
<html lang="en"     class="pb-page"  data-request-id="7f94ec37-533d-46f8-b7cd-df0a75f7856e"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/acs.jmedchem.0c01275;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2020.63.issue-20;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of Covalent Inhibitors Targeting the Transcriptional Enhanced Associate Domain Central Pocket" /></meta><meta name="dc.Creator" content="Hacer  Karatas" /></meta><meta name="dc.Creator" content="Mohammad  Akbarzadeh" /></meta><meta name="dc.Creator" content="Hélène  Adihou" /></meta><meta name="dc.Creator" content="Gernot  Hahne" /></meta><meta name="dc.Creator" content="Ajaybabu V.  Pobbati" /></meta><meta name="dc.Creator" content="Elizabeth  Yihui Ng" /></meta><meta name="dc.Creator" content="Stéphanie M.  Guéret" /></meta><meta name="dc.Creator" content="Sonja  Sievers" /></meta><meta name="dc.Creator" content="Axel  Pahl" /></meta><meta name="dc.Creator" content="Malte  Metz" /></meta><meta name="dc.Creator" content="Sarah  Zinken" /></meta><meta name="dc.Creator" content="Lara  Dötsch" /></meta><meta name="dc.Creator" content="Christine  Nowak" /></meta><meta name="dc.Creator" content="Sasikala  Thavam" /></meta><meta name="dc.Creator" content="Alexandra  Friese" /></meta><meta name="dc.Creator" content="CongBao  Kang" /></meta><meta name="dc.Creator" content="Wanjin  Hong" /></meta><meta name="dc.Creator" content="Herbert  Waldmann" /></meta><meta name="dc.Description" content="Transcriptional enhanced associate domain (TEAD) transcription factors together with coactivators and corepressors modulate the expression of genes that regulate fundamental processes, such as orga..." /></meta><meta name="Description" content="Transcriptional enhanced associate domain (TEAD) transcription factors together with coactivators and corepressors modulate the expression of genes that regulate fundamental processes, such as orga..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="September 10, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c01275" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2020 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01275" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c01275" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01275" /></link>
        
    
    

<title>Discovery of Covalent Inhibitors Targeting the Transcriptional Enhanced Associate Domain Central Pocket | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c01275" /></meta><meta property="og:title" content="Discovery of Covalent Inhibitors Targeting the Transcriptional Enhanced Associate Domain Central Pocket" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/large/jm0c01275_0011.jpeg" /></meta><meta property="og:description" content="Transcriptional enhanced associate domain (TEAD) transcription factors together with coactivators and corepressors modulate the expression of genes that regulate fundamental processes, such as organogenesis and cell growth, and elevated TEAD activity is associated with tumorigenesis. Hence, novel modulators of TEAD and methods for their identification are in high demand. We describe the development of a new “thiol conjugation assay” for identification of novel small molecules that bind to the TEAD central pocket. The assay monitors prevention of covalent binding of a fluorescence turn-on probe to a cysteine in the central pocket by small molecules. Screening of a collection of compounds revealed kojic acid analogues as TEAD inhibitors, which covalently target the cysteine in the central pocket, block the interaction with coactivator yes-associated protein with nanomolar apparent IC50 values, and reduce TEAD target gene expression. This methodology promises to enable new medicinal chemistry programs aimed at the modulation of TEAD activity." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c01275"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01275">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c01275&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c01275&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c01275&amp;href=/doi/10.1021/acs.jmedchem.0c01275" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 20</span><span class="cit-fg-pageRange">, 11972-11989</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/20" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c01148" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c01294" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of Covalent Inhibitors Targeting the Transcriptional Enhanced Associate Domain Central Pocket</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Hacer Karatas</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hacer Karatas</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemical Biology, Max Planck Institute of Molecular Physiology, Otto-Hahn Straße 11, 44227 Dortmund, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hacer++Karatas">Hacer Karatas</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7577-0320" title="Orcid link">http://orcid.org/0000-0002-7577-0320</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mohammad Akbarzadeh</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mohammad Akbarzadeh</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemical Biology, Max Planck Institute of Molecular Physiology, Otto-Hahn Straße 11, 44227 Dortmund, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mohammad++Akbarzadeh">Mohammad Akbarzadeh</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hélène Adihou</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hélène Adihou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemical Biology, AstraZeneca−Max Planck Institute Satellite Unit, Otto-Hahn-Strasse 11, 44227 Dortmund, Germany</div><div class="loa-info-affiliations-info">Medicinal Chemistry, Research and Early Development Cardiovascular, Renal and Metabolism, BioPharmaceuticals R&D, AstraZeneca, SE-431 83 Gothenburg, Sweden</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=H%C3%A9l%C3%A8ne++Adihou">Hélène Adihou</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Gernot Hahne</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gernot Hahne</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemical Biology, Max Planck Institute of Molecular Physiology, Otto-Hahn Straße 11, 44227 Dortmund, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gernot++Hahne">Gernot Hahne</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ajaybabu V. Pobbati</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ajaybabu V. Pobbati</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Molecular and Cell Biology, A*STAR, 61 Biopolis Drive, 138673 Singapore, Singapore</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ajaybabu+V.++Pobbati">Ajaybabu V. Pobbati</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5744-265X" title="Orcid link">http://orcid.org/0000-0001-5744-265X</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Elizabeth Yihui Ng</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Elizabeth Yihui Ng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Experimental Drug Development Centre (EDDC), Agency for Science, Technology and Research (A*STAR), 10 Biopolis Road, Chromos, #05-01, 138670, Singapore</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Elizabeth++Yihui+Ng">Elizabeth Yihui Ng</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Stéphanie M. Guéret</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stéphanie M. Guéret</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemical Biology, AstraZeneca−Max Planck Institute Satellite Unit, Otto-Hahn-Strasse 11, 44227 Dortmund, Germany</div><div class="loa-info-affiliations-info">Medicinal Chemistry, Research and Early Development Cardiovascular, Renal and Metabolism, BioPharmaceuticals R&D, AstraZeneca, SE-431 83 Gothenburg, Sweden</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=St%C3%A9phanie+M.++Gu%C3%A9ret">Stéphanie M. Guéret</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7220-9506" title="Orcid link">http://orcid.org/0000-0001-7220-9506</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sonja Sievers</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sonja Sievers</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemical Biology, Max Planck Institute of Molecular Physiology, Otto-Hahn Straße 11, 44227 Dortmund, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sonja++Sievers">Sonja Sievers</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Axel Pahl</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Axel Pahl</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemical Biology, Max Planck Institute of Molecular Physiology, Otto-Hahn Straße 11, 44227 Dortmund, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Axel++Pahl">Axel Pahl</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Malte Metz</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Malte Metz</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemical Biology, Max Planck Institute of Molecular Physiology, Otto-Hahn Straße 11, 44227 Dortmund, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Malte++Metz">Malte Metz</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sarah Zinken</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sarah Zinken</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemical Biology, Max Planck Institute of Molecular Physiology, Otto-Hahn Straße 11, 44227 Dortmund, Germany</div><div class="loa-info-affiliations-info">Faculty of Chemistry and Chemical Biology, Technical University Dortmund, Otto-Hahn-Straße 6, 44227 Dortmund, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sarah++Zinken">Sarah Zinken</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lara Dötsch</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lara Dötsch</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemical Biology, Max Planck Institute of Molecular Physiology, Otto-Hahn Straße 11, 44227 Dortmund, Germany</div><div class="loa-info-affiliations-info">Faculty of Chemistry and Chemical Biology, Technical University Dortmund, Otto-Hahn-Straße 6, 44227 Dortmund, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lara++D%C3%B6tsch">Lara Dötsch</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Christine Nowak</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christine Nowak</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemical Biology, Max Planck Institute of Molecular Physiology, Otto-Hahn Straße 11, 44227 Dortmund, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christine++Nowak">Christine Nowak</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sasikala Thavam</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sasikala Thavam</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemical Biology, Max Planck Institute of Molecular Physiology, Otto-Hahn Straße 11, 44227 Dortmund, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sasikala++Thavam">Sasikala Thavam</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Alexandra Friese</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alexandra Friese</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemical Biology, Max Planck Institute of Molecular Physiology, Otto-Hahn Straße 11, 44227 Dortmund, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alexandra++Friese">Alexandra Friese</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">CongBao Kang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">CongBao Kang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Experimental Drug Development Centre (EDDC), Agency for Science, Technology and Research (A*STAR), 10 Biopolis Road, Chromos, #05-01, 138670, Singapore</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=CongBao++Kang">CongBao Kang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9886-9374" title="Orcid link">http://orcid.org/0000-0002-9886-9374</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Wanjin Hong</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wanjin Hong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Molecular and Cell Biology, A*STAR, 61 Biopolis Drive, 138673 Singapore, Singapore</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wanjin++Hong">Wanjin Hong</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Herbert Waldmann</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Herbert Waldmann</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemical Biology, Max Planck Institute of Molecular Physiology, Otto-Hahn Straße 11, 44227 Dortmund, Germany</div><div class="loa-info-affiliations-info">Faculty of Chemistry and Chemical Biology, Technical University Dortmund, Otto-Hahn-Straße 6, 44227 Dortmund, Germany</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#bed6dbccdcdbccca90c9dfd2dad3dfd0d0fed3ced793dad1cccad3cbd0da90d3ced990dadb"><span class="__cf_email__" data-cfemail="dfb7baadbdbaadabf1a8beb3bbb2beb1b19fb2afb6f2bbb0adabb2aab1bbf1b2afb8f1bbba">[email protected]</span></a>. Phone: (+49)-231-133-2400. Fax: (+49)-231-133-2499.</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Herbert++Waldmann">Herbert Waldmann</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9606-7247" title="Orcid link">http://orcid.org/0000-0002-9606-7247</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01275&amp;href=/doi/10.1021%2Facs.jmedchem.0c01275" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 20</span><span class="cit-pageRange">, 11972–11989</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">September 10, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>22 July 2020</li><li><span class="item_label"><b>Published</b> online</span>10 September 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 22 October 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c01275" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01275</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice/index.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a><span class="article_header-cc-by text">with CC-BY<a href="/page/policy/authorchoice_ccby_termsofuse.html" title="License">license</a></span></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c01275"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">3570</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">5</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c01275" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of Covalent Inhibitors Targeting the Transcriptional Enhanced Associate Domain Central Pocket&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Hacer&quot;,&quot;last_name&quot;:&quot;Karatas&quot;},{&quot;first_name&quot;:&quot;Mohammad&quot;,&quot;last_name&quot;:&quot;Akbarzadeh&quot;},{&quot;first_name&quot;:&quot;Hélène&quot;,&quot;last_name&quot;:&quot;Adihou&quot;},{&quot;first_name&quot;:&quot;Gernot&quot;,&quot;last_name&quot;:&quot;Hahne&quot;},{&quot;first_name&quot;:&quot;Ajaybabu&quot;,&quot;last_name&quot;:&quot;V. Pobbati&quot;},{&quot;first_name&quot;:&quot;Elizabeth&quot;,&quot;last_name&quot;:&quot;Yihui Ng&quot;},{&quot;first_name&quot;:&quot;Stéphanie&quot;,&quot;last_name&quot;:&quot;M. Guéret&quot;},{&quot;first_name&quot;:&quot;Sonja&quot;,&quot;last_name&quot;:&quot;Sievers&quot;},{&quot;first_name&quot;:&quot;Axel&quot;,&quot;last_name&quot;:&quot;Pahl&quot;},{&quot;first_name&quot;:&quot;Malte&quot;,&quot;last_name&quot;:&quot;Metz&quot;},{&quot;first_name&quot;:&quot;Sarah&quot;,&quot;last_name&quot;:&quot;Zinken&quot;},{&quot;first_name&quot;:&quot;Lara&quot;,&quot;last_name&quot;:&quot;Dötsch&quot;},{&quot;first_name&quot;:&quot;Christine&quot;,&quot;last_name&quot;:&quot;Nowak&quot;},{&quot;first_name&quot;:&quot;Sasikala&quot;,&quot;last_name&quot;:&quot;Thavam&quot;},{&quot;first_name&quot;:&quot;Alexandra&quot;,&quot;last_name&quot;:&quot;Friese&quot;},{&quot;first_name&quot;:&quot;CongBao&quot;,&quot;last_name&quot;:&quot;Kang&quot;},{&quot;first_name&quot;:&quot;Wanjin&quot;,&quot;last_name&quot;:&quot;Hong&quot;},{&quot;first_name&quot;:&quot;Herbert&quot;,&quot;last_name&quot;:&quot;Waldmann&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;09&quot;,&quot;day&quot;:&quot;10&quot;,&quot;issue&quot;:&quot;20&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;11972-11989&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c01275&quot;},&quot;abstract&quot;:&quot;Transcriptional enhanced associate domain (TEAD) transcription factors together with coactivators and corepressors modulate the expression of genes that regulate fundamental processes, such as organogenesis and cell growth, and elevated TEAD activity is associated with tumorigenesis. Hence, novel modulators of TEAD and methods for their identification are in high demand. We describe the development of a new “thiol conjugation assay” for identification of novel small molecules that bind to the TEAD central pocket. The assay monitors prevention of covalent binding of a fluorescence turn-on probe to a cysteine in the central pocket by small molecules. Screening of a collection of compounds revealed kojic acid analogues as TEAD inhibitors, which covalently target the cysteine in the central pocket, block the interaction with coactivator yes-associated protein with nanomolar apparent IC50 values, and reduce TEAD target gene expression. This methodology promises to enable new medicinal chemistry programs aimed at t&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01275&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01275" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01275&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01275" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01275&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01275" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01275&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01275&amp;href=/doi/10.1021/acs.jmedchem.0c01275" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c01275" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c01275" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01275&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c01275%26sid%3Dliteratum%253Aachs%26pmid%3D32907324%26genre%3Darticle%26aulast%3DKaratas%26date%3D2020%26atitle%3DDiscovery%2Bof%2BCovalent%2BInhibitors%2BTargeting%2Bthe%2BTranscriptional%2BEnhanced%2BAssociate%2BDomain%2BCentral%2BPocket%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D20%26spage%3D11972%26epage%3D11989%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292225" title="Thiols">Thiols</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/20" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/jmcmar.2020.63.issue-20/20201022/jmcmar.2020.63.issue-20.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/medium/jm0c01275_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/large/jm0c01275_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01275&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Transcriptional enhanced associate domain (TEAD) transcription factors together with coactivators and corepressors modulate the expression of genes that regulate fundamental processes, such as organogenesis and cell growth, and elevated TEAD activity is associated with tumorigenesis. Hence, novel modulators of TEAD and methods for their identification are in high demand. We describe the development of a new “thiol conjugation assay” for identification of novel small molecules that bind to the TEAD central pocket. The assay monitors prevention of covalent binding of a fluorescence turn-on probe to a cysteine in the central pocket by small molecules. Screening of a collection of compounds revealed kojic acid analogues as TEAD inhibitors, which covalently target the cysteine in the central pocket, block the interaction with coactivator yes-associated protein with nanomolar apparent IC<sub>50</sub> values, and reduce TEAD target gene expression. This methodology promises to enable new medicinal chemistry programs aimed at the modulation of TEAD activity.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_60136" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_60136" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Transcriptional enhanced associate domain (TEAD) transcription factors play significant roles during development, tissue regeneration, cell growth, and migration and in human disease progression.<a onclick="showRef(event, 'ref1 ref2 ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4">(1−4)</a> Elevated TEAD expression has been observed in solid tumors such as prostate,<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> lung,<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> colorectal,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> gastric,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> and breast cancers.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> A number of TEAD target genes including cell surface receptor tyrosine kinase Axl,<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> connective tissue growth factor (CTGF),<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11,12)</a> apoptosis inhibitor survivin,<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> and tumor marker mesothelin<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> are also frequently associated with tumorigenesis. Therefore, TEAD transcription factors are potential therapeutic targets in cancer therapy.</div><div class="NLM_p">There are four TEAD homologues (TEAD1–4) in mammalian cells that are expressed in a tissue development stage-specific manner. TEAD1 is involved in cardiogenesis, while TEAD2 is critical for neural development and TEAD4 is necessary for embryo implantation.<a onclick="showRef(event, 'ref15 ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref15 ref16 ref17 ref18">(15−18)</a> Thus, pharmacological modulation of TEAD activity may enable novel medicinal chemistry approaches both in oncology and regenerative medicine.<a onclick="showRef(event, 'ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref19 ref20">(19,20)</a> Accordingly, the development of small-molecule and peptidic modulators of the TEAD function has been approached.<a onclick="showRef(event, 'ref1 ref20 ref21 ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref1 ref20 ref21 ref22 ref23">(1,20−23)</a></div><div class="NLM_p">TEAD homologues 1–4 share two highly conserved N- and C-terminal domains. The N-terminal DNA-binding domain forms a homeodomain and the C-terminal transactivation domain adopts a β-sandwich capped with a helix-turn-helix motif (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A, left panel, cyan).<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> The transactivation domain maintains the interaction with transcriptional comodulators, that is, coactivators yes-associated protein (YAP) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A, left panel, pale yellow), transcriptional coactivator with PDZ-binding motif (TAZ), vestigial-like (Vgll) family proteins, and corepressor Vgll4 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01275/suppl_file/jm0c01275_si_001.pdf" class="ext-link">Figure S1</a>).<a onclick="showRef(event, 'ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref24 ref25">(24,25)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/medium/jm0c01275_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/large/jm0c01275_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. hTEAD4 transactivation domain. (A) Structure of the TEAD transactivation domain (cyan) and the acylated cysteine residue in the central pocket (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5OAQ">5OAQ</a>). Cofactor YAP (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3KYS">3KYS</a>) is shown (pale yellow). The right panel is zoomed into the acylated cysteine residue (Cys367). (B) Small molecules that bind the TEAD central pocket.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/large/jm0c01275_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01275&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">TEADs contain a large hydrophobic central pocket within the globular β-sandwich to which the cofactors bind. The pocket embodies a conserved cysteine residue that can be acylated with palmitic or myristic acid (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A).<a onclick="showRef(event, 'ref26 ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref26 ref27 ref28">(26−28)</a> Recent studies suggest that TEAD acylation is dynamically regulated by auto-palmitoylation and depalmitoylases,<a onclick="showRef(event, 'ref26 ref29'); return false;" href="javascript:void(0);" class="ref ref26 ref29">(26,29)</a> pointing to a possibility of targeting the TEAD central pocket via the nonacylated form (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01275/suppl_file/jm0c01275_si_001.pdf" class="ext-link">Figure S2</a>).</div><div class="NLM_p">Indeed, the nonacylated hydrophobic central pocket can be targeted by small molecules that bind to the lipidation site (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B). Nonsteroidal anti-inflammatory drugs flufenamic acid (<b>1</b>, <i>K</i><sub>d</sub> = 73 μM) and niflumic acid (<b>2</b>, <i>K</i><sub>d</sub> = 28 μM) bind to the TEAD central pocket and inhibit expression of TEAD–YAP target genes and growth of cancer cell lines.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> Furthermore, a chloromethylketone analogue of flufenamic acid (<b>3</b>, <b>TED-347</b>) was developed as a covalent TEAD inhibitor that can allosterically inhibit the TEAD–YAP interaction (<i>K</i><sub>i</sub> = 10.4 μM).<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Another covalent TEAD inhibitor (<b>4</b>, <b>K-975</b>) showed efficacy in human malignant pleural mesothelioma xenograft models.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> More recently, a vinylsulfonamide derivative (<b>5</b>, <b>DC-TEADin02</b>) was developed via structure-based virtual screening and was shown to covalently bind to TEAD in cells transfected with Flag-TEAD4.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Triazole <b>6</b> efficiently inhibits TEAD palmitoylation in intestinal epithelium in vivo,<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> and compound <b>7</b> was identified via screening a DNA-ecoded library with indole-focused Ugi-peptidomimetics.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Furthermore, the hydroxyquinoline analogue <b>8</b> was reported to bind the central pocket, however, it activates the TEAD function.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> A recently identified TEAD palmitoylation inhibitor (<b>9</b>) mimics the palmitate-binding mode, activates TEAD to act as a transcriptional repressor independent from YAP, and inhibits tumor growth in a xenograft model.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> Overall, the number and diversity of small molecules that target the TEAD central pocket has been growing, and any methodology that will aid the development of novel TEAD modulators will be invaluable.</div><div class="NLM_p last">We describe the development of a new methodology for the identification of novel small-molecule inhibitors binding to the TEAD central pocket. By means of a newly developed “thiol conjugation assay” we identified kojic acid derivatives that covalently target the palmitoylatable cysteine in the TEAD central pocket and inhibit the TEAD–YAP interaction with nanomolar apparent IC<sub>50</sub> values. By means of a representative analogue, we have shown the TEAD engagement in cell lysates and the inhibition of the expression of the TEAD target gene ANKRD1. The novel methodology in general promises to enable new medicinal chemistry programs aimed at the discovery of covalent or noncovalent TEAD inhibitor classes.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_08577" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_08577" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Development of a “Thiol Conjugation Assay”</h3><div class="NLM_p">In order to develop a robust, homogeneous, and rapid binding assay for the TEAD central pocket, we utilized the thiol-reactive profluorescent probe, <i>N</i>-(4-(7-diethylamino-4-methylcoumarin-3-yl)phenyl)maleimide (CPM) that has been used for site-specific protein labeling, thermal stability analysis of membrane proteins, and assaying <i>N</i>-acyltransferase activity.<a onclick="showRef(event, 'ref38 ref39 ref40 ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref38 ref39 ref40 ref41 ref42">(38−42)</a> The fluorescence in CPM is quenched because of the maleimide substitution on the phenyl group that modulates the resonance between the coumarin carbonyl and 7-amino groups. However, upon reaction with a thiol, CPM fluorescence increases strongly (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A).<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> We hypothesized that the reaction of a free cysteine residue in the TEAD central pocket would yield a fluorescence signal and that small molecules binding to the TEAD central pocket with appreciable potency would prevent covalent labeling of the cysteine by CPM.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/medium/jm0c01275_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/large/jm0c01275_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Targeting the central pocket with small-molecule inhibitors. (A) Thiol conjugation assay principle. The free cysteine residue in the TEAD central pocket reacts with CPM yielding a fluorescence signal. (B) Kinetic measurement of hTEAD4 (nonacylated) treatment with CPM (purple square) yields a fluorescence signal, while CPM alone (black square) has only a minimal signal. (C) Competitive inhibition of hTEAD4 (nonacylated) CPM with known noncovalent central pocket binders. (D) Selected screening hits (<b>10–12</b>) and inactive analogues (<b>13–15</b>) with similar structures.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/large/jm0c01275_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01275&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">For assay development, hTEAD4 was delipidated with hydroxylamine to cleave the covalent thioester bond between the fatty acid and the cysteine (Cys367 for hTEAD4, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A right panel). Delipidation was monitored and confirmed with whole-protein electron spray ionization (ESI)–mass spectrometry (MS) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01275/suppl_file/jm0c01275_si_001.pdf" class="ext-link">Figure S3</a>). Through expression, a mixture of the three recombinant<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> nonacylated hTEAD4 (MW: 26,495) and hTEAD4 covalently bound to either myristic acid (C14-fatty acid, MW: 26,705) or palmitic acid (C16-fatty acid, MW: 26,733) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01275/suppl_file/jm0c01275_si_001.pdf" class="ext-link">Figure S3A</a>) was converted to a homogeneous nonacylated hTEAD4 protein (MW: 26,495) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01275/suppl_file/jm0c01275_si_001.pdf" class="ext-link">Figure S3B</a>).</div><div class="NLM_p">Addition of the nonacylated hTEAD4 to 1 equivalent CPM yielded a significant increase in the fluorescence signal, which reached a plateau in 30–40 min (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). Analysis of the reaction mixture with ESI–MS indicated that two cysteine residues might have been labeled with CPM (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01275/suppl_file/jm0c01275_si_001.pdf" class="ext-link">Figure S4A</a>). Subsequent tandem MS (MS–MS) analysis showed that Cys367 and Cys330, both of which are located in the hTEAD4 central pocket, were covalently modified with CPM (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01275/suppl_file/jm0c01275_si_001.pdf" class="ext-link">Figures S4B and S5</a>). These findings suggested that monitoring of CPM binding to the TEAD central pocket and its prevention by small molecules could be used to identify novel TEAD central pocket-targeting inhibitors.</div><div class="NLM_p last">For validation, we investigated inhibition of CPM binding and fluorescence decrease by the known TEAD central pocket binders, flufenamic acid (<b>1</b>), niflumic acid (<b>2</b>), palmitic acid, and the covalent inhibitor <b>3</b> (TED-347). As shown in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C for noncovalent binders and in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01275/suppl_file/jm0c01275_si_001.pdf" class="ext-link">Figure S6</a> for TED-347, these compounds inhibited the signal increase in a dose-dependent manner. To our best knowledge, this is the first example that CPM is used for a binding assay. Because a cysteine thiol is necessary in the reaction with CPM, we termed this novel assay “Thiol Conjugation Assay”.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Discovery of Small-Molecule Inhibitors of the TEAD Central Pocket</h3><div class="NLM_p">Investigation of a collection of 14,000 compounds using the newly established thiol conjugation assay identified 329 compounds, which showed >50% inhibition at a concentration of 12.5 μM (2.3% primary hit rate). Kojic acid analogues <b>10–12</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>D) were among the most potent hits. Analogues <b>13–15</b> lacking aryl substituents on the methyl amine, however, were inactive (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>D) such that for initial structure–activity relationship (SAR) investigations, a phenyl group at both R<sup>1</sup> and R<sup>3</sup> positions was retained (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). Compounds <b>19–44</b> were synthesized using a one-pot Betti reaction<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> starting from kojic acid (<b>16</b>), amines <b>17</b>, and aldehydes <b>18</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). In addition, the 2-hydroxyl group in compounds <b>19</b> and <b>10</b> was methylated to yield compounds <b>45</b> and <b>46</b>, respectively.</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/medium/jm0c01275_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/large/jm0c01275_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Kojic Acid Analogues<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/large/jm0c01275_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01275&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) EtOH, rt, 2–20 days. (b) MeI, K<sub>2</sub>CO<sub>3</sub>, DMF, 24 h, rt.</p></p></figure><div class="NLM_p">Direct binding of compounds <b>19–46</b> to the TEAD central pocket was monitored by means of a fluorescence polarization (FP)-based competitive binding assay employing a fluorescein isothiocyanate (FITC)-labeled palmitate tracer (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01275/suppl_file/jm0c01275_si_001.pdf" class="ext-link">Scheme S1</a>), which bound with <i>K</i><sub>d</sub> = 28 nM to nonacylated hTEAD4 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01275/suppl_file/jm0c01275_si_001.pdf" class="ext-link">Figure S7A</a>). The FP assay was validated employing the previously reported competitive inhibitor niflumic acid (<b>2</b>) as a positive control (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01275/suppl_file/jm0c01275_si_001.pdf" class="ext-link">Figure S7B</a>).</div><div class="NLM_p">IC<sub>50</sub> values obtained from the thiol conjugation assay and the FP-based competitive binding assay are shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. Not surprisingly, because of the different assay principles (i.e., competition with the covalent cysteine binder CPM <i>vs</i> competition with noncovalent binder FITC-palmitate) and protein concentrations, the IC<sub>50</sub> values obtained from the FP and thiol conjugation assays differ for all kojic acid analogues, consistently 5–10-fold.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. SAR Studies of the Kojic Acid Analogues</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/medium/jm0c01275_0009.gif" alt="" id="GRAPHIC-d7e790-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/medium/jm0c01275_0010.gif" alt="" id="gr8" /></img><div></div></div><div class="NLM_p">Initial SAR investigations focused on modifications of amine substituents R<sup>1</sup> and R<sup>2</sup> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Replacement of the phenyl group in <b>19</b> by an isopropyl group (<b>20</b>) led to a sevenfold increase in IC<sub>50</sub> values for both assays. Although a benzyl group (<b>21</b>) was slightly (twofold) less favorable than the phenyl group (<b>19</b>), the introduction of the constrained bis-benzyl moiety in <b>22</b> lowered the affinity nearly fivefold. A fluorine group at the meta-position of R<sup>1</sup> (<b>23</b>) did not influence TEAD binding, and introduction of an electron-deficient 2-pyridine ring (<b>24</b>) gave a similar IC<sub>50</sub> value as for a phenyl group (<b>19</b>). Substitution of the 2-pyridine ring (<b>25</b>) with a methyl group on the 5-position also did not alter the binding. However, the introduction of 2-pyrimidine- (<b>26</b>), 2-thiazole- (<b>27</b>), and 2-benzimidazole (<b>28</b>) groups resulted in more than 20-fold increase in the IC<sub>50</sub> values. Equipment of the phenyl ring at R<sup>1</sup> with an electron-withdrawing 4-carboxamide (<b>29</b>) did not influence the affinity.</div><div class="NLM_p">Replacement of the benzylic phenyl group at the R<sup>3</sup> position with an aliphatic cyclohexyl (<b>30</b>) or isopropyl group (<b>31</b>) or with a hydrogen (<b>32</b>) dramatically (100-fold) reduced TEAD binding in the thiol conjugation assay showing the importance of an aromatic residue at the R<sup>3</sup> position. Introduction of an electron-withdrawing carboxamide substituent at the para-position (<b>33</b>) slightly (twofold) improved binding. Hydrophobic substituents at the para-position (<b>34–38</b>) did not change binding significantly, and similarly, introduction of a fluorine group (<b>39</b>) at the ortho-position was tolerated. Replacement of the phenyl ring with an electron-deficient 2-pyridine group (<b>40</b>), however, lowered the inhibitory activities by 2–3-fold in the FP assay.</div><div class="NLM_p">Additional analogues with 2-pyridine residues at R<sup>1</sup> and R<sup>3</sup> (<b>41–44</b>), in particular, the combination of two 2-pyridines at both R<sup>1</sup> and R<sup>3</sup> positions (<b>43</b>), lowered the affinity significantly.</div><div class="NLM_p">The importance of a 2-OH group at the R<sup>4</sup> position was proven by investigation of compounds <b>45</b> and <b>46</b>, that is, the 2-OMe analogues of compounds <b>19</b> and <b>10</b>, respectively. Both compounds were inactive in the thiol conjugation assay and had >200-fold lower affinity in the FP assay, showing the critical influence of the 2-OH group for TEAD binding (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A–C).</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/medium/jm0c01275_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/large/jm0c01275_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Target engagement studies with compound <b>19</b> and its inactive analogue <b>45</b> in cell-free systems. (A) Structures of the compounds <b>19</b> and <b>45</b>. Representative inhibition curves in the (B) thiol conjugation assay and (C) FP-based competitive binding assay. (D) Differential scanning fluorimetry measurements of hTEAD4 (nonacylated) with selected derivatives.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/large/jm0c01275_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01275&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">We additionally confirmed the TEAD binding of selected analogues (<b>19</b>, <b>27</b>, <b>42</b>, and <b>45</b>) using differential scanning fluorimetry (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D). Compound <b>19</b> lowered the melting point (<i>T</i><sub>m</sub>) of nonacylated hTEAD4 by 5 °C, and its 2-pyrido (R<sup>1</sup>), 4-bromo (R<sup>3</sup>) analogue (<b>42</b>) induced a smaller <i>T</i><sub>m</sub> shift (−2 °C) aligning with the binding experiments (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Consistent with the competitive binding experiments, compounds <b>27</b> and <b>45</b> did not induce any change in the <i>T</i><sub>m</sub> of nonacylated hTEAD4.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Covalent Binding to the TEAD Central Pocket</h3><div class="NLM_p">In light of the observation that a nonmodified, acidic 2-OH group in kojic acid (p<i>K</i><sub>a</sub> = 7.9)<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> is required for activity, we reasoned that the compounds of type <b>19</b> might undergo a retro-Mannich reaction to yield a Michael acceptor, which then could covalently react with Cys367 in nonacylated hTEAD4 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A). In agreement with this hypothesis, methylation of the 2-OH group (<b>45</b> and <b>46</b>) led to compounds inactive in the thiol conjugation assay (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Indeed, treatment of nonacylated hTEAD4 with 8 equiv of compound <b>19</b> or <b>45</b> and subsequent analysis of the mixture with whole-protein ESI–MS spectrometry revealed that—consistent with our hypothesis—modification of the protein (+230 Da) was observed upon incubation with <b>19</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A–B), while modification of TEAD was not observed upon coincubation with <b>45</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C).</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/medium/jm0c01275_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/large/jm0c01275_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Mechanism of action studies with kojic acid analogues. (A) Hypothesized mechanism of action of compound <b>19</b>. Whole-protein ESI–MS analysis of hTEAD4 (nonacylated) treatment with kojic acid analogue (B) compound <b>19</b> and (C) compound <b>45</b>. Compound <b>19</b> is a covalent binder for nonacylated hTEAD4, and the −OH group at the second position is required for the predicted reaction with a thiol group. The −OMe analogue (<b>45</b>) is not a covalent TEAD binder. (D) Whole-protein ESI–MS analysis of hTEAD4 (acylated) incubation with <b>19</b>. No protein adduct with compound <b>19</b> was observed when Cys367 in hTEAD4 is lipidated. (E) Comparison of thiol conjugation inhibition with 10 μM compound <b>19</b> in the presence of hTEAD4 (nonacylated), hTEAD4 (acylated), or Cys to Ala/Ser mutants of mTEAD4. When the cysteine residue is replaced with alanine or serine, compound <b>19</b> cannot inhibit the thiol conjugation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/large/jm0c01275_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01275&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The hypothesis that compound <b>19</b> specifically targets Cys367 that is lipidated under physiological conditions was further strengthened by the observation that incubation of <b>19</b> with acylated (i.e., not treated with hydroxylamine) hTEAD4 did not result in the formation of an acylated hTEAD4-<b>19</b> adduct (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D and <a href="/doi/suppl/10.1021/acs.jmedchem.0c01275/suppl_file/jm0c01275_si_001.pdf" class="ext-link">S3A</a>).</div><div class="NLM_p">In addition, mutants of mouse TEAD4 (mTEAD4) in which the reactive cysteine was replaced with alanine or serine and that should not react with compound <b>19</b> were investigated in the thiol conjugation assay. As shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>E, compound <b>19</b> cannot inhibit an increase in the fluorescence signal when the Ser or Ala mutant of mTEAD4 is used. Collectively, these observations show that kojic acid analogue <b>19</b> covalently and specifically binds nonacylated TEAD4.</div><div class="NLM_p">To confirm the covalent reaction between compound <b>19</b> and nonacylated hTEAD4, thiol conjugation and FP assays were used to determine the time-dependent inhibition (TDI) of hTEAD4. Compound <b>19</b> showed a low-micromolar inhibition constant (<i>K</i><sub>i</sub> = 2.6 μM), which describes the potency of reversible binding between the compound and the protein before the covalent bond formation (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01275/suppl_file/jm0c01275_si_001.pdf" class="ext-link">Figure S8A,B</a>). Compound <b>19</b> inhibited nonacylated hTEAD4 with <i>k</i><sub>inact</sub>/<i>K</i><sub>i</sub> = 8.3 μM in the thiol conjugation assay and with <i>k</i><sub>inact</sub>/<i>K</i><sub>i</sub> = 4.0 μM in the FP assay (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01275/suppl_file/jm0c01275_si_001.pdf" class="ext-link">Figure S8C,D</a>).</div><div class="NLM_p last">Investigation of the specificity of compound <b>19</b> for binding to TEAD homologues by means of the thiol conjugation assay showed that <b>19</b> inhibits hTEAD1–4 with similar low-micromolar IC<sub>50</sub> values (2.6–6.9 μM, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01275/suppl_file/jm0c01275_si_001.pdf" class="ext-link">Figure S9</a>) consistent with the high homology (72.0–87.9%)<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> of the transactivation binding domains of TEAD1–4.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Binding Mode Analysis</h3><div class="NLM_p last">To get insight into the binding mode and binding site of the new covalent inhibitors, <sup>1</sup>H–<sup>15</sup>N HSQC spectra of mTEAD4 in the absence and presence of compound <b>19</b> or <b>45</b> were recorded, and signals were assigned based on the corresponding analysis<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> of the transactivation domain of mTEAD4. Compound <b>19</b> induced line broadening of several residues (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01275/suppl_file/jm0c01275_si_001.pdf" class="ext-link">Figure S10A</a>), including Phe386 (F386) and Phe408 (F408). These amino acids are in close proximity to the palmitoylation site. Conversely, the negative control <b>45</b> did not induce a significant change compared to the dimethyl sulfoxide (DMSO) control (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01275/suppl_file/jm0c01275_si_001.pdf" class="ext-link">Figure S10B</a>). These findings support the biochemical data suggesting that compound <b>19</b> binds to the central pocket in TEAD.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Covalent TEAD Binders Inhibit the TEAD–YAP Interaction</h3><div class="NLM_p">The chloromethyl ketone flufenamic acid analogue (<b>3</b>, TED-347) that binds the TEAD central pocket covalently allosterically inhibits TEAD and YAP interaction and consequently YAP-mediated target gene expression.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> In order to determine whether the new inhibitor class can also disrupt this protein–protein interaction, an FP assay that monitors the binding of the fluorescently labeled YAP peptide to nonacylated hTEAD4 was used. Preincubation with compound <b>19</b> for 10 min potently inhibited the interaction of nonacylated hTEAD4 with human YAP<sup>50–100</sup> with an IC<sub>50</sub> value of 70 nM. In contrast and in agreement with the covalent inhibition mechanism proposed above, the OMe analogue <b>45</b> had >100-fold lower activity (IC<sub>50</sub> = 9.5 μM) (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A) and covalent inhibitor <b>19</b> did not inhibit the interaction of YAP with acylated hTEAD4 and mTEAD4 with Cys360Ser or Cys360Ala mutations (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B–D) (saturation binding curves of FITC-YAP<sup>50–100</sup> with the TEAD4 proteins are shown in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01275/suppl_file/jm0c01275_si_001.pdf" class="ext-link">Figure S11A–D</a>). Overall, these results are in accordance with previous findings<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> for covalent allosteric inhibition of TEAD and show that the covalent and potent TEAD central pocket-binding small molecules identified here can block the TEAD–YAP interaction efficiently.</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/medium/jm0c01275_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/large/jm0c01275_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Inhibition of TEAD–YAP interaction as determined with an FP-based competitive binding assay using FITC-labeled hYAP<sup>50–100</sup>. Compound <b>19</b> (blue triangle) and compound <b>45</b> (black square) were tested. (A) Compound <b>19</b> can prevent YAP interaction with hTEAD4 (nonacylated) but not with (B) hTEAD4 (acylated), mTEAD4 with (C) Cys360Ser, or (D) Cys360Ala mutation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/large/jm0c01275_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01275&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> TEAD Engagement in a Cellular Environment</h3><div class="NLM_p">Target engagement in HCT116 cell lysates was proven by means of a cellular thermal shift assay (CETSA). Treatment of the lysates with compound <b>19</b> for 10 min increased the melting temperature of hTEAD4 from 49.8 to 53.1 °C (Δ<i>T</i><sub>m</sub> = 3.3 °C), providing evidence for the interaction with hTEAD4 in a biologically relevant environment (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A,B).</div><div class="NLM_p">TEADs, together with the coactivators (e.g., YAP), regulate the expression of genes involved in critical processes such as cardiogenesis, cell growth, and migration.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Small molecules that bind the TEAD central pocket have been shown to modulate the TEAD transcriptional activity, such as inhibition of the expression of well-established TEAD target genes (i.e., ANKRD1, CTGF, and CYR61).<a onclick="showRef(event, 'ref34 ref37'); return false;" href="javascript:void(0);" class="ref ref34 ref37">(34,37)</a> To evaluate the influence of kojic acid analogues on TEAD transcriptional activity, HEK293 cells were incubated with compound <b>19</b>, negative control <b>45</b>, or TED-347. Treatment with compound <b>19</b> for 6 h resulted in 90% reduction of the mRNA level of the TEAD target gene ANKRD1 compared to the DMSO control, while compound <b>45</b> had only a minimal effect at the same concentration, further supporting the on-target activity of the kojic acid analogues (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C).</div><figure id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/medium/jm0c01275_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/large/jm0c01275_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. TEAD inhibitory activity of <b>19</b> in a cellular environment. (A,B) Target engagement in HCT116 cell lysates. (A) Representative immunoblots of hTEAD4 treated with a vehicle control (DMSO) or compound <b>19</b>. (B) CETSA melting curves of hTEAD4 treated with a vehicle control (DMSO) or compound <b>19</b>. (C) Inhibition of the TEAD target gene ANKRD1 expression by compound <b>19</b> in comparison with the control compounds <b>45</b> and TED-347.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/large/jm0c01275_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01275&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51382" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51382" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">We have developed a new “thiol conjugation assay” that enables the systematic identification of small-molecule inhibitors of the TEAD central pocket. By means of this assay, kojic acid analogues were identified as a new class of TEAD inhibitors. Our data show that these inhibitors covalently bind to the Cys residue in the TEAD central pocket that is S-acylated under physiological conditions. The representative kojic acid analogue <b>19</b> inhibited the interaction of YAP with nonacylated hTEAD4 but not with the corresponding acylated protein and mutants, in which the Cys had been replaced by a serine or an alanine. Furthermore, CETSA in lysate provided evidence for compound <b>19</b> interacting with hTEAD4 in a biologically relevant medium. Consistent with the binding experiments, kojic acid analogue <b>19</b> inhibited TEAD target gene expression. Taken together, we have established a novel methodology that could be a guide for the development of novel classes of TEAD inhibitors interacting with the protein covalently or noncovalently.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_06857" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_06857" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Plasmids</h3><div class="NLM_p last">hTEAD4 plasmid was isolated from the cDNA clone of hTEAD4 (Dharmacon, clone ID: 3913870). The plasmid with mTEAD4 with a Cys360Ala mutant was obtained through site-directed mutagenesis using an mTEAD4 wild-type template (Dharmacon, clone ID: 40142595). All plasmids were verified by DNA sequencing.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Cloning, Expression, and Purification of Recombinant Proteins</h3><div class="NLM_p">Sequences of the proteins used in the study are shown in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01275/suppl_file/jm0c01275_si_001.pdf" class="ext-link">Table S1</a>.</div><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> N-His hTEAD4</h4><div class="NLM_p last">The plasmid of N-terminal His<sub>6</sub>-tagged hTEAD4<sup>217–434</sup> was subcloned into a pOPIN-neo vector, which was used to transform <i>Escherichia coli</i> BL21-CodonPlus (DE3)-RIPL (Agilent, 230280). The transformed cells were grown in the TB medium supplemented with 0.01% α-lactose monohydrate, 2 mM MgSO<sub>4</sub>, 100 μg/mL ampicillin, and 50 μg/mL chloramphenicol at 37 °C for 4 h. Then, the temperature was reduced to 25 °C and the culture was incubated for a further 20–24 h. The cells were harvested, and the pellets were taken in a lysis buffer containing 50 mM HEPES, 300 mM NaCl, 20 mM imidazole, and 1 mM TCEP, pH 8 and lysed by sonication, followed by ultracentrifugation. Then, the cleared lysate was loaded onto the ÄktaXpress (GE Healthcare) for affinity purification (His trap FF, GE Healthcare) which was followed by size exclusion chromatography (SD75 26/60, GE Healthcare) using 20 mM HEPES, 100 mM NaCl, 1 mM TCEP, 2 mM MgCl<sub>2</sub>, and 5% glycerol, pH 8. The purified protein was concentrated using Amicon Ultra Centrifugal Filters, 10K (Merck), and aliquots were snap-frozen and stored at −80 °C.</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> N-His mTEAD4_Cys360Ser</h4><div class="NLM_p last">A method provided earlier was used for the protein expression using 1 mM IPTG.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> The harvested cell pellets were lysed, and the protein was purified as described above for N-His hTEAD4.</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> N-His-MBP mTEAD4_Cys360Ala</h4><div class="NLM_p last">cDNA of N-terminal His<sub>6</sub>- and MBP-tagged mTEAD4<sup>209–427</sup> with Cys360Ala mutation was used to transform <i>E. coli</i> BL21-CodonPlus (DE3)-RIPL (Agilent, 230280). The protein was expressed, and the collected pellets were lysed as described for N-His hTEAD4 above. The cleared lysate was loaded onto the ÄktaXpress (GE Healthcare) for affinity purification (His trap FF, GE Healthcare) which was followed by size exclusion chromatography (SD200 16/60, GE Healthcare) using 25 mM HEPES, 150 mM NaCl, and 1 mM TCEP, pH 7.2. The purified protein was concentrated using Amicon Ultra Centrifugal Filters, 50K (Merck), and aliquots were snap-frozen and stored at −80 °C. The storage buffer was exchanged with pH 8 Tris-HCl (20 mM), NaCl (100 mM), and MgCl<sub>2</sub> (2 mM) using Amicon Ultra Centrifugal Filters (10K) for the binding studies.</div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> N-His-GST hTEAD1</h4><div class="NLM_p last">The plasmid of N-terminal His<sub>6</sub>-tagged GST-hTEAD1<sup>209–426</sup> was subcloned into a pOPIN-neo vector, which was used to transform <i>E. coli</i> BL21-CodonPlus (DE3)-RIPL (Agilent, 230280). The transformed cells were grown in TB medium supplemented with 2 mM MgSO<sub>4</sub>, 100 μg/mL ampicillin, and 50 μg/mL chloramphenicol at 37 °C until the optical density reached 0.6. Then, the temperature was reduced to 18 °C and the culture was incubated for a further 16 h. The cells were harvested, and the pellets were taken in a lysis buffer containing 50 mM HEPES, 300 mM NaCl, 20 mM imidazole, and 1 mM TCEP, pH 8 and lysed by sonication, followed by ultracentrifugation. Then, the cleared lysate was loaded onto the ÄktaXpress (GE Healthcare) for affinity purification (His trap FF, GE Healthcare) which was followed by size exclusion chromatography (SD75 26/60, GE Healthcare) using 20 mM HEPES, 100 mM NaCl, 1 mM TCEP, 2 mM MgCl<sub>2</sub>, and 5% glycerol, pH 8. The purified protein was concentrated using Amicon Ultra Centrifugal Filters, 10K (Merck), and aliquots were snap-frozen and stored at −80 °C.</div></div><div id="sec4_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> N-His hTEAD2</h4><div class="NLM_p last">The plasmid of N-terminal His<sub>6</sub>-tagged hTEAD4<sup>217–447</sup> was subcloned and the protein was expressed as described above for N-His hTEAD4.</div></div><div id="sec4_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> N-His hTEAD3</h4><div class="NLM_p last">N-terminal His<sub>6</sub>-tagged hTEAD4<sup>216–434</sup> was expressed using the protocol described for hTEAD4 earlier.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a></div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Deacylation (Depalmitoylation) of TEAD</h3><div class="NLM_p">To 500 μL of 105 μM (2.8 mg/mL) recombinant hTEAD4 (acylated) in an Eppendorf tube, 30 μL of freshly prepared 105 mM hydroxyl amine solution was added slowly and the mixture was gently shaken at room temperature for 3–4 h. The deacylation process was monitored with whole-protein ESI–MS. The protein was then purified over two consecutive size exclusion columns (i) Amicon, 0.5 mL 10K (Merck) and (ii) Zebaspin, 7K (Thermo Fisher) using Tris (20 mM pH 8), NaCl (100 mM), and MgCl<sub>2</sub> (2 mM) as the final storage buffer. Aliquots (5 μL) were snap-frozen and stored at −80 °C.</div><div class="NLM_p last">4 M hydroxylamine solution: 0.46 g (6.67 mmol) of hydroxylamine HCl was dissolved in 0.53 mL of H<sub>2</sub>O in a glass vial. Then, 0.33 mL of diethylamine was added dropwise, followed by additional 0.2 diethylamine and 0.63 mL H<sub>2</sub>O addition. Freshly prepared 4 M hydroxylamine solution was further diluted to 105 mM with Milli-Q water.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Binding Assays</h3><div class="NLM_p">All the experiments were performed using black, low-volume, round-bottom, nonbinding, 384-well plates (Corning 4514). The plates were imaged using Tecan Spark. Spectramax Paradigm (Molecular Devices) was used for imaging the plates during low-throughput screening. Stock solutions (10 mM) of the compounds in DMSO were used. The <i>K</i><sub>d</sub> of the tracers and the IC<sub>50</sub> value of the inhibitors were calculated using GraphPad Prism 7 software.</div><div id="sec4_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> Binding Assays for the TEAD Central Pocket</h4><div id="sec4_4_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i25" class="anchor-spacer"></div><h5 class="article-section__title" id="_i25"> Thiol Conjugation Assay</h5><div class="NLM_p last">To serially diluted solutions of the tested compounds in 5 μL of assay buffer [HEPES (25 mM) and NaCl (150 mM) pH 6.5], freshly prepared 5 μL of 0.45 μM protein solution was added and incubated at room temperature for 10 min, followed by addition of 5 μL of 0.45 μM CPM solution, giving a 15 μL final assay volume, 150 nM protein, and 150 nM CPM. The assay plate was shaken at room temperature for 1 h before measuring the fluorescence (Ex/Em: 380/470 nm). Compound <b>19</b> was used as an internal standard to determine 100% inhibition.</div></div><div id="sec4_4_1_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i26" class="anchor-spacer"></div><h5 class="article-section__title" id="_i26"> Thiol Conjugation Assay for Low-Throughput Screening</h5><div class="NLM_p last">A library of 14,000 compounds was screened in 1536-well plates using pH 6.5 MES buffer. To the preincubated solution of nonacylated hTEAD4 (0.15 μM final concentration) and compound solution (12.5 μM final concentration), CPM (0.15 μM final concentration) was added, and the assay plates were incubated at room temperature for 50 min before measuring the fluorescence (Ex/Em: 360/465 nm) using Spectramax Paradigm (Molecular Devices). <i>Z</i>′ = 0.6–0.8 (10-plates).</div></div><div id="sec4_4_1_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i27" class="anchor-spacer"></div><h5 class="article-section__title" id="_i27"> Saturation Binding Experiment</h5><div class="NLM_p last">To a serially diluted solution of nonacylated hTEAD4 in 12 μL of assay buffer [HEPES (25 mM) and NaCl (150 mM) pH 6.5], 8 μL of 25 nM FITC-palmitate solution was added, giving 20 μL final assay volume with a 10 nM tracer. The assay plate was incubated at room temperature for 1 h before measuring FP (Ex/Em: 485/535 nM).</div></div><div id="sec4_4_1_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i28" class="anchor-spacer"></div><h5 class="article-section__title" id="_i28"> FP-Based Competitive Inhibition Experiments</h5><div class="NLM_p last">To serially diluted solution of the tested compounds in 5 μL assay buffer [HEPES (25 mM) and NaCl (150 mM), pH 6.5 buffer], freshly prepared 5 μL of 0.15 μM protein solution was added and incubated at room temperature for 10 min followed by 5 μL of 30 nM FITC-palmitate solution, giving 15 μL final assay volume, 50 nM protein, and 10 nM tracer. The assay plate was shaken at room temperature for 1 h before measuring the FP (Ex/Em: 485/535 nm).</div></div></div><div id="sec4_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> Binding Assays for TEAD–YAP Interaction</h4><div class="NLM_p">All the TEAD–YAP binding experiments were performed in phosphate-buffered saline (PBS) (pH 7.5).</div><div id="sec4_4_2_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i30" class="anchor-spacer"></div><h5 class="article-section__title" id="_i30"> Saturation Binding Experiments</h5><div class="NLM_p last">To a serially diluted solution of the protein in 12 μL PBS, 8 μL of FAM-YAP solution was added, giving 20 μL final assay volume. The assay plate was incubated at room temperature for 1 h before measuring FP (Ex/Em: 485/530 nM). The final FAM concentrations are as follows: hTEAD4 (nonacylated): 2 nM, hTEAD4 (acylated): 2 nM, mTEAD4_Cys360Ser: 10 nM, and mTEAD4_Cys360Ala: 10 nM.</div></div><div id="sec4_4_2_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i31" class="anchor-spacer"></div><h5 class="article-section__title" id="_i31"> FP-Based Competitive Inhibition Experiments</h5><div class="NLM_p last">To serially diluted solution of the tested compounds in 5 μL of PBS, freshly prepared 5 μL of protein solution was added and incubated at room temperature for 10 min followed by addition of 5 μL of FAM-YAP<sup>50–100</sup> solution. The assay plate was shaken at room temperature for 1 h before measuring the FP (Ex/Em: 485/535 nm). Final protein and FAM concentrations are as follows (FAM/protein): hTEAD4 (nonacylated): 2 nM/5 nM, hTEAD4 (acylated): 3 nM/15 nM, mTEAD4_Cys360Ser: 10 nM/50 nM, and mTEAD4_Cys360Ala: 10 nM/50 nM.</div></div><div id="sec4_4_2_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i32" class="anchor-spacer"></div><h5 class="article-section__title" id="_i32"> Inhibition Efficiency Experiments</h5><div class="NLM_p last">TDI of hTEAD4 (nonacylated) by compound <b>19</b> was monitored in FP (central pocket binding) and thiol conjugation assays at 5–7 different concentrations (0.15–40 μM) following 1, 2, 4, and 24 h incubation at 4 °C (mean ± SD; <i>n</i> = 2). By fitting the percent of inhibition versus preincubation time at different concentrations, <i>k</i><sub>obs</sub> was calculated. Next, <i>k</i><sub>obs</sub> was plotted against concentration and was further fitted to a nonlinear regression model to determine the kinetic parameters of covalent inhibition of hTEAD4, where plateau is showing <i>k</i><sub>inact</sub> (min<sup>–1</sup>) and <i>t</i><sub>1/2</sub> is <i>K</i><sub>i</sub> (μM).</div></div></div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> Differential Scanning Fluorometry</h3><div class="NLM_p last">Nonacylated hTEAD4 (20 μM) was incubated in the absence or presence of inhibitors (80 μM) at 25 °C for 10 min. Then, SYPRO orange (Sigma-Aldrich) at a final concentration of 5× was added and the mixture was heated at a ramp rate of 1 °C/min in a qPCR machine (Applied Biosystems). The derivative of the SYPRO orange fluorescent intensity is plotted as a function of temperature.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> Whole-Protein ESI–MS</h3><div class="NLM_p last">The protein solution (1 μL 5 μM) was injected to UPLC-MS (ThermoScientific_UltiMate 3000 LC coupled to a Velos Pro MS) through a desalting column (MassPREP Online desalting, 2.1 × 10 mm). For reactions, 1 μM of the protein was incubated at room temperature for 1 h with 8 equiv of the compound in HEPES (25 mM) and NaCl (150 mM) pH 6.5 buffer. Then, the mixture was centrifuged before injecting 5 μL. The deconvolution of <i>m</i>/<i>z</i> values was performed using Promass Xcalibur software.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> Tandem MS Experiments</h3><div id="sec4_7_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> Nonacylated hTEAD4-CPM Reaction</h4><div class="NLM_p last">To 5 μL of assay buffer [HEPES (25 mM) and NaCl (150 mM) pH 6.5], 5 μL of 0.6 μM nonacylated hTEAD4 and 5 μL of 0.6 μM CPM solution were added and incubated at room temperature for 1 h in a 384-well plate (Corning 4514). The reaction mixtures were collected from 16 replicates and transferred to a nonbinding Eppendorf tube followed by coincubating with 10 μL of 60 μM dithiothreitol (DTT) for 30 min to quench the unreacted CPM. The protein mixture was then purified through Amicon (10K) to exchange the buffer with 50 mM Tris-HCl (pH 7.5).</div></div><div id="sec4_7_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> In-Solution Digestion and Purification</h4><div class="NLM_p last">To 50 μL of the protein sample in 50 mM Tris-HCl (pH 7.5), 250 μL of denaturizing/reducing buffer (10 M urea, 1.2 mM TCEP, and 50 mM Tris, pH 7.5) was added and shaken at room temperature (350 rpm, 30 min). Next, 33.3 μL of 50 mM alkylation solution (iodoacetamide prepared in a denaturizing/reducing buffer) was added and shaken (350 rpm) at room temperature for 30 min. Subsequently, 23.5 μL of 1 ng/μL solution of LysC (Wako, 125-05061) was added at 37 °C and shaken (350 rpm) for 1 h, followed by addition of 7.8 μL of 6 ng/μL trypsin (Roche, 03708969001). The mixture was shaken (350 rpm) at 37 °C overnight. The digested samples were stage tip-purified using C18 (octadecyl) tips (3M High Performance Extraction Discs, Empore) using a protocol described earlier.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> Bovine serum albumin (50 μL, 0.5 μM prepared in 50 mM Tris-HCl, pH 7.5) was used in parallel as a control for the digestion–purification experiments and treated as described above. The peptide solution was injected to NanoLC-MS (ThermoScientific_UltiMate 3000 nanoLC coupled to a QExactive Plus or QExactive HF MS) through a nanocolumn (ThermoScientific PEPMAP C18 2UM 75UMX500MM).</div></div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> mTEAD4 (Nonacylated) HSCQ Experiments</h3><div class="NLM_p last">The <sup>13</sup>C/<sup>15</sup>N-labeled mTEAD4 was prepared following a protocol established earlier,<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> and the protein was deacylated as described above for hTEAD4. <sup>1</sup>H–<sup>15</sup>N HSQC spectra in the presence of DMSO or the test compound (0.5 mM) were collected at 37 °C on a Bruker 600 MHz magnet equipped with a cryoprobe in 20 mM HEPES, pH 7.3 buffer containing 150 mM NaCl and 1 mM DTT. The spectra were processed and overlaid.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> CETSA Experiments</h3><div id="sec4_9_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> Lysate Preparation</h4><div class="NLM_p last">HCT116 cells were grown in Dulbecco’s modified Eagle’s medium (Sigma-Aldrich, #P04-03550) containing 10% fetal bovine serum, 1× NEAA (Sigma-Aldrich, # P08-32800) and 1 mM sodium pyruvate (Sigma-Aldrich, #S8636) until they reached 90% confluency. Cells were detached, transferred to two separate tubes, and washed three times with PBS. Then, 0.6 mL of PBS containing 0.04% NP-40 was added to each tube and cells were lysed by means of freeze and thaw. Lysates were centrifuged (Beckman Optima MAX-TL) at 100,000<i>g</i>, 4 °C for 25 min and protein concentration was determined using the DC Protein Assay Kit I (Bio-Rad, #5000111). Aliquots (1.5 mL) containing 1 mg/mL protein were stored at −80 °C until further usage.</div></div><div id="sec4_9_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> CETSA</h4><div class="NLM_p last">Compound <b>19</b> (2.75 μL 10 mM stock, 50 μM) or DMSO (2.75 μL) was added to 550 μL of HCT116 lysates (1 mg/mL), samples were mixed and incubated for 10 min at room temperature. Treated and nontreated lysates were divided into ten aliquots, each 50 μL in PCR tubes. The aliquots were individually heated at different temperatures (Eppendorf Mastercycler ep Gradient S). After the heat treatment, the cell lysates were directly completely transferred to polycarbonate tubes and centrifuged (Beckman Optima MAX-TL) at 100,000<i>g</i>, 4 °C for 25 min. To 16 μL of each supernatant was added 4 μL of 5× loading buffer and incubated for 5 min at 95 °C, before samples were analyzed by immunoblotting. Proteins were separated by sodium dodecyl sulphate-polyacrylamide gel electrophoresis (SDS-PAGE) and transferred to a polyvinylidene difluoride membrane using wet transfer. The membranes were blocked with Odyssey blocking buffer (TBS; <i>Li-Cor</i>) for 1 h and incubated with the primary antibody (Mouse anti-TEAD4 antibody corresponding to amino acids 151–261 of Human TEAD4—ChIP Grade (ab58310)) in Odyssey blocking buffer containing 0.2% Tween 20 at 4 °C overnight. After washing with TBS-T (TBS containing 0.1% Tween 20), the membrane was incubated with the secondary antibody coupled to IRDye 800CW (goat anti-mouse IgG, <i>Li-COR</i>) for 1 h, in Odyssey blocking buffer containing 0.2% Tween 20 and 0.1% SDS at room temperature. Membranes were washed with TBS-T and then TBS before images were taken (Bio-Rad ChemiDoc MP Imaging System).</div></div><div id="sec4_9_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> Analysis</h4><div class="NLM_p last">Images were analyzed with ImageJ (FUJI). Normalization and all calculations were performed with GraphPad Prism 7 software (Curves: IC<sub>50</sub> variable slope → turning points correspond to melting temperature in obtained curves).</div></div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> RNA Purification and RT-qPCR</h3><div class="NLM_p">HEK293 cells were treated with DMSO or the indicated compound for the indicated duration. RNA was extracted using the RNeasy Plus Mini Kit (QIAGEN, Hilden, Germany) following the manufacturer’s procedure. DNA was removed on the column by DNase digestion with the RNase-free DNase Set (QIAGEN, Hilden, Germany) according to the manufacturer’s instructions. Following the manufacturer’s protocol, cDNA templates were synthesized from 700 ng total RNA using the QuantiTect Reverse Transcription Kit (QIAGEN, Hilden, Germany).</div><div class="NLM_p">The expression levels of the TEAD target gene ANKRD1 was assessed by real-time quantitative PCR. A total of 100 ng of cDNA was amplified using 500 nM gene-specific primers and SsoAdvanced Universal SYBR Green Supermix (Bio-Rad Laboratories, Feldkirchen, Germany) in a total volume of 10 μL for 50 cycles. All measurements were performed in both technical and biological triplicates using a CFX96 Touch Real-Time PCR Detection System (Bio-Rad Laboratories, Feldkirchen, Germany).</div><div class="NLM_p last">The sequences of primers for ANKRD1 were forward 5′-AGACTCCTTCAGCCAACATGATG-3′ and reverse 5′-CTCTCCATCTCTGAAATCCTCAGG-3′. The sequences of primers for GAPDH were forward 5′-GTCTCCTCTGACTTCAACAGCG-3′ and reverse 5′-ACCACCCTGTTGCTGTAGCCAA-3′.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> Assay Media Stability</h3><div class="NLM_p last">The compounds were incubated in MES (25 mM), pH 6.5 buffer or HEPES (25 mM) and NaCl (150 mM) pH 6.5 buffer for an hour at room temperature before injecting into a reverse-phase HPLC system (ThermoScientific_Ultimate 3000) equipped with a C18 column (2/50 Nucleodur C18 gravity 1.8 μm von Macherey-Nagel).</div></div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i45">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_77271" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_77271" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">All the final compounds were characterized with <sup>1</sup>H NMR, <sup>13</sup>C NMR (500, 700 MHz, Bruker), and HR-MS (ESI<sup>+</sup>) (LTQ Orbitrap mass spectrometer). The intermediates were characterized with <sup>1</sup>H NMR, <sup>13</sup>C NMR (500, 700 MHz, Bruker), and LR-MS (ESI<sup>+</sup>) (Agilent 1290 Infinity UPLC/MS coupled to Agilent 6120 Quadripole or ThermoScientific_UltiMate 3000 LCMS coupled to LQC Fleet). Chemical shifts were reported in ppm relative to tetramethylsilane. CD<sub>3</sub>OD (3.31 ppm) or DMSO-<i>d</i><sub>6</sub> (2.50 ppm) was used as the internal standard for <sup>1</sup>H NMR spectra. CD<sub>3</sub>OD (49.0 ppm) or DMSO-<i>d</i><sub>6</sub> (39.52 ppm) was used as the internal standard for <sup>13</sup>C NMR spectra. All the final compounds have purity ≥95% as determined by reverse-phase UPLC (Agilent 1260 Infinity or ThermoScientific_Dionex Ultimate 3000 LCMS) equipped with a C18 column (Nucleodur C18 Gravity, 5 μm).</div><div class="NLM_p">Compounds <b>1–3</b> and <b>10–16</b> were obtained from commercial sources and used without further purification.</div><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> 2-((2-Fluorophenyl)(pyridin-2-ylamino)methyl)-3-hydroxy-6-(hydroxymethyl)-4<i>H</i>-pyran-4-one (<b>10</b>)</h3><div class="NLM_p">Each component (2 mmol) (kojic acid, 2-fluorobenzaldehyde, and 2-amino pyridine) in dimethylformamide (DMF) was stirred at room temperature for 2 days before being concentrated in vacuo. Then, the remaining crude was purified over a silica column using ethyl acetate/methanol (20:1) as an eluent. The collected fractions were concentrated and further washed with methanol yielding compound <b>10</b> as a white solid. HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>15</sub>FN<sub>2</sub>O<sub>4</sub> + H<sup>+</sup> [M + H]<sup>+</sup>, 343.1089; found, 343.1089. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.20 (s, 1H), 7.94 (dd, <i>J</i> = 5.2, 1.8 Hz, 1H), 7.51 (td, <i>J</i> = 7.8, 1.9 Hz, 1H), 7.44–7.37 (m, 2H), 7.33 (tdd, <i>J</i> = 7.4, 5.2, 1.8 Hz, 1H), 7.17 (q, <i>J</i> = 9.3, 8.1 Hz, 2H), 6.84 (d, <i>J</i> = 8.2 Hz, 1H), 6.68 (d, <i>J</i> = 8.4 Hz, 1H), 6.53 (dd, <i>J</i> = 7.0, 5.0 Hz, 1H), 6.30 (s, 1H), 5.64 (t, <i>J</i> = 6.2 Hz, 1H), 4.31–4.18 (m, 2H).</div><div class="NLM_p last"><sup>13</sup>C NMR (126 MHz, DMSO): δ 173.8, 167.5, 160.8, 158.8, 157.1, 148.1, 147.4, 141.2, 136.9, 129.5, 129.4, 128.99, 128.96, 126.8, 126.7, 124.5, 124.4, 115.4, 115.3, 112.8, 109.1, 108.9, 59.4, 44.80, 44.77, 40.1, 40.0, 39.94, 39.85, 39.78, 39.7, 39.5, 39.4, 39.2, 39.0.</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> 3-Hydroxy-6-(hydroxymethyl)-2-(phenyl(phenylamino)methyl)-4<i>H</i>-pyran-4-one (<b>19</b>)</h3><div class="NLM_p">Each component (2 mmol) (kojic acid, benzaldehyde, and aniline) in DMF was stirred at room temperature for 3 days before being concentrated in vacuo. Then, the remaining crude was washed with ethanol yielding compound <b>19</b> as a white solid (34% yield). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>17</sub>NO<sub>4</sub> + H<sup>+</sup> [M + H]<sup>+</sup>, 324.1230; found, 324.1236. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.36 (s, 1H), 7.50–7.45 (m, 2H), 7.37 (t, <i>J</i> = 7.7 Hz, 2H), 7.32–7.27 (m, 1H), 7.09–7.04 (m, 2H), 6.73–6.68 (m, 2H), 6.60–6.55 (m, 1H), 6.41 (d, <i>J</i> = 8.2 Hz, 1H), 6.27 (s, 1H), 5.89 (d, <i>J</i> = 8.2 Hz, 1H), 5.61 (t, <i>J</i> = 6.3 Hz, 1H), 4.25 (qd, <i>J</i> = 15.5, 5.8 Hz, 2H).</div><div class="NLM_p"><sup>13</sup>C NMR (176 MHz, DMSO): δ 173.6, 167.5, 148.9, 147.2, 141.5, 139.2, 128.9, 128.6, 127.6, 127.0, 116.9, 112.9, 108.9, 59.4, 52.9.</div><div class="NLM_p last"><sup>1</sup>H NMR, <sup>13</sup>C NMR, purity and assay media stability spectra of compound <b>19</b> is shown in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01275/suppl_file/jm0c01275_si_001.pdf" class="ext-link">Supporting Information</a> (pages S17, S19–S21).</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> 3-Hydroxy-6-(hydroxymethyl)-2-((phenylamino)(<i>p</i>-tolyl)methyl)-4<i>H</i>-pyran-4-one (<b>20</b>)</h3><div class="NLM_p">Each component (1 mmol) (kojic acid, benzaldehyde, and isopropylamine) in 15 mL of EtOH was stirred at room temperature for 7 h before being concentrating in vacuo. The remaining crude was purified using a silica column using ethyl acetate/methanol (25:1) as an eluent. The compound was dissolved in an acetonitrile/H<sub>2</sub>O mixture and lyophilized yielding 19 mg of compound <b>20</b> (6.5% yield). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>16</sub>H<sub>19</sub>NO<sub>4</sub> + H<sup>+</sup> [M + H]<sup>+</sup>, 290.1387; found, 290.1391. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.43–7.40 (m, 2H), 7.33 (dd, <i>J</i> = 8.4, 6.9 Hz, 2H), 7.28–7.24 (m, 1H), 6.26 (s, 1H), 5.62 (t, <i>J</i> = 6.4 Hz, 1H), 5.16 (s, 1H), 4.31–4.22 (m, 2H), 2.64 (h, <i>J</i> = 6.2 Hz, 1H), 1.01 (dd, <i>J</i> = 10.4, 6.2 Hz, 6H).</div><div class="NLM_p last"><sup>13</sup>C NMR (176 MHz, DMSO): δ 173.5, 167.1, 150.2, 141.9, 140.5, 128.4, 127.3, 127.1, 108.9, 59.5, 55.8, 45.8, 22.9, 22.3.</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> 2-((Benzylamino)(phenyl)methyl)-3-hydroxy-6-(hydroxymethyl)-4<i>H</i>-pyran-4-one (<b>21</b>)</h3><div class="NLM_p">Each component (1 mmol) (kojic acid, benzaldehyde, and benzyl amine) in 15 mL of EtOH was stirred at room temperature for 3 days before being concentrated in vacuo. The remaining crude was extracted with ethyl acetate and water. The organic layers were dried over anhydrous MgSO<sub>4</sub> and concentrated. The remaining crude was purified using a silica column using ethyl acetate as an eluent. A second silica column purification step was pursued using CH<sub>2</sub>Cl<sub>2</sub>/MeOH (10:0.75) as an eluent. The purified compound was dissolved in an acetonitrile/H<sub>2</sub>O mixture and lyophilized yielding 22 mg of compound <b>21</b> as a cream color solid (0.065 mmol, % 65 yield). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>19</sub>NO<sub>4</sub> + H<sup>+</sup> [M + H]<sup>+</sup>, 338.1387; found, 338.1375. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.09 (s, 1H), 7.43–7.40 (m, 2H), 7.36–7.25 (m, 7H), 7.24–7.20 (m, 1H), 6.26 (s, 1H), 5.65 (t, <i>J</i> = 6.4 Hz, 1H), 5.06 (s, 1H), 4.27 (qd, <i>J</i> = 15.5, 6.2 Hz, 2H), 3.66 (s, 2H).</div><div class="NLM_p last"><sup>13</sup>C NMR (126 MHz, DMSO): δ 173.5, 167.2, 149.8, 141.9, 140.3, 140.1, 128.5, 128.14, 128.11, 127.4, 127.1, 126.8, 108.8, 59.5, 57.4, 50.7.</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> 3-Hydroxy-6-(hydroxymethyl)-2-(isoindolin-2-yl(phenyl)methyl)-4<i>H</i>-pyran-4-one (<b>22</b>)</h3><div class="NLM_p">Each component (1 mmol) (kojic acid, benzaldehyde, and isoindoine) in 15 mL of EtOH was stirred at room temperature overnight before being concentrated in vacuo. The remaining crude was extracted with ethyl acetate and water. The organic layers were dried over anhydrous MgSO<sub>4</sub> and concentrated. The remaining crude was purified using a silica column using ethyl acetate as an eluent. Then, the compound was dissolved in an acetonitrile/H<sub>2</sub>O mixture and lyophilized yielding 38 mg of compound <b>22</b> as an off-white solid (11% yield). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>21</sub>H<sub>19</sub>NO<sub>4</sub> + H<sup>+</sup> [M + H]<sup>+</sup>, 350.1387; found, 350.1369. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.23 (s, 1H), 7.57–7.54 (m, 2H), 7.38 (t, <i>J</i> = 7.6 Hz, 2H), 7.32 (td, <i>J</i> = 7.1, 1.4 Hz, 1H), 7.22–7.16 (m, 4H), 6.30 (s, 1H), 5.65 (t, <i>J</i> = 6.2 Hz, 1H), 5.19 (s, 1H), 4.35–4.25 (m, 2H), 3.85–3.78 (m, 4H).</div><div class="NLM_p last"><sup>13</sup>C NMR (176 MHz, DMSO): δ 173.6, 167.7, 148.5, 142.0, 139.4, 139.1, 128.6, 128.1, 127.9, 126.6, 122.3, 108.8, 64.4, 59.5, 57.1.</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> 2-(((3-Fluorophenyl)amino)(phenyl)methyl)-3-hydroxy-6-(hydroxymethyl)-4<i>H</i>-pyran-4-one (<b>23</b>)</h3><div class="NLM_p">Each component (1 mmol) (kojic acid, benzaldehyde, and 3-fluoroaniline) in 15 mL of EtOH was stirred at room temperature for 5 days before being concentrated in vacuo. The remaining crude was extracted with ethyl acetate and water. The organic layers were dried over anhydrous MgSO<sub>4</sub> and concentrated and the remaining crude was purified using a silica column using ethyl acetate as an eluent. Next, the collected fractions were concentrated and washed with CH<sub>2</sub>Cl<sub>2</sub>. The remaining white powder was dissolved in an acetonitrile/H<sub>2</sub>O mixture and lyophilized yielding 46 mg of compound <b>23</b> as a cream color solid (0.135 mmol, 13.5% yield). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>16</sub>FNO<sub>4</sub> + H<sup>+</sup> [M + H]<sup>+</sup>, 342.1136; found, 342.1139. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.48 (s, 1H), 7.50–7.43 (m, 2H), 7.42–7.35 (m, 2H), 7.34–7.25 (m, 1H), 7.08 (td, <i>J</i> = 8.2, 7.0 Hz, 1H), 6.81 (d, <i>J</i> = 7.9 Hz, 1H), 6.55–6.51 (m, 1H), 6.46 (dt, <i>J</i> = 12.3, 2.3 Hz, 1H), 6.35 (td, <i>J</i> = 8.5, 2.5 Hz, 1H), 6.29 (s, 1H), 5.88 (d, <i>J</i> = 8.0 Hz, 1H), 5.63 (t, <i>J</i> = 6.2 Hz, 1H), 4.30–4.20 (m, 2H).</div><div class="NLM_p last"><sup>13</sup>C NMR (126 MHz, DMSO): δ 173.6, 167.6, 164.2, 162.3, 149.37, 149.28, 148.3, 141.7, 138.7, 130.45, 130.37, 128.7, 127.8, 127.1, 108.99, 108.98, 108.95, 103.1, 102.9, 99.4, 99.2, 59.4, 52.9.</div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> 3-Hydroxy-6-(hydroxymethyl)-2-(phenyl(pyridin-2-ylamino)methyl)-4<i>H</i>-pyran-4-one (<b>24</b>)</h3><div class="NLM_p">Each component (2 mmol) (kojic acid, benzaldehyde, and 2-aminopyridine) in DMF was stirred at room temperature for 3 days before being concentrated in vacuo. Then, the remaining crude was washed with ethanol yielding compound <b>24</b> as a white solid. HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>16</sub>N<sub>2</sub>O<sub>4</sub> + H<sup>+</sup> [M + H]<sup>+</sup>, 325.1183; found, 325.1169. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.25 (s, 1H), 7.95 (dd, <i>J</i> = 5.2, 1.8 Hz, 1H), 7.44–7.31 (m, 6H), 7.28–7.23 (m, 1H), 6.72 (d, <i>J</i> = 8.4 Hz, 1H), 6.67 (d, <i>J</i> = 8.6 Hz, 1H), 6.53 (dd, <i>J</i> = 7.0, 5.1 Hz, 1H), 6.29 (s, 1H), 5.64 (t, <i>J</i> = 6.3 Hz, 1H), 4.28 (qd, <i>J</i> = 15.6, 6.2 Hz, 2H).</div><div class="NLM_p last"><sup>13</sup>C NMR (126 MHz, DMSO): δ 173.8, 167.4, 157.4, 149.5, 147.4, 141.0, 140.2, 137.0, 128.5, 127.4, 126.8, 112.7, 109.1, 108.9, 59.5, 50.1.</div></div><div id="sec5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> 3-Hydroxy-6-(hydroxymethyl)-2-(((4-methylpyridin-2-yl)amino)(phenyl)methyl)-4<i>H</i>-pyran-4-one (<b>25</b>)</h3><div class="NLM_p">Each component (1 mmol) (kojic acid, benzaldehyde, and 2-amino-4-methylpyridine) in 15 mL of EtOH was stirred at room temperature for 2 days before being concentrated in vacuo. The remaining crude was extracted with ethyl acetate and water. The organic layers were dried over anhydrous MgSO<sub>4</sub>, concentrated, and dried. The remaining crude was taken in a mixture of ethyl acetate/MeOH (10:0.5) and sonicated for 10 min. The solution was discarded and the wash step was repeated. Next, the step was repeated using EtOH. Then, the compound was taken in an acetonitrile/H<sub>2</sub>O mixture and lyophilized yielding 156 mg of compound <b>25</b> as a white solid (0.46 mmol, 46% yield). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>18</sub>N<sub>2</sub>O<sub>4</sub> + H<sup>+</sup> [M + H]<sup>+</sup>, 339.1339; found, 339.1325. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.25 (s, 1H), 7.82 (d, <i>J</i> = 5.2 Hz, 1H), 7.41–7.36 (m, 2H), 7.36–7.30 (m, 2H), 7.29–7.22 (m, 2H), 6.67 (d, <i>J</i> = 8.7 Hz, 1H), 6.54 (d, <i>J</i> = 1.5 Hz, 1H), 6.39 (dd, <i>J</i> = 5.3, 1.4 Hz, 1H), 6.29 (s, 1H), 5.64 (t, <i>J</i> = 6.3 Hz, 1H), 4.28 (qdd, <i>J</i> = 15.6, 6.3, 0.9 Hz, 2H), 2.15 (s, 3H).</div><div class="NLM_p last"><sup>13</sup>C NMR (126 MHz, DMSO): δ 173.8, 167.4, 157.6, 149.6, 147.23, 147.07, 140.9, 140.3, 128.5, 127.3, 126.8, 114.4, 109.0, 108.9, 59.5, 50.1, 20.6.</div></div><div id="sec5_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> 3-Hydroxy-6-(hydroxymethyl)-2-(phenyl(pyrimidin-2-ylamino)methyl)-4<i>H</i>-pyran-4-one (<b>26</b>)</h3><div class="NLM_p">Each component (1 mmol) (kojic acid, benzaldehyde, and 2-aminopyrimidine) in 15 mL of EtOH was stirred at room temperature for 6 days before being concentrated in vacuo. The remaining crude was extracted with ethyl acetate and water. The organic layers together were dried over anhydrous MgSO<sub>4</sub>, concentrated, and dried. The remaining crude was purified using a silica column using CH<sub>2</sub>Cl<sub>2</sub>/MeOH (10:0.5) as an eluent. Then, the compound was taken in an acetonitrile/H<sub>2</sub>O mixture and lyophilized yielding 8 mg of compound <b>26</b> as a white solid (0.025 mmol, 2.5% yield). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>17</sub>H<sub>15</sub>N<sub>3</sub>O<sub>4</sub> + H<sup>+</sup> [M + H]<sup>+</sup>, 326.1135; found, 326.1128. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.29 (s, 1H), 8.33 (d, <i>J</i> = 4.8 Hz, 2H), 8.05 (d, <i>J</i> = 9.1 Hz, 1H), 7.41 (dd, <i>J</i> = 8.0, 1.4 Hz, 2H), 7.34 (t, <i>J</i> = 7.7 Hz, 2H), 7.30–7.22 (m, 1H), 6.69 (d, <i>J</i> = 9.1 Hz, 1H), 6.66 (t, <i>J</i> = 4.8 Hz, 1H), 6.29 (d, <i>J</i> = 0.9 Hz, 1H), 5.65 (t, <i>J</i> = 6.5 Hz, 1H), 4.29 (dddd, <i>J</i> = 56.1, 15.4, 6.5, 0.9 Hz, 2H).</div><div class="NLM_p last"><sup>13</sup>C NMR (176 MHz, DMSO): δ 173.8, 167.3, 158.14, 157.99, 148.6, 140.8, 139.8, 128.5, 127.4, 126.6, 111.2, 108.9, 59.5, 50.4.</div></div><div id="sec5_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> 3-Hydroxy-6-(hydroxymethyl)-2-(phenyl(thiazol-2-ylamino)methyl)-4<i>H</i>-pyran-4-one (<b>27</b>)</h3><div class="NLM_p">Each component (1 mmol) (kojic acid, benzaldehyde, and thiazol-2-amine) in 15 mL of EtOH was stirred at room temperature for 14 days before being concentrated in vacuo. The remaining crude was extracted with ethyl acetate and water. The organic layers were dried over anhydrous MgSO<sub>4</sub>, concentrated, and dried. The remaining crude was purified using a silica column using CH<sub>2</sub>Cl<sub>2</sub>/MeOH (10:1) as an eluent. Then, the compound was dissolved in an acetonitrile/H<sub>2</sub>O mixture and lyophilized yielding 34 mg of compound <b>27</b> as an off-white solid (10% yield). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>16</sub>H<sub>14</sub>N<sub>2</sub>O<sub>4</sub>S + H<sup>+</sup> [M + H]<sup>+</sup>, 331.0747; found, 331.0735. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.34 (s, 1H), 8.48 (d, <i>J</i> = 8.3 Hz, 1H), 7.42–7.38 (m, 2H), 7.36 (t, <i>J</i> = 7.7 Hz, 2H), 7.31–7.26 (m, 1H), 7.01 (d, <i>J</i> = 3.6 Hz, 1H), 6.67 (d, <i>J</i> = 3.6 Hz, 1H), 6.39 (d, <i>J</i> = 7.9 Hz, 1H), 6.30 (s, 1H), 5.63 (t, <i>J</i> = 6.3 Hz, 1H), 4.34–4.21 (m, 2H).</div><div class="NLM_p last"><sup>13</sup>C NMR (151 MHz, DMSO): δ 173.7, 167.54, 167.41, 148.3, 141.1, 139.1, 138.4, 128.6, 127.7, 126.8, 109.0, 107.6, 59.4, 53.8.</div></div><div id="sec5_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> 2-(((1<i>H</i>-Benzo[<i>d</i>]imidazol-2-yl)amino)(phenyl)methyl)-3-hydroxy-6-(hydroxymethyl)-4<i>H</i>-pyran-4-one (<b>28</b>)</h3><div class="NLM_p">Each component (1 mmol) (kojic acid, benzaldehyde, and 1<i>H</i>-benzimidazol-2-amine) in 15 mL of EtOH was stirred at room temperature for 20 days before being concentrated in vacuo. The remaining crude was extracted with ethyl acetate and water. The organic layers were dried over anhydrous MgSO<sub>4</sub>, concentrated, and dried. The remaining crude was taken in a mixture of ethyl acetate/MeOH (10:0.5) and sonicated for 5 min. The solution was discarded, and the wash step was repeated twice. Then, the compound was taken in an acetonitrile/H<sub>2</sub>O mixture and lyophilized yielding 22 mg of compound <b>28</b> as an off-white solid (0.06 mmol, 6% yield). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>17</sub>N<sub>3</sub>O<sub>4</sub> + H<sup>+</sup> [M + H]<sup>+</sup>, 364.1292; found, 364.1268. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.69 (bs, 1H), 9.62 (bs, 1H), 7.60 (d, <i>J</i> = 9.1 Hz, 1H), 7.47–7.43 (m, 2H), 7.37 (t, <i>J</i> = 7.6 Hz, 2H), 7.29 (t, <i>J</i> = 7.4 Hz, 1H), 7.17 (d, <i>J</i> = 8.7 Hz, 2H), 6.89 (s, 2H), 6.49 (d, <i>J</i> = 8.5 Hz, 1H), 6.31 (s, 1H), 5.65 (bs, 1H), 4.29 (q, <i>J</i> = 15.4 Hz, 2H).</div><div class="NLM_p last"><sup>13</sup>C NMR (176 MHz, DMSO): δ 173.9, 167.2, 154.1, 149.0, 141.1, 139.5, 128.6, 127.6, 126.7, 109.2, 59.5, 52.6.</div></div><div id="sec5_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> 4-(((3-Hydroxy-6-(hydroxymethyl)-4-oxo-4<i>H</i>-pyran-2-yl)(phenyl)methyl)amino)benzamide (<b>29</b>)</h3><div class="NLM_p">Each component (1 mmol) (kojic acid, benzaldehyde, and 4-aminobenzamide) in 15 mL of EtOH was stirred at room temperature for 3 days before being concentrated in vacuo. The remaining crude was partitioned between ethyl acetate and water. The observed white precipitate from both layers was collected and washed with ethyl acetate and water. The compound was then taken in an acetonitrile/H<sub>2</sub>O mixture and lyophilized yielding 95 mg of compound <b>29</b> as an off-white solid (26% yield). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>18</sub>N<sub>2</sub>O<sub>5</sub> + H<sup>+</sup> [M + H]<sup>+</sup>, 367.1288; found, 367.1289. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.46 (s, 1H), 7.65–7.61 (m, 2H), 7.55 (bs, 1H), 7.50–7.45 (m, 2H), 7.38 (t, <i>J</i> = 7.7 Hz, 2H), 7.34–7.30 (m, 1H), 6.94 (d, <i>J</i> = 7.9 Hz, 1H), 6.90 (bs, 1H), 6.71–6.67 (m, 2H), 6.29 (d, <i>J</i> = 0.9 Hz, 1H), 5.96 (d, <i>J</i> = 7.9 Hz, 1H), 5.62 (t, <i>J</i> = 6.3 Hz, 1H), 4.30–4.21 (m, 2H).</div><div class="NLM_p last"><sup>13</sup>C NMR (176 MHz, DMSO): δ 173.6, 167.8, 167.6, 149.7, 148.3, 141.7, 138.7, 128.96, 128.67, 127.8, 127.1, 122.4, 111.7, 108.9, 59.4, 52.7.</div></div><div id="sec5_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> 2-(Cyclohexyl(phenylamino)methyl)-3-hydroxy-6-(hydroxymethyl)-4<i>H</i>-pyran-4-one (<b>30</b>)</h3><div class="NLM_p">Each component (1 mmol) (kojic acid, cyclohexanecarboxaldehyde, and aniline) in 15 mL of EtOH was stirred at room temperature for 5 days before being concentrated in vacuo. The remaining crude was extracted with ethyl acetate and brine. The organic layers were dried over anhydrous MgSO<sub>4</sub> and concentrated. The remaining crude was purified using a silica column using ethyl acetate/CH<sub>2</sub>Cl<sub>2</sub> (1:1) as an eluent. Then, the compound was dissolved in an acetonitrile/H<sub>2</sub>O mixture and lyophilized yielding 80 mg of compound <b>30</b> as an off-white solid (0.24 mmol, 24% yield). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>23</sub>NO<sub>4</sub> + H<sup>+</sup> [M + H]<sup>+</sup>, 330.1700; found, 330.1702. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.03 (s, 1H), 7.04–6.95 (m, 2H), 6.64–6.56 (m, 2H), 6.52–6.44 (m, 1H), 6.24 (d, <i>J</i> = 0.9 Hz, 1H), 5.88 (d, <i>J</i> = 8.7 Hz, 1H), 5.59 (t, <i>J</i> = 6.3 Hz, 1H), 4.37 (t, <i>J</i> = 9.0 Hz, 1H), 4.25 (qdd, <i>J</i> = 15.4, 6.3, 0.9 Hz, 2H), 2.12–2.05 (m, 1H), 1.84–1.76 (m, 1H), 1.73–1.56 (m, 3H), 1.43 (d, <i>J</i> = 12.8 Hz, 1H), 1.27–1.11 (m, 3H), 1.08–0.95 (m, 2H).</div><div class="NLM_p last"><sup>13</sup>C NMR (151 MHz, DMSO): δ 173.3, 167.4, 149.6, 148.0, 142.7, 128.9, 116.0, 112.2, 108.7, 59.5, 54.3, 30.0, 29.0, 25.9, 25.4.</div></div><div id="sec5_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> 3-Hydroxy-6-(hydroxymethyl)-2-(2-methyl-1-(phenylamino)propyl)-4<i>H</i>-pyran-4-one (<b>31</b>)</h3><div class="NLM_p">Each component (1 mmol) (kojic acid, isobutyraldehyde, and aniline) in 15 mL of EtOH was stirred at room temperature for 5 days before being concentrated in vacuo. The remaining crude was extracted with ethyl acetate and brine. The organic layers were dried over anhydrous MgSO<sub>4</sub> and concentrated. The remaining crude was purified using a silica column using ethyl acetate/CH<sub>2</sub>Cl<sub>2</sub> (1:1) as an eluent. Then, the compound was dissolved in an acetonitrile/H<sub>2</sub>O mixture and lyophilized yielding 75 mg of compound <b>31</b> as an off-white solid (0.26 mmol, 26% yield). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>16</sub>H<sub>19</sub>NO<sub>4</sub> + H<sup>+</sup> [M + H]<sup>+</sup>, 290.1387; found, 290.1389. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.04 (s, 1H), 7.04–6.98 (m, 2H), 6.65–6.59 (m, 2H), 6.50 (tt, <i>J</i> = 7.3, 1.1 Hz, 1H), 6.24 (d, <i>J</i> = 0.9 Hz, 1H), 5.90 (d, <i>J</i> = 8.8 Hz, 1H), 5.59 (t, <i>J</i> = 6.2 Hz, 1H), 4.32–4.18 (m, 3H), 2.09 (dp, <i>J</i> = 9.0, 6.6 Hz, 1H), 1.08 (d, <i>J</i> = 6.6 Hz, 3H), 0.84 (d, <i>J</i> = 6.7 Hz, 3H).</div><div class="NLM_p last"><sup>13</sup>C NMR (151 MHz, DMSO): δ 173.3, 167.3, 149.8, 147.9, 142.5, 128.9, 116.1, 112.2, 108.7, 59.5, 55.5, 30.8, 20.1, 19.1.</div></div><div id="sec5_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> 3-Hydroxy-6-(hydroxymethyl)-2-((phenylamino)methyl)-4<i>H</i>-pyran-4-one (<b>32</b>)</h3><div class="NLM_p">Each component (1 mmol) (kojic acid, formaldehyde, and aniline) in 15 mL of EtOH was stirred at room temperature for 5 days before being concentrated in vacuo. The remaining crude was purified using a silica column using CH<sub>2</sub>Cl<sub>2</sub>/MeOH (10:1) as an eluent. Then, the compound was dissolved in an acetonitrile/H<sub>2</sub>O mixture and lyophilized yielding 100 mg of compound <b>32</b> as a light brown-orange solid (40% yield). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>13</sub>H<sub>13</sub>NO<sub>4</sub> + H<sup>+</sup> [M + H]<sup>+</sup>, 248.0917; found, 248.0918. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.19 (s, 1H), 7.09–7.01 (m, 2H), 6.67–6.61 (m, 2H), 6.57–6.50 (m, 1H), 6.29 (s, 1H), 6.16 (t, <i>J</i> = 6.1 Hz, 1H), 5.65 (t, <i>J</i> = 6.1 Hz, 1H), 4.26 (dd, <i>J</i> = 6.2, 0.9 Hz, 2H), 4.20 (d, <i>J</i> = 6.0 Hz, 2H).</div><div class="NLM_p last"><sup>13</sup>C NMR (126 MHz, DMSO): δ 173.6, 167.6, 148.0, 147.8, 142.3, 128.9, 116.3, 112.1, 108.9, 59.5.</div></div><div id="sec5_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> 4-((3-Hydroxy-6-(hydroxymethyl)-4-oxo-4<i>H</i>-pyran-2-yl)(phenylamino)methyl)benzamide (<b>33</b>)</h3><div class="NLM_p">Each component (1 mmol) (kojic acid, 4-formylbenzamide, and aniline) in 25 mL of EtOH was stirred at room temperature for 2 days before being concentrated in vacuo. The remaining crude was extracted with ethyl acetate and water. The organic layers were dried over anhydrous MgSO<sub>4</sub>, concentrated, and dried. The remaining crude was purified using a silica column using ethyl acetate/MeOH (10:1) as an eluent. Then, the compound was taken in an acetonitrile/H<sub>2</sub>O mixture and lyophilized yielding 40 mg of compound <b>33</b> as a bright-yellow solid (11% yield). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>18</sub>N<sub>2</sub>O<sub>5</sub> + H<sup>+</sup> [M + H]<sup>+</sup>, 367.1288; found, 367.1289. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.43 (s, 1H), 7.94 (bs, 1H), 7.87–7.83 (m, 2H), 7.56–7.52 (m, 2H), 7.34 (bs, 1H), 7.10–7.05 (m, 2H), 6.73–6.68 (m, 2H), 6.59 (tt, <i>J</i> = 7.3, 1.1 Hz, 1H), 6.48 (d, <i>J</i> = 8.2 Hz, 1H), 6.28 (d, <i>J</i> = 0.9 Hz, 1H), 5.94 (d, <i>J</i> = 8.2 Hz, 1H), 5.62 (t, <i>J</i> = 6.3 Hz, 1H), 4.30–4.20 (m, 2H).</div><div class="NLM_p last"><sup>13</sup>C NMR (176 MHz, DMSO): δ 173.6, 167.57, 167.55, 148.4, 147.1, 142.4, 141.6, 133.7, 128.9, 127.8, 126.8, 117.0, 113.0, 108.9, 59.4, 52.8.</div></div><div id="sec5_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> 3-Hydroxy-6-(hydroxymethyl)-2-((phenylamino)(<i>p</i>-tolyl)methyl)-4<i>H</i>-pyran-4-one (<b>34</b>)</h3><div class="NLM_p">Each component (2 mmol) (kojic acid, 4-methylbenzaldehyde, and aniline) in 15 mL of EtOH was stirred at room temperature for 2 days before being concentrated in vacuo. The remaining crude was purified using a silica column using ethyl acetate as an eluent. The purified compound was further washed with diethyl ether. Then, the compound was dissolved in an acetonitrile/H<sub>2</sub>O mixture and lyophilized yielding 0.16 g of compound <b>34</b> as a light-brown solid (24% yield). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>19</sub>NO<sub>4</sub> + H<sup>+</sup> [M + H]<sup>+</sup>, 338.1387; found, 338.1390. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.32 (s, 1H), 7.36 (d, <i>J</i> = 8.0 Hz, 2H), 7.17 (d, <i>J</i> = 7.9 Hz, 2H), 7.09–7.02 (m, 2H), 6.69 (d, <i>J</i> = 8.0 Hz, 2H), 6.57 (t, <i>J</i> = 7.3 Hz, 1H), 6.35 (d, <i>J</i> = 8.2 Hz, 1H), 6.27 (s, 1H), 5.84 (d, <i>J</i> = 8.1 Hz, 1H), 5.61 (t, <i>J</i> = 6.3 Hz, 1H), 4.25 (qd, <i>J</i> = 15.5, 6.2 Hz, 2H), 2.27 (s, 3H).</div><div class="NLM_p last"><sup>13</sup>C NMR (176 MHz, DMSO): δ 173.6, 167.5, 149.1, 147.3, 141.4, 136.8, 136.2, 129.13, 128.89, 126.9, 116.8, 112.9, 108.8, 59.4, 52.6, 20.6.</div></div><div id="sec5_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> 2-((4-Fluorophenyl)(phenylamino)methyl)-3-hydroxy-6-(hydroxymethyl)-4<i>H</i>-pyran-4-one (<b>35</b>)</h3><div class="NLM_p">Each component (1 mmol) (kojic acid, 4-fluorobenzaldehyde, and aniline) in 15 mL of EtOH was stirred at room temperature for 2 days before being concentrated in vacuo. The remaining crude was extracted with ethyl acetate and water. The organic layers were dried over anhydrous MgSO<sub>4</sub>, concentrated, and dried. The remaining crude was purified using a silica column using CH<sub>2</sub>Cl<sub>2</sub>/MeOH (10:1) as an eluent, and the dried solid was further washed with ether. Then, the compound was dissolved in an acetonitrile/H<sub>2</sub>O mixture and lyophilized yielding 20 mg of compound <b>35</b> as a brown-orange solid (6% yield). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>16</sub>FNO<sub>4</sub> + H<sup>+</sup> [M + H]<sup>+</sup>, 342.1136; found, 342.1139. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.44 (s, 1H), 7.55–7.48 (m, 2H), 7.20 (t, <i>J</i> = 8.9 Hz, 2H), 7.11–7.04 (m, 2H), 6.72–6.65 (m, 2H), 6.58 (td, <i>J</i> = 7.3, 1.2 Hz, 1H), 6.45 (d, <i>J</i> = 8.2 Hz, 1H), 6.28 (s, 1H), 5.89 (d, <i>J</i> = 8.2 Hz, 1H), 5.63 (t, <i>J</i> = 6.3 Hz, 1H), 4.32–4.18 (m, 2H).</div><div class="NLM_p last"><sup>13</sup>C NMR (126 MHz, DMSO): δ 173.6, 167.6, 162.5, 160.6, 148.6, 147.1, 141.5, 135.43, 135.41, 129.14, 129.07, 128.97, 117.0, 115.51, 115.34, 113.0, 108.9, 59.4, 52.3.</div></div><div id="sec5_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> 2-((4-Bromophenyl)(phenylamino)methyl)-3-hydroxy-6-(hydroxymethyl)-4<i>H</i>-pyran-4-one (<b>36</b>)</h3><div class="NLM_p">Each component (1 mmol) (kojic acid, 4-bromobenzaldehyde, and aniline) in 15 mL of EtOH was stirred at room temperature for 2 days before being concentrated in vacuo. The remaining crude was extracted with ethyl acetate and water. The organic layers were dried over anhydrous MgSO<sub>4</sub> and concentrated. The remaining crude was purified using a silica column using ethyl acetate as an eluent. Then, the compound was dissolved in an acetonitrile/H<sub>2</sub>O mixture and lyophilized yielding 21 mg of compound <b>36</b> as a light-brown solid (5% yield). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>16</sub>BrNO<sub>4</sub> + H<sup>+</sup> [M + H]<sup>+</sup>, 402.0335; found, 402.0334. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.44 (s, 1H), 7.59–7.53 (m, 2H), 7.45–7.39 (m, 2H), 7.10–7.04 (m, 2H), 6.70–6.67 (m, 2H), 6.59 (dd, <i>J</i> = 7.9, 6.6 Hz, 1H), 6.46 (d, <i>J</i> = 8.1 Hz, 1H), 6.28 (s, 1H), 5.87 (d, <i>J</i> = 8.1 Hz, 1H), 5.61 (t, <i>J</i> = 6.3 Hz, 1H), 4.29–4.18 (m, 2H).</div><div class="NLM_p last"><sup>13</sup>C NMR (176 MHz, DMSO): δ 173.6, 167.6, 148.3, 147.0, 141.5, 138.7, 131.5, 129.2, 128.9, 120.7, 117.1, 113.0, 108.9, 59.4, 52.4.</div></div><div id="sec5_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> 3-Hydroxy-6-(hydroxymethyl)-2-((4-isopropylphenyl)(phenylamino)methyl)-4<i>H</i>-pyran-4-one (<b>37</b>)</h3><div class="NLM_p">Each component (2 mmol) (kojic acid, 4-isopropylbenzaldehyde, and aniline) in 10 mL of EtOH was stirred at room temperature for 2 days before being concentrated in vacuo. The remaining crude was purified using a silica column using ethyl acetate as an eluent. The purified compound was further washed with diethyl ether and dried. Then, the compound was dissolved in an acetonitrile/H<sub>2</sub>O mixture and lyophilized yielding 0.3 g of compound <b>37</b> as a white-bright yellow solid (41% yield). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>22</sub>H<sub>23</sub>NO<sub>4</sub> + H<sup>+</sup> [M + H]<sup>+</sup>, 366.1700; found, 366.1682. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.32 (s, 1H), 7.41–7.38 (m, 2H), 7.25–7.23 (m, 2H), 7.08–7.04 (m, 2H), 6.71–6.68 (m, 2H), 6.57 (tt, <i>J</i> = 7.3, 1.2 Hz, 1H), 6.35 (d, <i>J</i> = 8.2 Hz, 1H), 6.27 (s, 1H), 5.84 (d, <i>J</i> = 8.1 Hz, 1H), 5.62 (t, <i>J</i> = 6.3 Hz, 1H), 4.30–4.22 (m, 2H), 2.86 (hept, <i>J</i> = 6.9 Hz, 1H), 1.18 (d, <i>J</i> = 7.0 Hz, 6H).</div><div class="NLM_p last"><sup>13</sup>C NMR (176 MHz, DMSO): δ 173.5, 167.4, 149.0, 147.8, 147.3, 141.5, 136.6, 128.9, 127.1, 126.5, 116.8, 112.8, 108.9, 59.4, 52.7, 33.1, 23.85, 23.80.</div></div><div id="sec5_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i66" class="anchor-spacer"></div><h3 class="article-section__title" id="_i66"> 3-Hydroxy-6-(hydroxymethyl)-2-(naphthalen-2-yl(phenylamino)methyl)-4<i>H</i>-pyran-4-one (<b>38</b>)</h3><div class="NLM_p">Each component (1 mmol) (kojic acid, 2-naphthaldehyde, and aniline) in 15 mL of EtOH was stirred at room temperature for 2 days before being concentrated in vacuo. The remaining crude was extracted with ethyl acetate and water. The organic layers were dried over anhydrous MgSO<sub>4</sub> and concentrated. The remaining crude was purified using a silica column using ethyl acetate as an eluent. Then, the compound was dissolved in an acetonitrile/H<sub>2</sub>O mixture and lyophilized yielding 10 mg of compound <b>38</b> as a cream color solid (3% yield). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>23</sub>H<sub>19</sub>NO<sub>4</sub> + H<sup>+</sup> [M + H]<sup>+</sup>, 374.1387; found, 374.1388. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.43 (s, 1H), 8.00 (d, <i>J</i> = 1.8 Hz, 1H), 7.94–7.88 (m, 3H), 7.62 (dd, <i>J</i> = 8.5, 1.8 Hz, 1H), 7.51 (qd, <i>J</i> = 7.2, 3.4 Hz, 2H), 7.08 (dd, <i>J</i> = 8.4, 7.1 Hz, 2H), 6.75 (d, <i>J</i> = 8.0 Hz, 2H), 6.58 (t, <i>J</i> = 7.3 Hz, 1H), 6.55 (d, <i>J</i> = 8.0 Hz, 1H), 6.28 (s, 1H), 6.06 (d, <i>J</i> = 8.0 Hz, 1H), 5.61 (t, <i>J</i> = 6.3 Hz, 1H), 4.26 (qd, <i>J</i> = 15.6, 6.2 Hz, 2H).</div><div class="NLM_p last"><sup>13</sup>C NMR (176 MHz, DMSO): δ 173.6, 167.6, 148.8, 147.3, 141.6, 136.8, 132.8, 132.4, 128.9, 128.3, 127.84, 127.50, 126.34, 126.11, 125.5, 125.2, 116.9, 113.0, 108.9, 59.4, 53.1.</div></div><div id="sec5_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i67" class="anchor-spacer"></div><h3 class="article-section__title" id="_i67"> 2-((2-Fluorophenyl)(phenylamino)methyl)-3-hydroxy-6-(hydroxymethyl)-4<i>H</i>-pyran-4-one (<b>39</b>)</h3><div class="NLM_p">Each component (1 mmol) (kojic acid, 2-fluorobenzaldehyde, and aniline) in 15 mL of EtOH was stirred at room temperature for 5 days before being concentrated in vacuo. The remaining crude was extracted with ethyl acetate and brine. The organic layers were dried over anhydrous MgSO<sub>4</sub> and concentrated. The remaining crude was purified using a silica column using ethyl acetate as an eluent. Then, the compound was dissolved in an acetonitrile/H<sub>2</sub>O mixture and lyophilized yielding 135 mg of compound <b>39</b> as an off-white solid (40% yield). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>16</sub>FNO<sub>4</sub> + H<sup>+</sup> [M + H]<sup>+</sup>, 342.1136; found, 342.1139. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.29 (s, 1H), 7.55 (td, <i>J</i> = 7.8, 1.8 Hz, 1H), 7.29 (tdd, <i>J</i> = 7.5, 5.3, 1.8 Hz, 1H), 7.17–7.12 (m, 2H), 7.03–6.98 (m, 2H), 6.59 (d, <i>J</i> = 8.0 Hz, 2H), 6.52 (t, <i>J</i> = 7.3 Hz, 1H), 6.39 (d, <i>J</i> = 8.0 Hz, 1H), 6.22 (s, 1H), 6.07 (d, <i>J</i> = 8.0 Hz, 1H), 5.56 (t, <i>J</i> = 6.2 Hz, 1H), 4.24–4.15 (m, 2H).</div><div class="NLM_p last"><sup>13</sup>C NMR (176 MHz, DMSO): δ 173.7, 167.6, 160.4, 159.0, 147.4, 146.9, 141.5, 129.78, 129.73, 129.13, 129.11, 129.0, 125.85, 125.77, 124.63, 124.61, 117.1, 115.45, 115.33, 112.7, 108.9, 59.4, 46.56, 46.54.</div></div><div id="sec5_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i68" class="anchor-spacer"></div><h3 class="article-section__title" id="_i68"> 3-Hydroxy-6-(hydroxymethyl)-2-((phenylamino)(pyridin-2-yl)methyl)-4<i>H</i>-pyran-4-one (<b>40</b>)</h3><div class="NLM_p">Each component (1 mmol) (kojic acid, 2-pyridinecarboxaldehyde, and aniline) in 15 mL of EtOH was stirred at room temperature for 2 days before being concentrated in vacuo. The remaining crude was extracted with ethyl acetate and water. The organic layers were dried over anhydrous MgSO<sub>4</sub> and concentrated. The remaining crude was purified using a silica column using ethyl acetate as an eluent. Then, the compound was dissolved in an acetonitrile/H<sub>2</sub>O mixture and lyophilized yielding 43 mg of compound <b>40</b> as a bright-yellow solid (13% yield). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>16</sub>N<sub>2</sub>O<sub>4</sub> + H<sup>+</sup> [M + H]<sup>+</sup>, 325.1183; found, 325.1188. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.37 (s, 1H), 8.56–8.53 (m, 1H), 7.82 (td, <i>J</i> = 7.7, 1.8 Hz, 1H), 7.56 (d, <i>J</i> = 7.9 Hz, 1H), 7.36–7.31 (m, 1H), 7.11–7.06 (m, 2H), 6.72 (d, <i>J</i> = 8.0 Hz, 2H), 6.59 (t, <i>J</i> = 7.3 Hz, 1H), 6.49 (d, <i>J</i> = 8.0 Hz, 1H), 6.26 (s, 1H), 5.99 (d, <i>J</i> = 8.0 Hz, 1H), 5.59 (t, <i>J</i> = 6.3 Hz, 1H), 4.23–4.13 (m, 2H).</div><div class="NLM_p last"><sup>13</sup>C NMR (176 MHz, DMSO): δ 173.6, 167.5, 157.8, 148.9, 148.4, 147.0, 142.3, 137.2, 129.0, 123.0, 121.9, 117.0, 112.9, 108.8, 59.4, 54.7.</div></div><div id="sec5_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i69" class="anchor-spacer"></div><h3 class="article-section__title" id="_i69"> 4-(((3-Hydroxy-6-(hydroxymethyl)-4-oxo-4<i>H</i>-pyran-2-yl)(pyridin-2-yl)methyl)amino)benzamide (<b>41</b>)</h3><div class="NLM_p">Each component (1 mmol) (kojic acid, 2-pyridinecarboxaldehyde, and 4-aminobenzamide) in 15 mL of EtOH was stirred at room temperature for 3 days before being concentrated in vacuo. The remaining crude was extracted with ethyl acetate and water. The organic layers were dried over anhydrous MgSO4, concentrated, and dried. The crude was then taken in a mixture of ethyl acetate/MeOH (10:1) and sonicated for 5 min. The solution was discarded and the wash step was repeated with ethyl acetate. Then, the compound was taken in an acetonitrile/H<sub>2</sub>O mixture and lyophilized yielding 235 mg of compound <b>41</b> as a cream color solid (64% yield). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>17</sub>N<sub>3</sub>O<sub>5</sub> + H<sup>+</sup> [M + H]<sup>+</sup>, 368.1241; found, 368.1242. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.46 (s, 1H), 8.56 (ddd, <i>J</i> = 4.8, 1.8, 0.9 Hz, 1H), 7.83 (td, <i>J</i> = 7.7, 1.8 Hz, 1H), 7.66–7.63 (m, 2H), 7.57 (bs, 1H), 7.54 (dd, <i>J</i> = 8.0, 1.2 Hz, 1H), 7.35 (ddd, <i>J</i> = 7.5, 4.8, 1.1 Hz, 1H), 7.01 (d, <i>J</i> = 7.8 Hz, 1H), 6.91 (bs, 1H), 6.73–6.69 (m, 2H), 6.28 (d, <i>J</i> = 1.0 Hz, 1H), 6.06 (d, <i>J</i> = 7.7 Hz, 1H), 5.59 (t, <i>J</i> = 6.2 Hz, 1H), 4.25–4.11 (m, 2H).</div><div class="NLM_p last"><sup>13</sup>C NMR (176 MHz, DMSO): δ 173.6, 167.78, 167.64, 157.2, 149.5, 149.0, 147.9, 142.4, 137.3, 129.0, 123.1, 122.5, 122.0, 111.7, 108.9, 59.4, 54.5.</div></div><div id="sec5_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i70" class="anchor-spacer"></div><h3 class="article-section__title" id="_i70"> 2-((4-Bromophenyl)(pyridin-2-ylamino)methyl)-3-hydroxy-6-(hydroxymethyl)-4<i>H</i>-pyran-4-one (<b>42</b>)</h3><div class="NLM_p">Each component (2 mmol) (kojic acid, 4-bromobenzaldehyde, and 2-aminopyridine) in DMF was stirred at room temperature for 3 days before being concentrated in vacuo. Then, the remaining crude was washed with ethanol yielding compound <b>42</b> as a white solid. HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>15</sub>BrN<sub>2</sub>O<sub>4</sub> + H<sup>+</sup> [M + H]<sup>+</sup>, 403.0288; found, 403.0259. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.31 (s, 1H), 7.95 (ddd, <i>J</i> = 5.1, 1.9, 0.9 Hz, 1H), 7.55–7.52 (m, 2H), 7.43 (ddd, <i>J</i> = 8.8, 7.1, 1.9 Hz, 1H), 7.39 (d, <i>J</i> = 8.5 Hz, 1H), 7.35–7.32 (m, 2H), 6.71 (dt, <i>J</i> = 8.4, 1.0 Hz, 1H), 6.64 (d, <i>J</i> = 8.5 Hz, 1H), 6.55 (ddd, <i>J</i> = 7.0, 5.1, 1.0 Hz, 1H), 6.30–6.29 (m, 1H), 5.62 (t, <i>J</i> = 6.3 Hz, 1H), 4.32–4.22 (m, 2H).</div><div class="NLM_p last"><sup>13</sup>C NMR (176 MHz, DMSO): δ 173.8, 167.5, 157.2, 148.8, 147.3, 141.0, 139.6, 137.0, 131.4, 128.9, 120.4, 112.9, 109.1, 108.9, 59.4, 49.7.</div></div><div id="sec5_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i71" class="anchor-spacer"></div><h3 class="article-section__title" id="_i71"> 3-Hydroxy-6-(hydroxymethyl)-2-(pyridin-2-yl(pyridin-2-ylamino)methyl)-4<i>H</i>-pyran-4-one (<b>43</b>)</h3><div class="NLM_p">Each component (1 mmol) (kojic acid, 2-pyridinecarboxaldehyde, and 2-aminopyridine) in 15 mL of EtOH was stirred at room temperature for 1 day before being concentrated in vacuo. The remaining crude was extracted with ethyl acetate and water. The organic layers were dried over anhydrous MgSO<sub>4</sub>, concentrated, and dried. The remaining crude was purified using a silica column using ethyl acetate/MeOH (1:2) as an eluent. Then, the compound was taken in an acetonitrile/H<sub>2</sub>O mixture and lyophilized yielding 75 mg of compound <b>43</b> as an off-white solid (23% yield). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>17</sub>H<sub>15</sub>N<sub>3</sub>O<sub>4</sub> + H<sup>+</sup> [M + H]<sup>+</sup>, 326.1135; found, 326.1122. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.21 (s, 1H), 8.51 (ddd, <i>J</i> = 4.9, 1.8, 0.9 Hz, 1H), 7.95 (ddd, <i>J</i> = 5.1, 1.9, 0.9 Hz, 1H), 7.78 (td, <i>J</i> = 7.7, 1.8 Hz, 1H), 7.46 (dt, <i>J</i> = 8.0, 1.0 Hz, 1H), 7.43 (ddd, <i>J</i> = 8.7, 7.0, 1.9 Hz, 1H), 7.37 (d, <i>J</i> = 8.3 Hz, 1H), 7.29 (ddd, <i>J</i> = 7.5, 4.8, 1.1 Hz, 1H), 6.76 (dt, <i>J</i> = 8.4, 1.0 Hz, 1H), 6.69 (d, <i>J</i> = 8.3 Hz, 1H), 6.55 (ddd, <i>J</i> = 7.0, 5.0, 1.0 Hz, 1H), 6.28 (d, <i>J</i> = 0.9 Hz, 1H), 5.60 (t, <i>J</i> = 6.3 Hz, 1H), 4.28–4.14 (m, 2H).</div><div class="NLM_p last"><sup>13</sup>C NMR (176 MHz, DMSO): δ 173.8, 167.4, 158.6, 157.3, 149.11, 148.87, 147.3, 141.8, 136.99, 136.97, 122.6, 121.6, 112.8, 109.1, 108.9, 59.4, 52.3.</div></div><div id="sec5_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i72" class="anchor-spacer"></div><h3 class="article-section__title" id="_i72"> 3-Hydroxy-6-(hydroxymethyl)-2-((pyridin-2-ylamino)(pyridin-3-yl)methyl)-4<i>H</i>-pyran-4-one (<b>44</b>)</h3><div class="NLM_p">Each component (1 mmol) (kojic acid, 3-pyridinecarboxaldehyde, and 2-aminopyridine) in 15 mL of EtOH was stirred at room temperature for 2 days before being concentrated in vacuo. The remaining crude was extracted with ethyl acetate and water. The organic layers were dried over anhydrous MgSO<sub>4</sub>, concentrated, and dried. The remaining crude was purified using a silica column using ethyl CH<sub>2</sub>Cl<sub>2</sub>/MeOH (20:1) as an eluent. The crude was then washed with ethyl acetate and sonicated. The wash step was repeated two more times. Then, the compound was taken in an acetonitrile/H<sub>2</sub>O mixture and lyophilized yielding 110 mg of compound <b>44</b> as a cream color solid (34% yield). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>17</sub>H<sub>15</sub>N<sub>3</sub>O<sub>4</sub> + H<sup>+</sup> [M + H]<sup>+</sup>, 326.1135; found, 326.1122. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.21 (s, 1H), 8.51 (ddd, <i>J</i> = 4.9, 1.9, 0.9 Hz, 1H), 7.95 (ddd, <i>J</i> = 5.1, 1.9, 0.8 Hz, 1H), 7.78 (td, <i>J</i> = 7.7, 1.8 Hz, 1H), 7.46 (dd, <i>J</i> = 8.1, 1.3 Hz, 1H), 7.43 (ddd, <i>J</i> = 8.6, 7.0, 1.9 Hz, 1H), 7.37 (d, <i>J</i> = 8.4 Hz, 1H), 7.29 (ddd, <i>J</i> = 7.5, 4.8, 1.2 Hz, 1H), 6.76 (dt, <i>J</i> = 8.4, 1.0 Hz, 1H), 6.69 (d, <i>J</i> = 8.3 Hz, 1H), 6.55 (ddd, <i>J</i> = 7.1, 5.0, 1.0 Hz, 1H), 6.28 (d, <i>J</i> = 1.0 Hz, 1H), 5.60 (t, <i>J</i> = 6.3 Hz, 1H), 4.26–4.14 (m, 2H).</div><div class="NLM_p last"><sup>13</sup>C NMR (176 MHz, DMSO): δ 173.8, 167.4, 158.6, 157.3, 149.1, 148.9, 147.3, 141.8, 137.00, 136.97, 122.6, 121.6, 112.8, 109.10, 108.88, 59.4, 52.3.</div></div><div id="sec5_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i73" class="anchor-spacer"></div><h3 class="article-section__title" id="_i73"> 6-(Hydroxymethyl)-3-methoxy-2-(phenyl(phenylamino)methyl)-4<i>H</i>-pyran-4-one (<b>45</b>)</h3><div class="NLM_p">To a stirring solution of 33 mg (0.1 mmol) of compound <b>19</b> and 13.8 mg of K<sub>2</sub>CO<sub>3</sub> (1 equiv) in 1 mL DMF, 14.1 mg (1 equiv) of iodomethane in 1 mL of DMF was added and stirred for 1 day in ambient temperature. The reaction mixture was then filtered and concentrated. The remaining crude was purified using a silica column using CH<sub>2</sub>Cl<sub>2</sub>/MeOH (10:0.5) as an eluent yielding 29 mg of white powder (86% yield). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>19</sub>NO<sub>4</sub> + H<sup>+</sup> [M + H]<sup>+</sup>, 338.1387; found, 338.1379. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.54–7.49 (m, 2H), 7.42–7.37 (m, 2H), 7.34–7.29 (m, 1H), 7.09 (dd, <i>J</i> = 8.6, 7.2 Hz, 2H), 6.75–6.71 (m, 2H), 6.60 (tt, <i>J</i> = 7.3, 1.1 Hz, 1H), 6.46 (d, <i>J</i> = 8.5 Hz, 1H), 6.27 (d, <i>J</i> = 1.0 Hz, 1H), 5.94 (d, <i>J</i> = 8.5 Hz, 1H), 5.67 (t, <i>J</i> = 6.3 Hz, 1H), 4.32–4.20 (m, 2H), 3.79 (s, 3H).</div><div class="NLM_p"><sup>13</sup>C NMR (126 MHz, DMSO): δ 174.5, 167.8, 158.0, 147.1, 143.3, 138.8, 129.0, 128.7, 127.9, 127.2, 117.2, 113.2, 111.7, 59.9, 59.2, 53.4.</div><div class="NLM_p last"><sup>1</sup>H NMR, <sup>13</sup>C- NMR, purity and assay media stability spectra of compound <b>45</b> is shown in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01275/suppl_file/jm0c01275_si_001.pdf" class="ext-link">Supporting Information</a> (pages S18–S21).</div></div><div id="sec5_29" class="NLM_sec NLM_sec_level_2"><div id="ac_i74" class="anchor-spacer"></div><h3 class="article-section__title" id="_i74"> 2-((2-Fluorophenyl)(pyridin-2-ylamino)methyl)-6-(hydroxymethyl)-3-methoxy-4<i>H</i>-pyran-4-one (<b>46</b>)</h3><div class="NLM_p">To a stirring solution of 25 mg (0.067 mmol) of compound <b>10</b> and 9.3 mg of K<sub>2</sub>CO<sub>3</sub> (0.063 mmol) in 1 mL DMF, 4.5 mg (0.5 equiv) of iodomethane in 1 mL DMF was added and stirred for 30 min. Then, another portion of 4.5 mg (0.5 equiv) of iodomethane in 0.5 mL of DMF was added and stirred for 30 min. The reaction mixture was then concentrated. The remaining crude was purified using a silica column using ethyl acetate as an eluent yielding 10 mg (0.028 mmol) of compound <b>46</b> as a white powder (42% yield). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C1<sub>9</sub>H<sub>17</sub>FN<sub>2</sub>O<sub>4</sub> + H<sup>+</sup> [M + H]<sup>+</sup>, 357.1245; found, 357.1222. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.97 (dd, <i>J</i> = 5.2, 1.8 Hz, 1H), 7.56 (td, <i>J</i> = 7.8, 1.9 Hz, 1H), 7.47 (d, <i>J</i> = 8.3 Hz, 1H), 7.43 (ddd, <i>J</i> = 8.7, 7.0, 2.0 Hz, 1H), 7.39–7.33 (m, 1H), 7.24–7.18 (m, 2H), 6.90 (d, <i>J</i> = 8.3 Hz, 1H), 6.71 (d, <i>J</i> = 8.3 Hz, 1H), 6.58–6.53 (m, 1H), 6.29 (s, 1H), 5.69 (t, <i>J</i> = 6.2 Hz, 1H), 4.30–4.20 (m, 2H), 3.69 (s, 3H).</div><div class="NLM_p last"><sup>13</sup>C NMR (126 MHz, DMSO): δ 174.7, 167.8, 160.7, 158.7, 157.7, 157.0, 147.4, 143.0, 137.0, 129.86, 129.80, 129.08, 129.06, 126.27, 126.16, 124.67, 124.64, 115.55, 115.38, 113.0, 111.8, 109.3, 59.34, 59.23, 44.82, 44.78.</div></div><div id="sec5_30" class="NLM_sec NLM_sec_level_2"><div id="ac_i75" class="anchor-spacer"></div><h3 class="article-section__title" id="_i75"> <i>tert</i>-Butyl(3-(2-(2-(3-aminopropoxy)ethoxy)ethoxy)propyl)carbamate (<b>S2</b>)</h3><div class="NLM_p last">Synthesis of <b>S2</b> was achieved using the procedure as described earlier.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> To a solution of 1.01 g (4.6 mmol) of 1,13-diamino-4,7,10-trioxatridecane (<b>S1</b>) in 20 mL of dry CH<sub>2</sub>Cl<sub>2</sub>, 0.5 g (2.3 mmol) of di-<i>tert</i>-butyl dicarbonate in 10 mL of dry CH<sub>2</sub>Cl<sub>2</sub> was added dropwise over 1 h and stirred at room temperature for overnight. Then, the reaction mixture was washed with H<sub>2</sub>O three times, dried over anhydrous MgSO<sub>4</sub>, and concentrated in vacuo. The remaining colorless oil was used without further purification.</div></div><div id="sec5_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i76" class="anchor-spacer"></div><h3 class="article-section__title" id="_i76"> <i>tert</i>-Butyl(15-oxo-4,7,10-trioxa-14-azatriacontyl)carbamate (<b>S3</b>)</h3><div class="NLM_p">Intermediate <b>S2</b> (30 mg, 0.1 mmol) was added to a solution of palmitic acid (38.5 mg, 0.15 mmol), EDCI (29 mg, 0.15 mmol), HOAt (20 mg, 0.15 mmol), and DIPEA (26 μL, 0.15 mmol) in 10 mL CH<sub>2</sub>Cl<sub>2</sub>, and the reaction mixture was stirred at room temperature for 4 h. Then, the reaction was quenched with H<sub>2</sub>O and extracted to CH<sub>2</sub>Cl<sub>2</sub>. The organic phase was dried over anhydrous MgSO<sub>4</sub> and concentrated in vacuo. The remaining crude was purified over a silica column using <i>n</i>-Hex–EtAc (1:1, then 1:4, then Et Ac only) as an eluent. <b>S3</b> was obtained as a white solid (60 mg, 0.11 mmol, 98% yield). LRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>31</sub>H<sub>62</sub>N<sub>2</sub>O<sub>6</sub> + H<sup>+</sup> [M + H]<sup>+</sup>, 559.47; found, 559.06. <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>): δ 3.64 (dp, <i>J</i> = 5.1, 2.4 Hz, 4H), 3.59 (dq, <i>J</i> = 7.0, 3.7, 3.0 Hz, 4H), 3.51 (td, <i>J</i> = 6.1, 4.2 Hz, 4H), 3.25 (t, <i>J</i> = 6.8 Hz, 2H), 3.12 (t, <i>J</i> = 6.8 Hz, 2H), 2.17 (t, <i>J</i> = 7.5 Hz, 2H), 1.74 (dp, <i>J</i> = 17.2, 6.5 Hz, 4H), 1.60 (q, <i>J</i> = 7.2 Hz, 2H), 1.43 (s, 9H), 1.36–1.25 (m, 24H), 0.90 (t, <i>J</i> = 6.9 Hz, 3H).</div><div class="NLM_p last"><sup>13</sup>C NMR (126 MHz, MeOD): δ 176.3, 158.5, 79.8, 71.6, 71.26, 71.25, 69.89, 69.86, 38.7, 37.8, 37.2, 33.1, 30.91, 30.83, 30.81, 30.79, 30.75, 30.67, 30.51, 30.47, 30.43, 30.32, 28.8, 27.1, 23.8, 14.5.</div></div><div id="sec5_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i77" class="anchor-spacer"></div><h3 class="article-section__title" id="_i77"> 2-(6-Hydroxy-3-oxoi-3H-xanthen-9-yl)-5-(3-(15-oxo-4,7,10-trioxa-14-azatriacontyl)thioureido)benzoic Acid (<b>S4</b>)</h3><div class="NLM_p last">A total of 2 mL of 4 M HCl solution in 1,4-dioxane was added to a solution of <b>S3</b> (10 mg, 0.017 mmol) in 10 mL of MeOH and was stirred at room temperature overnight. Then, the reaction mixture was concentrated, the remaining crude (7.4 mg, 0.015 mmol) was added into 2 mL of DMF, and DIPEA (7.8 μL, 0.045 mmol, 3 equiv) was added followed by FITC addition (8.2 mg, 0.021 mmol, 1.4 equiv) in 1 mL of DMF. The reaction mixture was stirred at room temperature overnight. Then, the solvent was evaporated and the remaining crude was purified over reverse-phase HPLC-MS (Agilent 1260 Infinity coupled to Agilent 6120 Quadripole), with a C18 reverse-phase column (VP125/21 Nucleodur C18 Gravity, 5 μm). <b>S4</b> was taken in a mixture of acetonitrile/H<sub>2</sub>O and lyophilized yielding 5 mg of orange powder (43% yield). HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>47</sub>H<sub>65</sub>N<sub>3</sub>O<sub>9</sub>S + H<sup>+</sup> [M + H]<sup>+</sup>, 848.4514; found, 848.4467. <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>): δ 8.14 (d, <i>J</i> = 2.0 Hz, 1H), 7.79 (d, <i>J</i> = 8.3 Hz, 1H), 7.17 (d, <i>J</i> = 8.2 Hz, 1H), 6.82–6.66 (m, 4H), 6.65–6.48 (m, 2H), 3.74 (s, 2H), 3.63 (qq, <i>J</i> = 5.0, 3.3, 2.5 Hz, 8H), 3.56 (dd, <i>J</i> = 5.8, 3.3 Hz, 2H), 3.49 (t, <i>J</i> = 6.2 Hz, 2H), 3.23 (t, <i>J</i> = 6.9 Hz, 2H), 2.15 (t, <i>J</i> = 7.5 Hz, 2H), 1.93 (p, <i>J</i> = 6.2 Hz, 2H), 1.73 (p, <i>J</i> = 6.5 Hz, 2H), 1.57 (t, <i>J</i> = 7.3 Hz, 3H), 1.28 (d, <i>J</i> = 9.6 Hz, 23H), 0.89 (t, <i>J</i> = 6.9 Hz, 3H). <sup>13</sup>C NMR (176 MHz, MeOD): δ 182.6, 176.3, 171.2, 154.2, 142.4, 130.3, 113.6, 111.5, 103.5, 71.52, 71.44, 71.23, 71.14, 70.4, 69.9, 37.8, 37.2, 33.1, 30.80, 30.79, 30.76, 30.72, 30.63, 30.48, 30.45, 30.44, 30.30, 29.9, 27.1, 23.7, 14.4.</div></div><div id="sec5_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i78" class="anchor-spacer"></div><h3 class="article-section__title" id="_i78"> Synthesis of FITC-YAP<sup>50–100</sup></h3><div class="NLM_p last">The YAP peptide (Fmoc-AGHQIVHVRGDSETDLEALFNAVMNPKTANVPQTVPMRLRKLPDSFFKPPE) was synthesized on a Syro I peptide synthesizer (Multisyntech) following standard Fmoc-protocols for solid-phase peptide synthesis using Rink amide ChemMatrix resin (Aldrich, 0.28 mmol/g, 0.05 mmol). The N-terminal Fmoc group in the resulting peptide on the resin was deprotected using 25% piperidine and reacted with Fmoc-O<sub>2</sub>Oc-OH (0.2 mmol, 4 equiv), HATU (0.2 mmol, 4 equiv), and DIPEA (0.4 mmol, 8 equiv) in DMF for 2 h. The N-terminal Fmoc group was then deprotected followed by the reaction with 5-isothiocyanatefluorescein (0.250 mmol, 5 equiv) and DIPEA (0.5 mmol, 10 equiv) in DMF at room temperature for 3 h in the dark. The resin was washed and the coupling procedure was repeated for an additional time and this time shaken in the dark at rt for 16 h. The peptidyl resin (0.05 mmol, 1 equiv) was cleaved with 1 mL of cleavage solution (triisopropylsilane 2.5%, water 2.5% in trifluoroacetyl) for 3 h at room temperature. The peptide was precipitated in 40 mL of cold diethyl ether/petroleum ether (1:1) upon 10 min of centrifugation. The pellet was washed twice with 40 mL of diethyl ether. The crude was dried over vacuum, dissolved in water, and lyophilized. Then, the crude was dissolved in 1 mL of DMSO and the peptide was purified by reverse-phase HPLC. HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>280</sub>H<sub>425</sub>N<sub>75</sub>O<sub>81</sub>S<sub>3</sub> + [M + 5H]<sup>5+</sup>, 1247.0194; found, 1247.0158 calcd for +[M + 6H]<sup>6+</sup>, 1039.3507; found, 1039.3481; calcd for +[M + 7H]<sup>7+</sup>, 891.0159; found, 891.0145.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i79"><a href="/doi/suppl/10.1021/acs.jmedchem.0c01275" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05737" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05737" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c01275?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01275</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Full-protein ESI–MS analysis of recombinant hTEAD4 (acylated), hTEAD4 (nonacylated), and hTEAD4 (nonacylated) reaction with CPM; tandem-MS analysis of the hTEAD4 (nonacylated) reaction with CPM; competitive inhibition of the thiol conjugation signal with TED-347; saturation binding curve of FITC-palmitate to hTEAD4 (nonacylated) and competitive inhibition curve of niflumic acid (<b>2</b>) in the FP assay; time- and concentration-dependent inhibition of hTEAD4; specificity of <b>19</b> for TEAD homologues; <sup>1</sup>H–<sup>15</sup>N HSQC spectrum of mTEAD4 (nonacylated) treated with <b>19</b> or <b>45</b>; saturation binding curves of FITC-labeled YAP<sup>50–100</sup> with TEAD4 proteins; sequences of the TEAD proteins used in the study; scheme for the synthesis of FITC-palmitate; and <sup>1</sup>H NMR, <sup>13</sup>C NMR, and UPLC spectra of compounds <b>19</b> and <b>45</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01275/suppl_file/jm0c01275_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings with biological data (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01275/suppl_file/jm0c01275_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01275/suppl_file/jm0c01275_si_001.pdf">jm0c01275_si_001.pdf (1.17 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01275/suppl_file/jm0c01275_si_002.csv">jm0c01275_si_002.csv (3.89 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c01275" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63050" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63050" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Herbert Waldmann</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemical Biology, Max Planck Institute
of Molecular Physiology, Otto-Hahn Straße 11, 44227 Dortmund, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Faculty
of
Chemistry and Chemical Biology, Technical
University Dortmund, Otto-Hahn-Straße 6, 44227 Dortmund, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9606-7247" title="Orcid link">http://orcid.org/0000-0002-9606-7247</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#6c04091e0e091e18421b0d0008010d02022c011c054108031e180119020842011c0b420809"><span class="__cf_email__" data-cfemail="533b3621313621277d24323f373e323d3d133e233a7e373c21273e263d377d3e23347d3736">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hacer Karatas</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemical Biology, Max Planck Institute
of Molecular Physiology, Otto-Hahn Straße 11, 44227 Dortmund, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7577-0320" title="Orcid link">http://orcid.org/0000-0002-7577-0320</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mohammad Akbarzadeh</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemical Biology, Max Planck Institute
of Molecular Physiology, Otto-Hahn Straße 11, 44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hélène Adihou</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemical Biology, AstraZeneca−Max
Planck Institute Satellite Unit, Otto-Hahn-Strasse 11, 44227 Dortmund, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry, Research and Early Development Cardiovascular, Renal and
Metabolism, BioPharmaceuticals R&D,
AstraZeneca, SE-431 83 Gothenburg, Sweden</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gernot Hahne</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemical Biology, Max Planck Institute
of Molecular Physiology, Otto-Hahn Straße 11, 44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ajaybabu V. Pobbati</span> - <span class="hlFld-Affiliation affiliation">Institute
of Molecular and Cell Biology, A*STAR, 61 Biopolis Drive, 138673 Singapore, Singapore</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5744-265X" title="Orcid link">http://orcid.org/0000-0001-5744-265X</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Elizabeth Yihui Ng</span> - <span class="hlFld-Affiliation affiliation">Experimental
Drug Development Centre (EDDC), Agency for
Science, Technology and Research (A*STAR), 10 Biopolis Road, Chromos, #05-01, 138670, Singapore</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stéphanie M. Guéret</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemical Biology, AstraZeneca−Max
Planck Institute Satellite Unit, Otto-Hahn-Strasse 11, 44227 Dortmund, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry, Research and Early Development Cardiovascular, Renal and
Metabolism, BioPharmaceuticals R&D,
AstraZeneca, SE-431 83 Gothenburg, Sweden</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7220-9506" title="Orcid link">http://orcid.org/0000-0001-7220-9506</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sonja Sievers</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemical Biology, Max Planck Institute
of Molecular Physiology, Otto-Hahn Straße 11, 44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Axel Pahl</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemical Biology, Max Planck Institute
of Molecular Physiology, Otto-Hahn Straße 11, 44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Malte Metz</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemical Biology, Max Planck Institute
of Molecular Physiology, Otto-Hahn Straße 11, 44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sarah Zinken</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemical Biology, Max Planck Institute
of Molecular Physiology, Otto-Hahn Straße 11, 44227 Dortmund, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Faculty
of
Chemistry and Chemical Biology, Technical
University Dortmund, Otto-Hahn-Straße 6, 44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lara Dötsch</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemical Biology, Max Planck Institute
of Molecular Physiology, Otto-Hahn Straße 11, 44227 Dortmund, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Faculty
of
Chemistry and Chemical Biology, Technical
University Dortmund, Otto-Hahn-Straße 6, 44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christine Nowak</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemical Biology, Max Planck Institute
of Molecular Physiology, Otto-Hahn Straße 11, 44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sasikala Thavam</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemical Biology, Max Planck Institute
of Molecular Physiology, Otto-Hahn Straße 11, 44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alexandra Friese</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemical Biology, Max Planck Institute
of Molecular Physiology, Otto-Hahn Straße 11, 44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">CongBao Kang</span> - <span class="hlFld-Affiliation affiliation">Experimental
Drug Development Centre (EDDC), Agency for
Science, Technology and Research (A*STAR), 10 Biopolis Road, Chromos, #05-01, 138670, Singapore</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9886-9374" title="Orcid link">http://orcid.org/0000-0002-9886-9374</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wanjin Hong</span> - <span class="hlFld-Affiliation affiliation">Institute
of Molecular and Cell Biology, A*STAR, 61 Biopolis Drive, 138673 Singapore, Singapore</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i80">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13600" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13600" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Max Planck Institute of Molecular Physiology Dortmund Protein Facility (DPF) for the expression and purification of hTEAD4 and mTEAD4_Cys360Ala and purification of mTEAD4_Cys360Ser proteins.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">Ala</td><td class="NLM_def"><p class="first last">alanine</p></td></tr><tr><td class="NLM_term">brs</td><td class="NLM_def"><p class="first last">broad signal</p></td></tr><tr><td class="NLM_term">CPM</td><td class="NLM_def"><p class="first last"><i>N</i>-(4-(7-diethylamino-4-methylcoumarin-3-yl)phenyl)maleimide</p></td></tr><tr><td class="NLM_term">Cys</td><td class="NLM_def"><p class="first last">cysteine</p></td></tr><tr><td class="NLM_term">ESI–MS</td><td class="NLM_def"><p class="first last">electron spray ionization–mass spectrometry</p></td></tr><tr><td class="NLM_term">FITC</td><td class="NLM_def"><p class="first last">fluorescein isothiocyanate</p></td></tr><tr><td class="NLM_term">FP</td><td class="NLM_def"><p class="first last">fluorescence polarization</p></td></tr><tr><td class="NLM_term">mTEAD4</td><td class="NLM_def"><p class="first last">mouse TEAD4</p></td></tr><tr><td class="NLM_term">Phe</td><td class="NLM_def"><p class="first last">phenylalanine</p></td></tr><tr><td class="NLM_term">Ser</td><td class="NLM_def"><p class="first last">serine</p></td></tr><tr><td class="NLM_term">TEAD</td><td class="NLM_def"><p class="first last">transcriptional-enhanced associate domain</p></td></tr><tr><td class="NLM_term"><i>T</i><sub>m</sub></td><td class="NLM_def"><p class="first last">melting temperature</p></td></tr><tr><td class="NLM_term">YAP</td><td class="NLM_def"><p class="first last">yes-associated protein</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i82">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_11851" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_11851" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 47 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Holden, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, C. N.</span></span> <span> </span><span class="NLM_article-title">Targeting the hippo pathway and cancer through the TEAD family of transcription factors</span>. <i>Cancers</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">81</span>, <span class="refDoi"> DOI: 10.3390/cancers10030081</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.3390%2Fcancers10030081" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtV2htLfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=81&author=J.+K.+Holdenauthor=C.+N.+Cunningham&title=Targeting+the+hippo+pathway+and+cancer+through+the+TEAD+family+of+transcription+factors&doi=10.3390%2Fcancers10030081"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the hippo pathway and cancer through the TEAD family of transcription factors</span></div><div class="casAuthors">Holden, Jeffrey K.; Cunningham, Christian N.</div><div class="citationInfo"><span class="NLM_cas:title">Cancers</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">81/1-81/15</span>CODEN:
                <span class="NLM_cas:coden">CANCCT</span>;
        ISSN:<span class="NLM_cas:issn">2072-6694</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  The Hippo pathway is a crit. transcriptional signaling pathway that regulates cell growth, proliferation and organ development.  The transcriptional enhanced assoc. domain (TEAD) protein family consists of four paralogous transcription factors that function to modulate gene expression in response to the Hippo signaling pathway.  Transcriptional activation of these proteins occurs upon binding to the co-activator YAP/TAZ whose entry into the nucleus is regulated by Lats1/2 kinase.  In recent years, it has become apparent that the dysregulation and/or overexpression of Hippo pathway effectors is implicated in a wide range of cancers, including prostate, gastric and liver cancer.  A large body of work has been dedicated to understanding the therapeutic potential of modulating the phosphorylation and localization of YAP/TAZ.  However, YAP/TAZ are considered to be natively unfolded and may be intractable as drug targets.  Therefore, TEAD proteins present themselves as an excellent therapeutic target for intervention of the Hippo pathway.  This review summarizes the functional role of TEAD proteins in cancer and assesses the therapeutic potential of antagonizing TEAD function in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFz0v8zUhnTrVg90H21EOLACvtfcHk0lgT4KvOevVJ2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtV2htLfJ&md5=793c03200885c0349d40a1d92460b652</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.3390%2Fcancers10030081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcancers10030081%26sid%3Dliteratum%253Aachs%26aulast%3DHolden%26aufirst%3DJ.%2BK.%26aulast%3DCunningham%26aufirst%3DC.%2BN.%26atitle%3DTargeting%2520the%2520hippo%2520pathway%2520and%2520cancer%2520through%2520the%2520TEAD%2520family%2520of%2520transcription%2520factors%26jtitle%3DCancers%26date%3D2018%26volume%3D10%26spage%3D81%26doi%3D10.3390%2Fcancers10030081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huh, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeong, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H.</span></span> <span> </span><span class="NLM_article-title">Regulation of TEAD transcription factors in cancer biology</span>. <i>Cells</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">600</span>, <span class="refDoi"> DOI: 10.3390/cells8060600</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.3390%2Fcells8060600" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjsFalsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2019&pages=600&author=H.+Huhauthor=D.+Kimauthor=H.-S.+Jeongauthor=H.+Park&title=Regulation+of+TEAD+transcription+factors+in+cancer+biology&doi=10.3390%2Fcells8060600"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of TEAD transcription factors in cancer biology</span></div><div class="casAuthors">Huh, Hyunbin D.; Kim, Dong Hyeon; Jeong, Han-Sol; Park, Hyun Woo</div><div class="citationInfo"><span class="NLM_cas:title">Cells</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">600</span>CODEN:
                <span class="NLM_cas:coden">CELLC6</span>;
        ISSN:<span class="NLM_cas:issn">2073-4409</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Transcriptional enhanced assoc. domain (TEAD) transcription factors play important roles during development, cell proliferation, regeneration, and tissue homeostasis.  TEAD integrates with and coordinates various signal transduction pathways including Hippo, Wnt, transforming growth factor beta (TGFβ), and epidermal growth factor receptor (EGFR) pathways.  TEAD deregulation affects well-established cancer genes such as KRAS, BRAF, LKB1, NF2, and MYC, and its transcriptional output plays an important role in tumor progression, metastasis, cancer metab., immunity, and drug resistance.  To date, TEADs have been recognized to be key transcription factors of the Hippo pathway.  Therefore, most studies are focused on the Hippo kinases and YAP/TAZ, whereas the Hippo-dependent and Hippo-independent regulators and regulations governing TEAD only emerged recently.  Deregulation of the TEAD transcriptional output plays important roles in tumor progression and serves as a prognostic biomarker due to high correlation with clinicopathol. parameters in human malignancies.  In addn., discovering the mol. mechanisms of TEAD, such as post-translational modifications and nucleocytoplasmic shuttling, represents an important means of modulating TEAD transcriptional activity.  Collectively, this review highlights the role of TEAD in multistep-tumorigenesis by interacting with upstream oncogenic signaling pathways and controlling downstream target genes, which provides unprecedented insight and rationale into developing TEAD-targeted anticancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqV83sN1ayn2LVg90H21EOLACvtfcHk0lhEJLTOTfD5TQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjsFalsrk%253D&md5=d5c87c97ba762881edc0dd4e354ad334</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.3390%2Fcells8060600&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcells8060600%26sid%3Dliteratum%253Aachs%26aulast%3DHuh%26aufirst%3DH.%26aulast%3DKim%26aufirst%3DD.%26aulast%3DJeong%26aufirst%3DH.-S.%26aulast%3DPark%26aufirst%3DH.%26atitle%3DRegulation%2520of%2520TEAD%2520transcription%2520factors%2520in%2520cancer%2520biology%26jtitle%3DCells%26date%3D2019%26volume%3D8%26spage%3D600%26doi%3D10.3390%2Fcells8060600" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, K.-L.</span></span> <span> </span><span class="NLM_article-title">Regulation of the hippo pathway transcription factor TEAD</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">862</span>– <span class="NLM_lpage">872</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2017.09.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1016%2Fj.tibs.2017.09.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=28964625" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFeqtr3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2017&pages=862-872&author=K.+C.+Linauthor=H.+W.+Parkauthor=K.-L.+Guan&title=Regulation+of+the+hippo+pathway+transcription+factor+TEAD&doi=10.1016%2Fj.tibs.2017.09.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of the Hippo Pathway Transcription Factor TEAD</span></div><div class="casAuthors">Lin, Kimberly C.; Park, Hyun Woo; Guan, Kun-Liang</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">862-872</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The TEAD transcription factor family is best known for transcriptional output of the Hippo signaling pathway and has been implicated in processes such as development, cell growth and proliferation, tissue homeostasis, and regeneration.  Our understanding of the functional importance of TEADs has increased dramatically since its initial discovery three decades ago.  The majority of our knowledge of TEADs is in the context of Hippo signaling as nuclear DNA-binding proteins passively activated by Yes-assocd. protein (YAP) and transcriptional activator with PDZ-binding domain (TAZ), transcription coactivators downstream of the Hippo pathway.  However, recent studies suggest that TEAD itself is actively regulated.  Here, we highlight evidence demonstrating Hippo-independent regulation of TEADs and the potential impacts these studies may have on new cancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4QrFEsHUdgbVg90H21EOLACvtfcHk0lhEJLTOTfD5TQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFeqtr3J&md5=0fa282492aa43a0a513f557037adee52</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2017.09.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2017.09.003%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DK.%2BC.%26aulast%3DPark%26aufirst%3DH.%2BW.%26aulast%3DGuan%26aufirst%3DK.-L.%26atitle%3DRegulation%2520of%2520the%2520hippo%2520pathway%2520transcription%2520factor%2520TEAD%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2017%26volume%3D42%26spage%3D862%26epage%3D872%26doi%3D10.1016%2Fj.tibs.2017.09.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">To, K. F.</span></span> <span> </span><span class="NLM_article-title">The TEAD family and its oncogenic role in promoting tumorigenesis</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">138</span>, <span class="refDoi"> DOI: 10.3390/ijms17010138</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.3390%2Fijms17010138" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=26805820" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC2sXktFOlsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=138&author=Y.+Zhouauthor=T.+Huangauthor=A.+S.+Chengauthor=J.+Yuauthor=W.+Kangauthor=K.+F.+To&title=The+TEAD+family+and+its+oncogenic+role+in+promoting+tumorigenesis&doi=10.3390%2Fijms17010138"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The TEAD family and its oncogenic role in promoting tumorigenesis</span></div><div class="casAuthors">Zhou, Yuhang; Huang, Tingting; Cheng, Alfred S. L.; Yu, Jun; Kang, Wei; To, Ka Fai</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">138/1-138/15</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">The TEAD family of transcription factors is necessary for developmental processes.  The family members contain a TEA domain for the binding with DNA elements and a transactivation domain for the interaction with transcription coactivators.  TEAD proteins are required for the participation of coactivators to transmit the signal of pathways for the downstream signaling processes.  TEADs also play an important role in tumor initiation and facilitate cancer progression via activating a series of progression-inducing genes, such as CTGF, Cyr61, Myc and Gli2.  Recent studies have highlighted that TEADs, together with their coactivators, promote or even act as the crucial parts in the development of various malignancies, such as liver, ovarian, breast and prostate cancers.  Furthermore, TEADs are proposed to be useful prognostic biomarkers due to the ideal correlation between high expression and clinicopathol. parameters in gastric, breast, ovarian and prostate cancers.  In this review, we summarize the functional role of TEAD proteins in tumorigenesis and discuss the key role of TEAD transcription factors in the linking of signal cascade transductions.  Improved knowledge of the TEAD proteins will be helpful for deep understanding of the mol. mechanisms of tumorigenesis and identifying ideal predictive or prognostic biomarkers, even providing clin. translation for anticancer therapy in human cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWR3IdQcw6YLVg90H21EOLACvtfcHk0lhEJLTOTfD5TQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXktFOlsw%253D%253D&md5=3ca058d5405586b1397cc1e0c5b70b79</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.3390%2Fijms17010138&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms17010138%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DT.%26aulast%3DCheng%26aufirst%3DA.%2BS.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DKang%26aufirst%3DW.%26aulast%3DTo%26aufirst%3DK.%2BF.%26atitle%3DThe%2520TEAD%2520family%2520and%2520its%2520oncogenic%2520role%2520in%2520promoting%2520tumorigenesis%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2016%26volume%3D17%26spage%3D138%26doi%3D10.3390%2Fijms17010138" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Knight, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shepherd, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizzo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brewer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jhavar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eeles, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falconer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovacs, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garrett, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norman, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shipley, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudson, D. L.</span></span> <span> </span><span class="NLM_article-title">TEAD1 and c-Cbl are novel prostate basal cell markers that correlate with poor clinical outcome in prostate cancer</span>. <i>Br. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">1849</span>– <span class="NLM_lpage">1858</span>, <span class="refDoi"> DOI: 10.1038/sj.bjc.6604774</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1038%2Fsj.bjc.6604774" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=19002168" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVajtbrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2008&pages=1849-1858&author=J.+F.+Knightauthor=C.+J.+Shepherdauthor=S.+Rizzoauthor=D.+Brewerauthor=S.+Jhavarauthor=A.+R.+Dodsonauthor=C.+S.+Cooperauthor=R.+Eelesauthor=A.+Falconerauthor=G.+Kovacsauthor=M.+D.+Garrettauthor=A.+R.+Normanauthor=J.+Shipleyauthor=D.+L.+Hudson&title=TEAD1+and+c-Cbl+are+novel+prostate+basal+cell+markers+that+correlate+with+poor+clinical+outcome+in+prostate+cancer&doi=10.1038%2Fsj.bjc.6604774"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">TEAD1 and c-Cbl are novel prostate basal cell markers that correlate with poor clinical outcome in prostate cancer</span></div><div class="casAuthors">Knight, J. F.; Shepherd, C. J.; Rizzo, S.; Brewer, D.; Jhavar, S.; Dodson, A. R.; Cooper, C. S.; Eeles, R.; Falconer, A.; Kovacs, G.; Garrett, M. D.; Norman, A. R.; Shipley, J.; Hudson, D. L.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Cancer</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1849-1858</span>CODEN:
                <span class="NLM_cas:coden">BJCAAI</span>;
        ISSN:<span class="NLM_cas:issn">0007-0920</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Prostate cancer is the most frequently diagnosed male cancer, and its clin. outcome is difficult to predict.  The disease may involve the inappropriate expression of genes that normally control the proliferation of epithelial cells in the basal layer and their differentiation into luminal cells.  Our aim was to identify novel basal cell markers and assess their prognostic and functional significance in prostate cancer.  RNA from basal and luminal cells isolated from benign tissue by immunoguided laser-capture microdissection was subjected to expression profiling.  We identified 112 and 267 genes defining basal and luminal populations, resp.  The transcription factor TEAD1 and the ubiquitin ligase c-Cbl were identified as novel basal cell markers.  Knockdown of either marker using siRNA in prostate cell lines led to decreased cell growth in PC3 and disrupted acinar formation in a 3D culture system of RWPE1.  Analyses of prostate cancer tissue microarray staining established that increased protein levels of either marker were assocd. with decreased patient survival independent of other clinicopathol. metrics.  These data are consistent with basal features impacting on the development and clin. course of prostate cancers.  British Journal of Cancer (2008) 99, 1849-1858. doi:10.1038/sj.bjc.6604774 www.bjcancer.com Published online 11 Nov. 2008.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3D_XIMg8w87Vg90H21EOLACvtfcHk0li0kifNVgwwZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVajtbrO&md5=f622b94acce5d78cacdae3225d0d26c7</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjc.6604774&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjc.6604774%26sid%3Dliteratum%253Aachs%26aulast%3DKnight%26aufirst%3DJ.%2BF.%26aulast%3DShepherd%26aufirst%3DC.%2BJ.%26aulast%3DRizzo%26aufirst%3DS.%26aulast%3DBrewer%26aufirst%3DD.%26aulast%3DJhavar%26aufirst%3DS.%26aulast%3DDodson%26aufirst%3DA.%2BR.%26aulast%3DCooper%26aufirst%3DC.%2BS.%26aulast%3DEeles%26aufirst%3DR.%26aulast%3DFalconer%26aufirst%3DA.%26aulast%3DKovacs%26aufirst%3DG.%26aulast%3DGarrett%26aufirst%3DM.%2BD.%26aulast%3DNorman%26aufirst%3DA.%2BR.%26aulast%3DShipley%26aufirst%3DJ.%26aulast%3DHudson%26aufirst%3DD.%2BL.%26atitle%3DTEAD1%2520and%2520c-Cbl%2520are%2520novel%2520prostate%2520basal%2520cell%2520markers%2520that%2520correlate%2520with%2520poor%2520clinical%2520outcome%2520in%2520prostate%2520cancer%26jtitle%3DBr.%2520J.%2520Cancer%26date%3D2008%26volume%3D99%26spage%3D1849%26epage%3D1858%26doi%3D10.1038%2Fsj.bjc.6604774" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span> <span> </span><span class="NLM_article-title">TEAD4 exerts pro-metastatic effects and is negatively regulated by miR6839-3p in lung adenocarcinoma progression</span>. <i>J. Cell. Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">3560</span>– <span class="NLM_lpage">3571</span>, <span class="refDoi"> DOI: 10.1111/jcmm.13634</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1111%2Fjcmm.13634" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=29667772" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtF2mtbbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2018&pages=3560-3571&author=Q.+Zhangauthor=H.+Fanauthor=Q.+Zouauthor=H.+Liuauthor=B.+Wanauthor=S.+Zhuauthor=Y.+Huauthor=H.+Liauthor=C.+Zhangauthor=L.+Zhouauthor=Q.+Zhuauthor=K.+Xiaoauthor=J.+Zhangauthor=P.+Zhanauthor=T.+Lvauthor=Y.+Song&title=TEAD4+exerts+pro-metastatic+effects+and+is+negatively+regulated+by+miR6839-3p+in+lung+adenocarcinoma+progression&doi=10.1111%2Fjcmm.13634"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">TEAD4 exerts pro-metastatic effects and is negatively regulated by miR6839-3p in lung adenocarcinoma progression</span></div><div class="casAuthors">Zhang, Qun; Fan, Hang; Zou, Qian; Liu, Hongda; Wan, Bing; Zhu, Suhua; Hu, Yangbo; Li, Huijuan; Zhang, ChenXi; Zhou, Li; Zhu, Qingqing; Xiao, Kunhong; Zhang, Jianya; Zhan, Ping; Lv, Tangfeng; Song, Yong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cellular and Molecular Medicine</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3560-3571</span>CODEN:
                <span class="NLM_cas:coden">JCMMC9</span>;
        ISSN:<span class="NLM_cas:issn">1582-4934</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Several studies have shown the tumorigenesis role of transcriptional enhancer assoc. domain (TEAD) proteins; here, we initially explored expression, function and signalling mechanisms of TEAD4 in lung adenocarcinoma (LAD).  Both the mRNA and protein levels of TEAD4 were increased in LAD tissues than those in adjacent nontumorous tissues.  Besides, database search indicated a poorer clin. outcome in LAD patients with higher TEAD4 expression, revealing its potential tumor-promoting role.  Therefore, we conducted cellular expts. to further investigate its effect on tumor phenotypes.  Accordingly, TEAD4 showed little impact on LAD cell cycle, proliferation, or apoptosis.  However, silencing TEAD4 remarkably attenuated cell migration and invasion capacities.  Consistently, several important epithelial-mesenchymal transition (EMT) markers such as E-cadherin and Slug were consequently altered by silencing TEAD4.  Furthermore, we identified a novel TEAD4-targeted microRNA, namely miR6839-3p, and confirmed its function in suppressing TEAD4 expression.  Finally, the impact of overexpressing miR6839-3p mimics on LAD progression was validated, which showed a similar pattern with TEAD4 knockdown cells.  Taken together, our data not only revealed the significant role of TEAD4 in promoting LAD progression and predicting clin. outcome but also distinguished miR6839-3p mimics as a promising therapeutic direction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEe22Yu3PQerVg90H21EOLACvtfcHk0li0kifNVgwwZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtF2mtbbJ&md5=d63af6f230a14d1f70af18b7ea985b82</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1111%2Fjcmm.13634&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fjcmm.13634%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DZou%26aufirst%3DQ.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DWan%26aufirst%3DB.%26aulast%3DZhu%26aufirst%3DS.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DZhu%26aufirst%3DQ.%26aulast%3DXiao%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DZhan%26aufirst%3DP.%26aulast%3DLv%26aufirst%3DT.%26aulast%3DSong%26aufirst%3DY.%26atitle%3DTEAD4%2520exerts%2520pro-metastatic%2520effects%2520and%2520is%2520negatively%2520regulated%2520by%2520miR6839-3p%2520in%2520lung%2520adenocarcinoma%2520progression%26jtitle%3DJ.%2520Cell.%2520Mol.%2520Med.%26date%3D2018%26volume%3D22%26spage%3D3560%26epage%3D3571%26doi%3D10.1111%2Fjcmm.13634" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mei, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, L.</span></span> <span> </span><span class="NLM_article-title">Increased TEAD4 expression and nuclear localization in colorectal cancer promote epithelial-mesenchymal transition and metastasis in a YAP-independent manner</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">2789</span>– <span class="NLM_lpage">2800</span>, <span class="refDoi"> DOI: 10.1038/onc.2015.342</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1038%2Fonc.2015.342" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=26387538" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFers7bJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2016&pages=2789-2800&author=Y.+Liuauthor=G.+Wangauthor=Y.+Yangauthor=Z.+Meiauthor=Z.+Liangauthor=A.+Cuiauthor=T.+Wuauthor=C.-Y.+Liuauthor=L.+Cui&title=Increased+TEAD4+expression+and+nuclear+localization+in+colorectal+cancer+promote+epithelial-mesenchymal+transition+and+metastasis+in+a+YAP-independent+manner&doi=10.1038%2Fonc.2015.342"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Increased TEAD4 expression and nuclear localization in colorectal cancer promote epithelial-mesenchymal transition and metastasis in a YAP-independent manner</span></div><div class="casAuthors">Liu, Y.; Wang, G.; Yang, Y.; Mei, Z.; Liang, Z.; Cui, A.; Wu, T.; Liu, C.-Y.; Cui, L.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">2789-2800</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Dysregulation of the Hippo pathway occurs in a variety of cancers and often correlates with a poor prognosis.  To further explore the potential role of Hippo pathway dysregulation in tumor development and progression, we investigated its downstream transcription factor TEAD4 in colorectal cancer (CRC).  Increased expression and nuclear localization of TEAD4 were found in a significant portion of CRC tissues, in assocn. with metastasis and a poor prognosis.  In CRC cells, TEAD4 knockdown induced the mesenchymal-epithelial transition and decreased cell mobility in vitro and metastasis in vivo.  Microarray anal. revealed that TEAD4 promoted cell adhesion and upregulated the epithelial-mesenchymal transition-related transcriptome in CRC cells.  Vimentin was identified as a new direct target gene mediating TEAD4 function in CRC cells, whereby forced vimentin expression markedly reversed TEAD4-knockdown-induced cell morphol. changes and decreased mobility.  Interestingly, rescued expression of both WT TEAD4 and a Y429H mutant can reverse the mesenchymal-epithelial transition and increase vimentin expression, cell mobility and metastatic potential in TEAD4-knockdown CRC cells.  The discrepant expression of YAP and TEAD4 in CRC tissues, the rescue ability of TEAD4 mutant defect in YAP binding and no effect on vimentin expression by YAP knockdown in CRC cells, all implicated a YAP-independent manner of TEAD4 function in CRC.  Furthermore, vimentin pos. correlated and CDH1 reversely correlated with the level of TEAD4 in CRC tissues and xenograft tumors.  Our results suggest that TEAD4 nuclear expression can serve as a biomarker for CRC progression and poor prognosis.  The transcription factor TEAD4 regulates a pro-metastasis transcription program in a YAP-independent manner in CRC, thus providing a novel mechanism of TEAD4 transcriptional regulation and its oncogenic role in CRC, independently of the Hippo pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfoIRDoJ1PIrVg90H21EOLACvtfcHk0li0kifNVgwwZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFers7bJ&md5=fbb07af174963a5f4ce3b7aeda162aa1</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fonc.2015.342&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2015.342%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DMei%26aufirst%3DZ.%26aulast%3DLiang%26aufirst%3DZ.%26aulast%3DCui%26aufirst%3DA.%26aulast%3DWu%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DC.-Y.%26aulast%3DCui%26aufirst%3DL.%26atitle%3DIncreased%2520TEAD4%2520expression%2520and%2520nuclear%2520localization%2520in%2520colorectal%2520cancer%2520promote%2520epithelial-mesenchymal%2520transition%2520and%2520metastasis%2520in%2520a%2520YAP-independent%2520manner%26jtitle%3DOncogene%26date%3D2016%26volume%3D35%26spage%3D2789%26epage%3D2800%26doi%3D10.1038%2Fonc.2015.342" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, J. H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, W. K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, A. S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">To, K. F.</span></span> <span> </span><span class="NLM_article-title">TEAD1/4 exerts oncogenic role and is negatively regulated by miR-4269 in gastric tumorigenesis</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">6518</span>– <span class="NLM_lpage">6530</span>, <span class="refDoi"> DOI: 10.1038/onc.2017.257</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1038%2Fonc.2017.257" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=28759040" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1CmtbvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2017&pages=6518-6530&author=Y.+Zhouauthor=T.+Huangauthor=J.+Zhangauthor=C.+C.+Wongauthor=B.+Zhangauthor=Y.+Dongauthor=F.+Wuauthor=J.+H.+M.+Tongauthor=W.+K.+K.+Wuauthor=A.+S.+L.+Chengauthor=J.+Yuauthor=W.+Kangauthor=K.+F.+To&title=TEAD1%2F4+exerts+oncogenic+role+and+is+negatively+regulated+by+miR-4269+in+gastric+tumorigenesis&doi=10.1038%2Fonc.2017.257"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">TEAD1/4 exerts oncogenic role and is negatively regulated by miR-4269 in gastric tumorigenesis</span></div><div class="casAuthors">Zhou, Y.; Huang, T.; Zhang, J.; Wong, C. C.; Zhang, B.; Dong, Y.; Wu, F.; Tong, J. H. M.; Wu, W. K. K.; Cheng, A. S. L.; Yu, J.; Kang, W.; To, K. F.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">6518-6530</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">TEA domain (TEAD) transcription factors are key components of the Hippo-YAP1 signaling pathway, but their functional role and regulatory mechanisms remain unclear.  This study aims to comprehensively explore the expression pattern and functional role of TEAD family in gastric carcinogenesis and investigate its regulation by microRNAs (miRNAs).  The mRNA and protein expression of TEAD family were examd. by quant. reverse transcription-PCR (qRT-PCR) and western blot.  Their functional roles were detd. by in vitro and in vivo studies.  The clinicopathol. assocn. of TEAD4 in gastric cancer (GC) was studied using immunohistochem. on tissue microarray.  The prediction of miRNAs, which potentially target TEAD1/4, was performed by TargetScan and miRDB.  The regulation of TEAD1/4 by miRNAs was confirmed by qRT-PCR, western blot and luciferase assays.  TEAD1/4 were overexpressed in GC cell lines and primary GC tissues.  Knockdown of TEAD1/4 induced a significant anticancer effect in vitro and in vivo.  TEAD1 was confirmed to be a direct target of miR-377-3p and miR-4269, while TEAD4 was neg. regulated by miR-1343-3p and miR-4269.  Among them, miR-4269 was the most effective inhibitor of TEAD1/4.  Ectopic expression of these miRNAs substantiated their tumor-suppressive effects.  In primary GC tumors, downregulation of miR-4269 was assocd. with poor disease-specific survival and showed a neg. correlation with TEAD4.  TEAD1 and TEAD4 are oncogenic factors, whose aberrant activation are, in part, mediated by the silence of miR-377-3p, miR-1343-3p and miR-4269.  For the first time, the nuclear accumulated TEAD4 and downregulated miR-4269 are proposed to serve as novel prognostic biomarkers in GC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTACkd06skNrVg90H21EOLACvtfcHk0lhX3OuW34u2MQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1CmtbvN&md5=3a0305563b06178d37487681a23c2bd7</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fonc.2017.257&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2017.257%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DWong%26aufirst%3DC.%2BC.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DDong%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DF.%26aulast%3DTong%26aufirst%3DJ.%2BH.%2BM.%26aulast%3DWu%26aufirst%3DW.%2BK.%2BK.%26aulast%3DCheng%26aufirst%3DA.%2BS.%2BL.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DKang%26aufirst%3DW.%26aulast%3DTo%26aufirst%3DK.%2BF.%26atitle%3DTEAD1%252F4%2520exerts%2520oncogenic%2520role%2520and%2520is%2520negatively%2520regulated%2520by%2520miR-4269%2520in%2520gastric%2520tumorigenesis%26jtitle%3DOncogene%26date%3D2017%26volume%3D36%26spage%3D6518%26epage%3D6530%26doi%3D10.1038%2Fonc.2017.257" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nie, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span> <span> </span><span class="NLM_article-title">The interplay between TEAD4 and KLF5 promotes breast cancer partially through inhibiting the transcription of p27Kip1</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">17685</span>– <span class="NLM_lpage">17697</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.3779</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.18632%2Foncotarget.3779" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=25970772" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A280%3ADC%252BC2MfjsFyqsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=17685-17697&author=C.+Wangauthor=Z.+Nieauthor=Z.+Zhouauthor=H.+Zhangauthor=R.+Liuauthor=J.+Wuauthor=J.+Qinauthor=Y.+Maauthor=L.+Chenauthor=S.+Liauthor=W.+Chenauthor=F.+Liauthor=P.+Shiauthor=Y.+Wuauthor=J.+Shenauthor=C.+Chen&title=The+interplay+between+TEAD4+and+KLF5+promotes+breast+cancer+partially+through+inhibiting+the+transcription+of+p27Kip1&doi=10.18632%2Foncotarget.3779"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The interplay between TEAD4 and KLF5 promotes breast cancer partially through inhibiting the transcription of p27Kip1</span></div><div class="casAuthors">Wang Chunyan; Zhou Zhongmei; Zhang Hailin; Liu Rong; Wu Jing; Qin Junying; Li Fubing; Shi Peiguo; Chen Ceshi; Wang Chunyan; Wu Jing; Qin Junying; Li Fubing; Shi Peiguo; Wang Chunyan; Nie Zhi; Ma Yun; Chen Liang; Li Shumo; Wu Yingying; Shen Jian; Wu Jing; Chen Wenlin</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">17685-97</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Growing evidence suggests that YAP/TAZ are mediators of the Hippo pathway and promote breast cancer.  However, the roles of YAP/TAZ transcription factor partners TEADs in breast cancer remain unclear.  Here we found that TEAD4 was expressed in breast cancer cell lines, especially in triple negative breast cancers (TNBC) cell lines.  TEAD4 binds to KLF5.  Knockdown of either TEAD4 or KLF5 in HCC1937 and HCC1806 cells induced the expression of CDK inhibitor p27.  Depletion of either TEAD4 or KLF5 activated the p27 gene promoter and increased the p27 mRNA levels.  Depletion of p27 partially prevents growth inhibition caused by TEAD4 and KLF5 knockdown.  TEAD4 overexpression stimulated proliferation in vitro and tumor growth in mice, while stable knockdown of TEAD4 inhibited proliferation in vitro and tumor growth in mice.  Thus TEAD4 and KLF5, in collaboration, promoted TNBC cell proliferation and tumor growth in part by inhibiting p27 gene transcription.  TEAD4 is a potential target and biomarker for the development of novel therapeutics for breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRn1i1qso7YZ8yail4HZd-ifW6udTcc2eZ1plVG3Mv6jbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MfjsFyqsw%253D%253D&md5=51a793fa622adc55368eab334a233791</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.3779&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.3779%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DC.%26aulast%3DNie%26aufirst%3DZ.%26aulast%3DZhou%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DR.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DQin%26aufirst%3DJ.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DShi%26aufirst%3DP.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DC.%26atitle%3DThe%2520interplay%2520between%2520TEAD4%2520and%2520KLF5%2520promotes%2520breast%2520cancer%2520partially%2520through%2520inhibiting%2520the%2520transcription%2520of%2520p27Kip1%26jtitle%3DOncotarget%26date%3D2015%26volume%3D6%26spage%3D17685%26epage%3D17697%26doi%3D10.18632%2Foncotarget.3779" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, M. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poon, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zender, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luk, J. M.</span></span> <span> </span><span class="NLM_article-title">AXL receptor kinase is a mediator of YAP-dependent oncogenic functions in hepatocellular carcinoma</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">1229</span>– <span class="NLM_lpage">1240</span>, <span class="refDoi"> DOI: 10.1038/onc.2010.504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1038%2Fonc.2010.504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=21076472" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVWmurjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2011&pages=1229-1240&author=M.+Z.+Xuauthor=S.+W.+Chanauthor=A.+M.+Liuauthor=K.+F.+Wongauthor=S.+T.+Fanauthor=J.+Chenauthor=R.+T.+Poonauthor=L.+Zenderauthor=S.+W.+Loweauthor=W.+Hongauthor=J.+M.+Luk&title=AXL+receptor+kinase+is+a+mediator+of+YAP-dependent+oncogenic+functions+in+hepatocellular+carcinoma&doi=10.1038%2Fonc.2010.504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">AXL receptor kinase is a mediator of YAP-dependent oncogenic functions in hepatocellular carcinoma</span></div><div class="casAuthors">Xu, M. Z.; Chan, S. W.; Liu, A. M.; Wong, K. F.; Fan, S. T.; Chen, J.; Poon, R. T.; Zender, L.; Lowe, S. W.; Hong, W.; Luk, J. M.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1229-1240</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Yes-assocd. protein (YAP) is a downstream effector of the Hippo signaling pathway, which controls organ expansion and tissue development.  We have recently defined the tumorigenic potential and clin. significance of the YAP1 oncogene in human hepatocellular carcinoma (HCC).  The present study aims to define the tumorigenic properties of YAP in HCC and elucidate the related downstream signaling mechanism.  In a gain-of-function study, we demonstrated that ectopic increased expression of YAP in the immortalized non-tumorigenic hepatocyte cell line MIHA confers tumorigenic and metastatic potentials, as evidenced by (1) enhanced aptitudes in cell viability, anchorage-independent growth, migration and invasion; (2) tumor formation in a xenograft mouse model; and (3) induction of HCC biomarker α-fetoprotein and activation of mitogen-activated protein kinase.  Furthermore, we have identified AXL, a receptor tyrosine kinase, as a key downstream target that drives YAP-dependent oncogenic functions.  RNAi-mediated knockdown of AXL expression decreased the ability of YAP-expressing MIHA cells and of the primary HCC cell line to proliferate and invade.  These results indicate that AXL is a mediator of YAP-dependent oncogenic activities and implicates it as a potential therapeutic target for HCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhMeq35gO5yrVg90H21EOLACvtfcHk0liiaKwPDzDvZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVWmurjJ&md5=7d49124ddec8fafc084a336e55ecb80b</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fonc.2010.504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2010.504%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DM.%2BZ.%26aulast%3DChan%26aufirst%3DS.%2BW.%26aulast%3DLiu%26aufirst%3DA.%2BM.%26aulast%3DWong%26aufirst%3DK.%2BF.%26aulast%3DFan%26aufirst%3DS.%2BT.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DPoon%26aufirst%3DR.%2BT.%26aulast%3DZender%26aufirst%3DL.%26aulast%3DLowe%26aufirst%3DS.%2BW.%26aulast%3DHong%26aufirst%3DW.%26aulast%3DLuk%26aufirst%3DJ.%2BM.%26atitle%3DAXL%2520receptor%2520kinase%2520is%2520a%2520mediator%2520of%2520YAP-dependent%2520oncogenic%2520functions%2520in%2520hepatocellular%2520carcinoma%26jtitle%3DOncogene%26date%3D2011%26volume%3D30%26spage%3D1229%26epage%3D1240%26doi%3D10.1038%2Fonc.2010.504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barreto, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ray, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ag Edgar, P.</span></span> <span> </span><span class="NLM_article-title">Biological characteristics of CCN proteins in tumor development</span>. <i>J. BUON</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1359</span>– <span class="NLM_lpage">1367</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=28039692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A280%3ADC%252BC1c7gvFOgtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=1359-1367&author=S.+C.+Barretoauthor=A.+Rayauthor=P.+Ag+Edgar&title=Biological+characteristics+of+CCN+proteins+in+tumor+development"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Biological characteristics of CCN proteins in tumor development</span></div><div class="casAuthors">Barreto Stephany C; Ray Amitabha; Ag Edgar Page</div><div class="citationInfo"><span class="NLM_cas:title">Journal of B.U.ON. : official journal of the Balkan Union of Oncology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1359-1367</span>
        ISSN:<span class="NLM_cas:issn">1107-0625</span>.
    </div><div class="casAbstract">The extracellular membrane (ECM) is no longer regarded as inert, rather it has multiple versatile physiologic functions.  Its diverse composition is implicated in each step of cancer progression including inflammation, angiogenesis, tumor invasion and metastasis.  In addition to structural proteins, the ECM also contains a family of non-structural proteins called matricellular proteins.  The six secreted CCN proteins, which belong to the matricellular protein family, include the following: Cyr61 (CCN1), CTGF (CCN2), Nov (CCN3), WISP- 1 (CCN4), WISP-2 (CCN5) and WISP-3(CCN6).  These proteins are capable of modulating a variety of biological processes in health as well as in disease conditions.  In tumor development and in tumor microenvironment, CCN proteins can influence multiple facets of pathophysiological processes including cellular proliferation, invasion and metastasis.  This review has attempted a cohesive look at the CCN family protein functions in a tumor-specific manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSroeed7ucn-DdblLE_E9PWfW6udTcc2eah6-aiHlxXILntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1c7gvFOgtQ%253D%253D&md5=cc10333a7fee86a0a0b0f8d935c32b0c</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBarreto%26aufirst%3DS.%2BC.%26aulast%3DRay%26aufirst%3DA.%26aulast%3DAg%2BEdgar%26aufirst%3DP.%26atitle%3DBiological%2520characteristics%2520of%2520CCN%2520proteins%2520in%2520tumor%2520development%26jtitle%3DJ.%2520BUON%26date%3D2016%26volume%3D21%26spage%3D1359%26epage%3D1367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaiyan, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, Z.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, K.-L.</span></span> <span> </span><span class="NLM_article-title">TEAD mediates YAP-dependent gene induction and growth control</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">1962</span>– <span class="NLM_lpage">1971</span>, <span class="refDoi"> DOI: 10.1101/gad.1664408</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1101%2Fgad.1664408" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=18579750" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BD1cXovFWlsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2008&pages=1962-1971&author=B.+Zhaoauthor=X.+Yeauthor=J.+Yuauthor=L.+Liauthor=W.+Liauthor=S.+Liauthor=J.+Yuauthor=J.+D.+Linauthor=C.-Y.+Wangauthor=A.+M.+Chinnaiyanauthor=Z.-C.+Laiauthor=K.-L.+Guan&title=TEAD+mediates+YAP-dependent+gene+induction+and+growth+control&doi=10.1101%2Fgad.1664408"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">TEAD mediates YAP-dependent gene induction and growth control</span></div><div class="casAuthors">Zhao, Bin; Ye, Xin; Yu, Jindan; Li, Li; Li, Weiquan; Li, Siming; Yu, Jianjun; Lin, Jiandie D.; Wang, Cun-Yu; Chinnaiyan, Arul M.; Lai, Zhi-Chun; Guan, Kun-Liang</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1962-1971</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">The YAP transcription coactivator has been implicated as an oncogene and is amplified in human cancers.  Recent studies have established that YAP is phosphorylated and inhibited by the Hippo tumor suppressor pathway.  Here we demonstrate that the TEAD family transcription factors are essential in mediating YAP-dependent gene expression.  TEAD is also required for YAP-induced cell growth, oncogenic transformation, and epithelial-mesenchymal transition.  CTGF is identified as a direct YAP target gene important for cell growth.  Moreover, the functional relationship between YAP and TEAD is conserved in Drosophila Yki (the YAP homolog) and Scalloped (the TEAD homolog).  Our study reveals TEAD as a new component in the Hippo pathway playing essential roles in mediating biol. functions of YAP.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVbRgjZPRWbbVg90H21EOLACvtfcHk0liiaKwPDzDvZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXovFWlsb8%253D&md5=53bf27e76bdb025ba0077004fd8ec968</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1101%2Fgad.1664408&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1664408%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DYe%26aufirst%3DX.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DLin%26aufirst%3DJ.%2BD.%26aulast%3DWang%26aufirst%3DC.-Y.%26aulast%3DChinnaiyan%26aufirst%3DA.%2BM.%26aulast%3DLai%26aufirst%3DZ.-C.%26aulast%3DGuan%26aufirst%3DK.-L.%26atitle%3DTEAD%2520mediates%2520YAP-dependent%2520gene%2520induction%2520and%2520growth%2520control%26jtitle%3DGenes%2520Dev.%26date%3D2008%26volume%3D22%26spage%3D1962%26epage%3D1971%26doi%3D10.1101%2Fgad.1664408" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feldmann, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Comerford, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gayyed, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anders, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maitra, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, D.</span></span> <span> </span><span class="NLM_article-title">Elucidation of a universal size-control mechanism in drosophila and mammals</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>130</i></span>,  <span class="NLM_fpage">1120</span>– <span class="NLM_lpage">1133</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2007.07.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1016%2Fj.cell.2007.07.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=17889654" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFKnurjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2007&pages=1120-1133&author=J.+Dongauthor=G.+Feldmannauthor=J.+Huangauthor=S.+Wuauthor=N.+Zhangauthor=S.+A.+Comerfordauthor=M.+F.+Gayyedauthor=R.+A.+Andersauthor=A.+Maitraauthor=D.+Pan&title=Elucidation+of+a+universal+size-control+mechanism+in+drosophila+and+mammals&doi=10.1016%2Fj.cell.2007.07.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Elucidation of a universal size-control mechanism in Drosophila and mammals</span></div><div class="casAuthors">Dong, Jixin; Feldmann, Georg; Huang, Jianbin; Wu, Shian; Zhang, Nailing; Comerford, Sarah A.; Gayyed, Mariana F.; Anders, Robert A.; Maitra, Anirban; Pan, Duojia</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1120-1133</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Coordination of cell proliferation and cell death is essential to attain proper organ size during development and for maintaining tissue homeostasis throughout postnatal life.  In Drosophila, these two processes are orchestrated by the Hippo kinase cascade, a growth-suppressive pathway that ultimately antagonizes the transcriptional coactivator Yorkie (Yki).  Here we demonstrate that a single phosphorylation site in Yki mediates the growth-suppressive output of the Hippo pathway.  Hippo-mediated phosphorylation inactivates Yki by excluding it from the nucleus, whereas loss of Hippo signaling leads to nuclear accumulation and therefore increased Yki activity.  We further delineate a mammalian Hippo signaling pathway that culminates in the phosphorylation of YAP, the mammalian homolog of Yki.  Using a conditional YAP transgenic mouse model, we demonstrate that the mammalian Hippo pathway is a potent regulator of organ size, and that its dysregulation leads to tumorigenesis.  These results uncover a universal size-control mechanism in metazoan.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFVuE0XSnE87Vg90H21EOLACvtfcHk0liiaKwPDzDvZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFKnurjO&md5=c8b5bfffd642be6b1fe2af7d012a1c3e</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.07.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.07.019%26sid%3Dliteratum%253Aachs%26aulast%3DDong%26aufirst%3DJ.%26aulast%3DFeldmann%26aufirst%3DG.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DComerford%26aufirst%3DS.%2BA.%26aulast%3DGayyed%26aufirst%3DM.%2BF.%26aulast%3DAnders%26aufirst%3DR.%2BA.%26aulast%3DMaitra%26aufirst%3DA.%26aulast%3DPan%26aufirst%3DD.%26atitle%3DElucidation%2520of%2520a%2520universal%2520size-control%2520mechanism%2520in%2520drosophila%2520and%2520mammals%26jtitle%3DCell%26date%3D2007%26volume%3D130%26spage%3D1120%26epage%3D1133%26doi%3D10.1016%2Fj.cell.2007.07.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Y. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaerkady, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kern, S. E.</span></span> <span> </span><span class="NLM_article-title">Unbiased discovery of interactions at a control locus driving expression of the cancer-specific therapeutic and diagnostic target, mesothelin</span>. <i>J. Proteome Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">5301</span>– <span class="NLM_lpage">5310</span>, <span class="refDoi"> DOI: 10.1021/pr300797v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/pr300797v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVejsrjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=5301-5310&author=Y.+R.+Renauthor=R.+Chaerkadyauthor=S.+Huauthor=J.+Wanauthor=J.+Qianauthor=H.+Zhuauthor=A.+Pandeyauthor=S.+E.+Kern&title=Unbiased+discovery+of+interactions+at+a+control+locus+driving+expression+of+the+cancer-specific+therapeutic+and+diagnostic+target%2C+mesothelin&doi=10.1021%2Fpr300797v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Unbiased discovery of interactions at a control locus driving expression of the cancer-specific therapeutic and diagnostic target, mesothelin</span></div><div class="casAuthors">Ren, Yunzhao R.; Chaerkady, Raghothama; Hu, Shaohui; Wan, Jun; Qian, Jiang; Zhu, Heng; Pandey, Akhilesh; Kern, Scott E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5301-5310</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Although significant effort is expended on identifying transcripts/proteins that are up-regulated in cancer, there are few reports on systematic elucidation of transcriptional mechanisms underlying such druggable cancer-specific targets.  The mesothelin (MSLN) gene offers a promising subject, being expressed in a restricted pattern normally, yet highly overexpressed in almost one-third of human malignancies and a target of cancer immunotherapeutic trials.  CanScript, a cis promoter element, appears to control MSLN cancer-specific expression; its related genomic sequences may up-regulate other cancer markers.  CanScript is a 20-nt bipartite element consisting of an SP1-like motif and a consensus MCAT sequence.  The latter recruits TEAD (TEA domain) family members, which are universally expressed.  Exploration of the active CanScript element, esp. the proteins binding to the SP1-like motif, thus could reveal cancer-specific features having diagnostic or therapeutic interest.  The efficient identification of sequence-specific DNA-binding proteins at a given locus, however, has lagged in biomarker explorations.  The authors used two orthogonal proteomics approaches-unbiased SILAC (stable isotope labeling by amino acids in cell culture)/DNA affinity-capture/mass spectrometry survey (SD-MS) and a large transcription factor protein microarray (TFM)-and functional validation to explore systematically the CanScript interactome.  SD-MS produced nine candidates, and TFM, 18.  The screens agreed in confirming binding by TEAD proteins and by newly identified NAB1 and NFATc.  Among other identified candidates, the authors found functional roles for ZNF24, NAB1 and RFX1 in MSLN expression by cancer cells.  Combined interactome screens yield an efficient, reproducible, sensitive, and unbiased approach to identify sequence-specific DNA-binding proteins and other participants in disease-specific DNA elements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmDKthGPBrALVg90H21EOLACvtfcHk0lhC1Hb8b8sFuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVejsrjM&md5=0453e4cf7e0b3559a445ae641813ea56</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fpr300797v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fpr300797v%26sid%3Dliteratum%253Aachs%26aulast%3DRen%26aufirst%3DY.%2BR.%26aulast%3DChaerkady%26aufirst%3DR.%26aulast%3DHu%26aufirst%3DS.%26aulast%3DWan%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DPandey%26aufirst%3DA.%26aulast%3DKern%26aufirst%3DS.%2BE.%26atitle%3DUnbiased%2520discovery%2520of%2520interactions%2520at%2520a%2520control%2520locus%2520driving%2520expression%2520of%2520the%2520cancer-specific%2520therapeutic%2520and%2520diagnostic%2520target%252C%2520mesothelin%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2012%26volume%3D11%26spage%3D5301%26epage%3D5310%26doi%3D10.1021%2Fpr300797v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedrich, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soriano, P.</span></span> <span> </span><span class="NLM_article-title">Transcriptional enhancer factor 1 disruption by a retroviral gene trap leads to heart defects and embryonic lethality in mice</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">2293</span>– <span class="NLM_lpage">2301</span>, <span class="refDoi"> DOI: 10.1101/gad.8.19.2293</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1101%2Fgad.8.19.2293" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=7958896" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADyaK2MXhtF2gsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1994&pages=2293-2301&author=Z.+Chenauthor=G.+A.+Friedrichauthor=P.+Soriano&title=Transcriptional+enhancer+factor+1+disruption+by+a+retroviral+gene+trap+leads+to+heart+defects+and+embryonic+lethality+in+mice&doi=10.1101%2Fgad.8.19.2293"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional enhancer factor 1 disruption by a retroviral gene trap leads to heart defects and embryonic lethality in mice</span></div><div class="casAuthors">Chen, Zhi; Friedrich, Glenn A.; Soriano, Philippe</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">2293-301</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    </div><div class="casAbstract">A retroviral gene trap in embryonic stem (ES) cells was used to derive a recessive embryonic lethal mouse strain, ROSAβ-geo5.  Mutant embryos display an enlarged pericardial cavity, brachycardia, a dilated fourth ventricle in the brain, and die between embryonic days 11 and 12.  Whereas heart development in the mutant embryos is extensive, the ventricular wall is abnormally thin with a reduced no. of trabeculae.  Cloning of the trapped gene indicates that proviral insertion creates a null mutation in the transcriptional enhancer factor 1 (TEF-1) gene.  Although transcription of a no. of muscle-specific genes believed to be TEF-1 targets appears normal, the defect in cardiogenesis is likely attributable to diminished transcription of one or several cardiac-specific genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEdQzY-4NhA7Vg90H21EOLACvtfcHk0lhC1Hb8b8sFuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXhtF2gsbg%253D&md5=720e70b11d73ceae5560a526fc53ff0c</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1101%2Fgad.8.19.2293&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.8.19.2293%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DFriedrich%26aufirst%3DG.%2BA.%26aulast%3DSoriano%26aufirst%3DP.%26atitle%3DTranscriptional%2520enhancer%2520factor%25201%2520disruption%2520by%2520a%2520retroviral%2520gene%2520trap%2520leads%2520to%2520heart%2520defects%2520and%2520embryonic%2520lethality%2520in%2520mice%26jtitle%3DGenes%2520Dev.%26date%3D1994%26volume%3D8%26spage%3D2293%26epage%3D2301%26doi%3D10.1101%2Fgad.8.19.2293" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaneko, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohn, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DePamphilis, M. L.</span></span> <span> </span><span class="NLM_article-title">Transcription factor TEAD2 is involved in neural tube closure</span>. <i>Genesis</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">577</span>– <span class="NLM_lpage">587</span>, <span class="refDoi"> DOI: 10.1002/dvg.20330</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1002%2Fdvg.20330" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=17868131" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlWjs7nI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2007&pages=577-587&author=K.+J.+Kanekoauthor=M.+J.+Kohnauthor=C.+Liuauthor=M.+L.+DePamphilis&title=Transcription+factor+TEAD2+is+involved+in+neural+tube+closure&doi=10.1002%2Fdvg.20330"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Transcription factor TEAD2 is involved in neural tube closure</span></div><div class="casAuthors">Kaneko, Kotaro J.; Kohn, Matthew J.; Liu, Chengyu; DePamphilis, Melvin L.</div><div class="citationInfo"><span class="NLM_cas:title">Genesis (Hoboken, NJ, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">577-587</span>CODEN:
                <span class="NLM_cas:coden">GNESFY</span>;
        ISSN:<span class="NLM_cas:issn">1526-954X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">TEAD2, one of the first transcription factors expressed at the beginning of mammalian development, appears to be required during neural development.  For example, Tead2 expression is greatest in the dorsal neural crest where it appears to regulate expression of Pax3, a gene essential for brain development.  Consistent with this hypothesis, we found that inactivation of the Tead2 gene in mice significantly increased the risk of exencephaly (a defect in neural tube closure).  However, none of the embryos exhibited spina bifida, the major phenotype of Pax3 nullizygous embryos, and expression of Pax3 in E11.5 Tead2 nullizygous embryos was normal.  Thus, Tead2 plays a role in neural tube closure that is independent of its putative role in Pax3 regulation.  In addn., the risk of exencephaly was greatest with Tead2 nullizygous females, and could be suppressed either by folic acid or pifithrin-α.  These results reveal a maternal genetic contribution to neural tube closure, and suggest that Tead2-deficient mice provide a model for anencephaly, a common human birth defect that can be prevented by folic acid.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDkxCNUlVolrVg90H21EOLACvtfcHk0lhC1Hb8b8sFuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlWjs7nI&md5=fe699b35849882fc648ccc8c089893c8</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1002%2Fdvg.20330&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fdvg.20330%26sid%3Dliteratum%253Aachs%26aulast%3DKaneko%26aufirst%3DK.%2BJ.%26aulast%3DKohn%26aufirst%3DM.%2BJ.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DDePamphilis%26aufirst%3DM.%2BL.%26atitle%3DTranscription%2520factor%2520TEAD2%2520is%2520involved%2520in%2520neural%2520tube%2520closure%26jtitle%3DGenesis%26date%3D2007%26volume%3D45%26spage%3D577%26epage%3D587%26doi%3D10.1002%2Fdvg.20330" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nishioka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiyonari, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawada, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ota, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, H.</span></span> <span> </span><span class="NLM_article-title">TEAD4 is required for specification of trophectoderm in pre-implantation mouse embryos</span>. <i>Mech. Dev.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">270</span>– <span class="NLM_lpage">283</span>, <span class="refDoi"> DOI: 10.1016/j.mod.2007.11.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1016%2Fj.mod.2007.11.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=18083014" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhslGjt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2008&pages=270-283&author=N.+Nishiokaauthor=S.+Yamamotoauthor=H.+Kiyonariauthor=H.+Satoauthor=A.+Sawadaauthor=M.+Otaauthor=K.+Nakaoauthor=H.+Sasaki&title=TEAD4+is+required+for+specification+of+trophectoderm+in+pre-implantation+mouse+embryos&doi=10.1016%2Fj.mod.2007.11.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Tead4 is required for specification of trophectoderm in preimplantation mouse embryos</span></div><div class="casAuthors">Nishioka, Noriyuki; Yamamoto, Shinji; Kiyonari, Hiroshi; Sato, Hiroko; Sawada, Atsushi; Ota, Mitsunori; Nakao, Kazuki; Sasaki, Hiroshi</div><div class="citationInfo"><span class="NLM_cas:title">Mechanisms of Development</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">3-4</span>),
    <span class="NLM_cas:pages">270-283</span>CODEN:
                <span class="NLM_cas:coden">MEDVE6</span>;
        ISSN:<span class="NLM_cas:issn">0925-4773</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">During preimplantation mouse development, embryos form blastocysts with establishment of the first two cell lineages: the trophectoderm (TE) which gives rise to the placenta, and the inner cell mass (ICM) which will form the embryo proper.  Differentiation of TE is regulated by the transcription factor Caudal-related homeobox 2 (Cdx2), but the mechanisms which act upstream of Cdx2 expression remain unknown.  Here we show that the TEA domain family transcription factor, Tead4, is required for TE development.  Tead1, Tead2 and Tead4 were expressed in preimplantation embryos, and at least Tead1 and Tead4 were expressed widely in both TE and ICM lineages.  Tead4 -/- embryos died at preimplantation stages without forming the blastocoel.  The mutant embryos continued cell proliferation, and adherens junction and cell polarity were not significantly affected.  In Tead4 -/- embryos, Cdx2 was weakly expressed at the morula stage but was not expressed in later stages.  None of the TE specific genes, including Eomes and a Cdx2 independent gene, Fgfr2, was detected in Tead4 -/- embryos.  Instead, the ICM specific transcription factors, Oct3/4 and Nanog, were expressed in all the blastomeres.  Tead4 -/- embryos also failed to differentiate trophoblast giant cells when they were cultured in vitro.  ES cells with normal in vitro differentiation abilities were established from Tead4 -/- embryos.  These results suggest that Tead4 has a distinct role from Tead1 and Tead2 in trophectoderm specification of preimplantation embryos, and that Tead4 is an early transcription factor required for specification and development of the trophectoderm lineage, which includes expression of Cdx2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEAvfctEY52LVg90H21EOLACvtfcHk0lhlUJaf2wx1AA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhslGjt78%253D&md5=68eba1036c39922c1e6776b49217bf99</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.mod.2007.11.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mod.2007.11.002%26sid%3Dliteratum%253Aachs%26aulast%3DNishioka%26aufirst%3DN.%26aulast%3DYamamoto%26aufirst%3DS.%26aulast%3DKiyonari%26aufirst%3DH.%26aulast%3DSato%26aufirst%3DH.%26aulast%3DSawada%26aufirst%3DA.%26aulast%3DOta%26aufirst%3DM.%26aulast%3DNakao%26aufirst%3DK.%26aulast%3DSasaki%26aufirst%3DH.%26atitle%3DTEAD4%2520is%2520required%2520for%2520specification%2520of%2520trophectoderm%2520in%2520pre-implantation%2520mouse%2520embryos%26jtitle%3DMech.%2520Dev.%26date%3D2008%26volume%3D125%26spage%3D270%26epage%3D283%26doi%3D10.1016%2Fj.mod.2007.11.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yagi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohn, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karavanova, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaneko, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vullhorst, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DePamphilis, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buonanno, A.</span></span> <span> </span><span class="NLM_article-title">Transcription factor TEAD4 specifies the trophectoderm lineage at the beginning of mammalian development</span>. <i>Development</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">3827</span>– <span class="NLM_lpage">3836</span>, <span class="refDoi"> DOI: 10.1242/dev.010223</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1242%2Fdev.010223" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=17913785" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVeku7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2007&pages=3827-3836&author=R.+Yagiauthor=M.+J.+Kohnauthor=I.+Karavanovaauthor=K.+J.+Kanekoauthor=D.+Vullhorstauthor=M.+L.+DePamphilisauthor=A.+Buonanno&title=Transcription+factor+TEAD4+specifies+the+trophectoderm+lineage+at+the+beginning+of+mammalian+development&doi=10.1242%2Fdev.010223"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Transcription factor TEAD4 specifies the trophectoderm lineage at the beginning of mammalian development</span></div><div class="casAuthors">Yagi, Rieko; Kohn, Matthew J.; Karavanova, Irina; Kaneko, Kotaro J.; Vullhorst, Detlef; DePamphilis, Melvin L.; Buonanno, Andres</div><div class="citationInfo"><span class="NLM_cas:title">Development (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">3827-3836</span>CODEN:
                <span class="NLM_cas:coden">DEVPED</span>;
        ISSN:<span class="NLM_cas:issn">0950-1991</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">Specification of cell lineages in mammals begins shortly after fertilization with formation of a blastocyst consisting of trophectoderm, which contributes exclusively to the placenta, and inner cell mass (ICM), from which the embryo develops.  Here we report that ablation of the mouse Tead4 gene results in a preimplantation lethal phenotype, and TEAD4 is one of two highly homologous TEAD transcription factors that are expressed during zygotic gene activation in mouse 2-cell embryos.  Tead4-/- embryos do not express trophectoderm-specific genes, such as Cdx2, but do express ICM-specific genes, such as Oct4 (also known as Pou5f1).  Consequently, Tead4-/- morulae do not produce trophoblast stem cells, trophectoderm or blastocoel cavities, and therefore do not implant into the uterine endometrium.  However, Tead4-/- embryos can produce embryonic stem cells, a deriv. of ICM, and if the Tead4 allele is not disrupted until after implantation, then Tead4-/- embryos complete development.  Thus, Tead4 is the earliest gene shown to be uniquely required for specification of the trophectoderm lineage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5_mWKmWY8KbVg90H21EOLACvtfcHk0lhlUJaf2wx1AA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVeku7rL&md5=e1fe8a57011737f1c83931c6878c3a59</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1242%2Fdev.010223&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fdev.010223%26sid%3Dliteratum%253Aachs%26aulast%3DYagi%26aufirst%3DR.%26aulast%3DKohn%26aufirst%3DM.%2BJ.%26aulast%3DKaravanova%26aufirst%3DI.%26aulast%3DKaneko%26aufirst%3DK.%2BJ.%26aulast%3DVullhorst%26aufirst%3DD.%26aulast%3DDePamphilis%26aufirst%3DM.%2BL.%26aulast%3DBuonanno%26aufirst%3DA.%26atitle%3DTranscription%2520factor%2520TEAD4%2520specifies%2520the%2520trophectoderm%2520lineage%2520at%2520the%2520beginning%2520of%2520mammalian%2520development%26jtitle%3DDevelopment%26date%3D2007%26volume%3D134%26spage%3D3827%26epage%3D3836%26doi%3D10.1242%2Fdev.010223" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Juan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, W.</span></span> <span> </span><span class="NLM_article-title">Targeting the hippo signaling pathway for tissue regeneration and cancer therapy</span>. <i>Genes</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">55</span>, <span class="refDoi"> DOI: 10.3390/genes7090055</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.3390%2Fgenes7090055" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVWju7nL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=55&author=W.+Juanauthor=W.+Hong&title=Targeting+the+hippo+signaling+pathway+for+tissue+regeneration+and+cancer+therapy&doi=10.3390%2Fgenes7090055"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the Hippo signaling pathway for tissue regeneration and cancer therapy</span></div><div class="casAuthors">Juan, Wen Chun; Hong, Wanjin</div><div class="citationInfo"><span class="NLM_cas:title">Genes</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">55/1-55/25</span>CODEN:
                <span class="NLM_cas:coden">GENEG9</span>;
        ISSN:<span class="NLM_cas:issn">2073-4425</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">The Hippo signaling pathway is a highly-conserved developmental pathway that plays an essential role in organ size control, tumor suppression, tissue regeneration and stem cell self-renewal.  The YES-assocd. protein (YAP) and the transcriptional co-activator with PDZ-binding motif (TAZ) are two important transcriptional co-activators that are neg. regulated by the Hippo signaling pathway.  By binding to transcription factors, esp. the TEA domain transcription factors (TEADs), YAP and TAZ induce the expression of growth-promoting genes, which can promote organ regeneration after injury.  Therefore, controlled activation of YAP and TAZ can be useful for regenerative medicine.  However, aberrant activation of YAP and TAZ due to deregulation of the Hippo pathway or overexpression of YAP/TAZ and TEADs can promote cancer development.  Hence, pharmacol. inhibition of YAP and TAZ may be a useful approach to treat tumors with high YAP and/or TAZ activity.  In this review, we present the mechanisms regulating the Hippo pathway, the role of the Hippo pathway in tissue repair and cancer, as well as a detailed anal. of the different strategies to target the Hippo signaling pathway and the genes regulated by YAP and TAZ for regenerative medicine and cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwVFa4phVsj7Vg90H21EOLACvtfcHk0liw1TiL4JIaUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVWju7nL&md5=19c4e707f22ac7a2c6130e0f4d07eb9e</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.3390%2Fgenes7090055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fgenes7090055%26sid%3Dliteratum%253Aachs%26aulast%3DJuan%26aufirst%3DW.%26aulast%3DHong%26aufirst%3DW.%26atitle%3DTargeting%2520the%2520hippo%2520signaling%2520pathway%2520for%2520tissue%2520regeneration%2520and%2520cancer%2520therapy%26jtitle%3DGenes%26date%3D2016%26volume%3D7%26spage%3D55%26doi%3D10.3390%2Fgenes7090055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Santucci, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vignudelli, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrari, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scalvini, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolognesi, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uliassi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costi, M. P.</span></span> <span> </span><span class="NLM_article-title">The hippo pathway and YAP/TAZ-TEAD protein-protein interaction as targets for regenerative medicine and cancer treatment</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">4857</span>– <span class="NLM_lpage">4873</span>, <span class="refDoi"> DOI: 10.1021/jm501615v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501615v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjsVWktrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=4857-4873&author=M.+Santucciauthor=T.+Vignudelliauthor=S.+Ferrariauthor=M.+Morauthor=L.+Scalviniauthor=M.+L.+Bolognesiauthor=E.+Uliassiauthor=M.+P.+Costi&title=The+hippo+pathway+and+YAP%2FTAZ-TEAD+protein-protein+interaction+as+targets+for+regenerative+medicine+and+cancer+treatment&doi=10.1021%2Fjm501615v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The Hippo Pathway and YAP/TAZ-TEAD Protein-Protein Interaction as Targets for Regenerative Medicine and Cancer Treatment</span></div><div class="casAuthors">Santucci, Matteo; Vignudelli, Tatiana; Ferrari, Stefania; Mor, Marco; Scalvini, Laura; Bolognesi, Maria Laura; Uliassi, Elisa; Costi, Maria Paola</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4857-4873</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The Hippo pathway is an important organ size control signaling network and the major regulatory mechanism of cell-contact inhibition.  Yes assocd. protein (YAP) and transcriptional co-activator with PDZ-binding motif (TAZ) are its targets and terminal effectors: inhibition of the pathway promotes YAP/TAZ translocation to the nucleus, where they interact with transcriptional enhancer assoc. domain (TEAD) transcription factors and coactivate the expression of target genes, promoting cell proliferation.  Defects in the pathway can result in overgrowth phenotypes due to deregulation of stem-cell proliferation and apoptosis; members of the pathway are directly involved in cancer development.  The pharmacol. regulation of the pathway might be useful in cancer prevention, treatment, and regenerative medicine applications; currently, a few compds. can selectively modulate the pathway.  In this review, we present an overview of the Hippo pathway, the sequence and structural anal. of YAP/TAZ, the known pharmacol. modulators of the pathway, esp. those targeting YAP/TAZ-TEAD interaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQkLuQrlwpNrVg90H21EOLACvtfcHk0liw1TiL4JIaUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjsVWktrs%253D&md5=22e4a9cda126edd3739884e8c11c0a24</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm501615v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501615v%26sid%3Dliteratum%253Aachs%26aulast%3DSantucci%26aufirst%3DM.%26aulast%3DVignudelli%26aufirst%3DT.%26aulast%3DFerrari%26aufirst%3DS.%26aulast%3DMor%26aufirst%3DM.%26aulast%3DScalvini%26aufirst%3DL.%26aulast%3DBolognesi%26aufirst%3DM.%2BL.%26aulast%3DUliassi%26aufirst%3DE.%26aulast%3DCosti%26aufirst%3DM.%2BP.%26atitle%3DThe%2520hippo%2520pathway%2520and%2520YAP%252FTAZ-TEAD%2520protein-protein%2520interaction%2520as%2520targets%2520for%2520regenerative%2520medicine%2520and%2520cancer%2520treatment%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D4857%26epage%3D4873%26doi%3D10.1021%2Fjm501615v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gibault, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coevoet, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sturbaut, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farce, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renault, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allemand, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guichou, J.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drucbert, A.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foulon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thuru, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corvaisier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huet, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chavatte, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnyk, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailly, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cotelle, P.</span></span> <span> </span><span class="NLM_article-title">Toward the discovery of a novel class of YAP(-)TEAD interaction inhibitors by virtual screening approach targeting YAP(-)TEAD protein(-)protein interface</span>. <i>Cancers</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">140</span>, <span class="refDoi"> DOI: 10.3390/cancers10050140</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.3390%2Fcancers10050140" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVemsbzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=140&author=F.+Gibaultauthor=M.+Coevoetauthor=M.+Sturbautauthor=A.+Farceauthor=N.+Renaultauthor=F.+Allemandauthor=J.-F.+Guichouauthor=A.-S.+Drucbertauthor=C.+Foulonauthor=R.+Magnezauthor=X.+Thuruauthor=M.+Corvaisierauthor=G.+Huetauthor=P.+Chavatteauthor=P.+Melnykauthor=F.+Baillyauthor=P.+Cotelle&title=Toward+the+discovery+of+a+novel+class+of+YAP%28-%29TEAD+interaction+inhibitors+by+virtual+screening+approach+targeting+YAP%28-%29TEAD+protein%28-%29protein+interface&doi=10.3390%2Fcancers10050140"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Toward the discovery of a novel class of YAP-TEAD interaction inhibitors by virtual screening approach targeting YAP-TEAD protein-protein interface</span></div><div class="casAuthors">Gibault, Floriane; Coevoet, Mathilde; Sturbaut, Manon; Farce, Amaury; Renault, Nicolas; Allemand, Frederic; Guichou, Jean-Francois; Drucbert, Anne-Sophie; Foulon, Catherine; Magnez, Romain; Thuru, Xavier; Corvaisier, Matthieu; Huet, Guillemette; Chavatte, Philippe; Melnyk, Patricia; Bailly, Fabrice; Cotelle, Philippe</div><div class="citationInfo"><span class="NLM_cas:title">Cancers</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">140/1-140/14</span>CODEN:
                <span class="NLM_cas:coden">CANCCT</span>;
        ISSN:<span class="NLM_cas:issn">2072-6694</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Intrinsically disordered protein YAP (yes-assocd. protein) interacts with TEADs transcriptional factors family (transcriptional enhancer assocd. domain) creating three interfaces.  Interface 3, between the Ω-loop of YAP and a shallow pocket of TEAD was identified as the most important TEAD zone for YAP-TEAD interaction.  Using the first X-ray structure of the hYAP50-71-hTEAD1209-426 complex (PDB 3KYS) published in 2010, a protein-protein interaction inhibitors-enriched library (175,000 chem. compds.) was screened against this hydrophobic pocket of TEAD.  Four different chem. families have been identified and evaluated using biophys. techniques (thermal shift assay, microscale thermophoresis) and in cellulo assays (luciferase activity in transfected HEK293 cells, RTqPCR in MDA-MB231 cells).  A first promising hit with micromolar inhibition in the luciferase gene reporter assay was discovered.  This hit also decreased mRNA levels of TEAD target genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSVG22N3nS17Vg90H21EOLACvtfcHk0liw1TiL4JIaUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVemsbzF&md5=a463164da91cbc8db69ad8b41d09270a</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.3390%2Fcancers10050140&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcancers10050140%26sid%3Dliteratum%253Aachs%26aulast%3DGibault%26aufirst%3DF.%26aulast%3DCoevoet%26aufirst%3DM.%26aulast%3DSturbaut%26aufirst%3DM.%26aulast%3DFarce%26aufirst%3DA.%26aulast%3DRenault%26aufirst%3DN.%26aulast%3DAllemand%26aufirst%3DF.%26aulast%3DGuichou%26aufirst%3DJ.-F.%26aulast%3DDrucbert%26aufirst%3DA.-S.%26aulast%3DFoulon%26aufirst%3DC.%26aulast%3DMagnez%26aufirst%3DR.%26aulast%3DThuru%26aufirst%3DX.%26aulast%3DCorvaisier%26aufirst%3DM.%26aulast%3DHuet%26aufirst%3DG.%26aulast%3DChavatte%26aufirst%3DP.%26aulast%3DMelnyk%26aufirst%3DP.%26aulast%3DBailly%26aufirst%3DF.%26aulast%3DCotelle%26aufirst%3DP.%26atitle%3DToward%2520the%2520discovery%2520of%2520a%2520novel%2520class%2520of%2520YAP%2528-%2529TEAD%2520interaction%2520inhibitors%2520by%2520virtual%2520screening%2520approach%2520targeting%2520YAP%2528-%2529TEAD%2520protein%2528-%2529protein%2520interface%26jtitle%3DCancers%26date%3D2018%26volume%3D10%26spage%3D140%26doi%3D10.3390%2Fcancers10050140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gibault, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sturbaut, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailly, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnyk, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cotelle, P.</span></span> <span> </span><span class="NLM_article-title">Targeting transcriptional enhanced associate domains (TEADs)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">5057</span>– <span class="NLM_lpage">5072</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00879</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00879" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvF2gsLbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=5057-5072&author=F.+Gibaultauthor=M.+Sturbautauthor=F.+Baillyauthor=P.+Melnykauthor=P.+Cotelle&title=Targeting+transcriptional+enhanced+associate+domains+%28TEADs%29&doi=10.1021%2Facs.jmedchem.7b00879"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Transcriptional Enhanced Associate Domains (TEADs)</span></div><div class="casAuthors">Gibault, Floriane; Sturbaut, Manon; Bailly, Fabrice; Melnyk, Patricia; Cotelle, Philippe</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5057-5072</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Transcriptional enhanced assoc. domain (TEAD) proteins are the downstream effectors of the Hippo signaling pathway that regulate cell proliferation and stem cell functions.  TEADs are unable to activate transcription and require the help of coactivators such as YAP, TAZ, VgLL, and p160 proteins.  The expression of TEAD family is up-regulated in many cancer types including gastric, colorectal, breast, and prostate cancers, which is correlated with poor survival in patients.  Pharmacol. modulators of TEADs could therefore find application in cancer treatment and regenerative medicine.  In this review, the authors present the very recent available structures of TEADs with or without coactivators or inhibitors and discuss the potential therapeutic application of their ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-oyYqx1TP_LVg90H21EOLACvtfcHk0lhKxsr8o1BZ3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvF2gsLbN&md5=52bed2d6b0f8253b3a417f2569b9df2d</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00879&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00879%26sid%3Dliteratum%253Aachs%26aulast%3DGibault%26aufirst%3DF.%26aulast%3DSturbaut%26aufirst%3DM.%26aulast%3DBailly%26aufirst%3DF.%26aulast%3DMelnyk%26aufirst%3DP.%26aulast%3DCotelle%26aufirst%3DP.%26atitle%3DTargeting%2520transcriptional%2520enhanced%2520associate%2520domains%2520%2528TEADs%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D5057%26epage%3D5072%26doi%3D10.1021%2Facs.jmedchem.7b00879" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sessions, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoemark, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebrahimighaei, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNeill, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crump, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKay, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newby, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bond, M.</span></span> <span> </span><span class="NLM_article-title">Antiproliferative and antimigratory effects of a novel YAP-TEAD interaction inhibitor identified using in silico molecular docking</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">1291</span>– <span class="NLM_lpage">1305</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01402</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01402" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC1MXpvFCrtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=1291-1305&author=S.+A.+Smithauthor=R.+B.+Sessionsauthor=D.+K.+Shoemarkauthor=C.+Williamsauthor=R.+Ebrahimighaeiauthor=M.+C.+McNeillauthor=M.+P.+Crumpauthor=T.+R.+McKayauthor=G.+Harrisauthor=A.+C.+Newbyauthor=M.+Bond&title=Antiproliferative+and+antimigratory+effects+of+a+novel+YAP-TEAD+interaction+inhibitor+identified+using+in+silico+molecular+docking&doi=10.1021%2Facs.jmedchem.8b01402"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Antiproliferative and Antimigratory Effects of a Novel YAP-TEAD Interaction Inhibitor Identified Using in Silico Molecular Docking</span></div><div class="casAuthors">Smith, Sarah A.; Sessions, Richard B.; Shoemark, Deborah K.; Williams, Christopher; Ebrahimighaei, Reza; McNeill, Madeleine C.; Crump, Matthew P.; McKay, Tristan R.; Harris, Gemma; Newby, Andrew C.; Bond, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1291-1305</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Hippo pathway is an important regulator of cell growth, proliferation, and migration.  TEAD transcription factors, which lie at the core of the Hippo pathway, are essential for regulation of organ growth and wound repair.  Dysregulation of TEAD and its regulatory cofactor Yes-assocd. protein (YAP) have been implicated in numerous human cancers and hyperproliferative pathol. processes.  Hence, the YAP-TEAD complex is a promising therapeutic target.  Here, we use in silico mol. docking using Bristol University Docking Engine to screen a library of more than 8 million druglike mols. for novel disrupters of the YAP-TEAD interaction.  We report the identification of a novel compd. (CPD3.1) with the ability to disrupt YAP-TEAD protein-protein interaction and inhibit TEAD activity, cell proliferation, and cell migration.  The YAP-TEAD complex is a viable drug target, and CPD3.1 is a lead compd. for the development of more potent TEAD inhibitors for treating cancer and other hyperproliferative pathologies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqLhehTsmBN7Vg90H21EOLACvtfcHk0lhKxsr8o1BZ3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXpvFCrtQ%253D%253D&md5=1dbe1315ab5036198e0f2d482d7b8cd0</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01402&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01402%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DS.%2BA.%26aulast%3DSessions%26aufirst%3DR.%2BB.%26aulast%3DShoemark%26aufirst%3DD.%2BK.%26aulast%3DWilliams%26aufirst%3DC.%26aulast%3DEbrahimighaei%26aufirst%3DR.%26aulast%3DMcNeill%26aufirst%3DM.%2BC.%26aulast%3DCrump%26aufirst%3DM.%2BP.%26aulast%3DMcKay%26aufirst%3DT.%2BR.%26aulast%3DHarris%26aufirst%3DG.%26aulast%3DNewby%26aufirst%3DA.%2BC.%26aulast%3DBond%26aufirst%3DM.%26atitle%3DAntiproliferative%2520and%2520antimigratory%2520effects%2520of%2520a%2520novel%2520YAP-TEAD%2520interaction%2520inhibitor%2520identified%2520using%2520in%2520silico%2520molecular%2520docking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D1291%26epage%3D1305%26doi%3D10.1021%2Facs.jmedchem.8b01402" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, L.</span></span> <span> </span><span class="NLM_article-title">VGLL4 is a transcriptional cofactor acting as a novel tumor suppressor via interacting with TEADs</span>. <i>Am. J. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">932</span>– <span class="NLM_lpage">943</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=30034932" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFOjs7vL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=932-943&author=X.+Dengauthor=L.+Fang&title=VGLL4+is+a+transcriptional+cofactor+acting+as+a+novel+tumor+suppressor+via+interacting+with+TEADs"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">VGLL4 is a transcriptional cofactor acting as a novel tumor suppressor via interacting with TEADs</span></div><div class="casAuthors">Deng, Xiaochong; Fang, Lin</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Cancer Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">932-943</span>CODEN:
                <span class="NLM_cas:coden">AJCRFT</span>;
        ISSN:<span class="NLM_cas:issn">2156-6976</span>.
    
            (<span class="NLM_cas:orgname">e-Century Publishing Corp.</span>)
        </div><div class="casAbstract">Vestigial Like Family Member 4 (VGLL4) is a transcriptional cofactor of VGLL family, which includes VGLL1- 4.  Unlike other members of VGLL family, VGLL4 was described as a novel tumor suppressor contg. two TDU motifs.  VGLL4 executes its biol. function through two TDU domains via interacting with TEA domain (TEAD) transcription factors.  Lower expression of VGLL4 usually indicates poor survival in many cancers, such as lung cancer, gastric cancer, breast cancer, colorectal cancer, bladder cancer, pancreatic adenocarcinoma and esophageal squamous cancer.  In cancer cells, the expression of VGLL4 is lower than that of normal tissues, moreover, expression level of VGLL4 is pos. related to survival rate.  VGLL4 is found to play an important role in several signal pathways, mainly acts as a tumor suppressor interacting with TEADs.  In Hippo signaling pathway, VGLL4 competes with YAP in binding to TEADs and inhibits the downstream of YAP.  In Wnt/β-catenin signaling pathway, VGLL4 neg. regulates Wnt/β-catenin signaling pathway via inhibiting β-catenin and TCF (T-cell factor).  VGLL4 can also suppress epithelial-mesenchymal transition (EMT) and contribute to apoptosis signaling pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0ATex5pIVN7Vg90H21EOLACvtfcHk0lhKxsr8o1BZ3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFOjs7vL&md5=a68a7902681c58f78d54b4ce285f1bdb</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DFang%26aufirst%3DL.%26atitle%3DVGLL4%2520is%2520a%2520transcriptional%2520cofactor%2520acting%2520as%2520a%2520novel%2520tumor%2520suppressor%2520via%2520interacting%2520with%2520TEADs%26jtitle%3DAm.%2520J.%2520Cancer%2520Res.%26date%3D2018%26volume%3D8%26spage%3D932%26epage%3D943" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pobbati, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, W.</span></span> <span> </span><span class="NLM_article-title">Emerging roles of TEAD transcription factors and its coactivators in cancers</span>. <i>Cancer Biol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">390</span>– <span class="NLM_lpage">398</span>, <span class="refDoi"> DOI: 10.4161/cbt.23788</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.4161%2Fcbt.23788" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=23380592" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjvFOrtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=390-398&author=A.+V.+Pobbatiauthor=W.+Hong&title=Emerging+roles+of+TEAD+transcription+factors+and+its+coactivators+in+cancers&doi=10.4161%2Fcbt.23788"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging roles of TEAD transcription factors and its coactivators in cancers</span></div><div class="casAuthors">Pobbati, Ajaybabu V.; Hong, Wanjin</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Biology & Therapy</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">390-398</span>CODEN:
                <span class="NLM_cas:coden">CBTAAO</span>;
        ISSN:<span class="NLM_cas:issn">1538-4047</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">A review.  TEAD proteins are transcription factors that are crucial for development, but also play a role in cancers.  Several developmentally and pathol. important genes are upregulated by TEADs.  TEADs have a TEA domain that enables them to bind specific DNA elements and a transactivation domain that enables them to interact with coactivators.  TEADs on their own are unable to activate transcription and they require the help of coactivators.  Several TEAD-interacting coactivators are known and they can be classified into three groups: (1) YAP and its paralog TAZ, (2) VgII proteins and (3) p160s.  Accordingly, these coactivators also play a role in development and cancers.  Recent studies have shown that TEADs and their coactivators aid in the progression of various cancers, including the difficult to treat glioblastoma, liver and ovarian cancers.  They facilitate cancer progression through expression of proliferation promoting genes such as c-myc, survivin, Axl, CTGF and Cyr61.  There is also a good correlation between high TEAD or its coactivator expression and poor prognosis in various cancers.  Given the fact that TEADs and their coactivators need to work together for a functional outcome, disrupting the interaction between them appears to be a viable option for cancer therapy.  Structures of TEAD-coactivator complexes have been elucidated and will facilitate drug design and development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaxbkfWmzipLVg90H21EOLACvtfcHk0lj4F_x1rP5LWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjvFOrtro%253D&md5=86bb7724f71105cd3d27fd0d12f52a8e</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.4161%2Fcbt.23788&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcbt.23788%26sid%3Dliteratum%253Aachs%26aulast%3DPobbati%26aufirst%3DA.%2BV.%26aulast%3DHong%26aufirst%3DW.%26atitle%3DEmerging%2520roles%2520of%2520TEAD%2520transcription%2520factors%2520and%2520its%2520coactivators%2520in%2520cancers%26jtitle%3DCancer%2520Biol.%2520Ther.%26date%3D2013%26volume%3D14%26spage%3D390%26epage%3D398%26doi%3D10.4161%2Fcbt.23788" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeRan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jarugumilli, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span> <span> </span><span class="NLM_article-title">Autopalmitoylation of TEAD proteins regulates transcriptional output of the hippo pathway</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">282</span>– <span class="NLM_lpage">289</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1038%2Fnchembio.2036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=26900866" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC28XivFKgtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=282-289&author=P.+Chanauthor=X.+Hanauthor=B.+Zhengauthor=M.+DeRanauthor=J.+Yuauthor=G.+K.+Jarugumilliauthor=H.+Dengauthor=D.+Panauthor=X.+Luoauthor=X.+Wu&title=Autopalmitoylation+of+TEAD+proteins+regulates+transcriptional+output+of+the+hippo+pathway&doi=10.1038%2Fnchembio.2036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Autopalmitoylation of TEAD proteins regulates transcriptional output of the Hippo pathway</span></div><div class="casAuthors">Chan, PuiYee; Han, Xiao; Zheng, Baohui; DeRan, Michael; Yu, Jianzhong; Jarugumilli, Gopala K.; Deng, Hua; Pan, Duojia; Luo, Xuelian; Wu, Xu</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">282-289</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">TEA domain (TEAD) transcription factors bind to the coactivators YAP and TAZ and regulate the transcriptional output of the Hippo pathway, playing crit. roles in organ size control and tumorigenesis.  Protein S-palmitoylation attaches the fatty acid palmitate to cysteine residues and regulates protein trafficking, membrane localization and signaling activities.  Using activity-based chem. probes, we discovered that human TEADs possess intrinsic palmitoylating enzyme-like activities and undergo autopalmitoylation at evolutionarily conserved cysteine residues under physiol. conditions.  We detd. the crystal structures of lipid-bound TEADs and found that the lipid chain of palmitate inserts into a conserved deep hydrophobic pocket.  Strikingly, palmitoylation did not alter TEAD's localization, but it was required for TEAD's binding to YAP and TAZ and was dispensable for its binding to the Vgll4 tumor suppressor.  Moreover, palmitoylation-deficient TEAD mutants impaired TAZ-mediated muscle differentiation in vitro and tissue overgrowth mediated by the Drosophila YAP homolog Yorkie in vivo.  Our study directly links autopalmitoylation to the transcriptional regulation of the Hippo pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_XknCE2cSXLVg90H21EOLACvtfcHk0lj4F_x1rP5LWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XivFKgtrs%253D&md5=1b63c4f8b84d04f672041228c2f044c0</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2036%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DP.%26aulast%3DHan%26aufirst%3DX.%26aulast%3DZheng%26aufirst%3DB.%26aulast%3DDeRan%26aufirst%3DM.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DJarugumilli%26aufirst%3DG.%2BK.%26aulast%3DDeng%26aufirst%3DH.%26aulast%3DPan%26aufirst%3DD.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DX.%26atitle%3DAutopalmitoylation%2520of%2520TEAD%2520proteins%2520regulates%2520transcriptional%2520output%2520of%2520the%2520hippo%2520pathway%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D282%26epage%3D289%26doi%3D10.1038%2Fnchembio.2036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mesrouze, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerhofer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bokhovchuk, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontana, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmermann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delaunay, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Izaac, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kallen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmelzle, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdmann, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chène, P.</span></span> <span> </span><span class="NLM_article-title">Effect of the acylation of TEAD4 on its interaction with co-activators YAP and TAZ</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">2399</span>– <span class="NLM_lpage">2409</span>, <span class="refDoi"> DOI: 10.1002/pro.3312</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1002%2Fpro.3312" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=28960584" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2nsbbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2017&pages=2399-2409&author=Y.+Mesrouzeauthor=M.+Meyerhoferauthor=F.+Bokhovchukauthor=P.+Fontanaauthor=C.+Zimmermannauthor=T.+Martinauthor=C.+Delaunayauthor=A.+Izaacauthor=J.+Kallenauthor=T.+Schmelzleauthor=D.+Erdmannauthor=P.+Ch%C3%A8ne&title=Effect+of+the+acylation+of+TEAD4+on+its+interaction+with+co-activators+YAP+and+TAZ&doi=10.1002%2Fpro.3312"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Effect of the acylation of TEAD4 on its interaction with co-activators YAP and TAZ</span></div><div class="casAuthors">Mesrouze, Yannick; Meyerhofer, Marco; Bokhovchuk, Fedir; Fontana, Patrizia; Zimmermann, Catherine; Martin, Typhaine; Delaunay, Clara; Izaac, Aude; Kallen, Joerg; Schmelzle, Tobias; Erdmann, Dirk; Chene, Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2399-2409</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">1469-896X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The Hippo pathway is deregulated in various cancers, and the discovery of mols. that modulate this pathway may open new therapeutic avenues in oncol.  TEA/ATTS domain (TEAD) transcription factors are the most distal elements of the Hippo pathway and their transcriptional activity is regulated by the Yes-assocd. protein (YAP).  Amongst the various possibilities for targeting this pathway, inhibition of the YAP:TEAD interaction is an attractive strategy.  It has been shown recently that TEAD proteins are covalently linked via a conserved cysteine to a fatty acid mol. (palmitate) that binds to a deep hydrophobic cavity present in these proteins.  This acylation of TEAD seems to be required for efficient binding to YAP, and understanding how it modulates the YAP:TEAD interaction may provide useful information on the regulation of TEAD function.  In this report we have studied the effect of TEAD4 acylation on its interaction with YAP and the other co-activator transcriptional co-activator with PDZ-binding motif (TAZ).  We show in our biochem. and cellular assays that YAP and TAZ bind in a similar manner to acylated and non-acylated TEAD4.  This indicates that TEAD4 acylation is not a prerequisite for its interaction with YAP or TAZ.  However, we obsd. that TEAD4 acylation significantly enhances its stability, suggesting that it may help this transcription factor to acquire and/or maintain its active conformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8jstEQT15tLVg90H21EOLACvtfcHk0lj4F_x1rP5LWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2nsbbE&md5=8604423205879435c87cced3203640e6</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fpro.3312&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.3312%26sid%3Dliteratum%253Aachs%26aulast%3DMesrouze%26aufirst%3DY.%26aulast%3DMeyerhofer%26aufirst%3DM.%26aulast%3DBokhovchuk%26aufirst%3DF.%26aulast%3DFontana%26aufirst%3DP.%26aulast%3DZimmermann%26aufirst%3DC.%26aulast%3DMartin%26aufirst%3DT.%26aulast%3DDelaunay%26aufirst%3DC.%26aulast%3DIzaac%26aufirst%3DA.%26aulast%3DKallen%26aufirst%3DJ.%26aulast%3DSchmelzle%26aufirst%3DT.%26aulast%3DErdmann%26aufirst%3DD.%26aulast%3DCh%25C3%25A8ne%26aufirst%3DP.%26atitle%3DEffect%2520of%2520the%2520acylation%2520of%2520TEAD4%2520on%2520its%2520interaction%2520with%2520co-activators%2520YAP%2520and%2520TAZ%26jtitle%3DProtein%2520Sci.%26date%3D2017%26volume%3D26%26spage%3D2399%26epage%3D2409%26doi%3D10.1002%2Fpro.3312" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Noland, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gierke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnier, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandoval, W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sagolla, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannoush, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairbrother, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, C. N.</span></span> <span> </span><span class="NLM_article-title">Palmitoylation of TEAD transcription factors is required for their stability and function in hippo pathway signaling</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">179</span>– <span class="NLM_lpage">186</span>, <span class="refDoi"> DOI: 10.1016/j.str.2015.11.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1016%2Fj.str.2015.11.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=26724994" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitV2msrrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=179-186&author=C.+L.+Nolandauthor=S.+Gierkeauthor=P.+D.+Schnierauthor=J.+Murrayauthor=W.+N.+Sandovalauthor=M.+Sagollaauthor=A.+Deyauthor=R.+N.+Hannoushauthor=W.+J.+Fairbrotherauthor=C.+N.+Cunningham&title=Palmitoylation+of+TEAD+transcription+factors+is+required+for+their+stability+and+function+in+hippo+pathway+signaling&doi=10.1016%2Fj.str.2015.11.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Palmitoylation of TEAD Transcription Factors Is Required for Their Stability and Function in Hippo Pathway Signaling</span></div><div class="casAuthors">Noland, Cameron L.; Gierke, Sarah; Schnier, Paul D.; Murray, Jeremy; Sandoval, Wendy N.; Sagolla, Meredith; Dey, Anwesha; Hannoush, Rami N.; Fairbrother, Wayne J.; Cunningham, Christian N.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">179-186</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The Hippo signaling pathway is responsible for regulating the function of TEAD family transcription factors in metazoans.  TEADs, with their co-activators YAP/TAZ, are crit. for controlling cell differentiation and organ size through their transcriptional activation of genes involved in cell growth and proliferation.  Dysregulation of the Hippo pathway has been implicated in multiple forms of cancer.  Here, we identify a novel form of regulation of TEAD family proteins.  We show that human TEADs are palmitoylated at a universally conserved cysteine, and report the crystal structures of the human TEAD2 and TEAD3 YAP-binding domains in their palmitoylated forms.  These structures show a palmitate bound within a highly conserved hydrophobic cavity at each protein's core.  Our findings also demonstrate that this modification is required for proper TEAD folding and stability, indicating a potential new avenue for pharmacol. regulating the Hippo pathway through the modulation of TEAD palmitoylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpu5u56RVLmHLVg90H21EOLACvtfcHk0ljMvhHPUPTQUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitV2msrrP&md5=6797e3861c599acaec9715dd14e89097</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2015.11.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2015.11.005%26sid%3Dliteratum%253Aachs%26aulast%3DNoland%26aufirst%3DC.%2BL.%26aulast%3DGierke%26aufirst%3DS.%26aulast%3DSchnier%26aufirst%3DP.%2BD.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DSandoval%26aufirst%3DW.%2BN.%26aulast%3DSagolla%26aufirst%3DM.%26aulast%3DDey%26aufirst%3DA.%26aulast%3DHannoush%26aufirst%3DR.%2BN.%26aulast%3DFairbrother%26aufirst%3DW.%2BJ.%26aulast%3DCunningham%26aufirst%3DC.%2BN.%26atitle%3DPalmitoylation%2520of%2520TEAD%2520transcription%2520factors%2520is%2520required%2520for%2520their%2520stability%2520and%2520function%2520in%2520hippo%2520pathway%2520signaling%26jtitle%3DStructure%26date%3D2016%26volume%3D24%26spage%3D179%26epage%3D186%26doi%3D10.1016%2Fj.str.2015.11.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, N.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gumbiner, B. M.</span></span> <span> </span><span class="NLM_article-title">Cell contact and Nf2/Merlin-dependent regulation of TEAD palmitoylation and activity</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">9877</span>– <span class="NLM_lpage">9882</span>, <span class="refDoi"> DOI: 10.1073/pnas.1819400116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1073%2Fpnas.1819400116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=31043565" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC1MXps1Clurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2019&pages=9877-9882&author=N.-G.+Kimauthor=B.+M.+Gumbiner&title=Cell+contact+and+Nf2%2FMerlin-dependent+regulation+of+TEAD+palmitoylation+and+activity&doi=10.1073%2Fpnas.1819400116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Cell contact and Nf2/Merlin-dependent regulation of TEAD palmitoylation and activity</span></div><div class="casAuthors">Kim, Nam-Gyun; Gumbiner, Barry M.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9877-9882</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The Hippo pathway is involved in regulating contact inhibition of proliferation and organ size control and responds to various phys. and biochem. stimuli.  It is a kinase cascade that neg. regulates the activity of cotranscription factors YAP and TAZ, which interact with DNA binding transcription factors including TEAD and activate the expression of target genes.  In this study, we show that the palmitoylation of TEAD, which controls the activity and stability of TEAD proteins, is actively regulated by cell d. independent of Lats, the key kinase of the Hippo pathway.  The expression of fatty acid synthase and acetyl-CoA carboxylase involved in de novo biosynthesis of palmitate is reduced by cell d. in an Nf2/Merlin-dependent manner.  Depalmitoylation of TEAD is mediated by depalmitoylases including APT2 and ABHD17A.  Palmitoylation-deficient TEAD4 mutant is unstable and degraded by proteasome through the activity of the E3 ubiquitin ligase CHIP.  These findings show that TEAD activity is tightly controlled through the regulation of palmitoylation and stability via the orchestration of FASN, depalmitoylases, and E3 ubiquitin ligase in response to cell contact.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9VITmJov-FLVg90H21EOLACvtfcHk0ljMvhHPUPTQUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXps1Clurs%253D&md5=a7520b16c5f37b6d7f347370176998d0</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1819400116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1819400116%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DN.-G.%26aulast%3DGumbiner%26aufirst%3DB.%2BM.%26atitle%3DCell%2520contact%2520and%2520Nf2%252FMerlin-dependent%2520regulation%2520of%2520TEAD%2520palmitoylation%2520and%2520activity%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2019%26volume%3D116%26spage%3D9877%26epage%3D9882%26doi%3D10.1073%2Fpnas.1819400116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pobbati, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiguang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huda, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, G.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chia, C. S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poulsen, A.</span></span> <span> </span><span class="NLM_article-title">Targeting the central pocket in human transcription factor TEAD as a potential cancer therapeutic strategy</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">2076</span>– <span class="NLM_lpage">2086</span>, <span class="refDoi"> DOI: 10.1016/j.str.2015.09.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1016%2Fj.str.2015.09.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=26592798" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslWrsLzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=2076-2086&author=A.+V.+Pobbatiauthor=X.+Hanauthor=A.+W.+Hungauthor=S.+Weiguangauthor=N.+Hudaauthor=G.-Y.+Chenauthor=C.+Kangauthor=C.+S.+B.+Chiaauthor=X.+Luoauthor=W.+Hongauthor=A.+Poulsen&title=Targeting+the+central+pocket+in+human+transcription+factor+TEAD+as+a+potential+cancer+therapeutic+strategy&doi=10.1016%2Fj.str.2015.09.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the Central Pocket in Human Transcription Factor TEAD as a Potential Cancer Therapeutic Strategy</span></div><div class="casAuthors">Pobbati, Ajaybabu V.; Han, Xiao; Hung, Alvin W.; Weiguang, Seetoh; Huda, Nur; Chen, Guo-Ying; Kang, CongBao; Chia, Cheng San Brian; Luo, Xuelian; Hong, Wanjin; Poulsen, Anders</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2076-2086</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The human TEAD family of transcription factors (TEAD1-4) is required for YAP-mediated transcription in the Hippo pathway.  Hyperactivation of TEAD's co-activator YAP contributes to tissue overgrowth and human cancers, suggesting that pharmacol. interference of TEAD-YAP activity may be an effective strategy for anticancer therapy.  Here we report the discovery of a central pocket in the YAP-binding domain (YBD) of TEAD that is targetable by small-mol. inhibitors.  Our X-ray crystallog. studies reveal that flufenamic acid, a non-steroidal anti-inflammatory drug (NSAID), binds to the central pocket of TEAD2 YBD.  Our biochem. and functional analyses further demonstrate that binding of NSAIDs to TEAD inhibits TEAD-YAP-dependent transcription, cell migration, and proliferation, indicating that the central pocket is important for TEAD function.  Therefore, our studies discover a novel way of targeting TEAD transcription factors and set the stage for therapeutic development of specific TEAD-YAP inhibitors against human cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzguxiNfMYsrVg90H21EOLACvtfcHk0ljMvhHPUPTQUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslWrsLzL&md5=614016edf5305d6a3be84e8a1ac42cb9</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2015.09.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2015.09.009%26sid%3Dliteratum%253Aachs%26aulast%3DPobbati%26aufirst%3DA.%2BV.%26aulast%3DHan%26aufirst%3DX.%26aulast%3DHung%26aufirst%3DA.%2BW.%26aulast%3DWeiguang%26aufirst%3DS.%26aulast%3DHuda%26aufirst%3DN.%26aulast%3DChen%26aufirst%3DG.-Y.%26aulast%3DKang%26aufirst%3DC.%26aulast%3DChia%26aufirst%3DC.%2BS.%2BB.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DHong%26aufirst%3DW.%26aulast%3DPoulsen%26aufirst%3DA.%26atitle%3DTargeting%2520the%2520central%2520pocket%2520in%2520human%2520transcription%2520factor%2520TEAD%2520as%2520a%2520potential%2520cancer%2520therapeutic%2520strategy%26jtitle%3DStructure%26date%3D2015%26volume%3D23%26spage%3D2076%26epage%3D2086%26doi%3D10.1016%2Fj.str.2015.09.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bum-Erdene, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez-Gutierrez, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghozayel, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Si, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shannon, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corson, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollok, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meroueh, S. O.</span></span> <span> </span><span class="NLM_article-title">Small-molecule covalent modification of conserved cysteine leads to allosteric inhibition of the TEAD-Yap protein-protein interaction</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">378</span>– <span class="NLM_lpage">389</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2018.11.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1016%2Fj.chembiol.2018.11.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=30581134" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisF2ksL7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2019&pages=378-389&author=K.+Bum-Erdeneauthor=D.+Zhouauthor=G.+Gonzalez-Gutierrezauthor=M.+K.+Ghozayelauthor=Y.+Siauthor=D.+Xuauthor=H.+E.+Shannonauthor=B.+J.+Baileyauthor=T.+W.+Corsonauthor=K.+E.+Pollokauthor=C.+D.+Wellsauthor=S.+O.+Meroueh&title=Small-molecule+covalent+modification+of+conserved+cysteine+leads+to+allosteric+inhibition+of+the+TEAD-Yap+protein-protein+interaction&doi=10.1016%2Fj.chembiol.2018.11.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Covalent Modification of Conserved Cysteine Leads to Allosteric Inhibition of the TEAD·Yap Protein-Protein Interaction</span></div><div class="casAuthors">Bum-Erdene, Khuchtumur; Zhou, Donghui; Gonzalez-Gutierrez, Giovanni; Ghozayel, Mona K.; Si, Yubing; Xu, David; Shannon, Harlan E.; Bailey, Barbara J.; Corson, Timothy W.; Pollok, Karen E.; Wells, Clark D.; Meroueh, Samy O.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">378-389.e13</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The Hippo pathway coordinates extracellular signals onto the control of tissue homeostasis and organ size.  Hippo signaling primarily regulates the ability of Yap1 to bind and co-activate TEA domain (TEAD) transcription factors.  Yap1 tightly binds to TEAD4 via a large flat interface, making the development of small-mol. orthosteric inhibitors highly challenging.  Here, we report small-mol. TEAD·Yap inhibitors that rapidly and selectively form a covalent bond with a conserved cysteine located within the unique deep hydrophobic palmitate-binding pocket of TEADs.  Inhibition of TEAD4 binding to Yap1 by these compds. was irreversible and occurred on a longer time scale.  In mammalian cells, the compds. formed a covalent complex with TEAD4, inhibited its binding to Yap1, blocked its transcriptional activity, and suppressed expression of connective tissue growth factor.  The compds. inhibited cell viability of patient-derived glioblastoma spheroids, making them suitable as chem. probes to explore Hippo signaling in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZkVd9Oyu-KbVg90H21EOLACvtfcHk0liXzXcLa73f8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisF2ksL7M&md5=13ab3a2b343da960c9106435ace5ed03</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2018.11.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2018.11.010%26sid%3Dliteratum%253Aachs%26aulast%3DBum-Erdene%26aufirst%3DK.%26aulast%3DZhou%26aufirst%3DD.%26aulast%3DGonzalez-Gutierrez%26aufirst%3DG.%26aulast%3DGhozayel%26aufirst%3DM.%2BK.%26aulast%3DSi%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DD.%26aulast%3DShannon%26aufirst%3DH.%2BE.%26aulast%3DBailey%26aufirst%3DB.%2BJ.%26aulast%3DCorson%26aufirst%3DT.%2BW.%26aulast%3DPollok%26aufirst%3DK.%2BE.%26aulast%3DWells%26aufirst%3DC.%2BD.%26aulast%3DMeroueh%26aufirst%3DS.%2BO.%26atitle%3DSmall-molecule%2520covalent%2520modification%2520of%2520conserved%2520cysteine%2520leads%2520to%2520allosteric%2520inhibition%2520of%2520the%2520TEAD-Yap%2520protein-protein%2520interaction%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2019%26volume%3D26%26spage%3D378%26epage%3D389%26doi%3D10.1016%2Fj.chembiol.2018.11.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaneda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seike, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uemori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myojo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aida, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danjo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakajima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuji, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamaguchi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yasunaga, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otsubo, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Onodera, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishiya, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saito, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishii, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakai, R.</span></span> <span> </span><span class="NLM_article-title">Abstract 3086 Discovery of a first-in-class TEAD inhibitor which directly inhibits YAP/TAZ-TEAD protein-protein interaction and shows a potent anti-tumor effect in malignant pleural mMesothelioma</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">3086</span>, <span class="refDoi"> DOI: 10.1158/1538-7445.AM2019-3086</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1158%2F1538-7445.AM2019-3086" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2019&pages=3086&author=A.+Kanedaauthor=T.+Seikeauthor=T.+Uemoriauthor=K.+Myojoauthor=K.+Aidaauthor=T.+Danjoauthor=T.+Nakajimaauthor=D.+Yamaguchiauthor=T.+Hamadaauthor=Y.+Tsujiauthor=K.+Hamaguchiauthor=M.+Yasunagaauthor=N.+Otsuboauthor=H.+Onoderaauthor=Y.+Nishiyaauthor=M.+Suzukiauthor=J.+Saitoauthor=T.+Ishiiauthor=R.+Nakai&title=Abstract+3086+Discovery+of+a+first-in-class+TEAD+inhibitor+which+directly+inhibits+YAP%2FTAZ-TEAD+protein-protein+interaction+and+shows+a+potent+anti-tumor+effect+in+malignant+pleural+mMesothelioma&doi=10.1158%2F1538-7445.AM2019-3086"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1158%2F1538-7445.AM2019-3086&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1538-7445.AM2019-3086%26sid%3Dliteratum%253Aachs%26aulast%3DKaneda%26aufirst%3DA.%26aulast%3DSeike%26aufirst%3DT.%26aulast%3DUemori%26aufirst%3DT.%26aulast%3DMyojo%26aufirst%3DK.%26aulast%3DAida%26aufirst%3DK.%26aulast%3DDanjo%26aufirst%3DT.%26aulast%3DNakajima%26aufirst%3DT.%26aulast%3DYamaguchi%26aufirst%3DD.%26aulast%3DHamada%26aufirst%3DT.%26aulast%3DTsuji%26aufirst%3DY.%26aulast%3DHamaguchi%26aufirst%3DK.%26aulast%3DYasunaga%26aufirst%3DM.%26aulast%3DOtsubo%26aufirst%3DN.%26aulast%3DOnodera%26aufirst%3DH.%26aulast%3DNishiya%26aufirst%3DY.%26aulast%3DSuzuki%26aufirst%3DM.%26aulast%3DSaito%26aufirst%3DJ.%26aulast%3DIshii%26aufirst%3DT.%26aulast%3DNakai%26aufirst%3DR.%26atitle%3DAbstract%25203086%2520Discovery%2520of%2520a%2520first-in-class%2520TEAD%2520inhibitor%2520which%2520directly%2520inhibits%2520YAP%252FTAZ-TEAD%2520protein-protein%2520interaction%2520and%2520shows%2520a%2520potent%2520anti-tumor%2520effect%2520in%2520malignant%2520pleural%2520mMesothelioma%26jtitle%3DCancer%2520Res.%26date%3D2019%26volume%3D79%26spage%3D3086%26doi%3D10.1158%2F1538-7445.AM2019-3086" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lian, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span> <span> </span><span class="NLM_article-title">Discovery and biological evaluation of vinylsulfonamide derivatives as highly potent, covalent TEAD autopalmitoylation inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>184</i></span>,  <span class="NLM_fpage">111767</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2019.111767</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1016%2Fj.ejmech.2019.111767" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=31622854" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFyhsLvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=184&publication_year=2019&pages=111767&author=W.+Luauthor=J.+Wangauthor=Y.+Liauthor=H.+Taoauthor=H.+Xiongauthor=F.+Lianauthor=J.+Gaoauthor=H.+Maauthor=T.+Luauthor=D.+Zhangauthor=X.+Yeauthor=H.+Dingauthor=L.+Yueauthor=Y.+Zhangauthor=H.+Tangauthor=N.+Zhangauthor=Y.+Yangauthor=H.+Jiangauthor=K.+Chenauthor=B.+Zhouauthor=C.+Luo&title=Discovery+and+biological+evaluation+of+vinylsulfonamide+derivatives+as+highly+potent%2C+covalent+TEAD+autopalmitoylation+inhibitors&doi=10.1016%2Fj.ejmech.2019.111767"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and biological evaluation of vinylsulfonamide derivatives as highly potent, covalent TEAD autopalmitoylation inhibitors</span></div><div class="casAuthors">Lu, Wenchao; Wang, Jun; Li, Yong; Tao, Hongru; Xiong, Huan; Lian, Fulin; Gao, Jing; Ma, Hongna; Lu, Tian; Zhang, Dan; Ye, Xiaoqing; Ding, Hong; Yue, Liyan; Zhang, Yuanyuan; Tang, Huanyu; Zhang, Naixia; Yang, Yaxi; Jiang, Hualiang; Chen, Kaixian; Zhou, Bing; Luo, Cheng</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">184</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">111767</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Transcriptional enhancer assocd. domain family members (TEADs) are the most important downstream effectors that play the pivotal role in the development, regeneration and tissue homeostasis.  Recent biochem. studies have demonstrated that TEADs could undergo autopalmitoylation that is indispensable for its function making the lipid-binding pocket an attractive target for chem. intervention.  Herein, through structure-based virtual screen and rational medicinal chem. optimization, we identified DC-TEADin02(I) as the most potent, selective, covalent TEAD autopalmitoylation inhibitor with the IC50 value of 197 ± 19 nM while it showed minimal effect on TEAD-YAP interaction.  Further biochem. counter-screens demonstrate the specific thiol reactivity and selectivity of I over the kinase family, lipid-binding proteins and epigenetic targets.  Notably, I inhibited TEADs transcription activity leading to downregulation of YAP-related downstream gene expression.  Taken together, our findings proved the validity of modulating transcriptional output in the Hippo signaling pathway through irreversible chem. interventions of TEADs autopalmitoylation activity, which may serve as a qualified chem. tool for TEADs palmitoylation-related studies in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGog-ZFmXN-ezLVg90H21EOLACvtfcHk0lilPvDOGvy2TA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFyhsLvN&md5=37289f125bb7e93056c058430b128e7e</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2019.111767&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2019.111767%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DTao%26aufirst%3DH.%26aulast%3DXiong%26aufirst%3DH.%26aulast%3DLian%26aufirst%3DF.%26aulast%3DGao%26aufirst%3DJ.%26aulast%3DMa%26aufirst%3DH.%26aulast%3DLu%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DYe%26aufirst%3DX.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DYue%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DTang%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DDiscovery%2520and%2520biological%2520evaluation%2520of%2520vinylsulfonamide%2520derivatives%2520as%2520highly%2520potent%252C%2520covalent%2520TEAD%2520autopalmitoylation%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2019%26volume%3D184%26spage%3D111767%26doi%3D10.1016%2Fj.ejmech.2019.111767" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jarugumilli, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cotton, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiol, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, P. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeRan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leiter, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ip, Y. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camargo, F. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, J.</span></span> <span> </span><span class="NLM_article-title">Lats1/2 Sustain intestinal stem cells and Wnt activation through TEAD-dependent and independent transcription</span>. <i>Cell Stem Cell</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">675</span>– <span class="NLM_lpage">692</span>, <span class="refDoi"> DOI: 10.1016/j.stem.2020.03.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1016%2Fj.stem.2020.03.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=32259481" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BB3cXmsFOltbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2020&pages=675-692&author=Q.+Liauthor=Y.+Sunauthor=G.+K.+Jarugumilliauthor=S.+Liuauthor=K.+Dangauthor=J.+L.+Cottonauthor=J.+Xiolauthor=P.+Y.+Chanauthor=M.+DeRanauthor=L.+Maauthor=R.+Liauthor=L.+J.+Zhuauthor=J.+H.+Liauthor=A.+B.+Leiterauthor=Y.+T.+Ipauthor=F.+D.+Camargoauthor=X.+Luoauthor=R.+L.+Johnsonauthor=X.+Wuauthor=J.+Mao&title=Lats1%2F2+Sustain+intestinal+stem+cells+and+Wnt+activation+through+TEAD-dependent+and+independent+transcription&doi=10.1016%2Fj.stem.2020.03.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Lats1/2 Sustain Intestinal Stem Cells and Wnt Activation through TEAD-Dependent and Independent Transcription</span></div><div class="casAuthors">Li, Qi; Sun, Yang; Jarugumilli, Gopala K.; Liu, Shun; Dang, Kyvan; Cotton, Jennifer L.; Xiol, Jordi; Chan, Pui Yee; DeRan, Michael; Ma, Lifang; Li, Rui; Zhu, Lihua J.; Li, Joyce H.; Leiter, Andrew B.; Ip, Y. Tony; Camargo, Fernando D.; Luo, Xuelian; Johnson, Randy L.; Wu, Xu; Mao, Junhao</div><div class="citationInfo"><span class="NLM_cas:title">Cell Stem Cell</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">675-692.e8</span>CODEN:
                <span class="NLM_cas:coden">CSCEC4</span>;
        ISSN:<span class="NLM_cas:issn">1875-9777</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Intestinal homeostasis is tightly regulated by complex yet poorly understood signaling networks.  Here, we demonstrate that Lats1/2, the core Hippo kinases, are essential to maintain Wnt pathway activity and intestinal stem cells.  Lats1/2 deletion leads to loss of intestinal stem cells but drives Wnt-uncoupled crypt expansion.  To explore the function of downstream transcriptional enhanced assoc. domain (TEAD) transcription factors, we identified a selective small-mol. reversible inhibitor of TEAD auto-palmitoylation that directly occupies its lipid-binding site and inhibits TEAD-mediated transcription in vivo.  Combining this chem. tool with genetic and proteomics approaches, we show that intestinal Wnt inhibition by Lats deletion is Yes-assocd. protein (YAP)/transcriptional activator with PDZ-binding domain (TAZ) dependent but TEAD independent.  Mechanistically, nuclear YAP/TAZ interact with Groucho/Transducin-Like Enhancer of Split (TLE) to block Wnt/T-cell factor (TCF)-mediated transcription, and dual inhibition of TEAD and Lats suppresses Wnt-uncoupled Myc upregulation and epithelial over-proliferation in Adenomatous polyposis coli (APC)-mutated intestine.  Our studies highlight a pharmacol. approach to inhibit TEAD palmitoylation and have important implications for targeting Wnt and Hippo signaling in human malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAIgIAq16ZxrVg90H21EOLACvtfcHk0lilPvDOGvy2TA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXmsFOltbc%253D&md5=ce6305aa48f08aceb2412d72beb85e0f</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.stem.2020.03.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.stem.2020.03.002%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DJarugumilli%26aufirst%3DG.%2BK.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DDang%26aufirst%3DK.%26aulast%3DCotton%26aufirst%3DJ.%2BL.%26aulast%3DXiol%26aufirst%3DJ.%26aulast%3DChan%26aufirst%3DP.%2BY.%26aulast%3DDeRan%26aufirst%3DM.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DR.%26aulast%3DZhu%26aufirst%3DL.%2BJ.%26aulast%3DLi%26aufirst%3DJ.%2BH.%26aulast%3DLeiter%26aufirst%3DA.%2BB.%26aulast%3DIp%26aufirst%3DY.%2BT.%26aulast%3DCamargo%26aufirst%3DF.%2BD.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DJohnson%26aufirst%3DR.%2BL.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DMao%26aufirst%3DJ.%26atitle%3DLats1%252F2%2520Sustain%2520intestinal%2520stem%2520cells%2520and%2520Wnt%2520activation%2520through%2520TEAD-dependent%2520and%2520independent%2520transcription%26jtitle%3DCell%2520Stem%2520Cell%26date%3D2020%26volume%3D26%26spage%3D675%26epage%3D692%26doi%3D10.1016%2Fj.stem.2020.03.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kunig, V. B. K.</span>; <span class="NLM_string-name">Potowski, M.</span>; <span class="NLM_string-name">Akbarzadeh, M.</span>; <span class="NLM_string-name">Klika Škopić, M.</span>; <span class="NLM_string-name">Santos Smith, D.</span>; <span class="NLM_string-name">Arendt, L.</span>; <span class="NLM_string-name">Dormuth, I.</span>; <span class="NLM_string-name">Adihou, H.</span>; <span class="NLM_string-name">Andlovic, B.</span>; <span class="NLM_string-name">Karatas, H.</span>; <span class="NLM_string-name">Shaabani, S.</span>; <span class="NLM_string-name">Zarganes-Tzitzikas, T.</span>; <span class="NLM_string-name">Neochoritis, C. G.</span>; <span class="NLM_string-name">Zhang, R.</span>; <span class="NLM_string-name">Groves, M.</span>; <span class="NLM_string-name">Guéret, S. M.</span>; <span class="NLM_string-name">Ottmann, C.</span>; <span class="NLM_string-name">Rahnenführer, J.</span>; <span class="NLM_string-name">Fried, R.</span>; <span class="NLM_string-name">Dömling, A.</span>; <span class="NLM_string-name">Brunschweiger, A.</span></span> <span> </span><span class="NLM_article-title">TEAD-YAP interaction inhibitors and MDM2 binders from DNA-encoded indole-focused Ugi-peptidomimetics</span>.  <i>Angew. Chem., Int. Ed. Engl.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume">59</span>, <span class="refDoi"> DOI: 10.1002/anie.202006280</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1002%2Fanie.202006280" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2020&author=V.+B.+K.+Kunig&author=M.+Potowski&author=M.+Akbarzadeh&author=M.+Klika+%C5%A0kopi%C4%87&author=D.+Santos+Smith&author=L.+Arendt&author=I.+Dormuth&author=H.+Adihou&author=B.+Andlovic&author=H.+Karatas&author=S.+Shaabani&author=T.+Zarganes-Tzitzikas&author=C.+G.+Neochoritis&author=R.+Zhang&author=M.+Groves&author=S.+M.+Gu%C3%A9ret&author=C.+Ottmann&author=J.+Rahnenf%C3%BChrer&author=R.+Fried&author=A.+D%C3%B6mling&author=A.+Brunschweiger&title=TEAD-YAP+interaction+inhibitors+and+MDM2+binders+from+DNA-encoded+indole-focused+Ugi-peptidomimetics&doi=10.1002%2Fanie.202006280"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1002%2Fanie.202006280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.202006280%26sid%3Dliteratum%253Aachs%26aulast%3DKunig%26aufirst%3DV.%2BB.%2BK.%26atitle%3DTEAD-YAP%2520interaction%2520inhibitors%2520and%2520MDM2%2520binders%2520from%2520DNA-encoded%2520indole-focused%2520Ugi-peptidomimetics%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%2520Engl.%26date%3D2020%26volume%3D59%26doi%3D10.1002%2Fanie.202006280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pobbati, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mejuch, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakraborty, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karatas, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bharath, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guéret, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goy, P.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahne, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pahl, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sievers, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, W.</span></span> <span> </span><span class="NLM_article-title">Identification of quinolinols as activators of TEAD-dependent transcription</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">2909</span>– <span class="NLM_lpage">2921</span>, <span class="refDoi"> DOI: 10.1021/acschembio.9b00786</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.9b00786" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFOmt7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=2909-2921&author=A.+V.+Pobbatiauthor=T.+Mejuchauthor=S.+Chakrabortyauthor=H.+Karatasauthor=S.+R.+Bharathauthor=S.+M.+Gu%C3%A9retauthor=P.-A.+Goyauthor=G.+Hahneauthor=A.+Pahlauthor=S.+Sieversauthor=E.+Guccioneauthor=H.+Songauthor=H.+Waldmannauthor=W.+Hong&title=Identification+of+quinolinols+as+activators+of+TEAD-dependent+transcription&doi=10.1021%2Facschembio.9b00786"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Quinolinols as Activators of TEAD-Dependent Transcription</span></div><div class="casAuthors">Pobbati, Ajaybabu V.; Mejuch, Tom; Chakraborty, Sayan; Karatas, Hacer; Bharath, Sakshibeedu R.; Gueret, Stephanie M.; Goy, Pierre-Alexis; Hahne, Gernot; Pahl, Axel; Sievers, Sonja; Guccione, Ernesto; Song, Haiwei; Waldmann, Herbert; Hong, Wanjin</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2909-2921</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The transcriptional co-regulators YAP (Yes-assocd. protein) and TAZ (Transcriptional co-activator with PDZ-binding motif) are the vertebrate downstream effectors of the Hippo signaling pathway that control various physiol. and pathol. processes.  YAP and TAZ pair with the TEAD (TEA domain) family of transcription factors to initiate transcription.  We previously identified a tractable pocket in TEADs, which has been physiol. shown to bind palmitate.  Herein, a TEAD-Palmitate interaction screen was developed to select small mols. occupying the palmitate-binding pocket (PBP) of TEADs.  We show that quinolinols were TEAD-binding compds. that augment YAP/TAZ-TEAD activity, which was verified using TEAD reporter assay, RT-qPCR and RNA-Seq analyses.  Structure activity relationship investigations uncovered the quinolinol substituents that are necessary for TEAD activation.  We reveal a novel mechanism where quinolinols stabilize YAP/TAZ protein levels by occupying the PBP.  The enhancement of YAP activity by quinolinols accelerates the in vivo wound closure in a mouse wound-healing model.  Although small mols. that occupy the PBP have been shown to inhibit YAP/TAZ-TEAD activity, leveraging PBP to activate TEADs is a novel approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJh56PW0Y3orVg90H21EOLACvtfcHk0lilPvDOGvy2TA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFOmt7rO&md5=b0ba938316568aeff3bd8a4136b7a26f</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Facschembio.9b00786&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.9b00786%26sid%3Dliteratum%253Aachs%26aulast%3DPobbati%26aufirst%3DA.%2BV.%26aulast%3DMejuch%26aufirst%3DT.%26aulast%3DChakraborty%26aufirst%3DS.%26aulast%3DKaratas%26aufirst%3DH.%26aulast%3DBharath%26aufirst%3DS.%2BR.%26aulast%3DGu%25C3%25A9ret%26aufirst%3DS.%2BM.%26aulast%3DGoy%26aufirst%3DP.-A.%26aulast%3DHahne%26aufirst%3DG.%26aulast%3DPahl%26aufirst%3DA.%26aulast%3DSievers%26aufirst%3DS.%26aulast%3DGuccione%26aufirst%3DE.%26aulast%3DSong%26aufirst%3DH.%26aulast%3DWaldmann%26aufirst%3DH.%26aulast%3DHong%26aufirst%3DW.%26atitle%3DIdentification%2520of%2520quinolinols%2520as%2520activators%2520of%2520TEAD-dependent%2520transcription%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2019%26volume%3D14%26spage%3D2909%26epage%3D2921%26doi%3D10.1021%2Facschembio.9b00786" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Holden, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noland, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zbieg, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lacap, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beroza, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reja, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tom, J. Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fong, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steffek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clausen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagenbeek, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, C. N.</span></span> <span> </span><span class="NLM_article-title">Small molecule dysregulation of TEAD lipidation induces a dominant-negative inhibition of hippo pathway signaling</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">107809</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2020.107809</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1016%2Fj.celrep.2020.107809" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=32579935" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1Kis7nN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2020&pages=107809&author=J.+K.+Holdenauthor=J.+J.+Crawfordauthor=C.+L.+Nolandauthor=S.+Schmidtauthor=J.+R.+Zbiegauthor=J.+A.+Lacapauthor=R.+Zangauthor=G.+M.+Millerauthor=Y.+Zhangauthor=P.+Berozaauthor=R.+Rejaauthor=W.+Leeauthor=J.+Y.+K.+Tomauthor=R.+Fongauthor=M.+Steffekauthor=S.+Clausenauthor=T.+J.+Hagenbeekauthor=T.+Huauthor=Z.+Zhouauthor=H.+C.+Shenauthor=C.+N.+Cunningham&title=Small+molecule+dysregulation+of+TEAD+lipidation+induces+a+dominant-negative+inhibition+of+hippo+pathway+signaling&doi=10.1016%2Fj.celrep.2020.107809"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Dysregulation of TEAD Lipidation Induces a Dominant-Negative Inhibition of Hippo Pathway Signaling</span></div><div class="casAuthors">Holden, Jeffrey K.; Crawford, James J.; Noland, Cameron L.; Schmidt, Stephen; Zbieg, Jason R.; Lacap, Jennifer A.; Zang, Richard; Miller, Gregory M.; Zhang, Yue; Beroza, Paul; Reja, Rohit; Lee, Wendy; Tom, Jeffrey Y. K.; Fong, Rina; Steffek, Micah; Clausen, Saundra; Hagenbeek, Thjis J.; Hu, Taishan; Zhou, Zheng; Shen, Hong C.; Cunningham, Christian N.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">107809</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The transcriptional enhanced assoc. domain (TEAD) family of transcription factors serves as the receptors for the downstream effectors of the Hippo pathway, YAP and TAZ, to upregulate the expression of multiple genes involved in cellular proliferation and survival.  Recent work identified TEAD S-palmitoylation as crit. for protein stability and activity as the lipid tail extends into a hydrophobic core of the protein.  Here, we report the identification and characterization of a potent small mol. that binds the TEAD lipid pocket (LP) and disrupts TEAD S-palmitoylation.  Using a variety of biochem., structural, and cellular methods, we uncover that TEAD S-palmitoylation functions as a TEAD homeostatic protein level checkpoint and that dysregulation of this lipidation affects TEAD transcriptional activity in a dominant-neg. manner.  Furthermore, we demonstrate that targeting the TEAD LP is a promising therapeutic strategy for modulating the Hippo pathway, showing tumor stasis in a mouse xenograft model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpg0_N1z8lt0LVg90H21EOLACvtfcHk0li08ejFYPAI6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1Kis7nN&md5=4c0c530d9cadf143c035a1ac0ae32633</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2020.107809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2020.107809%26sid%3Dliteratum%253Aachs%26aulast%3DHolden%26aufirst%3DJ.%2BK.%26aulast%3DCrawford%26aufirst%3DJ.%2BJ.%26aulast%3DNoland%26aufirst%3DC.%2BL.%26aulast%3DSchmidt%26aufirst%3DS.%26aulast%3DZbieg%26aufirst%3DJ.%2BR.%26aulast%3DLacap%26aufirst%3DJ.%2BA.%26aulast%3DZang%26aufirst%3DR.%26aulast%3DMiller%26aufirst%3DG.%2BM.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DBeroza%26aufirst%3DP.%26aulast%3DReja%26aufirst%3DR.%26aulast%3DLee%26aufirst%3DW.%26aulast%3DTom%26aufirst%3DJ.%2BY.%2BK.%26aulast%3DFong%26aufirst%3DR.%26aulast%3DSteffek%26aufirst%3DM.%26aulast%3DClausen%26aufirst%3DS.%26aulast%3DHagenbeek%26aufirst%3DT.%2BJ.%26aulast%3DHu%26aufirst%3DT.%26aulast%3DZhou%26aufirst%3DZ.%26aulast%3DShen%26aufirst%3DH.%2BC.%26aulast%3DCunningham%26aufirst%3DC.%2BN.%26atitle%3DSmall%2520molecule%2520dysregulation%2520of%2520TEAD%2520lipidation%2520induces%2520a%2520dominant-negative%2520inhibition%2520of%2520hippo%2520pathway%2520signaling%26jtitle%3DCell%2520Rep.%26date%3D2020%26volume%3D31%26spage%3D107809%26doi%3D10.1016%2Fj.celrep.2020.107809" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alexandrov, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mileni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chien, E. Y. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanson, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span> <span> </span><span class="NLM_article-title">Microscale fluorescent thermal stability assay for membrane proteins</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">351</span>– <span class="NLM_lpage">359</span>, <span class="refDoi"> DOI: 10.1016/j.str.2008.02.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1016%2Fj.str.2008.02.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=18334210" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjtVKjsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2008&pages=351-359&author=A.+I.+Alexandrovauthor=M.+Mileniauthor=E.+Y.+T.+Chienauthor=M.+A.+Hansonauthor=R.+C.+Stevens&title=Microscale+fluorescent+thermal+stability+assay+for+membrane+proteins&doi=10.1016%2Fj.str.2008.02.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Microscale fluorescent thermal stability assay for membrane proteins</span></div><div class="casAuthors">Alexandrov, Alexander I.; Mileni, Mauro; Chien, Ellen Y. T.; Hanson, Michael A.; Stevens, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">351-359</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Systematic efforts to understand membrane protein stability under a variety of different soln. conditions are not widely available for membrane proteins, mainly due to tech. problems stemming from the presence of detergents necessary to keep the proteins in the solubilized state and the background that such detergents usually generate during biophys. characterization.  In this report, the authors introduce an efficient microscale fluorescent stability screen using the thiol-specific fluorochrome N-[4-(7-diethylamino-4-methyl-3-coumarinyl)phenyl]maleimide (CPM) for stability profiling of membrane proteins under different soln. and ligand conditions.  The screen uses the chem. reactivity of the native cysteines embedded in the protein interior as a sensor for the overall integrity of the folded state.  The thermal information gained by thorough investigation of the protein stability landscape can be effectively used to guide purifn. and biophys. characterization efforts including crystn.  To evaluate the method, three different protein families were analyzed, including the Apelin G protein-coupled receptor (APJ).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzPDnouXzOg7Vg90H21EOLACvtfcHk0li08ejFYPAI6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjtVKjsro%253D&md5=ec6afa11f7fdb06b1e415e113d42b843</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2008.02.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2008.02.004%26sid%3Dliteratum%253Aachs%26aulast%3DAlexandrov%26aufirst%3DA.%2BI.%26aulast%3DMileni%26aufirst%3DM.%26aulast%3DChien%26aufirst%3DE.%2BY.%2BT.%26aulast%3DHanson%26aufirst%3DM.%2BA.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26atitle%3DMicroscale%2520fluorescent%2520thermal%2520stability%2520assay%2520for%2520membrane%2520proteins%26jtitle%3DStructure%26date%3D2008%26volume%3D16%26spage%3D351%26epage%3D359%26doi%3D10.1016%2Fj.str.2008.02.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brune, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corrie, J. E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webb, M. R.</span></span> <span> </span><span class="NLM_article-title">A fluorescent sensor of the phosphorylation state of nucleoside diphosphate kinase and its use to monitor nucleoside diphosphate concentrations in real time</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">5087</span>– <span class="NLM_lpage">5094</span>, <span class="refDoi"> DOI: 10.1021/bi002484h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi002484h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BD3MXit1Sksr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2001&pages=5087-5094&author=M.+Bruneauthor=J.+E.+T.+Corrieauthor=M.+R.+Webb&title=A+fluorescent+sensor+of+the+phosphorylation+state+of+nucleoside+diphosphate+kinase+and+its+use+to+monitor+nucleoside+diphosphate+concentrations+in+real+time&doi=10.1021%2Fbi002484h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">A fluorescent sensor of the phosphorylation state of nucleoside diphosphate kinase and its use to monitor nucleoside diphosphate concentrations in real time</span></div><div class="casAuthors">Brune, Martin; Corrie, John E. T.; Webb, Martin R.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">5087-5094</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A sensor for purine nucleoside diphosphates in soln. based on nucleoside diphosphate kinase (NDPK) has been developed.  A single cysteine was introduced into the protein and labeled with the environmentally sensitive fluorophore, N-[2-(iodoacetamido)ethyl]-7-diethylaminocoumarin-3-carboxamide.  The resultant mol. shows a 4-fold fluorescence increase when phosphorylated on His117; this phosphorylation is on the normal reaction pathway of the enzyme.  The emission max. of the phosphoenzyme is at 475 nm, with max. excitation at 430 nm.  The fluorescent phosphorylated NDPK is used to measure the amt. of ADP and the unphosphorylated to measure ATP.  The labeled protein is phosphorylated to >90%, and the resultant mol. is stable on ice or can be stored at -80°.  The fluorescence responds to the fraction of protein phosphorylated and so to the equil. between ADP plus NDPK∼P and ATP plus NDPK.  In effect, the sensor measures the ADP/ATP concn. ratio.  The enzyme has a broad specificity for the purine of the nucleotides, so the sensor also can measure GDP/GTP ratios.  The fluorescence and kinetic properties of the labeled protein are described.  The binding rate consts. of nucleotides are ∼105 M-1 s-1, and the fluorescence change is at >200 s-1 when the ADP concn. is >1 mM.  Results are presented with two well-defined systems, namely, the kinetics of ADP release from myosin subfragment 1 and GDP release from the small G protein, human rho.  The results obtained with this novel sensor agree with those from alternate methods and demonstrate the applicability for following micromolar changes in nucleoside diphosphate in real time.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodDeG3YnVMdLVg90H21EOLACvtfcHk0lggFm1c1bNpHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXit1Sksr4%253D&md5=0e461b9164c7ce973ba64583518dc38d</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fbi002484h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi002484h%26sid%3Dliteratum%253Aachs%26aulast%3DBrune%26aufirst%3DM.%26aulast%3DCorrie%26aufirst%3DJ.%2BE.%2BT.%26aulast%3DWebb%26aufirst%3DM.%2BR.%26atitle%3DA%2520fluorescent%2520sensor%2520of%2520the%2520phosphorylation%2520state%2520of%2520nucleoside%2520diphosphate%2520kinase%2520and%2520its%2520use%2520to%2520monitor%2520nucleoside%2520diphosphate%2520concentrations%2520in%2520real%2520time%26jtitle%3DBiochemistry%26date%3D2001%26volume%3D40%26spage%3D5087%26epage%3D5094%26doi%3D10.1021%2Fbi002484h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohwaki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneeweis, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stec, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varnerin, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goudreau, P. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cassaday, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamakawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kornienko, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodder, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inglese, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strulovici, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kusunoki, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tota, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takagi, T.</span></span> <span> </span><span class="NLM_article-title">A fluorescence-based thiol quantification assay for ultra-high-throughput screening for inhibitors of coenzyme A production</span>. <i>Assay Drug Dev. Technol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">361</span>– <span class="NLM_lpage">374</span>, <span class="refDoi"> DOI: 10.1089/adt.2007.105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1089%2Fadt.2007.105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=18452391" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnvFChu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2008&pages=361-374&author=C.+C.+Chungauthor=K.+Ohwakiauthor=J.+E.+Schneeweisauthor=E.+Stecauthor=J.+P.+Varnerinauthor=P.+N.+Goudreauauthor=A.+Changauthor=J.+Cassadayauthor=L.+Yangauthor=T.+Yamakawaauthor=O.+Kornienkoauthor=P.+Hodderauthor=J.+Ingleseauthor=M.+Ferrerauthor=B.+Struloviciauthor=J.+Kusunokiauthor=M.+R.+Totaauthor=T.+Takagi&title=A+fluorescence-based+thiol+quantification+assay+for+ultra-high-throughput+screening+for+inhibitors+of+coenzyme+A+production&doi=10.1089%2Fadt.2007.105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">A Fluorescence-Based Thiol Quantification Assay for Ultra-High-Throughput Screening for Inhibitors of Coenzyme A Production</span></div><div class="casAuthors">Chung, Christine C.; Ohwaki, Kenji; Schneeweis, Jonathan E.; Stec, Erica; Varnerin, Jeffrey P.; Goudreau, Paul N.; Chang, Amy; Cassaday, Jason; Yang, Lihu; Yamakawa, Takeru; Kornienko, Oleg; Hodder, Peter; Inglese, James; Ferrer, Marc; Strulovici, Berta; Kusunoki, Jun; Tota, Michael R.; Takagi, Toshimitsu</div><div class="citationInfo"><span class="NLM_cas:title">Assay and Drug Development Technologies</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">361-374</span>CODEN:
                <span class="NLM_cas:coden">ADDTAR</span>;
        ISSN:<span class="NLM_cas:issn">1540-658X</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">Here we report the development and miniaturization of a cell-free enzyme assay for ultra-high-throughput screening (uHTS) for inhibitors of two potential drug targets for obesity and cancer: fatty acid synthase (FAS) and acetyl-CoA carboxylase (ACC) 2.  This assay detects CoA, a product of the FAS-catalyzed condensation of malonyl-CoA and acetyl-CoA.  The free thiol of CoA can react with 7-diethylamino-3-(4'-maleimidylphenyl)-4-methylcoumarin (CPM), a profluorescent coumarin maleimide deriv. that becomes fluorescent upon reaction with thiols.  FAS produces long-chain fatty acid and CoA from the condensation of malonyl-CoA and acetyl-CoA.  In our FAS assay, CoA released in the FAS reaction forms a fluorescence adduct with CPM that emits at 530 nm when excited at 405 nm.  Using this detection method for CoA, we measured the activity of sequential enzymes in the fatty acid synthesis pathway to develop an ACC2/FAS-coupled assay where ACC2 produces malonyl-CoA from acetyl-CoA.  We miniaturized the FAS and ACC2/FAS assays to 3456- and 1536-well plate format, resp., and completed uHTSs for small mol. inhibitors of this enzyme system.  This report shows the results of assay development, miniaturization, and inhibitor screening for these potential drug targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeIh8mQ6DnfbVg90H21EOLACvtfcHk0lggFm1c1bNpHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnvFChu7g%253D&md5=c72d4b37902699489b68b3e45ba9a59a</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1089%2Fadt.2007.105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fadt.2007.105%26sid%3Dliteratum%253Aachs%26aulast%3DChung%26aufirst%3DC.%2BC.%26aulast%3DOhwaki%26aufirst%3DK.%26aulast%3DSchneeweis%26aufirst%3DJ.%2BE.%26aulast%3DStec%26aufirst%3DE.%26aulast%3DVarnerin%26aufirst%3DJ.%2BP.%26aulast%3DGoudreau%26aufirst%3DP.%2BN.%26aulast%3DChang%26aufirst%3DA.%26aulast%3DCassaday%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DL.%26aulast%3DYamakawa%26aufirst%3DT.%26aulast%3DKornienko%26aufirst%3DO.%26aulast%3DHodder%26aufirst%3DP.%26aulast%3DInglese%26aufirst%3DJ.%26aulast%3DFerrer%26aufirst%3DM.%26aulast%3DStrulovici%26aufirst%3DB.%26aulast%3DKusunoki%26aufirst%3DJ.%26aulast%3DTota%26aufirst%3DM.%2BR.%26aulast%3DTakagi%26aufirst%3DT.%26atitle%3DA%2520fluorescence-based%2520thiol%2520quantification%2520assay%2520for%2520ultra-high-throughput%2520screening%2520for%2520inhibitors%2520of%2520coenzyme%2520A%2520production%26jtitle%3DAssay%2520Drug%2520Dev.%2520Technol.%26date%3D2008%26volume%3D6%26spage%3D361%26epage%3D374%26doi%3D10.1089%2Fadt.2007.105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goncalves, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brannigan, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thinon, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olaleye, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serwa, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanzarone, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tate, E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leatherbarrow, R. J.</span></span> <span> </span><span class="NLM_article-title">A fluorescence-based assay for N-myristoyltransferase activity</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>421</i></span>,  <span class="NLM_fpage">342</span>– <span class="NLM_lpage">344</span>, <span class="refDoi"> DOI: 10.1016/j.ab.2011.10.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1016%2Fj.ab.2011.10.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=22051857" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC38XptVehtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=421&publication_year=2012&pages=342-344&author=V.+Goncalvesauthor=J.+A.+Branniganauthor=E.+Thinonauthor=T.+O.+Olaleyeauthor=R.+Serwaauthor=S.+Lanzaroneauthor=A.+J.+Wilkinsonauthor=E.+W.+Tateauthor=R.+J.+Leatherbarrow&title=A+fluorescence-based+assay+for+N-myristoyltransferase+activity&doi=10.1016%2Fj.ab.2011.10.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">A fluorescence-based assay for N-myristoyltransferase activity</span></div><div class="casAuthors">Goncalves, Victor; Brannigan, James A.; Thinon, Emmanuelle; Olaleye, Tayo O.; Serwa, Remigiusz; Lanzarone, Salvatore; Wilkinson, Anthony J.; Tate, Edward W.; Leatherbarrow, Robin J.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">421</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">342-344</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">N-myristoylation is the irreversible attachment of a C14 fatty acid, myristic acid, to the N-terminal glycine of a protein via formation of an amide bond.  This modification is catalyzed by myristoyl-CoA:protein N-myristoyltransferase (NMT), an enzyme ubiquitous in eukaryotes that is up-regulated in several cancers.  Here we report a sensitive fluorescence-based assay to study the enzymic activity of human NMT1 and NMT2 based on detection of CoA by 7-diethylamino-3-(4-maleimidophenyl)-4-methylcoumarin (CPM).  We also describe expression and characterization of NMT1 and NMT2 and assay validation with small mol. inhibitors.  This assay should be broadly applicable to NMTs from a range of organisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP8ouiGX0Ml7Vg90H21EOLACvtfcHk0lggFm1c1bNpHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XptVehtg%253D%253D&md5=b0b36c043d6947a0843e19eddb9d9f43</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2011.10.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2011.10.013%26sid%3Dliteratum%253Aachs%26aulast%3DGoncalves%26aufirst%3DV.%26aulast%3DBrannigan%26aufirst%3DJ.%2BA.%26aulast%3DThinon%26aufirst%3DE.%26aulast%3DOlaleye%26aufirst%3DT.%2BO.%26aulast%3DSerwa%26aufirst%3DR.%26aulast%3DLanzarone%26aufirst%3DS.%26aulast%3DWilkinson%26aufirst%3DA.%2BJ.%26aulast%3DTate%26aufirst%3DE.%2BW.%26aulast%3DLeatherbarrow%26aufirst%3DR.%2BJ.%26atitle%3DA%2520fluorescence-based%2520assay%2520for%2520N-myristoyltransferase%2520activity%26jtitle%3DAnal.%2520Biochem.%26date%3D2012%26volume%3D421%26spage%3D342%26epage%3D344%26doi%3D10.1016%2Fj.ab.2011.10.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conrad, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuneo, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hellinga, H. W.</span></span> <span> </span><span class="NLM_article-title">Orthogonal site-specific protein modification by engineering reversible thiol protection mechanisms</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">64</span>– <span class="NLM_lpage">73</span>, <span class="refDoi"> DOI: 10.1110/ps.04965405</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1110%2Fps.04965405" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=15576565" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFWmsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2005&pages=64-73&author=J.+J.+Smithauthor=D.+W.+Conradauthor=M.+J.+Cuneoauthor=H.+W.+Hellinga&title=Orthogonal+site-specific+protein+modification+by+engineering+reversible+thiol+protection+mechanisms&doi=10.1110%2Fps.04965405"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Orthogonal site-specific protein modification by engineering reversible thiol protection mechanisms</span></div><div class="casAuthors">Smith, J. Jefferson; Conrad, David W.; Cuneo, Matthew J.; Hellinga, Homme W.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">64-73</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Covalent modification is an important strategy for introducing new functions into proteins.  As engineered proteins become more sophisticated, it is often desirable to introduce multiple, modifications involving several different functionalities in a site-specific manner.  Such orthogonal labeling schemes require independent labeling of differentially reactive nucleophilic amino acid side chains.  We have developed two protein-mediated protection schemes that permit independent labeling of multiple thiols.  These schemes exploit metal coordination or disulfide bond formation to reversibly protect cysteines in a Cys2His2 zinc finger domain.  We constructed a variety of N- and C-terminal fusions of these domains with maltose-binding protein, which were labeled with two or three different fluorophores.  Multiple modifications were made by reacting an unprotected cysteine in MBP first, deprotecting the zinc finger, and then reacting the zinc finger cysteines.  The fusion proteins were orthogonally labeled with two different fluorophores, which exhibited intramol. fluorescene resonance energy transfer (FRET).  These conjugates showed up to a threefold ratiometric change in emission intensities in response to maltose binding.  We also demonstrated that the metal- and redox-mediated protection methods can be combined to produce triple independent modifications, and prepd. a protein labeled with three different fluorophores that exhibited a FRET relay.  Finally, labeled glucose-binding protein was covalently patterned on glass slides using thiol-mediated immobilization chemistries.  Together, these expts. demonstrated that reversible thiol protection schemes provide a rapid, straightforward method for producing multiple, site-specific modifications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrGQzcYV345rVg90H21EOLACvtfcHk0lgWU_p4ZRIxoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFWmsg%253D%253D&md5=b411f4b18b090392e54fc4307701b2b7</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1110%2Fps.04965405&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1110%252Fps.04965405%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DConrad%26aufirst%3DD.%2BW.%26aulast%3DCuneo%26aufirst%3DM.%2BJ.%26aulast%3DHellinga%26aufirst%3DH.%2BW.%26atitle%3DOrthogonal%2520site-specific%2520protein%2520modification%2520by%2520engineering%2520reversible%2520thiol%2520protection%2520mechanisms%26jtitle%3DProtein%2520Sci.%26date%3D2005%26volume%3D14%26spage%3D64%26epage%3D73%26doi%3D10.1110%2Fps.04965405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Konopíková, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uher, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bransová, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mastihuba, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudecová, D.</span></span> <span> </span><span class="NLM_article-title">Reaction of 5-hydroxy-2-hydroxymethyl-4H-pyran-4-one with aromatic aldehydes and amines (betti reaction)</span>. <i>Chem. Pap.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">182</span>– <span class="NLM_lpage">184</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADyaK2MXjsV2gs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=1994&pages=182-184&author=M.+Konop%C3%ADkov%C3%A1author=M.+Uherauthor=J.+Bransov%C3%A1author=V.+Mastihubaauthor=D.+Hudecov%C3%A1&title=Reaction+of+5-hydroxy-2-hydroxymethyl-4H-pyran-4-one+with+aromatic+aldehydes+and+amines+%28betti+reaction%29"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Reaction of 5-hydroxy-2-hydroxymethyl-4H-pyran-4-one with aromatic aldehydes and amines (Betti reaction)</span></div><div class="casAuthors">Konopikova, M.; Uher, M.; Bransova, J.; Mastihuba, V.; Hudecova, D.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Papers</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">182-4</span>CODEN:
                <span class="NLM_cas:coden">CHPAEG</span>;
        ISSN:<span class="NLM_cas:issn">0366-6352</span>.
    
            (<span class="NLM_cas:orgname">Slovak Academy of Sciences, Institute of Chemistry</span>)
        </div><div class="casAbstract">Prepn. and spectral characteristics of 6-(α-anilinobenzyl) derivs. of 5-hydroxy-2-hydroxymethyl-4H-pyran-4-one (kojic acid) were described.  Some products of Betti reaction are thermolabile and decomp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrf3Nr0h_MynbVg90H21EOLACvtfcHk0lgWU_p4ZRIxoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXjsV2gs7Y%253D&md5=38ab714dc3e7a702956d6349821a9cdd</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKonop%25C3%25ADkov%25C3%25A1%26aufirst%3DM.%26aulast%3DUher%26aufirst%3DM.%26aulast%3DBransov%25C3%25A1%26aufirst%3DJ.%26aulast%3DMastihuba%26aufirst%3DV.%26aulast%3DHudecov%25C3%25A1%26aufirst%3DD.%26atitle%3DReaction%2520of%25205-hydroxy-2-hydroxymethyl-4H-pyran-4-one%2520with%2520aromatic%2520aldehydes%2520and%2520amines%2520%2528betti%2520reaction%2529%26jtitle%3DChem.%2520Pap.%26date%3D1994%26volume%3D48%26spage%3D182%26epage%3D184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burnett, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergfeld, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belsito, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klaassen, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liebler, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marks, J. G.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shank, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slaga, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snyder, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, F. A.</span></span> <span> </span><span class="NLM_article-title">Final report of the safety assessment of kojic acid as used in cosmetics</span>. <i>Int. J. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">244S</span>– <span class="NLM_lpage">273S</span>, <span class="refDoi"> DOI: 10.1177/1091581810385956</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1177%2F1091581810385956" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=21164073" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtleiu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2010&pages=244S-273S&author=C.+L.+Burnettauthor=W.+F.+Bergfeldauthor=D.+V.+Belsitoauthor=R.+A.+Hillauthor=C.+D.+Klaassenauthor=D.+C.+Lieblerauthor=J.+G.+Marksauthor=R.+C.+Shankauthor=T.+J.+Slagaauthor=P.+W.+Snyderauthor=F.+A.+Andersen&title=Final+report+of+the+safety+assessment+of+kojic+acid+as+used+in+cosmetics&doi=10.1177%2F1091581810385956"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Final report of the safety assessment of kojic acid as used in cosmetics</span></div><div class="casAuthors">Burnett, Christina L.; Bergfeld, Wilma F.; Belsito, Donald V.; Hill, Ronald A.; Klaassen, Curtis D.; Liebler, Daniel C.; Marks, James G., Jr.; Shank, Ronald C.; Slaga, Thomas J.; Snyder, Paul W.; Andersen, F. Alan</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Toxicology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">Suppl. 4</span>),
    <span class="NLM_cas:pages">244S-273S</span>CODEN:
                <span class="NLM_cas:coden">IJTOFN</span>;
        ISSN:<span class="NLM_cas:issn">1091-5818</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A review.  Kojic acid functions as an antioxidant in cosmetic products.  Kojic acid was not a toxicant in acute, chronic, reproductive, and genotoxicity studies.  While some animal data suggested tumor promotion and weak carcinogenicity, kojic acid is slowly absorbed into the circulation from human skin and likely would not reach the threshold at which these effects were seen.  The available human sensitization data supported the safety of kojic acid at a use concn. of 2% in leave-on cosmetics.  Kojic acid depigmented black guinea pig skin at a concn. of 4%, but this effect was not seen at 1%.  The Cosmetic Ingredient Review (CIR) Expert Panel concluded that the 2 end points of concern, dermal sensitization and skin lightening, would not be seen at use concns. below 1%; therefore, this ingredient is safe for use in cosmetic products up to that level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmrJeP-lFMi7Vg90H21EOLACvtfcHk0lgWU_p4ZRIxoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtleiu7w%253D&md5=6285c4c02f9f18eaa6091d2f57b2383a</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1177%2F1091581810385956&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1091581810385956%26sid%3Dliteratum%253Aachs%26aulast%3DBurnett%26aufirst%3DC.%2BL.%26aulast%3DBergfeld%26aufirst%3DW.%2BF.%26aulast%3DBelsito%26aufirst%3DD.%2BV.%26aulast%3DHill%26aufirst%3DR.%2BA.%26aulast%3DKlaassen%26aufirst%3DC.%2BD.%26aulast%3DLiebler%26aufirst%3DD.%2BC.%26aulast%3DMarks%26aufirst%3DJ.%2BG.%26aulast%3DShank%26aufirst%3DR.%2BC.%26aulast%3DSlaga%26aufirst%3DT.%2BJ.%26aulast%3DSnyder%26aufirst%3DP.%2BW.%26aulast%3DAndersen%26aufirst%3DF.%2BA.%26atitle%3DFinal%2520report%2520of%2520the%2520safety%2520assessment%2520of%2520kojic%2520acid%2520as%2520used%2520in%2520cosmetics%26jtitle%3DInt.%2520J.%2520Toxicol.%26date%3D2010%26volume%3D29%26spage%3D244S%26epage%3D273S%26doi%3D10.1177%2F1091581810385956" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, E. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poulsen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pobbati, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, C.</span></span> <span> </span><span class="NLM_article-title">Structural and ligand-binding analysis of the YAP-binding domain of transcription factor TEAD4</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>475</i></span>,  <span class="NLM_fpage">2043</span>– <span class="NLM_lpage">2055</span>, <span class="refDoi"> DOI: 10.1042/bcj20180225</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1042%2FBCJ20180225" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=29760238" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlKqtbjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=475&publication_year=2018&pages=2043-2055&author=Y.+Liauthor=S.+Liuauthor=E.+Y.+Ngauthor=R.+Liauthor=A.+Poulsenauthor=J.+Hillauthor=A.+V.+Pobbatiauthor=A.+W.+Hungauthor=W.+Hongauthor=T.+H.+Kellerauthor=C.+Kang&title=Structural+and+ligand-binding+analysis+of+the+YAP-binding+domain+of+transcription+factor+TEAD4&doi=10.1042%2Fbcj20180225"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and ligand-binding analysis of the YAP-binding domain of transcription factor TEAD4</span></div><div class="casAuthors">Li, Yan; Liu, Shuang; Ng, Elizabeth Yihui; Li, Rong; Poulsen, Anders; Hill, Jeffrey; Pobbati, Ajaybabu V.; Hung, Alvin W.; Hong, Wanjin; Keller, Thomas H.; Kang, CongBao</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">475</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2043-2055</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">The oncoprotein YAP (Yes-assocd. protein) requires the TEAD family of transcription factors for the up-regulation of genes important for cell proliferation.  Disrupting YAP-TEAD interaction is an attractive strategy for cancer therapy.  Targeting TEADs using small mols. that either bind to the YAP-binding pocket or the palmitate-binding pocket is proposed to disrupt the YAP-TEAD interaction.  There is a need for methodologies to facilitate robust and reliable identification of compds. that occupy either YAP-binding pocket or palmitate-binding pocket.  Here, using NMR spectroscopy, we validated compds. that bind to these pockets and also identify the residues in mouse TEAD4 (mTEAD4) that interact with these compds.  Flufenamic acid (FA) was used as a pos. control for validation of palmitate-binding pocket-occupying compds. by NMR.  Furthermore, we identify a hit from a fragment screen and show that it occupies a site close to YAP-binding pocket on the TEAD surface.  Our results also indicate that purified mTEAD4 can catalyze autopalmitoylation.  NMR studies on mTEAD4 revealed that exchanges exist in TEAD as NMR signal broadening was obsd. for residues close to the palmitoylation site.  Mutating the palmitoylated cysteine (C360S mutant) abolished palmitoylation, while no significant changes in the NMR spectrum were obsd. for the mutant which still binds to YAP.  We also show that FA inhibits TEAD autopalmitoylation.  Our studies highlight the utility of NMR spectroscopy in identifying small mols. that bind to TEAD pockets and reinforce the notion that both palmitate-binding pocket and YAP-binding pocket are targetable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoq8Fg59GWiSLVg90H21EOLACvtfcHk0lgg81dm11opiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlKqtbjL&md5=9197b0611f10a4b5f6b82f73afa04f35</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1042%2FBCJ20180225&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBCJ20180225%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DNg%26aufirst%3DE.%2BY.%26aulast%3DLi%26aufirst%3DR.%26aulast%3DPoulsen%26aufirst%3DA.%26aulast%3DHill%26aufirst%3DJ.%26aulast%3DPobbati%26aufirst%3DA.%2BV.%26aulast%3DHung%26aufirst%3DA.%2BW.%26aulast%3DHong%26aufirst%3DW.%26aulast%3DKeller%26aufirst%3DT.%2BH.%26aulast%3DKang%26aufirst%3DC.%26atitle%3DStructural%2520and%2520ligand-binding%2520analysis%2520of%2520the%2520YAP-binding%2520domain%2520of%2520transcription%2520factor%2520TEAD4%26jtitle%3DBiochem.%2520J.%26date%3D2018%26volume%3D475%26spage%3D2043%26epage%3D2055%26doi%3D10.1042%2Fbcj20180225" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laraia, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friese, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corkery, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konstantinidis, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erwin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofer, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karatas, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klewer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brockmeyer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schölermann, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dwivedi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rios-Munoz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Köhn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vetter, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziegler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janning, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span> <span> </span><span class="NLM_article-title">The cholesterol transfer protein GRAMD1A regulates autophagosome biogenesis</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">710</span>– <span class="NLM_lpage">720</span>, <span class="refDoi"> DOI: 10.1038/s41589-019-0307-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1038%2Fs41589-019-0307-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=31222192" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1aqtbvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=710-720&author=L.+Laraiaauthor=A.+Frieseauthor=D.+P.+Corkeryauthor=G.+Konstantinidisauthor=N.+Erwinauthor=W.+Hoferauthor=H.+Karatasauthor=L.+Klewerauthor=A.+Brockmeyerauthor=M.+Metzauthor=B.+Sch%C3%B6lermannauthor=M.+Dwivediauthor=L.+Liauthor=P.+Rios-Munozauthor=M.+K%C3%B6hnauthor=R.+Winterauthor=I.+R.+Vetterauthor=S.+Zieglerauthor=P.+Janningauthor=Y.-W.+Wuauthor=H.+Waldmann&title=The+cholesterol+transfer+protein+GRAMD1A+regulates+autophagosome+biogenesis&doi=10.1038%2Fs41589-019-0307-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">The cholesterol transfer protein GRAMD1A regulates autophagosome biogenesis</span></div><div class="casAuthors">Laraia, Luca; Friese, Alexandra; Corkery, Dale P.; Konstantinidis, Georgios; Erwin, Nelli; Hofer, Walter; Karatas, Hacer; Klewer, Laura; Brockmeyer, Andreas; Metz, Malte; Schoelermann, Beate; Dwivedi, Mridula; Li, Lei; Rios-Munoz, Pablo; Koehn, Maja; Winter, Roland; Vetter, Ingrid R.; Ziegler, Slava; Janning, Petra; Wu, Yao-Wen; Waldmann, Herbert</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">710-720</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Autophagy mediates the degrdn. of damaged proteins, organelles and pathogens, and plays a key role in health and disease.  Thus, the identification of new mechanisms involved in the regulation of autophagy is of major interest.  In particular, little is known about the role of lipids and lipid-binding proteins in the early steps of autophagosome biogenesis.  Using target-agnostic, high-content, image-based identification of indicative phenotypic changes induced by small mols., we have identified autogramins as a new class of autophagy inhibitor.  Autogramins selectively target the recently discovered cholesterol transfer protein GRAM domain-contg. protein 1A (GRAMD1A, which had not previously been implicated in autophagy), and directly compete with cholesterol binding to the GRAMD1A StART domain.  GRAMD1A accumulates at sites of autophagosome initiation, affects cholesterol distribution in response to starvation and is required for autophagosome biogenesis.  These findings identify a new biol. function of GRAMD1A and a new role for cholesterol in autophagy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZaJquw1e9_bVg90H21EOLACvtfcHk0lgg81dm11opiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1aqtbvI&md5=ff7841edc7f22d611d6e681da5d74d0f</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fs41589-019-0307-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-019-0307-5%26sid%3Dliteratum%253Aachs%26aulast%3DLaraia%26aufirst%3DL.%26aulast%3DFriese%26aufirst%3DA.%26aulast%3DCorkery%26aufirst%3DD.%2BP.%26aulast%3DKonstantinidis%26aufirst%3DG.%26aulast%3DErwin%26aufirst%3DN.%26aulast%3DHofer%26aufirst%3DW.%26aulast%3DKaratas%26aufirst%3DH.%26aulast%3DKlewer%26aufirst%3DL.%26aulast%3DBrockmeyer%26aufirst%3DA.%26aulast%3DMetz%26aufirst%3DM.%26aulast%3DSch%25C3%25B6lermann%26aufirst%3DB.%26aulast%3DDwivedi%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DRios-Munoz%26aufirst%3DP.%26aulast%3DK%25C3%25B6hn%26aufirst%3DM.%26aulast%3DWinter%26aufirst%3DR.%26aulast%3DVetter%26aufirst%3DI.%2BR.%26aulast%3DZiegler%26aufirst%3DS.%26aulast%3DJanning%26aufirst%3DP.%26aulast%3DWu%26aufirst%3DY.-W.%26aulast%3DWaldmann%26aufirst%3DH.%26atitle%3DThe%2520cholesterol%2520transfer%2520protein%2520GRAMD1A%2520regulates%2520autophagosome%2520biogenesis%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D15%26spage%3D710%26epage%3D720%26doi%3D10.1038%2Fs41589-019-0307-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Medina-O’Donnell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivas, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reyes-Zurita, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin-Fonseca, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Granados, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrer-Martín, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lupiañez, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parra, A.</span></span> <span> </span><span class="NLM_article-title">Semi-synthesis and antiproliferative evaluation of PEGylated pentacyclic triterpenes</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">64</span>– <span class="NLM_lpage">78</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2016.04.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1016%2Fj.ejmech.2016.04.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=27128174" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC28XnslWqtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2016&pages=64-78&author=M.+Medina-O%E2%80%99Donnellauthor=F.+Rivasauthor=F.+J.+Reyes-Zuritaauthor=A.+Martinezauthor=S.+Martin-Fonsecaauthor=A.+Garcia-Granadosauthor=R.+M.+Ferrer-Mart%C3%ADnauthor=J.+A.+Lupia%C3%B1ezauthor=A.+Parra&title=Semi-synthesis+and+antiproliferative+evaluation+of+PEGylated+pentacyclic+triterpenes&doi=10.1016%2Fj.ejmech.2016.04.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Semi-synthesis and antiproliferative evaluation of PEGylated pentacyclic triterpenes</span></div><div class="casAuthors">Medina-O'Donnell, Marta; Rivas, Francisco; Reyes-Zurita, Fernando J.; Martinez, Antonio; Martin-Fonseca, Samuel; Garcia-Granados, Andres; Ferrer-Martin, Rosa M.; Lupianez, Jose A.; Parra, Andres</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">64-78</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Several PEGylated derivs. of oleanolic and maslinic acids have been semi-synthesized, attaching one acid-PEG reagent to the hydroxyl group/s at C-2 or C-2/C-3 of the A rings of these natural triterpenes, and also to their corresponding C-28 benzyl derivs.  Several monomeric and dimeric PEGylated compds. have also been produced by linking one diamine-PEG reagent to the carboxyl group at C-28 of the same natural triterpenes and also to their corresponding C-2 or C-2/C-3 acetylated derivs.  The cytotoxic effects of 12 triterpenic PEGylated derivs. in three cancer-cell lines (B16-F10, HT29, and Hep G2) have been assayed.  The best results have been achieved by the PEGylated-amine deriv. of oleanolic acid, with IC50 concns. between 0.22 and 3.78 μM, being between 28- and 963-fold more effective than its natural precursor.  The percentages of apoptosis induction have also been detd. for the five PEGylated derivs. with the lowest cytotoxicity data.  All five compds. showed apoptotic effects on the treated cells, with a total apoptosis rate of 99% in the B16-F10 cells, 80% in the Hep G2 cells, and 51% in the HT29 cells.  We have also studied the changes in the mitochondrial membrane potential (MMP) to elucidate the possible mechanism involved in the apoptotic responses (intrinsic or extrinsic).  Finally, to verify the results found in the cytometry assays, we have used fluorescence microscopy techniques to det. changes in the cell morphol.  These PEGylated derivs. of natural triterpenoids, which can induce apoptosis at very low concns. in different tumor lines, may represent new effective therapeutic drugs against these diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBNU2AmdXdsrVg90H21EOLACvtfcHk0lgg81dm11opiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnslWqtbw%253D&md5=9ff4513f20931132654c5a70a3ebc57a</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2016.04.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2016.04.016%26sid%3Dliteratum%253Aachs%26aulast%3DMedina-O%25E2%2580%2599Donnell%26aufirst%3DM.%26aulast%3DRivas%26aufirst%3DF.%26aulast%3DReyes-Zurita%26aufirst%3DF.%2BJ.%26aulast%3DMartinez%26aufirst%3DA.%26aulast%3DMartin-Fonseca%26aufirst%3DS.%26aulast%3DGarcia-Granados%26aufirst%3DA.%26aulast%3DFerrer-Mart%25C3%25ADn%26aufirst%3DR.%2BM.%26aulast%3DLupia%25C3%25B1ez%26aufirst%3DJ.%2BA.%26aulast%3DParra%26aufirst%3DA.%26atitle%3DSemi-synthesis%2520and%2520antiproliferative%2520evaluation%2520of%2520PEGylated%2520pentacyclic%2520triterpenes%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2016%26volume%3D118%26spage%3D64%26epage%3D78%26doi%3D10.1016%2Fj.ejmech.2016.04.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 5 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Floriane  Gibault</span>, <span class="hlFld-ContribAuthor ">Manon  Sturbaut</span>, <span class="hlFld-ContribAuthor ">Mathilde  Coevoet</span>, <span class="hlFld-ContribAuthor ">Martine  Pugnière</span>, <span class="hlFld-ContribAuthor ">Ashley  Burtscher</span>, <span class="hlFld-ContribAuthor ">Frédéric  Allemand</span>, <span class="hlFld-ContribAuthor ">Patricia  Melnyk</span>, <span class="hlFld-ContribAuthor ">Wanjin  Hong</span>, <span class="hlFld-ContribAuthor ">Brian P.  Rubin</span>, <span class="hlFld-ContribAuthor ">Ajaybabu V.  Pobbati</span>, <span class="hlFld-ContribAuthor ">Jean‐François  Guichou</span>, <span class="hlFld-ContribAuthor ">Philippe  Cotelle</span>, <span class="hlFld-ContribAuthor ">Fabrice  Bailly</span>. </span><span class="cited-content_cbyCitation_article-title">Design, Synthesis and Evaluation of a Series of 1,5‐Diaryl‐1,2,3‐triazole‐4‐carbohydrazones as Inhibitors of the YAP‐TAZ/TEAD Complex. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2021,</strong> <em>40 </em><a href="https://doi.org/10.1002/cmdc.202100153" title="DOI URL">https://doi.org/10.1002/cmdc.202100153</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.202100153&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.202100153%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DDesign%25252C%252BSynthesis%252Band%252BEvaluation%252Bof%252Ba%252BSeries%252Bof%252B1%25252C5%2525E2%252580%252590Diaryl%2525E2%252580%2525901%25252C2%25252C3%2525E2%252580%252590triazole%2525E2%252580%2525904%2525E2%252580%252590carbohydrazones%252Bas%252BInhibitors%252Bof%252Bthe%252BYAP%2525E2%252580%252590TAZ%25252FTEAD%252BComplex%26aulast%3DGibault%26aufirst%3DFloriane%26date%3D2021%26date%3D2021%26volume%3D40" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chee Wai  Fhu</span>, <span class="hlFld-ContribAuthor ">Azhar  Ali</span>. </span><span class="cited-content_cbyCitation_article-title">Protein Lipidation by Palmitoylation and Myristoylation in Cancer. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Cell and Developmental Biology</span><span> <strong>2021,</strong> <em>9 </em><a href="https://doi.org/10.3389/fcell.2021.673647" title="DOI URL">https://doi.org/10.3389/fcell.2021.673647</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fcell.2021.673647&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffcell.2021.673647%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Cell%2520and%2520Developmental%2520Biology%26atitle%3DProtein%252BLipidation%252Bby%252BPalmitoylation%252Band%252BMyristoylation%252Bin%252BCancer%26aulast%3DFhu%26aufirst%3DChee%2BWai%26date%3D2021%26date%3D2021%26volume%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tian  Lu</span>, <span class="hlFld-ContribAuthor ">Yong  Li</span>, <span class="hlFld-ContribAuthor ">Wenchao  Lu</span>, <span class="hlFld-ContribAuthor ">T.W.G.M.  Spitters</span>, <span class="hlFld-ContribAuthor ">Xueyu  Fang</span>, <span class="hlFld-ContribAuthor ">Jun  Wang</span>, <span class="hlFld-ContribAuthor ">Simian  Cai</span>, <span class="hlFld-ContribAuthor ">Jing  Gao</span>, <span class="hlFld-ContribAuthor ">Yanting  Zhou</span>, <span class="hlFld-ContribAuthor ">Zhe  Duan</span>, <span class="hlFld-ContribAuthor ">Huan  Xiong</span>, <span class="hlFld-ContribAuthor ">Liping  Liu</span>, <span class="hlFld-ContribAuthor ">Qi  Li</span>, <span class="hlFld-ContribAuthor ">Hualiang  Jiang</span>, <span class="hlFld-ContribAuthor ">Kaixian  Chen</span>, <span class="hlFld-ContribAuthor ">Hu  Zhou</span>, <span class="hlFld-ContribAuthor ">Hua  Lin</span>, <span class="hlFld-ContribAuthor ">Huijin  Feng</span>, <span class="hlFld-ContribAuthor ">Bing  Zhou</span>, <span class="hlFld-ContribAuthor ">Christopher L.  Antos</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a subtype-selective, covalent inhibitor against palmitoylation pocket of TEAD3. </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmaceutica Sinica B</span><span> <strong>2021,</strong> <em>11 </em><a href="https://doi.org/10.1016/j.apsb.2021.04.015" title="DOI URL">https://doi.org/10.1016/j.apsb.2021.04.015</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.apsb.2021.04.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.apsb.2021.04.015%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmaceutica%2520Sinica%2520B%26atitle%3DDiscovery%252Bof%252Ba%252Bsubtype-selective%25252C%252Bcovalent%252Binhibitor%252Bagainst%252Bpalmitoylation%252Bpocket%252Bof%252BTEAD3%26aulast%3DLu%26aufirst%3DTian%26date%3D2021%26volume%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sofiia  Karchugina</span>, <span class="hlFld-ContribAuthor ">Dorothy  Benton</span>, <span class="hlFld-ContribAuthor ">Jonathan  Chernoff</span>. </span><span class="cited-content_cbyCitation_article-title">Regulation of MST complexes and activity via SARAH domain modifications. </span><span class="cited-content_cbyCitation_journal-name">Biochemical Society Transactions</span><span> <strong>2021,</strong> <em>49 </em>
                                    (2)
                                     , 675-683. <a href="https://doi.org/10.1042/BST20200559" title="DOI URL">https://doi.org/10.1042/BST20200559</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1042/BST20200559&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1042%2FBST20200559%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Society%2520Transactions%26atitle%3DRegulation%252Bof%252BMST%252Bcomplexes%252Band%252Bactivity%252Bvia%252BSARAH%252Bdomain%252Bmodifications%26aulast%3DKarchugina%26aufirst%3DSofiia%26date%3D2021%26date%3D2021%26volume%3D49%26issue%3D2%26spage%3D675%26epage%3D683" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ajaybabu V.  Pobbati</span>, <span class="hlFld-ContribAuthor ">Brian P.  Rubin</span>. </span><span class="cited-content_cbyCitation_article-title">Protein-Protein Interaction Disruptors of the YAP/TAZ-TEAD Transcriptional Complex. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2020,</strong> <em>25 </em>
                                    (24)
                                     , 6001. <a href="https://doi.org/10.3390/molecules25246001" title="DOI URL">https://doi.org/10.3390/molecules25246001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules25246001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules25246001%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DProtein-Protein%252BInteraction%252BDisruptors%252Bof%252Bthe%252BYAP%25252FTAZ-TEAD%252BTranscriptional%252BComplex%26aulast%3DPobbati%26aufirst%3DAjaybabu%2BV.%26date%3D2020%26date%3D2020%26volume%3D25%26issue%3D24%26spage%3D6001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/medium/jm0c01275_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/large/jm0c01275_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01275&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/medium/jm0c01275_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/large/jm0c01275_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. hTEAD4 transactivation domain. (A) Structure of the TEAD transactivation domain (cyan) and the acylated cysteine residue in the central pocket (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5OAQ">5OAQ</a>). Cofactor YAP (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3KYS">3KYS</a>) is shown (pale yellow). The right panel is zoomed into the acylated cysteine residue (Cys367). (B) Small molecules that bind the TEAD central pocket.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/large/jm0c01275_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01275&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/medium/jm0c01275_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/large/jm0c01275_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Targeting the central pocket with small-molecule inhibitors. (A) Thiol conjugation assay principle. The free cysteine residue in the TEAD central pocket reacts with CPM yielding a fluorescence signal. (B) Kinetic measurement of hTEAD4 (nonacylated) treatment with CPM (purple square) yields a fluorescence signal, while CPM alone (black square) has only a minimal signal. (C) Competitive inhibition of hTEAD4 (nonacylated) CPM with known noncovalent central pocket binders. (D) Selected screening hits (<b>10–12</b>) and inactive analogues (<b>13–15</b>) with similar structures.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/large/jm0c01275_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01275&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/medium/jm0c01275_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/large/jm0c01275_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Kojic Acid Analogues<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/large/jm0c01275_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01275&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) EtOH, rt, 2–20 days. (b) MeI, K<sub>2</sub>CO<sub>3</sub>, DMF, 24 h, rt.</p></p></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/medium/jm0c01275_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/large/jm0c01275_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Target engagement studies with compound <b>19</b> and its inactive analogue <b>45</b> in cell-free systems. (A) Structures of the compounds <b>19</b> and <b>45</b>. Representative inhibition curves in the (B) thiol conjugation assay and (C) FP-based competitive binding assay. (D) Differential scanning fluorimetry measurements of hTEAD4 (nonacylated) with selected derivatives.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/large/jm0c01275_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01275&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/medium/jm0c01275_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/large/jm0c01275_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Mechanism of action studies with kojic acid analogues. (A) Hypothesized mechanism of action of compound <b>19</b>. Whole-protein ESI–MS analysis of hTEAD4 (nonacylated) treatment with kojic acid analogue (B) compound <b>19</b> and (C) compound <b>45</b>. Compound <b>19</b> is a covalent binder for nonacylated hTEAD4, and the −OH group at the second position is required for the predicted reaction with a thiol group. The −OMe analogue (<b>45</b>) is not a covalent TEAD binder. (D) Whole-protein ESI–MS analysis of hTEAD4 (acylated) incubation with <b>19</b>. No protein adduct with compound <b>19</b> was observed when Cys367 in hTEAD4 is lipidated. (E) Comparison of thiol conjugation inhibition with 10 μM compound <b>19</b> in the presence of hTEAD4 (nonacylated), hTEAD4 (acylated), or Cys to Ala/Ser mutants of mTEAD4. When the cysteine residue is replaced with alanine or serine, compound <b>19</b> cannot inhibit the thiol conjugation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/large/jm0c01275_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01275&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/medium/jm0c01275_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/large/jm0c01275_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Inhibition of TEAD–YAP interaction as determined with an FP-based competitive binding assay using FITC-labeled hYAP<sup>50–100</sup>. Compound <b>19</b> (blue triangle) and compound <b>45</b> (black square) were tested. (A) Compound <b>19</b> can prevent YAP interaction with hTEAD4 (nonacylated) but not with (B) hTEAD4 (acylated), mTEAD4 with (C) Cys360Ser, or (D) Cys360Ala mutation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/large/jm0c01275_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01275&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/medium/jm0c01275_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/large/jm0c01275_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. TEAD inhibitory activity of <b>19</b> in a cellular environment. (A,B) Target engagement in HCT116 cell lysates. (A) Representative immunoblots of hTEAD4 treated with a vehicle control (DMSO) or compound <b>19</b>. (B) CETSA melting curves of hTEAD4 treated with a vehicle control (DMSO) or compound <b>19</b>. (C) Inhibition of the TEAD target gene ANKRD1 expression by compound <b>19</b> in comparison with the control compounds <b>45</b> and TED-347.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01275/20201019/images/large/jm0c01275_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01275&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i82">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02675" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02675" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 47 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Holden, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, C. N.</span></span> <span> </span><span class="NLM_article-title">Targeting the hippo pathway and cancer through the TEAD family of transcription factors</span>. <i>Cancers</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">81</span>, <span class="refDoi"> DOI: 10.3390/cancers10030081</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.3390%2Fcancers10030081" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtV2htLfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=81&author=J.+K.+Holdenauthor=C.+N.+Cunningham&title=Targeting+the+hippo+pathway+and+cancer+through+the+TEAD+family+of+transcription+factors&doi=10.3390%2Fcancers10030081"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the hippo pathway and cancer through the TEAD family of transcription factors</span></div><div class="casAuthors">Holden, Jeffrey K.; Cunningham, Christian N.</div><div class="citationInfo"><span class="NLM_cas:title">Cancers</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">81/1-81/15</span>CODEN:
                <span class="NLM_cas:coden">CANCCT</span>;
        ISSN:<span class="NLM_cas:issn">2072-6694</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  The Hippo pathway is a crit. transcriptional signaling pathway that regulates cell growth, proliferation and organ development.  The transcriptional enhanced assoc. domain (TEAD) protein family consists of four paralogous transcription factors that function to modulate gene expression in response to the Hippo signaling pathway.  Transcriptional activation of these proteins occurs upon binding to the co-activator YAP/TAZ whose entry into the nucleus is regulated by Lats1/2 kinase.  In recent years, it has become apparent that the dysregulation and/or overexpression of Hippo pathway effectors is implicated in a wide range of cancers, including prostate, gastric and liver cancer.  A large body of work has been dedicated to understanding the therapeutic potential of modulating the phosphorylation and localization of YAP/TAZ.  However, YAP/TAZ are considered to be natively unfolded and may be intractable as drug targets.  Therefore, TEAD proteins present themselves as an excellent therapeutic target for intervention of the Hippo pathway.  This review summarizes the functional role of TEAD proteins in cancer and assesses the therapeutic potential of antagonizing TEAD function in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFz0v8zUhnTrVg90H21EOLACvtfcHk0ljHMAukswGP1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtV2htLfJ&md5=793c03200885c0349d40a1d92460b652</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.3390%2Fcancers10030081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcancers10030081%26sid%3Dliteratum%253Aachs%26aulast%3DHolden%26aufirst%3DJ.%2BK.%26aulast%3DCunningham%26aufirst%3DC.%2BN.%26atitle%3DTargeting%2520the%2520hippo%2520pathway%2520and%2520cancer%2520through%2520the%2520TEAD%2520family%2520of%2520transcription%2520factors%26jtitle%3DCancers%26date%3D2018%26volume%3D10%26spage%3D81%26doi%3D10.3390%2Fcancers10030081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huh, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeong, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H.</span></span> <span> </span><span class="NLM_article-title">Regulation of TEAD transcription factors in cancer biology</span>. <i>Cells</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">600</span>, <span class="refDoi"> DOI: 10.3390/cells8060600</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.3390%2Fcells8060600" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjsFalsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2019&pages=600&author=H.+Huhauthor=D.+Kimauthor=H.-S.+Jeongauthor=H.+Park&title=Regulation+of+TEAD+transcription+factors+in+cancer+biology&doi=10.3390%2Fcells8060600"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of TEAD transcription factors in cancer biology</span></div><div class="casAuthors">Huh, Hyunbin D.; Kim, Dong Hyeon; Jeong, Han-Sol; Park, Hyun Woo</div><div class="citationInfo"><span class="NLM_cas:title">Cells</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">600</span>CODEN:
                <span class="NLM_cas:coden">CELLC6</span>;
        ISSN:<span class="NLM_cas:issn">2073-4409</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Transcriptional enhanced assoc. domain (TEAD) transcription factors play important roles during development, cell proliferation, regeneration, and tissue homeostasis.  TEAD integrates with and coordinates various signal transduction pathways including Hippo, Wnt, transforming growth factor beta (TGFβ), and epidermal growth factor receptor (EGFR) pathways.  TEAD deregulation affects well-established cancer genes such as KRAS, BRAF, LKB1, NF2, and MYC, and its transcriptional output plays an important role in tumor progression, metastasis, cancer metab., immunity, and drug resistance.  To date, TEADs have been recognized to be key transcription factors of the Hippo pathway.  Therefore, most studies are focused on the Hippo kinases and YAP/TAZ, whereas the Hippo-dependent and Hippo-independent regulators and regulations governing TEAD only emerged recently.  Deregulation of the TEAD transcriptional output plays important roles in tumor progression and serves as a prognostic biomarker due to high correlation with clinicopathol. parameters in human malignancies.  In addn., discovering the mol. mechanisms of TEAD, such as post-translational modifications and nucleocytoplasmic shuttling, represents an important means of modulating TEAD transcriptional activity.  Collectively, this review highlights the role of TEAD in multistep-tumorigenesis by interacting with upstream oncogenic signaling pathways and controlling downstream target genes, which provides unprecedented insight and rationale into developing TEAD-targeted anticancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqV83sN1ayn2LVg90H21EOLACvtfcHk0ljHMAukswGP1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjsFalsrk%253D&md5=d5c87c97ba762881edc0dd4e354ad334</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.3390%2Fcells8060600&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcells8060600%26sid%3Dliteratum%253Aachs%26aulast%3DHuh%26aufirst%3DH.%26aulast%3DKim%26aufirst%3DD.%26aulast%3DJeong%26aufirst%3DH.-S.%26aulast%3DPark%26aufirst%3DH.%26atitle%3DRegulation%2520of%2520TEAD%2520transcription%2520factors%2520in%2520cancer%2520biology%26jtitle%3DCells%26date%3D2019%26volume%3D8%26spage%3D600%26doi%3D10.3390%2Fcells8060600" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, K.-L.</span></span> <span> </span><span class="NLM_article-title">Regulation of the hippo pathway transcription factor TEAD</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">862</span>– <span class="NLM_lpage">872</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2017.09.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1016%2Fj.tibs.2017.09.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=28964625" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFeqtr3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2017&pages=862-872&author=K.+C.+Linauthor=H.+W.+Parkauthor=K.-L.+Guan&title=Regulation+of+the+hippo+pathway+transcription+factor+TEAD&doi=10.1016%2Fj.tibs.2017.09.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of the Hippo Pathway Transcription Factor TEAD</span></div><div class="casAuthors">Lin, Kimberly C.; Park, Hyun Woo; Guan, Kun-Liang</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">862-872</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The TEAD transcription factor family is best known for transcriptional output of the Hippo signaling pathway and has been implicated in processes such as development, cell growth and proliferation, tissue homeostasis, and regeneration.  Our understanding of the functional importance of TEADs has increased dramatically since its initial discovery three decades ago.  The majority of our knowledge of TEADs is in the context of Hippo signaling as nuclear DNA-binding proteins passively activated by Yes-assocd. protein (YAP) and transcriptional activator with PDZ-binding domain (TAZ), transcription coactivators downstream of the Hippo pathway.  However, recent studies suggest that TEAD itself is actively regulated.  Here, we highlight evidence demonstrating Hippo-independent regulation of TEADs and the potential impacts these studies may have on new cancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4QrFEsHUdgbVg90H21EOLACvtfcHk0ljHMAukswGP1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFeqtr3J&md5=0fa282492aa43a0a513f557037adee52</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2017.09.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2017.09.003%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DK.%2BC.%26aulast%3DPark%26aufirst%3DH.%2BW.%26aulast%3DGuan%26aufirst%3DK.-L.%26atitle%3DRegulation%2520of%2520the%2520hippo%2520pathway%2520transcription%2520factor%2520TEAD%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2017%26volume%3D42%26spage%3D862%26epage%3D872%26doi%3D10.1016%2Fj.tibs.2017.09.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">To, K. F.</span></span> <span> </span><span class="NLM_article-title">The TEAD family and its oncogenic role in promoting tumorigenesis</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">138</span>, <span class="refDoi"> DOI: 10.3390/ijms17010138</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.3390%2Fijms17010138" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=26805820" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC2sXktFOlsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=138&author=Y.+Zhouauthor=T.+Huangauthor=A.+S.+Chengauthor=J.+Yuauthor=W.+Kangauthor=K.+F.+To&title=The+TEAD+family+and+its+oncogenic+role+in+promoting+tumorigenesis&doi=10.3390%2Fijms17010138"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The TEAD family and its oncogenic role in promoting tumorigenesis</span></div><div class="casAuthors">Zhou, Yuhang; Huang, Tingting; Cheng, Alfred S. L.; Yu, Jun; Kang, Wei; To, Ka Fai</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">138/1-138/15</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">The TEAD family of transcription factors is necessary for developmental processes.  The family members contain a TEA domain for the binding with DNA elements and a transactivation domain for the interaction with transcription coactivators.  TEAD proteins are required for the participation of coactivators to transmit the signal of pathways for the downstream signaling processes.  TEADs also play an important role in tumor initiation and facilitate cancer progression via activating a series of progression-inducing genes, such as CTGF, Cyr61, Myc and Gli2.  Recent studies have highlighted that TEADs, together with their coactivators, promote or even act as the crucial parts in the development of various malignancies, such as liver, ovarian, breast and prostate cancers.  Furthermore, TEADs are proposed to be useful prognostic biomarkers due to the ideal correlation between high expression and clinicopathol. parameters in gastric, breast, ovarian and prostate cancers.  In this review, we summarize the functional role of TEAD proteins in tumorigenesis and discuss the key role of TEAD transcription factors in the linking of signal cascade transductions.  Improved knowledge of the TEAD proteins will be helpful for deep understanding of the mol. mechanisms of tumorigenesis and identifying ideal predictive or prognostic biomarkers, even providing clin. translation for anticancer therapy in human cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWR3IdQcw6YLVg90H21EOLACvtfcHk0lhoFVxw6pVa-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXktFOlsw%253D%253D&md5=3ca058d5405586b1397cc1e0c5b70b79</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.3390%2Fijms17010138&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms17010138%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DT.%26aulast%3DCheng%26aufirst%3DA.%2BS.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DKang%26aufirst%3DW.%26aulast%3DTo%26aufirst%3DK.%2BF.%26atitle%3DThe%2520TEAD%2520family%2520and%2520its%2520oncogenic%2520role%2520in%2520promoting%2520tumorigenesis%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2016%26volume%3D17%26spage%3D138%26doi%3D10.3390%2Fijms17010138" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Knight, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shepherd, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizzo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brewer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jhavar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eeles, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falconer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovacs, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garrett, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norman, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shipley, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudson, D. L.</span></span> <span> </span><span class="NLM_article-title">TEAD1 and c-Cbl are novel prostate basal cell markers that correlate with poor clinical outcome in prostate cancer</span>. <i>Br. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">1849</span>– <span class="NLM_lpage">1858</span>, <span class="refDoi"> DOI: 10.1038/sj.bjc.6604774</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1038%2Fsj.bjc.6604774" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=19002168" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVajtbrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2008&pages=1849-1858&author=J.+F.+Knightauthor=C.+J.+Shepherdauthor=S.+Rizzoauthor=D.+Brewerauthor=S.+Jhavarauthor=A.+R.+Dodsonauthor=C.+S.+Cooperauthor=R.+Eelesauthor=A.+Falconerauthor=G.+Kovacsauthor=M.+D.+Garrettauthor=A.+R.+Normanauthor=J.+Shipleyauthor=D.+L.+Hudson&title=TEAD1+and+c-Cbl+are+novel+prostate+basal+cell+markers+that+correlate+with+poor+clinical+outcome+in+prostate+cancer&doi=10.1038%2Fsj.bjc.6604774"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">TEAD1 and c-Cbl are novel prostate basal cell markers that correlate with poor clinical outcome in prostate cancer</span></div><div class="casAuthors">Knight, J. F.; Shepherd, C. J.; Rizzo, S.; Brewer, D.; Jhavar, S.; Dodson, A. R.; Cooper, C. S.; Eeles, R.; Falconer, A.; Kovacs, G.; Garrett, M. D.; Norman, A. R.; Shipley, J.; Hudson, D. L.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Cancer</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1849-1858</span>CODEN:
                <span class="NLM_cas:coden">BJCAAI</span>;
        ISSN:<span class="NLM_cas:issn">0007-0920</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Prostate cancer is the most frequently diagnosed male cancer, and its clin. outcome is difficult to predict.  The disease may involve the inappropriate expression of genes that normally control the proliferation of epithelial cells in the basal layer and their differentiation into luminal cells.  Our aim was to identify novel basal cell markers and assess their prognostic and functional significance in prostate cancer.  RNA from basal and luminal cells isolated from benign tissue by immunoguided laser-capture microdissection was subjected to expression profiling.  We identified 112 and 267 genes defining basal and luminal populations, resp.  The transcription factor TEAD1 and the ubiquitin ligase c-Cbl were identified as novel basal cell markers.  Knockdown of either marker using siRNA in prostate cell lines led to decreased cell growth in PC3 and disrupted acinar formation in a 3D culture system of RWPE1.  Analyses of prostate cancer tissue microarray staining established that increased protein levels of either marker were assocd. with decreased patient survival independent of other clinicopathol. metrics.  These data are consistent with basal features impacting on the development and clin. course of prostate cancers.  British Journal of Cancer (2008) 99, 1849-1858. doi:10.1038/sj.bjc.6604774 www.bjcancer.com Published online 11 Nov. 2008.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3D_XIMg8w87Vg90H21EOLACvtfcHk0lhoFVxw6pVa-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVajtbrO&md5=f622b94acce5d78cacdae3225d0d26c7</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjc.6604774&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjc.6604774%26sid%3Dliteratum%253Aachs%26aulast%3DKnight%26aufirst%3DJ.%2BF.%26aulast%3DShepherd%26aufirst%3DC.%2BJ.%26aulast%3DRizzo%26aufirst%3DS.%26aulast%3DBrewer%26aufirst%3DD.%26aulast%3DJhavar%26aufirst%3DS.%26aulast%3DDodson%26aufirst%3DA.%2BR.%26aulast%3DCooper%26aufirst%3DC.%2BS.%26aulast%3DEeles%26aufirst%3DR.%26aulast%3DFalconer%26aufirst%3DA.%26aulast%3DKovacs%26aufirst%3DG.%26aulast%3DGarrett%26aufirst%3DM.%2BD.%26aulast%3DNorman%26aufirst%3DA.%2BR.%26aulast%3DShipley%26aufirst%3DJ.%26aulast%3DHudson%26aufirst%3DD.%2BL.%26atitle%3DTEAD1%2520and%2520c-Cbl%2520are%2520novel%2520prostate%2520basal%2520cell%2520markers%2520that%2520correlate%2520with%2520poor%2520clinical%2520outcome%2520in%2520prostate%2520cancer%26jtitle%3DBr.%2520J.%2520Cancer%26date%3D2008%26volume%3D99%26spage%3D1849%26epage%3D1858%26doi%3D10.1038%2Fsj.bjc.6604774" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span> <span> </span><span class="NLM_article-title">TEAD4 exerts pro-metastatic effects and is negatively regulated by miR6839-3p in lung adenocarcinoma progression</span>. <i>J. Cell. Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">3560</span>– <span class="NLM_lpage">3571</span>, <span class="refDoi"> DOI: 10.1111/jcmm.13634</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1111%2Fjcmm.13634" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=29667772" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtF2mtbbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2018&pages=3560-3571&author=Q.+Zhangauthor=H.+Fanauthor=Q.+Zouauthor=H.+Liuauthor=B.+Wanauthor=S.+Zhuauthor=Y.+Huauthor=H.+Liauthor=C.+Zhangauthor=L.+Zhouauthor=Q.+Zhuauthor=K.+Xiaoauthor=J.+Zhangauthor=P.+Zhanauthor=T.+Lvauthor=Y.+Song&title=TEAD4+exerts+pro-metastatic+effects+and+is+negatively+regulated+by+miR6839-3p+in+lung+adenocarcinoma+progression&doi=10.1111%2Fjcmm.13634"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">TEAD4 exerts pro-metastatic effects and is negatively regulated by miR6839-3p in lung adenocarcinoma progression</span></div><div class="casAuthors">Zhang, Qun; Fan, Hang; Zou, Qian; Liu, Hongda; Wan, Bing; Zhu, Suhua; Hu, Yangbo; Li, Huijuan; Zhang, ChenXi; Zhou, Li; Zhu, Qingqing; Xiao, Kunhong; Zhang, Jianya; Zhan, Ping; Lv, Tangfeng; Song, Yong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cellular and Molecular Medicine</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3560-3571</span>CODEN:
                <span class="NLM_cas:coden">JCMMC9</span>;
        ISSN:<span class="NLM_cas:issn">1582-4934</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Several studies have shown the tumorigenesis role of transcriptional enhancer assoc. domain (TEAD) proteins; here, we initially explored expression, function and signalling mechanisms of TEAD4 in lung adenocarcinoma (LAD).  Both the mRNA and protein levels of TEAD4 were increased in LAD tissues than those in adjacent nontumorous tissues.  Besides, database search indicated a poorer clin. outcome in LAD patients with higher TEAD4 expression, revealing its potential tumor-promoting role.  Therefore, we conducted cellular expts. to further investigate its effect on tumor phenotypes.  Accordingly, TEAD4 showed little impact on LAD cell cycle, proliferation, or apoptosis.  However, silencing TEAD4 remarkably attenuated cell migration and invasion capacities.  Consistently, several important epithelial-mesenchymal transition (EMT) markers such as E-cadherin and Slug were consequently altered by silencing TEAD4.  Furthermore, we identified a novel TEAD4-targeted microRNA, namely miR6839-3p, and confirmed its function in suppressing TEAD4 expression.  Finally, the impact of overexpressing miR6839-3p mimics on LAD progression was validated, which showed a similar pattern with TEAD4 knockdown cells.  Taken together, our data not only revealed the significant role of TEAD4 in promoting LAD progression and predicting clin. outcome but also distinguished miR6839-3p mimics as a promising therapeutic direction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEe22Yu3PQerVg90H21EOLACvtfcHk0lhoFVxw6pVa-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtF2mtbbJ&md5=d63af6f230a14d1f70af18b7ea985b82</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1111%2Fjcmm.13634&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fjcmm.13634%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DZou%26aufirst%3DQ.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DWan%26aufirst%3DB.%26aulast%3DZhu%26aufirst%3DS.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DZhu%26aufirst%3DQ.%26aulast%3DXiao%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DZhan%26aufirst%3DP.%26aulast%3DLv%26aufirst%3DT.%26aulast%3DSong%26aufirst%3DY.%26atitle%3DTEAD4%2520exerts%2520pro-metastatic%2520effects%2520and%2520is%2520negatively%2520regulated%2520by%2520miR6839-3p%2520in%2520lung%2520adenocarcinoma%2520progression%26jtitle%3DJ.%2520Cell.%2520Mol.%2520Med.%26date%3D2018%26volume%3D22%26spage%3D3560%26epage%3D3571%26doi%3D10.1111%2Fjcmm.13634" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mei, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, L.</span></span> <span> </span><span class="NLM_article-title">Increased TEAD4 expression and nuclear localization in colorectal cancer promote epithelial-mesenchymal transition and metastasis in a YAP-independent manner</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">2789</span>– <span class="NLM_lpage">2800</span>, <span class="refDoi"> DOI: 10.1038/onc.2015.342</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1038%2Fonc.2015.342" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=26387538" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFers7bJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2016&pages=2789-2800&author=Y.+Liuauthor=G.+Wangauthor=Y.+Yangauthor=Z.+Meiauthor=Z.+Liangauthor=A.+Cuiauthor=T.+Wuauthor=C.-Y.+Liuauthor=L.+Cui&title=Increased+TEAD4+expression+and+nuclear+localization+in+colorectal+cancer+promote+epithelial-mesenchymal+transition+and+metastasis+in+a+YAP-independent+manner&doi=10.1038%2Fonc.2015.342"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Increased TEAD4 expression and nuclear localization in colorectal cancer promote epithelial-mesenchymal transition and metastasis in a YAP-independent manner</span></div><div class="casAuthors">Liu, Y.; Wang, G.; Yang, Y.; Mei, Z.; Liang, Z.; Cui, A.; Wu, T.; Liu, C.-Y.; Cui, L.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">2789-2800</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Dysregulation of the Hippo pathway occurs in a variety of cancers and often correlates with a poor prognosis.  To further explore the potential role of Hippo pathway dysregulation in tumor development and progression, we investigated its downstream transcription factor TEAD4 in colorectal cancer (CRC).  Increased expression and nuclear localization of TEAD4 were found in a significant portion of CRC tissues, in assocn. with metastasis and a poor prognosis.  In CRC cells, TEAD4 knockdown induced the mesenchymal-epithelial transition and decreased cell mobility in vitro and metastasis in vivo.  Microarray anal. revealed that TEAD4 promoted cell adhesion and upregulated the epithelial-mesenchymal transition-related transcriptome in CRC cells.  Vimentin was identified as a new direct target gene mediating TEAD4 function in CRC cells, whereby forced vimentin expression markedly reversed TEAD4-knockdown-induced cell morphol. changes and decreased mobility.  Interestingly, rescued expression of both WT TEAD4 and a Y429H mutant can reverse the mesenchymal-epithelial transition and increase vimentin expression, cell mobility and metastatic potential in TEAD4-knockdown CRC cells.  The discrepant expression of YAP and TEAD4 in CRC tissues, the rescue ability of TEAD4 mutant defect in YAP binding and no effect on vimentin expression by YAP knockdown in CRC cells, all implicated a YAP-independent manner of TEAD4 function in CRC.  Furthermore, vimentin pos. correlated and CDH1 reversely correlated with the level of TEAD4 in CRC tissues and xenograft tumors.  Our results suggest that TEAD4 nuclear expression can serve as a biomarker for CRC progression and poor prognosis.  The transcription factor TEAD4 regulates a pro-metastasis transcription program in a YAP-independent manner in CRC, thus providing a novel mechanism of TEAD4 transcriptional regulation and its oncogenic role in CRC, independently of the Hippo pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfoIRDoJ1PIrVg90H21EOLACvtfcHk0ljZgxklTqn3VA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFers7bJ&md5=fbb07af174963a5f4ce3b7aeda162aa1</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fonc.2015.342&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2015.342%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DMei%26aufirst%3DZ.%26aulast%3DLiang%26aufirst%3DZ.%26aulast%3DCui%26aufirst%3DA.%26aulast%3DWu%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DC.-Y.%26aulast%3DCui%26aufirst%3DL.%26atitle%3DIncreased%2520TEAD4%2520expression%2520and%2520nuclear%2520localization%2520in%2520colorectal%2520cancer%2520promote%2520epithelial-mesenchymal%2520transition%2520and%2520metastasis%2520in%2520a%2520YAP-independent%2520manner%26jtitle%3DOncogene%26date%3D2016%26volume%3D35%26spage%3D2789%26epage%3D2800%26doi%3D10.1038%2Fonc.2015.342" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, J. H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, W. K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, A. S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">To, K. F.</span></span> <span> </span><span class="NLM_article-title">TEAD1/4 exerts oncogenic role and is negatively regulated by miR-4269 in gastric tumorigenesis</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">6518</span>– <span class="NLM_lpage">6530</span>, <span class="refDoi"> DOI: 10.1038/onc.2017.257</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1038%2Fonc.2017.257" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=28759040" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1CmtbvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2017&pages=6518-6530&author=Y.+Zhouauthor=T.+Huangauthor=J.+Zhangauthor=C.+C.+Wongauthor=B.+Zhangauthor=Y.+Dongauthor=F.+Wuauthor=J.+H.+M.+Tongauthor=W.+K.+K.+Wuauthor=A.+S.+L.+Chengauthor=J.+Yuauthor=W.+Kangauthor=K.+F.+To&title=TEAD1%2F4+exerts+oncogenic+role+and+is+negatively+regulated+by+miR-4269+in+gastric+tumorigenesis&doi=10.1038%2Fonc.2017.257"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">TEAD1/4 exerts oncogenic role and is negatively regulated by miR-4269 in gastric tumorigenesis</span></div><div class="casAuthors">Zhou, Y.; Huang, T.; Zhang, J.; Wong, C. C.; Zhang, B.; Dong, Y.; Wu, F.; Tong, J. H. M.; Wu, W. K. K.; Cheng, A. S. L.; Yu, J.; Kang, W.; To, K. F.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">6518-6530</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">TEA domain (TEAD) transcription factors are key components of the Hippo-YAP1 signaling pathway, but their functional role and regulatory mechanisms remain unclear.  This study aims to comprehensively explore the expression pattern and functional role of TEAD family in gastric carcinogenesis and investigate its regulation by microRNAs (miRNAs).  The mRNA and protein expression of TEAD family were examd. by quant. reverse transcription-PCR (qRT-PCR) and western blot.  Their functional roles were detd. by in vitro and in vivo studies.  The clinicopathol. assocn. of TEAD4 in gastric cancer (GC) was studied using immunohistochem. on tissue microarray.  The prediction of miRNAs, which potentially target TEAD1/4, was performed by TargetScan and miRDB.  The regulation of TEAD1/4 by miRNAs was confirmed by qRT-PCR, western blot and luciferase assays.  TEAD1/4 were overexpressed in GC cell lines and primary GC tissues.  Knockdown of TEAD1/4 induced a significant anticancer effect in vitro and in vivo.  TEAD1 was confirmed to be a direct target of miR-377-3p and miR-4269, while TEAD4 was neg. regulated by miR-1343-3p and miR-4269.  Among them, miR-4269 was the most effective inhibitor of TEAD1/4.  Ectopic expression of these miRNAs substantiated their tumor-suppressive effects.  In primary GC tumors, downregulation of miR-4269 was assocd. with poor disease-specific survival and showed a neg. correlation with TEAD4.  TEAD1 and TEAD4 are oncogenic factors, whose aberrant activation are, in part, mediated by the silence of miR-377-3p, miR-1343-3p and miR-4269.  For the first time, the nuclear accumulated TEAD4 and downregulated miR-4269 are proposed to serve as novel prognostic biomarkers in GC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTACkd06skNrVg90H21EOLACvtfcHk0ljZgxklTqn3VA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1CmtbvN&md5=3a0305563b06178d37487681a23c2bd7</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fonc.2017.257&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2017.257%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DWong%26aufirst%3DC.%2BC.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DDong%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DF.%26aulast%3DTong%26aufirst%3DJ.%2BH.%2BM.%26aulast%3DWu%26aufirst%3DW.%2BK.%2BK.%26aulast%3DCheng%26aufirst%3DA.%2BS.%2BL.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DKang%26aufirst%3DW.%26aulast%3DTo%26aufirst%3DK.%2BF.%26atitle%3DTEAD1%252F4%2520exerts%2520oncogenic%2520role%2520and%2520is%2520negatively%2520regulated%2520by%2520miR-4269%2520in%2520gastric%2520tumorigenesis%26jtitle%3DOncogene%26date%3D2017%26volume%3D36%26spage%3D6518%26epage%3D6530%26doi%3D10.1038%2Fonc.2017.257" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nie, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span> <span> </span><span class="NLM_article-title">The interplay between TEAD4 and KLF5 promotes breast cancer partially through inhibiting the transcription of p27Kip1</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">17685</span>– <span class="NLM_lpage">17697</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.3779</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.18632%2Foncotarget.3779" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=25970772" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A280%3ADC%252BC2MfjsFyqsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=17685-17697&author=C.+Wangauthor=Z.+Nieauthor=Z.+Zhouauthor=H.+Zhangauthor=R.+Liuauthor=J.+Wuauthor=J.+Qinauthor=Y.+Maauthor=L.+Chenauthor=S.+Liauthor=W.+Chenauthor=F.+Liauthor=P.+Shiauthor=Y.+Wuauthor=J.+Shenauthor=C.+Chen&title=The+interplay+between+TEAD4+and+KLF5+promotes+breast+cancer+partially+through+inhibiting+the+transcription+of+p27Kip1&doi=10.18632%2Foncotarget.3779"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The interplay between TEAD4 and KLF5 promotes breast cancer partially through inhibiting the transcription of p27Kip1</span></div><div class="casAuthors">Wang Chunyan; Zhou Zhongmei; Zhang Hailin; Liu Rong; Wu Jing; Qin Junying; Li Fubing; Shi Peiguo; Chen Ceshi; Wang Chunyan; Wu Jing; Qin Junying; Li Fubing; Shi Peiguo; Wang Chunyan; Nie Zhi; Ma Yun; Chen Liang; Li Shumo; Wu Yingying; Shen Jian; Wu Jing; Chen Wenlin</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">17685-97</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Growing evidence suggests that YAP/TAZ are mediators of the Hippo pathway and promote breast cancer.  However, the roles of YAP/TAZ transcription factor partners TEADs in breast cancer remain unclear.  Here we found that TEAD4 was expressed in breast cancer cell lines, especially in triple negative breast cancers (TNBC) cell lines.  TEAD4 binds to KLF5.  Knockdown of either TEAD4 or KLF5 in HCC1937 and HCC1806 cells induced the expression of CDK inhibitor p27.  Depletion of either TEAD4 or KLF5 activated the p27 gene promoter and increased the p27 mRNA levels.  Depletion of p27 partially prevents growth inhibition caused by TEAD4 and KLF5 knockdown.  TEAD4 overexpression stimulated proliferation in vitro and tumor growth in mice, while stable knockdown of TEAD4 inhibited proliferation in vitro and tumor growth in mice.  Thus TEAD4 and KLF5, in collaboration, promoted TNBC cell proliferation and tumor growth in part by inhibiting p27 gene transcription.  TEAD4 is a potential target and biomarker for the development of novel therapeutics for breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRn1i1qso7YZ8yail4HZd-ifW6udTcc2ebkfd1o_2qhxbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MfjsFyqsw%253D%253D&md5=51a793fa622adc55368eab334a233791</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.3779&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.3779%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DC.%26aulast%3DNie%26aufirst%3DZ.%26aulast%3DZhou%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DR.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DQin%26aufirst%3DJ.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DShi%26aufirst%3DP.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DC.%26atitle%3DThe%2520interplay%2520between%2520TEAD4%2520and%2520KLF5%2520promotes%2520breast%2520cancer%2520partially%2520through%2520inhibiting%2520the%2520transcription%2520of%2520p27Kip1%26jtitle%3DOncotarget%26date%3D2015%26volume%3D6%26spage%3D17685%26epage%3D17697%26doi%3D10.18632%2Foncotarget.3779" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, M. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poon, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zender, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luk, J. M.</span></span> <span> </span><span class="NLM_article-title">AXL receptor kinase is a mediator of YAP-dependent oncogenic functions in hepatocellular carcinoma</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">1229</span>– <span class="NLM_lpage">1240</span>, <span class="refDoi"> DOI: 10.1038/onc.2010.504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1038%2Fonc.2010.504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=21076472" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVWmurjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2011&pages=1229-1240&author=M.+Z.+Xuauthor=S.+W.+Chanauthor=A.+M.+Liuauthor=K.+F.+Wongauthor=S.+T.+Fanauthor=J.+Chenauthor=R.+T.+Poonauthor=L.+Zenderauthor=S.+W.+Loweauthor=W.+Hongauthor=J.+M.+Luk&title=AXL+receptor+kinase+is+a+mediator+of+YAP-dependent+oncogenic+functions+in+hepatocellular+carcinoma&doi=10.1038%2Fonc.2010.504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">AXL receptor kinase is a mediator of YAP-dependent oncogenic functions in hepatocellular carcinoma</span></div><div class="casAuthors">Xu, M. Z.; Chan, S. W.; Liu, A. M.; Wong, K. F.; Fan, S. T.; Chen, J.; Poon, R. T.; Zender, L.; Lowe, S. W.; Hong, W.; Luk, J. M.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1229-1240</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Yes-assocd. protein (YAP) is a downstream effector of the Hippo signaling pathway, which controls organ expansion and tissue development.  We have recently defined the tumorigenic potential and clin. significance of the YAP1 oncogene in human hepatocellular carcinoma (HCC).  The present study aims to define the tumorigenic properties of YAP in HCC and elucidate the related downstream signaling mechanism.  In a gain-of-function study, we demonstrated that ectopic increased expression of YAP in the immortalized non-tumorigenic hepatocyte cell line MIHA confers tumorigenic and metastatic potentials, as evidenced by (1) enhanced aptitudes in cell viability, anchorage-independent growth, migration and invasion; (2) tumor formation in a xenograft mouse model; and (3) induction of HCC biomarker α-fetoprotein and activation of mitogen-activated protein kinase.  Furthermore, we have identified AXL, a receptor tyrosine kinase, as a key downstream target that drives YAP-dependent oncogenic functions.  RNAi-mediated knockdown of AXL expression decreased the ability of YAP-expressing MIHA cells and of the primary HCC cell line to proliferate and invade.  These results indicate that AXL is a mediator of YAP-dependent oncogenic activities and implicates it as a potential therapeutic target for HCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhMeq35gO5yrVg90H21EOLACvtfcHk0lheKliQlGHknA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVWmurjJ&md5=7d49124ddec8fafc084a336e55ecb80b</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fonc.2010.504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2010.504%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DM.%2BZ.%26aulast%3DChan%26aufirst%3DS.%2BW.%26aulast%3DLiu%26aufirst%3DA.%2BM.%26aulast%3DWong%26aufirst%3DK.%2BF.%26aulast%3DFan%26aufirst%3DS.%2BT.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DPoon%26aufirst%3DR.%2BT.%26aulast%3DZender%26aufirst%3DL.%26aulast%3DLowe%26aufirst%3DS.%2BW.%26aulast%3DHong%26aufirst%3DW.%26aulast%3DLuk%26aufirst%3DJ.%2BM.%26atitle%3DAXL%2520receptor%2520kinase%2520is%2520a%2520mediator%2520of%2520YAP-dependent%2520oncogenic%2520functions%2520in%2520hepatocellular%2520carcinoma%26jtitle%3DOncogene%26date%3D2011%26volume%3D30%26spage%3D1229%26epage%3D1240%26doi%3D10.1038%2Fonc.2010.504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barreto, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ray, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ag Edgar, P.</span></span> <span> </span><span class="NLM_article-title">Biological characteristics of CCN proteins in tumor development</span>. <i>J. BUON</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1359</span>– <span class="NLM_lpage">1367</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=28039692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A280%3ADC%252BC1c7gvFOgtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=1359-1367&author=S.+C.+Barretoauthor=A.+Rayauthor=P.+Ag+Edgar&title=Biological+characteristics+of+CCN+proteins+in+tumor+development"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Biological characteristics of CCN proteins in tumor development</span></div><div class="casAuthors">Barreto Stephany C; Ray Amitabha; Ag Edgar Page</div><div class="citationInfo"><span class="NLM_cas:title">Journal of B.U.ON. : official journal of the Balkan Union of Oncology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1359-1367</span>
        ISSN:<span class="NLM_cas:issn">1107-0625</span>.
    </div><div class="casAbstract">The extracellular membrane (ECM) is no longer regarded as inert, rather it has multiple versatile physiologic functions.  Its diverse composition is implicated in each step of cancer progression including inflammation, angiogenesis, tumor invasion and metastasis.  In addition to structural proteins, the ECM also contains a family of non-structural proteins called matricellular proteins.  The six secreted CCN proteins, which belong to the matricellular protein family, include the following: Cyr61 (CCN1), CTGF (CCN2), Nov (CCN3), WISP- 1 (CCN4), WISP-2 (CCN5) and WISP-3(CCN6).  These proteins are capable of modulating a variety of biological processes in health as well as in disease conditions.  In tumor development and in tumor microenvironment, CCN proteins can influence multiple facets of pathophysiological processes including cellular proliferation, invasion and metastasis.  This review has attempted a cohesive look at the CCN family protein functions in a tumor-specific manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSroeed7ucn-DdblLE_E9PWfW6udTcc2ebkfd1o_2qhxbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1c7gvFOgtQ%253D%253D&md5=cc10333a7fee86a0a0b0f8d935c32b0c</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBarreto%26aufirst%3DS.%2BC.%26aulast%3DRay%26aufirst%3DA.%26aulast%3DAg%2BEdgar%26aufirst%3DP.%26atitle%3DBiological%2520characteristics%2520of%2520CCN%2520proteins%2520in%2520tumor%2520development%26jtitle%3DJ.%2520BUON%26date%3D2016%26volume%3D21%26spage%3D1359%26epage%3D1367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaiyan, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, Z.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, K.-L.</span></span> <span> </span><span class="NLM_article-title">TEAD mediates YAP-dependent gene induction and growth control</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">1962</span>– <span class="NLM_lpage">1971</span>, <span class="refDoi"> DOI: 10.1101/gad.1664408</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1101%2Fgad.1664408" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=18579750" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BD1cXovFWlsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2008&pages=1962-1971&author=B.+Zhaoauthor=X.+Yeauthor=J.+Yuauthor=L.+Liauthor=W.+Liauthor=S.+Liauthor=J.+Yuauthor=J.+D.+Linauthor=C.-Y.+Wangauthor=A.+M.+Chinnaiyanauthor=Z.-C.+Laiauthor=K.-L.+Guan&title=TEAD+mediates+YAP-dependent+gene+induction+and+growth+control&doi=10.1101%2Fgad.1664408"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">TEAD mediates YAP-dependent gene induction and growth control</span></div><div class="casAuthors">Zhao, Bin; Ye, Xin; Yu, Jindan; Li, Li; Li, Weiquan; Li, Siming; Yu, Jianjun; Lin, Jiandie D.; Wang, Cun-Yu; Chinnaiyan, Arul M.; Lai, Zhi-Chun; Guan, Kun-Liang</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1962-1971</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">The YAP transcription coactivator has been implicated as an oncogene and is amplified in human cancers.  Recent studies have established that YAP is phosphorylated and inhibited by the Hippo tumor suppressor pathway.  Here we demonstrate that the TEAD family transcription factors are essential in mediating YAP-dependent gene expression.  TEAD is also required for YAP-induced cell growth, oncogenic transformation, and epithelial-mesenchymal transition.  CTGF is identified as a direct YAP target gene important for cell growth.  Moreover, the functional relationship between YAP and TEAD is conserved in Drosophila Yki (the YAP homolog) and Scalloped (the TEAD homolog).  Our study reveals TEAD as a new component in the Hippo pathway playing essential roles in mediating biol. functions of YAP.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVbRgjZPRWbbVg90H21EOLACvtfcHk0lic1JpCpFev5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXovFWlsb8%253D&md5=53bf27e76bdb025ba0077004fd8ec968</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1101%2Fgad.1664408&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1664408%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DYe%26aufirst%3DX.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DLin%26aufirst%3DJ.%2BD.%26aulast%3DWang%26aufirst%3DC.-Y.%26aulast%3DChinnaiyan%26aufirst%3DA.%2BM.%26aulast%3DLai%26aufirst%3DZ.-C.%26aulast%3DGuan%26aufirst%3DK.-L.%26atitle%3DTEAD%2520mediates%2520YAP-dependent%2520gene%2520induction%2520and%2520growth%2520control%26jtitle%3DGenes%2520Dev.%26date%3D2008%26volume%3D22%26spage%3D1962%26epage%3D1971%26doi%3D10.1101%2Fgad.1664408" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feldmann, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Comerford, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gayyed, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anders, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maitra, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, D.</span></span> <span> </span><span class="NLM_article-title">Elucidation of a universal size-control mechanism in drosophila and mammals</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>130</i></span>,  <span class="NLM_fpage">1120</span>– <span class="NLM_lpage">1133</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2007.07.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1016%2Fj.cell.2007.07.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=17889654" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFKnurjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2007&pages=1120-1133&author=J.+Dongauthor=G.+Feldmannauthor=J.+Huangauthor=S.+Wuauthor=N.+Zhangauthor=S.+A.+Comerfordauthor=M.+F.+Gayyedauthor=R.+A.+Andersauthor=A.+Maitraauthor=D.+Pan&title=Elucidation+of+a+universal+size-control+mechanism+in+drosophila+and+mammals&doi=10.1016%2Fj.cell.2007.07.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Elucidation of a universal size-control mechanism in Drosophila and mammals</span></div><div class="casAuthors">Dong, Jixin; Feldmann, Georg; Huang, Jianbin; Wu, Shian; Zhang, Nailing; Comerford, Sarah A.; Gayyed, Mariana F.; Anders, Robert A.; Maitra, Anirban; Pan, Duojia</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1120-1133</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Coordination of cell proliferation and cell death is essential to attain proper organ size during development and for maintaining tissue homeostasis throughout postnatal life.  In Drosophila, these two processes are orchestrated by the Hippo kinase cascade, a growth-suppressive pathway that ultimately antagonizes the transcriptional coactivator Yorkie (Yki).  Here we demonstrate that a single phosphorylation site in Yki mediates the growth-suppressive output of the Hippo pathway.  Hippo-mediated phosphorylation inactivates Yki by excluding it from the nucleus, whereas loss of Hippo signaling leads to nuclear accumulation and therefore increased Yki activity.  We further delineate a mammalian Hippo signaling pathway that culminates in the phosphorylation of YAP, the mammalian homolog of Yki.  Using a conditional YAP transgenic mouse model, we demonstrate that the mammalian Hippo pathway is a potent regulator of organ size, and that its dysregulation leads to tumorigenesis.  These results uncover a universal size-control mechanism in metazoan.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFVuE0XSnE87Vg90H21EOLACvtfcHk0lic1JpCpFev5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFKnurjO&md5=c8b5bfffd642be6b1fe2af7d012a1c3e</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.07.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.07.019%26sid%3Dliteratum%253Aachs%26aulast%3DDong%26aufirst%3DJ.%26aulast%3DFeldmann%26aufirst%3DG.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DComerford%26aufirst%3DS.%2BA.%26aulast%3DGayyed%26aufirst%3DM.%2BF.%26aulast%3DAnders%26aufirst%3DR.%2BA.%26aulast%3DMaitra%26aufirst%3DA.%26aulast%3DPan%26aufirst%3DD.%26atitle%3DElucidation%2520of%2520a%2520universal%2520size-control%2520mechanism%2520in%2520drosophila%2520and%2520mammals%26jtitle%3DCell%26date%3D2007%26volume%3D130%26spage%3D1120%26epage%3D1133%26doi%3D10.1016%2Fj.cell.2007.07.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Y. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaerkady, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kern, S. E.</span></span> <span> </span><span class="NLM_article-title">Unbiased discovery of interactions at a control locus driving expression of the cancer-specific therapeutic and diagnostic target, mesothelin</span>. <i>J. Proteome Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">5301</span>– <span class="NLM_lpage">5310</span>, <span class="refDoi"> DOI: 10.1021/pr300797v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/pr300797v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVejsrjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=5301-5310&author=Y.+R.+Renauthor=R.+Chaerkadyauthor=S.+Huauthor=J.+Wanauthor=J.+Qianauthor=H.+Zhuauthor=A.+Pandeyauthor=S.+E.+Kern&title=Unbiased+discovery+of+interactions+at+a+control+locus+driving+expression+of+the+cancer-specific+therapeutic+and+diagnostic+target%2C+mesothelin&doi=10.1021%2Fpr300797v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Unbiased discovery of interactions at a control locus driving expression of the cancer-specific therapeutic and diagnostic target, mesothelin</span></div><div class="casAuthors">Ren, Yunzhao R.; Chaerkady, Raghothama; Hu, Shaohui; Wan, Jun; Qian, Jiang; Zhu, Heng; Pandey, Akhilesh; Kern, Scott E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5301-5310</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Although significant effort is expended on identifying transcripts/proteins that are up-regulated in cancer, there are few reports on systematic elucidation of transcriptional mechanisms underlying such druggable cancer-specific targets.  The mesothelin (MSLN) gene offers a promising subject, being expressed in a restricted pattern normally, yet highly overexpressed in almost one-third of human malignancies and a target of cancer immunotherapeutic trials.  CanScript, a cis promoter element, appears to control MSLN cancer-specific expression; its related genomic sequences may up-regulate other cancer markers.  CanScript is a 20-nt bipartite element consisting of an SP1-like motif and a consensus MCAT sequence.  The latter recruits TEAD (TEA domain) family members, which are universally expressed.  Exploration of the active CanScript element, esp. the proteins binding to the SP1-like motif, thus could reveal cancer-specific features having diagnostic or therapeutic interest.  The efficient identification of sequence-specific DNA-binding proteins at a given locus, however, has lagged in biomarker explorations.  The authors used two orthogonal proteomics approaches-unbiased SILAC (stable isotope labeling by amino acids in cell culture)/DNA affinity-capture/mass spectrometry survey (SD-MS) and a large transcription factor protein microarray (TFM)-and functional validation to explore systematically the CanScript interactome.  SD-MS produced nine candidates, and TFM, 18.  The screens agreed in confirming binding by TEAD proteins and by newly identified NAB1 and NFATc.  Among other identified candidates, the authors found functional roles for ZNF24, NAB1 and RFX1 in MSLN expression by cancer cells.  Combined interactome screens yield an efficient, reproducible, sensitive, and unbiased approach to identify sequence-specific DNA-binding proteins and other participants in disease-specific DNA elements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmDKthGPBrALVg90H21EOLACvtfcHk0lic1JpCpFev5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVejsrjM&md5=0453e4cf7e0b3559a445ae641813ea56</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fpr300797v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fpr300797v%26sid%3Dliteratum%253Aachs%26aulast%3DRen%26aufirst%3DY.%2BR.%26aulast%3DChaerkady%26aufirst%3DR.%26aulast%3DHu%26aufirst%3DS.%26aulast%3DWan%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DPandey%26aufirst%3DA.%26aulast%3DKern%26aufirst%3DS.%2BE.%26atitle%3DUnbiased%2520discovery%2520of%2520interactions%2520at%2520a%2520control%2520locus%2520driving%2520expression%2520of%2520the%2520cancer-specific%2520therapeutic%2520and%2520diagnostic%2520target%252C%2520mesothelin%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2012%26volume%3D11%26spage%3D5301%26epage%3D5310%26doi%3D10.1021%2Fpr300797v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedrich, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soriano, P.</span></span> <span> </span><span class="NLM_article-title">Transcriptional enhancer factor 1 disruption by a retroviral gene trap leads to heart defects and embryonic lethality in mice</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">2293</span>– <span class="NLM_lpage">2301</span>, <span class="refDoi"> DOI: 10.1101/gad.8.19.2293</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1101%2Fgad.8.19.2293" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=7958896" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADyaK2MXhtF2gsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1994&pages=2293-2301&author=Z.+Chenauthor=G.+A.+Friedrichauthor=P.+Soriano&title=Transcriptional+enhancer+factor+1+disruption+by+a+retroviral+gene+trap+leads+to+heart+defects+and+embryonic+lethality+in+mice&doi=10.1101%2Fgad.8.19.2293"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional enhancer factor 1 disruption by a retroviral gene trap leads to heart defects and embryonic lethality in mice</span></div><div class="casAuthors">Chen, Zhi; Friedrich, Glenn A.; Soriano, Philippe</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">2293-301</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    </div><div class="casAbstract">A retroviral gene trap in embryonic stem (ES) cells was used to derive a recessive embryonic lethal mouse strain, ROSAβ-geo5.  Mutant embryos display an enlarged pericardial cavity, brachycardia, a dilated fourth ventricle in the brain, and die between embryonic days 11 and 12.  Whereas heart development in the mutant embryos is extensive, the ventricular wall is abnormally thin with a reduced no. of trabeculae.  Cloning of the trapped gene indicates that proviral insertion creates a null mutation in the transcriptional enhancer factor 1 (TEF-1) gene.  Although transcription of a no. of muscle-specific genes believed to be TEF-1 targets appears normal, the defect in cardiogenesis is likely attributable to diminished transcription of one or several cardiac-specific genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEdQzY-4NhA7Vg90H21EOLACvtfcHk0lg9jNjdFvZVoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXhtF2gsbg%253D&md5=720e70b11d73ceae5560a526fc53ff0c</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1101%2Fgad.8.19.2293&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.8.19.2293%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DFriedrich%26aufirst%3DG.%2BA.%26aulast%3DSoriano%26aufirst%3DP.%26atitle%3DTranscriptional%2520enhancer%2520factor%25201%2520disruption%2520by%2520a%2520retroviral%2520gene%2520trap%2520leads%2520to%2520heart%2520defects%2520and%2520embryonic%2520lethality%2520in%2520mice%26jtitle%3DGenes%2520Dev.%26date%3D1994%26volume%3D8%26spage%3D2293%26epage%3D2301%26doi%3D10.1101%2Fgad.8.19.2293" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaneko, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohn, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DePamphilis, M. L.</span></span> <span> </span><span class="NLM_article-title">Transcription factor TEAD2 is involved in neural tube closure</span>. <i>Genesis</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">577</span>– <span class="NLM_lpage">587</span>, <span class="refDoi"> DOI: 10.1002/dvg.20330</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1002%2Fdvg.20330" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=17868131" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlWjs7nI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2007&pages=577-587&author=K.+J.+Kanekoauthor=M.+J.+Kohnauthor=C.+Liuauthor=M.+L.+DePamphilis&title=Transcription+factor+TEAD2+is+involved+in+neural+tube+closure&doi=10.1002%2Fdvg.20330"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Transcription factor TEAD2 is involved in neural tube closure</span></div><div class="casAuthors">Kaneko, Kotaro J.; Kohn, Matthew J.; Liu, Chengyu; DePamphilis, Melvin L.</div><div class="citationInfo"><span class="NLM_cas:title">Genesis (Hoboken, NJ, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">577-587</span>CODEN:
                <span class="NLM_cas:coden">GNESFY</span>;
        ISSN:<span class="NLM_cas:issn">1526-954X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">TEAD2, one of the first transcription factors expressed at the beginning of mammalian development, appears to be required during neural development.  For example, Tead2 expression is greatest in the dorsal neural crest where it appears to regulate expression of Pax3, a gene essential for brain development.  Consistent with this hypothesis, we found that inactivation of the Tead2 gene in mice significantly increased the risk of exencephaly (a defect in neural tube closure).  However, none of the embryos exhibited spina bifida, the major phenotype of Pax3 nullizygous embryos, and expression of Pax3 in E11.5 Tead2 nullizygous embryos was normal.  Thus, Tead2 plays a role in neural tube closure that is independent of its putative role in Pax3 regulation.  In addn., the risk of exencephaly was greatest with Tead2 nullizygous females, and could be suppressed either by folic acid or pifithrin-α.  These results reveal a maternal genetic contribution to neural tube closure, and suggest that Tead2-deficient mice provide a model for anencephaly, a common human birth defect that can be prevented by folic acid.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDkxCNUlVolrVg90H21EOLACvtfcHk0lj09TN1BN_rfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlWjs7nI&md5=fe699b35849882fc648ccc8c089893c8</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1002%2Fdvg.20330&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fdvg.20330%26sid%3Dliteratum%253Aachs%26aulast%3DKaneko%26aufirst%3DK.%2BJ.%26aulast%3DKohn%26aufirst%3DM.%2BJ.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DDePamphilis%26aufirst%3DM.%2BL.%26atitle%3DTranscription%2520factor%2520TEAD2%2520is%2520involved%2520in%2520neural%2520tube%2520closure%26jtitle%3DGenesis%26date%3D2007%26volume%3D45%26spage%3D577%26epage%3D587%26doi%3D10.1002%2Fdvg.20330" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nishioka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiyonari, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawada, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ota, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, H.</span></span> <span> </span><span class="NLM_article-title">TEAD4 is required for specification of trophectoderm in pre-implantation mouse embryos</span>. <i>Mech. Dev.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">270</span>– <span class="NLM_lpage">283</span>, <span class="refDoi"> DOI: 10.1016/j.mod.2007.11.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1016%2Fj.mod.2007.11.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=18083014" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhslGjt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2008&pages=270-283&author=N.+Nishiokaauthor=S.+Yamamotoauthor=H.+Kiyonariauthor=H.+Satoauthor=A.+Sawadaauthor=M.+Otaauthor=K.+Nakaoauthor=H.+Sasaki&title=TEAD4+is+required+for+specification+of+trophectoderm+in+pre-implantation+mouse+embryos&doi=10.1016%2Fj.mod.2007.11.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Tead4 is required for specification of trophectoderm in preimplantation mouse embryos</span></div><div class="casAuthors">Nishioka, Noriyuki; Yamamoto, Shinji; Kiyonari, Hiroshi; Sato, Hiroko; Sawada, Atsushi; Ota, Mitsunori; Nakao, Kazuki; Sasaki, Hiroshi</div><div class="citationInfo"><span class="NLM_cas:title">Mechanisms of Development</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">3-4</span>),
    <span class="NLM_cas:pages">270-283</span>CODEN:
                <span class="NLM_cas:coden">MEDVE6</span>;
        ISSN:<span class="NLM_cas:issn">0925-4773</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">During preimplantation mouse development, embryos form blastocysts with establishment of the first two cell lineages: the trophectoderm (TE) which gives rise to the placenta, and the inner cell mass (ICM) which will form the embryo proper.  Differentiation of TE is regulated by the transcription factor Caudal-related homeobox 2 (Cdx2), but the mechanisms which act upstream of Cdx2 expression remain unknown.  Here we show that the TEA domain family transcription factor, Tead4, is required for TE development.  Tead1, Tead2 and Tead4 were expressed in preimplantation embryos, and at least Tead1 and Tead4 were expressed widely in both TE and ICM lineages.  Tead4 -/- embryos died at preimplantation stages without forming the blastocoel.  The mutant embryos continued cell proliferation, and adherens junction and cell polarity were not significantly affected.  In Tead4 -/- embryos, Cdx2 was weakly expressed at the morula stage but was not expressed in later stages.  None of the TE specific genes, including Eomes and a Cdx2 independent gene, Fgfr2, was detected in Tead4 -/- embryos.  Instead, the ICM specific transcription factors, Oct3/4 and Nanog, were expressed in all the blastomeres.  Tead4 -/- embryos also failed to differentiate trophoblast giant cells when they were cultured in vitro.  ES cells with normal in vitro differentiation abilities were established from Tead4 -/- embryos.  These results suggest that Tead4 has a distinct role from Tead1 and Tead2 in trophectoderm specification of preimplantation embryos, and that Tead4 is an early transcription factor required for specification and development of the trophectoderm lineage, which includes expression of Cdx2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEAvfctEY52LVg90H21EOLACvtfcHk0lj09TN1BN_rfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhslGjt78%253D&md5=68eba1036c39922c1e6776b49217bf99</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.mod.2007.11.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mod.2007.11.002%26sid%3Dliteratum%253Aachs%26aulast%3DNishioka%26aufirst%3DN.%26aulast%3DYamamoto%26aufirst%3DS.%26aulast%3DKiyonari%26aufirst%3DH.%26aulast%3DSato%26aufirst%3DH.%26aulast%3DSawada%26aufirst%3DA.%26aulast%3DOta%26aufirst%3DM.%26aulast%3DNakao%26aufirst%3DK.%26aulast%3DSasaki%26aufirst%3DH.%26atitle%3DTEAD4%2520is%2520required%2520for%2520specification%2520of%2520trophectoderm%2520in%2520pre-implantation%2520mouse%2520embryos%26jtitle%3DMech.%2520Dev.%26date%3D2008%26volume%3D125%26spage%3D270%26epage%3D283%26doi%3D10.1016%2Fj.mod.2007.11.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yagi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohn, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karavanova, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaneko, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vullhorst, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DePamphilis, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buonanno, A.</span></span> <span> </span><span class="NLM_article-title">Transcription factor TEAD4 specifies the trophectoderm lineage at the beginning of mammalian development</span>. <i>Development</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">3827</span>– <span class="NLM_lpage">3836</span>, <span class="refDoi"> DOI: 10.1242/dev.010223</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1242%2Fdev.010223" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=17913785" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVeku7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2007&pages=3827-3836&author=R.+Yagiauthor=M.+J.+Kohnauthor=I.+Karavanovaauthor=K.+J.+Kanekoauthor=D.+Vullhorstauthor=M.+L.+DePamphilisauthor=A.+Buonanno&title=Transcription+factor+TEAD4+specifies+the+trophectoderm+lineage+at+the+beginning+of+mammalian+development&doi=10.1242%2Fdev.010223"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Transcription factor TEAD4 specifies the trophectoderm lineage at the beginning of mammalian development</span></div><div class="casAuthors">Yagi, Rieko; Kohn, Matthew J.; Karavanova, Irina; Kaneko, Kotaro J.; Vullhorst, Detlef; DePamphilis, Melvin L.; Buonanno, Andres</div><div class="citationInfo"><span class="NLM_cas:title">Development (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">3827-3836</span>CODEN:
                <span class="NLM_cas:coden">DEVPED</span>;
        ISSN:<span class="NLM_cas:issn">0950-1991</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">Specification of cell lineages in mammals begins shortly after fertilization with formation of a blastocyst consisting of trophectoderm, which contributes exclusively to the placenta, and inner cell mass (ICM), from which the embryo develops.  Here we report that ablation of the mouse Tead4 gene results in a preimplantation lethal phenotype, and TEAD4 is one of two highly homologous TEAD transcription factors that are expressed during zygotic gene activation in mouse 2-cell embryos.  Tead4-/- embryos do not express trophectoderm-specific genes, such as Cdx2, but do express ICM-specific genes, such as Oct4 (also known as Pou5f1).  Consequently, Tead4-/- morulae do not produce trophoblast stem cells, trophectoderm or blastocoel cavities, and therefore do not implant into the uterine endometrium.  However, Tead4-/- embryos can produce embryonic stem cells, a deriv. of ICM, and if the Tead4 allele is not disrupted until after implantation, then Tead4-/- embryos complete development.  Thus, Tead4 is the earliest gene shown to be uniquely required for specification of the trophectoderm lineage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5_mWKmWY8KbVg90H21EOLACvtfcHk0lj09TN1BN_rfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVeku7rL&md5=e1fe8a57011737f1c83931c6878c3a59</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1242%2Fdev.010223&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fdev.010223%26sid%3Dliteratum%253Aachs%26aulast%3DYagi%26aufirst%3DR.%26aulast%3DKohn%26aufirst%3DM.%2BJ.%26aulast%3DKaravanova%26aufirst%3DI.%26aulast%3DKaneko%26aufirst%3DK.%2BJ.%26aulast%3DVullhorst%26aufirst%3DD.%26aulast%3DDePamphilis%26aufirst%3DM.%2BL.%26aulast%3DBuonanno%26aufirst%3DA.%26atitle%3DTranscription%2520factor%2520TEAD4%2520specifies%2520the%2520trophectoderm%2520lineage%2520at%2520the%2520beginning%2520of%2520mammalian%2520development%26jtitle%3DDevelopment%26date%3D2007%26volume%3D134%26spage%3D3827%26epage%3D3836%26doi%3D10.1242%2Fdev.010223" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Juan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, W.</span></span> <span> </span><span class="NLM_article-title">Targeting the hippo signaling pathway for tissue regeneration and cancer therapy</span>. <i>Genes</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">55</span>, <span class="refDoi"> DOI: 10.3390/genes7090055</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.3390%2Fgenes7090055" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVWju7nL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=55&author=W.+Juanauthor=W.+Hong&title=Targeting+the+hippo+signaling+pathway+for+tissue+regeneration+and+cancer+therapy&doi=10.3390%2Fgenes7090055"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the Hippo signaling pathway for tissue regeneration and cancer therapy</span></div><div class="casAuthors">Juan, Wen Chun; Hong, Wanjin</div><div class="citationInfo"><span class="NLM_cas:title">Genes</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">55/1-55/25</span>CODEN:
                <span class="NLM_cas:coden">GENEG9</span>;
        ISSN:<span class="NLM_cas:issn">2073-4425</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">The Hippo signaling pathway is a highly-conserved developmental pathway that plays an essential role in organ size control, tumor suppression, tissue regeneration and stem cell self-renewal.  The YES-assocd. protein (YAP) and the transcriptional co-activator with PDZ-binding motif (TAZ) are two important transcriptional co-activators that are neg. regulated by the Hippo signaling pathway.  By binding to transcription factors, esp. the TEA domain transcription factors (TEADs), YAP and TAZ induce the expression of growth-promoting genes, which can promote organ regeneration after injury.  Therefore, controlled activation of YAP and TAZ can be useful for regenerative medicine.  However, aberrant activation of YAP and TAZ due to deregulation of the Hippo pathway or overexpression of YAP/TAZ and TEADs can promote cancer development.  Hence, pharmacol. inhibition of YAP and TAZ may be a useful approach to treat tumors with high YAP and/or TAZ activity.  In this review, we present the mechanisms regulating the Hippo pathway, the role of the Hippo pathway in tissue repair and cancer, as well as a detailed anal. of the different strategies to target the Hippo signaling pathway and the genes regulated by YAP and TAZ for regenerative medicine and cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwVFa4phVsj7Vg90H21EOLACvtfcHk0lj09TN1BN_rfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVWju7nL&md5=19c4e707f22ac7a2c6130e0f4d07eb9e</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.3390%2Fgenes7090055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fgenes7090055%26sid%3Dliteratum%253Aachs%26aulast%3DJuan%26aufirst%3DW.%26aulast%3DHong%26aufirst%3DW.%26atitle%3DTargeting%2520the%2520hippo%2520signaling%2520pathway%2520for%2520tissue%2520regeneration%2520and%2520cancer%2520therapy%26jtitle%3DGenes%26date%3D2016%26volume%3D7%26spage%3D55%26doi%3D10.3390%2Fgenes7090055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Santucci, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vignudelli, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrari, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scalvini, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolognesi, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uliassi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costi, M. P.</span></span> <span> </span><span class="NLM_article-title">The hippo pathway and YAP/TAZ-TEAD protein-protein interaction as targets for regenerative medicine and cancer treatment</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">4857</span>– <span class="NLM_lpage">4873</span>, <span class="refDoi"> DOI: 10.1021/jm501615v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501615v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjsVWktrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=4857-4873&author=M.+Santucciauthor=T.+Vignudelliauthor=S.+Ferrariauthor=M.+Morauthor=L.+Scalviniauthor=M.+L.+Bolognesiauthor=E.+Uliassiauthor=M.+P.+Costi&title=The+hippo+pathway+and+YAP%2FTAZ-TEAD+protein-protein+interaction+as+targets+for+regenerative+medicine+and+cancer+treatment&doi=10.1021%2Fjm501615v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The Hippo Pathway and YAP/TAZ-TEAD Protein-Protein Interaction as Targets for Regenerative Medicine and Cancer Treatment</span></div><div class="casAuthors">Santucci, Matteo; Vignudelli, Tatiana; Ferrari, Stefania; Mor, Marco; Scalvini, Laura; Bolognesi, Maria Laura; Uliassi, Elisa; Costi, Maria Paola</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4857-4873</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The Hippo pathway is an important organ size control signaling network and the major regulatory mechanism of cell-contact inhibition.  Yes assocd. protein (YAP) and transcriptional co-activator with PDZ-binding motif (TAZ) are its targets and terminal effectors: inhibition of the pathway promotes YAP/TAZ translocation to the nucleus, where they interact with transcriptional enhancer assoc. domain (TEAD) transcription factors and coactivate the expression of target genes, promoting cell proliferation.  Defects in the pathway can result in overgrowth phenotypes due to deregulation of stem-cell proliferation and apoptosis; members of the pathway are directly involved in cancer development.  The pharmacol. regulation of the pathway might be useful in cancer prevention, treatment, and regenerative medicine applications; currently, a few compds. can selectively modulate the pathway.  In this review, we present an overview of the Hippo pathway, the sequence and structural anal. of YAP/TAZ, the known pharmacol. modulators of the pathway, esp. those targeting YAP/TAZ-TEAD interaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQkLuQrlwpNrVg90H21EOLACvtfcHk0ljJzqwOogMVPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjsVWktrs%253D&md5=22e4a9cda126edd3739884e8c11c0a24</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm501615v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501615v%26sid%3Dliteratum%253Aachs%26aulast%3DSantucci%26aufirst%3DM.%26aulast%3DVignudelli%26aufirst%3DT.%26aulast%3DFerrari%26aufirst%3DS.%26aulast%3DMor%26aufirst%3DM.%26aulast%3DScalvini%26aufirst%3DL.%26aulast%3DBolognesi%26aufirst%3DM.%2BL.%26aulast%3DUliassi%26aufirst%3DE.%26aulast%3DCosti%26aufirst%3DM.%2BP.%26atitle%3DThe%2520hippo%2520pathway%2520and%2520YAP%252FTAZ-TEAD%2520protein-protein%2520interaction%2520as%2520targets%2520for%2520regenerative%2520medicine%2520and%2520cancer%2520treatment%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D4857%26epage%3D4873%26doi%3D10.1021%2Fjm501615v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gibault, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coevoet, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sturbaut, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farce, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renault, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allemand, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guichou, J.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drucbert, A.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foulon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thuru, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corvaisier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huet, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chavatte, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnyk, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailly, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cotelle, P.</span></span> <span> </span><span class="NLM_article-title">Toward the discovery of a novel class of YAP(-)TEAD interaction inhibitors by virtual screening approach targeting YAP(-)TEAD protein(-)protein interface</span>. <i>Cancers</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">140</span>, <span class="refDoi"> DOI: 10.3390/cancers10050140</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.3390%2Fcancers10050140" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVemsbzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=140&author=F.+Gibaultauthor=M.+Coevoetauthor=M.+Sturbautauthor=A.+Farceauthor=N.+Renaultauthor=F.+Allemandauthor=J.-F.+Guichouauthor=A.-S.+Drucbertauthor=C.+Foulonauthor=R.+Magnezauthor=X.+Thuruauthor=M.+Corvaisierauthor=G.+Huetauthor=P.+Chavatteauthor=P.+Melnykauthor=F.+Baillyauthor=P.+Cotelle&title=Toward+the+discovery+of+a+novel+class+of+YAP%28-%29TEAD+interaction+inhibitors+by+virtual+screening+approach+targeting+YAP%28-%29TEAD+protein%28-%29protein+interface&doi=10.3390%2Fcancers10050140"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Toward the discovery of a novel class of YAP-TEAD interaction inhibitors by virtual screening approach targeting YAP-TEAD protein-protein interface</span></div><div class="casAuthors">Gibault, Floriane; Coevoet, Mathilde; Sturbaut, Manon; Farce, Amaury; Renault, Nicolas; Allemand, Frederic; Guichou, Jean-Francois; Drucbert, Anne-Sophie; Foulon, Catherine; Magnez, Romain; Thuru, Xavier; Corvaisier, Matthieu; Huet, Guillemette; Chavatte, Philippe; Melnyk, Patricia; Bailly, Fabrice; Cotelle, Philippe</div><div class="citationInfo"><span class="NLM_cas:title">Cancers</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">140/1-140/14</span>CODEN:
                <span class="NLM_cas:coden">CANCCT</span>;
        ISSN:<span class="NLM_cas:issn">2072-6694</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Intrinsically disordered protein YAP (yes-assocd. protein) interacts with TEADs transcriptional factors family (transcriptional enhancer assocd. domain) creating three interfaces.  Interface 3, between the Ω-loop of YAP and a shallow pocket of TEAD was identified as the most important TEAD zone for YAP-TEAD interaction.  Using the first X-ray structure of the hYAP50-71-hTEAD1209-426 complex (PDB 3KYS) published in 2010, a protein-protein interaction inhibitors-enriched library (175,000 chem. compds.) was screened against this hydrophobic pocket of TEAD.  Four different chem. families have been identified and evaluated using biophys. techniques (thermal shift assay, microscale thermophoresis) and in cellulo assays (luciferase activity in transfected HEK293 cells, RTqPCR in MDA-MB231 cells).  A first promising hit with micromolar inhibition in the luciferase gene reporter assay was discovered.  This hit also decreased mRNA levels of TEAD target genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSVG22N3nS17Vg90H21EOLACvtfcHk0ljJzqwOogMVPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVemsbzF&md5=a463164da91cbc8db69ad8b41d09270a</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.3390%2Fcancers10050140&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcancers10050140%26sid%3Dliteratum%253Aachs%26aulast%3DGibault%26aufirst%3DF.%26aulast%3DCoevoet%26aufirst%3DM.%26aulast%3DSturbaut%26aufirst%3DM.%26aulast%3DFarce%26aufirst%3DA.%26aulast%3DRenault%26aufirst%3DN.%26aulast%3DAllemand%26aufirst%3DF.%26aulast%3DGuichou%26aufirst%3DJ.-F.%26aulast%3DDrucbert%26aufirst%3DA.-S.%26aulast%3DFoulon%26aufirst%3DC.%26aulast%3DMagnez%26aufirst%3DR.%26aulast%3DThuru%26aufirst%3DX.%26aulast%3DCorvaisier%26aufirst%3DM.%26aulast%3DHuet%26aufirst%3DG.%26aulast%3DChavatte%26aufirst%3DP.%26aulast%3DMelnyk%26aufirst%3DP.%26aulast%3DBailly%26aufirst%3DF.%26aulast%3DCotelle%26aufirst%3DP.%26atitle%3DToward%2520the%2520discovery%2520of%2520a%2520novel%2520class%2520of%2520YAP%2528-%2529TEAD%2520interaction%2520inhibitors%2520by%2520virtual%2520screening%2520approach%2520targeting%2520YAP%2528-%2529TEAD%2520protein%2528-%2529protein%2520interface%26jtitle%3DCancers%26date%3D2018%26volume%3D10%26spage%3D140%26doi%3D10.3390%2Fcancers10050140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gibault, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sturbaut, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailly, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnyk, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cotelle, P.</span></span> <span> </span><span class="NLM_article-title">Targeting transcriptional enhanced associate domains (TEADs)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">5057</span>– <span class="NLM_lpage">5072</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00879</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00879" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvF2gsLbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=5057-5072&author=F.+Gibaultauthor=M.+Sturbautauthor=F.+Baillyauthor=P.+Melnykauthor=P.+Cotelle&title=Targeting+transcriptional+enhanced+associate+domains+%28TEADs%29&doi=10.1021%2Facs.jmedchem.7b00879"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Transcriptional Enhanced Associate Domains (TEADs)</span></div><div class="casAuthors">Gibault, Floriane; Sturbaut, Manon; Bailly, Fabrice; Melnyk, Patricia; Cotelle, Philippe</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5057-5072</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Transcriptional enhanced assoc. domain (TEAD) proteins are the downstream effectors of the Hippo signaling pathway that regulate cell proliferation and stem cell functions.  TEADs are unable to activate transcription and require the help of coactivators such as YAP, TAZ, VgLL, and p160 proteins.  The expression of TEAD family is up-regulated in many cancer types including gastric, colorectal, breast, and prostate cancers, which is correlated with poor survival in patients.  Pharmacol. modulators of TEADs could therefore find application in cancer treatment and regenerative medicine.  In this review, the authors present the very recent available structures of TEADs with or without coactivators or inhibitors and discuss the potential therapeutic application of their ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-oyYqx1TP_LVg90H21EOLACvtfcHk0lgg0Tn_ee6M2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvF2gsLbN&md5=52bed2d6b0f8253b3a417f2569b9df2d</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00879&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00879%26sid%3Dliteratum%253Aachs%26aulast%3DGibault%26aufirst%3DF.%26aulast%3DSturbaut%26aufirst%3DM.%26aulast%3DBailly%26aufirst%3DF.%26aulast%3DMelnyk%26aufirst%3DP.%26aulast%3DCotelle%26aufirst%3DP.%26atitle%3DTargeting%2520transcriptional%2520enhanced%2520associate%2520domains%2520%2528TEADs%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D5057%26epage%3D5072%26doi%3D10.1021%2Facs.jmedchem.7b00879" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sessions, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoemark, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebrahimighaei, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNeill, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crump, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKay, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newby, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bond, M.</span></span> <span> </span><span class="NLM_article-title">Antiproliferative and antimigratory effects of a novel YAP-TEAD interaction inhibitor identified using in silico molecular docking</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">1291</span>– <span class="NLM_lpage">1305</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01402</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01402" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC1MXpvFCrtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=1291-1305&author=S.+A.+Smithauthor=R.+B.+Sessionsauthor=D.+K.+Shoemarkauthor=C.+Williamsauthor=R.+Ebrahimighaeiauthor=M.+C.+McNeillauthor=M.+P.+Crumpauthor=T.+R.+McKayauthor=G.+Harrisauthor=A.+C.+Newbyauthor=M.+Bond&title=Antiproliferative+and+antimigratory+effects+of+a+novel+YAP-TEAD+interaction+inhibitor+identified+using+in+silico+molecular+docking&doi=10.1021%2Facs.jmedchem.8b01402"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Antiproliferative and Antimigratory Effects of a Novel YAP-TEAD Interaction Inhibitor Identified Using in Silico Molecular Docking</span></div><div class="casAuthors">Smith, Sarah A.; Sessions, Richard B.; Shoemark, Deborah K.; Williams, Christopher; Ebrahimighaei, Reza; McNeill, Madeleine C.; Crump, Matthew P.; McKay, Tristan R.; Harris, Gemma; Newby, Andrew C.; Bond, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1291-1305</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Hippo pathway is an important regulator of cell growth, proliferation, and migration.  TEAD transcription factors, which lie at the core of the Hippo pathway, are essential for regulation of organ growth and wound repair.  Dysregulation of TEAD and its regulatory cofactor Yes-assocd. protein (YAP) have been implicated in numerous human cancers and hyperproliferative pathol. processes.  Hence, the YAP-TEAD complex is a promising therapeutic target.  Here, we use in silico mol. docking using Bristol University Docking Engine to screen a library of more than 8 million druglike mols. for novel disrupters of the YAP-TEAD interaction.  We report the identification of a novel compd. (CPD3.1) with the ability to disrupt YAP-TEAD protein-protein interaction and inhibit TEAD activity, cell proliferation, and cell migration.  The YAP-TEAD complex is a viable drug target, and CPD3.1 is a lead compd. for the development of more potent TEAD inhibitors for treating cancer and other hyperproliferative pathologies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqLhehTsmBN7Vg90H21EOLACvtfcHk0lgg0Tn_ee6M2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXpvFCrtQ%253D%253D&md5=1dbe1315ab5036198e0f2d482d7b8cd0</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01402&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01402%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DS.%2BA.%26aulast%3DSessions%26aufirst%3DR.%2BB.%26aulast%3DShoemark%26aufirst%3DD.%2BK.%26aulast%3DWilliams%26aufirst%3DC.%26aulast%3DEbrahimighaei%26aufirst%3DR.%26aulast%3DMcNeill%26aufirst%3DM.%2BC.%26aulast%3DCrump%26aufirst%3DM.%2BP.%26aulast%3DMcKay%26aufirst%3DT.%2BR.%26aulast%3DHarris%26aufirst%3DG.%26aulast%3DNewby%26aufirst%3DA.%2BC.%26aulast%3DBond%26aufirst%3DM.%26atitle%3DAntiproliferative%2520and%2520antimigratory%2520effects%2520of%2520a%2520novel%2520YAP-TEAD%2520interaction%2520inhibitor%2520identified%2520using%2520in%2520silico%2520molecular%2520docking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D1291%26epage%3D1305%26doi%3D10.1021%2Facs.jmedchem.8b01402" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, L.</span></span> <span> </span><span class="NLM_article-title">VGLL4 is a transcriptional cofactor acting as a novel tumor suppressor via interacting with TEADs</span>. <i>Am. J. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">932</span>– <span class="NLM_lpage">943</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=30034932" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFOjs7vL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=932-943&author=X.+Dengauthor=L.+Fang&title=VGLL4+is+a+transcriptional+cofactor+acting+as+a+novel+tumor+suppressor+via+interacting+with+TEADs"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">VGLL4 is a transcriptional cofactor acting as a novel tumor suppressor via interacting with TEADs</span></div><div class="casAuthors">Deng, Xiaochong; Fang, Lin</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Cancer Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">932-943</span>CODEN:
                <span class="NLM_cas:coden">AJCRFT</span>;
        ISSN:<span class="NLM_cas:issn">2156-6976</span>.
    
            (<span class="NLM_cas:orgname">e-Century Publishing Corp.</span>)
        </div><div class="casAbstract">Vestigial Like Family Member 4 (VGLL4) is a transcriptional cofactor of VGLL family, which includes VGLL1- 4.  Unlike other members of VGLL family, VGLL4 was described as a novel tumor suppressor contg. two TDU motifs.  VGLL4 executes its biol. function through two TDU domains via interacting with TEA domain (TEAD) transcription factors.  Lower expression of VGLL4 usually indicates poor survival in many cancers, such as lung cancer, gastric cancer, breast cancer, colorectal cancer, bladder cancer, pancreatic adenocarcinoma and esophageal squamous cancer.  In cancer cells, the expression of VGLL4 is lower than that of normal tissues, moreover, expression level of VGLL4 is pos. related to survival rate.  VGLL4 is found to play an important role in several signal pathways, mainly acts as a tumor suppressor interacting with TEADs.  In Hippo signaling pathway, VGLL4 competes with YAP in binding to TEADs and inhibits the downstream of YAP.  In Wnt/β-catenin signaling pathway, VGLL4 neg. regulates Wnt/β-catenin signaling pathway via inhibiting β-catenin and TCF (T-cell factor).  VGLL4 can also suppress epithelial-mesenchymal transition (EMT) and contribute to apoptosis signaling pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0ATex5pIVN7Vg90H21EOLACvtfcHk0lgg0Tn_ee6M2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFOjs7vL&md5=a68a7902681c58f78d54b4ce285f1bdb</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DFang%26aufirst%3DL.%26atitle%3DVGLL4%2520is%2520a%2520transcriptional%2520cofactor%2520acting%2520as%2520a%2520novel%2520tumor%2520suppressor%2520via%2520interacting%2520with%2520TEADs%26jtitle%3DAm.%2520J.%2520Cancer%2520Res.%26date%3D2018%26volume%3D8%26spage%3D932%26epage%3D943" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pobbati, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, W.</span></span> <span> </span><span class="NLM_article-title">Emerging roles of TEAD transcription factors and its coactivators in cancers</span>. <i>Cancer Biol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">390</span>– <span class="NLM_lpage">398</span>, <span class="refDoi"> DOI: 10.4161/cbt.23788</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.4161%2Fcbt.23788" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=23380592" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjvFOrtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=390-398&author=A.+V.+Pobbatiauthor=W.+Hong&title=Emerging+roles+of+TEAD+transcription+factors+and+its+coactivators+in+cancers&doi=10.4161%2Fcbt.23788"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging roles of TEAD transcription factors and its coactivators in cancers</span></div><div class="casAuthors">Pobbati, Ajaybabu V.; Hong, Wanjin</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Biology & Therapy</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">390-398</span>CODEN:
                <span class="NLM_cas:coden">CBTAAO</span>;
        ISSN:<span class="NLM_cas:issn">1538-4047</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">A review.  TEAD proteins are transcription factors that are crucial for development, but also play a role in cancers.  Several developmentally and pathol. important genes are upregulated by TEADs.  TEADs have a TEA domain that enables them to bind specific DNA elements and a transactivation domain that enables them to interact with coactivators.  TEADs on their own are unable to activate transcription and they require the help of coactivators.  Several TEAD-interacting coactivators are known and they can be classified into three groups: (1) YAP and its paralog TAZ, (2) VgII proteins and (3) p160s.  Accordingly, these coactivators also play a role in development and cancers.  Recent studies have shown that TEADs and their coactivators aid in the progression of various cancers, including the difficult to treat glioblastoma, liver and ovarian cancers.  They facilitate cancer progression through expression of proliferation promoting genes such as c-myc, survivin, Axl, CTGF and Cyr61.  There is also a good correlation between high TEAD or its coactivator expression and poor prognosis in various cancers.  Given the fact that TEADs and their coactivators need to work together for a functional outcome, disrupting the interaction between them appears to be a viable option for cancer therapy.  Structures of TEAD-coactivator complexes have been elucidated and will facilitate drug design and development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaxbkfWmzipLVg90H21EOLACvtfcHk0lg19vcegoFCzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjvFOrtro%253D&md5=86bb7724f71105cd3d27fd0d12f52a8e</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.4161%2Fcbt.23788&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcbt.23788%26sid%3Dliteratum%253Aachs%26aulast%3DPobbati%26aufirst%3DA.%2BV.%26aulast%3DHong%26aufirst%3DW.%26atitle%3DEmerging%2520roles%2520of%2520TEAD%2520transcription%2520factors%2520and%2520its%2520coactivators%2520in%2520cancers%26jtitle%3DCancer%2520Biol.%2520Ther.%26date%3D2013%26volume%3D14%26spage%3D390%26epage%3D398%26doi%3D10.4161%2Fcbt.23788" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeRan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jarugumilli, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span> <span> </span><span class="NLM_article-title">Autopalmitoylation of TEAD proteins regulates transcriptional output of the hippo pathway</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">282</span>– <span class="NLM_lpage">289</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1038%2Fnchembio.2036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=26900866" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC28XivFKgtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=282-289&author=P.+Chanauthor=X.+Hanauthor=B.+Zhengauthor=M.+DeRanauthor=J.+Yuauthor=G.+K.+Jarugumilliauthor=H.+Dengauthor=D.+Panauthor=X.+Luoauthor=X.+Wu&title=Autopalmitoylation+of+TEAD+proteins+regulates+transcriptional+output+of+the+hippo+pathway&doi=10.1038%2Fnchembio.2036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Autopalmitoylation of TEAD proteins regulates transcriptional output of the Hippo pathway</span></div><div class="casAuthors">Chan, PuiYee; Han, Xiao; Zheng, Baohui; DeRan, Michael; Yu, Jianzhong; Jarugumilli, Gopala K.; Deng, Hua; Pan, Duojia; Luo, Xuelian; Wu, Xu</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">282-289</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">TEA domain (TEAD) transcription factors bind to the coactivators YAP and TAZ and regulate the transcriptional output of the Hippo pathway, playing crit. roles in organ size control and tumorigenesis.  Protein S-palmitoylation attaches the fatty acid palmitate to cysteine residues and regulates protein trafficking, membrane localization and signaling activities.  Using activity-based chem. probes, we discovered that human TEADs possess intrinsic palmitoylating enzyme-like activities and undergo autopalmitoylation at evolutionarily conserved cysteine residues under physiol. conditions.  We detd. the crystal structures of lipid-bound TEADs and found that the lipid chain of palmitate inserts into a conserved deep hydrophobic pocket.  Strikingly, palmitoylation did not alter TEAD's localization, but it was required for TEAD's binding to YAP and TAZ and was dispensable for its binding to the Vgll4 tumor suppressor.  Moreover, palmitoylation-deficient TEAD mutants impaired TAZ-mediated muscle differentiation in vitro and tissue overgrowth mediated by the Drosophila YAP homolog Yorkie in vivo.  Our study directly links autopalmitoylation to the transcriptional regulation of the Hippo pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_XknCE2cSXLVg90H21EOLACvtfcHk0lg19vcegoFCzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XivFKgtrs%253D&md5=1b63c4f8b84d04f672041228c2f044c0</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2036%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DP.%26aulast%3DHan%26aufirst%3DX.%26aulast%3DZheng%26aufirst%3DB.%26aulast%3DDeRan%26aufirst%3DM.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DJarugumilli%26aufirst%3DG.%2BK.%26aulast%3DDeng%26aufirst%3DH.%26aulast%3DPan%26aufirst%3DD.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DX.%26atitle%3DAutopalmitoylation%2520of%2520TEAD%2520proteins%2520regulates%2520transcriptional%2520output%2520of%2520the%2520hippo%2520pathway%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D282%26epage%3D289%26doi%3D10.1038%2Fnchembio.2036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mesrouze, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerhofer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bokhovchuk, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontana, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmermann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delaunay, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Izaac, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kallen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmelzle, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdmann, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chène, P.</span></span> <span> </span><span class="NLM_article-title">Effect of the acylation of TEAD4 on its interaction with co-activators YAP and TAZ</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">2399</span>– <span class="NLM_lpage">2409</span>, <span class="refDoi"> DOI: 10.1002/pro.3312</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1002%2Fpro.3312" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=28960584" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2nsbbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2017&pages=2399-2409&author=Y.+Mesrouzeauthor=M.+Meyerhoferauthor=F.+Bokhovchukauthor=P.+Fontanaauthor=C.+Zimmermannauthor=T.+Martinauthor=C.+Delaunayauthor=A.+Izaacauthor=J.+Kallenauthor=T.+Schmelzleauthor=D.+Erdmannauthor=P.+Ch%C3%A8ne&title=Effect+of+the+acylation+of+TEAD4+on+its+interaction+with+co-activators+YAP+and+TAZ&doi=10.1002%2Fpro.3312"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Effect of the acylation of TEAD4 on its interaction with co-activators YAP and TAZ</span></div><div class="casAuthors">Mesrouze, Yannick; Meyerhofer, Marco; Bokhovchuk, Fedir; Fontana, Patrizia; Zimmermann, Catherine; Martin, Typhaine; Delaunay, Clara; Izaac, Aude; Kallen, Joerg; Schmelzle, Tobias; Erdmann, Dirk; Chene, Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2399-2409</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">1469-896X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The Hippo pathway is deregulated in various cancers, and the discovery of mols. that modulate this pathway may open new therapeutic avenues in oncol.  TEA/ATTS domain (TEAD) transcription factors are the most distal elements of the Hippo pathway and their transcriptional activity is regulated by the Yes-assocd. protein (YAP).  Amongst the various possibilities for targeting this pathway, inhibition of the YAP:TEAD interaction is an attractive strategy.  It has been shown recently that TEAD proteins are covalently linked via a conserved cysteine to a fatty acid mol. (palmitate) that binds to a deep hydrophobic cavity present in these proteins.  This acylation of TEAD seems to be required for efficient binding to YAP, and understanding how it modulates the YAP:TEAD interaction may provide useful information on the regulation of TEAD function.  In this report we have studied the effect of TEAD4 acylation on its interaction with YAP and the other co-activator transcriptional co-activator with PDZ-binding motif (TAZ).  We show in our biochem. and cellular assays that YAP and TAZ bind in a similar manner to acylated and non-acylated TEAD4.  This indicates that TEAD4 acylation is not a prerequisite for its interaction with YAP or TAZ.  However, we obsd. that TEAD4 acylation significantly enhances its stability, suggesting that it may help this transcription factor to acquire and/or maintain its active conformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8jstEQT15tLVg90H21EOLACvtfcHk0lg19vcegoFCzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2nsbbE&md5=8604423205879435c87cced3203640e6</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fpro.3312&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.3312%26sid%3Dliteratum%253Aachs%26aulast%3DMesrouze%26aufirst%3DY.%26aulast%3DMeyerhofer%26aufirst%3DM.%26aulast%3DBokhovchuk%26aufirst%3DF.%26aulast%3DFontana%26aufirst%3DP.%26aulast%3DZimmermann%26aufirst%3DC.%26aulast%3DMartin%26aufirst%3DT.%26aulast%3DDelaunay%26aufirst%3DC.%26aulast%3DIzaac%26aufirst%3DA.%26aulast%3DKallen%26aufirst%3DJ.%26aulast%3DSchmelzle%26aufirst%3DT.%26aulast%3DErdmann%26aufirst%3DD.%26aulast%3DCh%25C3%25A8ne%26aufirst%3DP.%26atitle%3DEffect%2520of%2520the%2520acylation%2520of%2520TEAD4%2520on%2520its%2520interaction%2520with%2520co-activators%2520YAP%2520and%2520TAZ%26jtitle%3DProtein%2520Sci.%26date%3D2017%26volume%3D26%26spage%3D2399%26epage%3D2409%26doi%3D10.1002%2Fpro.3312" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Noland, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gierke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnier, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandoval, W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sagolla, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannoush, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairbrother, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, C. N.</span></span> <span> </span><span class="NLM_article-title">Palmitoylation of TEAD transcription factors is required for their stability and function in hippo pathway signaling</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">179</span>– <span class="NLM_lpage">186</span>, <span class="refDoi"> DOI: 10.1016/j.str.2015.11.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1016%2Fj.str.2015.11.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=26724994" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitV2msrrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=179-186&author=C.+L.+Nolandauthor=S.+Gierkeauthor=P.+D.+Schnierauthor=J.+Murrayauthor=W.+N.+Sandovalauthor=M.+Sagollaauthor=A.+Deyauthor=R.+N.+Hannoushauthor=W.+J.+Fairbrotherauthor=C.+N.+Cunningham&title=Palmitoylation+of+TEAD+transcription+factors+is+required+for+their+stability+and+function+in+hippo+pathway+signaling&doi=10.1016%2Fj.str.2015.11.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Palmitoylation of TEAD Transcription Factors Is Required for Their Stability and Function in Hippo Pathway Signaling</span></div><div class="casAuthors">Noland, Cameron L.; Gierke, Sarah; Schnier, Paul D.; Murray, Jeremy; Sandoval, Wendy N.; Sagolla, Meredith; Dey, Anwesha; Hannoush, Rami N.; Fairbrother, Wayne J.; Cunningham, Christian N.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">179-186</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The Hippo signaling pathway is responsible for regulating the function of TEAD family transcription factors in metazoans.  TEADs, with their co-activators YAP/TAZ, are crit. for controlling cell differentiation and organ size through their transcriptional activation of genes involved in cell growth and proliferation.  Dysregulation of the Hippo pathway has been implicated in multiple forms of cancer.  Here, we identify a novel form of regulation of TEAD family proteins.  We show that human TEADs are palmitoylated at a universally conserved cysteine, and report the crystal structures of the human TEAD2 and TEAD3 YAP-binding domains in their palmitoylated forms.  These structures show a palmitate bound within a highly conserved hydrophobic cavity at each protein's core.  Our findings also demonstrate that this modification is required for proper TEAD folding and stability, indicating a potential new avenue for pharmacol. regulating the Hippo pathway through the modulation of TEAD palmitoylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpu5u56RVLmHLVg90H21EOLACvtfcHk0lgbB2fWfRph6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitV2msrrP&md5=6797e3861c599acaec9715dd14e89097</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2015.11.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2015.11.005%26sid%3Dliteratum%253Aachs%26aulast%3DNoland%26aufirst%3DC.%2BL.%26aulast%3DGierke%26aufirst%3DS.%26aulast%3DSchnier%26aufirst%3DP.%2BD.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DSandoval%26aufirst%3DW.%2BN.%26aulast%3DSagolla%26aufirst%3DM.%26aulast%3DDey%26aufirst%3DA.%26aulast%3DHannoush%26aufirst%3DR.%2BN.%26aulast%3DFairbrother%26aufirst%3DW.%2BJ.%26aulast%3DCunningham%26aufirst%3DC.%2BN.%26atitle%3DPalmitoylation%2520of%2520TEAD%2520transcription%2520factors%2520is%2520required%2520for%2520their%2520stability%2520and%2520function%2520in%2520hippo%2520pathway%2520signaling%26jtitle%3DStructure%26date%3D2016%26volume%3D24%26spage%3D179%26epage%3D186%26doi%3D10.1016%2Fj.str.2015.11.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, N.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gumbiner, B. M.</span></span> <span> </span><span class="NLM_article-title">Cell contact and Nf2/Merlin-dependent regulation of TEAD palmitoylation and activity</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">9877</span>– <span class="NLM_lpage">9882</span>, <span class="refDoi"> DOI: 10.1073/pnas.1819400116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1073%2Fpnas.1819400116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=31043565" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC1MXps1Clurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2019&pages=9877-9882&author=N.-G.+Kimauthor=B.+M.+Gumbiner&title=Cell+contact+and+Nf2%2FMerlin-dependent+regulation+of+TEAD+palmitoylation+and+activity&doi=10.1073%2Fpnas.1819400116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Cell contact and Nf2/Merlin-dependent regulation of TEAD palmitoylation and activity</span></div><div class="casAuthors">Kim, Nam-Gyun; Gumbiner, Barry M.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9877-9882</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The Hippo pathway is involved in regulating contact inhibition of proliferation and organ size control and responds to various phys. and biochem. stimuli.  It is a kinase cascade that neg. regulates the activity of cotranscription factors YAP and TAZ, which interact with DNA binding transcription factors including TEAD and activate the expression of target genes.  In this study, we show that the palmitoylation of TEAD, which controls the activity and stability of TEAD proteins, is actively regulated by cell d. independent of Lats, the key kinase of the Hippo pathway.  The expression of fatty acid synthase and acetyl-CoA carboxylase involved in de novo biosynthesis of palmitate is reduced by cell d. in an Nf2/Merlin-dependent manner.  Depalmitoylation of TEAD is mediated by depalmitoylases including APT2 and ABHD17A.  Palmitoylation-deficient TEAD4 mutant is unstable and degraded by proteasome through the activity of the E3 ubiquitin ligase CHIP.  These findings show that TEAD activity is tightly controlled through the regulation of palmitoylation and stability via the orchestration of FASN, depalmitoylases, and E3 ubiquitin ligase in response to cell contact.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9VITmJov-FLVg90H21EOLACvtfcHk0lgbB2fWfRph6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXps1Clurs%253D&md5=a7520b16c5f37b6d7f347370176998d0</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1819400116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1819400116%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DN.-G.%26aulast%3DGumbiner%26aufirst%3DB.%2BM.%26atitle%3DCell%2520contact%2520and%2520Nf2%252FMerlin-dependent%2520regulation%2520of%2520TEAD%2520palmitoylation%2520and%2520activity%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2019%26volume%3D116%26spage%3D9877%26epage%3D9882%26doi%3D10.1073%2Fpnas.1819400116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pobbati, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiguang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huda, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, G.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chia, C. S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poulsen, A.</span></span> <span> </span><span class="NLM_article-title">Targeting the central pocket in human transcription factor TEAD as a potential cancer therapeutic strategy</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">2076</span>– <span class="NLM_lpage">2086</span>, <span class="refDoi"> DOI: 10.1016/j.str.2015.09.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1016%2Fj.str.2015.09.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=26592798" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslWrsLzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=2076-2086&author=A.+V.+Pobbatiauthor=X.+Hanauthor=A.+W.+Hungauthor=S.+Weiguangauthor=N.+Hudaauthor=G.-Y.+Chenauthor=C.+Kangauthor=C.+S.+B.+Chiaauthor=X.+Luoauthor=W.+Hongauthor=A.+Poulsen&title=Targeting+the+central+pocket+in+human+transcription+factor+TEAD+as+a+potential+cancer+therapeutic+strategy&doi=10.1016%2Fj.str.2015.09.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the Central Pocket in Human Transcription Factor TEAD as a Potential Cancer Therapeutic Strategy</span></div><div class="casAuthors">Pobbati, Ajaybabu V.; Han, Xiao; Hung, Alvin W.; Weiguang, Seetoh; Huda, Nur; Chen, Guo-Ying; Kang, CongBao; Chia, Cheng San Brian; Luo, Xuelian; Hong, Wanjin; Poulsen, Anders</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2076-2086</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The human TEAD family of transcription factors (TEAD1-4) is required for YAP-mediated transcription in the Hippo pathway.  Hyperactivation of TEAD's co-activator YAP contributes to tissue overgrowth and human cancers, suggesting that pharmacol. interference of TEAD-YAP activity may be an effective strategy for anticancer therapy.  Here we report the discovery of a central pocket in the YAP-binding domain (YBD) of TEAD that is targetable by small-mol. inhibitors.  Our X-ray crystallog. studies reveal that flufenamic acid, a non-steroidal anti-inflammatory drug (NSAID), binds to the central pocket of TEAD2 YBD.  Our biochem. and functional analyses further demonstrate that binding of NSAIDs to TEAD inhibits TEAD-YAP-dependent transcription, cell migration, and proliferation, indicating that the central pocket is important for TEAD function.  Therefore, our studies discover a novel way of targeting TEAD transcription factors and set the stage for therapeutic development of specific TEAD-YAP inhibitors against human cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzguxiNfMYsrVg90H21EOLACvtfcHk0lgbB2fWfRph6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslWrsLzL&md5=614016edf5305d6a3be84e8a1ac42cb9</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2015.09.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2015.09.009%26sid%3Dliteratum%253Aachs%26aulast%3DPobbati%26aufirst%3DA.%2BV.%26aulast%3DHan%26aufirst%3DX.%26aulast%3DHung%26aufirst%3DA.%2BW.%26aulast%3DWeiguang%26aufirst%3DS.%26aulast%3DHuda%26aufirst%3DN.%26aulast%3DChen%26aufirst%3DG.-Y.%26aulast%3DKang%26aufirst%3DC.%26aulast%3DChia%26aufirst%3DC.%2BS.%2BB.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DHong%26aufirst%3DW.%26aulast%3DPoulsen%26aufirst%3DA.%26atitle%3DTargeting%2520the%2520central%2520pocket%2520in%2520human%2520transcription%2520factor%2520TEAD%2520as%2520a%2520potential%2520cancer%2520therapeutic%2520strategy%26jtitle%3DStructure%26date%3D2015%26volume%3D23%26spage%3D2076%26epage%3D2086%26doi%3D10.1016%2Fj.str.2015.09.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bum-Erdene, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez-Gutierrez, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghozayel, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Si, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shannon, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corson, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollok, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meroueh, S. O.</span></span> <span> </span><span class="NLM_article-title">Small-molecule covalent modification of conserved cysteine leads to allosteric inhibition of the TEAD-Yap protein-protein interaction</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">378</span>– <span class="NLM_lpage">389</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2018.11.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1016%2Fj.chembiol.2018.11.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=30581134" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisF2ksL7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2019&pages=378-389&author=K.+Bum-Erdeneauthor=D.+Zhouauthor=G.+Gonzalez-Gutierrezauthor=M.+K.+Ghozayelauthor=Y.+Siauthor=D.+Xuauthor=H.+E.+Shannonauthor=B.+J.+Baileyauthor=T.+W.+Corsonauthor=K.+E.+Pollokauthor=C.+D.+Wellsauthor=S.+O.+Meroueh&title=Small-molecule+covalent+modification+of+conserved+cysteine+leads+to+allosteric+inhibition+of+the+TEAD-Yap+protein-protein+interaction&doi=10.1016%2Fj.chembiol.2018.11.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Covalent Modification of Conserved Cysteine Leads to Allosteric Inhibition of the TEAD·Yap Protein-Protein Interaction</span></div><div class="casAuthors">Bum-Erdene, Khuchtumur; Zhou, Donghui; Gonzalez-Gutierrez, Giovanni; Ghozayel, Mona K.; Si, Yubing; Xu, David; Shannon, Harlan E.; Bailey, Barbara J.; Corson, Timothy W.; Pollok, Karen E.; Wells, Clark D.; Meroueh, Samy O.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">378-389.e13</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The Hippo pathway coordinates extracellular signals onto the control of tissue homeostasis and organ size.  Hippo signaling primarily regulates the ability of Yap1 to bind and co-activate TEA domain (TEAD) transcription factors.  Yap1 tightly binds to TEAD4 via a large flat interface, making the development of small-mol. orthosteric inhibitors highly challenging.  Here, we report small-mol. TEAD·Yap inhibitors that rapidly and selectively form a covalent bond with a conserved cysteine located within the unique deep hydrophobic palmitate-binding pocket of TEADs.  Inhibition of TEAD4 binding to Yap1 by these compds. was irreversible and occurred on a longer time scale.  In mammalian cells, the compds. formed a covalent complex with TEAD4, inhibited its binding to Yap1, blocked its transcriptional activity, and suppressed expression of connective tissue growth factor.  The compds. inhibited cell viability of patient-derived glioblastoma spheroids, making them suitable as chem. probes to explore Hippo signaling in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZkVd9Oyu-KbVg90H21EOLACvtfcHk0ljExQlgEzZzLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisF2ksL7M&md5=13ab3a2b343da960c9106435ace5ed03</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2018.11.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2018.11.010%26sid%3Dliteratum%253Aachs%26aulast%3DBum-Erdene%26aufirst%3DK.%26aulast%3DZhou%26aufirst%3DD.%26aulast%3DGonzalez-Gutierrez%26aufirst%3DG.%26aulast%3DGhozayel%26aufirst%3DM.%2BK.%26aulast%3DSi%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DD.%26aulast%3DShannon%26aufirst%3DH.%2BE.%26aulast%3DBailey%26aufirst%3DB.%2BJ.%26aulast%3DCorson%26aufirst%3DT.%2BW.%26aulast%3DPollok%26aufirst%3DK.%2BE.%26aulast%3DWells%26aufirst%3DC.%2BD.%26aulast%3DMeroueh%26aufirst%3DS.%2BO.%26atitle%3DSmall-molecule%2520covalent%2520modification%2520of%2520conserved%2520cysteine%2520leads%2520to%2520allosteric%2520inhibition%2520of%2520the%2520TEAD-Yap%2520protein-protein%2520interaction%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2019%26volume%3D26%26spage%3D378%26epage%3D389%26doi%3D10.1016%2Fj.chembiol.2018.11.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaneda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seike, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uemori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myojo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aida, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danjo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakajima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuji, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamaguchi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yasunaga, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otsubo, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Onodera, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishiya, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saito, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishii, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakai, R.</span></span> <span> </span><span class="NLM_article-title">Abstract 3086 Discovery of a first-in-class TEAD inhibitor which directly inhibits YAP/TAZ-TEAD protein-protein interaction and shows a potent anti-tumor effect in malignant pleural mMesothelioma</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">3086</span>, <span class="refDoi"> DOI: 10.1158/1538-7445.AM2019-3086</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1158%2F1538-7445.AM2019-3086" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2019&pages=3086&author=A.+Kanedaauthor=T.+Seikeauthor=T.+Uemoriauthor=K.+Myojoauthor=K.+Aidaauthor=T.+Danjoauthor=T.+Nakajimaauthor=D.+Yamaguchiauthor=T.+Hamadaauthor=Y.+Tsujiauthor=K.+Hamaguchiauthor=M.+Yasunagaauthor=N.+Otsuboauthor=H.+Onoderaauthor=Y.+Nishiyaauthor=M.+Suzukiauthor=J.+Saitoauthor=T.+Ishiiauthor=R.+Nakai&title=Abstract+3086+Discovery+of+a+first-in-class+TEAD+inhibitor+which+directly+inhibits+YAP%2FTAZ-TEAD+protein-protein+interaction+and+shows+a+potent+anti-tumor+effect+in+malignant+pleural+mMesothelioma&doi=10.1158%2F1538-7445.AM2019-3086"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1158%2F1538-7445.AM2019-3086&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1538-7445.AM2019-3086%26sid%3Dliteratum%253Aachs%26aulast%3DKaneda%26aufirst%3DA.%26aulast%3DSeike%26aufirst%3DT.%26aulast%3DUemori%26aufirst%3DT.%26aulast%3DMyojo%26aufirst%3DK.%26aulast%3DAida%26aufirst%3DK.%26aulast%3DDanjo%26aufirst%3DT.%26aulast%3DNakajima%26aufirst%3DT.%26aulast%3DYamaguchi%26aufirst%3DD.%26aulast%3DHamada%26aufirst%3DT.%26aulast%3DTsuji%26aufirst%3DY.%26aulast%3DHamaguchi%26aufirst%3DK.%26aulast%3DYasunaga%26aufirst%3DM.%26aulast%3DOtsubo%26aufirst%3DN.%26aulast%3DOnodera%26aufirst%3DH.%26aulast%3DNishiya%26aufirst%3DY.%26aulast%3DSuzuki%26aufirst%3DM.%26aulast%3DSaito%26aufirst%3DJ.%26aulast%3DIshii%26aufirst%3DT.%26aulast%3DNakai%26aufirst%3DR.%26atitle%3DAbstract%25203086%2520Discovery%2520of%2520a%2520first-in-class%2520TEAD%2520inhibitor%2520which%2520directly%2520inhibits%2520YAP%252FTAZ-TEAD%2520protein-protein%2520interaction%2520and%2520shows%2520a%2520potent%2520anti-tumor%2520effect%2520in%2520malignant%2520pleural%2520mMesothelioma%26jtitle%3DCancer%2520Res.%26date%3D2019%26volume%3D79%26spage%3D3086%26doi%3D10.1158%2F1538-7445.AM2019-3086" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lian, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span> <span> </span><span class="NLM_article-title">Discovery and biological evaluation of vinylsulfonamide derivatives as highly potent, covalent TEAD autopalmitoylation inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>184</i></span>,  <span class="NLM_fpage">111767</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2019.111767</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1016%2Fj.ejmech.2019.111767" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=31622854" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFyhsLvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=184&publication_year=2019&pages=111767&author=W.+Luauthor=J.+Wangauthor=Y.+Liauthor=H.+Taoauthor=H.+Xiongauthor=F.+Lianauthor=J.+Gaoauthor=H.+Maauthor=T.+Luauthor=D.+Zhangauthor=X.+Yeauthor=H.+Dingauthor=L.+Yueauthor=Y.+Zhangauthor=H.+Tangauthor=N.+Zhangauthor=Y.+Yangauthor=H.+Jiangauthor=K.+Chenauthor=B.+Zhouauthor=C.+Luo&title=Discovery+and+biological+evaluation+of+vinylsulfonamide+derivatives+as+highly+potent%2C+covalent+TEAD+autopalmitoylation+inhibitors&doi=10.1016%2Fj.ejmech.2019.111767"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and biological evaluation of vinylsulfonamide derivatives as highly potent, covalent TEAD autopalmitoylation inhibitors</span></div><div class="casAuthors">Lu, Wenchao; Wang, Jun; Li, Yong; Tao, Hongru; Xiong, Huan; Lian, Fulin; Gao, Jing; Ma, Hongna; Lu, Tian; Zhang, Dan; Ye, Xiaoqing; Ding, Hong; Yue, Liyan; Zhang, Yuanyuan; Tang, Huanyu; Zhang, Naixia; Yang, Yaxi; Jiang, Hualiang; Chen, Kaixian; Zhou, Bing; Luo, Cheng</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">184</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">111767</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Transcriptional enhancer assocd. domain family members (TEADs) are the most important downstream effectors that play the pivotal role in the development, regeneration and tissue homeostasis.  Recent biochem. studies have demonstrated that TEADs could undergo autopalmitoylation that is indispensable for its function making the lipid-binding pocket an attractive target for chem. intervention.  Herein, through structure-based virtual screen and rational medicinal chem. optimization, we identified DC-TEADin02(I) as the most potent, selective, covalent TEAD autopalmitoylation inhibitor with the IC50 value of 197 ± 19 nM while it showed minimal effect on TEAD-YAP interaction.  Further biochem. counter-screens demonstrate the specific thiol reactivity and selectivity of I over the kinase family, lipid-binding proteins and epigenetic targets.  Notably, I inhibited TEADs transcription activity leading to downregulation of YAP-related downstream gene expression.  Taken together, our findings proved the validity of modulating transcriptional output in the Hippo signaling pathway through irreversible chem. interventions of TEADs autopalmitoylation activity, which may serve as a qualified chem. tool for TEADs palmitoylation-related studies in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGog-ZFmXN-ezLVg90H21EOLACvtfcHk0ljExQlgEzZzLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFyhsLvN&md5=37289f125bb7e93056c058430b128e7e</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2019.111767&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2019.111767%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DTao%26aufirst%3DH.%26aulast%3DXiong%26aufirst%3DH.%26aulast%3DLian%26aufirst%3DF.%26aulast%3DGao%26aufirst%3DJ.%26aulast%3DMa%26aufirst%3DH.%26aulast%3DLu%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DYe%26aufirst%3DX.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DYue%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DTang%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DDiscovery%2520and%2520biological%2520evaluation%2520of%2520vinylsulfonamide%2520derivatives%2520as%2520highly%2520potent%252C%2520covalent%2520TEAD%2520autopalmitoylation%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2019%26volume%3D184%26spage%3D111767%26doi%3D10.1016%2Fj.ejmech.2019.111767" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jarugumilli, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cotton, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiol, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, P. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeRan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leiter, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ip, Y. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camargo, F. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, J.</span></span> <span> </span><span class="NLM_article-title">Lats1/2 Sustain intestinal stem cells and Wnt activation through TEAD-dependent and independent transcription</span>. <i>Cell Stem Cell</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">675</span>– <span class="NLM_lpage">692</span>, <span class="refDoi"> DOI: 10.1016/j.stem.2020.03.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1016%2Fj.stem.2020.03.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=32259481" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BB3cXmsFOltbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2020&pages=675-692&author=Q.+Liauthor=Y.+Sunauthor=G.+K.+Jarugumilliauthor=S.+Liuauthor=K.+Dangauthor=J.+L.+Cottonauthor=J.+Xiolauthor=P.+Y.+Chanauthor=M.+DeRanauthor=L.+Maauthor=R.+Liauthor=L.+J.+Zhuauthor=J.+H.+Liauthor=A.+B.+Leiterauthor=Y.+T.+Ipauthor=F.+D.+Camargoauthor=X.+Luoauthor=R.+L.+Johnsonauthor=X.+Wuauthor=J.+Mao&title=Lats1%2F2+Sustain+intestinal+stem+cells+and+Wnt+activation+through+TEAD-dependent+and+independent+transcription&doi=10.1016%2Fj.stem.2020.03.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Lats1/2 Sustain Intestinal Stem Cells and Wnt Activation through TEAD-Dependent and Independent Transcription</span></div><div class="casAuthors">Li, Qi; Sun, Yang; Jarugumilli, Gopala K.; Liu, Shun; Dang, Kyvan; Cotton, Jennifer L.; Xiol, Jordi; Chan, Pui Yee; DeRan, Michael; Ma, Lifang; Li, Rui; Zhu, Lihua J.; Li, Joyce H.; Leiter, Andrew B.; Ip, Y. Tony; Camargo, Fernando D.; Luo, Xuelian; Johnson, Randy L.; Wu, Xu; Mao, Junhao</div><div class="citationInfo"><span class="NLM_cas:title">Cell Stem Cell</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">675-692.e8</span>CODEN:
                <span class="NLM_cas:coden">CSCEC4</span>;
        ISSN:<span class="NLM_cas:issn">1875-9777</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Intestinal homeostasis is tightly regulated by complex yet poorly understood signaling networks.  Here, we demonstrate that Lats1/2, the core Hippo kinases, are essential to maintain Wnt pathway activity and intestinal stem cells.  Lats1/2 deletion leads to loss of intestinal stem cells but drives Wnt-uncoupled crypt expansion.  To explore the function of downstream transcriptional enhanced assoc. domain (TEAD) transcription factors, we identified a selective small-mol. reversible inhibitor of TEAD auto-palmitoylation that directly occupies its lipid-binding site and inhibits TEAD-mediated transcription in vivo.  Combining this chem. tool with genetic and proteomics approaches, we show that intestinal Wnt inhibition by Lats deletion is Yes-assocd. protein (YAP)/transcriptional activator with PDZ-binding domain (TAZ) dependent but TEAD independent.  Mechanistically, nuclear YAP/TAZ interact with Groucho/Transducin-Like Enhancer of Split (TLE) to block Wnt/T-cell factor (TCF)-mediated transcription, and dual inhibition of TEAD and Lats suppresses Wnt-uncoupled Myc upregulation and epithelial over-proliferation in Adenomatous polyposis coli (APC)-mutated intestine.  Our studies highlight a pharmacol. approach to inhibit TEAD palmitoylation and have important implications for targeting Wnt and Hippo signaling in human malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAIgIAq16ZxrVg90H21EOLACvtfcHk0lgMJ4szbL1ZWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXmsFOltbc%253D&md5=ce6305aa48f08aceb2412d72beb85e0f</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.stem.2020.03.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.stem.2020.03.002%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DJarugumilli%26aufirst%3DG.%2BK.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DDang%26aufirst%3DK.%26aulast%3DCotton%26aufirst%3DJ.%2BL.%26aulast%3DXiol%26aufirst%3DJ.%26aulast%3DChan%26aufirst%3DP.%2BY.%26aulast%3DDeRan%26aufirst%3DM.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DR.%26aulast%3DZhu%26aufirst%3DL.%2BJ.%26aulast%3DLi%26aufirst%3DJ.%2BH.%26aulast%3DLeiter%26aufirst%3DA.%2BB.%26aulast%3DIp%26aufirst%3DY.%2BT.%26aulast%3DCamargo%26aufirst%3DF.%2BD.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DJohnson%26aufirst%3DR.%2BL.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DMao%26aufirst%3DJ.%26atitle%3DLats1%252F2%2520Sustain%2520intestinal%2520stem%2520cells%2520and%2520Wnt%2520activation%2520through%2520TEAD-dependent%2520and%2520independent%2520transcription%26jtitle%3DCell%2520Stem%2520Cell%26date%3D2020%26volume%3D26%26spage%3D675%26epage%3D692%26doi%3D10.1016%2Fj.stem.2020.03.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kunig, V. B. K.</span>; <span class="NLM_string-name">Potowski, M.</span>; <span class="NLM_string-name">Akbarzadeh, M.</span>; <span class="NLM_string-name">Klika Škopić, M.</span>; <span class="NLM_string-name">Santos Smith, D.</span>; <span class="NLM_string-name">Arendt, L.</span>; <span class="NLM_string-name">Dormuth, I.</span>; <span class="NLM_string-name">Adihou, H.</span>; <span class="NLM_string-name">Andlovic, B.</span>; <span class="NLM_string-name">Karatas, H.</span>; <span class="NLM_string-name">Shaabani, S.</span>; <span class="NLM_string-name">Zarganes-Tzitzikas, T.</span>; <span class="NLM_string-name">Neochoritis, C. G.</span>; <span class="NLM_string-name">Zhang, R.</span>; <span class="NLM_string-name">Groves, M.</span>; <span class="NLM_string-name">Guéret, S. M.</span>; <span class="NLM_string-name">Ottmann, C.</span>; <span class="NLM_string-name">Rahnenführer, J.</span>; <span class="NLM_string-name">Fried, R.</span>; <span class="NLM_string-name">Dömling, A.</span>; <span class="NLM_string-name">Brunschweiger, A.</span></span> <span> </span><span class="NLM_article-title">TEAD-YAP interaction inhibitors and MDM2 binders from DNA-encoded indole-focused Ugi-peptidomimetics</span>.  <i>Angew. Chem., Int. Ed. Engl.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume">59</span>, <span class="refDoi"> DOI: 10.1002/anie.202006280</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1002%2Fanie.202006280" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2020&author=V.+B.+K.+Kunig&author=M.+Potowski&author=M.+Akbarzadeh&author=M.+Klika+%C5%A0kopi%C4%87&author=D.+Santos+Smith&author=L.+Arendt&author=I.+Dormuth&author=H.+Adihou&author=B.+Andlovic&author=H.+Karatas&author=S.+Shaabani&author=T.+Zarganes-Tzitzikas&author=C.+G.+Neochoritis&author=R.+Zhang&author=M.+Groves&author=S.+M.+Gu%C3%A9ret&author=C.+Ottmann&author=J.+Rahnenf%C3%BChrer&author=R.+Fried&author=A.+D%C3%B6mling&author=A.+Brunschweiger&title=TEAD-YAP+interaction+inhibitors+and+MDM2+binders+from+DNA-encoded+indole-focused+Ugi-peptidomimetics&doi=10.1002%2Fanie.202006280"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1002%2Fanie.202006280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.202006280%26sid%3Dliteratum%253Aachs%26aulast%3DKunig%26aufirst%3DV.%2BB.%2BK.%26atitle%3DTEAD-YAP%2520interaction%2520inhibitors%2520and%2520MDM2%2520binders%2520from%2520DNA-encoded%2520indole-focused%2520Ugi-peptidomimetics%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%2520Engl.%26date%3D2020%26volume%3D59%26doi%3D10.1002%2Fanie.202006280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pobbati, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mejuch, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakraborty, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karatas, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bharath, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guéret, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goy, P.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahne, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pahl, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sievers, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, W.</span></span> <span> </span><span class="NLM_article-title">Identification of quinolinols as activators of TEAD-dependent transcription</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">2909</span>– <span class="NLM_lpage">2921</span>, <span class="refDoi"> DOI: 10.1021/acschembio.9b00786</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.9b00786" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFOmt7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=2909-2921&author=A.+V.+Pobbatiauthor=T.+Mejuchauthor=S.+Chakrabortyauthor=H.+Karatasauthor=S.+R.+Bharathauthor=S.+M.+Gu%C3%A9retauthor=P.-A.+Goyauthor=G.+Hahneauthor=A.+Pahlauthor=S.+Sieversauthor=E.+Guccioneauthor=H.+Songauthor=H.+Waldmannauthor=W.+Hong&title=Identification+of+quinolinols+as+activators+of+TEAD-dependent+transcription&doi=10.1021%2Facschembio.9b00786"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Quinolinols as Activators of TEAD-Dependent Transcription</span></div><div class="casAuthors">Pobbati, Ajaybabu V.; Mejuch, Tom; Chakraborty, Sayan; Karatas, Hacer; Bharath, Sakshibeedu R.; Gueret, Stephanie M.; Goy, Pierre-Alexis; Hahne, Gernot; Pahl, Axel; Sievers, Sonja; Guccione, Ernesto; Song, Haiwei; Waldmann, Herbert; Hong, Wanjin</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2909-2921</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The transcriptional co-regulators YAP (Yes-assocd. protein) and TAZ (Transcriptional co-activator with PDZ-binding motif) are the vertebrate downstream effectors of the Hippo signaling pathway that control various physiol. and pathol. processes.  YAP and TAZ pair with the TEAD (TEA domain) family of transcription factors to initiate transcription.  We previously identified a tractable pocket in TEADs, which has been physiol. shown to bind palmitate.  Herein, a TEAD-Palmitate interaction screen was developed to select small mols. occupying the palmitate-binding pocket (PBP) of TEADs.  We show that quinolinols were TEAD-binding compds. that augment YAP/TAZ-TEAD activity, which was verified using TEAD reporter assay, RT-qPCR and RNA-Seq analyses.  Structure activity relationship investigations uncovered the quinolinol substituents that are necessary for TEAD activation.  We reveal a novel mechanism where quinolinols stabilize YAP/TAZ protein levels by occupying the PBP.  The enhancement of YAP activity by quinolinols accelerates the in vivo wound closure in a mouse wound-healing model.  Although small mols. that occupy the PBP have been shown to inhibit YAP/TAZ-TEAD activity, leveraging PBP to activate TEADs is a novel approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJh56PW0Y3orVg90H21EOLACvtfcHk0lgMJ4szbL1ZWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFOmt7rO&md5=b0ba938316568aeff3bd8a4136b7a26f</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Facschembio.9b00786&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.9b00786%26sid%3Dliteratum%253Aachs%26aulast%3DPobbati%26aufirst%3DA.%2BV.%26aulast%3DMejuch%26aufirst%3DT.%26aulast%3DChakraborty%26aufirst%3DS.%26aulast%3DKaratas%26aufirst%3DH.%26aulast%3DBharath%26aufirst%3DS.%2BR.%26aulast%3DGu%25C3%25A9ret%26aufirst%3DS.%2BM.%26aulast%3DGoy%26aufirst%3DP.-A.%26aulast%3DHahne%26aufirst%3DG.%26aulast%3DPahl%26aufirst%3DA.%26aulast%3DSievers%26aufirst%3DS.%26aulast%3DGuccione%26aufirst%3DE.%26aulast%3DSong%26aufirst%3DH.%26aulast%3DWaldmann%26aufirst%3DH.%26aulast%3DHong%26aufirst%3DW.%26atitle%3DIdentification%2520of%2520quinolinols%2520as%2520activators%2520of%2520TEAD-dependent%2520transcription%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2019%26volume%3D14%26spage%3D2909%26epage%3D2921%26doi%3D10.1021%2Facschembio.9b00786" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Holden, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noland, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zbieg, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lacap, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beroza, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reja, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tom, J. Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fong, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steffek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clausen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagenbeek, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, C. N.</span></span> <span> </span><span class="NLM_article-title">Small molecule dysregulation of TEAD lipidation induces a dominant-negative inhibition of hippo pathway signaling</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">107809</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2020.107809</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1016%2Fj.celrep.2020.107809" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=32579935" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1Kis7nN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2020&pages=107809&author=J.+K.+Holdenauthor=J.+J.+Crawfordauthor=C.+L.+Nolandauthor=S.+Schmidtauthor=J.+R.+Zbiegauthor=J.+A.+Lacapauthor=R.+Zangauthor=G.+M.+Millerauthor=Y.+Zhangauthor=P.+Berozaauthor=R.+Rejaauthor=W.+Leeauthor=J.+Y.+K.+Tomauthor=R.+Fongauthor=M.+Steffekauthor=S.+Clausenauthor=T.+J.+Hagenbeekauthor=T.+Huauthor=Z.+Zhouauthor=H.+C.+Shenauthor=C.+N.+Cunningham&title=Small+molecule+dysregulation+of+TEAD+lipidation+induces+a+dominant-negative+inhibition+of+hippo+pathway+signaling&doi=10.1016%2Fj.celrep.2020.107809"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Dysregulation of TEAD Lipidation Induces a Dominant-Negative Inhibition of Hippo Pathway Signaling</span></div><div class="casAuthors">Holden, Jeffrey K.; Crawford, James J.; Noland, Cameron L.; Schmidt, Stephen; Zbieg, Jason R.; Lacap, Jennifer A.; Zang, Richard; Miller, Gregory M.; Zhang, Yue; Beroza, Paul; Reja, Rohit; Lee, Wendy; Tom, Jeffrey Y. K.; Fong, Rina; Steffek, Micah; Clausen, Saundra; Hagenbeek, Thjis J.; Hu, Taishan; Zhou, Zheng; Shen, Hong C.; Cunningham, Christian N.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">107809</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The transcriptional enhanced assoc. domain (TEAD) family of transcription factors serves as the receptors for the downstream effectors of the Hippo pathway, YAP and TAZ, to upregulate the expression of multiple genes involved in cellular proliferation and survival.  Recent work identified TEAD S-palmitoylation as crit. for protein stability and activity as the lipid tail extends into a hydrophobic core of the protein.  Here, we report the identification and characterization of a potent small mol. that binds the TEAD lipid pocket (LP) and disrupts TEAD S-palmitoylation.  Using a variety of biochem., structural, and cellular methods, we uncover that TEAD S-palmitoylation functions as a TEAD homeostatic protein level checkpoint and that dysregulation of this lipidation affects TEAD transcriptional activity in a dominant-neg. manner.  Furthermore, we demonstrate that targeting the TEAD LP is a promising therapeutic strategy for modulating the Hippo pathway, showing tumor stasis in a mouse xenograft model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpg0_N1z8lt0LVg90H21EOLACvtfcHk0lhAEsKAhJxc0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1Kis7nN&md5=4c0c530d9cadf143c035a1ac0ae32633</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2020.107809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2020.107809%26sid%3Dliteratum%253Aachs%26aulast%3DHolden%26aufirst%3DJ.%2BK.%26aulast%3DCrawford%26aufirst%3DJ.%2BJ.%26aulast%3DNoland%26aufirst%3DC.%2BL.%26aulast%3DSchmidt%26aufirst%3DS.%26aulast%3DZbieg%26aufirst%3DJ.%2BR.%26aulast%3DLacap%26aufirst%3DJ.%2BA.%26aulast%3DZang%26aufirst%3DR.%26aulast%3DMiller%26aufirst%3DG.%2BM.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DBeroza%26aufirst%3DP.%26aulast%3DReja%26aufirst%3DR.%26aulast%3DLee%26aufirst%3DW.%26aulast%3DTom%26aufirst%3DJ.%2BY.%2BK.%26aulast%3DFong%26aufirst%3DR.%26aulast%3DSteffek%26aufirst%3DM.%26aulast%3DClausen%26aufirst%3DS.%26aulast%3DHagenbeek%26aufirst%3DT.%2BJ.%26aulast%3DHu%26aufirst%3DT.%26aulast%3DZhou%26aufirst%3DZ.%26aulast%3DShen%26aufirst%3DH.%2BC.%26aulast%3DCunningham%26aufirst%3DC.%2BN.%26atitle%3DSmall%2520molecule%2520dysregulation%2520of%2520TEAD%2520lipidation%2520induces%2520a%2520dominant-negative%2520inhibition%2520of%2520hippo%2520pathway%2520signaling%26jtitle%3DCell%2520Rep.%26date%3D2020%26volume%3D31%26spage%3D107809%26doi%3D10.1016%2Fj.celrep.2020.107809" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alexandrov, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mileni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chien, E. Y. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanson, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span> <span> </span><span class="NLM_article-title">Microscale fluorescent thermal stability assay for membrane proteins</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">351</span>– <span class="NLM_lpage">359</span>, <span class="refDoi"> DOI: 10.1016/j.str.2008.02.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1016%2Fj.str.2008.02.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=18334210" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjtVKjsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2008&pages=351-359&author=A.+I.+Alexandrovauthor=M.+Mileniauthor=E.+Y.+T.+Chienauthor=M.+A.+Hansonauthor=R.+C.+Stevens&title=Microscale+fluorescent+thermal+stability+assay+for+membrane+proteins&doi=10.1016%2Fj.str.2008.02.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Microscale fluorescent thermal stability assay for membrane proteins</span></div><div class="casAuthors">Alexandrov, Alexander I.; Mileni, Mauro; Chien, Ellen Y. T.; Hanson, Michael A.; Stevens, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">351-359</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Systematic efforts to understand membrane protein stability under a variety of different soln. conditions are not widely available for membrane proteins, mainly due to tech. problems stemming from the presence of detergents necessary to keep the proteins in the solubilized state and the background that such detergents usually generate during biophys. characterization.  In this report, the authors introduce an efficient microscale fluorescent stability screen using the thiol-specific fluorochrome N-[4-(7-diethylamino-4-methyl-3-coumarinyl)phenyl]maleimide (CPM) for stability profiling of membrane proteins under different soln. and ligand conditions.  The screen uses the chem. reactivity of the native cysteines embedded in the protein interior as a sensor for the overall integrity of the folded state.  The thermal information gained by thorough investigation of the protein stability landscape can be effectively used to guide purifn. and biophys. characterization efforts including crystn.  To evaluate the method, three different protein families were analyzed, including the Apelin G protein-coupled receptor (APJ).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzPDnouXzOg7Vg90H21EOLACvtfcHk0lhAEsKAhJxc0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjtVKjsro%253D&md5=ec6afa11f7fdb06b1e415e113d42b843</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2008.02.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2008.02.004%26sid%3Dliteratum%253Aachs%26aulast%3DAlexandrov%26aufirst%3DA.%2BI.%26aulast%3DMileni%26aufirst%3DM.%26aulast%3DChien%26aufirst%3DE.%2BY.%2BT.%26aulast%3DHanson%26aufirst%3DM.%2BA.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26atitle%3DMicroscale%2520fluorescent%2520thermal%2520stability%2520assay%2520for%2520membrane%2520proteins%26jtitle%3DStructure%26date%3D2008%26volume%3D16%26spage%3D351%26epage%3D359%26doi%3D10.1016%2Fj.str.2008.02.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brune, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corrie, J. E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webb, M. R.</span></span> <span> </span><span class="NLM_article-title">A fluorescent sensor of the phosphorylation state of nucleoside diphosphate kinase and its use to monitor nucleoside diphosphate concentrations in real time</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">5087</span>– <span class="NLM_lpage">5094</span>, <span class="refDoi"> DOI: 10.1021/bi002484h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi002484h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BD3MXit1Sksr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2001&pages=5087-5094&author=M.+Bruneauthor=J.+E.+T.+Corrieauthor=M.+R.+Webb&title=A+fluorescent+sensor+of+the+phosphorylation+state+of+nucleoside+diphosphate+kinase+and+its+use+to+monitor+nucleoside+diphosphate+concentrations+in+real+time&doi=10.1021%2Fbi002484h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">A fluorescent sensor of the phosphorylation state of nucleoside diphosphate kinase and its use to monitor nucleoside diphosphate concentrations in real time</span></div><div class="casAuthors">Brune, Martin; Corrie, John E. T.; Webb, Martin R.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">5087-5094</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A sensor for purine nucleoside diphosphates in soln. based on nucleoside diphosphate kinase (NDPK) has been developed.  A single cysteine was introduced into the protein and labeled with the environmentally sensitive fluorophore, N-[2-(iodoacetamido)ethyl]-7-diethylaminocoumarin-3-carboxamide.  The resultant mol. shows a 4-fold fluorescence increase when phosphorylated on His117; this phosphorylation is on the normal reaction pathway of the enzyme.  The emission max. of the phosphoenzyme is at 475 nm, with max. excitation at 430 nm.  The fluorescent phosphorylated NDPK is used to measure the amt. of ADP and the unphosphorylated to measure ATP.  The labeled protein is phosphorylated to >90%, and the resultant mol. is stable on ice or can be stored at -80°.  The fluorescence responds to the fraction of protein phosphorylated and so to the equil. between ADP plus NDPK∼P and ATP plus NDPK.  In effect, the sensor measures the ADP/ATP concn. ratio.  The enzyme has a broad specificity for the purine of the nucleotides, so the sensor also can measure GDP/GTP ratios.  The fluorescence and kinetic properties of the labeled protein are described.  The binding rate consts. of nucleotides are ∼105 M-1 s-1, and the fluorescence change is at >200 s-1 when the ADP concn. is >1 mM.  Results are presented with two well-defined systems, namely, the kinetics of ADP release from myosin subfragment 1 and GDP release from the small G protein, human rho.  The results obtained with this novel sensor agree with those from alternate methods and demonstrate the applicability for following micromolar changes in nucleoside diphosphate in real time.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodDeG3YnVMdLVg90H21EOLACvtfcHk0lhAEsKAhJxc0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXit1Sksr4%253D&md5=0e461b9164c7ce973ba64583518dc38d</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fbi002484h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi002484h%26sid%3Dliteratum%253Aachs%26aulast%3DBrune%26aufirst%3DM.%26aulast%3DCorrie%26aufirst%3DJ.%2BE.%2BT.%26aulast%3DWebb%26aufirst%3DM.%2BR.%26atitle%3DA%2520fluorescent%2520sensor%2520of%2520the%2520phosphorylation%2520state%2520of%2520nucleoside%2520diphosphate%2520kinase%2520and%2520its%2520use%2520to%2520monitor%2520nucleoside%2520diphosphate%2520concentrations%2520in%2520real%2520time%26jtitle%3DBiochemistry%26date%3D2001%26volume%3D40%26spage%3D5087%26epage%3D5094%26doi%3D10.1021%2Fbi002484h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohwaki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneeweis, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stec, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varnerin, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goudreau, P. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cassaday, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamakawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kornienko, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodder, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inglese, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strulovici, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kusunoki, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tota, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takagi, T.</span></span> <span> </span><span class="NLM_article-title">A fluorescence-based thiol quantification assay for ultra-high-throughput screening for inhibitors of coenzyme A production</span>. <i>Assay Drug Dev. Technol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">361</span>– <span class="NLM_lpage">374</span>, <span class="refDoi"> DOI: 10.1089/adt.2007.105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1089%2Fadt.2007.105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=18452391" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnvFChu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2008&pages=361-374&author=C.+C.+Chungauthor=K.+Ohwakiauthor=J.+E.+Schneeweisauthor=E.+Stecauthor=J.+P.+Varnerinauthor=P.+N.+Goudreauauthor=A.+Changauthor=J.+Cassadayauthor=L.+Yangauthor=T.+Yamakawaauthor=O.+Kornienkoauthor=P.+Hodderauthor=J.+Ingleseauthor=M.+Ferrerauthor=B.+Struloviciauthor=J.+Kusunokiauthor=M.+R.+Totaauthor=T.+Takagi&title=A+fluorescence-based+thiol+quantification+assay+for+ultra-high-throughput+screening+for+inhibitors+of+coenzyme+A+production&doi=10.1089%2Fadt.2007.105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">A Fluorescence-Based Thiol Quantification Assay for Ultra-High-Throughput Screening for Inhibitors of Coenzyme A Production</span></div><div class="casAuthors">Chung, Christine C.; Ohwaki, Kenji; Schneeweis, Jonathan E.; Stec, Erica; Varnerin, Jeffrey P.; Goudreau, Paul N.; Chang, Amy; Cassaday, Jason; Yang, Lihu; Yamakawa, Takeru; Kornienko, Oleg; Hodder, Peter; Inglese, James; Ferrer, Marc; Strulovici, Berta; Kusunoki, Jun; Tota, Michael R.; Takagi, Toshimitsu</div><div class="citationInfo"><span class="NLM_cas:title">Assay and Drug Development Technologies</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">361-374</span>CODEN:
                <span class="NLM_cas:coden">ADDTAR</span>;
        ISSN:<span class="NLM_cas:issn">1540-658X</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">Here we report the development and miniaturization of a cell-free enzyme assay for ultra-high-throughput screening (uHTS) for inhibitors of two potential drug targets for obesity and cancer: fatty acid synthase (FAS) and acetyl-CoA carboxylase (ACC) 2.  This assay detects CoA, a product of the FAS-catalyzed condensation of malonyl-CoA and acetyl-CoA.  The free thiol of CoA can react with 7-diethylamino-3-(4'-maleimidylphenyl)-4-methylcoumarin (CPM), a profluorescent coumarin maleimide deriv. that becomes fluorescent upon reaction with thiols.  FAS produces long-chain fatty acid and CoA from the condensation of malonyl-CoA and acetyl-CoA.  In our FAS assay, CoA released in the FAS reaction forms a fluorescence adduct with CPM that emits at 530 nm when excited at 405 nm.  Using this detection method for CoA, we measured the activity of sequential enzymes in the fatty acid synthesis pathway to develop an ACC2/FAS-coupled assay where ACC2 produces malonyl-CoA from acetyl-CoA.  We miniaturized the FAS and ACC2/FAS assays to 3456- and 1536-well plate format, resp., and completed uHTSs for small mol. inhibitors of this enzyme system.  This report shows the results of assay development, miniaturization, and inhibitor screening for these potential drug targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeIh8mQ6DnfbVg90H21EOLACvtfcHk0litWakcd8nCzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnvFChu7g%253D&md5=c72d4b37902699489b68b3e45ba9a59a</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1089%2Fadt.2007.105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fadt.2007.105%26sid%3Dliteratum%253Aachs%26aulast%3DChung%26aufirst%3DC.%2BC.%26aulast%3DOhwaki%26aufirst%3DK.%26aulast%3DSchneeweis%26aufirst%3DJ.%2BE.%26aulast%3DStec%26aufirst%3DE.%26aulast%3DVarnerin%26aufirst%3DJ.%2BP.%26aulast%3DGoudreau%26aufirst%3DP.%2BN.%26aulast%3DChang%26aufirst%3DA.%26aulast%3DCassaday%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DL.%26aulast%3DYamakawa%26aufirst%3DT.%26aulast%3DKornienko%26aufirst%3DO.%26aulast%3DHodder%26aufirst%3DP.%26aulast%3DInglese%26aufirst%3DJ.%26aulast%3DFerrer%26aufirst%3DM.%26aulast%3DStrulovici%26aufirst%3DB.%26aulast%3DKusunoki%26aufirst%3DJ.%26aulast%3DTota%26aufirst%3DM.%2BR.%26aulast%3DTakagi%26aufirst%3DT.%26atitle%3DA%2520fluorescence-based%2520thiol%2520quantification%2520assay%2520for%2520ultra-high-throughput%2520screening%2520for%2520inhibitors%2520of%2520coenzyme%2520A%2520production%26jtitle%3DAssay%2520Drug%2520Dev.%2520Technol.%26date%3D2008%26volume%3D6%26spage%3D361%26epage%3D374%26doi%3D10.1089%2Fadt.2007.105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goncalves, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brannigan, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thinon, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olaleye, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serwa, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanzarone, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tate, E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leatherbarrow, R. J.</span></span> <span> </span><span class="NLM_article-title">A fluorescence-based assay for N-myristoyltransferase activity</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>421</i></span>,  <span class="NLM_fpage">342</span>– <span class="NLM_lpage">344</span>, <span class="refDoi"> DOI: 10.1016/j.ab.2011.10.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1016%2Fj.ab.2011.10.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=22051857" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC38XptVehtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=421&publication_year=2012&pages=342-344&author=V.+Goncalvesauthor=J.+A.+Branniganauthor=E.+Thinonauthor=T.+O.+Olaleyeauthor=R.+Serwaauthor=S.+Lanzaroneauthor=A.+J.+Wilkinsonauthor=E.+W.+Tateauthor=R.+J.+Leatherbarrow&title=A+fluorescence-based+assay+for+N-myristoyltransferase+activity&doi=10.1016%2Fj.ab.2011.10.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">A fluorescence-based assay for N-myristoyltransferase activity</span></div><div class="casAuthors">Goncalves, Victor; Brannigan, James A.; Thinon, Emmanuelle; Olaleye, Tayo O.; Serwa, Remigiusz; Lanzarone, Salvatore; Wilkinson, Anthony J.; Tate, Edward W.; Leatherbarrow, Robin J.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">421</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">342-344</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">N-myristoylation is the irreversible attachment of a C14 fatty acid, myristic acid, to the N-terminal glycine of a protein via formation of an amide bond.  This modification is catalyzed by myristoyl-CoA:protein N-myristoyltransferase (NMT), an enzyme ubiquitous in eukaryotes that is up-regulated in several cancers.  Here we report a sensitive fluorescence-based assay to study the enzymic activity of human NMT1 and NMT2 based on detection of CoA by 7-diethylamino-3-(4-maleimidophenyl)-4-methylcoumarin (CPM).  We also describe expression and characterization of NMT1 and NMT2 and assay validation with small mol. inhibitors.  This assay should be broadly applicable to NMTs from a range of organisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP8ouiGX0Ml7Vg90H21EOLACvtfcHk0litWakcd8nCzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XptVehtg%253D%253D&md5=b0b36c043d6947a0843e19eddb9d9f43</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2011.10.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2011.10.013%26sid%3Dliteratum%253Aachs%26aulast%3DGoncalves%26aufirst%3DV.%26aulast%3DBrannigan%26aufirst%3DJ.%2BA.%26aulast%3DThinon%26aufirst%3DE.%26aulast%3DOlaleye%26aufirst%3DT.%2BO.%26aulast%3DSerwa%26aufirst%3DR.%26aulast%3DLanzarone%26aufirst%3DS.%26aulast%3DWilkinson%26aufirst%3DA.%2BJ.%26aulast%3DTate%26aufirst%3DE.%2BW.%26aulast%3DLeatherbarrow%26aufirst%3DR.%2BJ.%26atitle%3DA%2520fluorescence-based%2520assay%2520for%2520N-myristoyltransferase%2520activity%26jtitle%3DAnal.%2520Biochem.%26date%3D2012%26volume%3D421%26spage%3D342%26epage%3D344%26doi%3D10.1016%2Fj.ab.2011.10.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conrad, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuneo, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hellinga, H. W.</span></span> <span> </span><span class="NLM_article-title">Orthogonal site-specific protein modification by engineering reversible thiol protection mechanisms</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">64</span>– <span class="NLM_lpage">73</span>, <span class="refDoi"> DOI: 10.1110/ps.04965405</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1110%2Fps.04965405" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=15576565" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFWmsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2005&pages=64-73&author=J.+J.+Smithauthor=D.+W.+Conradauthor=M.+J.+Cuneoauthor=H.+W.+Hellinga&title=Orthogonal+site-specific+protein+modification+by+engineering+reversible+thiol+protection+mechanisms&doi=10.1110%2Fps.04965405"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Orthogonal site-specific protein modification by engineering reversible thiol protection mechanisms</span></div><div class="casAuthors">Smith, J. Jefferson; Conrad, David W.; Cuneo, Matthew J.; Hellinga, Homme W.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">64-73</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Covalent modification is an important strategy for introducing new functions into proteins.  As engineered proteins become more sophisticated, it is often desirable to introduce multiple, modifications involving several different functionalities in a site-specific manner.  Such orthogonal labeling schemes require independent labeling of differentially reactive nucleophilic amino acid side chains.  We have developed two protein-mediated protection schemes that permit independent labeling of multiple thiols.  These schemes exploit metal coordination or disulfide bond formation to reversibly protect cysteines in a Cys2His2 zinc finger domain.  We constructed a variety of N- and C-terminal fusions of these domains with maltose-binding protein, which were labeled with two or three different fluorophores.  Multiple modifications were made by reacting an unprotected cysteine in MBP first, deprotecting the zinc finger, and then reacting the zinc finger cysteines.  The fusion proteins were orthogonally labeled with two different fluorophores, which exhibited intramol. fluorescene resonance energy transfer (FRET).  These conjugates showed up to a threefold ratiometric change in emission intensities in response to maltose binding.  We also demonstrated that the metal- and redox-mediated protection methods can be combined to produce triple independent modifications, and prepd. a protein labeled with three different fluorophores that exhibited a FRET relay.  Finally, labeled glucose-binding protein was covalently patterned on glass slides using thiol-mediated immobilization chemistries.  Together, these expts. demonstrated that reversible thiol protection schemes provide a rapid, straightforward method for producing multiple, site-specific modifications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrGQzcYV345rVg90H21EOLACvtfcHk0litWakcd8nCzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFWmsg%253D%253D&md5=b411f4b18b090392e54fc4307701b2b7</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1110%2Fps.04965405&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1110%252Fps.04965405%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DConrad%26aufirst%3DD.%2BW.%26aulast%3DCuneo%26aufirst%3DM.%2BJ.%26aulast%3DHellinga%26aufirst%3DH.%2BW.%26atitle%3DOrthogonal%2520site-specific%2520protein%2520modification%2520by%2520engineering%2520reversible%2520thiol%2520protection%2520mechanisms%26jtitle%3DProtein%2520Sci.%26date%3D2005%26volume%3D14%26spage%3D64%26epage%3D73%26doi%3D10.1110%2Fps.04965405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Konopíková, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uher, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bransová, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mastihuba, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudecová, D.</span></span> <span> </span><span class="NLM_article-title">Reaction of 5-hydroxy-2-hydroxymethyl-4H-pyran-4-one with aromatic aldehydes and amines (betti reaction)</span>. <i>Chem. Pap.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">182</span>– <span class="NLM_lpage">184</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADyaK2MXjsV2gs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=1994&pages=182-184&author=M.+Konop%C3%ADkov%C3%A1author=M.+Uherauthor=J.+Bransov%C3%A1author=V.+Mastihubaauthor=D.+Hudecov%C3%A1&title=Reaction+of+5-hydroxy-2-hydroxymethyl-4H-pyran-4-one+with+aromatic+aldehydes+and+amines+%28betti+reaction%29"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Reaction of 5-hydroxy-2-hydroxymethyl-4H-pyran-4-one with aromatic aldehydes and amines (Betti reaction)</span></div><div class="casAuthors">Konopikova, M.; Uher, M.; Bransova, J.; Mastihuba, V.; Hudecova, D.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Papers</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">182-4</span>CODEN:
                <span class="NLM_cas:coden">CHPAEG</span>;
        ISSN:<span class="NLM_cas:issn">0366-6352</span>.
    
            (<span class="NLM_cas:orgname">Slovak Academy of Sciences, Institute of Chemistry</span>)
        </div><div class="casAbstract">Prepn. and spectral characteristics of 6-(α-anilinobenzyl) derivs. of 5-hydroxy-2-hydroxymethyl-4H-pyran-4-one (kojic acid) were described.  Some products of Betti reaction are thermolabile and decomp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrf3Nr0h_MynbVg90H21EOLACvtfcHk0ljzQiEr0r3bbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXjsV2gs7Y%253D&md5=38ab714dc3e7a702956d6349821a9cdd</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKonop%25C3%25ADkov%25C3%25A1%26aufirst%3DM.%26aulast%3DUher%26aufirst%3DM.%26aulast%3DBransov%25C3%25A1%26aufirst%3DJ.%26aulast%3DMastihuba%26aufirst%3DV.%26aulast%3DHudecov%25C3%25A1%26aufirst%3DD.%26atitle%3DReaction%2520of%25205-hydroxy-2-hydroxymethyl-4H-pyran-4-one%2520with%2520aromatic%2520aldehydes%2520and%2520amines%2520%2528betti%2520reaction%2529%26jtitle%3DChem.%2520Pap.%26date%3D1994%26volume%3D48%26spage%3D182%26epage%3D184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burnett, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergfeld, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belsito, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klaassen, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liebler, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marks, J. G.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shank, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slaga, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snyder, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, F. A.</span></span> <span> </span><span class="NLM_article-title">Final report of the safety assessment of kojic acid as used in cosmetics</span>. <i>Int. J. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">244S</span>– <span class="NLM_lpage">273S</span>, <span class="refDoi"> DOI: 10.1177/1091581810385956</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1177%2F1091581810385956" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=21164073" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtleiu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2010&pages=244S-273S&author=C.+L.+Burnettauthor=W.+F.+Bergfeldauthor=D.+V.+Belsitoauthor=R.+A.+Hillauthor=C.+D.+Klaassenauthor=D.+C.+Lieblerauthor=J.+G.+Marksauthor=R.+C.+Shankauthor=T.+J.+Slagaauthor=P.+W.+Snyderauthor=F.+A.+Andersen&title=Final+report+of+the+safety+assessment+of+kojic+acid+as+used+in+cosmetics&doi=10.1177%2F1091581810385956"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Final report of the safety assessment of kojic acid as used in cosmetics</span></div><div class="casAuthors">Burnett, Christina L.; Bergfeld, Wilma F.; Belsito, Donald V.; Hill, Ronald A.; Klaassen, Curtis D.; Liebler, Daniel C.; Marks, James G., Jr.; Shank, Ronald C.; Slaga, Thomas J.; Snyder, Paul W.; Andersen, F. Alan</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Toxicology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">Suppl. 4</span>),
    <span class="NLM_cas:pages">244S-273S</span>CODEN:
                <span class="NLM_cas:coden">IJTOFN</span>;
        ISSN:<span class="NLM_cas:issn">1091-5818</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A review.  Kojic acid functions as an antioxidant in cosmetic products.  Kojic acid was not a toxicant in acute, chronic, reproductive, and genotoxicity studies.  While some animal data suggested tumor promotion and weak carcinogenicity, kojic acid is slowly absorbed into the circulation from human skin and likely would not reach the threshold at which these effects were seen.  The available human sensitization data supported the safety of kojic acid at a use concn. of 2% in leave-on cosmetics.  Kojic acid depigmented black guinea pig skin at a concn. of 4%, but this effect was not seen at 1%.  The Cosmetic Ingredient Review (CIR) Expert Panel concluded that the 2 end points of concern, dermal sensitization and skin lightening, would not be seen at use concns. below 1%; therefore, this ingredient is safe for use in cosmetic products up to that level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmrJeP-lFMi7Vg90H21EOLACvtfcHk0ljzQiEr0r3bbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtleiu7w%253D&md5=6285c4c02f9f18eaa6091d2f57b2383a</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1177%2F1091581810385956&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1091581810385956%26sid%3Dliteratum%253Aachs%26aulast%3DBurnett%26aufirst%3DC.%2BL.%26aulast%3DBergfeld%26aufirst%3DW.%2BF.%26aulast%3DBelsito%26aufirst%3DD.%2BV.%26aulast%3DHill%26aufirst%3DR.%2BA.%26aulast%3DKlaassen%26aufirst%3DC.%2BD.%26aulast%3DLiebler%26aufirst%3DD.%2BC.%26aulast%3DMarks%26aufirst%3DJ.%2BG.%26aulast%3DShank%26aufirst%3DR.%2BC.%26aulast%3DSlaga%26aufirst%3DT.%2BJ.%26aulast%3DSnyder%26aufirst%3DP.%2BW.%26aulast%3DAndersen%26aufirst%3DF.%2BA.%26atitle%3DFinal%2520report%2520of%2520the%2520safety%2520assessment%2520of%2520kojic%2520acid%2520as%2520used%2520in%2520cosmetics%26jtitle%3DInt.%2520J.%2520Toxicol.%26date%3D2010%26volume%3D29%26spage%3D244S%26epage%3D273S%26doi%3D10.1177%2F1091581810385956" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, E. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poulsen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pobbati, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, C.</span></span> <span> </span><span class="NLM_article-title">Structural and ligand-binding analysis of the YAP-binding domain of transcription factor TEAD4</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>475</i></span>,  <span class="NLM_fpage">2043</span>– <span class="NLM_lpage">2055</span>, <span class="refDoi"> DOI: 10.1042/bcj20180225</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1042%2FBCJ20180225" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=29760238" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlKqtbjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=475&publication_year=2018&pages=2043-2055&author=Y.+Liauthor=S.+Liuauthor=E.+Y.+Ngauthor=R.+Liauthor=A.+Poulsenauthor=J.+Hillauthor=A.+V.+Pobbatiauthor=A.+W.+Hungauthor=W.+Hongauthor=T.+H.+Kellerauthor=C.+Kang&title=Structural+and+ligand-binding+analysis+of+the+YAP-binding+domain+of+transcription+factor+TEAD4&doi=10.1042%2Fbcj20180225"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and ligand-binding analysis of the YAP-binding domain of transcription factor TEAD4</span></div><div class="casAuthors">Li, Yan; Liu, Shuang; Ng, Elizabeth Yihui; Li, Rong; Poulsen, Anders; Hill, Jeffrey; Pobbati, Ajaybabu V.; Hung, Alvin W.; Hong, Wanjin; Keller, Thomas H.; Kang, CongBao</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">475</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2043-2055</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">The oncoprotein YAP (Yes-assocd. protein) requires the TEAD family of transcription factors for the up-regulation of genes important for cell proliferation.  Disrupting YAP-TEAD interaction is an attractive strategy for cancer therapy.  Targeting TEADs using small mols. that either bind to the YAP-binding pocket or the palmitate-binding pocket is proposed to disrupt the YAP-TEAD interaction.  There is a need for methodologies to facilitate robust and reliable identification of compds. that occupy either YAP-binding pocket or palmitate-binding pocket.  Here, using NMR spectroscopy, we validated compds. that bind to these pockets and also identify the residues in mouse TEAD4 (mTEAD4) that interact with these compds.  Flufenamic acid (FA) was used as a pos. control for validation of palmitate-binding pocket-occupying compds. by NMR.  Furthermore, we identify a hit from a fragment screen and show that it occupies a site close to YAP-binding pocket on the TEAD surface.  Our results also indicate that purified mTEAD4 can catalyze autopalmitoylation.  NMR studies on mTEAD4 revealed that exchanges exist in TEAD as NMR signal broadening was obsd. for residues close to the palmitoylation site.  Mutating the palmitoylated cysteine (C360S mutant) abolished palmitoylation, while no significant changes in the NMR spectrum were obsd. for the mutant which still binds to YAP.  We also show that FA inhibits TEAD autopalmitoylation.  Our studies highlight the utility of NMR spectroscopy in identifying small mols. that bind to TEAD pockets and reinforce the notion that both palmitate-binding pocket and YAP-binding pocket are targetable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoq8Fg59GWiSLVg90H21EOLACvtfcHk0ljzQiEr0r3bbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlKqtbjL&md5=9197b0611f10a4b5f6b82f73afa04f35</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1042%2FBCJ20180225&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBCJ20180225%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DNg%26aufirst%3DE.%2BY.%26aulast%3DLi%26aufirst%3DR.%26aulast%3DPoulsen%26aufirst%3DA.%26aulast%3DHill%26aufirst%3DJ.%26aulast%3DPobbati%26aufirst%3DA.%2BV.%26aulast%3DHung%26aufirst%3DA.%2BW.%26aulast%3DHong%26aufirst%3DW.%26aulast%3DKeller%26aufirst%3DT.%2BH.%26aulast%3DKang%26aufirst%3DC.%26atitle%3DStructural%2520and%2520ligand-binding%2520analysis%2520of%2520the%2520YAP-binding%2520domain%2520of%2520transcription%2520factor%2520TEAD4%26jtitle%3DBiochem.%2520J.%26date%3D2018%26volume%3D475%26spage%3D2043%26epage%3D2055%26doi%3D10.1042%2Fbcj20180225" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laraia, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friese, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corkery, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konstantinidis, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erwin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofer, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karatas, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klewer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brockmeyer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schölermann, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dwivedi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rios-Munoz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Köhn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vetter, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziegler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janning, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span> <span> </span><span class="NLM_article-title">The cholesterol transfer protein GRAMD1A regulates autophagosome biogenesis</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">710</span>– <span class="NLM_lpage">720</span>, <span class="refDoi"> DOI: 10.1038/s41589-019-0307-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1038%2Fs41589-019-0307-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=31222192" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1aqtbvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=710-720&author=L.+Laraiaauthor=A.+Frieseauthor=D.+P.+Corkeryauthor=G.+Konstantinidisauthor=N.+Erwinauthor=W.+Hoferauthor=H.+Karatasauthor=L.+Klewerauthor=A.+Brockmeyerauthor=M.+Metzauthor=B.+Sch%C3%B6lermannauthor=M.+Dwivediauthor=L.+Liauthor=P.+Rios-Munozauthor=M.+K%C3%B6hnauthor=R.+Winterauthor=I.+R.+Vetterauthor=S.+Zieglerauthor=P.+Janningauthor=Y.-W.+Wuauthor=H.+Waldmann&title=The+cholesterol+transfer+protein+GRAMD1A+regulates+autophagosome+biogenesis&doi=10.1038%2Fs41589-019-0307-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">The cholesterol transfer protein GRAMD1A regulates autophagosome biogenesis</span></div><div class="casAuthors">Laraia, Luca; Friese, Alexandra; Corkery, Dale P.; Konstantinidis, Georgios; Erwin, Nelli; Hofer, Walter; Karatas, Hacer; Klewer, Laura; Brockmeyer, Andreas; Metz, Malte; Schoelermann, Beate; Dwivedi, Mridula; Li, Lei; Rios-Munoz, Pablo; Koehn, Maja; Winter, Roland; Vetter, Ingrid R.; Ziegler, Slava; Janning, Petra; Wu, Yao-Wen; Waldmann, Herbert</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">710-720</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Autophagy mediates the degrdn. of damaged proteins, organelles and pathogens, and plays a key role in health and disease.  Thus, the identification of new mechanisms involved in the regulation of autophagy is of major interest.  In particular, little is known about the role of lipids and lipid-binding proteins in the early steps of autophagosome biogenesis.  Using target-agnostic, high-content, image-based identification of indicative phenotypic changes induced by small mols., we have identified autogramins as a new class of autophagy inhibitor.  Autogramins selectively target the recently discovered cholesterol transfer protein GRAM domain-contg. protein 1A (GRAMD1A, which had not previously been implicated in autophagy), and directly compete with cholesterol binding to the GRAMD1A StART domain.  GRAMD1A accumulates at sites of autophagosome initiation, affects cholesterol distribution in response to starvation and is required for autophagosome biogenesis.  These findings identify a new biol. function of GRAMD1A and a new role for cholesterol in autophagy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZaJquw1e9_bVg90H21EOLACvtfcHk0lioF1UDlIicog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1aqtbvI&md5=ff7841edc7f22d611d6e681da5d74d0f</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fs41589-019-0307-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-019-0307-5%26sid%3Dliteratum%253Aachs%26aulast%3DLaraia%26aufirst%3DL.%26aulast%3DFriese%26aufirst%3DA.%26aulast%3DCorkery%26aufirst%3DD.%2BP.%26aulast%3DKonstantinidis%26aufirst%3DG.%26aulast%3DErwin%26aufirst%3DN.%26aulast%3DHofer%26aufirst%3DW.%26aulast%3DKaratas%26aufirst%3DH.%26aulast%3DKlewer%26aufirst%3DL.%26aulast%3DBrockmeyer%26aufirst%3DA.%26aulast%3DMetz%26aufirst%3DM.%26aulast%3DSch%25C3%25B6lermann%26aufirst%3DB.%26aulast%3DDwivedi%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DRios-Munoz%26aufirst%3DP.%26aulast%3DK%25C3%25B6hn%26aufirst%3DM.%26aulast%3DWinter%26aufirst%3DR.%26aulast%3DVetter%26aufirst%3DI.%2BR.%26aulast%3DZiegler%26aufirst%3DS.%26aulast%3DJanning%26aufirst%3DP.%26aulast%3DWu%26aufirst%3DY.-W.%26aulast%3DWaldmann%26aufirst%3DH.%26atitle%3DThe%2520cholesterol%2520transfer%2520protein%2520GRAMD1A%2520regulates%2520autophagosome%2520biogenesis%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D15%26spage%3D710%26epage%3D720%26doi%3D10.1038%2Fs41589-019-0307-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Medina-O’Donnell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivas, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reyes-Zurita, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin-Fonseca, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Granados, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrer-Martín, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lupiañez, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parra, A.</span></span> <span> </span><span class="NLM_article-title">Semi-synthesis and antiproliferative evaluation of PEGylated pentacyclic triterpenes</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">64</span>– <span class="NLM_lpage">78</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2016.04.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=10.1016%2Fj.ejmech.2016.04.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=27128174" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;key=1%3ACAS%3A528%3ADC%252BC28XnslWqtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2016&pages=64-78&author=M.+Medina-O%E2%80%99Donnellauthor=F.+Rivasauthor=F.+J.+Reyes-Zuritaauthor=A.+Martinezauthor=S.+Martin-Fonsecaauthor=A.+Garcia-Granadosauthor=R.+M.+Ferrer-Mart%C3%ADnauthor=J.+A.+Lupia%C3%B1ezauthor=A.+Parra&title=Semi-synthesis+and+antiproliferative+evaluation+of+PEGylated+pentacyclic+triterpenes&doi=10.1016%2Fj.ejmech.2016.04.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Semi-synthesis and antiproliferative evaluation of PEGylated pentacyclic triterpenes</span></div><div class="casAuthors">Medina-O'Donnell, Marta; Rivas, Francisco; Reyes-Zurita, Fernando J.; Martinez, Antonio; Martin-Fonseca, Samuel; Garcia-Granados, Andres; Ferrer-Martin, Rosa M.; Lupianez, Jose A.; Parra, Andres</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">64-78</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Several PEGylated derivs. of oleanolic and maslinic acids have been semi-synthesized, attaching one acid-PEG reagent to the hydroxyl group/s at C-2 or C-2/C-3 of the A rings of these natural triterpenes, and also to their corresponding C-28 benzyl derivs.  Several monomeric and dimeric PEGylated compds. have also been produced by linking one diamine-PEG reagent to the carboxyl group at C-28 of the same natural triterpenes and also to their corresponding C-2 or C-2/C-3 acetylated derivs.  The cytotoxic effects of 12 triterpenic PEGylated derivs. in three cancer-cell lines (B16-F10, HT29, and Hep G2) have been assayed.  The best results have been achieved by the PEGylated-amine deriv. of oleanolic acid, with IC50 concns. between 0.22 and 3.78 μM, being between 28- and 963-fold more effective than its natural precursor.  The percentages of apoptosis induction have also been detd. for the five PEGylated derivs. with the lowest cytotoxicity data.  All five compds. showed apoptotic effects on the treated cells, with a total apoptosis rate of 99% in the B16-F10 cells, 80% in the Hep G2 cells, and 51% in the HT29 cells.  We have also studied the changes in the mitochondrial membrane potential (MMP) to elucidate the possible mechanism involved in the apoptotic responses (intrinsic or extrinsic).  Finally, to verify the results found in the cytometry assays, we have used fluorescence microscopy techniques to det. changes in the cell morphol.  These PEGylated derivs. of natural triterpenoids, which can induce apoptosis at very low concns. in different tumor lines, may represent new effective therapeutic drugs against these diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBNU2AmdXdsrVg90H21EOLACvtfcHk0lioF1UDlIicog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnslWqtbw%253D&md5=9ff4513f20931132654c5a70a3ebc57a</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2016.04.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2016.04.016%26sid%3Dliteratum%253Aachs%26aulast%3DMedina-O%25E2%2580%2599Donnell%26aufirst%3DM.%26aulast%3DRivas%26aufirst%3DF.%26aulast%3DReyes-Zurita%26aufirst%3DF.%2BJ.%26aulast%3DMartinez%26aufirst%3DA.%26aulast%3DMartin-Fonseca%26aufirst%3DS.%26aulast%3DGarcia-Granados%26aufirst%3DA.%26aulast%3DFerrer-Mart%25C3%25ADn%26aufirst%3DR.%2BM.%26aulast%3DLupia%25C3%25B1ez%26aufirst%3DJ.%2BA.%26aulast%3DParra%26aufirst%3DA.%26atitle%3DSemi-synthesis%2520and%2520antiproliferative%2520evaluation%2520of%2520PEGylated%2520pentacyclic%2520triterpenes%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2016%26volume%3D118%26spage%3D64%26epage%3D78%26doi%3D10.1016%2Fj.ejmech.2016.04.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5OAQ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5OAQ','PDB','5OAQ'); return false;">PDB: 5OAQ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3KYS" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3KYS','PDB','3KYS'); return false;">PDB: 3KYS</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i79"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01275">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_11181"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c01275?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01275</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Full-protein ESI–MS analysis of recombinant hTEAD4 (acylated), hTEAD4 (nonacylated), and hTEAD4 (nonacylated) reaction with CPM; tandem-MS analysis of the hTEAD4 (nonacylated) reaction with CPM; competitive inhibition of the thiol conjugation signal with TED-347; saturation binding curve of FITC-palmitate to hTEAD4 (nonacylated) and competitive inhibition curve of niflumic acid (<b>2</b>) in the FP assay; time- and concentration-dependent inhibition of hTEAD4; specificity of <b>19</b> for TEAD homologues; <sup>1</sup>H–<sup>15</sup>N HSQC spectrum of mTEAD4 (nonacylated) treated with <b>19</b> or <b>45</b>; saturation binding curves of FITC-labeled YAP<sup>50–100</sup> with TEAD4 proteins; sequences of the TEAD proteins used in the study; scheme for the synthesis of FITC-palmitate; and <sup>1</sup>H NMR, <sup>13</sup>C NMR, and UPLC spectra of compounds <b>19</b> and <b>45</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01275/suppl_file/jm0c01275_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings with biological data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01275/suppl_file/jm0c01275_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01275/suppl_file/jm0c01275_si_001.pdf">jm0c01275_si_001.pdf (1.17 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01275/suppl_file/jm0c01275_si_002.csv">jm0c01275_si_002.csv (3.89 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c01275&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c01275%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-20%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c01275" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679ac21a0cc52290","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
