
<html lang="en"     class="pb-page"  data-request-id="155247a6-f88a-4a12-8b68-4594a4c0a81e"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/jm300280e;issue:issue:10.1021/jmcmar.2012.55.issue-11;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="A Prospective Cross-Screening Study on G-Protein-Coupled Receptors: Lessons Learned in Virtual Compound Library Design" /></meta><meta name="dc.Creator" content="Marijn P. A.  Sanders" /></meta><meta name="dc.Creator" content="Luc  Roumen" /></meta><meta name="dc.Creator" content="Eelke  van der Horst" /></meta><meta name="dc.Creator" content="J. Robert  Lane" /></meta><meta name="dc.Creator" content="Henry F.  Vischer" /></meta><meta name="dc.Creator" content="Jody  van Offenbeek" /></meta><meta name="dc.Creator" content="Henk  de Vries" /></meta><meta name="dc.Creator" content="Stefan  Verhoeven" /></meta><meta name="dc.Creator" content="Ken Y.  Chow" /></meta><meta name="dc.Creator" content="Folkert  Verkaar" /></meta><meta name="dc.Creator" content="Margot W.  Beukers" /></meta><meta name="dc.Creator" content="Ross  McGuire" /></meta><meta name="dc.Creator" content="Rob  Leurs" /></meta><meta name="dc.Creator" content="Adriaan P.  IJzerman" /></meta><meta name="dc.Creator" content="Jacob  de Vlieg" /></meta><meta name="dc.Creator" content="Iwan J. P.  de Esch" /></meta><meta name="dc.Creator" content="Guido J. R.  Zaman" /></meta><meta name="dc.Creator" content="Jan P. G.  Klomp" /></meta><meta name="dc.Creator" content="Andreas  Bender" /></meta><meta name="dc.Creator" content="Chris  de Graaf" /></meta><meta name="dc.Description" content="We present the systematic prospective evaluation of a protein-based and a ligand-based virtual screening platform against a set of three G-protein-coupled receptors (GPCRs): the β-2 adrenoreceptor ..." /></meta><meta name="Description" content="We present the systematic prospective evaluation of a protein-based and a ligand-based virtual screening platform against a set of three G-protein-coupled receptors (GPCRs): the β-2 adrenoreceptor ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="May 23, 2012" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm300280e" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2012 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm300280e" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm300280e" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm300280e" /></link>
        
    
    

<title>A Prospective Cross-Screening Study on G-Protein-Coupled Receptors: Lessons Learned in Virtual Compound Library Design | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm300280e" /></meta><meta property="og:title" content="A Prospective Cross-Screening Study on G-Protein-Coupled Receptors: Lessons Learned in Virtual Compound Library Design" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/large/jm-2012-00280e_0013.jpeg" /></meta><meta property="og:description" content="We present the systematic prospective evaluation of a protein-based and a ligand-based virtual screening platform against a set of three G-protein-coupled receptors (GPCRs): the β-2 adrenoreceptor (ADRB2), the adenosine A2A receptor (AA2AR), and the sphingosine 1-phosphate receptor (S1PR1). Novel bioactive compounds were identified using a consensus scoring procedure combining ligand-based (frequent substructure ranking) and structure-based (Snooker) tools, and all 900 selected compounds were screened against all three receptors. A striking number of ligands showed affinity/activity for GPCRs other than the intended target, which could be partly attributed to the fuzziness and overlap of protein-based pharmacophore models. Surprisingly, the phosphodiesterase 5 (PDE5) inhibitor sildenafil was found to possess submicromolar affinity for AA2AR. Overall, this is one of the first published prospective chemogenomics studies that demonstrate the identification of novel cross-pharmacology between unrelated protein targets. The lessons learned from this study can be used to guide future virtual ligand design efforts." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm300280e"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm300280e">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm300280e&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm300280e&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm300280e&amp;href=/doi/10.1021/jm300280e" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2012</span><span class="cit-fg-volume">, 55</span><span class="cit-fg-issue">, 11</span><span class="cit-fg-pageRange">, 5311-5325</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/55/11" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm300276x" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm3002857" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">A Prospective Cross-Screening Study on G-Protein-Coupled Receptors: Lessons Learned in Virtual Compound Library Design</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Marijn+P.+A.++Sanders">Marijn P. A. Sanders</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Luc++Roumen">Luc Roumen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Eelke++van+der+Horst">Eelke van der Horst</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=J.+Robert++Lane">J. Robert Lane</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Henry+F.++Vischer">Henry F. Vischer</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jody++van+Offenbeek">Jody van Offenbeek</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Henk++de+Vries">Henk de Vries</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Stefan++Verhoeven">Stefan Verhoeven</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ken+Y.++Chow">Ken Y. Chow</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Folkert++Verkaar">Folkert Verkaar</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Margot+W.++Beukers">Margot W. Beukers</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ross++McGuire">Ross McGuire</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Rob++Leurs">Rob Leurs</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Adriaan+P.++IJzerman">Adriaan P. IJzerman</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jacob++de+Vlieg">Jacob de Vlieg</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Iwan+J.+P.++de+Esch">Iwan J. P. de Esch</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Guido+J.+R.++Zaman">Guido J. R. Zaman</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jan+P.+G.++Klomp">Jan P. G. Klomp</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Andreas++Bender">Andreas Bender</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Chris++de+Graaf">Chris de Graaf</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Computational Drug Discovery Group, Radboud University Nijmegen Medical Centre, Geert Grooteplein, Nijmegen, The Netherlands</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Leiden/Amsterdam Center of Drug Research (LACDR), Division of Medicinal Chemistry, Faculty of Sciences, VU University Amsterdam, De Boelelaan 1083, 1081 HV Amsterdam, The Netherlands</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">Leiden/Amsterdam Center of Drug Research (LACDR), Division of Medicinal Chemistry, Leiden University, Einsteinweg 55, 2333 CC Leiden, The Netherlands</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">∥</span> <span class="aff-text">Department of Molecular Pharmacology and DMPK, MSD, Molenstraat 110, 5342 CC Oss, The Netherlands</span></div><div class="aff-info" id="aff5"><span class="aff-symbol">⊥</span> <span class="aff-text">Department of Molecular Design and Informatics, MSD, Molenstraat 110, 5342 CC Oss, The Netherlands</span></div><div class="corresp-info"><strong>*</strong>For J.P.G.K.: phone, 0031 243505766; e-mail, <a href="/cdn-cgi/l/email-protection#076d6669296c6b686a77476b626663776f66756a662964686a"><span class="__cf_email__" data-cfemail="3b515a5515505754564b7b575e5a5f4b535a49565a15585456">[email protected]</span></a>. For A.B.: phone, 0044 1223 762983; e-mail, <a href="/cdn-cgi/l/email-protection#d697b8b2a4b3b7a5f894b3b8b2b3a496b5b7b8a2b7b4f8b8b3a2"><span class="__cf_email__" data-cfemail="30715e54425551431e72555e5455427053515e4451521e5e5544">[email protected]</span></a>. For C.d.G.: phone, 0031 20 5987553; e-mail, <a href="/cdn-cgi/l/email-protection#f6b5d89293d8b184979790b68083d8989a"><span class="__cf_email__" data-cfemail="92d1bcf6f7bcd5e0f3f3f4d2e4e7bcfcfe">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm300280e&amp;href=/doi/10.1021%2Fjm300280e" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2012</span></span><span class="cit-volume">, 55</span><span class="cit-issue">, 11</span><span class="cit-pageRange">, 5311–5325</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">May 7, 2012</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>28 February 2012</li><li><span class="item_label"><b>Published</b> online</span>23 May 2012</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 14 June 2012</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm300280e" title="DOI URL">https://doi.org/10.1021/jm300280e</a></div><div class="article_header-article-copyright"><strong>Copyright © 2012 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D5311%26pageCount%3D15%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DMarijn%2BP.%2BA.%2BSanders%252C%2BLuc%2BRoumen%252C%2BEelke%2Bvan%2Bder%2BHorst%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D55%26issueNum%3D11%26contentID%3Djm300280e%26title%3DA%2BProspective%2BCross-Screening%2BStudy%2Bon%2BG-Protein-Coupled%2BReceptors%253A%2BLessons%2BLearned%2Bin%2BVirtual%2BCompound%2BLibrary%2BDesign%26numPages%3D15%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D5325%26publicationDate%3DJune%2B2012">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm300280e"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1950</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">24</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm300280e" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;A Prospective Cross-Screening Study on G-Protein-Coupled Receptors: Lessons Learned in Virtual Compound Library Design&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Marijn&quot;,&quot;last_name&quot;:&quot;P. A. Sanders&quot;},{&quot;first_name&quot;:&quot;Luc&quot;,&quot;last_name&quot;:&quot;Roumen&quot;},{&quot;first_name&quot;:&quot;Eelke&quot;,&quot;last_name&quot;:&quot;van der Horst&quot;},{&quot;first_name&quot;:&quot;J.&quot;,&quot;last_name&quot;:&quot;Robert Lane&quot;},{&quot;first_name&quot;:&quot;Henry&quot;,&quot;last_name&quot;:&quot;F. Vischer&quot;},{&quot;first_name&quot;:&quot;Jody&quot;,&quot;last_name&quot;:&quot;van Offenbeek&quot;},{&quot;first_name&quot;:&quot;Henk&quot;,&quot;last_name&quot;:&quot;de Vries&quot;},{&quot;first_name&quot;:&quot;Stefan&quot;,&quot;last_name&quot;:&quot;Verhoeven&quot;},{&quot;first_name&quot;:&quot;Ken&quot;,&quot;last_name&quot;:&quot;Y. Chow&quot;},{&quot;first_name&quot;:&quot;Folkert&quot;,&quot;last_name&quot;:&quot;Verkaar&quot;},{&quot;first_name&quot;:&quot;Margot&quot;,&quot;last_name&quot;:&quot;W. Beukers&quot;},{&quot;first_name&quot;:&quot;Ross&quot;,&quot;last_name&quot;:&quot;McGuire&quot;},{&quot;first_name&quot;:&quot;Rob&quot;,&quot;last_name&quot;:&quot;Leurs&quot;},{&quot;first_name&quot;:&quot;Adriaan&quot;,&quot;last_name&quot;:&quot;P. IJzerman&quot;},{&quot;first_name&quot;:&quot;Jacob&quot;,&quot;last_name&quot;:&quot;de Vlieg&quot;},{&quot;first_name&quot;:&quot;Iwan&quot;,&quot;last_name&quot;:&quot;J. P. de Esch&quot;},{&quot;first_name&quot;:&quot;Guido&quot;,&quot;last_name&quot;:&quot;J. R. Zaman&quot;},{&quot;first_name&quot;:&quot;Jan&quot;,&quot;last_name&quot;:&quot;P. G. Klomp&quot;},{&quot;first_name&quot;:&quot;Andreas&quot;,&quot;last_name&quot;:&quot;Bender&quot;},{&quot;first_name&quot;:&quot;Chris&quot;,&quot;last_name&quot;:&quot;de Graaf&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2012&quot;,&quot;month&quot;:&quot;05&quot;,&quot;day&quot;:&quot;23&quot;,&quot;issue&quot;:&quot;11&quot;,&quot;volume&quot;:&quot;55&quot;,&quot;pages&quot;:&quot;5311-5325&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm300280e&quot;},&quot;abstract&quot;:&quot;We present the systematic prospective evaluation of a protein-based and a ligand-based virtual screening platform against a set of three G-protein-coupled receptors (GPCRs): the β-2 adrenoreceptor (ADRB2), the adenosine A2A receptor (AA2AR), and the sphingosine 1-phosphate receptor (S1PR1). Novel bioactive compounds were identified using a consensus scoring procedure combining ligand-based (frequent substructure ranking) and structure-based (Snooker) tools, and all 900 selected compounds were screened against all three receptors. A striking number of ligands showed affinity/activity for GPCRs other than the intended target, which could be partly attributed to the fuzziness and overlap of protein-based pharmacophore models. Surprisingly, the phosphodiesterase 5 (PDE5) inhibitor sildenafil was found to possess submicromolar affinity for AA2AR. Overall, this is one of the first published prospective chemogenomics studies that demonstrate the identification of novel cross-pharmacology between unrelated protein ta&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm300280e&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm300280e" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm300280e&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm300280e" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm300280e&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm300280e" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm300280e&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm300280e&amp;href=/doi/10.1021/jm300280e" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm300280e" /></input><a href="/doi/pdf/10.1021/jm300280e" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm300280e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm300280e%26sid%3Dliteratum%253Aachs%26pmid%3D22563707%26genre%3Darticle%26aulast%3DSanders%26date%3D2012%26atitle%3DA%2BProspective%2BCross-Screening%2BStudy%2Bon%2BG-Protein-Coupled%2BReceptors%253A%2BLessons%2BLearned%2Bin%2BVirtual%2BCompound%2BLibrary%2BDesign%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D55%26issue%3D11%26spage%3D5311%26epage%3D5325%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290663" title="Pharmacophores">Pharmacophores</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291880" title="Crystal structure">Crystal structure</a>,</li><li><a href="/action/doSearch?ConceptID=292202" title="Drug discovery">Drug discovery</a>,</li><li><a href="/action/doSearch?ConceptID=292186" title="Receptors">Receptors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/55/11" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jmcmar.2012.55.issue-11/production/jmcmar.2012.55.issue-11.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/medium/jm-2012-00280e_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/large/jm-2012-00280e_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300280e&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">We present the systematic prospective evaluation of a protein-based and a ligand-based virtual screening platform against a set of three G-protein-coupled receptors (GPCRs): the β-2 adrenoreceptor (ADRB2), the adenosine A<sub>2A</sub> receptor (AA2AR), and the sphingosine 1-phosphate receptor (S1PR1). Novel bioactive compounds were identified using a consensus scoring procedure combining ligand-based (frequent substructure ranking) and structure-based (Snooker) tools, and all 900 selected compounds were screened against all three receptors. A striking number of ligands showed affinity/activity for GPCRs other than the intended target, which could be partly attributed to the fuzziness and overlap of protein-based pharmacophore models. Surprisingly, the phosphodiesterase 5 (PDE5) inhibitor sildenafil was found to possess submicromolar affinity for AA2AR. Overall, this is one of the first published prospective chemogenomics studies that demonstrate the identification of novel cross-pharmacology between unrelated protein targets. The lessons learned from this study can be used to guide future virtual ligand design efforts.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div class="extra-info-sec articleNote"><h4><span>#</span> Author Contributions</h4><p class="last">These authors contributed equally to this work.</p></div><div class="extra-info-sec articleNote"><h4><span>∞</span> Author Present Address</h4><p class="last">Netherlands eScience Center, Science Park 140, 1098XG, Amsterdam, The Netherlands.</p></div><div class="extra-info-sec articleNote"><h4><span>×</span> Author Present Address</h4><p class="last">Bioinformatics Laboratory, Clinical Epidemiology, Biostatistics and Bioinformatics (KEBB), Academic Medical Center (AMC), Meibergdreef 9, J1B-207, 1100 DE Amsterdam, The Netherlands.</p></div><div class="extra-info-sec articleNote"><h4><span>△</span> Author Present Address</h4><p class="last">Drug Discovery Biology, Department of Pharmacy and Pharmaceutical Sciences, Monash University, Parkville, Victoria 3052, Australia.</p></div><div class="extra-info-sec articleNote"><h4><span>○</span> Author Present Address</h4><p class="last">BioAxis Research BV, Bergse Heihoek 56, 5351 SL, Berghem, The Netherlands.</p></div><div class="extra-info-sec articleNote"><h4><span>●</span> Author Present Address</h4><p class="last">Netherlands Translational Research Center B.V., P.O. Box 280, 5340 AG Oss, The Netherlands.</p></div><div class="extra-info-sec articleNote"><h4><span>◆</span> Author Present Address</h4><p class="last">Lead Pharma Medicine, Kapittelweg 29, 6525 EN Nijmegen, The Netherlands.</p></div><div class="extra-info-sec articleNote"><h4><span>◇</span> Author Present Address</h4><p class="last">Unilever Centre for Molecular Informatics, Department of Chemistry, University of Cambridge, Cambridge CB2 1EW, United Kingdom.</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i9">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_32344" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_32344" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Chemogenomics is a relatively novel research area aimed at systematically studying the biological effect of a large number of small molecules (ligands) on a large number of macromolecular targets (gene products).<a onclick="showRef(event, 'ref1 ref2 ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4">(1-4)</a> Since experimental measurements of a large number of ligand–target interactions are time-consuming and cost intensive, they are often complemented by high-throughput in silico chemogenomics approaches. These virtual screening methods are typically divided into ligand-based and target-based approaches, and both approaches can be employed to profile either a ligand against a set of proteins or a set of ligands against one specific protein target.<a onclick="showRef(event, 'ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref5 ref6">(5, 6)</a> Both methods have been successfully applied in the drug discovery context<a onclick="showRef(event, 'ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref5 ref6">(5, 6)</a> with the aim to guide the development of a compound with the desired bioactivity profile.<a onclick="showRef(event, 'ref7 ref8 ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref7 ref8 ref9 ref10">(7-10)</a></div><div class="NLM_p">What needs to be kept in mind is that the applicability of in silico chemogenomics models depends on the quality and completeness of the training sets that are used for model construction and validation.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Bioactivity data of small molecules in particular are often incomplete, since molecules are not usually screened systematically against a large panel of protein targets but often only on a select set comprising the target of interest and a limited number of proteins over which selectivity needs to be achieved.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Furthermore, most scientific studies focus on the presentation of “active” molecules and (potentially) “inactive” molecules are often not reported, leading to a selection bias of published bioactivity data.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a>Even in annotated ligand databases such as ChEMBLdb,<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> DrugBank,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> BindingDB,<a onclick="showRef(event, 'ref15 ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref15 ref16 ref17">(15-17)</a> PDBBind,<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18, 19)</a> MOAD,<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20, 21)</a>WOMBAT,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> and GLIDA<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> protein–ligand interaction matrices are incomplete.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a></div><div class="NLM_p">Computational methods have, however, been successfully used to fill the gap in experimental ligand–target affinity matrices<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> and to identify new drug–target associations.<a onclick="showRef(event, 'ref5 ref25'); return false;" href="javascript:void(0);" class="ref ref5 ref25">(5, 25)</a> From generalization of this analysis, the high number of active molecules in target-annotated chemical databases even allows for the identification of molecular features that determine binding to specific proteins and protein classes.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> The principle that similar receptors bind similar ligands<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> is used in ligand-based virtual screening methods that extrapolate from known active compounds<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> and aim to identify structurally diverse compounds having similar bioactivities<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> by use of techniques like substructure mining,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> molecular fingerprint similarity searches,<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> and ligand-based pharmacophore models.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> These techniques are generally faster than methods utilizing the structure of the protein target, such as molecular docking<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> and protein structure-based pharmacophore models,<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> and in most cases they can be trained on larger data sets. Structure-based methods on the other hand are more suitable for finding ligands that are structurally novel and offer insight in the atomic details of protein–ligand interactions.</div><div class="NLM_p">The recent elucidation of GPCR structures enables in silico screens based on structure-based approaches for this protein family,<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> which is particularly relevant for the identification of new ligands and the prediction of their binding modes.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a>The research project described herein is hence an effort to utilize, and evaluate, the increasing amount of information on the structural side (such as X-ray structures) as well as the bioactivity database side (such as the ChEMBL database<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a>). Accordingly, we in parallel applied two different virtual screening approaches in the current work, the first one being based on the recently elucidated receptor structures and the second one being based on ligand bioactivity information, in order to select potentially bioactive compounds.</div><div class="NLM_p">More specifically, our work focused on the discovery of novel bioactive compounds against a panel of three pharmaceutically relevant receptors, namely, the β-2 adrenoreceptor (ADRB2), the adenosine A<sub>2A</sub> receptor (AA2AR), and the sphingosine 1-phosphate receptor (S1PR1). While ADRB2 plays an important role in cardiovascular disorders,<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> asthma,<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> and other pulmonary disease states,<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> AA2AR is involved in coronary disease<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> and Parkinson’s disease.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> The S1PR1 on the other hand is an important target in the treatment of autoimmune diseases<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> and, potentially, cancer.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> ADRB2 and AA2AR were the first druggable GPCRs for which crystal structures have been obtained<a onclick="showRef(event, 'ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref39 ref40">(39, 40)</a> and are representative of the selected few GPCRs for which protein crystal structures are available. It was only very recently (February 17, 2012), during the preparation of the current manuscript, that the ligand bound crystal structure of S1PR1 was released.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> Interestingly, the S1PR1 homology model based pharmacophore constructed in the current study (prior to the release of this X-ray structure) contains the most essential protein–ligand interactions observed in the experimentally determined S1PR1 structure (as will be discussed).<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> One particular reason why a structure-based model of S1PR1 was relevant in the context of the current study is that while considerable ligand information is available for ADRB2 and AA2AR, this is not the case for S1PR1. Hence, structure-based confirmation would be of great importance for S1PR1. Because of the diversity and number of ligands available, when going from the AA2AR via the ADRB2 to the S1PR1 receptor, one would expect the relative importance of the ligand-based model to decrease and that of the structure-based model to increase for identifying novel bioactive compounds. An additional, practical consideration for selecting these receptors was our ability to perform experimental assays for the targets mentioned to prospectively validate our models.</div><div class="NLM_p last">In a prospective manner, we selected 300 compounds per receptor target and tested <i>all compounds</i> for bioactivity on <i>all</i> three <i>targets</i> in a cross-screening experiment. This approach allowed us to address the primary aim of our study, namely, to validate and compare structure-based and ligand-based screening approaches against complete biological data (i.e., for all ligands against all targets). This would give insights into what degree hits are (also) active against a different receptor than the target they were selected for. The overarching goal was to determine which method is superior in virtual compound screening and also to gauge whether there might exist any hidden relationship (such as shared information, resulting in mutual enrichment of bioactive compounds) between the different models employed.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16206" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16206" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The aim of the current study was to evaluate the performance of (combined) ligand- and structure-based virtual screening approaches (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>), optimized using retrospective screening simulations (Figures <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a> and <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>) and applied in a prospective, experimental <i>all-against-all</i> validation (Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>–<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a> and Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/medium/jm-2012-00280e_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/large/jm-2012-00280e_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Virtual screening flowchart. Numbers indicate the number of compounds that pass the respective structure-based and ligand-based bioactivity models. Bold numbers indicate the number of actives included in the selections. Note that out of the 300 compounds selected for each of the targets, some overlap between the hit lists had to be removed and not all compounds were available as physical samples, leading to slight modifications of the hit lists suggested by the virtual screening methods.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/large/jm-2012-00280e_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300280e&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Chemical Diversity of the Ligand Training Sets</h3><div class="NLM_p">A large diversity space of active compounds is desired in drug design projects because it allows the selection and synthesis of druglike compounds with good solubility, ADMET properties, and good selectivity toward only the desired targets. In order to understand similarities between the targets investigated in this work in more detail, we first analyzed the diversity of the bioactive ligand sets employed for training. The diversity of the 50 most dissimilar compounds for AA2AR and ADRB2 in BCI fingerprint space is presented in Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>. ADRB2 ligands are more similar to each other (average Euclidean distance of ∼11) and form a rather tight cluster, as compared to AA2AR ligands (with an average Euclidean distance of ∼14). This suggests that it is more challenging to find novel ligands for the ADRB2 receptor than for the AA2AR receptor. S1PR1 compounds include primarily compounds that are chemically similar to the endogeneous ligand sphingosine 1-phosphate. Diversity selection of S1PR1 compounds is therefore not very informative, and results based on S1PR1 training set compounds are expected to be biased toward these chemical series.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/medium/jm-2012-00280e_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/large/jm-2012-00280e_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Visualization of ligand similarities for the known actives on the adenosine A<sub>2A</sub> and the β-2 adrenergic receptor, represented by (left) stochastic proximity embedding and (right) distribution of Euclidean distances. While, on average, ADRB2 ligands (red) are more similar to each other, some atypical ligands can also be found that resemble compounds active on AA2AR (blue).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/large/jm-2012-00280e_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300280e&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Pharmacophore Models Match Experimentally Supported Ligand Binding Modes</h3><div class="NLM_p">In order to generate insight into our structure-based pharmacophore models, we first investigated binding modes of ligands and related this back to previous crystal structures as well as mutation studies.</div><div class="NLM_p">It can be seen (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>) that several polar residues, namely, T88<sup>3.36</sup>, Q89<sup>3.37</sup>, N253<sup>6.55</sup>, E169<sup>45.53</sup>, and S277<sup>7.42</sup>, form H-bond interactions with cocrystallized AA2AR ligands in crystal structures,<a onclick="showRef(event, 'ref40 ref41'); return false;" href="javascript:void(0);" class="ref ref40 ref41">(40, 41)</a> as well as being important in agonist (and xanthine antagonist) binding according to site-directed mutagenesis studies.<a onclick="showRef(event, 'ref42 ref243 ref244 ref245 ref43'); return false;" href="javascript:void(0);" class="ref ref42 ref243 ref244 ref245 ref43">(42-46)</a> The structure-based pharmacophore model of AA2AR used in our virtual screen includes interaction features derived from three of these polar residues, namely, T88<sup>3.36</sup>, N253<sup>6.55</sup>, and S277<sup>7.42</sup>.</div><div class="NLM_p">ADRB2 ligands share an essential positively charged amine, as well as an aromatic ring separated by a distance of about 5 Å (partial and full agonists) to 7 Å (inverse agonists and antagonists). The ADRB2 crystal structure<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> shows protein–ligand interactions to residues D113<sup>3.32</sup>, S203<sup>5.42</sup>, N312<sup>7.39</sup>, and F290<sup>6.52</sup>, which is in line with site-directed mutagenesis studies.<a onclick="showRef(event, 'ref44 ref45 ref249 ref250'); return false;" href="javascript:void(0);" class="ref ref44 ref45 ref249 ref250">(47-50)</a> Furthermore, mutation of residues S204<sup>5.43</sup>, S207<sup>5.46</sup>, N293<sup>6.55</sup>, and Y308<sup>7.35</sup> were shown to affect partial and full agonist binding.<a onclick="showRef(event, 'ref46 ref47 ref48'); return false;" href="javascript:void(0);" class="ref ref46 ref47 ref48">(51-53)</a> The ADRB2 pharmacophore model contains a hydrophobic contact with V114<sup>3.33</sup> and polar interactions with N293<sup>7.39</sup>, D113<sup>3.32</sup>, S203<sup>5.42</sup>, and S207<sup>5.46</sup> as supported by ligand cocrystallized ADRB2 crystal structures and site-directed mutagenesis studies<a onclick="showRef(event, 'ref44 ref45 ref249 ref250 ref46 ref47 ref48'); return false;" href="javascript:void(0);" class="ref ref44 ref45 ref249 ref250 ref46 ref47 ref48">(47-53)</a> (Figures <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a> and <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>).</div><div class="NLM_p">S1PR1 receptor ligands are characterized by the presence of a polar head region, which contains negatively and positively charged groups, and a long hydrophobic tail region.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(54)</a> On the basis of these chemical ligand properties, computational modeling and site-directed mutagenesis studies have identified important S1PR1 receptor–ligand interactions, including ionic interactions between the negatively charged phosphate oxygens of sphingosine 1-phosphate (S1P) and R120<sup>3.28</sup> and between the protonated amine of S1P and E121<sup>3.29</sup>.<a onclick="showRef(event, 'ref50 ref51'); return false;" href="javascript:void(0);" class="ref ref50 ref51">(55, 56)</a> In addition, L276<sup>6.55</sup> has been identified as a residue that contributes to ligand selectivity for S1PR1 over S1PR3 (containing a leucine residue at this position).<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(57)</a> The structure-based pharmacophore model for S1PR1 indeed includes features for R120<sup>3.28</sup>, E121<sup>3.29</sup>, and L276<sup>6.55</sup>, as supported by site-directed mutagenesis studies<a onclick="showRef(event, 'ref50 ref51'); return false;" href="javascript:void(0);" class="ref ref50 ref51">(55, 56)</a> (Figures <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a> and <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>). Interestingly these essential S1PR1–ligand interaction features are indeed observed in the recently solved S1PR1 crystal structure (released during the preparation of this manuscript),<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> corroborating our S1PR1 homology model<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(58)</a> and the structure-based pharmacophore model constructed in the current study.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/medium/jm-2012-00280e_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/large/jm-2012-00280e_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Important interacting residues for the different receptors with the pharmacophores and fitted compounds. AA2AR: related to acceptor features, N181<sup>5.42</sup> and N253<sup>6.55</sup>, T88<sup>3.36</sup> and S277<sup>7.42</sup>; related to donor features, N181<sup>5.42</sup> and N253<sup>6.55</sup>, T88<sup>3.36</sup> and S277<sup>7.42</sup>; hydrophobic features, V84<sup>3.32</sup> and L85<sup>3.33</sup>. ADRB2: acceptors, Y199<sup>5.38</sup> and S203<sup>5.42</sup> and S207<sup>5.46</sup>, N312<sup>7.39</sup>; donors, D113<sup>3.32</sup>, Y199<sup>5.38</sup>, and S203<sup>5.42</sup> and S207<sup>5.46</sup>; hydrophobic feature, V114<sup>3.33</sup>. S1PR1: acceptors, R120<sup>3.28</sup>, E121<sup>3.29</sup>; donor, R120<sup>3.28</sup>; hydrophobic features, Y202<sup>5.39</sup> and L276<sup>6.55</sup>, Y202<sup>5.39</sup> and L276<sup>6.55</sup>. For each compound, the highest scoring substructure that matched the compound is highlight in red.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/large/jm-2012-00280e_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300280e&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/medium/jm-2012-00280e_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/large/jm-2012-00280e_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Similarity between the structure based pharmacophore of S1PR1 and ADRB2 ligands, illustrated by compound <b>21</b> and carazolol (in the conformation as found in the crystal structure of ADRB2<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a>). Both compounds match the five-feature pharmacophore of S1PR1 and also fit in the ADRB2 receptor pharmacophore (not shown).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/large/jm-2012-00280e_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300280e&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Retrospective Virtual Screening Validation</h3><div class="NLM_p">In order to assess the likelihood of prospective performance of our structure-based and ligand-based virtual screening protocols, first a retrospective validation was performed, using the 50 diverse actives for AA2AR and ADRB2 vs the 50 308 assumed inactive compounds of the compound library. Here significant early enrichment was observed, as shown in Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>. Retrospective enrichment for AA2AR is poor for the structure-based method, probably (i) because of the fact that a large portion of AA2AR ligands bind in the extracellular domain that is not represented in Snooker pharmacophores and (ii) because it is likely that the A2A adenosine receptor possesses no conserved ligand binding site.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> The ligand-based method on the other hand exhibits excellent performance for this receptor; however, it might be biased toward known chemistry in our retrospective virtual screening studies. A combination of both methods is hence likely to result in a reduced number of identified actives when compared to the substructure-based method alone; however, it would be assumed to show more novelty among identified active compounds (as well as potentially a different set of ligand–protein interactions), compared to the training set molecules.</div><div class="NLM_p">In contrast to AA2AR ligands, the ADRB2 ligands bind largely within the transmembrane (TM) domain and share a common binding mode within a buried pocket. Therefore, structure-based searches perform better on ADRB2 than on AA2AR (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>). They not only are able to capture the actives faster but also retrieve a higher percentage of actives. In this particular case, a combination of structure-based and ligand-based methods for this receptor outperforms both the individual ligand-based and structure-based virtual screening methods (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>).</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/medium/jm-2012-00280e_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/large/jm-2012-00280e_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Receiver operating curves (ROC) for the ligand-based and structure-based virtual screening methods employed in this work. While the substructure-based method, on average, is able to achieve higher enrichment than the employed structure-based method, the extent of this difference is very much dependent on the target receptor considered, with AA2AR showing significantly higher retrieval of active compounds than ADRB2. Consensus scoring outperforms each individual method in the case of ADRB2.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/large/jm-2012-00280e_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300280e&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Prospective Cross-Screening</h3><div class="NLM_p">For each of the three bioactivity models a total of 300 compounds were selected. Duplicates were removed, and in the case of unavailability of the selected compound it was replaced with the next compound in the list. The final selection of compounds (including cross-receptor overlap) is visualized in Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a> (and <a class="ref internalNav" href="#notes-3" aria-label="Supporting Information Figures 5–7">Supporting Information Figures 5–7</a>) for all of the models. Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a> shows that the ADRB2 model and the S1PR1 models select the same compounds more often than other pairs of activity models. Overlap in “hit selection space” can indeed have implications on the results of prospective all-against-all virtual screening studies, as described in more detail below.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/medium/jm-2012-00280e_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/large/jm-2012-00280e_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Hit selection space based on the combined protein-based pharmacophore and ligand substructure virtual screens (3780 for AA2AR (blue), 810 for ADRB2 (green), and 1205 for S1PR1 (gray)). Most compounds are selected, since they are expected to have activity on only one of the three receptors. Some compounds are, however, expected to have activity on multiple receptors (front-upper, left-upper, right-upper, and front-lower corner), and a small set (front upper corner) is even expected to have activity on all three targets. Compounds are colored according to the receptor in which they receive the best rank.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/large/jm-2012-00280e_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300280e&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Number of Molecules in Training Set and Test Set As Used for the Construction of the Substructure-Based Model for AA2AR, ADRB2, and S1PR1<a class="ref internalNav" href="#tbl1-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char=".">training set</th><th class="colsep0 rowsep0" align="center" char=".">test set<a class="ref internalNav" href="#t1fn1" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">total</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AA2AR<a class="ref internalNav" href="#t1fn2" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="char" char=".">179</td><td class="colsep0 rowsep0" align="char" char=".">76</td><td class="colsep0 rowsep0" align="char" char=".">255</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ADRB2<a class="ref internalNav" href="#t1fn2" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="char" char=".">301</td><td class="colsep0 rowsep0" align="char" char=".">129</td><td class="colsep0 rowsep0" align="char" char=".">430</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">S1PR1<a class="ref internalNav" href="#t1fn3" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="char" char=".">172</td><td class="colsep0 rowsep0" align="char" char=".">74</td><td class="colsep0 rowsep0" align="char" char=".">246</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl1-fn1"><div class="footnote" id="tbl1-fn1"><sup>a</sup><p class="last">Ligands for all three receptors were retrieved from ChEMBL.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a></p></div><div class="footnote" id="t1fn1"><sup>b</sup><p class="last">The test set is 30% of the total set of selected ligands.</p></div><div class="footnote" id="t1fn2"><sup>c</sup><p class="last">Ligands with either <i>K</i><sub>i</sub>, IC<sub>50</sub>, or EC<sub>50</sub> of 10 nM or less.</p></div><div class="footnote" id="t1fn3"><sup>d</sup><p class="last">Ligands with either <i>K</i><sub>i</sub>, IC<sub>50</sub>, or EC<sub>50</sub> of 10 μM or less.</p></div></div></div><div class="NLM_p">Using our combined ligand- and structure-based virtual screening approach, we have successfully identified 18 AA2AR, 6 ADRB2, and 3 S1PR1 ligands (Figures <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a> and <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a>). ADRB2 and AA2AR were tested for ligand displacement, detecting both agonists and antagonists that bind to the orthosteric binding site. Both protein-based pharmacophore models (derived from orthosteric ligand binding sites of ADRB2 and AA2AR; see Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>) and ligand-based models (based on substructures of orthosteric ADRB2 and AA2AR ligands) are constructed based on information on the orthosteric ADRB2 and AA2AR binding cavities. S1P1R was tested in a functional assay in agonistic format, and experimentally confirmed S1PR1 hits might in principle also bind to binding sites other than the cavity included in the S1PR1-based pharmacophore model (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>). Many of the validated hits are chemically dissimilar to any known ADRB2, AA2AR, or S1PR1 ligand (ECFP-4 Tanimoto similarity below 0.4 (loose cutoff)<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(61)</a> or even below 0.26 (strict cutoff);<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(62)</a> see Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>). Structures of the new ligands are visualized in Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>, and plots of compound ranks versus receptor activity are displayed in Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>, also distinguishing between the in silico models used to select each bioactive compound identified in this study. As can be seen in Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>, AA2AR showed highest overall hit rates, which agrees with our initial hypothesis. However, a significant number of confirmed AA2AR hits also came from the S1PR1 model, which was rather surprising to us and which is discussed in more detail below. Experimentally determined radioligand displacement (AA2AR and ADRB2) and dose–response curves (S1PR1) are shown in Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a>.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. AA2AR and ADRB2 Receptor Binding Affinities and S1PR1 Potencies of Validated Hits Selected by a Combined Ligand-Based (Frequent Substructure Ranking<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(59)</a>) and Structure-Based (Snooker<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(60)</a>) Virtual Screening Protocol (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>)<a class="ref internalNav" href="#tbl2-fn1" aria-label="h">h</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/medium/jm-2012-00280e_0009.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/medium/jm-2012-00280e_0010.gif" alt="" id="GRAPHIC-d769e1014-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/medium/jm-2012-00280e_0011.gif" alt="" id="GRAPHIC-d769e1015-autogenerated" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>Table a</sup><p class="last">Average rank based on AA2AR proteins-based pharmacophore and frequent AA2AR ligand substructure rankings (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>).</p></div><div class="footnote" id="t2fn2"><sup>Table b</sup><p class="last">Average rank based on ADRB2 protein-based pharmacophore and frequent ADRB2 ligand substructure rankings (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>).</p></div><div class="footnote" id="t2fn3"><sup>Table c</sup><p class="last">Average rank based on S1PR1 protein-based pharmacophore and frequent S1PR1 ligand substructure rankings (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>).</p></div><div class="footnote" id="t2fn4"><sup>Table d</sup><p class="last">p<i>K</i><sub>i</sub> values are calculated by displacement of [<sup>3</sup>H]ZM241385 binding on membranes of HEK293T cells stably expressing the AA2AR (<i>n</i> = 3, each performed in duplicate).</p></div><div class="footnote" id="t2fn5"><sup>Table e</sup><p class="last">p<i>K</i><sub>i</sub> values are calculated by displacement of [<sup>3</sup>H]dihydroalprenolol binding on membranes of HEK293T cells transiently expressing the ADRB2 (<i>n</i> = 3, each performed in triplicate).</p></div><div class="footnote" id="t2fn6"><sup>Table f</sup><p class="last">pEC<sub>50</sub> values are calculated by induction of β-arrestin2 recruitment in CHO-K1 S1PR1 β-arrestin EFC cells (<i>n</i> = 1, each performed in triplicate).</p></div><div class="footnote" id="t2fn7"><sup>Table g</sup><p class="last">ECFP-4 circular fingerprint Tanimoto similarity to closest known AA2AR/ADRB2/S1PR1 active in ChEMBLdb. A similarity higher than 0.40 is considered significant.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(61)</a></p></div><div class="footnote" id="tbl2-fn1"><sup>Table h</sup><p class="last">Closest chemical similarity to any known AA2AR/ADRB2/S1PR1 ligand is given for each validated hit.</p></div></div><div></div></div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/medium/jm-2012-00280e_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/large/jm-2012-00280e_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Novel active compounds found using the respective models for the three receptors in this study (adenosine A<sub>2A</sub> receptor, β-2 adrenergic receptor, and sphingosine 1-phosphate receptor). The top of the plot shows the receptor for which the ligands were experimentally found to be active against, while the bottom of the plot shows the model that was used in silico to select each respective compound.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/large/jm-2012-00280e_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300280e&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/medium/jm-2012-00280e_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/large/jm-2012-00280e_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. (A) Compounds <b>1</b>–<b>3</b> selected from the AA2AR screen dose-dependently displacing [<sup>3</sup>H]ZM241385 from AA2AR. (B) Compounds <b>19</b>–<b>21</b> selected from the ADRB2 screen dose-dependently displacing [<sup>3</sup>H]dihydroalprenolol from ADRB2. (C) Compounds <b>25</b>–<b>27</b> selected from the S1PR1 screen dose-dependently inducing β-arrestin2 recruitment. PathHunter CHO-K1 S1PR1 β-arrestin EFC cells genetically engineered to increase β-galactosidase activity following the recruitment of β-arrestin2 to S1PR1 receptor were stimulated with increasing concentrations of <b>25</b>, <b>26</b>, <b>27</b> for 2 h before measurement of β-galactosidase activity. A single concentration of AUY954 (1 μM) was taken along to measure maximal β-arrestin2 recruitment.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/large/jm-2012-00280e_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300280e&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The virtual screening procedure was very successful for AA2AR and selected 12 out of the 18 active compounds for this receptor directly (including compound <b>3</b>, for which the proposed binding mode in AA2AR is depicted in Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>). Another three compounds were contained in the virtual AA2AR hit list; however, they were actually selected for testing because they had a better rank in the virtual screening for another receptor (namely, S1PR1). Surprisingly, one of those three compounds is the phosphodiesterase 5 (PDE5) inhibitor sildenafil (compound <b>1</b>) that has submicromolar affinity for the AA2AR receptor (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>, Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>). This is in line with previous in vivo studies suggesting that spinal adenosine receptors may play a role in sildenafil-induced antinociception<a onclick="showRef(event, 'ref263'); return false;" href="javascript:void(0);" class="ref ref263">(63)</a> and cardioprotective signaling.<a onclick="showRef(event, 'ref264'); return false;" href="javascript:void(0);" class="ref ref264">(64)</a> Comparison of the ZM241385 bound AA2AR<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> and sildenafil bound PDE5<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(65)</a> cocrystal structures reveals that both pockets possess similar interaction features for ligand binding. ZM241385 makes hydrogen bonding interactions with the amide side chain of N253<sup>6.55</sup> equivalent to the interaction of sildenafil with Q817 in PDE5. Similarly, the bicyclic rings of both ZM241385 and sildenafil are locked in a hydrophobic clamp between F168<sup>45.52</sup>, I274<sup>7.39</sup>, and L249<sup>6.51</sup> (AA2AR) and F820, V782, and F786 (PDE5), respectively (<a class="ref internalNav" href="#notes-3" aria-label="Supporting Information Figure 6">Supporting Information Figure 6</a>). Our unbiased computer-aided discovery of sildenafil as a true AA2AR ligand is therefore an interesting result, indicating that systematic in silico chemogenomics studies can be used to identify new interesting cross-pharmacology between unrelated protein targets. In this context the concepts of “GPCR-like compounds” and “privileged scaffolds”<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(66)</a> is also likely reflected in our selection procedure, given the observation that compounds active on the AA2AR receptor were retrieved when explicitly selecting for its ligands, as well in the ADRB2 and S1PR1 compound selection rounds.</div><div class="NLM_p">For the ADRB2 receptor 6 confirmed hits could be identified (including compound <b>21</b>, for which the proposed binding mode in ADRB2 is shown in Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>), which is a much smaller number of hits than in the case of AA2AR. This is in agreement with the restricted chemical diversity of ligands for this receptor, compared to those of the AA2AR receptor. The active compounds that were identified against ADRB2 interestingly resulted from the compound selection for the AA2AR and S1PR1 receptors. The fact that we were not able to identify any ADRB2 ligands from the ADRB2 virtual screening run might of course be related to the fact that all compounds similar to known ADRB2 actives were excluded from the compound set; however, the ability to identify compounds active on one receptor using a model for another (albeit related) receptor remains remarkable (as well as consistent throughout the receptors employed in this study). Notably, five of the ADRB2 bioactives were retrieved with the S1PR1 receptor pharmacophore. One possible reason we identified is that important hydrogen bonding and charge interactions are present in both ligand–receptor interactions (which correspond to a similar distance between D113<sup>3.32</sup> and Y199<sup>5.38</sup> in ADRB2, compared to R120<sup>3.28</sup> and E121<sup>3.29</sup> in S1PR1; Figures <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a> and <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>). In addition to these polar features, the pharmacophore model consists of hydrophobic features that are located in the vicinity of L276<sup>6.55</sup>, which forms the binding site of the aliphatic tail of the ligand in the S1PR1 cocrystal structure.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> The hydrophobic features are situated at the conjunction of L276<sup>6.55</sup> and Y202<sup>5.39</sup> in the S1PR1 homology model (Figures <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a> and <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>). However, the side chain of Y202<sup>5.39</sup> is located further from the ligand binding site (closest distance to the ligand is 6 Å) as a result of the different helical conformation of TM5 (the reason being that S1PR1 lacks the conserved P<sup>5.50</sup> residue that induces a helical kink in other GPCR crystal structures<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a>). Another aromatic residue, Y125<sup>3.33</sup>, recently shown to play a role in S1PR1 ligand binding based on site-directed mutagenesis studies<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(58)</a> and involved in S1PR1-ligand interactions in the S1PR1 crystal structure<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> is also very close to the location of some of the hydrophobic pharmacophore features in our S1PR1 homology model. While hydrophobic interactions originate in ADRB2 mainly from V114<sup>3.33</sup> and in S1PR1 from Y202<sup>5.39</sup> and L276<sup>6.55</sup>, they are in both cases located at fairly similar positions, further rationalizing the relatively high degree of overlap of the in silico hit lists of ADRB2 and S1PR1. The match of compound <b>21</b> in the S1PR1 pharmacophore is displayed in Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>, including a fit of the crystal structure conformation of carazolol of the ADRB2 receptor<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> in this pharmacophore. The complementarity of the S1PR1 pharmacophore to known ADRB2 ligands, in combination with the high similarity of the S1PR1 pharmacophore to the ADRB2 pharmacophore, is the best explanation of the ability of the S1PR1 structure-based pharmacophore to retrieve ligands bioactive against ADRB2. Although compounds <b>21</b>–<b>24</b> have relatively low affinity for ADRB2, these molecules offer interesting starting points for hit optimization and give insights into molecular determinants of ADRB2 binding. The low affinity of hit <b>21</b> suggests that small polar groups close to the essential basic nitrogen moiety are not favorable for ADRB2 binding. Compound <b>24</b> on the other hand has a relatively high ligand efficiency (LE = 0.42) and shares a 2-((dimethylamino)methyl)phenol scaffold with compound <b>22</b> that can be used for future ADRB2 ligand optimization studies.</div><div class="NLM_p last">For S1PR1, three agonists were identified, of which compound <b>26</b> originated from the ligands selected explicitly for this receptor. The proposed binding mode in the S1PR1 binding mode of <b>26</b> is presented in Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>. Overall, two of the three compounds active against S1PR1 were actually selected by the AA2AR model, again a surprising result of our prospectively validated all-against-all virtual screening study. Although the potency of the validated S1PR1 agonists <b>26</b>–<b>28</b> is relatively low, so far only very few S1PR1 ligands have been reported that lack a negatively ionizable functional group. The validated hits <b>26</b>–<b>28</b> therefore are potential interesting new starting points for S1PR1 ligand optimization studies.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Lessons Learned</h3><div class="NLM_p">The following lessons can be learned from our systematic prospective evaluation of a protein-based and a ligand-based virtual screening platform against ADRB2, AA2AR, and S1PR1:</div><div id="sec2_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18">1 Systematic validation of structure-based and ligand-based virtual screening approaches requires experimental biological activity data for all ligands against all investigated protein targets</h4><div class="NLM_p last">With the increasing amount of experimental information on protein targets and their ligands, complementary computational tools are required that are able to mine this large matrix of experimental data to predict novel protein–ligand interactions. Complete all-against-all data sets containing the biological activities of multiple molecules for a set of protein targets (like the one generated in our study) are essential for testing in silico methods in real life screening scenarios and for evaluating their ability to predict, for example, ligand cross-pharmacology and toxicity (i.e., interactions with antitargets).</div></div><div id="sec2_5_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19">2 Retrospective validation of virtual screening methods can be used to assess the limitations of (combining) ligand-based and protein-based discovery of chemically novel ligands</h4><div class="NLM_p last">The rather diverse AA2AR ligand training set facilitated the construction of a robust ligand-based model that was successfully used to discover a diverse set of novel AA2AR ligands. The lower chemical diversity of the ADRB2 and S1PR1 training sets made the discovery of novel ligands for these receptor more challenging and required a combination of ligand- and protein-based virtual screening models.</div></div><div id="sec2_5_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20">3 Overlap in protein-based pharmacophore models can yield experimentally validated hits that are not (only) active against the receptor they were selected for but (also) against a different member of the panel</h4><div class="NLM_p last">However, by use of protein-based virtual screening methods (complementary to ligand-based virtual screening), protein–ligand binding mode hypotheses can be generated to identify conserved and selective interaction features in different protein targets and to guide hit optimization studies.</div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i21">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66765" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66765" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We performed compound selections for three receptors (AA2AR, ADRB2, and S1PR1) based on a combined structure (pharmacophore) and ligand (substructure) based approach. Retrospective analysis of both individual methods on the AA2AR and ADRB2 receptor indicated that virtual screening accuracy is to a large extent dependent on the chemical diversity among the bioactive compound class:.The rather diverse AA2AR ligand training set facilitated the construction of a robust ligand-based model that turned out to be superior to the protein-based model, while a combination of a ligand- and protein-based model gave the best results for ADRB2.</div><div class="NLM_p">Using a consensus scoring procedure combining the ligand-based and structure-based approaches, we selected 300 compounds for each of the three GPCR targets and experimentally tested all 900 in silico hits against all other receptors in the study. Novel bioactive compounds were identified using a consensus scoring procedure combining ligand-based and structure-based tools (18 actives for AA2AR, 6 actives for ADRB2, and 3 actives for S1PR1). While these variations in hit rates might not be surprising by themselves, what is certainly remarkable is the high number of experimentally validated hits that were not active against the receptor they were selected for but rather against a different member of the panel. Interestingly, one of those ligands is the phosphodiesterase 5 (PDE5) inhibitor sildenafil that was found to have submicromolar affinity for the AA2AR adenosine receptor, demonstrating that our systematic in silico chemogenomics studies can be used to identify new interesting cross-pharmacology between unrelated protein targets.</div><div class="NLM_p last">Ideally, the chemogenomics approach presented here should be extended with more compounds and a larger panel of targets and should preferably be tested using the same assay technology. Such endeavors are already frequently employed for other protein families, such as for kinases, and they are hoped to provide more complete information for both hit identification and optimization than the current state-of-the-art single-target screens are able to do.<a onclick="showRef(event, 'ref267'); return false;" href="javascript:void(0);" class="ref ref267">(67)</a> The lessons learned from this exercise can be used to guide future virtual ligand design efforts.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i22">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12448" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12448" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The virtual screening setup employed in this work consists of (1) a structure-based protocol (based on inverse pharmacophores), (2) a ligand-based protocol (based on frequent substructure mining), and (3) a merging of the results of the two followed by a prospective validation. This setup is schematically displayed in Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>, and its parts will be described in detail in the below sections.</div><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Structure-Based Pharmacophore Screening</h3><div class="NLM_p">Structure-based pharmacophores describing the negative image of the pocket, located between the transmembrane helices, were generated for all three GPCRs with the previously described Snooker progam.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(60)</a> Snooker is a structure-based approach to generate pharmacophore hypotheses for compounds binding to the TM domain of GPCRs. Snooker does not require prior knowledge of ligands but selects residues based on their probability of being involved in ligand binding based on their sequence conservation (expressed as entropy values) inside and outside each GPCR subfamily.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(68)</a> Subsequently, protein properties of selected residues are projected to complementary ligand space, and pharmacophore features are generated at positions where protein–ligand interactions are likely.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(60)</a> For AA2AR and ADRB2 a consensus model based on eight different templates (PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1GZM">1GZM</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1L9H">1L9H</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2RH1">2RH1</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VT4">2VT4</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3CAP">3CAP</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3D4S">3D4S</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3DQB">3DQB</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3EML">3EML</a>) was constructed,<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(60)</a> whereas for S1PR1 a model was generated based on an experimentally validated agonist-bound homology model.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(58)</a> It was only very recently, during the preparation of the current manuscript, that the ligand bound crystal structure of S1PR1 was published<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> showing the same essential receptor–ligand interactions as predicted in our receptor structure-based pharmacophore model (constructed prior to the release of the S1PR1 crystal structure).<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> To gain specificity, directionality was added to the polar pharmacophore features as the average vector between the polar feature center and the Cα atom of the residues that constitute the respective features.</div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> Structure-Based Pharmacophore (SBP) Ligand Training Set</h4><div class="NLM_p last">The structure-based pharmacophores for virtual screening are selected by a training procedure that utilizes ligand information. Known active compounds for the three target proteins investigated, i.e., the β-2 adrenoreceptor (ADRB2), the adenosine A<sub>2A</sub> receptor (AA2AR), and the sphingosine 1-phosphate receptor (S1PR1), were extracted for all species from the ChEMBL database (release August 2009) independently of the functional class (agonist, partial agonist, antagonist, inverse agonist) using an activity cutoff on <i>K</i><sub>i</sub>, IC<sub>50</sub>, or EC<sub>50</sub> of less than 50 nM. To reduce the bias introduced by the deposition of compound series, diverse subsets of 50 compounds were generated for the AA2AR and ADRB2 receptors by exclusion sphere clustering on Tanimoto distances between the BCI fingerprints of the compounds. For the S1PR1 receptor all 43 compounds reported in ChEMBL were used as a training set.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> Pharmacophore Model Generation and Training</h4><div class="NLM_p last">Structure-based pharmacophores for all three targets were subsequently trained with their corresponding SBP ligand training sets of 50 known actives. In this training procedure, the subsets of pharmacophore features that are able to retrieve active compounds were ranked first according to the number of features in the pharmacophore and subsequently according to the number of known actives that they were able to retrieve. To optimize screening outcome, the ranking of pharmacophores was manually adjusted based on crystal structure information and mutagenesis data (see <a class="ref internalNav" href="#notes-3" aria-label="Supporting Information Tables 1–4">Supporting Information Tables 1–4</a>). Finally, shape restraints were defined by the Tanimoto distance of the compounds matching the ADRB2 and AA2AR pharmacophores to the cocrystallized ligands carazolol and ZM241385, respectively.<a onclick="showRef(event, 'ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref39 ref40">(39, 40)</a> For S1PR1, shape restraints were defined as the Tanimoto distance of compounds matching the S1PR1 pharmacophore to the average pose of known actives from the S1PR1 training set in the same pharmacophore. Tanimoto distance cutoffs were set to those values at which the enrichment of known actives over the 50 308 database compounds was optimal (0.700 for AA2AR, 0.675 for ADRB2, and 0.600 for S1PR1).</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> Pharmacophore Screening Settings</h4><div class="NLM_p last">Pharmacophore screening was performed via RDKit using the procedure described by Sanders et al.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(60)</a> Projected points were also calculated in RDKit and treated in the same manner as other feature types, with the exception that a heavy-atom–projected-point pair could not be separated and had to match a feature–projected-feature pair. The angular difference between the vectors describing the heavy-atom–projected-point pair and the feature–projected-feature pair was required to lie within 45° of each other.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> Preparation of Prospective Pharmacophore Screening Library</h4><div class="NLM_p last">The compound library used for screening (described in detail below) was prepared as follows for the structure-based virtual screening protocol. Initial three-dimensional conformations were generated with CORINA,<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(69)</a> and multiple 3D conformations were created with a genetic algorithm, Cyndi,<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(70)</a> employing a population size of 200 and finally outputting 100 conformations per molecule. Given that the algorithm used here is designed for screening millions of compounds of typical in-house or vendor libraries, no further force field minimization of compounds has been performed.</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Ligand (Frequent Substructure) Based Virtual Screening</h3><div class="NLM_p">For the ligand-based screening study, the substructure-based screening method of van der Horst et al.<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(59)</a> was employed. This method performs screening by searching library compounds for the occurrence of substructures characteristic for bioactivity against a particular receptor. These substructures are derived from existing ligands of the target investigated and are selected for their ability to distinguish ligands for this target from ligands for other, secondary proteins.</div><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> Ligand-Based (LB) Training Set</h4><div class="NLM_p last">For the human adenosine A<sub>2A</sub> receptor (AA2AR), β<sub>2</sub> adrenoceptor (ADRB2), and the S1P<sub>1</sub> lysophospholipid receptor (S1PR1), ligand structures and activity data were retrieved from the ChEMBL database,<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> selecting compounds with activity, i.e., a <i>K</i><sub>i</sub>, IC<sub>50</sub>, or EC<sub>50</sub>, of 10 nM or less for the adenosine A<sub>2A</sub> receptor and 10 μM or less for the other two receptors. For the A<sub>2A</sub> ligands, we chose a stricter selection, since this resulted in a set of high-affinity ligands that was still of comparable size to the other two ligand sets. Subsequent manual inspection was performed to ensure validity of the LB training set. All source sets were split into a training set and a test set using the “Diverse Molecules” component in PipelinePilot, version 6.1, using a 30% test set in combination with FCFP-4 fingerprints (both training and test set sizes are provided in Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>). For analysis of the substructures, all training sets were contrasted against a background compound distribution, consisting of 10 000 randomly selected compounds from the druglike subset of the ZINC database (accessed February 12, 2010).<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(71)</a> Chemical structures were represented as graphs indicating the type of the bonds, e.g., single, double, or aromatic.</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> Generation of “Frequently Occurring Substructures”</h4><div class="NLM_p last">Frequent substructure sets were generated using the frequent graph miner Gaston.<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(72)</a> For each substructure, the number of molecules containing that particular substructure was calculated. The difference between the relative occurrence (fraction) of a substructure in the antagonists set and the background set is the score contribution of that substructure. Substructures were ranked according to the score contribution in descending order, and the 50 highest-ranking substructures were selected for the virtual screening model.</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> Substructure-Based Virtual Screening: Ranking of Compounds</h4><div class="NLM_p last">Substructure scores were employed to rank database compounds (which might contain multiple substructures per compound). For this ranking, the score for a compound was calculated as the sum of all score contributions of substructures present in each particular structure.</div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> Virtual Screening Library</h3><div class="NLM_p last">A diverse subset of the MSD/Organon (Oss, The Netherlands) library consisting of 50 308 compounds was used for virtual screening. In order to characterize the library, the overlap of the available library with the ZINC purchasable compound set (∼23 700 000) was determined. The latter consists of compounds with a MW ≤ 500 Da from 26 vendors. To determine the overlap between the two libraries, structures were converted into unique hash codes without considering stereochemistry. From the 50 308 MSD compounds, 85% occurred in the 23 691 219 ZINC<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(71)</a> compound database and 60% occurred in the 6 981 556 CoCoCo<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(73)</a> compound database (<a class="ref internalNav" href="#notes-3" aria-label="Supporting Information Figure 1">Supporting Information Figure 1</a>). The MSD compounds possess physicochemical properties similar to those within the ZINC and CoCoCo compound databases, placing emphasis on druglikeness<a onclick="showRef(event, 'ref66 ref67'); return false;" href="javascript:void(0);" class="ref ref66 ref67">(74, 75)</a> (<a class="ref internalNav" href="#notes-3" aria-label="Supporting Information Figure 2">Supporting Information Figure 2</a>). Compounds that occurred in the training or test sets were removed from the screening library, as well as compounds that had already been tested against one of the targets. In addition, for the AA2A receptor, compounds with a typical adenosine receptor ligand scaffold, such as xanthines, have been removed in order to improve the likelihood of discovering structurally novel compounds through this work.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> Final Compound Selection</h3><div class="NLM_p last">After compound selection, we removed compounds very similar to known active structures in the cases of the AA2AR and ADRB2 virtual hit list, namely, those with a Tanimoto coefficient (<i>T</i><sub>c</sub>) of >0.5 to known bioactive compounds (as annotated in ChEMBL and employing ECFP_4 fingerprints). This step was performed to ensure that sufficiently novel active compounds would be discovered in this work, which is particularly relevant in the case of the AA2AR receptor for which already chemically diverse sets of ligands are known (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>). A selection of 300 compounds for each receptor was made after determination of the average rank of each compound that matched a pharmacophore and had a positive score in the frequent substructure procedure. Where compounds were not available for testing, the next best compound was selected. The compounds selected by virtual screening were part of a diverse subset of the in-house chemical library of MSD/Organon (Oss, The Netherlands) and originally purchased from available screening collections of eight vendors: Asinex (<a href="http://www.asinex.com" class="extLink">www.asinex.com</a>), Bionet (<a href="http://www.keyorganics.co.uk" class="extLink">www.keyorganics.co.uk</a>), Chembridge (<a href="http://www.chembridge.com" class="extLink">www.chembridge.com</a>), Chemdiv (<a href="http://www.chemdiv.com" class="extLink">www.chemdiv.com</a>), Contract Chemicals (<a href="http://www.contract-chemicals.com" class="extLink">www.contract-chemicals.com</a>), Interbioscreen (<a href="http://www.ibscreen.com" class="extLink">www.ibscreen.com</a>), Orion (<a href="http://www.orionscientific.in" class="extLink">www.orionscientific.in</a>), and Specs (<a href="http://www.specs.net" class="extLink">www.specs.net</a>). Purity of compounds was equal to or greater than 95% as verified by LC–MS experiments performed by the vendors.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> Experimental Validation</h3><div id="sec4_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> Adenosine A<sub>2A</sub> Receptor</h4><div class="NLM_p last">HEK293 cells stably expressing the human AA2AR receptor (gift from Dr. Wang, Biogen, Cambridge, MA) were used to determine the affinity of compounds in a radioligand binding assay with [<sup>3</sup>H]ZM241385 as the radioligand. Membranes containing 40 μg of protein were incubated in a total volume of 100 μL of Tris-HCl (50 mM, pH 7.4) and [<sup>3</sup>H]ZM241385 (final concentration of 1.7 nM) for 2 h at 25 °C in a shaking water bath. Nonspecific binding was determined in the presence of 100 μM CGS21680. The incubation was terminated by filtration over prewetted Whatman GF/B filters under reduced pressure with a Brandel harvester. Filters were washed three times with ice-cold buffer and placed in scintillation vials. Emulsifier Safe (3.5 mL) was added, and after 2 h radioactivity was counted in a TriCarb 2900TR liquid scintillation counter. Compounds that inhibited binding by ≥50% at 10 μM were subject to testing in concentration–response curves.</div></div><div id="sec4_5_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> β-2 Adrenoreceptor</h4><div class="NLM_p last">HEK293T cells were cultured and transiently transfected with 2.5 μg of ADRB2-pcDNA3.1+ (obtained from the Missouri S&T cDNA Resource Center ) per 10<sup>6</sup> cells using 12 μg of linear 25 kDa polyethylenimine (Polysciences, Warrington, PA, U.S.) as described previously.<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(76)</a> Cells were harvested 48 h after transfection, and membrane fractions were prepared as described previously.<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(76)</a> ADRB2-expressing membranes were incubated at room temperature in 96-well plates in binding buffer (50 mM HEPES, pH 7.4, 1 mM CaCl<sub>2</sub>, 5 mM MgCl<sub>2</sub>, 100 mM NaCl, and 0.5% (w/v) BSA) with 1 nM [<sup>3</sup>H]dihydroalprenolol (DHA, 104.4 Ci/mmol from PerkinElmer Life Sciences) and 10 μM or increasing concentrations of compounds. After 1 h, incubations were terminated by rapid filtration through Unifilter GF/C plates (PerkinElmer Life Sciences) presoaked in 0.5% polyethylenimine and washed with ice-cold binding buffer supplemented with 500 mM NaCl. Radioactivity was measured using a MicroBeta Trilux (PerkinElmer Life Sciences). Compounds that inhibited binding by ≥50% at 10 μM were subject to testing in concentration–response curves. Nonlinear regression analysis of data and calculation of <i>K</i><sub>d</sub> and <i>K</i><sub>i</sub> values was performed using GraphPad Prism, version 4, software.</div></div><div id="sec4_5_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> Sphingosine 1-Phosphate Receptor</h4><div class="NLM_p last">The S1PR1 assay was performed using the PathHunter enzyme fragment complementation β-arrestin recruitment technology as described previously.<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(77)</a> CHO-K1 S1PR1 β-arrestin EFC cells (DiscoveRx, Fremont, CA) were cultured in Dulbecco’s modified Eagle's medium F-12 (Invitrogen, Carlsbad, CA), supplemented with 10% heat-inactivated fetal calf serum (Cambrex, Verviers, Belgium), 100 U/mL penicillin, 100 μg/mL streptomycin, 300 μg/mL hygromycin B, and 800 μg/mL Geneticin (Invitrogen). Cells were seeded at a density of 10 000 cells per well of a 384-well culture plate (PerkinElmer, Boston, MA) in 20 μL of OPTI-MEM (Invitrogen). After overnight incubation at 37 °C in a humidified incubator (5% CO<sub>2</sub>, 95% humidity), 4 μL of compound dilution was added to cells and the plate was returned to the incubator for 2 h, followed by incubation at room temperature for 1 h. Cells were lysed using 8 μL of PathHunter detection reagent (DiscoveRx). Plates were incubated in the dark for 2 h at room temperature before measurement of β-galactosidase activity (chemiluminescence) on an Envision multilabel plate reader (PerkinElmer Life Sciences). Compounds that induced ≥30% β-arrestin recruitment compared to the reference compound AUY954 were selected and tested in dose–response curves.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-3"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i38"><a href="/doi/suppl/10.1021/jm300280e">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_04127" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_04127" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Pharmacophore definitions of AA2AR, ADRB2, and S1PR1; lists of ranked pharmacophores according to their expected retrieval of actives for all three targets; overview of property distributions of the 50.308 MSD compounds and the compounds available in the ZINC and CoCoCo database; overlap analysis between the 50.308 MSD compounds and ZINC and CoCoCo based on ECFP_4 fingerprints; plots of compound selections and their respective ranking in the structure-based and ligand-based ranking protocols. This material is available free of charge via the Internet at <a href="http://pubs.acs.org" class="extLink">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm300280e/suppl_file/jm300280e_si_001.pdf">jm300280e_si_001.pdf (1.18 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/jm300280e" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25266" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25266" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jan P. G. Klomp</span> - <span class="hlFld-Affiliation affiliation">Department of Molecular Design
and Informatics, MSD, Molenstraat 110,
5342 CC Oss, The Netherlands</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#97fdf6f9b9fcfbf8fae7d7fbf2f6f3e7fff6e5faf6b9f4f8fab7d6f9f3e5f2f6e4b9d5f2f9f3f2e5d7f4f6f9e3f6f5b9f9f2e3b7d4b9f3f2b9d0e5f6f6f1d7e1e2b9f9fb"><span class="__cf_email__" data-cfemail="751f141b5b1e191a18053519101411051d140718145b161a18">[email protected]</span> <span class="__cf_email__" data-cfemail="8ecfe0eafcebeffda0ccebe0eaebfcceedefe0faefeca0e0ebfa">[email protected]</span> <span class="__cf_email__" data-cfemail="33701d57561d74415252557345461d5d5f">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andreas Bender</span> - <span class="hlFld-Affiliation affiliation">Leiden/Amsterdam Center of Drug Research
(LACDR), Division of Medicinal Chemistry, Leiden University, Einsteinweg 55, 2333 CC Leiden, The Netherlands</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#1e747f7030757271736e5e727b7f7a6e767f6c737f307d71733e5f707a6c7b7f6d305c7b707a7b6c5e7d7f706a7f7c30707b6a3e5d307a7b30596c7f7f785e686b307072"><span class="__cf_email__" data-cfemail="ed878c83c3868182809dad81888c899d858c9f808cc38e8280">[email protected]</span> <span class="__cf_email__" data-cfemail="32735c56405753411c70575c5657407251535c4653501c5c5746">[email protected]</span> <span class="__cf_email__" data-cfemail="cf8ce1abaae188bdaeaea98fb9bae1a1a3">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chris de Graaf</span> - <span class="hlFld-Affiliation affiliation">Leiden/Amsterdam Center of Drug Research (LACDR), Division of Medicinal
Chemistry, Faculty of Sciences, VU University Amsterdam, De Boelelaan 1083, 1081 HV Amsterdam, The Netherlands</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#513b303f7f3a3d3e3c21113d343035213930233c307f323e3c71103f35233430227f13343f3534231132303f2530337f3f342571127f35347f16233030371127247f3f3d"><span class="__cf_email__" data-cfemail="acc6cdc282c7c0c3c1dcecc0c9cdc8dcc4cddec1cd82cfc3c1">[email protected]</span> <span class="__cf_email__" data-cfemail="da9bb4bea8bfbba9f498bfb4bebfa89ab9bbb4aebbb8f4b4bfae">[email protected]</span> <span class="__cf_email__" data-cfemail="9eddb0fafbb0d9ecfffff8dee8ebb0f0f2">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marijn P. A. Sanders</span> - <span class="hlFld-Affiliation affiliation">Computational Drug Discovery Group, Radboud University Nijmegen Medical Centre, Geert Grooteplein,
Nijmegen, The Netherlands</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Luc Roumen</span> - <span class="hlFld-Affiliation affiliation">Leiden/Amsterdam Center of Drug Research (LACDR), Division of Medicinal
Chemistry, Faculty of Sciences, VU University Amsterdam, De Boelelaan 1083, 1081 HV Amsterdam, The Netherlands</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Eelke van der Horst</span> - <span class="hlFld-Affiliation affiliation">Leiden/Amsterdam Center of Drug Research
(LACDR), Division of Medicinal Chemistry, Leiden University, Einsteinweg 55, 2333 CC Leiden, The Netherlands</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">J. Robert Lane</span> - <span class="hlFld-Affiliation affiliation">Leiden/Amsterdam Center of Drug Research
(LACDR), Division of Medicinal Chemistry, Leiden University, Einsteinweg 55, 2333 CC Leiden, The Netherlands</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Henry F. Vischer</span> - <span class="hlFld-Affiliation affiliation">Leiden/Amsterdam Center of Drug Research (LACDR), Division of Medicinal
Chemistry, Faculty of Sciences, VU University Amsterdam, De Boelelaan 1083, 1081 HV Amsterdam, The Netherlands</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jody van Offenbeek</span> - <span class="hlFld-Affiliation affiliation">Leiden/Amsterdam Center of Drug Research (LACDR), Division of Medicinal
Chemistry, Faculty of Sciences, VU University Amsterdam, De Boelelaan 1083, 1081 HV Amsterdam, The Netherlands</span>; 
    <span class="hlFld-Affiliation affiliation">Department of Molecular Pharmacology and DMPK, MSD, Molenstraat 110, 5342 CC Oss, The Netherlands</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Henk de Vries</span> - <span class="hlFld-Affiliation affiliation">Leiden/Amsterdam Center of Drug Research
(LACDR), Division of Medicinal Chemistry, Leiden University, Einsteinweg 55, 2333 CC Leiden, The Netherlands</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stefan Verhoeven</span> - <span class="hlFld-Affiliation affiliation">Department of Molecular Design
and Informatics, MSD, Molenstraat 110,
5342 CC Oss, The Netherlands</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ken Y. Chow</span> - <span class="hlFld-Affiliation affiliation">Leiden/Amsterdam Center of Drug Research (LACDR), Division of Medicinal
Chemistry, Faculty of Sciences, VU University Amsterdam, De Boelelaan 1083, 1081 HV Amsterdam, The Netherlands</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Folkert Verkaar</span> - <span class="hlFld-Affiliation affiliation">Leiden/Amsterdam Center of Drug Research (LACDR), Division of Medicinal
Chemistry, Faculty of Sciences, VU University Amsterdam, De Boelelaan 1083, 1081 HV Amsterdam, The Netherlands</span>; 
    <span class="hlFld-Affiliation affiliation">Department of Molecular Pharmacology and DMPK, MSD, Molenstraat 110, 5342 CC Oss, The Netherlands</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Margot W. Beukers</span> - <span class="hlFld-Affiliation affiliation">Leiden/Amsterdam Center of Drug Research
(LACDR), Division of Medicinal Chemistry, Leiden University, Einsteinweg 55, 2333 CC Leiden, The Netherlands</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ross McGuire</span> - <span class="hlFld-Affiliation affiliation">Department of Molecular Design
and Informatics, MSD, Molenstraat 110,
5342 CC Oss, The Netherlands</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rob Leurs</span> - <span class="hlFld-Affiliation affiliation">Leiden/Amsterdam Center of Drug Research (LACDR), Division of Medicinal
Chemistry, Faculty of Sciences, VU University Amsterdam, De Boelelaan 1083, 1081 HV Amsterdam, The Netherlands</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Adriaan P. IJzerman</span> - <span class="hlFld-Affiliation affiliation">Leiden/Amsterdam Center of Drug Research
(LACDR), Division of Medicinal Chemistry, Leiden University, Einsteinweg 55, 2333 CC Leiden, The Netherlands</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jacob de Vlieg</span> - <span class="hlFld-Affiliation affiliation">Computational Drug Discovery Group, Radboud University Nijmegen Medical Centre, Geert Grooteplein,
Nijmegen, The Netherlands</span>; 
    <span class="hlFld-Affiliation affiliation">Department of Molecular Design
and Informatics, MSD, Molenstraat 110,
5342 CC Oss, The Netherlands</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Iwan J. P. de Esch</span> - <span class="hlFld-Affiliation affiliation">Leiden/Amsterdam Center of Drug Research (LACDR), Division of Medicinal
Chemistry, Faculty of Sciences, VU University Amsterdam, De Boelelaan 1083, 1081 HV Amsterdam, The Netherlands</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Guido J. R. Zaman</span> - <span class="hlFld-Affiliation affiliation">Department of Molecular Pharmacology and DMPK, MSD, Molenstraat 110, 5342 CC Oss, The Netherlands</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d769e1611-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i39">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_06813" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_06813" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by the Dutch Top Institute Pharma (TI Pharma), Project No. D1-105, and The Netherlands Organisation for Scientific Research (NWO) through a VENI grant (No. 700.59.408 to C.d.G.).</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i40" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i40"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i41" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i41"> Abbreviations Used</h2><tr><td class="NLM_term">AA2AR</td><td class="NLM_def"><p class="first last">adenosine A<sub>2A</sub> receptor</p></td></tr><tr><td class="NLM_term">ADRB2</td><td class="NLM_def"><p class="first last">adrenergic β-2 receptor</p></td></tr><tr><td class="NLM_term">BCI</td><td class="NLM_def"><p class="first last">Barnard Chemical Information</p></td></tr><tr><td class="NLM_term">ECFP</td><td class="NLM_def"><p class="first last">extended connectivity fingerprint</p></td></tr><tr><td class="NLM_term">FCFP</td><td class="NLM_def"><p class="first last">functional class fingerprint</p></td></tr><tr><td class="NLM_term">GPCR</td><td class="NLM_def"><p class="first last">G-protein-coupled receptor</p></td></tr><tr><td class="NLM_term">LB</td><td class="NLM_def"><p class="first last">ligand based</p></td></tr><tr><td class="NLM_term">PDE5</td><td class="NLM_def"><p class="first last">phosphodiesterase 5</p></td></tr><tr><td class="NLM_term">S1PR1</td><td class="NLM_def"><p class="first last">sphingosine 1-phosphate receptor</p></td></tr><tr><td class="NLM_term">S1P</td><td class="NLM_def"><p class="first last">sphingosine 1-phosphate</p></td></tr><tr><td class="NLM_term"><i>T</i><sub>C</sub></td><td class="NLM_def"><p class="first last">Tanimoto coefficient</p></td></tr><tr><td class="NLM_term">TM</td><td class="NLM_def"><p class="first last">transmembrane (helix)</p></td></tr><tr><td class="NLM_term">SBP</td><td class="NLM_def"><p class="first last">structure based pharmacophore</p></td></tr><tr><td class="NLM_term">VS</td><td class="NLM_def"><p class="first last">virtual screening</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i42">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_72876" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_72876" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 77 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Kubinyi, H., Muller, G.</span>, Eds.  <span class="citation_source-book">Chemogenomics in Drug Discovery: A Medicinal Chemistry Perspective</span>; <span class="NLM_publisher-name">Wiley-VCH</span>: <span class="NLM_publisher-loc">Weinheim, Germany</span>,<span class="NLM_x"> </span><span class="NLM_year">2004</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1002%2F3527603948" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2004&author=H.+Kubinyi&author=G.+Muller&title=Chemogenomics+in+Drug+Discovery%3A+A+Medicinal+Chemistry+Perspective"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1002%2F3527603948&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F3527603948%26sid%3Dliteratum%253Aachs%26aulast%3DKubinyi%26aufirst%3DH.%26btitle%3DChemogenomics%2520in%2520Drug%2520Discovery%253A%2520A%2520Medicinal%2520Chemistry%2520Perspective%26pub%3DWiley-VCH%26date%3D2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Bredel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacoby, E.</span><span> </span><span class="NLM_article-title">Chemogenomics: an emerging strategy for rapid target and drug discovery</span> <span class="citation_source-journal">Nat. Rev. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">262</span><span class="NLM_x">–</span> <span class="NLM_lpage">275</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1038%2Fnrg1317" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=15131650" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjtVartL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2004&pages=262-275&author=M.+Bredelauthor=E.+Jacoby&title=Chemogenomics%3A+an+emerging+strategy+for+rapid+target+and+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Chemogenomics: an emerging strategy for rapid target and drug discovery</span></div><div class="casAuthors">Bredel, Markus; Jacoby, Edgar</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">262-275</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Chemogenomics is an emerging discipline that combines the latest tools of genomics and chem. and applies them to target and drug discovery.  Its strength lies in eliminating the bottleneck that currently occurs in target identification by measuring the broad, conditional effects of chem. libraries on whole biol. systems or by screening large chem. libraries quickly and efficiently against selected targets.  The hope is that chemogenomics will concurrently identify and validate therapeutic targets and detect drug candidates to rapidly and effectively generate new treatments for many human diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobwhXA51PclrVg90H21EOLACvtfcHk0licRcHKDCkWtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjtVartL0%253D&md5=ba7dbd1509189fcd2dd5e85a80a8e248</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrg1317&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg1317%26sid%3Dliteratum%253Aachs%26aulast%3DBredel%26aufirst%3DM.%26aulast%3DJacoby%26aufirst%3DE.%26atitle%3DChemogenomics%253A%2520an%2520emerging%2520strategy%2520for%2520rapid%2520target%2520and%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2004%26volume%3D5%26spage%3D262%26epage%3D275" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Harris, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevens, A. P.</span><span> </span><span class="NLM_article-title">Chemogenomics: structuring the drug discovery process to gene families</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">880</span><span class="NLM_x">–</span> <span class="NLM_lpage">888</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2006&pages=880-888&author=C.+J.+Harrisauthor=A.+P.+Stevens&title=Chemogenomics%3A+structuring+the+drug+discovery+process+to+gene+families"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHarris%26aufirst%3DC.%2BJ.%26aulast%3DStevens%26aufirst%3DA.%2BP.%26atitle%3DChemogenomics%253A%2520structuring%2520the%2520drug%2520discovery%2520process%2520to%2520gene%2520families%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2006%26volume%3D11%26spage%3D880%26epage%3D888" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Caron, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mullican, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mashal, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, K. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murcko, M. A.</span><span> </span><span class="NLM_article-title">Chemogenomic approaches to drug discovery</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">464</span><span class="NLM_x">–</span> <span class="NLM_lpage">470</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1016%2FS1367-5931%2800%2900229-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=11470611" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD3MXltlyqsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2001&pages=464-470&author=P.+R.+Caronauthor=M.+D.+Mullicanauthor=R.+D.+Mashalauthor=K.+P.+Wilsonauthor=M.+S.+Suauthor=M.+A.+Murcko&title=Chemogenomic+approaches+to+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Chemogenomic approaches to drug discovery</span></div><div class="casAuthors">Caron, P. R.; Mullican, M. D.; Mashal, R. D.; Wilson, K. P.; Su, M. S.; Murcko, M. A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">464-470</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review, with 39 refs., on the role of completed human genome sequence in developing novel therapeutics.  Topics discussed include efficiency of gene families; gene-family mapping; chemogenomics through mutagenesis and in the absence of structure; chemogenomic examples; key issues for chemogenomics; and bottlenecks in chemogenomics.  The chemogenomics approach, in which the completed human genome sequence is combined with protein structure and/or models to link to chem. inhibitors, is designed to fully use the sequence information by considering large families of gene targets at once.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSDlKQcY9zvrVg90H21EOLACvtfcHk0lgyOL-RgsWXdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXltlyqsb8%253D&md5=ea59ee2fcc9660c29b361537357e7e3f</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2FS1367-5931%2800%2900229-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1367-5931%252800%252900229-5%26sid%3Dliteratum%253Aachs%26aulast%3DCaron%26aufirst%3DP.%2BR.%26aulast%3DMullican%26aufirst%3DM.%2BD.%26aulast%3DMashal%26aufirst%3DR.%2BD.%26aulast%3DWilson%26aufirst%3DK.%2BP.%26aulast%3DSu%26aufirst%3DM.%2BS.%26aulast%3DMurcko%26aufirst%3DM.%2BA.%26atitle%3DChemogenomic%2520approaches%2520to%2520drug%2520discovery%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2001%26volume%3D5%26spage%3D464%26epage%3D470" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">Chemogenomic approaches to rational drug design</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">152</span><span class="NLM_x">, </span> <span class="NLM_fpage">38</span><span class="NLM_x">–</span> <span class="NLM_lpage">52</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1038%2Fsj.bjp.0707307" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=17533416" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpsFOjurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=152&publication_year=2007&pages=38-52&author=D.+Rognan&title=Chemogenomic+approaches+to+rational+drug+design"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Chemogenomic approaches to rational drug design</span></div><div class="casAuthors">Rognan, D.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">152</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">38-52</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Paradigms in drug design and discovery are changing at a significant pace.  Concomitant to the sequencing of over 180 several genomes, the high-throughput miniaturization of chem. synthesis and biol. evaluation of a multiple compds. on gene/protein expression and function opens the way to global drug-discovery approaches, no more focused on a single target but on an entire family of related proteins or on a full metabolic pathway.  Chemogenomics is this emerging research field aimed at systematically studying the biol. effect of a wide array of small mol.-wt. ligands on a wide array of macromol. targets.  Since the quantity of existing data (compds., targets and assays) and of produced information (gene/protein expression levels and binding consts.) are too large for manual manipulation, information technologies play a crucial role in planning, analyzing and predicting chemogenomic data.  The present review will focus on predictive in silico chemogenomic approaches to foster rational drug design and derive information from the simultaneous biol. evaluation of multiple compds. on multiple targets.  State-of-the-art methods for navigating in either ligand or target space will be presented and concrete drug design applications will be mentioned.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPERITBubs0LVg90H21EOLACvtfcHk0lgyOL-RgsWXdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpsFOjurw%253D&md5=47b3b2a5155a079b486e56d31e0ca59c</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjp.0707307&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjp.0707307%26sid%3Dliteratum%253Aachs%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DChemogenomic%2520approaches%2520to%2520rational%2520drug%2520design%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2007%26volume%3D152%26spage%3D38%26epage%3D52" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">Structure-based approaches to target fishing and ligand profiling</span> <span class="citation_source-journal">Mol. Inf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">176</span><span class="NLM_x">–</span> <span class="NLM_lpage">187</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1002%2Fminf.200900081" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=27462761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjs12ntLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2010&pages=176-187&author=D.+Rognan&title=Structure-based+approaches+to+target+fishing+and+ligand+profiling"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Approaches to Target Fishing and Ligand Profiling</span></div><div class="casAuthors">Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Informatics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">176-187</span>CODEN:
                <span class="NLM_cas:coden">MIONBS</span>;
        ISSN:<span class="NLM_cas:issn">1868-1743</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Chemogenomics is an emerging interdisciplinary field aiming at identifying all possible ligands of all possible targets.  If one groups targets in columns and ligands in rows, chemogenomic approaches to drug discovery just fill the interaction matrix.  Since exptl. data do not suffice, several computational methods are currently actively developed to supplement time-consuming and costly expts.  They are either designed to fill rows and thus profile a ligand towards a heterogeneous set of targets (target profiling) or to fill columns and thus identify novel ligands for an existing target (std. virtual screening).  At the interface of both strategies are now true chemogenomic computational methods filling well defined areas in the matrix.  The present review will focus on (protein) structure-based approaches and illustrates major advances in this novel exciting field which is supposed to massively impact rational drug design in the next decade.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzdpIh1yWS_7Vg90H21EOLACvtfcHk0liuOt1yGhhImw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjs12ntLo%253D&md5=e32b663e9a43bb1621d07d07277fa6b5</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1002%2Fminf.200900081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fminf.200900081%26sid%3Dliteratum%253Aachs%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DStructure-based%2520approaches%2520to%2520target%2520fishing%2520and%2520ligand%2520profiling%26jtitle%3DMol.%2520Inf.%26date%3D2010%26volume%3D29%26spage%3D176%26epage%3D187" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Keiser, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Setola, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laggner, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abbas, A. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hufeisen, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jensen, N. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuijer, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matos, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tran, T. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whaley, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glennon, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hert, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, D. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span> </span><span class="NLM_article-title">Predicting new molecular targets for known drugs</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">462</span><span class="NLM_x">, </span> <span class="NLM_fpage">175</span><span class="NLM_x">–</span> <span class="NLM_lpage">181</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1038%2Fnature08506" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=19881490" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlers7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=462&publication_year=2009&pages=175-181&author=M.+J.+Keiserauthor=V.+Setolaauthor=J.+J.+Irwinauthor=C.+Laggnerauthor=A.+I.+Abbasauthor=S.+J.+Hufeisenauthor=N.+H.+Jensenauthor=M.+B.+Kuijerauthor=R.+C.+Matosauthor=T.+B.+Tranauthor=R.+Whaleyauthor=R.+A.+Glennonauthor=J.+Hertauthor=K.+L.+Thomasauthor=D.+D.+Edwardsauthor=B.+K.+Shoichetauthor=B.+L.+Roth&title=Predicting+new+molecular+targets+for+known+drugs"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting new molecular targets for known drugs</span></div><div class="casAuthors">Keiser, Michael J.; Setola, Vincent; Irwin, John J.; Laggner, Christian; Abbas, Atheir I.; Hufeisen, Sandra J.; Jensen, Niels H.; Kuijer, Michael B.; Matos, Roberto C.; Tran, Thuy B.; Whaley, Ryan; Glennon, Richard A.; Hert, Jerome; Thomas, Kelan L. H.; Edwards, Douglas D.; Shoichet, Brian K.; Roth, Bryan L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">462</span>
        (<span class="NLM_cas:issue">7270</span>),
    <span class="NLM_cas:pages">175-181</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Although drugs are intended to be selective, at least some bind to several physiol. targets, explaining side effects and efficacy.  Because many drug-target combinations exist, it would be useful to explore possible interactions computationally.  Here we compared 3665 US Food and Drug Administration (FDA)-approved and investigational drugs against hundreds of targets, defining each target by its ligands.  Chem. similarities between drugs and ligand sets predicted thousands of unanticipated assocns.  Thirty were tested exptl., including the antagonism of the β1 receptor by the transporter inhibitor Prozac, the inhibition of the 5-hydroxytryptamine (5-HT) transporter by the ion channel drug Vadilex, and antagonism of the histamine H4 receptor by the enzyme inhibitor Rescriptor.  Overall, 23 new drug-target assocns. were confirmed, five of which were potent (<100 nM).  The physiol. relevance of one, the drug N,N-dimethyltryptamine (DMT) on serotonergic receptors, was confirmed in a knockout mouse.  The chem. similarity approach is systematic and comprehensive, and may suggest side-effects and new indications for many drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozd-8cCXUskrVg90H21EOLACvtfcHk0liuOt1yGhhImw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlers7vO&md5=974e65f5d5b4dabde1a463c429215546</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnature08506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08506%26sid%3Dliteratum%253Aachs%26aulast%3DKeiser%26aufirst%3DM.%2BJ.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DLaggner%26aufirst%3DC.%26aulast%3DAbbas%26aufirst%3DA.%2BI.%26aulast%3DHufeisen%26aufirst%3DS.%2BJ.%26aulast%3DJensen%26aufirst%3DN.%2BH.%26aulast%3DKuijer%26aufirst%3DM.%2BB.%26aulast%3DMatos%26aufirst%3DR.%2BC.%26aulast%3DTran%26aufirst%3DT.%2BB.%26aulast%3DWhaley%26aufirst%3DR.%26aulast%3DGlennon%26aufirst%3DR.%2BA.%26aulast%3DHert%26aufirst%3DJ.%26aulast%3DThomas%26aufirst%3DK.%2BL.%26aulast%3DEdwards%26aufirst%3DD.%2BD.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26atitle%3DPredicting%2520new%2520molecular%2520targets%2520for%2520known%2520drugs%26jtitle%3DNature%26date%3D2009%26volume%3D462%26spage%3D175%26epage%3D181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Badrinarayan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sastry, G. N.</span><span> </span><span class="NLM_article-title">Virtual high-throughput screening in new lead identification</span> <span class="citation_source-journal">Comb. Chem. High Throughput Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">840</span><span class="NLM_x">–</span> <span class="NLM_lpage">860</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.2174%2F138620711797537102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=21843146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFOhsLrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2011&pages=840-860&author=P.+Badrinarayanauthor=G.+N.+Sastry&title=Virtual+high-throughput+screening+in+new+lead+identification"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual high throughput screening in new lead identification</span></div><div class="casAuthors">Badrinarayan, Preethi; Sastry, G. Narahari</div><div class="citationInfo"><span class="NLM_cas:title">Combinatorial Chemistry & High Throughput Screening</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">840-860</span>CODEN:
                <span class="NLM_cas:coden">CCHSFU</span>;
        ISSN:<span class="NLM_cas:issn">1386-2073</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Drug discovery continues to be one of the greatest contemporary challenges and rational application of modeling approaches is the first important step to obtain lead compds., which can be optimized further.  Virtual high throughput screening (VHTS) is one of the efficient approaches to obtain lead structures for a given target.  Strategic application of different screening filters like pharmacophore mapping, shape-based, ligand-based, mol. similarity etc., in combination with other drug design protocols provide invaluable insights in lead identification and optimization.  Screening of large databases using these computational methods provides potential lead compds., thus triggering a meaningful interplay between computations and expts.  In this review, we present a crit. account on the relevance of mol. modeling approaches in general, lead optimization and virtual screening methods in particular for new lead identification.  The importance of developing reliable scoring functions for non-bonded interactions has been highlighted, as it is an extremely important measure for the reliability of scoring function.  The lead optimization and new lead design has also been illustrated with examples.  The importance of employing a combination of general and target specific screening protocols has also been highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPpEYhrdA6gbVg90H21EOLACvtfcHk0ljEaOdbNgefrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFOhsLrP&md5=a4f42fd1ae6ec14951648ee6ad6a588d</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.2174%2F138620711797537102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138620711797537102%26sid%3Dliteratum%253Aachs%26aulast%3DBadrinarayan%26aufirst%3DP.%26aulast%3DSastry%26aufirst%3DG.%2BN.%26atitle%3DVirtual%2520high-throughput%2520screening%2520in%2520new%2520lead%2520identification%26jtitle%3DComb.%2520Chem.%2520High%2520Throughput%2520Screening%26date%3D2011%26volume%3D14%26spage%3D840%26epage%3D860" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Seifert, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kraus, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kramer, B.</span><span> </span><span class="NLM_article-title">Virtual high-throughput screening of molecular databases</span> <span class="citation_source-journal">Curr. Opin. Drug Discovery Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">298</span><span class="NLM_x">–</span> <span class="NLM_lpage">307</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1007%2F978-1-59745-455-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=17554856" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmtVWlsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2007&pages=298-307&author=M.+H.+Seifertauthor=J.+Krausauthor=B.+Kramer&title=Virtual+high-throughput+screening+of+molecular+databases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual high-throughput screening of molecular databases</span></div><div class="casAuthors">Seifert, Markus H. J.; Kraus, Juergen; Kramer, Bernd</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Drug Discovery & Development</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">298-307</span>CODEN:
                <span class="NLM_cas:coden">CODDFF</span>;
        ISSN:<span class="NLM_cas:issn">1367-6733</span>.
    
            (<span class="NLM_cas:orgname">Thomson Scientific</span>)
        </div><div class="casAbstract">A review.  Virtual high-throughput screening (vHTS) is an efficient and widely applicable method used to identify initial hit compds. for pharmaceutical research.  Despite its widespread use, several aspects of protein structure-based vHTS can still be optimized, particularly its accuracy and speed in generating results.  Recent developments that address these issues include machine learning and implicit solvation methods.  Various machine learning methods are available to improve vHTS accuracy, for example, target-specific optimization of scoring functions, the integration of essential protein-ligand interactions, and the application of neg. training data.  Implicit solvation methods are exemplified by the mol. mechanics Poisson-Boltzmann solvent accessible surface area approach.  Furthermore, grid computing and intelligent database screening approaches are used to improve the speed of vHTS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRO6rcUw_QI7Vg90H21EOLACvtfcHk0ljEaOdbNgefrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmtVWlsrc%253D&md5=7e4efb79644d937514d93558f54cd575</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1007%2F978-1-59745-455-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-59745-455-1%26sid%3Dliteratum%253Aachs%26aulast%3DSeifert%26aufirst%3DM.%2BH.%26aulast%3DKraus%26aufirst%3DJ.%26aulast%3DKramer%26aufirst%3DB.%26atitle%3DVirtual%2520high-throughput%2520screening%2520of%2520molecular%2520databases%26jtitle%3DCurr.%2520Opin.%2520Drug%2520Discovery%2520Dev.%26date%3D2007%26volume%3D10%26spage%3D298%26epage%3D307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Reddy, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pati, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pradeep, H. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sastry, G. N.</span><span> </span><span class="NLM_article-title">Virtual screening in drug discovery—a computational perspective</span> <span class="citation_source-journal">Curr. Protein Pept. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">329</span><span class="NLM_x">–</span> <span class="NLM_lpage">351</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.2174%2F138920307781369427" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=17696867" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpslKrs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=329-351&author=A.+S.+Reddyauthor=S.+P.+Patiauthor=P.+P.+Kumarauthor=H.+N.+Pradeepauthor=G.+N.+Sastry&title=Virtual+screening+in+drug+discovery%E2%80%94a+computational+perspective"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual screening in drug discovery - a computational perspective</span></div><div class="casAuthors">Reddy, A. Srinivas; Pati, S. Priyadarshini; Kumar, P. Praveen; Pradeep, H. N.; Sastry, G. Narahari</div><div class="citationInfo"><span class="NLM_cas:title">Current Protein and Peptide Science</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">329-351</span>CODEN:
                <span class="NLM_cas:coden">CPPSCM</span>;
        ISSN:<span class="NLM_cas:issn">1389-2037</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Virtual screening emerged as an important tool in our quest to access novel drug like compds.  There are a wide range of comparable and contrasting methodol. protocols available in screening databases for the lead compds.  The no. of methods and software packages which employ the target and ligand based virtual screening are increasing at a rapid pace.  However, the general understanding on the applicability and limitations of these methodologies is not emerging as fast as the developments of various methods.  Therefore, it is extremely important to compare and contrast various protocols with practical examples to gauge the strength and applicability of various methods.  The review provides a comprehensive appraisal on several of the available virtual screening methods to-date.  Recent developments of the docking and similarity based methods have been discussed besides the descriptor selection and pharmacophore based searching.  The review touches upon the application of statistical, graph theory based methods machine learning tools in virtual screening and combinatorial library design.  Finally, several case studies are undertaken where the virtual screening technol. has been applied successfully.  A crit. anal. of these case studies provides a good platform to est. the applicability of various virtual screening methods in the new lead identification and optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUR2m_0xSzaLVg90H21EOLACvtfcHk0ljEaOdbNgefrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpslKrs70%253D&md5=ad933e5e65e213edc9c37e9c973c696e</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.2174%2F138920307781369427&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138920307781369427%26sid%3Dliteratum%253Aachs%26aulast%3DReddy%26aufirst%3DA.%2BS.%26aulast%3DPati%26aufirst%3DS.%2BP.%26aulast%3DKumar%26aufirst%3DP.%2BP.%26aulast%3DPradeep%26aufirst%3DH.%2BN.%26aulast%3DSastry%26aufirst%3DG.%2BN.%26atitle%3DVirtual%2520screening%2520in%2520drug%2520discovery%25E2%2580%2594a%2520computational%2520perspective%26jtitle%3DCurr.%2520Protein%2520Pept.%2520Sci.%26date%3D2007%26volume%3D8%26spage%3D329%26epage%3D351" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Brianso, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carrascosa, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oprea, T. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mestres, J.</span><span> </span><span class="NLM_article-title">Cross-pharmacology analysis of G protein-coupled receptors</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1956</span><span class="NLM_x">–</span> <span class="NLM_lpage">1963</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.2174%2F156802611796391285" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=21851335" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVSmsLzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2011&pages=1956-1963&author=F.+Briansoauthor=M.+C.+Carrascosaauthor=T.+I.+Opreaauthor=J.+Mestres&title=Cross-pharmacology+analysis+of+G+protein-coupled+receptors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Cross-pharmacology analysis of G protein-coupled receptors</span></div><div class="casAuthors">Brianso, Ferran; Carrascosa, Maria C.; Oprea, Tudor I.; Mestres, Jordi</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">1956-1963</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The degree of applicability of chemogenomic approaches to protein families depends on the accuracy and completeness of pharmacol. data and the corresponding level of pharmacol. similarity obsd. among their protein members.  The recent public domain availability of pharmacol. data for thousands of small mols. on 204 G protein-coupled receptors (GPCRs) provides a firm basis for an in-depth cross-pharmacol. anal. of this superfamily.  The no. of protein targets included in the cross-pharmacol. profile of the different GPCRs changes significantly upon varying the ligand similarity and binding affinity criteria.  However, with the exception of muscarinic receptors, aminergic GPCRs distinguish themselves from the rest of the members in the family by their remarkably high levels of pharmacol. similarity among them.  Clusters of non-GPCR targets related by cross-pharmacol. with particular GPCRs are identified and the implications for unwanted side-effects, as well as for repurposing opportunities, discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcxMaSMU_YxLVg90H21EOLACvtfcHk0lhU2lTfInOuhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVSmsLzO&md5=7805626c0844e84d16b56ed953823486</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.2174%2F156802611796391285&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802611796391285%26sid%3Dliteratum%253Aachs%26aulast%3DBrianso%26aufirst%3DF.%26aulast%3DCarrascosa%26aufirst%3DM.%2BC.%26aulast%3DOprea%26aufirst%3DT.%2BI.%26aulast%3DMestres%26aufirst%3DJ.%26atitle%3DCross-pharmacology%2520analysis%2520of%2520G%2520protein-coupled%2520receptors%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2011%26volume%3D11%26spage%3D1956%26epage%3D1963" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Mestres, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gregori-Puigjane, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valverde, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sole, R. V.</span><span> </span><span class="NLM_article-title">Data completeness—the Achilles heel of drug–target networks</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">983</span><span class="NLM_x">–</span> <span class="NLM_lpage">984</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=983-984&author=J.+Mestresauthor=E.+Gregori-Puigjaneauthor=S.+Valverdeauthor=R.+V.+Sole&title=Data+completeness%E2%80%94the+Achilles+heel+of+drug%E2%80%93target+networks"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMestres%26aufirst%3DJ.%26aulast%3DGregori-Puigjane%26aufirst%3DE.%26aulast%3DValverde%26aufirst%3DS.%26aulast%3DSole%26aufirst%3DR.%2BV.%26atitle%3DData%2520completeness%25E2%2580%2594the%2520Achilles%2520heel%2520of%2520drug%25E2%2580%2593target%2520networks%26jtitle%3DNat.%2520Biotechnol.%26date%3D2008%26volume%3D26%26spage%3D983%26epage%3D984" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Gaulton, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellis, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bento, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chambers, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hersey, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Light, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGlinchey, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michalovich, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Lazikani, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Overington, J. P.</span><span> </span><span class="NLM_article-title">ChEMBL: a large-scale bioactivity database for drug discovery</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">D1100</span><span class="NLM_x">–</span> <span class="NLM_lpage">D1107</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1093%2Fnar%2Fgkr777" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=21948594" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs12htbjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=D1100-D1107&author=A.+Gaultonauthor=L.+J.+Bellisauthor=A.+P.+Bentoauthor=J.+Chambersauthor=M.+Daviesauthor=A.+Herseyauthor=Y.+Lightauthor=S.+McGlincheyauthor=D.+Michalovichauthor=B.+Al-Lazikaniauthor=J.+P.+Overington&title=ChEMBL%3A+a+large-scale+bioactivity+database+for+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">ChEMBL: a large-scale bioactivity database for drug discovery</span></div><div class="casAuthors">Gaulton, Anna; Bellis, Louisa J.; Bento, A. Patricia; Chambers, Jon; Davies, Mark; Hersey, Anne; Light, Yvonne; McGlinchey, Shaun; Michalovich, David; Al-Lazikani, Bissan; Overington, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1100-D1107</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ChEMBL is an Open Data database contg. binding, functional and ADMET information for a large no. of drug-like bioactive compds.  These data are manually abstracted from the primary published literature on a regular basis, then further curated and standardized to maximize their quality and utility across a wide range of chem. biol. and drug-discovery research problems.  Currently, the database contains 5.4 million bioactivity measurements for more than 1 million compds. and 5200 protein targets.  Access is available through a web-based interface, data downloads and web services at: https://www.ebi.ac.uk/chembldb.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9e9jRENTtubVg90H21EOLACvtfcHk0lhU2lTfInOuhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs12htbjN&md5=aedf7793e1ca54b6a4fa272ea3ef7d0e</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr777&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr777%26sid%3Dliteratum%253Aachs%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DBellis%26aufirst%3DL.%2BJ.%26aulast%3DBento%26aufirst%3DA.%2BP.%26aulast%3DChambers%26aufirst%3DJ.%26aulast%3DDavies%26aufirst%3DM.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DLight%26aufirst%3DY.%26aulast%3DMcGlinchey%26aufirst%3DS.%26aulast%3DMichalovich%26aufirst%3DD.%26aulast%3DAl-Lazikani%26aufirst%3DB.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DChEMBL%253A%2520a%2520large-scale%2520bioactivity%2520database%2520for%2520drug%2520discovery%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3DD1100%26epage%3DD1107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Knox, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Law, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jewison, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ly, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frolkis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pon, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banco, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mak, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neveu, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Djoumbou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eisner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wishart, D. S.</span><span> </span><span class="NLM_article-title">DrugBank 3.0: a comprehensive resource for “omics” research on drugs</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">D1035</span><span class="NLM_x">–</span> <span class="NLM_lpage">D1041</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2011&pages=D1035-D1041&author=C.+Knoxauthor=V.+Lawauthor=T.+Jewisonauthor=P.+Liuauthor=S.+Lyauthor=A.+Frolkisauthor=A.+Ponauthor=K.+Bancoauthor=C.+Makauthor=V.+Neveuauthor=Y.+Djoumbouauthor=R.+Eisnerauthor=A.+C.+Guoauthor=D.+S.+Wishart&title=DrugBank+3.0%3A+a+comprehensive+resource+for+%E2%80%9Comics%E2%80%9D+research+on+drugs"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKnox%26aufirst%3DC.%26aulast%3DLaw%26aufirst%3DV.%26aulast%3DJewison%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DLy%26aufirst%3DS.%26aulast%3DFrolkis%26aufirst%3DA.%26aulast%3DPon%26aufirst%3DA.%26aulast%3DBanco%26aufirst%3DK.%26aulast%3DMak%26aufirst%3DC.%26aulast%3DNeveu%26aufirst%3DV.%26aulast%3DDjoumbou%26aufirst%3DY.%26aulast%3DEisner%26aufirst%3DR.%26aulast%3DGuo%26aufirst%3DA.%2BC.%26aulast%3DWishart%26aufirst%3DD.%2BS.%26atitle%3DDrugBank%25203.0%253A%2520a%2520comprehensive%2520resource%2520for%2520%25E2%2580%259Comics%25E2%2580%259D%2520research%2520on%2520drugs%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2011%26volume%3D39%26spage%3DD1035%26epage%3DD1041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilson, M. K.</span><span> </span><span class="NLM_article-title">The binding database: overview and user’s guide</span> <span class="citation_source-journal">Biopolymers</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x">, </span> <span class="NLM_fpage">127</span><span class="NLM_x">–</span> <span class="NLM_lpage">141</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2001&pages=127-141&author=X.+Chenauthor=Y.+Linauthor=M.+K.+Gilson&title=The+binding+database%3A+overview+and+user%E2%80%99s+guide"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DGilson%26aufirst%3DM.%2BK.%26atitle%3DThe%2520binding%2520database%253A%2520overview%2520and%2520user%25E2%2580%2599s%2520guide%26jtitle%3DBiopolymers%26date%3D2001%26volume%3D61%26spage%3D127%26epage%3D141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilson, M. K.</span><span> </span><span class="NLM_article-title">The binding database: data management and interface design</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">130</span><span class="NLM_x">–</span> <span class="NLM_lpage">139</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2002&pages=130-139&author=X.+Chenauthor=Y.+Linauthor=M.+Liuauthor=M.+K.+Gilson&title=The+binding+database%3A+data+management+and+interface+design"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DGilson%26aufirst%3DM.%2BK.%26atitle%3DThe%2520binding%2520database%253A%2520data%2520management%2520and%2520interface%2520design%26jtitle%3DBioinformatics%26date%3D2002%26volume%3D18%26spage%3D130%26epage%3D139" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilson, M. K.</span><span> </span><span class="NLM_article-title">BindingDB: a Web-accessible molecular recognition database</span> <span class="citation_source-journal">Comb. Chem. High Throughput Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">719</span><span class="NLM_x">–</span> <span class="NLM_lpage">725</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.2174%2F1386207013330670" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=11812264" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD38Xks1Orsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2001&pages=719-725&author=X.+Chenauthor=M.+Liuauthor=M.+K.+Gilson&title=BindingDB%3A+a+Web-accessible+molecular+recognition+database"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">BindingDB: A Web-accessible molecular recognition database</span></div><div class="casAuthors">Chen, Xi; Liu, Ming; Gilson, Michael K.</div><div class="citationInfo"><span class="NLM_cas:title">Combinatorial Chemistry and High Throughput Screening</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">719-725</span>CODEN:
                <span class="NLM_cas:coden">CCHSFU</span>;
        ISSN:<span class="NLM_cas:issn">1386-2073</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers</span>)
        </div><div class="casAbstract">The BindingDB, a public web-accessible database of measured binding affinities for various mol. types, is described.  It allows queries based upon a range of criteria, including chem. similarity or substructure, sequence homol., numerical criteria and reactant names.  Principles of Human-Computer Interactions are being employed in creating the query interface and user-feedback is being solicited.  The data specification includes significant exptl. detail.  A full dictionary has been created for isothermal titrn. calorimetry data in consultation with experimentalists and data dictionaries for enzyme inhibition and other measurement techniques are being developed.  Currently, the BindingDB contains several data sets of broad interest, such as antigen-antibody binding and cyclodextrin/small-mol. binding.  However, it is anticipated that online deposition by experimentalists will ultimately contribute a larger flow of data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzRKVfJZtKwrVg90H21EOLACvtfcHk0lgNG_DUTmi3Jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xks1Orsg%253D%253D&md5=9db61dc1ad10e8191ee11afcfa9e337e</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.2174%2F1386207013330670&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1386207013330670%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DGilson%26aufirst%3DM.%2BK.%26atitle%3DBindingDB%253A%2520a%2520Web-accessible%2520molecular%2520recognition%2520database%26jtitle%3DComb.%2520Chem.%2520High%2520Throughput%2520Screening%26date%3D2001%26volume%3D4%26spage%3D719%26epage%3D725" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Wang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span> </span><span class="NLM_article-title">The PDBbind database: collection of binding affinities for protein–ligand complexes with known three-dimensional structures</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">2977</span><span class="NLM_x">–</span> <span class="NLM_lpage">2980</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm030580l" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=2977-2980&author=R.+Wangauthor=X.+Fangauthor=Y.+Luauthor=S.+Wang&title=The+PDBbind+database%3A+collection+of+binding+affinities+for+protein%E2%80%93ligand+complexes+with+known+three-dimensional+structures"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm030580l&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030580l%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DR.%26aulast%3DFang%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DThe%2520PDBbind%2520database%253A%2520collection%2520of%2520binding%2520affinities%2520for%2520protein%25E2%2580%2593ligand%2520complexes%2520with%2520known%2520three-dimensional%2520structures%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D2977%26epage%3D2980" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Wang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, C. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span> </span><span class="NLM_article-title">The PDBbind database: methodologies and updates</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">4111</span><span class="NLM_x">–</span> <span class="NLM_lpage">4119</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm048957q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD2MXktlKisbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=4111-4119&author=R.+Wangauthor=X.+Fangauthor=Y.+Luauthor=C.+Y.+Yangauthor=S.+Wang&title=The+PDBbind+database%3A+methodologies+and+updates"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">The PDBbind Database: Methodologies and Updates</span></div><div class="casAuthors">Wang, Renxiao; Fang, Xueliang; Lu, Yipin; Yang, Chao-Yie; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4111-4119</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors have developed the PDBbind database to provide a comprehensive collection of binding affinities for the protein-ligand complexes in the Protein Data Bank (PDB).  This paper gives a full description of the latest version, i.e., version 2003, which is an update to our recently reported work.  Out of 23 790 entries in the PDB release No.107 (Jan. 2004), 5897 entries were identified as protein-ligand complexes that meet our definition.  Exptl. detd. binding affinities (Kd, Ki, and IC50) for 1622 of these were retrieved from the refs. assocd. with these complexes.  A total of 900 complexes were selected to form a "refined set", which is of particular value as a std. data set for docking and scoring studies.  All of the final data, including binding affinity data, ref. citations, and processed structural files, have been incorporated into the PDBbind database accessible online at http:// www.pdbbind.org/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD3MSeASmZZrVg90H21EOLACvtfcHk0lgNG_DUTmi3Jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXktlKisbg%253D&md5=4a5725b9464537310773862d65e8b2fe</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjm048957q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm048957q%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DR.%26aulast%3DFang%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DThe%2520PDBbind%2520database%253A%2520methodologies%2520and%2520updates%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D4111%26epage%3D4119" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Hu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benson, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lerner, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlson, H. A.</span><span> </span><span class="NLM_article-title">Binding MOAD (mother of all databases)</span> <span class="citation_source-journal">Proteins</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">333</span><span class="NLM_x">–</span> <span class="NLM_lpage">340</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2005&pages=333-340&author=L.+Huauthor=M.+L.+Bensonauthor=R.+D.+Smithauthor=M.+G.+Lernerauthor=H.+A.+Carlson&title=Binding+MOAD+%28mother+of+all+databases%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DL.%26aulast%3DBenson%26aufirst%3DM.%2BL.%26aulast%3DSmith%26aufirst%3DR.%2BD.%26aulast%3DLerner%26aufirst%3DM.%2BG.%26aulast%3DCarlson%26aufirst%3DH.%2BA.%26atitle%3DBinding%2520MOAD%2520%2528mother%2520of%2520all%2520databases%2529%26jtitle%3DProteins%26date%3D2005%26volume%3D60%26spage%3D333%26epage%3D340" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Smith, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Falkner, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benson, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nerothin, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlson, H. A.</span><span> </span><span class="NLM_article-title">Exploring protein–ligand recognition with binding MOAD</span> <span class="citation_source-journal">J. Mol. Graphics Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">414</span><span class="NLM_x">–</span> <span class="NLM_lpage">425</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1016%2Fj.jmgm.2005.08.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD28XjtFahu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2006&pages=414-425&author=R.+D.+Smithauthor=L.+Huauthor=J.+A.+Falknerauthor=M.+L.+Bensonauthor=J.+P.+Nerothinauthor=H.+A.+Carlson&title=Exploring+protein%E2%80%93ligand+recognition+with+binding+MOAD"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring protein-ligand recognition with Binding MOAD</span></div><div class="casAuthors">Smith, Richard D.; Hu, Liegi; Falkner, Jayson A.; Benson, Mark L.; Nerothin, Jason P.; Carlson, Heather A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">414-425</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">We have recently announced the largest database of protein-ligand complexes, Binding MOAD (Mother of All Databases).  After the August 2004 update, Binding MOAD contains 6816 complexes.  There are 2220 protein families and 3316 unique ligands.  After searching 6000+ crystallog. papers, we have obtained binding data for 1793 (27%) of the complexes.  We have also created a non-redundant set of complexes with only one complex from each protein family; in that set, 630 (28%) of the unique complexes have binding data.  Here, we present information about the data provided at the Binding MOAD website.  We also present the results of mining Binding MOAD to map the degree of solvent exposure for binding sites.  We have detd. that most cavities and ligands (70-85%) are well buried in the complexes.  This fits with the common paradigm that a large degree of contact between the ligand and protein is significant in mol. recognition.  GoCAV and the GoCAVviewer are the tools we created for this study.  To share our data and make our online dataset more useful to other research groups, we have integrated the viewer into the Binding MOAD website (www.BindingMOAD.org).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr08gepsx0bVbVg90H21EOLACvtfcHk0lgOFposuAx88A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjtFahu7Y%253D&md5=de65c9bf79e2a94cfdd5106642649a0e</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2005.08.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2005.08.002%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DR.%2BD.%26aulast%3DHu%26aufirst%3DL.%26aulast%3DFalkner%26aufirst%3DJ.%2BA.%26aulast%3DBenson%26aufirst%3DM.%2BL.%26aulast%3DNerothin%26aufirst%3DJ.%2BP.%26aulast%3DCarlson%26aufirst%3DH.%2BA.%26atitle%3DExploring%2520protein%25E2%2580%2593ligand%2520recognition%2520with%2520binding%2520MOAD%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2006%26volume%3D24%26spage%3D414%26epage%3D425" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Oprea, T. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allu, T. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fara, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rad, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ostopovici, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bologa, C. G.</span><span> </span><span class="NLM_article-title">Lead-like, drug-like or “Pub-like”: how different are they?</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">113</span><span class="NLM_x">–</span> <span class="NLM_lpage">119</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2007&pages=113-119&author=T.+I.+Opreaauthor=T.+K.+Alluauthor=D.+C.+Faraauthor=R.+F.+Radauthor=L.+Ostopoviciauthor=C.+G.+Bologa&title=Lead-like%2C+drug-like+or+%E2%80%9CPub-like%E2%80%9D%3A+how+different+are+they%3F"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DOprea%26aufirst%3DT.%2BI.%26aulast%3DAllu%26aufirst%3DT.%2BK.%26aulast%3DFara%26aufirst%3DD.%2BC.%26aulast%3DRad%26aufirst%3DR.%2BF.%26aulast%3DOstopovici%26aufirst%3DL.%26aulast%3DBologa%26aufirst%3DC.%2BG.%26atitle%3DLead-like%252C%2520drug-like%2520or%2520%25E2%2580%259CPub-like%25E2%2580%259D%253A%2520how%2520different%2520are%2520they%253F%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2007%26volume%3D21%26spage%3D113%26epage%3D119" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Okuno, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tamon, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yabuuchi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niijima, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minowa, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tonomura, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kunimoto, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, C.</span><span> </span><span class="NLM_article-title">GLIDA: GPCR—ligand database for chemical genomics drug discovery—database and tools update</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">D907</span><span class="NLM_x">–</span> <span class="NLM_lpage">D912</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2008&pages=D907-D912&author=Y.+Okunoauthor=A.+Tamonauthor=H.+Yabuuchiauthor=S.+Niijimaauthor=Y.+Minowaauthor=K.+Tonomuraauthor=R.+Kunimotoauthor=C.+Feng&title=GLIDA%3A+GPCR%E2%80%94ligand+database+for+chemical+genomics+drug+discovery%E2%80%94database+and+tools+update"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DOkuno%26aufirst%3DY.%26aulast%3DTamon%26aufirst%3DA.%26aulast%3DYabuuchi%26aufirst%3DH.%26aulast%3DNiijima%26aufirst%3DS.%26aulast%3DMinowa%26aufirst%3DY.%26aulast%3DTonomura%26aufirst%3DK.%26aulast%3DKunimoto%26aufirst%3DR.%26aulast%3DFeng%26aufirst%3DC.%26atitle%3DGLIDA%253A%2520GPCR%25E2%2580%2594ligand%2520database%2520for%2520chemical%2520genomics%2520drug%2520discovery%25E2%2580%2594database%2520and%2520tools%2520update%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2008%26volume%3D36%26spage%3DD907%26epage%3DD912" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Krejsa, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horvath, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogalski, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Penzotti, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mao, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbosa, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Migeon, J. C.</span><span> </span><span class="NLM_article-title">Predicting ADME properties and side effects: the BioPrint approach</span> <span class="citation_source-journal">Curr. Opin. Drug Discovery Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">470</span><span class="NLM_x">–</span> <span class="NLM_lpage">480</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=12951810" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD3sXntFymsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2003&pages=470-480&author=C.+M.+Krejsaauthor=D.+Horvathauthor=S.+L.+Rogalskiauthor=J.+E.+Penzottiauthor=B.+Maoauthor=F.+Barbosaauthor=J.+C.+Migeon&title=Predicting+ADME+properties+and+side+effects%3A+the+BioPrint+approach"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting ADME properties and side effects: The BioPrint approach</span></div><div class="casAuthors">Krejsa, Cecile M.; Horvath, Dragos; Rogalski, Sherri L.; Penzotti, Julie E.; Mao, Boryeu; Barbosa, Frederique; Migeon, Jacques C.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Drug Discovery & Development</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">470-480</span>CODEN:
                <span class="NLM_cas:coden">CODDFF</span>;
        ISSN:<span class="NLM_cas:issn">1367-6733</span>.
    
            (<span class="NLM_cas:orgname">Current Drugs</span>)
        </div><div class="casAbstract">A review.  Computational methods are increasingly used to streamline and enhance the lead discovery and optimization process.  However, accurate prediction of absorption, distribution, metab. and excretion (ADME) and adverse drug reactions (ADR) is often difficult, due to the complexity of underlying physiol. mechanisms.  Modeling approaches have been hampered by the lack of large, robust and standardized training datasets.  In an extensive effort to build such a dataset, the BioPrint database was constructed by systematic profiling of nearly all drugs available on the market, as well as numerous ref. compds.  The database is composed of several large datasets: compd. structures and mol. descriptors, in vitro ADME and pharmacol. profiles, and complementary clin. data including therapeutic use information, pharmacokinetics profiles and ADR profiles.  These data have allowed the development of computational tools designed to integrate a program of computational chem. into library design and lead development.  Models based on chem. structure are strengthened by in vitro results that can be used as addnl. compd. descriptors to predict complex in vivo endpoints.  The BioPrint pharmacoinformatics platform represents a systematic effort to accelerate the process of drug discovery, improve quant. structure-activity relationships and develop in vitro/in vivo assocns.  In this review, we will discuss the importance of training set size and diversity in model development, the implementation of linear and neighborhood modeling approaches, and the use of in silico methods to predict potential clin. liabilities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZQ71juUoErrVg90H21EOLACvtfcHk0linHLg4ihpCMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXntFymsLw%253D&md5=f8db4e37cc895d67bf4a82efe021c5c2</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKrejsa%26aufirst%3DC.%2BM.%26aulast%3DHorvath%26aufirst%3DD.%26aulast%3DRogalski%26aufirst%3DS.%2BL.%26aulast%3DPenzotti%26aufirst%3DJ.%2BE.%26aulast%3DMao%26aufirst%3DB.%26aulast%3DBarbosa%26aufirst%3DF.%26aulast%3DMigeon%26aufirst%3DJ.%2BC.%26atitle%3DPredicting%2520ADME%2520properties%2520and%2520side%2520effects%253A%2520the%2520BioPrint%2520approach%26jtitle%3DCurr.%2520Opin.%2520Drug%2520Discovery%2520Dev.%26date%3D2003%26volume%3D6%26spage%3D470%26epage%3D480" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Bender, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenkins, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Serrano, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mikhailov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clemons, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, J. W.</span><span> </span><span class="NLM_article-title">Chemogenomic data analysis: prediction of small-molecule targets and the advent of biological fingerprint</span> <span class="citation_source-journal">Comb. Chem. High Throughput Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">719</span><span class="NLM_x">–</span> <span class="NLM_lpage">731</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.2174%2F138620707782507313" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=18045083" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnsVOhsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2007&pages=719-731&author=A.+Benderauthor=D.+W.+Youngauthor=J.+L.+Jenkinsauthor=M.+Serranoauthor=D.+Mikhailovauthor=P.+A.+Clemonsauthor=J.+W.+Davies&title=Chemogenomic+data+analysis%3A+prediction+of+small-molecule+targets+and+the+advent+of+biological+fingerprint"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Chemogenomic data analysis: prediction of small-molecule targets and the advent of biological fingerprints</span></div><div class="casAuthors">Bender, Andreas; Young, Daniel W.; Jenkins, Jeremy L.; Serrano, Martin; Mikhailov, Dmitri; Clemons, Paul A.; Davies, John W.</div><div class="citationInfo"><span class="NLM_cas:title">Combinatorial Chemistry & High Throughput Screening</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">719-731</span>CODEN:
                <span class="NLM_cas:coden">CCHSFU</span>;
        ISSN:<span class="NLM_cas:issn">1386-2073</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Chemogenomics comprises a systematic relationship between targets and ligands that are used as target modulators in living systems such as cells or organisms.  In recent years, data on small mol.-bioactivity relationships have become increasingly available, and consequently so have the no. of approaches used to translate bioactivity data into knowledge.  This review will focus on 2 aspects of chemogenomics.  Firstly, in cases such as cell-based screens, the question of which target(s) a compd. is modulating to cause the obsd. phenotype is crucial.  In silico target prediction tools can suggest likely biol. targets of small mols. via data mining in target-annotated chem. databases.  This review presents some of the current tools available for this task and shows some sample applications relevant to a pharmaceutical industry setting.  These applications are the prediction of false-positives in cell-based reporter gene assays, the prediction of targets by linking bioassay data with protein domain annotations, and the direct prediction of adverse reactions.  Secondly, in recent years a shift from structure-derived chem. descriptors to biol. descriptors has occurred.  Here, the effect of a compd. on a no. of biol. endpoints is used to make predictions about other properties, such as putative targets, assocd. adverse reactions, and pathways modulated by the compd.  This review further summarizes these "performance" descriptors and their applications, focusing on gene expression profiles and high-content screening data.  The advent of such biol. fingerprints suggests that the field of drug discovery is currently at a crossroads, where single target bioassay results are supplanted by multidimensional biol. fingerprints that reflect a new awareness of biol. networks and polypharmacol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJuaT0KRiZF7Vg90H21EOLACvtfcHk0linHLg4ihpCMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnsVOhsg%253D%253D&md5=4b3152898f48cc7021904a0c983f6068</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.2174%2F138620707782507313&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138620707782507313%26sid%3Dliteratum%253Aachs%26aulast%3DBender%26aufirst%3DA.%26aulast%3DYoung%26aufirst%3DD.%2BW.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DSerrano%26aufirst%3DM.%26aulast%3DMikhailov%26aufirst%3DD.%26aulast%3DClemons%26aufirst%3DP.%2BA.%26aulast%3DDavies%26aufirst%3DJ.%2BW.%26atitle%3DChemogenomic%2520data%2520analysis%253A%2520prediction%2520of%2520small-molecule%2520targets%2520and%2520the%2520advent%2520of%2520biological%2520fingerprint%26jtitle%3DComb.%2520Chem.%2520High%2520Throughput%2520Screening%26date%3D2007%26volume%3D10%26spage%3D719%26epage%3D731" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">van der Horst, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okuno, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bender, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">AP, I. J.</span><span> </span><span class="NLM_article-title">Substructure mining of GPCR ligands reveals activity-class specific functional groups in an unbiased manner</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">348</span><span class="NLM_x">–</span> <span class="NLM_lpage">360</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci8003896" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlegt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=348-360&author=E.+van+der+Horstauthor=Y.+Okunoauthor=A.+Benderauthor=I.+J.+AP&title=Substructure+mining+of+GPCR+ligands+reveals+activity-class+specific+functional+groups+in+an+unbiased+manner"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Substructure Mining of GPCR Ligands Reveals Activity-Class Specific Functional Groups in an Unbiased Manner</span></div><div class="casAuthors">van der Horst, Eelke; Okuno, Yasushi; Bender, Andreas; IJzerman, Adriaan P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">348-360</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this study, we conducted frequent substructure mining to identify structural features that discriminate between ligands that do bind to G protein-coupled receptors (GPCRs) and those that do not.  In most cases, particular chem. representations resulted in the most significant substructures.  Substructures found to be characteristic for the background control set reflected reactions that may have been used to construct this library, e.g. for the ChemBridge DIVERSet library employed these are ester and carboxamide moieties.  Alkane amine substructures were identified as most important for GPCR ligands, e.g. the butylamine substructure, often linked to an arom. system.  Hierarchical anal. of targeted GPCRs revealed well-known motives and new substructural features.  One example is the imidazole-like substructure common for the histamine binding receptor ligands.  Another example is the planar ring system consisting of a fused five- and six-membered ring (indole-like substructure) common for the serotonin receptor ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVvg1lPYXlkbVg90H21EOLACvtfcHk0ljIFA5ct0GwiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlegt78%253D&md5=c7a95054a55fd0d58196ff858d3d500e</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fci8003896&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci8003896%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2Bder%2BHorst%26aufirst%3DE.%26aulast%3DOkuno%26aufirst%3DY.%26aulast%3DBender%26aufirst%3DA.%26aulast%3DAP%26aufirst%3DI.%2BJ.%26atitle%3DSubstructure%2520mining%2520of%2520GPCR%2520ligands%2520reveals%2520activity-class%2520specific%2520functional%2520groups%2520in%2520an%2520unbiased%2520manner%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D348%26epage%3D360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Klabunde, T.</span><span> </span><span class="NLM_article-title">Chemogenomic approaches to drug discovery: similar receptors bind similar ligands</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">152</span><span class="NLM_x">, </span> <span class="NLM_fpage">5</span><span class="NLM_x">–</span> <span class="NLM_lpage">7</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1038%2Fsj.bjp.0707308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=17533415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpsFOjur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=152&publication_year=2007&pages=5-7&author=T.+Klabunde&title=Chemogenomic+approaches+to+drug+discovery%3A+similar+receptors+bind+similar+ligands"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Chemogenomic approaches to drug discovery: similar receptors bind similar ligands</span></div><div class="casAuthors">Klabunde, T.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">152</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-7</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Within recent years, a paradigm shift from traditional receptor-specific studies to a cross-receptor view has taken place within pharmaceutical research to increase the efficiency of modern drug discovery.  Receptors are no longer viewed as single entities but grouped into sets of related proteins or receptor families that are explored in a systematic manner.  This interdisciplinary approach attempting to derive predictive links between the chem. structures of bioactive mols. and the receptors with which these mols. interact is referred to as chemogenomics.  Insights from chemogenomics are used for the rational compilation of screening sets and for the rational design and synthesis of directed chem. libraries to accelerate drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQeX3iyp_Z47Vg90H21EOLACvtfcHk0ljIFA5ct0GwiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpsFOjur0%253D&md5=a103cdd189d2bad6d70e799aad869290</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjp.0707308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjp.0707308%26sid%3Dliteratum%253Aachs%26aulast%3DKlabunde%26aufirst%3DT.%26atitle%3DChemogenomic%2520approaches%2520to%2520drug%2520discovery%253A%2520similar%2520receptors%2520bind%2520similar%2520ligands%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2007%26volume%3D152%26spage%3D5%26epage%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">van Westen, G. J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wegner, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">IJzerman, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Vlijmen, H. W. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bender, A.</span><span> </span><span class="NLM_article-title">Proteochemometric modeling as a tool to design selective compounds and for extrapolating to novel targets</span> <span class="citation_source-journal">Med. Chem. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">16</span><span class="NLM_x">–</span> <span class="NLM_lpage">30</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1039%2Fc0md00165a" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=16-30&author=G.+J.+P.+van+Westenauthor=J.+K.+Wegnerauthor=A.+P.+IJzermanauthor=H.+W.+T.+van+Vlijmenauthor=A.+Bender&title=Proteochemometric+modeling+as+a+tool+to+design+selective+compounds+and+for+extrapolating+to+novel+targets"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1039%2Fc0md00165a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc0md00165a%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BWesten%26aufirst%3DG.%2BJ.%2BP.%26aulast%3DWegner%26aufirst%3DJ.%2BK.%26aulast%3DIJzerman%26aufirst%3DA.%2BP.%26aulast%3Dvan%2BVlijmen%26aufirst%3DH.%2BW.%2BT.%26aulast%3DBender%26aufirst%3DA.%26atitle%3DProteochemometric%2520modeling%2520as%2520a%2520tool%2520to%2520design%2520selective%2520compounds%2520and%2520for%2520extrapolating%2520to%2520novel%2520targets%26jtitle%3DMed.%2520Chem.%2520Commun.%26date%3D2011%26volume%3D2%26spage%3D16%26epage%3D30" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Geppert, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vogt, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Current trends in ligand-based virtual screening: molecular representations, data mining methods, new application areas, and performance evaluation</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">205</span><span class="NLM_x">–</span> <span class="NLM_lpage">216</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci900419k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnvFWmsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=205-216&author=H.+Geppertauthor=M.+Vogtauthor=J.+Bajorath&title=Current+trends+in+ligand-based+virtual+screening%3A+molecular+representations%2C+data+mining+methods%2C+new+application+areas%2C+and+performance+evaluation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Current Trends in Ligand-Based Virtual Screening: Molecular Representations, Data Mining Methods, New Application Areas, and Performance Evaluation</span></div><div class="casAuthors">Geppert, Hanna; Vogt, Martin; Bajorath, Jurgen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">205-216</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpc1HgbkXa9o7Vg90H21EOLACvtfcHk0liMIALF6GkuRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnvFWmsA%253D%253D&md5=aeb674ef7c6c93711eb21452732e21d8</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fci900419k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900419k%26sid%3Dliteratum%253Aachs%26aulast%3DGeppert%26aufirst%3DH.%26aulast%3DVogt%26aufirst%3DM.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DCurrent%2520trends%2520in%2520ligand-based%2520virtual%2520screening%253A%2520molecular%2520representations%252C%2520data%2520mining%2520methods%252C%2520new%2520application%2520areas%252C%2520and%2520performance%2520evaluation%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D205%26epage%3D216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Bender, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenkins, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scheiber, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sukuru, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glick, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, J. W.</span><span> </span><span class="NLM_article-title">How similar are similarity searching methods? A principal component analysis of molecular descriptor space</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">108</span><span class="NLM_x">–</span> <span class="NLM_lpage">119</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci800249s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD1MXisVahtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=108-119&author=A.+Benderauthor=J.+L.+Jenkinsauthor=J.+Scheiberauthor=S.+C.+Sukuruauthor=M.+Glickauthor=J.+W.+Davies&title=How+similar+are+similarity+searching+methods%3F+A+principal+component+analysis+of+molecular+descriptor+space"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">How similar are similarity searching methods? A principal component analysis of molecular descriptor space</span></div><div class="casAuthors">Bender, Andreas; Jenkins, Jeremy L.; Scheiber, Josef; Sukuru, Sai Chetan K.; Glick, Meir; Davies, John W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">108-119</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Different mol. descriptors capture different aspects of mol. structures, but this effect has not yet been quantified systematically on a large scale.  In this work, we calc. the similarity of 37 descriptors by repeatedly selecting query compds. and ranking the rest of the database.  Euclidean distances between the rank-ordering of different descriptors are calcd. to det. descriptor (as opposed to compd.) similarity, followed by PCA for visualization.  Four broad descriptor classes are identified, which are circular fingerprints; circular fingerprints considering counts; path-based and keyed fingerprints; and pharmacophoric descriptors.  Descriptor behavior is much more defined by those four classes than the particular parametrization.  Using counts instead of the presence/absence of fingerprints significantly changes descriptor behavior, which is crucial for performance of topol. autocorrelation vectors, but not circular fingerprints.  Four-point pharmacophores (piDAPH4) surprisingly lead to much higher retrieval rates than three-point pharmacophores (28.21% vs 19.15%) but still similar rank-ordering of compds. (retrieval of similar actives).  Looking into individual rankings, circular fingerprints seem more appropriate than path-based fingerprints if complex ring systems or branching patterns are present; count-based fingerprints could be more suitable in databases with a large no. of repeated subunits (amide bonds, sugar rings, terpenes).  Information-based selection of diverse fingerprints for consensus scoring (ECFP4/TGD fingerprints) led only to marginal improvement over single fingerprint results.  While it seems to be nontrivial to exploit orthogonal descriptor behavior to improve retrieval rates in consensus virtual screening, those descriptors still each retrieve different actives which corroborates the strategy of employing diverse descriptors individually in prospective virtual screening settings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgAHu8jM0iuLVg90H21EOLACvtfcHk0liMIALF6GkuRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXisVahtQ%253D%253D&md5=edcbc5314c60e4607a7cbdfd60562f53</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fci800249s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800249s%26sid%3Dliteratum%253Aachs%26aulast%3DBender%26aufirst%3DA.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DScheiber%26aufirst%3DJ.%26aulast%3DSukuru%26aufirst%3DS.%2BC.%26aulast%3DGlick%26aufirst%3DM.%26aulast%3DDavies%26aufirst%3DJ.%2BW.%26atitle%3DHow%2520similar%2520are%2520similarity%2520searching%2520methods%253F%2520A%2520principal%2520component%2520analysis%2520of%2520molecular%2520descriptor%2520space%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D108%26epage%3D119" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Leach, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gillet, V. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, R.</span><span> </span><span class="NLM_article-title">Three-dimensional pharmacophore methods in drug discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">539</span><span class="NLM_x">–</span> <span class="NLM_lpage">558</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm900817u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1OktbjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=539-558&author=A.+R.+Leachauthor=V.+J.+Gilletauthor=R.+A.+Lewisauthor=R.+Taylor&title=Three-dimensional+pharmacophore+methods+in+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Three-Dimensional Pharmacophore Methods in Drug Discovery</span></div><div class="casAuthors">Leach, Andrew R.; Gillet, Valerie J.; Lewis, Richard A.; Taylor, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">539-558</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  In the first part of this Perspective the authors provide an overview of the computational 3D pharmacophore methods most commonly used in drug discovery.  The authors focus in particular on the key problem of pharmacophore elucidation: the identification from a set of active mols. and their biol. activities of the key common features and their relative orientations (also called pharmacophore mapping).  In the second part the authors reflect on the challenges in this field and provide some thoughts on future directions.  The authors' primary goal is to provide the interested reader with a fair assessment of the current state-of-the-art in this field with an emphasis on the methods and software that in the authors' experience are most widely used in real drug discovery projects.  The authors will cover both the practical aspects and some of the inherent limitations of such methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9x_N0BVt1DLVg90H21EOLACvtfcHk0liMIALF6GkuRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1OktbjP&md5=b9844e41a7140b58ee508aab07d9e4ed</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm900817u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm900817u%26sid%3Dliteratum%253Aachs%26aulast%3DLeach%26aufirst%3DA.%2BR.%26aulast%3DGillet%26aufirst%3DV.%2BJ.%26aulast%3DLewis%26aufirst%3DR.%2BA.%26aulast%3DTaylor%26aufirst%3DR.%26atitle%3DThree-dimensional%2520pharmacophore%2520methods%2520in%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D539%26epage%3D558" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Moitessier, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Englebienne, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lawandi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corbeil, C. R.</span><span> </span><span class="NLM_article-title">Towards the development of universal, fast and highly accurate docking/scoring methods: a long way to go</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">153</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 1</span><span class="NLM_x">) </span> <span class="NLM_fpage">S7</span><span class="NLM_x">–</span> <span class="NLM_lpage">S26</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=153&publication_year=2008&pages=S7-S26&issue=Suppl.+1&author=N.+Moitessierauthor=P.+Englebienneauthor=D.+Leeauthor=J.+Lawandiauthor=C.+R.+Corbeil&title=Towards+the+development+of+universal%2C+fast+and+highly+accurate+docking%2Fscoring+methods%3A+a+long+way+to+go"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMoitessier%26aufirst%3DN.%26aulast%3DEnglebienne%26aufirst%3DP.%26aulast%3DLee%26aufirst%3DD.%26aulast%3DLawandi%26aufirst%3DJ.%26aulast%3DCorbeil%26aufirst%3DC.%2BR.%26atitle%3DTowards%2520the%2520development%2520of%2520universal%252C%2520fast%2520and%2520highly%2520accurate%2520docking%252Fscoring%2520methods%253A%2520a%2520long%2520way%2520to%2520go%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2008%26volume%3D153%26issue%3DSuppl.%25201%26spage%3DS7%26epage%3DS26" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Sanders, M. P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGuire, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roumen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Esch, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Vlieg, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klomp, J. P. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Graaf, C.</span><span> </span><span class="NLM_article-title">From the protein’s perspective: the benefits and challenges of protein structure-based pharmacophore modeling</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">28</span><span class="NLM_x">–</span> <span class="NLM_lpage">38</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=28-38&author=M.+P.+A.+Sandersauthor=R.+McGuireauthor=L.+Roumenauthor=I.+J.+De+Eschauthor=J.+de+Vliegauthor=J.+P.+G.+Klompauthor=C.+de+Graaf&title=From+the+protein%E2%80%99s+perspective%3A+the+benefits+and+challenges+of+protein+structure-based+pharmacophore+modeling"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSanders%26aufirst%3DM.%2BP.%2BA.%26aulast%3DMcGuire%26aufirst%3DR.%26aulast%3DRoumen%26aufirst%3DL.%26aulast%3DDe%2BEsch%26aufirst%3DI.%2BJ.%26aulast%3Dde%2BVlieg%26aufirst%3DJ.%26aulast%3DKlomp%26aufirst%3DJ.%2BP.%2BG.%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26atitle%3DFrom%2520the%2520protein%25E2%2580%2599s%2520perspective%253A%2520the%2520benefits%2520and%2520challenges%2520of%2520protein%2520structure-based%2520pharmacophore%2520modeling%26jtitle%3DMedChemComm%26date%3D2011%26volume%3D3%26spage%3D28%26epage%3D38" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">de Graaf, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">Customizing G Protein-coupled receptor models for structure-based virtual screening</span> <span class="citation_source-journal">Curr. Pharm. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">4026</span><span class="NLM_x">–</span> <span class="NLM_lpage">4048</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.2174%2F138161209789824786" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=20028320" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlKlu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2009&pages=4026-4048&author=C.+de+Graafauthor=D.+Rognan&title=Customizing+G+Protein-coupled+receptor+models+for+structure-based+virtual+screening"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Customizing G protein-coupled receptor models for structure-based virtual screening</span></div><div class="casAuthors">de Graaf, Chris; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">4026-4048</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">This review will focus on the construction, refinement, and validation of G Protein-coupled receptor models for the purpose of structure-based virtual screening.  Practical tips and tricks derived from concrete modeling and virtual screening exercises to overcome the problems and pitfalls assocd. with the different steps of the receptor modeling workflow will be presented.  These examples will not only include rhodopsin-like (class A), but also secretine-like (class B), and glutamate-like (class C) receptors.  In addn., the review will present a careful comparative anal. of current crystal structures and their implication on homol. modeling.  The following themes will be discussed: (i) the use of exptl. anchors in guiding the modeling procedure; (ii) amino acid sequence alignments; (iii) ligand binding mode accommodation and binding cavity expansion; (iv) proline-induced kinks in transmembrane helixes; (v) binding mode prediction and virtual screening by receptor-ligand interaction fingerprint scoring; (vi) extracellular loop modeling; (vii) virtual filtering schemes.  Finally, an overview of several successful structure-based screening shows that receptor models, despite structural inaccuracies, can be efficiently used to find novel ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqxkKuCF95LrVg90H21EOLACvtfcHk0ljfUOG9KStBHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlKlu7Y%253D&md5=e3aefb5a191d0507cca6766ce191e82c</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.2174%2F138161209789824786&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138161209789824786%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DCustomizing%2520G%2520Protein-coupled%2520receptor%2520models%2520for%2520structure-based%2520virtual%2520screening%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2009%26volume%3D15%26spage%3D4026%26epage%3D4048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Liu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ouyang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gong, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H.</span><span> </span><span class="NLM_article-title">PharmMapper server: a Web server for potential drug target identification using pharmacophore mapping approach</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">W609</span><span class="NLM_x">–</span> <span class="NLM_lpage">W614</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1093%2Fnar%2Fgkq300" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=20430828" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BC3cXotVWjsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=W609-W614&author=X.+Liuauthor=S.+Ouyangauthor=B.+Yuauthor=Y.+Liuauthor=K.+Huangauthor=J.+Gongauthor=S.+Zhengauthor=Z.+Liauthor=H.+Liauthor=H.+Jiang&title=PharmMapper+server%3A+a+Web+server+for+potential+drug+target+identification+using+pharmacophore+mapping+approach"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">PharmMapper server: a web server for potential drug target identification using pharmacophore mapping approach</span></div><div class="casAuthors">Liu, Xiaofeng; Ouyang, Sisheng; Yu, Biao; Liu, Yabo; Huang, Kai; Gong, Jiayu; Zheng, Siyuan; Li, Zhihua; Li, Honglin; Jiang, Hualiang</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">Web Server</span>),
    <span class="NLM_cas:pages">W609-W614</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">In silico drug target identification, which includes many distinct algorithms for finding disease genes and proteins, is the first step in the drug discovery pipeline.  When the 3D structures of the targets are available, the problem of target identification is usually converted to finding the best interaction mode between the potential target candidates and small mol. probes.  Pharmacophore, which is the spatial arrangement of features essential for a mol. to interact with a specific target receptor, is an alternative method for achieving this goal apart from mol. docking method.  PharmMapper server is a freely accessed web server designed to identify potential target candidates for the given small mols. (drugs, natural products or other newly discovered compds. with unidentified binding targets) using pharmacophore mapping approach.  PharmMapper hosts a large, inhouse repertoire of pharmacophore database (namely PharmTargetDB) annotated from all the targets information in TargetBank, BindingDB, DrugBank and potential drug target database, including over 7000 receptor-based pharmacophore models (covering over 1500 drug targets information).  PharmMapper automatically finds the best mapping poses of the query mol. against all the pharmacophore models in PharmTargetDB and lists the top N best-fitted hits with appropriate target annotations, as well as resp. mol.'s aligned poses are presented.  Benefited from the highly efficient and robust triangle hashing mapping method, PharmMapper bears high throughput ability and only costs 1 h averagely to screen the whole PharmTargetDB.  The protocol was successful in finding the proper targets among the top 300 pharmacophore candidates in the retrospective benchmarking test of tamoxifen.  PharmMapper is available at http://59.78.96.61/pharmmapper.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqL-Gaa6UM_zLVg90H21EOLACvtfcHk0ljWeGs-qGFmgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXotVWjsL4%253D&md5=dadfc5a278243d22f2ea041914461e3a</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkq300&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkq300%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DOuyang%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DK.%26aulast%3DGong%26aufirst%3DJ.%26aulast%3DZheng%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DH.%26atitle%3DPharmMapper%2520server%253A%2520a%2520Web%2520server%2520for%2520potential%2520drug%2520target%2520identification%2520using%2520pharmacophore%2520mapping%2520approach%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2010%26volume%3D38%26spage%3DW609%26epage%3DW614" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Rask-Andersen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Almen, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schioth, H. B.</span><span> </span><span class="NLM_article-title">Trends in the exploitation of novel drug targets</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">579</span><span class="NLM_x">–</span> <span class="NLM_lpage">590</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1038%2Fnrd3478" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=21804595" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpsFKqsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=579-590&author=M.+Rask-Andersenauthor=M.+S.+Almenauthor=H.+B.+Schioth&title=Trends+in+the+exploitation+of+novel+drug+targets"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Trends in the exploitation of novel drug targets</span></div><div class="casAuthors">Rask-Andersen, Mathias; Almen, Markus Saellman; Schioeth, Helgi B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">579-590</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The discovery and exploitation of new drug targets is a key focus for both the pharmaceutical industry and academic biomedical research.  To provide an insight into trends in the exploitation of new drug targets, we have analyzed the drugs that were approved by the US Food and Drug Administration during the past three decades and examd. the interactions of these drugs with therapeutic targets that are encoded by the human genome, using the DrugBank database and extensive manual curation.  We have identified 435 effect-mediating drug targets in the human genome, which are modulated by 989 unique drugs, through 2,242 drug-target interactions.  We also analyze trends in the introduction of drugs that modulate previously unexploited targets, and discuss the network pharmacol. of the drugs in our data set.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo19FXXFB-PsLVg90H21EOLACvtfcHk0ljWeGs-qGFmgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpsFKqsLc%253D&md5=9255431a28f2c07759b0f36975711f84</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fnrd3478&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3478%26sid%3Dliteratum%253Aachs%26aulast%3DRask-Andersen%26aufirst%3DM.%26aulast%3DAlmen%26aufirst%3DM.%2BS.%26aulast%3DSchioth%26aufirst%3DH.%2BB.%26atitle%3DTrends%2520in%2520the%2520exploitation%2520of%2520novel%2520drug%2520targets%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D579%26epage%3D590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Szabo, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kincses, Z. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vecsei, L.</span><span> </span><span class="NLM_article-title">Novel therapy in Parkinson’s disease: adenosine A(2A) receptor antagonists</span> <span class="citation_source-journal">Expert Opin. Drug Metab. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">441</span><span class="NLM_x">–</span> <span class="NLM_lpage">455</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1517%2F17425255.2011.557066" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=21332415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A280%3ADC%252BC3M3ot1Oquw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=441-455&author=N.+Szaboauthor=Z.+T.+Kincsesauthor=L.+Vecsei&title=Novel+therapy+in+Parkinson%E2%80%99s+disease%3A+adenosine+A%282A%29+receptor+antagonists"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Novel therapy in Parkinson's disease: adenosine A(2A) receptor antagonists</span></div><div class="casAuthors">Szabo Nikoletta; Kincses Zsigmond Tamas; Vecsei Laszlo</div><div class="citationInfo"><span class="NLM_cas:title">Expert opinion on drug metabolism & toxicology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">441-55</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">INTRODUCTION: Parkinson's disease (PD) is a progressive neurodegenerative disorder.  To date, most of the currently available therapies in PD target the dopaminergic system and none of these therapeutic approaches have been proven to modify the course of the disease.  To various extents, these drugs can also cause motor and non-motor complications.  A novel target, the adenosine A(2A) receptor (AA2AR), was recently identified, blockade of which may alleviate Parkinsonian symptoms, reduce motor fluctuations and potentially afford neuroprotection.  AREAS COVERED: This review is based on a PubMed search covering the relationship of the adenosine receptors and PD.  The role of the AA2AR is reviewed and the results of preclinical investigations of antagonists are assessed.  A synopsis of current drug development is provided, with a special focus on the pharmacokinetics and relevant clinical trials.  EXPERT OPINION: The localization of the AA2AR in the central nervous system, the ultra structural localization and the molecular mechanism of its action reveal the potential importance of the AA2AR in movement disorders.  The theoretical background and experimental data indicate that AA2AR antagonists may have a potential therapeutic effect in Parkinson's disease.  More importantly, the putative neuroprotective effect needs further investigation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSkwTiCXUG5u5w7prLlxLOzfW6udTcc2ebWvQoCvL7k1rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3M3ot1Oquw%253D%253D&md5=c14a993e0dda01906cd19b08dd5db683</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1517%2F17425255.2011.557066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17425255.2011.557066%26sid%3Dliteratum%253Aachs%26aulast%3DSzabo%26aufirst%3DN.%26aulast%3DKincses%26aufirst%3DZ.%2BT.%26aulast%3DVecsei%26aufirst%3DL.%26atitle%3DNovel%2520therapy%2520in%2520Parkinson%25E2%2580%2599s%2520disease%253A%2520adenosine%2520A%25282A%2529%2520receptor%2520antagonists%26jtitle%3DExpert%2520Opin.%2520Drug%2520Metab.%2520Toxicol.%26date%3D2011%26volume%3D7%26spage%3D441%26epage%3D455" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Dorsam, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gutkind, J. S.</span><span> </span><span class="NLM_article-title">G-Protein-coupled receptors and cancer</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">79</span><span class="NLM_x">–</span> <span class="NLM_lpage">94</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1038%2Fnrc2069" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=17251915" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD2sXotFOisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2007&pages=79-94&author=R.+T.+Dorsamauthor=J.+S.+Gutkind&title=G-Protein-coupled+receptors+and+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">G-protein-coupled receptors and cancer</span></div><div class="casAuthors">Dorsam, Robert T.; Gutkind, J. Silvio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">79-94</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  G-protein-coupled receptors (GPCRs), the largest family of cell-surface mols. involved in signal transmission, have recently emerged as crucial players in tumor growth and metastasis.  Malignant cells often hijack the normal physiol. functions of GPCRs to survive, proliferate autonomously, evade the immune system, increase their blood supply, invade their surrounding tissues and disseminate to other organs.  This review will address our current understanding of the many roles of GPCRs and their signaling circuitry in tumor progression and metastasis.  We will also discuss how interfering with GPCRs might provide unique opportunities for cancer prevention and treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLfbCDb7EA8LVg90H21EOLACvtfcHk0ljBSXyrdrR9Ag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXotFOisw%253D%253D&md5=17b71299f0b532fbd01683c678f10397</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2Fnrc2069&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2069%26sid%3Dliteratum%253Aachs%26aulast%3DDorsam%26aufirst%3DR.%2BT.%26aulast%3DGutkind%26aufirst%3DJ.%2BS.%26atitle%3DG-Protein-coupled%2520receptors%2520and%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2007%26volume%3D7%26spage%3D79%26epage%3D94" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Cherezov, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenbaum, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanson, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rasmussen, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thian, F. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weis, W. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevens, R. C.</span><span> </span><span class="NLM_article-title">High-resolution crystal structure of an engineered human beta2-adrenergic G protein-coupled receptor</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">318</span><span class="NLM_x">, </span> <span class="NLM_fpage">1258</span><span class="NLM_x">–</span> <span class="NLM_lpage">1265</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1126%2Fscience.1150577" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=17962520" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlGmur7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=318&publication_year=2007&pages=1258-1265&author=V.+Cherezovauthor=D.+M.+Rosenbaumauthor=M.+A.+Hansonauthor=S.+G.+Rasmussenauthor=F.+S.+Thianauthor=T.+S.+Kobilkaauthor=H.+J.+Choiauthor=P.+Kuhnauthor=W.+I.+Weisauthor=B.+K.+Kobilkaauthor=R.+C.+Stevens&title=High-resolution+crystal+structure+of+an+engineered+human+beta2-adrenergic+G+protein-coupled+receptor"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">High-Resolution Crystal Structure of an Engineered Human β2-Adrenergic G Protein-Coupled Receptor</span></div><div class="casAuthors">Cherezov, Vadim; Rosenbaum, Daniel M.; Hanson, Michael A.; Rasmussen, Soren G. F.; Thian, Foon Sun; Kobilka, Tong Sun; Choi, Hee-Jung; Kuhn, Peter; Weis, William I.; Kobilka, Brian K.; Stevens, Raymond C.; Takeda, S.; Kadowaki, S.; Haga, T.; Takaesu, H.; Mitaku, S.; Fredriksson, R.; Lagerstrom, M. C.; Lundin, L. G.; Schioth, H. B.; Pierce, K. L.; Premont, R. T.; Lefkowitz, R. J.; Lefkowitz, R. J.; Shenoy, S. K.; Rosenbaum, D. M.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">318</span>
        (<span class="NLM_cas:issue">5854</span>),
    <span class="NLM_cas:pages">1258-1265</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Second extracellular loop, which in the β2-adrenergic receptor contains an unusual pair of disulfide bonds and an extra helix.  This loop and the absence Heterotrimeric guanine nucleotide-binding protein (G protein)-coupled receptors constitute the largest family of eukaryotic signal transduction proteins that communicate across the membrane.  We report the crystal structure of a human β2-adrenergic receptor-T4 lysozyme fusion protein bound to the partial inverse agonist carazolol at 2.4 angstrom resoln.  The structure provides a high-resoln. view of a human G protein-coupled receptor bound to a diffusible ligand.  Ligand-binding site accessibility is enabled by the second extracellular loop, which is held out of the binding cavity by a pair of closely spaced disulfide bridges and a short helical segment within the loop.  Cholesterol, a necessary component for crystn., mediates an intriguing parallel assocn. of receptor mols. in the crystal lattice.  Although the location of carazolol in the β2-adrenergic receptor is very similar to that of retinal in rhodopsin, structural differences in the ligand-binding site and other regions highlight the challenges in using rhodopsin as a template model for this large receptor family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpC-8wZ2k2GHLVg90H21EOLACvtfcHk0ljBSXyrdrR9Ag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlGmur7I&md5=12c5bacb8464a4b243fe9341192b5b3b</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1126%2Fscience.1150577&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1150577%26sid%3Dliteratum%253Aachs%26aulast%3DCherezov%26aufirst%3DV.%26aulast%3DRosenbaum%26aufirst%3DD.%2BM.%26aulast%3DHanson%26aufirst%3DM.%2BA.%26aulast%3DRasmussen%26aufirst%3DS.%2BG.%26aulast%3DThian%26aufirst%3DF.%2BS.%26aulast%3DKobilka%26aufirst%3DT.%2BS.%26aulast%3DChoi%26aufirst%3DH.%2BJ.%26aulast%3DKuhn%26aufirst%3DP.%26aulast%3DWeis%26aufirst%3DW.%2BI.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26atitle%3DHigh-resolution%2520crystal%2520structure%2520of%2520an%2520engineered%2520human%2520beta2-adrenergic%2520G%2520protein-coupled%2520receptor%26jtitle%3DScience%26date%3D2007%26volume%3D318%26spage%3D1258%26epage%3D1265" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Jaakola, V. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffith, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanson, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherezov, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chien, E. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lane, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ijzerman, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevens, R. C.</span><span> </span><span class="NLM_article-title">The 2.6 angstrom crystal structure of a human A2A adenosine receptor bound to an antagonist</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">322</span><span class="NLM_x">, </span> <span class="NLM_fpage">1211</span><span class="NLM_x">–</span> <span class="NLM_lpage">1217</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1126%2Fscience.1164772" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=18832607" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlyqtbfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=322&publication_year=2008&pages=1211-1217&author=V.+P.+Jaakolaauthor=M.+T.+Griffithauthor=M.+A.+Hansonauthor=V.+Cherezovauthor=E.+Y.+Chienauthor=J.+R.+Laneauthor=A.+P.+Ijzermanauthor=R.+C.+Stevens&title=The+2.6+angstrom+crystal+structure+of+a+human+A2A+adenosine+receptor+bound+to+an+antagonist"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">The 2.6 Angstrom Crystal Structure of a Human A2A Adenosine Receptor Bound to an Antagonist</span></div><div class="casAuthors">Jaakola, Veli-Pekka; Griffith, Mark T.; Hanson, Michael A.; Cherezov, Vadim; Chien, Ellen Y. T.; Lane, J. Robert; IJzerman, Adriaan P.; Stevens, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">322</span>
        (<span class="NLM_cas:issue">5905</span>),
    <span class="NLM_cas:pages">1211-1217</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The adenosine class of heterotrimeric guanine nucleotide-binding protein (G protein)-coupled receptors (GPCRs) mediates the important role of extracellular adenosine in many physiol. processes and is antagonized by caffeine.  The authors have detd. the crystal structure of the human A2A adenosine receptor, in complex with a high-affinity subtype-selective antagonist, ZM241385, to 2.6 angstrom resoln.  Four disulfide bridges in the extracellular domain, combined with a subtle repacking of the transmembrane helixes relative to the adrenergic and rhodopsin receptor structures, define a pocket distinct from that of other structurally detd. GPCRs.  The arrangement allows for the binding of the antagonist in an extended conformation, perpendicular to the membrane plane.  The binding site highlights an integral role for the extracellular loops, together with the helical core, in ligand recognition by this class of GPCRs and suggests a role for ZM241385 in restricting the movement of a tryptophan residue important in the activation mechanism of the class A receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7QL4wN4TdhrVg90H21EOLACvtfcHk0lgW2ZqQt_S6_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlyqtbfN&md5=5bdb862b41f345c244f3c162e058206b</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1126%2Fscience.1164772&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1164772%26sid%3Dliteratum%253Aachs%26aulast%3DJaakola%26aufirst%3DV.%2BP.%26aulast%3DGriffith%26aufirst%3DM.%2BT.%26aulast%3DHanson%26aufirst%3DM.%2BA.%26aulast%3DCherezov%26aufirst%3DV.%26aulast%3DChien%26aufirst%3DE.%2BY.%26aulast%3DLane%26aufirst%3DJ.%2BR.%26aulast%3DIjzerman%26aufirst%3DA.%2BP.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26atitle%3DThe%25202.6%2520angstrom%2520crystal%2520structure%2520of%2520a%2520human%2520A2A%2520adenosine%2520receptor%2520bound%2520to%2520an%2520antagonist%26jtitle%3DScience%26date%3D2008%26volume%3D322%26spage%3D1211%26epage%3D1217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Hanson, M. A</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffith, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, F. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinhart, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desale, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clemons, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cahalan, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schuerer, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanna, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevens, R. C.</span><span> </span><span class="NLM_article-title">Crystal structure of a lipid G protein-coupled receptor.</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">335</span><span class="NLM_x">, </span> <span class="NLM_fpage">851</span><span class="NLM_x">–</span> <span class="NLM_lpage">855</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=335&publication_year=2012&pages=851-855&author=M.+A+Hansonauthor=C.+B.+Rothauthor=E.+Joauthor=M.+T.+Griffithauthor=F.+L.+Scottauthor=G.+Reinhartauthor=H.+Desaleauthor=B.+Clemonsauthor=S.+M.+Cahalanauthor=S.+C.+Schuererauthor=M.+G.+Sannaauthor=G.+W.+Hanauthor=P.+Kuhnauthor=H.+Rosenauthor=R.+C.+Stevens&title=Crystal+structure+of+a+lipid+G+protein-coupled+receptor."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHanson%26aufirst%3DM.%2BA%26aulast%3DRoth%26aufirst%3DC.%2BB.%26aulast%3DJo%26aufirst%3DE.%26aulast%3DGriffith%26aufirst%3DM.%2BT.%26aulast%3DScott%26aufirst%3DF.%2BL.%26aulast%3DReinhart%26aufirst%3DG.%26aulast%3DDesale%26aufirst%3DH.%26aulast%3DClemons%26aufirst%3DB.%26aulast%3DCahalan%26aufirst%3DS.%2BM.%26aulast%3DSchuerer%26aufirst%3DS.%2BC.%26aulast%3DSanna%26aufirst%3DM.%2BG.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DKuhn%26aufirst%3DP.%26aulast%3DRosen%26aufirst%3DH.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26atitle%3DCrystal%2520structure%2520of%2520a%2520lipid%2520G%2520protein-coupled%2520receptor.%26jtitle%3DScience%26date%3D2012%26volume%3D335%26spage%3D851%26epage%3D855" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Xu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katritch, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, Z. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherezov, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevens, R. C.</span><span> </span><span class="NLM_article-title">Structure of an agonist-bound human A2A adenosine receptor</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">332</span><span class="NLM_x">, </span> <span class="NLM_fpage">322</span><span class="NLM_x">–</span> <span class="NLM_lpage">327</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1126%2Fscience.1202793" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=21393508" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BC3MXksFWjt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=332&publication_year=2011&pages=322-327&author=F.+Xuauthor=H.+Wuauthor=V.+Katritchauthor=G.+W.+Hanauthor=K.+A.+Jacobsonauthor=Z.+G.+Gaoauthor=V.+Cherezovauthor=R.+C.+Stevens&title=Structure+of+an+agonist-bound+human+A2A+adenosine+receptor"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of an Agonist-Bound Human A2A Adenosine Receptor</span></div><div class="casAuthors">Xu, Fei; Wu, Huixian; Katritch, Vsevolod; Han, Gye Won; Jacobson, Kenneth A.; Gao, Zhan-Guo; Cherezov, Vadim; Stevens, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">332</span>
        (<span class="NLM_cas:issue">6027</span>),
    <span class="NLM_cas:pages">322-327</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Activation of G protein-coupled receptors upon agonist binding is a crit. step in the signaling cascade for this family of cell surface proteins.  We report the crystal structure of the A2A adenosine receptor (A2AAR) bound to an agonist UK-432097 at 2.7 angstrom resoln.  Relative to inactive, antagonist-bound A2AAR, the agonist-bound structure displays an outward tilt and rotation of the cytoplasmic half of helix VI, a movement of helix V, and an axial shift of helix III, resembling the changes assocd. with the active-state opsin structure.  Addnl., a seesaw movement of helix VII and a shift of extracellular loop 3 are likely specific to A2AAR and its ligand.  The results define the mol. UK-432097 as a "conformationally selective agonist" capable of receptor stabilization in a specific active-state configuration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMkD1ni9m9GrVg90H21EOLACvtfcHk0lgW2ZqQt_S6_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXksFWjt7s%253D&md5=b1e074f38f45c95d4cc3b40378b24923</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1126%2Fscience.1202793&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1202793%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DF.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26aulast%3DGao%26aufirst%3DZ.%2BG.%26aulast%3DCherezov%26aufirst%3DV.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26atitle%3DStructure%2520of%2520an%2520agonist-bound%2520human%2520A2A%2520adenosine%2520receptor%26jtitle%3DScience%26date%3D2011%26volume%3D332%26spage%3D322%26epage%3D327" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref243"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref243'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit243"><span><span class="NLM_contrib-group">Jiang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Rhee, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yehle, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wess, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, K. A.</span><span> </span><span class="NLM_article-title">Hydrophilic side chains in the third and seventh transmembrane helical domains of human A2A adenosine receptors are required for ligand recognition</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">512</span><span class="NLM_x">–</span> <span class="NLM_lpage">521</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref243/cit243&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=8794889" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref243/cit243&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADyaK28XlslShurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1996&pages=512-521&author=Q.+Jiangauthor=A.+M.+Van+Rheeauthor=J.+Kimauthor=S.+Yehleauthor=J.+Wessauthor=K.+A.+Jacobson&title=Hydrophilic+side+chains+in+the+third+and+seventh+transmembrane+helical+domains+of+human+A2A+adenosine+receptors+are+required+for+ligand+recognition"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit243R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Hydrophilic side chains in the third and seventh transmembrane helical domains of human A2a adenosine receptors are required for ligand recognition</span></div><div class="casAuthors">Jiang, Qiaoling; Van Rhee, Michiel; Kim, Jeongho; Yehle, Susan; Wess, Juergen; Jacobson, Kenneth A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">512-521</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">0026-895X</span>.
    
            (<span class="NLM_cas:orgname">Williams & Wilkins</span>)
        </div><div class="casAbstract">Hydrophilic residues of the G protein-coupled human A2A adenosine receptor that are potentially involved in the binding of the ribose moiety of adenosine were targeted for mutagenesis.  Residues in a T88QSS sequence in the third transmembrane helical domain (TM3) were individually replaced with alanine and other amino acids.  Two addnl. serine residues in TM7 that were previously shown to be involved in ligand binding were mutated to other uncharged hydrophilic amino acids.  The binding affinity of agonist at T88 mutant receptors was greatly diminished, although the receptors were well expressed and bound antagonists similar to the wild-type receptor.  Thus, mutations that are specific for diminishing the affinity of ribose-contg. ligands (i.e., adenosine agonists) have been identified in both TM3 and TM7.  The T88A and T88S mutant receptor fully stimulated adenylyl cyclase, with the dose-responsive curves to CGS 21680 highly shifted to the right.  A Q89A mutant gained affinity for all agonist and antagonist ligands examd. in binding and functional assays.  Q89 likely plays an indirect role in ligand binding.  S90A, S91A, and S277C mutant receptors displayed only moderate changes in ligand affinity.  A S281N mutant gained affinity for all adenosine derivs. (agonists), but antagonist affinity was generally diminished, with the exception of a novel tetrahydrobenzothiophenone deriv.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojbNx_bMR3jLVg90H21EOLACvtfcHk0ljOz2WKqlYCHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XlslShurg%253D&md5=08a775d55751171209ecfd1f3c1e7d93</span></div><a href="/servlet/linkout?suffix=cit243&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DQ.%26aulast%3DVan%2BRhee%26aufirst%3DA.%2BM.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DYehle%26aufirst%3DS.%26aulast%3DWess%26aufirst%3DJ.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26atitle%3DHydrophilic%2520side%2520chains%2520in%2520the%2520third%2520and%2520seventh%2520transmembrane%2520helical%2520domains%2520of%2520human%2520A2A%2520adenosine%2520receptors%2520are%2520required%2520for%2520ligand%2520recognition%26jtitle%3DMol.%2520Pharmacol.%26date%3D1996%26volume%3D50%26spage%3D512%26epage%3D521" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref244"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref244'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit244"><span><span class="NLM_contrib-group">Kim, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wess, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Rhee, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schoneberg, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, K. A.</span><span> </span><span class="NLM_article-title">Site-directed mutagenesis identifies residues involved in ligand recognition in the human A2a adenosine receptor</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">270</span><span class="NLM_x">, </span> <span class="NLM_fpage">13987</span><span class="NLM_x">–</span> <span class="NLM_lpage">13997</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=270&publication_year=1995&pages=13987-13997&author=J.+Kimauthor=J.+Wessauthor=A.+M.+van+Rheeauthor=T.+Schonebergauthor=K.+A.+Jacobson&title=Site-directed+mutagenesis+identifies+residues+involved+in+ligand+recognition+in+the+human+A2a+adenosine+receptor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit244&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DWess%26aufirst%3DJ.%26aulast%3Dvan%2BRhee%26aufirst%3DA.%2BM.%26aulast%3DSchoneberg%26aufirst%3DT.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26atitle%3DSite-directed%2520mutagenesis%2520identifies%2520residues%2520involved%2520in%2520ligand%2520recognition%2520in%2520the%2520human%2520A2a%2520adenosine%2520receptor%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1995%26volume%3D270%26spage%3D13987%26epage%3D13997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref245"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref245'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit245"><span><span class="NLM_contrib-group">Kim, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glashofer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yehle, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wess, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, K. A.</span><span> </span><span class="NLM_article-title">Glutamate residues in the second extracellular loop of the human A2a adenosine receptor are required for ligand recognition</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">683</span><span class="NLM_x">–</span> <span class="NLM_lpage">691</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref245/cit245&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=8609897" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref245/cit245&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADyaK28XisVSksL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=1996&pages=683-691&author=J.+Kimauthor=Q.+Jiangauthor=M.+Glashoferauthor=S.+Yehleauthor=J.+Wessauthor=K.+A.+Jacobson&title=Glutamate+residues+in+the+second+extracellular+loop+of+the+human+A2a+adenosine+receptor+are+required+for+ligand+recognition"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit245R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Glutamate residues in the second extracellular loop of the human A2a adenosine receptor are required for ligand recognition</span></div><div class="casAuthors">Kim, Jeongho; Jiang, Qiaoling; Glashofer, Marc; Yehle, Susan; Wess, Juergen; Jacobson, Kenneth A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">683-91</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">0026-895X</span>.
    
            (<span class="NLM_cas:orgname">Williams & Wilkins</span>)
        </div><div class="casAbstract">The A2a adenosine receptor, a member of the G protein-coupled receptor family, is important in the regulation of dopaminergic pathways of the brain and in platelet and cardiovascular functions.  In this study, the role of extracellular loops in ligand binding to the human A2a receptor was explored through site-directed mutagenesis.  Four glutamate/aspartate residues (Glu151, Glu161, Glu169, and Asp170) in the second extracellular loop (E2) and a cysteine residue (Cys262) in the third extracellular loop (E3) were individually replaced with alanine and other amino acids.  A proline residue (Pro173) in E2 was mutated to arginine, the homologous amino acid in A3 receptors.  The binding properties of the resultant mutant receptors were detd. in transfected COS-7 cells.  The mutant receptors were tagged at their amino terminus with a hemagglutinin epitope, thus allowing their detection in the plasma membrane with immunol. techniques.  High affinity specific binding of [3H]2-[4-[(2-carboxyethyl)phenyl]ethyl-amino]-5'-N-ethylcarboxamidoadenosine (15 nM) and [3H]8-[4-[[[[(2-aminoethyl)-amino]carbonyl]methyl]oxy]phenyl]-1,3- dipropylxanthine (4 nM), an A2a agonist and antagonist, resp., was not obsd. with four of the mutant receptors, E151A, E151Q, E151D, and E169A, although they were well expressed at the cell surface.  The E151A and E169A mutant receptors showed nearly full stimulation of adenylyl cyclase at ∼103-fold higher concns. of 2-[4-[(2-carboxyethyl)phenyl]ethyl-amino]-5'-N-ethylcarboxamidoadenosine.  The E161A mutant receptor showed an increase in affinity for the nonxanthine adenosine antagonist 9-chloro-2-(furyl)[1,2,4]triazolo[1,5-c]quinazolin-5-amine (6-fold) but not for other ligands.  An E169Q mutant gained affinity (5-22-fold) for adenosine derivs. (agonists) substituted at N6 but not at C2 or C5' positions.  Mutant receptors D170K and P173R were similar to wild-type receptors in binding of both agonist and antagonist radioligands.  A C262G mutant also resembled the wild-type receptor in radioligand binding, indicating that a potential disulfide bridge with another cysteine residue in proximity is not required for the structural integrity of the receptor.  The authors' data suggest that certain amino acids in the second extracellular loop may be directly or indirectly involved in ligand binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxzW8wJBN6PLVg90H21EOLACvtfcHk0ljOz2WKqlYCHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XisVSksL0%253D&md5=f5e3742c67788d6c637cc97734b9614f</span></div><a href="/servlet/linkout?suffix=cit245&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DQ.%26aulast%3DGlashofer%26aufirst%3DM.%26aulast%3DYehle%26aufirst%3DS.%26aulast%3DWess%26aufirst%3DJ.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26atitle%3DGlutamate%2520residues%2520in%2520the%2520second%2520extracellular%2520loop%2520of%2520the%2520human%2520A2a%2520adenosine%2520receptor%2520are%2520required%2520for%2520ligand%2520recognition%26jtitle%3DMol.%2520Pharmacol.%26date%3D1996%26volume%3D49%26spage%3D683%26epage%3D691" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Moro, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, Z. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spalluto, G.</span><span> </span><span class="NLM_article-title">Progress in the pursuit of therapeutic adenosine receptor antagonists</span> <span class="citation_source-journal">Med. Res. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">131</span><span class="NLM_x">–</span> <span class="NLM_lpage">159</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1002%2Fmed.20048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=16380972" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD28XislOksLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2006&pages=131-159&author=S.+Moroauthor=Z.+G.+Gaoauthor=K.+A.+Jacobsonauthor=G.+Spalluto&title=Progress+in+the+pursuit+of+therapeutic+adenosine+receptor+antagonists"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Progress in the pursuit of therapeutic Adenosine receptor antagonists</span></div><div class="casAuthors">Moro, Stefano; Gao, Zhan-Guo; Jacobson, Kenneth A.; Spalluto, Giampiero</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">131-159</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Ever since the discovery of the hypotensive and bradycardiac effects of adenosine, adenosine receptors continue to represent promising drug targets.  First, this is due to the fact that the receptors are expressed in a large variety of tissues.  In particular, the actions of adenosine (or methylxanthine antagonists) in the central nervous system, in the circulation, on immune cells, and on other tissues can be beneficial in certain disorders.  Second, there exists a large no. of ligands, which have been generated by introducing several modifications in the structure of the lead compds. (adenosine and methylxanthine), some of them highly specific.  Four adenosine receptor subtypes (A1, A2A, A2B, and A3) have been cloned and pharmacol. characterized, all of which are G protein-coupled receptors.  Adenosine receptors can be distinguished according to their preferred mechanism of signal transduction: A1 and A3 receptors interact with pertussis toxin-sensitive G proteins of the Gi and Go family; the canonical signaling mechanism of the A2A and of the A2B receptors is stimulation of adenylyl cyclase via Gs proteins.  In addn. to the coupling to adenylyl cyclase, all four subtypes may pos. couple to phospholipase C via different G protein subunits.  The development of new ligands, in particular, potent and selective antagonists, for all subtypes of adenosine receptors has so far been directed by traditional medicinal chem.  The availability of genetic information promises to facilitate understanding of the drug-receptor interaction leading to the rational design of a potentially therapeutically important class of drugs.  Moreover, mol. modeling may further rationalize obsd. interactions between the receptors and their ligands.  In this review, we will summarize the most relevant progress in developing new therapeutic adenosine receptor antagonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrX73C53aX74rVg90H21EOLACvtfcHk0lh9rSMhBUqonQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XislOksLw%253D&md5=48f7a603f3c4044b2c48aba13a031e07</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1002%2Fmed.20048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.20048%26sid%3Dliteratum%253Aachs%26aulast%3DMoro%26aufirst%3DS.%26aulast%3DGao%26aufirst%3DZ.%2BG.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26aulast%3DSpalluto%26aufirst%3DG.%26atitle%3DProgress%2520in%2520the%2520pursuit%2520of%2520therapeutic%2520adenosine%2520receptor%2520antagonists%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2006%26volume%3D26%26spage%3D131%26epage%3D159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Strader, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sigal, I. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Candelore, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rands, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hill, W. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dixon, R. A.</span><span> </span><span class="NLM_article-title">Conserved aspartic acid residues 79 and 113 of the beta-adrenergic receptor have different roles in receptor function</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">263</span><span class="NLM_x">, </span> <span class="NLM_fpage">10267</span><span class="NLM_x">–</span> <span class="NLM_lpage">10271</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=263&publication_year=1988&pages=10267-10271&author=C.+D.+Straderauthor=I.+S.+Sigalauthor=M.+R.+Candeloreauthor=E.+Randsauthor=W.+S.+Hillauthor=R.+A.+Dixon&title=Conserved+aspartic+acid+residues+79+and+113+of+the+beta-adrenergic+receptor+have+different+roles+in+receptor+function"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DStrader%26aufirst%3DC.%2BD.%26aulast%3DSigal%26aufirst%3DI.%2BS.%26aulast%3DCandelore%26aufirst%3DM.%2BR.%26aulast%3DRands%26aufirst%3DE.%26aulast%3DHill%26aufirst%3DW.%2BS.%26aulast%3DDixon%26aufirst%3DR.%2BA.%26atitle%3DConserved%2520aspartic%2520acid%2520residues%252079%2520and%2520113%2520of%2520the%2520beta-adrenergic%2520receptor%2520have%2520different%2520roles%2520in%2520receptor%2520function%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1988%26volume%3D263%26spage%3D10267%26epage%3D10271" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Strader, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sigal, I. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Register, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Candelore, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rands, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dixon, R. A.</span><span> </span><span class="NLM_article-title">Identification of residues required for ligand binding to the beta-adrenergic receptor</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1987</span><span class="NLM_x">, </span> <span class="NLM_volume">84</span><span class="NLM_x">, </span> <span class="NLM_fpage">4384</span><span class="NLM_x">–</span> <span class="NLM_lpage">4388</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=1987&pages=4384-4388&author=C.+D.+Straderauthor=I.+S.+Sigalauthor=R.+B.+Registerauthor=M.+R.+Candeloreauthor=E.+Randsauthor=R.+A.+Dixon&title=Identification+of+residues+required+for+ligand+binding+to+the+beta-adrenergic+receptor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DStrader%26aufirst%3DC.%2BD.%26aulast%3DSigal%26aufirst%3DI.%2BS.%26aulast%3DRegister%26aufirst%3DR.%2BB.%26aulast%3DCandelore%26aufirst%3DM.%2BR.%26aulast%3DRands%26aufirst%3DE.%26aulast%3DDixon%26aufirst%3DR.%2BA.%26atitle%3DIdentification%2520of%2520residues%2520required%2520for%2520ligand%2520binding%2520to%2520the%2520beta-adrenergic%2520receptor%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1987%26volume%3D84%26spage%3D4384%26epage%3D4388" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref249"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref249'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit249"><span><span class="NLM_contrib-group">Liapakis, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ballesteros, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Papachristou, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Javitch, J. A.</span><span> </span><span class="NLM_article-title">The forgotten serine. A critical role for Ser-2035.42 in ligand binding to and activation of the beta 2-adrenergic receptor</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">275</span><span class="NLM_x">, </span> <span class="NLM_fpage">37779</span><span class="NLM_x">–</span> <span class="NLM_lpage">37788</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=275&publication_year=2000&pages=37779-37788&author=G.+Liapakisauthor=J.+A.+Ballesterosauthor=S.+Papachristouauthor=W.+C.+Chanauthor=X.+Chenauthor=J.+A.+Javitch&title=The+forgotten+serine.+A+critical+role+for+Ser-2035.42+in+ligand+binding+to+and+activation+of+the+beta+2-adrenergic+receptor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit249&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLiapakis%26aufirst%3DG.%26aulast%3DBallesteros%26aufirst%3DJ.%2BA.%26aulast%3DPapachristou%26aufirst%3DS.%26aulast%3DChan%26aufirst%3DW.%2BC.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DJavitch%26aufirst%3DJ.%2BA.%26atitle%3DThe%2520forgotten%2520serine.%2520A%2520critical%2520role%2520for%2520Ser-2035.42%2520in%2520ligand%2520binding%2520to%2520and%2520activation%2520of%2520the%2520beta%25202-adrenergic%2520receptor%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2000%26volume%3D275%26spage%3D37779%26epage%3D37788" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref250"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref250'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit250"><span><span class="NLM_contrib-group">Suryanarayana, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, B. K.</span><span> </span><span class="NLM_article-title">Amino acid substitutions at position 312 in the seventh hydrophobic segment of the beta 2-adrenergic receptor modify ligand-binding specificity</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">111</span><span class="NLM_x">–</span> <span class="NLM_lpage">114</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref250/cit250&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=8101966" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref250/cit250&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADyaK3sXlsFahtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=1993&pages=111-114&author=S.+Suryanarayanaauthor=B.+K.+Kobilka&title=Amino+acid+substitutions+at+position+312+in+the+seventh+hydrophobic+segment+of+the+beta+2-adrenergic+receptor+modify+ligand-binding+specificity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit250R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Amino acid substitutions at position 312 in the seventh hydrophobic segment of the β2-adrenergic receptor modify ligand-binding specificity</span></div><div class="casAuthors">Suryanarayana, Sankuratri; Kobilka, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">111-14</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">0026-895X</span>.
    </div><div class="casAbstract">The authors previously reported that Asn312 of the β2-adrenergic receptor and Asn385 in the homologs position in the 5-HT1A receptor are important for binding to a class of β-adrenergic receptor antagonists including propranolol and alprenolol.  The authors proposed that the asparagine may be forming a H bond with the phenoxy O common to these ligands.  To further test this hypothesis the authors made alanine, threonine, phenylalanine, and glutamine substitutions at position 312 in the β2-adrenergic receptor.  The authors obsd. that substitution with amino acids that permit formation of H bonds (threonine and glutamine) supported binding to aryloxyalkylamines, whereas substitution with amino acids that cannot form H bonds (alanine and phenylalanine) did not permit binding to these compds.  The authors were surprised to find that 2 of these substitutions led to an increase in affinity for α-adrenergic ligands.  Substitution with glutamine and threonine at position 312 led to a 11-15-fold increase in affinity for yohimbine and enabled p-aminoclonidine to act as an agonist.  These results further emphasize the role of position 312 in the formation of the ligand binding site for multiple ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGre0AZHzXQBV7Vg90H21EOLACvtfcHk0lh9rSMhBUqonQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXlsFahtb0%253D&md5=0e12c9a08aac895567384cb35632b78d</span></div><a href="/servlet/linkout?suffix=cit250&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSuryanarayana%26aufirst%3DS.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26atitle%3DAmino%2520acid%2520substitutions%2520at%2520position%2520312%2520in%2520the%2520seventh%2520hydrophobic%2520segment%2520of%2520the%2520beta%25202-adrenergic%2520receptor%2520modify%2520ligand-binding%2520specificity%26jtitle%3DMol.%2520Pharmacol.%26date%3D1993%26volume%3D44%26spage%3D111%26epage%3D114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Strader, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Candelore, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hill, W. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sigal, I. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dixon, R. A.</span><span> </span><span class="NLM_article-title">Identification of two serine residues involved in agonist activation of the beta-adrenergic receptor</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1989</span><span class="NLM_x">, </span> <span class="NLM_volume">264</span><span class="NLM_x">, </span> <span class="NLM_fpage">13572</span><span class="NLM_x">–</span> <span class="NLM_lpage">13578</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=2547766" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADyaL1MXlsVOht78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=264&publication_year=1989&pages=13572-13578&author=C.+D.+Straderauthor=M.+R.+Candeloreauthor=W.+S.+Hillauthor=I.+S.+Sigalauthor=R.+A.+Dixon&title=Identification+of+two+serine+residues+involved+in+agonist+activation+of+the+beta-adrenergic+receptor"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of two serine residues involved in agonist activation of the β-adrenergic receptor</span></div><div class="casAuthors">Strader, Catherine D.; Candelore, Mari Rios; Hill, Wendy S.; Sigal, Irving S.; Dixon, Richard A. F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">264</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">13572-8</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    </div><div class="casAbstract">Pharmacophore mapping of the ligand-binding domain of the β-adrenergic receptor has revealed specific mol. interactions which are important for agonist and antagonist binding to the receptor.  Previous site-directed mutagenesis expts. have demonstrated that the binding of amine agonists and antagonists to the receptor involves an interaction between the amine group of the ligand and the carboxylate side chain of Asp113 in the 3rd hydrophobic domain of the receptor.  Two serine residues, at positions 204 and 207 in the 5th hydrophobic domain of the β-adrenergic receptor, which are crit. for agonist binding and activation of the receptor have now been identified.  These serine residues are conserved with G-protein-coupled receptors which bind catecholamine agonists, but not with receptors whose endogenous ligands do not have the catechol moiety.  Removal of the OH side chain from either Ser204 or Ser207 by substitution of the serine residue with an alanine attenuates the activity of catecholamine agonists at the receptor.  The effects of these mutations on agonist activity are mimicked selectively by the removal of the catechol OH moieties from the arom. ring of the agonist.  The data suggest that the interaction of catecholamine agonists with the β-adrenergic receptor involves 2 H bonds, one between the OH side chain of Ser204 and the meta-OH group of the ligand and a second between the OH side chain of Ser207 and the para-OH group of the ligand.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEA_QZmFYKc7Vg90H21EOLACvtfcHk0lh9rSMhBUqonQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXlsVOht78%253D&md5=0e8e69f212b17a3c54a024d7497d79a7</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DStrader%26aufirst%3DC.%2BD.%26aulast%3DCandelore%26aufirst%3DM.%2BR.%26aulast%3DHill%26aufirst%3DW.%2BS.%26aulast%3DSigal%26aufirst%3DI.%2BS.%26aulast%3DDixon%26aufirst%3DR.%2BA.%26atitle%3DIdentification%2520of%2520two%2520serine%2520residues%2520involved%2520in%2520agonist%2520activation%2520of%2520the%2520beta-adrenergic%2520receptor%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1989%26volume%3D264%26spage%3D13572%26epage%3D13578" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Wieland, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zuurmond, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krasel, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ijzerman, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohse, M. J.</span><span> </span><span class="NLM_article-title">Involvement of Asn-293 in stereospecific agonist recognition and in activation of the beta 2-adrenergic receptor</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">93</span><span class="NLM_x">, </span> <span class="NLM_fpage">9276</span><span class="NLM_x">–</span> <span class="NLM_lpage">9281</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=1996&pages=9276-9281&author=K.+Wielandauthor=H.+M.+Zuurmondauthor=C.+Kraselauthor=A.+P.+Ijzermanauthor=M.+J.+Lohse&title=Involvement+of+Asn-293+in+stereospecific+agonist+recognition+and+in+activation+of+the+beta+2-adrenergic+receptor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWieland%26aufirst%3DK.%26aulast%3DZuurmond%26aufirst%3DH.%2BM.%26aulast%3DKrasel%26aufirst%3DC.%26aulast%3DIjzerman%26aufirst%3DA.%2BP.%26aulast%3DLohse%26aufirst%3DM.%2BJ.%26atitle%3DInvolvement%2520of%2520Asn-293%2520in%2520stereospecific%2520agonist%2520recognition%2520and%2520in%2520activation%2520of%2520the%2520beta%25202-adrenergic%2520receptor%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1996%26volume%3D93%26spage%3D9276%26epage%3D9281" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Kikkawa, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Isogaya, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagao, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurose, H.</span><span> </span><span class="NLM_article-title">The role of the seventh transmembrane region in high affinity binding of a beta 2-selective agonist TA-2005</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">128</span><span class="NLM_x">–</span> <span class="NLM_lpage">134</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1998&pages=128-134&author=H.+Kikkawaauthor=M.+Isogayaauthor=T.+Nagaoauthor=H.+Kurose&title=The+role+of+the+seventh+transmembrane+region+in+high+affinity+binding+of+a+beta+2-selective+agonist+TA-2005"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKikkawa%26aufirst%3DH.%26aulast%3DIsogaya%26aufirst%3DM.%26aulast%3DNagao%26aufirst%3DT.%26aulast%3DKurose%26aufirst%3DH.%26atitle%3DThe%2520role%2520of%2520the%2520seventh%2520transmembrane%2520region%2520in%2520high%2520affinity%2520binding%2520of%2520a%2520beta%25202-selective%2520agonist%2520TA-2005%26jtitle%3DMol.%2520Pharmacol.%26date%3D1998%26volume%3D53%26spage%3D128%26epage%3D134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Pham, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fells, J. I.,  Sr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Osborne, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">North, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Naor, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parrill, A. L.</span><span> </span><span class="NLM_article-title">Molecular recognition in the sphingosine 1-phosphate receptor family</span> <span class="citation_source-journal">J. Mol. Graphics Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">1189</span><span class="NLM_x">–</span> <span class="NLM_lpage">1201</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1016%2Fj.jmgm.2007.11.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=18165127" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlvVWnsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=1189-1201&author=T.+C.+Phamauthor=J.+I.+Fellsauthor=D.+A.+Osborneauthor=E.+J.+Northauthor=M.+M.+Naorauthor=A.+L.+Parrill&title=Molecular+recognition+in+the+sphingosine+1-phosphate+receptor+family"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular recognition in the sphingosine 1-phosphate receptor family</span></div><div class="casAuthors">Pham, Truc-Chi T.; Fells, James I.; Osborne, Daniel A.; North, E. Jeffrey; Naor, Mor M.; Parrill, Abby L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1189-1201</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Computational modeling and its application in ligand screening and ligand receptor interaction studies play important roles in structure-based drug design.  A series of sphingosine 1-phosphate (S1P) receptor ligands with varying potencies and receptor selectivities were docked into homol. models of the S1P1-5 receptors.  These studies provided mol. insights into pharmacol. trends both across the receptor family as well as at single receptors.  This study identifies ligand recognition features that generalize across the S1P receptor family, features unique to the S1P4 and S1P5 receptors, and suggests significant structural differences of the S1P2 receptor.  Docking results reveal a previously unknown sulfur-arom. interaction between the S1P4 C5.44 sulfur atom and the Ph ring of benzimidazole as well as π-π interaction between F3.33 of S1P1,4,5 and arom. ligands.  The findings not only confirm the importance of a cation-π interaction between W4.64 and the ammonium of S1P at S1P4 but also predict the same interaction at S1P5.  S1P receptor models are validated for pharmacophore development including database mining and new ligand discovery and serve as tools for ligand optimization to improve potency and selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpboag1kZ0ncbVg90H21EOLACvtfcHk0liZg18tO9zedg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlvVWnsbc%253D&md5=cde18a520b3536b68839196dd2ce989e</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2007.11.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2007.11.001%26sid%3Dliteratum%253Aachs%26aulast%3DPham%26aufirst%3DT.%2BC.%26aulast%3DFells%26aufirst%3DJ.%2BI.%26aulast%3DOsborne%26aufirst%3DD.%2BA.%26aulast%3DNorth%26aufirst%3DE.%2BJ.%26aulast%3DNaor%26aufirst%3DM.%2BM.%26aulast%3DParrill%26aufirst%3DA.%2BL.%26atitle%3DMolecular%2520recognition%2520in%2520the%2520sphingosine%25201-phosphate%2520receptor%2520family%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2008%26volume%3D26%26spage%3D1189%26epage%3D1201" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Parrill, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bautista, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Brocklyn, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lorincz, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischer, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baker, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liliom, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spiegel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tigyi, G.</span><span> </span><span class="NLM_article-title">Identification of Edg1 receptor residues that recognize sphingosine 1-phosphate</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">275</span><span class="NLM_x">, </span> <span class="NLM_fpage">39379</span><span class="NLM_x">–</span> <span class="NLM_lpage">39384</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1074%2Fjbc.M007680200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=10982820" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD3cXptFSrsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=275&publication_year=2000&pages=39379-39384&author=A.+L.+Parrillauthor=D.+Wangauthor=D.+L.+Bautistaauthor=J.+R.+Van+Brocklynauthor=Z.+Lorinczauthor=D.+J.+Fischerauthor=D.+L.+Bakerauthor=K.+Liliomauthor=S.+Spiegelauthor=G.+Tigyi&title=Identification+of+Edg1+receptor+residues+that+recognize+sphingosine+1-phosphate"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Edg1 receptor residues that recognize sphingosine 1-phosphate</span></div><div class="casAuthors">Parrill, Abby L.; Wang, De-An; Bautista, Debra L.; Van Brocklyn, James R.; Lorincz, Zsolt; Fischer, David J.; Baker, Daniel L.; Liliom, Karoly; Spiegel, Sarah; Tigyi, Gabor</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">275</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">39379-39384</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Originating from its DNA sequence, a computational model of the Edg1 receptor has been developed that predicts crit. interactions with its ligand, sphingosine 1-phosphate.  The basic amino acids Arg120 and Arg292 ion pair with the phosphate, whereas the acidic Glu121 residue ion pairs with the ammonium moiety of sphingosine 1-phosphate.  The requirement of these interactions for specific ligand recognition has been confirmed through examn. of site-directed mutants by radioligand binding, ligand-induced [35S]GTPγS binding, and receptor internalization assays.  These ion-pairing interactions explain the ligand specificity of the Edg1 receptor and provide insight into ligand specificity differences within the Edg receptor family.  This computational map of the ligand binding pocket provides information necessary for understanding the mol. pharmacol. of this receptor, thus underlining the potential of the computational method in predicting ligand-receptor interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8bLS2jWZ-dbVg90H21EOLACvtfcHk0liZg18tO9zedg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXptFSrsbg%253D&md5=2ff5a71db88fab6ff901d4c07e02f26c</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M007680200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M007680200%26sid%3Dliteratum%253Aachs%26aulast%3DParrill%26aufirst%3DA.%2BL.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DBautista%26aufirst%3DD.%2BL.%26aulast%3DVan%2BBrocklyn%26aufirst%3DJ.%2BR.%26aulast%3DLorincz%26aufirst%3DZ.%26aulast%3DFischer%26aufirst%3DD.%2BJ.%26aulast%3DBaker%26aufirst%3DD.%2BL.%26aulast%3DLiliom%26aufirst%3DK.%26aulast%3DSpiegel%26aufirst%3DS.%26aulast%3DTigyi%26aufirst%3DG.%26atitle%3DIdentification%2520of%2520Edg1%2520receptor%2520residues%2520that%2520recognize%2520sphingosine%25201-phosphate%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2000%26volume%3D275%26spage%3D39379%26epage%3D39384" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Wang, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lorincz, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bautista, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liliom, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tigyi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parrill, A. L.</span><span> </span><span class="NLM_article-title">A single amino acid determines lysophospholipid specificity of the S1P1 (EDG1) and LPA1 (EDG2) phospholipid growth factor receptors</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">276</span><span class="NLM_x">, </span> <span class="NLM_fpage">49213</span><span class="NLM_x">–</span> <span class="NLM_lpage">49220</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=2001&pages=49213-49220&author=D.+A.+Wangauthor=Z.+Lorinczauthor=D.+L.+Bautistaauthor=K.+Liliomauthor=G.+Tigyiauthor=A.+L.+Parrill&title=A+single+amino+acid+determines+lysophospholipid+specificity+of+the+S1P1+%28EDG1%29+and+LPA1+%28EDG2%29+phospholipid+growth+factor+receptors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DD.%2BA.%26aulast%3DLorincz%26aufirst%3DZ.%26aulast%3DBautista%26aufirst%3DD.%2BL.%26aulast%3DLiliom%26aufirst%3DK.%26aulast%3DTigyi%26aufirst%3DG.%26aulast%3DParrill%26aufirst%3DA.%2BL.%26atitle%3DA%2520single%2520amino%2520acid%2520determines%2520lysophospholipid%2520specificity%2520of%2520the%2520S1P1%2520%2528EDG1%2529%2520and%2520LPA1%2520%2528EDG2%2529%2520phospholipid%2520growth%2520factor%2520receptors%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2001%26volume%3D276%26spage%3D49213%26epage%3D49220" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Surgand, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodrigo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kellenberger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">A chemogenomic analysis of the transmembrane binding cavity of human G-protein-coupled receptors</span> <span class="citation_source-journal">Proteins</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">62</span><span class="NLM_x">, </span> <span class="NLM_fpage">509</span><span class="NLM_x">–</span> <span class="NLM_lpage">538</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1002%2Fprot.20768" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=16294340" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A280%3ADC%252BD2MnovFyrtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=509-538&author=J.+S.+Surgandauthor=J.+Rodrigoauthor=E.+Kellenbergerauthor=D.+Rognan&title=A+chemogenomic+analysis+of+the+transmembrane+binding+cavity+of+human+G-protein-coupled+receptors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">A chemogenomic analysis of the transmembrane binding cavity of human G-protein-coupled receptors</span></div><div class="casAuthors">Surgand Jean-Sebastien; Rodrigo Jordi; Kellenberger Esther; Rognan Didier</div><div class="citationInfo"><span class="NLM_cas:title">Proteins</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">509-38</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The amino acid sequences of 369 human nonolfactory G-protein-coupled receptors (GPCRs) have been aligned at the seven transmembrane domain (TM) and used to extract the nature of 30 critical residues supposed--from the X-ray structure of bovine rhodopsin bound to retinal--to line the TM binding cavity of ground-state receptors.  Interestingly, the clustering of human GPCRs from these 30 residues mirrors the recently described phylogenetic tree of full-sequence human GPCRs (Fredriksson et al., Mol Pharmacol 2003;63:1256-1272) with few exceptions.  A TM cavity could be found for all investigated GPCRs with physicochemical properties matching that of their cognate ligands.  The current approach allows a very fast comparison of most human GPCRs from the focused perspective of the predicted TM cavity and permits to easily detect key residues that drive ligand selectivity or promiscuity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTH-6l-CmE2uXWEEzDrymxmfW6udTcc2eZz9pkBILMJarntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2MnovFyrtw%253D%253D&md5=5f60612975dbffa52e440cd2d7ab080c</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1002%2Fprot.20768&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.20768%26sid%3Dliteratum%253Aachs%26aulast%3DSurgand%26aufirst%3DJ.%2BS.%26aulast%3DRodrigo%26aufirst%3DJ.%26aulast%3DKellenberger%26aufirst%3DE.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DA%2520chemogenomic%2520analysis%2520of%2520the%2520transmembrane%2520binding%2520cavity%2520of%2520human%2520G-protein-coupled%2520receptors%26jtitle%3DProteins%26date%3D2006%26volume%3D62%26spage%3D509%26epage%3D538" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">van Loenen, P. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Graaf, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verzijl, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leurs, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peters, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alewijnse, A. E.</span><span> </span><span class="NLM_article-title">Agonist-dependent effects of mutations in the sphingosine-1-phosphate type 1 receptor</span> <span class="citation_source-journal">Eur. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">667</span><span class="NLM_x">, </span> <span class="NLM_fpage">105</span><span class="NLM_x">–</span> <span class="NLM_lpage">112</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1016%2Fj.ejphar.2011.05.071" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=21663738" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVGltrnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=667&publication_year=2011&pages=105-112&author=P.+B.+van+Loenenauthor=C.+de+Graafauthor=D.+Verzijlauthor=R.+Leursauthor=D.+Rognanauthor=S.+L.+Petersauthor=A.+E.+Alewijnse&title=Agonist-dependent+effects+of+mutations+in+the+sphingosine-1-phosphate+type+1+receptor"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Agonist-dependent effects of mutations in the sphingosine-1-phosphate type 1 receptor</span></div><div class="casAuthors">van Loenen, Pieter B.; de Graaf, Chris; Verzijl, Dennis; Leurs, Rob; Rognan, Didier; Peters, Stephan L. M.; Alewijnse, Astrid E.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">667</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">105-112</span>CODEN:
                <span class="NLM_cas:coden">EJPHAZ</span>;
        ISSN:<span class="NLM_cas:issn">0014-2999</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The sphingosine-1-phosphate type 1 (S1P1) receptor is a new target in the treatment of auto-immune diseases as evidenced by the recent approval of FTY720 (Fingolimod).  The ligand-binding pocket of the S1P1 receptor was generally characterized but detailed insight into ligand-specific differences is still lacking.  The aim of the current study is to det. differences in ligand-induced S1P1 receptor activation using an in silico guided site-directed mutagenesis strategy.  S1P1 mutant receptors (modifications of residues Y982.57, R1203.28, F1253.33) were probed with a chem. diverse set of S1P1 agonists (S1P, dihydro-S1P (dhS1P), R-, S- and racemic FTY720-P, VPC24191, SEW2871).  Mutation of the R3.28 residue generally results in a redn. of the potency of all ligands although the synthetic ligands including FTY720-P are less sensitive to these mutations.  The Y2.57F mutation does not affect the potency of any of the ligands tested, but for all ligands except FTY720-P a significant decrease in potency is obsd. at the Y2.57A mutant.  The F3.33A mutation significantly decreased the potency of FTY720-P and is detrimental for SEW2871 and VPC24191.  The non-arom. endogenous ligands S1P and dhS1P are less affected by this mutation.  Our in silico guided mutagenesis studies identified new mol. determinants of ligand-induced S1P1 receptor activation: (1) the flexibility of the polar head of the agonist to maintain a tight H-bond network with R3.28 and (2) the ability of the agonist to make arom. π-stacking interactions with F3.33.  Interestingly, FTY720-P has both chem. properties and is the only ligand that can efficiently activate the Y2.57A mutant.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqua3AJCEtTi7Vg90H21EOLACvtfcHk0lhodEVxm0NyIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVGltrnJ&md5=128b2a264a474a637c7c9217aa34d406</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2Fj.ejphar.2011.05.071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejphar.2011.05.071%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BLoenen%26aufirst%3DP.%2BB.%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26aulast%3DVerzijl%26aufirst%3DD.%26aulast%3DLeurs%26aufirst%3DR.%26aulast%3DRognan%26aufirst%3DD.%26aulast%3DPeters%26aufirst%3DS.%2BL.%26aulast%3DAlewijnse%26aufirst%3DA.%2BE.%26atitle%3DAgonist-dependent%2520effects%2520of%2520mutations%2520in%2520the%2520sphingosine-1-phosphate%2520type%25201%2520receptor%26jtitle%3DEur.%2520J.%2520Pharmacol.%26date%3D2011%26volume%3D667%26spage%3D105%26epage%3D112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group">van der Horst, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van der Pijl, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulder-Krieger, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bender, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">IJzerman, A. P.</span><span> </span><span class="NLM_article-title">Substructure-based virtual screening for adenosine A2A receptor ligands</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">2302</span><span class="NLM_x">–</span> <span class="NLM_lpage">2311</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=2302-2311&author=E.+van+der+Horstauthor=R.+van+der+Pijlauthor=T.+Mulder-Kriegerauthor=A.+Benderauthor=A.+P.+IJzerman&title=Substructure-based+virtual+screening+for+adenosine+A2A+receptor+ligands"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2Bder%2BHorst%26aufirst%3DE.%26aulast%3Dvan%2Bder%2BPijl%26aufirst%3DR.%26aulast%3DMulder-Krieger%26aufirst%3DT.%26aulast%3DBender%26aufirst%3DA.%26aulast%3DIJzerman%26aufirst%3DA.%2BP.%26atitle%3DSubstructure-based%2520virtual%2520screening%2520for%2520adenosine%2520A2A%2520receptor%2520ligands%26jtitle%3DChemMedChem%26date%3D2011%26volume%3D6%26spage%3D2302%26epage%3D2311" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group">Sanders, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verhoeven, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Graaf, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roumen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vroling, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nabuurs, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Vlieg, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klomp, J. P.</span><span> </span><span class="NLM_article-title">Snooker: a structure-based pharmacophore generation tool applied to class A GPCRs</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2277</span><span class="NLM_x">–</span> <span class="NLM_lpage">2292</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci200088d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFahur7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=2277-2292&author=M.+P.+Sandersauthor=S.+Verhoevenauthor=C.+De+Graafauthor=L.+Roumenauthor=B.+Vrolingauthor=S.+B.+Nabuursauthor=J.+de+Vliegauthor=J.+P.+Klomp&title=Snooker%3A+a+structure-based+pharmacophore+generation+tool+applied+to+class+A+GPCRs"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Snooker: A Structure-Based Pharmacophore Generation Tool Applied to Class A GPCRs</span></div><div class="casAuthors">Sanders, Marijn P. A.; Verhoeven, Stefan; de Graaf, Chris; Roumen, Luc; Vroling, Bas; Nabuurs, Sander B.; de Vlieg, Jacob; Klomp, Jan P. G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2277-2292</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">G-protein coupled receptors (GPCRs) are important drug targets for various diseases and of major interest to pharmaceutical companies.  The function of individual members of this protein family can be modulated by the binding of small mols. at the extracellular side of the structurally conserved transmembrane (TM) domain.  Here, we present Snooker, a structure-based approach to generate pharmacophore hypotheses for compds. binding to this extracellular side of the TM domain.  Snooker does not require knowledge of ligands, is therefore suitable for apo-proteins, and can be applied to all receptors of the GPCR protein family.  The method comprises the construction of a homol. model of the TM domains and prioritization of residues on the probability of being ligand binding.  Subsequently, protein properties are converted to ligand space, and pharmacophore features are generated at positions where protein ligand interactions are likely.  Using this semiautomated knowledge-driven bioinformatics approach we have created pharmacophore hypotheses for 15 different GPCRs from several different subfamilies.  For the beta-2-adrenergic receptor we show that ligand poses predicted by Snooker pharmacophore hypotheses reproduce literature supported binding modes for ∼75% of compds. fulfilling pharmacophore constraints.  All 15 pharmacophore hypotheses represent interactions with essential residues for ligand binding as obsd. in mutagenesis expts. and compd. selections based on these hypotheses are shown to be target specific.  For 8 out of 15 targets enrichment factors above 10-fold are obsd. in the top 0.5% ranked compds. in a virtual screen.  Addnl., prospectively predicted ligand binding poses in the human dopamine D3 receptor based on Snooker pharmacophores were ranked among the best models in the community wide GPCR dock 2010.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNiE_rJlvsFbVg90H21EOLACvtfcHk0ljUpiJOuNwxcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFahur7J&md5=518d4365c9d417acac0645af83a3cd65</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fci200088d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200088d%26sid%3Dliteratum%253Aachs%26aulast%3DSanders%26aufirst%3DM.%2BP.%26aulast%3DVerhoeven%26aufirst%3DS.%26aulast%3DDe%2BGraaf%26aufirst%3DC.%26aulast%3DRoumen%26aufirst%3DL.%26aulast%3DVroling%26aufirst%3DB.%26aulast%3DNabuurs%26aufirst%3DS.%2BB.%26aulast%3Dde%2BVlieg%26aufirst%3DJ.%26aulast%3DKlomp%26aufirst%3DJ.%2BP.%26atitle%3DSnooker%253A%2520a%2520structure-based%2520pharmacophore%2520generation%2520tool%2520applied%2520to%2520class%2520A%2520GPCRs%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D2277%26epage%3D2292" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Wawer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Similarity-potency trees: a method to search for SAR information in compound data sets and derive SAR rules</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">1395</span><span class="NLM_x">–</span> <span class="NLM_lpage">409</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci100197b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BC3cXovVCiurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1395-409&author=M.+Wawerauthor=J.+Bajorath&title=Similarity-potency+trees%3A+a+method+to+search+for+SAR+information+in+compound+data+sets+and+derive+SAR+rules"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Similarity-Potency Trees: A Method to Search for SAR Information in Compound Data Sets and Derive SAR Rules</span></div><div class="casAuthors">Wawer, Mathias; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1395-1409</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An intuitive and generally applicable anal. method, termed similarity-potency tree (SPT), is introduced to mine structure-activity relation (SAR) information in compd. data sets of any source.  Only compd. potency values and nearest-neighbor similarity relationships are considered.  Rather than analyzing a data set as a whole, in part overlapping compd. neighborhoods are systematically generated and represented as SPTs.  This local anal. scheme simplifies the evaluation of SAR information and SPTs of high SAR information content are easily identified.  By inspecting only a limited no. of compd. neighborhoods, it is also straightforward to det. whether data sets contain only little or no interpretable SAR information.  Interactive anal. of SPTs is facilitated by reading the trees in two directions, which makes it possible to ext. SAR rules, if available, in a consistent manner.  The simplicity and interpretability of the data structure and the ease of calcn. are characteristic features of this approach.  We apply the methodol. to high-throughput screening and lead optimization data sets, compare the approach to std. clustering techniques, illustrate how SAR rules are derived, and provide some practical guidance how to best utilize the methodol.  The SPT program is made freely available to the scientific community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocIJW9gu2adrVg90H21EOLACvtfcHk0ljUpiJOuNwxcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXovVCiurc%253D&md5=7d4609c7ca3a695c0658233ca7867616</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fci100197b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100197b%26sid%3Dliteratum%253Aachs%26aulast%3DWawer%26aufirst%3DM.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DSimilarity-potency%2520trees%253A%2520a%2520method%2520to%2520search%2520for%2520SAR%2520information%2520in%2520compound%2520data%2520sets%2520and%2520derive%2520SAR%2520rules%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D1395%26epage%3D409" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Steffen, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kogej, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tyrchan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engkvist, O.</span><span> </span><span class="NLM_article-title">Comparison of molecular fingerprint methods on the basis of biological profile data</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">338</span><span class="NLM_x">–</span> <span class="NLM_lpage">347</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci800326z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFeitrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=338-347&author=A.+Steffenauthor=T.+Kogejauthor=C.+Tyrchanauthor=O.+Engkvist&title=Comparison+of+molecular+fingerprint+methods+on+the+basis+of+biological+profile+data"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of Molecular Fingerprint Methods on the Basis of Biological Profile Data</span></div><div class="casAuthors">Steffen, Andreas; Kogej, Thierry; Tyrchan, Christian; Engkvist, Ola</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">338-347</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this study we evaluated a set of mol. fingerprint methods with respect to their capability to reproduce similarities in the biol. activity space.  The evaluation presented in this paper is therefore different from many other fingerprint studies, in which the enrichment of active compds. binding to the same target as selected query structures was studied.  Conversely, our data set was extd. from the BioPrint database, which contains uniformly derived biol. activity profiles of mainly marketed drugs for a range of biol. assays relevant for the pharmaceutical industry.  We compared calcd. mol. fingerprint similarity values between all compd. pairs of the data set with the corresponding similarities in the biol. activity space and addnl. analyzed agreements of generated clusterings.  A closer anal. of the compd. pairs with a high biol. activity similarity revealed that fingerprint methods such as CHEMGPS or TRUST4, which describe global features of a mol. such as physicochem. properties and pharmacophore patterns, might be better suited to describe similarity of biol. activity profiles than purely structural fingerprint methods.  It is therefore suggested that the usage of these fingerprint methods could increase the probability of finding mols. with a similar biol. activity profile but yet a different chem. structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA5RyGYCpkULVg90H21EOLACvtfcHk0ljUpiJOuNwxcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFeitrY%253D&md5=0a296a850c61bea5a656c9e7e5c6df7c</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fci800326z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800326z%26sid%3Dliteratum%253Aachs%26aulast%3DSteffen%26aufirst%3DA.%26aulast%3DKogej%26aufirst%3DT.%26aulast%3DTyrchan%26aufirst%3DC.%26aulast%3DEngkvist%26aufirst%3DO.%26atitle%3DComparison%2520of%2520molecular%2520fingerprint%2520methods%2520on%2520the%2520basis%2520of%2520biological%2520profile%2520data%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D338%26epage%3D347" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref263"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref263'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit263"><span><span class="NLM_contrib-group">Lee, H. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, W. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, J. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoon, M. H.</span><span> </span><span class="NLM_article-title">Roles of adenosine receptor subtypes on the antinociceptive effect of sildenafil in rat spinal cord</span> <span class="citation_source-journal">Neurosci. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">480</span><span class="NLM_x">, </span> <span class="NLM_fpage">182</span><span class="NLM_x">–</span> <span class="NLM_lpage">185</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=480&publication_year=2010&pages=182-185&author=H.+G.+Leeauthor=W.+M.+Kimauthor=J.+I.+Choiauthor=M.+H.+Yoon&title=Roles+of+adenosine+receptor+subtypes+on+the+antinociceptive+effect+of+sildenafil+in+rat+spinal+cord"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit263&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DH.%2BG.%26aulast%3DKim%26aufirst%3DW.%2BM.%26aulast%3DChoi%26aufirst%3DJ.%2BI.%26aulast%3DYoon%26aufirst%3DM.%2BH.%26atitle%3DRoles%2520of%2520adenosine%2520receptor%2520subtypes%2520on%2520the%2520antinociceptive%2520effect%2520of%2520sildenafil%2520in%2520rat%2520spinal%2520cord%26jtitle%3DNeurosci.%2520Lett.%26date%3D2010%26volume%3D480%26spage%3D182%26epage%3D185" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref264"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref264'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit264"><span><span class="NLM_contrib-group">Salloum, F. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Das, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kukreja, R. C.</span><span> </span><span class="NLM_article-title">Adenosine A(1) receptor mediates delayed cardioprotective effect of sildenafil in mouse</span> <span class="citation_source-journal">J. Mol. Cell. Cardiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">545</span><span class="NLM_x">–</span> <span class="NLM_lpage">551</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref264/cit264&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1016%2Fj.yjmcc.2007.08.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref264/cit264&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=17920622" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref264/cit264&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1amsLrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2007&pages=545-551&author=F.+N.+Salloumauthor=A.+Dasauthor=C.+S.+Thomasauthor=C.+Yinauthor=R.+C.+Kukreja&title=Adenosine+A%281%29+receptor+mediates+delayed+cardioprotective+effect+of+sildenafil+in+mouse"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit264R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Adenosine A1 receptor mediates delayed cardioprotective effect of sildenafil in mouse</span></div><div class="casAuthors">Salloum, Fadi N.; Das, Anindita; Thomas, Christopher S.; Yin, Chang; Kukreja, Rakesh C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular and Cellular Cardiology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">545-551</span>CODEN:
                <span class="NLM_cas:coden">JMCDAY</span>;
        ISSN:<span class="NLM_cas:issn">0022-2828</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Sildenafil induces powerful cardioprotection against ischemia/reperfusion (I/R) injury.  Since adenosine is known to be a major trigger of ischemic preconditioning, we hypothesized that A1 adenosine receptor (A1AR) activation plays a role in sildenafil-induced cardioprotective signaling.  Adult male C57BL wild-type (WT) mice or their corresponding A1AR knockout (A1AR-KO) mice were treated i.p. with either sildenafil (0.71 mg/kg, equiv. to 50 mg dose for a 70 kg patient) or vol.-matched saline.  The selective A1AR antagonist 8-cyclopentyl-1,3-dipropyxanthine (DPCPX; 0.1 mg/kg, i.p.) was administered 30 min before sildenafil.  The hearts were isolated 24 h later and subjected to 30 min of global ischemia and 1 h of reperfusion in Langendorff mode.  Post-ischemic myocardial infarct size (mean; % of risk area) was reduced in C57BL-WT mice treated with sildenafil (5.6) vs. saline control group (27.3; n = 6/each).  However, sildenafil failed to protect the A1AR-KO hearts (31.6 vs. 32.3 with saline).  Addnl., DPCPX treatment abolished the infarct limiting effect of sildenafil (27.3).  DPCPX alone had no effect on infarct size as compared with the control group.  No significant changes in post-ischemic recovery of left ventricular pressure and heart rate were obsd. in the sildenafil-treated group.  We further examd. the effect of sildenafil in protection against simulated ischemia and reoxygenation injury in adult cardiomyocytes derived from WT and A1AR-KO mice.  WT myocytes treated with sildenafil (1 μM) demonstrated significantly lower trypan blue-pos. necrotic cells.  However, cardiomyocytes derived from A1AR-KO mice or DPCPX-treated WT cells failed to show protection against necrosis with sildenafil.  These results suggest that A1AR activation following treatment with sildenafil plays an integral role in the signaling cascade responsible for delayed protection against global I/R injury.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaEMDtjnKRrrVg90H21EOLACvtfcHk0li4BewW-a-iyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1amsLrN&md5=796f04cf01c42c4e3ea7ccaf56ec45af</span></div><a href="/servlet/linkout?suffix=cit264&amp;dbid=16384&amp;doi=10.1016%2Fj.yjmcc.2007.08.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.yjmcc.2007.08.014%26sid%3Dliteratum%253Aachs%26aulast%3DSalloum%26aufirst%3DF.%2BN.%26aulast%3DDas%26aufirst%3DA.%26aulast%3DThomas%26aufirst%3DC.%2BS.%26aulast%3DYin%26aufirst%3DC.%26aulast%3DKukreja%26aufirst%3DR.%2BC.%26atitle%3DAdenosine%2520A%25281%2529%2520receptor%2520mediates%2520delayed%2520cardioprotective%2520effect%2520of%2520sildenafil%2520in%2520mouse%26jtitle%3DJ.%2520Mol.%2520Cell.%2520Cardiol.%26date%3D2007%26volume%3D43%26spage%3D545%26epage%3D551" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Wang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huai, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cai, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zoraghi, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Francis, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corbin, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robinson, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xin, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ke, H.</span><span> </span><span class="NLM_article-title">Multiple conformations of phosphodiesterase-5: implications for enzyme function and drug development</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">281</span><span class="NLM_x">, </span> <span class="NLM_fpage">21469</span><span class="NLM_x">–</span> <span class="NLM_lpage">21479</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1074%2Fjbc.M512527200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=16735511" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD28Xnt1KjsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=21469-21479&author=H.+Wangauthor=Y.+Liuauthor=Q.+Huaiauthor=J.+Caiauthor=R.+Zoraghiauthor=S.+H.+Francisauthor=J.+D.+Corbinauthor=H.+Robinsonauthor=Z.+Xinauthor=G.+Linauthor=H.+Ke&title=Multiple+conformations+of+phosphodiesterase-5%3A+implications+for+enzyme+function+and+drug+development"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Multiple Conformations of Phosphodiesterase-5: implications for enzyme function and drug development</span></div><div class="casAuthors">Wang, Huanchen; Liu, Yudong; Huai, Qing; Cai, Jiwen; Zoraghi, Roya; Francis, Sharron H.; Corbin, Jackie D.; Robinson, Howard; Xin, Zhongcheng; Lin, Guiting; Ke, Hengming</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">281</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">21469-21479</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Phosphodiesterase-5 (PDE5) is the target for sildenafil, vardenafil, and tadalafil, which are drugs for treatment of erectile dysfunction and pulmonary hypertension.  We report here the crystal structures of a fully active catalytic domain of unliganded PDE5A1 and its complexes with sildenafil or icarisid II.  These structures together with the PDE5A1-isobutyl-1-methylxanthine complex show that the H-loop (residues 660-683) at the active site of PDE5A1 has four different conformations and migrates 7-35 Å upon inhibitor binding.  In addn., the conformation of sildenafil reported herein differs significantly from those in the previous structures of chimerically hybridized or almost inactive PDE5.  Mutagenesis and kinetic analyses confirm that the H-loop is particularly important for substrate recognition and that invariant Gly659, which immediately precedes the H-loop, is crit. for optimal substrate affinity and catalytic activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplF9ohkT9ogLVg90H21EOLACvtfcHk0li4BewW-a-iyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xnt1KjsLs%253D&md5=3a5ac86273a74da4a559bc6efc647181</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M512527200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M512527200%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DHuai%26aufirst%3DQ.%26aulast%3DCai%26aufirst%3DJ.%26aulast%3DZoraghi%26aufirst%3DR.%26aulast%3DFrancis%26aufirst%3DS.%2BH.%26aulast%3DCorbin%26aufirst%3DJ.%2BD.%26aulast%3DRobinson%26aufirst%3DH.%26aulast%3DXin%26aufirst%3DZ.%26aulast%3DLin%26aufirst%3DG.%26aulast%3DKe%26aufirst%3DH.%26atitle%3DMultiple%2520conformations%2520of%2520phosphodiesterase-5%253A%2520implications%2520for%2520enzyme%2520function%2520and%2520drug%2520development%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D21469%26epage%3D21479" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Gozalbes, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rolland, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicolai, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paugam, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coussy, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horvath, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbosa, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mao, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Revah, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Froloff, N.</span><span> </span><span class="NLM_article-title">QSAR strategy and experimental validation for the development of a GPCR focused library</span> <span class="citation_source-journal">QSAR Comb. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">508</span><span class="NLM_x">–</span> <span class="NLM_lpage">516</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2005&pages=508-516&author=R.+Gozalbesauthor=C.+Rollandauthor=E.+Nicolaiauthor=M.+Paugamauthor=L.+Coussyauthor=D.+Horvathauthor=F.+Barbosaauthor=B.+Maoauthor=F.+Revahauthor=N.+Froloff&title=QSAR+strategy+and+experimental+validation+for+the+development+of+a+GPCR+focused+library"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGozalbes%26aufirst%3DR.%26aulast%3DRolland%26aufirst%3DC.%26aulast%3DNicolai%26aufirst%3DE.%26aulast%3DPaugam%26aufirst%3DM.%26aulast%3DCoussy%26aufirst%3DL.%26aulast%3DHorvath%26aufirst%3DD.%26aulast%3DBarbosa%26aufirst%3DF.%26aulast%3DMao%26aufirst%3DB.%26aulast%3DRevah%26aufirst%3DF.%26aulast%3DFroloff%26aufirst%3DN.%26atitle%3DQSAR%2520strategy%2520and%2520experimental%2520validation%2520for%2520the%2520development%2520of%2520a%2520GPCR%2520focused%2520library%26jtitle%3DQSAR%2520Comb.%2520Sci.%26date%3D2005%26volume%3D24%26spage%3D508%26epage%3D516" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref267"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref267'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit267"><span><span class="NLM_contrib-group">Goldstein, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span> </span><span class="NLM_article-title">High-throughput kinase profiling as a platform for drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discov.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">391</span><span class="NLM_x">–</span> <span class="NLM_lpage">397</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=391-397&author=D.+M.+Goldsteinauthor=N.+S.+Grayauthor=P.+P.+Zarrinkar&title=High-throughput+kinase+profiling+as+a+platform+for+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit267&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGoldstein%26aufirst%3DD.%2BM.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DHigh-throughput%2520kinase%2520profiling%2520as%2520a%2520platform%2520for%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discov.%26date%3D2008%26volume%3D7%26spage%3D391%26epage%3D397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">Sanders, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fleuren, W. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verhoeven, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van den Beld, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alkema, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Vlieg, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klomp, J. P.</span><span> </span><span class="NLM_article-title">ss-TEA: entropy based identification of receptor specific ligand binding residues from a multiple sequence alignment of class A GPCRs</span> <span class="citation_source-journal">BMC Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">332</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1186%2F1471-2105-12-332" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=21831265" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFCktbrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=332&author=M.+P.+Sandersauthor=W.+W.+Fleurenauthor=S.+Verhoevenauthor=S.+van+den+Beldauthor=W.+Alkemaauthor=J.+de+Vliegauthor=J.+P.+Klomp&title=ss-TEA%3A+entropy+based+identification+of+receptor+specific+ligand+binding+residues+from+a+multiple+sequence+alignment+of+class+A+GPCRs"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">ss-TEA: entropy based identification of receptor specific ligand binding residues from a multiple sequence alignment of class A GPCRs</span></div><div class="casAuthors">Sanders, Marijn P. A.; Fleuren, Wilco W. M.; Verhoeven, Stefan; van den Beld, Sven; Alkema, Wynand; de Vlieg, Jacob; Klomp, Jan P. G.</div><div class="citationInfo"><span class="NLM_cas:title">BMC Bioinformatics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">332</span>CODEN:
                <span class="NLM_cas:coden">BBMIC4</span>;
        ISSN:<span class="NLM_cas:issn">1471-2105</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: G-protein coupled receptors (GPCRs) are involved in many different physiol. processes and their function can be modulated by small mols. which bind in the transmembrane (TM) domain.  Because of their structural and sequence conservation, the TM domains are often used in bioinformatics approaches to first create a multiple sequence alignment (MSA) and subsequently identify ligand binding positions.  So far methods have been developed to predict the common ligand binding residue positions for class A GPCRs.  Results: Here we present 1) ss-TEA, a method to identify specific ligand binding residue positions for any receptor, predicated on high quality sequence information.  2) The largest MSA of class A non olfactory GPCRs in the public domain consisting of 13324 sequences covering most of the species homologues of the human set of GPCRs.  A set of ligand binding residue positions extd. from literature of 10 different receptors shows that our method has the best ligand binding residue prediction for 9 of these 10 receptors compared to another state-of-the-art method.  Conclusions: The combination of the large multi species alignment and the newly introduced residue selection method ss-TEA can be used to rapidly identify subfamily specific ligand binding residues.  This approach can aid the design of site directed mutagenesis expts., explain receptor function and improve modeling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTGzbW8sSKHLVg90H21EOLACvtfcHk0li4BewW-a-iyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFCktbrF&md5=1673e238874e269dc59495e20d3241e8</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-12-332&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-12-332%26sid%3Dliteratum%253Aachs%26aulast%3DSanders%26aufirst%3DM.%2BP.%26aulast%3DFleuren%26aufirst%3DW.%2BW.%26aulast%3DVerhoeven%26aufirst%3DS.%26aulast%3Dvan%2Bden%2BBeld%26aufirst%3DS.%26aulast%3DAlkema%26aufirst%3DW.%26aulast%3Dde%2BVlieg%26aufirst%3DJ.%26aulast%3DKlomp%26aufirst%3DJ.%2BP.%26atitle%3Dss-TEA%253A%2520entropy%2520based%2520identification%2520of%2520receptor%2520specific%2520ligand%2520binding%2520residues%2520from%2520a%2520multiple%2520sequence%2520alignment%2520of%2520class%2520A%2520GPCRs%26jtitle%3DBMC%2520Bioinf.%26date%3D2011%26volume%3D12%26spage%3D332" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group">Gasteiger, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rudolph, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sadowski, J.</span><span> </span><span class="NLM_article-title">Automatic generation of 3D-atomic coordinates for organic molecules</span> <span class="citation_source-journal">Tetrahedron Comput. Methodol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">537</span><span class="NLM_x">–</span> <span class="NLM_lpage">547</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1016%2F0898-5529%2890%2990156-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADyaK3sXmtlCn" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1990&pages=537-547&author=J.+Gasteigerauthor=C.+Rudolphauthor=J.+Sadowski&title=Automatic+generation+of+3D-atomic+coordinates+for+organic+molecules"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Automatic generation of 3D atomic coordinates for organic molecules</span></div><div class="casAuthors">Gasteiger, J.; Rudolph, C.; Sadowski, J.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Computer Methodology</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">6c</span>),
    <span class="NLM_cas:pages">537-47</span>CODEN:
                <span class="NLM_cas:coden">TCMTE6</span>;
        ISSN:<span class="NLM_cas:issn">0898-5529</span>.
    </div><div class="casAbstract">A system has been developed that can automatically generate 3-dimensional at. coordinates from the constitution of a mol. as expressed by a connection table.  The program, CORINA, is applicable to the entire range of org. chem.  It can also handle structures that are beyond the scope of some other programs, e.g., macrocyclic and polymacrocyclic mols.  Computation times are short and the results compare favorably with data from x-ray crystallog. and with those of mol. mechanics calcns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrS-kIKXGPzYLVg90H21EOLACvtfcHk0lgXqGsR8FkyVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXmtlCn&md5=d5929de163b892f0354ea33a59097059</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2F0898-5529%2890%2990156-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0898-5529%252890%252990156-3%26sid%3Dliteratum%253Aachs%26aulast%3DGasteiger%26aufirst%3DJ.%26aulast%3DRudolph%26aufirst%3DC.%26aulast%3DSadowski%26aufirst%3DJ.%26atitle%3DAutomatic%2520generation%2520of%25203D-atomic%2520coordinates%2520for%2520organic%2520molecules%26jtitle%3DTetrahedron%2520Comput.%2520Methodol.%26date%3D1990%26volume%3D3%26spage%3D537%26epage%3D547" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group">Liu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bai, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ouyang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H.</span><span> </span><span class="NLM_article-title">Cyndi: a multi-objective evolution algorithm based method for bioactive molecular conformational generation</span> <span class="citation_source-journal">BMC Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">101</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1186%2F1471-2105-10-101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=19335906" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A280%3ADC%252BD1MzisFSktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2009&pages=101&author=X.+Liuauthor=F.+Baiauthor=S.+Ouyangauthor=X.+Wangauthor=H.+Liauthor=H.+Jiang&title=Cyndi%3A+a+multi-objective+evolution+algorithm+based+method+for+bioactive+molecular+conformational+generation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Cyndi: a multi-objective evolution algorithm based method for bioactive molecular conformational generation</span></div><div class="casAuthors">Liu Xiaofeng; Bai Fang; Ouyang Sisheng; Wang Xicheng; Li Honglin; Jiang Hualiang</div><div class="citationInfo"><span class="NLM_cas:title">BMC bioinformatics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">101</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Conformation generation is a ubiquitous problem in molecule modelling.  Many applications require sampling the broad molecular conformational space or perceiving the bioactive conformers to ensure success.  Numerous in silico methods have been proposed in an attempt to resolve the problem, ranging from deterministic to non-deterministic and systemic to stochastic ones.  In this work, we described an efficient conformation sampling method named Cyndi, which is based on multi-objective evolution algorithm.  RESULTS:  The conformational perturbation is subjected to evolutionary operation on the genome encoded with dihedral torsions.  Various objectives are designated to render the generated Pareto optimal conformers to be energy-favoured as well as evenly scattered across the conformational space.  An optional objective concerning the degree of molecular extension is added to achieve geometrically extended or compact conformations which have been observed to impact the molecular bioactivity (J Comput -Aided Mol Des 2002, 16: 105-112).  Testing the performance of Cyndi against a test set consisting of 329 small molecules reveals an average minimum RMSD of 0.864 A to corresponding bioactive conformations, indicating Cyndi is highly competitive against other conformation generation methods.  Meanwhile, the high-speed performance (0.49 +/- 0.18 seconds per molecule) renders Cyndi to be a practical toolkit for conformational database preparation and facilitates subsequent pharmacophore mapping or rigid docking.  The copy of precompiled executable of Cyndi and the test set molecules in mol2 format are accessible in Additional file 1.  CONCLUSION:  On the basis of MOEA algorithm, we present a new, highly efficient conformation generation method, Cyndi, and report the results of validation and performance studies comparing with other four methods.  The results reveal that Cyndi is capable of generating geometrically diverse conformers and outperforms other four multiple conformer generators in the case of reproducing the bioactive conformations against 329 structures.  The speed advantage indicates Cyndi is a powerful alternative method for extensive conformational sampling and large-scale conformer database preparation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQs2QpRXYXZbmGmni0r9GoKfW6udTcc2ebeIGbLGixQyLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MzisFSktg%253D%253D&md5=eec10b3b9f6c8e32c789d069450065df</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-10-101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-10-101%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DBai%26aufirst%3DF.%26aulast%3DOuyang%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DH.%26atitle%3DCyndi%253A%2520a%2520multi-objective%2520evolution%2520algorithm%2520based%2520method%2520for%2520bioactive%2520molecular%2520conformational%2520generation%26jtitle%3DBMC%2520Bioinf.%26date%3D2009%26volume%3D10%26spage%3D101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">ZINC—a free database of commercially available compounds for virtual screening</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">177</span><span class="NLM_x">–</span> <span class="NLM_lpage">182</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci049714%2B" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVOjt77J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2005&pages=177-182&author=J.+J.+Irwinauthor=B.+K.+Shoichet&title=ZINC%E2%80%94a+free+database+of+commercially+available+compounds+for+virtual+screening"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC - A Free Database of Commercially Available Compounds for Virtual Screening</span></div><div class="casAuthors">Irwin, John J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">177-182</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A crit. barrier to entry into structure-based virtual screening is the lack of a suitable, easy to access database of purchasable compds.  We have therefore prepd. a library of 727 842 mols., each with 3D structure, using catalogs of compds. from vendors (the size of this library continues to grow).  The mols. have been assigned biol. relevant protonation states and are annotated with properties such as mol. wt., calcd. LogP, and no. of rotatable bonds.  Each mol. in the library contains vendor and purchasing information and is ready for docking using a no. of popular docking programs.  Within certain limits, the mols. are prepd. in multiple protonation states and multiple tautomeric forms.  In one format, multiple conformations are available for the mols.  This database is available for free download (http://zinc.docking.org) in several common file formats including SMILES, mol2, 3D SDF, and DOCK flexibase format.  A Web-based query tool incorporating a mol. drawing interface enables the database to be searched and browsed and subsets to be created.  Users can process their own mols. by uploading them to a server.  Our hope is that this database will bring virtual screening libraries to a wide community of structural biologists and medicinal chemists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPhFw5E2XACbVg90H21EOLACvtfcHk0lgXqGsR8FkyVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVOjt77J&md5=e3892b7dc8608b17a3e63541a5ed60e6</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1021%2Fci049714%2B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci049714%252B%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DZINC%25E2%2580%2594a%2520free%2520database%2520of%2520commercially%2520available%2520compounds%2520for%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2005%26volume%3D45%26spage%3D177%26epage%3D182" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group">Nijssen, S.; Kok, J. N.</span><span> </span><span class="NLM_article-title">A Quickstart in Frequent Structure Mining Can Make a Difference</span>.  <span class="citation_source-book">Proceedings of the 10th ACM SIGKDD International Conference on Knowledge Discovery and Data Mining, 2004</span>; <span class="NLM_contrib-group">Kohavi, R.; Gehrke, J.; DuMouchel, W.; Ghosh, J.</span>, Eds.; <span class="NLM_publisher-name">ACM</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2004</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">647</span><span class="NLM_x">–</span> <span class="NLM_lpage">652</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1145%2F1014052.1014134" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2004&pages=647-652&author=S.+Nijssen&author=J.+N.+Kokauthor=R.+Kohavi&author=J.+Gehrke&author=W.+DuMouchel&author=J.+Ghosh&title=Proceedings+of+the+10th+ACM+SIGKDD+International+Conference+on+Knowledge+Discovery+and+Data+Mining%2C+2004"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1145%2F1014052.1014134&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1145%252F1014052.1014134%26sid%3Dliteratum%253Aachs%26aulast%3DNijssen%26aufirst%3DS.%26atitle%3DA%2520Quickstart%2520in%2520Frequent%2520Structure%2520Mining%2520Can%2520Make%2520a%2520Difference%26btitle%3DProceedings%2520of%2520the%252010th%2520ACM%2520SIGKDD%2520International%2520Conference%2520on%2520Knowledge%2520Discovery%2520and%2520Data%2520Mining%252C%25202004%26aulast%3DKohavi%26aufirst%3DR.%26pub%3DACM%26date%3D2004%26spage%3D647%26epage%3D652" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group">Del Rio, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbosa, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caporuscio, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mangiatordi, G. F.</span><span> </span><span class="NLM_article-title">CoCoCo: a free suite of multiconformational chemical databases for high-throughput virtual screening purposes</span> <span class="citation_source-journal">Mol. BioSyst.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">2122</span><span class="NLM_x">–</span> <span class="NLM_lpage">2128</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1039%2Fc0mb00039f" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=20694263" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1Kqt7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=2122-2128&author=A.+Del+Rioauthor=A.+J.+Barbosaauthor=F.+Caporuscioauthor=G.+F.+Mangiatordi&title=CoCoCo%3A+a+free+suite+of+multiconformational+chemical+databases+for+high-throughput+virtual+screening+purposes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">CoCoCo: a free suite of multiconformational chemical databases for high-throughput virtual screening purposes</span></div><div class="casAuthors">Del Rio, Alberto; Barbosa, Armenio Jorge Moura; Caporuscio, Fabiana; Mangiatordi, Giuseppe Felice</div><div class="citationInfo"><span class="NLM_cas:title">Molecular BioSystems</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2122-2128</span>CODEN:
                <span class="NLM_cas:coden">MBOIBW</span>;
        ISSN:<span class="NLM_cas:issn">1742-206X</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">In the last few decades, virtual screening has proved to be able to guide the selection of new hit compds. with predefined biol. activity.  However, the usage of these computational techniques is often assocd. with resource- and time-consuming prepn. protocols.  In this work we present Com. Compd. Collection (CoCoCo), a suite of free and ready-to-use chem. databases to help setting up in silico screening projects.  CoCoCo collects mol. structural information of com. compds. from various chem. vendors by merging them in a unique, non-redundant format.  CoCoCo databases are prepd. with transparent and straightforward routines based on state-of-the-art computational tools that introduce comprehensive structural information about tautomers, stereoisomers and conformational states of each compd.  CoCoCo suite is esp. conceived as a set of valuable tools that may help a wide range of researchers who wish to initiate their own project in the field of computational drug design.  CoCoCo suite is available free of charge at the website .</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4OJvTFBFY4bVg90H21EOLACvtfcHk0lhT7DxZorU_qQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1Kqt7fL&md5=2cddeafe06fb7afb949dc3600d095ff7</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1039%2Fc0mb00039f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc0mb00039f%26sid%3Dliteratum%253Aachs%26aulast%3DDel%2BRio%26aufirst%3DA.%26aulast%3DBarbosa%26aufirst%3DA.%2BJ.%26aulast%3DCaporuscio%26aufirst%3DF.%26aulast%3DMangiatordi%26aufirst%3DG.%2BF.%26atitle%3DCoCoCo%253A%2520a%2520free%2520suite%2520of%2520multiconformational%2520chemical%2520databases%2520for%2520high-throughput%2520virtual%2520screening%2520purposes%26jtitle%3DMol.%2520BioSyst.%26date%3D2010%26volume%3D6%26spage%3D2122%26epage%3D2128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group">Lipinski, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lombardo, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dominy, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feeney, P. J.</span><span> </span><span class="NLM_article-title">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span> <span class="citation_source-journal">Adv. Drug Delivery Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">3</span><span class="NLM_x">–</span> <span class="NLM_lpage">26</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1016%2FS0169-409X%2800%2900129-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=11259830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2001&pages=3-26&author=C.+A.+Lipinskiauthor=F.+Lombardoauthor=B.+W.+Dominyauthor=P.+J.+Feeney&title=Experimental+and+computational+approaches+to+estimate+solubility+and+permeability+in+drug+discovery+and+development+settings"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span></div><div class="casAuthors">Lipinski, C. A.; Lombardo, F.; Dominy, B. W.; Feeney, P. J.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">3-26</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ireland Ltd.</span>)
        </div><div class="casAbstract">A review with 50 refs.  Exptl. and computational approaches to est. soly. and permeability in discovery and development settings are described.  In the discovery setting 'the rule of 5' predicts that poor absorption or permeation is more likely when there are more than 5 H-bond donors, 10 H-bond acceptors, the mol. wt. (MWT) is greater than 500 and the calcd. Log P (CLogP) is greater than 5 (or MlogP >4.15).  Computational methodol. for the rule-based Moriguchi Log P (MLogP) calcn. is described.  Turbidimetric soly. measurement is described and applied to known drugs.  High throughput screening (HTS) leads tend to have higher MWT and Log P and lower turbidimetric soly. than leads in the pre-HTS era.  In the development setting, soly. calcns. focus on exact value prediction and are difficult because of polymorphism.  Recent work on linear free energy relationships and Log P approaches are critically reviewed.  Useful predictions are possible in closely related analog series when coupled with exptl. thermodn. soly. measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYDqa02g16urVg90H21EOLACvtfcHk0lhT7DxZorU_qQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D&md5=c60bb89da68f051c0ee7ac4c0468a0e4</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1016%2FS0169-409X%2800%2900129-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-409X%252800%252900129-0%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DDominy%26aufirst%3DB.%2BW.%26aulast%3DFeeney%26aufirst%3DP.%2BJ.%26atitle%3DExperimental%2520and%2520computational%2520approaches%2520to%2520estimate%2520solubility%2520and%2520permeability%2520in%2520drug%2520discovery%2520and%2520development%2520settings%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2001%26volume%3D46%26spage%3D3%26epage%3D26" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group">Wenlock, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Austin, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barton, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leeson, P. D.</span><span> </span><span class="NLM_article-title">A comparison of physiochemical property profiles of development and marketed oral drugs</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">1250</span><span class="NLM_x">–</span> <span class="NLM_lpage">1256</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm021053p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhslSlsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=1250-1256&author=M.+C.+Wenlockauthor=R.+P.+Austinauthor=P.+Bartonauthor=A.+M.+Davisauthor=P.+D.+Leeson&title=A+comparison+of+physiochemical+property+profiles+of+development+and+marketed+oral+drugs"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">A Comparison of Physiochemical Property Profiles of Development and Marketed Oral Drugs</span></div><div class="casAuthors">Wenlock, Mark C.; Austin, Rupert P.; Barton, Patrick; Davis, Andrew M.; Leeson, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1250-1256</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The process of drug discovery applies rigorous selection pressures.  Marketed oral drugs will generally possess favorable physiochem. properties with respect to absorption, metab., distribution, and clearance.  This paper describes a study in which the distributions of physiochem. properties of oral drugs in different phases of clin. development are compared to those already marketed.  The aim is to identify the trends in physiochem. properties that favor a drug's successful passage through clin. development and on to the market.  Two libraries were created, one of current development oral drugs and one of marketed oral drugs.  Statistical anal. of the two showed that the mean mol. wt. of orally administered drugs in development decreases on passing through each of the different clin. phases and gradually converges toward the mean mol. wt. of marketed oral drugs.  It is also clear that the most lipophilic compds. are being discontinued from development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoF0PksUFJq0bVg90H21EOLACvtfcHk0lhT7DxZorU_qQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhslSlsbY%253D&md5=b9809c803a61ec99a1b4171e97952452</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1021%2Fjm021053p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm021053p%26sid%3Dliteratum%253Aachs%26aulast%3DWenlock%26aufirst%3DM.%2BC.%26aulast%3DAustin%26aufirst%3DR.%2BP.%26aulast%3DBarton%26aufirst%3DP.%26aulast%3DDavis%26aufirst%3DA.%2BM.%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26atitle%3DA%2520comparison%2520of%2520physiochemical%2520property%2520profiles%2520of%2520development%2520and%2520marketed%2520oral%2520drugs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D1250%26epage%3D1256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group">Verzijl, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storelli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scholten, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bosch, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinhart, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Streblow, D. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tensen, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fitzsimons, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zaman, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pease, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Esch, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smit, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leurs, R.</span><span> </span><span class="NLM_article-title">Noncompetitive antagonism and inverse agonism as mechanism of action of nonpeptidergic antagonists at primate and rodent CXCR3 chemokine receptors</span> <span class="citation_source-journal">J. Pharmacol. Exp. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">325</span><span class="NLM_x">, </span> <span class="NLM_fpage">544</span><span class="NLM_x">–</span> <span class="NLM_lpage">555</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1124%2Fjpet.107.134783" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=18270317" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlsVOjtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=325&publication_year=2008&pages=544-555&author=D.+Verzijlauthor=S.+Storelliauthor=D.+J.+Scholtenauthor=L.+Boschauthor=T.+A.+Reinhartauthor=D.+N.+Streblowauthor=C.+P.+Tensenauthor=C.+P.+Fitzsimonsauthor=G.+J.+Zamanauthor=J.+E.+Peaseauthor=I.+J.+de+Eschauthor=M.+J.+Smitauthor=R.+Leurs&title=Noncompetitive+antagonism+and+inverse+agonism+as+mechanism+of+action+of+nonpeptidergic+antagonists+at+primate+and+rodent+CXCR3+chemokine+receptors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Noncompetitive antagonism and inverse agonism as mechanism of action of nonpeptidergic antagonists at primate and rodent CXCR3 chemokine receptors</span></div><div class="casAuthors">Verzijl, Dennis; Storelli, Stefania; Scholten, Danny J.; Bosch, Leontien; Reinhart, Todd A.; Streblow, Daniel N.; Tensen, Cornelis P.; Fitzsimons, Carlos P.; Zaman, Guido J. R.; Pease, James E.; de Esch, Iwan J. P.; Smit, Martine J.; Leurs, Rob</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">325</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">544-555</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">The chemokine receptor CXCR3 is involved in various inflammatory diseases, such as rheumatoid arthritis, multiple sclerosis, psoriasis, and allograft rejection in transplantation patients.  The CXCR3 ligands CXCL9, CXCL10, and CXCL11 are expressed at sites of inflammation, and they attract CXCR3-bearing lymphocytes, thus contributing to the inflammatory process.  In this study, we characterize five nonpeptidergic compds. of different chem. classes that block the action of CXCL10 and CXCL11 at the human CXCR3, i.e., the 3H-pyrido[2,3-d]pyrimidin-4-one derivs. N-1R-[3-(4-ethoxy-phenyl)-4-oxo-3,4-dihydro-pyrido[2,3-d]pyrimidin-2-yl]-ethyl-N-pyridin-3-ylmethyl-2-(4-fluoro-3-trifluoromethyl-phenyl)-acetamide (VUF10472/NBI-74330) and N-1R-[3-(4-ethoxy-phenyl)-4-oxo-3,4-dihydro-pyrido[2,3-d]pyrimidin-2-yl]-ethyl-N-pyridin-3-ylmethyl-2-(4-trifluoromethoxy-phenyl)-acetamide (VUF10085/AMG-487), the 3H-quinazolin-4-one decanoic acid {1-[3-(4-cyano-phenyl)-4-oxo-3,4-dihydro-quinazolin-2-yl]-ethyl}-(2-dimethylamino-ethyl)-amide (VUF5834), the imidazolium compd. 1,3-bis-[2-(3,4-dichloro-phenyl)-2-oxo-ethyl]-3H-imidazol-1-ium bromide (VUF10132), and the quaternary ammonium anilide N,N-dimethyl-N-[4-[[[2-(4-methylphenyl)-6,7-dihydro-5H-benzocyclohepten-8-yl]-carbonyl]amino]benzyl] tetrahydro-2H-pyran-4-aminium chloride (TAK-779).  To understand the action of these CXCR3 antagonists in various animal models of disease, the compds. were also tested at rat and mouse CXCR3, as well as at CXCR3 from rhesus macaque, which was cloned and characterized for the first time in this study.  Except for TAK-779, all compds. show slightly lower affinity for rodent CXCR3 than for primate CXCR3.  In addn., we have characterized the mol. mechanism of action of the various antagonists at the human CXCR3 receptor.  All tested compds. act as noncompetitive antagonists at CXCR3.  Moreover, this noncompetitive behavior is accompanied by inverse agonistic properties of all five compds. as detd. on an identified constitutively active mutant of CXCR3, CXCR3 N3.35A.  It is interesting to note that all compds. except TAK-779 act as full inverse agonists at CXCR3 N3.35A.  TAK-779 shows weak partial inverse agonism at CXCR3 N3.35A, and it probably has a different mode of interaction with CXCR3 than the other two classes of small-mol. inverse agonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDC_7W3UH75bVg90H21EOLACvtfcHk0lgSGjDFfF7BOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlsVOjtrg%253D&md5=10ee30aafa3473c34897de744139c8e3</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1124%2Fjpet.107.134783&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.107.134783%26sid%3Dliteratum%253Aachs%26aulast%3DVerzijl%26aufirst%3DD.%26aulast%3DStorelli%26aufirst%3DS.%26aulast%3DScholten%26aufirst%3DD.%2BJ.%26aulast%3DBosch%26aufirst%3DL.%26aulast%3DReinhart%26aufirst%3DT.%2BA.%26aulast%3DStreblow%26aufirst%3DD.%2BN.%26aulast%3DTensen%26aufirst%3DC.%2BP.%26aulast%3DFitzsimons%26aufirst%3DC.%2BP.%26aulast%3DZaman%26aufirst%3DG.%2BJ.%26aulast%3DPease%26aufirst%3DJ.%2BE.%26aulast%3Dde%2BEsch%26aufirst%3DI.%2BJ.%26aulast%3DSmit%26aufirst%3DM.%2BJ.%26aulast%3DLeurs%26aufirst%3DR.%26atitle%3DNoncompetitive%2520antagonism%2520and%2520inverse%2520agonism%2520as%2520mechanism%2520of%2520action%2520of%2520nonpeptidergic%2520antagonists%2520at%2520primate%2520and%2520rodent%2520CXCR3%2520chemokine%2520receptors%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2008%26volume%3D325%26spage%3D544%26epage%3D555" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group">van der Lee, M. M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bras, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Koppen, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zaman, G. J. R.</span><span> </span><span class="NLM_article-title">beta-Arrestin recruitment assay for the identification of agonists of the sphingosine 1-phosphate receptor EDG1</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">986</span><span class="NLM_x">–</span> <span class="NLM_lpage">998</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1177%2F1087057108326144" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=19036707" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD1MXkslGrsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2008&pages=986-998&author=M.+M.+C.+van+der+Leeauthor=M.+Brasauthor=C.+J.+van+Koppenauthor=G.+J.+R.+Zaman&title=beta-Arrestin+recruitment+assay+for+the+identification+of+agonists+of+the+sphingosine+1-phosphate+receptor+EDG1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">β-arrestin recruitment assay for the identification of agonists of the sphingosine 1-phosphate receptor EDG1</span></div><div class="casAuthors">Van Der Lee, Miranda M. C.; Bras, Maaike; Van Koppen, Chris J.; Zaman, Guido J. R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">986-998</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">β-Arrestin recruitment assays provide a generic assay platform for drug discovery on G-protein-coupled receptors (GPCRs).  The PathHunter assay technol. developed by DiscoveRx (Fremont, CA) uses enzyme fragment complementation of β-galactosidase to measure receptor-β-arrestin proximity by chemiluminescence.  This study describes an agonistic screen on the human endothelial differentiation sphingolipid GPCR 1 (EDG1), also known as S1P1, using PathHunter β-arrestin recruitment technol.  Screening of a collection of 345,052 compds. yielded 2157 agonistic hits.  Only 10 of these compds. showed β-arrestin recruitment activity on a nonrelated receptor, indicating high accuracy and specificity of the assay.  The authors show that receptor activation with ref. agonists can be detected within the same EDG1 PathHunter cell line at the level of β-arrestin recruitment, Gi/o protein-mediated inhibition of cyclic adenosine monophosphate (cAMP), and activation of downstream phosphorylation of extracellular signal-regulated protein kinases.  The degree of β-arrestin recruitment was largely unaffected upon blockade of Gi/o protein signaling with pertussis toxin, whereas kinetic studies demonstrated a lower rate of β-arrestin-receptor assocn.  In contrast, inhibition of cAMP and phosphorylation of extracellular signal-regulated protein kinases were fully Gi/o protein regulated.  The data indicate that the β-arrestin enzyme fragment complementation cell line can be used not only for agonistic screening of GPCRs but also for the identification of "biased ligands" (i.e., compds. that differ in G-protein coupling and β-arrestin-mediated cellular effects).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKfdOoQXzl47Vg90H21EOLACvtfcHk0lgSGjDFfF7BOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXkslGrsbY%253D&md5=23c0c7106cbbd7337ca0945f13299b1d</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1177%2F1087057108326144&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057108326144%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2Bder%2BLee%26aufirst%3DM.%2BM.%2BC.%26aulast%3DBras%26aufirst%3DM.%26aulast%3Dvan%2BKoppen%26aufirst%3DC.%2BJ.%26aulast%3DZaman%26aufirst%3DG.%2BJ.%2BR.%26atitle%3Dbeta-Arrestin%2520recruitment%2520assay%2520for%2520the%2520identification%2520of%2520agonists%2520of%2520the%2520sphingosine%25201-phosphate%2520receptor%2520EDG1%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2008%26volume%3D13%26spage%3D986%26epage%3D998" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref243':['cit243'],'ref244':['cit244'],'ref245':['cit245'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref249':['cit249'],'ref250':['cit250'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref62':['cit62'],'ref58':['cit58'],'ref54':['cit54'],'ref55':['cit55'],'ref263':['cit263'],'ref264':['cit264'],'ref56':['cit56'],'ref57':['cit57'],'ref267':['cit267'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 24 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Yan Lu, Shanshan Qin, Bingjie Zhang, Antao Dai, Xiaoqing Cai, Mengna Ma, Zhan-Guo Gao, Dehua Yang, Raymond C. Stevens, Kenneth A. Jacobson, Ming-Wei Wang, <span class="NLM_string-name hlFld-ContribAuthor">Wenqing Shui</span>. </span><span class="cited-content_cbyCitation_article-title">Accelerating the Throughput of Affinity Mass Spectrometry-Based Ligand Screening toward a G Protein-Coupled Receptor. </span><span class="cited-content_cbyCitation_journal-name">Analytical Chemistry</span><span> <strong>2019,</strong> <em>91 </em>
                                    (13)
                                     , 8162-8169. <a href="https://doi.org/10.1021/acs.analchem.9b00477" title="DOI URL">https://doi.org/10.1021/acs.analchem.9b00477</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.analchem.9b00477&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.analchem.9b00477%26sid%3Dliteratum%253Aachs%26jtitle%3DAnalytical%2520Chemistry%26atitle%3DAccelerating%252Bthe%252BThroughput%252Bof%252BAffinity%252BMass%252BSpectrometry-Based%252BLigand%252BScreening%252Btoward%252Ba%252BG%252BProtein-Coupled%252BReceptor%26aulast%3DLu%26aufirst%3DYan%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D26012019%26date%3D16052019%26date%3D12062019%26date%3D16052019%26volume%3D91%26issue%3D13%26spage%3D8162%26epage%3D8169" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ali  Jazayeri</span>, <span class="hlFld-ContribAuthor ">Stephen P.  Andrews</span>, and <span class="hlFld-ContribAuthor ">Fiona H.  Marshall</span>  . </span><span class="cited-content_cbyCitation_article-title">Structurally Enabled Discovery of Adenosine A2A Receptor Antagonists. </span><span class="cited-content_cbyCitation_journal-name">Chemical Reviews</span><span> <strong>2017,</strong> <em>117 </em>
                                    (1)
                                     , 21-37. <a href="https://doi.org/10.1021/acs.chemrev.6b00119" title="DOI URL">https://doi.org/10.1021/acs.chemrev.6b00119</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrev.6b00119&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrev.6b00119%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Reviews%26atitle%3DStructurally%252BEnabled%252BDiscovery%252Bof%252BAdenosine%252BA2A%252BReceptor%252BAntagonists%26aulast%3DJazayeri%26aufirst%3DAli%26date%3D2017%26date%3D2016%26date%3D2016%26date%3D16022016%26date%3D22062016%26date%3D11012017%26volume%3D117%26issue%3D1%26spage%3D21%26epage%3D37" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Trayder  Thomas</span>, <span class="hlFld-ContribAuthor ">Kimberley C.  McLean</span>, <span class="hlFld-ContribAuthor ">Fiona M.  McRobb</span>, <span class="hlFld-ContribAuthor ">David T.  Manallack</span>, <span class="hlFld-ContribAuthor ">David K.  Chalmers</span>, and <span class="hlFld-ContribAuthor ">Elizabeth  Yuriev</span>  . </span><span class="cited-content_cbyCitation_article-title">Homology Modeling of Human Muscarinic Acetylcholine Receptors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2014,</strong> <em>54 </em>
                                    (1)
                                     , 243-253. <a href="https://doi.org/10.1021/ci400502u" title="DOI URL">https://doi.org/10.1021/ci400502u</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci400502u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci400502u%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DHomology%252BModeling%252Bof%252BHuman%252BMuscarinic%252BAcetylcholine%252BReceptors%26aulast%3DThomas%26aufirst%3DTrayder%26date%3D2014%26date%3D2013%26date%3D2013%26date%3D02092013%26date%3D23122013%26date%3D27012014%26date%3D15122013%26volume%3D54%26issue%3D1%26spage%3D243%26epage%3D253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stefanie  Kampen</span>, <span class="hlFld-ContribAuthor ">Duc  Duy Vo</span>, <span class="hlFld-ContribAuthor ">Xiaoqun  Zhang</span>, <span class="hlFld-ContribAuthor ">Nicolas  Panel</span>, <span class="hlFld-ContribAuthor ">Yunting  Yang</span>, <span class="hlFld-ContribAuthor ">Mariama  Jaiteh</span>, <span class="hlFld-ContribAuthor ">Pierre  Matricon</span>, <span class="hlFld-ContribAuthor ">Per  Svenningsson</span>, <span class="hlFld-ContribAuthor ">Jose  Brea</span>, <span class="hlFld-ContribAuthor ">Maria Isabel  Loza</span>, <span class="hlFld-ContribAuthor ">Jan  Kihlberg</span>, <span class="hlFld-ContribAuthor ">Jens  Carlsson</span>. </span><span class="cited-content_cbyCitation_article-title">Structure‐Guided Design of G‐Protein‐Coupled Receptor Polypharmacology. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie International Edition</span><span> <strong>2021,</strong> <em>60 </em>
                                    (33)
                                     , 18022-18030. <a href="https://doi.org/10.1002/anie.202101478" title="DOI URL">https://doi.org/10.1002/anie.202101478</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/anie.202101478&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fanie.202101478%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%2520International%2520Edition%26atitle%3DStructure%2525E2%252580%252590Guided%252BDesign%252Bof%252BG%2525E2%252580%252590Protein%2525E2%252580%252590Coupled%252BReceptor%252BPolypharmacology%26aulast%3DKampen%26aufirst%3DStefanie%26date%3D2021%26date%3D2021%26volume%3D60%26issue%3D33%26spage%3D18022%26epage%3D18030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stefanie  Kampen</span>, <span class="hlFld-ContribAuthor ">Duc  Duy Vo</span>, <span class="hlFld-ContribAuthor ">Xiaoqun  Zhang</span>, <span class="hlFld-ContribAuthor ">Nicolas  Panel</span>, <span class="hlFld-ContribAuthor ">Yunting  Yang</span>, <span class="hlFld-ContribAuthor ">Mariama  Jaiteh</span>, <span class="hlFld-ContribAuthor ">Pierre  Matricon</span>, <span class="hlFld-ContribAuthor ">Per  Svenningsson</span>, <span class="hlFld-ContribAuthor ">Jose  Brea</span>, <span class="hlFld-ContribAuthor ">Maria Isabel  Loza</span>, <span class="hlFld-ContribAuthor ">Jan  Kihlberg</span>, <span class="hlFld-ContribAuthor ">Jens  Carlsson</span>. </span><span class="cited-content_cbyCitation_article-title">Structure‐Guided Design of G‐Protein‐Coupled Receptor Polypharmacology. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie</span><span> <strong>2021,</strong> <em>133 </em>
                                    (33)
                                     , 18170-18178. <a href="https://doi.org/10.1002/ange.202101478" title="DOI URL">https://doi.org/10.1002/ange.202101478</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ange.202101478&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fange.202101478%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%26atitle%3DStructure%2525E2%252580%252590Guided%252BDesign%252Bof%252BG%2525E2%252580%252590Protein%2525E2%252580%252590Coupled%252BReceptor%252BPolypharmacology%26aulast%3DKampen%26aufirst%3DStefanie%26date%3D2021%26date%3D2021%26volume%3D133%26issue%3D33%26spage%3D18170%26epage%3D18178" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Babs  Briels</span>, <span class="hlFld-ContribAuthor ">Chris  Graaf</span>, <span class="hlFld-ContribAuthor ">Andreas  Bender</span>. </span><span class="cited-content_cbyCitation_article-title">Structural Chemogenomics. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 53-77. <a href="https://doi.org/10.1002/9781118681121.ch3" title="DOI URL">https://doi.org/10.1002/9781118681121.ch3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9781118681121.ch3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9781118681121.ch3%26sid%3Dliteratum%253Aachs%26atitle%3DStructural%252BChemogenomics%26aulast%3DBriels%26aufirst%3DBabs%26date%3D2020%26date%3D2020%26spage%3D53%26epage%3D77%26pub%3DWiley%26atitle%3DStructural%252BBiology%252Bin%252BDrug%252BDiscovery%26aulast%3DRenaud%26aufirst%3DJean%25E2%2580%2590Paul%26date%3D2020%26date%3D2020%26volume%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rita C.  Acúrcio</span>, <span class="hlFld-ContribAuthor ">Anna  Scomparin</span>, <span class="hlFld-ContribAuthor ">Ronit  Satchi‐Fainaro</span>, <span class="hlFld-ContribAuthor ">Helena F.  Florindo</span>, <span class="hlFld-ContribAuthor ">Rita C.  Guedes</span>. </span><span class="cited-content_cbyCitation_article-title">Computer‐aided drug design in new druggable targets for the next generation of immune‐oncology therapies. </span><span class="cited-content_cbyCitation_journal-name">Wiley Interdisciplinary Reviews: Computational Molecular Science</span><span> <strong>2019,</strong> <em>9 </em>
                                    (3)
                                     , e1397. <a href="https://doi.org/10.1002/wcms.1397" title="DOI URL">https://doi.org/10.1002/wcms.1397</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/wcms.1397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fwcms.1397%26sid%3Dliteratum%253Aachs%26jtitle%3DWiley%2520Interdisciplinary%2520Reviews%253A%2520Computational%2520Molecular%2520Science%26atitle%3DComputer%2525E2%252580%252590aided%252Bdrug%252Bdesign%252Bin%252Bnew%252Bdruggable%252Btargets%252Bfor%252Bthe%252Bnext%252Bgeneration%252Bof%252Bimmune%2525E2%252580%252590oncology%252Btherapies%26aulast%3DAc%25C3%25BArcio%26aufirst%3DRita%2BC.%26date%3D2019%26date%3D2018%26volume%3D9%26issue%3D3%26spage%3De1397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Adriano  Mollica</span>, <span class="hlFld-ContribAuthor ">Gokhan  Zengin</span>, <span class="hlFld-ContribAuthor ">Serdar  Durdagi</span>, <span class="hlFld-ContribAuthor ">Ramin  Ekhteiari Salmas</span>, <span class="hlFld-ContribAuthor ">Giorgia  Macedonio</span>, <span class="hlFld-ContribAuthor ">Azzurra  Stefanucci</span>, <span class="hlFld-ContribAuthor ">Marilisa Pia  Dimmito</span>, <span class="hlFld-ContribAuthor ">Ettore  Novellino</span>. </span><span class="cited-content_cbyCitation_article-title">Combinatorial peptide library screening for discovery of diverse α-glucosidase inhibitors using molecular dynamics simulations and binary QSAR models. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biomolecular Structure and Dynamics</span><span> <strong>2019,</strong> <em>37 </em>
                                    (3)
                                     , 726-740. <a href="https://doi.org/10.1080/07391102.2018.1439403" title="DOI URL">https://doi.org/10.1080/07391102.2018.1439403</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/07391102.2018.1439403&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F07391102.2018.1439403%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biomolecular%2520Structure%2520and%2520Dynamics%26atitle%3DCombinatorial%252Bpeptide%252Blibrary%252Bscreening%252Bfor%252Bdiscovery%252Bof%252Bdiverse%252B%2525CE%2525B1-glucosidase%252Binhibitors%252Busing%252Bmolecular%252Bdynamics%252Bsimulations%252Band%252Bbinary%252BQSAR%252Bmodels%26aulast%3DMollica%26aufirst%3DAdriano%26date%3D2019%26date%3D2018%26volume%3D37%26issue%3D3%26spage%3D726%26epage%3D740" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Márton  Vass</span>, <span class="hlFld-ContribAuthor ">Albert J.  Kooistra</span>, <span class="hlFld-ContribAuthor ">Stefan  Verhoeven</span>, <span class="hlFld-ContribAuthor ">David  Gloriam</span>, <span class="hlFld-ContribAuthor ">Iwan J. P.  de Esch</span>, <span class="hlFld-ContribAuthor ">Chris  de Graaf</span>. </span><span class="cited-content_cbyCitation_article-title">A Structural Framework for GPCR Chemogenomics: What’s In a Residue Number?. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2018,</strong>,, 73-113. <a href="https://doi.org/10.1007/978-1-4939-7465-8_4" title="DOI URL">https://doi.org/10.1007/978-1-4939-7465-8_4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-1-4939-7465-8_4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-1-4939-7465-8_4%26sid%3Dliteratum%253Aachs%26atitle%3DA%252BStructural%252BFramework%252Bfor%252BGPCR%252BChemogenomics%25253A%252BWhat%2525E2%252580%252599s%252BIn%252Ba%252BResidue%252BNumber%25253F%26aulast%3DVass%26aufirst%3DM%25C3%25A1rton%26date%3D2018%26date%3D2017%26spage%3D73%26epage%3D113%26pub%3DSpringer%2520New%2520York%26atitle%3DComputational%252BMethods%252Bfor%252BGPCR%252BDrug%252BDiscovery%26aulast%3DHeifetz%26aufirst%3DAlexander%26date%3D2018%26volume%3D1705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Damian  Bartuzi</span>, <span class="hlFld-ContribAuthor ">Agnieszka  Kaczor</span>, <span class="hlFld-ContribAuthor ">Katarzyna  Targowska-Duda</span>, <span class="hlFld-ContribAuthor ">Dariusz  Matosiuk</span>. </span><span class="cited-content_cbyCitation_article-title">Recent Advances and Applications of Molecular Docking to G Protein-Coupled Receptors. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2017,</strong> <em>22 </em>
                                    (2)
                                     , 340. <a href="https://doi.org/10.3390/molecules22020340" title="DOI URL">https://doi.org/10.3390/molecules22020340</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules22020340&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules22020340%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DRecent%252BAdvances%252Band%252BApplications%252Bof%252BMolecular%252BDocking%252Bto%252BG%252BProtein-Coupled%252BReceptors%26aulast%3DBartuzi%26aufirst%3DDamian%26date%3D2017%26date%3D2017%26volume%3D22%26issue%3D2%26spage%3D340" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">M.  Congreve</span>, <span class="hlFld-ContribAuthor ">A.  Bortolato</span>, <span class="hlFld-ContribAuthor ">G.  Brown</span>, <span class="hlFld-ContribAuthor ">R.M.  Cooke</span>. </span><span class="cited-content_cbyCitation_article-title">Modeling and Design for Membrane Protein Targets. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 145-188. <a href="https://doi.org/10.1016/B978-0-12-409547-2.12358-3" title="DOI URL">https://doi.org/10.1016/B978-0-12-409547-2.12358-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-409547-2.12358-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-409547-2.12358-3%26sid%3Dliteratum%253Aachs%26atitle%3DModeling%252Band%252BDesign%252Bfor%252BMembrane%252BProtein%252BTargets%26aulast%3DCongreve%26aufirst%3DM.%26date%3D2017%26spage%3D145%26epage%3D188%26pub%3DElsevier%26atitle%3DComprehensive%252BMedicinal%252BChemistry%252BIII%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eelke B.  Lenselink</span>, <span class="hlFld-ContribAuthor ">Thijs  Beuming</span>, <span class="hlFld-ContribAuthor ">Corine  van Veen</span>, <span class="hlFld-ContribAuthor ">Arnault  Massink</span>, <span class="hlFld-ContribAuthor ">Woody  Sherman</span>, <span class="hlFld-ContribAuthor ">Herman W. T.  van Vlijmen</span>, <span class="hlFld-ContribAuthor ">Adriaan P.  IJzerman</span>. </span><span class="cited-content_cbyCitation_article-title">In search of novel ligands using a structure-based approach: a case study on the adenosine A2A receptor. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2016,</strong> <em>30 </em>
                                    (10)
                                     , 863-874. <a href="https://doi.org/10.1007/s10822-016-9963-7" title="DOI URL">https://doi.org/10.1007/s10822-016-9963-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-016-9963-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-016-9963-7%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DIn%252Bsearch%252Bof%252Bnovel%252Bligands%252Busing%252Ba%252Bstructure-based%252Bapproach%25253A%252Ba%252Bcase%252Bstudy%252Bon%252Bthe%252Badenosine%252BA2A%252Breceptor%26aulast%3DLenselink%26aufirst%3DEelke%2BB.%26date%3D2016%26date%3D2016%26volume%3D30%26issue%3D10%26spage%3D863%26epage%3D874" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Albert J.  Kooistra</span>, <span class="hlFld-ContribAuthor ">Henry F.  Vischer</span>, <span class="hlFld-ContribAuthor ">Daniel  McNaught-Flores</span>, <span class="hlFld-ContribAuthor ">Rob  Leurs</span>, <span class="hlFld-ContribAuthor ">Iwan J. P.  de Esch</span>, <span class="hlFld-ContribAuthor ">Chris  de Graaf</span>. </span><span class="cited-content_cbyCitation_article-title">Function-specific virtual screening for GPCR ligands using a combined scoring method. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2016,</strong> <em>6 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/srep28288" title="DOI URL">https://doi.org/10.1038/srep28288</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/srep28288&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fsrep28288%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DFunction-specific%252Bvirtual%252Bscreening%252Bfor%252BGPCR%252Bligands%252Busing%252Ba%252Bcombined%252Bscoring%252Bmethod%26aulast%3DKooistra%26aufirst%3DAlbert%2BJ.%26date%3D2016%26date%3D2016%26volume%3D6%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Punita  Kumari</span>, <span class="hlFld-ContribAuthor ">Eshan  Ghosh</span>, <span class="hlFld-ContribAuthor ">Arun K.  Shukla</span>. </span><span class="cited-content_cbyCitation_article-title">Emerging Approaches to GPCR Ligand Screening for Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">Trends in Molecular Medicine</span><span> <strong>2015,</strong> <em>21 </em>
                                    (11)
                                     , 687-701. <a href="https://doi.org/10.1016/j.molmed.2015.09.002" title="DOI URL">https://doi.org/10.1016/j.molmed.2015.09.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.molmed.2015.09.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.molmed.2015.09.002%26sid%3Dliteratum%253Aachs%26jtitle%3DTrends%2520in%2520Molecular%2520Medicine%26atitle%3DEmerging%252BApproaches%252Bto%252BGPCR%252BLigand%252BScreening%252Bfor%252BDrug%252BDiscovery%26aulast%3DKumari%26aufirst%3DPunita%26date%3D2015%26volume%3D21%26issue%3D11%26spage%3D687%26epage%3D701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sid  Topiol</span>, <span class="hlFld-ContribAuthor ">Michael  Sabio</span>. </span><span class="cited-content_cbyCitation_article-title">The role of experimental and computational structural approaches in 7TM drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2015,</strong> <em>10 </em>
                                    (10)
                                     , 1071-1084. <a href="https://doi.org/10.1517/17460441.2015.1072166" title="DOI URL">https://doi.org/10.1517/17460441.2015.1072166</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1517/17460441.2015.1072166&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1517%2F17460441.2015.1072166%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DThe%252Brole%252Bof%252Bexperimental%252Band%252Bcomputational%252Bstructural%252Bapproaches%252Bin%252B7TM%252Bdrug%252Bdiscovery%26aulast%3DTopiol%26aufirst%3DSid%26date%3D2015%26date%3D2015%26volume%3D10%26issue%3D10%26spage%3D1071%26epage%3D1084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ashutosh  Kumar</span>, <span class="hlFld-ContribAuthor ">Kam Y.J.  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Hierarchical virtual screening approaches in small molecule drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Methods</span><span> <strong>2015,</strong> <em>71 </em>, 26-37. <a href="https://doi.org/10.1016/j.ymeth.2014.07.007" title="DOI URL">https://doi.org/10.1016/j.ymeth.2014.07.007</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ymeth.2014.07.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ymeth.2014.07.007%26sid%3Dliteratum%253Aachs%26jtitle%3DMethods%26atitle%3DHierarchical%252Bvirtual%252Bscreening%252Bapproaches%252Bin%252Bsmall%252Bmolecule%252Bdrug%252Bdiscovery%26aulast%3DKumar%26aufirst%3DAshutosh%26date%3D2015%26volume%3D71%26spage%3D26%26epage%3D37" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stephen P.  Andrews</span>, <span class="hlFld-ContribAuthor ">Giles A.  Brown</span>, <span class="hlFld-ContribAuthor ">John A.  Christopher</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based and Fragment-Based GPCR Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2014,</strong> <em>9 </em>
                                    (2)
                                     , 256-275. <a href="https://doi.org/10.1002/cmdc.201300382" title="DOI URL">https://doi.org/10.1002/cmdc.201300382</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201300382&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201300382%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DStructure-Based%252Band%252BFragment-Based%252BGPCR%252BDrug%252BDiscovery%26aulast%3DAndrews%26aufirst%3DStephen%2BP.%26date%3D2014%26date%3D2013%26volume%3D9%26issue%3D2%26spage%3D256%26epage%3D275" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Miles  Congreve</span>, <span class="hlFld-ContribAuthor ">João M.  Dias</span>, <span class="hlFld-ContribAuthor ">Fiona H.  Marshall</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based Drug Design for G Protein-Coupled Receptors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2014,</strong>,, 1-63. <a href="https://doi.org/10.1016/B978-0-444-63380-4.00001-9" title="DOI URL">https://doi.org/10.1016/B978-0-444-63380-4.00001-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-444-63380-4.00001-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-444-63380-4.00001-9%26sid%3Dliteratum%253Aachs%26atitle%3DStructure-Based%252BDrug%252BDesign%252Bfor%252BG%252BProtein-Coupled%252BReceptors%26aulast%3DCongreve%26aufirst%3DMiles%26date%3D2014%26spage%3D1%26epage%3D63%26pub%3DElsevier%26date%3D2014%26volume%3D53" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sid  Topiol</span>. </span><span class="cited-content_cbyCitation_article-title">X-ray structural information of GPCRs in drug design: what are the limitations and where do we go?. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2013,</strong> <em>8 </em>
                                    (6)
                                     , 607-620. <a href="https://doi.org/10.1517/17460441.2013.783815" title="DOI URL">https://doi.org/10.1517/17460441.2013.783815</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1517/17460441.2013.783815&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1517%2F17460441.2013.783815%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DX-ray%252Bstructural%252Binformation%252Bof%252BGPCRs%252Bin%252Bdrug%252Bdesign%25253A%252Bwhat%252Bare%252Bthe%252Blimitations%252Band%252Bwhere%252Bdo%252Bwe%252Bgo%25253F%26aulast%3DTopiol%26aufirst%3DSid%26date%3D2013%26date%3D2013%26volume%3D8%26issue%3D6%26spage%3D607%26epage%3D620" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">A J  Kooistra</span>, <span class="hlFld-ContribAuthor ">S  Kuhne</span>, <span class="hlFld-ContribAuthor ">I J P  de Esch</span>, <span class="hlFld-ContribAuthor ">R  Leurs</span>, <span class="hlFld-ContribAuthor ">C  de Graaf</span>. </span><span class="cited-content_cbyCitation_article-title">A structural chemogenomics analysis of aminergic GPCRs: lessons for histamine receptor ligand design. </span><span class="cited-content_cbyCitation_journal-name">British Journal of Pharmacology</span><span> <strong>2013,</strong> <em>170 </em>
                                    (1)
                                     , 101-126. <a href="https://doi.org/10.1111/bph.12248" title="DOI URL">https://doi.org/10.1111/bph.12248</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/bph.12248&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Fbph.12248%26sid%3Dliteratum%253Aachs%26jtitle%3DBritish%2520Journal%2520of%2520Pharmacology%26atitle%3DA%252Bstructural%252Bchemogenomics%252Banalysis%252Bof%252Baminergic%252BGPCRs%25253A%252Blessons%252Bfor%252Bhistamine%252Breceptor%252Bligand%252Bdesign%26aulast%3DKooistra%26aufirst%3DA%2BJ%26date%3D2013%26date%3D2013%26volume%3D170%26issue%3D1%26spage%3D101%26epage%3D126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ha P.  Nguyen</span>, <span class="hlFld-ContribAuthor ">Alexios  Koutsoukas</span>, <span class="hlFld-ContribAuthor ">Fazlin  Mohd Fauzi</span>, <span class="hlFld-ContribAuthor ">Georgios  Drakakis</span>, <span class="hlFld-ContribAuthor ">Mateusz  Maciejewski</span>, <span class="hlFld-ContribAuthor ">Robert C.  Glen</span>, <span class="hlFld-ContribAuthor ">Andreas  Bender</span>. </span><span class="cited-content_cbyCitation_article-title">Diversity Selection of Compounds Based on ‘Protein Affinity Fingerprints’ Improves Sampling of
              Bioactive
              Chemical Space. </span><span class="cited-content_cbyCitation_journal-name">Chemical Biology & Drug Design</span><span> <strong>2013,</strong> <em>82 </em>
                                    (3)
                                     , 252-266. <a href="https://doi.org/10.1111/cbdd.12155" title="DOI URL">https://doi.org/10.1111/cbdd.12155</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/cbdd.12155&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Fcbdd.12155%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Biology%2520%2526%2520Drug%2520Design%26atitle%3DDiversity%252BSelection%252Bof%252BCompounds%252BBased%252Bon%252B%2525E2%252580%252598Protein%252BAffinity%252BFingerprints%2525E2%252580%252599%252BImproves%252BSampling%252Bof%252BBioactive%252BChemical%252BSpace%26aulast%3DNguyen%26aufirst%3DHa%2BP.%26date%3D2013%26date%3D2013%26volume%3D82%26issue%3D3%26spage%3D252%26epage%3D266" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">David E.  Gloriam</span>. </span><span class="cited-content_cbyCitation_article-title">Chemogenomics of allosteric binding sites in GPCRs. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today: Technologies</span><span> <strong>2013,</strong> <em>10 </em>
                                    (2)
                                     , e307-e313. <a href="https://doi.org/10.1016/j.ddtec.2012.07.010" title="DOI URL">https://doi.org/10.1016/j.ddtec.2012.07.010</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ddtec.2012.07.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ddtec.2012.07.010%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technologies%26atitle%3DChemogenomics%252Bof%252Ballosteric%252Bbinding%252Bsites%252Bin%252BGPCRs%26aulast%3DGloriam%26aufirst%3DDavid%2BE.%26date%3D2013%26volume%3D10%26issue%3D2%26spage%3De307%26epage%3De313" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chris  de Graaf</span>, <span class="hlFld-ContribAuthor ">Henry F.  Vischer</span>, <span class="hlFld-ContribAuthor ">Gerdien E.  de Kloe</span>, <span class="hlFld-ContribAuthor ">Albert J.  Kooistra</span>, <span class="hlFld-ContribAuthor ">Saskia  Nijmeijer</span>, <span class="hlFld-ContribAuthor ">Martien  Kuijer</span>, <span class="hlFld-ContribAuthor ">Mark H.P.  Verheij</span>, <span class="hlFld-ContribAuthor ">Paul J.  England</span>, <span class="hlFld-ContribAuthor ">Jacqueline E.  van Muijlwijk-Koezen</span>, <span class="hlFld-ContribAuthor ">Rob  Leurs</span>, <span class="hlFld-ContribAuthor ">Iwan J.P.  de Esch</span>. </span><span class="cited-content_cbyCitation_article-title">Small and colorful stones make beautiful mosaics: fragment-based chemogenomics. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2013,</strong> <em>18 </em>
                                    (7-8)
                                     , 323-330. <a href="https://doi.org/10.1016/j.drudis.2012.12.003" title="DOI URL">https://doi.org/10.1016/j.drudis.2012.12.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2012.12.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2012.12.003%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DSmall%252Band%252Bcolorful%252Bstones%252Bmake%252Bbeautiful%252Bmosaics%25253A%252Bfragment-based%252Bchemogenomics%26aulast%3Dde%2BGraaf%26aufirst%3DChris%26date%3D2013%26volume%3D18%26issue%3D7-8%26spage%3D323%26epage%3D330" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Valère  Lounnas</span>, <span class="hlFld-ContribAuthor ">Tina  Ritschel</span>, <span class="hlFld-ContribAuthor ">Jan  Kelder</span>, <span class="hlFld-ContribAuthor ">Ross  McGuire</span>, <span class="hlFld-ContribAuthor ">Robert P.  Bywater</span>, <span class="hlFld-ContribAuthor ">Nicolas  Foloppe</span>. </span><span class="cited-content_cbyCitation_article-title">CURRENT PROGRESS IN STRUCTURE-BASED RATIONAL DRUG DESIGN MARKS A NEW MINDSET IN DRUG DISCOVERY. </span><span class="cited-content_cbyCitation_journal-name">Computational and Structural Biotechnology Journal</span><span> <strong>2013,</strong> <em>5 </em>
                                    (6)
                                     , e201302011. <a href="https://doi.org/10.5936/csbj.201302011" title="DOI URL">https://doi.org/10.5936/csbj.201302011</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.5936/csbj.201302011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.5936%2Fcsbj.201302011%26sid%3Dliteratum%253Aachs%26jtitle%3DComputational%2520and%2520Structural%2520Biotechnology%2520Journal%26atitle%3DCURRENT%252BPROGRESS%252BIN%252BSTRUCTURE-BASED%252BRATIONAL%252BDRUG%252BDESIGN%252BMARKS%252BA%252BNEW%252BMINDSET%252BIN%252BDRUG%252BDISCOVERY%26aulast%3DLounnas%26aufirst%3DVal%25C3%25A8re%26date%3D2013%26volume%3D5%26issue%3D6%26spage%3De201302011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/medium/jm-2012-00280e_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/large/jm-2012-00280e_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300280e&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/medium/jm-2012-00280e_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/large/jm-2012-00280e_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Virtual screening flowchart. Numbers indicate the number of compounds that pass the respective structure-based and ligand-based bioactivity models. Bold numbers indicate the number of actives included in the selections. Note that out of the 300 compounds selected for each of the targets, some overlap between the hit lists had to be removed and not all compounds were available as physical samples, leading to slight modifications of the hit lists suggested by the virtual screening methods.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/large/jm-2012-00280e_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300280e&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/medium/jm-2012-00280e_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/large/jm-2012-00280e_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Visualization of ligand similarities for the known actives on the adenosine A<sub>2A</sub> and the β-2 adrenergic receptor, represented by (left) stochastic proximity embedding and (right) distribution of Euclidean distances. While, on average, ADRB2 ligands (red) are more similar to each other, some atypical ligands can also be found that resemble compounds active on AA2AR (blue).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/large/jm-2012-00280e_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300280e&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/medium/jm-2012-00280e_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/large/jm-2012-00280e_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Important interacting residues for the different receptors with the pharmacophores and fitted compounds. AA2AR: related to acceptor features, N181<sup>5.42</sup> and N253<sup>6.55</sup>, T88<sup>3.36</sup> and S277<sup>7.42</sup>; related to donor features, N181<sup>5.42</sup> and N253<sup>6.55</sup>, T88<sup>3.36</sup> and S277<sup>7.42</sup>; hydrophobic features, V84<sup>3.32</sup> and L85<sup>3.33</sup>. ADRB2: acceptors, Y199<sup>5.38</sup> and S203<sup>5.42</sup> and S207<sup>5.46</sup>, N312<sup>7.39</sup>; donors, D113<sup>3.32</sup>, Y199<sup>5.38</sup>, and S203<sup>5.42</sup> and S207<sup>5.46</sup>; hydrophobic feature, V114<sup>3.33</sup>. S1PR1: acceptors, R120<sup>3.28</sup>, E121<sup>3.29</sup>; donor, R120<sup>3.28</sup>; hydrophobic features, Y202<sup>5.39</sup> and L276<sup>6.55</sup>, Y202<sup>5.39</sup> and L276<sup>6.55</sup>. For each compound, the highest scoring substructure that matched the compound is highlight in red.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/large/jm-2012-00280e_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300280e&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/medium/jm-2012-00280e_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/large/jm-2012-00280e_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Similarity between the structure based pharmacophore of S1PR1 and ADRB2 ligands, illustrated by compound <b>21</b> and carazolol (in the conformation as found in the crystal structure of ADRB2<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a>). Both compounds match the five-feature pharmacophore of S1PR1 and also fit in the ADRB2 receptor pharmacophore (not shown).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/large/jm-2012-00280e_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300280e&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/medium/jm-2012-00280e_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/large/jm-2012-00280e_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Receiver operating curves (ROC) for the ligand-based and structure-based virtual screening methods employed in this work. While the substructure-based method, on average, is able to achieve higher enrichment than the employed structure-based method, the extent of this difference is very much dependent on the target receptor considered, with AA2AR showing significantly higher retrieval of active compounds than ADRB2. Consensus scoring outperforms each individual method in the case of ADRB2.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/large/jm-2012-00280e_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300280e&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/medium/jm-2012-00280e_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/large/jm-2012-00280e_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Hit selection space based on the combined protein-based pharmacophore and ligand substructure virtual screens (3780 for AA2AR (blue), 810 for ADRB2 (green), and 1205 for S1PR1 (gray)). Most compounds are selected, since they are expected to have activity on only one of the three receptors. Some compounds are, however, expected to have activity on multiple receptors (front-upper, left-upper, right-upper, and front-lower corner), and a small set (front upper corner) is even expected to have activity on all three targets. Compounds are colored according to the receptor in which they receive the best rank.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/large/jm-2012-00280e_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300280e&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/medium/jm-2012-00280e_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/large/jm-2012-00280e_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Novel active compounds found using the respective models for the three receptors in this study (adenosine A<sub>2A</sub> receptor, β-2 adrenergic receptor, and sphingosine 1-phosphate receptor). The top of the plot shows the receptor for which the ligands were experimentally found to be active against, while the bottom of the plot shows the model that was used in silico to select each respective compound.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/large/jm-2012-00280e_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300280e&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/medium/jm-2012-00280e_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/large/jm-2012-00280e_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. (A) Compounds <b>1</b>–<b>3</b> selected from the AA2AR screen dose-dependently displacing [<sup>3</sup>H]ZM241385 from AA2AR. (B) Compounds <b>19</b>–<b>21</b> selected from the ADRB2 screen dose-dependently displacing [<sup>3</sup>H]dihydroalprenolol from ADRB2. (C) Compounds <b>25</b>–<b>27</b> selected from the S1PR1 screen dose-dependently inducing β-arrestin2 recruitment. PathHunter CHO-K1 S1PR1 β-arrestin EFC cells genetically engineered to increase β-galactosidase activity following the recruitment of β-arrestin2 to S1PR1 receptor were stimulated with increasing concentrations of <b>25</b>, <b>26</b>, <b>27</b> for 2 h before measurement of β-galactosidase activity. A single concentration of AUY954 (1 μM) was taken along to measure maximal β-arrestin2 recruitment.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-11/jm300280e/production/images/large/jm-2012-00280e_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300280e&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i42">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_53756" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_53756" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 77 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Kubinyi, H., Muller, G.</span>, Eds.  <span class="citation_source-book">Chemogenomics in Drug Discovery: A Medicinal Chemistry Perspective</span>; <span class="NLM_publisher-name">Wiley-VCH</span>: <span class="NLM_publisher-loc">Weinheim, Germany</span>,<span class="NLM_x"> </span><span class="NLM_year">2004</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1002%2F3527603948" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2004&author=H.+Kubinyi&author=G.+Muller&title=Chemogenomics+in+Drug+Discovery%3A+A+Medicinal+Chemistry+Perspective"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1002%2F3527603948&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F3527603948%26sid%3Dliteratum%253Aachs%26aulast%3DKubinyi%26aufirst%3DH.%26btitle%3DChemogenomics%2520in%2520Drug%2520Discovery%253A%2520A%2520Medicinal%2520Chemistry%2520Perspective%26pub%3DWiley-VCH%26date%3D2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Bredel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacoby, E.</span><span> </span><span class="NLM_article-title">Chemogenomics: an emerging strategy for rapid target and drug discovery</span> <span class="citation_source-journal">Nat. Rev. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">262</span><span class="NLM_x">–</span> <span class="NLM_lpage">275</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1038%2Fnrg1317" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=15131650" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjtVartL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2004&pages=262-275&author=M.+Bredelauthor=E.+Jacoby&title=Chemogenomics%3A+an+emerging+strategy+for+rapid+target+and+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Chemogenomics: an emerging strategy for rapid target and drug discovery</span></div><div class="casAuthors">Bredel, Markus; Jacoby, Edgar</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">262-275</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Chemogenomics is an emerging discipline that combines the latest tools of genomics and chem. and applies them to target and drug discovery.  Its strength lies in eliminating the bottleneck that currently occurs in target identification by measuring the broad, conditional effects of chem. libraries on whole biol. systems or by screening large chem. libraries quickly and efficiently against selected targets.  The hope is that chemogenomics will concurrently identify and validate therapeutic targets and detect drug candidates to rapidly and effectively generate new treatments for many human diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobwhXA51PclrVg90H21EOLACvtfcHk0lgoGwDGbvoNyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjtVartL0%253D&md5=ba7dbd1509189fcd2dd5e85a80a8e248</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrg1317&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg1317%26sid%3Dliteratum%253Aachs%26aulast%3DBredel%26aufirst%3DM.%26aulast%3DJacoby%26aufirst%3DE.%26atitle%3DChemogenomics%253A%2520an%2520emerging%2520strategy%2520for%2520rapid%2520target%2520and%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2004%26volume%3D5%26spage%3D262%26epage%3D275" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Harris, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevens, A. P.</span><span> </span><span class="NLM_article-title">Chemogenomics: structuring the drug discovery process to gene families</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">880</span><span class="NLM_x">–</span> <span class="NLM_lpage">888</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2006&pages=880-888&author=C.+J.+Harrisauthor=A.+P.+Stevens&title=Chemogenomics%3A+structuring+the+drug+discovery+process+to+gene+families"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHarris%26aufirst%3DC.%2BJ.%26aulast%3DStevens%26aufirst%3DA.%2BP.%26atitle%3DChemogenomics%253A%2520structuring%2520the%2520drug%2520discovery%2520process%2520to%2520gene%2520families%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2006%26volume%3D11%26spage%3D880%26epage%3D888" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Caron, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mullican, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mashal, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, K. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murcko, M. A.</span><span> </span><span class="NLM_article-title">Chemogenomic approaches to drug discovery</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">464</span><span class="NLM_x">–</span> <span class="NLM_lpage">470</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1016%2FS1367-5931%2800%2900229-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=11470611" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD3MXltlyqsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2001&pages=464-470&author=P.+R.+Caronauthor=M.+D.+Mullicanauthor=R.+D.+Mashalauthor=K.+P.+Wilsonauthor=M.+S.+Suauthor=M.+A.+Murcko&title=Chemogenomic+approaches+to+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Chemogenomic approaches to drug discovery</span></div><div class="casAuthors">Caron, P. R.; Mullican, M. D.; Mashal, R. D.; Wilson, K. P.; Su, M. S.; Murcko, M. A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">464-470</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review, with 39 refs., on the role of completed human genome sequence in developing novel therapeutics.  Topics discussed include efficiency of gene families; gene-family mapping; chemogenomics through mutagenesis and in the absence of structure; chemogenomic examples; key issues for chemogenomics; and bottlenecks in chemogenomics.  The chemogenomics approach, in which the completed human genome sequence is combined with protein structure and/or models to link to chem. inhibitors, is designed to fully use the sequence information by considering large families of gene targets at once.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSDlKQcY9zvrVg90H21EOLACvtfcHk0lgoGwDGbvoNyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXltlyqsb8%253D&md5=ea59ee2fcc9660c29b361537357e7e3f</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2FS1367-5931%2800%2900229-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1367-5931%252800%252900229-5%26sid%3Dliteratum%253Aachs%26aulast%3DCaron%26aufirst%3DP.%2BR.%26aulast%3DMullican%26aufirst%3DM.%2BD.%26aulast%3DMashal%26aufirst%3DR.%2BD.%26aulast%3DWilson%26aufirst%3DK.%2BP.%26aulast%3DSu%26aufirst%3DM.%2BS.%26aulast%3DMurcko%26aufirst%3DM.%2BA.%26atitle%3DChemogenomic%2520approaches%2520to%2520drug%2520discovery%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2001%26volume%3D5%26spage%3D464%26epage%3D470" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">Chemogenomic approaches to rational drug design</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">152</span><span class="NLM_x">, </span> <span class="NLM_fpage">38</span><span class="NLM_x">–</span> <span class="NLM_lpage">52</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1038%2Fsj.bjp.0707307" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=17533416" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpsFOjurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=152&publication_year=2007&pages=38-52&author=D.+Rognan&title=Chemogenomic+approaches+to+rational+drug+design"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Chemogenomic approaches to rational drug design</span></div><div class="casAuthors">Rognan, D.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">152</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">38-52</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Paradigms in drug design and discovery are changing at a significant pace.  Concomitant to the sequencing of over 180 several genomes, the high-throughput miniaturization of chem. synthesis and biol. evaluation of a multiple compds. on gene/protein expression and function opens the way to global drug-discovery approaches, no more focused on a single target but on an entire family of related proteins or on a full metabolic pathway.  Chemogenomics is this emerging research field aimed at systematically studying the biol. effect of a wide array of small mol.-wt. ligands on a wide array of macromol. targets.  Since the quantity of existing data (compds., targets and assays) and of produced information (gene/protein expression levels and binding consts.) are too large for manual manipulation, information technologies play a crucial role in planning, analyzing and predicting chemogenomic data.  The present review will focus on predictive in silico chemogenomic approaches to foster rational drug design and derive information from the simultaneous biol. evaluation of multiple compds. on multiple targets.  State-of-the-art methods for navigating in either ligand or target space will be presented and concrete drug design applications will be mentioned.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPERITBubs0LVg90H21EOLACvtfcHk0lh_0xJ3P1kwdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpsFOjurw%253D&md5=47b3b2a5155a079b486e56d31e0ca59c</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjp.0707307&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjp.0707307%26sid%3Dliteratum%253Aachs%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DChemogenomic%2520approaches%2520to%2520rational%2520drug%2520design%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2007%26volume%3D152%26spage%3D38%26epage%3D52" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">Structure-based approaches to target fishing and ligand profiling</span> <span class="citation_source-journal">Mol. Inf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">176</span><span class="NLM_x">–</span> <span class="NLM_lpage">187</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1002%2Fminf.200900081" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=27462761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjs12ntLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2010&pages=176-187&author=D.+Rognan&title=Structure-based+approaches+to+target+fishing+and+ligand+profiling"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Approaches to Target Fishing and Ligand Profiling</span></div><div class="casAuthors">Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Informatics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">176-187</span>CODEN:
                <span class="NLM_cas:coden">MIONBS</span>;
        ISSN:<span class="NLM_cas:issn">1868-1743</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Chemogenomics is an emerging interdisciplinary field aiming at identifying all possible ligands of all possible targets.  If one groups targets in columns and ligands in rows, chemogenomic approaches to drug discovery just fill the interaction matrix.  Since exptl. data do not suffice, several computational methods are currently actively developed to supplement time-consuming and costly expts.  They are either designed to fill rows and thus profile a ligand towards a heterogeneous set of targets (target profiling) or to fill columns and thus identify novel ligands for an existing target (std. virtual screening).  At the interface of both strategies are now true chemogenomic computational methods filling well defined areas in the matrix.  The present review will focus on (protein) structure-based approaches and illustrates major advances in this novel exciting field which is supposed to massively impact rational drug design in the next decade.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzdpIh1yWS_7Vg90H21EOLACvtfcHk0lh_0xJ3P1kwdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjs12ntLo%253D&md5=e32b663e9a43bb1621d07d07277fa6b5</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1002%2Fminf.200900081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fminf.200900081%26sid%3Dliteratum%253Aachs%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DStructure-based%2520approaches%2520to%2520target%2520fishing%2520and%2520ligand%2520profiling%26jtitle%3DMol.%2520Inf.%26date%3D2010%26volume%3D29%26spage%3D176%26epage%3D187" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Keiser, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Setola, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laggner, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abbas, A. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hufeisen, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jensen, N. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuijer, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matos, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tran, T. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whaley, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glennon, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hert, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, D. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span> </span><span class="NLM_article-title">Predicting new molecular targets for known drugs</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">462</span><span class="NLM_x">, </span> <span class="NLM_fpage">175</span><span class="NLM_x">–</span> <span class="NLM_lpage">181</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1038%2Fnature08506" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=19881490" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlers7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=462&publication_year=2009&pages=175-181&author=M.+J.+Keiserauthor=V.+Setolaauthor=J.+J.+Irwinauthor=C.+Laggnerauthor=A.+I.+Abbasauthor=S.+J.+Hufeisenauthor=N.+H.+Jensenauthor=M.+B.+Kuijerauthor=R.+C.+Matosauthor=T.+B.+Tranauthor=R.+Whaleyauthor=R.+A.+Glennonauthor=J.+Hertauthor=K.+L.+Thomasauthor=D.+D.+Edwardsauthor=B.+K.+Shoichetauthor=B.+L.+Roth&title=Predicting+new+molecular+targets+for+known+drugs"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting new molecular targets for known drugs</span></div><div class="casAuthors">Keiser, Michael J.; Setola, Vincent; Irwin, John J.; Laggner, Christian; Abbas, Atheir I.; Hufeisen, Sandra J.; Jensen, Niels H.; Kuijer, Michael B.; Matos, Roberto C.; Tran, Thuy B.; Whaley, Ryan; Glennon, Richard A.; Hert, Jerome; Thomas, Kelan L. H.; Edwards, Douglas D.; Shoichet, Brian K.; Roth, Bryan L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">462</span>
        (<span class="NLM_cas:issue">7270</span>),
    <span class="NLM_cas:pages">175-181</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Although drugs are intended to be selective, at least some bind to several physiol. targets, explaining side effects and efficacy.  Because many drug-target combinations exist, it would be useful to explore possible interactions computationally.  Here we compared 3665 US Food and Drug Administration (FDA)-approved and investigational drugs against hundreds of targets, defining each target by its ligands.  Chem. similarities between drugs and ligand sets predicted thousands of unanticipated assocns.  Thirty were tested exptl., including the antagonism of the β1 receptor by the transporter inhibitor Prozac, the inhibition of the 5-hydroxytryptamine (5-HT) transporter by the ion channel drug Vadilex, and antagonism of the histamine H4 receptor by the enzyme inhibitor Rescriptor.  Overall, 23 new drug-target assocns. were confirmed, five of which were potent (<100 nM).  The physiol. relevance of one, the drug N,N-dimethyltryptamine (DMT) on serotonergic receptors, was confirmed in a knockout mouse.  The chem. similarity approach is systematic and comprehensive, and may suggest side-effects and new indications for many drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozd-8cCXUskrVg90H21EOLACvtfcHk0lh_0xJ3P1kwdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlers7vO&md5=974e65f5d5b4dabde1a463c429215546</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnature08506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08506%26sid%3Dliteratum%253Aachs%26aulast%3DKeiser%26aufirst%3DM.%2BJ.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DLaggner%26aufirst%3DC.%26aulast%3DAbbas%26aufirst%3DA.%2BI.%26aulast%3DHufeisen%26aufirst%3DS.%2BJ.%26aulast%3DJensen%26aufirst%3DN.%2BH.%26aulast%3DKuijer%26aufirst%3DM.%2BB.%26aulast%3DMatos%26aufirst%3DR.%2BC.%26aulast%3DTran%26aufirst%3DT.%2BB.%26aulast%3DWhaley%26aufirst%3DR.%26aulast%3DGlennon%26aufirst%3DR.%2BA.%26aulast%3DHert%26aufirst%3DJ.%26aulast%3DThomas%26aufirst%3DK.%2BL.%26aulast%3DEdwards%26aufirst%3DD.%2BD.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26atitle%3DPredicting%2520new%2520molecular%2520targets%2520for%2520known%2520drugs%26jtitle%3DNature%26date%3D2009%26volume%3D462%26spage%3D175%26epage%3D181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Badrinarayan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sastry, G. N.</span><span> </span><span class="NLM_article-title">Virtual high-throughput screening in new lead identification</span> <span class="citation_source-journal">Comb. Chem. High Throughput Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">840</span><span class="NLM_x">–</span> <span class="NLM_lpage">860</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.2174%2F138620711797537102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=21843146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFOhsLrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2011&pages=840-860&author=P.+Badrinarayanauthor=G.+N.+Sastry&title=Virtual+high-throughput+screening+in+new+lead+identification"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual high throughput screening in new lead identification</span></div><div class="casAuthors">Badrinarayan, Preethi; Sastry, G. Narahari</div><div class="citationInfo"><span class="NLM_cas:title">Combinatorial Chemistry & High Throughput Screening</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">840-860</span>CODEN:
                <span class="NLM_cas:coden">CCHSFU</span>;
        ISSN:<span class="NLM_cas:issn">1386-2073</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Drug discovery continues to be one of the greatest contemporary challenges and rational application of modeling approaches is the first important step to obtain lead compds., which can be optimized further.  Virtual high throughput screening (VHTS) is one of the efficient approaches to obtain lead structures for a given target.  Strategic application of different screening filters like pharmacophore mapping, shape-based, ligand-based, mol. similarity etc., in combination with other drug design protocols provide invaluable insights in lead identification and optimization.  Screening of large databases using these computational methods provides potential lead compds., thus triggering a meaningful interplay between computations and expts.  In this review, we present a crit. account on the relevance of mol. modeling approaches in general, lead optimization and virtual screening methods in particular for new lead identification.  The importance of developing reliable scoring functions for non-bonded interactions has been highlighted, as it is an extremely important measure for the reliability of scoring function.  The lead optimization and new lead design has also been illustrated with examples.  The importance of employing a combination of general and target specific screening protocols has also been highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPpEYhrdA6gbVg90H21EOLACvtfcHk0limMPLcb-W-FA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFOhsLrP&md5=a4f42fd1ae6ec14951648ee6ad6a588d</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.2174%2F138620711797537102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138620711797537102%26sid%3Dliteratum%253Aachs%26aulast%3DBadrinarayan%26aufirst%3DP.%26aulast%3DSastry%26aufirst%3DG.%2BN.%26atitle%3DVirtual%2520high-throughput%2520screening%2520in%2520new%2520lead%2520identification%26jtitle%3DComb.%2520Chem.%2520High%2520Throughput%2520Screening%26date%3D2011%26volume%3D14%26spage%3D840%26epage%3D860" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Seifert, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kraus, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kramer, B.</span><span> </span><span class="NLM_article-title">Virtual high-throughput screening of molecular databases</span> <span class="citation_source-journal">Curr. Opin. Drug Discovery Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">298</span><span class="NLM_x">–</span> <span class="NLM_lpage">307</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1007%2F978-1-59745-455-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=17554856" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmtVWlsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2007&pages=298-307&author=M.+H.+Seifertauthor=J.+Krausauthor=B.+Kramer&title=Virtual+high-throughput+screening+of+molecular+databases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual high-throughput screening of molecular databases</span></div><div class="casAuthors">Seifert, Markus H. J.; Kraus, Juergen; Kramer, Bernd</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Drug Discovery & Development</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">298-307</span>CODEN:
                <span class="NLM_cas:coden">CODDFF</span>;
        ISSN:<span class="NLM_cas:issn">1367-6733</span>.
    
            (<span class="NLM_cas:orgname">Thomson Scientific</span>)
        </div><div class="casAbstract">A review.  Virtual high-throughput screening (vHTS) is an efficient and widely applicable method used to identify initial hit compds. for pharmaceutical research.  Despite its widespread use, several aspects of protein structure-based vHTS can still be optimized, particularly its accuracy and speed in generating results.  Recent developments that address these issues include machine learning and implicit solvation methods.  Various machine learning methods are available to improve vHTS accuracy, for example, target-specific optimization of scoring functions, the integration of essential protein-ligand interactions, and the application of neg. training data.  Implicit solvation methods are exemplified by the mol. mechanics Poisson-Boltzmann solvent accessible surface area approach.  Furthermore, grid computing and intelligent database screening approaches are used to improve the speed of vHTS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRO6rcUw_QI7Vg90H21EOLACvtfcHk0lgsT643LLeasw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmtVWlsrc%253D&md5=7e4efb79644d937514d93558f54cd575</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1007%2F978-1-59745-455-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-59745-455-1%26sid%3Dliteratum%253Aachs%26aulast%3DSeifert%26aufirst%3DM.%2BH.%26aulast%3DKraus%26aufirst%3DJ.%26aulast%3DKramer%26aufirst%3DB.%26atitle%3DVirtual%2520high-throughput%2520screening%2520of%2520molecular%2520databases%26jtitle%3DCurr.%2520Opin.%2520Drug%2520Discovery%2520Dev.%26date%3D2007%26volume%3D10%26spage%3D298%26epage%3D307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Reddy, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pati, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pradeep, H. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sastry, G. N.</span><span> </span><span class="NLM_article-title">Virtual screening in drug discovery—a computational perspective</span> <span class="citation_source-journal">Curr. Protein Pept. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">329</span><span class="NLM_x">–</span> <span class="NLM_lpage">351</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.2174%2F138920307781369427" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=17696867" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpslKrs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=329-351&author=A.+S.+Reddyauthor=S.+P.+Patiauthor=P.+P.+Kumarauthor=H.+N.+Pradeepauthor=G.+N.+Sastry&title=Virtual+screening+in+drug+discovery%E2%80%94a+computational+perspective"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual screening in drug discovery - a computational perspective</span></div><div class="casAuthors">Reddy, A. Srinivas; Pati, S. Priyadarshini; Kumar, P. Praveen; Pradeep, H. N.; Sastry, G. Narahari</div><div class="citationInfo"><span class="NLM_cas:title">Current Protein and Peptide Science</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">329-351</span>CODEN:
                <span class="NLM_cas:coden">CPPSCM</span>;
        ISSN:<span class="NLM_cas:issn">1389-2037</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Virtual screening emerged as an important tool in our quest to access novel drug like compds.  There are a wide range of comparable and contrasting methodol. protocols available in screening databases for the lead compds.  The no. of methods and software packages which employ the target and ligand based virtual screening are increasing at a rapid pace.  However, the general understanding on the applicability and limitations of these methodologies is not emerging as fast as the developments of various methods.  Therefore, it is extremely important to compare and contrast various protocols with practical examples to gauge the strength and applicability of various methods.  The review provides a comprehensive appraisal on several of the available virtual screening methods to-date.  Recent developments of the docking and similarity based methods have been discussed besides the descriptor selection and pharmacophore based searching.  The review touches upon the application of statistical, graph theory based methods machine learning tools in virtual screening and combinatorial library design.  Finally, several case studies are undertaken where the virtual screening technol. has been applied successfully.  A crit. anal. of these case studies provides a good platform to est. the applicability of various virtual screening methods in the new lead identification and optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUR2m_0xSzaLVg90H21EOLACvtfcHk0lgsT643LLeasw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpslKrs70%253D&md5=ad933e5e65e213edc9c37e9c973c696e</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.2174%2F138920307781369427&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138920307781369427%26sid%3Dliteratum%253Aachs%26aulast%3DReddy%26aufirst%3DA.%2BS.%26aulast%3DPati%26aufirst%3DS.%2BP.%26aulast%3DKumar%26aufirst%3DP.%2BP.%26aulast%3DPradeep%26aufirst%3DH.%2BN.%26aulast%3DSastry%26aufirst%3DG.%2BN.%26atitle%3DVirtual%2520screening%2520in%2520drug%2520discovery%25E2%2580%2594a%2520computational%2520perspective%26jtitle%3DCurr.%2520Protein%2520Pept.%2520Sci.%26date%3D2007%26volume%3D8%26spage%3D329%26epage%3D351" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Brianso, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carrascosa, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oprea, T. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mestres, J.</span><span> </span><span class="NLM_article-title">Cross-pharmacology analysis of G protein-coupled receptors</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1956</span><span class="NLM_x">–</span> <span class="NLM_lpage">1963</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.2174%2F156802611796391285" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=21851335" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVSmsLzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2011&pages=1956-1963&author=F.+Briansoauthor=M.+C.+Carrascosaauthor=T.+I.+Opreaauthor=J.+Mestres&title=Cross-pharmacology+analysis+of+G+protein-coupled+receptors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Cross-pharmacology analysis of G protein-coupled receptors</span></div><div class="casAuthors">Brianso, Ferran; Carrascosa, Maria C.; Oprea, Tudor I.; Mestres, Jordi</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">1956-1963</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The degree of applicability of chemogenomic approaches to protein families depends on the accuracy and completeness of pharmacol. data and the corresponding level of pharmacol. similarity obsd. among their protein members.  The recent public domain availability of pharmacol. data for thousands of small mols. on 204 G protein-coupled receptors (GPCRs) provides a firm basis for an in-depth cross-pharmacol. anal. of this superfamily.  The no. of protein targets included in the cross-pharmacol. profile of the different GPCRs changes significantly upon varying the ligand similarity and binding affinity criteria.  However, with the exception of muscarinic receptors, aminergic GPCRs distinguish themselves from the rest of the members in the family by their remarkably high levels of pharmacol. similarity among them.  Clusters of non-GPCR targets related by cross-pharmacol. with particular GPCRs are identified and the implications for unwanted side-effects, as well as for repurposing opportunities, discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcxMaSMU_YxLVg90H21EOLACvtfcHk0ljmDycojBuf1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVSmsLzO&md5=7805626c0844e84d16b56ed953823486</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.2174%2F156802611796391285&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802611796391285%26sid%3Dliteratum%253Aachs%26aulast%3DBrianso%26aufirst%3DF.%26aulast%3DCarrascosa%26aufirst%3DM.%2BC.%26aulast%3DOprea%26aufirst%3DT.%2BI.%26aulast%3DMestres%26aufirst%3DJ.%26atitle%3DCross-pharmacology%2520analysis%2520of%2520G%2520protein-coupled%2520receptors%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2011%26volume%3D11%26spage%3D1956%26epage%3D1963" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Mestres, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gregori-Puigjane, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valverde, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sole, R. V.</span><span> </span><span class="NLM_article-title">Data completeness—the Achilles heel of drug–target networks</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">983</span><span class="NLM_x">–</span> <span class="NLM_lpage">984</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=983-984&author=J.+Mestresauthor=E.+Gregori-Puigjaneauthor=S.+Valverdeauthor=R.+V.+Sole&title=Data+completeness%E2%80%94the+Achilles+heel+of+drug%E2%80%93target+networks"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMestres%26aufirst%3DJ.%26aulast%3DGregori-Puigjane%26aufirst%3DE.%26aulast%3DValverde%26aufirst%3DS.%26aulast%3DSole%26aufirst%3DR.%2BV.%26atitle%3DData%2520completeness%25E2%2580%2594the%2520Achilles%2520heel%2520of%2520drug%25E2%2580%2593target%2520networks%26jtitle%3DNat.%2520Biotechnol.%26date%3D2008%26volume%3D26%26spage%3D983%26epage%3D984" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Gaulton, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellis, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bento, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chambers, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hersey, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Light, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGlinchey, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michalovich, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Lazikani, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Overington, J. P.</span><span> </span><span class="NLM_article-title">ChEMBL: a large-scale bioactivity database for drug discovery</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">D1100</span><span class="NLM_x">–</span> <span class="NLM_lpage">D1107</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1093%2Fnar%2Fgkr777" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=21948594" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs12htbjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=D1100-D1107&author=A.+Gaultonauthor=L.+J.+Bellisauthor=A.+P.+Bentoauthor=J.+Chambersauthor=M.+Daviesauthor=A.+Herseyauthor=Y.+Lightauthor=S.+McGlincheyauthor=D.+Michalovichauthor=B.+Al-Lazikaniauthor=J.+P.+Overington&title=ChEMBL%3A+a+large-scale+bioactivity+database+for+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">ChEMBL: a large-scale bioactivity database for drug discovery</span></div><div class="casAuthors">Gaulton, Anna; Bellis, Louisa J.; Bento, A. Patricia; Chambers, Jon; Davies, Mark; Hersey, Anne; Light, Yvonne; McGlinchey, Shaun; Michalovich, David; Al-Lazikani, Bissan; Overington, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1100-D1107</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ChEMBL is an Open Data database contg. binding, functional and ADMET information for a large no. of drug-like bioactive compds.  These data are manually abstracted from the primary published literature on a regular basis, then further curated and standardized to maximize their quality and utility across a wide range of chem. biol. and drug-discovery research problems.  Currently, the database contains 5.4 million bioactivity measurements for more than 1 million compds. and 5200 protein targets.  Access is available through a web-based interface, data downloads and web services at: https://www.ebi.ac.uk/chembldb.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9e9jRENTtubVg90H21EOLACvtfcHk0ljmDycojBuf1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs12htbjN&md5=aedf7793e1ca54b6a4fa272ea3ef7d0e</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr777&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr777%26sid%3Dliteratum%253Aachs%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DBellis%26aufirst%3DL.%2BJ.%26aulast%3DBento%26aufirst%3DA.%2BP.%26aulast%3DChambers%26aufirst%3DJ.%26aulast%3DDavies%26aufirst%3DM.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DLight%26aufirst%3DY.%26aulast%3DMcGlinchey%26aufirst%3DS.%26aulast%3DMichalovich%26aufirst%3DD.%26aulast%3DAl-Lazikani%26aufirst%3DB.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DChEMBL%253A%2520a%2520large-scale%2520bioactivity%2520database%2520for%2520drug%2520discovery%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3DD1100%26epage%3DD1107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Knox, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Law, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jewison, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ly, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frolkis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pon, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banco, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mak, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neveu, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Djoumbou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eisner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wishart, D. S.</span><span> </span><span class="NLM_article-title">DrugBank 3.0: a comprehensive resource for “omics” research on drugs</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">D1035</span><span class="NLM_x">–</span> <span class="NLM_lpage">D1041</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2011&pages=D1035-D1041&author=C.+Knoxauthor=V.+Lawauthor=T.+Jewisonauthor=P.+Liuauthor=S.+Lyauthor=A.+Frolkisauthor=A.+Ponauthor=K.+Bancoauthor=C.+Makauthor=V.+Neveuauthor=Y.+Djoumbouauthor=R.+Eisnerauthor=A.+C.+Guoauthor=D.+S.+Wishart&title=DrugBank+3.0%3A+a+comprehensive+resource+for+%E2%80%9Comics%E2%80%9D+research+on+drugs"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKnox%26aufirst%3DC.%26aulast%3DLaw%26aufirst%3DV.%26aulast%3DJewison%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DLy%26aufirst%3DS.%26aulast%3DFrolkis%26aufirst%3DA.%26aulast%3DPon%26aufirst%3DA.%26aulast%3DBanco%26aufirst%3DK.%26aulast%3DMak%26aufirst%3DC.%26aulast%3DNeveu%26aufirst%3DV.%26aulast%3DDjoumbou%26aufirst%3DY.%26aulast%3DEisner%26aufirst%3DR.%26aulast%3DGuo%26aufirst%3DA.%2BC.%26aulast%3DWishart%26aufirst%3DD.%2BS.%26atitle%3DDrugBank%25203.0%253A%2520a%2520comprehensive%2520resource%2520for%2520%25E2%2580%259Comics%25E2%2580%259D%2520research%2520on%2520drugs%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2011%26volume%3D39%26spage%3DD1035%26epage%3DD1041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilson, M. K.</span><span> </span><span class="NLM_article-title">The binding database: overview and user’s guide</span> <span class="citation_source-journal">Biopolymers</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x">, </span> <span class="NLM_fpage">127</span><span class="NLM_x">–</span> <span class="NLM_lpage">141</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2001&pages=127-141&author=X.+Chenauthor=Y.+Linauthor=M.+K.+Gilson&title=The+binding+database%3A+overview+and+user%E2%80%99s+guide"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DGilson%26aufirst%3DM.%2BK.%26atitle%3DThe%2520binding%2520database%253A%2520overview%2520and%2520user%25E2%2580%2599s%2520guide%26jtitle%3DBiopolymers%26date%3D2001%26volume%3D61%26spage%3D127%26epage%3D141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilson, M. K.</span><span> </span><span class="NLM_article-title">The binding database: data management and interface design</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">130</span><span class="NLM_x">–</span> <span class="NLM_lpage">139</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2002&pages=130-139&author=X.+Chenauthor=Y.+Linauthor=M.+Liuauthor=M.+K.+Gilson&title=The+binding+database%3A+data+management+and+interface+design"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DGilson%26aufirst%3DM.%2BK.%26atitle%3DThe%2520binding%2520database%253A%2520data%2520management%2520and%2520interface%2520design%26jtitle%3DBioinformatics%26date%3D2002%26volume%3D18%26spage%3D130%26epage%3D139" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilson, M. K.</span><span> </span><span class="NLM_article-title">BindingDB: a Web-accessible molecular recognition database</span> <span class="citation_source-journal">Comb. Chem. High Throughput Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">719</span><span class="NLM_x">–</span> <span class="NLM_lpage">725</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.2174%2F1386207013330670" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=11812264" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD38Xks1Orsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2001&pages=719-725&author=X.+Chenauthor=M.+Liuauthor=M.+K.+Gilson&title=BindingDB%3A+a+Web-accessible+molecular+recognition+database"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">BindingDB: A Web-accessible molecular recognition database</span></div><div class="casAuthors">Chen, Xi; Liu, Ming; Gilson, Michael K.</div><div class="citationInfo"><span class="NLM_cas:title">Combinatorial Chemistry and High Throughput Screening</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">719-725</span>CODEN:
                <span class="NLM_cas:coden">CCHSFU</span>;
        ISSN:<span class="NLM_cas:issn">1386-2073</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers</span>)
        </div><div class="casAbstract">The BindingDB, a public web-accessible database of measured binding affinities for various mol. types, is described.  It allows queries based upon a range of criteria, including chem. similarity or substructure, sequence homol., numerical criteria and reactant names.  Principles of Human-Computer Interactions are being employed in creating the query interface and user-feedback is being solicited.  The data specification includes significant exptl. detail.  A full dictionary has been created for isothermal titrn. calorimetry data in consultation with experimentalists and data dictionaries for enzyme inhibition and other measurement techniques are being developed.  Currently, the BindingDB contains several data sets of broad interest, such as antigen-antibody binding and cyclodextrin/small-mol. binding.  However, it is anticipated that online deposition by experimentalists will ultimately contribute a larger flow of data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzRKVfJZtKwrVg90H21EOLACvtfcHk0lihjsa924GA7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xks1Orsg%253D%253D&md5=9db61dc1ad10e8191ee11afcfa9e337e</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.2174%2F1386207013330670&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1386207013330670%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DGilson%26aufirst%3DM.%2BK.%26atitle%3DBindingDB%253A%2520a%2520Web-accessible%2520molecular%2520recognition%2520database%26jtitle%3DComb.%2520Chem.%2520High%2520Throughput%2520Screening%26date%3D2001%26volume%3D4%26spage%3D719%26epage%3D725" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Wang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span> </span><span class="NLM_article-title">The PDBbind database: collection of binding affinities for protein–ligand complexes with known three-dimensional structures</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">2977</span><span class="NLM_x">–</span> <span class="NLM_lpage">2980</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm030580l" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=2977-2980&author=R.+Wangauthor=X.+Fangauthor=Y.+Luauthor=S.+Wang&title=The+PDBbind+database%3A+collection+of+binding+affinities+for+protein%E2%80%93ligand+complexes+with+known+three-dimensional+structures"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm030580l&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030580l%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DR.%26aulast%3DFang%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DThe%2520PDBbind%2520database%253A%2520collection%2520of%2520binding%2520affinities%2520for%2520protein%25E2%2580%2593ligand%2520complexes%2520with%2520known%2520three-dimensional%2520structures%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D2977%26epage%3D2980" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Wang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, C. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span> </span><span class="NLM_article-title">The PDBbind database: methodologies and updates</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">4111</span><span class="NLM_x">–</span> <span class="NLM_lpage">4119</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm048957q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD2MXktlKisbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=4111-4119&author=R.+Wangauthor=X.+Fangauthor=Y.+Luauthor=C.+Y.+Yangauthor=S.+Wang&title=The+PDBbind+database%3A+methodologies+and+updates"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">The PDBbind Database: Methodologies and Updates</span></div><div class="casAuthors">Wang, Renxiao; Fang, Xueliang; Lu, Yipin; Yang, Chao-Yie; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4111-4119</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors have developed the PDBbind database to provide a comprehensive collection of binding affinities for the protein-ligand complexes in the Protein Data Bank (PDB).  This paper gives a full description of the latest version, i.e., version 2003, which is an update to our recently reported work.  Out of 23 790 entries in the PDB release No.107 (Jan. 2004), 5897 entries were identified as protein-ligand complexes that meet our definition.  Exptl. detd. binding affinities (Kd, Ki, and IC50) for 1622 of these were retrieved from the refs. assocd. with these complexes.  A total of 900 complexes were selected to form a "refined set", which is of particular value as a std. data set for docking and scoring studies.  All of the final data, including binding affinity data, ref. citations, and processed structural files, have been incorporated into the PDBbind database accessible online at http:// www.pdbbind.org/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD3MSeASmZZrVg90H21EOLACvtfcHk0lisfb-XruJmcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXktlKisbg%253D&md5=4a5725b9464537310773862d65e8b2fe</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjm048957q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm048957q%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DR.%26aulast%3DFang%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DThe%2520PDBbind%2520database%253A%2520methodologies%2520and%2520updates%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D4111%26epage%3D4119" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Hu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benson, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lerner, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlson, H. A.</span><span> </span><span class="NLM_article-title">Binding MOAD (mother of all databases)</span> <span class="citation_source-journal">Proteins</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">333</span><span class="NLM_x">–</span> <span class="NLM_lpage">340</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2005&pages=333-340&author=L.+Huauthor=M.+L.+Bensonauthor=R.+D.+Smithauthor=M.+G.+Lernerauthor=H.+A.+Carlson&title=Binding+MOAD+%28mother+of+all+databases%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DL.%26aulast%3DBenson%26aufirst%3DM.%2BL.%26aulast%3DSmith%26aufirst%3DR.%2BD.%26aulast%3DLerner%26aufirst%3DM.%2BG.%26aulast%3DCarlson%26aufirst%3DH.%2BA.%26atitle%3DBinding%2520MOAD%2520%2528mother%2520of%2520all%2520databases%2529%26jtitle%3DProteins%26date%3D2005%26volume%3D60%26spage%3D333%26epage%3D340" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Smith, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Falkner, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benson, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nerothin, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlson, H. A.</span><span> </span><span class="NLM_article-title">Exploring protein–ligand recognition with binding MOAD</span> <span class="citation_source-journal">J. Mol. Graphics Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">414</span><span class="NLM_x">–</span> <span class="NLM_lpage">425</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1016%2Fj.jmgm.2005.08.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD28XjtFahu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2006&pages=414-425&author=R.+D.+Smithauthor=L.+Huauthor=J.+A.+Falknerauthor=M.+L.+Bensonauthor=J.+P.+Nerothinauthor=H.+A.+Carlson&title=Exploring+protein%E2%80%93ligand+recognition+with+binding+MOAD"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring protein-ligand recognition with Binding MOAD</span></div><div class="casAuthors">Smith, Richard D.; Hu, Liegi; Falkner, Jayson A.; Benson, Mark L.; Nerothin, Jason P.; Carlson, Heather A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">414-425</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">We have recently announced the largest database of protein-ligand complexes, Binding MOAD (Mother of All Databases).  After the August 2004 update, Binding MOAD contains 6816 complexes.  There are 2220 protein families and 3316 unique ligands.  After searching 6000+ crystallog. papers, we have obtained binding data for 1793 (27%) of the complexes.  We have also created a non-redundant set of complexes with only one complex from each protein family; in that set, 630 (28%) of the unique complexes have binding data.  Here, we present information about the data provided at the Binding MOAD website.  We also present the results of mining Binding MOAD to map the degree of solvent exposure for binding sites.  We have detd. that most cavities and ligands (70-85%) are well buried in the complexes.  This fits with the common paradigm that a large degree of contact between the ligand and protein is significant in mol. recognition.  GoCAV and the GoCAVviewer are the tools we created for this study.  To share our data and make our online dataset more useful to other research groups, we have integrated the viewer into the Binding MOAD website (www.BindingMOAD.org).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr08gepsx0bVbVg90H21EOLACvtfcHk0lisfb-XruJmcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjtFahu7Y%253D&md5=de65c9bf79e2a94cfdd5106642649a0e</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2005.08.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2005.08.002%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DR.%2BD.%26aulast%3DHu%26aufirst%3DL.%26aulast%3DFalkner%26aufirst%3DJ.%2BA.%26aulast%3DBenson%26aufirst%3DM.%2BL.%26aulast%3DNerothin%26aufirst%3DJ.%2BP.%26aulast%3DCarlson%26aufirst%3DH.%2BA.%26atitle%3DExploring%2520protein%25E2%2580%2593ligand%2520recognition%2520with%2520binding%2520MOAD%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2006%26volume%3D24%26spage%3D414%26epage%3D425" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Oprea, T. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allu, T. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fara, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rad, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ostopovici, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bologa, C. G.</span><span> </span><span class="NLM_article-title">Lead-like, drug-like or “Pub-like”: how different are they?</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">113</span><span class="NLM_x">–</span> <span class="NLM_lpage">119</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2007&pages=113-119&author=T.+I.+Opreaauthor=T.+K.+Alluauthor=D.+C.+Faraauthor=R.+F.+Radauthor=L.+Ostopoviciauthor=C.+G.+Bologa&title=Lead-like%2C+drug-like+or+%E2%80%9CPub-like%E2%80%9D%3A+how+different+are+they%3F"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DOprea%26aufirst%3DT.%2BI.%26aulast%3DAllu%26aufirst%3DT.%2BK.%26aulast%3DFara%26aufirst%3DD.%2BC.%26aulast%3DRad%26aufirst%3DR.%2BF.%26aulast%3DOstopovici%26aufirst%3DL.%26aulast%3DBologa%26aufirst%3DC.%2BG.%26atitle%3DLead-like%252C%2520drug-like%2520or%2520%25E2%2580%259CPub-like%25E2%2580%259D%253A%2520how%2520different%2520are%2520they%253F%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2007%26volume%3D21%26spage%3D113%26epage%3D119" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Okuno, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tamon, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yabuuchi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niijima, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minowa, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tonomura, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kunimoto, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, C.</span><span> </span><span class="NLM_article-title">GLIDA: GPCR—ligand database for chemical genomics drug discovery—database and tools update</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">D907</span><span class="NLM_x">–</span> <span class="NLM_lpage">D912</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2008&pages=D907-D912&author=Y.+Okunoauthor=A.+Tamonauthor=H.+Yabuuchiauthor=S.+Niijimaauthor=Y.+Minowaauthor=K.+Tonomuraauthor=R.+Kunimotoauthor=C.+Feng&title=GLIDA%3A+GPCR%E2%80%94ligand+database+for+chemical+genomics+drug+discovery%E2%80%94database+and+tools+update"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DOkuno%26aufirst%3DY.%26aulast%3DTamon%26aufirst%3DA.%26aulast%3DYabuuchi%26aufirst%3DH.%26aulast%3DNiijima%26aufirst%3DS.%26aulast%3DMinowa%26aufirst%3DY.%26aulast%3DTonomura%26aufirst%3DK.%26aulast%3DKunimoto%26aufirst%3DR.%26aulast%3DFeng%26aufirst%3DC.%26atitle%3DGLIDA%253A%2520GPCR%25E2%2580%2594ligand%2520database%2520for%2520chemical%2520genomics%2520drug%2520discovery%25E2%2580%2594database%2520and%2520tools%2520update%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2008%26volume%3D36%26spage%3DD907%26epage%3DD912" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Krejsa, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horvath, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogalski, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Penzotti, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mao, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbosa, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Migeon, J. C.</span><span> </span><span class="NLM_article-title">Predicting ADME properties and side effects: the BioPrint approach</span> <span class="citation_source-journal">Curr. Opin. Drug Discovery Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">470</span><span class="NLM_x">–</span> <span class="NLM_lpage">480</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=12951810" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD3sXntFymsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2003&pages=470-480&author=C.+M.+Krejsaauthor=D.+Horvathauthor=S.+L.+Rogalskiauthor=J.+E.+Penzottiauthor=B.+Maoauthor=F.+Barbosaauthor=J.+C.+Migeon&title=Predicting+ADME+properties+and+side+effects%3A+the+BioPrint+approach"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting ADME properties and side effects: The BioPrint approach</span></div><div class="casAuthors">Krejsa, Cecile M.; Horvath, Dragos; Rogalski, Sherri L.; Penzotti, Julie E.; Mao, Boryeu; Barbosa, Frederique; Migeon, Jacques C.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Drug Discovery & Development</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">470-480</span>CODEN:
                <span class="NLM_cas:coden">CODDFF</span>;
        ISSN:<span class="NLM_cas:issn">1367-6733</span>.
    
            (<span class="NLM_cas:orgname">Current Drugs</span>)
        </div><div class="casAbstract">A review.  Computational methods are increasingly used to streamline and enhance the lead discovery and optimization process.  However, accurate prediction of absorption, distribution, metab. and excretion (ADME) and adverse drug reactions (ADR) is often difficult, due to the complexity of underlying physiol. mechanisms.  Modeling approaches have been hampered by the lack of large, robust and standardized training datasets.  In an extensive effort to build such a dataset, the BioPrint database was constructed by systematic profiling of nearly all drugs available on the market, as well as numerous ref. compds.  The database is composed of several large datasets: compd. structures and mol. descriptors, in vitro ADME and pharmacol. profiles, and complementary clin. data including therapeutic use information, pharmacokinetics profiles and ADR profiles.  These data have allowed the development of computational tools designed to integrate a program of computational chem. into library design and lead development.  Models based on chem. structure are strengthened by in vitro results that can be used as addnl. compd. descriptors to predict complex in vivo endpoints.  The BioPrint pharmacoinformatics platform represents a systematic effort to accelerate the process of drug discovery, improve quant. structure-activity relationships and develop in vitro/in vivo assocns.  In this review, we will discuss the importance of training set size and diversity in model development, the implementation of linear and neighborhood modeling approaches, and the use of in silico methods to predict potential clin. liabilities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZQ71juUoErrVg90H21EOLACvtfcHk0liP4VSoKNCw9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXntFymsLw%253D&md5=f8db4e37cc895d67bf4a82efe021c5c2</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKrejsa%26aufirst%3DC.%2BM.%26aulast%3DHorvath%26aufirst%3DD.%26aulast%3DRogalski%26aufirst%3DS.%2BL.%26aulast%3DPenzotti%26aufirst%3DJ.%2BE.%26aulast%3DMao%26aufirst%3DB.%26aulast%3DBarbosa%26aufirst%3DF.%26aulast%3DMigeon%26aufirst%3DJ.%2BC.%26atitle%3DPredicting%2520ADME%2520properties%2520and%2520side%2520effects%253A%2520the%2520BioPrint%2520approach%26jtitle%3DCurr.%2520Opin.%2520Drug%2520Discovery%2520Dev.%26date%3D2003%26volume%3D6%26spage%3D470%26epage%3D480" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Bender, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenkins, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Serrano, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mikhailov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clemons, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, J. W.</span><span> </span><span class="NLM_article-title">Chemogenomic data analysis: prediction of small-molecule targets and the advent of biological fingerprint</span> <span class="citation_source-journal">Comb. Chem. High Throughput Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">719</span><span class="NLM_x">–</span> <span class="NLM_lpage">731</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.2174%2F138620707782507313" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=18045083" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnsVOhsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2007&pages=719-731&author=A.+Benderauthor=D.+W.+Youngauthor=J.+L.+Jenkinsauthor=M.+Serranoauthor=D.+Mikhailovauthor=P.+A.+Clemonsauthor=J.+W.+Davies&title=Chemogenomic+data+analysis%3A+prediction+of+small-molecule+targets+and+the+advent+of+biological+fingerprint"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Chemogenomic data analysis: prediction of small-molecule targets and the advent of biological fingerprints</span></div><div class="casAuthors">Bender, Andreas; Young, Daniel W.; Jenkins, Jeremy L.; Serrano, Martin; Mikhailov, Dmitri; Clemons, Paul A.; Davies, John W.</div><div class="citationInfo"><span class="NLM_cas:title">Combinatorial Chemistry & High Throughput Screening</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">719-731</span>CODEN:
                <span class="NLM_cas:coden">CCHSFU</span>;
        ISSN:<span class="NLM_cas:issn">1386-2073</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Chemogenomics comprises a systematic relationship between targets and ligands that are used as target modulators in living systems such as cells or organisms.  In recent years, data on small mol.-bioactivity relationships have become increasingly available, and consequently so have the no. of approaches used to translate bioactivity data into knowledge.  This review will focus on 2 aspects of chemogenomics.  Firstly, in cases such as cell-based screens, the question of which target(s) a compd. is modulating to cause the obsd. phenotype is crucial.  In silico target prediction tools can suggest likely biol. targets of small mols. via data mining in target-annotated chem. databases.  This review presents some of the current tools available for this task and shows some sample applications relevant to a pharmaceutical industry setting.  These applications are the prediction of false-positives in cell-based reporter gene assays, the prediction of targets by linking bioassay data with protein domain annotations, and the direct prediction of adverse reactions.  Secondly, in recent years a shift from structure-derived chem. descriptors to biol. descriptors has occurred.  Here, the effect of a compd. on a no. of biol. endpoints is used to make predictions about other properties, such as putative targets, assocd. adverse reactions, and pathways modulated by the compd.  This review further summarizes these "performance" descriptors and their applications, focusing on gene expression profiles and high-content screening data.  The advent of such biol. fingerprints suggests that the field of drug discovery is currently at a crossroads, where single target bioassay results are supplanted by multidimensional biol. fingerprints that reflect a new awareness of biol. networks and polypharmacol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJuaT0KRiZF7Vg90H21EOLACvtfcHk0ljuK8fOE1OBrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnsVOhsg%253D%253D&md5=4b3152898f48cc7021904a0c983f6068</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.2174%2F138620707782507313&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138620707782507313%26sid%3Dliteratum%253Aachs%26aulast%3DBender%26aufirst%3DA.%26aulast%3DYoung%26aufirst%3DD.%2BW.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DSerrano%26aufirst%3DM.%26aulast%3DMikhailov%26aufirst%3DD.%26aulast%3DClemons%26aufirst%3DP.%2BA.%26aulast%3DDavies%26aufirst%3DJ.%2BW.%26atitle%3DChemogenomic%2520data%2520analysis%253A%2520prediction%2520of%2520small-molecule%2520targets%2520and%2520the%2520advent%2520of%2520biological%2520fingerprint%26jtitle%3DComb.%2520Chem.%2520High%2520Throughput%2520Screening%26date%3D2007%26volume%3D10%26spage%3D719%26epage%3D731" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">van der Horst, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okuno, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bender, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">AP, I. J.</span><span> </span><span class="NLM_article-title">Substructure mining of GPCR ligands reveals activity-class specific functional groups in an unbiased manner</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">348</span><span class="NLM_x">–</span> <span class="NLM_lpage">360</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci8003896" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlegt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=348-360&author=E.+van+der+Horstauthor=Y.+Okunoauthor=A.+Benderauthor=I.+J.+AP&title=Substructure+mining+of+GPCR+ligands+reveals+activity-class+specific+functional+groups+in+an+unbiased+manner"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Substructure Mining of GPCR Ligands Reveals Activity-Class Specific Functional Groups in an Unbiased Manner</span></div><div class="casAuthors">van der Horst, Eelke; Okuno, Yasushi; Bender, Andreas; IJzerman, Adriaan P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">348-360</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this study, we conducted frequent substructure mining to identify structural features that discriminate between ligands that do bind to G protein-coupled receptors (GPCRs) and those that do not.  In most cases, particular chem. representations resulted in the most significant substructures.  Substructures found to be characteristic for the background control set reflected reactions that may have been used to construct this library, e.g. for the ChemBridge DIVERSet library employed these are ester and carboxamide moieties.  Alkane amine substructures were identified as most important for GPCR ligands, e.g. the butylamine substructure, often linked to an arom. system.  Hierarchical anal. of targeted GPCRs revealed well-known motives and new substructural features.  One example is the imidazole-like substructure common for the histamine binding receptor ligands.  Another example is the planar ring system consisting of a fused five- and six-membered ring (indole-like substructure) common for the serotonin receptor ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVvg1lPYXlkbVg90H21EOLACvtfcHk0ljuK8fOE1OBrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlegt78%253D&md5=c7a95054a55fd0d58196ff858d3d500e</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fci8003896&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci8003896%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2Bder%2BHorst%26aufirst%3DE.%26aulast%3DOkuno%26aufirst%3DY.%26aulast%3DBender%26aufirst%3DA.%26aulast%3DAP%26aufirst%3DI.%2BJ.%26atitle%3DSubstructure%2520mining%2520of%2520GPCR%2520ligands%2520reveals%2520activity-class%2520specific%2520functional%2520groups%2520in%2520an%2520unbiased%2520manner%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D348%26epage%3D360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Klabunde, T.</span><span> </span><span class="NLM_article-title">Chemogenomic approaches to drug discovery: similar receptors bind similar ligands</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">152</span><span class="NLM_x">, </span> <span class="NLM_fpage">5</span><span class="NLM_x">–</span> <span class="NLM_lpage">7</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1038%2Fsj.bjp.0707308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=17533415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpsFOjur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=152&publication_year=2007&pages=5-7&author=T.+Klabunde&title=Chemogenomic+approaches+to+drug+discovery%3A+similar+receptors+bind+similar+ligands"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Chemogenomic approaches to drug discovery: similar receptors bind similar ligands</span></div><div class="casAuthors">Klabunde, T.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">152</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-7</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Within recent years, a paradigm shift from traditional receptor-specific studies to a cross-receptor view has taken place within pharmaceutical research to increase the efficiency of modern drug discovery.  Receptors are no longer viewed as single entities but grouped into sets of related proteins or receptor families that are explored in a systematic manner.  This interdisciplinary approach attempting to derive predictive links between the chem. structures of bioactive mols. and the receptors with which these mols. interact is referred to as chemogenomics.  Insights from chemogenomics are used for the rational compilation of screening sets and for the rational design and synthesis of directed chem. libraries to accelerate drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQeX3iyp_Z47Vg90H21EOLACvtfcHk0ljuK8fOE1OBrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpsFOjur0%253D&md5=a103cdd189d2bad6d70e799aad869290</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjp.0707308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjp.0707308%26sid%3Dliteratum%253Aachs%26aulast%3DKlabunde%26aufirst%3DT.%26atitle%3DChemogenomic%2520approaches%2520to%2520drug%2520discovery%253A%2520similar%2520receptors%2520bind%2520similar%2520ligands%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2007%26volume%3D152%26spage%3D5%26epage%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">van Westen, G. J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wegner, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">IJzerman, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Vlijmen, H. W. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bender, A.</span><span> </span><span class="NLM_article-title">Proteochemometric modeling as a tool to design selective compounds and for extrapolating to novel targets</span> <span class="citation_source-journal">Med. Chem. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">16</span><span class="NLM_x">–</span> <span class="NLM_lpage">30</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1039%2Fc0md00165a" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=16-30&author=G.+J.+P.+van+Westenauthor=J.+K.+Wegnerauthor=A.+P.+IJzermanauthor=H.+W.+T.+van+Vlijmenauthor=A.+Bender&title=Proteochemometric+modeling+as+a+tool+to+design+selective+compounds+and+for+extrapolating+to+novel+targets"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1039%2Fc0md00165a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc0md00165a%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BWesten%26aufirst%3DG.%2BJ.%2BP.%26aulast%3DWegner%26aufirst%3DJ.%2BK.%26aulast%3DIJzerman%26aufirst%3DA.%2BP.%26aulast%3Dvan%2BVlijmen%26aufirst%3DH.%2BW.%2BT.%26aulast%3DBender%26aufirst%3DA.%26atitle%3DProteochemometric%2520modeling%2520as%2520a%2520tool%2520to%2520design%2520selective%2520compounds%2520and%2520for%2520extrapolating%2520to%2520novel%2520targets%26jtitle%3DMed.%2520Chem.%2520Commun.%26date%3D2011%26volume%3D2%26spage%3D16%26epage%3D30" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Geppert, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vogt, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Current trends in ligand-based virtual screening: molecular representations, data mining methods, new application areas, and performance evaluation</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">205</span><span class="NLM_x">–</span> <span class="NLM_lpage">216</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci900419k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnvFWmsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=205-216&author=H.+Geppertauthor=M.+Vogtauthor=J.+Bajorath&title=Current+trends+in+ligand-based+virtual+screening%3A+molecular+representations%2C+data+mining+methods%2C+new+application+areas%2C+and+performance+evaluation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Current Trends in Ligand-Based Virtual Screening: Molecular Representations, Data Mining Methods, New Application Areas, and Performance Evaluation</span></div><div class="casAuthors">Geppert, Hanna; Vogt, Martin; Bajorath, Jurgen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">205-216</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpc1HgbkXa9o7Vg90H21EOLACvtfcHk0lj342K1wXiFTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnvFWmsA%253D%253D&md5=aeb674ef7c6c93711eb21452732e21d8</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fci900419k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900419k%26sid%3Dliteratum%253Aachs%26aulast%3DGeppert%26aufirst%3DH.%26aulast%3DVogt%26aufirst%3DM.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DCurrent%2520trends%2520in%2520ligand-based%2520virtual%2520screening%253A%2520molecular%2520representations%252C%2520data%2520mining%2520methods%252C%2520new%2520application%2520areas%252C%2520and%2520performance%2520evaluation%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D205%26epage%3D216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Bender, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenkins, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scheiber, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sukuru, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glick, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, J. W.</span><span> </span><span class="NLM_article-title">How similar are similarity searching methods? A principal component analysis of molecular descriptor space</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">108</span><span class="NLM_x">–</span> <span class="NLM_lpage">119</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci800249s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD1MXisVahtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=108-119&author=A.+Benderauthor=J.+L.+Jenkinsauthor=J.+Scheiberauthor=S.+C.+Sukuruauthor=M.+Glickauthor=J.+W.+Davies&title=How+similar+are+similarity+searching+methods%3F+A+principal+component+analysis+of+molecular+descriptor+space"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">How similar are similarity searching methods? A principal component analysis of molecular descriptor space</span></div><div class="casAuthors">Bender, Andreas; Jenkins, Jeremy L.; Scheiber, Josef; Sukuru, Sai Chetan K.; Glick, Meir; Davies, John W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">108-119</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Different mol. descriptors capture different aspects of mol. structures, but this effect has not yet been quantified systematically on a large scale.  In this work, we calc. the similarity of 37 descriptors by repeatedly selecting query compds. and ranking the rest of the database.  Euclidean distances between the rank-ordering of different descriptors are calcd. to det. descriptor (as opposed to compd.) similarity, followed by PCA for visualization.  Four broad descriptor classes are identified, which are circular fingerprints; circular fingerprints considering counts; path-based and keyed fingerprints; and pharmacophoric descriptors.  Descriptor behavior is much more defined by those four classes than the particular parametrization.  Using counts instead of the presence/absence of fingerprints significantly changes descriptor behavior, which is crucial for performance of topol. autocorrelation vectors, but not circular fingerprints.  Four-point pharmacophores (piDAPH4) surprisingly lead to much higher retrieval rates than three-point pharmacophores (28.21% vs 19.15%) but still similar rank-ordering of compds. (retrieval of similar actives).  Looking into individual rankings, circular fingerprints seem more appropriate than path-based fingerprints if complex ring systems or branching patterns are present; count-based fingerprints could be more suitable in databases with a large no. of repeated subunits (amide bonds, sugar rings, terpenes).  Information-based selection of diverse fingerprints for consensus scoring (ECFP4/TGD fingerprints) led only to marginal improvement over single fingerprint results.  While it seems to be nontrivial to exploit orthogonal descriptor behavior to improve retrieval rates in consensus virtual screening, those descriptors still each retrieve different actives which corroborates the strategy of employing diverse descriptors individually in prospective virtual screening settings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgAHu8jM0iuLVg90H21EOLACvtfcHk0lj342K1wXiFTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXisVahtQ%253D%253D&md5=edcbc5314c60e4607a7cbdfd60562f53</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fci800249s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800249s%26sid%3Dliteratum%253Aachs%26aulast%3DBender%26aufirst%3DA.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DScheiber%26aufirst%3DJ.%26aulast%3DSukuru%26aufirst%3DS.%2BC.%26aulast%3DGlick%26aufirst%3DM.%26aulast%3DDavies%26aufirst%3DJ.%2BW.%26atitle%3DHow%2520similar%2520are%2520similarity%2520searching%2520methods%253F%2520A%2520principal%2520component%2520analysis%2520of%2520molecular%2520descriptor%2520space%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D108%26epage%3D119" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Leach, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gillet, V. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, R.</span><span> </span><span class="NLM_article-title">Three-dimensional pharmacophore methods in drug discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">539</span><span class="NLM_x">–</span> <span class="NLM_lpage">558</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm900817u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1OktbjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=539-558&author=A.+R.+Leachauthor=V.+J.+Gilletauthor=R.+A.+Lewisauthor=R.+Taylor&title=Three-dimensional+pharmacophore+methods+in+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Three-Dimensional Pharmacophore Methods in Drug Discovery</span></div><div class="casAuthors">Leach, Andrew R.; Gillet, Valerie J.; Lewis, Richard A.; Taylor, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">539-558</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  In the first part of this Perspective the authors provide an overview of the computational 3D pharmacophore methods most commonly used in drug discovery.  The authors focus in particular on the key problem of pharmacophore elucidation: the identification from a set of active mols. and their biol. activities of the key common features and their relative orientations (also called pharmacophore mapping).  In the second part the authors reflect on the challenges in this field and provide some thoughts on future directions.  The authors' primary goal is to provide the interested reader with a fair assessment of the current state-of-the-art in this field with an emphasis on the methods and software that in the authors' experience are most widely used in real drug discovery projects.  The authors will cover both the practical aspects and some of the inherent limitations of such methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9x_N0BVt1DLVg90H21EOLACvtfcHk0lj342K1wXiFTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1OktbjP&md5=b9844e41a7140b58ee508aab07d9e4ed</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm900817u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm900817u%26sid%3Dliteratum%253Aachs%26aulast%3DLeach%26aufirst%3DA.%2BR.%26aulast%3DGillet%26aufirst%3DV.%2BJ.%26aulast%3DLewis%26aufirst%3DR.%2BA.%26aulast%3DTaylor%26aufirst%3DR.%26atitle%3DThree-dimensional%2520pharmacophore%2520methods%2520in%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D539%26epage%3D558" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Moitessier, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Englebienne, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lawandi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corbeil, C. R.</span><span> </span><span class="NLM_article-title">Towards the development of universal, fast and highly accurate docking/scoring methods: a long way to go</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">153</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 1</span><span class="NLM_x">) </span> <span class="NLM_fpage">S7</span><span class="NLM_x">–</span> <span class="NLM_lpage">S26</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=153&publication_year=2008&pages=S7-S26&issue=Suppl.+1&author=N.+Moitessierauthor=P.+Englebienneauthor=D.+Leeauthor=J.+Lawandiauthor=C.+R.+Corbeil&title=Towards+the+development+of+universal%2C+fast+and+highly+accurate+docking%2Fscoring+methods%3A+a+long+way+to+go"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMoitessier%26aufirst%3DN.%26aulast%3DEnglebienne%26aufirst%3DP.%26aulast%3DLee%26aufirst%3DD.%26aulast%3DLawandi%26aufirst%3DJ.%26aulast%3DCorbeil%26aufirst%3DC.%2BR.%26atitle%3DTowards%2520the%2520development%2520of%2520universal%252C%2520fast%2520and%2520highly%2520accurate%2520docking%252Fscoring%2520methods%253A%2520a%2520long%2520way%2520to%2520go%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2008%26volume%3D153%26issue%3DSuppl.%25201%26spage%3DS7%26epage%3DS26" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Sanders, M. P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGuire, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roumen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Esch, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Vlieg, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klomp, J. P. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Graaf, C.</span><span> </span><span class="NLM_article-title">From the protein’s perspective: the benefits and challenges of protein structure-based pharmacophore modeling</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">28</span><span class="NLM_x">–</span> <span class="NLM_lpage">38</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=28-38&author=M.+P.+A.+Sandersauthor=R.+McGuireauthor=L.+Roumenauthor=I.+J.+De+Eschauthor=J.+de+Vliegauthor=J.+P.+G.+Klompauthor=C.+de+Graaf&title=From+the+protein%E2%80%99s+perspective%3A+the+benefits+and+challenges+of+protein+structure-based+pharmacophore+modeling"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSanders%26aufirst%3DM.%2BP.%2BA.%26aulast%3DMcGuire%26aufirst%3DR.%26aulast%3DRoumen%26aufirst%3DL.%26aulast%3DDe%2BEsch%26aufirst%3DI.%2BJ.%26aulast%3Dde%2BVlieg%26aufirst%3DJ.%26aulast%3DKlomp%26aufirst%3DJ.%2BP.%2BG.%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26atitle%3DFrom%2520the%2520protein%25E2%2580%2599s%2520perspective%253A%2520the%2520benefits%2520and%2520challenges%2520of%2520protein%2520structure-based%2520pharmacophore%2520modeling%26jtitle%3DMedChemComm%26date%3D2011%26volume%3D3%26spage%3D28%26epage%3D38" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">de Graaf, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">Customizing G Protein-coupled receptor models for structure-based virtual screening</span> <span class="citation_source-journal">Curr. Pharm. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">4026</span><span class="NLM_x">–</span> <span class="NLM_lpage">4048</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.2174%2F138161209789824786" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=20028320" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlKlu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2009&pages=4026-4048&author=C.+de+Graafauthor=D.+Rognan&title=Customizing+G+Protein-coupled+receptor+models+for+structure-based+virtual+screening"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Customizing G protein-coupled receptor models for structure-based virtual screening</span></div><div class="casAuthors">de Graaf, Chris; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">4026-4048</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">This review will focus on the construction, refinement, and validation of G Protein-coupled receptor models for the purpose of structure-based virtual screening.  Practical tips and tricks derived from concrete modeling and virtual screening exercises to overcome the problems and pitfalls assocd. with the different steps of the receptor modeling workflow will be presented.  These examples will not only include rhodopsin-like (class A), but also secretine-like (class B), and glutamate-like (class C) receptors.  In addn., the review will present a careful comparative anal. of current crystal structures and their implication on homol. modeling.  The following themes will be discussed: (i) the use of exptl. anchors in guiding the modeling procedure; (ii) amino acid sequence alignments; (iii) ligand binding mode accommodation and binding cavity expansion; (iv) proline-induced kinks in transmembrane helixes; (v) binding mode prediction and virtual screening by receptor-ligand interaction fingerprint scoring; (vi) extracellular loop modeling; (vii) virtual filtering schemes.  Finally, an overview of several successful structure-based screening shows that receptor models, despite structural inaccuracies, can be efficiently used to find novel ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqxkKuCF95LrVg90H21EOLACvtfcHk0lgA7lhdvVURTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlKlu7Y%253D&md5=e3aefb5a191d0507cca6766ce191e82c</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.2174%2F138161209789824786&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138161209789824786%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DCustomizing%2520G%2520Protein-coupled%2520receptor%2520models%2520for%2520structure-based%2520virtual%2520screening%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2009%26volume%3D15%26spage%3D4026%26epage%3D4048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Liu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ouyang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gong, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H.</span><span> </span><span class="NLM_article-title">PharmMapper server: a Web server for potential drug target identification using pharmacophore mapping approach</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">W609</span><span class="NLM_x">–</span> <span class="NLM_lpage">W614</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1093%2Fnar%2Fgkq300" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=20430828" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BC3cXotVWjsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=W609-W614&author=X.+Liuauthor=S.+Ouyangauthor=B.+Yuauthor=Y.+Liuauthor=K.+Huangauthor=J.+Gongauthor=S.+Zhengauthor=Z.+Liauthor=H.+Liauthor=H.+Jiang&title=PharmMapper+server%3A+a+Web+server+for+potential+drug+target+identification+using+pharmacophore+mapping+approach"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">PharmMapper server: a web server for potential drug target identification using pharmacophore mapping approach</span></div><div class="casAuthors">Liu, Xiaofeng; Ouyang, Sisheng; Yu, Biao; Liu, Yabo; Huang, Kai; Gong, Jiayu; Zheng, Siyuan; Li, Zhihua; Li, Honglin; Jiang, Hualiang</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">Web Server</span>),
    <span class="NLM_cas:pages">W609-W614</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">In silico drug target identification, which includes many distinct algorithms for finding disease genes and proteins, is the first step in the drug discovery pipeline.  When the 3D structures of the targets are available, the problem of target identification is usually converted to finding the best interaction mode between the potential target candidates and small mol. probes.  Pharmacophore, which is the spatial arrangement of features essential for a mol. to interact with a specific target receptor, is an alternative method for achieving this goal apart from mol. docking method.  PharmMapper server is a freely accessed web server designed to identify potential target candidates for the given small mols. (drugs, natural products or other newly discovered compds. with unidentified binding targets) using pharmacophore mapping approach.  PharmMapper hosts a large, inhouse repertoire of pharmacophore database (namely PharmTargetDB) annotated from all the targets information in TargetBank, BindingDB, DrugBank and potential drug target database, including over 7000 receptor-based pharmacophore models (covering over 1500 drug targets information).  PharmMapper automatically finds the best mapping poses of the query mol. against all the pharmacophore models in PharmTargetDB and lists the top N best-fitted hits with appropriate target annotations, as well as resp. mol.'s aligned poses are presented.  Benefited from the highly efficient and robust triangle hashing mapping method, PharmMapper bears high throughput ability and only costs 1 h averagely to screen the whole PharmTargetDB.  The protocol was successful in finding the proper targets among the top 300 pharmacophore candidates in the retrospective benchmarking test of tamoxifen.  PharmMapper is available at http://59.78.96.61/pharmmapper.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqL-Gaa6UM_zLVg90H21EOLACvtfcHk0lgA7lhdvVURTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXotVWjsL4%253D&md5=dadfc5a278243d22f2ea041914461e3a</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkq300&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkq300%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DOuyang%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DK.%26aulast%3DGong%26aufirst%3DJ.%26aulast%3DZheng%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DH.%26atitle%3DPharmMapper%2520server%253A%2520a%2520Web%2520server%2520for%2520potential%2520drug%2520target%2520identification%2520using%2520pharmacophore%2520mapping%2520approach%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2010%26volume%3D38%26spage%3DW609%26epage%3DW614" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Rask-Andersen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Almen, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schioth, H. B.</span><span> </span><span class="NLM_article-title">Trends in the exploitation of novel drug targets</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">579</span><span class="NLM_x">–</span> <span class="NLM_lpage">590</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1038%2Fnrd3478" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=21804595" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpsFKqsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=579-590&author=M.+Rask-Andersenauthor=M.+S.+Almenauthor=H.+B.+Schioth&title=Trends+in+the+exploitation+of+novel+drug+targets"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Trends in the exploitation of novel drug targets</span></div><div class="casAuthors">Rask-Andersen, Mathias; Almen, Markus Saellman; Schioeth, Helgi B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">579-590</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The discovery and exploitation of new drug targets is a key focus for both the pharmaceutical industry and academic biomedical research.  To provide an insight into trends in the exploitation of new drug targets, we have analyzed the drugs that were approved by the US Food and Drug Administration during the past three decades and examd. the interactions of these drugs with therapeutic targets that are encoded by the human genome, using the DrugBank database and extensive manual curation.  We have identified 435 effect-mediating drug targets in the human genome, which are modulated by 989 unique drugs, through 2,242 drug-target interactions.  We also analyze trends in the introduction of drugs that modulate previously unexploited targets, and discuss the network pharmacol. of the drugs in our data set.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo19FXXFB-PsLVg90H21EOLACvtfcHk0lgA7lhdvVURTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpsFKqsLc%253D&md5=9255431a28f2c07759b0f36975711f84</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fnrd3478&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3478%26sid%3Dliteratum%253Aachs%26aulast%3DRask-Andersen%26aufirst%3DM.%26aulast%3DAlmen%26aufirst%3DM.%2BS.%26aulast%3DSchioth%26aufirst%3DH.%2BB.%26atitle%3DTrends%2520in%2520the%2520exploitation%2520of%2520novel%2520drug%2520targets%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D579%26epage%3D590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Szabo, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kincses, Z. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vecsei, L.</span><span> </span><span class="NLM_article-title">Novel therapy in Parkinson’s disease: adenosine A(2A) receptor antagonists</span> <span class="citation_source-journal">Expert Opin. Drug Metab. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">441</span><span class="NLM_x">–</span> <span class="NLM_lpage">455</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1517%2F17425255.2011.557066" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=21332415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A280%3ADC%252BC3M3ot1Oquw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=441-455&author=N.+Szaboauthor=Z.+T.+Kincsesauthor=L.+Vecsei&title=Novel+therapy+in+Parkinson%E2%80%99s+disease%3A+adenosine+A%282A%29+receptor+antagonists"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Novel therapy in Parkinson's disease: adenosine A(2A) receptor antagonists</span></div><div class="casAuthors">Szabo Nikoletta; Kincses Zsigmond Tamas; Vecsei Laszlo</div><div class="citationInfo"><span class="NLM_cas:title">Expert opinion on drug metabolism & toxicology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">441-55</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">INTRODUCTION: Parkinson's disease (PD) is a progressive neurodegenerative disorder.  To date, most of the currently available therapies in PD target the dopaminergic system and none of these therapeutic approaches have been proven to modify the course of the disease.  To various extents, these drugs can also cause motor and non-motor complications.  A novel target, the adenosine A(2A) receptor (AA2AR), was recently identified, blockade of which may alleviate Parkinsonian symptoms, reduce motor fluctuations and potentially afford neuroprotection.  AREAS COVERED: This review is based on a PubMed search covering the relationship of the adenosine receptors and PD.  The role of the AA2AR is reviewed and the results of preclinical investigations of antagonists are assessed.  A synopsis of current drug development is provided, with a special focus on the pharmacokinetics and relevant clinical trials.  EXPERT OPINION: The localization of the AA2AR in the central nervous system, the ultra structural localization and the molecular mechanism of its action reveal the potential importance of the AA2AR in movement disorders.  The theoretical background and experimental data indicate that AA2AR antagonists may have a potential therapeutic effect in Parkinson's disease.  More importantly, the putative neuroprotective effect needs further investigation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSkwTiCXUG5u5w7prLlxLOzfW6udTcc2eag___az8a2C7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3M3ot1Oquw%253D%253D&md5=c14a993e0dda01906cd19b08dd5db683</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1517%2F17425255.2011.557066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17425255.2011.557066%26sid%3Dliteratum%253Aachs%26aulast%3DSzabo%26aufirst%3DN.%26aulast%3DKincses%26aufirst%3DZ.%2BT.%26aulast%3DVecsei%26aufirst%3DL.%26atitle%3DNovel%2520therapy%2520in%2520Parkinson%25E2%2580%2599s%2520disease%253A%2520adenosine%2520A%25282A%2529%2520receptor%2520antagonists%26jtitle%3DExpert%2520Opin.%2520Drug%2520Metab.%2520Toxicol.%26date%3D2011%26volume%3D7%26spage%3D441%26epage%3D455" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Dorsam, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gutkind, J. S.</span><span> </span><span class="NLM_article-title">G-Protein-coupled receptors and cancer</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">79</span><span class="NLM_x">–</span> <span class="NLM_lpage">94</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1038%2Fnrc2069" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=17251915" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD2sXotFOisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2007&pages=79-94&author=R.+T.+Dorsamauthor=J.+S.+Gutkind&title=G-Protein-coupled+receptors+and+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">G-protein-coupled receptors and cancer</span></div><div class="casAuthors">Dorsam, Robert T.; Gutkind, J. Silvio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">79-94</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  G-protein-coupled receptors (GPCRs), the largest family of cell-surface mols. involved in signal transmission, have recently emerged as crucial players in tumor growth and metastasis.  Malignant cells often hijack the normal physiol. functions of GPCRs to survive, proliferate autonomously, evade the immune system, increase their blood supply, invade their surrounding tissues and disseminate to other organs.  This review will address our current understanding of the many roles of GPCRs and their signaling circuitry in tumor progression and metastasis.  We will also discuss how interfering with GPCRs might provide unique opportunities for cancer prevention and treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLfbCDb7EA8LVg90H21EOLACvtfcHk0ljhVsSkIpiGlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXotFOisw%253D%253D&md5=17b71299f0b532fbd01683c678f10397</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2Fnrc2069&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2069%26sid%3Dliteratum%253Aachs%26aulast%3DDorsam%26aufirst%3DR.%2BT.%26aulast%3DGutkind%26aufirst%3DJ.%2BS.%26atitle%3DG-Protein-coupled%2520receptors%2520and%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2007%26volume%3D7%26spage%3D79%26epage%3D94" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Cherezov, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenbaum, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanson, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rasmussen, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thian, F. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weis, W. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevens, R. C.</span><span> </span><span class="NLM_article-title">High-resolution crystal structure of an engineered human beta2-adrenergic G protein-coupled receptor</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">318</span><span class="NLM_x">, </span> <span class="NLM_fpage">1258</span><span class="NLM_x">–</span> <span class="NLM_lpage">1265</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1126%2Fscience.1150577" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=17962520" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlGmur7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=318&publication_year=2007&pages=1258-1265&author=V.+Cherezovauthor=D.+M.+Rosenbaumauthor=M.+A.+Hansonauthor=S.+G.+Rasmussenauthor=F.+S.+Thianauthor=T.+S.+Kobilkaauthor=H.+J.+Choiauthor=P.+Kuhnauthor=W.+I.+Weisauthor=B.+K.+Kobilkaauthor=R.+C.+Stevens&title=High-resolution+crystal+structure+of+an+engineered+human+beta2-adrenergic+G+protein-coupled+receptor"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">High-Resolution Crystal Structure of an Engineered Human β2-Adrenergic G Protein-Coupled Receptor</span></div><div class="casAuthors">Cherezov, Vadim; Rosenbaum, Daniel M.; Hanson, Michael A.; Rasmussen, Soren G. F.; Thian, Foon Sun; Kobilka, Tong Sun; Choi, Hee-Jung; Kuhn, Peter; Weis, William I.; Kobilka, Brian K.; Stevens, Raymond C.; Takeda, S.; Kadowaki, S.; Haga, T.; Takaesu, H.; Mitaku, S.; Fredriksson, R.; Lagerstrom, M. C.; Lundin, L. G.; Schioth, H. B.; Pierce, K. L.; Premont, R. T.; Lefkowitz, R. J.; Lefkowitz, R. J.; Shenoy, S. K.; Rosenbaum, D. M.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">318</span>
        (<span class="NLM_cas:issue">5854</span>),
    <span class="NLM_cas:pages">1258-1265</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Second extracellular loop, which in the β2-adrenergic receptor contains an unusual pair of disulfide bonds and an extra helix.  This loop and the absence Heterotrimeric guanine nucleotide-binding protein (G protein)-coupled receptors constitute the largest family of eukaryotic signal transduction proteins that communicate across the membrane.  We report the crystal structure of a human β2-adrenergic receptor-T4 lysozyme fusion protein bound to the partial inverse agonist carazolol at 2.4 angstrom resoln.  The structure provides a high-resoln. view of a human G protein-coupled receptor bound to a diffusible ligand.  Ligand-binding site accessibility is enabled by the second extracellular loop, which is held out of the binding cavity by a pair of closely spaced disulfide bridges and a short helical segment within the loop.  Cholesterol, a necessary component for crystn., mediates an intriguing parallel assocn. of receptor mols. in the crystal lattice.  Although the location of carazolol in the β2-adrenergic receptor is very similar to that of retinal in rhodopsin, structural differences in the ligand-binding site and other regions highlight the challenges in using rhodopsin as a template model for this large receptor family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpC-8wZ2k2GHLVg90H21EOLACvtfcHk0ljhVsSkIpiGlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlGmur7I&md5=12c5bacb8464a4b243fe9341192b5b3b</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1126%2Fscience.1150577&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1150577%26sid%3Dliteratum%253Aachs%26aulast%3DCherezov%26aufirst%3DV.%26aulast%3DRosenbaum%26aufirst%3DD.%2BM.%26aulast%3DHanson%26aufirst%3DM.%2BA.%26aulast%3DRasmussen%26aufirst%3DS.%2BG.%26aulast%3DThian%26aufirst%3DF.%2BS.%26aulast%3DKobilka%26aufirst%3DT.%2BS.%26aulast%3DChoi%26aufirst%3DH.%2BJ.%26aulast%3DKuhn%26aufirst%3DP.%26aulast%3DWeis%26aufirst%3DW.%2BI.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26atitle%3DHigh-resolution%2520crystal%2520structure%2520of%2520an%2520engineered%2520human%2520beta2-adrenergic%2520G%2520protein-coupled%2520receptor%26jtitle%3DScience%26date%3D2007%26volume%3D318%26spage%3D1258%26epage%3D1265" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Jaakola, V. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffith, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanson, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherezov, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chien, E. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lane, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ijzerman, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevens, R. C.</span><span> </span><span class="NLM_article-title">The 2.6 angstrom crystal structure of a human A2A adenosine receptor bound to an antagonist</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">322</span><span class="NLM_x">, </span> <span class="NLM_fpage">1211</span><span class="NLM_x">–</span> <span class="NLM_lpage">1217</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1126%2Fscience.1164772" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=18832607" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlyqtbfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=322&publication_year=2008&pages=1211-1217&author=V.+P.+Jaakolaauthor=M.+T.+Griffithauthor=M.+A.+Hansonauthor=V.+Cherezovauthor=E.+Y.+Chienauthor=J.+R.+Laneauthor=A.+P.+Ijzermanauthor=R.+C.+Stevens&title=The+2.6+angstrom+crystal+structure+of+a+human+A2A+adenosine+receptor+bound+to+an+antagonist"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">The 2.6 Angstrom Crystal Structure of a Human A2A Adenosine Receptor Bound to an Antagonist</span></div><div class="casAuthors">Jaakola, Veli-Pekka; Griffith, Mark T.; Hanson, Michael A.; Cherezov, Vadim; Chien, Ellen Y. T.; Lane, J. Robert; IJzerman, Adriaan P.; Stevens, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">322</span>
        (<span class="NLM_cas:issue">5905</span>),
    <span class="NLM_cas:pages">1211-1217</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The adenosine class of heterotrimeric guanine nucleotide-binding protein (G protein)-coupled receptors (GPCRs) mediates the important role of extracellular adenosine in many physiol. processes and is antagonized by caffeine.  The authors have detd. the crystal structure of the human A2A adenosine receptor, in complex with a high-affinity subtype-selective antagonist, ZM241385, to 2.6 angstrom resoln.  Four disulfide bridges in the extracellular domain, combined with a subtle repacking of the transmembrane helixes relative to the adrenergic and rhodopsin receptor structures, define a pocket distinct from that of other structurally detd. GPCRs.  The arrangement allows for the binding of the antagonist in an extended conformation, perpendicular to the membrane plane.  The binding site highlights an integral role for the extracellular loops, together with the helical core, in ligand recognition by this class of GPCRs and suggests a role for ZM241385 in restricting the movement of a tryptophan residue important in the activation mechanism of the class A receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7QL4wN4TdhrVg90H21EOLACvtfcHk0ljhVsSkIpiGlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlyqtbfN&md5=5bdb862b41f345c244f3c162e058206b</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1126%2Fscience.1164772&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1164772%26sid%3Dliteratum%253Aachs%26aulast%3DJaakola%26aufirst%3DV.%2BP.%26aulast%3DGriffith%26aufirst%3DM.%2BT.%26aulast%3DHanson%26aufirst%3DM.%2BA.%26aulast%3DCherezov%26aufirst%3DV.%26aulast%3DChien%26aufirst%3DE.%2BY.%26aulast%3DLane%26aufirst%3DJ.%2BR.%26aulast%3DIjzerman%26aufirst%3DA.%2BP.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26atitle%3DThe%25202.6%2520angstrom%2520crystal%2520structure%2520of%2520a%2520human%2520A2A%2520adenosine%2520receptor%2520bound%2520to%2520an%2520antagonist%26jtitle%3DScience%26date%3D2008%26volume%3D322%26spage%3D1211%26epage%3D1217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Hanson, M. A</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffith, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, F. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinhart, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desale, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clemons, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cahalan, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schuerer, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanna, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevens, R. C.</span><span> </span><span class="NLM_article-title">Crystal structure of a lipid G protein-coupled receptor.</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">335</span><span class="NLM_x">, </span> <span class="NLM_fpage">851</span><span class="NLM_x">–</span> <span class="NLM_lpage">855</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=335&publication_year=2012&pages=851-855&author=M.+A+Hansonauthor=C.+B.+Rothauthor=E.+Joauthor=M.+T.+Griffithauthor=F.+L.+Scottauthor=G.+Reinhartauthor=H.+Desaleauthor=B.+Clemonsauthor=S.+M.+Cahalanauthor=S.+C.+Schuererauthor=M.+G.+Sannaauthor=G.+W.+Hanauthor=P.+Kuhnauthor=H.+Rosenauthor=R.+C.+Stevens&title=Crystal+structure+of+a+lipid+G+protein-coupled+receptor."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHanson%26aufirst%3DM.%2BA%26aulast%3DRoth%26aufirst%3DC.%2BB.%26aulast%3DJo%26aufirst%3DE.%26aulast%3DGriffith%26aufirst%3DM.%2BT.%26aulast%3DScott%26aufirst%3DF.%2BL.%26aulast%3DReinhart%26aufirst%3DG.%26aulast%3DDesale%26aufirst%3DH.%26aulast%3DClemons%26aufirst%3DB.%26aulast%3DCahalan%26aufirst%3DS.%2BM.%26aulast%3DSchuerer%26aufirst%3DS.%2BC.%26aulast%3DSanna%26aufirst%3DM.%2BG.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DKuhn%26aufirst%3DP.%26aulast%3DRosen%26aufirst%3DH.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26atitle%3DCrystal%2520structure%2520of%2520a%2520lipid%2520G%2520protein-coupled%2520receptor.%26jtitle%3DScience%26date%3D2012%26volume%3D335%26spage%3D851%26epage%3D855" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Xu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katritch, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, Z. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherezov, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevens, R. C.</span><span> </span><span class="NLM_article-title">Structure of an agonist-bound human A2A adenosine receptor</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">332</span><span class="NLM_x">, </span> <span class="NLM_fpage">322</span><span class="NLM_x">–</span> <span class="NLM_lpage">327</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1126%2Fscience.1202793" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=21393508" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BC3MXksFWjt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=332&publication_year=2011&pages=322-327&author=F.+Xuauthor=H.+Wuauthor=V.+Katritchauthor=G.+W.+Hanauthor=K.+A.+Jacobsonauthor=Z.+G.+Gaoauthor=V.+Cherezovauthor=R.+C.+Stevens&title=Structure+of+an+agonist-bound+human+A2A+adenosine+receptor"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of an Agonist-Bound Human A2A Adenosine Receptor</span></div><div class="casAuthors">Xu, Fei; Wu, Huixian; Katritch, Vsevolod; Han, Gye Won; Jacobson, Kenneth A.; Gao, Zhan-Guo; Cherezov, Vadim; Stevens, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">332</span>
        (<span class="NLM_cas:issue">6027</span>),
    <span class="NLM_cas:pages">322-327</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Activation of G protein-coupled receptors upon agonist binding is a crit. step in the signaling cascade for this family of cell surface proteins.  We report the crystal structure of the A2A adenosine receptor (A2AAR) bound to an agonist UK-432097 at 2.7 angstrom resoln.  Relative to inactive, antagonist-bound A2AAR, the agonist-bound structure displays an outward tilt and rotation of the cytoplasmic half of helix VI, a movement of helix V, and an axial shift of helix III, resembling the changes assocd. with the active-state opsin structure.  Addnl., a seesaw movement of helix VII and a shift of extracellular loop 3 are likely specific to A2AAR and its ligand.  The results define the mol. UK-432097 as a "conformationally selective agonist" capable of receptor stabilization in a specific active-state configuration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMkD1ni9m9GrVg90H21EOLACvtfcHk0lgtPsKzmRYgHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXksFWjt7s%253D&md5=b1e074f38f45c95d4cc3b40378b24923</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1126%2Fscience.1202793&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1202793%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DF.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26aulast%3DGao%26aufirst%3DZ.%2BG.%26aulast%3DCherezov%26aufirst%3DV.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26atitle%3DStructure%2520of%2520an%2520agonist-bound%2520human%2520A2A%2520adenosine%2520receptor%26jtitle%3DScience%26date%3D2011%26volume%3D332%26spage%3D322%26epage%3D327" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref243"><div class="reference"><strong class="refLabel"><a href="#ref243" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit243"><span><span class="NLM_contrib-group">Jiang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Rhee, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yehle, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wess, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, K. A.</span><span> </span><span class="NLM_article-title">Hydrophilic side chains in the third and seventh transmembrane helical domains of human A2A adenosine receptors are required for ligand recognition</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">512</span><span class="NLM_x">–</span> <span class="NLM_lpage">521</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref243/cit243&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=8794889" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref243/cit243&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADyaK28XlslShurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1996&pages=512-521&author=Q.+Jiangauthor=A.+M.+Van+Rheeauthor=J.+Kimauthor=S.+Yehleauthor=J.+Wessauthor=K.+A.+Jacobson&title=Hydrophilic+side+chains+in+the+third+and+seventh+transmembrane+helical+domains+of+human+A2A+adenosine+receptors+are+required+for+ligand+recognition"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit243R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Hydrophilic side chains in the third and seventh transmembrane helical domains of human A2a adenosine receptors are required for ligand recognition</span></div><div class="casAuthors">Jiang, Qiaoling; Van Rhee, Michiel; Kim, Jeongho; Yehle, Susan; Wess, Juergen; Jacobson, Kenneth A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">512-521</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">0026-895X</span>.
    
            (<span class="NLM_cas:orgname">Williams & Wilkins</span>)
        </div><div class="casAbstract">Hydrophilic residues of the G protein-coupled human A2A adenosine receptor that are potentially involved in the binding of the ribose moiety of adenosine were targeted for mutagenesis.  Residues in a T88QSS sequence in the third transmembrane helical domain (TM3) were individually replaced with alanine and other amino acids.  Two addnl. serine residues in TM7 that were previously shown to be involved in ligand binding were mutated to other uncharged hydrophilic amino acids.  The binding affinity of agonist at T88 mutant receptors was greatly diminished, although the receptors were well expressed and bound antagonists similar to the wild-type receptor.  Thus, mutations that are specific for diminishing the affinity of ribose-contg. ligands (i.e., adenosine agonists) have been identified in both TM3 and TM7.  The T88A and T88S mutant receptor fully stimulated adenylyl cyclase, with the dose-responsive curves to CGS 21680 highly shifted to the right.  A Q89A mutant gained affinity for all agonist and antagonist ligands examd. in binding and functional assays.  Q89 likely plays an indirect role in ligand binding.  S90A, S91A, and S277C mutant receptors displayed only moderate changes in ligand affinity.  A S281N mutant gained affinity for all adenosine derivs. (agonists), but antagonist affinity was generally diminished, with the exception of a novel tetrahydrobenzothiophenone deriv.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojbNx_bMR3jLVg90H21EOLACvtfcHk0lgtPsKzmRYgHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XlslShurg%253D&md5=08a775d55751171209ecfd1f3c1e7d93</span></div><a href="/servlet/linkout?suffix=cit243&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DQ.%26aulast%3DVan%2BRhee%26aufirst%3DA.%2BM.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DYehle%26aufirst%3DS.%26aulast%3DWess%26aufirst%3DJ.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26atitle%3DHydrophilic%2520side%2520chains%2520in%2520the%2520third%2520and%2520seventh%2520transmembrane%2520helical%2520domains%2520of%2520human%2520A2A%2520adenosine%2520receptors%2520are%2520required%2520for%2520ligand%2520recognition%26jtitle%3DMol.%2520Pharmacol.%26date%3D1996%26volume%3D50%26spage%3D512%26epage%3D521" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref244"><div class="reference"><strong class="refLabel"><a href="#ref244" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit244"><span><span class="NLM_contrib-group">Kim, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wess, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Rhee, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schoneberg, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, K. A.</span><span> </span><span class="NLM_article-title">Site-directed mutagenesis identifies residues involved in ligand recognition in the human A2a adenosine receptor</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">270</span><span class="NLM_x">, </span> <span class="NLM_fpage">13987</span><span class="NLM_x">–</span> <span class="NLM_lpage">13997</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=270&publication_year=1995&pages=13987-13997&author=J.+Kimauthor=J.+Wessauthor=A.+M.+van+Rheeauthor=T.+Schonebergauthor=K.+A.+Jacobson&title=Site-directed+mutagenesis+identifies+residues+involved+in+ligand+recognition+in+the+human+A2a+adenosine+receptor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit244&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DWess%26aufirst%3DJ.%26aulast%3Dvan%2BRhee%26aufirst%3DA.%2BM.%26aulast%3DSchoneberg%26aufirst%3DT.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26atitle%3DSite-directed%2520mutagenesis%2520identifies%2520residues%2520involved%2520in%2520ligand%2520recognition%2520in%2520the%2520human%2520A2a%2520adenosine%2520receptor%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1995%26volume%3D270%26spage%3D13987%26epage%3D13997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref245"><div class="reference"><strong class="refLabel"><a href="#ref245" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit245"><span><span class="NLM_contrib-group">Kim, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glashofer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yehle, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wess, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, K. A.</span><span> </span><span class="NLM_article-title">Glutamate residues in the second extracellular loop of the human A2a adenosine receptor are required for ligand recognition</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">683</span><span class="NLM_x">–</span> <span class="NLM_lpage">691</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref245/cit245&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=8609897" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref245/cit245&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADyaK28XisVSksL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=1996&pages=683-691&author=J.+Kimauthor=Q.+Jiangauthor=M.+Glashoferauthor=S.+Yehleauthor=J.+Wessauthor=K.+A.+Jacobson&title=Glutamate+residues+in+the+second+extracellular+loop+of+the+human+A2a+adenosine+receptor+are+required+for+ligand+recognition"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit245R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Glutamate residues in the second extracellular loop of the human A2a adenosine receptor are required for ligand recognition</span></div><div class="casAuthors">Kim, Jeongho; Jiang, Qiaoling; Glashofer, Marc; Yehle, Susan; Wess, Juergen; Jacobson, Kenneth A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">683-91</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">0026-895X</span>.
    
            (<span class="NLM_cas:orgname">Williams & Wilkins</span>)
        </div><div class="casAbstract">The A2a adenosine receptor, a member of the G protein-coupled receptor family, is important in the regulation of dopaminergic pathways of the brain and in platelet and cardiovascular functions.  In this study, the role of extracellular loops in ligand binding to the human A2a receptor was explored through site-directed mutagenesis.  Four glutamate/aspartate residues (Glu151, Glu161, Glu169, and Asp170) in the second extracellular loop (E2) and a cysteine residue (Cys262) in the third extracellular loop (E3) were individually replaced with alanine and other amino acids.  A proline residue (Pro173) in E2 was mutated to arginine, the homologous amino acid in A3 receptors.  The binding properties of the resultant mutant receptors were detd. in transfected COS-7 cells.  The mutant receptors were tagged at their amino terminus with a hemagglutinin epitope, thus allowing their detection in the plasma membrane with immunol. techniques.  High affinity specific binding of [3H]2-[4-[(2-carboxyethyl)phenyl]ethyl-amino]-5'-N-ethylcarboxamidoadenosine (15 nM) and [3H]8-[4-[[[[(2-aminoethyl)-amino]carbonyl]methyl]oxy]phenyl]-1,3- dipropylxanthine (4 nM), an A2a agonist and antagonist, resp., was not obsd. with four of the mutant receptors, E151A, E151Q, E151D, and E169A, although they were well expressed at the cell surface.  The E151A and E169A mutant receptors showed nearly full stimulation of adenylyl cyclase at ∼103-fold higher concns. of 2-[4-[(2-carboxyethyl)phenyl]ethyl-amino]-5'-N-ethylcarboxamidoadenosine.  The E161A mutant receptor showed an increase in affinity for the nonxanthine adenosine antagonist 9-chloro-2-(furyl)[1,2,4]triazolo[1,5-c]quinazolin-5-amine (6-fold) but not for other ligands.  An E169Q mutant gained affinity (5-22-fold) for adenosine derivs. (agonists) substituted at N6 but not at C2 or C5' positions.  Mutant receptors D170K and P173R were similar to wild-type receptors in binding of both agonist and antagonist radioligands.  A C262G mutant also resembled the wild-type receptor in radioligand binding, indicating that a potential disulfide bridge with another cysteine residue in proximity is not required for the structural integrity of the receptor.  The authors' data suggest that certain amino acids in the second extracellular loop may be directly or indirectly involved in ligand binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxzW8wJBN6PLVg90H21EOLACvtfcHk0lhsNkhw5QIxNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XisVSksL0%253D&md5=f5e3742c67788d6c637cc97734b9614f</span></div><a href="/servlet/linkout?suffix=cit245&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DQ.%26aulast%3DGlashofer%26aufirst%3DM.%26aulast%3DYehle%26aufirst%3DS.%26aulast%3DWess%26aufirst%3DJ.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26atitle%3DGlutamate%2520residues%2520in%2520the%2520second%2520extracellular%2520loop%2520of%2520the%2520human%2520A2a%2520adenosine%2520receptor%2520are%2520required%2520for%2520ligand%2520recognition%26jtitle%3DMol.%2520Pharmacol.%26date%3D1996%26volume%3D49%26spage%3D683%26epage%3D691" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Moro, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, Z. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spalluto, G.</span><span> </span><span class="NLM_article-title">Progress in the pursuit of therapeutic adenosine receptor antagonists</span> <span class="citation_source-journal">Med. Res. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">131</span><span class="NLM_x">–</span> <span class="NLM_lpage">159</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1002%2Fmed.20048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=16380972" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD28XislOksLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2006&pages=131-159&author=S.+Moroauthor=Z.+G.+Gaoauthor=K.+A.+Jacobsonauthor=G.+Spalluto&title=Progress+in+the+pursuit+of+therapeutic+adenosine+receptor+antagonists"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Progress in the pursuit of therapeutic Adenosine receptor antagonists</span></div><div class="casAuthors">Moro, Stefano; Gao, Zhan-Guo; Jacobson, Kenneth A.; Spalluto, Giampiero</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">131-159</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Ever since the discovery of the hypotensive and bradycardiac effects of adenosine, adenosine receptors continue to represent promising drug targets.  First, this is due to the fact that the receptors are expressed in a large variety of tissues.  In particular, the actions of adenosine (or methylxanthine antagonists) in the central nervous system, in the circulation, on immune cells, and on other tissues can be beneficial in certain disorders.  Second, there exists a large no. of ligands, which have been generated by introducing several modifications in the structure of the lead compds. (adenosine and methylxanthine), some of them highly specific.  Four adenosine receptor subtypes (A1, A2A, A2B, and A3) have been cloned and pharmacol. characterized, all of which are G protein-coupled receptors.  Adenosine receptors can be distinguished according to their preferred mechanism of signal transduction: A1 and A3 receptors interact with pertussis toxin-sensitive G proteins of the Gi and Go family; the canonical signaling mechanism of the A2A and of the A2B receptors is stimulation of adenylyl cyclase via Gs proteins.  In addn. to the coupling to adenylyl cyclase, all four subtypes may pos. couple to phospholipase C via different G protein subunits.  The development of new ligands, in particular, potent and selective antagonists, for all subtypes of adenosine receptors has so far been directed by traditional medicinal chem.  The availability of genetic information promises to facilitate understanding of the drug-receptor interaction leading to the rational design of a potentially therapeutically important class of drugs.  Moreover, mol. modeling may further rationalize obsd. interactions between the receptors and their ligands.  In this review, we will summarize the most relevant progress in developing new therapeutic adenosine receptor antagonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrX73C53aX74rVg90H21EOLACvtfcHk0lhsNkhw5QIxNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XislOksLw%253D&md5=48f7a603f3c4044b2c48aba13a031e07</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1002%2Fmed.20048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.20048%26sid%3Dliteratum%253Aachs%26aulast%3DMoro%26aufirst%3DS.%26aulast%3DGao%26aufirst%3DZ.%2BG.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26aulast%3DSpalluto%26aufirst%3DG.%26atitle%3DProgress%2520in%2520the%2520pursuit%2520of%2520therapeutic%2520adenosine%2520receptor%2520antagonists%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2006%26volume%3D26%26spage%3D131%26epage%3D159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Strader, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sigal, I. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Candelore, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rands, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hill, W. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dixon, R. A.</span><span> </span><span class="NLM_article-title">Conserved aspartic acid residues 79 and 113 of the beta-adrenergic receptor have different roles in receptor function</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">263</span><span class="NLM_x">, </span> <span class="NLM_fpage">10267</span><span class="NLM_x">–</span> <span class="NLM_lpage">10271</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=263&publication_year=1988&pages=10267-10271&author=C.+D.+Straderauthor=I.+S.+Sigalauthor=M.+R.+Candeloreauthor=E.+Randsauthor=W.+S.+Hillauthor=R.+A.+Dixon&title=Conserved+aspartic+acid+residues+79+and+113+of+the+beta-adrenergic+receptor+have+different+roles+in+receptor+function"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DStrader%26aufirst%3DC.%2BD.%26aulast%3DSigal%26aufirst%3DI.%2BS.%26aulast%3DCandelore%26aufirst%3DM.%2BR.%26aulast%3DRands%26aufirst%3DE.%26aulast%3DHill%26aufirst%3DW.%2BS.%26aulast%3DDixon%26aufirst%3DR.%2BA.%26atitle%3DConserved%2520aspartic%2520acid%2520residues%252079%2520and%2520113%2520of%2520the%2520beta-adrenergic%2520receptor%2520have%2520different%2520roles%2520in%2520receptor%2520function%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1988%26volume%3D263%26spage%3D10267%26epage%3D10271" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Strader, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sigal, I. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Register, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Candelore, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rands, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dixon, R. A.</span><span> </span><span class="NLM_article-title">Identification of residues required for ligand binding to the beta-adrenergic receptor</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1987</span><span class="NLM_x">, </span> <span class="NLM_volume">84</span><span class="NLM_x">, </span> <span class="NLM_fpage">4384</span><span class="NLM_x">–</span> <span class="NLM_lpage">4388</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=1987&pages=4384-4388&author=C.+D.+Straderauthor=I.+S.+Sigalauthor=R.+B.+Registerauthor=M.+R.+Candeloreauthor=E.+Randsauthor=R.+A.+Dixon&title=Identification+of+residues+required+for+ligand+binding+to+the+beta-adrenergic+receptor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DStrader%26aufirst%3DC.%2BD.%26aulast%3DSigal%26aufirst%3DI.%2BS.%26aulast%3DRegister%26aufirst%3DR.%2BB.%26aulast%3DCandelore%26aufirst%3DM.%2BR.%26aulast%3DRands%26aufirst%3DE.%26aulast%3DDixon%26aufirst%3DR.%2BA.%26atitle%3DIdentification%2520of%2520residues%2520required%2520for%2520ligand%2520binding%2520to%2520the%2520beta-adrenergic%2520receptor%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1987%26volume%3D84%26spage%3D4384%26epage%3D4388" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref249"><div class="reference"><strong class="refLabel"><a href="#ref249" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit249"><span><span class="NLM_contrib-group">Liapakis, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ballesteros, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Papachristou, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Javitch, J. A.</span><span> </span><span class="NLM_article-title">The forgotten serine. A critical role for Ser-2035.42 in ligand binding to and activation of the beta 2-adrenergic receptor</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">275</span><span class="NLM_x">, </span> <span class="NLM_fpage">37779</span><span class="NLM_x">–</span> <span class="NLM_lpage">37788</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=275&publication_year=2000&pages=37779-37788&author=G.+Liapakisauthor=J.+A.+Ballesterosauthor=S.+Papachristouauthor=W.+C.+Chanauthor=X.+Chenauthor=J.+A.+Javitch&title=The+forgotten+serine.+A+critical+role+for+Ser-2035.42+in+ligand+binding+to+and+activation+of+the+beta+2-adrenergic+receptor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit249&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLiapakis%26aufirst%3DG.%26aulast%3DBallesteros%26aufirst%3DJ.%2BA.%26aulast%3DPapachristou%26aufirst%3DS.%26aulast%3DChan%26aufirst%3DW.%2BC.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DJavitch%26aufirst%3DJ.%2BA.%26atitle%3DThe%2520forgotten%2520serine.%2520A%2520critical%2520role%2520for%2520Ser-2035.42%2520in%2520ligand%2520binding%2520to%2520and%2520activation%2520of%2520the%2520beta%25202-adrenergic%2520receptor%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2000%26volume%3D275%26spage%3D37779%26epage%3D37788" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref250"><div class="reference"><strong class="refLabel"><a href="#ref250" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit250"><span><span class="NLM_contrib-group">Suryanarayana, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, B. K.</span><span> </span><span class="NLM_article-title">Amino acid substitutions at position 312 in the seventh hydrophobic segment of the beta 2-adrenergic receptor modify ligand-binding specificity</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">111</span><span class="NLM_x">–</span> <span class="NLM_lpage">114</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref250/cit250&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=8101966" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref250/cit250&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADyaK3sXlsFahtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=1993&pages=111-114&author=S.+Suryanarayanaauthor=B.+K.+Kobilka&title=Amino+acid+substitutions+at+position+312+in+the+seventh+hydrophobic+segment+of+the+beta+2-adrenergic+receptor+modify+ligand-binding+specificity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit250R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Amino acid substitutions at position 312 in the seventh hydrophobic segment of the β2-adrenergic receptor modify ligand-binding specificity</span></div><div class="casAuthors">Suryanarayana, Sankuratri; Kobilka, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">111-14</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">0026-895X</span>.
    </div><div class="casAbstract">The authors previously reported that Asn312 of the β2-adrenergic receptor and Asn385 in the homologs position in the 5-HT1A receptor are important for binding to a class of β-adrenergic receptor antagonists including propranolol and alprenolol.  The authors proposed that the asparagine may be forming a H bond with the phenoxy O common to these ligands.  To further test this hypothesis the authors made alanine, threonine, phenylalanine, and glutamine substitutions at position 312 in the β2-adrenergic receptor.  The authors obsd. that substitution with amino acids that permit formation of H bonds (threonine and glutamine) supported binding to aryloxyalkylamines, whereas substitution with amino acids that cannot form H bonds (alanine and phenylalanine) did not permit binding to these compds.  The authors were surprised to find that 2 of these substitutions led to an increase in affinity for α-adrenergic ligands.  Substitution with glutamine and threonine at position 312 led to a 11-15-fold increase in affinity for yohimbine and enabled p-aminoclonidine to act as an agonist.  These results further emphasize the role of position 312 in the formation of the ligand binding site for multiple ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGre0AZHzXQBV7Vg90H21EOLACvtfcHk0lipiBX9IwN_nA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXlsFahtb0%253D&md5=0e12c9a08aac895567384cb35632b78d</span></div><a href="/servlet/linkout?suffix=cit250&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSuryanarayana%26aufirst%3DS.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26atitle%3DAmino%2520acid%2520substitutions%2520at%2520position%2520312%2520in%2520the%2520seventh%2520hydrophobic%2520segment%2520of%2520the%2520beta%25202-adrenergic%2520receptor%2520modify%2520ligand-binding%2520specificity%26jtitle%3DMol.%2520Pharmacol.%26date%3D1993%26volume%3D44%26spage%3D111%26epage%3D114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Strader, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Candelore, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hill, W. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sigal, I. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dixon, R. A.</span><span> </span><span class="NLM_article-title">Identification of two serine residues involved in agonist activation of the beta-adrenergic receptor</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1989</span><span class="NLM_x">, </span> <span class="NLM_volume">264</span><span class="NLM_x">, </span> <span class="NLM_fpage">13572</span><span class="NLM_x">–</span> <span class="NLM_lpage">13578</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=2547766" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADyaL1MXlsVOht78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=264&publication_year=1989&pages=13572-13578&author=C.+D.+Straderauthor=M.+R.+Candeloreauthor=W.+S.+Hillauthor=I.+S.+Sigalauthor=R.+A.+Dixon&title=Identification+of+two+serine+residues+involved+in+agonist+activation+of+the+beta-adrenergic+receptor"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of two serine residues involved in agonist activation of the β-adrenergic receptor</span></div><div class="casAuthors">Strader, Catherine D.; Candelore, Mari Rios; Hill, Wendy S.; Sigal, Irving S.; Dixon, Richard A. F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">264</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">13572-8</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    </div><div class="casAbstract">Pharmacophore mapping of the ligand-binding domain of the β-adrenergic receptor has revealed specific mol. interactions which are important for agonist and antagonist binding to the receptor.  Previous site-directed mutagenesis expts. have demonstrated that the binding of amine agonists and antagonists to the receptor involves an interaction between the amine group of the ligand and the carboxylate side chain of Asp113 in the 3rd hydrophobic domain of the receptor.  Two serine residues, at positions 204 and 207 in the 5th hydrophobic domain of the β-adrenergic receptor, which are crit. for agonist binding and activation of the receptor have now been identified.  These serine residues are conserved with G-protein-coupled receptors which bind catecholamine agonists, but not with receptors whose endogenous ligands do not have the catechol moiety.  Removal of the OH side chain from either Ser204 or Ser207 by substitution of the serine residue with an alanine attenuates the activity of catecholamine agonists at the receptor.  The effects of these mutations on agonist activity are mimicked selectively by the removal of the catechol OH moieties from the arom. ring of the agonist.  The data suggest that the interaction of catecholamine agonists with the β-adrenergic receptor involves 2 H bonds, one between the OH side chain of Ser204 and the meta-OH group of the ligand and a second between the OH side chain of Ser207 and the para-OH group of the ligand.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEA_QZmFYKc7Vg90H21EOLACvtfcHk0lipiBX9IwN_nA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXlsVOht78%253D&md5=0e8e69f212b17a3c54a024d7497d79a7</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DStrader%26aufirst%3DC.%2BD.%26aulast%3DCandelore%26aufirst%3DM.%2BR.%26aulast%3DHill%26aufirst%3DW.%2BS.%26aulast%3DSigal%26aufirst%3DI.%2BS.%26aulast%3DDixon%26aufirst%3DR.%2BA.%26atitle%3DIdentification%2520of%2520two%2520serine%2520residues%2520involved%2520in%2520agonist%2520activation%2520of%2520the%2520beta-adrenergic%2520receptor%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1989%26volume%3D264%26spage%3D13572%26epage%3D13578" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Wieland, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zuurmond, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krasel, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ijzerman, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohse, M. J.</span><span> </span><span class="NLM_article-title">Involvement of Asn-293 in stereospecific agonist recognition and in activation of the beta 2-adrenergic receptor</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">93</span><span class="NLM_x">, </span> <span class="NLM_fpage">9276</span><span class="NLM_x">–</span> <span class="NLM_lpage">9281</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=1996&pages=9276-9281&author=K.+Wielandauthor=H.+M.+Zuurmondauthor=C.+Kraselauthor=A.+P.+Ijzermanauthor=M.+J.+Lohse&title=Involvement+of+Asn-293+in+stereospecific+agonist+recognition+and+in+activation+of+the+beta+2-adrenergic+receptor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWieland%26aufirst%3DK.%26aulast%3DZuurmond%26aufirst%3DH.%2BM.%26aulast%3DKrasel%26aufirst%3DC.%26aulast%3DIjzerman%26aufirst%3DA.%2BP.%26aulast%3DLohse%26aufirst%3DM.%2BJ.%26atitle%3DInvolvement%2520of%2520Asn-293%2520in%2520stereospecific%2520agonist%2520recognition%2520and%2520in%2520activation%2520of%2520the%2520beta%25202-adrenergic%2520receptor%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1996%26volume%3D93%26spage%3D9276%26epage%3D9281" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Kikkawa, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Isogaya, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagao, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurose, H.</span><span> </span><span class="NLM_article-title">The role of the seventh transmembrane region in high affinity binding of a beta 2-selective agonist TA-2005</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">128</span><span class="NLM_x">–</span> <span class="NLM_lpage">134</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1998&pages=128-134&author=H.+Kikkawaauthor=M.+Isogayaauthor=T.+Nagaoauthor=H.+Kurose&title=The+role+of+the+seventh+transmembrane+region+in+high+affinity+binding+of+a+beta+2-selective+agonist+TA-2005"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKikkawa%26aufirst%3DH.%26aulast%3DIsogaya%26aufirst%3DM.%26aulast%3DNagao%26aufirst%3DT.%26aulast%3DKurose%26aufirst%3DH.%26atitle%3DThe%2520role%2520of%2520the%2520seventh%2520transmembrane%2520region%2520in%2520high%2520affinity%2520binding%2520of%2520a%2520beta%25202-selective%2520agonist%2520TA-2005%26jtitle%3DMol.%2520Pharmacol.%26date%3D1998%26volume%3D53%26spage%3D128%26epage%3D134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Pham, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fells, J. I.,  Sr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Osborne, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">North, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Naor, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parrill, A. L.</span><span> </span><span class="NLM_article-title">Molecular recognition in the sphingosine 1-phosphate receptor family</span> <span class="citation_source-journal">J. Mol. Graphics Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">1189</span><span class="NLM_x">–</span> <span class="NLM_lpage">1201</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1016%2Fj.jmgm.2007.11.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=18165127" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlvVWnsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=1189-1201&author=T.+C.+Phamauthor=J.+I.+Fellsauthor=D.+A.+Osborneauthor=E.+J.+Northauthor=M.+M.+Naorauthor=A.+L.+Parrill&title=Molecular+recognition+in+the+sphingosine+1-phosphate+receptor+family"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular recognition in the sphingosine 1-phosphate receptor family</span></div><div class="casAuthors">Pham, Truc-Chi T.; Fells, James I.; Osborne, Daniel A.; North, E. Jeffrey; Naor, Mor M.; Parrill, Abby L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1189-1201</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Computational modeling and its application in ligand screening and ligand receptor interaction studies play important roles in structure-based drug design.  A series of sphingosine 1-phosphate (S1P) receptor ligands with varying potencies and receptor selectivities were docked into homol. models of the S1P1-5 receptors.  These studies provided mol. insights into pharmacol. trends both across the receptor family as well as at single receptors.  This study identifies ligand recognition features that generalize across the S1P receptor family, features unique to the S1P4 and S1P5 receptors, and suggests significant structural differences of the S1P2 receptor.  Docking results reveal a previously unknown sulfur-arom. interaction between the S1P4 C5.44 sulfur atom and the Ph ring of benzimidazole as well as π-π interaction between F3.33 of S1P1,4,5 and arom. ligands.  The findings not only confirm the importance of a cation-π interaction between W4.64 and the ammonium of S1P at S1P4 but also predict the same interaction at S1P5.  S1P receptor models are validated for pharmacophore development including database mining and new ligand discovery and serve as tools for ligand optimization to improve potency and selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpboag1kZ0ncbVg90H21EOLACvtfcHk0lh6SSwQlgG03A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlvVWnsbc%253D&md5=cde18a520b3536b68839196dd2ce989e</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2007.11.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2007.11.001%26sid%3Dliteratum%253Aachs%26aulast%3DPham%26aufirst%3DT.%2BC.%26aulast%3DFells%26aufirst%3DJ.%2BI.%26aulast%3DOsborne%26aufirst%3DD.%2BA.%26aulast%3DNorth%26aufirst%3DE.%2BJ.%26aulast%3DNaor%26aufirst%3DM.%2BM.%26aulast%3DParrill%26aufirst%3DA.%2BL.%26atitle%3DMolecular%2520recognition%2520in%2520the%2520sphingosine%25201-phosphate%2520receptor%2520family%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2008%26volume%3D26%26spage%3D1189%26epage%3D1201" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Parrill, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bautista, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Brocklyn, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lorincz, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischer, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baker, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liliom, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spiegel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tigyi, G.</span><span> </span><span class="NLM_article-title">Identification of Edg1 receptor residues that recognize sphingosine 1-phosphate</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">275</span><span class="NLM_x">, </span> <span class="NLM_fpage">39379</span><span class="NLM_x">–</span> <span class="NLM_lpage">39384</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1074%2Fjbc.M007680200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=10982820" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD3cXptFSrsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=275&publication_year=2000&pages=39379-39384&author=A.+L.+Parrillauthor=D.+Wangauthor=D.+L.+Bautistaauthor=J.+R.+Van+Brocklynauthor=Z.+Lorinczauthor=D.+J.+Fischerauthor=D.+L.+Bakerauthor=K.+Liliomauthor=S.+Spiegelauthor=G.+Tigyi&title=Identification+of+Edg1+receptor+residues+that+recognize+sphingosine+1-phosphate"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Edg1 receptor residues that recognize sphingosine 1-phosphate</span></div><div class="casAuthors">Parrill, Abby L.; Wang, De-An; Bautista, Debra L.; Van Brocklyn, James R.; Lorincz, Zsolt; Fischer, David J.; Baker, Daniel L.; Liliom, Karoly; Spiegel, Sarah; Tigyi, Gabor</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">275</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">39379-39384</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Originating from its DNA sequence, a computational model of the Edg1 receptor has been developed that predicts crit. interactions with its ligand, sphingosine 1-phosphate.  The basic amino acids Arg120 and Arg292 ion pair with the phosphate, whereas the acidic Glu121 residue ion pairs with the ammonium moiety of sphingosine 1-phosphate.  The requirement of these interactions for specific ligand recognition has been confirmed through examn. of site-directed mutants by radioligand binding, ligand-induced [35S]GTPγS binding, and receptor internalization assays.  These ion-pairing interactions explain the ligand specificity of the Edg1 receptor and provide insight into ligand specificity differences within the Edg receptor family.  This computational map of the ligand binding pocket provides information necessary for understanding the mol. pharmacol. of this receptor, thus underlining the potential of the computational method in predicting ligand-receptor interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8bLS2jWZ-dbVg90H21EOLACvtfcHk0lh6SSwQlgG03A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXptFSrsbg%253D&md5=2ff5a71db88fab6ff901d4c07e02f26c</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M007680200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M007680200%26sid%3Dliteratum%253Aachs%26aulast%3DParrill%26aufirst%3DA.%2BL.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DBautista%26aufirst%3DD.%2BL.%26aulast%3DVan%2BBrocklyn%26aufirst%3DJ.%2BR.%26aulast%3DLorincz%26aufirst%3DZ.%26aulast%3DFischer%26aufirst%3DD.%2BJ.%26aulast%3DBaker%26aufirst%3DD.%2BL.%26aulast%3DLiliom%26aufirst%3DK.%26aulast%3DSpiegel%26aufirst%3DS.%26aulast%3DTigyi%26aufirst%3DG.%26atitle%3DIdentification%2520of%2520Edg1%2520receptor%2520residues%2520that%2520recognize%2520sphingosine%25201-phosphate%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2000%26volume%3D275%26spage%3D39379%26epage%3D39384" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Wang, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lorincz, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bautista, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liliom, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tigyi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parrill, A. L.</span><span> </span><span class="NLM_article-title">A single amino acid determines lysophospholipid specificity of the S1P1 (EDG1) and LPA1 (EDG2) phospholipid growth factor receptors</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">276</span><span class="NLM_x">, </span> <span class="NLM_fpage">49213</span><span class="NLM_x">–</span> <span class="NLM_lpage">49220</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=2001&pages=49213-49220&author=D.+A.+Wangauthor=Z.+Lorinczauthor=D.+L.+Bautistaauthor=K.+Liliomauthor=G.+Tigyiauthor=A.+L.+Parrill&title=A+single+amino+acid+determines+lysophospholipid+specificity+of+the+S1P1+%28EDG1%29+and+LPA1+%28EDG2%29+phospholipid+growth+factor+receptors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DD.%2BA.%26aulast%3DLorincz%26aufirst%3DZ.%26aulast%3DBautista%26aufirst%3DD.%2BL.%26aulast%3DLiliom%26aufirst%3DK.%26aulast%3DTigyi%26aufirst%3DG.%26aulast%3DParrill%26aufirst%3DA.%2BL.%26atitle%3DA%2520single%2520amino%2520acid%2520determines%2520lysophospholipid%2520specificity%2520of%2520the%2520S1P1%2520%2528EDG1%2529%2520and%2520LPA1%2520%2528EDG2%2529%2520phospholipid%2520growth%2520factor%2520receptors%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2001%26volume%3D276%26spage%3D49213%26epage%3D49220" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Surgand, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodrigo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kellenberger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">A chemogenomic analysis of the transmembrane binding cavity of human G-protein-coupled receptors</span> <span class="citation_source-journal">Proteins</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">62</span><span class="NLM_x">, </span> <span class="NLM_fpage">509</span><span class="NLM_x">–</span> <span class="NLM_lpage">538</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1002%2Fprot.20768" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=16294340" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A280%3ADC%252BD2MnovFyrtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=509-538&author=J.+S.+Surgandauthor=J.+Rodrigoauthor=E.+Kellenbergerauthor=D.+Rognan&title=A+chemogenomic+analysis+of+the+transmembrane+binding+cavity+of+human+G-protein-coupled+receptors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">A chemogenomic analysis of the transmembrane binding cavity of human G-protein-coupled receptors</span></div><div class="casAuthors">Surgand Jean-Sebastien; Rodrigo Jordi; Kellenberger Esther; Rognan Didier</div><div class="citationInfo"><span class="NLM_cas:title">Proteins</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">509-38</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The amino acid sequences of 369 human nonolfactory G-protein-coupled receptors (GPCRs) have been aligned at the seven transmembrane domain (TM) and used to extract the nature of 30 critical residues supposed--from the X-ray structure of bovine rhodopsin bound to retinal--to line the TM binding cavity of ground-state receptors.  Interestingly, the clustering of human GPCRs from these 30 residues mirrors the recently described phylogenetic tree of full-sequence human GPCRs (Fredriksson et al., Mol Pharmacol 2003;63:1256-1272) with few exceptions.  A TM cavity could be found for all investigated GPCRs with physicochemical properties matching that of their cognate ligands.  The current approach allows a very fast comparison of most human GPCRs from the focused perspective of the predicted TM cavity and permits to easily detect key residues that drive ligand selectivity or promiscuity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTH-6l-CmE2uXWEEzDrymxmfW6udTcc2eY1IeQm_OpiWLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2MnovFyrtw%253D%253D&md5=5f60612975dbffa52e440cd2d7ab080c</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1002%2Fprot.20768&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.20768%26sid%3Dliteratum%253Aachs%26aulast%3DSurgand%26aufirst%3DJ.%2BS.%26aulast%3DRodrigo%26aufirst%3DJ.%26aulast%3DKellenberger%26aufirst%3DE.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DA%2520chemogenomic%2520analysis%2520of%2520the%2520transmembrane%2520binding%2520cavity%2520of%2520human%2520G-protein-coupled%2520receptors%26jtitle%3DProteins%26date%3D2006%26volume%3D62%26spage%3D509%26epage%3D538" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">van Loenen, P. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Graaf, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verzijl, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leurs, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peters, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alewijnse, A. E.</span><span> </span><span class="NLM_article-title">Agonist-dependent effects of mutations in the sphingosine-1-phosphate type 1 receptor</span> <span class="citation_source-journal">Eur. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">667</span><span class="NLM_x">, </span> <span class="NLM_fpage">105</span><span class="NLM_x">–</span> <span class="NLM_lpage">112</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1016%2Fj.ejphar.2011.05.071" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=21663738" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVGltrnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=667&publication_year=2011&pages=105-112&author=P.+B.+van+Loenenauthor=C.+de+Graafauthor=D.+Verzijlauthor=R.+Leursauthor=D.+Rognanauthor=S.+L.+Petersauthor=A.+E.+Alewijnse&title=Agonist-dependent+effects+of+mutations+in+the+sphingosine-1-phosphate+type+1+receptor"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Agonist-dependent effects of mutations in the sphingosine-1-phosphate type 1 receptor</span></div><div class="casAuthors">van Loenen, Pieter B.; de Graaf, Chris; Verzijl, Dennis; Leurs, Rob; Rognan, Didier; Peters, Stephan L. M.; Alewijnse, Astrid E.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">667</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">105-112</span>CODEN:
                <span class="NLM_cas:coden">EJPHAZ</span>;
        ISSN:<span class="NLM_cas:issn">0014-2999</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The sphingosine-1-phosphate type 1 (S1P1) receptor is a new target in the treatment of auto-immune diseases as evidenced by the recent approval of FTY720 (Fingolimod).  The ligand-binding pocket of the S1P1 receptor was generally characterized but detailed insight into ligand-specific differences is still lacking.  The aim of the current study is to det. differences in ligand-induced S1P1 receptor activation using an in silico guided site-directed mutagenesis strategy.  S1P1 mutant receptors (modifications of residues Y982.57, R1203.28, F1253.33) were probed with a chem. diverse set of S1P1 agonists (S1P, dihydro-S1P (dhS1P), R-, S- and racemic FTY720-P, VPC24191, SEW2871).  Mutation of the R3.28 residue generally results in a redn. of the potency of all ligands although the synthetic ligands including FTY720-P are less sensitive to these mutations.  The Y2.57F mutation does not affect the potency of any of the ligands tested, but for all ligands except FTY720-P a significant decrease in potency is obsd. at the Y2.57A mutant.  The F3.33A mutation significantly decreased the potency of FTY720-P and is detrimental for SEW2871 and VPC24191.  The non-arom. endogenous ligands S1P and dhS1P are less affected by this mutation.  Our in silico guided mutagenesis studies identified new mol. determinants of ligand-induced S1P1 receptor activation: (1) the flexibility of the polar head of the agonist to maintain a tight H-bond network with R3.28 and (2) the ability of the agonist to make arom. π-stacking interactions with F3.33.  Interestingly, FTY720-P has both chem. properties and is the only ligand that can efficiently activate the Y2.57A mutant.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqua3AJCEtTi7Vg90H21EOLACvtfcHk0liC7u_wPgCgsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVGltrnJ&md5=128b2a264a474a637c7c9217aa34d406</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2Fj.ejphar.2011.05.071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejphar.2011.05.071%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BLoenen%26aufirst%3DP.%2BB.%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26aulast%3DVerzijl%26aufirst%3DD.%26aulast%3DLeurs%26aufirst%3DR.%26aulast%3DRognan%26aufirst%3DD.%26aulast%3DPeters%26aufirst%3DS.%2BL.%26aulast%3DAlewijnse%26aufirst%3DA.%2BE.%26atitle%3DAgonist-dependent%2520effects%2520of%2520mutations%2520in%2520the%2520sphingosine-1-phosphate%2520type%25201%2520receptor%26jtitle%3DEur.%2520J.%2520Pharmacol.%26date%3D2011%26volume%3D667%26spage%3D105%26epage%3D112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group">van der Horst, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van der Pijl, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulder-Krieger, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bender, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">IJzerman, A. P.</span><span> </span><span class="NLM_article-title">Substructure-based virtual screening for adenosine A2A receptor ligands</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">2302</span><span class="NLM_x">–</span> <span class="NLM_lpage">2311</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=2302-2311&author=E.+van+der+Horstauthor=R.+van+der+Pijlauthor=T.+Mulder-Kriegerauthor=A.+Benderauthor=A.+P.+IJzerman&title=Substructure-based+virtual+screening+for+adenosine+A2A+receptor+ligands"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2Bder%2BHorst%26aufirst%3DE.%26aulast%3Dvan%2Bder%2BPijl%26aufirst%3DR.%26aulast%3DMulder-Krieger%26aufirst%3DT.%26aulast%3DBender%26aufirst%3DA.%26aulast%3DIJzerman%26aufirst%3DA.%2BP.%26atitle%3DSubstructure-based%2520virtual%2520screening%2520for%2520adenosine%2520A2A%2520receptor%2520ligands%26jtitle%3DChemMedChem%26date%3D2011%26volume%3D6%26spage%3D2302%26epage%3D2311" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group">Sanders, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verhoeven, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Graaf, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roumen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vroling, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nabuurs, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Vlieg, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klomp, J. P.</span><span> </span><span class="NLM_article-title">Snooker: a structure-based pharmacophore generation tool applied to class A GPCRs</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2277</span><span class="NLM_x">–</span> <span class="NLM_lpage">2292</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci200088d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFahur7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=2277-2292&author=M.+P.+Sandersauthor=S.+Verhoevenauthor=C.+De+Graafauthor=L.+Roumenauthor=B.+Vrolingauthor=S.+B.+Nabuursauthor=J.+de+Vliegauthor=J.+P.+Klomp&title=Snooker%3A+a+structure-based+pharmacophore+generation+tool+applied+to+class+A+GPCRs"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Snooker: A Structure-Based Pharmacophore Generation Tool Applied to Class A GPCRs</span></div><div class="casAuthors">Sanders, Marijn P. A.; Verhoeven, Stefan; de Graaf, Chris; Roumen, Luc; Vroling, Bas; Nabuurs, Sander B.; de Vlieg, Jacob; Klomp, Jan P. G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2277-2292</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">G-protein coupled receptors (GPCRs) are important drug targets for various diseases and of major interest to pharmaceutical companies.  The function of individual members of this protein family can be modulated by the binding of small mols. at the extracellular side of the structurally conserved transmembrane (TM) domain.  Here, we present Snooker, a structure-based approach to generate pharmacophore hypotheses for compds. binding to this extracellular side of the TM domain.  Snooker does not require knowledge of ligands, is therefore suitable for apo-proteins, and can be applied to all receptors of the GPCR protein family.  The method comprises the construction of a homol. model of the TM domains and prioritization of residues on the probability of being ligand binding.  Subsequently, protein properties are converted to ligand space, and pharmacophore features are generated at positions where protein ligand interactions are likely.  Using this semiautomated knowledge-driven bioinformatics approach we have created pharmacophore hypotheses for 15 different GPCRs from several different subfamilies.  For the beta-2-adrenergic receptor we show that ligand poses predicted by Snooker pharmacophore hypotheses reproduce literature supported binding modes for ∼75% of compds. fulfilling pharmacophore constraints.  All 15 pharmacophore hypotheses represent interactions with essential residues for ligand binding as obsd. in mutagenesis expts. and compd. selections based on these hypotheses are shown to be target specific.  For 8 out of 15 targets enrichment factors above 10-fold are obsd. in the top 0.5% ranked compds. in a virtual screen.  Addnl., prospectively predicted ligand binding poses in the human dopamine D3 receptor based on Snooker pharmacophores were ranked among the best models in the community wide GPCR dock 2010.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNiE_rJlvsFbVg90H21EOLACvtfcHk0ljqMSt3urJecA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFahur7J&md5=518d4365c9d417acac0645af83a3cd65</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fci200088d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200088d%26sid%3Dliteratum%253Aachs%26aulast%3DSanders%26aufirst%3DM.%2BP.%26aulast%3DVerhoeven%26aufirst%3DS.%26aulast%3DDe%2BGraaf%26aufirst%3DC.%26aulast%3DRoumen%26aufirst%3DL.%26aulast%3DVroling%26aufirst%3DB.%26aulast%3DNabuurs%26aufirst%3DS.%2BB.%26aulast%3Dde%2BVlieg%26aufirst%3DJ.%26aulast%3DKlomp%26aufirst%3DJ.%2BP.%26atitle%3DSnooker%253A%2520a%2520structure-based%2520pharmacophore%2520generation%2520tool%2520applied%2520to%2520class%2520A%2520GPCRs%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D2277%26epage%3D2292" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Wawer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Similarity-potency trees: a method to search for SAR information in compound data sets and derive SAR rules</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">1395</span><span class="NLM_x">–</span> <span class="NLM_lpage">409</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci100197b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BC3cXovVCiurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1395-409&author=M.+Wawerauthor=J.+Bajorath&title=Similarity-potency+trees%3A+a+method+to+search+for+SAR+information+in+compound+data+sets+and+derive+SAR+rules"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Similarity-Potency Trees: A Method to Search for SAR Information in Compound Data Sets and Derive SAR Rules</span></div><div class="casAuthors">Wawer, Mathias; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1395-1409</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An intuitive and generally applicable anal. method, termed similarity-potency tree (SPT), is introduced to mine structure-activity relation (SAR) information in compd. data sets of any source.  Only compd. potency values and nearest-neighbor similarity relationships are considered.  Rather than analyzing a data set as a whole, in part overlapping compd. neighborhoods are systematically generated and represented as SPTs.  This local anal. scheme simplifies the evaluation of SAR information and SPTs of high SAR information content are easily identified.  By inspecting only a limited no. of compd. neighborhoods, it is also straightforward to det. whether data sets contain only little or no interpretable SAR information.  Interactive anal. of SPTs is facilitated by reading the trees in two directions, which makes it possible to ext. SAR rules, if available, in a consistent manner.  The simplicity and interpretability of the data structure and the ease of calcn. are characteristic features of this approach.  We apply the methodol. to high-throughput screening and lead optimization data sets, compare the approach to std. clustering techniques, illustrate how SAR rules are derived, and provide some practical guidance how to best utilize the methodol.  The SPT program is made freely available to the scientific community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocIJW9gu2adrVg90H21EOLACvtfcHk0ljqMSt3urJecA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXovVCiurc%253D&md5=7d4609c7ca3a695c0658233ca7867616</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fci100197b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100197b%26sid%3Dliteratum%253Aachs%26aulast%3DWawer%26aufirst%3DM.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DSimilarity-potency%2520trees%253A%2520a%2520method%2520to%2520search%2520for%2520SAR%2520information%2520in%2520compound%2520data%2520sets%2520and%2520derive%2520SAR%2520rules%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D1395%26epage%3D409" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Steffen, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kogej, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tyrchan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engkvist, O.</span><span> </span><span class="NLM_article-title">Comparison of molecular fingerprint methods on the basis of biological profile data</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">338</span><span class="NLM_x">–</span> <span class="NLM_lpage">347</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci800326z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFeitrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=338-347&author=A.+Steffenauthor=T.+Kogejauthor=C.+Tyrchanauthor=O.+Engkvist&title=Comparison+of+molecular+fingerprint+methods+on+the+basis+of+biological+profile+data"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of Molecular Fingerprint Methods on the Basis of Biological Profile Data</span></div><div class="casAuthors">Steffen, Andreas; Kogej, Thierry; Tyrchan, Christian; Engkvist, Ola</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">338-347</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this study we evaluated a set of mol. fingerprint methods with respect to their capability to reproduce similarities in the biol. activity space.  The evaluation presented in this paper is therefore different from many other fingerprint studies, in which the enrichment of active compds. binding to the same target as selected query structures was studied.  Conversely, our data set was extd. from the BioPrint database, which contains uniformly derived biol. activity profiles of mainly marketed drugs for a range of biol. assays relevant for the pharmaceutical industry.  We compared calcd. mol. fingerprint similarity values between all compd. pairs of the data set with the corresponding similarities in the biol. activity space and addnl. analyzed agreements of generated clusterings.  A closer anal. of the compd. pairs with a high biol. activity similarity revealed that fingerprint methods such as CHEMGPS or TRUST4, which describe global features of a mol. such as physicochem. properties and pharmacophore patterns, might be better suited to describe similarity of biol. activity profiles than purely structural fingerprint methods.  It is therefore suggested that the usage of these fingerprint methods could increase the probability of finding mols. with a similar biol. activity profile but yet a different chem. structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA5RyGYCpkULVg90H21EOLACvtfcHk0lh757Lw7775fA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFeitrY%253D&md5=0a296a850c61bea5a656c9e7e5c6df7c</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fci800326z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800326z%26sid%3Dliteratum%253Aachs%26aulast%3DSteffen%26aufirst%3DA.%26aulast%3DKogej%26aufirst%3DT.%26aulast%3DTyrchan%26aufirst%3DC.%26aulast%3DEngkvist%26aufirst%3DO.%26atitle%3DComparison%2520of%2520molecular%2520fingerprint%2520methods%2520on%2520the%2520basis%2520of%2520biological%2520profile%2520data%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D338%26epage%3D347" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref263"><div class="reference"><strong class="refLabel"><a href="#ref263" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit263"><span><span class="NLM_contrib-group">Lee, H. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, W. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, J. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoon, M. H.</span><span> </span><span class="NLM_article-title">Roles of adenosine receptor subtypes on the antinociceptive effect of sildenafil in rat spinal cord</span> <span class="citation_source-journal">Neurosci. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">480</span><span class="NLM_x">, </span> <span class="NLM_fpage">182</span><span class="NLM_x">–</span> <span class="NLM_lpage">185</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=480&publication_year=2010&pages=182-185&author=H.+G.+Leeauthor=W.+M.+Kimauthor=J.+I.+Choiauthor=M.+H.+Yoon&title=Roles+of+adenosine+receptor+subtypes+on+the+antinociceptive+effect+of+sildenafil+in+rat+spinal+cord"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit263&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DH.%2BG.%26aulast%3DKim%26aufirst%3DW.%2BM.%26aulast%3DChoi%26aufirst%3DJ.%2BI.%26aulast%3DYoon%26aufirst%3DM.%2BH.%26atitle%3DRoles%2520of%2520adenosine%2520receptor%2520subtypes%2520on%2520the%2520antinociceptive%2520effect%2520of%2520sildenafil%2520in%2520rat%2520spinal%2520cord%26jtitle%3DNeurosci.%2520Lett.%26date%3D2010%26volume%3D480%26spage%3D182%26epage%3D185" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref264"><div class="reference"><strong class="refLabel"><a href="#ref264" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit264"><span><span class="NLM_contrib-group">Salloum, F. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Das, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kukreja, R. C.</span><span> </span><span class="NLM_article-title">Adenosine A(1) receptor mediates delayed cardioprotective effect of sildenafil in mouse</span> <span class="citation_source-journal">J. Mol. Cell. Cardiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">545</span><span class="NLM_x">–</span> <span class="NLM_lpage">551</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref264/cit264&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1016%2Fj.yjmcc.2007.08.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref264/cit264&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=17920622" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref264/cit264&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1amsLrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2007&pages=545-551&author=F.+N.+Salloumauthor=A.+Dasauthor=C.+S.+Thomasauthor=C.+Yinauthor=R.+C.+Kukreja&title=Adenosine+A%281%29+receptor+mediates+delayed+cardioprotective+effect+of+sildenafil+in+mouse"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit264R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Adenosine A1 receptor mediates delayed cardioprotective effect of sildenafil in mouse</span></div><div class="casAuthors">Salloum, Fadi N.; Das, Anindita; Thomas, Christopher S.; Yin, Chang; Kukreja, Rakesh C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular and Cellular Cardiology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">545-551</span>CODEN:
                <span class="NLM_cas:coden">JMCDAY</span>;
        ISSN:<span class="NLM_cas:issn">0022-2828</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Sildenafil induces powerful cardioprotection against ischemia/reperfusion (I/R) injury.  Since adenosine is known to be a major trigger of ischemic preconditioning, we hypothesized that A1 adenosine receptor (A1AR) activation plays a role in sildenafil-induced cardioprotective signaling.  Adult male C57BL wild-type (WT) mice or their corresponding A1AR knockout (A1AR-KO) mice were treated i.p. with either sildenafil (0.71 mg/kg, equiv. to 50 mg dose for a 70 kg patient) or vol.-matched saline.  The selective A1AR antagonist 8-cyclopentyl-1,3-dipropyxanthine (DPCPX; 0.1 mg/kg, i.p.) was administered 30 min before sildenafil.  The hearts were isolated 24 h later and subjected to 30 min of global ischemia and 1 h of reperfusion in Langendorff mode.  Post-ischemic myocardial infarct size (mean; % of risk area) was reduced in C57BL-WT mice treated with sildenafil (5.6) vs. saline control group (27.3; n = 6/each).  However, sildenafil failed to protect the A1AR-KO hearts (31.6 vs. 32.3 with saline).  Addnl., DPCPX treatment abolished the infarct limiting effect of sildenafil (27.3).  DPCPX alone had no effect on infarct size as compared with the control group.  No significant changes in post-ischemic recovery of left ventricular pressure and heart rate were obsd. in the sildenafil-treated group.  We further examd. the effect of sildenafil in protection against simulated ischemia and reoxygenation injury in adult cardiomyocytes derived from WT and A1AR-KO mice.  WT myocytes treated with sildenafil (1 μM) demonstrated significantly lower trypan blue-pos. necrotic cells.  However, cardiomyocytes derived from A1AR-KO mice or DPCPX-treated WT cells failed to show protection against necrosis with sildenafil.  These results suggest that A1AR activation following treatment with sildenafil plays an integral role in the signaling cascade responsible for delayed protection against global I/R injury.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaEMDtjnKRrrVg90H21EOLACvtfcHk0lh757Lw7775fA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1amsLrN&md5=796f04cf01c42c4e3ea7ccaf56ec45af</span></div><a href="/servlet/linkout?suffix=cit264&amp;dbid=16384&amp;doi=10.1016%2Fj.yjmcc.2007.08.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.yjmcc.2007.08.014%26sid%3Dliteratum%253Aachs%26aulast%3DSalloum%26aufirst%3DF.%2BN.%26aulast%3DDas%26aufirst%3DA.%26aulast%3DThomas%26aufirst%3DC.%2BS.%26aulast%3DYin%26aufirst%3DC.%26aulast%3DKukreja%26aufirst%3DR.%2BC.%26atitle%3DAdenosine%2520A%25281%2529%2520receptor%2520mediates%2520delayed%2520cardioprotective%2520effect%2520of%2520sildenafil%2520in%2520mouse%26jtitle%3DJ.%2520Mol.%2520Cell.%2520Cardiol.%26date%3D2007%26volume%3D43%26spage%3D545%26epage%3D551" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Wang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huai, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cai, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zoraghi, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Francis, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corbin, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robinson, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xin, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ke, H.</span><span> </span><span class="NLM_article-title">Multiple conformations of phosphodiesterase-5: implications for enzyme function and drug development</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">281</span><span class="NLM_x">, </span> <span class="NLM_fpage">21469</span><span class="NLM_x">–</span> <span class="NLM_lpage">21479</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1074%2Fjbc.M512527200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=16735511" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD28Xnt1KjsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=21469-21479&author=H.+Wangauthor=Y.+Liuauthor=Q.+Huaiauthor=J.+Caiauthor=R.+Zoraghiauthor=S.+H.+Francisauthor=J.+D.+Corbinauthor=H.+Robinsonauthor=Z.+Xinauthor=G.+Linauthor=H.+Ke&title=Multiple+conformations+of+phosphodiesterase-5%3A+implications+for+enzyme+function+and+drug+development"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Multiple Conformations of Phosphodiesterase-5: implications for enzyme function and drug development</span></div><div class="casAuthors">Wang, Huanchen; Liu, Yudong; Huai, Qing; Cai, Jiwen; Zoraghi, Roya; Francis, Sharron H.; Corbin, Jackie D.; Robinson, Howard; Xin, Zhongcheng; Lin, Guiting; Ke, Hengming</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">281</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">21469-21479</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Phosphodiesterase-5 (PDE5) is the target for sildenafil, vardenafil, and tadalafil, which are drugs for treatment of erectile dysfunction and pulmonary hypertension.  We report here the crystal structures of a fully active catalytic domain of unliganded PDE5A1 and its complexes with sildenafil or icarisid II.  These structures together with the PDE5A1-isobutyl-1-methylxanthine complex show that the H-loop (residues 660-683) at the active site of PDE5A1 has four different conformations and migrates 7-35 Å upon inhibitor binding.  In addn., the conformation of sildenafil reported herein differs significantly from those in the previous structures of chimerically hybridized or almost inactive PDE5.  Mutagenesis and kinetic analyses confirm that the H-loop is particularly important for substrate recognition and that invariant Gly659, which immediately precedes the H-loop, is crit. for optimal substrate affinity and catalytic activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplF9ohkT9ogLVg90H21EOLACvtfcHk0lgYWSklNo8DLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xnt1KjsLs%253D&md5=3a5ac86273a74da4a559bc6efc647181</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M512527200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M512527200%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DHuai%26aufirst%3DQ.%26aulast%3DCai%26aufirst%3DJ.%26aulast%3DZoraghi%26aufirst%3DR.%26aulast%3DFrancis%26aufirst%3DS.%2BH.%26aulast%3DCorbin%26aufirst%3DJ.%2BD.%26aulast%3DRobinson%26aufirst%3DH.%26aulast%3DXin%26aufirst%3DZ.%26aulast%3DLin%26aufirst%3DG.%26aulast%3DKe%26aufirst%3DH.%26atitle%3DMultiple%2520conformations%2520of%2520phosphodiesterase-5%253A%2520implications%2520for%2520enzyme%2520function%2520and%2520drug%2520development%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D21469%26epage%3D21479" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Gozalbes, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rolland, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicolai, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paugam, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coussy, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horvath, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbosa, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mao, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Revah, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Froloff, N.</span><span> </span><span class="NLM_article-title">QSAR strategy and experimental validation for the development of a GPCR focused library</span> <span class="citation_source-journal">QSAR Comb. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">508</span><span class="NLM_x">–</span> <span class="NLM_lpage">516</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2005&pages=508-516&author=R.+Gozalbesauthor=C.+Rollandauthor=E.+Nicolaiauthor=M.+Paugamauthor=L.+Coussyauthor=D.+Horvathauthor=F.+Barbosaauthor=B.+Maoauthor=F.+Revahauthor=N.+Froloff&title=QSAR+strategy+and+experimental+validation+for+the+development+of+a+GPCR+focused+library"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGozalbes%26aufirst%3DR.%26aulast%3DRolland%26aufirst%3DC.%26aulast%3DNicolai%26aufirst%3DE.%26aulast%3DPaugam%26aufirst%3DM.%26aulast%3DCoussy%26aufirst%3DL.%26aulast%3DHorvath%26aufirst%3DD.%26aulast%3DBarbosa%26aufirst%3DF.%26aulast%3DMao%26aufirst%3DB.%26aulast%3DRevah%26aufirst%3DF.%26aulast%3DFroloff%26aufirst%3DN.%26atitle%3DQSAR%2520strategy%2520and%2520experimental%2520validation%2520for%2520the%2520development%2520of%2520a%2520GPCR%2520focused%2520library%26jtitle%3DQSAR%2520Comb.%2520Sci.%26date%3D2005%26volume%3D24%26spage%3D508%26epage%3D516" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref267"><div class="reference"><strong class="refLabel"><a href="#ref267" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit267"><span><span class="NLM_contrib-group">Goldstein, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P. P.</span><span> </span><span class="NLM_article-title">High-throughput kinase profiling as a platform for drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discov.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">391</span><span class="NLM_x">–</span> <span class="NLM_lpage">397</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=391-397&author=D.+M.+Goldsteinauthor=N.+S.+Grayauthor=P.+P.+Zarrinkar&title=High-throughput+kinase+profiling+as+a+platform+for+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit267&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGoldstein%26aufirst%3DD.%2BM.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DHigh-throughput%2520kinase%2520profiling%2520as%2520a%2520platform%2520for%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discov.%26date%3D2008%26volume%3D7%26spage%3D391%26epage%3D397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">Sanders, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fleuren, W. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verhoeven, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van den Beld, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alkema, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Vlieg, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klomp, J. P.</span><span> </span><span class="NLM_article-title">ss-TEA: entropy based identification of receptor specific ligand binding residues from a multiple sequence alignment of class A GPCRs</span> <span class="citation_source-journal">BMC Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">332</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1186%2F1471-2105-12-332" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=21831265" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFCktbrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=332&author=M.+P.+Sandersauthor=W.+W.+Fleurenauthor=S.+Verhoevenauthor=S.+van+den+Beldauthor=W.+Alkemaauthor=J.+de+Vliegauthor=J.+P.+Klomp&title=ss-TEA%3A+entropy+based+identification+of+receptor+specific+ligand+binding+residues+from+a+multiple+sequence+alignment+of+class+A+GPCRs"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">ss-TEA: entropy based identification of receptor specific ligand binding residues from a multiple sequence alignment of class A GPCRs</span></div><div class="casAuthors">Sanders, Marijn P. A.; Fleuren, Wilco W. M.; Verhoeven, Stefan; van den Beld, Sven; Alkema, Wynand; de Vlieg, Jacob; Klomp, Jan P. G.</div><div class="citationInfo"><span class="NLM_cas:title">BMC Bioinformatics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">332</span>CODEN:
                <span class="NLM_cas:coden">BBMIC4</span>;
        ISSN:<span class="NLM_cas:issn">1471-2105</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: G-protein coupled receptors (GPCRs) are involved in many different physiol. processes and their function can be modulated by small mols. which bind in the transmembrane (TM) domain.  Because of their structural and sequence conservation, the TM domains are often used in bioinformatics approaches to first create a multiple sequence alignment (MSA) and subsequently identify ligand binding positions.  So far methods have been developed to predict the common ligand binding residue positions for class A GPCRs.  Results: Here we present 1) ss-TEA, a method to identify specific ligand binding residue positions for any receptor, predicated on high quality sequence information.  2) The largest MSA of class A non olfactory GPCRs in the public domain consisting of 13324 sequences covering most of the species homologues of the human set of GPCRs.  A set of ligand binding residue positions extd. from literature of 10 different receptors shows that our method has the best ligand binding residue prediction for 9 of these 10 receptors compared to another state-of-the-art method.  Conclusions: The combination of the large multi species alignment and the newly introduced residue selection method ss-TEA can be used to rapidly identify subfamily specific ligand binding residues.  This approach can aid the design of site directed mutagenesis expts., explain receptor function and improve modeling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTGzbW8sSKHLVg90H21EOLACvtfcHk0lgYWSklNo8DLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFCktbrF&md5=1673e238874e269dc59495e20d3241e8</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-12-332&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-12-332%26sid%3Dliteratum%253Aachs%26aulast%3DSanders%26aufirst%3DM.%2BP.%26aulast%3DFleuren%26aufirst%3DW.%2BW.%26aulast%3DVerhoeven%26aufirst%3DS.%26aulast%3Dvan%2Bden%2BBeld%26aufirst%3DS.%26aulast%3DAlkema%26aufirst%3DW.%26aulast%3Dde%2BVlieg%26aufirst%3DJ.%26aulast%3DKlomp%26aufirst%3DJ.%2BP.%26atitle%3Dss-TEA%253A%2520entropy%2520based%2520identification%2520of%2520receptor%2520specific%2520ligand%2520binding%2520residues%2520from%2520a%2520multiple%2520sequence%2520alignment%2520of%2520class%2520A%2520GPCRs%26jtitle%3DBMC%2520Bioinf.%26date%3D2011%26volume%3D12%26spage%3D332" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group">Gasteiger, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rudolph, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sadowski, J.</span><span> </span><span class="NLM_article-title">Automatic generation of 3D-atomic coordinates for organic molecules</span> <span class="citation_source-journal">Tetrahedron Comput. Methodol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">537</span><span class="NLM_x">–</span> <span class="NLM_lpage">547</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1016%2F0898-5529%2890%2990156-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADyaK3sXmtlCn" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1990&pages=537-547&author=J.+Gasteigerauthor=C.+Rudolphauthor=J.+Sadowski&title=Automatic+generation+of+3D-atomic+coordinates+for+organic+molecules"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Automatic generation of 3D atomic coordinates for organic molecules</span></div><div class="casAuthors">Gasteiger, J.; Rudolph, C.; Sadowski, J.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Computer Methodology</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">6c</span>),
    <span class="NLM_cas:pages">537-47</span>CODEN:
                <span class="NLM_cas:coden">TCMTE6</span>;
        ISSN:<span class="NLM_cas:issn">0898-5529</span>.
    </div><div class="casAbstract">A system has been developed that can automatically generate 3-dimensional at. coordinates from the constitution of a mol. as expressed by a connection table.  The program, CORINA, is applicable to the entire range of org. chem.  It can also handle structures that are beyond the scope of some other programs, e.g., macrocyclic and polymacrocyclic mols.  Computation times are short and the results compare favorably with data from x-ray crystallog. and with those of mol. mechanics calcns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrS-kIKXGPzYLVg90H21EOLACvtfcHk0ljjKszTNfBW5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXmtlCn&md5=d5929de163b892f0354ea33a59097059</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2F0898-5529%2890%2990156-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0898-5529%252890%252990156-3%26sid%3Dliteratum%253Aachs%26aulast%3DGasteiger%26aufirst%3DJ.%26aulast%3DRudolph%26aufirst%3DC.%26aulast%3DSadowski%26aufirst%3DJ.%26atitle%3DAutomatic%2520generation%2520of%25203D-atomic%2520coordinates%2520for%2520organic%2520molecules%26jtitle%3DTetrahedron%2520Comput.%2520Methodol.%26date%3D1990%26volume%3D3%26spage%3D537%26epage%3D547" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group">Liu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bai, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ouyang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H.</span><span> </span><span class="NLM_article-title">Cyndi: a multi-objective evolution algorithm based method for bioactive molecular conformational generation</span> <span class="citation_source-journal">BMC Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">101</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1186%2F1471-2105-10-101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=19335906" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A280%3ADC%252BD1MzisFSktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2009&pages=101&author=X.+Liuauthor=F.+Baiauthor=S.+Ouyangauthor=X.+Wangauthor=H.+Liauthor=H.+Jiang&title=Cyndi%3A+a+multi-objective+evolution+algorithm+based+method+for+bioactive+molecular+conformational+generation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Cyndi: a multi-objective evolution algorithm based method for bioactive molecular conformational generation</span></div><div class="casAuthors">Liu Xiaofeng; Bai Fang; Ouyang Sisheng; Wang Xicheng; Li Honglin; Jiang Hualiang</div><div class="citationInfo"><span class="NLM_cas:title">BMC bioinformatics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">101</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Conformation generation is a ubiquitous problem in molecule modelling.  Many applications require sampling the broad molecular conformational space or perceiving the bioactive conformers to ensure success.  Numerous in silico methods have been proposed in an attempt to resolve the problem, ranging from deterministic to non-deterministic and systemic to stochastic ones.  In this work, we described an efficient conformation sampling method named Cyndi, which is based on multi-objective evolution algorithm.  RESULTS:  The conformational perturbation is subjected to evolutionary operation on the genome encoded with dihedral torsions.  Various objectives are designated to render the generated Pareto optimal conformers to be energy-favoured as well as evenly scattered across the conformational space.  An optional objective concerning the degree of molecular extension is added to achieve geometrically extended or compact conformations which have been observed to impact the molecular bioactivity (J Comput -Aided Mol Des 2002, 16: 105-112).  Testing the performance of Cyndi against a test set consisting of 329 small molecules reveals an average minimum RMSD of 0.864 A to corresponding bioactive conformations, indicating Cyndi is highly competitive against other conformation generation methods.  Meanwhile, the high-speed performance (0.49 +/- 0.18 seconds per molecule) renders Cyndi to be a practical toolkit for conformational database preparation and facilitates subsequent pharmacophore mapping or rigid docking.  The copy of precompiled executable of Cyndi and the test set molecules in mol2 format are accessible in Additional file 1.  CONCLUSION:  On the basis of MOEA algorithm, we present a new, highly efficient conformation generation method, Cyndi, and report the results of validation and performance studies comparing with other four methods.  The results reveal that Cyndi is capable of generating geometrically diverse conformers and outperforms other four multiple conformer generators in the case of reproducing the bioactive conformations against 329 structures.  The speed advantage indicates Cyndi is a powerful alternative method for extensive conformational sampling and large-scale conformer database preparation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQs2QpRXYXZbmGmni0r9GoKfW6udTcc2eZ_FSSkNYxg2rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MzisFSktg%253D%253D&md5=eec10b3b9f6c8e32c789d069450065df</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-10-101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-10-101%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DBai%26aufirst%3DF.%26aulast%3DOuyang%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DH.%26atitle%3DCyndi%253A%2520a%2520multi-objective%2520evolution%2520algorithm%2520based%2520method%2520for%2520bioactive%2520molecular%2520conformational%2520generation%26jtitle%3DBMC%2520Bioinf.%26date%3D2009%26volume%3D10%26spage%3D101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">ZINC—a free database of commercially available compounds for virtual screening</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">177</span><span class="NLM_x">–</span> <span class="NLM_lpage">182</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci049714%2B" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVOjt77J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2005&pages=177-182&author=J.+J.+Irwinauthor=B.+K.+Shoichet&title=ZINC%E2%80%94a+free+database+of+commercially+available+compounds+for+virtual+screening"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC - A Free Database of Commercially Available Compounds for Virtual Screening</span></div><div class="casAuthors">Irwin, John J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">177-182</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A crit. barrier to entry into structure-based virtual screening is the lack of a suitable, easy to access database of purchasable compds.  We have therefore prepd. a library of 727 842 mols., each with 3D structure, using catalogs of compds. from vendors (the size of this library continues to grow).  The mols. have been assigned biol. relevant protonation states and are annotated with properties such as mol. wt., calcd. LogP, and no. of rotatable bonds.  Each mol. in the library contains vendor and purchasing information and is ready for docking using a no. of popular docking programs.  Within certain limits, the mols. are prepd. in multiple protonation states and multiple tautomeric forms.  In one format, multiple conformations are available for the mols.  This database is available for free download (http://zinc.docking.org) in several common file formats including SMILES, mol2, 3D SDF, and DOCK flexibase format.  A Web-based query tool incorporating a mol. drawing interface enables the database to be searched and browsed and subsets to be created.  Users can process their own mols. by uploading them to a server.  Our hope is that this database will bring virtual screening libraries to a wide community of structural biologists and medicinal chemists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPhFw5E2XACbVg90H21EOLACvtfcHk0ljjKszTNfBW5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVOjt77J&md5=e3892b7dc8608b17a3e63541a5ed60e6</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1021%2Fci049714%2B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci049714%252B%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DZINC%25E2%2580%2594a%2520free%2520database%2520of%2520commercially%2520available%2520compounds%2520for%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2005%26volume%3D45%26spage%3D177%26epage%3D182" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group">Nijssen, S.; Kok, J. N.</span><span> </span><span class="NLM_article-title">A Quickstart in Frequent Structure Mining Can Make a Difference</span>.  <span class="citation_source-book">Proceedings of the 10th ACM SIGKDD International Conference on Knowledge Discovery and Data Mining, 2004</span>; <span class="NLM_contrib-group">Kohavi, R.; Gehrke, J.; DuMouchel, W.; Ghosh, J.</span>, Eds.; <span class="NLM_publisher-name">ACM</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2004</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">647</span><span class="NLM_x">–</span> <span class="NLM_lpage">652</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1145%2F1014052.1014134" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2004&pages=647-652&author=S.+Nijssen&author=J.+N.+Kokauthor=R.+Kohavi&author=J.+Gehrke&author=W.+DuMouchel&author=J.+Ghosh&title=Proceedings+of+the+10th+ACM+SIGKDD+International+Conference+on+Knowledge+Discovery+and+Data+Mining%2C+2004"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1145%2F1014052.1014134&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1145%252F1014052.1014134%26sid%3Dliteratum%253Aachs%26aulast%3DNijssen%26aufirst%3DS.%26atitle%3DA%2520Quickstart%2520in%2520Frequent%2520Structure%2520Mining%2520Can%2520Make%2520a%2520Difference%26btitle%3DProceedings%2520of%2520the%252010th%2520ACM%2520SIGKDD%2520International%2520Conference%2520on%2520Knowledge%2520Discovery%2520and%2520Data%2520Mining%252C%25202004%26aulast%3DKohavi%26aufirst%3DR.%26pub%3DACM%26date%3D2004%26spage%3D647%26epage%3D652" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group">Del Rio, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbosa, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caporuscio, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mangiatordi, G. F.</span><span> </span><span class="NLM_article-title">CoCoCo: a free suite of multiconformational chemical databases for high-throughput virtual screening purposes</span> <span class="citation_source-journal">Mol. BioSyst.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">2122</span><span class="NLM_x">–</span> <span class="NLM_lpage">2128</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1039%2Fc0mb00039f" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=20694263" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1Kqt7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=2122-2128&author=A.+Del+Rioauthor=A.+J.+Barbosaauthor=F.+Caporuscioauthor=G.+F.+Mangiatordi&title=CoCoCo%3A+a+free+suite+of+multiconformational+chemical+databases+for+high-throughput+virtual+screening+purposes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">CoCoCo: a free suite of multiconformational chemical databases for high-throughput virtual screening purposes</span></div><div class="casAuthors">Del Rio, Alberto; Barbosa, Armenio Jorge Moura; Caporuscio, Fabiana; Mangiatordi, Giuseppe Felice</div><div class="citationInfo"><span class="NLM_cas:title">Molecular BioSystems</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2122-2128</span>CODEN:
                <span class="NLM_cas:coden">MBOIBW</span>;
        ISSN:<span class="NLM_cas:issn">1742-206X</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">In the last few decades, virtual screening has proved to be able to guide the selection of new hit compds. with predefined biol. activity.  However, the usage of these computational techniques is often assocd. with resource- and time-consuming prepn. protocols.  In this work we present Com. Compd. Collection (CoCoCo), a suite of free and ready-to-use chem. databases to help setting up in silico screening projects.  CoCoCo collects mol. structural information of com. compds. from various chem. vendors by merging them in a unique, non-redundant format.  CoCoCo databases are prepd. with transparent and straightforward routines based on state-of-the-art computational tools that introduce comprehensive structural information about tautomers, stereoisomers and conformational states of each compd.  CoCoCo suite is esp. conceived as a set of valuable tools that may help a wide range of researchers who wish to initiate their own project in the field of computational drug design.  CoCoCo suite is available free of charge at the website .</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4OJvTFBFY4bVg90H21EOLACvtfcHk0ljf13NMOfjjCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1Kqt7fL&md5=2cddeafe06fb7afb949dc3600d095ff7</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1039%2Fc0mb00039f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc0mb00039f%26sid%3Dliteratum%253Aachs%26aulast%3DDel%2BRio%26aufirst%3DA.%26aulast%3DBarbosa%26aufirst%3DA.%2BJ.%26aulast%3DCaporuscio%26aufirst%3DF.%26aulast%3DMangiatordi%26aufirst%3DG.%2BF.%26atitle%3DCoCoCo%253A%2520a%2520free%2520suite%2520of%2520multiconformational%2520chemical%2520databases%2520for%2520high-throughput%2520virtual%2520screening%2520purposes%26jtitle%3DMol.%2520BioSyst.%26date%3D2010%26volume%3D6%26spage%3D2122%26epage%3D2128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group">Lipinski, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lombardo, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dominy, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feeney, P. J.</span><span> </span><span class="NLM_article-title">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span> <span class="citation_source-journal">Adv. Drug Delivery Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">3</span><span class="NLM_x">–</span> <span class="NLM_lpage">26</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1016%2FS0169-409X%2800%2900129-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=11259830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2001&pages=3-26&author=C.+A.+Lipinskiauthor=F.+Lombardoauthor=B.+W.+Dominyauthor=P.+J.+Feeney&title=Experimental+and+computational+approaches+to+estimate+solubility+and+permeability+in+drug+discovery+and+development+settings"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span></div><div class="casAuthors">Lipinski, C. A.; Lombardo, F.; Dominy, B. W.; Feeney, P. J.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">3-26</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ireland Ltd.</span>)
        </div><div class="casAbstract">A review with 50 refs.  Exptl. and computational approaches to est. soly. and permeability in discovery and development settings are described.  In the discovery setting 'the rule of 5' predicts that poor absorption or permeation is more likely when there are more than 5 H-bond donors, 10 H-bond acceptors, the mol. wt. (MWT) is greater than 500 and the calcd. Log P (CLogP) is greater than 5 (or MlogP >4.15).  Computational methodol. for the rule-based Moriguchi Log P (MLogP) calcn. is described.  Turbidimetric soly. measurement is described and applied to known drugs.  High throughput screening (HTS) leads tend to have higher MWT and Log P and lower turbidimetric soly. than leads in the pre-HTS era.  In the development setting, soly. calcns. focus on exact value prediction and are difficult because of polymorphism.  Recent work on linear free energy relationships and Log P approaches are critically reviewed.  Useful predictions are possible in closely related analog series when coupled with exptl. thermodn. soly. measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYDqa02g16urVg90H21EOLACvtfcHk0ljf13NMOfjjCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D&md5=c60bb89da68f051c0ee7ac4c0468a0e4</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1016%2FS0169-409X%2800%2900129-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-409X%252800%252900129-0%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DDominy%26aufirst%3DB.%2BW.%26aulast%3DFeeney%26aufirst%3DP.%2BJ.%26atitle%3DExperimental%2520and%2520computational%2520approaches%2520to%2520estimate%2520solubility%2520and%2520permeability%2520in%2520drug%2520discovery%2520and%2520development%2520settings%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2001%26volume%3D46%26spage%3D3%26epage%3D26" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group">Wenlock, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Austin, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barton, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leeson, P. D.</span><span> </span><span class="NLM_article-title">A comparison of physiochemical property profiles of development and marketed oral drugs</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">1250</span><span class="NLM_x">–</span> <span class="NLM_lpage">1256</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm021053p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhslSlsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=1250-1256&author=M.+C.+Wenlockauthor=R.+P.+Austinauthor=P.+Bartonauthor=A.+M.+Davisauthor=P.+D.+Leeson&title=A+comparison+of+physiochemical+property+profiles+of+development+and+marketed+oral+drugs"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">A Comparison of Physiochemical Property Profiles of Development and Marketed Oral Drugs</span></div><div class="casAuthors">Wenlock, Mark C.; Austin, Rupert P.; Barton, Patrick; Davis, Andrew M.; Leeson, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1250-1256</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The process of drug discovery applies rigorous selection pressures.  Marketed oral drugs will generally possess favorable physiochem. properties with respect to absorption, metab., distribution, and clearance.  This paper describes a study in which the distributions of physiochem. properties of oral drugs in different phases of clin. development are compared to those already marketed.  The aim is to identify the trends in physiochem. properties that favor a drug's successful passage through clin. development and on to the market.  Two libraries were created, one of current development oral drugs and one of marketed oral drugs.  Statistical anal. of the two showed that the mean mol. wt. of orally administered drugs in development decreases on passing through each of the different clin. phases and gradually converges toward the mean mol. wt. of marketed oral drugs.  It is also clear that the most lipophilic compds. are being discontinued from development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoF0PksUFJq0bVg90H21EOLACvtfcHk0ljgu--XUP_bjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhslSlsbY%253D&md5=b9809c803a61ec99a1b4171e97952452</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1021%2Fjm021053p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm021053p%26sid%3Dliteratum%253Aachs%26aulast%3DWenlock%26aufirst%3DM.%2BC.%26aulast%3DAustin%26aufirst%3DR.%2BP.%26aulast%3DBarton%26aufirst%3DP.%26aulast%3DDavis%26aufirst%3DA.%2BM.%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26atitle%3DA%2520comparison%2520of%2520physiochemical%2520property%2520profiles%2520of%2520development%2520and%2520marketed%2520oral%2520drugs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D1250%26epage%3D1256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group">Verzijl, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storelli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scholten, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bosch, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinhart, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Streblow, D. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tensen, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fitzsimons, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zaman, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pease, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Esch, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smit, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leurs, R.</span><span> </span><span class="NLM_article-title">Noncompetitive antagonism and inverse agonism as mechanism of action of nonpeptidergic antagonists at primate and rodent CXCR3 chemokine receptors</span> <span class="citation_source-journal">J. Pharmacol. Exp. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">325</span><span class="NLM_x">, </span> <span class="NLM_fpage">544</span><span class="NLM_x">–</span> <span class="NLM_lpage">555</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1124%2Fjpet.107.134783" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=18270317" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlsVOjtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=325&publication_year=2008&pages=544-555&author=D.+Verzijlauthor=S.+Storelliauthor=D.+J.+Scholtenauthor=L.+Boschauthor=T.+A.+Reinhartauthor=D.+N.+Streblowauthor=C.+P.+Tensenauthor=C.+P.+Fitzsimonsauthor=G.+J.+Zamanauthor=J.+E.+Peaseauthor=I.+J.+de+Eschauthor=M.+J.+Smitauthor=R.+Leurs&title=Noncompetitive+antagonism+and+inverse+agonism+as+mechanism+of+action+of+nonpeptidergic+antagonists+at+primate+and+rodent+CXCR3+chemokine+receptors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Noncompetitive antagonism and inverse agonism as mechanism of action of nonpeptidergic antagonists at primate and rodent CXCR3 chemokine receptors</span></div><div class="casAuthors">Verzijl, Dennis; Storelli, Stefania; Scholten, Danny J.; Bosch, Leontien; Reinhart, Todd A.; Streblow, Daniel N.; Tensen, Cornelis P.; Fitzsimons, Carlos P.; Zaman, Guido J. R.; Pease, James E.; de Esch, Iwan J. P.; Smit, Martine J.; Leurs, Rob</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">325</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">544-555</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">The chemokine receptor CXCR3 is involved in various inflammatory diseases, such as rheumatoid arthritis, multiple sclerosis, psoriasis, and allograft rejection in transplantation patients.  The CXCR3 ligands CXCL9, CXCL10, and CXCL11 are expressed at sites of inflammation, and they attract CXCR3-bearing lymphocytes, thus contributing to the inflammatory process.  In this study, we characterize five nonpeptidergic compds. of different chem. classes that block the action of CXCL10 and CXCL11 at the human CXCR3, i.e., the 3H-pyrido[2,3-d]pyrimidin-4-one derivs. N-1R-[3-(4-ethoxy-phenyl)-4-oxo-3,4-dihydro-pyrido[2,3-d]pyrimidin-2-yl]-ethyl-N-pyridin-3-ylmethyl-2-(4-fluoro-3-trifluoromethyl-phenyl)-acetamide (VUF10472/NBI-74330) and N-1R-[3-(4-ethoxy-phenyl)-4-oxo-3,4-dihydro-pyrido[2,3-d]pyrimidin-2-yl]-ethyl-N-pyridin-3-ylmethyl-2-(4-trifluoromethoxy-phenyl)-acetamide (VUF10085/AMG-487), the 3H-quinazolin-4-one decanoic acid {1-[3-(4-cyano-phenyl)-4-oxo-3,4-dihydro-quinazolin-2-yl]-ethyl}-(2-dimethylamino-ethyl)-amide (VUF5834), the imidazolium compd. 1,3-bis-[2-(3,4-dichloro-phenyl)-2-oxo-ethyl]-3H-imidazol-1-ium bromide (VUF10132), and the quaternary ammonium anilide N,N-dimethyl-N-[4-[[[2-(4-methylphenyl)-6,7-dihydro-5H-benzocyclohepten-8-yl]-carbonyl]amino]benzyl] tetrahydro-2H-pyran-4-aminium chloride (TAK-779).  To understand the action of these CXCR3 antagonists in various animal models of disease, the compds. were also tested at rat and mouse CXCR3, as well as at CXCR3 from rhesus macaque, which was cloned and characterized for the first time in this study.  Except for TAK-779, all compds. show slightly lower affinity for rodent CXCR3 than for primate CXCR3.  In addn., we have characterized the mol. mechanism of action of the various antagonists at the human CXCR3 receptor.  All tested compds. act as noncompetitive antagonists at CXCR3.  Moreover, this noncompetitive behavior is accompanied by inverse agonistic properties of all five compds. as detd. on an identified constitutively active mutant of CXCR3, CXCR3 N3.35A.  It is interesting to note that all compds. except TAK-779 act as full inverse agonists at CXCR3 N3.35A.  TAK-779 shows weak partial inverse agonism at CXCR3 N3.35A, and it probably has a different mode of interaction with CXCR3 than the other two classes of small-mol. inverse agonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDC_7W3UH75bVg90H21EOLACvtfcHk0ljgu--XUP_bjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlsVOjtrg%253D&md5=10ee30aafa3473c34897de744139c8e3</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1124%2Fjpet.107.134783&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.107.134783%26sid%3Dliteratum%253Aachs%26aulast%3DVerzijl%26aufirst%3DD.%26aulast%3DStorelli%26aufirst%3DS.%26aulast%3DScholten%26aufirst%3DD.%2BJ.%26aulast%3DBosch%26aufirst%3DL.%26aulast%3DReinhart%26aufirst%3DT.%2BA.%26aulast%3DStreblow%26aufirst%3DD.%2BN.%26aulast%3DTensen%26aufirst%3DC.%2BP.%26aulast%3DFitzsimons%26aufirst%3DC.%2BP.%26aulast%3DZaman%26aufirst%3DG.%2BJ.%26aulast%3DPease%26aufirst%3DJ.%2BE.%26aulast%3Dde%2BEsch%26aufirst%3DI.%2BJ.%26aulast%3DSmit%26aufirst%3DM.%2BJ.%26aulast%3DLeurs%26aufirst%3DR.%26atitle%3DNoncompetitive%2520antagonism%2520and%2520inverse%2520agonism%2520as%2520mechanism%2520of%2520action%2520of%2520nonpeptidergic%2520antagonists%2520at%2520primate%2520and%2520rodent%2520CXCR3%2520chemokine%2520receptors%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2008%26volume%3D325%26spage%3D544%26epage%3D555" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group">van der Lee, M. M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bras, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Koppen, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zaman, G. J. R.</span><span> </span><span class="NLM_article-title">beta-Arrestin recruitment assay for the identification of agonists of the sphingosine 1-phosphate receptor EDG1</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">986</span><span class="NLM_x">–</span> <span class="NLM_lpage">998</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Fjm300280e&amp;key=10.1177%2F1087057108326144" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Fjm300280e&amp;key=19036707" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Fjm300280e&amp;key=1%3ACAS%3A528%3ADC%252BD1MXkslGrsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2008&pages=986-998&author=M.+M.+C.+van+der+Leeauthor=M.+Brasauthor=C.+J.+van+Koppenauthor=G.+J.+R.+Zaman&title=beta-Arrestin+recruitment+assay+for+the+identification+of+agonists+of+the+sphingosine+1-phosphate+receptor+EDG1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">β-arrestin recruitment assay for the identification of agonists of the sphingosine 1-phosphate receptor EDG1</span></div><div class="casAuthors">Van Der Lee, Miranda M. C.; Bras, Maaike; Van Koppen, Chris J.; Zaman, Guido J. R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">986-998</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">β-Arrestin recruitment assays provide a generic assay platform for drug discovery on G-protein-coupled receptors (GPCRs).  The PathHunter assay technol. developed by DiscoveRx (Fremont, CA) uses enzyme fragment complementation of β-galactosidase to measure receptor-β-arrestin proximity by chemiluminescence.  This study describes an agonistic screen on the human endothelial differentiation sphingolipid GPCR 1 (EDG1), also known as S1P1, using PathHunter β-arrestin recruitment technol.  Screening of a collection of 345,052 compds. yielded 2157 agonistic hits.  Only 10 of these compds. showed β-arrestin recruitment activity on a nonrelated receptor, indicating high accuracy and specificity of the assay.  The authors show that receptor activation with ref. agonists can be detected within the same EDG1 PathHunter cell line at the level of β-arrestin recruitment, Gi/o protein-mediated inhibition of cyclic adenosine monophosphate (cAMP), and activation of downstream phosphorylation of extracellular signal-regulated protein kinases.  The degree of β-arrestin recruitment was largely unaffected upon blockade of Gi/o protein signaling with pertussis toxin, whereas kinetic studies demonstrated a lower rate of β-arrestin-receptor assocn.  In contrast, inhibition of cAMP and phosphorylation of extracellular signal-regulated protein kinases were fully Gi/o protein regulated.  The data indicate that the β-arrestin enzyme fragment complementation cell line can be used not only for agonistic screening of GPCRs but also for the identification of "biased ligands" (i.e., compds. that differ in G-protein coupling and β-arrestin-mediated cellular effects).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKfdOoQXzl47Vg90H21EOLACvtfcHk0liDic883NNVLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXkslGrsbY%253D&md5=23c0c7106cbbd7337ca0945f13299b1d</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1177%2F1087057108326144&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057108326144%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2Bder%2BLee%26aufirst%3DM.%2BM.%2BC.%26aulast%3DBras%26aufirst%3DM.%26aulast%3Dvan%2BKoppen%26aufirst%3DC.%2BJ.%26aulast%3DZaman%26aufirst%3DG.%2BJ.%2BR.%26atitle%3Dbeta-Arrestin%2520recruitment%2520assay%2520for%2520the%2520identification%2520of%2520agonists%2520of%2520the%2520sphingosine%25201-phosphate%2520receptor%2520EDG1%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2008%26volume%3D13%26spage%3D986%26epage%3D998" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1GZM" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1GZM','PDB','1GZM'); return false;">PDB: 1GZM</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1L9H" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1L9H','PDB','1L9H'); return false;">PDB: 1L9H</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2RH1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2RH1','PDB','2RH1'); return false;">PDB: 2RH1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VT4" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VT4','PDB','2VT4'); return false;">PDB: 2VT4</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3CAP" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3CAP','PDB','3CAP'); return false;">PDB: 3CAP</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3D4S" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3D4S','PDB','3D4S'); return false;">PDB: 3D4S</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3DQB" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3DQB','PDB','3DQB'); return false;">PDB: 3DQB</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3EML" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3EML','PDB','3EML'); return false;">PDB: 3EML</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-3"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i38"><a href="/doi/suppl/10.1021/jm300280e">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_34380"></div></div></div></div></div><hr /></hr><p class="last">Pharmacophore definitions of AA2AR, ADRB2, and S1PR1; lists of ranked pharmacophores according to their expected retrieval of actives for all three targets; overview of property distributions of the 50.308 MSD compounds and the compounds available in the ZINC and CoCoCo database; overlap analysis between the 50.308 MSD compounds and ZINC and CoCoCo based on ECFP_4 fingerprints; plots of compound selections and their respective ranking in the structure-based and ligand-based ranking protocols. This material is available free of charge via the Internet at <a class="extLink" href="http://pubs.acs.org">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm300280e/suppl_file/jm300280e_si_001.pdf">jm300280e_si_001.pdf (1.18 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm300280e&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Fjm300280e%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2012.55.issue-11%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm300280e" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67997e21a9eed1a3","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
