
<html lang="en"     class="pb-page"  data-request-id="15c7e13a-e23d-4dc8-a363-45d289db570a"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.0c01019;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2020.63.issue-21"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of Roblitinib (FGF401) as a Reversible-Covalent Inhibitor of the Kinase Activity of Fibroblast Growth Factor Receptor 4" /></meta><meta name="dc.Creator" content="Robin A.  Fairhurst" /></meta><meta name="dc.Creator" content="Thomas  Knoepfel" /></meta><meta name="dc.Creator" content="Nicole  Buschmann" /></meta><meta name="dc.Creator" content="Catherine  Leblanc" /></meta><meta name="dc.Creator" content="Robert  Mah" /></meta><meta name="dc.Creator" content="Milen  Todorov" /></meta><meta name="dc.Creator" content="Pierre  Nimsgern" /></meta><meta name="dc.Creator" content="Sebastien  Ripoche" /></meta><meta name="dc.Creator" content="Michel  Niklaus" /></meta><meta name="dc.Creator" content="Nicolas  Warin" /></meta><meta name="dc.Creator" content="Van Huy  Luu" /></meta><meta name="dc.Creator" content="Mario  Madoerin" /></meta><meta name="dc.Creator" content="Jasmin  Wirth" /></meta><meta name="dc.Creator" content="Diana  Graus-Porta" /></meta><meta name="dc.Creator" content="Andreas  Weiss" /></meta><meta name="dc.Creator" content="Michael  Kiffe" /></meta><meta name="dc.Creator" content="Markus  Wartmann" /></meta><meta name="dc.Creator" content="Jacqueline  Kinyamu-Akunda" /></meta><meta name="dc.Creator" content="Dario  Sterker" /></meta><meta name="dc.Creator" content="Christelle  Stamm" /></meta><meta name="dc.Creator" content="Flavia  Adler" /></meta><meta name="dc.Creator" content="Alexandra  Buhles" /></meta><meta name="dc.Creator" content="Heiko  Schadt" /></meta><meta name="dc.Creator" content="Philippe  Couttet" /></meta><meta name="dc.Creator" content="Jutta  Blank" /></meta><meta name="dc.Creator" content="Inga  Galuba" /></meta><meta name="dc.Creator" content="Jörg  Trappe" /></meta><meta name="dc.Creator" content="Johannes  Voshol" /></meta><meta name="dc.Creator" content="Nils  Ostermann" /></meta><meta name="dc.Creator" content="Chao  Zou" /></meta><meta name="dc.Creator" content="Jörg  Berghausen" /></meta><meta name="dc.Creator" content="Alberto  Del Rio Espinola" /></meta><meta name="dc.Creator" content="Wolfgang  Jahnke" /></meta><meta name="dc.Creator" content="Pascal  Furet" /></meta><meta name="dc.Description" content="FGF19 signaling through the FGFR4/β-klotho receptor complex has been shown to be a key driver of growth and survival in a subset of hepatocellular carcinomas, making selective FGFR4 inhibition an a..." /></meta><meta name="Description" content="FGF19 signaling through the FGFR4/β-klotho receptor complex has been shown to be a key driver of growth and survival in a subset of hepatocellular carcinomas, making selective FGFR4 inhibition an a..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="September 15, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c01019" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01019" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c01019" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01019" /></link>
        
    
    

<title>Discovery of Roblitinib (FGF401) as a Reversible-Covalent Inhibitor of the Kinase Activity of Fibroblast Growth Factor Receptor 4 | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c01019" /></meta><meta property="og:title" content="Discovery of Roblitinib (FGF401) as a Reversible-Covalent Inhibitor of the Kinase Activity of Fibroblast Growth Factor Receptor 4" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0030.jpeg" /></meta><meta property="og:description" content="FGF19 signaling through the FGFR4/β-klotho receptor complex has been shown to be a key driver of growth and survival in a subset of hepatocellular carcinomas, making selective FGFR4 inhibition an attractive treatment opportunity. A kinome-wide sequence alignment highlighted a poorly conserved cysteine residue within the FGFR4 ATP-binding site at position 552, two positions beyond the gate-keeper residue. Several strategies for targeting this cysteine to identify FGFR4 selective inhibitor starting points are summarized which made use of both rational and unbiased screening approaches. The optimization of a 2-formylquinoline amide hit series is described in which the aldehyde makes a hemithioacetal reversible-covalent interaction with cysteine 552. Key challenges addressed during the optimization are improving the FGFR4 potency, metabolic stability, and solubility leading ultimately to the highly selective first-in-class clinical candidate roblitinib." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c01019"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01019">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c01019&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c01019&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c01019&amp;href=/doi/10.1021/acs.jmedchem.0c01019" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 21</span><span class="cit-fg-pageRange">, 12542-12573</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/21" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c00702" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Drug Annotation</span><a href="/doi/10.1021/acs.jmedchem.0c01035" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of Roblitinib (FGF401) as a Reversible-Covalent Inhibitor of the Kinase Activity of Fibroblast Growth Factor Receptor 4</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Robin A. Fairhurst</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Robin A. Fairhurst</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, CH-4002 Basel, Switzerland</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#40322f22292e6e262129322835323334002e2f3621323429336e232f2d"><span class="__cf_email__" data-cfemail="d3a1bcb1babdfdb5b2baa1bba6a1a0a793bdbca5b2a1a7baa0fdb0bcbe">[email protected]</span></a></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Robin+A.++Fairhurst">Robin A. Fairhurst</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9924-6291" title="Orcid link">http://orcid.org/0000-0002-9924-6291</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Thomas Knoepfel</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Thomas Knoepfel</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Thomas++Knoepfel">Thomas Knoepfel</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1675-6297" title="Orcid link">http://orcid.org/0000-0002-1675-6297</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nicole Buschmann</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nicole Buschmann</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nicole++Buschmann">Nicole Buschmann</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Catherine Leblanc</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Catherine Leblanc</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Catherine++Leblanc">Catherine Leblanc</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Robert Mah</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Robert Mah</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Robert++Mah">Robert Mah</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Milen Todorov</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Milen Todorov</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Milen++Todorov">Milen Todorov</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Pierre Nimsgern</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Pierre Nimsgern</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Pierre++Nimsgern">Pierre Nimsgern</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sebastien Ripoche</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sebastien Ripoche</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sebastien++Ripoche">Sebastien Ripoche</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Michel Niklaus</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Michel Niklaus</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Michel++Niklaus">Michel Niklaus</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nicolas Warin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nicolas Warin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nicolas++Warin">Nicolas Warin</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Van Huy Luu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Van Huy Luu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Van+Huy++Luu">Van Huy Luu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mario Madoerin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mario Madoerin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mario++Madoerin">Mario Madoerin</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jasmin Wirth</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jasmin Wirth</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jasmin++Wirth">Jasmin Wirth</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Diana Graus-Porta</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Diana Graus-Porta</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Diana++Graus-Porta">Diana Graus-Porta</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Andreas Weiss</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andreas Weiss</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andreas++Weiss">Andreas Weiss</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Michael Kiffe</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Michael Kiffe</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Michael++Kiffe">Michael Kiffe</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Markus Wartmann</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Markus Wartmann</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Markus++Wartmann">Markus Wartmann</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jacqueline Kinyamu-Akunda</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jacqueline Kinyamu-Akunda</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jacqueline++Kinyamu-Akunda">Jacqueline Kinyamu-Akunda</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dario Sterker</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dario Sterker</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dario++Sterker">Dario Sterker</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Christelle Stamm</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christelle Stamm</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christelle++Stamm">Christelle Stamm</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Flavia Adler</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Flavia Adler</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Flavia++Adler">Flavia Adler</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Alexandra Buhles</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alexandra Buhles</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alexandra++Buhles">Alexandra Buhles</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Heiko Schadt</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Heiko Schadt</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Heiko++Schadt">Heiko Schadt</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Philippe Couttet</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Philippe Couttet</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Philippe++Couttet">Philippe Couttet</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jutta Blank</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jutta Blank</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jutta++Blank">Jutta Blank</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Inga Galuba</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Inga Galuba</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Inga++Galuba">Inga Galuba</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jörg Trappe</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jörg Trappe</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=J%C3%B6rg++Trappe">Jörg Trappe</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Johannes Voshol</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Johannes Voshol</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Johannes++Voshol">Johannes Voshol</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nils Ostermann</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nils Ostermann</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nils++Ostermann">Nils Ostermann</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chao Zou</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chao Zou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chao++Zou">Chao Zou</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jörg Berghausen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jörg Berghausen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=J%C3%B6rg++Berghausen">Jörg Berghausen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Alberto Del Rio Espinola</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alberto Del Rio Espinola</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alberto++Del+Rio+Espinola">Alberto Del Rio Espinola</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Wolfgang Jahnke</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wolfgang Jahnke</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wolfgang++Jahnke">Wolfgang Jahnke</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7003-3305" title="Orcid link">http://orcid.org/0000-0002-7003-3305</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Pascal Furet</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Pascal Furet</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Novartis Institutes for BioMedical Research, CH-4002 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Pascal++Furet">Pascal Furet</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01019&amp;href=/doi/10.1021%2Facs.jmedchem.0c01019" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 21</span><span class="cit-pageRange">, 12542–12573</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">September 15, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>15 June 2020</li><li><span class="item_label"><b>Published</b> online</span>15 September 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 12 November 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c01019" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01019</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 <span class="NLM_copyright-holder">American Chemical Society</span></strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D12542%26pageCount%3D32%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DRobin%2BA.%2BFairhurst%252C%2BThomas%2BKnoepfel%252C%2BNicole%2BBuschmann%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D21%26contentID%3Dacs.jmedchem.0c01019%26title%3DDiscovery%2Bof%2BRoblitinib%2B%2528FGF401%2529%2Bas%2Ba%2BReversible-Covalent%2BInhibitor%2Bof%2Bthe%2BKinase%2BActivity%2Bof%2BFibroblast%2BGrowth%2BFactor%2BReceptor%2B4%26numPages%3D32%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D12573%26publicationDate%3DNovember%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c01019"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">3517</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">5</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c01019" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of Roblitinib (FGF401) as a Reversible-Covalent Inhibitor of the Kinase Activity of Fibroblast Growth Factor Receptor 4&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Robin&quot;,&quot;last_name&quot;:&quot;A. Fairhurst&quot;},{&quot;first_name&quot;:&quot;Thomas&quot;,&quot;last_name&quot;:&quot;Knoepfel&quot;},{&quot;first_name&quot;:&quot;Nicole&quot;,&quot;last_name&quot;:&quot;Buschmann&quot;},{&quot;first_name&quot;:&quot;Catherine&quot;,&quot;last_name&quot;:&quot;Leblanc&quot;},{&quot;first_name&quot;:&quot;Robert&quot;,&quot;last_name&quot;:&quot;Mah&quot;},{&quot;first_name&quot;:&quot;Milen&quot;,&quot;last_name&quot;:&quot;Todorov&quot;},{&quot;first_name&quot;:&quot;Pierre&quot;,&quot;last_name&quot;:&quot;Nimsgern&quot;},{&quot;first_name&quot;:&quot;Sebastien&quot;,&quot;last_name&quot;:&quot;Ripoche&quot;},{&quot;first_name&quot;:&quot;Michel&quot;,&quot;last_name&quot;:&quot;Niklaus&quot;},{&quot;first_name&quot;:&quot;Nicolas&quot;,&quot;last_name&quot;:&quot;Warin&quot;},{&quot;first_name&quot;:&quot;Van&quot;,&quot;last_name&quot;:&quot;Huy Luu&quot;},{&quot;first_name&quot;:&quot;Mario&quot;,&quot;last_name&quot;:&quot;Madoerin&quot;},{&quot;first_name&quot;:&quot;Jasmin&quot;,&quot;last_name&quot;:&quot;Wirth&quot;},{&quot;first_name&quot;:&quot;Diana&quot;,&quot;last_name&quot;:&quot;Graus-Porta&quot;},{&quot;first_name&quot;:&quot;Andreas&quot;,&quot;last_name&quot;:&quot;Weiss&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;Kiffe&quot;},{&quot;first_name&quot;:&quot;Markus&quot;,&quot;last_name&quot;:&quot;Wartmann&quot;},{&quot;first_name&quot;:&quot;Jacqueline&quot;,&quot;last_name&quot;:&quot;Kinyamu-Akunda&quot;},{&quot;first_name&quot;:&quot;Dario&quot;,&quot;last_name&quot;:&quot;Sterker&quot;},{&quot;first_name&quot;:&quot;Christelle&quot;,&quot;last_name&quot;:&quot;Stamm&quot;},{&quot;first_name&quot;:&quot;Flavia&quot;,&quot;last_name&quot;:&quot;Adler&quot;},{&quot;first_name&quot;:&quot;Alexandra&quot;,&quot;last_name&quot;:&quot;Buhles&quot;},{&quot;first_name&quot;:&quot;Heiko&quot;,&quot;last_name&quot;:&quot;Schadt&quot;},{&quot;first_name&quot;:&quot;Philippe&quot;,&quot;last_name&quot;:&quot;Couttet&quot;},{&quot;first_name&quot;:&quot;Jutta&quot;,&quot;last_name&quot;:&quot;Blank&quot;},{&quot;first_name&quot;:&quot;Inga&quot;,&quot;last_name&quot;:&quot;Galuba&quot;},{&quot;first_name&quot;:&quot;Jörg&quot;,&quot;last_name&quot;:&quot;Trappe&quot;},{&quot;first_name&quot;:&quot;Johannes&quot;,&quot;last_name&quot;:&quot;Voshol&quot;},{&quot;first_name&quot;:&quot;Nils&quot;,&quot;last_name&quot;:&quot;Ostermann&quot;},{&quot;first_name&quot;:&quot;Chao&quot;,&quot;last_name&quot;:&quot;Zou&quot;},{&quot;first_name&quot;:&quot;Jörg&quot;,&quot;last_name&quot;:&quot;Berghausen&quot;},{&quot;first_name&quot;:&quot;Alberto&quot;,&quot;last_name&quot;:&quot;Del Rio Espinola&quot;},{&quot;first_name&quot;:&quot;Wolfgang&quot;,&quot;last_name&quot;:&quot;Jahnke&quot;},{&quot;first_name&quot;:&quot;Pascal&quot;,&quot;last_name&quot;:&quot;Furet&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;09&quot;,&quot;day&quot;:&quot;15&quot;,&quot;issue&quot;:&quot;21&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;12542-12573&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c01019&quot;},&quot;abstract&quot;:&quot;FGF19 signaling through the FGFR4/β-klotho receptor complex has been shown to be a key driver of growth and survival in a subset of hepatocellular carcinomas, making selective FGFR4 inhibition an attractive treatment opportunity. A kinome-wide sequence alignment highlighted a poorly conserved cysteine residue within the FGFR4 ATP-binding site at position 552, two positions beyond the gate-keeper residue. Several strategies for targeting this cysteine to identify FGFR4 selective inhibitor starting points are summarized which made use of both rational and unbiased screening approaches. The optimization of a 2-formylquinoline amide hit series is described in which the aldehyde makes a hemithioacetal reversible-covalent interaction with cysteine 552. Key challenges addressed during the optimization are improving the FGFR4 potency, metabolic stability, and solubility leading ultimately to the highly selective first-in-class clinical candidate roblitinib.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01019&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01019" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01019&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01019" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01019&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01019" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01019&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01019&amp;href=/doi/10.1021/acs.jmedchem.0c01019" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c01019" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c01019" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (13 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c01019%26sid%3Dliteratum%253Aachs%26pmid%3D32930584%26genre%3Darticle%26aulast%3DFairhurst%26date%3D2020%26atitle%3DDiscovery%2Bof%2BRoblitinib%2B%2528FGF401%2529%2Bas%2Ba%2BReversible-Covalent%2BInhibitor%2Bof%2Bthe%2BKinase%2BActivity%2Bof%2BFibroblast%2BGrowth%2BFactor%2BReceptor%2B4%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D21%26spage%3D12542%26epage%3D12573%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=290831" title="Monomers">Monomers</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/21" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/jmcmar.2020.63.issue-21/20201112/jmcmar.2020.63.issue-21.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0030.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0030.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">FGF19 signaling through the FGFR4/β-klotho receptor complex has been shown to be a key driver of growth and survival in a subset of hepatocellular carcinomas, making selective FGFR4 inhibition an attractive treatment opportunity. A kinome-wide sequence alignment highlighted a poorly conserved cysteine residue within the FGFR4 ATP-binding site at position 552, two positions beyond the gate-keeper residue. Several strategies for targeting this cysteine to identify FGFR4 selective inhibitor starting points are summarized which made use of both rational and unbiased screening approaches. The optimization of a 2-formylquinoline amide hit series is described in which the aldehyde makes a hemithioacetal reversible-covalent interaction with cysteine 552. Key challenges addressed during the optimization are improving the FGFR4 potency, metabolic stability, and solubility leading ultimately to the highly selective first-in-class clinical candidate roblitinib.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64416" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64416" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec1_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Biological Rationale</h3><div class="NLM_p">Fibroblast growth factor receptor 4 (FGFR4) belongs to the FGFR family of four receptor tyrosine kinases that interact with fibroblast growth factors (FGFs) to regulate key developmental and physiological processes.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Aberrant FGFR signaling is implicated as the driver in a number of cancers and growth disorders including FGFR1 in non-small-cell lung cancer; FGFR2 in cholangiocarcinoma, endometrial, and gastric cancer; FGFR3 in bladder cancer; FGFR4 in rhabdomyosarcoma (RMS) and hepatocellular carcinoma (HCC).<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> As a result, numerous efforts have been made to identify FGFR inhibitors and a number of multitargeted kinase inhibitors have the FGFR family among the list of kinases which they inhibit (examples include derazantinib, dovitinib, lenvatinib, regorafenib, sorafenib). However, understanding the extent to which FGFR inhibition contributes to the safety and efficacy of these multitargeted agents is difficult to separate from the effects due to the inhibition of kinases outside the FGFR family. A number of selective pan-FGFR inhibitors have been developed that reversibly target the ATP binding site within the kinase domain which can be categorized as being either FGFR1–3 biased, with lower activity versus FGFR4 (the most well characterized examples being AZD4547 and infigratinib) or FGFR1–4 balanced, with equal activity versus all four isoforms (erdafitinib and LY2874455), <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> The reversible-covalent inhibitor <b>1</b> and the irreversible covalent inhibitor PRN1371 react with a cysteine on the P-loop which is conserved across all four family members leading to family selective pan-FGFR inhibition.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> The allosteric inhibitor SSR128129 has been described to bind at an extracellular site present on all four family members.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Antibodies targeting either the ligands or the receptors have also been developed to block FGFR signaling.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of selective pan-FGFR inhibitors: (A) FGFR1–3 biased; (B) FGFR1–4 balanced; (C) allosteric FGFR inhibitor; (D) covalent P-loop targeting FGFR1–4 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">HCC is the sixth most prevalent cancer worldwide, typically developing in patients with underlying viral infections, metabolic disorders, or a history of alcohol abuse.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Treatments currently approved for HCC include the multitargeted kinase inhibitors sorafenib, lenvatinib, carbozantinib, and regorafenib and the checkpoint inhibitors nivolumab and pembrolizumab.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> These treatments partially address the needs of patients, but there remains a clear requirement for agents that can provide improved outcomes in HCC.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Our interest centered on the observation that FGF19, signaling through FGFR4 and the coreceptor β-klotho (KLB), is implicated as the oncogenic driver in a subset of HCC. First, FGF19 is amplified in a number of HCCs, where it likely activates FGFR4 in an autocrine fashion.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Second, transgenic mice expressing FGF19 in skeletal muscle develop HCC, highlighting the transformative potential of elevated FGF19 and that FGF19 can also drive tumor growth in a paracrine manner.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> To address this opportunity, we sought to identify selective FGFR4 inhibitors as an improved approach to the already described selective pan-FGFR inhibitors, avoiding FGFR1–3 driven side effects. The principal side-effect limiting the tolerability of pan-FGFR inhibitors is hyperphosphatemia, driven mainly by the inhibition of FGF23 signaling through FGFR1,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> the hypothesis being that a selective FGFR4 inhibitor would be better tolerated and, as a result, enable a higher and/or a more sustained level of FGFR4 inhibition to be explored in patients.</div><div class="NLM_p">Ligand binding to FGFR4 induces receptor dimerization which leads to increased kinase activity through the autophosphorylation of key tyrosine residues, in particular at positions 642 and 643 on the activation loop.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> This in turn facilitates the recruitment and phosphorylation of the downstream substrates FRS2 and phospholipase C-γ. In cancer, aberrant activation of these phosphorylation events leads to the overstimulation of signals for survival and proliferation through the MAP kinase and phosphoinositide 3-kinase pathways, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a></div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. FGFR4 signaling cascades regulating BA biosynthesis, and survival and proliferation in cancer.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">In a healthy setting, FGF19 is secreted by enterocytes and signals through FGFR4/KLB in hepatocytes to regulate the biosynthesis of bile acids (BAs), <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> FGF19 represses the synthesis of the cytochrome P450 7A1 (CYP7A1), the enzyme which performs the rate-limiting step in the conversion of cholesterol into the primary BAs cholic acid and chenodeoxycholic acid. Consequently, FGFR4 inhibition was anticipated to perturb the enterohepatic bile acid/cholesterol system by blocking the above key feedback-loop, leading to an increase in BAs at the expense of cholesterol. At the start of the project some concerns were present regarding the extent to which an elevation in BA levels would limit the tolerability of an FGFR4 inhibitor and ultimately the degree and duration to which FGFR4 could be inhibited in patients. A study in primates with an FGF19 blocking antibody had highlighted elevated BA levels being the cause of liver toxicity and severe diarrhea.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> However, we anticipated that the coadministration of a bile acid sequestrant, such as the clinically used cholestyramine, could assist in maintaining BA levels in a tolerable range and minimize any associated side effects, if proven to be problematic in patients.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Cholestyramine is an orally administered tertiary amine-containing ion-exchange resin that complexes with BA in the gut to increase the amounts of BA eliminated in the feces and which was originally developed as a treatment to lower cholesterol.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a></div></div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13812" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13812" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Selective FGFR4 Inhibitor Hit Finding</h3><div class="NLM_p">At the onset of the project no low molecular weight FGFR4 inhibitors had been described, and analyses were carried out to explore how selectivity could be achieved, within both the FGFR family and the broader kinome. The most compelling of these was a kinome-wide kinase-domain sequence alignment.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> This analysis revealed what was considered to be an exploitable difference within the middle-hinge region of the ATP binding site, two positions beyond the gate-keeper residue (GK+2). In FGFR4, a cysteine is present at the GK+2 position, while the other FGFR family members contain a tyrosine at this position, <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>. Across the human kinome, only four other kinases were found to contain a cysteine at the GK+2 position: MAPKAPK2; MAPKAPK3; p70S6Kb; TTK.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> This GK+2 cysteine was thought to offer the opportunity to introduce an appropriately positioned electrophile to react with. Such a covalent interaction should then provide a way to achieve selectivity versus FGFR1–3 and also across a large proportion of the kinome. Additionally, the vast majority of kinases, like FGFR1–3, contain a larger aromatic amino acid residue at the GK+2 position, and only 12 other kinases possess a similar, or smaller, sized residue. This observation offered a second opportunity to gain kinase selectivity in this region if the larger subpocket, created by the relatively small cysteine residue, could be exploited. Both the middle-hinge covalent and steric approaches were considered equally viable for gaining FGFR4 selectivity at the start of the project and were incorporated into the hit-finding strategies.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Amino acid sequence alignment centered around the FGFR-family middle-hinge region.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">More recently, the efforts from several groups have been reported including fisogatinib (BLU-554), which was derived from BLU9931, and H3B-6527 which have now gone on to follow roblitinib into clinical studies in HCC patients (ClinicalTrials.gov identifiers NCT02508467 and NCT02834780), <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>.<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21,22)</a> It is interesting to compare the approaches taken by other groups, as in each case similar analyses have been made to those described above to target the middle-hinge region of the ATP binding site for FGFR4 selectivity.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> The majority have targeted a covalent interaction with Cys552 or, in the case of compound <b>4</b>, by occupying the larger subpocket.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> For fisogatinib, H3B-6527, <b>2</b>, and <b>3</b>, Cys552 is targeted for an irreversible-covalent interaction through a Michael-addition reaction to the acrylamide moiety. In addition to the FGFR4 selective inhibitors targeting the kinase domain, an antisense oligonucleotide ISIS-FGFR4RX has previously been described to block the production of FGFR4 as a potential treatment for obesity.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Structures of selective FGFR4 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Rational hit-finding activities were undertaken based upon the above middle-hinge covalent and steric hypotheses, in addition to unbiased screening approaches. At this time, no examples of covalent binding to a GK+2 cysteine-containing kinase had been described, so the reactivity of this cysteine residue in FGFR4 still needed to be established. However, the rational application of both middle-hinge hypotheses provided early successes and gave confidence that these approaches could yield starting points with high levels of kinase selectivity. Compounds <b>5</b> and <b>6</b> represent two of these early opportunities, <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> The rational introduction of an electrophile into a truncated version of the known FGFR inhibitor AZD4547 resulted in <b>5</b> which underwent a Michael reaction with Cys552 (<i>k</i><sub>inact</sub>/<i>K</i><sub>i</sub> = 5.1 × 10<sup>2</sup> M<sup>–1</sup> s<sup>–1</sup> C477A FGFR4 variant) and also bound covalently to Cys477 on the P-loop of FGFR4 at a much faster rate (<i>k</i><sub>inact</sub>/<i>K</i><sub>i</sub> = 5.1 × 10<sup>6</sup> M<sup>–1</sup> s<sup>–1</sup> wild-type FGFR4).<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> This duality of reactivity with both the middle-hinge and P-loop FGFR4 cysteines has subsequently been described for other acrylamide containing inhibitors.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> As highlighted earlier, the P-loop cysteine is poorly conserved across the human kinome but is present in all four FGFR family members, offering limited opportunity to gain selectivity within the FGFR family.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> However, although the rate of reaction was relatively low, <b>5</b> provided confidence that Cys552 could be targeted for a covalent interaction. Differentiation based upon the size of the GK+2 residue enabled a diaminopyrimidine series to be optimized to <b>6</b>, in which the isobutoxy residue occupies the larger subpocket created by Cys552. Compound <b>6</b> achieved >100-fold selectivity versus FGFR1–3 and a favorable level of selectivity against a diverse panel of >50 human kinases.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a></div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. FGFR4 hits identified by the rational modification of known kinase inhibitors and unbiased screening. Interactions with the middle-hinge region are determined from modeling for <b>6</b> and <b>8</b> and X-ray cocrystal structures for <b>5</b> and <b>7</b> (PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NWZ">5NWZ</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NUD">5NUD</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">A high throughput screening campaign with a biochemical FGFR4 kinase assay, using an FGFR2 counterscreen to assess selectivity, yielded a small number of validated hits with >50-fold selectivity versus FGFR2. Interestingly all of these selective hits could be assigned as ATP-site binders in which an interaction within the GK+2 position was believed to be the major contributor to the FGFR4/2 selectivity. In particular, two of these stood out as being both potent (FGFR4 IC<sub>50</sub> < 100 nM) and highly selective (FGFR2 IC<sub>50</sub> > 10 μM), compounds <b>7</b> and <b>8</b>, <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Compound <b>7</b> was shown to bind covalently to Cys552 through an S<sub>N</sub>Ar reaction with the 2-chloropyridine moiety (<i>k</i><sub>inact</sub>/<i>K</i><sub>i</sub> = 3.0 × 10<sup>4</sup> M<sup>–1</sup> s<sup>–1</sup>).<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Compound <b>8</b> was a representative example from a small number of closely related 2-formylquinoline amide (2-FQA) derivatives for which the Cys552 thiol and the formyl group of <b>8</b> were shown to be essential for activity. In addition to excellent FGFR family selectivity, <b>7</b> and <b>8</b> both showed excellent selectivity across a diverse panel of 50 human kinases.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Ultimately, the set of 2-FQAs represented by <b>8</b> provided the starting point that went on to be optimized to the clinical candidate roblitinib. Two observations were important to the early uptake of the 2-FQA hit series: the growing understanding of the potential of the GK+2 interaction as a way to deliver excellent kinase selectivity; the reversible-covalent mechanism of action in the context of the rapid protein resynthesis rate determined for FGFR4 in HCC cell lines; <i>vide infra</i>.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Evaluation of the 2-FQA Hit Series</h3><div class="NLM_p">The experience gained with several noncovalent inhibitor series suggested that a covalent interaction between <b>8</b> and FGFR4 was driving the activity, due to the high level of potency from a relatively small molecule (MW = 345). A hypothesis for hemithioacetal formation was then quickly arrived at, involving the side chain thiol of Cys552 and the 2-formyl group of <b>8</b>, <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> The bond formation was anticipated to be facilitated by the pyridine moiety making hydrogen-bonding interactions within the hinge region to position the two reacting centers proximal to each other. The intramolecular hydrogen-bond within <b>8</b> between the amide hydrogen and quinoline nitrogen facilitates the proximity of the two reacting centers. Initial attempts to expand the SAR surrounding <b>8</b> highlighted the poor solubility of the 2-FQA starting point (thermodynamic solubility pH 6.8 < 2.9 μM). Taking the 2-formylpyridyl moiety as a key element of the pharmacophore, scaffold morphing to introduce saturation as a way to improve solubility, led to the 2-formyl tetrahydronaphthyridine urea (2-FTHNU) <b>9</b> (numbering is retained from the 2FQA hit series). Although only a modest increase in solubility was observed by morphing to the 2-FTHNU series (thermodynamic solubility, pH 6.8, <b>9</b>, 4.9 μM), this enabled consistent in vitro SAR to be generated. Additionally this modification was also accompanied by a small increase in potency (4-fold for <b>9</b> versus <b>8</b>), and the 2-FTHNU quickly became the primary focus for the next phase of the optimization. Biochemical and cellular FGFR4 activity data are shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Key SAR generated to characterize the 2-FQA and 2-FTHNU hit series: (A) binding hypothesis for the interaction of <b>8</b> with FGFR4; (B) 2-FQA to 2-FTHNU scaffold morph; (C) inactive aldehyde replacements; (D) phenyl and N-methylated analogues disrupting the hinge-binding and intramolecular hydrogen bond; (E) cyanopyrimidine as an alternative electrophile; (F) analogues designed to assess the contribution of the 2-formylpyridyl group.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Biochemical and Cellular FGFR4 Inhibitory Activities and Hepatic Extraction Ratios Determined from Rat Liver Microsome Incubations for the Reference FGFR Inhibitors and Compounds <b>5</b>–<b>17</b><a class="ref internalNav" href="#tbl1-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="4" align="center">IC<sub>50</sub> (nM)</th><th class="colsep0 rowsep0" align="center"> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">biochemical FGFR4</th><th class="colsep0 rowsep0" align="center">BaF3 pFGFR4</th><th class="colsep0 rowsep0" align="center">FGFR4 BaF3 proliferation</th><th class="colsep0 rowsep0" align="center">HUH7 proliferation</th><th class="colsep0 rowsep0" align="center">rat microsome hepatic ER</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">infigratinib</td><td class="colsep0 rowsep0" align="left">71 ± 34</td><td class="colsep0 rowsep0" align="left">541 ± 239</td><td class="colsep0 rowsep0" align="left">581 ± 285</td><td class="colsep0 rowsep0" align="left">80 ± 4</td><td class="colsep0 rowsep0" align="left">0.77</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AZD4547</td><td class="colsep0 rowsep0" align="left">56 ± 21</td><td class="colsep0 rowsep0" align="left">519 ± 150</td><td class="colsep0 rowsep0" align="left">348 ± 26</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.43</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BLU9931</td><td class="colsep0 rowsep0" align="left">14 ± 10</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.65</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">erdafitinib</td><td class="colsep0 rowsep0" align="left">3.8</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.93</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">LY2874455</td><td class="colsep0 rowsep0" align="left">5.0</td><td class="colsep0 rowsep0" align="left">10 ± 1</td><td class="colsep0 rowsep0" align="left">7.6</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.86</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="left">1.5 ± 1.0</td><td class="colsep0 rowsep0" align="left">8.3 ± 1.6</td><td class="colsep0 rowsep0" align="left">4.9</td><td class="colsep0 rowsep0" align="left">26 ± 13</td><td class="colsep0 rowsep0" align="left">0.71</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="left">2.2 ± 0.9</td><td class="colsep0 rowsep0" align="left">162 ± 52</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">237 ± 95</td><td class="colsep0 rowsep0" align="left">0.10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="left">53 ± 18</td><td class="colsep0 rowsep0" align="left">482 ± 24</td><td class="colsep0 rowsep0" align="left">139 ± 5</td><td class="colsep0 rowsep0" align="left">135</td><td class="colsep0 rowsep0" align="left">0.71</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="left">57 ± 10</td><td class="colsep0 rowsep0" align="left">2755 ± 125</td><td class="colsep0 rowsep0" align="left">>3000</td><td class="colsep0 rowsep0" align="left">>3000</td><td class="colsep0 rowsep0" align="left">0.91</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="left">13 ± 8</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.91</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="left">8700</td><td class="colsep0 rowsep0" align="left">1180</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">>3000</td><td class="colsep0 rowsep0" align="left">0.78</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16</b></td><td class="colsep0 rowsep0" align="left">11 ± 5</td><td class="colsep0 rowsep0" align="left">88 ± 8</td><td class="colsep0 rowsep0" align="left">211</td><td class="colsep0 rowsep0" align="left">>3000</td><td class="colsep0 rowsep0" align="left">0.94</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="left">4400</td><td class="colsep0 rowsep0" align="left">>3000</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.90</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl1-fn1"><div class="footnote" id="tbl1-fn1"><sup><sup>a</sup></sup><p class="last">FGFR4 data are reported as the mean ± SD (<i>n</i> = 2–87 repeats) or as single measurements. Rat microsome hepatic ER data are reported as the mean (1–3 repeats).</p></div></div></div><div class="NLM_p last">Support for the above FGFR4 binding hypothesis was derived from a lack of activity for the benzylic alcohol <b>10</b>, carboxylic acid <b>11</b>, and primary amide <b>12</b> (biochemical FGFR4 IC<sub>50</sub> > 10 μM). Additionally, no inhibitory activity was determined for <b>8</b> or <b>9</b> with a biochemical assays using an FGFR4 kinase domain variant in which Cys552 had been swapped for alanine (C552A FGFR4 IC<sub>50</sub> values of >10 μM), <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>. These data supported the aldehyde group as the electrophile and that other carbonyl-containing derivatives were not suitable replacements. The key roles of the hinge-binding interaction and intramolecular hydrogen-bond were supported by the inactivity of the phenyl and N-methylated analogues <b>13</b> and <b>14</b> (biochemical FGFR4 IC<sub>50</sub> > 10 μM).<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Interestingly, attempts to introduce alternative electrophiles failed to deliver active compounds. For example, the cyanopyrimidine analogue <b>15</b> showed >1000-fold lower FGFR4 activity compared to a closely related 2-formylpyridine,<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> with an electrophile that has previously been used to successfully target cysteines for covalent interactions.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> The weak noncovalent FGFR4 activity of the THNU scaffold could be shown by comparing <b>16</b> with the analogue in which the aldehyde group is replaced by a hydrogen atom <b>17</b> and for which a 400-fold loss in FGFR4 biochemical activity was observed.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Characterization of the Reversible-Covalent FGFR4 Inhibition</h3><div class="NLM_p">To understand the specificity of the aldehyde interaction, Cys552 is located deep within the ATP binding site in a region that is structurally well-defined, with a relatively low level of conformational flexibility. As a result, any electrophile reacting with Cys552 would need to be of the correct size, be able to adopt the correct geometry, in addition to possessing a suitable level of chemical reactivity. With little movement in the protein to accommodate any poor alignments, the range of electrophiles that could react with Cys552 was therefore anticipated to be limited but with the possibility to lead to a high level of specificity. In contrast, the P-loop cysteine (Cys477 in FGFR4) is situated on a relatively flexible region of the protein with greater freedom of movement within the solvent. The P-loop cysteine of the FGFR family has been found to react with electrophiles in a number of different P-loop conformations, all of which are different from the ATP-bound catalytically active form in which the P-loop adopts a hairpin conformation.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> The flexibility of the two regions are compared in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>, by mapping the positions of the cysteine sulfur-atoms following covalent binding to ligands for which FGFR X-ray cocrystal structures have been solved, and modeling in the case of PRN1371. Consistent with the above analysis, the overlays show that variation in the position of C552 is limited to rotation about the thiomethyl side-chain moiety. In contrast, the P-loop cysteine is located in a number of different backbone conformations, following the addition reactions with inhibitors, which makes for a much larger volume in which reaction has been found to occur.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Overlay of the cysteine-thiol positions within the FGFR ATP-binding site of selective FGFR4 inhibitors covalently binding to Cys552, and pan-FGFR inhibitors binding to the P-loop cysteine. PDB codes are the following: <b>7</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NUD">5NUD</a>, in gray;<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> roblitinib, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YI8">6YI8</a>, in dark blue; BLU9931, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XCU">4XCU</a>, in brown;<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> H3B-6527, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VND">5VND</a>, in green;<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> FIIN-2, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4QQC">4QQC</a>. in light blue;<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> FIIN-3, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4R6V">4R6V</a>, in yellow;<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a><b>5</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NWZ">5NWZ</a>, in pink;<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> PRN1371, modeled interaction in magenta.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">With the opportunity for covalent interactions with Cys552 identified at an early stage of the project, the resynthesis rate of the target protein then became a key parameter to understand.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> The FGFR4 resynthesis rate in combination with an understanding of the temporal pharmacodynamic (PD) modulation required to deliver efficacy would then allow the optimal pharmacokinetic (PK) profile to be envisaged. For example, an ideal opportunity for an irreversible-covalent inhibitor would be to target a slowly resynthesized protein, where a transient exposure is sufficient to react and inhibit the target, and the response then slowly recovers over a much longer period of time, disconnecting effect from exposure in a favorable way.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> In parallel with the early hit-finding activities, studies were undertaken to establish the resynthesis rate of FGFR4 in relevant cell lines using stable-isotope labeling with amino acids (SILAC).<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> By use of this approach, the FGFR4 resynthesis rate was shown to be consistently less than 2 h for all the HCC cell lines studied: three with growth dependent upon FGFR4/KLB/FGF19 signaling (HUH7, Hep3B, and JHH7) and two not dependent on the pathway for growth (JHH5 and HEPG2), <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>. SILAC experiments were also conducted with the HUH7 and Hep3B cell lines in the presence of a selective FGFR4 inhibitor at a concentration selected to fully inhibit FGFR4. A similar result was obtained from these experiments, indicating that the FGFR4 resynthesis rate is unaffected by the activation state of the kinase, due to the presence of an ATP-site inhibitor. This is in contrast to the slower resynthesis rates that have been observed for some other kinases in the presence of inhibitors.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> The early identification of the FGFR4 resynthesis half-life being less than 2 h in HCC cell lines helped to further prioritise between the covalent hit series within the project as discussed below.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Resynthesis Rates of FGFR4 Determined with a Panel of HCC Cell Lines</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">cell line</th><th class="colsep0 rowsep0" align="center">FGFR4 resynthesis <i>t</i><sub>1/2</sub> (h)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HUH7</td><td class="colsep0 rowsep0" align="left"><2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Hep3B</td><td class="colsep0 rowsep0" align="left"><2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">JHH7</td><td class="colsep0 rowsep0" align="left"><2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">JHH5</td><td class="colsep0 rowsep0" align="left"><2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HEPG2</td><td class="colsep0 rowsep0" align="left"><2</td></tr></tbody></table></div></div><div class="NLM_p">Although only a small insight into the PK/PD/efficacy relationship was available at the beginning of the project, emerging cellular data supported that a high and sustained level of FGFR4 inhibition would be required for maximum efficacy in HCC. Such sustained-coverage PK/PD/efficacy relationships having been determined for a number of other kinase targets in the oncology field, including other FGFR family members, and this requirement became the working hypothesis to guide the prioritization of the series.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> The first consequence of this analysis was to deprioritize all irreversible-covalent inhibitor approaches, such as the acrylamides and 2-chloropyridines, represented by <b>5</b> and <b>7</b> above, the concern being that the rapid resynthesis rate of FGFR4 would not allow for a favorable disconnection of PD modulation from PK, and a sustained exposure to the irreversibly binding electrophile would be required throughout the dosing interval to continually engage and inhibit newly synthesized protein. Such an exposure profile was considered to be a higher risk for incurring toxicity due to nonspecific irreversible-covalent binding, in particular in the gut for a compound administered orally at the relatively high dose that was anticipated to be required to achieve a sustained high level of PD modulation. In this setting, even highly selective irreversible-covalent binders are anticipated to bind nonspecifically due to the initial high gut-levels, with the potential for protein adducts to remain long after the initial high compound concentration has been depleted.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> In contrast, a reversible-covalent binder, such as the 2-formylpyridyl-containing 2-FQA and 2-FTHNU, was considered an ideal opportunity in this situation, offering high FGFR4 potency and selectivity, likely a “standard” PK/PD relationship, and a reduced immunogenicity risk, due to the formation of reversible adducts with off-target proteins. The immunogenicity risk was considered to be lower because following protein degradation the affinity of a reversible-covalent binder for the cleaved oligopeptides was anticipated to be greatly reduced. This reduced affinity would then result in much lower levels of potential haptens.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> On the basis of the FGFR4 resynthesis rate measurements and following the above logic, the 2-FTHNU series became a primary focus for optimization within the project.</div><div class="NLM_p">However, some concerns remained for the 2-FTHNU series relating to the presence of the aryl aldehyde group: (i) as a metabolic liability such that an analogue with a suitable oral PK profile could be challenging to identify; (ii) as the source of a high level of nonspecific binding, potentially limiting or reducing efficacy; (iii) as a potential toxicophore.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> The nonspecific binding question arose due to both the 2-FQA and 2-FTHNU analogues showing a strong propensity to add nucleophiles in a reversible manner. For example, the equilibrium was typically displaced in favor of a hemiacetal species when alcoholic solutions were analyzed by LC/MS, but on evaporation the aldehyde form would be fully recovered from these solutions (determined by NMR analysis in CDCl<sub>3</sub> or DMSO-<i>d</i><sub>6</sub>). The positions of these equilibria proved to be variable between analogues but often favoring the addition products. An NMR study with the more-soluble <sup>13</sup>C-labeled analogue <b>18</b> in water supported this behavior, with both the aldehyde and hydrated species being detectable in solution, <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>. <sup>13</sup>C-labeled <b>18</b> was used for this study in order to gain full confidence in the identity of the hydrated species by 2D HSQC. These observations raised the question of weak nonspecific binding potentially limiting the amount of aldehyde available as “free drug” to inhibit FGFR4, the concern being that the high level of activity observed biochemically would be greatly reduced as the relative abundancy of nonspecific interactions increased in moving into cellular assays and then even further when moving into in vivo models. One way these weak nonspecific interactions could be overcome was thought to be with a longer target-residence time, as a result of the reversible-covalent nature of the interaction with FGFR4. Longer residence times have been highlighted as a way to increase potency and selectivity, both in vitro and in vivo.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> However, a residence time greater than 2 h was not anticipated to bring further benefit, due to the measured resynthesis rate of FGFR4 in HCC cell lines. In fact, a longer residence time was even considered to be detrimental because on rates would also increase if similar levels of FGFR4 affinity were maintained. A scenario could be envisioned where on rates would be limiting with such a rapidly resynthesized target protein, and higher inhibitor concentrations would be needed to react quickly enough with the newly synthesized protein to achieve and to maintain a high level of pathway inhibition.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. <sup>1</sup>H NMR study with <sup>13</sup>C-labeled <b>18</b> in water showing the ratio between the aldehyde and hydrated species. <sup>13</sup>C-HSQC: aldehyde <sup>1</sup>H/<sup>13</sup>C 9.92/192.5 ppm, <sup>1</sup><i>J</i>(<sup>1</sup>H–<sup>13</sup>C) coupling 182 Hz; hydrate <sup>1</sup>H/<sup>13</sup>C 6.09/89.4 ppm, <sup>1</sup><i>J</i>(<sup>1</sup>H–<sup>13</sup>C) coupling 166 Hz. Integration of the <sup>1</sup>H spectra shows an aldehyde to hydrate ratio of 9:1.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Binding kinetics were measured using a competition binding assay, and similar levels of FGFR4 affinity were determined for the noncovalent inhibitor <b>6</b> and the 2-FTHNU analogue <b>9</b>: <i>K</i><sub>d</sub> values of 20 nM and 28 nM, respectively.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> However, both compounds showed very different kinetic behavior. The binding kinetics for the noncovalent compound <b>6</b> were too fast to measure with the probe used for the assay, and a residence time of <1.4 min was assigned. The 2-FTHNU <b>9</b> exhibited slower binding kinetics with an FGFR4 residence time of 272 min, the reversible nature of the interaction being consistent with other hemithioacetal interactions under physiological-like conditions.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> These data provided some encouragement that a long FGFR4 residence time would not compromise the activity of the 2-FTHNU due to nonspecific binding but at the same time raised the question if the slower on rate could impact potency with a rapidly resynthesized target protein. Biochemical assays starting with nonphosphorylated FGFR4 and employing inhibitor preincubation periods between 0 and 4 h prior to the addition of ATP revealed minimal changes in potency for <b>9</b> and related 2-FTHNU analogues.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> These data suggested that the on rate for <b>9</b> was not limiting the FGFR4 inhibitory activity. However, on the basis of the rationale described above, further increases in potency in the optimization of the 2-FTHNU series would be more desirable through an increase in the on rate rather than by extending the residence time. Even though considered important, binding kinetics were not routinely monitored as a key parameter during the optimization phase. This approach was taken because any decrease in potency due to slow-binding kinetics was anticipated to be captured in the cellular assays. Additionally, rational design to control the balance between on and off-rates was considered challenging, and a spot-checking approach for measuring the binding kinetics was adopted during the optimization of the series.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Optimization of the 2-FTHNU Series</h3><div class="NLM_p">Further optimization of the 2-FTHNU series began by expanding the SAR surrounding the core and hinge binding elements, <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Changes to the partially saturated ring of the 2-FTHN that were found to be tolerated with minimal loss in activity included a reduction in size to the five-membered analogue <b>19</b>, introduction of a heteroatom to give the morpholine fused analogue <b>20</b>, and opening of the ring to give the 2-formylpyridyl urea analogue <b>21</b>. Increasing the size of the 2-THN partially saturated ring to the seven-membered analogue <b>22</b> led to a >50-fold loss in activity. This could be rationalized by an unfavorable steric interaction between the expanded ring and a hydrophobic slot formed between residues Leu473 and Gyl556 at the protein–solvent interface. This unfavorable interaction is in contrast to the analogues with smaller five- and six-membered rings and the ring-opened analogues with extended N-substituents, such as <b>23</b>, which make favorable contacts in this region.</div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Core and hinge-binding 2-FTHNU SAR: (A) THNU core variants; (B) hinge-binding heterocycle variants; (C) pyridine 4-substituents interacting with the Gly474/Val481 hydrophobic cleft; (D) modeled interaction of compound <b>28</b> within the ATP-binding site of FGFR4 highlighting the interactions between the pyridyl 4-methoxyethoxy substituent and the hydrophobic cleft formed between Gly474 and Val481.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Biochemical and Cellular FGFR4 Inhibitory Activities and Hepatic Extraction Ratios Determined from Rat Liver Microsome Incubations for Compounds <b>19</b>–<b>29</b><a class="ref internalNav" href="#tbl3-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="4" align="center">IC<sub>50</sub> (nM)</th><th class="colsep0 rowsep0" align="center"> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">biochemical FGFR4</th><th class="colsep0 rowsep0" align="center">BaF3 pFGFR4</th><th class="colsep0 rowsep0" align="center">FGFR4 BaF3 proliferation</th><th class="colsep0 rowsep0" align="center">HUH7 proliferation</th><th class="colsep0 rowsep0" align="center">rat microsome hepatic ER</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="left">11 ± 3</td><td class="colsep0 rowsep0" align="left">55 ± 3</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.95</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20</b></td><td class="colsep0 rowsep0" align="left">42 ± 10</td><td class="colsep0 rowsep0" align="left">531 ± 40</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.92</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="left">6.5 ± 2.5</td><td class="colsep0 rowsep0" align="left">41 ± 35</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">>3000</td><td class="colsep0 rowsep0" align="left">0.95</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22</b></td><td class="colsep0 rowsep0" align="left">660 ± 57</td><td class="colsep0 rowsep0" align="left">>3000</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">>3000</td><td class="colsep0 rowsep0" align="left">0.95</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23</b></td><td class="colsep0 rowsep0" align="left">7.4 ± 20</td><td class="colsep0 rowsep0" align="left">91 ± 23</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">>3000</td><td class="colsep0 rowsep0" align="left">0.96</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>24</b></td><td class="colsep0 rowsep0" align="left">16 ± 5</td><td class="colsep0 rowsep0" align="left">108 ± 29</td><td class="colsep0 rowsep0" align="left">171 ± 9</td><td class="colsep0 rowsep0" align="left">>3000</td><td class="colsep0 rowsep0" align="left">0.95</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25</b></td><td class="colsep0 rowsep0" align="left">45 ± 7</td><td class="colsep0 rowsep0" align="left">131 ± 18</td><td class="colsep0 rowsep0" align="left">301 ± 11</td><td class="colsep0 rowsep0" align="left">>3000</td><td class="colsep0 rowsep0" align="left">0.95</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>26</b></td><td class="colsep0 rowsep0" align="left">33 ± 3</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.97</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>27</b></td><td class="colsep0 rowsep0" align="left">0.7</td><td class="colsep0 rowsep0" align="left">29</td><td class="colsep0 rowsep0" align="left">28</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.95</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28</b></td><td class="colsep0 rowsep0" align="left">1.3 ± 0.5</td><td class="colsep0 rowsep0" align="left">18 ± 8</td><td class="colsep0 rowsep0" align="left">22 ± 9</td><td class="colsep0 rowsep0" align="left">>3000</td><td class="colsep0 rowsep0" align="left">0.97</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>29</b></td><td class="colsep0 rowsep0" align="left">1.2</td><td class="colsep0 rowsep0" align="left">24</td><td class="colsep0 rowsep0" align="left">18</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.95</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl3-fn1"><div class="footnote" id="tbl3-fn1"><sup><sup>a</sup></sup><p class="last">FGFR4 data are reported as the mean ± SD (<i>n</i> = 2–6 repeats) or as single measurements. Rat microsome hepatic ER data are reported as the mean (1–3 repeats).</p></div></div></div><div class="NLM_p">Molecular modeling indicated the hinge-binding pyridyl ring of the 2-FTHNU overlaid very closely with the pyrimidine ring of <b>6</b>. This analysis suggested that the 5-trifluoromethyl group in <b>9</b> could be replaced by the cyano group present in the equivalent position in <b>6</b>, which had been found to be optimal in the diaminopyrimidine series. Following this possibility, the 5-cyanopyridine analogue <b>24</b> was found to possess a similar level of FGFR4 biochemical activity compared to the trifluoromethyl matched pair <b>9</b>. In particular, the lower lipophilicity of the 5-cyano group was considered to be an attractive option for regulating the physical properties going forward (<b>9</b>, log <i>D</i><sub>7.4</sub> = 4.2; <b>24</b>, log <i>D</i><sub>7.4</sub> = 2.9). The overlay also suggested that the hinge-binding pyridine ring could be replaced by a number of five- and six-membered heterocycles. The most interesting heterocycles were found to be the closely related pyrimidine and pyrazine analogues <b>25</b> and <b>26</b>, with FGFR4 activities within 2-fold of the corresponding 5-cyanopyridine <b>24</b>. Further substitution of the hinge-binding ring indicated that a range of C-, N-, and O-substituents were tolerated at the pyridyl 4-position, and in several instances they provided an increase in potency. In particular, 4-substituents that could interact with a hydrophobic cleft between Gly474 and Val481, which is situated above the plane of the hinge-binding pyridine ring, were particularly active. The SAR from the diaminopyrimidine series was also consistent with this observation, the methylene bridge of the bicycle in <b>6</b> having been rationalized to interact within the cleft as way to increase potency.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Examples of 2-FTHNU hinge-binding ring 4-substituents that are believed to make favorable interactions within this hydrophobic cleft include the isopropoxy, 2-methoxyethoxy, and 4-hydroxy-4-methylpiperidnyl analogues <b>27</b>, <b>28</b>, and <b>29</b>, <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>.</div><div class="NLM_p">These initial investigations proved to be very encouraging for the 2-FTHNU series with analogues such as <b>27</b>–<b>29</b>, exhibiting biochemical activities of ∼1 nM, corresponding to the limit that could be quantified in the biochemical assay (FGFR4 concentration of 3 nM). These biochemical activities also translated consistently into cellular activities in Ba/F3 cells, measuring either FGFR4 autophosphorylation (pFGFR4), or cell proliferation. In the Ba/F3 cell line, the kinase domain is cytosolic and fused with the translocation ETS leukemia transcription factor (TEL) leading to constitutively activated FGFR4. Additionally, the Ba/F3 cells are engineered to be dependent upon signaling through the TEL fusion protein with FGFR4 for their growth, enabling the proliferation assay.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> The biochemical to Ba/F3 cellular assay IC<sub>50</sub> shifts observed for <b>27</b>–<b>29</b>, in the 15- to 41-fold range, are consistent with those determined for other kinases, when the biochemical assay is performed with ATP included at the <i>K</i><sub>M</sub> concentration. The shifts also compared favorably to the noncovalent inhibitor <b>6</b>, which exhibits a similar level of FGFR4 biochemical potency and for which a 73-fold shift was determined. Encouragingly, these data suggested no marked loss in activity for the 2-FTHNUs in a cellular setting, where an increased level of potential nonspecific interactions is present compared to the biochemical assay. These data helped to further alleviate the concerns that nonspecific covalent binding would lead to reduced potency, at least in a cellular setting.</div><div class="NLM_p">Further studies revealed that although the above 2-FTHNUs were active in the Ba/F3 cellular background, no activity was observed in proliferation assays in the HUH7 HCC cell line (IC<sub>50</sub> > 3 μM), <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. A similar lack of antiproliferative activity was also observed with an FGFR4-dependent RMS-derived RH-41 cell line. This was in contrast to the FGFR4-selective noncovalent inhibitors infigratinib and <b>6</b> and the irreversible-covalent inhibitor <b>5</b>, which inhibited HUH7 and RH41 proliferation with IC<sub>50</sub> values within 2- to 8-fold of those determined in the Ba/F3 pFGFR4 and proliferation assays. In terms of the incubation period, the HCC/RMS proliferation assays employed a similar 3 day incubation period compared to 2 days for the Ba/F3 proliferation assay. This was in contrast to the autophosphorylation assay which utilized a much shorter 40 min incubation period.</div><div class="NLM_p">In attempts to rationalize the lack of antiproliferative activity in the disease relevant cell lines, the HCC cells were anticipated to contain high levels of metabolizing enzymes due to their liver origin, and potentially a comparable level to that found in hepatocytes. Lower levels of metabolizing enzymes were considered likely for the Ba/F3 and RH-41 RMS cell lines which are hematopoietic and alveolar in origin.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> Therefore, a hypothesis was developed that the absence of activity in the HCC, proliferation assays was as a result of the 2-FTHNUs undergoing extensive metabolism.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> Removal of the parent compound would then lead to the loss of FGFR4 pathway inhibition during the 3-day proliferation assay, allowing the cells to proliferate, the assumption being that the generated metabolites would be of reduced FGFR4 activity compared to the parent compound. This hypothesis resonated with the possible metabolic weakness of the 2-pyridyl aldehyde that had been raised at the start of the optimization. However, this concern had originally been formulated in the context of achieving satisfactory oral exposure rather than limiting efficacy in disease relevant cell lines. To account for the observations within a single hypothesis, the RMS RH41 would also need to possess a sufficient level of metabolic capability to account for the reduced activity in this cell line.</div><div class="NLM_p">Further support for the above hypothesis came from the high in vitro clearance rates measured in liver microsomes and hepatocytes. <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a> include stability data from rat microsome incubations, and similarly high clearance rates were determined in incubations across species, including human (for example, <b>24</b> and <b>28</b>: human liver microsome ER 0.86 and 0.93). High-throughput metabolite identification by soft-spot identification (SSID) studies supported the aryl aldehyde to be the metabolic weak spot in both rat and human microsome incubations.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> The predominant, and in several instances only, metabolite identified by SSID was an M + 2 species across the 2-FTHNUs. The in silico prediction component of the SSID indicated the M + 2 species to result from the reduction of the aldehyde group to the corresponding benzylic alcohol, derivatives that had already been identified to be FGFR4 inactive when exploring the initial SAR: for example, comparing <b>9</b> with the corresponding benzyl alcohol <b>10</b>.</div><div class="NLM_p">Data supporting the above metabolic instability hypothesis came from time-course PD studies using the RH41 cell line. These studies revealed that the FGFR4 pathway was inhibited at an early time point (40 min), by measuring either proximal (pFRS2) or downstream (pMAPK/pERK) phosphorylation events. For both readouts, the inhibition was lost or decreased with the 2-FTHNU derivatives at the end of the incubation period (3 days). <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a> shows Western blot analyses for <b>24</b> and the pan-FGFR inhibitor infigratinib, and very similar profiles were obtained for all the 2-FTHNU derivatives unsubstituted at the 3-position that were investigated. The loss of pathway inhibition could be shown to correlate reasonably well with the disappearance of the parent compound <b>24</b> over time and the appearance of the M + 2 metabolite. Infigratinib was found to be stable throughout the incubation period and showed a high level of PD modulation throughout. However, the analyses of the parent and metabolite levels for <b>24</b> in the RH41 and HCC cell line (HUH7) incubation media proved to be challenging and variable. Monitoring the loss of 2-FTHNUs proved to be more straightforward in Caco-2 cell incubations: an epithelial colorectal adenocarcinoma cell line which has been used extensively as a routine in vitro PK screen to assess permeation across the gut epithelial barrier.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> Caco-2 cells have also previously been shown to be metabolically competent.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> Assessing gut permeation of the 2-FTHNUs in Caco-2 cells led to low compound recoveries (<10%) following a 2 h incubation period and prevented meaningful permeation rates to be assigned (compounds tested at 10 μM). The decrease in the 2-FTHNU levels and appearance of metabolites could be monitored over time in the incubation media by LC–MS/MS. Less variable data were generated in this way, and <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a> shows the change in parent, the M + 2, and an M + 16 metabolite over time for <b>25</b> from an incubation with Caco-2 cells (similar profiles were obtained for several 2-FTHNU derivatives unsubstituted at the 3-position). Greater than 80% depletion of the parent compound was observed after 1 h with a corresponding increase in the relative peak areas of the two metabolites and in particular the M + 2 metabolite. No other significant species were identified in these incubations.</div><figure id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0010.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Western blot pFRS2 and pERK PD measurements for compound <b>24</b> (50 nM) and infigratinib (50 nM) in RH41 cells after 40 min and 3 day incubation periods. Western blot for total Erk is provided as the protein loading control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0011.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Time course of the relative peak areas of parent, M + 2, and M + 16 metabolites in solution for <b>25</b> when incubated with Caco-2 cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In vivo rat PK studies were consistent with the in vitro data and revealed high in vivo clearance values. For example, <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a> summarizes the rat PK data obtained for <b>24</b> following intravenous and oral dosing. A time course competition assay showed the high binding to plasma proteins (>99%) not to be restrictive using the DCC method to assess the dissociation rate.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> These data indicated that any nonspecific reversible-covalent interactions with plasma proteins were not slowing the dynamics of the interaction. Incubation of <b>24</b> with rat blood showed >50% reduction to the M + 2 metabolite within 1 h, indicating that extrahepatic metabolism could also be contributing to the high in vivo clearance. Low bioavailability was also observed following oral dosing, and the high permeability supported first-past and/or gut metabolism to be the factors most likely limiting oral exposure. Incubation of <b>24</b> with rat feces under anaerobic conditions showed significant reduction to the M + 2 metabolite after 20 h and indicated that bacterial metabolism was likely also making some contribution to the low oral bioavailability.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> More rigorous metabolite identification studies (MetID) confirmed the identity of the M + 2 metabolite to be the benzylic alcohol <b>30</b>, both in vitro and in vivo, by comparison with an authentic sample. Further in vivo MetID studies also highlighted hydroxylated metabolites of the benzylic alcohol to be circulating in rat. Interestingly, with reference samples of the 2-pyridylcarboxylic acids in hand as putative oxidative metabolites of the aldehyde group, in the majority of MetID studies no increase in the M + 16 signal corresponding to the acid could be detected, for example, <b>31</b> in the case of <b>24</b>. Small amounts of the acid were present in several of the 2-FTHNU samples as byproducts from the syntheses or from oxidation upon storage (<1%), complicating the possibility of understanding their role in the aldehyde metabolism. The benzylic alcohol <b>30</b> was incubated in vitro with rat liver microsomes and dosed intravenously to rats (1 mg/kg). None of the samples analyzed from these studies showed the presence of the aldehyde <b>24</b> (lower limit of quantification 1.3 nM). Therefore, a simple elimination process appeared to be operating in which the aldehyde is cleared by reduction and not a more complex redox system where the aldehyde and benzylic alcohol would be in equilibrium (involving both oxidizing and reducing enzymes).</div><figure id="fig12" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0012.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Rat PK studies with <b>24</b>: (A) PK curves and parameters following iv (1.0 mg/kg in 7:3 PEG 200/NMP) and po (3.0 mg/kg suspension in 99.5:0.5 CMC-05/Tween 80) dosing; (B) acid and alcohol derivatives of <b>24</b>; (C) rat liver microsome MetID study.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Limited studies were carried out to understand the enzyme(s) responsible for the aldehyde reduction, and these are captured for <b>24</b> in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>. From these studies the highest levels of reduction to the benzylic alcohol <b>30</b> were found to occur in the presence of NADPH, including some nonenzymatic reduction in the control incubations with the NADPH regenerating system. In rat liver microsomes further C-hydroxylation of the THN (m2, m3) and a small amount of glucuronidation of <b>30</b> (m1) were observed. The small amount of the acid <b>31</b> present in the sample of <b>24</b>, as a byproduct from the synthesis, remained unchanged throughout the incubations, suggesting no appreciable oxidative metabolism to the acid was taking place. Of the liver fractions explored the highest levels of reduction were observed in the rat S9 incubations. Overall these observations suggested that the most efficient enzyme(s) for carrying out the aldehyde reduction was cytosolic and NADPH dependent. Such a profile is consistent with an enzyme(s) from the keto-reductase family.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> However, reduction could be observed in all the incubations, with the exception of rat plasma, also suggesting that a number of enzymes were capable of reducing the aldehyde but with varying efficiencies. Further efforts to characterize the metabolizing enzyme(s) were not undertaken; rather the focus was placed on ways to limit the reduction and identify analogues with greater metabolic stability.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Stability of <b>24</b> upon Incubation with a Panel of Rat Liver Fractions and Blood<a class="ref internalNav" href="#tbl4-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">incubation conditions</th><th class="colsep0 rowsep0" align="center" char=".">remaining <b>24</b> (%)</th><th class="colsep0 rowsep0" align="center" char=".">formed <b>30</b> (%)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">control</td><td class="colsep0 rowsep0" align="char" char=".">100</td><td class="colsep0 rowsep0" align="char" char=".">0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">control + NADPH</td><td class="colsep0 rowsep0" align="char" char=".">89</td><td class="colsep0 rowsep0" align="char" char=".">11</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rat liver microsomes</td><td class="colsep0 rowsep0" align="char" char=".">7</td><td class="colsep0 rowsep0" align="char" char=".">36</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rat liver S9 (−NADPH)</td><td class="colsep0 rowsep0" align="char" char=".">61</td><td class="colsep0 rowsep0" align="char" char=".">39</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rat liver S9 (+NADPH)</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">99</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rat blood</td><td class="colsep0 rowsep0" align="char" char=".">22</td><td class="colsep0 rowsep0" align="char" char=".">78</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rat plasma</td><td class="colsep0 rowsep0" align="char" char=".">100</td><td class="colsep0 rowsep0" align="char" char=".">0</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl4-fn1"><div class="footnote" id="tbl4-fn1"><sup><sup>a</sup></sup><p class="last">Data are uncorrected peak areas determined by HPLC/MS<sup>(<i>n</i>)</sup> following 60 min incubations.</p></div></div></div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Improving the Metabolic Stability of the 2-FTHNU Series</h3><div class="NLM_p">Tackling the metabolic instability of the aldehyde group subsequently became the focus for the optimization of the 2-FTHNU series to enable robust efficacy in disease relevant cell lines and also to achieve the targeted oral exposure profile. Two strategies were foreseen to increase the metabolic stability of the aldehyde moiety. The first was to make use of the mesomeric effect to reduce the reactivity of the aldehyde by introducing <i>ortho</i> or <i>para</i> π-electron donating amino or ether substituents. The notion was that the resulting vinylagous amides, or esters, would be chemically less reactive (electrophilic) and, as a result, less susceptible to reduction. The second was to introduce a substituent <i>ortho</i> to the aldehyde, at the THN 3-position, to increase the steric hindrance as a way to impede the reduction by metabolizing enzymes. The pseudo cycle formed by the intramolecular hydrogen-bond between the THN N-1 position and the hydrogen of the urea was considered to be a constant and further steric hindrance to attack on the aldehyde from the opposite side.</div><div class="NLM_p">The first analogue to be prepared to explore these approaches for increasing the metabolic stability of the aldehyde was the 3-hydroxy 2-FQA <b>32</b>. Encouragingly, <b>32</b> showed a reduced clearance rate in microsome incubations and maintained a comparable level of FGFR4 activity when compared to the 3-unsubstituted analogue <b>8</b>, <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>. However, <b>32</b> satisfied both the above electronic and steric hypotheses, and as a result, the best strategy to move forward remained an unanswered question.</div><figure id="fig13" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0013.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 13. Approaches taken to stabilize the 2-FTHNU aldehyde group toward metabolism: (A) 2-FQA and 2-FTHNU 3-substituent SAR; (B) interaction of the THN 3-substituent with Thr499 modeled with <b>37</b> bound within the ATP-pocket of FGFR4; (C) equilibrium between the aldehyde and hemiacetal forms of the 3-hydroxymethyl substituted analogues; (D) diols arising from the reduction of the aldehydes <b>42</b> and <b>43</b>; (E) THF-ether 4-substituted pyridine analogues.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig13"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Biochemical and Cellular FGFR4 Inhibitory Activities and Hepatic Extraction Ratios Determined from Rat Liver Microsome Incubations for Compounds <b>32</b>–<b>58</b><a class="ref internalNav" href="#tbl5-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="4" align="center">IC<sub>50</sub> (nM)</th><th class="colsep0 rowsep0" align="center"> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">biochemical FGFR4</th><th class="colsep0 rowsep0" align="center">BaF3 pFGFR4</th><th class="colsep0 rowsep0" align="center">FGFR4 BaF3 proliferation</th><th class="colsep0 rowsep0" align="center">HUH7 proliferation</th><th class="colsep0 rowsep0" align="center">rat microsome hepatic ER</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>32</b></td><td class="colsep0 rowsep0" align="left">58 ± 22</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.24</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>33</b></td><td class="colsep0 rowsep0" align="left">33 ± 19</td><td class="colsep0 rowsep0" align="left">78</td><td class="colsep0 rowsep0" align="left">244</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.94</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>34</b></td><td class="colsep0 rowsep0" align="left">630</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.67</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>35</b></td><td class="colsep0 rowsep0" align="left">387 ± 246</td><td class="colsep0 rowsep0" align="left">1007 ± 243</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.76</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>36</b></td><td class="colsep0 rowsep0" align="left">62</td><td class="colsep0 rowsep0" align="left">216</td><td class="colsep0 rowsep0" align="left">441</td><td class="colsep0 rowsep0" align="left">>3000</td><td class="colsep0 rowsep0" align="left">0.61</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>37</b></td><td class="colsep0 rowsep0" align="left">23</td><td class="colsep0 rowsep0" align="left">317</td><td class="colsep0 rowsep0" align="left">537</td><td class="colsep0 rowsep0" align="left">>3000</td><td class="colsep0 rowsep0" align="left">0.95</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>38</b></td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="left">126</td><td class="colsep0 rowsep0" align="left">317</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.75</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>39</b></td><td class="colsep0 rowsep0" align="left">800</td><td class="colsep0 rowsep0" align="left">>3000</td><td class="colsep0 rowsep0" align="left">>3000</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.44</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>40</b></td><td class="colsep0 rowsep0" align="left">28</td><td class="colsep0 rowsep0" align="left">>3000</td><td class="colsep0 rowsep0" align="left">>3000</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.33</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>41</b></td><td class="colsep0 rowsep0" align="left">2.0</td><td class="colsep0 rowsep0" align="left">2.9 ± 1.5</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.92</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>42</b></td><td class="colsep0 rowsep0" align="left">3.2 ± 1.9</td><td class="colsep0 rowsep0" align="left">8.7 ± 3.0</td><td class="colsep0 rowsep0" align="left">9.5</td><td class="colsep0 rowsep0" align="left">41 ± 7</td><td class="colsep0 rowsep0" align="left">0.66</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>43</b></td><td class="colsep0 rowsep0" align="left">2.9 ± 1.2</td><td class="colsep0 rowsep0" align="left">7.5 ± 1.5</td><td class="colsep0 rowsep0" align="left">13</td><td class="colsep0 rowsep0" align="left">72 ± 5</td><td class="colsep0 rowsep0" align="left">0.62</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>44</b></td><td class="colsep0 rowsep0" align="left">20 ± 7</td><td class="colsep0 rowsep0" align="left">52 ± 20</td><td class="colsep0 rowsep0" align="left">82 ± 18</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.64</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>45</b></td><td class="colsep0 rowsep0" align="left">2.4 ± 0.1</td><td class="colsep0 rowsep0" align="left">17</td><td class="colsep0 rowsep0" align="left">22</td><td class="colsep0 rowsep0" align="left">106 ± 8</td><td class="colsep0 rowsep0" align="left">0.68</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>46</b></td><td class="colsep0 rowsep0" align="left">0.9 ± 0.4</td><td class="colsep0 rowsep0" align="left">9.5 ± 3.8</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">27 ± 1</td><td class="colsep0 rowsep0" align="left">0.68</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>47</b></td><td class="colsep0 rowsep0" align="left">5.8</td><td class="colsep0 rowsep0" align="left">39</td><td class="colsep0 rowsep0" align="left">72</td><td class="colsep0 rowsep0" align="left">387</td><td class="colsep0 rowsep0" align="left">0.66</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>48</b></td><td class="colsep0 rowsep0" align="left">3.0 ± 1.0</td><td class="colsep0 rowsep0" align="left">21 ± 3</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">52</td><td class="colsep0 rowsep0" align="left">0.42</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>49</b></td><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left">18 ± 5</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">656</td><td class="colsep0 rowsep0" align="left">0.10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>50</b></td><td class="colsep0 rowsep0" align="left">14 ± 2</td><td class="colsep0 rowsep0" align="left">28 ± 12</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">340</td><td class="colsep0 rowsep0" align="left">0.54</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>51</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">50 ± 6</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">2056</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>52</b></td><td class="colsep0 rowsep0" align="left">2.5 ± 1.7</td><td class="colsep0 rowsep0" align="left">8.7 ± 1.4</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">53</td><td class="colsep0 rowsep0" align="left">0.12</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>53</b></td><td class="colsep0 rowsep0" align="left">8.2 ± 4.9</td><td class="colsep0 rowsep0" align="left">23</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">193</td><td class="colsep0 rowsep0" align="left">0.34</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>54</b></td><td class="colsep0 rowsep0" align="left">8.8</td><td class="colsep0 rowsep0" align="left">34 ± 1</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">1635 ± 355</td><td class="colsep0 rowsep0" align="left">0.29</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>57</b></td><td class="colsep0 rowsep0" align="left">4.5 ± 0.2</td><td class="colsep0 rowsep0" align="left">12 ± 1</td><td class="colsep0 rowsep0" align="left">22</td><td class="colsep0 rowsep0" align="left">463 ± 117</td><td class="colsep0 rowsep0" align="left">0.56</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>58</b></td><td class="colsep0 rowsep0" align="left">4.1 ± 0.4</td><td class="colsep0 rowsep0" align="left">14 ± 5</td><td class="colsep0 rowsep0" align="left">36</td><td class="colsep0 rowsep0" align="left">255 ± 205</td><td class="colsep0 rowsep0" align="left">0.52</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl5-fn1"><div class="footnote" id="tbl5-fn1"><sup><sup>a</sup></sup><p class="last">FGFR4 data are reported as the mean ± SD (<i>n</i> = 2–6 repeats) or as single measurements. Rat microsome hepatic ER data are reported as the mean (1–3 repeats).</p></div></div></div><div class="NLM_p">Further investigation revealed that a wide range of THN 3-substituents were tolerated with FGFR4 biochemical IC<sub>50</sub> values of <50 nM, and <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a> highlight some of the key SAR. THN 3-substituents that gave similar levels of FGFR4 activity to their unsubstituted analogues were methyl <b>33</b>, hydroxymethyl <b>36</b>, chloro <b>37</b>, cyclopropyl <b>38</b>, methylcarboxylic acid <b>40</b>, bromo <b>41</b>, methoxymethyl <b>44</b>, difluoromethyl <b>45</b>, trifluoromethyl <b>47</b>, C- and N-bonded aromatic heterocycles <b>48</b>–<b>51</b>, and saturated C-bonded heterocycles <b>52</b>–<b>54</b>. In contrast, the 3-amino <b>34</b>, 3-methoxy <b>35</b>, and 3-dimethylaminomethyl <b>39</b> substituents were found to result in a decrease in FGFR4 activity. These observations were consistent with molecular modeling which indicated that the THN 3-substituent would initially make van der Waals contact with the γ-methyl group of Thr499 and then be orientated out toward the protein/solvent interface, with space to accommodate a diverse range of substituents. The above analogues retaining FGFR4 activity were rationalized to be making favorable interactions with the side chain of Thr499. In contrast, the 3-amino in <b>34</b> and the 3-methoxy in <b>35</b> created an unfavorable hydrophobic mismatch at this position. Similarly, for <b>39</b>, the dimethylamino group is still in contact with the γ-methyl group of Thr499, creating the same hydrophobic mismatch, and also remains buried in the protein, as it is too short to fully reach into the solvent.</div><div class="NLM_p">Evidence for the electronic hypothesis to be operating could be derived from the stability of the 3-amino derivative <b>34</b>, but the reduced level of FGFR4 activity made this substituent less appealing for further follow-up. In contrast, the morpholine analogue <b>20</b>, in which oxygen atom <i>para</i> to the aldehyde also satisfied the electronic hypothesis, was found to be highly unstable in rat microsome incubations. The electron-withdrawing fluoromethyl analogues <b>45</b>–<b>47</b> were clearly more electrophilic, based upon readily forming reversible addition products (hydrates and hemiacetals), and showed an increase in metabolic stability. This observation suggested that the THN 3-substitutent could also stabilize more electrophilic aldehydes. However, an alternative rationale could be formulated that the equilibrium favoring the tetrahedral addition products provided protection for the aldehyde group toward reduction. To explore this possibility, an internal nucleophile was introduced to give the 3-hydroxymethyl analogues <b>42</b> and <b>43</b>. <sup>1</sup>H NMR analyses of <b>42</b> and <b>43</b> revealed the major species to be the internal hemiacetals <b>55</b> (23–27% aldehyde by <sup>1</sup>H NMR integration in DMSO-<i>d</i><sub>6</sub>). However, this aldehyde/hemiacetal equilibrium clearly provided sufficient aldehyde to inhibit FGFR4 with a high level of potency and to be enzymatically reduced to the diols <b>56</b> (assigned by comparison with authentic samples) at a similar rate to the corresponding methyl ether <b>44</b>. This suggested that the introduction of an internal equilibrium between aldehyde and a tetrahedral addition product was not an effective approach to influence metabolic stability. Encouragingly, for the examples where increased rat metabolic stability was determined, similar increases were observed across species, including human (human microsome ER: <b>42</b> 0.47; <b>43</b> 0.45; <b>45</b> 0.56; <b>46</b> 0.49).</div><div class="NLM_p">Having increased the potency and metabolic stability of the 2-FTHNUs, the antiproliferative activity in HCC cell lines was evaluated to understand to what extent metabolism was still potentially limiting efficacy.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> HUH7 antiproliferative data are included in <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>, and <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a> shows the antiproliferative activity of representative compounds in additional FGFR4-dependent HCC (Hep3B, JHH7) and RMS (RH-41) cell lines. The three reference compounds infigratinib, <b>5</b>, and <b>6</b> in <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a> inhibited the growth of all four cell lines with similar potency (<3-fold variation in the IC<sub>50</sub> values) and within 10-fold of their pFGFR4 Ba/F3 activities. Encouragingly, antiproliferative activity could now be observed with the 3-substituted 2-FTHNU derivatives. However, the 2-FTHNUs showed a much higher degree of variability across the four cell lines (up to >43-fold) and were up to >345-fold less active compared to their pFGFR4 Ba/F3 activities. Collectively these data suggested there was an improvement through the introduction of the 3-substituents (for the examples included in <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>), but there remained further scope to increase the antiproliferative activities of these compounds. Interestingly, by comparison of <b>42</b> with <b>43</b> and of <b>45</b> with <b>46</b>, both pairs only differ by a single atom that connects the substituent at the pyridyl 4-position. For both pairs the ether-linked analogues <b>42</b> and <b>45</b> showed no antiproliferative activity in the Hep3B and JHH7 assays (IC<sub>50</sub> > 3 μM), while the corresponding amino-linked analogues were active in each instance, with IC<sub>50</sub> values as low as 44 nM. However, this was not the case for all 2-methoxyethyl containing pyridine analogues: <b>47</b> exhibited similar IC<sub>50</sub> values across all three HCC cell lines, but IC<sub>50</sub> was >3 μM in the RH-41 assay. These observations highlight that additional factors other than the metabolic stability of the aldehyde are also likely contributing to the observed activities, potentially associated with other metabolic liabilities within the molecules.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Antiproliferative Activities in Hep3B, JHH7, and RH-41 Cell Lines for Representative 2-FTHNUs<a class="ref internalNav" href="#tbl6-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="3" align="center">growth inhibition IC<sub>50</sub> (nM)</th><th class="colsep0 rowsep0" align="center"> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">Hep3B</th><th class="colsep0 rowsep0" align="center">JHH7</th><th class="colsep0 rowsep0" align="center">RH-41</th><th class="colsep0 rowsep0" align="center">human microsome hepatic ER</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">infigratinib</td><td class="colsep0 rowsep0" align="left">85 ± 36</td><td class="colsep0 rowsep0" align="left">67 ± 28</td><td class="colsep0 rowsep0" align="left">172 ± 144</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="left">6.8</td><td class="colsep0 rowsep0" align="left">7.1</td><td class="colsep0 rowsep0" align="left">2.5</td><td class="colsep0 rowsep0" align="left">0.69</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="left">261 ± 230</td><td class="colsep0 rowsep0" align="left">282 ± 130</td><td class="colsep0 rowsep0" align="left">205 ± 92</td><td class="colsep0 rowsep0" align="left">0.24</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>36</b></td><td class="colsep0 rowsep0" align="left">719</td><td class="colsep0 rowsep0" align="left">>3000</td><td class="colsep0 rowsep0" align="left">>3000</td><td class="colsep0 rowsep0" align="left">0.43</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>42</b></td><td class="colsep0 rowsep0" align="left">>3000</td><td class="colsep0 rowsep0" align="left">>3000</td><td class="colsep0 rowsep0" align="left">>3000</td><td class="colsep0 rowsep0" align="left">0.47</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>43</b></td><td class="colsep0 rowsep0" align="left">73 ± 3</td><td class="colsep0 rowsep0" align="left">212 ± 116</td><td class="colsep0 rowsep0" align="left">>3000</td><td class="colsep0 rowsep0" align="left">0.45</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>45</b></td><td class="colsep0 rowsep0" align="left">>3000</td><td class="colsep0 rowsep0" align="left">>3000</td><td class="colsep0 rowsep0" align="left">>3000</td><td class="colsep0 rowsep0" align="left">0.56</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>46</b></td><td class="colsep0 rowsep0" align="left">44 ± 30</td><td class="colsep0 rowsep0" align="left">1003 ± 967</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.49</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>47</b></td><td class="colsep0 rowsep0" align="left">265</td><td class="colsep0 rowsep0" align="left">1183</td><td class="colsep0 rowsep0" align="left">1154</td><td class="colsep0 rowsep0" align="left">0.54</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>58</b></td><td class="colsep0 rowsep0" align="left">483 ± 180</td><td class="colsep0 rowsep0" align="left">418 ± 90</td><td class="colsep0 rowsep0" align="left">>3000</td><td class="colsep0 rowsep0" align="left">0.62</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl6-fn1"><div class="footnote" id="tbl6-fn1"><sup><sup>a</sup></sup><p class="last">Data are reported as the mean ± SD (<i>n</i> = 2–16 repeats) or as single measurements. Human microsome hepatic ER data are single measurements.</p></div></div></div><div class="NLM_p">To gain a further insight into what was still limiting the antiproliferative activity in cancer cell lines, more detailed time-course PD studies were carried out with selected 2-FTHNUs. Data for analogues <b>42</b> and <b>58</b> bearing 3-hydroxymethyl substituents, compared to infigratinib, are shown in <a data-tab="pane-pcw-Figures" href="#fig14" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig14" id="" class=" internalNav">14</a></a>. Both analogues exhibited similar levels of increased stability in rat and human liver microsome incubations when compared to the 3-unsubstituted examples <b>19</b>–<b>29</b>, including the matched pair <b>42</b> versus <b>28</b> (human ER of 0.93). An antiproliferative IC<sub>50</sub> < 1 μM was determined for <b>58</b>, but for <b>42</b> 50% inhibition of cell growth was not observed up to a concentration of 3 μM, possibly as a result of the instability of the 3-(2-methoxyethyl ether) substituent in the Hep3B cell line. For <b>58</b> this still represented a >20-fold shift in potency compared to the ability to inhibit FGFR4 autophosphorylation in BaF3 cells (pFGFR4). Consistent with the notion that metabolism was still limiting the efficacy of <b>42</b> and <b>58</b>, time and concentration dependent decreases in pathway inhibition were observed, based upon pFRS2 and pMAPK (pErk) Western blot analyses. For <b>42</b>, significant pathway inhibition was lost between 2 and 4 h of incubation at a concentration of 50 nM and between 6 and 24 h at a 10-fold higher concentration. For <b>58</b>, in line with the observed antiproliferative IC<sub>50</sub> value, pathway inhibition was maintained out to longer time points, with inhibition lost at the higher concentration between 24 and 48 h. In contrast, infigratinib, at a concentration of 500 nM, was shown to inhibit the pathway fully throughout the whole 72 h incubation period, leading to the lowest antiproliferation IC<sub>50</sub> value. Additionally, for <b>58</b>, the loss of PD modulation over time could be correlated with the disappearance of the parent compound in the incubation media and with the concurrent formation of an M + 2 species (data not shown). These observations supported the further exploration of THN 3-substituents as a way to limit the metabolism of the aldehyde by reducing enzyme(s); however the opportunity remained to further increase this effect.</div><figure id="fig14" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 14</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0014.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 14. PD time course profiles for infigratinib, <b>42</b>, and <b>58</b> in Hep3B cells. The Western blots show pFRS2 and pMAPK (pErk) as PD readouts with the α-actin protein levels as loading control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig14"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Although the opportunity to further improve upon the metabolic stability of the 2-FTHNU series had been highlighted, at this point in the project the possibility of in vivo target modulation was considered to be achievable and was explored for the first time. The 3-hydroxymethyl analogue <b>43</b> exhibited a low-dose oral PK profile in the mouse that suggested PD modulation could be achievable within an acceptable dose range, <a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">15</a></a>. Additionally, mouse plasma protein binding measurements indicated an unbound fraction of 12%, a value that was relatively high compared to the majority of other 2-FTHNU derivatives in <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a> (typically PPB > 99% where measured). Similar higher unbound fractions were observed in the mouse for the 3-hydroxymethyl analogues <b>42</b> and <b>58</b> (mouse PPB <b>42</b> 90%, <b>58</b> 91%) but were found to be more variable across species (rat PPB <b>42</b> 98%, <b>43</b> 96%, <b>58</b> >99%, human PPB <b>42</b> 98%, <b>43</b> 91%). For <b>43</b> this level of protein binding was considered to be potentially beneficial if the unbound fraction was found to be a key parameter for predicting efficacy. The PK/PD studies were conducted in two RMS mouse xenograft models derived from the RH30 and RH41 cell lines which constitutively overexpress activated FGFR4 and as a result provided strong pFGFR4 signals ideal for PD characterization.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> PD was assessed by the degree of inhibition of FGFR4 autophosphorylation in both studies, and in the RH30 study pErk was also measured as a second downstream PD readout. Dosed orally at 30 mg/kg, <b>43</b> produced strong PD responses at time points of 4 and 8 h following dosing in the RH41 and RH30 xenografts, respectively. The inhibitory effect was reduced at longer time-points and was consistent with lower levels of circulating parent compound (total plasma concentration of <40 nM). Observing that such a relatively low circulating concentration was still capable of inhibiting the pathway suggested that protein binding was not restricting the activity, likely as a consequence of the longer half-life associated with the reversible mechanism of FGFR4 inhibition. Some disconnection of effect from exposure could be anticipated for the reversible-covalent interaction, due to a slow inhibitor/FGFR4 dissociation rate; <i>vide infra</i>. However, the relatively rapid resynthesis rate of FGFR4 was anticipated to limit any disconnect to <2 h.</div><figure id="fig15" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Figure 15</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0015.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 15. In vivo profile of <b>43</b> in the mouse: (A) PK parameters following iv (1.0 mg/kg) and oral dosing (suspension, 3.0 mg/kg); (B) time course PK/PD and total plasma concentrations with RH30 and RH41 mouse xenografts (Western blot, RH30 normalized to total ERK; RH41 normalized to the lysate inputs, 1 mg protein) following oral dosing (suspension, 30 mg/kg); (C) efficacy studies with the Hep3B mouse xenograft model (<i>n</i> = 6, mean ± SEM), *<i>P</i> < 0.05 versus vehicle by one way ANOVA with post-hoc Dunnett’s test; (D) effects on liver CYP7A1 mRNA and circulating BA levels following 2 weeks dosing, *<i>P</i> < 0.05 versus vehicle by heteroscedastic Student’s <i>t</i>-test, **<i>P</i> < 0.05 versus vehicle by one way ANOVA with Dunnett’s multiple comparisons test.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig15"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Having demonstrated a sustained PD response, an efficacy study was carried out with <b>43</b> in a mouse xenograft model using the Hep3B HCC cell line.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> Dosed orally twice daily (b.i.d.) at 30 mg/kg, a significant decrease in tumor growth was observed with a treated over control tumor size ratio (T/C) of 32% determined after 14 days of dosing, <a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">15</a></a>. At double the dose (60 mg/kg b.i.d.), a further decease in tumor growth was observed, approximately leading to stasis after the 14 day study (T/C 1%). Both dose levels were considered to be well tolerated over the 2 week treatment period based upon observation and no difference in body weights between the inhibitor and vehicle treated groups (data included in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01019/suppl_file/jm0c01019_si_001.pdf" class="ext-link">Supporting Information</a>).</div><div class="NLM_p">In addition to the standard tolerability parameters, two of the anticipated biomarkers of FGFR4 inhibition in the host were also measured at the end of the 2 week studies: circulating BA levels and liver CYP7A1 levels. Both parameters were determined from necropsy samples collected 2 and 4 h following the final doses. At both dose levels, the expected increases in liver CYP7A1 mRNA and circulating BA levels were observed when compared to the vehicle treated animals.</div><div class="NLM_p">Within the 3-substituted THNU examples shown in <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>, evidence could be found for both the electronic and the steric hypotheses to be operating to stabilize the aldehyde group. However, the steric blocking effect proved to be the one that was consistently most effective, and the size of the substituent was found to play a role. Smaller 3-substituents only had a modest impact upon the stability in microsome incubations, such as methyl (<b>33</b>) and chloro substituted compound <b>37</b>. However, larger 3-substituents tended toward analogues with increased microsome stability, as exemplified by the cyclopropyl (<b>38</b>), difluoromethyl (<b>45</b> and <b>46</b>), and trifluoromethyl substituted compound <b>47</b>. <a data-tab="pane-pcw-Figures" href="#fig16" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig16" id="" class=" internalNav">16</a></a> shows the relationship between the size of the THN 3-substituent and the stability in rat liver microsome incubations from an analysis that was made during the optimization of the 2-FTHNU series, taking three heavy atoms as the differentiation point between large and small substituents. A measure of lipophilicity is also included in the plot (cLogP) as a parameter that typically exhibits a strong correlation with susceptibility to oxidative metabolism.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> The plot clearly shows a high level of instability in rat liver microsomes for all of the 3-unsubstituted 2-FTHNUs (ER > 0.9), the ER range 0.5–0.9 is equally populated with 2-FTHNUs with both large and small 3-substituents, whereas for ER values below 0.5 the plot is predominantly populated with the larger 3-substituents (three or more heavy atoms). Lipophilicity is a parameter that is typically inversely related to metabolic stability, and this relationship can be observed to some extent for the series: none of the 2-FTHNUs with large 3-substituents and of lower lipophilicity (cLogP < 1.0) show a hepatic ER of >0.6. Although multiple factors undoubtedly impact upon the observed clearance rates, the introduction of the larger THN 3-substituents produced a marked shift toward analogues with more favorable levels of metabolic stability.</div><figure id="fig16" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Figure 16</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0016.jpeg" id="gr16" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 16. Relationship between cLogP and metabolic stability in rat liver microsome incubations highlighting the effect of the size of the THN 3-substitutent. Compounds are colored according to the size of the 3-substitutent: unsubstituted in brown; less than three heavy atoms in light blue; three or more heavy atoms in dark blue.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig16"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Encouraged by the in vivo efficacy data with compound <b>43</b>, the further optimization of the 2-FTHNU series was focused upon identifying larger 3-THNU substituents to achieve the best balance between FGFR4 potency and metabolic stability. Following this approach, compound <b>59</b>, with a 3-substituent composed of six heavy atoms, highlighted what proved to be a key opportunity, <a data-tab="pane-pcw-Figures" href="#fig17" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig17" id="" class=" internalNav">17</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>. The 3-N-acylated aminomethyl analogue <b>59</b> combined both a high level of FGFR4 potency and a high level of metabolic stability across species (human liver microsome ER of 0.37). The increased activity of <b>59</b> could be rationalized through molecular modeling which suggested favorable interactions between the benzylic methylene and the γ-methyl group of Thr499, and a hydrogen bond between the side chain of Arg483 and acetamide carbonyl. Additionally, <b>59</b> showed a low but measurable level of solubility which was in contrast to the other analogues with encouraging profiles which contained a heterocycle directly attached at the THN 3-position such as <b>48</b> and <b>51</b> (aqueous solubility at pH 6.8: <b>59</b>, 18 μM; <b>48</b> and <b>51</b>, <4 μM). Introducing the optimal 4-pyridyl substituents into <b>59</b> followed the anticipated SAR to give the highly potent analogues <b>60</b>–<b>62</b> containing isopropylamino, methoxyethyl ether, and methoxyethylamino, respectively. Importantly, these analogues <b>60</b>–<b>62</b> also retained the increased level of microsome stability determined for <b>59</b>. Additionally, <20-fold difference between the cellular BaF3 pFGFR4 and HUH7 antiproliferative IC<sub>50</sub> values was observed for <b>60</b> and <b>61</b>, but the 2-methoxyethoxy substituted pyridine derivative <b>62</b> again showed a larger potency shift of 36-fold.</div><figure id="fig17" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Figure 17</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0017.jpeg" id="gr17" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 17. <i>N</i>-Acylaminomethyl 2-FTHNU 3-substituent SAR: (A) <i>N</i>-methyl-<i>N</i>-acyl analogues; (B) modeled interaction of compound <b>59</b> within the ATP-binding site of FGFR4 highlighting the H-bond between the carbonyl of the <i>N</i>-acetyl moiety and the side chain of Arg483; (C) selected <i>N</i>-acetyl and <i>N</i>-alkyl analogues; (D) modeled interaction of the enantiomers of compound <b>70</b> within the ATP-binding site of FGFR4 highlighting the unfavorable interactions made by the benzylic methyl moiety.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig17"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. Biochemical and Cellular FGFR4 Inhibitory Activities and Hepatic Extraction Ratios Determined from Rat Liver Microsome Incubations for Compounds <b>59</b>–<b>70</b><a class="ref internalNav" href="#tbl7-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="4" align="center">IC<sub>50</sub> (nM)</th><th class="colsep0 rowsep0" align="center"> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">biochemical FGFR4</th><th class="colsep0 rowsep0" align="center">BaF3 pFGFR4</th><th class="colsep0 rowsep0" align="center">FGFR4 BaF3 proliferation</th><th class="colsep0 rowsep0" align="center">HUH7 proliferation</th><th class="colsep0 rowsep0" align="center">rat microsome hepatic ER</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>59</b></td><td class="colsep0 rowsep0" align="left">35</td><td class="colsep0 rowsep0" align="left">55 ± 29</td><td class="colsep0 rowsep0" align="left">102 ± 17</td><td class="colsep0 rowsep0" align="left">1092 ± 451</td><td class="colsep0 rowsep0" align="left">0.30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>60</b></td><td class="colsep0 rowsep0" align="left">0.4</td><td class="colsep0 rowsep0" align="left">2.5 ± 0.1</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">21 ± 15</td><td class="colsep0 rowsep0" align="left">0.53</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>61</b></td><td class="colsep0 rowsep0" align="left">1.3 ± 0.6</td><td class="colsep0 rowsep0" align="left">20 ± 8</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">141 ± 32</td><td class="colsep0 rowsep0" align="left">0.57</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>62</b></td><td class="colsep0 rowsep0" align="left">2.1 ± 0.4</td><td class="colsep0 rowsep0" align="left">5.0 ± 0.7</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">178 ± 94</td><td class="colsep0 rowsep0" align="left">0.23</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>63</b></td><td class="colsep0 rowsep0" align="left">1.2 ± 0.2</td><td class="colsep0 rowsep0" align="left">5.9 ± 1.9</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">89 ± 14</td><td class="colsep0 rowsep0" align="left">0.57</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>64</b></td><td class="colsep0 rowsep0" align="left">0.3 ± 0.1</td><td class="colsep0 rowsep0" align="left">3.8 ± 0.4</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">68 ± 37</td><td class="colsep0 rowsep0" align="left">0.26</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>65</b></td><td class="colsep0 rowsep0" align="left">0.9 ± 0.1</td><td class="colsep0 rowsep0" align="left">4.5 ± 1.9</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">47 ± 6</td><td class="colsep0 rowsep0" align="left">0.41</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>66</b></td><td class="colsep0 rowsep0" align="left">1.1</td><td class="colsep0 rowsep0" align="left">13 ± 4</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">178 ± 48</td><td class="colsep0 rowsep0" align="left">0.68</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>67</b></td><td class="colsep0 rowsep0" align="left">0.8</td><td class="colsep0 rowsep0" align="left">23 ± 7</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">604</td><td class="colsep0 rowsep0" align="left">0.44</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>68</b></td><td class="colsep0 rowsep0" align="left">0.7 ± 0.5</td><td class="colsep0 rowsep0" align="left">7.0 ± 3.6</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">178 ± 110</td><td class="colsep0 rowsep0" align="left">0.66</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>69</b></td><td class="colsep0 rowsep0" align="left">0.4 ± 0.2</td><td class="colsep0 rowsep0" align="left">4.5 ± 1.2</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">25 ± 8</td><td class="colsep0 rowsep0" align="left">0.31</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>70</b></td><td class="colsep0 rowsep0" align="left">58 ± 1</td><td class="colsep0 rowsep0" align="left">100 ± 6</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.26</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl7-fn1"><div class="footnote" id="tbl7-fn1"><sup><sup>a</sup></sup><p class="last">FGFR4 data are reported as the mean ± SD (<i>n</i> = 2–6 repeats) or as single measurements. Rat microsome hepatic ER data are reported as the mean (1–3 repeats).</p></div></div></div><div class="NLM_p">From further exploration of the SAR around the 3-N-acylated aminomethyl substituent, increasing the size of the acyl group or the <i>N</i>-alkyl substituent was tolerated with little impact on the level of FGFR4 inhibitory activity, as represented by analogues <b>63</b>–<b>66</b>. N-Sulfonylation appeared to offer no advantage in cellular activity when compared to the amide derivatives, based upon the methylsulfonamide <b>67</b>. Cyclization between the acyl and N-substituents to form lactam derivatives was tolerated, as exemplified by the butyrolactam <b>68</b>, and morpholine lactam <b>69</b>, which maintained a high level of FGFR4 potency and rat microsome stability. Introducing a benzylic methyl group (<b>70</b>, racemic), with the rationale to further increase the steric bulk proximal to the aldehyde, did lower the rat microsome clearance. However, this methyl group led to a decrease in FGFR4 potency which was initially surprising considering that the equivalent level of substitution had been well tolerated at this position in the aromatic and saturated heterocycle substituted 2-FTHNUs <b>48</b>–<b>54</b>. Molecular modeling suggested the constraint of the H-bond between Arg483 and the acetamide carbonyl could explain the observations for both enantiomers. For the (<i>R</i>)-enantiomer of <b>70</b> with the H-bond in place, the benzylic methyl group would be anticipated to clash with the side chain of Cys552. For the (<i>S</i>)-enantiomer, the low energy conformation of the 3-N-acylated aminomethyl substituent is not compatible with maintaining a H-bond with Arg483, <a data-tab="pane-pcw-Figures" href="#fig17" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig17" id="" class=" internalNav">17</a></a>.</div><div class="NLM_p">To assess the impact of the increased in vitro metabolic stability in vivo, PK studies were performed in the rat for selected 2-FTHNUs, <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">Table <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">8</a></a>. For the analogues with smaller 3-THN substituents, such as <b>42</b> and <b>46</b> (two and three heavy atoms, respectively), similar clearance rates compared to the 3-unsubstituted <b>24</b> were observed. In contrast, oral dosing of <b>42</b> and <b>46</b> did produce some improvement in the oral PK profiles when comparing experiments run with suspension formulations of the crystalline material. In comparison to the 3-unsubstituted analogue <b>24</b>, <b>42</b> showed a 6-fold and <b>46</b> showed a 12-fold increase in oral bioavailability. However, the oral PK profiles of all 3-THNUs included in <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">Table <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">8</a></a> were predicted to be dissolution limited due to their low solubilities, the low solubility resulting from high heats of fusion (Δ<i>H</i><sub>fus</sub>) determined by differential scanning calorimetry (DSC). This was considered to be due to favorable packing of the flat molecules and supported by X-ray crystallography which indicated a high number of favorable intermolecular contacts within the crystal lattices.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> Consistent with this hypothesis, oral dosing of <b>46</b> as a nanosuspension produced a 10-fold increase in bioavailability (80 ± 66%) when compared to the material dosed as a suspension of the crystalline material, supporting the assumption of dissolution limited solubility. The high level of variability in the oral bioavailability determined with the enhanced formulation was also consistent with the profile anticipated for a poorly soluble compound, likely due to reprecipitation within the gut. This solubility dependency made understanding the influence of metabolic clearance on the level of oral bioavailability for each compound more challenging. With this in mind, comparisons of the 2-FTHNU oral PK profiles were made with caution at this stage, when solubility was predicted to be a limiting factor.</div><div class="NLM_table-wrap" id="tbl8"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 8. Rat PK Parameters for Selected 2-FTHNU Derivatives<a class="ref internalNav" href="#tbl8-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="5" align="center">compd</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">parameter</th><th class="colsep0 rowsep0" align="center"><b>42</b></th><th class="colsep0 rowsep0" align="center"><b>46</b></th><th class="colsep0 rowsep0" align="center"><b>48</b></th><th class="colsep0 rowsep0" align="center"><b>61</b></th><th class="colsep0 rowsep0" align="center"><b>68</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CL (mL min<sup>–1</sup> kg<sup>–1</sup>)</td><td class="colsep0 rowsep0" align="left">52 ± 8</td><td class="colsep0 rowsep0" align="left">40 ± 3</td><td class="colsep0 rowsep0" align="left">8 ± 1</td><td class="colsep0 rowsep0" align="left">16 ± 3</td><td class="colsep0 rowsep0" align="left">21 ± 4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>V</i><sub>ss</sub> (L/kg)</td><td class="colsep0 rowsep0" align="left">2.5 ± 0.2</td><td class="colsep0 rowsep0" align="left">4.7 ± 0.4</td><td class="colsep0 rowsep0" align="left">1.5 ± 0.3</td><td class="colsep0 rowsep0" align="left">8.6 ± 1.9</td><td class="colsep0 rowsep0" align="left">7.2 ± 1.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>t</i><sub>1/2</sub> (h)</td><td class="colsep0 rowsep0" align="left">2.0 ± 0.3</td><td class="colsep0 rowsep0" align="left">2.3 ± 0.3</td><td class="colsep0 rowsep0" align="left">3.5 ± 0.5</td><td class="colsep0 rowsep0" align="left">27.7 ± 4.9</td><td class="colsep0 rowsep0" align="left">14.2 ± 1.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AUC iv (nM·h)</td><td class="colsep0 rowsep0" align="left">802 ± 135</td><td class="colsep0 rowsep0" align="left">964 ± 65</td><td class="colsep0 rowsep0" align="left">4703 ± 793</td><td class="colsep0 rowsep0" align="left">2307 ± 485</td><td class="colsep0 rowsep0" align="left">1768 ± 306</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AUC po (nM·h)</td><td class="colsep0 rowsep0" align="left">194 ± 38</td><td class="colsep0 rowsep0" align="left">774 ± 635</td><td class="colsep0 rowsep0" align="left">2174 ± 327</td><td class="colsep0 rowsep0" align="left">599 ± 104</td><td class="colsep0 rowsep0" align="left">587 ± 145</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">oral <i>F</i> (%)</td><td class="colsep0 rowsep0" align="left">24 ± 5</td><td class="colsep0 rowsep0" align="left">7 ± 2</td><td class="colsep0 rowsep0" align="left">46 ± 7</td><td class="colsep0 rowsep0" align="left">26 ± 4</td><td class="colsep0 rowsep0" align="left">33 ± 8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PPB (%)</td><td class="colsep0 rowsep0" align="left">97.8</td><td class="colsep0 rowsep0" align="left">>99</td><td class="colsep0 rowsep0" align="left">>99</td><td class="colsep0 rowsep0" align="left">94.8</td><td class="colsep0 rowsep0" align="left">>99</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">solubility (μM)</td><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left"><6.0</td><td class="colsep0 rowsep0" align="left"><6.0</td><td class="colsep0 rowsep0" align="left">8.0</td><td class="colsep0 rowsep0" align="left"><6.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">mp (°C)</td><td class="colsep0 rowsep0" align="left">181</td><td class="colsep0 rowsep0" align="left">202</td><td class="colsep0 rowsep0" align="left">210</td><td class="colsep0 rowsep0" align="left">201</td><td class="colsep0 rowsep0" align="left">197</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Δ<i>H</i><sub>fus</sub> (kJ/mol)</td><td class="colsep0 rowsep0" align="left">39.6</td><td class="colsep0 rowsep0" align="left">45.7</td><td class="colsep0 rowsep0" align="left">40.3</td><td class="colsep0 rowsep0" align="left">44.1</td><td class="colsep0 rowsep0" align="left">35.7</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl8-fn1"><div class="footnote" id="tbl8-fn1"><sup><sup>a</sup></sup><p class="last">Dosed iv (1.0 mg/kg in in 7:3 PEG 200/NMP) and po (3.0 mg/kg in 99.5:0.5 CMC-05/Tween 80). Oral doses were administered as suspensions of the crystalline material (XRPD). AUC values are dose normalized to a 1 mg/kg dose. PK parameters are reported as the mean ± SD (<i>n</i> = 3). Thermodynamic solubilities were measured in pH 6.8 buffer using the shake-flask method. Δ<i>H</i><sub>fus</sub> and mp values were determined by DSC.</p></div></div></div><div class="NLM_p last">The analogues with the larger 3-THN substituents <b>48</b>, <b>61</b>, and <b>68</b> (six or more heavy atoms) exhibited low rat in vivo clearance values and in particular for <b>61</b> which achieved this with a higher unbound fraction. In the case of the two 3-N-acylated aminomethyl examples, the lower clearance in combination with higher volumes of distribution led to relatively long terminal half-lives. All three compounds showed respectable oral bioavailabilities in the rat. With no marked increase in thermodynamic solubility, compared to compounds <b>42</b> and <b>46</b>, the lower clearance was cautiously anticipated to be a contributor toward this increase in oral bioavailability for these three examples.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Evaluation of the PK, PD, Efficacy, and Toxicity of the 2-FTHNU Series</h3><div class="NLM_p">Compound <b>68</b> also exhibited favorable low-dose PK profiles in the mouse and dog, consistent with the reduced in vitro clearance observed with the larger 3-THN substituent, <a data-tab="pane-pcw-Figures" href="#fig18" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig18" id="" class=" internalNav">18</a></a>. A series of PK/PD studies in the mouse enabled a correlation between plasma exposure and FGFR4 autophosphorylation (data are pFGFR4 normalized to total FGFR4 by Western blot and expressed as a percentage versus the vehicle treated control) to be constructed for <b>68</b> using the RMS RH30 xenograft model, <a data-tab="pane-pcw-Figures" href="#fig18" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig18" id="" class=" internalNav">18</a></a>. Such plots proved to be a good way to predict the exposure levels needed to drive in vivo efficacy across a panel of xenograft models.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> The plots are compiled from exposure/effect (PK/PD) readouts at multiple time-points. These plots are believed to be effective due to the rapid phosphorylation/dephosphorylation of FGFR4, leading to a tight temporal relationship between plasma concentration and effect (PK/PD). Additionally, the rapid resynthesis rate of FGFR4 is anticipated to further enhance the temporal relationship between PK and PD for the reversible-covalent 2-FTHNU series. For <b>68</b>, a circulating concentration of 18 nM was required to reduce the PD (pFGFR4) effect by 50%, and 315 nM was required to reduce by 90% relative to the untreated control level. The plasma concentrations required for >50% PD modulation were found to be close to the HCC cellular proliferation IC<sub>50</sub> value (HUH7 128 nM), which also highlighted the efficiency of the reversible-covalent interaction in vivo and removed any final concerns that nonspecific binding could limit the efficacy of the 2-FTHNUs. The observation of similar inhibitory concentrations for the in vitro and in vivo experiments held true for all of the 2-FTHNU derivatives where such plots were constructed, albeit for only a modest number of examples in total.</div><figure id="fig18" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Figure 18</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0018.jpeg" id="gr18" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 18. In vivo studies with compound <b>68</b>: (A) PK parameters in the mouse (iv 0.5 mg/kg in 9:1 plasma/NMP) and dog (iv 0.1 mg/kg in 9:1 PEG 200/NMP and po 0.3 mg/kg as an aq suspension), where parameters are reported as the mean ± SD (<i>n</i> = 3); (B) total plasma concentration/response curves from PK/PD studies with the mouse RH30 xenograft model; (C) plot of tumor growth inhibition with the mouse HUH7 xenograft model (<i>n</i> = 6, mean ± SEM), *<i>P</i> < 0.05 versus vehicle by Student’s <i>t</i>-test.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig18"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">An in vivo efficacy experiment was carried out with <b>68</b> dosed orally, b.i.d., as a suspension of the crystalline material, at a dose of 30 mg/kg, in a mouse xenograft model using the human HUH7 HCC cell line. Significant inhibition of tumor growth was observed in this study, relative to a vehicle treated control group, approximately leading to tumor stasis (T/C 3%) at the end of the 14 day study, <a data-tab="pane-pcw-Figures" href="#fig18" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig18" id="" class=" internalNav">18</a></a>. Day 14 exposure levels were consistent with the PK profiles observed in the single dose PK/PD studies, with a day 14 trough concentration (12 h after the last dose) of 500 nM, indicating sustained PD inhibition throughout the study (1.6-fold the IC<sub>90</sub> from the RH30 plasma concentration versus p/tFGFR4 plot). The treatment was well tolerated throughout the study based upon observation and no body weight differences between the treated and control groups (data included in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01019/suppl_file/jm0c01019_si_001.pdf" class="ext-link">Supporting Information</a>).</div><div class="NLM_p">With <b>68</b> identified as a potent, metabolically stable, and in vivo efficacious 2-FTHNU derivative, the final key question for the series that remained to be explored was related to the toxicity of the 2-formylpyridyl group. To address this question, a cross-species sequence alignment had identified the rat as a species where an alternative amino acid residue was present at the GK+2 position. Compared to the cysteine residue present in human and the other commonly used preclinical species, the rat FGFR4 sequence contains a tyrosine at this position, <a data-tab="pane-pcw-Figures" href="#fig19" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig19" id="" class=" internalNav">19</a></a>. A tyrosine residue is present at the GK+2 position in the human FGFR1–3 kinase domains, and in line with this change the 2-FTHNU series were anticipated to be inactive as inhibitors of rat FGFR4. Using a version of the biochemical assay with the equivalent rat FGFR4 kinase domain construct confirmed this prediction for all the 2-FTHNU derivatives tested, including <b>68</b> (IC<sub>50</sub> > 10 μM). This observation provided the opportunity to explore the toxicity of the 2-FTHNU series in vivo in the rat without the complication of on-target FGFR4 activity. To assess this opportunity, the 2-FTHNU derivative <b>68</b> was selected for a rat toxicology study, in part based upon the very high level of selectivity observed when screened against a series of off-target panels (internal collections of 138 receptors and enzymes, all <50% inhibition at 3 μM), including a KINOME<i>scan</i> (screening concentration 3 μM, selectivity score 0.008; significant inhibition of probe binding was only determined to three kinases, FGFR4 > 99%, RIPK5 83%, RSK1 68%).<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a></div><figure id="fig19" data-index="19" class="article__inlineFigure"><h2 class="fig-label">Figure 19</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0019.jpeg" id="gr19" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 19. Evaluation of the 2-FTHNU toxicology with compound <b>68</b>: (A) FGFR4 cross-species sequence alignment centered around the middle-hinge region highlighting the absence of a cysteine residue at the GK+2 position in the rat; (B) KINOME<i>scan</i> TREE<i>spot</i> interaction map; (C) exposure data in blood (total) from the mouse efficacy (<i>n</i> = 2) and rat toxicology (<i>n</i> = 4) studies reported as the mean ± SD.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig19"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Compound <b>68</b> was dosed orally to rats at doses of 10, 30, and 100 mg kg<sup>–1</sup> day<sup>–1</sup>, once daily (q.d.), for 10 days. Dosed as a suspension of the crystalline material, the compound was rapidly absorbed and reached maximal concentrations within 2–3.5 h after dosing. Exposures increased proportionally between the 10 and 100 mg kg<sup>–1</sup> day<sup>–1</sup> treatment groups, with no significant exposure differences between the first dose and end of study samplings, <a data-tab="pane-pcw-Figures" href="#fig19" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig19" id="" class=" internalNav">19</a></a>. At doses up to 100 mg kg<sup>–1</sup> day<sup>–1</sup>, <b>68</b> was well tolerated based upon the absence of test article-related in-life findings and/or adverse clinical pathology and anatomic pathology observations. The exposure at the highest dose level in the rat achieved a comparable total AUC level when compared to the exposure level delivering efficacy in the mouse xenograft model shown in <a data-tab="pane-pcw-Figures" href="#fig18" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig18" id="" class=" internalNav">18</a></a>C. However, estimating the dose level that would be needed in the rat to maintain the <i>C</i><sub>trough</sub> above the plasma IC<sub>90</sub> to match the <i>C</i><sub>trough</sub> measured in the mouse HCC xenograft model (30 mg/kg b.i.d.), a b.i.d. dose of 10 mg/kg was predicted to satisfy this requirement. Taking this estimation would then give a >3-fold difference between the total daily dose level predicted to lead to stasis in the rat and the highest dose from the toxicology study. No changes in serum BA levels were observed in the rat study, in contrast to what was determined in the previous mouse efficacy studies. These data supported a favorable off-target toxicology profile for the aldehyde containing 2-FTHNU <b>68</b> and that the anticipated BA changes were FGFR4 driven. With these data in hand, our attention turned to driving the 2-FTHNU series toward the identification of a candidate molecule.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Improving the Solubility of the 2-FTHNU Series</h3><div class="NLM_p">Although compound <b>68</b> had exhibited a favorable oral PK profile in the rat toxicology study, covering multiples of the efficacious exposure, the thermodynamic solubility of the compound was low (<6 μM) and considered a risk for further development. Of the strategies available to increase solubility, the incorporation of a basic center was considered the most attractive, and some of the approaches taken to achieve this are highlighted in <a data-tab="pane-pcw-Figures" href="#fig20" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig20" id="" class=" internalNav">20</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl9" href="#tbl9">Table <a class="ref showTableEvent internalNav" data-ID="tbl9" href="#tbl9">9</a></a>. Optimization of the 4-position of the hinge-binding pyridyl moiety had shown that amine-containing substituents could be incorporated in this region of the binding pocket. A modest decrease in FGFR4 activity was typically observed when basic groups were introduced into this region; for example, the dimethylaminoethoxy analogue <b>16</b> is 8-fold less potent than the corresponding methoxyethoxy analogue <b>28</b>. Analogues with a tertiary amine designed to interact directly with the aspartic acid of the activation-loop DFG motif <b>71</b>–<b>74</b> were discovered to exhibit reasonable levels of FGFR4 activity. However, when combined with the larger aldehyde-stabilizing 3-THN substituents, this approach produced molecules with higher molecular weights and polar surface areas. The increased challenge for identifying orally bioavailable compounds within this property space has been well described,<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> and this encouraged an alternative location for the basic group to be sought.</div><figure id="fig20" data-index="20" class="article__inlineFigure"><h2 class="fig-label">Figure 20</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0020.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0020.jpeg" id="gr20" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 20. Approaches to increasing the solubility of the 2-FTHNU series: (A) introducing basic groups into the hinge-binding pyridine moiety at the 4-position to target the aspartic acid of the DFG motif; (B) modeled interaction of compound <b>74</b> within the ATP-pocket of FGFR4 highlighting the interaction of the dimethylamino moiety with Asp630 of the DFG motif; (C) selected SAR from introducing basic groups into the 3-position of the THN moiety; (D) further SAR around the 3-(<i>N</i>-methylpiperidin-2-one)methyl THN containing analogues.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig20"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl9"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 9. Key Profiling Data To Identify a Candidate with Improved Solubility and a Favorable Overall Profile<a class="ref internalNav" href="#tbl9-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="3" align="center">IC<sub>50</sub> (nM)</th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">biochemical FGFR4</th><th class="colsep0 rowsep0" align="center">BaF3 pFGFR4</th><th class="colsep0 rowsep0" align="center">HUH7 proliferation</th><th class="colsep0 rowsep0" align="center">HT sol., pH 6.8 (μM)</th><th class="colsep0 rowsep0" align="center">hERG IC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center">rat microsome hepatic ER</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>71</b></td><td class="colsep0 rowsep0" align="left">18</td><td class="colsep0 rowsep0" align="left">42 ± 3</td><td class="colsep0 rowsep0" align="left">638 ± 51</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">>30</td><td class="colsep0 rowsep0" align="left">0.64</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>72</b></td><td class="colsep0 rowsep0" align="left">18 ± 2</td><td class="colsep0 rowsep0" align="left">17 ± 4</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">25</td><td class="colsep0 rowsep0" align="left">0.66</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>73</b></td><td class="colsep0 rowsep0" align="left">3.4</td><td class="colsep0 rowsep0" align="left">64 ± 17</td><td class="colsep0 rowsep0" align="left">403 ± 263</td><td class="colsep0 rowsep0" align="left">472</td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">0.76</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>74</b></td><td class="colsep0 rowsep0" align="left">3.1</td><td class="colsep0 rowsep0" align="left">40 ± 15</td><td class="colsep0 rowsep0" align="left">573 ± 76</td><td class="colsep0 rowsep0" align="left">27</td><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left">0.91</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>75</b></td><td class="colsep0 rowsep0" align="left">8.1 ± 4.9</td><td class="colsep0 rowsep0" align="left">23</td><td class="colsep0 rowsep0" align="left">193</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">28</td><td class="colsep0 rowsep0" align="left">0.34</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>76</b></td><td class="colsep0 rowsep0" align="left">1.2</td><td class="colsep0 rowsep0" align="left">17 ± 8</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><4</td><td class="colsep0 rowsep0" align="left">0.5</td><td class="colsep0 rowsep0" align="left">0.64</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>77</b></td><td class="colsep0 rowsep0" align="left">0.9 ± 0.2</td><td class="colsep0 rowsep0" align="left">4.8 ± 0.8</td><td class="colsep0 rowsep0" align="left">33 ± 16</td><td class="colsep0 rowsep0" align="left">547</td><td class="colsep0 rowsep0" align="left">6.8</td><td class="colsep0 rowsep0" align="left">0.11</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>78</b></td><td class="colsep0 rowsep0" align="left">2.0 ± 0.1</td><td class="colsep0 rowsep0" align="left">17 ± 8</td><td class="colsep0 rowsep0" align="left">148 ± 44</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">0.42</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>79</b></td><td class="colsep0 rowsep0" align="left">2.3 ± 0.9</td><td class="colsep0 rowsep0" align="left">19 ± 3</td><td class="colsep0 rowsep0" align="left">504 ± 32</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="left">0.86</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>80</b></td><td class="colsep0 rowsep0" align="left">0.9</td><td class="colsep0 rowsep0" align="left">5.3 ± 1.3</td><td class="colsep0 rowsep0" align="left">39 ± 7</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">18</td><td class="colsep0 rowsep0" align="left">0.20</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>81</b></td><td class="colsep0 rowsep0" align="left">1.6</td><td class="colsep0 rowsep0" align="left">3.9 ± 0.7</td><td class="colsep0 rowsep0" align="left">46 ± 2</td><td class="colsep0 rowsep0" align="left">345</td><td class="colsep0 rowsep0" align="left">8.5</td><td class="colsep0 rowsep0" align="left">0.10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>82</b></td><td class="colsep0 rowsep0" align="left">0.2</td><td class="colsep0 rowsep0" align="left">6.1 ± 1.0</td><td class="colsep0 rowsep0" align="left">79 ± 12</td><td class="colsep0 rowsep0" align="left">79</td><td class="colsep0 rowsep0" align="left">>30</td><td class="colsep0 rowsep0" align="left">0.58</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>83</b></td><td class="colsep0 rowsep0" align="left">0.6</td><td class="colsep0 rowsep0" align="left">5.8 ± 0.2</td><td class="colsep0 rowsep0" align="left">114 ± 60</td><td class="colsep0 rowsep0" align="left">73</td><td class="colsep0 rowsep0" align="left">>30</td><td class="colsep0 rowsep0" align="left">0.35</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>84</b>, roblitinib</td><td class="colsep0 rowsep0" align="left">0.9 ± 0.4</td><td class="colsep0 rowsep0" align="left">4.3 ± 2.5</td><td class="colsep0 rowsep0" align="left">12 ± 6</td><td class="colsep0 rowsep0" align="left">31</td><td class="colsep0 rowsep0" align="left">>30</td><td class="colsep0 rowsep0" align="left">0.37</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>85</b></td><td class="colsep0 rowsep0" align="left">1.3 ± 0.7</td><td class="colsep0 rowsep0" align="left">7.1 ± 3.3</td><td class="colsep0 rowsep0" align="left">547 ± 171</td><td class="colsep0 rowsep0" align="left">39</td><td class="colsep0 rowsep0" align="left">22</td><td class="colsep0 rowsep0" align="left">0.47</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>86</b></td><td class="colsep0 rowsep0" align="left">2.6 ± 0.7</td><td class="colsep0 rowsep0" align="left">27 ± 1</td><td class="colsep0 rowsep0" align="left">1775 ± 735</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">13</td><td class="colsep0 rowsep0" align="left">0.43</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>87</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">12 ± 1</td><td class="colsep0 rowsep0" align="left">103</td><td class="colsep0 rowsep0" align="left"><4</td><td class="colsep0 rowsep0" align="left">>30</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>88</b></td><td class="colsep0 rowsep0" align="left">1.4</td><td class="colsep0 rowsep0" align="left">11 ± 1</td><td class="colsep0 rowsep0" align="left">185 ± 29</td><td class="colsep0 rowsep0" align="left"><4</td><td class="colsep0 rowsep0" align="left">>30</td><td class="colsep0 rowsep0" align="left">0.70</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>89</b></td><td class="colsep0 rowsep0" align="left">2.4 ± 0.7</td><td class="colsep0 rowsep0" align="left">5.7 ± 2.5</td><td class="colsep0 rowsep0" align="left">68 ± 33</td><td class="colsep0 rowsep0" align="left">276</td><td class="colsep0 rowsep0" align="left">>30</td><td class="colsep0 rowsep0" align="left">0.56</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>90</b></td><td class="colsep0 rowsep0" align="left">1.5 ± 0.4</td><td class="colsep0 rowsep0" align="left">5.0 ± 2.3</td><td class="colsep0 rowsep0" align="left">80 ± 36</td><td class="colsep0 rowsep0" align="left">394</td><td class="colsep0 rowsep0" align="left">>30</td><td class="colsep0 rowsep0" align="left">0.42</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>91</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">3.7 ± 0.2</td><td class="colsep0 rowsep0" align="left">66</td><td class="colsep0 rowsep0" align="left">416</td><td class="colsep0 rowsep0" align="left">>30</td><td class="colsep0 rowsep0" align="left">0.95</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl9-fn1"><div class="footnote" id="tbl9-fn1"><sup><sup>a</sup></sup><p class="last">FGFR4, solubility, and hERG data are reported as the mean ± SD (<i>n</i> = 2–15 repeats) or as single measurements. Rat microsome hepatic ER data are reported as the mean (1–3 repeats).</p></div></div></div><div class="NLM_p">Incorporating a basic group into the THN 3-substituent was considered to offer the possibility for a more atom-efficient approach. THN 3-substituents with a strongly basic tertiary amine proximal to the attachment point had resulted in a marked loss in FGFR4 activity, for example, compound <b>39</b> (calculated p<i>K</i><sub>a</sub> of 8.1). In contrast, moving the basic group further out from the main pharmacophore and reducing the basicity was found to be better tolerated, for example, the piperidine derivatives <b>75</b> and <b>76</b> (calculated p<i>K</i><sub>a</sub> of 7.8 and 6.0). This change also had a favorable effect on the selectivity versus the human ether-à-go-go-related gene channel (hERG) in the case of <b>75</b>, measured using a dofetilide competition binding assay (<b>39</b>, hERG IC<sub>50</sub> = 1.5 μM). These observations prompted the design of a number of 3-N-acylated aminomethyl derivatives incorporating tertiary amines four or more atoms removed from the point of attachment to the THN core and for which p<i>K</i><sub>a</sub> < 8 was predicted, <a data-tab="pane-pcw-Figures" href="#fig20" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig20" id="" class=" internalNav">20</a></a>. The profiles of the acyclic examples incorporating dimethylamino groups into the acyl, <b>77</b>, and <i>N</i>-alkyl moieties, <b>78</b> and <b>79</b>, were favorable and suggested that the optimization was close to meeting the criteria that had been set for candidate selection. As a result, a focused effort was made to probe the THN 3-substituent within this structural space, and lactam derivatives were of particular interest following the favorable efficacy and toxicity profile determined for <b>68</b>. The dimethylamino-substituted butyrolactam derivatives <b>80</b>–<b>83</b> provided favorable profiles, but based upon initial assessments of in vitro potency and selectivity and in vivo PK/PD studies, the N-methylated piperazine lactam <b>84</b> was found to provide the best all-round profile.</div><div class="NLM_p last">From further exploration of the SAR around <b>84</b>, the importance of the piperazine carbonyl is highlighted by <b>85</b> and <b>86</b>. Inverting the position of the carbonyl group in <b>84</b> leads to the basic group being positioned closer to the THN core resulting a 46-fold decrease in the HUH7 antiproliferation potency. The absence of the carbonyl group in <b>86</b> led to an even greater decrease in FGFR4 activity and an increase in binding affinity for the hERG channel. Substitution of the piperazine at the 3-, 5-, or 6-position was found to be reasonably well tolerated, as exemplified by <b>87</b> and <b>88</b>, but led to lower solubility and decreased metabolic stability. Analysis of several 2-FTHNU crystal structures had suggested that the introduction of a substituent into the 2-position of the methoxyethylamino pyridine moiety would be particularly disruptive to the crystal packing.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> The introduction of a methyl group into <b>84</b> at this position gave <b>89</b> and <b>90</b> of the (<i>S</i>)- and (<i>R</i>)-configurations. In line with the hypothesis, the enthalpies of fusion were halved leading to an 80 °C decrease in the melting point for <b>89</b> and <b>90</b> when compared to <b>84</b> (Δ<i>H</i><sub>fus</sub>/mp: <b>89</b> 21 kJ mol<sup>–1</sup>/134 °C, <b>90</b> 29 kJ mol<sup>–1</sup>/136 °C, <b>84</b> 53 kJ mol<sup>–1</sup>/224 °C; all crystalline by XRPD). However, higher clearance rates were determined for <b>89</b> and <b>90</b> (rat in vivo CL: <b>90</b> 35 mL min<sup>–1</sup> kg<sup>–1</sup>, <b>84</b> 19 mL min<sup>–1</sup> kg<sup>–1</sup>) which were assigned to the increased lipophilicity. The increase in clearance offset any solubility/dissolution gains, and there was no overall benefit in PK profile in the rat. The analogue in which the partially saturated ring of the THN is opened, <b>91</b>, exhibited a similar in vitro profile to <b>84</b>, but in line with earlier analogues with this modification, higher in vitro and in vivo clearance rates were observed (rat CL: <b>91</b> 88 mL min<sup>–1</sup> kg<sup>–1</sup>). From the head-to-head profiling of these lactam derivatives, compound <b>84</b> best satisfied the targeted profile and was progressed into more extensive characterization, ultimately going on to become the final selected candidate roblitinib.</div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Characterization of Roblitinib</h3><div class="NLM_p">Internal kinase profiling indicated an excellent selectivity profile for <b>84</b> (65 biochemical and 46 BaF3 cellular assays). More than 1000-fold selectivity was determined for FGFR4 compared to the next most potent kinase inhibited (Aurora A: biochemical IC<sub>50</sub> = 5.6 μM), including no appreciable activity in both biochemical and cellular assays versus the other FGFR family members (top concentration tested 10 μM). A KINOME<i>scan</i> with <b>84</b> showed <35% displacement of the reporter binding against the whole panel of 456 off-target kinases at a concentration of 3 μM, <a data-tab="pane-pcw-Figures" href="#fig21" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig21" id="" class=" internalNav">21</a></a>.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> Only reporter binding to FGFR4 was extensively inhibited (>99%; selectivity score 0.003). Of the other GK+2 cysteine containing kinases, only MAPKAPK2 (MK2) showed any appreciable inhibition (biochemical IC<sub>50</sub> = 9.4 μM). In enzyme and receptor screens, <b>84</b> exhibited no appreciable activity at a concentration of 10 μM against 138 representative off-targets. These screens indicated that <b>84</b> is a potent inhibitor of the kinase activity of FGFR4 with an exceptionally high level of selectivity within the human kinome and against a broader set of targets.</div><figure id="fig21" data-index="21" class="article__inlineFigure"><h2 class="fig-label">Figure 21</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0021.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0021.jpeg" id="gr21" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 21. In vitro profile of roblitinib <b>84</b>: (A) KINOME<i>scan</i> TREE<i>spot</i> interaction map; (B) selected biochemical kinase assay data; (C) selected cellular proliferation data in HCC cell lines both dependent on and independent of FGF19/FGFR4 signaling (methylene blue staining).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0021.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig21"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Consistent with the reversible-covalent interaction, <b>84</b> showed no activity in biochemical assays using the C552A variant and rat FGFR4 kinase domains. Mutated forms of FGFR4 have been described in RMS with increased signaling and oncogenic activity, with mutations most frequently occurring at the molecular-brake (N535) and gate-keeper (V550) positions.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> Recently, the first instances of acquired resistance have been described in HCC patients treated with the selective FGFR4 inhibitor fisogatinib, with mutation of the gate-keeper residue having been observed.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> The molecular brake is an asparagine residue that is highly conserved across the kinome and is involved in a hydrogen-bonding network that biases the kinase domain toward populating less active conformations.<a onclick="showRef(event, 'ref30 ref60'); return false;" href="javascript:void(0);" class="ref ref30 ref60">(30,60)</a> In biochemical assays, <b>84</b> was found to inhibit the N535 K mutant with similar activity to wild-type FGFR4. This highlights the ability of <b>84</b> to inhibit activated forms of FGFR4 equally well and is consistent with observing no shift in potency when an ATP preincubation-period is introduced prior to running the biochemical assay with nonphosphorylated FGFR4. In contrast, the V550E gate-keeper mutation incorporates a much larger glutamic acid residue at this position and consequently was anticipated to have a marked impact on the affinity of the majority of inhibitors interacting within the ATP-binding site.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> Consistent with this expectation, <b>84</b> was found to be 58-fold less active in a biochemical assay with the V550E mutant. These biochemical findings were also consistent with in vivo findings using NIH3T3 xenografts engineered to constitutively express the two mutated forms of FGFR4.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a></div><div class="NLM_p">By comparison of the difference between the BaF3 pFGFR4 and HUH7 proliferation IC<sub>50</sub> values, <b>84</b> exhibited a 3-fold shift in line with what would be anticipated for a “stable” kinase inhibitor. Additionally, in vitro sampling indicated that pathway inhibition and compound concentration were not extensively depleted over the course of the HCC proliferation assays. Across a panel of HCC lines with growth dependent on FGFR4/FGF19/KLB, <b>84</b> was found to inhibit them all with an IC<sub>50</sub> value in the range of 9–12 nM (HUH7, Hep-3B, JHH7). In contrast, for HCC cell lines with growth independent of FGFR4 signaling (HEPG2, JHH5), no antiproliferative activity was determined at concentrations up to 10 μM, further highlighting the excellent kinase selectivity of <b>84</b>.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a></div><div class="NLM_p">To better understand the mode of interaction of <b>84</b>, the crystal structure in complex with the kinase domain of FGFR4 was solved to a resolution of 2.1 Å, <a data-tab="pane-pcw-Figures" href="#fig22" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig22" id="" class=" internalNav">22</a></a>. The structure confirmed that <b>84</b> binds within the ATP pocket of the kinase domain to make a hemithioacetal carbon to sulfur bond with the side chain of the hinge residue Cys552. A number of direct or water mediated hydrogen bonds between the inhibitor and residues of the ATP pocket are observed in the complex. In particular, the pyridyl nitrogen, the piperazinone oxygen, and the hemithioacetal oxygen of <b>84</b> form direct hydrogen bonds with residues Ala553, Arg483, and Val500, respectively. Water mediated hydrogen bonds are formed between the inhibitor urea oxygen and residue Asn557 and also between the 5-cyano group and residue Lys503. Hydrophobic interactions are also evident: the pyridyl moiety of <b>84</b> is sandwiched between the side chains of residues Leu619 and Ala501, while residue Leu473 makes extensive hydrophobic contacts with the inhibitor THN moiety. In addition, the terminal methyl group of the methoxyethyl chain of <b>84</b> fits snugly into the small hydrophobic slot formed by the side chains of residues Val481 and Gly474. The stereochemistry of the hemithioacetal center can be assigned as being of the (<i>R</i>)-configuration in this structure. This outcome is also consistent with addition occurring to the low energy conformation of the aldehyde (determined by ab initio calculation) in which the carbonyl group is orientated away from the THN ring nitrogen to minimize electrostatic repulsion. When the hinge-binding interaction is in place or being formed, only the <i>re</i> face of the aldehyde in the low energy conformation is accessible to react with the thiol of Cys552, approximately following the Bürgi–Dunitz angle.<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> This low energy aldehyde conformation was found to be present in the single-crystal X-ray structure determined for <b>84</b>. However, the selective crystallization of a single diastereosiomer from a mixture of diastereoisomers in the crystallization media cannot be ruled out. The cocrystal structure of <b>84</b> with the kinase domain of FGFR4 has subsequently been confirmed by others.<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a></div><figure id="fig22" data-index="22" class="article__inlineFigure"><h2 class="fig-label">Figure 22</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0022.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0022.jpeg" id="gr22" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 22. X-ray structures of <b>84</b>: (A) cocrystal structure in complex with the FGFR4 kinase domain ATP binding site (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YI8">6YI8</a>). After reaction, the aldehyde group of <b>84</b> forms a hemithioacetal with the hinge residue Cys552 specific to FGFR4. <b>84</b> forms direct or water mediated hydrogen bonds with residues Ala553, Val500, Arg483, Lys503, and Asn557 of the ATP binding site (indicated as dashed lines). The intramolecular hydrogen bond between the inhibitor free urea nitrogen and the pyridyl nitrogen of the THN moiety, previously observed in this series of compounds, is also present. (B) ORTEP representation of a single-crystal X-ray structure of <b>84</b> (CCDC code 2009566).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0022.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig22"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To further characterize the reversible-covalent interaction, binding kinetics were determined using an FGFR4 competition binding assay.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> The measured time-course profiles were consistent with a reversible interaction and the data for <b>84</b>, and selected analogues are shown in <a class="ref showTableEvent internalNav" data-ID="tbl10" href="#tbl10">Table <a class="ref showTableEvent internalNav" data-ID="tbl10" href="#tbl10">10</a></a>. Binding affinities determined with the kinetic binding assay correlated well with the biochemical IC<sub>50</sub> values. The kinetic parameters determined for <b>84</b> were similar to the other 2-FTHN analogues characterized as part of the optimization. In each case, the increase in FGFR4 affinity was driven by a faster on-rate, when taking compound <b>23</b> as the reference point for the start of the optimization. Off-rates were in a similar range for all the analogues, with <b>84</b> and the other analogues showing slightly decreased values in comparison to compound <b>23</b>. Even though the kinetic assay was not used routinely during the optimization of the series, the kinetic profile of <b>84</b> appeared to be ideal for the needs of an FGFR4 inhibitor to treat HCC, highlighting that the optimization strategy taken, to use a biochemical followed by a cellular proliferation assay, may have naturally selected for the optimal kinetic profile. With an FGFR4 protein resynthesis rate of <2 h determined in HCC cells, a residence time in excess of this was considered to offer no advantage. To satisfy the observed PK/PD/efficacy relationship, a residence time that matched the resynthesis rate appeared ideal to ensure continuous and robust inhibition once the target had been engaged. In addition, a faster on-rate should ensure rapid target-engagement to quickly inhibit the newly synthesized protein. For a target with a rapid resynthesis rate, such as determined for FGFR4 in HCC, achieving a faster on-rate may be particularly important to minimize the need for higher inhibitor concentrations to be present to achieve a high target coverage.</div><div class="NLM_table-wrap" id="tbl10"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 10. Kinetic Binding Data for Selected 2-FTHNU Derivatives<a class="ref internalNav" href="#t10fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub> (nM)</th><th class="colsep0 rowsep0" align="center"><i>k</i><sub>on</sub> (s<sup>–1</sup> M<sup>–1</sup>)</th><th class="colsep0 rowsep0" align="center"><i>k</i><sub>off</sub> (s<sup>–1</sup>)</th><th class="colsep0 rowsep0" align="center" char=".">residence time (min)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>24</b></td><td class="colsep0 rowsep0" align="left">21</td><td class="colsep0 rowsep0" align="left">3.9 × 10<sup>3</sup></td><td class="colsep0 rowsep0" align="left">8.2 × 10<sup>–5</sup></td><td class="colsep0 rowsep0" align="char" char=".">204</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>43</b></td><td class="colsep0 rowsep0" align="left">4.0</td><td class="colsep0 rowsep0" align="left">4.4 × 10<sup>4</sup></td><td class="colsep0 rowsep0" align="left">1.8 × 10<sup>–4</sup></td><td class="colsep0 rowsep0" align="char" char=".">95</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>57</b></td><td class="colsep0 rowsep0" align="left">6.0</td><td class="colsep0 rowsep0" align="left">2.7 × 10<sup>4</sup></td><td class="colsep0 rowsep0" align="left">1.6 × 10<sup>–4</sup></td><td class="colsep0 rowsep0" align="char" char=".">102</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>62</b></td><td class="colsep0 rowsep0" align="left">3.6</td><td class="colsep0 rowsep0" align="left">4.3 × 10<sup>4</sup></td><td class="colsep0 rowsep0" align="left">1.6 × 10<sup>–4</sup></td><td class="colsep0 rowsep0" align="char" char=".">107</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>68</b></td><td class="colsep0 rowsep0" align="left">3.8</td><td class="colsep0 rowsep0" align="left">6.4 × 10<sup>4</sup></td><td class="colsep0 rowsep0" align="left">2.4 × 10<sup>–4</sup></td><td class="colsep0 rowsep0" align="char" char=".">68</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>84</b></td><td class="colsep0 rowsep0" align="left">2.9</td><td class="colsep0 rowsep0" align="left">1.1 × 10<sup>5</sup></td><td class="colsep0 rowsep0" align="left">3.2 × 10<sup>–4</sup></td><td class="colsep0 rowsep0" align="char" char=".">52</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>85</b></td><td class="colsep0 rowsep0" align="left">3.5</td><td class="colsep0 rowsep0" align="left">2.4 × 10<sup>5</sup></td><td class="colsep0 rowsep0" align="left">8.6 × 10<sup>–4</sup></td><td class="colsep0 rowsep0" align="char" char=".">19</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t10fn1"><div class="footnote" id="t10fn1"><sup><sup>a</sup></sup><p class="last">Data are single measurements.</p></div></div></div><div class="NLM_p">The <i>N</i>-methylpiperazine lactam of <b>84</b> introduces a weakly basic amine with a measured p<i>K</i><sub>a</sub> value of 5.3 (potentiometric titration), which contributes to the increased solubility at pH 6.8 and faster dissolution rates at lower pH values when compared to the nonbasic lactam analogues such as <b>68</b>. Additionally, a lower p<i>K</i><sub>a</sub> in this range was found to be important for achieving a clean off-target profile. Measurement of the thermodynamic solubility in pH 6.8 buffer (16 μM) provided a very similar value to the high throughput method, consistent with what was observed for other analogues from the series. Permeability was assessed in Caco-2 cells, and the uptake of <b>84</b> from the gut was predicted to be high following oral delivery, with no evidence for efflux (<i>P</i><sub>app</sub> A–B/B–A of 7.4/11 × 10<sup>–6</sup> cm s<sup>–1</sup>). The measured permeability is relatively high considering the topological PSA of 144 Å<sup>2</sup>. However, the presence of the internal hydrogen bond reduces the effective PSA into the range where favorable permeation rates would be anticipated (calculated 3D PSA of 122 Å<sup>2</sup>).<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a></div><div class="NLM_p">In vitro MetID studies revealed that some reduction of the aldehyde to the benzylic alcohol <b>92</b> was occurring but not as the major pathway, <a data-tab="pane-pcw-Figures" href="#fig23" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig23" id="" class=" internalNav">23</a></a>. Major metabolites observed in these incubations were derived from oxidative clearance mechanisms leading to <b>93</b> and <b>94</b> as a result of O- and N-demethylations and oxidation of the THN core. No evidence for the significant inhibition or induction of metabolizing enzymes was observed, suggesting <b>84</b> would be suitable for use in combination with a wide range of other treatments.</div><figure id="fig23" data-index="23" class="article__inlineFigure"><h2 class="fig-label">Figure 23</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0023.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0023.jpeg" id="gr23" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 23. Preclinical pharmacokinetics determined for <b>84</b>: (A) table showing key in vitro and in vivo parameters, mouse (female Balb/c, <i>n</i> = 3) dosed iv at 1 mg/kg in 9:1 NMP/plasma and po at 3 mg/kg as a suspension of the crystalline free base in 99:0.5:0.5 water/methyl cellulose/Tween-80, rat (Sprague Dawley, <i>n</i> = 4) dosed iv at 0.5 mg/kg in 7:3 PEG 200/NMP and po at 3 mg/kg as a suspension of the crystalline free base in 99.5:0.5 methylcellulose-05/Tween-80, dog (male, beagle, <i>n</i> = 3) dosed iv at 0.1 mg/kg in 9:1 PEG 200/NMP and po at 0.3 mg/kg as a suspension in water; (B) structures of the reduced <b>92</b> and predominant demethylated metabolites <b>93</b> and <b>94</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0023.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig23"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The low dose PK profiles for <b>84</b> in mouse, rat, and dog are summarized in the table of <a data-tab="pane-pcw-Figures" href="#fig23" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig23" id="" class=" internalNav">23</a></a>. In all three species, in vivo clearance values were consistent with the low clearance predictions derived from the in vitro measurements with microsomes and hepatocytes. Oral bioavailability was modest in the two rodent species but higher in the dog. Importantly, extrapolating the in vitro data to man suggested a favorable PK profile for <b>84</b> (prediction: plasma CL = 4–8 mL min<sup>–1</sup> kg<sup>–1</sup>, <i>V</i><sub>ss</sub> = 3.5–5 L/kg, moderate to high oral <i>F</i>).<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a></div><div class="NLM_p">PK/PD studies using the RH30 RMS mouse xenograft model confirmed <b>84</b> to potently inhibit FGFR4 autophosphorylation in vivo with plasma IC<sub>50</sub> and IC<sub>90</sub> values of 2.1 and 52 nM.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> By employment of these values to guide the design of mouse efficacy studies, <b>84</b> consistently delivered the maximum efficacy observable through FGFR4 inhibition in HUH7, Hep3B, and JHH7 xenografts when dosed using a 30 mg/kg b.i.d. regimen. Representative data from the Hep3B model are shown in <a data-tab="pane-pcw-Figures" href="#fig24" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig24" id="" class=" internalNav">24</a></a> for doses of 10, 30, and 100 mg/kg, administered b.i.d.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> In contrast to the PK/PD experiments, exposures for the Hep3B efficacy study were measured in blood. A blood to plasma ratio of 1.39 ± 0.10 (±SD) was determined for compound <b>84</b> in the mouse, leading to adjusted PK/PD values in blood of IC<sub>50</sub> = 2.9 and IC<sub>90</sub> = 72 nM. At the lowest dose of 10 mg/kg, day 6 blood concentrations dropped below the IC<sub>90</sub> threshold level within 8 h of dosing, and tumor growth was controlled to the level of stasis (day 14 T/C 17%). With the 30 mg/kg b.i.d. regimen, blood concentrations dropped below the IC<sub>90</sub> threshold between 8 and 12 h following dosing, and the maximal level of inhibition of FGFR4-dependent tumor growth was observed in this model (day 14 T/C −63%). The 30 mg/kg b.i.d. dose was associated with a <i>C</i><sub>trough</sub> of 30 nM, with no evidence for any significant accumulation over the 2 week study. At the highest dose level tested of 100 mg/kg, b.i.d., blood concentrations remained above the IC<sub>90</sub> level throughout the dosing interval and comparable efficacy to the 30 mg/kg group was observed (day 14 T/C −85%). In these studies, roblitinib was well tolerated and could be shown to produce the anticipated FGFR-driven pharmacology in the host of increased CYP7A1 transcript levels in the liver and elevated levels of the circulating BA intermediate 7α-hydroxy-4-cholesten-3-one.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> These data were consistent with the notion that a sustained high level of FGFR4 inhibition would be required to deliver maximum efficacy.</div><figure id="fig24" data-index="24" class="article__inlineFigure"><h2 class="fig-label">Figure 24</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0024.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0024.jpeg" id="gr24" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 24. Mouse xenograft studies with <b>84</b>: (A) total plasma concentration–response curve in the RH30 xenograft model, IC<sub>50</sub> = 2.1 nM, IC<sub>90</sub> = 52 nM; (B) change in tumor volume over time in the Hep3B xenograft model, where data are the mean ± SD, *<i>P</i> < 0.05 versus vehicle by Kruskal–Wallis (Dunn’s); (C) day 6 blood exposure profiles.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0024.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig24"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Further support for this PK/PD/efficacy relationship could be derived from a dose fractionation study carried out in the rat with HUH7 xenografts.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> In this study, a daily dose of 6 mg/kg was administered either as a 3 mg b.i.d., 6 mg q.d., or 12 mg every other day (q2d) regimen for 14 days, <a data-tab="pane-pcw-Figures" href="#fig25" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig25" id="" class=" internalNav">25</a></a>. Lower dose levels were required in the rat to probe the same <i>C</i><sub>trough</sub> exposure range due to a more sustained oral PK profile in this species relative to the mouse. From the assessment of the PK/PD/efficacy relationship, for the b.i.d. regimen the blood concentration remained above the IC<sub>90</sub> for 60% of the 12 h dosing interval, leading to a high degree of regression of the tumor growth (day 14 T/C −84%). The q.d. regimen blood level remained above the IC<sub>90</sub> level for 42% of the 24 h doing interval and approximately led to tumor stasis (day 14 T/C 1%). The q2d regimen blood level remained above the IC<sub>90</sub> level for only 26% of the 48 h dosing interval and led to only a slowing of the tumor growth rate (day 14 T/C 23%). These data strongly supported a PK/PD/efficacy relationship in which covering a <i>C</i><sub>trough</sub> value throughout the dosing interval was critical for achieving maximum efficacy and that taking this level as the RH30 xenograft-model derived IC<sub>90</sub> value would be a conservative estimate to target going forward.</div><figure id="fig25" data-index="25" class="article__inlineFigure"><h2 class="fig-label">Figure 25</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0025.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0025.jpeg" id="gr25" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 25. Dose fraction study with <b>84</b> in the HUH7 xenograft model in the rat (<i>n</i> = 8, data are the mean ± SD): (A) change in tumor volume over time, *<i>P</i> < 0.05 versus vehicle by Kruskal–Wallis (Dunn’s); (B) day 8 blood exposure profiles; (C) efficacy and exposure parameters.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0025.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig25"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In vitro and in vivo toxicology studies in mouse, rat, and dog revealed <b>84</b> to be well tolerated at exposures and doses above those associated with efficacy. The anticipated FGFR4 mediated effects on BA homeostasis were observed in the mouse and dog, as well as mild hepatic changes (increases in alkaline phosphatase and serum aminotransferase). Additional studies in the dog revealed that the mild hepatic changes were a consequence of the increased levels of BAs and that they could be moderated by the coadministration of the BA sequestrant cholestyramine.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a></div><div class="NLM_p">A salt screen revealed the 1:1 citrate salt form of <b>84</b> to have favorable properties and to be predicted to maintain proportional increases in exposure up to higher dose levels compared to the free base form. Upon the basis of this profile, the citrate salt was selected for the first in man studies, with predictions indicating that the necessary exposures should be readily achievable to reach the levels targeted for efficacy.</div><div class="NLM_p last">Overall the data package for <b>84</b> was favorable and the compound was selected to go forward for clinical evaluation as a potential treatment for FGFR/FGF19/KLB driven solid tumors. The compound became the first selective FGFR4 inhibitor to enter into clinical studies at the end of 2014.<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> At this stage, <b>84</b> was assigned the INN roblitinib, having previously been named FGF401. Initial data from these trials have been encouraging and suggest that <b>84</b> has a favorable human PK profile, has a manageable safety profile that is consistent with inhibiting the FGF19 regulation of BAs, and provided clinical benefit in patients with advanced and heavily pretreated HCC.<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> Clinical trials remain ongoing with roblitinib in HCC.<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a></div></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Synthesis of the 2-FTHNU Analogues</h3><div class="NLM_p">The preparation of the 2-FTHNU derivatives involved the development of routes to key 2-aminopyridine and THN intermediates as well as the identification of methodologies for constructing the central urea linkage. Representative sequences are highlighted in <a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a>–<a class="ref internalNav" href="#sch4" aria-label="4">4</a>, including for the synthesis of <b>68</b> and <b>84</b> which exemplify the two key approaches used to prepare the central urea linkage.</div><figure id="sch1" data-index="26" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0026.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0026.jpeg" id="gr26" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthetic Routes to the Hinge-Binding 2-Aminopyridine Intermediates <b>96</b> and <b>97</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0026.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NIS, DMA, rt, 24 h (76–81%); (b) Zn(CN)<sub>2</sub>, Zn, Pd<sub>2</sub>dba<sub>3</sub>, dppf, DMA, 150 °C, 2 h (84–97%); (c) 2-methoxyethylamine, DIPEA, DMA, 50 °C, 18 h (80–85%); (d) <i>i</i>PrOH, KHMDS, THF, rt, 18 h (55–62%).</p></p></figure><div class="NLM_p">The hinge-binding aminopyridine fragments were readily prepared as shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. Starting from 2-amino-4-fluoropyridine, iodination, followed by Pd-catalyzed cyanation, gave the key intermediate <b>95</b>. Nucleophilic aromatic substitution reactions with the 4-fluoropyridine derivative <b>95</b> enabled a range of O- and N-substituted analogues to be prepared, as exemplified by the synthesis of <b>96</b> and <b>97</b>.</div><div class="NLM_p">Generation of the 2-FTHNU aldehyde functionality was found to be best carried out as the final step of the synthesis, and acetal hydrolysis proved to be the most efficient approach. Final products were then typically isolated with high purity and required only minimal purification. Late stage oxidation of either the 2-methyl or 2-hydroxymethyl THNU analogues was also explored, but both approaches were found to be less efficient. In particular, the isolated products often required more extensive purification which was complicated by the presence of the aldehyde group.</div><div class="NLM_p">To facilitate this approach, the 3-bromotetrahydronaphthyridine acetal <b>99</b> proved to be a key intermediate for the preparation of a range of 2-FTHN building blocks and was readily prepared from the previously described 2-dimethoxymethyl THN <b>98</b> (THN numbering is based upon the original 2-FQA hit), <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>.<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> The 3-bromo functionality enabled the introduction of substituents <i>ortho</i> to the protected aldehyde. Chemistry carried out with the 3-bromo intermediate included transition metal mediated cross-coupling reactions which enabled the synthesis of aromatic and saturated heterocyclic derivatives and halogen–lithium exchange which provided ready access to the putative dilithiated THN intermediate <b>100</b>, which could be reacted with a range of electrophiles. In particular, formylation of <b>100</b> with DMF, to give the aldehyde <b>101</b>, proved to be a pivotal approach for the synthesis of several key analogues: fluorination of <b>101</b> with DAST provided access to the difluoromethyl analogues, such as <b>45</b> and <b>46</b>, via <b>102</b>; reduction of <b>101</b> gave access to the benzylic alcohol derivatives, such as <b>43</b> and <b>57</b>, via the protected intermediate <b>103</b>; reductive amination with <b>101</b> provided access to secondary and tertiary amine derivatives, such as <b>104</b> for the preparation of <b>39</b>.</div><figure id="sch2" data-index="27" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0027.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0027.jpeg" id="gr27" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of the Key THN Intermediates <b>102</b>, <b>103</b>, and <b>104</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0027.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NBS, MeCN, rt, 0.5 h (73%); (b) 1 equiv of MeLi, then 1 equiv of <i>n</i>BuLi, then DMF, −78 °C to rt (>95%); (c) DAST, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to rt, 18 h (63–69%); (d) NaBH<sub>4</sub>, MeOH, DCM, rt, 0.5 h (>95%); (e) TBDMSCl, Et<sub>3</sub>N, DMAP, DCM, DMF, rt, 2 h (80–95%); (f) 2 M Me<sub>2</sub>NH in THF, Na(OAc)<sub>3</sub>BH, DCE, rt, 48 h (>95%).</p></p></figure><div class="NLM_p">Due to the low nucleophilicity of the 2-aminopyridine and THN fragments and the poor stability of 2-aminopyridine derived isocyanates,<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> the identification of efficient ways to construct the central urea linkage proved to be crucial for exploring the 2-FTHNU series. The preparation of the butyrolactam containing derivative <b>68</b> highlights one of the key urea-bond forming reactions employed in the optimization, the aza-anion approach, <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Deprotonation of the silyl protected hydroxymethyl THN intermediate <b>103</b> with LiHMDS facilitated acylation with diphenyl carbonate to give the carbamate <b>105</b>. The second phenoxide originating from the diphenyl carbonate in <b>105</b> is then displaced with the lithium anion of the 2-aminopyridine <b>97</b>, similarly generated at low temperature by treatment with LiHMDS, to complete the urea formation. To introduce the butyrolactam, the silyl ether was cleaved with fluoride, the resulting benzylic alcohol <b>106</b> activated as the mesylate which was then used to alkylate methyl 4-aminobutyrate to give <b>107</b>. The γ-amino ester <b>107</b> was then cyclized under basic conditions to the lactam. Subsequent hydrolysis of the acetal efficiently generated <b>68</b>, which could be isolated as a crystalline solid of high purity following sonication in ethyl acetate.</div><figure id="sch3" data-index="28" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0028.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0028.jpeg" id="gr28" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthetic Route to Compound <b>68</b>, Using the Aza-Anion Approach for Urea Formation<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0028.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) LiHMDS, THF, −78 °C, then (PhO)<sub>2</sub>CO, warm to rt (59–75%); (b) <b>97</b>, LiHMDS, THF, −78 °C, 2 h (79–82%); (c) HF-pyridine, THF, rt, 4 h (88-95%); (d) (MeSO<sub>2</sub>)<sub>2</sub>O, DIPEA, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to rt, 2 h; (e) methyl 4-aminobutyrate, DIPEA, KI, 5:4 THF:CH<sub>2</sub>Cl<sub>2</sub>, rt, 22 h (44–52% 2 steps); (f) Et<sub>3</sub>N, 1,4-dioxane, 90 °C, 24 h (39–65%); (g) HCl<sub>(aq)</sub>, rt, 2 h (94–95%).</p></p></figure><div class="NLM_p">For certain combinations of coupling partners the above aza-anion approach was found to be unsuitable and resulted in low yields or no detectable formation of the desired urea products. In some of these instances, the outcomes could be improved by the use of 2 equiv of base to deprotonate any additional acidic positions within the reacting partners. However, alternative conditions still needed to be identified for some combinations, in particular with 2-aminopyridines bearing a secondary 4-amino substituent and THN moieties bearing N-acylated aminomethyl 3-substituents. Following the screening of a range of doubly activated carbonyl reagents, carbonyl ditriazole (CDT) was found to be a reliable way to effect the urea formation with these poorly reactive coupling partners.<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a> The preparation of <b>84</b> highlights the use of the CDT urea-forming approach, as shown in <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>. The amino ester <b>108</b> was readily prepared in two steps from Boc-protected <i>N</i>-methyl ethylene diamine and ethyl bromoacetate. A reductive amination with the amino ester <b>108</b> and the 3-formyl THN acetal <b>101</b> gave <i>N</i>-methylpiperazine lactam <b>109</b> directly, the initially formed secondary amine cyclizing under the reaction conditions. Urea formation with the THN <b>109</b> and 2-aminopyridine <b>96</b> proceeded smoothly in DMF with CDT at room temperature to give <b>110</b>. The difference in reactivity compared to the more commonly employed carbonyl dimidazole (CDI) can be clearly seen in this reaction. Under a range of conditions, at best only a modest conversion to <b>110</b> could be achieved with CDI. Finally, the acetal was cleaved in the final step of the synthesis to cleanly release the aldehyde functionality with only minimal purification required to obtain material of >95% purity. Roblitinib <b>84</b> displayed good stability during storage with no special precautions being required due to the presence of the aldehyde functionality.</div><figure id="sch4" data-index="29" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0029.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0029.jpeg" id="gr29" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthetic Route to <b>84</b> (Roblitinib), Using CDT for Urea Formation<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0029.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) TEA, THF, rt (98%); (b) conc HCl, THF, EtOH, rt (92%); (c) <b>101</b>, NaBH(OAc)<sub>3</sub>, TEA, DCE, rt (54–67%); (d) <b>96</b>, CDT, DMF, rt (82–86%); (e) conc HCl, THF<sub>(aq)</sub>, rt (82–88%).</p></p></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i28">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_58657" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_58657" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Hypotheses for identifying selective FGFR4 inhibitors were generated following a kinome-wide sequence alignment which targeted Cys552 within the middle-hinge region of the FGFR4 ATP-binding site. Both steric and covalent hypotheses targeting the middle-hinge region proved to be successful with FGFR4 selective hits being identified through the application of both rational design and unbiased screening approaches. Of particular interest, a 2-FQA hit series was identified for optimization based upon a high level of potency and selectivity for FGFR4, within both the FGFR family and also the human kinome, and a reversible-covalent mechanism of action that was considered ideally suited to match the resynthesis rate of FGFR4 in the HCC cell lines of interest. Morphing of the initial 2-FQA hit to a 2-FTHNU scaffold and optimization of the hinge binding pyridyl substitution pattern led to improved solubility and highly potent FGFR4 inhibition. Key to the further optimization of the series was the stabilization of the aldehyde group by the introduction of a 3-THNU substituent that slowed down the metabolism to the corresponding benzylic alcohol. 3-(<i>N</i>-Acyl-<i>N</i>-alkylaminomethyl) THNU substituents were found to be particularly effective for blocking the metabolism of the aldehyde group and maintaining a high level of FGFR4 activity in vitro and in vivo. Further refinement of this subseries identified the methyl-(<i>N</i>-methylpiperazine lactam) THNU 3-substituent as a way to increase solubility and to be optimal in combination with a 4-methoxyethylamino substituent in the hinge-binding pyridine ring leading to the identification of roblitinib <b>84</b>. Roblitinib satisfied the target product profile for a highly selective orally administered FGFR4 inhibitor and was promoted as the first candidate from the project. Roblitinib subsequently progressed to become the first selective FGFR4 inhibitor to enter into clinical studies in 2014. Encouraging data have been obtained from these first-in-human studies in patients with FGFR4/FGF19 driven tumors and will be the subject of future publications.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i29">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25786" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25786" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">The purity of all compounds by HPLC was ≥95% except where noted in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01019/suppl_file/jm0c01019_si_001.pdf" class="ext-link">Supporting Information</a>. All experiments utilizing animals were conducted under licenses from the appropriate local authorities following approved protocols as detailed in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01019/suppl_file/jm0c01019_si_001.pdf" class="ext-link">Supporting Information</a>.</div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i30"><a href="/doi/suppl/10.1021/acs.jmedchem.0c01019" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14467" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14467" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c01019?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01019</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Experimental procedures for the synthetic sequences shown in <a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a>–<a class="ref internalNav" href="#sch4" aria-label="4">4</a>, methods of synthesis, characterizing data for all the final compounds,  details of how the molecular modeling, biological, and physicochemical assays were conducted (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01019/suppl_file/jm0c01019_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">FGFR4 kinase-domain homology model (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01019/suppl_file/jm0c01019_si_002.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Molecular formula strings (SMILES) with associated biological data (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01019/suppl_file/jm0c01019_si_003.xlsx" class="ext-link">XLSX</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01019/suppl_file/jm0c01019_si_001.pdf">jm0c01019_si_001.pdf (6.98 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01019/suppl_file/jm0c01019_si_002.pdb">jm0c01019_si_002.pdb (197.42 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01019/suppl_file/jm0c01019_si_003.xlsx">jm0c01019_si_003.xlsx (15.72 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last"><b>84</b> cocrystallized with the kinase domain of FGFR4 (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YI8">6YI8</a>). Authors will release the atomic coordinates upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.0c01019" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45336" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45336" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Robin A. Fairhurst</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes for
BioMedical Research, CH-4002 Basel, Switzerland</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9924-6291" title="Orcid link">http://orcid.org/0000-0002-9924-6291</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#8af8e5e8e3e4a4ecebe3f8e2fff8f9fecae4e5fcebf8fee3f9a4e9e5e7"><span class="__cf_email__" data-cfemail="10627f72797e3e767179627865626364507e7f6671626479633e737f7d">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Thomas Knoepfel</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes for
BioMedical Research, CH-4002 Basel, Switzerland</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1675-6297" title="Orcid link">http://orcid.org/0000-0002-1675-6297</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nicole Buschmann</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes for
BioMedical Research, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Catherine Leblanc</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes for
BioMedical Research, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Robert Mah</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes for
BioMedical Research, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Milen Todorov</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes for
BioMedical Research, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Pierre Nimsgern</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes for
BioMedical Research, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sebastien Ripoche</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes for
BioMedical Research, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michel Niklaus</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes for
BioMedical Research, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nicolas Warin</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes for
BioMedical Research, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Van Huy Luu</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes for
BioMedical Research, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mario Madoerin</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes for
BioMedical Research, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jasmin Wirth</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes for
BioMedical Research, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Diana Graus-Porta</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes for
BioMedical Research, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andreas Weiss</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes for
BioMedical Research, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael Kiffe</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes for
BioMedical Research, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Markus Wartmann</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes for
BioMedical Research, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jacqueline Kinyamu-Akunda</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes for
BioMedical Research, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dario Sterker</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes for
BioMedical Research, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christelle Stamm</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes for
BioMedical Research, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Flavia Adler</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes for
BioMedical Research, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alexandra Buhles</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes for
BioMedical Research, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Heiko Schadt</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes for
BioMedical Research, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Philippe Couttet</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes for
BioMedical Research, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jutta Blank</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes for
BioMedical Research, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Inga Galuba</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes for
BioMedical Research, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jörg Trappe</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes for
BioMedical Research, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Johannes Voshol</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes for
BioMedical Research, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nils Ostermann</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes for
BioMedical Research, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chao Zou</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes for
BioMedical Research, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jörg Berghausen</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes for
BioMedical Research, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alberto Del Rio Espinola</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes for
BioMedical Research, CH-4002 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wolfgang Jahnke</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes for
BioMedical Research, CH-4002 Basel, Switzerland</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7003-3305" title="Orcid link">http://orcid.org/0000-0002-7003-3305</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Pascal Furet</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes for
BioMedical Research, CH-4002 Basel, Switzerland</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): Authors include employees of Novartis AG and/or stockholders of Novartis AG.<br /></br><p class="inlineNote">X-ray structural information for the structure of <b>84</b> (CCDC code: 2009566).</p></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i33">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64617" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64617" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We are grateful to Elvira Masso and Sandro Giger for technical assistance in preparing the compounds; Hansjoerg Lehmann, Michael Parmentier, and Bernhard Erb for supporting the large scale synthesis of intermediates and final compounds; Astrid Pornon for technical assistance with the proliferation assays; Christelle Henry for conducting the <sup>1</sup>H NMR experiments with <sup>13</sup>C-labeled <b>18</b>; Ina Dix, Philippe Piechon, and Trixi Wagner for carrying out the single crystal X-ray crystallography to determine the structure of compound <b>84</b>; Paulo G. Santos and Alessandro Cavecchi for carrying out the in vivo formulation work; Sandrine Desrayaud, Dallas Bednarczyk, David Pearson, Gaelle Chenal, Grit Laue, Matthias Kittelmann, and Fabian Eggimann for conducting the metabolism and PK investigations; Alex A. Pérez for directing and Kuno Wuersch and Elizabeth Skuba for carrying out the pathology evaluation for the toxicology study with compound <b>68</b>; Andrew Pape, Richard Ducray, and Min Dong for project management; Luigi Manenti and Andrea Myers for enabling the clinical studies with <b>84</b>; Susanne Oswald for assistance with the preparation of the manuscript; Alan Abrams for preparing the cover artwork; Proteros Biostructures GmbH for conducting the kinetic binding assay studies and determining the X-ray cocrystal structure of <b>84</b> with FGFR4; Tobias Gabriel, Mike Dillon, Francesco Hofmann, and William R. Sellers for their insight guidance and support throughout the project.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">2-FQA</td><td class="NLM_def"><p class="first last">2-formylquinoline amide</p></td></tr><tr><td class="NLM_term">2-FTHNU</td><td class="NLM_def"><p class="first last">2-formyl tetrahydronaphthyridine urea</p></td></tr><tr><td class="NLM_term">AUC</td><td class="NLM_def"><p class="first last">area under the curve</p></td></tr><tr><td class="NLM_term">aq</td><td class="NLM_def"><p class="first last">aqueous</p></td></tr><tr><td class="NLM_term">BA</td><td class="NLM_def"><p class="first last">bile acid</p></td></tr><tr><td class="NLM_term">b.i.d.</td><td class="NLM_def"><p class="first last">twice a day</p></td></tr><tr><td class="NLM_term"><i>C</i></td><td class="NLM_def"><p class="first last">concentration</p></td></tr><tr><td class="NLM_term">CCDC</td><td class="NLM_def"><p class="first last">Cambridge Crystallographic Data Centre</p></td></tr><tr><td class="NLM_term">CDI</td><td class="NLM_def"><p class="first last">carbonyl dimidazole</p></td></tr><tr><td class="NLM_term">CDT</td><td class="NLM_def"><p class="first last">carbonyl ditriazole</p></td></tr><tr><td class="NLM_term">CL</td><td class="NLM_def"><p class="first last">clearance</p></td></tr><tr><td class="NLM_term">CMC</td><td class="NLM_def"><p class="first last">carboxymethylcellulose</p></td></tr><tr><td class="NLM_term">CYP7A1</td><td class="NLM_def"><p class="first last">cytochrome P450 7A1</p></td></tr><tr><td class="NLM_term">DAST</td><td class="NLM_def"><p class="first last">diethylaminosulfur trifluoride</p></td></tr><tr><td class="NLM_term">DCC</td><td class="NLM_def"><p class="first last">dextran-coated charcoal</p></td></tr><tr><td class="NLM_term">DCE</td><td class="NLM_def"><p class="first last">1,2-dichloroethane</p></td></tr><tr><td class="NLM_term">Δ<i>H</i><sub>fus</sub></td><td class="NLM_def"><p class="first last">heat of fusion</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">DMA</td><td class="NLM_def"><p class="first last">dimethylacetamide</p></td></tr><tr><td class="NLM_term">DMAP</td><td class="NLM_def"><p class="first last">4-dimethylaminopyridine</p></td></tr><tr><td class="NLM_term">dppf</td><td class="NLM_def"><p class="first last">bis(diphenylphosphino)ferrocene</p></td></tr><tr><td class="NLM_term">DSC</td><td class="NLM_def"><p class="first last">differential scanning calorimetry</p></td></tr><tr><td class="NLM_term">ER</td><td class="NLM_def"><p class="first last">extraction ratio</p></td></tr><tr><td class="NLM_term">ERK</td><td class="NLM_def"><p class="first last">extracellular signal-regulated kinase</p></td></tr><tr><td class="NLM_term"><i>F</i></td><td class="NLM_def"><p class="first last">bioavailability</p></td></tr><tr><td class="NLM_term">FGF</td><td class="NLM_def"><p class="first last">fibroblast growth factor</p></td></tr><tr><td class="NLM_term">RMS</td><td class="NLM_def"><p class="first last">rhabdomyosarcoma</p></td></tr><tr><td class="NLM_term">FGFR</td><td class="NLM_def"><p class="first last">fibroblast growth factor receptor</p></td></tr><tr><td class="NLM_term">FRS2</td><td class="NLM_def"><p class="first last">fibroblast growth factor receptor substrate 2</p></td></tr><tr><td class="NLM_term">GK+2</td><td class="NLM_def"><p class="first last">two positions beyond the gate-keeper residue</p></td></tr><tr><td class="NLM_term">HCC</td><td class="NLM_def"><p class="first last">hepatocellular carcinoma</p></td></tr><tr><td class="NLM_term">hERG</td><td class="NLM_def"><p class="first last">human ether-à-go-go-related gene channel</p></td></tr><tr><td class="NLM_term">iv</td><td class="NLM_def"><p class="first last">intravenously</p></td></tr><tr><td class="NLM_term">KLB</td><td class="NLM_def"><p class="first last">β-klotho</p></td></tr><tr><td class="NLM_term">LC–MS/MS</td><td class="NLM_def"><p class="first last">liquid chromatography–tandem mass spectrometry</p></td></tr><tr><td class="NLM_term">LiHMDS</td><td class="NLM_def"><p class="first last">lithium bis(trimethylsilyl)amide</p></td></tr><tr><td class="NLM_term">MAPK</td><td class="NLM_def"><p class="first last">mitogen-activated protein kinase</p></td></tr><tr><td class="NLM_term">MetID</td><td class="NLM_def"><p class="first last">metabolite identification</p></td></tr><tr><td class="NLM_term">MW</td><td class="NLM_def"><p class="first last">molecular weight</p></td></tr><tr><td class="NLM_term">NADPH</td><td class="NLM_def"><p class="first last">nicotinamide adenine dinucleotide phosphate</p></td></tr><tr><td class="NLM_term">NBS</td><td class="NLM_def"><p class="first last"><i>N</i>-bromosuccinimide</p></td></tr><tr><td class="NLM_term">NIS</td><td class="NLM_def"><p class="first last"><i>N</i>-iodosuccinimide</p></td></tr><tr><td class="NLM_term">NMR</td><td class="NLM_def"><p class="first last">nuclear magnetic resonance spectroscopy</p></td></tr><tr><td class="NLM_term">ORTEP</td><td class="NLM_def"><p class="first last">Oak Ridge thermal ellipsoid plot</p></td></tr><tr><td class="NLM_term"><i>P</i><sub>app</sub></td><td class="NLM_def"><p class="first last">apparent permeability coefficient</p></td></tr><tr><td class="NLM_term">PD</td><td class="NLM_def"><p class="first last">pharmacodynamic</p></td></tr><tr><td class="NLM_term">PDB</td><td class="NLM_def"><p class="first last">Protein Data Bank</p></td></tr><tr><td class="NLM_term">PEG</td><td class="NLM_def"><p class="first last">polyethylene glycol</p></td></tr><tr><td class="NLM_term">PK</td><td class="NLM_def"><p class="first last">pharmacokinetic</p></td></tr><tr><td class="NLM_term">po</td><td class="NLM_def"><p class="first last">per os (orally)</p></td></tr><tr><td class="NLM_term">PPB</td><td class="NLM_def"><p class="first last">plasma protein binding</p></td></tr><tr><td class="NLM_term">PSA</td><td class="NLM_def"><p class="first last">polar surface area</p></td></tr><tr><td class="NLM_term">q2d</td><td class="NLM_def"><p class="first last">every other day</p></td></tr><tr><td class="NLM_term">q.d.</td><td class="NLM_def"><p class="first last">once a day</p></td></tr><tr><td class="NLM_term">SD</td><td class="NLM_def"><p class="first last">standard deviation</p></td></tr><tr><td class="NLM_term">SEM</td><td class="NLM_def"><p class="first last">standard error of the mean</p></td></tr><tr><td class="NLM_term">SILAC</td><td class="NLM_def"><p class="first last">stable-isotope labeling with amino acids</p></td></tr><tr><td class="NLM_term">SSID</td><td class="NLM_def"><p class="first last">soft-spot identification</p></td></tr><tr><td class="NLM_term"><i>t</i><sub>1/2</sub></td><td class="NLM_def"><p class="first last">half-life</p></td></tr><tr><td class="NLM_term">T/C</td><td class="NLM_def"><p class="first last">treatment versus control</p></td></tr><tr><td class="NLM_term">TEL</td><td class="NLM_def"><p class="first last">ETS leukemia transcription factor</p></td></tr><tr><td class="NLM_term">TBDMS</td><td class="NLM_def"><p class="first last"><i>tert</i>-butyldimethylsilyl</p></td></tr><tr><td class="NLM_term"><i>V</i><sub>ss</sub></td><td class="NLM_def"><p class="first last">volume of distribution</p></td></tr><tr><td class="NLM_term">XRPD</td><td class="NLM_def"><p class="first last">X-ray powder diffraction.</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i35">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34856" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34856" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 71 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grose, R.</span></span> <span> </span><span class="NLM_article-title">Fibroblast growth factor signaling: from development to cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">116</span>– <span class="NLM_lpage">129</span>, <span class="refDoi"> DOI: 10.1038/nrc2780</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1038%2Fnrc2780" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=20094046" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC3cXps1Gktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=116-129&author=N.+Turnerauthor=R.+Grose&title=Fibroblast+growth+factor+signaling%3A+from+development+to+cancer&doi=10.1038%2Fnrc2780"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1aR"><div class="casContent"><span class="casTitleNuber">1a</span><div class="casTitle"><span class="NLM_cas:atitle">Fibroblast growth factor signalling: from development to cancer</span></div><div class="casAuthors">Turner, Nicholas; Grose, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">116-129</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Fibroblast growth factors (FGFs) and their receptors control a wide range of biol. functions, regulating cellular proliferation, survival, migration and differentiation.  Although targeting FGF signalling as a cancer therapeutic target has lagged behind that of other receptor tyrosine kinases, there is now substantial evidence for the importance of FGF signalling in the pathogenesis of diverse tumor types, and clin. reagents that specifically target the FGFs or FGF receptors are being developed.  Although FGF signalling can drive tumorigenesis, in different contexts FGF signalling can mediate tumor protective functions; the identification of the mechanisms that underlie these differential effects will be important to understand how FGF signalling can be most appropriately therapeutically targeted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUBfz6uAu97bVg90H21EOLACvtfcHk0lg6saHcinaW9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXps1Gktg%253D%253D&md5=7963bc75c149981560dffc8d91b4aaf1</span></div><a href="/servlet/linkout?suffix=cit1a&amp;dbid=16384&amp;doi=10.1038%2Fnrc2780&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2780%26sid%3Dliteratum%253Aachs%26aulast%3DTurner%26aufirst%3DN.%26aulast%3DGrose%26aufirst%3DR.%26atitle%3DFibroblast%2520growth%2520factor%2520signaling%253A%2520from%2520development%2520to%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2010%26volume%3D10%26spage%3D116%26epage%3D129%26doi%3D10.1038%2Fnrc2780" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit1b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Beenken, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohammadi, M.</span></span> <span> </span><span class="NLM_article-title">The FGF family: biology, pathophysiology and therapy</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">253</span>, <span class="refDoi"> DOI: 10.1038/nrd2792</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1038%2Fnrd2792" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=19247306" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BD1MXisVShur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=235-253&author=A.+Beenkenauthor=M.+Mohammadi&title=The+FGF+family%3A+biology%2C+pathophysiology+and+therapy&doi=10.1038%2Fnrd2792"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1bR"><div class="casContent"><span class="casTitleNuber">1b</span><div class="casTitle"><span class="NLM_cas:atitle">The FGF family: biology, pathophysiology and therapy</span></div><div class="casAuthors">Beenken, Andrew; Mohammadi, Moosa</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">235-253</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The family of fibroblast growth factors (FGFs) regulates a plethora of developmental processes, including brain patterning, branching morphogenesis and limb development.  Several mitogenic, cytoprotective and angiogenic therapeutic applications of FGFs are already being explored, and the recent discovery of the crucial roles of the endocrine-acting FGF19 subfamily in bile acid, glucose and phosphate homeostasis has sparked renewed interest in the pharmacol. potential of this family.  This review discusses traditional applications of recombinant FGFs and small-mol. FGF receptor kinase inhibitors in the treatment of cancer and cardiovascular disease and their emerging potential in the treatment of metabolic syndrome and hypophosphatemic diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmidKyrdG0lbVg90H21EOLACvtfcHk0linwSy0ak-NYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXisVShur4%253D&md5=4789ed4cb521a92114063a400a138645</span></div><a href="/servlet/linkout?suffix=cit1b&amp;dbid=16384&amp;doi=10.1038%2Fnrd2792&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2792%26sid%3Dliteratum%253Aachs%26aulast%3DBeenken%26aufirst%3DA.%26aulast%3DMohammadi%26aufirst%3DM.%26atitle%3DThe%2520FGF%2520family%253A%2520biology%252C%2520pathophysiology%2520and%2520therapy%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2009%26volume%3D8%26spage%3D235%26epage%3D253%26doi%3D10.1038%2Fnrd2792" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Katoh, M.</span></span> <span> </span><span class="NLM_article-title">Fibroblast growth factor receptors as treatment targets in clinical oncology</span>. <i>Nat. Rev. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">105</span>– <span class="NLM_lpage">122</span>, <span class="refDoi"> DOI: 10.1038/s41571-018-0115-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1038%2Fs41571-018-0115-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=30367139" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVaqt7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2019&pages=105-122&author=M.+Katoh&title=Fibroblast+growth+factor+receptors+as+treatment+targets+in+clinical+oncology&doi=10.1038%2Fs41571-018-0115-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2aR"><div class="casContent"><span class="casTitleNuber">2a</span><div class="casTitle"><span class="NLM_cas:atitle">Fibroblast growth factor receptors as treatment targets in clinical oncology</span></div><div class="casAuthors">Katoh, Masaru</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Clinical Oncology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">105-122</span>CODEN:
                <span class="NLM_cas:coden">NRCOAA</span>;
        ISSN:<span class="NLM_cas:issn">1759-4774</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  FGFRs are receptor tyrosine kinases with a role in several biol. processes, such as the regulation of development and tissue repair.  However, alterations in FGFRs 1-4, such as amplifications, fusions and mutations, as well as aberrant epigenetic or transcriptional regulation and changes in tumor-stromal interactions in the tumor microenvironment, can lead to the development and/or progression of cancer.  Similar to other kinase alterations, such alterations are targetable using small mols. or antibodies, and the benefits of FGFR inhibitors have been demonstrated in clin. trials involving subsets of patients with solid tumors harbouring FGFR alterations.  However, the response rates in patients with FGFR alterations were relatively low, and responses in patients without detectable FGFR alterations were also obsd.  In this Review, the author describes the clin. experience with FGFR inhibitors to date, and highlights key aspects that might lead to improved response rates and/or the avoidance of acquired resistance, including the selection of patients who are most likely to benefit from treatment, and the use of FGFR inhibitors in combination regimens with other agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8oCjTdYY2DLVg90H21EOLACvtfcHk0linwSy0ak-NYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVaqt7vO&md5=41440ace3cb26bac6d7cdb0e18b46c39</span></div><a href="/servlet/linkout?suffix=cit2a&amp;dbid=16384&amp;doi=10.1038%2Fs41571-018-0115-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41571-018-0115-y%26sid%3Dliteratum%253Aachs%26aulast%3DKatoh%26aufirst%3DM.%26atitle%3DFibroblast%2520growth%2520factor%2520receptors%2520as%2520treatment%2520targets%2520in%2520clinical%2520oncology%26jtitle%3DNat.%2520Rev.%2520Clin.%2520Oncol.%26date%3D2019%26volume%3D16%26spage%3D105%26epage%3D122%26doi%3D10.1038%2Fs41571-018-0115-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit2b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Ghedini, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ronca, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Presta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giacomini, A.</span></span> <span> </span><span class="NLM_article-title">Future applications of FGF/FGFR inhibitors in cancer</span>. <i>Expert Rev. Anticancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">861</span>– <span class="NLM_lpage">872</span>, <span class="refDoi"> DOI: 10.1080/14737140.2018.1491795</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1080%2F14737140.2018.1491795" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=29936878" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1KlsLrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2018&pages=861-872&author=G.+C.+Ghediniauthor=R.+Roncaauthor=M.+Prestaauthor=A.+Giacomini&title=Future+applications+of+FGF%2FFGFR+inhibitors+in+cancer&doi=10.1080%2F14737140.2018.1491795"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2bR"><div class="casContent"><span class="casTitleNuber">2b</span><div class="casTitle"><span class="NLM_cas:atitle">Future applications of FGF/FGFR inhibitors in cancer</span></div><div class="casAuthors">Ghedini, Gaia Cristina; Ronca, Roberto; Presta, Marco; Giacomini, Arianna</div><div class="citationInfo"><span class="NLM_cas:title">Expert Review of Anticancer Therapy</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">861-872</span>CODEN:
                <span class="NLM_cas:coden">ERATBJ</span>;
        ISSN:<span class="NLM_cas:issn">1473-7140</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  Deregulation of the fibroblast growth factor (FGF)/FGF receptor (FGFR) network occurs frequently in tumors due to gene amplification, activating mutations, and oncogenic fusions.  Thus, the development of FGF/FGFR-targeting therapies is the focus of several basic, preclin., and clin. studies.  This review will recapitulate the status of current FGF/FGFR-targeted drugs.  Non-selective FGF/FGFR inhibitors have been approved for cancer treatment but evidence highlights various complications affecting their use in the clin. practice.  It appears mandatory to identify FGF/FGFR alterations and appropriate biomarkers that may predict and monitor response to treatment, to establish the contribution of the FGF/FGFR system to the onset of mechanisms of drug resistance, and to develop effective combinations of FGF/FGFR inhibitors with other targeted therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4xcl4aHaVQ7Vg90H21EOLACvtfcHk0li9rhGi1ggyyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1KlsLrF&md5=98c33ddab04d181711a9439f7ebc75ca</span></div><a href="/servlet/linkout?suffix=cit2b&amp;dbid=16384&amp;doi=10.1080%2F14737140.2018.1491795&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F14737140.2018.1491795%26sid%3Dliteratum%253Aachs%26aulast%3DGhedini%26aufirst%3DG.%2BC.%26aulast%3DRonca%26aufirst%3DR.%26aulast%3DPresta%26aufirst%3DM.%26aulast%3DGiacomini%26aufirst%3DA.%26atitle%3DFuture%2520applications%2520of%2520FGF%252FFGFR%2520inhibitors%2520in%2520cancer%26jtitle%3DExpert%2520Rev.%2520Anticancer%2520Ther.%26date%3D2018%26volume%3D18%26spage%3D861%26epage%3D872%26doi%3D10.1080%2F14737140.2018.1491795" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.,  Jr.</span></span> <span> </span><span class="NLM_article-title">The role of fibroblast growth factor receptor (FGFR) protein-tyrosine kinase inhibitors in the treatment of cancers including those of the urinary bladder</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>151</i></span>,  <span class="NLM_fpage">104567</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2019.104567</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1016%2Fj.phrs.2019.104567" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=31770593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlegtr3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2020&pages=104567&author=R.+Roskoski&title=The+role+of+fibroblast+growth+factor+receptor+%28FGFR%29+protein-tyrosine+kinase+inhibitors+in+the+treatment+of+cancers+including+those+of+the+urinary+bladder&doi=10.1016%2Fj.phrs.2019.104567"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3aR"><div class="casContent"><span class="casTitleNuber">3a</span><div class="casTitle"><span class="NLM_cas:atitle">The role of fibroblast growth factor receptor (FGFR) protein-tyrosine kinase inhibitors in the treatment of cancers including those of the urinary bladder</span></div><div class="casAuthors">Roskoski, Robert Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">104567</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The human fibroblast growth factor family consists of 22 factors and five transmembrane receptors.  Of the 22 factors, eighteen are secreted while four of them function exclusively within the cell.  Four of the fibroblast growth factor receptors (FGFRs) possess intracellular protein-tyrosine kinase activity while the fifth (FGFRL1) has a short 105-residue intracellular non-enzymic component.  The FGFR protein kinase domain consists of a bi-lobed structure that is similar to that of all other protein kinases.  FGFR gene alterations occur in a wide variety of cancers including those of the urinary bladder, breast, ovary, prostate, endometrium, lung, and stomach.  The majority (66 %) of FGFR gene alterations involve gene amplifications, followed by mutations (26 %), and rearrangements that produce fusion proteins (8 %).  Erdafitinib was the first orally effective FGFR antagonist approved by the FDA (2019) for the treatment of advanced cancer, that of the urinary bladder.  FGF23 suppresses phosphate reabsorption in the proximal tubules of the kidney; FGF23 blockade allows phosphate reabsorption to occur and leads to elevated serum phosphate levels.  Erdafitinib and several other, but not all, FGFR antagonists produce hyperphosphatemia.  Erdafitinib binds to an inactive DGF-Din conformation of FGFR1 and is classified as a type I1/2 inhibitor.  Similarly, dovitinib, AZD4547, CH5183284, infigratinib, lenvatinib, LY2874455, and lucitanib are type I1/2 inhibitors.  The inactive conformations contain an autoinhibitory brake that is made up of three main residues: an asparagine (N) within the αC-β4 back loop, a glutamate (E) corresponding to the second hinge residue, and a lysine (K) in the β8-strand (the NEK triad).  PDGFRα/β, Kit, CSF1R, VEGFR1/2/3, Flt3, Tek, and Tie protein kinases are also regulated by a similar autoinhibitory brake mechanism.  Ponatinib binds to FGFR4 in a DFG-Dout conformation and is classified as a type II inhibitor.  Futibatinib, roblitinib, H3B-6527, fisogatinib, and PRN1371 bind covalently to their FGFR target and are classified as type VI inhibitors.  Nintedanib, pazopanib, pemigatinib, rogaratinib, fisogatinib, and PRN1371 are FGFR inhibitors lacking drug-enzyme crystal structures.  All of the aforementioned FGFR antagonists are orally effective.  The development of FGFR inhibitors has lagged behind those of other receptor protein-tyrosine kinases.  However, the FDA approval of erdafitinib for the treatment of urinary bladder cancers may stimulate addnl. work targeting the many other FGFR-driven neoplasms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoocEMK0pmYkbVg90H21EOLACvtfcHk0li9rhGi1ggyyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlegtr3I&md5=004af19e5027e49768812596aebef459</span></div><a href="/servlet/linkout?suffix=cit3a&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2019.104567&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2019.104567%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DThe%2520role%2520of%2520fibroblast%2520growth%2520factor%2520receptor%2520%2528FGFR%2529%2520protein-tyrosine%2520kinase%2520inhibitors%2520in%2520the%2520treatment%2520of%2520cancers%2520including%2520those%2520of%2520the%2520urinary%2520bladder%26jtitle%3DPharmacol.%2520Res.%26date%3D2020%26volume%3D151%26spage%3D104567%26doi%3D10.1016%2Fj.phrs.2019.104567" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit3b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, W.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, L.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.-Y.</span></span> <span> </span><span class="NLM_article-title">Recent developments and advances of FGFR as a potential target in cancer</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">2109</span>– <span class="NLM_lpage">2126</span>, <span class="refDoi"> DOI: 10.4155/fmc-2018-0103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.4155%2Ffmc-2018-0103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=30066580" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsF2gsLzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=2109-2126&author=W.-J.+Xueauthor=M.-T.+Liauthor=L.+Chenauthor=L.-P.+Sunauthor=Y.-Y.+Li&title=Recent+developments+and+advances+of+FGFR+as+a+potential+target+in+cancer&doi=10.4155%2Ffmc-2018-0103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3bR"><div class="casContent"><span class="casTitleNuber">3b</span><div class="casTitle"><span class="NLM_cas:atitle">Recent developments and advances of FGFR as a potential target in cancer</span></div><div class="casAuthors">Xue, Wen-Jun; Li, Ming-Tao; Chen, Lin; Sun, Li-Ping; Li, Yu-Yan</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2109-2126</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">FGFs and their receptors (FGFRs) are crit. for many biol. processes, including angiogenesis, wound healing and tissue regeneration.  Aberrations in FGFR signaling are common in cancer, making FGFRs a promising target in antitumor studies.  To date, many FGFR inhibitors are being detected in clin. studies, and resistance to some inhibitors has emerged.  Understanding the mechanisms of resistance is a fundamental step for further implementation of targeted therapies.  In this review, we will describe the basic knowledge regarding FGF/FGFR signaling and categorize the clin. FGFR inhibitors.  The mechanisms of resistance to FGFR inhibitors and corresponding strategies of overcoming drug resistance will also be discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXiPsfDmCflbVg90H21EOLACvtfcHk0li9rhGi1ggyyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsF2gsLzL&md5=98c18a707727b4bc20961fa664cfabb0</span></div><a href="/servlet/linkout?suffix=cit3b&amp;dbid=16384&amp;doi=10.4155%2Ffmc-2018-0103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc-2018-0103%26sid%3Dliteratum%253Aachs%26aulast%3DXue%26aufirst%3DW.-J.%26aulast%3DLi%26aufirst%3DM.-T.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DSun%26aufirst%3DL.-P.%26aulast%3DLi%26aufirst%3DY.-Y.%26atitle%3DRecent%2520developments%2520and%2520advances%2520of%2520FGFR%2520as%2520a%2520potential%2520target%2520in%2520cancer%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2018%26volume%3D10%26spage%3D2109%26epage%3D2126%26doi%3D10.4155%2Ffmc-2018-0103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brameld, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owens, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verner, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venetsanakos, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradshaw, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, V. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tam, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaStant, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loughhead, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ton, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karr, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerritsen, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldstein, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Funk, J. O.</span></span> <span> </span><span class="NLM_article-title">Discovery of the irreversible covalent FGFR inhibitor 8-(3-(4-acryloylpiperazin-1-yl)propyl)-6-(2,6-dichloro-3,5-dimethoxyphenyl)-2-(methylamino)pyrido[2,3-<i>d</i>]pyrimidin-7(8<i>H</i>)-one (PRN1371) for the treatment of solid tumors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">6516</span>– <span class="NLM_lpage">6527</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00360</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00360" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVykt7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=6516-6527&author=K.+A.+Brameldauthor=T.+D.+Owensauthor=E.+Vernerauthor=E.+Venetsanakosauthor=J.+M.+Bradshawauthor=V.+T.+Phanauthor=D.+Tamauthor=K.+Leungauthor=J.+Shuauthor=J.+LaStantauthor=D.+G.+Loughheadauthor=T.+Tonauthor=D.+E.+Karrauthor=M.+E.+Gerritsenauthor=D.+M.+Goldsteinauthor=J.+O.+Funk&title=Discovery+of+the+irreversible+covalent+FGFR+inhibitor+8-%283-%284-acryloylpiperazin-1-yl%29propyl%29-6-%282%2C6-dichloro-3%2C5-dimethoxyphenyl%29-2-%28methylamino%29pyrido%5B2%2C3-d%5Dpyrimidin-7%288H%29-one+%28PRN1371%29+for+the+treatment+of+solid+tumors&doi=10.1021%2Facs.jmedchem.7b00360"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of the Irreversible Covalent FGFR Inhibitor 8-(3-(4-Acryloylpiperazin-1-yl)propyl)-6-(2,6-dichloro-3,5-dimethoxyphenyl)-2-(methylamino)pyrido[2,3-d]pyrimidin-7(8H)-one (PRN1371) for the Treatment of Solid Tumors</span></div><div class="casAuthors">Brameld, Ken A.; Owens, Timothy D.; Verner, Erik; Venetsanakos, Eleni; Bradshaw, J. Michael; Phan, Vernon T.; Tam, Danny; Leung, Kwan; Shu, Jin; LaStant, Jacob; Loughhead, David G.; Ton, Tony; Karr, Dane E.; Gerritsen, Mary E.; Goldstein, David M.; Funk, Jens Oliver</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6516-6527</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Aberrant signaling of the FGF/FGFR pathway occurs frequently in cancers and is an oncogenic driver in many solid tumors.  Clin. validation of FGFR as a therapeutic target has been demonstrated in bladder, liver, lung, breast, and gastric cancers.  Our goal was to develop an irreversible covalent inhibitor of FGFR1-4 for use in oncol. indications.  An irreversible covalent binding mechanism imparts many desirable pharmacol. benefits including high potency, selectivity, and prolonged target inhibition.  Herein we report the structure-based design, medicinal chem. optimization, and unique ADME assays of our irreversible covalent drug discovery program which culminated in the discovery of compd. 34 (PRN1371), a highly selective and potent FGFR1-4 inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpklTuvm1niy7Vg90H21EOLACvtfcHk0lhf9dHTiY0STQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVykt7fK&md5=987069b645d5e0e36ccda5e5c81520b9</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00360&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00360%26sid%3Dliteratum%253Aachs%26aulast%3DBrameld%26aufirst%3DK.%2BA.%26aulast%3DOwens%26aufirst%3DT.%2BD.%26aulast%3DVerner%26aufirst%3DE.%26aulast%3DVenetsanakos%26aufirst%3DE.%26aulast%3DBradshaw%26aufirst%3DJ.%2BM.%26aulast%3DPhan%26aufirst%3DV.%2BT.%26aulast%3DTam%26aufirst%3DD.%26aulast%3DLeung%26aufirst%3DK.%26aulast%3DShu%26aufirst%3DJ.%26aulast%3DLaStant%26aufirst%3DJ.%26aulast%3DLoughhead%26aufirst%3DD.%2BG.%26aulast%3DTon%26aufirst%3DT.%26aulast%3DKarr%26aufirst%3DD.%2BE.%26aulast%3DGerritsen%26aufirst%3DM.%2BE.%26aulast%3DGoldstein%26aufirst%3DD.%2BM.%26aulast%3DFunk%26aufirst%3DJ.%2BO.%26atitle%3DDiscovery%2520of%2520the%2520irreversible%2520covalent%2520FGFR%2520inhibitor%25208-%25283-%25284-acryloylpiperazin-1-yl%2529propyl%2529-6-%25282%252C6-dichloro-3%252C5-dimethoxyphenyl%2529-2-%2528methylamino%2529pyrido%255B2%252C3-d%255Dpyrimidin-7%25288H%2529-one%2520%2528PRN1371%2529%2520for%2520the%2520treatment%2520of%2520solid%2520tumors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D6516%26epage%3D6527%26doi%3D10.1021%2Facs.jmedchem.7b00360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bono, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Smet, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herbert, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Bock, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georgiadou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fons, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tjwa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alcouffe, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ny, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bianciotto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jonckx, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murakami, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanahan, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michielsen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sibrac, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dol-Gleizes, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazzone, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zacchigna, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herault, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rigon, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz de
Almodovar, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Claes, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blanc, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poesen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Segura, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gueguen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bordes, M.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambrechts, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Broussy, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van de Wouwer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michaux, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jean, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blacher, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Motte, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rom, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rakic, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katsuma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaeffer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yayon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Schepdael, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwalbe, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gervasio, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmeliet, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rozensky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dewerchin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simons, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herbert, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmeliet, P.</span></span> <span> </span><span class="NLM_article-title">Inhibition of tumor angiogenesis and growth by a small-molecule multi-FGF receptor blocker with allosteric properties</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">477</span>– <span class="NLM_lpage">488</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2013.02.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1016%2Fj.ccr.2013.02.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=23597562" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmtV2nsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=477-488&author=F.+Bonoauthor=F.+De+Smetauthor=C.+Herbertauthor=K.+De+Bockauthor=M.+Georgiadouauthor=P.+Fonsauthor=M.+Tjwaauthor=C.+Alcouffeauthor=A.+Nyauthor=M.+Bianciottoauthor=B.+Jonckxauthor=M.+Murakamiauthor=A.+A.+Lanahanauthor=C.+Michielsenauthor=D.+Sibracauthor=F.+Dol-Gleizesauthor=M.+Mazzoneauthor=S.+Zacchignaauthor=J.-P.+Heraultauthor=C.+Fischerauthor=P.+Rigonauthor=C.+Ruiz+de%0AAlmodovarauthor=F.+Claesauthor=I.+Blancauthor=K.+Poesenauthor=J.+Zhangauthor=I.+Seguraauthor=G.+Gueguenauthor=M.-F.+Bordesauthor=D.+Lambrechtsauthor=R.+Broussyauthor=M.+van+de+Wouwerauthor=C.+Michauxauthor=T.+Shimadaauthor=I.+Jeanauthor=S.+Blacherauthor=A.+Noelauthor=P.+Motteauthor=E.+Romauthor=J.-M.+Rakicauthor=S.+Katsumaauthor=P.+Schaefferauthor=A.+Yayonauthor=A.+Van+Schepdaelauthor=H.+Schwalbeauthor=F.+L.+Gervasioauthor=G.+Carmelietauthor=J.+Rozenskyauthor=M.+Dewerchinauthor=M.+Simonsauthor=A.+Christopoulosauthor=J.-M.+Herbertauthor=P.+Carmeliet&title=Inhibition+of+tumor+angiogenesis+and+growth+by+a+small-molecule+multi-FGF+receptor+blocker+with+allosteric+properties&doi=10.1016%2Fj.ccr.2013.02.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of Tumor Angiogenesis and Growth by a Small-Molecule Multi-FGF Receptor Blocker with Allosteric Properties</span></div><div class="casAuthors">Bono, Francoise; De Smet, Frederik; Herbert, Corentin; De Bock, Katrien; Georgiadou, Maria; Fons, Pierre; Tjwa, Marc; Alcouffe, Chantal; Ny, Annelii; Bianciotto, Marc; Jonckx, Bart; Murakami, Masahiro; Lanahan, Anthony A.; Michielsen, Christof; Sibrac, David; Dol-Gleizes, Frederique; Mazzone, Massimiliano; Zacchigna, Serena; Herault, Jean-Pascal; Fischer, Christian; Rigon, Patrice; Ruiz de Almodovar, Carmen; Claes, Filip; Blanc, Isabelle; Poesen, Koen; Zhang, Jie; Segura, Inmaculada; Gueguen, Genevieve; Bordes, Marie-Francoise; Lambrechts, Diether; Broussy, Roselyne; van de Wouwer, Marlies; Michaux, Corinne; Shimada, Toru; Jean, Isabelle; Blacher, Silvia; Noel, Agnes; Motte, Patrick; Rom, Eran; Rakic, Jean-Marie; Katsuma, Susumu; Schaeffer, Paul; Yayon, Avner; Van Schepdael, Ann; Schwalbe, Harald; Gervasio, Francesco Luigi; Carmeliet, Geert; Rozensky, Jef; Dewerchin, Mieke; Simons, Michael; Christopoulos, Arthur; Herbert, Jean-Marc; Carmeliet, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">477-488</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Receptor tyrosine kinases (RTK) are targets for anticancer drug development.  To date, only RTK inhibitors that block orthosteric binding of ligands and substrates have been developed.  Here, we report the pharmacol. characterization of the chem. SSR128129E (SSR), which inhibits fibroblast growth factor receptor (FGFR) signaling by binding to the extracellular FGFR domain without affecting orthosteric FGF binding.  SSR exhibits allosteric properties, including probe dependence, signaling bias, and ceiling effects.  Inhibition by SSR is highly conserved throughout the animal kingdom.  Oral delivery of SSR inhibits arthritis and tumors that are relatively refractory to anti-vascular endothelial growth factor receptor-2 antibodies.  Thus, orally-active extracellularly acting small-mol. modulators of RTKs with allosteric properties can be developed and may offer opportunities to improve anticancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuE-3w5S7QY7Vg90H21EOLACvtfcHk0lhf9dHTiY0STQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmtV2nsLg%253D&md5=ee0c854130ad23f910eb4783e081e6e5</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2013.02.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2013.02.019%26sid%3Dliteratum%253Aachs%26aulast%3DBono%26aufirst%3DF.%26aulast%3DDe%2BSmet%26aufirst%3DF.%26aulast%3DHerbert%26aufirst%3DC.%26aulast%3DDe%2BBock%26aufirst%3DK.%26aulast%3DGeorgiadou%26aufirst%3DM.%26aulast%3DFons%26aufirst%3DP.%26aulast%3DTjwa%26aufirst%3DM.%26aulast%3DAlcouffe%26aufirst%3DC.%26aulast%3DNy%26aufirst%3DA.%26aulast%3DBianciotto%26aufirst%3DM.%26aulast%3DJonckx%26aufirst%3DB.%26aulast%3DMurakami%26aufirst%3DM.%26aulast%3DLanahan%26aufirst%3DA.%2BA.%26aulast%3DMichielsen%26aufirst%3DC.%26aulast%3DSibrac%26aufirst%3DD.%26aulast%3DDol-Gleizes%26aufirst%3DF.%26aulast%3DMazzone%26aufirst%3DM.%26aulast%3DZacchigna%26aufirst%3DS.%26aulast%3DHerault%26aufirst%3DJ.-P.%26aulast%3DFischer%26aufirst%3DC.%26aulast%3DRigon%26aufirst%3DP.%26aulast%3DRuiz%2Bde%2BAlmodovar%26aufirst%3DC.%26aulast%3DClaes%26aufirst%3DF.%26aulast%3DBlanc%26aufirst%3DI.%26aulast%3DPoesen%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DSegura%26aufirst%3DI.%26aulast%3DGueguen%26aufirst%3DG.%26aulast%3DBordes%26aufirst%3DM.-F.%26aulast%3DLambrechts%26aufirst%3DD.%26aulast%3DBroussy%26aufirst%3DR.%26aulast%3Dvan%2Bde%2BWouwer%26aufirst%3DM.%26aulast%3DMichaux%26aufirst%3DC.%26aulast%3DShimada%26aufirst%3DT.%26aulast%3DJean%26aufirst%3DI.%26aulast%3DBlacher%26aufirst%3DS.%26aulast%3DNoel%26aufirst%3DA.%26aulast%3DMotte%26aufirst%3DP.%26aulast%3DRom%26aufirst%3DE.%26aulast%3DRakic%26aufirst%3DJ.-M.%26aulast%3DKatsuma%26aufirst%3DS.%26aulast%3DSchaeffer%26aufirst%3DP.%26aulast%3DYayon%26aufirst%3DA.%26aulast%3DVan%2BSchepdael%26aufirst%3DA.%26aulast%3DSchwalbe%26aufirst%3DH.%26aulast%3DGervasio%26aufirst%3DF.%2BL.%26aulast%3DCarmeliet%26aufirst%3DG.%26aulast%3DRozensky%26aufirst%3DJ.%26aulast%3DDewerchin%26aufirst%3DM.%26aulast%3DSimons%26aufirst%3DM.%26aulast%3DChristopoulos%26aufirst%3DA.%26aulast%3DHerbert%26aufirst%3DJ.-M.%26aulast%3DCarmeliet%26aufirst%3DP.%26atitle%3DInhibition%2520of%2520tumor%2520angiogenesis%2520and%2520growth%2520by%2520a%2520small-molecule%2520multi-FGF%2520receptor%2520blocker%2520with%2520allosteric%2520properties%26jtitle%3DCancer%2520Cell%26date%3D2013%26volume%3D23%26spage%3D477%26epage%3D488%26doi%3D10.1016%2Fj.ccr.2013.02.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Bartz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukuchi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohtsuka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lange, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gruner, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watanabe, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawaida, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komori, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kashimoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wirtz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, J-P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Redondo-Muller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saito, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanzawa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imai, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haussinger, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agatsuma, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bange, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abraham, R.</span></span> <span> </span><span class="NLM_article-title">Preclinical development of U3–1784, a Novel FGFR4 antibody against cancer, and avoidance of its on-target toxicity</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1832</span>– <span class="NLM_lpage">1843</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-18-0048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1158%2F1535-7163.MCT-18-0048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=31350344" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BB3cXmsFKruw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=1832-1843&author=R.+Bartzauthor=K.+Fukuchiauthor=T.+Ohtsukaauthor=T.+Langeauthor=K.+Grunerauthor=I.+Watanabeauthor=S.+Hayashiauthor=Y.+Odaauthor=R.+Kawaidaauthor=H.+Komoriauthor=Y.+Kashimotoauthor=P.+Wirtzauthor=J-P.+A.+Mayerauthor=M.+Redondo-Mullerauthor=S.+Saitoauthor=M.+Takahashiauthor=H.+Hanzawaauthor=E.+Imaiauthor=A.+Martinezauthor=M.+Hanaiauthor=D.+Haussingerauthor=R.+W.+Chapmanauthor=T.+Agatsumaauthor=J.+Bangeauthor=R.+Abraham&title=Preclinical+development+of+U3%E2%80%931784%2C+a+Novel+FGFR4+antibody+against+cancer%2C+and+avoidance+of+its+on-target+toxicity&doi=10.1158%2F1535-7163.MCT-18-0048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6aR"><div class="casContent"><span class="casTitleNuber">6a</span><div class="casTitle"><span class="NLM_cas:atitle">Preclinical development of U3-1784, a novel FGFR4 antibody against cancer, and avoidance of its on-target toxicity</span></div><div class="casAuthors">Bartz, Rene; Fukuchi, Keisuke; Ohtsuka, Toshiaki; Lange, Tanja; Gruner, Katrin; Watanabe, Ichiro; Hayashi, Shinko; Oda, Yoko; Kawaida, Reimi; Komori, Hironobu; Kashimoto, Yoshinori; Wirtz, Peter; Mayer, Jan-Peter A.; Redondo-Mueller, Mauricio; Saito, Shuntaro; Takahashi, Mizuki; Hanzawa, Hiroyuki; Imai, Emi; Martinez, Alberto; Hanai, Masaharu; Haeussinger, Dieter; Chapman, Roger W.; Agatsuma, Toshinori; Bange, Johannes; Abraham, Reimar</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1832-1843</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The FGFR4/FGF19 signaling axis is overactivated in 20% of liver tumors and currently represents a promising targetable signaling mechanism in this cancer type.  However, blocking FGFR4 or FGF19 has proven challenging due to its physiol. role in suppressing bile acid synthesis which leads to increased toxic bile acid plasma levels upon FGFR4 inhibition.  An FGFR4-targeting antibody, U3-1784, was generated in order to investigate its suitability as a cancer treatment without major side effects.  U3-1784 is a high-affinity fully human antibody that was obtained by phage display technol. and specifically binds to FGFR4.  The antibody inhibits cell signaling by competing with various FGFs for their FGFR4 binding site thereby inhibiting receptor activation and downstream signaling via FRS2 and Erk.  The antibody specifically slowed tumor growth of models overexpressing FGF19 by up to 90% whereas tumor growth of models not expressing FGF19 was unaffected.  In cynomolgus monkeys, i.v. injection of U3-1784 caused elevated serum bile acid and liver enzyme levels indicating potential liver damage.  These effects could be completely prevented by the concomitant oral treatment with the bile acid sequestrant colestyramine, which binds and eliminates bile acids in the gut.  These results offer a new biomarker-driven treatment modality in liver cancer without toxicity and they suggest a general strategy for avoiding adverse events with FGFR4 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7V7MCgojB7bVg90H21EOLACvtfcHk0lg5v-Be7GjwIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXmsFKruw%253D%253D&md5=22e8bfdf0a246648b637074de07af74f</span></div><a href="/servlet/linkout?suffix=cit6a&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-18-0048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-18-0048%26sid%3Dliteratum%253Aachs%26aulast%3DBartz%26aufirst%3DR.%26aulast%3DFukuchi%26aufirst%3DK.%26aulast%3DOhtsuka%26aufirst%3DT.%26aulast%3DLange%26aufirst%3DT.%26aulast%3DGruner%26aufirst%3DK.%26aulast%3DWatanabe%26aufirst%3DI.%26aulast%3DHayashi%26aufirst%3DS.%26aulast%3DOda%26aufirst%3DY.%26aulast%3DKawaida%26aufirst%3DR.%26aulast%3DKomori%26aufirst%3DH.%26aulast%3DKashimoto%26aufirst%3DY.%26aulast%3DWirtz%26aufirst%3DP.%26aulast%3DMayer%26aufirst%3DJ-P.%2BA.%26aulast%3DRedondo-Muller%26aufirst%3DM.%26aulast%3DSaito%26aufirst%3DS.%26aulast%3DTakahashi%26aufirst%3DM.%26aulast%3DHanzawa%26aufirst%3DH.%26aulast%3DImai%26aufirst%3DE.%26aulast%3DMartinez%26aufirst%3DA.%26aulast%3DHanai%26aufirst%3DM.%26aulast%3DHaussinger%26aufirst%3DD.%26aulast%3DChapman%26aufirst%3DR.%2BW.%26aulast%3DAgatsuma%26aufirst%3DT.%26aulast%3DBange%26aufirst%3DJ.%26aulast%3DAbraham%26aufirst%3DR.%26atitle%3DPreclinical%2520development%2520of%2520U3%25E2%2580%25931784%252C%2520a%2520Novel%2520FGFR4%2520antibody%2520against%2520cancer%252C%2520and%2520avoidance%2520of%2520its%2520on-target%2520toxicity%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2019%26volume%3D18%26spage%3D1832%26epage%3D1843%26doi%3D10.1158%2F1535-7163.MCT-18-0048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit6b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Desnoyers, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pai, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrando, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hötzel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carano, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Souza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qing, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohtashemi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashkenazi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, D. M.</span></span> <span> </span><span class="NLM_article-title">Targeting FGF19 inhibits tumor growth in colon cancer xenograft and FGF19 transgenic hepatocellular carcinoma models</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">97</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1210623</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1038%2Fsj.onc.1210623" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=17599042" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlSntg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2008&pages=85-97&author=L.+R.+Desnoyersauthor=R.+Paiauthor=R.+E.+Ferrandoauthor=K.+H%C3%B6tzelauthor=T.+Leauthor=J.+Rossauthor=R.+Caranoauthor=A.+D%E2%80%99Souzaauthor=J.+Qingauthor=I.+Mohtashemiauthor=A.+Ashkenaziauthor=D.+M.+French&title=Targeting+FGF19+inhibits+tumor+growth+in+colon+cancer+xenograft+and+FGF19+transgenic+hepatocellular+carcinoma+models&doi=10.1038%2Fsj.onc.1210623"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6bR"><div class="casContent"><span class="casTitleNuber">6b</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting FGF19 inhibits tumor growth in colon cancer xenograft and FGF19 transgenic hepatocellular carcinoma models</span></div><div class="casAuthors">Desnoyers, L. R.; Pai, R.; Ferrando, R. E.; Hoetzel, K.; Le, T.; Ross, J.; Carano, R.; D'Souza, A.; Qing, J.; Mohtashemi, I.; Ashkenazi, A.; French, D. M.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">85-97</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Although fibroblast growth factor 19 (FGF19) can promote liver carcinogenesis in mice its involvement in human cancer is not well characterized.  Here we report that FGF19 and its cognate receptor FGF receptor 4 (FGFR4) are coexpressed in primary human liver, lung and colon tumors and in a subset of human colon cancer cell lines.  To test the importance of FGF19 for tumor growth, we developed an anti-FGF19 monoclonal antibody that selectively blocks the interaction of FGF19 with FGFR4.  This antibody abolished FGF19-mediated activity in vitro and inhibited growth of colon tumor xenografts in vivo and effectively prevented hepatocellular carcinomas in FGF19 transgenic mice.  The efficacy of the antibody in these models was linked to inhibition of FGF19-dependent activation of FGFR4, FRS2, ERK and β-catenin.  These findings suggest that the inactivation of FGF19 could be beneficial for the treatment of colon cancer, liver cancer and other malignancies involving interaction of FGF19 and FGFR4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkYfmQMrOH_bVg90H21EOLACvtfcHk0lg5v-Be7GjwIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlSntg%253D%253D&md5=5b8980bcf38575516b375b8c6b6e8f19</span></div><a href="/servlet/linkout?suffix=cit6b&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1210623&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1210623%26sid%3Dliteratum%253Aachs%26aulast%3DDesnoyers%26aufirst%3DL.%2BR.%26aulast%3DPai%26aufirst%3DR.%26aulast%3DFerrando%26aufirst%3DR.%2BE.%26aulast%3DH%25C3%25B6tzel%26aufirst%3DK.%26aulast%3DLe%26aufirst%3DT.%26aulast%3DRoss%26aufirst%3DJ.%26aulast%3DCarano%26aufirst%3DR.%26aulast%3DD%25E2%2580%2599Souza%26aufirst%3DA.%26aulast%3DQing%26aufirst%3DJ.%26aulast%3DMohtashemi%26aufirst%3DI.%26aulast%3DAshkenazi%26aufirst%3DA.%26aulast%3DFrench%26aufirst%3DD.%2BM.%26atitle%3DTargeting%2520FGF19%2520inhibits%2520tumor%2520growth%2520in%2520colon%2520cancer%2520xenograft%2520and%2520FGF19%2520transgenic%2520hepatocellular%2520carcinoma%2520models%26jtitle%3DOncogene%26date%3D2008%26volume%3D27%26spage%3D85%26epage%3D97%26doi%3D10.1038%2Fsj.onc.1210623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferlay, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colombet, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soerjomataram, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathers, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parkin, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piñeros, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Znaor, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bray, F.</span></span> <span> </span><span class="NLM_article-title">Estimating the global cancer incidence and mortality in 2018: GLOBOCAN sources and methods</span>. <i>Int. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>144</i></span>,  <span class="NLM_fpage">1941</span>– <span class="NLM_lpage">1953</span>, <span class="refDoi"> DOI: 10.1002/ijc.31937</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1002%2Fijc.31937" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=30350310" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVaku77O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=144&publication_year=2019&pages=1941-1953&author=J.+Ferlayauthor=M.+Colombetauthor=I.+Soerjomataramauthor=C.+Mathersauthor=D.+M.+Parkinauthor=M.+Pi%C3%B1erosauthor=A.+Znaorauthor=F.+Bray&title=Estimating+the+global+cancer+incidence+and+mortality+in+2018%3A+GLOBOCAN+sources+and+methods&doi=10.1002%2Fijc.31937"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Estimating the global cancer incidence and mortality in 2018: GLOBOCAN sources and methods</span></div><div class="casAuthors">Ferlay, J.; Colombet, M.; Soerjomataram, I.; Mathers, C.; Parkin, D. M.; Pineros, M.; Znaor, A.; Bray, F.</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Cancer</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">144</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1941-1953</span>CODEN:
                <span class="NLM_cas:coden">IJCNAW</span>;
        ISSN:<span class="NLM_cas:issn">0020-7136</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Ests. of the worldwide incidence and mortality from 36 cancers and for all cancers combined for the year 2018 are now available in the GLOBOCAN 2018 database, compiled and disseminated by the International Agency for Research on Cancer (IARC).  This paper reviews the sources and methods used in compiling the cancer statistics in 185 countries.  The validity of the national ests. depends upon the representativeness of the source information, and to take into account possible sources of bias, uncertainty intervals are now provided for the estd. sex- and site-specific all-ages no. of new cancer cases and cancer deaths.  We briefly describe the key results globally and by world region.  There were an estd. 18.1 million (95% UI: 17.5-18.7 million) new cases of cancer (17 million excluding non-melanoma skin cancer) and 9.6 million (95% UI: 9.3-9.8 million) deaths from cancer (9.5 million excluding non-melanoma skin cancer) worldwide in 2018.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFkXwTr-9_ibVg90H21EOLACvtfcHk0lgODZFXoQSCZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVaku77O&md5=db8d895060c7770078fec16e2d9eb816</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1002%2Fijc.31937&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.31937%26sid%3Dliteratum%253Aachs%26aulast%3DFerlay%26aufirst%3DJ.%26aulast%3DColombet%26aufirst%3DM.%26aulast%3DSoerjomataram%26aufirst%3DI.%26aulast%3DMathers%26aufirst%3DC.%26aulast%3DParkin%26aufirst%3DD.%2BM.%26aulast%3DPi%25C3%25B1eros%26aufirst%3DM.%26aulast%3DZnaor%26aufirst%3DA.%26aulast%3DBray%26aufirst%3DF.%26atitle%3DEstimating%2520the%2520global%2520cancer%2520incidence%2520and%2520mortality%2520in%25202018%253A%2520GLOBOCAN%2520sources%2520and%2520methods%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D2019%26volume%3D144%26spage%3D1941%26epage%3D1953%26doi%3D10.1002%2Fijc.31937" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hainaut, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gores, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amadou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plymoth, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, L. R.</span></span> <span> </span><span class="NLM_article-title">A global view of hepatocellular carcinoma: trends, risk, prevention and management</span>. <i>Nat. Rev. Gastroenterol. Hepatol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">589</span>– <span class="NLM_lpage">604</span>, <span class="refDoi"> DOI: 10.1038/s41575-019-0186-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1038%2Fs41575-019-0186-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=31439937" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A280%3ADC%252BB3Mrgs1yntg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2019&pages=589-604&author=J.+D.+Yangauthor=P.+Hainautauthor=G.+J.+Goresauthor=A.+Amadouauthor=A.+Plymothauthor=L.+R.+Roberts&title=A+global+view+of+hepatocellular+carcinoma%3A+trends%2C+risk%2C+prevention+and+management&doi=10.1038%2Fs41575-019-0186-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">A global view of hepatocellular carcinoma: trends, risk, prevention and management</span></div><div class="casAuthors">Yang Ju Dong; Gores Gregory J; Roberts Lewis R; Yang Ju Dong; Yang Ju Dong; Yang Ju Dong; Hainaut Pierre; Amadou Amina; Plymoth Amelie</div><div class="citationInfo"><span class="NLM_cas:title">Nature reviews. Gastroenterology & hepatology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">589-604</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Hepatocellular carcinoma (HCC) is the fourth most common cause of cancer-related death worldwide.  Risk factors for HCC include chronic hepatitis B and hepatitis C, alcohol addiction, metabolic liver disease (particularly nonalcoholic fatty liver disease) and exposure to dietary toxins such as aflatoxins and aristolochic acid.  All these risk factors are potentially preventable, highlighting the considerable potential of risk prevention for decreasing the global burden of HCC.  HCC surveillance and early detection increase the chance of potentially curative treatment; however, HCC surveillance is substantially underutilized, even in countries with sufficient medical resources.  Early-stage HCC can be treated curatively by local ablation, surgical resection or liver transplantation.  Treatment selection depends on tumour characteristics, the severity of underlying liver dysfunction, age, other medical comorbidities, and available medical resources and local expertise.  Catheter-based locoregional treatment is used in patients with intermediate-stage cancer.  Kinase and immune checkpoint inhibitors have been shown to be effective treatment options in patients with advanced-stage HCC.  Together, rational deployment of prevention, attainment of global goals for viral hepatitis eradication, and improvements in HCC surveillance and therapy hold promise for achieving a substantial reduction in the worldwide HCC burden within the next few decades.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT_e_17NcpJhcULxIEhxUjPfW6udTcc2ebja88zKXKxhbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3Mrgs1yntg%253D%253D&md5=5a0d26728621fabe00c683714a206d18</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fs41575-019-0186-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41575-019-0186-y%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DJ.%2BD.%26aulast%3DHainaut%26aufirst%3DP.%26aulast%3DGores%26aufirst%3DG.%2BJ.%26aulast%3DAmadou%26aufirst%3DA.%26aulast%3DPlymoth%26aufirst%3DA.%26aulast%3DRoberts%26aufirst%3DL.%2BR.%26atitle%3DA%2520global%2520view%2520of%2520hepatocellular%2520carcinoma%253A%2520trends%252C%2520risk%252C%2520prevention%2520and%2520management%26jtitle%3DNat.%2520Rev.%2520Gastroenterol.%2520Hepatol.%26date%3D2019%26volume%3D16%26spage%3D589%26epage%3D604%26doi%3D10.1038%2Fs41575-019-0186-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Repana, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, P.</span></span> <span> </span><span class="NLM_article-title">Targeting FGF19/FGFR4 pathway: a novel therapeutic strategy for hepatocellular carcinoma</span>. <i>Diseases</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">294</span>– <span class="NLM_lpage">305</span>, <span class="refDoi"> DOI: 10.3390/diseases3040294</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.3390%2Fdiseases3040294" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=28943626" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC1cXltFCgurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2015&pages=294-305&author=D.+Repanaauthor=P.+Ross&title=Targeting+FGF19%2FFGFR4+pathway%3A+a+novel+therapeutic+strategy+for+hepatocellular+carcinoma&doi=10.3390%2Fdiseases3040294"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting FGF19/FGFR4 pathway: a novel therapeutic strategy for hepatocellular carcinoma</span></div><div class="casAuthors">Repana, Dimitra; Ross, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Diseases</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">294-305</span>CODEN:
                <span class="NLM_cas:coden">DISECP</span>;
        ISSN:<span class="NLM_cas:issn">2079-9721</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Hepatocellular carcinoma (HCC) is a lethal cancer with limited systemic therapeutic options.  Liver carcinogenesis is a complex procedure and various pathways have been found to be deregulated which are potential targets for novel treatments.  Aberrant signalling through FGF19 and its receptor FGFR4 seems to be the oncogenic driver for a subset of HCCs and is assocd. with poor prognosis.  Inhibition of the pathway in preclin. models has shown antitumor activity and has triggered further evaluation of this strategy to in vivo models.  This review aims to describe the role of the FGF19/FGFR4 pathway in hepatocellular carcinoma and its role as a potential predictive biomarker for novel targeted agents against FGF19/FGFR4 signalling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpREsw9FM-_67Vg90H21EOLACvtfcHk0lgODZFXoQSCZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXltFCgurs%253D&md5=319e81e1d0135c096c86f024dc1694cf</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.3390%2Fdiseases3040294&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fdiseases3040294%26sid%3Dliteratum%253Aachs%26aulast%3DRepana%26aufirst%3DD.%26aulast%3DRoss%26aufirst%3DP.%26atitle%3DTargeting%2520FGF19%252FFGFR4%2520pathway%253A%2520a%2520novel%2520therapeutic%2520strategy%2520for%2520hepatocellular%2520carcinoma%26jtitle%3DDiseases%26date%3D2015%26volume%3D3%26spage%3D294%26epage%3D305%26doi%3D10.3390%2Fdiseases3040294" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shim, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sung, C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baek, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haq, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ansari, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chun, S.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sung, W.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zucman-Rossi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, G.</span></span> <span> </span><span class="NLM_article-title">Genomic portrait of resectable hepatocellular carcinomas: implications of RB1 and FGF19 aberrations for patient stratification</span>. <i>Hepatology</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1972</span>– <span class="NLM_lpage">1982</span>, <span class="refDoi"> DOI: 10.1002/hep.27198</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1002%2Fhep.27198" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=24798001" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVCis7%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2014&pages=1972-1982&author=S.+M.+Ahnauthor=S.+J.+Jangauthor=J.+H.+Shimauthor=D.+Kimauthor=S.+M.+Hongauthor=C.+O.+Sungauthor=D.+Baekauthor=F.+Haqauthor=A.+A.+Ansariauthor=S.+Y.+Leeauthor=S.-M.+Chunauthor=S.+Choiauthor=H.-J.+Choiauthor=J.+Kimauthor=S.+Kimauthor=S.+Hwangauthor=Y.-J.+Leeauthor=J.+Leeauthor=W.+Jungauthor=H.+Y.+Jangauthor=E.+Yangauthor=W.-K.+Sungauthor=N.+P.+Leeauthor=M.+Maoauthor=C.+Leeauthor=J.+Zucman-Rossiauthor=E.+Yuauthor=H.+C.+Leeauthor=G.+Kong&title=Genomic+portrait+of+resectable+hepatocellular+carcinomas%3A+implications+of+RB1+and+FGF19+aberrations+for+patient+stratification&doi=10.1002%2Fhep.27198"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10aR"><div class="casContent"><span class="casTitleNuber">10a</span><div class="casTitle"><span class="NLM_cas:atitle">Genomic portrait of resectable hepatocellular carcinomas: Implications of RB1 and FGF19 aberrations for patient stratification</span></div><div class="casAuthors">Ahn, Sung-Min; Jang, Se Jin; Shim, Ju Hyun; Kim, Deokhoon; Hong, Seung-Mo; Sung, Chang Ohk; Baek, Daehyun; Haq, Farhan; Ansari, Adnan Ahmad; Lee, Sun Young; Chun, Sung-Min; Choi, Seongmin; Choi, Hyun-Jeung; Kim, Jongkyu; Kim, Sukjun; Hwang, Shin; Lee, Young-Joo; Lee, Jong-eun; Jung, Wang-rim; Jang, Hye Yoon; Yang, Eunho; Sung, Wing-Kin; Lee, Nikki P.; Mao, Mao; Lee, Charles; Zucman-Rossi, Jessica; Yu, Eunsil; Lee, Han Chu; Kong, Gu</div><div class="citationInfo"><span class="NLM_cas:title">Hepatology (Hoboken, NJ, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1972-1982</span>CODEN:
                <span class="NLM_cas:coden">HPTLD9</span>;
        ISSN:<span class="NLM_cas:issn">0270-9139</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Hepatic resection is the most curative treatment option for early-stage hepatocellular carcinoma, but is assocd. with a high recurrence rate, which exceeds 50% at 5 years after surgery.  Understanding the genetic basis of hepatocellular carcinoma at surgically curable stages may enable the identification of new mol. biomarkers that accurately identify patients in need of addnl. early therapeutic interventions.  Whole exome sequencing and copy no. anal. was performed on 231 hepatocellular carcinomas (72% with hepatitis B viral infection) that were classified as early-stage hepatocellular carcinomas, candidates for surgical resection.  Recurrent mutations were validated by Sanger sequencing.  Unsupervised genomic analyses identified an assocn. between specific genetic aberrations and postoperative clin. outcomes.  Recurrent somatic mutations were identified in nine genes, including TP53, CTNNB1, AXIN1, RPS6KA3, and RB1.  Recurrent homozygous deletions in FAM123A, RB1, and CDKN2A, and high-copy amplifications in MYC, RSPO2, CCND1, and FGF19 were detected.  Pathway analyses of these genes revealed aberrations in the p53, Wnt, PIK3/Ras, cell cycle, and chromatin remodeling pathways.  RB1 mutations were significantly assocd. with cancer-specific and recurrence-free survival after resection (multivariate P = 0.038 and P = 0.012, resp.).  FGF19 amplifications, known to activate Wnt signaling, were mutually exclusive with CTNNB1 and AXIN1 mutations, and significantly assocd. with cirrhosis (P = 0.017).  Conclusion: RB1 mutations can be used as a prognostic mol. biomarker for resectable hepatocellular carcinoma.  Further study is required to investigate the potential role of FGF19 amplification in driving hepatocarcinogenesis in patients with liver cirrhosis and to investigate the potential of anti-FGF19 treatment in these patients. (Hepatol. 2014;60:1971-1981).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIUWBV3H0t5rVg90H21EOLACvtfcHk0lgpyqZpabRJgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVCis7%252FN&md5=57e2374a0731562433edabe109f21025</span></div><a href="/servlet/linkout?suffix=cit10a&amp;dbid=16384&amp;doi=10.1002%2Fhep.27198&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fhep.27198%26sid%3Dliteratum%253Aachs%26aulast%3DAhn%26aufirst%3DS.%2BM.%26aulast%3DJang%26aufirst%3DS.%2BJ.%26aulast%3DShim%26aufirst%3DJ.%2BH.%26aulast%3DKim%26aufirst%3DD.%26aulast%3DHong%26aufirst%3DS.%2BM.%26aulast%3DSung%26aufirst%3DC.%2BO.%26aulast%3DBaek%26aufirst%3DD.%26aulast%3DHaq%26aufirst%3DF.%26aulast%3DAnsari%26aufirst%3DA.%2BA.%26aulast%3DLee%26aufirst%3DS.%2BY.%26aulast%3DChun%26aufirst%3DS.-M.%26aulast%3DChoi%26aufirst%3DS.%26aulast%3DChoi%26aufirst%3DH.-J.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DS.%26aulast%3DHwang%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DY.-J.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DJung%26aufirst%3DW.%26aulast%3DJang%26aufirst%3DH.%2BY.%26aulast%3DYang%26aufirst%3DE.%26aulast%3DSung%26aufirst%3DW.-K.%26aulast%3DLee%26aufirst%3DN.%2BP.%26aulast%3DMao%26aufirst%3DM.%26aulast%3DLee%26aufirst%3DC.%26aulast%3DZucman-Rossi%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DE.%26aulast%3DLee%26aufirst%3DH.%2BC.%26aulast%3DKong%26aufirst%3DG.%26atitle%3DGenomic%2520portrait%2520of%2520resectable%2520hepatocellular%2520carcinomas%253A%2520implications%2520of%2520RB1%2520and%2520FGF19%2520aberrations%2520for%2520patient%2520stratification%26jtitle%3DHepatology%26date%3D2014%26volume%3D60%26spage%3D1972%26epage%3D1982%26doi%3D10.1002%2Fhep.27198" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit10b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Sawey, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chanrion, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zender, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Busuttil, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yee, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stein, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finn, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powers, S.</span></span> <span> </span><span class="NLM_article-title">Identification of a therapeutic strategy targeting amplified FGF19 in liver cancer by oncogenomic screening</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">347</span>– <span class="NLM_lpage">358</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2011.01.040</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1016%2Fj.ccr.2011.01.040" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=21397858" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjtFKisrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=347-358&author=E.+T.+Saweyauthor=M.+Chanrionauthor=C.+Caiauthor=G.+Wuauthor=J.+Zhangauthor=L.+Zenderauthor=A.+Zhaoauthor=R.+W.+Busuttilauthor=H.+Yeeauthor=L.+Steinauthor=D.+M.+Frenchauthor=R.+S.+Finnauthor=S.+W.+Loweauthor=S.+Powers&title=Identification+of+a+therapeutic+strategy+targeting+amplified+FGF19+in+liver+cancer+by+oncogenomic+screening&doi=10.1016%2Fj.ccr.2011.01.040"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10bR"><div class="casContent"><span class="casTitleNuber">10b</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a Therapeutic Strategy Targeting Amplified FGF19 in Liver Cancer by Oncogenomic Screening</span></div><div class="casAuthors">Sawey, Eric T.; Chanrion, Maia; Cai, Chunlin; Wu, Guanming; Zhang, Jianping; Zender, Lars; Zhao, Alice; Busuttil, Ronald W.; Yee, Herman; Stein, Lincoln; French, Dorothy M.; Finn, Richard S.; Lowe, Scott W.; Powers, Scott</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">347-358</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: We screened 124 genes that are amplified in human hepatocellular carcinoma (HCC) using a mouse hepatoblast model and identified 18 tumor-promoting genes, including CCND1 and its neighbor on 11q13.3, FGF19.  Although it is widely assumed that CCND1 is the main driving oncogene of this common amplicon (15% frequency in HCC), both forward-transformation assays and RNAi-mediated inhibition in human HCC cells established that FGF19 is an equally important driver gene in HCC.  Furthermore, clonal growth and tumorigenicity of HCC cells harboring the 11q13.3 amplicon were selectively inhibited by RNAi-mediated knockdown of CCND1 or FGF19, as well as by an anti-FGF19 antibody.  These results show that 11q13.3 amplification could be an effective biomarker for patients most likely to respond to anti-FGF19 therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbWNCR5PCyb7Vg90H21EOLACvtfcHk0lgpyqZpabRJgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjtFKisrs%253D&md5=eb5934a5f2fde11c7e7bf385582ac0af</span></div><a href="/servlet/linkout?suffix=cit10b&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2011.01.040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2011.01.040%26sid%3Dliteratum%253Aachs%26aulast%3DSawey%26aufirst%3DE.%2BT.%26aulast%3DChanrion%26aufirst%3DM.%26aulast%3DCai%26aufirst%3DC.%26aulast%3DWu%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DZender%26aufirst%3DL.%26aulast%3DZhao%26aufirst%3DA.%26aulast%3DBusuttil%26aufirst%3DR.%2BW.%26aulast%3DYee%26aufirst%3DH.%26aulast%3DStein%26aufirst%3DL.%26aulast%3DFrench%26aufirst%3DD.%2BM.%26aulast%3DFinn%26aufirst%3DR.%2BS.%26aulast%3DLowe%26aufirst%3DS.%2BW.%26aulast%3DPowers%26aufirst%3DS.%26atitle%3DIdentification%2520of%2520a%2520therapeutic%2520strategy%2520targeting%2520amplified%2520FGF19%2520in%2520liver%2520cancer%2520by%2520oncogenomic%2520screening%26jtitle%3DCancer%2520Cell%26date%3D2011%26volume%3D19%26spage%3D347%26epage%3D358%26doi%3D10.1016%2Fj.ccr.2011.01.040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicholes, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guillet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomlinson, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hillan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frantz, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pham, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillard-Telm, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephan, J.-P</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stinson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stewart, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, D. M.</span></span> <span> </span><span class="NLM_article-title">A mouse model of hepatocellular carcinoma</span>. <i>Am. J. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>160</i></span>,  <span class="NLM_fpage">2295</span>– <span class="NLM_lpage">2307</span>, <span class="refDoi"> DOI: 10.1016/S0002-9440(10)61177-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1016%2FS0002-9440%2810%2961177-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=12057932" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BD38XltFOrsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=160&publication_year=2002&pages=2295-2307&author=K.+Nicholesauthor=S.+Guilletauthor=E.+Tomlinsonauthor=K.+Hillanauthor=B.+Wrightauthor=G.+D.+Frantzauthor=T.+A.+Phamauthor=L.+Dillard-Telmauthor=S.+P.+Tsaiauthor=J.-P+Stephanauthor=J.+Stinsonauthor=T.+Stewartauthor=D.+M.+French&title=A+mouse+model+of+hepatocellular+carcinoma&doi=10.1016%2FS0002-9440%2810%2961177-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">A mouse model of hepatocellular carcinoma: Ectopic expression of fibroblast growth factor 19 in skeletal muscle of transgenic mice</span></div><div class="casAuthors">Nicholes, Katrina; Guillet, Susan; Tomlinson, Elizabeth; Hillan, Kenneth; Wright, Barbara; Frantz, Gretchen D.; Pham, Thinh A.; Dillard-Telm, Lisa; Tsai, Siao Ping; Stephan, Jean-Philippe; Stinson, Jeremy; Stewart, Timothy; French, Dorothy M.</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Pathology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">160</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2295-2307</span>CODEN:
                <span class="NLM_cas:coden">AJPAA4</span>;
        ISSN:<span class="NLM_cas:issn">0002-9440</span>.
    
            (<span class="NLM_cas:orgname">American Society for Investigative Pathology</span>)
        </div><div class="casAbstract">Most mouse models of hepatocellular carcinoma have expressed growth factors and oncogenes under the control of a liver-specific promoter.  In contrast, the authors describe here the formation of liver tumors in transgenic mice overexpressing human fibroblast growth factor 19 (FGF19) in skeletal muscle.  FGF19 transgenic mice had elevated hepatic α-fetoprotein mRNA as early as 2 mo of age, and hepatocellular carcinomas were evident by 10 mo of age.  Increased proliferation of pericentral hepatocytes was demonstrated by 5-bromo-2'-deoxyuridine incorporation in the FGF19 transgenic mice before tumor formation and in nontransgenic mice injected with recombinant FGF19 protein.  Areas of small cell dysplasia were initially evident pericentrally, and dysplastic/neoplastic foci throughout the hepatic lobule were glutamine synthetase-pos., suggestive of a pericentral origin.  Consistent with chronic activation of the Wingless/Wnt pathway, 44% of the hepatocellular tumors from FGF19 transgenic mice had nuclear staining for β-catenin.  Sequencing of the tumor DNA encoding β-catenin revealed point mutations that resulted in amino acid substitutions.  These findings suggest a previously unknown role for FGF19 in hepatocellular carcinomas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9ekQaICJqerVg90H21EOLACvtfcHk0li8X6JnAqZKqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XltFOrsbs%253D&md5=c3355b5f65fea242cfb7ee357e2dd4a7</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2FS0002-9440%2810%2961177-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0002-9440%252810%252961177-7%26sid%3Dliteratum%253Aachs%26aulast%3DNicholes%26aufirst%3DK.%26aulast%3DGuillet%26aufirst%3DS.%26aulast%3DTomlinson%26aufirst%3DE.%26aulast%3DHillan%26aufirst%3DK.%26aulast%3DWright%26aufirst%3DB.%26aulast%3DFrantz%26aufirst%3DG.%2BD.%26aulast%3DPham%26aufirst%3DT.%2BA.%26aulast%3DDillard-Telm%26aufirst%3DL.%26aulast%3DTsai%26aufirst%3DS.%2BP.%26aulast%3DStephan%26aufirst%3DJ.-P%26aulast%3DStinson%26aufirst%3DJ.%26aulast%3DStewart%26aufirst%3DT.%26aulast%3DFrench%26aufirst%3DD.%2BM.%26atitle%3DA%2520mouse%2520model%2520of%2520hepatocellular%2520carcinoma%26jtitle%3DAm.%2520J.%2520Pathol.%26date%3D2002%26volume%3D160%26spage%3D2295%26epage%3D2307%26doi%3D10.1016%2FS0002-9440%2810%2961177-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Nogova, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sequist, L. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez
Garcia, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andre, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delord, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hidalgo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schellens, J. H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cassier, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camidge, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaishampayan, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burris, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campone, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wainberg, Z. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, W.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LoRusso, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choudhury, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ringeisen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graus-Porta, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isaacs, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buettner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, J.</span></span> <span> </span><span class="NLM_article-title">Evaluation of BGJ398, a fibroblast growth factor receptor 1–3 kinase inhibitor, in patients with advanced solid tumors harboring genetic alterations in fibroblast growth factor receptors: results of a global phase I, dose-escalation and dose-expansion study</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">157</span>– <span class="NLM_lpage">165</span>, <span class="refDoi"> DOI: 10.1200/JCO.2016.67.2048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1200%2FJCO.2016.67.2048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=27870574" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlensr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2017&pages=157-165&author=L.+Nogovaauthor=L.+V.+Sequistauthor=J.+M.+Perez%0AGarciaauthor=F.+Andreauthor=J.-P.+Delordauthor=M.+Hidalgoauthor=J.+H.+M.+Schellensauthor=P.+A.+Cassierauthor=D.+R.+Camidgeauthor=M.+Schulerauthor=U.+Vaishampayanauthor=H.+Burrisauthor=G.+G.+Tianauthor=M.+Camponeauthor=Z.+A.+Wainbergauthor=W.-T.+Limauthor=P.+LoRussoauthor=G.+I.+Shapiroauthor=K.+Parkerauthor=X.+Chenauthor=S.+Choudhuryauthor=F.+Ringeisenauthor=D.+Graus-Portaauthor=D.+Porterauthor=R.+Isaacsauthor=R.+Buettnerauthor=J.+Wolf&title=Evaluation+of+BGJ398%2C+a+fibroblast+growth+factor+receptor+1%E2%80%933+kinase+inhibitor%2C+in+patients+with+advanced+solid+tumors+harboring+genetic+alterations+in+fibroblast+growth+factor+receptors%3A+results+of+a+global+phase+I%2C+dose-escalation+and+dose-expansion+study&doi=10.1200%2FJCO.2016.67.2048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12aR"><div class="casContent"><span class="casTitleNuber">12a</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of BGJ398, a fibroblast growth factor receptor 1-3 kinase inhibitor, in patients with advanced solid tumors harboring genetic alterations in fibroblast growth factor receptors: results of a global phase I, dose-escalation and dose-expansion study</span></div><div class="casAuthors">Nogova, Lucia; Sequist, Lecia V.; Garcia, Jose Manuel Perez; Andre, Fabrice; Delord, Jean-Pierre; Hidalgo, Manuel; Schellens, Jan H. M.; Cassier, Philippe A.; Camidge, D. Ross; Schuler, Martin; Vaishampayan, Ulka; Burris, Howard; Tian, G. Gary; Campone, Mario; Wainberg, Zev A.; Lim, Wan-Teck; Lo Russo, Patricia; Shapiro, Geoffrey I.; Parker, Katie; Chen, Xueying; Choudhury, Somesh; Ringeisen, Francois; Graus-Porta, Diana; Porter, Dale; Isaacs, Randi; Buettner, Reinhard; Wolf, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Oncology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">157-175</span>CODEN:
                <span class="NLM_cas:coden">JCONDN</span>;
        ISSN:<span class="NLM_cas:issn">1527-7755</span>.
    
            (<span class="NLM_cas:orgname">American Society of Clinical Oncology</span>)
        </div><div class="casAbstract">Purpose: This two-part, first-in-human study was initiated in patient swith advanced solid tumors harboring genetic alterations in fibroblast growth factor receptors (FGFRs) to det. the max. tolerated dose (MTD), the recommended phase II dose (RP2D), and the schedule, safety, pharmacokinetics, pharmacodynamics, and antitumor activity of oral BGJ398, a selective FGFR1-3 tyrosine kinase inhibitor.  Patients and Methods: Adult patients were treated with escalating dosages of BGJ398 5 to 150 mg once daily or 50 mg twice daily continuously in 28-day cycles.  During expansion at the MTD, patients with FGFR1-amplified squamous cell non-small-cell lung cancer (sqNSCLC; arm 1) or other solid tumors with FGFR genetic alterations (mutations/amplifications/fusions) received BGJ398 daily on a continuous schedule (arm 2), or on a 3-wk-on/1-wk-off schedule (arm 3).  Results: Data in 132 patients from the escalation and expansion arms are reported (May 15, 2015, cutoff).  The MTD, 125 mg daily, was detd. on the basis of dose-limiting toxicities in four patients (100 mg, grade 3 aminotransferase elevations [n = 1]; 125 mg, hyperphosphatemia [n = 1]; 150 mg, grade 1 corneal toxicity [n = 1] and grade 3 aminotransferase elevations [n = 1]).  Common adverse events in patients treated at the MTD (n = 57) included hyperphosphatemia (82.5%), constipation (50.9%), decreased appetite (45.6%), and stomatitis (45.6%).  A similar safety profile was obsd. using the 3-wk-on/1-wk-off schedule (RP2D).  However, adverse event-related dose adjustments/interruptions were less frequent with the 3-wk-on/1-wk-off (50.0%) vs. the continuous (73.7%) schedule.  Antitumor activity (seven partial responses [six confirmed]) was demonstrated with BGJ398 doses ≥ 100 mg in patients with FGFR1-amplified sqNSCLC and FGFR3-mutant bladder/urothelial cancer.  Conclusion: BGJ398 at the MTD/RP2D had a tolerable and manageable safety profile and showed antitumor activity in several tumor types, including FGFR1-amplified sqNSCLC and FGFR3-mutant bladder/urothelial cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0KJ9yf2nMQrVg90H21EOLACvtfcHk0li8X6JnAqZKqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlensr0%253D&md5=b0204f7a46eeaa059fe732ad35508ca9</span></div><a href="/servlet/linkout?suffix=cit12a&amp;dbid=16384&amp;doi=10.1200%2FJCO.2016.67.2048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2016.67.2048%26sid%3Dliteratum%253Aachs%26aulast%3DNogova%26aufirst%3DL.%26aulast%3DSequist%26aufirst%3DL.%2BV.%26aulast%3DPerez%2BGarcia%26aufirst%3DJ.%2BM.%26aulast%3DAndre%26aufirst%3DF.%26aulast%3DDelord%26aufirst%3DJ.-P.%26aulast%3DHidalgo%26aufirst%3DM.%26aulast%3DSchellens%26aufirst%3DJ.%2BH.%2BM.%26aulast%3DCassier%26aufirst%3DP.%2BA.%26aulast%3DCamidge%26aufirst%3DD.%2BR.%26aulast%3DSchuler%26aufirst%3DM.%26aulast%3DVaishampayan%26aufirst%3DU.%26aulast%3DBurris%26aufirst%3DH.%26aulast%3DTian%26aufirst%3DG.%2BG.%26aulast%3DCampone%26aufirst%3DM.%26aulast%3DWainberg%26aufirst%3DZ.%2BA.%26aulast%3DLim%26aufirst%3DW.-T.%26aulast%3DLoRusso%26aufirst%3DP.%26aulast%3DShapiro%26aufirst%3DG.%2BI.%26aulast%3DParker%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DChoudhury%26aufirst%3DS.%26aulast%3DRingeisen%26aufirst%3DF.%26aulast%3DGraus-Porta%26aufirst%3DD.%26aulast%3DPorter%26aufirst%3DD.%26aulast%3DIsaacs%26aufirst%3DR.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DWolf%26aufirst%3DJ.%26atitle%3DEvaluation%2520of%2520BGJ398%252C%2520a%2520fibroblast%2520growth%2520factor%2520receptor%25201%25E2%2580%25933%2520kinase%2520inhibitor%252C%2520in%2520patients%2520with%2520advanced%2520solid%2520tumors%2520harboring%2520genetic%2520alterations%2520in%2520fibroblast%2520growth%2520factor%2520receptors%253A%2520results%2520of%2520a%2520global%2520phase%2520I%252C%2520dose-escalation%2520and%2520dose-expansion%2520study%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2017%26volume%3D35%26spage%3D157%26epage%3D165%26doi%3D10.1200%2FJCO.2016.67.2048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit12b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Hierro, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tabernero, J.</span></span> <span> </span><span class="NLM_article-title">Fibroblast growth factor (FGF) receptor/FGF inhibitors: novel targets and strategies for optimization of response of solid tumors</span>. <i>Semin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">801</span>– <span class="NLM_lpage">819</span>, <span class="refDoi"> DOI: 10.1053/j.seminoncol.2015.09.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1053%2Fj.seminoncol.2015.09.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=26615127" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFWksL7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2015&pages=801-819&author=C.+Hierroauthor=J.+Rodonauthor=J.+Tabernero&title=Fibroblast+growth+factor+%28FGF%29+receptor%2FFGF+inhibitors%3A+novel+targets+and+strategies+for+optimization+of+response+of+solid+tumors&doi=10.1053%2Fj.seminoncol.2015.09.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12bR"><div class="casContent"><span class="casTitleNuber">12b</span><div class="casTitle"><span class="NLM_cas:atitle">Fibroblast Growth Factor (FGF) Receptor/FGF Inhibitors: Novel Targets and Strategies for Optimization of Response of Solid Tumors</span></div><div class="casAuthors">Hierro, Cinta; Rodon, Jordi; Tabernero, Josep</div><div class="citationInfo"><span class="NLM_cas:title">Seminars in Oncology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">801-819</span>CODEN:
                <span class="NLM_cas:coden">SOLGAV</span>;
        ISSN:<span class="NLM_cas:issn">0093-7754</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The fibroblast growth factor receptor (FGFR) pathway plays a major role in several biol. processes, from organogenesis to metab. homeostasis and angiogenesis.  Several aberrations, including gene amplifications, point mutations, and chromosomal translocations have been described across solid tumors.  Most of these mol. alterations promote multiple steps of carcinogenesis in FGFR oncogene-addicted cells, increasing cell proliferation, angiogenesis, and drug resistance.  Data suggest that upregulation of FGFR signaling is a common event in many cancer types.  The FGFR pathway thus arises as a potential promising target for cancer treatment.  Several FGFR inhibitors are currently under development.  Initial preclin. results have translated into limited successful clin. responses when first-generation, nonspecific FGFR inhibitors were evaluated in patients.  The future development of selective and unselective FGFR inhibitors will rely on a better understanding of the tissue-specific role of FGFR signaling and identification of biomarkers to select those patients who will benefit the most from these drugs.  Further studies are warranted to establish the predictive significance of the different FGFR-aberrations and to incorporate them into clin. algorithms, now that second-generation, selective FGFR inhibitors exist.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruNHv_T2hq8bVg90H21EOLACvtfcHk0ljxbIhOLMA3QQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFWksL7F&md5=b1b150e2f8a903aab696070b71a9247c</span></div><a href="/servlet/linkout?suffix=cit12b&amp;dbid=16384&amp;doi=10.1053%2Fj.seminoncol.2015.09.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1053%252Fj.seminoncol.2015.09.027%26sid%3Dliteratum%253Aachs%26aulast%3DHierro%26aufirst%3DC.%26aulast%3DRodon%26aufirst%3DJ.%26aulast%3DTabernero%26aufirst%3DJ.%26atitle%3DFibroblast%2520growth%2520factor%2520%2528FGF%2529%2520receptor%252FFGF%2520inhibitors%253A%2520novel%2520targets%2520and%2520strategies%2520for%2520optimization%2520of%2520response%2520of%2520solid%2520tumors%26jtitle%3DSemin.%2520Oncol.%26date%3D2015%26volume%3D42%26spage%3D801%26epage%3D819%26doi%3D10.1053%2Fj.seminoncol.2015.09.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Sarabipour, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hristova, K.</span></span> <span> </span><span class="NLM_article-title">Mechanism of FGF receptor dimerization and activation</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">10262</span>, <span class="refDoi"> DOI: 10.1038/ncomms10262</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1038%2Fncomms10262" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=26725515" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC28Xjslanug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=10262&author=S.+Sarabipourauthor=K.+Hristova&title=Mechanism+of+FGF+receptor+dimerization+and+activation&doi=10.1038%2Fncomms10262"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13aR"><div class="casContent"><span class="casTitleNuber">13a</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism of FGF receptor dimerization and activation</span></div><div class="casAuthors">Sarabipour, Sarvenaz; Hristova, Kalina</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">10262</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Fibroblast growth factors (fgfs) are widely believed to activate their receptors by mediating receptor dimerization.  Here we show, however, that the FGF receptors form dimers in the absence of ligand, and that these unliganded dimers are phosphorylated.  We further show that ligand binding triggers structural changes in the FGFR dimers, which increase FGFR phosphorylation.  The obsd. effects due to the ligands fgf1 and fgf2 are very different.  The fgf2-bound dimer structure ensures the smallest sepn. between the transmembrane (TM) domains and the highest possible phosphorylation, a conclusion that is supported by a strong correlation between TM helix sepn. in the dimer and kinase phosphorylation.  The pathogenic A391E mutation in FGFR3 TM domain emulates the action of fgf2, trapping the FGFR3 dimer in its most active state.  This study establishes the existence of multiple active ligand-bound states, and uncovers a novel mol. mechanism through which FGFR-linked pathologies can arise.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrl3F_9xgzAOLVg90H21EOLACvtfcHk0ljxbIhOLMA3QQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xjslanug%253D%253D&md5=3c87746c27a7e92fb437f91649ad6c8e</span></div><a href="/servlet/linkout?suffix=cit13a&amp;dbid=16384&amp;doi=10.1038%2Fncomms10262&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms10262%26sid%3Dliteratum%253Aachs%26aulast%3DSarabipour%26aufirst%3DS.%26aulast%3DHristova%26aufirst%3DK.%26atitle%3DMechanism%2520of%2520FGF%2520receptor%2520dimerization%2520and%2520activation%26jtitle%3DNat.%2520Commun.%26date%3D2016%26volume%3D7%26spage%3D10262%26doi%3D10.1038%2Fncomms10262" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit13b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Lesca, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lammens, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Augustin, M.</span></span> <span> </span><span class="NLM_article-title">Structural analysis of the human fibroblast growth factor receptor 4 kinase</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>426</i></span>,  <span class="NLM_fpage">3744</span>– <span class="NLM_lpage">3756</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2014.09.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1016%2Fj.jmb.2014.09.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=25219510" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFyitb7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=426&publication_year=2014&pages=3744-3756&author=E.+Lescaauthor=A.+Lammensauthor=R.+Huberauthor=M.+Augustin&title=Structural+analysis+of+the+human+fibroblast+growth+factor+receptor+4+kinase&doi=10.1016%2Fj.jmb.2014.09.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13bR"><div class="casContent"><span class="casTitleNuber">13b</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Analysis of the Human Fibroblast Growth Factor Receptor 4 Kinase</span></div><div class="casAuthors">Lesca, E.; Lammens, A.; Huber, R.; Augustin, M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">426</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">3744-3756</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The family of fibroblast growth factor receptors (FGFRs) plays an important and well-characterized role in a variety of pathol. disorders.  FGFR4 is involved in myogenesis and muscle regeneration.  Mutations affecting the kinase domain of FGFR4 may cause cancer, for example, breast cancer or rhabdomyosarcoma.  Whereas FGFR1-FGFR3 have been structurally characterized, the structure of the FGFR4 kinase domain has not yet been reported.  In this study, we present four structures of the kinase domain of FGFR4, in its apo-form and in complex with different types of small-mol. inhibitors.  The two apo-FGFR4 kinase domain structures show an activation segment similar in conformation to an autoinhibitory segment obsd. in the hepatocyte growth factor receptor kinase but different from the known structures of other FGFR kinases.  The structures of FGFR4 in complex with the type I inhibitor Dovitinib and the type II inhibitor Ponatinib reveal the mol. interactions with different types of kinase inhibitors and may assist in the design and development of FGFR4 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9Wm0Poe_gXrVg90H21EOLACvtfcHk0ljxbIhOLMA3QQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFyitb7E&md5=ee9d7f9bc9aa10dbec92f3eb6b80b44f</span></div><a href="/servlet/linkout?suffix=cit13b&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2014.09.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2014.09.004%26sid%3Dliteratum%253Aachs%26aulast%3DLesca%26aufirst%3DE.%26aulast%3DLammens%26aufirst%3DA.%26aulast%3DHuber%26aufirst%3DR.%26aulast%3DAugustin%26aufirst%3DM.%26atitle%3DStructural%2520analysis%2520of%2520the%2520human%2520fibroblast%2520growth%2520factor%2520receptor%25204%2520kinase%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2014%26volume%3D426%26spage%3D3744%26epage%3D3756%26doi%3D10.1016%2Fj.jmb.2014.09.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q.</span></span> <span> </span><span class="NLM_article-title">Role of fibroblast growth factor receptor 4 in cancer</span>. <i>Cancer Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">3024</span>– <span class="NLM_lpage">3031</span>, <span class="refDoi"> DOI: 10.1111/cas.13759</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1111%2Fcas.13759" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=30070748" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVCqtb7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2018&pages=3024-3031&author=S.+Tangauthor=Y.+Haoauthor=Y.+Yuanauthor=R.+Liuauthor=Q.+Chen&title=Role+of+fibroblast+growth+factor+receptor+4+in+cancer&doi=10.1111%2Fcas.13759"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14aR"><div class="casContent"><span class="casTitleNuber">14a</span><div class="casTitle"><span class="NLM_cas:atitle">Role of fibroblast growth factor receptor 4 in cancer</span></div><div class="casAuthors">Tang, Shuya; Hao, Yilong; Yuan, Yao; Liu, Rui; Chen, Qianming</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Science</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3024-3031</span>CODEN:
                <span class="NLM_cas:coden">CSACCM</span>;
        ISSN:<span class="NLM_cas:issn">1349-7006</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Fibroblast growth factor receptors (FGFR) play a significant role in both embryonic development and in adults.  Upon binding with ligands, FGFR signaling is activated and triggers various downstream signal cascades that are implicated in diverse biol. processes.  Aberrant regulations of FGFR signaling are detected in numerous cancers.  Although FGFR4 was discovered later than other FGFR, information on the involvement of FGFR4 in cancers has significantly increased in recent years.  In this review, the recent findings in FGFR4 structure, signaling transduction, physiol. function, aberrant regulations, and effects in cancers as well as its potential applications as an anticancer therapeutic target are summarized.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdw5EJC-Ta-rVg90H21EOLACvtfcHk0lgOFpILqXcpiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVCqtb7F&md5=d65681255072f30701fc8730fb7938fa</span></div><a href="/servlet/linkout?suffix=cit14a&amp;dbid=16384&amp;doi=10.1111%2Fcas.13759&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcas.13759%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DS.%26aulast%3DHao%26aufirst%3DY.%26aulast%3DYuan%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DQ.%26atitle%3DRole%2520of%2520fibroblast%2520growth%2520factor%2520receptor%25204%2520in%2520cancer%26jtitle%3DCancer%2520Sci.%26date%3D2018%26volume%3D109%26spage%3D3024%26epage%3D3031%26doi%3D10.1111%2Fcas.13759" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit14b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Gotoh, N.</span></span> <span> </span><span class="NLM_article-title">Regulation of growth factor signaling by FRS2 family docking/scaffold adaptor proteins</span>. <i>Cancer Sci.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">1319</span>– <span class="NLM_lpage">1325</span>, <span class="refDoi"> DOI: 10.1111/j.1349-7006.2008.00840.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1111%2Fj.1349-7006.2008.00840.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=18452557" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BD1cXot1eitrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2008&pages=1319-1325&author=N.+Gotoh&title=Regulation+of+growth+factor+signaling+by+FRS2+family+docking%2Fscaffold+adaptor+proteins&doi=10.1111%2Fj.1349-7006.2008.00840.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14bR"><div class="casContent"><span class="casTitleNuber">14b</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of growth factor signaling by FRS2 family docking/scaffold adaptor proteins</span></div><div class="casAuthors">Gotoh, Noriko</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Science</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1319-1325</span>CODEN:
                <span class="NLM_cas:coden">CSACCM</span>;
        ISSN:<span class="NLM_cas:issn">1347-9032</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Asia Pty Ltd.</span>)
        </div><div class="casAbstract">A review.  The FRS2 family of adaptor/scaffold proteins has two members, FRS2α and FRS2β.  Both proteins contain N-terminal myristylation sites for localization on the plasma membrane and a PTB domain for binding to limited species of receptor tyrosine kinases (RTKs), including the FGF receptor, the neurotrophin receptor, RET, and ALK.  Activation of these RTKs allows FRS2 proteins to become phosphorylated of tyrosine residues and then bind to Grb2 and Shp2, a SH2 domain-contg. adaptor and a tyrosine phosphatase, resp.  Subsequently, Shp2 activates a Ras/ERK pathway and Grb2 activates a Ras/ERK, phosphatidylinositol (PI)-3 kinase and ubiquitination/degrdn. pathways by binding to SOS, Gab1, and Cbl via the SH3 domains of Grb2.  FRS2α acts as 'a conning center' in FGF signaling mainly because it induces sustained levels of activation of ERK via Shp2-binding sites and Grb2-binding sites, though the contribution of the former is greater.  Indeed, FRS2α knockout mice and mice with mutated Shp2-binding sites exhibit a variety of phenotypes due to defects in FGF signaling in vivo.  Although FRS2β binds to the EGF receptor, it does not induce tyrosine phosphorylation on the receptor.  Instead, it inhibits EGF signaling, resulting in inhibition of EGF-induced cell proliferation and cell transformation.  Based on these findings, the involvement of FRS2 proteins in tumorigenesis should be studied extensively to be validated as candidate biomarkers for the effectiveness of treatments targeting RTKs such as the FGF receptor and EGF receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnJLrQGKGN0rVg90H21EOLACvtfcHk0lgOFpILqXcpiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXot1eitrg%253D&md5=51684ae103a15bd6161590568b305082</span></div><a href="/servlet/linkout?suffix=cit14b&amp;dbid=16384&amp;doi=10.1111%2Fj.1349-7006.2008.00840.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1349-7006.2008.00840.x%26sid%3Dliteratum%253Aachs%26aulast%3DGotoh%26aufirst%3DN.%26atitle%3DRegulation%2520of%2520growth%2520factor%2520signaling%2520by%2520FRS2%2520family%2520docking%252Fscaffold%2520adaptor%2520proteins%26jtitle%3DCancer%2520Sci.%26date%3D2008%26volume%3D99%26spage%3D1319%26epage%3D1325%26doi%3D10.1111%2Fj.1349-7006.2008.00840.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackmore, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desnoyers, L. R.</span></span> <span> </span><span class="NLM_article-title">Liver-specific activities of FGF19 require Klotho beta</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>282</i></span>,  <span class="NLM_fpage">27277</span>– <span class="NLM_lpage">27284</span>, <span class="refDoi"> DOI: 10.1074/jbc.M704244200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1074%2Fjbc.M704244200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=17627937" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVWjsLfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=27277-27284&author=B.+C.+Linauthor=M.+Wangauthor=C.+Blackmoreauthor=L.+R.+Desnoyers&title=Liver-specific+activities+of+FGF19+require+Klotho+beta&doi=10.1074%2Fjbc.M704244200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15aR"><div class="casContent"><span class="casTitleNuber">15a</span><div class="casTitle"><span class="NLM_cas:atitle">Liver-specific activities of FGF19 Require Klotho beta</span></div><div class="casAuthors">Lin, Benjamin C.; Wang, Manping; Blackmore, Craig; Desnoyers, Luc R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">27277-27284</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Hepatocyte function is regulated by members of the fibroblast growth factor (FGF) family of proteins, but little is known about the specific mol. mechanisms of this endocrine pathway.  FGF19 regulates bile acid homeostasis and gall bladder filling; FGF19 binds only to FGF receptor 4 (FGFR4), but its liver-specific activity cannot be explained solely by the distribution of this receptor.  Although it has been suggested that Klotho beta (KLB) may have a role in mediating FGF19 activity, we have provided for the first time definitive evidence that KLB is required for FGF19 binding to FGFR4, intracellular signaling, and downstream modulation of gene expression.  We have shown that FGFR4 is widely distributed in mouse, whereas KLB distribution is more restricted.  Liver was the only organ in which both genes were abundantly expressed.  We show that in mice, FGF19 injection triggers liver-specific induction of c-Fos and repression of CYP7A1.  The tissue-specific activity of FGF19 supports the unique intersection of KLB and FGFR4 distribution in liver.  These studies define KLB as a novel FGFR4 coreceptor required for FGF19 liver specific functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPLlo-ZnNt-LVg90H21EOLACvtfcHk0lgOFpILqXcpiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVWjsLfE&md5=432206e9ae0424491964bb99d1db41f0</span></div><a href="/servlet/linkout?suffix=cit15a&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M704244200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M704244200%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DB.%2BC.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DBlackmore%26aufirst%3DC.%26aulast%3DDesnoyers%26aufirst%3DL.%2BR.%26atitle%3DLiver-specific%2520activities%2520of%2520FGF19%2520require%2520Klotho%2520beta%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D27277%26epage%3D27284%26doi%3D10.1074%2Fjbc.M704244200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit15b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Inagaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moschetta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummins, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodwin, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerard, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repa, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangelsdorf, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kliewer, S. A.</span></span> <span> </span><span class="NLM_article-title">Fibroblast growth factor 15 functions as an enterohepatic signal to regulate bile acid homeostasis</span>. <i>Cell Metab.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">217</span>– <span class="NLM_lpage">225</span>, <span class="refDoi"> DOI: 10.1016/j.cmet.2005.09.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1016%2Fj.cmet.2005.09.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=16213224" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFGktrjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2005&pages=217-225&author=T.+Inagakiauthor=M.+Choiauthor=A.+Moschettaauthor=L.+Pengauthor=C.+L.+Cumminsauthor=J.+G.+McDonaldauthor=G.+Luoauthor=S.+A.+Jonesauthor=B.+Goodwinauthor=J.+A.+Richardsonauthor=R.+D.+Gerardauthor=J.+J.+Repaauthor=D.+J.+Mangelsdorfauthor=S.+A.+Kliewer&title=Fibroblast+growth+factor+15+functions+as+an+enterohepatic+signal+to+regulate+bile+acid+homeostasis&doi=10.1016%2Fj.cmet.2005.09.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15bR"><div class="casContent"><span class="casTitleNuber">15b</span><div class="casTitle"><span class="NLM_cas:atitle">Fibroblast growth factor 15 functions as an enterohepatic signal to regulate bile acid homeostasis</span></div><div class="casAuthors">Inagaki, Takeshi; Choi, Mihwa; Moschetta, Antonio; Peng, Li; Cummins, Carolyn L.; McDonald, Jeffrey G.; Luo, Guizhen; Jones, Stacey A.; Goodwin, Bryan; Richardson, James A.; Gerard, Robert D.; Repa, Joyce J.; Mangelsdorf, David J.; Kliewer, Steven A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Metabolism</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">217-225</span>CODEN:
                <span class="NLM_cas:coden">CMEEB5</span>;
        ISSN:<span class="NLM_cas:issn">1550-4131</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The liver and intestine play crucial roles in maintaining bile acid homeostasis.  Fibroblast growth factor 15 (FGF15) signals from intestine to liver to repress the gene encoding cholesterol 7α-hydroxylase (CYP7A1), which catalyzes the first and rate-limiting step in the classical bile acid synthetic pathway.  FGF15 expression is stimulated in the small intestine by the nuclear bile acid receptor FXR and represses Cyp7a1 in liver through a mechanism that involves FGF receptor 4 (FGFR4) and the orphan nuclear receptor SHP.  Mice lacking FGF15 have increased hepatic CYP7A1 mRNA and protein levels and corresponding increases in CYP7A1 enzyme activity and fecal bile acid excretion.  These studies define FGF15 and FGFR4 as components of a gut-liver signaling pathway that synergizes with SHP to regulate bile acid synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMd0UxjYyXL7Vg90H21EOLACvtfcHk0lgOFpILqXcpiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFGktrjM&md5=b206ac5a248d774e26787a4a4f98ffeb</span></div><a href="/servlet/linkout?suffix=cit15b&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2005.09.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2005.09.001%26sid%3Dliteratum%253Aachs%26aulast%3DInagaki%26aufirst%3DT.%26aulast%3DChoi%26aufirst%3DM.%26aulast%3DMoschetta%26aufirst%3DA.%26aulast%3DPeng%26aufirst%3DL.%26aulast%3DCummins%26aufirst%3DC.%2BL.%26aulast%3DMcDonald%26aufirst%3DJ.%2BG.%26aulast%3DLuo%26aufirst%3DG.%26aulast%3DJones%26aufirst%3DS.%2BA.%26aulast%3DGoodwin%26aufirst%3DB.%26aulast%3DRichardson%26aufirst%3DJ.%2BA.%26aulast%3DGerard%26aufirst%3DR.%2BD.%26aulast%3DRepa%26aufirst%3DJ.%2BJ.%26aulast%3DMangelsdorf%26aufirst%3DD.%2BJ.%26aulast%3DKliewer%26aufirst%3DS.%2BA.%26atitle%3DFibroblast%2520growth%2520factor%252015%2520functions%2520as%2520an%2520enterohepatic%2520signal%2520to%2520regulate%2520bile%2520acid%2520homeostasis%26jtitle%3DCell%2520Metab.%26date%3D2005%26volume%3D2%26spage%3D217%26epage%3D225%26doi%3D10.1016%2Fj.cmet.2005.09.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pai, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hotzel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plise, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salphati, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Setchell, K. D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ware, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lauriault, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schutt, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dambach, D.</span></span> <span> </span><span class="NLM_article-title">Antibody-mediated inhibition of fibroblast growth factor 19 results in increased bile acid synthesis and ileal malabsorption of bile acids in cynomolgus monkeys</span>. <i>Toxicol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">446</span>– <span class="NLM_lpage">456</span>, <span class="refDoi"> DOI: 10.1093/toxsci/kfs011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1093%2Ftoxsci%2Fkfs011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=22268002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC38Xks1SitbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2012&pages=446-456&author=R.+Paiauthor=D.+Frenchauthor=N.+Maauthor=K.+Hotzelauthor=E.+Pliseauthor=L.+Salphatiauthor=K.+D.+R.+Setchellauthor=J.+Wareauthor=V.+Lauriaultauthor=L.+Schuttauthor=D.+Hartleyauthor=D.+Dambach&title=Antibody-mediated+inhibition+of+fibroblast+growth+factor+19+results+in+increased+bile+acid+synthesis+and+ileal+malabsorption+of+bile+acids+in+cynomolgus+monkeys&doi=10.1093%2Ftoxsci%2Fkfs011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Antibody-Mediated Inhibition of Fibroblast Growth Factor 19 Results in Increased Bile Acids Synthesis and Ileal Malabsorption of Bile Acids in Cynomolgus Monkeys</span></div><div class="casAuthors">Pai, Rama; French, Dorothy; Ma, Ning; Hotzel, Kathy; Plise, Emile; Salphati, Laurent; Setchell, Kenneth D. R.; Ware, Joseph; Lauriault, Veronique; Schutt, Leah; Hartley, Dylan; Dambach, Donna</div><div class="citationInfo"><span class="NLM_cas:title">Toxicological Sciences</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">446-456</span>CODEN:
                <span class="NLM_cas:coden">TOSCF2</span>;
        ISSN:<span class="NLM_cas:issn">1096-0929</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Fibroblast growth factor 19 (FGF19) represses cholesterol 7α-hydroxylase (Cyp7α1) and inhibits bile acid synthesis in vitro and in vivo.  Previous studies have shown that anti-FGF19 antibody treatment reduces growth of colon tumor xenografts and prevents hepatocellular carcinomas in FGF19 transgenic mice and thus may be a useful cancer target.  In a repeat dose safety study in cynomolgus monkeys, anti-FGF19 treatment (3-100 mg/kg) demonstrated dose-related liver toxicity accompanied by severe diarrhea and low food consumption.  The mechanism of anti-FGF19 toxicity was investigated using in vitro and in vivo approaches.  The authors' results show that anti-FGF19 antibody had no direct cytotoxic effect on monkey hepatocytes.  Anti-FGF19 increased Cyp7α1, as expected, but also increased bile acid efflux transporter gene (bile salt export pump, multidrug resistant protein 2 [MRP2], and MRP3) expression and reduced sodium taurocholate cotransporting polypeptide and org. anion transporter 2 expression in liver tissues from treated monkeys and in primary hepatocytes.  In addn., anti-FGF19 treatment increased solute transporter gene (ileal bile acid-binding protein, org. solute transporter α [OST-α], and OST-β) expression in ileal tissues from treated monkeys but not in Caco-2 cells.  However, deoxycholic acid (a secondary bile acid) increased expression of FGF19 and these solute transporter genes in Caco-2 cells.  Gas chromatog.-mass spectrometry anal. of monkey feces showed an increase in total bile acids and cholic acid derivs.  These findings suggest that high doses of anti-FGF19 increase Cyp7α1 expression and bile acid synthesis and alter the expression of bile transporters in the liver resulting in enhanced bile acid efflux and reduced uptake.  Increased bile acids alter expression of solute transporters in the ileum causing diarrhea and the enhanced enterohepatic recirculation of bile acids leading to liver toxicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqORkYjNAK2bLVg90H21EOLACvtfcHk0ljezRexgZ_RAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xks1SitbY%253D&md5=b70cb6104a023ef79adda0cea34fc8a4</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1093%2Ftoxsci%2Fkfs011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Ftoxsci%252Fkfs011%26sid%3Dliteratum%253Aachs%26aulast%3DPai%26aufirst%3DR.%26aulast%3DFrench%26aufirst%3DD.%26aulast%3DMa%26aufirst%3DN.%26aulast%3DHotzel%26aufirst%3DK.%26aulast%3DPlise%26aufirst%3DE.%26aulast%3DSalphati%26aufirst%3DL.%26aulast%3DSetchell%26aufirst%3DK.%2BD.%2BR.%26aulast%3DWare%26aufirst%3DJ.%26aulast%3DLauriault%26aufirst%3DV.%26aulast%3DSchutt%26aufirst%3DL.%26aulast%3DHartley%26aufirst%3DD.%26aulast%3DDambach%26aufirst%3DD.%26atitle%3DAntibody-mediated%2520inhibition%2520of%2520fibroblast%2520growth%2520factor%252019%2520results%2520in%2520increased%2520bile%2520acid%2520synthesis%2520and%2520ileal%2520malabsorption%2520of%2520bile%2520acids%2520in%2520cynomolgus%2520monkeys%26jtitle%3DToxicol.%2520Sci.%26date%3D2012%26volume%3D126%26spage%3D446%26epage%3D456%26doi%3D10.1093%2Ftoxsci%2Fkfs011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schadt, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahl, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wuersch, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couttet, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwald, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lienard, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emotte, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skuba, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manenti, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graus Porta, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairhurst, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kullak-Ublick, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chibout, S.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pognan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kluwe, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kinyamu-Akunda, J.</span></span> <span> </span><span class="NLM_article-title">Bile acid sequestration by cholestyramine mitigates FGFR4 inhibition-induced ALT elevation</span>. <i>Toxicol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>163</i></span>,  <span class="NLM_fpage">265</span>– <span class="NLM_lpage">278</span>, <span class="refDoi"> DOI: 10.1093/toxsci/kfy031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1093%2Ftoxsci%2Fkfy031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=29432567" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlGksr%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=163&publication_year=2018&pages=265-278&author=H.+S.+Schadtauthor=A.+Wolfauthor=J.+A.+Mahlauthor=K.+Wuerschauthor=P.+Couttetauthor=M.+Schwaldauthor=A.+Fischerauthor=M.+Lienardauthor=C.+Emotteauthor=C.-H.+Tengauthor=E.+Skubaauthor=T.+A.+Richardsonauthor=L.+Manentiauthor=A.+Weissauthor=D.+Graus+Portaauthor=R.+A.+Fairhurstauthor=G.+A.+Kullak-Ublickauthor=S.-D.+Chiboutauthor=F.+Pognanauthor=W.+Kluweauthor=J.+Kinyamu-Akunda&title=Bile+acid+sequestration+by+cholestyramine+mitigates+FGFR4+inhibition-induced+ALT+elevation&doi=10.1093%2Ftoxsci%2Fkfy031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Bile acid sequestration by cholestyramine mitigates FGFR4 inhibition-induced ALT elevation</span></div><div class="casAuthors">Schadt, Heiko S.; Wolf, Armin; Mahl, Joerg Andreas; Wuersch, Kuno; Couttet, Philippe; Schwald, Marianne; Fischer, Audrey; Lienard, Mathilde; Emotte, Corinne; Teng, Chi-Hse; Skuba, Elizabeth; Richardson, Terrilyn A.; Manenti, Luigi; Weiss, Andreas; Porta, Diana Graus; Fairhurst, Robin A.; Kullak-Ublick, Gerd A.; Chibout, Salah-Dine; Pognan, Francois; Kluwe, William; Kinyamu-Akunda, Jacqueline</div><div class="citationInfo"><span class="NLM_cas:title">Toxicological Sciences</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">163</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">265-278</span>CODEN:
                <span class="NLM_cas:coden">TOSCF2</span>;
        ISSN:<span class="NLM_cas:issn">1096-0929</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The FGF19-fibroblast growth factor receptor (FGFR4)-βKlotho (KLB) pathway plays an important role in the regulation of bile acid (BA) homeostasis.  Aberrant activation of this pathway has been described in the development and progression of a subset of liver cancers including hepatocellular carcinoma, establishing FGFR4 as an attractive therapeutic target for such solid tumors.  FGF401 is a highly selective FGFR4 kinase inhibitor being developed for hepatocellular carcinoma, currently in phase I/II clin. studies.  In preclin. studies in mice and dogs, oral administration of FGF401 led to induction of Cyp7a1, elevation of its peripheral marker 7alpha-hydroxy-4-cholesten-3-one, increased BA pool size, decreased serum cholesterol and diarrhea in dogs.  FGF401 was also assocd. with increases of serum aminotransferases, primarily alanine aminotransferase (ALT), in the absence of any observable adverse histopathol. in the liver, or in any other organs.  The authors hypothesized that the increase in ALT could be secondary to increased BAs and conducted an investigative study in dogs with FGF401 and coadministration of the BA sequestrant cholestyramine (CHO).  CHO prevented and reversed FGF401-related increases in ALT in dogs in parallel to its ability to reduce BAs in the circulation.  Correlation anal. showed that FGF401-mediated increases in ALT strongly correlated with increases in taurolithocholic acid and taurodeoxycholic acid, the major secondary BAs in dog plasma, indicating a mechanistic link between ALT elevation and changes in BA pool hydrophobicity.  Thus, CHO may offer the potential to mitigate elevations in serum amtransferases in human subjects that are caused by targeted FGFR4 inhibition and elevated intracellular BA levels.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4tzKAwiqUR7Vg90H21EOLACvtfcHk0ljezRexgZ_RAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlGksr%252FF&md5=e11d415d68d5b1b326a9547d35ef727a</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1093%2Ftoxsci%2Fkfy031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Ftoxsci%252Fkfy031%26sid%3Dliteratum%253Aachs%26aulast%3DSchadt%26aufirst%3DH.%2BS.%26aulast%3DWolf%26aufirst%3DA.%26aulast%3DMahl%26aufirst%3DJ.%2BA.%26aulast%3DWuersch%26aufirst%3DK.%26aulast%3DCouttet%26aufirst%3DP.%26aulast%3DSchwald%26aufirst%3DM.%26aulast%3DFischer%26aufirst%3DA.%26aulast%3DLienard%26aufirst%3DM.%26aulast%3DEmotte%26aufirst%3DC.%26aulast%3DTeng%26aufirst%3DC.-H.%26aulast%3DSkuba%26aufirst%3DE.%26aulast%3DRichardson%26aufirst%3DT.%2BA.%26aulast%3DManenti%26aufirst%3DL.%26aulast%3DWeiss%26aufirst%3DA.%26aulast%3DGraus%2BPorta%26aufirst%3DD.%26aulast%3DFairhurst%26aufirst%3DR.%2BA.%26aulast%3DKullak-Ublick%26aufirst%3DG.%2BA.%26aulast%3DChibout%26aufirst%3DS.-D.%26aulast%3DPognan%26aufirst%3DF.%26aulast%3DKluwe%26aufirst%3DW.%26aulast%3DKinyamu-Akunda%26aufirst%3DJ.%26atitle%3DBile%2520acid%2520sequestration%2520by%2520cholestyramine%2520mitigates%2520FGFR4%2520inhibition-induced%2520ALT%2520elevation%26jtitle%3DToxicol.%2520Sci.%26date%3D2018%26volume%3D163%26spage%3D265%26epage%3D278%26doi%3D10.1093%2Ftoxsci%2Fkfy031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, J.</span></span> <span> </span><span class="NLM_article-title">Systematic review: the management of chronic diarrhoea due to bile acid malabsorption</span>. <i>Aliment. Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">923</span>– <span class="NLM_lpage">939</span>, <span class="refDoi"> DOI: 10.1111/apt.12684</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1111%2Fapt.12684" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=24602022" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC2cXlvFWis7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2014&pages=923-939&author=C.+Wilcoxauthor=J.+Turnerauthor=J.+Green&title=Systematic+review%3A+the+management+of+chronic+diarrhoea+due+to+bile+acid+malabsorption&doi=10.1111%2Fapt.12684"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Systematic review: the management of chronic diarrhoea due to bile acid malabsorption</span></div><div class="casAuthors">Wilcox, C.; Turner, J.; Green, J.</div><div class="citationInfo"><span class="NLM_cas:title">Alimentary Pharmacology and Therapeutics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">923-939</span>CODEN:
                <span class="NLM_cas:coden">APTHEN</span>;
        ISSN:<span class="NLM_cas:issn">0269-2813</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Summary : Background : Bile acid malabsorption (BAM) is a common, yet under-recognized, cause of chronic diarrhea, with limited guidance available on the appropriate management of patients with BAM.  Aim : To summarize the evidence supporting different treatments available for patients with bile acid malabsorption, noting their impact on clin. outcomes, tolerability and assocd. side effects.  Methods : A literature search was conducted through PubMed, the Cochrane Database of Systematic Reviews and Scopus.  Relevant articles studied patients who had been diagnosed with BAM and were clin. assessed before and after therapy.  Results : A total of 30 relevant publications (1241 adult patients) were identified, which investigated the clin. response to drugs, including colestyramine, colestipol, colesevelam, aluminum hydroxide and obeticholic acid.  The most commonly used diagnostic test of bile acid malabsorption was the SeHCAT test (24 studies).  Colestyramine treatment was by far the most studied of these agents, and was successful in 70% of 801 patients (range: 63-100%).  Conclusions : Colestyramine and colestipol are generally effective treatments of gastrointestinal symptoms from BAM, but may be poorly tolerated and reduce the bioavailability of co-administered agents.  Alternative therapies (including colesevelam and aluminum hydroxide) as well as dietary intervention may also have a role, and the promising results of the first proof-of-concept study of obeticholic acid suggest that its novel approach may have an exciting future in the treatment of this condition.  Future trials should employ accurate diagnostic testing and be conducted over longer periods so that the long-term benefits and tolerability of these different approaches can be evaluated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq17n0aYFw-kLVg90H21EOLACvtfcHk0lgWD6QJzPECtg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXlvFWis7w%253D&md5=b4d6ad463b6544420028dd22300ad292</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1111%2Fapt.12684&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fapt.12684%26sid%3Dliteratum%253Aachs%26aulast%3DWilcox%26aufirst%3DC.%26aulast%3DTurner%26aufirst%3DJ.%26aulast%3DGreen%26aufirst%3DJ.%26atitle%3DSystematic%2520review%253A%2520the%2520management%2520of%2520chronic%2520diarrhoea%2520due%2520to%2520bile%2520acid%2520malabsorption%26jtitle%3DAliment.%2520Pharmacol.%2520Ther.%26date%3D2014%26volume%3D39%26spage%3D923%26epage%3D939%26doi%3D10.1111%2Fapt.12684" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fairhurst, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knoepfel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leblanc, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buschmann, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaul, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blank, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galuba, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trappe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voshol, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Genick, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunet-Lefeuvre, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bitsch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graus-Porta, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furet, P.</span></span> <span> </span><span class="NLM_article-title">Approaches to selective fibroblast growth factor receptor 4 inhibition through targeting the ATP-pocket middle-hinge region</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1604</span>– <span class="NLM_lpage">1613</span>, <span class="refDoi"> DOI: 10.1039/C7MD00213K</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1039%2FC7MD00213K" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=30108871" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC2sXps1GlsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1604-1613&author=R.+A.+Fairhurstauthor=T.+Knoepfelauthor=C.+Leblancauthor=N.+Buschmannauthor=C.+Gaulauthor=J.+Blankauthor=I.+Galubaauthor=J.+Trappeauthor=C.+Zouauthor=J.+Vosholauthor=C.+Genickauthor=P.+Brunet-Lefeuvreauthor=F.+Bitschauthor=D.+Graus-Portaauthor=P.+Furet&title=Approaches+to+selective+fibroblast+growth+factor+receptor+4+inhibition+through+targeting+the+ATP-pocket+middle-hinge+region&doi=10.1039%2FC7MD00213K"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Approaches to selective fibroblast growth factor receptor 4 inhibition through targeting the ATP-pocket middle-hinge region</span></div><div class="casAuthors">Fairhurst, Robin A.; Knoepfel, Thomas; Leblanc, Catherine; Buschmann, Nicole; Gaul, Christoph; Blank, Jutta; Galuba, Inga; Trappe, Jorg; Zou, Chao; Voshol, Johannes; Genick, Christine; Brunet-Lefeuvre, Peggy; Bitsch, Francis; Graus-Porta, Diana; Furet, Pascal</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1604-1613</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A diverse range of selective FGFR4 inhibitor hit series were identified using unbiased screening approaches and by the modification of known kinase inhibitor scaffolds.  In each case the origin of the selectivity was consistent with an interaction with a poorly conserved cysteine residue within the middle-hinge region of the kinase domain of FGFR4, at position 552.  Targeting this region identified a non-covalent diaminopyrimidine series differentiating by size, an irreversible-covalent inhibitor in which Cys552 undergoes an SNAr reaction with a 2-chloropyridine, and a reversible-covalent inhibitor series in which Cys552 forms a hemithioacetal adduct with a 2-formyl naphthalene.  In addn., the introduction of an acrylamide into a known FGFR scaffold identified a pan-FGFR inhibitor which reacted with both Cys552 and a second poorly conserved cysteine on the P-loop of FGFR4 at position 477 which is present in all four FGFR family members.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwzyJE17BWsLVg90H21EOLACvtfcHk0lgWD6QJzPECtg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXps1GlsL8%253D&md5=04f1aff88385b915e2672509dd3f6954</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1039%2FC7MD00213K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7MD00213K%26sid%3Dliteratum%253Aachs%26aulast%3DFairhurst%26aufirst%3DR.%2BA.%26aulast%3DKnoepfel%26aufirst%3DT.%26aulast%3DLeblanc%26aufirst%3DC.%26aulast%3DBuschmann%26aufirst%3DN.%26aulast%3DGaul%26aufirst%3DC.%26aulast%3DBlank%26aufirst%3DJ.%26aulast%3DGaluba%26aufirst%3DI.%26aulast%3DTrappe%26aufirst%3DJ.%26aulast%3DZou%26aufirst%3DC.%26aulast%3DVoshol%26aufirst%3DJ.%26aulast%3DGenick%26aufirst%3DC.%26aulast%3DBrunet-Lefeuvre%26aufirst%3DP.%26aulast%3DBitsch%26aufirst%3DF.%26aulast%3DGraus-Porta%26aufirst%3DD.%26aulast%3DFuret%26aufirst%3DP.%26atitle%3DApproaches%2520to%2520selective%2520fibroblast%2520growth%2520factor%2520receptor%25204%2520inhibition%2520through%2520targeting%2520the%2520ATP-pocket%2520middle-hinge%2520region%26jtitle%3DMedChemComm%26date%3D2017%26volume%3D8%26spage%3D1604%26epage%3D1613%26doi%3D10.1039%2FC7MD00213K" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leproult, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barluenga, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moras, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wurtz, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winssinger, N.</span></span> <span> </span><span class="NLM_article-title">Cysteine mapping in conformationally distinct kinase nucleotide binding sites: application to the design of selective covalent inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">1347</span>– <span class="NLM_lpage">1355</span>, <span class="refDoi"> DOI: 10.1021/jm101396q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm101396q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhvFeis70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=1347-1355&author=E.+Leproultauthor=S.+Barluengaauthor=D.+Morasauthor=J.-M.+Wurtzauthor=N.+Winssinger&title=Cysteine+mapping+in+conformationally+distinct+kinase+nucleotide+binding+sites%3A+application+to+the+design+of+selective+covalent+inhibitors&doi=10.1021%2Fjm101396q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Cysteine mapping in conformationally distinct kinase nucleotide binding sites: application to the design of selective covalent inhibitors</span></div><div class="casAuthors">Leproult, Emeline; Barluenga, Sofia; Moras, Dino; Wurtz, Jean-Marie; Winssinger, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1347-1355</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Kinases have emerged as one of the most prolific therapeutic targets.  An important criterion in the therapeutic success of inhibitors targeting the nucleotide binding pocket of kinases is the inhibitor residence time.  Recently, covalent kinase inhibitors have attracted attention since they confer terminal inhibition and should thus be more effective than reversible inhibitors with transient inhibition.  The most robust approach to design irreversible inhibitors is to capitalize on the nucleophilicity of a cysteine thiol group present in the target protein.  Herein, the authors report a systematic anal. of cysteine residues present in the nucleotide binding site of kinases, which could be harnessed for irreversible inhibition, taking into consideration the different kinase conformations.  The authors demonstrate the predictive power of this anal. with the design and validation of an irreversible inhibitor of KIT/PDGFR kinases.  This is the first example of a covalent kinase inhibitor that combines a pharmacophore addressing the DFG-out conformation with a covalent trap.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkzqt0gMrz3rVg90H21EOLACvtfcHk0lgWD6QJzPECtg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhvFeis70%253D&md5=329ae5eaa6102b6dd34a0d1274631247</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm101396q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm101396q%26sid%3Dliteratum%253Aachs%26aulast%3DLeproult%26aufirst%3DE.%26aulast%3DBarluenga%26aufirst%3DS.%26aulast%3DMoras%26aufirst%3DD.%26aulast%3DWurtz%26aufirst%3DJ.-M.%26aulast%3DWinssinger%26aufirst%3DN.%26atitle%3DCysteine%2520mapping%2520in%2520conformationally%2520distinct%2520kinase%2520nucleotide%2520binding%2520sites%253A%2520application%2520to%2520the%2520design%2520of%2520selective%2520covalent%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D1347%26epage%3D1355%26doi%3D10.1021%2Fjm101396q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yau, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macarulla, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, J.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choo, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollebecque, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sung, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, H.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazzaferro, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trojan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, A. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoon, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Z.-Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faivre, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feun, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yen, C.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dufour, J.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Llovet, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manoogian, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tugnait, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stransky, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohl, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lengauer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherwin, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt-Kittler, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoeflich, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, B. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, Y.-K.</span></span> <span> </span><span class="NLM_article-title">First-in-human phase I study of fisogatinib (BLU-554) validates aberrant FGF19 signaling as a driver event in hepatocellular carcinoma</span>. <i>Cancer Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1696</span>– <span class="NLM_lpage">1707</span>, <span class="refDoi"> DOI: 10.1158/2159-8290.CD-19-0555</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1158%2F2159-8290.CD-19-0555" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=31575541" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1Ght77L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=1696-1707&author=R.+D.+Kimauthor=D.+Sarkerauthor=T.+Meyerauthor=T.+Yauauthor=T.+Macarullaauthor=J.-W.+Parkauthor=S.+P.+Chooauthor=A.+Hollebecqueauthor=M.+W.+Sungauthor=H.-Y.+Limauthor=V.+Mazzaferroauthor=J.+Trojanauthor=A.+X.+Zhuauthor=J.-H.+Yoonauthor=S.+Sharmaauthor=Z.-Z.+Linauthor=S.+L.+Chanauthor=S.+Faivreauthor=L.+G.+Feunauthor=C.-J.+Yenauthor=J.-F.+Dufourauthor=D.+H.+Palmerauthor=J.+M.+Llovetauthor=M.+Manoogianauthor=M.+Tugnaitauthor=N.+Stranskyauthor=M.+Hagelauthor=N.+E.+Kohlauthor=C.+Lengauerauthor=C.+A.+Sherwinauthor=O.+Schmidt-Kittlerauthor=K.+P.+Hoeflichauthor=H.+Shiauthor=B.+B.+Wolfauthor=Y.-K.+Kang&title=First-in-human+phase+I+study+of+fisogatinib+%28BLU-554%29+validates+aberrant+FGF19+signaling+as+a+driver+event+in+hepatocellular+carcinoma&doi=10.1158%2F2159-8290.CD-19-0555"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">First-in-human phase I study of fisogatinib (BLU-554) validates aberrant FGF19 signaling as a driver event in hepatocellular carcinoma</span></div><div class="casAuthors">Kim, Richard D.; Sarker, Debashis; Meyer, Tim; Yau, Thomas; Macarulla, Teresa; Park, Joong-Won; Choo, Su Pin; Hollebecque, Antoine; Sung, Max W.; Lim, Ho-Yeong; Mazzaferro, Vincenzo; Trojan, Joerg; Zhu, Andrew X.; Yoon, Jung-Hwan; Sharma, Sunil; Lin, Zhong-Zhe; Chan, Stephen L.; Faivre, Sandrine; Feun, Lynn G.; Yen, Chia-Jui; Dufour, Jean-Francois; Palmer, Daniel H.; Llovet, Josep M.; Manoogian, Melissa; Tugnait, Meera; Stransky, Nicolas; Hagel, Margit; Kohl, Nancy E.; Lengauer, Christoph; Sherwin, Cori Ann; Schmidt-Kittler, Oleg; Hoeflich, Klaus P.; Shi, Hongliang; Wolf, Beni B.; Kang, Yoon-Koo</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1696-1707</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Outcomes for patients with advanced hepatocellular carcinoma (HCC) remain poor despite recent progress in drug development.  Emerging data implicate FGF19 as a potential HCC driver, suggesting its receptor, FGFR4, as a novel therapeutic target.  We evaluated fisogatinib (BLU-554), a highly potent and selective oral FGFR4 inhibitor, in a phase I dose-escalation/dose-expansion study in advanced HCC using FGF19 expression measured by IHC as a biomarker for pathway activation.  For dose escalation, 25 patients received 140 to 900 mg fisogatinib once daily; the max. tolerated dose (600 mg once daily) was expanded in 81 patients.  Fisogatinib was well tolerated; most adverse events were manageable, grade 1/2 gastrointestinal events, primarily diarrhea, nausea, and vomiting.  Across doses, the overall response rate was 17% in FGF19-pos. patients [median duration of response: 5.3 mo (95% CI, 3.7-not reached)] and 0% in FGF19-neg. patients.  These results validate FGFR4 as a targetable driver in FGF19-pos. advanced HCC.  SIGNIFICANCE: Fisogatinib elicited clin. responses in patients with tumor FGF19 overexpression in advanced HCC.  These results validate the oncogenic driver role of the FGFR4 pathway in HCC and the use of FGF19 as a biomarker for patient selection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpthlOsQ713p7Vg90H21EOLACvtfcHk0lh-5bSuYgBeMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1Ght77L&md5=84a5d6ca43e37a9caad36f513eb79438</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-19-0555&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-19-0555%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DR.%2BD.%26aulast%3DSarker%26aufirst%3DD.%26aulast%3DMeyer%26aufirst%3DT.%26aulast%3DYau%26aufirst%3DT.%26aulast%3DMacarulla%26aufirst%3DT.%26aulast%3DPark%26aufirst%3DJ.-W.%26aulast%3DChoo%26aufirst%3DS.%2BP.%26aulast%3DHollebecque%26aufirst%3DA.%26aulast%3DSung%26aufirst%3DM.%2BW.%26aulast%3DLim%26aufirst%3DH.-Y.%26aulast%3DMazzaferro%26aufirst%3DV.%26aulast%3DTrojan%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DA.%2BX.%26aulast%3DYoon%26aufirst%3DJ.-H.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DLin%26aufirst%3DZ.-Z.%26aulast%3DChan%26aufirst%3DS.%2BL.%26aulast%3DFaivre%26aufirst%3DS.%26aulast%3DFeun%26aufirst%3DL.%2BG.%26aulast%3DYen%26aufirst%3DC.-J.%26aulast%3DDufour%26aufirst%3DJ.-F.%26aulast%3DPalmer%26aufirst%3DD.%2BH.%26aulast%3DLlovet%26aufirst%3DJ.%2BM.%26aulast%3DManoogian%26aufirst%3DM.%26aulast%3DTugnait%26aufirst%3DM.%26aulast%3DStransky%26aufirst%3DN.%26aulast%3DHagel%26aufirst%3DM.%26aulast%3DKohl%26aufirst%3DN.%2BE.%26aulast%3DLengauer%26aufirst%3DC.%26aulast%3DSherwin%26aufirst%3DC.%2BA.%26aulast%3DSchmidt-Kittler%26aufirst%3DO.%26aulast%3DHoeflich%26aufirst%3DK.%2BP.%26aulast%3DShi%26aufirst%3DH.%26aulast%3DWolf%26aufirst%3DB.%2BB.%26aulast%3DKang%26aufirst%3DY.-K.%26atitle%3DFirst-in-human%2520phase%2520I%2520study%2520of%2520fisogatinib%2520%2528BLU-554%2529%2520validates%2520aberrant%2520FGF19%2520signaling%2520as%2520a%2520driver%2520event%2520in%2520hepatocellular%2520carcinoma%26jtitle%3DCancer%2520Discovery%26date%3D2019%26volume%3D9%26spage%3D1696%26epage%3D1707%26doi%3D10.1158%2F2159-8290.CD-19-0555" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coffey, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corcoran, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichikawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prajapati, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rimkunas, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karr, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramanian, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKenzie, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Satoh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioux, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buonamici, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fekkes, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warmuth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reynolds, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Selvaraj, A.</span></span> <span> </span><span class="NLM_article-title">H3B-6527 Is a potent and selective inhibitor of FGFR4 in FGF19-Driven hepatocellular carcinoma</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">6999</span>– <span class="NLM_lpage">7013</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-17-1865</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1158%2F0008-5472.CAN-17-1865" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=29247039" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFOgur%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2017&pages=6999-7013&author=J.+J.+Joshiauthor=H.+Coffeyauthor=E.+Corcoranauthor=J.+Tsaiauthor=C.+L.+Huangauthor=K.+Ichikawaauthor=S.+Prajapatiauthor=M.+H.+Haoauthor=S.+Baileyauthor=J.+Wuauthor=V.+Rimkunasauthor=C.+Karrauthor=V.+Subramanianauthor=P.+Kumarauthor=C.+MacKenzieauthor=R.+Hurleyauthor=T.+Satohauthor=K.+Yuauthor=E.+Parkauthor=N.+Riouxauthor=A.+Kimauthor=W.+G.+Laiauthor=L.+Yuauthor=P.+Zhuauthor=S.+Buonamiciauthor=N.+Larsenauthor=P.+Fekkesauthor=J.+Wangauthor=M.+Warmuthauthor=D.+J.+Reynoldsauthor=P.+G.+Smithauthor=A.+Selvaraj&title=H3B-6527+Is+a+potent+and+selective+inhibitor+of+FGFR4+in+FGF19-Driven+hepatocellular+carcinoma&doi=10.1158%2F0008-5472.CAN-17-1865"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">H3B-6527 Is a Potent and Selective Inhibitor of FGFR4 in FGF19-Driven Hepatocellular Carcinoma</span></div><div class="casAuthors">Joshi, Jaya Julie; Coffey, Heather; Corcoran, Erik; Tsai, Jennifer; Huang, Chia-Ling; Ichikawa, Kana; Prajapati, Sudeep; Hao, Ming-Hong; Bailey, Suzanna; Wu, Jeremy; Rimkunas, Victoria; Karr, Craig; Subramanian, Vanitha; Kumar, Pavan; MacKenzie, Crystal; Hurley, Raelene; Satoh, Takashi; Yu, Kun; Park, Eunice; Rioux, Nathalie; Kim, Amy; Lai, Weidong G.; Yu, Lihua; Zhu, Ping; Buonamici, Silvia; Larsen, Nicholas; Fekkes, Peter; Wang, John; Warmuth, Markus; Reynolds, Dominic J.; Smith, Peter G.; Selvaraj, Anand</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">6999-7013</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Activation of the fibroblast growth factor receptor FGFR4 by FGF19 drives hepatocellular carcinoma (HCC), a disease with few, if any, effective treatment options.  While a no. of pan-FGFR inhibitors are being clin. evaluated, their application to FGF19-driven HCC may be limited by dose-limiting toxicities mediated by FGFR1-3 receptors.  To evade the potential limitations of pan-FGFR inhibitors, we generated H3B-6527, a highly selective covalent FGFR4 inhibitor, through structure-guided drug design.  Studies in a panel of 40 HCC cell lines and 30 HCC PDX models showed that FGF19 expression is a predictive biomarker for H3B-6527 response.  Moreover, coadministration of the CDK4/6 inhibitor palbociclib in combination with H3B-6527 could effectively trigger tumor regression in a xenograft model of HCC.  Overall, our results offer preclin. proof of concept for H3B-6527 as a candidate therapeutic agent for HCC cases that exhibit increased expression of FGF19.  Cancer Res; 77(24); 6999-7013. ©2017 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohONN1gEpft7Vg90H21EOLACvtfcHk0lh-5bSuYgBeMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFOgur%252FI&md5=9d7ea6245e3b047aa85fc6961a88e5eb</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-17-1865&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-17-1865%26sid%3Dliteratum%253Aachs%26aulast%3DJoshi%26aufirst%3DJ.%2BJ.%26aulast%3DCoffey%26aufirst%3DH.%26aulast%3DCorcoran%26aufirst%3DE.%26aulast%3DTsai%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DC.%2BL.%26aulast%3DIchikawa%26aufirst%3DK.%26aulast%3DPrajapati%26aufirst%3DS.%26aulast%3DHao%26aufirst%3DM.%2BH.%26aulast%3DBailey%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DRimkunas%26aufirst%3DV.%26aulast%3DKarr%26aufirst%3DC.%26aulast%3DSubramanian%26aufirst%3DV.%26aulast%3DKumar%26aufirst%3DP.%26aulast%3DMacKenzie%26aufirst%3DC.%26aulast%3DHurley%26aufirst%3DR.%26aulast%3DSatoh%26aufirst%3DT.%26aulast%3DYu%26aufirst%3DK.%26aulast%3DPark%26aufirst%3DE.%26aulast%3DRioux%26aufirst%3DN.%26aulast%3DKim%26aufirst%3DA.%26aulast%3DLai%26aufirst%3DW.%2BG.%26aulast%3DYu%26aufirst%3DL.%26aulast%3DZhu%26aufirst%3DP.%26aulast%3DBuonamici%26aufirst%3DS.%26aulast%3DLarsen%26aufirst%3DN.%26aulast%3DFekkes%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWarmuth%26aufirst%3DM.%26aulast%3DReynolds%26aufirst%3DD.%2BJ.%26aulast%3DSmith%26aufirst%3DP.%2BG.%26aulast%3DSelvaraj%26aufirst%3DA.%26atitle%3DH3B-6527%2520Is%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520FGFR4%2520in%2520FGF19-Driven%2520hepatocellular%2520carcinoma%26jtitle%3DCancer%2520Res.%26date%3D2017%26volume%3D77%26spage%3D6999%26epage%3D7013%26doi%3D10.1158%2F0008-5472.CAN-17-1865" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patterson, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smaill, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, K.</span></span> <span> </span><span class="NLM_article-title">Fibroblast growth factor receptor 4 (FGFR4) selective inhibitors as hepatocellular carcinoma therapy: advances and prospects</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">2905</span>– <span class="NLM_lpage">2915</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01531</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01531" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFagur3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=2905-2915&author=X.+Luauthor=H.+Chenauthor=A.+V.+Pattersonauthor=J.+B.+Smaillauthor=K.+Ding&title=Fibroblast+growth+factor+receptor+4+%28FGFR4%29+selective+inhibitors+as+hepatocellular+carcinoma+therapy%3A+advances+and+prospects&doi=10.1021%2Facs.jmedchem.8b01531"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Fibroblast Growth Factor Receptor 4 (FGFR4) Selective Inhibitors as Hepatocellular Carcinoma Therapy: Advances and Prospects</span></div><div class="casAuthors">Lu, Xiaoyun; Chen, Hao; Patterson, Adam V.; Smaill, Jeff B.; Ding, Ke</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2905-2915</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Hepatocellular carcinoma (HCC) is a lethal disease with limited therapeutic options and a particularly poor prognosis.  Aberrant fibroblast growth factor 19 (FGF19) signaling through fibroblast growth factor receptor 4 (FGFR4) has been identified as an oncogenic driver for a subset of patients with HCC.  FGFR4 is therefore a promising target for the treatment of HCC harboring aberrant FGF19-FGFR4 signaling, and several FGFR4 inhibitors have advanced to clin. trial.  In this review, the authors summarize the latest developments in FGFR4 inhibitors, including the known pharmacophores, their binding mode, selectivity, and clin. implications, as well as the optimization strategy of introducing an acrylamide into a known pan-FGFR inhibitor targeting Cys552 of FGFR4 to provide selective covalent FGFR4 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6Ev_KhmOEBbVg90H21EOLACvtfcHk0lh2SW34VVVnTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFagur3M&md5=9c1c111108e0ec7f33bca7b9227c246a</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01531&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01531%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DPatterson%26aufirst%3DA.%2BV.%26aulast%3DSmaill%26aufirst%3DJ.%2BB.%26aulast%3DDing%26aufirst%3DK.%26atitle%3DFibroblast%2520growth%2520factor%2520receptor%25204%2520%2528FGFR4%2529%2520selective%2520inhibitors%2520as%2520hepatocellular%2520carcinoma%2520therapy%253A%2520advances%2520and%2520prospects%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D2905%26epage%3D2915%26doi%3D10.1021%2Facs.jmedchem.8b01531" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pike, K. G.</span>; <span class="NLM_string-name">Buttar, D.</span>; <span class="NLM_string-name">Tucker, J.</span>; <span class="NLM_string-name">Jones, C.</span></span> <span> </span><span class="NLM_article-title">Identifying Selective Inhibitors of FGFR4 Kinase</span>. Presented at the 23rd International Symposium on Medicinal Chemistry, Lisbon, Sep 7–11, <span class="NLM_year">2014</span>, European Federation for Medicinal Chemistry, LE098.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=K.+G.+Pike&author=D.+Buttar&author=J.+Tucker&author=C.+Jones&title=Identifying+Selective+Inhibitors+of+FGFR4+Kinase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DPike%26aufirst%3DK.%2BG.%26atitle%3DIdentifying%2520Selective%2520Inhibitors%2520of%2520FGFR4%2520Kinase%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><a href="https://clinicaltrials.gov/ct2/show/NCT02476019" class="extLink">https://clinicaltrials.gov/ct2/show/NCT02476019</a> (accessed Sep 14, <span class="NLM_year">2020</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=https%3A%2F%2Fclinicaltrials.gov%2Fct2%2Fshow%2FNCT02476019+%28accessed+Sep+14%2C+2020%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gavine, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mooney, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kilgour, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Kadhimi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rooney, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mellor, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinowska, T.</span></span> <span> </span><span class="NLM_article-title">AZD4547: An orally bioavailable, potent, and selective inhibitor of the fibroblast growth factor receptor tyrosine kinase family</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">2045</span>– <span class="NLM_lpage">2056</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-11-3034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1158%2F0008-5472.CAN-11-3034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=22369928" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC38Xls1Wgsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2012&pages=2045-2056&author=P.+R.+Gavineauthor=L.+Mooneyauthor=E.+Kilgourauthor=A.+P.+Thomasauthor=K.+Al-Kadhimiauthor=S.+Beckauthor=C.+Rooneyauthor=T.+Colemanauthor=D.+Bakerauthor=M.+J.+Mellorauthor=A.+N.+Brooksauthor=T.+Klinowska&title=AZD4547%3A+An+orally+bioavailable%2C+potent%2C+and+selective+inhibitor+of+the+fibroblast+growth+factor+receptor+tyrosine+kinase+family&doi=10.1158%2F0008-5472.CAN-11-3034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">AZD4547: An Orally Bioavailable, Potent, and Selective Inhibitor of the Fibroblast Growth Factor Receptor Tyrosine Kinase Family</span></div><div class="casAuthors">Gavine, Paul R.; Mooney, Lorraine; Kilgour, Elaine; Thomas, Andrew P.; Al-Kadhimi, Katherine; Beck, Sarah; Rooney, Claire; Coleman, Tanya; Baker, Dawn; Mellor, Martine J.; Brooks, A. Nigel; Klinowska, Teresa</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2045-2056</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The fibroblast growth factor (FGF) signaling axis is increasingly implicated in tumorigenesis and chemoresistance.  Several small-mol. FGF receptor (FGFR) kinase inhibitors are currently in clin. development; however, the predominant activity of the most advanced of these agents is against the kinase insert domain receptor (KDR), which compromises the FGFR selectivity.  Here, we report the pharmacol. profile of AZD4547, a novel and selective inhibitor of the FGFR1, 2, and 3 tyrosine kinases.  AZD4547 inhibited recombinant FGFR kinase activity in vitro and suppressed FGFR signaling and growth in tumor cell lines with deregulated FGFR expression.  In a representative FGFR-driven human tumor xenograft model, oral administration of AZD4547 was well tolerated and resulted in potent dose-dependent antitumor activity, consistent with plasma exposure and pharmacodynamic modulation of tumor FGFR.  Importantly, at efficacious doses, no evidence of anti-KDR-related effects were obsd., confirming the in vivo FGFR selectivity of AZD4547.  Taken together, our findings show that AZD4547 is a novel selective small-mol. inhibitor of FGFR with potent antitumor activity against FGFR-deregulated tumors in preclin. models.  AZD4547 is under clin. investigation for the treatment of FGFR-dependent tumors.  Cancer Res; 72(8); 2045-56.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnpExryJc9xbVg90H21EOLACvtfcHk0lh2SW34VVVnTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xls1Wgsrs%253D&md5=cba90ad8f9b9e51cd10288868bbcc85e</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-11-3034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-11-3034%26sid%3Dliteratum%253Aachs%26aulast%3DGavine%26aufirst%3DP.%2BR.%26aulast%3DMooney%26aufirst%3DL.%26aulast%3DKilgour%26aufirst%3DE.%26aulast%3DThomas%26aufirst%3DA.%2BP.%26aulast%3DAl-Kadhimi%26aufirst%3DK.%26aulast%3DBeck%26aufirst%3DS.%26aulast%3DRooney%26aufirst%3DC.%26aulast%3DColeman%26aufirst%3DT.%26aulast%3DBaker%26aufirst%3DD.%26aulast%3DMellor%26aufirst%3DM.%2BJ.%26aulast%3DBrooks%26aufirst%3DA.%2BN.%26aulast%3DKlinowska%26aufirst%3DT.%26atitle%3DAZD4547%253A%2520An%2520orally%2520bioavailable%252C%2520potent%252C%2520and%2520selective%2520inhibitor%2520of%2520the%2520fibroblast%2520growth%2520factor%2520receptor%2520tyrosine%2520kinase%2520family%26jtitle%3DCancer%2520Res.%26date%3D2012%26volume%3D72%26spage%3D2045%26epage%3D2056%26doi%3D10.1158%2F0008-5472.CAN-11-3034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yosaatmadja, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalyukina, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Middleditch, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patterson, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smaill, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Squire, C. J.</span></span> <span> </span><span class="NLM_article-title">Rotational freedom, steric hindrance, and protein dynamics explain BLU554 selectivity for the hinge cysteine of FGFR4</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1180</span>– <span class="NLM_lpage">1186</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00196</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00196" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlams7vN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1180-1186&author=X.+Linauthor=Y.+Yosaatmadjaauthor=M.+Kalyukinaauthor=M.+J.+Middleditchauthor=Z.+Zhangauthor=X.+Luauthor=K.+Dingauthor=A.+V.+Pattersonauthor=J.+B.+Smaillauthor=C.+J.+Squire&title=Rotational+freedom%2C+steric+hindrance%2C+and+protein+dynamics+explain+BLU554+selectivity+for+the+hinge+cysteine+of+FGFR4&doi=10.1021%2Facsmedchemlett.9b00196"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Rotational Freedom, Steric Hindrance, and Protein Dynamics Explain BLU554 Selectivity for the Hinge Cysteine of FGFR4</span></div><div class="casAuthors">Lin, Xiaojing; Yosaatmadja, Yuliana; Kalyukina, Maria; Middleditch, Martin J.; Zhang, Zhen; Lu, Xiaoyun; Ding, Ke; Patterson, Adam V.; Smaill, Jeff B.; Squire, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1180-1186</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Aberration in FGFR4 signaling drives carcinogenesis and progression in a subset of hepatocellular carcinoma (HCC) patients, thereby making FGFR4 an attractive mol. target for this disease.  Selective FGFR4 inhibition can be achieved through covalently targeting a poorly conserved cysteine residue in the FGFR4 kinase domain.  We report mass spectrometry assays and cocrystal structures of FGFR4 in covalent complex with the clin. candidate BLU554 and with a series of four structurally related inhibitors that define the inherent reactivity and selectivity profile of these mols.  We further reveal the structure of FGFR1 with one of our inhibitors and show that off-target covalent binding can occur through an alternative conformation that supports targeting of a cysteine conserved in all members of the FGFR family.  Collectively, we propose that rotational freedom, steric hindrance, and protein dynamics explain the exceptional selectivity profile of BLU554 for targeting FGFR4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqag7zwpLVc7bVg90H21EOLACvtfcHk0liypBP4sJutnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlams7vN&md5=43f32bb16bbb3dd3873f8d7a16730a3e</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00196&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00196%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DX.%26aulast%3DYosaatmadja%26aufirst%3DY.%26aulast%3DKalyukina%26aufirst%3DM.%26aulast%3DMiddleditch%26aufirst%3DM.%2BJ.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DDing%26aufirst%3DK.%26aulast%3DPatterson%26aufirst%3DA.%2BV.%26aulast%3DSmaill%26aufirst%3DJ.%2BB.%26aulast%3DSquire%26aufirst%3DC.%2BJ.%26atitle%3DRotational%2520freedom%252C%2520steric%2520hindrance%252C%2520and%2520protein%2520dynamics%2520explain%2520BLU554%2520selectivity%2520for%2520the%2520hinge%2520cysteine%2520of%2520FGFR4%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D10%26spage%3D1180%26epage%3D1186%26doi%3D10.1021%2Facsmedchemlett.9b00196" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Knoepfel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furet, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mah, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buschmann, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leblanc, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ripoche, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graus-Porta, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wartmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galuba, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairhurst, R. A.</span></span> <span> </span><span class="NLM_article-title">2-Formylpyridyl ureas as highly selective reversible-covalent inhibitors of fibroblast growth factor receptor 4</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">215</span>– <span class="NLM_lpage">220</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.7b00485</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.7b00485" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1Srsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=215-220&author=T.+Knoepfelauthor=P.+Furetauthor=R.+Mahauthor=N.+Buschmannauthor=C.+Leblancauthor=S.+Ripocheauthor=D.+Graus-Portaauthor=M.+Wartmannauthor=I.+Galubaauthor=R.+A.+Fairhurst&title=2-Formylpyridyl+ureas+as+highly+selective+reversible-covalent+inhibitors+of+fibroblast+growth+factor+receptor+4&doi=10.1021%2Facsmedchemlett.7b00485"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">2-Formylpyridyl Ureas as Highly Selective Reversible-Covalent Inhibitors of Fibroblast Growth Factor Receptor 4</span></div><div class="casAuthors">Knoepfel, Thomas; Furet, Pascal; Mah, Robert; Buschmann, Nicole; Leblanc, Catherine; Ripoche, Sebastien; Graus-Porta, Diana; Wartmann, Markus; Galuba, Inga; Fairhurst, Robin A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">215-220</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">As part of a project to identify FGFR4 selective inhibitors, scaffold morphing of a 2-formylquinoline amide hit identified series of 2-formylpyridine ureas (2-FPUs) with improved potency and physicochem. properties.  In particular, tetrahydronaphthyridine urea analogs with cellular activities below 30 nM have been identified.  Consistent with the hypothesized reversible-covalent mechanism of inhibition, the 2-FPUs exhibited slow binding kinetics, and the aldehyde, as the putative electrophile, could be demonstrated to be a key structural element for activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpA1PRltYPuWrVg90H21EOLACvtfcHk0liypBP4sJutnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit1Srsb4%253D&md5=fd43811b8a9e5f7a02f27096569423f8</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.7b00485&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.7b00485%26sid%3Dliteratum%253Aachs%26aulast%3DKnoepfel%26aufirst%3DT.%26aulast%3DFuret%26aufirst%3DP.%26aulast%3DMah%26aufirst%3DR.%26aulast%3DBuschmann%26aufirst%3DN.%26aulast%3DLeblanc%26aufirst%3DC.%26aulast%3DRipoche%26aufirst%3DS.%26aulast%3DGraus-Porta%26aufirst%3DD.%26aulast%3DWartmann%26aufirst%3DM.%26aulast%3DGaluba%26aufirst%3DI.%26aulast%3DFairhurst%26aufirst%3DR.%2BA.%26atitle%3D2-Formylpyridyl%2520ureas%2520as%2520highly%2520selective%2520reversible-covalent%2520inhibitors%2520of%2520fibroblast%2520growth%2520factor%2520receptor%25204%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D9%26spage%3D215%26epage%3D220%26doi%3D10.1021%2Facsmedchemlett.7b00485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gehringer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laufer, S. A.</span></span> <span> </span><span class="NLM_article-title">Emerging and re-emerging warheads for targeted covalent inhibitors: applications in medicinal chemistry and chemical biology</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">5673</span>– <span class="NLM_lpage">5724</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01153</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01153" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFKiur%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=5673-5724&author=M.+Gehringerauthor=S.+A.+Laufer&title=Emerging+and+re-emerging+warheads+for+targeted+covalent+inhibitors%3A+applications+in+medicinal+chemistry+and+chemical+biology&doi=10.1021%2Facs.jmedchem.8b01153"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging and Re-Emerging Warheads for Targeted Covalent Inhibitors: Applications in Medicinal Chemistry and Chemical Biology</span></div><div class="casAuthors">Gehringer, Matthias; Laufer, Stefan A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5673-5724</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Targeted covalent inhibitors (TCIs) are designed to bind poorly conserved amino acids by means of reactive groups, the so-called warheads.  Currently, targeting noncatalytic cysteine residues with acrylamides and other α,β-unsatd. carbonyl compds. is the predominant strategy in TCI development.  The recent ascent of covalent drugs has stimulated considerable efforts to characterize alternative warheads for the covalent-reversible and irreversible engagement of noncatalytic cysteine residues as well as other amino acids.  This Perspective article provides an overview of warheads-beyond α,β-unsatd. amides-recently used in the design of targeted covalent ligands.  Promising reactive groups that have not yet demonstrated their utility in TCI development are also highlighted.  Special emphasis is placed on the discussion of reactivity and of case studies illustrating applications in medicinal chem. and chem. biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDhc4PWV293LVg90H21EOLACvtfcHk0liypBP4sJutnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFKiur%252FL&md5=4623733558ff99aadebfc13950b816ad</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01153&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01153%26sid%3Dliteratum%253Aachs%26aulast%3DGehringer%26aufirst%3DM.%26aulast%3DLaufer%26aufirst%3DS.%2BA.%26atitle%3DEmerging%2520and%2520re-emerging%2520warheads%2520for%2520targeted%2520covalent%2520inhibitors%253A%2520applications%2520in%2520medicinal%2520chemistry%2520and%2520chemical%2520biology%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D5673%26epage%3D5724%26doi%3D10.1021%2Facs.jmedchem.8b01153" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eliseenkova, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neubert, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohammadi, M.</span></span> <span> </span><span class="NLM_article-title">A ‘molecular brake’ in the kinase hinge region regulates the activity of receptor tyrosine kinases</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">717</span>– <span class="NLM_lpage">730</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2007.06.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1016%2Fj.molcel.2007.06.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=17803937" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFWlsL7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2007&pages=717-730&author=H.+Chenauthor=J.+Maauthor=W.+Liauthor=A.+V.+Eliseenkovaauthor=C.+Xuauthor=T.+A.+Neubertauthor=W.+T.+Millerauthor=M.+Mohammadi&title=A+%E2%80%98molecular+brake%E2%80%99+in+the+kinase+hinge+region+regulates+the+activity+of+receptor+tyrosine+kinases&doi=10.1016%2Fj.molcel.2007.06.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">A molecular brake in the kinase hinge region regulates the activity of receptor tyrosine kinases</span></div><div class="casAuthors">Chen, Huaibin; Ma, Jinghong; Li, Wanqing; Eliseenkova, Anna V.; Xu, Chongfeng; Neubert, Thomas A.; Miller, W. Todd; Mohammadi, Moosa</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">717-730</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Activating mutations in the tyrosine kinase domain of receptor tyrosine kinases (RTKs) cause cancer and skeletal disorders.  Comparison of the crystal structures of unphosphorylated and phosphorylated wild-type FGFR2 kinase domains (FGFR2Ks) with those of seven unphosphorylated pathogenic mutants reveals an autoinhibitory "mol. brake" mediated by a triad of residues in the kinase hinge region of all FGFRs.  Structural anal. shows that many other RTKs, including PDGFRs, VEGFRs, KIT, CSF1R, FLT3, TEK, and TIE, are also subject to regulation by this brake.  Pathogenic mutations activate FGFRs and other RTKs by disengaging the brake either directly or indirectly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobYTkvrN7lYrVg90H21EOLACvtfcHk0lipkpNVEAveIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFWlsL7I&md5=dba8a6b97b9d1826b2223146be137963</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2007.06.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2007.06.028%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DH.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DEliseenkova%26aufirst%3DA.%2BV.%26aulast%3DXu%26aufirst%3DC.%26aulast%3DNeubert%26aufirst%3DT.%2BA.%26aulast%3DMiller%26aufirst%3DW.%2BT.%26aulast%3DMohammadi%26aufirst%3DM.%26atitle%3DA%2520%25E2%2580%2598molecular%2520brake%25E2%2580%2599%2520in%2520the%2520kinase%2520hinge%2520region%2520regulates%2520the%2520activity%2520of%2520receptor%2520tyrosine%2520kinases%26jtitle%3DMol.%2520Cell%26date%3D2007%26volume%3D27%26spage%3D717%26epage%3D730%26doi%3D10.1016%2Fj.molcel.2007.06.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hagel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miduturu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheets, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stransky, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bifulco, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodous, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooijmans, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shutes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lengauer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohl, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzi, T.</span></span> <span> </span><span class="NLM_article-title">First selective small molecule inhibitor of FGFR4 for the treatment of hepatocellular carcinomas with an activated FGFR4 signaling pathway</span>. <i>Cancer Discovery</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">424</span>– <span class="NLM_lpage">437</span>, <span class="refDoi"> DOI: 10.1158/2159-8290.CD-14-1029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1158%2F2159-8290.CD-14-1029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=25776529" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmsVSrtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=424-437&author=M.+Hagelauthor=C.+Miduturuauthor=M.+Sheetsauthor=N.+Rubinauthor=W.+Wengauthor=N.+Stranskyauthor=N.+Bifulcoauthor=J.+L.+Kimauthor=B.+Hodousauthor=N.+Brooijmansauthor=A.+Shutesauthor=C.+Winterauthor=C.+Lengauerauthor=N.+E.+Kohlauthor=T.+Guzi&title=First+selective+small+molecule+inhibitor+of+FGFR4+for+the+treatment+of+hepatocellular+carcinomas+with+an+activated+FGFR4+signaling+pathway&doi=10.1158%2F2159-8290.CD-14-1029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">First Selective Small Molecule Inhibitor of FGFR4 for the Treatment of Hepatocellular Carcinomas with an Activated FGFR4 Signaling Pathway</span></div><div class="casAuthors">Hagel, Margit; Miduturu, Chandra; Sheets, Michael; Rubin, Nooreen; Weng, Weifan; Stransky, Nicolas; Bifulco, Neil; Kim, Joseph L.; Hodous, Brian; Brooijmans, Natasja; Shutes, Adam; Winter, Christopher; Lengauer, Christoph; Kohl, Nancy E.; Guzi, Timothy</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">424-437</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">HCC cells with FGF19 gene amplification or overexpression and an intact FGFR4 signaling pathway are sensitive to potent and selective FGFR4 inhibition by the small mol. BLU9931 in vitro and in xenograft models.  Aberrant signaling through the fibroblast growth factor 19 (FGF19)/fibroblast growth factor receptor 4 (FGFR 4) signaling complex has been shown to cause hepatocellular carcinoma (HCC) in mice and has been implicated to play a similar role in humans.  We have developed BLU9931, a potent and irreversible small-mol. inhibitor of FGFR4, as a targeted therapy to treat patients with HCC whose tumors have an activated FGFR4 signaling pathway.  BLU9931 is exquisitely selective for FGFR4 vs. other FGFR family members and all other kinases.  BLU9931 shows remarkable antitumor activity in mice bearing an HCC tumor xenograft that overexpresses FGF19 due to amplification as well as a liver tumor xenograft that overexpresses FGF19 mRNA but lacks FGF19 amplification.  Approx. one third of patients with HCC whose tumors express FGF19 together with FGFR4 and its coreceptor klotho β (KLB) could potentially respond to treatment with an FGFR4 inhibitor.  These findings are the first demonstration of a therapeutic strategy that targets a subset of patients with HCC.  Significance: This article documents the discovery of BLU9931, a novel irreversible kinase inhibitor that specifically targets FGFR4 while sparing all other FGFR paralogs and demonstrates exquisite kinome selectivity.  BLU9931 is efficacious in tumors with an intact FGFR4 signaling pathway that includes FGF19, FGFR4, and KLB.  BLU9931 is the first FGFR4-selective mol. for the treatment of patients with HCC with aberrant FGFR4 signaling.  Cancer Discov; 5(4); 424-37. cpr2015 AACR.  See related commentary by Packer and Pollock, p. 355 This article is highlighted in the In This Issue feature, p.  333.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqcHNr-1ZnG7Vg90H21EOLACvtfcHk0lipkpNVEAveIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmsVSrtL8%253D&md5=4a69439aee01a3c090c14851a785362e</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-14-1029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-14-1029%26sid%3Dliteratum%253Aachs%26aulast%3DHagel%26aufirst%3DM.%26aulast%3DMiduturu%26aufirst%3DC.%26aulast%3DSheets%26aufirst%3DM.%26aulast%3DRubin%26aufirst%3DN.%26aulast%3DWeng%26aufirst%3DW.%26aulast%3DStransky%26aufirst%3DN.%26aulast%3DBifulco%26aufirst%3DN.%26aulast%3DKim%26aufirst%3DJ.%2BL.%26aulast%3DHodous%26aufirst%3DB.%26aulast%3DBrooijmans%26aufirst%3DN.%26aulast%3DShutes%26aufirst%3DA.%26aulast%3DWinter%26aufirst%3DC.%26aulast%3DLengauer%26aufirst%3DC.%26aulast%3DKohl%26aufirst%3DN.%2BE.%26aulast%3DGuzi%26aufirst%3DT.%26atitle%3DFirst%2520selective%2520small%2520molecule%2520inhibitor%2520of%2520FGFR4%2520for%2520the%2520treatment%2520of%2520hepatocellular%2520carcinomas%2520with%2520an%2520activated%2520FGFR4%2520signaling%2520pathway%26jtitle%3DCancer%2520Discovery%26date%3D2015%26volume%3D5%26spage%3D424%26epage%3D437%26doi%3D10.1158%2F2159-8290.CD-14-1029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanizaki, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aref, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rusan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, S.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ercan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capelletti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hur, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sim, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaudet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbie, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, J-R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yun, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammerman, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohammadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jänne, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Development of covalent inhibitors that can overcome resistance to first-generation FGFR kinase inhibitors</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">E4869</span>– <span class="NLM_lpage">E4877</span>, <span class="refDoi"> DOI: 10.1073/pnas.1403438111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1073%2Fpnas.1403438111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=25349422" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVWmsL%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=E4869-E4877&author=L.+Tanauthor=J.+Wangauthor=J.+Tanizakiauthor=Z.+Huangauthor=A.+R.+Arefauthor=M.+Rusanauthor=S.-J.+Zhuauthor=Y.+Zhangauthor=D.+Ercanauthor=R.+G.+Liaoauthor=M.+Capellettiauthor=W.+Zhouauthor=W.+Hurauthor=N.+Kimauthor=T.+Simauthor=S.+Gaudetauthor=D.+A.+Barbieauthor=J-R.+J.+Yehauthor=C.-H.+Yunauthor=P.+S.+Hammermanauthor=M.+Mohammadiauthor=P.+A.+J%C3%A4nneauthor=N.+S.+Gray&title=Development+of+covalent+inhibitors+that+can+overcome+resistance+to+first-generation+FGFR+kinase+inhibitors&doi=10.1073%2Fpnas.1403438111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Development of covalent inhibitors that can overcome resistance to first-generation FGFR kinase inhibitors</span></div><div class="casAuthors">Tan, Li; Wang, Jun; Tanizaki, Junko; Huang, Zhifeng; Aref, Amir R.; Rusan, Maria; Zhu, Su-Jie; Zhang, Yiyun; Ercan, Dalia; Liao, Rachel G.; Capelletti, Marzia; Zhou, Wenjun; Hur, Wooyoung; Kim, NamDoo; Sim, Taebo; Gaudet, Suzanne; Barbie, David A.; Yeh, Jing-Ruey Joanna; Yun, Cai-Hong; Hammerman, Peter S.; Mohammadi, Moosa; Janne, Pasi A.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">E4869-E4877</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The human FGF receptors (FGFRs) play crit. roles in various human cancers, and several FGFR inhibitors are currently under clin. investigation.  Resistance usually results from selection for mutant kinases that are impervious to the action of the drug or from up-regulation of compensatory signaling pathways.  Preclin. studies have demonstrated that resistance to FGFR inhibitors can be acquired through mutations in the FGFR gatekeeper residue, as clin. obsd. for FGFR4 in embryonal rhabdomyosarcoma and neuroendocrine breast carcinomas.  Here the authors report on the use of a structure-based drug design to develop two selective, next-generation covalent FGFR inhibitors, the FGFR irreversible inhibitors I and II.  To our knowledge, I and II are the first inhibitors that can potently inhibit the proliferation of cells dependent upon the gatekeeper mutants of FGFR1 or FGFR2, which confer resistance to first-generation clin. FGFR inhibitors such as NVP-BGJ398 and AZD4547.  Because of the conformational flexibility of the reactive acrylamide substituent, II has the unprecedented ability to inhibit both the EGF receptor (EGFR) and FGFR covalently by targeting two distinct cysteine residues.  The authors report the cocrystal structure of FGFR4 with I, which unexpectedly exhibits a "DFG-out" covalent binding mode.  The structural basis for dual FGFR and EGFR targeting by FIIN3 also is illustrated by crystal structures of II bound with FGFR4 V550L and EGFR L858R.  These results have important implications for the design of covalent FGFR inhibitors that can overcome clin. resistance and provide the first example, to the knowledge, of a kinase inhibitor that covalently targets cysteines located in different positions within the ATP-binding pocket.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmCbTDKPA32LVg90H21EOLACvtfcHk0lhnV0Zgr_99nQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVWmsL%252FP&md5=fd4a7eac1e0c17e7c9f2651dd2d76ac7</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1403438111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1403438111%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DTanizaki%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DAref%26aufirst%3DA.%2BR.%26aulast%3DRusan%26aufirst%3DM.%26aulast%3DZhu%26aufirst%3DS.-J.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DErcan%26aufirst%3DD.%26aulast%3DLiao%26aufirst%3DR.%2BG.%26aulast%3DCapelletti%26aufirst%3DM.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DHur%26aufirst%3DW.%26aulast%3DKim%26aufirst%3DN.%26aulast%3DSim%26aufirst%3DT.%26aulast%3DGaudet%26aufirst%3DS.%26aulast%3DBarbie%26aufirst%3DD.%2BA.%26aulast%3DYeh%26aufirst%3DJ-R.%2BJ.%26aulast%3DYun%26aufirst%3DC.-H.%26aulast%3DHammerman%26aufirst%3DP.%2BS.%26aulast%3DMohammadi%26aufirst%3DM.%26aulast%3DJ%25C3%25A4nne%26aufirst%3DP.%2BA.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DDevelopment%2520of%2520covalent%2520inhibitors%2520that%2520can%2520overcome%2520resistance%2520to%2520first-generation%2520FGFR%2520kinase%2520inhibitors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3DE4869%26epage%3DE4877%26doi%3D10.1073%2Fpnas.1403438111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hett, E. C.</span></span> <span> </span><span class="NLM_article-title">Know your target, know your molecule</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">368</span>– <span class="NLM_lpage">372</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1813</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1038%2Fnchembio.1813" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=25978985" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFemtrvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=368-372&author=M.+E.+Bunnageauthor=A.+M.+Gilbertauthor=L.+H.+Jonesauthor=E.+C.+Hett&title=Know+your+target%2C+know+your+molecule&doi=10.1038%2Fnchembio.1813"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Know your target, know your molecule</span></div><div class="casAuthors">Bunnage, Mark E.; Gilbert, Adam M.; Jones, Lyn H.; Hett, Erik C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">368-372</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review. The pharmaceutical industry continues to experience significant attrition of drug candidates during phase 2 proof-of-concept clin. studies.  We describe some questions about the characteristics of protein targets and small-mol. drugs that may be important to consider in drug-discovery projects and could improve prospects for future clin. success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRthCFSFvOXrVg90H21EOLACvtfcHk0lhnV0Zgr_99nQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFemtrvK&md5=f9606135ccea6af432040d8afb630ede</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1813&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1813%26sid%3Dliteratum%253Aachs%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DGilbert%26aufirst%3DA.%2BM.%26aulast%3DJones%26aufirst%3DL.%2BH.%26aulast%3DHett%26aufirst%3DE.%2BC.%26atitle%3DKnow%2520your%2520target%252C%2520know%2520your%2520molecule%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D368%26epage%3D372%26doi%3D10.1038%2Fnchembio.1813" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barf, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaptein, A.</span></span> <span> </span><span class="NLM_article-title">Irreversible protein kinase inhibitors: balancing the benefits and risks</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">6243</span>– <span class="NLM_lpage">6262</span>, <span class="refDoi"> DOI: 10.1021/jm3003203</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3003203" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC38XnsVerur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=6243-6262&author=T.+Barfauthor=A.+Kaptein&title=Irreversible+protein+kinase+inhibitors%3A+balancing+the+benefits+and+risks&doi=10.1021%2Fjm3003203"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Irreversible Protein Kinase Inhibitors: Balancing the Benefits and Risks</span></div><div class="casAuthors">Barf, Tjeerd; Kaptein, Allard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6243-6262</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Not applicable for a Perspective.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4PLlisO0OVbVg90H21EOLACvtfcHk0lhnV0Zgr_99nQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XnsVerur8%253D&md5=5d87850857d37faab02e6037ef8140a4</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fjm3003203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3003203%26sid%3Dliteratum%253Aachs%26aulast%3DBarf%26aufirst%3DT.%26aulast%3DKaptein%26aufirst%3DA.%26atitle%3DIrreversible%2520protein%2520kinase%2520inhibitors%253A%2520balancing%2520the%2520benefits%2520and%2520risks%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D6243%26epage%3D6262%26doi%3D10.1021%2Fjm3003203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schwanhäusser, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gossen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dittmar, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Selbach, M.</span></span> <span> </span><span class="NLM_article-title">Global analysis of cellular protein translation by pulsed SILAC</span>. <i>Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">205</span>– <span class="NLM_lpage">209</span>, <span class="refDoi"> DOI: 10.1002/pmic.200800275</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1002%2Fpmic.200800275" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=19053139" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A280%3ADC%252BD1M%252FkvFWhsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=205-209&author=B.+Schwanh%C3%A4usserauthor=M.+Gossenauthor=G.+Dittmarauthor=M.+Selbach&title=Global+analysis+of+cellular+protein+translation+by+pulsed+SILAC&doi=10.1002%2Fpmic.200800275"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Global analysis of cellular protein translation by pulsed SILAC</span></div><div class="casAuthors">Schwanhausser Bjorn; Gossen Manfred; Dittmar Gunnar; Selbach Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Proteomics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">205-9</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Current methods for system-wide gene expression analysis detect changes in mRNA abundance, but neglect regulation at the level of translation.  Pulse labeling with stable isotopes has been used to measure protein turnover rates, but this does not directly provide information about translation rates.  Here, we developed pulsed stable isotope labeling by amino acids in cell culture (pSILAC) with two heavy isotope labels to directly quantify protein translation on a proteome-wide scale.  We applied the method to cellular iron homeostasis as a model system and demonstrate that it can confidently identify proteins that are translationally regulated by iron availability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTlGBzGGsSeDG52AMpI4GK2fW6udTcc2eaFOBxM8PREQrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1M%252FkvFWhsQ%253D%253D&md5=3f223ce86b049cf6cd41a321e710f07b</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1002%2Fpmic.200800275&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpmic.200800275%26sid%3Dliteratum%253Aachs%26aulast%3DSchwanh%25C3%25A4usser%26aufirst%3DB.%26aulast%3DGossen%26aufirst%3DM.%26aulast%3DDittmar%26aufirst%3DG.%26aulast%3DSelbach%26aufirst%3DM.%26atitle%3DGlobal%2520analysis%2520of%2520cellular%2520protein%2520translation%2520by%2520pulsed%2520SILAC%26jtitle%3DProteomics%26date%3D2009%26volume%3D9%26spage%3D205%26epage%3D209%26doi%3D10.1002%2Fpmic.200800275" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zapf, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstenberger, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Limburg, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caspers, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aulabaugh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurumbail, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shakya, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spaulding, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czerwinski, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seth, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medley, Q. G.</span></span> <span> </span><span class="NLM_article-title">Covalent inhibitors of interleukin-2 inducible T cell kinase (Itk) with nanomolar potency in a whole-blood assay</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">10047</span>– <span class="NLM_lpage">10063</span>, <span class="refDoi"> DOI: 10.1021/jm301190s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301190s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFOlsbnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=10047-10063&author=C.+W.+Zapfauthor=B.+S.+Gerstenbergerauthor=L.+Xingauthor=D.+C.+Limburgauthor=D.+R.+Andersonauthor=N.+Caspersauthor=S.+Hanauthor=A.+Aulabaughauthor=R.+Kurumbailauthor=S.+Shakyaauthor=X.+Liauthor=V.+Spauldingauthor=R.+M.+Czerwinskiauthor=N.+Sethauthor=Q.+G.+Medley&title=Covalent+inhibitors+of+interleukin-2+inducible+T+cell+kinase+%28Itk%29+with+nanomolar+potency+in+a+whole-blood+assay&doi=10.1021%2Fjm301190s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent Inhibitors of Interleukin-2 Inducible T Cell Kinase (Itk) with Nanomolar Potency in a Whole-Blood Assay</span></div><div class="casAuthors">Zapf, Christoph W.; Gerstenberger, Brian S.; Xing, Li; Limburg, David C.; Anderson, David R.; Caspers, Nicole; Han, Seungil; Aulabaugh, Ann; Kurumbail, Ravi; Shakya, Subarna; Li, Xin; Spaulding, Vikki; Czerwinski, Robert M.; Seth, Nilufer; Medley, Quintus G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">10047-10063</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We wish to report a strategy that targets interleukin-2 inducible T cell kinase (Itk) with covalent inhibitors.  Thus far, covalent inhibition of Itk has not been disclosed in the literature.  Structure-based drug design was utilized to achieve low nanomolar potency of the disclosed series even at high ATP concns.  Kinetic measurements confirmed an irreversible binding mode with off-rate half-lives exceeding 24 h and moderate on-rates.  The analogs are highly potent in a cellular IP1 assay as well as in a human whole-blood (hWB) assay.  Despite a half-life of approx. 2 h in resting primary T cells, the covalent inhibition of Itk resulted in functional silencing of the TCR pathway for more than 24 h.  This prolonged effect indicates that covalent inhibition is a viable strategy to target the inactivation of Itk.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnh9j28SM8jbVg90H21EOLACvtfcHk0lif50mYBZivNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFOlsbnP&md5=7e39a17151dc9493d13439bc48fa83a3</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm301190s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301190s%26sid%3Dliteratum%253Aachs%26aulast%3DZapf%26aufirst%3DC.%2BW.%26aulast%3DGerstenberger%26aufirst%3DB.%2BS.%26aulast%3DXing%26aufirst%3DL.%26aulast%3DLimburg%26aufirst%3DD.%2BC.%26aulast%3DAnderson%26aufirst%3DD.%2BR.%26aulast%3DCaspers%26aufirst%3DN.%26aulast%3DHan%26aufirst%3DS.%26aulast%3DAulabaugh%26aufirst%3DA.%26aulast%3DKurumbail%26aufirst%3DR.%26aulast%3DShakya%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DSpaulding%26aufirst%3DV.%26aulast%3DCzerwinski%26aufirst%3DR.%2BM.%26aulast%3DSeth%26aufirst%3DN.%26aulast%3DMedley%26aufirst%3DQ.%2BG.%26atitle%3DCovalent%2520inhibitors%2520of%2520interleukin-2%2520inducible%2520T%2520cell%2520kinase%2520%2528Itk%2529%2520with%2520nanomolar%2520potency%2520in%2520a%2520whole-blood%2520assay%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D10047%26epage%3D10063%26doi%3D10.1021%2Fjm301190s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guagnano, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furet, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spanka, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bordas, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Douget, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stamm, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brueggen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wartmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berghausen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drueckes, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerlin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bussiere, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graus Porta, D.</span></span> <span> </span><span class="NLM_article-title">Discovery of 3-(2,6-dichloro-3,5-dimethoxy-phenyl)-1-{6-[4-(4-ethyl-piperazin-1-yl)-phenylamino]-pyrimidin-4-yl}-1-methyl-urea (NVP-BGJ398), a potent and selective inhibitor of the fibroblast growth factor receptor family of receptor tyrosine kinases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">7066</span>– <span class="NLM_lpage">7083</span>, <span class="refDoi"> DOI: 10.1021/jm2006222</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2006222" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1SrsrvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=7066-7083&author=V.+Guagnanoauthor=P.+Furetauthor=C.+Spankaauthor=V.+Bordasauthor=M.+Le+Dougetauthor=C.+Stammauthor=J.+Brueggenauthor=M.+R.+Jensenauthor=C.+Schnellauthor=H.+Schmidauthor=M.+Wartmannauthor=J.+Berghausenauthor=P.+Drueckesauthor=A.+Zimmerlinauthor=D.+Bussiereauthor=J.+Murrayauthor=D.+Graus+Porta&title=Discovery+of+3-%282%2C6-dichloro-3%2C5-dimethoxy-phenyl%29-1-%7B6-%5B4-%284-ethyl-piperazin-1-yl%29-phenylamino%5D-pyrimidin-4-yl%7D-1-methyl-urea+%28NVP-BGJ398%29%2C+a+potent+and+selective+inhibitor+of+the+fibroblast+growth+factor+receptor+family+of+receptor+tyrosine+kinases&doi=10.1021%2Fjm2006222"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 3-(2,6-Dichloro-3,5-dimethoxy-phenyl)-1-{6-[4-(4-ethyl-piperazin-1-yl)-phenylamino]-pyrimidin-4-yl}-1-methyl-urea (NVP-BGJ398), A Potent and Selective Inhibitor of the Fibroblast Growth Factor Receptor Family of Receptor Tyrosine Kinase</span></div><div class="casAuthors">Guagnano, Vito; Furet, Pascal; Spanka, Carsten; Bordas, Vincent; Le Douget, Mickael; Stamm, Christelle; Brueggen, Josef; Jensen, Michael R.; Schnell, Christian; Schmid, Herbert; Wartmann, Markus; Berghausen, Joerg; Drueckes, Peter; Zimmerlin, Alfred; Bussiere, Dirksen; Murray, Jeremy; Graus Porta, Diana</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">7066-7083</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel series of N-aryl-N'-pyrimidin-4-yl ureas has been optimized to afford potent and selective inhibitors of the fibroblast growth factor receptor tyrosine kinases 1, 2, and 3 by rationally designing the substitution pattern of the aryl ring.  On the basis of its in vitro profile, compd. 1h (NVP-BGJ398) was selected for in vivo evaluation and showed significant antitumor activity in RT112 bladder cancer xenografts models overexpressing wild-type FGFR3.  These results support the potential therapeutic use of 1h as a new anticancer agent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSC1v0A_6I_bVg90H21EOLACvtfcHk0lijJoN0RjSDQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1SrsrvK&md5=f9a2d742294cd3814e872bdd8add1ffa</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fjm2006222&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2006222%26sid%3Dliteratum%253Aachs%26aulast%3DGuagnano%26aufirst%3DV.%26aulast%3DFuret%26aufirst%3DP.%26aulast%3DSpanka%26aufirst%3DC.%26aulast%3DBordas%26aufirst%3DV.%26aulast%3DLe%2BDouget%26aufirst%3DM.%26aulast%3DStamm%26aufirst%3DC.%26aulast%3DBrueggen%26aufirst%3DJ.%26aulast%3DJensen%26aufirst%3DM.%2BR.%26aulast%3DSchnell%26aufirst%3DC.%26aulast%3DSchmid%26aufirst%3DH.%26aulast%3DWartmann%26aufirst%3DM.%26aulast%3DBerghausen%26aufirst%3DJ.%26aulast%3DDrueckes%26aufirst%3DP.%26aulast%3DZimmerlin%26aufirst%3DA.%26aulast%3DBussiere%26aufirst%3DD.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DGraus%2BPorta%26aufirst%3DD.%26atitle%3DDiscovery%2520of%25203-%25282%252C6-dichloro-3%252C5-dimethoxy-phenyl%2529-1-%257B6-%255B4-%25284-ethyl-piperazin-1-yl%2529-phenylamino%255D-pyrimidin-4-yl%257D-1-methyl-urea%2520%2528NVP-BGJ398%2529%252C%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520the%2520fibroblast%2520growth%2520factor%2520receptor%2520family%2520of%2520receptor%2520tyrosine%2520kinases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D7066%26epage%3D7083%26doi%3D10.1021%2Fjm2006222" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, R. A.</span></span> <span> </span><span class="NLM_article-title">Covalent inhibitors in drug discovery: from accidental discoveries to avoided liabilities and designed therapies</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">1061</span>– <span class="NLM_lpage">1073</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2015.05.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1016%2Fj.drudis.2015.05.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=26002380" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptlOnu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=1061-1073&author=R.+A.+Bauer&title=Covalent+inhibitors+in+drug+discovery%3A+from+accidental+discoveries+to+avoided+liabilities+and+designed+therapies&doi=10.1016%2Fj.drudis.2015.05.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent inhibitors in drug discovery: from accidental discoveries to avoided liabilities and designed therapies</span></div><div class="casAuthors">Bauer, Renato A.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1061-1073</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Drugs that covalently bond to their biol. targets have a long history in drug discovery.  A look at drug approvals in recent years suggests that covalent drugs will continue to make impacts on human health for years to come.  Although fraught with concerns about toxicity, the high potencies and prolonged effects achievable with covalent drugs may result in less-frequent drug dosing and in wide therapeutic margins for patients.  Covalent inhibition can also dissoc. drug pharmacodynamics (PD) from pharmacokinetics (PK), which can result in desired drug efficacy for inhibitors that have short systemic exposure.  Evidence suggests that there is a reduced risk for the development of resistance against covalent drugs, which is a major challenge in areas such as oncol. and infectious disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMkBDsUEu4arVg90H21EOLACvtfcHk0lijJoN0RjSDQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptlOnu7s%253D&md5=6ba5b016d2804c7d56838a2724b9a0b9</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2015.05.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2015.05.005%26sid%3Dliteratum%253Aachs%26aulast%3DBauer%26aufirst%3DR.%2BA.%26atitle%3DCovalent%2520inhibitors%2520in%2520drug%2520discovery%253A%2520from%2520accidental%2520discoveries%2520to%2520avoided%2520liabilities%2520and%2520designed%2520therapies%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2015%26volume%3D20%26spage%3D1061%26epage%3D1073%26doi%3D10.1016%2Fj.drudis.2015.05.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Uetrecht, J.</span></span> <span> </span><span class="NLM_article-title">Idiosyncratic drug reactions: past, present, and future</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">84</span>– <span class="NLM_lpage">92</span>, <span class="refDoi"> DOI: 10.1021/tx700186p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx700186p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlygsr%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2008&pages=84-92&author=J.+Uetrecht&title=Idiosyncratic+drug+reactions%3A+past%2C+present%2C+and+future&doi=10.1021%2Ftx700186p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Idiosyncratic Drug Reactions: Past, Present, and Future</span></div><div class="casAuthors">Uetrecht, Jack</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">84-92</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Although the major working hypothesis for the mechanism of idiosyncratic drug reactions (IDRs), the hapten hypothesis, has not changed since 1987, several hypotheses have been added, for example, the danger hypothesis and the pharmaceutical interaction hypothesis.  Genetic studies have found that several IDRs are linked to specific HLA genes, providing addnl. evidence that they are immune-mediated.  Evidence that most IDRs are caused by reactive metabolites has led pharmaceutical companies to avoid drug candidates that form significant amts. of reactive metabolites; however, at least one IDR, ximelagatran-induced liver toxicity, does not appear to be caused by a reactive metabolite.  It is possible that there are biomarkers such as those related to cell stress that would predict that a drug candidate would cause a significant incidence of IDRs; however, there has been no systematic study of the changes in gene expression induced by drugs known to cause IDRs.  A major impediment to the study of the mechanisms of IDRs is the paucity of valid animal models, and if we had a better mechanistic understanding, it should be easier to develop such models.  There is growing evidence that these adverse reactions are more varied and complex than previously recognized, and it is unlikely that a quick fix will be achieved.  However, IDRs are an important cause of patient morbidity and mortality and markedly increase the uncertainty of drug development; therefore, continued basic research in this area is essential.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLG5_bNuaw4LVg90H21EOLACvtfcHk0ljl_DsyEmK1yw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlygsr%252FN&md5=eaed35d895118a05034c7762efd060dc</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Ftx700186p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx700186p%26sid%3Dliteratum%253Aachs%26aulast%3DUetrecht%26aufirst%3DJ.%26atitle%3DIdiosyncratic%2520drug%2520reactions%253A%2520past%252C%2520present%252C%2520and%2520future%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2008%26volume%3D21%26spage%3D84%26epage%3D92%26doi%3D10.1021%2Ftx700186p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">LoPachin, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gavin, T.</span></span> <span> </span><span class="NLM_article-title">Molecular mechanisms of aldehyde toxicity: a chemical perspective</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1081</span>– <span class="NLM_lpage">1091</span>, <span class="refDoi"> DOI: 10.1021/tx5001046</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx5001046" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpsVensrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2014&pages=1081-1091&author=R.+M.+LoPachinauthor=T.+Gavin&title=Molecular+mechanisms+of+aldehyde+toxicity%3A+a+chemical+perspective&doi=10.1021%2Ftx5001046"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Mechanisms of Aldehyde Toxicity: A Chemical Perspective</span></div><div class="casAuthors">LoPachin, Richard M.; Gavin, Terrence</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1081-1091</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Aldehydes are electrophilic compds. to which humans are pervasively exposed.  Despite a significant health risk due to exposure, the mechanisms of aldehyde toxicity are poorly understood.  This ambiguity is likely due to the structural diversity of aldehyde derivs. and corresponding differences in chem. reactions and biol. targets.  To gain mechanistic insight, the authors have used parameters based on the hard and soft, acids and bases (HSAB) theory to profile the different aldehyde subclasses with respect to electronic character (softness, hardness), electrophilic reactivity (electrophilic index), and biol. nucleophilic targets.  The authors' analyses indicate that short chain aldehydes and longer chain satd. alkanals are hard electrophiles that cause toxicity by forming adducts with hard biol. nucleophiles, e.g., primary nitrogen groups on lysine residues.  In contrast, α,β-unsatd. carbonyl derivs., alkenals, and the α-oxoaldehydes are soft electrophiles that preferentially react with soft nucleophilic thiolate groups on cysteine residues.  The aldehydes can therefore be grouped into subclasses according to common electronic characteristics (softness/hardness) and mol. mechanisms of toxicity.  As the authors will discuss, the toxic potencies of these subgroups are generally related to corresponding electrophilicities.  For some aldehydes, however, predictions of toxicity based on electrophilicity are less accurate due to inherent physicochem. variables that limit target accessibility, e.g., steric hindrance and soly.  The unsatd. aldehydes are also members of the conjugated type-2 alkene chem. class that includes α,β-unsatd. amide, ketone, and ester derivs.  Type-2 alkenes are electrophiles of varying softness and electrophilicity that share a common mechanism of toxicity.  Therefore, exposure to an environmental mixt. of unsatd. carbonyl derivs. could cause "type-2 alkene toxicity" through additive interactions.  Finally, the authors propose that environmentally derived aldehydes can accelerate diseases by interacting with endogenous aldehydes generated during oxidative stress.  This review provides a basis for understanding aldehyde mechanisms and environmental toxicity through the context of electronic structure, electrophilicity, and nucleophile target selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrATFAViXVn7rVg90H21EOLACvtfcHk0ljl_DsyEmK1yw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpsVensrg%253D&md5=e72707796ea59ccaff0283848ebbdf76</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Ftx5001046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx5001046%26sid%3Dliteratum%253Aachs%26aulast%3DLoPachin%26aufirst%3DR.%2BM.%26aulast%3DGavin%26aufirst%3DT.%26atitle%3DMolecular%2520mechanisms%2520of%2520aldehyde%2520toxicity%253A%2520a%2520chemical%2520perspective%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2014%26volume%3D27%26spage%3D1081%26epage%3D1091%26doi%3D10.1021%2Ftx5001046" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span> <span> </span><span class="NLM_article-title">The dynamics of drug-target interactions: drug-target residence time and its impact on efficacy and safety</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">305</span>– <span class="NLM_lpage">310</span>, <span class="refDoi"> DOI: 10.1517/17460441003677725</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1517%2F17460441003677725" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=22823083" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjvFSgsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=305-310&author=R.+A.+Copeland&title=The+dynamics+of+drug-target+interactions%3A+drug-target+residence+time+and+its+impact+on+efficacy+and+safety&doi=10.1517%2F17460441003677725"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41aR"><div class="casContent"><span class="casTitleNuber">41a</span><div class="casTitle"><span class="NLM_cas:atitle">The dynamics of drug-target interactions: drug-target residence time and its impact on efficacy and safety</span></div><div class="casAuthors">Copeland, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">305-310</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  The extent and duration of pharmacol. action is detd. by the lifetime of drug occupancy on a mol. target.  This lifetime is defined by dynamic processes that control the rates of drug assocn. and dissocn. from the target.  Recently, the term residence time has been coined to describe exptl. measurements that can be related to the lifetime of the binary drug-target complex, and this in turn to durable, pharmacodynamic activity.  The residence time concept and its impact on drug optimization are reviewed here.  Examples are provided that demonstrate how a long residence time can improve drug efficacy in vivo.  Addnl., optimization of drug-target residence time can help to mitigate off-target mediated toxicity, hence, improving drug safety and tolerability.  Recent applications of the residence time concept to both drug discovery and development are also presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyt3KrIYdRjbVg90H21EOLACvtfcHk0ljl_DsyEmK1yw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjvFSgsbo%253D&md5=aca925e00abefddf6e92d83b470d96f1</span></div><a href="/servlet/linkout?suffix=cit41a&amp;dbid=16384&amp;doi=10.1517%2F17460441003677725&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441003677725%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26atitle%3DThe%2520dynamics%2520of%2520drug-target%2520interactions%253A%2520drug-target%2520residence%2520time%2520and%2520its%2520impact%2520on%2520efficacy%2520and%2520safety%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2010%26volume%3D5%26spage%3D305%26epage%3D310%26doi%3D10.1517%2F17460441003677725" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit41b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Swinney, D. C.</span></span> <span> </span><span class="NLM_article-title">The role of binding kinetics in therapeutically useful drug action</span>. <i>Curr. Opin. Drug Discovery Dev.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">31</span>– <span class="NLM_lpage">39</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=19152211" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnt12ltrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2009&pages=31-39&author=D.+C.+Swinney&title=The+role+of+binding+kinetics+in+therapeutically+useful+drug+action"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41bR"><div class="casContent"><span class="casTitleNuber">41b</span><div class="casTitle"><span class="NLM_cas:atitle">The role of binding kinetics in therapeutically useful drug action</span></div><div class="casAuthors">Swinney, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Drug Discovery & Development</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">31-39</span>CODEN:
                <span class="NLM_cas:coden">CODDFF</span>;
        ISSN:<span class="NLM_cas:issn">2040-3437</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">A review.  Binding kinetics help define how a medicine will communicate with physiol. to produce a desired therapeutic response.  Clin. efficacy, duration of action, clin. differentiation and safety may all be influenced by binding kinetics.  The optimization of binding kinetics can be used to maximize a drug's therapeutic index and thereby decrease drug attrition.  The gap between basic scientific principles and the potential medical value of a drug is currently bridged by the use of empirical assays.  The value of binding kinetics to drug discovery will be increased through an improved ability to identify optimal kinetic mechanisms and define kinetic structure activity relationships.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8IoReBfGAabVg90H21EOLACvtfcHk0liOE__YGFU8PQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnt12ltrk%253D&md5=129948a7fb0a60f3b9995e5aedb700a4</span></div><a href="/servlet/linkout?suffix=cit41b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSwinney%26aufirst%3DD.%2BC.%26atitle%3DThe%2520role%2520of%2520binding%2520kinetics%2520in%2520therapeutically%2520useful%2520drug%2520action%26jtitle%3DCurr.%2520Opin.%2520Drug%2520Discovery%2520Dev.%26date%3D2009%26volume%3D12%26spage%3D31%26epage%3D39" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Caraballo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribeiro, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiménez-Barbero, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramström, O.</span></span> <span> </span><span class="NLM_article-title">Direct STD NMR identification of β-galactose inhibitors from a virtual dynamic hemithioacetal system</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">589</span>– <span class="NLM_lpage">593</span>, <span class="refDoi"> DOI: 10.1002/anie.200903920</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1002%2Fanie.200903920" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC3cXktlaksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2010&pages=589-593&author=R.+Caraballoauthor=H.+Dongauthor=J.+P.+Ribeiroauthor=J.+Jim%C3%A9nez-Barberoauthor=O.+Ramstr%C3%B6m&title=Direct+STD+NMR+identification+of+%CE%B2-galactose+inhibitors+from+a+virtual+dynamic+hemithioacetal+system&doi=10.1002%2Fanie.200903920"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Direct STD NMR Identification of β-Galactosidase Inhibitors from a Virtual Dynamic Hemithioacetal System</span></div><div class="casAuthors">Caraballo, Remi; Dong, Hai; Ribeiro, Joao P.; Jimenez-Barbero, Jesus; Ramstroem, Olof</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">589-593, S589/1-S589/20</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Herein we show that hemithioacetal (HTA) formation is a fast and efficient reversible reaction that can be used to generate systems of potential enzyme inhibitors in aq. media under neutral conditions.  These systems were also subjected to direct binding anal. by satn. transfer difference (STD) NMR spectroscopy, a rapid and efficient technique for protein-ligand binding studies, in which the optimal HTA interactions with the biol. target can be readily deduced.  We have successfully demonstrated, for the first time, hemithioacetal formation applied to dynamic combinatorial system generation in aq. media.  Equil. formation using this chem. proved very rapid and resulted in truly virtual dynamic systems in which 1H NMR analyses confirmed the presence of transient HTA constituents.  It has further been demonstrated that 1H STD NMR spectroscopy can be used for efficient and direct in situ identification of the best enzyme binders of the virtual dynamic system.  Inhibition studies unequivocally support the 1H STD NMR.exptl. data and enable the distinction of different β-galactosidase inhibitors and nonspecific binders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_95z4ob2R57Vg90H21EOLACvtfcHk0liOE__YGFU8PQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXktlaksw%253D%253D&md5=012f6210d6e743fe5dd958cc4500d937</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1002%2Fanie.200903920&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200903920%26sid%3Dliteratum%253Aachs%26aulast%3DCaraballo%26aufirst%3DR.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DRibeiro%26aufirst%3DJ.%2BP.%26aulast%3DJim%25C3%25A9nez-Barbero%26aufirst%3DJ.%26aulast%3DRamstr%25C3%25B6m%26aufirst%3DO.%26atitle%3DDirect%2520STD%2520NMR%2520identification%2520of%2520%25CE%25B2-galactose%2520inhibitors%2520from%2520a%2520virtual%2520dynamic%2520hemithioacetal%2520system%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2010%26volume%3D49%26spage%3D589%26epage%3D593%26doi%3D10.1002%2Fanie.200903920" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Warmuth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adrián, F.</span></span> <span> </span><span class="NLM_article-title">Ba/F3 cells and their use in kinase drug discovery</span>. <i>Curr. Opin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">55</span>– <span class="NLM_lpage">60</span>, <span class="refDoi"> DOI: 10.1097/CCO.0b013e328011a25f</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1097%2FCCO.0b013e328011a25f" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=17133113" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1Cnt7%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2007&pages=55-60&author=M.+Warmuthauthor=S.+Kimauthor=X.+Guauthor=G.+Xiaauthor=F.+Adri%C3%A1n&title=Ba%2FF3+cells+and+their+use+in+kinase+drug+discovery&doi=10.1097%2FCCO.0b013e328011a25f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Ba/F3 cells and their use in kinase drug discovery</span></div><div class="casAuthors">Warmuth, Markus; Kim, Sungjoon; Gu, Xiang-ju; Xia, Gang; Adrian, Francisco</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Oncology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">55-60</span>CODEN:
                <span class="NLM_cas:coden">CUOOE8</span>;
        ISSN:<span class="NLM_cas:issn">1040-8746</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">A review.  Purpose of review: Due to their ability to function as dominant oncogenes, protein kinases have become favored targets in the quest for 'molecularly-targeted' cancer chemotherapeutics.  The discovery of a large no. of cancer-assocd. mutations in the kinome, and the progress in developing specific small-mol. kinase inhibitors has increased the need for accurate, reproducible, and efficient kinase activity-dependent cellular assay systems.  Recent findings: Ba/F3, a murine interleukin-3 dependent pro-B cell line is increasingly popular as a model system for assessing both the potency and downstream signaling of kinase oncogenes, and the ability of small-mol. kinase inhibitors to block kinase activity.  Facilitated by their growth properties, Ba/F3 cells have recently been adapted to high-throughput assay formats for compd. profiling.  Further, several published approaches show promise in predicting resistance to small-mol. kinase inhibitors elicited by point mutations interfering with inhibitor binding.  Summary: Ba/F3 cells are an increasingly popular tool in kinase drug discovery.  The ability to test the transforming capacity of newly identified kinase mutations, and to profile drug candidates and compd. libraries in high-throughput fashion, combined with the use of Ba/F3 cells to predict clin. resistance will greatly facilitate developments in this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXQmhiRwGlDLVg90H21EOLACvtfcHk0liOE__YGFU8PQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1Cnt7%252FL&md5=a7f8d84dfedea69d408edb571e2eead9</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1097%2FCCO.0b013e328011a25f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FCCO.0b013e328011a25f%26sid%3Dliteratum%253Aachs%26aulast%3DWarmuth%26aufirst%3DM.%26aulast%3DKim%26aufirst%3DS.%26aulast%3DGu%26aufirst%3DX.%26aulast%3DXia%26aufirst%3DG.%26aulast%3DAdri%25C3%25A1n%26aufirst%3DF.%26atitle%3DBa%252FF3%2520cells%2520and%2520their%2520use%2520in%2520kinase%2520drug%2520discovery%26jtitle%3DCurr.%2520Opin.%2520Oncol.%26date%3D2007%26volume%3D19%26spage%3D55%26epage%3D60%26doi%3D10.1097%2FCCO.0b013e328011a25f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hinson, A. R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crose, L. E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belyea, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barr, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linardic, C. M.</span></span> <span> </span><span class="NLM_article-title">Human rhabdomyosarcoma cell lines for rhabdomyosarcoma research: utility and pitfalls</span>. <i>Front. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">183</span>, <span class="refDoi"> DOI: 10.3389/fonc.2013.00183</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.3389%2Ffonc.2013.00183" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=23882450" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A280%3ADC%252BC3sfjsFOhtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2013&pages=183&author=A.+R.+P.+Hinsonauthor=R.+Jonesauthor=L.+E.+S.+Croseauthor=B.+C.+Belyeaauthor=F.+G.+Barrauthor=C.+M.+Linardic&title=Human+rhabdomyosarcoma+cell+lines+for+rhabdomyosarcoma+research%3A+utility+and+pitfalls&doi=10.3389%2Ffonc.2013.00183"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Human rhabdomyosarcoma cell lines for rhabdomyosarcoma research: utility and pitfalls</span></div><div class="casAuthors">Hinson Ashley R P; Jones Rosanne; Crose Lisa E S; Belyea Brian C; Barr Frederic G; Linardic Corinne M</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in oncology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">183</span>
        ISSN:<span class="NLM_cas:issn">2234-943X</span>.
    </div><div class="casAbstract">Rhabdomyosarcoma (RMS) is the most common soft tissue sarcoma of childhood and adolescence.  Despite intergroup clinical trials conducted in Europe and North America, outcomes for high risk patients with this disease have not significantly improved in the last several decades, and survival of metastatic or relapsed disease remains extremely poor.  Accrual into new clinical trials is slow and difficult, so in vitro cell-line research and in vivo xenograft models present an attractive alternative for preclinical research for this cancer type.  Currently, 30 commonly used human RMS cell lines exist, with differing origins, karyotypes, histologies, and methods of validation.  Selecting an appropriate cell line for RMS research has important implications for outcomes.  There are also potential pitfalls in using certain cell lines including contamination with murine stromal cells, cross-contamination between cell lines, discordance between the cell line and its associated original tumor, imposter cell lines, and nomenclature errors that result in the circulation of two or more presumed unique cell lines that are actually from the same origin.  These pitfalls can be avoided by testing for species-specific isoenzymes, microarray analysis, assays for subtype-specific fusion products, and short tandem repeat analysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSN0jeRw0uUgDEC-75aqa5AfW6udTcc2eaS-H2FCIMzNrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sfjsFOhtA%253D%253D&md5=ff7c421705574030c7566122a562d59a</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.3389%2Ffonc.2013.00183&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffonc.2013.00183%26sid%3Dliteratum%253Aachs%26aulast%3DHinson%26aufirst%3DA.%2BR.%2BP.%26aulast%3DJones%26aufirst%3DR.%26aulast%3DCrose%26aufirst%3DL.%2BE.%2BS.%26aulast%3DBelyea%26aufirst%3DB.%2BC.%26aulast%3DBarr%26aufirst%3DF.%2BG.%26aulast%3DLinardic%26aufirst%3DC.%2BM.%26atitle%3DHuman%2520rhabdomyosarcoma%2520cell%2520lines%2520for%2520rhabdomyosarcoma%2520research%253A%2520utility%2520and%2520pitfalls%26jtitle%3DFront.%2520Oncol.%26date%3D2013%26volume%3D3%26spage%3D183%26doi%3D10.3389%2Ffonc.2013.00183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Michael, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doherty, M. M.</span></span> <span> </span><span class="NLM_article-title">Drug metabolism by tumors: its nature, relevance and therapeutic implications</span>. <i>Expert Opin. Drug Metab. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">783</span>– <span class="NLM_lpage">803</span>, <span class="refDoi"> DOI: 10.1517/17425255.3.6.783</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1517%2F17425255.3.6.783" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=18028025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlCls7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=783-803&author=M.+Michaelauthor=M.+M.+Doherty&title=Drug+metabolism+by+tumors%3A+its+nature%2C+relevance+and+therapeutic+implications&doi=10.1517%2F17425255.3.6.783"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Drug metabolism by tumors: its nature, relevance and therapeutic implications</span></div><div class="casAuthors">Michael, Michael; Doherty, Margaret M.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Metabolism & Toxicology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">783-803</span>CODEN:
                <span class="NLM_cas:coden">EODMAP</span>;
        ISSN:<span class="NLM_cas:issn">1742-5255</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Drug-metabolizing enzymes (DMEs) are present in tumors and are capable of biotransforming a variety of antineoplastics.  Tumoral drug metab. is both a potential mechanism of resistance and a means of achieving optimal therapy.  This review addresses the classes of DMEs, their cytotoxic substrates and distribution in specific malignancies.  The limitations of preclin. and clin. studies are highlighted.  Their role in predicting therapeutic response, the activation of prodrugs and the potential for their modulation for gain is also addressed.  The contribution of tumoral DMEs to cancer therapy can only be ascertained through large prospective studies and supported by new technologies.  Only then can efforts be concd. in the design of better prodrugs or combination therapy to optimize individual therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5aLOaiaVgPrVg90H21EOLACvtfcHk0lg_LJx7-TgQ_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlCls7fK&md5=40fa6309e929f7f7ba1bc063d76744ef</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1517%2F17425255.3.6.783&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17425255.3.6.783%26sid%3Dliteratum%253Aachs%26aulast%3DMichael%26aufirst%3DM.%26aulast%3DDoherty%26aufirst%3DM.%2BM.%26atitle%3DDrug%2520metabolism%2520by%2520tumors%253A%2520its%2520nature%252C%2520relevance%2520and%2520therapeutic%2520implications%26jtitle%3DExpert%2520Opin.%2520Drug%2520Metab.%2520Toxicol.%26date%3D2007%26volume%3D3%26spage%3D783%26epage%3D803%26doi%3D10.1517%2F17425255.3.6.783" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Trunzer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faller, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerlin, A.</span></span> <span> </span><span class="NLM_article-title">Metabolic soft spot identification and compound optimization in early discovery phases using metaSite and LC-MS/MS validation</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">329</span>– <span class="NLM_lpage">335</span>, <span class="refDoi"> DOI: 10.1021/jm8008663</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm8008663" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFChsLfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=329-335&author=M.+Trunzerauthor=B.+Fallerauthor=A.+Zimmerlin&title=Metabolic+soft+spot+identification+and+compound+optimization+in+early+discovery+phases+using+metaSite+and+LC-MS%2FMS+validation&doi=10.1021%2Fjm8008663"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolic Soft Spot Identification and Compound Optimization in Early Discovery Phases Using MetaSite and LC-MS/MS Validation</span></div><div class="casAuthors">Trunzer, Markus; Faller, Bernard; Zimmerlin, Alfred</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">329-335</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Metabolic stability is a key property to enable drugs to reach therapeutic concns.  Microsomal clearance assays are used to dial out labile compds. in early discovery phases.  However, because they do not provide any information on soft spots, the rational design of more stable compds. remains challenging.  A robust soft spot identification procedure combining in silico prediction ranking using MetaSite and mass-spectrometric confirmation is described.  MetaSite's first rank order predictions were exptl. confirmed for only about 55% of the compds.  For another 29% of the compds., the second (20%) or the third (9%) rank order predictions were detected.  This automatic and high-throughput reprioritization of a likely soft-spot increases the likelihood of working on the right soft spot from about 50% to more than 80%.  With this information, the structure-metab. relationships are likely to be understood faster and earlier in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoV4vzsZ7zQ_rVg90H21EOLACvtfcHk0lg_LJx7-TgQ_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFChsLfJ&md5=8628722a0cd75b0c6497cc383bd4ec16</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fjm8008663&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8008663%26sid%3Dliteratum%253Aachs%26aulast%3DTrunzer%26aufirst%3DM.%26aulast%3DFaller%26aufirst%3DB.%26aulast%3DZimmerlin%26aufirst%3DA.%26atitle%3DMetabolic%2520soft%2520spot%2520identification%2520and%2520compound%2520optimization%2520in%2520early%2520discovery%2520phases%2520using%2520metaSite%2520and%2520LC-MS%252FMS%2520validation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D329%26epage%3D335%26doi%3D10.1021%2Fjm8008663" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hidalgo, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raub, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borchardt, R. T.</span></span> <span> </span><span class="NLM_article-title">Characterization of the human colon carcinoma cell line (Caco-2) as a model system for intestinal epithelial permeability</span>. <i>Gastroenterology</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>96</i></span>,  <span class="NLM_fpage">736</span>– <span class="NLM_lpage">749</span>, <span class="refDoi"> DOI: 10.1016/0016-5085(89)90897-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1016%2FS0016-5085%2889%2980072-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=2914637" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A280%3ADyaL1M7itFeqsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=1989&pages=736-749&author=I.+J.+Hidalgoauthor=T.+J.+Raubauthor=R.+T.+Borchardt&title=Characterization+of+the+human+colon+carcinoma+cell+line+%28Caco-2%29+as+a+model+system+for+intestinal+epithelial+permeability&doi=10.1016%2F0016-5085%2889%2990897-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of the human colon carcinoma cell line (Caco-2) as a model system for intestinal epithelial permeability</span></div><div class="casAuthors">Hidalgo I J; Raub T J; Borchardt R T</div><div class="citationInfo"><span class="NLM_cas:title">Gastroenterology</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">96</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">736-49</span>
        ISSN:<span class="NLM_cas:issn">0016-5085</span>.
    </div><div class="casAbstract">Caco-2 cells develop morphologic characteristics of normal enterocytes when grown on plastic dishes or nitrocellulose filters.  The purpose of this study was to determine whether Caco-2 cells undergo similar differentiation when grown on Transwell polycarbonate membranes, and to study the suitability of Caco-2 monolayers as an intestinal epithelial transport model system.  Transepithelial electrical resistance values after confluence were 173.5 omega.cm2 and remained unchanged through day 17.  Permeabilities to the water-soluble fluid-phase markers that do not permeate the membrane, Lucifer yellow CH, [14C]inulin, [14C]polyethylene glycol, and [3H] dextran were less than 0.25% of the administered amount per hour after day 10.  Qualitative evaluation of uptake and permeability to horseradish peroxidase confirmed the similarity in uptake and barrier properties between this cell system and the small intestinal epithelial layer.  We conclude that Caco-2 cells grown on collagen-coated polycarbonate membranes should represent a valuable transport model system for the small intestinal epithelium.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSOn97989rzEIqdXsLcGTEUfW6udTcc2eaS-H2FCIMzNrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaL1M7itFeqsg%253D%253D&md5=59ffccc1dd4dccb01400c05df95ac1ff</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2FS0016-5085%2889%2980072-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0016-5085%252889%252980072-1%26sid%3Dliteratum%253Aachs%26aulast%3DHidalgo%26aufirst%3DI.%2BJ.%26aulast%3DRaub%26aufirst%3DT.%2BJ.%26aulast%3DBorchardt%26aufirst%3DR.%2BT.%26atitle%3DCharacterization%2520of%2520the%2520human%2520colon%2520carcinoma%2520cell%2520line%2520%2528Caco-2%2529%2520as%2520a%2520model%2520system%2520for%2520intestinal%2520epithelial%2520permeability%26jtitle%3DGastroenterology%26date%3D1989%26volume%3D96%26spage%3D736%26epage%3D749%26doi%3D10.1016%2F0016-5085%2889%2990897-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buur, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mork, N.</span></span> <span> </span><span class="NLM_article-title">Metabolism of testosterone during in vitro transport across Caco-2 cell monolayers: evidence for β-hydroxysteroid dehydrogenase activity in differentiated Caco-2 cells</span>. <i>Pharm. Res.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1290</span>– <span class="NLM_lpage">1294</span>, <span class="refDoi"> DOI: 10.1023/A:1015805317375</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1023%2FA%3A1015805317375" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1448428" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADyaK38Xmt1KmtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1992&pages=1290-1294&author=A.+Buurauthor=N.+Mork&title=Metabolism+of+testosterone+during+in+vitro+transport+across+Caco-2+cell+monolayers%3A+evidence+for+%CE%B2-hydroxysteroid+dehydrogenase+activity+in+differentiated+Caco-2+cells&doi=10.1023%2FA%3A1015805317375"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolism of testosterone during in vitro transport across CACO-2 cell monolayers:  evidence for β-hydroxysteroid dehydrogenase activity in differentiated CACO-2 cells</span></div><div class="casAuthors">Buur, Anders; Moerk, Niels</div><div class="citationInfo"><span class="NLM_cas:title">Pharmaceutical Research</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1290-4</span>CODEN:
                <span class="NLM_cas:coden">PHREEB</span>;
        ISSN:<span class="NLM_cas:issn">0724-8741</span>.
    </div><div class="casAbstract">Testosterone has previously been used as a model compd. for the detn. of unstirred water layer thickness in the CACO-2 transport model.  The authors have found, however, that testosterone is metabolized during in vitro transport across the CACO-2 cell monolayers.  This suggests that testosterone is not an ideal model substance.  Testosterone is metabolized to androstenedione, indicating the presence of 17β-hydroxy steroid dehydrogenase in differentiated CACO-2 cells.  No reverse metab. is obsd., so androstenedione is considered superior to testosterone for detn. of unstirred water layer thickness in the CACO-2 system.  Permeability coeffs. for testosterone and androstenedione obtained under identical transport conditions were 66 × 10-6 and 84 × 10-6 cm/s, resp.  The unstirred water layer thicknesses at different agitation rates are detd. for the CACO-2 transport model used in the lab., utilizing androstenedione as a model compd.  The system is capable of controlling the water layer thickness from about 200 to 1000 μm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqimXFucsn9F7Vg90H21EOLACvtfcHk0ljdc0G7Z-UKDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38Xmt1KmtL4%253D&md5=2f33efcf5c8b388b5d2046fe98da8460</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1023%2FA%3A1015805317375&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1015805317375%26sid%3Dliteratum%253Aachs%26aulast%3DBuur%26aufirst%3DA.%26aulast%3DMork%26aufirst%3DN.%26atitle%3DMetabolism%2520of%2520testosterone%2520during%2520in%2520vitro%2520transport%2520across%2520Caco-2%2520cell%2520monolayers%253A%2520evidence%2520for%2520%25CE%25B2-hydroxysteroid%2520dehydrogenase%2520activity%2520in%2520differentiated%2520Caco-2%2520cells%26jtitle%3DPharm.%2520Res.%26date%3D1992%26volume%3D9%26spage%3D1290%26epage%3D1294%26doi%3D10.1023%2FA%3A1015805317375" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berezhkovskiy, L. M.</span></span> <span> </span><span class="NLM_article-title">Some features of the kinetics and equilibrium of drug binding to plasma proteins</span>. <i>Expert Opin. Drug Metab. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">1479</span>– <span class="NLM_lpage">1498</span>, <span class="refDoi"> DOI: 10.1517/17425250802503808</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1517%2F17425250802503808" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=19040325" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVent7zI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=1479-1498&author=L.+M.+Berezhkovskiy&title=Some+features+of+the+kinetics+and+equilibrium+of+drug+binding+to+plasma+proteins&doi=10.1517%2F17425250802503808"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Some features of the kinetics and equilibrium of drug binding to plasma proteins</span></div><div class="casAuthors">Berezhkovskiy, Leonid M.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Metabolism & Toxicology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1479-1498</span>CODEN:
                <span class="NLM_cas:coden">EODMAP</span>;
        ISSN:<span class="NLM_cas:issn">1742-5255</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  The reaction of drug-protein binding is considered based on the common calcns. of chem. equil. (using the mass action and mass balance laws) and formal chem. kinetics.  The calculative and theor. aspects presented in the review are pertinent to routine drug development.  Numerous real-life examples are provided throughout the article to illustrate the practical utility of the presented concepts.  This may be very helpful in the interpretation of protein binding and pharmacokinetic data.  Considerable resources may be saved by the proper setting of protein binding expts., using relatively simple calcns. and estns. instead of doing exptl. measurements, and also avoiding improvements' that are destined to failure.  The presented material may be also useful for the simulations of pharmacokinetics and pharmacodynamics, which attempt the complete account of drug-protein binding.  A complete description of the considered topics is given in the last paragraph of the Introduction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1Bltn8ibfTbVg90H21EOLACvtfcHk0ljdc0G7Z-UKDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVent7zI&md5=f9098016ae1af78e01546ff0ce3648ef</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1517%2F17425250802503808&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17425250802503808%26sid%3Dliteratum%253Aachs%26aulast%3DBerezhkovskiy%26aufirst%3DL.%2BM.%26atitle%3DSome%2520features%2520of%2520the%2520kinetics%2520and%2520equilibrium%2520of%2520drug%2520binding%2520to%2520plasma%2520proteins%26jtitle%3DExpert%2520Opin.%2520Drug%2520Metab.%2520Toxicol.%26date%3D2008%26volume%3D4%26spage%3D1479%26epage%3D1498%26doi%3D10.1517%2F17425250802503808" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zimmermann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmermann-Kogadeeva, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wegmann, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodman, A. L.</span></span> <span> </span><span class="NLM_article-title">Mapping human microbiome drug metabolism by gut bacteria and their genes</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>570</i></span>,  <span class="NLM_fpage">462</span>– <span class="NLM_lpage">467</span>, <span class="refDoi"> DOI: 10.1038/s41586-019-1291-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1038%2Fs41586-019-1291-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=31158845" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1aqtbjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=570&publication_year=2019&pages=462-467&author=M.+Zimmermannauthor=M.+Zimmermann-Kogadeevaauthor=R.+Wegmannauthor=A.+L.+Goodman&title=Mapping+human+microbiome+drug+metabolism+by+gut+bacteria+and+their+genes&doi=10.1038%2Fs41586-019-1291-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Mapping human microbiome drug metabolism by gut bacteria and their genes</span></div><div class="casAuthors">Zimmermann, Michael; Zimmermann-Kogadeeva, Maria; Wegmann, Rebekka; Goodman, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">570</span>
        (<span class="NLM_cas:issue">7762</span>),
    <span class="NLM_cas:pages">462-467</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Individuals vary widely in their responses to medicinal drugs, which can be dangerous and expensive owing to treatment delays and adverse effects.  Although increasing evidence implicates the gut microbiome in this variability, the mol. mechanisms involved remain largely unknown.  Here we show, by measuring the ability of 76 human gut bacteria from diverse clades to metabolize 271 orally administered drugs, that many drugs are chem. modified by microorganisms.  We combined high-throughput genetic analyses with mass spectrometry to systematically identify microbial gene products that metabolize drugs.  These microbiome-encoded enzymes can directly and substantially affect intestinal and systemic drug metab. in mice, and can explain the drug-metabolizing activities of human gut bacteria and communities on the basis of their genomic contents.  These causal links between the gene content and metabolic activities of the microbiota connect interpersonal variability in microbiomes to interpersonal differences in drug metab., which has implications for medical therapy and drug development across multiple disease indications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNgThCVG-l2bVg90H21EOLACvtfcHk0ljdc0G7Z-UKDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1aqtbjN&md5=fc92271e1b2b7bad9c2396ac9cf7517e</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1038%2Fs41586-019-1291-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-019-1291-3%26sid%3Dliteratum%253Aachs%26aulast%3DZimmermann%26aufirst%3DM.%26aulast%3DZimmermann-Kogadeeva%26aufirst%3DM.%26aulast%3DWegmann%26aufirst%3DR.%26aulast%3DGoodman%26aufirst%3DA.%2BL.%26atitle%3DMapping%2520human%2520microbiome%2520drug%2520metabolism%2520by%2520gut%2520bacteria%2520and%2520their%2520genes%26jtitle%3DNature%26date%3D2019%26volume%3D570%26spage%3D462%26epage%3D467%26doi%3D10.1038%2Fs41586-019-1291-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Škarydová, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wsól, V.</span></span> <span> </span><span class="NLM_article-title">Human microsomal carbonyl reducing enzymes in the metabolism of xenobiotics: well-known and promising members of the SDR superfamily</span>. <i>Drug Metab. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">173</span>– <span class="NLM_lpage">191</span>, <span class="refDoi"> DOI: 10.3109/03602532.2011.638304</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.3109%2F03602532.2011.638304" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=22181347" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC38Xls1Whsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2012&pages=173-191&author=L.+%C5%A0karydov%C3%A1author=V.+Ws%C3%B3l&title=Human+microsomal+carbonyl+reducing+enzymes+in+the+metabolism+of+xenobiotics%3A+well-known+and+promising+members+of+the+SDR+superfamily&doi=10.3109%2F03602532.2011.638304"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Human microsomal carbonyl reducing enzymes in the metabolism of xenobiotics: well-known and promising members of the SDR superfamily</span></div><div class="casAuthors">Skarydova, Lucie; Wsol, Vladimir</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism Reviews</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">173-191</span>CODEN:
                <span class="NLM_cas:coden">DMTRAR</span>;
        ISSN:<span class="NLM_cas:issn">0360-2532</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  The best known, most widely studied enzyme system in phase I biotransformation is cytochrome P 450 (CYP), which participates in the metab. of roughly 9 of 10 drugs in use today.  The main biotransformation isoforms of CYP are assocd. with the membrane of the endoplasmatic reticulum (ER).  Other enzymes that are also active in phase I biotransformation are carbonyl reducing enzymes.  Much is known about the role of cytosolic forms of carbonyl reducing enzymes in the metab. of xenobiotics, but their microsomal forms have been mostly poorly studied.  The only well-known microsomal carbonyl reducing enzyme taking part in the biotransformation of xenobiotics is 11β-hydroxysteroid dehydrogenase 1, a member of the short-chain dehydrogenase/reductase superfamily.  Physiol. roles of microsomal carbonyl reducing enzymes are better known than their participation in the metab. of xenobiotics.  This review is a summary of the fragmentary information known about the roles of the microsomal forms.  Besides 11β-hydroxysteroid dehydrogenase 1, it has been reported, so far, that retinol dehydrogenase 12 participates only in the detoxification of unsatd. aldehydes formed upon oxidative stress.  Another promising group of microsomal biotransformation carbonyl reducing enzymes are some members of 17β-hydroxysteroid dehydrogenases.  Generally, it is clear that this area is, overall, quite unexplored, but carbonyl reducing enzymes located in the ER have proven very interesting.  The study of these enzymes could shed new light on the metab. of several clin. used drugs or they could become an important target in connection with some diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUT5DSEsK3C7Vg90H21EOLACvtfcHk0ljHlqnowTKhXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xls1Whsb0%253D&md5=8265d72837c0fa0265fb7f686fa5c738</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.3109%2F03602532.2011.638304&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F03602532.2011.638304%26sid%3Dliteratum%253Aachs%26aulast%3D%25C5%25A0karydov%25C3%25A1%26aufirst%3DL.%26aulast%3DWs%25C3%25B3l%26aufirst%3DV.%26atitle%3DHuman%2520microsomal%2520carbonyl%2520reducing%2520enzymes%2520in%2520the%2520metabolism%2520of%2520xenobiotics%253A%2520well-known%2520and%2520promising%2520members%2520of%2520the%2520SDR%2520superfamily%26jtitle%3DDrug%2520Metab.%2520Rev.%26date%3D2012%26volume%3D44%26spage%3D173%26epage%3D191%26doi%3D10.3109%2F03602532.2011.638304" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adler, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhles, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stamm, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairhurst, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiffe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sterker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centeleghe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wartmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kinyamu-Akunda, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schadt, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couttet, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barzaghi-Rinaudo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murakami, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kauffmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knoepfel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buschmann, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leblanc, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mah, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furet, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blank, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graus Porta, D.</span></span> <span> </span><span class="NLM_article-title">FGF401, a first-in-class highly selective and potent FGFR4 inhibitor for the treatment of FGF19-driven hepatocellular cancer</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">2194</span>– <span class="NLM_lpage">2206</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-18-1291</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1158%2F1535-7163.MCT-18-1291" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=31409633" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BB3cXktFOit7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=2194-2206&author=A.+Weissauthor=F.+Adlerauthor=A.+Buhlesauthor=C.+Stammauthor=R.+A.+Fairhurstauthor=M.+Kiffeauthor=D.+Sterkerauthor=M.+Centelegheauthor=M.+Wartmannauthor=J.+Kinyamu-Akundaauthor=H.+S.+Schadtauthor=P.+Couttetauthor=A.+Wolfauthor=Y.+Wangauthor=P.+Barzaghi-Rinaudoauthor=M.+Murakamiauthor=A.+Kauffmannauthor=T.+Knoepfelauthor=N.+Buschmannauthor=C.+Leblancauthor=R.+Mahauthor=P.+Furetauthor=J.+Blankauthor=F.+Hofmannauthor=W.+R.+Sellersauthor=D.+Graus+Porta&title=FGF401%2C+a+first-in-class+highly+selective+and+potent+FGFR4+inhibitor+for+the+treatment+of+FGF19-driven+hepatocellular+cancer&doi=10.1158%2F1535-7163.MCT-18-1291"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">FGF401, a first-in-class highly selective and potent FGFR4 inhibitor for the treatment of FGF19-driven hepatocellular cancer</span></div><div class="casAuthors">Weiss, Andreas; Adler, Flavia; Buhles, Alexandra; Stamm, Christelle; Fairhurst, Robin A.; Kiffe, Michael; Sterker, Dario; Centeleghe, Mario; Wartmann, Markus; Kinyamu-Akunda, Jacqueline; Schadt, Heiko S.; Couttet, Philippe; Wolf, Armin; Wang, Youzhen; Barzaghi-Rinaudo, Patrizia; Murakami, Masato; Kauffmann, Audrey; Knoepfel, Thomas; Buschmann, Nicole; Leblanc, Catherine; Mah, Robert; Furet, Pascal; Blank, Jutta; Hofmann, Francesco; Sellers, William R.; Porta, Diana Graus</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2194-2206</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Hepatocellular carcinoma (HCC) is the most common primary malignancy of the liver and it is the third leading cause of cancer-related deaths worldwide.  Recently, aberrant signaling through the FGF19/FGFR4 axis has been implicated in HCC.  Here, we describe the development of FGF401, a highly potent and selective, first in class, reversible-covalent small-mol. inhibitor of the kinase activity of FGFR4.  FGF401 is exquisitely selective for FGFR4 vs. the other FGFR paralogues FGFR1, FGFR2, FGFR3, and all other kinases in the kinome.  FGF401 has excellent drug-like properties showing a robust pharmacokinetic/pharmacodynamics/efficacy relationship, driven by a fraction of time above the phospho-FGFR4 IC90 value.  FGF401 has remarkable antitumor activity in mice bearing HCC tumor xenografts and patient-derived xenograft models that are pos. for FGF19, FGFR4, and KLB.  FGF401 is the first FGFR4 inhibitor to enter clin. trials, and a phase I/II study is currently ongoing in HCC and other solid malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBspWk4KWyurVg90H21EOLACvtfcHk0ljHlqnowTKhXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXktFOit7Y%253D&md5=81a3b04d779295a9c9a037cf5ca73fca</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-18-1291&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-18-1291%26sid%3Dliteratum%253Aachs%26aulast%3DWeiss%26aufirst%3DA.%26aulast%3DAdler%26aufirst%3DF.%26aulast%3DBuhles%26aufirst%3DA.%26aulast%3DStamm%26aufirst%3DC.%26aulast%3DFairhurst%26aufirst%3DR.%2BA.%26aulast%3DKiffe%26aufirst%3DM.%26aulast%3DSterker%26aufirst%3DD.%26aulast%3DCenteleghe%26aufirst%3DM.%26aulast%3DWartmann%26aufirst%3DM.%26aulast%3DKinyamu-Akunda%26aufirst%3DJ.%26aulast%3DSchadt%26aufirst%3DH.%2BS.%26aulast%3DCouttet%26aufirst%3DP.%26aulast%3DWolf%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DBarzaghi-Rinaudo%26aufirst%3DP.%26aulast%3DMurakami%26aufirst%3DM.%26aulast%3DKauffmann%26aufirst%3DA.%26aulast%3DKnoepfel%26aufirst%3DT.%26aulast%3DBuschmann%26aufirst%3DN.%26aulast%3DLeblanc%26aufirst%3DC.%26aulast%3DMah%26aufirst%3DR.%26aulast%3DFuret%26aufirst%3DP.%26aulast%3DBlank%26aufirst%3DJ.%26aulast%3DHofmann%26aufirst%3DF.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26aulast%3DGraus%2BPorta%26aufirst%3DD.%26atitle%3DFGF401%252C%2520a%2520first-in-class%2520highly%2520selective%2520and%2520potent%2520FGFR4%2520inhibitor%2520for%2520the%2520treatment%2520of%2520FGF19-driven%2520hepatocellular%2520cancer%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2019%26volume%3D18%26spage%3D2194%26epage%3D2206%26doi%3D10.1158%2F1535-7163.MCT-18-1291" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arnott, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lobo Planey, S.</span></span> <span> </span><span class="NLM_article-title">The influence of lipophilicity in drug discovery and design</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">863</span>– <span class="NLM_lpage">875</span>, <span class="refDoi"> DOI: 10.1517/17460441.2012.714363</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1517%2F17460441.2012.714363" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=22992175" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhtl2lsbvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=863-875&author=J.+A.+Arnottauthor=S.+Lobo+Planey&title=The+influence+of+lipophilicity+in+drug+discovery+and+design&doi=10.1517%2F17460441.2012.714363"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">The influence of lipophilicity in drug discovery and design</span></div><div class="casAuthors">Arnott, John A.; Planey, Sonia Lobo</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">863-875</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Introduction: The role of lipophilicity in drug discovery and design is a crit. one.  Lipophilicity is a key physicochem. property that plays a crucial role in detg. ADMET (absorption, distribution, metab., excretion, and toxicity) properties and the overall suitability of drug candidates.  There is increasing evidence to suggest that control of physicochem. properties such as lipophilicity, within a defined optimal range, can improve compd. quality and the likelihood of therapeutic success.Areas covered: This review focuses on understanding lipophilicity, techniques used to measure lipophilicity, and summarizes the importance of lipophilicity in drug discovery and development, including a discussion of its impact on individual ADMET parameters as well as its overall influence on the drug discovery and design process, specifically within the past 15 years.Expert opinion: A current review of the literature reveals a continued reliance on the synthesis of novel structures with increased potency, rather than a focus on maintaining optimal physicochem. properties assocd. with ADMET throughout drug optimization.  Particular attention to the optimum region of lipophilicity, as well as monitoring of lipophilic efficiency indexes, may contribute significantly to the overall quality of candidate drugs at different stages of discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolWJgvv9yK87Vg90H21EOLACvtfcHk0ligKHugcPX-Vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhtl2lsbvJ&md5=4dc5d7268216c9a59fcbd841fbf66735</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1517%2F17460441.2012.714363&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.2012.714363%26sid%3Dliteratum%253Aachs%26aulast%3DArnott%26aufirst%3DJ.%2BA.%26aulast%3DLobo%2BPlaney%26aufirst%3DS.%26atitle%3DThe%2520influence%2520of%2520lipophilicity%2520in%2520drug%2520discovery%2520and%2520design%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2012%26volume%3D7%26spage%3D863%26epage%3D875%26doi%3D10.1517%2F17460441.2012.714363" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yalkowsky, S. H.</span></span> <span> </span><span class="NLM_article-title">An interesting relationship between drug absorption and melting point</span>. <i>Int. J. Pharm.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>373</i></span>,  <span class="NLM_fpage">24</span>– <span class="NLM_lpage">40</span>, <span class="refDoi"> DOI: 10.1016/j.ijpharm.2009.01.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1016%2Fj.ijpharm.2009.01.026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=19429285" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BD1MXltlSjtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=373&publication_year=2009&pages=24-40&author=K.+A.+Chuauthor=S.+H.+Yalkowsky&title=An+interesting+relationship+between+drug+absorption+and+melting+point&doi=10.1016%2Fj.ijpharm.2009.01.026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54aR"><div class="casContent"><span class="casTitleNuber">54a</span><div class="casTitle"><span class="NLM_cas:atitle">An interesting relationship between drug absorption and melting point</span></div><div class="casAuthors">Chu, Katherine A.; Yalkowsky, Samuel H.</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Pharmaceutics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">373</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">24-40</span>CODEN:
                <span class="NLM_cas:coden">IJPHDE</span>;
        ISSN:<span class="NLM_cas:issn">0378-5173</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The ability to predict the extent of passive intestinal drug absorption is very important for efficient lead candidate selection and development.  Physicochem.-based absorption predictive models previously developed use soly., partition coeff. and pKa as drug input parameters for intestinal absorption.  Alternatively, this study looks at the relationship between m.p. and passive transport for poorly sol. drugs.  It is based entirely on the expression derived from the general soly. equation (GSE) that relates m.p. to the product of intrinsic soly. and partition coeff.  Given that the m.p. of a compd. is one of the first and more reliable phys. properties measured, it can be advantageously used as a guide in early drug discovery and development.  This paper elucidates the interesting relationship between the m.p. and dose to the fraction absorbed of poorly sol. drugs, i.e., class II and IV compds. in the biopharmaceutics classification system.  The newly defined m.p. based absorption potential (MPbAP) parameter is successful at distinguishing 90% of the 91 drugs considered being well absorbed (FA > 0.5) or poorly absorbed.  In general, lower melting compds. are more likely to be well absorbed than higher melting compds. for any given dose.  The fraction absorbed for drugs with high melting temps. is limited by the dose to a greater degree than it is for low melting compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLyfNgwnwHI7Vg90H21EOLACvtfcHk0ligKHugcPX-Vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXltlSjtrc%253D&md5=6ce9ff030bfc5c5c20c630ee18e502c3</span></div><a href="/servlet/linkout?suffix=cit54a&amp;dbid=16384&amp;doi=10.1016%2Fj.ijpharm.2009.01.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ijpharm.2009.01.026%26sid%3Dliteratum%253Aachs%26aulast%3DChu%26aufirst%3DK.%2BA.%26aulast%3DYalkowsky%26aufirst%3DS.%2BH.%26atitle%3DAn%2520interesting%2520relationship%2520between%2520drug%2520absorption%2520and%2520melting%2520point%26jtitle%3DInt.%2520J.%2520Pharm.%26date%3D2009%26volume%3D373%26spage%3D24%26epage%3D40%26doi%3D10.1016%2Fj.ijpharm.2009.01.026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit54b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, V. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhattachar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hitchingham, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zocharski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naath, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Surendran, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoner, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Kattan, A.</span></span> <span> </span><span class="NLM_article-title">The road map to oral bioavailability: an industrial perspective</span>. <i>Expert Opin. Drug Metab. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">591</span>– <span class="NLM_lpage">608</span>, <span class="refDoi"> DOI: 10.1517/17425255.2.4.591</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1517%2F17425255.2.4.591" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=16859407" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BD28Xnt1ahsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2006&pages=591-608&author=V.+H.+Thomasauthor=S.+Bhattacharauthor=L.+Hitchinghamauthor=P.+Zocharskiauthor=M.+Naathauthor=N.+Surendranauthor=C.+L.+Stonerauthor=A.+El-Kattan&title=The+road+map+to+oral+bioavailability%3A+an+industrial+perspective&doi=10.1517%2F17425255.2.4.591"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54bR"><div class="casContent"><span class="casTitleNuber">54b</span><div class="casTitle"><span class="NLM_cas:atitle">The road map to oral bioavailability: an industrial perspective</span></div><div class="casAuthors">Thomas, V. Hayden; Bhattachar, Shobha; Hitchingham, Linda; Zocharski, Philip; Naath, Maryanne; Surendran, Narayanan; Stoner, Chad L.; El-Kattan, Ayman</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Metabolism & Toxicology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">591-608</span>CODEN:
                <span class="NLM_cas:coden">EODMAP</span>;
        ISSN:<span class="NLM_cas:issn">1742-5255</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Optimization of oral bioavailability is a continuing challenge for the pharmaceutical and biotechnol. industries.  The no. of potential drug candidates requiring in vivo evaluation has significantly increased with the advent of combinatorial chem.  In addn., drug discovery programs are increasingly forced into more lipophilic and lower soly. chem. space.  To aid in the use of in vitro and in silico tools as well as reduce the no. of in vivo studies required, a team-based discussion tool is proposed that provides a road map' to guide the selection of profiling assays that should be considered when optimizing oral bioavailability.  This road map divides the factors that contribute to poor oral bioavailability into two interrelated categories: absorption and metab.  This road map provides an interface for cross discipline discussions and a systematic approach to the experimentation that drives the drug discovery process towards a common goal - acceptable oral bioavailability using minimal resources in an acceptable time frame.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrh7_zZKacXE7Vg90H21EOLACvtfcHk0ligKHugcPX-Vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xnt1ahsLs%253D&md5=79cf4cea1b89914bd3a8826b5d7360f9</span></div><a href="/servlet/linkout?suffix=cit54b&amp;dbid=16384&amp;doi=10.1517%2F17425255.2.4.591&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17425255.2.4.591%26sid%3Dliteratum%253Aachs%26aulast%3DThomas%26aufirst%3DV.%2BH.%26aulast%3DBhattachar%26aufirst%3DS.%26aulast%3DHitchingham%26aufirst%3DL.%26aulast%3DZocharski%26aufirst%3DP.%26aulast%3DNaath%26aufirst%3DM.%26aulast%3DSurendran%26aufirst%3DN.%26aulast%3DStoner%26aufirst%3DC.%2BL.%26aulast%3DEl-Kattan%26aufirst%3DA.%26atitle%3DThe%2520road%2520map%2520to%2520oral%2520bioavailability%253A%2520an%2520industrial%2520perspective%26jtitle%3DExpert%2520Opin.%2520Drug%2520Metab.%2520Toxicol.%26date%3D2006%26volume%3D2%26spage%3D591%26epage%3D608%26doi%3D10.1517%2F17425255.2.4.591" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><a href="https://www.discoverx.com/services/drug-discovery-development-services/kinase-profiling/kinomescan" class="extLink">https://www.discoverx.com/services/drug-discovery-development-services/kinase-profiling/kinomescan</a> (accessed Sep 14, <span class="NLM_year">2020</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=https%3A%2F%2Fwww.discoverx.com%2Fservices%2Fdrug-discovery-development-services%2Fkinase-profiling%2Fkinomescan+%28accessed+Sep+14%2C+2020%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Veber, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, H.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kopple, K. D.</span></span> <span> </span><span class="NLM_article-title">Molecular properties that influence the oral bioavailability of drug candidates</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">2615</span>– <span class="NLM_lpage">2623</span>, <span class="refDoi"> DOI: 10.1021/jm020017n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm020017n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BD38XjsFCmt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=2615-2623&author=D.+F.+Veberauthor=S.+R.+Johnsonauthor=H.-Y.+Chengauthor=B.+R.+Smithauthor=K.+W.+Wardauthor=K.+D.+Kopple&title=Molecular+properties+that+influence+the+oral+bioavailability+of+drug+candidates&doi=10.1021%2Fjm020017n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Properties That Influence the Oral Bioavailability of Drug Candidates</span></div><div class="casAuthors">Veber, Daniel F.; Johnson, Stephen R.; Cheng, Hung-Yuan; Smith, Brian R.; Ward, Keith W.; Kopple, Kenneth D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2615-2623</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Oral bioavailability measurements in rats for over 1100 drug candidates studied at Smith-Kline Beecham Pharmaceuticals (now Glaxo Smith-Kline) have allowed us to analyze the relative importance of mol. properties considered to influence that drug property.  Reduced mol. flexibility, as measured by the no. of rotatable bonds, and low polar surface area or total hydrogen bond count (sum of donors and acceptors) are found to be important predictors of good oral bioavailability, independent of mol. wt.  That on av. both the no. of rotatable bonds and polar surface area or hydrogen bond count tend to increase with mol. wt. may in part explain the success of the mol. wt. parameter in predicting oral bioavailability.  The commonly applied mol. wt. cutoff at 500 does not itself significantly sep. compds. with poor oral bioavailability from those with acceptable values in this extensive data set.  Our observations suggest that compds. which meet only the 2 criteria of (1) 10 or fewer rotatable bonds and (2) polar surface area ≤140 Å2 (or 12 or fewer H-bond donors and acceptors) will have a high probability of good oral bioavailability in the rat.  Data sets for the artificial membrane permeation rate and for clearance in the rat were also examd.  Reduced polar surface area correlates better with increased permeation rate than does lipophilicity (C log P), and increased rotatable bond count has a neg. effect on the permeation rate.  A threshold permeation rate is a prerequisite of oral bioavailability.  The rotatable bond count does not correlate with the data examd. here for the in vivo clearance rate in the rat.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3JtJEkO7yVLVg90H21EOLACvtfcHk0lh5ihJSoEWSZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjsFCmt7g%253D&md5=eaad26ed6a259de82ad65a8834fc397d</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Fjm020017n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm020017n%26sid%3Dliteratum%253Aachs%26aulast%3DVeber%26aufirst%3DD.%2BF.%26aulast%3DJohnson%26aufirst%3DS.%2BR.%26aulast%3DCheng%26aufirst%3DH.-Y.%26aulast%3DSmith%26aufirst%3DB.%2BR.%26aulast%3DWard%26aufirst%3DK.%2BW.%26aulast%3DKopple%26aufirst%3DK.%2BD.%26atitle%3DMolecular%2520properties%2520that%2520influence%2520the%2520oral%2520bioavailability%2520of%2520drug%2520candidates%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D2615%26epage%3D2623%26doi%3D10.1021%2Fjm020017n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Docherty, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pencheva, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abramov, Y. A.</span></span> <span> </span><span class="NLM_article-title">Low solubility in drug development: de-convoluting the relative importance of solvation and crystal packing</span>. <i>J. Pharm. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">847</span>– <span class="NLM_lpage">856</span>, <span class="refDoi"> DOI: 10.1111/jphp.12393</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1111%2Fjphp.12393" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=25880016" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmsFersLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2015&pages=847-856&author=R.+Dochertyauthor=K.+Penchevaauthor=Y.+A.+Abramov&title=Low+solubility+in+drug+development%3A+de-convoluting+the+relative+importance+of+solvation+and+crystal+packing&doi=10.1111%2Fjphp.12393"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57aR"><div class="casContent"><span class="casTitleNuber">57a</span><div class="casTitle"><span class="NLM_cas:atitle">Low solubility in drug development: de-convoluting the relative importance of solvation and crystal packing</span></div><div class="casAuthors">Docherty, Robert; Pencheva, Klimentina; Abramov, Yuriy A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacy and Pharmacology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">847-856</span>CODEN:
                <span class="NLM_cas:coden">JPPMAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3573</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">Objectives : An increasing trend towards low soly. is a major issue for drug development as formulation of low soly. compds. can be problematic.  This paper presents a model which de-convolutes the soly. of pharmaceutical compds. into solvation and packing properties with the intention to understand the soly. limiting features.  Methods : The Cambridge Crystallog. Database was the source of structural information.  Lattice energies were calcd. via force-field based approaches using Materials Studio.  The solvation energies were calcd. applying quantum chem. models using Cosmotherm software.  Key findings : The solubilities of 54 drug-like compds. were mapped onto a solvation energy/crystal packing grid.  Four quadrants were identified were different balances of solvation and packing were defining the soly.  A version of the model was developed which allows for the calcn. of the two features even in absence of crystal structure.  Conclusion : Although there are significant no. of in-silico models, it has been proven very difficult to predict aq. soly. accurately.  Therefore, we have taken a different approach where the soly. is not predicted directly but is de-convoluted into two constituent features.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSkolyShKsU7Vg90H21EOLACvtfcHk0lh5ihJSoEWSZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmsFersLg%253D&md5=a74b378d2f71167a8af1d1c142c92c53</span></div><a href="/servlet/linkout?suffix=cit57a&amp;dbid=16384&amp;doi=10.1111%2Fjphp.12393&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fjphp.12393%26sid%3Dliteratum%253Aachs%26aulast%3DDocherty%26aufirst%3DR.%26aulast%3DPencheva%26aufirst%3DK.%26aulast%3DAbramov%26aufirst%3DY.%2BA.%26atitle%3DLow%2520solubility%2520in%2520drug%2520development%253A%2520de-convoluting%2520the%2520relative%2520importance%2520of%2520solvation%2520and%2520crystal%2520packing%26jtitle%3DJ.%2520Pharm.%2520Pharmacol.%26date%3D2015%26volume%3D67%26spage%3D847%26epage%3D856%26doi%3D10.1111%2Fjphp.12393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit57b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Lovering, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bikker, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humblet, C.</span></span> <span> </span><span class="NLM_article-title">Escape from flatland: increasing saturation as an approach to improving clinical success</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">6752</span>– <span class="NLM_lpage">6756</span>, <span class="refDoi"> DOI: 10.1021/jm901241e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901241e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1KjtLvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=6752-6756&author=F.+Loveringauthor=J.+Bikkerauthor=C.+Humblet&title=Escape+from+flatland%3A+increasing+saturation+as+an+approach+to+improving+clinical+success&doi=10.1021%2Fjm901241e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57bR"><div class="casContent"><span class="casTitleNuber">57b</span><div class="casTitle"><span class="NLM_cas:atitle">Escape from Flatland: Increasing Saturation as an Approach to Improving Clinical Success</span></div><div class="casAuthors">Lovering, Frank; Bikker, Jack; Humblet, Christine</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6752-6756</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The medicinal chem. community has become increasingly aware of the value of tracking calcd. phys. properties such as mol. wt., topol. polar surface area, rotatable bonds, and hydrogen bond donors and acceptors.  The authors hypothesized that the shift to high-throughput synthetic practices over the past decade may be another factor that may predispose mols. to fail by steering discovery efforts toward achiral, arom. compds.  The authors have proposed two simple and interpretable measures of the complexity of mols. prepd. as potential drug candidates.  The first is carbon bond satn. as defined by fraction Sp3 (Fsp3) where Fsp3 = (no. of Sp3 hybridized carbons/total carbon count).  The second is simply whether a chiral carbon exists in the mol.  The authors demonstrate that both complexity (as measured by Fsp3) and the presence of chiral centers correlate with success as compds. transition from discovery, through clin. testing, to drugs.  To explain these observations, the authors further demonstrate that satn. correlates with soly., an exptl. phys. property important to success in the drug discovery setting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoC1f3Ed7nUBbVg90H21EOLACvtfcHk0lh5ihJSoEWSZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1KjtLvN&md5=4ca92c30c17c53d77ad376719bad951e</span></div><a href="/servlet/linkout?suffix=cit57b&amp;dbid=16384&amp;doi=10.1021%2Fjm901241e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901241e%26sid%3Dliteratum%253Aachs%26aulast%3DLovering%26aufirst%3DF.%26aulast%3DBikker%26aufirst%3DJ.%26aulast%3DHumblet%26aufirst%3DC.%26atitle%3DEscape%2520from%2520flatland%253A%2520increasing%2520saturation%2520as%2520an%2520approach%2520to%2520improving%2520clinical%2520success%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D6752%26epage%3D6756%26doi%3D10.1021%2Fjm901241e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit57c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Wassvik, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmén, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Draheim, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artursson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergström, C. A. S.</span></span> <span> </span><span class="NLM_article-title">Molecular characteristics for solid-state limited solubility</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">3035</span>– <span class="NLM_lpage">3039</span>, <span class="refDoi"> DOI: 10.1021/jm701587d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm701587d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BD1cXksVSgt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=3035-3039&author=C.+M.+Wassvikauthor=A.+G.+Holm%C3%A9nauthor=R.+Draheimauthor=P.+Arturssonauthor=C.+A.+S.+Bergstr%C3%B6m&title=Molecular+characteristics+for+solid-state+limited+solubility&doi=10.1021%2Fjm701587d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57cR"><div class="casContent"><span class="casTitleNuber">57c</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Characteristics for Solid-State Limited Solubility</span></div><div class="casAuthors">Wassvik, Carola M.; Holmen, Anders G.; Draheim, Rieke; Artursson, Per; Bergstroem, Christel A. S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3035-3039</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Soly. and solid-state characteristics were detd. and multivariate data anal. was used to deduce structural features important for solid-state limited soly. of marketed drugs.  Mols. with extended ring structures and large conjugated systems were less sol., indicating that structural features related to rigidity and aromaticity result in soly. restricted by stable crystal structures.  These descriptors successfully predicted the applied test set and can be useful for avoiding synthesis of compds. behaving like "brick dust".</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4YD2_Ul4qabVg90H21EOLACvtfcHk0lieMZNNh9B2Kg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXksVSgt7o%253D&md5=f27982a173dfbc1491b6ce1749f486a8</span></div><a href="/servlet/linkout?suffix=cit57c&amp;dbid=16384&amp;doi=10.1021%2Fjm701587d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701587d%26sid%3Dliteratum%253Aachs%26aulast%3DWassvik%26aufirst%3DC.%2BM.%26aulast%3DHolm%25C3%25A9n%26aufirst%3DA.%2BG.%26aulast%3DDraheim%26aufirst%3DR.%26aulast%3DArtursson%26aufirst%3DP.%26aulast%3DBergstr%25C3%25B6m%26aufirst%3DC.%2BA.%2BS.%26atitle%3DMolecular%2520characteristics%2520for%2520solid-state%2520limited%2520solubility%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D3035%26epage%3D3039%26doi%3D10.1021%2Fjm701587d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tayl, J. G.,  VI</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheuk, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsang, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, J.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Youngblood, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeung, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helman, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendoza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ngo, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staudt, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khanna, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Catchpoole, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qualman, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merlino, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chanock, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, J.</span></span> <span> </span><span class="NLM_article-title">Identification of FGFR4-activating mutations in human rhabdomyosarcomas that promote metastasis in xenotransplanted models</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">3395</span>– <span class="NLM_lpage">3407</span>, <span class="refDoi"> DOI: 10.1172/JCI39703</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1172%2FJCI39703" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=19809159" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2009&pages=3395-3407&author=J.+G.+Taylauthor=A.+T.+Cheukauthor=P.+S.+Tsangauthor=J.-Y.+Chungauthor=Y.+K.+Songauthor=K.+Desaiauthor=Y.+Yuauthor=Q.-R.+Chenauthor=K.+Shahauthor=V.+Youngbloodauthor=J.+Fangauthor=S.+Y.+Kimauthor=C.+Yeungauthor=L.+J.+Helmanauthor=A.+Mendozaauthor=V.+Ngoauthor=L.+M.+Staudtauthor=J.+S.+Weiauthor=C.+Khannaauthor=D.+Catchpooleauthor=S.+J.+Qualmanauthor=S.+M.+Hewittauthor=G.+Merlinoauthor=S.+J.+Chanockauthor=J.+Khan&title=Identification+of+FGFR4-activating+mutations+in+human+rhabdomyosarcomas+that+promote+metastasis+in+xenotransplanted+models&doi=10.1172%2FJCI39703"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1172%2FJCI39703&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI39703%26sid%3Dliteratum%253Aachs%26aulast%3DTayl%26aufirst%3DJ.%2BG.%26aulast%3DCheuk%26aufirst%3DA.%2BT.%26aulast%3DTsang%26aufirst%3DP.%2BS.%26aulast%3DChung%26aufirst%3DJ.-Y.%26aulast%3DSong%26aufirst%3DY.%2BK.%26aulast%3DDesai%26aufirst%3DK.%26aulast%3DYu%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DQ.-R.%26aulast%3DShah%26aufirst%3DK.%26aulast%3DYoungblood%26aufirst%3DV.%26aulast%3DFang%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DS.%2BY.%26aulast%3DYeung%26aufirst%3DC.%26aulast%3DHelman%26aufirst%3DL.%2BJ.%26aulast%3DMendoza%26aufirst%3DA.%26aulast%3DNgo%26aufirst%3DV.%26aulast%3DStaudt%26aufirst%3DL.%2BM.%26aulast%3DWei%26aufirst%3DJ.%2BS.%26aulast%3DKhanna%26aufirst%3DC.%26aulast%3DCatchpoole%26aufirst%3DD.%26aulast%3DQualman%26aufirst%3DS.%2BJ.%26aulast%3DHewitt%26aufirst%3DS.%2BM.%26aulast%3DMerlino%26aufirst%3DG.%26aulast%3DChanock%26aufirst%3DS.%2BJ.%26aulast%3DKhan%26aufirst%3DJ.%26atitle%3DIdentification%2520of%2520FGFR4-activating%2520mutations%2520in%2520human%2520rhabdomyosarcomas%2520that%2520promote%2520metastasis%2520in%2520xenotransplanted%2520models%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2009%26volume%3D119%26spage%3D3395%26epage%3D3407%26doi%3D10.1172%2FJCI39703" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hatlen, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt-Kittler, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherwin, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rozsahegyi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheets, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miduturu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bifulco, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooijmans, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boral, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lengauer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorsch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, Y.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, B. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoeflich, K. P.</span></span> <span> </span><span class="NLM_article-title">Acquired on-target clinical resistance validates FGFR4 as a driver of hepatocellular carcinoma</span>. <i>Cancer Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1686</span>– <span class="NLM_lpage">1695</span>, <span class="refDoi"> DOI: 10.1158/2159-8290.CD-19-0367</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1158%2F2159-8290.CD-19-0367" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=31575540" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1Ght77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=1686-1695&author=M.+A.+Hatlenauthor=O.+Schmidt-Kittlerauthor=C.+A.+Sherwinauthor=E.+Rozsahegyiauthor=N.+Rubinauthor=M.+P.+Sheetsauthor=J.+L.+Kimauthor=C.+Miduturuauthor=N.+Bifulcoauthor=N.+Brooijmansauthor=H.+Shiauthor=T.+Guziauthor=A.+Boralauthor=C.+Lengauerauthor=M.+Dorschauthor=R.+D.+Kimauthor=Y.-K.+Kangauthor=B.+B.+Wolfauthor=K.+P.+Hoeflich&title=Acquired+on-target+clinical+resistance+validates+FGFR4+as+a+driver+of+hepatocellular+carcinoma&doi=10.1158%2F2159-8290.CD-19-0367"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Acquired on-target clinical resistance validates FGFR4 as a driver of hepatocellular carcinoma</span></div><div class="casAuthors">Hatlen, Megan A.; Schmidt-Kittler, Oleg; Sherwin, Cori Ann; Rozsahegyi, Emily; Rubin, Nooreen; Sheets, Michael P.; Kim, Joseph L.; Miduturu, Chandrasekhar; Bifulco, Neil; Brooijmans, Natasja; Shi, Hongliang; Guzi, Timothy; Boral, Andy; Lengauer, Christoph; Dorsch, Marion; Kim, Richard D.; Kang, Yoon-Koo; Wolf, Beni B.; Hoeflich, Klaus P.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1686-1695</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Hepatocellular carcinoma (HCC) is a leading cause of cancer mortality worldwide with no clin. confirmed oncogenic driver.  Although preclin. studies implicate the FGF19 receptor FGFR4 in hepatocarcinogenesis, the dependence of human cancer on FGFR4 has not been demonstrated.  Fisogatinib (BLU-554) is a potent and selective inhibitor of FGFR4 and demonstrates clin. benefit and tumor regression in patients with HCC with aberrant FGF19 expression.  Mutations were identified in the gatekeeper and hinge-1 residues in the kinase domain of FGFR4 upon disease progression in 2 patients treated with fisogatinib, which were confirmed to mediate resistance in vitro and in vivo.  A gatekeeper-agnostic, pan-FGFR inhibitor decreased HCC xenograft growth in the presence of these mutations, demonstrating continued FGF19-FGFR4 pathway dependence.  These results validate FGFR4 as an oncogenic driver and warrant further therapeutic targeting of this kinase in the clinic.  SIGNIFICANCE: Our study is the first to demonstrate on-target FGFR4 kinase domain mutations as a mechanism of acquired clin. resistance to targeted therapy.  This further establishes FGF19-FGFR4 pathway activation as an oncogenic driver.  These findings support further investigation of fisogatinib in HCC and inform the profile of potential next-generation inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqO2LPyX4ffo7Vg90H21EOLACvtfcHk0lieMZNNh9B2Kg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1Ght77F&md5=8d5ac6ec96fb9c179635f2b4a31b990f</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-19-0367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-19-0367%26sid%3Dliteratum%253Aachs%26aulast%3DHatlen%26aufirst%3DM.%2BA.%26aulast%3DSchmidt-Kittler%26aufirst%3DO.%26aulast%3DSherwin%26aufirst%3DC.%2BA.%26aulast%3DRozsahegyi%26aufirst%3DE.%26aulast%3DRubin%26aufirst%3DN.%26aulast%3DSheets%26aufirst%3DM.%2BP.%26aulast%3DKim%26aufirst%3DJ.%2BL.%26aulast%3DMiduturu%26aufirst%3DC.%26aulast%3DBifulco%26aufirst%3DN.%26aulast%3DBrooijmans%26aufirst%3DN.%26aulast%3DShi%26aufirst%3DH.%26aulast%3DGuzi%26aufirst%3DT.%26aulast%3DBoral%26aufirst%3DA.%26aulast%3DLengauer%26aufirst%3DC.%26aulast%3DDorsch%26aufirst%3DM.%26aulast%3DKim%26aufirst%3DR.%2BD.%26aulast%3DKang%26aufirst%3DY.-K.%26aulast%3DWolf%26aufirst%3DB.%2BB.%26aulast%3DHoeflich%26aufirst%3DK.%2BP.%26atitle%3DAcquired%2520on-target%2520clinical%2520resistance%2520validates%2520FGFR4%2520as%2520a%2520driver%2520of%2520hepatocellular%2520carcinoma%26jtitle%3DCancer%2520Discovery%26date%3D2019%26volume%3D9%26spage%3D1686%26epage%3D1695%26doi%3D10.1158%2F2159-8290.CD-19-0367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marsiglia, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katigbak, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohammadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traaseth, N. J.</span></span> <span> </span><span class="NLM_article-title">A conserved allosteric pathway in tyrosine kinase regulation</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1308</span>– <span class="NLM_lpage">1315</span>, <span class="refDoi"> DOI: 10.1016/j.str.2019.05.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1016%2Fj.str.2019.05.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=31204250" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFOku7nP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2019&pages=1308-1315&author=W.+M.+Marsigliaauthor=J.+Katigbakauthor=S.+Zhengauthor=M.+Mohammadiauthor=Y.+Zhangauthor=N.+J.+Traaseth&title=A+conserved+allosteric+pathway+in+tyrosine+kinase+regulation&doi=10.1016%2Fj.str.2019.05.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">A Conserved Allosteric Pathway in Tyrosine Kinase Regulation</span></div><div class="casAuthors">Marsiglia, William M.; Katigbak, Joseph; Zheng, Sijin; Mohammadi, Moosa; Zhang, Yingkai; Traaseth, Nathaniel J.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1308-1315.e3</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">An autoinhibitory network of hydrogen bonds located at the kinase hinge (referred to as the "mol. brake") regulates the activity of several receptor tyrosine kinases.  The mechanism whereby mutational disengagement of the brake allosterically activates the kinase in human disease is incompletely understood.  We used a combination of NMR, bioinformatics, and mol. dynamics simulation to show that mutational disruption of the mol. brake triggers localized conformational perturbations that propagate to the active site.  This entails changes in interactions of an isoleucine, one of three hydrophobic residues that lock the phenylalanine of the DFG motif in an inactive conformation.  Structural anal. of tyrosine kinases provides evidence that this allosteric control mechanism is shared across the tyrosine kinase family.  We also show that highly activating mutations at the brake diminish the enzyme's thermostability, thereby explaining why these mutations cause milder skeletal syndromes compared with less-activating mutations in the activation loop.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2xwjFECEogrVg90H21EOLACvtfcHk0ljVQY64SWVu7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFOku7nP&md5=6da80dec30b0ead22399e1f5d7491711</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2019.05.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2019.05.002%26sid%3Dliteratum%253Aachs%26aulast%3DMarsiglia%26aufirst%3DW.%2BM.%26aulast%3DKatigbak%26aufirst%3DJ.%26aulast%3DZheng%26aufirst%3DS.%26aulast%3DMohammadi%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DTraaseth%26aufirst%3DN.%2BJ.%26atitle%3DA%2520conserved%2520allosteric%2520pathway%2520in%2520tyrosine%2520kinase%2520regulation%26jtitle%3DStructure%26date%3D2019%26volume%3D27%26spage%3D1308%26epage%3D1315%26doi%3D10.1016%2Fj.str.2019.05.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blais, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neubert, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohammadi, M.</span></span> <span> </span><span class="NLM_article-title">DFG-out mode of inhibition by an irreversible type-1 inhibitor capable of overcoming gate-keeper mutations in FGF receptors</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">299</span>– <span class="NLM_lpage">309</span>, <span class="refDoi"> DOI: 10.1021/cb500674s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500674s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslGgt7vM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=299-309&author=Z.+Huangauthor=L.+Tanauthor=H.+Wangauthor=Y.+Liuauthor=S.+Blaisauthor=J.+Dengauthor=T.+A.+Neubertauthor=N.+S.+Grayauthor=X.+Liauthor=M.+Mohammadi&title=DFG-out+mode+of+inhibition+by+an+irreversible+type-1+inhibitor+capable+of+overcoming+gate-keeper+mutations+in+FGF+receptors&doi=10.1021%2Fcb500674s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">DFG-out Mode of Inhibition by an Irreversible Type-1 Inhibitor Capable of Overcoming Gate-Keeper Mutations in FGF Receptors</span></div><div class="casAuthors">Huang, Zhifeng; Tan, Li; Wang, Huiyan; Liu, Yang; Blais, Steven; Deng, Jingjing; Neubert, Thomas A.; Gray, Nathanael S.; Li, Xiaokun; Mohammadi, Moosa</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">299-309</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Drug-resistance acquisition through kinase gate-keeper mutations is a major hurdle in the clinic.  Here, we detd. the first crystal structures of the human FGFR4 kinase domain (FGFR4K) alone and complexed with ponatinib, a promiscuous type-2 (DFG-out) kinase inhibitor, and an oncogenic FGFR4K harboring the V550L gate-keeper mutation bound to FIIN-2, a new type-1 irreversible inhibitor.  Remarkably, like ponatinib, FIIN-2 also binds in the DFG-out mode despite lacking a functional group necessary to occupy the pocket vacated upon the DFG-out flip.  Structural anal. reveals that the covalent bond between FIIN-2 and a cysteine, uniquely present in the glycine-rich loop of FGFR kinases, facilitates the DFG-out conformation, which together with the internal flexibility of FIIN-2 enables FIIN-2 to avoid the steric clash with the gate-keeper mutation that causes the ponatinib resistance.  The structural data provide a blueprint for the development of next generation anticancer inhibitors through combining the salient inhibitory mechanisms of ponatinib and FIIN-2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpf97N01ZUSpbVg90H21EOLACvtfcHk0ljVQY64SWVu7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslGgt7vM&md5=0c741cc5492596f9ea9323e4d45e274b</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fcb500674s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500674s%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DTan%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DBlais%26aufirst%3DS.%26aulast%3DDeng%26aufirst%3DJ.%26aulast%3DNeubert%26aufirst%3DT.%2BA.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DMohammadi%26aufirst%3DM.%26atitle%3DDFG-out%2520mode%2520of%2520inhibition%2520by%2520an%2520irreversible%2520type-1%2520inhibitor%2520capable%2520of%2520overcoming%2520gate-keeper%2520mutations%2520in%2520FGF%2520receptors%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D299%26epage%3D309%26doi%3D10.1021%2Fcb500674s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Burgi, H. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunitz, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehn, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wipff, G.</span></span> <span> </span><span class="NLM_article-title">Sterochemistry of reaction paths at carbonyl centres</span>. <i>Tetrahedron</i> <span class="NLM_year" style="font-weight: bold;">1974</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">1563</span>– <span class="NLM_lpage">1572</span>, <span class="refDoi"> DOI: 10.1016/S0040-4020(01)90678-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1016%2FS0040-4020%2801%2990678-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADyaE2MXlsleqtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1974&pages=1563-1572&author=H.+B.+Burgiauthor=J.+D.+Dunitzauthor=J.+M.+Lehnauthor=G.+Wipff&title=Sterochemistry+of+reaction+paths+at+carbonyl+centres&doi=10.1016%2FS0040-4020%2801%2990678-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62aR"><div class="casContent"><span class="casTitleNuber">62a</span><div class="casTitle"><span class="NLM_cas:atitle">Stereochemistry of reaction paths at carbonyl centers</span></div><div class="casAuthors">Buergi, H. B.; Dunitz, J. D.; Lehn, J. M.; Wipff, G.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron</span>
        (<span class="NLM_cas:date">1974</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1563-72</span>CODEN:
                <span class="NLM_cas:coden">TETRAB</span>;
        ISSN:<span class="NLM_cas:issn">0040-4020</span>.
    </div><div class="casAbstract">Ab initio MO calcns. and crystal structure structure data were used to infer the reaction paths of nucleophilic addn. to carbonyl compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgIyobc_DXFrVg90H21EOLACvtfcHk0ljVQY64SWVu7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2MXlsleqtA%253D%253D&md5=b2d3956e505f323b2a475fa69ce01d23</span></div><a href="/servlet/linkout?suffix=cit62a&amp;dbid=16384&amp;doi=10.1016%2FS0040-4020%2801%2990678-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0040-4020%252801%252990678-7%26sid%3Dliteratum%253Aachs%26aulast%3DBurgi%26aufirst%3DH.%2BB.%26aulast%3DDunitz%26aufirst%3DJ.%2BD.%26aulast%3DLehn%26aufirst%3DJ.%2BM.%26aulast%3DWipff%26aufirst%3DG.%26atitle%3DSterochemistry%2520of%2520reaction%2520paths%2520at%2520carbonyl%2520centres%26jtitle%3DTetrahedron%26date%3D1974%26volume%3D30%26spage%3D1563%26epage%3D1572%26doi%3D10.1016%2FS0040-4020%2801%2990678-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit62b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Burgi, H. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunitz, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shefter, E.</span></span> <span> </span><span class="NLM_article-title">Geometrical reaction coordinates II nucleophilic addition to a carbonyl group</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1973</span>,  <span class="NLM_volume"><i>95</i></span>,  <span class="NLM_fpage">5065</span>– <span class="NLM_lpage">5067</span>, <span class="refDoi"> DOI: 10.1021/ja00796a058</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00796a058" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADyaE3sXkvFWhs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=1973&pages=5065-5067&author=H.+B.+Burgiauthor=J.+D.+Dunitzauthor=E.+Shefter&title=Geometrical+reaction+coordinates+II+nucleophilic+addition+to+a+carbonyl+group&doi=10.1021%2Fja00796a058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62bR"><div class="casContent"><span class="casTitleNuber">62b</span><div class="casTitle"><span class="NLM_cas:atitle">Geometrical reaction coordinates.  II.  Nucleophilic addition to a carbonyl group</span></div><div class="casAuthors">Burgi, H. B.; Dunitz, J. D.; Shefter, Eli</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1973</span>),
    <span class="NLM_cas:volume">95</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5065-7</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">Analysis of intramol. N...C:O interactions obsd. in six crystal structures provides an exptl. basis for mapping the reaction coordinate (min. energy pathway) for the addn. reaction of a nucleophile to a carbonyl group.  The line of approach of the nucleophile is not perpendicular to the C-O bond but is inclined at about 107° to it.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoG98LdihhQYLVg90H21EOLACvtfcHk0liuls9y7b3IFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE3sXkvFWhs7w%253D&md5=c7aa2b9f31ad151d8f334bf6032d7da4</span></div><a href="/servlet/linkout?suffix=cit62b&amp;dbid=16384&amp;doi=10.1021%2Fja00796a058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00796a058%26sid%3Dliteratum%253Aachs%26aulast%3DBurgi%26aufirst%3DH.%2BB.%26aulast%3DDunitz%26aufirst%3DJ.%2BD.%26aulast%3DShefter%26aufirst%3DE.%26atitle%3DGeometrical%2520reaction%2520coordinates%2520II%2520nucleophilic%2520addition%2520to%2520a%2520carbonyl%2520group%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1973%26volume%3D95%26spage%3D5065%26epage%3D5067%26doi%3D10.1021%2Fja00796a058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span> <span> </span><span class="NLM_article-title">Characterization of FGF401 as a reversible covalent inhibitor of fibroblast growth factor receptor 4</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">5890</span>– <span class="NLM_lpage">5893</span>, <span class="refDoi"> DOI: 10.1039/C9CC02052G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1039%2FC9CC02052G" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=31041948" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC1MXosVSitrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2019&pages=5890-5893&author=Z.+Zhouauthor=X.+Chenauthor=Y.+Fuauthor=Y.+Zhangauthor=S.+Daiauthor=J.+Liauthor=L.+Chenauthor=G.+Xuauthor=Z.+Chenauthor=Y.+Chen&title=Characterization+of+FGF401+as+a+reversible+covalent+inhibitor+of+fibroblast+growth+factor+receptor+4&doi=10.1039%2FC9CC02052G"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of FGF401 as a reversible covalent inhibitor of fibroblast growth factor receptor 4</span></div><div class="casAuthors">Zhou, Zhan; Chen, Xiaojuan; Fu, Ying; Zhang, Ye; Dai, Shuyan; Li, Jun; Chen, Lin; Xu, Guangyu; Chen, Zhuchu; Chen, Yongheng</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">5890-5893</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Biochem. and structural studies provide information on the mode of action of FGF401 as a selective, reversible covalent inhibitor of FGFR4.  Kinase and proliferation assays reveal that FGF401 has the ability to overcome gatekeeper mutations in FGFR4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolsnzMt2X_B7Vg90H21EOLACvtfcHk0liuls9y7b3IFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXosVSitrc%253D&md5=00b2da2e9df52673444bf51e5c8befe9</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1039%2FC9CC02052G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC9CC02052G%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DFu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DDai%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DY.%26atitle%3DCharacterization%2520of%2520FGF401%2520as%2520a%2520reversible%2520covalent%2520inhibitor%2520of%2520fibroblast%2520growth%2520factor%2520receptor%25204%26jtitle%3DChem.%2520Commun.%26date%3D2019%26volume%3D55%26spage%3D5890%26epage%3D5893%26doi%3D10.1039%2FC9CC02052G" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alex, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Millan, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakenhut, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitlock, G. A.</span></span> <span> </span><span class="NLM_article-title">Intramolecular hydrogen bonding to improve membrane permeability and absorption in beyond rule of five chemical space</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">669</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1039/c1md00093d</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1039%2Fc1md00093d" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotl2gt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=669-674&author=A.+Alexauthor=D.+S.+Millanauthor=M.+Perezauthor=F.+Wakenhutauthor=G.+A.+Whitlock&title=Intramolecular+hydrogen+bonding+to+improve+membrane+permeability+and+absorption+in+beyond+rule+of+five+chemical+space&doi=10.1039%2Fc1md00093d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Intramolecular hydrogen bonding to improve membrane permeability and absorption in beyond rule of five chemical space</span></div><div class="casAuthors">Alex, Alexander; Millan, David S.; Perez, Manuel; Wakenhut, Florian; Whitlock, Gavin A.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">669-674</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Utilizing 'beyond rule of five' chem. space is becoming increasingly important in drug design, but is usually at odds with good oral absorption.  The formation of intramol. hydrogen bonds in drug mols. is hypothesised to shield polarity facilitating improved membrane permeability and intestinal absorption.  NMR based evidence for intramol. hydrogen bonding in several 'beyond rule of five' oral drugs is described.  Furthermore, the propensity for these drugs to form intramol. hydrogen bonds could be predicted for through modeling the lowest energy conformation in the gas phase.  The modulation of apparent lipophilicity through intramol. hydrogen bonding in these mols. is supported by intrinsic cell permeability and intestinal absorption data in rat and human.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp93l2KJMdB2bVg90H21EOLACvtfcHk0liuls9y7b3IFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotl2gt7o%253D&md5=3effbdd29832ac7b57eabbc26c66217b</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1039%2Fc1md00093d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc1md00093d%26sid%3Dliteratum%253Aachs%26aulast%3DAlex%26aufirst%3DA.%26aulast%3DMillan%26aufirst%3DD.%2BS.%26aulast%3DPerez%26aufirst%3DM.%26aulast%3DWakenhut%26aufirst%3DF.%26aulast%3DWhitlock%26aufirst%3DG.%2BA.%26atitle%3DIntramolecular%2520hydrogen%2520bonding%2520to%2520improve%2520membrane%2520permeability%2520and%2520absorption%2520in%2520beyond%2520rule%2520of%2520five%2520chemical%2520space%26jtitle%3DMedChemComm%26date%3D2011%26volume%3D2%26spage%3D669%26epage%3D674%26doi%3D10.1039%2Fc1md00093d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gu, Y.</span>; <span class="NLM_string-name">Wilbaux, M.</span>; <span class="NLM_string-name">Yang, S.</span>; <span class="NLM_string-name">Demanse, D.</span>; <span class="NLM_string-name">Jullion, A.</span>; <span class="NLM_string-name">Meille, C.</span>; <span class="NLM_string-name">Siddani, R.</span>; <span class="NLM_string-name">Graus Porta, D.</span>; <span class="NLM_string-name">Myers, A.</span></span> <span> </span><span class="NLM_article-title">Integrated Pharmacokinetic-Pharmacodynamic (PK-PD) and Exposure-Response (E-R) Analyses to Support the First-In-Human (FIH) Phase I/II Study of FGF401</span>. Presented at the 30th Joint European Organisation for Research and Treatment of Cancer, National Cancer Institute, American Association for Cancer Research Symposium on Molecular Targets and Cancer Therapeutics, Dublin, Ireland, Nov 13–16, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=Y.+Gu&author=M.+Wilbaux&author=S.+Yang&author=D.+Demanse&author=A.+Jullion&author=C.+Meille&author=R.+Siddani&author=D.+Graus+Porta&author=A.+Myers&title=Integrated+Pharmacokinetic-Pharmacodynamic+%28PK-PD%29+and+Exposure-Response+%28E-R%29+Analyses+to+Support+the+First-In-Human+%28FIH%29+Phase+I%2FII+Study+of+FGF401"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DGu%26aufirst%3DY.%26atitle%3DIntegrated%2520Pharmacokinetic-Pharmacodynamic%2520%2528PK-PD%2529%2520and%2520Exposure-Response%2520%2528E-R%2529%2520Analyses%2520to%2520Support%2520the%2520First-In-Human%2520%2528FIH%2529%2520Phase%2520I%252FII%2520Study%2520of%2520FGF401%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit65b"><span>Abstract:  <i>Eur. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume">103</span>,  <span class="NLM_fpage">E122</span>– <span class="NLM_lpage">E123</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Abstract%3A+Eur.+J.+Cancer+2018%2C+103%2C+E122%E2%80%93E123."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit65b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26jtitle%3DEur.%2520J.%2520Cancer%26date%3D2018%26volume%3D103%26spage%3DE122%26epage%3DE123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><a href="https://clinicaltrials.gov/ct2/show/NCT02325739" class="extLink">https://clinicaltrials.gov/ct2/show/NCT02325739</a> (accessed Sep 14, <span class="NLM_year">2020</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=https%3A%2F%2Fclinicaltrials.gov%2Fct2%2Fshow%2FNCT02325739+%28accessed+Sep+14%2C+2020%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, S. L.</span>; <span class="NLM_string-name">Yen, C.-J.</span>; <span class="NLM_string-name">Schuler, M.</span>; <span class="NLM_string-name">Lin, C.-C.</span>; <span class="NLM_string-name">Choo, S. P.</span>; <span class="NLM_string-name">Weiss, K. H.</span>; <span class="NLM_string-name">Geier, A.</span>; <span class="NLM_string-name">Okusaka, T.</span>; <span class="NLM_string-name">Lim, H. Y.</span>; <span class="NLM_string-name">Macarulla, T.</span>; <span class="NLM_string-name">Zhu, A. X.</span>; <span class="NLM_string-name">Kakizume, T.</span>; <span class="NLM_string-name">Gu, Y.</span>; <span class="NLM_string-name">Graus-Porta, D.</span>; <span class="NLM_string-name">Myers, A. P.</span>; <span class="NLM_string-name">Delord, J.-P.</span></span> <span> </span><span class="NLM_article-title">Phase I/II Study of FGF401 in Adult Patients with Hepatocellular Carcinoma or Solid Tumors Characterized by FGFR4/KLB Expression</span>. Presented at the American Association for Cancer Research Annual Meeting, Washington, DC, Apr 1–5, <span class="NLM_year">2017</span>; Abstract CT106.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=S.+L.+Chan&author=C.-J.+Yen&author=M.+Schuler&author=C.-C.+Lin&author=S.+P.+Choo&author=K.+H.+Weiss&author=A.+Geier&author=T.+Okusaka&author=H.+Y.+Lim&author=T.+Macarulla&author=A.+X.+Zhu&author=T.+Kakizume&author=Y.+Gu&author=D.+Graus-Porta&author=A.+P.+Myers&author=J.-P.+Delord&title=Phase+I%2FII+Study+of+FGF401+in+Adult+Patients+with+Hepatocellular+Carcinoma+or+Solid+Tumors+Characterized+by+FGFR4%2FKLB+Expression"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DS.%2BL.%26atitle%3DPhase%2520I%252FII%2520Study%2520of%2520FGF401%2520in%2520Adult%2520Patients%2520with%2520Hepatocellular%2520Carcinoma%2520or%2520Solid%2520Tumors%2520Characterized%2520by%2520FGFR4%252FKLB%2520Expression%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><a href="http://www.everestmedicines.com/Pipeline.aspx#news_138" class="extLink">http://www.everestmedicines.com/Pipeline.aspx#news_138</a> (accessed Sep 14, <span class="NLM_year">2020</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=http%3A%2F%2Fwww.everestmedicines.com%2FPipeline.aspx%23news_138+%28accessed+Sep+14%2C+2020%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yasuda, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsiao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivera, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yau, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palucki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dormer, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volante, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reider, P. J.</span></span> <span> </span><span class="NLM_article-title">An Efficient synthesis of an α<sub>v</sub>β<sub>3</sub> antagonist <i>J</i></span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">1959</span>– <span class="NLM_lpage">1966</span>, <span class="refDoi"> DOI: 10.1021/jo030297u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo030297u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BD2cXht12gtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2004&pages=1959-1966&author=N.+Yasudaauthor=Y.+Hsiaoauthor=M.+S.+Jensenauthor=N.+R.+Riveraauthor=C.+Yangauthor=K.+M.+Wellsauthor=J.+Yauauthor=M.+Paluckiauthor=L.+Tanauthor=P.+G.+Dormerauthor=R.+P.+Volanteauthor=D.+L.+Hughesauthor=P.+J.+Reider&title=An+Efficient+synthesis+of+an+%CE%B1v%CE%B23+antagonist+J&doi=10.1021%2Fjo030297u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">An Efficient Synthesis of an αvβ3 Antagonist</span></div><div class="casAuthors">Yasuda, Nobuyoshi; Hsiao, Yi; Jensen, Mark S.; Rivera, Nelo R.; Yang, Chunhua; Wells, Kenneth M.; Yau, James; Palucki, Michael; Tan, Lushi; Dormer, Peter G.; Volante, Ralph P.; Hughes, David L.; Reider, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1959-1966</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A practical prepn. of an αvβ3 antagonist, (βS)-6-methoxy-β-[2-oxo-3-[3-(5,6,7,8-tetrahydro-1,8-naphthyridin-2-yl)propyl]-1-imidazolidinyl]-3-pyridinepropanoic acid (I), is reported.  The antagonist consists of three key components, a tetrahydronaphthyridine moiety, a β-alanine moiety, and a central imidazolidone moiety.  The tetrahydronaphthyridine component was prepd. using two different methods, both of which relied on variations of the Friedlander reaction to establish the desired regiochem.  The β-alanine component was prepd. using Davies' asym. 1,4-addn. methodol. as the key stereo-defining step.  The central imidazolidone portion was created from these two components using an effective three-step cyclization protocol.  Thus, a highly convergent process for the drug candidate was defined.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-OkegSdO1JLVg90H21EOLACvtfcHk0lgrCfXOGuTt9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXht12gtrY%253D&md5=178065fe8be7e5a2e4b236c3392f839f</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1021%2Fjo030297u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo030297u%26sid%3Dliteratum%253Aachs%26aulast%3DYasuda%26aufirst%3DN.%26aulast%3DHsiao%26aufirst%3DY.%26aulast%3DJensen%26aufirst%3DM.%2BS.%26aulast%3DRivera%26aufirst%3DN.%2BR.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DWells%26aufirst%3DK.%2BM.%26aulast%3DYau%26aufirst%3DJ.%26aulast%3DPalucki%26aufirst%3DM.%26aulast%3DTan%26aufirst%3DL.%26aulast%3DDormer%26aufirst%3DP.%2BG.%26aulast%3DVolante%26aufirst%3DR.%2BP.%26aulast%3DHughes%26aufirst%3DD.%2BL.%26aulast%3DReider%26aufirst%3DP.%2BJ.%26atitle%3DAn%2520Efficient%2520synthesis%2520of%2520an%2520%25CE%25B1v%25CE%25B23%2520antagonist%2520J%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2004%26volume%3D69%26spage%3D1959%26epage%3D1966%26doi%3D10.1021%2Fjo030297u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Holt, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreassen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bakke, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiksdahl, A.</span></span> <span> </span><span class="NLM_article-title">Nitropyridyl isocyanates</span>. <i>J. Heterocycl. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">259</span>– <span class="NLM_lpage">264</span>, <span class="refDoi"> DOI: 10.1002/jhet.5570420213</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1002%2Fjhet.5570420213" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BD2MXivFGlsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2005&pages=259-264&author=J.+Holtauthor=T.+Andreassenauthor=J.+M.+Bakkeauthor=A.+Fiksdahl&title=Nitropyridyl+isocyanates&doi=10.1002%2Fjhet.5570420213"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Nitropyridyl isocyanates</span></div><div class="casAuthors">Holt, Jarle; Andreassen, Trygve; Bakke, Jan M.; Fiksdahl, Anne</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Heterocyclic Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">259-264</span>CODEN:
                <span class="NLM_cas:coden">JHTCAD</span>;
        ISSN:<span class="NLM_cas:issn">0022-152X</span>.
    
            (<span class="NLM_cas:orgname">HeteroCorporation</span>)
        </div><div class="casAbstract">The authors report the first prepn. of 3-nitro-4-pyridyl isocyanate (I) and 5-nitro-2-pyridyl isocyanate (II).  They were formed by Curtius rearrangement of the corresponding acyl azides, prepd. from Me 3-nitro-4-pyridinecarboxylate.  I can be stored in dry soln. and is stable at room temp. for several weeks while II was less stable and should be used for synthetic purposes immediately.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTaMMqynXYeLVg90H21EOLACvtfcHk0lgrCfXOGuTt9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXivFGlsrw%253D&md5=f95cc37035ae9bda443eeee8893fcf1d</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1002%2Fjhet.5570420213&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjhet.5570420213%26sid%3Dliteratum%253Aachs%26aulast%3DHolt%26aufirst%3DJ.%26aulast%3DAndreassen%26aufirst%3DT.%26aulast%3DBakke%26aufirst%3DJ.%2BM.%26aulast%3DFiksdahl%26aufirst%3DA.%26atitle%3DNitropyridyl%2520isocyanates%26jtitle%3DJ.%2520Heterocycl.%2520Chem.%26date%3D2005%26volume%3D42%26spage%3D259%26epage%3D264%26doi%3D10.1002%2Fjhet.5570420213" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wellenzohn, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lessel, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beller, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isambert, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoenke, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nosse, B.</span></span> <span> </span><span class="NLM_article-title">Identification of new potent GPR119 agonists by combining virtual screening and combinatorial chemistry</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">11031</span>– <span class="NLM_lpage">11041</span>, <span class="refDoi"> DOI: 10.1021/jm301549a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301549a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslOlsL%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=11031-11041&author=B.+Wellenzohnauthor=U.+Lesselauthor=A.+Bellerauthor=T.+Isambertauthor=C.+Hoenkeauthor=B.+Nosse&title=Identification+of+new+potent+GPR119+agonists+by+combining+virtual+screening+and+combinatorial+chemistry&doi=10.1021%2Fjm301549a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of New Potent GPR119 Agonists by Combining Virtual Screening and Combinatorial Chemistry</span></div><div class="casAuthors">Wellenzohn, Bernd; Lessel, Uta; Beller, Andreas; Isambert, Timo; Hoenke, Christoph; Nosse, Bernd</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">11031-11041</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Virtual screening in a huge collection of virtual combinatorial libraries has led to the identification of two new structural classes of GPR119 agonists with submicromolar in vitro potencies.  Herein, the authors describe the virtual screening process involving feature trees fragment space searches followed by a 3D postprocessing step.  The in silico findings were then filtered and prioritized, and finally, combinatorial libraries of target mols. were synthesized.  Furthermore the so-called "activity-anchor principle" is introduced as an element to increase the chance to generate true hits.  An activity anchor is a structural element expected to provide key contributions to a certain biol. activity.  Application of this technique has led to the discovery of two new GPR119-agonist hit series, one of which was further optimized to progress as a novel lead class.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqe1IfuyaYeNrVg90H21EOLACvtfcHk0lgYXFC7X0VfIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslOlsL%252FJ&md5=18e23e2eab61f8ef0015d0b76f7e467b</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1021%2Fjm301549a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301549a%26sid%3Dliteratum%253Aachs%26aulast%3DWellenzohn%26aufirst%3DB.%26aulast%3DLessel%26aufirst%3DU.%26aulast%3DBeller%26aufirst%3DA.%26aulast%3DIsambert%26aufirst%3DT.%26aulast%3DHoenke%26aufirst%3DC.%26aulast%3DNosse%26aufirst%3DB.%26atitle%3DIdentification%2520of%2520new%2520potent%2520GPR119%2520agonists%2520by%2520combining%2520virtual%2520screening%2520and%2520combinatorial%2520chemistry%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D11031%26epage%3D11041%26doi%3D10.1021%2Fjm301549a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1a','cit1b'],'ref2':['cit2a','cit2b'],'ref3':['cit3a','cit3b'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6a','cit6b'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10a','cit10b'],'ref11':['cit11'],'ref12':['cit12a','cit12b'],'ref13':['cit13a','cit13b'],'ref14':['cit14a','cit14b'],'ref15':['cit15a','cit15b'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41a','cit41b'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54a','cit54b'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57a','cit57b','cit57c'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62a','cit62b'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65','cit65b'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 5 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Wuqing Deng, Xiaojuan Chen, Kaili Jiang, Xiaojuan Song, Minhao Huang, Zheng-Chao Tu, Zhang Zhang, Xiaojing Lin, Raquel Ortega, Adam V. Patterson, Jeff B. Smaill, Ke Ding, Suming Chen, Yongheng Chen, <span class="NLM_string-name hlFld-ContribAuthor">Xiaoyun Lu</span>. </span><span class="cited-content_cbyCitation_article-title">Investigation of Covalent Warheads in the Design of 2-Aminopyrimidine-based FGFR4 Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2021,</strong> <em>12 </em>
                                    (4)
                                     , 647-652. <a href="https://doi.org/10.1021/acsmedchemlett.1c00052" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.1c00052</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.1c00052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.1c00052%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DInvestigation%252Bof%252BCovalent%252BWarheads%252Bin%252Bthe%252BDesign%252Bof%252B2-Aminopyrimidine-based%252BFGFR4%252BInhibitors%26aulast%3DDeng%26aufirst%3DWuqing%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D23012021%26date%3D19032021%26date%3D22032021%26volume%3D12%26issue%3D4%26spage%3D647%26epage%3D652" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Christian Gampe, <span class="NLM_string-name hlFld-ContribAuthor">Vishal A. Verma</span>. </span><span class="cited-content_cbyCitation_article-title">Curse or Cure? A Perspective on the Developability of Aldehydes as Active Pharmaceutical Ingredients. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (23)
                                     , 14357-14381. <a href="https://doi.org/10.1021/acs.jmedchem.0c01177" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01177</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01177&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01177%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DCurse%252Bor%252BCure%25253F%252BA%252BPerspective%252Bon%252Bthe%252BDevelopability%252Bof%252BAldehydes%252Bas%252BActive%252BPharmaceutical%252BIngredients%26aulast%3DGampe%26aufirst%3DChristian%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D07072020%26date%3D29092020%26date%3D11092020%26volume%3D63%26issue%3D23%26spage%3D14357%26epage%3D14381" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">. </span><span class="cited-content_cbyCitation_article-title">Synthesis of Roblitinib. </span><span class="cited-content_cbyCitation_journal-name">Synfacts</span><span> <strong>2021,</strong>,, 0125. <a href="https://doi.org/10.1055/s-0040-1719320" title="DOI URL">https://doi.org/10.1055/s-0040-1719320</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1055/s-0040-1719320&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1055%2Fs-0040-1719320%26sid%3Dliteratum%253Aachs%26jtitle%3DSynfacts%26atitle%3DSynthesis%252Bof%252BRoblitinib%26date%3D2021%26date%3D2021%26volume%3D17%26issue%3D02%26spage%3D0125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jianmin  Gao</span>, <span class="hlFld-ContribAuthor ">Vincent  Nobile</span>. </span><span class="cited-content_cbyCitation_article-title">Chemistry perspectives of reversible covalent drugs. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 75-94. <a href="https://doi.org/10.1016/bs.armc.2020.10.004" title="DOI URL">https://doi.org/10.1016/bs.armc.2020.10.004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.armc.2020.10.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.armc.2020.10.004%26sid%3Dliteratum%253Aachs%26atitle%3DChemistry%252Bperspectives%252Bof%252Breversible%252Bcovalent%252Bdrugs%26aulast%3DGao%26aufirst%3DJianmin%26date%3D2021%26spage%3D75%26epage%3D94%26pub%3DElsevier%26atitle%3DThe%252BDesign%252Bof%252BCovalent-Based%252BInhibitors%26date%3D2021%26volume%3D56" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Robert  Abel</span>, <span class="hlFld-ContribAuthor ">Richard  Friesner</span>. </span><span class="cited-content_cbyCitation_article-title">Computational support of structure-based targeted covalent inhibitor design. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 229-242. <a href="https://doi.org/10.1016/bs.armc.2021.01.001" title="DOI URL">https://doi.org/10.1016/bs.armc.2021.01.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.armc.2021.01.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.armc.2021.01.001%26sid%3Dliteratum%253Aachs%26atitle%3DComputational%252Bsupport%252Bof%252Bstructure-based%252Btargeted%252Bcovalent%252Binhibitor%252Bdesign%26aulast%3DAbel%26aufirst%3DRobert%26date%3D2021%26spage%3D229%26epage%3D242%26pub%3DElsevier%26atitle%3DThe%252BDesign%252Bof%252BCovalent-Based%252BInhibitors%26date%3D2021%26volume%3D56" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0030.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0030.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of selective pan-FGFR inhibitors: (A) FGFR1–3 biased; (B) FGFR1–4 balanced; (C) allosteric FGFR inhibitor; (D) covalent P-loop targeting FGFR1–4 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. FGFR4 signaling cascades regulating BA biosynthesis, and survival and proliferation in cancer.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Amino acid sequence alignment centered around the FGFR-family middle-hinge region.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Structures of selective FGFR4 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. FGFR4 hits identified by the rational modification of known kinase inhibitors and unbiased screening. Interactions with the middle-hinge region are determined from modeling for <b>6</b> and <b>8</b> and X-ray cocrystal structures for <b>5</b> and <b>7</b> (PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NWZ">5NWZ</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NUD">5NUD</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Key SAR generated to characterize the 2-FQA and 2-FTHNU hit series: (A) binding hypothesis for the interaction of <b>8</b> with FGFR4; (B) 2-FQA to 2-FTHNU scaffold morph; (C) inactive aldehyde replacements; (D) phenyl and N-methylated analogues disrupting the hinge-binding and intramolecular hydrogen bond; (E) cyanopyrimidine as an alternative electrophile; (F) analogues designed to assess the contribution of the 2-formylpyridyl group.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Overlay of the cysteine-thiol positions within the FGFR ATP-binding site of selective FGFR4 inhibitors covalently binding to Cys552, and pan-FGFR inhibitors binding to the P-loop cysteine. PDB codes are the following: <b>7</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NUD">5NUD</a>, in gray;<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> roblitinib, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YI8">6YI8</a>, in dark blue; BLU9931, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XCU">4XCU</a>, in brown;<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> H3B-6527, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VND">5VND</a>, in green;<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> FIIN-2, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4QQC">4QQC</a>. in light blue;<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> FIIN-3, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4R6V">4R6V</a>, in yellow;<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a><b>5</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NWZ">5NWZ</a>, in pink;<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> PRN1371, modeled interaction in magenta.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. <sup>1</sup>H NMR study with <sup>13</sup>C-labeled <b>18</b> in water showing the ratio between the aldehyde and hydrated species. <sup>13</sup>C-HSQC: aldehyde <sup>1</sup>H/<sup>13</sup>C 9.92/192.5 ppm, <sup>1</sup><i>J</i>(<sup>1</sup>H–<sup>13</sup>C) coupling 182 Hz; hydrate <sup>1</sup>H/<sup>13</sup>C 6.09/89.4 ppm, <sup>1</sup><i>J</i>(<sup>1</sup>H–<sup>13</sup>C) coupling 166 Hz. Integration of the <sup>1</sup>H spectra shows an aldehyde to hydrate ratio of 9:1.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Core and hinge-binding 2-FTHNU SAR: (A) THNU core variants; (B) hinge-binding heterocycle variants; (C) pyridine 4-substituents interacting with the Gly474/Val481 hydrophobic cleft; (D) modeled interaction of compound <b>28</b> within the ATP-binding site of FGFR4 highlighting the interactions between the pyridyl 4-methoxyethoxy substituent and the hydrophobic cleft formed between Gly474 and Val481.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0010.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Western blot pFRS2 and pERK PD measurements for compound <b>24</b> (50 nM) and infigratinib (50 nM) in RH41 cells after 40 min and 3 day incubation periods. Western blot for total Erk is provided as the protein loading control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0011.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Time course of the relative peak areas of parent, M + 2, and M + 16 metabolites in solution for <b>25</b> when incubated with Caco-2 cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig12" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0012.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Rat PK studies with <b>24</b>: (A) PK curves and parameters following iv (1.0 mg/kg in 7:3 PEG 200/NMP) and po (3.0 mg/kg suspension in 99.5:0.5 CMC-05/Tween 80) dosing; (B) acid and alcohol derivatives of <b>24</b>; (C) rat liver microsome MetID study.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig13" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0013.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 13. Approaches taken to stabilize the 2-FTHNU aldehyde group toward metabolism: (A) 2-FQA and 2-FTHNU 3-substituent SAR; (B) interaction of the THN 3-substituent with Thr499 modeled with <b>37</b> bound within the ATP-pocket of FGFR4; (C) equilibrium between the aldehyde and hemiacetal forms of the 3-hydroxymethyl substituted analogues; (D) diols arising from the reduction of the aldehydes <b>42</b> and <b>43</b>; (E) THF-ether 4-substituted pyridine analogues.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig13"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig14" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 14</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0014.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 14. PD time course profiles for infigratinib, <b>42</b>, and <b>58</b> in Hep3B cells. The Western blots show pFRS2 and pMAPK (pErk) as PD readouts with the α-actin protein levels as loading control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig14"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig15" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Figure 15</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0015.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 15. In vivo profile of <b>43</b> in the mouse: (A) PK parameters following iv (1.0 mg/kg) and oral dosing (suspension, 3.0 mg/kg); (B) time course PK/PD and total plasma concentrations with RH30 and RH41 mouse xenografts (Western blot, RH30 normalized to total ERK; RH41 normalized to the lysate inputs, 1 mg protein) following oral dosing (suspension, 30 mg/kg); (C) efficacy studies with the Hep3B mouse xenograft model (<i>n</i> = 6, mean ± SEM), *<i>P</i> < 0.05 versus vehicle by one way ANOVA with post-hoc Dunnett’s test; (D) effects on liver CYP7A1 mRNA and circulating BA levels following 2 weeks dosing, *<i>P</i> < 0.05 versus vehicle by heteroscedastic Student’s <i>t</i>-test, **<i>P</i> < 0.05 versus vehicle by one way ANOVA with Dunnett’s multiple comparisons test.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig15"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig16" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Figure 16</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0016.jpeg" id="rightTab-gr16" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 16. Relationship between cLogP and metabolic stability in rat liver microsome incubations highlighting the effect of the size of the THN 3-substitutent. Compounds are colored according to the size of the 3-substitutent: unsubstituted in brown; less than three heavy atoms in light blue; three or more heavy atoms in dark blue.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig16"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig17" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Figure 17</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0017.jpeg" id="rightTab-gr17" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 17. <i>N</i>-Acylaminomethyl 2-FTHNU 3-substituent SAR: (A) <i>N</i>-methyl-<i>N</i>-acyl analogues; (B) modeled interaction of compound <b>59</b> within the ATP-binding site of FGFR4 highlighting the H-bond between the carbonyl of the <i>N</i>-acetyl moiety and the side chain of Arg483; (C) selected <i>N</i>-acetyl and <i>N</i>-alkyl analogues; (D) modeled interaction of the enantiomers of compound <b>70</b> within the ATP-binding site of FGFR4 highlighting the unfavorable interactions made by the benzylic methyl moiety.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig17"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig18" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Figure 18</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0018.jpeg" id="rightTab-gr18" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 18. In vivo studies with compound <b>68</b>: (A) PK parameters in the mouse (iv 0.5 mg/kg in 9:1 plasma/NMP) and dog (iv 0.1 mg/kg in 9:1 PEG 200/NMP and po 0.3 mg/kg as an aq suspension), where parameters are reported as the mean ± SD (<i>n</i> = 3); (B) total plasma concentration/response curves from PK/PD studies with the mouse RH30 xenograft model; (C) plot of tumor growth inhibition with the mouse HUH7 xenograft model (<i>n</i> = 6, mean ± SEM), *<i>P</i> < 0.05 versus vehicle by Student’s <i>t</i>-test.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig18"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig19" data-index="19" class="article__inlineFigure"><h2 class="fig-label">Figure 19</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0019.jpeg" id="rightTab-gr19" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 19. Evaluation of the 2-FTHNU toxicology with compound <b>68</b>: (A) FGFR4 cross-species sequence alignment centered around the middle-hinge region highlighting the absence of a cysteine residue at the GK+2 position in the rat; (B) KINOME<i>scan</i> TREE<i>spot</i> interaction map; (C) exposure data in blood (total) from the mouse efficacy (<i>n</i> = 2) and rat toxicology (<i>n</i> = 4) studies reported as the mean ± SD.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig19"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig20" data-index="20" class="article__inlineFigure"><h2 class="fig-label">Figure 20</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0020.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0020.jpeg" id="rightTab-gr20" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 20. Approaches to increasing the solubility of the 2-FTHNU series: (A) introducing basic groups into the hinge-binding pyridine moiety at the 4-position to target the aspartic acid of the DFG motif; (B) modeled interaction of compound <b>74</b> within the ATP-pocket of FGFR4 highlighting the interaction of the dimethylamino moiety with Asp630 of the DFG motif; (C) selected SAR from introducing basic groups into the 3-position of the THN moiety; (D) further SAR around the 3-(<i>N</i>-methylpiperidin-2-one)methyl THN containing analogues.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig20"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig21" data-index="21" class="article__inlineFigure"><h2 class="fig-label">Figure 21</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0021.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0021.jpeg" id="rightTab-gr21" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 21. In vitro profile of roblitinib <b>84</b>: (A) KINOME<i>scan</i> TREE<i>spot</i> interaction map; (B) selected biochemical kinase assay data; (C) selected cellular proliferation data in HCC cell lines both dependent on and independent of FGF19/FGFR4 signaling (methylene blue staining).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0021.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig21"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig22" data-index="22" class="article__inlineFigure"><h2 class="fig-label">Figure 22</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0022.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0022.jpeg" id="rightTab-gr22" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 22. X-ray structures of <b>84</b>: (A) cocrystal structure in complex with the FGFR4 kinase domain ATP binding site (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YI8">6YI8</a>). After reaction, the aldehyde group of <b>84</b> forms a hemithioacetal with the hinge residue Cys552 specific to FGFR4. <b>84</b> forms direct or water mediated hydrogen bonds with residues Ala553, Val500, Arg483, Lys503, and Asn557 of the ATP binding site (indicated as dashed lines). The intramolecular hydrogen bond between the inhibitor free urea nitrogen and the pyridyl nitrogen of the THN moiety, previously observed in this series of compounds, is also present. (B) ORTEP representation of a single-crystal X-ray structure of <b>84</b> (CCDC code 2009566).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0022.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig22"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig23" data-index="23" class="article__inlineFigure"><h2 class="fig-label">Figure 23</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0023.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0023.jpeg" id="rightTab-gr23" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 23. Preclinical pharmacokinetics determined for <b>84</b>: (A) table showing key in vitro and in vivo parameters, mouse (female Balb/c, <i>n</i> = 3) dosed iv at 1 mg/kg in 9:1 NMP/plasma and po at 3 mg/kg as a suspension of the crystalline free base in 99:0.5:0.5 water/methyl cellulose/Tween-80, rat (Sprague Dawley, <i>n</i> = 4) dosed iv at 0.5 mg/kg in 7:3 PEG 200/NMP and po at 3 mg/kg as a suspension of the crystalline free base in 99.5:0.5 methylcellulose-05/Tween-80, dog (male, beagle, <i>n</i> = 3) dosed iv at 0.1 mg/kg in 9:1 PEG 200/NMP and po at 0.3 mg/kg as a suspension in water; (B) structures of the reduced <b>92</b> and predominant demethylated metabolites <b>93</b> and <b>94</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0023.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig23"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig24" data-index="24" class="article__inlineFigure"><h2 class="fig-label">Figure 24</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0024.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0024.jpeg" id="rightTab-gr24" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 24. Mouse xenograft studies with <b>84</b>: (A) total plasma concentration–response curve in the RH30 xenograft model, IC<sub>50</sub> = 2.1 nM, IC<sub>90</sub> = 52 nM; (B) change in tumor volume over time in the Hep3B xenograft model, where data are the mean ± SD, *<i>P</i> < 0.05 versus vehicle by Kruskal–Wallis (Dunn’s); (C) day 6 blood exposure profiles.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0024.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig24"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig25" data-index="25" class="article__inlineFigure"><h2 class="fig-label">Figure 25</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0025.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0025.jpeg" id="rightTab-gr25" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 25. Dose fraction study with <b>84</b> in the HUH7 xenograft model in the rat (<i>n</i> = 8, data are the mean ± SD): (A) change in tumor volume over time, *<i>P</i> < 0.05 versus vehicle by Kruskal–Wallis (Dunn’s); (B) day 8 blood exposure profiles; (C) efficacy and exposure parameters.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0025.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=fig25"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="26" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0026.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0026.jpeg" id="rightTab-gr26" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthetic Routes to the Hinge-Binding 2-Aminopyridine Intermediates <b>96</b> and <b>97</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0026.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NIS, DMA, rt, 24 h (76–81%); (b) Zn(CN)<sub>2</sub>, Zn, Pd<sub>2</sub>dba<sub>3</sub>, dppf, DMA, 150 °C, 2 h (84–97%); (c) 2-methoxyethylamine, DIPEA, DMA, 50 °C, 18 h (80–85%); (d) <i>i</i>PrOH, KHMDS, THF, rt, 18 h (55–62%).</p></p></figure><figure data-id="sch2" data-index="27" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0027.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0027.jpeg" id="rightTab-gr27" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of the Key THN Intermediates <b>102</b>, <b>103</b>, and <b>104</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0027.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NBS, MeCN, rt, 0.5 h (73%); (b) 1 equiv of MeLi, then 1 equiv of <i>n</i>BuLi, then DMF, −78 °C to rt (>95%); (c) DAST, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to rt, 18 h (63–69%); (d) NaBH<sub>4</sub>, MeOH, DCM, rt, 0.5 h (>95%); (e) TBDMSCl, Et<sub>3</sub>N, DMAP, DCM, DMF, rt, 2 h (80–95%); (f) 2 M Me<sub>2</sub>NH in THF, Na(OAc)<sub>3</sub>BH, DCE, rt, 48 h (>95%).</p></p></figure><figure data-id="sch3" data-index="28" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0028.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0028.jpeg" id="rightTab-gr28" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthetic Route to Compound <b>68</b>, Using the Aza-Anion Approach for Urea Formation<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0028.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) LiHMDS, THF, −78 °C, then (PhO)<sub>2</sub>CO, warm to rt (59–75%); (b) <b>97</b>, LiHMDS, THF, −78 °C, 2 h (79–82%); (c) HF-pyridine, THF, rt, 4 h (88-95%); (d) (MeSO<sub>2</sub>)<sub>2</sub>O, DIPEA, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to rt, 2 h; (e) methyl 4-aminobutyrate, DIPEA, KI, 5:4 THF:CH<sub>2</sub>Cl<sub>2</sub>, rt, 22 h (44–52% 2 steps); (f) Et<sub>3</sub>N, 1,4-dioxane, 90 °C, 24 h (39–65%); (g) HCl<sub>(aq)</sub>, rt, 2 h (94–95%).</p></p></figure><figure data-id="sch4" data-index="29" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/medium/jm0c01019_0029.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0029.jpeg" id="rightTab-gr29" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthetic Route to <b>84</b> (Roblitinib), Using CDT for Urea Formation<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01019/20201105/images/large/jm0c01019_0029.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01019&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) TEA, THF, rt (98%); (b) conc HCl, THF, EtOH, rt (92%); (c) <b>101</b>, NaBH(OAc)<sub>3</sub>, TEA, DCE, rt (54–67%); (d) <b>96</b>, CDT, DMF, rt (82–86%); (e) conc HCl, THF<sub>(aq)</sub>, rt (82–88%).</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i35">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88266" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88266" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 71 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grose, R.</span></span> <span> </span><span class="NLM_article-title">Fibroblast growth factor signaling: from development to cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">116</span>– <span class="NLM_lpage">129</span>, <span class="refDoi"> DOI: 10.1038/nrc2780</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1038%2Fnrc2780" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=20094046" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC3cXps1Gktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=116-129&author=N.+Turnerauthor=R.+Grose&title=Fibroblast+growth+factor+signaling%3A+from+development+to+cancer&doi=10.1038%2Fnrc2780"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1aR"><div class="casContent"><span class="casTitleNuber">1a</span><div class="casTitle"><span class="NLM_cas:atitle">Fibroblast growth factor signalling: from development to cancer</span></div><div class="casAuthors">Turner, Nicholas; Grose, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">116-129</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Fibroblast growth factors (FGFs) and their receptors control a wide range of biol. functions, regulating cellular proliferation, survival, migration and differentiation.  Although targeting FGF signalling as a cancer therapeutic target has lagged behind that of other receptor tyrosine kinases, there is now substantial evidence for the importance of FGF signalling in the pathogenesis of diverse tumor types, and clin. reagents that specifically target the FGFs or FGF receptors are being developed.  Although FGF signalling can drive tumorigenesis, in different contexts FGF signalling can mediate tumor protective functions; the identification of the mechanisms that underlie these differential effects will be important to understand how FGF signalling can be most appropriately therapeutically targeted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUBfz6uAu97bVg90H21EOLACvtfcHk0lg2QDLnt6cesg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXps1Gktg%253D%253D&md5=7963bc75c149981560dffc8d91b4aaf1</span></div><a href="/servlet/linkout?suffix=cit1a&amp;dbid=16384&amp;doi=10.1038%2Fnrc2780&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2780%26sid%3Dliteratum%253Aachs%26aulast%3DTurner%26aufirst%3DN.%26aulast%3DGrose%26aufirst%3DR.%26atitle%3DFibroblast%2520growth%2520factor%2520signaling%253A%2520from%2520development%2520to%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2010%26volume%3D10%26spage%3D116%26epage%3D129%26doi%3D10.1038%2Fnrc2780" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit1b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Beenken, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohammadi, M.</span></span> <span> </span><span class="NLM_article-title">The FGF family: biology, pathophysiology and therapy</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">253</span>, <span class="refDoi"> DOI: 10.1038/nrd2792</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1038%2Fnrd2792" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=19247306" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BD1MXisVShur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=235-253&author=A.+Beenkenauthor=M.+Mohammadi&title=The+FGF+family%3A+biology%2C+pathophysiology+and+therapy&doi=10.1038%2Fnrd2792"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1bR"><div class="casContent"><span class="casTitleNuber">1b</span><div class="casTitle"><span class="NLM_cas:atitle">The FGF family: biology, pathophysiology and therapy</span></div><div class="casAuthors">Beenken, Andrew; Mohammadi, Moosa</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">235-253</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The family of fibroblast growth factors (FGFs) regulates a plethora of developmental processes, including brain patterning, branching morphogenesis and limb development.  Several mitogenic, cytoprotective and angiogenic therapeutic applications of FGFs are already being explored, and the recent discovery of the crucial roles of the endocrine-acting FGF19 subfamily in bile acid, glucose and phosphate homeostasis has sparked renewed interest in the pharmacol. potential of this family.  This review discusses traditional applications of recombinant FGFs and small-mol. FGF receptor kinase inhibitors in the treatment of cancer and cardiovascular disease and their emerging potential in the treatment of metabolic syndrome and hypophosphatemic diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmidKyrdG0lbVg90H21EOLACvtfcHk0lh-HYvqVzrx9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXisVShur4%253D&md5=4789ed4cb521a92114063a400a138645</span></div><a href="/servlet/linkout?suffix=cit1b&amp;dbid=16384&amp;doi=10.1038%2Fnrd2792&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2792%26sid%3Dliteratum%253Aachs%26aulast%3DBeenken%26aufirst%3DA.%26aulast%3DMohammadi%26aufirst%3DM.%26atitle%3DThe%2520FGF%2520family%253A%2520biology%252C%2520pathophysiology%2520and%2520therapy%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2009%26volume%3D8%26spage%3D235%26epage%3D253%26doi%3D10.1038%2Fnrd2792" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Katoh, M.</span></span> <span> </span><span class="NLM_article-title">Fibroblast growth factor receptors as treatment targets in clinical oncology</span>. <i>Nat. Rev. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">105</span>– <span class="NLM_lpage">122</span>, <span class="refDoi"> DOI: 10.1038/s41571-018-0115-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1038%2Fs41571-018-0115-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=30367139" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVaqt7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2019&pages=105-122&author=M.+Katoh&title=Fibroblast+growth+factor+receptors+as+treatment+targets+in+clinical+oncology&doi=10.1038%2Fs41571-018-0115-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2aR"><div class="casContent"><span class="casTitleNuber">2a</span><div class="casTitle"><span class="NLM_cas:atitle">Fibroblast growth factor receptors as treatment targets in clinical oncology</span></div><div class="casAuthors">Katoh, Masaru</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Clinical Oncology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">105-122</span>CODEN:
                <span class="NLM_cas:coden">NRCOAA</span>;
        ISSN:<span class="NLM_cas:issn">1759-4774</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  FGFRs are receptor tyrosine kinases with a role in several biol. processes, such as the regulation of development and tissue repair.  However, alterations in FGFRs 1-4, such as amplifications, fusions and mutations, as well as aberrant epigenetic or transcriptional regulation and changes in tumor-stromal interactions in the tumor microenvironment, can lead to the development and/or progression of cancer.  Similar to other kinase alterations, such alterations are targetable using small mols. or antibodies, and the benefits of FGFR inhibitors have been demonstrated in clin. trials involving subsets of patients with solid tumors harbouring FGFR alterations.  However, the response rates in patients with FGFR alterations were relatively low, and responses in patients without detectable FGFR alterations were also obsd.  In this Review, the author describes the clin. experience with FGFR inhibitors to date, and highlights key aspects that might lead to improved response rates and/or the avoidance of acquired resistance, including the selection of patients who are most likely to benefit from treatment, and the use of FGFR inhibitors in combination regimens with other agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8oCjTdYY2DLVg90H21EOLACvtfcHk0lh-HYvqVzrx9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVaqt7vO&md5=41440ace3cb26bac6d7cdb0e18b46c39</span></div><a href="/servlet/linkout?suffix=cit2a&amp;dbid=16384&amp;doi=10.1038%2Fs41571-018-0115-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41571-018-0115-y%26sid%3Dliteratum%253Aachs%26aulast%3DKatoh%26aufirst%3DM.%26atitle%3DFibroblast%2520growth%2520factor%2520receptors%2520as%2520treatment%2520targets%2520in%2520clinical%2520oncology%26jtitle%3DNat.%2520Rev.%2520Clin.%2520Oncol.%26date%3D2019%26volume%3D16%26spage%3D105%26epage%3D122%26doi%3D10.1038%2Fs41571-018-0115-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit2b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Ghedini, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ronca, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Presta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giacomini, A.</span></span> <span> </span><span class="NLM_article-title">Future applications of FGF/FGFR inhibitors in cancer</span>. <i>Expert Rev. Anticancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">861</span>– <span class="NLM_lpage">872</span>, <span class="refDoi"> DOI: 10.1080/14737140.2018.1491795</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1080%2F14737140.2018.1491795" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=29936878" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1KlsLrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2018&pages=861-872&author=G.+C.+Ghediniauthor=R.+Roncaauthor=M.+Prestaauthor=A.+Giacomini&title=Future+applications+of+FGF%2FFGFR+inhibitors+in+cancer&doi=10.1080%2F14737140.2018.1491795"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2bR"><div class="casContent"><span class="casTitleNuber">2b</span><div class="casTitle"><span class="NLM_cas:atitle">Future applications of FGF/FGFR inhibitors in cancer</span></div><div class="casAuthors">Ghedini, Gaia Cristina; Ronca, Roberto; Presta, Marco; Giacomini, Arianna</div><div class="citationInfo"><span class="NLM_cas:title">Expert Review of Anticancer Therapy</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">861-872</span>CODEN:
                <span class="NLM_cas:coden">ERATBJ</span>;
        ISSN:<span class="NLM_cas:issn">1473-7140</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  Deregulation of the fibroblast growth factor (FGF)/FGF receptor (FGFR) network occurs frequently in tumors due to gene amplification, activating mutations, and oncogenic fusions.  Thus, the development of FGF/FGFR-targeting therapies is the focus of several basic, preclin., and clin. studies.  This review will recapitulate the status of current FGF/FGFR-targeted drugs.  Non-selective FGF/FGFR inhibitors have been approved for cancer treatment but evidence highlights various complications affecting their use in the clin. practice.  It appears mandatory to identify FGF/FGFR alterations and appropriate biomarkers that may predict and monitor response to treatment, to establish the contribution of the FGF/FGFR system to the onset of mechanisms of drug resistance, and to develop effective combinations of FGF/FGFR inhibitors with other targeted therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4xcl4aHaVQ7Vg90H21EOLACvtfcHk0lh-HYvqVzrx9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1KlsLrF&md5=98c33ddab04d181711a9439f7ebc75ca</span></div><a href="/servlet/linkout?suffix=cit2b&amp;dbid=16384&amp;doi=10.1080%2F14737140.2018.1491795&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F14737140.2018.1491795%26sid%3Dliteratum%253Aachs%26aulast%3DGhedini%26aufirst%3DG.%2BC.%26aulast%3DRonca%26aufirst%3DR.%26aulast%3DPresta%26aufirst%3DM.%26aulast%3DGiacomini%26aufirst%3DA.%26atitle%3DFuture%2520applications%2520of%2520FGF%252FFGFR%2520inhibitors%2520in%2520cancer%26jtitle%3DExpert%2520Rev.%2520Anticancer%2520Ther.%26date%3D2018%26volume%3D18%26spage%3D861%26epage%3D872%26doi%3D10.1080%2F14737140.2018.1491795" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.,  Jr.</span></span> <span> </span><span class="NLM_article-title">The role of fibroblast growth factor receptor (FGFR) protein-tyrosine kinase inhibitors in the treatment of cancers including those of the urinary bladder</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>151</i></span>,  <span class="NLM_fpage">104567</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2019.104567</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1016%2Fj.phrs.2019.104567" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=31770593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlegtr3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2020&pages=104567&author=R.+Roskoski&title=The+role+of+fibroblast+growth+factor+receptor+%28FGFR%29+protein-tyrosine+kinase+inhibitors+in+the+treatment+of+cancers+including+those+of+the+urinary+bladder&doi=10.1016%2Fj.phrs.2019.104567"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3aR"><div class="casContent"><span class="casTitleNuber">3a</span><div class="casTitle"><span class="NLM_cas:atitle">The role of fibroblast growth factor receptor (FGFR) protein-tyrosine kinase inhibitors in the treatment of cancers including those of the urinary bladder</span></div><div class="casAuthors">Roskoski, Robert Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">104567</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The human fibroblast growth factor family consists of 22 factors and five transmembrane receptors.  Of the 22 factors, eighteen are secreted while four of them function exclusively within the cell.  Four of the fibroblast growth factor receptors (FGFRs) possess intracellular protein-tyrosine kinase activity while the fifth (FGFRL1) has a short 105-residue intracellular non-enzymic component.  The FGFR protein kinase domain consists of a bi-lobed structure that is similar to that of all other protein kinases.  FGFR gene alterations occur in a wide variety of cancers including those of the urinary bladder, breast, ovary, prostate, endometrium, lung, and stomach.  The majority (66 %) of FGFR gene alterations involve gene amplifications, followed by mutations (26 %), and rearrangements that produce fusion proteins (8 %).  Erdafitinib was the first orally effective FGFR antagonist approved by the FDA (2019) for the treatment of advanced cancer, that of the urinary bladder.  FGF23 suppresses phosphate reabsorption in the proximal tubules of the kidney; FGF23 blockade allows phosphate reabsorption to occur and leads to elevated serum phosphate levels.  Erdafitinib and several other, but not all, FGFR antagonists produce hyperphosphatemia.  Erdafitinib binds to an inactive DGF-Din conformation of FGFR1 and is classified as a type I1/2 inhibitor.  Similarly, dovitinib, AZD4547, CH5183284, infigratinib, lenvatinib, LY2874455, and lucitanib are type I1/2 inhibitors.  The inactive conformations contain an autoinhibitory brake that is made up of three main residues: an asparagine (N) within the αC-β4 back loop, a glutamate (E) corresponding to the second hinge residue, and a lysine (K) in the β8-strand (the NEK triad).  PDGFRα/β, Kit, CSF1R, VEGFR1/2/3, Flt3, Tek, and Tie protein kinases are also regulated by a similar autoinhibitory brake mechanism.  Ponatinib binds to FGFR4 in a DFG-Dout conformation and is classified as a type II inhibitor.  Futibatinib, roblitinib, H3B-6527, fisogatinib, and PRN1371 bind covalently to their FGFR target and are classified as type VI inhibitors.  Nintedanib, pazopanib, pemigatinib, rogaratinib, fisogatinib, and PRN1371 are FGFR inhibitors lacking drug-enzyme crystal structures.  All of the aforementioned FGFR antagonists are orally effective.  The development of FGFR inhibitors has lagged behind those of other receptor protein-tyrosine kinases.  However, the FDA approval of erdafitinib for the treatment of urinary bladder cancers may stimulate addnl. work targeting the many other FGFR-driven neoplasms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoocEMK0pmYkbVg90H21EOLACvtfcHk0lijJFI7iLYkxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlegtr3I&md5=004af19e5027e49768812596aebef459</span></div><a href="/servlet/linkout?suffix=cit3a&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2019.104567&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2019.104567%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DThe%2520role%2520of%2520fibroblast%2520growth%2520factor%2520receptor%2520%2528FGFR%2529%2520protein-tyrosine%2520kinase%2520inhibitors%2520in%2520the%2520treatment%2520of%2520cancers%2520including%2520those%2520of%2520the%2520urinary%2520bladder%26jtitle%3DPharmacol.%2520Res.%26date%3D2020%26volume%3D151%26spage%3D104567%26doi%3D10.1016%2Fj.phrs.2019.104567" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit3b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, W.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, L.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.-Y.</span></span> <span> </span><span class="NLM_article-title">Recent developments and advances of FGFR as a potential target in cancer</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">2109</span>– <span class="NLM_lpage">2126</span>, <span class="refDoi"> DOI: 10.4155/fmc-2018-0103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.4155%2Ffmc-2018-0103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=30066580" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsF2gsLzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=2109-2126&author=W.-J.+Xueauthor=M.-T.+Liauthor=L.+Chenauthor=L.-P.+Sunauthor=Y.-Y.+Li&title=Recent+developments+and+advances+of+FGFR+as+a+potential+target+in+cancer&doi=10.4155%2Ffmc-2018-0103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3bR"><div class="casContent"><span class="casTitleNuber">3b</span><div class="casTitle"><span class="NLM_cas:atitle">Recent developments and advances of FGFR as a potential target in cancer</span></div><div class="casAuthors">Xue, Wen-Jun; Li, Ming-Tao; Chen, Lin; Sun, Li-Ping; Li, Yu-Yan</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2109-2126</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">FGFs and their receptors (FGFRs) are crit. for many biol. processes, including angiogenesis, wound healing and tissue regeneration.  Aberrations in FGFR signaling are common in cancer, making FGFRs a promising target in antitumor studies.  To date, many FGFR inhibitors are being detected in clin. studies, and resistance to some inhibitors has emerged.  Understanding the mechanisms of resistance is a fundamental step for further implementation of targeted therapies.  In this review, we will describe the basic knowledge regarding FGF/FGFR signaling and categorize the clin. FGFR inhibitors.  The mechanisms of resistance to FGFR inhibitors and corresponding strategies of overcoming drug resistance will also be discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXiPsfDmCflbVg90H21EOLACvtfcHk0lijJFI7iLYkxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsF2gsLzL&md5=98c18a707727b4bc20961fa664cfabb0</span></div><a href="/servlet/linkout?suffix=cit3b&amp;dbid=16384&amp;doi=10.4155%2Ffmc-2018-0103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc-2018-0103%26sid%3Dliteratum%253Aachs%26aulast%3DXue%26aufirst%3DW.-J.%26aulast%3DLi%26aufirst%3DM.-T.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DSun%26aufirst%3DL.-P.%26aulast%3DLi%26aufirst%3DY.-Y.%26atitle%3DRecent%2520developments%2520and%2520advances%2520of%2520FGFR%2520as%2520a%2520potential%2520target%2520in%2520cancer%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2018%26volume%3D10%26spage%3D2109%26epage%3D2126%26doi%3D10.4155%2Ffmc-2018-0103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brameld, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owens, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verner, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venetsanakos, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradshaw, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, V. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tam, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaStant, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loughhead, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ton, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karr, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerritsen, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldstein, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Funk, J. O.</span></span> <span> </span><span class="NLM_article-title">Discovery of the irreversible covalent FGFR inhibitor 8-(3-(4-acryloylpiperazin-1-yl)propyl)-6-(2,6-dichloro-3,5-dimethoxyphenyl)-2-(methylamino)pyrido[2,3-<i>d</i>]pyrimidin-7(8<i>H</i>)-one (PRN1371) for the treatment of solid tumors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">6516</span>– <span class="NLM_lpage">6527</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00360</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00360" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVykt7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=6516-6527&author=K.+A.+Brameldauthor=T.+D.+Owensauthor=E.+Vernerauthor=E.+Venetsanakosauthor=J.+M.+Bradshawauthor=V.+T.+Phanauthor=D.+Tamauthor=K.+Leungauthor=J.+Shuauthor=J.+LaStantauthor=D.+G.+Loughheadauthor=T.+Tonauthor=D.+E.+Karrauthor=M.+E.+Gerritsenauthor=D.+M.+Goldsteinauthor=J.+O.+Funk&title=Discovery+of+the+irreversible+covalent+FGFR+inhibitor+8-%283-%284-acryloylpiperazin-1-yl%29propyl%29-6-%282%2C6-dichloro-3%2C5-dimethoxyphenyl%29-2-%28methylamino%29pyrido%5B2%2C3-d%5Dpyrimidin-7%288H%29-one+%28PRN1371%29+for+the+treatment+of+solid+tumors&doi=10.1021%2Facs.jmedchem.7b00360"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of the Irreversible Covalent FGFR Inhibitor 8-(3-(4-Acryloylpiperazin-1-yl)propyl)-6-(2,6-dichloro-3,5-dimethoxyphenyl)-2-(methylamino)pyrido[2,3-d]pyrimidin-7(8H)-one (PRN1371) for the Treatment of Solid Tumors</span></div><div class="casAuthors">Brameld, Ken A.; Owens, Timothy D.; Verner, Erik; Venetsanakos, Eleni; Bradshaw, J. Michael; Phan, Vernon T.; Tam, Danny; Leung, Kwan; Shu, Jin; LaStant, Jacob; Loughhead, David G.; Ton, Tony; Karr, Dane E.; Gerritsen, Mary E.; Goldstein, David M.; Funk, Jens Oliver</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6516-6527</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Aberrant signaling of the FGF/FGFR pathway occurs frequently in cancers and is an oncogenic driver in many solid tumors.  Clin. validation of FGFR as a therapeutic target has been demonstrated in bladder, liver, lung, breast, and gastric cancers.  Our goal was to develop an irreversible covalent inhibitor of FGFR1-4 for use in oncol. indications.  An irreversible covalent binding mechanism imparts many desirable pharmacol. benefits including high potency, selectivity, and prolonged target inhibition.  Herein we report the structure-based design, medicinal chem. optimization, and unique ADME assays of our irreversible covalent drug discovery program which culminated in the discovery of compd. 34 (PRN1371), a highly selective and potent FGFR1-4 inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpklTuvm1niy7Vg90H21EOLACvtfcHk0lijJFI7iLYkxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVykt7fK&md5=987069b645d5e0e36ccda5e5c81520b9</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00360&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00360%26sid%3Dliteratum%253Aachs%26aulast%3DBrameld%26aufirst%3DK.%2BA.%26aulast%3DOwens%26aufirst%3DT.%2BD.%26aulast%3DVerner%26aufirst%3DE.%26aulast%3DVenetsanakos%26aufirst%3DE.%26aulast%3DBradshaw%26aufirst%3DJ.%2BM.%26aulast%3DPhan%26aufirst%3DV.%2BT.%26aulast%3DTam%26aufirst%3DD.%26aulast%3DLeung%26aufirst%3DK.%26aulast%3DShu%26aufirst%3DJ.%26aulast%3DLaStant%26aufirst%3DJ.%26aulast%3DLoughhead%26aufirst%3DD.%2BG.%26aulast%3DTon%26aufirst%3DT.%26aulast%3DKarr%26aufirst%3DD.%2BE.%26aulast%3DGerritsen%26aufirst%3DM.%2BE.%26aulast%3DGoldstein%26aufirst%3DD.%2BM.%26aulast%3DFunk%26aufirst%3DJ.%2BO.%26atitle%3DDiscovery%2520of%2520the%2520irreversible%2520covalent%2520FGFR%2520inhibitor%25208-%25283-%25284-acryloylpiperazin-1-yl%2529propyl%2529-6-%25282%252C6-dichloro-3%252C5-dimethoxyphenyl%2529-2-%2528methylamino%2529pyrido%255B2%252C3-d%255Dpyrimidin-7%25288H%2529-one%2520%2528PRN1371%2529%2520for%2520the%2520treatment%2520of%2520solid%2520tumors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D6516%26epage%3D6527%26doi%3D10.1021%2Facs.jmedchem.7b00360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bono, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Smet, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herbert, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Bock, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georgiadou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fons, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tjwa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alcouffe, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ny, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bianciotto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jonckx, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murakami, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanahan, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michielsen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sibrac, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dol-Gleizes, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazzone, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zacchigna, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herault, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rigon, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz de
Almodovar, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Claes, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blanc, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poesen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Segura, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gueguen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bordes, M.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambrechts, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Broussy, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van de Wouwer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michaux, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jean, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blacher, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Motte, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rom, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rakic, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katsuma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaeffer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yayon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Schepdael, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwalbe, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gervasio, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmeliet, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rozensky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dewerchin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simons, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herbert, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmeliet, P.</span></span> <span> </span><span class="NLM_article-title">Inhibition of tumor angiogenesis and growth by a small-molecule multi-FGF receptor blocker with allosteric properties</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">477</span>– <span class="NLM_lpage">488</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2013.02.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1016%2Fj.ccr.2013.02.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=23597562" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmtV2nsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=477-488&author=F.+Bonoauthor=F.+De+Smetauthor=C.+Herbertauthor=K.+De+Bockauthor=M.+Georgiadouauthor=P.+Fonsauthor=M.+Tjwaauthor=C.+Alcouffeauthor=A.+Nyauthor=M.+Bianciottoauthor=B.+Jonckxauthor=M.+Murakamiauthor=A.+A.+Lanahanauthor=C.+Michielsenauthor=D.+Sibracauthor=F.+Dol-Gleizesauthor=M.+Mazzoneauthor=S.+Zacchignaauthor=J.-P.+Heraultauthor=C.+Fischerauthor=P.+Rigonauthor=C.+Ruiz+de%0AAlmodovarauthor=F.+Claesauthor=I.+Blancauthor=K.+Poesenauthor=J.+Zhangauthor=I.+Seguraauthor=G.+Gueguenauthor=M.-F.+Bordesauthor=D.+Lambrechtsauthor=R.+Broussyauthor=M.+van+de+Wouwerauthor=C.+Michauxauthor=T.+Shimadaauthor=I.+Jeanauthor=S.+Blacherauthor=A.+Noelauthor=P.+Motteauthor=E.+Romauthor=J.-M.+Rakicauthor=S.+Katsumaauthor=P.+Schaefferauthor=A.+Yayonauthor=A.+Van+Schepdaelauthor=H.+Schwalbeauthor=F.+L.+Gervasioauthor=G.+Carmelietauthor=J.+Rozenskyauthor=M.+Dewerchinauthor=M.+Simonsauthor=A.+Christopoulosauthor=J.-M.+Herbertauthor=P.+Carmeliet&title=Inhibition+of+tumor+angiogenesis+and+growth+by+a+small-molecule+multi-FGF+receptor+blocker+with+allosteric+properties&doi=10.1016%2Fj.ccr.2013.02.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of Tumor Angiogenesis and Growth by a Small-Molecule Multi-FGF Receptor Blocker with Allosteric Properties</span></div><div class="casAuthors">Bono, Francoise; De Smet, Frederik; Herbert, Corentin; De Bock, Katrien; Georgiadou, Maria; Fons, Pierre; Tjwa, Marc; Alcouffe, Chantal; Ny, Annelii; Bianciotto, Marc; Jonckx, Bart; Murakami, Masahiro; Lanahan, Anthony A.; Michielsen, Christof; Sibrac, David; Dol-Gleizes, Frederique; Mazzone, Massimiliano; Zacchigna, Serena; Herault, Jean-Pascal; Fischer, Christian; Rigon, Patrice; Ruiz de Almodovar, Carmen; Claes, Filip; Blanc, Isabelle; Poesen, Koen; Zhang, Jie; Segura, Inmaculada; Gueguen, Genevieve; Bordes, Marie-Francoise; Lambrechts, Diether; Broussy, Roselyne; van de Wouwer, Marlies; Michaux, Corinne; Shimada, Toru; Jean, Isabelle; Blacher, Silvia; Noel, Agnes; Motte, Patrick; Rom, Eran; Rakic, Jean-Marie; Katsuma, Susumu; Schaeffer, Paul; Yayon, Avner; Van Schepdael, Ann; Schwalbe, Harald; Gervasio, Francesco Luigi; Carmeliet, Geert; Rozensky, Jef; Dewerchin, Mieke; Simons, Michael; Christopoulos, Arthur; Herbert, Jean-Marc; Carmeliet, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">477-488</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Receptor tyrosine kinases (RTK) are targets for anticancer drug development.  To date, only RTK inhibitors that block orthosteric binding of ligands and substrates have been developed.  Here, we report the pharmacol. characterization of the chem. SSR128129E (SSR), which inhibits fibroblast growth factor receptor (FGFR) signaling by binding to the extracellular FGFR domain without affecting orthosteric FGF binding.  SSR exhibits allosteric properties, including probe dependence, signaling bias, and ceiling effects.  Inhibition by SSR is highly conserved throughout the animal kingdom.  Oral delivery of SSR inhibits arthritis and tumors that are relatively refractory to anti-vascular endothelial growth factor receptor-2 antibodies.  Thus, orally-active extracellularly acting small-mol. modulators of RTKs with allosteric properties can be developed and may offer opportunities to improve anticancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuE-3w5S7QY7Vg90H21EOLACvtfcHk0lgazFLMlynZiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmtV2nsLg%253D&md5=ee0c854130ad23f910eb4783e081e6e5</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2013.02.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2013.02.019%26sid%3Dliteratum%253Aachs%26aulast%3DBono%26aufirst%3DF.%26aulast%3DDe%2BSmet%26aufirst%3DF.%26aulast%3DHerbert%26aufirst%3DC.%26aulast%3DDe%2BBock%26aufirst%3DK.%26aulast%3DGeorgiadou%26aufirst%3DM.%26aulast%3DFons%26aufirst%3DP.%26aulast%3DTjwa%26aufirst%3DM.%26aulast%3DAlcouffe%26aufirst%3DC.%26aulast%3DNy%26aufirst%3DA.%26aulast%3DBianciotto%26aufirst%3DM.%26aulast%3DJonckx%26aufirst%3DB.%26aulast%3DMurakami%26aufirst%3DM.%26aulast%3DLanahan%26aufirst%3DA.%2BA.%26aulast%3DMichielsen%26aufirst%3DC.%26aulast%3DSibrac%26aufirst%3DD.%26aulast%3DDol-Gleizes%26aufirst%3DF.%26aulast%3DMazzone%26aufirst%3DM.%26aulast%3DZacchigna%26aufirst%3DS.%26aulast%3DHerault%26aufirst%3DJ.-P.%26aulast%3DFischer%26aufirst%3DC.%26aulast%3DRigon%26aufirst%3DP.%26aulast%3DRuiz%2Bde%2BAlmodovar%26aufirst%3DC.%26aulast%3DClaes%26aufirst%3DF.%26aulast%3DBlanc%26aufirst%3DI.%26aulast%3DPoesen%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DSegura%26aufirst%3DI.%26aulast%3DGueguen%26aufirst%3DG.%26aulast%3DBordes%26aufirst%3DM.-F.%26aulast%3DLambrechts%26aufirst%3DD.%26aulast%3DBroussy%26aufirst%3DR.%26aulast%3Dvan%2Bde%2BWouwer%26aufirst%3DM.%26aulast%3DMichaux%26aufirst%3DC.%26aulast%3DShimada%26aufirst%3DT.%26aulast%3DJean%26aufirst%3DI.%26aulast%3DBlacher%26aufirst%3DS.%26aulast%3DNoel%26aufirst%3DA.%26aulast%3DMotte%26aufirst%3DP.%26aulast%3DRom%26aufirst%3DE.%26aulast%3DRakic%26aufirst%3DJ.-M.%26aulast%3DKatsuma%26aufirst%3DS.%26aulast%3DSchaeffer%26aufirst%3DP.%26aulast%3DYayon%26aufirst%3DA.%26aulast%3DVan%2BSchepdael%26aufirst%3DA.%26aulast%3DSchwalbe%26aufirst%3DH.%26aulast%3DGervasio%26aufirst%3DF.%2BL.%26aulast%3DCarmeliet%26aufirst%3DG.%26aulast%3DRozensky%26aufirst%3DJ.%26aulast%3DDewerchin%26aufirst%3DM.%26aulast%3DSimons%26aufirst%3DM.%26aulast%3DChristopoulos%26aufirst%3DA.%26aulast%3DHerbert%26aufirst%3DJ.-M.%26aulast%3DCarmeliet%26aufirst%3DP.%26atitle%3DInhibition%2520of%2520tumor%2520angiogenesis%2520and%2520growth%2520by%2520a%2520small-molecule%2520multi-FGF%2520receptor%2520blocker%2520with%2520allosteric%2520properties%26jtitle%3DCancer%2520Cell%26date%3D2013%26volume%3D23%26spage%3D477%26epage%3D488%26doi%3D10.1016%2Fj.ccr.2013.02.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Bartz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukuchi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohtsuka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lange, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gruner, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watanabe, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawaida, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komori, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kashimoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wirtz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, J-P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Redondo-Muller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saito, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanzawa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imai, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haussinger, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agatsuma, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bange, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abraham, R.</span></span> <span> </span><span class="NLM_article-title">Preclinical development of U3–1784, a Novel FGFR4 antibody against cancer, and avoidance of its on-target toxicity</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1832</span>– <span class="NLM_lpage">1843</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-18-0048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1158%2F1535-7163.MCT-18-0048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=31350344" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BB3cXmsFKruw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=1832-1843&author=R.+Bartzauthor=K.+Fukuchiauthor=T.+Ohtsukaauthor=T.+Langeauthor=K.+Grunerauthor=I.+Watanabeauthor=S.+Hayashiauthor=Y.+Odaauthor=R.+Kawaidaauthor=H.+Komoriauthor=Y.+Kashimotoauthor=P.+Wirtzauthor=J-P.+A.+Mayerauthor=M.+Redondo-Mullerauthor=S.+Saitoauthor=M.+Takahashiauthor=H.+Hanzawaauthor=E.+Imaiauthor=A.+Martinezauthor=M.+Hanaiauthor=D.+Haussingerauthor=R.+W.+Chapmanauthor=T.+Agatsumaauthor=J.+Bangeauthor=R.+Abraham&title=Preclinical+development+of+U3%E2%80%931784%2C+a+Novel+FGFR4+antibody+against+cancer%2C+and+avoidance+of+its+on-target+toxicity&doi=10.1158%2F1535-7163.MCT-18-0048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6aR"><div class="casContent"><span class="casTitleNuber">6a</span><div class="casTitle"><span class="NLM_cas:atitle">Preclinical development of U3-1784, a novel FGFR4 antibody against cancer, and avoidance of its on-target toxicity</span></div><div class="casAuthors">Bartz, Rene; Fukuchi, Keisuke; Ohtsuka, Toshiaki; Lange, Tanja; Gruner, Katrin; Watanabe, Ichiro; Hayashi, Shinko; Oda, Yoko; Kawaida, Reimi; Komori, Hironobu; Kashimoto, Yoshinori; Wirtz, Peter; Mayer, Jan-Peter A.; Redondo-Mueller, Mauricio; Saito, Shuntaro; Takahashi, Mizuki; Hanzawa, Hiroyuki; Imai, Emi; Martinez, Alberto; Hanai, Masaharu; Haeussinger, Dieter; Chapman, Roger W.; Agatsuma, Toshinori; Bange, Johannes; Abraham, Reimar</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1832-1843</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The FGFR4/FGF19 signaling axis is overactivated in 20% of liver tumors and currently represents a promising targetable signaling mechanism in this cancer type.  However, blocking FGFR4 or FGF19 has proven challenging due to its physiol. role in suppressing bile acid synthesis which leads to increased toxic bile acid plasma levels upon FGFR4 inhibition.  An FGFR4-targeting antibody, U3-1784, was generated in order to investigate its suitability as a cancer treatment without major side effects.  U3-1784 is a high-affinity fully human antibody that was obtained by phage display technol. and specifically binds to FGFR4.  The antibody inhibits cell signaling by competing with various FGFs for their FGFR4 binding site thereby inhibiting receptor activation and downstream signaling via FRS2 and Erk.  The antibody specifically slowed tumor growth of models overexpressing FGF19 by up to 90% whereas tumor growth of models not expressing FGF19 was unaffected.  In cynomolgus monkeys, i.v. injection of U3-1784 caused elevated serum bile acid and liver enzyme levels indicating potential liver damage.  These effects could be completely prevented by the concomitant oral treatment with the bile acid sequestrant colestyramine, which binds and eliminates bile acids in the gut.  These results offer a new biomarker-driven treatment modality in liver cancer without toxicity and they suggest a general strategy for avoiding adverse events with FGFR4 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7V7MCgojB7bVg90H21EOLACvtfcHk0lh4AMGp7agszQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXmsFKruw%253D%253D&md5=22e8bfdf0a246648b637074de07af74f</span></div><a href="/servlet/linkout?suffix=cit6a&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-18-0048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-18-0048%26sid%3Dliteratum%253Aachs%26aulast%3DBartz%26aufirst%3DR.%26aulast%3DFukuchi%26aufirst%3DK.%26aulast%3DOhtsuka%26aufirst%3DT.%26aulast%3DLange%26aufirst%3DT.%26aulast%3DGruner%26aufirst%3DK.%26aulast%3DWatanabe%26aufirst%3DI.%26aulast%3DHayashi%26aufirst%3DS.%26aulast%3DOda%26aufirst%3DY.%26aulast%3DKawaida%26aufirst%3DR.%26aulast%3DKomori%26aufirst%3DH.%26aulast%3DKashimoto%26aufirst%3DY.%26aulast%3DWirtz%26aufirst%3DP.%26aulast%3DMayer%26aufirst%3DJ-P.%2BA.%26aulast%3DRedondo-Muller%26aufirst%3DM.%26aulast%3DSaito%26aufirst%3DS.%26aulast%3DTakahashi%26aufirst%3DM.%26aulast%3DHanzawa%26aufirst%3DH.%26aulast%3DImai%26aufirst%3DE.%26aulast%3DMartinez%26aufirst%3DA.%26aulast%3DHanai%26aufirst%3DM.%26aulast%3DHaussinger%26aufirst%3DD.%26aulast%3DChapman%26aufirst%3DR.%2BW.%26aulast%3DAgatsuma%26aufirst%3DT.%26aulast%3DBange%26aufirst%3DJ.%26aulast%3DAbraham%26aufirst%3DR.%26atitle%3DPreclinical%2520development%2520of%2520U3%25E2%2580%25931784%252C%2520a%2520Novel%2520FGFR4%2520antibody%2520against%2520cancer%252C%2520and%2520avoidance%2520of%2520its%2520on-target%2520toxicity%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2019%26volume%3D18%26spage%3D1832%26epage%3D1843%26doi%3D10.1158%2F1535-7163.MCT-18-0048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit6b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Desnoyers, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pai, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrando, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hötzel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carano, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Souza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qing, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohtashemi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashkenazi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, D. M.</span></span> <span> </span><span class="NLM_article-title">Targeting FGF19 inhibits tumor growth in colon cancer xenograft and FGF19 transgenic hepatocellular carcinoma models</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">97</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1210623</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1038%2Fsj.onc.1210623" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=17599042" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlSntg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2008&pages=85-97&author=L.+R.+Desnoyersauthor=R.+Paiauthor=R.+E.+Ferrandoauthor=K.+H%C3%B6tzelauthor=T.+Leauthor=J.+Rossauthor=R.+Caranoauthor=A.+D%E2%80%99Souzaauthor=J.+Qingauthor=I.+Mohtashemiauthor=A.+Ashkenaziauthor=D.+M.+French&title=Targeting+FGF19+inhibits+tumor+growth+in+colon+cancer+xenograft+and+FGF19+transgenic+hepatocellular+carcinoma+models&doi=10.1038%2Fsj.onc.1210623"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6bR"><div class="casContent"><span class="casTitleNuber">6b</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting FGF19 inhibits tumor growth in colon cancer xenograft and FGF19 transgenic hepatocellular carcinoma models</span></div><div class="casAuthors">Desnoyers, L. R.; Pai, R.; Ferrando, R. E.; Hoetzel, K.; Le, T.; Ross, J.; Carano, R.; D'Souza, A.; Qing, J.; Mohtashemi, I.; Ashkenazi, A.; French, D. M.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">85-97</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Although fibroblast growth factor 19 (FGF19) can promote liver carcinogenesis in mice its involvement in human cancer is not well characterized.  Here we report that FGF19 and its cognate receptor FGF receptor 4 (FGFR4) are coexpressed in primary human liver, lung and colon tumors and in a subset of human colon cancer cell lines.  To test the importance of FGF19 for tumor growth, we developed an anti-FGF19 monoclonal antibody that selectively blocks the interaction of FGF19 with FGFR4.  This antibody abolished FGF19-mediated activity in vitro and inhibited growth of colon tumor xenografts in vivo and effectively prevented hepatocellular carcinomas in FGF19 transgenic mice.  The efficacy of the antibody in these models was linked to inhibition of FGF19-dependent activation of FGFR4, FRS2, ERK and β-catenin.  These findings suggest that the inactivation of FGF19 could be beneficial for the treatment of colon cancer, liver cancer and other malignancies involving interaction of FGF19 and FGFR4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkYfmQMrOH_bVg90H21EOLACvtfcHk0lh4AMGp7agszQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlSntg%253D%253D&md5=5b8980bcf38575516b375b8c6b6e8f19</span></div><a href="/servlet/linkout?suffix=cit6b&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1210623&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1210623%26sid%3Dliteratum%253Aachs%26aulast%3DDesnoyers%26aufirst%3DL.%2BR.%26aulast%3DPai%26aufirst%3DR.%26aulast%3DFerrando%26aufirst%3DR.%2BE.%26aulast%3DH%25C3%25B6tzel%26aufirst%3DK.%26aulast%3DLe%26aufirst%3DT.%26aulast%3DRoss%26aufirst%3DJ.%26aulast%3DCarano%26aufirst%3DR.%26aulast%3DD%25E2%2580%2599Souza%26aufirst%3DA.%26aulast%3DQing%26aufirst%3DJ.%26aulast%3DMohtashemi%26aufirst%3DI.%26aulast%3DAshkenazi%26aufirst%3DA.%26aulast%3DFrench%26aufirst%3DD.%2BM.%26atitle%3DTargeting%2520FGF19%2520inhibits%2520tumor%2520growth%2520in%2520colon%2520cancer%2520xenograft%2520and%2520FGF19%2520transgenic%2520hepatocellular%2520carcinoma%2520models%26jtitle%3DOncogene%26date%3D2008%26volume%3D27%26spage%3D85%26epage%3D97%26doi%3D10.1038%2Fsj.onc.1210623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferlay, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colombet, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soerjomataram, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathers, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parkin, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piñeros, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Znaor, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bray, F.</span></span> <span> </span><span class="NLM_article-title">Estimating the global cancer incidence and mortality in 2018: GLOBOCAN sources and methods</span>. <i>Int. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>144</i></span>,  <span class="NLM_fpage">1941</span>– <span class="NLM_lpage">1953</span>, <span class="refDoi"> DOI: 10.1002/ijc.31937</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1002%2Fijc.31937" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=30350310" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVaku77O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=144&publication_year=2019&pages=1941-1953&author=J.+Ferlayauthor=M.+Colombetauthor=I.+Soerjomataramauthor=C.+Mathersauthor=D.+M.+Parkinauthor=M.+Pi%C3%B1erosauthor=A.+Znaorauthor=F.+Bray&title=Estimating+the+global+cancer+incidence+and+mortality+in+2018%3A+GLOBOCAN+sources+and+methods&doi=10.1002%2Fijc.31937"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Estimating the global cancer incidence and mortality in 2018: GLOBOCAN sources and methods</span></div><div class="casAuthors">Ferlay, J.; Colombet, M.; Soerjomataram, I.; Mathers, C.; Parkin, D. M.; Pineros, M.; Znaor, A.; Bray, F.</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Cancer</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">144</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1941-1953</span>CODEN:
                <span class="NLM_cas:coden">IJCNAW</span>;
        ISSN:<span class="NLM_cas:issn">0020-7136</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Ests. of the worldwide incidence and mortality from 36 cancers and for all cancers combined for the year 2018 are now available in the GLOBOCAN 2018 database, compiled and disseminated by the International Agency for Research on Cancer (IARC).  This paper reviews the sources and methods used in compiling the cancer statistics in 185 countries.  The validity of the national ests. depends upon the representativeness of the source information, and to take into account possible sources of bias, uncertainty intervals are now provided for the estd. sex- and site-specific all-ages no. of new cancer cases and cancer deaths.  We briefly describe the key results globally and by world region.  There were an estd. 18.1 million (95% UI: 17.5-18.7 million) new cases of cancer (17 million excluding non-melanoma skin cancer) and 9.6 million (95% UI: 9.3-9.8 million) deaths from cancer (9.5 million excluding non-melanoma skin cancer) worldwide in 2018.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFkXwTr-9_ibVg90H21EOLACvtfcHk0lh4AMGp7agszQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVaku77O&md5=db8d895060c7770078fec16e2d9eb816</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1002%2Fijc.31937&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.31937%26sid%3Dliteratum%253Aachs%26aulast%3DFerlay%26aufirst%3DJ.%26aulast%3DColombet%26aufirst%3DM.%26aulast%3DSoerjomataram%26aufirst%3DI.%26aulast%3DMathers%26aufirst%3DC.%26aulast%3DParkin%26aufirst%3DD.%2BM.%26aulast%3DPi%25C3%25B1eros%26aufirst%3DM.%26aulast%3DZnaor%26aufirst%3DA.%26aulast%3DBray%26aufirst%3DF.%26atitle%3DEstimating%2520the%2520global%2520cancer%2520incidence%2520and%2520mortality%2520in%25202018%253A%2520GLOBOCAN%2520sources%2520and%2520methods%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D2019%26volume%3D144%26spage%3D1941%26epage%3D1953%26doi%3D10.1002%2Fijc.31937" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hainaut, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gores, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amadou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plymoth, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, L. R.</span></span> <span> </span><span class="NLM_article-title">A global view of hepatocellular carcinoma: trends, risk, prevention and management</span>. <i>Nat. Rev. Gastroenterol. Hepatol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">589</span>– <span class="NLM_lpage">604</span>, <span class="refDoi"> DOI: 10.1038/s41575-019-0186-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1038%2Fs41575-019-0186-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=31439937" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A280%3ADC%252BB3Mrgs1yntg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2019&pages=589-604&author=J.+D.+Yangauthor=P.+Hainautauthor=G.+J.+Goresauthor=A.+Amadouauthor=A.+Plymothauthor=L.+R.+Roberts&title=A+global+view+of+hepatocellular+carcinoma%3A+trends%2C+risk%2C+prevention+and+management&doi=10.1038%2Fs41575-019-0186-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">A global view of hepatocellular carcinoma: trends, risk, prevention and management</span></div><div class="casAuthors">Yang Ju Dong; Gores Gregory J; Roberts Lewis R; Yang Ju Dong; Yang Ju Dong; Yang Ju Dong; Hainaut Pierre; Amadou Amina; Plymoth Amelie</div><div class="citationInfo"><span class="NLM_cas:title">Nature reviews. Gastroenterology & hepatology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">589-604</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Hepatocellular carcinoma (HCC) is the fourth most common cause of cancer-related death worldwide.  Risk factors for HCC include chronic hepatitis B and hepatitis C, alcohol addiction, metabolic liver disease (particularly nonalcoholic fatty liver disease) and exposure to dietary toxins such as aflatoxins and aristolochic acid.  All these risk factors are potentially preventable, highlighting the considerable potential of risk prevention for decreasing the global burden of HCC.  HCC surveillance and early detection increase the chance of potentially curative treatment; however, HCC surveillance is substantially underutilized, even in countries with sufficient medical resources.  Early-stage HCC can be treated curatively by local ablation, surgical resection or liver transplantation.  Treatment selection depends on tumour characteristics, the severity of underlying liver dysfunction, age, other medical comorbidities, and available medical resources and local expertise.  Catheter-based locoregional treatment is used in patients with intermediate-stage cancer.  Kinase and immune checkpoint inhibitors have been shown to be effective treatment options in patients with advanced-stage HCC.  Together, rational deployment of prevention, attainment of global goals for viral hepatitis eradication, and improvements in HCC surveillance and therapy hold promise for achieving a substantial reduction in the worldwide HCC burden within the next few decades.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT_e_17NcpJhcULxIEhxUjPfW6udTcc2ebrF1DA_Afzzrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3Mrgs1yntg%253D%253D&md5=5a0d26728621fabe00c683714a206d18</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fs41575-019-0186-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41575-019-0186-y%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DJ.%2BD.%26aulast%3DHainaut%26aufirst%3DP.%26aulast%3DGores%26aufirst%3DG.%2BJ.%26aulast%3DAmadou%26aufirst%3DA.%26aulast%3DPlymoth%26aufirst%3DA.%26aulast%3DRoberts%26aufirst%3DL.%2BR.%26atitle%3DA%2520global%2520view%2520of%2520hepatocellular%2520carcinoma%253A%2520trends%252C%2520risk%252C%2520prevention%2520and%2520management%26jtitle%3DNat.%2520Rev.%2520Gastroenterol.%2520Hepatol.%26date%3D2019%26volume%3D16%26spage%3D589%26epage%3D604%26doi%3D10.1038%2Fs41575-019-0186-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Repana, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, P.</span></span> <span> </span><span class="NLM_article-title">Targeting FGF19/FGFR4 pathway: a novel therapeutic strategy for hepatocellular carcinoma</span>. <i>Diseases</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">294</span>– <span class="NLM_lpage">305</span>, <span class="refDoi"> DOI: 10.3390/diseases3040294</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.3390%2Fdiseases3040294" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=28943626" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC1cXltFCgurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2015&pages=294-305&author=D.+Repanaauthor=P.+Ross&title=Targeting+FGF19%2FFGFR4+pathway%3A+a+novel+therapeutic+strategy+for+hepatocellular+carcinoma&doi=10.3390%2Fdiseases3040294"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting FGF19/FGFR4 pathway: a novel therapeutic strategy for hepatocellular carcinoma</span></div><div class="casAuthors">Repana, Dimitra; Ross, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Diseases</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">294-305</span>CODEN:
                <span class="NLM_cas:coden">DISECP</span>;
        ISSN:<span class="NLM_cas:issn">2079-9721</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Hepatocellular carcinoma (HCC) is a lethal cancer with limited systemic therapeutic options.  Liver carcinogenesis is a complex procedure and various pathways have been found to be deregulated which are potential targets for novel treatments.  Aberrant signalling through FGF19 and its receptor FGFR4 seems to be the oncogenic driver for a subset of HCCs and is assocd. with poor prognosis.  Inhibition of the pathway in preclin. models has shown antitumor activity and has triggered further evaluation of this strategy to in vivo models.  This review aims to describe the role of the FGF19/FGFR4 pathway in hepatocellular carcinoma and its role as a potential predictive biomarker for novel targeted agents against FGF19/FGFR4 signalling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpREsw9FM-_67Vg90H21EOLACvtfcHk0lhu7Ht9mJPpRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXltFCgurs%253D&md5=319e81e1d0135c096c86f024dc1694cf</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.3390%2Fdiseases3040294&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fdiseases3040294%26sid%3Dliteratum%253Aachs%26aulast%3DRepana%26aufirst%3DD.%26aulast%3DRoss%26aufirst%3DP.%26atitle%3DTargeting%2520FGF19%252FFGFR4%2520pathway%253A%2520a%2520novel%2520therapeutic%2520strategy%2520for%2520hepatocellular%2520carcinoma%26jtitle%3DDiseases%26date%3D2015%26volume%3D3%26spage%3D294%26epage%3D305%26doi%3D10.3390%2Fdiseases3040294" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shim, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sung, C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baek, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haq, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ansari, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chun, S.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sung, W.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zucman-Rossi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, G.</span></span> <span> </span><span class="NLM_article-title">Genomic portrait of resectable hepatocellular carcinomas: implications of RB1 and FGF19 aberrations for patient stratification</span>. <i>Hepatology</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1972</span>– <span class="NLM_lpage">1982</span>, <span class="refDoi"> DOI: 10.1002/hep.27198</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1002%2Fhep.27198" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=24798001" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVCis7%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2014&pages=1972-1982&author=S.+M.+Ahnauthor=S.+J.+Jangauthor=J.+H.+Shimauthor=D.+Kimauthor=S.+M.+Hongauthor=C.+O.+Sungauthor=D.+Baekauthor=F.+Haqauthor=A.+A.+Ansariauthor=S.+Y.+Leeauthor=S.-M.+Chunauthor=S.+Choiauthor=H.-J.+Choiauthor=J.+Kimauthor=S.+Kimauthor=S.+Hwangauthor=Y.-J.+Leeauthor=J.+Leeauthor=W.+Jungauthor=H.+Y.+Jangauthor=E.+Yangauthor=W.-K.+Sungauthor=N.+P.+Leeauthor=M.+Maoauthor=C.+Leeauthor=J.+Zucman-Rossiauthor=E.+Yuauthor=H.+C.+Leeauthor=G.+Kong&title=Genomic+portrait+of+resectable+hepatocellular+carcinomas%3A+implications+of+RB1+and+FGF19+aberrations+for+patient+stratification&doi=10.1002%2Fhep.27198"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10aR"><div class="casContent"><span class="casTitleNuber">10a</span><div class="casTitle"><span class="NLM_cas:atitle">Genomic portrait of resectable hepatocellular carcinomas: Implications of RB1 and FGF19 aberrations for patient stratification</span></div><div class="casAuthors">Ahn, Sung-Min; Jang, Se Jin; Shim, Ju Hyun; Kim, Deokhoon; Hong, Seung-Mo; Sung, Chang Ohk; Baek, Daehyun; Haq, Farhan; Ansari, Adnan Ahmad; Lee, Sun Young; Chun, Sung-Min; Choi, Seongmin; Choi, Hyun-Jeung; Kim, Jongkyu; Kim, Sukjun; Hwang, Shin; Lee, Young-Joo; Lee, Jong-eun; Jung, Wang-rim; Jang, Hye Yoon; Yang, Eunho; Sung, Wing-Kin; Lee, Nikki P.; Mao, Mao; Lee, Charles; Zucman-Rossi, Jessica; Yu, Eunsil; Lee, Han Chu; Kong, Gu</div><div class="citationInfo"><span class="NLM_cas:title">Hepatology (Hoboken, NJ, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1972-1982</span>CODEN:
                <span class="NLM_cas:coden">HPTLD9</span>;
        ISSN:<span class="NLM_cas:issn">0270-9139</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Hepatic resection is the most curative treatment option for early-stage hepatocellular carcinoma, but is assocd. with a high recurrence rate, which exceeds 50% at 5 years after surgery.  Understanding the genetic basis of hepatocellular carcinoma at surgically curable stages may enable the identification of new mol. biomarkers that accurately identify patients in need of addnl. early therapeutic interventions.  Whole exome sequencing and copy no. anal. was performed on 231 hepatocellular carcinomas (72% with hepatitis B viral infection) that were classified as early-stage hepatocellular carcinomas, candidates for surgical resection.  Recurrent mutations were validated by Sanger sequencing.  Unsupervised genomic analyses identified an assocn. between specific genetic aberrations and postoperative clin. outcomes.  Recurrent somatic mutations were identified in nine genes, including TP53, CTNNB1, AXIN1, RPS6KA3, and RB1.  Recurrent homozygous deletions in FAM123A, RB1, and CDKN2A, and high-copy amplifications in MYC, RSPO2, CCND1, and FGF19 were detected.  Pathway analyses of these genes revealed aberrations in the p53, Wnt, PIK3/Ras, cell cycle, and chromatin remodeling pathways.  RB1 mutations were significantly assocd. with cancer-specific and recurrence-free survival after resection (multivariate P = 0.038 and P = 0.012, resp.).  FGF19 amplifications, known to activate Wnt signaling, were mutually exclusive with CTNNB1 and AXIN1 mutations, and significantly assocd. with cirrhosis (P = 0.017).  Conclusion: RB1 mutations can be used as a prognostic mol. biomarker for resectable hepatocellular carcinoma.  Further study is required to investigate the potential role of FGF19 amplification in driving hepatocarcinogenesis in patients with liver cirrhosis and to investigate the potential of anti-FGF19 treatment in these patients. (Hepatol. 2014;60:1971-1981).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIUWBV3H0t5rVg90H21EOLACvtfcHk0lhu7Ht9mJPpRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVCis7%252FN&md5=57e2374a0731562433edabe109f21025</span></div><a href="/servlet/linkout?suffix=cit10a&amp;dbid=16384&amp;doi=10.1002%2Fhep.27198&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fhep.27198%26sid%3Dliteratum%253Aachs%26aulast%3DAhn%26aufirst%3DS.%2BM.%26aulast%3DJang%26aufirst%3DS.%2BJ.%26aulast%3DShim%26aufirst%3DJ.%2BH.%26aulast%3DKim%26aufirst%3DD.%26aulast%3DHong%26aufirst%3DS.%2BM.%26aulast%3DSung%26aufirst%3DC.%2BO.%26aulast%3DBaek%26aufirst%3DD.%26aulast%3DHaq%26aufirst%3DF.%26aulast%3DAnsari%26aufirst%3DA.%2BA.%26aulast%3DLee%26aufirst%3DS.%2BY.%26aulast%3DChun%26aufirst%3DS.-M.%26aulast%3DChoi%26aufirst%3DS.%26aulast%3DChoi%26aufirst%3DH.-J.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DS.%26aulast%3DHwang%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DY.-J.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DJung%26aufirst%3DW.%26aulast%3DJang%26aufirst%3DH.%2BY.%26aulast%3DYang%26aufirst%3DE.%26aulast%3DSung%26aufirst%3DW.-K.%26aulast%3DLee%26aufirst%3DN.%2BP.%26aulast%3DMao%26aufirst%3DM.%26aulast%3DLee%26aufirst%3DC.%26aulast%3DZucman-Rossi%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DE.%26aulast%3DLee%26aufirst%3DH.%2BC.%26aulast%3DKong%26aufirst%3DG.%26atitle%3DGenomic%2520portrait%2520of%2520resectable%2520hepatocellular%2520carcinomas%253A%2520implications%2520of%2520RB1%2520and%2520FGF19%2520aberrations%2520for%2520patient%2520stratification%26jtitle%3DHepatology%26date%3D2014%26volume%3D60%26spage%3D1972%26epage%3D1982%26doi%3D10.1002%2Fhep.27198" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit10b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Sawey, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chanrion, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zender, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Busuttil, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yee, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stein, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finn, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powers, S.</span></span> <span> </span><span class="NLM_article-title">Identification of a therapeutic strategy targeting amplified FGF19 in liver cancer by oncogenomic screening</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">347</span>– <span class="NLM_lpage">358</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2011.01.040</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1016%2Fj.ccr.2011.01.040" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=21397858" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjtFKisrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=347-358&author=E.+T.+Saweyauthor=M.+Chanrionauthor=C.+Caiauthor=G.+Wuauthor=J.+Zhangauthor=L.+Zenderauthor=A.+Zhaoauthor=R.+W.+Busuttilauthor=H.+Yeeauthor=L.+Steinauthor=D.+M.+Frenchauthor=R.+S.+Finnauthor=S.+W.+Loweauthor=S.+Powers&title=Identification+of+a+therapeutic+strategy+targeting+amplified+FGF19+in+liver+cancer+by+oncogenomic+screening&doi=10.1016%2Fj.ccr.2011.01.040"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10bR"><div class="casContent"><span class="casTitleNuber">10b</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a Therapeutic Strategy Targeting Amplified FGF19 in Liver Cancer by Oncogenomic Screening</span></div><div class="casAuthors">Sawey, Eric T.; Chanrion, Maia; Cai, Chunlin; Wu, Guanming; Zhang, Jianping; Zender, Lars; Zhao, Alice; Busuttil, Ronald W.; Yee, Herman; Stein, Lincoln; French, Dorothy M.; Finn, Richard S.; Lowe, Scott W.; Powers, Scott</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">347-358</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: We screened 124 genes that are amplified in human hepatocellular carcinoma (HCC) using a mouse hepatoblast model and identified 18 tumor-promoting genes, including CCND1 and its neighbor on 11q13.3, FGF19.  Although it is widely assumed that CCND1 is the main driving oncogene of this common amplicon (15% frequency in HCC), both forward-transformation assays and RNAi-mediated inhibition in human HCC cells established that FGF19 is an equally important driver gene in HCC.  Furthermore, clonal growth and tumorigenicity of HCC cells harboring the 11q13.3 amplicon were selectively inhibited by RNAi-mediated knockdown of CCND1 or FGF19, as well as by an anti-FGF19 antibody.  These results show that 11q13.3 amplification could be an effective biomarker for patients most likely to respond to anti-FGF19 therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbWNCR5PCyb7Vg90H21EOLACvtfcHk0ljOHZ4E11X9iA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjtFKisrs%253D&md5=eb5934a5f2fde11c7e7bf385582ac0af</span></div><a href="/servlet/linkout?suffix=cit10b&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2011.01.040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2011.01.040%26sid%3Dliteratum%253Aachs%26aulast%3DSawey%26aufirst%3DE.%2BT.%26aulast%3DChanrion%26aufirst%3DM.%26aulast%3DCai%26aufirst%3DC.%26aulast%3DWu%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DZender%26aufirst%3DL.%26aulast%3DZhao%26aufirst%3DA.%26aulast%3DBusuttil%26aufirst%3DR.%2BW.%26aulast%3DYee%26aufirst%3DH.%26aulast%3DStein%26aufirst%3DL.%26aulast%3DFrench%26aufirst%3DD.%2BM.%26aulast%3DFinn%26aufirst%3DR.%2BS.%26aulast%3DLowe%26aufirst%3DS.%2BW.%26aulast%3DPowers%26aufirst%3DS.%26atitle%3DIdentification%2520of%2520a%2520therapeutic%2520strategy%2520targeting%2520amplified%2520FGF19%2520in%2520liver%2520cancer%2520by%2520oncogenomic%2520screening%26jtitle%3DCancer%2520Cell%26date%3D2011%26volume%3D19%26spage%3D347%26epage%3D358%26doi%3D10.1016%2Fj.ccr.2011.01.040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicholes, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guillet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomlinson, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hillan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frantz, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pham, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillard-Telm, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephan, J.-P</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stinson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stewart, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, D. M.</span></span> <span> </span><span class="NLM_article-title">A mouse model of hepatocellular carcinoma</span>. <i>Am. J. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>160</i></span>,  <span class="NLM_fpage">2295</span>– <span class="NLM_lpage">2307</span>, <span class="refDoi"> DOI: 10.1016/S0002-9440(10)61177-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1016%2FS0002-9440%2810%2961177-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=12057932" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BD38XltFOrsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=160&publication_year=2002&pages=2295-2307&author=K.+Nicholesauthor=S.+Guilletauthor=E.+Tomlinsonauthor=K.+Hillanauthor=B.+Wrightauthor=G.+D.+Frantzauthor=T.+A.+Phamauthor=L.+Dillard-Telmauthor=S.+P.+Tsaiauthor=J.-P+Stephanauthor=J.+Stinsonauthor=T.+Stewartauthor=D.+M.+French&title=A+mouse+model+of+hepatocellular+carcinoma&doi=10.1016%2FS0002-9440%2810%2961177-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">A mouse model of hepatocellular carcinoma: Ectopic expression of fibroblast growth factor 19 in skeletal muscle of transgenic mice</span></div><div class="casAuthors">Nicholes, Katrina; Guillet, Susan; Tomlinson, Elizabeth; Hillan, Kenneth; Wright, Barbara; Frantz, Gretchen D.; Pham, Thinh A.; Dillard-Telm, Lisa; Tsai, Siao Ping; Stephan, Jean-Philippe; Stinson, Jeremy; Stewart, Timothy; French, Dorothy M.</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Pathology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">160</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2295-2307</span>CODEN:
                <span class="NLM_cas:coden">AJPAA4</span>;
        ISSN:<span class="NLM_cas:issn">0002-9440</span>.
    
            (<span class="NLM_cas:orgname">American Society for Investigative Pathology</span>)
        </div><div class="casAbstract">Most mouse models of hepatocellular carcinoma have expressed growth factors and oncogenes under the control of a liver-specific promoter.  In contrast, the authors describe here the formation of liver tumors in transgenic mice overexpressing human fibroblast growth factor 19 (FGF19) in skeletal muscle.  FGF19 transgenic mice had elevated hepatic α-fetoprotein mRNA as early as 2 mo of age, and hepatocellular carcinomas were evident by 10 mo of age.  Increased proliferation of pericentral hepatocytes was demonstrated by 5-bromo-2'-deoxyuridine incorporation in the FGF19 transgenic mice before tumor formation and in nontransgenic mice injected with recombinant FGF19 protein.  Areas of small cell dysplasia were initially evident pericentrally, and dysplastic/neoplastic foci throughout the hepatic lobule were glutamine synthetase-pos., suggestive of a pericentral origin.  Consistent with chronic activation of the Wingless/Wnt pathway, 44% of the hepatocellular tumors from FGF19 transgenic mice had nuclear staining for β-catenin.  Sequencing of the tumor DNA encoding β-catenin revealed point mutations that resulted in amino acid substitutions.  These findings suggest a previously unknown role for FGF19 in hepatocellular carcinomas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9ekQaICJqerVg90H21EOLACvtfcHk0ljOHZ4E11X9iA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XltFOrsbs%253D&md5=c3355b5f65fea242cfb7ee357e2dd4a7</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2FS0002-9440%2810%2961177-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0002-9440%252810%252961177-7%26sid%3Dliteratum%253Aachs%26aulast%3DNicholes%26aufirst%3DK.%26aulast%3DGuillet%26aufirst%3DS.%26aulast%3DTomlinson%26aufirst%3DE.%26aulast%3DHillan%26aufirst%3DK.%26aulast%3DWright%26aufirst%3DB.%26aulast%3DFrantz%26aufirst%3DG.%2BD.%26aulast%3DPham%26aufirst%3DT.%2BA.%26aulast%3DDillard-Telm%26aufirst%3DL.%26aulast%3DTsai%26aufirst%3DS.%2BP.%26aulast%3DStephan%26aufirst%3DJ.-P%26aulast%3DStinson%26aufirst%3DJ.%26aulast%3DStewart%26aufirst%3DT.%26aulast%3DFrench%26aufirst%3DD.%2BM.%26atitle%3DA%2520mouse%2520model%2520of%2520hepatocellular%2520carcinoma%26jtitle%3DAm.%2520J.%2520Pathol.%26date%3D2002%26volume%3D160%26spage%3D2295%26epage%3D2307%26doi%3D10.1016%2FS0002-9440%2810%2961177-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Nogova, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sequist, L. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez
Garcia, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andre, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delord, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hidalgo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schellens, J. H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cassier, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camidge, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaishampayan, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burris, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campone, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wainberg, Z. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, W.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LoRusso, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choudhury, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ringeisen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graus-Porta, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isaacs, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buettner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, J.</span></span> <span> </span><span class="NLM_article-title">Evaluation of BGJ398, a fibroblast growth factor receptor 1–3 kinase inhibitor, in patients with advanced solid tumors harboring genetic alterations in fibroblast growth factor receptors: results of a global phase I, dose-escalation and dose-expansion study</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">157</span>– <span class="NLM_lpage">165</span>, <span class="refDoi"> DOI: 10.1200/JCO.2016.67.2048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1200%2FJCO.2016.67.2048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=27870574" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlensr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2017&pages=157-165&author=L.+Nogovaauthor=L.+V.+Sequistauthor=J.+M.+Perez%0AGarciaauthor=F.+Andreauthor=J.-P.+Delordauthor=M.+Hidalgoauthor=J.+H.+M.+Schellensauthor=P.+A.+Cassierauthor=D.+R.+Camidgeauthor=M.+Schulerauthor=U.+Vaishampayanauthor=H.+Burrisauthor=G.+G.+Tianauthor=M.+Camponeauthor=Z.+A.+Wainbergauthor=W.-T.+Limauthor=P.+LoRussoauthor=G.+I.+Shapiroauthor=K.+Parkerauthor=X.+Chenauthor=S.+Choudhuryauthor=F.+Ringeisenauthor=D.+Graus-Portaauthor=D.+Porterauthor=R.+Isaacsauthor=R.+Buettnerauthor=J.+Wolf&title=Evaluation+of+BGJ398%2C+a+fibroblast+growth+factor+receptor+1%E2%80%933+kinase+inhibitor%2C+in+patients+with+advanced+solid+tumors+harboring+genetic+alterations+in+fibroblast+growth+factor+receptors%3A+results+of+a+global+phase+I%2C+dose-escalation+and+dose-expansion+study&doi=10.1200%2FJCO.2016.67.2048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12aR"><div class="casContent"><span class="casTitleNuber">12a</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of BGJ398, a fibroblast growth factor receptor 1-3 kinase inhibitor, in patients with advanced solid tumors harboring genetic alterations in fibroblast growth factor receptors: results of a global phase I, dose-escalation and dose-expansion study</span></div><div class="casAuthors">Nogova, Lucia; Sequist, Lecia V.; Garcia, Jose Manuel Perez; Andre, Fabrice; Delord, Jean-Pierre; Hidalgo, Manuel; Schellens, Jan H. M.; Cassier, Philippe A.; Camidge, D. Ross; Schuler, Martin; Vaishampayan, Ulka; Burris, Howard; Tian, G. Gary; Campone, Mario; Wainberg, Zev A.; Lim, Wan-Teck; Lo Russo, Patricia; Shapiro, Geoffrey I.; Parker, Katie; Chen, Xueying; Choudhury, Somesh; Ringeisen, Francois; Graus-Porta, Diana; Porter, Dale; Isaacs, Randi; Buettner, Reinhard; Wolf, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Oncology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">157-175</span>CODEN:
                <span class="NLM_cas:coden">JCONDN</span>;
        ISSN:<span class="NLM_cas:issn">1527-7755</span>.
    
            (<span class="NLM_cas:orgname">American Society of Clinical Oncology</span>)
        </div><div class="casAbstract">Purpose: This two-part, first-in-human study was initiated in patient swith advanced solid tumors harboring genetic alterations in fibroblast growth factor receptors (FGFRs) to det. the max. tolerated dose (MTD), the recommended phase II dose (RP2D), and the schedule, safety, pharmacokinetics, pharmacodynamics, and antitumor activity of oral BGJ398, a selective FGFR1-3 tyrosine kinase inhibitor.  Patients and Methods: Adult patients were treated with escalating dosages of BGJ398 5 to 150 mg once daily or 50 mg twice daily continuously in 28-day cycles.  During expansion at the MTD, patients with FGFR1-amplified squamous cell non-small-cell lung cancer (sqNSCLC; arm 1) or other solid tumors with FGFR genetic alterations (mutations/amplifications/fusions) received BGJ398 daily on a continuous schedule (arm 2), or on a 3-wk-on/1-wk-off schedule (arm 3).  Results: Data in 132 patients from the escalation and expansion arms are reported (May 15, 2015, cutoff).  The MTD, 125 mg daily, was detd. on the basis of dose-limiting toxicities in four patients (100 mg, grade 3 aminotransferase elevations [n = 1]; 125 mg, hyperphosphatemia [n = 1]; 150 mg, grade 1 corneal toxicity [n = 1] and grade 3 aminotransferase elevations [n = 1]).  Common adverse events in patients treated at the MTD (n = 57) included hyperphosphatemia (82.5%), constipation (50.9%), decreased appetite (45.6%), and stomatitis (45.6%).  A similar safety profile was obsd. using the 3-wk-on/1-wk-off schedule (RP2D).  However, adverse event-related dose adjustments/interruptions were less frequent with the 3-wk-on/1-wk-off (50.0%) vs. the continuous (73.7%) schedule.  Antitumor activity (seven partial responses [six confirmed]) was demonstrated with BGJ398 doses ≥ 100 mg in patients with FGFR1-amplified sqNSCLC and FGFR3-mutant bladder/urothelial cancer.  Conclusion: BGJ398 at the MTD/RP2D had a tolerable and manageable safety profile and showed antitumor activity in several tumor types, including FGFR1-amplified sqNSCLC and FGFR3-mutant bladder/urothelial cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0KJ9yf2nMQrVg90H21EOLACvtfcHk0lg86JWE2xv11Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlensr0%253D&md5=b0204f7a46eeaa059fe732ad35508ca9</span></div><a href="/servlet/linkout?suffix=cit12a&amp;dbid=16384&amp;doi=10.1200%2FJCO.2016.67.2048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2016.67.2048%26sid%3Dliteratum%253Aachs%26aulast%3DNogova%26aufirst%3DL.%26aulast%3DSequist%26aufirst%3DL.%2BV.%26aulast%3DPerez%2BGarcia%26aufirst%3DJ.%2BM.%26aulast%3DAndre%26aufirst%3DF.%26aulast%3DDelord%26aufirst%3DJ.-P.%26aulast%3DHidalgo%26aufirst%3DM.%26aulast%3DSchellens%26aufirst%3DJ.%2BH.%2BM.%26aulast%3DCassier%26aufirst%3DP.%2BA.%26aulast%3DCamidge%26aufirst%3DD.%2BR.%26aulast%3DSchuler%26aufirst%3DM.%26aulast%3DVaishampayan%26aufirst%3DU.%26aulast%3DBurris%26aufirst%3DH.%26aulast%3DTian%26aufirst%3DG.%2BG.%26aulast%3DCampone%26aufirst%3DM.%26aulast%3DWainberg%26aufirst%3DZ.%2BA.%26aulast%3DLim%26aufirst%3DW.-T.%26aulast%3DLoRusso%26aufirst%3DP.%26aulast%3DShapiro%26aufirst%3DG.%2BI.%26aulast%3DParker%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DChoudhury%26aufirst%3DS.%26aulast%3DRingeisen%26aufirst%3DF.%26aulast%3DGraus-Porta%26aufirst%3DD.%26aulast%3DPorter%26aufirst%3DD.%26aulast%3DIsaacs%26aufirst%3DR.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DWolf%26aufirst%3DJ.%26atitle%3DEvaluation%2520of%2520BGJ398%252C%2520a%2520fibroblast%2520growth%2520factor%2520receptor%25201%25E2%2580%25933%2520kinase%2520inhibitor%252C%2520in%2520patients%2520with%2520advanced%2520solid%2520tumors%2520harboring%2520genetic%2520alterations%2520in%2520fibroblast%2520growth%2520factor%2520receptors%253A%2520results%2520of%2520a%2520global%2520phase%2520I%252C%2520dose-escalation%2520and%2520dose-expansion%2520study%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2017%26volume%3D35%26spage%3D157%26epage%3D165%26doi%3D10.1200%2FJCO.2016.67.2048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit12b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Hierro, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tabernero, J.</span></span> <span> </span><span class="NLM_article-title">Fibroblast growth factor (FGF) receptor/FGF inhibitors: novel targets and strategies for optimization of response of solid tumors</span>. <i>Semin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">801</span>– <span class="NLM_lpage">819</span>, <span class="refDoi"> DOI: 10.1053/j.seminoncol.2015.09.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1053%2Fj.seminoncol.2015.09.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=26615127" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFWksL7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2015&pages=801-819&author=C.+Hierroauthor=J.+Rodonauthor=J.+Tabernero&title=Fibroblast+growth+factor+%28FGF%29+receptor%2FFGF+inhibitors%3A+novel+targets+and+strategies+for+optimization+of+response+of+solid+tumors&doi=10.1053%2Fj.seminoncol.2015.09.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12bR"><div class="casContent"><span class="casTitleNuber">12b</span><div class="casTitle"><span class="NLM_cas:atitle">Fibroblast Growth Factor (FGF) Receptor/FGF Inhibitors: Novel Targets and Strategies for Optimization of Response of Solid Tumors</span></div><div class="casAuthors">Hierro, Cinta; Rodon, Jordi; Tabernero, Josep</div><div class="citationInfo"><span class="NLM_cas:title">Seminars in Oncology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">801-819</span>CODEN:
                <span class="NLM_cas:coden">SOLGAV</span>;
        ISSN:<span class="NLM_cas:issn">0093-7754</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The fibroblast growth factor receptor (FGFR) pathway plays a major role in several biol. processes, from organogenesis to metab. homeostasis and angiogenesis.  Several aberrations, including gene amplifications, point mutations, and chromosomal translocations have been described across solid tumors.  Most of these mol. alterations promote multiple steps of carcinogenesis in FGFR oncogene-addicted cells, increasing cell proliferation, angiogenesis, and drug resistance.  Data suggest that upregulation of FGFR signaling is a common event in many cancer types.  The FGFR pathway thus arises as a potential promising target for cancer treatment.  Several FGFR inhibitors are currently under development.  Initial preclin. results have translated into limited successful clin. responses when first-generation, nonspecific FGFR inhibitors were evaluated in patients.  The future development of selective and unselective FGFR inhibitors will rely on a better understanding of the tissue-specific role of FGFR signaling and identification of biomarkers to select those patients who will benefit the most from these drugs.  Further studies are warranted to establish the predictive significance of the different FGFR-aberrations and to incorporate them into clin. algorithms, now that second-generation, selective FGFR inhibitors exist.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruNHv_T2hq8bVg90H21EOLACvtfcHk0lg86JWE2xv11Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFWksL7F&md5=b1b150e2f8a903aab696070b71a9247c</span></div><a href="/servlet/linkout?suffix=cit12b&amp;dbid=16384&amp;doi=10.1053%2Fj.seminoncol.2015.09.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1053%252Fj.seminoncol.2015.09.027%26sid%3Dliteratum%253Aachs%26aulast%3DHierro%26aufirst%3DC.%26aulast%3DRodon%26aufirst%3DJ.%26aulast%3DTabernero%26aufirst%3DJ.%26atitle%3DFibroblast%2520growth%2520factor%2520%2528FGF%2529%2520receptor%252FFGF%2520inhibitors%253A%2520novel%2520targets%2520and%2520strategies%2520for%2520optimization%2520of%2520response%2520of%2520solid%2520tumors%26jtitle%3DSemin.%2520Oncol.%26date%3D2015%26volume%3D42%26spage%3D801%26epage%3D819%26doi%3D10.1053%2Fj.seminoncol.2015.09.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Sarabipour, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hristova, K.</span></span> <span> </span><span class="NLM_article-title">Mechanism of FGF receptor dimerization and activation</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">10262</span>, <span class="refDoi"> DOI: 10.1038/ncomms10262</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1038%2Fncomms10262" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=26725515" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC28Xjslanug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=10262&author=S.+Sarabipourauthor=K.+Hristova&title=Mechanism+of+FGF+receptor+dimerization+and+activation&doi=10.1038%2Fncomms10262"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13aR"><div class="casContent"><span class="casTitleNuber">13a</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism of FGF receptor dimerization and activation</span></div><div class="casAuthors">Sarabipour, Sarvenaz; Hristova, Kalina</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">10262</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Fibroblast growth factors (fgfs) are widely believed to activate their receptors by mediating receptor dimerization.  Here we show, however, that the FGF receptors form dimers in the absence of ligand, and that these unliganded dimers are phosphorylated.  We further show that ligand binding triggers structural changes in the FGFR dimers, which increase FGFR phosphorylation.  The obsd. effects due to the ligands fgf1 and fgf2 are very different.  The fgf2-bound dimer structure ensures the smallest sepn. between the transmembrane (TM) domains and the highest possible phosphorylation, a conclusion that is supported by a strong correlation between TM helix sepn. in the dimer and kinase phosphorylation.  The pathogenic A391E mutation in FGFR3 TM domain emulates the action of fgf2, trapping the FGFR3 dimer in its most active state.  This study establishes the existence of multiple active ligand-bound states, and uncovers a novel mol. mechanism through which FGFR-linked pathologies can arise.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrl3F_9xgzAOLVg90H21EOLACvtfcHk0lg86JWE2xv11Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xjslanug%253D%253D&md5=3c87746c27a7e92fb437f91649ad6c8e</span></div><a href="/servlet/linkout?suffix=cit13a&amp;dbid=16384&amp;doi=10.1038%2Fncomms10262&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms10262%26sid%3Dliteratum%253Aachs%26aulast%3DSarabipour%26aufirst%3DS.%26aulast%3DHristova%26aufirst%3DK.%26atitle%3DMechanism%2520of%2520FGF%2520receptor%2520dimerization%2520and%2520activation%26jtitle%3DNat.%2520Commun.%26date%3D2016%26volume%3D7%26spage%3D10262%26doi%3D10.1038%2Fncomms10262" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit13b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Lesca, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lammens, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Augustin, M.</span></span> <span> </span><span class="NLM_article-title">Structural analysis of the human fibroblast growth factor receptor 4 kinase</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>426</i></span>,  <span class="NLM_fpage">3744</span>– <span class="NLM_lpage">3756</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2014.09.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1016%2Fj.jmb.2014.09.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=25219510" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFyitb7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=426&publication_year=2014&pages=3744-3756&author=E.+Lescaauthor=A.+Lammensauthor=R.+Huberauthor=M.+Augustin&title=Structural+analysis+of+the+human+fibroblast+growth+factor+receptor+4+kinase&doi=10.1016%2Fj.jmb.2014.09.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13bR"><div class="casContent"><span class="casTitleNuber">13b</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Analysis of the Human Fibroblast Growth Factor Receptor 4 Kinase</span></div><div class="casAuthors">Lesca, E.; Lammens, A.; Huber, R.; Augustin, M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">426</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">3744-3756</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The family of fibroblast growth factor receptors (FGFRs) plays an important and well-characterized role in a variety of pathol. disorders.  FGFR4 is involved in myogenesis and muscle regeneration.  Mutations affecting the kinase domain of FGFR4 may cause cancer, for example, breast cancer or rhabdomyosarcoma.  Whereas FGFR1-FGFR3 have been structurally characterized, the structure of the FGFR4 kinase domain has not yet been reported.  In this study, we present four structures of the kinase domain of FGFR4, in its apo-form and in complex with different types of small-mol. inhibitors.  The two apo-FGFR4 kinase domain structures show an activation segment similar in conformation to an autoinhibitory segment obsd. in the hepatocyte growth factor receptor kinase but different from the known structures of other FGFR kinases.  The structures of FGFR4 in complex with the type I inhibitor Dovitinib and the type II inhibitor Ponatinib reveal the mol. interactions with different types of kinase inhibitors and may assist in the design and development of FGFR4 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9Wm0Poe_gXrVg90H21EOLACvtfcHk0lgBEzcENb-qIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFyitb7E&md5=ee9d7f9bc9aa10dbec92f3eb6b80b44f</span></div><a href="/servlet/linkout?suffix=cit13b&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2014.09.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2014.09.004%26sid%3Dliteratum%253Aachs%26aulast%3DLesca%26aufirst%3DE.%26aulast%3DLammens%26aufirst%3DA.%26aulast%3DHuber%26aufirst%3DR.%26aulast%3DAugustin%26aufirst%3DM.%26atitle%3DStructural%2520analysis%2520of%2520the%2520human%2520fibroblast%2520growth%2520factor%2520receptor%25204%2520kinase%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2014%26volume%3D426%26spage%3D3744%26epage%3D3756%26doi%3D10.1016%2Fj.jmb.2014.09.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q.</span></span> <span> </span><span class="NLM_article-title">Role of fibroblast growth factor receptor 4 in cancer</span>. <i>Cancer Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">3024</span>– <span class="NLM_lpage">3031</span>, <span class="refDoi"> DOI: 10.1111/cas.13759</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1111%2Fcas.13759" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=30070748" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVCqtb7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2018&pages=3024-3031&author=S.+Tangauthor=Y.+Haoauthor=Y.+Yuanauthor=R.+Liuauthor=Q.+Chen&title=Role+of+fibroblast+growth+factor+receptor+4+in+cancer&doi=10.1111%2Fcas.13759"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14aR"><div class="casContent"><span class="casTitleNuber">14a</span><div class="casTitle"><span class="NLM_cas:atitle">Role of fibroblast growth factor receptor 4 in cancer</span></div><div class="casAuthors">Tang, Shuya; Hao, Yilong; Yuan, Yao; Liu, Rui; Chen, Qianming</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Science</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3024-3031</span>CODEN:
                <span class="NLM_cas:coden">CSACCM</span>;
        ISSN:<span class="NLM_cas:issn">1349-7006</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Fibroblast growth factor receptors (FGFR) play a significant role in both embryonic development and in adults.  Upon binding with ligands, FGFR signaling is activated and triggers various downstream signal cascades that are implicated in diverse biol. processes.  Aberrant regulations of FGFR signaling are detected in numerous cancers.  Although FGFR4 was discovered later than other FGFR, information on the involvement of FGFR4 in cancers has significantly increased in recent years.  In this review, the recent findings in FGFR4 structure, signaling transduction, physiol. function, aberrant regulations, and effects in cancers as well as its potential applications as an anticancer therapeutic target are summarized.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdw5EJC-Ta-rVg90H21EOLACvtfcHk0lgBEzcENb-qIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVCqtb7F&md5=d65681255072f30701fc8730fb7938fa</span></div><a href="/servlet/linkout?suffix=cit14a&amp;dbid=16384&amp;doi=10.1111%2Fcas.13759&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcas.13759%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DS.%26aulast%3DHao%26aufirst%3DY.%26aulast%3DYuan%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DQ.%26atitle%3DRole%2520of%2520fibroblast%2520growth%2520factor%2520receptor%25204%2520in%2520cancer%26jtitle%3DCancer%2520Sci.%26date%3D2018%26volume%3D109%26spage%3D3024%26epage%3D3031%26doi%3D10.1111%2Fcas.13759" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit14b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Gotoh, N.</span></span> <span> </span><span class="NLM_article-title">Regulation of growth factor signaling by FRS2 family docking/scaffold adaptor proteins</span>. <i>Cancer Sci.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">1319</span>– <span class="NLM_lpage">1325</span>, <span class="refDoi"> DOI: 10.1111/j.1349-7006.2008.00840.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1111%2Fj.1349-7006.2008.00840.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=18452557" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BD1cXot1eitrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2008&pages=1319-1325&author=N.+Gotoh&title=Regulation+of+growth+factor+signaling+by+FRS2+family+docking%2Fscaffold+adaptor+proteins&doi=10.1111%2Fj.1349-7006.2008.00840.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14bR"><div class="casContent"><span class="casTitleNuber">14b</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of growth factor signaling by FRS2 family docking/scaffold adaptor proteins</span></div><div class="casAuthors">Gotoh, Noriko</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Science</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1319-1325</span>CODEN:
                <span class="NLM_cas:coden">CSACCM</span>;
        ISSN:<span class="NLM_cas:issn">1347-9032</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Asia Pty Ltd.</span>)
        </div><div class="casAbstract">A review.  The FRS2 family of adaptor/scaffold proteins has two members, FRS2α and FRS2β.  Both proteins contain N-terminal myristylation sites for localization on the plasma membrane and a PTB domain for binding to limited species of receptor tyrosine kinases (RTKs), including the FGF receptor, the neurotrophin receptor, RET, and ALK.  Activation of these RTKs allows FRS2 proteins to become phosphorylated of tyrosine residues and then bind to Grb2 and Shp2, a SH2 domain-contg. adaptor and a tyrosine phosphatase, resp.  Subsequently, Shp2 activates a Ras/ERK pathway and Grb2 activates a Ras/ERK, phosphatidylinositol (PI)-3 kinase and ubiquitination/degrdn. pathways by binding to SOS, Gab1, and Cbl via the SH3 domains of Grb2.  FRS2α acts as 'a conning center' in FGF signaling mainly because it induces sustained levels of activation of ERK via Shp2-binding sites and Grb2-binding sites, though the contribution of the former is greater.  Indeed, FRS2α knockout mice and mice with mutated Shp2-binding sites exhibit a variety of phenotypes due to defects in FGF signaling in vivo.  Although FRS2β binds to the EGF receptor, it does not induce tyrosine phosphorylation on the receptor.  Instead, it inhibits EGF signaling, resulting in inhibition of EGF-induced cell proliferation and cell transformation.  Based on these findings, the involvement of FRS2 proteins in tumorigenesis should be studied extensively to be validated as candidate biomarkers for the effectiveness of treatments targeting RTKs such as the FGF receptor and EGF receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnJLrQGKGN0rVg90H21EOLACvtfcHk0lgBEzcENb-qIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXot1eitrg%253D&md5=51684ae103a15bd6161590568b305082</span></div><a href="/servlet/linkout?suffix=cit14b&amp;dbid=16384&amp;doi=10.1111%2Fj.1349-7006.2008.00840.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1349-7006.2008.00840.x%26sid%3Dliteratum%253Aachs%26aulast%3DGotoh%26aufirst%3DN.%26atitle%3DRegulation%2520of%2520growth%2520factor%2520signaling%2520by%2520FRS2%2520family%2520docking%252Fscaffold%2520adaptor%2520proteins%26jtitle%3DCancer%2520Sci.%26date%3D2008%26volume%3D99%26spage%3D1319%26epage%3D1325%26doi%3D10.1111%2Fj.1349-7006.2008.00840.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackmore, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desnoyers, L. R.</span></span> <span> </span><span class="NLM_article-title">Liver-specific activities of FGF19 require Klotho beta</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>282</i></span>,  <span class="NLM_fpage">27277</span>– <span class="NLM_lpage">27284</span>, <span class="refDoi"> DOI: 10.1074/jbc.M704244200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1074%2Fjbc.M704244200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=17627937" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVWjsLfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=27277-27284&author=B.+C.+Linauthor=M.+Wangauthor=C.+Blackmoreauthor=L.+R.+Desnoyers&title=Liver-specific+activities+of+FGF19+require+Klotho+beta&doi=10.1074%2Fjbc.M704244200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15aR"><div class="casContent"><span class="casTitleNuber">15a</span><div class="casTitle"><span class="NLM_cas:atitle">Liver-specific activities of FGF19 Require Klotho beta</span></div><div class="casAuthors">Lin, Benjamin C.; Wang, Manping; Blackmore, Craig; Desnoyers, Luc R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">27277-27284</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Hepatocyte function is regulated by members of the fibroblast growth factor (FGF) family of proteins, but little is known about the specific mol. mechanisms of this endocrine pathway.  FGF19 regulates bile acid homeostasis and gall bladder filling; FGF19 binds only to FGF receptor 4 (FGFR4), but its liver-specific activity cannot be explained solely by the distribution of this receptor.  Although it has been suggested that Klotho beta (KLB) may have a role in mediating FGF19 activity, we have provided for the first time definitive evidence that KLB is required for FGF19 binding to FGFR4, intracellular signaling, and downstream modulation of gene expression.  We have shown that FGFR4 is widely distributed in mouse, whereas KLB distribution is more restricted.  Liver was the only organ in which both genes were abundantly expressed.  We show that in mice, FGF19 injection triggers liver-specific induction of c-Fos and repression of CYP7A1.  The tissue-specific activity of FGF19 supports the unique intersection of KLB and FGFR4 distribution in liver.  These studies define KLB as a novel FGFR4 coreceptor required for FGF19 liver specific functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPLlo-ZnNt-LVg90H21EOLACvtfcHk0lgBEzcENb-qIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVWjsLfE&md5=432206e9ae0424491964bb99d1db41f0</span></div><a href="/servlet/linkout?suffix=cit15a&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M704244200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M704244200%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DB.%2BC.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DBlackmore%26aufirst%3DC.%26aulast%3DDesnoyers%26aufirst%3DL.%2BR.%26atitle%3DLiver-specific%2520activities%2520of%2520FGF19%2520require%2520Klotho%2520beta%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D27277%26epage%3D27284%26doi%3D10.1074%2Fjbc.M704244200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit15b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Inagaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moschetta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummins, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodwin, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerard, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repa, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangelsdorf, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kliewer, S. A.</span></span> <span> </span><span class="NLM_article-title">Fibroblast growth factor 15 functions as an enterohepatic signal to regulate bile acid homeostasis</span>. <i>Cell Metab.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">217</span>– <span class="NLM_lpage">225</span>, <span class="refDoi"> DOI: 10.1016/j.cmet.2005.09.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1016%2Fj.cmet.2005.09.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=16213224" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFGktrjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2005&pages=217-225&author=T.+Inagakiauthor=M.+Choiauthor=A.+Moschettaauthor=L.+Pengauthor=C.+L.+Cumminsauthor=J.+G.+McDonaldauthor=G.+Luoauthor=S.+A.+Jonesauthor=B.+Goodwinauthor=J.+A.+Richardsonauthor=R.+D.+Gerardauthor=J.+J.+Repaauthor=D.+J.+Mangelsdorfauthor=S.+A.+Kliewer&title=Fibroblast+growth+factor+15+functions+as+an+enterohepatic+signal+to+regulate+bile+acid+homeostasis&doi=10.1016%2Fj.cmet.2005.09.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15bR"><div class="casContent"><span class="casTitleNuber">15b</span><div class="casTitle"><span class="NLM_cas:atitle">Fibroblast growth factor 15 functions as an enterohepatic signal to regulate bile acid homeostasis</span></div><div class="casAuthors">Inagaki, Takeshi; Choi, Mihwa; Moschetta, Antonio; Peng, Li; Cummins, Carolyn L.; McDonald, Jeffrey G.; Luo, Guizhen; Jones, Stacey A.; Goodwin, Bryan; Richardson, James A.; Gerard, Robert D.; Repa, Joyce J.; Mangelsdorf, David J.; Kliewer, Steven A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Metabolism</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">217-225</span>CODEN:
                <span class="NLM_cas:coden">CMEEB5</span>;
        ISSN:<span class="NLM_cas:issn">1550-4131</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The liver and intestine play crucial roles in maintaining bile acid homeostasis.  Fibroblast growth factor 15 (FGF15) signals from intestine to liver to repress the gene encoding cholesterol 7α-hydroxylase (CYP7A1), which catalyzes the first and rate-limiting step in the classical bile acid synthetic pathway.  FGF15 expression is stimulated in the small intestine by the nuclear bile acid receptor FXR and represses Cyp7a1 in liver through a mechanism that involves FGF receptor 4 (FGFR4) and the orphan nuclear receptor SHP.  Mice lacking FGF15 have increased hepatic CYP7A1 mRNA and protein levels and corresponding increases in CYP7A1 enzyme activity and fecal bile acid excretion.  These studies define FGF15 and FGFR4 as components of a gut-liver signaling pathway that synergizes with SHP to regulate bile acid synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMd0UxjYyXL7Vg90H21EOLACvtfcHk0lgtNVAemM5P6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFGktrjM&md5=b206ac5a248d774e26787a4a4f98ffeb</span></div><a href="/servlet/linkout?suffix=cit15b&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2005.09.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2005.09.001%26sid%3Dliteratum%253Aachs%26aulast%3DInagaki%26aufirst%3DT.%26aulast%3DChoi%26aufirst%3DM.%26aulast%3DMoschetta%26aufirst%3DA.%26aulast%3DPeng%26aufirst%3DL.%26aulast%3DCummins%26aufirst%3DC.%2BL.%26aulast%3DMcDonald%26aufirst%3DJ.%2BG.%26aulast%3DLuo%26aufirst%3DG.%26aulast%3DJones%26aufirst%3DS.%2BA.%26aulast%3DGoodwin%26aufirst%3DB.%26aulast%3DRichardson%26aufirst%3DJ.%2BA.%26aulast%3DGerard%26aufirst%3DR.%2BD.%26aulast%3DRepa%26aufirst%3DJ.%2BJ.%26aulast%3DMangelsdorf%26aufirst%3DD.%2BJ.%26aulast%3DKliewer%26aufirst%3DS.%2BA.%26atitle%3DFibroblast%2520growth%2520factor%252015%2520functions%2520as%2520an%2520enterohepatic%2520signal%2520to%2520regulate%2520bile%2520acid%2520homeostasis%26jtitle%3DCell%2520Metab.%26date%3D2005%26volume%3D2%26spage%3D217%26epage%3D225%26doi%3D10.1016%2Fj.cmet.2005.09.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pai, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hotzel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plise, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salphati, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Setchell, K. D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ware, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lauriault, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schutt, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dambach, D.</span></span> <span> </span><span class="NLM_article-title">Antibody-mediated inhibition of fibroblast growth factor 19 results in increased bile acid synthesis and ileal malabsorption of bile acids in cynomolgus monkeys</span>. <i>Toxicol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">446</span>– <span class="NLM_lpage">456</span>, <span class="refDoi"> DOI: 10.1093/toxsci/kfs011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1093%2Ftoxsci%2Fkfs011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=22268002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC38Xks1SitbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2012&pages=446-456&author=R.+Paiauthor=D.+Frenchauthor=N.+Maauthor=K.+Hotzelauthor=E.+Pliseauthor=L.+Salphatiauthor=K.+D.+R.+Setchellauthor=J.+Wareauthor=V.+Lauriaultauthor=L.+Schuttauthor=D.+Hartleyauthor=D.+Dambach&title=Antibody-mediated+inhibition+of+fibroblast+growth+factor+19+results+in+increased+bile+acid+synthesis+and+ileal+malabsorption+of+bile+acids+in+cynomolgus+monkeys&doi=10.1093%2Ftoxsci%2Fkfs011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Antibody-Mediated Inhibition of Fibroblast Growth Factor 19 Results in Increased Bile Acids Synthesis and Ileal Malabsorption of Bile Acids in Cynomolgus Monkeys</span></div><div class="casAuthors">Pai, Rama; French, Dorothy; Ma, Ning; Hotzel, Kathy; Plise, Emile; Salphati, Laurent; Setchell, Kenneth D. R.; Ware, Joseph; Lauriault, Veronique; Schutt, Leah; Hartley, Dylan; Dambach, Donna</div><div class="citationInfo"><span class="NLM_cas:title">Toxicological Sciences</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">446-456</span>CODEN:
                <span class="NLM_cas:coden">TOSCF2</span>;
        ISSN:<span class="NLM_cas:issn">1096-0929</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Fibroblast growth factor 19 (FGF19) represses cholesterol 7α-hydroxylase (Cyp7α1) and inhibits bile acid synthesis in vitro and in vivo.  Previous studies have shown that anti-FGF19 antibody treatment reduces growth of colon tumor xenografts and prevents hepatocellular carcinomas in FGF19 transgenic mice and thus may be a useful cancer target.  In a repeat dose safety study in cynomolgus monkeys, anti-FGF19 treatment (3-100 mg/kg) demonstrated dose-related liver toxicity accompanied by severe diarrhea and low food consumption.  The mechanism of anti-FGF19 toxicity was investigated using in vitro and in vivo approaches.  The authors' results show that anti-FGF19 antibody had no direct cytotoxic effect on monkey hepatocytes.  Anti-FGF19 increased Cyp7α1, as expected, but also increased bile acid efflux transporter gene (bile salt export pump, multidrug resistant protein 2 [MRP2], and MRP3) expression and reduced sodium taurocholate cotransporting polypeptide and org. anion transporter 2 expression in liver tissues from treated monkeys and in primary hepatocytes.  In addn., anti-FGF19 treatment increased solute transporter gene (ileal bile acid-binding protein, org. solute transporter α [OST-α], and OST-β) expression in ileal tissues from treated monkeys but not in Caco-2 cells.  However, deoxycholic acid (a secondary bile acid) increased expression of FGF19 and these solute transporter genes in Caco-2 cells.  Gas chromatog.-mass spectrometry anal. of monkey feces showed an increase in total bile acids and cholic acid derivs.  These findings suggest that high doses of anti-FGF19 increase Cyp7α1 expression and bile acid synthesis and alter the expression of bile transporters in the liver resulting in enhanced bile acid efflux and reduced uptake.  Increased bile acids alter expression of solute transporters in the ileum causing diarrhea and the enhanced enterohepatic recirculation of bile acids leading to liver toxicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqORkYjNAK2bLVg90H21EOLACvtfcHk0lgtNVAemM5P6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xks1SitbY%253D&md5=b70cb6104a023ef79adda0cea34fc8a4</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1093%2Ftoxsci%2Fkfs011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Ftoxsci%252Fkfs011%26sid%3Dliteratum%253Aachs%26aulast%3DPai%26aufirst%3DR.%26aulast%3DFrench%26aufirst%3DD.%26aulast%3DMa%26aufirst%3DN.%26aulast%3DHotzel%26aufirst%3DK.%26aulast%3DPlise%26aufirst%3DE.%26aulast%3DSalphati%26aufirst%3DL.%26aulast%3DSetchell%26aufirst%3DK.%2BD.%2BR.%26aulast%3DWare%26aufirst%3DJ.%26aulast%3DLauriault%26aufirst%3DV.%26aulast%3DSchutt%26aufirst%3DL.%26aulast%3DHartley%26aufirst%3DD.%26aulast%3DDambach%26aufirst%3DD.%26atitle%3DAntibody-mediated%2520inhibition%2520of%2520fibroblast%2520growth%2520factor%252019%2520results%2520in%2520increased%2520bile%2520acid%2520synthesis%2520and%2520ileal%2520malabsorption%2520of%2520bile%2520acids%2520in%2520cynomolgus%2520monkeys%26jtitle%3DToxicol.%2520Sci.%26date%3D2012%26volume%3D126%26spage%3D446%26epage%3D456%26doi%3D10.1093%2Ftoxsci%2Fkfs011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schadt, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahl, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wuersch, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couttet, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwald, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lienard, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emotte, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skuba, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manenti, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graus Porta, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairhurst, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kullak-Ublick, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chibout, S.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pognan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kluwe, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kinyamu-Akunda, J.</span></span> <span> </span><span class="NLM_article-title">Bile acid sequestration by cholestyramine mitigates FGFR4 inhibition-induced ALT elevation</span>. <i>Toxicol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>163</i></span>,  <span class="NLM_fpage">265</span>– <span class="NLM_lpage">278</span>, <span class="refDoi"> DOI: 10.1093/toxsci/kfy031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1093%2Ftoxsci%2Fkfy031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=29432567" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlGksr%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=163&publication_year=2018&pages=265-278&author=H.+S.+Schadtauthor=A.+Wolfauthor=J.+A.+Mahlauthor=K.+Wuerschauthor=P.+Couttetauthor=M.+Schwaldauthor=A.+Fischerauthor=M.+Lienardauthor=C.+Emotteauthor=C.-H.+Tengauthor=E.+Skubaauthor=T.+A.+Richardsonauthor=L.+Manentiauthor=A.+Weissauthor=D.+Graus+Portaauthor=R.+A.+Fairhurstauthor=G.+A.+Kullak-Ublickauthor=S.-D.+Chiboutauthor=F.+Pognanauthor=W.+Kluweauthor=J.+Kinyamu-Akunda&title=Bile+acid+sequestration+by+cholestyramine+mitigates+FGFR4+inhibition-induced+ALT+elevation&doi=10.1093%2Ftoxsci%2Fkfy031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Bile acid sequestration by cholestyramine mitigates FGFR4 inhibition-induced ALT elevation</span></div><div class="casAuthors">Schadt, Heiko S.; Wolf, Armin; Mahl, Joerg Andreas; Wuersch, Kuno; Couttet, Philippe; Schwald, Marianne; Fischer, Audrey; Lienard, Mathilde; Emotte, Corinne; Teng, Chi-Hse; Skuba, Elizabeth; Richardson, Terrilyn A.; Manenti, Luigi; Weiss, Andreas; Porta, Diana Graus; Fairhurst, Robin A.; Kullak-Ublick, Gerd A.; Chibout, Salah-Dine; Pognan, Francois; Kluwe, William; Kinyamu-Akunda, Jacqueline</div><div class="citationInfo"><span class="NLM_cas:title">Toxicological Sciences</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">163</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">265-278</span>CODEN:
                <span class="NLM_cas:coden">TOSCF2</span>;
        ISSN:<span class="NLM_cas:issn">1096-0929</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The FGF19-fibroblast growth factor receptor (FGFR4)-βKlotho (KLB) pathway plays an important role in the regulation of bile acid (BA) homeostasis.  Aberrant activation of this pathway has been described in the development and progression of a subset of liver cancers including hepatocellular carcinoma, establishing FGFR4 as an attractive therapeutic target for such solid tumors.  FGF401 is a highly selective FGFR4 kinase inhibitor being developed for hepatocellular carcinoma, currently in phase I/II clin. studies.  In preclin. studies in mice and dogs, oral administration of FGF401 led to induction of Cyp7a1, elevation of its peripheral marker 7alpha-hydroxy-4-cholesten-3-one, increased BA pool size, decreased serum cholesterol and diarrhea in dogs.  FGF401 was also assocd. with increases of serum aminotransferases, primarily alanine aminotransferase (ALT), in the absence of any observable adverse histopathol. in the liver, or in any other organs.  The authors hypothesized that the increase in ALT could be secondary to increased BAs and conducted an investigative study in dogs with FGF401 and coadministration of the BA sequestrant cholestyramine (CHO).  CHO prevented and reversed FGF401-related increases in ALT in dogs in parallel to its ability to reduce BAs in the circulation.  Correlation anal. showed that FGF401-mediated increases in ALT strongly correlated with increases in taurolithocholic acid and taurodeoxycholic acid, the major secondary BAs in dog plasma, indicating a mechanistic link between ALT elevation and changes in BA pool hydrophobicity.  Thus, CHO may offer the potential to mitigate elevations in serum amtransferases in human subjects that are caused by targeted FGFR4 inhibition and elevated intracellular BA levels.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4tzKAwiqUR7Vg90H21EOLACvtfcHk0lgtNVAemM5P6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlGksr%252FF&md5=e11d415d68d5b1b326a9547d35ef727a</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1093%2Ftoxsci%2Fkfy031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Ftoxsci%252Fkfy031%26sid%3Dliteratum%253Aachs%26aulast%3DSchadt%26aufirst%3DH.%2BS.%26aulast%3DWolf%26aufirst%3DA.%26aulast%3DMahl%26aufirst%3DJ.%2BA.%26aulast%3DWuersch%26aufirst%3DK.%26aulast%3DCouttet%26aufirst%3DP.%26aulast%3DSchwald%26aufirst%3DM.%26aulast%3DFischer%26aufirst%3DA.%26aulast%3DLienard%26aufirst%3DM.%26aulast%3DEmotte%26aufirst%3DC.%26aulast%3DTeng%26aufirst%3DC.-H.%26aulast%3DSkuba%26aufirst%3DE.%26aulast%3DRichardson%26aufirst%3DT.%2BA.%26aulast%3DManenti%26aufirst%3DL.%26aulast%3DWeiss%26aufirst%3DA.%26aulast%3DGraus%2BPorta%26aufirst%3DD.%26aulast%3DFairhurst%26aufirst%3DR.%2BA.%26aulast%3DKullak-Ublick%26aufirst%3DG.%2BA.%26aulast%3DChibout%26aufirst%3DS.-D.%26aulast%3DPognan%26aufirst%3DF.%26aulast%3DKluwe%26aufirst%3DW.%26aulast%3DKinyamu-Akunda%26aufirst%3DJ.%26atitle%3DBile%2520acid%2520sequestration%2520by%2520cholestyramine%2520mitigates%2520FGFR4%2520inhibition-induced%2520ALT%2520elevation%26jtitle%3DToxicol.%2520Sci.%26date%3D2018%26volume%3D163%26spage%3D265%26epage%3D278%26doi%3D10.1093%2Ftoxsci%2Fkfy031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, J.</span></span> <span> </span><span class="NLM_article-title">Systematic review: the management of chronic diarrhoea due to bile acid malabsorption</span>. <i>Aliment. Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">923</span>– <span class="NLM_lpage">939</span>, <span class="refDoi"> DOI: 10.1111/apt.12684</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1111%2Fapt.12684" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=24602022" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC2cXlvFWis7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2014&pages=923-939&author=C.+Wilcoxauthor=J.+Turnerauthor=J.+Green&title=Systematic+review%3A+the+management+of+chronic+diarrhoea+due+to+bile+acid+malabsorption&doi=10.1111%2Fapt.12684"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Systematic review: the management of chronic diarrhoea due to bile acid malabsorption</span></div><div class="casAuthors">Wilcox, C.; Turner, J.; Green, J.</div><div class="citationInfo"><span class="NLM_cas:title">Alimentary Pharmacology and Therapeutics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">923-939</span>CODEN:
                <span class="NLM_cas:coden">APTHEN</span>;
        ISSN:<span class="NLM_cas:issn">0269-2813</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Summary : Background : Bile acid malabsorption (BAM) is a common, yet under-recognized, cause of chronic diarrhea, with limited guidance available on the appropriate management of patients with BAM.  Aim : To summarize the evidence supporting different treatments available for patients with bile acid malabsorption, noting their impact on clin. outcomes, tolerability and assocd. side effects.  Methods : A literature search was conducted through PubMed, the Cochrane Database of Systematic Reviews and Scopus.  Relevant articles studied patients who had been diagnosed with BAM and were clin. assessed before and after therapy.  Results : A total of 30 relevant publications (1241 adult patients) were identified, which investigated the clin. response to drugs, including colestyramine, colestipol, colesevelam, aluminum hydroxide and obeticholic acid.  The most commonly used diagnostic test of bile acid malabsorption was the SeHCAT test (24 studies).  Colestyramine treatment was by far the most studied of these agents, and was successful in 70% of 801 patients (range: 63-100%).  Conclusions : Colestyramine and colestipol are generally effective treatments of gastrointestinal symptoms from BAM, but may be poorly tolerated and reduce the bioavailability of co-administered agents.  Alternative therapies (including colesevelam and aluminum hydroxide) as well as dietary intervention may also have a role, and the promising results of the first proof-of-concept study of obeticholic acid suggest that its novel approach may have an exciting future in the treatment of this condition.  Future trials should employ accurate diagnostic testing and be conducted over longer periods so that the long-term benefits and tolerability of these different approaches can be evaluated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq17n0aYFw-kLVg90H21EOLACvtfcHk0lisdI13SiGmaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXlvFWis7w%253D&md5=b4d6ad463b6544420028dd22300ad292</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1111%2Fapt.12684&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fapt.12684%26sid%3Dliteratum%253Aachs%26aulast%3DWilcox%26aufirst%3DC.%26aulast%3DTurner%26aufirst%3DJ.%26aulast%3DGreen%26aufirst%3DJ.%26atitle%3DSystematic%2520review%253A%2520the%2520management%2520of%2520chronic%2520diarrhoea%2520due%2520to%2520bile%2520acid%2520malabsorption%26jtitle%3DAliment.%2520Pharmacol.%2520Ther.%26date%3D2014%26volume%3D39%26spage%3D923%26epage%3D939%26doi%3D10.1111%2Fapt.12684" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fairhurst, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knoepfel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leblanc, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buschmann, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaul, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blank, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galuba, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trappe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voshol, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Genick, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunet-Lefeuvre, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bitsch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graus-Porta, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furet, P.</span></span> <span> </span><span class="NLM_article-title">Approaches to selective fibroblast growth factor receptor 4 inhibition through targeting the ATP-pocket middle-hinge region</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1604</span>– <span class="NLM_lpage">1613</span>, <span class="refDoi"> DOI: 10.1039/C7MD00213K</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1039%2FC7MD00213K" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=30108871" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC2sXps1GlsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1604-1613&author=R.+A.+Fairhurstauthor=T.+Knoepfelauthor=C.+Leblancauthor=N.+Buschmannauthor=C.+Gaulauthor=J.+Blankauthor=I.+Galubaauthor=J.+Trappeauthor=C.+Zouauthor=J.+Vosholauthor=C.+Genickauthor=P.+Brunet-Lefeuvreauthor=F.+Bitschauthor=D.+Graus-Portaauthor=P.+Furet&title=Approaches+to+selective+fibroblast+growth+factor+receptor+4+inhibition+through+targeting+the+ATP-pocket+middle-hinge+region&doi=10.1039%2FC7MD00213K"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Approaches to selective fibroblast growth factor receptor 4 inhibition through targeting the ATP-pocket middle-hinge region</span></div><div class="casAuthors">Fairhurst, Robin A.; Knoepfel, Thomas; Leblanc, Catherine; Buschmann, Nicole; Gaul, Christoph; Blank, Jutta; Galuba, Inga; Trappe, Jorg; Zou, Chao; Voshol, Johannes; Genick, Christine; Brunet-Lefeuvre, Peggy; Bitsch, Francis; Graus-Porta, Diana; Furet, Pascal</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1604-1613</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A diverse range of selective FGFR4 inhibitor hit series were identified using unbiased screening approaches and by the modification of known kinase inhibitor scaffolds.  In each case the origin of the selectivity was consistent with an interaction with a poorly conserved cysteine residue within the middle-hinge region of the kinase domain of FGFR4, at position 552.  Targeting this region identified a non-covalent diaminopyrimidine series differentiating by size, an irreversible-covalent inhibitor in which Cys552 undergoes an SNAr reaction with a 2-chloropyridine, and a reversible-covalent inhibitor series in which Cys552 forms a hemithioacetal adduct with a 2-formyl naphthalene.  In addn., the introduction of an acrylamide into a known FGFR scaffold identified a pan-FGFR inhibitor which reacted with both Cys552 and a second poorly conserved cysteine on the P-loop of FGFR4 at position 477 which is present in all four FGFR family members.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwzyJE17BWsLVg90H21EOLACvtfcHk0lisdI13SiGmaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXps1GlsL8%253D&md5=04f1aff88385b915e2672509dd3f6954</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1039%2FC7MD00213K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7MD00213K%26sid%3Dliteratum%253Aachs%26aulast%3DFairhurst%26aufirst%3DR.%2BA.%26aulast%3DKnoepfel%26aufirst%3DT.%26aulast%3DLeblanc%26aufirst%3DC.%26aulast%3DBuschmann%26aufirst%3DN.%26aulast%3DGaul%26aufirst%3DC.%26aulast%3DBlank%26aufirst%3DJ.%26aulast%3DGaluba%26aufirst%3DI.%26aulast%3DTrappe%26aufirst%3DJ.%26aulast%3DZou%26aufirst%3DC.%26aulast%3DVoshol%26aufirst%3DJ.%26aulast%3DGenick%26aufirst%3DC.%26aulast%3DBrunet-Lefeuvre%26aufirst%3DP.%26aulast%3DBitsch%26aufirst%3DF.%26aulast%3DGraus-Porta%26aufirst%3DD.%26aulast%3DFuret%26aufirst%3DP.%26atitle%3DApproaches%2520to%2520selective%2520fibroblast%2520growth%2520factor%2520receptor%25204%2520inhibition%2520through%2520targeting%2520the%2520ATP-pocket%2520middle-hinge%2520region%26jtitle%3DMedChemComm%26date%3D2017%26volume%3D8%26spage%3D1604%26epage%3D1613%26doi%3D10.1039%2FC7MD00213K" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leproult, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barluenga, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moras, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wurtz, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winssinger, N.</span></span> <span> </span><span class="NLM_article-title">Cysteine mapping in conformationally distinct kinase nucleotide binding sites: application to the design of selective covalent inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">1347</span>– <span class="NLM_lpage">1355</span>, <span class="refDoi"> DOI: 10.1021/jm101396q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm101396q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhvFeis70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=1347-1355&author=E.+Leproultauthor=S.+Barluengaauthor=D.+Morasauthor=J.-M.+Wurtzauthor=N.+Winssinger&title=Cysteine+mapping+in+conformationally+distinct+kinase+nucleotide+binding+sites%3A+application+to+the+design+of+selective+covalent+inhibitors&doi=10.1021%2Fjm101396q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Cysteine mapping in conformationally distinct kinase nucleotide binding sites: application to the design of selective covalent inhibitors</span></div><div class="casAuthors">Leproult, Emeline; Barluenga, Sofia; Moras, Dino; Wurtz, Jean-Marie; Winssinger, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1347-1355</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Kinases have emerged as one of the most prolific therapeutic targets.  An important criterion in the therapeutic success of inhibitors targeting the nucleotide binding pocket of kinases is the inhibitor residence time.  Recently, covalent kinase inhibitors have attracted attention since they confer terminal inhibition and should thus be more effective than reversible inhibitors with transient inhibition.  The most robust approach to design irreversible inhibitors is to capitalize on the nucleophilicity of a cysteine thiol group present in the target protein.  Herein, the authors report a systematic anal. of cysteine residues present in the nucleotide binding site of kinases, which could be harnessed for irreversible inhibition, taking into consideration the different kinase conformations.  The authors demonstrate the predictive power of this anal. with the design and validation of an irreversible inhibitor of KIT/PDGFR kinases.  This is the first example of a covalent kinase inhibitor that combines a pharmacophore addressing the DFG-out conformation with a covalent trap.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkzqt0gMrz3rVg90H21EOLACvtfcHk0lisdI13SiGmaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhvFeis70%253D&md5=329ae5eaa6102b6dd34a0d1274631247</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm101396q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm101396q%26sid%3Dliteratum%253Aachs%26aulast%3DLeproult%26aufirst%3DE.%26aulast%3DBarluenga%26aufirst%3DS.%26aulast%3DMoras%26aufirst%3DD.%26aulast%3DWurtz%26aufirst%3DJ.-M.%26aulast%3DWinssinger%26aufirst%3DN.%26atitle%3DCysteine%2520mapping%2520in%2520conformationally%2520distinct%2520kinase%2520nucleotide%2520binding%2520sites%253A%2520application%2520to%2520the%2520design%2520of%2520selective%2520covalent%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D1347%26epage%3D1355%26doi%3D10.1021%2Fjm101396q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yau, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macarulla, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, J.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choo, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollebecque, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sung, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, H.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazzaferro, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trojan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, A. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoon, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Z.-Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faivre, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feun, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yen, C.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dufour, J.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Llovet, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manoogian, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tugnait, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stransky, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohl, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lengauer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherwin, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt-Kittler, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoeflich, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, B. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, Y.-K.</span></span> <span> </span><span class="NLM_article-title">First-in-human phase I study of fisogatinib (BLU-554) validates aberrant FGF19 signaling as a driver event in hepatocellular carcinoma</span>. <i>Cancer Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1696</span>– <span class="NLM_lpage">1707</span>, <span class="refDoi"> DOI: 10.1158/2159-8290.CD-19-0555</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1158%2F2159-8290.CD-19-0555" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=31575541" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1Ght77L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=1696-1707&author=R.+D.+Kimauthor=D.+Sarkerauthor=T.+Meyerauthor=T.+Yauauthor=T.+Macarullaauthor=J.-W.+Parkauthor=S.+P.+Chooauthor=A.+Hollebecqueauthor=M.+W.+Sungauthor=H.-Y.+Limauthor=V.+Mazzaferroauthor=J.+Trojanauthor=A.+X.+Zhuauthor=J.-H.+Yoonauthor=S.+Sharmaauthor=Z.-Z.+Linauthor=S.+L.+Chanauthor=S.+Faivreauthor=L.+G.+Feunauthor=C.-J.+Yenauthor=J.-F.+Dufourauthor=D.+H.+Palmerauthor=J.+M.+Llovetauthor=M.+Manoogianauthor=M.+Tugnaitauthor=N.+Stranskyauthor=M.+Hagelauthor=N.+E.+Kohlauthor=C.+Lengauerauthor=C.+A.+Sherwinauthor=O.+Schmidt-Kittlerauthor=K.+P.+Hoeflichauthor=H.+Shiauthor=B.+B.+Wolfauthor=Y.-K.+Kang&title=First-in-human+phase+I+study+of+fisogatinib+%28BLU-554%29+validates+aberrant+FGF19+signaling+as+a+driver+event+in+hepatocellular+carcinoma&doi=10.1158%2F2159-8290.CD-19-0555"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">First-in-human phase I study of fisogatinib (BLU-554) validates aberrant FGF19 signaling as a driver event in hepatocellular carcinoma</span></div><div class="casAuthors">Kim, Richard D.; Sarker, Debashis; Meyer, Tim; Yau, Thomas; Macarulla, Teresa; Park, Joong-Won; Choo, Su Pin; Hollebecque, Antoine; Sung, Max W.; Lim, Ho-Yeong; Mazzaferro, Vincenzo; Trojan, Joerg; Zhu, Andrew X.; Yoon, Jung-Hwan; Sharma, Sunil; Lin, Zhong-Zhe; Chan, Stephen L.; Faivre, Sandrine; Feun, Lynn G.; Yen, Chia-Jui; Dufour, Jean-Francois; Palmer, Daniel H.; Llovet, Josep M.; Manoogian, Melissa; Tugnait, Meera; Stransky, Nicolas; Hagel, Margit; Kohl, Nancy E.; Lengauer, Christoph; Sherwin, Cori Ann; Schmidt-Kittler, Oleg; Hoeflich, Klaus P.; Shi, Hongliang; Wolf, Beni B.; Kang, Yoon-Koo</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1696-1707</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Outcomes for patients with advanced hepatocellular carcinoma (HCC) remain poor despite recent progress in drug development.  Emerging data implicate FGF19 as a potential HCC driver, suggesting its receptor, FGFR4, as a novel therapeutic target.  We evaluated fisogatinib (BLU-554), a highly potent and selective oral FGFR4 inhibitor, in a phase I dose-escalation/dose-expansion study in advanced HCC using FGF19 expression measured by IHC as a biomarker for pathway activation.  For dose escalation, 25 patients received 140 to 900 mg fisogatinib once daily; the max. tolerated dose (600 mg once daily) was expanded in 81 patients.  Fisogatinib was well tolerated; most adverse events were manageable, grade 1/2 gastrointestinal events, primarily diarrhea, nausea, and vomiting.  Across doses, the overall response rate was 17% in FGF19-pos. patients [median duration of response: 5.3 mo (95% CI, 3.7-not reached)] and 0% in FGF19-neg. patients.  These results validate FGFR4 as a targetable driver in FGF19-pos. advanced HCC.  SIGNIFICANCE: Fisogatinib elicited clin. responses in patients with tumor FGF19 overexpression in advanced HCC.  These results validate the oncogenic driver role of the FGFR4 pathway in HCC and the use of FGF19 as a biomarker for patient selection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpthlOsQ713p7Vg90H21EOLACvtfcHk0lhBxgK8B3eclQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1Ght77L&md5=84a5d6ca43e37a9caad36f513eb79438</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-19-0555&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-19-0555%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DR.%2BD.%26aulast%3DSarker%26aufirst%3DD.%26aulast%3DMeyer%26aufirst%3DT.%26aulast%3DYau%26aufirst%3DT.%26aulast%3DMacarulla%26aufirst%3DT.%26aulast%3DPark%26aufirst%3DJ.-W.%26aulast%3DChoo%26aufirst%3DS.%2BP.%26aulast%3DHollebecque%26aufirst%3DA.%26aulast%3DSung%26aufirst%3DM.%2BW.%26aulast%3DLim%26aufirst%3DH.-Y.%26aulast%3DMazzaferro%26aufirst%3DV.%26aulast%3DTrojan%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DA.%2BX.%26aulast%3DYoon%26aufirst%3DJ.-H.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DLin%26aufirst%3DZ.-Z.%26aulast%3DChan%26aufirst%3DS.%2BL.%26aulast%3DFaivre%26aufirst%3DS.%26aulast%3DFeun%26aufirst%3DL.%2BG.%26aulast%3DYen%26aufirst%3DC.-J.%26aulast%3DDufour%26aufirst%3DJ.-F.%26aulast%3DPalmer%26aufirst%3DD.%2BH.%26aulast%3DLlovet%26aufirst%3DJ.%2BM.%26aulast%3DManoogian%26aufirst%3DM.%26aulast%3DTugnait%26aufirst%3DM.%26aulast%3DStransky%26aufirst%3DN.%26aulast%3DHagel%26aufirst%3DM.%26aulast%3DKohl%26aufirst%3DN.%2BE.%26aulast%3DLengauer%26aufirst%3DC.%26aulast%3DSherwin%26aufirst%3DC.%2BA.%26aulast%3DSchmidt-Kittler%26aufirst%3DO.%26aulast%3DHoeflich%26aufirst%3DK.%2BP.%26aulast%3DShi%26aufirst%3DH.%26aulast%3DWolf%26aufirst%3DB.%2BB.%26aulast%3DKang%26aufirst%3DY.-K.%26atitle%3DFirst-in-human%2520phase%2520I%2520study%2520of%2520fisogatinib%2520%2528BLU-554%2529%2520validates%2520aberrant%2520FGF19%2520signaling%2520as%2520a%2520driver%2520event%2520in%2520hepatocellular%2520carcinoma%26jtitle%3DCancer%2520Discovery%26date%3D2019%26volume%3D9%26spage%3D1696%26epage%3D1707%26doi%3D10.1158%2F2159-8290.CD-19-0555" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coffey, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corcoran, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichikawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prajapati, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rimkunas, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karr, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramanian, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKenzie, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Satoh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioux, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buonamici, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fekkes, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warmuth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reynolds, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Selvaraj, A.</span></span> <span> </span><span class="NLM_article-title">H3B-6527 Is a potent and selective inhibitor of FGFR4 in FGF19-Driven hepatocellular carcinoma</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">6999</span>– <span class="NLM_lpage">7013</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-17-1865</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1158%2F0008-5472.CAN-17-1865" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=29247039" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFOgur%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2017&pages=6999-7013&author=J.+J.+Joshiauthor=H.+Coffeyauthor=E.+Corcoranauthor=J.+Tsaiauthor=C.+L.+Huangauthor=K.+Ichikawaauthor=S.+Prajapatiauthor=M.+H.+Haoauthor=S.+Baileyauthor=J.+Wuauthor=V.+Rimkunasauthor=C.+Karrauthor=V.+Subramanianauthor=P.+Kumarauthor=C.+MacKenzieauthor=R.+Hurleyauthor=T.+Satohauthor=K.+Yuauthor=E.+Parkauthor=N.+Riouxauthor=A.+Kimauthor=W.+G.+Laiauthor=L.+Yuauthor=P.+Zhuauthor=S.+Buonamiciauthor=N.+Larsenauthor=P.+Fekkesauthor=J.+Wangauthor=M.+Warmuthauthor=D.+J.+Reynoldsauthor=P.+G.+Smithauthor=A.+Selvaraj&title=H3B-6527+Is+a+potent+and+selective+inhibitor+of+FGFR4+in+FGF19-Driven+hepatocellular+carcinoma&doi=10.1158%2F0008-5472.CAN-17-1865"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">H3B-6527 Is a Potent and Selective Inhibitor of FGFR4 in FGF19-Driven Hepatocellular Carcinoma</span></div><div class="casAuthors">Joshi, Jaya Julie; Coffey, Heather; Corcoran, Erik; Tsai, Jennifer; Huang, Chia-Ling; Ichikawa, Kana; Prajapati, Sudeep; Hao, Ming-Hong; Bailey, Suzanna; Wu, Jeremy; Rimkunas, Victoria; Karr, Craig; Subramanian, Vanitha; Kumar, Pavan; MacKenzie, Crystal; Hurley, Raelene; Satoh, Takashi; Yu, Kun; Park, Eunice; Rioux, Nathalie; Kim, Amy; Lai, Weidong G.; Yu, Lihua; Zhu, Ping; Buonamici, Silvia; Larsen, Nicholas; Fekkes, Peter; Wang, John; Warmuth, Markus; Reynolds, Dominic J.; Smith, Peter G.; Selvaraj, Anand</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">6999-7013</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Activation of the fibroblast growth factor receptor FGFR4 by FGF19 drives hepatocellular carcinoma (HCC), a disease with few, if any, effective treatment options.  While a no. of pan-FGFR inhibitors are being clin. evaluated, their application to FGF19-driven HCC may be limited by dose-limiting toxicities mediated by FGFR1-3 receptors.  To evade the potential limitations of pan-FGFR inhibitors, we generated H3B-6527, a highly selective covalent FGFR4 inhibitor, through structure-guided drug design.  Studies in a panel of 40 HCC cell lines and 30 HCC PDX models showed that FGF19 expression is a predictive biomarker for H3B-6527 response.  Moreover, coadministration of the CDK4/6 inhibitor palbociclib in combination with H3B-6527 could effectively trigger tumor regression in a xenograft model of HCC.  Overall, our results offer preclin. proof of concept for H3B-6527 as a candidate therapeutic agent for HCC cases that exhibit increased expression of FGF19.  Cancer Res; 77(24); 6999-7013. ©2017 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohONN1gEpft7Vg90H21EOLACvtfcHk0lhBxgK8B3eclQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFOgur%252FI&md5=9d7ea6245e3b047aa85fc6961a88e5eb</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-17-1865&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-17-1865%26sid%3Dliteratum%253Aachs%26aulast%3DJoshi%26aufirst%3DJ.%2BJ.%26aulast%3DCoffey%26aufirst%3DH.%26aulast%3DCorcoran%26aufirst%3DE.%26aulast%3DTsai%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DC.%2BL.%26aulast%3DIchikawa%26aufirst%3DK.%26aulast%3DPrajapati%26aufirst%3DS.%26aulast%3DHao%26aufirst%3DM.%2BH.%26aulast%3DBailey%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DRimkunas%26aufirst%3DV.%26aulast%3DKarr%26aufirst%3DC.%26aulast%3DSubramanian%26aufirst%3DV.%26aulast%3DKumar%26aufirst%3DP.%26aulast%3DMacKenzie%26aufirst%3DC.%26aulast%3DHurley%26aufirst%3DR.%26aulast%3DSatoh%26aufirst%3DT.%26aulast%3DYu%26aufirst%3DK.%26aulast%3DPark%26aufirst%3DE.%26aulast%3DRioux%26aufirst%3DN.%26aulast%3DKim%26aufirst%3DA.%26aulast%3DLai%26aufirst%3DW.%2BG.%26aulast%3DYu%26aufirst%3DL.%26aulast%3DZhu%26aufirst%3DP.%26aulast%3DBuonamici%26aufirst%3DS.%26aulast%3DLarsen%26aufirst%3DN.%26aulast%3DFekkes%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWarmuth%26aufirst%3DM.%26aulast%3DReynolds%26aufirst%3DD.%2BJ.%26aulast%3DSmith%26aufirst%3DP.%2BG.%26aulast%3DSelvaraj%26aufirst%3DA.%26atitle%3DH3B-6527%2520Is%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520FGFR4%2520in%2520FGF19-Driven%2520hepatocellular%2520carcinoma%26jtitle%3DCancer%2520Res.%26date%3D2017%26volume%3D77%26spage%3D6999%26epage%3D7013%26doi%3D10.1158%2F0008-5472.CAN-17-1865" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patterson, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smaill, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, K.</span></span> <span> </span><span class="NLM_article-title">Fibroblast growth factor receptor 4 (FGFR4) selective inhibitors as hepatocellular carcinoma therapy: advances and prospects</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">2905</span>– <span class="NLM_lpage">2915</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01531</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01531" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFagur3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=2905-2915&author=X.+Luauthor=H.+Chenauthor=A.+V.+Pattersonauthor=J.+B.+Smaillauthor=K.+Ding&title=Fibroblast+growth+factor+receptor+4+%28FGFR4%29+selective+inhibitors+as+hepatocellular+carcinoma+therapy%3A+advances+and+prospects&doi=10.1021%2Facs.jmedchem.8b01531"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Fibroblast Growth Factor Receptor 4 (FGFR4) Selective Inhibitors as Hepatocellular Carcinoma Therapy: Advances and Prospects</span></div><div class="casAuthors">Lu, Xiaoyun; Chen, Hao; Patterson, Adam V.; Smaill, Jeff B.; Ding, Ke</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2905-2915</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Hepatocellular carcinoma (HCC) is a lethal disease with limited therapeutic options and a particularly poor prognosis.  Aberrant fibroblast growth factor 19 (FGF19) signaling through fibroblast growth factor receptor 4 (FGFR4) has been identified as an oncogenic driver for a subset of patients with HCC.  FGFR4 is therefore a promising target for the treatment of HCC harboring aberrant FGF19-FGFR4 signaling, and several FGFR4 inhibitors have advanced to clin. trial.  In this review, the authors summarize the latest developments in FGFR4 inhibitors, including the known pharmacophores, their binding mode, selectivity, and clin. implications, as well as the optimization strategy of introducing an acrylamide into a known pan-FGFR inhibitor targeting Cys552 of FGFR4 to provide selective covalent FGFR4 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6Ev_KhmOEBbVg90H21EOLACvtfcHk0lhMXEmNh_yamw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFagur3M&md5=9c1c111108e0ec7f33bca7b9227c246a</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01531&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01531%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DPatterson%26aufirst%3DA.%2BV.%26aulast%3DSmaill%26aufirst%3DJ.%2BB.%26aulast%3DDing%26aufirst%3DK.%26atitle%3DFibroblast%2520growth%2520factor%2520receptor%25204%2520%2528FGFR4%2529%2520selective%2520inhibitors%2520as%2520hepatocellular%2520carcinoma%2520therapy%253A%2520advances%2520and%2520prospects%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D2905%26epage%3D2915%26doi%3D10.1021%2Facs.jmedchem.8b01531" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pike, K. G.</span>; <span class="NLM_string-name">Buttar, D.</span>; <span class="NLM_string-name">Tucker, J.</span>; <span class="NLM_string-name">Jones, C.</span></span> <span> </span><span class="NLM_article-title">Identifying Selective Inhibitors of FGFR4 Kinase</span>. Presented at the 23rd International Symposium on Medicinal Chemistry, Lisbon, Sep 7–11, <span class="NLM_year">2014</span>, European Federation for Medicinal Chemistry, LE098.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=K.+G.+Pike&author=D.+Buttar&author=J.+Tucker&author=C.+Jones&title=Identifying+Selective+Inhibitors+of+FGFR4+Kinase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DPike%26aufirst%3DK.%2BG.%26atitle%3DIdentifying%2520Selective%2520Inhibitors%2520of%2520FGFR4%2520Kinase%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><a href="https://clinicaltrials.gov/ct2/show/NCT02476019" class="extLink">https://clinicaltrials.gov/ct2/show/NCT02476019</a> (accessed Sep 14, <span class="NLM_year">2020</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=https%3A%2F%2Fclinicaltrials.gov%2Fct2%2Fshow%2FNCT02476019+%28accessed+Sep+14%2C+2020%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gavine, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mooney, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kilgour, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Kadhimi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rooney, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mellor, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinowska, T.</span></span> <span> </span><span class="NLM_article-title">AZD4547: An orally bioavailable, potent, and selective inhibitor of the fibroblast growth factor receptor tyrosine kinase family</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">2045</span>– <span class="NLM_lpage">2056</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-11-3034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1158%2F0008-5472.CAN-11-3034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=22369928" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC38Xls1Wgsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2012&pages=2045-2056&author=P.+R.+Gavineauthor=L.+Mooneyauthor=E.+Kilgourauthor=A.+P.+Thomasauthor=K.+Al-Kadhimiauthor=S.+Beckauthor=C.+Rooneyauthor=T.+Colemanauthor=D.+Bakerauthor=M.+J.+Mellorauthor=A.+N.+Brooksauthor=T.+Klinowska&title=AZD4547%3A+An+orally+bioavailable%2C+potent%2C+and+selective+inhibitor+of+the+fibroblast+growth+factor+receptor+tyrosine+kinase+family&doi=10.1158%2F0008-5472.CAN-11-3034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">AZD4547: An Orally Bioavailable, Potent, and Selective Inhibitor of the Fibroblast Growth Factor Receptor Tyrosine Kinase Family</span></div><div class="casAuthors">Gavine, Paul R.; Mooney, Lorraine; Kilgour, Elaine; Thomas, Andrew P.; Al-Kadhimi, Katherine; Beck, Sarah; Rooney, Claire; Coleman, Tanya; Baker, Dawn; Mellor, Martine J.; Brooks, A. Nigel; Klinowska, Teresa</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2045-2056</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The fibroblast growth factor (FGF) signaling axis is increasingly implicated in tumorigenesis and chemoresistance.  Several small-mol. FGF receptor (FGFR) kinase inhibitors are currently in clin. development; however, the predominant activity of the most advanced of these agents is against the kinase insert domain receptor (KDR), which compromises the FGFR selectivity.  Here, we report the pharmacol. profile of AZD4547, a novel and selective inhibitor of the FGFR1, 2, and 3 tyrosine kinases.  AZD4547 inhibited recombinant FGFR kinase activity in vitro and suppressed FGFR signaling and growth in tumor cell lines with deregulated FGFR expression.  In a representative FGFR-driven human tumor xenograft model, oral administration of AZD4547 was well tolerated and resulted in potent dose-dependent antitumor activity, consistent with plasma exposure and pharmacodynamic modulation of tumor FGFR.  Importantly, at efficacious doses, no evidence of anti-KDR-related effects were obsd., confirming the in vivo FGFR selectivity of AZD4547.  Taken together, our findings show that AZD4547 is a novel selective small-mol. inhibitor of FGFR with potent antitumor activity against FGFR-deregulated tumors in preclin. models.  AZD4547 is under clin. investigation for the treatment of FGFR-dependent tumors.  Cancer Res; 72(8); 2045-56.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnpExryJc9xbVg90H21EOLACvtfcHk0lhMXEmNh_yamw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xls1Wgsrs%253D&md5=cba90ad8f9b9e51cd10288868bbcc85e</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-11-3034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-11-3034%26sid%3Dliteratum%253Aachs%26aulast%3DGavine%26aufirst%3DP.%2BR.%26aulast%3DMooney%26aufirst%3DL.%26aulast%3DKilgour%26aufirst%3DE.%26aulast%3DThomas%26aufirst%3DA.%2BP.%26aulast%3DAl-Kadhimi%26aufirst%3DK.%26aulast%3DBeck%26aufirst%3DS.%26aulast%3DRooney%26aufirst%3DC.%26aulast%3DColeman%26aufirst%3DT.%26aulast%3DBaker%26aufirst%3DD.%26aulast%3DMellor%26aufirst%3DM.%2BJ.%26aulast%3DBrooks%26aufirst%3DA.%2BN.%26aulast%3DKlinowska%26aufirst%3DT.%26atitle%3DAZD4547%253A%2520An%2520orally%2520bioavailable%252C%2520potent%252C%2520and%2520selective%2520inhibitor%2520of%2520the%2520fibroblast%2520growth%2520factor%2520receptor%2520tyrosine%2520kinase%2520family%26jtitle%3DCancer%2520Res.%26date%3D2012%26volume%3D72%26spage%3D2045%26epage%3D2056%26doi%3D10.1158%2F0008-5472.CAN-11-3034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yosaatmadja, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalyukina, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Middleditch, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patterson, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smaill, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Squire, C. J.</span></span> <span> </span><span class="NLM_article-title">Rotational freedom, steric hindrance, and protein dynamics explain BLU554 selectivity for the hinge cysteine of FGFR4</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1180</span>– <span class="NLM_lpage">1186</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00196</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00196" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlams7vN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1180-1186&author=X.+Linauthor=Y.+Yosaatmadjaauthor=M.+Kalyukinaauthor=M.+J.+Middleditchauthor=Z.+Zhangauthor=X.+Luauthor=K.+Dingauthor=A.+V.+Pattersonauthor=J.+B.+Smaillauthor=C.+J.+Squire&title=Rotational+freedom%2C+steric+hindrance%2C+and+protein+dynamics+explain+BLU554+selectivity+for+the+hinge+cysteine+of+FGFR4&doi=10.1021%2Facsmedchemlett.9b00196"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Rotational Freedom, Steric Hindrance, and Protein Dynamics Explain BLU554 Selectivity for the Hinge Cysteine of FGFR4</span></div><div class="casAuthors">Lin, Xiaojing; Yosaatmadja, Yuliana; Kalyukina, Maria; Middleditch, Martin J.; Zhang, Zhen; Lu, Xiaoyun; Ding, Ke; Patterson, Adam V.; Smaill, Jeff B.; Squire, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1180-1186</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Aberration in FGFR4 signaling drives carcinogenesis and progression in a subset of hepatocellular carcinoma (HCC) patients, thereby making FGFR4 an attractive mol. target for this disease.  Selective FGFR4 inhibition can be achieved through covalently targeting a poorly conserved cysteine residue in the FGFR4 kinase domain.  We report mass spectrometry assays and cocrystal structures of FGFR4 in covalent complex with the clin. candidate BLU554 and with a series of four structurally related inhibitors that define the inherent reactivity and selectivity profile of these mols.  We further reveal the structure of FGFR1 with one of our inhibitors and show that off-target covalent binding can occur through an alternative conformation that supports targeting of a cysteine conserved in all members of the FGFR family.  Collectively, we propose that rotational freedom, steric hindrance, and protein dynamics explain the exceptional selectivity profile of BLU554 for targeting FGFR4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqag7zwpLVc7bVg90H21EOLACvtfcHk0lhTp1vXv2Ny0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlams7vN&md5=43f32bb16bbb3dd3873f8d7a16730a3e</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00196&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00196%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DX.%26aulast%3DYosaatmadja%26aufirst%3DY.%26aulast%3DKalyukina%26aufirst%3DM.%26aulast%3DMiddleditch%26aufirst%3DM.%2BJ.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DDing%26aufirst%3DK.%26aulast%3DPatterson%26aufirst%3DA.%2BV.%26aulast%3DSmaill%26aufirst%3DJ.%2BB.%26aulast%3DSquire%26aufirst%3DC.%2BJ.%26atitle%3DRotational%2520freedom%252C%2520steric%2520hindrance%252C%2520and%2520protein%2520dynamics%2520explain%2520BLU554%2520selectivity%2520for%2520the%2520hinge%2520cysteine%2520of%2520FGFR4%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D10%26spage%3D1180%26epage%3D1186%26doi%3D10.1021%2Facsmedchemlett.9b00196" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Knoepfel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furet, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mah, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buschmann, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leblanc, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ripoche, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graus-Porta, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wartmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galuba, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairhurst, R. A.</span></span> <span> </span><span class="NLM_article-title">2-Formylpyridyl ureas as highly selective reversible-covalent inhibitors of fibroblast growth factor receptor 4</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">215</span>– <span class="NLM_lpage">220</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.7b00485</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.7b00485" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1Srsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=215-220&author=T.+Knoepfelauthor=P.+Furetauthor=R.+Mahauthor=N.+Buschmannauthor=C.+Leblancauthor=S.+Ripocheauthor=D.+Graus-Portaauthor=M.+Wartmannauthor=I.+Galubaauthor=R.+A.+Fairhurst&title=2-Formylpyridyl+ureas+as+highly+selective+reversible-covalent+inhibitors+of+fibroblast+growth+factor+receptor+4&doi=10.1021%2Facsmedchemlett.7b00485"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">2-Formylpyridyl Ureas as Highly Selective Reversible-Covalent Inhibitors of Fibroblast Growth Factor Receptor 4</span></div><div class="casAuthors">Knoepfel, Thomas; Furet, Pascal; Mah, Robert; Buschmann, Nicole; Leblanc, Catherine; Ripoche, Sebastien; Graus-Porta, Diana; Wartmann, Markus; Galuba, Inga; Fairhurst, Robin A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">215-220</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">As part of a project to identify FGFR4 selective inhibitors, scaffold morphing of a 2-formylquinoline amide hit identified series of 2-formylpyridine ureas (2-FPUs) with improved potency and physicochem. properties.  In particular, tetrahydronaphthyridine urea analogs with cellular activities below 30 nM have been identified.  Consistent with the hypothesized reversible-covalent mechanism of inhibition, the 2-FPUs exhibited slow binding kinetics, and the aldehyde, as the putative electrophile, could be demonstrated to be a key structural element for activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpA1PRltYPuWrVg90H21EOLACvtfcHk0lhTp1vXv2Ny0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit1Srsb4%253D&md5=fd43811b8a9e5f7a02f27096569423f8</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.7b00485&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.7b00485%26sid%3Dliteratum%253Aachs%26aulast%3DKnoepfel%26aufirst%3DT.%26aulast%3DFuret%26aufirst%3DP.%26aulast%3DMah%26aufirst%3DR.%26aulast%3DBuschmann%26aufirst%3DN.%26aulast%3DLeblanc%26aufirst%3DC.%26aulast%3DRipoche%26aufirst%3DS.%26aulast%3DGraus-Porta%26aufirst%3DD.%26aulast%3DWartmann%26aufirst%3DM.%26aulast%3DGaluba%26aufirst%3DI.%26aulast%3DFairhurst%26aufirst%3DR.%2BA.%26atitle%3D2-Formylpyridyl%2520ureas%2520as%2520highly%2520selective%2520reversible-covalent%2520inhibitors%2520of%2520fibroblast%2520growth%2520factor%2520receptor%25204%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D9%26spage%3D215%26epage%3D220%26doi%3D10.1021%2Facsmedchemlett.7b00485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gehringer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laufer, S. A.</span></span> <span> </span><span class="NLM_article-title">Emerging and re-emerging warheads for targeted covalent inhibitors: applications in medicinal chemistry and chemical biology</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">5673</span>– <span class="NLM_lpage">5724</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01153</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01153" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFKiur%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=5673-5724&author=M.+Gehringerauthor=S.+A.+Laufer&title=Emerging+and+re-emerging+warheads+for+targeted+covalent+inhibitors%3A+applications+in+medicinal+chemistry+and+chemical+biology&doi=10.1021%2Facs.jmedchem.8b01153"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging and Re-Emerging Warheads for Targeted Covalent Inhibitors: Applications in Medicinal Chemistry and Chemical Biology</span></div><div class="casAuthors">Gehringer, Matthias; Laufer, Stefan A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5673-5724</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Targeted covalent inhibitors (TCIs) are designed to bind poorly conserved amino acids by means of reactive groups, the so-called warheads.  Currently, targeting noncatalytic cysteine residues with acrylamides and other α,β-unsatd. carbonyl compds. is the predominant strategy in TCI development.  The recent ascent of covalent drugs has stimulated considerable efforts to characterize alternative warheads for the covalent-reversible and irreversible engagement of noncatalytic cysteine residues as well as other amino acids.  This Perspective article provides an overview of warheads-beyond α,β-unsatd. amides-recently used in the design of targeted covalent ligands.  Promising reactive groups that have not yet demonstrated their utility in TCI development are also highlighted.  Special emphasis is placed on the discussion of reactivity and of case studies illustrating applications in medicinal chem. and chem. biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDhc4PWV293LVg90H21EOLACvtfcHk0lhTp1vXv2Ny0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFKiur%252FL&md5=4623733558ff99aadebfc13950b816ad</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01153&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01153%26sid%3Dliteratum%253Aachs%26aulast%3DGehringer%26aufirst%3DM.%26aulast%3DLaufer%26aufirst%3DS.%2BA.%26atitle%3DEmerging%2520and%2520re-emerging%2520warheads%2520for%2520targeted%2520covalent%2520inhibitors%253A%2520applications%2520in%2520medicinal%2520chemistry%2520and%2520chemical%2520biology%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D5673%26epage%3D5724%26doi%3D10.1021%2Facs.jmedchem.8b01153" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eliseenkova, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neubert, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohammadi, M.</span></span> <span> </span><span class="NLM_article-title">A ‘molecular brake’ in the kinase hinge region regulates the activity of receptor tyrosine kinases</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">717</span>– <span class="NLM_lpage">730</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2007.06.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1016%2Fj.molcel.2007.06.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=17803937" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFWlsL7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2007&pages=717-730&author=H.+Chenauthor=J.+Maauthor=W.+Liauthor=A.+V.+Eliseenkovaauthor=C.+Xuauthor=T.+A.+Neubertauthor=W.+T.+Millerauthor=M.+Mohammadi&title=A+%E2%80%98molecular+brake%E2%80%99+in+the+kinase+hinge+region+regulates+the+activity+of+receptor+tyrosine+kinases&doi=10.1016%2Fj.molcel.2007.06.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">A molecular brake in the kinase hinge region regulates the activity of receptor tyrosine kinases</span></div><div class="casAuthors">Chen, Huaibin; Ma, Jinghong; Li, Wanqing; Eliseenkova, Anna V.; Xu, Chongfeng; Neubert, Thomas A.; Miller, W. Todd; Mohammadi, Moosa</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">717-730</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Activating mutations in the tyrosine kinase domain of receptor tyrosine kinases (RTKs) cause cancer and skeletal disorders.  Comparison of the crystal structures of unphosphorylated and phosphorylated wild-type FGFR2 kinase domains (FGFR2Ks) with those of seven unphosphorylated pathogenic mutants reveals an autoinhibitory "mol. brake" mediated by a triad of residues in the kinase hinge region of all FGFRs.  Structural anal. shows that many other RTKs, including PDGFRs, VEGFRs, KIT, CSF1R, FLT3, TEK, and TIE, are also subject to regulation by this brake.  Pathogenic mutations activate FGFRs and other RTKs by disengaging the brake either directly or indirectly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobYTkvrN7lYrVg90H21EOLACvtfcHk0lha2437caNO3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFWlsL7I&md5=dba8a6b97b9d1826b2223146be137963</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2007.06.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2007.06.028%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DH.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DEliseenkova%26aufirst%3DA.%2BV.%26aulast%3DXu%26aufirst%3DC.%26aulast%3DNeubert%26aufirst%3DT.%2BA.%26aulast%3DMiller%26aufirst%3DW.%2BT.%26aulast%3DMohammadi%26aufirst%3DM.%26atitle%3DA%2520%25E2%2580%2598molecular%2520brake%25E2%2580%2599%2520in%2520the%2520kinase%2520hinge%2520region%2520regulates%2520the%2520activity%2520of%2520receptor%2520tyrosine%2520kinases%26jtitle%3DMol.%2520Cell%26date%3D2007%26volume%3D27%26spage%3D717%26epage%3D730%26doi%3D10.1016%2Fj.molcel.2007.06.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hagel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miduturu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheets, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stransky, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bifulco, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodous, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooijmans, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shutes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lengauer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohl, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzi, T.</span></span> <span> </span><span class="NLM_article-title">First selective small molecule inhibitor of FGFR4 for the treatment of hepatocellular carcinomas with an activated FGFR4 signaling pathway</span>. <i>Cancer Discovery</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">424</span>– <span class="NLM_lpage">437</span>, <span class="refDoi"> DOI: 10.1158/2159-8290.CD-14-1029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1158%2F2159-8290.CD-14-1029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=25776529" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmsVSrtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=424-437&author=M.+Hagelauthor=C.+Miduturuauthor=M.+Sheetsauthor=N.+Rubinauthor=W.+Wengauthor=N.+Stranskyauthor=N.+Bifulcoauthor=J.+L.+Kimauthor=B.+Hodousauthor=N.+Brooijmansauthor=A.+Shutesauthor=C.+Winterauthor=C.+Lengauerauthor=N.+E.+Kohlauthor=T.+Guzi&title=First+selective+small+molecule+inhibitor+of+FGFR4+for+the+treatment+of+hepatocellular+carcinomas+with+an+activated+FGFR4+signaling+pathway&doi=10.1158%2F2159-8290.CD-14-1029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">First Selective Small Molecule Inhibitor of FGFR4 for the Treatment of Hepatocellular Carcinomas with an Activated FGFR4 Signaling Pathway</span></div><div class="casAuthors">Hagel, Margit; Miduturu, Chandra; Sheets, Michael; Rubin, Nooreen; Weng, Weifan; Stransky, Nicolas; Bifulco, Neil; Kim, Joseph L.; Hodous, Brian; Brooijmans, Natasja; Shutes, Adam; Winter, Christopher; Lengauer, Christoph; Kohl, Nancy E.; Guzi, Timothy</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">424-437</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">HCC cells with FGF19 gene amplification or overexpression and an intact FGFR4 signaling pathway are sensitive to potent and selective FGFR4 inhibition by the small mol. BLU9931 in vitro and in xenograft models.  Aberrant signaling through the fibroblast growth factor 19 (FGF19)/fibroblast growth factor receptor 4 (FGFR 4) signaling complex has been shown to cause hepatocellular carcinoma (HCC) in mice and has been implicated to play a similar role in humans.  We have developed BLU9931, a potent and irreversible small-mol. inhibitor of FGFR4, as a targeted therapy to treat patients with HCC whose tumors have an activated FGFR4 signaling pathway.  BLU9931 is exquisitely selective for FGFR4 vs. other FGFR family members and all other kinases.  BLU9931 shows remarkable antitumor activity in mice bearing an HCC tumor xenograft that overexpresses FGF19 due to amplification as well as a liver tumor xenograft that overexpresses FGF19 mRNA but lacks FGF19 amplification.  Approx. one third of patients with HCC whose tumors express FGF19 together with FGFR4 and its coreceptor klotho β (KLB) could potentially respond to treatment with an FGFR4 inhibitor.  These findings are the first demonstration of a therapeutic strategy that targets a subset of patients with HCC.  Significance: This article documents the discovery of BLU9931, a novel irreversible kinase inhibitor that specifically targets FGFR4 while sparing all other FGFR paralogs and demonstrates exquisite kinome selectivity.  BLU9931 is efficacious in tumors with an intact FGFR4 signaling pathway that includes FGF19, FGFR4, and KLB.  BLU9931 is the first FGFR4-selective mol. for the treatment of patients with HCC with aberrant FGFR4 signaling.  Cancer Discov; 5(4); 424-37. cpr2015 AACR.  See related commentary by Packer and Pollock, p. 355 This article is highlighted in the In This Issue feature, p.  333.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqcHNr-1ZnG7Vg90H21EOLACvtfcHk0lha2437caNO3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmsVSrtL8%253D&md5=4a69439aee01a3c090c14851a785362e</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-14-1029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-14-1029%26sid%3Dliteratum%253Aachs%26aulast%3DHagel%26aufirst%3DM.%26aulast%3DMiduturu%26aufirst%3DC.%26aulast%3DSheets%26aufirst%3DM.%26aulast%3DRubin%26aufirst%3DN.%26aulast%3DWeng%26aufirst%3DW.%26aulast%3DStransky%26aufirst%3DN.%26aulast%3DBifulco%26aufirst%3DN.%26aulast%3DKim%26aufirst%3DJ.%2BL.%26aulast%3DHodous%26aufirst%3DB.%26aulast%3DBrooijmans%26aufirst%3DN.%26aulast%3DShutes%26aufirst%3DA.%26aulast%3DWinter%26aufirst%3DC.%26aulast%3DLengauer%26aufirst%3DC.%26aulast%3DKohl%26aufirst%3DN.%2BE.%26aulast%3DGuzi%26aufirst%3DT.%26atitle%3DFirst%2520selective%2520small%2520molecule%2520inhibitor%2520of%2520FGFR4%2520for%2520the%2520treatment%2520of%2520hepatocellular%2520carcinomas%2520with%2520an%2520activated%2520FGFR4%2520signaling%2520pathway%26jtitle%3DCancer%2520Discovery%26date%3D2015%26volume%3D5%26spage%3D424%26epage%3D437%26doi%3D10.1158%2F2159-8290.CD-14-1029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanizaki, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aref, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rusan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, S.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ercan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capelletti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hur, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sim, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaudet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbie, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, J-R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yun, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammerman, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohammadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jänne, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Development of covalent inhibitors that can overcome resistance to first-generation FGFR kinase inhibitors</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">E4869</span>– <span class="NLM_lpage">E4877</span>, <span class="refDoi"> DOI: 10.1073/pnas.1403438111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1073%2Fpnas.1403438111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=25349422" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVWmsL%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=E4869-E4877&author=L.+Tanauthor=J.+Wangauthor=J.+Tanizakiauthor=Z.+Huangauthor=A.+R.+Arefauthor=M.+Rusanauthor=S.-J.+Zhuauthor=Y.+Zhangauthor=D.+Ercanauthor=R.+G.+Liaoauthor=M.+Capellettiauthor=W.+Zhouauthor=W.+Hurauthor=N.+Kimauthor=T.+Simauthor=S.+Gaudetauthor=D.+A.+Barbieauthor=J-R.+J.+Yehauthor=C.-H.+Yunauthor=P.+S.+Hammermanauthor=M.+Mohammadiauthor=P.+A.+J%C3%A4nneauthor=N.+S.+Gray&title=Development+of+covalent+inhibitors+that+can+overcome+resistance+to+first-generation+FGFR+kinase+inhibitors&doi=10.1073%2Fpnas.1403438111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Development of covalent inhibitors that can overcome resistance to first-generation FGFR kinase inhibitors</span></div><div class="casAuthors">Tan, Li; Wang, Jun; Tanizaki, Junko; Huang, Zhifeng; Aref, Amir R.; Rusan, Maria; Zhu, Su-Jie; Zhang, Yiyun; Ercan, Dalia; Liao, Rachel G.; Capelletti, Marzia; Zhou, Wenjun; Hur, Wooyoung; Kim, NamDoo; Sim, Taebo; Gaudet, Suzanne; Barbie, David A.; Yeh, Jing-Ruey Joanna; Yun, Cai-Hong; Hammerman, Peter S.; Mohammadi, Moosa; Janne, Pasi A.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">E4869-E4877</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The human FGF receptors (FGFRs) play crit. roles in various human cancers, and several FGFR inhibitors are currently under clin. investigation.  Resistance usually results from selection for mutant kinases that are impervious to the action of the drug or from up-regulation of compensatory signaling pathways.  Preclin. studies have demonstrated that resistance to FGFR inhibitors can be acquired through mutations in the FGFR gatekeeper residue, as clin. obsd. for FGFR4 in embryonal rhabdomyosarcoma and neuroendocrine breast carcinomas.  Here the authors report on the use of a structure-based drug design to develop two selective, next-generation covalent FGFR inhibitors, the FGFR irreversible inhibitors I and II.  To our knowledge, I and II are the first inhibitors that can potently inhibit the proliferation of cells dependent upon the gatekeeper mutants of FGFR1 or FGFR2, which confer resistance to first-generation clin. FGFR inhibitors such as NVP-BGJ398 and AZD4547.  Because of the conformational flexibility of the reactive acrylamide substituent, II has the unprecedented ability to inhibit both the EGF receptor (EGFR) and FGFR covalently by targeting two distinct cysteine residues.  The authors report the cocrystal structure of FGFR4 with I, which unexpectedly exhibits a "DFG-out" covalent binding mode.  The structural basis for dual FGFR and EGFR targeting by FIIN3 also is illustrated by crystal structures of II bound with FGFR4 V550L and EGFR L858R.  These results have important implications for the design of covalent FGFR inhibitors that can overcome clin. resistance and provide the first example, to the knowledge, of a kinase inhibitor that covalently targets cysteines located in different positions within the ATP-binding pocket.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmCbTDKPA32LVg90H21EOLACvtfcHk0lha2437caNO3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVWmsL%252FP&md5=fd4a7eac1e0c17e7c9f2651dd2d76ac7</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1403438111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1403438111%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DTanizaki%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DAref%26aufirst%3DA.%2BR.%26aulast%3DRusan%26aufirst%3DM.%26aulast%3DZhu%26aufirst%3DS.-J.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DErcan%26aufirst%3DD.%26aulast%3DLiao%26aufirst%3DR.%2BG.%26aulast%3DCapelletti%26aufirst%3DM.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DHur%26aufirst%3DW.%26aulast%3DKim%26aufirst%3DN.%26aulast%3DSim%26aufirst%3DT.%26aulast%3DGaudet%26aufirst%3DS.%26aulast%3DBarbie%26aufirst%3DD.%2BA.%26aulast%3DYeh%26aufirst%3DJ-R.%2BJ.%26aulast%3DYun%26aufirst%3DC.-H.%26aulast%3DHammerman%26aufirst%3DP.%2BS.%26aulast%3DMohammadi%26aufirst%3DM.%26aulast%3DJ%25C3%25A4nne%26aufirst%3DP.%2BA.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DDevelopment%2520of%2520covalent%2520inhibitors%2520that%2520can%2520overcome%2520resistance%2520to%2520first-generation%2520FGFR%2520kinase%2520inhibitors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3DE4869%26epage%3DE4877%26doi%3D10.1073%2Fpnas.1403438111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hett, E. C.</span></span> <span> </span><span class="NLM_article-title">Know your target, know your molecule</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">368</span>– <span class="NLM_lpage">372</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1813</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1038%2Fnchembio.1813" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=25978985" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFemtrvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=368-372&author=M.+E.+Bunnageauthor=A.+M.+Gilbertauthor=L.+H.+Jonesauthor=E.+C.+Hett&title=Know+your+target%2C+know+your+molecule&doi=10.1038%2Fnchembio.1813"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Know your target, know your molecule</span></div><div class="casAuthors">Bunnage, Mark E.; Gilbert, Adam M.; Jones, Lyn H.; Hett, Erik C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">368-372</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review. The pharmaceutical industry continues to experience significant attrition of drug candidates during phase 2 proof-of-concept clin. studies.  We describe some questions about the characteristics of protein targets and small-mol. drugs that may be important to consider in drug-discovery projects and could improve prospects for future clin. success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRthCFSFvOXrVg90H21EOLACvtfcHk0lgAAYjF0J1I7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFemtrvK&md5=f9606135ccea6af432040d8afb630ede</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1813&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1813%26sid%3Dliteratum%253Aachs%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DGilbert%26aufirst%3DA.%2BM.%26aulast%3DJones%26aufirst%3DL.%2BH.%26aulast%3DHett%26aufirst%3DE.%2BC.%26atitle%3DKnow%2520your%2520target%252C%2520know%2520your%2520molecule%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D368%26epage%3D372%26doi%3D10.1038%2Fnchembio.1813" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barf, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaptein, A.</span></span> <span> </span><span class="NLM_article-title">Irreversible protein kinase inhibitors: balancing the benefits and risks</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">6243</span>– <span class="NLM_lpage">6262</span>, <span class="refDoi"> DOI: 10.1021/jm3003203</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3003203" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC38XnsVerur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=6243-6262&author=T.+Barfauthor=A.+Kaptein&title=Irreversible+protein+kinase+inhibitors%3A+balancing+the+benefits+and+risks&doi=10.1021%2Fjm3003203"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Irreversible Protein Kinase Inhibitors: Balancing the Benefits and Risks</span></div><div class="casAuthors">Barf, Tjeerd; Kaptein, Allard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6243-6262</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Not applicable for a Perspective.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4PLlisO0OVbVg90H21EOLACvtfcHk0lgAAYjF0J1I7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XnsVerur8%253D&md5=5d87850857d37faab02e6037ef8140a4</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fjm3003203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3003203%26sid%3Dliteratum%253Aachs%26aulast%3DBarf%26aufirst%3DT.%26aulast%3DKaptein%26aufirst%3DA.%26atitle%3DIrreversible%2520protein%2520kinase%2520inhibitors%253A%2520balancing%2520the%2520benefits%2520and%2520risks%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D6243%26epage%3D6262%26doi%3D10.1021%2Fjm3003203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schwanhäusser, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gossen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dittmar, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Selbach, M.</span></span> <span> </span><span class="NLM_article-title">Global analysis of cellular protein translation by pulsed SILAC</span>. <i>Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">205</span>– <span class="NLM_lpage">209</span>, <span class="refDoi"> DOI: 10.1002/pmic.200800275</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1002%2Fpmic.200800275" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=19053139" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A280%3ADC%252BD1M%252FkvFWhsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=205-209&author=B.+Schwanh%C3%A4usserauthor=M.+Gossenauthor=G.+Dittmarauthor=M.+Selbach&title=Global+analysis+of+cellular+protein+translation+by+pulsed+SILAC&doi=10.1002%2Fpmic.200800275"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Global analysis of cellular protein translation by pulsed SILAC</span></div><div class="casAuthors">Schwanhausser Bjorn; Gossen Manfred; Dittmar Gunnar; Selbach Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Proteomics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">205-9</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Current methods for system-wide gene expression analysis detect changes in mRNA abundance, but neglect regulation at the level of translation.  Pulse labeling with stable isotopes has been used to measure protein turnover rates, but this does not directly provide information about translation rates.  Here, we developed pulsed stable isotope labeling by amino acids in cell culture (pSILAC) with two heavy isotope labels to directly quantify protein translation on a proteome-wide scale.  We applied the method to cellular iron homeostasis as a model system and demonstrate that it can confidently identify proteins that are translationally regulated by iron availability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTlGBzGGsSeDG52AMpI4GK2fW6udTcc2eZnvIS-b94tMLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1M%252FkvFWhsQ%253D%253D&md5=3f223ce86b049cf6cd41a321e710f07b</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1002%2Fpmic.200800275&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpmic.200800275%26sid%3Dliteratum%253Aachs%26aulast%3DSchwanh%25C3%25A4usser%26aufirst%3DB.%26aulast%3DGossen%26aufirst%3DM.%26aulast%3DDittmar%26aufirst%3DG.%26aulast%3DSelbach%26aufirst%3DM.%26atitle%3DGlobal%2520analysis%2520of%2520cellular%2520protein%2520translation%2520by%2520pulsed%2520SILAC%26jtitle%3DProteomics%26date%3D2009%26volume%3D9%26spage%3D205%26epage%3D209%26doi%3D10.1002%2Fpmic.200800275" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zapf, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstenberger, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Limburg, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caspers, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aulabaugh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurumbail, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shakya, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spaulding, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czerwinski, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seth, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medley, Q. G.</span></span> <span> </span><span class="NLM_article-title">Covalent inhibitors of interleukin-2 inducible T cell kinase (Itk) with nanomolar potency in a whole-blood assay</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">10047</span>– <span class="NLM_lpage">10063</span>, <span class="refDoi"> DOI: 10.1021/jm301190s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301190s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFOlsbnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=10047-10063&author=C.+W.+Zapfauthor=B.+S.+Gerstenbergerauthor=L.+Xingauthor=D.+C.+Limburgauthor=D.+R.+Andersonauthor=N.+Caspersauthor=S.+Hanauthor=A.+Aulabaughauthor=R.+Kurumbailauthor=S.+Shakyaauthor=X.+Liauthor=V.+Spauldingauthor=R.+M.+Czerwinskiauthor=N.+Sethauthor=Q.+G.+Medley&title=Covalent+inhibitors+of+interleukin-2+inducible+T+cell+kinase+%28Itk%29+with+nanomolar+potency+in+a+whole-blood+assay&doi=10.1021%2Fjm301190s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent Inhibitors of Interleukin-2 Inducible T Cell Kinase (Itk) with Nanomolar Potency in a Whole-Blood Assay</span></div><div class="casAuthors">Zapf, Christoph W.; Gerstenberger, Brian S.; Xing, Li; Limburg, David C.; Anderson, David R.; Caspers, Nicole; Han, Seungil; Aulabaugh, Ann; Kurumbail, Ravi; Shakya, Subarna; Li, Xin; Spaulding, Vikki; Czerwinski, Robert M.; Seth, Nilufer; Medley, Quintus G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">10047-10063</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We wish to report a strategy that targets interleukin-2 inducible T cell kinase (Itk) with covalent inhibitors.  Thus far, covalent inhibition of Itk has not been disclosed in the literature.  Structure-based drug design was utilized to achieve low nanomolar potency of the disclosed series even at high ATP concns.  Kinetic measurements confirmed an irreversible binding mode with off-rate half-lives exceeding 24 h and moderate on-rates.  The analogs are highly potent in a cellular IP1 assay as well as in a human whole-blood (hWB) assay.  Despite a half-life of approx. 2 h in resting primary T cells, the covalent inhibition of Itk resulted in functional silencing of the TCR pathway for more than 24 h.  This prolonged effect indicates that covalent inhibition is a viable strategy to target the inactivation of Itk.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnh9j28SM8jbVg90H21EOLACvtfcHk0ljMt_1MpQHIYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFOlsbnP&md5=7e39a17151dc9493d13439bc48fa83a3</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm301190s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301190s%26sid%3Dliteratum%253Aachs%26aulast%3DZapf%26aufirst%3DC.%2BW.%26aulast%3DGerstenberger%26aufirst%3DB.%2BS.%26aulast%3DXing%26aufirst%3DL.%26aulast%3DLimburg%26aufirst%3DD.%2BC.%26aulast%3DAnderson%26aufirst%3DD.%2BR.%26aulast%3DCaspers%26aufirst%3DN.%26aulast%3DHan%26aufirst%3DS.%26aulast%3DAulabaugh%26aufirst%3DA.%26aulast%3DKurumbail%26aufirst%3DR.%26aulast%3DShakya%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DSpaulding%26aufirst%3DV.%26aulast%3DCzerwinski%26aufirst%3DR.%2BM.%26aulast%3DSeth%26aufirst%3DN.%26aulast%3DMedley%26aufirst%3DQ.%2BG.%26atitle%3DCovalent%2520inhibitors%2520of%2520interleukin-2%2520inducible%2520T%2520cell%2520kinase%2520%2528Itk%2529%2520with%2520nanomolar%2520potency%2520in%2520a%2520whole-blood%2520assay%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D10047%26epage%3D10063%26doi%3D10.1021%2Fjm301190s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guagnano, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furet, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spanka, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bordas, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Douget, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stamm, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brueggen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wartmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berghausen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drueckes, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerlin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bussiere, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graus Porta, D.</span></span> <span> </span><span class="NLM_article-title">Discovery of 3-(2,6-dichloro-3,5-dimethoxy-phenyl)-1-{6-[4-(4-ethyl-piperazin-1-yl)-phenylamino]-pyrimidin-4-yl}-1-methyl-urea (NVP-BGJ398), a potent and selective inhibitor of the fibroblast growth factor receptor family of receptor tyrosine kinases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">7066</span>– <span class="NLM_lpage">7083</span>, <span class="refDoi"> DOI: 10.1021/jm2006222</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2006222" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1SrsrvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=7066-7083&author=V.+Guagnanoauthor=P.+Furetauthor=C.+Spankaauthor=V.+Bordasauthor=M.+Le+Dougetauthor=C.+Stammauthor=J.+Brueggenauthor=M.+R.+Jensenauthor=C.+Schnellauthor=H.+Schmidauthor=M.+Wartmannauthor=J.+Berghausenauthor=P.+Drueckesauthor=A.+Zimmerlinauthor=D.+Bussiereauthor=J.+Murrayauthor=D.+Graus+Porta&title=Discovery+of+3-%282%2C6-dichloro-3%2C5-dimethoxy-phenyl%29-1-%7B6-%5B4-%284-ethyl-piperazin-1-yl%29-phenylamino%5D-pyrimidin-4-yl%7D-1-methyl-urea+%28NVP-BGJ398%29%2C+a+potent+and+selective+inhibitor+of+the+fibroblast+growth+factor+receptor+family+of+receptor+tyrosine+kinases&doi=10.1021%2Fjm2006222"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 3-(2,6-Dichloro-3,5-dimethoxy-phenyl)-1-{6-[4-(4-ethyl-piperazin-1-yl)-phenylamino]-pyrimidin-4-yl}-1-methyl-urea (NVP-BGJ398), A Potent and Selective Inhibitor of the Fibroblast Growth Factor Receptor Family of Receptor Tyrosine Kinase</span></div><div class="casAuthors">Guagnano, Vito; Furet, Pascal; Spanka, Carsten; Bordas, Vincent; Le Douget, Mickael; Stamm, Christelle; Brueggen, Josef; Jensen, Michael R.; Schnell, Christian; Schmid, Herbert; Wartmann, Markus; Berghausen, Joerg; Drueckes, Peter; Zimmerlin, Alfred; Bussiere, Dirksen; Murray, Jeremy; Graus Porta, Diana</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">7066-7083</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel series of N-aryl-N'-pyrimidin-4-yl ureas has been optimized to afford potent and selective inhibitors of the fibroblast growth factor receptor tyrosine kinases 1, 2, and 3 by rationally designing the substitution pattern of the aryl ring.  On the basis of its in vitro profile, compd. 1h (NVP-BGJ398) was selected for in vivo evaluation and showed significant antitumor activity in RT112 bladder cancer xenografts models overexpressing wild-type FGFR3.  These results support the potential therapeutic use of 1h as a new anticancer agent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSC1v0A_6I_bVg90H21EOLACvtfcHk0ljMt_1MpQHIYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1SrsrvK&md5=f9a2d742294cd3814e872bdd8add1ffa</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fjm2006222&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2006222%26sid%3Dliteratum%253Aachs%26aulast%3DGuagnano%26aufirst%3DV.%26aulast%3DFuret%26aufirst%3DP.%26aulast%3DSpanka%26aufirst%3DC.%26aulast%3DBordas%26aufirst%3DV.%26aulast%3DLe%2BDouget%26aufirst%3DM.%26aulast%3DStamm%26aufirst%3DC.%26aulast%3DBrueggen%26aufirst%3DJ.%26aulast%3DJensen%26aufirst%3DM.%2BR.%26aulast%3DSchnell%26aufirst%3DC.%26aulast%3DSchmid%26aufirst%3DH.%26aulast%3DWartmann%26aufirst%3DM.%26aulast%3DBerghausen%26aufirst%3DJ.%26aulast%3DDrueckes%26aufirst%3DP.%26aulast%3DZimmerlin%26aufirst%3DA.%26aulast%3DBussiere%26aufirst%3DD.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DGraus%2BPorta%26aufirst%3DD.%26atitle%3DDiscovery%2520of%25203-%25282%252C6-dichloro-3%252C5-dimethoxy-phenyl%2529-1-%257B6-%255B4-%25284-ethyl-piperazin-1-yl%2529-phenylamino%255D-pyrimidin-4-yl%257D-1-methyl-urea%2520%2528NVP-BGJ398%2529%252C%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520the%2520fibroblast%2520growth%2520factor%2520receptor%2520family%2520of%2520receptor%2520tyrosine%2520kinases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D7066%26epage%3D7083%26doi%3D10.1021%2Fjm2006222" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, R. A.</span></span> <span> </span><span class="NLM_article-title">Covalent inhibitors in drug discovery: from accidental discoveries to avoided liabilities and designed therapies</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">1061</span>– <span class="NLM_lpage">1073</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2015.05.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1016%2Fj.drudis.2015.05.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=26002380" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptlOnu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=1061-1073&author=R.+A.+Bauer&title=Covalent+inhibitors+in+drug+discovery%3A+from+accidental+discoveries+to+avoided+liabilities+and+designed+therapies&doi=10.1016%2Fj.drudis.2015.05.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent inhibitors in drug discovery: from accidental discoveries to avoided liabilities and designed therapies</span></div><div class="casAuthors">Bauer, Renato A.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1061-1073</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Drugs that covalently bond to their biol. targets have a long history in drug discovery.  A look at drug approvals in recent years suggests that covalent drugs will continue to make impacts on human health for years to come.  Although fraught with concerns about toxicity, the high potencies and prolonged effects achievable with covalent drugs may result in less-frequent drug dosing and in wide therapeutic margins for patients.  Covalent inhibition can also dissoc. drug pharmacodynamics (PD) from pharmacokinetics (PK), which can result in desired drug efficacy for inhibitors that have short systemic exposure.  Evidence suggests that there is a reduced risk for the development of resistance against covalent drugs, which is a major challenge in areas such as oncol. and infectious disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMkBDsUEu4arVg90H21EOLACvtfcHk0ljMt_1MpQHIYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptlOnu7s%253D&md5=6ba5b016d2804c7d56838a2724b9a0b9</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2015.05.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2015.05.005%26sid%3Dliteratum%253Aachs%26aulast%3DBauer%26aufirst%3DR.%2BA.%26atitle%3DCovalent%2520inhibitors%2520in%2520drug%2520discovery%253A%2520from%2520accidental%2520discoveries%2520to%2520avoided%2520liabilities%2520and%2520designed%2520therapies%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2015%26volume%3D20%26spage%3D1061%26epage%3D1073%26doi%3D10.1016%2Fj.drudis.2015.05.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Uetrecht, J.</span></span> <span> </span><span class="NLM_article-title">Idiosyncratic drug reactions: past, present, and future</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">84</span>– <span class="NLM_lpage">92</span>, <span class="refDoi"> DOI: 10.1021/tx700186p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx700186p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlygsr%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2008&pages=84-92&author=J.+Uetrecht&title=Idiosyncratic+drug+reactions%3A+past%2C+present%2C+and+future&doi=10.1021%2Ftx700186p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Idiosyncratic Drug Reactions: Past, Present, and Future</span></div><div class="casAuthors">Uetrecht, Jack</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">84-92</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Although the major working hypothesis for the mechanism of idiosyncratic drug reactions (IDRs), the hapten hypothesis, has not changed since 1987, several hypotheses have been added, for example, the danger hypothesis and the pharmaceutical interaction hypothesis.  Genetic studies have found that several IDRs are linked to specific HLA genes, providing addnl. evidence that they are immune-mediated.  Evidence that most IDRs are caused by reactive metabolites has led pharmaceutical companies to avoid drug candidates that form significant amts. of reactive metabolites; however, at least one IDR, ximelagatran-induced liver toxicity, does not appear to be caused by a reactive metabolite.  It is possible that there are biomarkers such as those related to cell stress that would predict that a drug candidate would cause a significant incidence of IDRs; however, there has been no systematic study of the changes in gene expression induced by drugs known to cause IDRs.  A major impediment to the study of the mechanisms of IDRs is the paucity of valid animal models, and if we had a better mechanistic understanding, it should be easier to develop such models.  There is growing evidence that these adverse reactions are more varied and complex than previously recognized, and it is unlikely that a quick fix will be achieved.  However, IDRs are an important cause of patient morbidity and mortality and markedly increase the uncertainty of drug development; therefore, continued basic research in this area is essential.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLG5_bNuaw4LVg90H21EOLACvtfcHk0lhyu8BsJEf09A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlygsr%252FN&md5=eaed35d895118a05034c7762efd060dc</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Ftx700186p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx700186p%26sid%3Dliteratum%253Aachs%26aulast%3DUetrecht%26aufirst%3DJ.%26atitle%3DIdiosyncratic%2520drug%2520reactions%253A%2520past%252C%2520present%252C%2520and%2520future%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2008%26volume%3D21%26spage%3D84%26epage%3D92%26doi%3D10.1021%2Ftx700186p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">LoPachin, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gavin, T.</span></span> <span> </span><span class="NLM_article-title">Molecular mechanisms of aldehyde toxicity: a chemical perspective</span>. <i>Chem. Res. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1081</span>– <span class="NLM_lpage">1091</span>, <span class="refDoi"> DOI: 10.1021/tx5001046</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx5001046" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpsVensrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2014&pages=1081-1091&author=R.+M.+LoPachinauthor=T.+Gavin&title=Molecular+mechanisms+of+aldehyde+toxicity%3A+a+chemical+perspective&doi=10.1021%2Ftx5001046"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Mechanisms of Aldehyde Toxicity: A Chemical Perspective</span></div><div class="casAuthors">LoPachin, Richard M.; Gavin, Terrence</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1081-1091</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Aldehydes are electrophilic compds. to which humans are pervasively exposed.  Despite a significant health risk due to exposure, the mechanisms of aldehyde toxicity are poorly understood.  This ambiguity is likely due to the structural diversity of aldehyde derivs. and corresponding differences in chem. reactions and biol. targets.  To gain mechanistic insight, the authors have used parameters based on the hard and soft, acids and bases (HSAB) theory to profile the different aldehyde subclasses with respect to electronic character (softness, hardness), electrophilic reactivity (electrophilic index), and biol. nucleophilic targets.  The authors' analyses indicate that short chain aldehydes and longer chain satd. alkanals are hard electrophiles that cause toxicity by forming adducts with hard biol. nucleophiles, e.g., primary nitrogen groups on lysine residues.  In contrast, α,β-unsatd. carbonyl derivs., alkenals, and the α-oxoaldehydes are soft electrophiles that preferentially react with soft nucleophilic thiolate groups on cysteine residues.  The aldehydes can therefore be grouped into subclasses according to common electronic characteristics (softness/hardness) and mol. mechanisms of toxicity.  As the authors will discuss, the toxic potencies of these subgroups are generally related to corresponding electrophilicities.  For some aldehydes, however, predictions of toxicity based on electrophilicity are less accurate due to inherent physicochem. variables that limit target accessibility, e.g., steric hindrance and soly.  The unsatd. aldehydes are also members of the conjugated type-2 alkene chem. class that includes α,β-unsatd. amide, ketone, and ester derivs.  Type-2 alkenes are electrophiles of varying softness and electrophilicity that share a common mechanism of toxicity.  Therefore, exposure to an environmental mixt. of unsatd. carbonyl derivs. could cause "type-2 alkene toxicity" through additive interactions.  Finally, the authors propose that environmentally derived aldehydes can accelerate diseases by interacting with endogenous aldehydes generated during oxidative stress.  This review provides a basis for understanding aldehyde mechanisms and environmental toxicity through the context of electronic structure, electrophilicity, and nucleophile target selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrATFAViXVn7rVg90H21EOLACvtfcHk0lhyu8BsJEf09A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpsVensrg%253D&md5=e72707796ea59ccaff0283848ebbdf76</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Ftx5001046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx5001046%26sid%3Dliteratum%253Aachs%26aulast%3DLoPachin%26aufirst%3DR.%2BM.%26aulast%3DGavin%26aufirst%3DT.%26atitle%3DMolecular%2520mechanisms%2520of%2520aldehyde%2520toxicity%253A%2520a%2520chemical%2520perspective%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2014%26volume%3D27%26spage%3D1081%26epage%3D1091%26doi%3D10.1021%2Ftx5001046" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span> <span> </span><span class="NLM_article-title">The dynamics of drug-target interactions: drug-target residence time and its impact on efficacy and safety</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">305</span>– <span class="NLM_lpage">310</span>, <span class="refDoi"> DOI: 10.1517/17460441003677725</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1517%2F17460441003677725" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=22823083" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjvFSgsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=305-310&author=R.+A.+Copeland&title=The+dynamics+of+drug-target+interactions%3A+drug-target+residence+time+and+its+impact+on+efficacy+and+safety&doi=10.1517%2F17460441003677725"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41aR"><div class="casContent"><span class="casTitleNuber">41a</span><div class="casTitle"><span class="NLM_cas:atitle">The dynamics of drug-target interactions: drug-target residence time and its impact on efficacy and safety</span></div><div class="casAuthors">Copeland, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">305-310</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  The extent and duration of pharmacol. action is detd. by the lifetime of drug occupancy on a mol. target.  This lifetime is defined by dynamic processes that control the rates of drug assocn. and dissocn. from the target.  Recently, the term residence time has been coined to describe exptl. measurements that can be related to the lifetime of the binary drug-target complex, and this in turn to durable, pharmacodynamic activity.  The residence time concept and its impact on drug optimization are reviewed here.  Examples are provided that demonstrate how a long residence time can improve drug efficacy in vivo.  Addnl., optimization of drug-target residence time can help to mitigate off-target mediated toxicity, hence, improving drug safety and tolerability.  Recent applications of the residence time concept to both drug discovery and development are also presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyt3KrIYdRjbVg90H21EOLACvtfcHk0li35ANmQXSzYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjvFSgsbo%253D&md5=aca925e00abefddf6e92d83b470d96f1</span></div><a href="/servlet/linkout?suffix=cit41a&amp;dbid=16384&amp;doi=10.1517%2F17460441003677725&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441003677725%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26atitle%3DThe%2520dynamics%2520of%2520drug-target%2520interactions%253A%2520drug-target%2520residence%2520time%2520and%2520its%2520impact%2520on%2520efficacy%2520and%2520safety%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2010%26volume%3D5%26spage%3D305%26epage%3D310%26doi%3D10.1517%2F17460441003677725" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit41b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Swinney, D. C.</span></span> <span> </span><span class="NLM_article-title">The role of binding kinetics in therapeutically useful drug action</span>. <i>Curr. Opin. Drug Discovery Dev.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">31</span>– <span class="NLM_lpage">39</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=19152211" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnt12ltrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2009&pages=31-39&author=D.+C.+Swinney&title=The+role+of+binding+kinetics+in+therapeutically+useful+drug+action"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41bR"><div class="casContent"><span class="casTitleNuber">41b</span><div class="casTitle"><span class="NLM_cas:atitle">The role of binding kinetics in therapeutically useful drug action</span></div><div class="casAuthors">Swinney, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Drug Discovery & Development</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">31-39</span>CODEN:
                <span class="NLM_cas:coden">CODDFF</span>;
        ISSN:<span class="NLM_cas:issn">2040-3437</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">A review.  Binding kinetics help define how a medicine will communicate with physiol. to produce a desired therapeutic response.  Clin. efficacy, duration of action, clin. differentiation and safety may all be influenced by binding kinetics.  The optimization of binding kinetics can be used to maximize a drug's therapeutic index and thereby decrease drug attrition.  The gap between basic scientific principles and the potential medical value of a drug is currently bridged by the use of empirical assays.  The value of binding kinetics to drug discovery will be increased through an improved ability to identify optimal kinetic mechanisms and define kinetic structure activity relationships.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8IoReBfGAabVg90H21EOLACvtfcHk0ljgQ2_mVeUrjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnt12ltrk%253D&md5=129948a7fb0a60f3b9995e5aedb700a4</span></div><a href="/servlet/linkout?suffix=cit41b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSwinney%26aufirst%3DD.%2BC.%26atitle%3DThe%2520role%2520of%2520binding%2520kinetics%2520in%2520therapeutically%2520useful%2520drug%2520action%26jtitle%3DCurr.%2520Opin.%2520Drug%2520Discovery%2520Dev.%26date%3D2009%26volume%3D12%26spage%3D31%26epage%3D39" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Caraballo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribeiro, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiménez-Barbero, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramström, O.</span></span> <span> </span><span class="NLM_article-title">Direct STD NMR identification of β-galactose inhibitors from a virtual dynamic hemithioacetal system</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">589</span>– <span class="NLM_lpage">593</span>, <span class="refDoi"> DOI: 10.1002/anie.200903920</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1002%2Fanie.200903920" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC3cXktlaksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2010&pages=589-593&author=R.+Caraballoauthor=H.+Dongauthor=J.+P.+Ribeiroauthor=J.+Jim%C3%A9nez-Barberoauthor=O.+Ramstr%C3%B6m&title=Direct+STD+NMR+identification+of+%CE%B2-galactose+inhibitors+from+a+virtual+dynamic+hemithioacetal+system&doi=10.1002%2Fanie.200903920"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Direct STD NMR Identification of β-Galactosidase Inhibitors from a Virtual Dynamic Hemithioacetal System</span></div><div class="casAuthors">Caraballo, Remi; Dong, Hai; Ribeiro, Joao P.; Jimenez-Barbero, Jesus; Ramstroem, Olof</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">589-593, S589/1-S589/20</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Herein we show that hemithioacetal (HTA) formation is a fast and efficient reversible reaction that can be used to generate systems of potential enzyme inhibitors in aq. media under neutral conditions.  These systems were also subjected to direct binding anal. by satn. transfer difference (STD) NMR spectroscopy, a rapid and efficient technique for protein-ligand binding studies, in which the optimal HTA interactions with the biol. target can be readily deduced.  We have successfully demonstrated, for the first time, hemithioacetal formation applied to dynamic combinatorial system generation in aq. media.  Equil. formation using this chem. proved very rapid and resulted in truly virtual dynamic systems in which 1H NMR analyses confirmed the presence of transient HTA constituents.  It has further been demonstrated that 1H STD NMR spectroscopy can be used for efficient and direct in situ identification of the best enzyme binders of the virtual dynamic system.  Inhibition studies unequivocally support the 1H STD NMR.exptl. data and enable the distinction of different β-galactosidase inhibitors and nonspecific binders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_95z4ob2R57Vg90H21EOLACvtfcHk0ljgQ2_mVeUrjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXktlaksw%253D%253D&md5=012f6210d6e743fe5dd958cc4500d937</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1002%2Fanie.200903920&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200903920%26sid%3Dliteratum%253Aachs%26aulast%3DCaraballo%26aufirst%3DR.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DRibeiro%26aufirst%3DJ.%2BP.%26aulast%3DJim%25C3%25A9nez-Barbero%26aufirst%3DJ.%26aulast%3DRamstr%25C3%25B6m%26aufirst%3DO.%26atitle%3DDirect%2520STD%2520NMR%2520identification%2520of%2520%25CE%25B2-galactose%2520inhibitors%2520from%2520a%2520virtual%2520dynamic%2520hemithioacetal%2520system%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2010%26volume%3D49%26spage%3D589%26epage%3D593%26doi%3D10.1002%2Fanie.200903920" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Warmuth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adrián, F.</span></span> <span> </span><span class="NLM_article-title">Ba/F3 cells and their use in kinase drug discovery</span>. <i>Curr. Opin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">55</span>– <span class="NLM_lpage">60</span>, <span class="refDoi"> DOI: 10.1097/CCO.0b013e328011a25f</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1097%2FCCO.0b013e328011a25f" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=17133113" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1Cnt7%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2007&pages=55-60&author=M.+Warmuthauthor=S.+Kimauthor=X.+Guauthor=G.+Xiaauthor=F.+Adri%C3%A1n&title=Ba%2FF3+cells+and+their+use+in+kinase+drug+discovery&doi=10.1097%2FCCO.0b013e328011a25f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Ba/F3 cells and their use in kinase drug discovery</span></div><div class="casAuthors">Warmuth, Markus; Kim, Sungjoon; Gu, Xiang-ju; Xia, Gang; Adrian, Francisco</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Oncology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">55-60</span>CODEN:
                <span class="NLM_cas:coden">CUOOE8</span>;
        ISSN:<span class="NLM_cas:issn">1040-8746</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">A review.  Purpose of review: Due to their ability to function as dominant oncogenes, protein kinases have become favored targets in the quest for 'molecularly-targeted' cancer chemotherapeutics.  The discovery of a large no. of cancer-assocd. mutations in the kinome, and the progress in developing specific small-mol. kinase inhibitors has increased the need for accurate, reproducible, and efficient kinase activity-dependent cellular assay systems.  Recent findings: Ba/F3, a murine interleukin-3 dependent pro-B cell line is increasingly popular as a model system for assessing both the potency and downstream signaling of kinase oncogenes, and the ability of small-mol. kinase inhibitors to block kinase activity.  Facilitated by their growth properties, Ba/F3 cells have recently been adapted to high-throughput assay formats for compd. profiling.  Further, several published approaches show promise in predicting resistance to small-mol. kinase inhibitors elicited by point mutations interfering with inhibitor binding.  Summary: Ba/F3 cells are an increasingly popular tool in kinase drug discovery.  The ability to test the transforming capacity of newly identified kinase mutations, and to profile drug candidates and compd. libraries in high-throughput fashion, combined with the use of Ba/F3 cells to predict clin. resistance will greatly facilitate developments in this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXQmhiRwGlDLVg90H21EOLACvtfcHk0ljgQ2_mVeUrjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1Cnt7%252FL&md5=a7f8d84dfedea69d408edb571e2eead9</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1097%2FCCO.0b013e328011a25f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FCCO.0b013e328011a25f%26sid%3Dliteratum%253Aachs%26aulast%3DWarmuth%26aufirst%3DM.%26aulast%3DKim%26aufirst%3DS.%26aulast%3DGu%26aufirst%3DX.%26aulast%3DXia%26aufirst%3DG.%26aulast%3DAdri%25C3%25A1n%26aufirst%3DF.%26atitle%3DBa%252FF3%2520cells%2520and%2520their%2520use%2520in%2520kinase%2520drug%2520discovery%26jtitle%3DCurr.%2520Opin.%2520Oncol.%26date%3D2007%26volume%3D19%26spage%3D55%26epage%3D60%26doi%3D10.1097%2FCCO.0b013e328011a25f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hinson, A. R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crose, L. E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belyea, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barr, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linardic, C. M.</span></span> <span> </span><span class="NLM_article-title">Human rhabdomyosarcoma cell lines for rhabdomyosarcoma research: utility and pitfalls</span>. <i>Front. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">183</span>, <span class="refDoi"> DOI: 10.3389/fonc.2013.00183</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.3389%2Ffonc.2013.00183" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=23882450" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A280%3ADC%252BC3sfjsFOhtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2013&pages=183&author=A.+R.+P.+Hinsonauthor=R.+Jonesauthor=L.+E.+S.+Croseauthor=B.+C.+Belyeaauthor=F.+G.+Barrauthor=C.+M.+Linardic&title=Human+rhabdomyosarcoma+cell+lines+for+rhabdomyosarcoma+research%3A+utility+and+pitfalls&doi=10.3389%2Ffonc.2013.00183"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Human rhabdomyosarcoma cell lines for rhabdomyosarcoma research: utility and pitfalls</span></div><div class="casAuthors">Hinson Ashley R P; Jones Rosanne; Crose Lisa E S; Belyea Brian C; Barr Frederic G; Linardic Corinne M</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in oncology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">183</span>
        ISSN:<span class="NLM_cas:issn">2234-943X</span>.
    </div><div class="casAbstract">Rhabdomyosarcoma (RMS) is the most common soft tissue sarcoma of childhood and adolescence.  Despite intergroup clinical trials conducted in Europe and North America, outcomes for high risk patients with this disease have not significantly improved in the last several decades, and survival of metastatic or relapsed disease remains extremely poor.  Accrual into new clinical trials is slow and difficult, so in vitro cell-line research and in vivo xenograft models present an attractive alternative for preclinical research for this cancer type.  Currently, 30 commonly used human RMS cell lines exist, with differing origins, karyotypes, histologies, and methods of validation.  Selecting an appropriate cell line for RMS research has important implications for outcomes.  There are also potential pitfalls in using certain cell lines including contamination with murine stromal cells, cross-contamination between cell lines, discordance between the cell line and its associated original tumor, imposter cell lines, and nomenclature errors that result in the circulation of two or more presumed unique cell lines that are actually from the same origin.  These pitfalls can be avoided by testing for species-specific isoenzymes, microarray analysis, assays for subtype-specific fusion products, and short tandem repeat analysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSN0jeRw0uUgDEC-75aqa5AfW6udTcc2ealAXp8arI52rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sfjsFOhtA%253D%253D&md5=ff7c421705574030c7566122a562d59a</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.3389%2Ffonc.2013.00183&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffonc.2013.00183%26sid%3Dliteratum%253Aachs%26aulast%3DHinson%26aufirst%3DA.%2BR.%2BP.%26aulast%3DJones%26aufirst%3DR.%26aulast%3DCrose%26aufirst%3DL.%2BE.%2BS.%26aulast%3DBelyea%26aufirst%3DB.%2BC.%26aulast%3DBarr%26aufirst%3DF.%2BG.%26aulast%3DLinardic%26aufirst%3DC.%2BM.%26atitle%3DHuman%2520rhabdomyosarcoma%2520cell%2520lines%2520for%2520rhabdomyosarcoma%2520research%253A%2520utility%2520and%2520pitfalls%26jtitle%3DFront.%2520Oncol.%26date%3D2013%26volume%3D3%26spage%3D183%26doi%3D10.3389%2Ffonc.2013.00183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Michael, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doherty, M. M.</span></span> <span> </span><span class="NLM_article-title">Drug metabolism by tumors: its nature, relevance and therapeutic implications</span>. <i>Expert Opin. Drug Metab. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">783</span>– <span class="NLM_lpage">803</span>, <span class="refDoi"> DOI: 10.1517/17425255.3.6.783</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1517%2F17425255.3.6.783" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=18028025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlCls7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=783-803&author=M.+Michaelauthor=M.+M.+Doherty&title=Drug+metabolism+by+tumors%3A+its+nature%2C+relevance+and+therapeutic+implications&doi=10.1517%2F17425255.3.6.783"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Drug metabolism by tumors: its nature, relevance and therapeutic implications</span></div><div class="casAuthors">Michael, Michael; Doherty, Margaret M.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Metabolism & Toxicology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">783-803</span>CODEN:
                <span class="NLM_cas:coden">EODMAP</span>;
        ISSN:<span class="NLM_cas:issn">1742-5255</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Drug-metabolizing enzymes (DMEs) are present in tumors and are capable of biotransforming a variety of antineoplastics.  Tumoral drug metab. is both a potential mechanism of resistance and a means of achieving optimal therapy.  This review addresses the classes of DMEs, their cytotoxic substrates and distribution in specific malignancies.  The limitations of preclin. and clin. studies are highlighted.  Their role in predicting therapeutic response, the activation of prodrugs and the potential for their modulation for gain is also addressed.  The contribution of tumoral DMEs to cancer therapy can only be ascertained through large prospective studies and supported by new technologies.  Only then can efforts be concd. in the design of better prodrugs or combination therapy to optimize individual therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5aLOaiaVgPrVg90H21EOLACvtfcHk0li6ulXlH42R0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlCls7fK&md5=40fa6309e929f7f7ba1bc063d76744ef</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1517%2F17425255.3.6.783&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17425255.3.6.783%26sid%3Dliteratum%253Aachs%26aulast%3DMichael%26aufirst%3DM.%26aulast%3DDoherty%26aufirst%3DM.%2BM.%26atitle%3DDrug%2520metabolism%2520by%2520tumors%253A%2520its%2520nature%252C%2520relevance%2520and%2520therapeutic%2520implications%26jtitle%3DExpert%2520Opin.%2520Drug%2520Metab.%2520Toxicol.%26date%3D2007%26volume%3D3%26spage%3D783%26epage%3D803%26doi%3D10.1517%2F17425255.3.6.783" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Trunzer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faller, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerlin, A.</span></span> <span> </span><span class="NLM_article-title">Metabolic soft spot identification and compound optimization in early discovery phases using metaSite and LC-MS/MS validation</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">329</span>– <span class="NLM_lpage">335</span>, <span class="refDoi"> DOI: 10.1021/jm8008663</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm8008663" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFChsLfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=329-335&author=M.+Trunzerauthor=B.+Fallerauthor=A.+Zimmerlin&title=Metabolic+soft+spot+identification+and+compound+optimization+in+early+discovery+phases+using+metaSite+and+LC-MS%2FMS+validation&doi=10.1021%2Fjm8008663"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolic Soft Spot Identification and Compound Optimization in Early Discovery Phases Using MetaSite and LC-MS/MS Validation</span></div><div class="casAuthors">Trunzer, Markus; Faller, Bernard; Zimmerlin, Alfred</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">329-335</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Metabolic stability is a key property to enable drugs to reach therapeutic concns.  Microsomal clearance assays are used to dial out labile compds. in early discovery phases.  However, because they do not provide any information on soft spots, the rational design of more stable compds. remains challenging.  A robust soft spot identification procedure combining in silico prediction ranking using MetaSite and mass-spectrometric confirmation is described.  MetaSite's first rank order predictions were exptl. confirmed for only about 55% of the compds.  For another 29% of the compds., the second (20%) or the third (9%) rank order predictions were detected.  This automatic and high-throughput reprioritization of a likely soft-spot increases the likelihood of working on the right soft spot from about 50% to more than 80%.  With this information, the structure-metab. relationships are likely to be understood faster and earlier in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoV4vzsZ7zQ_rVg90H21EOLACvtfcHk0li6ulXlH42R0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFChsLfJ&md5=8628722a0cd75b0c6497cc383bd4ec16</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fjm8008663&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8008663%26sid%3Dliteratum%253Aachs%26aulast%3DTrunzer%26aufirst%3DM.%26aulast%3DFaller%26aufirst%3DB.%26aulast%3DZimmerlin%26aufirst%3DA.%26atitle%3DMetabolic%2520soft%2520spot%2520identification%2520and%2520compound%2520optimization%2520in%2520early%2520discovery%2520phases%2520using%2520metaSite%2520and%2520LC-MS%252FMS%2520validation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D329%26epage%3D335%26doi%3D10.1021%2Fjm8008663" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hidalgo, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raub, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borchardt, R. T.</span></span> <span> </span><span class="NLM_article-title">Characterization of the human colon carcinoma cell line (Caco-2) as a model system for intestinal epithelial permeability</span>. <i>Gastroenterology</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>96</i></span>,  <span class="NLM_fpage">736</span>– <span class="NLM_lpage">749</span>, <span class="refDoi"> DOI: 10.1016/0016-5085(89)90897-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1016%2FS0016-5085%2889%2980072-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=2914637" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A280%3ADyaL1M7itFeqsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=1989&pages=736-749&author=I.+J.+Hidalgoauthor=T.+J.+Raubauthor=R.+T.+Borchardt&title=Characterization+of+the+human+colon+carcinoma+cell+line+%28Caco-2%29+as+a+model+system+for+intestinal+epithelial+permeability&doi=10.1016%2F0016-5085%2889%2990897-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of the human colon carcinoma cell line (Caco-2) as a model system for intestinal epithelial permeability</span></div><div class="casAuthors">Hidalgo I J; Raub T J; Borchardt R T</div><div class="citationInfo"><span class="NLM_cas:title">Gastroenterology</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">96</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">736-49</span>
        ISSN:<span class="NLM_cas:issn">0016-5085</span>.
    </div><div class="casAbstract">Caco-2 cells develop morphologic characteristics of normal enterocytes when grown on plastic dishes or nitrocellulose filters.  The purpose of this study was to determine whether Caco-2 cells undergo similar differentiation when grown on Transwell polycarbonate membranes, and to study the suitability of Caco-2 monolayers as an intestinal epithelial transport model system.  Transepithelial electrical resistance values after confluence were 173.5 omega.cm2 and remained unchanged through day 17.  Permeabilities to the water-soluble fluid-phase markers that do not permeate the membrane, Lucifer yellow CH, [14C]inulin, [14C]polyethylene glycol, and [3H] dextran were less than 0.25% of the administered amount per hour after day 10.  Qualitative evaluation of uptake and permeability to horseradish peroxidase confirmed the similarity in uptake and barrier properties between this cell system and the small intestinal epithelial layer.  We conclude that Caco-2 cells grown on collagen-coated polycarbonate membranes should represent a valuable transport model system for the small intestinal epithelium.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSOn97989rzEIqdXsLcGTEUfW6udTcc2ealAXp8arI52rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaL1M7itFeqsg%253D%253D&md5=59ffccc1dd4dccb01400c05df95ac1ff</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2FS0016-5085%2889%2980072-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0016-5085%252889%252980072-1%26sid%3Dliteratum%253Aachs%26aulast%3DHidalgo%26aufirst%3DI.%2BJ.%26aulast%3DRaub%26aufirst%3DT.%2BJ.%26aulast%3DBorchardt%26aufirst%3DR.%2BT.%26atitle%3DCharacterization%2520of%2520the%2520human%2520colon%2520carcinoma%2520cell%2520line%2520%2528Caco-2%2529%2520as%2520a%2520model%2520system%2520for%2520intestinal%2520epithelial%2520permeability%26jtitle%3DGastroenterology%26date%3D1989%26volume%3D96%26spage%3D736%26epage%3D749%26doi%3D10.1016%2F0016-5085%2889%2990897-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buur, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mork, N.</span></span> <span> </span><span class="NLM_article-title">Metabolism of testosterone during in vitro transport across Caco-2 cell monolayers: evidence for β-hydroxysteroid dehydrogenase activity in differentiated Caco-2 cells</span>. <i>Pharm. Res.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1290</span>– <span class="NLM_lpage">1294</span>, <span class="refDoi"> DOI: 10.1023/A:1015805317375</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1023%2FA%3A1015805317375" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1448428" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADyaK38Xmt1KmtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1992&pages=1290-1294&author=A.+Buurauthor=N.+Mork&title=Metabolism+of+testosterone+during+in+vitro+transport+across+Caco-2+cell+monolayers%3A+evidence+for+%CE%B2-hydroxysteroid+dehydrogenase+activity+in+differentiated+Caco-2+cells&doi=10.1023%2FA%3A1015805317375"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolism of testosterone during in vitro transport across CACO-2 cell monolayers:  evidence for β-hydroxysteroid dehydrogenase activity in differentiated CACO-2 cells</span></div><div class="casAuthors">Buur, Anders; Moerk, Niels</div><div class="citationInfo"><span class="NLM_cas:title">Pharmaceutical Research</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1290-4</span>CODEN:
                <span class="NLM_cas:coden">PHREEB</span>;
        ISSN:<span class="NLM_cas:issn">0724-8741</span>.
    </div><div class="casAbstract">Testosterone has previously been used as a model compd. for the detn. of unstirred water layer thickness in the CACO-2 transport model.  The authors have found, however, that testosterone is metabolized during in vitro transport across the CACO-2 cell monolayers.  This suggests that testosterone is not an ideal model substance.  Testosterone is metabolized to androstenedione, indicating the presence of 17β-hydroxy steroid dehydrogenase in differentiated CACO-2 cells.  No reverse metab. is obsd., so androstenedione is considered superior to testosterone for detn. of unstirred water layer thickness in the CACO-2 system.  Permeability coeffs. for testosterone and androstenedione obtained under identical transport conditions were 66 × 10-6 and 84 × 10-6 cm/s, resp.  The unstirred water layer thicknesses at different agitation rates are detd. for the CACO-2 transport model used in the lab., utilizing androstenedione as a model compd.  The system is capable of controlling the water layer thickness from about 200 to 1000 μm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqimXFucsn9F7Vg90H21EOLACvtfcHk0ljvmFwWoy4yzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38Xmt1KmtL4%253D&md5=2f33efcf5c8b388b5d2046fe98da8460</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1023%2FA%3A1015805317375&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1015805317375%26sid%3Dliteratum%253Aachs%26aulast%3DBuur%26aufirst%3DA.%26aulast%3DMork%26aufirst%3DN.%26atitle%3DMetabolism%2520of%2520testosterone%2520during%2520in%2520vitro%2520transport%2520across%2520Caco-2%2520cell%2520monolayers%253A%2520evidence%2520for%2520%25CE%25B2-hydroxysteroid%2520dehydrogenase%2520activity%2520in%2520differentiated%2520Caco-2%2520cells%26jtitle%3DPharm.%2520Res.%26date%3D1992%26volume%3D9%26spage%3D1290%26epage%3D1294%26doi%3D10.1023%2FA%3A1015805317375" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berezhkovskiy, L. M.</span></span> <span> </span><span class="NLM_article-title">Some features of the kinetics and equilibrium of drug binding to plasma proteins</span>. <i>Expert Opin. Drug Metab. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">1479</span>– <span class="NLM_lpage">1498</span>, <span class="refDoi"> DOI: 10.1517/17425250802503808</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1517%2F17425250802503808" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=19040325" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVent7zI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=1479-1498&author=L.+M.+Berezhkovskiy&title=Some+features+of+the+kinetics+and+equilibrium+of+drug+binding+to+plasma+proteins&doi=10.1517%2F17425250802503808"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Some features of the kinetics and equilibrium of drug binding to plasma proteins</span></div><div class="casAuthors">Berezhkovskiy, Leonid M.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Metabolism & Toxicology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1479-1498</span>CODEN:
                <span class="NLM_cas:coden">EODMAP</span>;
        ISSN:<span class="NLM_cas:issn">1742-5255</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  The reaction of drug-protein binding is considered based on the common calcns. of chem. equil. (using the mass action and mass balance laws) and formal chem. kinetics.  The calculative and theor. aspects presented in the review are pertinent to routine drug development.  Numerous real-life examples are provided throughout the article to illustrate the practical utility of the presented concepts.  This may be very helpful in the interpretation of protein binding and pharmacokinetic data.  Considerable resources may be saved by the proper setting of protein binding expts., using relatively simple calcns. and estns. instead of doing exptl. measurements, and also avoiding improvements' that are destined to failure.  The presented material may be also useful for the simulations of pharmacokinetics and pharmacodynamics, which attempt the complete account of drug-protein binding.  A complete description of the considered topics is given in the last paragraph of the Introduction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1Bltn8ibfTbVg90H21EOLACvtfcHk0ljvmFwWoy4yzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVent7zI&md5=f9098016ae1af78e01546ff0ce3648ef</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1517%2F17425250802503808&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17425250802503808%26sid%3Dliteratum%253Aachs%26aulast%3DBerezhkovskiy%26aufirst%3DL.%2BM.%26atitle%3DSome%2520features%2520of%2520the%2520kinetics%2520and%2520equilibrium%2520of%2520drug%2520binding%2520to%2520plasma%2520proteins%26jtitle%3DExpert%2520Opin.%2520Drug%2520Metab.%2520Toxicol.%26date%3D2008%26volume%3D4%26spage%3D1479%26epage%3D1498%26doi%3D10.1517%2F17425250802503808" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zimmermann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmermann-Kogadeeva, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wegmann, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodman, A. L.</span></span> <span> </span><span class="NLM_article-title">Mapping human microbiome drug metabolism by gut bacteria and their genes</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>570</i></span>,  <span class="NLM_fpage">462</span>– <span class="NLM_lpage">467</span>, <span class="refDoi"> DOI: 10.1038/s41586-019-1291-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1038%2Fs41586-019-1291-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=31158845" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1aqtbjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=570&publication_year=2019&pages=462-467&author=M.+Zimmermannauthor=M.+Zimmermann-Kogadeevaauthor=R.+Wegmannauthor=A.+L.+Goodman&title=Mapping+human+microbiome+drug+metabolism+by+gut+bacteria+and+their+genes&doi=10.1038%2Fs41586-019-1291-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Mapping human microbiome drug metabolism by gut bacteria and their genes</span></div><div class="casAuthors">Zimmermann, Michael; Zimmermann-Kogadeeva, Maria; Wegmann, Rebekka; Goodman, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">570</span>
        (<span class="NLM_cas:issue">7762</span>),
    <span class="NLM_cas:pages">462-467</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Individuals vary widely in their responses to medicinal drugs, which can be dangerous and expensive owing to treatment delays and adverse effects.  Although increasing evidence implicates the gut microbiome in this variability, the mol. mechanisms involved remain largely unknown.  Here we show, by measuring the ability of 76 human gut bacteria from diverse clades to metabolize 271 orally administered drugs, that many drugs are chem. modified by microorganisms.  We combined high-throughput genetic analyses with mass spectrometry to systematically identify microbial gene products that metabolize drugs.  These microbiome-encoded enzymes can directly and substantially affect intestinal and systemic drug metab. in mice, and can explain the drug-metabolizing activities of human gut bacteria and communities on the basis of their genomic contents.  These causal links between the gene content and metabolic activities of the microbiota connect interpersonal variability in microbiomes to interpersonal differences in drug metab., which has implications for medical therapy and drug development across multiple disease indications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNgThCVG-l2bVg90H21EOLACvtfcHk0ljvmFwWoy4yzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1aqtbjN&md5=fc92271e1b2b7bad9c2396ac9cf7517e</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1038%2Fs41586-019-1291-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-019-1291-3%26sid%3Dliteratum%253Aachs%26aulast%3DZimmermann%26aufirst%3DM.%26aulast%3DZimmermann-Kogadeeva%26aufirst%3DM.%26aulast%3DWegmann%26aufirst%3DR.%26aulast%3DGoodman%26aufirst%3DA.%2BL.%26atitle%3DMapping%2520human%2520microbiome%2520drug%2520metabolism%2520by%2520gut%2520bacteria%2520and%2520their%2520genes%26jtitle%3DNature%26date%3D2019%26volume%3D570%26spage%3D462%26epage%3D467%26doi%3D10.1038%2Fs41586-019-1291-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Škarydová, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wsól, V.</span></span> <span> </span><span class="NLM_article-title">Human microsomal carbonyl reducing enzymes in the metabolism of xenobiotics: well-known and promising members of the SDR superfamily</span>. <i>Drug Metab. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">173</span>– <span class="NLM_lpage">191</span>, <span class="refDoi"> DOI: 10.3109/03602532.2011.638304</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.3109%2F03602532.2011.638304" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=22181347" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC38Xls1Whsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2012&pages=173-191&author=L.+%C5%A0karydov%C3%A1author=V.+Ws%C3%B3l&title=Human+microsomal+carbonyl+reducing+enzymes+in+the+metabolism+of+xenobiotics%3A+well-known+and+promising+members+of+the+SDR+superfamily&doi=10.3109%2F03602532.2011.638304"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Human microsomal carbonyl reducing enzymes in the metabolism of xenobiotics: well-known and promising members of the SDR superfamily</span></div><div class="casAuthors">Skarydova, Lucie; Wsol, Vladimir</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism Reviews</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">173-191</span>CODEN:
                <span class="NLM_cas:coden">DMTRAR</span>;
        ISSN:<span class="NLM_cas:issn">0360-2532</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  The best known, most widely studied enzyme system in phase I biotransformation is cytochrome P 450 (CYP), which participates in the metab. of roughly 9 of 10 drugs in use today.  The main biotransformation isoforms of CYP are assocd. with the membrane of the endoplasmatic reticulum (ER).  Other enzymes that are also active in phase I biotransformation are carbonyl reducing enzymes.  Much is known about the role of cytosolic forms of carbonyl reducing enzymes in the metab. of xenobiotics, but their microsomal forms have been mostly poorly studied.  The only well-known microsomal carbonyl reducing enzyme taking part in the biotransformation of xenobiotics is 11β-hydroxysteroid dehydrogenase 1, a member of the short-chain dehydrogenase/reductase superfamily.  Physiol. roles of microsomal carbonyl reducing enzymes are better known than their participation in the metab. of xenobiotics.  This review is a summary of the fragmentary information known about the roles of the microsomal forms.  Besides 11β-hydroxysteroid dehydrogenase 1, it has been reported, so far, that retinol dehydrogenase 12 participates only in the detoxification of unsatd. aldehydes formed upon oxidative stress.  Another promising group of microsomal biotransformation carbonyl reducing enzymes are some members of 17β-hydroxysteroid dehydrogenases.  Generally, it is clear that this area is, overall, quite unexplored, but carbonyl reducing enzymes located in the ER have proven very interesting.  The study of these enzymes could shed new light on the metab. of several clin. used drugs or they could become an important target in connection with some diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUT5DSEsK3C7Vg90H21EOLACvtfcHk0lhkQnGumX_Nfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xls1Whsb0%253D&md5=8265d72837c0fa0265fb7f686fa5c738</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.3109%2F03602532.2011.638304&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F03602532.2011.638304%26sid%3Dliteratum%253Aachs%26aulast%3D%25C5%25A0karydov%25C3%25A1%26aufirst%3DL.%26aulast%3DWs%25C3%25B3l%26aufirst%3DV.%26atitle%3DHuman%2520microsomal%2520carbonyl%2520reducing%2520enzymes%2520in%2520the%2520metabolism%2520of%2520xenobiotics%253A%2520well-known%2520and%2520promising%2520members%2520of%2520the%2520SDR%2520superfamily%26jtitle%3DDrug%2520Metab.%2520Rev.%26date%3D2012%26volume%3D44%26spage%3D173%26epage%3D191%26doi%3D10.3109%2F03602532.2011.638304" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adler, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhles, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stamm, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairhurst, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiffe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sterker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centeleghe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wartmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kinyamu-Akunda, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schadt, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couttet, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barzaghi-Rinaudo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murakami, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kauffmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knoepfel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buschmann, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leblanc, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mah, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furet, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blank, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graus Porta, D.</span></span> <span> </span><span class="NLM_article-title">FGF401, a first-in-class highly selective and potent FGFR4 inhibitor for the treatment of FGF19-driven hepatocellular cancer</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">2194</span>– <span class="NLM_lpage">2206</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-18-1291</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1158%2F1535-7163.MCT-18-1291" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=31409633" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BB3cXktFOit7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=2194-2206&author=A.+Weissauthor=F.+Adlerauthor=A.+Buhlesauthor=C.+Stammauthor=R.+A.+Fairhurstauthor=M.+Kiffeauthor=D.+Sterkerauthor=M.+Centelegheauthor=M.+Wartmannauthor=J.+Kinyamu-Akundaauthor=H.+S.+Schadtauthor=P.+Couttetauthor=A.+Wolfauthor=Y.+Wangauthor=P.+Barzaghi-Rinaudoauthor=M.+Murakamiauthor=A.+Kauffmannauthor=T.+Knoepfelauthor=N.+Buschmannauthor=C.+Leblancauthor=R.+Mahauthor=P.+Furetauthor=J.+Blankauthor=F.+Hofmannauthor=W.+R.+Sellersauthor=D.+Graus+Porta&title=FGF401%2C+a+first-in-class+highly+selective+and+potent+FGFR4+inhibitor+for+the+treatment+of+FGF19-driven+hepatocellular+cancer&doi=10.1158%2F1535-7163.MCT-18-1291"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">FGF401, a first-in-class highly selective and potent FGFR4 inhibitor for the treatment of FGF19-driven hepatocellular cancer</span></div><div class="casAuthors">Weiss, Andreas; Adler, Flavia; Buhles, Alexandra; Stamm, Christelle; Fairhurst, Robin A.; Kiffe, Michael; Sterker, Dario; Centeleghe, Mario; Wartmann, Markus; Kinyamu-Akunda, Jacqueline; Schadt, Heiko S.; Couttet, Philippe; Wolf, Armin; Wang, Youzhen; Barzaghi-Rinaudo, Patrizia; Murakami, Masato; Kauffmann, Audrey; Knoepfel, Thomas; Buschmann, Nicole; Leblanc, Catherine; Mah, Robert; Furet, Pascal; Blank, Jutta; Hofmann, Francesco; Sellers, William R.; Porta, Diana Graus</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2194-2206</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Hepatocellular carcinoma (HCC) is the most common primary malignancy of the liver and it is the third leading cause of cancer-related deaths worldwide.  Recently, aberrant signaling through the FGF19/FGFR4 axis has been implicated in HCC.  Here, we describe the development of FGF401, a highly potent and selective, first in class, reversible-covalent small-mol. inhibitor of the kinase activity of FGFR4.  FGF401 is exquisitely selective for FGFR4 vs. the other FGFR paralogues FGFR1, FGFR2, FGFR3, and all other kinases in the kinome.  FGF401 has excellent drug-like properties showing a robust pharmacokinetic/pharmacodynamics/efficacy relationship, driven by a fraction of time above the phospho-FGFR4 IC90 value.  FGF401 has remarkable antitumor activity in mice bearing HCC tumor xenografts and patient-derived xenograft models that are pos. for FGF19, FGFR4, and KLB.  FGF401 is the first FGFR4 inhibitor to enter clin. trials, and a phase I/II study is currently ongoing in HCC and other solid malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBspWk4KWyurVg90H21EOLACvtfcHk0lhkQnGumX_Nfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXktFOit7Y%253D&md5=81a3b04d779295a9c9a037cf5ca73fca</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-18-1291&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-18-1291%26sid%3Dliteratum%253Aachs%26aulast%3DWeiss%26aufirst%3DA.%26aulast%3DAdler%26aufirst%3DF.%26aulast%3DBuhles%26aufirst%3DA.%26aulast%3DStamm%26aufirst%3DC.%26aulast%3DFairhurst%26aufirst%3DR.%2BA.%26aulast%3DKiffe%26aufirst%3DM.%26aulast%3DSterker%26aufirst%3DD.%26aulast%3DCenteleghe%26aufirst%3DM.%26aulast%3DWartmann%26aufirst%3DM.%26aulast%3DKinyamu-Akunda%26aufirst%3DJ.%26aulast%3DSchadt%26aufirst%3DH.%2BS.%26aulast%3DCouttet%26aufirst%3DP.%26aulast%3DWolf%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DBarzaghi-Rinaudo%26aufirst%3DP.%26aulast%3DMurakami%26aufirst%3DM.%26aulast%3DKauffmann%26aufirst%3DA.%26aulast%3DKnoepfel%26aufirst%3DT.%26aulast%3DBuschmann%26aufirst%3DN.%26aulast%3DLeblanc%26aufirst%3DC.%26aulast%3DMah%26aufirst%3DR.%26aulast%3DFuret%26aufirst%3DP.%26aulast%3DBlank%26aufirst%3DJ.%26aulast%3DHofmann%26aufirst%3DF.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26aulast%3DGraus%2BPorta%26aufirst%3DD.%26atitle%3DFGF401%252C%2520a%2520first-in-class%2520highly%2520selective%2520and%2520potent%2520FGFR4%2520inhibitor%2520for%2520the%2520treatment%2520of%2520FGF19-driven%2520hepatocellular%2520cancer%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2019%26volume%3D18%26spage%3D2194%26epage%3D2206%26doi%3D10.1158%2F1535-7163.MCT-18-1291" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arnott, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lobo Planey, S.</span></span> <span> </span><span class="NLM_article-title">The influence of lipophilicity in drug discovery and design</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">863</span>– <span class="NLM_lpage">875</span>, <span class="refDoi"> DOI: 10.1517/17460441.2012.714363</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1517%2F17460441.2012.714363" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=22992175" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhtl2lsbvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=863-875&author=J.+A.+Arnottauthor=S.+Lobo+Planey&title=The+influence+of+lipophilicity+in+drug+discovery+and+design&doi=10.1517%2F17460441.2012.714363"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">The influence of lipophilicity in drug discovery and design</span></div><div class="casAuthors">Arnott, John A.; Planey, Sonia Lobo</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">863-875</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Introduction: The role of lipophilicity in drug discovery and design is a crit. one.  Lipophilicity is a key physicochem. property that plays a crucial role in detg. ADMET (absorption, distribution, metab., excretion, and toxicity) properties and the overall suitability of drug candidates.  There is increasing evidence to suggest that control of physicochem. properties such as lipophilicity, within a defined optimal range, can improve compd. quality and the likelihood of therapeutic success.Areas covered: This review focuses on understanding lipophilicity, techniques used to measure lipophilicity, and summarizes the importance of lipophilicity in drug discovery and development, including a discussion of its impact on individual ADMET parameters as well as its overall influence on the drug discovery and design process, specifically within the past 15 years.Expert opinion: A current review of the literature reveals a continued reliance on the synthesis of novel structures with increased potency, rather than a focus on maintaining optimal physicochem. properties assocd. with ADMET throughout drug optimization.  Particular attention to the optimum region of lipophilicity, as well as monitoring of lipophilic efficiency indexes, may contribute significantly to the overall quality of candidate drugs at different stages of discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolWJgvv9yK87Vg90H21EOLACvtfcHk0lgMJBKtXQ3rpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhtl2lsbvJ&md5=4dc5d7268216c9a59fcbd841fbf66735</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1517%2F17460441.2012.714363&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.2012.714363%26sid%3Dliteratum%253Aachs%26aulast%3DArnott%26aufirst%3DJ.%2BA.%26aulast%3DLobo%2BPlaney%26aufirst%3DS.%26atitle%3DThe%2520influence%2520of%2520lipophilicity%2520in%2520drug%2520discovery%2520and%2520design%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2012%26volume%3D7%26spage%3D863%26epage%3D875%26doi%3D10.1517%2F17460441.2012.714363" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yalkowsky, S. H.</span></span> <span> </span><span class="NLM_article-title">An interesting relationship between drug absorption and melting point</span>. <i>Int. J. Pharm.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>373</i></span>,  <span class="NLM_fpage">24</span>– <span class="NLM_lpage">40</span>, <span class="refDoi"> DOI: 10.1016/j.ijpharm.2009.01.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1016%2Fj.ijpharm.2009.01.026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=19429285" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BD1MXltlSjtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=373&publication_year=2009&pages=24-40&author=K.+A.+Chuauthor=S.+H.+Yalkowsky&title=An+interesting+relationship+between+drug+absorption+and+melting+point&doi=10.1016%2Fj.ijpharm.2009.01.026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54aR"><div class="casContent"><span class="casTitleNuber">54a</span><div class="casTitle"><span class="NLM_cas:atitle">An interesting relationship between drug absorption and melting point</span></div><div class="casAuthors">Chu, Katherine A.; Yalkowsky, Samuel H.</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Pharmaceutics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">373</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">24-40</span>CODEN:
                <span class="NLM_cas:coden">IJPHDE</span>;
        ISSN:<span class="NLM_cas:issn">0378-5173</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The ability to predict the extent of passive intestinal drug absorption is very important for efficient lead candidate selection and development.  Physicochem.-based absorption predictive models previously developed use soly., partition coeff. and pKa as drug input parameters for intestinal absorption.  Alternatively, this study looks at the relationship between m.p. and passive transport for poorly sol. drugs.  It is based entirely on the expression derived from the general soly. equation (GSE) that relates m.p. to the product of intrinsic soly. and partition coeff.  Given that the m.p. of a compd. is one of the first and more reliable phys. properties measured, it can be advantageously used as a guide in early drug discovery and development.  This paper elucidates the interesting relationship between the m.p. and dose to the fraction absorbed of poorly sol. drugs, i.e., class II and IV compds. in the biopharmaceutics classification system.  The newly defined m.p. based absorption potential (MPbAP) parameter is successful at distinguishing 90% of the 91 drugs considered being well absorbed (FA > 0.5) or poorly absorbed.  In general, lower melting compds. are more likely to be well absorbed than higher melting compds. for any given dose.  The fraction absorbed for drugs with high melting temps. is limited by the dose to a greater degree than it is for low melting compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLyfNgwnwHI7Vg90H21EOLACvtfcHk0lgMJBKtXQ3rpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXltlSjtrc%253D&md5=6ce9ff030bfc5c5c20c630ee18e502c3</span></div><a href="/servlet/linkout?suffix=cit54a&amp;dbid=16384&amp;doi=10.1016%2Fj.ijpharm.2009.01.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ijpharm.2009.01.026%26sid%3Dliteratum%253Aachs%26aulast%3DChu%26aufirst%3DK.%2BA.%26aulast%3DYalkowsky%26aufirst%3DS.%2BH.%26atitle%3DAn%2520interesting%2520relationship%2520between%2520drug%2520absorption%2520and%2520melting%2520point%26jtitle%3DInt.%2520J.%2520Pharm.%26date%3D2009%26volume%3D373%26spage%3D24%26epage%3D40%26doi%3D10.1016%2Fj.ijpharm.2009.01.026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit54b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, V. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhattachar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hitchingham, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zocharski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naath, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Surendran, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoner, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Kattan, A.</span></span> <span> </span><span class="NLM_article-title">The road map to oral bioavailability: an industrial perspective</span>. <i>Expert Opin. Drug Metab. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">591</span>– <span class="NLM_lpage">608</span>, <span class="refDoi"> DOI: 10.1517/17425255.2.4.591</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1517%2F17425255.2.4.591" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=16859407" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BD28Xnt1ahsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2006&pages=591-608&author=V.+H.+Thomasauthor=S.+Bhattacharauthor=L.+Hitchinghamauthor=P.+Zocharskiauthor=M.+Naathauthor=N.+Surendranauthor=C.+L.+Stonerauthor=A.+El-Kattan&title=The+road+map+to+oral+bioavailability%3A+an+industrial+perspective&doi=10.1517%2F17425255.2.4.591"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54bR"><div class="casContent"><span class="casTitleNuber">54b</span><div class="casTitle"><span class="NLM_cas:atitle">The road map to oral bioavailability: an industrial perspective</span></div><div class="casAuthors">Thomas, V. Hayden; Bhattachar, Shobha; Hitchingham, Linda; Zocharski, Philip; Naath, Maryanne; Surendran, Narayanan; Stoner, Chad L.; El-Kattan, Ayman</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Metabolism & Toxicology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">591-608</span>CODEN:
                <span class="NLM_cas:coden">EODMAP</span>;
        ISSN:<span class="NLM_cas:issn">1742-5255</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Optimization of oral bioavailability is a continuing challenge for the pharmaceutical and biotechnol. industries.  The no. of potential drug candidates requiring in vivo evaluation has significantly increased with the advent of combinatorial chem.  In addn., drug discovery programs are increasingly forced into more lipophilic and lower soly. chem. space.  To aid in the use of in vitro and in silico tools as well as reduce the no. of in vivo studies required, a team-based discussion tool is proposed that provides a road map' to guide the selection of profiling assays that should be considered when optimizing oral bioavailability.  This road map divides the factors that contribute to poor oral bioavailability into two interrelated categories: absorption and metab.  This road map provides an interface for cross discipline discussions and a systematic approach to the experimentation that drives the drug discovery process towards a common goal - acceptable oral bioavailability using minimal resources in an acceptable time frame.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrh7_zZKacXE7Vg90H21EOLACvtfcHk0lgMJBKtXQ3rpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xnt1ahsLs%253D&md5=79cf4cea1b89914bd3a8826b5d7360f9</span></div><a href="/servlet/linkout?suffix=cit54b&amp;dbid=16384&amp;doi=10.1517%2F17425255.2.4.591&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17425255.2.4.591%26sid%3Dliteratum%253Aachs%26aulast%3DThomas%26aufirst%3DV.%2BH.%26aulast%3DBhattachar%26aufirst%3DS.%26aulast%3DHitchingham%26aufirst%3DL.%26aulast%3DZocharski%26aufirst%3DP.%26aulast%3DNaath%26aufirst%3DM.%26aulast%3DSurendran%26aufirst%3DN.%26aulast%3DStoner%26aufirst%3DC.%2BL.%26aulast%3DEl-Kattan%26aufirst%3DA.%26atitle%3DThe%2520road%2520map%2520to%2520oral%2520bioavailability%253A%2520an%2520industrial%2520perspective%26jtitle%3DExpert%2520Opin.%2520Drug%2520Metab.%2520Toxicol.%26date%3D2006%26volume%3D2%26spage%3D591%26epage%3D608%26doi%3D10.1517%2F17425255.2.4.591" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><a href="https://www.discoverx.com/services/drug-discovery-development-services/kinase-profiling/kinomescan" class="extLink">https://www.discoverx.com/services/drug-discovery-development-services/kinase-profiling/kinomescan</a> (accessed Sep 14, <span class="NLM_year">2020</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=https%3A%2F%2Fwww.discoverx.com%2Fservices%2Fdrug-discovery-development-services%2Fkinase-profiling%2Fkinomescan+%28accessed+Sep+14%2C+2020%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Veber, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, H.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kopple, K. D.</span></span> <span> </span><span class="NLM_article-title">Molecular properties that influence the oral bioavailability of drug candidates</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">2615</span>– <span class="NLM_lpage">2623</span>, <span class="refDoi"> DOI: 10.1021/jm020017n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm020017n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BD38XjsFCmt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=2615-2623&author=D.+F.+Veberauthor=S.+R.+Johnsonauthor=H.-Y.+Chengauthor=B.+R.+Smithauthor=K.+W.+Wardauthor=K.+D.+Kopple&title=Molecular+properties+that+influence+the+oral+bioavailability+of+drug+candidates&doi=10.1021%2Fjm020017n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Properties That Influence the Oral Bioavailability of Drug Candidates</span></div><div class="casAuthors">Veber, Daniel F.; Johnson, Stephen R.; Cheng, Hung-Yuan; Smith, Brian R.; Ward, Keith W.; Kopple, Kenneth D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2615-2623</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Oral bioavailability measurements in rats for over 1100 drug candidates studied at Smith-Kline Beecham Pharmaceuticals (now Glaxo Smith-Kline) have allowed us to analyze the relative importance of mol. properties considered to influence that drug property.  Reduced mol. flexibility, as measured by the no. of rotatable bonds, and low polar surface area or total hydrogen bond count (sum of donors and acceptors) are found to be important predictors of good oral bioavailability, independent of mol. wt.  That on av. both the no. of rotatable bonds and polar surface area or hydrogen bond count tend to increase with mol. wt. may in part explain the success of the mol. wt. parameter in predicting oral bioavailability.  The commonly applied mol. wt. cutoff at 500 does not itself significantly sep. compds. with poor oral bioavailability from those with acceptable values in this extensive data set.  Our observations suggest that compds. which meet only the 2 criteria of (1) 10 or fewer rotatable bonds and (2) polar surface area ≤140 Å2 (or 12 or fewer H-bond donors and acceptors) will have a high probability of good oral bioavailability in the rat.  Data sets for the artificial membrane permeation rate and for clearance in the rat were also examd.  Reduced polar surface area correlates better with increased permeation rate than does lipophilicity (C log P), and increased rotatable bond count has a neg. effect on the permeation rate.  A threshold permeation rate is a prerequisite of oral bioavailability.  The rotatable bond count does not correlate with the data examd. here for the in vivo clearance rate in the rat.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3JtJEkO7yVLVg90H21EOLACvtfcHk0liAzxAHvJrAFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjsFCmt7g%253D&md5=eaad26ed6a259de82ad65a8834fc397d</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Fjm020017n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm020017n%26sid%3Dliteratum%253Aachs%26aulast%3DVeber%26aufirst%3DD.%2BF.%26aulast%3DJohnson%26aufirst%3DS.%2BR.%26aulast%3DCheng%26aufirst%3DH.-Y.%26aulast%3DSmith%26aufirst%3DB.%2BR.%26aulast%3DWard%26aufirst%3DK.%2BW.%26aulast%3DKopple%26aufirst%3DK.%2BD.%26atitle%3DMolecular%2520properties%2520that%2520influence%2520the%2520oral%2520bioavailability%2520of%2520drug%2520candidates%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D2615%26epage%3D2623%26doi%3D10.1021%2Fjm020017n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Docherty, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pencheva, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abramov, Y. A.</span></span> <span> </span><span class="NLM_article-title">Low solubility in drug development: de-convoluting the relative importance of solvation and crystal packing</span>. <i>J. Pharm. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">847</span>– <span class="NLM_lpage">856</span>, <span class="refDoi"> DOI: 10.1111/jphp.12393</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1111%2Fjphp.12393" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=25880016" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmsFersLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2015&pages=847-856&author=R.+Dochertyauthor=K.+Penchevaauthor=Y.+A.+Abramov&title=Low+solubility+in+drug+development%3A+de-convoluting+the+relative+importance+of+solvation+and+crystal+packing&doi=10.1111%2Fjphp.12393"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57aR"><div class="casContent"><span class="casTitleNuber">57a</span><div class="casTitle"><span class="NLM_cas:atitle">Low solubility in drug development: de-convoluting the relative importance of solvation and crystal packing</span></div><div class="casAuthors">Docherty, Robert; Pencheva, Klimentina; Abramov, Yuriy A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacy and Pharmacology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">847-856</span>CODEN:
                <span class="NLM_cas:coden">JPPMAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3573</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">Objectives : An increasing trend towards low soly. is a major issue for drug development as formulation of low soly. compds. can be problematic.  This paper presents a model which de-convolutes the soly. of pharmaceutical compds. into solvation and packing properties with the intention to understand the soly. limiting features.  Methods : The Cambridge Crystallog. Database was the source of structural information.  Lattice energies were calcd. via force-field based approaches using Materials Studio.  The solvation energies were calcd. applying quantum chem. models using Cosmotherm software.  Key findings : The solubilities of 54 drug-like compds. were mapped onto a solvation energy/crystal packing grid.  Four quadrants were identified were different balances of solvation and packing were defining the soly.  A version of the model was developed which allows for the calcn. of the two features even in absence of crystal structure.  Conclusion : Although there are significant no. of in-silico models, it has been proven very difficult to predict aq. soly. accurately.  Therefore, we have taken a different approach where the soly. is not predicted directly but is de-convoluted into two constituent features.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSkolyShKsU7Vg90H21EOLACvtfcHk0liAzxAHvJrAFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmsFersLg%253D&md5=a74b378d2f71167a8af1d1c142c92c53</span></div><a href="/servlet/linkout?suffix=cit57a&amp;dbid=16384&amp;doi=10.1111%2Fjphp.12393&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fjphp.12393%26sid%3Dliteratum%253Aachs%26aulast%3DDocherty%26aufirst%3DR.%26aulast%3DPencheva%26aufirst%3DK.%26aulast%3DAbramov%26aufirst%3DY.%2BA.%26atitle%3DLow%2520solubility%2520in%2520drug%2520development%253A%2520de-convoluting%2520the%2520relative%2520importance%2520of%2520solvation%2520and%2520crystal%2520packing%26jtitle%3DJ.%2520Pharm.%2520Pharmacol.%26date%3D2015%26volume%3D67%26spage%3D847%26epage%3D856%26doi%3D10.1111%2Fjphp.12393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit57b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Lovering, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bikker, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humblet, C.</span></span> <span> </span><span class="NLM_article-title">Escape from flatland: increasing saturation as an approach to improving clinical success</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">6752</span>– <span class="NLM_lpage">6756</span>, <span class="refDoi"> DOI: 10.1021/jm901241e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901241e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1KjtLvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=6752-6756&author=F.+Loveringauthor=J.+Bikkerauthor=C.+Humblet&title=Escape+from+flatland%3A+increasing+saturation+as+an+approach+to+improving+clinical+success&doi=10.1021%2Fjm901241e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57bR"><div class="casContent"><span class="casTitleNuber">57b</span><div class="casTitle"><span class="NLM_cas:atitle">Escape from Flatland: Increasing Saturation as an Approach to Improving Clinical Success</span></div><div class="casAuthors">Lovering, Frank; Bikker, Jack; Humblet, Christine</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6752-6756</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The medicinal chem. community has become increasingly aware of the value of tracking calcd. phys. properties such as mol. wt., topol. polar surface area, rotatable bonds, and hydrogen bond donors and acceptors.  The authors hypothesized that the shift to high-throughput synthetic practices over the past decade may be another factor that may predispose mols. to fail by steering discovery efforts toward achiral, arom. compds.  The authors have proposed two simple and interpretable measures of the complexity of mols. prepd. as potential drug candidates.  The first is carbon bond satn. as defined by fraction Sp3 (Fsp3) where Fsp3 = (no. of Sp3 hybridized carbons/total carbon count).  The second is simply whether a chiral carbon exists in the mol.  The authors demonstrate that both complexity (as measured by Fsp3) and the presence of chiral centers correlate with success as compds. transition from discovery, through clin. testing, to drugs.  To explain these observations, the authors further demonstrate that satn. correlates with soly., an exptl. phys. property important to success in the drug discovery setting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoC1f3Ed7nUBbVg90H21EOLACvtfcHk0liAzxAHvJrAFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1KjtLvN&md5=4ca92c30c17c53d77ad376719bad951e</span></div><a href="/servlet/linkout?suffix=cit57b&amp;dbid=16384&amp;doi=10.1021%2Fjm901241e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901241e%26sid%3Dliteratum%253Aachs%26aulast%3DLovering%26aufirst%3DF.%26aulast%3DBikker%26aufirst%3DJ.%26aulast%3DHumblet%26aufirst%3DC.%26atitle%3DEscape%2520from%2520flatland%253A%2520increasing%2520saturation%2520as%2520an%2520approach%2520to%2520improving%2520clinical%2520success%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D6752%26epage%3D6756%26doi%3D10.1021%2Fjm901241e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit57c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Wassvik, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmén, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Draheim, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artursson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergström, C. A. S.</span></span> <span> </span><span class="NLM_article-title">Molecular characteristics for solid-state limited solubility</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">3035</span>– <span class="NLM_lpage">3039</span>, <span class="refDoi"> DOI: 10.1021/jm701587d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm701587d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BD1cXksVSgt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=3035-3039&author=C.+M.+Wassvikauthor=A.+G.+Holm%C3%A9nauthor=R.+Draheimauthor=P.+Arturssonauthor=C.+A.+S.+Bergstr%C3%B6m&title=Molecular+characteristics+for+solid-state+limited+solubility&doi=10.1021%2Fjm701587d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57cR"><div class="casContent"><span class="casTitleNuber">57c</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Characteristics for Solid-State Limited Solubility</span></div><div class="casAuthors">Wassvik, Carola M.; Holmen, Anders G.; Draheim, Rieke; Artursson, Per; Bergstroem, Christel A. S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3035-3039</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Soly. and solid-state characteristics were detd. and multivariate data anal. was used to deduce structural features important for solid-state limited soly. of marketed drugs.  Mols. with extended ring structures and large conjugated systems were less sol., indicating that structural features related to rigidity and aromaticity result in soly. restricted by stable crystal structures.  These descriptors successfully predicted the applied test set and can be useful for avoiding synthesis of compds. behaving like "brick dust".</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4YD2_Ul4qabVg90H21EOLACvtfcHk0liAzxAHvJrAFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXksVSgt7o%253D&md5=f27982a173dfbc1491b6ce1749f486a8</span></div><a href="/servlet/linkout?suffix=cit57c&amp;dbid=16384&amp;doi=10.1021%2Fjm701587d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701587d%26sid%3Dliteratum%253Aachs%26aulast%3DWassvik%26aufirst%3DC.%2BM.%26aulast%3DHolm%25C3%25A9n%26aufirst%3DA.%2BG.%26aulast%3DDraheim%26aufirst%3DR.%26aulast%3DArtursson%26aufirst%3DP.%26aulast%3DBergstr%25C3%25B6m%26aufirst%3DC.%2BA.%2BS.%26atitle%3DMolecular%2520characteristics%2520for%2520solid-state%2520limited%2520solubility%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D3035%26epage%3D3039%26doi%3D10.1021%2Fjm701587d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tayl, J. G.,  VI</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheuk, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsang, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, J.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Youngblood, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeung, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helman, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendoza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ngo, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staudt, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khanna, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Catchpoole, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qualman, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merlino, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chanock, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, J.</span></span> <span> </span><span class="NLM_article-title">Identification of FGFR4-activating mutations in human rhabdomyosarcomas that promote metastasis in xenotransplanted models</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">3395</span>– <span class="NLM_lpage">3407</span>, <span class="refDoi"> DOI: 10.1172/JCI39703</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1172%2FJCI39703" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=19809159" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2009&pages=3395-3407&author=J.+G.+Taylauthor=A.+T.+Cheukauthor=P.+S.+Tsangauthor=J.-Y.+Chungauthor=Y.+K.+Songauthor=K.+Desaiauthor=Y.+Yuauthor=Q.-R.+Chenauthor=K.+Shahauthor=V.+Youngbloodauthor=J.+Fangauthor=S.+Y.+Kimauthor=C.+Yeungauthor=L.+J.+Helmanauthor=A.+Mendozaauthor=V.+Ngoauthor=L.+M.+Staudtauthor=J.+S.+Weiauthor=C.+Khannaauthor=D.+Catchpooleauthor=S.+J.+Qualmanauthor=S.+M.+Hewittauthor=G.+Merlinoauthor=S.+J.+Chanockauthor=J.+Khan&title=Identification+of+FGFR4-activating+mutations+in+human+rhabdomyosarcomas+that+promote+metastasis+in+xenotransplanted+models&doi=10.1172%2FJCI39703"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1172%2FJCI39703&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI39703%26sid%3Dliteratum%253Aachs%26aulast%3DTayl%26aufirst%3DJ.%2BG.%26aulast%3DCheuk%26aufirst%3DA.%2BT.%26aulast%3DTsang%26aufirst%3DP.%2BS.%26aulast%3DChung%26aufirst%3DJ.-Y.%26aulast%3DSong%26aufirst%3DY.%2BK.%26aulast%3DDesai%26aufirst%3DK.%26aulast%3DYu%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DQ.-R.%26aulast%3DShah%26aufirst%3DK.%26aulast%3DYoungblood%26aufirst%3DV.%26aulast%3DFang%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DS.%2BY.%26aulast%3DYeung%26aufirst%3DC.%26aulast%3DHelman%26aufirst%3DL.%2BJ.%26aulast%3DMendoza%26aufirst%3DA.%26aulast%3DNgo%26aufirst%3DV.%26aulast%3DStaudt%26aufirst%3DL.%2BM.%26aulast%3DWei%26aufirst%3DJ.%2BS.%26aulast%3DKhanna%26aufirst%3DC.%26aulast%3DCatchpoole%26aufirst%3DD.%26aulast%3DQualman%26aufirst%3DS.%2BJ.%26aulast%3DHewitt%26aufirst%3DS.%2BM.%26aulast%3DMerlino%26aufirst%3DG.%26aulast%3DChanock%26aufirst%3DS.%2BJ.%26aulast%3DKhan%26aufirst%3DJ.%26atitle%3DIdentification%2520of%2520FGFR4-activating%2520mutations%2520in%2520human%2520rhabdomyosarcomas%2520that%2520promote%2520metastasis%2520in%2520xenotransplanted%2520models%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2009%26volume%3D119%26spage%3D3395%26epage%3D3407%26doi%3D10.1172%2FJCI39703" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hatlen, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt-Kittler, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherwin, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rozsahegyi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheets, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miduturu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bifulco, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooijmans, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boral, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lengauer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorsch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, Y.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, B. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoeflich, K. P.</span></span> <span> </span><span class="NLM_article-title">Acquired on-target clinical resistance validates FGFR4 as a driver of hepatocellular carcinoma</span>. <i>Cancer Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1686</span>– <span class="NLM_lpage">1695</span>, <span class="refDoi"> DOI: 10.1158/2159-8290.CD-19-0367</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1158%2F2159-8290.CD-19-0367" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=31575540" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1Ght77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=1686-1695&author=M.+A.+Hatlenauthor=O.+Schmidt-Kittlerauthor=C.+A.+Sherwinauthor=E.+Rozsahegyiauthor=N.+Rubinauthor=M.+P.+Sheetsauthor=J.+L.+Kimauthor=C.+Miduturuauthor=N.+Bifulcoauthor=N.+Brooijmansauthor=H.+Shiauthor=T.+Guziauthor=A.+Boralauthor=C.+Lengauerauthor=M.+Dorschauthor=R.+D.+Kimauthor=Y.-K.+Kangauthor=B.+B.+Wolfauthor=K.+P.+Hoeflich&title=Acquired+on-target+clinical+resistance+validates+FGFR4+as+a+driver+of+hepatocellular+carcinoma&doi=10.1158%2F2159-8290.CD-19-0367"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Acquired on-target clinical resistance validates FGFR4 as a driver of hepatocellular carcinoma</span></div><div class="casAuthors">Hatlen, Megan A.; Schmidt-Kittler, Oleg; Sherwin, Cori Ann; Rozsahegyi, Emily; Rubin, Nooreen; Sheets, Michael P.; Kim, Joseph L.; Miduturu, Chandrasekhar; Bifulco, Neil; Brooijmans, Natasja; Shi, Hongliang; Guzi, Timothy; Boral, Andy; Lengauer, Christoph; Dorsch, Marion; Kim, Richard D.; Kang, Yoon-Koo; Wolf, Beni B.; Hoeflich, Klaus P.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1686-1695</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Hepatocellular carcinoma (HCC) is a leading cause of cancer mortality worldwide with no clin. confirmed oncogenic driver.  Although preclin. studies implicate the FGF19 receptor FGFR4 in hepatocarcinogenesis, the dependence of human cancer on FGFR4 has not been demonstrated.  Fisogatinib (BLU-554) is a potent and selective inhibitor of FGFR4 and demonstrates clin. benefit and tumor regression in patients with HCC with aberrant FGF19 expression.  Mutations were identified in the gatekeeper and hinge-1 residues in the kinase domain of FGFR4 upon disease progression in 2 patients treated with fisogatinib, which were confirmed to mediate resistance in vitro and in vivo.  A gatekeeper-agnostic, pan-FGFR inhibitor decreased HCC xenograft growth in the presence of these mutations, demonstrating continued FGF19-FGFR4 pathway dependence.  These results validate FGFR4 as an oncogenic driver and warrant further therapeutic targeting of this kinase in the clinic.  SIGNIFICANCE: Our study is the first to demonstrate on-target FGFR4 kinase domain mutations as a mechanism of acquired clin. resistance to targeted therapy.  This further establishes FGF19-FGFR4 pathway activation as an oncogenic driver.  These findings support further investigation of fisogatinib in HCC and inform the profile of potential next-generation inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqO2LPyX4ffo7Vg90H21EOLACvtfcHk0lh1KvgeRmz1rQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1Ght77F&md5=8d5ac6ec96fb9c179635f2b4a31b990f</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-19-0367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-19-0367%26sid%3Dliteratum%253Aachs%26aulast%3DHatlen%26aufirst%3DM.%2BA.%26aulast%3DSchmidt-Kittler%26aufirst%3DO.%26aulast%3DSherwin%26aufirst%3DC.%2BA.%26aulast%3DRozsahegyi%26aufirst%3DE.%26aulast%3DRubin%26aufirst%3DN.%26aulast%3DSheets%26aufirst%3DM.%2BP.%26aulast%3DKim%26aufirst%3DJ.%2BL.%26aulast%3DMiduturu%26aufirst%3DC.%26aulast%3DBifulco%26aufirst%3DN.%26aulast%3DBrooijmans%26aufirst%3DN.%26aulast%3DShi%26aufirst%3DH.%26aulast%3DGuzi%26aufirst%3DT.%26aulast%3DBoral%26aufirst%3DA.%26aulast%3DLengauer%26aufirst%3DC.%26aulast%3DDorsch%26aufirst%3DM.%26aulast%3DKim%26aufirst%3DR.%2BD.%26aulast%3DKang%26aufirst%3DY.-K.%26aulast%3DWolf%26aufirst%3DB.%2BB.%26aulast%3DHoeflich%26aufirst%3DK.%2BP.%26atitle%3DAcquired%2520on-target%2520clinical%2520resistance%2520validates%2520FGFR4%2520as%2520a%2520driver%2520of%2520hepatocellular%2520carcinoma%26jtitle%3DCancer%2520Discovery%26date%3D2019%26volume%3D9%26spage%3D1686%26epage%3D1695%26doi%3D10.1158%2F2159-8290.CD-19-0367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marsiglia, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katigbak, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohammadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traaseth, N. J.</span></span> <span> </span><span class="NLM_article-title">A conserved allosteric pathway in tyrosine kinase regulation</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1308</span>– <span class="NLM_lpage">1315</span>, <span class="refDoi"> DOI: 10.1016/j.str.2019.05.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1016%2Fj.str.2019.05.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=31204250" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFOku7nP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2019&pages=1308-1315&author=W.+M.+Marsigliaauthor=J.+Katigbakauthor=S.+Zhengauthor=M.+Mohammadiauthor=Y.+Zhangauthor=N.+J.+Traaseth&title=A+conserved+allosteric+pathway+in+tyrosine+kinase+regulation&doi=10.1016%2Fj.str.2019.05.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">A Conserved Allosteric Pathway in Tyrosine Kinase Regulation</span></div><div class="casAuthors">Marsiglia, William M.; Katigbak, Joseph; Zheng, Sijin; Mohammadi, Moosa; Zhang, Yingkai; Traaseth, Nathaniel J.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1308-1315.e3</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">An autoinhibitory network of hydrogen bonds located at the kinase hinge (referred to as the "mol. brake") regulates the activity of several receptor tyrosine kinases.  The mechanism whereby mutational disengagement of the brake allosterically activates the kinase in human disease is incompletely understood.  We used a combination of NMR, bioinformatics, and mol. dynamics simulation to show that mutational disruption of the mol. brake triggers localized conformational perturbations that propagate to the active site.  This entails changes in interactions of an isoleucine, one of three hydrophobic residues that lock the phenylalanine of the DFG motif in an inactive conformation.  Structural anal. of tyrosine kinases provides evidence that this allosteric control mechanism is shared across the tyrosine kinase family.  We also show that highly activating mutations at the brake diminish the enzyme's thermostability, thereby explaining why these mutations cause milder skeletal syndromes compared with less-activating mutations in the activation loop.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2xwjFECEogrVg90H21EOLACvtfcHk0ljvrIsdZjOP2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFOku7nP&md5=6da80dec30b0ead22399e1f5d7491711</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2019.05.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2019.05.002%26sid%3Dliteratum%253Aachs%26aulast%3DMarsiglia%26aufirst%3DW.%2BM.%26aulast%3DKatigbak%26aufirst%3DJ.%26aulast%3DZheng%26aufirst%3DS.%26aulast%3DMohammadi%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DTraaseth%26aufirst%3DN.%2BJ.%26atitle%3DA%2520conserved%2520allosteric%2520pathway%2520in%2520tyrosine%2520kinase%2520regulation%26jtitle%3DStructure%26date%3D2019%26volume%3D27%26spage%3D1308%26epage%3D1315%26doi%3D10.1016%2Fj.str.2019.05.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blais, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neubert, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohammadi, M.</span></span> <span> </span><span class="NLM_article-title">DFG-out mode of inhibition by an irreversible type-1 inhibitor capable of overcoming gate-keeper mutations in FGF receptors</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">299</span>– <span class="NLM_lpage">309</span>, <span class="refDoi"> DOI: 10.1021/cb500674s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500674s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslGgt7vM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=299-309&author=Z.+Huangauthor=L.+Tanauthor=H.+Wangauthor=Y.+Liuauthor=S.+Blaisauthor=J.+Dengauthor=T.+A.+Neubertauthor=N.+S.+Grayauthor=X.+Liauthor=M.+Mohammadi&title=DFG-out+mode+of+inhibition+by+an+irreversible+type-1+inhibitor+capable+of+overcoming+gate-keeper+mutations+in+FGF+receptors&doi=10.1021%2Fcb500674s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">DFG-out Mode of Inhibition by an Irreversible Type-1 Inhibitor Capable of Overcoming Gate-Keeper Mutations in FGF Receptors</span></div><div class="casAuthors">Huang, Zhifeng; Tan, Li; Wang, Huiyan; Liu, Yang; Blais, Steven; Deng, Jingjing; Neubert, Thomas A.; Gray, Nathanael S.; Li, Xiaokun; Mohammadi, Moosa</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">299-309</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Drug-resistance acquisition through kinase gate-keeper mutations is a major hurdle in the clinic.  Here, we detd. the first crystal structures of the human FGFR4 kinase domain (FGFR4K) alone and complexed with ponatinib, a promiscuous type-2 (DFG-out) kinase inhibitor, and an oncogenic FGFR4K harboring the V550L gate-keeper mutation bound to FIIN-2, a new type-1 irreversible inhibitor.  Remarkably, like ponatinib, FIIN-2 also binds in the DFG-out mode despite lacking a functional group necessary to occupy the pocket vacated upon the DFG-out flip.  Structural anal. reveals that the covalent bond between FIIN-2 and a cysteine, uniquely present in the glycine-rich loop of FGFR kinases, facilitates the DFG-out conformation, which together with the internal flexibility of FIIN-2 enables FIIN-2 to avoid the steric clash with the gate-keeper mutation that causes the ponatinib resistance.  The structural data provide a blueprint for the development of next generation anticancer inhibitors through combining the salient inhibitory mechanisms of ponatinib and FIIN-2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpf97N01ZUSpbVg90H21EOLACvtfcHk0ljvrIsdZjOP2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslGgt7vM&md5=0c741cc5492596f9ea9323e4d45e274b</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fcb500674s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500674s%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DTan%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DBlais%26aufirst%3DS.%26aulast%3DDeng%26aufirst%3DJ.%26aulast%3DNeubert%26aufirst%3DT.%2BA.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DMohammadi%26aufirst%3DM.%26atitle%3DDFG-out%2520mode%2520of%2520inhibition%2520by%2520an%2520irreversible%2520type-1%2520inhibitor%2520capable%2520of%2520overcoming%2520gate-keeper%2520mutations%2520in%2520FGF%2520receptors%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D299%26epage%3D309%26doi%3D10.1021%2Fcb500674s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Burgi, H. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunitz, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehn, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wipff, G.</span></span> <span> </span><span class="NLM_article-title">Sterochemistry of reaction paths at carbonyl centres</span>. <i>Tetrahedron</i> <span class="NLM_year" style="font-weight: bold;">1974</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">1563</span>– <span class="NLM_lpage">1572</span>, <span class="refDoi"> DOI: 10.1016/S0040-4020(01)90678-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1016%2FS0040-4020%2801%2990678-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADyaE2MXlsleqtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1974&pages=1563-1572&author=H.+B.+Burgiauthor=J.+D.+Dunitzauthor=J.+M.+Lehnauthor=G.+Wipff&title=Sterochemistry+of+reaction+paths+at+carbonyl+centres&doi=10.1016%2FS0040-4020%2801%2990678-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62aR"><div class="casContent"><span class="casTitleNuber">62a</span><div class="casTitle"><span class="NLM_cas:atitle">Stereochemistry of reaction paths at carbonyl centers</span></div><div class="casAuthors">Buergi, H. B.; Dunitz, J. D.; Lehn, J. M.; Wipff, G.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron</span>
        (<span class="NLM_cas:date">1974</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1563-72</span>CODEN:
                <span class="NLM_cas:coden">TETRAB</span>;
        ISSN:<span class="NLM_cas:issn">0040-4020</span>.
    </div><div class="casAbstract">Ab initio MO calcns. and crystal structure structure data were used to infer the reaction paths of nucleophilic addn. to carbonyl compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgIyobc_DXFrVg90H21EOLACvtfcHk0ljvrIsdZjOP2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2MXlsleqtA%253D%253D&md5=b2d3956e505f323b2a475fa69ce01d23</span></div><a href="/servlet/linkout?suffix=cit62a&amp;dbid=16384&amp;doi=10.1016%2FS0040-4020%2801%2990678-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0040-4020%252801%252990678-7%26sid%3Dliteratum%253Aachs%26aulast%3DBurgi%26aufirst%3DH.%2BB.%26aulast%3DDunitz%26aufirst%3DJ.%2BD.%26aulast%3DLehn%26aufirst%3DJ.%2BM.%26aulast%3DWipff%26aufirst%3DG.%26atitle%3DSterochemistry%2520of%2520reaction%2520paths%2520at%2520carbonyl%2520centres%26jtitle%3DTetrahedron%26date%3D1974%26volume%3D30%26spage%3D1563%26epage%3D1572%26doi%3D10.1016%2FS0040-4020%2801%2990678-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit62b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Burgi, H. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunitz, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shefter, E.</span></span> <span> </span><span class="NLM_article-title">Geometrical reaction coordinates II nucleophilic addition to a carbonyl group</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1973</span>,  <span class="NLM_volume"><i>95</i></span>,  <span class="NLM_fpage">5065</span>– <span class="NLM_lpage">5067</span>, <span class="refDoi"> DOI: 10.1021/ja00796a058</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00796a058" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADyaE3sXkvFWhs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=1973&pages=5065-5067&author=H.+B.+Burgiauthor=J.+D.+Dunitzauthor=E.+Shefter&title=Geometrical+reaction+coordinates+II+nucleophilic+addition+to+a+carbonyl+group&doi=10.1021%2Fja00796a058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62bR"><div class="casContent"><span class="casTitleNuber">62b</span><div class="casTitle"><span class="NLM_cas:atitle">Geometrical reaction coordinates.  II.  Nucleophilic addition to a carbonyl group</span></div><div class="casAuthors">Burgi, H. B.; Dunitz, J. D.; Shefter, Eli</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1973</span>),
    <span class="NLM_cas:volume">95</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5065-7</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">Analysis of intramol. N...C:O interactions obsd. in six crystal structures provides an exptl. basis for mapping the reaction coordinate (min. energy pathway) for the addn. reaction of a nucleophile to a carbonyl group.  The line of approach of the nucleophile is not perpendicular to the C-O bond but is inclined at about 107° to it.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoG98LdihhQYLVg90H21EOLACvtfcHk0lhqaDWwFapPaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE3sXkvFWhs7w%253D&md5=c7aa2b9f31ad151d8f334bf6032d7da4</span></div><a href="/servlet/linkout?suffix=cit62b&amp;dbid=16384&amp;doi=10.1021%2Fja00796a058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00796a058%26sid%3Dliteratum%253Aachs%26aulast%3DBurgi%26aufirst%3DH.%2BB.%26aulast%3DDunitz%26aufirst%3DJ.%2BD.%26aulast%3DShefter%26aufirst%3DE.%26atitle%3DGeometrical%2520reaction%2520coordinates%2520II%2520nucleophilic%2520addition%2520to%2520a%2520carbonyl%2520group%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1973%26volume%3D95%26spage%3D5065%26epage%3D5067%26doi%3D10.1021%2Fja00796a058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span> <span> </span><span class="NLM_article-title">Characterization of FGF401 as a reversible covalent inhibitor of fibroblast growth factor receptor 4</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">5890</span>– <span class="NLM_lpage">5893</span>, <span class="refDoi"> DOI: 10.1039/C9CC02052G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1039%2FC9CC02052G" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=31041948" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC1MXosVSitrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2019&pages=5890-5893&author=Z.+Zhouauthor=X.+Chenauthor=Y.+Fuauthor=Y.+Zhangauthor=S.+Daiauthor=J.+Liauthor=L.+Chenauthor=G.+Xuauthor=Z.+Chenauthor=Y.+Chen&title=Characterization+of+FGF401+as+a+reversible+covalent+inhibitor+of+fibroblast+growth+factor+receptor+4&doi=10.1039%2FC9CC02052G"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of FGF401 as a reversible covalent inhibitor of fibroblast growth factor receptor 4</span></div><div class="casAuthors">Zhou, Zhan; Chen, Xiaojuan; Fu, Ying; Zhang, Ye; Dai, Shuyan; Li, Jun; Chen, Lin; Xu, Guangyu; Chen, Zhuchu; Chen, Yongheng</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">5890-5893</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Biochem. and structural studies provide information on the mode of action of FGF401 as a selective, reversible covalent inhibitor of FGFR4.  Kinase and proliferation assays reveal that FGF401 has the ability to overcome gatekeeper mutations in FGFR4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolsnzMt2X_B7Vg90H21EOLACvtfcHk0lhqaDWwFapPaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXosVSitrc%253D&md5=00b2da2e9df52673444bf51e5c8befe9</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1039%2FC9CC02052G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC9CC02052G%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DFu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DDai%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DY.%26atitle%3DCharacterization%2520of%2520FGF401%2520as%2520a%2520reversible%2520covalent%2520inhibitor%2520of%2520fibroblast%2520growth%2520factor%2520receptor%25204%26jtitle%3DChem.%2520Commun.%26date%3D2019%26volume%3D55%26spage%3D5890%26epage%3D5893%26doi%3D10.1039%2FC9CC02052G" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alex, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Millan, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakenhut, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitlock, G. A.</span></span> <span> </span><span class="NLM_article-title">Intramolecular hydrogen bonding to improve membrane permeability and absorption in beyond rule of five chemical space</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">669</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1039/c1md00093d</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1039%2Fc1md00093d" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotl2gt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=669-674&author=A.+Alexauthor=D.+S.+Millanauthor=M.+Perezauthor=F.+Wakenhutauthor=G.+A.+Whitlock&title=Intramolecular+hydrogen+bonding+to+improve+membrane+permeability+and+absorption+in+beyond+rule+of+five+chemical+space&doi=10.1039%2Fc1md00093d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Intramolecular hydrogen bonding to improve membrane permeability and absorption in beyond rule of five chemical space</span></div><div class="casAuthors">Alex, Alexander; Millan, David S.; Perez, Manuel; Wakenhut, Florian; Whitlock, Gavin A.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">669-674</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Utilizing 'beyond rule of five' chem. space is becoming increasingly important in drug design, but is usually at odds with good oral absorption.  The formation of intramol. hydrogen bonds in drug mols. is hypothesised to shield polarity facilitating improved membrane permeability and intestinal absorption.  NMR based evidence for intramol. hydrogen bonding in several 'beyond rule of five' oral drugs is described.  Furthermore, the propensity for these drugs to form intramol. hydrogen bonds could be predicted for through modeling the lowest energy conformation in the gas phase.  The modulation of apparent lipophilicity through intramol. hydrogen bonding in these mols. is supported by intrinsic cell permeability and intestinal absorption data in rat and human.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp93l2KJMdB2bVg90H21EOLACvtfcHk0lhqaDWwFapPaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotl2gt7o%253D&md5=3effbdd29832ac7b57eabbc26c66217b</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1039%2Fc1md00093d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc1md00093d%26sid%3Dliteratum%253Aachs%26aulast%3DAlex%26aufirst%3DA.%26aulast%3DMillan%26aufirst%3DD.%2BS.%26aulast%3DPerez%26aufirst%3DM.%26aulast%3DWakenhut%26aufirst%3DF.%26aulast%3DWhitlock%26aufirst%3DG.%2BA.%26atitle%3DIntramolecular%2520hydrogen%2520bonding%2520to%2520improve%2520membrane%2520permeability%2520and%2520absorption%2520in%2520beyond%2520rule%2520of%2520five%2520chemical%2520space%26jtitle%3DMedChemComm%26date%3D2011%26volume%3D2%26spage%3D669%26epage%3D674%26doi%3D10.1039%2Fc1md00093d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gu, Y.</span>; <span class="NLM_string-name">Wilbaux, M.</span>; <span class="NLM_string-name">Yang, S.</span>; <span class="NLM_string-name">Demanse, D.</span>; <span class="NLM_string-name">Jullion, A.</span>; <span class="NLM_string-name">Meille, C.</span>; <span class="NLM_string-name">Siddani, R.</span>; <span class="NLM_string-name">Graus Porta, D.</span>; <span class="NLM_string-name">Myers, A.</span></span> <span> </span><span class="NLM_article-title">Integrated Pharmacokinetic-Pharmacodynamic (PK-PD) and Exposure-Response (E-R) Analyses to Support the First-In-Human (FIH) Phase I/II Study of FGF401</span>. Presented at the 30th Joint European Organisation for Research and Treatment of Cancer, National Cancer Institute, American Association for Cancer Research Symposium on Molecular Targets and Cancer Therapeutics, Dublin, Ireland, Nov 13–16, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=Y.+Gu&author=M.+Wilbaux&author=S.+Yang&author=D.+Demanse&author=A.+Jullion&author=C.+Meille&author=R.+Siddani&author=D.+Graus+Porta&author=A.+Myers&title=Integrated+Pharmacokinetic-Pharmacodynamic+%28PK-PD%29+and+Exposure-Response+%28E-R%29+Analyses+to+Support+the+First-In-Human+%28FIH%29+Phase+I%2FII+Study+of+FGF401"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DGu%26aufirst%3DY.%26atitle%3DIntegrated%2520Pharmacokinetic-Pharmacodynamic%2520%2528PK-PD%2529%2520and%2520Exposure-Response%2520%2528E-R%2529%2520Analyses%2520to%2520Support%2520the%2520First-In-Human%2520%2528FIH%2529%2520Phase%2520I%252FII%2520Study%2520of%2520FGF401%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit65b"><span>Abstract:  <i>Eur. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume">103</span>,  <span class="NLM_fpage">E122</span>– <span class="NLM_lpage">E123</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Abstract%3A+Eur.+J.+Cancer+2018%2C+103%2C+E122%E2%80%93E123."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit65b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26jtitle%3DEur.%2520J.%2520Cancer%26date%3D2018%26volume%3D103%26spage%3DE122%26epage%3DE123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><a href="https://clinicaltrials.gov/ct2/show/NCT02325739" class="extLink">https://clinicaltrials.gov/ct2/show/NCT02325739</a> (accessed Sep 14, <span class="NLM_year">2020</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=https%3A%2F%2Fclinicaltrials.gov%2Fct2%2Fshow%2FNCT02325739+%28accessed+Sep+14%2C+2020%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, S. L.</span>; <span class="NLM_string-name">Yen, C.-J.</span>; <span class="NLM_string-name">Schuler, M.</span>; <span class="NLM_string-name">Lin, C.-C.</span>; <span class="NLM_string-name">Choo, S. P.</span>; <span class="NLM_string-name">Weiss, K. H.</span>; <span class="NLM_string-name">Geier, A.</span>; <span class="NLM_string-name">Okusaka, T.</span>; <span class="NLM_string-name">Lim, H. Y.</span>; <span class="NLM_string-name">Macarulla, T.</span>; <span class="NLM_string-name">Zhu, A. X.</span>; <span class="NLM_string-name">Kakizume, T.</span>; <span class="NLM_string-name">Gu, Y.</span>; <span class="NLM_string-name">Graus-Porta, D.</span>; <span class="NLM_string-name">Myers, A. P.</span>; <span class="NLM_string-name">Delord, J.-P.</span></span> <span> </span><span class="NLM_article-title">Phase I/II Study of FGF401 in Adult Patients with Hepatocellular Carcinoma or Solid Tumors Characterized by FGFR4/KLB Expression</span>. Presented at the American Association for Cancer Research Annual Meeting, Washington, DC, Apr 1–5, <span class="NLM_year">2017</span>; Abstract CT106.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=S.+L.+Chan&author=C.-J.+Yen&author=M.+Schuler&author=C.-C.+Lin&author=S.+P.+Choo&author=K.+H.+Weiss&author=A.+Geier&author=T.+Okusaka&author=H.+Y.+Lim&author=T.+Macarulla&author=A.+X.+Zhu&author=T.+Kakizume&author=Y.+Gu&author=D.+Graus-Porta&author=A.+P.+Myers&author=J.-P.+Delord&title=Phase+I%2FII+Study+of+FGF401+in+Adult+Patients+with+Hepatocellular+Carcinoma+or+Solid+Tumors+Characterized+by+FGFR4%2FKLB+Expression"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DS.%2BL.%26atitle%3DPhase%2520I%252FII%2520Study%2520of%2520FGF401%2520in%2520Adult%2520Patients%2520with%2520Hepatocellular%2520Carcinoma%2520or%2520Solid%2520Tumors%2520Characterized%2520by%2520FGFR4%252FKLB%2520Expression%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><a href="http://www.everestmedicines.com/Pipeline.aspx#news_138" class="extLink">http://www.everestmedicines.com/Pipeline.aspx#news_138</a> (accessed Sep 14, <span class="NLM_year">2020</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=http%3A%2F%2Fwww.everestmedicines.com%2FPipeline.aspx%23news_138+%28accessed+Sep+14%2C+2020%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yasuda, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsiao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivera, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yau, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palucki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dormer, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volante, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reider, P. J.</span></span> <span> </span><span class="NLM_article-title">An Efficient synthesis of an α<sub>v</sub>β<sub>3</sub> antagonist <i>J</i></span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">1959</span>– <span class="NLM_lpage">1966</span>, <span class="refDoi"> DOI: 10.1021/jo030297u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo030297u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BD2cXht12gtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2004&pages=1959-1966&author=N.+Yasudaauthor=Y.+Hsiaoauthor=M.+S.+Jensenauthor=N.+R.+Riveraauthor=C.+Yangauthor=K.+M.+Wellsauthor=J.+Yauauthor=M.+Paluckiauthor=L.+Tanauthor=P.+G.+Dormerauthor=R.+P.+Volanteauthor=D.+L.+Hughesauthor=P.+J.+Reider&title=An+Efficient+synthesis+of+an+%CE%B1v%CE%B23+antagonist+J&doi=10.1021%2Fjo030297u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">An Efficient Synthesis of an αvβ3 Antagonist</span></div><div class="casAuthors">Yasuda, Nobuyoshi; Hsiao, Yi; Jensen, Mark S.; Rivera, Nelo R.; Yang, Chunhua; Wells, Kenneth M.; Yau, James; Palucki, Michael; Tan, Lushi; Dormer, Peter G.; Volante, Ralph P.; Hughes, David L.; Reider, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1959-1966</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A practical prepn. of an αvβ3 antagonist, (βS)-6-methoxy-β-[2-oxo-3-[3-(5,6,7,8-tetrahydro-1,8-naphthyridin-2-yl)propyl]-1-imidazolidinyl]-3-pyridinepropanoic acid (I), is reported.  The antagonist consists of three key components, a tetrahydronaphthyridine moiety, a β-alanine moiety, and a central imidazolidone moiety.  The tetrahydronaphthyridine component was prepd. using two different methods, both of which relied on variations of the Friedlander reaction to establish the desired regiochem.  The β-alanine component was prepd. using Davies' asym. 1,4-addn. methodol. as the key stereo-defining step.  The central imidazolidone portion was created from these two components using an effective three-step cyclization protocol.  Thus, a highly convergent process for the drug candidate was defined.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-OkegSdO1JLVg90H21EOLACvtfcHk0liAnqMMymTvXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXht12gtrY%253D&md5=178065fe8be7e5a2e4b236c3392f839f</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1021%2Fjo030297u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo030297u%26sid%3Dliteratum%253Aachs%26aulast%3DYasuda%26aufirst%3DN.%26aulast%3DHsiao%26aufirst%3DY.%26aulast%3DJensen%26aufirst%3DM.%2BS.%26aulast%3DRivera%26aufirst%3DN.%2BR.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DWells%26aufirst%3DK.%2BM.%26aulast%3DYau%26aufirst%3DJ.%26aulast%3DPalucki%26aufirst%3DM.%26aulast%3DTan%26aufirst%3DL.%26aulast%3DDormer%26aufirst%3DP.%2BG.%26aulast%3DVolante%26aufirst%3DR.%2BP.%26aulast%3DHughes%26aufirst%3DD.%2BL.%26aulast%3DReider%26aufirst%3DP.%2BJ.%26atitle%3DAn%2520Efficient%2520synthesis%2520of%2520an%2520%25CE%25B1v%25CE%25B23%2520antagonist%2520J%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2004%26volume%3D69%26spage%3D1959%26epage%3D1966%26doi%3D10.1021%2Fjo030297u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Holt, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreassen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bakke, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiksdahl, A.</span></span> <span> </span><span class="NLM_article-title">Nitropyridyl isocyanates</span>. <i>J. Heterocycl. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">259</span>– <span class="NLM_lpage">264</span>, <span class="refDoi"> DOI: 10.1002/jhet.5570420213</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=10.1002%2Fjhet.5570420213" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BD2MXivFGlsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2005&pages=259-264&author=J.+Holtauthor=T.+Andreassenauthor=J.+M.+Bakkeauthor=A.+Fiksdahl&title=Nitropyridyl+isocyanates&doi=10.1002%2Fjhet.5570420213"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Nitropyridyl isocyanates</span></div><div class="casAuthors">Holt, Jarle; Andreassen, Trygve; Bakke, Jan M.; Fiksdahl, Anne</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Heterocyclic Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">259-264</span>CODEN:
                <span class="NLM_cas:coden">JHTCAD</span>;
        ISSN:<span class="NLM_cas:issn">0022-152X</span>.
    
            (<span class="NLM_cas:orgname">HeteroCorporation</span>)
        </div><div class="casAbstract">The authors report the first prepn. of 3-nitro-4-pyridyl isocyanate (I) and 5-nitro-2-pyridyl isocyanate (II).  They were formed by Curtius rearrangement of the corresponding acyl azides, prepd. from Me 3-nitro-4-pyridinecarboxylate.  I can be stored in dry soln. and is stable at room temp. for several weeks while II was less stable and should be used for synthetic purposes immediately.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTaMMqynXYeLVg90H21EOLACvtfcHk0liAnqMMymTvXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXivFGlsrw%253D&md5=f95cc37035ae9bda443eeee8893fcf1d</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1002%2Fjhet.5570420213&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjhet.5570420213%26sid%3Dliteratum%253Aachs%26aulast%3DHolt%26aufirst%3DJ.%26aulast%3DAndreassen%26aufirst%3DT.%26aulast%3DBakke%26aufirst%3DJ.%2BM.%26aulast%3DFiksdahl%26aufirst%3DA.%26atitle%3DNitropyridyl%2520isocyanates%26jtitle%3DJ.%2520Heterocycl.%2520Chem.%26date%3D2005%26volume%3D42%26spage%3D259%26epage%3D264%26doi%3D10.1002%2Fjhet.5570420213" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wellenzohn, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lessel, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beller, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isambert, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoenke, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nosse, B.</span></span> <span> </span><span class="NLM_article-title">Identification of new potent GPR119 agonists by combining virtual screening and combinatorial chemistry</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">11031</span>– <span class="NLM_lpage">11041</span>, <span class="refDoi"> DOI: 10.1021/jm301549a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301549a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslOlsL%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=11031-11041&author=B.+Wellenzohnauthor=U.+Lesselauthor=A.+Bellerauthor=T.+Isambertauthor=C.+Hoenkeauthor=B.+Nosse&title=Identification+of+new+potent+GPR119+agonists+by+combining+virtual+screening+and+combinatorial+chemistry&doi=10.1021%2Fjm301549a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of New Potent GPR119 Agonists by Combining Virtual Screening and Combinatorial Chemistry</span></div><div class="casAuthors">Wellenzohn, Bernd; Lessel, Uta; Beller, Andreas; Isambert, Timo; Hoenke, Christoph; Nosse, Bernd</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">11031-11041</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Virtual screening in a huge collection of virtual combinatorial libraries has led to the identification of two new structural classes of GPR119 agonists with submicromolar in vitro potencies.  Herein, the authors describe the virtual screening process involving feature trees fragment space searches followed by a 3D postprocessing step.  The in silico findings were then filtered and prioritized, and finally, combinatorial libraries of target mols. were synthesized.  Furthermore the so-called "activity-anchor principle" is introduced as an element to increase the chance to generate true hits.  An activity anchor is a structural element expected to provide key contributions to a certain biol. activity.  Application of this technique has led to the discovery of two new GPR119-agonist hit series, one of which was further optimized to progress as a novel lead class.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqe1IfuyaYeNrVg90H21EOLACvtfcHk0liAnqMMymTvXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslOlsL%252FJ&md5=18e23e2eab61f8ef0015d0b76f7e467b</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1021%2Fjm301549a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301549a%26sid%3Dliteratum%253Aachs%26aulast%3DWellenzohn%26aufirst%3DB.%26aulast%3DLessel%26aufirst%3DU.%26aulast%3DBeller%26aufirst%3DA.%26aulast%3DIsambert%26aufirst%3DT.%26aulast%3DHoenke%26aufirst%3DC.%26aulast%3DNosse%26aufirst%3DB.%26atitle%3DIdentification%2520of%2520new%2520potent%2520GPR119%2520agonists%2520by%2520combining%2520virtual%2520screening%2520and%2520combinatorial%2520chemistry%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D11031%26epage%3D11041%26doi%3D10.1021%2Fjm301549a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NWZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NWZ','PDB','5NWZ'); return false;">PDB: 5NWZ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NUD" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NUD','PDB','5NUD'); return false;">PDB: 5NUD</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YI8" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YI8','PDB','6YI8'); return false;">PDB: 6YI8</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XCU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XCU','PDB','4XCU'); return false;">PDB: 4XCU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VND" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VND','PDB','5VND'); return false;">PDB: 5VND</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4QQC" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4QQC','PDB','4QQC'); return false;">PDB: 4QQC</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4R6V" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4R6V','PDB','4R6V'); return false;">PDB: 4R6V</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i30"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01019">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_08302"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c01019?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01019</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Experimental procedures for the synthetic sequences shown in <a class="ref internalNav" href="#sch1">Schemes 1</a>–<a class="ref internalNav" href="#sch4">4</a>, methods of synthesis, characterizing data for all the final compounds,  details of how the molecular modeling, biological, and physicochemical assays were conducted (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01019/suppl_file/jm0c01019_si_001.pdf">PDF</a>)</p></li><li><p class="inline">FGFR4 kinase-domain homology model (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01019/suppl_file/jm0c01019_si_002.pdb">PDB</a>)</p></li><li><p class="inline">Molecular formula strings (SMILES) with associated biological data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01019/suppl_file/jm0c01019_si_003.xlsx">XLSX</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01019/suppl_file/jm0c01019_si_001.pdf">jm0c01019_si_001.pdf (6.98 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01019/suppl_file/jm0c01019_si_002.pdb">jm0c01019_si_002.pdb (197.42 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01019/suppl_file/jm0c01019_si_003.xlsx">jm0c01019_si_003.xlsx (15.72 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last"><b>84</b> cocrystallized with the kinase domain of FGFR4 (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YI8">6YI8</a>). Authors will release the atomic coordinates upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c01019&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c01019%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-21" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c01019" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                13MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679997c0ab5c2521","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
