
<html lang="en"     class="pb-page"  data-request-id="df49a654-c4bb-49a8-98b5-a0477f89082c"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.0c02076;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Structure- and Similarity-Based Survey of Allosteric Kinase Inhibitors, Activators, and Closely Related Compounds" /></meta><meta name="dc.Creator" content="Oliver  Laufkötter" /></meta><meta name="dc.Creator" content="Huabin  Hu" /></meta><meta name="dc.Creator" content="Filip  Miljković" /></meta><meta name="dc.Creator" content="Jürgen  Bajorath" /></meta><meta name="dc.Description" content="Allosteric kinase inhibitors are thought to have high selectivity and are prime candidates for kinase drug discovery. In addition, the exploration of allosteric mechanisms represents an attractive ..." /></meta><meta name="Description" content="Allosteric kinase inhibitors are thought to have high selectivity and are prime candidates for kinase drug discovery. In addition, the exploration of allosteric mechanisms represents an attractive ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="January 21, 2021" /></meta><meta name="dc.Type" content="review-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c02076" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02076" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c02076" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02076" /></link>
        
    
    

<title>Structure- and Similarity-Based Survey of Allosteric Kinase Inhibitors, Activators, and Closely Related Compounds | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c02076" /></meta><meta property="og:title" content="Structure- and Similarity-Based Survey of Allosteric Kinase Inhibitors, Activators, and Closely Related Compounds" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/large/jm0c02076_0007.jpeg" /></meta><meta property="og:description" content="Allosteric kinase inhibitors are thought to have high selectivity and are prime candidates for kinase drug discovery. In addition, the exploration of allosteric mechanisms represents an attractive topic for basic research and drug design. Although the identification and characterization of allosteric kinase inhibitors is still far from being routine, X-ray structures of kinase complexes have been determined for a significant number of such inhibitors. On the basis of structural data, allosteric inhibitors can be confirmed. We report a comprehensive survey of allosteric kinase inhibitors and activators from publicly available X-ray structures, map their binding sites, and determine their distribution over binding pockets in kinases. In addition, we discuss structural features of these compounds and identify active structural analogues and high-confidence target annotations, indicating additional activities for a subset of allosteric inhibitors. This contribution aims to provide a detailed structure-based view of allosteric kinase inhibition." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c02076"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02076">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c02076&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c02076&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c02076&amp;href=/doi/10.1021/acs.jmedchem.0c02076" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, XXXX</span><span class="cit-fg-issue">, XXX</span><span class="cit-fg-pageRange">, XXX-XXX</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/0/0" title="Return to Articles ASAP" id="returnToASAP" class="content-navigation__btn--return"><span>RETURN TO ARTICLES ASAP</span></a><a href="/doi/10.1021/acs.jmedchem.0c01929" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Perspective</span><a href="/doi/10.1021/acs.jmedchem.0c01140" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Structure- and Similarity-Based Survey of Allosteric Kinase Inhibitors, Activators, and Closely Related Compounds</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Oliver Laufkötter</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Oliver Laufkötter</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Life Science Informatics, B-IT, LIMES Program Unit Chemical Biology and Medicinal Chemistry, Rheinische Friedrich-Wilhelms-Universität, Friedrich-Hirzebruch-Allee 6, D-53115 Bonn, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Oliver++Laufk%C3%B6tter">Oliver Laufkötter</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath1">View Biography</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Huabin Hu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Huabin Hu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Life Science Informatics, B-IT, LIMES Program Unit Chemical Biology and Medicinal Chemistry, Rheinische Friedrich-Wilhelms-Universität, Friedrich-Hirzebruch-Allee 6, D-53115 Bonn, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Huabin++Hu">Huabin Hu</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath2">View Biography</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Filip Miljković</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Filip Miljković</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Life Science Informatics, B-IT, LIMES Program Unit Chemical Biology and Medicinal Chemistry, Rheinische Friedrich-Wilhelms-Universität, Friedrich-Hirzebruch-Allee 6, D-53115 Bonn, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Filip++Miljkovi%C4%87">Filip Miljković</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath3">View Biography</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Jürgen Bajorath</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jürgen Bajorath</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Life Science Informatics, B-IT, LIMES Program Unit Chemical Biology and Medicinal Chemistry, Rheinische Friedrich-Wilhelms-Universität, Friedrich-Hirzebruch-Allee 6, D-53115 Bonn, Germany</div></div><span class="conrtib-corresp"><strong>*</strong>Phone: 49-228-7369-100, Email: <a href="/cdn-cgi/l/email-protection#fd9f9c97928f9c8995bd9f9489d3889394d09f929393d39998"><span class="__cf_email__" data-cfemail="e88a8982879a899c80a88a819cc69d8681c58a878686c68c8d">[email protected]</span></a></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=J%C3%BCrgen++Bajorath">Jürgen Bajorath</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath4">View Biography</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0557-5714" title="Orcid link">http://orcid.org/0000-0002-0557-5714</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02076&amp;href=/doi/10.1021%2Facs.jmedchem.0c02076" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, XXXX</span><span class="cit-issue">, XXX</span><span class="cit-pageRange">, XXX-XXX</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">January 21, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>30 November 2020</li><li><span class="item_label"><b>Published</b> online</span>21 January 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c02076" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02076</a></div><div class="article_header-article-copyright"><strong>© 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1%26pageCount%3D13%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DOliver%2BLaufk%25C3%25B6tter%252C%2BHuabin%2BHu%252C%2BFilip%2BMiljkovi%25C4%2587%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D0%26issueNum%3D0%26contentID%3Dacs.jmedchem.0c02076%26title%3DStructure-%2Band%2BSimilarity-Based%2BSurvey%2Bof%2BAllosteric%2BKinase%2BInhibitors%252C%2BActivators%252C%2Band%2BClosely%2BRelated%2BCompounds%26numPages%3D13%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D13%26publicationDate%3DJanuary%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c02076"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">3413</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c02076" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Structure- and Similarity-Based Survey of Allosteric Kinase Inhibitors, Activators, and Closely Related Compounds&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Oliver&quot;,&quot;last_name&quot;:&quot;Laufkötter&quot;},{&quot;first_name&quot;:&quot;Huabin&quot;,&quot;last_name&quot;:&quot;Hu&quot;},{&quot;first_name&quot;:&quot;Filip&quot;,&quot;last_name&quot;:&quot;Miljković&quot;},{&quot;first_name&quot;:&quot;Jürgen&quot;,&quot;last_name&quot;:&quot;Bajorath&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;01&quot;,&quot;day&quot;:&quot;21&quot;,&quot;issue&quot;:&quot;0&quot;,&quot;volume&quot;:&quot;0&quot;,&quot;pages&quot;:&quot;&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c02076&quot;},&quot;abstract&quot;:&quot;Allosteric kinase inhibitors are thought to have high selectivity and are prime candidates for kinase drug discovery. In addition, the exploration of allosteric mechanisms represents an attractive topic for basic research and drug design. Although the identification and characterization of allosteric kinase inhibitors is still far from being routine, X-ray structures of kinase complexes have been determined for a significant number of such inhibitors. On the basis of structural data, allosteric inhibitors can be confirmed. We report a comprehensive survey of allosteric kinase inhibitors and activators from publicly available X-ray structures, map their binding sites, and determine their distribution over binding pockets in kinases. In addition, we discuss structural features of these compounds and identify active structural analogues and high-confidence target annotations, indicating additional activities for a subset of allosteric inhibitors. This contribution aims to provide a detailed structure-based view &quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02076&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02076" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02076&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02076" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02076&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02076" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02076&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02076&amp;href=/doi/10.1021/acs.jmedchem.0c02076" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c02076" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c02076" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02076&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c02076%26sid%3Dliteratum%253Aachs%26pmid%3D33476146%26genre%3Darticle%26aulast%3DLaufk%25C3%25B6tter%26date%3D2021%26atitle%3DStructure-%2Band%2BSimilarity-Based%2BSurvey%2Bof%2BAllosteric%2BKinase%2BInhibitors%252C%2BActivators%252C%2Band%2BClosely%2BRelated%2BCompounds%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D0%26issue%3D0%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291445" title="Chemical structure">Chemical structure</a>,</li><li><a href="/action/doSearch?ConceptID=291264" title="Kinase inhibitors">Kinase inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/0/0" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/medium/jm0c02076_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/large/jm0c02076_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02076&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Allosteric kinase inhibitors are thought to have high selectivity and are prime candidates for kinase drug discovery. In addition, the exploration of allosteric mechanisms represents an attractive topic for basic research and drug design. Although the identification and characterization of allosteric kinase inhibitors is still far from being routine, X-ray structures of kinase complexes have been determined for a significant number of such inhibitors. On the basis of structural data, allosteric inhibitors can be confirmed. We report a comprehensive survey of allosteric kinase inhibitors and activators from publicly available X-ray structures, map their binding sites, and determine their distribution over binding pockets in kinases. In addition, we discuss structural features of these compounds and identify active structural analogues and high-confidence target annotations, indicating additional activities for a subset of allosteric inhibitors. This contribution aims to provide a detailed structure-based view of allosteric kinase inhibition.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="specialIssueNotice" class="extra-info-sec articleNote"><h4>SPECIAL ISSUE</h4><p class="last">This article is part of the <a href="/toc/jmcmar"><issue-title>New Horizons in Drug Discovery - Understanding and Advancing Kinase Inhibitors</issue-title></a> special issue.</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">1.  Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05761" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05761" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Since the first approval of a protein kinase inhibitor for therapy (imatinib, 2001), the field of kinase inhibitor drug discovery has excelled.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1−3)</a> In September 2020, 62 kinase inhibitors were approved by the U.S. Food and Drug Administration (FDA). In addition, more than 200 other compounds with Nonproprietary International Names are currently in clinical development. Resulting from the high interest in kinase inhibitors in oncology and other therapeutic areas<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> and a plethora of research efforts, a wealth of kinase and inhibitor data have become available in the public domain (with many more proprietary data remaining in pharmaceutical companies). On the basis of our periodic data surveys, more than 130 000 inhibitors with reliable activity measurements are currently available in public repositories that cover more than 80% of the 518 kinases comprising the human kinome.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> In addition, more than 5000 X-ray structures of complexes formed by more than 300 kinases and 3000 inhibitors have been deposited in the Protein Data Bank (PDB).<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Taken together, these data provide a rich source for exploring kinase inhibitor characteristics and a knowledge base for developing new compounds.</div><div class="NLM_p">Kinase inhibitors have been classified and assigned to different types depending on their mode of action and location of their binding sites. Type I, I<span class="NLM_inline-formula"><img class="imgWithoutFigViewer inline-formula-styles" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/medium/jm0c02076_m001.gif" alt="" id="_i3" /></img></span>, and II inhibitors are ATP-site-directed, whereas type III and IV inhibitors bind outside the ATP site and act by allosteric mechanisms.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a> Type I and II inhibitors bind to an active and inactive kinase conformation, respectively, while type I<span class="NLM_inline-formula"><img class="imgWithoutFigViewer inline-formula-styles" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/medium/jm0c02076_m002.gif" alt="" id="_i4" /></img></span> inhibitors interact with an intermediate conformational state. Type II inhibitors have been further classified. Binding of type IIA inhibitors extends into the so-called back pocket while type IIB inhibitors only bind to the front cleft and gate area of the active site region. The binding site of type III inhibitors is proximal to the ATP site and includes at least two variants, depending on whether the kinase contains a pleckstrin homology domain or not. By contrast, type IV inhibitors target sites that are distant from the ATP site and have different locations. Type V inhibitors are bivalent constructs typically combining an ATP-site-directed inhibitor through a linker fragment with a substrate analogue targeting another site in the catalytic or regulatory domain of kinases.<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8,9)</a> As substrates, peptides (or peptidomimetics) are often used such as, for example, phosphorylated peptides targeting SH2 domains. However, a variety of bivalent constructs targeting different binding sites are principally possible. In addition, covalent kinase inhibitors are classified as type VI inhibitors.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> There has been increasing interest in covalent kinase inhibition in recent years, especially reversible covalent inhibition, because of increased efficacy and kinase (isoform) selectivity, prolonged residence times, or improved pharmacokinetics that are often observed for such compounds compared with competitive inhibitors.<a onclick="showRef(event, 'ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref10 ref11">(10,11)</a> Type VI inhibitors mostly, but not always, target free cysteine residues in kinases using thiol-reactive groups (warheads)<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> and include both ATP-site-directed compounds that block ATP binding and covalent-allosteric inhibitors.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> In addition, type VII inhibitors have been proposed that bind to extracellular regions of receptor Tyr kinases and trigger allosteric inhibitory effects.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Kinase activators have also been increasingly considered as tools for gaining a deeper understanding of kinase regulation and mechanisms of action. Currently available activators exclusively bind to allosteric sites.</div><div class="NLM_p">Much of the high interest in allosteric kinase inhibitors stems from the expectation that these compounds should be more specific than ATP-site-directed inhibitors, taking into consideration that the ATP binding site is largely conserved across the kinome. The vast majority of kinase inhibitors available in the public domain are type I inhibitors.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> While these inhibitors include highly promiscuous compounds,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> the expectation that ATP-site-directed compounds should generally be promiscuous cannot be supported on the basis of available activity data.<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14,15)</a> If reliable activity measurements in target-based assays are considered, more than 60% of currently available inhibitors are only active against a single kinase, and less than five percent have five or more kinase annotations.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> Even taking activity data incompleteness into account (i.e., not all inhibitors have been tested against the entire kinome), promiscuity of ATP-site-directed kinase inhibitors cannot be generalized. This is also consistent with results of selectivity-oriented kinase profiling<a onclick="showRef(event, 'ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref16 ref17 ref18">(16−18)</a> and increasing insights into the varying influence of kinase inhibition kinetics<a onclick="showRef(event, 'ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref19 ref20">(19,20)</a> and the dynamics of active versus inactive conformational states adapted by kinases.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a></div><div class="NLM_p">While ATP-site-directed kinase inhibitors may also be highly selective, allosteric inhibitors display a greater variety of binding modes and mechanisms than other kinase inhibitors and have a principally higher potential for selective inhibition. However, the discovery of allosteric kinase inhibitors is far from being routine and has often been serendipitous. Moreover, confirming allosteric mechanisms of action is prone to complications.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a></div><div class="NLM_p">Although it has thus far not been possible to develop experimental approaches for a more routine discovery of allosteric kinase inhibitors, a significant number of such inhibitors have become available that were confirmed on the basis of X-ray structures. Binding modes of these compounds have typically been explored on a case-by-case basis.</div><div class="NLM_p last">We have been interested in comprehensively analyzing and comparing allosteric ligands for which X-ray structures of complexes with kinases were determined. Herein, we report a survey of these ligands by studying their distribution over different binding pockets and kinases and analyzing characteristic structural features. In addition, a search was carried out for reported nonkinase targets of these ligands, which is a topic of high interest in polypharmacology-oriented drug discovery.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Furthermore, structural analogues of allosteric ligands were systematically identified, and available activity annotations were collected to derive additional target hypotheses.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i5">2.  Allosteric Kinase Ligands from X-ray Structures</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_28868" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_28868" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">X-ray structures of complexes between allosteric ligands and human kinases (excluding atypical kinases) and associated data were extracted from the PDB,<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> KLIFS,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Profkin,<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> and ASD<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> databases. For X-ray structures, PDB codes were collected and UniProt IDs<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> of kinases identified. Only X-ray structures with a resolution higher than 3.5 Å were considered. The resolution of selected complex X-ray structures ranged from 1.06–3.49 Å, with a median of 2.00 Å. Each structure was inspected and analyzed using the Molecular Operating Environment,<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> and the binding sites of any crystallographic ligands were determined. Structures were compared with the aid of the structural alignment web tool from PDBe-KB.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Interactive data curation yielded a total of 232 allosteric ligands that were classified according to inhibitor types and additional compound categories, as summarized in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Allosteric Ligands from X-ray Structures<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">type/category</th><th class="colsep0 rowsep0" align="center">binding site</th><th class="colsep0 rowsep0" align="center">count</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">type III</td><td class="colsep0 rowsep0" align="left">adjacent to ATP site</td><td class="colsep0 rowsep0" align="center">81</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">type IV</td><td class="colsep0 rowsep0" align="left">distant from ATP site</td><td class="colsep0 rowsep0" align="center">81</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">type V</td><td class="colsep0 rowsep0" align="left">ATP site plus allosteric pocket</td><td class="colsep0 rowsep0" align="center">15</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">type VI</td><td class="colsep0 rowsep0" align="left">allosteric pocket</td><td class="colsep0 rowsep0" align="center">10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">activators</td><td class="colsep0 rowsep0" align="left">allosteric pocket</td><td class="colsep0 rowsep0" align="center">19</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">multisite ligands</td><td class="colsep0 rowsep0" align="left">ATP site and allosteric pocket(s)/</td><td class="colsep0 rowsep0" align="center">26</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">different allosteric pockets</td><td class="colsep0 rowsep0" align="center"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>total</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"><b>232</b></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Listed are the numbers of identified unique allosteric kinase ligands from X-ray structures that were assigned to different inhibitor types or other compound categories.</p></div></div></div><div class="NLM_p">We found 81 type III inhibitors binding to the well-characterized allosteric pocket adjacent to the ATP site and 15 bivalent inhibitors occupying the ATP site and the adjacent allosteric pocket. In addition, 81 type IV inhibitors were identified and 10 (covalent) type VI inhibitors that bound to various pockets distant from the ATP site. Furthermore, 19 designated activators of kinase activity were found to be accommodated in different allosteric pockets.</div><div class="NLM_p last">In addition to the unique allosteric ligands reported in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, we also identified 26 multisite ligands that occupied more than one pocket in the same or different kinase structures. These multisite ligands bound to the ATP site plus an allosteric pocket or different allosteric pockets, as further discussed below.</div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i7">3.  Mapping of Allosteric Binding Sites</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15037" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15037" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We systematically mapped the binding pockets of the 232 allosteric compounds we identified. Analysis of the corresponding X-ray structures revealed a total of 12 binding sites (designated A-L) including the ATP site (A), depicted in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>. In this figure, the locations of all pockets identified in different kinases are mapped on the structure of the catalytic domain of an exemplary kinase to facilitate a comparison of these sites (no individual kinase contains all of these pockets). Eleven allosteric binding pockets (B-L) were identified. For each mapped pocket, we determined the number of kinases from X-ray structures in which it was present, reported in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> as the number of UniProt IDs (UPIDs), and the number of unique crystallographic ligands. For example, pocket B located adjacent to the ATP site was found in 19 kinases and accommodated a total of 90 different type III (or VI) inhibitors. By contrast, pocket L was only detected for a single inhibitor in a single kinase.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/medium/jm0c02076_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/large/jm0c02076_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Binding pockets. The location of all identified binding pockets is mapped on an exemplary kinase structure shown from a front-side view focusing on the ATP site (red) and a back-side view. For each pocket, the number of unique X-ray ligands is given in bold, and UniProt IDs (UPIDs) in parentheses report the number of kinases containing the pocket.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/large/jm0c02076_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02076&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">While some of these binding sites were already well-characterized in the literature, for others, only little, if any information was available. For each pocket, we identified inhibitor types on the basis of structural data and literature information, if available, and also determined binding sites of activators. The results are summarized in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Allosteric Binding Pockets<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="4" align="center">exemplary X-ray structure</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">pocket</th><th class="colsep0 rowsep0" align="center">inhibitor types/activators</th><th class="colsep0 rowsep0" align="center">kinase</th><th class="colsep0 rowsep0" align="center">PDB ID</th><th class="colsep0 rowsep0" align="center">ligand ID</th><th class="colsep0 rowsep0" align="center">resolution (Å)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">B</td><td class="colsep0 rowsep0" align="left">type III, VI</td><td class="colsep0 rowsep0" align="left">ITK</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4M15">4M15</a></td><td class="colsep0 rowsep0" align="left">QWS</td><td class="colsep0 rowsep0" align="center">1.52</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">type IV, VI, activators</td><td class="colsep0 rowsep0" align="left">PDPK1</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LVO">5LVO</a></td><td class="colsep0 rowsep0" align="left">78W</td><td class="colsep0 rowsep0" align="center">1.09</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">D</td><td class="colsep0 rowsep0" align="left">type IV, VI, activators</td><td class="colsep0 rowsep0" align="left">MAPK14</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ROC">3ROC</a></td><td class="colsep0 rowsep0" align="left">29B</td><td class="colsep0 rowsep0" align="center">1.70</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">E</td><td class="colsep0 rowsep0" align="left">type IV, activators</td><td class="colsep0 rowsep0" align="left">ABL1</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PYY">3PYY</a></td><td class="colsep0 rowsep0" align="left">3YY</td><td class="colsep0 rowsep0" align="center">1.85</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">F</td><td class="colsep0 rowsep0" align="left">type IV, activators</td><td class="colsep0 rowsep0" align="left">CSNK2A1</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6FVF">6FVF</a></td><td class="colsep0 rowsep0" align="left">503</td><td class="colsep0 rowsep0" align="center">1.47</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">G</td><td class="colsep0 rowsep0" align="left">type IV</td><td class="colsep0 rowsep0" align="left">CHK1</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3JVR">3JVR</a></td><td class="colsep0 rowsep0" align="left">AGX</td><td class="colsep0 rowsep0" align="center">1.76</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">type IV</td><td class="colsep0 rowsep0" align="left">CSNK2A1</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5OTZ">5OTZ</a></td><td class="colsep0 rowsep0" align="left">AUT</td><td class="colsep0 rowsep0" align="center">1.46</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">H+A</td><td class="colsep0 rowsep0" align="left">type V</td><td class="colsep0 rowsep0" align="left">CSNK2A1</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6EHK">6EHK</a></td><td class="colsep0 rowsep0" align="left">AUW</td><td class="colsep0 rowsep0" align="center">1.40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">I</td><td class="colsep0 rowsep0" align="left">type VI</td><td class="colsep0 rowsep0" align="left">CDK2</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5OSJ">5OSJ</a></td><td class="colsep0 rowsep0" align="left">AAK</td><td class="colsep0 rowsep0" align="center">1.83</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">J</td><td class="colsep0 rowsep0" align="left">type IV</td><td class="colsep0 rowsep0" align="left">BTK</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BKW">6BKW</a></td><td class="colsep0 rowsep0" align="left">FPZ</td><td class="colsep0 rowsep0" align="center">1.50</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">K</td><td class="colsep0 rowsep0" align="left">type IV, VI</td><td class="colsep0 rowsep0" align="left">AURKA</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ORL">5ORL</a></td><td class="colsep0 rowsep0" align="left">A4W</td><td class="colsep0 rowsep0" align="center">1.69</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">L</td><td class="colsep0 rowsep0" align="left">type IV</td><td class="colsep0 rowsep0" align="left">NTRK1</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6D1Z">6D1Z</a></td><td class="colsep0 rowsep0" align="left">FQD</td><td class="colsep0 rowsep0" align="center">1.87</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Mapped allosteric pockets are reported with their respective kinase inhibitor types and activators, if available. For each pocket, an exemplary X-ray structure is given.</p></div></div></div><div class="NLM_p">Seven of the 11 allosteric pockets mapped to the front side of the kinase where the catalytic site is located while four pockets mapped to the back side (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). They were widely distributed across the surface of the catalytic domain. While pocket B represented the known binding site of type III (plus covalent) allosteric inhibitors, Nine of the 10 remaining allosteric pockets were classified as type IV inhibitor binding sites, with the exception of pocket I, for which only type VI inhibitors were available. Structurally characterized bivalent inhibitors bound exclusively to the combination of the A (ATP) site and the adjacent allosteric H pocket. Bivalent inhibitors targeting other pocket combinations were not found. However, Eight of 23 identified bivalent compounds were also classified as multisite ligands because they bound to the H+A pocket combination as well as the F or D pocket, a surprising finding. Four of the 11 allosteric pockets also contained known kinase activators (C, D, E, and F). Both noncovalent and covalent inhibitors (type IV and VI) were accommodated in pockets C, D, and K. Hence, allosteric pockets C on the back and D on the front side (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) accommodated noncovalent and covalent kinase inhibitors as well as activators and were thus most versatile from a mechanistic point of view. All covalent inhibitors found in allosteric sites exclusively targeted cysteine residues. Seven of the 10 covalent compounds were acrylamides, one was an enone, and the remaining two inhibitors formed disulfide bonds. In the following, characteristics of individual pockets are discussed. For kinases and kinase groups, standard abbreviations are used.<a onclick="showRef(event, 'ref4 ref27'); return false;" href="javascript:void(0);" class="ref ref4 ref27">(4,27)</a></div><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> B Pocket</h3><div class="NLM_p last">The binding site of type III inhibitors was confirmed by X-ray crystallography in 2004 (PDB IDs: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1S9I">1S9I</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1S9J">1S9J</a>)<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> and is most extensively characterized, with structures containing 90 unique type III or VI inhibitors (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). The pocket participates in the formation of the catalytic hinge region. Trametinib, a MEK inhibitor, was approved by the FDA in 2013 as the first kinase inhibitor acting by a type III allosteric mechanism.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a></div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> C Pocket</h3><div class="NLM_p last">This binding pocket is located near the αC-helix proximal to the B pocket and has thus far been identified in PDK1, AURKA, and CDK2. It has also been termed the PDK1-interacting fragment (PIF) pocket in PDK1<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> or Y-pocket in AURKA.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> The first X-ray structure of a complex with an inhibitor bound to the C pocket was reported in 2009 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3HRF">3HRF</a>).<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> For this binding site, a total of 41 unique ligands were identified including type IV and VI inhibitors as well as activators.</div></div><div id="sec3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> D Pocket</h3><div class="NLM_p last">The first X-ray structure with a ligand binding to the D pocket, also termed lipid pocket, was reported in 2008 (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3C5U">3C5U</a>).<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> So far, 24 unique compounds have been identified binding to this pocket including type IV and VI inhibitors as well as activators. Several compounds have no detectable effect on the catalytic activity, while kinase activation by lipids binding to pocket D has also been reported.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> This binding site has been found in four kinases including three MAP kinase isoforms (MAPK8, MAPK10, MAPK14, CSNK2A1).</div></div><div id="sec3_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> E Pocket</h3><div class="NLM_p last">The E pocket accommodates type IV inhibitors and activators.<a onclick="showRef(event, 'ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref37 ref38">(37,38)</a> It represents one of the first allosteric sites in kinases that were identified. In 2003, the structure of a complex formed by c-ABL kinase and an endogenous myristoyl inhibitor was determined revealing the E pocket (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1OPL">1OPL</a>),<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> which has since then been termed the myristoyl pocket. While myristoyl acts as an inhibitor, activators can also bind to this site.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> The inhibitory mechanism of action at this site is well understood. Bound ligands closely interact with the αI′-helix. As a consequence, helix bending may occur, which then leads to autoinhibition by the SH2 domain of the kinase. However, not all bound ligands induce helix bending. In the absence of induced helix bending, the kinase remains in an active conformation, and bound activators can further promote catalytic activity. A total of three inhibitors and four activators were identified to bind to this pocket. In addition to c-ABL, this pocket was also detected in CDK2. Asciminib was the first allosteric BCR-ABL1 inhibitor targeting the myristoyl pocket, which currently is in a phase 3 clinical trial for the treatment of chronic myelogenous leukemia.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a></div></div><div id="sec3_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> F Pocket</h3><div class="NLM_p last">This pocket has been structurally confirmed and first described in 2013.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> It was then identified in four AMP-dependent kinases at the interface between the kinase domain of the α-subunit and the carbohydrate binding region of the β-subunit. Inhibitors binding at this site trigger conformational changes that prohibit α-/β-subunit interactions.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> Structures with 25 type IV inhibitors or activators have been determined so far. Activators binding to this pocket are thought to stabilize the association of regulatory subunits.</div></div><div id="sec3_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> G Pocket</h3><div class="NLM_p last">This pocket was first described in 2009 for CHK1.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> It was reported that ligands binding at this site had inhibitory effects but did not induce detectable conformational changes.<a onclick="showRef(event, 'ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref42 ref43">(42,43)</a> In our structural survey, we also identified this pocket in kinase PDK1. Six different compounds were found to bind to this site.</div></div><div id="sec3_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> H Pocket</h3><div class="NLM_p last">The H pocket proximal to the ATP site is the primary target of bivalent (type V) inhibitors, 23 of which have so far been characterized. In addition, structures with 25 primarily fragment-like type IV inhibitors bound to this pocket are available. This site was first described in 2017 and termed αD-pocket<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> because its presence depends on the position of the αD-helix.</div></div><div id="sec3_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> J Pocket</h3><div class="NLM_p last">This pocket was first described in 2017 for AURKA,<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> and structures with eight type IV inhibitors targeting this site have been determined. Similar to the F pocket, the J pocket maps to the top of the catalytic domain, albeit to the back side, rather than the front (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). Inhibitors binding to both of these pockets are implicated in similar allosteric mechanisms preventing dimerization with regulatory subunits.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> The J pocket was also detected in another kinase (BTK).</div></div><div id="sec3_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> I, K, and L Pockets</h3><div class="NLM_p last">Different from some allosteric binding sites that have been intensely investigated, as discussed above, only little information is available for pockets I, K, and L, which have been first described two to three years ago. So far, each pocket has only been observed in a single kinase including CDK2 (I), AURKA (K), and NTRK1 (L) and structures with only one or two inhibitors are available. For the I pocket, both of these inhibitors were covalently bound.</div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18">4.  Induced Allosteric Sites</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10611" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10611" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">An important question concerning allosteric kinase inhibition with immediate implications for drug design is whether allosteric binding sites are preformed in kinases or whether they are induced upon ligand binding. To address this question, we have compared structures of apo and holo forms of kinases with type IV (or VI) inhibitors. A summary of the results is reported in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>. We generally observed that allosteric pockets were often rather shallow, which complicated the assessment of conformational changes in some instances. For example, pocket C (or PIF pocket<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a>) is a shallow pocket that was suggested to be conserved across AGC kinases, a family with ∼60 members, on the basis of sequence comparisons.<a onclick="showRef(event, 'ref32 ref47'); return false;" href="javascript:void(0);" class="ref ref32 ref47">(32,47)</a> Structural comparison of complexes (5DN3, 5DOS, 5ORR) and corresponding apo forms of kinases (5DNR, 1MQ4, 6HJJ) revealed the presence of this pocket in the absence of bound compounds and confirmed additional conformational changes upon binding. These changes included a rotation of the αC-helix and reorientation of side chains in the binding site. By contrast, pocket D, which is located in the C-terminal lobe between the MAP kinase insert (MKI) loop and the αEF-/αF-helices, is deep. In this case, structural comparisons (complexes: 2YIS, 4E8A, 5N67; apo forms: 2YIX, 5MZ3, 5TBE) also revealed the presence of a preformed pocket in this region whose entrance was obstructed by the side chain of residue Trp197. Furthermore, large conformational changes occurred upon ligand binding in the MKI loop and the αEF-/αF- helices. Induced conformational changes were also reported for backbone segments of the MKI loop and αEF-/αF-helices as well as a side chain reorientation of the catalytically relevant Trp197 residue.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Similar observations were made for the deep E pocket in the C-terminal lobe (complexes: 3PYY, 2FO0, 5MO4, 6NPE; apo: 3QRJ, 4TWP, 2HZI) that interfaces with the αI′-helix. In this case, the presence of a pocket was also evident in the uncomplexed structures. Some ligands including the endogenous myristoyl as well as synthetic compounds induce a large conformational change in the αI′-helix causing a bend beginning at residue Phe516 that triggers autoinhibition.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Other ligands stabilize the native conformation of the helix providing a basis for activator effects.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a></div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Ligand-Induced Conformational Changes in Allosteric Binding Sites<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">pocket</th><th class="colsep0 rowsep0" align="center">location</th><th class="colsep0 rowsep0" align="center">representative kinase</th><th class="colsep0 rowsep0" align="center">complex structures</th><th class="colsep0 rowsep0" align="center">Apo form structures</th><th class="colsep0 rowsep0" align="center">ligand-induced changes</th><th class="colsep0 rowsep0" align="center">pocket depth</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">N-terminal lobe</td><td class="colsep0 rowsep0" align="left">AURKA</td><td class="colsep0 rowsep0" align="left">5DN3, 5DOS, 5ORR</td><td class="colsep0 rowsep0" align="left">5DNR, 1MQ4, 6HJJ</td><td class="colsep0 rowsep0" align="left">rotation of αC-helix, reorientation of side chains</td><td class="colsep0 rowsep0" align="left">shallow</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">D</td><td class="colsep0 rowsep0" align="left">C-terminal lobe, inside MKI loop</td><td class="colsep0 rowsep0" align="left">MAPK14</td><td class="colsep0 rowsep0" align="left">2YIS, 4E8A, 5N67</td><td class="colsep0 rowsep0" align="left">3QRJ, 4TWP, 2HZI</td><td class="colsep0 rowsep0" align="left">move of MKI loop and the αEF/αF helices, Trp197 reorientation</td><td class="colsep0 rowsep0" align="left">deep</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">E</td><td class="colsep0 rowsep0" align="left">C-terminal lobe, distal to active site</td><td class="colsep0 rowsep0" align="left">ABL1</td><td class="colsep0 rowsep0" align="left">3PYY, 2FO0, 5MO4, 6NPE</td><td class="colsep0 rowsep0" align="left">3QRJ, 4TWP, 2HZI</td><td class="colsep0 rowsep0" align="left">bending of αI′-helix</td><td class="colsep0 rowsep0" align="left">deep</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">F</td><td class="colsep0 rowsep0" align="left">N-terminal lobe, distal to active site</td><td class="colsep0 rowsep0" align="left">CSNK2A1</td><td class="colsep0 rowsep0" align="left">5CSH, 6FVF, 5MOE</td><td class="colsep0 rowsep0" align="left">5CVH, 5MO8, 4GUB</td><td class="colsep0 rowsep0" align="left">changes in β4-β5 loop, Gln36 side chain</td><td class="colsep0 rowsep0" align="left">shallow</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">G</td><td class="colsep0 rowsep0" align="left">C-terminal lobe, proximal to H pocket</td><td class="colsep0 rowsep0" align="left">CHK1</td><td class="colsep0 rowsep0" align="left">3F9N, 3JVS</td><td class="colsep0 rowsep0" align="left">3NLB, 3OT3</td><td class="colsep0 rowsep0" align="left">none detected or reported</td><td class="colsep0 rowsep0" align="left">very shallow</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">C-terminal lobe, proximal to ATP site</td><td class="colsep0 rowsep0" align="left">CSNK2A1</td><td class="colsep0 rowsep0" align="left">5CS6, 5CU3, 5CT0</td><td class="colsep0 rowsep0" align="left">5CSV, 5CVH, 6L22</td><td class="colsep0 rowsep0" align="left">move of αD-helix</td><td class="colsep0 rowsep0" align="left">deep</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">I</td><td class="colsep0 rowsep0" align="left">C-terminal lobe, entrance to D pocket</td><td class="colsep0 rowsep0" align="left">CDK2</td><td class="colsep0 rowsep0" align="left">5OO0, 5OSJ</td><td class="colsep0 rowsep0" align="left">5OO1, 5OSM</td><td class="colsep0 rowsep0" align="left">none detected or reported</td><td class="colsep0 rowsep0" align="left">very shallow</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">J</td><td class="colsep0 rowsep0" align="left">N-terminal lobe, proximal to F site</td><td class="colsep0 rowsep0" align="left">BTK</td><td class="colsep0 rowsep0" align="left">6X3N, 6BKW</td><td class="colsep0 rowsep0" align="left">6BLN, 5BQ0</td><td class="colsep0 rowsep0" align="left">none detected or reported</td><td class="colsep0 rowsep0" align="left">very shallow</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">K</td><td class="colsep0 rowsep0" align="left">back side of active cleft</td><td class="colsep0 rowsep0" align="left">AURKA</td><td class="colsep0 rowsep0" align="left">5ORL, 5LXM</td><td class="colsep0 rowsep0" align="left">6C2R, 3LAU</td><td class="colsep0 rowsep0" align="left">none detected or reported</td><td class="colsep0 rowsep0" align="left">very shallow</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">L</td><td class="colsep0 rowsep0" align="left">N-terminal lobe, between pocket F and ATP site</td><td class="colsep0 rowsep0" align="left">NTRK1</td><td class="colsep0 rowsep0" align="left">6D1Z, 6D20</td><td class="colsep0 rowsep0" align="left">6D22, 6D1Y</td><td class="colsep0 rowsep0" align="left">none detected or reported</td><td class="colsep0 rowsep0" align="left">very shallow</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">A summary of compound-induced conformational changes in binding sites of type IV (and VI) inhibitors is provided that were identified by comparing X-ray structures of kinase apo and holo forms augmented by literature searches, as discussed in the text.</p></div></div></div><div class="NLM_p">Different from D and E, pocket F in the N-terminal lobe of the catalytic kinase domain is relatively shallow, but also visible in apo form structures. Structural comparison (complexes: 5CSH, 6FVF, 5MOE; apo: 5CVH, 5MO8, 4GUB) revealed large conformational changes in the β4-β5 loop (Asp103-Thr108) of CK2α kinase as well as a side chain rotamer change of residue Gln36. It was reported previously that the interaction between the CK2α and CK2β subunits was essential for substrate selectivity and that ligands bound at this allosteric site prevented this subunit interaction.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> In addition to the changes described above, inhibitor binding at this site also induced a conformational change in the distal αD-helix adjacent to the ATP site.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a></div><div class="NLM_p">Pocket H is another deep pocket in the C-terminal lobe proximal to the ATP pocket. In this case, no evidence for this pocket was observed in the absence of bound ligands, whereas structural comparison (complexes: 5CS6, 5CU3, 5CT0; apo: 5CSV, 5CVH, 6L22) confirmed a large conformational change in the αD-helix (Asn117-Leu128) as a consequence of compound binding, inducing the formation of the H pocket. Previously, ligand binding was found to force the displacement of residue Tyr125 from this site, releasing the αD-helix from the C-lobe and effectively opening this pocket.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a></div><div class="NLM_p">The G, I, J, K, and L pockets were very shallow and only limited structural information was available, as discussed above. In these cases, structural comparisons did not yield evidence for preformation of these pockets or for significant conformational changes occurring upon ligand binding. Such effects were not reported in the literature.</div><div class="NLM_p last">Only three of the binding sites of type IV inhibitors represented deep pockets (D, E, and H). From structural comparisons, the picture emerged that the deep pockets D and E as well as the comparably shallow pockets C and F were partly preformed in kinase structures. However, they were substantially altered or extended through conformational changes induced by ligand binding. The deep H pocket was the only one that we classified as fully induced by compound binding because there was no evidence for its formation in apo structures. The remaining pockets had only little depth, and in these instances, no firm conclusions concerning preformation or ligand-based induction could be drawn on the basis of our analysis. It is conceivable that these very shallow allosteric pockets only form upon ligand binding, but associated conformational effects were hardly detectable.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">5.  Distribution of Allosteric Pockets, Inhibitors, and Multisite Ligands</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00661" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00661" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The distribution of allosteric pockets across the human kinome was determined, as shown in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>. Allosteric binding sites were detected in all major kinase groups with the exception of the CK1 group, indicating that allosteric compounds could be identified for many different kinases. Kinases from the TKL and STE groups were found to exclusively contain the B pocket adjacent to the ATP site hosting type III and VI inhibitors. Two Tyr kinases and five Ser/Thr kinases contained multiple allosteric pockets including binding sites for type III, IV, V, or VI compounds. Furthermore, a total of four kinases from different groups were found to contain at least three different allosteric sites in different combinations including TRKA (B, F, and L pockets), AURKA (C, J, K), CDK2 (B, E, I), and CK2a1 (B, D, F, H), which had four different allosteric pockets, the largest number we detected.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/medium/jm0c02076_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/large/jm0c02076_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Distribution of allosteric pockets. Kinases for which X-ray structures of complexes with ligands occupying allosteric binding sites were available are indicated as color-coded circles in a phylogenetic tree representation of the human kinome. Each kinase is numbered, and its name is reported below the tree representation. The resulting map reveals the distribution of allosteric pockets across the kinome.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/large/jm0c02076_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02076&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We also determined the distribution of allosteric compounds and different properties across the binding pockets. <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A reports the release dates for individual X-ray structures containing allosteric inhibitors, and <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B shows the distribution of their crystallographic resolution. There were no significant differences in resolution between these structures. In <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C, the size (molecular weight) distribution of the allosteric inhibitors is reported. The largest compounds were type III/VI inhibitors occupying the B pocket with a median molecular weight >450 Da. Type IV/VI inhibitors distributed across the C–G pockets were generally small, with a median weight <400 Da. However, for all of these pockets, the size of bound compounds significantly varied. Largest variations were observed for compounds occupying the F pocket, followed by the G pocket, with median values close to 400 Da. Interestingly, bivalent inhibitors binding to the H+A sites had molecular weights comparable to many type III/VI inhibitors of the B pocket. By contrast, compounds binding to the H, I, J, and K pockets were much smaller, with molecular weights between 200 and 300 Da, and often fragment-like. As discussed above, these compounds occupied the shallowest allosteric pockets. <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D reports the corresponding distribution of binding efficiency index values for inhibitors for which potency values were available, revealing that allosteric inhibitors of pocket D had particularly high ligand efficiency. Despite size variations, the ligand efficiency of inhibitors targeting the other pockets was comparable.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/medium/jm0c02076_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/large/jm0c02076_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Size distribution of allosteric compounds. (A) The years are reported when individual X-ray structures containing an allosteric ligand targeting each pocket were released. In addition, boxplots report for each pocket the distribution of (B) crystallographic resolution (Å) of X-ray structures with allosteric inhibitors, (C) molecular weights of allosteric inhibitors, and (D) binding efficiency index (BEI)<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> values for allosteric inhibitors. At the top of each boxplot (D), the number of inhibitors is reported for which defined potency values were available.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/large/jm0c02076_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02076&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Rather unexpectedly, we also identified 26 multisite ligands that bound to distinct allosteric pockets or the ATP site in the same or different X-ray structures. Nine of these compounds bound to the ATP site and one or two of the allosteric pockets, eight others bound to the H+A sites (bivalent ligands) and one of or both of the allosteric F or D pockets, and nine others to diverse allosteric pockets. Seven of the latter compounds bound to the F and H pockets, indicating that these pockets were capable of accommodating the same (or similar) ligands.</div><div class="NLM_p"><a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a> shows two examples of multisite ligands. On the left, an inhibitor is shown that bound to the ATP site and the B (type III) and C (type IV) pocket in different structures and kinases (DAPK1, CDK2), thus representing an exemplary “promiscuous allosteric” kinase inhibitor. This compound was found in six different X-ray structures. In these structures, it occupied the ATP site twice and the B and C pockets five times. On the right in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>, an inhibitor is shown that simultaneously bound to the ATP site and the D pocket in a single complex structure, hence providing an example for a multisite ligand displaying a mixed competitive/allosteric inhibitory phenotype.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/medium/jm0c02076_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/large/jm0c02076_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Exemplary multisite ligands. On the left, a kinase inhibitor is shown that bound to multiple sites (and site combinations) in different X-ray structures. Two distinct kinase targets are superimposed and displayed in ribbon representations (cyan, DAPK1; gray, CDK2). On the right, another kinase inhibitor is shown that occupied both the A (ATP) and D (allosteric) pocket in MAPK14.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/large/jm0c02076_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02076&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec6" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i21">6.  Allosteric Inhibitors with Privileged Substructures</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47335" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47335" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">To further characterize allosteric kinase inhibitors, we examined our collection for the presence of privileged substructures from medicinal chemistry using the compendium of Welsch et al.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> complemented with quinazoline<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> and 5<i>H</i>-dibenzo[b,e][1,4]diazepine.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> Substructure search calculations were carried out with the aid of the RDKit toolkit.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> At least one privileged substructure was found to be embedded in 88 allosteric inhibitors. Representative examples are shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>.<named-content content-type="anchor" rid="fig5" type="simple"></named-content></div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/medium/jm0c02076_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/large/jm0c02076_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Allosteric inhibitors with privileged substructures. Shown are six allosteric inhibitors with activity against different kinases. For each compound, the PDB ligand ID is provided, and privileged substructures are colored red.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/large/jm0c02076_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02076&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The frequent occurrence of privileged substructures in allosteric kinase inhibitors illustrates that structural novelty or uniqueness is not a prerequisite for compounds acting by specific allosteric mechanisms. Different from privileged substructures, pan-assay interference motifs (i.e., substructures with potential assay liabilities)<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> were only detected in four confirmed allosteric kinase ligands.</div></div><div id="sec7" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i22">7.  Structural Analogues</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82564" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82564" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We also searched for structural analogues of allosteric kinase inhibitors from X-ray structures in ChEMBL<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> (version 26). Activators were removed as well as inhibitors with a molecular weight <250 Da (hence excluding fragment-like compounds from further consideration), resulting in a set of 169 allosteric inhibitors. For these compounds, analogue searching was facilitated with an in-house variant of a fragmentation algorithm for the identification of matched molecular pairs.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> Analogues with modifications at one or two substitution sites were considered. For 130 allosteric inhibitors, 907 structural analogues were identified in ChEMBL. Reliable activity data were available for 485 of these ChEMBL analogues (i.e., compounds tested in high-confidence assays against single human targets with numerically specified standard measurements such as IC<sub>50</sub>, <i>K</i><sub>i</sub>, or <i>K</i><sub>d</sub> values, applying a potency threshold of 10 μM).</div><div class="NLM_p last">Notably, 114 analogue pairs were exclusively formed by single-pocket allosteric kinase inhibitors. In 92 out of these cases, inhibitors forming a pair were found to occupy the same allosteric pocket, thus confirming that structurally analogous inhibitors acted by the same (or very similar) allosteric mechanisms.<named-content content-type="anchor" rid="tbl4" type="simple"></named-content></div></div><div id="sec8" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i23">8.  Target Annotations</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_72122" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_72122" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">For the 169 allosteric inhibitors and 485 ChEMBL analogues, all high-confidence target annotations were collected, yielding a total of 115 targets.</div><div class="NLM_p">For 14 of the 169 allosteric inhibitors, high-confidence activity annotations for additional targets (other than their crystallographic target) were available in ChEMBL (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). These annotations included 24 kinases and, in addition, nine nonkinase targets for four of the 14 inhibitors. Among others, these targets included G-protein coupled receptors such as the histamine H1 receptor and muscarinic acetylcholine receptor M1. As reported in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>, allosteric kinase inhibitors with multiple confirmed targets included five promiscuous compounds with five or more reported targets, one of which was active against both kinase and nonkinase targets.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Allosteric Kinase Inhibitors with Multiple Targets<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/medium/jm0c02076_0006.gif" alt="" id="fx1" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">Shown are allosteric kinase inhibitors for which additional high-confidence targets were available in ChEMBL. For each inhibitor, the total number of targets is reported (including kinase targets given in parentheses). For targets, UniProt abbreviations are used (nonkinase targets are given in bold).</p></div></div><div></div></div><div class="NLM_p">Close structural analogues are likely to share similar activities. Accordingly, ChEMBL analogues of allosteric inhibitors with high-confidence target annotations were of particular interest. Therefore, the 169 allosteric inhibitors and the 485 ChEMBL analogues were combined and structural and target relationships were systematically analyzed in a pseudotripartite network.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> In this network representation, allosteric inhibitors, ChEMBL compounds, and target proteins were represented by three different types of nodes. Different types of edges were drawn between compounds that formed analogue pairs and between compounds and their experimentally confirmed targets. Allosteric inhibitors without structural analogues were included in the network on the basis of their target annotations. <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a> shows representative sections of this network representation. For 37 allosteric inhibitors, additional plausible target hypotheses were deduced on the basis of analogue relationships, which yielded a total of 75 potential kinase and nonkinase targets. Among these were seven members of the G-protein coupled receptor 1 family. Hence, for a subset of allosteric kinase inhibitors, additional targets were confirmed (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>) or are likely.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/medium/jm0c02076_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/large/jm0c02076_0010.jpeg" id="GRAPHIC-d7e1161-autogenerated" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Compound/target network analysis. Shown are network sections capturing multiple analogues and their compound-target relationships. Red and green circular nodes represent ChEMBL compounds and allosteric inhibitors, respectively, and triangular nodes represent targets. Pink and blue triangular nodes indicate kinase and nonkinase targets, respectively. Three types of edges are distinguished including edges between allosteric compounds and their crystallographic kinase targets (solid black lines), any compounds and their target(s) (solid gray), and compounds forming analogue pairs (dashed gray). Allosteric kinase inhibitors from X-ray structures are labeled with their PDB ligand IDs. In the inset at the bottom defining nodes and edges, CPD stands for compound.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/large/jm0c02076_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02076&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec9" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i25">9.  Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05437" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05437" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Compounds acting by allosteric mechanisms are of high interest in the quest for selective kinase inhibition and often considered prime candidates for drug development. The discovery of such inhibitors is still serendipitous in many instances. Nonetheless, in recent years, increasing numbers of allosteric kinase inhibitors have been reported. The assignment and elucidation of allosteric mechanisms is far from trivial and prone to errors. Experimentally determined structures of kinase-inhibitor complexes provide a sound basis for confirming allosteric inhibitors and their binding sites. Complex structures have preferentially been studied on a case-by-case basis, yielding incremental insights into allosteric sites and mechanisms.</div><div class="NLM_p">Our systematic survey of publicly available X-ray structures of complexes of kinases and designated allosteric ligands identified 232 unique compounds including allosteric inhibitors and 19 kinase activators. Mapping of the binding sites of these compounds revealed 11 allosteric sites. These sites were found in varying numbers of kinases and hosted different numbers of ligands. Some allosteric pockets and associated mechanisms of action were well characterized in the literature such as the B pocket (type III/VI inhibitors) or E pocket (type IV inhibitors and activators). By contrast, for other binding sites we mapped, only little information was available. Structurally analogous allosteric inhibitors mostly bound to the same pockets, indicating the presence of specific mechanisms. At the same time, we also identified a number of multisite ligands that were found in different pockets or combinations of pockets. Furthermore, analysis of high-confidence activity data and structural analogues identified a number of confirmed or likely additional targets for a confined subset of allosteric kinase inhibitors. Hence, selectivity of allosteric ligands could not be generalized and there was considerable heterogeneity in their binding characteristics. Since many allosteric kinase inhibitors contained privileged substructures that also occurred in compounds with other activities, such as ligands of G-protein coupled receptors, additional activities of such allosteric compounds might be anticipated.</div><div class="NLM_p">For drug design, an important question is whether or not allosteric binding sites are induced upon compound binding or whether they can be targeted in structures of uncomplexed kinases. On the basis of structural comparisons, we found that several allosteric pockets were preformed in kinases but significantly altered and extended upon ligand binding. This provides a difficult scenario for “static” structure-based drug design approaches and indicates the need to complement drug design with an extensive assessment of protein dynamics in such regions. On the other hand, allosteric pockets were often rather shallow, making it difficult to obtain evidence for defined conformational changes upon ligand binding.</div><div class="NLM_p last">Taken together, the findings presented herein provide a refined structure-based view of allosteric kinase inhibition and should encourage further structural and computational investigations of allosteric mechanisms as well as a comparative assessment of structure–activity relationships among allosteric kinase inhibitors.</div></div><div class="NLM_back"><div class="article_supporting-info" id=""></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c02076" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21134" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21134" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jürgen Bajorath</span> - <span class="hlFld-Affiliation affiliation">Department of Life Science
Informatics, B-IT, LIMES Program Unit Chemical Biology and Medicinal
Chemistry, Rheinische Friedrich-Wilhelms-Universität, Friedrich-Hirzebruch-Allee 6, D-53115 Bonn, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0557-5714" title="Orcid link">http://orcid.org/0000-0002-0557-5714</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#6507040f0a1704110d25070c114b100b0c48070a0b0b4b0100"><span class="__cf_email__" data-cfemail="87e5e6ede8f5e6f3efc7e5eef3a9f2e9eeaae5e8e9e9a9e3e2">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Oliver Laufkötter</span> - <span class="hlFld-Affiliation affiliation">Department of Life Science
Informatics, B-IT, LIMES Program Unit Chemical Biology and Medicinal
Chemistry, Rheinische Friedrich-Wilhelms-Universität, Friedrich-Hirzebruch-Allee 6, D-53115 Bonn, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Huabin Hu</span> - <span class="hlFld-Affiliation affiliation">Department of Life Science
Informatics, B-IT, LIMES Program Unit Chemical Biology and Medicinal
Chemistry, Rheinische Friedrich-Wilhelms-Universität, Friedrich-Hirzebruch-Allee 6, D-53115 Bonn, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Filip Miljković</span> - <span class="hlFld-Affiliation affiliation">Department of Life Science
Informatics, B-IT, LIMES Program Unit Chemical Biology and Medicinal
Chemistry, Rheinische Friedrich-Wilhelms-Universität, Friedrich-Hirzebruch-Allee 6, D-53115 Bonn, Germany</span>; 
    <span>Present Address:
                        F.M.: Data Science & Artificial Intelligence, Imaging & Data Analytics, Clinical Pharmacology & Safety Sciences, R&D, AstraZeneca Gothenburg, 43183 Mölndal, Sweden</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>O.L., H.H.: These authors have contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="bio" rid="ath1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="biographies">Biographies</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37703" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37703" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><figure id="ath1" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Oliver Laufkötter</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02076&amp;id=BIO-d7e1176-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Oliver Laufkötter</b> obtained his bachelor’s degree in chemistry and biochemistry from the university of Canterbury, Christchurch, New Zealand, in 2012. He completed his master’s degree in drug research at the Pharmaceutical Institute of the University of Bonn in 2018 and began his doctoral studies in the Department of Life Science Informatics, Bonn-Aachen International Center for Information Technology (B-IT), University of Bonn, Germany, under the supervision of Prof. Dr. Jürgen Bajorath. His current work focuses on compound data-driven studies of kinase promiscuity and large-scale analysis of kinase inhibitors.</p></figure></div><div class="bio" rid="ath2"><figure id="ath2" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Huabin Hu</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02076&amp;id=BIO-d7e1181-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Huabin Hu</b> obtained his M.D. degree in medicinal chemistry from Peking Union Medical College in 2017. He then joined the Department of Life Science Informatics at the University of Bonn as a Ph.D. student under the supervision of Prof. Jürgen Bajorath. Currently, his work mainly focuses on large-scale investigation of activity cliffs and structure–activity relationships.</p></figure></div><div class="bio" rid="ath3"><figure id="ath3" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Filip Miljković</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02076&amp;id=BIO-d7e1186-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Filip Miljković</b> obtained a master’s degree in pharmacy from the Medical Faculty, University of Niš, Serbia in 2014. In 2016 he began doctoral studies in Computational Life Sciences at the Department of Life Science Informatics, Bonn-Aachen International Center for Information Technology (B-IT), University of Bonn, Germany, under the supervision of Prof. Dr. Jürgen Bajorath. He obtained his Ph.D. degree in 2019 for exploring and developing chemoinformatics approaches for kinase drug discovery. He then joined AstraZeneca R&D as a Data Scientist at the Department of Clinical Pharmacology & Safety Sciences in Gothenburg, Sweden. His research interests include chemoinformatics and computational methodologies for medicinal chemistry and chemical biology, including machine learning and structure-based approaches. For further details, visit: <a href="https://filipm90.github.io/" class="extLink">https://filipm90.github.io/</a>.</p></figure></div><div class="bio" rid="ath4"><figure id="ath4" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Jürgen Bajorath</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02076&amp;id=BIO-d7e1194-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Jürgen Bajorath</b> is Professor and Chair of Life Science Informatics at the University of Bonn and an Affiliate Professor in the Department of Biological Structure at the University of Washington, Seattle. His research interests include computer-aided medicinal chemistry, molecular promiscuity, advanced machine learning concepts, and drug discovery. For further details, see: <a href="http://bajorath.bit.uni-bonn.de" class="extLink">http://bajorath.bit.uni-bonn.de</a>.</p></figure></div><div class="ack" id="ACK-d7e1205-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i32">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_60268" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_60268" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">H.H. is supported by a fellowship from the China Scholarship Council.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">CPD</td><td class="NLM_def"><p class="first last">compound</p></td></tr><tr><td class="NLM_term">Da</td><td class="NLM_def"><p class="first last">Daltons</p></td></tr><tr><td class="NLM_term">FDA</td><td class="NLM_def"><p class="first last">Food and Drug Administration</p></td></tr><tr><td class="NLM_term">ID</td><td class="NLM_def"><p class="first last">identifier</p></td></tr><tr><td class="NLM_term">MKI</td><td class="NLM_def"><p class="first last">MAP kinase insert</p></td></tr><tr><td class="NLM_term">PDB</td><td class="NLM_def"><p class="first last">Protein Data Bank</p></td></tr><tr><td class="NLM_term">PIF</td><td class="NLM_def"><p class="first last">PDK1-interacting fragment</p></td></tr><tr><td class="NLM_term">UPID</td><td class="NLM_def"><p class="first last">UniProt ID</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i34">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_83212" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_83212" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 58 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ward, R. A.</span>; <span class="NLM_string-name">Goldberg, F. W.</span></span>, Eds.;  <i>Kinase Drug Discovery</i>; <span class="NLM_publisher-name">RSC</span>: <span class="NLM_publisher-loc">Cambridge</span>, <span class="NLM_year">2011</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1039%2F9781849733557" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=R.+A.+Ward&author=F.+W.+Goldberg&title=Kinase+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1039%2F9781849733557&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1039%252F9781849733557%26sid%3Dliteratum%253Aachs%26aulast%3DWard%26aufirst%3DR.%2BA.%26btitle%3DKinase%2520Drug%2520Discovery%26pub%3DRSC%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ward, R. A.</span>; <span class="NLM_string-name">Goldberg, F. W.</span></span>, Eds.;  <i>Kinase Drug Discovery: Modern Approaches</i>; <span class="NLM_publisher-name">RSC</span>: <span class="NLM_publisher-loc">Cambridge</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1039%2F9781788013093" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=R.+A.+Ward&author=F.+W.+Goldberg&title=Kinase+Drug+Discovery%3A+Modern+Approaches"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1039%2F9781788013093&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1039%252F9781788013093%26sid%3Dliteratum%253Aachs%26aulast%3DWard%26aufirst%3DR.%2BA.%26btitle%3DKinase%2520Drug%2520Discovery%253A%2520Modern%2520Approaches%26pub%3DRSC%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Kinase Inhibitors: The Road Ahead</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">353</span>– <span class="NLM_lpage">376</span>, <span class="refDoi"> DOI: 10.1038/nrd.2018.21</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1038%2Fnrd.2018.21" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=29545548" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC1cXks12ktbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=353-376&author=F.+M.+Fergusonauthor=N.+S.+Gray&title=Kinase+Inhibitors%3A+The+Road+Ahead&doi=10.1038%2Fnrd.2018.21"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase inhibitors: the road ahead</span></div><div class="casAuthors">Ferguson, Fleur M.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">353-377</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Receptor tyrosine kinase signaling pathways have been successfully targeted to inhibit proliferation and angiogenesis for cancer therapy.  However, kinase deregulation has been firmly demonstrated to play an essential role in virtually all major disease areas.  Kinase inhibitor drug discovery programs have recently broadened their focus to include an expanded range of kinase targets and therapeutic areas.  In this Review, we provide an overview of the novel targets, biol. processes and disease areas that kinase-targeting small mols. are being developed against, highlight the assocd. challenges and assess the strategies and technologies that are enabling efficient generation of highly optimized kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEZPospb1TP7Vg90H21EOLACvtfcHk0lg3gyqg2yIGdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXks12ktbg%253D&md5=34e42911d8a70301905460c3507e737a</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2018.21&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2018.21%26sid%3Dliteratum%253Aachs%26aulast%3DFerguson%26aufirst%3DF.%2BM.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DKinase%2520Inhibitors%253A%2520The%2520Road%2520Ahead%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2018%26volume%3D17%26spage%3D353%26epage%3D376%26doi%3D10.1038%2Fnrd.2018.21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Manning, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whyte, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunter, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sudarsanam, S.</span></span> <span> </span><span class="NLM_article-title">The Protein Kinase Complement of the Human Genome</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>298</i></span>,  <span class="NLM_fpage">1912</span>– <span class="NLM_lpage">1934</span>, <span class="refDoi"> DOI: 10.1126/science.1075762</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1126%2Fscience.1075762" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=12471243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BD38Xpt1Wisb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=298&publication_year=2002&pages=1912-1934&author=G.+Manningauthor=D.+Whyteauthor=R.+Martinezauthor=T.+Hunterauthor=S.+Sudarsanam&title=The+Protein+Kinase+Complement+of+the+Human+Genome&doi=10.1126%2Fscience.1075762"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Kinase Complement of the Human Genome</span></div><div class="casAuthors">Manning, G.; Whyte, D. B.; Martinez, R.; Hunter, T.; Sudarsanam, S.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">298</span>
        (<span class="NLM_cas:issue">5600</span>),
    <span class="NLM_cas:pages">1912-1916, 1933-1934</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">We have catalogued the protein kinase complement of the human genome (the "kinome") using public and proprietary genomic, complementary DNA, and expressed sequence tag (EST) sequences.  This provides a starting point for comprehensive anal. of protein phosphorylation in normal and disease states, as well as a detailed view of the current state of human genome anal. through a focus on one large gene family.  We identify 518 putative protein kinase genes, of which 71 have not previously been reported or described as kinases, and we extend or correct the protein sequences of 56 more kinases.  New genes include members of well-studied families as well as previously unidentified families, some of which are conserved in model organisms.  Classification and comparison with model organism kinomes identified orthologous groups and highlighted expansions specific to human and other lineages.  We also identified 106 protein kinase pseudogenes.  Chromosomal mapping revealed several small clusters of kinase genes and revealed that 244 kinases map to disease loci or cancer amplicons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTj1hf_0cQcrVg90H21EOLACvtfcHk0ljvJfLx7ifTew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xpt1Wisb0%253D&md5=b3def83bace52257a3252a3347b8bb92</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1126%2Fscience.1075762&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1075762%26sid%3Dliteratum%253Aachs%26aulast%3DManning%26aufirst%3DG.%26aulast%3DWhyte%26aufirst%3DD.%26aulast%3DMartinez%26aufirst%3DR.%26aulast%3DHunter%26aufirst%3DT.%26aulast%3DSudarsanam%26aufirst%3DS.%26atitle%3DThe%2520Protein%2520Kinase%2520Complement%2520of%2520the%2520Human%2520Genome%26jtitle%3DScience%26date%3D2002%26volume%3D298%26spage%3D1912%26epage%3D1934%26doi%3D10.1126%2Fscience.1075762" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berman, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westbrook, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilliland, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhat, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissig, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shindyalov, I. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourne, P. E.</span></span> <span> </span><span class="NLM_article-title">The Protein Data Bank</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1093/nar/28.1.235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1093%2Fnar%2F28.1.235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10592235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=235-242&author=H.+M.+Bermanauthor=J.+Westbrookauthor=Z.+Fengauthor=G.+Gillilandauthor=T.+N.+Bhatauthor=H.+Weissigauthor=I.+N.+Shindyalovauthor=P.+E.+Bourne&title=The+Protein+Data+Bank&doi=10.1093%2Fnar%2F28.1.235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank</span></div><div class="casAuthors">Berman, Helen M.; Westbrook, John; Feng, Zukang; Gilliland, Gary; Bhat, T. N.; Weissig, Helge; Shindyalov, Ilya N.; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Protein Data Bank (PDB; http://www.rcsb.org/pdb/)is the single worldwide archive of structural data of biol. macromols.  This paper describes the goals of the PDB, the systems in place for data deposition and access, how to obtain further information, and near-term plans for the future development of the resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpStGCxT8QzirVg90H21EOLACvtfcHk0ljvJfLx7ifTew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D&md5=227fb393f754be2be375ab727bfd05dc</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F28.1.235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F28.1.235%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DWestbrook%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGilliland%26aufirst%3DG.%26aulast%3DBhat%26aufirst%3DT.%2BN.%26aulast%3DWeissig%26aufirst%3DH.%26aulast%3DShindyalov%26aufirst%3DI.%2BN.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DThe%2520Protein%2520Data%2520Bank%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2000%26volume%3D28%26spage%3D235%26epage%3D242%26doi%3D10.1093%2Fnar%2F28.1.235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gavrin, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saiah, E.</span></span> <span> </span><span class="NLM_article-title">Approaches to Discover Non-ATP Site Kinase Inhibitors</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">41</span>– <span class="NLM_lpage">51</span>, <span class="refDoi"> DOI: 10.1039/C2MD20180A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1039%2FC2MD20180A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVymsr7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=41-51&author=L.+K.+Gavrinauthor=E.+Saiah&title=Approaches+to+Discover+Non-ATP+Site+Kinase+Inhibitors&doi=10.1039%2FC2MD20180A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Approaches to discover non-ATP site kinase inhibitors</span></div><div class="casAuthors">Gavrin, Lori Krim; Saiah, Eddine</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">41-51</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The catalytic domain of kinases shows a high degree of sequence homol., esp. for kinases that belong to the same family.  They share a common ATP binding site with a conserved activation loop and similar three-dimensional structure.  Consequently, a major challenge in kinase research exists in achieving selectivity among the >500 family members, since they all process the same substrate.  In addn. to requiring selectivity against other kinases, ATP site inhibitors must also bind tightly to overcome the high physiol. concn. of ATP in the cell.  Furthermore, the development of novel ATP site inhibitors is becoming increasingly challenging, as many ATP competitive scaffolds have previously been disclosed.  In order to develop compds. with better selectivity among kinases, inhibitors that bind outside the ATP site show great promise and are currently being explored by many groups.  This review will highlight the most commonly used methods to discover small mol. Type III and IV kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7xJcf4-RJ9rVg90H21EOLACvtfcHk0ljvJfLx7ifTew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVymsr7N&md5=97acf2247e5516c1649c1d964c867656</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1039%2FC2MD20180A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC2MD20180A%26sid%3Dliteratum%253Aachs%26aulast%3DGavrin%26aufirst%3DL.%2BK.%26aulast%3DSaiah%26aufirst%3DE.%26atitle%3DApproaches%2520to%2520Discover%2520Non-ATP%2520Site%2520Kinase%2520Inhibitors%26jtitle%3DMedChemComm%26date%3D2013%26volume%3D4%26spage%3D41%26epage%3D51%26doi%3D10.1039%2FC2MD20180A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abdeldayem, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raouf, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Constantinescu, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriggl, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunning, P. T.</span></span> <span> </span><span class="NLM_article-title">Advances in Covalent Kinase Inhibition</span>. <i>Chem. Soc. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">2617</span>– <span class="NLM_lpage">2687</span>, <span class="refDoi"> DOI: 10.1039/C9CS00720B</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1039%2FC9CS00720B" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=32227030" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BB3cXlvVygsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2020&pages=2617-2687&author=A.+Abdeldayemauthor=Y.+S.+Raoufauthor=S.+N.+Constantinescuauthor=R.+Morigglauthor=P.+T.+Gunning&title=Advances+in+Covalent+Kinase+Inhibition&doi=10.1039%2FC9CS00720B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in covalent kinase inhibitors</span></div><div class="casAuthors">Abdeldayem, Ayah; Raouf, Yasir S.; Constantinescu, Stefan N.; Moriggl, Richard; Gunning, Patrick T.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Society Reviews</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2617-2687</span>CODEN:
                <span class="NLM_cas:coden">CSRVBR</span>;
        ISSN:<span class="NLM_cas:issn">0306-0012</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Over the past decade, covalent kinase inhibitors (CKI) have seen a resurgence in drug discovery.  Covalency affords a unique set of advantages as well as challenges relative to their non-covalent counterpart.  After reversible protein target recognition and binding, covalent inhibitors irreversibly modify a proximal nucleophilic residue on the protein via reaction with an electrophile.  To date, the acrylamide group remains the predominantly employed electrophile in CKI development, with its incorporation in the majority of clin. candidates and FDA approved covalent therapies.  Nonetheless, in recent years considerable efforts have ensued to characterize alternative electrophiles that exhibit irreversible or reversibly covalent binding mechanisms towards cysteine thiols and other amino acids.  This review article provides a comprehensive overview of CKIs reported in the literature over a decade period, 2007-2018.  Emphasis is placed on the rationale behind warhead choice, optimization approach, and inhibitor design.  Current FDA approved CKIs are also highlighted, in addn. to a detailed anal. of the common trends and themes obsd. within the listed data set.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT7Unx-xJoarVg90H21EOLACvtfcHk0lhKKIHx0WHbbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXlvVygsr8%253D&md5=88c6e097cb0b3619a53ceb1e03d19971</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1039%2FC9CS00720B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC9CS00720B%26sid%3Dliteratum%253Aachs%26aulast%3DAbdeldayem%26aufirst%3DA.%26aulast%3DRaouf%26aufirst%3DY.%2BS.%26aulast%3DConstantinescu%26aufirst%3DS.%2BN.%26aulast%3DMoriggl%26aufirst%3DR.%26aulast%3DGunning%26aufirst%3DP.%2BT.%26atitle%3DAdvances%2520in%2520Covalent%2520Kinase%2520Inhibition%26jtitle%3DChem.%2520Soc.%2520Rev.%26date%3D2020%26volume%3D49%26spage%3D2617%26epage%3D2687%26doi%3D10.1039%2FC9CS00720B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamba, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, I.</span></span> <span> </span><span class="NLM_article-title">New Directions in Targeting Protein Kinases: Focusing Upon True Allosteric and Bivalent Inhibitors</span>. <i>Curr. Pharm. Des.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">2936</span>– <span class="NLM_lpage">2945</span>, <span class="refDoi"> DOI: 10.2174/138161212800672813</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.2174%2F138161212800672813" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=22571662" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFaltrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=2936-2945&author=V.+Lambaauthor=I.+Ghosh&title=New+Directions+in+Targeting+Protein+Kinases%3A+Focusing+Upon+True+Allosteric+and+Bivalent+Inhibitors&doi=10.2174%2F138161212800672813"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">New directions in targeting protein kinases: focusing upon true allosteric and bivalent inhibitors</span></div><div class="casAuthors">Lamba, Vandana; Ghosh, Indraneel</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">2936-2945</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Over the past decade, therapeutics that target subsets of the 518 human protein kinases have played a vital role in the fight against cancer.  Protein kinases are typically targeted at the ATP-binding cleft by type I and II inhibitors; however, the high sequence and structural homol. shared by protein kinases, esp. at the ATP-binding site, inherently leads to polypharmacol.  In order to discover or design truly selective protein kinase inhibitors as both pharmacol. reagents and safer therapeutic leads, new efforts are needed to target kinases outside the ATP cleft.  Recent advances include the serendipitous discovery of type III inhibitors that bind a site proximal to the ATP pocket as well as the truly allosteric type IV inhibitors that target protein kinases distal to the substrate-binding pocket.  These new classes of inhibitors are often selective, but usually display moderate affinities.  Here, the authors discuss the different classes of inhibitors with an emphasis on bisubstrate and bivalent inhibitors (type V) that combine different inhibitor classes.  These inhibitors have the potential to couple the high affinity and potency of traditional active site-targeted small-mol. inhibitors with the selectivity of inhibitors that target the protein kinase surface outside ATP cleft.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkIIgeMxhCrrVg90H21EOLACvtfcHk0lhKKIHx0WHbbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFaltrk%253D&md5=36a44e36709507b5053bfdfe7d09e5e9</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.2174%2F138161212800672813&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138161212800672813%26sid%3Dliteratum%253Aachs%26aulast%3DLamba%26aufirst%3DV.%26aulast%3DGhosh%26aufirst%3DI.%26atitle%3DNew%2520Directions%2520in%2520Targeting%2520Protein%2520Kinases%253A%2520Focusing%2520Upon%2520True%2520Allosteric%2520and%2520Bivalent%2520Inhibitors%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2012%26volume%3D18%26spage%3D2936%26epage%3D2945%26doi%3D10.2174%2F138161212800672813" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gower, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maly, D. J.</span></span> <span> </span><span class="NLM_article-title">Bivalent Inhibitors of Protein Kinases</span>. <i>Crit. Rev. Biochem. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">102</span>– <span class="NLM_lpage">115</span>, <span class="refDoi"> DOI: 10.3109/10409238.2013.875513</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.3109%2F10409238.2013.875513" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=24564382" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC2cXks1Smtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2014&pages=102-115&author=C.+M.+Gowerauthor=M.+E.+Changauthor=D.+J.+Maly&title=Bivalent+Inhibitors+of+Protein+Kinases&doi=10.3109%2F10409238.2013.875513"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Bivalent inhibitors of protein kinases</span></div><div class="casAuthors">Gower, Carrie M.; Chang, Matthew E. K.; Maly, Dustin J.</div><div class="citationInfo"><span class="NLM_cas:title">Critical Reviews in Biochemistry and Molecular Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">102-115</span>CODEN:
                <span class="NLM_cas:coden">CRBBEJ</span>;
        ISSN:<span class="NLM_cas:issn">1040-9238</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Protein kinases are key players in a large no. of cellular signaling pathways.  Dysregulated kinase activity has been implicated in a no. of diseases, and members of this enzyme family are of therapeutic interest.  However, due to the fact that most inhibitors interact with the highly conserved ATP-binding sites of kinases, it is a significant challenge to develop pharmacol. agents that target only one of the greater than 500 kinases present in humans.  A potential soln. to this problem is the development of bisubstrate and bivalent kinase inhibitors, in which an active site-directed moiety is tethered to another ligand that targets a location outside of the ATP-binding cleft.  Because kinase signaling specificity is modulated by regions outside of the ATP-binding site, strategies that exploit these interactions have the potential to provide reagents with high target selectivity.  This review highlights examples of kinase interaction sites that can potentially be exploited by bisubstrate and bivalent inhibitors.  Furthermore, an overview of efforts to target these interactions with bisubstrate and bivalent inhibitors is provided.  Finally, several examples of the successful application of these reagents in a cellular setting are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbnNuzK1xqRLVg90H21EOLACvtfcHk0lhKKIHx0WHbbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXks1Smtr8%253D&md5=66249a71e01aaedb02ef8ed00d27096f</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.3109%2F10409238.2013.875513&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F10409238.2013.875513%26sid%3Dliteratum%253Aachs%26aulast%3DGower%26aufirst%3DC.%2BM.%26aulast%3DChang%26aufirst%3DM.%2BE.%26aulast%3DMaly%26aufirst%3DD.%2BJ.%26atitle%3DBivalent%2520Inhibitors%2520of%2520Protein%2520Kinases%26jtitle%3DCrit.%2520Rev.%2520Biochem.%2520Mol.%2520Biol.%26date%3D2014%26volume%3D49%26spage%3D102%26epage%3D115%26doi%3D10.3109%2F10409238.2013.875513" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laufer, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">The Cysteinome of Protein Kinases as a Target in Drug Development</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">4372</span>– <span class="NLM_lpage">4385</span>, <span class="refDoi"> DOI: 10.1002/anie.201707875</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1002%2Fanie.201707875" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFGgs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=4372-4385&author=A.+Chaikuadauthor=P.+Kochauthor=S.+A.+Lauferauthor=S.+Knapp&title=The+Cysteinome+of+Protein+Kinases+as+a+Target+in+Drug+Development&doi=10.1002%2Fanie.201707875"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The Cysteinome of Protein Kinases as a Target in Drug Development</span></div><div class="casAuthors">Chaikuad, Apirat; Koch, Pierre; Laufer, Stefan A.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4372-4385</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Drugs that function through covalent bond formation represent a considerable fraction of the authors' repository of effective medicines but safety concerns and the complexity of developing covalent inhibitors has rendered covalent targeting a less attractive strategy for rational drug design.  The recent approval of four covalent kinase inhibitors and the development of highly potent covalent kinase probes with exceptional selectivity has raised significant interest in industry and academic research and validated the concept of covalent kinase targeting for clin. applications.  The abundance of cysteines at diverse positions in and around the kinase active site suggests that a large fraction of kinases can be targeted by covalent inhibitors.  Herein, the authors review recent developments of this rapidly growing area in kinase drug development and highlight the unique opportunities and challenges of this strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphHGXsIr-dpbVg90H21EOLACvtfcHk0ljGN-lV-cKqBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFGgs7c%253D&md5=311ba5ab53dc4a5dc1dc92ddd4b6d4c2</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1002%2Fanie.201707875&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201707875%26sid%3Dliteratum%253Aachs%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DKoch%26aufirst%3DP.%26aulast%3DLaufer%26aufirst%3DS.%2BA.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DThe%2520Cysteinome%2520of%2520Protein%2520Kinases%2520as%2520a%2520Target%2520in%2520Drug%2520Development%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26spage%3D4372%26epage%3D4385%26doi%3D10.1002%2Fanie.201707875" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gehringer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laufer, S. A.</span></span> <span> </span><span class="NLM_article-title">Emerging and Re-emerging Warheads for Targeted Covalent Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">5673</span>– <span class="NLM_lpage">5724</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01153</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01153" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFKiur%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=5673-5724&author=M.+Gehringerauthor=S.+A.+Laufer&title=Emerging+and+Re-emerging+Warheads+for+Targeted+Covalent+Inhibitors&doi=10.1021%2Facs.jmedchem.8b01153"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging and Re-Emerging Warheads for Targeted Covalent Inhibitors: Applications in Medicinal Chemistry and Chemical Biology</span></div><div class="casAuthors">Gehringer, Matthias; Laufer, Stefan A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5673-5724</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Targeted covalent inhibitors (TCIs) are designed to bind poorly conserved amino acids by means of reactive groups, the so-called warheads.  Currently, targeting noncatalytic cysteine residues with acrylamides and other α,β-unsatd. carbonyl compds. is the predominant strategy in TCI development.  The recent ascent of covalent drugs has stimulated considerable efforts to characterize alternative warheads for the covalent-reversible and irreversible engagement of noncatalytic cysteine residues as well as other amino acids.  This Perspective article provides an overview of warheads-beyond α,β-unsatd. amides-recently used in the design of targeted covalent ligands.  Promising reactive groups that have not yet demonstrated their utility in TCI development are also highlighted.  Special emphasis is placed on the discussion of reactivity and of case studies illustrating applications in medicinal chem. and chem. biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDhc4PWV293LVg90H21EOLACvtfcHk0ljGN-lV-cKqBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFKiur%252FL&md5=4623733558ff99aadebfc13950b816ad</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01153&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01153%26sid%3Dliteratum%253Aachs%26aulast%3DGehringer%26aufirst%3DM.%26aulast%3DLaufer%26aufirst%3DS.%2BA.%26atitle%3DEmerging%2520and%2520Re-emerging%2520Warheads%2520for%2520Targeted%2520Covalent%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D5673%26epage%3D5724%26doi%3D10.1021%2Facs.jmedchem.8b01153" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weisner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gontla, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Westhuizen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeck, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ketzer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janning, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richters, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muhlenberg, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taher, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jendrossek, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pelly, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Otterlo, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauh, D.</span></span> <span> </span><span class="NLM_article-title">Covalent-allosteric Kinase Inhibitors</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">10313</span>– <span class="NLM_lpage">10316</span>, <span class="refDoi"> DOI: 10.1002/anie.201502142</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1002%2Fanie.201502142" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVKms7fE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=10313-10316&author=J.+Weisnerauthor=R.+Gontlaauthor=L.+van+der%0AWesthuizenauthor=S.+Oeckauthor=J.+Ketzerauthor=P.+Janningauthor=A.+Richtersauthor=T.+Muhlenbergauthor=Z.+Fangauthor=A.+Taherauthor=V.+Jendrossekauthor=S.+C.+Pellyauthor=S.+Bauerauthor=W.+A.+van+Otterloauthor=D.+Rauh&title=Covalent-allosteric+Kinase+Inhibitors&doi=10.1002%2Fanie.201502142"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent-Allosteric Kinase Inhibitors</span></div><div class="casAuthors">Weisner, Joern; Gontla, Rajesh; van der Westhuizen, Leandi; Oeck, Sebastian; Ketzer, Julia; Janning, Petra; Richters, Andre; Muehlenberg, Thomas; Fang, Zhizhou; Taher, Abu; Jendrossek, Verena; Pelly, Stephen C.; Bauer, Sebastian; van Otterlo, Willem A. L.; Rauh, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">10313-10316</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Targeting and stabilizing distinct kinase conformations is an instrumental strategy for dissecting conformation-dependent signaling of protein kinases.  Herein the structure-based design, synthesis, and evaluation of pleckstrin homol. (PH) domain-dependent covalent-allosteric inhibitors (CAIs) of the kinase Akt is reported.  These inhibitors bind covalently to a distinct cysteine of the kinase and thereby stabilize the inactive kinase conformation.  These modulators exhibit high potency and selectivity, and represent an innovative approach for chem. biol. and medicinal chem. research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodImeERqKXoLVg90H21EOLACvtfcHk0ljGN-lV-cKqBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVKms7fE&md5=9502910ef46da0e7e7ae7d2156e0a292</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1002%2Fanie.201502142&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201502142%26sid%3Dliteratum%253Aachs%26aulast%3DWeisner%26aufirst%3DJ.%26aulast%3DGontla%26aufirst%3DR.%26aulast%3Dvan%2Bder%2BWesthuizen%26aufirst%3DL.%26aulast%3DOeck%26aufirst%3DS.%26aulast%3DKetzer%26aufirst%3DJ.%26aulast%3DJanning%26aufirst%3DP.%26aulast%3DRichters%26aufirst%3DA.%26aulast%3DMuhlenberg%26aufirst%3DT.%26aulast%3DFang%26aufirst%3DZ.%26aulast%3DTaher%26aufirst%3DA.%26aulast%3DJendrossek%26aufirst%3DV.%26aulast%3DPelly%26aufirst%3DS.%2BC.%26aulast%3DBauer%26aufirst%3DS.%26aulast%3Dvan%2BOtterlo%26aufirst%3DW.%2BA.%26aulast%3DRauh%26aufirst%3DD.%26atitle%3DCovalent-allosteric%2520Kinase%2520Inhibitors%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D54%26spage%3D10313%26epage%3D10316%26doi%3D10.1002%2Fanie.201502142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smaill, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, K.</span></span> <span> </span><span class="NLM_article-title">New Promise and Opportunities for Allosteric Kinase Inhibitors</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">13764</span>– <span class="NLM_lpage">13776</span>, <span class="refDoi"> DOI: 10.1002/anie.201914525</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1002%2Fanie.201914525" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BB3cXmt1Giurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2020&pages=13764-13776&author=X.+Luauthor=J.+B.+Smaillauthor=K.+Ding&title=New+Promise+and+Opportunities+for+Allosteric+Kinase+Inhibitors&doi=10.1002%2Fanie.201914525"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">New Promise and Opportunities for Allosteric Kinase Inhibitors</span></div><div class="casAuthors">Lu, Xiaoyun; Smaill, Jeff B.; Ding, Ke</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">33</span>),
    <span class="NLM_cas:pages">13764-13776</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Drugs that function through allosteric inhibition of kinase signaling represent a promising approach for the targeted discovery of therapeutics.  The majority of developed allosteric kinase inhibitors are characterized as type III and IV inhibitors that show good kinome selectivity but generally lack the subtype selectivity of same kinase family.  Recently allosteric inhibitors have been developed that bind outside the catalytic kinase domain with high selectivity for specific kinase subtypes.  Allosteric inhibitors that bind to the pseudokinase domain of pseudokinase or the extracellular domain of receptor tyrosine kinases are reviewed.  We also review recent developments in the field of allosteric kinase inhibitors including examples of proteolysis targeting chimeras, and highlight the unique binding modes for each type of inhibitors and address future opportunities in this area.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIeFGyjfBKrLVg90H21EOLACvtfcHk0lgpLokELZyE6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXmt1Giurw%253D&md5=27959fdb1a96202db7ca65b2a8e19bc7</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1002%2Fanie.201914525&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201914525%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DX.%26aulast%3DSmaill%26aufirst%3DJ.%2BB.%26aulast%3DDing%26aufirst%3DK.%26atitle%3DNew%2520Promise%2520and%2520Opportunities%2520for%2520Allosteric%2520Kinase%2520Inhibitors%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2020%26volume%3D59%26spage%3D13764%26epage%3D13776%26doi%3D10.1002%2Fanie.201914525" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furtmann, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Current Compound Coverage of the Kinome</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">30</span>– <span class="NLM_lpage">40</span>, <span class="refDoi"> DOI: 10.1021/jm5008159</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5008159" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFyqs77J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=30-40&author=Y.+Huauthor=N.+Furtmannauthor=J.+Bajorath&title=Current+Compound+Coverage+of+the+Kinome&doi=10.1021%2Fjm5008159"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Current compound coverage of the kinome</span></div><div class="casAuthors">Hu, Ye; Furtmann, Norbert; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">30-40</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Publicly available protein kinase inhibitors have been analyzed in detail.  Nearly 19,000 inhibitors have been identified with activity against 266 different kinases.  Thus, about half of the human kinome is currently covered with active small mols.  The distribution of inhibitors across the kinome is uneven.  Most available kinase inhibitors are likely to be type I inhibitors.  By contrast, type II inhibitors are rare but usually have high potency.  Kinase inhibitors generally display high scaffold diversity.  Activity cliffs with an at least 100-fold difference in potency are only found for inhibitors of 106 kinases, which is partly due to only small nos. of compds. available for many kinases, in addn. to scaffold diversity.  Moreover, kinase inhibitors are less promiscuous than often thought.  More than 70% of available inhibitors are only annotated with a single kinase activity, and only ∼1% of the inhibitors are active against ≥5 kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpT_AYGB-DpHbVg90H21EOLACvtfcHk0lgpLokELZyE6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFyqs77J&md5=83267a7f8500cdaa1a5e9285c1bb7099</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fjm5008159&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5008159%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DY.%26aulast%3DFurtmann%26aufirst%3DN.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DCurrent%2520Compound%2520Coverage%2520of%2520the%2520Kinome%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D30%26epage%3D40%26doi%3D10.1021%2Fjm5008159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miljković, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Computational Analysis of Kinase Inhibitors Identifies Promiscuity Cliffs across the Human Kinome</span>. <i>ACS Omega</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">17295</span>– <span class="NLM_lpage">17308</span>, <span class="refDoi"> DOI: 10.1021/acsomega.8b02998</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsomega.8b02998" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFSjsL7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2018&pages=17295-17308&author=F.+Miljkovi%C4%87author=J.+Bajorath&title=Computational+Analysis+of+Kinase+Inhibitors+Identifies+Promiscuity+Cliffs+across+the+Human+Kinome&doi=10.1021%2Facsomega.8b02998"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Computational Analysis of Kinase Inhibitors Identifies Promiscuity Cliffs across the Human Kinome</span></div><div class="casAuthors">Miljkovic, Filip; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">ACS Omega</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">17295-17308</span>CODEN:
                <span class="NLM_cas:coden">ACSODF</span>;
        ISSN:<span class="NLM_cas:issn">2470-1343</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Kinase inhibitors are high-priority drug candidates for a variety of therapeutic applications.  Accordingly, there has been rapid growth in the no. of kinase inhibitors and vols. of assocd. activity data.  A paradigm for the use of kinase inhibitors in oncol. is that these compds. have multitarget activity and elicit their therapeutic effects through polypharmacol.  Anal. of kinase inhibitors and activity data from medicinal chem. has so far only identified small subsets of highly promiscuous kinase inhibitors.  In this study, the authors have collected inhibitors of human kinases and their activity data from seven public repositories, curated, and combined these data, yielding more than 112,000 inhibitors with well-defined activity measurements from which qual. target annotations were derived.  Anal. of this unprecedentedly large data sets revealed that nearly 40% of human kinase inhibitors have multikinase activity but that only 4% are known to be active against five or more kinases.  However, structurally analogous inhibitors often displayed significant differences in the no. of kinase annotations, leading to the formation of nearly 16,000 "promiscuity cliffs".  Moreover, 2236 PCs (14.03%) were formed by kinase inhibitors at different stages of clin. development.  Overall, these cliffs suggested many target hypotheses for kinase inhibitors, taking data incompleteness into consideration, as well as hypotheses for structural modifications leading to kinase selectivity.  Furthermore, from network representations, pathways comprising sequences of promiscuity cliffs were extd. that revealed unexpected structure-promiscuity relationships.  To enable follow-up investigations, all promiscuity cliffs formed by human kinase inhibitors will be made freely available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFnDkPulJXJrVg90H21EOLACvtfcHk0lgpLokELZyE6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFSjsL7M&md5=684eace3a768628d1ec8c4054f06361e</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facsomega.8b02998&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsomega.8b02998%26sid%3Dliteratum%253Aachs%26aulast%3DMiljkovi%25C4%2587%26aufirst%3DF.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DComputational%2520Analysis%2520of%2520Kinase%2520Inhibitors%2520Identifies%2520Promiscuity%2520Cliffs%2520across%2520the%2520Human%2520Kinome%26jtitle%3DACS%2520Omega%26date%3D2018%26volume%3D3%26spage%3D17295%26epage%3D17308%26doi%3D10.1021%2Facsomega.8b02998" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davis, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrgard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocker, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span> <span> </span><span class="NLM_article-title">Comprehensive Analysis of Kinase Inhibitor Selectivity</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1046</span>– <span class="NLM_lpage">1051</span>, <span class="refDoi"> DOI: 10.1038/nbt.1990</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1038%2Fnbt.1990" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=22037378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=1046-1051&author=M.+I.+Davisauthor=J.+P.+Huntauthor=S.+Herrgardauthor=P.+Ciceriauthor=L.+M.+Wodickaauthor=G.+Pallaresauthor=M.+Hockerauthor=D.+K.+Treiberauthor=P.+P.+Zarrinkar&title=Comprehensive+Analysis+of+Kinase+Inhibitor+Selectivity&doi=10.1038%2Fnbt.1990"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Davis, Mindy I.; Hunt, Jeremy P.; Herrgard, Sanna; Ciceri, Pietro; Wodicka, Lisa M.; Pallares, Gabriel; Hocker, Michael; Treiber, Daniel K.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1046-1051</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We tested the interaction of 72 kinase inhibitors with 442 kinases covering >80% of the human catalytic protein kinome.  Our data show that, as a class, type II inhibitors are more selective than type I inhibitors, but that there are important exceptions to this trend.  The data further illustrate that selective inhibitors have been developed against the majority of kinases targeted by the compds. tested.  Anal. of the interaction patterns reveals a class of 'group-selective' inhibitors broadly active against a single subfamily of kinases, but selective outside that subfamily.  The data set suggests compds. to use as tools to study kinases for which no dedicated inhibitors exist.  It also provides a foundation for further exploring kinase inhibitor biol. and toxicity, as well as for studying the structural basis of the obsd. interaction patterns.  Our findings will help to realize the direct enabling potential of genomics for drug development and basic research about cellular signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEZnKVs6Xt5rVg90H21EOLACvtfcHk0lgpLokELZyE6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK&md5=67639f702cb48ffb2c60e9a250f0df5e</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1990&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1990%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DHocker%26aufirst%3DM.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DComprehensive%2520Analysis%2520of%2520Kinase%2520Inhibitor%2520Selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26spage%3D1046%26epage%3D1051%26doi%3D10.1038%2Fnbt.1990" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klaeger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinzlmeir, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilhelm, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polzer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vick, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koenig, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinecke, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruprecht, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petzoldt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zecha, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiter, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helm, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schoof, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canevari, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casale, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Depaolini, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feuchtinger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rueckert, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huenges, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garz, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gohlke, B. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zolg, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kayser, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vooder, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preissner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahne, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tõnisson, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kramer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Götze, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassermann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlegl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ehrlich, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aiche, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greif, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felder, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruland, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Médard, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeremias, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spiekermann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuster, B.</span></span> <span> </span><span class="NLM_article-title">The Target Landscape of Clinical Kinase Inhibitors</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>358</i></span>,  <span class="NLM_fpage">eaan4368</span>, <span class="refDoi"> DOI: 10.1126/science.aan4368</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1126%2Fscience.aan4368" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=29191878" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=358&publication_year=2017&pages=eaan4368&author=S.+Klaegerauthor=S.+Heinzlmeirauthor=M.+Wilhelmauthor=H.+Polzerauthor=B.+Vickauthor=P.+A.+Koenigauthor=M.+Reineckeauthor=B.+Ruprechtauthor=S.+Petzoldtauthor=C.+Mengauthor=J.+Zechaauthor=K.+Reiterauthor=H.+Qiaoauthor=D.+Helmauthor=H.+Kochauthor=M.+Schoofauthor=G.+Canevariauthor=E.+Casaleauthor=S.+R.+Depaoliniauthor=A.+Feuchtingerauthor=Z.+Wuauthor=T.+Schmidtauthor=L.+Rueckertauthor=W.+Beckerauthor=J.+Huengesauthor=A.+K.+Garzauthor=B.+O.+Gohlkeauthor=D.+P.+Zolgauthor=G.+Kayserauthor=T.+Vooderauthor=R.+Preissnerauthor=H.+Hahneauthor=N.+T%C3%B5nissonauthor=K.+Kramerauthor=K.+G%C3%B6tzeauthor=F.+Bassermannauthor=J.+Schleglauthor=H.+C.+Ehrlichauthor=S.+Aicheauthor=A.+Walchauthor=P.+A.+Greifauthor=S.+Schneiderauthor=E.+R.+Felderauthor=J.+Rulandauthor=G.+M%C3%A9dardauthor=I.+Jeremiasauthor=K.+Spiekermannauthor=B.+Kuster&title=The+Target+Landscape+of+Clinical+Kinase+Inhibitors&doi=10.1126%2Fscience.aan4368"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1126%2Fscience.aan4368&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aan4368%26sid%3Dliteratum%253Aachs%26aulast%3DKlaeger%26aufirst%3DS.%26aulast%3DHeinzlmeir%26aufirst%3DS.%26aulast%3DWilhelm%26aufirst%3DM.%26aulast%3DPolzer%26aufirst%3DH.%26aulast%3DVick%26aufirst%3DB.%26aulast%3DKoenig%26aufirst%3DP.%2BA.%26aulast%3DReinecke%26aufirst%3DM.%26aulast%3DRuprecht%26aufirst%3DB.%26aulast%3DPetzoldt%26aufirst%3DS.%26aulast%3DMeng%26aufirst%3DC.%26aulast%3DZecha%26aufirst%3DJ.%26aulast%3DReiter%26aufirst%3DK.%26aulast%3DQiao%26aufirst%3DH.%26aulast%3DHelm%26aufirst%3DD.%26aulast%3DKoch%26aufirst%3DH.%26aulast%3DSchoof%26aufirst%3DM.%26aulast%3DCanevari%26aufirst%3DG.%26aulast%3DCasale%26aufirst%3DE.%26aulast%3DDepaolini%26aufirst%3DS.%2BR.%26aulast%3DFeuchtinger%26aufirst%3DA.%26aulast%3DWu%26aufirst%3DZ.%26aulast%3DSchmidt%26aufirst%3DT.%26aulast%3DRueckert%26aufirst%3DL.%26aulast%3DBecker%26aufirst%3DW.%26aulast%3DHuenges%26aufirst%3DJ.%26aulast%3DGarz%26aufirst%3DA.%2BK.%26aulast%3DGohlke%26aufirst%3DB.%2BO.%26aulast%3DZolg%26aufirst%3DD.%2BP.%26aulast%3DKayser%26aufirst%3DG.%26aulast%3DVooder%26aufirst%3DT.%26aulast%3DPreissner%26aufirst%3DR.%26aulast%3DHahne%26aufirst%3DH.%26aulast%3DT%25C3%25B5nisson%26aufirst%3DN.%26aulast%3DKramer%26aufirst%3DK.%26aulast%3DG%25C3%25B6tze%26aufirst%3DK.%26aulast%3DBassermann%26aufirst%3DF.%26aulast%3DSchlegl%26aufirst%3DJ.%26aulast%3DEhrlich%26aufirst%3DH.%2BC.%26aulast%3DAiche%26aufirst%3DS.%26aulast%3DWalch%26aufirst%3DA.%26aulast%3DGreif%26aufirst%3DP.%2BA.%26aulast%3DSchneider%26aufirst%3DS.%26aulast%3DFelder%26aufirst%3DE.%2BR.%26aulast%3DRuland%26aufirst%3DJ.%26aulast%3DM%25C3%25A9dard%26aufirst%3DG.%26aulast%3DJeremias%26aufirst%3DI.%26aulast%3DSpiekermann%26aufirst%3DK.%26aulast%3DKuster%26aufirst%3DB.%26atitle%3DThe%2520Target%2520Landscape%2520of%2520Clinical%2520Kinase%2520Inhibitors%26jtitle%3DScience%26date%3D2017%26volume%3D358%26spage%3Deaan4368%26doi%3D10.1126%2Fscience.aan4368" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">González-Medina, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miljković, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haase, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drueckes, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trappe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laufer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Promiscuity Analysis of a Kinase Panel Screen with Designated p38 Alpha Inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>187</i></span>,  <span class="NLM_fpage">112004</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2019.112004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1016%2Fj.ejmech.2019.112004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=31881458" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVyrurfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=187&publication_year=2020&pages=112004&author=M.+Gonz%C3%A1lez-Medinaauthor=F.+Miljkovi%C4%87author=G.+S.+Haaseauthor=P.+Drueckesauthor=J.+Trappeauthor=S.+Lauferauthor=J.+Bajorath&title=Promiscuity+Analysis+of+a+Kinase+Panel+Screen+with+Designated+p38+Alpha+Inhibitors&doi=10.1016%2Fj.ejmech.2019.112004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Promiscuity analysis of a kinase panel screen with designated p38 alpha inhibitors</span></div><div class="casAuthors">Gonzalez-Medina, Mariana; Miljkovic, Filip; Haase, Gernot S.; Drueckes, Peter; Trappe, Joerg; Laufer, Stefan; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">187</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112004</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Protein phosphorylation by kinases is of crit. importance for the regulation of many cellular functions.  When kinases are deregulated numerous biol. processes are affected, which may cause a variety of diseases.  Therefore, kinase inhibition plays an important role for therapeutic intervention.  A no. of kinase inhibitors have been approved as drugs, initially in oncol. where promiscuous (multi-kinase) inhibitors were most efficacious.  Exploring kinase inhibitor selectivity and promiscuity for therapy is among the most challenging aspects of kinase drug discovery.  Herein, we thoroughly analyze a kinase profiling expt. in which 637 designated inhibitors of p38α MAP kinase (p38α) were tested against a panel of 60 kinases distributed across the human kinome.  In this expt., only 19% of the inhibitors were found to be promiscuous when the median p38α inhibition level was applied as an activity threshold.  Promiscuous inhibitors had a median value of two targets per compd., and many of these inhibitors were only active against the p38α and closely related JNK3 enzymes.  Promiscuity cliffs were identified and analyzed in a network representation revealing structural modifications that were implicated in triggering compd. promiscuity.  Taken together, the findings revealed a high degree of selectivity of designated p38α directed inhibitors although they target the ATP binding site that is largely conserved across the human kinome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6AAFV2qsAnbVg90H21EOLACvtfcHk0lh1w8Ld4hmDbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVyrurfP&md5=e615c05fc221b69dc13ec804d428dd03</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2019.112004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2019.112004%26sid%3Dliteratum%253Aachs%26aulast%3DGonz%25C3%25A1lez-Medina%26aufirst%3DM.%26aulast%3DMiljkovi%25C4%2587%26aufirst%3DF.%26aulast%3DHaase%26aufirst%3DG.%2BS.%26aulast%3DDrueckes%26aufirst%3DP.%26aulast%3DTrappe%26aufirst%3DJ.%26aulast%3DLaufer%26aufirst%3DS.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DPromiscuity%2520Analysis%2520of%2520a%2520Kinase%2520Panel%2520Screen%2520with%2520Designated%2520p38%2520Alpha%2520Inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D187%26spage%3D112004%26doi%3D10.1016%2Fj.ejmech.2019.112004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tacconi, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarsounas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">A Unique Inhibitor Binding Site in ERK1/2 is Associated with Slow Binding Kinetics</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">853</span>– <span class="NLM_lpage">860</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1629</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1038%2Fnchembio.1629" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=25195011" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFKms7vN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=853-860&author=A.+Chaikuadauthor=E.+M.+Tacconiauthor=J.+Zimmerauthor=Y.+Liangauthor=N.+S.+Grayauthor=M.+Tarsounasauthor=S.+Knapp&title=A+Unique+Inhibitor+Binding+Site+in+ERK1%2F2+is+Associated+with+Slow+Binding+Kinetics&doi=10.1038%2Fnchembio.1629"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">A unique inhibitor binding site in ERK1/2 is associated with slow binding kinetics</span></div><div class="casAuthors">Chaikuad, Apirat; M C Tacconi, Eliana; Zimmer, Jutta; Liang, Yanke; Gray, Nathanael S.; Tarsounas, Madalena; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">853-860</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Activation of the ERK pathway is a hallmark of cancer, and targeting of upstream signaling partners led to the development of approved drugs.  Recently, SCH772984 has been shown to be a selective and potent ERK1/2 inhibitor.  Here we report the structural mechanism for its remarkable selectivity.  In ERK1/2, SCH772984 induces a so-far-unknown binding pocket that accommodates the piperazine-phenyl-pyrimidine decoration.  This new binding pocket was created by an inactive conformation of the phosphate-binding loop and an outward tilt of helix αC.  In contrast, structure detn. of SCH772984 with the off-target haspin and JNK1 revealed two canonical but distinct type I binding modes.  Notably, the new binding mode with ERK1/2 was assocd. with slow binding kinetics in vitro as well as in cell-based assay systems.  The described binding mode of SCH772984 with ERK1/2 enables the design of a new type of specific kinase inhibitors with prolonged on-target activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdNjmmeHD_-LVg90H21EOLACvtfcHk0ljaCotAwfO7yQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFKms7vN&md5=b54251bd7486f0df266d8423ae9c627d</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1629&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1629%26sid%3Dliteratum%253Aachs%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DTacconi%26aufirst%3DE.%2BM.%26aulast%3DZimmer%26aufirst%3DJ.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DTarsounas%26aufirst%3DM.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DA%2520Unique%2520Inhibitor%2520Binding%2520Site%2520in%2520ERK1%252F2%2520is%2520Associated%2520with%2520Slow%2520Binding%2520Kinetics%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D853%26epage%3D860%26doi%3D10.1038%2Fnchembio.1629" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Georgi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiele, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, B.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steffen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marin Zapata, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briem, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preusse, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasta, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brands, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernández-Montalván, A.</span></span> <span> </span><span class="NLM_article-title">Binding Kinetics Survey of the Drugged Kinome</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">15774</span>– <span class="NLM_lpage">15782</span>, <span class="refDoi"> DOI: 10.1021/jacs.8b08048</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.8b08048" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVWrtbzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2018&pages=15774-15782&author=V.+Georgiauthor=F.+Schieleauthor=B.-T.+Bergerauthor=A.+Steffenauthor=P.+A.+Marin+Zapataauthor=H.+Briemauthor=S.+Menzauthor=C.+Preusseauthor=J.+D.+Vastaauthor=M.+B.+Robersauthor=M.+Brandsauthor=S.+Knappauthor=A.+Fern%C3%A1ndez-Montalv%C3%A1n&title=Binding+Kinetics+Survey+of+the+Drugged+Kinome&doi=10.1021%2Fjacs.8b08048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Binding Kinetics Survey of the Drugged Kinome</span></div><div class="casAuthors">Georgi, Victoria; Schiele, Felix; Berger, Benedict-Tilman; Steffen, Andreas; Marin Zapata, Paula A.; Briem, Hans; Menz, Stephan; Preusse, Cornelia; Vasta, James D.; Robers, Matthew B.; Brands, Michael; Knapp, Stefan; Fernandez-Montalvan, Amaury</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">15774-15782</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Target residence time is emerging as an important optimization parameter in drug discovery, yet target and off-target engagement dynamics have not been clearly linked to the clin. performance of drugs.  Here the authors developed high-throughput binding kinetics assays to characterize the interactions of 270 protein kinase inhibitors with 40 clin. relevant targets.  Anal. of the results revealed that on-rates are better correlated with affinity than off-rates and that the fraction of slowly dissocg. drug-target complexes increases from early/preclin. to late stage and FDA-approved compds., suggesting distinct contributions by each parameter to clin. success.  Combining binding parameters with PK/ADME properties, the authors illustrate in silico and in cells how kinetic selectivity could be exploited as an optimization strategy.  Furthermore, using bio- and chemoinformatics the authors uncovered structural features influencing rate consts.  The results underscore the value of binding kinetics information in rational drug design and provide a resource for future studies on this subject.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnD0Psx3Ks9rVg90H21EOLACvtfcHk0ljaCotAwfO7yQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVWrtbzO&md5=e2e54da7e0b00be3f674a3e4448333d5</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjacs.8b08048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.8b08048%26sid%3Dliteratum%253Aachs%26aulast%3DGeorgi%26aufirst%3DV.%26aulast%3DSchiele%26aufirst%3DF.%26aulast%3DBerger%26aufirst%3DB.-T.%26aulast%3DSteffen%26aufirst%3DA.%26aulast%3DMarin%2BZapata%26aufirst%3DP.%2BA.%26aulast%3DBriem%26aufirst%3DH.%26aulast%3DMenz%26aufirst%3DS.%26aulast%3DPreusse%26aufirst%3DC.%26aulast%3DVasta%26aufirst%3DJ.%2BD.%26aulast%3DRobers%26aufirst%3DM.%2BB.%26aulast%3DBrands%26aufirst%3DM.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DFern%25C3%25A1ndez-Montalv%25C3%25A1n%26aufirst%3DA.%26atitle%3DBinding%2520Kinetics%2520Survey%2520of%2520the%2520Drugged%2520Kinome%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D15774%26epage%3D15782%26doi%3D10.1021%2Fjacs.8b08048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xie, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saleh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalodimos, C. G.</span></span> <span> </span><span class="NLM_article-title">Conformational States Dynamically Populated by a Kinase Determine Its Function</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>370</i></span>,  <span class="NLM_fpage">eabc2754</span>, <span class="refDoi"> DOI: 10.1126/science.abc2754</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1126%2Fscience.abc2754" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=33004676" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=370&publication_year=2020&pages=eabc2754&author=T.+Xieauthor=T.+Salehauthor=P.+Rossiauthor=C.+G.+Kalodimos&title=Conformational+States+Dynamically+Populated+by+a+Kinase+Determine+Its+Function&doi=10.1126%2Fscience.abc2754"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1126%2Fscience.abc2754&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.abc2754%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DT.%26aulast%3DSaleh%26aufirst%3DT.%26aulast%3DRossi%26aufirst%3DP.%26aulast%3DKalodimos%26aufirst%3DC.%2BG.%26atitle%3DConformational%2520States%2520Dynamically%2520Populated%2520by%2520a%2520Kinase%2520Determine%2520Its%2520Function%26jtitle%3DScience%26date%3D2020%26volume%3D370%26spage%3Deabc2754%26doi%3D10.1126%2Fscience.abc2754" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brear, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ball, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stott, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Arcy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hyvönen, M.</span></span> <span> </span><span class="NLM_article-title">Proposed Allosteric Inhibitors Bind to the ATP Site of CK2α</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">12786</span>– <span class="NLM_lpage">12789</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c01173</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c01173" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitFygsLbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=12786-12789&author=P.+Brearauthor=D.+Ballauthor=K.+Stottauthor=S.+D%E2%80%99Arcyauthor=M.+Hyv%C3%B6nen&title=Proposed+Allosteric+Inhibitors+Bind+to+the+ATP+Site+of+CK2%CE%B1&doi=10.1021%2Facs.jmedchem.0c01173"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Proposed Allosteric Inhibitors Bind to the ATP Site of CK2α</span></div><div class="casAuthors">Brear, Paul; Ball, Darby; Stott, Katherine; D'Arcy, Sheena; Hyvonen, Marko</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">12786-12798</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">CK2α is a ubiquitous, well-studied kinase that is a target for small-mol. inhibition, for treatment of cancers.  While many different classes of ATP (ATP)-competitive inhibitors have been described for CK2α, they tend to suffer from significant off-target activity and new approaches are needed.  A series of inhibitors of CK2α has recently been described as allosteric, acting at a previously unidentified binding site.  Given the similarity of these inhibitors to known ATP-competitive inhibitors, we have investigated them further.  In our thorough structural and biophys. analyses, we have found no evidence that these inhibitors bind to the proposed allosteric site.  Rather, we report crystal structures, competitive isothermal titrn. calorimetry (ITC) and NMR, hydrogen-deuterium exchange (HDX) mass spectrometry, and chemoinformatic analyses that all point to these compds. binding in the ATP pocket.  Comparisons of our results and exptl. approach with the data presented in the original report suggest that the primary reason for the disparity is nonspecific inhibition by aggregation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpl2o_1mN1qerVg90H21EOLACvtfcHk0ljaCotAwfO7yQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitFygsLbF&md5=bf59960c798b3f53140b0d16d1ee20b3</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c01173&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c01173%26sid%3Dliteratum%253Aachs%26aulast%3DBrear%26aufirst%3DP.%26aulast%3DBall%26aufirst%3DD.%26aulast%3DStott%26aufirst%3DK.%26aulast%3DD%25E2%2580%2599Arcy%26aufirst%3DS.%26aulast%3DHyv%25C3%25B6nen%26aufirst%3DM.%26atitle%3DProposed%2520Allosteric%2520Inhibitors%2520Bind%2520to%2520the%2520ATP%2520Site%2520of%2520CK2%25CE%25B1%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D12786%26epage%3D12789%26doi%3D10.1021%2Facs.jmedchem.0c01173" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Munoz, L.</span></span> <span> </span><span class="NLM_article-title">Non-kinase Targets of Protein Kinase Inhibitors</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">424</span>– <span class="NLM_lpage">440</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.266</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1038%2Fnrd.2016.266" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=28280261" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkt1OnsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=424-440&author=L.+Munoz&title=Non-kinase+Targets+of+Protein+Kinase+Inhibitors&doi=10.1038%2Fnrd.2016.266"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Non-kinase targets of protein kinase inhibitors</span></div><div class="casAuthors">Munoz, Lenka</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">424-440</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Kinome-wide profiling platforms have comprehensively identified the relevant kinases that are targeted by numerous protein kinase inhibitors.  However, recent projects have begun to discover non-kinase targets of kinase inhibitors.  These non-kinase targets can contribute to the desired or undesired activities of inhibitors, or act as silent bystanders.  As a full awareness of a drug's mechanism of action is crucial for the interpretation of results and for successful preclin. and clin. drug development, these discoveries highlight the importance of understanding the pharmacol. of kinase inhibitors beyond the kinome.  In this Review, I discuss kinase inhibitors for which non-kinase targets have been identified and the application of emerging techniques to validate drug-target engagement in intact cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXNOHFmb6t1LVg90H21EOLACvtfcHk0lgR3jBTlQMJlQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkt1OnsbY%253D&md5=1145594c60795de101c75e522f001242</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.266&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.266%26sid%3Dliteratum%253Aachs%26aulast%3DMunoz%26aufirst%3DL.%26atitle%3DNon-kinase%2520Targets%2520of%2520Protein%2520Kinase%2520Inhibitors%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D424%26epage%3D440%26doi%3D10.1038%2Fnrd.2016.266" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">van
Linden, O. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kooistra, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leurs, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Esch, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Graaf, C.</span></span> <span> </span><span class="NLM_article-title">KLIFS: A Knowledge-Based Structural Database to Navigate Kinase-Ligand Interaction Space</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">249</span>– <span class="NLM_lpage">277</span>, <span class="refDoi"> DOI: 10.1021/jm400378w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400378w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Ojur7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=249-277&author=O.+P.+van%0ALindenauthor=A.+J.+Kooistraauthor=R.+Leursauthor=I.+J.+de+Eschauthor=C.+de+Graaf&title=KLIFS%3A+A+Knowledge-Based+Structural+Database+to+Navigate+Kinase-Ligand+Interaction+Space&doi=10.1021%2Fjm400378w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">KLIFS: A Knowledge-Based Structural Database To Navigate Kinase-Ligand Interaction Space</span></div><div class="casAuthors">van Linden, Oscar P. J.; Kooistra, Albert J.; Leurs, Rob; de Esch, Iwan J. P.; de Graaf, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">249-277</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Protein kinases regulate the majority of signal transduction pathways in cells and have become important targets for the development of designer drugs.  We present a systematic anal. of kinase-ligand interactions in all regions of the catalytic cleft of all 1252 human kinase-ligand cocrystal structures present in the Protein Data Bank (PDB).  The kinase-ligand interaction fingerprints and structure database (KLIFS) contains a consistent alignment of 85 kinase ligand binding site residues that enables the identification of family specific interaction features and classification of ligands according to their binding modes.  We illustrate how systematic mining of kinase-ligand interaction space gives new insights into how conserved and selective kinase interaction hot spots can accommodate the large diversity of chem. scaffolds in kinase ligands.  These analyses lead to an improved understanding of the structural requirements of kinase binding that will be useful in ligand discovery and design studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4tMmT6S-vRrVg90H21EOLACvtfcHk0lgR3jBTlQMJlQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1Ojur7P&md5=8519942e1703ba04f64797e07df4b712</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm400378w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400378w%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BLinden%26aufirst%3DO.%2BP.%26aulast%3DKooistra%26aufirst%3DA.%2BJ.%26aulast%3DLeurs%26aufirst%3DR.%26aulast%3Dde%2BEsch%26aufirst%3DI.%2BJ.%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26atitle%3DKLIFS%253A%2520A%2520Knowledge-Based%2520Structural%2520Database%2520to%2520Navigate%2520Kinase-Ligand%2520Interaction%2520Space%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D249%26epage%3D277%26doi%3D10.1021%2Fjm400378w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Z.</span>; <span class="NLM_string-name">Yan, Y.-H.</span>; <span class="NLM_string-name">Yang, S.</span>; <span class="NLM_string-name">Zhu, S.</span>; <span class="NLM_string-name">Yuan, Y.</span>; <span class="NLM_string-name">Qiu, Z.</span>; <span class="NLM_string-name">Jia, H.</span>; <span class="NLM_string-name">Wang, R.</span>; <span class="NLM_string-name">Li, G.-B.</span>; <span class="NLM_string-name">Li, H.</span></span> P<span> </span><span class="NLM_article-title">rofKin: A Comprehensive Web Server for Structure-Based Kinase Selectivity Profiling</span>, <span class="NLM_year">2019</span>. <a href="http://www.lilab-ecust.cn/profkin/" class="extLink">http://www.lilab-ecust.cn/profkin/</a> (accessed Aug 01, 2020).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=Z.+Shen&author=Y.-H.+Yan&author=S.+Yang&author=S.+Zhu&author=Y.+Yuan&author=Z.+Qiu&author=H.+Jia&author=R.+Wang&author=G.-B.+Li&author=H.+Li&title=rofKin%3A+A+Comprehensive+Web+Server+for+Structure-Based+Kinase+Selectivity+Profiling"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DZ.%26atitle%3DrofKin%253A%2520A%2520Comprehensive%2520Web%2520Server%2520for%2520Structure-Based%2520Kinase%2520Selectivity%2520Profiling%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">ASD: A Comprehensive Database of Allosteric Proteins and Modulators</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">D663</span>– <span class="NLM_lpage">D669</span>, <span class="refDoi"> DOI: 10.1093/nar/gkq1022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1093%2Fnar%2Fgkq1022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=21051350" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC3sXivF2ks74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2011&pages=D663-D669&author=Z.+Huangauthor=L.+Zhuauthor=Y.+Caoauthor=G.+Wuauthor=X.+Liuauthor=Y.+Chenauthor=Q.+Wangauthor=T.+Shiauthor=Y.+Zhaoauthor=Y.+Wangauthor=W.+Liauthor=Y.+Liauthor=H.+Chenauthor=G.+Chenauthor=J.+Zhang&title=ASD%3A+A+Comprehensive+Database+of+Allosteric+Proteins+and+Modulators&doi=10.1093%2Fnar%2Fgkq1022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">ASD: a comprehensive database of allosteric proteins and modulators</span></div><div class="casAuthors">Huang, Zhimin; Zhu, Liang; Cao, Yan; Wu, Geng; Liu, Xinyi; Chen, Yingyi; Wang, Qi; Shi, Ting; Zhao, Yaxue; Wang, Yuefei; Li, Weihua; Li, Yixue; Chen, Haifeng; Chen, Guoqiang; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">Suppl. 1</span>),
    <span class="NLM_cas:pages">D663-D669</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Allostery is the most direct, rapid and efficient way of regulating protein function, ranging from the control of metabolic mechanisms to signal-transduction pathways.  However, an enormous amt. of unsystematic allosteric information has deterred scientists who could benefit from this field.  Here, we present the AlloSteric Database (ASD), the first online database that provides a central resource for the display, search and anal. of structure, function and related annotation for allosteric mols.  Currently, ASD contains 336 allosteric proteins from 101 species and 8095 modulators in three categories (activators, inhibitors and regulators).  Proteins are annotated with a detailed description of allostery, biol. process and related diseases, and modulators with binding affinity, physicochem. properties and therapeutic area.  Integrating the information of allosteric proteins in ASD should allow for the identification of specific allosteric sites of a given subtype among proteins of the same family that can potentially serve as ideal targets for exptl. validation.  In addn., modulators curated in ASD can be used to investigate potent allosteric targets for the query compd., and also help chemists to implement structure modifications for novel allosteric drug design.  Therefore, ASD could be a platform and a starting point for biologists and medicinal chemists for furthering allosteric research.  ASD is freely available at http://mdl.shsmu.edu.cn/ASD/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpg8WPj2tR4JbVg90H21EOLACvtfcHk0lgR3jBTlQMJlQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXivF2ks74%253D&md5=28a63caabbc2942d98ca03ca1b4256ee</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkq1022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkq1022%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DZhu%26aufirst%3DL.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DG.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DShi%26aufirst%3DT.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DASD%253A%2520A%2520Comprehensive%2520Database%2520of%2520Allosteric%2520Proteins%2520and%2520Modulators%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2011%26volume%3D39%26spage%3DD663%26epage%3DD669%26doi%3D10.1093%2Fnar%2Fgkq1022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">UniProt
Consortium</span> <span> </span><span class="NLM_article-title">Reorganizing the
Protein Space at the Universal Protein Resource (UniProt)</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">D142</span>– <span class="NLM_lpage">D148</span>, <span class="refDoi"> DOI: 10.1093/nar/gkr981</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1093%2Fnar%2Fgkr981" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=D142-D148&author=UniProt%0AConsortium&title=Reorganizing+the%0AProtein+Space+at+the+Universal+Protein+Resource+%28UniProt%29&doi=10.1093%2Fnar%2Fgkr981"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr981&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr981%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DReorganizing%2520the%250AProtein%2520Space%2520at%2520the%2520Universal%2520Protein%2520Resource%2520%2528UniProt%2529%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3DD142%26epage%3DD148%26doi%3D10.1093%2Fnar%2Fgkr981" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span> <i>Molecular Operating Environment
(MOE)</i>, version 2014.09; <span class="NLM_publisher-name">Chemical Computing
Group ULC</span>: <span class="NLM_publisher-loc">Montreal,
QC, Canada</span>, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Molecular+Operating+Environment%0A%28MOE%29%2C+version+2014.09%3B+Chemical+Computing%0AGroup+ULC%3A+Montreal%2C%0AQC%2C+Canada%2C+2014."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DMolecular%2520Operating%2520Environment%250A%2528MOE%2529%26pub%3DChemical%2520Computing%250AGroup%2520ULC%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">PDBe-KB
consortium</span> <span> </span><span class="NLM_article-title">PDBe-KB: A Community-driven
Resource for Structural and Functional Annotations</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">D344</span>– <span class="NLM_lpage">D353</span>, <span class="refDoi"> DOI: 10.1093/nar/gkz853</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1093%2Fnar%2Fgkz853" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=31584092" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2020&pages=D344-D353&author=PDBe-KB%0Aconsortium&title=PDBe-KB%3A+A+Community-driven%0AResource+for+Structural+and+Functional+Annotations&doi=10.1093%2Fnar%2Fgkz853"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkz853&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkz853%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DPDBe-KB%253A%2520A%2520Community-driven%250AResource%2520for%2520Structural%2520and%2520Functional%2520Annotations%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2020%26volume%3D48%26spage%3DD344%26epage%3DD353%26doi%3D10.1093%2Fnar%2Fgkz853" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ohren, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavlovsky, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitehead, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuffa, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConnell, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spessard, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banotai, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delaney, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Omer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebolt-Leopold, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dudley, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, I. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flamme, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warmus, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tecle, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasemann, C. A.</span></span> <span> </span><span class="NLM_article-title">Structures of Human Map Kinase Kinase 1 (MEK1) and MEK2 Describe Novel Noncompetitive Kinase Inhibition</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1192</span>– <span class="NLM_lpage">1197</span>, <span class="refDoi"> DOI: 10.1038/nsmb859</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1038%2Fnsmb859" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=15543157" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVens7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2004&pages=1192-1197&author=J.+F.+Ohrenauthor=H.+Chenauthor=A.+Pavlovskyauthor=C.+Whiteheadauthor=E.+Zhangauthor=P.+Kuffaauthor=C.+Yanauthor=P.+McConnellauthor=C.+Spessardauthor=C.+Banotaiauthor=W.+T.+Muellerauthor=A.+Delaneyauthor=C.+Omerauthor=J.+Sebolt-Leopoldauthor=D.+T.+Dudleyauthor=I.+K.+Leungauthor=C.+Flammeauthor=J.+Warmusauthor=M.+Kaufmanauthor=S.+Barrettauthor=H.+Tecleauthor=C.+A.+Hasemann&title=Structures+of+Human+Map+Kinase+Kinase+1+%28MEK1%29+and+MEK2+Describe+Novel+Noncompetitive+Kinase+Inhibition&doi=10.1038%2Fnsmb859"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Structures of human MAP kinase kinase 1 (MEK1) and MEK2 describe novel noncompetitive kinase inhibition</span></div><div class="casAuthors">Ohren, Jeffrey F.; Chen, Huifen; Pavlovsky, Alexander; Whitehead, Christopher; Zhang, Erli; Kuffa, Peter; Yan, Chunhong; McConnell, Patrick; Spessard, Cindy; Banotai, Craig; Mueller, W. Thomas; Delaney, Amy; Omer, Charles; Sebolt-Leopold, Judith; Dudley, David T.; Leung, Iris K.; Flamme, Cathlin; Warmus, Joseph; Kaufman, Michael; Barrett, Stephen; Tecle, Haile; Hasemann, Charles A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1192-1197</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">MEK1 and MEK2 kinases are closely related, dual-specificity tyrosine/threonine protein kinases found in the Ras/Raf/MEK/ERK MAP kinase (MAPK) signaling pathway.  Approx. 30% of all human cancers have a constitutively activated MAPK pathway, and constitutive activation of MEK1 results in cellular transformation.  Here, the authors present the x-ray crystal structures of human MEK1 and MEK2 kinases, each detd. as a ternary complex with MgATP and an inhibitor to a resoln. of 2.4 and 3.2 Å, resp.  The structures revealed that MEK1 and MEK2 each had a unique inhibitor-binding pocket adjacent to the MgATP-binding site.  The presence of the potent inhibitors induced several conformational changes in the unphosphorylated MEK1 and MEK2 kinases that locked them into a closed but catalytically inactive species.  Thus, the structures reported here reveal a novel, noncompetitive mechanism for protein kinase inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIY9begVVfW7Vg90H21EOLACvtfcHk0lgyATdmdNF9DQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVens7jM&md5=79ebbc2e3a9cdd3fbecb8bb35ea9dba3</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnsmb859&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb859%26sid%3Dliteratum%253Aachs%26aulast%3DOhren%26aufirst%3DJ.%2BF.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DPavlovsky%26aufirst%3DA.%26aulast%3DWhitehead%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DE.%26aulast%3DKuffa%26aufirst%3DP.%26aulast%3DYan%26aufirst%3DC.%26aulast%3DMcConnell%26aufirst%3DP.%26aulast%3DSpessard%26aufirst%3DC.%26aulast%3DBanotai%26aufirst%3DC.%26aulast%3DMueller%26aufirst%3DW.%2BT.%26aulast%3DDelaney%26aufirst%3DA.%26aulast%3DOmer%26aufirst%3DC.%26aulast%3DSebolt-Leopold%26aufirst%3DJ.%26aulast%3DDudley%26aufirst%3DD.%2BT.%26aulast%3DLeung%26aufirst%3DI.%2BK.%26aulast%3DFlamme%26aufirst%3DC.%26aulast%3DWarmus%26aufirst%3DJ.%26aulast%3DKaufman%26aufirst%3DM.%26aulast%3DBarrett%26aufirst%3DS.%26aulast%3DTecle%26aufirst%3DH.%26aulast%3DHasemann%26aufirst%3DC.%2BA.%26atitle%3DStructures%2520of%2520Human%2520Map%2520Kinase%2520Kinase%25201%2520%2528MEK1%2529%2520and%2520MEK2%2520Describe%2520Novel%2520Noncompetitive%2520Kinase%2520Inhibition%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2004%26volume%3D11%26spage%3D1192%26epage%3D1197%26doi%3D10.1038%2Fnsmb859" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wright, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCormack, P. L.</span></span> <span> </span><span class="NLM_article-title">Trametinib: First Global Approval</span>. <i>Drugs</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">1245</span>– <span class="NLM_lpage">1254</span>, <span class="refDoi"> DOI: 10.1007/s40265-013-0096-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1007%2Fs40265-013-0096-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=23846731" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A280%3ADC%252BC3sfgtFSjtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=1245-1254&author=C.+J.+Wrightauthor=P.+L.+McCormack&title=Trametinib%3A+First+Global+Approval&doi=10.1007%2Fs40265-013-0096-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Trametinib: first global approval</span></div><div class="casAuthors">Wright Cameron J M; McCormack Paul L</div><div class="citationInfo"><span class="NLM_cas:title">Drugs</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1245-54</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Trametinib is an orally bioavailable mitogen-activated protein kinase (MAPK) kinase (MEK) inhibitor with antineoplastic activity.  The compound specifically binds to MEK1 and MEK2, resulting in inhibition of growth factor-mediated cell signalling and cellular proliferation in various cancers.  Originally developed by Japan Tobacco, GlaxoSmithKline has licensed exclusive worldwide rights to the compound and conducted development in a number of different cancer types.  Trametinib, as a monotherapy, has been approved in the US for the treatment of unresectable or metastatic malignant melanoma with BRAF V600E or V600K mutations, as detected by an FDA-approved test.  The compound, as a monotherapy, has also been submitted for regulatory review in the EU for BRAF mutation-positive malignant melanoma, and is in phase III development in Europe, Argentina, Canada and Oceania.  Phase II development is underway for pancreatic cancer, non-small cell lung cancer and relapsed or refractory leukaemias.  GlaxoSmithKline is also developing trametinib for use in combination with dabrafenib in BRAF V600 mutation-positive metastatic cutaneous melanoma; the combination is at the preregistration stage in the EU and a phase III clinical programme is underway worldwide.  Phase II development for this combination is also underway in colorectal cancer.  Several phase I trials have also been initiated to evaluate trametinib in combination with other drugs for the treatment of various solid tumours and haematological malignancies.  A paediatric oral solution formulation has been assessed against the oral tablet formulation in a phase I trial.  This article summarizes the milestones in the development of trametinib leading to this first approval for unresectable or metastatic BRAF mutation-positive malignant melanoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRw3yHtWXvrR2BAiYWy6n3SfW6udTcc2eYWzl9WYc0sgrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sfgtFSjtg%253D%253D&md5=133d953030ec0c3f62fbba730fabbfa9</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1007%2Fs40265-013-0096-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40265-013-0096-1%26sid%3Dliteratum%253Aachs%26aulast%3DWright%26aufirst%3DC.%2BJ.%26aulast%3DMcCormack%26aufirst%3DP.%2BL.%26atitle%3DTrametinib%253A%2520First%2520Global%2520Approval%26jtitle%3DDrugs%26date%3D2013%26volume%3D73%26spage%3D1245%26epage%3D1254%26doi%3D10.1007%2Fs40265-013-0096-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Biondi, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casamayor, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deak, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Currie, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alessi, D. R.</span></span> <span> </span><span class="NLM_article-title">Identification of a Pocket in the PDK1 Kinase Domain That Interacts with PIF and the C-Terminal Residues of PKA</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">979</span>– <span class="NLM_lpage">988</span>, <span class="refDoi"> DOI: 10.1093/emboj/19.5.979</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1093%2Femboj%2F19.5.979" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10698939" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BD3cXitVyrsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2000&pages=979-988&author=R.+M.+Biondiauthor=P.+C.+Cheungauthor=A.+Casamayorauthor=M.+Deakauthor=R.+A.+Currieauthor=D.+R.+Alessi&title=Identification+of+a+Pocket+in+the+PDK1+Kinase+Domain+That+Interacts+with+PIF+and+the+C-Terminal+Residues+of+PKA&doi=10.1093%2Femboj%2F19.5.979"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a pocket in the PDK1 kinase domain that interacts with PIF and the C-terminal residues of PKA</span></div><div class="casAuthors">Biondi, Ricardo M.; Cheung, Peter C. F.; Casamayor, Antonio; Deak, Maria; Currie, Richard A.; Alessi, Dario R.</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">979-988</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The 3-phosphoinositide-dependent protein kinase-1 (PDK1) phosphorylates and activates a no. of protein kinases of the AGC subfamily.  The kinase domain of PDK1 interacts with a region of protein kinase C-related kinase-2 (PRK2), termed the PDK1-interacting fragment (PIF), through a hydrophobic motif.  Here we identify a hydrophobic pocket in the small lobe of the PDK1 kinase domain, sep. from the ATP- and substrate-binding sites, that interacts with PIF.  Mutation of residues predicted to form part of this hydrophobic pocket either abolished or significantly diminished the affinity of PDK1 for PIF.  PIF increased the rate at which PDK1 phosphorylated a synthetic dodecapeptide (T308tide), corresponding to the sequences surrounding the PDK1 phosphorylation site of PKB.  This peptide is a poor substrate for PDK1, but a peptide comprising T308tide fused to the PDK1-binding motif of PIF was a vastly superior substrate for PDK1.  Our results suggest that the PIF-binding pocket on the kinase domain of PDK1 acts as a "docking site", enabling it to interact with and enhance the phosphorylation of its substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9OG4ALKuz-LVg90H21EOLACvtfcHk0lhwBKyvLsXErg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXitVyrsrk%253D&md5=dfcdfbf35d7d4039725ee1b371837ed8</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1093%2Femboj%2F19.5.979&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Femboj%252F19.5.979%26sid%3Dliteratum%253Aachs%26aulast%3DBiondi%26aufirst%3DR.%2BM.%26aulast%3DCheung%26aufirst%3DP.%2BC.%26aulast%3DCasamayor%26aufirst%3DA.%26aulast%3DDeak%26aufirst%3DM.%26aulast%3DCurrie%26aufirst%3DR.%2BA.%26aulast%3DAlessi%26aufirst%3DD.%2BR.%26atitle%3DIdentification%2520of%2520a%2520Pocket%2520in%2520the%2520PDK1%2520Kinase%2520Domain%2520That%2520Interacts%2520with%2520PIF%2520and%2520the%2520C-Terminal%2520Residues%2520of%2520PKA%26jtitle%3DEMBO%2520J.%26date%3D2000%26volume%3D19%26spage%3D979%26epage%3D988%26doi%3D10.1093%2Femboj%2F19.5.979" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Janecek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emery, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huggins, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stockwell, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stokes, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almeida, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardwick, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narvaez, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hyvonen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spring, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKenzie, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkitaraman, A. R.</span></span> <span> </span><span class="NLM_article-title">Allosteric Modulation of AURKA Kinase Activity by a Small-Molecule Inhibitor of Its Protein-Protein Interaction with Tpx2</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">28528</span>, <span class="refDoi"> DOI: 10.1038/srep28528</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1038%2Fsrep28528" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=27339427" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVKiurjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=28528&author=M.+Janecekauthor=M.+Rossmannauthor=P.+Sharmaauthor=A.+Emeryauthor=D.+J.+Hugginsauthor=S.+R.+Stockwellauthor=J.+E.+Stokesauthor=Y.+S.+Tanauthor=E.+G.+Almeidaauthor=B.+Hardwickauthor=A.+J.+Narvaezauthor=M.+Hyvonenauthor=D.+R.+Springauthor=G.+J.+McKenzieauthor=A.+R.+Venkitaraman&title=Allosteric+Modulation+of+AURKA+Kinase+Activity+by+a+Small-Molecule+Inhibitor+of+Its+Protein-Protein+Interaction+with+Tpx2&doi=10.1038%2Fsrep28528"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric modulation of AURKA kinase activity by a small-molecule inhibitor of its protein-protein interaction with TPX2</span></div><div class="casAuthors">Janecek, Matej; Rossmann, Maxim; Sharma, Pooja; Emery, Amy; Huggins, David J.; Stockwell, Simon R.; Stokes, Jamie E.; Tan, Yaw S.; Almeida, Estrella Guarino; Hardwick, Bryn; Narvaez, Ana J.; Hyvonen, Marko; Spring, David R.; McKenzie, Grahame J.; Venkitaraman, Ashok R.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">28528</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The essential mitotic kinase Aurora A (AURKA) is controlled during cell cycle progression via two distinct mechanisms.  Following activation loop autophosphorylation early in mitosis when it localizes to centrosomes, AURKA is allosterically activated on the mitotic spindle via binding to the microtubule-assocd. protein, TPX2.  Here, we report the discovery of AurkinA, a novel chem. inhibitor of the AURKA-TPX2 interaction, which acts via an unexpected structural mechanism to inhibit AURKA activity and mitotic localization.  In crystal structures, AurkinA binds to a hydrophobic pocket (the 'Y pocket') that normally accommodates a conserved Tyr-Ser-Tyr motif from TPX2, blocking the AURKA-TPX2 interaction.  AurkinA binding to the Y- pocket induces structural changes in AURKA that inhibit catalytic activity in vitro and in cells, without affecting ATP binding to the active site, defining a novel mechanism of allosteric inhibition.  Consistent with this mechanism, cells exposed to AurkinA mislocalize AURKA from mitotic spindle microtubules.  Thus, our findings provide fresh insight into the catalytic mechanism of AURKA, and identify a key structural feature as the target for a new class of dual-mode AURKA inhibitors, with implications for the chem. biol. and selective therapeutic targeting of structurally related kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_RxbdNKlklbVg90H21EOLACvtfcHk0lhwBKyvLsXErg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVKiurjL&md5=39ed09c96dee028e980e391c033a624c</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fsrep28528&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep28528%26sid%3Dliteratum%253Aachs%26aulast%3DJanecek%26aufirst%3DM.%26aulast%3DRossmann%26aufirst%3DM.%26aulast%3DSharma%26aufirst%3DP.%26aulast%3DEmery%26aufirst%3DA.%26aulast%3DHuggins%26aufirst%3DD.%2BJ.%26aulast%3DStockwell%26aufirst%3DS.%2BR.%26aulast%3DStokes%26aufirst%3DJ.%2BE.%26aulast%3DTan%26aufirst%3DY.%2BS.%26aulast%3DAlmeida%26aufirst%3DE.%2BG.%26aulast%3DHardwick%26aufirst%3DB.%26aulast%3DNarvaez%26aufirst%3DA.%2BJ.%26aulast%3DHyvonen%26aufirst%3DM.%26aulast%3DSpring%26aufirst%3DD.%2BR.%26aulast%3DMcKenzie%26aufirst%3DG.%2BJ.%26aulast%3DVenkitaraman%26aufirst%3DA.%2BR.%26atitle%3DAllosteric%2520Modulation%2520of%2520AURKA%2520Kinase%2520Activity%2520by%2520a%2520Small-Molecule%2520Inhibitor%2520of%2520Its%2520Protein-Protein%2520Interaction%2520with%2520Tpx2%26jtitle%3DSci.%2520Rep.%26date%3D2016%26volume%3D6%26spage%3D28528%26doi%3D10.1038%2Fsrep28528" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hindie, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stroba, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Garcia, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Idrissova, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeuzem, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirschberg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaeffer, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jørgensen, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alzari, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biondi, R. M.</span></span> <span> </span><span class="NLM_article-title">Structure and Allosteric Effects of Low-Molecular-Weight Activators on the Protein kinase PDK1</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">758</span>– <span class="NLM_lpage">764</span>, <span class="refDoi"> DOI: 10.1038/nchembio.208</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1038%2Fnchembio.208" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=19718043" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVKhs73F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=758-764&author=V.+Hindieauthor=A.+Strobaauthor=H.+Zhangauthor=L.+A.+Lopez-Garciaauthor=L.+Idrissovaauthor=S.+Zeuzemauthor=D.+Hirschbergauthor=F.+Schaefferauthor=T.+J.+J%C3%B8rgensenauthor=M.+Engelauthor=P.+M.+Alzariauthor=R.+M.+Biondi&title=Structure+and+Allosteric+Effects+of+Low-Molecular-Weight+Activators+on+the+Protein+kinase+PDK1&doi=10.1038%2Fnchembio.208"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and allosteric effects of low-molecular-weight activators on the protein kinase PDK1</span></div><div class="casAuthors">Hindie, Valerie; Stroba, Adriana; Zhang, Hua; Lopez-Garcia, Laura A.; Idrissova, Leila; Zeuzem, Stefan; Hirschberg, Daniel; Schaeffer, Francis; Jorgensen, Thomas J. D.; Engel, Matthias; Alzari, Pedro M.; Biondi, Ricardo M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">758-764</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein phosphorylation transduces a large set of intracellular signals.  One mechanism by which phosphorylation mediates signal transduction is by prompting conformational changes in the target protein or interacting proteins.  Previous work described an allosteric site mediating phosphorylation-dependent activation of AGC kinases.  AGC kinase PDK1 is activated by the docking of a phosphorylated motif from substrates.  Here, the authors present the crystallog. of PDK1 bound to a rationally developed low-mol.-wt. activator and describe the conformational changes induced by small compds. in the crystal and in soln. using a fluorescence-based assay and deuterium exchange expts.  The results indicate that the binding of the compd. produces local changes at the target site, the PIF-binding pocket, and also allosteric changes at the ATP-binding site and the activation loop.  Altogether, the authors present mol. details of the allosteric changes induced by small compds. that trigger the activation of PDK1 through mimicry of phosphorylation-dependent conformational changes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBilNGbn-krbVg90H21EOLACvtfcHk0lhwBKyvLsXErg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVKhs73F&md5=c9afd03c1d2c954fbcf04709570fdb76</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.208&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.208%26sid%3Dliteratum%253Aachs%26aulast%3DHindie%26aufirst%3DV.%26aulast%3DStroba%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DLopez-Garcia%26aufirst%3DL.%2BA.%26aulast%3DIdrissova%26aufirst%3DL.%26aulast%3DZeuzem%26aufirst%3DS.%26aulast%3DHirschberg%26aufirst%3DD.%26aulast%3DSchaeffer%26aufirst%3DF.%26aulast%3DJ%25C3%25B8rgensen%26aufirst%3DT.%2BJ.%26aulast%3DEngel%26aufirst%3DM.%26aulast%3DAlzari%26aufirst%3DP.%2BM.%26aulast%3DBiondi%26aufirst%3DR.%2BM.%26atitle%3DStructure%2520and%2520Allosteric%2520Effects%2520of%2520Low-Molecular-Weight%2520Activators%2520on%2520the%2520Protein%2520kinase%2520PDK1%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D758%26epage%3D764%26doi%3D10.1038%2Fnchembio.208" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Diskin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engelberg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Livnah, O.</span></span> <span> </span><span class="NLM_article-title">A Novel Lipid Binding Site Formed by the MAP Kinase Insert in p38 Alpha</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>375</i></span>,  <span class="NLM_fpage">70</span>– <span class="NLM_lpage">79</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2007.09.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1016%2Fj.jmb.2007.09.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=17999933" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlylu7vI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=375&publication_year=2008&pages=70-79&author=R.+Diskinauthor=D.+Engelbergauthor=O.+Livnah&title=A+Novel+Lipid+Binding+Site+Formed+by+the+MAP+Kinase+Insert+in+p38+Alpha&doi=10.1016%2Fj.jmb.2007.09.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">A Novel Lipid Binding Site Formed by the MAP Kinase Insert in p38α</span></div><div class="casAuthors">Diskin, Ron; Engelberg, David; Livnah, Oded</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">375</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">70-79</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The p38 mitogen-activated protein (MAP) kinases function as signaling mols. essential for many cellular processes, particularly mediating stress response.  The activity of p38 MAP kinases is meticulously regulated to reach the desired cellular phenotype.  Several alternative activation and attenuation mechanisms have been characterized recently which include new phosphorylation sites.  Here we present the crystal structure of p38α MAP kinase in complex with n-octyl-β-glucopyranoside (β-OG) detergent.  The complex unveils a novel lipid-binding site formed by a local conformational change of the MAP kinase insert.  This binding is the first evidence of a possible role for the MAP kinase insert in p38.  The binding site can accommodate a large selection of lipidic mols.  In addn., we also show via biophys. methods that arachidonic acid (AA) and its derivs. bind p38α in vitro.  Based on our anal. we propose that the binding of lipids could fine-tune p38α catalytic activity towards a preferred phenotype.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6J5fd2Jqe77Vg90H21EOLACvtfcHk0lh5Ds6hyvoMpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlylu7vI&md5=88b97152503182d41fa0ea7b53271229</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2007.09.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2007.09.002%26sid%3Dliteratum%253Aachs%26aulast%3DDiskin%26aufirst%3DR.%26aulast%3DEngelberg%26aufirst%3DD.%26aulast%3DLivnah%26aufirst%3DO.%26atitle%3DA%2520Novel%2520Lipid%2520Binding%2520Site%2520Formed%2520by%2520the%2520MAP%2520Kinase%2520Insert%2520in%2520p38%2520Alpha%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2008%26volume%3D375%26spage%3D70%26epage%3D79%26doi%3D10.1016%2Fj.jmb.2007.09.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tzarum, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eisenberg-Domovich, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gills, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dennis, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Livnah, O.</span></span> <span> </span><span class="NLM_article-title">Lipid Molecules Induce P38alpha Activation Via a Novel Molecular Switch</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>424</i></span>,  <span class="NLM_fpage">339</span>– <span class="NLM_lpage">353</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2012.10.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1016%2Fj.jmb.2012.10.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=23079240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFymu73I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=424&publication_year=2012&pages=339-353&author=N.+Tzarumauthor=Y.+Eisenberg-Domovichauthor=J.+J.+Gillsauthor=P.+A.+Dennisauthor=O.+Livnah&title=Lipid+Molecules+Induce+P38alpha+Activation+Via+a+Novel+Molecular+Switch&doi=10.1016%2Fj.jmb.2012.10.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Lipid Molecules Induce p38α Activation via a Novel Molecular Switch</span></div><div class="casAuthors">Tzarum, Netanel; Eisenberg-Domovich, Yael; Gills, Joell J.; Dennis, Phillip A.; Livnah, Oded</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">424</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">339-353</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">P38α mitogen-activated protein kinase (MAPK) is generally activated by dual phosphorylation but has also been shown to exhibit alternative activation modes.  One of these modes included a direct interaction with phosphatidylinositol ether lipid analogs (PIA), inducing p38α autoactivation and apoptosis.  Perifosine, an Akt inhibitor in phase II clin. trials, also showed p38α activation properties similar to those of PIAs.  The crystal structures of p38α in complex with PIA23, PIA24 and perifosine provide insights into this unique activation mode.  The activating mols. bind a unique hydrophobic binding site in the kinase C'-lobe formed in part by the MAPK insert region.  In addn., there are conformational changes in the short αEF/αF loop region that acts as an activation switch, inducing autophosphorylation.  Structural and biochem. characterization of the αEF/αF loop identified Trp197 as a key residue in the lipid binding and in p38α catalytic activity.  The lipid binding site also accommodates hydrophobic inhibitor mols., and thus can serve as a novel p38α-target for specific activation or inhibition, with novel therapeutic implications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQ5MaNgn-kFbVg90H21EOLACvtfcHk0lh5Ds6hyvoMpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFymu73I&md5=4341b3620e35b8a54f68448817ca157b</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2012.10.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2012.10.007%26sid%3Dliteratum%253Aachs%26aulast%3DTzarum%26aufirst%3DN.%26aulast%3DEisenberg-Domovich%26aufirst%3DY.%26aulast%3DGills%26aufirst%3DJ.%2BJ.%26aulast%3DDennis%26aufirst%3DP.%2BA.%26aulast%3DLivnah%26aufirst%3DO.%26atitle%3DLipid%2520Molecules%2520Induce%2520P38alpha%2520Activation%2520Via%2520a%2520Novel%2520Molecular%2520Switch%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D424%26spage%3D339%26epage%3D353%26doi%3D10.1016%2Fj.jmb.2012.10.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nagar, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hantschel, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheffzek, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veach, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bornmann, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarkson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Superti-Furga, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuriyan, J.</span></span> <span> </span><span class="NLM_article-title">Structural Basis for the Autoinhibition of c-Abl Tyrosine Kinase</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">859</span>– <span class="NLM_lpage">871</span>, <span class="refDoi"> DOI: 10.1016/S0092-8674(03)00194-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1016%2FS0092-8674%2803%2900194-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=12654251" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BD3sXivVWjtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2003&pages=859-871&author=B.+Nagarauthor=O.+Hantschelauthor=M.+A.+Youngauthor=K.+Scheffzekauthor=D.+Veachauthor=W.+Bornmannauthor=B.+Clarksonauthor=G.+Superti-Furgaauthor=J.+Kuriyan&title=Structural+Basis+for+the+Autoinhibition+of+c-Abl+Tyrosine+Kinase&doi=10.1016%2FS0092-8674%2803%2900194-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for the autoinhibition of c-Abl tyrosine kinase</span></div><div class="casAuthors">Nagar, Bhushan; Hantschel, Oliver; Young, Matthew A.; Scheffzek, Klaus; Veach, Darren; Bornmann, William; Clarkson, Bayard; Superti-Furga, Giulio; Kuriyan, John</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">859-871</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">C-Abl is normally regulated by an autoinhibitory mechanism, the disruption of which leads to chronic myelogenous leukemia.  The details of this mechanism have been elusive because c-Abl lacks a phosphotyrosine residue that triggers the assembly of the autoinhibited form of the closely related Src kinases by internally engaging the SH2 domain.  Crystal structures of c-Abl show that the N-terminal myristoyl modification of c-Abl 1b binds to the kinase domain and induces conformational changes that allow the SH2 and SH3 domains to dock onto it.  Autoinhibited c-Abl forms an assembly that is strikingly similar to that of inactive Src kinases but with specific differences that explain the differential ability of the drug STI-571/Gleevec/imatinib (STI-571) to inhibit the catalytic activity of Abl, but not that of c-Src.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaks_bu7ZlC7Vg90H21EOLACvtfcHk0lh5Ds6hyvoMpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXivVWjtr4%253D&md5=19245c92b1e36779e2024bc071aa723b</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2803%2900194-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252803%252900194-6%26sid%3Dliteratum%253Aachs%26aulast%3DNagar%26aufirst%3DB.%26aulast%3DHantschel%26aufirst%3DO.%26aulast%3DYoung%26aufirst%3DM.%2BA.%26aulast%3DScheffzek%26aufirst%3DK.%26aulast%3DVeach%26aufirst%3DD.%26aulast%3DBornmann%26aufirst%3DW.%26aulast%3DClarkson%26aufirst%3DB.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26aulast%3DKuriyan%26aufirst%3DJ.%26atitle%3DStructural%2520Basis%2520for%2520the%2520Autoinhibition%2520of%2520c-Abl%2520Tyrosine%2520Kinase%26jtitle%3DCell%26date%3D2003%26volume%3D112%26spage%3D859%26epage%3D871%26doi%3D10.1016%2FS0092-8674%2803%2900194-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campobasso, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biju, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, X.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cottom, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galbraith, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegfried, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zappacosta, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Washio, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertrand, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Head, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moores, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johanson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burton, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erickson-Miller, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simpson, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tummino, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliff, A.</span></span> <span> </span><span class="NLM_article-title">Discovery and Characterization of a Cell-Permeable, Small-Molecule c-Abl Kinase Activator that Binds to the Myristoyl Binding Site</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">177</span>– <span class="NLM_lpage">186</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2010.12.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1016%2Fj.chembiol.2010.12.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=21338916" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC3MXit1Kjtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=177-186&author=J.+Yangauthor=N.+Campobassoauthor=M.+P.+Bijuauthor=K.+Fisherauthor=X.-Q.+Panauthor=J.+Cottomauthor=S.+Galbraithauthor=T.+Hoauthor=H.+Zhangauthor=X.+Hongauthor=P.+Wardauthor=G.+Hofmannauthor=B.+Siegfriedauthor=F.+Zappacostaauthor=Y.+Washioauthor=P.+Caoauthor=J.+Quauthor=S.+Bertrandauthor=D.-Y.+Wangauthor=M.+S.+Headauthor=H.+Liauthor=S.+Mooresauthor=Z.+Laiauthor=K.+Johansonauthor=G.+Burtonauthor=C.+Erickson-Millerauthor=G.+Simpsonauthor=P.+Tumminoauthor=R.+A.+Copelandauthor=A.+Oliff&title=Discovery+and+Characterization+of+a+Cell-Permeable%2C+Small-Molecule+c-Abl+Kinase+Activator+that+Binds+to+the+Myristoyl+Binding+Site&doi=10.1016%2Fj.chembiol.2010.12.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Characterization of a Cell-Permeable, Small-Molecule c-Abl Kinase Activator that Binds to the Myristoyl Binding Site</span></div><div class="casAuthors">Yang, Jingsong; Campobasso, Nino; Biju, Mangatt P.; Fisher, Kelly; Pan, Xiao-Qing; Cottom, Josh; Galbraith, Sarah; Ho, Thau; Zhang, Hong; Hong, Xuan; Ward, Paris; Hofmann, Glenn; Siegfried, Brett; Zappacosta, Francesca; Washio, Yoshiaki; Cao, Ping; Qu, Junya; Bertrand, Sophie; Wang, Da-Yuan; Head, Martha S.; Li, Hu; Moores, Sheri; Lai, Zhihong; Johanson, Kyung; Burton, George; Erickson-Miller, Connie; Simpson, Graham; Tummino, Peter; Copeland, Robert A.; Oliff, Allen</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">177-186</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: c-Abl kinase activity is regulated by a unique mechanism involving the formation of an autoinhibited conformation in which the N-terminal myristoyl group binds intramolecularly to the myristoyl binding site on the kinase domain and induces the bending of the αI helix that creates a docking surface for the SH2 domain.  Here, the authors report a small-mol. c-Abl activator, DPH, that displays potent enzymic and cellular activity in stimulating c-Abl activation.  Structural analyses indicate that DPH binds to the myristoyl binding site and prevents the formation of the bent conformation of the αI helix through steric hindrance, a mode of action distinct from the previously identified allosteric c-Abl inhibitor, GNF-2, that also binds to the myristoyl binding site.  DPH represents the first cell-permeable, small-mol. tool compd. for c-Abl activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKyxGGBs719rVg90H21EOLACvtfcHk0liezcc0RFz-ZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXit1Kjtrg%253D&md5=27986a6ebec4220a39045cae27570775</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2010.12.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2010.12.013%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DCampobasso%26aufirst%3DN.%26aulast%3DBiju%26aufirst%3DM.%2BP.%26aulast%3DFisher%26aufirst%3DK.%26aulast%3DPan%26aufirst%3DX.-Q.%26aulast%3DCottom%26aufirst%3DJ.%26aulast%3DGalbraith%26aufirst%3DS.%26aulast%3DHo%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DHong%26aufirst%3DX.%26aulast%3DWard%26aufirst%3DP.%26aulast%3DHofmann%26aufirst%3DG.%26aulast%3DSiegfried%26aufirst%3DB.%26aulast%3DZappacosta%26aufirst%3DF.%26aulast%3DWashio%26aufirst%3DY.%26aulast%3DCao%26aufirst%3DP.%26aulast%3DQu%26aufirst%3DJ.%26aulast%3DBertrand%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DD.-Y.%26aulast%3DHead%26aufirst%3DM.%2BS.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DMoores%26aufirst%3DS.%26aulast%3DLai%26aufirst%3DZ.%26aulast%3DJohanson%26aufirst%3DK.%26aulast%3DBurton%26aufirst%3DG.%26aulast%3DErickson-Miller%26aufirst%3DC.%26aulast%3DSimpson%26aufirst%3DG.%26aulast%3DTummino%26aufirst%3DP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DOliff%26aufirst%3DA.%26atitle%3DDiscovery%2520and%2520Characterization%2520of%2520a%2520Cell-Permeable%252C%2520Small-Molecule%2520c-Abl%2520Kinase%2520Activator%2520that%2520Binds%2520to%2520the%2520Myristoyl%2520Binding%2520Site%26jtitle%3DChem.%2520Biol.%26date%3D2011%26volume%3D18%26spage%3D177%26epage%3D186%26doi%3D10.1016%2Fj.chembiol.2010.12.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wylie, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schoepfer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowan-Jacob, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furet, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marzinzik, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pelle, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buonamici, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassan, A. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iyer, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berellini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodd, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thohan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bitter, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Branford, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petruzzelli, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanasse, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warmuth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keen, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span> <span> </span><span class="NLM_article-title">The Allosteric Inhibitor ABL001 Enables Dual Targeting of BCR–ABL1</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>543</i></span>,  <span class="NLM_fpage">733</span>– <span class="NLM_lpage">737</span>, <span class="refDoi"> DOI: 10.1038/nature21702</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1038%2Fnature21702" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=28329763" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC2sXltVWntbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=543&publication_year=2017&pages=733-737&author=A.+A.+Wylieauthor=J.+Schoepferauthor=W.+Jahnkeauthor=S.+W.+Cowan-Jacobauthor=A.+Looauthor=P.+Furetauthor=A.+L.+Marzinzikauthor=X.+Pelleauthor=J.+Donovanauthor=W.+Zhuauthor=S.+Buonamiciauthor=A.+Q.+Hassanauthor=F.+Lombardoauthor=V.+Iyerauthor=M.+Palmerauthor=G.+Berelliniauthor=S.+Doddauthor=S.+Thohanauthor=H.+Bitterauthor=S.+Branfordauthor=D.+M.+Rossauthor=T.+P.+Hughesauthor=L.+Petruzzelliauthor=K.+G.+Vanasseauthor=M.+Warmuthauthor=F.+Hofmannauthor=N.+J.+Keenauthor=W.+R.+Sellers&title=The+Allosteric+Inhibitor+ABL001+Enables+Dual+Targeting+of+BCR%E2%80%93ABL1&doi=10.1038%2Fnature21702"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">The allosteric inhibitor ABL001 enables dual targeting of BCR-ABL1</span></div><div class="casAuthors">Wylie, Andrew A.; Schoepfer, Joseph; Jahnke, Wolfgang; Cowan-Jacob, Sandra W.; Loo, Alice; Furet, Pascal; Marzinzik, Andreas L.; Pelle, Xavier; Donovan, Jerry; Zhu, Wenjing; Buonamici, Silvia; Hassan, A. Quamrul; Lombardo, Franco; Iyer, Varsha; Palmer, Michael; Berellini, Giuliano; Dodd, Stephanie; Thohan, Sanjeev; Bitter, Hans; Branford, Susan; Ross, David M.; Hughes, Timothy P.; Petruzzelli, Lilli; Vanasse, K. Gary; Warmuth, Markus; Hofmann, Francesco; Keen, Nicholas J.; Sellers, William R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">543</span>
        (<span class="NLM_cas:issue">7647</span>),
    <span class="NLM_cas:pages">733-737</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chronic myeloid leukemia (CML) is driven by the activity of the BCR-ABL1 fusion oncoprotein.  ABL1 kinase inhibitors have improved the clin. outcomes for patients with CML, with over 80% of patients treated with imatinib surviving for more than 10 years.  Second-generation ABL1 kinase inhibitors induce more potent mol. responses in both previously untreated and imatinib-resistant patients with CML.  Studies in patients with chronic-phase CML have shown that around 50% of patients who achieve and maintain undetectable BCR-ABL1 transcript levels for at least 2 years remain disease-free after the withdrawal of treatment.  Here we characterize ABL001 (asciminib), a potent and selective allosteric ABL1 inhibitor that is undergoing clin. development testing in patients with CML and Philadelphia chromosome-pos. (Ph+) acute lymphoblastic leukemia.  In contrast to catalytic-site ABL1 kinase inhibitors, ABL001 binds to the myristoyl pocket of ABL1 and induces the formation of an inactive kinase conformation.  ABL001 and second-generation catalytic inhibitors have similar cellular potencies but distinct patterns of resistance mutations, with genetic barcoding studies revealing pre-existing clonal populations with no shared resistance between ABL001 and the catalytic inhibitor nilotinib.  Consistent with this profile, acquired resistance was obsd. with single-agent therapy in mice; however, the combination of ABL001 and nilotinib led to complete disease control and eradicated CML xenograft tumors without recurrence after the cessation of treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreJ5YSS_wiS7Vg90H21EOLACvtfcHk0liezcc0RFz-ZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXltVWntbc%253D&md5=20d00f222ab8d9387cb340a9531b8ba5</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fnature21702&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature21702%26sid%3Dliteratum%253Aachs%26aulast%3DWylie%26aufirst%3DA.%2BA.%26aulast%3DSchoepfer%26aufirst%3DJ.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DCowan-Jacob%26aufirst%3DS.%2BW.%26aulast%3DLoo%26aufirst%3DA.%26aulast%3DFuret%26aufirst%3DP.%26aulast%3DMarzinzik%26aufirst%3DA.%2BL.%26aulast%3DPelle%26aufirst%3DX.%26aulast%3DDonovan%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DW.%26aulast%3DBuonamici%26aufirst%3DS.%26aulast%3DHassan%26aufirst%3DA.%2BQ.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DIyer%26aufirst%3DV.%26aulast%3DPalmer%26aufirst%3DM.%26aulast%3DBerellini%26aufirst%3DG.%26aulast%3DDodd%26aufirst%3DS.%26aulast%3DThohan%26aufirst%3DS.%26aulast%3DBitter%26aufirst%3DH.%26aulast%3DBranford%26aufirst%3DS.%26aulast%3DRoss%26aufirst%3DD.%2BM.%26aulast%3DHughes%26aufirst%3DT.%2BP.%26aulast%3DPetruzzelli%26aufirst%3DL.%26aulast%3DVanasse%26aufirst%3DK.%2BG.%26aulast%3DWarmuth%26aufirst%3DM.%26aulast%3DHofmann%26aufirst%3DF.%26aulast%3DKeen%26aufirst%3DN.%2BJ.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26atitle%3DThe%2520Allosteric%2520Inhibitor%2520ABL001%2520Enables%2520Dual%2520Targeting%2520of%2520BCR%25E2%2580%2593ABL1%26jtitle%3DNature%26date%3D2017%26volume%3D543%26spage%3D733%26epage%3D737%26doi%3D10.1038%2Fnature21702" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanders, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmena, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bright, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haire, L. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Underwood, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heath, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giordanetto, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carling, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gamblin, S. J.</span></span> <span> </span><span class="NLM_article-title">Structural Basis of AMPK Regulation by Small Molecule Activators</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">3017</span>, <span class="refDoi"> DOI: 10.1038/ncomms4017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1038%2Fncomms4017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=24352254" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A280%3ADC%252BC2c3oslGmtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=3017&author=B.+Xiaoauthor=M.+J.+Sandersauthor=D.+Carmenaauthor=N.+J.+Brightauthor=L.+F.+Haireauthor=E.+Underwoodauthor=B.+R.+Patelauthor=R.+B.+Heathauthor=P.+A.+Walkerauthor=S.+Hallenauthor=F.+Giordanettoauthor=S.+R.+Martinauthor=D.+Carlingauthor=S.+J.+Gamblin&title=Structural+Basis+of+AMPK+Regulation+by+Small+Molecule+Activators&doi=10.1038%2Fncomms4017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of AMPK regulation by small molecule activators</span></div><div class="casAuthors">Xiao Bing; Sanders Matthew J; Heath Richard B; Carmena David; Bright Nicola J; Patel Bhakti R; Carling David; Haire Lesley F; Underwood Elizabeth; Walker Philip A; Martin Stephen R; Gamblin Steven J; Hallen Stefan; Giordanetto Fabrizio</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">3017</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">AMP-activated protein kinase (AMPK) plays a major role in regulating cellular energy balance by sensing and responding to increases in AMP/ADP concentration relative to ATP.  Binding of AMP causes allosteric activation of the enzyme and binding of either AMP or ADP promotes and maintains the phosphorylation of threonine 172 within the activation loop of the kinase.  AMPK has attracted widespread interest as a potential therapeutic target for metabolic diseases including type 2 diabetes and, more recently, cancer.  A number of direct AMPK activators have been reported as having beneficial effects in treating metabolic diseases, but there has been no structural basis for activator binding to AMPK.  Here we present the crystal structure of human AMPK in complex with a small molecule activator that binds at a site between the kinase domain and the carbohydrate-binding module, stabilising the interaction between these two components.  The nature of the activator-binding pocket suggests the involvement of an additional, as yet unidentified, metabolite in the physiological regulation of AMPK.  Importantly, the structure offers new opportunities for the design of small molecule activators of AMPK for treatment of metabolic disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTKRk4-m-rMp1w2ofyr07jXfW6udTcc2eaO3jAPAlA6qrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2c3oslGmtQ%253D%253D&md5=b261b7498514e258ea6ec99a34d04794</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fncomms4017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms4017%26sid%3Dliteratum%253Aachs%26aulast%3DXiao%26aufirst%3DB.%26aulast%3DSanders%26aufirst%3DM.%2BJ.%26aulast%3DCarmena%26aufirst%3DD.%26aulast%3DBright%26aufirst%3DN.%2BJ.%26aulast%3DHaire%26aufirst%3DL.%2BF.%26aulast%3DUnderwood%26aufirst%3DE.%26aulast%3DPatel%26aufirst%3DB.%2BR.%26aulast%3DHeath%26aufirst%3DR.%2BB.%26aulast%3DWalker%26aufirst%3DP.%2BA.%26aulast%3DHallen%26aufirst%3DS.%26aulast%3DGiordanetto%26aufirst%3DF.%26aulast%3DMartin%26aufirst%3DS.%2BR.%26aulast%3DCarling%26aufirst%3DD.%26aulast%3DGamblin%26aufirst%3DS.%2BJ.%26atitle%3DStructural%2520Basis%2520of%2520AMPK%2520Regulation%2520by%2520Small%2520Molecule%2520Activators%26jtitle%3DNat.%2520Commun.%26date%3D2013%26volume%3D4%26spage%3D3017%26doi%3D10.1038%2Fncomms4017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kufareva, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bestgen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brear, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prudent, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laudet, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moucadel, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ettaoussi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sautel, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krimm, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filhol, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borgne, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lomberget, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cochet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abagyan, R.</span></span> <span> </span><span class="NLM_article-title">Discovery of Holoenzyme-Disrupting Chemicals as Substrate-Selective Ck2 Inhibitors</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">15893</span>, <span class="refDoi"> DOI: 10.1038/s41598-019-52141-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1038%2Fs41598-019-52141-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=31685885" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A280%3ADC%252BB3MjltVOnuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=15893&author=I.+Kufarevaauthor=B.+Bestgenauthor=P.+Brearauthor=R.+Prudentauthor=B.+Laudetauthor=V.+Moucadelauthor=M.+Ettaoussiauthor=C.+F.+Sautelauthor=I.+Krimmauthor=M.+Engelauthor=O.+Filholauthor=M.+L.+Borgneauthor=T.+Lombergetauthor=C.+Cochetauthor=R.+Abagyan&title=Discovery+of+Holoenzyme-Disrupting+Chemicals+as+Substrate-Selective+Ck2+Inhibitors&doi=10.1038%2Fs41598-019-52141-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of holoenzyme-disrupting chemicals as substrate-selective CK2 inhibitors</span></div><div class="casAuthors">Kufareva Irina; Abagyan Ruben; Bestgen Benoit; Ettaoussi Mohamed; Borgne Marc Le; Lomberget Thierry; Bestgen Benoit; Engel Matthias; Bestgen Benoit; Prudent Renaud; Laudet Beatrice; Moucadel Virginie; Sautel Celine F; Filhol Odile; Cochet Claude; Bestgen Benoit; Brear Paul; Prudent Renaud; Laudet Beatrice; Moucadel Virginie; Sautel Celine F; Krimm Isabelle</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">15893</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">CK2 is a constitutively active protein kinase overexpressed in numerous malignancies.  Interaction between CK2α and CK2β subunits is essential for substrate selectivity.  The CK2α/CK2β interface has been previously targeted by peptides to achieve functional effects; however, no small molecules modulators were identified due to pocket flexibility and open shape.  Here we generated numerous plausible conformations of the interface using the fumigation modeling protocol, and virtually screened a compound library to discover compound 1 that suppressed CK2α/CK2β interaction in vitro and inhibited CK2 in a substrate-selective manner.  Orthogonal SPR, crystallography, and NMR experiments demonstrated that 4 and 6, improved analogs of 1, bind to CK2α as predicted.  Both inhibitors alter CK2 activity in cells through inhibition of CK2 holoenzyme formation.  Treatment with 6 suppressed MDA-MB231 triple negative breast cancer cell growth and induced apoptosis.  Altogether, our findings exemplify an innovative computational-experimental approach and identify novel non-peptidic inhibitors of CK2 subunit interface disclosing substrate-selective functional effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSmE_81mdt7jNenUz0lpNqnfW6udTcc2eaO3jAPAlA6qrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MjltVOnuw%253D%253D&md5=9cbb832ee1fc84a087151c72111f6808</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fs41598-019-52141-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-019-52141-5%26sid%3Dliteratum%253Aachs%26aulast%3DKufareva%26aufirst%3DI.%26aulast%3DBestgen%26aufirst%3DB.%26aulast%3DBrear%26aufirst%3DP.%26aulast%3DPrudent%26aufirst%3DR.%26aulast%3DLaudet%26aufirst%3DB.%26aulast%3DMoucadel%26aufirst%3DV.%26aulast%3DEttaoussi%26aufirst%3DM.%26aulast%3DSautel%26aufirst%3DC.%2BF.%26aulast%3DKrimm%26aufirst%3DI.%26aulast%3DEngel%26aufirst%3DM.%26aulast%3DFilhol%26aufirst%3DO.%26aulast%3DBorgne%26aufirst%3DM.%2BL.%26aulast%3DLomberget%26aufirst%3DT.%26aulast%3DCochet%26aufirst%3DC.%26aulast%3DAbagyan%26aufirst%3DR.%26atitle%3DDiscovery%2520of%2520Holoenzyme-Disrupting%2520Chemicals%2520as%2520Substrate-Selective%2520Ck2%2520Inhibitors%26jtitle%3DSci.%2520Rep.%26date%3D2019%26volume%3D9%26spage%3D15893%26doi%3D10.1038%2Fs41598-019-52141-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Converso, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartingh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garbaccio, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tasber, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rickert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraley, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kreatsoulas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stirdivant, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drakas, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamilton, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buser, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abrams, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lobell, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sepp-Lorenzino, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zugay-Murphy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sardana, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munshi, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jezequel-Sur, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuck, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartman, G. D.</span></span> <span> </span><span class="NLM_article-title">Development of Thioquinazolinones, Allosteric Chk1 Kinase Inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1240</span>– <span class="NLM_lpage">1244</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2008.12.076</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1016%2Fj.bmcl.2008.12.076" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=19155174" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BD1MXit1Wrurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=1240-1244&author=A.+Conversoauthor=T.+Hartinghauthor=R.+M.+Garbaccioauthor=E.+Tasberauthor=K.+Rickertauthor=M.+E.+Fraleyauthor=Y.+Yanauthor=C.+Kreatsoulasauthor=S.+Stirdivantauthor=B.+Drakasauthor=E.+S.+Walshauthor=K.+Hamiltonauthor=C.+A.+Buserauthor=X.+Maoauthor=M.+T.+Abramsauthor=S.+C.+Beckauthor=W.+Taoauthor=R.+Lobellauthor=L.+Sepp-Lorenzinoauthor=J.+Zugay-Murphyauthor=V.+Sardanaauthor=S.+K.+Munshiauthor=S.+M.+Jezequel-Surauthor=P.+D.+Zuckauthor=G.+D.+Hartman&title=Development+of+Thioquinazolinones%2C+Allosteric+Chk1+Kinase+Inhibitors&doi=10.1016%2Fj.bmcl.2008.12.076"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Development of thioquinazolinones, allosteric Chk1 kinase inhibitors</span></div><div class="casAuthors">Converso, Antonella; Hartingh, Timothy; Garbaccio, Robert M.; Tasber, Edward; Rickert, Keith; Fraley, Mark E.; Yan, Youwei; Kreatsoulas, Constantine; Stirdivant, Steve; Drakas, Bob; Walsh, Eileen S.; Hamilton, Kelly; Buser, Carolyn A.; Mao, Xianzhi; Abrams, Marc T.; Beck, Stephen C.; Tao, Weikang; Lobell, Rob; Sepp-Lorenzino, Laura; Zugay-Murphy, Joan; Sardana, Vinod; Munshi, Sanjeev K.; Jezequel-Sur, Sylvie Marie; Zuck, Paul D.; Hartman, George D.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1240-1244</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A high throughput screening campaign was designed to identify allosteric inhibitors of Chk1 kinase by testing compds. at high concn.  Activity was then obsd. at Km for ATP and at near-physiol. concns. of ATP.  This strategy led to the discovery of a non-ATP competitive thioquinazolinone series which was optimized for potency and stability.  An X-ray crystal structure for the complex of our best inhibitor bound to Chk1 was solved, indicating that it binds to an allosteric site ∼13 Å from the ATP binding site.  Preliminary data is presented for several of these compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9bIRNnirJ4bVg90H21EOLACvtfcHk0likx4OISG1HdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXit1Wrurc%253D&md5=e91440930e976b9aafcfbb1597679511</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2008.12.076&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2008.12.076%26sid%3Dliteratum%253Aachs%26aulast%3DConverso%26aufirst%3DA.%26aulast%3DHartingh%26aufirst%3DT.%26aulast%3DGarbaccio%26aufirst%3DR.%2BM.%26aulast%3DTasber%26aufirst%3DE.%26aulast%3DRickert%26aufirst%3DK.%26aulast%3DFraley%26aufirst%3DM.%2BE.%26aulast%3DYan%26aufirst%3DY.%26aulast%3DKreatsoulas%26aufirst%3DC.%26aulast%3DStirdivant%26aufirst%3DS.%26aulast%3DDrakas%26aufirst%3DB.%26aulast%3DWalsh%26aufirst%3DE.%2BS.%26aulast%3DHamilton%26aufirst%3DK.%26aulast%3DBuser%26aufirst%3DC.%2BA.%26aulast%3DMao%26aufirst%3DX.%26aulast%3DAbrams%26aufirst%3DM.%2BT.%26aulast%3DBeck%26aufirst%3DS.%2BC.%26aulast%3DTao%26aufirst%3DW.%26aulast%3DLobell%26aufirst%3DR.%26aulast%3DSepp-Lorenzino%26aufirst%3DL.%26aulast%3DZugay-Murphy%26aufirst%3DJ.%26aulast%3DSardana%26aufirst%3DV.%26aulast%3DMunshi%26aufirst%3DS.%2BK.%26aulast%3DJezequel-Sur%26aufirst%3DS.%2BM.%26aulast%3DZuck%26aufirst%3DP.%2BD.%26aulast%3DHartman%26aufirst%3DG.%2BD.%26atitle%3DDevelopment%2520of%2520Thioquinazolinones%252C%2520Allosteric%2520Chk1%2520Kinase%2520Inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D1240%26epage%3D1244%26doi%3D10.1016%2Fj.bmcl.2008.12.076" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vanderpool, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ping, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergqvist, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alton, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phonephaly, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rui, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grant, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quenzer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Margosiak, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Register, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ermolieff, J.</span></span> <span> </span><span class="NLM_article-title">Characterization of the Chk1 Allosteric Inhibitor Binding Site</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">9823</span>– <span class="NLM_lpage">9830</span>, <span class="refDoi"> DOI: 10.1021/bi900258v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi900258v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFOiu7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2009&pages=9823-9830&author=D.+Vanderpoolauthor=T.+O.+Johnsonauthor=C.+Pingauthor=S.+Bergqvistauthor=G.+Altonauthor=S.+Phonephalyauthor=E.+Ruiauthor=C.+Luoauthor=Y.+L.+Dengauthor=S.+Grantauthor=T.+Quenzerauthor=S.+Margosiakauthor=J.+Registerauthor=E.+Brownauthor=J.+Ermolieff&title=Characterization+of+the+Chk1+Allosteric+Inhibitor+Binding+Site&doi=10.1021%2Fbi900258v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of the CHK1 Allosteric Inhibitor Binding Site</span></div><div class="casAuthors">Vanderpool, Darin; Johnson, Ted O.; Ping, Chen; Bergqvist, Simon; Alton, Gordon; Phonephaly, Soneprasith; Rui, Eugene; Luo, Chun; Deng, Ya-Li; Grant, Stephan; Quenzer, Terri; Margosiak, Steve; Register, James; Brown, Ed; Ermolieff, Jacques</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">9823-9830</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Checkpoint kinase 1 (CHK1) is a key element in the DNA damage response pathway and plays a crucial role in the S-G2-phase checkpoint.  Inhibiting CHK1 is a therapeutic strategy involving abrogation of the G2/M mitotic checkpoint defense of tumor cells toward lethal damage induced by DNA-directed chemotherapeutic agents.  To date, most CHK1 inhibition approaches have involved targeting the ATP site of this kinase.  In this study, we provide crystallog. and kinetic characterization of two small mol. inhibitors that bind to an allosteric site in the proximity of the CHK1 substrate site.  Anal. of kinetic and biophys. data has led to the conclusion that these small mol. allosteric site inhibitors of CHK1 are reversible and are neither ATP- nor peptide substrate-competitive.  Ki values of 1.89 and 0.15 μM, resp., have been detd. for these compds. using a mixed inhibitor kinetic anal.  Cocrystal structures of the inhibitors bound to CHK1 reveal an allosteric site, unique to CHK1, located in the C-terminal domain and consisting of a shallow groove linked to a small hydrophobic pocket.  The pocket displays induced fit characteristics in the presence of the two inhibitors.  These findings establish the potential for the development of highly selective CHK1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMx2Syic38ULVg90H21EOLACvtfcHk0likx4OISG1HdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFOiu7vO&md5=463e6df560e0cc8b3e8fed96be567284</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fbi900258v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi900258v%26sid%3Dliteratum%253Aachs%26aulast%3DVanderpool%26aufirst%3DD.%26aulast%3DJohnson%26aufirst%3DT.%2BO.%26aulast%3DPing%26aufirst%3DC.%26aulast%3DBergqvist%26aufirst%3DS.%26aulast%3DAlton%26aufirst%3DG.%26aulast%3DPhonephaly%26aufirst%3DS.%26aulast%3DRui%26aufirst%3DE.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DDeng%26aufirst%3DY.%2BL.%26aulast%3DGrant%26aufirst%3DS.%26aulast%3DQuenzer%26aufirst%3DT.%26aulast%3DMargosiak%26aufirst%3DS.%26aulast%3DRegister%26aufirst%3DJ.%26aulast%3DBrown%26aufirst%3DE.%26aulast%3DErmolieff%26aufirst%3DJ.%26atitle%3DCharacterization%2520of%2520the%2520Chk1%2520Allosteric%2520Inhibitor%2520Binding%2520Site%26jtitle%3DBiochemistry%26date%3D2009%26volume%3D48%26spage%3D9823%26epage%3D9830%26doi%3D10.1021%2Fbi900258v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Fusco, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brear, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iegre, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georgiou, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sore, H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hyvönen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spring, D. R.</span></span> <span> </span><span class="NLM_article-title">A Fragment-Based Approach Leading to the Discovery of a Novel Binding Site and the Selective CK2 Inhibitor CAM4066</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">3471</span>– <span class="NLM_lpage">3482</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2017.04.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1016%2Fj.bmc.2017.04.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=28495381" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC2sXntlWktbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=3471-3482&author=C.+De%0AFuscoauthor=P.+Brearauthor=J.+Iegreauthor=K.+H.+Georgiouauthor=H.+F.+Soreauthor=M.+Hyv%C3%B6nenauthor=D.+R.+Spring&title=A+Fragment-Based+Approach+Leading+to+the+Discovery+of+a+Novel+Binding+Site+and+the+Selective+CK2+Inhibitor+CAM4066&doi=10.1016%2Fj.bmc.2017.04.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">A fragment-based approach leading to the discovery of a novel binding site and the selective CK2 inhibitor CAM4066</span></div><div class="casAuthors">De Fusco, Claudia; Brear, Paul; Iegre, Jessica; Georgiou, Kathy Hadje; Sore, Hannah F.; Hyvonen, Marko; Spring, David R.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3471-3482</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Recently the authors reported the discovery of a potent and selective CK2α inhibitor CAM4066.  This compd. inhibits CK2 activity by exploiting a pocket located outside the ATP binding site (αD pocket).  Here the authors describe in detail the journey that led to the discovery of CAM4066 using the challenging fragment linking strategy.  Specifically, the authors aimed to develop inhibitors by linking a high-affinity fragment anchored in the αD site to a weakly binding warhead fragment occupying the ATP site.  Moreover, the authors describe the remarkable impact that mol. modeling had on the development of this novel chem. tool.  The work described herein shows potential for the development of a novel class of CK2 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-es_MXrRt5bVg90H21EOLACvtfcHk0likx4OISG1HdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXntlWktbc%253D&md5=2c4bf2e6b0528149dc09d04badfdeab3</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2017.04.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2017.04.037%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BFusco%26aufirst%3DC.%26aulast%3DBrear%26aufirst%3DP.%26aulast%3DIegre%26aufirst%3DJ.%26aulast%3DGeorgiou%26aufirst%3DK.%2BH.%26aulast%3DSore%26aufirst%3DH.%2BF.%26aulast%3DHyv%25C3%25B6nen%26aufirst%3DM.%26aulast%3DSpring%26aufirst%3DD.%2BR.%26atitle%3DA%2520Fragment-Based%2520Approach%2520Leading%2520to%2520the%2520Discovery%2520of%2520a%2520Novel%2520Binding%2520Site%2520and%2520the%2520Selective%2520CK2%2520Inhibitor%2520CAM4066%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2017%26volume%3D25%26spage%3D3471%26epage%3D3482%26doi%3D10.1016%2Fj.bmc.2017.04.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McIntyre, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vrzal, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birchall, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mpamhanga, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coombs, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richards, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veverka, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delft, F. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merritt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayliss, R.</span></span> <span> </span><span class="NLM_article-title">Characterization of Three Druggable Hot-Spots in the Aurora-a/Tpx2 Interaction Using Biochemical, Biophysical, and Fragment-Based Approaches</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">2906</span>– <span class="NLM_lpage">2914</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00537</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00537" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1KqsrbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=2906-2914&author=P.+J.+McIntyreauthor=P.+M.+Collinsauthor=L.+Vrzalauthor=K.+Birchallauthor=L.+H.+Arnoldauthor=C.+Mpamhangaauthor=P.+J.+Coombsauthor=S.+G.+Burgessauthor=M.+W.+Richardsauthor=A.+Winterauthor=V.+Veverkaauthor=F.+V.+Delftauthor=A.+Merrittauthor=R.+Bayliss&title=Characterization+of+Three+Druggable+Hot-Spots+in+the+Aurora-a%2FTpx2+Interaction+Using+Biochemical%2C+Biophysical%2C+and+Fragment-Based+Approaches&doi=10.1021%2Facschembio.7b00537"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of Three Druggable Hot-Spots in the Aurora-A/TPX2 Interaction Using Biochemical, Biophysical, and Fragment-Based Approaches</span></div><div class="casAuthors">McIntyre, Patrick J.; Collins, Patrick M.; Vrzal, Lukas; Birchall, Kristian; Arnold, Laurence H.; Mpamhanga, Chido; Coombs, Peter J.; Burgess, Selena G.; Richards, Mark W.; Winter, Anja; Veverka, Vaclav; Delft, Frank von; Merritt, Andy; Bayliss, Richard</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2906-2914</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The mitotic kinase Aurora-A and its partner protein TPX2 (Targeting Protein for Xenopus kinesin-like protein 2), are overexpressed in cancers and it has been proposed that they work together as an oncogenic holoenzyme.  TPX2 is responsible for activating Aurora-A during mitosis, ensuring proper cell division.  Disruption of the interface with TPX2 is therefore a potential target for novel anti-cancer drugs that exploit the increased sensitivity of cancer cells to mitotic stress.  Here, we investigate the interface using co-pptn. assays and isothermal titrn. calorimetry to quantify the energetic contribution of individual residues of TPX2.  Residues Tyr8, Tyr10, Phe16 and Trp34 of TPX2 are shown to be crucial for robust complex formation, suggesting that the interaction could be abrogated through blocking any of the three pockets on Aurora-A that complement these residues.  Phosphorylation of Aurora-A on Thr288 is also necessary for high-affinity binding and here we identify arginine residues that communicate the phosphorylation of Thr288 to the TPX2 binding site.  With these findings in mind, we conducted a high-throughput X-ray crystallog.-based screen of 1255 fragments against Aurora-A and identified 59 hits.  Over three-quarters of these hits bound to the pockets described above, both validating our identification of hot-spots and demonstrating the druggability of this protein-protein interaction.  Our study exemplifies the potential of high-throughput crystallog. facilities such as XChem to aid drug discovery.  These results will accelerate the development of chem. inhibitors of the Aurora-A/TPX2 interaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojoCM8ethY8rVg90H21EOLACvtfcHk0ljw2g1rO7H21w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1KqsrbK&md5=5d4fcd13e24ae8776fcfce019ec10a0c</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00537&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00537%26sid%3Dliteratum%253Aachs%26aulast%3DMcIntyre%26aufirst%3DP.%2BJ.%26aulast%3DCollins%26aufirst%3DP.%2BM.%26aulast%3DVrzal%26aufirst%3DL.%26aulast%3DBirchall%26aufirst%3DK.%26aulast%3DArnold%26aufirst%3DL.%2BH.%26aulast%3DMpamhanga%26aufirst%3DC.%26aulast%3DCoombs%26aufirst%3DP.%2BJ.%26aulast%3DBurgess%26aufirst%3DS.%2BG.%26aulast%3DRichards%26aufirst%3DM.%2BW.%26aulast%3DWinter%26aufirst%3DA.%26aulast%3DVeverka%26aufirst%3DV.%26aulast%3DDelft%26aufirst%3DF.%2BV.%26aulast%3DMerritt%26aufirst%3DA.%26aulast%3DBayliss%26aufirst%3DR.%26atitle%3DCharacterization%2520of%2520Three%2520Druggable%2520Hot-Spots%2520in%2520the%2520Aurora-a%252FTpx2%2520Interaction%2520Using%2520Biochemical%252C%2520Biophysical%252C%2520and%2520Fragment-Based%2520Approaches%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D2906%26epage%3D2914%26doi%3D10.1021%2Facschembio.7b00537" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Engel, M.</span></span> <span> </span><span class="NLM_article-title">The PIF Pocket of AGC Kinases: A Target Site for Allosteric Modulators and Protein–Protein Interaction Inhibitors</span>. In  <i>Protein-Protein Interactions in Drug Discovery; Methods and Principles in Medicinal Chemistry</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mannhold, R.</span>, <span class="NLM_string-name">Kubinyi, H.</span>, <span class="NLM_string-name">Folkers, G.</span></span>; Eds.; <span class="NLM_publisher-name">Wiley-VCH</span>: <span class="NLM_publisher-loc">Weinheim, Germany</span>, <span class="NLM_year">2013</span>; pp  <span class="NLM_fpage">187</span>– <span class="NLM_lpage">223</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1002%2F9783527648207.ch9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&pages=187-223&author=M.+Engelauthor=R.+Mannhold&author=H.+Kubinyi&author=G.+Folkers&title=Protein-Protein+Interactions+in+Drug+Discovery%3B+Methods+and+Principles+in+Medicinal+Chemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1002%2F9783527648207.ch9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9783527648207.ch9%26sid%3Dliteratum%253Aachs%26aulast%3DEngel%26aufirst%3DM.%26atitle%3DThe%2520PIF%2520Pocket%2520of%2520AGC%2520Kinases%253A%2520A%2520Target%2520Site%2520for%2520Allosteric%2520Modulators%2520and%2520Protein%25E2%2580%2593Protein%2520Interaction%2520Inhibitors%26btitle%3DProtein-Protein%2520Interactions%2520in%2520Drug%2520Discovery%253B%2520Methods%2520and%2520Principles%2520in%2520Medicinal%2520Chemistry%26aulast%3DMannhold%26aufirst%3DR.%26pub%3DWiley-VCH%26date%3D2013%26spage%3D187%26epage%3D223" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frödin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antal, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dummler, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deak, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gammeltoft, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biondi, R. M.</span></span> <span> </span><span class="NLM_article-title">A Phosphoserine/Threonine-Binding Pocket in AGC Kinases and PDK1Mediates Activation by Hydrophobic Motif Phosphorylation</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">5396</span>– <span class="NLM_lpage">5407</span>, <span class="refDoi"> DOI: 10.1093/emboj/cdf551</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1093%2Femboj%2Fcdf551" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=12374740" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A280%3ADC%252BD38nlvVWhsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2002&pages=5396-5407&author=M.+Fr%C3%B6dinauthor=T.+L.+Antalauthor=B.+A.+Dummlerauthor=C.+J.+Jensenauthor=M.+Deakauthor=S.+Gammeltoftauthor=R.+M.+Biondi&title=A+Phosphoserine%2FThreonine-Binding+Pocket+in+AGC+Kinases+and+PDK1Mediates+Activation+by+Hydrophobic+Motif+Phosphorylation&doi=10.1093%2Femboj%2Fcdf551"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">A phosphoserine/threonine-binding pocket in AGC kinases and PDK1 mediates activation by hydrophobic motif phosphorylation</span></div><div class="casAuthors">Frodin Morten; Antal Torben L; Dummler Bettina A; Jensen Claus J; Deak Maria; Gammeltoft Steen; Biondi Ricardo M</div><div class="citationInfo"><span class="NLM_cas:title">The EMBO journal</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5396-407</span>
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    </div><div class="casAbstract">The growth factor-activated AGC protein kinases RSK, S6K, PKB, MSK and SGK are activated by serine/threonine phosphorylation in the activation loop and in the hydrophobic motif, C-terminal to the kinase domain.  In some of these kinases, phosphorylation of the hydrophobic motif creates a specific docking site that recruits and activates PDK1, which then phosphorylates the activation loop.  Here, we discover a pocket in the kinase domain of PDK1 that recognizes the phosphoserine/phosphothreonine in the hydrophobic motif by identifying two oppositely positioned arginine and lysine residues that bind the phosphate.  Moreover, we demonstrate that RSK2, S6K1, PKBalpha, MSK1 and SGK1 contain a similar phosphate-binding pocket, which they use for intramolecular interaction with their own phosphorylated hydrophobic motif.  Molecular modelling and experimental data provide evidence for a common activation mechanism in which the phosphorylated hydrophobic motif and activation loop act on the alphaC-helix of the kinase structure to induce synergistic stimulation of catalytic activity.  Sequence conservation suggests that this mechanism is a key feature in activation of >40 human AGC kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSb-TVEWss6nftos8R3JUhXfW6udTcc2ebg1HmBk9I0bbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD38nlvVWhsw%253D%253D&md5=40a9f8370c7f7634dcc961988f252924</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1093%2Femboj%2Fcdf551&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Femboj%252Fcdf551%26sid%3Dliteratum%253Aachs%26aulast%3DFr%25C3%25B6din%26aufirst%3DM.%26aulast%3DAntal%26aufirst%3DT.%2BL.%26aulast%3DDummler%26aufirst%3DB.%2BA.%26aulast%3DJensen%26aufirst%3DC.%2BJ.%26aulast%3DDeak%26aufirst%3DM.%26aulast%3DGammeltoft%26aufirst%3DS.%26aulast%3DBiondi%26aufirst%3DR.%2BM.%26atitle%3DA%2520Phosphoserine%252FThreonine-Binding%2520Pocket%2520in%2520AGC%2520Kinases%2520and%2520PDK1Mediates%2520Activation%2520by%2520Hydrophobic%2520Motif%2520Phosphorylation%26jtitle%3DEMBO%2520J.%26date%3D2002%26volume%3D21%26spage%3D5396%26epage%3D5407%26doi%3D10.1093%2Femboj%2Fcdf551" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Simpson, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertrand, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borthwick, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campobasso, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chabanet, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coggins, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cottom, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christensen, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hobbs, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangatt, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munoz-Muriedas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliff, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott-Stevens, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Washio, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. J.</span></span> <span> </span><span class="NLM_article-title">Identification and Optimization of Novel Small C-Abl Kinase Activators Using Fragment and HTS Methodologies</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">2154</span>– <span class="NLM_lpage">2171</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01872</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01872" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFKjsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=2154-2171&author=G.+L.+Simpsonauthor=S.+M.+Bertrandauthor=J.+A.+Borthwickauthor=N.+Campobassoauthor=J.+Chabanetauthor=S.+Chenauthor=J.+Cogginsauthor=J.+Cottomauthor=S.+B.+Christensenauthor=H.+C.+Dawsonauthor=H.+L.+Evansauthor=A.+N.+Hobbsauthor=X.+Hongauthor=B.+Mangattauthor=J.+Munoz-Muriedasauthor=A.+Oliffauthor=D.+Qinauthor=P.+Scott-Stevensauthor=P.+Wardauthor=Y.+Washioauthor=J.+Yangauthor=R.+J.+Young&title=Identification+and+Optimization+of+Novel+Small+C-Abl+Kinase+Activators+Using+Fragment+and+HTS+Methodologies&doi=10.1021%2Facs.jmedchem.8b01872"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Optimization of Novel Small c-Abl Kinase Activators Using Fragment and HTS Methodologies</span></div><div class="casAuthors">Simpson, Graham L.; Bertrand, Sophie M.; Borthwick, Jennifer A.; Campobasso, Nino; Chabanet, Julien; Chen, Susan; Coggins, Julia; Cottom, Josh; Christensen, Siegfried B.; Dawson, Helen C.; Evans, Helen L.; Hobbs, Andrew N.; Hong, Xuan; Mangatt, Biju; Munoz-Muriedas, Jordi; Oliff, Allen; Qin, Donghui; Scott-Stevens, Paul; Ward, Paris; Washio, Yoshiaki; Yang, Jingsong; Young, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2154-2171</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Abelson kinase (c-Abl) is a ubiquitously expressed, nonreceptor tyrosine kinase which plays a key role in cell differentiation and survival.  It was hypothesized that transient activation of c-Abl kinase via displacement of the N-terminal autoinhibitory "myristoyl latch", may lead to an increased hematopoietic stem cell differentiation.  This would increase the nos. of circulating neutrophils and so be an effective treatment for chemotherapy-induced neutropenia.  This paper describes the discovery and optimization of a thiazole series of novel small mol. c-Abl activators, initially identified by a high throughput screening.  Subsequently, a scaffold-hop, which exploited the improved physicochem. properties of a dihydropyrazole analog, identified through fragment screening, delivered potent, sol., cell-active c-Abl activators, which demonstrated the intracellular activation of c-Abl in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoA0_N7VrekSbVg90H21EOLACvtfcHk0lg5suJaD7WH4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFKjsr8%253D&md5=c187254649943b5dd3c94d64204d4aa1</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01872&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01872%26sid%3Dliteratum%253Aachs%26aulast%3DSimpson%26aufirst%3DG.%2BL.%26aulast%3DBertrand%26aufirst%3DS.%2BM.%26aulast%3DBorthwick%26aufirst%3DJ.%2BA.%26aulast%3DCampobasso%26aufirst%3DN.%26aulast%3DChabanet%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DCoggins%26aufirst%3DJ.%26aulast%3DCottom%26aufirst%3DJ.%26aulast%3DChristensen%26aufirst%3DS.%2BB.%26aulast%3DDawson%26aufirst%3DH.%2BC.%26aulast%3DEvans%26aufirst%3DH.%2BL.%26aulast%3DHobbs%26aufirst%3DA.%2BN.%26aulast%3DHong%26aufirst%3DX.%26aulast%3DMangatt%26aufirst%3DB.%26aulast%3DMunoz-Muriedas%26aufirst%3DJ.%26aulast%3DOliff%26aufirst%3DA.%26aulast%3DQin%26aufirst%3DD.%26aulast%3DScott-Stevens%26aufirst%3DP.%26aulast%3DWard%26aufirst%3DP.%26aulast%3DWashio%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DYoung%26aufirst%3DR.%2BJ.%26atitle%3DIdentification%2520and%2520Optimization%2520of%2520Novel%2520Small%2520C-Abl%2520Kinase%2520Activators%2520Using%2520Fragment%2520and%2520HTS%2520Methodologies%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D2154%26epage%3D2171%26doi%3D10.1021%2Facs.jmedchem.8b01872" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brear, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Fusco, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hadje Georgiou, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis-Newton, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stubbs, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sore, H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkitaraman, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spring, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hyvonen, M.</span></span> <span> </span><span class="NLM_article-title">Specific Inhibition of Ck2alpha from an Anchor Outside the Active Site</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">6839</span>– <span class="NLM_lpage">6845</span>, <span class="refDoi"> DOI: 10.1039/C6SC02335E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1039%2FC6SC02335E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=28451126" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFCrtbbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=6839-6845&author=P.+Brearauthor=C.+De+Fuscoauthor=K.+Hadje+Georgiouauthor=N.+J.+Francis-Newtonauthor=C.+J.+Stubbsauthor=H.+F.+Soreauthor=A.+R.+Venkitaramanauthor=C.+Abellauthor=D.+R.+Springauthor=M.+Hyvonen&title=Specific+Inhibition+of+Ck2alpha+from+an+Anchor+Outside+the+Active+Site&doi=10.1039%2FC6SC02335E"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Specific inhibition of CK2α from an anchor outside the active site</span></div><div class="casAuthors">Brear, Paul; De Fusco, Claudia; Hadje Georgiou, Kathy; Francis-Newton, Nicola J.; Stubbs, Christopher J.; Sore, Hannah F.; Venkitaraman, Ashok R.; Abell, Chris; Spring, David R.; Hyvonen, Marko</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">6839-6845</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The development of selective inhibitors of protein kinases is challenging because of the significant conservation of the ATP binding site.  Here, we describe a new mechanism by which the protein kinase CK2α can be selectively inhibited using features outside the ATP site.  We have identified a new binding site for small mols. on CK2α adjacent to the ATP site and behind the αD loop, termed the αD pocket.  An elaborated fragment anchored in this site has been linked with a low affinity fragment binding in the ATP site, creating a novel and selective inhibitor (CAM4066) that binds CK2α with a Kd of 320 nM and shows significantly improved selectivity compared to other CK2α inhibitors.  CAM4066 shows target engagement in several cell lines and similar potency to clin. trial candidate CX4945.  Our data demonstrate that targeting a poorly conserved, cryptic pocket allows inhibition of CK2α via a novel mechanism, enabling the development of a new generation of selective CK2α inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3FqxiBkU9frVg90H21EOLACvtfcHk0lg5suJaD7WH4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFCrtbbP&md5=c4652010f4da70a128fb1dab64f811d0</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1039%2FC6SC02335E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6SC02335E%26sid%3Dliteratum%253Aachs%26aulast%3DBrear%26aufirst%3DP.%26aulast%3DDe%2BFusco%26aufirst%3DC.%26aulast%3DHadje%2BGeorgiou%26aufirst%3DK.%26aulast%3DFrancis-Newton%26aufirst%3DN.%2BJ.%26aulast%3DStubbs%26aufirst%3DC.%2BJ.%26aulast%3DSore%26aufirst%3DH.%2BF.%26aulast%3DVenkitaraman%26aufirst%3DA.%2BR.%26aulast%3DAbell%26aufirst%3DC.%26aulast%3DSpring%26aufirst%3DD.%2BR.%26aulast%3DHyvonen%26aufirst%3DM.%26atitle%3DSpecific%2520Inhibition%2520of%2520Ck2alpha%2520from%2520an%2520Anchor%2520Outside%2520the%2520Active%2520Site%26jtitle%3DChem.%2520Sci.%26date%3D2016%26volume%3D7%26spage%3D6839%26epage%3D6845%26doi%3D10.1039%2FC6SC02335E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Welsch, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snyder, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stockwell, B. R.</span></span> <span> </span><span class="NLM_article-title">Privileged Scaffolds for Library Design and Drug Discovery</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">347</span>– <span class="NLM_lpage">361</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2010.02.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1016%2Fj.cbpa.2010.02.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=20303320" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmvVWqsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2010&pages=347-361&author=M.+E.+Welschauthor=S.+A.+Snyderauthor=B.+R.+Stockwell&title=Privileged+Scaffolds+for+Library+Design+and+Drug+Discovery&doi=10.1016%2Fj.cbpa.2010.02.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Privileged scaffolds for library design and drug discovery</span></div><div class="casAuthors">Welsch, Matthew E.; Snyder, Scott A.; Stockwell, Brent R.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">347-361</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">This review explores the concept of using privileged scaffolds to identify biol. active compds. through building chem. libraries.  We hope to accomplish three main objectives: to provide one of the most comprehensive listings of privileged scaffolds; to reveal through four selected examples the present state of the art in privileged scaffold library synthesis (in hopes of inspiring new and even more creative approaches); and also to offer some thoughts on how new privileged scaffolds might be identified and exploited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWQPxdsWawiLVg90H21EOLACvtfcHk0lg5suJaD7WH4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmvVWqsLs%253D&md5=fd0c2a642f063fd6f42f2c34fd186f8a</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2010.02.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2010.02.018%26sid%3Dliteratum%253Aachs%26aulast%3DWelsch%26aufirst%3DM.%2BE.%26aulast%3DSnyder%26aufirst%3DS.%2BA.%26aulast%3DStockwell%26aufirst%3DB.%2BR.%26atitle%3DPrivileged%2520Scaffolds%2520for%2520Library%2520Design%2520and%2520Drug%2520Discovery%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D14%26spage%3D347%26epage%3D361%26doi%3D10.1016%2Fj.cbpa.2010.02.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mehndiratta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sapra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nepali, K.</span></span> <span> </span><span class="NLM_article-title">Quinazolines as Apoptosis Inducers and Inhibitors: A Review of Patent Literature</span>. <i>Recent Pat. Anti-Cancer Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2</span>– <span class="NLM_lpage">66</span>, <span class="refDoi"> DOI: 10.2174/1574892811666151218151506</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.2174%2F1574892811666151218151506" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=26681186" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC28XisVGls70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=2-66&author=S.+Mehndirattaauthor=S.+Sapraauthor=G.+Singhauthor=M.+Singhauthor=K.+Nepali&title=Quinazolines+as+Apoptosis+Inducers+and+Inhibitors%3A+A+Review+of+Patent+Literature&doi=10.2174%2F1574892811666151218151506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Quinazolines as Apoptosis Inducers and Inhibitors: A Review of Patent Literature</span></div><div class="casAuthors">Mehndiratta, Samir; Sapra, Sameer; Singh, Gurpreet; Singh, Manwinder; Nepali, Kunal</div><div class="citationInfo"><span class="NLM_cas:title">Recent Patents on Anti-Cancer Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2-66</span>CODEN:
                <span class="NLM_cas:coden">RPADDY</span>;
        ISSN:<span class="NLM_cas:issn">1574-8928</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Quinazoline scaffold has been successfully utilized for development of various inhibitors of tubulin, epidermal growth factor receptor (EGFR), polo like kinases (PLKs), Hedgehog-Gli signaling pathway and protein kinase B (PKB) /Akt pathway.  Compds. based on quinazolines have shown efficacies in μM to nM range in various cancer cell lines and thus could be useful scaffolds for development of both apoptosis inducers as well as inhibitors.  This compilation is based on various patents published till 2015 and divides the quinazolines in two main categories: Quinazolines as apoptosis inducers and as apoptosis inhibitors.  These two main categories are further sub-categorized based on the use/pharmacol. indications for these classes of patented compds.  This review will act as a tool for the researchers working on exploring the anticancer potential of quinazoline as a privileged heterocyclic.  The promising anticancer profile of some of the quinazoline based compds. clearly highlights the clin. potential of this heterocycle.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRFoI4uJ8BA7Vg90H21EOLACvtfcHk0lidtEkoYVVouw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XisVGls70%253D&md5=3ad6dd317a6a8e19d0be41e7b59d70e1</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.2174%2F1574892811666151218151506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1574892811666151218151506%26sid%3Dliteratum%253Aachs%26aulast%3DMehndiratta%26aufirst%3DS.%26aulast%3DSapra%26aufirst%3DS.%26aulast%3DSingh%26aufirst%3DG.%26aulast%3DSingh%26aufirst%3DM.%26aulast%3DNepali%26aufirst%3DK.%26atitle%3DQuinazolines%2520as%2520Apoptosis%2520Inducers%2520and%2520Inhibitors%253A%2520A%2520Review%2520of%2520Patent%2520Literature%26jtitle%3DRecent%2520Pat.%2520Anti-Cancer%2520Drug%2520Discovery%26date%3D2016%26volume%3D11%26spage%3D2%26epage%3D66%26doi%3D10.2174%2F1574892811666151218151506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ramdas, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnin, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plunkett, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallick, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Budde, R. J.</span></span> <span> </span><span class="NLM_article-title">Benzodiazepine Compounds as Inhibitors of the Src Protein Tyrosine Kinase: Screening of a Combinatorial Library of 1,4-Benzodiazepines</span>. <i>Arch. Biochem. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>368</i></span>,  <span class="NLM_fpage">394</span>– <span class="NLM_lpage">400</span>, <span class="refDoi"> DOI: 10.1006/abbi.1999.1313</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1006%2Fabbi.1999.1313" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10441393" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADyaK1MXltVahsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=1999&pages=394-400&author=L.+Ramdasauthor=B.+A.+Bunninauthor=M.+J.+Plunkettauthor=G.+Sunauthor=J.+Ellmanauthor=G.+Gallickauthor=R.+J.+Budde&title=Benzodiazepine+Compounds+as+Inhibitors+of+the+Src+Protein+Tyrosine+Kinase%3A+Screening+of+a+Combinatorial+Library+of+1%2C4-Benzodiazepines&doi=10.1006%2Fabbi.1999.1313"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Benzodiazepine compounds as inhibitors of the Src protein tyrosine kinase: screening of a combinatorial library of 1,4-benzodiazepines</span></div><div class="casAuthors">Ramdas, Latha; Bunnin, Barry A.; Plunkett, Matthew J.; Sun, Gongqin; Ellman, Jonathan; Gallick, Gary; Budde, Raymond J. A.</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">394-400</span>CODEN:
                <span class="NLM_cas:coden">ABBIA4</span>;
        ISSN:<span class="NLM_cas:issn">0003-9861</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">We screened 1680 spatially sepd. compds. of a diverse combinatorial library of 1,4-benzodiazepines for their ability to inhibit the kinase activity of protein tyrosine kinases Src, Yes, Abl, Lck, Csk, and fibroblast growth factor receptor.  This screening yielded novel ligands for the protein tyrosine kinase Src.  In the 1,4-benzodiazepine-2-one scaffold, the preferred substituent at position R1 was 4-hydroxyphenylmethyl or a 3-indolemethyl derived from a tyrosine or tryptophan used in building the benzodiazepine, while the substituent at R2, introduced by alkylating agents, was preferably arom. in nature.  The preferred ring structure introduced on the bicyclic ring of the scaffold by acid chlorides was a p-hydroxy Ph group.  The lead compd., designated as N-l-Yaa, has a l-4-hydroxyphenylmethyl ring at R1 and a biphenylmethyl substituent at R2.  The compd. has an IC50 of 73 μM against Src, 2- to 6-fold lower than against other protein tyrosine kinases and >10-fold lower than against other nucleotide-utilizing enzymes.  The mechanism of binding of N-l-Yaa to Src is mixed against the peptidic substrate with a Ki of 35 μM and noncompetitive against ATP-Mg with a Ki of 17 μM.  Multiple inhibition anal. of the lead compd. in the presence of other competitive inhibitors demonstrated that the binding of the lead compd. is nonexclusive to the other competitive inhibitor.  The inhibitor was found to be nontoxic to the AFB-13-human fibroblasts cells and inhibited the colony formation of HT-29 colon adenocarcinoma cells that are dependent on Src activity.  (c) 1999 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr94Bg8htImNrVg90H21EOLACvtfcHk0lidtEkoYVVouw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXltVahsL0%253D&md5=438dac1a4c7eb8f1cb26315e0b1b2a4b</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1006%2Fabbi.1999.1313&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fabbi.1999.1313%26sid%3Dliteratum%253Aachs%26aulast%3DRamdas%26aufirst%3DL.%26aulast%3DBunnin%26aufirst%3DB.%2BA.%26aulast%3DPlunkett%26aufirst%3DM.%2BJ.%26aulast%3DSun%26aufirst%3DG.%26aulast%3DEllman%26aufirst%3DJ.%26aulast%3DGallick%26aufirst%3DG.%26aulast%3DBudde%26aufirst%3DR.%2BJ.%26atitle%3DBenzodiazepine%2520Compounds%2520as%2520Inhibitors%2520of%2520the%2520Src%2520Protein%2520Tyrosine%2520Kinase%253A%2520Screening%2520of%2520a%2520Combinatorial%2520Library%2520of%25201%252C4-Benzodiazepines%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D1999%26volume%3D368%26spage%3D394%26epage%3D400%26doi%3D10.1006%2Fabbi.1999.1313" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span>RDKit:
Cheminformatics and Machine Learning Software, <span class="NLM_year">2013</span>; <a href="http://www.rdkit.org" class="extLink">http://www.rdkit.org</a> (accessed Apr
25, 2019).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=RDKit%3A%0ACheminformatics+and+Machine+Learning+Software%2C+2013%3B+http%3A%2F%2Fwww.rdkit.org+%28accessed+Apr%0A25%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, G. A.</span></span> <span> </span><span class="NLM_article-title">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2740</span>, <span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+Substructure+Filters+for+Removal+of+Pan+Assay+Interference+Compounds+%28PAINS%29+from+Screening+Libraries+and+for+Their+Exclusion+in+Bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0lidtEkoYVVouw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520Substructure%2520Filters%2520for%2520Removal%2520of%2520Pan%2520Assay%2520Interference%2520Compounds%2520%2528PAINS%2529%2520from%2520Screening%2520Libraries%2520and%2520for%2520Their%2520Exclusion%2520in%2520Bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaulton, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellis, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bento, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chambers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hersey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Light, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGlinchey, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michalovich, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Lazikani, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span> <span> </span><span class="NLM_article-title">ChEMBL: A Large-scale Bioactivity Database for Drug Discovery</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">D1100</span>– <span class="NLM_lpage">D1107</span>, <span class="refDoi"> DOI: 10.1093/nar/gkr777</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1093%2Fnar%2Fgkr777" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=21948594" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs12htbjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=D1100-D1107&author=A.+Gaultonauthor=L.+J.+Bellisauthor=A.+P.+Bentoauthor=J.+Chambersauthor=M.+Daviesauthor=A.+Herseyauthor=Y.+Lightauthor=S.+McGlincheyauthor=D.+Michalovichauthor=B.+Al-Lazikaniauthor=J.+P.+Overington&title=ChEMBL%3A+A+Large-scale+Bioactivity+Database+for+Drug+Discovery&doi=10.1093%2Fnar%2Fgkr777"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">ChEMBL: a large-scale bioactivity database for drug discovery</span></div><div class="casAuthors">Gaulton, Anna; Bellis, Louisa J.; Bento, A. Patricia; Chambers, Jon; Davies, Mark; Hersey, Anne; Light, Yvonne; McGlinchey, Shaun; Michalovich, David; Al-Lazikani, Bissan; Overington, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1100-D1107</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ChEMBL is an Open Data database contg. binding, functional and ADMET information for a large no. of drug-like bioactive compds.  These data are manually abstracted from the primary published literature on a regular basis, then further curated and standardized to maximize their quality and utility across a wide range of chem. biol. and drug-discovery research problems.  Currently, the database contains 5.4 million bioactivity measurements for more than 1 million compds. and 5200 protein targets.  Access is available through a web-based interface, data downloads and web services at: https://www.ebi.ac.uk/chembldb.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9e9jRENTtubVg90H21EOLACvtfcHk0liha-uh38zTCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs12htbjN&md5=aedf7793e1ca54b6a4fa272ea3ef7d0e</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr777&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr777%26sid%3Dliteratum%253Aachs%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DBellis%26aufirst%3DL.%2BJ.%26aulast%3DBento%26aufirst%3DA.%2BP.%26aulast%3DChambers%26aufirst%3DJ.%26aulast%3DDavies%26aufirst%3DM.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DLight%26aufirst%3DY.%26aulast%3DMcGlinchey%26aufirst%3DS.%26aulast%3DMichalovich%26aufirst%3DD.%26aulast%3DAl-Lazikani%26aufirst%3DB.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DChEMBL%253A%2520A%2520Large-scale%2520Bioactivity%2520Database%2520for%2520Drug%2520Discovery%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3DD1100%26epage%3DD1107%26doi%3D10.1093%2Fnar%2Fgkr777" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hussain, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rea, C.</span></span> <span> </span><span class="NLM_article-title">Computationally Efficient Algorithm to Identify Matched Molecular Pairs (MMPs) in Large Data Sets</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">339</span>– <span class="NLM_lpage">348</span>, <span class="refDoi"> DOI: 10.1021/ci900450m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci900450m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlWltr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=339-348&author=J.+Hussainauthor=C.+Rea&title=Computationally+Efficient+Algorithm+to+Identify+Matched+Molecular+Pairs+%28MMPs%29+in+Large+Data+Sets&doi=10.1021%2Fci900450m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Computationally Efficient Algorithm to Identify Matched Molecular Pairs (MMPs) in Large Data Sets</span></div><div class="casAuthors">Hussain, Jameed; Rea, Ceara</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">339-348</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Modern drug discovery organizations generate large vols. of SAR data.  A promising methodol. that can be used to mine this chem. data to identify novel structure-activity relationships is the matched mol. pair (MMP) methodol.  However, before the full potential of the MMP methodol. can be utilized, a MMP identification method that is capable of identifying all MMPs in large chem. data sets on modest computational hardware is required.  In this paper we report an algorithm that is capable of systematically generating all MMPs in chem. data sets.  Addnl., the algorithm is computationally efficient enough to be applied on large data sets.  As an example the algorithm was used to identify the MMPs in the ∼300k NIH MLSMR set.  The algorithm identified ∼5.3 million matched mol. pairs in the set.  These pairs cover ∼2.6 million unique mol. transformations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoF0rN8jaKSjrVg90H21EOLACvtfcHk0liha-uh38zTCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlWltr4%253D&md5=23964abebc23274f5c36c319bee52bab</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Fci900450m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900450m%26sid%3Dliteratum%253Aachs%26aulast%3DHussain%26aufirst%3DJ.%26aulast%3DRea%26aufirst%3DC.%26atitle%3DComputationally%2520Efficient%2520Algorithm%2520to%2520Identify%2520Matched%2520Molecular%2520Pairs%2520%2528MMPs%2529%2520in%2520Large%2520Data%2520Sets%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D339%26epage%3D348%26doi%3D10.1021%2Fci900450m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kunimoto, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Design of a Tripartite Network for the Prediction of Drug Targets</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">321</span>– <span class="NLM_lpage">330</span>, <span class="refDoi"> DOI: 10.1007/s10822-018-0098-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1007%2Fs10822-018-0098-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=29340865" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtVKru7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2018&pages=321-330&author=R.+Kunimotoauthor=J.+Bajorath&title=Design+of+a+Tripartite+Network+for+the+Prediction+of+Drug+Targets&doi=10.1007%2Fs10822-018-0098-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Design of a tripartite network for the prediction of drug targets</span></div><div class="casAuthors">Kunimoto, Ryo; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">321-330</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Drug-target networks have aided in many target prediction studies aiming at drug repurposing or the anal. of side effects.  Conventional drug-target networks are bipartite.  They contain two different types of nodes representing drugs and targets, resp., and edges indicating pairwise drug-target interactions.  In this work, we introduce a tripartite network consisting of drugs, other bioactive compds., and targets from different sources.  On the basis of analog relationships captured in the network and so-called neighbor targets of drugs, new drug targets can be inferred.  The tripartite network was found to have a stable structure and simulated network growth was accompanied by a steady increase in assortativity, reflecting increasing correlation between degrees of connected nodes leading to even network connectivity.  Local drug environments in the tripartite network typically contained neighbor targets and revealed interesting drug-compd.-target relationships for further anal.  Candidate targets were prioritized.  The tripartite network design extends std. drug-target networks and provides addnl. opportunities for drug target prediction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIv0Wn7vN4p7Vg90H21EOLACvtfcHk0liha-uh38zTCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtVKru7o%253D&md5=ea402669c22e80cfeebd7c1b954650b2</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1007%2Fs10822-018-0098-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-018-0098-x%26sid%3Dliteratum%253Aachs%26aulast%3DKunimoto%26aufirst%3DR.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DDesign%2520of%2520a%2520Tripartite%2520Network%2520for%2520the%2520Prediction%2520of%2520Drug%2520Targets%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2018%26volume%3D32%26spage%3D321%26epage%3D330%26doi%3D10.1007%2Fs10822-018-0098-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abad-Zapatero, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metz, J. T.</span></span> <span> </span><span class="NLM_article-title">Ligand Efficiency Indices as Guideposts for Drug Discovery</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">464</span>– <span class="NLM_lpage">469</span>, <span class="refDoi"> DOI: 10.1016/S1359-6446(05)03386-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1016%2FS1359-6446%2805%2903386-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=15809192" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A280%3ADC%252BD2M7nsF2iug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2005&pages=464-469&author=C.+Abad-Zapateroauthor=J.+T.+Metz&title=Ligand+Efficiency+Indices+as+Guideposts+for+Drug+Discovery&doi=10.1016%2FS1359-6446%2805%2903386-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand efficiency indices as guideposts for drug discovery</span></div><div class="casAuthors">Abad-Zapatero Cele; Metz James T</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">464-9</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSHqVfZXUq224qdXsLcGTEUfW6udTcc2eYNUcDVD3hrs7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2M7nsF2iug%253D%253D&md5=1720f263a306122c86a57463e5a39287</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2805%2903386-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252805%252903386-6%26sid%3Dliteratum%253Aachs%26aulast%3DAbad-Zapatero%26aufirst%3DC.%26aulast%3DMetz%26aufirst%3DJ.%2BT.%26atitle%3DLigand%2520Efficiency%2520Indices%2520as%2520Guideposts%2520for%2520Drug%2520Discovery%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2005%26volume%3D10%26spage%3D464%26epage%3D469%26doi%3D10.1016%2FS1359-6446%2805%2903386-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/medium/jm0c02076_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/large/jm0c02076_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02076&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/medium/jm0c02076_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/large/jm0c02076_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Binding pockets. The location of all identified binding pockets is mapped on an exemplary kinase structure shown from a front-side view focusing on the ATP site (red) and a back-side view. For each pocket, the number of unique X-ray ligands is given in bold, and UniProt IDs (UPIDs) in parentheses report the number of kinases containing the pocket.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/large/jm0c02076_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02076&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/medium/jm0c02076_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/large/jm0c02076_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Distribution of allosteric pockets. Kinases for which X-ray structures of complexes with ligands occupying allosteric binding sites were available are indicated as color-coded circles in a phylogenetic tree representation of the human kinome. Each kinase is numbered, and its name is reported below the tree representation. The resulting map reveals the distribution of allosteric pockets across the kinome.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/large/jm0c02076_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02076&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/medium/jm0c02076_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/large/jm0c02076_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Size distribution of allosteric compounds. (A) The years are reported when individual X-ray structures containing an allosteric ligand targeting each pocket were released. In addition, boxplots report for each pocket the distribution of (B) crystallographic resolution (Å) of X-ray structures with allosteric inhibitors, (C) molecular weights of allosteric inhibitors, and (D) binding efficiency index (BEI)<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> values for allosteric inhibitors. At the top of each boxplot (D), the number of inhibitors is reported for which defined potency values were available.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/large/jm0c02076_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02076&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/medium/jm0c02076_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/large/jm0c02076_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Exemplary multisite ligands. On the left, a kinase inhibitor is shown that bound to multiple sites (and site combinations) in different X-ray structures. Two distinct kinase targets are superimposed and displayed in ribbon representations (cyan, DAPK1; gray, CDK2). On the right, another kinase inhibitor is shown that occupied both the A (ATP) and D (allosteric) pocket in MAPK14.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/large/jm0c02076_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02076&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/medium/jm0c02076_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/large/jm0c02076_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Allosteric inhibitors with privileged substructures. Shown are six allosteric inhibitors with activity against different kinases. For each compound, the PDB ligand ID is provided, and privileged substructures are colored red.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/large/jm0c02076_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02076&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/medium/jm0c02076_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/large/jm0c02076_0010.jpeg" id="rightTab-GRAPHIC-d7e1161-autogenerated" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Compound/target network analysis. Shown are network sections capturing multiple analogues and their compound-target relationships. Red and green circular nodes represent ChEMBL compounds and allosteric inhibitors, respectively, and triangular nodes represent targets. Pink and blue triangular nodes indicate kinase and nonkinase targets, respectively. Three types of edges are distinguished including edges between allosteric compounds and their crystallographic kinase targets (solid black lines), any compounds and their target(s) (solid gray), and compounds forming analogue pairs (dashed gray). Allosteric kinase inhibitors from X-ray structures are labeled with their PDB ligand IDs. In the inset at the bottom defining nodes and edges, CPD stands for compound.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02076/20210121/images/large/jm0c02076_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02076&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i34">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45560" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45560" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 58 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ward, R. A.</span>; <span class="NLM_string-name">Goldberg, F. W.</span></span>, Eds.;  <i>Kinase Drug Discovery</i>; <span class="NLM_publisher-name">RSC</span>: <span class="NLM_publisher-loc">Cambridge</span>, <span class="NLM_year">2011</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1039%2F9781849733557" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=R.+A.+Ward&author=F.+W.+Goldberg&title=Kinase+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1039%2F9781849733557&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1039%252F9781849733557%26sid%3Dliteratum%253Aachs%26aulast%3DWard%26aufirst%3DR.%2BA.%26btitle%3DKinase%2520Drug%2520Discovery%26pub%3DRSC%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ward, R. A.</span>; <span class="NLM_string-name">Goldberg, F. W.</span></span>, Eds.;  <i>Kinase Drug Discovery: Modern Approaches</i>; <span class="NLM_publisher-name">RSC</span>: <span class="NLM_publisher-loc">Cambridge</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1039%2F9781788013093" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=R.+A.+Ward&author=F.+W.+Goldberg&title=Kinase+Drug+Discovery%3A+Modern+Approaches"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1039%2F9781788013093&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1039%252F9781788013093%26sid%3Dliteratum%253Aachs%26aulast%3DWard%26aufirst%3DR.%2BA.%26btitle%3DKinase%2520Drug%2520Discovery%253A%2520Modern%2520Approaches%26pub%3DRSC%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Kinase Inhibitors: The Road Ahead</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">353</span>– <span class="NLM_lpage">376</span>, <span class="refDoi"> DOI: 10.1038/nrd.2018.21</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1038%2Fnrd.2018.21" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=29545548" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC1cXks12ktbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=353-376&author=F.+M.+Fergusonauthor=N.+S.+Gray&title=Kinase+Inhibitors%3A+The+Road+Ahead&doi=10.1038%2Fnrd.2018.21"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase inhibitors: the road ahead</span></div><div class="casAuthors">Ferguson, Fleur M.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">353-377</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Receptor tyrosine kinase signaling pathways have been successfully targeted to inhibit proliferation and angiogenesis for cancer therapy.  However, kinase deregulation has been firmly demonstrated to play an essential role in virtually all major disease areas.  Kinase inhibitor drug discovery programs have recently broadened their focus to include an expanded range of kinase targets and therapeutic areas.  In this Review, we provide an overview of the novel targets, biol. processes and disease areas that kinase-targeting small mols. are being developed against, highlight the assocd. challenges and assess the strategies and technologies that are enabling efficient generation of highly optimized kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEZPospb1TP7Vg90H21EOLACvtfcHk0ljo3fc-5Jq9Aw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXks12ktbg%253D&md5=34e42911d8a70301905460c3507e737a</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2018.21&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2018.21%26sid%3Dliteratum%253Aachs%26aulast%3DFerguson%26aufirst%3DF.%2BM.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DKinase%2520Inhibitors%253A%2520The%2520Road%2520Ahead%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2018%26volume%3D17%26spage%3D353%26epage%3D376%26doi%3D10.1038%2Fnrd.2018.21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Manning, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whyte, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunter, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sudarsanam, S.</span></span> <span> </span><span class="NLM_article-title">The Protein Kinase Complement of the Human Genome</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>298</i></span>,  <span class="NLM_fpage">1912</span>– <span class="NLM_lpage">1934</span>, <span class="refDoi"> DOI: 10.1126/science.1075762</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1126%2Fscience.1075762" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=12471243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BD38Xpt1Wisb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=298&publication_year=2002&pages=1912-1934&author=G.+Manningauthor=D.+Whyteauthor=R.+Martinezauthor=T.+Hunterauthor=S.+Sudarsanam&title=The+Protein+Kinase+Complement+of+the+Human+Genome&doi=10.1126%2Fscience.1075762"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Kinase Complement of the Human Genome</span></div><div class="casAuthors">Manning, G.; Whyte, D. B.; Martinez, R.; Hunter, T.; Sudarsanam, S.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">298</span>
        (<span class="NLM_cas:issue">5600</span>),
    <span class="NLM_cas:pages">1912-1916, 1933-1934</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">We have catalogued the protein kinase complement of the human genome (the "kinome") using public and proprietary genomic, complementary DNA, and expressed sequence tag (EST) sequences.  This provides a starting point for comprehensive anal. of protein phosphorylation in normal and disease states, as well as a detailed view of the current state of human genome anal. through a focus on one large gene family.  We identify 518 putative protein kinase genes, of which 71 have not previously been reported or described as kinases, and we extend or correct the protein sequences of 56 more kinases.  New genes include members of well-studied families as well as previously unidentified families, some of which are conserved in model organisms.  Classification and comparison with model organism kinomes identified orthologous groups and highlighted expansions specific to human and other lineages.  We also identified 106 protein kinase pseudogenes.  Chromosomal mapping revealed several small clusters of kinase genes and revealed that 244 kinases map to disease loci or cancer amplicons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTj1hf_0cQcrVg90H21EOLACvtfcHk0ljo3fc-5Jq9Aw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xpt1Wisb0%253D&md5=b3def83bace52257a3252a3347b8bb92</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1126%2Fscience.1075762&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1075762%26sid%3Dliteratum%253Aachs%26aulast%3DManning%26aufirst%3DG.%26aulast%3DWhyte%26aufirst%3DD.%26aulast%3DMartinez%26aufirst%3DR.%26aulast%3DHunter%26aufirst%3DT.%26aulast%3DSudarsanam%26aufirst%3DS.%26atitle%3DThe%2520Protein%2520Kinase%2520Complement%2520of%2520the%2520Human%2520Genome%26jtitle%3DScience%26date%3D2002%26volume%3D298%26spage%3D1912%26epage%3D1934%26doi%3D10.1126%2Fscience.1075762" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berman, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westbrook, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilliland, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhat, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissig, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shindyalov, I. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourne, P. E.</span></span> <span> </span><span class="NLM_article-title">The Protein Data Bank</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1093/nar/28.1.235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1093%2Fnar%2F28.1.235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10592235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=235-242&author=H.+M.+Bermanauthor=J.+Westbrookauthor=Z.+Fengauthor=G.+Gillilandauthor=T.+N.+Bhatauthor=H.+Weissigauthor=I.+N.+Shindyalovauthor=P.+E.+Bourne&title=The+Protein+Data+Bank&doi=10.1093%2Fnar%2F28.1.235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank</span></div><div class="casAuthors">Berman, Helen M.; Westbrook, John; Feng, Zukang; Gilliland, Gary; Bhat, T. N.; Weissig, Helge; Shindyalov, Ilya N.; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Protein Data Bank (PDB; http://www.rcsb.org/pdb/)is the single worldwide archive of structural data of biol. macromols.  This paper describes the goals of the PDB, the systems in place for data deposition and access, how to obtain further information, and near-term plans for the future development of the resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpStGCxT8QzirVg90H21EOLACvtfcHk0ljo3fc-5Jq9Aw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D&md5=227fb393f754be2be375ab727bfd05dc</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F28.1.235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F28.1.235%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DWestbrook%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGilliland%26aufirst%3DG.%26aulast%3DBhat%26aufirst%3DT.%2BN.%26aulast%3DWeissig%26aufirst%3DH.%26aulast%3DShindyalov%26aufirst%3DI.%2BN.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DThe%2520Protein%2520Data%2520Bank%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2000%26volume%3D28%26spage%3D235%26epage%3D242%26doi%3D10.1093%2Fnar%2F28.1.235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gavrin, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saiah, E.</span></span> <span> </span><span class="NLM_article-title">Approaches to Discover Non-ATP Site Kinase Inhibitors</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">41</span>– <span class="NLM_lpage">51</span>, <span class="refDoi"> DOI: 10.1039/C2MD20180A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1039%2FC2MD20180A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVymsr7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=41-51&author=L.+K.+Gavrinauthor=E.+Saiah&title=Approaches+to+Discover+Non-ATP+Site+Kinase+Inhibitors&doi=10.1039%2FC2MD20180A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Approaches to discover non-ATP site kinase inhibitors</span></div><div class="casAuthors">Gavrin, Lori Krim; Saiah, Eddine</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">41-51</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The catalytic domain of kinases shows a high degree of sequence homol., esp. for kinases that belong to the same family.  They share a common ATP binding site with a conserved activation loop and similar three-dimensional structure.  Consequently, a major challenge in kinase research exists in achieving selectivity among the >500 family members, since they all process the same substrate.  In addn. to requiring selectivity against other kinases, ATP site inhibitors must also bind tightly to overcome the high physiol. concn. of ATP in the cell.  Furthermore, the development of novel ATP site inhibitors is becoming increasingly challenging, as many ATP competitive scaffolds have previously been disclosed.  In order to develop compds. with better selectivity among kinases, inhibitors that bind outside the ATP site show great promise and are currently being explored by many groups.  This review will highlight the most commonly used methods to discover small mol. Type III and IV kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7xJcf4-RJ9rVg90H21EOLACvtfcHk0lh68w9CIwodEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVymsr7N&md5=97acf2247e5516c1649c1d964c867656</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1039%2FC2MD20180A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC2MD20180A%26sid%3Dliteratum%253Aachs%26aulast%3DGavrin%26aufirst%3DL.%2BK.%26aulast%3DSaiah%26aufirst%3DE.%26atitle%3DApproaches%2520to%2520Discover%2520Non-ATP%2520Site%2520Kinase%2520Inhibitors%26jtitle%3DMedChemComm%26date%3D2013%26volume%3D4%26spage%3D41%26epage%3D51%26doi%3D10.1039%2FC2MD20180A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abdeldayem, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raouf, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Constantinescu, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriggl, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunning, P. T.</span></span> <span> </span><span class="NLM_article-title">Advances in Covalent Kinase Inhibition</span>. <i>Chem. Soc. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">2617</span>– <span class="NLM_lpage">2687</span>, <span class="refDoi"> DOI: 10.1039/C9CS00720B</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1039%2FC9CS00720B" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=32227030" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BB3cXlvVygsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2020&pages=2617-2687&author=A.+Abdeldayemauthor=Y.+S.+Raoufauthor=S.+N.+Constantinescuauthor=R.+Morigglauthor=P.+T.+Gunning&title=Advances+in+Covalent+Kinase+Inhibition&doi=10.1039%2FC9CS00720B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in covalent kinase inhibitors</span></div><div class="casAuthors">Abdeldayem, Ayah; Raouf, Yasir S.; Constantinescu, Stefan N.; Moriggl, Richard; Gunning, Patrick T.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Society Reviews</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2617-2687</span>CODEN:
                <span class="NLM_cas:coden">CSRVBR</span>;
        ISSN:<span class="NLM_cas:issn">0306-0012</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Over the past decade, covalent kinase inhibitors (CKI) have seen a resurgence in drug discovery.  Covalency affords a unique set of advantages as well as challenges relative to their non-covalent counterpart.  After reversible protein target recognition and binding, covalent inhibitors irreversibly modify a proximal nucleophilic residue on the protein via reaction with an electrophile.  To date, the acrylamide group remains the predominantly employed electrophile in CKI development, with its incorporation in the majority of clin. candidates and FDA approved covalent therapies.  Nonetheless, in recent years considerable efforts have ensued to characterize alternative electrophiles that exhibit irreversible or reversibly covalent binding mechanisms towards cysteine thiols and other amino acids.  This review article provides a comprehensive overview of CKIs reported in the literature over a decade period, 2007-2018.  Emphasis is placed on the rationale behind warhead choice, optimization approach, and inhibitor design.  Current FDA approved CKIs are also highlighted, in addn. to a detailed anal. of the common trends and themes obsd. within the listed data set.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT7Unx-xJoarVg90H21EOLACvtfcHk0lh68w9CIwodEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXlvVygsr8%253D&md5=88c6e097cb0b3619a53ceb1e03d19971</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1039%2FC9CS00720B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC9CS00720B%26sid%3Dliteratum%253Aachs%26aulast%3DAbdeldayem%26aufirst%3DA.%26aulast%3DRaouf%26aufirst%3DY.%2BS.%26aulast%3DConstantinescu%26aufirst%3DS.%2BN.%26aulast%3DMoriggl%26aufirst%3DR.%26aulast%3DGunning%26aufirst%3DP.%2BT.%26atitle%3DAdvances%2520in%2520Covalent%2520Kinase%2520Inhibition%26jtitle%3DChem.%2520Soc.%2520Rev.%26date%3D2020%26volume%3D49%26spage%3D2617%26epage%3D2687%26doi%3D10.1039%2FC9CS00720B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamba, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, I.</span></span> <span> </span><span class="NLM_article-title">New Directions in Targeting Protein Kinases: Focusing Upon True Allosteric and Bivalent Inhibitors</span>. <i>Curr. Pharm. Des.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">2936</span>– <span class="NLM_lpage">2945</span>, <span class="refDoi"> DOI: 10.2174/138161212800672813</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.2174%2F138161212800672813" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=22571662" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFaltrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=2936-2945&author=V.+Lambaauthor=I.+Ghosh&title=New+Directions+in+Targeting+Protein+Kinases%3A+Focusing+Upon+True+Allosteric+and+Bivalent+Inhibitors&doi=10.2174%2F138161212800672813"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">New directions in targeting protein kinases: focusing upon true allosteric and bivalent inhibitors</span></div><div class="casAuthors">Lamba, Vandana; Ghosh, Indraneel</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">2936-2945</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Over the past decade, therapeutics that target subsets of the 518 human protein kinases have played a vital role in the fight against cancer.  Protein kinases are typically targeted at the ATP-binding cleft by type I and II inhibitors; however, the high sequence and structural homol. shared by protein kinases, esp. at the ATP-binding site, inherently leads to polypharmacol.  In order to discover or design truly selective protein kinase inhibitors as both pharmacol. reagents and safer therapeutic leads, new efforts are needed to target kinases outside the ATP cleft.  Recent advances include the serendipitous discovery of type III inhibitors that bind a site proximal to the ATP pocket as well as the truly allosteric type IV inhibitors that target protein kinases distal to the substrate-binding pocket.  These new classes of inhibitors are often selective, but usually display moderate affinities.  Here, the authors discuss the different classes of inhibitors with an emphasis on bisubstrate and bivalent inhibitors (type V) that combine different inhibitor classes.  These inhibitors have the potential to couple the high affinity and potency of traditional active site-targeted small-mol. inhibitors with the selectivity of inhibitors that target the protein kinase surface outside ATP cleft.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkIIgeMxhCrrVg90H21EOLACvtfcHk0lh68w9CIwodEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFaltrk%253D&md5=36a44e36709507b5053bfdfe7d09e5e9</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.2174%2F138161212800672813&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138161212800672813%26sid%3Dliteratum%253Aachs%26aulast%3DLamba%26aufirst%3DV.%26aulast%3DGhosh%26aufirst%3DI.%26atitle%3DNew%2520Directions%2520in%2520Targeting%2520Protein%2520Kinases%253A%2520Focusing%2520Upon%2520True%2520Allosteric%2520and%2520Bivalent%2520Inhibitors%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2012%26volume%3D18%26spage%3D2936%26epage%3D2945%26doi%3D10.2174%2F138161212800672813" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gower, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maly, D. J.</span></span> <span> </span><span class="NLM_article-title">Bivalent Inhibitors of Protein Kinases</span>. <i>Crit. Rev. Biochem. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">102</span>– <span class="NLM_lpage">115</span>, <span class="refDoi"> DOI: 10.3109/10409238.2013.875513</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.3109%2F10409238.2013.875513" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=24564382" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC2cXks1Smtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2014&pages=102-115&author=C.+M.+Gowerauthor=M.+E.+Changauthor=D.+J.+Maly&title=Bivalent+Inhibitors+of+Protein+Kinases&doi=10.3109%2F10409238.2013.875513"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Bivalent inhibitors of protein kinases</span></div><div class="casAuthors">Gower, Carrie M.; Chang, Matthew E. K.; Maly, Dustin J.</div><div class="citationInfo"><span class="NLM_cas:title">Critical Reviews in Biochemistry and Molecular Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">102-115</span>CODEN:
                <span class="NLM_cas:coden">CRBBEJ</span>;
        ISSN:<span class="NLM_cas:issn">1040-9238</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Protein kinases are key players in a large no. of cellular signaling pathways.  Dysregulated kinase activity has been implicated in a no. of diseases, and members of this enzyme family are of therapeutic interest.  However, due to the fact that most inhibitors interact with the highly conserved ATP-binding sites of kinases, it is a significant challenge to develop pharmacol. agents that target only one of the greater than 500 kinases present in humans.  A potential soln. to this problem is the development of bisubstrate and bivalent kinase inhibitors, in which an active site-directed moiety is tethered to another ligand that targets a location outside of the ATP-binding cleft.  Because kinase signaling specificity is modulated by regions outside of the ATP-binding site, strategies that exploit these interactions have the potential to provide reagents with high target selectivity.  This review highlights examples of kinase interaction sites that can potentially be exploited by bisubstrate and bivalent inhibitors.  Furthermore, an overview of efforts to target these interactions with bisubstrate and bivalent inhibitors is provided.  Finally, several examples of the successful application of these reagents in a cellular setting are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbnNuzK1xqRLVg90H21EOLACvtfcHk0lgehNHBQ7AP3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXks1Smtr8%253D&md5=66249a71e01aaedb02ef8ed00d27096f</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.3109%2F10409238.2013.875513&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F10409238.2013.875513%26sid%3Dliteratum%253Aachs%26aulast%3DGower%26aufirst%3DC.%2BM.%26aulast%3DChang%26aufirst%3DM.%2BE.%26aulast%3DMaly%26aufirst%3DD.%2BJ.%26atitle%3DBivalent%2520Inhibitors%2520of%2520Protein%2520Kinases%26jtitle%3DCrit.%2520Rev.%2520Biochem.%2520Mol.%2520Biol.%26date%3D2014%26volume%3D49%26spage%3D102%26epage%3D115%26doi%3D10.3109%2F10409238.2013.875513" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laufer, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">The Cysteinome of Protein Kinases as a Target in Drug Development</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">4372</span>– <span class="NLM_lpage">4385</span>, <span class="refDoi"> DOI: 10.1002/anie.201707875</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1002%2Fanie.201707875" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFGgs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=4372-4385&author=A.+Chaikuadauthor=P.+Kochauthor=S.+A.+Lauferauthor=S.+Knapp&title=The+Cysteinome+of+Protein+Kinases+as+a+Target+in+Drug+Development&doi=10.1002%2Fanie.201707875"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The Cysteinome of Protein Kinases as a Target in Drug Development</span></div><div class="casAuthors">Chaikuad, Apirat; Koch, Pierre; Laufer, Stefan A.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4372-4385</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Drugs that function through covalent bond formation represent a considerable fraction of the authors' repository of effective medicines but safety concerns and the complexity of developing covalent inhibitors has rendered covalent targeting a less attractive strategy for rational drug design.  The recent approval of four covalent kinase inhibitors and the development of highly potent covalent kinase probes with exceptional selectivity has raised significant interest in industry and academic research and validated the concept of covalent kinase targeting for clin. applications.  The abundance of cysteines at diverse positions in and around the kinase active site suggests that a large fraction of kinases can be targeted by covalent inhibitors.  Herein, the authors review recent developments of this rapidly growing area in kinase drug development and highlight the unique opportunities and challenges of this strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphHGXsIr-dpbVg90H21EOLACvtfcHk0lgehNHBQ7AP3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFGgs7c%253D&md5=311ba5ab53dc4a5dc1dc92ddd4b6d4c2</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1002%2Fanie.201707875&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201707875%26sid%3Dliteratum%253Aachs%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DKoch%26aufirst%3DP.%26aulast%3DLaufer%26aufirst%3DS.%2BA.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DThe%2520Cysteinome%2520of%2520Protein%2520Kinases%2520as%2520a%2520Target%2520in%2520Drug%2520Development%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26spage%3D4372%26epage%3D4385%26doi%3D10.1002%2Fanie.201707875" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gehringer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laufer, S. A.</span></span> <span> </span><span class="NLM_article-title">Emerging and Re-emerging Warheads for Targeted Covalent Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">5673</span>– <span class="NLM_lpage">5724</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01153</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01153" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFKiur%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=5673-5724&author=M.+Gehringerauthor=S.+A.+Laufer&title=Emerging+and+Re-emerging+Warheads+for+Targeted+Covalent+Inhibitors&doi=10.1021%2Facs.jmedchem.8b01153"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging and Re-Emerging Warheads for Targeted Covalent Inhibitors: Applications in Medicinal Chemistry and Chemical Biology</span></div><div class="casAuthors">Gehringer, Matthias; Laufer, Stefan A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5673-5724</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Targeted covalent inhibitors (TCIs) are designed to bind poorly conserved amino acids by means of reactive groups, the so-called warheads.  Currently, targeting noncatalytic cysteine residues with acrylamides and other α,β-unsatd. carbonyl compds. is the predominant strategy in TCI development.  The recent ascent of covalent drugs has stimulated considerable efforts to characterize alternative warheads for the covalent-reversible and irreversible engagement of noncatalytic cysteine residues as well as other amino acids.  This Perspective article provides an overview of warheads-beyond α,β-unsatd. amides-recently used in the design of targeted covalent ligands.  Promising reactive groups that have not yet demonstrated their utility in TCI development are also highlighted.  Special emphasis is placed on the discussion of reactivity and of case studies illustrating applications in medicinal chem. and chem. biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDhc4PWV293LVg90H21EOLACvtfcHk0lgehNHBQ7AP3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFKiur%252FL&md5=4623733558ff99aadebfc13950b816ad</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01153&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01153%26sid%3Dliteratum%253Aachs%26aulast%3DGehringer%26aufirst%3DM.%26aulast%3DLaufer%26aufirst%3DS.%2BA.%26atitle%3DEmerging%2520and%2520Re-emerging%2520Warheads%2520for%2520Targeted%2520Covalent%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D5673%26epage%3D5724%26doi%3D10.1021%2Facs.jmedchem.8b01153" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weisner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gontla, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Westhuizen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeck, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ketzer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janning, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richters, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muhlenberg, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taher, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jendrossek, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pelly, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Otterlo, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauh, D.</span></span> <span> </span><span class="NLM_article-title">Covalent-allosteric Kinase Inhibitors</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">10313</span>– <span class="NLM_lpage">10316</span>, <span class="refDoi"> DOI: 10.1002/anie.201502142</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1002%2Fanie.201502142" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVKms7fE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=10313-10316&author=J.+Weisnerauthor=R.+Gontlaauthor=L.+van+der%0AWesthuizenauthor=S.+Oeckauthor=J.+Ketzerauthor=P.+Janningauthor=A.+Richtersauthor=T.+Muhlenbergauthor=Z.+Fangauthor=A.+Taherauthor=V.+Jendrossekauthor=S.+C.+Pellyauthor=S.+Bauerauthor=W.+A.+van+Otterloauthor=D.+Rauh&title=Covalent-allosteric+Kinase+Inhibitors&doi=10.1002%2Fanie.201502142"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent-Allosteric Kinase Inhibitors</span></div><div class="casAuthors">Weisner, Joern; Gontla, Rajesh; van der Westhuizen, Leandi; Oeck, Sebastian; Ketzer, Julia; Janning, Petra; Richters, Andre; Muehlenberg, Thomas; Fang, Zhizhou; Taher, Abu; Jendrossek, Verena; Pelly, Stephen C.; Bauer, Sebastian; van Otterlo, Willem A. L.; Rauh, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">10313-10316</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Targeting and stabilizing distinct kinase conformations is an instrumental strategy for dissecting conformation-dependent signaling of protein kinases.  Herein the structure-based design, synthesis, and evaluation of pleckstrin homol. (PH) domain-dependent covalent-allosteric inhibitors (CAIs) of the kinase Akt is reported.  These inhibitors bind covalently to a distinct cysteine of the kinase and thereby stabilize the inactive kinase conformation.  These modulators exhibit high potency and selectivity, and represent an innovative approach for chem. biol. and medicinal chem. research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodImeERqKXoLVg90H21EOLACvtfcHk0lhP-gJr7d3gkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVKms7fE&md5=9502910ef46da0e7e7ae7d2156e0a292</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1002%2Fanie.201502142&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201502142%26sid%3Dliteratum%253Aachs%26aulast%3DWeisner%26aufirst%3DJ.%26aulast%3DGontla%26aufirst%3DR.%26aulast%3Dvan%2Bder%2BWesthuizen%26aufirst%3DL.%26aulast%3DOeck%26aufirst%3DS.%26aulast%3DKetzer%26aufirst%3DJ.%26aulast%3DJanning%26aufirst%3DP.%26aulast%3DRichters%26aufirst%3DA.%26aulast%3DMuhlenberg%26aufirst%3DT.%26aulast%3DFang%26aufirst%3DZ.%26aulast%3DTaher%26aufirst%3DA.%26aulast%3DJendrossek%26aufirst%3DV.%26aulast%3DPelly%26aufirst%3DS.%2BC.%26aulast%3DBauer%26aufirst%3DS.%26aulast%3Dvan%2BOtterlo%26aufirst%3DW.%2BA.%26aulast%3DRauh%26aufirst%3DD.%26atitle%3DCovalent-allosteric%2520Kinase%2520Inhibitors%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D54%26spage%3D10313%26epage%3D10316%26doi%3D10.1002%2Fanie.201502142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smaill, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, K.</span></span> <span> </span><span class="NLM_article-title">New Promise and Opportunities for Allosteric Kinase Inhibitors</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">13764</span>– <span class="NLM_lpage">13776</span>, <span class="refDoi"> DOI: 10.1002/anie.201914525</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1002%2Fanie.201914525" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BB3cXmt1Giurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2020&pages=13764-13776&author=X.+Luauthor=J.+B.+Smaillauthor=K.+Ding&title=New+Promise+and+Opportunities+for+Allosteric+Kinase+Inhibitors&doi=10.1002%2Fanie.201914525"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">New Promise and Opportunities for Allosteric Kinase Inhibitors</span></div><div class="casAuthors">Lu, Xiaoyun; Smaill, Jeff B.; Ding, Ke</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">33</span>),
    <span class="NLM_cas:pages">13764-13776</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Drugs that function through allosteric inhibition of kinase signaling represent a promising approach for the targeted discovery of therapeutics.  The majority of developed allosteric kinase inhibitors are characterized as type III and IV inhibitors that show good kinome selectivity but generally lack the subtype selectivity of same kinase family.  Recently allosteric inhibitors have been developed that bind outside the catalytic kinase domain with high selectivity for specific kinase subtypes.  Allosteric inhibitors that bind to the pseudokinase domain of pseudokinase or the extracellular domain of receptor tyrosine kinases are reviewed.  We also review recent developments in the field of allosteric kinase inhibitors including examples of proteolysis targeting chimeras, and highlight the unique binding modes for each type of inhibitors and address future opportunities in this area.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIeFGyjfBKrLVg90H21EOLACvtfcHk0lhP-gJr7d3gkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXmt1Giurw%253D&md5=27959fdb1a96202db7ca65b2a8e19bc7</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1002%2Fanie.201914525&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201914525%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DX.%26aulast%3DSmaill%26aufirst%3DJ.%2BB.%26aulast%3DDing%26aufirst%3DK.%26atitle%3DNew%2520Promise%2520and%2520Opportunities%2520for%2520Allosteric%2520Kinase%2520Inhibitors%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2020%26volume%3D59%26spage%3D13764%26epage%3D13776%26doi%3D10.1002%2Fanie.201914525" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furtmann, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Current Compound Coverage of the Kinome</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">30</span>– <span class="NLM_lpage">40</span>, <span class="refDoi"> DOI: 10.1021/jm5008159</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5008159" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFyqs77J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=30-40&author=Y.+Huauthor=N.+Furtmannauthor=J.+Bajorath&title=Current+Compound+Coverage+of+the+Kinome&doi=10.1021%2Fjm5008159"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Current compound coverage of the kinome</span></div><div class="casAuthors">Hu, Ye; Furtmann, Norbert; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">30-40</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Publicly available protein kinase inhibitors have been analyzed in detail.  Nearly 19,000 inhibitors have been identified with activity against 266 different kinases.  Thus, about half of the human kinome is currently covered with active small mols.  The distribution of inhibitors across the kinome is uneven.  Most available kinase inhibitors are likely to be type I inhibitors.  By contrast, type II inhibitors are rare but usually have high potency.  Kinase inhibitors generally display high scaffold diversity.  Activity cliffs with an at least 100-fold difference in potency are only found for inhibitors of 106 kinases, which is partly due to only small nos. of compds. available for many kinases, in addn. to scaffold diversity.  Moreover, kinase inhibitors are less promiscuous than often thought.  More than 70% of available inhibitors are only annotated with a single kinase activity, and only ∼1% of the inhibitors are active against ≥5 kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpT_AYGB-DpHbVg90H21EOLACvtfcHk0lhP-gJr7d3gkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFyqs77J&md5=83267a7f8500cdaa1a5e9285c1bb7099</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fjm5008159&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5008159%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DY.%26aulast%3DFurtmann%26aufirst%3DN.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DCurrent%2520Compound%2520Coverage%2520of%2520the%2520Kinome%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D30%26epage%3D40%26doi%3D10.1021%2Fjm5008159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miljković, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Computational Analysis of Kinase Inhibitors Identifies Promiscuity Cliffs across the Human Kinome</span>. <i>ACS Omega</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">17295</span>– <span class="NLM_lpage">17308</span>, <span class="refDoi"> DOI: 10.1021/acsomega.8b02998</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsomega.8b02998" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFSjsL7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2018&pages=17295-17308&author=F.+Miljkovi%C4%87author=J.+Bajorath&title=Computational+Analysis+of+Kinase+Inhibitors+Identifies+Promiscuity+Cliffs+across+the+Human+Kinome&doi=10.1021%2Facsomega.8b02998"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Computational Analysis of Kinase Inhibitors Identifies Promiscuity Cliffs across the Human Kinome</span></div><div class="casAuthors">Miljkovic, Filip; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">ACS Omega</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">17295-17308</span>CODEN:
                <span class="NLM_cas:coden">ACSODF</span>;
        ISSN:<span class="NLM_cas:issn">2470-1343</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Kinase inhibitors are high-priority drug candidates for a variety of therapeutic applications.  Accordingly, there has been rapid growth in the no. of kinase inhibitors and vols. of assocd. activity data.  A paradigm for the use of kinase inhibitors in oncol. is that these compds. have multitarget activity and elicit their therapeutic effects through polypharmacol.  Anal. of kinase inhibitors and activity data from medicinal chem. has so far only identified small subsets of highly promiscuous kinase inhibitors.  In this study, the authors have collected inhibitors of human kinases and their activity data from seven public repositories, curated, and combined these data, yielding more than 112,000 inhibitors with well-defined activity measurements from which qual. target annotations were derived.  Anal. of this unprecedentedly large data sets revealed that nearly 40% of human kinase inhibitors have multikinase activity but that only 4% are known to be active against five or more kinases.  However, structurally analogous inhibitors often displayed significant differences in the no. of kinase annotations, leading to the formation of nearly 16,000 "promiscuity cliffs".  Moreover, 2236 PCs (14.03%) were formed by kinase inhibitors at different stages of clin. development.  Overall, these cliffs suggested many target hypotheses for kinase inhibitors, taking data incompleteness into consideration, as well as hypotheses for structural modifications leading to kinase selectivity.  Furthermore, from network representations, pathways comprising sequences of promiscuity cliffs were extd. that revealed unexpected structure-promiscuity relationships.  To enable follow-up investigations, all promiscuity cliffs formed by human kinase inhibitors will be made freely available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFnDkPulJXJrVg90H21EOLACvtfcHk0lhP-gJr7d3gkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFSjsL7M&md5=684eace3a768628d1ec8c4054f06361e</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facsomega.8b02998&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsomega.8b02998%26sid%3Dliteratum%253Aachs%26aulast%3DMiljkovi%25C4%2587%26aufirst%3DF.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DComputational%2520Analysis%2520of%2520Kinase%2520Inhibitors%2520Identifies%2520Promiscuity%2520Cliffs%2520across%2520the%2520Human%2520Kinome%26jtitle%3DACS%2520Omega%26date%3D2018%26volume%3D3%26spage%3D17295%26epage%3D17308%26doi%3D10.1021%2Facsomega.8b02998" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davis, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrgard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocker, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span> <span> </span><span class="NLM_article-title">Comprehensive Analysis of Kinase Inhibitor Selectivity</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1046</span>– <span class="NLM_lpage">1051</span>, <span class="refDoi"> DOI: 10.1038/nbt.1990</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1038%2Fnbt.1990" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=22037378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=1046-1051&author=M.+I.+Davisauthor=J.+P.+Huntauthor=S.+Herrgardauthor=P.+Ciceriauthor=L.+M.+Wodickaauthor=G.+Pallaresauthor=M.+Hockerauthor=D.+K.+Treiberauthor=P.+P.+Zarrinkar&title=Comprehensive+Analysis+of+Kinase+Inhibitor+Selectivity&doi=10.1038%2Fnbt.1990"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Davis, Mindy I.; Hunt, Jeremy P.; Herrgard, Sanna; Ciceri, Pietro; Wodicka, Lisa M.; Pallares, Gabriel; Hocker, Michael; Treiber, Daniel K.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1046-1051</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We tested the interaction of 72 kinase inhibitors with 442 kinases covering >80% of the human catalytic protein kinome.  Our data show that, as a class, type II inhibitors are more selective than type I inhibitors, but that there are important exceptions to this trend.  The data further illustrate that selective inhibitors have been developed against the majority of kinases targeted by the compds. tested.  Anal. of the interaction patterns reveals a class of 'group-selective' inhibitors broadly active against a single subfamily of kinases, but selective outside that subfamily.  The data set suggests compds. to use as tools to study kinases for which no dedicated inhibitors exist.  It also provides a foundation for further exploring kinase inhibitor biol. and toxicity, as well as for studying the structural basis of the obsd. interaction patterns.  Our findings will help to realize the direct enabling potential of genomics for drug development and basic research about cellular signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEZnKVs6Xt5rVg90H21EOLACvtfcHk0lhmZVCMVqvlFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK&md5=67639f702cb48ffb2c60e9a250f0df5e</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1990&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1990%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DHocker%26aufirst%3DM.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DComprehensive%2520Analysis%2520of%2520Kinase%2520Inhibitor%2520Selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26spage%3D1046%26epage%3D1051%26doi%3D10.1038%2Fnbt.1990" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klaeger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinzlmeir, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilhelm, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polzer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vick, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koenig, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinecke, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruprecht, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petzoldt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zecha, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiter, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helm, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schoof, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canevari, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casale, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Depaolini, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feuchtinger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rueckert, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huenges, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garz, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gohlke, B. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zolg, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kayser, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vooder, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preissner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahne, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tõnisson, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kramer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Götze, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassermann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlegl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ehrlich, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aiche, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greif, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felder, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruland, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Médard, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeremias, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spiekermann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuster, B.</span></span> <span> </span><span class="NLM_article-title">The Target Landscape of Clinical Kinase Inhibitors</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>358</i></span>,  <span class="NLM_fpage">eaan4368</span>, <span class="refDoi"> DOI: 10.1126/science.aan4368</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1126%2Fscience.aan4368" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=29191878" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=358&publication_year=2017&pages=eaan4368&author=S.+Klaegerauthor=S.+Heinzlmeirauthor=M.+Wilhelmauthor=H.+Polzerauthor=B.+Vickauthor=P.+A.+Koenigauthor=M.+Reineckeauthor=B.+Ruprechtauthor=S.+Petzoldtauthor=C.+Mengauthor=J.+Zechaauthor=K.+Reiterauthor=H.+Qiaoauthor=D.+Helmauthor=H.+Kochauthor=M.+Schoofauthor=G.+Canevariauthor=E.+Casaleauthor=S.+R.+Depaoliniauthor=A.+Feuchtingerauthor=Z.+Wuauthor=T.+Schmidtauthor=L.+Rueckertauthor=W.+Beckerauthor=J.+Huengesauthor=A.+K.+Garzauthor=B.+O.+Gohlkeauthor=D.+P.+Zolgauthor=G.+Kayserauthor=T.+Vooderauthor=R.+Preissnerauthor=H.+Hahneauthor=N.+T%C3%B5nissonauthor=K.+Kramerauthor=K.+G%C3%B6tzeauthor=F.+Bassermannauthor=J.+Schleglauthor=H.+C.+Ehrlichauthor=S.+Aicheauthor=A.+Walchauthor=P.+A.+Greifauthor=S.+Schneiderauthor=E.+R.+Felderauthor=J.+Rulandauthor=G.+M%C3%A9dardauthor=I.+Jeremiasauthor=K.+Spiekermannauthor=B.+Kuster&title=The+Target+Landscape+of+Clinical+Kinase+Inhibitors&doi=10.1126%2Fscience.aan4368"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1126%2Fscience.aan4368&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aan4368%26sid%3Dliteratum%253Aachs%26aulast%3DKlaeger%26aufirst%3DS.%26aulast%3DHeinzlmeir%26aufirst%3DS.%26aulast%3DWilhelm%26aufirst%3DM.%26aulast%3DPolzer%26aufirst%3DH.%26aulast%3DVick%26aufirst%3DB.%26aulast%3DKoenig%26aufirst%3DP.%2BA.%26aulast%3DReinecke%26aufirst%3DM.%26aulast%3DRuprecht%26aufirst%3DB.%26aulast%3DPetzoldt%26aufirst%3DS.%26aulast%3DMeng%26aufirst%3DC.%26aulast%3DZecha%26aufirst%3DJ.%26aulast%3DReiter%26aufirst%3DK.%26aulast%3DQiao%26aufirst%3DH.%26aulast%3DHelm%26aufirst%3DD.%26aulast%3DKoch%26aufirst%3DH.%26aulast%3DSchoof%26aufirst%3DM.%26aulast%3DCanevari%26aufirst%3DG.%26aulast%3DCasale%26aufirst%3DE.%26aulast%3DDepaolini%26aufirst%3DS.%2BR.%26aulast%3DFeuchtinger%26aufirst%3DA.%26aulast%3DWu%26aufirst%3DZ.%26aulast%3DSchmidt%26aufirst%3DT.%26aulast%3DRueckert%26aufirst%3DL.%26aulast%3DBecker%26aufirst%3DW.%26aulast%3DHuenges%26aufirst%3DJ.%26aulast%3DGarz%26aufirst%3DA.%2BK.%26aulast%3DGohlke%26aufirst%3DB.%2BO.%26aulast%3DZolg%26aufirst%3DD.%2BP.%26aulast%3DKayser%26aufirst%3DG.%26aulast%3DVooder%26aufirst%3DT.%26aulast%3DPreissner%26aufirst%3DR.%26aulast%3DHahne%26aufirst%3DH.%26aulast%3DT%25C3%25B5nisson%26aufirst%3DN.%26aulast%3DKramer%26aufirst%3DK.%26aulast%3DG%25C3%25B6tze%26aufirst%3DK.%26aulast%3DBassermann%26aufirst%3DF.%26aulast%3DSchlegl%26aufirst%3DJ.%26aulast%3DEhrlich%26aufirst%3DH.%2BC.%26aulast%3DAiche%26aufirst%3DS.%26aulast%3DWalch%26aufirst%3DA.%26aulast%3DGreif%26aufirst%3DP.%2BA.%26aulast%3DSchneider%26aufirst%3DS.%26aulast%3DFelder%26aufirst%3DE.%2BR.%26aulast%3DRuland%26aufirst%3DJ.%26aulast%3DM%25C3%25A9dard%26aufirst%3DG.%26aulast%3DJeremias%26aufirst%3DI.%26aulast%3DSpiekermann%26aufirst%3DK.%26aulast%3DKuster%26aufirst%3DB.%26atitle%3DThe%2520Target%2520Landscape%2520of%2520Clinical%2520Kinase%2520Inhibitors%26jtitle%3DScience%26date%3D2017%26volume%3D358%26spage%3Deaan4368%26doi%3D10.1126%2Fscience.aan4368" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">González-Medina, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miljković, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haase, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drueckes, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trappe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laufer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Promiscuity Analysis of a Kinase Panel Screen with Designated p38 Alpha Inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>187</i></span>,  <span class="NLM_fpage">112004</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2019.112004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1016%2Fj.ejmech.2019.112004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=31881458" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVyrurfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=187&publication_year=2020&pages=112004&author=M.+Gonz%C3%A1lez-Medinaauthor=F.+Miljkovi%C4%87author=G.+S.+Haaseauthor=P.+Drueckesauthor=J.+Trappeauthor=S.+Lauferauthor=J.+Bajorath&title=Promiscuity+Analysis+of+a+Kinase+Panel+Screen+with+Designated+p38+Alpha+Inhibitors&doi=10.1016%2Fj.ejmech.2019.112004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Promiscuity analysis of a kinase panel screen with designated p38 alpha inhibitors</span></div><div class="casAuthors">Gonzalez-Medina, Mariana; Miljkovic, Filip; Haase, Gernot S.; Drueckes, Peter; Trappe, Joerg; Laufer, Stefan; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">187</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112004</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Protein phosphorylation by kinases is of crit. importance for the regulation of many cellular functions.  When kinases are deregulated numerous biol. processes are affected, which may cause a variety of diseases.  Therefore, kinase inhibition plays an important role for therapeutic intervention.  A no. of kinase inhibitors have been approved as drugs, initially in oncol. where promiscuous (multi-kinase) inhibitors were most efficacious.  Exploring kinase inhibitor selectivity and promiscuity for therapy is among the most challenging aspects of kinase drug discovery.  Herein, we thoroughly analyze a kinase profiling expt. in which 637 designated inhibitors of p38α MAP kinase (p38α) were tested against a panel of 60 kinases distributed across the human kinome.  In this expt., only 19% of the inhibitors were found to be promiscuous when the median p38α inhibition level was applied as an activity threshold.  Promiscuous inhibitors had a median value of two targets per compd., and many of these inhibitors were only active against the p38α and closely related JNK3 enzymes.  Promiscuity cliffs were identified and analyzed in a network representation revealing structural modifications that were implicated in triggering compd. promiscuity.  Taken together, the findings revealed a high degree of selectivity of designated p38α directed inhibitors although they target the ATP binding site that is largely conserved across the human kinome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6AAFV2qsAnbVg90H21EOLACvtfcHk0lhknoRfa_9b9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVyrurfP&md5=e615c05fc221b69dc13ec804d428dd03</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2019.112004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2019.112004%26sid%3Dliteratum%253Aachs%26aulast%3DGonz%25C3%25A1lez-Medina%26aufirst%3DM.%26aulast%3DMiljkovi%25C4%2587%26aufirst%3DF.%26aulast%3DHaase%26aufirst%3DG.%2BS.%26aulast%3DDrueckes%26aufirst%3DP.%26aulast%3DTrappe%26aufirst%3DJ.%26aulast%3DLaufer%26aufirst%3DS.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DPromiscuity%2520Analysis%2520of%2520a%2520Kinase%2520Panel%2520Screen%2520with%2520Designated%2520p38%2520Alpha%2520Inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D187%26spage%3D112004%26doi%3D10.1016%2Fj.ejmech.2019.112004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tacconi, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarsounas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">A Unique Inhibitor Binding Site in ERK1/2 is Associated with Slow Binding Kinetics</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">853</span>– <span class="NLM_lpage">860</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1629</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1038%2Fnchembio.1629" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=25195011" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFKms7vN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=853-860&author=A.+Chaikuadauthor=E.+M.+Tacconiauthor=J.+Zimmerauthor=Y.+Liangauthor=N.+S.+Grayauthor=M.+Tarsounasauthor=S.+Knapp&title=A+Unique+Inhibitor+Binding+Site+in+ERK1%2F2+is+Associated+with+Slow+Binding+Kinetics&doi=10.1038%2Fnchembio.1629"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">A unique inhibitor binding site in ERK1/2 is associated with slow binding kinetics</span></div><div class="casAuthors">Chaikuad, Apirat; M C Tacconi, Eliana; Zimmer, Jutta; Liang, Yanke; Gray, Nathanael S.; Tarsounas, Madalena; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">853-860</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Activation of the ERK pathway is a hallmark of cancer, and targeting of upstream signaling partners led to the development of approved drugs.  Recently, SCH772984 has been shown to be a selective and potent ERK1/2 inhibitor.  Here we report the structural mechanism for its remarkable selectivity.  In ERK1/2, SCH772984 induces a so-far-unknown binding pocket that accommodates the piperazine-phenyl-pyrimidine decoration.  This new binding pocket was created by an inactive conformation of the phosphate-binding loop and an outward tilt of helix αC.  In contrast, structure detn. of SCH772984 with the off-target haspin and JNK1 revealed two canonical but distinct type I binding modes.  Notably, the new binding mode with ERK1/2 was assocd. with slow binding kinetics in vitro as well as in cell-based assay systems.  The described binding mode of SCH772984 with ERK1/2 enables the design of a new type of specific kinase inhibitors with prolonged on-target activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdNjmmeHD_-LVg90H21EOLACvtfcHk0lhknoRfa_9b9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFKms7vN&md5=b54251bd7486f0df266d8423ae9c627d</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1629&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1629%26sid%3Dliteratum%253Aachs%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DTacconi%26aufirst%3DE.%2BM.%26aulast%3DZimmer%26aufirst%3DJ.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DTarsounas%26aufirst%3DM.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DA%2520Unique%2520Inhibitor%2520Binding%2520Site%2520in%2520ERK1%252F2%2520is%2520Associated%2520with%2520Slow%2520Binding%2520Kinetics%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D853%26epage%3D860%26doi%3D10.1038%2Fnchembio.1629" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Georgi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiele, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, B.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steffen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marin Zapata, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briem, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preusse, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasta, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brands, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernández-Montalván, A.</span></span> <span> </span><span class="NLM_article-title">Binding Kinetics Survey of the Drugged Kinome</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">15774</span>– <span class="NLM_lpage">15782</span>, <span class="refDoi"> DOI: 10.1021/jacs.8b08048</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.8b08048" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVWrtbzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2018&pages=15774-15782&author=V.+Georgiauthor=F.+Schieleauthor=B.-T.+Bergerauthor=A.+Steffenauthor=P.+A.+Marin+Zapataauthor=H.+Briemauthor=S.+Menzauthor=C.+Preusseauthor=J.+D.+Vastaauthor=M.+B.+Robersauthor=M.+Brandsauthor=S.+Knappauthor=A.+Fern%C3%A1ndez-Montalv%C3%A1n&title=Binding+Kinetics+Survey+of+the+Drugged+Kinome&doi=10.1021%2Fjacs.8b08048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Binding Kinetics Survey of the Drugged Kinome</span></div><div class="casAuthors">Georgi, Victoria; Schiele, Felix; Berger, Benedict-Tilman; Steffen, Andreas; Marin Zapata, Paula A.; Briem, Hans; Menz, Stephan; Preusse, Cornelia; Vasta, James D.; Robers, Matthew B.; Brands, Michael; Knapp, Stefan; Fernandez-Montalvan, Amaury</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">15774-15782</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Target residence time is emerging as an important optimization parameter in drug discovery, yet target and off-target engagement dynamics have not been clearly linked to the clin. performance of drugs.  Here the authors developed high-throughput binding kinetics assays to characterize the interactions of 270 protein kinase inhibitors with 40 clin. relevant targets.  Anal. of the results revealed that on-rates are better correlated with affinity than off-rates and that the fraction of slowly dissocg. drug-target complexes increases from early/preclin. to late stage and FDA-approved compds., suggesting distinct contributions by each parameter to clin. success.  Combining binding parameters with PK/ADME properties, the authors illustrate in silico and in cells how kinetic selectivity could be exploited as an optimization strategy.  Furthermore, using bio- and chemoinformatics the authors uncovered structural features influencing rate consts.  The results underscore the value of binding kinetics information in rational drug design and provide a resource for future studies on this subject.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnD0Psx3Ks9rVg90H21EOLACvtfcHk0lhknoRfa_9b9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVWrtbzO&md5=e2e54da7e0b00be3f674a3e4448333d5</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjacs.8b08048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.8b08048%26sid%3Dliteratum%253Aachs%26aulast%3DGeorgi%26aufirst%3DV.%26aulast%3DSchiele%26aufirst%3DF.%26aulast%3DBerger%26aufirst%3DB.-T.%26aulast%3DSteffen%26aufirst%3DA.%26aulast%3DMarin%2BZapata%26aufirst%3DP.%2BA.%26aulast%3DBriem%26aufirst%3DH.%26aulast%3DMenz%26aufirst%3DS.%26aulast%3DPreusse%26aufirst%3DC.%26aulast%3DVasta%26aufirst%3DJ.%2BD.%26aulast%3DRobers%26aufirst%3DM.%2BB.%26aulast%3DBrands%26aufirst%3DM.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DFern%25C3%25A1ndez-Montalv%25C3%25A1n%26aufirst%3DA.%26atitle%3DBinding%2520Kinetics%2520Survey%2520of%2520the%2520Drugged%2520Kinome%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D15774%26epage%3D15782%26doi%3D10.1021%2Fjacs.8b08048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xie, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saleh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalodimos, C. G.</span></span> <span> </span><span class="NLM_article-title">Conformational States Dynamically Populated by a Kinase Determine Its Function</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>370</i></span>,  <span class="NLM_fpage">eabc2754</span>, <span class="refDoi"> DOI: 10.1126/science.abc2754</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1126%2Fscience.abc2754" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=33004676" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=370&publication_year=2020&pages=eabc2754&author=T.+Xieauthor=T.+Salehauthor=P.+Rossiauthor=C.+G.+Kalodimos&title=Conformational+States+Dynamically+Populated+by+a+Kinase+Determine+Its+Function&doi=10.1126%2Fscience.abc2754"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1126%2Fscience.abc2754&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.abc2754%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DT.%26aulast%3DSaleh%26aufirst%3DT.%26aulast%3DRossi%26aufirst%3DP.%26aulast%3DKalodimos%26aufirst%3DC.%2BG.%26atitle%3DConformational%2520States%2520Dynamically%2520Populated%2520by%2520a%2520Kinase%2520Determine%2520Its%2520Function%26jtitle%3DScience%26date%3D2020%26volume%3D370%26spage%3Deabc2754%26doi%3D10.1126%2Fscience.abc2754" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brear, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ball, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stott, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Arcy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hyvönen, M.</span></span> <span> </span><span class="NLM_article-title">Proposed Allosteric Inhibitors Bind to the ATP Site of CK2α</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">12786</span>– <span class="NLM_lpage">12789</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c01173</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c01173" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitFygsLbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=12786-12789&author=P.+Brearauthor=D.+Ballauthor=K.+Stottauthor=S.+D%E2%80%99Arcyauthor=M.+Hyv%C3%B6nen&title=Proposed+Allosteric+Inhibitors+Bind+to+the+ATP+Site+of+CK2%CE%B1&doi=10.1021%2Facs.jmedchem.0c01173"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Proposed Allosteric Inhibitors Bind to the ATP Site of CK2α</span></div><div class="casAuthors">Brear, Paul; Ball, Darby; Stott, Katherine; D'Arcy, Sheena; Hyvonen, Marko</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">12786-12798</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">CK2α is a ubiquitous, well-studied kinase that is a target for small-mol. inhibition, for treatment of cancers.  While many different classes of ATP (ATP)-competitive inhibitors have been described for CK2α, they tend to suffer from significant off-target activity and new approaches are needed.  A series of inhibitors of CK2α has recently been described as allosteric, acting at a previously unidentified binding site.  Given the similarity of these inhibitors to known ATP-competitive inhibitors, we have investigated them further.  In our thorough structural and biophys. analyses, we have found no evidence that these inhibitors bind to the proposed allosteric site.  Rather, we report crystal structures, competitive isothermal titrn. calorimetry (ITC) and NMR, hydrogen-deuterium exchange (HDX) mass spectrometry, and chemoinformatic analyses that all point to these compds. binding in the ATP pocket.  Comparisons of our results and exptl. approach with the data presented in the original report suggest that the primary reason for the disparity is nonspecific inhibition by aggregation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpl2o_1mN1qerVg90H21EOLACvtfcHk0lin3XtIymyLxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitFygsLbF&md5=bf59960c798b3f53140b0d16d1ee20b3</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c01173&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c01173%26sid%3Dliteratum%253Aachs%26aulast%3DBrear%26aufirst%3DP.%26aulast%3DBall%26aufirst%3DD.%26aulast%3DStott%26aufirst%3DK.%26aulast%3DD%25E2%2580%2599Arcy%26aufirst%3DS.%26aulast%3DHyv%25C3%25B6nen%26aufirst%3DM.%26atitle%3DProposed%2520Allosteric%2520Inhibitors%2520Bind%2520to%2520the%2520ATP%2520Site%2520of%2520CK2%25CE%25B1%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D12786%26epage%3D12789%26doi%3D10.1021%2Facs.jmedchem.0c01173" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Munoz, L.</span></span> <span> </span><span class="NLM_article-title">Non-kinase Targets of Protein Kinase Inhibitors</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">424</span>– <span class="NLM_lpage">440</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.266</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1038%2Fnrd.2016.266" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=28280261" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkt1OnsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=424-440&author=L.+Munoz&title=Non-kinase+Targets+of+Protein+Kinase+Inhibitors&doi=10.1038%2Fnrd.2016.266"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Non-kinase targets of protein kinase inhibitors</span></div><div class="casAuthors">Munoz, Lenka</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">424-440</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Kinome-wide profiling platforms have comprehensively identified the relevant kinases that are targeted by numerous protein kinase inhibitors.  However, recent projects have begun to discover non-kinase targets of kinase inhibitors.  These non-kinase targets can contribute to the desired or undesired activities of inhibitors, or act as silent bystanders.  As a full awareness of a drug's mechanism of action is crucial for the interpretation of results and for successful preclin. and clin. drug development, these discoveries highlight the importance of understanding the pharmacol. of kinase inhibitors beyond the kinome.  In this Review, I discuss kinase inhibitors for which non-kinase targets have been identified and the application of emerging techniques to validate drug-target engagement in intact cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXNOHFmb6t1LVg90H21EOLACvtfcHk0lin3XtIymyLxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkt1OnsbY%253D&md5=1145594c60795de101c75e522f001242</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.266&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.266%26sid%3Dliteratum%253Aachs%26aulast%3DMunoz%26aufirst%3DL.%26atitle%3DNon-kinase%2520Targets%2520of%2520Protein%2520Kinase%2520Inhibitors%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D424%26epage%3D440%26doi%3D10.1038%2Fnrd.2016.266" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">van
Linden, O. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kooistra, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leurs, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Esch, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Graaf, C.</span></span> <span> </span><span class="NLM_article-title">KLIFS: A Knowledge-Based Structural Database to Navigate Kinase-Ligand Interaction Space</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">249</span>– <span class="NLM_lpage">277</span>, <span class="refDoi"> DOI: 10.1021/jm400378w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400378w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Ojur7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=249-277&author=O.+P.+van%0ALindenauthor=A.+J.+Kooistraauthor=R.+Leursauthor=I.+J.+de+Eschauthor=C.+de+Graaf&title=KLIFS%3A+A+Knowledge-Based+Structural+Database+to+Navigate+Kinase-Ligand+Interaction+Space&doi=10.1021%2Fjm400378w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">KLIFS: A Knowledge-Based Structural Database To Navigate Kinase-Ligand Interaction Space</span></div><div class="casAuthors">van Linden, Oscar P. J.; Kooistra, Albert J.; Leurs, Rob; de Esch, Iwan J. P.; de Graaf, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">249-277</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Protein kinases regulate the majority of signal transduction pathways in cells and have become important targets for the development of designer drugs.  We present a systematic anal. of kinase-ligand interactions in all regions of the catalytic cleft of all 1252 human kinase-ligand cocrystal structures present in the Protein Data Bank (PDB).  The kinase-ligand interaction fingerprints and structure database (KLIFS) contains a consistent alignment of 85 kinase ligand binding site residues that enables the identification of family specific interaction features and classification of ligands according to their binding modes.  We illustrate how systematic mining of kinase-ligand interaction space gives new insights into how conserved and selective kinase interaction hot spots can accommodate the large diversity of chem. scaffolds in kinase ligands.  These analyses lead to an improved understanding of the structural requirements of kinase binding that will be useful in ligand discovery and design studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4tMmT6S-vRrVg90H21EOLACvtfcHk0lin3XtIymyLxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1Ojur7P&md5=8519942e1703ba04f64797e07df4b712</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm400378w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400378w%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BLinden%26aufirst%3DO.%2BP.%26aulast%3DKooistra%26aufirst%3DA.%2BJ.%26aulast%3DLeurs%26aufirst%3DR.%26aulast%3Dde%2BEsch%26aufirst%3DI.%2BJ.%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26atitle%3DKLIFS%253A%2520A%2520Knowledge-Based%2520Structural%2520Database%2520to%2520Navigate%2520Kinase-Ligand%2520Interaction%2520Space%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D249%26epage%3D277%26doi%3D10.1021%2Fjm400378w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Z.</span>; <span class="NLM_string-name">Yan, Y.-H.</span>; <span class="NLM_string-name">Yang, S.</span>; <span class="NLM_string-name">Zhu, S.</span>; <span class="NLM_string-name">Yuan, Y.</span>; <span class="NLM_string-name">Qiu, Z.</span>; <span class="NLM_string-name">Jia, H.</span>; <span class="NLM_string-name">Wang, R.</span>; <span class="NLM_string-name">Li, G.-B.</span>; <span class="NLM_string-name">Li, H.</span></span> P<span> </span><span class="NLM_article-title">rofKin: A Comprehensive Web Server for Structure-Based Kinase Selectivity Profiling</span>, <span class="NLM_year">2019</span>. <a href="http://www.lilab-ecust.cn/profkin/" class="extLink">http://www.lilab-ecust.cn/profkin/</a> (accessed Aug 01, 2020).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=Z.+Shen&author=Y.-H.+Yan&author=S.+Yang&author=S.+Zhu&author=Y.+Yuan&author=Z.+Qiu&author=H.+Jia&author=R.+Wang&author=G.-B.+Li&author=H.+Li&title=rofKin%3A+A+Comprehensive+Web+Server+for+Structure-Based+Kinase+Selectivity+Profiling"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DZ.%26atitle%3DrofKin%253A%2520A%2520Comprehensive%2520Web%2520Server%2520for%2520Structure-Based%2520Kinase%2520Selectivity%2520Profiling%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">ASD: A Comprehensive Database of Allosteric Proteins and Modulators</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">D663</span>– <span class="NLM_lpage">D669</span>, <span class="refDoi"> DOI: 10.1093/nar/gkq1022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1093%2Fnar%2Fgkq1022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=21051350" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC3sXivF2ks74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2011&pages=D663-D669&author=Z.+Huangauthor=L.+Zhuauthor=Y.+Caoauthor=G.+Wuauthor=X.+Liuauthor=Y.+Chenauthor=Q.+Wangauthor=T.+Shiauthor=Y.+Zhaoauthor=Y.+Wangauthor=W.+Liauthor=Y.+Liauthor=H.+Chenauthor=G.+Chenauthor=J.+Zhang&title=ASD%3A+A+Comprehensive+Database+of+Allosteric+Proteins+and+Modulators&doi=10.1093%2Fnar%2Fgkq1022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">ASD: a comprehensive database of allosteric proteins and modulators</span></div><div class="casAuthors">Huang, Zhimin; Zhu, Liang; Cao, Yan; Wu, Geng; Liu, Xinyi; Chen, Yingyi; Wang, Qi; Shi, Ting; Zhao, Yaxue; Wang, Yuefei; Li, Weihua; Li, Yixue; Chen, Haifeng; Chen, Guoqiang; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">Suppl. 1</span>),
    <span class="NLM_cas:pages">D663-D669</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Allostery is the most direct, rapid and efficient way of regulating protein function, ranging from the control of metabolic mechanisms to signal-transduction pathways.  However, an enormous amt. of unsystematic allosteric information has deterred scientists who could benefit from this field.  Here, we present the AlloSteric Database (ASD), the first online database that provides a central resource for the display, search and anal. of structure, function and related annotation for allosteric mols.  Currently, ASD contains 336 allosteric proteins from 101 species and 8095 modulators in three categories (activators, inhibitors and regulators).  Proteins are annotated with a detailed description of allostery, biol. process and related diseases, and modulators with binding affinity, physicochem. properties and therapeutic area.  Integrating the information of allosteric proteins in ASD should allow for the identification of specific allosteric sites of a given subtype among proteins of the same family that can potentially serve as ideal targets for exptl. validation.  In addn., modulators curated in ASD can be used to investigate potent allosteric targets for the query compd., and also help chemists to implement structure modifications for novel allosteric drug design.  Therefore, ASD could be a platform and a starting point for biologists and medicinal chemists for furthering allosteric research.  ASD is freely available at http://mdl.shsmu.edu.cn/ASD/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpg8WPj2tR4JbVg90H21EOLACvtfcHk0lhxXul0kWX-7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXivF2ks74%253D&md5=28a63caabbc2942d98ca03ca1b4256ee</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkq1022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkq1022%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DZhu%26aufirst%3DL.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DG.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DShi%26aufirst%3DT.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DASD%253A%2520A%2520Comprehensive%2520Database%2520of%2520Allosteric%2520Proteins%2520and%2520Modulators%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2011%26volume%3D39%26spage%3DD663%26epage%3DD669%26doi%3D10.1093%2Fnar%2Fgkq1022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">UniProt
Consortium</span> <span> </span><span class="NLM_article-title">Reorganizing the
Protein Space at the Universal Protein Resource (UniProt)</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">D142</span>– <span class="NLM_lpage">D148</span>, <span class="refDoi"> DOI: 10.1093/nar/gkr981</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1093%2Fnar%2Fgkr981" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=D142-D148&author=UniProt%0AConsortium&title=Reorganizing+the%0AProtein+Space+at+the+Universal+Protein+Resource+%28UniProt%29&doi=10.1093%2Fnar%2Fgkr981"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr981&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr981%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DReorganizing%2520the%250AProtein%2520Space%2520at%2520the%2520Universal%2520Protein%2520Resource%2520%2528UniProt%2529%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3DD142%26epage%3DD148%26doi%3D10.1093%2Fnar%2Fgkr981" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span> <i>Molecular Operating Environment
(MOE)</i>, version 2014.09; <span class="NLM_publisher-name">Chemical Computing
Group ULC</span>: <span class="NLM_publisher-loc">Montreal,
QC, Canada</span>, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Molecular+Operating+Environment%0A%28MOE%29%2C+version+2014.09%3B+Chemical+Computing%0AGroup+ULC%3A+Montreal%2C%0AQC%2C+Canada%2C+2014."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DMolecular%2520Operating%2520Environment%250A%2528MOE%2529%26pub%3DChemical%2520Computing%250AGroup%2520ULC%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">PDBe-KB
consortium</span> <span> </span><span class="NLM_article-title">PDBe-KB: A Community-driven
Resource for Structural and Functional Annotations</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">D344</span>– <span class="NLM_lpage">D353</span>, <span class="refDoi"> DOI: 10.1093/nar/gkz853</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1093%2Fnar%2Fgkz853" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=31584092" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2020&pages=D344-D353&author=PDBe-KB%0Aconsortium&title=PDBe-KB%3A+A+Community-driven%0AResource+for+Structural+and+Functional+Annotations&doi=10.1093%2Fnar%2Fgkz853"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkz853&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkz853%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DPDBe-KB%253A%2520A%2520Community-driven%250AResource%2520for%2520Structural%2520and%2520Functional%2520Annotations%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2020%26volume%3D48%26spage%3DD344%26epage%3DD353%26doi%3D10.1093%2Fnar%2Fgkz853" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ohren, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavlovsky, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitehead, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuffa, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConnell, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spessard, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banotai, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delaney, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Omer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebolt-Leopold, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dudley, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, I. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flamme, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warmus, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tecle, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasemann, C. A.</span></span> <span> </span><span class="NLM_article-title">Structures of Human Map Kinase Kinase 1 (MEK1) and MEK2 Describe Novel Noncompetitive Kinase Inhibition</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1192</span>– <span class="NLM_lpage">1197</span>, <span class="refDoi"> DOI: 10.1038/nsmb859</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1038%2Fnsmb859" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=15543157" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVens7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2004&pages=1192-1197&author=J.+F.+Ohrenauthor=H.+Chenauthor=A.+Pavlovskyauthor=C.+Whiteheadauthor=E.+Zhangauthor=P.+Kuffaauthor=C.+Yanauthor=P.+McConnellauthor=C.+Spessardauthor=C.+Banotaiauthor=W.+T.+Muellerauthor=A.+Delaneyauthor=C.+Omerauthor=J.+Sebolt-Leopoldauthor=D.+T.+Dudleyauthor=I.+K.+Leungauthor=C.+Flammeauthor=J.+Warmusauthor=M.+Kaufmanauthor=S.+Barrettauthor=H.+Tecleauthor=C.+A.+Hasemann&title=Structures+of+Human+Map+Kinase+Kinase+1+%28MEK1%29+and+MEK2+Describe+Novel+Noncompetitive+Kinase+Inhibition&doi=10.1038%2Fnsmb859"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Structures of human MAP kinase kinase 1 (MEK1) and MEK2 describe novel noncompetitive kinase inhibition</span></div><div class="casAuthors">Ohren, Jeffrey F.; Chen, Huifen; Pavlovsky, Alexander; Whitehead, Christopher; Zhang, Erli; Kuffa, Peter; Yan, Chunhong; McConnell, Patrick; Spessard, Cindy; Banotai, Craig; Mueller, W. Thomas; Delaney, Amy; Omer, Charles; Sebolt-Leopold, Judith; Dudley, David T.; Leung, Iris K.; Flamme, Cathlin; Warmus, Joseph; Kaufman, Michael; Barrett, Stephen; Tecle, Haile; Hasemann, Charles A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1192-1197</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">MEK1 and MEK2 kinases are closely related, dual-specificity tyrosine/threonine protein kinases found in the Ras/Raf/MEK/ERK MAP kinase (MAPK) signaling pathway.  Approx. 30% of all human cancers have a constitutively activated MAPK pathway, and constitutive activation of MEK1 results in cellular transformation.  Here, the authors present the x-ray crystal structures of human MEK1 and MEK2 kinases, each detd. as a ternary complex with MgATP and an inhibitor to a resoln. of 2.4 and 3.2 Å, resp.  The structures revealed that MEK1 and MEK2 each had a unique inhibitor-binding pocket adjacent to the MgATP-binding site.  The presence of the potent inhibitors induced several conformational changes in the unphosphorylated MEK1 and MEK2 kinases that locked them into a closed but catalytically inactive species.  Thus, the structures reported here reveal a novel, noncompetitive mechanism for protein kinase inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIY9begVVfW7Vg90H21EOLACvtfcHk0ljxsYoJQV5yrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVens7jM&md5=79ebbc2e3a9cdd3fbecb8bb35ea9dba3</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnsmb859&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb859%26sid%3Dliteratum%253Aachs%26aulast%3DOhren%26aufirst%3DJ.%2BF.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DPavlovsky%26aufirst%3DA.%26aulast%3DWhitehead%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DE.%26aulast%3DKuffa%26aufirst%3DP.%26aulast%3DYan%26aufirst%3DC.%26aulast%3DMcConnell%26aufirst%3DP.%26aulast%3DSpessard%26aufirst%3DC.%26aulast%3DBanotai%26aufirst%3DC.%26aulast%3DMueller%26aufirst%3DW.%2BT.%26aulast%3DDelaney%26aufirst%3DA.%26aulast%3DOmer%26aufirst%3DC.%26aulast%3DSebolt-Leopold%26aufirst%3DJ.%26aulast%3DDudley%26aufirst%3DD.%2BT.%26aulast%3DLeung%26aufirst%3DI.%2BK.%26aulast%3DFlamme%26aufirst%3DC.%26aulast%3DWarmus%26aufirst%3DJ.%26aulast%3DKaufman%26aufirst%3DM.%26aulast%3DBarrett%26aufirst%3DS.%26aulast%3DTecle%26aufirst%3DH.%26aulast%3DHasemann%26aufirst%3DC.%2BA.%26atitle%3DStructures%2520of%2520Human%2520Map%2520Kinase%2520Kinase%25201%2520%2528MEK1%2529%2520and%2520MEK2%2520Describe%2520Novel%2520Noncompetitive%2520Kinase%2520Inhibition%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2004%26volume%3D11%26spage%3D1192%26epage%3D1197%26doi%3D10.1038%2Fnsmb859" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wright, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCormack, P. L.</span></span> <span> </span><span class="NLM_article-title">Trametinib: First Global Approval</span>. <i>Drugs</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">1245</span>– <span class="NLM_lpage">1254</span>, <span class="refDoi"> DOI: 10.1007/s40265-013-0096-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1007%2Fs40265-013-0096-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=23846731" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A280%3ADC%252BC3sfgtFSjtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=1245-1254&author=C.+J.+Wrightauthor=P.+L.+McCormack&title=Trametinib%3A+First+Global+Approval&doi=10.1007%2Fs40265-013-0096-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Trametinib: first global approval</span></div><div class="casAuthors">Wright Cameron J M; McCormack Paul L</div><div class="citationInfo"><span class="NLM_cas:title">Drugs</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1245-54</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Trametinib is an orally bioavailable mitogen-activated protein kinase (MAPK) kinase (MEK) inhibitor with antineoplastic activity.  The compound specifically binds to MEK1 and MEK2, resulting in inhibition of growth factor-mediated cell signalling and cellular proliferation in various cancers.  Originally developed by Japan Tobacco, GlaxoSmithKline has licensed exclusive worldwide rights to the compound and conducted development in a number of different cancer types.  Trametinib, as a monotherapy, has been approved in the US for the treatment of unresectable or metastatic malignant melanoma with BRAF V600E or V600K mutations, as detected by an FDA-approved test.  The compound, as a monotherapy, has also been submitted for regulatory review in the EU for BRAF mutation-positive malignant melanoma, and is in phase III development in Europe, Argentina, Canada and Oceania.  Phase II development is underway for pancreatic cancer, non-small cell lung cancer and relapsed or refractory leukaemias.  GlaxoSmithKline is also developing trametinib for use in combination with dabrafenib in BRAF V600 mutation-positive metastatic cutaneous melanoma; the combination is at the preregistration stage in the EU and a phase III clinical programme is underway worldwide.  Phase II development for this combination is also underway in colorectal cancer.  Several phase I trials have also been initiated to evaluate trametinib in combination with other drugs for the treatment of various solid tumours and haematological malignancies.  A paediatric oral solution formulation has been assessed against the oral tablet formulation in a phase I trial.  This article summarizes the milestones in the development of trametinib leading to this first approval for unresectable or metastatic BRAF mutation-positive malignant melanoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRw3yHtWXvrR2BAiYWy6n3SfW6udTcc2eZBDZ1zaNna27ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sfgtFSjtg%253D%253D&md5=133d953030ec0c3f62fbba730fabbfa9</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1007%2Fs40265-013-0096-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40265-013-0096-1%26sid%3Dliteratum%253Aachs%26aulast%3DWright%26aufirst%3DC.%2BJ.%26aulast%3DMcCormack%26aufirst%3DP.%2BL.%26atitle%3DTrametinib%253A%2520First%2520Global%2520Approval%26jtitle%3DDrugs%26date%3D2013%26volume%3D73%26spage%3D1245%26epage%3D1254%26doi%3D10.1007%2Fs40265-013-0096-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Biondi, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casamayor, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deak, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Currie, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alessi, D. R.</span></span> <span> </span><span class="NLM_article-title">Identification of a Pocket in the PDK1 Kinase Domain That Interacts with PIF and the C-Terminal Residues of PKA</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">979</span>– <span class="NLM_lpage">988</span>, <span class="refDoi"> DOI: 10.1093/emboj/19.5.979</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1093%2Femboj%2F19.5.979" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10698939" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BD3cXitVyrsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2000&pages=979-988&author=R.+M.+Biondiauthor=P.+C.+Cheungauthor=A.+Casamayorauthor=M.+Deakauthor=R.+A.+Currieauthor=D.+R.+Alessi&title=Identification+of+a+Pocket+in+the+PDK1+Kinase+Domain+That+Interacts+with+PIF+and+the+C-Terminal+Residues+of+PKA&doi=10.1093%2Femboj%2F19.5.979"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a pocket in the PDK1 kinase domain that interacts with PIF and the C-terminal residues of PKA</span></div><div class="casAuthors">Biondi, Ricardo M.; Cheung, Peter C. F.; Casamayor, Antonio; Deak, Maria; Currie, Richard A.; Alessi, Dario R.</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">979-988</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The 3-phosphoinositide-dependent protein kinase-1 (PDK1) phosphorylates and activates a no. of protein kinases of the AGC subfamily.  The kinase domain of PDK1 interacts with a region of protein kinase C-related kinase-2 (PRK2), termed the PDK1-interacting fragment (PIF), through a hydrophobic motif.  Here we identify a hydrophobic pocket in the small lobe of the PDK1 kinase domain, sep. from the ATP- and substrate-binding sites, that interacts with PIF.  Mutation of residues predicted to form part of this hydrophobic pocket either abolished or significantly diminished the affinity of PDK1 for PIF.  PIF increased the rate at which PDK1 phosphorylated a synthetic dodecapeptide (T308tide), corresponding to the sequences surrounding the PDK1 phosphorylation site of PKB.  This peptide is a poor substrate for PDK1, but a peptide comprising T308tide fused to the PDK1-binding motif of PIF was a vastly superior substrate for PDK1.  Our results suggest that the PIF-binding pocket on the kinase domain of PDK1 acts as a "docking site", enabling it to interact with and enhance the phosphorylation of its substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9OG4ALKuz-LVg90H21EOLACvtfcHk0li7QRXAK501EQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXitVyrsrk%253D&md5=dfcdfbf35d7d4039725ee1b371837ed8</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1093%2Femboj%2F19.5.979&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Femboj%252F19.5.979%26sid%3Dliteratum%253Aachs%26aulast%3DBiondi%26aufirst%3DR.%2BM.%26aulast%3DCheung%26aufirst%3DP.%2BC.%26aulast%3DCasamayor%26aufirst%3DA.%26aulast%3DDeak%26aufirst%3DM.%26aulast%3DCurrie%26aufirst%3DR.%2BA.%26aulast%3DAlessi%26aufirst%3DD.%2BR.%26atitle%3DIdentification%2520of%2520a%2520Pocket%2520in%2520the%2520PDK1%2520Kinase%2520Domain%2520That%2520Interacts%2520with%2520PIF%2520and%2520the%2520C-Terminal%2520Residues%2520of%2520PKA%26jtitle%3DEMBO%2520J.%26date%3D2000%26volume%3D19%26spage%3D979%26epage%3D988%26doi%3D10.1093%2Femboj%2F19.5.979" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Janecek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emery, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huggins, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stockwell, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stokes, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almeida, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardwick, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narvaez, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hyvonen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spring, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKenzie, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkitaraman, A. R.</span></span> <span> </span><span class="NLM_article-title">Allosteric Modulation of AURKA Kinase Activity by a Small-Molecule Inhibitor of Its Protein-Protein Interaction with Tpx2</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">28528</span>, <span class="refDoi"> DOI: 10.1038/srep28528</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1038%2Fsrep28528" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=27339427" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVKiurjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=28528&author=M.+Janecekauthor=M.+Rossmannauthor=P.+Sharmaauthor=A.+Emeryauthor=D.+J.+Hugginsauthor=S.+R.+Stockwellauthor=J.+E.+Stokesauthor=Y.+S.+Tanauthor=E.+G.+Almeidaauthor=B.+Hardwickauthor=A.+J.+Narvaezauthor=M.+Hyvonenauthor=D.+R.+Springauthor=G.+J.+McKenzieauthor=A.+R.+Venkitaraman&title=Allosteric+Modulation+of+AURKA+Kinase+Activity+by+a+Small-Molecule+Inhibitor+of+Its+Protein-Protein+Interaction+with+Tpx2&doi=10.1038%2Fsrep28528"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric modulation of AURKA kinase activity by a small-molecule inhibitor of its protein-protein interaction with TPX2</span></div><div class="casAuthors">Janecek, Matej; Rossmann, Maxim; Sharma, Pooja; Emery, Amy; Huggins, David J.; Stockwell, Simon R.; Stokes, Jamie E.; Tan, Yaw S.; Almeida, Estrella Guarino; Hardwick, Bryn; Narvaez, Ana J.; Hyvonen, Marko; Spring, David R.; McKenzie, Grahame J.; Venkitaraman, Ashok R.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">28528</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The essential mitotic kinase Aurora A (AURKA) is controlled during cell cycle progression via two distinct mechanisms.  Following activation loop autophosphorylation early in mitosis when it localizes to centrosomes, AURKA is allosterically activated on the mitotic spindle via binding to the microtubule-assocd. protein, TPX2.  Here, we report the discovery of AurkinA, a novel chem. inhibitor of the AURKA-TPX2 interaction, which acts via an unexpected structural mechanism to inhibit AURKA activity and mitotic localization.  In crystal structures, AurkinA binds to a hydrophobic pocket (the 'Y pocket') that normally accommodates a conserved Tyr-Ser-Tyr motif from TPX2, blocking the AURKA-TPX2 interaction.  AurkinA binding to the Y- pocket induces structural changes in AURKA that inhibit catalytic activity in vitro and in cells, without affecting ATP binding to the active site, defining a novel mechanism of allosteric inhibition.  Consistent with this mechanism, cells exposed to AurkinA mislocalize AURKA from mitotic spindle microtubules.  Thus, our findings provide fresh insight into the catalytic mechanism of AURKA, and identify a key structural feature as the target for a new class of dual-mode AURKA inhibitors, with implications for the chem. biol. and selective therapeutic targeting of structurally related kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_RxbdNKlklbVg90H21EOLACvtfcHk0li7QRXAK501EQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVKiurjL&md5=39ed09c96dee028e980e391c033a624c</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fsrep28528&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep28528%26sid%3Dliteratum%253Aachs%26aulast%3DJanecek%26aufirst%3DM.%26aulast%3DRossmann%26aufirst%3DM.%26aulast%3DSharma%26aufirst%3DP.%26aulast%3DEmery%26aufirst%3DA.%26aulast%3DHuggins%26aufirst%3DD.%2BJ.%26aulast%3DStockwell%26aufirst%3DS.%2BR.%26aulast%3DStokes%26aufirst%3DJ.%2BE.%26aulast%3DTan%26aufirst%3DY.%2BS.%26aulast%3DAlmeida%26aufirst%3DE.%2BG.%26aulast%3DHardwick%26aufirst%3DB.%26aulast%3DNarvaez%26aufirst%3DA.%2BJ.%26aulast%3DHyvonen%26aufirst%3DM.%26aulast%3DSpring%26aufirst%3DD.%2BR.%26aulast%3DMcKenzie%26aufirst%3DG.%2BJ.%26aulast%3DVenkitaraman%26aufirst%3DA.%2BR.%26atitle%3DAllosteric%2520Modulation%2520of%2520AURKA%2520Kinase%2520Activity%2520by%2520a%2520Small-Molecule%2520Inhibitor%2520of%2520Its%2520Protein-Protein%2520Interaction%2520with%2520Tpx2%26jtitle%3DSci.%2520Rep.%26date%3D2016%26volume%3D6%26spage%3D28528%26doi%3D10.1038%2Fsrep28528" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hindie, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stroba, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Garcia, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Idrissova, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeuzem, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirschberg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaeffer, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jørgensen, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alzari, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biondi, R. M.</span></span> <span> </span><span class="NLM_article-title">Structure and Allosteric Effects of Low-Molecular-Weight Activators on the Protein kinase PDK1</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">758</span>– <span class="NLM_lpage">764</span>, <span class="refDoi"> DOI: 10.1038/nchembio.208</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1038%2Fnchembio.208" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=19718043" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVKhs73F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=758-764&author=V.+Hindieauthor=A.+Strobaauthor=H.+Zhangauthor=L.+A.+Lopez-Garciaauthor=L.+Idrissovaauthor=S.+Zeuzemauthor=D.+Hirschbergauthor=F.+Schaefferauthor=T.+J.+J%C3%B8rgensenauthor=M.+Engelauthor=P.+M.+Alzariauthor=R.+M.+Biondi&title=Structure+and+Allosteric+Effects+of+Low-Molecular-Weight+Activators+on+the+Protein+kinase+PDK1&doi=10.1038%2Fnchembio.208"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and allosteric effects of low-molecular-weight activators on the protein kinase PDK1</span></div><div class="casAuthors">Hindie, Valerie; Stroba, Adriana; Zhang, Hua; Lopez-Garcia, Laura A.; Idrissova, Leila; Zeuzem, Stefan; Hirschberg, Daniel; Schaeffer, Francis; Jorgensen, Thomas J. D.; Engel, Matthias; Alzari, Pedro M.; Biondi, Ricardo M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">758-764</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein phosphorylation transduces a large set of intracellular signals.  One mechanism by which phosphorylation mediates signal transduction is by prompting conformational changes in the target protein or interacting proteins.  Previous work described an allosteric site mediating phosphorylation-dependent activation of AGC kinases.  AGC kinase PDK1 is activated by the docking of a phosphorylated motif from substrates.  Here, the authors present the crystallog. of PDK1 bound to a rationally developed low-mol.-wt. activator and describe the conformational changes induced by small compds. in the crystal and in soln. using a fluorescence-based assay and deuterium exchange expts.  The results indicate that the binding of the compd. produces local changes at the target site, the PIF-binding pocket, and also allosteric changes at the ATP-binding site and the activation loop.  Altogether, the authors present mol. details of the allosteric changes induced by small compds. that trigger the activation of PDK1 through mimicry of phosphorylation-dependent conformational changes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBilNGbn-krbVg90H21EOLACvtfcHk0li7QRXAK501EQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVKhs73F&md5=c9afd03c1d2c954fbcf04709570fdb76</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.208&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.208%26sid%3Dliteratum%253Aachs%26aulast%3DHindie%26aufirst%3DV.%26aulast%3DStroba%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DLopez-Garcia%26aufirst%3DL.%2BA.%26aulast%3DIdrissova%26aufirst%3DL.%26aulast%3DZeuzem%26aufirst%3DS.%26aulast%3DHirschberg%26aufirst%3DD.%26aulast%3DSchaeffer%26aufirst%3DF.%26aulast%3DJ%25C3%25B8rgensen%26aufirst%3DT.%2BJ.%26aulast%3DEngel%26aufirst%3DM.%26aulast%3DAlzari%26aufirst%3DP.%2BM.%26aulast%3DBiondi%26aufirst%3DR.%2BM.%26atitle%3DStructure%2520and%2520Allosteric%2520Effects%2520of%2520Low-Molecular-Weight%2520Activators%2520on%2520the%2520Protein%2520kinase%2520PDK1%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D758%26epage%3D764%26doi%3D10.1038%2Fnchembio.208" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Diskin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engelberg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Livnah, O.</span></span> <span> </span><span class="NLM_article-title">A Novel Lipid Binding Site Formed by the MAP Kinase Insert in p38 Alpha</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>375</i></span>,  <span class="NLM_fpage">70</span>– <span class="NLM_lpage">79</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2007.09.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1016%2Fj.jmb.2007.09.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=17999933" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlylu7vI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=375&publication_year=2008&pages=70-79&author=R.+Diskinauthor=D.+Engelbergauthor=O.+Livnah&title=A+Novel+Lipid+Binding+Site+Formed+by+the+MAP+Kinase+Insert+in+p38+Alpha&doi=10.1016%2Fj.jmb.2007.09.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">A Novel Lipid Binding Site Formed by the MAP Kinase Insert in p38α</span></div><div class="casAuthors">Diskin, Ron; Engelberg, David; Livnah, Oded</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">375</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">70-79</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The p38 mitogen-activated protein (MAP) kinases function as signaling mols. essential for many cellular processes, particularly mediating stress response.  The activity of p38 MAP kinases is meticulously regulated to reach the desired cellular phenotype.  Several alternative activation and attenuation mechanisms have been characterized recently which include new phosphorylation sites.  Here we present the crystal structure of p38α MAP kinase in complex with n-octyl-β-glucopyranoside (β-OG) detergent.  The complex unveils a novel lipid-binding site formed by a local conformational change of the MAP kinase insert.  This binding is the first evidence of a possible role for the MAP kinase insert in p38.  The binding site can accommodate a large selection of lipidic mols.  In addn., we also show via biophys. methods that arachidonic acid (AA) and its derivs. bind p38α in vitro.  Based on our anal. we propose that the binding of lipids could fine-tune p38α catalytic activity towards a preferred phenotype.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6J5fd2Jqe77Vg90H21EOLACvtfcHk0li3payJrbysdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlylu7vI&md5=88b97152503182d41fa0ea7b53271229</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2007.09.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2007.09.002%26sid%3Dliteratum%253Aachs%26aulast%3DDiskin%26aufirst%3DR.%26aulast%3DEngelberg%26aufirst%3DD.%26aulast%3DLivnah%26aufirst%3DO.%26atitle%3DA%2520Novel%2520Lipid%2520Binding%2520Site%2520Formed%2520by%2520the%2520MAP%2520Kinase%2520Insert%2520in%2520p38%2520Alpha%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2008%26volume%3D375%26spage%3D70%26epage%3D79%26doi%3D10.1016%2Fj.jmb.2007.09.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tzarum, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eisenberg-Domovich, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gills, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dennis, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Livnah, O.</span></span> <span> </span><span class="NLM_article-title">Lipid Molecules Induce P38alpha Activation Via a Novel Molecular Switch</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>424</i></span>,  <span class="NLM_fpage">339</span>– <span class="NLM_lpage">353</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2012.10.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1016%2Fj.jmb.2012.10.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=23079240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFymu73I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=424&publication_year=2012&pages=339-353&author=N.+Tzarumauthor=Y.+Eisenberg-Domovichauthor=J.+J.+Gillsauthor=P.+A.+Dennisauthor=O.+Livnah&title=Lipid+Molecules+Induce+P38alpha+Activation+Via+a+Novel+Molecular+Switch&doi=10.1016%2Fj.jmb.2012.10.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Lipid Molecules Induce p38α Activation via a Novel Molecular Switch</span></div><div class="casAuthors">Tzarum, Netanel; Eisenberg-Domovich, Yael; Gills, Joell J.; Dennis, Phillip A.; Livnah, Oded</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">424</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">339-353</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">P38α mitogen-activated protein kinase (MAPK) is generally activated by dual phosphorylation but has also been shown to exhibit alternative activation modes.  One of these modes included a direct interaction with phosphatidylinositol ether lipid analogs (PIA), inducing p38α autoactivation and apoptosis.  Perifosine, an Akt inhibitor in phase II clin. trials, also showed p38α activation properties similar to those of PIAs.  The crystal structures of p38α in complex with PIA23, PIA24 and perifosine provide insights into this unique activation mode.  The activating mols. bind a unique hydrophobic binding site in the kinase C'-lobe formed in part by the MAPK insert region.  In addn., there are conformational changes in the short αEF/αF loop region that acts as an activation switch, inducing autophosphorylation.  Structural and biochem. characterization of the αEF/αF loop identified Trp197 as a key residue in the lipid binding and in p38α catalytic activity.  The lipid binding site also accommodates hydrophobic inhibitor mols., and thus can serve as a novel p38α-target for specific activation or inhibition, with novel therapeutic implications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQ5MaNgn-kFbVg90H21EOLACvtfcHk0li3payJrbysdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFymu73I&md5=4341b3620e35b8a54f68448817ca157b</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2012.10.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2012.10.007%26sid%3Dliteratum%253Aachs%26aulast%3DTzarum%26aufirst%3DN.%26aulast%3DEisenberg-Domovich%26aufirst%3DY.%26aulast%3DGills%26aufirst%3DJ.%2BJ.%26aulast%3DDennis%26aufirst%3DP.%2BA.%26aulast%3DLivnah%26aufirst%3DO.%26atitle%3DLipid%2520Molecules%2520Induce%2520P38alpha%2520Activation%2520Via%2520a%2520Novel%2520Molecular%2520Switch%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D424%26spage%3D339%26epage%3D353%26doi%3D10.1016%2Fj.jmb.2012.10.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nagar, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hantschel, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheffzek, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veach, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bornmann, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarkson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Superti-Furga, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuriyan, J.</span></span> <span> </span><span class="NLM_article-title">Structural Basis for the Autoinhibition of c-Abl Tyrosine Kinase</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">859</span>– <span class="NLM_lpage">871</span>, <span class="refDoi"> DOI: 10.1016/S0092-8674(03)00194-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1016%2FS0092-8674%2803%2900194-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=12654251" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BD3sXivVWjtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2003&pages=859-871&author=B.+Nagarauthor=O.+Hantschelauthor=M.+A.+Youngauthor=K.+Scheffzekauthor=D.+Veachauthor=W.+Bornmannauthor=B.+Clarksonauthor=G.+Superti-Furgaauthor=J.+Kuriyan&title=Structural+Basis+for+the+Autoinhibition+of+c-Abl+Tyrosine+Kinase&doi=10.1016%2FS0092-8674%2803%2900194-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for the autoinhibition of c-Abl tyrosine kinase</span></div><div class="casAuthors">Nagar, Bhushan; Hantschel, Oliver; Young, Matthew A.; Scheffzek, Klaus; Veach, Darren; Bornmann, William; Clarkson, Bayard; Superti-Furga, Giulio; Kuriyan, John</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">859-871</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">C-Abl is normally regulated by an autoinhibitory mechanism, the disruption of which leads to chronic myelogenous leukemia.  The details of this mechanism have been elusive because c-Abl lacks a phosphotyrosine residue that triggers the assembly of the autoinhibited form of the closely related Src kinases by internally engaging the SH2 domain.  Crystal structures of c-Abl show that the N-terminal myristoyl modification of c-Abl 1b binds to the kinase domain and induces conformational changes that allow the SH2 and SH3 domains to dock onto it.  Autoinhibited c-Abl forms an assembly that is strikingly similar to that of inactive Src kinases but with specific differences that explain the differential ability of the drug STI-571/Gleevec/imatinib (STI-571) to inhibit the catalytic activity of Abl, but not that of c-Src.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaks_bu7ZlC7Vg90H21EOLACvtfcHk0li3payJrbysdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXivVWjtr4%253D&md5=19245c92b1e36779e2024bc071aa723b</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2803%2900194-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252803%252900194-6%26sid%3Dliteratum%253Aachs%26aulast%3DNagar%26aufirst%3DB.%26aulast%3DHantschel%26aufirst%3DO.%26aulast%3DYoung%26aufirst%3DM.%2BA.%26aulast%3DScheffzek%26aufirst%3DK.%26aulast%3DVeach%26aufirst%3DD.%26aulast%3DBornmann%26aufirst%3DW.%26aulast%3DClarkson%26aufirst%3DB.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26aulast%3DKuriyan%26aufirst%3DJ.%26atitle%3DStructural%2520Basis%2520for%2520the%2520Autoinhibition%2520of%2520c-Abl%2520Tyrosine%2520Kinase%26jtitle%3DCell%26date%3D2003%26volume%3D112%26spage%3D859%26epage%3D871%26doi%3D10.1016%2FS0092-8674%2803%2900194-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campobasso, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biju, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, X.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cottom, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galbraith, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegfried, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zappacosta, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Washio, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertrand, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Head, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moores, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johanson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burton, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erickson-Miller, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simpson, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tummino, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliff, A.</span></span> <span> </span><span class="NLM_article-title">Discovery and Characterization of a Cell-Permeable, Small-Molecule c-Abl Kinase Activator that Binds to the Myristoyl Binding Site</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">177</span>– <span class="NLM_lpage">186</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2010.12.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1016%2Fj.chembiol.2010.12.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=21338916" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC3MXit1Kjtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=177-186&author=J.+Yangauthor=N.+Campobassoauthor=M.+P.+Bijuauthor=K.+Fisherauthor=X.-Q.+Panauthor=J.+Cottomauthor=S.+Galbraithauthor=T.+Hoauthor=H.+Zhangauthor=X.+Hongauthor=P.+Wardauthor=G.+Hofmannauthor=B.+Siegfriedauthor=F.+Zappacostaauthor=Y.+Washioauthor=P.+Caoauthor=J.+Quauthor=S.+Bertrandauthor=D.-Y.+Wangauthor=M.+S.+Headauthor=H.+Liauthor=S.+Mooresauthor=Z.+Laiauthor=K.+Johansonauthor=G.+Burtonauthor=C.+Erickson-Millerauthor=G.+Simpsonauthor=P.+Tumminoauthor=R.+A.+Copelandauthor=A.+Oliff&title=Discovery+and+Characterization+of+a+Cell-Permeable%2C+Small-Molecule+c-Abl+Kinase+Activator+that+Binds+to+the+Myristoyl+Binding+Site&doi=10.1016%2Fj.chembiol.2010.12.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Characterization of a Cell-Permeable, Small-Molecule c-Abl Kinase Activator that Binds to the Myristoyl Binding Site</span></div><div class="casAuthors">Yang, Jingsong; Campobasso, Nino; Biju, Mangatt P.; Fisher, Kelly; Pan, Xiao-Qing; Cottom, Josh; Galbraith, Sarah; Ho, Thau; Zhang, Hong; Hong, Xuan; Ward, Paris; Hofmann, Glenn; Siegfried, Brett; Zappacosta, Francesca; Washio, Yoshiaki; Cao, Ping; Qu, Junya; Bertrand, Sophie; Wang, Da-Yuan; Head, Martha S.; Li, Hu; Moores, Sheri; Lai, Zhihong; Johanson, Kyung; Burton, George; Erickson-Miller, Connie; Simpson, Graham; Tummino, Peter; Copeland, Robert A.; Oliff, Allen</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">177-186</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: c-Abl kinase activity is regulated by a unique mechanism involving the formation of an autoinhibited conformation in which the N-terminal myristoyl group binds intramolecularly to the myristoyl binding site on the kinase domain and induces the bending of the αI helix that creates a docking surface for the SH2 domain.  Here, the authors report a small-mol. c-Abl activator, DPH, that displays potent enzymic and cellular activity in stimulating c-Abl activation.  Structural analyses indicate that DPH binds to the myristoyl binding site and prevents the formation of the bent conformation of the αI helix through steric hindrance, a mode of action distinct from the previously identified allosteric c-Abl inhibitor, GNF-2, that also binds to the myristoyl binding site.  DPH represents the first cell-permeable, small-mol. tool compd. for c-Abl activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKyxGGBs719rVg90H21EOLACvtfcHk0livxQiTe2AOZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXit1Kjtrg%253D&md5=27986a6ebec4220a39045cae27570775</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2010.12.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2010.12.013%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DCampobasso%26aufirst%3DN.%26aulast%3DBiju%26aufirst%3DM.%2BP.%26aulast%3DFisher%26aufirst%3DK.%26aulast%3DPan%26aufirst%3DX.-Q.%26aulast%3DCottom%26aufirst%3DJ.%26aulast%3DGalbraith%26aufirst%3DS.%26aulast%3DHo%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DHong%26aufirst%3DX.%26aulast%3DWard%26aufirst%3DP.%26aulast%3DHofmann%26aufirst%3DG.%26aulast%3DSiegfried%26aufirst%3DB.%26aulast%3DZappacosta%26aufirst%3DF.%26aulast%3DWashio%26aufirst%3DY.%26aulast%3DCao%26aufirst%3DP.%26aulast%3DQu%26aufirst%3DJ.%26aulast%3DBertrand%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DD.-Y.%26aulast%3DHead%26aufirst%3DM.%2BS.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DMoores%26aufirst%3DS.%26aulast%3DLai%26aufirst%3DZ.%26aulast%3DJohanson%26aufirst%3DK.%26aulast%3DBurton%26aufirst%3DG.%26aulast%3DErickson-Miller%26aufirst%3DC.%26aulast%3DSimpson%26aufirst%3DG.%26aulast%3DTummino%26aufirst%3DP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DOliff%26aufirst%3DA.%26atitle%3DDiscovery%2520and%2520Characterization%2520of%2520a%2520Cell-Permeable%252C%2520Small-Molecule%2520c-Abl%2520Kinase%2520Activator%2520that%2520Binds%2520to%2520the%2520Myristoyl%2520Binding%2520Site%26jtitle%3DChem.%2520Biol.%26date%3D2011%26volume%3D18%26spage%3D177%26epage%3D186%26doi%3D10.1016%2Fj.chembiol.2010.12.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wylie, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schoepfer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowan-Jacob, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furet, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marzinzik, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pelle, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buonamici, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassan, A. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iyer, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berellini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodd, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thohan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bitter, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Branford, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petruzzelli, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanasse, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warmuth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keen, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span> <span> </span><span class="NLM_article-title">The Allosteric Inhibitor ABL001 Enables Dual Targeting of BCR–ABL1</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>543</i></span>,  <span class="NLM_fpage">733</span>– <span class="NLM_lpage">737</span>, <span class="refDoi"> DOI: 10.1038/nature21702</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1038%2Fnature21702" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=28329763" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC2sXltVWntbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=543&publication_year=2017&pages=733-737&author=A.+A.+Wylieauthor=J.+Schoepferauthor=W.+Jahnkeauthor=S.+W.+Cowan-Jacobauthor=A.+Looauthor=P.+Furetauthor=A.+L.+Marzinzikauthor=X.+Pelleauthor=J.+Donovanauthor=W.+Zhuauthor=S.+Buonamiciauthor=A.+Q.+Hassanauthor=F.+Lombardoauthor=V.+Iyerauthor=M.+Palmerauthor=G.+Berelliniauthor=S.+Doddauthor=S.+Thohanauthor=H.+Bitterauthor=S.+Branfordauthor=D.+M.+Rossauthor=T.+P.+Hughesauthor=L.+Petruzzelliauthor=K.+G.+Vanasseauthor=M.+Warmuthauthor=F.+Hofmannauthor=N.+J.+Keenauthor=W.+R.+Sellers&title=The+Allosteric+Inhibitor+ABL001+Enables+Dual+Targeting+of+BCR%E2%80%93ABL1&doi=10.1038%2Fnature21702"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">The allosteric inhibitor ABL001 enables dual targeting of BCR-ABL1</span></div><div class="casAuthors">Wylie, Andrew A.; Schoepfer, Joseph; Jahnke, Wolfgang; Cowan-Jacob, Sandra W.; Loo, Alice; Furet, Pascal; Marzinzik, Andreas L.; Pelle, Xavier; Donovan, Jerry; Zhu, Wenjing; Buonamici, Silvia; Hassan, A. Quamrul; Lombardo, Franco; Iyer, Varsha; Palmer, Michael; Berellini, Giuliano; Dodd, Stephanie; Thohan, Sanjeev; Bitter, Hans; Branford, Susan; Ross, David M.; Hughes, Timothy P.; Petruzzelli, Lilli; Vanasse, K. Gary; Warmuth, Markus; Hofmann, Francesco; Keen, Nicholas J.; Sellers, William R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">543</span>
        (<span class="NLM_cas:issue">7647</span>),
    <span class="NLM_cas:pages">733-737</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chronic myeloid leukemia (CML) is driven by the activity of the BCR-ABL1 fusion oncoprotein.  ABL1 kinase inhibitors have improved the clin. outcomes for patients with CML, with over 80% of patients treated with imatinib surviving for more than 10 years.  Second-generation ABL1 kinase inhibitors induce more potent mol. responses in both previously untreated and imatinib-resistant patients with CML.  Studies in patients with chronic-phase CML have shown that around 50% of patients who achieve and maintain undetectable BCR-ABL1 transcript levels for at least 2 years remain disease-free after the withdrawal of treatment.  Here we characterize ABL001 (asciminib), a potent and selective allosteric ABL1 inhibitor that is undergoing clin. development testing in patients with CML and Philadelphia chromosome-pos. (Ph+) acute lymphoblastic leukemia.  In contrast to catalytic-site ABL1 kinase inhibitors, ABL001 binds to the myristoyl pocket of ABL1 and induces the formation of an inactive kinase conformation.  ABL001 and second-generation catalytic inhibitors have similar cellular potencies but distinct patterns of resistance mutations, with genetic barcoding studies revealing pre-existing clonal populations with no shared resistance between ABL001 and the catalytic inhibitor nilotinib.  Consistent with this profile, acquired resistance was obsd. with single-agent therapy in mice; however, the combination of ABL001 and nilotinib led to complete disease control and eradicated CML xenograft tumors without recurrence after the cessation of treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreJ5YSS_wiS7Vg90H21EOLACvtfcHk0livxQiTe2AOZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXltVWntbc%253D&md5=20d00f222ab8d9387cb340a9531b8ba5</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fnature21702&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature21702%26sid%3Dliteratum%253Aachs%26aulast%3DWylie%26aufirst%3DA.%2BA.%26aulast%3DSchoepfer%26aufirst%3DJ.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DCowan-Jacob%26aufirst%3DS.%2BW.%26aulast%3DLoo%26aufirst%3DA.%26aulast%3DFuret%26aufirst%3DP.%26aulast%3DMarzinzik%26aufirst%3DA.%2BL.%26aulast%3DPelle%26aufirst%3DX.%26aulast%3DDonovan%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DW.%26aulast%3DBuonamici%26aufirst%3DS.%26aulast%3DHassan%26aufirst%3DA.%2BQ.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DIyer%26aufirst%3DV.%26aulast%3DPalmer%26aufirst%3DM.%26aulast%3DBerellini%26aufirst%3DG.%26aulast%3DDodd%26aufirst%3DS.%26aulast%3DThohan%26aufirst%3DS.%26aulast%3DBitter%26aufirst%3DH.%26aulast%3DBranford%26aufirst%3DS.%26aulast%3DRoss%26aufirst%3DD.%2BM.%26aulast%3DHughes%26aufirst%3DT.%2BP.%26aulast%3DPetruzzelli%26aufirst%3DL.%26aulast%3DVanasse%26aufirst%3DK.%2BG.%26aulast%3DWarmuth%26aufirst%3DM.%26aulast%3DHofmann%26aufirst%3DF.%26aulast%3DKeen%26aufirst%3DN.%2BJ.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26atitle%3DThe%2520Allosteric%2520Inhibitor%2520ABL001%2520Enables%2520Dual%2520Targeting%2520of%2520BCR%25E2%2580%2593ABL1%26jtitle%3DNature%26date%3D2017%26volume%3D543%26spage%3D733%26epage%3D737%26doi%3D10.1038%2Fnature21702" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanders, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmena, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bright, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haire, L. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Underwood, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heath, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giordanetto, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carling, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gamblin, S. J.</span></span> <span> </span><span class="NLM_article-title">Structural Basis of AMPK Regulation by Small Molecule Activators</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">3017</span>, <span class="refDoi"> DOI: 10.1038/ncomms4017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1038%2Fncomms4017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=24352254" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A280%3ADC%252BC2c3oslGmtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=3017&author=B.+Xiaoauthor=M.+J.+Sandersauthor=D.+Carmenaauthor=N.+J.+Brightauthor=L.+F.+Haireauthor=E.+Underwoodauthor=B.+R.+Patelauthor=R.+B.+Heathauthor=P.+A.+Walkerauthor=S.+Hallenauthor=F.+Giordanettoauthor=S.+R.+Martinauthor=D.+Carlingauthor=S.+J.+Gamblin&title=Structural+Basis+of+AMPK+Regulation+by+Small+Molecule+Activators&doi=10.1038%2Fncomms4017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of AMPK regulation by small molecule activators</span></div><div class="casAuthors">Xiao Bing; Sanders Matthew J; Heath Richard B; Carmena David; Bright Nicola J; Patel Bhakti R; Carling David; Haire Lesley F; Underwood Elizabeth; Walker Philip A; Martin Stephen R; Gamblin Steven J; Hallen Stefan; Giordanetto Fabrizio</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">3017</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">AMP-activated protein kinase (AMPK) plays a major role in regulating cellular energy balance by sensing and responding to increases in AMP/ADP concentration relative to ATP.  Binding of AMP causes allosteric activation of the enzyme and binding of either AMP or ADP promotes and maintains the phosphorylation of threonine 172 within the activation loop of the kinase.  AMPK has attracted widespread interest as a potential therapeutic target for metabolic diseases including type 2 diabetes and, more recently, cancer.  A number of direct AMPK activators have been reported as having beneficial effects in treating metabolic diseases, but there has been no structural basis for activator binding to AMPK.  Here we present the crystal structure of human AMPK in complex with a small molecule activator that binds at a site between the kinase domain and the carbohydrate-binding module, stabilising the interaction between these two components.  The nature of the activator-binding pocket suggests the involvement of an additional, as yet unidentified, metabolite in the physiological regulation of AMPK.  Importantly, the structure offers new opportunities for the design of small molecule activators of AMPK for treatment of metabolic disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTKRk4-m-rMp1w2ofyr07jXfW6udTcc2ea1Y7emnmHot7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2c3oslGmtQ%253D%253D&md5=b261b7498514e258ea6ec99a34d04794</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fncomms4017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms4017%26sid%3Dliteratum%253Aachs%26aulast%3DXiao%26aufirst%3DB.%26aulast%3DSanders%26aufirst%3DM.%2BJ.%26aulast%3DCarmena%26aufirst%3DD.%26aulast%3DBright%26aufirst%3DN.%2BJ.%26aulast%3DHaire%26aufirst%3DL.%2BF.%26aulast%3DUnderwood%26aufirst%3DE.%26aulast%3DPatel%26aufirst%3DB.%2BR.%26aulast%3DHeath%26aufirst%3DR.%2BB.%26aulast%3DWalker%26aufirst%3DP.%2BA.%26aulast%3DHallen%26aufirst%3DS.%26aulast%3DGiordanetto%26aufirst%3DF.%26aulast%3DMartin%26aufirst%3DS.%2BR.%26aulast%3DCarling%26aufirst%3DD.%26aulast%3DGamblin%26aufirst%3DS.%2BJ.%26atitle%3DStructural%2520Basis%2520of%2520AMPK%2520Regulation%2520by%2520Small%2520Molecule%2520Activators%26jtitle%3DNat.%2520Commun.%26date%3D2013%26volume%3D4%26spage%3D3017%26doi%3D10.1038%2Fncomms4017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kufareva, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bestgen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brear, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prudent, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laudet, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moucadel, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ettaoussi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sautel, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krimm, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filhol, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borgne, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lomberget, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cochet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abagyan, R.</span></span> <span> </span><span class="NLM_article-title">Discovery of Holoenzyme-Disrupting Chemicals as Substrate-Selective Ck2 Inhibitors</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">15893</span>, <span class="refDoi"> DOI: 10.1038/s41598-019-52141-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1038%2Fs41598-019-52141-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=31685885" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A280%3ADC%252BB3MjltVOnuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=15893&author=I.+Kufarevaauthor=B.+Bestgenauthor=P.+Brearauthor=R.+Prudentauthor=B.+Laudetauthor=V.+Moucadelauthor=M.+Ettaoussiauthor=C.+F.+Sautelauthor=I.+Krimmauthor=M.+Engelauthor=O.+Filholauthor=M.+L.+Borgneauthor=T.+Lombergetauthor=C.+Cochetauthor=R.+Abagyan&title=Discovery+of+Holoenzyme-Disrupting+Chemicals+as+Substrate-Selective+Ck2+Inhibitors&doi=10.1038%2Fs41598-019-52141-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of holoenzyme-disrupting chemicals as substrate-selective CK2 inhibitors</span></div><div class="casAuthors">Kufareva Irina; Abagyan Ruben; Bestgen Benoit; Ettaoussi Mohamed; Borgne Marc Le; Lomberget Thierry; Bestgen Benoit; Engel Matthias; Bestgen Benoit; Prudent Renaud; Laudet Beatrice; Moucadel Virginie; Sautel Celine F; Filhol Odile; Cochet Claude; Bestgen Benoit; Brear Paul; Prudent Renaud; Laudet Beatrice; Moucadel Virginie; Sautel Celine F; Krimm Isabelle</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">15893</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">CK2 is a constitutively active protein kinase overexpressed in numerous malignancies.  Interaction between CK2α and CK2β subunits is essential for substrate selectivity.  The CK2α/CK2β interface has been previously targeted by peptides to achieve functional effects; however, no small molecules modulators were identified due to pocket flexibility and open shape.  Here we generated numerous plausible conformations of the interface using the fumigation modeling protocol, and virtually screened a compound library to discover compound 1 that suppressed CK2α/CK2β interaction in vitro and inhibited CK2 in a substrate-selective manner.  Orthogonal SPR, crystallography, and NMR experiments demonstrated that 4 and 6, improved analogs of 1, bind to CK2α as predicted.  Both inhibitors alter CK2 activity in cells through inhibition of CK2 holoenzyme formation.  Treatment with 6 suppressed MDA-MB231 triple negative breast cancer cell growth and induced apoptosis.  Altogether, our findings exemplify an innovative computational-experimental approach and identify novel non-peptidic inhibitors of CK2 subunit interface disclosing substrate-selective functional effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSmE_81mdt7jNenUz0lpNqnfW6udTcc2eaLPhpUjT07KLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MjltVOnuw%253D%253D&md5=9cbb832ee1fc84a087151c72111f6808</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fs41598-019-52141-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-019-52141-5%26sid%3Dliteratum%253Aachs%26aulast%3DKufareva%26aufirst%3DI.%26aulast%3DBestgen%26aufirst%3DB.%26aulast%3DBrear%26aufirst%3DP.%26aulast%3DPrudent%26aufirst%3DR.%26aulast%3DLaudet%26aufirst%3DB.%26aulast%3DMoucadel%26aufirst%3DV.%26aulast%3DEttaoussi%26aufirst%3DM.%26aulast%3DSautel%26aufirst%3DC.%2BF.%26aulast%3DKrimm%26aufirst%3DI.%26aulast%3DEngel%26aufirst%3DM.%26aulast%3DFilhol%26aufirst%3DO.%26aulast%3DBorgne%26aufirst%3DM.%2BL.%26aulast%3DLomberget%26aufirst%3DT.%26aulast%3DCochet%26aufirst%3DC.%26aulast%3DAbagyan%26aufirst%3DR.%26atitle%3DDiscovery%2520of%2520Holoenzyme-Disrupting%2520Chemicals%2520as%2520Substrate-Selective%2520Ck2%2520Inhibitors%26jtitle%3DSci.%2520Rep.%26date%3D2019%26volume%3D9%26spage%3D15893%26doi%3D10.1038%2Fs41598-019-52141-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Converso, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartingh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garbaccio, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tasber, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rickert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraley, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kreatsoulas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stirdivant, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drakas, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamilton, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buser, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abrams, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lobell, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sepp-Lorenzino, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zugay-Murphy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sardana, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munshi, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jezequel-Sur, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuck, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartman, G. D.</span></span> <span> </span><span class="NLM_article-title">Development of Thioquinazolinones, Allosteric Chk1 Kinase Inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1240</span>– <span class="NLM_lpage">1244</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2008.12.076</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1016%2Fj.bmcl.2008.12.076" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=19155174" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BD1MXit1Wrurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=1240-1244&author=A.+Conversoauthor=T.+Hartinghauthor=R.+M.+Garbaccioauthor=E.+Tasberauthor=K.+Rickertauthor=M.+E.+Fraleyauthor=Y.+Yanauthor=C.+Kreatsoulasauthor=S.+Stirdivantauthor=B.+Drakasauthor=E.+S.+Walshauthor=K.+Hamiltonauthor=C.+A.+Buserauthor=X.+Maoauthor=M.+T.+Abramsauthor=S.+C.+Beckauthor=W.+Taoauthor=R.+Lobellauthor=L.+Sepp-Lorenzinoauthor=J.+Zugay-Murphyauthor=V.+Sardanaauthor=S.+K.+Munshiauthor=S.+M.+Jezequel-Surauthor=P.+D.+Zuckauthor=G.+D.+Hartman&title=Development+of+Thioquinazolinones%2C+Allosteric+Chk1+Kinase+Inhibitors&doi=10.1016%2Fj.bmcl.2008.12.076"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Development of thioquinazolinones, allosteric Chk1 kinase inhibitors</span></div><div class="casAuthors">Converso, Antonella; Hartingh, Timothy; Garbaccio, Robert M.; Tasber, Edward; Rickert, Keith; Fraley, Mark E.; Yan, Youwei; Kreatsoulas, Constantine; Stirdivant, Steve; Drakas, Bob; Walsh, Eileen S.; Hamilton, Kelly; Buser, Carolyn A.; Mao, Xianzhi; Abrams, Marc T.; Beck, Stephen C.; Tao, Weikang; Lobell, Rob; Sepp-Lorenzino, Laura; Zugay-Murphy, Joan; Sardana, Vinod; Munshi, Sanjeev K.; Jezequel-Sur, Sylvie Marie; Zuck, Paul D.; Hartman, George D.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1240-1244</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A high throughput screening campaign was designed to identify allosteric inhibitors of Chk1 kinase by testing compds. at high concn.  Activity was then obsd. at Km for ATP and at near-physiol. concns. of ATP.  This strategy led to the discovery of a non-ATP competitive thioquinazolinone series which was optimized for potency and stability.  An X-ray crystal structure for the complex of our best inhibitor bound to Chk1 was solved, indicating that it binds to an allosteric site ∼13 Å from the ATP binding site.  Preliminary data is presented for several of these compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9bIRNnirJ4bVg90H21EOLACvtfcHk0lik9MpEFHRDWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXit1Wrurc%253D&md5=e91440930e976b9aafcfbb1597679511</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2008.12.076&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2008.12.076%26sid%3Dliteratum%253Aachs%26aulast%3DConverso%26aufirst%3DA.%26aulast%3DHartingh%26aufirst%3DT.%26aulast%3DGarbaccio%26aufirst%3DR.%2BM.%26aulast%3DTasber%26aufirst%3DE.%26aulast%3DRickert%26aufirst%3DK.%26aulast%3DFraley%26aufirst%3DM.%2BE.%26aulast%3DYan%26aufirst%3DY.%26aulast%3DKreatsoulas%26aufirst%3DC.%26aulast%3DStirdivant%26aufirst%3DS.%26aulast%3DDrakas%26aufirst%3DB.%26aulast%3DWalsh%26aufirst%3DE.%2BS.%26aulast%3DHamilton%26aufirst%3DK.%26aulast%3DBuser%26aufirst%3DC.%2BA.%26aulast%3DMao%26aufirst%3DX.%26aulast%3DAbrams%26aufirst%3DM.%2BT.%26aulast%3DBeck%26aufirst%3DS.%2BC.%26aulast%3DTao%26aufirst%3DW.%26aulast%3DLobell%26aufirst%3DR.%26aulast%3DSepp-Lorenzino%26aufirst%3DL.%26aulast%3DZugay-Murphy%26aufirst%3DJ.%26aulast%3DSardana%26aufirst%3DV.%26aulast%3DMunshi%26aufirst%3DS.%2BK.%26aulast%3DJezequel-Sur%26aufirst%3DS.%2BM.%26aulast%3DZuck%26aufirst%3DP.%2BD.%26aulast%3DHartman%26aufirst%3DG.%2BD.%26atitle%3DDevelopment%2520of%2520Thioquinazolinones%252C%2520Allosteric%2520Chk1%2520Kinase%2520Inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D1240%26epage%3D1244%26doi%3D10.1016%2Fj.bmcl.2008.12.076" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vanderpool, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ping, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergqvist, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alton, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phonephaly, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rui, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grant, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quenzer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Margosiak, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Register, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ermolieff, J.</span></span> <span> </span><span class="NLM_article-title">Characterization of the Chk1 Allosteric Inhibitor Binding Site</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">9823</span>– <span class="NLM_lpage">9830</span>, <span class="refDoi"> DOI: 10.1021/bi900258v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi900258v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFOiu7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2009&pages=9823-9830&author=D.+Vanderpoolauthor=T.+O.+Johnsonauthor=C.+Pingauthor=S.+Bergqvistauthor=G.+Altonauthor=S.+Phonephalyauthor=E.+Ruiauthor=C.+Luoauthor=Y.+L.+Dengauthor=S.+Grantauthor=T.+Quenzerauthor=S.+Margosiakauthor=J.+Registerauthor=E.+Brownauthor=J.+Ermolieff&title=Characterization+of+the+Chk1+Allosteric+Inhibitor+Binding+Site&doi=10.1021%2Fbi900258v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of the CHK1 Allosteric Inhibitor Binding Site</span></div><div class="casAuthors">Vanderpool, Darin; Johnson, Ted O.; Ping, Chen; Bergqvist, Simon; Alton, Gordon; Phonephaly, Soneprasith; Rui, Eugene; Luo, Chun; Deng, Ya-Li; Grant, Stephan; Quenzer, Terri; Margosiak, Steve; Register, James; Brown, Ed; Ermolieff, Jacques</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">9823-9830</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Checkpoint kinase 1 (CHK1) is a key element in the DNA damage response pathway and plays a crucial role in the S-G2-phase checkpoint.  Inhibiting CHK1 is a therapeutic strategy involving abrogation of the G2/M mitotic checkpoint defense of tumor cells toward lethal damage induced by DNA-directed chemotherapeutic agents.  To date, most CHK1 inhibition approaches have involved targeting the ATP site of this kinase.  In this study, we provide crystallog. and kinetic characterization of two small mol. inhibitors that bind to an allosteric site in the proximity of the CHK1 substrate site.  Anal. of kinetic and biophys. data has led to the conclusion that these small mol. allosteric site inhibitors of CHK1 are reversible and are neither ATP- nor peptide substrate-competitive.  Ki values of 1.89 and 0.15 μM, resp., have been detd. for these compds. using a mixed inhibitor kinetic anal.  Cocrystal structures of the inhibitors bound to CHK1 reveal an allosteric site, unique to CHK1, located in the C-terminal domain and consisting of a shallow groove linked to a small hydrophobic pocket.  The pocket displays induced fit characteristics in the presence of the two inhibitors.  These findings establish the potential for the development of highly selective CHK1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMx2Syic38ULVg90H21EOLACvtfcHk0ljw8HgcUgll7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFOiu7vO&md5=463e6df560e0cc8b3e8fed96be567284</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fbi900258v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi900258v%26sid%3Dliteratum%253Aachs%26aulast%3DVanderpool%26aufirst%3DD.%26aulast%3DJohnson%26aufirst%3DT.%2BO.%26aulast%3DPing%26aufirst%3DC.%26aulast%3DBergqvist%26aufirst%3DS.%26aulast%3DAlton%26aufirst%3DG.%26aulast%3DPhonephaly%26aufirst%3DS.%26aulast%3DRui%26aufirst%3DE.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DDeng%26aufirst%3DY.%2BL.%26aulast%3DGrant%26aufirst%3DS.%26aulast%3DQuenzer%26aufirst%3DT.%26aulast%3DMargosiak%26aufirst%3DS.%26aulast%3DRegister%26aufirst%3DJ.%26aulast%3DBrown%26aufirst%3DE.%26aulast%3DErmolieff%26aufirst%3DJ.%26atitle%3DCharacterization%2520of%2520the%2520Chk1%2520Allosteric%2520Inhibitor%2520Binding%2520Site%26jtitle%3DBiochemistry%26date%3D2009%26volume%3D48%26spage%3D9823%26epage%3D9830%26doi%3D10.1021%2Fbi900258v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Fusco, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brear, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iegre, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georgiou, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sore, H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hyvönen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spring, D. R.</span></span> <span> </span><span class="NLM_article-title">A Fragment-Based Approach Leading to the Discovery of a Novel Binding Site and the Selective CK2 Inhibitor CAM4066</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">3471</span>– <span class="NLM_lpage">3482</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2017.04.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1016%2Fj.bmc.2017.04.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=28495381" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC2sXntlWktbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=3471-3482&author=C.+De%0AFuscoauthor=P.+Brearauthor=J.+Iegreauthor=K.+H.+Georgiouauthor=H.+F.+Soreauthor=M.+Hyv%C3%B6nenauthor=D.+R.+Spring&title=A+Fragment-Based+Approach+Leading+to+the+Discovery+of+a+Novel+Binding+Site+and+the+Selective+CK2+Inhibitor+CAM4066&doi=10.1016%2Fj.bmc.2017.04.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">A fragment-based approach leading to the discovery of a novel binding site and the selective CK2 inhibitor CAM4066</span></div><div class="casAuthors">De Fusco, Claudia; Brear, Paul; Iegre, Jessica; Georgiou, Kathy Hadje; Sore, Hannah F.; Hyvonen, Marko; Spring, David R.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3471-3482</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Recently the authors reported the discovery of a potent and selective CK2α inhibitor CAM4066.  This compd. inhibits CK2 activity by exploiting a pocket located outside the ATP binding site (αD pocket).  Here the authors describe in detail the journey that led to the discovery of CAM4066 using the challenging fragment linking strategy.  Specifically, the authors aimed to develop inhibitors by linking a high-affinity fragment anchored in the αD site to a weakly binding warhead fragment occupying the ATP site.  Moreover, the authors describe the remarkable impact that mol. modeling had on the development of this novel chem. tool.  The work described herein shows potential for the development of a novel class of CK2 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-es_MXrRt5bVg90H21EOLACvtfcHk0ljw8HgcUgll7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXntlWktbc%253D&md5=2c4bf2e6b0528149dc09d04badfdeab3</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2017.04.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2017.04.037%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BFusco%26aufirst%3DC.%26aulast%3DBrear%26aufirst%3DP.%26aulast%3DIegre%26aufirst%3DJ.%26aulast%3DGeorgiou%26aufirst%3DK.%2BH.%26aulast%3DSore%26aufirst%3DH.%2BF.%26aulast%3DHyv%25C3%25B6nen%26aufirst%3DM.%26aulast%3DSpring%26aufirst%3DD.%2BR.%26atitle%3DA%2520Fragment-Based%2520Approach%2520Leading%2520to%2520the%2520Discovery%2520of%2520a%2520Novel%2520Binding%2520Site%2520and%2520the%2520Selective%2520CK2%2520Inhibitor%2520CAM4066%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2017%26volume%3D25%26spage%3D3471%26epage%3D3482%26doi%3D10.1016%2Fj.bmc.2017.04.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McIntyre, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vrzal, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birchall, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mpamhanga, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coombs, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richards, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veverka, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delft, F. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merritt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayliss, R.</span></span> <span> </span><span class="NLM_article-title">Characterization of Three Druggable Hot-Spots in the Aurora-a/Tpx2 Interaction Using Biochemical, Biophysical, and Fragment-Based Approaches</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">2906</span>– <span class="NLM_lpage">2914</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00537</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00537" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1KqsrbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=2906-2914&author=P.+J.+McIntyreauthor=P.+M.+Collinsauthor=L.+Vrzalauthor=K.+Birchallauthor=L.+H.+Arnoldauthor=C.+Mpamhangaauthor=P.+J.+Coombsauthor=S.+G.+Burgessauthor=M.+W.+Richardsauthor=A.+Winterauthor=V.+Veverkaauthor=F.+V.+Delftauthor=A.+Merrittauthor=R.+Bayliss&title=Characterization+of+Three+Druggable+Hot-Spots+in+the+Aurora-a%2FTpx2+Interaction+Using+Biochemical%2C+Biophysical%2C+and+Fragment-Based+Approaches&doi=10.1021%2Facschembio.7b00537"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of Three Druggable Hot-Spots in the Aurora-A/TPX2 Interaction Using Biochemical, Biophysical, and Fragment-Based Approaches</span></div><div class="casAuthors">McIntyre, Patrick J.; Collins, Patrick M.; Vrzal, Lukas; Birchall, Kristian; Arnold, Laurence H.; Mpamhanga, Chido; Coombs, Peter J.; Burgess, Selena G.; Richards, Mark W.; Winter, Anja; Veverka, Vaclav; Delft, Frank von; Merritt, Andy; Bayliss, Richard</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2906-2914</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The mitotic kinase Aurora-A and its partner protein TPX2 (Targeting Protein for Xenopus kinesin-like protein 2), are overexpressed in cancers and it has been proposed that they work together as an oncogenic holoenzyme.  TPX2 is responsible for activating Aurora-A during mitosis, ensuring proper cell division.  Disruption of the interface with TPX2 is therefore a potential target for novel anti-cancer drugs that exploit the increased sensitivity of cancer cells to mitotic stress.  Here, we investigate the interface using co-pptn. assays and isothermal titrn. calorimetry to quantify the energetic contribution of individual residues of TPX2.  Residues Tyr8, Tyr10, Phe16 and Trp34 of TPX2 are shown to be crucial for robust complex formation, suggesting that the interaction could be abrogated through blocking any of the three pockets on Aurora-A that complement these residues.  Phosphorylation of Aurora-A on Thr288 is also necessary for high-affinity binding and here we identify arginine residues that communicate the phosphorylation of Thr288 to the TPX2 binding site.  With these findings in mind, we conducted a high-throughput X-ray crystallog.-based screen of 1255 fragments against Aurora-A and identified 59 hits.  Over three-quarters of these hits bound to the pockets described above, both validating our identification of hot-spots and demonstrating the druggability of this protein-protein interaction.  Our study exemplifies the potential of high-throughput crystallog. facilities such as XChem to aid drug discovery.  These results will accelerate the development of chem. inhibitors of the Aurora-A/TPX2 interaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojoCM8ethY8rVg90H21EOLACvtfcHk0ljw8HgcUgll7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1KqsrbK&md5=5d4fcd13e24ae8776fcfce019ec10a0c</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00537&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00537%26sid%3Dliteratum%253Aachs%26aulast%3DMcIntyre%26aufirst%3DP.%2BJ.%26aulast%3DCollins%26aufirst%3DP.%2BM.%26aulast%3DVrzal%26aufirst%3DL.%26aulast%3DBirchall%26aufirst%3DK.%26aulast%3DArnold%26aufirst%3DL.%2BH.%26aulast%3DMpamhanga%26aufirst%3DC.%26aulast%3DCoombs%26aufirst%3DP.%2BJ.%26aulast%3DBurgess%26aufirst%3DS.%2BG.%26aulast%3DRichards%26aufirst%3DM.%2BW.%26aulast%3DWinter%26aufirst%3DA.%26aulast%3DVeverka%26aufirst%3DV.%26aulast%3DDelft%26aufirst%3DF.%2BV.%26aulast%3DMerritt%26aufirst%3DA.%26aulast%3DBayliss%26aufirst%3DR.%26atitle%3DCharacterization%2520of%2520Three%2520Druggable%2520Hot-Spots%2520in%2520the%2520Aurora-a%252FTpx2%2520Interaction%2520Using%2520Biochemical%252C%2520Biophysical%252C%2520and%2520Fragment-Based%2520Approaches%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D2906%26epage%3D2914%26doi%3D10.1021%2Facschembio.7b00537" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Engel, M.</span></span> <span> </span><span class="NLM_article-title">The PIF Pocket of AGC Kinases: A Target Site for Allosteric Modulators and Protein–Protein Interaction Inhibitors</span>. In  <i>Protein-Protein Interactions in Drug Discovery; Methods and Principles in Medicinal Chemistry</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mannhold, R.</span>, <span class="NLM_string-name">Kubinyi, H.</span>, <span class="NLM_string-name">Folkers, G.</span></span>; Eds.; <span class="NLM_publisher-name">Wiley-VCH</span>: <span class="NLM_publisher-loc">Weinheim, Germany</span>, <span class="NLM_year">2013</span>; pp  <span class="NLM_fpage">187</span>– <span class="NLM_lpage">223</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1002%2F9783527648207.ch9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&pages=187-223&author=M.+Engelauthor=R.+Mannhold&author=H.+Kubinyi&author=G.+Folkers&title=Protein-Protein+Interactions+in+Drug+Discovery%3B+Methods+and+Principles+in+Medicinal+Chemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1002%2F9783527648207.ch9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9783527648207.ch9%26sid%3Dliteratum%253Aachs%26aulast%3DEngel%26aufirst%3DM.%26atitle%3DThe%2520PIF%2520Pocket%2520of%2520AGC%2520Kinases%253A%2520A%2520Target%2520Site%2520for%2520Allosteric%2520Modulators%2520and%2520Protein%25E2%2580%2593Protein%2520Interaction%2520Inhibitors%26btitle%3DProtein-Protein%2520Interactions%2520in%2520Drug%2520Discovery%253B%2520Methods%2520and%2520Principles%2520in%2520Medicinal%2520Chemistry%26aulast%3DMannhold%26aufirst%3DR.%26pub%3DWiley-VCH%26date%3D2013%26spage%3D187%26epage%3D223" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frödin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antal, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dummler, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deak, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gammeltoft, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biondi, R. M.</span></span> <span> </span><span class="NLM_article-title">A Phosphoserine/Threonine-Binding Pocket in AGC Kinases and PDK1Mediates Activation by Hydrophobic Motif Phosphorylation</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">5396</span>– <span class="NLM_lpage">5407</span>, <span class="refDoi"> DOI: 10.1093/emboj/cdf551</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1093%2Femboj%2Fcdf551" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=12374740" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A280%3ADC%252BD38nlvVWhsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2002&pages=5396-5407&author=M.+Fr%C3%B6dinauthor=T.+L.+Antalauthor=B.+A.+Dummlerauthor=C.+J.+Jensenauthor=M.+Deakauthor=S.+Gammeltoftauthor=R.+M.+Biondi&title=A+Phosphoserine%2FThreonine-Binding+Pocket+in+AGC+Kinases+and+PDK1Mediates+Activation+by+Hydrophobic+Motif+Phosphorylation&doi=10.1093%2Femboj%2Fcdf551"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">A phosphoserine/threonine-binding pocket in AGC kinases and PDK1 mediates activation by hydrophobic motif phosphorylation</span></div><div class="casAuthors">Frodin Morten; Antal Torben L; Dummler Bettina A; Jensen Claus J; Deak Maria; Gammeltoft Steen; Biondi Ricardo M</div><div class="citationInfo"><span class="NLM_cas:title">The EMBO journal</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5396-407</span>
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    </div><div class="casAbstract">The growth factor-activated AGC protein kinases RSK, S6K, PKB, MSK and SGK are activated by serine/threonine phosphorylation in the activation loop and in the hydrophobic motif, C-terminal to the kinase domain.  In some of these kinases, phosphorylation of the hydrophobic motif creates a specific docking site that recruits and activates PDK1, which then phosphorylates the activation loop.  Here, we discover a pocket in the kinase domain of PDK1 that recognizes the phosphoserine/phosphothreonine in the hydrophobic motif by identifying two oppositely positioned arginine and lysine residues that bind the phosphate.  Moreover, we demonstrate that RSK2, S6K1, PKBalpha, MSK1 and SGK1 contain a similar phosphate-binding pocket, which they use for intramolecular interaction with their own phosphorylated hydrophobic motif.  Molecular modelling and experimental data provide evidence for a common activation mechanism in which the phosphorylated hydrophobic motif and activation loop act on the alphaC-helix of the kinase structure to induce synergistic stimulation of catalytic activity.  Sequence conservation suggests that this mechanism is a key feature in activation of >40 human AGC kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSb-TVEWss6nftos8R3JUhXfW6udTcc2ea3tnW0qWYHSbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD38nlvVWhsw%253D%253D&md5=40a9f8370c7f7634dcc961988f252924</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1093%2Femboj%2Fcdf551&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Femboj%252Fcdf551%26sid%3Dliteratum%253Aachs%26aulast%3DFr%25C3%25B6din%26aufirst%3DM.%26aulast%3DAntal%26aufirst%3DT.%2BL.%26aulast%3DDummler%26aufirst%3DB.%2BA.%26aulast%3DJensen%26aufirst%3DC.%2BJ.%26aulast%3DDeak%26aufirst%3DM.%26aulast%3DGammeltoft%26aufirst%3DS.%26aulast%3DBiondi%26aufirst%3DR.%2BM.%26atitle%3DA%2520Phosphoserine%252FThreonine-Binding%2520Pocket%2520in%2520AGC%2520Kinases%2520and%2520PDK1Mediates%2520Activation%2520by%2520Hydrophobic%2520Motif%2520Phosphorylation%26jtitle%3DEMBO%2520J.%26date%3D2002%26volume%3D21%26spage%3D5396%26epage%3D5407%26doi%3D10.1093%2Femboj%2Fcdf551" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Simpson, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertrand, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borthwick, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campobasso, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chabanet, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coggins, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cottom, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christensen, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hobbs, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangatt, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munoz-Muriedas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliff, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott-Stevens, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Washio, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. J.</span></span> <span> </span><span class="NLM_article-title">Identification and Optimization of Novel Small C-Abl Kinase Activators Using Fragment and HTS Methodologies</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">2154</span>– <span class="NLM_lpage">2171</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01872</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01872" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFKjsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=2154-2171&author=G.+L.+Simpsonauthor=S.+M.+Bertrandauthor=J.+A.+Borthwickauthor=N.+Campobassoauthor=J.+Chabanetauthor=S.+Chenauthor=J.+Cogginsauthor=J.+Cottomauthor=S.+B.+Christensenauthor=H.+C.+Dawsonauthor=H.+L.+Evansauthor=A.+N.+Hobbsauthor=X.+Hongauthor=B.+Mangattauthor=J.+Munoz-Muriedasauthor=A.+Oliffauthor=D.+Qinauthor=P.+Scott-Stevensauthor=P.+Wardauthor=Y.+Washioauthor=J.+Yangauthor=R.+J.+Young&title=Identification+and+Optimization+of+Novel+Small+C-Abl+Kinase+Activators+Using+Fragment+and+HTS+Methodologies&doi=10.1021%2Facs.jmedchem.8b01872"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Optimization of Novel Small c-Abl Kinase Activators Using Fragment and HTS Methodologies</span></div><div class="casAuthors">Simpson, Graham L.; Bertrand, Sophie M.; Borthwick, Jennifer A.; Campobasso, Nino; Chabanet, Julien; Chen, Susan; Coggins, Julia; Cottom, Josh; Christensen, Siegfried B.; Dawson, Helen C.; Evans, Helen L.; Hobbs, Andrew N.; Hong, Xuan; Mangatt, Biju; Munoz-Muriedas, Jordi; Oliff, Allen; Qin, Donghui; Scott-Stevens, Paul; Ward, Paris; Washio, Yoshiaki; Yang, Jingsong; Young, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2154-2171</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Abelson kinase (c-Abl) is a ubiquitously expressed, nonreceptor tyrosine kinase which plays a key role in cell differentiation and survival.  It was hypothesized that transient activation of c-Abl kinase via displacement of the N-terminal autoinhibitory "myristoyl latch", may lead to an increased hematopoietic stem cell differentiation.  This would increase the nos. of circulating neutrophils and so be an effective treatment for chemotherapy-induced neutropenia.  This paper describes the discovery and optimization of a thiazole series of novel small mol. c-Abl activators, initially identified by a high throughput screening.  Subsequently, a scaffold-hop, which exploited the improved physicochem. properties of a dihydropyrazole analog, identified through fragment screening, delivered potent, sol., cell-active c-Abl activators, which demonstrated the intracellular activation of c-Abl in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoA0_N7VrekSbVg90H21EOLACvtfcHk0lgT_jjHfD7POA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFKjsr8%253D&md5=c187254649943b5dd3c94d64204d4aa1</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01872&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01872%26sid%3Dliteratum%253Aachs%26aulast%3DSimpson%26aufirst%3DG.%2BL.%26aulast%3DBertrand%26aufirst%3DS.%2BM.%26aulast%3DBorthwick%26aufirst%3DJ.%2BA.%26aulast%3DCampobasso%26aufirst%3DN.%26aulast%3DChabanet%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DCoggins%26aufirst%3DJ.%26aulast%3DCottom%26aufirst%3DJ.%26aulast%3DChristensen%26aufirst%3DS.%2BB.%26aulast%3DDawson%26aufirst%3DH.%2BC.%26aulast%3DEvans%26aufirst%3DH.%2BL.%26aulast%3DHobbs%26aufirst%3DA.%2BN.%26aulast%3DHong%26aufirst%3DX.%26aulast%3DMangatt%26aufirst%3DB.%26aulast%3DMunoz-Muriedas%26aufirst%3DJ.%26aulast%3DOliff%26aufirst%3DA.%26aulast%3DQin%26aufirst%3DD.%26aulast%3DScott-Stevens%26aufirst%3DP.%26aulast%3DWard%26aufirst%3DP.%26aulast%3DWashio%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DYoung%26aufirst%3DR.%2BJ.%26atitle%3DIdentification%2520and%2520Optimization%2520of%2520Novel%2520Small%2520C-Abl%2520Kinase%2520Activators%2520Using%2520Fragment%2520and%2520HTS%2520Methodologies%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D2154%26epage%3D2171%26doi%3D10.1021%2Facs.jmedchem.8b01872" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brear, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Fusco, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hadje Georgiou, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis-Newton, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stubbs, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sore, H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkitaraman, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spring, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hyvonen, M.</span></span> <span> </span><span class="NLM_article-title">Specific Inhibition of Ck2alpha from an Anchor Outside the Active Site</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">6839</span>– <span class="NLM_lpage">6845</span>, <span class="refDoi"> DOI: 10.1039/C6SC02335E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1039%2FC6SC02335E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=28451126" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFCrtbbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=6839-6845&author=P.+Brearauthor=C.+De+Fuscoauthor=K.+Hadje+Georgiouauthor=N.+J.+Francis-Newtonauthor=C.+J.+Stubbsauthor=H.+F.+Soreauthor=A.+R.+Venkitaramanauthor=C.+Abellauthor=D.+R.+Springauthor=M.+Hyvonen&title=Specific+Inhibition+of+Ck2alpha+from+an+Anchor+Outside+the+Active+Site&doi=10.1039%2FC6SC02335E"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Specific inhibition of CK2α from an anchor outside the active site</span></div><div class="casAuthors">Brear, Paul; De Fusco, Claudia; Hadje Georgiou, Kathy; Francis-Newton, Nicola J.; Stubbs, Christopher J.; Sore, Hannah F.; Venkitaraman, Ashok R.; Abell, Chris; Spring, David R.; Hyvonen, Marko</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">6839-6845</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The development of selective inhibitors of protein kinases is challenging because of the significant conservation of the ATP binding site.  Here, we describe a new mechanism by which the protein kinase CK2α can be selectively inhibited using features outside the ATP site.  We have identified a new binding site for small mols. on CK2α adjacent to the ATP site and behind the αD loop, termed the αD pocket.  An elaborated fragment anchored in this site has been linked with a low affinity fragment binding in the ATP site, creating a novel and selective inhibitor (CAM4066) that binds CK2α with a Kd of 320 nM and shows significantly improved selectivity compared to other CK2α inhibitors.  CAM4066 shows target engagement in several cell lines and similar potency to clin. trial candidate CX4945.  Our data demonstrate that targeting a poorly conserved, cryptic pocket allows inhibition of CK2α via a novel mechanism, enabling the development of a new generation of selective CK2α inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3FqxiBkU9frVg90H21EOLACvtfcHk0lg6S9RNMKq5bA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFCrtbbP&md5=c4652010f4da70a128fb1dab64f811d0</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1039%2FC6SC02335E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6SC02335E%26sid%3Dliteratum%253Aachs%26aulast%3DBrear%26aufirst%3DP.%26aulast%3DDe%2BFusco%26aufirst%3DC.%26aulast%3DHadje%2BGeorgiou%26aufirst%3DK.%26aulast%3DFrancis-Newton%26aufirst%3DN.%2BJ.%26aulast%3DStubbs%26aufirst%3DC.%2BJ.%26aulast%3DSore%26aufirst%3DH.%2BF.%26aulast%3DVenkitaraman%26aufirst%3DA.%2BR.%26aulast%3DAbell%26aufirst%3DC.%26aulast%3DSpring%26aufirst%3DD.%2BR.%26aulast%3DHyvonen%26aufirst%3DM.%26atitle%3DSpecific%2520Inhibition%2520of%2520Ck2alpha%2520from%2520an%2520Anchor%2520Outside%2520the%2520Active%2520Site%26jtitle%3DChem.%2520Sci.%26date%3D2016%26volume%3D7%26spage%3D6839%26epage%3D6845%26doi%3D10.1039%2FC6SC02335E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Welsch, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snyder, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stockwell, B. R.</span></span> <span> </span><span class="NLM_article-title">Privileged Scaffolds for Library Design and Drug Discovery</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">347</span>– <span class="NLM_lpage">361</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2010.02.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1016%2Fj.cbpa.2010.02.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=20303320" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmvVWqsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2010&pages=347-361&author=M.+E.+Welschauthor=S.+A.+Snyderauthor=B.+R.+Stockwell&title=Privileged+Scaffolds+for+Library+Design+and+Drug+Discovery&doi=10.1016%2Fj.cbpa.2010.02.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Privileged scaffolds for library design and drug discovery</span></div><div class="casAuthors">Welsch, Matthew E.; Snyder, Scott A.; Stockwell, Brent R.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">347-361</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">This review explores the concept of using privileged scaffolds to identify biol. active compds. through building chem. libraries.  We hope to accomplish three main objectives: to provide one of the most comprehensive listings of privileged scaffolds; to reveal through four selected examples the present state of the art in privileged scaffold library synthesis (in hopes of inspiring new and even more creative approaches); and also to offer some thoughts on how new privileged scaffolds might be identified and exploited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWQPxdsWawiLVg90H21EOLACvtfcHk0lg6S9RNMKq5bA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmvVWqsLs%253D&md5=fd0c2a642f063fd6f42f2c34fd186f8a</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2010.02.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2010.02.018%26sid%3Dliteratum%253Aachs%26aulast%3DWelsch%26aufirst%3DM.%2BE.%26aulast%3DSnyder%26aufirst%3DS.%2BA.%26aulast%3DStockwell%26aufirst%3DB.%2BR.%26atitle%3DPrivileged%2520Scaffolds%2520for%2520Library%2520Design%2520and%2520Drug%2520Discovery%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D14%26spage%3D347%26epage%3D361%26doi%3D10.1016%2Fj.cbpa.2010.02.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mehndiratta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sapra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nepali, K.</span></span> <span> </span><span class="NLM_article-title">Quinazolines as Apoptosis Inducers and Inhibitors: A Review of Patent Literature</span>. <i>Recent Pat. Anti-Cancer Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2</span>– <span class="NLM_lpage">66</span>, <span class="refDoi"> DOI: 10.2174/1574892811666151218151506</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.2174%2F1574892811666151218151506" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=26681186" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC28XisVGls70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=2-66&author=S.+Mehndirattaauthor=S.+Sapraauthor=G.+Singhauthor=M.+Singhauthor=K.+Nepali&title=Quinazolines+as+Apoptosis+Inducers+and+Inhibitors%3A+A+Review+of+Patent+Literature&doi=10.2174%2F1574892811666151218151506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Quinazolines as Apoptosis Inducers and Inhibitors: A Review of Patent Literature</span></div><div class="casAuthors">Mehndiratta, Samir; Sapra, Sameer; Singh, Gurpreet; Singh, Manwinder; Nepali, Kunal</div><div class="citationInfo"><span class="NLM_cas:title">Recent Patents on Anti-Cancer Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2-66</span>CODEN:
                <span class="NLM_cas:coden">RPADDY</span>;
        ISSN:<span class="NLM_cas:issn">1574-8928</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Quinazoline scaffold has been successfully utilized for development of various inhibitors of tubulin, epidermal growth factor receptor (EGFR), polo like kinases (PLKs), Hedgehog-Gli signaling pathway and protein kinase B (PKB) /Akt pathway.  Compds. based on quinazolines have shown efficacies in μM to nM range in various cancer cell lines and thus could be useful scaffolds for development of both apoptosis inducers as well as inhibitors.  This compilation is based on various patents published till 2015 and divides the quinazolines in two main categories: Quinazolines as apoptosis inducers and as apoptosis inhibitors.  These two main categories are further sub-categorized based on the use/pharmacol. indications for these classes of patented compds.  This review will act as a tool for the researchers working on exploring the anticancer potential of quinazoline as a privileged heterocyclic.  The promising anticancer profile of some of the quinazoline based compds. clearly highlights the clin. potential of this heterocycle.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRFoI4uJ8BA7Vg90H21EOLACvtfcHk0lg6S9RNMKq5bA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XisVGls70%253D&md5=3ad6dd317a6a8e19d0be41e7b59d70e1</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.2174%2F1574892811666151218151506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1574892811666151218151506%26sid%3Dliteratum%253Aachs%26aulast%3DMehndiratta%26aufirst%3DS.%26aulast%3DSapra%26aufirst%3DS.%26aulast%3DSingh%26aufirst%3DG.%26aulast%3DSingh%26aufirst%3DM.%26aulast%3DNepali%26aufirst%3DK.%26atitle%3DQuinazolines%2520as%2520Apoptosis%2520Inducers%2520and%2520Inhibitors%253A%2520A%2520Review%2520of%2520Patent%2520Literature%26jtitle%3DRecent%2520Pat.%2520Anti-Cancer%2520Drug%2520Discovery%26date%3D2016%26volume%3D11%26spage%3D2%26epage%3D66%26doi%3D10.2174%2F1574892811666151218151506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ramdas, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnin, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plunkett, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallick, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Budde, R. J.</span></span> <span> </span><span class="NLM_article-title">Benzodiazepine Compounds as Inhibitors of the Src Protein Tyrosine Kinase: Screening of a Combinatorial Library of 1,4-Benzodiazepines</span>. <i>Arch. Biochem. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>368</i></span>,  <span class="NLM_fpage">394</span>– <span class="NLM_lpage">400</span>, <span class="refDoi"> DOI: 10.1006/abbi.1999.1313</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1006%2Fabbi.1999.1313" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10441393" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADyaK1MXltVahsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=1999&pages=394-400&author=L.+Ramdasauthor=B.+A.+Bunninauthor=M.+J.+Plunkettauthor=G.+Sunauthor=J.+Ellmanauthor=G.+Gallickauthor=R.+J.+Budde&title=Benzodiazepine+Compounds+as+Inhibitors+of+the+Src+Protein+Tyrosine+Kinase%3A+Screening+of+a+Combinatorial+Library+of+1%2C4-Benzodiazepines&doi=10.1006%2Fabbi.1999.1313"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Benzodiazepine compounds as inhibitors of the Src protein tyrosine kinase: screening of a combinatorial library of 1,4-benzodiazepines</span></div><div class="casAuthors">Ramdas, Latha; Bunnin, Barry A.; Plunkett, Matthew J.; Sun, Gongqin; Ellman, Jonathan; Gallick, Gary; Budde, Raymond J. A.</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">394-400</span>CODEN:
                <span class="NLM_cas:coden">ABBIA4</span>;
        ISSN:<span class="NLM_cas:issn">0003-9861</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">We screened 1680 spatially sepd. compds. of a diverse combinatorial library of 1,4-benzodiazepines for their ability to inhibit the kinase activity of protein tyrosine kinases Src, Yes, Abl, Lck, Csk, and fibroblast growth factor receptor.  This screening yielded novel ligands for the protein tyrosine kinase Src.  In the 1,4-benzodiazepine-2-one scaffold, the preferred substituent at position R1 was 4-hydroxyphenylmethyl or a 3-indolemethyl derived from a tyrosine or tryptophan used in building the benzodiazepine, while the substituent at R2, introduced by alkylating agents, was preferably arom. in nature.  The preferred ring structure introduced on the bicyclic ring of the scaffold by acid chlorides was a p-hydroxy Ph group.  The lead compd., designated as N-l-Yaa, has a l-4-hydroxyphenylmethyl ring at R1 and a biphenylmethyl substituent at R2.  The compd. has an IC50 of 73 μM against Src, 2- to 6-fold lower than against other protein tyrosine kinases and >10-fold lower than against other nucleotide-utilizing enzymes.  The mechanism of binding of N-l-Yaa to Src is mixed against the peptidic substrate with a Ki of 35 μM and noncompetitive against ATP-Mg with a Ki of 17 μM.  Multiple inhibition anal. of the lead compd. in the presence of other competitive inhibitors demonstrated that the binding of the lead compd. is nonexclusive to the other competitive inhibitor.  The inhibitor was found to be nontoxic to the AFB-13-human fibroblasts cells and inhibited the colony formation of HT-29 colon adenocarcinoma cells that are dependent on Src activity.  (c) 1999 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr94Bg8htImNrVg90H21EOLACvtfcHk0lg6S9RNMKq5bA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXltVahsL0%253D&md5=438dac1a4c7eb8f1cb26315e0b1b2a4b</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1006%2Fabbi.1999.1313&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fabbi.1999.1313%26sid%3Dliteratum%253Aachs%26aulast%3DRamdas%26aufirst%3DL.%26aulast%3DBunnin%26aufirst%3DB.%2BA.%26aulast%3DPlunkett%26aufirst%3DM.%2BJ.%26aulast%3DSun%26aufirst%3DG.%26aulast%3DEllman%26aufirst%3DJ.%26aulast%3DGallick%26aufirst%3DG.%26aulast%3DBudde%26aufirst%3DR.%2BJ.%26atitle%3DBenzodiazepine%2520Compounds%2520as%2520Inhibitors%2520of%2520the%2520Src%2520Protein%2520Tyrosine%2520Kinase%253A%2520Screening%2520of%2520a%2520Combinatorial%2520Library%2520of%25201%252C4-Benzodiazepines%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D1999%26volume%3D368%26spage%3D394%26epage%3D400%26doi%3D10.1006%2Fabbi.1999.1313" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span>RDKit:
Cheminformatics and Machine Learning Software, <span class="NLM_year">2013</span>; <a href="http://www.rdkit.org" class="extLink">http://www.rdkit.org</a> (accessed Apr
25, 2019).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=RDKit%3A%0ACheminformatics+and+Machine+Learning+Software%2C+2013%3B+http%3A%2F%2Fwww.rdkit.org+%28accessed+Apr%0A25%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, G. A.</span></span> <span> </span><span class="NLM_article-title">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2740</span>, <span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+Substructure+Filters+for+Removal+of+Pan+Assay+Interference+Compounds+%28PAINS%29+from+Screening+Libraries+and+for+Their+Exclusion+in+Bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0lg2puaypSLNSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520Substructure%2520Filters%2520for%2520Removal%2520of%2520Pan%2520Assay%2520Interference%2520Compounds%2520%2528PAINS%2529%2520from%2520Screening%2520Libraries%2520and%2520for%2520Their%2520Exclusion%2520in%2520Bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaulton, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellis, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bento, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chambers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hersey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Light, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGlinchey, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michalovich, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Lazikani, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span> <span> </span><span class="NLM_article-title">ChEMBL: A Large-scale Bioactivity Database for Drug Discovery</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">D1100</span>– <span class="NLM_lpage">D1107</span>, <span class="refDoi"> DOI: 10.1093/nar/gkr777</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1093%2Fnar%2Fgkr777" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=21948594" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs12htbjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=D1100-D1107&author=A.+Gaultonauthor=L.+J.+Bellisauthor=A.+P.+Bentoauthor=J.+Chambersauthor=M.+Daviesauthor=A.+Herseyauthor=Y.+Lightauthor=S.+McGlincheyauthor=D.+Michalovichauthor=B.+Al-Lazikaniauthor=J.+P.+Overington&title=ChEMBL%3A+A+Large-scale+Bioactivity+Database+for+Drug+Discovery&doi=10.1093%2Fnar%2Fgkr777"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">ChEMBL: a large-scale bioactivity database for drug discovery</span></div><div class="casAuthors">Gaulton, Anna; Bellis, Louisa J.; Bento, A. Patricia; Chambers, Jon; Davies, Mark; Hersey, Anne; Light, Yvonne; McGlinchey, Shaun; Michalovich, David; Al-Lazikani, Bissan; Overington, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1100-D1107</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ChEMBL is an Open Data database contg. binding, functional and ADMET information for a large no. of drug-like bioactive compds.  These data are manually abstracted from the primary published literature on a regular basis, then further curated and standardized to maximize their quality and utility across a wide range of chem. biol. and drug-discovery research problems.  Currently, the database contains 5.4 million bioactivity measurements for more than 1 million compds. and 5200 protein targets.  Access is available through a web-based interface, data downloads and web services at: https://www.ebi.ac.uk/chembldb.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9e9jRENTtubVg90H21EOLACvtfcHk0lg2puaypSLNSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs12htbjN&md5=aedf7793e1ca54b6a4fa272ea3ef7d0e</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr777&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr777%26sid%3Dliteratum%253Aachs%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DBellis%26aufirst%3DL.%2BJ.%26aulast%3DBento%26aufirst%3DA.%2BP.%26aulast%3DChambers%26aufirst%3DJ.%26aulast%3DDavies%26aufirst%3DM.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DLight%26aufirst%3DY.%26aulast%3DMcGlinchey%26aufirst%3DS.%26aulast%3DMichalovich%26aufirst%3DD.%26aulast%3DAl-Lazikani%26aufirst%3DB.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DChEMBL%253A%2520A%2520Large-scale%2520Bioactivity%2520Database%2520for%2520Drug%2520Discovery%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3DD1100%26epage%3DD1107%26doi%3D10.1093%2Fnar%2Fgkr777" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hussain, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rea, C.</span></span> <span> </span><span class="NLM_article-title">Computationally Efficient Algorithm to Identify Matched Molecular Pairs (MMPs) in Large Data Sets</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">339</span>– <span class="NLM_lpage">348</span>, <span class="refDoi"> DOI: 10.1021/ci900450m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci900450m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlWltr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=339-348&author=J.+Hussainauthor=C.+Rea&title=Computationally+Efficient+Algorithm+to+Identify+Matched+Molecular+Pairs+%28MMPs%29+in+Large+Data+Sets&doi=10.1021%2Fci900450m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Computationally Efficient Algorithm to Identify Matched Molecular Pairs (MMPs) in Large Data Sets</span></div><div class="casAuthors">Hussain, Jameed; Rea, Ceara</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">339-348</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Modern drug discovery organizations generate large vols. of SAR data.  A promising methodol. that can be used to mine this chem. data to identify novel structure-activity relationships is the matched mol. pair (MMP) methodol.  However, before the full potential of the MMP methodol. can be utilized, a MMP identification method that is capable of identifying all MMPs in large chem. data sets on modest computational hardware is required.  In this paper we report an algorithm that is capable of systematically generating all MMPs in chem. data sets.  Addnl., the algorithm is computationally efficient enough to be applied on large data sets.  As an example the algorithm was used to identify the MMPs in the ∼300k NIH MLSMR set.  The algorithm identified ∼5.3 million matched mol. pairs in the set.  These pairs cover ∼2.6 million unique mol. transformations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoF0rN8jaKSjrVg90H21EOLACvtfcHk0li0sWWHoqhsPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlWltr4%253D&md5=23964abebc23274f5c36c319bee52bab</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Fci900450m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900450m%26sid%3Dliteratum%253Aachs%26aulast%3DHussain%26aufirst%3DJ.%26aulast%3DRea%26aufirst%3DC.%26atitle%3DComputationally%2520Efficient%2520Algorithm%2520to%2520Identify%2520Matched%2520Molecular%2520Pairs%2520%2528MMPs%2529%2520in%2520Large%2520Data%2520Sets%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D339%26epage%3D348%26doi%3D10.1021%2Fci900450m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kunimoto, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Design of a Tripartite Network for the Prediction of Drug Targets</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">321</span>– <span class="NLM_lpage">330</span>, <span class="refDoi"> DOI: 10.1007/s10822-018-0098-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1007%2Fs10822-018-0098-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=29340865" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtVKru7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2018&pages=321-330&author=R.+Kunimotoauthor=J.+Bajorath&title=Design+of+a+Tripartite+Network+for+the+Prediction+of+Drug+Targets&doi=10.1007%2Fs10822-018-0098-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Design of a tripartite network for the prediction of drug targets</span></div><div class="casAuthors">Kunimoto, Ryo; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">321-330</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Drug-target networks have aided in many target prediction studies aiming at drug repurposing or the anal. of side effects.  Conventional drug-target networks are bipartite.  They contain two different types of nodes representing drugs and targets, resp., and edges indicating pairwise drug-target interactions.  In this work, we introduce a tripartite network consisting of drugs, other bioactive compds., and targets from different sources.  On the basis of analog relationships captured in the network and so-called neighbor targets of drugs, new drug targets can be inferred.  The tripartite network was found to have a stable structure and simulated network growth was accompanied by a steady increase in assortativity, reflecting increasing correlation between degrees of connected nodes leading to even network connectivity.  Local drug environments in the tripartite network typically contained neighbor targets and revealed interesting drug-compd.-target relationships for further anal.  Candidate targets were prioritized.  The tripartite network design extends std. drug-target networks and provides addnl. opportunities for drug target prediction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIv0Wn7vN4p7Vg90H21EOLACvtfcHk0li0sWWHoqhsPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtVKru7o%253D&md5=ea402669c22e80cfeebd7c1b954650b2</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1007%2Fs10822-018-0098-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-018-0098-x%26sid%3Dliteratum%253Aachs%26aulast%3DKunimoto%26aufirst%3DR.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DDesign%2520of%2520a%2520Tripartite%2520Network%2520for%2520the%2520Prediction%2520of%2520Drug%2520Targets%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2018%26volume%3D32%26spage%3D321%26epage%3D330%26doi%3D10.1007%2Fs10822-018-0098-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abad-Zapatero, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metz, J. T.</span></span> <span> </span><span class="NLM_article-title">Ligand Efficiency Indices as Guideposts for Drug Discovery</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">464</span>– <span class="NLM_lpage">469</span>, <span class="refDoi"> DOI: 10.1016/S1359-6446(05)03386-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=10.1016%2FS1359-6446%2805%2903386-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=15809192" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;key=1%3ACAS%3A280%3ADC%252BD2M7nsF2iug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2005&pages=464-469&author=C.+Abad-Zapateroauthor=J.+T.+Metz&title=Ligand+Efficiency+Indices+as+Guideposts+for+Drug+Discovery&doi=10.1016%2FS1359-6446%2805%2903386-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand efficiency indices as guideposts for drug discovery</span></div><div class="casAuthors">Abad-Zapatero Cele; Metz James T</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">464-9</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSHqVfZXUq224qdXsLcGTEUfW6udTcc2eZZKh2ArAdK67ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2M7nsF2iug%253D%253D&md5=1720f263a306122c86a57463e5a39287</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2805%2903386-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252805%252903386-6%26sid%3Dliteratum%253Aachs%26aulast%3DAbad-Zapatero%26aufirst%3DC.%26aulast%3DMetz%26aufirst%3DJ.%2BT.%26atitle%3DLigand%2520Efficiency%2520Indices%2520as%2520Guideposts%2520for%2520Drug%2520Discovery%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2005%26volume%3D10%26spage%3D464%26epage%3D469%26doi%3D10.1016%2FS1359-6446%2805%2903386-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4M15" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4M15','PDB','4M15'); return false;">PDB: 4M15</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LVO" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LVO','PDB','5LVO'); return false;">PDB: 5LVO</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ROC" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ROC','PDB','3ROC'); return false;">PDB: 3ROC</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PYY" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PYY','PDB','3PYY'); return false;">PDB: 3PYY</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6FVF" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6FVF','PDB','6FVF'); return false;">PDB: 6FVF</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3JVR" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3JVR','PDB','3JVR'); return false;">PDB: 3JVR</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5OTZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5OTZ','PDB','5OTZ'); return false;">PDB: 5OTZ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6EHK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6EHK','PDB','6EHK'); return false;">PDB: 6EHK</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5OSJ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5OSJ','PDB','5OSJ'); return false;">PDB: 5OSJ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BKW" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BKW','PDB','6BKW'); return false;">PDB: 6BKW</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ORL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ORL','PDB','5ORL'); return false;">PDB: 5ORL</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6D1Z" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6D1Z','PDB','6D1Z'); return false;">PDB: 6D1Z</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1S9I" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1S9I','PDB','1S9I'); return false;">PDB: 1S9I</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1S9J" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1S9J','PDB','1S9J'); return false;">PDB: 1S9J</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3HRF" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3HRF','PDB','3HRF'); return false;">PDB: 3HRF</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3C5U" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3C5U','PDB','3C5U'); return false;">PDB: 3C5U</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1OPL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1OPL','PDB','1OPL'); return false;">PDB: 1OPL</a></li></ul></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c02076&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c02076%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c02076" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679980c5af153ce8","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
