
<html lang="en"     class="pb-page"  data-request-id="83402e20-64b8-464a-aa66-2d1e32201f3a"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;issue:issue:10.1021/jmcmar.2021.64.issue-9;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;article:article:10.1021/acs.jmedchem.1c00078;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Design of Dimeric Bile Acid Derivatives as Potent and Selective Human NTCP Inhibitors" /></meta><meta name="dc.Creator" content="Yang  Liu" /></meta><meta name="dc.Creator" content="Lei  Zhang" /></meta><meta name="dc.Creator" content="Huan  Yan" /></meta><meta name="dc.Creator" content="Zhiqiang  Wang" /></meta><meta name="dc.Creator" content="Guoliang  Sun" /></meta><meta name="dc.Creator" content="Xiao  Song" /></meta><meta name="dc.Creator" content="Zhongmin  Zhou" /></meta><meta name="dc.Creator" content="Bo  Peng" /></meta><meta name="dc.Creator" content="Liwei  Yan" /></meta><meta name="dc.Creator" content="Qingcui  Wu" /></meta><meta name="dc.Creator" content="Wenhui  Li" /></meta><meta name="dc.Creator" content="Xiangbing  Qi" /></meta><meta name="dc.Description" content="Dimeric bile acid derivatives (DBADs) were developed and tested for their anti-HBV and anti-HDV activities as sodium taurocholate cotransporting polypeptide (NTCP) inhibitors. DBADs exhibited stron..." /></meta><meta name="Description" content="Dimeric bile acid derivatives (DBADs) were developed and tested for their anti-HBV and anti-HDV activities as sodium taurocholate cotransporting polypeptide (NTCP) inhibitors. DBADs exhibited stron..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="April 28, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.1c00078" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00078" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.1c00078" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00078" /></link>
        
    
    

<title>Design of Dimeric Bile Acid Derivatives as Potent and Selective Human NTCP Inhibitors | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.1c00078" /></meta><meta property="og:title" content="Design of Dimeric Bile Acid Derivatives as Potent and Selective Human NTCP Inhibitors" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0058.jpeg" /></meta><meta property="og:description" content="Dimeric bile acid derivatives (DBADs) were developed and tested for their anti-HBV and anti-HDV activities as sodium taurocholate cotransporting polypeptide (NTCP) inhibitors. DBADs exhibited strong and persistent potency of NTCP inhibition, whereas diverse linkers and constitutions showed distinct inhibition features. Motif aa157–165 on NTCP was shown to be a possible binding site of DBADs; therefore, we determined DBADs’ selectivity among NTCPs from different species. A cyclized DBAD scaffold DBA-41 exhibited a high affinity to human NTCP (hNTCP). Intraperitoneal administration of DBA-41 to hNTCP-tg mice induced serum total bile acid elevation. DBA-41 may serve as a biological tool to study NTCP physiological function." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.1c00078"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00078">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.1c00078&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.1c00078&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.1c00078&amp;href=/doi/10.1021/acs.jmedchem.1c00078" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 9</span><span class="cit-fg-pageRange">, 5973-6007</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/9" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.1c00076" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.1c00085" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Design of Dimeric Bile Acid Derivatives as Potent and Selective Human NTCP Inhibitors</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Yang Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yang Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Institute of Biological Sciences, Beijing 102206, China</div><div class="loa-info-affiliations-info">Graduate Program, Tsinghua University, Beijing 100084, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yang++Liu">Yang Liu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lei Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lei Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Institute of Biological Sciences, Beijing 102206, China</div><div class="loa-info-affiliations-info">College of Life Sciences, Beijing Normal University, Beijing 100875, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lei++Zhang">Lei Zhang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Huan Yan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Huan Yan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Institute of Biological Sciences, Beijing 102206, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Huan++Yan">Huan Yan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Zhiqiang Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zhiqiang Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Institute of Biological Sciences, Beijing 102206, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zhiqiang++Wang">Zhiqiang Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Guoliang Sun</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Guoliang Sun</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Institute of Biological Sciences, Beijing 102206, China</div><div class="loa-info-affiliations-info">Graduate Program, Peking University, Beijing 100080, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Guoliang++Sun">Guoliang Sun</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xiao Song</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiao Song</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Institute of Biological Sciences, Beijing 102206, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiao++Song">Xiao Song</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Zhongmin Zhou</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zhongmin Zhou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Institute of Biological Sciences, Beijing 102206, China</div><div class="loa-info-affiliations-info">College of Life Sciences, Beijing Normal University, Beijing 100875, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zhongmin++Zhou">Zhongmin Zhou</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Bo Peng</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bo Peng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Institute of Biological Sciences, Beijing 102206, China</div><div class="loa-info-affiliations-info">Graduate Program, Peking University, Beijing 100080, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bo++Peng">Bo Peng</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Liwei Yan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Liwei Yan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Institute of Biological Sciences, Beijing 102206, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Liwei++Yan">Liwei Yan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Qingcui Wu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Qingcui Wu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Institute of Biological Sciences, Beijing 102206, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Qingcui++Wu">Qingcui Wu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Wenhui Li</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wenhui Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Institute of Biological Sciences, Beijing 102206, China</div><div class="loa-info-affiliations-info">Tsinghua Institute of Multidisciplinary Biomedical Research, Tsinghua University, Beijing 100084, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#6e0207190b00061b072e00070c1d400f0d400d00"><span class="__cf_email__" data-cfemail="127e7b65777c7a677b527c7b70613c73713c717c">[email protected]</span></a>. Phone: +86-10-80726688-8580.</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wenhui++Li">Wenhui Li</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Xiangbing Qi</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiangbing Qi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Institute of Biological Sciences, Beijing 102206, China</div><div class="loa-info-affiliations-info">Tsinghua Institute of Multidisciplinary Biomedical Research, Tsinghua University, Beijing 100084, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#384951405159565f5a51565f7856515a4b16595b165b56"><span class="__cf_email__" data-cfemail="d4a5bdacbdb5bab3b6bdbab394babdb6a7fab5b7fab7ba">[email protected]</span></a>. Phone: +86-10-80726688-8655.</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiangbing++Qi">Xiangbing Qi</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7139-5164" title="Orcid link">http://orcid.org/0000-0002-7139-5164</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00078&amp;href=/doi/10.1021%2Facs.jmedchem.1c00078" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 9</span><span class="cit-pageRange">, 5973–6007</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">April 27, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>15 January 2021</li><li><span class="item_label"><b>Published</b> online</span>28 April 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 13 May 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.1c00078" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00078</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D5973%26pageCount%3D35%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DYang%2BLiu%252C%2BLei%2BZhang%252C%2BHuan%2BYan%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D9%26contentID%3Dacs.jmedchem.1c00078%26title%3DDesign%2Bof%2BDimeric%2BBile%2BAcid%2BDerivatives%2Bas%2BPotent%2Band%2BSelective%2BHuman%2BNTCP%2BInhibitors%26numPages%3D35%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D6007%26publicationDate%3DMay%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.1c00078"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">797</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.1c00078" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Design of Dimeric Bile Acid Derivatives as Potent and Selective Human NTCP Inhibitors&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Yang&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Lei&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Huan&quot;,&quot;last_name&quot;:&quot;Yan&quot;},{&quot;first_name&quot;:&quot;Zhiqiang&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Guoliang&quot;,&quot;last_name&quot;:&quot;Sun&quot;},{&quot;first_name&quot;:&quot;Xiao&quot;,&quot;last_name&quot;:&quot;Song&quot;},{&quot;first_name&quot;:&quot;Zhongmin&quot;,&quot;last_name&quot;:&quot;Zhou&quot;},{&quot;first_name&quot;:&quot;Bo&quot;,&quot;last_name&quot;:&quot;Peng&quot;},{&quot;first_name&quot;:&quot;Liwei&quot;,&quot;last_name&quot;:&quot;Yan&quot;},{&quot;first_name&quot;:&quot;Qingcui&quot;,&quot;last_name&quot;:&quot;Wu&quot;},{&quot;first_name&quot;:&quot;Wenhui&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Xiangbing&quot;,&quot;last_name&quot;:&quot;Qi&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;04&quot;,&quot;day&quot;:&quot;28&quot;,&quot;issue&quot;:&quot;9&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;5973-6007&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.1c00078&quot;},&quot;abstract&quot;:&quot;Dimeric bile acid derivatives (DBADs) were developed and tested for their anti-HBV and anti-HDV activities as sodium taurocholate cotransporting polypeptide (NTCP) inhibitors. DBADs exhibited strong and persistent potency of NTCP inhibition, whereas diverse linkers and constitutions showed distinct inhibition features. Motif aa157–165 on NTCP was shown to be a possible binding site of DBADs; therefore, we determined DBADs’ selectivity among NTCPs from different species. A cyclized DBAD scaffold DBA-41 exhibited a high affinity to human NTCP (hNTCP). Intraperitoneal administration of DBA-41 to hNTCP-tg mice induced serum total bile acid elevation. DBA-41 may serve as a biological tool to study NTCP physiological function.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00078&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00078" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00078&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00078" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00078&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00078" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00078&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00078&amp;href=/doi/10.1021/acs.jmedchem.1c00078" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.1c00078" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.1c00078" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (7 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00078&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.1c00078%26sid%3Dliteratum%253Aachs%26pmid%3D33906348%26genre%3Darticle%26aulast%3DLiu%26date%3D2021%26atitle%3DDesign%2Bof%2BDimeric%2BBile%2BAcid%2BDerivatives%2Bas%2BPotent%2Band%2BSelective%2BHuman%2BNTCP%2BInhibitors%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D9%26spage%3D5973%26epage%3D6007%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291430" title="Infectious diseases">Infectious diseases</a>,</li><li><a href="/action/doSearch?ConceptID=290758" title="Anatomy">Anatomy</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292464" title="Rodent models">Rodent models</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/9" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/jmcmar.2021.64.issue-9/20210513/jmcmar.2021.64.issue-9.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0058.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0058.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Dimeric bile acid derivatives (DBADs) were developed and tested for their anti-HBV and anti-HDV activities as sodium taurocholate cotransporting polypeptide (NTCP) inhibitors. DBADs exhibited strong and persistent potency of NTCP inhibition, whereas diverse linkers and constitutions showed distinct inhibition features. Motif aa157–165 on NTCP was shown to be a possible binding site of DBADs; therefore, we determined DBADs’ selectivity among NTCPs from different species. A cyclized DBAD scaffold DBA-41 exhibited a high affinity to human NTCP (hNTCP). Intraperitoneal administration of DBA-41 to hNTCP-tg mice induced serum total bile acid elevation. DBA-41 may serve as a biological tool to study NTCP physiological function.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82425" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82425" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Hepatitis B virus (HBV) infection remains a global public health problem. A global prevalence study indicated that about 290 million people are chronically infected.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> This chronic infection often leads to the development of cirrhosis and hepatocellular carcinoma (HCC).<a onclick="showRef(event, 'ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref2 ref3">(2,3)</a> Hepatitis D virus (HDV) is a satellite viroid that hijacks HBV envelope proteins for viral assembly and thus shares a similar early entry mechanism to HBV.<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4,5)</a> Sodium taurocholate cotransporting polypeptide (NTCP, encoded by the <i>SLC</i>10<i>A</i>1 gene) was identified to be a functional receptor for the entry of HBV and HDV into hepatocytes.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> It is a membrane protein predominantly expressed on the sinusoidal membrane of hepatocytes, contributing to the majority of the hepatic uptake of bile acids, sulfated steroids hormones, sulfated thyroidal hormones, and various drugs.<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7,8)</a> NTCP mRNA expression is low in human hepatocarcinoma cell lines; thus, HepG2 cells stably expressing human NTCP (HepG2-NTCP cells) offers a convenient and efficient HBV and HDV <i>in vitro</i> infection model.<a onclick="showRef(event, 'ref6 ref9'); return false;" href="javascript:void(0);" class="ref ref6 ref9">(6,9)</a></div><div class="NLM_p">We and other research groups have reported that viral entry of HBV/HDV and bile salt transport share common amino acid determinant motifs on NTCP; compounds inhibiting NTCP substrate transport could also inhibit HBV and HDV infection with varied efficiency from several to hundreds of micromolar level.<a onclick="showRef(event, 'ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref10 ref11">(10,11)</a> These compounds include NTCP substrates such as natural bile acids tauroursodeoxycholic acid (TUDCA) and chenodeoxycholic acid (CDCA);<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> steroidal compounds such as progesterone, 22(<i>S</i>)-hydroxycholesterol,<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> and proscillaridin A;<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> natural product vanitaracin A;<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> FDA-approved drugs such as Cyclosporine A (CsA),<a onclick="showRef(event, 'ref14 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref14 ref15 ref16">(14−16)</a> Ezetimibe,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Irbesartan, Ritonavir, and Bosentan; etc.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Previous studies from our group have reported that TUDCA showed decent anti-HBV activity among common bile acids with an IC<sub>50</sub> of around 3 μM.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> However, the low inhibitory potency of these small molecules restricts them to be used as drug candidates for HBV/HDV infection.</div><div class="NLM_p last">Despite that modulating NTCP activity has pronounced potential for controlling HBV/HDV infection and the associated diseases, to date, the only and first-in-class NTCP inhibitor is Myrcludex B, which is a polypeptide derived from HBV Pre-S1 peptide<a onclick="showRef(event, 'ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref19 ref20">(19,20)</a> and has been approved in Europe for the treatment of chronic hepatitis D. However, this compound is limited in intravenous or subcutaneous administration. In this study, we fortuitously discovered a series of dimeric bile acids (DBADs) as potent NTCP inhibitors and therefore designed and synthesized dozens of dimeric bile acid derivatives. We then conducted thorough structure–activity relationship (SAR) studies and evaluated their anti-HBV activities on HepG2-NTCP cells. This bivalent drug design with distinctive structure features has also been demonstrated by different inhibition properties and species selectivities. Encouragingly, we found that some DBADs exhibit strong and persistent inhibition potency with IC<sub>50</sub> lower than 50 nM. In addition, we found that motif aa157–165, a known motif for monkey-specific host range of HBV Pre-S1 binding on NTCP, was involved in DBAD binding. Finally, we developed a highly potent, highly affinitive to human NTCP compound DBA-41 and investigated its <i>in vivo</i> efficacy in human NTCP knock-in mice.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_54085" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_54085" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Identification of a Dimeric Bile Acid Derivative with Potent Anti-HBV Activity</h3><div class="NLM_p">In our previous studies, we have found that TUDCA exhibited strong anti-HBV activity.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Hoping to investigate TUDCA in further detail, we tried to synthesize TUDCA instead of using commercial TUDCA. However, when we test our in-house synthesized TUDCA (>99.5% purity) in anti-HBV assays and NTCP substrate uptake assays, we noticed a significant decrease in the activity of the synthesized TUDCA as compared to commercially sourced TUDCA (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A). We therefore examined the purity of the commercial TUDCA using liquid chromatography–mass spectroscopy (LC–MS with Waters 3100 Mass Detector) analysis: the commercial TUDCA contained at least two impurities (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B): one at a retention time of 5.90 min (around 10% based on the peak area with a molecular weight of 541.31 Daltons) and one at a retention time of 7.65 min (around 1% based on the peak area with a molecular weight of 873.58 Daltons).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0002.jpeg" id="gr44" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Discovery of potent NTCP inhibitors: DBADs. (A) HBV infection inhibition assay for commercial TUDCA (labeled as TUDCA-(com)) and in-house synthesized TUDCA (TUDCA-(syn)). HepG2-NTCP cells were inoculated with HBV in the presence of the indicated compounds for 24 h. After the culture medium was replenished with fresh medium, the cells were cultured for another 4 days. The HBeAg titers secreted into the medium were measured at 5 dpi using a commercial ELISA kit. (B) Liquid chromatography–mass spectroscopy (LC–MS) analysis of TUDCA-(com) and TUDCA-(syn). The retention time window between 4 min and 8 min is shown. The peaks indicated by arrows illustrate two impurities in TUDCA-(com). (C) HBV infection inhibition assay and NTCP substrate uptake assay for 3-OAc-TUDCA. The HBV infection assay was conducted as in (A). The substrate uptake assay was conducted with HepG2-NTCP cells using <sup>3</sup>H-TCA as the substrate. (D) HBV infection inhibition assay and NTCP substrate uptake assay for dimeric TUDCA. The HBV infection and substrate uptake assays were as in (C).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Having dug out these impurities, we then assumed that these impurities may contribute to the higher antiviral activity we had observed for commercial TUDCA. Guided by the LC–MS data, we presumably proposed two potential structures for these impurities and subsequently synthesized 3-acetylated TUDCA (3-OAc-TUDCA; MW, 541.31) and a dimeric form of UDCA (DBA-1; MW, 874.58) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B). These synthetic samples showed identical retention time and molecular weight with the impurities in commercial TUDCA (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00078/suppl_file/jm1c00078_si_001.pdf" class="ext-link">Supplement Figure 1</a>). We then tested the activities of both molecules using anti-HBV and NTCP substrate uptake assays. 3-OAc-TUDCA had decreased activities in both the anti-HBV and NTCP substrate uptake inhibition assays as compared to commercial TUDCA (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C). However, to our thrill, the dimeric UDCA showed remarkably enhanced activity in both anti-HBV and NTCP substrate uptake inhibition assays (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D). The potency we detected for this dimeric bile acid suggested a new strategy for developing NTCP inhibitors and anti-HBV agents.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Structure–Activity Relationship Analysis of Dimeric Bile Acid Derivatives</h3><div class="NLM_p">To explore the idea of dimeric bile acid design, we synthesized a number of dimeric bile acid derivatives (DBADs) by linking two or more bile acid moieties to generate molecules bearing a variety of structural features. In the nomenclature we here used for DBAD design, we defined the C-3 hydroxy group at the steroid part of a bile acid as the “tail” and the C-24 side chain as the “head”. Accordingly, most of our DBADs molecular structures can be classified as “head-to-head”, “head-to-tail”, or “tail-to-tail”. We then evaluated the antiviral activity of these synthesized DBADs using an <i>in vitro</i> HBV infection system based on HepG2-NTCP cells.</div><div class="NLM_p">The predicted structure from the impurities—a head-to-tail DBAD DBA-1—was synthesized by linking C-3 of a TUDCA subunit to C-24 of a UDCA by an ester linkage. DBA-1 exerted significant anti-HBV activity (IC<sub>50</sub> = 58 nM). This preliminary data encouraged us to investigate more polyvalent molecules with a variety of connection motifs to optimize the antiviral efficacy. We then designed several types of polymeric bile acid molecules, and the activities are examined and summarized in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. The head-to-head DBAD DBA-2 was synthesized by linking C-24 of two UDCA subunits through a bis-ester linkage; this compound had decreased antiviral activity (IC<sub>50</sub> = 2700 nM). Replacement of the ester linkage of DBA-2 with an amide linkage (DBA-3) resulted in a total loss of antiviral activity (IC<sub>50</sub> > 10 000 nM). Introducing a tauro-conjugation at the linker site of DBA-2 significantly increased the antiviral activity (DBA-4, IC<sub>50</sub> = 110 nM). In addition, DBA-5 with a tail-to-tail arrangement had antiviral activity consistent with DBA-1 (IC<sub>50</sub> = 65 nM). The decreased antiviral activity was observed for DBA-6 (formed by linking C-7 of TUDCA to C-24 of UDCA) and for “trimer” DBA-7 (head-to-tail linking of three UDCA subunits) (IC<sub>50</sub> = 1878 and 2653 nM, respectively). These results suggest that head-to-tail and tail-to-tail DBADs exert higher antiviral activities compared to other compositions; head-to-head DBADs appear to be less potent but still better than monomers, whereas tail-to-middle and trimeric bile acid derivatives had no significant antiviral activity.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. SAR of Different Constitutions of DBADs</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0050.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0051.gif" alt="" id="fx2" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> determined via HBV infection of HepG2-NTCP cells.</p></div></div><div></div></div><div class="NLM_p">Given the fact that head-to-tail and tail-to-tail constitutions have decent antiviral activity, hoping to improve the potency, we first examined the impact of various monomers in the DBAD composition (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Dimeric taurochenodeoxycholic acid (TCDCA) (DBA-8) and dimeric taurolithocholic acid (TLCA) (DBA-9) had decreased activity (IC<sub>50</sub> = 187 nM and 3244 nM, respectively) compared to DBA-5, implying that the presence of a TUDCA monomer is beneficial for antiviral activity. We further evaluated the contribution of taurine conjugation to the potency of DBADs. Using compound DBA-10 as a starting point, we observed decreased activity for DBADs with fewer conjugation groups, for example, the unconjugated form DBA-11 (IC<sub>50</sub> = 3571 nM), single taurine conjugation DBA-12 (IC<sub>50</sub> = 93 nM), and double taurine conjugation DBA-10 (IC<sub>50</sub> = 41 nM).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. SAR of the Different Monomers in Composition of DBADs</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0052.gif" alt="" id="fx3" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0053.gif" alt="" id="fx4" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> determined via HBV infection of HepG2-NTCP cells.</p></div></div><div></div></div><div class="NLM_p">We next examined the effect of different linkage types of these two scaffolds (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Using compound DBA-5 as a starting point, the bis-ester linker was replaced by a bis-amide linker to form DBA-10, which had slightly improved antiviral activity as compared with DBA-5 (IC<sub>50</sub> = 41 nM). Replacing the bis-ester with bis-ether (DBA-13), bis-alkyne (DBA-14), or bis-amine (DBA-15) linker significantly decreased the antiviral activity (IC<sub>50</sub> = 935, 1866, and 9651 nM, respectively). Ethylenediamine-linked compounds DBA-16, with linkage comprising only two carbons, had no apparent antiviral activity (IC<sub>50</sub> = 7683 nM), whereas compounds with two TUDCA monomers directly linked at the C-3 position (DBA-17, 18, and 19) had no antiviral activity. Besides investigating the linker type, we also examined the impact of linker length of bis-amide and bis-ester motifs on DBAD antiviral activity (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). The results indicate that the penta-carbon glutaric acid or glutaramide linkage is apparently more suitable for maintaining the antiviral activity of DBADs and also highlighted the impactful structural feature of DBADs (DBA-5 or DBA-19 vs DBA-23 or DBA-24).</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. SAR of the Linker Type of Tail-to-Tail DBADs</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0054.gif" alt="" id="fx5" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0055.gif" alt="" id="fx6" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> determined via HBV infection of HepG2-NTCP cells.</p></div></div><div></div></div><div class="NLM_p">To further diversify the connection motif and optimize the activity, we designed a more rigid heterocycle linkage with fewer rotational bonds. To our delight, a triazole-based linkage (DBA-20) significantly increased the activity (IC<sub>50</sub> = 12 nM), indicating that a rigid heterocyclic linkage of two bile acid monomers is favorable for activity. Consistent with this heterocyclic design, a triazole-based linkage DBA-21 and DBA-22 (IC<sub>50</sub> = 25 and 24 nM, respectively) also have enhanced activity compared to the head-to-tail ester type of DBAD DBA-1. In addition, due to the structural stability and inert metabolization, the triazole motif stands out as the preferred linkage design over amide and ester.</div><div class="NLM_p">Since the structural feature of all of these linear types of dimeric DBADs is bRO5 (beyond the rule of 5),<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> we then conducted more challenging SAR by optimizing the physicochemical properties for better druggability and bioavailability. We designed and synthesized a series of cyclic triazole-based DBADs by minimizing the number of free rotational bonds and flexibility to optimize the potential bioavailability according to the [AB-MPS = Abs(<i>c</i> log <i>D</i>-3) + NAR (#of aromatic rings) + NRB (#of rotatable bonds)] value of calculation.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Based on the calculation, the [AB-MPS] values of all of the cyclic dimers are below 14, which is much lower than the value of its corresponding linear counterpart. In addition, the diversity of these cyclic DBADs provided more opportunities for <i>in vivo</i> activity optimization. Based on the observations and results we have collected so far, we subsequently focused on the most potent DBADs representing each of the constitutions (DBA-1, 4, 5, 25 (an enantiomerically pure version of DBA-10), 20, and 29) for the follow-up studies. Their structures are illustrated in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0003.jpeg" id="gr45" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Chemical structures of selected DBAD candidates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. SAR of the Cyclic DBADs</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0056.gif" alt="" id="fx7" /></img><div></div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> DBADs Specifically Inhibit NTCP-Mediated Viral Entry and Substrate Uptake</h3><div class="NLM_p">The antiviral activities of the selected DBADs were systematically examined in assays with HepG2-NTCP cells. HepG2-NTCP cells treated with DBADs were significantly protected from HBV infection, as assessed based on the measurement of secreted HBeAg in the culture medium (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). Consistently, quantitative polymerase chain reaction (qPCR) showed significant reductions in intracellular HBV RNA levels upon treatment with DBADs (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B), and immunostaining supported DBAD-mediated significant reductions in the intracellular accumulation of HBcAg and HBsAg (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00078/suppl_file/jm1c00078_si_001.pdf" class="ext-link">Supplement Figure 2</a>). Further, DBADs showed excellent activity against HDV infection as indicated by decreased intranuclear accumulation of the HDV delta antigen (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C). These results collectively indicate that DBADs exert significant inhibition activity against HBV/HDV infection.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0004.jpeg" id="gr46" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Antiviral activity of DBADs. (A) HBV infection inhibition assay for DBADs in HepG2-NTCP cells. The treatment and HBeAg-detection protocol here were similar to those in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A. (B) DBADs decrease intracellular HBV RNA in HepG2-NTCP cells. RNA was extracted from infected HepG2-NTCP cells and assessed using qPCR. (C) HDV infection inhibition assay for DBADs in HepG2-NTCP cells. The cells were inoculated with HBV in the presence of the indicated compounds (5 μM concentration) and incubated for 24 h. HDV delta antigen was stained using a homemade FITC-labeled antidelta antigen antibody at 5 dpi and imaged via confocal microscopy (Zeiss Meta). (D) VSV infection inhibition assay for DBADs in HepG2-NTCP cells. The cells were inoculated with VSV in the presence of the indicated compounds (1 μM concentration) and incubated for 24 h. Secreted HBeAg and HBsAg titers were detected using the same kit as those in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A. (E) Time-course study of DBAD inhibition of HBV infection. The indicated compounds were added to HepG2-NTCP cells at different time points. Secreted HBeAg was detected using the methods detailed in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A. (F) HBV Pre-S1 binding for DBADs. The FITC-labeled HBV Pre-S1 polypeptide was incubated with HepG2-NTCP cells in the presence of the indicated compounds; images were captured via confocal microscopy (Zeiss Meta).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In contrast, no inhibitory effect of DBAD treatment was observed for infection of HepG2-NTCP cells by a vesicular stomatitis virus (VSV) pseudo-typed Lentivirus expressing recombinant HBeAg and HBsAg (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D), indicating that the reduced infection we observed is not associated with some artifacts of viral antigen expression or secretion and the inhibition may be specific to NTCP-mediated virus entry. Notably, a time-course analysis indicated that most of the tested DBADs potently inhibited HBV infection when added as preincubation treatment or at the point of viral incubation but did not inhibit HBV infection when added postinoculation (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>E). These results supported that DBADs act during the early viral entry steps by somehow inhibiting the susceptibility of host cells, rather than by impairing the infectivity of the virus or by reducing the expression of viral proteins.</div><div class="NLM_p">We next investigated the inhibition mechanism(s) of DBADs. Using fluorescein isothiocyanate (FITC)-labeled HBV Pre-S1 lipopeptide as a surrogate, we were able to investigate the interaction between the Pre-S1 domain and NTCP. Most of the tested DBADs significantly inhibited the FITC-Pre-S1 peptide binding on HepG2-NTCP cells (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>F). We further excluded nonspecific effects such as cytotoxicity or NTCP surface expression level: DBADs were noncytotoxic at as high as 10 μM on HepG2-NTCP cells (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00078/suppl_file/jm1c00078_si_001.pdf" class="ext-link">Supplement Figure 3</a>) and more than 100 μM on primary tupaia hepatocytes (PTHs) (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00078/suppl_file/jm1c00078_si_001.pdf" class="ext-link">Supplement Figure 3</a>). To explore the possibility that DBADs may affect NTCP localization, we examined NTCP surface expression on DBAD-treated HepG2-NTCP cells using immunostaining with a mAb known to recognize an extracellular epitope of NTCP. Taurolithocholic acid (TLCA), which is known to induce NTCP endocytosis,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> was employed here as a positive control. DBADs do not cause significant reductions in NTCP surface expression (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00078/suppl_file/jm1c00078_si_001.pdf" class="ext-link">Supplement Figure 4</a>). Collectively, these results indicate that DBADs inhibit HBV infection by interfering with the Pre-S1-NTCP interaction and exclude a reduction of NTCP accumulation on the cell surface in DBAD-mediated antiviral effects.</div><div class="NLM_p">Given that bile acids are natural substrates of NTCP, it is reasonably assumed that DBADs are potent NTCP inhibitors, we next examined whether DBADs may inhibit NTCP’s substrate transport function(s). <sup>3</sup>H labeled taurocholic acid (<sup>3</sup>H-TCA) and estrone sulfate (<sup>3</sup>H-Estrone sulfate) uptake inhibition assays showed that all of the tested DBADs significantly inhibited the uptake of these substrates. (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A–B), whereas uptake assays for ASBT and OATP1B1 showed that DBADs did not inhibit their uptake transporting functions, indicating high specificity of DBADs for human NTCP.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0005.jpeg" id="gr47" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. DBADs inhibit NTCP substrate uptake. (A) DBADs inhibit NTCP- and ASBT-mediated uptake of <sup>3</sup>H-TCA. HepG2 cells transfected with a plasmid expressing human NTCP or human-ASBT were used in substrate uptake assays with 2 μM of the indicated compounds using <sup>3</sup>H-TCA as a substrate. (B) DBADs inhibit NTCP and OATP1B1 uptake of <sup>3</sup>H-Estrone sulfate. HepG2 cells transfected with a plasmid expressing human NTCP or human-OATP1B1 were used in substrate uptake assays with 2 μM of the indicated compounds using <sup>3</sup>H-Estrone sulfate as a substrate. (C, D) Time course of DBAD inhibition of HBV infection and <sup>3</sup>H-TCA uptake in HepG2-NTCP cells. HepG2-NTCP cells were treated with 2 μM of the indicated DBADs at different time points. (C) DBADs inhibit HBV infection in HepG2-NTCP cells. Secreted HBeAg was measured using ELISA. (D) DBADs inhibit NTCP substrate uptake. <sup>3</sup>H-TCA was used as a substrate. (E) IC<sub>50</sub> determination of DBA-25 inhibiting substrate uptake of multiple NTCP homologues. HepG2 cells overexpressing indicated NTCP homologues were used in substrate uptake assays using <sup>3</sup>H-TCA as a substrate. (F) DBADs inhibit substrate uptake of multiple NTCP chimeras. HepG2 cells overexpressing indicated NTCP chimeras were used in substrate uptake assays using <sup>3</sup>H-TCA as a substrate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">We also designed and synthesized two fluorescent bile acids (FBAs) based on conjugation of DBD-F (4-(<i>N</i>,<i>N</i>-dimethylaminosulfonyl)-7-fluoro-2,1,3-benzoxadiazole) at the C3 (NQL-064) or C24 (NQL-065) of TCA to support dynamic visualization of NTCP transport in cells. Both of the FBAs were NTCP substrates and could be distinctly observed to accumulate in NTCP-expressing cells within 15 min; notably, NQL-064 was uptaken more efficiently than NQL-065 (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00078/suppl_file/jm1c00078_si_001.pdf" class="ext-link">Supplement Figure 5</a>). FBA uptake by HepG2-NTCP cells was blocked by DBADs (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00078/suppl_file/jm1c00078_si_001.pdf" class="ext-link">Supplement Figure 6</a>), and NTCP substrate uptake assays using <sup>3</sup>H-TCA as a substrate showed that all of the tested DBADs (except for DBA-4) significantly inhibited TCA uptake, with IC<sub>50</sub> values of around 14–36 nM, a level around 400 times lower than TUDCA (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00078/suppl_file/jm1c00078_si_001.pdf" class="ext-link">Supplement Figure 7</a>). Collectively, these results demonstrate that DBADs are potent and selective NTCP transport inhibitors.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Kinetics and Mechanism Study of DBADs Inhibiting NTCP-Mediated Viral Infection and Substrate Transport</h3><div class="NLM_p">Having determined the antiviral and NTCP substrate inhibition efficacies of the DBADs, we next investigated their inhibition kinetics. We treated HepG2-NTCP cells with DBADs at different time points for fixed 2 h, prior to challenging cells with HBV virus. Interestingly, whereas DBA-1 and DBA-4 showed significant inhibition of HBV infection regardless of the preincubation time, the DBADs with tail-to-tail constitutions (DBA-5, DBA-25, and DBA-20) had reduced or even no effect when they were added 2 or more hours prior to the infection assay (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C). In addition, an FITC-Pre-S1 peptide binding assay indicated that the persistent effect of DBA-1 and DBA-4 can be attributed to sustained interference of the Pre-S1-NTCP interaction (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00078/suppl_file/jm1c00078_si_001.pdf" class="ext-link">Supplement Figure 8</a>).</div><div class="NLM_p">We also examined the effects of DBA-1 and DBA-4 on substrate uptake with inhibition assays in HepG2-NTCP cells using <sup>3</sup>H-TCA as a substrate: both of these compounds could inhibit substrate uptake when examined as agents for a 1 h pretreatment (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D). An uptake assay examining a fluorescent bile acid revealed that DBA-1 and DBA-4 showed significant inhibition of NQL-064 uptake compared with the other DBADs (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00078/suppl_file/jm1c00078_si_001.pdf" class="ext-link">Supplement Figure 9</a>). These results indicated that head-to-tail (DBA-1) and head-to-head (DBA-4) configuration DBADs apparently may have relatively low constants of dissociation (<i>K</i><sub>off</sub>) that may allow them to remain in and block the NTCP binding site, whereas the tail-to-tail DBADs do not persistently inhibit the NTCP function.</div><div class="NLM_p">Previous studies have shown that genetic variations in the NTCP sequences from different species variously influence the susceptibility of each species to HBV. Two host range determinants on NTCP have been found to determine the HBV species specificity: motif aa84–87 for mouse NTCP and motif aa157–165 for monkey NTCP.<a onclick="showRef(event, 'ref6 ref23'); return false;" href="javascript:void(0);" class="ref ref6 ref23">(6,23)</a> To identify whether DBAD-mediated inhibition of NTCP is species-specific, we conducted substrate uptake assays in HepG2 cells expressing a variety of NTCP orthologues. Head-to-tail DBAD DBA-1 and head-to-head DBAD DBA-4 strongly inhibited all of the assayed NTCP orthologues. In contrast, the tail-to-tail DBADs DBA-25 and DBA-20 showed significant inhibition of <sup>3</sup>H-TCA uptake by human NTCP, but these compounds had sharply decreased inhibition effects on mouse, monkey, and tupaia NTCPs (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>E). Similar species-specific inhibition was also observed using the FBA NQL-064 as a substrate (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00078/suppl_file/jm1c00078_si_001.pdf" class="ext-link">Supplement Figure 10</a>). We therefore investigated DBA-25 in greater detail by replacing human NTCP residues 157–165 with the corresponding amino acids from monkey NTCP and observed a dramatic reduction in DBA-25’s potency (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>F).</div><div class="NLM_p">We further examined the inhibitory efficiency of DBADs on HepG2 cells overexpressing human NTCP, monkey NTCP, and NTCP chimeras with substitutions at aa157–165. The inhibition activity of tail-to-tail DBADs was strongly dependent on the specific sequence at aa157–165 swap. These compounds showed significant inhibition activity against NTCP with human residues, whereas weaker inhibition activity with monkey residues. We again found that the inhibition activity of DBADs with other constitutions was consistent for this set of NTCP variants (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>G). Thus, it is reasonable to assume that aa157–165 mediates the binding of tail-to-tail DBADs with NTCP. Given that aa157–165 also determines HBV Pre-S1 binding to NTCP, these HBV inhibition and substrate uptake assay results imply that HBV Pre-S1 and tail-to-tail DBADs may bind to a common binding motif on NTCP.</div><div id="sec2_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i12" class="anchor-spacer"></div><h4 class="article-section__title" id="_i12"> Pharmacological and <i>In Vivo</i> Studies of DBADs</h4><div class="NLM_p">Having investigated the molecular mode of action of DBADs <i>in vitro</i>, we next investigated their efficacy <i>in vivo</i>. Given that high activity coupled with a low constant of dissociation would be attractive for drug development, we chose DBA-1 as a starting point for further optimization. Briefly, DBA-26, in which the ester linker of DBA-1 was substituted with an amide linker, showed the best antiviral activity in our DBAD series (IC<sub>50</sub> = 5 nM). However, a liver microsome stability assay indicated that DBA-26 was degraded rapidly by liver enzymes (<i>T</i><sub>1/2</sub> = 0.7 min, CL<sub>int</sub> = 994.30 μL/min/mg protein).</div><div class="NLM_p">Inspired by the stable structure and decent activity from the 1,2,3-triazole linker on DBA-20, we tested DBA-22, which had a similar IC<sub>50</sub> value to DBA-1 (IC<sub>50</sub> = 24.5 nM), but was almost 20 times more stable than DBA-26 according to mouse liver microsome assays (<i>T</i><sub>1/2</sub> = 12.56 min, Cl<sub>int</sub> = 55.20 μL/min/mg protein). However, when we applied DBA-22 to mice, we did not detect any significant <i>in vivo</i> activities (data not shown). Investigating how DBA-22 lost its activity <i>in vivo</i>, substrate uptake inhibition assays using <sup>3</sup>H-TCA as the substrate on HepG2-NTCP cells indicated that DBA-22 totally lost its activity upon addition of 20% fetal bovine serum (FBS) to the cell growth medium (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B), suggesting DBA-22 likely interacts extensively with components in serum.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0006.jpeg" id="gr48" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. In vivo activity evaluation of DBADs. (A) Chemical structures of DBA-1, DBA-26, DBA-22, DBA-38, and DBA-41, with key chemical modifications, compared with the previous generation highlighted in yellow and red circles. (B–D) IC<sub>50</sub> values of DBA-22 (B), DBA-38 (C), and DBA-41 (D) on inhibiting NTCP uptake of <sup>3</sup>H-TCA in HepG2-NTCP cells with different concentrations of FBS. (E) DBA-41 inhibits human NTCP (left panel) and mouse NTCP (right panel) uptake of <sup>3</sup>H-TCA in HepG2 cells with no serum, human serum, mouse serum, and bovine serum. (F) Total bile acids level responses in WT/hNTCP-tg mice treated with DBA-41. WT-C57B/L mice and hNTCP-tg-C57B/L mice were intraperitoneally administrated vehicle or DBA-41 50 mg/kg B.I.D. for 3 days. Mice serum samples were collected at the start point and end point. The total bile acid level was determined at Peking Union Lawke Center for Functional Medicine.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We therefore rescreened our DBADs in the presence of 20% FBS using 3H-TCA uptake assays with HepG2-NTCP cells. Seeking potent NTCP inhibitor(s) exhibiting a persistent inhibition capacity, we treated the cells with DBADs for 2 h prior to conducting the uptake assays. To our surprise, unlike the performance in anti-HBV assays, cyclized DBADs have high potential in TCA uptake inhibition assays, while they retained more of their activity in the presence of FBS as compared to other DBADs (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>). The cyclized DBAD with triazole linker DBA-38 has an IC50 value of 17.9 nM for inhibition of substrate uptake. Although its IC50 values are 109.2 and 167.2 nM in the presence of 20% FBS and 50% FBS respectively, its inhibition activity is not absolutely abolished as with DBA-22 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C).</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. TCA Uptake Inhibition IC<sub>50</sub> of DBADs</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0057.gif" alt="" id="fx8" /></img><div></div></div><div class="NLM_p">Hoping to further improve the potency of DBADs in the presence of serum, we next examined structure–activity relationships by comparing similar cyclized structures DBA-35, DBA-38, and DBA-41. Given the IC<sub>50</sub> values of these compounds were similar without FBS in the assay environment, we found that their respective hydrophilic properties correlated well with their activities in the presence of FBS. That is, the compound lacking any conjugation on its linker site (DBA-35) showed the strongest decline in activity, whereas a compound with taurine conjugation on its linker site (DBA-41) retained much of its activity in FBS (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D). Time-course assays of NTCP uptake inhibition on HepG2-NTCP cells indicating that DBA-38 and DBA-41 persistently inhibited NTCP substrate uptake (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00078/suppl_file/jm1c00078_si_001.pdf" class="ext-link">Supplement Figure 11</a>). Thus, we chose DBA-41 as our primary candidate for further <i>in vivo</i> verification of DBADs.</div><div class="NLM_p">It is known that bile acids bind to serum components like albumin and lipoproteins<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> and the hydrophobicity of bile acids is the governing factor in this bile acid–albumin interaction.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Given the observation that hydrophobicity of our DBAD structures impacts their in-serum activity, considering the bile acid nature of DBADs, it is reasonably assumed that DBADs may have similar binding properties to bile acids. To assess the plasma protein binding ratio of DBADs, we used a rapid equilibrium device system (RED System, Thermo Fisher) to determine the protein binding percentage of DBA-41 in serum and found that DBA-41 has a greater than 99% protein binding percentage in both human and mouse serum, indicating that DBA-41 may bind readily with serum components but having a higher affinity for NTCP. We next investigated the affinity of DBA-41 by NTCP uptake inhibition assays using HepG2 cells transfected with plasmids expressing human NTCP or mouse NTCP in the presence of serum from different species. Interestingly, when DBA-41 was applied to inhibit human NTCP, serum from humans, mice, and bovine did not affect its efficacy. However, when this compound was applied to inhibit mouse NTCP, serum from humans and mice significantly decreased its efficacy (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>E). Collectively, the DBADs exhibit a plasma protein binding profile similar to natural bile acids, which are strongly binding to plasma protein components, while keeping the ability to interact with NTCP. In addition, we found that DBADs have a divergent affinity to NTCP orthologues and the cyclized DBADs DBA-38 and DBA-41 have a higher affinity to human NTCP than that to mouse NTCP.</div><div class="NLM_p">To evaluate the potential therapeutic effects of DBA-41, we tested the efficacy of intraperitoneally delivered DBA-41 in human NTCP knock-in C57B/L mice (hNTCP-tg mice).<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Having tested that liver microsomes assays showed that DBA-41 has decent stability in both mouse liver microsomes (<i>T</i><sub>1/2</sub> = 57.76 min, CL<sub>int</sub> = 12.0 μL/min/mg protein) and human liver microsomes (<i>T</i><sub>1/2</sub> = 113.6 min, CL<sub>int</sub> = 6.1 μL/min/mg protein). We then determined the pharmacokinetic profile of DBA-41 in C57B/L mice following intravenous injection, intraperitoneal injection, and oral gavage. DBA-41 exhibited systemic exposure, having a half-life of 1.6 h, with bioavailability at 32% upon intraperitoneal injection and at 1.3% upon oral intake.</div><div class="NLM_p last">We tested a 3 day B.I.D. dosing paradigm in which 50 mg/kg DBA-41 was injected intraperitoneally into hNTCP-tg or wild-type (WT) mice. Treatment with DBA-41 significantly increased the total bile acid level in hNTCP-tg mice but had no effect on WT mice (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>F). Importantly, no differences were observed in their alanine aminotransferase (ALT) levels or body weights during the treatment (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00078/suppl_file/jm1c00078_si_001.pdf" class="ext-link">Supplement Figure 12</a>). These results both demonstrated <i>in vivo</i> efficacy for DBA-41 in inhibiting NTCP substrate uptake and meanwhile exhibit decent safety, collectively implying that DBA-41 could be developed as a possible candidate for HBV entry inhibitor.</div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26233" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26233" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In this study, by chemically linking two bile acid monomers with a variety of linkers, we synthesized a series of dimeric bile acid derivatives (DBADs) and then examined their antiviral activities using HepG2-NTCP cell-based anti-HBV assays. Extensive SAR studies were also conducted to delineate that DBADs with head-to-tail, head-to-head, tail-to-tail, and cyclized constitutions have low micromolar potency. Common features including optional distances between the two steroidal cores, linker types, and hydrophilic properties also highly impact the efficacy. By performing a series of biological tests, we showed that DBADs efficiently inhibit HBV infection by interfering HBV Pre-S1-NTCP interaction without disturbing the membrane localization of NTCP. Kinetic studies of DBADs showed that head-to-head and head-to-tail DBADs have relatively low constants of dissociation and these DBADs can inhibit NTCP from multiple species. In contrast, tail-to-tail DBADs may inhibit NTCP-mediated bile acid uptake and Pre-S1 binding in a competitive way, and these compounds specifically inhibit human NTCP on a molecular base of residues 157–165. We ultimately obtained a highly potent compound DBA-41 with profound affinity to human NTCP and moderate shift in potency upon addition of 50% serum and showed its <i>in vivo</i> efficacy in mice models. Compared to the acceptable bioavailability (32%) upon intraperitoneal injection, the low oral bioavailability of DBA-41 (1.4%) after p.o. limits its clinical translation and the oral bioavailability optimization is imperative for further drug development and is currently ongoing within the group. Although the oral bioavailability of the lead compound DBA-41 is not ideal for an oral-drug candidate so far, the molecular cyclization and conformational rigidity design enforced herein resulted in superior physical properties and symbolized promising optimization direction. We believe that this type of hit-to-lead optimization strategy will inspire more multivalent drug discoveries not only for virus infection but also for other protein–protein interaction modulations.</div><div class="NLM_p">Current studies on HBV entry inhibitor development focused on NTCP substrate-derived inhibitors. These molecules, however, may have a high uptake rate by NTCP or other anion transporters and thus unable to sustain inhibition activity. To overcome this, we fortuitously discovered bivalent dimeric bile acid derivatives that have a relatively low constant of dissociation that allows persistent inhibition. Although currently there is no NTCP structure information available, we could reasonably assume based on the observations that this bivalent bile acid structure may interact with multiple binding sites on NTCP, yielding much stronger interaction and substantially improved binding affinity. This assumption is supported by the observation that the conformational rigid linkage of bivalent bile acids exhibits increased potency, and the distance between the two steroidal cores strongly impacts the activity of the DBADs.</div><div class="NLM_p">In addition, tail-to-tail DBADs specifically inhibit human NTCP on a molecular base of aa157–165. Since substitution of this motif alone could result in more than 100 reduction of potency, it is interesting that although residues in this motif have no significant influence on the uptake efficiency of TCA, this motif can be highlighted in both HBV Pre-S1 domain binding and DBAD binding, perhaps also involved in recognition with other NTCP substrates. It would be interesting to evaluate the contribution of this motif to the species-specific transport of NTCP during the evolutionary process.</div><div class="NLM_p">Our results also indicate that dimeric bile acid derivatives have a high plasma protein binding rate. However, cyclized DBADs retained the binding affinity to NTCP, similarly to natural bile acids in physiological conditions. We believe that NTCP here acts as an active transporter that mediates the binding and influx of these compounds into hepatocytes in spite of plasma protein binding, committing these compounds as exceptions to free drug theory. Thus, the binding affinity of bile acid derivatives to NTCP should be taken into consideration when determining the hepatic clearance and drug distribution in bile acid derivative drug discovery, and this binding affinity may vary among NTCP orthologues from different species.</div><div class="NLM_p last">To the best of our knowledge, our design of DBADs provides the first example of bivalent small-molecule inhibitors of NTCP, many with excellent potency and good selectivity. These DBAD compounds offer a new design strategy for drug development for inhibiting NTCP-mediated HBV viral entry and substrate transport, which also can be utilized as useful tools to delineate the molecular mechanisms of viral entry and substrate transport mediated by NTCP.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78484" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78484" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Ethical Statement</h3><div class="NLM_p last">All animals were housed in the animal facility of the National Institute of Biological Sciences (NIBS), Beijing. Animal experiments were conducted following the National Guidelines for Housing and Care of Laboratory Animals and performed in accordance with NIBS institutional regulations after approval by the institution’s Institutional Animal Care and Use Committee (IACUC). The protocol number is NIBS-0012.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> FITC-pre-S1 Peptide Binding Assay</h3><div class="NLM_p last">NTCP-expressing cells were cultured in the PTH maintenance medium (PMM) for 24 h before conducting this assay. For Immunofluorescence microscopy, cells were incubated with 400 nM FITC-pre-S1 peptide diluted in PMM at 37 °C for about 2–3 h. Subsequently, cells were washed once by plain medium and then directly visualized with a Fluorescence Microscope.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> HBV and HDV Infection Assay</h3><div class="NLM_p last">HepG2-NTCP Cells were cultured in PMM for 12–24 h before infection and then inoculated with HBV or HDV inoculum in the presence of 5% PEG8000 in PMM at 37 °C for about 24 h. Chemicals to be tested were added to the cells as indicated in the specific assays. The inoculum was replenished by PMM postinfection. The infections were detected on 5 day postinfection (dpi).</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> ELISA Assay for the Detection of HBeAg and HBsAg</h3><div class="NLM_p last">ELISA kits for HBeAg and HBsAg detection were from Wantai Pharm Inc. (Beijing, China). The supernatant from infected HepG2-NTCP cells was collected on 2–5 day postinfection. The secreted HBeAg and HBsAg in the culture medium were measured with a commercial kit from Wantai Pharm Inc. (Beijing, China) by following the manufacturer’s instructions.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Immunostaining Assay</h3><div class="NLM_p last">For HBV infection, infected cells were washed twice with phosphate-buffered saline (PBS) and fixed in 3.7% paraformaldehyde (PFA) at room temperature for 10 min. Subsequently, cells were permeabilized with 0.5% TritonX-100/PBS for 10 min at room temperature, blocked with 3% BSA at 37 °C for 1 h, and followed by incubation with 5 μg/mL mouse mAb1C10, which recognizes HBcAg, and by staining with FITC-conjugated secondary antibodies. Nuclei were stained with DAPI in blue. The stained cells were imaged with a fluorescence microscope (Nikon). For HDV infection, on 5 dpi, HDV-infected cells were fixed with 100% methanol at room temperature for 10 min, intracellular delta antigens were then stained with 5 μg/mL FITC-conjugated 4G5, and nuclei were stained with DAPI in blue. Images were collected by an Eclipse Ti Fluorescence Microscope (Nikon), and a representative picture was shown.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> [<sup>3</sup>H] Substrate Uptake Assay</h3><div class="NLM_p last">The [<sup>3</sup>H]-taurocholate uptake assay was conducted following a protocol as previously described. In general, HepG2-NTCP cells were cultured in PMM for 12–24 h and then treated with or without indicated chemicals before the uptake assay. For the substrate uptake assay, cells were generally incubated with 0.5 μL of (0.5 μCi) [<sup>3</sup>H]-taurocholate dissolved in Na<sup>+</sup> Ringer’s solutions for 10 min at 37 °C with or without the presence of chemicals. Subsequently, cells were washed once by PBS and lysed by 100 μL of 1% TritonX-100 in H<sub>2</sub>O for 5 min at room temperature. The lysate was transferred into a liquid scintillation tube and mixed with 900 μL of liquid scintillation cocktail (Ultima GOLD XR, PerkinElmer). Liquid scintillation counting was performed on a PerkinElmer 1450 LSC Liquid Scintillation Counter and Luminescence Counter.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Liver Microsome Stability Assay</h3><div class="NLM_p last">Commercial liver microsomes were diluted using fresh 0.05 M phosphate buffer (prepared by dissolving 8.7 g of K<sub>2</sub>HPO<sub>4</sub> in 1000 mL of H<sub>2</sub>O and then adjusting the pH to 7.4 using H<sub>3</sub>PO<sub>4</sub>) to a final concentration of 1 mg/mL. Compounds were mixed with the liver microsome at a final concentration of 1 μM. The mixture was then preincubated at 37 °C for 5 min. The enzyme activity was initiated by adding 1 mM NADPH. A small volume of the mixture (20 μL) was collected at different time points (0, 5, 15, 30, and 60 min) and quenched by adding 300 μL of methanol. The samples were vigorously vortexed for 1 min and then placed at −20 °C for 1 h. The samples were then centrifuged at 3500 rpm for 15 min, and the supernatant was collected for LC–MS analysis.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Pharmacological Kinetics Studies</h3><div class="NLM_p last">The pharmacological kinetics studies on DBA-041 were conducted using adult C57B/L mice. For each route (I.V., I.P., and P.O.), 12 mice were administrated and sampling in four groups (<i>n</i> = 3). The blood samples were collected at dosing time 5 min and 8 h (group 1), 15 min and 2 h (group 2), 30 min and 4 h (group 3), and 1 and 24 h (group 4) by eye orbital puncture and then eyeball pluck. The plasma samples were collected by centrifugation and extracted by methanol. The concentration of testing compounds in plasma was determined by LC–MS analysis.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Chemistry</h3><div class="NLM_p">All reactions were carried out under a nitrogen atmosphere in flame-dried glassware with magnetic stirring unless otherwise indicated. Commercially obtained reagents were used as received. Solvents were dried by passage through an activated alumina column under argon. Liquids and solutions were transferred via a syringe. All reactions were monitored by thin-layer chromatography with E. Merck silica gel 60 F254 precoated plates (0.25 mm). <sup>1</sup>H and <sup>13</sup>C NMR spectra were recorded on a Varian Inova-400 or -500 spectrometer. Data for <sup>1</sup>H NMR spectra are reported relative to CDCl<sub>3</sub> (7.26 ppm), CD<sub>3</sub>OD (3.31 ppm), or DMSO-<i>d</i><sub>6</sub> (2.50 ppm) as an internal standard and are reported as follows: chemical shift (δ ppm), multiplicity (s = singlet, d = doublet, t = triplet, q = quartet, sept = septet, m = multiplet, br = broad), coupling constant J (Hz), and integration. Data for 13C NMR spectra are reported relative to CDCl<sub>3</sub> (77.23 ppm), CD<sub>3</sub>OD (49.00 ppm), or DMSO-<i>d</i><sub>6</sub> (39.52 ppm) as an internal standard and are reported in terms of chemical shift (δ ppm). Sample preparation and purity analysis were conducted on a Waters high-performance liquid chromatograph (HPLC; Column, XBridge C18, 5 μm, 19 mm × 150 mm) with 2998PDA and 3100MS detectors and a Waters ultrahigh-pressure liquid chromatograph (UPLC; Column, BEH C18, 1.7 μm, 2.1 mm × 50 mm) with PDA and SQD MS detectors in electrospray ionization (ESI) mode.</div><div class="NLM_p">All new compounds were synthesized as indicated in detail in the following schemes. For compounds DBA-1–4 in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, the synthesis was a condensation step between OH or −NH<sub>2</sub> of TUDCA and carboxylic acid of ursodeoxycholic acid (UDCA); the double condensation reactions of two TUDCAs with glutaric acid give bis-ester DBA-5. The trimeric molecule DBA-7 was synthesized via a sequential intermolecular condensation of three TUDCA at 3- and 24-positions. The starting materials used for these reactions were commercially available or synthesized from known procedures, such as 24-amino-UDCA and 24-hydroxyl-UDCA were generated from 24-COOH of UDCA (references for the synthesis of 24-amine, 24-OH are necessary). Compounds DBA-8-12 in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> were synthesized according to the same synthetic route as that for compound DBA-5 using different bile acids as starting materials. Since compounds in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> were dimeric compounds with different types of linkers, the synthesis was complicated. Specifically, the DBA-13, 15, and 16 were synthesized using double nucleophilic substitutions of 3-hydroxyl- or 3-amino-TUDCAs. The DBA-23, 24 were synthesized according to the same synthetic route as that for compound DBA-5 using different bis-acids as linkers. The triazole-linked compounds 20, 21, and 22 were constructed through Cu-catalyzed click chemistry from Azido-TUDCA and alkyne-substituted TUDCA. The alkyne-substituted TUDCA monomer was also dimerized to generate bis-alkyne-linked DBA-14. Finally, the diol-linked compound DBA-17 was built via an interrupted McMurry coupling of the ketone substrate, which is oxidized from TUDCA.</div><div class="NLM_p">Cyclic compounds 27, 28, 29, 30, 31, 32, 33, and 34 were constructed through double condensation between the linear diamide TUDCA dimer and the corresponding diol or diamines. Cyclic compounds 35, 36, 37, 38, 39, 40, and 41 were synthesized from the condensation between the linear triazole TUDCA dimer and the corresponding diamines and followed by another intramolecular condensation to form the cyclization products (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>).</div><figure id="sch1" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0007.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of <b>DBA-1</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, UDCA (43.1 mg, 0.11 mmol, 1.0 equiv) was dissolved in <i>N</i>,<i>N</i>-dimethylformamide (DMF) (2.0 mL) at room temperature. Then, 2-aminoethanesulfonic acid (42.5 mg, 0.34 mmol, 3.0 equiv), <i>N</i>,<i>N-</i>diisopropylethylamine (DIEA) (58.8 mg, 0.46 mmol, 4.0 equiv), 4-dimethylaminopyridine (DMAP) (7 mg, 0.05 mmol, 0.5 equiv), and EDC (32.6 mg, 0.17 mmol, 1.5 equiv) was added. After the addition was complete, the mixture was stirred at room temperature for 14 h. The mixture was concentrated and purified by pre-HPLC (Anal: XBridge C18, H<sub>2</sub>O/ACN (10–90% in 10 min), retention time: 5.30 min) to get TUDCA as a white solid (11.1 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO): 7.65 (t, <i>J</i> = 5.2 Hz, 1H), 4.41 (d, <i>J</i> = 4.8 Hz, 1H), 3.86 (d, <i>J</i> = 7.2 Hz, 1H), 3.27 (dd, <i>J</i> = 13.9, 6.1 Hz, 4H), 2.08–1.56 (m, 9H), 1.56–0.93 (m, 18H), 0.93–0.81 (m, 6H), 0.61 (s, 3H).<sup>13</sup>C NMR (100 MHz, <i>d</i><sub>6</sub>-DMSO) δ 172.2, 69.8, 69.6, 56.0, 54.8, 50.8, 43.2, 43.1, 42.3, 37.8, 37.4, 35.6, 35.1, 35.0, 33.9, 32.7, 31.6, 30.4, 28.3, 26.8, 23.4, 21.0, 18.6, 12.2. LRMS (ESI) calcd for [M – H]<sup>−</sup> 498.3, found 498.5.</div><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, UDCA (23.5 mg, 0.06 mmol, 1.5 equiv) was dissolved in DMF (1 mL) at room temperature. Then, TUDCA (20 mg, 0.04 mmol, 1.0 equiv), DIEA (15.5 mg, 0.12 mmol, 3.0 equiv), DMAP (5 mg, 0.04 mmol, 1.0 equiv), and <i>N</i>,<i>N</i>′-dicyclohexylcarbodiimide (DCC, 16.5 mg, 0.08 mmol, 2.0 equiv) were added. After the addition was complete, the mixture was stirred at room temperature for 14 h. The mixture was concentrated and purified by pre-HPLC (Anal: XBridge C18, H<sub>2</sub>O (0.1% NH<sub>4</sub>OAC)/ACN (50–70% in 10 min), retention time: 7.65 min) to get the target product <b>DBA-1</b> as a white solid (5.0 mg, purity is >98%). <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO): δ 7.65 (t, <i>J</i> = 5.2 Hz, 1H), 4.54 (s, 1H), 4.43 (s, 1H), 3.88 (s, 2H), 3.28–3.24 (m, 4H), 2.31–1.98 (m, 8H), 1.92 (m, 3H), 1.77 (s, 3H), 1.77–1.54 (m, 10H), 1.20 (m, 28H), 0.95–0.79 (m, 12H), 0.62–0.61 (m, 6H). <sup>13</sup>C NMR (100 MHz, CD<sub>3</sub>OD) δ 176.5, 175.5, 75.2, 72.1, 71.9, 71.8, 57.5, 57.4, 56.6, 56.4, 51.5, 44.8, 44.5, 44.0, 43.8, 43.7, 41.5, 40.7, 40.6, 38.6, 38.4, 38.0, 36.7, 36.6, 36.4, 35.7, 35.2, 34.3, 34.2, 33.2, 32.6, 32.3, 31.0, 29.7, 29.6, 27.9, 27.6, 24.0, 23.9, 22.4, 19.1, 18.9, 12.8, 12.7, 12.6. LRMS (ESI) calcd for [M – H]<sup>−</sup> 873.3, found 873.0 (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>).</div><figure id="sch2" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0008.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of <b>DBA-2</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, UDCA (39.2 mg, 0.1 mmol, 1.0 equiv) was dissolved in Tol (2 mL) at room temperature. Then, (3<i>R</i>,7<i>S</i>,9<i>S</i>,10<i>S</i>,13<i>R</i>,14<i>S</i>,17<i>R</i>)-17-((<i>R</i>)-5-hydroxypentan-2-yl)-10,13-dimethylhexadecahydro-1<i>H</i>-cyclopenta[α]phenanthrene-3,7-diol (80.0 mg, 0.2 mmol, 2.0 equiv) and TsOH (1.0 mg, cat) were added. After the addition was complete, the mixture was stirred at 120 °C for 2 h. The mixture was quenched by the addition of H<sub>2</sub>O and extracted with dichloromethane (DCM). The organic layers were concentrated under reduced pressure and purified by silica column chromatography (PE:EA = 1:4) to get the target product <b>DBA-2</b> as a white solid (8.2 mg) which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO): δ 4.43 (d, <i>J</i> = 4.5 Hz, 2H), 3.97 (t, <i>J</i> = 6.3 Hz, 2H), 3.86 (dd, <i>J</i> = 6.8, 2.4 Hz, 2H), 3.30–3.24 (m, 4H), 2.33–2.14 (m, 6H), 1.96–1.53 (m, 17H), 1.53–0.94 (m, 43H), 0.93–0.81 (m, 15H), 0.62–0.61 (m, 6H).<sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 174.3, 71.2, 71.1, 71.0, 64.6, 55.9, 55.8, 55.0, 54.9, 43.7, 43.6, 43.5, 43.2, 42.4, 40.1, 39.3, 37.3, 37.0, 35.4, 35.3, 35.0, 34.1, 32.0, 31.6, 31.3, 30.2, 28.8, 28.7, 26.9, 25.1, 23.4, 21.1, 18.6, 18.3, 12.1. LRMS (ESI) calcd for [M + H]<sup>+</sup> 753.6, found 753.8 (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>).</div><figure id="sch3" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0009.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of <b>DBA-3</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a 25 mL, one-necked, round-bottomed flask, PPh<sub>3</sub> (260 mg, 1.0 mmol, 2.0 equiv) was dissolved in anhydrous tetrahydrofuran (THF, 20 mL) at −15 °C and then diisopropyl azodicarboxylate (DIAD, 200 mg, 1.0 mmol, 2.0 equiv) was added dropwise under Ar. After 10 min, (3<i>R</i>,7<i>S</i>,9<i>S</i>,10<i>S</i>,13<i>R</i>,14<i>S</i>,17<i>R</i>)-17-((<i>R</i>)-5-hydroxypentan-2-yl)-10,13-dimethylhexadecahydro-1<i>H</i>-cyclopenta[α]phenanthrene-3,7-diol (189 mg, 0.5 mmol, 1.0 equiv) dissolved in THF (10 mL) was added dropwise. The mixture was stirred at room temperature for 30 min. Then, isoindoline-1,3-dione (147 mg, 1.0 mmol, 2.0 equiv) was added. After the addition was complete, the mixture was stirred at room temperature for 14 h. The mixture was quenched by the addition of conc. NaHCO<sub>3</sub> to pH > 8, extracted with EA, concentrated under reduced pressure, and purified by silica column chromatography (PE:EA = 1:1) to get DBA-3-A (280 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO): δ 7.86–7.78 (m, 4H), 4.45 (d, <i>J</i> = 4.3 Hz, 1H), 3.86 (d, <i>J</i> = 6.8 Hz, 1H), 3.32–3.21 (m, 2H), 1.96–1.54 (m, 8H), 1.53–0.96 (m, 20H), 0.86–0.83 (m, 6H), 0.60 (s, 3H).</div><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, DBA-3-A (50 mg, 0.1 mmol, 1.0 equiv) was dissolved in EtOH (5 mL) at room temperature and then hydrazine hydrate (25 mg, 0.5 mmol, 5.0 equiv) was added dropwise under Ar. After the addition was complete, the mixture was stirred at 90 °C for 14 h. The mixture was filtered and concentrated under reduced pressure to get a crude oil of DBA-3-B, which was used directly.</div><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, DBA-3-B (crude, 0.1 mmol, 1.0 equiv) was dissolved in dry DMF (2.0 mL) at room temperature. Then, UDCA (43.1 mg, 0.11 mmol, 1.1 equiv), HATU (41.8 mg, 0.11 mmol, 1.1 equiv) and DIEA (38.7 mg, 0.3 mmol, 3.0 equiv) were added at room temperature. After the addition was complete, the mixture was stirred at room temperature for 2 h. The mixture was concentrated and purified by silica column chromatography (DCM:MeOH = 40:1) to get the target product <b>DBA-3</b> as a white solid (3.0 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO): δ 7.56–7.54 (m, 1H), 4.45 (d, <i>J</i> = 4.3 Hz, 2H), 3.86 (d, <i>J</i> = 6.8 Hz, 2H), 3.10–3.03 (m, 4H), 1.96–1.54 (m, 24H), 1.53–0.96 (m, 25H), 0.94–0.85 (m, 12H), 0.62 (s, 6H) (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>).</div><figure id="sch4" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0010.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of <b>DBA-4</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a 25 mL, one-necked, round-bottomed flask, TUDCA (200.0 mg, 0.4 mmol, 1.0 equiv) was dissolved in THF (10 mL) at room temperature. Then, lithium aluminum hydride (LAH) (152.0 mg, 4.0 mmol, 10.0 equiv) was added in portions at 0 °C. After the addition was complete, the mixture was stirred at room temperature for 120 h. The mixture was concentrated and purified by pre-HPLC to get the target product DBA-4-A (26.8 mg) as a white solid, which was confirmed by UPLC. LRMS (ESI) calcd for [M – H]<sup>−</sup> 484.3, found 484.4.</div><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, DBA-4-A (24.2 mg, 0.05 mmol, 1.0 equiv) was dissolved in DMF (1 mL) at room temperature. Then, UDCA (23.5 mg, 0.06 mmol, 1.2 equiv), DIEA (19.3 mg, 0.15 mmol, 3.0 equiv), and HATU (26.6 mg, 0.07 mmol, 1.4 equiv) were added. After the addition was complete, the mixture was stirred at room temperature for 14 h. The mixture was concentrated and purified by pre-HPLC (Anal: XBridge C18, H<sub>2</sub>O (0.1% NH<sub>4</sub>OAC)/ACN (25–95% in 10 min), retention time: 5.48 min) to get the target product <b>DBA-4</b> as a white solid (6.8 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO): δ 4.45–4.38 (m, 2H), 3.89–3.85 (m, 2H), 3.50–3.36 (m, 4H), 3.31–3.27 (m, 2H), 3.17–3.14 (m, 2H), 2.67–2.57 (m, 2H), 2.33–2.02 (m, 5H), 1.97–1.60 (m, 14H), 1.46–1.10 (m, 35H), 0.95–0.84 (m, 12H), 0.63–0.61 (m, 6H). LRMS (ESI) calcd for [M – H]<sup>−</sup> 858.6, found 858.9 (<a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>).</div><figure id="sch5" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0011.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of <b>DBA-5</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, TUDCA (50.0 mg, 0.1 mmol, 2.0 equiv) was dissolved in DMF (1 mL) at room temperature. Then, glutaric acid (6.6 mg, 0.05 mmol, 1.0 equiv), DIEA (25.8 mg, 0.2 mmol, 4.0 equiv), DMAP (6.1 mg, 0.05 mmol, 1.0 equiv), and EDC (23.0 mg, 0.12 mmol, 2.4 equiv) were added. After the addition was complete, the mixture was stirred at room temperature for 14 h. The mixture was concentrated and used directly.</div><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, DBA-5-A (crude, 0.05 mmol, 1.0 equiv) was dissolved in DMF (1 mL) at room temperature. Then, TUDCA (25.0 mg, 0.05 mmol, 1.0 equiv), HATU (19.0 mg, 0.05 mmol, 1.0 equiv), and DIEA (25.8 mg, 0.2 mmol, 4.0 equiv) were added. After the addition was complete, the mixture was stirred at room temperature for 14 h. The mixture was concentrated and purified by pre-HPLC (Anal: XBridge C18, H<sub>2</sub>O (0.1% NH<sub>4</sub>OAC)/ACN (10–95% in 10 min), retention time: 7.39 min) to get the target product <b>DBA-5</b> as a white solid (9.3 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO): δ 7.66 (t, <i>J</i> = 5.4 Hz, 2H), 4.59–4.52 (m, 2H), 3.90 (d, <i>J</i> = 7.2 Hz, 2H), 3.31–3.21 (m, 8H), 2.28 (t, <i>J</i> = 7.3 Hz, 6H), 2.07–1.99 (m, 4H), 1.96–1.50 (m, 24H), 1.50–1.09 (m, 22H), 1.02 (s, 6H), 0.89 (d, <i>J</i> = 8.2 Hz, 12H), 0.61 (s, 6H). <sup>13</sup>C NMR (100 MHz, CD<sub>3</sub>OD) 176.5, 174.2, 75.4, 71.8, 57.4, 56.4, 51.5, 44.8, 44.5, 43.8, 41.5, 40.6, 38.4, 37.1, 36.9, 36.8, 36.6, 35.7, 35.2, 34.3, 34.2, 33.2, 29.7, 27.9, 27.6, 23.8, 22.4, 21.4, 19.1, 12.6. LRMS (ESI) calcd for [M – H]<sup>−</sup> 1094.4, found 1094.2 (<a class="ref internalNav" href="#sch6" aria-label="Scheme 6">Scheme 6</a>).</div><figure id="sch6" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0012.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of <b>DBA-6</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a 25 mL, one-necked, round-bottomed flask, UDCA (500.0 mg, 1.27 mmol, 1.0 equiv) was dissolved in MeCN (10 mL) at room temperature. Then, BnBr (1.09 g, 6.38 mmol, 5.0 equiv) was added dropwise under Ar. After the addition was complete, the mixture was stirred at 80 °C for 14 h. The mixture was quenched by the addition of MeOH and extracted with EA, concentrated under reduced pressure and purified by silica column chromatography (DCM:MeOH = 50:1) to get DBA-6-A (250.0 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO): δ 7.37–7.33 (m, 5H), 5.08 (d, <i>J</i> = 4.0 Hz, 2H), 4.43 (d, <i>J</i> = 4.8 Hz, 1H), 3.86 (d, <i>J</i> = 7.2 Hz, 1H), 3.32–3.21 (m, 2H), 2.43–3.21 (m, 2H), 1.96–1.54 (m, 6H), 1.53–0.90 (m, 17H), 0.89–0.86 (m, 6H), 0.58 (s, 3H).</div><div class="NLM_p">In a 25 mL, one-necked, round-bottomed flask, DBA-6-A (240.0 mg, 0.5 mmol, 1.0 equiv) was dissolved in DCM (10 mL) at room temperature. Then, pyridine (Py) (98 mg, 1.25 mmol, 2.5 equiv) was added, followed by adding CbzCl (213 mg, 1.25 mmol, 2.5 equiv) dropwise under Ar. The mixture was stirred at room temperature for 1 h. Py (59 mg, 0.75 mmol, 1.5 equiv) and CbzCl (127 mg, 0.75 mmol, 1.5 equiv) were added. After the addition was complete, the mixture was stirred at room temperature for 2 h. The mixture was quenched by the addition of 1 M HCl, extracted with DCM, concentrated under reduced pressure, and purified by silica column chromatography (DCM:MeOH = 50:1) to get DBA-6-B (62.0 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO): δ 7.40–7.30 (m, 10H), 5.11 (s, 2H), 5.07 (d, <i>J</i> = 4.0 Hz, 2H), 3.86 (d, <i>J</i> = 7.2 Hz, 1H), 3.32–3.21 (m, 2H), 2.43–3.21 (m, 2H), 1.96–1.54 (m, 6H), 1.53–0.90 (m, 17H), 0.89–0.86 (m, 6H), 0.58 (s, 3H).</div><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, DBA-6-B (61 mg, 0.1 mmol, 1.0 equiv) was dissolved in DMF (1 mL) at room temperature. Then, UDCA (392 mg, 1.0 mmol, 10 equiv), DIEA (129 mg, 3.0 mmol, 30 equiv), DMAP (122 mg, 1.0 mmol, 10 equiv), and EDC (230 mg, 1.2 mmol, 12 equiv) was added. After the addition was complete, the mixture was stirred at room temperature for 14 h. The mixture was concentrated and used directly.</div><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, DBA-6-C (crude, 0.04 mmol, 1.0 equiv) was dissolved in THF (0.5 mL) and EtOH (0.5 mL) at room temperature. Then, HCOOH (three drops) and Pd/C (10 mg) were added. The mixture was bubbled with H<sub>2</sub> for 5 min and stirred at room temperature for 2 h. The mixture was filtered and concentrated to get crude product, which was used directly.</div><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, DBA-6-D (30.0 mg, 0.04 mmol, 1.0 equiv) was dissolved in DMF (1 mL) at room temperature. Then, 2-aminoethanesulfonic acid (20 mg, 0.16 mmol, 4.0 equiv), DIEA (25.8 mg, 0.2 mmol, 5.0 equiv), DMAP (5 mg, 0.04 mmol, 1.0 equiv), and EDC (9.2 mg, 0.05 mmol, 1.2 equiv) were added. After the addition was complete, the mixture was stirred at room temperature for 14 h. The mixture was concentrated and purified by pre-HPLC (Anal: XBridge C18, H<sub>2</sub>O (0.1% NH<sub>4</sub>OAC)/ACN (45–95% in 10 min), retention time: 6.35 min) to get the target product <b>DBA-6</b> as a white solid (1.4 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 3.62–3.58 (m, 2H), 3.53–3.45 (m, 4H), 2.99–2.96 (m, 2H), 2.37–1.98 (m, 15H), 1.98–1.02 (m, 61H), 1.02–0.87 (m, 13H), 0.72–0.71 (m, 6H). LRMS (ESI) calcd for [M – H]<sup>−</sup> 873.3, found 873.2 (<a class="ref internalNav" href="#sch7" aria-label="Scheme 7">Scheme 7</a>).</div><figure id="sch7" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0013.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Synthesis of <b>DBA-7</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, DBA-1 (44.0 mg, 0.05 mmol, 1.0 equiv) was dissolved in DMF (1 mL) at room temperature. Then, UDCA (197 mg, 0.5 mmol, 10 equiv), DIEA (193.5 mg, 1.5 mmol, 30 equiv), DMAP (61 mg, 0.5 mmol, 10 equiv), and EDC (115 mg, 0.6 mmol, 1.2 equiv) were added. After the addition was complete, the mixture was stirred at room temperature for 14 h. The mixture was concentrated and purified by pre-HPLC (Anal: XBridge C18, H<sub>2</sub>O (0.1% NH<sub>4</sub>OAC)/ACN (80–100% in 10 min), retention time: 7.25 min) to get the target product <b>DBA-7</b> as a white solid (6.1 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 7.89 (s, 1H), 4.67–4.62 (m, 2H), 3.58 (t, <i>J</i> = 5.8 Hz, 2H), 3.52–3.43 (m, 4H), 2.96 (t, <i>J</i> = 6.8 Hz, 2H), 2.39–1.99 (m, 12H), 1.99–1.05 (m, 65H), 1.05–0.90 (m, 18H), 0.72 (s, 9H). <sup>13</sup>C NMR (100 MHz, CD<sub>3</sub>OD) δ 176.5, 175.5, 175.4, 75.2, 72.1, 71.9, 71.8, 57.5, 57.4, 56.5, 56.4, 51.5, 44.9, 44.8, 44.8, 44.5, 44.1, 44.0, 43.8, 41.6, 40.7, 38.6, 38.4, 38.0, 36.9, 36.6, 36.4, 36.3, 35.7, 35.3, 35.2, 34.4, 34.3, 33.3, 32.7, 32.4, 31.1, 29.8, 29.7, 27.9, 27.6, 24.2, 23.8, 22.4, 19.1, 19.0, 12.9, 12.8, 12.7. LRMS (ESI) calcd for [M – H]<sup>−</sup> 1247.4, found 1247.2 (<a class="ref internalNav" href="#sch8" aria-label="Scheme 8">Scheme 8</a>).</div><figure id="sch8" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0014.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 8. Synthesis of <b>DBA-8</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a 10 mL, one-necked, round-bottom flask, chenodeoxycholic acid (CDCA) (43.1 mg, 0.11 mmol, 1.0 equiv) was dissolved in DMF (2.0 mL) at room temperature. Then, 2-aminoethanesulfonic acid (42.5 mg, 0.34 mmol, 3.0 equiv), DIEA (58.8 mg, 0.46 mmol, 4.0 equiv), DMAP (7 mg, 0.05 mmol, 0.5 equiv), and EDC (32.6 mg, 0.17 mmol, 1.5 equiv) were added. After the addition was complete, the mixture was stirred at room temperature for 14 h. The mixture was concentrated and purified by pre-HPLC to get taurochenodeoxycholic acid (TCDCA) as a white solid (11.1 mg), which was confirmed by UPLC.</div><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, taurochenodeoxycholic acid (TCDCA) (100 mg, 0.2 mmol, 1.0 equiv) was dissolved in DMF (1 mL) at room temperature. Then, glutaric acid (26.4 mg, 0.2 mmol, 1.0 equiv), DIEA (103.2 mg, 0.8 mmol, 4.0 equiv), DMAP (48.8 mg, 0.4 mmol, 2.0 equiv), and EDC (92.0 mg, 0.48 mmol, 2.4 equiv) were added. After the addition was complete, the mixture was stirred at room temperature for 48 h. The mixture was concentrated and purified by pre-HPLC (Anal: XBridge C18, H<sub>2</sub>O (0.1% NH<sub>4</sub>OAC)/ACN (20–75% in 10 min), retention time: 6.69 min) to get the target product DBA-8 as a white solid (5.4 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO): δ 7.68 (t, <i>J</i> = 5.5 Hz, 2H), 4.48–4.43 (m, 2H), 4.20 (d, <i>J</i> = 3.4 Hz, 2H), 3.63 (s, 2H), 3.47 (t, <i>J</i> = 6.8 Hz, 2H), 3.26 (d, <i>J</i> = 11.2 Hz, 4H), 3.02–2.98 (m, 4H), 2.38–2.25 (m, 10H), 2.12– 0.94 (m, 48H), 0.88–0.86 (m, 12H), 0.60 (s, 6H). LRMS (ESI) calcd for [M/2 – H]<sup>−</sup> 546.3, found 546.5 (<a class="ref internalNav" href="#sch9" aria-label="Scheme 9">Scheme 9</a>).</div><figure id="sch9" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0015.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 9. Synthesis of <b>DBA-9</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, lithocholic acid (LCA) (414 mg, 1.1 mmol, 1.0 equiv) was dissolved in DMF (5.0 mL) at room temperature. Then, 2-aminoethanesulfonic acid (425.0 mg, 3.4 mmol, 3.0 equiv), DIEA (593.4 mg, 4.6 mmol, 4.0 equiv), DMAP (61.0 mg, 0.5 mmol, 0.5 equiv), and EDC (334.9 mg, 1.7 mmol, 1.5 equiv) were added. After the addition was complete, the mixture was stirred at room temperature for 14 h. The mixture was concentrated and purified by pre-HPLC to get taurolithocholic acid (TLCA) as a white solid (111 mg), which was confirmed by UPLC.</div><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, taurolithocholic acid (TLCA) (96.7 mg, 0.2 mmol, 1.0 equiv) was dissolved in DMF (1 mL) at room temperature. Then, glutaric acid (26.4 mg, 0.2 mmol, 1.0 equiv), DIEA (103.2 mg, 0.8 mmol, 4.0 equiv), DMAP (48.8 mg, 0.4 mmol, 2.0 equiv), and EDC (92.0 mg, 0.48 mmol, 2.4 equiv) were added. After the addition was complete, the mixture was stirred at room temperature for 48 h. The mixture was concentrated and purified by pre-HPLC (Anal: XBridge C18, H<sub>2</sub>O (0.1% NH<sub>4</sub>OAC)/ACN (40–80% in 10 min), retention time: 6.75 min) to get the target product DBA-9 as a white solid (1.0 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 4.77–4.65 (m, 2H), 3.58 (t, <i>J</i> = 6.9 Hz, 4H), 2.96 (t, <i>J</i> = 6.9 Hz, 4H), 2.35 (t, <i>J</i> = 7.3 Hz, 4H), 2.32–1.83 (m, 24H), 1.83–1.03 (m, 34H), 1.03–0.91 (m, 12H), 0.69 (s, 6H). LRMS (ESI) calcd for [M – H]<sup>−</sup> 1061.5, found 1061.8 (<a class="ref internalNav" href="#sch10" aria-label="Scheme 10">Scheme 10</a>).</div><figure id="sch10" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Scheme 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0016.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 10. Synthesis of <b>DBA-10</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, DBA-11 (30.0 mg, 0.037 mmol, 1.0 equiv) was dissolved in dry DMF (2.0 mL) at room temperature. Then, 2-aminoethane-1-sulfonic acid (37 mg, 0.296 mmol, 8.0 equiv), HATU (33.4 mg, 0.089 mmol, 2.4 equiv), and DIEA (14.3 mg, 0.111 mmol, 3.0 equiv) were added at room temperature. After the addition was complete, the mixture was stirred at room temperature for 14 h. The mixture was concentrated and purified by pre-HPLC (Anal: XBridge C18, H<sub>2</sub>O (0.1% NH<sub>4</sub>OAC)/ACN (15–60% in 10 min), retention time: 7.45 min) to get the target product <b>DBA-10</b> as a white solid (7.3 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 4.00 (s, 2H), 3.57–3.54 (m, 6H), 3.48–3.39 (m, 2H), 2.93 (d, <i>J</i> = 7.3, 4H), 2.29–1.95 (m, 15H), 1.95–1.64 (m, 16H), 1.63–1.02 (m, 41H), 1.02–0.91 (m, 12H), 0.69 (s, 6H). LRMS (ESI) calcd for [M – H]<sup>−</sup> 1092.5, found 1092.2 (<a class="ref internalNav" href="#sch11" aria-label="Scheme 11">Scheme 11</a>).</div><figure id="sch11" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Scheme 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0017.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 11. Synthesis of <b>DBA-11</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a 250 mL, one-necked, round-bottomed flask, methyl(4<i>R</i>)-4-((3<i>R</i>,7<i>S</i>,10<i>S</i>,13<i>R</i>,17<i>R</i>)-3,7-dihydroxy-10,13-dimethylhexadecahydro-1<i>H</i>-cyclopenta[α]phenanthren-17-yl)pentanoate (4.06 g, 10 mmol, 1.0 equiv) was dissolved in dry DCM (100.0 mL) at room temperature. Then, silica gel (5.0 g) and PCC (2.37 g, 11.0 mmol, 1.1 equiv) was added in portions at 0 °C under Ar. After the addition was complete, the mixture was stirred at 0 °C for 30 min. The mixture was filtered and concentrated to get a crude oil, which was purified by silica chromatography (DCM:MeOH = 40:1) to get DBA-11-A as a white solid (610.0 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 3.94 (d, <i>J</i> = 6.8 Hz, 1H), 3.57 (s, 3H), 3.32–3.21 (m, 1H), 2.41–3.20 (m, 2H), 1.96–1.54 (m, 6H), 1.53–0.90 (m, 18H), 0.95 (s, 3H), 0.87 (d, <i>J</i> = 6.4 Hz, 3H), 0.64 (s, 3H).</div><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, DBA-11-A (202 mg, 0.5 mmol, 1.0 equiv) was dissolved in dry Py (2.0 mL) at room temperature. Then, NH<sub>2</sub>OH·HCl (139 mg, 2.0 mmol, 4.0 equiv) was added in portions at room temperature. After the addition was complete, the mixture was stirred at 100 °C for 2 h. The mixture was acidified with 1 M HCl to pH < 6, extracted with DCM, concentrated to get a crude product DBA-11-B, which was used directly. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 3.89 (d, <i>J</i> = 6.8 Hz, 1H), 3.54 (s, 3H), 2.89–2.82 (m, 1H), 2.38–3.14 (m, 3H), 1.96–1.54 (m, 8H), 1.53–0.90 (m, 15H), 0.98–0.82 (m, 6H), 0.63 (s, 3H).</div><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, DBA-11-B (85.0 mg, 0.2 mmol, 1.0 equiv) was dissolved in dry MeOH (2.0 mL) at room temperature. Then, HCOONH<sub>4</sub> (126 mg, 2.0 mmol, 10.0 equiv) and Pd/C (8.5 mg, 10 wt %) were added in portions at room temperature. After the addition was complete, the mixture was stirred at 90 °C for 2 h under H<sub>2</sub>. The mixture was filtered and concentrated to get a crude product DBA-11-C, which was used directly.</div><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, DBA-11-C (crude, 0.1 mmol, 1.0 equiv) was dissolved in dry DMF (2.0 mL) at room temperature. Then, glutaric acid (13.2 mg, 0.1 mmol, 1.0 equiv), HATU (114 mg, 0.3 mmol, 3.0 equiv), and DIEA (38.7 mg, 0.3 mmol, 3.0 equiv) were added at room temperature. After the addition was complete, the mixture was stirred at room temperature for 14 h. The mixture was concentrated and dissolved in THF (2.0 mL), and then, LiOH·H<sub>2</sub>O (21.0 mg, 0.5 mmol, 4.0 equiv) in H<sub>2</sub>O (2.0 mL) was added. The mixture was stirred at room temperature for another 2 h. The mixture was concentrated and purified by pre-HPLC (Anal: XBridge C18, H<sub>2</sub>O (0.1% NH<sub>4</sub>OAC)/ACN (15–60% in 12 min), retention time: 7.22 min) to get the target product <b>DBA-11</b> as a white solid (6.1 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 4.03 (br, 2H), 3.49–3.41 (m, 2H), 2.37–2.13 (m, 9H), 2.13–2.00 (m, 6H), 1.89–1.83 (m, 14H), 1.75–0.93 (m, 45H), 0.72 (s, 6H). LRMS (ESI) calcd for [M – H]<sup>−</sup> 878.3, found 878.1 (<a class="ref internalNav" href="#sch12" aria-label="Scheme 12">Scheme 12</a>).</div><figure id="sch12" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Scheme 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0018.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 12. Synthesis of <b>DBA-12</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch12"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a 250 mL, three-necked, round-bottomed flask, PPh<sub>3</sub> (2.6 g, 10.0 mmol, 2.0 equiv) was dissolved in anhydrous THF (100 mL) at −15 °C and then DIAD (2.02 g, 10 mmol, 2.0 equiv) was added dropwise under Ar. After 10 min, UDCA-Ester (2.0 g, 5.0 mmol, 1.0 equiv) dissolved in THF (10 mL) was added dropwise. The mixture was stirred at room temperature for 30 min. Then, methanesulfonic acid (960 mg, 10.0 mmol, 2.0 equiv) was added. After the addition was complete, the mixture was stirred at room temperature for 14 h. The mixture was quenched by the addition of conc. NaHCO<sub>3</sub> to pH > 8, extracted with EA, concentrated under reduced pressure, and purified by silica column chromatography (PE:EA = 1:1) to get DBA-12-A (0.6 g), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 4.88 (s, 1H), 3.93 (d, <i>J</i> = 7.0 Hz, 1H), 3.57 (s, 3H), 3.26 (s, 1H), 3.14 (s, 3H), 2.38–2.12 (m, 4H), 2.02–1.49 (m, 14H), 1.45–0.94 (m, 18H), 0.94–0.82 (m, 6H), 0.62 (s, 3H).</div><div class="NLM_p">In a 25 mL, one-necked, round-bottomed flask, DBA-12-A (1.2 g, 2.5 mmol, 1.0 equiv) was dissolved in DMF (10 mL) at room temperature. Then, NaN<sub>3</sub> (325 mg, 5.0 mmol, 2.0 equiv) was added. After the addition was complete, the mixture was stirred at 80 °C for 1.5 h. The mixture was washed with water, extracted with EA, concentrated, and purified by column chromatography (PE:EA = 5:1) to get DBA-12-B as a yellow solid (700 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 3.66 (s, 3H), 3.59–3.51 (m, 1H), 3.31–3.20 (m, 1H), 2.38–2.12 (m, 4H), 2.06–0.98 (m, 22H), 0.97–0.90 (m, 6H), 0.67 (s, 3H).</div><div class="NLM_p">In a 25 mL, one-necked, round-bottomed flask, DBA-12-B (200 mg, 0.5 mmol, 1.0 equiv) was dissolved in THF/H<sub>2</sub>O (50:3) (10 mL) at room temperature. Then, PPh<sub>3</sub> (170 mg, 0.7 mmol, 1.4 equiv) was added. After the addition was complete, the mixture was stirred at 65 °C for 15 h. The mixture was washed with water, extracted with EA, concentrated, and purified by column chromatography (DCM:MeOH = 7:1) to get DBA-12-C as a yellow oil (120 mg).</div><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, DBA-12-C (40 mg, 0.1 mmol, 1.0 equiv) was dissolved in dry DMF (2.0 mL) at room temperature. Then, glutaric acid (13.2 mg, 0.1 mmol, 1.0 equiv), HATU (114 mg, 0.3 mmol, 3.0 equiv), and DIEA (38.7 mg, 0.3 mmol, 3.0 equiv) were added at room temperature. After the addition was complete, the mixture was stirred at room temperature for 14 h. The mixture was concentrated to give DBA-12-D as a crude product, which was used directly.</div><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, DBA-12-D (52.0 mg, 0.057 mmol, 1.0 equiv) was dissolved in THF (1.0 mL)/MeOH (0.2 mL)/H<sub>2</sub>O (1.0 mL) at room temperature. Then, LiOH·H<sub>2</sub>O (7.2 mg, 0.17 mmol, 3.0 equiv) was added and stirred at room temperature for 4 h. The mixture was acidified with 1 M HCl to pH < 3, extracted with EA, concentrated, and used directly.</div><div class="NLM_p">The crude product was dissolved in DMF (1.5 mL) at room temperature. Then, 2-aminoethanesulfonic acid (42.5 mg, 0.34 mmol, 6.0 equiv), DIEA (58.8 mg, 0.46 mmol, 8.0 equiv), DMAP (7 mg, 0.05 mmol, 1.0 equiv), and EDC (32.6 mg, 0.17 mmol, 3.0 equiv) were added. After the addition was complete, the mixture was stirred at room temperature for 14 h. The mixture was concentrated and purified by pre-HPLC (Anal: XBridge C18, H<sub>2</sub>O (0.1% NH<sub>4</sub>OAC)/ACN (20–40% in 15 min), retention time: 12.25 min) to get the target product <b>DBA-12</b> as a white solid (0.7 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 3.60–3.57 (m, 2H), 3.51–3.40 (m, 4H), 2.97–3.95 (m, 2H), 1.62–1.40 (m, 22H), 1.40–1.04 (m, 36H), 1.04–0.93 (m, 12H), 0.72 (s, 6H). LRMS (ESI) calcd for [M – H]<sup>−</sup> 985.4, found 985.1 (<a class="ref internalNav" href="#sch13" aria-label="Scheme 13">Scheme 13</a>).</div><figure id="sch13" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Scheme 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0019.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 13. Synthesis of <b>DBA-13</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch13"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a 25 mL, one-necked, round-bottomed flask, UDCA-Ester (200 mg, 0.5 mmol, 1.0 equiv) was dissolved in DMF (10 mL) at room temperature. Then, 1,5-dibromopentane (111.5 mg, 0.5 mmol, 1.0 equiv) and KOH (56.0 mg, 1.0 mmol, 2.0 equiv) were added. After the addition was complete, the mixture was stirred at room temperature for 36 h. The mixture was washed with water, extracted with EA, and concentrated to get a crude product, which was used directly, confirmed by UPLC.</div><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, DBA-13-A (crude, 0.05 mmol, 1.0 equiv) was dissolved in DMF (1.5 mL) at room temperature. Then, 2-aminoethanesulfonic acid (42.5 mg, 0.34 mmol, 6.0 equiv), DIEA (58.8 mg, 0.46 mmol, 8.0 equiv), DMAP (7 mg, 0.05 mmol, 1.0 equiv), and EDC (32.6 mg, 0.17 mmol, 3.0 equiv) were added. After the addition was complete, the mixture was stirred at room temperature for 14 h. The mixture was concentrated and purified by pre-HPLC (Anal: XBridge C18, H<sub>2</sub>O (0.1% NH<sub>4</sub>OAC)/ACN (60–100% in 10 min), retention time: 4.94 min) to get the target product <b>DBA-13</b> as a white solid (0.7 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 3.60–3.41 (m, 10H), 2.95 (d, <i>J</i> = 6.6 Hz, 2H), 2.37–2.01 (m, 15H), 1.96–1.07 (m, 60H), 1.05–0.89 (m, 12H), 0.71 (s, 6H). LRMS (ESI) calcd for [M – H]<sup>−</sup> 959.4, found 959.0 (<a class="ref internalNav" href="#sch14" aria-label="Scheme 14">Scheme 14</a>).</div><figure id="sch14" data-index="19" class="article__inlineFigure"><h2 class="fig-label">Scheme 14</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0020.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0020.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 14. Synthesis of <b>DBA-14</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch14"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a 100 mL, one-necked, round-bottomed flask, UDCA-Ester (406.0 mg, 1.0 mmol, 1.0 equiv) and 3-bromoprop-1-yne (476.0 mg, 4.0 mmol, 4.0 equiv) were dissolved in DMF (2 mL) at room temperature. After the addition was complete, the mixture was stirred at room temperature for 14 h. The mixture was washed with water, extracted with EA, concentrated, and purified by column chromatography (DCM:MeOH = 20:1) to get the target product DBA-14-A (50 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 4.13 (d, <i>J</i> = 2.4 Hz, 2H), 3.60 (s, 3H), 3.55–3.49 (m, 1H), 3.43–3.35 (m, 1H), 2.37 (t, <i>J</i> = 2.4 Hz, 1H), 2.25 (m, 3H), 2.04–0.92 (m, 27H), 0.92–0.84 (m, 6H), 0.61 (s, 3H).</div><div class="NLM_p">In a 100 mL, one-necked, round-bottomed flask, DBA-14-A (25.0 mg, 0.056 mmol, 1.0 equiv) was dissolved in MeCN (2 mL) at room temperature. Then, Cu(OAc)<sub>2</sub> (13.0 mg, 0.067 mmol, 1.2 equiv) and pyridine (26.0 mg, 0.336 mmol, 6.0 equiv) were added. After the addition was complete, the mixture was stirred at 80 °C for 1 h. The mixture was filtered and concentrated to use directly.</div><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, the above mixture of DBA-14-B (crude, 0.03 mmol, 1.0 equiv) was dissolved in THF (0.5 mL)/MeOH (0.1 mL)/H<sub>2</sub>O (0.5 mL) at room temperature. Then, LiOH·H<sub>2</sub>O (6.3 mg, 0.15 mmol, 5.0 equiv) was added and stirred at room temperature for 4 h. The mixture was acidified with 1 M HCl to pH < 3, extracted with EA, concentrated, and used directly.</div><div class="NLM_p">The crude product was dissolved in DMF (1.0 mL) at room temperature. Then, 2-aminoethanesulfonic acid (22.5 mg, 0.18 mmol, 6.0 equiv), DIEA (31.0 mg, 0.24 mmol, 8.0 equiv), DMAP (3.6 mg, 0.03 mmol, 1.0 equiv), and EDC (17.3 mg, 0.09 mmol, 3.0 equiv) were added. After the addition was complete, the mixture was stirred at room temperature for 14 h. The mixture was concentrated and purified by pre-HPLC (Anal: XBridge C18, H<sub>2</sub>O (0.1% NH<sub>4</sub>OAC)/ACN (10–95% in 10 min), retention time: 7.19 min) to get the target product <b>DBA-14</b> as a white solid (2.6 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 4.28 (s, 4H), 3.58 (t, <i>J</i> = 6.9 Hz, 4H), 3.48–3.42 (m, 4H), 2.95 (t, <i>J</i> = 6.9 Hz, 4H), 2.23– 1.99 (m, 8H), 1.98–1.61 (m, 16H), 1.62–1.01 (m, 34H), 0.98–0.96 (m, 12H), 0.71 (s, 6H). LRMS (ESI) calcd for [M/2 – H]<sup>−</sup> 535.8, found 535.6 (<a class="ref internalNav" href="#sch15" aria-label="Scheme 15">Scheme 15</a>).</div><figure id="sch15" data-index="20" class="article__inlineFigure"><h2 class="fig-label">Scheme 15</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0021.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0021.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 15. Synthesis of <b>DBA-15</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0021.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch15"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, DBA-12-D (45.0 mg, 0.05 mmol, 1.0 equiv) was dissolved in THF (2.0 mL) at room temperature. Then, BH<sub>3</sub>·THF (0.4 mL, 1 mol/L, 8.0 equiv) was added dropwise. After the addition was complete, the mixture was stirred at 80 °C for 4 h. The mixture was quenched by 1 M HCl, stirred at room temperature for 30 min, then concentrated, and used directly.</div><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, the above mixture of DBA-15-A (crude, 0.05 mmol, 1.0 equiv) was dissolved in THF (0.5 mL)/MeOH (0.1 mL)/H<sub>2</sub>O (0.5 mL) at room temperature. Then, LiOH·H<sub>2</sub>O (10.5 mg, 0.25 mmol, 5.0 equiv) was added and stirred at room temperature for 4 h. The mixture was acidified with 1 M HCl to pH < 3, extracted with EA, concentrated, and used directly.</div><div class="NLM_p">The crude product was dissolved in DMF (1.0 mL) at room temperature. Then, 2-aminoethanesulfonic acid (37.5 mg, 0.3 mmol, 6.0 equiv), DIEA (48.8 mg, 0.4 mmol, 8.0 equiv), DMAP (6.1 mg, 0.05 mmol, 1.0 equiv), and EDC (28.7 mg, 0.15 mmol, 3.0 equiv) were added. After the addition was complete, the mixture was stirred at room temperature for 14 h. The mixture was concentrated and purified by pre-HPLC (Anal: XBridge C18, H<sub>2</sub>O (0.1% NH<sub>4</sub>OAC)/ACN (45–95% in 15 min), retention time: 8.2 min) to get the target product <b>DBA-15</b> as a white solid (0.8 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 3.85–3.78 (m, 2H), 3.48–3.39 (m, 4H), 3.14–3.02 (m, 6H), 2.95–2.91 (m, 4H), 2.31–2.25 (m, 4H), 2.12–1.71 (m, 23H), 1.64–1.11 (m, 29H), 1.11–0.93 (m, 12H), 0.72 (s, 6H). LRMS (ESI) calcd for [M/2 – H]<sup>−</sup> 531.8, found 531.6 (<a class="ref internalNav" href="#sch16" aria-label="Scheme 16">Scheme 16</a>).</div><figure id="sch16" data-index="21" class="article__inlineFigure"><h2 class="fig-label">Scheme 16</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0022.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0022.jpeg" id="gr16" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 16. Synthesis of <b>DBA-16</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0022.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch16"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a 25 mL, one-necked, round-bottomed flask, DBA-11-A (202 mg, 0.5 mmol, 1.0 equiv) was dissolved in MeOH (5.0 mL) at room temperature. Then, ethane-1,2-diamine (300 mg, 5.0 mmol, 10.0 equiv) and NaBH<sub>3</sub>CN (94 mg, 1.5 mmol, 3.0 equiv) were added in portions at room temperature. After the addition was complete, the mixture was stirred at room temperature for 14 h. The mixture was concentrated to get a crude oil, which was purified by silica chromatography (DCM:MeOH = 10:1) to get DBA-16-A as a yellow solid (100.0 mg), which was confirmed by UPLC.</div><div class="NLM_p">In a 25 mL, one-necked, round-bottomed flask, DBA-16-A (22.4 mg, 0.05 mmol, 1.0 equiv) was dissolved in MeOH (5.0 mL) at room temperature. Then, DBA-11-A (30.5 mg, 0.075 mmol, 1.5 equiv) and PTSA (1.0 mg) were added at room temperature. After the addition was complete, the mixture was stirred at room temperature. After 14 h, NaBH<sub>3</sub>CN (9.5 mg, 0.15 mmol, 3.0 equiv) was added at room temperature. The mixture was stirred at room temperature for another 14 h. Thin-layer chromatography (TLC) showed that the starting material had disappeared, the mixture was concentrated and dissolved in THF (2.0 mL), and then LiOH·H<sub>2</sub>O (10.5 mg, 0.2 mmol, 4.0 equiv) in H<sub>2</sub>O (2.0 mL) was added. The mixture was stirred at room temperature for another 2 h. The mixture was concentrated and purified by pre-HPLC (Anal: XBridge C18, H<sub>2</sub>O (0.1% NH<sub>4</sub>OAC)/ACN (10–95% in 10 min), retention time: 5.40 min) to get DBA-16-B as a white solid (4.3 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 3.50–3.42 (m, 2H), 3.05–2.97 (m, 4H), 2.81–2.72 (m, 2H), 2.34–2.18 (m, 4H), 1.99–1.78 (m, 12H), 1.78–1.64 (m, 7H), 1.64–1.05 (m, 30H), 1.05–0.91 (m, 12H), 0.72 (s, 6H). LRMS (ESI) calcd for [M – H]<sup>−</sup> 807.6, found 807.9.</div><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, DBA-16-B (10.6 mg, 0.013 mmol, 1.0 equiv) was dissolved in dry DMF (2.0 mL) at room temperature. Then, 2-aminoethane-1-sulfonic acid (13.1 mg, 0.105 mmol, 8.0 equiv), HATU (11.9 mg, 0.031 mmol, 2.4 equiv), and DIEA (5.0 mg, 0.039 mmol, 3.0 equiv) were added at room temperature. After the addition was complete, the mixture was stirred at room temperature for 14 h. The mixture was concentrated and purified by pre-HPLC (Anal: XBridge C18, H<sub>2</sub>O (0.1%NH<sub>4</sub>OAC)/ACN (10–95% in 10 min), retention time: 7.15 min) to get the target product DBA-16 as a white solid (2.4 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 3.79–3.71 (m, 4H), 3.66–3.57 (m, 4H), 3.14–3.08 (m, 4H), 3.01–2.94 (m, 4H), 2.29–2.24 (m, 4H), 1.97–1.08 (m, 58H), 1.02–0.96 (m, 12H), 0.71 (s, 6H). LRMS (ESI) calcd for [M – H]<sup>−</sup> 1021.6, found 1021.8 (<a class="ref internalNav" href="#sch17" aria-label="Scheme 17">Scheme 17</a>).</div><figure id="sch17" data-index="22" class="article__inlineFigure"><h2 class="fig-label">Scheme 17</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0023.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0023.jpeg" id="gr17" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 17. Synthesis of <b>DBA-17</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0023.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch17"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a 25 mL, three-necked, round-bottomed flask, Zn powder (322 mg, 5 mmol, 5.0 equiv) in dry THF (10 mL) was added to TiCl<sub>4</sub> (378 mg, 2.0 mmol, 2.0 equiv) dropwise at −15 °C within 30 min. After the addition was complete, the mixture was stirred at 80 °C for 2 h. After cooling to room temperature, DBA-11-A (404 mg, 1.0 mmol, 1.0 equiv) dissolved in MeOH (5.0 mL) was added. The mixture was stirred at 80 °C for 14 h in the dark. The mixture was filtered and concentrated to get a crude product, which was purified by silica chromatography (DCM:MeOH = 20:1) to get the target product DBA-17-A (40 mg), which was confirmed by UPLC.</div><div class="NLM_p">In a 25 mL, one-necked, round-bottomed flask, DBA-17-A (40.0 mg, 0.05 mmol, 1.0 equiv) was dissolved in THF (2.0 mL) and then LiOH·H<sub>2</sub>O (8.6 mg, 0.2 mmol, 4.0 equiv) in H<sub>2</sub>O (1.0 mL) was added. The mixture was stirred at room temperature for 2 h, concentrated, and dissolved in dry DMF (2.0 mL) at room temperature. Then, 2-aminoethane-1-sulfonic acid (37.5 mg, 0.3 mmol, 6.0 equiv), HATU (46.5 mg, 0.12 mmol, 2.4 equiv), and DIEA (52.6 mg, 0.41 mmol, 8.0 equiv) were added at room temperature. After the addition was complete, the mixture was stirred at room temperature for 14 h. The mixture was concentrated and purified by pre-HPLC (Anal: XBridge C18, H<sub>2</sub>O (0.1% NH<sub>4</sub>OAC)/ACN (25–100% in 10 min), retention time: 7.40 min) to get the target product <b>DBA-17</b> as a white solid (10.8 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO): δ 7.66 (t, <i>J</i> = 5.2 Hz, 2H), 3.80 (d, <i>J</i> = 7.0 Hz, 2H), 3.67 (s, 2H), 3.31–3.17 (m, 8H), 2.19–1.54 (m, 25H), 1.54–0.94 (m, 37H), 0.93–0.86 (m, 12H), 0.60 (s, 6H). LRMS (ESI) calcd for [M – H]<sup>−</sup> 995.6, found 995.6 (<a class="ref internalNav" href="#sch18" aria-label="Scheme 18">Scheme 18</a>).</div><figure id="sch18" data-index="23" class="article__inlineFigure"><h2 class="fig-label">Scheme 18</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0024.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0024.jpeg" id="gr18" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 18. Synthesis of <b>DBA-18</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0024.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch18"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a 25 mL, three-necked, round-bottomed flask, Zn powder (322 mg, 5 mmol, 5.0 equiv) in dry THF (10 mL) was added to TiCl<sub>4</sub> (378 mg, 2.0 mmol, 2.0 equiv) dropwise at −15 °C within 30 min. After the addition was complete, the mixture was stirred at 80 °C for 2 h. After cooling to room temperature, DBA-11-A (404 mg, 1.0 mmol, 1.0 equiv) dissolved in MeOH (5.0 mL) was added. The mixture was stirred at 80 °C for 14 h in the dark. The mixture was filtered and concentrated to get a crude product, which was purified by silica chromatography (DCM:MeOH = 20:1) to get DBA-18-A (35 mg), which was confirmed by UPLC.</div><div class="NLM_p">In a 25 mL, one-necked, round-bottomed flask, DBA-18-A (35 mg, 0.047 mmol, 1.0 equiv) was dissolved in THF (2.0 mL) and then LiOH·H<sub>2</sub>O (7.9 mg, 0.19 mmol, 4.0 equiv) in H<sub>2</sub>O (1.0 mL) was added. The mixture was stirred at room temperature for 2 h, concentrated, and dissolved in dry DMF (2.0 mL) at room temperature. Then, 2-aminoethane-1-sulfonic acid (35.2 mg, 0.28 mmol, 6.0 equiv), HATU (42.8 mg, 0.11 mmol, 2.4 equiv), and DIEA (48.5 mg, 0.38 mmol, 8.0 equiv) were added at room temperature. After the addition was complete, the mixture was stirred at room temperature for 14 h. The mixture was concentrated and purified by pre-HPLC (Anal: XBridge C18, H<sub>2</sub>O (0.1% NH<sub>4</sub>OAC)/ACN (60–95% in 10 min), retention time: 7.71 min) to get the target product <b>DBA-18</b> as a white solid (1.2 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO): δ 7.66 (d, <i>J</i> = 4.3 Hz, 2H), 3.91 (d, <i>J</i> = 7.0 Hz, 1H), 3.86 (d, <i>J</i> = 6.6 Hz, 1H), 3.33–3.21 (m, 8H), 3.00 (s, 2H), 2.09–1.37 (m, 20H), 1.37–1.25 (m, 12H), 1.25–0.95 (m, 28H), 0.90–0.86 (m, 12H), 0.61 (s, 6H). LRMS (ESI) calcd for [M – H]<sup>−</sup> 961.6, found 961.6 (<a class="ref internalNav" href="#sch19" aria-label="Scheme 19">Scheme 19</a>).</div><figure id="sch19" data-index="24" class="article__inlineFigure"><h2 class="fig-label">Scheme 19</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0025.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0025.jpeg" id="gr19" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 19. Synthesis of <b>DBA-19</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0025.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch19"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a 25 mL, one-necked, round-bottomed flask, DBA-18 (17 mg, 0.017 mmol, 1.0 equiv) was dissolved in MeOH (4.0 mL) and then Pd/C (5 mg, 10 wt %) was added at room temperature. After the addition was complete, the mixture was stirred at room temperature for 14 h under H<sub>2</sub>. The mixture was concentrated and purified by pre-HPLC (Anal: XBridge C18, H<sub>2</sub>O (0.1% NH<sub>4</sub>OAC)/ACN (60–95% in 10 min), retention time: 7.71 min) to get the target product <b>DBA-19</b> as a white solid (3.4 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 3.58 (t, <i>J</i> = 6.9 Hz, 4H), 3.48–3.40 (m, 2H), 2.96 (t, <i>J</i> = 6.9 Hz, 4H), 2.29–2.14 (m, 4H), 2.10–1.94 (m, 6H), 1.94–1.82 (m, 12H), 1.61–1.01 (m, 36H), 0.98–0.94 (d, <i>J</i> = 7.0 Hz, 12H), 0.71 (s, 6H). LRMS (ESI) calcd for [M – H]<sup>−</sup> 963.6, found 963.9 (<a class="ref internalNav" href="#sch20" aria-label="Scheme 20">Scheme 20</a>).</div><figure id="sch20" data-index="25" class="article__inlineFigure"><h2 class="fig-label">Scheme 20</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0026.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0026.jpeg" id="gr20" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 20. Synthesis of <b>DBA-20</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0026.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch20"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a 10 mL, one-necked, round-bottom flask, DBA-12-B (35.5 mg, 0.084 mmol, 1.5 equiv) and DBA-14-A (25.0 mg, 0.056 mmol, 1.0 equiv) were dissolved in DMF (2.0 mL) at room temperature. Then, CuSO<sub>4</sub>·5H<sub>2</sub>O (15.12 mg, 0.056 mmol, 1.0 equiv) and sodium-<span class="smallcaps smallerCapital">l</span>-ascorbate (22.0 mg, 0.112 mmol, 2.0 equiv) in H<sub>2</sub>O (2.0 mL) were added. After the addition was complete, the mixture was stirred at room temperature for 24 h. The mixture was washed with water, extracted with DCM, concentrated, and purified by column chromatography (DCM:MeOH = 20:1) to get the target product DBA-20-A (20 mg).</div><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, DBA-20-A (20.0 mg, 0.023 mmol, 1.0 equiv) was dissolved in THF (1.0 mL)/MeOH (0.2 mL)/H<sub>2</sub>O (1.0 mL) at room temperature. Then, LiOH·H<sub>2</sub>O (4.2 mg, 0.114 mmol, 5.0 equiv) was added and stirred at room temperature for 4 h. The mixture was acidified with 1 M HCl to pH < 3, extracted with EA, concentrated, and used directly.</div><div class="NLM_p">The crude product was dissolved in DMF (1.5 mL) at room temperature. Then, 2-aminoethanesulfonic acid (17.8 mg, 0.142 mmol, 6.0 equiv), DIEA (24.7 mg, 0.192 mmol, 8.0 equiv), DMAP (3 mg, 0.024 mmol, 1.0 equiv), and EDC (13.6 mg, 0.71 mmol, 3.0 equiv) were added. After the addition was complete, the mixture was stirred at room temperature for 14 h. The mixture was concentrated and purified by pre-HPLC (Anal: XBridge C18, H<sub>2</sub>O (0.1% NH<sub>4</sub>OAC)/ACN (10–95% in 10 min), retention time: 6.21 min) to get the target product <b>DBA-20</b> as a white solid (4.4 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 8.07 (s, 1H), 7.93 (t, <i>J</i> = 5.6, 1H), 4.62 (s, 2H), 4.52–4.46 (m, 1H), 3.61–3.53 (m, 4H), 3.52–3.36 (m, 3H), 3.01–2.92 (m, 4H), 2.29–2.03 (m, 9H), 1.97–1.68 (m, 14H), 1.66–1.01 (m, 32H), 0.98–0.96 (m, 6H), 0.73–0.71 (m, 6H). <sup>13</sup>C NMR (100 MHz, CD<sub>3</sub>OD) δ 176.6, 176.5, 146.1, 123.3, 79.9, 71.9, 71.6, 62.2, 61.9, 57.5, 57.1, 56.5, 56.4, 51.5, 44.8, 44.7, 44.5, 44.4, 43.9, 41.6, 41.4, 40.7, 40.5, 38.6, 38.3, 36.8, 36.7, 36.6, 36.5, 35.9, 35.7, 35.4, 35.1, 34.2, 33.2, 29.7, 29.0, 28.1, 28.0, 24.0, 23.9, 22.5, 22.4, 19.1, 12.7. LRMS (ESI) calcd for [M – H]<sup>−</sup> 1061.5, found 1061.3 (<a class="ref internalNav" href="#sch21" aria-label="Scheme 21">Scheme 21</a>).</div><figure id="sch21" data-index="26" class="article__inlineFigure"><h2 class="fig-label">Scheme 21</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0027.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0027.jpeg" id="gr21" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 21. Synthesis of <b>DBA-21</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0027.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch21"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To a solution of UDCA-Ester (3.000 g, 7.4 mmol, 1.0 equiv) in DMF (20 mL) were added imidazole (1.505 g, 22.13 mmol, 3.0 equiv) and tert-butyldimethylsilyl chloride (2.786 g, 18.45 mmol, 2.5 equiv) at room temperature. The mixture was stirred at room temperature for 16 h. TLC showed that the starting material was consumed. The mixture was poured into ice water, extracted with EA, washed with brine, and concentrated. The residue was purified by silica gel column chromatography (PE:EA = 30:1) to get the target product DBA-21-A as a white solid (4.0 g).</div><div class="NLM_p">To a solution of DBA-21-A (2.500 g, 3.95 mmol, 1.0 equiv) in THF (30 mL) was added DIBAL-H (5.3 mL, 2.635 mmol, 2.0 equiv) at 0 °C under Ar. The solution was stirred at room temperature for 15 h. TLC showed that a trace amount of the starting material remained. Aqueous sodium hydroxide (1 N) was added to the above reaction. The mixture was extracted with EA and concentrated. The residue was purified by column chromatography (PE:EA = 10:1) to get DBA-21-B as oily solid (1.8 g), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 3.701–3.585 (m, 3H), 3.576–3.510 (m, 1H), 1.996–1.913 (m, 1H), 1.856–0.836 (m, 60H), 0.646 (s, 3H), 0.059–0.039 (m, 12H).</div><div class="NLM_p">To a solution of DBA-21-B (640.0 mg, 1.054 mmol, 1.0 equiv) in DCM (4 mL) was added PCC (569 mg, 2.635 mmol, 2.5 equiv) at room temperature. The mixture was stirred at room temperature for 2 h. TLC showed that most of the starting material was consumed. The mixture was concentrated and purified by column chromatography (DCM: 100%) to get DBA-21-C as a white solid (600 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 9.762 (s, 1H), 3.689–3.612 (m, 1H), 3.596–3.492 (m, 1H), 1.952–0.730 (m, 60H), 0.646 (s, 3H), 0.062–0.029 (m, 12H).</div><div class="NLM_p">To a mixture of K<sub>2</sub>CO<sub>3</sub> (251 mg, 1.82 mmol, 2.5 equiv) in MeOH was added a solution of dimethyl (1-diazo-2-oxopropyl)phosphonate (280.0 mg, 1.45 mmol, 2.0 equiv) in MeOH (10 mL) at 0 °C; then, a solution of DBA-21-C (440.0 mg, 0.727 mmol, 1.0 equiv) in DCM (10 mL) was added to the above mixture at 0 °C. Then, the mixture was stirred at room temperature under an Ar balloon for 5 h. TLC showed that most of the starting material was consumed. The mixture was concentrated and purified by column chromatography (PE:EA = 100:1) to get the target product (DBA-21-D) as a white solid (190 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 3.661–3.546 (m, 1H), 3.531–3.493 (m, 1H), 2.356–0.886 (m, 63H), 0.653 (s, 3H), 0.068–0.046 (m, 12H).</div><div class="NLM_p">To a mixture of DBA-21-D (85.0 mg, 0.141 mmol, 1.0 equiv) and DBA-12-B (74.0 mg, 0.170 mmol, 1.2 equiv) in DCM (4 mL) were added CuSO<sub>4</sub>·5H<sub>2</sub>O (36.0 mg, 0.141 mmol, 1.0 equiv) and (+)-sodium-<span class="smallcaps smallerCapital">l</span>-ascorbate (252 mg, 1.272 mmol, 9.0 equiv) at room temperature. The mixture was stirred at room temperature for 29 h. TLC showed that the starting material (30%) was remaining. The mixture was concentrated and purified by column chromatography (DCM:MeOH = 25:1) to get DBA-21-E as a yellow solid (70 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.243 (s, 1H), 4.421–4.301 (m, 1H), 3.755–3.410 (m, 6H), 2.783–0.834 (m, 98H), 0.696–0.644 (m, 6H), 0.062–0.046 (m, 12H).</div><div class="NLM_p">To a solution of DBA-21-E (65.0 mg, 0.063 mmol, 1.0 equiv) in dioxane (2 mL) was added 2 N hydrochloric acid (0.4 mL) at room temperature. The mixture was stirred at room temperature for 16 h. TLC showed that the reaction was complete. The mixture was washed with water, extracted with EA, concentrated, and purified by column chromatography (DCM:MeOH = 25:1) to get DBA-21-F as a white solid (30 mg).</div><div class="NLM_p">To a solution of DBA-21-F (30.0 mg, 0.037 mmol, 1.0 equiv) in MeOH/THF/H<sub>2</sub>O (1:4:4, 3 mL) was added LiOH (6.0 mg, 0.223 mmol, 6.0 equiv) at room temperature. Then, the mixture was stirred at room temperature for 16 h. UPLC showed that the desired product was formed. The mixture was concentrated to remove the solvent. The residue was dissolved in H<sub>2</sub>O and adjusted to pH < 3 by adding 2 N HCl. Some suspensions appeared and filtered. The filtered cake was dissolved in MeOH and concentrated to get the target product <b>DBA-21</b> (20 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 7.826 (s, 1H), 4.496–4.402 (m, 1H), 3.576–3.412 (m, 3H), 2.813–2.711 (m, 1H), 2.642–2.557 (m, 1H), 2.351–0.902 (m, 65H), 0.738–0.700 (m, 6H) (<a class="ref internalNav" href="#sch22" aria-label="Scheme 22">Scheme 22</a>).</div><figure id="sch22" data-index="27" class="article__inlineFigure"><h2 class="fig-label">Scheme 22</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0028.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0028.jpeg" id="gr22" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 22. Synthesis of <b>DBA-22</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0028.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch22"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To a solution of DBA-21 (15.0 mg, 0.019 mmol, 1.0 equiv) in DCM/DMF (3 mL) were added 2-aminoethane-1-sulfonic acid (8.0 mg, 0.057 mmol, 3.0 equiv), HATU (15.0 mg, 0.038 mmol, 2.0 equiv), and <i>N</i>-ethyl-<i>N</i>-isopropylpropan-2-amine (10.0 mg, 0.076 mmol, 4.0 equiv) at room temperature. The mixture was stirred at room temperature for 17 h. UPLC showed that the desired product was formed. The mixture was concentrated and purified by Prep-HPLC to get the target product <b>DBA-22</b> as a solid (21 mg, purity is 98%), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 7.927 (m, 1H), 4.483–4.426 (m, 1H), 3.597–3.439 (m, 5H), 2.963 (t, <i>J</i> = 6.8 Hz, 2H), 2.802–2.738 (m, 1H), 2.618–2.557 (m, 1H), 2.292–0.967 (m, 66H), 0.731–0.698 (m, 1H). LRMS (ESI) calcd for [M – 1]<sup>−</sup> 896.3, found 896.1 (<a class="ref internalNav" href="#sch23" aria-label="Scheme 23">Scheme 23</a>).</div><figure id="sch23" data-index="28" class="article__inlineFigure"><h2 class="fig-label">Scheme 23</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0029.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0029.jpeg" id="gr23" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 23. Synthesis of <b>DBA-23</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0029.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch23"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, TUDCA (100 mg, 0.2 mmol, 1.0 equiv) was dissolved in DMF (1 mL) at room temperature. Then, heptanedioic acid (32 mg, 0.2 mmol, 1.0 equiv), DIEA (103.2 mg, 0.8 mmol, 4.0 equiv), DMAP (48.8 mg, 0.4 mmol, 2.0 equiv), and EDC (92.0 mg, 0.48 mmol, 2.4 equiv) were added. After the addition was complete, the mixture was stirred at room temperature for 48 h. The mixture was concentrated and purified by pre-HPLC (Anal: XBridge C18, H<sub>2</sub>O (0.1% NH<sub>4</sub>OAC)/ACN (10–95% in 10 min), retention time: 7.44 min) to get the target product <b>DBA-23</b> as a white solid (1.4 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO): δ 7.66 (s, 2H), 4.55 (s, 2H), 3.89 (s, 2H), 3.28–3.22 (m, 10H), 2.33–1.79 (m, 30H), 1.78–0.94 (m, 42H), 0.91–0.85 (m, 12H), 0.61 (s, 6H). LRMS (ESI) calcd for [M – 1]<sup>−</sup> 1122.5, found 1122.2 (<a class="ref internalNav" href="#sch24" aria-label="Scheme 24">Scheme 24</a>).</div><figure id="sch24" data-index="29" class="article__inlineFigure"><h2 class="fig-label">Scheme 24</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0030.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0030.jpeg" id="gr24" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 24. Synthesis of <b>DBA-24</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0030.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch24"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, DBA-11-C (50.0 mg, 0.123 mmol, 1.0 equiv) was dissolved in dry DMF (2.0 mL) at room temperature. Then, malonic acid (12.8 mg, 0.123 mmol, 1.0 equiv), HATU (114 mg, 0.3 mmol, 2.4 equiv), and DIEA (47.6 mg, 0.369 mmol, 3.0 equiv) were added at room temperature. After the addition was complete, the mixture was stirred at room temperature for 14 h. The mixture was concentrated and dissolved in THF (2.0 mL); then, LiOH·H<sub>2</sub>O (21.0 mg, 0.5 mmol, 4.0 equiv) in H<sub>2</sub>O (2.0 mL) was added. The mixture was stirred at room temperature for another 2 h. The mixture was concentrated and purified by silica chromatography (DCM:MeOH = 20:1) to get DBA-24-A as a white solid (31.4 mg), which was confirmed by UPLC.</div><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, DBA-24-A (31.4 mg, 0.037 mmol, 1.0 equiv) was dissolved in dry DMF (2.0 mL) at room temperature. Then, 2-aminoethane- 1-sulfonic acid (37 mg, 0.296 mmol, 8.0 equiv), HATU (33.4 mg, 0.089 mmol, 2.4 equiv), and DIEA (14.3 mg, 0.111 mmol, 3.0 equiv) were added at room temperature. After the addition was complete, the mixture was stirred at room temperature for 14 h. The mixture was concentrated and purified by pre-HPLC (Anal: XBridge C18, H<sub>2</sub>O (0.1% NH<sub>4</sub>OAC)/MeOH (60–90% in 10 min), retention time: 5.49 min) to get the target product <b>DBA-24</b> as a white solid (2.6 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 4.05 (s, 2H), 3.58 (t, <i>J</i> = 6.9 Hz, 4H), 3.48–3.41 (m, 2H), 2.93 (t, <i>J</i> = 6.9 Hz, 6H), 2.31–2.18 (m, 4H), 2.17–2.01 (m, 7H), 1.99–1.13 (m, 45H), 1.04–0.97 (m, 12H), 0.71 (s, 6H). LRMS (ESI) calcd for [M/2 – H]<sup>−</sup> 531.3, found 531.4 (<a class="ref internalNav" href="#sch25" aria-label="Scheme 25">Scheme 25</a>).</div><figure id="sch25" data-index="30" class="article__inlineFigure"><h2 class="fig-label">Scheme 25</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0031.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0031.jpeg" id="gr25" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 25. Synthesis of <b>DBA-25</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0031.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch25"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, DBA-12-D (52.0 mg, 0.057 mmol, 1.0 equiv) was dissolved in THF (1.0 mL)/MeOH (0.2 mL)/H<sub>2</sub>O (1.0 mL) at room temperature. Then, LiOH·H<sub>2</sub>O (7.2 mg, 0.17 mmol, 3.0 equiv) was added and stirred at room temperature for 4 h. The mixture was acidified with 1 M HCl to pH < 3, extracted with EA, concentrated, and used directly.</div><div class="NLM_p">The crude product was dissolved in DMF (1.5 mL) at room temperature. Then, 2-aminoethanesulfonic acid (42.5 mg, 0.34 mmol, 6.0 equiv), DIEA (58.8 mg, 0.46 mmol, 8.0 equiv), DMAP (7 mg, 0.05 mmol, 1.0 equiv), and EDC (32.6 mg, 0.17 mmol, 3.0 equiv) were added. After the addition was complete, the mixture was stirred at room temperature for 14 h. The mixture was concentrated and purified by pre-HPLC (Anal: XBridge C18, H<sub>2</sub>O (0.1% NH<sub>4</sub>OAC)/ACN (50–70% in 10 min), retention time: 7.6 min) to get the target product <b>DBA-25</b> as a white solid (15.1 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO): δ 7.66 (t, <i>J</i> = 5.2 Hz, 2H), 7.59 (d, <i>J</i> = 8.0 Hz, 2H), 3.93 (d, <i>J</i> = 6.9 Hz, 2H), 3.49–3.43 (m, 4H), 3.26 (dd, <i>J</i> = 13.4, 5.8 Hz, 8H), 2.12–1.57 (m, 32H), 1.57–0.94 (m, 45H), 0.92–0.82 (m, 12H), 0.61 (s, 6H).<sup>13</sup>C NMR (100 MHz, CD<sub>3</sub>OD) δ 176.5, 176.4, 174.4, 174.3, 71.9, 57.6, 57.5, 56.3, 51.5, 51.0, 50.5, 44.8, 44.5, 44.3, 41.6, 40.7, 38.5, 36.9, 36.8, 36.7, 36.3, 36.2, 35.2, 35.0, 34.1, 33.2, 29.7, 28.3, 28.0, 26.5, 24.0, 23.6, 22.4, 19.0, 12.7. LRMS (ESI) calcd for [M – H]<sup>−</sup> 1091.6, found 1091.8 (<a class="ref internalNav" href="#sch26" aria-label="Scheme 26">Scheme 26</a>).</div><figure id="sch26" data-index="31" class="article__inlineFigure"><h2 class="fig-label">Scheme 26</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0032.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0032.jpeg" id="gr26" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 26. Synthesis of <b>DBA-26</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0032.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch26"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To a solution of UDCA (291 mg, 0.740 mmol, 1.5 equiv) in DCM/DMF (6 mL) were added DBA-12-C (200.0 mg, 0.493 mmol, 1.0 equiv), HATU (301.0 mg, 0.790 mmol, 1.6 equiv), and DIPEA (128.0 mg, 0.988 mmol, 2.0 equiv) at room temperature. The mixture was stirred at room temperature for 17 h. TLC showed that the reaction was complete. The mixture was washed with water, extracted with DCM, concentrated, and purified by column chromatography (DCM:MeOH = 25:1) to get the target product DBA-26-A as a yellow solid (350 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 5.210–5.258 (m, 1H), 3.703–3.795 (m, 1H), 3.655 (s, 3H), 3.500–3.623 (m, 3H), 2.410–2.311 (m, 1H), 2.112–2.298 (m, 2H), 2.102–1.012 (m, 66H), 0.94–0.93 (m, 12H), 0.665 (s, 6H).</div><div class="NLM_p">To a solution of methyl DBA-26-A (345.0 mg, 0.442 mmol, 1.0 equiv) in MeOH/THF/H<sub>2</sub>O (1:4:4, 15 mL) was added lithium hydroxide (85.0 mg, 3.54 mmol, 8.0 equiv) at room temperature. Then, the mixture was stirred at room temperature for 16 h. UPLC showed that the desired product was formed. The mixture was concentrated to remove the solvent. The residue was dissolved in H<sub>2</sub>O and adjusted to pH < 3 by adding 2 N HCl. Some suspensions appeared and filtered. The filtered cake was dissolved in MeOH and concentrated to get the target product DBA-26-B (315 mg), which was used for the next step directly.</div><div class="NLM_p">To a solution of DBA-26-B (310 mg, 0.404 mmol, 1.0 equiv) in DCM/DMF (8 mL) were added 2-aminoethane-1-sulfonic acid (152.0 mg, 1.214 mmol, 3.0 equiv), HATU (307.0 mg, 0.809 mmol, 2.0 equiv), and DIPEA (261.0 mg, 2.023 mmol, 5.0 equiv) at room temperature. The mixture was stirred at room temperature for 17 h. UPLC showed that the desired product was formed. The mixture was concentrated and purified by Prep-HPLC to get the target product <b>DBA-26</b> as a yellow solid (170 mg, purity is 99%), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO): δ 7.683–7.656 (m, 1H), 7.605–7.585 (m, 1H), 7.080 (br, 3H), 4.455–4.444 (d, <i>J</i> = 4.4 Hz, 1H), 4.126–4.087 (m, 1H), 3.930–3.878 (m, 2H), 3.337–3.313 (m, 1H), 3.289–3.239 (m, 5H), 3.171–3.157 (m, 2H), 2.074–0.866 (m, 66H), 0.612–0.600 (m, 6H). LRMS (ESI) calcd for [M – 1]<sup>−</sup> 872.3, found 872.0 (<a class="ref internalNav" href="#sch27" aria-label="Scheme 27">Scheme 27</a>).</div><figure id="sch27" data-index="32" class="article__inlineFigure"><h2 class="fig-label">Scheme 27</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0033.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0033.jpeg" id="gr27" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 27. Synthesis of <b>DBA-27</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0033.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch27"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a 25 mL, one-necked, round-bottomed flask, DBA-12-B (200.0 mg, 0.46 mmol, 1.0 equiv) was dissolved in THF (10 mL) at room temperature. Then, LAH (88 mg, 2.3 mmol, 5.0 equiv) was added. After the addition was complete, the mixture was stirred at room temperature for 2 h. The mixture was quenched by Na<sub>2</sub>SO<sub>4</sub>·10H<sub>2</sub>O (1.48 g, 4.6 mmol, 10.0 equiv), filtered, and concentrated to get a crude solid, which purified by silica column chromatography (DCM:MeOH = 10:1) to get the target product DBA-27-A (70 mg), which was confirmed by UPLC.</div><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, DBA-12-C (230.0 mg, 0.53 mmol, 1.0 equiv) was dissolved in THF (10 mL) at room temperature. Then, dihydro-2<i>H</i>-pyran-2,6(3<i>H</i>)-dione (64 mg, 0.56 mmol, 1.05 equiv) and DMAP (cat) were added. After the addition was complete, the mixture was stirred at room temperature for 2 h. The mixture was concentrated to use directly.</div><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, the above mixture of DBA-27-B (54.6 mg, 0.1 mmol, 1.0 equiv) was dissolved in DCM (5.0 mL) at room temperature. Then, DBA-27-A (37.7 mg, 0.1 mmol, 1.0 equiv), DIEA (38.7 mg, 0.3 mmol, 3.0 equiv), DMAP (12.2 mg, 0.1 mmol, 1.0 equiv), and EDC (192 mg, 0.15 mmol, 1.5 equiv) were added. After the addition was complete, the mixture was stirred at room temperature for 24 h. The mixture was concentrated and purified by column chromatography (DCM:MeOH = 20:1) to get the target product DBA-27-C (18 mg).</div><div class="NLM_p">In a 5 mL, one-necked, round-bottomed flask, DBA-27-C (13.0 mg, 0.015 mmol, 1.0 equiv) was dissolved in THF (1.0 mL)/MeOH (0.2 mL)/H<sub>2</sub>O (1.0 mL) at room temperature. Then, LiOH·H<sub>2</sub>O (1.9 mg, 0.045 mmol, 3.0 equiv) was added and stirred at room temperature for 4 h. The mixture was acidified with 1 M HCl to pH < 3, extracted with EA, concentrated, and used directly.</div><div class="NLM_p">The crude product was dissolved in THF (1.5 mL) at room temperature. Then, DMAP (5.5 mg, 0.045 mmol, 3.0 equiv) was added, followed by addition of 2,4,6-trichlorobenzoyl chloride (7.3 mg, 0.03 mmol, 2.0 equiv) under Ar at room temperature. After the addition was complete, the mixture was stirred at room temperature for 48 h. The mixture was concentrated and purified by pre-HPLC (Anal: XBridge C18, H<sub>2</sub>O (0.1% NH<sub>4</sub>OAC)/ACN (80–100% in 10 min), retention time: 5.71 min) to get the target product <b>DBA-27</b> as a white solid (1.3 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 5.40–5.32 (m, 2H), 4.24–4.17 (m, 1H), 3.96–3.90 (m, 1H), 3.56–3.49 (m, 2H), 2.40–1.05 (m, 58H), 1.05–0.87 (m, 12H), 0.70 (s, 6H). LRMS (ESI) calcd for [M + H]<sup>+</sup> 847.6, found 847.7 (<a class="ref internalNav" href="#sch28" aria-label="Scheme 28">Scheme 28</a>).</div><figure id="sch28" data-index="33" class="article__inlineFigure"><h2 class="fig-label">Scheme 28</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0034.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0034.jpeg" id="gr28" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 28. Synthesis of <b>DBA-28</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0034.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch28"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, DBA-12-D (45.0 mg, 0.05 mmol, 1.0 equiv) was dissolved in THF (1.0 mL)/MeOH (0.2 mL)/H<sub>2</sub>O (1.0 mL) at room temperature. Then, LiOH·H<sub>2</sub>O (7.2 mg, 0.17 mmol, 3.0 equiv) was added and stirred at room temperature for 4 h. The mixture was acidified with 1 M HCl to pH < 3, extracted with EA, concentrated, and used directly.</div><div class="NLM_p">The crude product was dissolved in DMF (1.5 mL) at room temperature. Then, ethane-1,2-diol (2.2 mg, 0.035 mmol, 0.7 equiv), DIEA (25.8 mg, 0.2 mmol, 4.0 equiv), DMAP (6 mg, 0.05 mmol, 1.0 equiv), and DCC (31.0 mg, 0.15 mmol, 3.0 equiv) were added. After the addition was complete, the mixture was stirred at room temperature for 14 h. The mixture was concentrated and purified by pre-HPLC (Anal: XBridge C18, H<sub>2</sub>O (0.1% NH<sub>4</sub>OAC)/ACN (70–75% in 10 min), retention time: 5.96 min) to get the target product <b>DBA-28</b> as a white solid (3.4 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 5.43 (d, <i>J</i> = 7.1 Hz, 2H), 4.36–4.19 (m, 4H), 3.64–3.45 (m, 4H), 2.44–2.05 (m, 10H), 2.05–1.01 (m, 54H), 0.98–0.89 (m, 12H), 0.68 (s, 6H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 174.0, 171.5, 71.4, 62.3, 55.9, 54.3, 50.1, 43.9, 43.8, 42.7, 40.2, 39.4, 36.8, 36.1, 35.4, 34.8, 34.4, 34.2, 31.0, 30.5, 28.5, 27.4, 26.9, 23.6, 21.7, 21.1, 18.6, 12.1. LRMS (ESI) calcd for [M + H]<sup>+</sup> 905.6, found 905.8 (<a class="ref internalNav" href="#sch29" aria-label="Scheme 29">Scheme 29</a>).</div><figure id="sch29" data-index="34" class="article__inlineFigure"><h2 class="fig-label">Scheme 29</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0035.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0035.jpeg" id="gr29" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 29. Synthesis of <b>DBA-29</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0035.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch29"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, DBA-12-D (45.0 mg, 0.05 mmol, 1.0 equiv) was dissolved in THF (1.0 mL)/MeOH (0.2 mL)/H<sub>2</sub>O (1.0 mL) at room temperature. Then, LiOH·H<sub>2</sub>O (7.2 mg, 0.17 mmol, 3.0 equiv) was added and stirred at room temperature for 4 h. The mixture was acidified with 1 M HCl to pH < 3, extracted with EA, concentrated, and used directly. The crude product was dissolved in DMF (1.5 mL) at room temperature. Then, ethane-1,2-diamine (2.1 mg, 0.035 mmol, 0.7 equiv), DIEA (25.8 mg, 0.2 mmol, 4.0 equiv), DMAP (6 mg, 0.05 mmol, 1.0 equiv), and EDC (29.5 mg, 0.15 mmol, 3.0 equiv) were added. After the addition was complete, the mixture was stirred at room temperature for 14 h. The mixture was concentrated and purified by pre-HPLC (Anal: XBridge C18, H<sub>2</sub>O (0.1% NH<sub>4</sub>OAC)/MeOH (80–100% in 10 min), retention time: 7.30 min) to get the target product <b>DBA-29</b> as a white solid (4.5 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 3.61–3.50 (m, 2H), 3.49–3.39 (m, 2H), 3.38–3.20 (m, 8H), 2.29–1.98 (m, 11H), 1.96–1.66 (m, 14H), 1.66–1.04 (m, 29H), 1.03–0.91 (m, 12H), 0.71 (s, 6H).<sup>13</sup>C NMR (100 MHz, CD<sub>3</sub>OD) δ 177.4, 174.5, 71.9, 57.7, 56.5, 50.9, 44.8, 44.5, 44.4, 41.7, 40.7, 40.4, 38.4, 37.1, 36.6, 36.5, 35.2, 34.8, 34.3, 33.5, 29.8, 28.0, 24.0, 23.1, 22.4, 19.1, 12.6. LRMS (ESI) calcd for [M + H]<sup>+</sup> 903.7, found 903.8 (<a class="ref internalNav" href="#sch30" aria-label="Scheme 30">Scheme 30</a>).</div><figure id="sch30" data-index="35" class="article__inlineFigure"><h2 class="fig-label">Scheme 30</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0036.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0036.jpeg" id="gr30" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 30. Synthesis of <b>DBA-30</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0036.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch30"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, DBA-29 (45.0 mg, 0.05 mmol, 1.0 equiv) was dissolved in DCM (1.0 mL) at room temperature. Then, (BnO)<sub>2</sub>PN(<i>i</i>-Pr)<sub>2</sub> (58.6 mg, 0.17 mmol, 3.0 equiv) and Tetrazole (0.38 mL, 0.45 M, 0.17 mmol, 3.0 equiv) were added and stirred at room temperature for 2 h. The mixture was cooled down to 0 °C, and 1 mL 30% H<sub>2</sub>O<sub>2</sub> was added. After stirring at 0 °C for 30 min, the solution was quenched by conc. Na<sub>2</sub>S<sub>2</sub>SO<sub>3</sub> and extracted with DCM. The mixture was concentrated and purified by pre-HPLC (Anal: XBridge C18, H<sub>2</sub>O (0.1% NH<sub>4</sub>OAC)/MeOH (80–100% in 10 min), retention time: 6.56 min) to get the target product <b>DBA-30</b> as a white solid (7.8 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.37–7.26 (m, 20H), 6.22 (t, <i>J</i> = 4.3 Hz, 2H), 5.48 (d, <i>J</i> = 7.2 Hz, 2H), 5.10–4.85 (m, 8H), 4.36 (dq, <i>J</i> = 11.0, 5.5 Hz, 2H), 3.62 (ddd, <i>J</i> = 11.3, 7.5, 3.6 Hz, 2H), 3.47 (dd, <i>J</i> = 9.9, 3.5 Hz, 2H), 3.34–3.27 (m, 2H), 2.28–0.81 (m, 70H), 0.64 (s, 6H). LRMS (ESI) calcd for [M + H]<sup>+</sup> 1423.8, found 1424.6 (<a class="ref internalNav" href="#sch31" aria-label="Scheme 31">Scheme 31</a>).</div><figure id="sch31" data-index="36" class="article__inlineFigure"><h2 class="fig-label">Scheme 31</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0037.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0037.jpeg" id="gr31" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 31. Synthesis of <b>DBA-31</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0037.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch31"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, DBA-12-D (45.0 mg, 0.05 mmol, 1.0 equiv) was dissolved in THF (1.0 mL)/MeOH (0.2 mL)/H<sub>2</sub>O (1.0 mL) at room temperature. Then, LiOH·H<sub>2</sub>O (7.2 mg, 0.17 mmol, 3.0 equiv) was added and stirred at room temperature for 4 h. The mixture was acidified with 1 M HCl to pH < 3, extracted with EA, concentrated, and used directly. The crude product was dissolved in DMF (1.5 mL) at room temperature. Then, 1,3-phenylenedimethanamine (4.7 mg, 0.035 mmol, 0.7 equiv), DIEA (25.8 mg, 0.2 mmol, 4.0 equiv), DMAP (6 mg, 0.05 mmol, 1.0 equiv), and EDC (29.5 mg, 0.15 mmol, 3.0 equiv) were added. After the addition was complete, the mixture was stirred at room temperature for 14 h. The mixture was concentrated and purified by pre-HPLC (Anal: XBridge C18, H<sub>2</sub>O (0.1% NH<sub>4</sub>OAC)/MeOH (80–100% in 10 min), retention time: 6.56 min) to get the target product <b>DBA-31</b> as a white solid (7.8 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 7.34–7.11 (m, 4H), 4.44 (d, <i>J</i> = 14.7 Hz, 2H), 4.18 (d, <i>J</i> = 14.8 Hz, 2H), 3.56–3.40 (m, 4H), 2.30–0.94 (m, 70H), 0.75 (s, 6H). LRMS (ESI) calcd for [M + H]<sup>+</sup> 979.7, found 979.8 (<a class="ref internalNav" href="#sch32" aria-label="Scheme 32">Scheme 32</a>).</div><figure id="sch32" data-index="37" class="article__inlineFigure"><h2 class="fig-label">Scheme 32</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0038.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0038.jpeg" id="gr32" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 32. Synthesis of <b>DBA-32</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0038.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch32"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, DBA-12-C (120.0 mg, 0.3 mmol, 1.0 equiv) was dissolved in DMF (2.0 mL) at room temperature. Then, 2,2′-oxydiacetic acid (40.2 mg, 0.3 mmol, 1.0 equiv), DIEA (116.0 mg, 0.9 mmol, 3.0 equiv), DMAP (36 mg, 0.3 mmol, 1.0 equiv), and EDC (173 mg, 0.9 mmol, 3.0 equiv) were added. After the addition was complete, the mixture was stirred at room temperature for 24 h. The mixture was confirmed by UPLC, concentrated, and purified by chromatography column (DCM:MeOH = 20:1) to get the target product DBA-32-A (69 mg).</div><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, DBA-32-A (45.4 mg, 0.05 mmol, 1.0 equiv) was dissolved in THF (1.0 mL)/MeOH (0.2 mL)/H<sub>2</sub>O (1.0 mL) at room temperature. Then, LiOH·H<sub>2</sub>O (7.2 mg, 0.17 mmol, 3.0 equiv) was added and stirred at room temperature for 4 h. The mixture was acidified with 1 M HCl to pH < 3, extracted with EA, concentrated, and used directly.</div><div class="NLM_p">The crude product was dissolved in DMF (1.5 mL) at room temperature. Then, ethane-1,2-diamine (2.1 mg, 0.035 mmol, 0.7 equiv), DIEA (25.8 mg, 0.2 mmol, 4.0 equiv), DMAP (6 mg, 0.05 mmol, 1.0 equiv), and DCC (31.0 mg, 0.15 mmol, 3.0 equiv) were added. After the addition was complete, the mixture was stirred at room temperature for 14 h. The mixture was concentrated and purified by pre-HPLC (Anal: XBridge C18, H<sub>2</sub>O (0.1% NH<sub>4</sub>OAC)/ACN (70–75% in 10 min), retention time: 5.25 min) to get the target product <b>DBA-32</b> as a white solid (3.5 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 4.04–3.93 (m, 4H), 3.76–3.67 (m, 2H), 3.49–3.43 (m, 2H), 3.25–3.15 (m, 4H), 2.26 (dt, <i>J</i> = 12.7, 6.4 Hz, 3H), 2.17–0.78 (m, 61H), 0.73 (s, 6H). LRMS (ESI) calcd for [M + H]<sup>+</sup> 905.7, found 906.1 (<a class="ref internalNav" href="#sch33" aria-label="Scheme 33">Scheme 33</a>).</div><figure id="sch33" data-index="38" class="article__inlineFigure"><h2 class="fig-label">Scheme 33</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0039.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0039.jpeg" id="gr33" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 33. Synthesis of <b>DBA-33</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0039.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch33"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, DBA-12-C (120.0 mg, 0.3 mmol, 1.0 equiv) was dissolved in DMF (2.0 mL) at room temperature. Then, 2,2′-(methylazanediyl)diacetic acid (44.1 mg, 0.3 mmol, 1.0 equiv), DIEA (116.0 mg, 0.9 mmol, 3.0 equiv), DMAP (36 mg, 0.3 mmol, 1.0 equiv), and EDC (173 mg, 0.9 mmol, 3.0 equiv) were added. After the addition was complete, the mixture was stirred at room temperature for 24 h. The mixture was confirmed by UPLC, concentrated, and purified by column chromatography (DCM:MeOH = 20:1) to get the target product DBA-33-A (33 mg).</div><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, DBA-33-A (46.1 mg, 0.05 mmol, 1.0 equiv) was dissolved in THF (1.0 mL)/MeOH (0.2 mL)/H<sub>2</sub>O (1.0 mL) at room temperature. Then, LiOH·H<sub>2</sub>O (7.2 mg, 0.17 mmol, 3.0 equiv) was added and stirred at room temperature for 4 h. The mixture was acidified with 1 M HCl to pH < 3, extracted with EA, concentrated, and used directly.</div><div class="NLM_p">The crude product was dissolved in DMF (1.5 mL) at room temperature. Then, ethane-1,2-diamine (2.1 mg, 0.035 mmol, 0.7 equiv), DIEA (25.8 mg, 0.2 mmol, 4.0 equiv), DMAP (6 mg, 0.05 mmol, 1.0 equiv), and DCC (31.0 mg, 0.15 mmol, 3.0 equiv) were added. After the addition was complete, the mixture was stirred at room temperature for 14 h. The mixture was concentrated and purified by pre-HPLC (Anal: XBridge C18, H<sub>2</sub>O (0.1% NH<sub>4</sub>OAC)/ACN (70–75% in 10 min), retention time: 6.72 min) to get the target product <b>DBA-33</b> as a white solid (5.9 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 3.75–3.63 (m, 2H), 3.44 (q, <i>J</i> = 7.8, 6.6 Hz, 2H), 3.33 (d, <i>J</i> = 12.7 Hz, 3H), 3.28 (d, <i>J</i> = 8.8 Hz, 1H), 3.12–2.95 (m, 4H), 2.37–2.20 (m, 5H), 2.15–0.93 (m, 62H), 0.72 (s, 6H). LRMS (ESI) calcd for [M + H]<sup>+</sup> 918.7, found 919.1 (<a class="ref internalNav" href="#sch34" aria-label="Scheme 34">Scheme 34</a>).</div><figure id="sch34" data-index="39" class="article__inlineFigure"><h2 class="fig-label">Scheme 34</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0040.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0040.jpeg" id="gr34" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 34. Synthesis of <b>DBA-34</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0040.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch34"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, DBA-12-D (45.0 mg, 0.05 mmol, 1.0 equiv) was dissolved in THF (1.0 mL)/MeOH (0.2 mL)/H<sub>2</sub>O (1.0 mL) at room temperature. Then, LiOH·H<sub>2</sub>O (7.2 mg, 0.17 mmol, 3.0 equiv) was added and stirred at room temperature for 4 h. The mixture was acidified with 1 M HCl to pH < 3, extracted with EA, concentrated, and used directly. The crude product was dissolved in DMF (1.5 mL) at room temperature. Then, propane-1,3-diamine (2.6 mg, 0.035 mmol, 0.7 equiv), DIEA (25.8 mg, 0.2 mmol, 4.0 equiv), DMAP (6 mg, 0.05 mmol, 1.0 equiv), and EDC (29.5 mg, 0.15 mmol, 3.0 equiv) were added. After the addition was complete, the mixture was stirred at room temperature for 14 h. The mixture was concentrated and purified by pre-HPLC (Anal: XBridge C18, H<sub>2</sub>O (0.1% NH<sub>4</sub>OAC)/MeOH (80–100% in 10 min), retention time: 7.12 min) to get the target product <b>DBA-34</b> as a white solid (3.0 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 3.58–3.54 (m, 2H), 3.48–3.43 (m, 2H), 3.23–3.16 (m, 4H), 2.31–2.00 (m, 14H), 1.97–1.72 (m, 15H), 1.71–0.94 (m, 43H), 0.73 (s, 6H). LRMS (ESI) calcd for [M + H]<sup>+</sup> 917.7, found 918.1 (<a class="ref internalNav" href="#sch35" aria-label="Scheme 35">Scheme 35</a>).</div><figure id="sch35" data-index="40" class="article__inlineFigure"><h2 class="fig-label">Scheme 35</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0041.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0041.jpeg" id="gr35" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 35. Synthesis of <b>DBA-35</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0041.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch35"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, DBA-20-A (20.0 mg, 0.023 mmol, 1.0 equiv) was dissolved in THF (1.0 mL)/MeOH (0.2 mL)/H<sub>2</sub>O (1.0 mL) at room temperature. Then, LiOH·H<sub>2</sub>O (4.2 mg, 0.114 mmol, 5.0 equiv) was added and stirred at room temperature for 4 h. The mixture was acidified with 1 M HCl to pH < 3, extracted with EA, concentrated, and used directly.</div><div class="NLM_p">The crude product was dissolved in DMF (1.5 mL) at room temperature. Then, ethane-1,2-diamine (1.4 mg, 0.023 mmol, 1.0 equiv), DIEA (12.3 mg, 0.096 mmol, 4.0 equiv), DMAP (3 mg, 0.024 mmol, 1.0 equiv), and EDC (13.6 mg, 0.71 mmol, 3.0 equiv) were added. After the addition was complete, the mixture was stirred at room temperature for 14 h. The mixture was concentrated and purified by pre-HPLC (Anal: XBridge C18, H<sub>2</sub>O (0.1% NH<sub>4</sub>OAC)/ACN (10–95% in 10 min), retention time: 8.45 min) to get the target product <b>DBA-35</b> as a white solid (2.4 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.60 (s, 1H), 6.49 (s, 1H), 6.16 (s, 1H), 4.69 (s, 2H), 4.62–4.49 (m, 1H), 3.64–3.35 (m, 5H), 3.28 (d, <i>J</i> = 11.2 Hz, 2H), 2.30–1.06–0.91 (m, 64H), 0.69 (d, <i>J</i> = 7.5 Hz, 6H). LRMS (ESI) calcd for [M – H]<sup>−</sup> 870.6, found 870.5 (<a class="ref internalNav" href="#sch36" aria-label="Scheme 36">Scheme 36</a>).</div><figure id="sch36" data-index="41" class="article__inlineFigure"><h2 class="fig-label">Scheme 36</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0042.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0042.jpeg" id="gr36" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 36. Synthesis of <b>DBA-36</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0042.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch36"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, DBA-20-A (20.0 mg, 0.023 mmol, 1.0 equiv) was dissolved in THF (1.0 mL)/MeOH (0.2 mL)/H<sub>2</sub>O (1.0 mL) at room temperature. Then, LiOH·H<sub>2</sub>O (4.2 mg, 0.114 mmol, 5.0 equiv) was added and stirred at room temperature for 4 h. The mixture was acidified with 1 M HCl to pH < 3, extracted with EA, concentrated, and used directly.</div><div class="NLM_p">The crude product was dissolved in DMF (1.5 mL) at room temperature. Then, propane-1,3-diamine (1.7 mg, 0.023 mmol, 1.0 equiv), DIEA (12.3 mg, 0.096 mmol, 4.0 equiv), DMAP (3 mg, 0.024 mmol, 1.0 equiv), and EDC (13.6 mg, 0.71 mmol, 3.0 equiv) were added. After the addition was complete, the mixture was stirred at room temperature for 14 h. The mixture was purified by pre-HPLC (Anal: XBridge C18, H<sub>2</sub>O (0.1% NH<sub>4</sub>OAC)/ACN (10–95% in 10 min), retention time: 5.36 min) to get the target product <b>DBA-36</b> as a white solid (2.3 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 7.98 (s, 1H), 4.70–4.50 (m, 4H), 3.72 (q, <i>J</i> = 6.6 Hz, 1H), 3.54–3.40 (m, 3H), 3.25–3.08 (m, 4H), 2.18–0.93 (m, 64H), 0.74 (d, <i>J</i> = 5.6 Hz, 6H). LRMS (ESI) calcd for [M – H]<sup>+</sup> 886.7, found 887.1 (<a class="ref internalNav" href="#sch37" aria-label="Scheme 37">Scheme 37</a>).</div><figure id="sch37" data-index="42" class="article__inlineFigure"><h2 class="fig-label">Scheme 37</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0043.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0043.jpeg" id="gr37" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 37. Synthesis of <b>DBA-37</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0043.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch37"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, DBA-20-A (20.0 mg, 0.023 mmol, 1.0 equiv) was dissolved in THF (1.0 mL)/MeOH (0.2 mL)/H<sub>2</sub>O (1.0 mL) at room temperature. Then, LiOH·H<sub>2</sub>O (4.2 mg, 0.114 mmol, 5.0 equiv) was added and stirred at room temperature for 4 h. The mixture was acidified with 1 M HCl to pH < 3, extracted with EA, concentrated, and used directly.</div><div class="NLM_p">The crude product was dissolved in DMF (1.5 mL) at room temperature. Then, butane-1,4-diamine (2.0 mg, 0.023 mmol, 1.0 equiv), DIEA (12.3 mg, 0.096 mmol, 4.0 equiv), DMAP (3 mg, 0.024 mmol, 1.0 equiv), and EDC (13.6 mg, 0.71 mmol, 3.0 equiv) were added. After the addition was complete, the mixture was stirred at room temperature for 14 h. The mixture was purified by pre-HPLC (Anal: XBridge C18, H<sub>2</sub>O (0.1% NH<sub>4</sub>OAC)/ACN (10–95% in 10 min), retention time: 5.44 min) to get the target product <b>DBA-37</b> as a white solid (3.2 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 8.07 (s, 1H), 4.66–4.49 (m, 4H), 3.73 (p, <i>J</i> = 6.6 Hz, 1H), 3.55–3.39 (m, 4H), 3.27–3.07 (m, 5H), 2.31–0.95 (m, 64H), 0.73 (d, <i>J</i> = 6.2 Hz, 6H). LRMS (ESI) calcd for [M + H]<sup>+</sup> 900.7, found 901.1 (<a class="ref internalNav" href="#sch38" aria-label="Scheme 38">Scheme 38</a>).</div><figure id="sch38" data-index="43" class="article__inlineFigure"><h2 class="fig-label">Scheme 38</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0044.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0044.jpeg" id="gr38" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 38. Synthesis of <b>DBA-38</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0044.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch38"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, DBA-20-A (20.0 mg, 0.023 mmol, 1.0 equiv) was dissolved in THF (1.0 mL)/MeOH (0.2 mL)/H<sub>2</sub>O (1.0 mL) at room temperature. Then, LiOH·H<sub>2</sub>O (4.2 mg, 0.114 mmol, 5.0 equiv) was added and stirred at room temperature for 4 h. The mixture was acidified with 1 M HCl to pH < 3, extracted with EA, concentrated, and used directly.</div><div class="NLM_p">The crude product was dissolved in DMF (1.5 mL) at room temperature. Then, methyl 2,3-diaminopropanoate (2.7 mg, 0.023 mmol, 1.0 equiv), DIEA (12.3 mg, 0.096 mmol, 4.0 equiv), DMAP (3 mg, 0.024 mmol, 1.0 equiv), and EDC (13.6 mg, 0.71 mmol, 3.0 equiv) were added. After the addition was complete, the mixture was stirred at room temperature for 14 h. The mixture was washed with water, extracted with EA, concentrated, and redissolved in THF (1.0 mL)/MeOH (0.2 mL)/H<sub>2</sub>O (1.0 mL) at room temperature. Then, LiOH·H<sub>2</sub>O (4.2 mg, 0.114 mmol, 5.0 equiv) was added and stirred at room temperature for 4 h. The mixture was acidified with 1 M HCl to pH < 3, extracted with EA, concentrated, and purified by pre-HPLC (Anal: XBridge C18, H<sub>2</sub>O (0.1% NH<sub>4</sub>OAC)/ACN (10–95% in 10 min), retention time: 6.77 min) to get the target product <b>DBA-38</b> as a white solid (3.1 mg, purity is 96%), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 8.16–8.12 (m, 1H), 4.73–4.49 (m, 4H), 4.41–4.33 (m, 1H), 3.65–3.36 (m, 6H), 2.34–0.94 (m, 62H), 0.82–0.65 (m, 6H). LRMS (ESI) calcd for [M – H]<sup>−</sup> 915.3, found 915.1 (<a class="ref internalNav" href="#sch39" aria-label="Scheme 39">Scheme 39</a>).</div><figure id="sch39" data-index="44" class="article__inlineFigure"><h2 class="fig-label">Scheme 39</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0045.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0045.jpeg" id="gr39" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 39. Synthesis of <b>DBA-39</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0045.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch39"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, DBA-20-A (20.0 mg, 0.023 mmol, 1.0 equiv) was dissolved in THF (1.0 mL)/MeOH (0.2 mL)/H<sub>2</sub>O (1.0 mL) at room temperature. Then, LiOH·H<sub>2</sub>O (4.2 mg, 0.114 mmol, 5.0 equiv) was added and stirred at room temperature for 4 h. The mixture was acidified with 1 M HCl to pH < 3, extracted with EA, concentrated, and used directly.</div><div class="NLM_p">The crude product was dissolved in DMF (1.5 mL) at room temperature. Then, pentane-1,5-diamine (2.5 mg, 0.023 mmol, 1.0 equiv), DIEA (12.3 mg, 0.096 mmol, 4.0 equiv), DMAP (3 mg, 0.024 mmol, 1.0 equiv), and EDC (13.6 mg, 0.71 mmol, 3.0 equiv) were added. After the addition was complete, the mixture was stirred at room temperature for 14 h. The mixture was purified by pre-HPLC (Anal: XBridge C18, H<sub>2</sub>O (0.1% NH<sub>4</sub>OAC)/ACN (10–95% in 10 min), retention time: 6.16 min) to get the target product <b>DBA-39</b> as a white solid (5.3 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 8.08 (s, 1H), 4.65–4.51 (m, 6H), 3.56–3.38 (m, 4H), 3.28–3.05 (m, 5H), 2.30–2.06 (m, 6H), 1.97–0.94 (m, 59H), 0.73 (d, <i>J</i> = 7.1 Hz, 6H). LRMS (ESI) calcd for [M – H]<sup>−</sup> 912.7, found 912.6 (<a class="ref internalNav" href="#sch40" aria-label="Scheme 40">Scheme 40</a>).</div><figure id="sch40" data-index="45" class="article__inlineFigure"><h2 class="fig-label">Scheme 40</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0046.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0046.jpeg" id="gr40" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 40. Synthesis of <b>DBA-40</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0046.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch40"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, DBA-20-A (20.0 mg, 0.023 mmol, 1.0 equiv) was dissolved in THF (1.0 mL)/MeOH (0.2 mL)/H<sub>2</sub>O (1.0 mL) at room temperature. Then, LiOH·H<sub>2</sub>O (4.2 mg, 0.114 mmol, 5.0 equiv) was added and stirred at room temperature for 4 h. The mixture was acidified with 1 M HCl to pH < 3, extracted with EA, concentrated, and used directly.</div><div class="NLM_p">The crude product was dissolved in DMF (1.5 mL) at room temperature. Then, hexane-1,6-diamine (2.7 mg, 0.023 mmol, 1.0 equiv), DIEA (12.3 mg, 0.096 mmol, 4.0 equiv), DMAP (3 mg, 0.024 mmol, 1.0 equiv), and EDC (13.6 mg, 0.71 mmol, 3.0 equiv) were added. After the addition was complete, the mixture was stirred at room temperature for 14 h. The mixture was purified by pre-HPLC (Anal: XBridge C18, H<sub>2</sub>O (0.1% NH<sub>4</sub>OAC)/ACN (10–95% in 10 min), retention time: 6.25 min) to get the target product <b>DBA-40</b> as a white solid (10.8 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 8.09 (s, 1H), 4.63 (s, 2H), 4.59–4.50 (m, 1H), 3.58–3.40 (m, 4H), 3.26–3.06 (m, 5H), 2.28–2.19 (m, 2H), 2.08–0.94 (m, 68H), 0.73 (d, <i>J</i> = 7.3 Hz, 6H). LRMS (ESI) calcd for [M – H]<sup>−</sup> 926.7, found 926.4 (<a class="ref internalNav" href="#sch41" aria-label="Scheme 41">Scheme 41</a>).</div><figure id="sch41" data-index="46" class="article__inlineFigure"><h2 class="fig-label">Scheme 41</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0047.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0047.jpeg" id="gr41" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 41. Synthesis of <b>DBA-41</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0047.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch41"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, DBA-38 (7.0 mg, 0.008 mmol, 1.0 equiv) was dissolved in dry DMF (1.0 mL) at room temperature. Then, 2-aminoethane-1-sulfonic acid (7.5 mg, 0.06 mmol, 8.0 equiv), HATU (7.3 mg, 0.02 mmol, 2.4 equiv), and DIEA (3.1 mg, 0.02 mmol, 3.0 equiv) were added at room temperature. After the addition was complete, the mixture was stirred at room temperature for 14 h. The mixture was concentrated and purified by pre-HPLC (Anal: XBridge C18, H<sub>2</sub>O (0.1% NH<sub>4</sub>OAC)/ACN (15–60% in 10 min), retention time: 4.31 min) to get the target product as a white solid <b>DBA-41</b> (2.7 mg, purity is 97%), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 8.15–8.12 (m, 1H), 4.69–4.41 (m, 6H), 3.69–3.38 (m, 8H), 2.99 (d, <i>J</i> = 7.1 Hz, 2H), 2.32–0.95 (m, 62H), 0.80–0.67 (m, 6H). LRMS (ESI) calcd for [M – H]<sup>−</sup> 1022.4, found 1022.0 (<a class="ref internalNav" href="#sch42" aria-label="Scheme 42">Scheme 42</a>).</div><figure id="sch42" data-index="47" class="article__inlineFigure"><h2 class="fig-label">Scheme 42</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0048.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0048.jpeg" id="gr42" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 42. Synthesis of <b>NQL-064</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0048.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch42"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, DBA-12-C (24.3 mg, 0.06 mmol, 1.5 equiv) was dissolved in MeCN (3 mL) at room temperature. Then, 7-fluoro-<i>N</i>,<i>N</i>-dimethylbenzo[<i>c</i>][1,2,5]oxadiazole-4-sulfonamide (10.0 mg, 0.04 mmol, 1.0 equiv) and K<sub>2</sub>CO<sub>3</sub> (11.4 mg, 0.08 mmol, 2.0 equiv) were added. After the addition was complete, the mixture was stirred at room temperature for 0.5 h. The mixture was washed with water, extracted with EA, concentrated, and purified by silica column chromatography (DCM:MeOH = 20:1) to get the target product NQL-064-A (14 mg). <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>Cl): δ 7.84 (d, <i>J</i> = 8.3 Hz, 1H), 6.11 (d, <i>J</i> = 8.3 Hz, 1H), 3.67 (s, 3H), 3.48–3.47 (m, 1H), 2.96 (s, 3H), 2.87 (s, 3H), 2.42–1.01 (m, 27H), 1.02 (s, 3H), 0.92 (d, <i>J</i> = 6.4 Hz, 3H), 0.68 (s, 3H).</div><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, NQL-064-A (17.0 mg, 0.027 mmol, 1.0 equiv) was dissolved in THF (1.0 mL)/MeOH (0.2 mL)/H<sub>2</sub>O (1.0 mL) at room temperature. Then, LiOH·H<sub>2</sub>O (3.4 mg, 0.081 mmol, 3.0 equiv) was added and stirred at room temperature for 4 h. The mixture was acidified with 1 M HCl to pH < 3, extracted with EA, and concentrated to get a white solid, which was used directly. The solid was dissolved in DMF (2.0 mL) at room temperature. Then, 2-aminoethanesulfonic acid (10.2 mg, 0.081 mmol, 3.0 equiv), DIEA (14 mg, 0.108 mmol, 4.0 equiv), DMAP (1.6 mg, 0.0135 mmol, 0.5 equiv), and EDC (10.3 mg, 0.054 mmol, 2.0 equiv) were added. After the addition was complete, the mixture was stirred at room temperature for 14 h. The mixture was concentrated and purified by pre-HPLC (Anal: XBridge C18, H<sub>2</sub>O (0.1% NH<sub>4</sub>OAC)/MeCN (30–60% in 10 min), retention time: 5.39 min) to get the target product <b>NQL-064</b> as a white solid (5.1 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 7.86 (d, <i>J</i> = 8.3 Hz, 1H), 6.31 (d, <i>J</i> = 8.3 Hz, 1H), 3.70 (s, 1H), 3.59 (t, <i>J</i> = 6.8 Hz, 2H), 3.51 (s, 1H), 2.96 (t, <i>J</i> = 6.8 Hz, 2H), 2.79 (s, 6H), 2.24–1.73 (m, 12H), 1. 73–1.06 (m, 16H), 1.05 (s, 3H), 0.98 (d, <i>J</i> = 6.3 Hz, 3H), 0.73 (s, 3H). LRMS (ESI) calcd for [M – H]<sup>−</sup> 722.9, found 722.9 (<a class="ref internalNav" href="#sch43" aria-label="Scheme 43">Scheme 43</a>).</div><figure id="sch43" data-index="48" class="article__inlineFigure"><h2 class="fig-label">Scheme 43</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0049.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0049.jpeg" id="gr43" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 43. Synthesis of <b>NQL-065</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0049.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch43"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, cholic acid (917 mg, 2.0 mmol, 1.0 equiv) was dissolved in HCOOH (1.5 mL) at room temperature. Then, two drops of HClO<sub>4</sub> were added. After the addition was complete, the mixture was stirred at 60 °C for 3.0 h. The mixture was cooled to 40 °C and Ac<sub>2</sub>O (2 mL) was added dropwise. Then, the mixture was slowly poured into ice-cold water and washed with water until the washings were neutral. The solid was dissolved in EA and crystallized by addition of hexane to give the target product <b>NQL-65-A</b> (1.01 g). <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>Cl): 8.16 (s, 1H), 8.10 (s, 1H), 8.02 (s, 1H), 5.29 (s, 1H), 5.06 (s, 1H), 4.76–4.64 (m, 1H), 2.40–0.99 (m, 27H), 0.94 (s, 3H). 0.82 (d, <i>J</i> = 6.8 Hz, 3H), 0.76 (s, 3H).</div><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, potassium 3-ethoxy-3-oxopropanoate (357 mg, 2.1 mmol, 2.1 equiv) was dissolved in MeCN (2.0 mL). Then, MgCl<sub>2</sub> (237 mg, 2.5 mmol, 2.5 equiv) and Et<sub>3</sub>N (323 mg, 3.2 mmol, 3.2 equiv) were added and stirred at room temperature for 2.5 h. In the meanwhile, NQL-65-A (492 mg, 1.0 mmol, 1.0 equiv) and CDI (194 mg, 1.2 mmol, 1.2 equiv) dissolved in MeCN (3.0 mL) were stirred at room temperature for 2.5 h. Then, the two components were mixed together and stirred at room temperature for 26 h. The mixture was concentrated and purified by silica column chromatography (PE:EA = 4:1) to get the target product <b>NQL-65-B</b> (330 mg). <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>Cl): 8.16 (s, 1H), 8.10 (s, 1H), 8.02 (s, 1H), 5.29 (s, 1H), 5.06 (s, 1H), 4.76–4.64 (m, 1H), 4.18 (q, <i>J</i> = 6.4 Hz, 1H), 3.42 (s, 2H), 2.62–2.38 (m, 2H), 2.18–1.01 (m, 26H), 0.94 (s, 3H). 0.82 (d, <i>J</i> = 6.8 Hz, 3H), 0.76 (s, 3H).</div><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, NQL-65-B (56.2 mg, 0.1 mmol, 1.0 equiv) was dissolved in MeOH (2.0 mL). Then, CH<sub>3</sub>COONH<sub>4</sub> (77 mg, 10.0 mmol, 1.0 equiv), NaBH<sub>3</sub>CN (12.4 mg, 0.2 mmol, 2 equiv), and AcOH (cat) were added and stirred at 80 °C for 14 h. The mixture was concentrated and purified by silica column chromatography (DCM:MeOH = 4:1) to get a crude product mixture (35 mg).</div><div class="NLM_p">In a 10 mL, one-necked, round-bottomed flask, NQL-65-C (crude, 0.05 mmol, 1.0 equiv) was dissolved in MeCN (2 mL) at room temperature. Then, 7-fluoro-<i>N</i>,<i>N</i>-dimethylbenzo[<i>c</i>][1,2,5]oxadiazole-4-sulfonamide (10.0 mg, 0.04 mmol, 0.8 equiv) and K<sub>2</sub>CO<sub>3</sub> (13.8.4 mg, 0.1 mmol, 2.0 equiv) were added. After the addition was complete, the mixture was stirred at room temperature for 0.5 h. The mixture was washed with water, extracted with EA, concentrated, and purified by silica gel column chromatography (DCM:MeOH = 20:1) to get the target product <b>NQL-65-D</b> (32 mg).</div><div class="NLM_p last">In a 10 mL, one-necked, round-bottomed flask, NQL-65-D (32.0 mg, 0.04 mmol, 1.0 equiv) was dissolved in THF (1.0 mL)/MeOH (0.2 mL)/H<sub>2</sub>O (1.0 mL) at room temperature. Then, LiOH·H<sub>2</sub>O (5.0 mg, 0.12 mmol, 3.0 equiv) was added and stirred at room temperature for 4 h. The mixture was acidified with 1 M HCl to pH < 3, extracted with EA, and concentrated to get a solid, which was used directly. The solid was dissolved in DMF (2.0 mL) at room temperature. Then, 2-aminoethanesulfonic acid (15 mg, 0.12 mmol, 3.0 equiv), DIEA (2.06 mg, 0.16 mmol, 4.0 equiv), DMAP (2.4 mg, 0.02 mmol, 0.5 equiv), and EDC (15.8 mg, 0.08 mmol, 2.0 equiv) were added. After the addition was complete, the mixture was stirred at room temperature for 14 h. The mixture was concentrated and purified by pre-HPLC (Anal: XBridge C18, H<sub>2</sub>O (0.1% NH<sub>4</sub>OAC)/MeCN (30–60% in 10 min), retention time: 5.39 min) to get the target product <b>NQL-65</b> as a white solid (2.0 mg), which was confirmed by NMR. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): 7.88–7.86 (m, 1H), 7.43–7.39 (m, 1H), 4.21 (br, 1H), 3.93 (br, 1H), 3.76 (br, 1H), 3.60–3.52 (m, 2H), 2.90–2.86 (m, 2H), 2.82 (s, 6H), 2.60–2.51 (m, 1H), 2.28–2.18 (m, 2H), 1.96–1.74 (m, 8H), 1.74–0.96 (m, 21H), 0.91–0.89 (m, 3H), 0.66–0.64 (m, 3H). LRMS (ESI) calcd for [M – H]<sup>−</sup> 783.0, found 782.9.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i68"><a href="/doi/suppl/10.1021/acs.jmedchem.1c00078" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81715" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81715" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.1c00078?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00078</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">LC–MS result of commercial TUDCA and synthesized dimeric TUDCA, HBV infection inhibition by selected DBADs, cytotoxicity analysis of selected DBADs on HepG2-NTCP cells and PTHs, surface and total expression of NTCP after DBAD treatment, NTCP-dependent uptake of fluorescence bile acids NQL-064 and NQL-065 by HepG2-NTCP cells, fluorescence-labeled bile acid uptake inhibition assay, <sup>3</sup>H-TCA uptake inhibition by selected DBADs, persistent inhibition of FITC-pre-S1 peptide binding by DBADs, persistent inhibition of fluorescence bile acid NQL-064 uptake by DBADs, inhibitory effect of DBADs on FITC-pre-S1 peptide binding to NTCPs of different species, time course of DBAD inhibition of 3H-TCA uptake in HepG2-NTCP cells, and ALT and body weight surveillance during DBA-41 treatment (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00078/suppl_file/jm1c00078_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00078/suppl_file/jm1c00078_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00078/suppl_file/jm1c00078_si_001.pdf">jm1c00078_si_001.pdf (1.29 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00078/suppl_file/jm1c00078_si_002.csv">jm1c00078_si_002.csv (9.28 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.1c00078" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_85656" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_85656" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wenhui Li</span> - <span class="hlFld-Affiliation affiliation">National
Institute of Biological Sciences, Beijing 102206, China</span>; 
    <span class="hlFld-Affiliation affiliation">Tsinghua
Institute of Multidisciplinary Biomedical Research, Tsinghua University, Beijing 100084, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#5b37322c3e35332e321b35323928753a38753835"><span class="__cf_email__" data-cfemail="a8c4c1dfcdc6c0ddc1e8c6c1cadb86c9cb86cbc6">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiangbing Qi</span> - <span class="hlFld-Affiliation affiliation">National
Institute of Biological Sciences, Beijing 102206, China</span>; 
    <span class="hlFld-Affiliation affiliation">Tsinghua
Institute of Multidisciplinary Biomedical Research, Tsinghua University, Beijing 100084, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7139-5164" title="Orcid link">http://orcid.org/0000-0002-7139-5164</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#d3a2baabbab2bdb4b1babdb493bdbab1a0fdb2b0fdb0bd"><span class="__cf_email__" data-cfemail="e2938b9a8b838c85808b8c85a28c8b8091cc8381cc818c">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yang Liu</span> - <span class="hlFld-Affiliation affiliation">National
Institute of Biological Sciences, Beijing 102206, China</span>; 
    <span class="hlFld-Affiliation affiliation">Graduate
Program, Tsinghua University, Beijing 100084, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lei Zhang</span> - <span class="hlFld-Affiliation affiliation">National
Institute of Biological Sciences, Beijing 102206, China</span>; 
    <span class="hlFld-Affiliation affiliation">College
of Life Sciences, Beijing Normal University, Beijing 100875, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Huan Yan</span> - <span class="hlFld-Affiliation affiliation">National
Institute of Biological Sciences, Beijing 102206, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhiqiang Wang</span> - <span class="hlFld-Affiliation affiliation">National
Institute of Biological Sciences, Beijing 102206, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Guoliang Sun</span> - <span class="hlFld-Affiliation affiliation">National
Institute of Biological Sciences, Beijing 102206, China</span>; 
    <span class="hlFld-Affiliation affiliation">Graduate
Program, Peking University, Beijing 100080, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiao Song</span> - <span class="hlFld-Affiliation affiliation">National
Institute of Biological Sciences, Beijing 102206, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhongmin Zhou</span> - <span class="hlFld-Affiliation affiliation">National
Institute of Biological Sciences, Beijing 102206, China</span>; 
    <span class="hlFld-Affiliation affiliation">College
of Life Sciences, Beijing Normal University, Beijing 100875, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bo Peng</span> - <span class="hlFld-Affiliation affiliation">National
Institute of Biological Sciences, Beijing 102206, China</span>; 
    <span class="hlFld-Affiliation affiliation">Graduate
Program, Peking University, Beijing 100080, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Liwei Yan</span> - <span class="hlFld-Affiliation affiliation">National
Institute of Biological Sciences, Beijing 102206, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Qingcui Wu</span> - <span class="hlFld-Affiliation affiliation">National
Institute of Biological Sciences, Beijing 102206, China</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>Y.L., L.Z., H.Y., and Z.W. contributed equally to this work.</p><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through the contribution of all authors. All authors have given approval to the final version of the manuscript. X.Q. and W.L. conceived the project; X.Q., Z.W., and L.Z. designed; Z.W., L.Z., and X.S. synthesized all of the compounds; W.L., Y.L., and H.Y. designed, Y.L. and H.Y. conducted the biological experiments in vitro; Y.L. conducted the pharmacological experiments in vitro and in vivo; G.S. helped with biological experiments in vitro; Z.Z. helped with mice experiments in vivo; B.P. provided the HepG2-NTCP cell line; L.Y. produced HBV and HDV inocula for antiviral assays; Q.W. conducted HPLC-MS analysis; and Y.L., H.Y., and X.Q. drafted the manuscript.</p></li><li><span class="author-information-subsection-header">Funding</span><p>This project is funded by the National Natural Science Foundation of China to W.L. (81525018), the Science and Technology Major Project of Beijing to W.L. (D171100003117003), the Beijing Scholar Project to W.L., the Science and Technology Bureau of Beijing Municipal Government to W.L., and the Chinese Ministry of Science and Technology 973 Program grant (2014CB849603) to X.Q.</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): The authors (WL, XQ, LZ, ZW, HY, YL, BP, QW) declare a patent application based on this study (WIPO publication no. WO2016045642A1, Polymeric bile acid derivatives inhibit hepatitis b and d virus and ntcp transport).<br /></br><p class="inlineNote">The authors (W.L., X.Q., L.Z., Z.W., H.Y., Y.L., B.P., and Q.W.) declare a patent application based on this study (WIPO publication no. WO2016045642A1, polymeric bile acid derivatives inhibit hepatitis B and D viruses and NTCP transport).</p></div></li></ul></div><div class="ack" id="ACK-d7e3648-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i72">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67506" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67506" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors thank Tsinghua University and Peking University for Ph.D. program support. We gratefully acknowledge the Beijing Municipal Government and Tsinghua University for their financial support.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">HBV</td><td class="NLM_def"><p class="first last">hepatitis B virus</p></td></tr><tr><td class="NLM_term">HDV</td><td class="NLM_def"><p class="first last">hepatitis D virus</p></td></tr><tr><td class="NLM_term">NTCP</td><td class="NLM_def"><p class="first last">sodium taurocholate cotransporting peptide</p></td></tr><tr><td class="NLM_term">DBAD</td><td class="NLM_def"><p class="first last">dimeric bile acid derivative</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">TCA</td><td class="NLM_def"><p class="first last">taurocholic acid</p></td></tr><tr><td class="NLM_term">TUDCA</td><td class="NLM_def"><p class="first last">tauroursodeoxycholic acid</p></td></tr><tr><td class="NLM_term">UDCA</td><td class="NLM_def"><p class="first last">ursodeoxycholic acid</p></td></tr><tr><td class="NLM_term">TLCA</td><td class="NLM_def"><p class="first last">taurolithocholic acid</p></td></tr><tr><td class="NLM_term">FITC</td><td class="NLM_def"><p class="first last">fluorescein isothiocyanate</p></td></tr><tr><td class="NLM_term">ASBT</td><td class="NLM_def"><p class="first last">apical sodium-dependent bile acid transporter</p></td></tr><tr><td class="NLM_term">OATP</td><td class="NLM_def"><p class="first last">organic anion-transporting polypeptide</p></td></tr><tr><td class="NLM_term">FBS</td><td class="NLM_def"><p class="first last">fetal bovine serum</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i74">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30706" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30706" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 26 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Razavi-Shearer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gamkrelidze, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, M. H.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Global prevalence, treatment, and prevention of hepatitis B virus infection in 2016: a modelling study</span>. <i>Lancet Gastroenterol. Hepatol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">383</span>– <span class="NLM_lpage">403</span>, <span class="refDoi"> DOI: 10.1016/S2468-1253(18)30056-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1016%2FS2468-1253%2818%2930056-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=29599078" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2018&pages=383-403&author=D.+Razavi-Shearerauthor=I.+Gamkrelidzeauthor=M.+H.+Nguyen&title=Global+prevalence%2C+treatment%2C+and+prevention+of+hepatitis+B+virus+infection+in+2016%3A+a+modelling+study&doi=10.1016%2FS2468-1253%2818%2930056-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2FS2468-1253%2818%2930056-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS2468-1253%252818%252930056-6%26sid%3Dliteratum%253Aachs%26aulast%3DRazavi-Shearer%26aufirst%3DD.%26aulast%3DGamkrelidze%26aufirst%3DI.%26aulast%3DNguyen%26aufirst%3DM.%2BH.%26atitle%3DGlobal%2520prevalence%252C%2520treatment%252C%2520and%2520prevention%2520of%2520hepatitis%2520B%2520virus%2520infection%2520in%25202016%253A%2520a%2520modelling%2520study%26jtitle%3DLancet%2520Gastroenterol.%2520Hepatol.%26date%3D2018%26volume%3D3%26spage%3D383%26epage%3D403%26doi%3D10.1016%2FS2468-1253%2818%2930056-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beasley, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chien, C. S.</span></span> <span> </span><span class="NLM_article-title">Hepatocellular carcinoma and hepatitis B virus. A prospective study of 22 707 men in Taiwan</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">1981</span>,  <span class="NLM_volume"><i>318</i></span>,  <span class="NLM_fpage">1129</span>– <span class="NLM_lpage">1133</span>, <span class="refDoi"> DOI: 10.1016/S0140-6736(81)90585-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1016%2FS0140-6736%2881%2990585-7" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=318&publication_year=1981&pages=1129-1133&author=R.+P.+Beasleyauthor=L.+Y.+Hwangauthor=C.+C.+Linauthor=C.+S.+Chien&title=Hepatocellular+carcinoma+and+hepatitis+B+virus.+A+prospective+study+of+22+707+men+in+Taiwan&doi=10.1016%2FS0140-6736%2881%2990585-7"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2881%2990585-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252881%252990585-7%26sid%3Dliteratum%253Aachs%26aulast%3DBeasley%26aufirst%3DR.%2BP.%26aulast%3DHwang%26aufirst%3DL.%2BY.%26aulast%3DLin%26aufirst%3DC.%2BC.%26aulast%3DChien%26aufirst%3DC.%2BS.%26atitle%3DHepatocellular%2520carcinoma%2520and%2520hepatitis%2520B%2520virus.%2520A%2520prospective%2520study%2520of%252022%2520707%2520men%2520in%2520Taiwan%26jtitle%3DLancet%26date%3D1981%26volume%3D318%26spage%3D1129%26epage%3D1133%26doi%3D10.1016%2FS0140-6736%2881%2990585-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jen, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, G. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iloeje, U. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Group, R.-H. S.</span></span> <span> </span><span class="NLM_article-title">Risk of hepatocellular carcinoma across a biological gradient of serum hepatitis B virus DNA level</span>. <i>JAMA</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>295</i></span>,  <span class="NLM_fpage">65</span>– <span class="NLM_lpage">73</span>, <span class="refDoi"> DOI: 10.1001/jama.295.1.65</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1001%2Fjama.295.1.65" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=16391218" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlKqtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=295&publication_year=2006&pages=65-73&author=C.-J.+Chenauthor=H.+I.+Yangauthor=J.+Suauthor=C.+L.+Jenauthor=S.+L.+Youauthor=S.+N.+Luauthor=G.+T.+Huangauthor=U.+H.+Iloejeauthor=R.-H.+S.+Group&title=Risk+of+hepatocellular+carcinoma+across+a+biological+gradient+of+serum+hepatitis+B+virus+DNA+level&doi=10.1001%2Fjama.295.1.65"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Risk of hepatocellular carcinoma across a biological gradient of serum hepatitis B virus DNA level</span></div><div class="casAuthors">Chen, Chien-Jen; Yang, Hwai-I.; Su, Jun; Jen, Chin-Lan; You, San-Lin; Lu, Sheng-Nan; Huang, Guan-Tarn; Iloeje, Uchenna H.</div><div class="citationInfo"><span class="NLM_cas:title">JAMA, the Journal of the American Medical Association</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">295</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">65-73</span>CODEN:
                <span class="NLM_cas:coden">JAMAAP</span>;
        ISSN:<span class="NLM_cas:issn">0098-7484</span>.
    
            (<span class="NLM_cas:orgname">American Medical Association</span>)
        </div><div class="casAbstract">Serum hepatitis B virus (HBV) DNA level is a marker of viral replication and efficacy of antiviral treatment in individuals with chronic hepatitis B.  To evaluate the relationship between serum HBV DNA level and risk of hepatocellular carcinoma.  Prospective cohort study of 3653 participants (aged 30-65 years), who were seropos. for the hepatitis B surface antigen and seroneg. for antibodies against the hepatitis C virus, recruited to a community-based cancer screening program in Taiwan between 1991 and 1992.  Incidence of hepatocellular carcinoma during follow-up examn. and by data linkage with the national cancer registry and the death certification systems.  There were 164 incident cases of hepatocellular carcinoma and 346 deaths during a mean follow-up of 11.4 years and 41 779 person-years of follow-up.  The incidence of hepatocellular carcinoma increased with serum HBV DNA level at study entry in a dose-response relationship ranging from 108 per 100.000 person-years for an HBV DNA level of less than 300 copies/mL to 1152 per 100.000 person-years for an HBV DNA level of 1 million copies/mL or greater.  The corresponding cumulative incidence rates of hepatocellular carcinoma were 1.3% and 14.9%, resp.  The biol. gradient of hepatocellular carcinoma by serum HBV DNA levels remained significant (P < .001) after adjustment for sex, age, cigarette smoking, alc. consumption, serostatus for the hepatitis B e antigen (HBeAg), serum alanine aminotransferase level, and liver cirrhosis at study entry.  The dose-response relationship was most prominent for participants who were seroneg. for HBeAg with normal serum alanine aminotransferase levels and no liver cirrhosis at study entry.  Participants with persistent elevation of serum HBV DNA level during follow-up had the highest hepatocellular carcinoma risk.  Elevated serum HBV DNA level (-10000 copies/mL) is a strong risk predictor of hepatocellular carcinoma independent of HBeAg, serum alanine aminotransferase level, and liver cirrhosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofTLNTbgs5X7Vg90H21EOLACvtfcHk0lhO3d1VD-J49w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlKqtQ%253D%253D&md5=3fb9792cfca230106e13a8c405382ed7</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1001%2Fjama.295.1.65&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1001%252Fjama.295.1.65%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DC.-J.%26aulast%3DYang%26aufirst%3DH.%2BI.%26aulast%3DSu%26aufirst%3DJ.%26aulast%3DJen%26aufirst%3DC.%2BL.%26aulast%3DYou%26aufirst%3DS.%2BL.%26aulast%3DLu%26aufirst%3DS.%2BN.%26aulast%3DHuang%26aufirst%3DG.%2BT.%26aulast%3DIloeje%26aufirst%3DU.%2BH.%26aulast%3DGroup%26aufirst%3DR.-H.%2BS.%26atitle%3DRisk%2520of%2520hepatocellular%2520carcinoma%2520across%2520a%2520biological%2520gradient%2520of%2520serum%2520hepatitis%2520B%2520virus%2520DNA%2520level%26jtitle%3DJAMA%26date%3D2006%26volume%3D295%26spage%3D65%26epage%3D73%26doi%3D10.1001%2Fjama.295.1.65" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sureau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guerra, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanford, R. E.</span></span> <span> </span><span class="NLM_article-title">Role of the large hepatitis B virus envelope protein in infectivity of the hepatitis delta virion</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">366</span>– <span class="NLM_lpage">372</span>, <span class="refDoi"> DOI: 10.1128/JVI.67.1.366-372.1993</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1128%2FJVI.67.1.366-372.1993" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=8416375" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADyaK3sXlsleitA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=1993&pages=366-372&author=C.+Sureauauthor=B.+Guerraauthor=R.+E.+Lanford&title=Role+of+the+large+hepatitis+B+virus+envelope+protein+in+infectivity+of+the+hepatitis+delta+virion&doi=10.1128%2FJVI.67.1.366-372.1993"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Role of the large hepatitis B virus envelope protein in infectivity of the hepatitis delta virion</span></div><div class="casAuthors">Sureau, Camille; Guerra, Bernadette; Lanford, Robert E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">366-72</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    </div><div class="casAbstract">The hepatitis delta virus (HDV) is coated with large (L), middle (M), and small (S) envelope proteins encoded by coinfecting hepatitis B virus (HBV).  To study the role of the HBV envelope proteins in the assembly and infectivity of HDV, we produced three types of recombinant particles in Huh7 cells by transfection with HBV DNA and HDV cDNA: (i) particles with an envelope contg. the S HBV envelope protein only, (ii) particles with an envelope contg. S and M proteins, and (iii) particles with an envelope contg. S, M, and L proteins.  Although the resulting S-, SM-, and SML-HDV particles contained both hepatitis delta antigen and HDV RNA, only particles coated with all three envelope proteins (SML) showed evidence of infectivity in an in vitro culture system susceptible to HDV infection.  We concluded that the L HBV envelope protein, and more specifically the pre- S1 domain, is important for infectivity of HDV particles and that the M protein, which has been reported to bear a site for binding to polymd. albumin in the pre-S2 domain, is not sufficient for infectivity.  Our data also show that the helper HBV is not required for initiation of HDV infection.  The mechanism by which the L protein may affect HDV infectivity is discussed herein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiHc4X4_5QV7Vg90H21EOLACvtfcHk0lgeshE4H89rYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXlsleitA%253D%253D&md5=3e436e69840f20e918c08cfbc7c8418f</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1128%2FJVI.67.1.366-372.1993&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.67.1.366-372.1993%26sid%3Dliteratum%253Aachs%26aulast%3DSureau%26aufirst%3DC.%26aulast%3DGuerra%26aufirst%3DB.%26aulast%3DLanford%26aufirst%3DR.%2BE.%26atitle%3DRole%2520of%2520the%2520large%2520hepatitis%2520B%2520virus%2520envelope%2520protein%2520in%2520infectivity%2520of%2520the%2520hepatitis%2520delta%2520virion%26jtitle%3DJ.%2520Virol.%26date%3D1993%26volume%3D67%26spage%3D366%26epage%3D372%26doi%3D10.1128%2FJVI.67.1.366-372.1993" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sureau, C.</span></span> <span> </span><span class="NLM_article-title">The role of the HBV envelope proteins in the HDV replication cycle</span>. <i>Curr. Top. Microbiol. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>307</i></span>,  <span class="NLM_fpage">113</span>– <span class="NLM_lpage">131</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=16903223" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADC%252BD28XosFeqsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=307&publication_year=2006&pages=113-131&author=C.+Sureau&title=The+role+of+the+HBV+envelope+proteins+in+the+HDV+replication+cycle"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The role of the HBV envelope proteins in the HDV replication cycle</span></div><div class="casAuthors">Sureau, C.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Microbiology and Immunology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">307</span>
        (<span class="NLM_cas:issue">Hepatitis Delta Virus</span>),
    <span class="NLM_cas:pages">113-131</span>CODEN:
                <span class="NLM_cas:coden">CTMIA3</span>;
        ISSN:<span class="NLM_cas:issn">0070-217X</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">A review.  The hepatitis delta virus (HDV) is a subviral agent that utilizes the envelope proteins of the hepatitis B virus (HBV) for propagation.  When introduced into permissive cells, the HDV RNA genome replicates and assocs. with multiple copies of the HDV-encoded proteins to assemble a ribonucleoprotein (RNP) complex.  The mechanism necessary to export the RNP from the cell is provided by the HBV envelope proteins, which have the capacity to assemble lipoprotein vesicles that bud into the lumen of a pre-Golgi compartment before being secreted.  In addn. to allowing the release of the HDV RNP, the HBV envelope proteins also provide a means for its targeting to an uninfected cell, thereby ensuring the spread of HDV.  This chapter covers the mol. aspects of the HBV envelope protein functions in the HDV replication cycle, in particular the activity of the small envelope protein in RNP export and the function of the large envelope protein at viral entry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoEJaSXW9Fg7Vg90H21EOLACvtfcHk0lgeshE4H89rYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XosFeqsro%253D&md5=aff046b7f493724aaaec0288b4173826</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSureau%26aufirst%3DC.%26atitle%3DThe%2520role%2520of%2520the%2520HBV%2520envelope%2520proteins%2520in%2520the%2520HDV%2520replication%2520cycle%26jtitle%3DCurr.%2520Top.%2520Microbiol.%2520Immunol.%26date%3D2006%26volume%3D307%26spage%3D113%26epage%3D131" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jing, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sui, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span> <span> </span><span class="NLM_article-title">Sodium taurocholate cotransporting polypeptide is a functional receptor for human hepatitis B and D virus</span>. <i>Elife</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>1</i></span>, <span class="NLM_elocation-id">e00049</span> <span class="refDoi"> DOI: 10.7554/eLife.00049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.7554%2FeLife.00049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=23150796" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht12ms7nK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2012&author=H.+Yanauthor=G.+Zhongauthor=G.+Xuauthor=W.+Heauthor=Z.+Jingauthor=Z.+Gaoauthor=Y.+Huangauthor=Y.+Qiauthor=B.+Pengauthor=H.+Wangauthor=L.+Fuauthor=M.+Songauthor=P.+Chenauthor=W.+Gaoauthor=B.+Renauthor=Y.+Sunauthor=T.+Caiauthor=X.+Fengauthor=J.+Suiauthor=W.+Li&title=Sodium+taurocholate+cotransporting+polypeptide+is+a+functional+receptor+for+human+hepatitis+B+and+D+virus&doi=10.7554%2FeLife.00049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Sodium taurocholate cotransporting polypeptide is a functional receptor for human hepatitis B and D virus</span></div><div class="casAuthors">Yan, Huan; Zhong, Guocai; Xu, Guangwei; He, Wenhui; Jing, Zhiyi; Gao, Zhenchao; Huang, Yi; Qi, Yonghe; Peng, Bo; Wang, Haimin; Fu, Liran; Song, Mei; Chen, Pan; Gao, Wenqing; Ren, Bijie; Sun, Yinyan; Cai, Tao; Feng, Xiaofeng; Sui, Jianhua; Li, Wenhui</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e00049, 28 pp.</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">Human hepatitis B virus (HBV) infection and HBV-related diseases remain a major public health problem.  Individuals coinfected with its satellite hepatitis D virus (HDV) have more severe disease.  Cellular entry of both viruses is mediated by HBV envelope proteins.  The pre-S1 domain of the large envelope protein is a key determinant for receptor(s) binding.  However, the identity of the receptor(s) is unknown.  Here, by using near zero distance photo-crosslinking and tandem affinity purifn., the authors revealed that the receptor-binding region of pre-S1 specifically interacts with sodium taurocholate cotransporting polypeptide (NTCP), a multiple transmembrane transporter predominantly expressed in the liver.  Silencing NTCP inhibited HBV and HDV infection, while exogenous NTCP expression rendered nonsusceptible hepatocarcinoma cells susceptible to these viral infections.  Moreover, replacing amino acids 157-165 of nonfunctional monkey NTCP with the human counterpart conferred its ability in supporting both viral infections.  The authors' results demonstrate that NTCP is a functional receptor for HBV and HDV.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrY0tqZhmqs37Vg90H21EOLACvtfcHk0lgeshE4H89rYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht12ms7nK&md5=7380905d92bd82b9cf3407b7c113bdfb</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.7554%2FeLife.00049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.00049%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DH.%26aulast%3DZhong%26aufirst%3DG.%26aulast%3DXu%26aufirst%3DG.%26aulast%3DHe%26aufirst%3DW.%26aulast%3DJing%26aufirst%3DZ.%26aulast%3DGao%26aufirst%3DZ.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DQi%26aufirst%3DY.%26aulast%3DPeng%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DFu%26aufirst%3DL.%26aulast%3DSong%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DP.%26aulast%3DGao%26aufirst%3DW.%26aulast%3DRen%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DCai%26aufirst%3DT.%26aulast%3DFeng%26aufirst%3DX.%26aulast%3DSui%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DW.%26atitle%3DSodium%2520taurocholate%2520cotransporting%2520polypeptide%2520is%2520a%2520functional%2520receptor%2520for%2520human%2520hepatitis%2520B%2520and%2520D%2520virus%26jtitle%3DElife%26date%3D2012%26volume%3D1%26doi%3D10.7554%2FeLife.00049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hagenbuch, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stieger, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foguet, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lubbert, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meier, P. J.</span></span> <span> </span><span class="NLM_article-title">Functional expression cloning and characterization of the hepatocyte Na+/bile acid cotransport system</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>88</i></span>,  <span class="NLM_fpage">10629</span>– <span class="NLM_lpage">10633</span>, <span class="refDoi"> DOI: 10.1073/pnas.88.23.10629</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1073%2Fpnas.88.23.10629" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1961729" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADyaK3sXnsFGjuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=1991&pages=10629-10633&author=B.+Hagenbuchauthor=B.+Stiegerauthor=M.+Foguetauthor=H.+Lubbertauthor=P.+J.+Meier&title=Functional+expression+cloning+and+characterization+of+the+hepatocyte+Na%2B%2Fbile+acid+cotransport+system&doi=10.1073%2Fpnas.88.23.10629"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Functional expression cloning and characterization of the hepatocyte sodium/bile acid cotransport system</span></div><div class="casAuthors">Hagenbuch, Bruno; Stieger, Bruno; Foguet, Montserrat; Luebbert, Hermann; Meier, Peter J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10629-33</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">A cDNA encoding the rat liver bile acid uptake system was isolated by expression cloning in Xenopus laevis oocytes.  The cloned transporter is strictly sodium-dependent and can be inhibited by various nonbile acid org. compds.  Sequence anal. of the cDNA revealed an open reading frame of 1086 nucleotides coding for a protein of 362 amino acids (calcd. mol. mass 39 kDa) with five possible N-linked glycosylation sites and seven putative transmembrane domains.  Translation expts. in vitro and in oocytes indicate that the transporter is indeed glycosylated and that its polypeptide backbone has an apparent mol. mass of 33-35 kDa.  Northern blot anal. with the cloned probe revealed crossreactivity with mRNA species from rat kidney and intestine as well as from liver tissues of mouse, guinea pig, rabbit, and man.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptcBpGcNwgibVg90H21EOLACvtfcHk0liDQb7O17wFmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXnsFGjuw%253D%253D&md5=b5cc0c3d427d9e7df22d92e5f92d412d</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1073%2Fpnas.88.23.10629&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.88.23.10629%26sid%3Dliteratum%253Aachs%26aulast%3DHagenbuch%26aufirst%3DB.%26aulast%3DStieger%26aufirst%3DB.%26aulast%3DFoguet%26aufirst%3DM.%26aulast%3DLubbert%26aufirst%3DH.%26aulast%3DMeier%26aufirst%3DP.%2BJ.%26atitle%3DFunctional%2520expression%2520cloning%2520and%2520characterization%2520of%2520the%2520hepatocyte%2520Na%252B%252Fbile%2520acid%2520cotransport%2520system%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1991%26volume%3D88%26spage%3D10629%26epage%3D10633%26doi%3D10.1073%2Fpnas.88.23.10629" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hagenbuch, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meier, P. J.</span></span> <span> </span><span class="NLM_article-title">Molecular cloning, chromosomal localization, and functional characterization of a human liver Na+/bile acid cotransporter</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">1326</span>– <span class="NLM_lpage">1331</span>, <span class="refDoi"> DOI: 10.1172/JCI117091</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1172%2FJCI117091" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=8132774" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADyaK2cXjtFKqtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=1994&pages=1326-1331&author=B.+Hagenbuchauthor=P.+J.+Meier&title=Molecular+cloning%2C+chromosomal+localization%2C+and+functional+characterization+of+a+human+liver+Na%2B%2Fbile+acid+cotransporter&doi=10.1172%2FJCI117091"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular cloning, chromosomal localization, and functional characterization of a human liver Na+/bile acid cotransporter</span></div><div class="casAuthors">Hagenbuch, Bruno; Meier, Peter J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Investigation</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1326-31</span>CODEN:
                <span class="NLM_cas:coden">JCINAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-9738</span>.
    </div><div class="casAbstract">A cDNA probe from a cloned rat liver Na+/taurocholate-cotransporting polypeptide (Ntcp) was used to screen a human liver cDNA library.  A 1599-bp cDNA clone that encodes a human Na+/taurocholate cotransporting polypeptide (NTCP) was isolated.  The human NTCP consists of 349 amino acids (calcd. mol. mass of 38 kDa) and exhibits 77% amino acid homol. with the rat Ntcp.  In vitro translation expts. indicate that the protein is glycosylated and has a mol. wt. similar to the rat Ntcp.  Injection of in vitro transcribed cRNA into Xenopus laevis oocytes resulted in the expression of Na+-dependent taurocholate uptake.  Satn. kinetics indicated that the human NTCP has a higher affinity for taurocholate (apparent Km = 6 μM) than the previously cloned rat protein (apparent Km = 25 μM).  NTCP-mediated taurocholate uptake into oocytes was inhibited by all major bile acid derivs. (100 μM), bumetanide (500 μM), and bromosulfophthalein (100 μM).  Southern blot anal. of genomic DNA from a panel of human/hamster somatic cell hybrids mapped the human NTCP gene to chromosome 14.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqmQsr-mlVh7Vg90H21EOLACvtfcHk0liDQb7O17wFmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXjtFKqtLk%253D&md5=1e49aa2044dce90f56b3671d4cca53c4</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1172%2FJCI117091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI117091%26sid%3Dliteratum%253Aachs%26aulast%3DHagenbuch%26aufirst%3DB.%26aulast%3DMeier%26aufirst%3DP.%2BJ.%26atitle%3DMolecular%2520cloning%252C%2520chromosomal%2520localization%252C%2520and%2520functional%2520characterization%2520of%2520a%2520human%2520liver%2520Na%252B%252Fbile%2520acid%2520cotransporter%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D1994%26volume%3D93%26spage%3D1326%26epage%3D1331%26doi%3D10.1172%2FJCI117091" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span> <span> </span><span class="NLM_article-title">DNA Polymerase kappa Is a Key Cellular Factor for the Formation of Covalently Closed Circular DNA of Hepatitis B Virus</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e1005893</span> <span class="refDoi"> DOI: 10.1371/journal.ppat.1005893</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1371%2Fjournal.ppat.1005893" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=27783675" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFCnu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&author=Y.+Qiauthor=Z.+Gaoauthor=G.+Xuauthor=B.+Pengauthor=C.+Liuauthor=H.+Yanauthor=Q.+Yaoauthor=G.+Sunauthor=Y.+Liuauthor=D.+Tangauthor=Z.+Songauthor=W.+Heauthor=Y.+Sunauthor=J.+T.+Guoauthor=W.+Li&title=DNA+Polymerase+kappa+Is+a+Key+Cellular+Factor+for+the+Formation+of+Covalently+Closed+Circular+DNA+of+Hepatitis+B+Virus&doi=10.1371%2Fjournal.ppat.1005893"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">DNA polymerase κ is a key cellular factor for the formation of covalently closed circular DNA of hepatitis B virus</span></div><div class="casAuthors">Qi, Yonghe; Gao, Zhenchao; Xu, Guangwei; Peng, Bo; Liu, Chenxuan; Yan, Huan; Yao, Qiyan; Sun, Guoliang; Liu, Yang; Tang, Dingbin; Song, Zilin; He, Wenhui; Sun, Yinyan; Guo, Ju-Tao; Li, Wenhui</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Pathogens</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e1005893/1-e1005893/26</span>CODEN:
                <span class="NLM_cas:coden">PPLACN</span>;
        ISSN:<span class="NLM_cas:issn">1553-7374</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Hepatitis B virus (HBV) infection of hepatocytes begins by binding to its cellular receptor sodium taurocholate cotransporting polypeptide (NTCP), followed by the internalization of viral nucleocapsid into the cytoplasm.  The viral relaxed circular (rc) DNA genome in nucleocapsid is transported into the nucleus and converted into covalently closed circular (ccc) DNA to serve as a viral persistence reservoir that is refractory to current antiviral therapies.  Host DNA repair enzymes have been speculated to catalyze the conversion of rcDNA to cccDNA, however, the DNA polymerase(s) that fills the gap in the plus strand of rcDNA remains to be detd.  Here we conducted targeted genetic screening in combination with chem. inhibition to identify the cellular DNA polymerase(s) responsible for cccDNA formation, and exploited recombinant HBV with capsid coding deficiency which infects HepG2-NTCP cells with similar efficiency of wild-type HBV to assure cccDNA synthesis is exclusively from de novo HBV infection.  We found that DNA polymerase κ (POLK), a Y-family DNA polymerase with max. activity in non-dividing cells, substantially contributes to cccDNA formation during de novo HBV infection.  Depleting gene expression of POLK in HepG2-NTCP cells by either siRNA knockdown or CRISPR/Cas9 knockout inhibited the conversion of rcDNA into cccDNA, while the diminished cccDNA formation in, and hence the viral infection of, the knockout cells could be effectively rescued by ectopic expression of POLK.  These studies revealed that POLK is a crucial host factor required for cccDNA formation during a de novo HBV infection and suggest that POLK may be a potential target for developing antivirals against HBV.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD8pcGjwB5mLVg90H21EOLACvtfcHk0livg77gQfCkDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFCnu7g%253D&md5=5a72d28fa023cb3f51dd976606f59f07</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1005893&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1005893%26sid%3Dliteratum%253Aachs%26aulast%3DQi%26aufirst%3DY.%26aulast%3DGao%26aufirst%3DZ.%26aulast%3DXu%26aufirst%3DG.%26aulast%3DPeng%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DYan%26aufirst%3DH.%26aulast%3DYao%26aufirst%3DQ.%26aulast%3DSun%26aufirst%3DG.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DTang%26aufirst%3DD.%26aulast%3DSong%26aufirst%3DZ.%26aulast%3DHe%26aufirst%3DW.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DGuo%26aufirst%3DJ.%2BT.%26aulast%3DLi%26aufirst%3DW.%26atitle%3DDNA%2520Polymerase%2520kappa%2520Is%2520a%2520Key%2520Cellular%2520Factor%2520for%2520the%2520Formation%2520of%2520Covalently%2520Closed%2520Circular%2520DNA%2520of%2520Hepatitis%2520B%2520Virus%26jtitle%3DPLoS%2520Pathog.%26date%3D2016%26volume%3D12%26doi%3D10.1371%2Fjournal.ppat.1005893" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">König, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doring, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohr, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geipel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geyer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glebe, D.</span></span> <span> </span><span class="NLM_article-title">Kinetics of the bile acid transporter and hepatitis B virus receptor Na+/taurocholate cotransporting polypeptide (NTCP) in hepatocytes</span>. <i>J. Hepatol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">867</span>– <span class="NLM_lpage">875</span>, <span class="refDoi"> DOI: 10.1016/j.jhep.2014.05.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1016%2Fj.jhep.2014.05.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=24845614" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A280%3ADC%252BC2cjkslSgug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2014&pages=867-875&author=A.+K%C3%B6nigauthor=B.+Doringauthor=C.+Mohrauthor=A.+Geipelauthor=J.+Geyerauthor=D.+Glebe&title=Kinetics+of+the+bile+acid+transporter+and+hepatitis+B+virus+receptor+Na%2B%2Ftaurocholate+cotransporting+polypeptide+%28NTCP%29+in+hepatocytes&doi=10.1016%2Fj.jhep.2014.05.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetics of the bile acid transporter and hepatitis B virus receptor Na+/taurocholate cotransporting polypeptide (NTCP) in hepatocytes</span></div><div class="casAuthors">Konig Alexander; Mohr Christina; Geipel Andreas; Doring Barbara; Geyer Joachim; Glebe Dieter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of hepatology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">867-75</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND & AIMS:  The human liver bile acid transporter Na(+)/taurocholate cotransporting polypeptide (NTCP) has recently been identified as liver-specific receptor for infection of hepatitis B virus (HBV), which attaches via the myristoylated preS1 (myr-preS1) peptide domain of its large surface protein to NTCP.  Since binding of the myr-preS1 peptide to NTCP is an initiating step of HBV infection, we investigated if this process interferes with the physiological bile acid transport function of NTCP.  METHODS:  HBV infection, myr-preS1 peptide binding, and bile acid transport assays were performed with primary Tupaia belangeri (PTH) and human (PHH) hepatocytes as well as NTCP-transfected human hepatoma HepG2 cells allowing regulated NTCP expression, in the presence of various bile acids, ezetimibe, and myr-preS1 peptides.  RESULTS:  The myr-preS1 peptide of HBV inhibited bile acid transport in PTH and PHH as well as in NTCP-expressing HEK293 and HepG2 cells.  Inversely, HBV infection of PTH, PHH, and NTCP-transfected HepG2 cells was inhibited in a concentration-dependent manner by taurine and glycine conjugates of cholic acid and ursodeoxycholic acid as well as by ezetimibe.  In NTCP-HepG2 cells and PTH, NTCP expression, NTCP transport function, myr-preS1 peptide binding, and HBV infection followed comparable kinetics.  CONCLUSIONS:  Myr-preS1 virus binding to NTCP, necessary for productive HBV infection, interferes with the physiological bile acid transport function of NTCP.  Therefore, HBV infection via NTCP may be lockable by NTCP substrates and NTCP-inhibiting drugs.  This opens a completely new way for an efficient management of HBV infection by the use of NTCP-directed drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQL8uPrQ26gtXtlzo9xastyfW6udTcc2eYYiGIuXFodtbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cjkslSgug%253D%253D&md5=46106af8958ba2cd35a321f29fe58970</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.jhep.2014.05.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jhep.2014.05.018%26sid%3Dliteratum%253Aachs%26aulast%3DK%25C3%25B6nig%26aufirst%3DA.%26aulast%3DDoring%26aufirst%3DB.%26aulast%3DMohr%26aufirst%3DC.%26aulast%3DGeipel%26aufirst%3DA.%26aulast%3DGeyer%26aufirst%3DJ.%26aulast%3DGlebe%26aufirst%3DD.%26atitle%3DKinetics%2520of%2520the%2520bile%2520acid%2520transporter%2520and%2520hepatitis%2520B%2520virus%2520receptor%2520Na%252B%252Ftaurocholate%2520cotransporting%2520polypeptide%2520%2528NTCP%2529%2520in%2520hepatocytes%26jtitle%3DJ.%2520Hepatol.%26date%3D2014%26volume%3D61%26spage%3D867%26epage%3D875%26doi%3D10.1016%2Fj.jhep.2014.05.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jing, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sui, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span> <span> </span><span class="NLM_article-title">Viral entry of hepatitis B and D viruses and bile salts transportation share common molecular determinants on sodium taurocholate cotransporting polypeptide</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>88</i></span>,  <span class="NLM_fpage">3273</span>– <span class="NLM_lpage">3284</span>, <span class="refDoi"> DOI: 10.1128/JVI.03478-13</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1128%2FJVI.03478-13" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=24390325" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsVOqtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=2014&pages=3273-3284&author=H.+Yanauthor=B.+Pengauthor=Y.+Liuauthor=G.+Xuauthor=W.+Heauthor=B.+Renauthor=Z.+Jingauthor=J.+Suiauthor=W.+Li&title=Viral+entry+of+hepatitis+B+and+D+viruses+and+bile+salts+transportation+share+common+molecular+determinants+on+sodium+taurocholate+cotransporting+polypeptide&doi=10.1128%2FJVI.03478-13"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Viral entry of hepatitis B and D viruses and bile salts transportation share common molecular determinants on sodium taurocholate cotransporting polypeptide</span></div><div class="casAuthors">Yan, Huan; Peng, Bo; Liu, Yang; Xu, Guangwei; He, Wenhui; Ren, Bijie; Jing, Zhiyi; Sui, Jianhua; Li, Wenhui</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3273-3284, 13 pp.</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">1098-5514</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The liver bile acids transporter sodium taurocholate cotransporting polypeptide (NTCP) is responsible for the majority of sodium-dependent bile salts uptake by hepatocytes.  NTCP also functions as a cellular receptor for viral entry of hepatitis B virus (HBV) and hepatitis D virus (HDV) through a specific interaction between NTCP and the pre-S1 domain of HBV large envelope protein.  However, it remains unknown if these two functions of NTCP are independent or if they interfere with each other.  Here we show that binding of the pre-S1 domain to human NTCP blocks taurocholate uptake by the receptor; conversely, some bile acid substrates of NTCP inhibit HBV and HDV entry.  Mutations of NTCP residues crit. for bile salts binding severely impair viral infection by HDV and HBV; to a lesser extent, the residues important for sodium binding also inhibit viral infection.  The mutation S267F, corresponding to a single nucleotide polymorphism (SNP) found in about 9% of the East Asian population, renders NTCP without either taurocholate transporting activity or the ability to support HBV or HDV infection in cell culture.  These results demonstrate that mol. determinants crit. for HBV and HDV entry overlap with that for bile salts uptake by NTCP, indicating that viral infection may interfere with the normal function of NTCP and bile acids and their derivs. hold the potential for further development into antiviral drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsAWAqX4p6L7Vg90H21EOLACvtfcHk0livg77gQfCkDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsVOqtr4%253D&md5=22fe09e746ad3f6ee76002a316de82b1</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1128%2FJVI.03478-13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.03478-13%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DH.%26aulast%3DPeng%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DG.%26aulast%3DHe%26aufirst%3DW.%26aulast%3DRen%26aufirst%3DB.%26aulast%3DJing%26aufirst%3DZ.%26aulast%3DSui%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DW.%26atitle%3DViral%2520entry%2520of%2520hepatitis%2520B%2520and%2520D%2520viruses%2520and%2520bile%2520salts%2520transportation%2520share%2520common%2520molecular%2520determinants%2520on%2520sodium%2520taurocholate%2520cotransporting%2520polypeptide%26jtitle%3DJ.%2520Virol.%26date%3D2014%26volume%3D88%26spage%3D3273%26epage%3D3284%26doi%3D10.1128%2FJVI.03478-13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Okuyama-Dobashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasai, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamashita, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yasumoto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maekawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakita, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enomoto, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriishi, K.</span></span> <span> </span><span class="NLM_article-title">Hepatitis B virus efficiently infects non-adherent hepatoma cells via human sodium taurocholate cotransporting polypeptide</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>, <span class="NLM_elocation-id">17047</span> <span class="refDoi"> DOI: 10.1038/srep17047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1038%2Fsrep17047" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=26592202" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFensbnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&author=K.+Okuyama-Dobashiauthor=H.+Kasaiauthor=T.+Tanakaauthor=A.+Yamashitaauthor=J.+Yasumotoauthor=W.+Chenauthor=T.+Okamotoauthor=S.+Maekawaauthor=K.+Watashiauthor=T.+Wakitaauthor=A.+Ryoauthor=T.+Suzukiauthor=Y.+Matsuuraauthor=N.+Enomotoauthor=K.+Moriishi&title=Hepatitis+B+virus+efficiently+infects+non-adherent+hepatoma+cells+via+human+sodium+taurocholate+cotransporting+polypeptide&doi=10.1038%2Fsrep17047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Hepatitis B virus efficiently infects non-adherent hepatoma cells via human sodium taurocholate cotransporting polypeptide</span></div><div class="casAuthors">Okuyama-Dobashi, Kaori; Kasai, Hirotake; Tanaka, Tomohisa; Yamashita, Atsuya; Yasumoto, Jun; Chen, Wenjia; Okamoto, Toru; Maekawa, Shinya; Watashi, Koichi; Wakita, Takaji; Ryo, Akihide; Suzuki, Tetsuro; Matsuura, Yoshiharu; Enomoto, Nobuyuki; Moriishi, Kohji</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">17047</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Sodium taurocholate cotransporting polypeptide (NTCP) has been reported as a functional receptor for hepatitis B virus (HBV) infection.  However, HBV could not efficiently infect HepG2 cells expressing NTCP (NTCP-HepG2 cells) under adherent monolayer-cell conditions.  In this study, NTCP was mainly detected in the basolateral membrane region, but not the apical site, of monolayer NTCP-HepG2 cells.  We hypothesized that non-adherent cell conditions of infection would enhance HBV infectivity.  Non-adherent NTCP-HepG2 cells were prepd. by treatment with trypsin and EDTA, which did not degrade NTCP in the membrane fraction.  HBV successfully infected NTCP-HepG2 cells at a viral dose 10 times lower in non-adherent phase than in adherent phase.  Efficient infection of non-adherent NTCP-HepG2 cells with blood-borne or cell-culture-derived HBV was obsd. and was remarkably impaired in the presence of the myristoylated preS1 peptide.  HBV could also efficiently infect HepaRG cells under non-adherent cell conditions.  We screened several compds. using our culture system and identified proscillaridin A as a potent anti-HBV agent with an IC50 value of 7.2 nM.  In conclusion, non-adherent host cell conditions of infection augmented HBV infectivity in an NTCP-dependent manner, thus providing a novel strategy to identify anti-HBV drugs and investigate the mechanism of HBV infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGow9Dq2g5ytTLVg90H21EOLACvtfcHk0lg8rFaiKeUzHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFensbnJ&md5=0990ab0c427e276fcea727a037e17c9d</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fsrep17047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep17047%26sid%3Dliteratum%253Aachs%26aulast%3DOkuyama-Dobashi%26aufirst%3DK.%26aulast%3DKasai%26aufirst%3DH.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DYamashita%26aufirst%3DA.%26aulast%3DYasumoto%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DOkamoto%26aufirst%3DT.%26aulast%3DMaekawa%26aufirst%3DS.%26aulast%3DWatashi%26aufirst%3DK.%26aulast%3DWakita%26aufirst%3DT.%26aulast%3DRyo%26aufirst%3DA.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DMatsuura%26aufirst%3DY.%26aulast%3DEnomoto%26aufirst%3DN.%26aulast%3DMoriishi%26aufirst%3DK.%26atitle%3DHepatitis%2520B%2520virus%2520efficiently%2520infects%2520non-adherent%2520hepatoma%2520cells%2520via%2520human%2520sodium%2520taurocholate%2520cotransporting%2520polypeptide%26jtitle%3DSci.%2520Rep.%26date%3D2015%26volume%3D5%26doi%3D10.1038%2Fsrep17047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaneko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamisuki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsunaga, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawai, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohashi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsukuda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aizaki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugiyama, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohtani, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugawara, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizokami, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sureau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakita, T.</span></span> <span> </span><span class="NLM_article-title">A Novel Tricyclic Polyketide, Vanitaracin A, Specifically Inhibits the Entry of Hepatitis B and D Viruses by Targeting Sodium Taurocholate Cotransporting Polypeptide</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">11945</span>– <span class="NLM_lpage">11953</span>, <span class="refDoi"> DOI: 10.1128/JVI.01855-15</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1128%2FJVI.01855-15" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=26378168" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADC%252BC28XjsFyiur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2015&pages=11945-11953&author=M.+Kanekoauthor=K.+Watashiauthor=S.+Kamisukiauthor=H.+Matsunagaauthor=M.+Iwamotoauthor=F.+Kawaiauthor=H.+Ohashiauthor=S.+Tsukudaauthor=S.+Shimuraauthor=R.+Suzukiauthor=H.+Aizakiauthor=M.+Sugiyamaauthor=S.+Y.+Parkauthor=T.+Itoauthor=N.+Ohtaniauthor=F.+Sugawaraauthor=Y.+Tanakaauthor=M.+Mizokamiauthor=C.+Sureauauthor=T.+Wakita&title=A+Novel+Tricyclic+Polyketide%2C+Vanitaracin+A%2C+Specifically+Inhibits+the+Entry+of+Hepatitis+B+and+D+Viruses+by+Targeting+Sodium+Taurocholate+Cotransporting+Polypeptide&doi=10.1128%2FJVI.01855-15"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">A novel tricyclic polyketide, vanitaracin A, specifically inhibits the entry of hepatitis B and D viruses by targeting sodium taurocholate cotransporting polypeptide</span></div><div class="casAuthors">Kaneko, Manabu; Watashi, Koichi; Kamisuki, Shinji; Matsunaga, Hiroki; Iwamoto, Masashi; Kawai, Fumihiro; Ohashi, Hirofumi; Tsukuda, Senko; Shimura, Satomi; Suzuki, Ryosuke; Aizaki, Hideki; Sugiyama, Masaya; Park, Sam-Yong; Ito, Takayoshi; Ohtani, Naoko; Sugawara, Fumio; Tanaka, Yasuhito; Mizokami, Masashi; Sureau, Camille; Wakita, Takaji</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">11945-11953</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">1098-5514</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Anti-hepatitis B virus (HBV) drugs are currently limited to nucleos(t)ide analogs (NAs) and interferons.  A challenge of drug development is the identification of small mols. that suppress HBV infection from new chem. sources.  Here, from a fungus-derived secondary metabolite library, we identify a structurally novel tricyclic polyketide, named vanitaracin A, which specifically inhibits HBV infection.  Vanitaracin A inhibited the viral entry process with a submicromolar 50% inhibitory concn. (IC50) (IC50 = 0.61 ± 0.23 μM), without evident cytotoxicity (50% cytotoxic concn. of >256 μM; selectivity index value of >419) in primary human hepatocytes.  Vanitaracin A did not affect the HBV replication process.  This compd. was found to directly interact with the HBV entry receptor sodium taurocholate cotransporting polypeptide (NTCP) and impaired its bile acid transport activity.  Consistent with this NTCP targeting, antiviral activity of vanitaracin A was obsd. with hepatitis D virus (HDV) but not hepatitis C virus.  Importantly, vanitaracin A inhibited infection by all HBV genotypes tested (genotypes A to D) and clin. relevant NA-resistant HBV isolate.  Thus, we identified a fungal metabolite, vanitaracin A, which was a potent, well-tolerated, and broadly active inhibitor of HBV and HDV entry.  This compd., or its related analogs, could be part of an antiviral strategy for preventing reinfection with HBV, including clin. relevant nucleos(t)ide analog-resistant virus.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiy8gZypawebVg90H21EOLACvtfcHk0lg8rFaiKeUzHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjsFyiur0%253D&md5=a474e23328a4f350415610cb08697512</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1128%2FJVI.01855-15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.01855-15%26sid%3Dliteratum%253Aachs%26aulast%3DKaneko%26aufirst%3DM.%26aulast%3DWatashi%26aufirst%3DK.%26aulast%3DKamisuki%26aufirst%3DS.%26aulast%3DMatsunaga%26aufirst%3DH.%26aulast%3DIwamoto%26aufirst%3DM.%26aulast%3DKawai%26aufirst%3DF.%26aulast%3DOhashi%26aufirst%3DH.%26aulast%3DTsukuda%26aufirst%3DS.%26aulast%3DShimura%26aufirst%3DS.%26aulast%3DSuzuki%26aufirst%3DR.%26aulast%3DAizaki%26aufirst%3DH.%26aulast%3DSugiyama%26aufirst%3DM.%26aulast%3DPark%26aufirst%3DS.%2BY.%26aulast%3DIto%26aufirst%3DT.%26aulast%3DOhtani%26aufirst%3DN.%26aulast%3DSugawara%26aufirst%3DF.%26aulast%3DTanaka%26aufirst%3DY.%26aulast%3DMizokami%26aufirst%3DM.%26aulast%3DSureau%26aufirst%3DC.%26aulast%3DWakita%26aufirst%3DT.%26atitle%3DA%2520Novel%2520Tricyclic%2520Polyketide%252C%2520Vanitaracin%2520A%252C%2520Specifically%2520Inhibits%2520the%2520Entry%2520of%2520Hepatitis%2520B%2520and%2520D%2520Viruses%2520by%2520Targeting%2520Sodium%2520Taurocholate%2520Cotransporting%2520Polypeptide%26jtitle%3DJ.%2520Virol.%26date%3D2015%26volume%3D89%26spage%3D11945%26epage%3D11953%26doi%3D10.1128%2FJVI.01855-15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nkongolo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ni, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lempp, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufman, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esser-Nobis, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohmann, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mier, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehrle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urban, S.</span></span> <span> </span><span class="NLM_article-title">Cyclosporin A inhibits hepatitis B and hepatitis D virus entry by cyclophilin-independent interference with the NTCP receptor</span>. <i>J. Hepatol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">723</span>– <span class="NLM_lpage">731</span>, <span class="refDoi"> DOI: 10.1016/j.jhep.2013.11.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1016%2Fj.jhep.2013.11.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=24295872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtlagtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2014&pages=723-731&author=S.+Nkongoloauthor=Y.+Niauthor=F.+A.+Lemppauthor=C.+Kaufmanauthor=T.+Lindnerauthor=K.+Esser-Nobisauthor=V.+Lohmannauthor=W.+Mierauthor=S.+Mehrleauthor=S.+Urban&title=Cyclosporin+A+inhibits+hepatitis+B+and+hepatitis+D+virus+entry+by+cyclophilin-independent+interference+with+the+NTCP+receptor&doi=10.1016%2Fj.jhep.2013.11.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclosporin A inhibits hepatitis B and hepatitis D virus entry by cyclophilin-independent interference with the NTCP receptor</span></div><div class="casAuthors">Nkongolo, Shirin; Ni, Yi; Lempp, Florian A.; Kaufman, Christina; Lindner, Thomas; Esser-Nobis, Katharina; Lohmann, Volker; Mier, Walter; Mehrle, Stefan; Urban, Stephan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Hepatology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">723-731</span>CODEN:
                <span class="NLM_cas:coden">JOHEEC</span>;
        ISSN:<span class="NLM_cas:issn">0168-8278</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Chronic hepatitis B and hepatitis D are global health problems caused by the human hepatitis B and hepatitis D virus.  The myristoylated preS1 domain of the large envelope protein mediates specific binding to hepatocytes by sodium taurocholate co-transporting polypeptide (NTCP).  NTCP is a bile salt transporter known to be inhibited by cyclosporin A.  This study aimed to characterize the effect of cyclosporin A on HBV/HDV infection.  HepaRG cells, primary human hepatocytes, and susceptible NTCP-expressing hepatoma cell lines were applied for infection expts.  The mode of action of cyclosporin A was studied by comparing the effect of different inhibitors, cyclophilin A/B/C-silenced cell lines as well as NTCP variants and mutants.  Bile salt transporter and HBV receptor functions were investigated by taurocholate uptake and quantification of HBVpreS binding.Cyclosporin A inhibited hepatitis B and D virus infections during and - less pronounced - prior to virus inoculation.  Binding of HBVpreS to NTCP was blocked by cyclosporin A concns. at 8 μM.  An NTCP variant deficient in HBVpreS binding but competent for bile salt transport showed resistance to cyclosporin A.  Silencing of cyclophilins A/B/C did not abrogate transporter and receptor inhibition.  In contrast, tacrolimus, a cyclophilin-independent calcineurin inhibitor, was inactive.HBV and HDV entry via sodium taurocholate co-transporting polypeptide is inhibited by cyclosporin A.  The interaction between the drug and the viral receptor is direct and overlaps with a functional binding site of the preS1 domain, which mediates viral entry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTAVAhQe6_ErVg90H21EOLACvtfcHk0lg8rFaiKeUzHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtlagtw%253D%253D&md5=d8c7e3698ac926cd5c06a9e7e2c4717b</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.jhep.2013.11.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jhep.2013.11.022%26sid%3Dliteratum%253Aachs%26aulast%3DNkongolo%26aufirst%3DS.%26aulast%3DNi%26aufirst%3DY.%26aulast%3DLempp%26aufirst%3DF.%2BA.%26aulast%3DKaufman%26aufirst%3DC.%26aulast%3DLindner%26aufirst%3DT.%26aulast%3DEsser-Nobis%26aufirst%3DK.%26aulast%3DLohmann%26aufirst%3DV.%26aulast%3DMier%26aufirst%3DW.%26aulast%3DMehrle%26aufirst%3DS.%26aulast%3DUrban%26aufirst%3DS.%26atitle%3DCyclosporin%2520A%2520inhibits%2520hepatitis%2520B%2520and%2520hepatitis%2520D%2520virus%2520entry%2520by%2520cyclophilin-independent%2520interference%2520with%2520the%2520NTCP%2520receptor%26jtitle%3DJ.%2520Hepatol.%26date%3D2014%26volume%3D60%26spage%3D723%26epage%3D731%26doi%3D10.1016%2Fj.jhep.2013.11.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Watashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sluder, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daito, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsunaga, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagamori, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakajima, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsukuda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borroto-Esoda, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugiyama, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kusuhara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizokami, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakita, T.</span></span> <span> </span><span class="NLM_article-title">Cyclosporin A and its analogs inhibit hepatitis B virus entry into cultured hepatocytes through targeting a membrane transporter, sodium taurocholate cotransporting polypeptide (NTCP)</span>. <i>Hepatology</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1726</span>– <span class="NLM_lpage">1737</span>, <span class="refDoi"> DOI: 10.1002/hep.26982</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1002%2Fhep.26982" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=24375637" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADC%252BC2cXntFCltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2014&pages=1726-1737&author=K.+Watashiauthor=A.+Sluderauthor=T.+Daitoauthor=S.+Matsunagaauthor=A.+Ryoauthor=S.+Nagamoriauthor=M.+Iwamotoauthor=S.+Nakajimaauthor=S.+Tsukudaauthor=K.+Borroto-Esodaauthor=M.+Sugiyamaauthor=Y.+Tanakaauthor=Y.+Kanaiauthor=H.+Kusuharaauthor=M.+Mizokamiauthor=T.+Wakita&title=Cyclosporin+A+and+its+analogs+inhibit+hepatitis+B+virus+entry+into+cultured+hepatocytes+through+targeting+a+membrane+transporter%2C+sodium+taurocholate+cotransporting+polypeptide+%28NTCP%29&doi=10.1002%2Fhep.26982"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclosporin A and its analogs inhibit hepatitis B virus entry into cultured hepatocytes through targeting a membrane transporter, sodium taurocholate cotransporting polypeptide (NTCP)</span></div><div class="casAuthors">Watashi, Koichi; Sluder, Ann; Daito, Takuji; Matsunaga, Satoko; Ryo, Akihide; Nagamori, Shushi; Iwamoto, Masashi; Nakajima, Syo; Tsukuda, Senko; Borroto-Esoda, Katyna; Sugiyama, Masaya; Tanaka, Yasuhito; Kanai, Yoshikatsu; Kusuhara, Hiroyuki; Mizokami, Masashi; Wakita, Takaji</div><div class="citationInfo"><span class="NLM_cas:title">Hepatology (Hoboken, NJ, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1726-1737</span>CODEN:
                <span class="NLM_cas:coden">HPTLD9</span>;
        ISSN:<span class="NLM_cas:issn">0270-9139</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Chronic hepatitis B virus (HBV) infection is a major public health problem worldwide.  Although nucleos(t)ide analogs inhibiting viral reverse transcriptase are clin. available as anti-HBV agents, emergence of drug-resistant viruses highlights the need for new anti-HBV agents interfering with other targets.  Here we report that cyclosporin A (CsA) can inhibit HBV entry into cultured hepatocytes.  The anti-HBV effect of CsA was independent of binding to cyclophilin and calcineurin.  Rather, blockade of HBV infection correlated with the ability to inhibit the transporter activity of sodium taurocholate cotransporting polypeptide (NTCP).  We also found that HBV infection-susceptible cells, differentiated HepaRG cells and primary human hepatocytes expressed NTCP, while nonsusceptible cell lines did not.  A series of compds. targeting NTCP could inhibit HBV infection.  CsA inhibited the binding between NTCP and large envelope protein in vitro.  Evaluation of CsA analogs identified a compd. with higher anti-HBV potency, having a median inhibitory concn. <0.2 μM.  Conclusion: This study provides a proof of concept for the novel strategy to identify anti-HBV agents by targeting the candidate HBV receptor, NTCP, using CsA as a structural platform. (Hepatol. 2014;59:1726-1737).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRV21kvExmObVg90H21EOLACvtfcHk0lhzfsuO8BcIKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXntFCltro%253D&md5=d83ae63fbe3e8b827f9fbf5f6601b316</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1002%2Fhep.26982&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fhep.26982%26sid%3Dliteratum%253Aachs%26aulast%3DWatashi%26aufirst%3DK.%26aulast%3DSluder%26aufirst%3DA.%26aulast%3DDaito%26aufirst%3DT.%26aulast%3DMatsunaga%26aufirst%3DS.%26aulast%3DRyo%26aufirst%3DA.%26aulast%3DNagamori%26aufirst%3DS.%26aulast%3DIwamoto%26aufirst%3DM.%26aulast%3DNakajima%26aufirst%3DS.%26aulast%3DTsukuda%26aufirst%3DS.%26aulast%3DBorroto-Esoda%26aufirst%3DK.%26aulast%3DSugiyama%26aufirst%3DM.%26aulast%3DTanaka%26aufirst%3DY.%26aulast%3DKanai%26aufirst%3DY.%26aulast%3DKusuhara%26aufirst%3DH.%26aulast%3DMizokami%26aufirst%3DM.%26aulast%3DWakita%26aufirst%3DT.%26atitle%3DCyclosporin%2520A%2520and%2520its%2520analogs%2520inhibit%2520hepatitis%2520B%2520virus%2520entry%2520into%2520cultured%2520hepatocytes%2520through%2520targeting%2520a%2520membrane%2520transporter%252C%2520sodium%2520taurocholate%2520cotransporting%2520polypeptide%2520%2528NTCP%2529%26jtitle%3DHepatology%26date%3D2014%26volume%3D59%26spage%3D1726%26epage%3D1737%26doi%3D10.1002%2Fhep.26982" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shimura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukano, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sluder, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawai, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsukuda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeuchi, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyake, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugiyama, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogasawara, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kusuhara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizokami, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sureau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakita, T.</span></span> <span> </span><span class="NLM_article-title">Cyclosporin derivatives inhibit hepatitis B virus entry without interfering with NTCP transporter activity</span>. <i>J. Hepatol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">685</span>– <span class="NLM_lpage">692</span>, <span class="refDoi"> DOI: 10.1016/j.jhep.2016.11.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1016%2Fj.jhep.2016.11.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=27890789" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFGis7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2017&pages=685-692&author=S.+Shimuraauthor=K.+Watashiauthor=K.+Fukanoauthor=M.+Peelauthor=A.+Sluderauthor=F.+Kawaiauthor=M.+Iwamotoauthor=S.+Tsukudaauthor=J.+S.+Takeuchiauthor=T.+Miyakeauthor=M.+Sugiyamaauthor=Y.+Ogasawaraauthor=S.+Y.+Parkauthor=Y.+Tanakaauthor=H.+Kusuharaauthor=M.+Mizokamiauthor=C.+Sureauauthor=T.+Wakita&title=Cyclosporin+derivatives+inhibit+hepatitis+B+virus+entry+without+interfering+with+NTCP+transporter+activity&doi=10.1016%2Fj.jhep.2016.11.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclosporin derivatives inhibit hepatitis B virus entry without interfering with NTCP transporter activity</span></div><div class="casAuthors">Shimura, Satomi; Watashi, Koichi; Fukano, Kento; Peel, Michael; Sluder, Ann; Kawai, Fumihiro; Iwamoto, Masashi; Tsukuda, Senko; Takeuchi, Junko S.; Miyake, Takeshi; Sugiyama, Masaya; Ogasawara, Yuki; Park, Sam-Yong; Tanaka, Yasuhito; Kusuhara, Hiroyuki; Mizokami, Masashi; Sureau, Camille; Wakita, Takaji</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Hepatology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">685-692</span>CODEN:
                <span class="NLM_cas:coden">JOHEEC</span>;
        ISSN:<span class="NLM_cas:issn">0168-8278</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The sodium taurocholate co-transporting polypeptide (NTCP) is the main target of most hepatitis B virus (HBV) specific entry inhibitors.  Unfortunately, these agents also block NTCP transport of bile acids into hepatocytes, and thus have the potential to cause adverse effects.  We aimed to identify small mols. that inhibit HBV entry while maintaining NTCP transporter function.  We characterized a series of cyclosporine (CsA) derivs. for their anti-HBV activity and NTCP binding specificity using HepG2 cells overexpressing NTCP and primary human hepatocytes.  The four most potent derivs. were tested for their capacity to prevent HBV entry, but maintain NTCP transporter function.  Their antiviral activity against different HBV genotypes was analyzed.We identified several CsA derivs. that inhibited HBV infection with a sub-micromolar IC50.  Among them, SCY446 and SCY450 showed low activity against calcineurin (CN) and cyclophilins (CyPs), two major CsA cellular targets.  This suggested that instead, these compds. interacted directly with NTCP to inhibit viral attachment to host cells, and have no immunosuppressive function.  Importantly, we found that SCY450 and SCY995 did not impair the NTCP-dependent uptake of bile acids, and inhibited multiple HBV genotypes including a clin. relevant nucleoside analog-resistant HBV isolate.  This is the first example of small mol. selective inhibition of HBV entry with no decrease in NTCP transporter activity.  It suggests that the anti-HBV activity can be functionally sepd. from bile acid transport.  These broadly active anti-HBV mols. are potential candidates for developing new drugs with fewer adverse effects.  In this study, we identified new compds. that selectively inhibited hepatitis B virus (HBV) entry, and did not impair bile acid uptake.  Our evidence offers a new strategy for developing anti-HBV drugs with fewer side effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqK9TO_Uo2M-bVg90H21EOLACvtfcHk0lhzfsuO8BcIKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFGis7%252FO&md5=7cb0015063c0a8c2016283d875076478</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.jhep.2016.11.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jhep.2016.11.009%26sid%3Dliteratum%253Aachs%26aulast%3DShimura%26aufirst%3DS.%26aulast%3DWatashi%26aufirst%3DK.%26aulast%3DFukano%26aufirst%3DK.%26aulast%3DPeel%26aufirst%3DM.%26aulast%3DSluder%26aufirst%3DA.%26aulast%3DKawai%26aufirst%3DF.%26aulast%3DIwamoto%26aufirst%3DM.%26aulast%3DTsukuda%26aufirst%3DS.%26aulast%3DTakeuchi%26aufirst%3DJ.%2BS.%26aulast%3DMiyake%26aufirst%3DT.%26aulast%3DSugiyama%26aufirst%3DM.%26aulast%3DOgasawara%26aufirst%3DY.%26aulast%3DPark%26aufirst%3DS.%2BY.%26aulast%3DTanaka%26aufirst%3DY.%26aulast%3DKusuhara%26aufirst%3DH.%26aulast%3DMizokami%26aufirst%3DM.%26aulast%3DSureau%26aufirst%3DC.%26aulast%3DWakita%26aufirst%3DT.%26atitle%3DCyclosporin%2520derivatives%2520inhibit%2520hepatitis%2520B%2520virus%2520entry%2520without%2520interfering%2520with%2520NTCP%2520transporter%2520activity%26jtitle%3DJ.%2520Hepatol.%26date%3D2017%26volume%3D66%26spage%3D685%26epage%3D692%26doi%3D10.1016%2Fj.jhep.2016.11.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lucifora, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esser, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Protzer, U.</span></span> <span> </span><span class="NLM_article-title">Ezetimibe blocks hepatitis B virus infection after virus uptake into hepatocytes</span>. <i>Antiviral Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">195</span>– <span class="NLM_lpage">197</span>, <span class="refDoi"> DOI: 10.1016/j.antiviral.2012.12.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1016%2Fj.antiviral.2012.12.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=23266293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFSrsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2013&pages=195-197&author=J.+Luciforaauthor=K.+Esserauthor=U.+Protzer&title=Ezetimibe+blocks+hepatitis+B+virus+infection+after+virus+uptake+into+hepatocytes&doi=10.1016%2Fj.antiviral.2012.12.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Ezetimibe blocks hepatitis B virus infection after virus uptake into hepatocytes</span></div><div class="casAuthors">Lucifora, Julie; Esser, Knud; Protzer, Ulrike</div><div class="citationInfo"><span class="NLM_cas:title">Antiviral Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">195-197</span>CODEN:
                <span class="NLM_cas:coden">ARSRDR</span>;
        ISSN:<span class="NLM_cas:issn">0166-3542</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Current treatment of chronic hepatitis B virus (HBV) infection mainly targets viral replication in hepatocytes and leads to curing only in exceptional cases.  Despite their potential to improve therapeutic success, no drugs interfering with early infection steps of the hepatotropic pathogen HBV are available to date.  Recently, entry of the hepatitis C virus (HCV) was shown to occur along hepatic cholesterol uptake pathways and ezetimibe, a drug which blocks this lipid transport, was shown to inhibit HCV infection.  We here investigated the effect of ezetimibe on HBV infection using differentiated HepaRG cells as a cell-culture infection model.  Treatment with ezetimibe inhibited establishment of intrahepatic cccDNA and expression of viral replication markers when cells were infected with HBV virions, while we obsd. no effect when the HBV viral genome was transduced via an adenoviral vector.  Our data suggest that modulating hepatic cholesterol uptake by ezetimibe inhibits early HBV infection and that ezetimibe sensitive lipid transport pathways represent new targets for antiviral therapy in HBV infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNJ1wQJ_RUkbVg90H21EOLACvtfcHk0lgS7qF5-sYyWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFSrsrw%253D&md5=dcfca7855b8cacdad584203e23690f85</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.antiviral.2012.12.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.antiviral.2012.12.008%26sid%3Dliteratum%253Aachs%26aulast%3DLucifora%26aufirst%3DJ.%26aulast%3DEsser%26aufirst%3DK.%26aulast%3DProtzer%26aufirst%3DU.%26atitle%3DEzetimibe%2520blocks%2520hepatitis%2520B%2520virus%2520infection%2520after%2520virus%2520uptake%2520into%2520hepatocytes%26jtitle%3DAntiviral%2520Res.%26date%3D2013%26volume%3D97%26spage%3D195%26epage%3D197%26doi%3D10.1016%2Fj.antiviral.2012.12.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blanchet, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sureau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labonte, P.</span></span> <span> </span><span class="NLM_article-title">Use of FDA approved therapeutics with hNTCP metabolic inhibitory properties to impair the HDV lifecycle</span>. <i>Antiviral Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">111</span>– <span class="NLM_lpage">115</span>, <span class="refDoi"> DOI: 10.1016/j.antiviral.2014.03.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1016%2Fj.antiviral.2014.03.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=24717262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnvFSrsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2014&pages=111-115&author=M.+Blanchetauthor=C.+Sureauauthor=P.+Labonte&title=Use+of+FDA+approved+therapeutics+with+hNTCP+metabolic+inhibitory+properties+to+impair+the+HDV+lifecycle&doi=10.1016%2Fj.antiviral.2014.03.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Use of FDA approved therapeutics with hNTCP metabolic inhibitory properties to impair the HDV lifecycle</span></div><div class="casAuthors">Blanchet, Matthieu; Sureau, Camille; Labonte, Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Antiviral Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">111-115</span>CODEN:
                <span class="NLM_cas:coden">ARSRDR</span>;
        ISSN:<span class="NLM_cas:issn">0166-3542</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Worldwide there are approx. 240 million individuals chronically infected with the hepatitis B virus (HBV), including 15-20 million coinfected with the hepatitis delta virus (HDV).  Treatments available today are not fully efficient and often assocd. to important side effects and development of drug resistance.  Targeting the HBV/HDV entry step using preS1-specific lipopeptides appears as a promising strategy to block viral entry for both HBV and HDV (Gripon et al., 2005; Petersen et al., 2008).  Recently, the human Sodium Taurocholate Cotransporting Polypeptide (hNTCP) has been identified as a functional, preS1-specific receptor for HBV and HDV.  This groundbreaking discovery has opened a very promising avenue for the treatment of chronic HBV and HDV infections.  Here we investigated the ability of FDA approved therapeutics with documented inhibitory effect on hNTCP cellular function to impair viral entry using a HDV in vitro infection model based on a hNTCP-expressing Huh7 cell line.  We demonstrate the potential of three FDA approved mols., irbesartan, ezetimibe, and ritonavir, to alter HDV infection in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5uuuK9vPHVbVg90H21EOLACvtfcHk0lgS7qF5-sYyWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnvFSrsLc%253D&md5=abe5262a9fa3fc4daaff030ae1a080c1</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.antiviral.2014.03.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.antiviral.2014.03.017%26sid%3Dliteratum%253Aachs%26aulast%3DBlanchet%26aufirst%3DM.%26aulast%3DSureau%26aufirst%3DC.%26aulast%3DLabonte%26aufirst%3DP.%26atitle%3DUse%2520of%2520FDA%2520approved%2520therapeutics%2520with%2520hNTCP%2520metabolic%2520inhibitory%2520properties%2520to%2520impair%2520the%2520HDV%2520lifecycle%26jtitle%3DAntiviral%2520Res.%26date%3D2014%26volume%3D106%26spage%3D111%26epage%3D115%26doi%3D10.1016%2Fj.antiviral.2014.03.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blank, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markert, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hohmann, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carls, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikus, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehr, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alexandrov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haag, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwab, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urban, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haefeli, W. E.</span></span> <span> </span><span class="NLM_article-title">First-in-human application of the novel hepatitis B and hepatitis D virus entry inhibitor myrcludex B</span>. <i>J. Hepatol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">483</span>– <span class="NLM_lpage">489</span>, <span class="refDoi"> DOI: 10.1016/j.jhep.2016.04.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1016%2Fj.jhep.2016.04.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=27132172" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADC%252BC28Xptl2msrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2016&pages=483-489&author=A.+Blankauthor=C.+Markertauthor=N.+Hohmannauthor=A.+Carlsauthor=G.+Mikusauthor=T.+Lehrauthor=A.+Alexandrovauthor=M.+Haagauthor=M.+Schwabauthor=S.+Urbanauthor=W.+E.+Haefeli&title=First-in-human+application+of+the+novel+hepatitis+B+and+hepatitis+D+virus+entry+inhibitor+myrcludex+B&doi=10.1016%2Fj.jhep.2016.04.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">First-in-human application of the novel hepatitis B and hepatitis D virus entry inhibitor Myrcludex B</span></div><div class="casAuthors">Blank, Antje; Markert, Christoph; Hohmann, Nicolas; Carls, Alexandra; Mikus, Gerd; Lehr, Thorsten; Alexandrov, Alexander; Haag, Mathias; Schwab, Matthias; Urban, Stephan; Haefeli, Walter E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Hepatology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">483-489</span>CODEN:
                <span class="NLM_cas:coden">JOHEEC</span>;
        ISSN:<span class="NLM_cas:issn">0168-8278</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Myrcludex B is a first-in-class compd., which blocks entry of hepatitis B and D virus into hepatocytes in vitro and in animal models.  Based on the required preclin. data we aimed to translate this compd. into the first application in humans.Single ascending doses of myrcludex B, a 47 amino acid peptide, were administered up to 20 mg i.v. and 10 mg s.c. in a prospective open first-in-human, phase I clin. trial to 36 healthy volunteers.  Safety, tolerability and plasma concns. of myrcludex B were assessed and a pharmacokinetic model was derived.Myrcludex B was well tolerated and no serious or relevant AEs representing off-target effects, and no immunogenic effects were obsd. up to the highest applied dose of 20 mg (i.v.).  Myrcludex B showed dose-dependent pharmacokinetics, best described by a 2-compartment target-mediated drug disposition model.  Bioavailability of the s.c. application was large (85%).  Interindividual variability was moderate.  The pharmacokinetic model suggested that s.c. doses of 10 mg and above reach a target satn. of over 80% for at least 15 h.Myrcludex B showed excellent tolerability up to high doses.  Pharmacol. properties followed a 2-compartment target-mediated drug disposition model.  These findings are vital for planning of further multiple dose efficacy trials in patients.After showing antiviral activity in cell culture and animal models, myrcludex B, a new drug intended for the treatment of hepatitis B and D, has been administered the first time in humans.  Healthy volunteers received the drug i.v. and s.c. up to high doses (20 mg).  The drug was well tolerated and the characteristics of the drug detg. its way in the human body could be described.  These results will allow testing myrcludex B in hepatitis B and D patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJoBJ0yS6pu7Vg90H21EOLACvtfcHk0lgS7qF5-sYyWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xptl2msrc%253D&md5=04a54ac56423c8692e19bd6baabff25b</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.jhep.2016.04.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jhep.2016.04.013%26sid%3Dliteratum%253Aachs%26aulast%3DBlank%26aufirst%3DA.%26aulast%3DMarkert%26aufirst%3DC.%26aulast%3DHohmann%26aufirst%3DN.%26aulast%3DCarls%26aufirst%3DA.%26aulast%3DMikus%26aufirst%3DG.%26aulast%3DLehr%26aufirst%3DT.%26aulast%3DAlexandrov%26aufirst%3DA.%26aulast%3DHaag%26aufirst%3DM.%26aulast%3DSchwab%26aufirst%3DM.%26aulast%3DUrban%26aufirst%3DS.%26aulast%3DHaefeli%26aufirst%3DW.%2BE.%26atitle%3DFirst-in-human%2520application%2520of%2520the%2520novel%2520hepatitis%2520B%2520and%2520hepatitis%2520D%2520virus%2520entry%2520inhibitor%2520myrcludex%2520B%26jtitle%3DJ.%2520Hepatol.%26date%3D2016%26volume%3D65%26spage%3D483%26epage%3D489%26doi%3D10.1016%2Fj.jhep.2016.04.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bogomolov, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alexandrov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voronkova, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macievich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kokina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petrachenkova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehr, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lempp, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wedemeyer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haag, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwab, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haefeli, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blank, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urban, S.</span></span> <span> </span><span class="NLM_article-title">Treatment of chronic hepatitis D with the entry inhibitor myrcludex B: First results of a phase Ib/IIa study</span>. <i>J. Hepatol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">490</span>– <span class="NLM_lpage">498</span>, <span class="refDoi"> DOI: 10.1016/j.jhep.2016.04.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1016%2Fj.jhep.2016.04.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=27132170" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVynsbrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2016&pages=490-498&author=P.+Bogomolovauthor=A.+Alexandrovauthor=N.+Voronkovaauthor=M.+Macievichauthor=K.+Kokinaauthor=M.+Petrachenkovaauthor=T.+Lehrauthor=F.+A.+Lemppauthor=H.+Wedemeyerauthor=M.+Haagauthor=M.+Schwabauthor=W.+E.+Haefeliauthor=A.+Blankauthor=S.+Urban&title=Treatment+of+chronic+hepatitis+D+with+the+entry+inhibitor+myrcludex+B%3A+First+results+of+a+phase+Ib%2FIIa+study&doi=10.1016%2Fj.jhep.2016.04.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Treatment of chronic hepatitis D with the entry inhibitor myrcludex B: First results of a phase Ib/IIa study</span></div><div class="casAuthors">Bogomolov, Pavel; Alexandrov, Alexander; Voronkova, Natalia; Macievich, Maria; Kokina, Ksenia; Petrachenkova, Maria; Lehr, Thorsten; Lempp, Florian A.; Wedemeyer, Heiner; Haag, Mathias; Schwab, Matthias; Haefeli, Walter E.; Blank, Antje; Urban, Stephan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Hepatology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">490-498</span>CODEN:
                <span class="NLM_cas:coden">JOHEEC</span>;
        ISSN:<span class="NLM_cas:issn">0168-8278</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The therapeutic option for patients with chronic hepatitis delta virus infection (CHD) is limited to interferon alpha with rare curative outcome.  Myrcludex B is a first-in-class entry inhibitor inactivating the hepatitis B virus (HBV) and hepatitis D virus (HDV) receptor sodium taurocholate co-transporting polypeptide.  We report the interim results of a pilot trial on chronically infected HDV patients treated with myrcludex B, or pegylated interferon alpha (PegIFNα-2a) or their combination.  Twenty-four patients with CHD infection were equally randomized (1:1:1) to receive myrcludex B, or PegIFNα-2a or their combination.  Patients were evaluated for virol. and biochem. response and tolerability of the study drugs at weeks 12 and 24.  Myrcludex B was well tolerated and no serious adverse event occurred.  Although hepatitis B surface antigen levels remained unchanged, HDV RNA significantly declined at week 24 in all cohorts.  HDV RNA became neg. in two patients each in the Myrcludex B and PegIFNα-2a cohorts, and in five patients of the Myrcludex B + PegIFNα-2a cohort.  ALT decreased significantly in the Myrcludex B cohort (six of eight patients), and HBV DNA was significantly reduced at week 24 in the Myrcludex B + PegIFNα-2a cohort.  Virus kinetic modeling suggested a strong synergistic effect of myrcludex B and PegIFNα-2a on both HDV and HBV.  Myrcludex B showed a strong effect on HDV RNA serum levels and induced ALT normalization under monotherapy.  Synergistic antiviral effects on HDV RNA and HBV DNA in the Myr-IFN cohort indicated a benefit of the combination of entry inhibition with PegIFNα-2a to treat CHD patients.  Myrcludex B is a new drug to treat hepatitis B and D infection.  After 24 wk of treatment with myrcludex B and/or pegylated interferon α-2a, HDV RNA, a relevant marker for hepatitis D infection, decreased in all patients with chronic hepatitis B and D.  Two of eight patients which received either myrcludex B or pegylated interferon α-2a, became neg. for HDV RNA, and five of seven patients who received both drugs at the same time became neg.  The drug was well tolerated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo29Ewz7ZuCMrVg90H21EOLACvtfcHk0liiIyxE0BwGHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVynsbrE&md5=9b31bd4df1423aa2da1e49d41a89192a</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.jhep.2016.04.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jhep.2016.04.016%26sid%3Dliteratum%253Aachs%26aulast%3DBogomolov%26aufirst%3DP.%26aulast%3DAlexandrov%26aufirst%3DA.%26aulast%3DVoronkova%26aufirst%3DN.%26aulast%3DMacievich%26aufirst%3DM.%26aulast%3DKokina%26aufirst%3DK.%26aulast%3DPetrachenkova%26aufirst%3DM.%26aulast%3DLehr%26aufirst%3DT.%26aulast%3DLempp%26aufirst%3DF.%2BA.%26aulast%3DWedemeyer%26aufirst%3DH.%26aulast%3DHaag%26aufirst%3DM.%26aulast%3DSchwab%26aufirst%3DM.%26aulast%3DHaefeli%26aufirst%3DW.%2BE.%26aulast%3DBlank%26aufirst%3DA.%26aulast%3DUrban%26aufirst%3DS.%26atitle%3DTreatment%2520of%2520chronic%2520hepatitis%2520D%2520with%2520the%2520entry%2520inhibitor%2520myrcludex%2520B%253A%2520First%2520results%2520of%2520a%2520phase%2520Ib%252FIIa%2520study%26jtitle%3DJ.%2520Hepatol.%26date%3D2016%26volume%3D65%26spage%3D490%26epage%3D498%26doi%3D10.1016%2Fj.jhep.2016.04.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lipinski, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dominy, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feeney, P. J.</span></span> <span> </span><span class="NLM_article-title">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span>. <i>Adv. Drug Delivery Rev.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">26</span>, <span class="refDoi"> DOI: 10.1016/S0169-409X(00)00129-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1016%2FS0169-409X%2800%2900129-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=11259830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2001&pages=3-26&author=C.+A.+Lipinskiauthor=F.+Lombardoauthor=B.+W.+Dominyauthor=P.+J.+Feeney&title=Experimental+and+computational+approaches+to+estimate+solubility+and+permeability+in+drug+discovery+and+development+settings&doi=10.1016%2FS0169-409X%2800%2900129-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span></div><div class="casAuthors">Lipinski, C. A.; Lombardo, F.; Dominy, B. W.; Feeney, P. J.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">3-26</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ireland Ltd.</span>)
        </div><div class="casAbstract">A review with 50 refs.  Exptl. and computational approaches to est. soly. and permeability in discovery and development settings are described.  In the discovery setting 'the rule of 5' predicts that poor absorption or permeation is more likely when there are more than 5 H-bond donors, 10 H-bond acceptors, the mol. wt. (MWT) is greater than 500 and the calcd. Log P (CLogP) is greater than 5 (or MlogP >4.15).  Computational methodol. for the rule-based Moriguchi Log P (MLogP) calcn. is described.  Turbidimetric soly. measurement is described and applied to known drugs.  High throughput screening (HTS) leads tend to have higher MWT and Log P and lower turbidimetric soly. than leads in the pre-HTS era.  In the development setting, soly. calcns. focus on exact value prediction and are difficult because of polymorphism.  Recent work on linear free energy relationships and Log P approaches are critically reviewed.  Useful predictions are possible in closely related analog series when coupled with exptl. thermodn. soly. measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYDqa02g16urVg90H21EOLACvtfcHk0liiIyxE0BwGHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D&md5=c60bb89da68f051c0ee7ac4c0468a0e4</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2FS0169-409X%2800%2900129-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-409X%252800%252900129-0%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DDominy%26aufirst%3DB.%2BW.%26aulast%3DFeeney%26aufirst%3DP.%2BJ.%26atitle%3DExperimental%2520and%2520computational%2520approaches%2520to%2520estimate%2520solubility%2520and%2520permeability%2520in%2520drug%2520discovery%2520and%2520development%2520settings%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2001%26volume%3D46%26spage%3D3%26epage%3D26%26doi%3D10.1016%2FS0169-409X%2800%2900129-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">DeGoey, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wendt, M. D.</span></span> <span> </span><span class="NLM_article-title">Beyond the Rule of 5: Lessons Learned from AbbVie’s Drugs and Compound Collection</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">2636</span>– <span class="NLM_lpage">2651</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00717</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00717" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFeitbjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=2636-2651&author=D.+A.+DeGoeyauthor=H.+J.+Chenauthor=P.+B.+Coxauthor=M.+D.+Wendt&title=Beyond+the+Rule+of+5%3A+Lessons+Learned+from+AbbVie%E2%80%99s+Drugs+and+Compound+Collection&doi=10.1021%2Facs.jmedchem.7b00717"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Beyond the Rule of 5: Lessons Learned from AbbVie's Drugs and Compound Collection</span></div><div class="casAuthors">DeGoey, David A.; Chen, Hui-Ju; Cox, Philip B.; Wendt, Michael D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2636-2651</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Recently, there has been an increasing focus on the pursuit of targets considered to be less druggable that offer potential for development of promising new therapeutic agents for the treatment of diseases with large unmet medical need, particularly in the areas of oncol. and virol.  However, conducting drug discovery campaigns in "beyond rule of 5" (bRo5) chem. space presents a significant drug design and development challenge to medicinal chemists to achieve acceptable oral pharmacokinetics.  Retrospective anal. of past successes and failures in drug discovery bRo5 may shed light on the key principles that contribute to the oral bioavailability of successful bRo5 compds. and improve the efficiency of drug design for future projects.  We present here highlights and case studies of lessons learned from discovery of bRo5 compds.  A simple multiparametric scoring function (AB-MPS) was devised that correlated preclin. PK results with cLogD, no. of rotatable bonds, and no. of arom. rings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTHuaXbtjlALVg90H21EOLACvtfcHk0liiIyxE0BwGHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFeitbjJ&md5=48984dc65410d83d4ac8ddaca030b024</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00717&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00717%26sid%3Dliteratum%253Aachs%26aulast%3DDeGoey%26aufirst%3DD.%2BA.%26aulast%3DChen%26aufirst%3DH.%2BJ.%26aulast%3DCox%26aufirst%3DP.%2BB.%26aulast%3DWendt%26aufirst%3DM.%2BD.%26atitle%3DBeyond%2520the%2520Rule%2520of%25205%253A%2520Lessons%2520Learned%2520from%2520AbbVie%25E2%2580%2599s%2520Drugs%2520and%2520Compound%2520Collection%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D2636%26epage%3D2651%26doi%3D10.1021%2Facs.jmedchem.7b00717" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jing, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sui, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span> <span> </span><span class="NLM_article-title">Molecular determinants of hepatitis B and D virus entry restriction in mouse sodium taurocholate cotransporting polypeptide</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">7977</span>– <span class="NLM_lpage">7991</span>, <span class="refDoi"> DOI: 10.1128/JVI.03540-12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1128%2FJVI.03540-12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=23678176" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFWjt7zK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2013&pages=7977-7991&author=H.+Yanauthor=B.+Pengauthor=W.+Heauthor=G.+Zhongauthor=Y.+Qiauthor=B.+Renauthor=Z.+Gaoauthor=Z.+Jingauthor=M.+Songauthor=G.+Xuauthor=J.+Suiauthor=W.+Li&title=Molecular+determinants+of+hepatitis+B+and+D+virus+entry+restriction+in+mouse+sodium+taurocholate+cotransporting+polypeptide&doi=10.1128%2FJVI.03540-12"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular determinants of hepatitis B and D virus entry restriction in mouse sodium taurocholate cotransporting polypeptide</span></div><div class="casAuthors">Yan, Huan; Peng, Bo; He, Wenhui; Zhong, Guocai; Qi, Yonghe; Ren, Bijie; Gao, Zhenchao; Jing, Zhiyi; Song, Mei; Xu, Guangwei; Sui, Jianhua; Li, Wenhui</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">7977-7991</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Human hepatitis B virus (HBV) and its satellite virus, hepatitis D virus (HDV), primarily infect humans, chimpanzees, or tree shrews (Tupaia belangeri).  Viral infections in other species are known to be mainly restricted at the entry level since viral replication can be achieved in the cells by transfection of the viral genome.  Sodium taurocholate cotransporting polypeptide (NTCP) is a functional receptor for HBV and HDV, and amino acids 157 to 165 of NTCP are crit. for viral entry and likely limit viral infection of macaques.  However, the mol. determinants for viral entry restriction in mouse NTCP (mNTCP) remain unclear.  In this study, mNTCP was found to be unable to support either HBV or HDV infection, although it can bind to pre-S1 of HBV L protein and is functional in transporting substrate taurocholate; comprehensive swapping and point mutations of human NTCP (hNTCP) and mNTCP revealed mol. determinants restricting mNTCP for viral entry of HBV and HDV.  Remarkably, when mNTCP residues 84 to 87 were substituted by human counterparts, mNTCP can effectively support viral infections.  In addn., a no. of cell lines, regardless of their species or tissue origin, supported HDV infection when transfected with hNTCP or mNTCP with residues 84 to 87 replaced by human counterparts, highlighting the central role of NTCP for viral infections mediated by HBV envelope proteins.  These studies advance our understanding of NTCP-mediated viral entry of HBV and HDV and have important implications for developing the mouse model for their infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo181QjmyElmLVg90H21EOLACvtfcHk0liys7jNf_yBZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFWjt7zK&md5=59f5669452aad2a667b6dfd1ba54d525</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1128%2FJVI.03540-12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.03540-12%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DH.%26aulast%3DPeng%26aufirst%3DB.%26aulast%3DHe%26aufirst%3DW.%26aulast%3DZhong%26aufirst%3DG.%26aulast%3DQi%26aufirst%3DY.%26aulast%3DRen%26aufirst%3DB.%26aulast%3DGao%26aufirst%3DZ.%26aulast%3DJing%26aufirst%3DZ.%26aulast%3DSong%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DG.%26aulast%3DSui%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DW.%26atitle%3DMolecular%2520determinants%2520of%2520hepatitis%2520B%2520and%2520D%2520virus%2520entry%2520restriction%2520in%2520mouse%2520sodium%2520taurocholate%2520cotransporting%2520polypeptide%26jtitle%3DJ.%2520Virol.%26date%3D2013%26volume%3D87%26spage%3D7977%26epage%3D7991%26doi%3D10.1128%2FJVI.03540-12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rudman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kendall, F. E.</span></span> <span> </span><span class="NLM_article-title">Bile acid content of human serum. II. The binding of cholanic acids by human plasma proteins</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">1957</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">538</span>– <span class="NLM_lpage">542</span>, <span class="refDoi"> DOI: 10.1172/JCI103451</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1172%2FJCI103451" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=13416382" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADyaG2sXlslOnuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=1957&pages=538-542&author=D.+Rudmanauthor=F.+E.+Kendall&title=Bile+acid+content+of+human+serum.+II.+The+binding+of+cholanic+acids+by+human+plasma+proteins&doi=10.1172%2FJCI103451"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Bile acid content of human serum. II. The binding of cholanic acids by human plasma proteins</span></div><div class="casAuthors">Rudman, Daniel; Kendall, Forrest E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Investigation</span>
        (<span class="NLM_cas:date">1957</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">538-42</span>CODEN:
                <span class="NLM_cas:coden">JCINAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-9738</span>.
    </div><div class="casAbstract">The binding of bile acids and derivs. by the protein fractions of human plasma was studied by the dialysis-equil. method.  Albumin exhibits the greatest binding activity towards these compds.  The lipoprotein-contg. globulins, Cohn fractions III and IV-1 bind approx. half as much deoxycholic acid and cholic acid as does albumin. γ-Globulin and fibrinogen do not interact with the bile acids.  The affinity for albumin is reduced by the introduction of polar groups into the steroid nucleus.  Thus the extent of binding decreases in the order of monohydroxy > dihydroxy > trihydroxy cholanic acid.  Binding of the ketocholanic acids to albumin could not be detected.  The effect of pH on the binding reaction suggests that the primary attraction between albumin and the cholanic acids is an electrostatic bond between the pos. charged lysine and side chains of the former and the neg. charged carboxy groups of the latter.  The binding consts. of albumin with 2 bile acids are calcd.  53 refs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCDr5ugBl7ebVg90H21EOLACvtfcHk0liys7jNf_yBZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaG2sXlslOnuw%253D%253D&md5=4dc95127a15788b10799103fca3077db</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1172%2FJCI103451&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI103451%26sid%3Dliteratum%253Aachs%26aulast%3DRudman%26aufirst%3DD.%26aulast%3DKendall%26aufirst%3DF.%2BE.%26atitle%3DBile%2520acid%2520content%2520of%2520human%2520serum.%2520II.%2520The%2520binding%2520of%2520cholanic%2520acids%2520by%2520human%2520plasma%2520proteins%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D1957%26volume%3D36%26spage%3D538%26epage%3D542%26doi%3D10.1172%2FJCI103451" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mondal, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, S.</span></span> <span> </span><span class="NLM_article-title">Hydrophobicity is the governing factor in the interaction of human serum albumin with bile salts</span>. <i>Langmuir</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">1095</span>– <span class="NLM_lpage">1104</span>, <span class="refDoi"> DOI: 10.1021/la504270a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/la504270a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFyhtbzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2015&pages=1095-1104&author=N.+Ghoshauthor=R.+Mondalauthor=S.+Mukherjee&title=Hydrophobicity+is+the+governing+factor+in+the+interaction+of+human+serum+albumin+with+bile+salts&doi=10.1021%2Fla504270a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Hydrophobicity Is the Governing Factor in the Interaction of Human Serum Albumin with Bile Salts</span></div><div class="casAuthors">Ghosh, Narayani; Mondal, Ramakanta; Mukherjee, Saptarshi</div><div class="citationInfo"><span class="NLM_cas:title">Langmuir</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1095-1104</span>CODEN:
                <span class="NLM_cas:coden">LANGD5</span>;
        ISSN:<span class="NLM_cas:issn">0743-7463</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The present study demonstrates a detailed characterization of the interaction of a series of bile salts, sodium deoxycholate (NaDC), sodium cholate (NaC), and sodium taurocholate (NaTC), with a model transport protein, human serum albumin (HSA).  Here, steady-state and time-resolved fluorescence spectroscopic techniques have been used to characterize the interaction of the bile salts with HSA.  The binding isotherms constructed from steady-state fluorescence intensity measurements demonstrate that the interaction of the bile salts with HSA can be characterized by three distinct regions, which were also successfully reproduced from the significant variation of the emission wavelength (λem) of the intrinsic tryptophan (Trp) moiety of HSA.  The time-resolved fluorescence decay behavior of the Trp residue of HSA was also found to corroborate the steady-state results.  The effect of interaction with the bile salts on the native conformation of the protein has been explored in a CD study, which reveals a decrease in α-helicity of HSA induced by the bile salts.  In accordance with this, the esterase activity of the protein-bile salt aggregates is found to be reduced in comparison to that of the native protein.  Our results exclusively highlight the fact that it is the hydrophobic character of the bile salt that governs the extent of interaction with the protein.  Isothermal titrn. calorimetry (ITC) and mol. docking studies further substantiate our other exptl. findings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7AbgP5sfdFbVg90H21EOLACvtfcHk0liys7jNf_yBZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFyhtbzP&md5=c2df6db5bbb26f72fdb9bfd66061f00b</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fla504270a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fla504270a%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DN.%26aulast%3DMondal%26aufirst%3DR.%26aulast%3DMukherjee%26aufirst%3DS.%26atitle%3DHydrophobicity%2520is%2520the%2520governing%2520factor%2520in%2520the%2520interaction%2520of%2520human%2520serum%2520albumin%2520with%2520bile%2520salts%26jtitle%3DLangmuir%26date%3D2015%26volume%3D31%26spage%3D1095%26epage%3D1104%26doi%3D10.1021%2Fla504270a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jing, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sui, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span> <span> </span><span class="NLM_article-title">Hepatitis D Virus Infection of Mice Expressing Human Sodium Taurocholate Co-transporting Polypeptide</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>, <span class="NLM_elocation-id">e1004840</span> <span class="refDoi"> DOI: 10.1371/journal.ppat.1004840</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1371%2Fjournal.ppat.1004840" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=25902143" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&author=W.+Heauthor=B.+Renauthor=F.+Maoauthor=Z.+Jingauthor=Y.+Liauthor=Y.+Liuauthor=B.+Pengauthor=H.+Yanauthor=Y.+Qiauthor=Y.+Sunauthor=J.+T.+Guoauthor=J.+Suiauthor=F.+Wangauthor=W.+Li&title=Hepatitis+D+Virus+Infection+of+Mice+Expressing+Human+Sodium+Taurocholate+Co-transporting+Polypeptide&doi=10.1371%2Fjournal.ppat.1004840"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1004840&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1004840%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DW.%26aulast%3DRen%26aufirst%3DB.%26aulast%3DMao%26aufirst%3DF.%26aulast%3DJing%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DPeng%26aufirst%3DB.%26aulast%3DYan%26aufirst%3DH.%26aulast%3DQi%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DGuo%26aufirst%3DJ.%2BT.%26aulast%3DSui%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DW.%26atitle%3DHepatitis%2520D%2520Virus%2520Infection%2520of%2520Mice%2520Expressing%2520Human%2520Sodium%2520Taurocholate%2520Co-transporting%2520Polypeptide%26jtitle%3DPLoS%2520Pathog.%26date%3D2015%26volume%3D11%26doi%3D10.1371%2Fjournal.ppat.1004840" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0058.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0058.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0002.jpeg" id="rightTab-gr44" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Discovery of potent NTCP inhibitors: DBADs. (A) HBV infection inhibition assay for commercial TUDCA (labeled as TUDCA-(com)) and in-house synthesized TUDCA (TUDCA-(syn)). HepG2-NTCP cells were inoculated with HBV in the presence of the indicated compounds for 24 h. After the culture medium was replenished with fresh medium, the cells were cultured for another 4 days. The HBeAg titers secreted into the medium were measured at 5 dpi using a commercial ELISA kit. (B) Liquid chromatography–mass spectroscopy (LC–MS) analysis of TUDCA-(com) and TUDCA-(syn). The retention time window between 4 min and 8 min is shown. The peaks indicated by arrows illustrate two impurities in TUDCA-(com). (C) HBV infection inhibition assay and NTCP substrate uptake assay for 3-OAc-TUDCA. The HBV infection assay was conducted as in (A). The substrate uptake assay was conducted with HepG2-NTCP cells using <sup>3</sup>H-TCA as the substrate. (D) HBV infection inhibition assay and NTCP substrate uptake assay for dimeric TUDCA. The HBV infection and substrate uptake assays were as in (C).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0003.jpeg" id="rightTab-gr45" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Chemical structures of selected DBAD candidates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0004.jpeg" id="rightTab-gr46" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Antiviral activity of DBADs. (A) HBV infection inhibition assay for DBADs in HepG2-NTCP cells. The treatment and HBeAg-detection protocol here were similar to those in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A. (B) DBADs decrease intracellular HBV RNA in HepG2-NTCP cells. RNA was extracted from infected HepG2-NTCP cells and assessed using qPCR. (C) HDV infection inhibition assay for DBADs in HepG2-NTCP cells. The cells were inoculated with HBV in the presence of the indicated compounds (5 μM concentration) and incubated for 24 h. HDV delta antigen was stained using a homemade FITC-labeled antidelta antigen antibody at 5 dpi and imaged via confocal microscopy (Zeiss Meta). (D) VSV infection inhibition assay for DBADs in HepG2-NTCP cells. The cells were inoculated with VSV in the presence of the indicated compounds (1 μM concentration) and incubated for 24 h. Secreted HBeAg and HBsAg titers were detected using the same kit as those in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A. (E) Time-course study of DBAD inhibition of HBV infection. The indicated compounds were added to HepG2-NTCP cells at different time points. Secreted HBeAg was detected using the methods detailed in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A. (F) HBV Pre-S1 binding for DBADs. The FITC-labeled HBV Pre-S1 polypeptide was incubated with HepG2-NTCP cells in the presence of the indicated compounds; images were captured via confocal microscopy (Zeiss Meta).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0005.jpeg" id="rightTab-gr47" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. DBADs inhibit NTCP substrate uptake. (A) DBADs inhibit NTCP- and ASBT-mediated uptake of <sup>3</sup>H-TCA. HepG2 cells transfected with a plasmid expressing human NTCP or human-ASBT were used in substrate uptake assays with 2 μM of the indicated compounds using <sup>3</sup>H-TCA as a substrate. (B) DBADs inhibit NTCP and OATP1B1 uptake of <sup>3</sup>H-Estrone sulfate. HepG2 cells transfected with a plasmid expressing human NTCP or human-OATP1B1 were used in substrate uptake assays with 2 μM of the indicated compounds using <sup>3</sup>H-Estrone sulfate as a substrate. (C, D) Time course of DBAD inhibition of HBV infection and <sup>3</sup>H-TCA uptake in HepG2-NTCP cells. HepG2-NTCP cells were treated with 2 μM of the indicated DBADs at different time points. (C) DBADs inhibit HBV infection in HepG2-NTCP cells. Secreted HBeAg was measured using ELISA. (D) DBADs inhibit NTCP substrate uptake. <sup>3</sup>H-TCA was used as a substrate. (E) IC<sub>50</sub> determination of DBA-25 inhibiting substrate uptake of multiple NTCP homologues. HepG2 cells overexpressing indicated NTCP homologues were used in substrate uptake assays using <sup>3</sup>H-TCA as a substrate. (F) DBADs inhibit substrate uptake of multiple NTCP chimeras. HepG2 cells overexpressing indicated NTCP chimeras were used in substrate uptake assays using <sup>3</sup>H-TCA as a substrate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0006.jpeg" id="rightTab-gr48" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. In vivo activity evaluation of DBADs. (A) Chemical structures of DBA-1, DBA-26, DBA-22, DBA-38, and DBA-41, with key chemical modifications, compared with the previous generation highlighted in yellow and red circles. (B–D) IC<sub>50</sub> values of DBA-22 (B), DBA-38 (C), and DBA-41 (D) on inhibiting NTCP uptake of <sup>3</sup>H-TCA in HepG2-NTCP cells with different concentrations of FBS. (E) DBA-41 inhibits human NTCP (left panel) and mouse NTCP (right panel) uptake of <sup>3</sup>H-TCA in HepG2 cells with no serum, human serum, mouse serum, and bovine serum. (F) Total bile acids level responses in WT/hNTCP-tg mice treated with DBA-41. WT-C57B/L mice and hNTCP-tg-C57B/L mice were intraperitoneally administrated vehicle or DBA-41 50 mg/kg B.I.D. for 3 days. Mice serum samples were collected at the start point and end point. The total bile acid level was determined at Peking Union Lawke Center for Functional Medicine.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0007.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of <b>DBA-1</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0008.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of <b>DBA-2</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch3" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0009.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of <b>DBA-3</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch4" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0010.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of <b>DBA-4</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch5" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0011.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of <b>DBA-5</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch6" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0012.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of <b>DBA-6</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch7" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0013.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Synthesis of <b>DBA-7</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch8" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0014.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 8. Synthesis of <b>DBA-8</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch9" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0015.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 9. Synthesis of <b>DBA-9</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch10" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Scheme 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0016.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 10. Synthesis of <b>DBA-10</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch11" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Scheme 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0017.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 11. Synthesis of <b>DBA-11</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch12" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Scheme 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0018.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 12. Synthesis of <b>DBA-12</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch12"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch13" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Scheme 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0019.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 13. Synthesis of <b>DBA-13</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch13"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch14" data-index="19" class="article__inlineFigure"><h2 class="fig-label">Scheme 14</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0020.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0020.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 14. Synthesis of <b>DBA-14</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch14"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch15" data-index="20" class="article__inlineFigure"><h2 class="fig-label">Scheme 15</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0021.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0021.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 15. Synthesis of <b>DBA-15</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0021.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch15"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch16" data-index="21" class="article__inlineFigure"><h2 class="fig-label">Scheme 16</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0022.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0022.jpeg" id="rightTab-gr16" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 16. Synthesis of <b>DBA-16</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0022.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch16"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch17" data-index="22" class="article__inlineFigure"><h2 class="fig-label">Scheme 17</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0023.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0023.jpeg" id="rightTab-gr17" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 17. Synthesis of <b>DBA-17</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0023.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch17"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch18" data-index="23" class="article__inlineFigure"><h2 class="fig-label">Scheme 18</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0024.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0024.jpeg" id="rightTab-gr18" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 18. Synthesis of <b>DBA-18</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0024.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch18"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch19" data-index="24" class="article__inlineFigure"><h2 class="fig-label">Scheme 19</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0025.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0025.jpeg" id="rightTab-gr19" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 19. Synthesis of <b>DBA-19</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0025.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch19"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch20" data-index="25" class="article__inlineFigure"><h2 class="fig-label">Scheme 20</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0026.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0026.jpeg" id="rightTab-gr20" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 20. Synthesis of <b>DBA-20</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0026.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch20"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch21" data-index="26" class="article__inlineFigure"><h2 class="fig-label">Scheme 21</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0027.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0027.jpeg" id="rightTab-gr21" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 21. Synthesis of <b>DBA-21</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0027.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch21"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch22" data-index="27" class="article__inlineFigure"><h2 class="fig-label">Scheme 22</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0028.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0028.jpeg" id="rightTab-gr22" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 22. Synthesis of <b>DBA-22</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0028.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch22"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch23" data-index="28" class="article__inlineFigure"><h2 class="fig-label">Scheme 23</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0029.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0029.jpeg" id="rightTab-gr23" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 23. Synthesis of <b>DBA-23</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0029.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch23"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch24" data-index="29" class="article__inlineFigure"><h2 class="fig-label">Scheme 24</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0030.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0030.jpeg" id="rightTab-gr24" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 24. Synthesis of <b>DBA-24</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0030.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch24"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch25" data-index="30" class="article__inlineFigure"><h2 class="fig-label">Scheme 25</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0031.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0031.jpeg" id="rightTab-gr25" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 25. Synthesis of <b>DBA-25</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0031.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch25"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch26" data-index="31" class="article__inlineFigure"><h2 class="fig-label">Scheme 26</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0032.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0032.jpeg" id="rightTab-gr26" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 26. Synthesis of <b>DBA-26</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0032.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch26"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch27" data-index="32" class="article__inlineFigure"><h2 class="fig-label">Scheme 27</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0033.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0033.jpeg" id="rightTab-gr27" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 27. Synthesis of <b>DBA-27</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0033.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch27"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch28" data-index="33" class="article__inlineFigure"><h2 class="fig-label">Scheme 28</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0034.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0034.jpeg" id="rightTab-gr28" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 28. Synthesis of <b>DBA-28</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0034.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch28"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch29" data-index="34" class="article__inlineFigure"><h2 class="fig-label">Scheme 29</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0035.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0035.jpeg" id="rightTab-gr29" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 29. Synthesis of <b>DBA-29</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0035.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch29"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch30" data-index="35" class="article__inlineFigure"><h2 class="fig-label">Scheme 30</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0036.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0036.jpeg" id="rightTab-gr30" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 30. Synthesis of <b>DBA-30</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0036.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch30"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch31" data-index="36" class="article__inlineFigure"><h2 class="fig-label">Scheme 31</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0037.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0037.jpeg" id="rightTab-gr31" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 31. Synthesis of <b>DBA-31</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0037.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch31"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch32" data-index="37" class="article__inlineFigure"><h2 class="fig-label">Scheme 32</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0038.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0038.jpeg" id="rightTab-gr32" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 32. Synthesis of <b>DBA-32</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0038.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch32"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch33" data-index="38" class="article__inlineFigure"><h2 class="fig-label">Scheme 33</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0039.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0039.jpeg" id="rightTab-gr33" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 33. Synthesis of <b>DBA-33</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0039.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch33"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch34" data-index="39" class="article__inlineFigure"><h2 class="fig-label">Scheme 34</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0040.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0040.jpeg" id="rightTab-gr34" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 34. Synthesis of <b>DBA-34</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0040.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch34"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch35" data-index="40" class="article__inlineFigure"><h2 class="fig-label">Scheme 35</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0041.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0041.jpeg" id="rightTab-gr35" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 35. Synthesis of <b>DBA-35</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0041.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch35"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch36" data-index="41" class="article__inlineFigure"><h2 class="fig-label">Scheme 36</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0042.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0042.jpeg" id="rightTab-gr36" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 36. Synthesis of <b>DBA-36</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0042.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch36"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch37" data-index="42" class="article__inlineFigure"><h2 class="fig-label">Scheme 37</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0043.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0043.jpeg" id="rightTab-gr37" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 37. Synthesis of <b>DBA-37</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0043.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch37"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch38" data-index="43" class="article__inlineFigure"><h2 class="fig-label">Scheme 38</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0044.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0044.jpeg" id="rightTab-gr38" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 38. Synthesis of <b>DBA-38</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0044.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch38"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch39" data-index="44" class="article__inlineFigure"><h2 class="fig-label">Scheme 39</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0045.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0045.jpeg" id="rightTab-gr39" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 39. Synthesis of <b>DBA-39</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0045.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch39"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch40" data-index="45" class="article__inlineFigure"><h2 class="fig-label">Scheme 40</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0046.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0046.jpeg" id="rightTab-gr40" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 40. Synthesis of <b>DBA-40</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0046.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch40"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch41" data-index="46" class="article__inlineFigure"><h2 class="fig-label">Scheme 41</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0047.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0047.jpeg" id="rightTab-gr41" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 41. Synthesis of <b>DBA-41</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0047.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch41"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch42" data-index="47" class="article__inlineFigure"><h2 class="fig-label">Scheme 42</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0048.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0048.jpeg" id="rightTab-gr42" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 42. Synthesis of <b>NQL-064</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0048.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch42"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch43" data-index="48" class="article__inlineFigure"><h2 class="fig-label">Scheme 43</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/medium/jm1c00078_0049.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0049.jpeg" id="rightTab-gr43" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 43. Synthesis of <b>NQL-065</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.1c00078/20210506/images/large/jm1c00078_0049.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00078&amp;id=sch43"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i74">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_68213" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_68213" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 26 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Razavi-Shearer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gamkrelidze, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, M. H.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Global prevalence, treatment, and prevention of hepatitis B virus infection in 2016: a modelling study</span>. <i>Lancet Gastroenterol. Hepatol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">383</span>– <span class="NLM_lpage">403</span>, <span class="refDoi"> DOI: 10.1016/S2468-1253(18)30056-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1016%2FS2468-1253%2818%2930056-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=29599078" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2018&pages=383-403&author=D.+Razavi-Shearerauthor=I.+Gamkrelidzeauthor=M.+H.+Nguyen&title=Global+prevalence%2C+treatment%2C+and+prevention+of+hepatitis+B+virus+infection+in+2016%3A+a+modelling+study&doi=10.1016%2FS2468-1253%2818%2930056-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2FS2468-1253%2818%2930056-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS2468-1253%252818%252930056-6%26sid%3Dliteratum%253Aachs%26aulast%3DRazavi-Shearer%26aufirst%3DD.%26aulast%3DGamkrelidze%26aufirst%3DI.%26aulast%3DNguyen%26aufirst%3DM.%2BH.%26atitle%3DGlobal%2520prevalence%252C%2520treatment%252C%2520and%2520prevention%2520of%2520hepatitis%2520B%2520virus%2520infection%2520in%25202016%253A%2520a%2520modelling%2520study%26jtitle%3DLancet%2520Gastroenterol.%2520Hepatol.%26date%3D2018%26volume%3D3%26spage%3D383%26epage%3D403%26doi%3D10.1016%2FS2468-1253%2818%2930056-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beasley, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chien, C. S.</span></span> <span> </span><span class="NLM_article-title">Hepatocellular carcinoma and hepatitis B virus. A prospective study of 22 707 men in Taiwan</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">1981</span>,  <span class="NLM_volume"><i>318</i></span>,  <span class="NLM_fpage">1129</span>– <span class="NLM_lpage">1133</span>, <span class="refDoi"> DOI: 10.1016/S0140-6736(81)90585-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1016%2FS0140-6736%2881%2990585-7" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=318&publication_year=1981&pages=1129-1133&author=R.+P.+Beasleyauthor=L.+Y.+Hwangauthor=C.+C.+Linauthor=C.+S.+Chien&title=Hepatocellular+carcinoma+and+hepatitis+B+virus.+A+prospective+study+of+22+707+men+in+Taiwan&doi=10.1016%2FS0140-6736%2881%2990585-7"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2881%2990585-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252881%252990585-7%26sid%3Dliteratum%253Aachs%26aulast%3DBeasley%26aufirst%3DR.%2BP.%26aulast%3DHwang%26aufirst%3DL.%2BY.%26aulast%3DLin%26aufirst%3DC.%2BC.%26aulast%3DChien%26aufirst%3DC.%2BS.%26atitle%3DHepatocellular%2520carcinoma%2520and%2520hepatitis%2520B%2520virus.%2520A%2520prospective%2520study%2520of%252022%2520707%2520men%2520in%2520Taiwan%26jtitle%3DLancet%26date%3D1981%26volume%3D318%26spage%3D1129%26epage%3D1133%26doi%3D10.1016%2FS0140-6736%2881%2990585-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jen, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, G. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iloeje, U. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Group, R.-H. S.</span></span> <span> </span><span class="NLM_article-title">Risk of hepatocellular carcinoma across a biological gradient of serum hepatitis B virus DNA level</span>. <i>JAMA</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>295</i></span>,  <span class="NLM_fpage">65</span>– <span class="NLM_lpage">73</span>, <span class="refDoi"> DOI: 10.1001/jama.295.1.65</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1001%2Fjama.295.1.65" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=16391218" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlKqtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=295&publication_year=2006&pages=65-73&author=C.-J.+Chenauthor=H.+I.+Yangauthor=J.+Suauthor=C.+L.+Jenauthor=S.+L.+Youauthor=S.+N.+Luauthor=G.+T.+Huangauthor=U.+H.+Iloejeauthor=R.-H.+S.+Group&title=Risk+of+hepatocellular+carcinoma+across+a+biological+gradient+of+serum+hepatitis+B+virus+DNA+level&doi=10.1001%2Fjama.295.1.65"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Risk of hepatocellular carcinoma across a biological gradient of serum hepatitis B virus DNA level</span></div><div class="casAuthors">Chen, Chien-Jen; Yang, Hwai-I.; Su, Jun; Jen, Chin-Lan; You, San-Lin; Lu, Sheng-Nan; Huang, Guan-Tarn; Iloeje, Uchenna H.</div><div class="citationInfo"><span class="NLM_cas:title">JAMA, the Journal of the American Medical Association</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">295</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">65-73</span>CODEN:
                <span class="NLM_cas:coden">JAMAAP</span>;
        ISSN:<span class="NLM_cas:issn">0098-7484</span>.
    
            (<span class="NLM_cas:orgname">American Medical Association</span>)
        </div><div class="casAbstract">Serum hepatitis B virus (HBV) DNA level is a marker of viral replication and efficacy of antiviral treatment in individuals with chronic hepatitis B.  To evaluate the relationship between serum HBV DNA level and risk of hepatocellular carcinoma.  Prospective cohort study of 3653 participants (aged 30-65 years), who were seropos. for the hepatitis B surface antigen and seroneg. for antibodies against the hepatitis C virus, recruited to a community-based cancer screening program in Taiwan between 1991 and 1992.  Incidence of hepatocellular carcinoma during follow-up examn. and by data linkage with the national cancer registry and the death certification systems.  There were 164 incident cases of hepatocellular carcinoma and 346 deaths during a mean follow-up of 11.4 years and 41 779 person-years of follow-up.  The incidence of hepatocellular carcinoma increased with serum HBV DNA level at study entry in a dose-response relationship ranging from 108 per 100.000 person-years for an HBV DNA level of less than 300 copies/mL to 1152 per 100.000 person-years for an HBV DNA level of 1 million copies/mL or greater.  The corresponding cumulative incidence rates of hepatocellular carcinoma were 1.3% and 14.9%, resp.  The biol. gradient of hepatocellular carcinoma by serum HBV DNA levels remained significant (P < .001) after adjustment for sex, age, cigarette smoking, alc. consumption, serostatus for the hepatitis B e antigen (HBeAg), serum alanine aminotransferase level, and liver cirrhosis at study entry.  The dose-response relationship was most prominent for participants who were seroneg. for HBeAg with normal serum alanine aminotransferase levels and no liver cirrhosis at study entry.  Participants with persistent elevation of serum HBV DNA level during follow-up had the highest hepatocellular carcinoma risk.  Elevated serum HBV DNA level (-10000 copies/mL) is a strong risk predictor of hepatocellular carcinoma independent of HBeAg, serum alanine aminotransferase level, and liver cirrhosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofTLNTbgs5X7Vg90H21EOLACvtfcHk0lgbThc2-LJBJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlKqtQ%253D%253D&md5=3fb9792cfca230106e13a8c405382ed7</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1001%2Fjama.295.1.65&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1001%252Fjama.295.1.65%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DC.-J.%26aulast%3DYang%26aufirst%3DH.%2BI.%26aulast%3DSu%26aufirst%3DJ.%26aulast%3DJen%26aufirst%3DC.%2BL.%26aulast%3DYou%26aufirst%3DS.%2BL.%26aulast%3DLu%26aufirst%3DS.%2BN.%26aulast%3DHuang%26aufirst%3DG.%2BT.%26aulast%3DIloeje%26aufirst%3DU.%2BH.%26aulast%3DGroup%26aufirst%3DR.-H.%2BS.%26atitle%3DRisk%2520of%2520hepatocellular%2520carcinoma%2520across%2520a%2520biological%2520gradient%2520of%2520serum%2520hepatitis%2520B%2520virus%2520DNA%2520level%26jtitle%3DJAMA%26date%3D2006%26volume%3D295%26spage%3D65%26epage%3D73%26doi%3D10.1001%2Fjama.295.1.65" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sureau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guerra, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanford, R. E.</span></span> <span> </span><span class="NLM_article-title">Role of the large hepatitis B virus envelope protein in infectivity of the hepatitis delta virion</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">366</span>– <span class="NLM_lpage">372</span>, <span class="refDoi"> DOI: 10.1128/JVI.67.1.366-372.1993</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1128%2FJVI.67.1.366-372.1993" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=8416375" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADyaK3sXlsleitA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=1993&pages=366-372&author=C.+Sureauauthor=B.+Guerraauthor=R.+E.+Lanford&title=Role+of+the+large+hepatitis+B+virus+envelope+protein+in+infectivity+of+the+hepatitis+delta+virion&doi=10.1128%2FJVI.67.1.366-372.1993"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Role of the large hepatitis B virus envelope protein in infectivity of the hepatitis delta virion</span></div><div class="casAuthors">Sureau, Camille; Guerra, Bernadette; Lanford, Robert E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">366-72</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    </div><div class="casAbstract">The hepatitis delta virus (HDV) is coated with large (L), middle (M), and small (S) envelope proteins encoded by coinfecting hepatitis B virus (HBV).  To study the role of the HBV envelope proteins in the assembly and infectivity of HDV, we produced three types of recombinant particles in Huh7 cells by transfection with HBV DNA and HDV cDNA: (i) particles with an envelope contg. the S HBV envelope protein only, (ii) particles with an envelope contg. S and M proteins, and (iii) particles with an envelope contg. S, M, and L proteins.  Although the resulting S-, SM-, and SML-HDV particles contained both hepatitis delta antigen and HDV RNA, only particles coated with all three envelope proteins (SML) showed evidence of infectivity in an in vitro culture system susceptible to HDV infection.  We concluded that the L HBV envelope protein, and more specifically the pre- S1 domain, is important for infectivity of HDV particles and that the M protein, which has been reported to bear a site for binding to polymd. albumin in the pre-S2 domain, is not sufficient for infectivity.  Our data also show that the helper HBV is not required for initiation of HDV infection.  The mechanism by which the L protein may affect HDV infectivity is discussed herein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiHc4X4_5QV7Vg90H21EOLACvtfcHk0lgbThc2-LJBJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXlsleitA%253D%253D&md5=3e436e69840f20e918c08cfbc7c8418f</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1128%2FJVI.67.1.366-372.1993&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.67.1.366-372.1993%26sid%3Dliteratum%253Aachs%26aulast%3DSureau%26aufirst%3DC.%26aulast%3DGuerra%26aufirst%3DB.%26aulast%3DLanford%26aufirst%3DR.%2BE.%26atitle%3DRole%2520of%2520the%2520large%2520hepatitis%2520B%2520virus%2520envelope%2520protein%2520in%2520infectivity%2520of%2520the%2520hepatitis%2520delta%2520virion%26jtitle%3DJ.%2520Virol.%26date%3D1993%26volume%3D67%26spage%3D366%26epage%3D372%26doi%3D10.1128%2FJVI.67.1.366-372.1993" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sureau, C.</span></span> <span> </span><span class="NLM_article-title">The role of the HBV envelope proteins in the HDV replication cycle</span>. <i>Curr. Top. Microbiol. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>307</i></span>,  <span class="NLM_fpage">113</span>– <span class="NLM_lpage">131</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=16903223" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADC%252BD28XosFeqsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=307&publication_year=2006&pages=113-131&author=C.+Sureau&title=The+role+of+the+HBV+envelope+proteins+in+the+HDV+replication+cycle"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The role of the HBV envelope proteins in the HDV replication cycle</span></div><div class="casAuthors">Sureau, C.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Microbiology and Immunology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">307</span>
        (<span class="NLM_cas:issue">Hepatitis Delta Virus</span>),
    <span class="NLM_cas:pages">113-131</span>CODEN:
                <span class="NLM_cas:coden">CTMIA3</span>;
        ISSN:<span class="NLM_cas:issn">0070-217X</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">A review.  The hepatitis delta virus (HDV) is a subviral agent that utilizes the envelope proteins of the hepatitis B virus (HBV) for propagation.  When introduced into permissive cells, the HDV RNA genome replicates and assocs. with multiple copies of the HDV-encoded proteins to assemble a ribonucleoprotein (RNP) complex.  The mechanism necessary to export the RNP from the cell is provided by the HBV envelope proteins, which have the capacity to assemble lipoprotein vesicles that bud into the lumen of a pre-Golgi compartment before being secreted.  In addn. to allowing the release of the HDV RNP, the HBV envelope proteins also provide a means for its targeting to an uninfected cell, thereby ensuring the spread of HDV.  This chapter covers the mol. aspects of the HBV envelope protein functions in the HDV replication cycle, in particular the activity of the small envelope protein in RNP export and the function of the large envelope protein at viral entry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoEJaSXW9Fg7Vg90H21EOLACvtfcHk0ljOqesUY7DTUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XosFeqsro%253D&md5=aff046b7f493724aaaec0288b4173826</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSureau%26aufirst%3DC.%26atitle%3DThe%2520role%2520of%2520the%2520HBV%2520envelope%2520proteins%2520in%2520the%2520HDV%2520replication%2520cycle%26jtitle%3DCurr.%2520Top.%2520Microbiol.%2520Immunol.%26date%3D2006%26volume%3D307%26spage%3D113%26epage%3D131" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jing, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sui, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span> <span> </span><span class="NLM_article-title">Sodium taurocholate cotransporting polypeptide is a functional receptor for human hepatitis B and D virus</span>. <i>Elife</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>1</i></span>, <span class="NLM_elocation-id">e00049</span> <span class="refDoi"> DOI: 10.7554/eLife.00049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.7554%2FeLife.00049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=23150796" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht12ms7nK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2012&author=H.+Yanauthor=G.+Zhongauthor=G.+Xuauthor=W.+Heauthor=Z.+Jingauthor=Z.+Gaoauthor=Y.+Huangauthor=Y.+Qiauthor=B.+Pengauthor=H.+Wangauthor=L.+Fuauthor=M.+Songauthor=P.+Chenauthor=W.+Gaoauthor=B.+Renauthor=Y.+Sunauthor=T.+Caiauthor=X.+Fengauthor=J.+Suiauthor=W.+Li&title=Sodium+taurocholate+cotransporting+polypeptide+is+a+functional+receptor+for+human+hepatitis+B+and+D+virus&doi=10.7554%2FeLife.00049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Sodium taurocholate cotransporting polypeptide is a functional receptor for human hepatitis B and D virus</span></div><div class="casAuthors">Yan, Huan; Zhong, Guocai; Xu, Guangwei; He, Wenhui; Jing, Zhiyi; Gao, Zhenchao; Huang, Yi; Qi, Yonghe; Peng, Bo; Wang, Haimin; Fu, Liran; Song, Mei; Chen, Pan; Gao, Wenqing; Ren, Bijie; Sun, Yinyan; Cai, Tao; Feng, Xiaofeng; Sui, Jianhua; Li, Wenhui</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e00049, 28 pp.</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">Human hepatitis B virus (HBV) infection and HBV-related diseases remain a major public health problem.  Individuals coinfected with its satellite hepatitis D virus (HDV) have more severe disease.  Cellular entry of both viruses is mediated by HBV envelope proteins.  The pre-S1 domain of the large envelope protein is a key determinant for receptor(s) binding.  However, the identity of the receptor(s) is unknown.  Here, by using near zero distance photo-crosslinking and tandem affinity purifn., the authors revealed that the receptor-binding region of pre-S1 specifically interacts with sodium taurocholate cotransporting polypeptide (NTCP), a multiple transmembrane transporter predominantly expressed in the liver.  Silencing NTCP inhibited HBV and HDV infection, while exogenous NTCP expression rendered nonsusceptible hepatocarcinoma cells susceptible to these viral infections.  Moreover, replacing amino acids 157-165 of nonfunctional monkey NTCP with the human counterpart conferred its ability in supporting both viral infections.  The authors' results demonstrate that NTCP is a functional receptor for HBV and HDV.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrY0tqZhmqs37Vg90H21EOLACvtfcHk0ljOqesUY7DTUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht12ms7nK&md5=7380905d92bd82b9cf3407b7c113bdfb</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.7554%2FeLife.00049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.00049%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DH.%26aulast%3DZhong%26aufirst%3DG.%26aulast%3DXu%26aufirst%3DG.%26aulast%3DHe%26aufirst%3DW.%26aulast%3DJing%26aufirst%3DZ.%26aulast%3DGao%26aufirst%3DZ.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DQi%26aufirst%3DY.%26aulast%3DPeng%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DFu%26aufirst%3DL.%26aulast%3DSong%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DP.%26aulast%3DGao%26aufirst%3DW.%26aulast%3DRen%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DCai%26aufirst%3DT.%26aulast%3DFeng%26aufirst%3DX.%26aulast%3DSui%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DW.%26atitle%3DSodium%2520taurocholate%2520cotransporting%2520polypeptide%2520is%2520a%2520functional%2520receptor%2520for%2520human%2520hepatitis%2520B%2520and%2520D%2520virus%26jtitle%3DElife%26date%3D2012%26volume%3D1%26doi%3D10.7554%2FeLife.00049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hagenbuch, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stieger, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foguet, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lubbert, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meier, P. J.</span></span> <span> </span><span class="NLM_article-title">Functional expression cloning and characterization of the hepatocyte Na+/bile acid cotransport system</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>88</i></span>,  <span class="NLM_fpage">10629</span>– <span class="NLM_lpage">10633</span>, <span class="refDoi"> DOI: 10.1073/pnas.88.23.10629</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1073%2Fpnas.88.23.10629" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1961729" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADyaK3sXnsFGjuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=1991&pages=10629-10633&author=B.+Hagenbuchauthor=B.+Stiegerauthor=M.+Foguetauthor=H.+Lubbertauthor=P.+J.+Meier&title=Functional+expression+cloning+and+characterization+of+the+hepatocyte+Na%2B%2Fbile+acid+cotransport+system&doi=10.1073%2Fpnas.88.23.10629"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Functional expression cloning and characterization of the hepatocyte sodium/bile acid cotransport system</span></div><div class="casAuthors">Hagenbuch, Bruno; Stieger, Bruno; Foguet, Montserrat; Luebbert, Hermann; Meier, Peter J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10629-33</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">A cDNA encoding the rat liver bile acid uptake system was isolated by expression cloning in Xenopus laevis oocytes.  The cloned transporter is strictly sodium-dependent and can be inhibited by various nonbile acid org. compds.  Sequence anal. of the cDNA revealed an open reading frame of 1086 nucleotides coding for a protein of 362 amino acids (calcd. mol. mass 39 kDa) with five possible N-linked glycosylation sites and seven putative transmembrane domains.  Translation expts. in vitro and in oocytes indicate that the transporter is indeed glycosylated and that its polypeptide backbone has an apparent mol. mass of 33-35 kDa.  Northern blot anal. with the cloned probe revealed crossreactivity with mRNA species from rat kidney and intestine as well as from liver tissues of mouse, guinea pig, rabbit, and man.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptcBpGcNwgibVg90H21EOLACvtfcHk0ljOqesUY7DTUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXnsFGjuw%253D%253D&md5=b5cc0c3d427d9e7df22d92e5f92d412d</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1073%2Fpnas.88.23.10629&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.88.23.10629%26sid%3Dliteratum%253Aachs%26aulast%3DHagenbuch%26aufirst%3DB.%26aulast%3DStieger%26aufirst%3DB.%26aulast%3DFoguet%26aufirst%3DM.%26aulast%3DLubbert%26aufirst%3DH.%26aulast%3DMeier%26aufirst%3DP.%2BJ.%26atitle%3DFunctional%2520expression%2520cloning%2520and%2520characterization%2520of%2520the%2520hepatocyte%2520Na%252B%252Fbile%2520acid%2520cotransport%2520system%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1991%26volume%3D88%26spage%3D10629%26epage%3D10633%26doi%3D10.1073%2Fpnas.88.23.10629" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hagenbuch, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meier, P. J.</span></span> <span> </span><span class="NLM_article-title">Molecular cloning, chromosomal localization, and functional characterization of a human liver Na+/bile acid cotransporter</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">1326</span>– <span class="NLM_lpage">1331</span>, <span class="refDoi"> DOI: 10.1172/JCI117091</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1172%2FJCI117091" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=8132774" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADyaK2cXjtFKqtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=1994&pages=1326-1331&author=B.+Hagenbuchauthor=P.+J.+Meier&title=Molecular+cloning%2C+chromosomal+localization%2C+and+functional+characterization+of+a+human+liver+Na%2B%2Fbile+acid+cotransporter&doi=10.1172%2FJCI117091"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular cloning, chromosomal localization, and functional characterization of a human liver Na+/bile acid cotransporter</span></div><div class="casAuthors">Hagenbuch, Bruno; Meier, Peter J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Investigation</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1326-31</span>CODEN:
                <span class="NLM_cas:coden">JCINAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-9738</span>.
    </div><div class="casAbstract">A cDNA probe from a cloned rat liver Na+/taurocholate-cotransporting polypeptide (Ntcp) was used to screen a human liver cDNA library.  A 1599-bp cDNA clone that encodes a human Na+/taurocholate cotransporting polypeptide (NTCP) was isolated.  The human NTCP consists of 349 amino acids (calcd. mol. mass of 38 kDa) and exhibits 77% amino acid homol. with the rat Ntcp.  In vitro translation expts. indicate that the protein is glycosylated and has a mol. wt. similar to the rat Ntcp.  Injection of in vitro transcribed cRNA into Xenopus laevis oocytes resulted in the expression of Na+-dependent taurocholate uptake.  Satn. kinetics indicated that the human NTCP has a higher affinity for taurocholate (apparent Km = 6 μM) than the previously cloned rat protein (apparent Km = 25 μM).  NTCP-mediated taurocholate uptake into oocytes was inhibited by all major bile acid derivs. (100 μM), bumetanide (500 μM), and bromosulfophthalein (100 μM).  Southern blot anal. of genomic DNA from a panel of human/hamster somatic cell hybrids mapped the human NTCP gene to chromosome 14.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqmQsr-mlVh7Vg90H21EOLACvtfcHk0lj8P4RNt3KOJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXjtFKqtLk%253D&md5=1e49aa2044dce90f56b3671d4cca53c4</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1172%2FJCI117091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI117091%26sid%3Dliteratum%253Aachs%26aulast%3DHagenbuch%26aufirst%3DB.%26aulast%3DMeier%26aufirst%3DP.%2BJ.%26atitle%3DMolecular%2520cloning%252C%2520chromosomal%2520localization%252C%2520and%2520functional%2520characterization%2520of%2520a%2520human%2520liver%2520Na%252B%252Fbile%2520acid%2520cotransporter%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D1994%26volume%3D93%26spage%3D1326%26epage%3D1331%26doi%3D10.1172%2FJCI117091" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span> <span> </span><span class="NLM_article-title">DNA Polymerase kappa Is a Key Cellular Factor for the Formation of Covalently Closed Circular DNA of Hepatitis B Virus</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e1005893</span> <span class="refDoi"> DOI: 10.1371/journal.ppat.1005893</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1371%2Fjournal.ppat.1005893" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=27783675" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFCnu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&author=Y.+Qiauthor=Z.+Gaoauthor=G.+Xuauthor=B.+Pengauthor=C.+Liuauthor=H.+Yanauthor=Q.+Yaoauthor=G.+Sunauthor=Y.+Liuauthor=D.+Tangauthor=Z.+Songauthor=W.+Heauthor=Y.+Sunauthor=J.+T.+Guoauthor=W.+Li&title=DNA+Polymerase+kappa+Is+a+Key+Cellular+Factor+for+the+Formation+of+Covalently+Closed+Circular+DNA+of+Hepatitis+B+Virus&doi=10.1371%2Fjournal.ppat.1005893"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">DNA polymerase κ is a key cellular factor for the formation of covalently closed circular DNA of hepatitis B virus</span></div><div class="casAuthors">Qi, Yonghe; Gao, Zhenchao; Xu, Guangwei; Peng, Bo; Liu, Chenxuan; Yan, Huan; Yao, Qiyan; Sun, Guoliang; Liu, Yang; Tang, Dingbin; Song, Zilin; He, Wenhui; Sun, Yinyan; Guo, Ju-Tao; Li, Wenhui</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Pathogens</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e1005893/1-e1005893/26</span>CODEN:
                <span class="NLM_cas:coden">PPLACN</span>;
        ISSN:<span class="NLM_cas:issn">1553-7374</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Hepatitis B virus (HBV) infection of hepatocytes begins by binding to its cellular receptor sodium taurocholate cotransporting polypeptide (NTCP), followed by the internalization of viral nucleocapsid into the cytoplasm.  The viral relaxed circular (rc) DNA genome in nucleocapsid is transported into the nucleus and converted into covalently closed circular (ccc) DNA to serve as a viral persistence reservoir that is refractory to current antiviral therapies.  Host DNA repair enzymes have been speculated to catalyze the conversion of rcDNA to cccDNA, however, the DNA polymerase(s) that fills the gap in the plus strand of rcDNA remains to be detd.  Here we conducted targeted genetic screening in combination with chem. inhibition to identify the cellular DNA polymerase(s) responsible for cccDNA formation, and exploited recombinant HBV with capsid coding deficiency which infects HepG2-NTCP cells with similar efficiency of wild-type HBV to assure cccDNA synthesis is exclusively from de novo HBV infection.  We found that DNA polymerase κ (POLK), a Y-family DNA polymerase with max. activity in non-dividing cells, substantially contributes to cccDNA formation during de novo HBV infection.  Depleting gene expression of POLK in HepG2-NTCP cells by either siRNA knockdown or CRISPR/Cas9 knockout inhibited the conversion of rcDNA into cccDNA, while the diminished cccDNA formation in, and hence the viral infection of, the knockout cells could be effectively rescued by ectopic expression of POLK.  These studies revealed that POLK is a crucial host factor required for cccDNA formation during a de novo HBV infection and suggest that POLK may be a potential target for developing antivirals against HBV.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD8pcGjwB5mLVg90H21EOLACvtfcHk0lj8P4RNt3KOJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFCnu7g%253D&md5=5a72d28fa023cb3f51dd976606f59f07</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1005893&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1005893%26sid%3Dliteratum%253Aachs%26aulast%3DQi%26aufirst%3DY.%26aulast%3DGao%26aufirst%3DZ.%26aulast%3DXu%26aufirst%3DG.%26aulast%3DPeng%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DYan%26aufirst%3DH.%26aulast%3DYao%26aufirst%3DQ.%26aulast%3DSun%26aufirst%3DG.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DTang%26aufirst%3DD.%26aulast%3DSong%26aufirst%3DZ.%26aulast%3DHe%26aufirst%3DW.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DGuo%26aufirst%3DJ.%2BT.%26aulast%3DLi%26aufirst%3DW.%26atitle%3DDNA%2520Polymerase%2520kappa%2520Is%2520a%2520Key%2520Cellular%2520Factor%2520for%2520the%2520Formation%2520of%2520Covalently%2520Closed%2520Circular%2520DNA%2520of%2520Hepatitis%2520B%2520Virus%26jtitle%3DPLoS%2520Pathog.%26date%3D2016%26volume%3D12%26doi%3D10.1371%2Fjournal.ppat.1005893" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">König, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doring, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohr, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geipel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geyer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glebe, D.</span></span> <span> </span><span class="NLM_article-title">Kinetics of the bile acid transporter and hepatitis B virus receptor Na+/taurocholate cotransporting polypeptide (NTCP) in hepatocytes</span>. <i>J. Hepatol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">867</span>– <span class="NLM_lpage">875</span>, <span class="refDoi"> DOI: 10.1016/j.jhep.2014.05.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1016%2Fj.jhep.2014.05.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=24845614" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A280%3ADC%252BC2cjkslSgug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2014&pages=867-875&author=A.+K%C3%B6nigauthor=B.+Doringauthor=C.+Mohrauthor=A.+Geipelauthor=J.+Geyerauthor=D.+Glebe&title=Kinetics+of+the+bile+acid+transporter+and+hepatitis+B+virus+receptor+Na%2B%2Ftaurocholate+cotransporting+polypeptide+%28NTCP%29+in+hepatocytes&doi=10.1016%2Fj.jhep.2014.05.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetics of the bile acid transporter and hepatitis B virus receptor Na+/taurocholate cotransporting polypeptide (NTCP) in hepatocytes</span></div><div class="casAuthors">Konig Alexander; Mohr Christina; Geipel Andreas; Doring Barbara; Geyer Joachim; Glebe Dieter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of hepatology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">867-75</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND & AIMS:  The human liver bile acid transporter Na(+)/taurocholate cotransporting polypeptide (NTCP) has recently been identified as liver-specific receptor for infection of hepatitis B virus (HBV), which attaches via the myristoylated preS1 (myr-preS1) peptide domain of its large surface protein to NTCP.  Since binding of the myr-preS1 peptide to NTCP is an initiating step of HBV infection, we investigated if this process interferes with the physiological bile acid transport function of NTCP.  METHODS:  HBV infection, myr-preS1 peptide binding, and bile acid transport assays were performed with primary Tupaia belangeri (PTH) and human (PHH) hepatocytes as well as NTCP-transfected human hepatoma HepG2 cells allowing regulated NTCP expression, in the presence of various bile acids, ezetimibe, and myr-preS1 peptides.  RESULTS:  The myr-preS1 peptide of HBV inhibited bile acid transport in PTH and PHH as well as in NTCP-expressing HEK293 and HepG2 cells.  Inversely, HBV infection of PTH, PHH, and NTCP-transfected HepG2 cells was inhibited in a concentration-dependent manner by taurine and glycine conjugates of cholic acid and ursodeoxycholic acid as well as by ezetimibe.  In NTCP-HepG2 cells and PTH, NTCP expression, NTCP transport function, myr-preS1 peptide binding, and HBV infection followed comparable kinetics.  CONCLUSIONS:  Myr-preS1 virus binding to NTCP, necessary for productive HBV infection, interferes with the physiological bile acid transport function of NTCP.  Therefore, HBV infection via NTCP may be lockable by NTCP substrates and NTCP-inhibiting drugs.  This opens a completely new way for an efficient management of HBV infection by the use of NTCP-directed drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQL8uPrQ26gtXtlzo9xastyfW6udTcc2eZVb9r-Y2td77ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cjkslSgug%253D%253D&md5=46106af8958ba2cd35a321f29fe58970</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.jhep.2014.05.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jhep.2014.05.018%26sid%3Dliteratum%253Aachs%26aulast%3DK%25C3%25B6nig%26aufirst%3DA.%26aulast%3DDoring%26aufirst%3DB.%26aulast%3DMohr%26aufirst%3DC.%26aulast%3DGeipel%26aufirst%3DA.%26aulast%3DGeyer%26aufirst%3DJ.%26aulast%3DGlebe%26aufirst%3DD.%26atitle%3DKinetics%2520of%2520the%2520bile%2520acid%2520transporter%2520and%2520hepatitis%2520B%2520virus%2520receptor%2520Na%252B%252Ftaurocholate%2520cotransporting%2520polypeptide%2520%2528NTCP%2529%2520in%2520hepatocytes%26jtitle%3DJ.%2520Hepatol.%26date%3D2014%26volume%3D61%26spage%3D867%26epage%3D875%26doi%3D10.1016%2Fj.jhep.2014.05.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jing, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sui, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span> <span> </span><span class="NLM_article-title">Viral entry of hepatitis B and D viruses and bile salts transportation share common molecular determinants on sodium taurocholate cotransporting polypeptide</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>88</i></span>,  <span class="NLM_fpage">3273</span>– <span class="NLM_lpage">3284</span>, <span class="refDoi"> DOI: 10.1128/JVI.03478-13</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1128%2FJVI.03478-13" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=24390325" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsVOqtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=2014&pages=3273-3284&author=H.+Yanauthor=B.+Pengauthor=Y.+Liuauthor=G.+Xuauthor=W.+Heauthor=B.+Renauthor=Z.+Jingauthor=J.+Suiauthor=W.+Li&title=Viral+entry+of+hepatitis+B+and+D+viruses+and+bile+salts+transportation+share+common+molecular+determinants+on+sodium+taurocholate+cotransporting+polypeptide&doi=10.1128%2FJVI.03478-13"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Viral entry of hepatitis B and D viruses and bile salts transportation share common molecular determinants on sodium taurocholate cotransporting polypeptide</span></div><div class="casAuthors">Yan, Huan; Peng, Bo; Liu, Yang; Xu, Guangwei; He, Wenhui; Ren, Bijie; Jing, Zhiyi; Sui, Jianhua; Li, Wenhui</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3273-3284, 13 pp.</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">1098-5514</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The liver bile acids transporter sodium taurocholate cotransporting polypeptide (NTCP) is responsible for the majority of sodium-dependent bile salts uptake by hepatocytes.  NTCP also functions as a cellular receptor for viral entry of hepatitis B virus (HBV) and hepatitis D virus (HDV) through a specific interaction between NTCP and the pre-S1 domain of HBV large envelope protein.  However, it remains unknown if these two functions of NTCP are independent or if they interfere with each other.  Here we show that binding of the pre-S1 domain to human NTCP blocks taurocholate uptake by the receptor; conversely, some bile acid substrates of NTCP inhibit HBV and HDV entry.  Mutations of NTCP residues crit. for bile salts binding severely impair viral infection by HDV and HBV; to a lesser extent, the residues important for sodium binding also inhibit viral infection.  The mutation S267F, corresponding to a single nucleotide polymorphism (SNP) found in about 9% of the East Asian population, renders NTCP without either taurocholate transporting activity or the ability to support HBV or HDV infection in cell culture.  These results demonstrate that mol. determinants crit. for HBV and HDV entry overlap with that for bile salts uptake by NTCP, indicating that viral infection may interfere with the normal function of NTCP and bile acids and their derivs. hold the potential for further development into antiviral drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsAWAqX4p6L7Vg90H21EOLACvtfcHk0lh-AJOtCYGJ5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsVOqtr4%253D&md5=22fe09e746ad3f6ee76002a316de82b1</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1128%2FJVI.03478-13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.03478-13%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DH.%26aulast%3DPeng%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DG.%26aulast%3DHe%26aufirst%3DW.%26aulast%3DRen%26aufirst%3DB.%26aulast%3DJing%26aufirst%3DZ.%26aulast%3DSui%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DW.%26atitle%3DViral%2520entry%2520of%2520hepatitis%2520B%2520and%2520D%2520viruses%2520and%2520bile%2520salts%2520transportation%2520share%2520common%2520molecular%2520determinants%2520on%2520sodium%2520taurocholate%2520cotransporting%2520polypeptide%26jtitle%3DJ.%2520Virol.%26date%3D2014%26volume%3D88%26spage%3D3273%26epage%3D3284%26doi%3D10.1128%2FJVI.03478-13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Okuyama-Dobashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasai, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamashita, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yasumoto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maekawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakita, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enomoto, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriishi, K.</span></span> <span> </span><span class="NLM_article-title">Hepatitis B virus efficiently infects non-adherent hepatoma cells via human sodium taurocholate cotransporting polypeptide</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>, <span class="NLM_elocation-id">17047</span> <span class="refDoi"> DOI: 10.1038/srep17047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1038%2Fsrep17047" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=26592202" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFensbnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&author=K.+Okuyama-Dobashiauthor=H.+Kasaiauthor=T.+Tanakaauthor=A.+Yamashitaauthor=J.+Yasumotoauthor=W.+Chenauthor=T.+Okamotoauthor=S.+Maekawaauthor=K.+Watashiauthor=T.+Wakitaauthor=A.+Ryoauthor=T.+Suzukiauthor=Y.+Matsuuraauthor=N.+Enomotoauthor=K.+Moriishi&title=Hepatitis+B+virus+efficiently+infects+non-adherent+hepatoma+cells+via+human+sodium+taurocholate+cotransporting+polypeptide&doi=10.1038%2Fsrep17047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Hepatitis B virus efficiently infects non-adherent hepatoma cells via human sodium taurocholate cotransporting polypeptide</span></div><div class="casAuthors">Okuyama-Dobashi, Kaori; Kasai, Hirotake; Tanaka, Tomohisa; Yamashita, Atsuya; Yasumoto, Jun; Chen, Wenjia; Okamoto, Toru; Maekawa, Shinya; Watashi, Koichi; Wakita, Takaji; Ryo, Akihide; Suzuki, Tetsuro; Matsuura, Yoshiharu; Enomoto, Nobuyuki; Moriishi, Kohji</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">17047</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Sodium taurocholate cotransporting polypeptide (NTCP) has been reported as a functional receptor for hepatitis B virus (HBV) infection.  However, HBV could not efficiently infect HepG2 cells expressing NTCP (NTCP-HepG2 cells) under adherent monolayer-cell conditions.  In this study, NTCP was mainly detected in the basolateral membrane region, but not the apical site, of monolayer NTCP-HepG2 cells.  We hypothesized that non-adherent cell conditions of infection would enhance HBV infectivity.  Non-adherent NTCP-HepG2 cells were prepd. by treatment with trypsin and EDTA, which did not degrade NTCP in the membrane fraction.  HBV successfully infected NTCP-HepG2 cells at a viral dose 10 times lower in non-adherent phase than in adherent phase.  Efficient infection of non-adherent NTCP-HepG2 cells with blood-borne or cell-culture-derived HBV was obsd. and was remarkably impaired in the presence of the myristoylated preS1 peptide.  HBV could also efficiently infect HepaRG cells under non-adherent cell conditions.  We screened several compds. using our culture system and identified proscillaridin A as a potent anti-HBV agent with an IC50 value of 7.2 nM.  In conclusion, non-adherent host cell conditions of infection augmented HBV infectivity in an NTCP-dependent manner, thus providing a novel strategy to identify anti-HBV drugs and investigate the mechanism of HBV infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGow9Dq2g5ytTLVg90H21EOLACvtfcHk0lh-AJOtCYGJ5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFensbnJ&md5=0990ab0c427e276fcea727a037e17c9d</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fsrep17047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep17047%26sid%3Dliteratum%253Aachs%26aulast%3DOkuyama-Dobashi%26aufirst%3DK.%26aulast%3DKasai%26aufirst%3DH.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DYamashita%26aufirst%3DA.%26aulast%3DYasumoto%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DOkamoto%26aufirst%3DT.%26aulast%3DMaekawa%26aufirst%3DS.%26aulast%3DWatashi%26aufirst%3DK.%26aulast%3DWakita%26aufirst%3DT.%26aulast%3DRyo%26aufirst%3DA.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DMatsuura%26aufirst%3DY.%26aulast%3DEnomoto%26aufirst%3DN.%26aulast%3DMoriishi%26aufirst%3DK.%26atitle%3DHepatitis%2520B%2520virus%2520efficiently%2520infects%2520non-adherent%2520hepatoma%2520cells%2520via%2520human%2520sodium%2520taurocholate%2520cotransporting%2520polypeptide%26jtitle%3DSci.%2520Rep.%26date%3D2015%26volume%3D5%26doi%3D10.1038%2Fsrep17047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaneko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamisuki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsunaga, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawai, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohashi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsukuda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aizaki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugiyama, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohtani, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugawara, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizokami, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sureau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakita, T.</span></span> <span> </span><span class="NLM_article-title">A Novel Tricyclic Polyketide, Vanitaracin A, Specifically Inhibits the Entry of Hepatitis B and D Viruses by Targeting Sodium Taurocholate Cotransporting Polypeptide</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">11945</span>– <span class="NLM_lpage">11953</span>, <span class="refDoi"> DOI: 10.1128/JVI.01855-15</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1128%2FJVI.01855-15" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=26378168" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADC%252BC28XjsFyiur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2015&pages=11945-11953&author=M.+Kanekoauthor=K.+Watashiauthor=S.+Kamisukiauthor=H.+Matsunagaauthor=M.+Iwamotoauthor=F.+Kawaiauthor=H.+Ohashiauthor=S.+Tsukudaauthor=S.+Shimuraauthor=R.+Suzukiauthor=H.+Aizakiauthor=M.+Sugiyamaauthor=S.+Y.+Parkauthor=T.+Itoauthor=N.+Ohtaniauthor=F.+Sugawaraauthor=Y.+Tanakaauthor=M.+Mizokamiauthor=C.+Sureauauthor=T.+Wakita&title=A+Novel+Tricyclic+Polyketide%2C+Vanitaracin+A%2C+Specifically+Inhibits+the+Entry+of+Hepatitis+B+and+D+Viruses+by+Targeting+Sodium+Taurocholate+Cotransporting+Polypeptide&doi=10.1128%2FJVI.01855-15"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">A novel tricyclic polyketide, vanitaracin A, specifically inhibits the entry of hepatitis B and D viruses by targeting sodium taurocholate cotransporting polypeptide</span></div><div class="casAuthors">Kaneko, Manabu; Watashi, Koichi; Kamisuki, Shinji; Matsunaga, Hiroki; Iwamoto, Masashi; Kawai, Fumihiro; Ohashi, Hirofumi; Tsukuda, Senko; Shimura, Satomi; Suzuki, Ryosuke; Aizaki, Hideki; Sugiyama, Masaya; Park, Sam-Yong; Ito, Takayoshi; Ohtani, Naoko; Sugawara, Fumio; Tanaka, Yasuhito; Mizokami, Masashi; Sureau, Camille; Wakita, Takaji</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">11945-11953</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">1098-5514</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Anti-hepatitis B virus (HBV) drugs are currently limited to nucleos(t)ide analogs (NAs) and interferons.  A challenge of drug development is the identification of small mols. that suppress HBV infection from new chem. sources.  Here, from a fungus-derived secondary metabolite library, we identify a structurally novel tricyclic polyketide, named vanitaracin A, which specifically inhibits HBV infection.  Vanitaracin A inhibited the viral entry process with a submicromolar 50% inhibitory concn. (IC50) (IC50 = 0.61 ± 0.23 μM), without evident cytotoxicity (50% cytotoxic concn. of >256 μM; selectivity index value of >419) in primary human hepatocytes.  Vanitaracin A did not affect the HBV replication process.  This compd. was found to directly interact with the HBV entry receptor sodium taurocholate cotransporting polypeptide (NTCP) and impaired its bile acid transport activity.  Consistent with this NTCP targeting, antiviral activity of vanitaracin A was obsd. with hepatitis D virus (HDV) but not hepatitis C virus.  Importantly, vanitaracin A inhibited infection by all HBV genotypes tested (genotypes A to D) and clin. relevant NA-resistant HBV isolate.  Thus, we identified a fungal metabolite, vanitaracin A, which was a potent, well-tolerated, and broadly active inhibitor of HBV and HDV entry.  This compd., or its related analogs, could be part of an antiviral strategy for preventing reinfection with HBV, including clin. relevant nucleos(t)ide analog-resistant virus.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiy8gZypawebVg90H21EOLACvtfcHk0ljriwz4r7-LuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjsFyiur0%253D&md5=a474e23328a4f350415610cb08697512</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1128%2FJVI.01855-15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.01855-15%26sid%3Dliteratum%253Aachs%26aulast%3DKaneko%26aufirst%3DM.%26aulast%3DWatashi%26aufirst%3DK.%26aulast%3DKamisuki%26aufirst%3DS.%26aulast%3DMatsunaga%26aufirst%3DH.%26aulast%3DIwamoto%26aufirst%3DM.%26aulast%3DKawai%26aufirst%3DF.%26aulast%3DOhashi%26aufirst%3DH.%26aulast%3DTsukuda%26aufirst%3DS.%26aulast%3DShimura%26aufirst%3DS.%26aulast%3DSuzuki%26aufirst%3DR.%26aulast%3DAizaki%26aufirst%3DH.%26aulast%3DSugiyama%26aufirst%3DM.%26aulast%3DPark%26aufirst%3DS.%2BY.%26aulast%3DIto%26aufirst%3DT.%26aulast%3DOhtani%26aufirst%3DN.%26aulast%3DSugawara%26aufirst%3DF.%26aulast%3DTanaka%26aufirst%3DY.%26aulast%3DMizokami%26aufirst%3DM.%26aulast%3DSureau%26aufirst%3DC.%26aulast%3DWakita%26aufirst%3DT.%26atitle%3DA%2520Novel%2520Tricyclic%2520Polyketide%252C%2520Vanitaracin%2520A%252C%2520Specifically%2520Inhibits%2520the%2520Entry%2520of%2520Hepatitis%2520B%2520and%2520D%2520Viruses%2520by%2520Targeting%2520Sodium%2520Taurocholate%2520Cotransporting%2520Polypeptide%26jtitle%3DJ.%2520Virol.%26date%3D2015%26volume%3D89%26spage%3D11945%26epage%3D11953%26doi%3D10.1128%2FJVI.01855-15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nkongolo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ni, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lempp, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufman, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esser-Nobis, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohmann, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mier, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehrle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urban, S.</span></span> <span> </span><span class="NLM_article-title">Cyclosporin A inhibits hepatitis B and hepatitis D virus entry by cyclophilin-independent interference with the NTCP receptor</span>. <i>J. Hepatol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">723</span>– <span class="NLM_lpage">731</span>, <span class="refDoi"> DOI: 10.1016/j.jhep.2013.11.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1016%2Fj.jhep.2013.11.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=24295872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtlagtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2014&pages=723-731&author=S.+Nkongoloauthor=Y.+Niauthor=F.+A.+Lemppauthor=C.+Kaufmanauthor=T.+Lindnerauthor=K.+Esser-Nobisauthor=V.+Lohmannauthor=W.+Mierauthor=S.+Mehrleauthor=S.+Urban&title=Cyclosporin+A+inhibits+hepatitis+B+and+hepatitis+D+virus+entry+by+cyclophilin-independent+interference+with+the+NTCP+receptor&doi=10.1016%2Fj.jhep.2013.11.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclosporin A inhibits hepatitis B and hepatitis D virus entry by cyclophilin-independent interference with the NTCP receptor</span></div><div class="casAuthors">Nkongolo, Shirin; Ni, Yi; Lempp, Florian A.; Kaufman, Christina; Lindner, Thomas; Esser-Nobis, Katharina; Lohmann, Volker; Mier, Walter; Mehrle, Stefan; Urban, Stephan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Hepatology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">723-731</span>CODEN:
                <span class="NLM_cas:coden">JOHEEC</span>;
        ISSN:<span class="NLM_cas:issn">0168-8278</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Chronic hepatitis B and hepatitis D are global health problems caused by the human hepatitis B and hepatitis D virus.  The myristoylated preS1 domain of the large envelope protein mediates specific binding to hepatocytes by sodium taurocholate co-transporting polypeptide (NTCP).  NTCP is a bile salt transporter known to be inhibited by cyclosporin A.  This study aimed to characterize the effect of cyclosporin A on HBV/HDV infection.  HepaRG cells, primary human hepatocytes, and susceptible NTCP-expressing hepatoma cell lines were applied for infection expts.  The mode of action of cyclosporin A was studied by comparing the effect of different inhibitors, cyclophilin A/B/C-silenced cell lines as well as NTCP variants and mutants.  Bile salt transporter and HBV receptor functions were investigated by taurocholate uptake and quantification of HBVpreS binding.Cyclosporin A inhibited hepatitis B and D virus infections during and - less pronounced - prior to virus inoculation.  Binding of HBVpreS to NTCP was blocked by cyclosporin A concns. at 8 μM.  An NTCP variant deficient in HBVpreS binding but competent for bile salt transport showed resistance to cyclosporin A.  Silencing of cyclophilins A/B/C did not abrogate transporter and receptor inhibition.  In contrast, tacrolimus, a cyclophilin-independent calcineurin inhibitor, was inactive.HBV and HDV entry via sodium taurocholate co-transporting polypeptide is inhibited by cyclosporin A.  The interaction between the drug and the viral receptor is direct and overlaps with a functional binding site of the preS1 domain, which mediates viral entry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTAVAhQe6_ErVg90H21EOLACvtfcHk0ljriwz4r7-LuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtlagtw%253D%253D&md5=d8c7e3698ac926cd5c06a9e7e2c4717b</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.jhep.2013.11.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jhep.2013.11.022%26sid%3Dliteratum%253Aachs%26aulast%3DNkongolo%26aufirst%3DS.%26aulast%3DNi%26aufirst%3DY.%26aulast%3DLempp%26aufirst%3DF.%2BA.%26aulast%3DKaufman%26aufirst%3DC.%26aulast%3DLindner%26aufirst%3DT.%26aulast%3DEsser-Nobis%26aufirst%3DK.%26aulast%3DLohmann%26aufirst%3DV.%26aulast%3DMier%26aufirst%3DW.%26aulast%3DMehrle%26aufirst%3DS.%26aulast%3DUrban%26aufirst%3DS.%26atitle%3DCyclosporin%2520A%2520inhibits%2520hepatitis%2520B%2520and%2520hepatitis%2520D%2520virus%2520entry%2520by%2520cyclophilin-independent%2520interference%2520with%2520the%2520NTCP%2520receptor%26jtitle%3DJ.%2520Hepatol.%26date%3D2014%26volume%3D60%26spage%3D723%26epage%3D731%26doi%3D10.1016%2Fj.jhep.2013.11.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Watashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sluder, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daito, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsunaga, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagamori, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakajima, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsukuda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borroto-Esoda, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugiyama, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kusuhara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizokami, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakita, T.</span></span> <span> </span><span class="NLM_article-title">Cyclosporin A and its analogs inhibit hepatitis B virus entry into cultured hepatocytes through targeting a membrane transporter, sodium taurocholate cotransporting polypeptide (NTCP)</span>. <i>Hepatology</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1726</span>– <span class="NLM_lpage">1737</span>, <span class="refDoi"> DOI: 10.1002/hep.26982</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1002%2Fhep.26982" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=24375637" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADC%252BC2cXntFCltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2014&pages=1726-1737&author=K.+Watashiauthor=A.+Sluderauthor=T.+Daitoauthor=S.+Matsunagaauthor=A.+Ryoauthor=S.+Nagamoriauthor=M.+Iwamotoauthor=S.+Nakajimaauthor=S.+Tsukudaauthor=K.+Borroto-Esodaauthor=M.+Sugiyamaauthor=Y.+Tanakaauthor=Y.+Kanaiauthor=H.+Kusuharaauthor=M.+Mizokamiauthor=T.+Wakita&title=Cyclosporin+A+and+its+analogs+inhibit+hepatitis+B+virus+entry+into+cultured+hepatocytes+through+targeting+a+membrane+transporter%2C+sodium+taurocholate+cotransporting+polypeptide+%28NTCP%29&doi=10.1002%2Fhep.26982"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclosporin A and its analogs inhibit hepatitis B virus entry into cultured hepatocytes through targeting a membrane transporter, sodium taurocholate cotransporting polypeptide (NTCP)</span></div><div class="casAuthors">Watashi, Koichi; Sluder, Ann; Daito, Takuji; Matsunaga, Satoko; Ryo, Akihide; Nagamori, Shushi; Iwamoto, Masashi; Nakajima, Syo; Tsukuda, Senko; Borroto-Esoda, Katyna; Sugiyama, Masaya; Tanaka, Yasuhito; Kanai, Yoshikatsu; Kusuhara, Hiroyuki; Mizokami, Masashi; Wakita, Takaji</div><div class="citationInfo"><span class="NLM_cas:title">Hepatology (Hoboken, NJ, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1726-1737</span>CODEN:
                <span class="NLM_cas:coden">HPTLD9</span>;
        ISSN:<span class="NLM_cas:issn">0270-9139</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Chronic hepatitis B virus (HBV) infection is a major public health problem worldwide.  Although nucleos(t)ide analogs inhibiting viral reverse transcriptase are clin. available as anti-HBV agents, emergence of drug-resistant viruses highlights the need for new anti-HBV agents interfering with other targets.  Here we report that cyclosporin A (CsA) can inhibit HBV entry into cultured hepatocytes.  The anti-HBV effect of CsA was independent of binding to cyclophilin and calcineurin.  Rather, blockade of HBV infection correlated with the ability to inhibit the transporter activity of sodium taurocholate cotransporting polypeptide (NTCP).  We also found that HBV infection-susceptible cells, differentiated HepaRG cells and primary human hepatocytes expressed NTCP, while nonsusceptible cell lines did not.  A series of compds. targeting NTCP could inhibit HBV infection.  CsA inhibited the binding between NTCP and large envelope protein in vitro.  Evaluation of CsA analogs identified a compd. with higher anti-HBV potency, having a median inhibitory concn. <0.2 μM.  Conclusion: This study provides a proof of concept for the novel strategy to identify anti-HBV agents by targeting the candidate HBV receptor, NTCP, using CsA as a structural platform. (Hepatol. 2014;59:1726-1737).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRV21kvExmObVg90H21EOLACvtfcHk0ljriwz4r7-LuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXntFCltro%253D&md5=d83ae63fbe3e8b827f9fbf5f6601b316</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1002%2Fhep.26982&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fhep.26982%26sid%3Dliteratum%253Aachs%26aulast%3DWatashi%26aufirst%3DK.%26aulast%3DSluder%26aufirst%3DA.%26aulast%3DDaito%26aufirst%3DT.%26aulast%3DMatsunaga%26aufirst%3DS.%26aulast%3DRyo%26aufirst%3DA.%26aulast%3DNagamori%26aufirst%3DS.%26aulast%3DIwamoto%26aufirst%3DM.%26aulast%3DNakajima%26aufirst%3DS.%26aulast%3DTsukuda%26aufirst%3DS.%26aulast%3DBorroto-Esoda%26aufirst%3DK.%26aulast%3DSugiyama%26aufirst%3DM.%26aulast%3DTanaka%26aufirst%3DY.%26aulast%3DKanai%26aufirst%3DY.%26aulast%3DKusuhara%26aufirst%3DH.%26aulast%3DMizokami%26aufirst%3DM.%26aulast%3DWakita%26aufirst%3DT.%26atitle%3DCyclosporin%2520A%2520and%2520its%2520analogs%2520inhibit%2520hepatitis%2520B%2520virus%2520entry%2520into%2520cultured%2520hepatocytes%2520through%2520targeting%2520a%2520membrane%2520transporter%252C%2520sodium%2520taurocholate%2520cotransporting%2520polypeptide%2520%2528NTCP%2529%26jtitle%3DHepatology%26date%3D2014%26volume%3D59%26spage%3D1726%26epage%3D1737%26doi%3D10.1002%2Fhep.26982" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shimura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukano, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sluder, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawai, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsukuda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeuchi, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyake, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugiyama, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogasawara, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kusuhara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizokami, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sureau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakita, T.</span></span> <span> </span><span class="NLM_article-title">Cyclosporin derivatives inhibit hepatitis B virus entry without interfering with NTCP transporter activity</span>. <i>J. Hepatol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">685</span>– <span class="NLM_lpage">692</span>, <span class="refDoi"> DOI: 10.1016/j.jhep.2016.11.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1016%2Fj.jhep.2016.11.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=27890789" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFGis7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2017&pages=685-692&author=S.+Shimuraauthor=K.+Watashiauthor=K.+Fukanoauthor=M.+Peelauthor=A.+Sluderauthor=F.+Kawaiauthor=M.+Iwamotoauthor=S.+Tsukudaauthor=J.+S.+Takeuchiauthor=T.+Miyakeauthor=M.+Sugiyamaauthor=Y.+Ogasawaraauthor=S.+Y.+Parkauthor=Y.+Tanakaauthor=H.+Kusuharaauthor=M.+Mizokamiauthor=C.+Sureauauthor=T.+Wakita&title=Cyclosporin+derivatives+inhibit+hepatitis+B+virus+entry+without+interfering+with+NTCP+transporter+activity&doi=10.1016%2Fj.jhep.2016.11.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclosporin derivatives inhibit hepatitis B virus entry without interfering with NTCP transporter activity</span></div><div class="casAuthors">Shimura, Satomi; Watashi, Koichi; Fukano, Kento; Peel, Michael; Sluder, Ann; Kawai, Fumihiro; Iwamoto, Masashi; Tsukuda, Senko; Takeuchi, Junko S.; Miyake, Takeshi; Sugiyama, Masaya; Ogasawara, Yuki; Park, Sam-Yong; Tanaka, Yasuhito; Kusuhara, Hiroyuki; Mizokami, Masashi; Sureau, Camille; Wakita, Takaji</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Hepatology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">685-692</span>CODEN:
                <span class="NLM_cas:coden">JOHEEC</span>;
        ISSN:<span class="NLM_cas:issn">0168-8278</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The sodium taurocholate co-transporting polypeptide (NTCP) is the main target of most hepatitis B virus (HBV) specific entry inhibitors.  Unfortunately, these agents also block NTCP transport of bile acids into hepatocytes, and thus have the potential to cause adverse effects.  We aimed to identify small mols. that inhibit HBV entry while maintaining NTCP transporter function.  We characterized a series of cyclosporine (CsA) derivs. for their anti-HBV activity and NTCP binding specificity using HepG2 cells overexpressing NTCP and primary human hepatocytes.  The four most potent derivs. were tested for their capacity to prevent HBV entry, but maintain NTCP transporter function.  Their antiviral activity against different HBV genotypes was analyzed.We identified several CsA derivs. that inhibited HBV infection with a sub-micromolar IC50.  Among them, SCY446 and SCY450 showed low activity against calcineurin (CN) and cyclophilins (CyPs), two major CsA cellular targets.  This suggested that instead, these compds. interacted directly with NTCP to inhibit viral attachment to host cells, and have no immunosuppressive function.  Importantly, we found that SCY450 and SCY995 did not impair the NTCP-dependent uptake of bile acids, and inhibited multiple HBV genotypes including a clin. relevant nucleoside analog-resistant HBV isolate.  This is the first example of small mol. selective inhibition of HBV entry with no decrease in NTCP transporter activity.  It suggests that the anti-HBV activity can be functionally sepd. from bile acid transport.  These broadly active anti-HBV mols. are potential candidates for developing new drugs with fewer adverse effects.  In this study, we identified new compds. that selectively inhibited hepatitis B virus (HBV) entry, and did not impair bile acid uptake.  Our evidence offers a new strategy for developing anti-HBV drugs with fewer side effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqK9TO_Uo2M-bVg90H21EOLACvtfcHk0ljt8qAw7U1gpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFGis7%252FO&md5=7cb0015063c0a8c2016283d875076478</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.jhep.2016.11.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jhep.2016.11.009%26sid%3Dliteratum%253Aachs%26aulast%3DShimura%26aufirst%3DS.%26aulast%3DWatashi%26aufirst%3DK.%26aulast%3DFukano%26aufirst%3DK.%26aulast%3DPeel%26aufirst%3DM.%26aulast%3DSluder%26aufirst%3DA.%26aulast%3DKawai%26aufirst%3DF.%26aulast%3DIwamoto%26aufirst%3DM.%26aulast%3DTsukuda%26aufirst%3DS.%26aulast%3DTakeuchi%26aufirst%3DJ.%2BS.%26aulast%3DMiyake%26aufirst%3DT.%26aulast%3DSugiyama%26aufirst%3DM.%26aulast%3DOgasawara%26aufirst%3DY.%26aulast%3DPark%26aufirst%3DS.%2BY.%26aulast%3DTanaka%26aufirst%3DY.%26aulast%3DKusuhara%26aufirst%3DH.%26aulast%3DMizokami%26aufirst%3DM.%26aulast%3DSureau%26aufirst%3DC.%26aulast%3DWakita%26aufirst%3DT.%26atitle%3DCyclosporin%2520derivatives%2520inhibit%2520hepatitis%2520B%2520virus%2520entry%2520without%2520interfering%2520with%2520NTCP%2520transporter%2520activity%26jtitle%3DJ.%2520Hepatol.%26date%3D2017%26volume%3D66%26spage%3D685%26epage%3D692%26doi%3D10.1016%2Fj.jhep.2016.11.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lucifora, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esser, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Protzer, U.</span></span> <span> </span><span class="NLM_article-title">Ezetimibe blocks hepatitis B virus infection after virus uptake into hepatocytes</span>. <i>Antiviral Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">195</span>– <span class="NLM_lpage">197</span>, <span class="refDoi"> DOI: 10.1016/j.antiviral.2012.12.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1016%2Fj.antiviral.2012.12.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=23266293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFSrsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2013&pages=195-197&author=J.+Luciforaauthor=K.+Esserauthor=U.+Protzer&title=Ezetimibe+blocks+hepatitis+B+virus+infection+after+virus+uptake+into+hepatocytes&doi=10.1016%2Fj.antiviral.2012.12.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Ezetimibe blocks hepatitis B virus infection after virus uptake into hepatocytes</span></div><div class="casAuthors">Lucifora, Julie; Esser, Knud; Protzer, Ulrike</div><div class="citationInfo"><span class="NLM_cas:title">Antiviral Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">195-197</span>CODEN:
                <span class="NLM_cas:coden">ARSRDR</span>;
        ISSN:<span class="NLM_cas:issn">0166-3542</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Current treatment of chronic hepatitis B virus (HBV) infection mainly targets viral replication in hepatocytes and leads to curing only in exceptional cases.  Despite their potential to improve therapeutic success, no drugs interfering with early infection steps of the hepatotropic pathogen HBV are available to date.  Recently, entry of the hepatitis C virus (HCV) was shown to occur along hepatic cholesterol uptake pathways and ezetimibe, a drug which blocks this lipid transport, was shown to inhibit HCV infection.  We here investigated the effect of ezetimibe on HBV infection using differentiated HepaRG cells as a cell-culture infection model.  Treatment with ezetimibe inhibited establishment of intrahepatic cccDNA and expression of viral replication markers when cells were infected with HBV virions, while we obsd. no effect when the HBV viral genome was transduced via an adenoviral vector.  Our data suggest that modulating hepatic cholesterol uptake by ezetimibe inhibits early HBV infection and that ezetimibe sensitive lipid transport pathways represent new targets for antiviral therapy in HBV infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNJ1wQJ_RUkbVg90H21EOLACvtfcHk0li1Qb1Gw8PTQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFSrsrw%253D&md5=dcfca7855b8cacdad584203e23690f85</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.antiviral.2012.12.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.antiviral.2012.12.008%26sid%3Dliteratum%253Aachs%26aulast%3DLucifora%26aufirst%3DJ.%26aulast%3DEsser%26aufirst%3DK.%26aulast%3DProtzer%26aufirst%3DU.%26atitle%3DEzetimibe%2520blocks%2520hepatitis%2520B%2520virus%2520infection%2520after%2520virus%2520uptake%2520into%2520hepatocytes%26jtitle%3DAntiviral%2520Res.%26date%3D2013%26volume%3D97%26spage%3D195%26epage%3D197%26doi%3D10.1016%2Fj.antiviral.2012.12.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blanchet, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sureau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labonte, P.</span></span> <span> </span><span class="NLM_article-title">Use of FDA approved therapeutics with hNTCP metabolic inhibitory properties to impair the HDV lifecycle</span>. <i>Antiviral Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">111</span>– <span class="NLM_lpage">115</span>, <span class="refDoi"> DOI: 10.1016/j.antiviral.2014.03.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1016%2Fj.antiviral.2014.03.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=24717262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnvFSrsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2014&pages=111-115&author=M.+Blanchetauthor=C.+Sureauauthor=P.+Labonte&title=Use+of+FDA+approved+therapeutics+with+hNTCP+metabolic+inhibitory+properties+to+impair+the+HDV+lifecycle&doi=10.1016%2Fj.antiviral.2014.03.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Use of FDA approved therapeutics with hNTCP metabolic inhibitory properties to impair the HDV lifecycle</span></div><div class="casAuthors">Blanchet, Matthieu; Sureau, Camille; Labonte, Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Antiviral Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">111-115</span>CODEN:
                <span class="NLM_cas:coden">ARSRDR</span>;
        ISSN:<span class="NLM_cas:issn">0166-3542</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Worldwide there are approx. 240 million individuals chronically infected with the hepatitis B virus (HBV), including 15-20 million coinfected with the hepatitis delta virus (HDV).  Treatments available today are not fully efficient and often assocd. to important side effects and development of drug resistance.  Targeting the HBV/HDV entry step using preS1-specific lipopeptides appears as a promising strategy to block viral entry for both HBV and HDV (Gripon et al., 2005; Petersen et al., 2008).  Recently, the human Sodium Taurocholate Cotransporting Polypeptide (hNTCP) has been identified as a functional, preS1-specific receptor for HBV and HDV.  This groundbreaking discovery has opened a very promising avenue for the treatment of chronic HBV and HDV infections.  Here we investigated the ability of FDA approved therapeutics with documented inhibitory effect on hNTCP cellular function to impair viral entry using a HDV in vitro infection model based on a hNTCP-expressing Huh7 cell line.  We demonstrate the potential of three FDA approved mols., irbesartan, ezetimibe, and ritonavir, to alter HDV infection in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5uuuK9vPHVbVg90H21EOLACvtfcHk0li1Qb1Gw8PTQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnvFSrsLc%253D&md5=abe5262a9fa3fc4daaff030ae1a080c1</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.antiviral.2014.03.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.antiviral.2014.03.017%26sid%3Dliteratum%253Aachs%26aulast%3DBlanchet%26aufirst%3DM.%26aulast%3DSureau%26aufirst%3DC.%26aulast%3DLabonte%26aufirst%3DP.%26atitle%3DUse%2520of%2520FDA%2520approved%2520therapeutics%2520with%2520hNTCP%2520metabolic%2520inhibitory%2520properties%2520to%2520impair%2520the%2520HDV%2520lifecycle%26jtitle%3DAntiviral%2520Res.%26date%3D2014%26volume%3D106%26spage%3D111%26epage%3D115%26doi%3D10.1016%2Fj.antiviral.2014.03.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blank, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markert, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hohmann, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carls, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikus, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehr, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alexandrov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haag, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwab, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urban, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haefeli, W. E.</span></span> <span> </span><span class="NLM_article-title">First-in-human application of the novel hepatitis B and hepatitis D virus entry inhibitor myrcludex B</span>. <i>J. Hepatol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">483</span>– <span class="NLM_lpage">489</span>, <span class="refDoi"> DOI: 10.1016/j.jhep.2016.04.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1016%2Fj.jhep.2016.04.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=27132172" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADC%252BC28Xptl2msrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2016&pages=483-489&author=A.+Blankauthor=C.+Markertauthor=N.+Hohmannauthor=A.+Carlsauthor=G.+Mikusauthor=T.+Lehrauthor=A.+Alexandrovauthor=M.+Haagauthor=M.+Schwabauthor=S.+Urbanauthor=W.+E.+Haefeli&title=First-in-human+application+of+the+novel+hepatitis+B+and+hepatitis+D+virus+entry+inhibitor+myrcludex+B&doi=10.1016%2Fj.jhep.2016.04.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">First-in-human application of the novel hepatitis B and hepatitis D virus entry inhibitor Myrcludex B</span></div><div class="casAuthors">Blank, Antje; Markert, Christoph; Hohmann, Nicolas; Carls, Alexandra; Mikus, Gerd; Lehr, Thorsten; Alexandrov, Alexander; Haag, Mathias; Schwab, Matthias; Urban, Stephan; Haefeli, Walter E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Hepatology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">483-489</span>CODEN:
                <span class="NLM_cas:coden">JOHEEC</span>;
        ISSN:<span class="NLM_cas:issn">0168-8278</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Myrcludex B is a first-in-class compd., which blocks entry of hepatitis B and D virus into hepatocytes in vitro and in animal models.  Based on the required preclin. data we aimed to translate this compd. into the first application in humans.Single ascending doses of myrcludex B, a 47 amino acid peptide, were administered up to 20 mg i.v. and 10 mg s.c. in a prospective open first-in-human, phase I clin. trial to 36 healthy volunteers.  Safety, tolerability and plasma concns. of myrcludex B were assessed and a pharmacokinetic model was derived.Myrcludex B was well tolerated and no serious or relevant AEs representing off-target effects, and no immunogenic effects were obsd. up to the highest applied dose of 20 mg (i.v.).  Myrcludex B showed dose-dependent pharmacokinetics, best described by a 2-compartment target-mediated drug disposition model.  Bioavailability of the s.c. application was large (85%).  Interindividual variability was moderate.  The pharmacokinetic model suggested that s.c. doses of 10 mg and above reach a target satn. of over 80% for at least 15 h.Myrcludex B showed excellent tolerability up to high doses.  Pharmacol. properties followed a 2-compartment target-mediated drug disposition model.  These findings are vital for planning of further multiple dose efficacy trials in patients.After showing antiviral activity in cell culture and animal models, myrcludex B, a new drug intended for the treatment of hepatitis B and D, has been administered the first time in humans.  Healthy volunteers received the drug i.v. and s.c. up to high doses (20 mg).  The drug was well tolerated and the characteristics of the drug detg. its way in the human body could be described.  These results will allow testing myrcludex B in hepatitis B and D patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJoBJ0yS6pu7Vg90H21EOLACvtfcHk0li1Qb1Gw8PTQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xptl2msrc%253D&md5=04a54ac56423c8692e19bd6baabff25b</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.jhep.2016.04.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jhep.2016.04.013%26sid%3Dliteratum%253Aachs%26aulast%3DBlank%26aufirst%3DA.%26aulast%3DMarkert%26aufirst%3DC.%26aulast%3DHohmann%26aufirst%3DN.%26aulast%3DCarls%26aufirst%3DA.%26aulast%3DMikus%26aufirst%3DG.%26aulast%3DLehr%26aufirst%3DT.%26aulast%3DAlexandrov%26aufirst%3DA.%26aulast%3DHaag%26aufirst%3DM.%26aulast%3DSchwab%26aufirst%3DM.%26aulast%3DUrban%26aufirst%3DS.%26aulast%3DHaefeli%26aufirst%3DW.%2BE.%26atitle%3DFirst-in-human%2520application%2520of%2520the%2520novel%2520hepatitis%2520B%2520and%2520hepatitis%2520D%2520virus%2520entry%2520inhibitor%2520myrcludex%2520B%26jtitle%3DJ.%2520Hepatol.%26date%3D2016%26volume%3D65%26spage%3D483%26epage%3D489%26doi%3D10.1016%2Fj.jhep.2016.04.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bogomolov, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alexandrov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voronkova, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macievich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kokina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petrachenkova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehr, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lempp, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wedemeyer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haag, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwab, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haefeli, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blank, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urban, S.</span></span> <span> </span><span class="NLM_article-title">Treatment of chronic hepatitis D with the entry inhibitor myrcludex B: First results of a phase Ib/IIa study</span>. <i>J. Hepatol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">490</span>– <span class="NLM_lpage">498</span>, <span class="refDoi"> DOI: 10.1016/j.jhep.2016.04.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1016%2Fj.jhep.2016.04.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=27132170" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVynsbrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2016&pages=490-498&author=P.+Bogomolovauthor=A.+Alexandrovauthor=N.+Voronkovaauthor=M.+Macievichauthor=K.+Kokinaauthor=M.+Petrachenkovaauthor=T.+Lehrauthor=F.+A.+Lemppauthor=H.+Wedemeyerauthor=M.+Haagauthor=M.+Schwabauthor=W.+E.+Haefeliauthor=A.+Blankauthor=S.+Urban&title=Treatment+of+chronic+hepatitis+D+with+the+entry+inhibitor+myrcludex+B%3A+First+results+of+a+phase+Ib%2FIIa+study&doi=10.1016%2Fj.jhep.2016.04.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Treatment of chronic hepatitis D with the entry inhibitor myrcludex B: First results of a phase Ib/IIa study</span></div><div class="casAuthors">Bogomolov, Pavel; Alexandrov, Alexander; Voronkova, Natalia; Macievich, Maria; Kokina, Ksenia; Petrachenkova, Maria; Lehr, Thorsten; Lempp, Florian A.; Wedemeyer, Heiner; Haag, Mathias; Schwab, Matthias; Haefeli, Walter E.; Blank, Antje; Urban, Stephan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Hepatology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">490-498</span>CODEN:
                <span class="NLM_cas:coden">JOHEEC</span>;
        ISSN:<span class="NLM_cas:issn">0168-8278</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The therapeutic option for patients with chronic hepatitis delta virus infection (CHD) is limited to interferon alpha with rare curative outcome.  Myrcludex B is a first-in-class entry inhibitor inactivating the hepatitis B virus (HBV) and hepatitis D virus (HDV) receptor sodium taurocholate co-transporting polypeptide.  We report the interim results of a pilot trial on chronically infected HDV patients treated with myrcludex B, or pegylated interferon alpha (PegIFNα-2a) or their combination.  Twenty-four patients with CHD infection were equally randomized (1:1:1) to receive myrcludex B, or PegIFNα-2a or their combination.  Patients were evaluated for virol. and biochem. response and tolerability of the study drugs at weeks 12 and 24.  Myrcludex B was well tolerated and no serious adverse event occurred.  Although hepatitis B surface antigen levels remained unchanged, HDV RNA significantly declined at week 24 in all cohorts.  HDV RNA became neg. in two patients each in the Myrcludex B and PegIFNα-2a cohorts, and in five patients of the Myrcludex B + PegIFNα-2a cohort.  ALT decreased significantly in the Myrcludex B cohort (six of eight patients), and HBV DNA was significantly reduced at week 24 in the Myrcludex B + PegIFNα-2a cohort.  Virus kinetic modeling suggested a strong synergistic effect of myrcludex B and PegIFNα-2a on both HDV and HBV.  Myrcludex B showed a strong effect on HDV RNA serum levels and induced ALT normalization under monotherapy.  Synergistic antiviral effects on HDV RNA and HBV DNA in the Myr-IFN cohort indicated a benefit of the combination of entry inhibition with PegIFNα-2a to treat CHD patients.  Myrcludex B is a new drug to treat hepatitis B and D infection.  After 24 wk of treatment with myrcludex B and/or pegylated interferon α-2a, HDV RNA, a relevant marker for hepatitis D infection, decreased in all patients with chronic hepatitis B and D.  Two of eight patients which received either myrcludex B or pegylated interferon α-2a, became neg. for HDV RNA, and five of seven patients who received both drugs at the same time became neg.  The drug was well tolerated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo29Ewz7ZuCMrVg90H21EOLACvtfcHk0lj2Do3bA2pEqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVynsbrE&md5=9b31bd4df1423aa2da1e49d41a89192a</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.jhep.2016.04.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jhep.2016.04.016%26sid%3Dliteratum%253Aachs%26aulast%3DBogomolov%26aufirst%3DP.%26aulast%3DAlexandrov%26aufirst%3DA.%26aulast%3DVoronkova%26aufirst%3DN.%26aulast%3DMacievich%26aufirst%3DM.%26aulast%3DKokina%26aufirst%3DK.%26aulast%3DPetrachenkova%26aufirst%3DM.%26aulast%3DLehr%26aufirst%3DT.%26aulast%3DLempp%26aufirst%3DF.%2BA.%26aulast%3DWedemeyer%26aufirst%3DH.%26aulast%3DHaag%26aufirst%3DM.%26aulast%3DSchwab%26aufirst%3DM.%26aulast%3DHaefeli%26aufirst%3DW.%2BE.%26aulast%3DBlank%26aufirst%3DA.%26aulast%3DUrban%26aufirst%3DS.%26atitle%3DTreatment%2520of%2520chronic%2520hepatitis%2520D%2520with%2520the%2520entry%2520inhibitor%2520myrcludex%2520B%253A%2520First%2520results%2520of%2520a%2520phase%2520Ib%252FIIa%2520study%26jtitle%3DJ.%2520Hepatol.%26date%3D2016%26volume%3D65%26spage%3D490%26epage%3D498%26doi%3D10.1016%2Fj.jhep.2016.04.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lipinski, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dominy, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feeney, P. J.</span></span> <span> </span><span class="NLM_article-title">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span>. <i>Adv. Drug Delivery Rev.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">26</span>, <span class="refDoi"> DOI: 10.1016/S0169-409X(00)00129-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1016%2FS0169-409X%2800%2900129-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=11259830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2001&pages=3-26&author=C.+A.+Lipinskiauthor=F.+Lombardoauthor=B.+W.+Dominyauthor=P.+J.+Feeney&title=Experimental+and+computational+approaches+to+estimate+solubility+and+permeability+in+drug+discovery+and+development+settings&doi=10.1016%2FS0169-409X%2800%2900129-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span></div><div class="casAuthors">Lipinski, C. A.; Lombardo, F.; Dominy, B. W.; Feeney, P. J.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">3-26</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ireland Ltd.</span>)
        </div><div class="casAbstract">A review with 50 refs.  Exptl. and computational approaches to est. soly. and permeability in discovery and development settings are described.  In the discovery setting 'the rule of 5' predicts that poor absorption or permeation is more likely when there are more than 5 H-bond donors, 10 H-bond acceptors, the mol. wt. (MWT) is greater than 500 and the calcd. Log P (CLogP) is greater than 5 (or MlogP >4.15).  Computational methodol. for the rule-based Moriguchi Log P (MLogP) calcn. is described.  Turbidimetric soly. measurement is described and applied to known drugs.  High throughput screening (HTS) leads tend to have higher MWT and Log P and lower turbidimetric soly. than leads in the pre-HTS era.  In the development setting, soly. calcns. focus on exact value prediction and are difficult because of polymorphism.  Recent work on linear free energy relationships and Log P approaches are critically reviewed.  Useful predictions are possible in closely related analog series when coupled with exptl. thermodn. soly. measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYDqa02g16urVg90H21EOLACvtfcHk0lj2Do3bA2pEqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D&md5=c60bb89da68f051c0ee7ac4c0468a0e4</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2FS0169-409X%2800%2900129-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-409X%252800%252900129-0%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DDominy%26aufirst%3DB.%2BW.%26aulast%3DFeeney%26aufirst%3DP.%2BJ.%26atitle%3DExperimental%2520and%2520computational%2520approaches%2520to%2520estimate%2520solubility%2520and%2520permeability%2520in%2520drug%2520discovery%2520and%2520development%2520settings%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2001%26volume%3D46%26spage%3D3%26epage%3D26%26doi%3D10.1016%2FS0169-409X%2800%2900129-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">DeGoey, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wendt, M. D.</span></span> <span> </span><span class="NLM_article-title">Beyond the Rule of 5: Lessons Learned from AbbVie’s Drugs and Compound Collection</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">2636</span>– <span class="NLM_lpage">2651</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00717</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00717" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFeitbjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=2636-2651&author=D.+A.+DeGoeyauthor=H.+J.+Chenauthor=P.+B.+Coxauthor=M.+D.+Wendt&title=Beyond+the+Rule+of+5%3A+Lessons+Learned+from+AbbVie%E2%80%99s+Drugs+and+Compound+Collection&doi=10.1021%2Facs.jmedchem.7b00717"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Beyond the Rule of 5: Lessons Learned from AbbVie's Drugs and Compound Collection</span></div><div class="casAuthors">DeGoey, David A.; Chen, Hui-Ju; Cox, Philip B.; Wendt, Michael D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2636-2651</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Recently, there has been an increasing focus on the pursuit of targets considered to be less druggable that offer potential for development of promising new therapeutic agents for the treatment of diseases with large unmet medical need, particularly in the areas of oncol. and virol.  However, conducting drug discovery campaigns in "beyond rule of 5" (bRo5) chem. space presents a significant drug design and development challenge to medicinal chemists to achieve acceptable oral pharmacokinetics.  Retrospective anal. of past successes and failures in drug discovery bRo5 may shed light on the key principles that contribute to the oral bioavailability of successful bRo5 compds. and improve the efficiency of drug design for future projects.  We present here highlights and case studies of lessons learned from discovery of bRo5 compds.  A simple multiparametric scoring function (AB-MPS) was devised that correlated preclin. PK results with cLogD, no. of rotatable bonds, and no. of arom. rings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTHuaXbtjlALVg90H21EOLACvtfcHk0lj2Do3bA2pEqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFeitbjJ&md5=48984dc65410d83d4ac8ddaca030b024</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00717&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00717%26sid%3Dliteratum%253Aachs%26aulast%3DDeGoey%26aufirst%3DD.%2BA.%26aulast%3DChen%26aufirst%3DH.%2BJ.%26aulast%3DCox%26aufirst%3DP.%2BB.%26aulast%3DWendt%26aufirst%3DM.%2BD.%26atitle%3DBeyond%2520the%2520Rule%2520of%25205%253A%2520Lessons%2520Learned%2520from%2520AbbVie%25E2%2580%2599s%2520Drugs%2520and%2520Compound%2520Collection%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D2636%26epage%3D2651%26doi%3D10.1021%2Facs.jmedchem.7b00717" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jing, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sui, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span> <span> </span><span class="NLM_article-title">Molecular determinants of hepatitis B and D virus entry restriction in mouse sodium taurocholate cotransporting polypeptide</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">7977</span>– <span class="NLM_lpage">7991</span>, <span class="refDoi"> DOI: 10.1128/JVI.03540-12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1128%2FJVI.03540-12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=23678176" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFWjt7zK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2013&pages=7977-7991&author=H.+Yanauthor=B.+Pengauthor=W.+Heauthor=G.+Zhongauthor=Y.+Qiauthor=B.+Renauthor=Z.+Gaoauthor=Z.+Jingauthor=M.+Songauthor=G.+Xuauthor=J.+Suiauthor=W.+Li&title=Molecular+determinants+of+hepatitis+B+and+D+virus+entry+restriction+in+mouse+sodium+taurocholate+cotransporting+polypeptide&doi=10.1128%2FJVI.03540-12"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular determinants of hepatitis B and D virus entry restriction in mouse sodium taurocholate cotransporting polypeptide</span></div><div class="casAuthors">Yan, Huan; Peng, Bo; He, Wenhui; Zhong, Guocai; Qi, Yonghe; Ren, Bijie; Gao, Zhenchao; Jing, Zhiyi; Song, Mei; Xu, Guangwei; Sui, Jianhua; Li, Wenhui</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">7977-7991</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Human hepatitis B virus (HBV) and its satellite virus, hepatitis D virus (HDV), primarily infect humans, chimpanzees, or tree shrews (Tupaia belangeri).  Viral infections in other species are known to be mainly restricted at the entry level since viral replication can be achieved in the cells by transfection of the viral genome.  Sodium taurocholate cotransporting polypeptide (NTCP) is a functional receptor for HBV and HDV, and amino acids 157 to 165 of NTCP are crit. for viral entry and likely limit viral infection of macaques.  However, the mol. determinants for viral entry restriction in mouse NTCP (mNTCP) remain unclear.  In this study, mNTCP was found to be unable to support either HBV or HDV infection, although it can bind to pre-S1 of HBV L protein and is functional in transporting substrate taurocholate; comprehensive swapping and point mutations of human NTCP (hNTCP) and mNTCP revealed mol. determinants restricting mNTCP for viral entry of HBV and HDV.  Remarkably, when mNTCP residues 84 to 87 were substituted by human counterparts, mNTCP can effectively support viral infections.  In addn., a no. of cell lines, regardless of their species or tissue origin, supported HDV infection when transfected with hNTCP or mNTCP with residues 84 to 87 replaced by human counterparts, highlighting the central role of NTCP for viral infections mediated by HBV envelope proteins.  These studies advance our understanding of NTCP-mediated viral entry of HBV and HDV and have important implications for developing the mouse model for their infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo181QjmyElmLVg90H21EOLACvtfcHk0lh-8kkJ4PpskQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFWjt7zK&md5=59f5669452aad2a667b6dfd1ba54d525</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1128%2FJVI.03540-12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.03540-12%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DH.%26aulast%3DPeng%26aufirst%3DB.%26aulast%3DHe%26aufirst%3DW.%26aulast%3DZhong%26aufirst%3DG.%26aulast%3DQi%26aufirst%3DY.%26aulast%3DRen%26aufirst%3DB.%26aulast%3DGao%26aufirst%3DZ.%26aulast%3DJing%26aufirst%3DZ.%26aulast%3DSong%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DG.%26aulast%3DSui%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DW.%26atitle%3DMolecular%2520determinants%2520of%2520hepatitis%2520B%2520and%2520D%2520virus%2520entry%2520restriction%2520in%2520mouse%2520sodium%2520taurocholate%2520cotransporting%2520polypeptide%26jtitle%3DJ.%2520Virol.%26date%3D2013%26volume%3D87%26spage%3D7977%26epage%3D7991%26doi%3D10.1128%2FJVI.03540-12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rudman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kendall, F. E.</span></span> <span> </span><span class="NLM_article-title">Bile acid content of human serum. II. The binding of cholanic acids by human plasma proteins</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">1957</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">538</span>– <span class="NLM_lpage">542</span>, <span class="refDoi"> DOI: 10.1172/JCI103451</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1172%2FJCI103451" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=13416382" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADyaG2sXlslOnuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=1957&pages=538-542&author=D.+Rudmanauthor=F.+E.+Kendall&title=Bile+acid+content+of+human+serum.+II.+The+binding+of+cholanic+acids+by+human+plasma+proteins&doi=10.1172%2FJCI103451"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Bile acid content of human serum. II. The binding of cholanic acids by human plasma proteins</span></div><div class="casAuthors">Rudman, Daniel; Kendall, Forrest E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Investigation</span>
        (<span class="NLM_cas:date">1957</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">538-42</span>CODEN:
                <span class="NLM_cas:coden">JCINAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-9738</span>.
    </div><div class="casAbstract">The binding of bile acids and derivs. by the protein fractions of human plasma was studied by the dialysis-equil. method.  Albumin exhibits the greatest binding activity towards these compds.  The lipoprotein-contg. globulins, Cohn fractions III and IV-1 bind approx. half as much deoxycholic acid and cholic acid as does albumin. γ-Globulin and fibrinogen do not interact with the bile acids.  The affinity for albumin is reduced by the introduction of polar groups into the steroid nucleus.  Thus the extent of binding decreases in the order of monohydroxy > dihydroxy > trihydroxy cholanic acid.  Binding of the ketocholanic acids to albumin could not be detected.  The effect of pH on the binding reaction suggests that the primary attraction between albumin and the cholanic acids is an electrostatic bond between the pos. charged lysine and side chains of the former and the neg. charged carboxy groups of the latter.  The binding consts. of albumin with 2 bile acids are calcd.  53 refs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCDr5ugBl7ebVg90H21EOLACvtfcHk0lh-8kkJ4PpskQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaG2sXlslOnuw%253D%253D&md5=4dc95127a15788b10799103fca3077db</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1172%2FJCI103451&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI103451%26sid%3Dliteratum%253Aachs%26aulast%3DRudman%26aufirst%3DD.%26aulast%3DKendall%26aufirst%3DF.%2BE.%26atitle%3DBile%2520acid%2520content%2520of%2520human%2520serum.%2520II.%2520The%2520binding%2520of%2520cholanic%2520acids%2520by%2520human%2520plasma%2520proteins%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D1957%26volume%3D36%26spage%3D538%26epage%3D542%26doi%3D10.1172%2FJCI103451" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mondal, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, S.</span></span> <span> </span><span class="NLM_article-title">Hydrophobicity is the governing factor in the interaction of human serum albumin with bile salts</span>. <i>Langmuir</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">1095</span>– <span class="NLM_lpage">1104</span>, <span class="refDoi"> DOI: 10.1021/la504270a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/la504270a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFyhtbzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2015&pages=1095-1104&author=N.+Ghoshauthor=R.+Mondalauthor=S.+Mukherjee&title=Hydrophobicity+is+the+governing+factor+in+the+interaction+of+human+serum+albumin+with+bile+salts&doi=10.1021%2Fla504270a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Hydrophobicity Is the Governing Factor in the Interaction of Human Serum Albumin with Bile Salts</span></div><div class="casAuthors">Ghosh, Narayani; Mondal, Ramakanta; Mukherjee, Saptarshi</div><div class="citationInfo"><span class="NLM_cas:title">Langmuir</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1095-1104</span>CODEN:
                <span class="NLM_cas:coden">LANGD5</span>;
        ISSN:<span class="NLM_cas:issn">0743-7463</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The present study demonstrates a detailed characterization of the interaction of a series of bile salts, sodium deoxycholate (NaDC), sodium cholate (NaC), and sodium taurocholate (NaTC), with a model transport protein, human serum albumin (HSA).  Here, steady-state and time-resolved fluorescence spectroscopic techniques have been used to characterize the interaction of the bile salts with HSA.  The binding isotherms constructed from steady-state fluorescence intensity measurements demonstrate that the interaction of the bile salts with HSA can be characterized by three distinct regions, which were also successfully reproduced from the significant variation of the emission wavelength (λem) of the intrinsic tryptophan (Trp) moiety of HSA.  The time-resolved fluorescence decay behavior of the Trp residue of HSA was also found to corroborate the steady-state results.  The effect of interaction with the bile salts on the native conformation of the protein has been explored in a CD study, which reveals a decrease in α-helicity of HSA induced by the bile salts.  In accordance with this, the esterase activity of the protein-bile salt aggregates is found to be reduced in comparison to that of the native protein.  Our results exclusively highlight the fact that it is the hydrophobic character of the bile salt that governs the extent of interaction with the protein.  Isothermal titrn. calorimetry (ITC) and mol. docking studies further substantiate our other exptl. findings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7AbgP5sfdFbVg90H21EOLACvtfcHk0lh-8kkJ4PpskQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFyhtbzP&md5=c2df6db5bbb26f72fdb9bfd66061f00b</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fla504270a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fla504270a%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DN.%26aulast%3DMondal%26aufirst%3DR.%26aulast%3DMukherjee%26aufirst%3DS.%26atitle%3DHydrophobicity%2520is%2520the%2520governing%2520factor%2520in%2520the%2520interaction%2520of%2520human%2520serum%2520albumin%2520with%2520bile%2520salts%26jtitle%3DLangmuir%26date%3D2015%26volume%3D31%26spage%3D1095%26epage%3D1104%26doi%3D10.1021%2Fla504270a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jing, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sui, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span> <span> </span><span class="NLM_article-title">Hepatitis D Virus Infection of Mice Expressing Human Sodium Taurocholate Co-transporting Polypeptide</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>, <span class="NLM_elocation-id">e1004840</span> <span class="refDoi"> DOI: 10.1371/journal.ppat.1004840</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=10.1371%2Fjournal.ppat.1004840" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;key=25902143" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&author=W.+Heauthor=B.+Renauthor=F.+Maoauthor=Z.+Jingauthor=Y.+Liauthor=Y.+Liuauthor=B.+Pengauthor=H.+Yanauthor=Y.+Qiauthor=Y.+Sunauthor=J.+T.+Guoauthor=J.+Suiauthor=F.+Wangauthor=W.+Li&title=Hepatitis+D+Virus+Infection+of+Mice+Expressing+Human+Sodium+Taurocholate+Co-transporting+Polypeptide&doi=10.1371%2Fjournal.ppat.1004840"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1004840&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1004840%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DW.%26aulast%3DRen%26aufirst%3DB.%26aulast%3DMao%26aufirst%3DF.%26aulast%3DJing%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DPeng%26aufirst%3DB.%26aulast%3DYan%26aufirst%3DH.%26aulast%3DQi%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DGuo%26aufirst%3DJ.%2BT.%26aulast%3DSui%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DW.%26atitle%3DHepatitis%2520D%2520Virus%2520Infection%2520of%2520Mice%2520Expressing%2520Human%2520Sodium%2520Taurocholate%2520Co-transporting%2520Polypeptide%26jtitle%3DPLoS%2520Pathog.%26date%3D2015%26volume%3D11%26doi%3D10.1371%2Fjournal.ppat.1004840" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i68"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00078">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_10502"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.1c00078?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00078</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">LC–MS result of commercial TUDCA and synthesized dimeric TUDCA, HBV infection inhibition by selected DBADs, cytotoxicity analysis of selected DBADs on HepG2-NTCP cells and PTHs, surface and total expression of NTCP after DBAD treatment, NTCP-dependent uptake of fluorescence bile acids NQL-064 and NQL-065 by HepG2-NTCP cells, fluorescence-labeled bile acid uptake inhibition assay, <sup>3</sup>H-TCA uptake inhibition by selected DBADs, persistent inhibition of FITC-pre-S1 peptide binding by DBADs, persistent inhibition of fluorescence bile acid NQL-064 uptake by DBADs, inhibitory effect of DBADs on FITC-pre-S1 peptide binding to NTCPs of different species, time course of DBAD inhibition of 3H-TCA uptake in HepG2-NTCP cells, and ALT and body weight surveillance during DBA-41 treatment (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00078/suppl_file/jm1c00078_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00078/suppl_file/jm1c00078_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00078/suppl_file/jm1c00078_si_001.pdf">jm1c00078_si_001.pdf (1.29 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00078/suppl_file/jm1c00078_si_002.csv">jm1c00078_si_002.csv (9.28 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.1c00078&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-9%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.1c00078%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.1c00078" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                7MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799f7195b092390","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
