
<html lang="en"     class="pb-page"  data-request-id="406a80fd-3c94-4bb4-942f-c211692191bd"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;article:article:10.1021/acs.jmedchem.0c00406;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2020.63.issue-10"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of a First-in-Class Protein Arginine Methyltransferase 6 (PRMT6) Covalent Inhibitor" /></meta><meta name="dc.Creator" content="Yudao  Shen" /></meta><meta name="dc.Creator" content="Fengling  Li" /></meta><meta name="dc.Creator" content="Magdalena M.  Szewczyk" /></meta><meta name="dc.Creator" content="Levon  Halabelian" /></meta><meta name="dc.Creator" content="Kwang-su  Park" /></meta><meta name="dc.Creator" content="Irene  Chau" /></meta><meta name="dc.Creator" content="Aiping  Dong" /></meta><meta name="dc.Creator" content="Hong  Zeng" /></meta><meta name="dc.Creator" content="He  Chen" /></meta><meta name="dc.Creator" content="Fanye  Meng" /></meta><meta name="dc.Creator" content="Dalia  Barsyte-Lovejoy" /></meta><meta name="dc.Creator" content="Cheryl H.  Arrowsmith" /></meta><meta name="dc.Creator" content="Peter J.  Brown" /></meta><meta name="dc.Creator" content="Jing  Liu" /></meta><meta name="dc.Creator" content="Masoud  Vedadi" /></meta><meta name="dc.Creator" content="Jian  Jin" /></meta><meta name="dc.Description" content="Protein arginine methyltransferase 6 (PRMT6) plays important roles in several biological processes associated with multiple cancers. Well-characterized potent, selective, and cell-active PRMT6 inhi..." /></meta><meta name="Description" content="Protein arginine methyltransferase 6 (PRMT6) plays important roles in several biological processes associated with multiple cancers. Well-characterized potent, selective, and cell-active PRMT6 inhi..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="May 5, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c00406" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00406" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c00406" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00406" /></link>
        
    
    

<title>Discovery of a First-in-Class Protein Arginine Methyltransferase 6 (PRMT6) Covalent Inhibitor | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c00406" /></meta><meta property="og:title" content="Discovery of a First-in-Class Protein Arginine Methyltransferase 6 (PRMT6) Covalent Inhibitor" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0013.jpeg" /></meta><meta property="og:description" content="Protein arginine methyltransferase 6 (PRMT6) plays important roles in several biological processes associated with multiple cancers. Well-characterized potent, selective, and cell-active PRMT6 inhibitors are invaluable tools for testing biological and therapeutic hypotheses. Although there are several known reversible PRMT6 inhibitors, covalent PRMT6 inhibitors have not been reported. Based on a cocrystal structure of PRMT6–MS023 (a type I PRMT inhibitor), we discovered the first potent and cell-active irreversible PRMT6 inhibitor, 4 (MS117). The covalent binding mode of compound 4 to PRMT6 was confirmed by mass spectrometry and kinetic studies and by a cocrystal structure. Compound 4 did not covalently modify other closely related PRMTs, potently inhibited PRMT6 in cells, and was selective for PRMT6 over other methyltransferases. We also developed two structurally similar control compounds, 5 (MS167) and 7 (MS168). We provide these valuable chemical tools to the scientific community for further studying PRMT6 physiological and pathophysiological functions." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c00406"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00406">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c00406&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c00406&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c00406&amp;href=/doi/10.1021/acs.jmedchem.0c00406" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 10</span><span class="cit-fg-pageRange">, 5477-5487</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/10" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c00337" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c00441" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of a First-in-Class Protein Arginine Methyltransferase 6 (PRMT6) Covalent Inhibitor</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Yudao Shen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yudao Shen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yudao++Shen">Yudao Shen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Fengling Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Fengling Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Fengling++Li">Fengling Li</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Magdalena M. Szewczyk</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Magdalena M. Szewczyk</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Magdalena+M.++Szewczyk">Magdalena M. Szewczyk</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Levon Halabelian</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Levon Halabelian</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Levon++Halabelian">Levon Halabelian</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kwang-su Park</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kwang-su Park</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kwang-su++Park">Kwang-su Park</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Irene Chau</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Irene Chau</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Irene++Chau">Irene Chau</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Aiping Dong</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Aiping Dong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Aiping++Dong">Aiping Dong</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hong Zeng</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hong Zeng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hong++Zeng">Hong Zeng</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">He Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">He Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=He++Chen">He Chen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Fanye Meng</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Fanye Meng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Fanye++Meng">Fanye Meng</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dalia Barsyte-Lovejoy</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dalia Barsyte-Lovejoy</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dalia++Barsyte-Lovejoy">Dalia Barsyte-Lovejoy</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Cheryl H. Arrowsmith</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Cheryl H. Arrowsmith</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div><div class="loa-info-affiliations-info">Princess Margaret Cancer Centre and Department of Medical Biophysics, University of Toronto, Toronto, Ontario M5G 2M9, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Cheryl+H.++Arrowsmith">Cheryl H. Arrowsmith</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4971-3250" title="Orcid link">http://orcid.org/0000-0002-4971-3250</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Peter J. Brown</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Peter J. Brown</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Peter+J.++Brown">Peter J. Brown</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jing Liu</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jing Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#214b484f460f4d4854614c52524c0f444554"><span class="__cf_email__" data-cfemail="2a4043444d0446435f6a47595947044f4e5f">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jing++Liu">Jing Liu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Masoud Vedadi</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Masoud Vedadi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div><div class="loa-info-affiliations-info">Department of Pharmacology and Toxicology, University of Toronto, Toronto, Ontario M5S 1A8, Canada</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#3954174f5c5d585d50794c4d564b56574d56175a58"><span class="__cf_email__" data-cfemail="177a3961727376737e576263786578796378397476">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Masoud++Vedadi">Masoud Vedadi</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Jian Jin</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jian Jin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#8de7e4ece3a3e7e4e3cde0fefee0a3e8e9f8"><span class="__cf_email__" data-cfemail="5d37343c33733734331d302e2e3073383928">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jian++Jin">Jian Jin</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2387-3862" title="Orcid link">http://orcid.org/0000-0002-2387-3862</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00406&amp;href=/doi/10.1021%2Facs.jmedchem.0c00406" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 10</span><span class="cit-pageRange">, 5477–5487</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">May 5, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>9 March 2020</li><li><span class="item_label"><b>Published</b> online</span>5 May 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 28 May 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c00406" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00406</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D5477%26pageCount%3D11%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DYudao%2BShen%252C%2BFengling%2BLi%252C%2BMagdalena%2BM.%2BSzewczyk%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D10%26contentID%3Dacs.jmedchem.0c00406%26title%3DDiscovery%2Bof%2Ba%2BFirst-in-Class%2BProtein%2BArginine%2BMethyltransferase%2B6%2B%2528PRMT6%2529%2BCovalent%2BInhibitor%26numPages%3D11%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D5487%26publicationDate%3DMay%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c00406"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2237</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">8</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c00406" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of a First-in-Class Protein Arginine Methyltransferase 6 (PRMT6) Covalent Inhibitor&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Yudao&quot;,&quot;last_name&quot;:&quot;Shen&quot;},{&quot;first_name&quot;:&quot;Fengling&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Magdalena&quot;,&quot;last_name&quot;:&quot;M. Szewczyk&quot;},{&quot;first_name&quot;:&quot;Levon&quot;,&quot;last_name&quot;:&quot;Halabelian&quot;},{&quot;first_name&quot;:&quot;Kwang-su&quot;,&quot;last_name&quot;:&quot;Park&quot;},{&quot;first_name&quot;:&quot;Irene&quot;,&quot;last_name&quot;:&quot;Chau&quot;},{&quot;first_name&quot;:&quot;Aiping&quot;,&quot;last_name&quot;:&quot;Dong&quot;},{&quot;first_name&quot;:&quot;Hong&quot;,&quot;last_name&quot;:&quot;Zeng&quot;},{&quot;first_name&quot;:&quot;He&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Fanye&quot;,&quot;last_name&quot;:&quot;Meng&quot;},{&quot;first_name&quot;:&quot;Dalia&quot;,&quot;last_name&quot;:&quot;Barsyte-Lovejoy&quot;},{&quot;first_name&quot;:&quot;Cheryl&quot;,&quot;last_name&quot;:&quot;H. Arrowsmith&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;J. Brown&quot;},{&quot;first_name&quot;:&quot;Jing&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Masoud&quot;,&quot;last_name&quot;:&quot;Vedadi&quot;},{&quot;first_name&quot;:&quot;Jian&quot;,&quot;last_name&quot;:&quot;Jin&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;05&quot;,&quot;day&quot;:&quot;05&quot;,&quot;issue&quot;:&quot;10&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;5477-5487&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c00406&quot;},&quot;abstract&quot;:&quot;Protein arginine methyltransferase 6 (PRMT6) plays important roles in several biological processes associated with multiple cancers. Well-characterized potent, selective, and cell-active PRMT6 inhibitors are invaluable tools for testing biological and therapeutic hypotheses. Although there are several known reversible PRMT6 inhibitors, covalent PRMT6 inhibitors have not been reported. Based on a cocrystal structure of PRMT6–MS023 (a type I PRMT inhibitor), we discovered the first potent and cell-active irreversible PRMT6 inhibitor, 4 (MS117). The covalent binding mode of compound 4 to PRMT6 was confirmed by mass spectrometry and kinetic studies and by a cocrystal structure. Compound 4 did not covalently modify other closely related PRMTs, potently inhibited PRMT6 in cells, and was selective for PRMT6 over other methyltransferases. We also developed two structurally similar control compounds, 5 (MS167) and 7 (MS168). We provide these valuable chemical tools to the scientific community for further studying PRMT&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00406&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00406" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00406&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00406" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00406&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00406" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00406&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00406&amp;href=/doi/10.1021/acs.jmedchem.0c00406" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c00406" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c00406" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00406&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c00406%26sid%3Dliteratum%253Aachs%26pmid%3D32367723%26genre%3Darticle%26aulast%3DShen%26date%3D2020%26atitle%3DDiscovery%2Bof%2Ba%2BFirst-in-Class%2BProtein%2BArginine%2BMethyltransferase%2B6%2B%2528PRMT6%2529%2BCovalent%2BInhibitor%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D10%26spage%3D5477%26epage%3D5487%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291445" title="Chemical structure">Chemical structure</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/10" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/jmcmar.2020.63.issue-10/20200528/jmcmar.2020.63.issue-10.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/medium/jm0c00406_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00406&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Protein arginine methyltransferase 6 (PRMT6) plays important roles in several biological processes associated with multiple cancers. Well-characterized potent, selective, and cell-active PRMT6 inhibitors are invaluable tools for testing biological and therapeutic hypotheses. Although there are several known reversible PRMT6 inhibitors, covalent PRMT6 inhibitors have not been reported. Based on a cocrystal structure of PRMT6–MS023 (a type I PRMT inhibitor), we discovered the first potent and cell-active irreversible PRMT6 inhibitor, <b>4</b> (MS117). The covalent binding mode of compound <b>4</b> to PRMT6 was confirmed by mass spectrometry and kinetic studies and by a cocrystal structure. Compound <b>4</b> did not covalently modify other closely related PRMTs, potently inhibited PRMT6 in cells, and was selective for PRMT6 over other methyltransferases. We also developed two structurally similar control compounds, <b>5</b> (MS167) and <b>7</b> (MS168). We provide these valuable chemical tools to the scientific community for further studying PRMT6 physiological and pathophysiological functions.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48602" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48602" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Protein arginine methyltransferase 6 (PRMT6) solely resides in the nucleus, and its polypeptide chain consists of a catalytic core sequence that is common to other protein arginine methyltransferases (PRMTs).<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> As a member of type I PRMTs, PRMT6 catalyzes the transfer of the methyl group from the cofactor <i>S</i>-5′-adenosyl-<span class="smallcaps smallerCapital">l</span>-methionine (SAM) to arginine residues of its substrates to provide post-translationally modified products with monomethylated or asymmetric dimethylated arginine residues.<a onclick="showRef(event, 'ref2 ref3 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref2 ref3 ref4 ref5">(2−5)</a> Besides the well-characterized histone substrate H3R2 (histone H3 arginine 2),<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> nonhistone proteins, such as HIV Tat,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> HMGA1a,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> and DNA polymerase β,<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> are also PRMT6 substrates. PRMT6 overexpression and hypermethylation of its substrates are associated with multiple cancers, such as prostate,<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> bladder,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> melanoma,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> and colorectal<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> cancers. Potent, selective, and cell-active reversible or irreversible inhibitors of PRMT6 could be useful chemical tools to investigate cellular processes and/or serve as lead compounds in drug discovery. Several small-molecule noncovalent inhibitors of PRMT6 have been developed,<a onclick="showRef(event, 'ref14 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref14 ref15 ref16">(14−16)</a> but covalent, irreversible inhibitors of PRMT6 have not been reported to date. Covalent, irreversible inhibitors have been rapidly gaining momentum in the clinic and more frequently becoming FDA-approved drugs.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> They form inhibitor–target adducts with no dissociation or virtually unlimited residence time. This continuous target engagement could provide extended pharmacodynamic duration without the need to maintain high exposures of the inhibitors.</div><div class="NLM_p last">Based on a cocrystal structure of PRMT6 in complex with MS023, a type I PRMT inhibitor we previously developed,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> we discovered the first potent, selective, and cell-active covalent inhibitor of PRMT6, MS117 (compound <b>4</b>). We also designed and synthesized two structurally highly similar control compounds, MS167 (compound <b>5</b>), a potent and cell-active reversible PRMT6 inhibitor, and MS168 (compound <b>7</b>), a very poor PRMT6 inhibitor but with the same reactive warhead as compound <b>4</b>. Using mass spectrometry (MS) and kinetic studies, we have demonstrated that compound <b>4</b> is a covalent PRMT6 inhibitor, which was further confirmed by the cocrystal structure of PRMT6 conjugated with compound <b>4</b>. We also found that compound <b>4</b> did not covalently modify other closely related PRMTs and was selective for PRMT6 over other methyltransferases. In addition, compound <b>4</b> effectively inhibited PRMT6 in cells and was more potent than its close analogue, compound <b>5</b>, which is a noncovalent inhibitor of PRMT6. Here, we describe structure-based design, chemical synthesis, and biological characterization of these valuable chemical tools in a battery of biochemical, biophysical, and cellular assays.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_68074" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_68074" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Structure-Based Inhibitor Design</h3><div class="NLM_p">We aligned protein sequences of PRMTs and identified that Cys50 is a unique cysteine residue located in the PRMT6 substrate-binding site and other PRMTs lack such a cysteine residue in their substrate-binding sites (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00406/suppl_file/jm0c00406_si_001.pdf" class="ext-link">Figure S1</a>). Through analysis of the crystal structure of PRMT6 in complex with the type I PRMT inhibitor MS023 we previously developed,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> we noted that the distance between the thiol group of Cys50 and the meta-position of the MS023 phenyl ring is 4.9 Å, which is approximately equal to the length of four to five chemical bonds (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A). This distance is suitable for introducing an acrylamide group as a Michael addition acceptor to react with the thiol group of Cys50 and form a covalent bond.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Thus, we designed compound <b>4</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B), which contains an acrylamide moiety at the meta-position of the MS023 phenyl ring. In addition, we designed two structurally highly similar control compounds: <b>5</b> and <b>7</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B). Compound <b>5</b>, which contains the propionamide group instead of the acrylamide moiety, was designed to maintain similar binding affinity to PRMT6 as compound <b>4</b>, but be incapable of covalently modifying Cys50. We previously reported that the terminal primary amino group of MS023 formed critical hydrogen bonds (H-bonds) with PRMT6 and the replacement of the primary amino group with a primary alcohol group led to significant decreased potency.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Thus, compound <b>7</b>, which contains a primary alcohol group, was designed as another control compound to possess the same reactive moiety (warhead) but have much weaker binding affinity to PRMT6.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/medium/jm0c00406_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0001.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Design of the PRMT6 covalent inhibitor, compound <b>4</b>, and its two control compounds. (A) Crystal structure (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5E8R">5E8R</a>) of PRMT6 (green) in complex with MS023 (cyan) and SAH (gray) reveals that the distance between the Cys50 thiol group and the meta-position of the MS023 phenyl ring (shown by the orange dotted line) is 4.9 Å. (B) Chemical structures of MS023, compound <b>4</b> (PRMT6 covalent inhibitor), and two structurally highly similar controls, compounds <b>5</b> and <b>7</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00406&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Synthesis</h3><div class="NLM_p">The synthetic routes for preparing compounds <b>4</b>, <b>5</b>, and <b>7</b> are depicted in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. 1,3-Dipolar cycloaddition between commercially available ethyl (<i>E</i>)-3-(3-nitrophenyl)acrylate and 1-((isocyanomethyl)sulfonyl)-4-methylbenzene under basic condition, followed by protecting the resulting pyrrole nitrogen with the Boc group, provided compound <b>1</b>. DIBAL-H-mediated reduction converted the ethyl ester group of compound <b>1</b> to an alcohol group, which was subsequently oxidized to yield aldehyde <b>2</b> by Dess–Martin periodinane. Reduction amination of the resulting aldehyde intermediate <b>2</b> with <i>tert</i>-butyl (2-(methylamino)ethyl)carbamate or 2-(methylamino)ethan-1-ol afforded intermediates <b>3</b> and <b>6</b>, respectively. Under hydrogenation reaction conditions, the nitro group of compound <b>3</b> was reduced to an amino group. The subsequent amide coupling reactions between the resulting amine and acryloyl chloride or propionyl chloride, followed by the removal of the Boc-protecting group, provided compounds <b>4</b> and <b>5</b>, respectively. Using a similar reaction sequence, e.g., reductive amination, nitro group reduction, amide coupling, and Boc group deprotection, intermediate <b>6</b> was converted to compound <b>7</b>.</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/medium/jm0c00406_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0012.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthetic Route of Compounds <b>4</b>, <b>5</b>, and <b>7</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00406&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaH, dimethylformamide/tetrahydrofuran (DMF/THF); (b) Et<sub>3</sub>N, Boc<sub>2</sub>O, 4-dimethylaminopyridine (DMAP), dichloromethane (DCM), 37% over two steps. (c) DIBAL-H, THF, −78 °C; (d) Dess–Martin periodinane, DCM, 71%; (e) 2-(methylamino)ethan-1-ol, NaBH(OAc)<sub>3</sub>, DCM, 74%; (f) <i>tert</i>-butyl (2-(methylamino)ethyl)carbamate, NaBH(OAc)<sub>3</sub>, DCM, 88%; (g) Pd/C (10%), H<sub>2</sub>, MeOH; (h) acryloyl chloride, DCM; (i) trifluoroacetic acid (TFA)/DCM, 29–42%; (j) propionyl chloride, Et<sub>3</sub>N, DCM.</p></p></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Evaluation in Biochemical and Kinetic Studies</h3><div class="NLM_p">We first assessed the PRMT6 inhibitory effect of these three compounds using a scintillation proximity assay (SPA), which measures the activity of human PRMT6 in catalyzing the transfer of the tritiated methyl group from <sup>3</sup>H-SAM to a peptide substrate in the presence and absence of compounds.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> With 1 h preincubation, compound <b>4</b> displayed excellent inhibition potency against PRMT6 (IC<sub>50</sub> = 18 ± 2 nM, pIC<sub>50</sub> = 7.8 ± 0.01) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). The propionate amide control compound <b>5</b> exhibited similar potency to compound <b>4</b> (IC<sub>50</sub> = 28 ± 1 nM, pIC<sub>50</sub> = 7.6 ± 0.1) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). As expected, the alcohol-containing control compound <b>7</b> showed poor potency against PRMT6 (IC<sub>50</sub> = 9800 ± 980 nM, pIC<sub>50</sub> = 5.0 ± 0.04) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C).</div><figure id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/medium/jm0c00406_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Compounds <b>4</b> and <b>5</b>, but not compound <b>7</b>, potently inhibited PRMT6. Concentration–response curves of compounds <b>4</b> (A), <b>5</b> (B), and <b>7</b> (C) in inhibiting PRMT6 in a scintillation proximity assay. IC<sub>50</sub> determination experiments were performed in triplicate, and the values are presented as mean ± standard deviation (SD).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00406&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">While IC<sub>50</sub> values are appropriate for quantifying the potency of reversible inhibitors, the IC<sub>50</sub> values of irreversible inhibitors are time-dependent. Thus, irreversible inhibitors are commonly quantified using the <i>k</i><sub>inact</sub>/<i>K</i><sub>I</sub> rate constants, derived from the inhibition constant (<i>K</i><sub>I</sub>) and the rate of enzyme inactivation (<i>k</i><sub>inact</sub>).<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> To further characterize the PRMT6 kinetic inhibition nature of compounds <b>4</b> and <b>5</b>, a series of IC<sub>50</sub> values were measured using the SPA assay by preincubating the compounds with PRMT6 at different time points ranging from 1 to 30 min. Compound <b>4</b> displayed time-dependent potencies against PRMT6 with a <i>k</i><sub>inact</sub>/<i>K</i><sub>I</sub> value of 8.3 × 10<sup>4</sup> M<sup>–1</sup> s<sup>–1</sup>, indicating the irreversible inhibition mechanism (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A).<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> On the other hand, no significant change in IC<sub>50</sub> values was observed for compound <b>5</b> at various time points ranging from 1 to 30 min, indicating that compound <b>5</b> is a reversible inhibitor (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B).</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/medium/jm0c00406_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Kinetics of PRMT6 inhibition by compounds <b>4</b> and <b>5</b>. (A) Compound <b>4</b> inhibited the PRMT6 methyltransferase activity in a time-dependent manner. (B) Compound <b>5</b> inhibited PRMT6 in a time-independent manner. The indicated compound was preincubated with PRMT6 at different time points ranging from 1 to 30 min, and IC<sub>50</sub> values were determined using the SPA assay. The experiments were performed in triplicate. The IC<sub>50</sub> values are presented as mean ± SD.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00406&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Characterization in a Mass Spectrometry-Based Assay</h3><div class="NLM_p">To confirm that compound <b>4</b> covalently modifies PRMT6, an excess amount of compound <b>4</b> was preincubated with PRMT6 for 1 h and potential protein–ligand adducts were analyzed by mass spectrometry (MS). Our MS analyses revealed an adduct with mass equal to PRMT6 plus compound <b>4</b>, with no detectable unmodified PRMT6 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A,B), indicating that compound <b>4</b> efficiently forms a single modified covalent adduct. Under the same assay conditions, neither compound <b>5</b> nor compound <b>7</b> could modify PRMT6 in a detected level (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C,D). In addition, compound <b>4</b> was screened for modification of other type I PRMTs (PRMT1, 3, 4, and 8) and no adducts with other type I PRMTs could be detected (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00406/suppl_file/jm0c00406_si_001.pdf" class="ext-link">Figures S2–S5</a>). This is in agreement with the PRMT protein sequence alignment results that PRMT6, but not other PRMTs, contains a cysteine residue (Cys50) in its substrate-binding site, which can be selectively targeted to form a covalent bond (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00406/suppl_file/jm0c00406_si_001.pdf" class="ext-link">Figure S1</a>).</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/medium/jm0c00406_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Compound <b>4</b>, but not compounds <b>5</b> and <b>7</b>, covalently modified PRMT6 in an MS assay. Mass spectra of PRMT6 alone (A), PRMT6 preincubated with compound <b>4</b> (MW: 298) (B), PRMT6 preincubated with compound <b>5</b> (C), and PRMT6 preincubated with compound <b>7</b> (D).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00406&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Cocrystal Structure of the PRMT6–Compound <b>4</b> Complex</h3><div class="NLM_p">To confirm that Cys50 was covalently modified, we co-crystallized PRMT6 with compound <b>4</b> in the presence of the cofactor product <i>S</i>-5′-adenosyl-<span class="smallcaps smallerCapital">l</span>-homocysteine (SAH) and solved a high-resolution (2 Å) cocrystal structure (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6P7I">6P7I</a>, <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A). The asymmetric cocrystal structure contains four conformational similar copies of PRMT6, with both compound <b>4</b> and SAH in all of the four active sites (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A). Electron density map at the active sites indicates that the acrylamide group is close to the Cys50 residue. Although there is not enough electron density to confirm the covalent C–S bond in chains a, b, and c, the C–S bond formation is unambiguously confirmed by the electron density in chain d (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A). The alignment of MS023–PRMT6 and compound <b>4</b>-PRMT6 cocrystal structures reveals that compound <b>4</b> occupies the substrate pocket of PRMT6 with a similar binding mode to MS023 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B).<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Compared to MS023, the phenyl ring of compound <b>4</b> slightly flipped about 0.9 Å away from Cys50 to favor the Michael addition between acrylamide and Cys50 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B). As the isopropoxy group of MS023 was removed in compound <b>4</b>, the H-bond between the isopropoxy group and residue His163 in the MS023–PRMT6 complex cannot be observed anymore in the PRMT6–compound <b>4</b> adduct. Otherwise, all other H-bonds can be found in the PRMT6–compound <b>4</b> adduct, not only including direct H-bonds between the primary amino group and Glu155 or Met157, between the ternary amino group and His317, and between the pyrrole nitrogen and Glu59, but also including the water-mediated H-bonds between the primary amino group and Trp156 and Glu164 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A).</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/medium/jm0c00406_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) X-ray cocrystal structure of PRMT6 (green, chain d) conjugated with compound <b>4</b> (cyan) in complex with SAH (gray) (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6P7I">6P7I</a>). Compound <b>4</b> covalently bound to the PRMT6 Cys50 residue. Key H-bond interactions between compound <b>4</b> and PRMT6 are shown in orange dotted lines. (B) Superimposition of the PRMT6–MS023 (magenta) (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5E8R">5E8R</a>) and PRMT6–compound <b>4</b> cocrystal structures.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00406&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Selectivity Assessment</h3><div class="NLM_p">To assess the selectivity of compound <b>4</b>, we tested it against 22 protein lysine methyltransferases (PKMTs), 3 DNA methyltransferases (DNMTs), and 8 PRMTs at 10 and 50 μM (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A). At 10 μM, compound <b>4</b> did not significantly inhibit any PKMTs, DNMTs, and type II, III PRMTs tested. However, compound <b>4</b> significantly inhibited type I PRMTs with >90% inhibition. At 50 μM, in addition to type I PRMTs, compound <b>4</b> inhibited SUV39H1, SETD7, PRMT7, and DNMT3A/3L (>50% inhibition). We also tested control compounds <b>5</b> and <b>7</b> against these 33 methyltransferases (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B,C). Similar to compound <b>4</b>, compound <b>5</b> was very selective for type I PRMTs over PKMTs, DNMTs, and type II, III PRMTs but inhibited all type I PRMTs (>40% inhibition at 10 μM; >80% inhibition at 50 μM) (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B). On the other hand, compound <b>7</b> showed modest inhibition activity only for PRMT1 (∼60% inhibition at 10 μM; ∼70% inhibition at 50 μM) and PRMT6 (∼50% inhibition at 10 μM; ∼80% inhibition at 50 μM) (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C).</div><figure id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/medium/jm0c00406_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Selectivity assessment of compounds <b>4</b> (A), <b>5</b> (B), and <b>7</b> (C) against a variety of methyltransferases at 10 μM (blue) and 50 μM (gray).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00406&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To further assess the selectivity profiles of compounds <b>4</b>, <b>5</b>, and <b>7</b> within the type I PRMT subfamily, we determined potencies of these compounds against PRMT1, PRMT3, PRMT4, and PRMT8 using the SPA assay (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Compound <b>4</b> showed good selectivity for PRMT6 (IC<sub>50</sub> = 0.018 ± 0.002 μM) over PRMT3 (IC<sub>50</sub> = 3 ± 0.2 μM) and PRMT4 (IC<sub>50</sub> = 0.48 ± 0.1 μM), with 167- and 27-fold selectivities, respectively. In addition, compound <b>4</b> showed moderate selectivity (6-fold) for PRMT6 over PRMT1 (IC<sub>50</sub> = 0.1 ± 0.025 μM) and PRMT8 (IC<sub>50</sub> = 0.11 ± 0.01 μM). Similar to compound <b>4</b>, compound <b>5</b> showed good selectivity for PRMT6 (IC<sub>50</sub> = 0.028 ± 0.001 μM) over PRMT3 (IC<sub>50</sub> = 15 ± 5 μM) and PRMT4 (IC<sub>50</sub> = 2 ± 0.2 μM) and moderate selectivity over PRMT1 (IC<sub>50</sub> = 0.17 ± 0.03 μM) and PRMT8 (IC<sub>50</sub> = 0.13 ± 0.01 μM). As expected, compared to compounds <b>4</b> and <b>5</b>, compound <b>7</b> was much weaker at inhibiting the methyltransferase activity of all type I PRMTs.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Potencies of Compounds <b>4</b>, <b>5</b>, and <b>7</b> at Inhibiting Type I PRMTs<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="5" align="center" char="±">IC<sub>50</sub> (μM)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center" char="±">PRMT1</th><th class="colsep0 rowsep0" align="center">PRMT3</th><th class="colsep0 rowsep0" align="center" char="±">PRMT4</th><th class="colsep0 rowsep0" align="center" char="±">PRMT6</th><th class="colsep0 rowsep0" align="center" char="±">PRMT8</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="char" char="±">0.1 ± 0.025</td><td class="colsep0 rowsep0" align="left">3 ± 0.2</td><td class="colsep0 rowsep0" align="char" char="±">0.48 ± 0.1</td><td class="colsep0 rowsep0" align="char" char="±">0.018 ± 0.002</td><td class="colsep0 rowsep0" align="char" char="±">0.11 ± 0.01</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="char" char="±">0.17 ± 0.03</td><td class="colsep0 rowsep0" align="left">15 ± 5</td><td class="colsep0 rowsep0" align="char" char="±">2 ± 0.2</td><td class="colsep0 rowsep0" align="char" char="±">0.028 ± 0.001</td><td class="colsep0 rowsep0" align="char" char="±">0.13 ± 0.01</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="char" char="±">15 ± 3</td><td class="colsep0 rowsep0" align="left">>200</td><td class="colsep0 rowsep0" align="char" char="±">51 ± 1.6</td><td class="colsep0 rowsep0" align="char" char="±">9.8 ± 0.098</td><td class="colsep0 rowsep0" align="char" char="±">23 ± 1.3</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> determination experiments were performed at substrate and cofactor concentrations equal to the respective <i>K</i><sub>m</sub> values for each enzyme. IC<sub>50</sub> determination experiments were performed in triplicate, and the values are presented as mean ± SD.</p></div></div></div><div class="NLM_p last">Taken together, these results indicate that while compound <b>7</b> is a very weak inhibitor against methyltransferases in general, compounds <b>4</b> and <b>5</b> are highly selective for PRMT6 over PKMTs, DNMTs, and type II and III PRMTs. Within the type I PRMT subfamily, compounds <b>4</b> and <b>5</b> displayed good selectivity for PRMT6 over PRMT3 and PRMT4 and were moderately selective for PRMT6 over PRMT1 and PRMT8.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Evaluation in Cellular Assays</h3><div class="NLM_p">Encouraged by the good potency and selectivity data of compound <b>4</b> in biochemical assays, we next evaluated the effects of pharmacological inhibition of PRMT6 and other type I PRMT off-targets, including PRMT1, PRMT3, and PRMT4, by compound <b>4</b> and its two control compounds <b>5</b> and <b>7</b> in cells (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>). Because PRMT8 is mainly expressed in the brain,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> we did not investigate the cellular effect of compound <b>4</b> against this target. First, the cellular potency of compound <b>4</b> against ectopically expressed PRMT6 in HEK293 cells was evaluated (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A–C). Compound <b>4</b> potently and concentration-dependently reduced cellular levels of H3R2 asymmetric dimethylation (H3R2me2a, IC<sub>50</sub> = 1.3 ± 0.2 μM), the known PRMT6 asymmetric dimethylation product (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A). Similarly, both the PRMT6 catalytically inactive V86K/D88A mutation and the type I PRMT inhibitor MS023<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> at 2 μM effectively inhibited H3R2me2a. As expected, compound <b>5</b> also inhibited H3R2me2a, but was less effective than compound <b>4</b>, and compound <b>7</b> did not significantly inhibit H3R2me2a at concentrations up to 10 μM (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A,B).</div><figure id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/medium/jm0c00406_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Compound <b>4</b> inhibits PRMT6-dependent H3R2 asymmetric dimethylation in cells with the highest potency. HEK293T cells were transfected with Flag-tagged wild-type PRMT6 or PRMT6-V86K/D88A catalytically inactive mutant (*) and treated with indicated compounds for 20 h. Type I PRMT inhibitor MS023 was used as a positive control. (A) Western blot representation of the effect of compounds <b>4</b>, <b>5</b>, and <b>7</b> on PRMT6 activity. (B) Quantification of the effects of compounds <b>4</b>, <b>5</b>, and <b>7</b> on PRMT6 activity. The graph shows H3R2me2a fluorescence intensities normalized to intensities of H3. The dotted line indicates the methylation level after MS023 treatment, which represents the background signal. The results are presented as the mean ± SD of three replicates. (C) IC<sub>50</sub> determination of compound <b>4</b>. The graph represents the nonlinear fit of H3R2me2a fluorescence intensities normalized to intensities of H3. The individual data points are indicated.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00406&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Next, we used MCF7 cells to assess the cellular activity of compound <b>4</b> against endogenous PRMT1-catalyzed asymmetric dimethylation of H4R3 (H4R3me2a) using MS023 (1 μM) as a positive control (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A,B). Compound <b>4</b> treatment resulted in a concentration-dependent inhibition of H4R3me2a levels with an IC<sub>50</sub> value of 5.6 ± 0.3 μM, indicating that compound <b>4</b> is less potent at inhibiting PRMT1 than PRMT6 in cells. Because compound <b>4</b> is not a covalent inhibitor of PRMT1, we expected that compounds <b>4</b> and <b>5</b> would have similar potencies in this cellular assay. Indeed, compound <b>5</b> displayed similar cellular potency (IC<sub>50</sub> = 6.7 ± 0.6 μM) to compound <b>4</b> (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A,B). In addition, as predicted, compound <b>7</b> was inactive in this cellular assay at concentrations up to 20 μM.</div><figure id="fig8" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/medium/jm0c00406_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Inhibition of PRMT1-dependent H4R3 asymmetric dimethylation in cells. (A) Compounds <b>4</b> and <b>5</b>, but not compound <b>7</b>, inhibited H4R3me2a. MCF7 cells were treated with compounds for 2 days. MS023 was used as a positive control. (B) Graph representing the nonlinear fit of H4R3me2a fluorescence intensities normalized to intensities of H4. The results are presented as the mean ± SD of three replicates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00406&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We next investigated the cellular effects of compounds <b>4</b>, <b>5</b>, and <b>7</b> on the overexpressed PRMT3-catalyzed methylation of exogenously introduced H4 (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>).<a onclick="showRef(event, 'ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref22 ref23">(22,23)</a> While the catalytically inactive PRMT3 mutant (E335Q) and the PRMT3 selective inhibitor SGC707<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> (1 μM) effectively inhibited H4R3me2a, compounds <b>4</b>, <b>5</b>, and <b>7</b> did not significantly reduce this mark at concentrations up to 20 μM.</div><figure id="fig9" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/medium/jm0c00406_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0011.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Inhibition of PRMT3-dependent H4R3 arginine asymmetric dimethylation in cells. Compounds <b>4</b>, <b>5</b>, and <b>7</b> did not inhibit the PRMT3 activity at concentrations up to 20 μM. HEK293T cells were co-transfected with GFP-tagged histone H4 peptide (1–50) and PRMT3 WT or catalytically inactive mutant (E338Q) and treated with compounds for 20 h. PRMT3 selective inhibitor SGC707 was used as a positive control. The WB results are representative of two independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00406&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We also assessed the effects of compounds <b>4</b>, <b>5</b>, and <b>7</b> on inhibiting the endogenously expressed PRMT4 in MCF7 cells (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>). In contrast to the PRMT4 selective inhibitor TP064 (2 μM),<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> compounds <b>4</b>, <b>5</b>, and <b>7</b> did not reduce BAF155 arginine asymmetric dimethylation (BAF155-Rme2a) levels at concentrations up to 20 μM.</div><figure id="fig10" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/medium/jm0c00406_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0002.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Inhibition of PRMT4-dependent BAF155 asymmetric dimethylation in cells. Compounds <b>4</b>, <b>5</b>, and <b>7</b> did not inhibit PRMT4 activity at concentrations up to 20 μM. MCF7 cells were treated with compounds for 2 days. PRMT4 selective inhibitor TP064 was used as a positive control. The WB results are representative of two independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00406&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Finally, to assess the cellular toxicity of compounds <b>4</b>, <b>5</b>, and <b>7</b>, we investigated the effect of these compounds on inhibiting the growth of MCF-7, PNT2 (a nontumorigenic prostate cell line), and HEK293T cells. Each of the three cell lines was treated with compound <b>4</b>, <b>5</b>, or <b>7</b> at concentrations up to 20 μM for 3 days, and we found that none of these compounds was cytotoxic in MCF-7, PNT2, and HEK293T cells (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>). Thus, compounds <b>4</b>, <b>5</b>, and <b>7</b> are valuable chemical tools without significant cell toxicity.</div><figure id="fig11" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/medium/jm0c00406_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0003.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Compounds <b>4</b>, <b>5</b>, and <b>7</b> are not cytotoxic in MCF-7, PNT2, and HEK293T cells. MCF-7, PNT2, and HEK293T cells were treated with compound <b>4</b> (A), <b>5</b> (B), or <b>7</b> (C) at indicated concentrations for 3 days. Results are presented as mean ± SD from three replicate experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00406&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Taken together, these results indicate that compound <b>4</b> can potently inhibit PRMT6 and is selective for PRMT6 over PRMT3 (>20-fold), PRMT4 (>20-fold), and PRMT1 (∼4-fold) in cells without significant cell toxicity, and compounds <b>5</b> and <b>7</b> are excellent control compounds for compound <b>4</b>.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_57405" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_57405" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">We discovered a first-in-class PRMT6 covalent inhibitor, compound <b>4</b>. The covalent inhibition nature of compound <b>4</b> was confirmed by kinetic and MS studies. We also obtained a cocrystal structure of PRMT6 conjugated with compound <b>4</b>, which confirmed that Cys50, a unique cysteine residue in the PRMT6 active site, was modified by compound <b>4</b>. Compound <b>4</b> is a potent PRMT6 inhibitor and remarkably selective for PRMT6 over 28 PKMTs, DNMTs, and type II and III PRMTs. Within the type I PRMT subfamily, compound <b>4</b> displayed excellent selectivity over PRMT3 and PRMT4 and moderate selectivity over PRMT1 and PRMT8 and did not covalently modify these closely related type I PRMTs. In cellular assays, compound <b>4</b> potently inhibited PRMT6, was less potent against PRMT1 and inactive against PRMT3 and PRMT4, and was not cytotoxic in general. In addition, we developed two structurally highly similar control compounds: compound <b>5</b>, a potent and cell-active reversible PRMT6 inhibitor, and compound <b>7</b>, a very poor inhibitor but with the same reactive acrylamide warhead. Collectively, compounds <b>4</b>, <b>5</b>, and <b>7</b> are a set of valuable tool compounds for the biomedical community to investigate PRMT6 biological functions and disease associations. Furthermore, our discovery has paved the way for the research community to develop the next generation of improved PRMT6 covalent inhibitors.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27076" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27076" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Chemistry General Procedure</h3><div class="NLM_p">All commercially available chemical reagents were directly used without further purification. A Teledyne ISCO CombiFlash Rf<sup>+</sup> instrument equipped with a variable wavelength UV detector and a fraction collector was used to conduct flash column chromatography. RediSepRf HP C18 and flash silica columns were used for purification. High-performance liquid chromatography (HPLC) spectra for all compounds were acquired using an Agilent 1200 Series system with a DAD detector. Chromatography was performed on a 2.1 mm × 150 mm Zorbax 300SB-C18 5 μm column with water containing 0.1% formic acid as solvent A and acetonitrile containing 0.1% formic acid as solvent B at a flow rate of 0.4 mL/min. The gradient program was as follows: 1% B (0–1 min), 1–99% B (1–4 min), and 99% B (4–8 min). High-resolution mass spectra (HRMS) data were acquired in positive-ion mode using an Agilent G1969A API-TOF with an electrospray ionization (ESI) source. Nuclear magnetic resonance (NMR) spectra were acquired on a Bruker Avance-III 800 MHz spectrometer (800 MHz <sup>1</sup>H NMR, 201 MHz <sup>13</sup>C NMR). Chemical shifts are reported in ppm (δ). Preparative HPLC was performed on Agilent Prep 1200 series with UV detector set to 254 nm or 220 nM. Samples were injected into a Phenomenex Luna 250 mm × 30 mm, 5 μM C<sub>18</sub> column at room temperature (rt). The flow rate was 40 mL/min. A linear gradient was used with 10% of MeOH or CH<sub>3</sub>CN (A) in H<sub>2</sub>O (with 0.1% TFA) (B) to 100% of MeOH or CH<sub>3</sub>CN (A). HPLC was used to establish the purity of target compounds. All final compounds had >95% purity using the HPLC methods described above. All final compounds are characterized as trifluoroacetic acid salt form.</div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> 1-(<i>tert</i>-Butyl) 3-Ethyl 4-(3-nitrophenyl)-1<i>H</i>-pyrrole-1,3-dicarboxylate (<b>1</b>)</h4><div class="NLM_p last">To a suspension of sodium hydride (60% in mineral oil, 1.44 g, 36.0 mmol) in DMF/THF (50 mL, v/v, 1:1) at 0 °C was slowly added a solution of 1-((isocyanomethyl)sulfonyl)-4-methylbenzene (2.3 g, 12.0 mmol) and ethyl (<i>E</i>)-3-(3-nitrophenyl)acrylate (2.6 g, 12.0 mmol) in DMF/THF (50 mL, v/v, 1:1) over 30 min. After the completion of addition, the reaction was warmed to room temperature and stirred for 1 h. The reaction was then quenched with water and diluted with ethyl acetate. The organic layer was washed twice with water, dried over anhydrous sodium sulfate, filtered, and concentrated under reduced pressure. The resulting crude intermediate was used in the next step without further purification. To a solution of the above intermediate, triethylamine (1.8 g, 18.0 mmol), and 4-dimethylaminopyridine (110 mg, 0.9 mmol) in dichloromethane (100 mL) was added a solution of di-<i>tert</i>-butyl dicarbonate (2.6 g, 12.0 mmol) in dichloromethane (20 mL). After being stirred overnight, the reaction was quenched with water, separated, dried over anhydrous sodium sulfate, concentrated, and purified by flash chromatography on silica gel column (EtOAc/hexane, 0–25%) to afford the title compound as a white solid (1.6 g, yield 37%). <sup>1</sup>H NMR (800 MHz, CDCl<sub>3</sub>) δ 8.31 (s, 1H), 8.11 (dd, <i>J</i> = 8.3, 2.2 Hz, 1H), 7.92 (d, <i>J</i> = 2.5 Hz, 1H), 7.79 (d, <i>J</i> = 7.6 Hz, 1H), 7.48 (t, <i>J</i> = 7.9 Hz, 1H), 7.33–7.27 (m, 1H), 4.22 (q, <i>J</i> = 7.0 Hz, 2H), 1.63 (s, 9H), 1.24 (t, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (201 MHz, CDCl<sub>3</sub>) δ 163.42, 147.83, 147.59, 135.39, 135.29, 128.53, 126.86, 126.06, 124.07, 121.83, 119.99, 116.86, 85.68, 60.26, 27.80, 14.12.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> <i>tert</i>-Butyl 3-Formyl-4-(3-nitrophenyl)-1<i>H</i>-pyrrole-1-carboxylate (<b>2</b>)</h4><div class="NLM_p last">To a solution of 1-(<i>tert</i>-butyl) 3-ethyl 4-(3-nitrophenyl)-1<i>H</i>-pyrrole-1,3-dicarboxylate (1.6 g, 4.4 mmol) in THF (100 mL) at −78 °C was slowly added diisobutylaluminium hydride (13.2 mL, 1 M in hexane, 13.2 mmol) over 30 min. The resulting solution was stirred at −78 °C for 3 h before the reaction was quenched with 2 mL of methanol. The solution was poured to saturated aqueous solution of potassium sodium tartrate (150 mL). The resulting mixture was vigorously stirred until two phases can be separated. The organic phase was dried over anhydrous sodium sulfate and concentrated under reduced pressure. The crude intermediate was used in the next step without further purification. To a solution of the above intermediate in dichloromethane (50 mL) was added Dess–Martin periodinane (1.87 g, 4.4 mmol) in portions. The resulting reaction mixture was stirred for 30 min and filtered. The filtrate was concentrated and purified by silica gel flash chromatography (EtOAc/hexane, 0–25%) to afford the title compound as an oil (1 g, yield 71%). <sup>1</sup>H NMR (800 MHz, CDCl<sub>3</sub>) δ 9.92 (s, 1H), 8.39 (s, 1H), 8.19–8.14 (m, 1H), 8.01–7.97 (m, 1H), 7.91 (d, <i>J</i> = 7.6 Hz, 1H), 7.56 (t, <i>J</i> = 7.9 Hz, 1H), 7.45 (d, <i>J</i> = 2.3 Hz, 1H), 1.67 (s, 9H). <sup>13</sup>C NMR (201 MHz, CDCl<sub>3</sub>) δ 185.21, 148.21, 147.36, 134.95, 134.39, 130.97, 129.14, 125.54, 125.04, 123.40, 122.22, 120.93, 86.41, 27.86.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> <i>tert</i>-Butyl 3-(((2-((<i>tert</i>-Butoxycarbonyl)amino)ethyl)(methyl)amino)methyl)-4-(3-nitrophenyl)-1<i>H</i>-pyrrole-1-carboxylate (<b>3</b>)</h4><div class="NLM_p last">To a solution of <i>tert</i>-butyl 3-formyl-4-(3-nitrophenyl)-1<i>H</i>-pyrrole-1-carboxylate (760 mg, 2.41 mmol) and <i>tert</i>-butyl (2-(methylamino)ethyl)carbamate (630 mg, 3.62 mmol) in dichloromethane (30 mL) was added sodium triacetoxylborohydride (1.02 g, 4.82 mmol) in portions. The reaction mixture was stirred overnight at room temperature before it was quenched with 20 mL of saturated aqueous solution of sodium bicarbonate. The aqueous layer was extracted with dichloromethane (3 × 20 mL). The combined organic phase was dried over anhydrous sodium sulfate, filtered, and concentrated under reduced pressure. The residue was purified by silica gel flash chromatography (EtOAc/hexane: 0–50%) to afford the title compound <b>3</b> as an oil (1.0 g, yield 88%). <sup>1</sup>H NMR (800 MHz, CD<sub>3</sub>OD) δ 8.67 (s, 1H), 8.17 (d, <i>J</i> = 8.2 Hz, 1H), 8.03 (d, <i>J</i> = 7.8 Hz, 1H), 7.67 (t, <i>J</i> = 7.9 Hz, 1H), 7.55 (d, <i>J</i> = 2.3 Hz, 1H), 7.43 (s, 1H), 3.78–3.65 (m, 2H), 3.25 (t, <i>J</i> = 6.3 Hz, 2H), 2.76–2.64 (m, 2H), 2.42 (s, 3H), 1.69 (s, 9H), 1.42 (s, 9H). <sup>13</sup>C NMR (201 MHz, CD<sub>3</sub>OD) δ 155.47, 147.10, 146.78, 134.69, 132.43, 127.97, 124.82, 121.26, 120.43, 119.62, 117.39, 82.99, 58.69, 53.82, 50.98, 38.71, 35.66, 25.89, 25.38. MS (ESI) <i>m</i>/<i>z</i> 475.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> <i>N</i>-(3-(4-(((2-Aminoethyl)(methyl)amino)methyl)-1<i>H</i>-pyrrol-3-yl)phenyl)acrylamide (<b>4</b>)</h4><div class="NLM_p last">To a solution of <i>tert</i>-butyl 3-(((2-((<i>tert</i>-butoxycarbonyl)amino)ethyl)(methyl)amino)methyl)-4-(3-nitrophenyl)-1<i>H</i>-pyrrole-1-carboxylate (200 mg, 0.42 mmol) in methanol (8 mL) was added Pd/C (30 mg, 10 wt % Pd basis). After the resulting mixture was stirred under a hydrogen atmosphere for 2 h, the mixture was filtered through a Celite cup and the filtrate was concentrated under reduced pressure. The crude intermediate was used in the next step without further purification. To a solution of the above intermediate in dichloromethane (10 mL) at 0 °C was added a solution of acryloyl chloride (38 mg, 0.42 mmol) in dichloromethane (2 mL). The resulting solution was stirred for 10 min before half of the solvent was removed under reduced pressure. The reaction was treated with trifluoroacetic acid (4 mL) for 1 h before the volatile was removed under reduced pressure. The resulting residue was purified by reverse prep-HPLC to afford compound <b>4</b> as TFA salt (82 mg, yield 37%) after lyophilization. <sup>1</sup>H NMR (800 MHz, CD<sub>3</sub>OD) δ 7.84 (s, 1H), 7.43–7.36 (m, 2H), 7.19 (d, <i>J</i> = 8.0 Hz, 2H), 6.99 (d, <i>J</i> = 2.1 Hz, 1H), 6.50 (dd, <i>J</i> = 17.0, 10.2 Hz, 1H), 6.41 (d, <i>J</i> = 16.9 Hz, 1H), 5.82 (d, <i>J</i> = 10.2 Hz, 1H), 4.53 (s, 2H), 3.44–3.13 (m, 4H), 2.73 (s, 3H). <sup>13</sup>C NMR (201 MHz, CD<sub>3</sub>OD) δ 165.14, 138.56, 135.86, 131.06, 129.26, 126.65, 125.05, 124.56, 122.18, 120.68, 118.33, 117.86, 107.60, 52.06, 50.83, 38.61, 33.94. MS (ESI) <i>m</i>/<i>z</i> 299.3 [M + H]<sup>+</sup>. HRMS (ESI): calcd for C<sub>17</sub>H<sub>22</sub>N<sub>4</sub>O + H: 299.1866; found: 299.1868 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> <i>N</i>-(3-(4-(((2-Aminoethyl)(methyl)amino)methyl)-1<i>H</i>-pyrrol-3-yl)phenyl)propionamide (<b>5</b>)</h4><div class="NLM_p last">Compound <b>5</b> was synthesized according to the procedures for the preparation of compound <b>4</b>. <i>tert</i>-Butyl 3-(((2-((<i>tert</i>-butoxycarbonyl)amino)ethyl)(methyl)amino)methyl)-4-(3-nitrophenyl)-1<i>H</i>-pyrrole-1-carboxylate (100 mg, 0.21 mmol) and propionyl chloride (27 mg, 0.30 mmol) were used to afford compound <b>5</b> (32 mg, yield 29%) after lyophilization. <sup>1</sup>H NMR (800 MHz, CD<sub>3</sub>OD) δ 7.73 (s, 1H), 7.38 (t, <i>J</i> = 7.8 Hz, 1H), 7.32 (d, <i>J</i> = 8.0 Hz, 1H), 7.25–7.06 (m, 2H), 6.97 (d, <i>J</i> = 2.1 Hz, 1H), 4.51 (s, 2H), 3.60–3.04 (m, 4H), 2.72 (s, 3H), 2.45 (q, <i>J</i> = 7.6 Hz, 2H), 1.24 (t, <i>J</i> = 7.5 Hz, 3H). <sup>13</sup>C NMR (201 MHz, CD<sub>3</sub>OD) δ 174.46, 138.76, 135.75, 129.18, 125.13, 124.23, 122.15, 120.68, 118.26, 117.80, 107.55, 52.02, 50.84, 38.60, 33.95, 29.62, 8.82. MS (ESI) <i>m</i>/<i>z</i> 301.2 [M + H]<sup>+</sup>. HRMS (ESI): calcd for C<sub>17</sub>H<sub>25</sub>N<sub>4</sub>O + H: 301.2023; found: 301.2008 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> <i>tert</i>-Butyl 3-(((2-Hydroxyethyl)(methyl)amino)methyl)-4-(3-nitrophenyl)-1<i>H</i>-pyrrole-1-carboxylate (<b>6</b>)</h4><div class="NLM_p last">To a solution of <i>tert</i>-butyl 3-formyl-4-(3-nitrophenyl)-1<i>H</i>-pyrrole-1-carboxylate (240 mg, 0.76 mmol) and 2-(methylamino)ethan-1-ol (85.5 mg, 1.14 mmol) in dichloromethane (10 mL) was added sodium triacetoxylborohydride (322 mg, 1.52 mmol) in portions. The mixture was stirred overnight at room temperature before the reaction was quenched with 5 mL of saturated aqueous solution of sodium bicarbonate. The aqueous layer was extracted with dichloromethane (3 × 10 mL). The combined organic phase was dried over anhydrous sodium sulfate, filtered, and concentrated under reduced pressure. The residue was purified by silica gel flash chromatography (EtOAc/hexane: 0–50%) to afford the title compound as an oil (210 mg, yield 74%). <sup>1</sup>H NMR (800 MHz, CD<sub>3</sub>OD) δ 8.63 (s, 1H), 8.17–8.07 (m, 1H), 8.02 (d, <i>J</i> = 7.7 Hz, 1H), 7.60 (t, <i>J</i> = 7.9 Hz, 1H), 7.50 (d, <i>J</i> = 2.4 Hz, 1H), 7.31 (d, <i>J</i> = 2.3 Hz, 1H), 3.65 (t, <i>J</i> = 6.3 Hz, 2H), 3.48 (s, 2H), 2.60 (t, <i>J</i> = 6.3 Hz, 2H), 2.28 (s, 3H), 1.65 (s, 9H). <sup>13</sup>C NMR (201 MHz, CD<sub>3</sub>OD) δ 148.51, 148.44, 136.45, 133.92, 129.20, 126.45, 122.61, 122.05, 121.11, 120.85, 118.42, 84.19, 59.18, 58.24, 53.06, 40.93, 26.79. MS (ESI) <i>m</i>/<i>z</i> 376.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> <i>N</i>-(3-(4-(((2-Hydroxyethyl)(methyl)amino)methyl)-1<i>H</i>-pyrrol-3-yl)phenyl)acrylamide (<b>7</b>)</h4><div class="NLM_p last">To a solution of <i>tert</i>-butyl 3-(((2-hydroxyethyl)(methyl)amino)methyl)-4-(3-nitrophenyl)-1<i>H</i>-pyrrole-1-carboxylate (210 mg, 0.56 mmol) in methanol (8 mL) was added Pd/C (30 mg, 10 wt % Pd basis). After the resulting mixture was stirred under a hydrogen atmosphere for 2 h, it was filtered through a Celite cup, and the filtrate was concentrated under reduced pressure. The crude intermediate was used in the next step without further purification. To a solution of the above intermediate in dichloromethane (10 mL) at 0 °C was added a solution of acryloyl chloride (50 mg, 0.56 mmol) in dichloromethane (2 mL). The resulting solution was stirred for 10 min before half of the solvent was removed under reduced pressure. The reaction was treated with trifluoroacetic acid (4 mL) for 1 h before the volatile was removed under reduced pressure. The resulting residue was purified by reverse prep-HPLC to afford compound <b>7</b> as TFA salt (98 mg, yield 42%) after lyophilization. <sup>1</sup>H NMR (800 MHz, CD<sub>3</sub>OD) δ 7.83 (s, 1H), 7.41 (dt, <i>J</i> = 15.3, 8.1 Hz, 2H), 7.18 (d, <i>J</i> = 7.2 Hz, 1H), 7.14 (d, <i>J</i> = 2.3 Hz, 1H), 6.98 (d, <i>J</i> = 2.1 Hz, 1H), 6.51–6.45 (m, 1H), 6.40 (d, <i>J</i> = 16.9 Hz, 1H), 5.81 (d, <i>J</i> = 10.2 Hz, 1H), 4.53 (d, <i>J</i> = 14.0 Hz, 1H), 4.43 (d, <i>J</i> = 14.0 Hz, 1H), 3.71 (t, <i>J</i> = 5.2 Hz, 2H), 3.18 (dt, <i>J</i> = 12.2, 5.6 Hz, 1H), 2.92 (dt, <i>J</i> = 13.3, 4.8 Hz, 1H), 2.66 (s, 3H). <sup>13</sup>C NMR (201 MHz, CD<sub>3</sub>OD) δ 164.99, 138.67, 136.13, 131.08, 129.15, 126.56, 125.14, 124.36, 122.05, 120.37, 118.01, 117.73, 107.86, 55.86, 55.00, 51.13, 38.60. MS (ESI) <i>m</i>/<i>z</i> 300.1 [M + H]<sup>+</sup>. HRMS (ESI): calcd for C<sub>17</sub>H<sub>21</sub>N<sub>3</sub>O<sub>2</sub> + H: 300.1707; found: 300.1699 [M + H]<sup>+</sup>.</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Protein Expression, Purification, Co-crystallization, and Structural Determination</h3><div class="NLM_p last">Human PRMT6 protein was expressed and purified according to the previously published protocol.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> PRMT6 at 5.6 mg/mL was mixed with a 5-fold molar excess of <i>S</i>-adenosyl-<span class="smallcaps smallerCapital">l</span>-homocysteine (SAH) and a 3-fold molar excess of compound <b>4</b> (dissolved from a previously prepared 100 mM dimethyl sulfoxide (DMSO) stock solution). Diffraction-quality crystals were obtained by setting a 96-well vapor-diffusion sitting drop at room temperature, in a precipitant solution containing 0.1 M sodium malonate pH 7.0, 12% (w/v) PEG 3350. The PRMT6–compound <b>4</b> crystal was cryo-protected by immersing it into a precipitant solution supplemented with 10% glycerol and then into paratone, and cryo-cooled in liquid nitrogen. The PRMT6–compound <b>4</b> dataset was collected with an in-house Rigaku FR-E SuperBright and processed with HKL3000.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Initial phases were obtained by Fourier transform in refmac5<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> using PRMT6 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EGS">5EGS</a>) as the initial model. Model building was performed in COOT,<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> and the structure was validated with MolProbity.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Restraints for compound <b>4</b> was generated using Grade Web Server (<a href="http://grade.globalphasing.org" class="extLink">http://grade.globalphasing.org</a>).</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Biochemical Assays</h3><div class="NLM_p last">The inhibitory effects of compound <b>4</b>, compound <b>5</b>, and compound <b>7</b> against PRMT1, PRMT3, PRMT4, PRMT6, and PRMT8 were tested using the published scintillation proximity assay (SPA).<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> The tritiated <i>S</i>-adenosyl-<span class="smallcaps smallerCapital">l</span>-methionine (<sup>3</sup>H-SAM, PerkinElmer Life Sciences) was used as the donor of the methyl group. The <sup>3</sup>H-methylated biotin-labeled peptide (subsrate) was captured in a streptavidin/scintillant-coated microplate (FlashPlate PLUS; PerkinElmer Life Sciences), which brings the incorporated <sup>3</sup>H-methyl and the scintillant to close proximity resulting in light emission that is quantified by tracing the radioactivity signal (counts per minute), as measured by a TopCount NXT Microplate Scintillation and Luminescence Counter (PerkinElmer Life Sciences). The IC<sub>50</sub> values were determined under balanced conditions at <i>K</i><sub>m</sub> concentrations of both substrate and cofactor by titration of test compounds in the reaction mixture.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Mass Spectrometry Analysis for Assessing the Covalent Binding</h3><div class="NLM_p last">PRMT proteins (15 μM) were incubated with 20 molar excess of the PRMT inhibitors (300 μM) for 1 h at rt before being quenched by adding 0.1% trifluoroacetic acid (aq.). The resulting samples were separated over an HPLC column with 5–95% acetonitrile in water as an eluent. And the MS data were analyzed using an Agilent LC/MSD Time-of-Flight Mass Spectrometer equipped with an electrospray ionization source.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Methyltransferase Selectivity Assays</h3><div class="NLM_p last">The inhibitory effects of compound <b>4</b>, compound <b>5</b>, and compound <b>7</b> on 33 methyltransferase (MT) activities were assessed at two concentrations of 10 and 50 μM, as previously described.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a></div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> <i>K</i><sub>I</sub> and <i>k</i><sub>inact</sub> Determinations</h3><div class="NLM_p last">The values of <i>K</i><sub>I</sub> and <i>k</i><sub>inact</sub> were estimated by time-dependent IC<sub>50</sub> values using XLfit, as described previously.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> A series of IC<sub>50</sub> values were measured using the SPA by preincubating compound <b>4</b> and compound <b>5</b> with PRMT6 at different time points ranging from 1 to 30 min.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Cellular Assay</h3><div class="NLM_p last">The effects of compound <b>4</b>, compound <b>5</b>, and compound <b>7</b> inhibiting PRMT1 in MCF7 cells,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> PRMT3 in HEK293T cells,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> PRMT4 in MCF7 cells,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> or PRMT6 in HEK293T cells<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> were assessed as previously reported.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> Cell Viability Assay</h3><div class="NLM_p last">MCF7, PNT2, and HEK293T were seeded (1 × 10<sup>4</sup>) into 96-well plates in triplicate. After 24 h, the cells were treated with serially diluted compounds (compound <b>4</b>, compound <b>5</b>, and compound <b>7</b>) from 20 μM for 3 days. Cell viability was assessed using CCK-8 (cell counting kit-8, WST-8). Briefly, a 1× solution of CCK-8 (Dojindo, CK04) was warmed up at room temperature. Then, CCK-8 was added to each of the wells (10 μL per 100 μL medium) and then incubated for 4 h at 37 °C. The absorbance was read at 450 nm using Infinite F PLEX plate reader (TECAN, Morrisville, NC). The data were analyzed using GraphPad Prism 8. Results are shown as mean ± SD from three replicate experiments.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i30"><a href="/doi/suppl/10.1021/acs.jmedchem.0c00406" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_62853" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_62853" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c00406?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00406</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Crystallography data and refinement statistics; selectivity data of compound <b>4</b> against 33 methyltransferases; sequence alignment of type I PRMTs; MS assay results for PRMT1, PRMT3, PRMT4, and PRMT8 preincubated with compound <b>4</b>; and <sup>1</sup>H and <sup>13</sup>C NMR spectra of compounds <b>1</b>–<b>7</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00406/suppl_file/jm0c00406_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings for all compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00406/suppl_file/jm0c00406_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00406/suppl_file/jm0c00406_si_001.pdf">jm0c00406_si_001.pdf (2.53 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00406/suppl_file/jm0c00406_si_002.csv">jm0c00406_si_002.csv (0.94 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The structure presented in this article has been deposited in the Protein Data Bank under accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6P7I">6P7I</a> for the PRMT6–MS117 complex. The authors will release the atomic coordinates upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.0c00406" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66248" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66248" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jing Liu</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch
Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#670d0e0900490b0e12270a14140a49020312"><span class="__cf_email__" data-cfemail="bfd5d6d1d891d3d6caffd2ccccd291dadbca">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Masoud Vedadi</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmacology and Toxicology, University
of Toronto, Toronto, Ontario M5S 1A8, Canada</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#1a77346c7f7e7b7e735a6f6e756875746e7534797b"><span class="__cf_email__" data-cfemail="9bf6b5edfefffafff2dbeeeff4e9f4f5eff4b5f8fa">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jian Jin</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch
Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2387-3862" title="Orcid link">http://orcid.org/0000-0002-2387-3862</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#e882818986c6828186a8859b9b85c68d8c9d"><span class="__cf_email__" data-cfemail="bdd7d4dcd393d7d4d3fdd0ceced093d8d9c8">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yudao Shen</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch
Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fengling Li</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Magdalena M. Szewczyk</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Levon Halabelian</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kwang-su Park</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch
Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Irene Chau</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Aiping Dong</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hong Zeng</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">He Chen</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch
Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fanye Meng</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch
Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dalia Barsyte-Lovejoy</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cheryl H. Arrowsmith</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Princess
Margaret Cancer Centre and Department of Medical Biophysics, University of Toronto, Toronto, Ontario M5G 2M9, Canada</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4971-3250" title="Orcid link">http://orcid.org/0000-0002-4971-3250</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter J. Brown</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>Y.S., F.L., and M.M.S. contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e2323-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i33">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_68282" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_68282" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The research described here was supported in part by the grant R01GM122749 (to J.J.) from the U.S. National Institutes of Health. The SGC is a registered charity (number 1097737) that receives funds from AbbVie, Bayer Pharma AG, Boehringer Ingelheim, Canada Foundation for Innovation, Eshelman Institute for Innovation, Genome Canada through Ontario Genomics Institute [OGI-055], Innovative Medicines Initiative (EU/EFPIA) [ULTRA-DD grant no. 115766], Janssen, Merck KGaA, Darmstadt, Germany, MSD, Novartis Pharma AG, Ontario Ministry of Research, Innovation and Science (MRIS), Pfizer, São Paulo Research Foundation-FAPESP, Takeda, and Wellcome [106169/ZZ14/Z]. The authors also thank Drs. Jithesh Kottur and Prashasti Kumar for helpful suggestions.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">PRMT</td><td class="NLM_def"><p class="first last">protein arginine methyltransferase</p></td></tr><tr><td class="NLM_term">SAM</td><td class="NLM_def"><p class="first last"><i>S</i>-adenosylmethionine</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">ESI</td><td class="NLM_def"><p class="first last">electrospray ionization</p></td></tr><tr><td class="NLM_term">SPA</td><td class="NLM_def"><p class="first last">scintillation proximity assay</p></td></tr><tr><td class="NLM_term">H3R2me2a</td><td class="NLM_def"><p class="first last">H3R2 asymmetric dimethylation</p></td></tr><tr><td class="NLM_term">H4R3me2a</td><td class="NLM_def"><p class="first last">H4R3 asymmetric dimethylation</p></td></tr><tr><td class="NLM_term">BAF155-Rme2a</td><td class="NLM_def"><p class="first last">BAF155 arginine asymmetric demethylation</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i35">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07137" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07137" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 30 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frankel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yadav, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Branscombe, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span> <span> </span><span class="NLM_article-title">The novel human protein arginine N-methyltransferase PRMT6 is a nuclear enzyme displaying unique substrate specificity</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>277</i></span>,  <span class="NLM_fpage">3537</span>– <span class="NLM_lpage">3543</span>, <span class="refDoi"> DOI: 10.1074/jbc.M108786200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.1074%2Fjbc.M108786200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=11724789" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BD38XhtVaktL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2002&pages=3537-3543&author=A.+Frankelauthor=N.+Yadavauthor=J.+H.+Leeauthor=T.+L.+Branscombeauthor=S.+Clarkeauthor=M.+T.+Bedford&title=The+novel+human+protein+arginine+N-methyltransferase+PRMT6+is+a+nuclear+enzyme+displaying+unique+substrate+specificity&doi=10.1074%2Fjbc.M108786200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The novel human protein arginine N-methyltransferase PRMT6 is a nuclear enzyme displaying unique substrate specificity</span></div><div class="casAuthors">Frankel, Adam; Yadav, Neelu; Lee, Jaeho; Branscombe, Tina L.; Clarke, Steven; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">277</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">3537-3543</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Protein arginine methylation is a prevalent posttranslational modification in eukaryotic cells that has been implicated in signal transduction, the metab. of nascent pre-RNA, and the transcriptional activation processes.  In searching the human genome for protein arginine N-methyltransferase (PRMT) family members, a novel gene has been found on chromosome 1 that encodes for an apparent methyltransferase, PRMT6.  The polypeptide chain of PRMT6 is 41.9 kDa consisting of a catalytic core sequence common to other PRMT enzymes.  Expressed as a glutathione S-transferase fusion protein, PRMT6 demonstrates type I PRMT activity, capable of forming both ω-NG-monomethylarginine and asym. ω-NG,NG-dimethylarginine derivs. on the recombinant glycine- and arginine-rich substrate in a processive manner with a specific activity of 144 pmol Me groups transferred min-1 mg-1 enzyme.  A comparison of substrate specificity reveals that PRMT6 is functionally distinct from two previously characterized type I enzymes, PRMT1 and PRMT4.  In addn., PRMT6 displays automethylation activity; it is the first PRMT to do so.  This novel human PRMT, which resides solely in the nucleus when fused to the green fluorescent protein, joins a family of enzymes with diverse functions within cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqs_K0kYqP2NbVg90H21EOLACvtfcHk0ljGJd3tG0_hOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XhtVaktL4%253D&md5=1faaf27f44d72742749bdee2998ec9c2</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M108786200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M108786200%26sid%3Dliteratum%253Aachs%26aulast%3DFrankel%26aufirst%3DA.%26aulast%3DYadav%26aufirst%3DN.%26aulast%3DLee%26aufirst%3DJ.%2BH.%26aulast%3DBranscombe%26aufirst%3DT.%2BL.%26aulast%3DClarke%26aufirst%3DS.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DThe%2520novel%2520human%2520protein%2520arginine%2520N-methyltransferase%2520PRMT6%2520is%2520a%2520nuclear%2520enzyme%2520displaying%2520unique%2520substrate%2520specificity%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2002%26volume%3D277%26spage%3D3537%26epage%3D3543%26doi%3D10.1074%2Fjbc.M108786200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, S. G.</span></span> <span> </span><span class="NLM_article-title">Protein arginine methylation in mammals: who, what, and why</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">13</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2008.12.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.1016%2Fj.molcel.2008.12.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=19150423" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFOitrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2009&pages=1-13&author=M.+T.+Bedfordauthor=S.+G.+Clarke&title=Protein+arginine+methylation+in+mammals%3A+who%2C+what%2C+and+why&doi=10.1016%2Fj.molcel.2008.12.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methylation in mammals: who, what, and why</span></div><div class="casAuthors">Bedford, Mark T.; Clarke, Steven G.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-13</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The covalent marking of proteins by Me group addn. to arginine residues can promote their recognition by binding partners or can modulate their biol. activity.  A small family of gene products that catalyze such methylation reactions in eukaryotes (PRMTs) works in conjunction with a changing cast of assocd. subunits to recognize distinct cellular substrates.  These reactions display many of the attributes of reversible covalent modifications such as protein phosphorylation or protein lysine methylation; however, it is unclear to what extent protein arginine demethylation occurs.  Physiol. roles for protein arginine methylation have been established in signal transduction, mRNA splicing, transcriptional control, DNA repair, and protein translocation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqN0aG9XUXV27Vg90H21EOLACvtfcHk0ljfBO-kHrmW_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFOitrg%253D&md5=0c8f178d730d06c37a87b8bafd0255a1</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2008.12.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2008.12.013%26sid%3Dliteratum%253Aachs%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DClarke%26aufirst%3DS.%2BG.%26atitle%3DProtein%2520arginine%2520methylation%2520in%2520mammals%253A%2520who%252C%2520what%252C%2520and%2520why%26jtitle%3DMol.%2520Cell%26date%3D2009%26volume%3D33%26spage%3D1%26epage%3D13%26doi%3D10.1016%2Fj.molcel.2008.12.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. Ü.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martini, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of protein methyltransferases and dimethylases</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">989</span>– <span class="NLM_lpage">1068</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.6b00801</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.6b00801" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkslynsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2018&pages=989-1068&author=H.+%C3%9C.+Kaniskanauthor=M.+L.+Martiniauthor=J.+Jin&title=Inhibitors+of+protein+methyltransferases+and+dimethylases&doi=10.1021%2Facs.chemrev.6b00801"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of Protein Methyltransferases and Demethylases</span></div><div class="casAuthors">Kaniskan, H. Umit; Martini, Michael L.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">989-1068</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Post-translational modifications of histones by protein methyltransferases (PMTs) and histone demethylases (KDMs) play an important role in the regulation of gene expression and transcription and are implicated in cancer and many other diseases.  Many of these enzymes also target various nonhistone proteins impacting numerous crucial biol. pathways.  Given their key biol. functions and implications in human diseases, there has been a growing interest in assessing these enzymes as potential therapeutic targets.  Consequently, discovering and developing inhibitors of these enzymes has become a very active and fast-growing research area over the past decade.  In this review, the authors cover the discovery, characterization, and biol. application of inhibitors of PMTs and KDMs with emphasis on key advancements in the field.  The authors also discuss challenges, opportunities, and future directions in this emerging, exciting research field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJYjiYtw_n4rVg90H21EOLACvtfcHk0ljfBO-kHrmW_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkslynsL8%253D&md5=1ed520c6b0c42d2aa8a7f66b0a2d4e40</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.6b00801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.6b00801%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3DMartini%26aufirst%3DM.%2BL.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DInhibitors%2520of%2520protein%2520methyltransferases%2520and%2520dimethylases%26jtitle%3DChem.%2520Rev.%26date%3D2018%26volume%3D118%26spage%3D989%26epage%3D1068%26doi%3D10.1021%2Facs.chemrev.6b00801" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span> <span> </span><span class="NLM_article-title">Protein arginine methyltransferases and cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">37</span>– <span class="NLM_lpage">50</span>, <span class="refDoi"> DOI: 10.1038/nrc3409</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.1038%2Fnrc3409" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=23235912" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVajur7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2013&pages=37-50&author=Y.+Yangauthor=M.+T.+Bedford&title=Protein+arginine+methyltransferases+and+cancer&doi=10.1038%2Fnrc3409"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methyltransferases and cancer</span></div><div class="casAuthors">Yang, Yanzhong; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">37-50</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  There are nine protein arginine methyltransferases (PRMTs) encoded in mammalian genomes, the protein products of which catalyze three types of arginine methylation - monomethylation and two types of dimethylation.  Protein arginine methylation is an abundant modification that has been implicated in signal transduction, gene transcription, DNA repair and mRNA splicing, among others.  Studies have only recently linked this modification to carcinogenesis and metastasis.  Sequencing studies have not generally found alterations to the PRMTs; however, overexpression of these enzymes is often assocd. with various cancers, which might make some of them viable targets for therapeutic strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnb4O4wtn5fLVg90H21EOLACvtfcHk0li9nOVehNfpyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVajur7I&md5=9df3b58d92e7932e214e6de26a10fa46</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrc3409&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3409%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DY.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DProtein%2520arginine%2520methyltransferases%2520and%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2013%26volume%3D13%26spage%3D37%26epage%3D50%26doi%3D10.1038%2Fnrc3409" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wei, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mundade, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lange, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span> <span> </span><span class="NLM_article-title">Protein arginine methylation of non-histone proteins and its role in diseases</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">32</span>– <span class="NLM_lpage">41</span>, <span class="refDoi"> DOI: 10.4161/cc.27353</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.4161%2Fcc.27353" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=24296620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltFeruro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=32-41&author=H.+Weiauthor=R.+Mundadeauthor=K.+C.+Langeauthor=T.+Lu&title=Protein+arginine+methylation+of+non-histone+proteins+and+its+role+in+diseases&doi=10.4161%2Fcc.27353"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methylation of non-histone proteins and its role in diseases</span></div><div class="casAuthors">Wei, Han; Mundade, Rasika; Lange, Kevin C.; Lu, Tao</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">32-41</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1538-4101</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">A review.  Protein arginine methyltransferases (PRMTs) are a family of enzymes that can methylate arginine residues on histones and other proteins.  PRMTs play a crucial role in influencing various cellular functions, including cellular development and tumorigenesis.  Arginine methylation by PRMTs is found on both nuclear and cytoplasmic proteins.  Recently, there is increasing evidence regarding post-translational modifications of non-histone proteins by PRMTs, illustrating the previously unknown importance of PRMTs in the regulation of various cellular functions by post-translational modifications.  In this review, we present the recent developments in the regulation of non-histone proteins by PRMTs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIW3UyE1q0IbVg90H21EOLACvtfcHk0li9nOVehNfpyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltFeruro%253D&md5=55f3ea91e0b85ccf07b66eed1c58e471</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.4161%2Fcc.27353&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.27353%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DH.%26aulast%3DMundade%26aufirst%3DR.%26aulast%3DLange%26aufirst%3DK.%2BC.%26aulast%3DLu%26aufirst%3DT.%26atitle%3DProtein%2520arginine%2520methylation%2520of%2520non-histone%2520proteins%2520and%2520its%2520role%2520in%2520diseases%26jtitle%3DCell%2520Cycle%26date%3D2014%26volume%3D13%26spage%3D32%26epage%3D41%26doi%3D10.4161%2Fcc.27353" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casadio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinato, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cesaroni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuchlautz, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luscher, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amati, B.</span></span> <span> </span><span class="NLM_article-title">Methylation of histone H3R2 by PRMT6 and H3K4 by an MLL complex are mutually exclusive</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>449</i></span>,  <span class="NLM_fpage">933</span>– <span class="NLM_lpage">937</span>, <span class="refDoi"> DOI: 10.1038/nature06166</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.1038%2Fnature06166" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=17898714" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFOjt7bK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=449&publication_year=2007&pages=933-937&author=E.+Guccioneauthor=C.+Bassiauthor=F.+Casadioauthor=F.+Martinatoauthor=M.+Cesaroniauthor=H.+Schuchlautzauthor=B.+Luscherauthor=B.+Amati&title=Methylation+of+histone+H3R2+by+PRMT6+and+H3K4+by+an+MLL+complex+are+mutually+exclusive&doi=10.1038%2Fnature06166"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Methylation of histone H3R2 by PRMT6 and H3K4 by an MLL complex are mutually exclusive</span></div><div class="casAuthors">Guccione, Ernesto; Bassi, Christian; Casadio, Fabio; Martinato, Francesca; Cesaroni, Matteo; Schuchlautz, Henning; Luescher, Bernhard; Amati, Bruno</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">449</span>
        (<span class="NLM_cas:issue">7164</span>),
    <span class="NLM_cas:pages">933-937</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Eukaryotic genomes are organized into active (euchromatic) and inactive (heterochromatic) chromatin domains.  Post-translational modifications of histones (or 'marks') are key in defining these functional states, particularly in promoter regions.  Mutual regulatory interactions between these marks-and the enzymes that catalyze them-contribute to the shaping of this epigenetic landscape, in a manner that remains to be fully elucidated.  We previously obsd. that asym. di-methylation of histone H3 arginine 2 (H3R2me2a) counter-correlates with di- and tri- methylation of H3 lysine 4 (H3K4me2, H3K4me3) on human promoters.  Here we show that the arginine methyltransferase PRMT6 catalyzes H3R2 di-methylation in vitro and controls global levels of H3R2me2a in vivo.  H3R2 methylation by PRMT6 was prevented by the presence of H3K4me3 on the H3 tail.  Conversely, the H3R2me2a mark prevented methylation of H3K4 as well as binding to the H3 tail by an ASH2/WDR5/MLL-family methyltransferase complex.  Chromatin immunopptn. showed that H3R2me2a was distributed within the body and at the 3' end of human genes, regardless of their transcriptional state, whereas it was selectively and locally depleted from active promoters, coincident with the presence of H3K4me3.  Hence, the mutual antagonism between H3R2 and H3K4 methylation, together with the assocn. of MLL-family complexes with the basal transcription machinery, may contribute to the localized patterns of H3K4 tri-methylation characteristic of transcriptionally poised or active promoters in mammalian genomes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWqE8pDpHmKLVg90H21EOLACvtfcHk0li994WIwaVTiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFOjt7bK&md5=f4cc9371491088af7b74b614c90d6bd4</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnature06166&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature06166%26sid%3Dliteratum%253Aachs%26aulast%3DGuccione%26aufirst%3DE.%26aulast%3DBassi%26aufirst%3DC.%26aulast%3DCasadio%26aufirst%3DF.%26aulast%3DMartinato%26aufirst%3DF.%26aulast%3DCesaroni%26aufirst%3DM.%26aulast%3DSchuchlautz%26aufirst%3DH.%26aulast%3DLuscher%26aufirst%3DB.%26aulast%3DAmati%26aufirst%3DB.%26atitle%3DMethylation%2520of%2520histone%2520H3R2%2520by%2520PRMT6%2520and%2520H3K4%2520by%2520an%2520MLL%2520complex%2520are%2520mutually%2520exclusive%26jtitle%3DNature%26date%3D2007%26volume%3D449%26spage%3D933%26epage%3D937%26doi%3D10.1038%2Fnature06166" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boulanger, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russell, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wainberg, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richard, S.</span></span> <span> </span><span class="NLM_article-title">Methylation of Tat by PRMT6 regulates human immunodeficiency virus type 1 gene expression</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">124</span>– <span class="NLM_lpage">131</span>, <span class="refDoi"> DOI: 10.1128/JVI.79.1.124-131.2005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.1128%2FJVI.79.1.124-131.2005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=15596808" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhslemsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2005&pages=124-131&author=M.+C.+Boulangerauthor=C.+Liangauthor=R.+S.+Russellauthor=R.+Linauthor=M.+T.+Bedfordauthor=M.+A.+Wainbergauthor=S.+Richard&title=Methylation+of+Tat+by+PRMT6+regulates+human+immunodeficiency+virus+type+1+gene+expression&doi=10.1128%2FJVI.79.1.124-131.2005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Methylation of Tat by PRMT6 regulates human immunodeficiency virus type 1 gene expression</span></div><div class="casAuthors">Boulanger, Marie-Chloe; Liang, Chen; Russell, Rodney S.; Lin, Rongtuan; Bedford, Mark T.; Wainberg, Mark A.; Richard, Stephane</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">124-131</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The human immunodeficiency virus (HIV) transactivator protein, Tat, stimulates transcription from the viral long terminal repeats via an arginine-rich transactivating domain.  Since arginines are often known to be methylated, authors investigated whether HIV type 1 (HIV-1) Tat was a substrate for known protein arginine methyltransferases (PRMTs).  Here authors identify Tat as a substrate for the arginine methyltransferase, PRMT6.  Tat is specifically assocd. with and methylated by PRMT6 within cells.  Overexpression of wild-type PRMT6, but not a methylase-inactive PRMT6 mutant, decreased Tat transactivation of an HIV-1 long terminal repeat luciferase reporter plasmid in a dose-dependent manner.  Knocking down PRMT6 consistently increased HIV-1 prodn. in HEK293T cells and also led to increased viral infectiousness as shown in multinuclear activation of a galactosidase indicator assays.  This study demonstrates that arginine methylation of Tat neg. regulates its transactivation activity and that PRMT6 acts as a restriction factor for HIV replication.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0JF5C5Agwq7Vg90H21EOLACvtfcHk0li994WIwaVTiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhslemsg%253D%253D&md5=763ac9af247b3f81962ae4e19419493a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1128%2FJVI.79.1.124-131.2005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.79.1.124-131.2005%26sid%3Dliteratum%253Aachs%26aulast%3DBoulanger%26aufirst%3DM.%2BC.%26aulast%3DLiang%26aufirst%3DC.%26aulast%3DRussell%26aufirst%3DR.%2BS.%26aulast%3DLin%26aufirst%3DR.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DWainberg%26aufirst%3DM.%2BA.%26aulast%3DRichard%26aufirst%3DS.%26atitle%3DMethylation%2520of%2520Tat%2520by%2520PRMT6%2520regulates%2520human%2520immunodeficiency%2520virus%2520type%25201%2520gene%2520expression%26jtitle%3DJ.%2520Virol.%26date%3D2005%26volume%3D79%26spage%3D124%26epage%3D131%26doi%3D10.1128%2FJVI.79.1.124-131.2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sgarra, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tessari, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spolaore, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giancotti, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manfioletti, G.</span></span> <span> </span><span class="NLM_article-title">The AT-hook of the chromatin architectural transcription factor high mobility group A1a is arginine-methylated by protein arginine methyltransferase 6</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>281</i></span>,  <span class="NLM_fpage">3764</span>– <span class="NLM_lpage">3772</span>, <span class="refDoi"> DOI: 10.1074/jbc.M510231200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.1074%2Fjbc.M510231200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=16293633" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlKksLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=3764-3772&author=R.+Sgarraauthor=J.+Leeauthor=M.+A.+Tessariauthor=S.+Altamuraauthor=B.+Spolaoreauthor=V.+Giancottiauthor=M.+T.+Bedfordauthor=G.+Manfioletti&title=The+AT-hook+of+the+chromatin+architectural+transcription+factor+high+mobility+group+A1a+is+arginine-methylated+by+protein+arginine+methyltransferase+6&doi=10.1074%2Fjbc.M510231200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The AT-hook of the Chromatin Architectural Transcription Factor High Mobility Group A1a Is Arginine-methylated by Protein Arginine Methyltransferase 6</span></div><div class="casAuthors">Sgarra, Riccardo; Lee, Jaeho; Tessari, Michela A.; Altamura, Sandro; Spolaore, Barbara; Giancotti, Vincenzo; Bedford, Mark T.; Manfioletti, Guidalberto</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">281</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3764-3772</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The HMGA1a protein belongs to the high mobility group A (HMGA) family of architectural nuclear factors, a group of proteins that plays an important role in chromatin dynamics.  HMGA proteins are multifunctional factors that assoc. both with DNA and nuclear proteins that have been involved in several nuclear processes, such as transcriptional regulation, viral integration, DNA repair, RNA processing, and chromatin remodeling.  The activity of HMGA proteins is finely modulated by a variety of post-translational modifications.  Arginine methylation was recently demonstrated to occur on HMGA1a protein, and it correlates with the apoptotic process and neoplastic progression.  Methyltransferases responsible for these modifications are unknown.  Here we show that the protein arginine methyltransferase PRMT6 specifically methylates HMGA1a protein both in vitro and in vivo.  By mass spectrometry, the sites of methylation were unambiguously mapped to Arg57 and Arg59, two residues which are embedded in the second AT-hook, a region crit. for both protein-DNA and protein-protein interactions and whose modification may cause profound alterations in the HMGA network.  The in vivo assocn. of HMGA and PRMT6 place this yet functionally uncharacterized methyltransferase in the well established functional context of the chromatin structure organization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKv8D5yCRQ4rVg90H21EOLACvtfcHk0li994WIwaVTiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlKksLs%253D&md5=803d5cd75cc725a9c381220bc067d330</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M510231200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M510231200%26sid%3Dliteratum%253Aachs%26aulast%3DSgarra%26aufirst%3DR.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DTessari%26aufirst%3DM.%2BA.%26aulast%3DAltamura%26aufirst%3DS.%26aulast%3DSpolaore%26aufirst%3DB.%26aulast%3DGiancotti%26aufirst%3DV.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DManfioletti%26aufirst%3DG.%26atitle%3DThe%2520AT-hook%2520of%2520the%2520chromatin%2520architectural%2520transcription%2520factor%2520high%2520mobility%2520group%2520A1a%2520is%2520arginine-methylated%2520by%2520protein%2520arginine%2520methyltransferase%25206%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D3764%26epage%3D3772%26doi%3D10.1074%2Fjbc.M510231200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">El-Andaloussi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valovka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toueille, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinacher, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Focke, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gehrig, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Covic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassa, P. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubscher, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hottiger, M. O.</span></span> <span> </span><span class="NLM_article-title">Arginine methylation regulates DNA polymerase beta</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">51</span>– <span class="NLM_lpage">62</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2006.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.1016%2Fj.molcel.2006.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=16600869" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BD28XjvFWiu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2006&pages=51-62&author=N.+El-Andaloussiauthor=T.+Valovkaauthor=M.+Toueilleauthor=R.+Steinacherauthor=F.+Fockeauthor=P.+Gehrigauthor=M.+Covicauthor=P.+O.+Hassaauthor=P.+Scharauthor=U.+Hubscherauthor=M.+O.+Hottiger&title=Arginine+methylation+regulates+DNA+polymerase+beta&doi=10.1016%2Fj.molcel.2006.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Arginine methylation regulates DNA polymerase β</span></div><div class="casAuthors">El-Andaloussi, Nazim; Valovka, Taras; Toueille, Magali; Steinacher, Roland; Focke, Frauke; Gehrig, Peter; Covic, Marcela; Hassa, Paul O.; Schar, Primo; Hubscher, Ulrich; Hottiger, Michael O.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">51-62</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Alterations in DNA repair lead to genomic instability and higher risk of cancer.  DNA base excision repair (BER) corrects damaged bases, apurinic sites, and single-strand DNA breaks.  Here, a regulatory mechanism for DNA polymerase β (Pol β) is described.  Pol β was found to form a complex with the protein arginine methyltransferase 6 (PRMT6) and was specifically methylated in vitro and in vivo.  Methylation of Pol β by PRMT6 strongly stimulated DNA polymerase activity by enhancing DNA binding and processivity, while single nucleotide insertion and dRP-lyase activity were not affected.  Two residues, R83 and R152, were identified in Pol β as the sites of methylation by PRMT6.  Genetic complementation of Pol β knockout cells with R83/152K mutant revealed the importance of these residues for the cellular resistance to DNA alkylating agent.  Based on our findings, we propose that PRMT6 plays a role as a regulator of BER.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonqui7wNvLvrVg90H21EOLACvtfcHk0ljaKrXNk8jCTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjvFWiu74%253D&md5=987419489fb0439acb13f3b78d2d5a1e</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2006.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2006.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DEl-Andaloussi%26aufirst%3DN.%26aulast%3DValovka%26aufirst%3DT.%26aulast%3DToueille%26aufirst%3DM.%26aulast%3DSteinacher%26aufirst%3DR.%26aulast%3DFocke%26aufirst%3DF.%26aulast%3DGehrig%26aufirst%3DP.%26aulast%3DCovic%26aufirst%3DM.%26aulast%3DHassa%26aufirst%3DP.%2BO.%26aulast%3DSchar%26aufirst%3DP.%26aulast%3DHubscher%26aufirst%3DU.%26aulast%3DHottiger%26aufirst%3DM.%2BO.%26atitle%3DArginine%2520methylation%2520regulates%2520DNA%2520polymerase%2520beta%26jtitle%3DMol.%2520Cell%26date%3D2006%26volume%3D22%26spage%3D51%26epage%3D62%26doi%3D10.1016%2Fj.molcel.2006.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vieira, F. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costa-Pinheiro, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramalho-Carvalho, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pereira, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menezes, F. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antunes, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carneiro, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliveira, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henrique, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeronimo, C.</span></span> <span> </span><span class="NLM_article-title">Deregulated expression of selected histone methylases and dimethylases in prostate carcinoma</span>. <i>Endocr. Relat. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">51</span>– <span class="NLM_lpage">61</span>, <span class="refDoi"> DOI: 10.1530/ERC-13-0375</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.1530%2FERC-13-0375" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=24200674" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtl2gsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=51-61&author=F.+Q.+Vieiraauthor=P.+Costa-Pinheiroauthor=J.+Ramalho-Carvalhoauthor=A.+Pereiraauthor=F.+D.+Menezesauthor=L.+Antunesauthor=I.+Carneiroauthor=J.+Oliveiraauthor=R.+Henriqueauthor=C.+Jeronimo&title=Deregulated+expression+of+selected+histone+methylases+and+dimethylases+in+prostate+carcinoma&doi=10.1530%2FERC-13-0375"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Deregulated expression of selected histone methylases and demethylases in prostate carcinoma</span></div><div class="casAuthors">Vieira, Filipa Quintela; Costa-Pinheiro, Pedro; Ramalho-Carvalho, Joao; Pereira, Andreia; Menezes, Francisco Duarte; Antunes, Luis; Carneiro, Isa; Oliveira, Jorge; Henrique, Rui; Jeronimo, Carmen</div><div class="citationInfo"><span class="NLM_cas:title">Endocrine-Related Cancer</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">51-61</span>CODEN:
                <span class="NLM_cas:coden">ERCAE9</span>;
        ISSN:<span class="NLM_cas:issn">1351-0088</span>.
    
            (<span class="NLM_cas:orgname">BioScientifica Ltd.</span>)
        </div><div class="casAbstract">Prostate cancer (PCa), a leading cause of cancer-related morbidity and mortality, arises through the acquisition of genetic and epigenetic alterations.  Deregulation of histone methyltransferases (HMTs) or demethylases (HDMs) has been assocd. with PCa development and progression.  However, the precise influence of altered HMTs or HDMs expression and resp. histone marks in PCa onset and progression remains largely unknown.  To clarify the role of HMTs and HDMs in prostate carcinogenesis, expression levels of 37 HMTs and 20 HDMs were assessed in normal prostate and PCa tissue samples by RT-qPCR.  SMYD3, SUV39H2, PRMT6, KDM5A, and KDM6A were upregulated, whereas KMT2A-E (MLL1-5) and KDM4B were downregulated in PCa, compared with normal prostate tissues.  Remarkably, PRMT6 was the histone modifier that best discriminated normal from tumorous tissue samples.  Interestingly, EZH2 and SMYD3 expression levels significantly correlated with less differentiated and more aggressive tumors.  Remarkably, SMYD3 expression levels were of independent prognostic value for the prediction of disease-specific survival of PCa patients with clin. localized disease submitted to radical prostatectomy.  We concluded that expression profiling of HMTs and HDMs, esp. SMYD3, might be of clin. usefulness for the assessment of PCa patients and assist in pre-therapeutic decision-making.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-hUmMxxi157Vg90H21EOLACvtfcHk0ljaKrXNk8jCTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtl2gsbs%253D&md5=bbd4eed8bf88222c179aa9a079f5d20b</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1530%2FERC-13-0375&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1530%252FERC-13-0375%26sid%3Dliteratum%253Aachs%26aulast%3DVieira%26aufirst%3DF.%2BQ.%26aulast%3DCosta-Pinheiro%26aufirst%3DP.%26aulast%3DRamalho-Carvalho%26aufirst%3DJ.%26aulast%3DPereira%26aufirst%3DA.%26aulast%3DMenezes%26aufirst%3DF.%2BD.%26aulast%3DAntunes%26aufirst%3DL.%26aulast%3DCarneiro%26aufirst%3DI.%26aulast%3DOliveira%26aufirst%3DJ.%26aulast%3DHenrique%26aufirst%3DR.%26aulast%3DJeronimo%26aufirst%3DC.%26atitle%3DDeregulated%2520expression%2520of%2520selected%2520histone%2520methylases%2520and%2520dimethylases%2520in%2520prostate%2520carcinoma%26jtitle%3DEndocr.%2520Relat.%2520Cancer%26date%3D2014%26volume%3D21%26spage%3D51%26epage%3D61%26doi%3D10.1530%2FERC-13-0375" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yoshimatsu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toyokawa, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayami, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unoki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsunoda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Field, H. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neal, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maehara, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponder, B. A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamamoto, R.</span></span> <span> </span><span class="NLM_article-title">Dysregulation of PRMT1 and PRMT6, type I arginine methyltransferases, is involved in various types of human cancers</span>. <i>Int. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">562</span>– <span class="NLM_lpage">573</span>, <span class="refDoi"> DOI: 10.1002/ijc.25366</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.1002%2Fijc.25366" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=20473859" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A280%3ADC%252BC3cbpslChtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2011&pages=562-573&author=M.+Yoshimatsuauthor=G.+Toyokawaauthor=S.+Hayamiauthor=M.+Unokiauthor=T.+Tsunodaauthor=H.+I.+Fieldauthor=J.+D.+Kellyauthor=D.+E.+Nealauthor=Y.+Maeharaauthor=B.+A.+J.+Ponderauthor=Y.+Nakamuraauthor=R.+Hamamoto&title=Dysregulation+of+PRMT1+and+PRMT6%2C+type+I+arginine+methyltransferases%2C+is+involved+in+various+types+of+human+cancers&doi=10.1002%2Fijc.25366"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Dysregulation of PRMT1 and PRMT6, Type I arginine methyltransferases, is involved in various types of human cancers</span></div><div class="casAuthors">Yoshimatsu Masanori; Toyokawa Gouji; Hayami Shinya; Unoki Motoko; Tsunoda Tatsuhiko; Field Helen I; Kelly John D; Neal David E; Maehara Yoshihiko; Ponder Bruce A J; Nakamura Yusuke; Hamamoto Ryuji</div><div class="citationInfo"><span class="NLM_cas:title">International journal of cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">562-73</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Protein arginine methylation is a novel post-translational modification regulating a diversity of cellular processes, including histone functions, but the roles of protein arginine methyltransferases (PRMTs) in human cancer are not well investigated.  To address this issue, we first examined expression levels of genes belonging to the PRMT family and found significantly higher expression of PRMT1 and PRMT6, both of which are Type I PRMTs, in cancer cells of various tissues than in non-neoplastic cells.  Abrogation of the expression of these genes with specific siRNAs significantly suppressed growth of bladder and lung cancer cells.  Expression profile analysis using the cells transfected with the siRNAs indicated that PRMT1 and PRMT6 interplay in multiple pathways, supporting regulatory roles in the cell cycle, RNA processing and also DNA replication that are fundamentally important for cancer cell proliferation.  Furthermore, we demonstrated that serum asymmetric dimethylarginine (ADMA) levels of a number of cancer cases are significantly higher than those of nontumor control cases.  In summary, our results suggest that dysregulation of PRMT1 and PRMT6 can be involved in human carcinogenesis and that these Type I arginine methyltransferases are good therapeutic targets for various types of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQAVHBzjMN_WjEwO064Z95rfW6udTcc2eaAS70kHIT2Brntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cbpslChtw%253D%253D&md5=34889d958208a4d852b0833e21a045de</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1002%2Fijc.25366&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.25366%26sid%3Dliteratum%253Aachs%26aulast%3DYoshimatsu%26aufirst%3DM.%26aulast%3DToyokawa%26aufirst%3DG.%26aulast%3DHayami%26aufirst%3DS.%26aulast%3DUnoki%26aufirst%3DM.%26aulast%3DTsunoda%26aufirst%3DT.%26aulast%3DField%26aufirst%3DH.%2BI.%26aulast%3DKelly%26aufirst%3DJ.%2BD.%26aulast%3DNeal%26aufirst%3DD.%2BE.%26aulast%3DMaehara%26aufirst%3DY.%26aulast%3DPonder%26aufirst%3DB.%2BA.%2BJ.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DHamamoto%26aufirst%3DR.%26atitle%3DDysregulation%2520of%2520PRMT1%2520and%2520PRMT6%252C%2520type%2520I%2520arginine%2520methyltransferases%252C%2520is%2520involved%2520in%2520various%2520types%2520of%2520human%2520cancers%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D2011%26volume%3D128%26spage%3D562%26epage%3D573%26doi%3D10.1002%2Fijc.25366" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Limm, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oefner, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hellerbrand, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bosserhoff, A. K.</span></span> <span> </span><span class="NLM_article-title">Deregulation of protein methylation in melanoma</span>. <i>Eur. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">1305</span>– <span class="NLM_lpage">1313</span>, <span class="refDoi"> DOI: 10.1016/j.ejca.2012.11.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.1016%2Fj.ejca.2012.11.026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=23265702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvFWhsbnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2013&pages=1305-1313&author=K.+Limmauthor=C.+Ottauthor=S.+Wallnerauthor=D.+W.+Muellerauthor=P.+Oefnerauthor=C.+Hellerbrandauthor=A.+K.+Bosserhoff&title=Deregulation+of+protein+methylation+in+melanoma&doi=10.1016%2Fj.ejca.2012.11.026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Deregulation of protein methylation in melanoma</span></div><div class="casAuthors">Limm, Katharina; Ott, Corinna; Wallner, Susanne; Mueller, Daniel W.; Oefner, Peter; Hellerbrand, Claus; Bosserhoff, Anja-Katrin</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Cancer</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1305-1313</span>CODEN:
                <span class="NLM_cas:coden">EJCAEL</span>;
        ISSN:<span class="NLM_cas:issn">0959-8049</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Loss of methylthioadenosine phosphorylase (MTAP) expression and a concomitant accumulation of 5'-methyl-thioadenosine (MTA) characterize several tumor entities including malignant melanoma.  MTA affects cellular signalling, proliferation and migration not only of cancer but also surrounding cells including lymphocytes and stromal fibroblasts.  The mode of action of MTA is still not known.  Interestingly, MTA is a known potent inhibitor of protein arginine methyltransferases (PRMTs) and is used as a tool in studying activity and impact of PRMTs.  This study aimed at analyzing PRMTs in melanoma and the potential impact of MTA on tumorigenesis.Our findings demonstrate that expression of PRMT4/CARM1 and PRMT6 is deregulated in melanoma, whereas expression of the remaining PRMTs stays unchanged.  General PRMT activity and, consequently, sym. and asym. protein methylation are reduced significantly in melanoma cells and tissues.  This is due to a loss of MTAP expression and accumulation of MTA.  Redn. of protein methylation by MTA affects cell signalling and leads, for example, to an activation of extracellular signal-regulated kinase (ERK) activity.  The effects of endogeneous MTA on PRMTs as presented in this study can strongly support the migratory and invasive phenotype of melanoma cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqujaHA2MQGG7Vg90H21EOLACvtfcHk0lj9JvB47pZDOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvFWhsbnF&md5=a29a9efe0acd75da35bb7af04bd26dc4</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.ejca.2012.11.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejca.2012.11.026%26sid%3Dliteratum%253Aachs%26aulast%3DLimm%26aufirst%3DK.%26aulast%3DOtt%26aufirst%3DC.%26aulast%3DWallner%26aufirst%3DS.%26aulast%3DMueller%26aufirst%3DD.%2BW.%26aulast%3DOefner%26aufirst%3DP.%26aulast%3DHellerbrand%26aufirst%3DC.%26aulast%3DBosserhoff%26aufirst%3DA.%2BK.%26atitle%3DDeregulation%2520of%2520protein%2520methylation%2520in%2520melanoma%26jtitle%3DEur.%2520J.%2520Cancer%26date%3D2013%26volume%3D49%26spage%3D1305%26epage%3D1313%26doi%3D10.1016%2Fj.ejca.2012.11.026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ying, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ying, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, S.</span></span> <span> </span><span class="NLM_article-title">Significant association of PRMT6 hypomethylation with colorectal cancer</span>. <i>J. Clin. Lab. Anal.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>32</i></span>, <span class="NLM_elocation-id">e22590</span> <span class="refDoi"> DOI: 10.1002/jcla.22590</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.1002%2Fjcla.22590" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=29927001" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2018&author=R.+Panauthor=H.+Yuauthor=J.+Daiauthor=C.+Zhouauthor=X.+Yingauthor=J.+Zhongauthor=J.+Zhaoauthor=Y.+Zhangauthor=B.+Wuauthor=Y.+Maoauthor=D.+Wuauthor=J.+Yingauthor=S.+Duan&title=Significant+association+of+PRMT6+hypomethylation+with+colorectal+cancer&doi=10.1002%2Fjcla.22590"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1002%2Fjcla.22590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcla.22590%26sid%3Dliteratum%253Aachs%26aulast%3DPan%26aufirst%3DR.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DDai%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DC.%26aulast%3DYing%26aufirst%3DX.%26aulast%3DZhong%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DB.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DD.%26aulast%3DYing%26aufirst%3DJ.%26aulast%3DDuan%26aufirst%3DS.%26atitle%3DSignificant%2520association%2520of%2520PRMT6%2520hypomethylation%2520with%2520colorectal%2520cancer%26jtitle%3DJ.%2520Clin.%2520Lab.%2520Anal.%26date%3D2018%26volume%3D32%26doi%3D10.1002%2Fjcla.22590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senisterra, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butler, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. Ü.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speed, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">dela Sena, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">A potent, selective, and cell-active inhibitor of human type I protein arginine methyltransferases</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">772</span>– <span class="NLM_lpage">781</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00839</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00839" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVyru7vE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=772-781&author=M.+S.+Eramauthor=Y.+D.+Shenauthor=M.+M.+Szewczykauthor=H.+Wuauthor=G.+Senisterraauthor=F.+L.+Liauthor=K.+V.+Butlerauthor=H.+%C3%9C.+Kaniskanauthor=B.+A.+Speedauthor=C.+dela+Senaauthor=A.+P.+Dongauthor=H.+Zengauthor=M.+Schapiraauthor=P.+J.+Brownauthor=C.+H.+Arrowsmithauthor=D.+Barsyte-Lovejoyauthor=J.+Liuauthor=M.+Vedadiauthor=J.+Jin&title=A+potent%2C+selective%2C+and+cell-active+inhibitor+of+human+type+I+protein+arginine+methyltransferases&doi=10.1021%2Facschembio.5b00839"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">A Potent, Selective, and Cell-Active Inhibitor of Human Type I Protein Arginine Methyltransferases</span></div><div class="casAuthors">Eram, Mohammad S.; Shen, Yudao; Szewczyk, Magdalena M.; Wu, Hong; Senisterra, Guillermo; Li, Fengling; Butler, Kyle V.; Kaniskan, H. Umit; Speed, Brandon A.; dela Sena, Carlo; Dong, Aiping; Zeng, Hong; Schapira, Matthieu; Brown, Peter J.; Arrowsmith, Cheryl H.; Barsyte-Lovejoy, Dalia; Liu, Jing; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">772-781</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferases (PRMTs) play a crucial role in a variety of biol. processes.  Overexpression of PRMTs has been implicated in various human diseases including cancer.  Consequently, selective small-mol. inhibitors of PRMTs have been pursued by both academia and the pharmaceutical industry as chem. tools for testing biol. and therapeutic hypotheses.  PRMTs are divided into three categories: type I PRMTs which catalyze mono- and asym. dimethylation of arginine residues, type II PRMTs which catalyze mono- and sym. dimethylation of arginine residues, and type III PRMT which catalyzes only monomethylation of arginine residues.  Here, we report the discovery of a potent, selective, and cell-active inhibitor of human type I PRMTs, MS023, and characterization of this inhibitor in a battery of biochem., biophys., and cellular assays.  MS023 displayed high potency for type I PRMTs including PRMT1, -3, -4, -6, and -8 but was completely inactive against type II and type III PRMTs, protein lysine methyltransferases and DNA methyltransferases.  A crystal structure of PRMT6 in complex with MS023 revealed that MS023 binds the substrate binding site.  MS023 potently decreased cellular levels of histone arginine asym. dimethylation.  It also reduced global levels of arginine asym. dimethylation and concurrently increased levels of arginine monomethylation and sym. dimethylation in cells.  We also developed MS094, a close analog of MS023, which was inactive in biochem. and cellular assays, as a neg. control for chem. biol. studies.  MS023 and MS094 are useful chem. tools for investigating the role of type I PRMTs in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_ysJCXRFiPbVg90H21EOLACvtfcHk0lj9JvB47pZDOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVyru7vE&md5=d4e103b75c2ae19572b4738ca53a271d</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00839&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00839%26sid%3Dliteratum%253Aachs%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DShen%26aufirst%3DY.%2BD.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DF.%2BL.%26aulast%3DButler%26aufirst%3DK.%2BV.%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3DSpeed%26aufirst%3DB.%2BA.%26aulast%3Ddela%2BSena%26aufirst%3DC.%26aulast%3DDong%26aufirst%3DA.%2BP.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DA%2520potent%252C%2520selective%252C%2520and%2520cell-active%2520inhibitor%2520of%2520human%2520type%2520I%2520protein%2520arginine%2520methyltransferases%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D772%26epage%3D781%26doi%3D10.1021%2Facschembio.5b00839" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drew, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribich, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioux, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swinger, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacques, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingaraj, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boriack-Sjodin, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigle, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moradei, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riera, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter-Scott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesworth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span> <span> </span><span class="NLM_article-title">Aryl pyrazoles as potent inhibitors of arginine methyltransferases: identification of the first PRMT6 tool compound</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">655</span>– <span class="NLM_lpage">659</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00071</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00071" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmtVWkurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=655-659&author=L.+H.+Mitchellauthor=A.+E.+Drewauthor=S.+A.+Ribichauthor=N.+Riouxauthor=K.+K.+Swingerauthor=S.+L.+Jacquesauthor=T.+Lingarajauthor=P.+A.+Boriack-Sjodinauthor=N.+J.+Watersauthor=T.+J.+Wigleauthor=O.+Moradeiauthor=L.+Jinauthor=T.+Rieraauthor=M.+Porter-Scottauthor=M.+P.+Moyerauthor=J.+J.+Smithauthor=R.+Chesworthauthor=R.+A.+Copeland&title=Aryl+pyrazoles+as+potent+inhibitors+of+arginine+methyltransferases%3A+identification+of+the+first+PRMT6+tool+compound&doi=10.1021%2Facsmedchemlett.5b00071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Aryl Pyrazoles as Potent Inhibitors of Arginine Methyltransferases: Identification of the First PRMT6 Tool Compound</span></div><div class="casAuthors">Mitchell, Lorna H.; Drew, Allison E.; Ribich, Scott A.; Rioux, Nathalie; Swinger, Kerren K.; Jacques, Suzanne L.; Lingaraj, Trupti; Boriack-Sjodin, P. Ann; Waters, Nigel J.; Wigle, Tim J.; Moradei, Oscar; Jin, Lei; Riera, Tom; Porter-Scott, Margaret; Moyer, Mikel P.; Smith, Jesse J.; Chesworth, Richard; Copeland, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">655-659</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel aryl pyrazole series of arginine methyltransferase inhibitors has been identified.  Synthesis of analogs within this series yielded the first potent, selective, small mol. PRMT6 inhibitor tool compd., EPZ020411.  PRMT6 overexpression has been reported in several cancer types suggesting that inhibition of PRMT6 activity may have therapeutic utility.  Identification of EPZ020411 provides the field with the first small mol. tool compd. for target validation studies.  EPZ020411 shows good bioavailability following s.c. dosing in rats making it a suitable tool for in vivo studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2OMyuVpnWSbVg90H21EOLACvtfcHk0li3JOZeu25HBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmtVWkurg%253D&md5=df8a4c190ea07b525d41f9dac519fa91</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00071%26sid%3Dliteratum%253Aachs%26aulast%3DMitchell%26aufirst%3DL.%2BH.%26aulast%3DDrew%26aufirst%3DA.%2BE.%26aulast%3DRibich%26aufirst%3DS.%2BA.%26aulast%3DRioux%26aufirst%3DN.%26aulast%3DSwinger%26aufirst%3DK.%2BK.%26aulast%3DJacques%26aufirst%3DS.%2BL.%26aulast%3DLingaraj%26aufirst%3DT.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DMoradei%26aufirst%3DO.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DRiera%26aufirst%3DT.%26aulast%3DPorter-Scott%26aufirst%3DM.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26atitle%3DAryl%2520pyrazoles%2520as%2520potent%2520inhibitors%2520of%2520arginine%2520methyltransferases%253A%2520identification%2520of%2520the%2520first%2520PRMT6%2520tool%2520compound%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D655%26epage%3D659%26doi%3D10.1021%2Facsmedchemlett.5b00071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. Ü.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Freitas, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senisterra, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a potent, selective, and cell-active dual inhibitor of protein arginine methyltransferase 4 and protein arginine methyltransferase 6</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">9124</span>– <span class="NLM_lpage">9139</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01033</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01033" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVKjs77O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=9124-9139&author=Y.+Shenauthor=M.+M.+Szewczykauthor=M.+S.+Eramauthor=D.+Smilauthor=H.+%C3%9C.+Kaniskanauthor=R.+F.+de+Freitasauthor=G.+Senisterraauthor=F.+Liauthor=M.+Schapiraauthor=P.+J.+Brownauthor=C.+H.+Arrowsmithauthor=D.+Barsyte-Lovejoyauthor=J.+Liuauthor=M.+Vedadiauthor=J.+Jin&title=Discovery+of+a+potent%2C+selective%2C+and+cell-active+dual+inhibitor+of+protein+arginine+methyltransferase+4+and+protein+arginine+methyltransferase+6&doi=10.1021%2Facs.jmedchem.6b01033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent, Selective, and Cell-Active Dual Inhibitor of Protein Arginine Methyltransferase 4 and Protein Arginine Methyltransferase 6</span></div><div class="casAuthors">Shen, Yudao; Szewczyk, Magdalena M.; Eram, Mohammad S.; Smil, David; Kaniskan, H. Umit; Ferreira de Freitas, Renato; Senisterra, Guillermo; Li, Fengling; Schapira, Matthieu; Brown, Peter J.; Arrowsmith, Cheryl H.; Barsyte-Lovejoy, Dalia; Liu, Jing; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">9124-9139</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Well-characterized selective inhibitors of protein arginine methyltransferases (PRMTs) are invaluable chem. tools for testing biol. and therapeutic hypotheses.  Based on the lead compd., a fragment-like inhibitor of type I PRMTs, the authors conducted structure-activity relationship (SAR) studies and explored three regions of this scaffold.  The studies led to the discovery of a potent, selective and cell-active dual inhibitor of PRMT4 and PRMT6, I (MS049).  As compared to the lead, I displayed much improved potency for PRMT4 and PRMT6 in both biochem. and cellular assays.  It was selective for PRMT4 and PRMT6 over other PRMTs and a broad range of other epigenetic modifiers and nonepigenetic targets.  The authors also developed II (MS049N), which was inactive in biochem. and cellular assays, as a neg. control for chem. biol. studies.  Considering possible overlapping substrate specificity of PRMTs, I and II are valuable chem. tools for dissecting specific biol. functions and dysregulation of PRMT4 and PRMT6 in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZENTB29LcELVg90H21EOLACvtfcHk0li3JOZeu25HBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVKjs77O&md5=2dcdad732107d0a64f7e317a05be1284</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01033%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3Dde%2BFreitas%26aufirst%3DR.%2BF.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%2520potent%252C%2520selective%252C%2520and%2520cell-active%2520dual%2520inhibitor%2520of%2520protein%2520arginine%2520methyltransferase%25204%2520and%2520protein%2520arginine%2520methyltransferase%25206%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D9124%26epage%3D9139%26doi%3D10.1021%2Facs.jmedchem.6b01033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samanta, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mondal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W. R.</span></span> <span> </span><span class="NLM_article-title">Covalent inhibition in drug discovery</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">889</span>– <span class="NLM_lpage">906</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201900107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.1002%2Fcmdc.201900107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=30816012" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlvFensb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=889-906&author=A.+K.+Ghoshauthor=I.+Samantaauthor=A.+Mondalauthor=W.+R.+Liu&title=Covalent+inhibition+in+drug+discovery&doi=10.1002%2Fcmdc.201900107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent Inhibition in Drug Discovery</span></div><div class="casAuthors">Ghosh, Avick Kumar; Samanta, Indranil; Mondal, Anushree; Liu, Wenshe Ray</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">889-906</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Although covalent inhibitors have been used as therapeutics for more than a century, there has been general resistance in the pharmaceutical industry against their further development due to safety concerns.  This inclination has recently been reverted after the development of a wide variety of covalent inhibitors to address human health conditions along with the US Food and Drug Administration (FDA) approval of several covalent therapeutics for use in humans.  Along with this exciting resurrection of an old drug discovery concept, this review surveys enzymes that can be targeted by covalent inhibitors for the treatment of human diseases.  We focus on protein kinases, RAS proteins, and a few other enzymes that have been studied extensively as targets for covalent inhibition, with the aim to address challenges in designing effective covalent drugs and to provide suggestions in the area that have yet to be explored.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1p5IlI637AbVg90H21EOLACvtfcHk0lj9puABo-z88w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlvFensb4%253D&md5=aaa9bd889193c02f8d9acdecaa45e64b</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201900107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201900107%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DA.%2BK.%26aulast%3DSamanta%26aufirst%3DI.%26aulast%3DMondal%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DW.%2BR.%26atitle%3DCovalent%2520inhibition%2520in%2520drug%2520discovery%26jtitle%3DChemMedChem%26date%3D2019%26volume%3D14%26spage%3D889%26epage%3D906%26doi%3D10.1002%2Fcmdc.201900107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Flanagan, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abramite, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aulabaugh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahal, U. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oppenheimer, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryder, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuff, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uccello, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayward, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noe, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obach, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philippe, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanmugasundaram, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Starr, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stroh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, Y.</span></span> <span> </span><span class="NLM_article-title">Chemical and computational methods for the characterization of covalent reactive groups for the prospective design of irreversible inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">10072</span>– <span class="NLM_lpage">10079</span>, <span class="refDoi"> DOI: 10.1021/jm501412a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501412a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVyjtLbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=10072-10079&author=M.+E.+Flanaganauthor=J.+A.+Abramiteauthor=D.+P.+Andersonauthor=A.+Aulabaughauthor=U.+P.+Dahalauthor=A.+M.+Gilbertauthor=C.+Liauthor=J.+Montgomeryauthor=S.+R.+Oppenheimerauthor=T.+Ryderauthor=B.+P.+Schuffauthor=D.+P.+Uccelloauthor=G.+S.+Walkerauthor=Y.+Wuauthor=M.+F.+Brownauthor=J.+M.+Chenauthor=M.+M.+Haywardauthor=M.+C.+Noeauthor=R.+S.+Obachauthor=L.+Philippeauthor=V.+Shanmugasundaramauthor=M.+J.+Shapiroauthor=J.+Starrauthor=J.+Strohauthor=Y.+Che&title=Chemical+and+computational+methods+for+the+characterization+of+covalent+reactive+groups+for+the+prospective+design+of+irreversible+inhibitors&doi=10.1021%2Fjm501412a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical and Computational Methods for the Characterization of Covalent Reactive Groups for the Prospective Design of Irreversible Inhibitors</span></div><div class="casAuthors">Flanagan, Mark E.; Abramite, Joseph A.; Anderson, Dennis P.; Aulabaugh, Ann; Dahal, Upendra P.; Gilbert, Adam M.; Li, Chao; Montgomery, Justin; Oppenheimer, Stacey R.; Ryder, Tim; Schuff, Brandon P.; Uccello, Daniel P.; Walker, Gregory S.; Wu, Yan; Brown, Matthew F.; Chen, Jinshan M.; Hayward, Matthew M.; Noe, Mark C.; Obach, R. Scott; Philippe, Laurence; Shanmugasundaram, Veerabahu; Shapiro, Michael J.; Starr, Jeremy; Stroh, Justin; Che, Ye</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10072-10079</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Interest in drugs that covalently modify their target is driven by the desire for enhanced efficacy that can result from the silencing of enzymic activity until protein resynthesis can occur, along with the potential for increased selectivity by targeting uniquely positioned nucleophilic residues in the protein.  However, covalent approaches carry addnl. risk for toxicities or hypersensitivity reactions that can result from covalent modification of unintended targets.  Here we describe methods for measuring the reactivity of covalent reactive groups (CRGs) with a biol. relevant nucleophile, glutathione (GSH), along with kinetic data for a broad array of electrophiles.  We also describe a computational method for predicting electrophilic reactivity, which taken together can be applied to the prospective design of thiol-reactive covalent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAzTSCv_Nj6bVg90H21EOLACvtfcHk0lj9puABo-z88w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVyjtLbM&md5=3ccf59ab7a494655185e5eb5becf8c48</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm501412a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501412a%26sid%3Dliteratum%253Aachs%26aulast%3DFlanagan%26aufirst%3DM.%2BE.%26aulast%3DAbramite%26aufirst%3DJ.%2BA.%26aulast%3DAnderson%26aufirst%3DD.%2BP.%26aulast%3DAulabaugh%26aufirst%3DA.%26aulast%3DDahal%26aufirst%3DU.%2BP.%26aulast%3DGilbert%26aufirst%3DA.%2BM.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DMontgomery%26aufirst%3DJ.%26aulast%3DOppenheimer%26aufirst%3DS.%2BR.%26aulast%3DRyder%26aufirst%3DT.%26aulast%3DSchuff%26aufirst%3DB.%2BP.%26aulast%3DUccello%26aufirst%3DD.%2BP.%26aulast%3DWalker%26aufirst%3DG.%2BS.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DBrown%26aufirst%3DM.%2BF.%26aulast%3DChen%26aufirst%3DJ.%2BM.%26aulast%3DHayward%26aufirst%3DM.%2BM.%26aulast%3DNoe%26aufirst%3DM.%2BC.%26aulast%3DObach%26aufirst%3DR.%2BS.%26aulast%3DPhilippe%26aufirst%3DL.%26aulast%3DShanmugasundaram%26aufirst%3DV.%26aulast%3DShapiro%26aufirst%3DM.%2BJ.%26aulast%3DStarr%26aufirst%3DJ.%26aulast%3DStroh%26aufirst%3DJ.%26aulast%3DChe%26aufirst%3DY.%26atitle%3DChemical%2520and%2520computational%2520methods%2520for%2520the%2520characterization%2520of%2520covalent%2520reactive%2520groups%2520for%2520the%2520prospective%2520design%2520of%2520irreversible%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D10072%26epage%3D10079%26doi%3D10.1021%2Fjm501412a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobrovetsky, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korboukh, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Exploiting an allosteric binding site of PRMT3 yields potent and selective inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">2110</span>– <span class="NLM_lpage">2124</span>, <span class="refDoi"> DOI: 10.1021/jm3018332</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3018332" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtlWgt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=2110-2124&author=F.+Liuauthor=F.+Liauthor=A.+Maauthor=E.+Dobrovetskyauthor=A.+Dongauthor=C.+Gaoauthor=I.+Korboukhauthor=J.+Liuauthor=D.+Smilauthor=P.+J.+Brownauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=M.+Schapiraauthor=M.+Vedadiauthor=J.+Jin&title=Exploiting+an+allosteric+binding+site+of+PRMT3+yields+potent+and+selective+inhibitors&doi=10.1021%2Fjm3018332"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Exploiting an Allosteric Binding Site of PRMT3 Yields Potent and Selective Inhibitors</span></div><div class="casAuthors">Liu, Feng; Li, Fengling; Ma, Anqi; Dobrovetsky, Elena; Dong, Aiping; Gao, Cen; Korboukh, Ilia; Liu, Jing; Smil, David; Brown, Peter J.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Schapira, Matthieu; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2110-2124</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferases (PRMTs) play an important role in diverse biol. processes.  Among the nine known human PRMTs, PRMT3 has been implicated in ribosomal biosynthesis via asym. dimethylation of the 40S ribosomal protein S2 and in cancer via interaction with the DAL-1 tumor suppressor protein.  However, few selective inhibitors of PRMTs have been discovered.  We recently disclosed the first selective PRMT3 inhibitor, which occupies a novel allosteric binding site and is noncompetitive with both the peptide substrate and cofactor.  Here we report comprehensive structure-activity relationship studies of this series, which resulted in the discovery of multiple PRMT3 inhibitors with submicromolar potencies.  An X-ray crystal structure of compd. I in complex with PRMT3 confirmed that this inhibitor occupied the same allosteric binding site as our initial lead compd.  These studies provide the first exptl. evidence that potent and selective inhibitors can be created by exploiting the allosteric binding site of PRMT3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgc4z8jYHwWbVg90H21EOLACvtfcHk0lj9puABo-z88w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtlWgt74%253D&md5=d538b236cbe1326ed74d84ab12778e47</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjm3018332&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3018332%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DMa%26aufirst%3DA.%26aulast%3DDobrovetsky%26aufirst%3DE.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DKorboukh%26aufirst%3DI.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DExploiting%2520an%2520allosteric%2520binding%2520site%2520of%2520PRMT3%2520yields%2520potent%2520and%2520selective%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D2110%26epage%3D2124%26doi%3D10.1021%2Fjm3018332" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krippendorff, B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neuhaus, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lienau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reichel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huisinga, W.</span></span> <span> </span><span class="NLM_article-title">Mechanism-based inhibition: deriving K(I) and k(inact) directly from time-dependent IC(50) values</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">913</span>– <span class="NLM_lpage">923</span>, <span class="refDoi"> DOI: 10.1177/1087057109336751</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.1177%2F1087057109336751" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=19675314" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlSjtLjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=913-923&author=B.+F.+Krippendorffauthor=R.+Neuhausauthor=P.+Lienauauthor=A.+Reichelauthor=W.+Huisinga&title=Mechanism-based+inhibition%3A+deriving+K%28I%29+and+k%28inact%29+directly+from+time-dependent+IC%2850%29+values&doi=10.1177%2F1087057109336751"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism-based inhibition: deriving KI and kinact directly from time-dependent IC50 values</span></div><div class="casAuthors">Krippendorff, Ben-Fillippo; Neuhaus, Roland; Lienau, Philip; Reichel, Andreas; Huisinga, Wilhelm</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">913-923</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The potential of enzyme inhibition of a drug is frequently quantified in terms of IC50 values.  Although this is a suitable quantity for reversible inhibitors, concerns arise when dealing with irreversible or mechanism-based inhibitors (MBIs).  IC50 values of MBIs are time dependent, causing serious problems when aiming at ranking different compds. with respect to their inhibitory potential.  As a consequence, most studies and ranking schemes related to MBIs rely on the inhibition const. (KI) and the rate of enzyme inactivation (kinact) rather than on IC50 values.  In this article, the authors derive a novel relation between potentially time-dependent IC50 values and KI, kinact parameters for different types of inhibition.  This allows for direct estn. of KI and kinact values from time-dependent IC50 values, even without the need of addnl. preincubation expts.  The application of this approach is illustrated using a fluorimetric assay to access the drug-drug interaction potential assocd. with new chem. entities.  The approach can easily be implemented using std. software tools (e.g., XLfit) and may also be suitable for applications where mechanism-based inhibition is a desired mode of action (e.g., at particular pharmacol. drug targets).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSOIpe0vN8JLVg90H21EOLACvtfcHk0lhuIcpo7feEOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlSjtLjF&md5=93cb5f8644c7e1bdf872cf46756039cf</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1177%2F1087057109336751&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057109336751%26sid%3Dliteratum%253Aachs%26aulast%3DKrippendorff%26aufirst%3DB.%2BF.%26aulast%3DNeuhaus%26aufirst%3DR.%26aulast%3DLienau%26aufirst%3DP.%26aulast%3DReichel%26aufirst%3DA.%26aulast%3DHuisinga%26aufirst%3DW.%26atitle%3DMechanism-based%2520inhibition%253A%2520deriving%2520K%2528I%2529%2520and%2520k%2528inact%2529%2520directly%2520from%2520time-dependent%2520IC%252850%2529%2520values%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2009%26volume%3D14%26spage%3D913%26epage%3D923%26doi%3D10.1177%2F1087057109336751" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sayegh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span> <span> </span><span class="NLM_article-title">PRMT8, a new membrane-bound tissue-specific member of the protein arginine methyltransferase family</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>280</i></span>,  <span class="NLM_fpage">32890</span>– <span class="NLM_lpage">32896</span>, <span class="refDoi"> DOI: 10.1074/jbc.M506944200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.1074%2Fjbc.M506944200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=16051612" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVWiur3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2005&pages=32890-32896&author=J.+Leeauthor=J.+Sayeghauthor=J.+Danielauthor=S.+Clarkeauthor=M.+T.+Bedford&title=PRMT8%2C+a+new+membrane-bound+tissue-specific+member+of+the+protein+arginine+methyltransferase+family&doi=10.1074%2Fjbc.M506944200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">PRMT8, a New Membrane-bound Tissue-specific Member of the Protein Arginine Methyltransferase Family</span></div><div class="casAuthors">Lee, Jaeho; Sayegh, Joyce; Daniel, Jeremy; Clarke, Steven; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">280</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">32890-32896</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Protein arginine methylation is a common post-translational modification that has been implicated in signal transduction, RNA processing, transcriptional regulation, and DNA repair.  A search of the human genome for addnl. members of the protein arginine N-methyltransferase (PRMT) family of enzymes has identified a gene on chromosome 12 that we have termed PRMT8.  This novel enzyme is most closely related to PRMT1, although it has a distinctive N-terminal region.  The unique N-terminal end harbors a myristoylation motif, and we have shown here that PRMT8 is indeed modified by the attachment of a myristate to the glycine residue after the initiator methionine.  The myristoylation of PRMT8 results in its assocn. with the plasma membrane.  The second singular property of PRMT8 is its tissue-specific expression pattern; it is largely expressed in the brain.  A glutathione S-transferase fusion protein of PRMT8 has type I PRMT activity, catalyzing the formation of ω-NG-monomethylated and asym. ω-NG,NG-dimethylated arginine residues on a recombinant glycine- and arginine-rich substrate.  PRMT8 is thus an active arginine methyltransferase that is membrane-assocd. and tissue-specific, two firsts for this family of enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorefolZCmgTrVg90H21EOLACvtfcHk0lhuIcpo7feEOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVWiur3E&md5=2a61ea03371c7bb01f0e79f25fc37099</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M506944200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M506944200%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DSayegh%26aufirst%3DJ.%26aulast%3DDaniel%26aufirst%3DJ.%26aulast%3DClarke%26aufirst%3DS.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DPRMT8%252C%2520a%2520new%2520membrane-bound%2520tissue-specific%2520member%2520of%2520the%2520protein%2520arginine%2520methyltransferase%2520family%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2005%26volume%3D280%26spage%3D32890%26epage%3D32896%26doi%3D10.1074%2Fjbc.M506944200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. Ü.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zang, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobrovetsky, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin-Jones, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atadja, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span> <span> </span><span class="NLM_article-title">A potent, selective and cell-active allosteric inhibitor of protein arginine methyltransferase 3 (PRMT3)</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">5166</span>– <span class="NLM_lpage">5170</span>, <span class="refDoi"> DOI: 10.1002/anie.201412154</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.1002%2Fanie.201412154" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BC2MXkvF2msLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=5166-5170&author=H.+%C3%9C.+Kaniskanauthor=M.+M.+Szewczykauthor=Z.+Yuauthor=M.+S.+Eramauthor=X.+Yangauthor=K.+Schmidtauthor=X.+Luoauthor=M.+Daiauthor=F.+Heauthor=I.+Zangauthor=Y.+Linauthor=S.+Kennedyauthor=F.+Liauthor=E.+Dobrovetskyauthor=A.+Dongauthor=D.+Smilauthor=S.+J.+Minauthor=M.+Landonauthor=J.+Lin-Jonesauthor=X.+P.+Huangauthor=B.+L.+Rothauthor=M.+Schapiraauthor=P.+Atadjaauthor=D.+Barsyte-Lovejoyauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=K.+Zhaoauthor=J.+Jinauthor=M.+Vedadi&title=A+potent%2C+selective+and+cell-active+allosteric+inhibitor+of+protein+arginine+methyltransferase+3+%28PRMT3%29&doi=10.1002%2Fanie.201412154"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">A Potent, Selective and Cell-Active Allosteric Inhibitor of Protein Arginine Methyltransferase 3 (PRMT3)</span></div><div class="casAuthors">Kaniskan, H. Uemit; Szewczyk, Magdalena M.; Yu, Zhengtian; Eram, Mohammad S.; Yang, Xiaobao; Schmidt, Keith; Luo, Xiao; Dai, Miao; He, Feng; Zang, Irene; Lin, Ying; Kennedy, Steven; Li, Fengling; Dobrovetsky, Elena; Dong, Aiping; Smil, David; Min, Sun-Joon; Landon, Melissa; Lin-Jones, Jennifer; Huang, Xi-Ping; Roth, Bryan L.; Schapira, Matthieu; Atadja, Peter; Barsyte-Lovejoy, Dalia; Arrowsmith, Cheryl H.; Brown, Peter J.; Zhao, Kehao; Jin, Jian; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">5166-5170</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">PRMT3 catalyzes the asym. dimethylation of arginine residues of various proteins.  It is essential for maturation of ribosomes, may have a role in lipogenesis, and is implicated in several diseases.  A potent, selective, and cell-active PRMT3 inhibitor would be a valuable tool for further investigating PRMT3 biol.  Here we report the discovery of the first PRMT3 chem. probe, SGC707, by structure-based optimization of the allosteric PRMT3 inhibitors we reported previously, and thorough characterization of this probe in biochem., biophys., and cellular assays.  SGC707 is a potent PRMT3 inhibitor (IC50=31±2 nM, KD=53±2 nM) with outstanding selectivity (selective against 31 other methyltransferases and more than 250 non-epigenetic targets).  The mechanism of action studies and crystal structure of the PRMT3-SGC707 complex confirm the allosteric inhibition mode.  Importantly, SGC707 engages PRMT3 and potently inhibits its methyltransferase activity in cells.  It is also bioavailable and suitable for animal studies.  This well-characterized chem. probe is an excellent tool to further study the role of PRMT3 in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTk4dFfkDLm7Vg90H21EOLACvtfcHk0lhuIcpo7feEOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXkvF2msLo%253D&md5=283b0875813af0f7631235e8ba7811c2</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1002%2Fanie.201412154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201412154%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DYu%26aufirst%3DZ.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DSchmidt%26aufirst%3DK.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DDai%26aufirst%3DM.%26aulast%3DHe%26aufirst%3DF.%26aulast%3DZang%26aufirst%3DI.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DDobrovetsky%26aufirst%3DE.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DMin%26aufirst%3DS.%2BJ.%26aulast%3DLandon%26aufirst%3DM.%26aulast%3DLin-Jones%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DAtadja%26aufirst%3DP.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DZhao%26aufirst%3DK.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DA%2520potent%252C%2520selective%2520and%2520cell-active%2520allosteric%2520inhibitor%2520of%2520protein%2520arginine%2520methyltransferase%25203%2520%2528PRMT3%2529%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D54%26spage%3D5166%26epage%3D5170%26doi%3D10.1002%2Fanie.201412154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. Ü.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zang, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobrovetsky, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin-Jones, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atadja, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of potent and selective allosteric inhibitors of protein arginine methyltransferase 3 (PRMT3)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1204</span>– <span class="NLM_lpage">1217</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01674</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01674" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvF2gsbzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=1204-1217&author=H.+%C3%9C.+Kaniskanauthor=M.+S.+Eramauthor=K.+Zhaoauthor=M.+M.+Szewczykauthor=X.+Yangauthor=K.+Schmidtauthor=X.+Luoauthor=S.+Xiaoauthor=M.+Daiauthor=F.+Heauthor=I.+Zangauthor=Y.+Linauthor=F.+Liauthor=E.+Dobrovetskyauthor=D.+Smilauthor=S.+J.+Minauthor=J.+Lin-Jonesauthor=M.+Schapiraauthor=P.+Atadjaauthor=E.+Liauthor=D.+Barsyte-Lovejoyauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=F.+Liuauthor=Z.+Yuauthor=M.+Vedadiauthor=J.+Jin&title=Discovery+of+potent+and+selective+allosteric+inhibitors+of+protein+arginine+methyltransferase+3+%28PRMT3%29&doi=10.1021%2Facs.jmedchem.7b01674"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Potent and Selective Allosteric Inhibitors of Protein Arginine Methyltransferase 3 (PRMT3)</span></div><div class="casAuthors">Kaniskan, H. Umit; Eram, Mohammad S.; Zhao, Kehao; Szewczyk, Magdalena M.; Yang, Xiaobao; Schmidt, Keith; Luo, Xiao; Xiao, Sean; Dai, Miao; He, Feng; Zang, Irene; Lin, Ying; Li, Fengling; Dobrovetsky, Elena; Smil, David; Min, Sun-Joon; Lin-Jones, Jennifer; Schapira, Matthieu; Atadja, Peter; Li, En; Barsyte-Lovejoy, Dalia; Arrowsmith, Cheryl H.; Brown, Peter J.; Liu, Feng; Yu, Zhengtian; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1204-1217</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">PRMT3 catalyzes the asym. dimethylation of arginine residues of various proteins.  It is crucial for maturation of ribosomes, and has been implicated in several diseases.  We recently disclosed a highly potent, selective and cell-active allosteric inhibitor of PRMT3, compd. 4.  Here, we report comprehensive structure-activity relationship studies that target the allosteric binding site of PRMT3.  We conducted design, synthesis and evaluation of novel compds. in biochem., selectivity and cellular assays that culminated in the discovery of 4 and other highly potent (IC50 values: ∼10-36 nM), selective and cell-active allosteric inhibitors of PRMT3 (compds. 29, 30, 36 and 37).  In addn., we generated compds. that are very close analogs of these potent inhibitors, but displayed drastically reduced potency as neg. controls (compds. 49-51).  These inhibitors and neg. controls are valuable chem. tools for the biomedical community to further investigate biol. functions and disease assocns. of PRMT3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodgUnzNZOB2rVg90H21EOLACvtfcHk0ljE2gwVId1xDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvF2gsbzO&md5=a784663c26234c3a99386ea4fe3f8abc</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01674%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DZhao%26aufirst%3DK.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DSchmidt%26aufirst%3DK.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DXiao%26aufirst%3DS.%26aulast%3DDai%26aufirst%3DM.%26aulast%3DHe%26aufirst%3DF.%26aulast%3DZang%26aufirst%3DI.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DDobrovetsky%26aufirst%3DE.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DMin%26aufirst%3DS.%2BJ.%26aulast%3DLin-Jones%26aufirst%3DJ.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DAtadja%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DE.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DYu%26aufirst%3DZ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520potent%2520and%2520selective%2520allosteric%2520inhibitors%2520of%2520protein%2520arginine%2520methyltransferase%25203%2520%2528PRMT3%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D1204%26epage%3D1217%26doi%3D10.1021%2Facs.jmedchem.7b01674" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakayama, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dela Sena, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Freitas, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baba, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunitomo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cary, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tawada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohashi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imaeda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saikatendu, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimshaw, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiba, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomita, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span> <span> </span><span class="NLM_article-title">TP-064, a potent and selective small molecule inhibitor of PRMT4 for multiple myeloma</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">18480</span>– <span class="NLM_lpage">18493</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.24883</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.18632%2Foncotarget.24883" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=29719619" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A280%3ADC%252BC1MjptFWhtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=18480-18493&author=K.+Nakayamaauthor=M.+M.+Szewczykauthor=C.+Dela+Senaauthor=H.+Wuauthor=A.+Dongauthor=H.+Zengauthor=F.+Liauthor=R.+F.+de+Freitasauthor=M.+S.+Eramauthor=M.+Schapiraauthor=Y.+Babaauthor=M.+Kunitomoauthor=D.+R.+Caryauthor=M.+Tawadaauthor=A.+Ohashiauthor=Y.+Imaedaauthor=K.+S.+Saikatenduauthor=C.+E.+Grimshawauthor=M.+Vedadiauthor=C.+H.+Arrowsmithauthor=D.+Barsyte-Lovejoyauthor=A.+Kibaauthor=D.+Tomitaauthor=P.+J.+Brown&title=TP-064%2C+a+potent+and+selective+small+molecule+inhibitor+of+PRMT4+for+multiple+myeloma&doi=10.18632%2Foncotarget.24883"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">TP-064, a potent and selective small molecule inhibitor of PRMT4 for multiple myeloma</span></div><div class="casAuthors">Nakayama Kazuhide; Baba Yuji; Kunitomo Mihoko; Cary Douglas R; Ohashi Akihiro; Imaeda Yasuhiro; Kiba Atsushi; Tomita Daisuke; Szewczyk Magdalena M; Dela Sena Carlo; Wu Hong; Dong Aiping; Zeng Hong; Li Fengling; de Freitas Renato Ferreira; Eram Mohammad S; Schapira Matthieu; Vedadi Masoud; Arrowsmith Cheryl H; Barsyte-Lovejoy Dalia; Brown Peter J; Schapira Matthieu; Vedadi Masoud; Tawada Michiko; Saikatendu Kumar Singh; Grimshaw Charles E; Arrowsmith Cheryl H</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">18480-18493</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Protein arginine methyltransferase (PRMT) 4 (also known as coactivator-associated arginine methyltransferase 1; CARM1) is involved in a variety of biological processes and is considered as a candidate oncogene owing to its overexpression in several types of cancer.  Selective PRMT4 inhibitors are useful tools for clarifying the molecular events regulated by PRMT4 and for validating PRMT4 as a therapeutic target.  Here, we report the discovery of TP-064, a potent, selective, and cell-active chemical probe of human PRMT4 and its co-crystal structure with PRMT4.  TP-064 inhibited the methyltransferase activity of PRMT4 with high potency (half-maximal inhibitory concentration, IC50 < 10 nM) and selectivity over other PRMT family proteins, and reduced arginine dimethylation of the PRMT4 substrates BRG1-associated factor 155 (BAF155; IC50= 340 ± 30 nM) and Mediator complex subunit 12 (MED12; IC50 = 43 ± 10 nM).  TP-064 treatment inhibited the proliferation of a subset of multiple myeloma cell lines, with affected cells arrested in G1 phase of the cell cycle.  TP-064 and its negative control (TP-064N) will be valuable tools to further investigate the biology of PRMT4 and the therapeutic potential of PRMT4 inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTsNG0-q3aOlw20zIbnCCI3fW6udTcc2eZ85sFYjYUbnbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MjptFWhtA%253D%253D&md5=f910a11dc3aa023564499ba2e6209e31</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.24883&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.24883%26sid%3Dliteratum%253Aachs%26aulast%3DNakayama%26aufirst%3DK.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DDela%2BSena%26aufirst%3DC.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DF.%26aulast%3Dde%2BFreitas%26aufirst%3DR.%2BF.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DBaba%26aufirst%3DY.%26aulast%3DKunitomo%26aufirst%3DM.%26aulast%3DCary%26aufirst%3DD.%2BR.%26aulast%3DTawada%26aufirst%3DM.%26aulast%3DOhashi%26aufirst%3DA.%26aulast%3DImaeda%26aufirst%3DY.%26aulast%3DSaikatendu%26aufirst%3DK.%2BS.%26aulast%3DGrimshaw%26aufirst%3DC.%2BE.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DKiba%26aufirst%3DA.%26aulast%3DTomita%26aufirst%3DD.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26atitle%3DTP-064%252C%2520a%2520potent%2520and%2520selective%2520small%2520molecule%2520inhibitor%2520of%2520PRMT4%2520for%2520multiple%2520myeloma%26jtitle%3DOncotarget%26date%3D2018%26volume%3D9%26spage%3D18480%26epage%3D18493%26doi%3D10.18632%2Foncotarget.24883" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira
de Freitas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steuber, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senisterra, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hitchcock, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moosmayer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmann, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Egner, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span> <span> </span><span class="NLM_article-title">Discovery of a potent class I protein arginine methyltransferase fragment inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1176</span>– <span class="NLM_lpage">1183</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01772</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01772" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVWls78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1176-1183&author=R.+Ferreira%0Ade+Freitasauthor=M.+S.+Eramauthor=M.+M.+Szewczykauthor=H.+Steuberauthor=D.+Smilauthor=H.+Wuauthor=F.+Liauthor=G.+Senisterraauthor=A.+Dongauthor=P.+J.+Brownauthor=M.+Hitchcockauthor=D.+Moosmayerauthor=C.+M.+Stegmannauthor=U.+Egnerauthor=C.+Arrowsmithauthor=D.+Barsyte-Lovejoyauthor=M.+Vedadiauthor=M.+Schapira&title=Discovery+of+a+potent+class+I+protein+arginine+methyltransferase+fragment+inhibitor&doi=10.1021%2Facs.jmedchem.5b01772"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent Class I Protein Arginine Methyltransferase Fragment Inhibitor</span></div><div class="casAuthors">Ferreira de Freitas, Renato; Eram, Mohammad S.; Szewczyk, Magdalena M.; Steuber, Holger; Smil, David; Wu, Hong; Li, Fengling; Senisterra, Guillermo; Dong, Aiping; Brown, Peter J.; Hitchcock, Marion; Moosmayer, Dieter; Stegmann, Christian M.; Egner, Ursula; Arrowsmith, Cheryl; Barsyte-Lovejoy, Dalia; Vedadi, Masoud; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1176-1183</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein methyltransferases (PMTs) are a promising target class in oncol. and other disease areas.  They are composed of SET domain methyltransferases and structurally unrelated Rossman-fold enzymes that include protein arginine methyltransferases (PRMTs).  In the absence of a well-defined medicinal chem. tool-kit focused on PMTs, most current inhibitors were identified by screening large and diverse libraries of leadlike mols.  So far, no successful fragment-based approach was reported against this target class.  Here, by deconstructing potent PRMT inhibitors, we find that chem. moieties occupying the substrate arginine-binding site can act as efficient fragment inhibitors.  Screening a fragment library against PRMT6 produced numerous hits, including a 300 nM inhibitor (ligand efficiency of 0.56) that decreased global histone 3 arginine 2 methylation in cells, and can serve as a warhead for the development of PRMT chem. probes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplB_HPKhzNcLVg90H21EOLACvtfcHk0li0XLt5EomnZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVWls78%253D&md5=03cbe3214b3a80f05032692037cbfd52</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01772&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01772%26sid%3Dliteratum%253Aachs%26aulast%3DFerreira%2Bde%2BFreitas%26aufirst%3DR.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DSteuber%26aufirst%3DH.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DHitchcock%26aufirst%3DM.%26aulast%3DMoosmayer%26aufirst%3DD.%26aulast%3DStegmann%26aufirst%3DC.%2BM.%26aulast%3DEgner%26aufirst%3DU.%26aulast%3DArrowsmith%26aufirst%3DC.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520a%2520potent%2520class%2520I%2520protein%2520arginine%2520methyltransferase%2520fragment%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1176%26epage%3D1183%26doi%3D10.1021%2Facs.jmedchem.5b01772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cymborowski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chruszcz, M.</span></span> <span> </span><span class="NLM_article-title">HKL-3000: the integration of data reduction and structure solution--from diffraction images to an initial model in minutes</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">859</span>– <span class="NLM_lpage">866</span>, <span class="refDoi"> DOI: 10.1107/S0907444906019949</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.1107%2FS0907444906019949" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=16855301" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BD28XntVWksL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=859-866&author=W.+Minorauthor=M.+Cymborowskiauthor=Z.+Otwinowskiauthor=M.+Chruszcz&title=HKL-3000%3A+the+integration+of+data+reduction+and+structure+solution%2D%2Dfrom+diffraction+images+to+an+initial+model+in+minutes&doi=10.1107%2FS0907444906019949"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">HKL-3000: the integration of data reduction and structure solution - from diffraction images to an initial model in minutes</span></div><div class="casAuthors">Minor, Wladek; Cymborowski, Marcin; Otwinowski, Zbyszek; Chruszcz, Maksymilian</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">D62</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">859-866</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A new approach that integrates data collection, data redn., phasing and model building significantly accelerates the process of structure detn. and on av. minimizes the no. of data sets and synchrotron time required for structure soln.  Initial testing of the HKL-3000 system (the beta version was named HKL-2000_ph) with more than 140 novel structure detns. has proven its high value for MAD/SAD expts.  The heuristics for choosing the best computational strategy at different data resoln. limits of phasing signal and crystal diffraction are being optimized.  The typical end result is an interpretable electron-d. map with a partially built structure and, in some cases, an almost complete refined model.  The current development is oriented towards very fast structure soln. in order to provide feedback during the diffraction expt.  Work is also proceeding towards improving the quality of phasing calcn. and model building.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0c0DYtjFP7rVg90H21EOLACvtfcHk0li0XLt5EomnZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XntVWksL8%253D&md5=2cd2fe288f954837abb4acae4c008a2f</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1107%2FS0907444906019949&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444906019949%26sid%3Dliteratum%253Aachs%26aulast%3DMinor%26aufirst%3DW.%26aulast%3DCymborowski%26aufirst%3DM.%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DChruszcz%26aufirst%3DM.%26atitle%3DHKL-3000%253A%2520the%2520integration%2520of%2520data%2520reduction%2520and%2520structure%2520solution--from%2520diffraction%2520images%2520to%2520an%2520initial%2520model%2520in%2520minutes%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2006%26volume%3D62%26spage%3D859%26epage%3D866%26doi%3D10.1107%2FS0907444906019949" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Steiner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebedev, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span> <span> </span><span class="NLM_article-title">Fisher’s information in maximum-likelihood macromolecular crystallographic refinement</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">2114</span>– <span class="NLM_lpage">2124</span>, <span class="refDoi"> DOI: 10.1107/S0907444903018675</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.1107%2FS0907444903018675" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=14646069" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BD3sXptlGit7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2003&pages=2114-2124&author=R.+A.+Steinerauthor=A.+A.+Lebedevauthor=G.+N.+Murshudov&title=Fisher%E2%80%99s+information+in+maximum-likelihood+macromolecular+crystallographic+refinement&doi=10.1107%2FS0907444903018675"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Fisher's information in maximum-likelihood macromolecular crystallographic refinement</span></div><div class="casAuthors">Steiner, Roberto A.; Lebedev, Andrey A.; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">D59</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2114-2124</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">Fisher's information is a statistical quantity related to max.-likelihood theory.  It is a matrix defined as the expected value of the squared gradient of minus the log-likelihood function.  This matrix is pos. semidefinite for any parameter value.  Fisher's information is used in the quasi-Newton scoring method of minimization to calc. the shift vectors of model parameters.  If the matrix is non-singular, the scoring-minimization step is always downhill.  In this article, it is shown how the scoring method can be applied to macromol. crystallog. refinement.  It is also shown how the computational costs involved in calcn. of the Fisher's matrix can be efficiently reduced.  Speed is achieved by assuming a continuous distribution of reciprocal-lattice points.  Matrix elements calcd. with this method agree very well with those calcd. anal.  The scoring algorithm has been implemented in the program REFMAC5 of the CCP4 suite.  The Fisher's matrix is used in its sparse approxn.  Tests indicate that the algorithm performs satisfactorily.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgph0YGiViZLVg90H21EOLACvtfcHk0lg4fGYoHza7IQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXptlGit7k%253D&md5=c8bd3bece9333f75ce2cffff26ac55e1</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1107%2FS0907444903018675&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444903018675%26sid%3Dliteratum%253Aachs%26aulast%3DSteiner%26aufirst%3DR.%2BA.%26aulast%3DLebedev%26aufirst%3DA.%2BA.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DFisher%25E2%2580%2599s%2520information%2520in%2520maximum-likelihood%2520macromolecular%2520crystallographic%2520refinement%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2003%26volume%3D59%26spage%3D2114%26epage%3D2124%26doi%3D10.1107%2FS0907444903018675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lg4fGYoHza7IQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Williams, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prisant, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Videau, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deis, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keedy, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hintze, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arendall, W. B.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snoeyink, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lovell, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span> <span> </span><span class="NLM_article-title">MolProbity: more and better reference data for improved all-atom structure validation</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">293</span>– <span class="NLM_lpage">315</span>, <span class="refDoi"> DOI: 10.1002/pro.3330</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.1002%2Fpro.3330" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=29067766" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVKgsrnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2018&pages=293-315&author=C.+J.+Williamsauthor=J.+J.+Headdauthor=N.+W.+Moriartyauthor=M.+G.+Prisantauthor=L.+L.+Videauauthor=L.+N.+Deisauthor=V.+Vermaauthor=D.+A.+Keedyauthor=B.+J.+Hintzeauthor=V.+B.+Chenauthor=S.+Jainauthor=S.+M.+Lewisauthor=W.+B.+Arendallauthor=J.+Snoeyinkauthor=P.+D.+Adamsauthor=S.+C.+Lovellauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+more+and+better+reference+data+for+improved+all-atom+structure+validation&doi=10.1002%2Fpro.3330"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: More and better reference data for improved all-atom structure validation</span></div><div class="casAuthors">Williams, Christopher J.; Headd, Jeffrey J.; Moriarty, Nigel W.; Prisant, Michael G.; Videau, Lizbeth L.; Deis, Lindsay N.; Verma, Vishal; Keedy, Daniel A.; Hintze, Bradley J.; Chen, Vincent B.; Jain, Swati; Lewis, Steven M.; Arendall, W. Bryan, III; Snoeyink, Jack; Adams, Paul D.; Lovell, Simon C.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">293-315</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">1469-896X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">This paper describes the current update on macromol. model validation services that are provided at the MolProbity website, emphasizing changes and addns. since the previous review in 2010.  There have been many infrastructure improvements, including rewrite of previous Java utilities to now use existing or newly written Python utilities in the open-source CCTBX portion of the Phenix software system.  This improves long-term maintainability and enhances the thorough integration of MolProbity-style validation within Phenix.  There is now a complete MolProbity mirror site at .  GitHub serves our open-source code, ref. datasets, and the resulting multi-dimensional distributions that define most validation criteria.  Coordinate output after Asn/Gln/His "flip" correction is now more idealized, since the post-refinement step has apparently often been skipped in the past.  Two distinct sets of heavy-atom-to-hydrogen distances and accompanying van der Waals radii have been researched and improved in accuracy, one for the electron-cloud-center positions suitable for X-ray crystallog. and one for nuclear positions.  New validations include messages at input about problem-causing format irregularities, updates of Ramachandran and rotamer criteria from the million quality-filtered residues in a new ref. dataset, the CaBLAM Cα-CO virtual-angle anal. of backbone and secondary structure for cryoEM or low-resoln. X-ray, and flagging of the very rare cis-nonProline and twisted peptides which have recently been greatly overused.  Due to wide application of MolProbity validation and corrections by the research community, in Phenix, and at the worldwide Protein Data Bank, newly deposited structures have continued to improve greatly as measured by MolProbity's unique all-atom clashscore.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQFbgiBJUVTbVg90H21EOLACvtfcHk0lg4fGYoHza7IQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVKgsrnO&md5=a8d8f315bc29e1ecdeb0f60a313baca6</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1002%2Fpro.3330&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.3330%26sid%3Dliteratum%253Aachs%26aulast%3DWilliams%26aufirst%3DC.%2BJ.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DPrisant%26aufirst%3DM.%2BG.%26aulast%3DVideau%26aufirst%3DL.%2BL.%26aulast%3DDeis%26aufirst%3DL.%2BN.%26aulast%3DVerma%26aufirst%3DV.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DHintze%26aufirst%3DB.%2BJ.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DJain%26aufirst%3DS.%26aulast%3DLewis%26aufirst%3DS.%2BM.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DSnoeyink%26aufirst%3DJ.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DLovell%26aufirst%3DS.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520more%2520and%2520better%2520reference%2520data%2520for%2520improved%2520all-atom%2520structure%2520validation%26jtitle%3DProtein%2520Sci.%26date%3D2018%26volume%3D27%26spage%3D293%26epage%3D315%26doi%3D10.1002%2Fpro.3330" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scheer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ackloo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medina, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Northrop, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. Ü.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLeod, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zepeda-Velazquez, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Carvalho, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaph, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span> <span> </span><span class="NLM_article-title">A chemical biology toolbox to study protein methyltransferases and epigenetic signaling</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>, <span class="NLM_elocation-id">19</span> <span class="refDoi"> DOI: 10.1038/s41467-018-07905-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.1038%2Fs41467-018-07905-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=30604761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmvVCmtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&author=S.+Scheerauthor=S.+Acklooauthor=T.+S.+Medinaauthor=M.+Schapiraauthor=F.+Liauthor=J.+A.+Wardauthor=A.+M.+Lewisauthor=J.+P.+Northropauthor=P.+L.+Richardsonauthor=H.+%C3%9C.+Kaniskanauthor=Y.+Shenauthor=J.+Liuauthor=D.+Smilauthor=D.+McLeodauthor=C.+A.+Zepeda-Velazquezauthor=M.+Luoauthor=J.+Jinauthor=D.+Barsyte-Lovejoyauthor=K.+V.+M.+Huberauthor=D.+D.+De+Carvalhoauthor=M.+Vedadiauthor=C.+Zaphauthor=P.+J.+Brownauthor=C.+H.+Arrowsmith&title=A+chemical+biology+toolbox+to+study+protein+methyltransferases+and+epigenetic+signaling&doi=10.1038%2Fs41467-018-07905-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">A chemical biology toolbox to study protein methyltransferases and epigenetic signaling</span></div><div class="casAuthors">Scheer, Sebastian; Ackloo, Suzanne; Medina, Tiago S.; Schapira, Matthieu; Li, Fengling; Ward, Jennifer A.; Lewis, Andrew M.; Northrop, Jeffrey P.; Richardson, Paul L.; Kaniskan, H. Umit; Shen, Yudao; Liu, Jing; Smil, David; McLeod, David; Zepeda-Velazquez, Carlos A.; Luo, Minkui; Jin, Jian; Barsyte-Lovejoy, Dalia; Huber, Kilian V. M.; De Carvalho, Daniel D.; Vedadi, Masoud; Zaph, Colby; Brown, Peter J.; Arrowsmith, Cheryl H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">19</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Protein methyltransferases (PMTs) comprise a major class of epigenetic regulatory enzymes with therapeutic relevance.  Here we present a collection of chem. probes and assocd. reagents and data to elucidate the function of human and murine PMTs in cellular studies.  Our collection provides inhibitors and antagonists that together modulate most of the key regulatory methylation marks on histones H3 and H4, providing an important resource for modulating cellular epigenomes.  We describe a comprehensive and comparative characterization of the probe collection with respect to their potency, selectivity, and mode of inhibition.  We demonstrate the utility of this collection in CD4+ T cell differentiation assays revealing the potential of individual probes to alter multiple T cell subpopulations which may have implications for T cell-mediated processes such as inflammation and immuno-oncol.  In particular, we demonstrate a role for DOT1L in limiting Th1 cell differentiation and maintaining lineage integrity.  This chem. probe collection and assocd. data form a resource for the study of methylation-mediated signaling in epigenetics, inflammation and beyond.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopA5rgeiWDvrVg90H21EOLACvtfcHk0lgghvMsQGjYNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmvVCmtro%253D&md5=008fec084426171f5c91657e54b73900</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fs41467-018-07905-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-018-07905-4%26sid%3Dliteratum%253Aachs%26aulast%3DScheer%26aufirst%3DS.%26aulast%3DAckloo%26aufirst%3DS.%26aulast%3DMedina%26aufirst%3DT.%2BS.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DWard%26aufirst%3DJ.%2BA.%26aulast%3DLewis%26aufirst%3DA.%2BM.%26aulast%3DNorthrop%26aufirst%3DJ.%2BP.%26aulast%3DRichardson%26aufirst%3DP.%2BL.%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DMcLeod%26aufirst%3DD.%26aulast%3DZepeda-Velazquez%26aufirst%3DC.%2BA.%26aulast%3DLuo%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DDe%2BCarvalho%26aufirst%3DD.%2BD.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DZaph%26aufirst%3DC.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26atitle%3DA%2520chemical%2520biology%2520toolbox%2520to%2520study%2520protein%2520methyltransferases%2520and%2520epigenetic%2520signaling%26jtitle%3DNat.%2520Commun.%26date%3D2019%26volume%3D10%26doi%3D10.1038%2Fs41467-018-07905-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 8 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Yi Zhang, Deqin Rong, Bingbing Li, <span class="NLM_string-name hlFld-ContribAuthor">Yuanxiang Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Epigenetic Regulators with Covalent Small-Molecule Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (12)
                                     , 7900-7925. <a href="https://doi.org/10.1021/acs.jmedchem.0c02055" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02055</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02055%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252BEpigenetic%252BRegulators%252Bwith%252BCovalent%252BSmall-Molecule%252BInhibitors%26aulast%3DZhang%26aufirst%3DYi%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D26112020%26date%3D18022021%26volume%3D64%26issue%3D12%26spage%3D7900%26epage%3D7925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yudao Shen, Fengling Li, Magdalena M. Szewczyk, Levon Halabelian, Irene Chau, Mohammad S. Eram, Carlo Dela Seña, Kwang-Su Park, Fanye Meng, He Chen, Hong Zeng, Aiping Dong, Hong Wu, Viacheslav V. Trush, David McLeod, Carlos A. Zepeda-Velázquez, Robert M. Campbell, Mary M. Mader, Brian M. Watson, Matthieu Schapira, Cheryl H. Arrowsmith, Rima Al-Awar, Dalia Barsyte-Lovejoy, H. Ümit Kaniskan, Peter J. Brown, Masoud Vedadi, <span class="NLM_string-name hlFld-ContribAuthor">Jian Jin</span>. </span><span class="cited-content_cbyCitation_article-title">A First-in-Class, Highly Selective and Cell-Active Allosteric Inhibitor of Protein Arginine Methyltransferase 6. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (7)
                                     , 3697-3706. <a href="https://doi.org/10.1021/acs.jmedchem.0c02160" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02160</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02160&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02160%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DA%252BFirst-in-Class%25252C%252BHighly%252BSelective%252Band%252BCell-Active%252BAllosteric%252BInhibitor%252Bof%252BProtein%252BArginine%252BMethyltransferase%252B6%26aulast%3DShen%26aufirst%3DYudao%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D14122020%26date%3D16022021%26volume%3D64%26issue%3D7%26spage%3D3697%26epage%3D3706" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mamta  Verma</span>, <span class="hlFld-ContribAuthor ">Mohd. Imran K.  Khan</span>, <span class="hlFld-ContribAuthor ">Rajashekar Varma  Kadumuri</span>, <span class="hlFld-ContribAuthor ">Baskar  Chakrapani</span>, <span class="hlFld-ContribAuthor ">Sharad  Awasthi</span>, <span class="hlFld-ContribAuthor ">Arun  Mahesh</span>, <span class="hlFld-ContribAuthor ">Gayathri  Govindaraju</span>, <span class="hlFld-ContribAuthor ">Pavithra L  Chavali</span>, <span class="hlFld-ContribAuthor ">Arumugam  Rajavelu</span>, <span class="hlFld-ContribAuthor ">Sreenivas  Chavali</span>, <span class="hlFld-ContribAuthor ">Arunkumar  Dhayalan</span>. </span><span class="cited-content_cbyCitation_article-title">PRMT3 interacts with ALDH1A1 and regulates gene-expression by inhibiting retinoic acid signaling. </span><span class="cited-content_cbyCitation_journal-name">Communications Biology</span><span> <strong>2021,</strong> <em>4 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s42003-020-01644-3" title="DOI URL">https://doi.org/10.1038/s42003-020-01644-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s42003-020-01644-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs42003-020-01644-3%26sid%3Dliteratum%253Aachs%26jtitle%3DCommunications%2520Biology%26atitle%3DPRMT3%252Binteracts%252Bwith%252BALDH1A1%252Band%252Bregulates%252Bgene-expression%252Bby%252Binhibiting%252Bretinoic%252Bacid%252Bsignaling%26aulast%3DVerma%26aufirst%3DMamta%26date%3D2021%26date%3D2021%26volume%3D4%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qin  Wu</span>, <span class="hlFld-ContribAuthor ">Matthieu  Schapira</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>, <span class="hlFld-ContribAuthor ">Dalia  Barsyte-Lovejoy</span>. </span><span class="cited-content_cbyCitation_article-title">Protein arginine methylation: from enigmatic functions to therapeutic targeting. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Drug Discovery</span><span> <strong>2021,</strong> <em>20 </em>
                                    (7)
                                     , 509-530. <a href="https://doi.org/10.1038/s41573-021-00159-8" title="DOI URL">https://doi.org/10.1038/s41573-021-00159-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41573-021-00159-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41573-021-00159-8%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Drug%2520Discovery%26atitle%3DProtein%252Barginine%252Bmethylation%25253A%252Bfrom%252Benigmatic%252Bfunctions%252Bto%252Btherapeutic%252Btargeting%26aulast%3DWu%26aufirst%3DQin%26date%3D2021%26date%3D2021%26volume%3D20%26issue%3D7%26spage%3D509%26epage%3D530" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lucas  Schneider</span>, <span class="hlFld-ContribAuthor ">Stefanie  Herkt</span>, <span class="hlFld-ContribAuthor ">Lei  Wang</span>, <span class="hlFld-ContribAuthor ">Christine  Feld</span>, <span class="hlFld-ContribAuthor ">Josephine  Wesely</span>, <span class="hlFld-ContribAuthor ">Olga N.  Kuvardina</span>, <span class="hlFld-ContribAuthor ">Annekarin  Meyer</span>, <span class="hlFld-ContribAuthor ">Thomas  Oellerich</span>, <span class="hlFld-ContribAuthor ">Björn  Häupl</span>, <span class="hlFld-ContribAuthor ">Erhard  Seifried</span>, <span class="hlFld-ContribAuthor ">Halvard  Bonig</span>, <span class="hlFld-ContribAuthor ">Joern  Lausen</span>. </span><span class="cited-content_cbyCitation_article-title">PRMT6 activates cyclin D1 expression in conjunction with the transcription factor LEF1. </span><span class="cited-content_cbyCitation_journal-name">Oncogenesis</span><span> <strong>2021,</strong> <em>10 </em>
                                    (5)
                                     <a href="https://doi.org/10.1038/s41389-021-00332-z" title="DOI URL">https://doi.org/10.1038/s41389-021-00332-z</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41389-021-00332-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41389-021-00332-z%26sid%3Dliteratum%253Aachs%26jtitle%3DOncogenesis%26atitle%3DPRMT6%252Bactivates%252Bcyclin%252BD1%252Bexpression%252Bin%252Bconjunction%252Bwith%252Bthe%252Btranscription%252Bfactor%252BLEF1%26aulast%3DSchneider%26aufirst%3DLucas%26date%3D2021%26date%3D2021%26volume%3D10%26issue%3D5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jee Won  Hwang</span>, <span class="hlFld-ContribAuthor ">Yena  Cho</span>, <span class="hlFld-ContribAuthor ">Gyu-Un  Bae</span>, <span class="hlFld-ContribAuthor ">Su-Nam  Kim</span>, <span class="hlFld-ContribAuthor ">Yong Kee  Kim</span>. </span><span class="cited-content_cbyCitation_article-title">Protein arginine methyltransferases: promising targets for cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">Experimental & Molecular Medicine</span><span> <strong>2021,</strong> <em>53 </em>
                                    (5)
                                     , 788-808. <a href="https://doi.org/10.1038/s12276-021-00613-y" title="DOI URL">https://doi.org/10.1038/s12276-021-00613-y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s12276-021-00613-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs12276-021-00613-y%26sid%3Dliteratum%253Aachs%26jtitle%3DExperimental%2520%2526%2520Molecular%2520Medicine%26atitle%3DProtein%252Barginine%252Bmethyltransferases%25253A%252Bpromising%252Btargets%252Bfor%252Bcancer%252Btherapy%26aulast%3DHwang%26aufirst%3DJee%2BWon%26date%3D2021%26date%3D2021%26volume%3D53%26issue%3D5%26spage%3D788%26epage%3D808" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Joshua J.  Hamey</span>, <span class="hlFld-ContribAuthor ">Sinja  Rakow</span>, <span class="hlFld-ContribAuthor ">Caroline  Bouchard</span>, <span class="hlFld-ContribAuthor ">Johanna M.  Senst</span>, <span class="hlFld-ContribAuthor ">Peter  Kolb</span>, <span class="hlFld-ContribAuthor ">Uta‐Maria  Bauer</span>, <span class="hlFld-ContribAuthor ">Marc R.  Wilkins</span>, <span class="hlFld-ContribAuthor ">Gene  Hart‐Smith</span>. </span><span class="cited-content_cbyCitation_article-title">Systematic investigation of PRMT6 substrate recognition reveals broad specificity with a preference for an RG motif or basic and bulky residues. </span><span class="cited-content_cbyCitation_journal-name">The FEBS Journal</span><span> <strong>2021,</strong> <em>20 </em><a href="https://doi.org/10.1111/febs.15837" title="DOI URL">https://doi.org/10.1111/febs.15837</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/febs.15837&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Ffebs.15837%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520FEBS%2520Journal%26atitle%3DSystematic%252Binvestigation%252Bof%252BPRMT6%252Bsubstrate%252Brecognition%252Breveals%252Bbroad%252Bspecificity%252Bwith%252Ba%252Bpreference%252Bfor%252Ban%252BRG%252Bmotif%252Bor%252Bbasic%252Band%252Bbulky%252Bresidues%26aulast%3DHamey%26aufirst%3DJoshua%2BJ.%26date%3D2021%26date%3D2021%26volume%3D20" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shuai  Gong</span>, <span class="hlFld-ContribAuthor ">Shinji  Maegawa</span>, <span class="hlFld-ContribAuthor ">Yanwen  Yang</span>, <span class="hlFld-ContribAuthor ">Vidya  Gopalakrishnan</span>, <span class="hlFld-ContribAuthor ">Guangrong  Zheng</span>, <span class="hlFld-ContribAuthor ">Donghang  Cheng</span>. </span><span class="cited-content_cbyCitation_article-title">Licochalcone A is a natural selective inhibitor of arginine methyltransferase 6. </span><span class="cited-content_cbyCitation_journal-name">Biochemical Journal</span><span> <strong>2021,</strong> <em>478 </em>
                                    (2)
                                     , 389-406. <a href="https://doi.org/10.1042/BCJ20200411" title="DOI URL">https://doi.org/10.1042/BCJ20200411</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1042/BCJ20200411&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1042%2FBCJ20200411%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Journal%26atitle%3DLicochalcone%252BA%252Bis%252Ba%252Bnatural%252Bselective%252Binhibitor%252Bof%252Barginine%252Bmethyltransferase%252B6%26aulast%3DGong%26aufirst%3DShuai%26date%3D2021%26date%3D2021%26volume%3D478%26issue%3D2%26spage%3D389%26epage%3D406" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/medium/jm0c00406_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00406&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/medium/jm0c00406_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0001.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Design of the PRMT6 covalent inhibitor, compound <b>4</b>, and its two control compounds. (A) Crystal structure (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5E8R">5E8R</a>) of PRMT6 (green) in complex with MS023 (cyan) and SAH (gray) reveals that the distance between the Cys50 thiol group and the meta-position of the MS023 phenyl ring (shown by the orange dotted line) is 4.9 Å. (B) Chemical structures of MS023, compound <b>4</b> (PRMT6 covalent inhibitor), and two structurally highly similar controls, compounds <b>5</b> and <b>7</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00406&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/medium/jm0c00406_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0012.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthetic Route of Compounds <b>4</b>, <b>5</b>, and <b>7</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00406&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaH, dimethylformamide/tetrahydrofuran (DMF/THF); (b) Et<sub>3</sub>N, Boc<sub>2</sub>O, 4-dimethylaminopyridine (DMAP), dichloromethane (DCM), 37% over two steps. (c) DIBAL-H, THF, −78 °C; (d) Dess–Martin periodinane, DCM, 71%; (e) 2-(methylamino)ethan-1-ol, NaBH(OAc)<sub>3</sub>, DCM, 74%; (f) <i>tert</i>-butyl (2-(methylamino)ethyl)carbamate, NaBH(OAc)<sub>3</sub>, DCM, 88%; (g) Pd/C (10%), H<sub>2</sub>, MeOH; (h) acryloyl chloride, DCM; (i) trifluoroacetic acid (TFA)/DCM, 29–42%; (j) propionyl chloride, Et<sub>3</sub>N, DCM.</p></p></figure><figure data-id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/medium/jm0c00406_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Compounds <b>4</b> and <b>5</b>, but not compound <b>7</b>, potently inhibited PRMT6. Concentration–response curves of compounds <b>4</b> (A), <b>5</b> (B), and <b>7</b> (C) in inhibiting PRMT6 in a scintillation proximity assay. IC<sub>50</sub> determination experiments were performed in triplicate, and the values are presented as mean ± standard deviation (SD).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00406&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/medium/jm0c00406_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Kinetics of PRMT6 inhibition by compounds <b>4</b> and <b>5</b>. (A) Compound <b>4</b> inhibited the PRMT6 methyltransferase activity in a time-dependent manner. (B) Compound <b>5</b> inhibited PRMT6 in a time-independent manner. The indicated compound was preincubated with PRMT6 at different time points ranging from 1 to 30 min, and IC<sub>50</sub> values were determined using the SPA assay. The experiments were performed in triplicate. The IC<sub>50</sub> values are presented as mean ± SD.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00406&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/medium/jm0c00406_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Compound <b>4</b>, but not compounds <b>5</b> and <b>7</b>, covalently modified PRMT6 in an MS assay. Mass spectra of PRMT6 alone (A), PRMT6 preincubated with compound <b>4</b> (MW: 298) (B), PRMT6 preincubated with compound <b>5</b> (C), and PRMT6 preincubated with compound <b>7</b> (D).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00406&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/medium/jm0c00406_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) X-ray cocrystal structure of PRMT6 (green, chain d) conjugated with compound <b>4</b> (cyan) in complex with SAH (gray) (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6P7I">6P7I</a>). Compound <b>4</b> covalently bound to the PRMT6 Cys50 residue. Key H-bond interactions between compound <b>4</b> and PRMT6 are shown in orange dotted lines. (B) Superimposition of the PRMT6–MS023 (magenta) (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5E8R">5E8R</a>) and PRMT6–compound <b>4</b> cocrystal structures.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00406&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/medium/jm0c00406_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Selectivity assessment of compounds <b>4</b> (A), <b>5</b> (B), and <b>7</b> (C) against a variety of methyltransferases at 10 μM (blue) and 50 μM (gray).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00406&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/medium/jm0c00406_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Compound <b>4</b> inhibits PRMT6-dependent H3R2 asymmetric dimethylation in cells with the highest potency. HEK293T cells were transfected with Flag-tagged wild-type PRMT6 or PRMT6-V86K/D88A catalytically inactive mutant (*) and treated with indicated compounds for 20 h. Type I PRMT inhibitor MS023 was used as a positive control. (A) Western blot representation of the effect of compounds <b>4</b>, <b>5</b>, and <b>7</b> on PRMT6 activity. (B) Quantification of the effects of compounds <b>4</b>, <b>5</b>, and <b>7</b> on PRMT6 activity. The graph shows H3R2me2a fluorescence intensities normalized to intensities of H3. The dotted line indicates the methylation level after MS023 treatment, which represents the background signal. The results are presented as the mean ± SD of three replicates. (C) IC<sub>50</sub> determination of compound <b>4</b>. The graph represents the nonlinear fit of H3R2me2a fluorescence intensities normalized to intensities of H3. The individual data points are indicated.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00406&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/medium/jm0c00406_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Inhibition of PRMT1-dependent H4R3 asymmetric dimethylation in cells. (A) Compounds <b>4</b> and <b>5</b>, but not compound <b>7</b>, inhibited H4R3me2a. MCF7 cells were treated with compounds for 2 days. MS023 was used as a positive control. (B) Graph representing the nonlinear fit of H4R3me2a fluorescence intensities normalized to intensities of H4. The results are presented as the mean ± SD of three replicates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00406&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/medium/jm0c00406_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0011.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Inhibition of PRMT3-dependent H4R3 arginine asymmetric dimethylation in cells. Compounds <b>4</b>, <b>5</b>, and <b>7</b> did not inhibit the PRMT3 activity at concentrations up to 20 μM. HEK293T cells were co-transfected with GFP-tagged histone H4 peptide (1–50) and PRMT3 WT or catalytically inactive mutant (E338Q) and treated with compounds for 20 h. PRMT3 selective inhibitor SGC707 was used as a positive control. The WB results are representative of two independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00406&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/medium/jm0c00406_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0002.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Inhibition of PRMT4-dependent BAF155 asymmetric dimethylation in cells. Compounds <b>4</b>, <b>5</b>, and <b>7</b> did not inhibit PRMT4 activity at concentrations up to 20 μM. MCF7 cells were treated with compounds for 2 days. PRMT4 selective inhibitor TP064 was used as a positive control. The WB results are representative of two independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00406&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig11" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/medium/jm0c00406_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0003.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Compounds <b>4</b>, <b>5</b>, and <b>7</b> are not cytotoxic in MCF-7, PNT2, and HEK293T cells. MCF-7, PNT2, and HEK293T cells were treated with compound <b>4</b> (A), <b>5</b> (B), or <b>7</b> (C) at indicated concentrations for 3 days. Results are presented as mean ± SD from three replicate experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-10/acs.jmedchem.0c00406/20200520/images/large/jm0c00406_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00406&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i35">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_50404" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_50404" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 30 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frankel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yadav, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Branscombe, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span> <span> </span><span class="NLM_article-title">The novel human protein arginine N-methyltransferase PRMT6 is a nuclear enzyme displaying unique substrate specificity</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>277</i></span>,  <span class="NLM_fpage">3537</span>– <span class="NLM_lpage">3543</span>, <span class="refDoi"> DOI: 10.1074/jbc.M108786200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.1074%2Fjbc.M108786200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=11724789" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BD38XhtVaktL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2002&pages=3537-3543&author=A.+Frankelauthor=N.+Yadavauthor=J.+H.+Leeauthor=T.+L.+Branscombeauthor=S.+Clarkeauthor=M.+T.+Bedford&title=The+novel+human+protein+arginine+N-methyltransferase+PRMT6+is+a+nuclear+enzyme+displaying+unique+substrate+specificity&doi=10.1074%2Fjbc.M108786200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The novel human protein arginine N-methyltransferase PRMT6 is a nuclear enzyme displaying unique substrate specificity</span></div><div class="casAuthors">Frankel, Adam; Yadav, Neelu; Lee, Jaeho; Branscombe, Tina L.; Clarke, Steven; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">277</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">3537-3543</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Protein arginine methylation is a prevalent posttranslational modification in eukaryotic cells that has been implicated in signal transduction, the metab. of nascent pre-RNA, and the transcriptional activation processes.  In searching the human genome for protein arginine N-methyltransferase (PRMT) family members, a novel gene has been found on chromosome 1 that encodes for an apparent methyltransferase, PRMT6.  The polypeptide chain of PRMT6 is 41.9 kDa consisting of a catalytic core sequence common to other PRMT enzymes.  Expressed as a glutathione S-transferase fusion protein, PRMT6 demonstrates type I PRMT activity, capable of forming both ω-NG-monomethylarginine and asym. ω-NG,NG-dimethylarginine derivs. on the recombinant glycine- and arginine-rich substrate in a processive manner with a specific activity of 144 pmol Me groups transferred min-1 mg-1 enzyme.  A comparison of substrate specificity reveals that PRMT6 is functionally distinct from two previously characterized type I enzymes, PRMT1 and PRMT4.  In addn., PRMT6 displays automethylation activity; it is the first PRMT to do so.  This novel human PRMT, which resides solely in the nucleus when fused to the green fluorescent protein, joins a family of enzymes with diverse functions within cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqs_K0kYqP2NbVg90H21EOLACvtfcHk0li7The0xlWPWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XhtVaktL4%253D&md5=1faaf27f44d72742749bdee2998ec9c2</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M108786200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M108786200%26sid%3Dliteratum%253Aachs%26aulast%3DFrankel%26aufirst%3DA.%26aulast%3DYadav%26aufirst%3DN.%26aulast%3DLee%26aufirst%3DJ.%2BH.%26aulast%3DBranscombe%26aufirst%3DT.%2BL.%26aulast%3DClarke%26aufirst%3DS.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DThe%2520novel%2520human%2520protein%2520arginine%2520N-methyltransferase%2520PRMT6%2520is%2520a%2520nuclear%2520enzyme%2520displaying%2520unique%2520substrate%2520specificity%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2002%26volume%3D277%26spage%3D3537%26epage%3D3543%26doi%3D10.1074%2Fjbc.M108786200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, S. G.</span></span> <span> </span><span class="NLM_article-title">Protein arginine methylation in mammals: who, what, and why</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">13</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2008.12.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.1016%2Fj.molcel.2008.12.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=19150423" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFOitrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2009&pages=1-13&author=M.+T.+Bedfordauthor=S.+G.+Clarke&title=Protein+arginine+methylation+in+mammals%3A+who%2C+what%2C+and+why&doi=10.1016%2Fj.molcel.2008.12.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methylation in mammals: who, what, and why</span></div><div class="casAuthors">Bedford, Mark T.; Clarke, Steven G.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-13</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The covalent marking of proteins by Me group addn. to arginine residues can promote their recognition by binding partners or can modulate their biol. activity.  A small family of gene products that catalyze such methylation reactions in eukaryotes (PRMTs) works in conjunction with a changing cast of assocd. subunits to recognize distinct cellular substrates.  These reactions display many of the attributes of reversible covalent modifications such as protein phosphorylation or protein lysine methylation; however, it is unclear to what extent protein arginine demethylation occurs.  Physiol. roles for protein arginine methylation have been established in signal transduction, mRNA splicing, transcriptional control, DNA repair, and protein translocation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqN0aG9XUXV27Vg90H21EOLACvtfcHk0li7The0xlWPWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFOitrg%253D&md5=0c8f178d730d06c37a87b8bafd0255a1</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2008.12.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2008.12.013%26sid%3Dliteratum%253Aachs%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DClarke%26aufirst%3DS.%2BG.%26atitle%3DProtein%2520arginine%2520methylation%2520in%2520mammals%253A%2520who%252C%2520what%252C%2520and%2520why%26jtitle%3DMol.%2520Cell%26date%3D2009%26volume%3D33%26spage%3D1%26epage%3D13%26doi%3D10.1016%2Fj.molcel.2008.12.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. Ü.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martini, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of protein methyltransferases and dimethylases</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">989</span>– <span class="NLM_lpage">1068</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.6b00801</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.6b00801" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkslynsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2018&pages=989-1068&author=H.+%C3%9C.+Kaniskanauthor=M.+L.+Martiniauthor=J.+Jin&title=Inhibitors+of+protein+methyltransferases+and+dimethylases&doi=10.1021%2Facs.chemrev.6b00801"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of Protein Methyltransferases and Demethylases</span></div><div class="casAuthors">Kaniskan, H. Umit; Martini, Michael L.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">989-1068</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Post-translational modifications of histones by protein methyltransferases (PMTs) and histone demethylases (KDMs) play an important role in the regulation of gene expression and transcription and are implicated in cancer and many other diseases.  Many of these enzymes also target various nonhistone proteins impacting numerous crucial biol. pathways.  Given their key biol. functions and implications in human diseases, there has been a growing interest in assessing these enzymes as potential therapeutic targets.  Consequently, discovering and developing inhibitors of these enzymes has become a very active and fast-growing research area over the past decade.  In this review, the authors cover the discovery, characterization, and biol. application of inhibitors of PMTs and KDMs with emphasis on key advancements in the field.  The authors also discuss challenges, opportunities, and future directions in this emerging, exciting research field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJYjiYtw_n4rVg90H21EOLACvtfcHk0liMa2l4XxOe-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkslynsL8%253D&md5=1ed520c6b0c42d2aa8a7f66b0a2d4e40</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.6b00801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.6b00801%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3DMartini%26aufirst%3DM.%2BL.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DInhibitors%2520of%2520protein%2520methyltransferases%2520and%2520dimethylases%26jtitle%3DChem.%2520Rev.%26date%3D2018%26volume%3D118%26spage%3D989%26epage%3D1068%26doi%3D10.1021%2Facs.chemrev.6b00801" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span> <span> </span><span class="NLM_article-title">Protein arginine methyltransferases and cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">37</span>– <span class="NLM_lpage">50</span>, <span class="refDoi"> DOI: 10.1038/nrc3409</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.1038%2Fnrc3409" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=23235912" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVajur7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2013&pages=37-50&author=Y.+Yangauthor=M.+T.+Bedford&title=Protein+arginine+methyltransferases+and+cancer&doi=10.1038%2Fnrc3409"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methyltransferases and cancer</span></div><div class="casAuthors">Yang, Yanzhong; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">37-50</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  There are nine protein arginine methyltransferases (PRMTs) encoded in mammalian genomes, the protein products of which catalyze three types of arginine methylation - monomethylation and two types of dimethylation.  Protein arginine methylation is an abundant modification that has been implicated in signal transduction, gene transcription, DNA repair and mRNA splicing, among others.  Studies have only recently linked this modification to carcinogenesis and metastasis.  Sequencing studies have not generally found alterations to the PRMTs; however, overexpression of these enzymes is often assocd. with various cancers, which might make some of them viable targets for therapeutic strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnb4O4wtn5fLVg90H21EOLACvtfcHk0liMa2l4XxOe-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVajur7I&md5=9df3b58d92e7932e214e6de26a10fa46</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrc3409&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3409%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DY.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DProtein%2520arginine%2520methyltransferases%2520and%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2013%26volume%3D13%26spage%3D37%26epage%3D50%26doi%3D10.1038%2Fnrc3409" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wei, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mundade, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lange, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span> <span> </span><span class="NLM_article-title">Protein arginine methylation of non-histone proteins and its role in diseases</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">32</span>– <span class="NLM_lpage">41</span>, <span class="refDoi"> DOI: 10.4161/cc.27353</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.4161%2Fcc.27353" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=24296620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltFeruro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=32-41&author=H.+Weiauthor=R.+Mundadeauthor=K.+C.+Langeauthor=T.+Lu&title=Protein+arginine+methylation+of+non-histone+proteins+and+its+role+in+diseases&doi=10.4161%2Fcc.27353"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methylation of non-histone proteins and its role in diseases</span></div><div class="casAuthors">Wei, Han; Mundade, Rasika; Lange, Kevin C.; Lu, Tao</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">32-41</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1538-4101</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">A review.  Protein arginine methyltransferases (PRMTs) are a family of enzymes that can methylate arginine residues on histones and other proteins.  PRMTs play a crucial role in influencing various cellular functions, including cellular development and tumorigenesis.  Arginine methylation by PRMTs is found on both nuclear and cytoplasmic proteins.  Recently, there is increasing evidence regarding post-translational modifications of non-histone proteins by PRMTs, illustrating the previously unknown importance of PRMTs in the regulation of various cellular functions by post-translational modifications.  In this review, we present the recent developments in the regulation of non-histone proteins by PRMTs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIW3UyE1q0IbVg90H21EOLACvtfcHk0liMa2l4XxOe-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltFeruro%253D&md5=55f3ea91e0b85ccf07b66eed1c58e471</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.4161%2Fcc.27353&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.27353%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DH.%26aulast%3DMundade%26aufirst%3DR.%26aulast%3DLange%26aufirst%3DK.%2BC.%26aulast%3DLu%26aufirst%3DT.%26atitle%3DProtein%2520arginine%2520methylation%2520of%2520non-histone%2520proteins%2520and%2520its%2520role%2520in%2520diseases%26jtitle%3DCell%2520Cycle%26date%3D2014%26volume%3D13%26spage%3D32%26epage%3D41%26doi%3D10.4161%2Fcc.27353" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casadio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinato, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cesaroni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuchlautz, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luscher, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amati, B.</span></span> <span> </span><span class="NLM_article-title">Methylation of histone H3R2 by PRMT6 and H3K4 by an MLL complex are mutually exclusive</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>449</i></span>,  <span class="NLM_fpage">933</span>– <span class="NLM_lpage">937</span>, <span class="refDoi"> DOI: 10.1038/nature06166</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.1038%2Fnature06166" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=17898714" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFOjt7bK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=449&publication_year=2007&pages=933-937&author=E.+Guccioneauthor=C.+Bassiauthor=F.+Casadioauthor=F.+Martinatoauthor=M.+Cesaroniauthor=H.+Schuchlautzauthor=B.+Luscherauthor=B.+Amati&title=Methylation+of+histone+H3R2+by+PRMT6+and+H3K4+by+an+MLL+complex+are+mutually+exclusive&doi=10.1038%2Fnature06166"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Methylation of histone H3R2 by PRMT6 and H3K4 by an MLL complex are mutually exclusive</span></div><div class="casAuthors">Guccione, Ernesto; Bassi, Christian; Casadio, Fabio; Martinato, Francesca; Cesaroni, Matteo; Schuchlautz, Henning; Luescher, Bernhard; Amati, Bruno</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">449</span>
        (<span class="NLM_cas:issue">7164</span>),
    <span class="NLM_cas:pages">933-937</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Eukaryotic genomes are organized into active (euchromatic) and inactive (heterochromatic) chromatin domains.  Post-translational modifications of histones (or 'marks') are key in defining these functional states, particularly in promoter regions.  Mutual regulatory interactions between these marks-and the enzymes that catalyze them-contribute to the shaping of this epigenetic landscape, in a manner that remains to be fully elucidated.  We previously obsd. that asym. di-methylation of histone H3 arginine 2 (H3R2me2a) counter-correlates with di- and tri- methylation of H3 lysine 4 (H3K4me2, H3K4me3) on human promoters.  Here we show that the arginine methyltransferase PRMT6 catalyzes H3R2 di-methylation in vitro and controls global levels of H3R2me2a in vivo.  H3R2 methylation by PRMT6 was prevented by the presence of H3K4me3 on the H3 tail.  Conversely, the H3R2me2a mark prevented methylation of H3K4 as well as binding to the H3 tail by an ASH2/WDR5/MLL-family methyltransferase complex.  Chromatin immunopptn. showed that H3R2me2a was distributed within the body and at the 3' end of human genes, regardless of their transcriptional state, whereas it was selectively and locally depleted from active promoters, coincident with the presence of H3K4me3.  Hence, the mutual antagonism between H3R2 and H3K4 methylation, together with the assocn. of MLL-family complexes with the basal transcription machinery, may contribute to the localized patterns of H3K4 tri-methylation characteristic of transcriptionally poised or active promoters in mammalian genomes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWqE8pDpHmKLVg90H21EOLACvtfcHk0liMa2l4XxOe-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFOjt7bK&md5=f4cc9371491088af7b74b614c90d6bd4</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnature06166&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature06166%26sid%3Dliteratum%253Aachs%26aulast%3DGuccione%26aufirst%3DE.%26aulast%3DBassi%26aufirst%3DC.%26aulast%3DCasadio%26aufirst%3DF.%26aulast%3DMartinato%26aufirst%3DF.%26aulast%3DCesaroni%26aufirst%3DM.%26aulast%3DSchuchlautz%26aufirst%3DH.%26aulast%3DLuscher%26aufirst%3DB.%26aulast%3DAmati%26aufirst%3DB.%26atitle%3DMethylation%2520of%2520histone%2520H3R2%2520by%2520PRMT6%2520and%2520H3K4%2520by%2520an%2520MLL%2520complex%2520are%2520mutually%2520exclusive%26jtitle%3DNature%26date%3D2007%26volume%3D449%26spage%3D933%26epage%3D937%26doi%3D10.1038%2Fnature06166" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boulanger, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russell, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wainberg, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richard, S.</span></span> <span> </span><span class="NLM_article-title">Methylation of Tat by PRMT6 regulates human immunodeficiency virus type 1 gene expression</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">124</span>– <span class="NLM_lpage">131</span>, <span class="refDoi"> DOI: 10.1128/JVI.79.1.124-131.2005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.1128%2FJVI.79.1.124-131.2005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=15596808" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhslemsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2005&pages=124-131&author=M.+C.+Boulangerauthor=C.+Liangauthor=R.+S.+Russellauthor=R.+Linauthor=M.+T.+Bedfordauthor=M.+A.+Wainbergauthor=S.+Richard&title=Methylation+of+Tat+by+PRMT6+regulates+human+immunodeficiency+virus+type+1+gene+expression&doi=10.1128%2FJVI.79.1.124-131.2005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Methylation of Tat by PRMT6 regulates human immunodeficiency virus type 1 gene expression</span></div><div class="casAuthors">Boulanger, Marie-Chloe; Liang, Chen; Russell, Rodney S.; Lin, Rongtuan; Bedford, Mark T.; Wainberg, Mark A.; Richard, Stephane</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">124-131</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The human immunodeficiency virus (HIV) transactivator protein, Tat, stimulates transcription from the viral long terminal repeats via an arginine-rich transactivating domain.  Since arginines are often known to be methylated, authors investigated whether HIV type 1 (HIV-1) Tat was a substrate for known protein arginine methyltransferases (PRMTs).  Here authors identify Tat as a substrate for the arginine methyltransferase, PRMT6.  Tat is specifically assocd. with and methylated by PRMT6 within cells.  Overexpression of wild-type PRMT6, but not a methylase-inactive PRMT6 mutant, decreased Tat transactivation of an HIV-1 long terminal repeat luciferase reporter plasmid in a dose-dependent manner.  Knocking down PRMT6 consistently increased HIV-1 prodn. in HEK293T cells and also led to increased viral infectiousness as shown in multinuclear activation of a galactosidase indicator assays.  This study demonstrates that arginine methylation of Tat neg. regulates its transactivation activity and that PRMT6 acts as a restriction factor for HIV replication.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0JF5C5Agwq7Vg90H21EOLACvtfcHk0lgfv8mqx7vFTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhslemsg%253D%253D&md5=763ac9af247b3f81962ae4e19419493a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1128%2FJVI.79.1.124-131.2005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.79.1.124-131.2005%26sid%3Dliteratum%253Aachs%26aulast%3DBoulanger%26aufirst%3DM.%2BC.%26aulast%3DLiang%26aufirst%3DC.%26aulast%3DRussell%26aufirst%3DR.%2BS.%26aulast%3DLin%26aufirst%3DR.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DWainberg%26aufirst%3DM.%2BA.%26aulast%3DRichard%26aufirst%3DS.%26atitle%3DMethylation%2520of%2520Tat%2520by%2520PRMT6%2520regulates%2520human%2520immunodeficiency%2520virus%2520type%25201%2520gene%2520expression%26jtitle%3DJ.%2520Virol.%26date%3D2005%26volume%3D79%26spage%3D124%26epage%3D131%26doi%3D10.1128%2FJVI.79.1.124-131.2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sgarra, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tessari, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spolaore, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giancotti, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manfioletti, G.</span></span> <span> </span><span class="NLM_article-title">The AT-hook of the chromatin architectural transcription factor high mobility group A1a is arginine-methylated by protein arginine methyltransferase 6</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>281</i></span>,  <span class="NLM_fpage">3764</span>– <span class="NLM_lpage">3772</span>, <span class="refDoi"> DOI: 10.1074/jbc.M510231200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.1074%2Fjbc.M510231200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=16293633" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlKksLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=3764-3772&author=R.+Sgarraauthor=J.+Leeauthor=M.+A.+Tessariauthor=S.+Altamuraauthor=B.+Spolaoreauthor=V.+Giancottiauthor=M.+T.+Bedfordauthor=G.+Manfioletti&title=The+AT-hook+of+the+chromatin+architectural+transcription+factor+high+mobility+group+A1a+is+arginine-methylated+by+protein+arginine+methyltransferase+6&doi=10.1074%2Fjbc.M510231200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The AT-hook of the Chromatin Architectural Transcription Factor High Mobility Group A1a Is Arginine-methylated by Protein Arginine Methyltransferase 6</span></div><div class="casAuthors">Sgarra, Riccardo; Lee, Jaeho; Tessari, Michela A.; Altamura, Sandro; Spolaore, Barbara; Giancotti, Vincenzo; Bedford, Mark T.; Manfioletti, Guidalberto</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">281</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3764-3772</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The HMGA1a protein belongs to the high mobility group A (HMGA) family of architectural nuclear factors, a group of proteins that plays an important role in chromatin dynamics.  HMGA proteins are multifunctional factors that assoc. both with DNA and nuclear proteins that have been involved in several nuclear processes, such as transcriptional regulation, viral integration, DNA repair, RNA processing, and chromatin remodeling.  The activity of HMGA proteins is finely modulated by a variety of post-translational modifications.  Arginine methylation was recently demonstrated to occur on HMGA1a protein, and it correlates with the apoptotic process and neoplastic progression.  Methyltransferases responsible for these modifications are unknown.  Here we show that the protein arginine methyltransferase PRMT6 specifically methylates HMGA1a protein both in vitro and in vivo.  By mass spectrometry, the sites of methylation were unambiguously mapped to Arg57 and Arg59, two residues which are embedded in the second AT-hook, a region crit. for both protein-DNA and protein-protein interactions and whose modification may cause profound alterations in the HMGA network.  The in vivo assocn. of HMGA and PRMT6 place this yet functionally uncharacterized methyltransferase in the well established functional context of the chromatin structure organization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKv8D5yCRQ4rVg90H21EOLACvtfcHk0lgfv8mqx7vFTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlKksLs%253D&md5=803d5cd75cc725a9c381220bc067d330</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M510231200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M510231200%26sid%3Dliteratum%253Aachs%26aulast%3DSgarra%26aufirst%3DR.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DTessari%26aufirst%3DM.%2BA.%26aulast%3DAltamura%26aufirst%3DS.%26aulast%3DSpolaore%26aufirst%3DB.%26aulast%3DGiancotti%26aufirst%3DV.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DManfioletti%26aufirst%3DG.%26atitle%3DThe%2520AT-hook%2520of%2520the%2520chromatin%2520architectural%2520transcription%2520factor%2520high%2520mobility%2520group%2520A1a%2520is%2520arginine-methylated%2520by%2520protein%2520arginine%2520methyltransferase%25206%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D3764%26epage%3D3772%26doi%3D10.1074%2Fjbc.M510231200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">El-Andaloussi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valovka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toueille, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinacher, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Focke, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gehrig, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Covic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassa, P. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubscher, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hottiger, M. O.</span></span> <span> </span><span class="NLM_article-title">Arginine methylation regulates DNA polymerase beta</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">51</span>– <span class="NLM_lpage">62</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2006.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.1016%2Fj.molcel.2006.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=16600869" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BD28XjvFWiu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2006&pages=51-62&author=N.+El-Andaloussiauthor=T.+Valovkaauthor=M.+Toueilleauthor=R.+Steinacherauthor=F.+Fockeauthor=P.+Gehrigauthor=M.+Covicauthor=P.+O.+Hassaauthor=P.+Scharauthor=U.+Hubscherauthor=M.+O.+Hottiger&title=Arginine+methylation+regulates+DNA+polymerase+beta&doi=10.1016%2Fj.molcel.2006.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Arginine methylation regulates DNA polymerase β</span></div><div class="casAuthors">El-Andaloussi, Nazim; Valovka, Taras; Toueille, Magali; Steinacher, Roland; Focke, Frauke; Gehrig, Peter; Covic, Marcela; Hassa, Paul O.; Schar, Primo; Hubscher, Ulrich; Hottiger, Michael O.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">51-62</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Alterations in DNA repair lead to genomic instability and higher risk of cancer.  DNA base excision repair (BER) corrects damaged bases, apurinic sites, and single-strand DNA breaks.  Here, a regulatory mechanism for DNA polymerase β (Pol β) is described.  Pol β was found to form a complex with the protein arginine methyltransferase 6 (PRMT6) and was specifically methylated in vitro and in vivo.  Methylation of Pol β by PRMT6 strongly stimulated DNA polymerase activity by enhancing DNA binding and processivity, while single nucleotide insertion and dRP-lyase activity were not affected.  Two residues, R83 and R152, were identified in Pol β as the sites of methylation by PRMT6.  Genetic complementation of Pol β knockout cells with R83/152K mutant revealed the importance of these residues for the cellular resistance to DNA alkylating agent.  Based on our findings, we propose that PRMT6 plays a role as a regulator of BER.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonqui7wNvLvrVg90H21EOLACvtfcHk0lgfv8mqx7vFTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjvFWiu74%253D&md5=987419489fb0439acb13f3b78d2d5a1e</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2006.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2006.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DEl-Andaloussi%26aufirst%3DN.%26aulast%3DValovka%26aufirst%3DT.%26aulast%3DToueille%26aufirst%3DM.%26aulast%3DSteinacher%26aufirst%3DR.%26aulast%3DFocke%26aufirst%3DF.%26aulast%3DGehrig%26aufirst%3DP.%26aulast%3DCovic%26aufirst%3DM.%26aulast%3DHassa%26aufirst%3DP.%2BO.%26aulast%3DSchar%26aufirst%3DP.%26aulast%3DHubscher%26aufirst%3DU.%26aulast%3DHottiger%26aufirst%3DM.%2BO.%26atitle%3DArginine%2520methylation%2520regulates%2520DNA%2520polymerase%2520beta%26jtitle%3DMol.%2520Cell%26date%3D2006%26volume%3D22%26spage%3D51%26epage%3D62%26doi%3D10.1016%2Fj.molcel.2006.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vieira, F. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costa-Pinheiro, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramalho-Carvalho, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pereira, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menezes, F. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antunes, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carneiro, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliveira, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henrique, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeronimo, C.</span></span> <span> </span><span class="NLM_article-title">Deregulated expression of selected histone methylases and dimethylases in prostate carcinoma</span>. <i>Endocr. Relat. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">51</span>– <span class="NLM_lpage">61</span>, <span class="refDoi"> DOI: 10.1530/ERC-13-0375</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.1530%2FERC-13-0375" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=24200674" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtl2gsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=51-61&author=F.+Q.+Vieiraauthor=P.+Costa-Pinheiroauthor=J.+Ramalho-Carvalhoauthor=A.+Pereiraauthor=F.+D.+Menezesauthor=L.+Antunesauthor=I.+Carneiroauthor=J.+Oliveiraauthor=R.+Henriqueauthor=C.+Jeronimo&title=Deregulated+expression+of+selected+histone+methylases+and+dimethylases+in+prostate+carcinoma&doi=10.1530%2FERC-13-0375"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Deregulated expression of selected histone methylases and demethylases in prostate carcinoma</span></div><div class="casAuthors">Vieira, Filipa Quintela; Costa-Pinheiro, Pedro; Ramalho-Carvalho, Joao; Pereira, Andreia; Menezes, Francisco Duarte; Antunes, Luis; Carneiro, Isa; Oliveira, Jorge; Henrique, Rui; Jeronimo, Carmen</div><div class="citationInfo"><span class="NLM_cas:title">Endocrine-Related Cancer</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">51-61</span>CODEN:
                <span class="NLM_cas:coden">ERCAE9</span>;
        ISSN:<span class="NLM_cas:issn">1351-0088</span>.
    
            (<span class="NLM_cas:orgname">BioScientifica Ltd.</span>)
        </div><div class="casAbstract">Prostate cancer (PCa), a leading cause of cancer-related morbidity and mortality, arises through the acquisition of genetic and epigenetic alterations.  Deregulation of histone methyltransferases (HMTs) or demethylases (HDMs) has been assocd. with PCa development and progression.  However, the precise influence of altered HMTs or HDMs expression and resp. histone marks in PCa onset and progression remains largely unknown.  To clarify the role of HMTs and HDMs in prostate carcinogenesis, expression levels of 37 HMTs and 20 HDMs were assessed in normal prostate and PCa tissue samples by RT-qPCR.  SMYD3, SUV39H2, PRMT6, KDM5A, and KDM6A were upregulated, whereas KMT2A-E (MLL1-5) and KDM4B were downregulated in PCa, compared with normal prostate tissues.  Remarkably, PRMT6 was the histone modifier that best discriminated normal from tumorous tissue samples.  Interestingly, EZH2 and SMYD3 expression levels significantly correlated with less differentiated and more aggressive tumors.  Remarkably, SMYD3 expression levels were of independent prognostic value for the prediction of disease-specific survival of PCa patients with clin. localized disease submitted to radical prostatectomy.  We concluded that expression profiling of HMTs and HDMs, esp. SMYD3, might be of clin. usefulness for the assessment of PCa patients and assist in pre-therapeutic decision-making.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-hUmMxxi157Vg90H21EOLACvtfcHk0ljtnr8Qlpi6xQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtl2gsbs%253D&md5=bbd4eed8bf88222c179aa9a079f5d20b</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1530%2FERC-13-0375&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1530%252FERC-13-0375%26sid%3Dliteratum%253Aachs%26aulast%3DVieira%26aufirst%3DF.%2BQ.%26aulast%3DCosta-Pinheiro%26aufirst%3DP.%26aulast%3DRamalho-Carvalho%26aufirst%3DJ.%26aulast%3DPereira%26aufirst%3DA.%26aulast%3DMenezes%26aufirst%3DF.%2BD.%26aulast%3DAntunes%26aufirst%3DL.%26aulast%3DCarneiro%26aufirst%3DI.%26aulast%3DOliveira%26aufirst%3DJ.%26aulast%3DHenrique%26aufirst%3DR.%26aulast%3DJeronimo%26aufirst%3DC.%26atitle%3DDeregulated%2520expression%2520of%2520selected%2520histone%2520methylases%2520and%2520dimethylases%2520in%2520prostate%2520carcinoma%26jtitle%3DEndocr.%2520Relat.%2520Cancer%26date%3D2014%26volume%3D21%26spage%3D51%26epage%3D61%26doi%3D10.1530%2FERC-13-0375" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yoshimatsu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toyokawa, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayami, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unoki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsunoda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Field, H. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neal, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maehara, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponder, B. A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamamoto, R.</span></span> <span> </span><span class="NLM_article-title">Dysregulation of PRMT1 and PRMT6, type I arginine methyltransferases, is involved in various types of human cancers</span>. <i>Int. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">562</span>– <span class="NLM_lpage">573</span>, <span class="refDoi"> DOI: 10.1002/ijc.25366</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.1002%2Fijc.25366" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=20473859" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A280%3ADC%252BC3cbpslChtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2011&pages=562-573&author=M.+Yoshimatsuauthor=G.+Toyokawaauthor=S.+Hayamiauthor=M.+Unokiauthor=T.+Tsunodaauthor=H.+I.+Fieldauthor=J.+D.+Kellyauthor=D.+E.+Nealauthor=Y.+Maeharaauthor=B.+A.+J.+Ponderauthor=Y.+Nakamuraauthor=R.+Hamamoto&title=Dysregulation+of+PRMT1+and+PRMT6%2C+type+I+arginine+methyltransferases%2C+is+involved+in+various+types+of+human+cancers&doi=10.1002%2Fijc.25366"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Dysregulation of PRMT1 and PRMT6, Type I arginine methyltransferases, is involved in various types of human cancers</span></div><div class="casAuthors">Yoshimatsu Masanori; Toyokawa Gouji; Hayami Shinya; Unoki Motoko; Tsunoda Tatsuhiko; Field Helen I; Kelly John D; Neal David E; Maehara Yoshihiko; Ponder Bruce A J; Nakamura Yusuke; Hamamoto Ryuji</div><div class="citationInfo"><span class="NLM_cas:title">International journal of cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">562-73</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Protein arginine methylation is a novel post-translational modification regulating a diversity of cellular processes, including histone functions, but the roles of protein arginine methyltransferases (PRMTs) in human cancer are not well investigated.  To address this issue, we first examined expression levels of genes belonging to the PRMT family and found significantly higher expression of PRMT1 and PRMT6, both of which are Type I PRMTs, in cancer cells of various tissues than in non-neoplastic cells.  Abrogation of the expression of these genes with specific siRNAs significantly suppressed growth of bladder and lung cancer cells.  Expression profile analysis using the cells transfected with the siRNAs indicated that PRMT1 and PRMT6 interplay in multiple pathways, supporting regulatory roles in the cell cycle, RNA processing and also DNA replication that are fundamentally important for cancer cell proliferation.  Furthermore, we demonstrated that serum asymmetric dimethylarginine (ADMA) levels of a number of cancer cases are significantly higher than those of nontumor control cases.  In summary, our results suggest that dysregulation of PRMT1 and PRMT6 can be involved in human carcinogenesis and that these Type I arginine methyltransferases are good therapeutic targets for various types of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQAVHBzjMN_WjEwO064Z95rfW6udTcc2eYpHLsAF3t3Vbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cbpslChtw%253D%253D&md5=34889d958208a4d852b0833e21a045de</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1002%2Fijc.25366&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.25366%26sid%3Dliteratum%253Aachs%26aulast%3DYoshimatsu%26aufirst%3DM.%26aulast%3DToyokawa%26aufirst%3DG.%26aulast%3DHayami%26aufirst%3DS.%26aulast%3DUnoki%26aufirst%3DM.%26aulast%3DTsunoda%26aufirst%3DT.%26aulast%3DField%26aufirst%3DH.%2BI.%26aulast%3DKelly%26aufirst%3DJ.%2BD.%26aulast%3DNeal%26aufirst%3DD.%2BE.%26aulast%3DMaehara%26aufirst%3DY.%26aulast%3DPonder%26aufirst%3DB.%2BA.%2BJ.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DHamamoto%26aufirst%3DR.%26atitle%3DDysregulation%2520of%2520PRMT1%2520and%2520PRMT6%252C%2520type%2520I%2520arginine%2520methyltransferases%252C%2520is%2520involved%2520in%2520various%2520types%2520of%2520human%2520cancers%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D2011%26volume%3D128%26spage%3D562%26epage%3D573%26doi%3D10.1002%2Fijc.25366" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Limm, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oefner, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hellerbrand, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bosserhoff, A. K.</span></span> <span> </span><span class="NLM_article-title">Deregulation of protein methylation in melanoma</span>. <i>Eur. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">1305</span>– <span class="NLM_lpage">1313</span>, <span class="refDoi"> DOI: 10.1016/j.ejca.2012.11.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.1016%2Fj.ejca.2012.11.026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=23265702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvFWhsbnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2013&pages=1305-1313&author=K.+Limmauthor=C.+Ottauthor=S.+Wallnerauthor=D.+W.+Muellerauthor=P.+Oefnerauthor=C.+Hellerbrandauthor=A.+K.+Bosserhoff&title=Deregulation+of+protein+methylation+in+melanoma&doi=10.1016%2Fj.ejca.2012.11.026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Deregulation of protein methylation in melanoma</span></div><div class="casAuthors">Limm, Katharina; Ott, Corinna; Wallner, Susanne; Mueller, Daniel W.; Oefner, Peter; Hellerbrand, Claus; Bosserhoff, Anja-Katrin</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Cancer</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1305-1313</span>CODEN:
                <span class="NLM_cas:coden">EJCAEL</span>;
        ISSN:<span class="NLM_cas:issn">0959-8049</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Loss of methylthioadenosine phosphorylase (MTAP) expression and a concomitant accumulation of 5'-methyl-thioadenosine (MTA) characterize several tumor entities including malignant melanoma.  MTA affects cellular signalling, proliferation and migration not only of cancer but also surrounding cells including lymphocytes and stromal fibroblasts.  The mode of action of MTA is still not known.  Interestingly, MTA is a known potent inhibitor of protein arginine methyltransferases (PRMTs) and is used as a tool in studying activity and impact of PRMTs.  This study aimed at analyzing PRMTs in melanoma and the potential impact of MTA on tumorigenesis.Our findings demonstrate that expression of PRMT4/CARM1 and PRMT6 is deregulated in melanoma, whereas expression of the remaining PRMTs stays unchanged.  General PRMT activity and, consequently, sym. and asym. protein methylation are reduced significantly in melanoma cells and tissues.  This is due to a loss of MTAP expression and accumulation of MTA.  Redn. of protein methylation by MTA affects cell signalling and leads, for example, to an activation of extracellular signal-regulated kinase (ERK) activity.  The effects of endogeneous MTA on PRMTs as presented in this study can strongly support the migratory and invasive phenotype of melanoma cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqujaHA2MQGG7Vg90H21EOLACvtfcHk0ljZ5uR1stwYfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvFWhsbnF&md5=a29a9efe0acd75da35bb7af04bd26dc4</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.ejca.2012.11.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejca.2012.11.026%26sid%3Dliteratum%253Aachs%26aulast%3DLimm%26aufirst%3DK.%26aulast%3DOtt%26aufirst%3DC.%26aulast%3DWallner%26aufirst%3DS.%26aulast%3DMueller%26aufirst%3DD.%2BW.%26aulast%3DOefner%26aufirst%3DP.%26aulast%3DHellerbrand%26aufirst%3DC.%26aulast%3DBosserhoff%26aufirst%3DA.%2BK.%26atitle%3DDeregulation%2520of%2520protein%2520methylation%2520in%2520melanoma%26jtitle%3DEur.%2520J.%2520Cancer%26date%3D2013%26volume%3D49%26spage%3D1305%26epage%3D1313%26doi%3D10.1016%2Fj.ejca.2012.11.026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ying, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ying, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, S.</span></span> <span> </span><span class="NLM_article-title">Significant association of PRMT6 hypomethylation with colorectal cancer</span>. <i>J. Clin. Lab. Anal.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>32</i></span>, <span class="NLM_elocation-id">e22590</span> <span class="refDoi"> DOI: 10.1002/jcla.22590</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.1002%2Fjcla.22590" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=29927001" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2018&author=R.+Panauthor=H.+Yuauthor=J.+Daiauthor=C.+Zhouauthor=X.+Yingauthor=J.+Zhongauthor=J.+Zhaoauthor=Y.+Zhangauthor=B.+Wuauthor=Y.+Maoauthor=D.+Wuauthor=J.+Yingauthor=S.+Duan&title=Significant+association+of+PRMT6+hypomethylation+with+colorectal+cancer&doi=10.1002%2Fjcla.22590"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1002%2Fjcla.22590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcla.22590%26sid%3Dliteratum%253Aachs%26aulast%3DPan%26aufirst%3DR.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DDai%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DC.%26aulast%3DYing%26aufirst%3DX.%26aulast%3DZhong%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DB.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DD.%26aulast%3DYing%26aufirst%3DJ.%26aulast%3DDuan%26aufirst%3DS.%26atitle%3DSignificant%2520association%2520of%2520PRMT6%2520hypomethylation%2520with%2520colorectal%2520cancer%26jtitle%3DJ.%2520Clin.%2520Lab.%2520Anal.%26date%3D2018%26volume%3D32%26doi%3D10.1002%2Fjcla.22590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senisterra, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butler, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. Ü.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speed, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">dela Sena, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">A potent, selective, and cell-active inhibitor of human type I protein arginine methyltransferases</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">772</span>– <span class="NLM_lpage">781</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00839</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00839" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVyru7vE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=772-781&author=M.+S.+Eramauthor=Y.+D.+Shenauthor=M.+M.+Szewczykauthor=H.+Wuauthor=G.+Senisterraauthor=F.+L.+Liauthor=K.+V.+Butlerauthor=H.+%C3%9C.+Kaniskanauthor=B.+A.+Speedauthor=C.+dela+Senaauthor=A.+P.+Dongauthor=H.+Zengauthor=M.+Schapiraauthor=P.+J.+Brownauthor=C.+H.+Arrowsmithauthor=D.+Barsyte-Lovejoyauthor=J.+Liuauthor=M.+Vedadiauthor=J.+Jin&title=A+potent%2C+selective%2C+and+cell-active+inhibitor+of+human+type+I+protein+arginine+methyltransferases&doi=10.1021%2Facschembio.5b00839"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">A Potent, Selective, and Cell-Active Inhibitor of Human Type I Protein Arginine Methyltransferases</span></div><div class="casAuthors">Eram, Mohammad S.; Shen, Yudao; Szewczyk, Magdalena M.; Wu, Hong; Senisterra, Guillermo; Li, Fengling; Butler, Kyle V.; Kaniskan, H. Umit; Speed, Brandon A.; dela Sena, Carlo; Dong, Aiping; Zeng, Hong; Schapira, Matthieu; Brown, Peter J.; Arrowsmith, Cheryl H.; Barsyte-Lovejoy, Dalia; Liu, Jing; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">772-781</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferases (PRMTs) play a crucial role in a variety of biol. processes.  Overexpression of PRMTs has been implicated in various human diseases including cancer.  Consequently, selective small-mol. inhibitors of PRMTs have been pursued by both academia and the pharmaceutical industry as chem. tools for testing biol. and therapeutic hypotheses.  PRMTs are divided into three categories: type I PRMTs which catalyze mono- and asym. dimethylation of arginine residues, type II PRMTs which catalyze mono- and sym. dimethylation of arginine residues, and type III PRMT which catalyzes only monomethylation of arginine residues.  Here, we report the discovery of a potent, selective, and cell-active inhibitor of human type I PRMTs, MS023, and characterization of this inhibitor in a battery of biochem., biophys., and cellular assays.  MS023 displayed high potency for type I PRMTs including PRMT1, -3, -4, -6, and -8 but was completely inactive against type II and type III PRMTs, protein lysine methyltransferases and DNA methyltransferases.  A crystal structure of PRMT6 in complex with MS023 revealed that MS023 binds the substrate binding site.  MS023 potently decreased cellular levels of histone arginine asym. dimethylation.  It also reduced global levels of arginine asym. dimethylation and concurrently increased levels of arginine monomethylation and sym. dimethylation in cells.  We also developed MS094, a close analog of MS023, which was inactive in biochem. and cellular assays, as a neg. control for chem. biol. studies.  MS023 and MS094 are useful chem. tools for investigating the role of type I PRMTs in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_ysJCXRFiPbVg90H21EOLACvtfcHk0ljZ5uR1stwYfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVyru7vE&md5=d4e103b75c2ae19572b4738ca53a271d</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00839&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00839%26sid%3Dliteratum%253Aachs%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DShen%26aufirst%3DY.%2BD.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DF.%2BL.%26aulast%3DButler%26aufirst%3DK.%2BV.%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3DSpeed%26aufirst%3DB.%2BA.%26aulast%3Ddela%2BSena%26aufirst%3DC.%26aulast%3DDong%26aufirst%3DA.%2BP.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DA%2520potent%252C%2520selective%252C%2520and%2520cell-active%2520inhibitor%2520of%2520human%2520type%2520I%2520protein%2520arginine%2520methyltransferases%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D772%26epage%3D781%26doi%3D10.1021%2Facschembio.5b00839" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drew, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribich, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioux, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swinger, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacques, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingaraj, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boriack-Sjodin, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigle, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moradei, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riera, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter-Scott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesworth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span> <span> </span><span class="NLM_article-title">Aryl pyrazoles as potent inhibitors of arginine methyltransferases: identification of the first PRMT6 tool compound</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">655</span>– <span class="NLM_lpage">659</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00071</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00071" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmtVWkurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=655-659&author=L.+H.+Mitchellauthor=A.+E.+Drewauthor=S.+A.+Ribichauthor=N.+Riouxauthor=K.+K.+Swingerauthor=S.+L.+Jacquesauthor=T.+Lingarajauthor=P.+A.+Boriack-Sjodinauthor=N.+J.+Watersauthor=T.+J.+Wigleauthor=O.+Moradeiauthor=L.+Jinauthor=T.+Rieraauthor=M.+Porter-Scottauthor=M.+P.+Moyerauthor=J.+J.+Smithauthor=R.+Chesworthauthor=R.+A.+Copeland&title=Aryl+pyrazoles+as+potent+inhibitors+of+arginine+methyltransferases%3A+identification+of+the+first+PRMT6+tool+compound&doi=10.1021%2Facsmedchemlett.5b00071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Aryl Pyrazoles as Potent Inhibitors of Arginine Methyltransferases: Identification of the First PRMT6 Tool Compound</span></div><div class="casAuthors">Mitchell, Lorna H.; Drew, Allison E.; Ribich, Scott A.; Rioux, Nathalie; Swinger, Kerren K.; Jacques, Suzanne L.; Lingaraj, Trupti; Boriack-Sjodin, P. Ann; Waters, Nigel J.; Wigle, Tim J.; Moradei, Oscar; Jin, Lei; Riera, Tom; Porter-Scott, Margaret; Moyer, Mikel P.; Smith, Jesse J.; Chesworth, Richard; Copeland, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">655-659</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel aryl pyrazole series of arginine methyltransferase inhibitors has been identified.  Synthesis of analogs within this series yielded the first potent, selective, small mol. PRMT6 inhibitor tool compd., EPZ020411.  PRMT6 overexpression has been reported in several cancer types suggesting that inhibition of PRMT6 activity may have therapeutic utility.  Identification of EPZ020411 provides the field with the first small mol. tool compd. for target validation studies.  EPZ020411 shows good bioavailability following s.c. dosing in rats making it a suitable tool for in vivo studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2OMyuVpnWSbVg90H21EOLACvtfcHk0lhHWExHyfkirw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmtVWkurg%253D&md5=df8a4c190ea07b525d41f9dac519fa91</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00071%26sid%3Dliteratum%253Aachs%26aulast%3DMitchell%26aufirst%3DL.%2BH.%26aulast%3DDrew%26aufirst%3DA.%2BE.%26aulast%3DRibich%26aufirst%3DS.%2BA.%26aulast%3DRioux%26aufirst%3DN.%26aulast%3DSwinger%26aufirst%3DK.%2BK.%26aulast%3DJacques%26aufirst%3DS.%2BL.%26aulast%3DLingaraj%26aufirst%3DT.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DMoradei%26aufirst%3DO.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DRiera%26aufirst%3DT.%26aulast%3DPorter-Scott%26aufirst%3DM.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26atitle%3DAryl%2520pyrazoles%2520as%2520potent%2520inhibitors%2520of%2520arginine%2520methyltransferases%253A%2520identification%2520of%2520the%2520first%2520PRMT6%2520tool%2520compound%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D655%26epage%3D659%26doi%3D10.1021%2Facsmedchemlett.5b00071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. Ü.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Freitas, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senisterra, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a potent, selective, and cell-active dual inhibitor of protein arginine methyltransferase 4 and protein arginine methyltransferase 6</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">9124</span>– <span class="NLM_lpage">9139</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01033</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01033" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVKjs77O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=9124-9139&author=Y.+Shenauthor=M.+M.+Szewczykauthor=M.+S.+Eramauthor=D.+Smilauthor=H.+%C3%9C.+Kaniskanauthor=R.+F.+de+Freitasauthor=G.+Senisterraauthor=F.+Liauthor=M.+Schapiraauthor=P.+J.+Brownauthor=C.+H.+Arrowsmithauthor=D.+Barsyte-Lovejoyauthor=J.+Liuauthor=M.+Vedadiauthor=J.+Jin&title=Discovery+of+a+potent%2C+selective%2C+and+cell-active+dual+inhibitor+of+protein+arginine+methyltransferase+4+and+protein+arginine+methyltransferase+6&doi=10.1021%2Facs.jmedchem.6b01033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent, Selective, and Cell-Active Dual Inhibitor of Protein Arginine Methyltransferase 4 and Protein Arginine Methyltransferase 6</span></div><div class="casAuthors">Shen, Yudao; Szewczyk, Magdalena M.; Eram, Mohammad S.; Smil, David; Kaniskan, H. Umit; Ferreira de Freitas, Renato; Senisterra, Guillermo; Li, Fengling; Schapira, Matthieu; Brown, Peter J.; Arrowsmith, Cheryl H.; Barsyte-Lovejoy, Dalia; Liu, Jing; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">9124-9139</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Well-characterized selective inhibitors of protein arginine methyltransferases (PRMTs) are invaluable chem. tools for testing biol. and therapeutic hypotheses.  Based on the lead compd., a fragment-like inhibitor of type I PRMTs, the authors conducted structure-activity relationship (SAR) studies and explored three regions of this scaffold.  The studies led to the discovery of a potent, selective and cell-active dual inhibitor of PRMT4 and PRMT6, I (MS049).  As compared to the lead, I displayed much improved potency for PRMT4 and PRMT6 in both biochem. and cellular assays.  It was selective for PRMT4 and PRMT6 over other PRMTs and a broad range of other epigenetic modifiers and nonepigenetic targets.  The authors also developed II (MS049N), which was inactive in biochem. and cellular assays, as a neg. control for chem. biol. studies.  Considering possible overlapping substrate specificity of PRMTs, I and II are valuable chem. tools for dissecting specific biol. functions and dysregulation of PRMT4 and PRMT6 in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZENTB29LcELVg90H21EOLACvtfcHk0lhHWExHyfkirw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVKjs77O&md5=2dcdad732107d0a64f7e317a05be1284</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01033%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3Dde%2BFreitas%26aufirst%3DR.%2BF.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%2520potent%252C%2520selective%252C%2520and%2520cell-active%2520dual%2520inhibitor%2520of%2520protein%2520arginine%2520methyltransferase%25204%2520and%2520protein%2520arginine%2520methyltransferase%25206%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D9124%26epage%3D9139%26doi%3D10.1021%2Facs.jmedchem.6b01033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samanta, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mondal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W. R.</span></span> <span> </span><span class="NLM_article-title">Covalent inhibition in drug discovery</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">889</span>– <span class="NLM_lpage">906</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201900107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.1002%2Fcmdc.201900107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=30816012" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlvFensb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=889-906&author=A.+K.+Ghoshauthor=I.+Samantaauthor=A.+Mondalauthor=W.+R.+Liu&title=Covalent+inhibition+in+drug+discovery&doi=10.1002%2Fcmdc.201900107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent Inhibition in Drug Discovery</span></div><div class="casAuthors">Ghosh, Avick Kumar; Samanta, Indranil; Mondal, Anushree; Liu, Wenshe Ray</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">889-906</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Although covalent inhibitors have been used as therapeutics for more than a century, there has been general resistance in the pharmaceutical industry against their further development due to safety concerns.  This inclination has recently been reverted after the development of a wide variety of covalent inhibitors to address human health conditions along with the US Food and Drug Administration (FDA) approval of several covalent therapeutics for use in humans.  Along with this exciting resurrection of an old drug discovery concept, this review surveys enzymes that can be targeted by covalent inhibitors for the treatment of human diseases.  We focus on protein kinases, RAS proteins, and a few other enzymes that have been studied extensively as targets for covalent inhibition, with the aim to address challenges in designing effective covalent drugs and to provide suggestions in the area that have yet to be explored.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1p5IlI637AbVg90H21EOLACvtfcHk0lhHWExHyfkirw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlvFensb4%253D&md5=aaa9bd889193c02f8d9acdecaa45e64b</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201900107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201900107%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DA.%2BK.%26aulast%3DSamanta%26aufirst%3DI.%26aulast%3DMondal%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DW.%2BR.%26atitle%3DCovalent%2520inhibition%2520in%2520drug%2520discovery%26jtitle%3DChemMedChem%26date%3D2019%26volume%3D14%26spage%3D889%26epage%3D906%26doi%3D10.1002%2Fcmdc.201900107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Flanagan, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abramite, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aulabaugh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahal, U. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oppenheimer, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryder, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuff, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uccello, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayward, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noe, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obach, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philippe, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanmugasundaram, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Starr, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stroh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, Y.</span></span> <span> </span><span class="NLM_article-title">Chemical and computational methods for the characterization of covalent reactive groups for the prospective design of irreversible inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">10072</span>– <span class="NLM_lpage">10079</span>, <span class="refDoi"> DOI: 10.1021/jm501412a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501412a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVyjtLbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=10072-10079&author=M.+E.+Flanaganauthor=J.+A.+Abramiteauthor=D.+P.+Andersonauthor=A.+Aulabaughauthor=U.+P.+Dahalauthor=A.+M.+Gilbertauthor=C.+Liauthor=J.+Montgomeryauthor=S.+R.+Oppenheimerauthor=T.+Ryderauthor=B.+P.+Schuffauthor=D.+P.+Uccelloauthor=G.+S.+Walkerauthor=Y.+Wuauthor=M.+F.+Brownauthor=J.+M.+Chenauthor=M.+M.+Haywardauthor=M.+C.+Noeauthor=R.+S.+Obachauthor=L.+Philippeauthor=V.+Shanmugasundaramauthor=M.+J.+Shapiroauthor=J.+Starrauthor=J.+Strohauthor=Y.+Che&title=Chemical+and+computational+methods+for+the+characterization+of+covalent+reactive+groups+for+the+prospective+design+of+irreversible+inhibitors&doi=10.1021%2Fjm501412a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical and Computational Methods for the Characterization of Covalent Reactive Groups for the Prospective Design of Irreversible Inhibitors</span></div><div class="casAuthors">Flanagan, Mark E.; Abramite, Joseph A.; Anderson, Dennis P.; Aulabaugh, Ann; Dahal, Upendra P.; Gilbert, Adam M.; Li, Chao; Montgomery, Justin; Oppenheimer, Stacey R.; Ryder, Tim; Schuff, Brandon P.; Uccello, Daniel P.; Walker, Gregory S.; Wu, Yan; Brown, Matthew F.; Chen, Jinshan M.; Hayward, Matthew M.; Noe, Mark C.; Obach, R. Scott; Philippe, Laurence; Shanmugasundaram, Veerabahu; Shapiro, Michael J.; Starr, Jeremy; Stroh, Justin; Che, Ye</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10072-10079</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Interest in drugs that covalently modify their target is driven by the desire for enhanced efficacy that can result from the silencing of enzymic activity until protein resynthesis can occur, along with the potential for increased selectivity by targeting uniquely positioned nucleophilic residues in the protein.  However, covalent approaches carry addnl. risk for toxicities or hypersensitivity reactions that can result from covalent modification of unintended targets.  Here we describe methods for measuring the reactivity of covalent reactive groups (CRGs) with a biol. relevant nucleophile, glutathione (GSH), along with kinetic data for a broad array of electrophiles.  We also describe a computational method for predicting electrophilic reactivity, which taken together can be applied to the prospective design of thiol-reactive covalent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAzTSCv_Nj6bVg90H21EOLACvtfcHk0lh4OUeZ7TdxfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVyjtLbM&md5=3ccf59ab7a494655185e5eb5becf8c48</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm501412a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501412a%26sid%3Dliteratum%253Aachs%26aulast%3DFlanagan%26aufirst%3DM.%2BE.%26aulast%3DAbramite%26aufirst%3DJ.%2BA.%26aulast%3DAnderson%26aufirst%3DD.%2BP.%26aulast%3DAulabaugh%26aufirst%3DA.%26aulast%3DDahal%26aufirst%3DU.%2BP.%26aulast%3DGilbert%26aufirst%3DA.%2BM.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DMontgomery%26aufirst%3DJ.%26aulast%3DOppenheimer%26aufirst%3DS.%2BR.%26aulast%3DRyder%26aufirst%3DT.%26aulast%3DSchuff%26aufirst%3DB.%2BP.%26aulast%3DUccello%26aufirst%3DD.%2BP.%26aulast%3DWalker%26aufirst%3DG.%2BS.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DBrown%26aufirst%3DM.%2BF.%26aulast%3DChen%26aufirst%3DJ.%2BM.%26aulast%3DHayward%26aufirst%3DM.%2BM.%26aulast%3DNoe%26aufirst%3DM.%2BC.%26aulast%3DObach%26aufirst%3DR.%2BS.%26aulast%3DPhilippe%26aufirst%3DL.%26aulast%3DShanmugasundaram%26aufirst%3DV.%26aulast%3DShapiro%26aufirst%3DM.%2BJ.%26aulast%3DStarr%26aufirst%3DJ.%26aulast%3DStroh%26aufirst%3DJ.%26aulast%3DChe%26aufirst%3DY.%26atitle%3DChemical%2520and%2520computational%2520methods%2520for%2520the%2520characterization%2520of%2520covalent%2520reactive%2520groups%2520for%2520the%2520prospective%2520design%2520of%2520irreversible%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D10072%26epage%3D10079%26doi%3D10.1021%2Fjm501412a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobrovetsky, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korboukh, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Exploiting an allosteric binding site of PRMT3 yields potent and selective inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">2110</span>– <span class="NLM_lpage">2124</span>, <span class="refDoi"> DOI: 10.1021/jm3018332</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3018332" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtlWgt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=2110-2124&author=F.+Liuauthor=F.+Liauthor=A.+Maauthor=E.+Dobrovetskyauthor=A.+Dongauthor=C.+Gaoauthor=I.+Korboukhauthor=J.+Liuauthor=D.+Smilauthor=P.+J.+Brownauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=M.+Schapiraauthor=M.+Vedadiauthor=J.+Jin&title=Exploiting+an+allosteric+binding+site+of+PRMT3+yields+potent+and+selective+inhibitors&doi=10.1021%2Fjm3018332"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Exploiting an Allosteric Binding Site of PRMT3 Yields Potent and Selective Inhibitors</span></div><div class="casAuthors">Liu, Feng; Li, Fengling; Ma, Anqi; Dobrovetsky, Elena; Dong, Aiping; Gao, Cen; Korboukh, Ilia; Liu, Jing; Smil, David; Brown, Peter J.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Schapira, Matthieu; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2110-2124</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferases (PRMTs) play an important role in diverse biol. processes.  Among the nine known human PRMTs, PRMT3 has been implicated in ribosomal biosynthesis via asym. dimethylation of the 40S ribosomal protein S2 and in cancer via interaction with the DAL-1 tumor suppressor protein.  However, few selective inhibitors of PRMTs have been discovered.  We recently disclosed the first selective PRMT3 inhibitor, which occupies a novel allosteric binding site and is noncompetitive with both the peptide substrate and cofactor.  Here we report comprehensive structure-activity relationship studies of this series, which resulted in the discovery of multiple PRMT3 inhibitors with submicromolar potencies.  An X-ray crystal structure of compd. I in complex with PRMT3 confirmed that this inhibitor occupied the same allosteric binding site as our initial lead compd.  These studies provide the first exptl. evidence that potent and selective inhibitors can be created by exploiting the allosteric binding site of PRMT3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgc4z8jYHwWbVg90H21EOLACvtfcHk0lh4OUeZ7TdxfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtlWgt74%253D&md5=d538b236cbe1326ed74d84ab12778e47</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjm3018332&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3018332%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DMa%26aufirst%3DA.%26aulast%3DDobrovetsky%26aufirst%3DE.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DKorboukh%26aufirst%3DI.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DExploiting%2520an%2520allosteric%2520binding%2520site%2520of%2520PRMT3%2520yields%2520potent%2520and%2520selective%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D2110%26epage%3D2124%26doi%3D10.1021%2Fjm3018332" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krippendorff, B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neuhaus, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lienau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reichel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huisinga, W.</span></span> <span> </span><span class="NLM_article-title">Mechanism-based inhibition: deriving K(I) and k(inact) directly from time-dependent IC(50) values</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">913</span>– <span class="NLM_lpage">923</span>, <span class="refDoi"> DOI: 10.1177/1087057109336751</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.1177%2F1087057109336751" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=19675314" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlSjtLjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=913-923&author=B.+F.+Krippendorffauthor=R.+Neuhausauthor=P.+Lienauauthor=A.+Reichelauthor=W.+Huisinga&title=Mechanism-based+inhibition%3A+deriving+K%28I%29+and+k%28inact%29+directly+from+time-dependent+IC%2850%29+values&doi=10.1177%2F1087057109336751"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism-based inhibition: deriving KI and kinact directly from time-dependent IC50 values</span></div><div class="casAuthors">Krippendorff, Ben-Fillippo; Neuhaus, Roland; Lienau, Philip; Reichel, Andreas; Huisinga, Wilhelm</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">913-923</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The potential of enzyme inhibition of a drug is frequently quantified in terms of IC50 values.  Although this is a suitable quantity for reversible inhibitors, concerns arise when dealing with irreversible or mechanism-based inhibitors (MBIs).  IC50 values of MBIs are time dependent, causing serious problems when aiming at ranking different compds. with respect to their inhibitory potential.  As a consequence, most studies and ranking schemes related to MBIs rely on the inhibition const. (KI) and the rate of enzyme inactivation (kinact) rather than on IC50 values.  In this article, the authors derive a novel relation between potentially time-dependent IC50 values and KI, kinact parameters for different types of inhibition.  This allows for direct estn. of KI and kinact values from time-dependent IC50 values, even without the need of addnl. preincubation expts.  The application of this approach is illustrated using a fluorimetric assay to access the drug-drug interaction potential assocd. with new chem. entities.  The approach can easily be implemented using std. software tools (e.g., XLfit) and may also be suitable for applications where mechanism-based inhibition is a desired mode of action (e.g., at particular pharmacol. drug targets).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSOIpe0vN8JLVg90H21EOLACvtfcHk0lh4OUeZ7TdxfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlSjtLjF&md5=93cb5f8644c7e1bdf872cf46756039cf</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1177%2F1087057109336751&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057109336751%26sid%3Dliteratum%253Aachs%26aulast%3DKrippendorff%26aufirst%3DB.%2BF.%26aulast%3DNeuhaus%26aufirst%3DR.%26aulast%3DLienau%26aufirst%3DP.%26aulast%3DReichel%26aufirst%3DA.%26aulast%3DHuisinga%26aufirst%3DW.%26atitle%3DMechanism-based%2520inhibition%253A%2520deriving%2520K%2528I%2529%2520and%2520k%2528inact%2529%2520directly%2520from%2520time-dependent%2520IC%252850%2529%2520values%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2009%26volume%3D14%26spage%3D913%26epage%3D923%26doi%3D10.1177%2F1087057109336751" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sayegh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span> <span> </span><span class="NLM_article-title">PRMT8, a new membrane-bound tissue-specific member of the protein arginine methyltransferase family</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>280</i></span>,  <span class="NLM_fpage">32890</span>– <span class="NLM_lpage">32896</span>, <span class="refDoi"> DOI: 10.1074/jbc.M506944200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.1074%2Fjbc.M506944200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=16051612" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVWiur3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2005&pages=32890-32896&author=J.+Leeauthor=J.+Sayeghauthor=J.+Danielauthor=S.+Clarkeauthor=M.+T.+Bedford&title=PRMT8%2C+a+new+membrane-bound+tissue-specific+member+of+the+protein+arginine+methyltransferase+family&doi=10.1074%2Fjbc.M506944200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">PRMT8, a New Membrane-bound Tissue-specific Member of the Protein Arginine Methyltransferase Family</span></div><div class="casAuthors">Lee, Jaeho; Sayegh, Joyce; Daniel, Jeremy; Clarke, Steven; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">280</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">32890-32896</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Protein arginine methylation is a common post-translational modification that has been implicated in signal transduction, RNA processing, transcriptional regulation, and DNA repair.  A search of the human genome for addnl. members of the protein arginine N-methyltransferase (PRMT) family of enzymes has identified a gene on chromosome 12 that we have termed PRMT8.  This novel enzyme is most closely related to PRMT1, although it has a distinctive N-terminal region.  The unique N-terminal end harbors a myristoylation motif, and we have shown here that PRMT8 is indeed modified by the attachment of a myristate to the glycine residue after the initiator methionine.  The myristoylation of PRMT8 results in its assocn. with the plasma membrane.  The second singular property of PRMT8 is its tissue-specific expression pattern; it is largely expressed in the brain.  A glutathione S-transferase fusion protein of PRMT8 has type I PRMT activity, catalyzing the formation of ω-NG-monomethylated and asym. ω-NG,NG-dimethylated arginine residues on a recombinant glycine- and arginine-rich substrate.  PRMT8 is thus an active arginine methyltransferase that is membrane-assocd. and tissue-specific, two firsts for this family of enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorefolZCmgTrVg90H21EOLACvtfcHk0ljpulroSbQJRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVWiur3E&md5=2a61ea03371c7bb01f0e79f25fc37099</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M506944200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M506944200%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DSayegh%26aufirst%3DJ.%26aulast%3DDaniel%26aufirst%3DJ.%26aulast%3DClarke%26aufirst%3DS.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DPRMT8%252C%2520a%2520new%2520membrane-bound%2520tissue-specific%2520member%2520of%2520the%2520protein%2520arginine%2520methyltransferase%2520family%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2005%26volume%3D280%26spage%3D32890%26epage%3D32896%26doi%3D10.1074%2Fjbc.M506944200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. Ü.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zang, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobrovetsky, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin-Jones, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atadja, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span> <span> </span><span class="NLM_article-title">A potent, selective and cell-active allosteric inhibitor of protein arginine methyltransferase 3 (PRMT3)</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">5166</span>– <span class="NLM_lpage">5170</span>, <span class="refDoi"> DOI: 10.1002/anie.201412154</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.1002%2Fanie.201412154" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BC2MXkvF2msLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=5166-5170&author=H.+%C3%9C.+Kaniskanauthor=M.+M.+Szewczykauthor=Z.+Yuauthor=M.+S.+Eramauthor=X.+Yangauthor=K.+Schmidtauthor=X.+Luoauthor=M.+Daiauthor=F.+Heauthor=I.+Zangauthor=Y.+Linauthor=S.+Kennedyauthor=F.+Liauthor=E.+Dobrovetskyauthor=A.+Dongauthor=D.+Smilauthor=S.+J.+Minauthor=M.+Landonauthor=J.+Lin-Jonesauthor=X.+P.+Huangauthor=B.+L.+Rothauthor=M.+Schapiraauthor=P.+Atadjaauthor=D.+Barsyte-Lovejoyauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=K.+Zhaoauthor=J.+Jinauthor=M.+Vedadi&title=A+potent%2C+selective+and+cell-active+allosteric+inhibitor+of+protein+arginine+methyltransferase+3+%28PRMT3%29&doi=10.1002%2Fanie.201412154"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">A Potent, Selective and Cell-Active Allosteric Inhibitor of Protein Arginine Methyltransferase 3 (PRMT3)</span></div><div class="casAuthors">Kaniskan, H. Uemit; Szewczyk, Magdalena M.; Yu, Zhengtian; Eram, Mohammad S.; Yang, Xiaobao; Schmidt, Keith; Luo, Xiao; Dai, Miao; He, Feng; Zang, Irene; Lin, Ying; Kennedy, Steven; Li, Fengling; Dobrovetsky, Elena; Dong, Aiping; Smil, David; Min, Sun-Joon; Landon, Melissa; Lin-Jones, Jennifer; Huang, Xi-Ping; Roth, Bryan L.; Schapira, Matthieu; Atadja, Peter; Barsyte-Lovejoy, Dalia; Arrowsmith, Cheryl H.; Brown, Peter J.; Zhao, Kehao; Jin, Jian; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">5166-5170</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">PRMT3 catalyzes the asym. dimethylation of arginine residues of various proteins.  It is essential for maturation of ribosomes, may have a role in lipogenesis, and is implicated in several diseases.  A potent, selective, and cell-active PRMT3 inhibitor would be a valuable tool for further investigating PRMT3 biol.  Here we report the discovery of the first PRMT3 chem. probe, SGC707, by structure-based optimization of the allosteric PRMT3 inhibitors we reported previously, and thorough characterization of this probe in biochem., biophys., and cellular assays.  SGC707 is a potent PRMT3 inhibitor (IC50=31±2 nM, KD=53±2 nM) with outstanding selectivity (selective against 31 other methyltransferases and more than 250 non-epigenetic targets).  The mechanism of action studies and crystal structure of the PRMT3-SGC707 complex confirm the allosteric inhibition mode.  Importantly, SGC707 engages PRMT3 and potently inhibits its methyltransferase activity in cells.  It is also bioavailable and suitable for animal studies.  This well-characterized chem. probe is an excellent tool to further study the role of PRMT3 in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTk4dFfkDLm7Vg90H21EOLACvtfcHk0ljpulroSbQJRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXkvF2msLo%253D&md5=283b0875813af0f7631235e8ba7811c2</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1002%2Fanie.201412154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201412154%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DYu%26aufirst%3DZ.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DSchmidt%26aufirst%3DK.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DDai%26aufirst%3DM.%26aulast%3DHe%26aufirst%3DF.%26aulast%3DZang%26aufirst%3DI.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DDobrovetsky%26aufirst%3DE.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DMin%26aufirst%3DS.%2BJ.%26aulast%3DLandon%26aufirst%3DM.%26aulast%3DLin-Jones%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DAtadja%26aufirst%3DP.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DZhao%26aufirst%3DK.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DA%2520potent%252C%2520selective%2520and%2520cell-active%2520allosteric%2520inhibitor%2520of%2520protein%2520arginine%2520methyltransferase%25203%2520%2528PRMT3%2529%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D54%26spage%3D5166%26epage%3D5170%26doi%3D10.1002%2Fanie.201412154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. Ü.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zang, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobrovetsky, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin-Jones, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atadja, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of potent and selective allosteric inhibitors of protein arginine methyltransferase 3 (PRMT3)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1204</span>– <span class="NLM_lpage">1217</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01674</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01674" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvF2gsbzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=1204-1217&author=H.+%C3%9C.+Kaniskanauthor=M.+S.+Eramauthor=K.+Zhaoauthor=M.+M.+Szewczykauthor=X.+Yangauthor=K.+Schmidtauthor=X.+Luoauthor=S.+Xiaoauthor=M.+Daiauthor=F.+Heauthor=I.+Zangauthor=Y.+Linauthor=F.+Liauthor=E.+Dobrovetskyauthor=D.+Smilauthor=S.+J.+Minauthor=J.+Lin-Jonesauthor=M.+Schapiraauthor=P.+Atadjaauthor=E.+Liauthor=D.+Barsyte-Lovejoyauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=F.+Liuauthor=Z.+Yuauthor=M.+Vedadiauthor=J.+Jin&title=Discovery+of+potent+and+selective+allosteric+inhibitors+of+protein+arginine+methyltransferase+3+%28PRMT3%29&doi=10.1021%2Facs.jmedchem.7b01674"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Potent and Selective Allosteric Inhibitors of Protein Arginine Methyltransferase 3 (PRMT3)</span></div><div class="casAuthors">Kaniskan, H. Umit; Eram, Mohammad S.; Zhao, Kehao; Szewczyk, Magdalena M.; Yang, Xiaobao; Schmidt, Keith; Luo, Xiao; Xiao, Sean; Dai, Miao; He, Feng; Zang, Irene; Lin, Ying; Li, Fengling; Dobrovetsky, Elena; Smil, David; Min, Sun-Joon; Lin-Jones, Jennifer; Schapira, Matthieu; Atadja, Peter; Li, En; Barsyte-Lovejoy, Dalia; Arrowsmith, Cheryl H.; Brown, Peter J.; Liu, Feng; Yu, Zhengtian; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1204-1217</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">PRMT3 catalyzes the asym. dimethylation of arginine residues of various proteins.  It is crucial for maturation of ribosomes, and has been implicated in several diseases.  We recently disclosed a highly potent, selective and cell-active allosteric inhibitor of PRMT3, compd. 4.  Here, we report comprehensive structure-activity relationship studies that target the allosteric binding site of PRMT3.  We conducted design, synthesis and evaluation of novel compds. in biochem., selectivity and cellular assays that culminated in the discovery of 4 and other highly potent (IC50 values: ∼10-36 nM), selective and cell-active allosteric inhibitors of PRMT3 (compds. 29, 30, 36 and 37).  In addn., we generated compds. that are very close analogs of these potent inhibitors, but displayed drastically reduced potency as neg. controls (compds. 49-51).  These inhibitors and neg. controls are valuable chem. tools for the biomedical community to further investigate biol. functions and disease assocns. of PRMT3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodgUnzNZOB2rVg90H21EOLACvtfcHk0lgUMyAqvnHJrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvF2gsbzO&md5=a784663c26234c3a99386ea4fe3f8abc</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01674%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DZhao%26aufirst%3DK.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DSchmidt%26aufirst%3DK.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DXiao%26aufirst%3DS.%26aulast%3DDai%26aufirst%3DM.%26aulast%3DHe%26aufirst%3DF.%26aulast%3DZang%26aufirst%3DI.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DDobrovetsky%26aufirst%3DE.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DMin%26aufirst%3DS.%2BJ.%26aulast%3DLin-Jones%26aufirst%3DJ.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DAtadja%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DE.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DYu%26aufirst%3DZ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520potent%2520and%2520selective%2520allosteric%2520inhibitors%2520of%2520protein%2520arginine%2520methyltransferase%25203%2520%2528PRMT3%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D1204%26epage%3D1217%26doi%3D10.1021%2Facs.jmedchem.7b01674" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakayama, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dela Sena, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Freitas, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baba, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunitomo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cary, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tawada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohashi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imaeda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saikatendu, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimshaw, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiba, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomita, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span> <span> </span><span class="NLM_article-title">TP-064, a potent and selective small molecule inhibitor of PRMT4 for multiple myeloma</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">18480</span>– <span class="NLM_lpage">18493</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.24883</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.18632%2Foncotarget.24883" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=29719619" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A280%3ADC%252BC1MjptFWhtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=18480-18493&author=K.+Nakayamaauthor=M.+M.+Szewczykauthor=C.+Dela+Senaauthor=H.+Wuauthor=A.+Dongauthor=H.+Zengauthor=F.+Liauthor=R.+F.+de+Freitasauthor=M.+S.+Eramauthor=M.+Schapiraauthor=Y.+Babaauthor=M.+Kunitomoauthor=D.+R.+Caryauthor=M.+Tawadaauthor=A.+Ohashiauthor=Y.+Imaedaauthor=K.+S.+Saikatenduauthor=C.+E.+Grimshawauthor=M.+Vedadiauthor=C.+H.+Arrowsmithauthor=D.+Barsyte-Lovejoyauthor=A.+Kibaauthor=D.+Tomitaauthor=P.+J.+Brown&title=TP-064%2C+a+potent+and+selective+small+molecule+inhibitor+of+PRMT4+for+multiple+myeloma&doi=10.18632%2Foncotarget.24883"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">TP-064, a potent and selective small molecule inhibitor of PRMT4 for multiple myeloma</span></div><div class="casAuthors">Nakayama Kazuhide; Baba Yuji; Kunitomo Mihoko; Cary Douglas R; Ohashi Akihiro; Imaeda Yasuhiro; Kiba Atsushi; Tomita Daisuke; Szewczyk Magdalena M; Dela Sena Carlo; Wu Hong; Dong Aiping; Zeng Hong; Li Fengling; de Freitas Renato Ferreira; Eram Mohammad S; Schapira Matthieu; Vedadi Masoud; Arrowsmith Cheryl H; Barsyte-Lovejoy Dalia; Brown Peter J; Schapira Matthieu; Vedadi Masoud; Tawada Michiko; Saikatendu Kumar Singh; Grimshaw Charles E; Arrowsmith Cheryl H</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">18480-18493</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Protein arginine methyltransferase (PRMT) 4 (also known as coactivator-associated arginine methyltransferase 1; CARM1) is involved in a variety of biological processes and is considered as a candidate oncogene owing to its overexpression in several types of cancer.  Selective PRMT4 inhibitors are useful tools for clarifying the molecular events regulated by PRMT4 and for validating PRMT4 as a therapeutic target.  Here, we report the discovery of TP-064, a potent, selective, and cell-active chemical probe of human PRMT4 and its co-crystal structure with PRMT4.  TP-064 inhibited the methyltransferase activity of PRMT4 with high potency (half-maximal inhibitory concentration, IC50 < 10 nM) and selectivity over other PRMT family proteins, and reduced arginine dimethylation of the PRMT4 substrates BRG1-associated factor 155 (BAF155; IC50= 340 ± 30 nM) and Mediator complex subunit 12 (MED12; IC50 = 43 ± 10 nM).  TP-064 treatment inhibited the proliferation of a subset of multiple myeloma cell lines, with affected cells arrested in G1 phase of the cell cycle.  TP-064 and its negative control (TP-064N) will be valuable tools to further investigate the biology of PRMT4 and the therapeutic potential of PRMT4 inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTsNG0-q3aOlw20zIbnCCI3fW6udTcc2ebYFfM9oHqBULntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MjptFWhtA%253D%253D&md5=f910a11dc3aa023564499ba2e6209e31</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.24883&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.24883%26sid%3Dliteratum%253Aachs%26aulast%3DNakayama%26aufirst%3DK.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DDela%2BSena%26aufirst%3DC.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DF.%26aulast%3Dde%2BFreitas%26aufirst%3DR.%2BF.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DBaba%26aufirst%3DY.%26aulast%3DKunitomo%26aufirst%3DM.%26aulast%3DCary%26aufirst%3DD.%2BR.%26aulast%3DTawada%26aufirst%3DM.%26aulast%3DOhashi%26aufirst%3DA.%26aulast%3DImaeda%26aufirst%3DY.%26aulast%3DSaikatendu%26aufirst%3DK.%2BS.%26aulast%3DGrimshaw%26aufirst%3DC.%2BE.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DKiba%26aufirst%3DA.%26aulast%3DTomita%26aufirst%3DD.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26atitle%3DTP-064%252C%2520a%2520potent%2520and%2520selective%2520small%2520molecule%2520inhibitor%2520of%2520PRMT4%2520for%2520multiple%2520myeloma%26jtitle%3DOncotarget%26date%3D2018%26volume%3D9%26spage%3D18480%26epage%3D18493%26doi%3D10.18632%2Foncotarget.24883" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira
de Freitas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steuber, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senisterra, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hitchcock, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moosmayer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmann, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Egner, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span> <span> </span><span class="NLM_article-title">Discovery of a potent class I protein arginine methyltransferase fragment inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1176</span>– <span class="NLM_lpage">1183</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01772</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01772" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVWls78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1176-1183&author=R.+Ferreira%0Ade+Freitasauthor=M.+S.+Eramauthor=M.+M.+Szewczykauthor=H.+Steuberauthor=D.+Smilauthor=H.+Wuauthor=F.+Liauthor=G.+Senisterraauthor=A.+Dongauthor=P.+J.+Brownauthor=M.+Hitchcockauthor=D.+Moosmayerauthor=C.+M.+Stegmannauthor=U.+Egnerauthor=C.+Arrowsmithauthor=D.+Barsyte-Lovejoyauthor=M.+Vedadiauthor=M.+Schapira&title=Discovery+of+a+potent+class+I+protein+arginine+methyltransferase+fragment+inhibitor&doi=10.1021%2Facs.jmedchem.5b01772"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent Class I Protein Arginine Methyltransferase Fragment Inhibitor</span></div><div class="casAuthors">Ferreira de Freitas, Renato; Eram, Mohammad S.; Szewczyk, Magdalena M.; Steuber, Holger; Smil, David; Wu, Hong; Li, Fengling; Senisterra, Guillermo; Dong, Aiping; Brown, Peter J.; Hitchcock, Marion; Moosmayer, Dieter; Stegmann, Christian M.; Egner, Ursula; Arrowsmith, Cheryl; Barsyte-Lovejoy, Dalia; Vedadi, Masoud; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1176-1183</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein methyltransferases (PMTs) are a promising target class in oncol. and other disease areas.  They are composed of SET domain methyltransferases and structurally unrelated Rossman-fold enzymes that include protein arginine methyltransferases (PRMTs).  In the absence of a well-defined medicinal chem. tool-kit focused on PMTs, most current inhibitors were identified by screening large and diverse libraries of leadlike mols.  So far, no successful fragment-based approach was reported against this target class.  Here, by deconstructing potent PRMT inhibitors, we find that chem. moieties occupying the substrate arginine-binding site can act as efficient fragment inhibitors.  Screening a fragment library against PRMT6 produced numerous hits, including a 300 nM inhibitor (ligand efficiency of 0.56) that decreased global histone 3 arginine 2 methylation in cells, and can serve as a warhead for the development of PRMT chem. probes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplB_HPKhzNcLVg90H21EOLACvtfcHk0ljiEuo3x27Nvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVWls78%253D&md5=03cbe3214b3a80f05032692037cbfd52</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01772&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01772%26sid%3Dliteratum%253Aachs%26aulast%3DFerreira%2Bde%2BFreitas%26aufirst%3DR.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DSteuber%26aufirst%3DH.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DHitchcock%26aufirst%3DM.%26aulast%3DMoosmayer%26aufirst%3DD.%26aulast%3DStegmann%26aufirst%3DC.%2BM.%26aulast%3DEgner%26aufirst%3DU.%26aulast%3DArrowsmith%26aufirst%3DC.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520a%2520potent%2520class%2520I%2520protein%2520arginine%2520methyltransferase%2520fragment%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1176%26epage%3D1183%26doi%3D10.1021%2Facs.jmedchem.5b01772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cymborowski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chruszcz, M.</span></span> <span> </span><span class="NLM_article-title">HKL-3000: the integration of data reduction and structure solution--from diffraction images to an initial model in minutes</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">859</span>– <span class="NLM_lpage">866</span>, <span class="refDoi"> DOI: 10.1107/S0907444906019949</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.1107%2FS0907444906019949" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=16855301" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BD28XntVWksL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=859-866&author=W.+Minorauthor=M.+Cymborowskiauthor=Z.+Otwinowskiauthor=M.+Chruszcz&title=HKL-3000%3A+the+integration+of+data+reduction+and+structure+solution%2D%2Dfrom+diffraction+images+to+an+initial+model+in+minutes&doi=10.1107%2FS0907444906019949"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">HKL-3000: the integration of data reduction and structure solution - from diffraction images to an initial model in minutes</span></div><div class="casAuthors">Minor, Wladek; Cymborowski, Marcin; Otwinowski, Zbyszek; Chruszcz, Maksymilian</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">D62</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">859-866</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A new approach that integrates data collection, data redn., phasing and model building significantly accelerates the process of structure detn. and on av. minimizes the no. of data sets and synchrotron time required for structure soln.  Initial testing of the HKL-3000 system (the beta version was named HKL-2000_ph) with more than 140 novel structure detns. has proven its high value for MAD/SAD expts.  The heuristics for choosing the best computational strategy at different data resoln. limits of phasing signal and crystal diffraction are being optimized.  The typical end result is an interpretable electron-d. map with a partially built structure and, in some cases, an almost complete refined model.  The current development is oriented towards very fast structure soln. in order to provide feedback during the diffraction expt.  Work is also proceeding towards improving the quality of phasing calcn. and model building.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0c0DYtjFP7rVg90H21EOLACvtfcHk0ljiEuo3x27Nvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XntVWksL8%253D&md5=2cd2fe288f954837abb4acae4c008a2f</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1107%2FS0907444906019949&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444906019949%26sid%3Dliteratum%253Aachs%26aulast%3DMinor%26aufirst%3DW.%26aulast%3DCymborowski%26aufirst%3DM.%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DChruszcz%26aufirst%3DM.%26atitle%3DHKL-3000%253A%2520the%2520integration%2520of%2520data%2520reduction%2520and%2520structure%2520solution--from%2520diffraction%2520images%2520to%2520an%2520initial%2520model%2520in%2520minutes%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2006%26volume%3D62%26spage%3D859%26epage%3D866%26doi%3D10.1107%2FS0907444906019949" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Steiner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebedev, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span> <span> </span><span class="NLM_article-title">Fisher’s information in maximum-likelihood macromolecular crystallographic refinement</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">2114</span>– <span class="NLM_lpage">2124</span>, <span class="refDoi"> DOI: 10.1107/S0907444903018675</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.1107%2FS0907444903018675" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=14646069" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BD3sXptlGit7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2003&pages=2114-2124&author=R.+A.+Steinerauthor=A.+A.+Lebedevauthor=G.+N.+Murshudov&title=Fisher%E2%80%99s+information+in+maximum-likelihood+macromolecular+crystallographic+refinement&doi=10.1107%2FS0907444903018675"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Fisher's information in maximum-likelihood macromolecular crystallographic refinement</span></div><div class="casAuthors">Steiner, Roberto A.; Lebedev, Andrey A.; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">D59</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2114-2124</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">Fisher's information is a statistical quantity related to max.-likelihood theory.  It is a matrix defined as the expected value of the squared gradient of minus the log-likelihood function.  This matrix is pos. semidefinite for any parameter value.  Fisher's information is used in the quasi-Newton scoring method of minimization to calc. the shift vectors of model parameters.  If the matrix is non-singular, the scoring-minimization step is always downhill.  In this article, it is shown how the scoring method can be applied to macromol. crystallog. refinement.  It is also shown how the computational costs involved in calcn. of the Fisher's matrix can be efficiently reduced.  Speed is achieved by assuming a continuous distribution of reciprocal-lattice points.  Matrix elements calcd. with this method agree very well with those calcd. anal.  The scoring algorithm has been implemented in the program REFMAC5 of the CCP4 suite.  The Fisher's matrix is used in its sparse approxn.  Tests indicate that the algorithm performs satisfactorily.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgph0YGiViZLVg90H21EOLACvtfcHk0ljiEuo3x27Nvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXptlGit7k%253D&md5=c8bd3bece9333f75ce2cffff26ac55e1</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1107%2FS0907444903018675&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444903018675%26sid%3Dliteratum%253Aachs%26aulast%3DSteiner%26aufirst%3DR.%2BA.%26aulast%3DLebedev%26aufirst%3DA.%2BA.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DFisher%25E2%2580%2599s%2520information%2520in%2520maximum-likelihood%2520macromolecular%2520crystallographic%2520refinement%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2003%26volume%3D59%26spage%3D2114%26epage%3D2124%26doi%3D10.1107%2FS0907444903018675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0ljiEuo3x27Nvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Williams, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prisant, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Videau, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deis, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keedy, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hintze, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arendall, W. B.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snoeyink, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lovell, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span> <span> </span><span class="NLM_article-title">MolProbity: more and better reference data for improved all-atom structure validation</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">293</span>– <span class="NLM_lpage">315</span>, <span class="refDoi"> DOI: 10.1002/pro.3330</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.1002%2Fpro.3330" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=29067766" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVKgsrnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2018&pages=293-315&author=C.+J.+Williamsauthor=J.+J.+Headdauthor=N.+W.+Moriartyauthor=M.+G.+Prisantauthor=L.+L.+Videauauthor=L.+N.+Deisauthor=V.+Vermaauthor=D.+A.+Keedyauthor=B.+J.+Hintzeauthor=V.+B.+Chenauthor=S.+Jainauthor=S.+M.+Lewisauthor=W.+B.+Arendallauthor=J.+Snoeyinkauthor=P.+D.+Adamsauthor=S.+C.+Lovellauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+more+and+better+reference+data+for+improved+all-atom+structure+validation&doi=10.1002%2Fpro.3330"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: More and better reference data for improved all-atom structure validation</span></div><div class="casAuthors">Williams, Christopher J.; Headd, Jeffrey J.; Moriarty, Nigel W.; Prisant, Michael G.; Videau, Lizbeth L.; Deis, Lindsay N.; Verma, Vishal; Keedy, Daniel A.; Hintze, Bradley J.; Chen, Vincent B.; Jain, Swati; Lewis, Steven M.; Arendall, W. Bryan, III; Snoeyink, Jack; Adams, Paul D.; Lovell, Simon C.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">293-315</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">1469-896X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">This paper describes the current update on macromol. model validation services that are provided at the MolProbity website, emphasizing changes and addns. since the previous review in 2010.  There have been many infrastructure improvements, including rewrite of previous Java utilities to now use existing or newly written Python utilities in the open-source CCTBX portion of the Phenix software system.  This improves long-term maintainability and enhances the thorough integration of MolProbity-style validation within Phenix.  There is now a complete MolProbity mirror site at .  GitHub serves our open-source code, ref. datasets, and the resulting multi-dimensional distributions that define most validation criteria.  Coordinate output after Asn/Gln/His "flip" correction is now more idealized, since the post-refinement step has apparently often been skipped in the past.  Two distinct sets of heavy-atom-to-hydrogen distances and accompanying van der Waals radii have been researched and improved in accuracy, one for the electron-cloud-center positions suitable for X-ray crystallog. and one for nuclear positions.  New validations include messages at input about problem-causing format irregularities, updates of Ramachandran and rotamer criteria from the million quality-filtered residues in a new ref. dataset, the CaBLAM Cα-CO virtual-angle anal. of backbone and secondary structure for cryoEM or low-resoln. X-ray, and flagging of the very rare cis-nonProline and twisted peptides which have recently been greatly overused.  Due to wide application of MolProbity validation and corrections by the research community, in Phenix, and at the worldwide Protein Data Bank, newly deposited structures have continued to improve greatly as measured by MolProbity's unique all-atom clashscore.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQFbgiBJUVTbVg90H21EOLACvtfcHk0lgQchCtvxrVWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVKgsrnO&md5=a8d8f315bc29e1ecdeb0f60a313baca6</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1002%2Fpro.3330&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.3330%26sid%3Dliteratum%253Aachs%26aulast%3DWilliams%26aufirst%3DC.%2BJ.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DPrisant%26aufirst%3DM.%2BG.%26aulast%3DVideau%26aufirst%3DL.%2BL.%26aulast%3DDeis%26aufirst%3DL.%2BN.%26aulast%3DVerma%26aufirst%3DV.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DHintze%26aufirst%3DB.%2BJ.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DJain%26aufirst%3DS.%26aulast%3DLewis%26aufirst%3DS.%2BM.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DSnoeyink%26aufirst%3DJ.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DLovell%26aufirst%3DS.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520more%2520and%2520better%2520reference%2520data%2520for%2520improved%2520all-atom%2520structure%2520validation%26jtitle%3DProtein%2520Sci.%26date%3D2018%26volume%3D27%26spage%3D293%26epage%3D315%26doi%3D10.1002%2Fpro.3330" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scheer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ackloo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medina, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Northrop, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. Ü.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLeod, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zepeda-Velazquez, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Carvalho, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaph, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span> <span> </span><span class="NLM_article-title">A chemical biology toolbox to study protein methyltransferases and epigenetic signaling</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>, <span class="NLM_elocation-id">19</span> <span class="refDoi"> DOI: 10.1038/s41467-018-07905-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=10.1038%2Fs41467-018-07905-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=30604761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmvVCmtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&author=S.+Scheerauthor=S.+Acklooauthor=T.+S.+Medinaauthor=M.+Schapiraauthor=F.+Liauthor=J.+A.+Wardauthor=A.+M.+Lewisauthor=J.+P.+Northropauthor=P.+L.+Richardsonauthor=H.+%C3%9C.+Kaniskanauthor=Y.+Shenauthor=J.+Liuauthor=D.+Smilauthor=D.+McLeodauthor=C.+A.+Zepeda-Velazquezauthor=M.+Luoauthor=J.+Jinauthor=D.+Barsyte-Lovejoyauthor=K.+V.+M.+Huberauthor=D.+D.+De+Carvalhoauthor=M.+Vedadiauthor=C.+Zaphauthor=P.+J.+Brownauthor=C.+H.+Arrowsmith&title=A+chemical+biology+toolbox+to+study+protein+methyltransferases+and+epigenetic+signaling&doi=10.1038%2Fs41467-018-07905-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">A chemical biology toolbox to study protein methyltransferases and epigenetic signaling</span></div><div class="casAuthors">Scheer, Sebastian; Ackloo, Suzanne; Medina, Tiago S.; Schapira, Matthieu; Li, Fengling; Ward, Jennifer A.; Lewis, Andrew M.; Northrop, Jeffrey P.; Richardson, Paul L.; Kaniskan, H. Umit; Shen, Yudao; Liu, Jing; Smil, David; McLeod, David; Zepeda-Velazquez, Carlos A.; Luo, Minkui; Jin, Jian; Barsyte-Lovejoy, Dalia; Huber, Kilian V. M.; De Carvalho, Daniel D.; Vedadi, Masoud; Zaph, Colby; Brown, Peter J.; Arrowsmith, Cheryl H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">19</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Protein methyltransferases (PMTs) comprise a major class of epigenetic regulatory enzymes with therapeutic relevance.  Here we present a collection of chem. probes and assocd. reagents and data to elucidate the function of human and murine PMTs in cellular studies.  Our collection provides inhibitors and antagonists that together modulate most of the key regulatory methylation marks on histones H3 and H4, providing an important resource for modulating cellular epigenomes.  We describe a comprehensive and comparative characterization of the probe collection with respect to their potency, selectivity, and mode of inhibition.  We demonstrate the utility of this collection in CD4+ T cell differentiation assays revealing the potential of individual probes to alter multiple T cell subpopulations which may have implications for T cell-mediated processes such as inflammation and immuno-oncol.  In particular, we demonstrate a role for DOT1L in limiting Th1 cell differentiation and maintaining lineage integrity.  This chem. probe collection and assocd. data form a resource for the study of methylation-mediated signaling in epigenetics, inflammation and beyond.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopA5rgeiWDvrVg90H21EOLACvtfcHk0lgQchCtvxrVWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmvVCmtro%253D&md5=008fec084426171f5c91657e54b73900</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fs41467-018-07905-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-018-07905-4%26sid%3Dliteratum%253Aachs%26aulast%3DScheer%26aufirst%3DS.%26aulast%3DAckloo%26aufirst%3DS.%26aulast%3DMedina%26aufirst%3DT.%2BS.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DWard%26aufirst%3DJ.%2BA.%26aulast%3DLewis%26aufirst%3DA.%2BM.%26aulast%3DNorthrop%26aufirst%3DJ.%2BP.%26aulast%3DRichardson%26aufirst%3DP.%2BL.%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DMcLeod%26aufirst%3DD.%26aulast%3DZepeda-Velazquez%26aufirst%3DC.%2BA.%26aulast%3DLuo%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DDe%2BCarvalho%26aufirst%3DD.%2BD.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DZaph%26aufirst%3DC.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26atitle%3DA%2520chemical%2520biology%2520toolbox%2520to%2520study%2520protein%2520methyltransferases%2520and%2520epigenetic%2520signaling%26jtitle%3DNat.%2520Commun.%26date%3D2019%26volume%3D10%26doi%3D10.1038%2Fs41467-018-07905-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5E8R" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5E8R','PDB','5E8R'); return false;">PDB: 5E8R</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6P7I" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6P7I','PDB','6P7I'); return false;">PDB: 6P7I</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EGS" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EGS','PDB','5EGS'); return false;">PDB: 5EGS</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i30"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00406">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_48204"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c00406?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00406</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Crystallography data and refinement statistics; selectivity data of compound <b>4</b> against 33 methyltransferases; sequence alignment of type I PRMTs; MS assay results for PRMT1, PRMT3, PRMT4, and PRMT8 preincubated with compound <b>4</b>; and <sup>1</sup>H and <sup>13</sup>C NMR spectra of compounds <b>1</b>–<b>7</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00406/suppl_file/jm0c00406_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings for all compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00406/suppl_file/jm0c00406_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00406/suppl_file/jm0c00406_si_001.pdf">jm0c00406_si_001.pdf (2.53 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00406/suppl_file/jm0c00406_si_002.csv">jm0c00406_si_002.csv (0.94 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The structure presented in this article has been deposited in the Protein Data Bank under accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6P7I">6P7I</a> for the PRMT6–MS117 complex. The authors will release the atomic coordinates upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c00406&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c00406%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-10" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c00406" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679a4fbf081bd1c3","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
