
<html lang="en"     class="pb-page"  data-request-id="86baca8f-89b7-4730-84f1-217e2f7ef5ea"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/acs.jmedchem.0c00770;requestedJournal:journal:jmcmar;journal:journal:jmcmar;issue:issue:10.1021/jmcmar.2020.63.issue-15;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Probing the Plasticity in the Active Site of Protein N-terminal Methyltransferase 1 Using Bisubstrate Analogues" /></meta><meta name="dc.Creator" content="Dongxing  Chen" /></meta><meta name="dc.Creator" content="Cheng  Dong" /></meta><meta name="dc.Creator" content="Guangping  Dong" /></meta><meta name="dc.Creator" content="Karthik  Srinivasan" /></meta><meta name="dc.Creator" content="Jinrong  Min" /></meta><meta name="dc.Creator" content="Nicholas  Noinaj" /></meta><meta name="dc.Creator" content="Rong  Huang" /></meta><meta name="dc.Description" content="The bisubstrate analogue strategy is a promising approach to develop potent and selective inhibitors for protein methyltransferases. Herein, the interactions of a series of bisubstrate analogues wi..." /></meta><meta name="Description" content="The bisubstrate analogue strategy is a promising approach to develop potent and selective inhibitors for protein methyltransferases. Herein, the interactions of a series of bisubstrate analogues wi..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 1, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c00770" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00770" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c00770" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00770" /></link>
        
    
    

<title>Probing the Plasticity in the Active Site of Protein N-terminal Methyltransferase 1 Using Bisubstrate Analogues | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c00770" /></meta><meta property="og:title" content="Probing the Plasticity in the Active Site of Protein N-terminal Methyltransferase 1 Using Bisubstrate Analogues" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/large/jm0c00770_0011.jpeg" /></meta><meta property="og:description" content="The bisubstrate analogue strategy is a promising approach to develop potent and selective inhibitors for protein methyltransferases. Herein, the interactions of a series of bisubstrate analogues with protein N-terminal methyltransferase 1 (NTMT1) were examined to probe the molecular properties of the active site of NTMT1. Our results indicate that a 2-C to 4-C atom linker enables its respective bisubstrate analogue to occupy both substrate- and cofactor-binding sites of NTMT1, but the bisubstrate analogue with a 5-C atom linker only interacts with the substrate-binding site and functions as a substrate. Furthermore, the 4-C atom linker is the optimal and produces the most potent inhibitor (Ki,app = 130 ± 40 pM) for NTMT1 to date, displaying more than 3000-fold selectivity for other methyltransferases and even for its homologue NTMT2. This study reveals the molecular basis for the plasticity of the active site of NTMT1. Additionally, our study outlines general guidance on the development of bisubstrate inhibitors for any methyltransferases." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c00770"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00770">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c00770&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c00770&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c00770&amp;href=/doi/10.1021/acs.jmedchem.0c00770" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 15</span><span class="cit-fg-pageRange">, 8419-8431</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/15" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c00754" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c00799" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Probing the Plasticity in the Active Site of Protein N-terminal Methyltransferase 1 Using Bisubstrate Analogues</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Dongxing Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dongxing Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry and Molecular Pharmacology, Center for Cancer Research, Institute for Drug Discovery, Purdue University, West Lafayette, Indiana 47907, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dongxing++Chen">Dongxing Chen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Cheng Dong</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Cheng Dong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, Department of Physiology, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Cheng++Dong">Cheng Dong</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Guangping Dong</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Guangping Dong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry and Molecular Pharmacology, Center for Cancer Research, Institute for Drug Discovery, Purdue University, West Lafayette, Indiana 47907, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Guangping++Dong">Guangping Dong</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Karthik Srinivasan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Karthik Srinivasan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Markey Center for Structural Biology, Department of Biological Sciences and the Purdue Institute of Inflammation, Immunology and Infectious Disease, Purdue University, West Lafayette, Indiana 47907, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Karthik++Srinivasan">Karthik Srinivasan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jinrong Min</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jinrong Min</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, Department of Physiology, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jinrong++Min">Jinrong Min</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nicholas Noinaj</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nicholas Noinaj</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Markey Center for Structural Biology, Department of Biological Sciences and the Purdue Institute of Inflammation, Immunology and Infectious Disease, Purdue University, West Lafayette, Indiana 47907, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nicholas++Noinaj">Nicholas Noinaj</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Rong Huang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rong Huang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry and Molecular Pharmacology, Center for Cancer Research, Institute for Drug Discovery, Purdue University, West Lafayette, Indiana 47907, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#0d65786c636a207f4d7d787f69786823686978"><span class="__cf_email__" data-cfemail="b6dec3d7d8d19bc4f6c6c3c4d2c3d398d3d2c3">[email protected]</span></a>. Phone: 765 494 3426.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rong++Huang">Rong Huang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1477-3165" title="Orcid link">http://orcid.org/0000-0002-1477-3165</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00770&amp;href=/doi/10.1021%2Facs.jmedchem.0c00770" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 15</span><span class="cit-pageRange">, 8419–8431</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 1, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>6 May 2020</li><li><span class="item_label"><b>Published</b> online</span>1 July 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 13 August 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c00770" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00770</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D8419%26pageCount%3D13%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DDongxing%2BChen%252C%2BCheng%2BDong%252C%2BGuangping%2BDong%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D15%26contentID%3Dacs.jmedchem.0c00770%26title%3DProbing%2Bthe%2BPlasticity%2Bin%2Bthe%2BActive%2BSite%2Bof%2BProtein%2BN-terminal%2BMethyltransferase%2B1%2BUsing%2BBisubstrate%2BAnalogues%26numPages%3D13%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D8431%26publicationDate%3DAugust%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c00770"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">802</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">7</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c00770" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Probing the Plasticity in the Active Site of Protein N-terminal Methyltransferase 1 Using Bisubstrate Analogues&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Dongxing&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Cheng&quot;,&quot;last_name&quot;:&quot;Dong&quot;},{&quot;first_name&quot;:&quot;Guangping&quot;,&quot;last_name&quot;:&quot;Dong&quot;},{&quot;first_name&quot;:&quot;Karthik&quot;,&quot;last_name&quot;:&quot;Srinivasan&quot;},{&quot;first_name&quot;:&quot;Jinrong&quot;,&quot;last_name&quot;:&quot;Min&quot;},{&quot;first_name&quot;:&quot;Nicholas&quot;,&quot;last_name&quot;:&quot;Noinaj&quot;},{&quot;first_name&quot;:&quot;Rong&quot;,&quot;last_name&quot;:&quot;Huang&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;01&quot;,&quot;issue&quot;:&quot;15&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;8419-8431&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c00770&quot;},&quot;abstract&quot;:&quot;The bisubstrate analogue strategy is a promising approach to develop potent and selective inhibitors for protein methyltransferases. Herein, the interactions of a series of bisubstrate analogues with protein N-terminal methyltransferase 1 (NTMT1) were examined to probe the molecular properties of the active site of NTMT1. Our results indicate that a 2-C to 4-C atom linker enables its respective bisubstrate analogue to occupy both substrate- and cofactor-binding sites of NTMT1, but the bisubstrate analogue with a 5-C atom linker only interacts with the substrate-binding site and functions as a substrate. Furthermore, the 4-C atom linker is the optimal and produces the most potent inhibitor (Ki,app = 130 ± 40 pM) for NTMT1 to date, displaying more than 3000-fold selectivity for other methyltransferases and even for its homologue NTMT2. This study reveals the molecular basis for the plasticity of the active site of NTMT1. Additionally, our study outlines general guidance on the development of bisubstrate inhibit&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00770&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00770" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00770&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00770" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00770&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00770" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00770&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00770&amp;href=/doi/10.1021/acs.jmedchem.0c00770" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c00770" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c00770" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00770&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c00770%26sid%3Dliteratum%253Aachs%26pmid%3D32605369%26genre%3Darticle%26aulast%3DChen%26date%3D2020%26atitle%3DProbing%2Bthe%2BPlasticity%2Bin%2Bthe%2BActive%2BSite%2Bof%2BProtein%2BN-terminal%2BMethyltransferase%2B1%2BUsing%2BBisubstrate%2BAnalogues%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D15%26spage%3D8419%26epage%3D8431%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291316" title="Mixtures">Mixtures</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/15" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/jmcmar.2020.63.issue-15/20200813/jmcmar.2020.63.issue-15.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/medium/jm0c00770_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/large/jm0c00770_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00770&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The bisubstrate analogue strategy is a promising approach to develop potent and selective inhibitors for protein methyltransferases. Herein, the interactions of a series of bisubstrate analogues with protein N-terminal methyltransferase 1 (NTMT1) were examined to probe the molecular properties of the active site of NTMT1. Our results indicate that a 2-C to 4-C atom linker enables its respective bisubstrate analogue to occupy both substrate- and cofactor-binding sites of NTMT1, but the bisubstrate analogue with a 5-C atom linker only interacts with the substrate-binding site and functions as a substrate. Furthermore, the 4-C atom linker is the optimal and produces the most potent inhibitor (<i>K</i><sub>i,app</sub> = 130 ± 40 pM) for NTMT1 to date, displaying more than 3000-fold selectivity for other methyltransferases and even for its homologue NTMT2. This study reveals the molecular basis for the plasticity of the active site of NTMT1. Additionally, our study outlines general guidance on the development of bisubstrate inhibitors for any methyltransferases.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47122" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47122" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Protein methylation, a widespread epigenetic modification that is catalyzed by protein methyltransferases (PMTs), plays critical roles in many cellular processes such as gene expression, transcriptional regulation, DNA replication, and damage repair. Dysregulation of PMT expression has been implicated in a variety of diseases including cancer, cardiovascular, neurodegenerative, and other diseases.<a onclick="showRef(event, 'ref1 ref2 ref3 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4 ref5">(1−5)</a> Therefore, there has been emerging interest in the development of selective inhibitors for PMTs as potential therapeutic agents. Recently, several inhibitors of PMTs have been developed and have entered human clinical trials, such as DOT1-like protein inhibitor EPZ-5676 (phase I/II), protein arginine methyltransferase 5 inhibitor GSK3326595 (phase I/II), enhancer of zeste homologue 2 inhibitors EPZ-6438 (phase I/II), and CPI-1205 (phase I/II).<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a></div><div class="NLM_p">PMTs utilize the cofactor <i>S</i>-adenosyl-<span class="smallcaps smallerCapital">l</span>-methionine (SAM) as a methyl donor to transfer the methyl group from SAM onto the substrate with concomitant generation of <i>S</i>-adenosine-<span class="smallcaps smallerCapital">l</span>-homocysteine (SAH) and methylated substrate.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> The PMT-catalyzed methylation reaction typically undergoes a Bi–Bi mechanism and involves the formation of a ternary complex.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Bisubstrate analogues that occupy both substrate- and cofactor-binding sites have the potential to achieve higher binding affinities than a substrate analogue.<a onclick="showRef(event, 'ref7 ref8 ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref7 ref8 ref9 ref10">(7−10)</a> Furthermore, bisubstrate analogues may offer superior selectivity for their targeted enzymes.<a onclick="showRef(event, 'ref9 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref9 ref11 ref12">(9,11,12)</a> On account of a shared SAM cofactor-binding pocket across PMT families, it is rational to hypothesize that bisubstrate analogues can be a general approach to offer potent and specific inhibitors for each individual PMT. Recent progresses have been made in the development of bisubstrate analogues for a few PMTs, such as protein N-terminal methyltransferase 1 (NTMT1),<a onclick="showRef(event, 'ref11 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref11 ref13 ref14">(11,13,14)</a> coactivator-associated arginine methyltransferase 1 (CARM1),<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> protein arginine methyltransferase 1 (PRMT1),<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> and PRMT6.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> These bisubstrate analogues were created by connecting a SAM analogue to a substrate or a functional group of the corresponding substrate. However, the potency of those bisubstrate analogues for their respective targets varied from nanomolar to micromolar. This variable potency imposed the need for explicit instructions for the design of bisubstrate analogues for PMTs.</div><div class="NLM_p">The aforementioned need has prompted us to investigate how the linker length would affect the interaction of bisubstrate analogue to its target PMT. Here, we used NTMT1 as a model system to examine the optimal length of the linker, with the intention to provide a general guide to facilitate the design of optimal bisubstrate analogues for other PMTs. NTMT1 recognizes a unique X-P-K/R motif at the protein α-N-terminus.<a onclick="showRef(event, 'ref18 ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref18 ref19 ref20">(18−20)</a> It methylates centromere-specific histone H3 variant centromere protein A/B, the regulator of chromatin condensation 1, damaged DNA-binding protein 2, and poly(ADP-ribose) polymerase 3.<a onclick="showRef(event, 'ref21 ref22 ref23 ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref21 ref22 ref23 ref24 ref25 ref26">(21−26)</a> Hence, NTMT1 has been implicated in regulating chromosome segregation, mitosis, and DNA damage repair.<a onclick="showRef(event, 'ref22 ref24 ref27'); return false;" href="javascript:void(0);" class="ref ref22 ref24 ref27">(22,24,27)</a> We have previously developed bisubstrate analogues for NTMT1. Among them, compound <b>NAH-C3-PPKRIA</b> showed an IC<sub>50</sub> value of 35 ± 2 nM in a matrix-assisted laser desorption/ionization-mass spectrometry (MALDI-MS)-based assay and displayed more than 100-fold selectivity for other methyltransferases (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A).<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> In addition, compound <b>NAH-C3-GPRRRS</b> showed an IC<sub>50</sub> value of 0.94 ± 0.16 μM for NTMT1.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Both compounds have a propylene linker, which raises the question if a propylene linker is optimal. To address this question, we design and synthesize a series of NTMT1 bisubstrate analogues with variable linker lengths to probe the active site of NTMT1.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/medium/jm0c00770_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/large/jm0c00770_0001.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Chemical structures of NTMT1 inhibitors. (B) Design of the bisubstrate analogues for NTMT1.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/large/jm0c00770_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00770&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec1_1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_55648" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_55648" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec555555" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Design and Synthesis</h3><div class="NLM_p">The linear distance between the sulfur atom of SAH and the nitrogen atom of substrate ranges 4.6–5.5 Å in the ternary structures of NTMT1.<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18,19)</a> A propyl linker was able to offer high potent inhibitor for NTMT1.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Herein, we proposed to vary the linker from 2 to 5 methylene groups to explore the optimal length to build bisubstrate analogues <b>1</b>–<b>7</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B and <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). In addition, the peptide sequence was also altered to ensure that the effect of different linkers was not sequence-specific. We chose the peptide starting with Gly due to its synthetic versatility. Meanwhile, compounds <b>8</b>–<b>12</b> were designed to investigate the contribution of each part of the bisubstrate analogue to the inhibitory activity. The synthesis of compounds <b>1</b>–<b>7</b> and <b>10</b>–<b>12</b> was accomplished in a convergent manner by reacting the primary amines (<b>16</b>–<b>19</b> and <b>24</b>–<b>26</b>) with α-bromo peptide on resin, followed by acid treatment (<a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a> and <a class="ref internalNav" href="#sch2" aria-label="2">2</a>).<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Compounds <b>8</b> and <b>9</b> were obtained by acidic deprotection of <b>14</b> and <b>18</b>, respectively (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). The primary amines <b>18</b> and <b>19</b> were prepared through reductive amination of a common starting material <b>14</b> with the respective aldehydes followed by removal of the protection group (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>).<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28,29)</a> Commercially available amine <b>20</b> was subjected to reductive amination with aldehyde <b>21</b> to provide compound <b>22</b>. The primary amine <b>24</b> was obtained after removal of the Fmoc group of compound <b>23</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>).</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/medium/jm0c00770_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/large/jm0c00770_0009.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthetic Route for Compounds <b>1</b>–<b>9</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/large/jm0c00770_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00770&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaBH<sub>3</sub>CN, CH<sub>3</sub>OH, r.t., 38–68%; (b) 10% Pd/C, H<sub>2</sub>, MeOH, 56%; (c) 1,8-diazabicyclo[5.4.0]undec-7-ene (DBU), CH<sub>3</sub>CN, 0 °C to r.t., 71%; (d) α-bromo peptides, <i>N</i>,<i>N</i>-diisopropylethylamine (DIPEA), dimethylformamide (DMF), r.t.; (e) TFA/DODT/H<sub>2</sub>O/TIPS = 94:2.5:2.5:1; (f) trifluoroacetic acid (TFA), CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to r.t., 42%; and (g) 4N HCl in dioxane, H<sub>2</sub>O, 0 °C to r.t., 45%.</p></p></figure><figure id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/medium/jm0c00770_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/large/jm0c00770_0010.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthetic Route for Compounds <b>10</b>–<b>12</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/large/jm0c00770_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00770&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaBH<sub>3</sub>CN, CH<sub>3</sub>OH, r.t., 71%; (b) (Boc)<sub>2</sub>O, NaHCO<sub>3</sub>, tetrahydrofuran (THF), H<sub>2</sub>O, 85%; (c) DBU, CH<sub>3</sub>CN, 0 °C to r.t., 50%; (d) α-bromo peptides, DIPEA, DMF, r.t.; and (e) TFA/DODT/H<sub>2</sub>O/TIPS = 94: 2.5: 2.5: 1.</p></p></figure></div><div id="sec1_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> MALDI-MS Methylation Assay</h3><div class="NLM_p">Compounds <b>1</b>–<b>4</b> were first subjected to the methylation assay catalyzed by NTMT1 in the presence of SAM to first check if they may act as possible substrates. Surprisingly, compound <b>4</b> containing the 5-C atom linker was able to be methylated by NTMT1 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>), indicating <b>4</b> as a substrate for NTMT1.</div><figure id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/medium/jm0c00770_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/large/jm0c00770_0002.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. MALDI-MS methylation assay for bisubstrate analogues <b>1</b>–<b>4</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/large/jm0c00770_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00770&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec1_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Inhibition Evaluation</h3><div class="NLM_p">The inhibitory activities of compounds <b>1</b>–<b>3</b> and <b>5</b>–<b>7</b> were evaluated in the SAH hydrolase (SAHH)-coupled fluorescence assay with 10 min of preincubation time.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> Both SAM and the peptide substrate GPKRIA were at their respective <i>K</i><sub>m</sub> values. To compare the inhibitory activity under the same condition, we resynthesized and purified <b>1</b>, <b>2</b>, and <b>6</b>, respectively. Their IC<sub>50</sub> values were lower than the previous values,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> which may be caused by an improved S/N ratio with a different GPKRIA peptide substrate and fluorescent dye ThioGlo4. Among the NAH-GPKRIA series, compounds <b>3</b> (IC<sub>50</sub> = 82 ± 17 nM) containing a 4-C atom length displayed the best inhibition for NTMT1 (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). When the linker length decreased from a 4-C atom to a 3-C and 2-C, compounds <b>2</b> (IC<sub>50</sub> = 130 ± 40 nM) and <b>1</b> (IC<sub>50</sub> = 260 ± 70 nM) exhibited a 1.5- and 3-fold decrease compared to <b>3</b>, respectively. Therefore, a 4-C atom length is optimal for the NAM-GPKRIA series. To further validate the optimal linker length, we synthesized three additional bisubstrate analogues that contained a tetrapeptide instead of hexapeptide in the substrate portion. For NAH-GPKR bisubstrate analogues, compounds <b>5</b> and <b>6</b> showed less than twofold activity decrease compared to their respective NAH-GPKRIA compounds <b>1</b> and <b>2</b>, while compound <b>7</b> displayed comparable inhibition with compound <b>3</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Furthermore, the 4-C atom linker is also the best inhibitor, as <b>7</b> (IC<sub>50</sub> = 84 ± 12 nM) showed two- and fourfold better activity than <b>6</b> and <b>5</b>, respectively. Therefore, the 4-C atom linker is the most favorable length for NTMT1 bisubstrate analogues in both the NAH-GPKRIA and NAH-GPKR series.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. IC<sub>50</sub> Values of Bisubstrate Compounds</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound ID</th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1 (NAH-C2-GPKRIA)</b></td><td class="colsep0 rowsep0" align="left">260 ± 70</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2 (NAH-C3-GPKRIA)</b></td><td class="colsep0 rowsep0" align="left">130 ± 40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3 (NAH-C4-GPKRIA)</b></td><td class="colsep0 rowsep0" align="left">82 ± 17</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4 (NAH-C5-GPKRIA)</b></td><td class="colsep0 rowsep0" align="left">NA</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5 (NAH-C2-GPKR)</b></td><td class="colsep0 rowsep0" align="left">320 ± 80</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6 (NAH-C3-GPKR)</b></td><td class="colsep0 rowsep0" align="left">200 ± 30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7 (NAH-C4-GPKR)</b></td><td class="colsep0 rowsep0" align="left">84 ± 12</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Triplicates (<i>n</i> = 3). NA, not applied.</p></div></div></div><div class="NLM_p">The bisubstrate analogues contained three parts: a SAM analogue <b>8</b> (NAH), a linker, and a peptide substrate. To understand the contribution of each component of the bisubstrate analogue to the inhibitory activity for NTMT1, a deconstruction strategy was applied to design and synthesize only partial structures of the bisubstrate analogue (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>): (1) compound <b>8</b> contained only the NAH part, (2) compound <b>9</b> with only the NAH and linker, (3) compound <b>10</b> without the adenine portion, (4) compound <b>11</b> without the NAH part, and (5) compound <b>12</b> without the aspartic acid moiety of the SAH. Compounds <b>10</b>–<b>12</b> were determined to be substrates for NTMT1 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00770/suppl_file/jm0c00770_si_002.pdf" class="ext-link">Figure S1</a>). Compound <b>9</b> displayed an IC<sub>50</sub> value higher than 10 μM in the SAHH-coupling assay. Since <b>8</b> is a potent inhibitor (IC<sub>50</sub> = 23 ± 2 nM) for SAHH, it does not exhibit significant inhibition against NTMT1 (IC<sub>50</sub> > 10 μM) through an orthogonal MALDI-MS inhibition assay (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00770/suppl_file/jm0c00770_si_002.pdf" class="ext-link">Figure S2</a>). These results suggested that each part of the bisubstrate inhibitor was critical for its inhibitory activity for NTMT1.</div><figure id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/medium/jm0c00770_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/large/jm0c00770_0003.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Deconstruction strategy and inhibition evaluation of the compounds. The IC<sub>50</sub> value of <b>8</b> was determined by the MALDI-MS methylation inhibition assay, while <b>9</b> was determined by the SAHH-coupling assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/large/jm0c00770_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00770&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec1_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Time-Dependent Inhibition Study</h3><div class="NLM_p">The measured IC<sub>50</sub> values for compounds <b>2</b>, <b>3</b>, and <b>7</b> converged to the enzyme concentration (100 nM) present in the assay (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), suggesting tight-binding inhibition. Therefore, Morrison’s equation is a better method to analyze the concentration–response data for these tight-binding inhibitors to define their apparent <i>K</i><sub>i</sub> values.<a onclick="showRef(event, 'ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref31 ref32">(31,32)</a> The tight-binding inhibitors often exhibit slow binding traits, so compounds <b>1</b>–<b>3</b> were chosen to investigate the effects of preincubation time on the inhibitory activity of tight-binding inhibitors (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>).<a onclick="showRef(event, 'ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref31 ref32">(31,32)</a> The <i>K</i><sub>i,app</sub> values of the three compounds were inversely proportional to the preincubation time (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B,D,F), indicating their slow binding feature. The results showed the interaction between the compounds and NTMT1 reached equilibrium after 30 min of incubation. Compound <b>3</b> containing a 4-C atom linker displayed the lowest <i>K</i><sub>i,app</sub> value (130 ± 40 pM) (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B) after 120 min, further substantiating its slow and tight binding feature ([E]/<i>K</i><sub>i,app</sub> > 100).<a onclick="showRef(event, 'ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref31 ref32">(31,32)</a> When the linker length decreased from a 4-C atom to a 3-C and 2-C, compounds <b>2</b> (<i>K</i><sub>i,app</sub> = 1.3 ± 0.6 nM) and <b>1</b> (<i>K</i><sub>i,app</sub> = 5.4 ± 2.6 nM) exhibited a 10- and 40-fold decrease in comparison with <b>3</b>, respectively. The above results infer decent plasticity in the transition state of NTMT1 to accommodate a linker length ranging from 2-C to 4-C atom. Among them, a 4-C atom linker is the optimal length to achieve the best inhibition. Notably, a 5-C atom linker would convert a tight-binding bisubstrate analogue into a substrate for NTMT1 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>), indicating an upper limit for the linker length to achieve potent inhibition.</div><figure id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/medium/jm0c00770_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/large/jm0c00770_0004.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Time-dependent inhibition study of compounds <b>1</b>–<b>3</b>. All experiments were run in duplicate (<i>n</i> = 2), except for 10 min of preincubation (<i>n</i> = 3). (A), (C), and (E) Concentration–response plots for different preincubation times fitted to Morrison’s quadratic equation for compounds <b>1</b>, <b>2</b>, and <b>3</b>, respectively. (B), (D), and (F) Nonlinear regression of Morrison <i>K</i><sub>i,app</sub> values with preincubation times for compounds <b>1</b>, <b>2</b>, and <b>3</b>, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/large/jm0c00770_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00770&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec1_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Isothermal Titration Calorimetry (ITC)</h3><div class="NLM_p">NTMT2 is a closely related homologue of NTMT1, which shares more than 50% sequence similarity and the same X-P-K recognition motif.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> To assess the interactions of the bisubstrate analogues with both enzymes, the binding affinities of <b>1</b>–<b>4</b> for both NTMT1 and NTMT2 were determined by the isothermal titration calorimetry (ITC) experiment (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). All four compounds bound to NTMT1, especially compound <b>3</b> (<i>K</i><sub>d1/d2</sub> = 0.31 μM/2.47 nM), showing the tightest binding affinity. Notably, the <i>K</i><sub>d</sub> values of bisubstrate analogues were higher than their <i>K</i><sub>i,app</sub> values, except for compound <b>4</b> (<i>K</i><sub>d1/d2</sub> = 1.6/0.48 μM). This difference may be caused by endogenous amounts of SAM/SAH binding to recombinant NTMTs and slow binding characteristic of tight-binding inhibitors <b>1</b>–<b>3</b>.<a onclick="showRef(event, 'ref18 ref33'); return false;" href="javascript:void(0);" class="ref ref18 ref33">(18,33)</a> However, the ITC binding data was still informative since the bisubstrate analogues at 1 mM showed negligible interactions with NTMT2.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> This delicate selectivity further corroborates the benefits of bisubstrate analogues on account of their high specificity.</div><figure id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/medium/jm0c00770_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/large/jm0c00770_0005.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Binding affinity for NTMT1 (A) and NTMT2 (B). ND, not detected.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/large/jm0c00770_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00770&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec1_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> MALDI-MS Methylation Inhibition Assay</h3><div class="NLM_p">To further confirm the inhibitory activity of bisubstrate inhibitor, the MALDI-MS methylation assay was performed as an orthogonal assay to monitor the production of methylated peptides with both substrate peptide GPKRIA and SAM at their <i>K</i><sub>m</sub> values after 10 min of preincubation time (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A,B). At 50 nM compound <b>3</b>, trimethylation of the GPKRIA peptide was substantially reduced. At 200 nM compound <b>3</b>, trimethylation of the GPKRIA peptide was barely detected (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B).</div><figure id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/medium/jm0c00770_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/large/jm0c00770_0006.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (A, B) MALDI-MS methylation inhibition assay for compound <b>3</b> (<i>n</i> = 2). (A) MALDI-MS results of MALDI-MS methylation inhibition assay for compound <b>3</b> and (B) quantification of methylation progression of GPKRIA by NTMT1 with <b>3</b> at 20 min. (C) Selectivity evaluation of compound <b>3</b> (<i>n</i> = 2).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/large/jm0c00770_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00770&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec1_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Selectivity Assessment</h3><div class="NLM_p last">The selectivity of <b>3</b> for NTMT1 was further examined against a panel of methyltransferases that shared a SAM-binding pocket, which also included the coupling enzyme SAHH that has a SAH-binding site and was used in the fluorescence assay (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00770/suppl_file/jm0c00770_si_002.pdf" class="ext-link">Table S1</a>). The compound barely showed inhibitory activity over euchromatic histone-lysine <i>N</i>-methyltransferase 2 (G9a), SET domain-containing protein 7 (SETD7), and nicotinamide <i>N</i>-methyltransferase (NNMT) at 100 μM. At 33 μM, <b>3</b> demonstrated less than 50% inhibition over protein arginine methyltransferase 1 (PRMT1), protein arginine methyltransferase 3 (PRMT3), <i>Trypanosoma brucei</i> protein arginine methyltransferase 7 (<i>Tb</i>PRMT7), and SAHH. Therefore, compound <b>3</b> showed more than 3000-fold selectivity for NTMT1 over other methyltransferases.</div></div><div id="sec1_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Cocrystal Structures of Compounds <b>1</b> and <b>4</b> with NTMT1</h3><div class="NLM_p">To understand the molecular basis of the effect of different linkers, we obtained the cocrystal structures of bisubstrate analogues <b>1</b> and <b>4</b> with NTMT1 (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>). The binary complex X-ray crystal structure of the NTMT1–<b>1</b> complex (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WJ7">6WJ7</a>) illuminates the inhibition mechanism since compound <b>1</b> does occupy both cofactor- and substrate-binding sites of NTMT1. It also overlays well with previously reported bisubstrate inhibitor <b>NAH-C3-PPKRIA</b> in the X-ray crystal structure of NTMT1-<b>NAH-C3-PPKRIA</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DTN">6DTN</a>) (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>B). The SAM analogue moiety of <b>1</b> in the binary complex binds almost the same as <b>NAH-C3-PPKRIA</b>, which is nearly identical to SAH in the ternary complex of substrate peptide/SAH with NTMT1. When increasing the carbon linker to 5-C atom linker, the peptide part of the compound <b>4</b> retains its interaction with the substrate-binding pocket and overlays very well with the PPKRIA peptide portion of <b>NAH-C3-PPKRIA</b> (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>D–F). However, the NAH moiety of compound <b>4</b> no longer occupied the SAM-binding pocket. Interestingly, the SAM analogue part inserted into a new pocket adjacent to the substrate-binding pocket, which resulted in the exposure of the secondary amine (α-N-terminal amine of the peptide portion) of compound <b>4</b> to receive the methyl group from SAM. Nevertheless, it infers that the length of the 5-C atom linker is too long to allow the bisubstrate analogue <b>4</b> to simultaneously occupy both SAM- and peptide-binding sites.</div><figure id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/medium/jm0c00770_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/large/jm0c00770_0007.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. X-ray cocrystal structures. (A) X-ray crystal structure of compound <b>1</b> (purple stick) in complex with NTMT1 (gray ribbon) (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WJ7">6WJ7</a>). H-bond interactions are shown in yellow dotted lines. (B) Overlay of the crystal structure of NTMT1 (gray ribbon)–<b>1</b> (purple stick) complex with the crystal structure of the NTMT1-<b>NAH-C3-PPKRIA</b> (orange stick) binary complex (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DTN">6DTN</a>). (C) compound <b>1</b> two-dimensional (2D) interaction diagram (Schrödinger Maestro) with NTMT1. (D) X-ray crystal structure of NTMT1 (gray ribbon)–<b>4</b> (yellow stick)–SAH (gray stick) (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PVB">6PVB</a>). H-bond interactions are shown in yellow dotted lines. (E) Overlay of the crystal structure of NTMT1 (gray ribbon)–<b>4</b> (yellow stick)–SAH (gray stick) complex (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PVB">6PVB</a>) with the crystal structure of the NTMT1-<b>NAH-C3-PPKRIA</b> (orange stick) binary complex (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DTN">6DTN</a>). (F) Compound <b>4</b> 2D interaction diagram (Schrödinger Maestro) with NTMT1.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/large/jm0c00770_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00770&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec1_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Inhibition Mechanism Studies</h3><div class="NLM_p">Despite additional efforts to screen and optimize the conditions, the cocrystal structure of NTMT1 in complex with <b>2</b> or <b>3</b> was not obtained. Hence, inhibition mechanism studies were performed to confirm the binding modes for 4-C atom linker compound. Since the cocrystal structures of <b>NAH-C3-PPKRIA</b> containing the 3-C linker revealed that the compound could simultaneously occupy SAM- and peptide-binding sites, it was reasonable to assume that compound <b>2</b> with 3-C linker could also interact with both SAM- and peptide-binding sites. So, inhibition mechanism studies for compound <b>3</b> were performed along with compounds <b>1</b> and <b>2</b> as the control under the condition of 2 h of preincubation (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>). Even at both 6 and 12<i>K</i><sub>m</sub> concentrations of the peptide substrate, there was slight competition for compound <b>3</b>. Moreover, ∼10% of enzymatic activity was recovered in the presence of 0.3 μM or higher concentrations of compounds <b>2</b> and <b>3</b>. When the competing peptide substrate concentration was increased to 50<i>K</i><sub>m</sub>, a right shift of the IC<sub>50</sub> curves was observed for <b>2</b> and <b>3</b>. In addition, ∼20% enzymatic activity was recovered, further reflecting their tight and slow binding character. Upon increasing the concentration of SAM from 6 to 25<i>K</i><sub>m</sub>, similar trends were observed. The catalytic activity of NTMT1 could be recovered with a higher concentration of SAM or peptide, suggesting that the compounds could be competitive with the peptide and SAM. These results indicated that all three bisubstrate inhibitors <b>1</b>–<b>3</b> occupied both SAM- and peptide substrate-binding sites in a reversible manner.</div><figure id="fig8" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/medium/jm0c00770_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/large/jm0c00770_0008.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Inhibition mechanism studies for compounds <b>1</b>–<b>3</b>. (A–C) IC<sub>50</sub> curves of <b>1</b>–<b>3</b> at varying concentrations of the peptide with a fixed concentration of SAM. (D–F) IC<sub>50</sub> curves of <b>1</b>–<b>3</b> at varying concentrations of SAM with a fixed concentration of the peptide. The preincubation time for the assay was 120 min, and all of the experiments were run in duplicates (<i>n</i> = 2).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/large/jm0c00770_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00770&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_57623" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_57623" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">This series of tight-binding and selective bisubstrate analogues to NTMT1 represent valuable tools to shed light on the plasticity in the transition state of NTMT1 since it can accommodate a linker length from 2-C to 4-C atom. Among them, a 4-C atom linker of compound <b>3</b> is the optimal length to achieve the best inhibition with a <i>K</i><sub>i,app</sub> value of 130 pM. Notably, a 5-C atom linker of compound <b>4</b> would convert a tight-binding bisubstrate analogue to a substrate for NTMT1, suggesting the upper limit of the aforementioned plasticity. The cocrystal structure of <b>4</b> in complex with NTMT1 (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PVB">6PVB</a>) implied that extending the linker to a 5-C atom linker may cause tension to prevent the bisubstrate analogue from engaging with both binding sites. Given the transient nature of the transition state, these bisubstrate analogues provide dynamic snapshots of the reorganization within the active site of NTMT1 during catalysis. Furthermore, top inhibitor <b>3</b> (<b>NAH-C4-GPKRIA</b>) containing a 4-C atom linker showed picomolar inhibition (<i>K</i><sub>i,app</sub> = 130 ± 40 pM) for NTMT1 and more than 100 000-fold selectivity for other MTases. In addition, all bisubstrate analogues <b>1</b>–<b>3</b> demonstrate no interaction with its close homologue NTMT2 despite similar catalytic sites of NTMT1/2.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> This delicate selectivity further corroborates the benefits of bisubstrate analogues on account of their high specificity to offer tight and selective inhibitors for an individual target. However, increasing the carbon linker length to a 5-C atom resulted in a substrate instead of an inhibitor for NTMT1, which defined the scope or limit of plasticity. To our knowledge, this is the first case that delineates the optimal linker range of methyltransferase (MTase) bisubstrate analogue and provides the molecular basis for the obliterated activity for longer linkers.</div><div class="NLM_p last">The linear distance between the sulfonium ion and the nitrogen atom that accept a methyl group is very similar for most PMTs, which ranges from 3.5 to 5.5 Å among the ternary complex of all examined PMTs including NTMT1, SET domain-containing protein 8, and euchromatic histone-lysine <i>N</i>-methyltransferase 1.<a onclick="showRef(event, 'ref18 ref19 ref34 ref35 ref36 ref37'); return false;" href="javascript:void(0);" class="ref ref18 ref19 ref34 ref35 ref36 ref37">(18,19,34−37)</a> Therefore, we propose that a 3-C or 4-C atom linker would serve as a general and feasible starting point to build bisubstrate analogues for any MTase. The optimal linker length has also been corroborated by successful inhibitors for RNA MTase, as a 3-C atom linker showed better inhibition compared to a 2-C atom linker.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Small-molecule inhibitors have been reported targeting either SAM- or substrate-binding sites for MTase.<a onclick="showRef(event, 'ref2 ref39'); return false;" href="javascript:void(0);" class="ref ref2 ref39">(2,39)</a> Future directions will focus on applying such linkers to conjugate small-molecule inhibitors that target the substrate-binding site with SAM mimic or SAM competitive small-molecule inhibitors to achieve higher selectivity and cellular potency for the respective methyltransferases.</div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_83777" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_83777" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> General Chemical Procedures</h3><div class="NLM_p">The reagents and solvents were purchased from commercial sources (Fisher) and used directly. The intermediates and products were purified using an ISCO CombiFlash system and prepacked SiO<sub>2</sub> cartridges. Final compounds were purified on preparative high-pressure liquid chromatography (RP-HPLC) performed on an Agilent 1260 Series system. Systems were run with 0–20% methanol/water gradient with 0.1% TFA. NMR spectra were acquired on a Bruker AV500 instrument (500 MHz for <sup>1</sup>H NMR, 126 MHz for <sup>13</sup>C NMR). Thin-layer chromatography–mass spectrometry (TLC–MS) was performed using an Advion CMS-L MS. Matrix-assisted laser desorption ionization-mass spectrometry (MALDI-MS) data were acquired in positive-ion mode using a Sciex 4800 MALDI TOF/TOF MS. The peptides were synthesized using a CEM Liberty Blue automated microwave peptide synthesizer with the manufacturers’ standard coupling cycles at a 0.1 mmol scale. The purity of final compounds was confirmed by a Waters LC–MS system and/or an Agilent 1260 Series HPLC system. Systems were run with 0–40% methanol/water gradient with 0.1% TFA. All target compounds showed purity >95% in RP-HPLC.</div><div id="sec3_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> α-Bromo Peptides (G*PKRIA or G*PKR)</h4><div class="NLM_p">Bromo acetic acid (139 mg, 1 mmol) was activated by DIC (155 μL, 1 mmol) for 20 min. Then, the solution was added into the peptide PKRIA or PKR on resin (0.1 mmol). The mixture was shaken overnight at r.t. The suspension was filtered, and the resin was washed with DMF (3 mL × 3). The procedures were repeated again. Test cleavage (TFA/TIPS/H<sub>2</sub>O = 95:2.5:2.5) and MALDI-MS were performed to check the progress of the reaction.</div><div id="sec3_1_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i20" class="anchor-spacer"></div><h5 class="article-section__title" id="_i20"> NAH-C2-GPKRIA (<b>1</b>)</h5><div class="NLM_p last">To a suspension of G*PKRIA on a resin (0.1 mmol) in DMF (3 mL) was added compound <b>8</b> (91 mg, 0.15 mmol) and DIPEA (26 μL, 0.15 mmol). The mixture was shaken at room temperature overnight. The suspension was filtered, and the resin was subsequently washed with DMF (3 mL × 3), MeOH (3 mL × 3), and CH<sub>2</sub>Cl<sub>2</sub> (3 mL × 3). The peptide conjugates were cleaved from the resin in the presence of a cleavage cocktail (10 mL) containing trifluoroacetic acid (TFA)/2,2′-(ethylenedioxy)-diethanethiol (DODT)/triisopropylsilane (TIPS)/water (94:2.5:1:2.5 v/v) at room temperature for 4–5 h, and the resin was rinsed with a small amount of TFA. Volatiles of the filtrate were removed under N<sub>2</sub>, and the residue was precipitated with 10 vol. cold anhydrous ether and collected by centrifugation. The supernatant was discarded, and the pellet was washed with chilled ether and air-dried. The white precipitate was dissolved in ddH<sub>2</sub>O and purified by reverse-phase HPLC using a Waters system with an XBridge (BEH C18, 5 μm, 10 × 250 mm<sup>2</sup>) column and 0.1% TFA in water (A) and MeOH (B) as the mobile phase with monitoring at 214 and 254 nm. MALDI-MS (positive) <i>m</i>/<i>z</i>: calcd for C<sub>44</sub>H<sub>77</sub>N<sub>18</sub>O<sub>11</sub> [M + H]<sup>+</sup><i>m</i>/<i>z</i> 1033.6019, found m/z 1033.7422.</div></div><div id="sec3_1_1_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i21" class="anchor-spacer"></div><h5 class="article-section__title" id="_i21"> <b>NAH-C3-GPKRIA</b> (<b>2</b>)</h5><div class="NLM_p last">The compound was prepared according to the procedure followed for <b>NAH-C2-GPKRIA</b> (<b>1</b>) and purified by reverse-phase HPLC. MALDI-MS (positive) <i>m</i>/<i>z</i>: calcd for C<sub>45</sub>H<sub>79</sub>N<sub>18</sub>O<sub>11</sub> [M + H]<sup>+</sup><i>m</i>/<i>z</i> 1047.6176, found <i>m</i>/<i>z</i> 1047.8483.</div></div><div id="sec3_1_1_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i22" class="anchor-spacer"></div><h5 class="article-section__title" id="_i22"> <b>NAH-C4-GPKRIA</b> (<b>3</b>)</h5><div class="NLM_p last">The compound was prepared according to the procedure followed for <b>NAH-C2-GPKRIA</b> (<b>1</b>) and purified by reverse-phase HPLC. MALDI-MS (positive) <i>m</i>/<i>z</i>: calcd for C<sub>46</sub>H<sub>81</sub>N<sub>18</sub>O<sub>11</sub> [M + H]<sup>+</sup><i>m</i>/<i>z</i> 1061.6332, found <i>m</i>/<i>z</i> 1062.0251.</div></div><div id="sec3_1_1_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i23" class="anchor-spacer"></div><h5 class="article-section__title" id="_i23"> <b>NAH-C5-GPKRIA</b> (<b>4</b>)</h5><div class="NLM_p last">The compound was prepared according to the procedure followed for <b>NAH-C2-GPKRIA</b> (<b>1</b>) and purified by reverse-phase HPLC. MALDI-MS (positive) <i>m</i>/<i>z</i>: calcd for C<sub>46</sub>H<sub>81</sub>N<sub>18</sub>O<sub>11</sub> [M + H]<sup>+</sup><i>m</i>/<i>z</i> 1075.6489, found <i>m</i>/<i>z</i> 1075.6666.</div></div><div id="sec3_1_1_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i24" class="anchor-spacer"></div><h5 class="article-section__title" id="_i24"> <b>NAH-C2-GPKR</b> (<b>5</b>)</h5><div class="NLM_p last">The compound was prepared according to the procedure followed for <b>NAH-C2-GPKRIA</b> (<b>1</b>) and purified by reverse-phase HPLC. MALDI-MS (positive) <i>m</i>/<i>z</i>: calcd for C<sub>35</sub>H<sub>61</sub>N<sub>16</sub>O<sub>9</sub> [M + H]<sup>+</sup><i>m</i>/<i>z</i> 849.4807, found <i>m</i>/<i>z</i> 849.7723.</div></div><div id="sec3_1_1_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i25" class="anchor-spacer"></div><h5 class="article-section__title" id="_i25"> <b>NAH-C3-GPKR</b> (<b>6</b>)</h5><div class="NLM_p last">The compound was prepared according to the procedure followed for <b>NAH-C2-GPKRIA</b> (<b>1</b>) and purified by reverse-phase HPLC. MALDI-MS (positive) <i>m</i>/<i>z</i>: calcd for C<sub>36</sub>H<sub>63</sub>N<sub>16</sub>O<sub>9</sub> [M + H]<sup>+</sup><i>m</i>/<i>z</i> 863.4964, found <i>m</i>/<i>z</i> 863.6786.</div></div><div id="sec3_1_1_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i26" class="anchor-spacer"></div><h5 class="article-section__title" id="_i26"> <b>NAH-C4-GPKR</b> (<b>7</b>)</h5><div class="NLM_p last">The compound was prepared according to the procedure followed for <b>NAH-C2-GPKRIA</b> (<b>1</b>) and purified by reverse-phase HPLC. MALDI-MS (positive) <i>m</i>/<i>z</i>: calcd for C<sub>37</sub>H<sub>65</sub>N<sub>16</sub>O<sub>9</sub> [M + H]<sup>+</sup><i>m</i>/<i>z</i> 877.5120, found <i>m</i>/<i>z</i> 877.7390.</div></div><div id="sec3_1_1_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i27" class="anchor-spacer"></div><h5 class="article-section__title" id="_i27"> (<i>S</i>)-2-Amino-4-((((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>S</i>)-5-(6-amino-9H-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)amino)butanoic Acid (<b>8</b>)</h5><div class="NLM_p last">To a solution of compound <b>14</b> (10mg, 0.018 mmol) in CH<sub>2</sub>Cl<sub>2</sub> was added TFA under an ice bath and stirred for 1 h. TFA was removed in vacuum, and the residue was purified by cation ion-exchange chromatography (0.1 M NH<sub>4</sub>HCO<sub>3</sub> aq solution). After lyophilization, 2.7 mg (41%) of white solid was obtained. <sup>1</sup>H NMR (500 MHz, deuterium oxide) δ 8.13 (dt, <i>J</i> = 12.4, 4.0 Hz, 2H), 5.95 (d, <i>J</i> = 5.0 Hz, 1H), 4.80–4.73 (m, 1H), 4.27 (dt, <i>J</i> = 19.5, 4.5 Hz, 2H), 3.62 (t, <i>J</i> = 6.5 Hz, 1H), 3.27 (dd, <i>J</i> = 43.7, 9.1 Hz, 2H), 3.02 (t, <i>J</i> = 7.3 Hz, 2H), 2.00 (ddd, <i>J</i> = 29.3, 14.8, 7.7 Hz, 2H). <sup>13</sup>C NMR (126 MHz, deuterium oxide) δ 174.19, 155.63, 152.75, 148.71, 140.75, 119.20, 88.61, 80.84, 72.94, 71.43, 53.29, 49.51, 45.30, 28.15. MALDI-MS (positive) <i>m</i>/<i>z</i>: calcd for C<sub>14</sub>H<sub>22</sub>N<sub>7</sub>O<sub>5</sub> [M + H]<sup>+</sup><i>m</i>/<i>z</i> 368.1682, found <i>m</i>/<i>z</i> 368.2080.</div></div><div id="sec3_1_1_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i28" class="anchor-spacer"></div><h5 class="article-section__title" id="_i28"> (<i>S</i>)-2-Amino-4-((((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>S</i>)-5-(6-amino-9H-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)(4-aminobutyl)amino)butanoic Acid (<b>9</b>)</h5><div class="NLM_p last">To a suspension of compound <b>18</b> (20 mg, 0.032 mmol) in ddH<sub>2</sub>O was added 4N HCl in dioxane under an ice bath. Then, the suspension was stirred for 2 h at r.t. The solvent was removed in vacuum, and the residue was purified by cation ion-exchange chromatography (0.1 M NH<sub>4</sub>HCO<sub>3</sub> aq solution). After lyophilization, 6 mg (45%) of white solid was obtained. <sup>1</sup>H NMR (500 MHz, deuterium oxide) δ 8.23–8.10 (m, 2H), 5.98–5.92 (m, 1H), 4.27–4.09 (m, 2H), 3.50–3.38 (m, 1H), 2.85–2.68 (m, 4H), 2.66–2.55 (m, 2H), 2.52–2.39 (m, 2H), 1.91–1.64 (m, 2H), 1.51–1.27 (m, 4H). MALDI-MS (positive) <i>m</i>/<i>z</i>: calcd for C<sub>18</sub>H<sub>31</sub>N<sub>8</sub>O<sub>5</sub> [M + H]<sup>+</sup><i>m</i>/<i>z</i> 439.2417, found <i>m</i>/<i>z</i> 439.3303.</div></div><div id="sec3_1_1_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i29" class="anchor-spacer"></div><h5 class="article-section__title" id="_i29"> (<i>S</i>)-2-Amino-4-((4-((2-((<i>S</i>)-2-(((<i>S</i>)-6-amino-1-(((<i>S</i>)-1-(((2<i>S</i>,3<i>S</i>)-1-(((<i>S</i>)-1-amino-1-oxopropan-2-yl)amino)-3-methyl-1-oxopentan-2-yl)amino)-5-guanidino-1-oxopentan-2-yl)amino)-1-oxohexan-2-yl)carbamoyl)pyrrolidin-1-yl)-2-oxoethyl)amino)butyl)amino)butanoic Acid (<b>10</b>)</h5><div class="NLM_p last">The compound was prepared according to the procedure followed for <b>NAH-C2-GPKRIA</b> (<b>1</b>) and purified by reverse-phase HPLC. MALDI-MS (positive) <i>m</i>/<i>z</i>: calcd for C<sub>36</sub>H<sub>70</sub>N<sub>13</sub>O<sub>8</sub> [M + H]<sup>+</sup><i>m</i>/<i>z</i> 812.5470, found <i>m</i>/<i>z</i> 812.7323.</div></div><div id="sec3_1_1_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i30" class="anchor-spacer"></div><h5 class="article-section__title" id="_i30"> (<i>S</i>)-<i>N</i>-((<i>S</i>)-6-Amino-1-(((<i>S</i>)-1-(((2<i>S</i>,3<i>S</i>)-1-(((<i>S</i>)-1-amino-1-oxopropan-2-yl)amino)-3-methyl-1-oxopentan-2-yl)amino)-5-guanidino-1-oxopentan-2-yl)amino)-1-oxohexan-2-yl)-1-((4-aminobutyl)glycyl)pyrrolidine-2-carboxamide (<b>11</b>)</h5><div class="NLM_p last">The compound was prepared according to the procedure followed for <b>NAH-C2-GPKRIA</b> (<b>1</b>) and purified by reverse-phase HPLC. MALDI-MS (positive) <i>m</i>/<i>z</i>: calcd for C<sub>32</sub>H<sub>63</sub>N<sub>12</sub>O<sub>6</sub> [M + H]<sup>+</sup><i>m</i>/<i>z</i> 711.4994, found <i>m</i>/<i>z</i> 711.5203.</div></div><div id="sec3_1_1_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i31" class="anchor-spacer"></div><h5 class="article-section__title" id="_i31"> (<i>S</i>)-<i>N</i>-((S)-6-Amino-1-(((S)-1-(((2<i>S</i>,3<i>S</i>)-1-(((<i>S</i>)-1-amino-1-oxopropan-2-yl)amino)-3-methyl-1-oxopentan-2-yl)amino)-5-guanidino-1-oxopentan-2-yl)amino)-1-oxohexan-2-yl)-1-((3-((((2<i>S</i>,3<i>S</i>,4<i>R</i>,5<i>S</i>)-5-(6-amino-9H-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)thio)propyl)glycyl)pyrrolidine-2-carboxamide (<b>12</b>)</h5><div class="NLM_p last">The compound was prepared according to the procedure followed for <b>NAH-C2-GPKRIA</b> (<b>1</b>) and purified by reverse-phase HPLC. MALDI-MS (positive) <i>m</i>/<i>z</i>: calcd for C<sub>41</sub>H<sub>71</sub>N<sub>16</sub>O<sub>9</sub>S [M + H]<sup>+</sup><i>m</i>/<i>z</i> 963.5311, found <i>m</i>/<i>z</i> 963.6245.t</div></div><div id="sec3_1_1_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i32" class="anchor-spacer"></div><h5 class="article-section__title" id="_i32"> <i>tert</i>-Butyl (<i>S</i>)-4-((((3a<i>R</i>,4<i>R</i>,6<i>S</i>,6a<i>R</i>)-6-(6-amino-9H-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)(4-(((benzyloxy)carbonyl)amino)butyl)amino)-2-((<i>tert</i>-butoxycarbonyl)amino)butanoate (<b>15a</b>)</h5><div class="NLM_p last">To a solution of <i>tert</i>-butyl (<i>S</i>)-4-((((3a<i>R</i>,4<i>R</i>,6<i>S</i>,6a<i>R</i>)-6-(6-amino-9H-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)amino)-2-((<i>tert</i>-butoxycarbonyl)amino)butanoate (<b>14</b>, 0.5 g, 0.89 mmol) in methanol, benzyl (4-oxobutyl)carbamate (<b>13a</b>, 0.26 g, 1.15 mmol) was added into the reaction. The mixture was stirred for 20 min at r.t. Then, NaBH<sub>3</sub>CN (82 mg, 1.34 mmol) was added in portions, and the reaction mixture was continuously stirred for 6 h at r.t. After removing the solvent in vacuum, the residue was purified by column chromatography (CH<sub>2</sub>Cl<sub>2</sub>/MeOH = 30:1) to obtain 0.46 g (68%) of white solid. <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.25 (s, 1H), 8.22 (s, 1H), 7.39–7.24 (m, 5H), 6.16 (d, <i>J</i> = 2.4 Hz, 1H), 5.53–5.49 (m, 1H), 5.05 (s, 2H), 5.04–5.01 (m, 1H), 4.37–4.28 (m, 1H), 4.09–4.02 (m, 1H), 3.14–3.00 (m, 2H), 2.89–2.75 (m, 1H), 2.68–2.54 (m, 2H), 2.53–2.31 (m, 3H), 1.95–1.84 (m, 1H), 1.72–1.61 (m, 1H), 1.58 (s, 3H), 1.45–1.44 (m, 2H), 1.43 (s, 9H), 1.42 (s, 9H), 1.38 (s, 3H).</div></div><div id="sec3_1_1_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i33" class="anchor-spacer"></div><h5 class="article-section__title" id="_i33"> <i>tert</i>-Butyl (<i>S</i>)-4-((5-((((9H-fluoren-9-yl)methoxy)carbonyl)amino)pentyl)(((3a<i>R</i>,4<i>R</i>,6<i>S</i>,6a<i>R</i>)-6-(6-amino-9H-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)amino)-2-((<i>tert</i>-butoxycarbonyl)amino)butanoate (<b>15b</b>)</h5><div class="NLM_p last">To a solution of <i>tert</i>-butyl (<i>S</i>)-4-((((3a<i>R</i>,4<i>R</i>,6<i>S</i>,6a<i>R</i>)-6-(6-amino-9H-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)amino)-2-((<i>tert</i>-butoxycarbonyl)amino)butanoate (<b>14</b>, 0.17 g, 0.30 mmol) in methanol, (9H-fluoren-9-yl)methyl (5-oxopentyl)carbamate (<b>13b</b>, 0.13 g, 0.40 mmol) was added into the reaction. The mixture was stirred for 20 min at r.t. Then, NaBH<sub>3</sub>CN (26 mg, 0.42 mmol) was added in portions, and the reaction mixture was continuously stirred overnight at r.t. After removing the solvent in vacuum, the residue was purified by column chromatography (CH<sub>2</sub>Cl<sub>2</sub>/MeOH = 50:1) to obtain 0.1 g (38%) of white solid. <sup>1</sup>H NMR (500 MHz, chloroform-<i>d</i>) δ 8.34 (s, 1H), 7.93 (s, 1H), 7.76 (d, <i>J</i> = 7.5 Hz, 2H), 7.60 (d, <i>J</i> = 6.9 Hz, 2H), 7.39 (t, <i>J</i> = 7.4 Hz, 2H), 7.30 (t, <i>J</i> = 7.3 Hz, 2H), 6.09–6.01 (m, 1H), 5.67–5.43 (m, 3H), 5.03–4.91 (m, 1H), 4.49–4.32 (m, 3H), 4.31–4.08 (m, 2H), 3.31–3.01 (m, 2H), 2.83–2.21 (m, 6H), 1.99–1.84 (m, 1H), 1.70 (ws, 3H), 1.61 (s, 3H), 1.44 (s, 18H), 1.39 (s, 3H), 1.34–1.10 (m, 4H).</div></div><div id="sec3_1_1_15" class="NLM_sec NLM_sec_level_4"><div id="ac_i34" class="anchor-spacer"></div><h5 class="article-section__title" id="_i34"> <i>tert</i>-Butyl (<i>S</i>)-4-((((3a<i>R</i>,4<i>R</i>,6<i>S</i>,6a<i>R</i>)-6-(6-amino-9H-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)(4-aminobutyl)amino)-2-((<i>tert</i>-butoxycarbonyl)amino)butanoate (<b>18</b>)</h5><div class="NLM_p last">To a solution of <b>15a</b> (0.46 g, 0.6 mmol) in MeOH, 10% Pd/C was added slowly into the reaction. The mixture was degassed under vacuum and then under H2 atmosphere and stirred overnight. After filtration and concentration, the residue was purified by column chromatography (CH<sub>2</sub>Cl<sub>2</sub>/MeOH = 10:1) to get 0.21 g (56%) of white solid. <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.26 (s, 1H), 8.22 (s, 1H), 6.17 (d, <i>J</i> = 2.1 Hz, 1H), 5.54–5.51 (m, 1H), 5.08–5.01 (m, 1H), 4.37–4.26 (m, 1H), 4.13–4.00 (m, 1H), 2.84–2.74 (m, 1H), 2.66–2.52 (m, 4H), 2.50–2.39 (m, 2H), 2.38–2.28 (m, 1H), 2.00–1.86 (m, 1H), 1.69–1.60 (m, 1H), 1.59 (s, 3H), 1.46–1.41 (m, 20H), 1.38 (s, 3H).</div></div><div id="sec3_1_1_16" class="NLM_sec NLM_sec_level_4"><div id="ac_i35" class="anchor-spacer"></div><h5 class="article-section__title" id="_i35"> <i>tert</i>-butyl (<i>S</i>)-4-((((3a<i>R</i>,4<i>R</i>,6S,6a<i>R</i>)-6-(6-amino-9H-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)(5-aminopentyl)amino)-2-((<i>tert</i>-butoxycarbonyl)amino)butanoate (<b>19</b>)</h5><div class="NLM_p last">To a solution of <b>15b</b> (0.1 g, 0.11 mmol) in CH<sub>3</sub>CN cooled in an ice bath, DBU (34 μL, 0.22 mmol) was added dropwise. The reaction mixture was stirred for 1 h at r.t. The solvent was removed in vacuum, and the residue was purified by column chromatography (CH<sub>2</sub>Cl<sub>2</sub>/MeOH = 10:1) to get 50 mg (71%) of the product. <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.26 (s, 1H), 8.22 (s, 1H), 6.17 (d, <i>J</i> = 2.2 Hz, 1H), 5.57–5.50 (m, 1H), 5.08–5.02 (m, 1H), 4.36–4.28 (m, 1H), 4.11–4.01 (m, 1H), 2.84–2.74 (m, 1H), 2.66–2.54 (m, 4H), 2.48–2.39 (m, 2H), 2.38–2.29 (m, 1H), 1.99–1.88 (m, 1H), 1.68–1.61 (m, 1H), 1.59 (s, 3H), 1.44 (s, 9H), 1.44 (s, 9H), 1.42–1.40 (m, 2H), 1.39 (s, 3H), 1.37–1.30 (m, 2H), 1.30–1.17 (m, 2H).</div></div><div id="sec3_1_1_17" class="NLM_sec NLM_sec_level_4"><div id="ac_i36" class="anchor-spacer"></div><h5 class="article-section__title" id="_i36"> <i>tert</i>-Butyl (<i>S</i>)-4-((4-((((9H-fluoren-9-yl)methoxy)carbonyl)amino)butyl)amino)-2-((<i>tert</i>-butoxycarbonyl)amino)butanoate (<b>22</b>)</h5><div class="NLM_p last">To a solution of (9H-fluoren-9-yl)methyl (4-aminobutyl)carbamate (<b>20</b>, 0.2g, 0.58 mmol) in methanol, <i>tert</i>-butyl (S)-2-((<i>tert</i>-butoxycarbonyl)amino)-4-oxobutanoate (<b>21</b>, 0.24g, 0.86 mmol) was added into the reaction. The mixture was stirred for 30 min at r.t. Then, NaBH<sub>3</sub>CN (51 mg, 0.81 mmol) was added in portions, and the reaction mixture was continuously stirred for 6 h at r.t. After removing the solvent in vacuum, the residue was purified by column chromatography (CH<sub>2</sub>Cl<sub>2</sub>/MeOH = 20:1) to obtain 0.23 g (71%) of the product. <sup>1</sup>H NMR (500 MHz, chloroform-<i>d</i>) δ 7.75 (d, <i>J</i> = 7.6 Hz, 2H), 7.60 (d, <i>J</i> = 7.4 Hz, 2H), 7.39 (t, <i>J</i> = 7.4 Hz, 2H), 7.34–7.28 (m, 2H), 5.63–5.55 (m, 1H), 5.27–5.20 (m, 1H), 4.43–4.29 (m, 2H), 4.21 (t, <i>J</i> = 6.8 Hz, 1H), 4.18–4.09 (m, 1H), 3.31–3.17 (m, 2H), 3.16–2.79 (m, 5H), 2.55–2.45 (m, 1H), 1.84–1.75 (m, 2H), 1.72–1.58 (m, 2H), 1.48 (s, 9H), 1.43 (s, 9H). TLC–MS (ESI<sup>+</sup>) <i>m</i>/<i>z</i> 568.6 [M + H]<sup>+</sup></div></div><div id="sec3_1_1_18" class="NLM_sec NLM_sec_level_4"><div id="ac_i37" class="anchor-spacer"></div><h5 class="article-section__title" id="_i37"> <i>tert</i>-Butyl (<i>S</i>)-4-((4-((((9H-fluoren-9-yl)methoxy)carbonyl)amino)butyl)(<i>tert</i>-butoxycarbonyl)amino)-2-((<i>tert</i>-butoxycarbonyl)amino)butanoate (<b>23</b>)</h5><div class="NLM_p last">To a mixture of <b>22</b> (0.21 g, 0.37 mmol) and NaHCO<sub>3</sub> (95 mg, 1.11 mmol) in THF and H<sub>2</sub>O cooled in an ice bath, (Boc)<sub>2</sub>O (0.16 g, 0.74 mmol) was added in portions. Then, the reaction mixture was stirred at r.t. overnight. The mixture was diluted by CH<sub>2</sub>Cl<sub>2</sub>, and the organic layer was separated and dried over anhydrous NaSO<sub>4</sub>. After filtration, the solvent of the organic layer was removed in vacuum. The residue was purified by column chromatography (CH<sub>2</sub>Cl<sub>2</sub>/MeOH = 20:1) to get 0.21 g (85%) of the product. <sup>1</sup>H NMR (500 MHz, chloroform-<i>d</i>) δ 7.74 (d, <i>J</i> = 7.5 Hz, 2H), 7.58 (d, <i>J</i> = 7.3 Hz, 2H), 7.37 (t, <i>J</i> = 7.4 Hz, 2H), 7.28 (t, <i>J</i> = 7.3 Hz, 2H), 5.24–5.09 (m, 1H), 4.37 (d, <i>J</i> = 6.2 Hz, 2H), 4.19 (t, <i>J</i> = 6.9 Hz, 1H), 4.17–4.10 (m, 1H), 3.35–3.07 (m, 6H), 2.03–1.96 (m, 1H), 1.90–1.75 (m, 1H), 1.47–1.42 (m, 31H).</div></div><div id="sec3_1_1_19" class="NLM_sec NLM_sec_level_4"><div id="ac_i38" class="anchor-spacer"></div><h5 class="article-section__title" id="_i38"> <i>tert</i>-Butyl (<i>S</i>)-4-((4-aminobutyl)(<i>tert</i>-butoxycarbonyl)amino)-2-((<i>tert</i>-butoxycarbonyl)amino)butanoate (<b>24</b>)</h5><div class="NLM_p last">To a solution of <b>23</b> (0.21 g, 0.31 mmol) in CH<sub>3</sub>CN cooled in an ice bath, DBU (94 μL, 0.62 mmol) was added dropwise. The reaction mixture was stirred for 1 h at r.t. The solvent was removed in vacuum, and the residue was purified by column chromatography (CH<sub>2</sub>Cl<sub>2</sub>/MeOH = 10:1) to get 69 mg (50%) of the product. <sup>1</sup>H NMR (500 MHz, chloroform-<i>d</i>) δ 5.27–5.13 (m, 1H), 4.18–4.03 (m, 1H), 3.38–3.02 (m, 4H), 3.02–2.85 (m, 2H), 2.06–1.92 (m, 1H), 1.85–1.73 (m, 1H), 1.68–1.59 (m, 2H), 1.60–1.52 (m, 2H), 1.52–1.33 (m, 27H).</div></div></div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> Expression and Purification of Protein</h3><div class="NLM_p last">Expression and purification of NTMT1, G9a, SETD7, PRMT1, PRMT3, <i>Tb</i>PRMT7, and NNMT were performed as previously described<a onclick="showRef(event, 'ref6 ref18 ref37 ref40 ref41 ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref6 ref18 ref37 ref40 ref41 ref42 ref43">(6,18,37,40−43)</a></div></div><div id="sec3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> NTMT1 Biochemical Assays and Enzyme Kinetics Studies</h3><div class="NLM_p">A fluorescence-based SAHH-coupled assay was applied to study the IC<sub>50</sub> values of compounds with both SAM and the peptide substrate at their <i>K</i><sub>m</sub> values. After 10 min of preincubation, the methylation assay was performed under the following conditions in a final well volume of 40 μL: 25 mM Tris (pH = 7.5), 50 mM KCl, 0.01% Triton X-100, 5 μM SAHH, 0.1 μM NTMT1, 3 μM SAM, and 10 μM ThioGlo4. The inhibitors were added at concentrations ranging from 0.15 nM to 10 μM. After 10 min of incubation, reactions were initiated by the addition of 0.5 μM GPKRIA peptide. For 120 min preincubation, NTMT1 and compounds were preincubated for 120 min. Then, the preincubation mixture was added into the reaction mixture: 25 mM Tris (pH = 7.5), 50 mM KCl, 0.01% Triton X-100, 5 μM SAHH, 3 μM SAM, and 10 μM ThioGlo4. The reactions were initiated by the addition of 0.5 μM GPKRIA peptide. Fluorescence was monitored on a BMG CLARIOstar microplate reader with excitation at 400 nm and emission at 465 nm. Data were processed using GraphPad Prism software 7.0.</div><div class="NLM_p last">For the inhibition mechanism studies, varying concentrations of SAM with a fixed concentration of GPKRIA (0.5 μM) or varying concentrations of GPKRIA with a fixed concentration of SAM (3 μM) were included in reactions at the concentration of inhibitors ranging from 0.15 nM to 10 μM. Fluorescence was monitored on a BMG CLARIOstar microplate reader with excitation at 400 nm and emission at 465 nm. All experiments were determined in duplicate. Data were processed using GraphPad Prism software 7.0.</div></div><div id="sec3_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> ITC</h3><div class="NLM_p last">Isothermal titration calorimetry (ITC) measurements were performed by a VP-ITC MicroCal calorimeter at 25 °C. Titration buffer contains 20 mM Tris–HCl, pH 7.5, 150 mM NaCl, and 0.5 mM TECP (tris(2-carboxyethyl)phosphine). Purified NTMT1 and NTMT2 were diluted with the ITC buffer to a final concentration of 30–50 μM, and the compounds were dissolved to 0.5–1 mM in the same buffer. The compound was titrated into the protein solution with 26 injections of 10 μL each. Injections were spaced 180 s with a reference power of 15 μcal/s. The ITC data were processed with MicroCal PEAQ-ITC analysis software using a one-site binding or two-site binding model.</div></div><div id="sec3_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> Time-Dependent Inhibition Study</h3><div class="NLM_p last">The assay was performed under the following conditions in a final well volume of 40 μL: 25 mM Tris (pH = 7.5), 50 mM KCl, 0.01% Triton X-100, 5 μM SAHH, 0.1 μM NTMT1, 3 μM SAM, 0.5 μM GPKRIA, and 10 μM ThioGlo4. The inhibitors were added at concentrations ranging from 0.15 nM to 10 μM. NTMT1 and compounds were preincubated for 30, 60, 90, and 120 min, Then, the preincubation mixture was added into the reaction mixture: 25 mM Tris (pH = 7.5), 50 mM KCl, 0.01% Triton X-100, 5 μM SAHH, 3 μM SAM, and 10 μM ThioGlo4. The reactions were initiated by the addition of 0.5 μM GPKRIA peptide. Fluorescence was monitored on a BMG CLARIOstar microplate reader with excitation at 400 nm and emission at 465 nm. Data were processed using GraphPad Prism software 7.0 with Morrison’s quadratic equation.</div></div><div id="sec3_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> Selectivity Assays</h3><div class="NLM_p">The selectivity studies of G9a, SETD7, PRMT1, <i>Tb</i>PRMT7, and NNMT were performed as previously described.<a onclick="showRef(event, 'ref11 ref44'); return false;" href="javascript:void(0);" class="ref ref11 ref44">(11,44)</a></div><div class="NLM_p last">For PRMT3, the assay was also performed in a final well volume of 40 μL: 2.5 mM HEPES (pH = 7.0), 25 mM NaCl, 25 μM ethylenediaminetetraacetic acid (EDTA), 50 μM tris(2-carboxyethyl)phosphine (TCEP), 0.01% Triton X-100, 5 μM SAHH, 1 μM PRMT3, 25 μM SAM, and 15 μM ThioGlo1. After 10 min incubation with inhibitors, reactions were initiated by the addition of 10 μM H4-21 peptide. All experiments were determined in duplicate. Fluorescence was monitored on a BMG CLARIOstar microplate reader with excitation at 380 nm and emission at 505 nm.</div></div><div id="sec3_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> MALDI-MS Methylation and Methylation Inhibition Assays</h3><div class="NLM_p">MALDI-MS methylation and methylation inhibition assays were performed and analyzed via a Sciex 4800 MALDI TOF/TOF MS. MALDI-MS methylation assay was performed in a final well volume of 40 μL: 0.1 μM NTMT1, 20 mM Tris (pH = 7.5), 50 mM KCl, 3 μM SAM and 10 μM compounds. After incubation overnight at 37 °C, the samples were quenched in a 1:1 ratio with a quenching solution (20 mM NH<sub>4</sub>H<sub>2</sub>PO<sub>4</sub>, 0.4% (v/v) TFA in 1:1 acetonitrile/water). Samples were analyzed by MALDI-MS with 2,5-dihydroxybenzoic acid matrix solution.</div><div class="NLM_p last">The inhibition assay was performed in a final well volume of 40 μL: 0.1 μM NTMT1, 20 mM Tris (pH = 7.5), 50 mM KCl, 3 μM SAM, and various concentrations of compounds at 37 °C for 10 min before the addition of 0.5 μM GPKRIA peptide to initiate the reaction. After incubation for 20 min, the samples were quenched in a 1:1 ratio with a quenching solution (20 mM NH<sub>4</sub>H<sub>2</sub>PO<sub>4</sub>, 0.4% (v/v) TFA in 1:1 acetonitrile/water). Samples were analyzed by MALDI-MS with 2,5-dihydroxybenzoic acid matrix solution. A duplicate assay was performed for all experiments. Data were processed in Data Explorer.</div></div><div id="sec3_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> Cocrystallization and Structure Determination</h3><div class="NLM_p">For cocrystallization, NTMT1 was mixed with 2 mM compounds incubated for 1 h at 4 °C, diluted sixfold in 1× <i>tert</i>-butyldimethylsilyl chloride (TBS), pH 7.6, with 1 mM TCEP, and then reconcentrated to 30 mg/mL. Broad matrix and optimization crystallization screening were performed using a Mosquito-LCP high-throughput crystallization robot (TTP LabTech) with a hanging-drop vapor diffusion method, incubated at 20 °C, and crystals harvested directly from the 96-well crystallization plates. Crystals containing <b>1</b> were grown in 32% (w/v) PEG 4000, 0.8 M lithium chloride, 0.1 M Tris–HCl (pH = 8.5), and crystals containing <b>4</b> were grown in 30% (w/v) PEG3350, 0.2 M Li<sub>2</sub>SO<sub>4</sub>, and 0.1 M bis–tris propane (pH = 7.0); all crystals were harvested directly from the drop and flash-cooled into liquid nitrogen.</div><div class="NLM_p last">Data were collected on single crystals at 12.0 keV at the GM/CA-CAT ID-D beamline at the Advanced Photon Source, Argonne National Laboratory. The data were processed using the on-site automated pipeline with Fast DP, and molecular replacement was performed with Phaser-MR (PHENIX).<a onclick="showRef(event, 'ref45 ref46'); return false;" href="javascript:void(0);" class="ref ref45 ref46">(45,46)</a> Whole model building was performed using COOT, and the subsequent refinement was done using phenix.refine (PHENIX).<a onclick="showRef(event, 'ref46 ref47'); return false;" href="javascript:void(0);" class="ref ref46 ref47">(46,47)</a> Structure factors and model coordinates have been deposited in the Protein Data Bank with PDB IDs <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WJ7">6WJ7</a> (<b>1</b>) and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PVB">6PVB</a> (<b>4</b>). Structure-related figures were made with PyMOL (Schrödinger) and annotated and finalized with Adobe Photoshop and Illustrator.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i46"><a href="/doi/suppl/10.1021/acs.jmedchem.0c00770" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07620" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07620" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c00770?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00770</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00770/suppl_file/jm0c00770_si_001.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">NMR and MS spectra of compounds <b>15a</b>,<b>b</b>, <b>18</b>, <b>19</b>, and <b>22</b>–<b>24</b>; NMR, MALDI-MS, and LC–MS/HPLC spectra of compounds <b>1</b>–<b>12</b>; methylation assay of compounds <b>10</b>–<b>12</b> for NTMT1 (Figure S1); MALDI-MS methylation inhibition assay compound <b>8</b> (Figure S2); selectivity evaluation of compound <b>3</b> (Table S1); and crystallography data and refinement statistics (PDB IDs: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WJ7">6WJ7</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PVB">6PVB</a>) (Table S2) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00770/suppl_file/jm0c00770_si_002.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00770/suppl_file/jm0c00770_si_001.csv">jm0c00770_si_001.csv (1.91 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00770/suppl_file/jm0c00770_si_002.pdf">jm0c00770_si_002.pdf (2.94 MB)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The coordinates for the structure of human NTMT1 in complex with compound <b>1</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WJ7">6WJ7</a>) and <b>4</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PVB">6PVB</a>) have been deposited in the Protein Data Bank. The authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.0c00770" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_11135" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_11135" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rong Huang</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry and Molecular Pharmacology, Center for Cancer
Research, Institute for Drug Discovery, Purdue University, West Lafayette, Indiana 47907, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1477-3165" title="Orcid link">http://orcid.org/0000-0002-1477-3165</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#610914000f064c13211114130514044f040514"><span class="__cf_email__" data-cfemail="442c31252a236936043431362031216a212031">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dongxing Chen</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry and Molecular Pharmacology, Center for Cancer
Research, Institute for Drug Discovery, Purdue University, West Lafayette, Indiana 47907, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cheng Dong</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, Department of Physiology, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Guangping Dong</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry and Molecular Pharmacology, Center for Cancer
Research, Institute for Drug Discovery, Purdue University, West Lafayette, Indiana 47907, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Karthik Srinivasan</span> - <span class="hlFld-Affiliation affiliation">Markey
Center for Structural Biology, Department of Biological Sciences and
the Purdue Institute of Inflammation, Immunology and Infectious Disease, Purdue University, West Lafayette, Indiana 47907, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jinrong Min</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, Department of Physiology, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nicholas Noinaj</span> - <span class="hlFld-Affiliation affiliation">Markey
Center for Structural Biology, Department of Biological Sciences and
the Purdue Institute of Inflammation, Immunology and Infectious Disease, Purdue University, West Lafayette, Indiana 47907, United States</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e2876-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i48">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48177" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48177" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors appreciate Dr. Darrel L. Peterson for the purification of SAHH. The authors thank Krystal Diaz for purifying <i>Tb</i>PRMT7. The authors acknowledge the support from NIH grants R01GM117275 (R.H.), 1R01GM127896 (N.N.), 1R01AI127793 (N.N.), and P30 CA023168 (Purdue University Center for Cancer Research). SGC is a registered charity (no. 1097737) that receives funds from AbbVie, Bayer Pharma AG, Boehringer Ingelheim, Canada Foundation for Innovation, Eshelman Institute for Innovation, Genome Canada through Ontario Genomics Institute [OGI-055], Innovative Medicines Initiative (EU/EFPIA) [ULTRA-DD grant no. 115766], Janssen, Merck KGaA, Darmstadt, Germany, MSD, Novartis Pharma AG, Ontario Ministry of Research, Innovation and Science (MRIS), Pfizer, São Paulo Research Foundation-FAPESP, Takeda, and Wellcome. The authors also thank the Department of Medicinal Chemistry and Molecular Pharmacology (RH) and the Department of Biological Sciences (N.N.) at Purdue University for their support. This work is supported by Natural Sciences and Engineering Research Council Grant RGPIN-2016-06300 (J.M.).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">NTMT1</td><td class="NLM_def"><p class="first last">protein N-terminal methyltransferase 1</p></td></tr><tr><td class="NLM_term">PMTs</td><td class="NLM_def"><p class="first last">protein methyltransferases</p></td></tr><tr><td class="NLM_term">SAM</td><td class="NLM_def"><p class="first last"><i>S</i>-5′-adenosyl-<span class="smallcaps smallerCapital">l</span>-methionine</p></td></tr><tr><td class="NLM_term">SAH</td><td class="NLM_def"><p class="first last"><i>S</i>-5′-adenosyl-<span class="smallcaps smallerCapital">l</span>-homocysteines</p></td></tr><tr><td class="NLM_term">SAHH</td><td class="NLM_def"><p class="first last">SAH hydrolase</p></td></tr><tr><td class="NLM_term">PRMT1</td><td class="NLM_def"><p class="first last">protein arginine methyltransferase 1</p></td></tr><tr><td class="NLM_term">CARM1</td><td class="NLM_def"><p class="first last">coactivator-associated arginine methyltransferase 1</p></td></tr><tr><td class="NLM_term">G9a</td><td class="NLM_def"><p class="first last">euchromatic histone-lysine <i>N</i>-methyltransferase 2</p></td></tr><tr><td class="NLM_term">SETD7</td><td class="NLM_def"><p class="first last">SET domain-containing protein 7</p></td></tr><tr><td class="NLM_term">NNMT</td><td class="NLM_def"><p class="first last">nicotinamide <i>N</i>-methyltransferase</p></td></tr><tr><td class="NLM_term">PRMT3</td><td class="NLM_def"><p class="first last">protein arginine methyltransferase 3</p></td></tr><tr><td class="NLM_term"><i>Tb</i>PRMT7</td><td class="NLM_def"><p class="first last"><i>Trypanosoma brucei</i> protein arginine methyltransferase 7</p></td></tr><tr><td class="NLM_term">MTase</td><td class="NLM_def"><p class="first last">methyltransferase</p></td></tr><tr><td class="NLM_term">TCEP</td><td class="NLM_def"><p class="first last">tris(2-carboxyethyl)phosphine</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i50">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16666" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16666" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 47 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowhan, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muscat, G. E. O.</span></span> <span> </span><span class="NLM_article-title">Epigenetic Arginine Methylation in Breast Cancer: Emerging Therapeutic Strategies</span>. <i>J. Mol. Endocrinol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">R223</span>– <span class="NLM_lpage">R227</span>, <span class="refDoi"> DOI: 10.1530/JME-18-0224</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1530%2FJME-18-0224" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=30620710" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1eiu7vI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=R223-R227&author=S.+M.+Wangauthor=S.+M.+Wangauthor=D.+H.+Dowhanauthor=G.+E.+O.+Muscat&title=Epigenetic+Arginine+Methylation+in+Breast+Cancer%3A+Emerging+Therapeutic+Strategies&doi=10.1530%2FJME-18-0224"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic arginine methylation in breast cancer: emerging therapeutic strategies</span></div><div class="casAuthors">Wang, Shu-Ching M.; Dowhan, Dennis H.; Muscat, George E. O.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Endocrinology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">R223-R237</span>CODEN:
                <span class="NLM_cas:coden">JMLEEI</span>;
        ISSN:<span class="NLM_cas:issn">1479-6813</span>.
    
            (<span class="NLM_cas:orgname">Bioscientifica Ltd.</span>)
        </div><div class="casAbstract">A review.  Breast cancer is a heterogeneous disease, and the complexity of breast carcinogenesis is assocd. with epigenetic modifiation.  There are several major classes of epigenetic enzymes that regulate chromatin activity.  This review will focus on the nine mammalian protein arginine methyltransferases (PRMTs) and the dysregulation of PRMT expression and function in breast cancer.  This class of enzymes catalyze the mono- and (sym. and asym.) di-methylation of arginine residues on histone and non-histone target proteins.  PRMT signalling (and R methylation) drives cellular proliferation, cell invasion and metastasis, targeting (i) nuclear hormone receptor signalling, (ii) tumor suppressors, (iii) TGF- β and EMT signalling and (iv) alternative splicing and DNA/chromatin stability, inflencing the clin. and survival outcomes in breast cancer.  Emerging reports suggest that PRMTs are also implicated in the development of drug/endocrine resistance providing another prospective avenue for the treatment of hormone resistance and assocd. metastasis.  The complexity of PRMT signalling is further underscored by the degree of alternative splicing and the scope of variant isoforms (with distinct properties) within each PRMT family member.  The evolution of PRMT inhibitors, and the ongoing clin. trials of PRMT inhibitors against a subgroup of solid cancers, coupled to the track record of lysine methyltransferases inhibitors in phase I/II clin. trials against cancer underscores the potential therapeutic utility of targeting PRMT epigenetic enzymes to improve survival outcomes in aggressive and metastatic breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYtQu27dxxebVg90H21EOLACvtfcHk0lhA8i_-TJvFCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1eiu7vI&md5=221a3f95d307b8a4db0720d2164792b3</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1530%2FJME-18-0224&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1530%252FJME-18-0224%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DS.%2BM.%26aulast%3DWang%26aufirst%3DS.%2BM.%26aulast%3DDowhan%26aufirst%3DD.%2BH.%26aulast%3DMuscat%26aufirst%3DG.%2BE.%2BO.%26atitle%3DEpigenetic%2520Arginine%2520Methylation%2520in%2520Breast%2520Cancer%253A%2520Emerging%2520Therapeutic%2520Strategies%26jtitle%3DJ.%2520Mol.%2520Endocrinol.%26date%3D2019%26volume%3D62%26spage%3DR223%26epage%3DR227%26doi%3D10.1530%2FJME-18-0224" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. Ü.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martini, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of Protein Methyltransferases and Demethylases</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">989</span>– <span class="NLM_lpage">1068</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.6b00801</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.6b00801" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkslynsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2018&pages=989-1068&author=H.+%C3%9C.+Kaniskanauthor=M.+L.+Martiniauthor=J.+Jin&title=Inhibitors+of+Protein+Methyltransferases+and+Demethylases&doi=10.1021%2Facs.chemrev.6b00801"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of Protein Methyltransferases and Demethylases</span></div><div class="casAuthors">Kaniskan, H. Umit; Martini, Michael L.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">989-1068</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Post-translational modifications of histones by protein methyltransferases (PMTs) and histone demethylases (KDMs) play an important role in the regulation of gene expression and transcription and are implicated in cancer and many other diseases.  Many of these enzymes also target various nonhistone proteins impacting numerous crucial biol. pathways.  Given their key biol. functions and implications in human diseases, there has been a growing interest in assessing these enzymes as potential therapeutic targets.  Consequently, discovering and developing inhibitors of these enzymes has become a very active and fast-growing research area over the past decade.  In this review, the authors cover the discovery, characterization, and biol. application of inhibitors of PMTs and KDMs with emphasis on key advancements in the field.  The authors also discuss challenges, opportunities, and future directions in this emerging, exciting research field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJYjiYtw_n4rVg90H21EOLACvtfcHk0lhA8i_-TJvFCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkslynsL8%253D&md5=1ed520c6b0c42d2aa8a7f66b0a2d4e40</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.6b00801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.6b00801%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3DMartini%26aufirst%3DM.%2BL.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DInhibitors%2520of%2520Protein%2520Methyltransferases%2520and%2520Demethylases%26jtitle%3DChem.%2520Rev.%26date%3D2018%26volume%3D118%26spage%3D989%26epage%3D1068%26doi%3D10.1021%2Facs.chemrev.6b00801" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niessen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goebel, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yates, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montminy, M.</span></span> <span> </span><span class="NLM_article-title">PRMT5 Modulates the Metabolic Response to Fasting Signals</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">8870</span>– <span class="NLM_lpage">8875</span>, <span class="refDoi"> DOI: 10.1073/pnas.1304602110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1073%2Fpnas.1304602110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=23671120" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFalsbjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=8870-8875&author=W.+Tsaiauthor=S.+Niessenauthor=N.+Goebelauthor=J.+R.+Yatesauthor=E.+Guccioneauthor=M.+Montminy&title=PRMT5+Modulates+the+Metabolic+Response+to+Fasting+Signals&doi=10.1073%2Fpnas.1304602110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">PRMT5 modulates the metabolic response to fasting signals</span></div><div class="casAuthors">Tsai, Wen-Wei; Niessen, Sherry; Goebel, Naomi; Yates, John R., III; Guccione, Ernesto; Montminy, Marc</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">8870-8875, S8870/1-S8870/5</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Under fasting conditions, increases in circulating glucagon maintain glucose balance by promoting hepatic gluconeogenesis.  Triggering of the cAMP pathway stimulates gluconeogenic gene expression through the PKA-mediated phosphorylation of the cAMP response element binding (CREB) protein and via the dephosphorylation of the latent cytoplasmic CREB regulated transcriptional coactivator 2 (CRTC2).  CREB and CRTC2 activities are increased in insulin resistance, in which they promote hyperglycemia because of constitutive induction of the gluconeogenic program.  The extent to which CREB and CRTC2 are coordinately up-regulated in response to glucagon, however, remains unclear.  Here it is shown that, following its activation, CRTC2 enhances CREB phosphorylation through an assocn. with the protein arginine methyltransferase 5 (PRMT5).  In turn, PRMT5 was found to stimulate CREB phosphorylation via increases in histone H3 Arg2 methylation that enhanced chromatin accessibility at gluconeogenic promoters.  Because depletion of PRMT5 lowers hepatic glucose prodn. and gluconeogenic gene expression, these results demonstrate how a chromatin-modifying enzyme regulates a metabolic program through epigenetic changes that impact the phosphorylation of a transcription factor in response to hormonal stimuli.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkZvybDjBzyrVg90H21EOLACvtfcHk0likkOzpgucgbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFalsbjF&md5=6a46c62348b78963e2ae08a5eb2a01bb</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1304602110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1304602110%26sid%3Dliteratum%253Aachs%26aulast%3DTsai%26aufirst%3DW.%26aulast%3DNiessen%26aufirst%3DS.%26aulast%3DGoebel%26aufirst%3DN.%26aulast%3DYates%26aufirst%3DJ.%2BR.%26aulast%3DGuccione%26aufirst%3DE.%26aulast%3DMontminy%26aufirst%3DM.%26atitle%3DPRMT5%2520Modulates%2520the%2520Metabolic%2520Response%2520to%2520Fasting%2520Signals%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2013%26volume%3D110%26spage%3D8870%26epage%3D8875%26doi%3D10.1073%2Fpnas.1304602110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, D.</span></span> <span> </span><span class="NLM_article-title">Coactivator-Associated Arginine Methyltransferase 1 Targeted by MiR-15a Regulates Inflammation in Acute Coronary Syndrome</span>. <i>Atherosclerosis</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>233</i></span>,  <span class="NLM_fpage">349</span>– <span class="NLM_lpage">356</span>, <span class="refDoi"> DOI: 10.1016/j.atherosclerosis.2014.01.039</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1016%2Fj.atherosclerosis.2014.01.039" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=24530761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC2cXks1Wntrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=233&publication_year=2014&pages=349-356&author=X.+Liuauthor=L.+Wangauthor=H.+Liauthor=X.+Luauthor=Y.+Huauthor=X.+Yangauthor=C.+Huangauthor=D.+Gu&title=Coactivator-Associated+Arginine+Methyltransferase+1+Targeted+by+MiR-15a+Regulates+Inflammation+in+Acute+Coronary+Syndrome&doi=10.1016%2Fj.atherosclerosis.2014.01.039"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Coactivator-associated arginine methyltransferase 1 targeted by miR-15a regulates inflammation in acute coronary syndrome</span></div><div class="casAuthors">Liu, Xuehui; Wang, Laiyuan; Li, Hongfan; Lu, Xiangfeng; Hu, Yongyan; Yang, Xueli; Huang, Chen; Gu, Dongfeng</div><div class="citationInfo"><span class="NLM_cas:title">Atherosclerosis (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">233</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">349-356</span>CODEN:
                <span class="NLM_cas:coden">ATHSBL</span>;
        ISSN:<span class="NLM_cas:issn">0021-9150</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Coactivator-assocd. arginine methyltransferase 1 (CARM1) is essential for the activation of a subset of NF-κB-dependent genes, which code the chemokines triggering plaque vulnerability.  Unstable atherosclerotic plaques lead to the onset of acute coronary syndrome (ACS).  Therefore, we aimed to investigate whether CARM1 is involved in the pathogenesis of ACS and ascertain the regulatory mechanism of CARM1 expression at posttranscriptional level.  Peripheral blood mononuclear cells (PBMCs) were isolated from peripheral blood of 19 patients with ACS and 22 subjects with risk factors for coronary heart disease.  Gene expression was detd. by quant. real-time PCR and Western blot.  The effects of CARM1 and miR-15a on their target genes expression were assessed by gain-of-function and loss-of-function approaches.  PBMCs from patients with ACS showed higher levels of CARM1 mRNA and protein expression.  The levels of CARM1 mRNA were pos. correlated with three chemokines including interferon-inducible protein-10 (IP-10), monocyte chemoattractant protein 1 (MCP-1), and interleukin-8 (IL-8) in PBMCs (CARM1 and IP-10: r = 0.55, P = 0.008; CARM1 and MCP-1: r = 0.64, P = 0.002; CARM1 and IL-8: r = 0.55, P = 0.008).  Moreover, CARM1 regulated the transcription of these chemokines in human embryonic kidney 293T (HEK293T) cells.  We also found that the levels of miR-15a were decreased by 37% in patients with ACS and miR-15a modulated CARM1 expression through targeted binding to CARM1 3'-UTR.  The present study demonstrated that CARM1 targeted by miR-15a played an important role in chemokine activation in the pathogenesis of ACS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrODlfcuAxfYrVg90H21EOLACvtfcHk0likkOzpgucgbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXks1Wntrc%253D&md5=dcb8ced710aad25aa69e9c5e30943f21</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.atherosclerosis.2014.01.039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.atherosclerosis.2014.01.039%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DC.%26aulast%3DGu%26aufirst%3DD.%26atitle%3DCoactivator-Associated%2520Arginine%2520Methyltransferase%25201%2520Targeted%2520by%2520MiR-15a%2520Regulates%2520Inflammation%2520in%2520Acute%2520Coronary%2520Syndrome%26jtitle%3DAtherosclerosis%26date%3D2014%26volume%3D233%26spage%3D349%26epage%3D356%26doi%3D10.1016%2Fj.atherosclerosis.2014.01.039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M.</span></span> <span> </span><span class="NLM_article-title">Chemical and Biochemical Perspectives of Protein Lysine Methylation</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">6656</span>– <span class="NLM_lpage">6705</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.8b00008</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.8b00008" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFKks7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2018&pages=6656-6705&author=M.+Luo&title=Chemical+and+Biochemical+Perspectives+of+Protein+Lysine+Methylation&doi=10.1021%2Facs.chemrev.8b00008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical and Biochemical Perspectives of Protein Lysine Methylation</span></div><div class="casAuthors">Luo, Minkui</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6656-6705</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Protein lysine methylation is a distinct post-translational modification that causes minimal changes in the size and electrostatic status of lysine residues.  Lysine methylation plays essential roles in regulating fates and functions of target proteins in an epigenetic manner.  As a result, substrates and degrees (free vs. mono/di/tri) of protein lysine methylation are orchestrated within cells by balanced activities of protein lysine methyltransferases (PKMTs) and demethylases (KDMs).  Their dysregulation is often assocd. with neurol. disorders, developmental abnormalities or cancer.  Methyllysine-contg. proteins can be recognized by downstream effector proteins, which contain methyllysine reader domains, to relay their biol. functions.  While numerous efforts have been made to annotate biol. roles of protein lysine methylation, limited work has been done to uncover mechanisms assocd. with this modification at a mol. or at. level.  Given distinct biophys. and biochem. properties of methyllysine, this review will focus on chem. and biochem. aspects in addn., recognition, and removal of this post-translational mark.  Chem. and biophys. methods to profile PKMT substrates will be discussed along with classification of PKMT inhibitors for accurate perturbation of methyltransferase activities.  Semisynthesis of methyllysine-contg. proteins will also be covered given the crit. need for these reagents to unambiguously define functional roles of protein lysine methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbSRWg3jP5bLVg90H21EOLACvtfcHk0lhc6YnRZsX_eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFKks7zO&md5=d846db925be49839c5954f5fc34f17d9</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.8b00008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.8b00008%26sid%3Dliteratum%253Aachs%26aulast%3DLuo%26aufirst%3DM.%26atitle%3DChemical%2520and%2520Biochemical%2520Perspectives%2520of%2520Protein%2520Lysine%2520Methylation%26jtitle%3DChem.%2520Rev.%26date%3D2018%26volume%3D118%26spage%3D6656%26epage%3D6705%26doi%3D10.1021%2Facs.chemrev.8b00008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanjra, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Kinetic Mechanism of Protein N-Terminal Methyltransferase 1</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>290</i></span>,  <span class="NLM_fpage">11601</span>– <span class="NLM_lpage">11610</span>, <span class="refDoi"> DOI: 10.1074/jbc.M114.626846</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1074%2Fjbc.M114.626846" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=25771539" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnsFCrs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=11601-11610&author=S.+L.+Richardsonauthor=Y.+Maoauthor=G.+Zhangauthor=P.+Hanjraauthor=D.+L.+Petersonauthor=R.+Huang&title=Kinetic+Mechanism+of+Protein+N-Terminal+Methyltransferase+1&doi=10.1074%2Fjbc.M114.626846"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic Mechanism of Protein N-terminal Methyltransferase 1</span></div><div class="casAuthors">Richardson, Stacie L.; Mao, Yunfei; Zhang, Gang; Hanjra, Pahul; Peterson, Darrell L.; Huang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">290</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">11601-11610</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The protein N-terminal methyltransferase 1 (NTMT1) catalyzes the transfer of the Me group from the S-adenosyl-L-methionine (AdoMet) to the protein α-amine, resulting in formation of S-adenosyl-L-homocysteine (AdoHcy) and α-N-methylated proteins.  NTMT1 is an interesting potential anticancer target because it is overexpressed in gastrointestinal cancers and plays an important role in cell mitosis.  To gain insight into the biochem. mechanism of NTMT1, we have characterized the kinetic mechanism of recombinant NTMT1 using a fluorescence assay and mass spectrometry.  The results of initial velocity, product, and dead-end inhibition studies indicate that methylation by NTMT1 proceeds via a random sequential Bi Bi mechanism.  In addn., our processivity studies demonstrate that NTMT1 proceeds via a distributive mechanism for multiple methylations.  Together, our studies provide new knowledge about the kinetic mechanism of NTMT1 and lay the foundation for the development of mechanism-based inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYCNmI8h5YRbVg90H21EOLACvtfcHk0lhc6YnRZsX_eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnsFCrs7g%253D&md5=28a62d8c1b378fa4fabceca206e61464</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M114.626846&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M114.626846%26sid%3Dliteratum%253Aachs%26aulast%3DRichardson%26aufirst%3DS.%2BL.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DHanjra%26aufirst%3DP.%26aulast%3DPeterson%26aufirst%3DD.%2BL.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DKinetic%2520Mechanism%2520of%2520Protein%2520N-Terminal%2520Methyltransferase%25201%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2015%26volume%3D290%26spage%3D11601%26epage%3D11610%26doi%3D10.1074%2Fjbc.M114.626846" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Khalil, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Angelis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishii, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span> <span> </span><span class="NLM_article-title">Mechanism-based Inhibition of the Melatonin Rhythem Enzyme: Pharmacologic Exploitation of Active Site Functional Plasticity</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>96</i></span>,  <span class="NLM_fpage">12418</span>– <span class="NLM_lpage">12423</span>, <span class="refDoi"> DOI: 10.1073/pnas.96.22.12418</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1073%2Fpnas.96.22.12418" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10535937" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADyaK1MXnt1yrtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=1999&pages=12418-12423&author=E.+M.+Khalilauthor=J.+De+Angelisauthor=M.+Ishiiauthor=P.+A.+Cole&title=Mechanism-based+Inhibition+of+the+Melatonin+Rhythem+Enzyme%3A+Pharmacologic+Exploitation+of+Active+Site+Functional+Plasticity&doi=10.1073%2Fpnas.96.22.12418"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism-based inhibition of the melatonin rhythm enzyme: pharmacologic exploitation of active site functional plasticity</span></div><div class="casAuthors">Khalil, Ehab M.; De Angelis, Jacqueline; Ishii, Makoto; Cole, Philip A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">96</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">12418-12423</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Serotonin N-acetyltransferase is the enzyme responsible for the diurnal rhythm of melatonin prodn. in the pineal gland of animals and humans.  Inhibitors of this enzyme active in cell culture have not been reported previously.  The compd. N-bromoacetyltryptamine was shown to be a potent inhibitor of this enzyme in vitro and in a pineal cell culture assay (IC50 ≈ 500 nM).  The mechanism of inhibition is suggested to involve a serotonin N-acetyltransferase-catalyzed alkylation reaction between N-bromoacetyltryptamine and reduced CoA, resulting in the prodn. of a tight-binding bisubstrate analog inhibitor.  This alkyltransferase activity is apparently catalyzed at a functionally distinct site compared with the acetyltransferase activity active site on serotonin N-acetyltransferase.  Such active site plasticity is suggested to result from a subtle conformational alteration in the protein.  This plasticity allows for an unusual form of mechanism-based inhibition with multiple turnovers, resulting in "mol. fratricide.".  N-bromoacetyltryptamine should serve as a useful tool for dissecting the role of melatonin in circadian rhythm as well as a potential lead compd. for therapeutic use in mood and sleep disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOeqv_lFsxOLVg90H21EOLACvtfcHk0lhc6YnRZsX_eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXnt1yrtbY%253D&md5=dda5816db0455e31a50cb72cd4e02883</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1073%2Fpnas.96.22.12418&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.96.22.12418%26sid%3Dliteratum%253Aachs%26aulast%3DKhalil%26aufirst%3DE.%2BM.%26aulast%3DDe%2BAngelis%26aufirst%3DJ.%26aulast%3DIshii%26aufirst%3DM.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DMechanism-based%2520Inhibition%2520of%2520the%2520Melatonin%2520Rhythem%2520Enzyme%253A%2520Pharmacologic%2520Exploitation%2520of%2520Active%2520Site%2520Functional%2520Plasticity%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1999%26volume%3D96%26spage%3D12418%26epage%3D12423%26doi%3D10.1073%2Fpnas.96.22.12418" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magalha, M. L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blanchard, J. S.</span></span> <span> </span><span class="NLM_article-title">Bisubstrate Inhibition: Theory and Application to N-Acetyltransferases</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">14788</span>– <span class="NLM_lpage">14794</span>, <span class="refDoi"> DOI: 10.1021/bi061621t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi061621t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1SjtbfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2006&pages=14788-14794&author=M.+Yuauthor=M.+L.+B.+Magalhaauthor=P.+F.+Cookauthor=J.+S.+Blanchard&title=Bisubstrate+Inhibition%3A+Theory+and+Application+to+N-Acetyltransferases&doi=10.1021%2Fbi061621t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Bisubstrate Inhibition: Theory and Application to N-Acetyltransferases</span></div><div class="casAuthors">Yu, Michael; Magalhaes, Maria L. B.; Cook, Paul F.; Blanchard, John S.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">14788-14794</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bisubstrate inhibitors represent a potentially powerful group of compds. that have found significant therapeutic utility.  Although these compds. have been synthesized and tested against a no. of enzymes that catalyze sequential bireactant reactions, the detailed theory for predicting the expected patterns of inhibition against the two substrates for various bireactant kinetic mechanisms has, heretofore, not been presented.  We have derived the rate equations for all likely sequential bireactant mechanisms and provide two examples in which bisubstrate inhibitors allow the kinetic mechanism to be detd.  Bisubstrate inhibitor kinetics is a powerful diagnostic for the detn. of kinetic mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrk_sv2txdK0LVg90H21EOLACvtfcHk0lg8uoVKg15aQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1SjtbfN&md5=6ddb7dcb30e7d97a3577a6088db49433</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fbi061621t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi061621t%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DM.%26aulast%3DMagalha%26aufirst%3DM.%2BL.%2BB.%26aulast%3DCook%26aufirst%3DP.%2BF.%26aulast%3DBlanchard%26aufirst%3DJ.%2BS.%26atitle%3DBisubstrate%2520Inhibition%253A%2520Theory%2520and%2520Application%2520to%2520N-Acetyltransferases%26jtitle%3DBiochemistry%26date%3D2006%26volume%3D45%26spage%3D14788%26epage%3D14794%26doi%3D10.1021%2Fbi061621t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Parang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span> <span> </span><span class="NLM_article-title">Designing Bisubstrate Analog Inhibitors for Protein Kinases</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">145</span>– <span class="NLM_lpage">157</span>, <span class="refDoi"> DOI: 10.1016/S0163-7258(02)00184-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1016%2FS0163-7258%2802%2900184-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=12191607" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BD38XksVShtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2002&pages=145-157&author=K.+Parangauthor=P.+A.+Cole&title=Designing+Bisubstrate+Analog+Inhibitors+for+Protein+Kinases&doi=10.1016%2FS0163-7258%2802%2900184-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Designing bisubstrate analog inhibitors for protein kinases</span></div><div class="casAuthors">Parang, Keykavous; Cole, Philip A.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">2-3</span>),
    <span class="NLM_cas:pages">145-157</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Inc.</span>)
        </div><div class="casAbstract">A review.  Protein kinases play crit. roles in signal transduction pathways by transmitting extracellular signals across the cell membrane to distant locations in the cytoplasm and the nucleus.  The development of protein kinase inhibitors has been hindered by the broad overlapping substrate specificities exhibited by these enzymes.  The design of bisubstrate analog inhibitors could provide for the enhancement of specificity and potency in protein kinase inhibition.  Bisubstrate analog inhibitors form a special group of protein kinase inhibitors that mimic two natural substrates/ligands and that simultaneously assoc. with two regions of given kinases.  Most bisubstrate analogs have been designed to mimic the phosphate donor (ATP) and the acceptor components (Ser-, Thr-, or Tyr-contg. peptides).  Recent studies have emphasized the importance of maintaining a specific distance between these two components to achieve potent inhibition.  In this review, we present a discussion of the methods for designing protein kinase inhibitors by mechanism-based approaches.  Emphasis is given to bivalent approaches, with an interpretation of what has been learned from more and less successful examples.  Future challenges in this area are also highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoM_dF2knydcbVg90H21EOLACvtfcHk0lg8uoVKg15aQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XksVShtbk%253D&md5=95c8c3d60901a53ffc712d83c6cfba01</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2FS0163-7258%2802%2900184-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0163-7258%252802%252900184-5%26sid%3Dliteratum%253Aachs%26aulast%3DParang%26aufirst%3DK.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DDesigning%2520Bisubstrate%2520Analog%2520Inhibitors%2520for%2520Protein%2520Kinases%26jtitle%3DPharmacol.%2520Ther.%26date%3D2002%26volume%3D93%26spage%3D145%26epage%3D157%26doi%3D10.1016%2FS0163-7258%2802%2900184-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez-Ferrando, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span> <span> </span><span class="NLM_article-title">Enhanced Interrogation: Emerging Strategies for Cell Signaling Inhibition</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">646</span>– <span class="NLM_lpage">649</span>, <span class="refDoi"> DOI: 10.1038/nsmb0610-646</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1038%2Fnsmb0610-646" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2011&pages=646-649&author=R.+Huangauthor=I.+Martinez-Ferrandoauthor=P.+A.+Cole&title=Enhanced+Interrogation%3A+Emerging+Strategies+for+Cell+Signaling+Inhibition&doi=10.1038%2Fnsmb0610-646"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnsmb0610-646&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb0610-646%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DR.%26aulast%3DMartinez-Ferrando%26aufirst%3DI.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DEnhanced%2520Interrogation%253A%2520Emerging%2520Strategies%2520for%2520Cell%2520Signaling%2520Inhibition%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2011%26volume%3D17%26spage%3D646%26epage%3D649%26doi%3D10.1038%2Fnsmb0610-646" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noinaj, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Discovery of Bisubstrate Inhibitors for Protein N-Terminal Methyltransferase 1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">3773</span>– <span class="NLM_lpage">3779</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00206</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00206" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC1MXltFequ7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=3773-3779&author=D.+Chenauthor=G.+Dongauthor=N.+Noinajauthor=R.+Huang&title=Discovery+of+Bisubstrate+Inhibitors+for+Protein+N-Terminal+Methyltransferase+1&doi=10.1021%2Facs.jmedchem.9b00206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of bisubstrate inhibitors for protein N-terminal methyltransferase 1</span></div><div class="casAuthors">Chen, Dongxing; Dong, Guangping; Noinaj, Nicholas; Huang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3773-3779</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein N-terminal methyltransferase 1 (NTMT1) plays an important role in regulating mitosis and DNA repair.  Here, we describe the discovery of a potent NTMT1 bisubstrate inhibitor 4 (IC50 = 35 ± 2 nM) that exhibits greater than 100-fold selectivity against a panel of methyltransferases.  We also report the first crystal structure of NTMT1 in complex with an inhibitor, which revealed that 4 occupies substrate and cofactor binding sites of NTMT1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqM1ZcSF10eY7Vg90H21EOLACvtfcHk0lg8uoVKg15aQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXltFequ7o%253D&md5=34782952c04423e217abf6cd3d228f94</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00206%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DD.%26aulast%3DDong%26aufirst%3DG.%26aulast%3DNoinaj%26aufirst%3DN.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DDiscovery%2520of%2520Bisubstrate%2520Inhibitors%2520for%2520Protein%2520N-Terminal%2520Methyltransferase%25201%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D3773%26epage%3D3779%26doi%3D10.1021%2Facs.jmedchem.9b00206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brandvold, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breen, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lachacz, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steffey, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soellner, M. B.</span></span> <span> </span><span class="NLM_article-title">Exquisitely Specific Bisubstrate Inhibitors of C-Src Kinase</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1387</span>– <span class="NLM_lpage">1391</span>, <span class="refDoi"> DOI: 10.1021/cb501048b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb501048b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC2MXkvFWjurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1387-1391&author=K.+R.+Brandvoldauthor=S.+M.+Santosauthor=M.+E.+Breenauthor=E.+J.+Lachaczauthor=M.+E.+Steffeyauthor=M.+B.+Soellner&title=Exquisitely+Specific+Bisubstrate+Inhibitors+of+C-Src+Kinase&doi=10.1021%2Fcb501048b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Exquisitely Specific Bisubstrate Inhibitors of c-Src Kinase</span></div><div class="casAuthors">Brandvold, Kristoffer R.; Santos, Shana M.; Breen, Meghan E.; Lachacz, Eric J.; Steffey, Michael E.; Soellner, Matthew B.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1387-1391</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have developed a modular approach to bisubstrate inhibition of protein kinases.  We apply our methodol. to c-Src and identify a highly selective bisubstrate inhibitor for this target.  Our approach has yielded the most selective c-Src inhibitor to date, and the methodol. to render the bisubstrate inhibitor cell-permeable provides a highly valuable tool for the study of c-Src signaling.  In addn., we have applied our bisubstrate inhibitor to develop a novel screening methodol. to identify non-ATP-competitive inhibitors of c-Src.  Using this methodol., we have discovered the most potent non-ATP-competitive inhibitor reported to date.  Our methodol. is designed to be general and could be applicable to addnl. kinases inhibited by the promiscuous ATP-competitive fragment used in our studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLbpY-YSwyA7Vg90H21EOLACvtfcHk0ljtySFEauLTHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXkvFWjurs%253D&md5=3d981ee3bfa8053f43de9ab4034c89b3</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fcb501048b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb501048b%26sid%3Dliteratum%253Aachs%26aulast%3DBrandvold%26aufirst%3DK.%2BR.%26aulast%3DSantos%26aufirst%3DS.%2BM.%26aulast%3DBreen%26aufirst%3DM.%2BE.%26aulast%3DLachacz%26aufirst%3DE.%2BJ.%26aulast%3DSteffey%26aufirst%3DM.%2BE.%26aulast%3DSoellner%26aufirst%3DM.%2BB.%26atitle%3DExquisitely%2520Specific%2520Bisubstrate%2520Inhibitors%2520of%2520C-Src%2520Kinase%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D1387%26epage%3D1391%26doi%3D10.1021%2Fcb501048b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, and Kinetic Analysis of Potent Protein N-Terminal Methyltransferase 1 Inhibitors</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">4149</span>– <span class="NLM_lpage">4154</span>, <span class="refDoi"> DOI: 10.1039/C5OB00120J</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1039%2FC5OB00120J" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=25712161" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjtVemtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2015&pages=4149-4154&author=G.+Zhangauthor=S.+L.+Richardsonauthor=Y.+Maoauthor=R.+Huang&title=Design%2C+Synthesis%2C+and+Kinetic+Analysis+of+Potent+Protein+N-Terminal+Methyltransferase+1+Inhibitors&doi=10.1039%2FC5OB00120J"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis, and kinetic analysis of potent protein N-terminal methyltransferase 1 inhibitors</span></div><div class="casAuthors">Zhang, Gang; Richardson, Stacie Lynn; Mao, Yunfei; Huang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4149-4154</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The protein N-terminal methyltransferase 1 (NTMT1) methylates the α-N-terminal amines of proteins.  NTMT1 is upregulated in a variety of cancers and knockdown of NTMT1 results in cell mitotic defects.  Therefore, NTMT1 inhibitors could be potential anticancer therapeutics.  This study describes the design and synthesis of the first inhibitor targeting NTMT1.  A novel bisubstrate analog (NAM-TZ-SPKRIA) was shown to be a potent inhibitor (Ki = 0.20 μM) for NTMT1 and was selective vs. protein lysine methyltransferase G9a and arginine methyltransferase 1.  NAM-TZ-SPKRIA was found to exhibit a competitive inhibition pattern for both substrates, and mass spectrometry expts. revealed that the inhibitor substantially suppressed the methylation progression.  Our results demonstrate the feasibility of using a triazole group to link an S-adenosyl-L-methionine (SAM) analog with a peptide substrate to construct bisubstrate analogs as NTMT1 potent and selective inhibitors.  This study lays a foundation to further discover small mol. NTMT1 inhibitors to interrogate its biol. functions, and suggests a general strategy for the development of selective protein methyltransferase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpChRkaQTjE77Vg90H21EOLACvtfcHk0ljtySFEauLTHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjtVemtL4%253D&md5=b39327aa5e759ec16c3551367609c24f</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1039%2FC5OB00120J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5OB00120J%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DRichardson%26aufirst%3DS.%2BL.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520Kinetic%2520Analysis%2520of%2520Potent%2520Protein%2520N-Terminal%2520Methyltransferase%25201%2520Inhibitors%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2015%26volume%3D13%26spage%3D4149%26epage%3D4154%26doi%3D10.1039%2FC5OB00120J" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Facile Synthesis of SAM-Peptide Conjugates through Alkyl Linkers Targeting Protein N-Terminal Methyltransferase 1</span>. <i>RSC Adv.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">6768</span>– <span class="NLM_lpage">6771</span>, <span class="refDoi"> DOI: 10.1039/C5RA20625A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1039%2FC5RA20625A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=27588169" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC28Xlt1Krtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=6768-6771&author=G.+Zhangauthor=R.+Huang&title=Facile+Synthesis+of+SAM-Peptide+Conjugates+through+Alkyl+Linkers+Targeting+Protein+N-Terminal+Methyltransferase+1&doi=10.1039%2FC5RA20625A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Facile synthesis of SAM-peptide conjugates through alkyl linkers targeting protein N-terminal methyltransferase 1</span></div><div class="casAuthors">Zhang, Gang; Huang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">RSC Advances</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">6768-6771</span>CODEN:
                <span class="NLM_cas:coden">RSCACL</span>;
        ISSN:<span class="NLM_cas:issn">2046-2069</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The authors report the first chem. synthesis of SAM-peptide (SAM = S-adenosyl-L-methionine) conjugates through alkyl linkers to prep. bisubstrate analogs for protein methyltransferases.  The authors demonstrate its application by developing a series of bisubstrate inhibitors for protein N-terminal methyltransferase 1 (NTMT1) and the most potent one exhibits a Ki = 310 ± 55 nM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_MNtkSc3Ne7Vg90H21EOLACvtfcHk0ljtySFEauLTHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xlt1Krtw%253D%253D&md5=e267ad97193784986c32b86c2e1ae88a</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1039%2FC5RA20625A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5RA20625A%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DFacile%2520Synthesis%2520of%2520SAM-Peptide%2520Conjugates%2520through%2520Alkyl%2520Linkers%2520Targeting%2520Protein%2520N-Terminal%2520Methyltransferase%25201%26jtitle%3DRSC%2520Adv.%26date%3D2016%26volume%3D6%26spage%3D6768%26epage%3D6771%26doi%3D10.1039%2FC5RA20625A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haren, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marechal, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Troffer-charlier, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cianciulli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sbardella, G.</span></span> <span> </span><span class="NLM_article-title">Transition State Mimics Are Valuable Mechanistic Probes for Structural Studies with the Arginine</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">E3652</span>– <span class="NLM_lpage">E3658</span>, <span class="refDoi"> DOI: 10.1073/pnas.1620470114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1073%2Fpnas.1620470114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=28416684" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=E3652-E3658&author=M.+J.+Harenauthor=+Vanauthor=N.+Marechalauthor=N.+Troffer-charlierauthor=A.+Cianciulliauthor=G.+Sbardella&title=Transition+State+Mimics+Are+Valuable+Mechanistic+Probes+for+Structural+Studies+with+the+Arginine&doi=10.1073%2Fpnas.1620470114"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1620470114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1620470114%26sid%3Dliteratum%253Aachs%26aulast%3DHaren%26aufirst%3DM.%2BJ.%26aulast%3DVan%26aulast%3DMarechal%26aufirst%3DN.%26aulast%3DTroffer-charlier%26aufirst%3DN.%26aulast%3DCianciulli%26aufirst%3DA.%26aulast%3DSbardella%26aufirst%3DG.%26atitle%3DTransition%2520State%2520Mimics%2520Are%2520Valuable%2520Mechanistic%2520Probes%2520for%2520Structural%2520Studies%2520with%2520the%2520Arginine%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2017%26volume%3D114%26spage%3DE3652%26epage%3DE3658%26doi%3D10.1073%2Fpnas.1620470114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Osborne, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roska, R. L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajski, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, P. R.</span></span> <span> </span><span class="NLM_article-title">In Situ Generation of a Bisubstrate Analogue for Protein Arginine Methyltransferase 1</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>130</i></span>,  <span class="NLM_fpage">4574</span>– <span class="NLM_lpage">4575</span>, <span class="refDoi"> DOI: 10.1021/ja077104v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja077104v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjtFygtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2008&pages=4574-4575&author=T.+Osborneauthor=R.+L.+W.+Roskaauthor=S.+R.+Rajskiauthor=P.+R.+Thompson&title=In+Situ+Generation+of+a+Bisubstrate+Analogue+for+Protein+Arginine+Methyltransferase+1&doi=10.1021%2Fja077104v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">In Situ Generation of a Bisubstrate Analogue for Protein Arginine Methyltransferase 1</span></div><div class="casAuthors">Osborne, Tanesha; Roska, Rachel L. Weller; Rajski, Scott R.; Thompson, Paul R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4574-4575</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferases (PRMTs) are (S)-adenosylmethionine (SAM)-dependent methyltransferases that catalyze the post-translational methylation of Arg residues in a variety of different proteins involved in transcriptional regulation and RNA splicing (e.g., histones H2A, H3, and H4).  Herein, we describe the use of an N-mustard, 5'-(diaminobutyric acid)-N-iodoethyl-5'-deoxyadenosine ammonium hydrochloride (AAI), to generate a bisubstrate analog inhibitor of PRMT1.  Using the approach outlined in this communication, it should be possible to generate bisubstrate analog-based inhibitors of PRMT isoenzymes that are potent and highly selective for a particular isoenzyme.  The fact that PRMT1 catalyzes AAI transfer is also significant because with appropriate modifications (e.g., functionalization with pendant azido or alkyne functionalities) this compd. could be used for proteomic applications to identify novel PRMT substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9Yqi1phgFBLVg90H21EOLACvtfcHk0lhMSeM8fXFIZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjtFygtb0%253D&md5=1433435a8942d1075a01cc39a6ffce99</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fja077104v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja077104v%26sid%3Dliteratum%253Aachs%26aulast%3DOsborne%26aufirst%3DT.%26aulast%3DRoska%26aufirst%3DR.%2BL.%2BW.%26aulast%3DRajski%26aufirst%3DS.%2BR.%26aulast%3DThompson%26aufirst%3DP.%2BR.%26atitle%3DIn%2520Situ%2520Generation%2520of%2520a%2520Bisubstrate%2520Analogue%2520for%2520Protein%2520Arginine%2520Methyltransferase%25201%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26spage%3D4574%26epage%3D4575%26doi%3D10.1021%2Fja077104v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Policarpo, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Decultot, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">May, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuzmic, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhakshinamoorthy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kannt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dittakavi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danarese, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaudet, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shair, M. D.</span></span> <span> </span><span class="NLM_article-title">High-Affinity Alkynyl Bisubstrate Inhibitors of Nicotinamide N-Methyltransferase (NNMT)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">9837</span>– <span class="NLM_lpage">9873</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01238</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01238" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFWltrzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=9837-9873&author=R.+L.+Policarpoauthor=L.+Decultotauthor=E.+Mayauthor=P.+Kuzmicauthor=S.+Carlsonauthor=D.+Huangauthor=V.+Chuauthor=B.+A.+Wrightauthor=S.+Dhakshinamoorthyauthor=A.+Kanntauthor=S.+Raniauthor=S.+Dittakaviauthor=J.+D.+Danareseauthor=R.+Gaudetauthor=M.+D.+Shair&title=High-Affinity+Alkynyl+Bisubstrate+Inhibitors+of+Nicotinamide+N-Methyltransferase+%28NNMT%29&doi=10.1021%2Facs.jmedchem.9b01238"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">High-Affinity Alkynyl Bisubstrate Inhibitors of Nicotinamide N-Methyltransferase (NNMT)</span></div><div class="casAuthors">Policarpo, Rocco L.; Decultot, Ludovic; May, Elizabeth; Kuzmic, Petr; Carlson, Samuel; Huang, Danny; Chu, Vincent; Wright, Brandon A.; Dhakshinamoorthy, Saravanakumar; Kannt, Aimo; Rani, Shilpa; Dittakavi, Sreekanth; Panarese, Joseph D.; Gaudet, Rachelle; Shair, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9837-9873</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) is a metabolic enzyme that methylates nicotinamide (NAM) using cofactor S-adenosylmethionine (SAM).  NNMT overexpression has been linked to diabetes, obesity, and various cancers.  In this work, structure-based rational design led to the development of potent and selective alkynyl bisubstrate inhibitors of NNMT.  The reported nicotinamide-SAM conjugate (named NS1) features an alkyne as a key design element that closely mimics the linear, 180° transition state geometry found in the NNMT-catalyzed SAM → NAM Me transfer reaction.  NS1 was synthesized in 14 steps and found to be a high-affinity, subnanomolar NNMT inhibitor.  An X-ray cocrystal structure and SAR study revealed the ability of an alkynyl linker to span the Me transfer tunnel of NNMT with ideal shape complementarity.  The compds. reported in this work represent the most potent and selective NNMT inhibitors reported to date.  The rational design principle described herein could potentially be extended to other methyltransferase enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLrzDcwfc0ibVg90H21EOLACvtfcHk0lhMSeM8fXFIZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFWltrzM&md5=f62a8055840292563d07cd25b0fcb4c2</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01238%26sid%3Dliteratum%253Aachs%26aulast%3DPolicarpo%26aufirst%3DR.%2BL.%26aulast%3DDecultot%26aufirst%3DL.%26aulast%3DMay%26aufirst%3DE.%26aulast%3DKuzmic%26aufirst%3DP.%26aulast%3DCarlson%26aufirst%3DS.%26aulast%3DHuang%26aufirst%3DD.%26aulast%3DChu%26aufirst%3DV.%26aulast%3DWright%26aufirst%3DB.%2BA.%26aulast%3DDhakshinamoorthy%26aufirst%3DS.%26aulast%3DKannt%26aufirst%3DA.%26aulast%3DRani%26aufirst%3DS.%26aulast%3DDittakavi%26aufirst%3DS.%26aulast%3DDanarese%26aufirst%3DJ.%2BD.%26aulast%3DGaudet%26aufirst%3DR.%26aulast%3DShair%26aufirst%3DM.%2BD.%26atitle%3DHigh-Affinity%2520Alkynyl%2520Bisubstrate%2520Inhibitors%2520of%2520Nicotinamide%2520N-Methyltransferase%2520%2528NNMT%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D9837%26epage%3D9873%26doi%3D10.1021%2Facs.jmedchem.9b01238" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loppnau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span> <span> </span><span class="NLM_article-title">Structural Basis for Substrate Recognition by the Human N-Terminal Methyltransferase 1</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">2343</span>– <span class="NLM_lpage">2348</span>, <span class="refDoi"> DOI: 10.1101/gad.270611.115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1101%2Fgad.270611.115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=26543161" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC28XntV2rtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=2343-2348&author=C.+Dongauthor=Y.+Maoauthor=W.+Tempelauthor=S.+Qinauthor=L.+Liauthor=P.+Loppnauauthor=R.+Huangauthor=J.+Min&title=Structural+Basis+for+Substrate+Recognition+by+the+Human+N-Terminal+Methyltransferase+1&doi=10.1101%2Fgad.270611.115"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for substrate recognition by the human N-terminal methyltransferase 1</span></div><div class="casAuthors">Dong, Cheng; Mao, Yunfei; Tempel, Wolfram; Qin, Su; Li, Li; Loppnau, Peter; Huang, Rong; Min, Jinrong</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">2343-2348</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">α-N-terminal methylation represents a highly conserved and prevalent post-translational modification, yet its biol. function has remained largely speculative.  The recent discovery of α-N-terminal methyltransferase 1 (NTMT1) and its physiol. substrates propels the elucidation of a general role of α-N-terminal methylation in mediating DNA-binding ability of the modified proteins.  The phenotypes, obsd. from both NTMT1 knockdown in breast cancer cell lines and knockout mouse models, suggest the potential involvement of α-N-terminal methylation in DNA damage response and cancer development.  In this study, we report the first crystal structures of human NTMT1 in complex with cofactor S-adenosyl-L-homocysteine (SAH) and six substrate peptides, resp., and reveal that NTMT1 contains two characteristic structural elements (a β hairpin and an N-terminal extension) that contribute to its substrate specificity.  Our complex structures, coupled with mutagenesis, binding, and enzymic studies, also present the key elements involved in locking the consensus substrate motif XPK (X indicates any residue type other than D/E) into the catalytic pocket for α-N-terminal methylation and explain why NTMT1 prefers an XPK sequence motif.  We propose a catalytic mechanism for α-N-terminal methylation.  Overall, this study gives us the first glimpse of the mol. mechanism of α-N-terminal methylation and potentially contributes to the advent of therapeutic agents for human diseases assocd. with deregulated α-N-terminal methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrv_R_NbrFwa7Vg90H21EOLACvtfcHk0lhMSeM8fXFIZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XntV2rtLc%253D&md5=3f9e7d4ad9c2abd5af04d615e71d1777</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1101%2Fgad.270611.115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.270611.115%26sid%3Dliteratum%253Aachs%26aulast%3DDong%26aufirst%3DC.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DQin%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLoppnau%26aufirst%3DP.%26aulast%3DHuang%26aufirst%3DR.%26aulast%3DMin%26aufirst%3DJ.%26atitle%3DStructural%2520Basis%2520for%2520Substrate%2520Recognition%2520by%2520the%2520Human%2520N-Terminal%2520Methyltransferase%25201%26jtitle%3DGenes%2520Dev.%26date%3D2015%26volume%3D29%26spage%3D2343%26epage%3D2348%26doi%3D10.1101%2Fgad.270611.115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span> <span> </span><span class="NLM_article-title">Molecular Basis for Histone N-Terminal Methylation by NRMT1</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">2337</span>– <span class="NLM_lpage">2342</span>, <span class="refDoi"> DOI: 10.1101/gad.270926.115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1101%2Fgad.270926.115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=26543159" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1yltb7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=2337-2342&author=R.+Wuauthor=Y.+Yueauthor=X.+Zhengauthor=H.+Li&title=Molecular+Basis+for+Histone+N-Terminal+Methylation+by+NRMT1&doi=10.1101%2Fgad.270926.115"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular basis for histone N-terminal methylation by NRMT1</span></div><div class="casAuthors">Wu, Ruoxi; Yue, Yuan; Zheng, Xiangdong; Li, Haitao</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">2337-2342</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">NRMT1 is an N-terminal methyltransferase that methylates histone CENP-A as well as nonhistone substrates.  Here, we report the crystal structure of human NRMT1 bound to CENP-A peptide at 1.3 Å.  NRMT1 adopts a core methyltransferase fold that resembles DOT1L and PRMT but not SET domain family histone methyltransferases.  Key substrate recognition and catalytic residues were identified by mutagenesis studies.  Histone peptide profiling revealed that human NRMT1 is highly selective to human CENP-A and fruit fly H2B, which share a common "Xaa-Pro-Lys/Arg" motif.  These results, along with a 1.5 Å costructure of human NRMT1 bound to the fruit fly H2B peptide, underscore the importance of the NRMT1 recognition motif.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGIuplVjQhULVg90H21EOLACvtfcHk0lhO0V6Hl9yBlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1yltb7J&md5=f3601b0966da6c89d0acf9aeae4799be</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1101%2Fgad.270926.115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.270926.115%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DR.%26aulast%3DYue%26aufirst%3DY.%26aulast%3DZheng%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DH.%26atitle%3DMolecular%2520Basis%2520for%2520Histone%2520N-Terminal%2520Methylation%2520by%2520NRMT1%26jtitle%3DGenes%2520Dev.%26date%3D2015%26volume%3D29%26spage%3D2337%26epage%3D2342%26doi%3D10.1101%2Fgad.270926.115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petkowski, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaner Tooley, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shumilin, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balsbaugh, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macara, I. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shabanowitz, J.</span></span> <span> </span><span class="NLM_article-title">Substrate Specificity of Mammalian N-Terminal α -Amino Methyltransferase NRMT</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">5942</span>– <span class="NLM_lpage">5950</span>, <span class="refDoi"> DOI: 10.1021/bi300278f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi300278f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC38XpvVars78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=5942-5950&author=J.+J.+Petkowskiauthor=C.+E.+Schaner+Tooleyauthor=L.+C.+Andersonauthor=I.+A.+Shumilinauthor=J.+L.+Balsbaughauthor=D.+F.+Huntauthor=W.+Minorauthor=I.+G.+Macaraauthor=J.+Shabanowitz&title=Substrate+Specificity+of+Mammalian+N-Terminal+%CE%B1+-Amino+Methyltransferase+NRMT&doi=10.1021%2Fbi300278f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Substrate Specificity of Mammalian N-Terminal α-Amino Methyltransferase NRMT</span></div><div class="casAuthors">Petkowski, Janusz J.; Schaner Tooley, Christine E.; Anderson, Lissa C.; Shumilin, Igor A.; Balsbaugh, Jeremy L.; Shabanowitz, Jeffrey; Hunt, Donald F.; Minor, Wladek; Macara, Ian G.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">5942-5950</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">N-Terminal methylation of free α-amino groups is a post-translational modification of proteins that was first described 30 years ago but has been studied very little.  In this modification, the initiating M residue is cleaved and the exposed α-amino group is mono-, di-, or trimethylated by NRMT, a recently identified N-terminal methyltransferase.  Currently, all known eukaryotic α-amino-methylated proteins have a unique N-terminal motif, M-X-P-K, where X is A, P, or S.  NRMT can also methylate artificial substrates in vitro in which X is G, F, Y, C, M, K, R, N, Q, or H.  Methylation efficiencies of N-terminal amino acids are variable with respect to the identity of X.  Here we use in vitro peptide methylation assays and substrate immunopptns. to show that the canonical M-X-P-K methylation motif is not the only one recognized by NRMT.  We predict that N-terminal methylation is a widespread post-translational modification and that there is interplay between N-terminal acetylation and N-terminal methylation.  We also use isothermal calorimetry expts. to demonstrate that NRMT can efficiently recognize and bind to its fully methylated products.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoR8dl8ByIopbVg90H21EOLACvtfcHk0lhO0V6Hl9yBlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpvVars78%253D&md5=dbddebac4bc6f6045fd144604e157bdb</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fbi300278f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi300278f%26sid%3Dliteratum%253Aachs%26aulast%3DPetkowski%26aufirst%3DJ.%2BJ.%26aulast%3DSchaner%2BTooley%26aufirst%3DC.%2BE.%26aulast%3DAnderson%26aufirst%3DL.%2BC.%26aulast%3DShumilin%26aufirst%3DI.%2BA.%26aulast%3DBalsbaugh%26aufirst%3DJ.%2BL.%26aulast%3DHunt%26aufirst%3DD.%2BF.%26aulast%3DMinor%26aufirst%3DW.%26aulast%3DMacara%26aufirst%3DI.%2BG.%26aulast%3DShabanowitz%26aufirst%3DJ.%26atitle%3DSubstrate%2520Specificity%2520of%2520Mammalian%2520N-Terminal%2520%25CE%25B1%2520-Amino%2520Methyltransferase%2520NRMT%26jtitle%3DBiochemistry%26date%3D2012%26volume%3D51%26spage%3D5942%26epage%3D5950%26doi%3D10.1021%2Fbi300278f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dorgan, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wooderchak, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wynn, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karschner, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alfaro, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Z. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hevel, J. M.</span></span> <span> </span><span class="NLM_article-title">An Enzyme-Coupled Continuous Spectrophotometric Assay for S-Adenosylmethionine-Dependent Methyltransferases</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>350</i></span>,  <span class="NLM_fpage">249</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1016/j.ab.2006.01.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1016%2Fj.ab.2006.01.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=16460659" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BD28XitVeht7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=350&publication_year=2006&pages=249-255&author=K.+M.+Dorganauthor=W.+L.+Wooderchakauthor=D.+P.+Wynnauthor=E.+L.+Karschnerauthor=J.+F.+Alfaroauthor=Y.+Cuiauthor=Z.+S.+Zhouauthor=J.+M.+Hevel&title=An+Enzyme-Coupled+Continuous+Spectrophotometric+Assay+for+S-Adenosylmethionine-Dependent+Methyltransferases&doi=10.1016%2Fj.ab.2006.01.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">An enzyme-coupled continuous spectrophotometric assay for S-adenosylmethionine-dependent methyltransferases</span></div><div class="casAuthors">Dorgan, Kathleen M.; Wooderchak, Whitney L.; Wynn, DonRaphael P.; Karschner, Erin L.; Alfaro, Joshua F.; Cui, Yinqiu; Zhou, Zhaohui Sunny; Hevel, Joan M.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">350</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">249-255</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Modification of small mols. and proteins by methyltransferases affects a wide range of biol. processes.  Here, we report an enzyme-coupled continuous spectrophotometric assay to quant. characterize S-adenosyl-L-methionine (AdoMet/SAM)-dependent methyltransferase activity.  In this assay, S-adenosyl-L-homocysteine (AdoHcy/SAH), the transmethylation product of AdoMet-dependent methyltransferases, is hydrolyzed to S-ribosylhomocysteine and adenine by recombinant S-adenosylhomocysteine/5'-methylthioadenosine nucleosidase (SAHN/MTAN, EC 3.2.2.9).  Subsequently, adenine generated from AdoHcy is further hydrolyzed to hypoxanthine and ammonia by recombinant adenine deaminase (EC 3.5.4.2).  This deamination is assocd. with a decrease in absorbance at 265 nm that can be monitored continuously.  Coupling enzymes are recombinant and easily purified.  The utility of this assay was shown using recombinant rat protein arginine N-methyltransferase 1 (PRMT1, EC 2.1.1.125), which catalyzes the mono- and dimethylation of guanidino nitrogens of arginine residues in select proteins.  Using this assay, the kinetic parameters of PRMT1 with three synthetic peptides were detd.  An advantage of this assay is the destruction of AdoHcy by AdoHcy nucleosidase, which alleviates AdoHcy product feedback inhibition of S-adenosylmethionine-dependent methyltransferases.  Finally, this method may be used to assay other enzymes that produce AdoHcy, 5'-methylthioadenosine, or compds. that can be cleaved by AdoHcy nucleosidase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkcuo70A9XQbVg90H21EOLACvtfcHk0li3O1rXzu6HQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XitVeht7s%253D&md5=f83416f4f10bbe745b7a0873f9950a38</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2006.01.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2006.01.004%26sid%3Dliteratum%253Aachs%26aulast%3DDorgan%26aufirst%3DK.%2BM.%26aulast%3DWooderchak%26aufirst%3DW.%2BL.%26aulast%3DWynn%26aufirst%3DD.%2BP.%26aulast%3DKarschner%26aufirst%3DE.%2BL.%26aulast%3DAlfaro%26aufirst%3DJ.%2BF.%26aulast%3DCui%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DZ.%2BS.%26aulast%3DHevel%26aufirst%3DJ.%2BM.%26atitle%3DAn%2520Enzyme-Coupled%2520Continuous%2520Spectrophotometric%2520Assay%2520for%2520S-Adenosylmethionine-Dependent%2520Methyltransferases%26jtitle%3DAnal.%2520Biochem.%26date%3D2006%26volume%3D350%26spage%3D249%26epage%3D255%26doi%3D10.1016%2Fj.ab.2006.01.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, A. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panchenko, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sathyan, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petkowski, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pai, P.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russell, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macara, I. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shabanowitz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Black, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flotz, D. R.</span></span> <span> </span><span class="NLM_article-title">Posttranslational Modification of CENP-A Influences the Conformation of Centromeric Chromatin</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">11827</span>– <span class="NLM_lpage">11832</span>, <span class="refDoi"> DOI: 10.1073/pnas.1300325110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1073%2Fpnas.1300325110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=23818633" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1emurvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=11827-11832&author=A.+O.+Baileyauthor=T.+Panchenkoauthor=K.+M.+Sathyanauthor=J.+J.+Petkowskiauthor=P.-J.+Paiauthor=D.+L.+Baiauthor=D.+H.+Russellauthor=I.+G.+Macaraauthor=J.+Shabanowitzauthor=D.+F.+Huntauthor=B.+E.+Blackauthor=D.+R.+Flotz&title=Posttranslational+Modification+of+CENP-A+Influences+the+Conformation+of+Centromeric+Chromatin&doi=10.1073%2Fpnas.1300325110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Posttranslational modification of CENP-A influences the conformation of centromeric chromatin</span></div><div class="casAuthors">Bailey, Aaron O.; Panchenko, Tanya; Sathyan, Kizhakke M.; Petkowski, Janusz J.; Pai, Pei-Jing; Bai, Dina L.; Russell, David H.; Macara, Ian G.; Shabanowitz, Jeffrey; Hunt, Donald F.; Black, Ben E.; Foltz, Daniel R.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">11827-11832,S11827/1-S11827/14</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Centromeres are chromosomal loci required for accurate segregation of sister chromatids during mitosis.  The location of the centromere on the chromosome is not dependent on DNA sequence, but rather it is epigenetically specified by the histone H3 variant centromere protein A (CENP-A).  The N-terminal tail of CENP-A is highly divergent from other H3 variants.  Canonical histone N termini are hotspots of conserved post-translational modification; however, no broadly conserved modifications of the vertebrate CENP-A tail have been previously obsd.  Here, we report three post-translational modifications on human CENP-A N termini using high-resoln. MS: trimethylation of Gly1 and phosphorylation of Ser16 and Ser18.  Our results demonstrate that CENP-A is subjected to constitutive initiating methionine removal, similar to other H3 variants.  The nascent N-terminal residue Gly1 becomes trimethylated on the α-amino group.  We demonstrate that the N-terminal RCC1 methyltransferase is capable of modifying the CENP-A N terminus.  Methylation occurs in the prenucleosomal form and marks the majority of CENP-A nucleosomes.  Serine 16 and 18 become phosphorylated in prenucleosomal CENP-A and are phosphorylated on asynchronous and mitotic nucleosomal CENP-A and are important for chromosome segregation during mitosis.  The double phosphorylation motif forms a salt-bridged secondary structure and causes CENP-A N-terminal tails to form intramol. assocns.  Anal. ultracentrifugation of phospho-mimetic CENP-A nucleosome arrays demonstrates that phosphorylation results in greater intranucleosome assocns. and counteracts the hyperoligomerized state exhibited by unmodified CENP-A nucleosome arrays.  Our studies have revealed that the major modifications on the N-terminal tail of CENP-A alter the phys. properties of the chromatin fiber at the centromere.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7Tb1icoBc4LVg90H21EOLACvtfcHk0li3O1rXzu6HQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1emurvE&md5=246e391333b51fa7ba0e0a2d80498c03</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1300325110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1300325110%26sid%3Dliteratum%253Aachs%26aulast%3DBailey%26aufirst%3DA.%2BO.%26aulast%3DPanchenko%26aufirst%3DT.%26aulast%3DSathyan%26aufirst%3DK.%2BM.%26aulast%3DPetkowski%26aufirst%3DJ.%2BJ.%26aulast%3DPai%26aufirst%3DP.-J.%26aulast%3DBai%26aufirst%3DD.%2BL.%26aulast%3DRussell%26aufirst%3DD.%2BH.%26aulast%3DMacara%26aufirst%3DI.%2BG.%26aulast%3DShabanowitz%26aufirst%3DJ.%26aulast%3DHunt%26aufirst%3DD.%2BF.%26aulast%3DBlack%26aufirst%3DB.%2BE.%26aulast%3DFlotz%26aufirst%3DD.%2BR.%26atitle%3DPosttranslational%2520Modification%2520of%2520CENP-A%2520Influences%2520the%2520Conformation%2520of%2520Centromeric%2520Chromatin%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2013%26volume%3D110%26spage%3D11827%26epage%3D11832%26doi%3D10.1073%2Fpnas.1300325110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otake, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masumoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span> <span> </span><span class="NLM_article-title">Identification of Novel α-N-Methylation of CENP-B That Regulates Its Binding to the Centromeric DNA</span>. <i>J. Proteome Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">4167</span>– <span class="NLM_lpage">4175</span>, <span class="refDoi"> DOI: 10.1021/pr400498y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/pr400498y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Wqt7zK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=4167-4175&author=X.+Daiauthor=K.+Otakeauthor=C.+Youauthor=Q.+Caiauthor=Z.+Wangauthor=H.+Masumotoauthor=Y.+Wang&title=Identification+of+Novel+%CE%B1-N-Methylation+of+CENP-B+That+Regulates+Its+Binding+to+the+Centromeric+DNA&doi=10.1021%2Fpr400498y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of novel α-N-methylation of CENP-B that regulates its binding to the centromeric DNA</span></div><div class="casAuthors">Dai, Xiaoxia; Otake, Koichiro; You, Changjun; Cai, Qian; Wang, Zi; Masumoto, Hiroshi; Wang, Yinsheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4167-4175</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The eukaryotic centromere is an essential chromatin region required for accurate segregation of sister chromatids during cell division.  Centromere protein B (CENP-B) is a highly conserved protein which can bind to the 17-bp CENP-B box on the centromeric DNA.  In this study, the authors found that CENP-B could be α-N-methylated in human cells.  The level of the α-N-methylation was stimulated in cells in response to a variety of extracellular stimuli, including increased cell d., heat shock, and arsenite treatment, although the methylation level was not altered upon metaphase arrest.  N-terminal RCC1 methyltransferase (NRMT) was identified as a major enzyme required for the CENP-B methylation.  Addnl., chromatin-bound CENP-B was primarily trimethylated and α-N-trimethylation could enhance CENP-B's binding to CENP-B box in cells.  These study also expands the function of protein α-N-methylation that has been known for decades and whose function remains largely unexplored.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCWj3AJ08MjLVg90H21EOLACvtfcHk0li3O1rXzu6HQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1Wqt7zK&md5=f0a6400d0765d888861badd1aafe7c73</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fpr400498y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fpr400498y%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DX.%26aulast%3DOtake%26aufirst%3DK.%26aulast%3DYou%26aufirst%3DC.%26aulast%3DCai%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DMasumoto%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DY.%26atitle%3DIdentification%2520of%2520Novel%2520%25CE%25B1-N-Methylation%2520of%2520CENP-B%2520That%2520Regulates%2520Its%2520Binding%2520to%2520the%2520Centromeric%2520DNA%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2013%26volume%3D12%26spage%3D4167%26epage%3D4175%26doi%3D10.1021%2Fpr400498y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cai, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span> <span> </span><span class="NLM_article-title">α-N-Methylation of Damaged DNA-Binding Protein 2 (DDB2) and Its Function in Nucleotide Excision Repair</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>289</i></span>,  <span class="NLM_fpage">16046</span>– <span class="NLM_lpage">16056</span>, <span class="refDoi"> DOI: 10.1074/jbc.M114.558510</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1074%2Fjbc.M114.558510" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=24753253" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC2cXptl2qsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=2014&pages=16046-16056&author=Q.+Caiauthor=L.+Fuauthor=Z.+Wangauthor=N.+Ganauthor=X.+Daiauthor=Y.+Wang&title=%CE%B1-N-Methylation+of+Damaged+DNA-Binding+Protein+2+%28DDB2%29+and+Its+Function+in+Nucleotide+Excision+Repair&doi=10.1074%2Fjbc.M114.558510"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">α-N-Methylation of damaged DNA-binding protein 2 (DDB2) and its function in nucleotide excision repair</span></div><div class="casAuthors">Cai, Qian; Fu, Lijuan; Wang, Zi; Gan, Nanqin; Dai, Xiaoxia; Wang, Yinsheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">289</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">16046-16056</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">DDB2 exhibits a high affinity toward UV-damaged DNA, and it is involved in the initial steps of global genome nucleotide excision repair.  Mutations in the DDB2 gene cause the genetic complementation group E of xeroderma pigmentosum, an autosomal recessive disease manifested clin. by hypersensitivity to sunlight exposure and an increased predisposition to skin cancer.  Here the authors found that, in human cells, the initiating methionine residue in DDB2 was removed and that the N-terminal alanine could be methylated on its α-amino group in human cells, with trimethylation being the major form.  The α-N-methylation of DDB2 is catalyzed by the N-terminal RCC1 methyltransferase.  In addn., a methylation-defective mutant of DDB2 displayed diminished nuclear localization and was recruited at a reduced efficiency to UV-induced cyclobutane pyrimidine dimer foci.  Moreover, loss of this methylation conferred compromised ATM (ataxia telangiectasia mutated) activation, decreased efficiency in cyclobutane pyrimidine dimer repair, and elevated sensitivity of cells toward UV light exposure.  This study provides new knowledge about the posttranslational regulation of DDB2 and expands the biol. functions of protein α-N-methylation to DNA repair.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpc3yE8db8RO7Vg90H21EOLACvtfcHk0lhn3zKkxpF9GQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXptl2qsLs%253D&md5=e52a0c303e3420df4722d42cb0478af1</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M114.558510&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M114.558510%26sid%3Dliteratum%253Aachs%26aulast%3DCai%26aufirst%3DQ.%26aulast%3DFu%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DGan%26aufirst%3DN.%26aulast%3DDai%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DY.%26atitle%3D%25CE%25B1-N-Methylation%2520of%2520Damaged%2520DNA-Binding%2520Protein%25202%2520%2528DDB2%2529%2520and%2520Its%2520Function%2520in%2520Nucleotide%2520Excision%2520Repair%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2014%26volume%3D289%26spage%3D16046%26epage%3D16056%26doi%3D10.1074%2Fjbc.M114.558510" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rulten, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldecott, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span> <span> </span><span class="NLM_article-title">Identification and Functional Characterizations of N-Terminal α-N-Methylation and Phosphorylation of Serine 461 in Human Poly(ADP-Ribose) Polymerase 3</span>. <i>J. Proteome Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">2575</span>– <span class="NLM_lpage">2582</span>, <span class="refDoi"> DOI: 10.1021/acs.jproteome.5b00126</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jproteome.5b00126" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC2MXms1OgsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=2575-2582&author=X.+Daiauthor=S.+L.+Rultenauthor=C.+Youauthor=K.+W.+Caldecottauthor=Y.+Wang&title=Identification+and+Functional+Characterizations+of+N-Terminal+%CE%B1-N-Methylation+and+Phosphorylation+of+Serine+461+in+Human+Poly%28ADP-Ribose%29+Polymerase+3&doi=10.1021%2Facs.jproteome.5b00126"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Functional Characterizations of N-Terminal α-N-Methylation and Phosphorylation of Serine 461 in Human Poly(ADP-ribose) Polymerase 3</span></div><div class="casAuthors">Dai, Xiaoxia; Rulten, Stuart L.; You, Changjun; Caldecott, Keith W.; Wang, Yinsheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2575-2582</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Poly(ADP-ribose) polymerase 3 (PARP3) is a member of the PARP family enzymes which catalyze the ADP-ribosylation of proteins.  PARP3 plays an important role in DNA damage repair and mitotic progression.  In this study, we identified, using mass spectrometric (MS) techniques, two novel post-translational modification sites in PARP3, α-N-methylation and phosphorylation of serine 461 (S461).  We found that the N-terminal α-amino group of PARP3 is heavily methylated in human cells, and N-terminal RCC1 methyltransferase (NRMT) is a key enzyme required for this methylation.  We also obsd. that the phosphorylation level of S461 in PARP3 could be reduced in human cells upon treatment with flavopiridol, a cyclin-dependent kinase inhibitor.  Moreover, we demonstrated that S461 phosphorylation, but not α-N-methylation of PARP3, may be involved in the cellular response toward DNA double-strand breaks (DSBs).  These findings provide novel insights into the post-translational regulation of PARP3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrr0mFKyXPLnrVg90H21EOLACvtfcHk0lhn3zKkxpF9GQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXms1OgsLw%253D&md5=6ff05a312e38b2105b71fbbd17388e64</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.jproteome.5b00126&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jproteome.5b00126%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DX.%26aulast%3DRulten%26aufirst%3DS.%2BL.%26aulast%3DYou%26aufirst%3DC.%26aulast%3DCaldecott%26aufirst%3DK.%2BW.%26aulast%3DWang%26aufirst%3DY.%26atitle%3DIdentification%2520and%2520Functional%2520Characterizations%2520of%2520N-Terminal%2520%25CE%25B1-N-Methylation%2520and%2520Phosphorylation%2520of%2520Serine%2520461%2520in%2520Human%2520Poly%2528ADP-Ribose%2529%2520Polymerase%25203%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2015%26volume%3D14%26spage%3D2575%26epage%3D2582%26doi%3D10.1021%2Facs.jproteome.5b00126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tooley, C. E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petkowski, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muratore-Schroeder, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balsbaugh, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shabanowitz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabat, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macara, I. G.</span></span> <span> </span><span class="NLM_article-title">NRMT Is an Alpha-N-Methyltransferase That Methylates RCC1 and Retinoblastoma Protein</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>466</i></span>,  <span class="NLM_fpage">1125</span>– <span class="NLM_lpage">1128</span>, <span class="refDoi"> DOI: 10.1038/nature09343</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1038%2Fnature09343" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=20668449" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A280%3ADC%252BC3cfhtVensQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=466&publication_year=2010&pages=1125-1128&author=C.+E.+S.+Tooleyauthor=J.+J.+Petkowskiauthor=T.+L.+Muratore-Schroederauthor=J.+L.+Balsbaughauthor=J.+Shabanowitzauthor=M.+Sabatauthor=W.+Minorauthor=D.+F.+Huntauthor=I.+G.+Macara&title=NRMT+Is+an+Alpha-N-Methyltransferase+That+Methylates+RCC1+and+Retinoblastoma+Protein&doi=10.1038%2Fnature09343"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">NRMT is an alpha-N-methyltransferase that methylates RCC1 and retinoblastoma protein</span></div><div class="casAuthors">Tooley Christine E Schaner; Petkowski Janusz J; Muratore-Schroeder Tara L; Balsbaugh Jeremy L; Shabanowitz Jeffrey; Sabat Michal; Minor Wladek; Hunt Donald F; Macara Ian G</div><div class="citationInfo"><span class="NLM_cas:title">Nature</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">466</span>
        (<span class="NLM_cas:issue">7310</span>),
    <span class="NLM_cas:pages">1125-8</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The post-translational methylation of alpha-amino groups was first discovered over 30 years ago on the bacterial ribosomal proteins L16 and L33 (refs 1, 2), but almost nothing is known about the function or enzymology of this modification.  Several other bacterial and eukaryotic proteins have since been shown to be alpha-N-methylated.  However, the Ran guanine nucleotide-exchange factor, RCC1, is the only protein for which any biological function of alpha-N-methylation has been identified.  Methylation-defective mutants of RCC1 have reduced affinity for DNA and cause mitotic defects, but further characterization of this modification has been hindered by ignorance of the responsible methyltransferase.  All fungal and animal N-terminally methylated proteins contain a unique N-terminal motif, Met-(Ala/Pro/Ser)-Pro-Lys, indicating that they may be targets of the same, unknown enzyme.  The initiating Met is cleaved, and the exposed alpha-amino group is mono-, di- or trimethylated.  Here we report the discovery of the first alpha-N-methyltransferase, which we named N-terminal RCC1 methyltransferase (NRMT).  Substrate docking and mutational analysis of RCC1 defined the NRMT recognition sequence and enabled the identification of numerous new methylation targets, including SET (also known as TAF-I or PHAPII) and the retinoblastoma protein, RB.  Knockdown of NRMT recapitulates the multi-spindle phenotype seen with methylation-defective RCC1 mutants, demonstrating the importance of alpha-N-methylation for normal bipolar spindle formation and chromosome segregation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRzswcolxsHgOxsqfvU0ziafW6udTcc2eaWj88VcZNvuLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cfhtVensQ%253D%253D&md5=e5fe130b420fc030b5af04ef304a9024</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnature09343&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09343%26sid%3Dliteratum%253Aachs%26aulast%3DTooley%26aufirst%3DC.%2BE.%2BS.%26aulast%3DPetkowski%26aufirst%3DJ.%2BJ.%26aulast%3DMuratore-Schroeder%26aufirst%3DT.%2BL.%26aulast%3DBalsbaugh%26aufirst%3DJ.%2BL.%26aulast%3DShabanowitz%26aufirst%3DJ.%26aulast%3DSabat%26aufirst%3DM.%26aulast%3DMinor%26aufirst%3DW.%26aulast%3DHunt%26aufirst%3DD.%2BF.%26aulast%3DMacara%26aufirst%3DI.%2BG.%26atitle%3DNRMT%2520Is%2520an%2520Alpha-N-Methyltransferase%2520That%2520Methylates%2520RCC1%2520and%2520Retinoblastoma%2520Protein%26jtitle%3DNature%26date%3D2010%26volume%3D466%26spage%3D1125%26epage%3D1128%26doi%3D10.1038%2Fnature09343" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonsignore, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tooley, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Hoose, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaner
Tooley, C. E.</span></span> <span> </span><span class="NLM_article-title">NRMT1 Knockout Mice Exhibit Phenotypes Associated with Impaired DNA Repair and Premature Aging</span>. <i>Mech. Ageing Dev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>146–148</i></span>,  <span class="NLM_fpage">42</span>– <span class="NLM_lpage">52</span>, <span class="refDoi"> DOI: 10.1016/j.mad.2015.03.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1016%2Fj.mad.2015.03.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=25843235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvVSmu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=146%E2%80%93148&publication_year=2015&pages=42-52&author=L.+A.+Bonsignoreauthor=J.+G.+Tooleyauthor=P.+M.+Van+Hooseauthor=E.+Wangauthor=A.+Chengauthor=M.+P.+Coleauthor=C.+E.+Schaner%0ATooley&title=NRMT1+Knockout+Mice+Exhibit+Phenotypes+Associated+with+Impaired+DNA+Repair+and+Premature+Aging&doi=10.1016%2Fj.mad.2015.03.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">NRMT1 knockout mice exhibit phenotypes associated with impaired DNA repair and premature aging</span></div><div class="casAuthors">Bonsignore, Lindsay A.; Tooley, John G.; Van Hoose, Patrick M.; Wang, Eugenia; Cheng, Alan; Cole, Marsha P.; Schaner Tooley, Christine E.</div><div class="citationInfo"><span class="NLM_cas:title">Mechanisms of Ageing and Development</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">146-148</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">42-52</span>CODEN:
                <span class="NLM_cas:coden">MAGDA3</span>;
        ISSN:<span class="NLM_cas:issn">0047-6374</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">Though defective genome maintenance and DNA repair have long been known to promote phenotypes of premature aging, the role protein methylation plays in these processes is only now emerging.  We have recently identified the first N-terminal methyltransferase, NRMT1, which regulates protein-DNA interactions and is necessary for both accurate mitotic division and nucleotide excision repair.  To demonstrate if complete loss of NRMT1 subsequently resulted in developmental or aging phenotypes, we constructed the first NRMT1 knockout (Nrmt1-/-) mouse.  The majority of these mice die shortly after birth.  However, the ones that survive, exhibit decreased body size, female-specific infertility, kyphosis, decreased mitochondrial function, and early-onset liver degeneration; phenotypes characteristic of other mouse models deficient in DNA repair.  The livers from Nrmt1-/- mice produce less reactive oxygen species (ROS) than wild type controls, and Nrmt1-/- mouse embryonic fibroblasts show a decreased capacity for handling oxidative damage.  This indicates that decreased mitochondrial function may benefit Nrmt1-/- mice and protect them from excess internal ROS and subsequent DNA damage.  These studies position the NRMT1 knockout mouse as a useful new system for studying the effects of genomic instability and defective DNA damage repair on organismal and tissue-specific aging.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWx010KT-fbrVg90H21EOLACvtfcHk0lg3DABubG8mkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvVSmu7Y%253D&md5=501bd0e3dade5777c3965efb8fe10b93</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.mad.2015.03.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mad.2015.03.012%26sid%3Dliteratum%253Aachs%26aulast%3DBonsignore%26aufirst%3DL.%2BA.%26aulast%3DTooley%26aufirst%3DJ.%2BG.%26aulast%3DVan%2BHoose%26aufirst%3DP.%2BM.%26aulast%3DWang%26aufirst%3DE.%26aulast%3DCheng%26aufirst%3DA.%26aulast%3DCole%26aufirst%3DM.%2BP.%26aulast%3DSchaner%2BTooley%26aufirst%3DC.%2BE.%26atitle%3DNRMT1%2520Knockout%2520Mice%2520Exhibit%2520Phenotypes%2520Associated%2520with%2520Impaired%2520DNA%2520Repair%2520and%2520Premature%2520Aging%26jtitle%3DMech.%2520Ageing%2520Dev.%26date%3D2015%26volume%3D146%25E2%2580%2593148%26spage%3D42%26epage%3D52%26doi%3D10.1016%2Fj.mad.2015.03.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nitrone, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keirs, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moffat, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overton, K.</span></span> <span> </span><span class="NLM_article-title">Enantioselective synthesis of optically pure (R)- and (S)-β-lysine via nitrone cycloaddition</span>. <i>J. Chem. Soc., Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_fpage">654</span>– <span class="NLM_lpage">655</span>, <span class="refDoi"> DOI: 10.1039/C39880000654</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1039%2FC39880000654" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1988&pages=654-655&author=S.+Nitroneauthor=D.+Keirsauthor=D.+Moffatauthor=K.+Overton&title=Enantioselective+synthesis+of+optically+pure+%28R%29-+and+%28S%29-%CE%B2-lysine+via+nitrone+cycloaddition&doi=10.1039%2FC39880000654"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1039%2FC39880000654&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC39880000654%26sid%3Dliteratum%253Aachs%26aulast%3DNitrone%26aufirst%3DS.%26aulast%3DKeirs%26aufirst%3DD.%26aulast%3DMoffat%26aufirst%3DD.%26aulast%3DOverton%26aufirst%3DK.%26atitle%3DEnantioselective%2520synthesis%2520of%2520optically%2520pure%2520%2528R%2529-%2520and%2520%2528S%2529-%25CE%25B2-lysine%2520via%2520nitrone%2520cycloaddition%26jtitle%3DJ.%2520Chem.%2520Soc.%252C%2520Chem.%2520Commun.%26date%3D1988%26spage%3D654%26epage%3D655%26doi%3D10.1039%2FC39880000654" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nasopoulou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georgiadis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matziari, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dive, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cedex, G. Y. V.</span></span> <span> </span><span class="NLM_article-title">A Versatile Annulation Protocol toward Novel Constrained Phosphinic Peptidomimetics</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">7222</span>– <span class="NLM_lpage">7228</span>, <span class="refDoi"> DOI: 10.1021/jo071081l</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo071081l" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpsVeltLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2007&pages=7222-7228&author=M.+Nasopoulouauthor=D.+Georgiadisauthor=M.+Matziariauthor=V.+Diveauthor=G.+Y.+V.+Cedex&title=A+Versatile+Annulation+Protocol+toward+Novel+Constrained+Phosphinic+Peptidomimetics&doi=10.1021%2Fjo071081l"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">A versatile annulation protocol toward novel constrained phosphinic peptidomimetics</span></div><div class="casAuthors">Nasopoulou, Magdalini; Georgiadis, Dimitris; Matziari, Magdalini; Dive, Vincent; Yiotakis, Athanasios</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7222-7228</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The development of a novel 3-center 2-component annulation reaction between α,ω-carbamoylaldehydes and suitably monoalkylated phosphinic acids is reported.  Depending on the starting α,ω-carbamoylaldehyde, diverse phosphinic scaffolds varying in the size of their rigidity element, the nature and stereochem. of substituents, and the participation of heteroatoms in the azacyclic ring system can be obtained in one synthetic step and in high yield.  In addn., this methodol. allows the synthesis of Fmoc-protected (Fmoc = 9-fluorenylmethyloxycarbonyl) constrained aminophosphinic acids that can be easily converted to suitable pseudodipeptide building blocks compatible with the requirements of peptide synthesis on the solid phase.  Finally, the careful choice of both substituents and protecting groups can provide functionally diverse, orthogonally protected constrained scaffolds for extended derivatization of the target phosphinic peptidomimetic structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTlVy0upnrWrVg90H21EOLACvtfcHk0ljZ5O4q5UnvXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpsVeltLs%253D&md5=07017bd083b952e1bca5527309ecad8a</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjo071081l&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo071081l%26sid%3Dliteratum%253Aachs%26aulast%3DNasopoulou%26aufirst%3DM.%26aulast%3DGeorgiadis%26aufirst%3DD.%26aulast%3DMatziari%26aufirst%3DM.%26aulast%3DDive%26aufirst%3DV.%26aulast%3DCedex%26aufirst%3DG.%2BY.%2BV.%26atitle%3DA%2520Versatile%2520Annulation%2520Protocol%2520toward%2520Novel%2520Constrained%2520Phosphinic%2520Peptidomimetics%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2007%26volume%3D72%26spage%3D7222%26epage%3D7228%26doi%3D10.1021%2Fjo071081l" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wasney, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siarheyeva, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajian, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheryl, H.</span></span> <span> </span><span class="NLM_article-title">Fluorescence-Based Methods for Screening Writers and Readers of Histone Methyl Marks</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">71</span>– <span class="NLM_lpage">84</span>, <span class="refDoi"> DOI: 10.1177/1087057111422256</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1177%2F1087057111422256" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=21972038" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC38Xit1Olt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2012&pages=71-84&author=A.+Allali-Hassaniauthor=G.+A.+Wasneyauthor=A.+Siarheyevaauthor=T.+Hajianauthor=M.+Vedadiauthor=H.+Cheryl&title=Fluorescence-Based+Methods+for+Screening+Writers+and+Readers+of+Histone+Methyl+Marks&doi=10.1177%2F1087057111422256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescence-based methods for screening writers and readers of histone methyl marks</span></div><div class="casAuthors">Allali-Hassani, Abdellah; Wasney, Gregory A.; Siarheyeva, Alena; Hajian, Taraneh; Arrowsmith, Cheryl H.; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">71-84</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The histone methyltransferase (HMT) family of proteins consists of enzymes that methylate lysine or arginine residues on histone tails as well as other proteins.  Such modifications affect chromatin structure and play a significant regulatory role in gene expression.  Many HMTs have been implicated in tumorigenesis and progression of multiple malignancies and play essential roles in embryonic development and stem cell renewal.  Overexpression of some HMTs has been obsd. and is correlated pos. with various types of cancer.  Here the authors report development of a continuous fluorescence-based methyltransferase assay in a 384-well format and its application in detg. kinetic parameters for EHMT1, G9a, PRMT3, SETD7, and SUV39H2 as well as for screening against libraries of small mols. to identify enzyme inhibitors.  They also report the development of a peptide displacement assay using fluorescence polarization in a 384-well format to assay and screen protein peptide interactions such as those of WDR5 and EED, components of MLL and EZH2 methyltransferase complexes.  Using these high-throughput screening methods, the authors have identified potent inhibitors and ligands for some of these proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0s8Ex9rZOXrVg90H21EOLACvtfcHk0ljZ5O4q5UnvXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xit1Olt7g%253D&md5=105a20c2473ab603ade42a38764def09</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1177%2F1087057111422256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057111422256%26sid%3Dliteratum%253Aachs%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DCheryl%26aufirst%3DH.%26atitle%3DFluorescence-Based%2520Methods%2520for%2520Screening%2520Writers%2520and%2520Readers%2520of%2520Histone%2520Methyl%2520Marks%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2012%26volume%3D17%26spage%3D71%26epage%3D84%26doi%3D10.1177%2F1087057111422256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, J. F.</span></span> <span> </span><span class="NLM_article-title">The Slow-Binding and Slow, Tight-Binding Inhibition of Enzyme-Catalysed Reactions</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">1982</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">102</span>– <span class="NLM_lpage">105</span>, <span class="refDoi"> DOI: 10.1016/0968-0004(82)90157-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1016%2F0968-0004%2882%2990157-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADyaL38XhslKrtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=1982&pages=102-105&author=J.+F.+Morrison&title=The+Slow-Binding+and+Slow%2C+Tight-Binding+Inhibition+of+Enzyme-Catalysed+Reactions&doi=10.1016%2F0968-0004%2882%2990157-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">The slow-binding and slow, tight-binding inhibition of enzyme-catalyzed reactions</span></div><div class="casAuthors">Morrison, John F.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">1982</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">102-5</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    </div><div class="casAbstract">The mechanisms of enzyme inhibition by slow-binding and slow, tight-binding reversible inhibitors are discussed, as well as the kinetics of these processes as opposed to classical and tight-binding inhibitor mechanisms.  Examples of these types of inhibitors in the enzyme literature are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoun0ayjodbYrVg90H21EOLACvtfcHk0ljZ5O4q5UnvXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL38XhslKrtrw%253D&md5=1740627c1ad2adf182f4c0ae355bb4de</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2F0968-0004%2882%2990157-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0968-0004%252882%252990157-8%26sid%3Dliteratum%253Aachs%26aulast%3DMorrison%26aufirst%3DJ.%2BF.%26atitle%3DThe%2520Slow-Binding%2520and%2520Slow%252C%2520Tight-Binding%2520Inhibition%2520of%2520Enzyme-Catalysed%2520Reactions%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D1982%26volume%3D7%26spage%3D102%26epage%3D105%26doi%3D10.1016%2F0968-0004%2882%2990157-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span> <span> </span><span class="NLM_article-title">Tight Binding Inhibitors</span>. In  <i>Evaluation of Enzyme Inhibitors in Drug Discovery: A Guide for Medicinal Chemists and Pharmacologists</i>, <span class="NLM_edition">2</span>nd ed.; <span class="NLM_publisher-name">John Wiley & Sons</span>: <span class="NLM_publisher-loc">New Jersey</span>, <span class="NLM_year">2013</span>; pp  <span class="NLM_fpage">245</span>– <span class="NLM_lpage">285</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1002%2F9781118540398.ch7" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&pages=245-285&author=R.+A.+Copeland&title=Evaluation+of+Enzyme+Inhibitors+in+Drug+Discovery%3A+A+Guide+for+Medicinal+Chemists+and+Pharmacologists"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2F9781118540398.ch7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9781118540398.ch7%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26atitle%3DTight%2520Binding%2520Inhibitors%26btitle%3DEvaluation%2520of%2520Enzyme%2520Inhibitors%2520in%2520Drug%2520Discovery%253A%2520A%2520Guide%2520for%2520Medicinal%2520Chemists%2520and%2520Pharmacologists%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%26date%3D2013%26spage%3D245%26epage%3D285" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span> <span> </span><span class="NLM_article-title">An Asparagine/Glycine Switch Governs Product Specificity of Human N-Terminal Methyltransferase NTMT2</span>. <i>Commun. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>1</i></span>, <span class="NLM_elocation-id">183</span> <span class="refDoi"> DOI: 10.1038/s42003-018-0196-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1038%2Fs42003-018-0196-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=30417120" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A280%3ADC%252BB3cvovV2qug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2018&author=C.+Dongauthor=G.+Dongauthor=L.+Liauthor=L.+Zhuauthor=W.+Tempelauthor=Y.+Liuauthor=R.+Huangauthor=J.+Min&title=An+Asparagine%2FGlycine+Switch+Governs+Product+Specificity+of+Human+N-Terminal+Methyltransferase+NTMT2&doi=10.1038%2Fs42003-018-0196-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">An asparagine/glycine switch governs product specificity of human N-terminal methyltransferase NTMT2</span></div><div class="casAuthors">Dong Cheng; Li Li; Zhu Licheng; Tempel Wolfram; Liu Yanli; Min Jinrong; Dong Guangping; Huang Rong; Zhu Licheng; Min Jinrong</div><div class="citationInfo"><span class="NLM_cas:title">Communications biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">183</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">α-N-terminal methylation of proteins is an important post-translational modification that is catalyzed by two different N-terminal methyltransferases, namely NTMT1 and NTMT2.  Previous studies have suggested that NTMT1 is a tri-methyltransferase, whereas NTMT2 is a mono-methyltransferase.  Here, we report the first crystal structures, to our knowledge, of NTMT2 in binary complex with S-adenosyl-L-methionine as well as in ternary complex with S-adenosyl-L-homocysteine and a substrate peptide.  Our structural observations combined with biochemical studies reveal that NTMT2 is also able to di-/tri-methylate the GPKRIA peptide and di-methylate the PPKRIA peptide, otherwise it is predominantly a mono-methyltransferase.  The residue N89 of NTMT2 serves as a gatekeeper residue that regulates the binding of unmethylated versus monomethylated substrate peptide.  Structural comparison of NTMT1 and NTMT2 prompts us to design a N89G mutant of NTMT2 that can profoundly alter its catalytic activities and product specificities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSKFvwVHUgF0kwlCcfExbvbfW6udTcc2eamneuUw_Sw-Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cvovV2qug%253D%253D&md5=edc8c87334f423ab98872442f81d7928</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fs42003-018-0196-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs42003-018-0196-2%26sid%3Dliteratum%253Aachs%26aulast%3DDong%26aufirst%3DC.%26aulast%3DDong%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DZhu%26aufirst%3DL.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DR.%26aulast%3DMin%26aufirst%3DJ.%26atitle%3DAn%2520Asparagine%252FGlycine%2520Switch%2520Governs%2520Product%2520Specificity%2520of%2520Human%2520N-Terminal%2520Methyltransferase%2520NTMT2%26jtitle%3DCommun.%2520Biol.%26date%3D2018%26volume%3D1%26doi%3D10.1038%2Fs42003-018-0196-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rizzo, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dirk, L. M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strunk, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roiko, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunzelle, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houtz, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trievel, R. C.</span></span> <span> </span><span class="NLM_article-title">SET7/9 Catalytic Mutants Reveal the Role of Active Site Water Molecules in Lysine Multiple Methylation</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>285</i></span>,  <span class="NLM_fpage">31849</span>– <span class="NLM_lpage">31858</span>, <span class="refDoi"> DOI: 10.1074/jbc.M110.114587</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1074%2Fjbc.M110.114587" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=20675860" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=31849-31858&author=P.+A.+Rizzoauthor=+Delauthor=L.+M.+A.+Dirkauthor=B.+S.+Strunkauthor=M.+S.+Roikoauthor=J.+S.+Brunzelleauthor=R.+L.+Houtzauthor=R.+C.+Trievel&title=SET7%2F9+Catalytic+Mutants+Reveal+the+Role+of+Active+Site+Water+Molecules+in+Lysine+Multiple+Methylation&doi=10.1074%2Fjbc.M110.114587"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.114587&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.114587%26sid%3Dliteratum%253Aachs%26aulast%3DRizzo%26aufirst%3DP.%2BA.%26aulast%3DDel%26aulast%3DDirk%26aufirst%3DL.%2BM.%2BA.%26aulast%3DStrunk%26aufirst%3DB.%2BS.%26aulast%3DRoiko%26aufirst%3DM.%2BS.%26aulast%3DBrunzelle%26aufirst%3DJ.%2BS.%26aulast%3DHoutz%26aufirst%3DR.%2BL.%26aulast%3DTrievel%26aufirst%3DR.%2BC.%26atitle%3DSET7%252F9%2520Catalytic%2520Mutants%2520Reveal%2520the%2520Role%2520of%2520Active%2520Site%2520Water%2520Molecules%2520in%2520Lysine%2520Multiple%2520Methylation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D31849%26epage%3D31858%26doi%3D10.1074%2Fjbc.M110.114587" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chuikov, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurash, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Justin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ivanov, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mckinney, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempst, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prives, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gamblin, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barlev, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinberg, D.</span></span> <span> </span><span class="NLM_article-title">Regulation of P53 Activity through Lysine Methylation</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>432</i></span>,  <span class="NLM_fpage">353</span>– <span class="NLM_lpage">360</span>, <span class="refDoi"> DOI: 10.1038/nature03117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1038%2Fnature03117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=15525938" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BD2cXpvVCjtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=432&publication_year=2004&pages=353-360&author=S.+Chuikovauthor=J.+K.+Kurashauthor=J.+R.+Wilsonauthor=B.+Xiaoauthor=N.+Justinauthor=G.+S.+Ivanovauthor=K.+Mckinneyauthor=P.+Tempstauthor=C.+Privesauthor=S.+J.+Gamblinauthor=N.+A.+Barlevauthor=D.+Reinberg&title=Regulation+of+P53+Activity+through+Lysine+Methylation&doi=10.1038%2Fnature03117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of p53 activity through lysine methylation</span></div><div class="casAuthors">Chuikov, Sergei; Kurash, Julia K.; Wilson, Jonathan R.; Xiao, Bing; Justin, Neil; Ivanov, Gleb S.; McKinney, Kristine; Tempst, Paul; Prives, Carol; Gamblin, Steven J.; Barlev, Nickolai A.; Reinberg, Danny</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">432</span>
        (<span class="NLM_cas:issue">7015</span>),
    <span class="NLM_cas:pages">353-360</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">P53 is a tumor suppressor that regulates the cellular response to genotoxic stresses. p53 is a short-lived protein and its activity is regulated mostly by stabilization via different post-translational modifications.  Here we report a novel mechanism of p53 regulation through lysine methylation by Set9 methyltransferase.  Set9 specifically methylates p53 at one residue within the carboxyl-terminus regulatory region.  Methylated p53 is restricted to the nucleus and the modification pos. affects its stability.  Set9 regulates the expression of p53 target genes in a manner dependent on the p53-methylation site.  The crystal structure of a ternary complex of Set9 with a p53 peptide and the cofactor product S-adenosyl-L-homocysteine (AdoHcy) provides the mol. basis for recognition of p53 by this lysine methyltransferase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQEN46GC7xzLVg90H21EOLACvtfcHk0ljPWncgLHAq0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXpvVCjtbo%253D&md5=eff3a7ec7f6adbf1fc43016325f074d0</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fnature03117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03117%26sid%3Dliteratum%253Aachs%26aulast%3DChuikov%26aufirst%3DS.%26aulast%3DKurash%26aufirst%3DJ.%2BK.%26aulast%3DWilson%26aufirst%3DJ.%2BR.%26aulast%3DXiao%26aufirst%3DB.%26aulast%3DJustin%26aufirst%3DN.%26aulast%3DIvanov%26aufirst%3DG.%2BS.%26aulast%3DMckinney%26aufirst%3DK.%26aulast%3DTempst%26aufirst%3DP.%26aulast%3DPrives%26aufirst%3DC.%26aulast%3DGamblin%26aufirst%3DS.%2BJ.%26aulast%3DBarlev%26aufirst%3DN.%2BA.%26aulast%3DReinberg%26aufirst%3DD.%26atitle%3DRegulation%2520of%2520P53%2520Activity%2520through%2520Lysine%2520Methylation%26jtitle%3DNature%26date%3D2004%26volume%3D432%26spage%3D353%26epage%3D360%26doi%3D10.1038%2Fnature03117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Couture, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collazo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunzelle, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trievel, R. C.</span></span> <span> </span><span class="NLM_article-title">Structural and Functional Analysis of SET8, a Histone H4 Lys-20 Methyltransferase.</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_fpage">1455</span>– <span class="NLM_lpage">1465</span>, <span class="refDoi"> DOI: 10.1101/gad.1318405</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1101%2Fgad.1318405" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=15933070" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlslChu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&pages=1455-1465&author=J.+Coutureauthor=E.+Collazoauthor=J.+S.+Brunzelleauthor=R.+C.+Trievel&title=Structural+and+Functional+Analysis+of+SET8%2C+a+Histone+H4+Lys-20+Methyltransferase.&doi=10.1101%2Fgad.1318405"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and functional analysis of SET8, a histone H4 Lys-20 methyltransferase</span></div><div class="casAuthors">Couture, Jean-Francois; Collazo, Evys; Brunzelle, Joseph S.; Trievel, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1455-1465</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">SET8 (also known as PR-SET7) is a histone H4-Lys-20-specific methyltransferase that is implicated in cell-cycle-dependent transcriptional silencing and mitotic regulation in metazoans.  Herein we report the crystal structure of human SET8 (hSET8) bound to a histone H4 peptide bearing Lys-20 and the product cofactor S-adenosylhomocysteine.  Histone H4 intercalates in the substrate-binding cleft as an extended parallel β-strand.  Residues preceding Lys-20 in H4 engage in an extensive array of salt bridge, hydrogen bond, and van der Waals interactions with hSET8, while the C-terminal residues bind through predominantly hydrophobic interactions.  Mutational anal. of both the substrate-binding cleft and histone H4 reveals that interactions with residues in the N and C termini of the H4 peptide are crit. for conferring substrate specificity.  Finally, anal. of the product specificity indicates that hSET8 is a monomethylase, consistent with its role in the maintenance of Lys-20 monomethylation during cell division.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphqNcPOAjIbbVg90H21EOLACvtfcHk0ljPWncgLHAq0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlslChu7w%253D&md5=26dd3cdbaee412a5904ba8268b8efb40</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1101%2Fgad.1318405&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1318405%26sid%3Dliteratum%253Aachs%26aulast%3DCouture%26aufirst%3DJ.%26aulast%3DCollazo%26aufirst%3DE.%26aulast%3DBrunzelle%26aufirst%3DJ.%2BS.%26aulast%3DTrievel%26aufirst%3DR.%2BC.%26atitle%3DStructural%2520and%2520Functional%2520Analysis%2520of%2520SET8%252C%2520a%2520Histone%2520H4%2520Lys-20%2520Methyltransferase.%26jtitle%3DGenes%2520Dev.%26date%3D2005%26spage%3D1455%26epage%3D1465%26doi%3D10.1101%2Fgad.1318405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lunin, V. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antoshenko, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dombrovski, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campagna-Slater, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotnikov, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span> <span> </span><span class="NLM_article-title">Structural Biology of Human H3K9 Methyltransferases</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>5</i></span>, <span class="NLM_elocation-id">e8570</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0008570</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1371%2Fjournal.pone.0008570" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=20084102" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A280%3ADC%252BC3c%252FjtlGgsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&author=H.+Wuauthor=J.+Minauthor=V.+V.+Luninauthor=T.+Antoshenkoauthor=L.+Dombrovskiauthor=H.+Zengauthor=A.+Allali-Hassaniauthor=V.+Campagna-Slaterauthor=M.+Vedadiauthor=C.+H.+Arrowsmithauthor=A.+N.+Plotnikovauthor=M.+Schapira&title=Structural+Biology+of+Human+H3K9+Methyltransferases&doi=10.1371%2Fjournal.pone.0008570"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Structural biology of human H3K9 methyltransferases</span></div><div class="casAuthors">Wu Hong; Min Jinrong; Lunin Vladimir V; Antoshenko Tatiana; Dombrovski Ludmila; Zeng Hong; Allali-Hassani Abdellah; Campagna-Slater Valerie; Vedadi Masoud; Arrowsmith Cheryl H; Plotnikov Alexander N; Schapira Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">PloS one</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">e8570</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">SET domain methyltransferases deposit methyl marks on specific histone tail lysine residues and play a major role in epigenetic regulation of gene transcription.  We solved the structures of the catalytic domains of GLP, G9a, Suv39H2 and PRDM2, four of the eight known human H3K9 methyltransferases in their apo conformation or in complex with the methyl donating cofactor, and peptide substrates.  We analyzed the structural determinants for methylation state specificity, and designed a G9a mutant able to tri-methylate H3K9.  We show that the I-SET domain acts as a rigid docking platform, while induced-fit of the Post-SET domain is necessary to achieve a catalytically competent conformation.  We also propose a model where long-range electrostatics bring enzyme and histone substrate together, while the presence of an arginine upstream of the target lysine is critical for binding and specificity.  ENHANCED VERSION: This article can also be viewed as an enhanced version in which the text of the article is integrated with interactive 3D representations and animated transitions.  Please note that a web plugin is required to access this enhanced functionality.  Instructions for the installation and use of the web plugin are available in Text S1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTAEo3Y9d9dncRAV4D_Ins_fW6udTcc2eaZ-bIzeLpQz7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3c%252FjtlGgsA%253D%253D&md5=2649472b1401c69035f99e0ca20822a3</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0008570&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0008570%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DH.%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DLunin%26aufirst%3DV.%2BV.%26aulast%3DAntoshenko%26aufirst%3DT.%26aulast%3DDombrovski%26aufirst%3DL.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DCampagna-Slater%26aufirst%3DV.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DPlotnikov%26aufirst%3DA.%2BN.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DStructural%2520Biology%2520of%2520Human%2520H3K9%2520Methyltransferases%26jtitle%3DPLoS%2520One%26date%3D2010%26volume%3D5%26doi%3D10.1371%2Fjournal.pone.0008570" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oerum, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Catala, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atdjian, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brachet, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barraud, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iannazzo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Droogmans, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ethève-Quelquejeu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tisné, C.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Bisubstrate Analogues as Structural Tools to Investigate m A Methyltransferase Active Sites</span>. <i>RNA Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">798</span>– <span class="NLM_lpage">808</span>, <span class="refDoi"> DOI: 10.1080/15476286.2019.1589360</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1080%2F15476286.2019.1589360" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=30879411" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A280%3ADC%252BB3cbkvFehsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2019&pages=798-808&author=S.+Oerumauthor=M.+Catalaauthor=C.+Atdjianauthor=F.+Brachetauthor=P.+Barraudauthor=L.+Iannazzoauthor=L.+Droogmansauthor=M.+Eth%C3%A8ve-Quelquejeuauthor=C.+Tisn%C3%A9&title=Bisubstrate+Analogues+as+Structural+Tools+to+Investigate+m+A+Methyltransferase+Active+Sites&doi=10.1080%2F15476286.2019.1589360"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Bisubstrate analogues as structural tools to investigate m(6)A methyltransferase active sites</span></div><div class="casAuthors">Oerum Stephanie; Catala Marjorie; Barraud Pierre; Tisne Carine; Atdjian Colette; Iannazzo Laura; Braud Emmanuelle; Etheve-Quelquejeu Melanie; Brachet Franck; Ponchon Luc; Droogmans Louis</div><div class="citationInfo"><span class="NLM_cas:title">RNA biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">798-808</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">RNA methyltransferases (MTases) catalyse the transfer of a methyl group to their RNA substrates using most-often S-adenosyl-L-methionine (SAM) as cofactor.  Only few RNA-bound MTases structures are currently available due to the difficulties in crystallising RNA:protein complexes.  The lack of complex structures results in poorly understood RNA recognition patterns and methylation reaction mechanisms.  On the contrary, many cofactor-bound MTase structures are available, resulting in well-understood protein:cofactor recognition, that can guide the design of bisubstrate analogues that mimic the state at which both the substrate and the cofactor is bound.  Such bisubstrate analogues were recently synthesized for proteins monomethylating the N6-atom of adenine (m(6)A).  These proteins include, amongst others, RlmJ in E. coli and METLL3:METT14 and METTL16 in human.  As a proof-of-concept, we here test the ability of the bisubstrate analogues to mimic the substrate:cofactor bound state during catalysis by studying their binding to RlmJ using differential scanning fluorimetry, isothermal titration calorimetry and X-ray crystallography.  We find that the methylated adenine base binds in the correct pocket, and thus these analogues could potentially be used broadly to study the RNA recognition and catalytic mechanism of m(6)A MTases.  Two bisubstrate analogues bind RlmJ with micro-molar affinity, and could serve as starting scaffolds for inhibitor design against m(6)A RNA MTases.  The same analogues cause changes in the melting temperature of the m(1)A RNA MTase, TrmK, indicating non-selective protein:compound complex formation.  Thus, optimization of these molecular scaffolds for m(6)A RNA MTase inhibition should aim to increase selectivity, as well as affinity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRDdwiIxGiYOv6xYuD7tu2XfW6udTcc2eZmDB7Rxw1Eibntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cbkvFehsg%253D%253D&md5=1c13082943751faaf85ebe743756df1d</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1080%2F15476286.2019.1589360&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15476286.2019.1589360%26sid%3Dliteratum%253Aachs%26aulast%3DOerum%26aufirst%3DS.%26aulast%3DCatala%26aufirst%3DM.%26aulast%3DAtdjian%26aufirst%3DC.%26aulast%3DBrachet%26aufirst%3DF.%26aulast%3DBarraud%26aufirst%3DP.%26aulast%3DIannazzo%26aufirst%3DL.%26aulast%3DDroogmans%26aufirst%3DL.%26aulast%3DEth%25C3%25A8ve-Quelquejeu%26aufirst%3DM.%26aulast%3DTisn%25C3%25A9%26aufirst%3DC.%26atitle%3DBisubstrate%2520Analogues%2520as%2520Structural%2520Tools%2520to%2520Investigate%2520m%2520A%2520Methyltransferase%2520Active%2520Sites%26jtitle%3DRNA%2520Biol.%26date%3D2019%26volume%3D16%26spage%3D798%26epage%3D808%26doi%3D10.1080%2F15476286.2019.1589360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swewczyk, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trush, V. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freitas, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tatlock, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumpf, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wythes, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casimiro-Garcia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denny, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parikh, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. R.</span></span> <span> </span><span class="NLM_article-title">Selective, Small-Molecule Co-Factor Binding Site Inhibition of a Su(Var)3 – 9, Enhancer of Zeste, Trithorax Domain Containing Lysine Methyltransferase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">7669</span>– <span class="NLM_lpage">7683</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00112</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00112" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFyrur7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=7669-7683&author=A.+P.+Taylorauthor=M.+Swewczykauthor=S.+Kennedyauthor=V.+V.+Trushauthor=H.+Wuauthor=H.+Zengauthor=A.+Dongauthor=R.+F.+Freitasauthor=J.+Tatlockauthor=R.+A.+Kumpfauthor=M.+Wythesauthor=A.+Casimiro-Garciaauthor=R.+A.+Dennyauthor=M.+D.+Parikhauthor=F.+Liauthor=D.+Barsyte-Lovejoyauthor=M.+Schapiraauthor=M.+Vedadiauthor=M.+Schapiraauthor=P.+J.+Brownauthor=C.+H.+Arrowsmithauthor=D.+R.+Owen&title=Selective%2C+Small-Molecule+Co-Factor+Binding+Site+Inhibition+of+a+Su%28Var%293+%E2%80%93+9%2C+Enhancer+of+Zeste%2C+Trithorax+Domain+Containing+Lysine+Methyltransferase&doi=10.1021%2Facs.jmedchem.9b00112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Selective, Small-Molecule Co-Factor Binding Site Inhibition of a Su(var)3-9, Enhancer of Zeste, Trithorax Domain Containing Lysine Methyltransferase</span></div><div class="casAuthors">Taylor, Alexandria P.; Swewczyk, Magdalena; Kennedy, Steven; Trush, Viacheslav V.; Wu, Hong; Zeng, Hong; Dong, Aiping; Ferreira de Freitas, Renato; Tatlock, John; Kumpf, Robert A.; Wythes, Martin; Casimiro-Garcia, Agustin; Denny, Rajiah Aldrin; Parikh, Mihir D.; Li, Fengling; Barsyte-Lovejoy, Dalia; Schapira, Matthieu; Vedadi, Masoud; Brown, Peter J.; Arrowsmith, Cheryl H.; Owen, Dafydd R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7669-7683</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The first chem. probe to primarily occupy the co-factor binding site of a Su(var)3-9, enhancer of a zeste, trithorax (SET) domain contg. protein lysine methyltransferase (PKMT) is reported.  Protein methyltransferases require S-adenosylmethionine (SAM) as a co-factor (Me donor) for enzymic activity.  However, SAM itself represents a poor medicinal chem. starting point for a selective, cell-active inhibitor given its extreme physicochem. properties and its role in multiple cellular processes.  A previously untested medicinal chem. strategy of deliberate file enrichment around mols. bearing the hallmarks of SAM, but with improved lead-like properties from the outset, yielded viable hits against SET and MYND domain-contg. protein 2 (SMYD2) that were shown to bind in the co-factor site.  These leads were optimized to identify a highly biochem. potent, PKMT-selective, and cell-active chem. probe.  While substrate-based inhibitors of PKMTs are known, this represents a novel, co-factor-derived strategy for the inhibition of SMYD2 which may also prove applicable to lysine methyltransferase family members previously thought of as intractable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrayuqhEmC-MLVg90H21EOLACvtfcHk0li_BkHGYdTFXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFyrur7N&md5=6a9f875b93804141716a85d77b93c5ee</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00112%26sid%3Dliteratum%253Aachs%26aulast%3DTaylor%26aufirst%3DA.%2BP.%26aulast%3DSwewczyk%26aufirst%3DM.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DTrush%26aufirst%3DV.%2BV.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DFreitas%26aufirst%3DR.%2BF.%26aulast%3DTatlock%26aufirst%3DJ.%26aulast%3DKumpf%26aufirst%3DR.%2BA.%26aulast%3DWythes%26aufirst%3DM.%26aulast%3DCasimiro-Garcia%26aufirst%3DA.%26aulast%3DDenny%26aufirst%3DR.%2BA.%26aulast%3DParikh%26aufirst%3DM.%2BD.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26atitle%3DSelective%252C%2520Small-Molecule%2520Co-Factor%2520Binding%2520Site%2520Inhibition%2520of%2520a%2520Su%2528Var%25293%2520%25E2%2580%2593%25209%252C%2520Enhancer%2520of%2520Zeste%252C%2520Trithorax%2520Domain%2520Containing%2520Lysine%2520Methyltransferase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D7669%26epage%3D7683%26doi%3D10.1021%2Facs.jmedchem.9b00112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Babault, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ju, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of Bisubstrate Inhibitors of Nicotinamide <i>N</i> -Methyltransferase (NNMT)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1541</span>– <span class="NLM_lpage">1551</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01422</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01422" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC1cXotVOruw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=1541-1551&author=N.+Babaultauthor=A.+Allali-Hassaniauthor=F.+Liauthor=J.+Fanauthor=A.+Yueauthor=K.+Juauthor=F.+Liuauthor=M.+Vedadiauthor=J.+Liuauthor=J.+Jin&title=Discovery+of+Bisubstrate+Inhibitors+of+Nicotinamide+N+-Methyltransferase+%28NNMT%29&doi=10.1021%2Facs.jmedchem.7b01422"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Bisubstrate Inhibitors of Nicotinamide N-Methyltransferase (NNMT)</span></div><div class="casAuthors">Babault, Nicolas; Allali-Hassani, Abdellah; Li, Fengling; Fan, Jie; Yue, Alex; Ju, Kevin; Liu, Feng; Vedadi, Masoud; Liu, Jing; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1541-1551</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) catalyzes the N-methylation of pyridine-contg. compds. using the cofactor S-5'-adenosyl-L-methionine (SAM) as the Me group donor.  Through the regulation of the levels of its substrates, cofactor, and products, NNMT plays important role in physiol. and pathophysiol.  Overexpression of NNMT has been implicated in various human diseases.  Potent and selective small-mol. NNMT inhibitors are valuable chem. tools for testing biol. and therapeutic hypotheses.  However, very few NNMT inhibitors have been reported.  Here, we describe the discovery of a bisubstrate NNMT inhibitor MS2734 (6), and characterization of this inhibitor in biochem., biophys., kinetic, and structural studies.  Importantly, we obtained the first crystal structure of human NNMT in complex with a small-mol. inhibitor.  The structure of the NNMT-6 complex has unambiguously demonstrated that 6 occupied both substrate and cofactor binding sites.  The findings paved the way for developing more potent and selective NNMT inhibitors in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9ThNiA1GTwLVg90H21EOLACvtfcHk0li_BkHGYdTFXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXotVOruw%253D%253D&md5=50e23e5f60beb64c184f6eadc6a75883</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01422%26sid%3Dliteratum%253Aachs%26aulast%3DBabault%26aufirst%3DN.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DFan%26aufirst%3DJ.%26aulast%3DYue%26aufirst%3DA.%26aulast%3DJu%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520Bisubstrate%2520Inhibitors%2520of%2520Nicotinamide%2520N%2520-Methyltransferase%2520%2528NNMT%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D1541%26epage%3D1551%26doi%3D10.1021%2Facs.jmedchem.7b01422" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahley, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stivers, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. G.</span></span> <span> </span><span class="NLM_article-title">A Transient Kinetic Analysis of PRMT1 Catalysis</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">7033</span>– <span class="NLM_lpage">7044</span>, <span class="refDoi"> DOI: 10.1021/bi200456u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi200456u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC3MXptFKltLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2011&pages=7033-7044&author=Y.+Fengauthor=N.+Xieauthor=M.+Jinauthor=M.+R.+Stahleyauthor=J.+T.+Stiversauthor=Y.+G.+Zheng&title=A+Transient+Kinetic+Analysis+of+PRMT1+Catalysis&doi=10.1021%2Fbi200456u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">A transient kinetic analysis of PRMT1 catalysis</span></div><div class="casAuthors">Feng, You; Xie, Nan; Jin, Miyeong; Stahley, Mary R.; Stivers, JamesT.; Zheng, Yujun George</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">7033-7044</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Post-translational modifications (PTMs) are important strategies used by eukaryotic organisms to modulate their phenotypes.  One of the well-studied PTMs, Arg residue methylation, is catalyzed by protein arginine methyltransferases (PRMTs) with S-adenosyl-L-methionine (SAM) as the Me donor.  The functions of PRMTs have been broadly studied in different biol. processes and diseased states, but the mol. basis for Arg methylation is not well-defined.  Here, the authors report the transient-state kinetic anal. of rat PRMT1 catalysis.  The fast assocn. and dissocn. rates suggested that PRMT1 catalysis of histone H4 methylation followed a rapid equil. sequential kinetic mechanism.  The data provided direct evidence that the chem. of Me transfer is the major rate-limiting step and that binding of cofactors SAM or S-adenosyl-L-homocysteine (SAH) affects the assocn. and dissocn. of H4 with PRMT1.  Importantly, from stopped-flow fluorescence measurements, the authors identified a crit. kinetic step suggesting a pre-catalytic conformational transition induced by substrate binding.  These results provide new insights into the mechanism of Arg methylation and the rational design of PRMT inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpImJRI93ID4rVg90H21EOLACvtfcHk0ljLBpabxmkjxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXptFKltLc%253D&md5=f2562d7ece9b10754b551c2b6dec7cb1</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fbi200456u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi200456u%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DXie%26aufirst%3DN.%26aulast%3DJin%26aufirst%3DM.%26aulast%3DStahley%26aufirst%3DM.%2BR.%26aulast%3DStivers%26aufirst%3DJ.%2BT.%26aulast%3DZheng%26aufirst%3DY.%2BG.%26atitle%3DA%2520Transient%2520Kinetic%2520Analysis%2520of%2520PRMT1%2520Catalysis%26jtitle%3DBiochemistry%26date%3D2011%26volume%3D50%26spage%3D7033%26epage%3D7044%26doi%3D10.1021%2Fbi200456u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oudhoff, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tatlock, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freeman, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senisterra, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuznetsova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcellus, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couzens, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, Al.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Awar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fish, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstenberger, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benn, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimley, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braam, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, F. M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sudol, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaph, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span> <span> </span><span class="NLM_article-title">(R)-PFI-2 Is a Potent and Selective Inhibitor of SETD7 Methyltransferase Activity in Cells</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">12853</span>– <span class="NLM_lpage">12858</span>, <span class="refDoi"> DOI: 10.1073/pnas.1407358111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1073%2Fpnas.1407358111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=25136132" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVartr7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=12853-12858&author=D.+Barsyte-Lovejoyauthor=F.+Liauthor=M.+J.+Oudhoffauthor=J.+H.+Tatlockauthor=A.+Dongauthor=H.+Zengauthor=H.+Wuauthor=S.+A.+Freemanauthor=M.+Schapiraauthor=G.+A.+Senisterraauthor=E.+Kuznetsovaauthor=R.+Marcellusauthor=A.+Allali-Hassaniauthor=S.+Kennedyauthor=P.+Lambertauthor=A.+L.+Couzensauthor=A.+Amanauthor=Al.+Gingrasauthor=R.+Al-Awarauthor=P.+V.+Fishauthor=B.+S.+Gerstenbergerauthor=L.+Robertsauthor=C.+L.+Bennauthor=R.+L.+Grimleyauthor=M.+J.+Braamauthor=F.+M.+V.+Rossiauthor=M.+Sudolauthor=P.+J.+Brownauthor=M.+E.+Bunnageauthor=D.+R.+Owenauthor=C.+Zaphauthor=M.+Vedadiauthor=C.+H.+Arrowsmith&title=%28R%29-PFI-2+Is+a+Potent+and+Selective+Inhibitor+of+SETD7+Methyltransferase+Activity+in+Cells&doi=10.1073%2Fpnas.1407358111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">(R)-PFI-2 is a potent and selective inhibitor of SETD7 methyltransferase activity in cells</span></div><div class="casAuthors">Barsyte-Lovejoy, Dalia; Li, Fengling; Oudhoff, Menno J.; Tatlock, John H.; Dong, Aiping; Zeng, Hong; Wu, Hong; Freeman, Spencer A.; Schapira, Matthieu; Senisterra, Guillermo A.; Kuznetsova, Ekaterina; Marcellus, Richard; Allali-Hassani, Abdellah; Kennedy, Steven; Lambert, Jean-Philippe; Couzens, Amber L.; Aman, Ahmed; Gingras, Anne-Claude; Al-Awar, Rima; Fish, Paul V.; Gerstenberger, Brian S.; Roberts, Lee; Benn, Caroline L.; Grimley, Rachel L.; Braam, Mitchell J. S.; Rossi, Fabio M. V.; Sudol, Marius; Brown, Peter J.; Bunnage, Mark E.; Owen, Dafydd R.; Zaph, Colby; Vedadi, Masoud; Arrowsmith, Cheryl H.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">12853-12858</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">SET domain contg. (lysine methyltransferase) 7 (SETD7) is implicated in multiple signaling and disease related pathways with a broad diversity of reported substrates.  Here, we report the discovery of (R)-PFI-2-a first-in-class, potent (Kiapp = 0.33 nM), selective, and cell-active inhibitor of the methyltransferase activity of human SETD7-and its 500-fold less active enantiomer, (S)-PFI-2.  (R)-PFI-2 exhibits an unusual cofactor-dependent and substrate-competitive inhibitory mechanism by occupying the substrate peptide binding groove of SETD7, including the catalytic lysine-binding channel, and by making direct contact with the donor Me group of the cofactor, S-adenosylmethionine.  Chemoproteomics expts. using a biotinylated deriv. of (R)-PFI-2 demonstrated dose-dependent competition for binding to endogenous SETD7 in MCF7 cells pretreated with (R)-PFI-2.  In murine embryonic fibroblasts, (R)-PFI-2 treatment phenocopied the effects of Setd7 deficiency on Hippo pathway signaling, via modulation of the transcriptional coactivator Yes-assocd. protein (YAP) and regulation of YAP target genes.  In confluent MCF7 cells, (R)-PFI-2 rapidly altered YAP localization, suggesting continuous and dynamic regulation of YAP by the methyltransferase activity of SETD7.  These data establish (R)-PFI-2 and related compds. as a valuable tool-kit for the study of the diverse roles of SETD7 in cells and further validate protein methyltransferases as a druggable target class.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqv-HTBIcjMnbVg90H21EOLACvtfcHk0ljLBpabxmkjxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVartr7E&md5=0b8ad07ae23c4eba5cf11060d231b3f4</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1407358111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1407358111%26sid%3Dliteratum%253Aachs%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DOudhoff%26aufirst%3DM.%2BJ.%26aulast%3DTatlock%26aufirst%3DJ.%2BH.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DFreeman%26aufirst%3DS.%2BA.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DSenisterra%26aufirst%3DG.%2BA.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DMarcellus%26aufirst%3DR.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DLambert%26aufirst%3DP.%26aulast%3DCouzens%26aufirst%3DA.%2BL.%26aulast%3DAman%26aufirst%3DA.%26aulast%3DGingras%26aufirst%3DAl.%26aulast%3DAl-Awar%26aufirst%3DR.%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DGerstenberger%26aufirst%3DB.%2BS.%26aulast%3DRoberts%26aufirst%3DL.%26aulast%3DBenn%26aufirst%3DC.%2BL.%26aulast%3DGrimley%26aufirst%3DR.%2BL.%26aulast%3DBraam%26aufirst%3DM.%2BJ.%26aulast%3DRossi%26aufirst%3DF.%2BM.%2BV.%26aulast%3DSudol%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DZaph%26aufirst%3DC.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26atitle%3D%2528R%2529-PFI-2%2520Is%2520a%2520Potent%2520and%2520Selective%2520Inhibitor%2520of%2520SETD7%2520Methyltransferase%2520Activity%2520in%2520Cells%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2014%26volume%3D111%26spage%3D12853%26epage%3D12858%26doi%3D10.1073%2Fpnas.1407358111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Debler, E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warmack, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blobel, G.</span></span> <span> </span><span class="NLM_article-title">A Glutamate/Aspartate Switch Controls Product Specificity in a Protein Arginine Methyltransferase</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">2068</span>– <span class="NLM_lpage">2073</span>, <span class="refDoi"> DOI: 10.1073/pnas.1525783113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1073%2Fpnas.1525783113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=26858449" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC28Xit12ku7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=2068-2073&author=E.+W.+Deblerauthor=K.+Jainauthor=R.+A.+Warmackauthor=Y.+Fengauthor=S.+G.+Clarkeauthor=G.+Blobel&title=A+Glutamate%2FAspartate+Switch+Controls+Product+Specificity+in+a+Protein+Arginine+Methyltransferase&doi=10.1073%2Fpnas.1525783113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">A glutamate/aspartate switch controls product specificity in a protein arginine methyltransferase</span></div><div class="casAuthors">Debler, Erik W.; Jain, Kanishk; Warmack, Rebeccah A.; Feng, You; Clarke, Steven G.; Blobel, Gunter; Stavropoulos, Pete</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2068-2073</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Trypanosoma brucei PRMT7 (TbPRMT7) is a protein arginine methyltransferase (PRMT) that strictly monomethylates various substrates, thus classifying it as a type III PRMT.  However, the mol. basis of its unique product specificity has remained elusive.  Here, we present the structure of TbPRMT7 in complex with its cofactor product S-adenosyl-L-homocysteine (AdoHcy) at 2.8 Å resoln. and identify a glutamate residue crit. for its monomethylation behavior.  TbPRMT7 comprises the conserved methyltransferase and β-barrel domains, an N-terminal extension, and a dimerization arm.  The active site at the interface of the N-terminal extension, methyltransferase, and β-barrel domains is stabilized by the dimerization arm of the neighboring protomer, providing a structural basis for dimerization as a prerequisite for catalytic activity.  Mutagenesis of active-site residues highlights the importance of Glu181, the second of the two invariant glutamate residues of the double E loop that coordinate the target arginine in substrate peptides/proteins and that increase its nucleophilicity.  Strikingly, mutation of Glu181 to aspartate converts TbPRMT7 into a type I PRMT, producing asym. dimethylarginine (ADMA).  Isothermal titrn. calorimetry (ITC) using a histone H4 peptide showed that the Glu181Asp mutant has markedly increased affinity for monomethylated peptide with respect to the WT, suggesting that the enlarged active site can favorably accommodate monomethylated peptide and provide sufficient space for ADMA formation.  In conclusion, these findings yield valuable insights into the product specificity and the catalytic mechanism of protein arginine methyltransferases and have important implications for the rational (re)design of PRMTs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrj2myjluq1B7Vg90H21EOLACvtfcHk0ljUJ-uo_qw79A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xit12ku7k%253D&md5=93f9d17569159c432b5801e2439c6df0</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1525783113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1525783113%26sid%3Dliteratum%253Aachs%26aulast%3DDebler%26aufirst%3DE.%2BW.%26aulast%3DJain%26aufirst%3DK.%26aulast%3DWarmack%26aufirst%3DR.%2BA.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DClarke%26aufirst%3DS.%2BG.%26aulast%3DBlobel%26aufirst%3DG.%26atitle%3DA%2520Glutamate%252FAspartate%2520Switch%2520Controls%2520Product%2520Specificity%2520in%2520a%2520Protein%2520Arginine%2520Methyltransferase%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2016%26volume%3D113%26spage%3D2068%26epage%3D2073%26doi%3D10.1073%2Fpnas.1525783113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iyamu, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yadav, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noinaj, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Novel Propargyl-Linked Bisubstrate Analogues as Tight-Binding Inhibitors for Nicotinamide N-Methyltransferase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">10783</span>– <span class="NLM_lpage">10797</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01255</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01255" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFKksLvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=10783-10797&author=D.+Chenauthor=L.+Liauthor=K.+Diazauthor=I.+D.+Iyamuauthor=R.+Yadavauthor=N.+Noinajauthor=R.+Huang&title=Novel+Propargyl-Linked+Bisubstrate+Analogues+as+Tight-Binding+Inhibitors+for+Nicotinamide+N-Methyltransferase&doi=10.1021%2Facs.jmedchem.9b01255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Novel propargyl-linked bisubstrate analogues as tight-binding inhibitors for nicotinamide N-methyltransferase</span></div><div class="casAuthors">Chen, Dongxing; Li, Linjie; Diaz, Krystal; Iyamu, Iredia D.; Yadav, Ravi; Noinaj, Nicholas; Huang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10783-10797</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) catalyzes the Me transfer from the cofactor S-adenosylmethionine to nicotinamide and other pyridine-contg. compds.  NNMT is an important regulator for nicotinamide metab. and methylation potential.  Aberrant expression levels of NNMT have been implicated in cancer, metabolic, and neurodegenerative diseases, which makes NNMT a potential therapeutic target.  Therefore, potent and selective NNMT inhibitors can serve as valuable tools to investigate the roles of NNMT in its mediated diseases.  Here, we applied a rational strategy to design and synthesize the tight-binding bisubstrate inhibitor LL320 through a novel propargyl linker.  LL320 demonstrates a Ki value of 1.6 ± 0.3 nM, which is the most potent inhibitor to date.  The cocrystal structure of LL320 confirms its interaction with both the substrate and cofactor binding sites on NNMT.  Importantly, this is the first example of using the propargyl linker to construct potent methyltransferase inhibitors, which will expand our understanding of the transition state of Me transfer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqs2XyawAg9ebVg90H21EOLACvtfcHk0ljUJ-uo_qw79A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFKksLvN&md5=a6e3a5dc2044dc50ede1596c84804f61</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01255%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DDiaz%26aufirst%3DK.%26aulast%3DIyamu%26aufirst%3DI.%2BD.%26aulast%3DYadav%26aufirst%3DR.%26aulast%3DNoinaj%26aufirst%3DN.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DNovel%2520Propargyl-Linked%2520Bisubstrate%2520Analogues%2520as%2520Tight-Binding%2520Inhibitors%2520for%2520Nicotinamide%2520N-Methyltransferase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D10783%26epage%3D10797%26doi%3D10.1021%2Facs.jmedchem.9b01255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G.</span>; <span class="NLM_string-name">Ashton, A.</span></span>.  <i>Fast_dp. Methods Zenodo</i>, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=G.+Winter&author=A.+Ashton&title=Fast_dp.+Methods+Zenodo"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%26jtitle%3DFast_dp.%2520Methods%2520Zenodo%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkóczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: A Comprehensive Python-Based System for Macromolecular Structure Solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+A+Comprehensive+Python-Based+System+for+Macromolecular+Structure+Solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0ljUJ-uo_qw79A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520A%2520Comprehensive%2520Python-Based%2520System%2520for%2520Macromolecular%2520Structure%2520Solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and Development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+Development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lj24hPT891_6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520Development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 7 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Dongxing Chen, Ying Meng, Dan Yu, Nicholas Noinaj, Xiaodong Cheng, <span class="NLM_string-name hlFld-ContribAuthor">Rong Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Chemoproteomic Study Uncovers HemK2/KMT9 As a New Target for NTMT1 Bisubstrate Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2021,</strong> <em>16 </em>
                                    (7)
                                     , 1234-1242. <a href="https://doi.org/10.1021/acschembio.1c00279" title="DOI URL">https://doi.org/10.1021/acschembio.1c00279</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.1c00279&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.1c00279%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DChemoproteomic%252BStudy%252BUncovers%252BHemK2%25252FKMT9%252BAs%252Ba%252BNew%252BTarget%252Bfor%252BNTMT1%252BBisubstrate%252BInhibitors%26aulast%3DChen%26aufirst%3DDongxing%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D13042021%26date%3D18062021%26date%3D01072021%26volume%3D16%26issue%3D7%26spage%3D1234%26epage%3D1242" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Dongxing Chen, Guangping Dong, Youchao Deng, Nicholas Noinaj, <span class="NLM_string-name hlFld-ContribAuthor">Rong Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-based Discovery of Cell-Potent Peptidomimetic Inhibitors for Protein N-Terminal Methyltransferase 1. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2021,</strong> <em>12 </em>
                                    (3)
                                     , 485-493. <a href="https://doi.org/10.1021/acsmedchemlett.1c00012" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.1c00012</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.1c00012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.1c00012%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DStructure-based%252BDiscovery%252Bof%252BCell-Potent%252BPeptidomimetic%252BInhibitors%252Bfor%252BProtein%252BN-Terminal%252BMethyltransferase%252B1%26aulast%3DChen%26aufirst%3DDongxing%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D06012021%26date%3D24022021%26date%3D01032021%26volume%3D12%26issue%3D3%26spage%3D485%26epage%3D493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jian Lu, Aaron G. Bart, Qian Wu, Kevin R. Criscione, Michael J. McLeish, Emily E. Scott, <span class="NLM_string-name hlFld-ContribAuthor">Gary L. Grunewald</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based Drug Design of Bisubstrate Inhibitors of Phenylethanolamine N-Methyltransferase Possessing Low Nanomolar Affinity at Both Substrate Binding Domains1. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (22)
                                     , 13878-13898. <a href="https://doi.org/10.1021/acs.jmedchem.0c01475" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01475</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01475&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01475%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-Based%252BDrug%252BDesign%252Bof%252BBisubstrate%252BInhibitors%252Bof%252BPhenylethanolamine%252BN-Methyltransferase%252BPossessing%252BLow%252BNanomolar%252BAffinity%252Bat%252BBoth%252BSubstrate%252BBinding%252BDomains1%26aulast%3DLu%26aufirst%3DJian%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D24082020%26date%3D04112020%26volume%3D63%26issue%3D22%26spage%3D13878%26epage%3D13898" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Brianna D. Mackie, Dongxing Chen, Guangping Dong, Cheng Dong, Haley Parker, Christine E. Schaner Tooley, Nicholas Noinaj, Jinrong Min, <span class="NLM_string-name hlFld-ContribAuthor">Rong Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Selective Peptidomimetic Inhibitors of NTMT1/2: Rational Design, Synthesis, Characterization, and Crystallographic Studies. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (17)
                                     , 9512-9522. <a href="https://doi.org/10.1021/acs.jmedchem.0c00689" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00689</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00689&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00689%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSelective%252BPeptidomimetic%252BInhibitors%252Bof%252BNTMT1%25252F2%25253A%252BRational%252BDesign%25252C%252BSynthesis%25252C%252BCharacterization%25252C%252Band%252BCrystallographic%252BStudies%26aulast%3DMackie%26aufirst%3DBrianna%2BD.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D24042020%26date%3D05082020%26date%3D21072020%26volume%3D63%26issue%3D17%26spage%3D9512%26epage%3D9522" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Krystal  Diaz</span>, <span class="hlFld-ContribAuthor ">Ying  Meng</span>, <span class="hlFld-ContribAuthor ">Rong  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Past, present, and perspectives of protein N-terminal methylation. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2021,</strong> <em>63 </em>, 115-122. <a href="https://doi.org/10.1016/j.cbpa.2021.02.017" title="DOI URL">https://doi.org/10.1016/j.cbpa.2021.02.017</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2021.02.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2021.02.017%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DPast%25252C%252Bpresent%25252C%252Band%252Bperspectives%252Bof%252Bprotein%252BN-terminal%252Bmethylation%26aulast%3DDiaz%26aufirst%3DKrystal%26date%3D2021%26volume%3D63%26spage%3D115%26epage%3D122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Iredia D.  Iyamu</span>, <span class="hlFld-ContribAuthor ">Rong  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Mechanisms and inhibitors of nicotinamide
              N
              -methyltransferase. </span><span class="cited-content_cbyCitation_journal-name">RSC Medicinal Chemistry</span><span> <strong>2021,</strong> <em>28 </em><a href="https://doi.org/10.1039/D1MD00016K" title="DOI URL">https://doi.org/10.1039/D1MD00016K</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1MD00016K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1MD00016K%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Medicinal%2520Chemistry%26atitle%3DMechanisms%252Band%252Binhibitors%252Bof%252Bnicotinamide%252BN%252B-methyltransferase%26aulast%3DIyamu%26aufirst%3DIredia%2BD.%26date%3D2021%26date%3D2021%26volume%3D28" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ayad A.  Al-Hamashi</span>, <span class="hlFld-ContribAuthor ">Dongxing  Chen</span>, <span class="hlFld-ContribAuthor ">Youchao  Deng</span>, <span class="hlFld-ContribAuthor ">Guangping  Dong</span>, <span class="hlFld-ContribAuthor ">Rong  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a potent and dual-selective bisubstrate inhibitor for protein arginine methyltransferase 4/5. </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmaceutica Sinica B</span><span> <strong>2020,</strong> <em>57 </em><a href="https://doi.org/10.1016/j.apsb.2020.10.013" title="DOI URL">https://doi.org/10.1016/j.apsb.2020.10.013</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.apsb.2020.10.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.apsb.2020.10.013%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmaceutica%2520Sinica%2520B%26atitle%3DDiscovery%252Bof%252Ba%252Bpotent%252Band%252Bdual-selective%252Bbisubstrate%252Binhibitor%252Bfor%252Bprotein%252Barginine%252Bmethyltransferase%252B4%25252F5%26aulast%3DAl-Hamashi%26aufirst%3DAyad%2BA.%26date%3D2020%26volume%3D57" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/medium/jm0c00770_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/large/jm0c00770_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00770&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/medium/jm0c00770_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/large/jm0c00770_0001.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Chemical structures of NTMT1 inhibitors. (B) Design of the bisubstrate analogues for NTMT1.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/large/jm0c00770_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00770&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/medium/jm0c00770_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/large/jm0c00770_0009.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthetic Route for Compounds <b>1</b>–<b>9</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/large/jm0c00770_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00770&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaBH<sub>3</sub>CN, CH<sub>3</sub>OH, r.t., 38–68%; (b) 10% Pd/C, H<sub>2</sub>, MeOH, 56%; (c) 1,8-diazabicyclo[5.4.0]undec-7-ene (DBU), CH<sub>3</sub>CN, 0 °C to r.t., 71%; (d) α-bromo peptides, <i>N</i>,<i>N</i>-diisopropylethylamine (DIPEA), dimethylformamide (DMF), r.t.; (e) TFA/DODT/H<sub>2</sub>O/TIPS = 94:2.5:2.5:1; (f) trifluoroacetic acid (TFA), CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to r.t., 42%; and (g) 4N HCl in dioxane, H<sub>2</sub>O, 0 °C to r.t., 45%.</p></p></figure><figure data-id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/medium/jm0c00770_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/large/jm0c00770_0010.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthetic Route for Compounds <b>10</b>–<b>12</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/large/jm0c00770_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00770&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaBH<sub>3</sub>CN, CH<sub>3</sub>OH, r.t., 71%; (b) (Boc)<sub>2</sub>O, NaHCO<sub>3</sub>, tetrahydrofuran (THF), H<sub>2</sub>O, 85%; (c) DBU, CH<sub>3</sub>CN, 0 °C to r.t., 50%; (d) α-bromo peptides, DIPEA, DMF, r.t.; and (e) TFA/DODT/H<sub>2</sub>O/TIPS = 94: 2.5: 2.5: 1.</p></p></figure><figure data-id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/medium/jm0c00770_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/large/jm0c00770_0002.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. MALDI-MS methylation assay for bisubstrate analogues <b>1</b>–<b>4</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/large/jm0c00770_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00770&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/medium/jm0c00770_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/large/jm0c00770_0003.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Deconstruction strategy and inhibition evaluation of the compounds. The IC<sub>50</sub> value of <b>8</b> was determined by the MALDI-MS methylation inhibition assay, while <b>9</b> was determined by the SAHH-coupling assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/large/jm0c00770_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00770&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/medium/jm0c00770_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/large/jm0c00770_0004.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Time-dependent inhibition study of compounds <b>1</b>–<b>3</b>. All experiments were run in duplicate (<i>n</i> = 2), except for 10 min of preincubation (<i>n</i> = 3). (A), (C), and (E) Concentration–response plots for different preincubation times fitted to Morrison’s quadratic equation for compounds <b>1</b>, <b>2</b>, and <b>3</b>, respectively. (B), (D), and (F) Nonlinear regression of Morrison <i>K</i><sub>i,app</sub> values with preincubation times for compounds <b>1</b>, <b>2</b>, and <b>3</b>, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/large/jm0c00770_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00770&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/medium/jm0c00770_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/large/jm0c00770_0005.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Binding affinity for NTMT1 (A) and NTMT2 (B). ND, not detected.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/large/jm0c00770_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00770&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/medium/jm0c00770_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/large/jm0c00770_0006.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (A, B) MALDI-MS methylation inhibition assay for compound <b>3</b> (<i>n</i> = 2). (A) MALDI-MS results of MALDI-MS methylation inhibition assay for compound <b>3</b> and (B) quantification of methylation progression of GPKRIA by NTMT1 with <b>3</b> at 20 min. (C) Selectivity evaluation of compound <b>3</b> (<i>n</i> = 2).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/large/jm0c00770_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00770&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/medium/jm0c00770_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/large/jm0c00770_0007.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. X-ray cocrystal structures. (A) X-ray crystal structure of compound <b>1</b> (purple stick) in complex with NTMT1 (gray ribbon) (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WJ7">6WJ7</a>). H-bond interactions are shown in yellow dotted lines. (B) Overlay of the crystal structure of NTMT1 (gray ribbon)–<b>1</b> (purple stick) complex with the crystal structure of the NTMT1-<b>NAH-C3-PPKRIA</b> (orange stick) binary complex (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DTN">6DTN</a>). (C) compound <b>1</b> two-dimensional (2D) interaction diagram (Schrödinger Maestro) with NTMT1. (D) X-ray crystal structure of NTMT1 (gray ribbon)–<b>4</b> (yellow stick)–SAH (gray stick) (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PVB">6PVB</a>). H-bond interactions are shown in yellow dotted lines. (E) Overlay of the crystal structure of NTMT1 (gray ribbon)–<b>4</b> (yellow stick)–SAH (gray stick) complex (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PVB">6PVB</a>) with the crystal structure of the NTMT1-<b>NAH-C3-PPKRIA</b> (orange stick) binary complex (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DTN">6DTN</a>). (F) Compound <b>4</b> 2D interaction diagram (Schrödinger Maestro) with NTMT1.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/large/jm0c00770_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00770&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/medium/jm0c00770_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/large/jm0c00770_0008.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Inhibition mechanism studies for compounds <b>1</b>–<b>3</b>. (A–C) IC<sub>50</sub> curves of <b>1</b>–<b>3</b> at varying concentrations of the peptide with a fixed concentration of SAM. (D–F) IC<sub>50</sub> curves of <b>1</b>–<b>3</b> at varying concentrations of SAM with a fixed concentration of the peptide. The preincubation time for the assay was 120 min, and all of the experiments were run in duplicates (<i>n</i> = 2).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00770/20200806/images/large/jm0c00770_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00770&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i50">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25156" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25156" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 47 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowhan, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muscat, G. E. O.</span></span> <span> </span><span class="NLM_article-title">Epigenetic Arginine Methylation in Breast Cancer: Emerging Therapeutic Strategies</span>. <i>J. Mol. Endocrinol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">R223</span>– <span class="NLM_lpage">R227</span>, <span class="refDoi"> DOI: 10.1530/JME-18-0224</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1530%2FJME-18-0224" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=30620710" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1eiu7vI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=R223-R227&author=S.+M.+Wangauthor=S.+M.+Wangauthor=D.+H.+Dowhanauthor=G.+E.+O.+Muscat&title=Epigenetic+Arginine+Methylation+in+Breast+Cancer%3A+Emerging+Therapeutic+Strategies&doi=10.1530%2FJME-18-0224"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic arginine methylation in breast cancer: emerging therapeutic strategies</span></div><div class="casAuthors">Wang, Shu-Ching M.; Dowhan, Dennis H.; Muscat, George E. O.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Endocrinology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">R223-R237</span>CODEN:
                <span class="NLM_cas:coden">JMLEEI</span>;
        ISSN:<span class="NLM_cas:issn">1479-6813</span>.
    
            (<span class="NLM_cas:orgname">Bioscientifica Ltd.</span>)
        </div><div class="casAbstract">A review.  Breast cancer is a heterogeneous disease, and the complexity of breast carcinogenesis is assocd. with epigenetic modifiation.  There are several major classes of epigenetic enzymes that regulate chromatin activity.  This review will focus on the nine mammalian protein arginine methyltransferases (PRMTs) and the dysregulation of PRMT expression and function in breast cancer.  This class of enzymes catalyze the mono- and (sym. and asym.) di-methylation of arginine residues on histone and non-histone target proteins.  PRMT signalling (and R methylation) drives cellular proliferation, cell invasion and metastasis, targeting (i) nuclear hormone receptor signalling, (ii) tumor suppressors, (iii) TGF- β and EMT signalling and (iv) alternative splicing and DNA/chromatin stability, inflencing the clin. and survival outcomes in breast cancer.  Emerging reports suggest that PRMTs are also implicated in the development of drug/endocrine resistance providing another prospective avenue for the treatment of hormone resistance and assocd. metastasis.  The complexity of PRMT signalling is further underscored by the degree of alternative splicing and the scope of variant isoforms (with distinct properties) within each PRMT family member.  The evolution of PRMT inhibitors, and the ongoing clin. trials of PRMT inhibitors against a subgroup of solid cancers, coupled to the track record of lysine methyltransferases inhibitors in phase I/II clin. trials against cancer underscores the potential therapeutic utility of targeting PRMT epigenetic enzymes to improve survival outcomes in aggressive and metastatic breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYtQu27dxxebVg90H21EOLACvtfcHk0ljGFgOCW3nXQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1eiu7vI&md5=221a3f95d307b8a4db0720d2164792b3</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1530%2FJME-18-0224&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1530%252FJME-18-0224%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DS.%2BM.%26aulast%3DWang%26aufirst%3DS.%2BM.%26aulast%3DDowhan%26aufirst%3DD.%2BH.%26aulast%3DMuscat%26aufirst%3DG.%2BE.%2BO.%26atitle%3DEpigenetic%2520Arginine%2520Methylation%2520in%2520Breast%2520Cancer%253A%2520Emerging%2520Therapeutic%2520Strategies%26jtitle%3DJ.%2520Mol.%2520Endocrinol.%26date%3D2019%26volume%3D62%26spage%3DR223%26epage%3DR227%26doi%3D10.1530%2FJME-18-0224" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. Ü.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martini, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of Protein Methyltransferases and Demethylases</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">989</span>– <span class="NLM_lpage">1068</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.6b00801</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.6b00801" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkslynsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2018&pages=989-1068&author=H.+%C3%9C.+Kaniskanauthor=M.+L.+Martiniauthor=J.+Jin&title=Inhibitors+of+Protein+Methyltransferases+and+Demethylases&doi=10.1021%2Facs.chemrev.6b00801"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of Protein Methyltransferases and Demethylases</span></div><div class="casAuthors">Kaniskan, H. Umit; Martini, Michael L.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">989-1068</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Post-translational modifications of histones by protein methyltransferases (PMTs) and histone demethylases (KDMs) play an important role in the regulation of gene expression and transcription and are implicated in cancer and many other diseases.  Many of these enzymes also target various nonhistone proteins impacting numerous crucial biol. pathways.  Given their key biol. functions and implications in human diseases, there has been a growing interest in assessing these enzymes as potential therapeutic targets.  Consequently, discovering and developing inhibitors of these enzymes has become a very active and fast-growing research area over the past decade.  In this review, the authors cover the discovery, characterization, and biol. application of inhibitors of PMTs and KDMs with emphasis on key advancements in the field.  The authors also discuss challenges, opportunities, and future directions in this emerging, exciting research field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJYjiYtw_n4rVg90H21EOLACvtfcHk0ljGFgOCW3nXQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkslynsL8%253D&md5=1ed520c6b0c42d2aa8a7f66b0a2d4e40</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.6b00801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.6b00801%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3DMartini%26aufirst%3DM.%2BL.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DInhibitors%2520of%2520Protein%2520Methyltransferases%2520and%2520Demethylases%26jtitle%3DChem.%2520Rev.%26date%3D2018%26volume%3D118%26spage%3D989%26epage%3D1068%26doi%3D10.1021%2Facs.chemrev.6b00801" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niessen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goebel, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yates, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montminy, M.</span></span> <span> </span><span class="NLM_article-title">PRMT5 Modulates the Metabolic Response to Fasting Signals</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">8870</span>– <span class="NLM_lpage">8875</span>, <span class="refDoi"> DOI: 10.1073/pnas.1304602110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1073%2Fpnas.1304602110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=23671120" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFalsbjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=8870-8875&author=W.+Tsaiauthor=S.+Niessenauthor=N.+Goebelauthor=J.+R.+Yatesauthor=E.+Guccioneauthor=M.+Montminy&title=PRMT5+Modulates+the+Metabolic+Response+to+Fasting+Signals&doi=10.1073%2Fpnas.1304602110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">PRMT5 modulates the metabolic response to fasting signals</span></div><div class="casAuthors">Tsai, Wen-Wei; Niessen, Sherry; Goebel, Naomi; Yates, John R., III; Guccione, Ernesto; Montminy, Marc</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">8870-8875, S8870/1-S8870/5</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Under fasting conditions, increases in circulating glucagon maintain glucose balance by promoting hepatic gluconeogenesis.  Triggering of the cAMP pathway stimulates gluconeogenic gene expression through the PKA-mediated phosphorylation of the cAMP response element binding (CREB) protein and via the dephosphorylation of the latent cytoplasmic CREB regulated transcriptional coactivator 2 (CRTC2).  CREB and CRTC2 activities are increased in insulin resistance, in which they promote hyperglycemia because of constitutive induction of the gluconeogenic program.  The extent to which CREB and CRTC2 are coordinately up-regulated in response to glucagon, however, remains unclear.  Here it is shown that, following its activation, CRTC2 enhances CREB phosphorylation through an assocn. with the protein arginine methyltransferase 5 (PRMT5).  In turn, PRMT5 was found to stimulate CREB phosphorylation via increases in histone H3 Arg2 methylation that enhanced chromatin accessibility at gluconeogenic promoters.  Because depletion of PRMT5 lowers hepatic glucose prodn. and gluconeogenic gene expression, these results demonstrate how a chromatin-modifying enzyme regulates a metabolic program through epigenetic changes that impact the phosphorylation of a transcription factor in response to hormonal stimuli.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkZvybDjBzyrVg90H21EOLACvtfcHk0ljGFgOCW3nXQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFalsbjF&md5=6a46c62348b78963e2ae08a5eb2a01bb</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1304602110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1304602110%26sid%3Dliteratum%253Aachs%26aulast%3DTsai%26aufirst%3DW.%26aulast%3DNiessen%26aufirst%3DS.%26aulast%3DGoebel%26aufirst%3DN.%26aulast%3DYates%26aufirst%3DJ.%2BR.%26aulast%3DGuccione%26aufirst%3DE.%26aulast%3DMontminy%26aufirst%3DM.%26atitle%3DPRMT5%2520Modulates%2520the%2520Metabolic%2520Response%2520to%2520Fasting%2520Signals%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2013%26volume%3D110%26spage%3D8870%26epage%3D8875%26doi%3D10.1073%2Fpnas.1304602110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, D.</span></span> <span> </span><span class="NLM_article-title">Coactivator-Associated Arginine Methyltransferase 1 Targeted by MiR-15a Regulates Inflammation in Acute Coronary Syndrome</span>. <i>Atherosclerosis</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>233</i></span>,  <span class="NLM_fpage">349</span>– <span class="NLM_lpage">356</span>, <span class="refDoi"> DOI: 10.1016/j.atherosclerosis.2014.01.039</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1016%2Fj.atherosclerosis.2014.01.039" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=24530761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC2cXks1Wntrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=233&publication_year=2014&pages=349-356&author=X.+Liuauthor=L.+Wangauthor=H.+Liauthor=X.+Luauthor=Y.+Huauthor=X.+Yangauthor=C.+Huangauthor=D.+Gu&title=Coactivator-Associated+Arginine+Methyltransferase+1+Targeted+by+MiR-15a+Regulates+Inflammation+in+Acute+Coronary+Syndrome&doi=10.1016%2Fj.atherosclerosis.2014.01.039"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Coactivator-associated arginine methyltransferase 1 targeted by miR-15a regulates inflammation in acute coronary syndrome</span></div><div class="casAuthors">Liu, Xuehui; Wang, Laiyuan; Li, Hongfan; Lu, Xiangfeng; Hu, Yongyan; Yang, Xueli; Huang, Chen; Gu, Dongfeng</div><div class="citationInfo"><span class="NLM_cas:title">Atherosclerosis (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">233</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">349-356</span>CODEN:
                <span class="NLM_cas:coden">ATHSBL</span>;
        ISSN:<span class="NLM_cas:issn">0021-9150</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Coactivator-assocd. arginine methyltransferase 1 (CARM1) is essential for the activation of a subset of NF-κB-dependent genes, which code the chemokines triggering plaque vulnerability.  Unstable atherosclerotic plaques lead to the onset of acute coronary syndrome (ACS).  Therefore, we aimed to investigate whether CARM1 is involved in the pathogenesis of ACS and ascertain the regulatory mechanism of CARM1 expression at posttranscriptional level.  Peripheral blood mononuclear cells (PBMCs) were isolated from peripheral blood of 19 patients with ACS and 22 subjects with risk factors for coronary heart disease.  Gene expression was detd. by quant. real-time PCR and Western blot.  The effects of CARM1 and miR-15a on their target genes expression were assessed by gain-of-function and loss-of-function approaches.  PBMCs from patients with ACS showed higher levels of CARM1 mRNA and protein expression.  The levels of CARM1 mRNA were pos. correlated with three chemokines including interferon-inducible protein-10 (IP-10), monocyte chemoattractant protein 1 (MCP-1), and interleukin-8 (IL-8) in PBMCs (CARM1 and IP-10: r = 0.55, P = 0.008; CARM1 and MCP-1: r = 0.64, P = 0.002; CARM1 and IL-8: r = 0.55, P = 0.008).  Moreover, CARM1 regulated the transcription of these chemokines in human embryonic kidney 293T (HEK293T) cells.  We also found that the levels of miR-15a were decreased by 37% in patients with ACS and miR-15a modulated CARM1 expression through targeted binding to CARM1 3'-UTR.  The present study demonstrated that CARM1 targeted by miR-15a played an important role in chemokine activation in the pathogenesis of ACS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrODlfcuAxfYrVg90H21EOLACvtfcHk0ljK4GWFxPl-FQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXks1Wntrc%253D&md5=dcb8ced710aad25aa69e9c5e30943f21</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.atherosclerosis.2014.01.039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.atherosclerosis.2014.01.039%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DC.%26aulast%3DGu%26aufirst%3DD.%26atitle%3DCoactivator-Associated%2520Arginine%2520Methyltransferase%25201%2520Targeted%2520by%2520MiR-15a%2520Regulates%2520Inflammation%2520in%2520Acute%2520Coronary%2520Syndrome%26jtitle%3DAtherosclerosis%26date%3D2014%26volume%3D233%26spage%3D349%26epage%3D356%26doi%3D10.1016%2Fj.atherosclerosis.2014.01.039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M.</span></span> <span> </span><span class="NLM_article-title">Chemical and Biochemical Perspectives of Protein Lysine Methylation</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">6656</span>– <span class="NLM_lpage">6705</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.8b00008</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.8b00008" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFKks7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2018&pages=6656-6705&author=M.+Luo&title=Chemical+and+Biochemical+Perspectives+of+Protein+Lysine+Methylation&doi=10.1021%2Facs.chemrev.8b00008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical and Biochemical Perspectives of Protein Lysine Methylation</span></div><div class="casAuthors">Luo, Minkui</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6656-6705</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Protein lysine methylation is a distinct post-translational modification that causes minimal changes in the size and electrostatic status of lysine residues.  Lysine methylation plays essential roles in regulating fates and functions of target proteins in an epigenetic manner.  As a result, substrates and degrees (free vs. mono/di/tri) of protein lysine methylation are orchestrated within cells by balanced activities of protein lysine methyltransferases (PKMTs) and demethylases (KDMs).  Their dysregulation is often assocd. with neurol. disorders, developmental abnormalities or cancer.  Methyllysine-contg. proteins can be recognized by downstream effector proteins, which contain methyllysine reader domains, to relay their biol. functions.  While numerous efforts have been made to annotate biol. roles of protein lysine methylation, limited work has been done to uncover mechanisms assocd. with this modification at a mol. or at. level.  Given distinct biophys. and biochem. properties of methyllysine, this review will focus on chem. and biochem. aspects in addn., recognition, and removal of this post-translational mark.  Chem. and biophys. methods to profile PKMT substrates will be discussed along with classification of PKMT inhibitors for accurate perturbation of methyltransferase activities.  Semisynthesis of methyllysine-contg. proteins will also be covered given the crit. need for these reagents to unambiguously define functional roles of protein lysine methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbSRWg3jP5bLVg90H21EOLACvtfcHk0ljK4GWFxPl-FQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFKks7zO&md5=d846db925be49839c5954f5fc34f17d9</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.8b00008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.8b00008%26sid%3Dliteratum%253Aachs%26aulast%3DLuo%26aufirst%3DM.%26atitle%3DChemical%2520and%2520Biochemical%2520Perspectives%2520of%2520Protein%2520Lysine%2520Methylation%26jtitle%3DChem.%2520Rev.%26date%3D2018%26volume%3D118%26spage%3D6656%26epage%3D6705%26doi%3D10.1021%2Facs.chemrev.8b00008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanjra, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Kinetic Mechanism of Protein N-Terminal Methyltransferase 1</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>290</i></span>,  <span class="NLM_fpage">11601</span>– <span class="NLM_lpage">11610</span>, <span class="refDoi"> DOI: 10.1074/jbc.M114.626846</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1074%2Fjbc.M114.626846" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=25771539" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnsFCrs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=11601-11610&author=S.+L.+Richardsonauthor=Y.+Maoauthor=G.+Zhangauthor=P.+Hanjraauthor=D.+L.+Petersonauthor=R.+Huang&title=Kinetic+Mechanism+of+Protein+N-Terminal+Methyltransferase+1&doi=10.1074%2Fjbc.M114.626846"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic Mechanism of Protein N-terminal Methyltransferase 1</span></div><div class="casAuthors">Richardson, Stacie L.; Mao, Yunfei; Zhang, Gang; Hanjra, Pahul; Peterson, Darrell L.; Huang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">290</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">11601-11610</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The protein N-terminal methyltransferase 1 (NTMT1) catalyzes the transfer of the Me group from the S-adenosyl-L-methionine (AdoMet) to the protein α-amine, resulting in formation of S-adenosyl-L-homocysteine (AdoHcy) and α-N-methylated proteins.  NTMT1 is an interesting potential anticancer target because it is overexpressed in gastrointestinal cancers and plays an important role in cell mitosis.  To gain insight into the biochem. mechanism of NTMT1, we have characterized the kinetic mechanism of recombinant NTMT1 using a fluorescence assay and mass spectrometry.  The results of initial velocity, product, and dead-end inhibition studies indicate that methylation by NTMT1 proceeds via a random sequential Bi Bi mechanism.  In addn., our processivity studies demonstrate that NTMT1 proceeds via a distributive mechanism for multiple methylations.  Together, our studies provide new knowledge about the kinetic mechanism of NTMT1 and lay the foundation for the development of mechanism-based inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYCNmI8h5YRbVg90H21EOLACvtfcHk0ljK4GWFxPl-FQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnsFCrs7g%253D&md5=28a62d8c1b378fa4fabceca206e61464</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M114.626846&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M114.626846%26sid%3Dliteratum%253Aachs%26aulast%3DRichardson%26aufirst%3DS.%2BL.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DHanjra%26aufirst%3DP.%26aulast%3DPeterson%26aufirst%3DD.%2BL.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DKinetic%2520Mechanism%2520of%2520Protein%2520N-Terminal%2520Methyltransferase%25201%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2015%26volume%3D290%26spage%3D11601%26epage%3D11610%26doi%3D10.1074%2Fjbc.M114.626846" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Khalil, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Angelis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishii, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span> <span> </span><span class="NLM_article-title">Mechanism-based Inhibition of the Melatonin Rhythem Enzyme: Pharmacologic Exploitation of Active Site Functional Plasticity</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>96</i></span>,  <span class="NLM_fpage">12418</span>– <span class="NLM_lpage">12423</span>, <span class="refDoi"> DOI: 10.1073/pnas.96.22.12418</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1073%2Fpnas.96.22.12418" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10535937" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADyaK1MXnt1yrtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=1999&pages=12418-12423&author=E.+M.+Khalilauthor=J.+De+Angelisauthor=M.+Ishiiauthor=P.+A.+Cole&title=Mechanism-based+Inhibition+of+the+Melatonin+Rhythem+Enzyme%3A+Pharmacologic+Exploitation+of+Active+Site+Functional+Plasticity&doi=10.1073%2Fpnas.96.22.12418"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism-based inhibition of the melatonin rhythm enzyme: pharmacologic exploitation of active site functional plasticity</span></div><div class="casAuthors">Khalil, Ehab M.; De Angelis, Jacqueline; Ishii, Makoto; Cole, Philip A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">96</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">12418-12423</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Serotonin N-acetyltransferase is the enzyme responsible for the diurnal rhythm of melatonin prodn. in the pineal gland of animals and humans.  Inhibitors of this enzyme active in cell culture have not been reported previously.  The compd. N-bromoacetyltryptamine was shown to be a potent inhibitor of this enzyme in vitro and in a pineal cell culture assay (IC50 ≈ 500 nM).  The mechanism of inhibition is suggested to involve a serotonin N-acetyltransferase-catalyzed alkylation reaction between N-bromoacetyltryptamine and reduced CoA, resulting in the prodn. of a tight-binding bisubstrate analog inhibitor.  This alkyltransferase activity is apparently catalyzed at a functionally distinct site compared with the acetyltransferase activity active site on serotonin N-acetyltransferase.  Such active site plasticity is suggested to result from a subtle conformational alteration in the protein.  This plasticity allows for an unusual form of mechanism-based inhibition with multiple turnovers, resulting in "mol. fratricide.".  N-bromoacetyltryptamine should serve as a useful tool for dissecting the role of melatonin in circadian rhythm as well as a potential lead compd. for therapeutic use in mood and sleep disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOeqv_lFsxOLVg90H21EOLACvtfcHk0liAQqRUl8p7jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXnt1yrtbY%253D&md5=dda5816db0455e31a50cb72cd4e02883</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1073%2Fpnas.96.22.12418&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.96.22.12418%26sid%3Dliteratum%253Aachs%26aulast%3DKhalil%26aufirst%3DE.%2BM.%26aulast%3DDe%2BAngelis%26aufirst%3DJ.%26aulast%3DIshii%26aufirst%3DM.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DMechanism-based%2520Inhibition%2520of%2520the%2520Melatonin%2520Rhythem%2520Enzyme%253A%2520Pharmacologic%2520Exploitation%2520of%2520Active%2520Site%2520Functional%2520Plasticity%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1999%26volume%3D96%26spage%3D12418%26epage%3D12423%26doi%3D10.1073%2Fpnas.96.22.12418" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magalha, M. L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blanchard, J. S.</span></span> <span> </span><span class="NLM_article-title">Bisubstrate Inhibition: Theory and Application to N-Acetyltransferases</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">14788</span>– <span class="NLM_lpage">14794</span>, <span class="refDoi"> DOI: 10.1021/bi061621t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi061621t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1SjtbfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2006&pages=14788-14794&author=M.+Yuauthor=M.+L.+B.+Magalhaauthor=P.+F.+Cookauthor=J.+S.+Blanchard&title=Bisubstrate+Inhibition%3A+Theory+and+Application+to+N-Acetyltransferases&doi=10.1021%2Fbi061621t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Bisubstrate Inhibition: Theory and Application to N-Acetyltransferases</span></div><div class="casAuthors">Yu, Michael; Magalhaes, Maria L. B.; Cook, Paul F.; Blanchard, John S.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">14788-14794</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bisubstrate inhibitors represent a potentially powerful group of compds. that have found significant therapeutic utility.  Although these compds. have been synthesized and tested against a no. of enzymes that catalyze sequential bireactant reactions, the detailed theory for predicting the expected patterns of inhibition against the two substrates for various bireactant kinetic mechanisms has, heretofore, not been presented.  We have derived the rate equations for all likely sequential bireactant mechanisms and provide two examples in which bisubstrate inhibitors allow the kinetic mechanism to be detd.  Bisubstrate inhibitor kinetics is a powerful diagnostic for the detn. of kinetic mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrk_sv2txdK0LVg90H21EOLACvtfcHk0liAQqRUl8p7jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1SjtbfN&md5=6ddb7dcb30e7d97a3577a6088db49433</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fbi061621t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi061621t%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DM.%26aulast%3DMagalha%26aufirst%3DM.%2BL.%2BB.%26aulast%3DCook%26aufirst%3DP.%2BF.%26aulast%3DBlanchard%26aufirst%3DJ.%2BS.%26atitle%3DBisubstrate%2520Inhibition%253A%2520Theory%2520and%2520Application%2520to%2520N-Acetyltransferases%26jtitle%3DBiochemistry%26date%3D2006%26volume%3D45%26spage%3D14788%26epage%3D14794%26doi%3D10.1021%2Fbi061621t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Parang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span> <span> </span><span class="NLM_article-title">Designing Bisubstrate Analog Inhibitors for Protein Kinases</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">145</span>– <span class="NLM_lpage">157</span>, <span class="refDoi"> DOI: 10.1016/S0163-7258(02)00184-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1016%2FS0163-7258%2802%2900184-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=12191607" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BD38XksVShtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2002&pages=145-157&author=K.+Parangauthor=P.+A.+Cole&title=Designing+Bisubstrate+Analog+Inhibitors+for+Protein+Kinases&doi=10.1016%2FS0163-7258%2802%2900184-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Designing bisubstrate analog inhibitors for protein kinases</span></div><div class="casAuthors">Parang, Keykavous; Cole, Philip A.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">2-3</span>),
    <span class="NLM_cas:pages">145-157</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Inc.</span>)
        </div><div class="casAbstract">A review.  Protein kinases play crit. roles in signal transduction pathways by transmitting extracellular signals across the cell membrane to distant locations in the cytoplasm and the nucleus.  The development of protein kinase inhibitors has been hindered by the broad overlapping substrate specificities exhibited by these enzymes.  The design of bisubstrate analog inhibitors could provide for the enhancement of specificity and potency in protein kinase inhibition.  Bisubstrate analog inhibitors form a special group of protein kinase inhibitors that mimic two natural substrates/ligands and that simultaneously assoc. with two regions of given kinases.  Most bisubstrate analogs have been designed to mimic the phosphate donor (ATP) and the acceptor components (Ser-, Thr-, or Tyr-contg. peptides).  Recent studies have emphasized the importance of maintaining a specific distance between these two components to achieve potent inhibition.  In this review, we present a discussion of the methods for designing protein kinase inhibitors by mechanism-based approaches.  Emphasis is given to bivalent approaches, with an interpretation of what has been learned from more and less successful examples.  Future challenges in this area are also highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoM_dF2knydcbVg90H21EOLACvtfcHk0lhQCQMKh9dxmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XksVShtbk%253D&md5=95c8c3d60901a53ffc712d83c6cfba01</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2FS0163-7258%2802%2900184-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0163-7258%252802%252900184-5%26sid%3Dliteratum%253Aachs%26aulast%3DParang%26aufirst%3DK.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DDesigning%2520Bisubstrate%2520Analog%2520Inhibitors%2520for%2520Protein%2520Kinases%26jtitle%3DPharmacol.%2520Ther.%26date%3D2002%26volume%3D93%26spage%3D145%26epage%3D157%26doi%3D10.1016%2FS0163-7258%2802%2900184-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez-Ferrando, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span> <span> </span><span class="NLM_article-title">Enhanced Interrogation: Emerging Strategies for Cell Signaling Inhibition</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">646</span>– <span class="NLM_lpage">649</span>, <span class="refDoi"> DOI: 10.1038/nsmb0610-646</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1038%2Fnsmb0610-646" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2011&pages=646-649&author=R.+Huangauthor=I.+Martinez-Ferrandoauthor=P.+A.+Cole&title=Enhanced+Interrogation%3A+Emerging+Strategies+for+Cell+Signaling+Inhibition&doi=10.1038%2Fnsmb0610-646"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnsmb0610-646&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb0610-646%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DR.%26aulast%3DMartinez-Ferrando%26aufirst%3DI.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DEnhanced%2520Interrogation%253A%2520Emerging%2520Strategies%2520for%2520Cell%2520Signaling%2520Inhibition%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2011%26volume%3D17%26spage%3D646%26epage%3D649%26doi%3D10.1038%2Fnsmb0610-646" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noinaj, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Discovery of Bisubstrate Inhibitors for Protein N-Terminal Methyltransferase 1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">3773</span>– <span class="NLM_lpage">3779</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00206</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00206" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC1MXltFequ7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=3773-3779&author=D.+Chenauthor=G.+Dongauthor=N.+Noinajauthor=R.+Huang&title=Discovery+of+Bisubstrate+Inhibitors+for+Protein+N-Terminal+Methyltransferase+1&doi=10.1021%2Facs.jmedchem.9b00206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of bisubstrate inhibitors for protein N-terminal methyltransferase 1</span></div><div class="casAuthors">Chen, Dongxing; Dong, Guangping; Noinaj, Nicholas; Huang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3773-3779</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein N-terminal methyltransferase 1 (NTMT1) plays an important role in regulating mitosis and DNA repair.  Here, we describe the discovery of a potent NTMT1 bisubstrate inhibitor 4 (IC50 = 35 ± 2 nM) that exhibits greater than 100-fold selectivity against a panel of methyltransferases.  We also report the first crystal structure of NTMT1 in complex with an inhibitor, which revealed that 4 occupies substrate and cofactor binding sites of NTMT1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqM1ZcSF10eY7Vg90H21EOLACvtfcHk0lhQCQMKh9dxmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXltFequ7o%253D&md5=34782952c04423e217abf6cd3d228f94</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00206%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DD.%26aulast%3DDong%26aufirst%3DG.%26aulast%3DNoinaj%26aufirst%3DN.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DDiscovery%2520of%2520Bisubstrate%2520Inhibitors%2520for%2520Protein%2520N-Terminal%2520Methyltransferase%25201%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D3773%26epage%3D3779%26doi%3D10.1021%2Facs.jmedchem.9b00206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brandvold, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breen, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lachacz, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steffey, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soellner, M. B.</span></span> <span> </span><span class="NLM_article-title">Exquisitely Specific Bisubstrate Inhibitors of C-Src Kinase</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1387</span>– <span class="NLM_lpage">1391</span>, <span class="refDoi"> DOI: 10.1021/cb501048b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb501048b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC2MXkvFWjurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1387-1391&author=K.+R.+Brandvoldauthor=S.+M.+Santosauthor=M.+E.+Breenauthor=E.+J.+Lachaczauthor=M.+E.+Steffeyauthor=M.+B.+Soellner&title=Exquisitely+Specific+Bisubstrate+Inhibitors+of+C-Src+Kinase&doi=10.1021%2Fcb501048b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Exquisitely Specific Bisubstrate Inhibitors of c-Src Kinase</span></div><div class="casAuthors">Brandvold, Kristoffer R.; Santos, Shana M.; Breen, Meghan E.; Lachacz, Eric J.; Steffey, Michael E.; Soellner, Matthew B.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1387-1391</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have developed a modular approach to bisubstrate inhibition of protein kinases.  We apply our methodol. to c-Src and identify a highly selective bisubstrate inhibitor for this target.  Our approach has yielded the most selective c-Src inhibitor to date, and the methodol. to render the bisubstrate inhibitor cell-permeable provides a highly valuable tool for the study of c-Src signaling.  In addn., we have applied our bisubstrate inhibitor to develop a novel screening methodol. to identify non-ATP-competitive inhibitors of c-Src.  Using this methodol., we have discovered the most potent non-ATP-competitive inhibitor reported to date.  Our methodol. is designed to be general and could be applicable to addnl. kinases inhibited by the promiscuous ATP-competitive fragment used in our studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLbpY-YSwyA7Vg90H21EOLACvtfcHk0lhQCQMKh9dxmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXkvFWjurs%253D&md5=3d981ee3bfa8053f43de9ab4034c89b3</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fcb501048b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb501048b%26sid%3Dliteratum%253Aachs%26aulast%3DBrandvold%26aufirst%3DK.%2BR.%26aulast%3DSantos%26aufirst%3DS.%2BM.%26aulast%3DBreen%26aufirst%3DM.%2BE.%26aulast%3DLachacz%26aufirst%3DE.%2BJ.%26aulast%3DSteffey%26aufirst%3DM.%2BE.%26aulast%3DSoellner%26aufirst%3DM.%2BB.%26atitle%3DExquisitely%2520Specific%2520Bisubstrate%2520Inhibitors%2520of%2520C-Src%2520Kinase%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D1387%26epage%3D1391%26doi%3D10.1021%2Fcb501048b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, and Kinetic Analysis of Potent Protein N-Terminal Methyltransferase 1 Inhibitors</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">4149</span>– <span class="NLM_lpage">4154</span>, <span class="refDoi"> DOI: 10.1039/C5OB00120J</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1039%2FC5OB00120J" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=25712161" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjtVemtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2015&pages=4149-4154&author=G.+Zhangauthor=S.+L.+Richardsonauthor=Y.+Maoauthor=R.+Huang&title=Design%2C+Synthesis%2C+and+Kinetic+Analysis+of+Potent+Protein+N-Terminal+Methyltransferase+1+Inhibitors&doi=10.1039%2FC5OB00120J"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis, and kinetic analysis of potent protein N-terminal methyltransferase 1 inhibitors</span></div><div class="casAuthors">Zhang, Gang; Richardson, Stacie Lynn; Mao, Yunfei; Huang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4149-4154</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The protein N-terminal methyltransferase 1 (NTMT1) methylates the α-N-terminal amines of proteins.  NTMT1 is upregulated in a variety of cancers and knockdown of NTMT1 results in cell mitotic defects.  Therefore, NTMT1 inhibitors could be potential anticancer therapeutics.  This study describes the design and synthesis of the first inhibitor targeting NTMT1.  A novel bisubstrate analog (NAM-TZ-SPKRIA) was shown to be a potent inhibitor (Ki = 0.20 μM) for NTMT1 and was selective vs. protein lysine methyltransferase G9a and arginine methyltransferase 1.  NAM-TZ-SPKRIA was found to exhibit a competitive inhibition pattern for both substrates, and mass spectrometry expts. revealed that the inhibitor substantially suppressed the methylation progression.  Our results demonstrate the feasibility of using a triazole group to link an S-adenosyl-L-methionine (SAM) analog with a peptide substrate to construct bisubstrate analogs as NTMT1 potent and selective inhibitors.  This study lays a foundation to further discover small mol. NTMT1 inhibitors to interrogate its biol. functions, and suggests a general strategy for the development of selective protein methyltransferase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpChRkaQTjE77Vg90H21EOLACvtfcHk0lhhVozT_MxztA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjtVemtL4%253D&md5=b39327aa5e759ec16c3551367609c24f</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1039%2FC5OB00120J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5OB00120J%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DRichardson%26aufirst%3DS.%2BL.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520Kinetic%2520Analysis%2520of%2520Potent%2520Protein%2520N-Terminal%2520Methyltransferase%25201%2520Inhibitors%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2015%26volume%3D13%26spage%3D4149%26epage%3D4154%26doi%3D10.1039%2FC5OB00120J" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Facile Synthesis of SAM-Peptide Conjugates through Alkyl Linkers Targeting Protein N-Terminal Methyltransferase 1</span>. <i>RSC Adv.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">6768</span>– <span class="NLM_lpage">6771</span>, <span class="refDoi"> DOI: 10.1039/C5RA20625A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1039%2FC5RA20625A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=27588169" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC28Xlt1Krtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=6768-6771&author=G.+Zhangauthor=R.+Huang&title=Facile+Synthesis+of+SAM-Peptide+Conjugates+through+Alkyl+Linkers+Targeting+Protein+N-Terminal+Methyltransferase+1&doi=10.1039%2FC5RA20625A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Facile synthesis of SAM-peptide conjugates through alkyl linkers targeting protein N-terminal methyltransferase 1</span></div><div class="casAuthors">Zhang, Gang; Huang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">RSC Advances</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">6768-6771</span>CODEN:
                <span class="NLM_cas:coden">RSCACL</span>;
        ISSN:<span class="NLM_cas:issn">2046-2069</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The authors report the first chem. synthesis of SAM-peptide (SAM = S-adenosyl-L-methionine) conjugates through alkyl linkers to prep. bisubstrate analogs for protein methyltransferases.  The authors demonstrate its application by developing a series of bisubstrate inhibitors for protein N-terminal methyltransferase 1 (NTMT1) and the most potent one exhibits a Ki = 310 ± 55 nM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_MNtkSc3Ne7Vg90H21EOLACvtfcHk0lhhVozT_MxztA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xlt1Krtw%253D%253D&md5=e267ad97193784986c32b86c2e1ae88a</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1039%2FC5RA20625A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5RA20625A%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DFacile%2520Synthesis%2520of%2520SAM-Peptide%2520Conjugates%2520through%2520Alkyl%2520Linkers%2520Targeting%2520Protein%2520N-Terminal%2520Methyltransferase%25201%26jtitle%3DRSC%2520Adv.%26date%3D2016%26volume%3D6%26spage%3D6768%26epage%3D6771%26doi%3D10.1039%2FC5RA20625A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haren, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marechal, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Troffer-charlier, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cianciulli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sbardella, G.</span></span> <span> </span><span class="NLM_article-title">Transition State Mimics Are Valuable Mechanistic Probes for Structural Studies with the Arginine</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">E3652</span>– <span class="NLM_lpage">E3658</span>, <span class="refDoi"> DOI: 10.1073/pnas.1620470114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1073%2Fpnas.1620470114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=28416684" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=E3652-E3658&author=M.+J.+Harenauthor=+Vanauthor=N.+Marechalauthor=N.+Troffer-charlierauthor=A.+Cianciulliauthor=G.+Sbardella&title=Transition+State+Mimics+Are+Valuable+Mechanistic+Probes+for+Structural+Studies+with+the+Arginine&doi=10.1073%2Fpnas.1620470114"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1620470114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1620470114%26sid%3Dliteratum%253Aachs%26aulast%3DHaren%26aufirst%3DM.%2BJ.%26aulast%3DVan%26aulast%3DMarechal%26aufirst%3DN.%26aulast%3DTroffer-charlier%26aufirst%3DN.%26aulast%3DCianciulli%26aufirst%3DA.%26aulast%3DSbardella%26aufirst%3DG.%26atitle%3DTransition%2520State%2520Mimics%2520Are%2520Valuable%2520Mechanistic%2520Probes%2520for%2520Structural%2520Studies%2520with%2520the%2520Arginine%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2017%26volume%3D114%26spage%3DE3652%26epage%3DE3658%26doi%3D10.1073%2Fpnas.1620470114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Osborne, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roska, R. L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajski, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, P. R.</span></span> <span> </span><span class="NLM_article-title">In Situ Generation of a Bisubstrate Analogue for Protein Arginine Methyltransferase 1</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>130</i></span>,  <span class="NLM_fpage">4574</span>– <span class="NLM_lpage">4575</span>, <span class="refDoi"> DOI: 10.1021/ja077104v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja077104v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjtFygtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2008&pages=4574-4575&author=T.+Osborneauthor=R.+L.+W.+Roskaauthor=S.+R.+Rajskiauthor=P.+R.+Thompson&title=In+Situ+Generation+of+a+Bisubstrate+Analogue+for+Protein+Arginine+Methyltransferase+1&doi=10.1021%2Fja077104v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">In Situ Generation of a Bisubstrate Analogue for Protein Arginine Methyltransferase 1</span></div><div class="casAuthors">Osborne, Tanesha; Roska, Rachel L. Weller; Rajski, Scott R.; Thompson, Paul R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4574-4575</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferases (PRMTs) are (S)-adenosylmethionine (SAM)-dependent methyltransferases that catalyze the post-translational methylation of Arg residues in a variety of different proteins involved in transcriptional regulation and RNA splicing (e.g., histones H2A, H3, and H4).  Herein, we describe the use of an N-mustard, 5'-(diaminobutyric acid)-N-iodoethyl-5'-deoxyadenosine ammonium hydrochloride (AAI), to generate a bisubstrate analog inhibitor of PRMT1.  Using the approach outlined in this communication, it should be possible to generate bisubstrate analog-based inhibitors of PRMT isoenzymes that are potent and highly selective for a particular isoenzyme.  The fact that PRMT1 catalyzes AAI transfer is also significant because with appropriate modifications (e.g., functionalization with pendant azido or alkyne functionalities) this compd. could be used for proteomic applications to identify novel PRMT substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9Yqi1phgFBLVg90H21EOLACvtfcHk0lh0uvcm-gWQIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjtFygtb0%253D&md5=1433435a8942d1075a01cc39a6ffce99</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fja077104v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja077104v%26sid%3Dliteratum%253Aachs%26aulast%3DOsborne%26aufirst%3DT.%26aulast%3DRoska%26aufirst%3DR.%2BL.%2BW.%26aulast%3DRajski%26aufirst%3DS.%2BR.%26aulast%3DThompson%26aufirst%3DP.%2BR.%26atitle%3DIn%2520Situ%2520Generation%2520of%2520a%2520Bisubstrate%2520Analogue%2520for%2520Protein%2520Arginine%2520Methyltransferase%25201%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26spage%3D4574%26epage%3D4575%26doi%3D10.1021%2Fja077104v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Policarpo, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Decultot, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">May, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuzmic, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhakshinamoorthy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kannt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dittakavi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danarese, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaudet, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shair, M. D.</span></span> <span> </span><span class="NLM_article-title">High-Affinity Alkynyl Bisubstrate Inhibitors of Nicotinamide N-Methyltransferase (NNMT)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">9837</span>– <span class="NLM_lpage">9873</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01238</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01238" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFWltrzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=9837-9873&author=R.+L.+Policarpoauthor=L.+Decultotauthor=E.+Mayauthor=P.+Kuzmicauthor=S.+Carlsonauthor=D.+Huangauthor=V.+Chuauthor=B.+A.+Wrightauthor=S.+Dhakshinamoorthyauthor=A.+Kanntauthor=S.+Raniauthor=S.+Dittakaviauthor=J.+D.+Danareseauthor=R.+Gaudetauthor=M.+D.+Shair&title=High-Affinity+Alkynyl+Bisubstrate+Inhibitors+of+Nicotinamide+N-Methyltransferase+%28NNMT%29&doi=10.1021%2Facs.jmedchem.9b01238"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">High-Affinity Alkynyl Bisubstrate Inhibitors of Nicotinamide N-Methyltransferase (NNMT)</span></div><div class="casAuthors">Policarpo, Rocco L.; Decultot, Ludovic; May, Elizabeth; Kuzmic, Petr; Carlson, Samuel; Huang, Danny; Chu, Vincent; Wright, Brandon A.; Dhakshinamoorthy, Saravanakumar; Kannt, Aimo; Rani, Shilpa; Dittakavi, Sreekanth; Panarese, Joseph D.; Gaudet, Rachelle; Shair, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9837-9873</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) is a metabolic enzyme that methylates nicotinamide (NAM) using cofactor S-adenosylmethionine (SAM).  NNMT overexpression has been linked to diabetes, obesity, and various cancers.  In this work, structure-based rational design led to the development of potent and selective alkynyl bisubstrate inhibitors of NNMT.  The reported nicotinamide-SAM conjugate (named NS1) features an alkyne as a key design element that closely mimics the linear, 180° transition state geometry found in the NNMT-catalyzed SAM → NAM Me transfer reaction.  NS1 was synthesized in 14 steps and found to be a high-affinity, subnanomolar NNMT inhibitor.  An X-ray cocrystal structure and SAR study revealed the ability of an alkynyl linker to span the Me transfer tunnel of NNMT with ideal shape complementarity.  The compds. reported in this work represent the most potent and selective NNMT inhibitors reported to date.  The rational design principle described herein could potentially be extended to other methyltransferase enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLrzDcwfc0ibVg90H21EOLACvtfcHk0lh0uvcm-gWQIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFWltrzM&md5=f62a8055840292563d07cd25b0fcb4c2</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01238%26sid%3Dliteratum%253Aachs%26aulast%3DPolicarpo%26aufirst%3DR.%2BL.%26aulast%3DDecultot%26aufirst%3DL.%26aulast%3DMay%26aufirst%3DE.%26aulast%3DKuzmic%26aufirst%3DP.%26aulast%3DCarlson%26aufirst%3DS.%26aulast%3DHuang%26aufirst%3DD.%26aulast%3DChu%26aufirst%3DV.%26aulast%3DWright%26aufirst%3DB.%2BA.%26aulast%3DDhakshinamoorthy%26aufirst%3DS.%26aulast%3DKannt%26aufirst%3DA.%26aulast%3DRani%26aufirst%3DS.%26aulast%3DDittakavi%26aufirst%3DS.%26aulast%3DDanarese%26aufirst%3DJ.%2BD.%26aulast%3DGaudet%26aufirst%3DR.%26aulast%3DShair%26aufirst%3DM.%2BD.%26atitle%3DHigh-Affinity%2520Alkynyl%2520Bisubstrate%2520Inhibitors%2520of%2520Nicotinamide%2520N-Methyltransferase%2520%2528NNMT%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D9837%26epage%3D9873%26doi%3D10.1021%2Facs.jmedchem.9b01238" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loppnau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span> <span> </span><span class="NLM_article-title">Structural Basis for Substrate Recognition by the Human N-Terminal Methyltransferase 1</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">2343</span>– <span class="NLM_lpage">2348</span>, <span class="refDoi"> DOI: 10.1101/gad.270611.115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1101%2Fgad.270611.115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=26543161" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC28XntV2rtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=2343-2348&author=C.+Dongauthor=Y.+Maoauthor=W.+Tempelauthor=S.+Qinauthor=L.+Liauthor=P.+Loppnauauthor=R.+Huangauthor=J.+Min&title=Structural+Basis+for+Substrate+Recognition+by+the+Human+N-Terminal+Methyltransferase+1&doi=10.1101%2Fgad.270611.115"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for substrate recognition by the human N-terminal methyltransferase 1</span></div><div class="casAuthors">Dong, Cheng; Mao, Yunfei; Tempel, Wolfram; Qin, Su; Li, Li; Loppnau, Peter; Huang, Rong; Min, Jinrong</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">2343-2348</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">α-N-terminal methylation represents a highly conserved and prevalent post-translational modification, yet its biol. function has remained largely speculative.  The recent discovery of α-N-terminal methyltransferase 1 (NTMT1) and its physiol. substrates propels the elucidation of a general role of α-N-terminal methylation in mediating DNA-binding ability of the modified proteins.  The phenotypes, obsd. from both NTMT1 knockdown in breast cancer cell lines and knockout mouse models, suggest the potential involvement of α-N-terminal methylation in DNA damage response and cancer development.  In this study, we report the first crystal structures of human NTMT1 in complex with cofactor S-adenosyl-L-homocysteine (SAH) and six substrate peptides, resp., and reveal that NTMT1 contains two characteristic structural elements (a β hairpin and an N-terminal extension) that contribute to its substrate specificity.  Our complex structures, coupled with mutagenesis, binding, and enzymic studies, also present the key elements involved in locking the consensus substrate motif XPK (X indicates any residue type other than D/E) into the catalytic pocket for α-N-terminal methylation and explain why NTMT1 prefers an XPK sequence motif.  We propose a catalytic mechanism for α-N-terminal methylation.  Overall, this study gives us the first glimpse of the mol. mechanism of α-N-terminal methylation and potentially contributes to the advent of therapeutic agents for human diseases assocd. with deregulated α-N-terminal methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrv_R_NbrFwa7Vg90H21EOLACvtfcHk0lh0uvcm-gWQIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XntV2rtLc%253D&md5=3f9e7d4ad9c2abd5af04d615e71d1777</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1101%2Fgad.270611.115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.270611.115%26sid%3Dliteratum%253Aachs%26aulast%3DDong%26aufirst%3DC.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DQin%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLoppnau%26aufirst%3DP.%26aulast%3DHuang%26aufirst%3DR.%26aulast%3DMin%26aufirst%3DJ.%26atitle%3DStructural%2520Basis%2520for%2520Substrate%2520Recognition%2520by%2520the%2520Human%2520N-Terminal%2520Methyltransferase%25201%26jtitle%3DGenes%2520Dev.%26date%3D2015%26volume%3D29%26spage%3D2343%26epage%3D2348%26doi%3D10.1101%2Fgad.270611.115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span> <span> </span><span class="NLM_article-title">Molecular Basis for Histone N-Terminal Methylation by NRMT1</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">2337</span>– <span class="NLM_lpage">2342</span>, <span class="refDoi"> DOI: 10.1101/gad.270926.115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1101%2Fgad.270926.115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=26543159" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1yltb7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=2337-2342&author=R.+Wuauthor=Y.+Yueauthor=X.+Zhengauthor=H.+Li&title=Molecular+Basis+for+Histone+N-Terminal+Methylation+by+NRMT1&doi=10.1101%2Fgad.270926.115"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular basis for histone N-terminal methylation by NRMT1</span></div><div class="casAuthors">Wu, Ruoxi; Yue, Yuan; Zheng, Xiangdong; Li, Haitao</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">2337-2342</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">NRMT1 is an N-terminal methyltransferase that methylates histone CENP-A as well as nonhistone substrates.  Here, we report the crystal structure of human NRMT1 bound to CENP-A peptide at 1.3 Å.  NRMT1 adopts a core methyltransferase fold that resembles DOT1L and PRMT but not SET domain family histone methyltransferases.  Key substrate recognition and catalytic residues were identified by mutagenesis studies.  Histone peptide profiling revealed that human NRMT1 is highly selective to human CENP-A and fruit fly H2B, which share a common "Xaa-Pro-Lys/Arg" motif.  These results, along with a 1.5 Å costructure of human NRMT1 bound to the fruit fly H2B peptide, underscore the importance of the NRMT1 recognition motif.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGIuplVjQhULVg90H21EOLACvtfcHk0ljdNQWhwhAsnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1yltb7J&md5=f3601b0966da6c89d0acf9aeae4799be</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1101%2Fgad.270926.115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.270926.115%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DR.%26aulast%3DYue%26aufirst%3DY.%26aulast%3DZheng%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DH.%26atitle%3DMolecular%2520Basis%2520for%2520Histone%2520N-Terminal%2520Methylation%2520by%2520NRMT1%26jtitle%3DGenes%2520Dev.%26date%3D2015%26volume%3D29%26spage%3D2337%26epage%3D2342%26doi%3D10.1101%2Fgad.270926.115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petkowski, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaner Tooley, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shumilin, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balsbaugh, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macara, I. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shabanowitz, J.</span></span> <span> </span><span class="NLM_article-title">Substrate Specificity of Mammalian N-Terminal α -Amino Methyltransferase NRMT</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">5942</span>– <span class="NLM_lpage">5950</span>, <span class="refDoi"> DOI: 10.1021/bi300278f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi300278f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC38XpvVars78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=5942-5950&author=J.+J.+Petkowskiauthor=C.+E.+Schaner+Tooleyauthor=L.+C.+Andersonauthor=I.+A.+Shumilinauthor=J.+L.+Balsbaughauthor=D.+F.+Huntauthor=W.+Minorauthor=I.+G.+Macaraauthor=J.+Shabanowitz&title=Substrate+Specificity+of+Mammalian+N-Terminal+%CE%B1+-Amino+Methyltransferase+NRMT&doi=10.1021%2Fbi300278f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Substrate Specificity of Mammalian N-Terminal α-Amino Methyltransferase NRMT</span></div><div class="casAuthors">Petkowski, Janusz J.; Schaner Tooley, Christine E.; Anderson, Lissa C.; Shumilin, Igor A.; Balsbaugh, Jeremy L.; Shabanowitz, Jeffrey; Hunt, Donald F.; Minor, Wladek; Macara, Ian G.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">5942-5950</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">N-Terminal methylation of free α-amino groups is a post-translational modification of proteins that was first described 30 years ago but has been studied very little.  In this modification, the initiating M residue is cleaved and the exposed α-amino group is mono-, di-, or trimethylated by NRMT, a recently identified N-terminal methyltransferase.  Currently, all known eukaryotic α-amino-methylated proteins have a unique N-terminal motif, M-X-P-K, where X is A, P, or S.  NRMT can also methylate artificial substrates in vitro in which X is G, F, Y, C, M, K, R, N, Q, or H.  Methylation efficiencies of N-terminal amino acids are variable with respect to the identity of X.  Here we use in vitro peptide methylation assays and substrate immunopptns. to show that the canonical M-X-P-K methylation motif is not the only one recognized by NRMT.  We predict that N-terminal methylation is a widespread post-translational modification and that there is interplay between N-terminal acetylation and N-terminal methylation.  We also use isothermal calorimetry expts. to demonstrate that NRMT can efficiently recognize and bind to its fully methylated products.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoR8dl8ByIopbVg90H21EOLACvtfcHk0ljdNQWhwhAsnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpvVars78%253D&md5=dbddebac4bc6f6045fd144604e157bdb</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fbi300278f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi300278f%26sid%3Dliteratum%253Aachs%26aulast%3DPetkowski%26aufirst%3DJ.%2BJ.%26aulast%3DSchaner%2BTooley%26aufirst%3DC.%2BE.%26aulast%3DAnderson%26aufirst%3DL.%2BC.%26aulast%3DShumilin%26aufirst%3DI.%2BA.%26aulast%3DBalsbaugh%26aufirst%3DJ.%2BL.%26aulast%3DHunt%26aufirst%3DD.%2BF.%26aulast%3DMinor%26aufirst%3DW.%26aulast%3DMacara%26aufirst%3DI.%2BG.%26aulast%3DShabanowitz%26aufirst%3DJ.%26atitle%3DSubstrate%2520Specificity%2520of%2520Mammalian%2520N-Terminal%2520%25CE%25B1%2520-Amino%2520Methyltransferase%2520NRMT%26jtitle%3DBiochemistry%26date%3D2012%26volume%3D51%26spage%3D5942%26epage%3D5950%26doi%3D10.1021%2Fbi300278f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dorgan, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wooderchak, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wynn, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karschner, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alfaro, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Z. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hevel, J. M.</span></span> <span> </span><span class="NLM_article-title">An Enzyme-Coupled Continuous Spectrophotometric Assay for S-Adenosylmethionine-Dependent Methyltransferases</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>350</i></span>,  <span class="NLM_fpage">249</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1016/j.ab.2006.01.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1016%2Fj.ab.2006.01.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=16460659" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BD28XitVeht7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=350&publication_year=2006&pages=249-255&author=K.+M.+Dorganauthor=W.+L.+Wooderchakauthor=D.+P.+Wynnauthor=E.+L.+Karschnerauthor=J.+F.+Alfaroauthor=Y.+Cuiauthor=Z.+S.+Zhouauthor=J.+M.+Hevel&title=An+Enzyme-Coupled+Continuous+Spectrophotometric+Assay+for+S-Adenosylmethionine-Dependent+Methyltransferases&doi=10.1016%2Fj.ab.2006.01.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">An enzyme-coupled continuous spectrophotometric assay for S-adenosylmethionine-dependent methyltransferases</span></div><div class="casAuthors">Dorgan, Kathleen M.; Wooderchak, Whitney L.; Wynn, DonRaphael P.; Karschner, Erin L.; Alfaro, Joshua F.; Cui, Yinqiu; Zhou, Zhaohui Sunny; Hevel, Joan M.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">350</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">249-255</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Modification of small mols. and proteins by methyltransferases affects a wide range of biol. processes.  Here, we report an enzyme-coupled continuous spectrophotometric assay to quant. characterize S-adenosyl-L-methionine (AdoMet/SAM)-dependent methyltransferase activity.  In this assay, S-adenosyl-L-homocysteine (AdoHcy/SAH), the transmethylation product of AdoMet-dependent methyltransferases, is hydrolyzed to S-ribosylhomocysteine and adenine by recombinant S-adenosylhomocysteine/5'-methylthioadenosine nucleosidase (SAHN/MTAN, EC 3.2.2.9).  Subsequently, adenine generated from AdoHcy is further hydrolyzed to hypoxanthine and ammonia by recombinant adenine deaminase (EC 3.5.4.2).  This deamination is assocd. with a decrease in absorbance at 265 nm that can be monitored continuously.  Coupling enzymes are recombinant and easily purified.  The utility of this assay was shown using recombinant rat protein arginine N-methyltransferase 1 (PRMT1, EC 2.1.1.125), which catalyzes the mono- and dimethylation of guanidino nitrogens of arginine residues in select proteins.  Using this assay, the kinetic parameters of PRMT1 with three synthetic peptides were detd.  An advantage of this assay is the destruction of AdoHcy by AdoHcy nucleosidase, which alleviates AdoHcy product feedback inhibition of S-adenosylmethionine-dependent methyltransferases.  Finally, this method may be used to assay other enzymes that produce AdoHcy, 5'-methylthioadenosine, or compds. that can be cleaved by AdoHcy nucleosidase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkcuo70A9XQbVg90H21EOLACvtfcHk0ljdNQWhwhAsnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XitVeht7s%253D&md5=f83416f4f10bbe745b7a0873f9950a38</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2006.01.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2006.01.004%26sid%3Dliteratum%253Aachs%26aulast%3DDorgan%26aufirst%3DK.%2BM.%26aulast%3DWooderchak%26aufirst%3DW.%2BL.%26aulast%3DWynn%26aufirst%3DD.%2BP.%26aulast%3DKarschner%26aufirst%3DE.%2BL.%26aulast%3DAlfaro%26aufirst%3DJ.%2BF.%26aulast%3DCui%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DZ.%2BS.%26aulast%3DHevel%26aufirst%3DJ.%2BM.%26atitle%3DAn%2520Enzyme-Coupled%2520Continuous%2520Spectrophotometric%2520Assay%2520for%2520S-Adenosylmethionine-Dependent%2520Methyltransferases%26jtitle%3DAnal.%2520Biochem.%26date%3D2006%26volume%3D350%26spage%3D249%26epage%3D255%26doi%3D10.1016%2Fj.ab.2006.01.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, A. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panchenko, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sathyan, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petkowski, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pai, P.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russell, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macara, I. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shabanowitz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Black, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flotz, D. R.</span></span> <span> </span><span class="NLM_article-title">Posttranslational Modification of CENP-A Influences the Conformation of Centromeric Chromatin</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">11827</span>– <span class="NLM_lpage">11832</span>, <span class="refDoi"> DOI: 10.1073/pnas.1300325110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1073%2Fpnas.1300325110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=23818633" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1emurvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=11827-11832&author=A.+O.+Baileyauthor=T.+Panchenkoauthor=K.+M.+Sathyanauthor=J.+J.+Petkowskiauthor=P.-J.+Paiauthor=D.+L.+Baiauthor=D.+H.+Russellauthor=I.+G.+Macaraauthor=J.+Shabanowitzauthor=D.+F.+Huntauthor=B.+E.+Blackauthor=D.+R.+Flotz&title=Posttranslational+Modification+of+CENP-A+Influences+the+Conformation+of+Centromeric+Chromatin&doi=10.1073%2Fpnas.1300325110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Posttranslational modification of CENP-A influences the conformation of centromeric chromatin</span></div><div class="casAuthors">Bailey, Aaron O.; Panchenko, Tanya; Sathyan, Kizhakke M.; Petkowski, Janusz J.; Pai, Pei-Jing; Bai, Dina L.; Russell, David H.; Macara, Ian G.; Shabanowitz, Jeffrey; Hunt, Donald F.; Black, Ben E.; Foltz, Daniel R.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">11827-11832,S11827/1-S11827/14</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Centromeres are chromosomal loci required for accurate segregation of sister chromatids during mitosis.  The location of the centromere on the chromosome is not dependent on DNA sequence, but rather it is epigenetically specified by the histone H3 variant centromere protein A (CENP-A).  The N-terminal tail of CENP-A is highly divergent from other H3 variants.  Canonical histone N termini are hotspots of conserved post-translational modification; however, no broadly conserved modifications of the vertebrate CENP-A tail have been previously obsd.  Here, we report three post-translational modifications on human CENP-A N termini using high-resoln. MS: trimethylation of Gly1 and phosphorylation of Ser16 and Ser18.  Our results demonstrate that CENP-A is subjected to constitutive initiating methionine removal, similar to other H3 variants.  The nascent N-terminal residue Gly1 becomes trimethylated on the α-amino group.  We demonstrate that the N-terminal RCC1 methyltransferase is capable of modifying the CENP-A N terminus.  Methylation occurs in the prenucleosomal form and marks the majority of CENP-A nucleosomes.  Serine 16 and 18 become phosphorylated in prenucleosomal CENP-A and are phosphorylated on asynchronous and mitotic nucleosomal CENP-A and are important for chromosome segregation during mitosis.  The double phosphorylation motif forms a salt-bridged secondary structure and causes CENP-A N-terminal tails to form intramol. assocns.  Anal. ultracentrifugation of phospho-mimetic CENP-A nucleosome arrays demonstrates that phosphorylation results in greater intranucleosome assocns. and counteracts the hyperoligomerized state exhibited by unmodified CENP-A nucleosome arrays.  Our studies have revealed that the major modifications on the N-terminal tail of CENP-A alter the phys. properties of the chromatin fiber at the centromere.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7Tb1icoBc4LVg90H21EOLACvtfcHk0lhmqyF2zfWugQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1emurvE&md5=246e391333b51fa7ba0e0a2d80498c03</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1300325110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1300325110%26sid%3Dliteratum%253Aachs%26aulast%3DBailey%26aufirst%3DA.%2BO.%26aulast%3DPanchenko%26aufirst%3DT.%26aulast%3DSathyan%26aufirst%3DK.%2BM.%26aulast%3DPetkowski%26aufirst%3DJ.%2BJ.%26aulast%3DPai%26aufirst%3DP.-J.%26aulast%3DBai%26aufirst%3DD.%2BL.%26aulast%3DRussell%26aufirst%3DD.%2BH.%26aulast%3DMacara%26aufirst%3DI.%2BG.%26aulast%3DShabanowitz%26aufirst%3DJ.%26aulast%3DHunt%26aufirst%3DD.%2BF.%26aulast%3DBlack%26aufirst%3DB.%2BE.%26aulast%3DFlotz%26aufirst%3DD.%2BR.%26atitle%3DPosttranslational%2520Modification%2520of%2520CENP-A%2520Influences%2520the%2520Conformation%2520of%2520Centromeric%2520Chromatin%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2013%26volume%3D110%26spage%3D11827%26epage%3D11832%26doi%3D10.1073%2Fpnas.1300325110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otake, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masumoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span> <span> </span><span class="NLM_article-title">Identification of Novel α-N-Methylation of CENP-B That Regulates Its Binding to the Centromeric DNA</span>. <i>J. Proteome Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">4167</span>– <span class="NLM_lpage">4175</span>, <span class="refDoi"> DOI: 10.1021/pr400498y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/pr400498y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Wqt7zK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=4167-4175&author=X.+Daiauthor=K.+Otakeauthor=C.+Youauthor=Q.+Caiauthor=Z.+Wangauthor=H.+Masumotoauthor=Y.+Wang&title=Identification+of+Novel+%CE%B1-N-Methylation+of+CENP-B+That+Regulates+Its+Binding+to+the+Centromeric+DNA&doi=10.1021%2Fpr400498y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of novel α-N-methylation of CENP-B that regulates its binding to the centromeric DNA</span></div><div class="casAuthors">Dai, Xiaoxia; Otake, Koichiro; You, Changjun; Cai, Qian; Wang, Zi; Masumoto, Hiroshi; Wang, Yinsheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4167-4175</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The eukaryotic centromere is an essential chromatin region required for accurate segregation of sister chromatids during cell division.  Centromere protein B (CENP-B) is a highly conserved protein which can bind to the 17-bp CENP-B box on the centromeric DNA.  In this study, the authors found that CENP-B could be α-N-methylated in human cells.  The level of the α-N-methylation was stimulated in cells in response to a variety of extracellular stimuli, including increased cell d., heat shock, and arsenite treatment, although the methylation level was not altered upon metaphase arrest.  N-terminal RCC1 methyltransferase (NRMT) was identified as a major enzyme required for the CENP-B methylation.  Addnl., chromatin-bound CENP-B was primarily trimethylated and α-N-trimethylation could enhance CENP-B's binding to CENP-B box in cells.  These study also expands the function of protein α-N-methylation that has been known for decades and whose function remains largely unexplored.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCWj3AJ08MjLVg90H21EOLACvtfcHk0lhmqyF2zfWugQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1Wqt7zK&md5=f0a6400d0765d888861badd1aafe7c73</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fpr400498y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fpr400498y%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DX.%26aulast%3DOtake%26aufirst%3DK.%26aulast%3DYou%26aufirst%3DC.%26aulast%3DCai%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DMasumoto%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DY.%26atitle%3DIdentification%2520of%2520Novel%2520%25CE%25B1-N-Methylation%2520of%2520CENP-B%2520That%2520Regulates%2520Its%2520Binding%2520to%2520the%2520Centromeric%2520DNA%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2013%26volume%3D12%26spage%3D4167%26epage%3D4175%26doi%3D10.1021%2Fpr400498y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cai, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span> <span> </span><span class="NLM_article-title">α-N-Methylation of Damaged DNA-Binding Protein 2 (DDB2) and Its Function in Nucleotide Excision Repair</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>289</i></span>,  <span class="NLM_fpage">16046</span>– <span class="NLM_lpage">16056</span>, <span class="refDoi"> DOI: 10.1074/jbc.M114.558510</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1074%2Fjbc.M114.558510" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=24753253" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC2cXptl2qsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=2014&pages=16046-16056&author=Q.+Caiauthor=L.+Fuauthor=Z.+Wangauthor=N.+Ganauthor=X.+Daiauthor=Y.+Wang&title=%CE%B1-N-Methylation+of+Damaged+DNA-Binding+Protein+2+%28DDB2%29+and+Its+Function+in+Nucleotide+Excision+Repair&doi=10.1074%2Fjbc.M114.558510"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">α-N-Methylation of damaged DNA-binding protein 2 (DDB2) and its function in nucleotide excision repair</span></div><div class="casAuthors">Cai, Qian; Fu, Lijuan; Wang, Zi; Gan, Nanqin; Dai, Xiaoxia; Wang, Yinsheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">289</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">16046-16056</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">DDB2 exhibits a high affinity toward UV-damaged DNA, and it is involved in the initial steps of global genome nucleotide excision repair.  Mutations in the DDB2 gene cause the genetic complementation group E of xeroderma pigmentosum, an autosomal recessive disease manifested clin. by hypersensitivity to sunlight exposure and an increased predisposition to skin cancer.  Here the authors found that, in human cells, the initiating methionine residue in DDB2 was removed and that the N-terminal alanine could be methylated on its α-amino group in human cells, with trimethylation being the major form.  The α-N-methylation of DDB2 is catalyzed by the N-terminal RCC1 methyltransferase.  In addn., a methylation-defective mutant of DDB2 displayed diminished nuclear localization and was recruited at a reduced efficiency to UV-induced cyclobutane pyrimidine dimer foci.  Moreover, loss of this methylation conferred compromised ATM (ataxia telangiectasia mutated) activation, decreased efficiency in cyclobutane pyrimidine dimer repair, and elevated sensitivity of cells toward UV light exposure.  This study provides new knowledge about the posttranslational regulation of DDB2 and expands the biol. functions of protein α-N-methylation to DNA repair.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpc3yE8db8RO7Vg90H21EOLACvtfcHk0lhmqyF2zfWugQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXptl2qsLs%253D&md5=e52a0c303e3420df4722d42cb0478af1</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M114.558510&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M114.558510%26sid%3Dliteratum%253Aachs%26aulast%3DCai%26aufirst%3DQ.%26aulast%3DFu%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DGan%26aufirst%3DN.%26aulast%3DDai%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DY.%26atitle%3D%25CE%25B1-N-Methylation%2520of%2520Damaged%2520DNA-Binding%2520Protein%25202%2520%2528DDB2%2529%2520and%2520Its%2520Function%2520in%2520Nucleotide%2520Excision%2520Repair%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2014%26volume%3D289%26spage%3D16046%26epage%3D16056%26doi%3D10.1074%2Fjbc.M114.558510" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rulten, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldecott, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span> <span> </span><span class="NLM_article-title">Identification and Functional Characterizations of N-Terminal α-N-Methylation and Phosphorylation of Serine 461 in Human Poly(ADP-Ribose) Polymerase 3</span>. <i>J. Proteome Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">2575</span>– <span class="NLM_lpage">2582</span>, <span class="refDoi"> DOI: 10.1021/acs.jproteome.5b00126</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jproteome.5b00126" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC2MXms1OgsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=2575-2582&author=X.+Daiauthor=S.+L.+Rultenauthor=C.+Youauthor=K.+W.+Caldecottauthor=Y.+Wang&title=Identification+and+Functional+Characterizations+of+N-Terminal+%CE%B1-N-Methylation+and+Phosphorylation+of+Serine+461+in+Human+Poly%28ADP-Ribose%29+Polymerase+3&doi=10.1021%2Facs.jproteome.5b00126"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Functional Characterizations of N-Terminal α-N-Methylation and Phosphorylation of Serine 461 in Human Poly(ADP-ribose) Polymerase 3</span></div><div class="casAuthors">Dai, Xiaoxia; Rulten, Stuart L.; You, Changjun; Caldecott, Keith W.; Wang, Yinsheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2575-2582</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Poly(ADP-ribose) polymerase 3 (PARP3) is a member of the PARP family enzymes which catalyze the ADP-ribosylation of proteins.  PARP3 plays an important role in DNA damage repair and mitotic progression.  In this study, we identified, using mass spectrometric (MS) techniques, two novel post-translational modification sites in PARP3, α-N-methylation and phosphorylation of serine 461 (S461).  We found that the N-terminal α-amino group of PARP3 is heavily methylated in human cells, and N-terminal RCC1 methyltransferase (NRMT) is a key enzyme required for this methylation.  We also obsd. that the phosphorylation level of S461 in PARP3 could be reduced in human cells upon treatment with flavopiridol, a cyclin-dependent kinase inhibitor.  Moreover, we demonstrated that S461 phosphorylation, but not α-N-methylation of PARP3, may be involved in the cellular response toward DNA double-strand breaks (DSBs).  These findings provide novel insights into the post-translational regulation of PARP3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrr0mFKyXPLnrVg90H21EOLACvtfcHk0lgu32B7XOBjEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXms1OgsLw%253D&md5=6ff05a312e38b2105b71fbbd17388e64</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.jproteome.5b00126&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jproteome.5b00126%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DX.%26aulast%3DRulten%26aufirst%3DS.%2BL.%26aulast%3DYou%26aufirst%3DC.%26aulast%3DCaldecott%26aufirst%3DK.%2BW.%26aulast%3DWang%26aufirst%3DY.%26atitle%3DIdentification%2520and%2520Functional%2520Characterizations%2520of%2520N-Terminal%2520%25CE%25B1-N-Methylation%2520and%2520Phosphorylation%2520of%2520Serine%2520461%2520in%2520Human%2520Poly%2528ADP-Ribose%2529%2520Polymerase%25203%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2015%26volume%3D14%26spage%3D2575%26epage%3D2582%26doi%3D10.1021%2Facs.jproteome.5b00126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tooley, C. E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petkowski, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muratore-Schroeder, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balsbaugh, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shabanowitz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabat, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macara, I. G.</span></span> <span> </span><span class="NLM_article-title">NRMT Is an Alpha-N-Methyltransferase That Methylates RCC1 and Retinoblastoma Protein</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>466</i></span>,  <span class="NLM_fpage">1125</span>– <span class="NLM_lpage">1128</span>, <span class="refDoi"> DOI: 10.1038/nature09343</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1038%2Fnature09343" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=20668449" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A280%3ADC%252BC3cfhtVensQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=466&publication_year=2010&pages=1125-1128&author=C.+E.+S.+Tooleyauthor=J.+J.+Petkowskiauthor=T.+L.+Muratore-Schroederauthor=J.+L.+Balsbaughauthor=J.+Shabanowitzauthor=M.+Sabatauthor=W.+Minorauthor=D.+F.+Huntauthor=I.+G.+Macara&title=NRMT+Is+an+Alpha-N-Methyltransferase+That+Methylates+RCC1+and+Retinoblastoma+Protein&doi=10.1038%2Fnature09343"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">NRMT is an alpha-N-methyltransferase that methylates RCC1 and retinoblastoma protein</span></div><div class="casAuthors">Tooley Christine E Schaner; Petkowski Janusz J; Muratore-Schroeder Tara L; Balsbaugh Jeremy L; Shabanowitz Jeffrey; Sabat Michal; Minor Wladek; Hunt Donald F; Macara Ian G</div><div class="citationInfo"><span class="NLM_cas:title">Nature</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">466</span>
        (<span class="NLM_cas:issue">7310</span>),
    <span class="NLM_cas:pages">1125-8</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The post-translational methylation of alpha-amino groups was first discovered over 30 years ago on the bacterial ribosomal proteins L16 and L33 (refs 1, 2), but almost nothing is known about the function or enzymology of this modification.  Several other bacterial and eukaryotic proteins have since been shown to be alpha-N-methylated.  However, the Ran guanine nucleotide-exchange factor, RCC1, is the only protein for which any biological function of alpha-N-methylation has been identified.  Methylation-defective mutants of RCC1 have reduced affinity for DNA and cause mitotic defects, but further characterization of this modification has been hindered by ignorance of the responsible methyltransferase.  All fungal and animal N-terminally methylated proteins contain a unique N-terminal motif, Met-(Ala/Pro/Ser)-Pro-Lys, indicating that they may be targets of the same, unknown enzyme.  The initiating Met is cleaved, and the exposed alpha-amino group is mono-, di- or trimethylated.  Here we report the discovery of the first alpha-N-methyltransferase, which we named N-terminal RCC1 methyltransferase (NRMT).  Substrate docking and mutational analysis of RCC1 defined the NRMT recognition sequence and enabled the identification of numerous new methylation targets, including SET (also known as TAF-I or PHAPII) and the retinoblastoma protein, RB.  Knockdown of NRMT recapitulates the multi-spindle phenotype seen with methylation-defective RCC1 mutants, demonstrating the importance of alpha-N-methylation for normal bipolar spindle formation and chromosome segregation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRzswcolxsHgOxsqfvU0ziafW6udTcc2eaXw_iZXUhK0bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cfhtVensQ%253D%253D&md5=e5fe130b420fc030b5af04ef304a9024</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnature09343&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09343%26sid%3Dliteratum%253Aachs%26aulast%3DTooley%26aufirst%3DC.%2BE.%2BS.%26aulast%3DPetkowski%26aufirst%3DJ.%2BJ.%26aulast%3DMuratore-Schroeder%26aufirst%3DT.%2BL.%26aulast%3DBalsbaugh%26aufirst%3DJ.%2BL.%26aulast%3DShabanowitz%26aufirst%3DJ.%26aulast%3DSabat%26aufirst%3DM.%26aulast%3DMinor%26aufirst%3DW.%26aulast%3DHunt%26aufirst%3DD.%2BF.%26aulast%3DMacara%26aufirst%3DI.%2BG.%26atitle%3DNRMT%2520Is%2520an%2520Alpha-N-Methyltransferase%2520That%2520Methylates%2520RCC1%2520and%2520Retinoblastoma%2520Protein%26jtitle%3DNature%26date%3D2010%26volume%3D466%26spage%3D1125%26epage%3D1128%26doi%3D10.1038%2Fnature09343" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonsignore, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tooley, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Hoose, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaner
Tooley, C. E.</span></span> <span> </span><span class="NLM_article-title">NRMT1 Knockout Mice Exhibit Phenotypes Associated with Impaired DNA Repair and Premature Aging</span>. <i>Mech. Ageing Dev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>146–148</i></span>,  <span class="NLM_fpage">42</span>– <span class="NLM_lpage">52</span>, <span class="refDoi"> DOI: 10.1016/j.mad.2015.03.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1016%2Fj.mad.2015.03.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=25843235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvVSmu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=146%E2%80%93148&publication_year=2015&pages=42-52&author=L.+A.+Bonsignoreauthor=J.+G.+Tooleyauthor=P.+M.+Van+Hooseauthor=E.+Wangauthor=A.+Chengauthor=M.+P.+Coleauthor=C.+E.+Schaner%0ATooley&title=NRMT1+Knockout+Mice+Exhibit+Phenotypes+Associated+with+Impaired+DNA+Repair+and+Premature+Aging&doi=10.1016%2Fj.mad.2015.03.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">NRMT1 knockout mice exhibit phenotypes associated with impaired DNA repair and premature aging</span></div><div class="casAuthors">Bonsignore, Lindsay A.; Tooley, John G.; Van Hoose, Patrick M.; Wang, Eugenia; Cheng, Alan; Cole, Marsha P.; Schaner Tooley, Christine E.</div><div class="citationInfo"><span class="NLM_cas:title">Mechanisms of Ageing and Development</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">146-148</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">42-52</span>CODEN:
                <span class="NLM_cas:coden">MAGDA3</span>;
        ISSN:<span class="NLM_cas:issn">0047-6374</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">Though defective genome maintenance and DNA repair have long been known to promote phenotypes of premature aging, the role protein methylation plays in these processes is only now emerging.  We have recently identified the first N-terminal methyltransferase, NRMT1, which regulates protein-DNA interactions and is necessary for both accurate mitotic division and nucleotide excision repair.  To demonstrate if complete loss of NRMT1 subsequently resulted in developmental or aging phenotypes, we constructed the first NRMT1 knockout (Nrmt1-/-) mouse.  The majority of these mice die shortly after birth.  However, the ones that survive, exhibit decreased body size, female-specific infertility, kyphosis, decreased mitochondrial function, and early-onset liver degeneration; phenotypes characteristic of other mouse models deficient in DNA repair.  The livers from Nrmt1-/- mice produce less reactive oxygen species (ROS) than wild type controls, and Nrmt1-/- mouse embryonic fibroblasts show a decreased capacity for handling oxidative damage.  This indicates that decreased mitochondrial function may benefit Nrmt1-/- mice and protect them from excess internal ROS and subsequent DNA damage.  These studies position the NRMT1 knockout mouse as a useful new system for studying the effects of genomic instability and defective DNA damage repair on organismal and tissue-specific aging.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWx010KT-fbrVg90H21EOLACvtfcHk0lgThtYyw_UghQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvVSmu7Y%253D&md5=501bd0e3dade5777c3965efb8fe10b93</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.mad.2015.03.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mad.2015.03.012%26sid%3Dliteratum%253Aachs%26aulast%3DBonsignore%26aufirst%3DL.%2BA.%26aulast%3DTooley%26aufirst%3DJ.%2BG.%26aulast%3DVan%2BHoose%26aufirst%3DP.%2BM.%26aulast%3DWang%26aufirst%3DE.%26aulast%3DCheng%26aufirst%3DA.%26aulast%3DCole%26aufirst%3DM.%2BP.%26aulast%3DSchaner%2BTooley%26aufirst%3DC.%2BE.%26atitle%3DNRMT1%2520Knockout%2520Mice%2520Exhibit%2520Phenotypes%2520Associated%2520with%2520Impaired%2520DNA%2520Repair%2520and%2520Premature%2520Aging%26jtitle%3DMech.%2520Ageing%2520Dev.%26date%3D2015%26volume%3D146%25E2%2580%2593148%26spage%3D42%26epage%3D52%26doi%3D10.1016%2Fj.mad.2015.03.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nitrone, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keirs, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moffat, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overton, K.</span></span> <span> </span><span class="NLM_article-title">Enantioselective synthesis of optically pure (R)- and (S)-β-lysine via nitrone cycloaddition</span>. <i>J. Chem. Soc., Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_fpage">654</span>– <span class="NLM_lpage">655</span>, <span class="refDoi"> DOI: 10.1039/C39880000654</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1039%2FC39880000654" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1988&pages=654-655&author=S.+Nitroneauthor=D.+Keirsauthor=D.+Moffatauthor=K.+Overton&title=Enantioselective+synthesis+of+optically+pure+%28R%29-+and+%28S%29-%CE%B2-lysine+via+nitrone+cycloaddition&doi=10.1039%2FC39880000654"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1039%2FC39880000654&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC39880000654%26sid%3Dliteratum%253Aachs%26aulast%3DNitrone%26aufirst%3DS.%26aulast%3DKeirs%26aufirst%3DD.%26aulast%3DMoffat%26aufirst%3DD.%26aulast%3DOverton%26aufirst%3DK.%26atitle%3DEnantioselective%2520synthesis%2520of%2520optically%2520pure%2520%2528R%2529-%2520and%2520%2528S%2529-%25CE%25B2-lysine%2520via%2520nitrone%2520cycloaddition%26jtitle%3DJ.%2520Chem.%2520Soc.%252C%2520Chem.%2520Commun.%26date%3D1988%26spage%3D654%26epage%3D655%26doi%3D10.1039%2FC39880000654" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nasopoulou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georgiadis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matziari, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dive, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cedex, G. Y. V.</span></span> <span> </span><span class="NLM_article-title">A Versatile Annulation Protocol toward Novel Constrained Phosphinic Peptidomimetics</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">7222</span>– <span class="NLM_lpage">7228</span>, <span class="refDoi"> DOI: 10.1021/jo071081l</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo071081l" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpsVeltLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2007&pages=7222-7228&author=M.+Nasopoulouauthor=D.+Georgiadisauthor=M.+Matziariauthor=V.+Diveauthor=G.+Y.+V.+Cedex&title=A+Versatile+Annulation+Protocol+toward+Novel+Constrained+Phosphinic+Peptidomimetics&doi=10.1021%2Fjo071081l"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">A versatile annulation protocol toward novel constrained phosphinic peptidomimetics</span></div><div class="casAuthors">Nasopoulou, Magdalini; Georgiadis, Dimitris; Matziari, Magdalini; Dive, Vincent; Yiotakis, Athanasios</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7222-7228</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The development of a novel 3-center 2-component annulation reaction between α,ω-carbamoylaldehydes and suitably monoalkylated phosphinic acids is reported.  Depending on the starting α,ω-carbamoylaldehyde, diverse phosphinic scaffolds varying in the size of their rigidity element, the nature and stereochem. of substituents, and the participation of heteroatoms in the azacyclic ring system can be obtained in one synthetic step and in high yield.  In addn., this methodol. allows the synthesis of Fmoc-protected (Fmoc = 9-fluorenylmethyloxycarbonyl) constrained aminophosphinic acids that can be easily converted to suitable pseudodipeptide building blocks compatible with the requirements of peptide synthesis on the solid phase.  Finally, the careful choice of both substituents and protecting groups can provide functionally diverse, orthogonally protected constrained scaffolds for extended derivatization of the target phosphinic peptidomimetic structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTlVy0upnrWrVg90H21EOLACvtfcHk0lgThtYyw_UghQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpsVeltLs%253D&md5=07017bd083b952e1bca5527309ecad8a</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjo071081l&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo071081l%26sid%3Dliteratum%253Aachs%26aulast%3DNasopoulou%26aufirst%3DM.%26aulast%3DGeorgiadis%26aufirst%3DD.%26aulast%3DMatziari%26aufirst%3DM.%26aulast%3DDive%26aufirst%3DV.%26aulast%3DCedex%26aufirst%3DG.%2BY.%2BV.%26atitle%3DA%2520Versatile%2520Annulation%2520Protocol%2520toward%2520Novel%2520Constrained%2520Phosphinic%2520Peptidomimetics%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2007%26volume%3D72%26spage%3D7222%26epage%3D7228%26doi%3D10.1021%2Fjo071081l" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wasney, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siarheyeva, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajian, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheryl, H.</span></span> <span> </span><span class="NLM_article-title">Fluorescence-Based Methods for Screening Writers and Readers of Histone Methyl Marks</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">71</span>– <span class="NLM_lpage">84</span>, <span class="refDoi"> DOI: 10.1177/1087057111422256</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1177%2F1087057111422256" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=21972038" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC38Xit1Olt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2012&pages=71-84&author=A.+Allali-Hassaniauthor=G.+A.+Wasneyauthor=A.+Siarheyevaauthor=T.+Hajianauthor=M.+Vedadiauthor=H.+Cheryl&title=Fluorescence-Based+Methods+for+Screening+Writers+and+Readers+of+Histone+Methyl+Marks&doi=10.1177%2F1087057111422256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescence-based methods for screening writers and readers of histone methyl marks</span></div><div class="casAuthors">Allali-Hassani, Abdellah; Wasney, Gregory A.; Siarheyeva, Alena; Hajian, Taraneh; Arrowsmith, Cheryl H.; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">71-84</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The histone methyltransferase (HMT) family of proteins consists of enzymes that methylate lysine or arginine residues on histone tails as well as other proteins.  Such modifications affect chromatin structure and play a significant regulatory role in gene expression.  Many HMTs have been implicated in tumorigenesis and progression of multiple malignancies and play essential roles in embryonic development and stem cell renewal.  Overexpression of some HMTs has been obsd. and is correlated pos. with various types of cancer.  Here the authors report development of a continuous fluorescence-based methyltransferase assay in a 384-well format and its application in detg. kinetic parameters for EHMT1, G9a, PRMT3, SETD7, and SUV39H2 as well as for screening against libraries of small mols. to identify enzyme inhibitors.  They also report the development of a peptide displacement assay using fluorescence polarization in a 384-well format to assay and screen protein peptide interactions such as those of WDR5 and EED, components of MLL and EZH2 methyltransferase complexes.  Using these high-throughput screening methods, the authors have identified potent inhibitors and ligands for some of these proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0s8Ex9rZOXrVg90H21EOLACvtfcHk0lgThtYyw_UghQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xit1Olt7g%253D&md5=105a20c2473ab603ade42a38764def09</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1177%2F1087057111422256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057111422256%26sid%3Dliteratum%253Aachs%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DCheryl%26aufirst%3DH.%26atitle%3DFluorescence-Based%2520Methods%2520for%2520Screening%2520Writers%2520and%2520Readers%2520of%2520Histone%2520Methyl%2520Marks%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2012%26volume%3D17%26spage%3D71%26epage%3D84%26doi%3D10.1177%2F1087057111422256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, J. F.</span></span> <span> </span><span class="NLM_article-title">The Slow-Binding and Slow, Tight-Binding Inhibition of Enzyme-Catalysed Reactions</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">1982</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">102</span>– <span class="NLM_lpage">105</span>, <span class="refDoi"> DOI: 10.1016/0968-0004(82)90157-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1016%2F0968-0004%2882%2990157-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADyaL38XhslKrtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=1982&pages=102-105&author=J.+F.+Morrison&title=The+Slow-Binding+and+Slow%2C+Tight-Binding+Inhibition+of+Enzyme-Catalysed+Reactions&doi=10.1016%2F0968-0004%2882%2990157-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">The slow-binding and slow, tight-binding inhibition of enzyme-catalyzed reactions</span></div><div class="casAuthors">Morrison, John F.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">1982</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">102-5</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    </div><div class="casAbstract">The mechanisms of enzyme inhibition by slow-binding and slow, tight-binding reversible inhibitors are discussed, as well as the kinetics of these processes as opposed to classical and tight-binding inhibitor mechanisms.  Examples of these types of inhibitors in the enzyme literature are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoun0ayjodbYrVg90H21EOLACvtfcHk0ljr8_17Hh43PA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL38XhslKrtrw%253D&md5=1740627c1ad2adf182f4c0ae355bb4de</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2F0968-0004%2882%2990157-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0968-0004%252882%252990157-8%26sid%3Dliteratum%253Aachs%26aulast%3DMorrison%26aufirst%3DJ.%2BF.%26atitle%3DThe%2520Slow-Binding%2520and%2520Slow%252C%2520Tight-Binding%2520Inhibition%2520of%2520Enzyme-Catalysed%2520Reactions%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D1982%26volume%3D7%26spage%3D102%26epage%3D105%26doi%3D10.1016%2F0968-0004%2882%2990157-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span> <span> </span><span class="NLM_article-title">Tight Binding Inhibitors</span>. In  <i>Evaluation of Enzyme Inhibitors in Drug Discovery: A Guide for Medicinal Chemists and Pharmacologists</i>, <span class="NLM_edition">2</span>nd ed.; <span class="NLM_publisher-name">John Wiley & Sons</span>: <span class="NLM_publisher-loc">New Jersey</span>, <span class="NLM_year">2013</span>; pp  <span class="NLM_fpage">245</span>– <span class="NLM_lpage">285</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1002%2F9781118540398.ch7" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&pages=245-285&author=R.+A.+Copeland&title=Evaluation+of+Enzyme+Inhibitors+in+Drug+Discovery%3A+A+Guide+for+Medicinal+Chemists+and+Pharmacologists"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2F9781118540398.ch7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9781118540398.ch7%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26atitle%3DTight%2520Binding%2520Inhibitors%26btitle%3DEvaluation%2520of%2520Enzyme%2520Inhibitors%2520in%2520Drug%2520Discovery%253A%2520A%2520Guide%2520for%2520Medicinal%2520Chemists%2520and%2520Pharmacologists%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%26date%3D2013%26spage%3D245%26epage%3D285" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span> <span> </span><span class="NLM_article-title">An Asparagine/Glycine Switch Governs Product Specificity of Human N-Terminal Methyltransferase NTMT2</span>. <i>Commun. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>1</i></span>, <span class="NLM_elocation-id">183</span> <span class="refDoi"> DOI: 10.1038/s42003-018-0196-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1038%2Fs42003-018-0196-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=30417120" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A280%3ADC%252BB3cvovV2qug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2018&author=C.+Dongauthor=G.+Dongauthor=L.+Liauthor=L.+Zhuauthor=W.+Tempelauthor=Y.+Liuauthor=R.+Huangauthor=J.+Min&title=An+Asparagine%2FGlycine+Switch+Governs+Product+Specificity+of+Human+N-Terminal+Methyltransferase+NTMT2&doi=10.1038%2Fs42003-018-0196-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">An asparagine/glycine switch governs product specificity of human N-terminal methyltransferase NTMT2</span></div><div class="casAuthors">Dong Cheng; Li Li; Zhu Licheng; Tempel Wolfram; Liu Yanli; Min Jinrong; Dong Guangping; Huang Rong; Zhu Licheng; Min Jinrong</div><div class="citationInfo"><span class="NLM_cas:title">Communications biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">183</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">α-N-terminal methylation of proteins is an important post-translational modification that is catalyzed by two different N-terminal methyltransferases, namely NTMT1 and NTMT2.  Previous studies have suggested that NTMT1 is a tri-methyltransferase, whereas NTMT2 is a mono-methyltransferase.  Here, we report the first crystal structures, to our knowledge, of NTMT2 in binary complex with S-adenosyl-L-methionine as well as in ternary complex with S-adenosyl-L-homocysteine and a substrate peptide.  Our structural observations combined with biochemical studies reveal that NTMT2 is also able to di-/tri-methylate the GPKRIA peptide and di-methylate the PPKRIA peptide, otherwise it is predominantly a mono-methyltransferase.  The residue N89 of NTMT2 serves as a gatekeeper residue that regulates the binding of unmethylated versus monomethylated substrate peptide.  Structural comparison of NTMT1 and NTMT2 prompts us to design a N89G mutant of NTMT2 that can profoundly alter its catalytic activities and product specificities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSKFvwVHUgF0kwlCcfExbvbfW6udTcc2eZMTzYi1qMl5rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cvovV2qug%253D%253D&md5=edc8c87334f423ab98872442f81d7928</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fs42003-018-0196-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs42003-018-0196-2%26sid%3Dliteratum%253Aachs%26aulast%3DDong%26aufirst%3DC.%26aulast%3DDong%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DZhu%26aufirst%3DL.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DR.%26aulast%3DMin%26aufirst%3DJ.%26atitle%3DAn%2520Asparagine%252FGlycine%2520Switch%2520Governs%2520Product%2520Specificity%2520of%2520Human%2520N-Terminal%2520Methyltransferase%2520NTMT2%26jtitle%3DCommun.%2520Biol.%26date%3D2018%26volume%3D1%26doi%3D10.1038%2Fs42003-018-0196-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rizzo, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dirk, L. M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strunk, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roiko, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunzelle, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houtz, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trievel, R. C.</span></span> <span> </span><span class="NLM_article-title">SET7/9 Catalytic Mutants Reveal the Role of Active Site Water Molecules in Lysine Multiple Methylation</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>285</i></span>,  <span class="NLM_fpage">31849</span>– <span class="NLM_lpage">31858</span>, <span class="refDoi"> DOI: 10.1074/jbc.M110.114587</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1074%2Fjbc.M110.114587" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=20675860" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=31849-31858&author=P.+A.+Rizzoauthor=+Delauthor=L.+M.+A.+Dirkauthor=B.+S.+Strunkauthor=M.+S.+Roikoauthor=J.+S.+Brunzelleauthor=R.+L.+Houtzauthor=R.+C.+Trievel&title=SET7%2F9+Catalytic+Mutants+Reveal+the+Role+of+Active+Site+Water+Molecules+in+Lysine+Multiple+Methylation&doi=10.1074%2Fjbc.M110.114587"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.114587&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.114587%26sid%3Dliteratum%253Aachs%26aulast%3DRizzo%26aufirst%3DP.%2BA.%26aulast%3DDel%26aulast%3DDirk%26aufirst%3DL.%2BM.%2BA.%26aulast%3DStrunk%26aufirst%3DB.%2BS.%26aulast%3DRoiko%26aufirst%3DM.%2BS.%26aulast%3DBrunzelle%26aufirst%3DJ.%2BS.%26aulast%3DHoutz%26aufirst%3DR.%2BL.%26aulast%3DTrievel%26aufirst%3DR.%2BC.%26atitle%3DSET7%252F9%2520Catalytic%2520Mutants%2520Reveal%2520the%2520Role%2520of%2520Active%2520Site%2520Water%2520Molecules%2520in%2520Lysine%2520Multiple%2520Methylation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D31849%26epage%3D31858%26doi%3D10.1074%2Fjbc.M110.114587" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chuikov, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurash, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Justin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ivanov, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mckinney, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempst, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prives, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gamblin, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barlev, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinberg, D.</span></span> <span> </span><span class="NLM_article-title">Regulation of P53 Activity through Lysine Methylation</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>432</i></span>,  <span class="NLM_fpage">353</span>– <span class="NLM_lpage">360</span>, <span class="refDoi"> DOI: 10.1038/nature03117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1038%2Fnature03117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=15525938" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BD2cXpvVCjtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=432&publication_year=2004&pages=353-360&author=S.+Chuikovauthor=J.+K.+Kurashauthor=J.+R.+Wilsonauthor=B.+Xiaoauthor=N.+Justinauthor=G.+S.+Ivanovauthor=K.+Mckinneyauthor=P.+Tempstauthor=C.+Privesauthor=S.+J.+Gamblinauthor=N.+A.+Barlevauthor=D.+Reinberg&title=Regulation+of+P53+Activity+through+Lysine+Methylation&doi=10.1038%2Fnature03117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of p53 activity through lysine methylation</span></div><div class="casAuthors">Chuikov, Sergei; Kurash, Julia K.; Wilson, Jonathan R.; Xiao, Bing; Justin, Neil; Ivanov, Gleb S.; McKinney, Kristine; Tempst, Paul; Prives, Carol; Gamblin, Steven J.; Barlev, Nickolai A.; Reinberg, Danny</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">432</span>
        (<span class="NLM_cas:issue">7015</span>),
    <span class="NLM_cas:pages">353-360</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">P53 is a tumor suppressor that regulates the cellular response to genotoxic stresses. p53 is a short-lived protein and its activity is regulated mostly by stabilization via different post-translational modifications.  Here we report a novel mechanism of p53 regulation through lysine methylation by Set9 methyltransferase.  Set9 specifically methylates p53 at one residue within the carboxyl-terminus regulatory region.  Methylated p53 is restricted to the nucleus and the modification pos. affects its stability.  Set9 regulates the expression of p53 target genes in a manner dependent on the p53-methylation site.  The crystal structure of a ternary complex of Set9 with a p53 peptide and the cofactor product S-adenosyl-L-homocysteine (AdoHcy) provides the mol. basis for recognition of p53 by this lysine methyltransferase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQEN46GC7xzLVg90H21EOLACvtfcHk0ljjMEmFliA-AA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXpvVCjtbo%253D&md5=eff3a7ec7f6adbf1fc43016325f074d0</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fnature03117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03117%26sid%3Dliteratum%253Aachs%26aulast%3DChuikov%26aufirst%3DS.%26aulast%3DKurash%26aufirst%3DJ.%2BK.%26aulast%3DWilson%26aufirst%3DJ.%2BR.%26aulast%3DXiao%26aufirst%3DB.%26aulast%3DJustin%26aufirst%3DN.%26aulast%3DIvanov%26aufirst%3DG.%2BS.%26aulast%3DMckinney%26aufirst%3DK.%26aulast%3DTempst%26aufirst%3DP.%26aulast%3DPrives%26aufirst%3DC.%26aulast%3DGamblin%26aufirst%3DS.%2BJ.%26aulast%3DBarlev%26aufirst%3DN.%2BA.%26aulast%3DReinberg%26aufirst%3DD.%26atitle%3DRegulation%2520of%2520P53%2520Activity%2520through%2520Lysine%2520Methylation%26jtitle%3DNature%26date%3D2004%26volume%3D432%26spage%3D353%26epage%3D360%26doi%3D10.1038%2Fnature03117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Couture, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collazo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunzelle, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trievel, R. C.</span></span> <span> </span><span class="NLM_article-title">Structural and Functional Analysis of SET8, a Histone H4 Lys-20 Methyltransferase.</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_fpage">1455</span>– <span class="NLM_lpage">1465</span>, <span class="refDoi"> DOI: 10.1101/gad.1318405</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1101%2Fgad.1318405" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=15933070" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlslChu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&pages=1455-1465&author=J.+Coutureauthor=E.+Collazoauthor=J.+S.+Brunzelleauthor=R.+C.+Trievel&title=Structural+and+Functional+Analysis+of+SET8%2C+a+Histone+H4+Lys-20+Methyltransferase.&doi=10.1101%2Fgad.1318405"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and functional analysis of SET8, a histone H4 Lys-20 methyltransferase</span></div><div class="casAuthors">Couture, Jean-Francois; Collazo, Evys; Brunzelle, Joseph S.; Trievel, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1455-1465</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">SET8 (also known as PR-SET7) is a histone H4-Lys-20-specific methyltransferase that is implicated in cell-cycle-dependent transcriptional silencing and mitotic regulation in metazoans.  Herein we report the crystal structure of human SET8 (hSET8) bound to a histone H4 peptide bearing Lys-20 and the product cofactor S-adenosylhomocysteine.  Histone H4 intercalates in the substrate-binding cleft as an extended parallel β-strand.  Residues preceding Lys-20 in H4 engage in an extensive array of salt bridge, hydrogen bond, and van der Waals interactions with hSET8, while the C-terminal residues bind through predominantly hydrophobic interactions.  Mutational anal. of both the substrate-binding cleft and histone H4 reveals that interactions with residues in the N and C termini of the H4 peptide are crit. for conferring substrate specificity.  Finally, anal. of the product specificity indicates that hSET8 is a monomethylase, consistent with its role in the maintenance of Lys-20 monomethylation during cell division.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphqNcPOAjIbbVg90H21EOLACvtfcHk0likeqW58KJbJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlslChu7w%253D&md5=26dd3cdbaee412a5904ba8268b8efb40</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1101%2Fgad.1318405&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1318405%26sid%3Dliteratum%253Aachs%26aulast%3DCouture%26aufirst%3DJ.%26aulast%3DCollazo%26aufirst%3DE.%26aulast%3DBrunzelle%26aufirst%3DJ.%2BS.%26aulast%3DTrievel%26aufirst%3DR.%2BC.%26atitle%3DStructural%2520and%2520Functional%2520Analysis%2520of%2520SET8%252C%2520a%2520Histone%2520H4%2520Lys-20%2520Methyltransferase.%26jtitle%3DGenes%2520Dev.%26date%3D2005%26spage%3D1455%26epage%3D1465%26doi%3D10.1101%2Fgad.1318405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lunin, V. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antoshenko, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dombrovski, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campagna-Slater, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotnikov, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span> <span> </span><span class="NLM_article-title">Structural Biology of Human H3K9 Methyltransferases</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>5</i></span>, <span class="NLM_elocation-id">e8570</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0008570</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1371%2Fjournal.pone.0008570" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=20084102" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A280%3ADC%252BC3c%252FjtlGgsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&author=H.+Wuauthor=J.+Minauthor=V.+V.+Luninauthor=T.+Antoshenkoauthor=L.+Dombrovskiauthor=H.+Zengauthor=A.+Allali-Hassaniauthor=V.+Campagna-Slaterauthor=M.+Vedadiauthor=C.+H.+Arrowsmithauthor=A.+N.+Plotnikovauthor=M.+Schapira&title=Structural+Biology+of+Human+H3K9+Methyltransferases&doi=10.1371%2Fjournal.pone.0008570"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Structural biology of human H3K9 methyltransferases</span></div><div class="casAuthors">Wu Hong; Min Jinrong; Lunin Vladimir V; Antoshenko Tatiana; Dombrovski Ludmila; Zeng Hong; Allali-Hassani Abdellah; Campagna-Slater Valerie; Vedadi Masoud; Arrowsmith Cheryl H; Plotnikov Alexander N; Schapira Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">PloS one</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">e8570</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">SET domain methyltransferases deposit methyl marks on specific histone tail lysine residues and play a major role in epigenetic regulation of gene transcription.  We solved the structures of the catalytic domains of GLP, G9a, Suv39H2 and PRDM2, four of the eight known human H3K9 methyltransferases in their apo conformation or in complex with the methyl donating cofactor, and peptide substrates.  We analyzed the structural determinants for methylation state specificity, and designed a G9a mutant able to tri-methylate H3K9.  We show that the I-SET domain acts as a rigid docking platform, while induced-fit of the Post-SET domain is necessary to achieve a catalytically competent conformation.  We also propose a model where long-range electrostatics bring enzyme and histone substrate together, while the presence of an arginine upstream of the target lysine is critical for binding and specificity.  ENHANCED VERSION: This article can also be viewed as an enhanced version in which the text of the article is integrated with interactive 3D representations and animated transitions.  Please note that a web plugin is required to access this enhanced functionality.  Instructions for the installation and use of the web plugin are available in Text S1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTAEo3Y9d9dncRAV4D_Ins_fW6udTcc2eYh6dO-4yYw77ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3c%252FjtlGgsA%253D%253D&md5=2649472b1401c69035f99e0ca20822a3</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0008570&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0008570%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DH.%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DLunin%26aufirst%3DV.%2BV.%26aulast%3DAntoshenko%26aufirst%3DT.%26aulast%3DDombrovski%26aufirst%3DL.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DCampagna-Slater%26aufirst%3DV.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DPlotnikov%26aufirst%3DA.%2BN.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DStructural%2520Biology%2520of%2520Human%2520H3K9%2520Methyltransferases%26jtitle%3DPLoS%2520One%26date%3D2010%26volume%3D5%26doi%3D10.1371%2Fjournal.pone.0008570" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oerum, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Catala, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atdjian, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brachet, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barraud, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iannazzo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Droogmans, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ethève-Quelquejeu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tisné, C.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Bisubstrate Analogues as Structural Tools to Investigate m A Methyltransferase Active Sites</span>. <i>RNA Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">798</span>– <span class="NLM_lpage">808</span>, <span class="refDoi"> DOI: 10.1080/15476286.2019.1589360</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1080%2F15476286.2019.1589360" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=30879411" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A280%3ADC%252BB3cbkvFehsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2019&pages=798-808&author=S.+Oerumauthor=M.+Catalaauthor=C.+Atdjianauthor=F.+Brachetauthor=P.+Barraudauthor=L.+Iannazzoauthor=L.+Droogmansauthor=M.+Eth%C3%A8ve-Quelquejeuauthor=C.+Tisn%C3%A9&title=Bisubstrate+Analogues+as+Structural+Tools+to+Investigate+m+A+Methyltransferase+Active+Sites&doi=10.1080%2F15476286.2019.1589360"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Bisubstrate analogues as structural tools to investigate m(6)A methyltransferase active sites</span></div><div class="casAuthors">Oerum Stephanie; Catala Marjorie; Barraud Pierre; Tisne Carine; Atdjian Colette; Iannazzo Laura; Braud Emmanuelle; Etheve-Quelquejeu Melanie; Brachet Franck; Ponchon Luc; Droogmans Louis</div><div class="citationInfo"><span class="NLM_cas:title">RNA biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">798-808</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">RNA methyltransferases (MTases) catalyse the transfer of a methyl group to their RNA substrates using most-often S-adenosyl-L-methionine (SAM) as cofactor.  Only few RNA-bound MTases structures are currently available due to the difficulties in crystallising RNA:protein complexes.  The lack of complex structures results in poorly understood RNA recognition patterns and methylation reaction mechanisms.  On the contrary, many cofactor-bound MTase structures are available, resulting in well-understood protein:cofactor recognition, that can guide the design of bisubstrate analogues that mimic the state at which both the substrate and the cofactor is bound.  Such bisubstrate analogues were recently synthesized for proteins monomethylating the N6-atom of adenine (m(6)A).  These proteins include, amongst others, RlmJ in E. coli and METLL3:METT14 and METTL16 in human.  As a proof-of-concept, we here test the ability of the bisubstrate analogues to mimic the substrate:cofactor bound state during catalysis by studying their binding to RlmJ using differential scanning fluorimetry, isothermal titration calorimetry and X-ray crystallography.  We find that the methylated adenine base binds in the correct pocket, and thus these analogues could potentially be used broadly to study the RNA recognition and catalytic mechanism of m(6)A MTases.  Two bisubstrate analogues bind RlmJ with micro-molar affinity, and could serve as starting scaffolds for inhibitor design against m(6)A RNA MTases.  The same analogues cause changes in the melting temperature of the m(1)A RNA MTase, TrmK, indicating non-selective protein:compound complex formation.  Thus, optimization of these molecular scaffolds for m(6)A RNA MTase inhibition should aim to increase selectivity, as well as affinity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRDdwiIxGiYOv6xYuD7tu2XfW6udTcc2ebbpaljoQUMLrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cbkvFehsg%253D%253D&md5=1c13082943751faaf85ebe743756df1d</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1080%2F15476286.2019.1589360&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15476286.2019.1589360%26sid%3Dliteratum%253Aachs%26aulast%3DOerum%26aufirst%3DS.%26aulast%3DCatala%26aufirst%3DM.%26aulast%3DAtdjian%26aufirst%3DC.%26aulast%3DBrachet%26aufirst%3DF.%26aulast%3DBarraud%26aufirst%3DP.%26aulast%3DIannazzo%26aufirst%3DL.%26aulast%3DDroogmans%26aufirst%3DL.%26aulast%3DEth%25C3%25A8ve-Quelquejeu%26aufirst%3DM.%26aulast%3DTisn%25C3%25A9%26aufirst%3DC.%26atitle%3DBisubstrate%2520Analogues%2520as%2520Structural%2520Tools%2520to%2520Investigate%2520m%2520A%2520Methyltransferase%2520Active%2520Sites%26jtitle%3DRNA%2520Biol.%26date%3D2019%26volume%3D16%26spage%3D798%26epage%3D808%26doi%3D10.1080%2F15476286.2019.1589360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swewczyk, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trush, V. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freitas, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tatlock, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumpf, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wythes, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casimiro-Garcia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denny, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parikh, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. R.</span></span> <span> </span><span class="NLM_article-title">Selective, Small-Molecule Co-Factor Binding Site Inhibition of a Su(Var)3 – 9, Enhancer of Zeste, Trithorax Domain Containing Lysine Methyltransferase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">7669</span>– <span class="NLM_lpage">7683</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00112</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00112" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFyrur7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=7669-7683&author=A.+P.+Taylorauthor=M.+Swewczykauthor=S.+Kennedyauthor=V.+V.+Trushauthor=H.+Wuauthor=H.+Zengauthor=A.+Dongauthor=R.+F.+Freitasauthor=J.+Tatlockauthor=R.+A.+Kumpfauthor=M.+Wythesauthor=A.+Casimiro-Garciaauthor=R.+A.+Dennyauthor=M.+D.+Parikhauthor=F.+Liauthor=D.+Barsyte-Lovejoyauthor=M.+Schapiraauthor=M.+Vedadiauthor=M.+Schapiraauthor=P.+J.+Brownauthor=C.+H.+Arrowsmithauthor=D.+R.+Owen&title=Selective%2C+Small-Molecule+Co-Factor+Binding+Site+Inhibition+of+a+Su%28Var%293+%E2%80%93+9%2C+Enhancer+of+Zeste%2C+Trithorax+Domain+Containing+Lysine+Methyltransferase&doi=10.1021%2Facs.jmedchem.9b00112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Selective, Small-Molecule Co-Factor Binding Site Inhibition of a Su(var)3-9, Enhancer of Zeste, Trithorax Domain Containing Lysine Methyltransferase</span></div><div class="casAuthors">Taylor, Alexandria P.; Swewczyk, Magdalena; Kennedy, Steven; Trush, Viacheslav V.; Wu, Hong; Zeng, Hong; Dong, Aiping; Ferreira de Freitas, Renato; Tatlock, John; Kumpf, Robert A.; Wythes, Martin; Casimiro-Garcia, Agustin; Denny, Rajiah Aldrin; Parikh, Mihir D.; Li, Fengling; Barsyte-Lovejoy, Dalia; Schapira, Matthieu; Vedadi, Masoud; Brown, Peter J.; Arrowsmith, Cheryl H.; Owen, Dafydd R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7669-7683</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The first chem. probe to primarily occupy the co-factor binding site of a Su(var)3-9, enhancer of a zeste, trithorax (SET) domain contg. protein lysine methyltransferase (PKMT) is reported.  Protein methyltransferases require S-adenosylmethionine (SAM) as a co-factor (Me donor) for enzymic activity.  However, SAM itself represents a poor medicinal chem. starting point for a selective, cell-active inhibitor given its extreme physicochem. properties and its role in multiple cellular processes.  A previously untested medicinal chem. strategy of deliberate file enrichment around mols. bearing the hallmarks of SAM, but with improved lead-like properties from the outset, yielded viable hits against SET and MYND domain-contg. protein 2 (SMYD2) that were shown to bind in the co-factor site.  These leads were optimized to identify a highly biochem. potent, PKMT-selective, and cell-active chem. probe.  While substrate-based inhibitors of PKMTs are known, this represents a novel, co-factor-derived strategy for the inhibition of SMYD2 which may also prove applicable to lysine methyltransferase family members previously thought of as intractable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrayuqhEmC-MLVg90H21EOLACvtfcHk0lhnbRSsCOOPuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFyrur7N&md5=6a9f875b93804141716a85d77b93c5ee</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00112%26sid%3Dliteratum%253Aachs%26aulast%3DTaylor%26aufirst%3DA.%2BP.%26aulast%3DSwewczyk%26aufirst%3DM.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DTrush%26aufirst%3DV.%2BV.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DFreitas%26aufirst%3DR.%2BF.%26aulast%3DTatlock%26aufirst%3DJ.%26aulast%3DKumpf%26aufirst%3DR.%2BA.%26aulast%3DWythes%26aufirst%3DM.%26aulast%3DCasimiro-Garcia%26aufirst%3DA.%26aulast%3DDenny%26aufirst%3DR.%2BA.%26aulast%3DParikh%26aufirst%3DM.%2BD.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26atitle%3DSelective%252C%2520Small-Molecule%2520Co-Factor%2520Binding%2520Site%2520Inhibition%2520of%2520a%2520Su%2528Var%25293%2520%25E2%2580%2593%25209%252C%2520Enhancer%2520of%2520Zeste%252C%2520Trithorax%2520Domain%2520Containing%2520Lysine%2520Methyltransferase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D7669%26epage%3D7683%26doi%3D10.1021%2Facs.jmedchem.9b00112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Babault, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ju, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of Bisubstrate Inhibitors of Nicotinamide <i>N</i> -Methyltransferase (NNMT)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1541</span>– <span class="NLM_lpage">1551</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01422</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01422" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC1cXotVOruw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=1541-1551&author=N.+Babaultauthor=A.+Allali-Hassaniauthor=F.+Liauthor=J.+Fanauthor=A.+Yueauthor=K.+Juauthor=F.+Liuauthor=M.+Vedadiauthor=J.+Liuauthor=J.+Jin&title=Discovery+of+Bisubstrate+Inhibitors+of+Nicotinamide+N+-Methyltransferase+%28NNMT%29&doi=10.1021%2Facs.jmedchem.7b01422"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Bisubstrate Inhibitors of Nicotinamide N-Methyltransferase (NNMT)</span></div><div class="casAuthors">Babault, Nicolas; Allali-Hassani, Abdellah; Li, Fengling; Fan, Jie; Yue, Alex; Ju, Kevin; Liu, Feng; Vedadi, Masoud; Liu, Jing; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1541-1551</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) catalyzes the N-methylation of pyridine-contg. compds. using the cofactor S-5'-adenosyl-L-methionine (SAM) as the Me group donor.  Through the regulation of the levels of its substrates, cofactor, and products, NNMT plays important role in physiol. and pathophysiol.  Overexpression of NNMT has been implicated in various human diseases.  Potent and selective small-mol. NNMT inhibitors are valuable chem. tools for testing biol. and therapeutic hypotheses.  However, very few NNMT inhibitors have been reported.  Here, we describe the discovery of a bisubstrate NNMT inhibitor MS2734 (6), and characterization of this inhibitor in biochem., biophys., kinetic, and structural studies.  Importantly, we obtained the first crystal structure of human NNMT in complex with a small-mol. inhibitor.  The structure of the NNMT-6 complex has unambiguously demonstrated that 6 occupied both substrate and cofactor binding sites.  The findings paved the way for developing more potent and selective NNMT inhibitors in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9ThNiA1GTwLVg90H21EOLACvtfcHk0lhnbRSsCOOPuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXotVOruw%253D%253D&md5=50e23e5f60beb64c184f6eadc6a75883</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01422%26sid%3Dliteratum%253Aachs%26aulast%3DBabault%26aufirst%3DN.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DFan%26aufirst%3DJ.%26aulast%3DYue%26aufirst%3DA.%26aulast%3DJu%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520Bisubstrate%2520Inhibitors%2520of%2520Nicotinamide%2520N%2520-Methyltransferase%2520%2528NNMT%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D1541%26epage%3D1551%26doi%3D10.1021%2Facs.jmedchem.7b01422" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahley, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stivers, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. G.</span></span> <span> </span><span class="NLM_article-title">A Transient Kinetic Analysis of PRMT1 Catalysis</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">7033</span>– <span class="NLM_lpage">7044</span>, <span class="refDoi"> DOI: 10.1021/bi200456u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi200456u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC3MXptFKltLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2011&pages=7033-7044&author=Y.+Fengauthor=N.+Xieauthor=M.+Jinauthor=M.+R.+Stahleyauthor=J.+T.+Stiversauthor=Y.+G.+Zheng&title=A+Transient+Kinetic+Analysis+of+PRMT1+Catalysis&doi=10.1021%2Fbi200456u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">A transient kinetic analysis of PRMT1 catalysis</span></div><div class="casAuthors">Feng, You; Xie, Nan; Jin, Miyeong; Stahley, Mary R.; Stivers, JamesT.; Zheng, Yujun George</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">7033-7044</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Post-translational modifications (PTMs) are important strategies used by eukaryotic organisms to modulate their phenotypes.  One of the well-studied PTMs, Arg residue methylation, is catalyzed by protein arginine methyltransferases (PRMTs) with S-adenosyl-L-methionine (SAM) as the Me donor.  The functions of PRMTs have been broadly studied in different biol. processes and diseased states, but the mol. basis for Arg methylation is not well-defined.  Here, the authors report the transient-state kinetic anal. of rat PRMT1 catalysis.  The fast assocn. and dissocn. rates suggested that PRMT1 catalysis of histone H4 methylation followed a rapid equil. sequential kinetic mechanism.  The data provided direct evidence that the chem. of Me transfer is the major rate-limiting step and that binding of cofactors SAM or S-adenosyl-L-homocysteine (SAH) affects the assocn. and dissocn. of H4 with PRMT1.  Importantly, from stopped-flow fluorescence measurements, the authors identified a crit. kinetic step suggesting a pre-catalytic conformational transition induced by substrate binding.  These results provide new insights into the mechanism of Arg methylation and the rational design of PRMT inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpImJRI93ID4rVg90H21EOLACvtfcHk0ljY7G4Z0N-rkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXptFKltLc%253D&md5=f2562d7ece9b10754b551c2b6dec7cb1</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fbi200456u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi200456u%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DXie%26aufirst%3DN.%26aulast%3DJin%26aufirst%3DM.%26aulast%3DStahley%26aufirst%3DM.%2BR.%26aulast%3DStivers%26aufirst%3DJ.%2BT.%26aulast%3DZheng%26aufirst%3DY.%2BG.%26atitle%3DA%2520Transient%2520Kinetic%2520Analysis%2520of%2520PRMT1%2520Catalysis%26jtitle%3DBiochemistry%26date%3D2011%26volume%3D50%26spage%3D7033%26epage%3D7044%26doi%3D10.1021%2Fbi200456u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oudhoff, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tatlock, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freeman, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senisterra, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuznetsova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcellus, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couzens, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, Al.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Awar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fish, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstenberger, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benn, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimley, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braam, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, F. M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sudol, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaph, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span> <span> </span><span class="NLM_article-title">(R)-PFI-2 Is a Potent and Selective Inhibitor of SETD7 Methyltransferase Activity in Cells</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">12853</span>– <span class="NLM_lpage">12858</span>, <span class="refDoi"> DOI: 10.1073/pnas.1407358111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1073%2Fpnas.1407358111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=25136132" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVartr7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=12853-12858&author=D.+Barsyte-Lovejoyauthor=F.+Liauthor=M.+J.+Oudhoffauthor=J.+H.+Tatlockauthor=A.+Dongauthor=H.+Zengauthor=H.+Wuauthor=S.+A.+Freemanauthor=M.+Schapiraauthor=G.+A.+Senisterraauthor=E.+Kuznetsovaauthor=R.+Marcellusauthor=A.+Allali-Hassaniauthor=S.+Kennedyauthor=P.+Lambertauthor=A.+L.+Couzensauthor=A.+Amanauthor=Al.+Gingrasauthor=R.+Al-Awarauthor=P.+V.+Fishauthor=B.+S.+Gerstenbergerauthor=L.+Robertsauthor=C.+L.+Bennauthor=R.+L.+Grimleyauthor=M.+J.+Braamauthor=F.+M.+V.+Rossiauthor=M.+Sudolauthor=P.+J.+Brownauthor=M.+E.+Bunnageauthor=D.+R.+Owenauthor=C.+Zaphauthor=M.+Vedadiauthor=C.+H.+Arrowsmith&title=%28R%29-PFI-2+Is+a+Potent+and+Selective+Inhibitor+of+SETD7+Methyltransferase+Activity+in+Cells&doi=10.1073%2Fpnas.1407358111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">(R)-PFI-2 is a potent and selective inhibitor of SETD7 methyltransferase activity in cells</span></div><div class="casAuthors">Barsyte-Lovejoy, Dalia; Li, Fengling; Oudhoff, Menno J.; Tatlock, John H.; Dong, Aiping; Zeng, Hong; Wu, Hong; Freeman, Spencer A.; Schapira, Matthieu; Senisterra, Guillermo A.; Kuznetsova, Ekaterina; Marcellus, Richard; Allali-Hassani, Abdellah; Kennedy, Steven; Lambert, Jean-Philippe; Couzens, Amber L.; Aman, Ahmed; Gingras, Anne-Claude; Al-Awar, Rima; Fish, Paul V.; Gerstenberger, Brian S.; Roberts, Lee; Benn, Caroline L.; Grimley, Rachel L.; Braam, Mitchell J. S.; Rossi, Fabio M. V.; Sudol, Marius; Brown, Peter J.; Bunnage, Mark E.; Owen, Dafydd R.; Zaph, Colby; Vedadi, Masoud; Arrowsmith, Cheryl H.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">12853-12858</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">SET domain contg. (lysine methyltransferase) 7 (SETD7) is implicated in multiple signaling and disease related pathways with a broad diversity of reported substrates.  Here, we report the discovery of (R)-PFI-2-a first-in-class, potent (Kiapp = 0.33 nM), selective, and cell-active inhibitor of the methyltransferase activity of human SETD7-and its 500-fold less active enantiomer, (S)-PFI-2.  (R)-PFI-2 exhibits an unusual cofactor-dependent and substrate-competitive inhibitory mechanism by occupying the substrate peptide binding groove of SETD7, including the catalytic lysine-binding channel, and by making direct contact with the donor Me group of the cofactor, S-adenosylmethionine.  Chemoproteomics expts. using a biotinylated deriv. of (R)-PFI-2 demonstrated dose-dependent competition for binding to endogenous SETD7 in MCF7 cells pretreated with (R)-PFI-2.  In murine embryonic fibroblasts, (R)-PFI-2 treatment phenocopied the effects of Setd7 deficiency on Hippo pathway signaling, via modulation of the transcriptional coactivator Yes-assocd. protein (YAP) and regulation of YAP target genes.  In confluent MCF7 cells, (R)-PFI-2 rapidly altered YAP localization, suggesting continuous and dynamic regulation of YAP by the methyltransferase activity of SETD7.  These data establish (R)-PFI-2 and related compds. as a valuable tool-kit for the study of the diverse roles of SETD7 in cells and further validate protein methyltransferases as a druggable target class.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqv-HTBIcjMnbVg90H21EOLACvtfcHk0ljY7G4Z0N-rkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVartr7E&md5=0b8ad07ae23c4eba5cf11060d231b3f4</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1407358111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1407358111%26sid%3Dliteratum%253Aachs%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DOudhoff%26aufirst%3DM.%2BJ.%26aulast%3DTatlock%26aufirst%3DJ.%2BH.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DFreeman%26aufirst%3DS.%2BA.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DSenisterra%26aufirst%3DG.%2BA.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DMarcellus%26aufirst%3DR.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DLambert%26aufirst%3DP.%26aulast%3DCouzens%26aufirst%3DA.%2BL.%26aulast%3DAman%26aufirst%3DA.%26aulast%3DGingras%26aufirst%3DAl.%26aulast%3DAl-Awar%26aufirst%3DR.%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DGerstenberger%26aufirst%3DB.%2BS.%26aulast%3DRoberts%26aufirst%3DL.%26aulast%3DBenn%26aufirst%3DC.%2BL.%26aulast%3DGrimley%26aufirst%3DR.%2BL.%26aulast%3DBraam%26aufirst%3DM.%2BJ.%26aulast%3DRossi%26aufirst%3DF.%2BM.%2BV.%26aulast%3DSudol%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DZaph%26aufirst%3DC.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26atitle%3D%2528R%2529-PFI-2%2520Is%2520a%2520Potent%2520and%2520Selective%2520Inhibitor%2520of%2520SETD7%2520Methyltransferase%2520Activity%2520in%2520Cells%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2014%26volume%3D111%26spage%3D12853%26epage%3D12858%26doi%3D10.1073%2Fpnas.1407358111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Debler, E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warmack, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blobel, G.</span></span> <span> </span><span class="NLM_article-title">A Glutamate/Aspartate Switch Controls Product Specificity in a Protein Arginine Methyltransferase</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">2068</span>– <span class="NLM_lpage">2073</span>, <span class="refDoi"> DOI: 10.1073/pnas.1525783113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1073%2Fpnas.1525783113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=26858449" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC28Xit12ku7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=2068-2073&author=E.+W.+Deblerauthor=K.+Jainauthor=R.+A.+Warmackauthor=Y.+Fengauthor=S.+G.+Clarkeauthor=G.+Blobel&title=A+Glutamate%2FAspartate+Switch+Controls+Product+Specificity+in+a+Protein+Arginine+Methyltransferase&doi=10.1073%2Fpnas.1525783113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">A glutamate/aspartate switch controls product specificity in a protein arginine methyltransferase</span></div><div class="casAuthors">Debler, Erik W.; Jain, Kanishk; Warmack, Rebeccah A.; Feng, You; Clarke, Steven G.; Blobel, Gunter; Stavropoulos, Pete</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2068-2073</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Trypanosoma brucei PRMT7 (TbPRMT7) is a protein arginine methyltransferase (PRMT) that strictly monomethylates various substrates, thus classifying it as a type III PRMT.  However, the mol. basis of its unique product specificity has remained elusive.  Here, we present the structure of TbPRMT7 in complex with its cofactor product S-adenosyl-L-homocysteine (AdoHcy) at 2.8 Å resoln. and identify a glutamate residue crit. for its monomethylation behavior.  TbPRMT7 comprises the conserved methyltransferase and β-barrel domains, an N-terminal extension, and a dimerization arm.  The active site at the interface of the N-terminal extension, methyltransferase, and β-barrel domains is stabilized by the dimerization arm of the neighboring protomer, providing a structural basis for dimerization as a prerequisite for catalytic activity.  Mutagenesis of active-site residues highlights the importance of Glu181, the second of the two invariant glutamate residues of the double E loop that coordinate the target arginine in substrate peptides/proteins and that increase its nucleophilicity.  Strikingly, mutation of Glu181 to aspartate converts TbPRMT7 into a type I PRMT, producing asym. dimethylarginine (ADMA).  Isothermal titrn. calorimetry (ITC) using a histone H4 peptide showed that the Glu181Asp mutant has markedly increased affinity for monomethylated peptide with respect to the WT, suggesting that the enlarged active site can favorably accommodate monomethylated peptide and provide sufficient space for ADMA formation.  In conclusion, these findings yield valuable insights into the product specificity and the catalytic mechanism of protein arginine methyltransferases and have important implications for the rational (re)design of PRMTs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrj2myjluq1B7Vg90H21EOLACvtfcHk0lhs2SDgVD2PFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xit12ku7k%253D&md5=93f9d17569159c432b5801e2439c6df0</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1525783113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1525783113%26sid%3Dliteratum%253Aachs%26aulast%3DDebler%26aufirst%3DE.%2BW.%26aulast%3DJain%26aufirst%3DK.%26aulast%3DWarmack%26aufirst%3DR.%2BA.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DClarke%26aufirst%3DS.%2BG.%26aulast%3DBlobel%26aufirst%3DG.%26atitle%3DA%2520Glutamate%252FAspartate%2520Switch%2520Controls%2520Product%2520Specificity%2520in%2520a%2520Protein%2520Arginine%2520Methyltransferase%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2016%26volume%3D113%26spage%3D2068%26epage%3D2073%26doi%3D10.1073%2Fpnas.1525783113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iyamu, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yadav, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noinaj, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Novel Propargyl-Linked Bisubstrate Analogues as Tight-Binding Inhibitors for Nicotinamide N-Methyltransferase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">10783</span>– <span class="NLM_lpage">10797</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01255</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01255" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFKksLvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=10783-10797&author=D.+Chenauthor=L.+Liauthor=K.+Diazauthor=I.+D.+Iyamuauthor=R.+Yadavauthor=N.+Noinajauthor=R.+Huang&title=Novel+Propargyl-Linked+Bisubstrate+Analogues+as+Tight-Binding+Inhibitors+for+Nicotinamide+N-Methyltransferase&doi=10.1021%2Facs.jmedchem.9b01255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Novel propargyl-linked bisubstrate analogues as tight-binding inhibitors for nicotinamide N-methyltransferase</span></div><div class="casAuthors">Chen, Dongxing; Li, Linjie; Diaz, Krystal; Iyamu, Iredia D.; Yadav, Ravi; Noinaj, Nicholas; Huang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10783-10797</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) catalyzes the Me transfer from the cofactor S-adenosylmethionine to nicotinamide and other pyridine-contg. compds.  NNMT is an important regulator for nicotinamide metab. and methylation potential.  Aberrant expression levels of NNMT have been implicated in cancer, metabolic, and neurodegenerative diseases, which makes NNMT a potential therapeutic target.  Therefore, potent and selective NNMT inhibitors can serve as valuable tools to investigate the roles of NNMT in its mediated diseases.  Here, we applied a rational strategy to design and synthesize the tight-binding bisubstrate inhibitor LL320 through a novel propargyl linker.  LL320 demonstrates a Ki value of 1.6 ± 0.3 nM, which is the most potent inhibitor to date.  The cocrystal structure of LL320 confirms its interaction with both the substrate and cofactor binding sites on NNMT.  Importantly, this is the first example of using the propargyl linker to construct potent methyltransferase inhibitors, which will expand our understanding of the transition state of Me transfer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqs2XyawAg9ebVg90H21EOLACvtfcHk0lhs2SDgVD2PFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFKksLvN&md5=a6e3a5dc2044dc50ede1596c84804f61</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01255%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DDiaz%26aufirst%3DK.%26aulast%3DIyamu%26aufirst%3DI.%2BD.%26aulast%3DYadav%26aufirst%3DR.%26aulast%3DNoinaj%26aufirst%3DN.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DNovel%2520Propargyl-Linked%2520Bisubstrate%2520Analogues%2520as%2520Tight-Binding%2520Inhibitors%2520for%2520Nicotinamide%2520N-Methyltransferase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D10783%26epage%3D10797%26doi%3D10.1021%2Facs.jmedchem.9b01255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G.</span>; <span class="NLM_string-name">Ashton, A.</span></span>.  <i>Fast_dp. Methods Zenodo</i>, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=G.+Winter&author=A.+Ashton&title=Fast_dp.+Methods+Zenodo"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%26jtitle%3DFast_dp.%2520Methods%2520Zenodo%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkóczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: A Comprehensive Python-Based System for Macromolecular Structure Solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+A+Comprehensive+Python-Based+System+for+Macromolecular+Structure+Solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lhp1GucHh_OcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520A%2520Comprehensive%2520Python-Based%2520System%2520for%2520Macromolecular%2520Structure%2520Solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and Development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+Development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lhp1GucHh_OcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520Development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WJ7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WJ7','PDB','6WJ7'); return false;">PDB: 6WJ7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DTN" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DTN','PDB','6DTN'); return false;">PDB: 6DTN</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PVB" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PVB','PDB','6PVB'); return false;">PDB: 6PVB</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i46"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00770">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_36520"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c00770?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00770</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00770/suppl_file/jm0c00770_si_001.csv">CSV</a>)</p></li><li><p class="inline">NMR and MS spectra of compounds <b>15a</b>,<b>b</b>, <b>18</b>, <b>19</b>, and <b>22</b>–<b>24</b>; NMR, MALDI-MS, and LC–MS/HPLC spectra of compounds <b>1</b>–<b>12</b>; methylation assay of compounds <b>10</b>–<b>12</b> for NTMT1 (Figure S1); MALDI-MS methylation inhibition assay compound <b>8</b> (Figure S2); selectivity evaluation of compound <b>3</b> (Table S1); and crystallography data and refinement statistics (PDB IDs: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WJ7">6WJ7</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PVB">6PVB</a>) (Table S2) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00770/suppl_file/jm0c00770_si_002.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00770/suppl_file/jm0c00770_si_001.csv">jm0c00770_si_001.csv (1.91 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00770/suppl_file/jm0c00770_si_002.pdf">jm0c00770_si_002.pdf (2.94 MB)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The coordinates for the structure of human NTMT1 in complex with compound <b>1</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WJ7">6WJ7</a>) and <b>4</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PVB">6PVB</a>) have been deposited in the Protein Data Bank. The authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c00770%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-15%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c00770" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679ab67af8813da0","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
