
<html lang="en"     class="pb-page"  data-request-id="c85e912e-c049-4bae-8a88-01fca73eda28"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/jm301032j;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;issue:issue:10.1021/jmcmar.2013.56.issue-3;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Discovery and Mechanism Study of SIRT1 Activators that Promote the Deacetylation of Fluorophore-Labeled Substrate" /></meta><meta name="dc.Creator" content="Jiahui  Wu" /></meta><meta name="dc.Creator" content="Dengyou  Zhang" /></meta><meta name="dc.Creator" content="Lei  Chen" /></meta><meta name="dc.Creator" content="Jianneng  Li" /></meta><meta name="dc.Creator" content="Jianling  Wang" /></meta><meta name="dc.Creator" content="Chengqing  Ning" /></meta><meta name="dc.Creator" content="Niefang  Yu" /></meta><meta name="dc.Creator" content="Fei  Zhao" /></meta><meta name="dc.Creator" content="Dongying  Chen" /></meta><meta name="dc.Creator" content="Xiaoyan  Chen" /></meta><meta name="dc.Creator" content="Kaixian  Chen" /></meta><meta name="dc.Creator" content="Hualiang  Jiang" /></meta><meta name="dc.Creator" content="Hong  Liu" /></meta><meta name="dc.Creator" content="Dongxiang  Liu" /></meta><meta name="dc.Description" content="SIRT1 is an NAD+-dependent deacetylase, whose activators have potential therapeutic applications in age-related diseases. Here we report a new class of SIRT1 activators. The activation is dependent..." /></meta><meta name="Description" content="SIRT1 is an NAD+-dependent deacetylase, whose activators have potential therapeutic applications in age-related diseases. Here we report a new class of SIRT1 activators. The activation is dependent..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="January 28, 2013" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm301032j" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2013 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm301032j" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm301032j" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm301032j" /></link>
        
    
    

<title>Discovery and Mechanism Study of SIRT1 Activators that Promote the Deacetylation of Fluorophore-Labeled Substrate | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm301032j" /></meta><meta property="og:title" content="Discovery and Mechanism Study of SIRT1 Activators that Promote the Deacetylation of Fluorophore-Labeled Substrate" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0020.jpeg" /></meta><meta property="og:description" content="SIRT1 is an NAD+-dependent deacetylase, whose activators have potential therapeutic applications in age-related diseases. Here we report a new class of SIRT1 activators. The activation is dependent on the fluorophore labeled to the substrate. To elucidate the activation mechanism, we solved the crystal structure of SIRT3/ac-RHKKac-AMC complex. The structure revealed that the fluorophore blocked the H-bond formation and created a cavity between the substrate and the Rossmann fold. We built the SIRT1/ac-RHKKac-AMC complex model based on the crystal structure. Km and Kd determinations demonstrated that the fluorophore decreased the peptide binding affinity. The binding modes of SIRT1 activators indicated that a portion of the activators interacts with the fluorophore through π-stacking, while the other portion inserts into the cavity or interacts with the Rossmann fold, thus increasing the substrate affinity. Our study provides new insights into the mechanism of SIRT1 activation and may aid the design of novel SIRT1 activators." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm301032j"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm301032j">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm301032j&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm301032j&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm301032j&amp;href=/doi/10.1021/jm301032j" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2013</span><span class="cit-fg-volume">, 56</span><span class="cit-fg-issue">, 3</span><span class="cit-fg-pageRange">, 761-780</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/56/3" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm301441w" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm301302s" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery and Mechanism Study of SIRT1 Activators that Promote the Deacetylation of Fluorophore-Labeled Substrate</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jiahui++Wu">Jiahui Wu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Dengyou++Zhang">Dengyou Zhang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Lei++Chen">Lei Chen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jianneng++Li">Jianneng Li</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jianling++Wang">Jianling Wang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Chengqing++Ning">Chengqing Ning</a></span><span class="author-xref-symbol author-aff-symbol"><sup>#</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Niefang++Yu">Niefang Yu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>#</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Fei++Zhao">Fei Zhao</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Dongying++Chen">Dongying Chen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Xiaoyan++Chen">Xiaoyan Chen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Kaixian++Chen">Kaixian Chen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Hualiang++Jiang">Hualiang Jiang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Hong++Liu">Hong Liu</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Dongxiang++Liu">Dongxiang Liu</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-text"><sup>†</sup>Department of Pharmacology III, <sup>‡</sup>Drug Discovery and Design Center, <sup>§</sup>Department of Analytical Chemistry, and <sup>∥</sup>Center for Drug Metabolism and Pharmacokinetics, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu-Chong-Zhi Road, Shanghai 201203, China</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">⊥</span> <span class="aff-text">School of Life Sciences, Fudan University, Shanghai 200433, China</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">#</span> <span class="aff-text">Institute of Molecular Design and Drug Discovery, School of Pharmaceutical Sciences, Central South University, Changsha 410078, Hunan, China</span></div><div class="corresp-info"><strong>*</strong>For D.L.: E-mail: <a href="/cdn-cgi/l/email-protection#74100c183419151d185a071c171a175a15175a171a"><span class="__cf_email__" data-cfemail="f89c8094b895999194d68b909b969bd6999bd69b96">[email protected]</span></a>. For H.L.: phone, +86-21-50806600; fax, +86-21-50807088; E-mail, <a href="/cdn-cgi/l/email-protection#80e8ece9f5c0ede1e9ecaef3e8e3eee3aee1e3aee3ee"><span class="__cf_email__" data-cfemail="147c787d615479757d783a677c777a773a75773a777a">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm301032j&amp;href=/doi/10.1021%2Fjm301032j" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2013</span></span><span class="cit-volume">, 56</span><span class="cit-issue">, 3</span><span class="cit-pageRange">, 761–780</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">January 14, 2013</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>13 July 2012</li><li><span class="item_label"><b>Published</b> online</span>28 January 2013</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 14 February 2013</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm301032j" title="DOI URL">https://doi.org/10.1021/jm301032j</a></div><div class="article_header-article-copyright"><strong>Copyright © 2013 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D761%26pageCount%3D20%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DJiahui%2BWu%252C%2BDengyou%2BZhang%252C%2BLei%2BChen%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D56%26issueNum%3D3%26contentID%3Djm301032j%26title%3DDiscovery%2Band%2BMechanism%2BStudy%2Bof%2BSIRT1%2BActivators%2Bthat%2BPromote%2Bthe%2BDeacetylation%2Bof%2BFluorophore-Labeled%2BSubstrate%26numPages%3D20%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D780%26publicationDate%3DFebruary%2B2013">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm301032j"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">3353</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">24</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm301032j" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery and Mechanism Study of SIRT1 Activators that Promote the Deacetylation of Fluorophore-Labeled Substrate&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Jiahui&quot;,&quot;last_name&quot;:&quot;Wu&quot;},{&quot;first_name&quot;:&quot;Dengyou&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Lei&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Jianneng&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Jianling&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Chengqing&quot;,&quot;last_name&quot;:&quot;Ning&quot;},{&quot;first_name&quot;:&quot;Niefang&quot;,&quot;last_name&quot;:&quot;Yu&quot;},{&quot;first_name&quot;:&quot;Fei&quot;,&quot;last_name&quot;:&quot;Zhao&quot;},{&quot;first_name&quot;:&quot;Dongying&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Xiaoyan&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Kaixian&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Hualiang&quot;,&quot;last_name&quot;:&quot;Jiang&quot;},{&quot;first_name&quot;:&quot;Hong&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Dongxiang&quot;,&quot;last_name&quot;:&quot;Liu&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2013&quot;,&quot;month&quot;:&quot;01&quot;,&quot;day&quot;:&quot;28&quot;,&quot;issue&quot;:&quot;3&quot;,&quot;volume&quot;:&quot;56&quot;,&quot;pages&quot;:&quot;761-780&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm301032j&quot;},&quot;abstract&quot;:&quot;SIRT1 is an NAD+-dependent deacetylase, whose activators have potential therapeutic applications in age-related diseases. Here we report a new class of SIRT1 activators. The activation is dependent on the fluorophore labeled to the substrate. To elucidate the activation mechanism, we solved the crystal structure of SIRT3/ac-RHKKac-AMC complex. The structure revealed that the fluorophore blocked the H-bond formation and created a cavity between the substrate and the Rossmann fold. We built the SIRT1/ac-RHKKac-AMC complex model based on the crystal structure. Km and Kd determinations demonstrated that the fluorophore decreased the peptide binding affinity. The binding modes of SIRT1 activators indicated that a portion of the activators interacts with the fluorophore through π-stacking, while the other portion inserts into the cavity or interacts with the Rossmann fold, thus increasing the substrate affinity. Our study provides new insights into the mechanism of SIRT1 activation and may aid the design of novel S&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm301032j&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm301032j" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm301032j&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm301032j" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm301032j&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm301032j" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm301032j&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm301032j&amp;href=/doi/10.1021/jm301032j" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm301032j" /></input><a href="/doi/pdf/10.1021/jm301032j" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm301032j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm301032j%26sid%3Dliteratum%253Aachs%26pmid%3D23316803%26genre%3Darticle%26aulast%3DWu%26date%3D2013%26atitle%3DDiscovery%2Band%2BMechanism%2BStudy%2Bof%2BSIRT1%2BActivators%2Bthat%2BPromote%2Bthe%2BDeacetylation%2Bof%2BFluorophore-Labeled%2BSubstrate%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D56%26issue%3D3%26spage%3D761%26epage%3D780%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292239" title="Surface interactions">Surface interactions</a>,</li><li><a href="/action/doSearch?ConceptID=291445" title="Chemical structure">Chemical structure</a>,</li><li><a href="/action/doSearch?ConceptID=292529" title="Aromatic compounds">Aromatic compounds</a>,</li><li><a href="/action/doSearch?ConceptID=292606" title="Imaging probes">Imaging probes</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/56/3" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jmcmar.2013.56.issue-3/production/jmcmar.2013.56.issue-3.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/medium/jm-2012-01032j_0020.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm301032j&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">SIRT1 is an NAD<sup>+</sup>-dependent deacetylase, whose activators have potential therapeutic applications in age-related diseases. Here we report a new class of SIRT1 activators. The activation is dependent on the fluorophore labeled to the substrate. To elucidate the activation mechanism, we solved the crystal structure of SIRT3/ac-RHKK<sub>ac</sub>-AMC complex. The structure revealed that the fluorophore blocked the H-bond formation and created a cavity between the substrate and the Rossmann fold. We built the SIRT1/ac-RHKK<sub>ac</sub>-AMC complex model based on the crystal structure. <i>K</i><sub>m</sub> and <i>K</i><sub>d</sub> determinations demonstrated that the fluorophore decreased the peptide binding affinity. The binding modes of SIRT1 activators indicated that a portion of the activators interacts with the fluorophore through π-stacking, while the other portion inserts into the cavity or interacts with the Rossmann fold, thus increasing the substrate affinity. Our study provides new insights into the mechanism of SIRT1 activation and may aid the design of novel SIRT1 activators.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div class="extra-info-sec articleNote"><h4><span>¶</span> Author Present Address</h4><p class="last">Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu-Chong-Zhi Road, Shanghai 201203, China</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_20570" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_20570" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Sirtuins are a class of protein deacetylases that belong to KDAC family. Unlike other KDACs, the deacetylation by sirtuins requires the involvement of nicotinamide adenine dinucleotide (NAD<sup>+</sup>).<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Mammal possesses seven sirtuins (SIRT1–SIRT7), and they are widely distributed in different subcellular compartments.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> SIRT1, SIRT6, and SIRT7 are localized in the nucleus, while SIRT2 is in the cytoplasm and SIRT3, SIRT4, and SIRT5 are in mitochondria. Among them, SIRT1 shares the highest sequence homology with yeast sirtuin information regulator 2 (Sir2) and is the most extensively studied.</div><div class="NLM_p">SIRT1 deacetylates a broad range of substrates and plays important roles in many biological functions. For instance, SIRT1 deacetylates histone H1 and promotes the heterochromatin formation.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> H3 and H4 could be deacetylated by SIRT1 and the RNA interference of SIRT1 induced hyperacetylation of the two proteins in human cells.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> SIRT1 binds to the fat regulator peroxisome proliferator activated receptor γ (PPARγ) and represses the genes controlled by PPARγ. Studies revealed that overexpression of SIRT1 attenuates the adipogenesis.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Also, adiponectin transcription could be enhanced by SIRT1 in adipocytes through activating FOXO1<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> and cell resistance to oxidative stress could be increased by SIRT1 through regulating FOXO family of Forkhead transcription factors.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Besides, SIRT1 can regulate the genes involved in insulin secretion in pancreatic β cell lines and β cell-specific SIRT1-overexpressing (BESTO) mice by repressing uncoupling protein 2 (Ucp2).<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8, 9)</a> Additionally, SIRT1 promotes glucose homeostasis and mitochondrial biogenesis through PPARγ coactivator 1α (PGC-1α).<a onclick="showRef(event, 'ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref10 ref11">(10, 11)</a> Therefore, SIRT1 is involved in a wide range of physiological activities including fatty acid oxidization, stress resistance, insulin secretion, and glucose production, which are closely related to age-related diseases such as type 2 diabetes, cardiovascular disease, metabolic syndrome, inflammation, and also to aging and longevity.<a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(12, 13)</a> Development of SIRT1 activators has potential therapeutic applications in the treatment of the age-related diseases.</div><div class="NLM_p">Up to the present, several SIRT1 activators were identified (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>). They could be mainly categorized into two classes: natural plant polyphenols and synthesized small molecules. <b>1</b> (resveratrol) and several other plant polyphenols were first discovered by using a fluorescent assay with a fluorophore-labeled peptide as the substrate.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Treatment of <i>Saccharomyces cerevisiae</i> with <b>1</b> significantly increased the life span of the yeast. Studies in <i>Caenorhabditis elegans</i>,<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a><i>Drosophila</i>,<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15, 16)</a> and vertebrates<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> had proven the positive effect of <b>1</b> in increasing the life span. Besides, <b>1</b> was found to attenuated insulin resistance under high-fat diet induced conditions in vivo,<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> and treatment of type 2 diabetic mice with <b>1</b> induced mitochondrial biogenesis.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Meanwhile, small molecules with structures unrelated to <b>1</b> had been identified to activate SIRT1 in vitro by using the fluorophore-labeled substrates, which include compound <b>2</b>,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a><b>3</b>,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> and the compounds discovered by the Sirtris company like <b>4</b> (SRT1460), <b>5</b> (SRT1720),<a onclick="showRef(event, 'ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref22 ref23">(22, 23)</a> and <b>6</b><a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/medium/jm-2012-01032j_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical structures of representative SIRT1 activators.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm301032j&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">However, it is still controversial whether <b>1</b> and Sirtris compounds are real SIRT1 activators. In vitro studies revealed that the activation of SIRT1 by these compounds requires the fluorophore attached to the peptide substrate, and the activation could not be observed in deacetylating the substrate lacking of fluorophore.<a onclick="showRef(event, 'ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref25 ref26">(25, 26)</a> Even though Dai et al. provided the biophysical evidence of the direct binding between the Sirtris compounds and SIRT1, they suggested that the activators bind in an allosteric site of the enzyme.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> In terms of in vivo effect of <b>1</b>, in contrast with the positive results mentioned above, some studies showed that there is neither detectable effect of <b>1</b> on Sir2 activity in yeast<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> nor significant effect on lifespan in <i>Drosophila</i>.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> As for the SIRT1 small molecule activator <b>5</b>, some results indicated that it could ameliorate insulin sensitivity and glucose tolerance in obese mice<a onclick="showRef(event, 'ref22 ref29'); return false;" href="javascript:void(0);" class="ref ref22 ref29">(22, 29)</a> and enhance mitochondrial biogenesis in mice fed with a high calorie diet,<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> whereas Pacholec et al. reported that no effect of lowering plasma glucose or improving mitochondrial capacity could be detected in mice.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></div><div class="NLM_p last">Considering the numerous and paradoxical reports about SIRT1 activators, obviously it is very important to elucidate the activation mechanism of the activators on molecular level. In this study, a series of diaryl acylhydrazone compounds with the activation activities of SIRT1 were first identified from our in-house compound library by employing a coumarin-labeled acetyl peptide. We synthesized a series of analogues of these compounds and established their structure–activity relationship. Combining the protein crystallography, molecular modeling, and site-directed protein mutagenesis, we determined the binding site of our compounds, <b>1</b> and <b>4</b>. The activity, stability, and bioavailability of our compounds were also tested on cell level. Our goal is to explain the substrate selectivity and the activation mechanism of these SIRT1 activators from the structural point of view. The mechanism of SIRT1 activation by small molecules presented here may shed light in the design of SIRT1 novel activators.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21414" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21414" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Identification of Chemical Compounds That Enhance SIRT1 Enzyme Activity</h3><div class="NLM_p">A high-throughput fluorescent assay was deployed to screen our in-house compound library aiming to discover novel SIRT1 activators. To our delight, five compounds based on diaryl acylhydrazone scaffold were identified with the activation effect of SIRT1 deacetylation (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>). Notably, our compounds displayed EC<sub>1.5</sub> values ranging from 1.6 to 7.0 μM and the highest maximum activation was up to 375%. The EC<sub>1.5</sub> values were better than that of <b>1</b> (EC<sub>1.5</sub> of <b>1</b> was 23.6 μM). These results encouraged us to further optimize the diaryl acylhydrazone scaffold. Our efforts were mainly focused on diversifying the aryl fragments due to their convenient modification.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. The Activities of the Initial Hits Identified by Our Fluorescent Assay<a class="ref internalNav" href="#tbl1-fn1" aria-label="c">c</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/medium/jm-2012-01032j_0013.gif" alt="" id="fx1" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>Table a</sup><p class="last">EC<sub>1.5</sub> value represents the concentration of an activator required to increase the enzyme activity by 50%.</p></div><div class="footnote" id="t1fn2"><sup>Table b</sup><p class="last">Percentage of maximum activation achieved.</p></div><div class="footnote" id="tbl1-fn1"><sup>Table c</sup><p class="last">For comparison, the activity of compound <b>1</b> was also listed in this table.</p></div></div><div></div></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Synthesis of Diaryl Acylhydrozone Analogues</h3><div class="NLM_p">Compounds <b>16a</b>–<b>n</b> were prepared according to Scheme <a class="ref internalNav" href="#sch1" aria-label="1">1</a>. Suzuki coupling of methyl 4-bromobenzoate <b>12a</b> with aryl bromide or 4-(methoxycarbonyl)phenylboronic acid <b>12b</b> with aryl boric acid smoothly afforded biphenyl esters <b>13a</b>–<b>k</b>. Hydrazinolyzation of <b>13a</b>–<b>k</b> and <b>12c</b> gave hydrozide intermediate <b>14a</b>–<b>k</b>. Condensation of <b>14a</b>–<b>k</b> with benzaldehyde moieties <b>15a</b>–<b>d</b> readily provided compounds <b>16a</b>–<b>k</b>. <b>15b</b>–<b>d</b> were prepared according to the refs <a onclick="showRef(event, 'ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref31 ref32">31 and 32</a>. Preparation of compounds <b>16l</b>–<b>n</b> was from commercial available hydrazides <b>17l</b>–<b>n</b>. Naphthyl analogues <b>20a</b>–<b>i</b> were synthesized in a similar manner as <b>16a</b>–<b>n</b> (Scheme <a class="ref internalNav" href="#sch2" aria-label="2">2</a>). The details of the synthetic procedures and structural characterizations are described in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>.</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/medium/jm-2012-01032j_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0011.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>16a</b>–<b>n</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm301032j&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) Pd(PPh<sub>3</sub>)<sub>4</sub>, KF·2H<sub>2</sub>O, DME/H<sub>2</sub>O/EtOH (7:3:2), <i>M</i><sub>w</sub>, 110 °C; (ii) N<sub>2</sub>H<sub>4</sub>·H<sub>2</sub>O, MeOH, reflux; (iii) AcOH, EtOH, reflux.</p></p></figure><figure id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/medium/jm-2012-01032j_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0012.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>20a</b>–<b>i</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm301032j&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) Pd(PPh<sub>3</sub>)<sub>4</sub>, KF·2H<sub>2</sub>O, DME/H<sub>2</sub>O/EtOH (7:3:2), <i>M</i><sub>w</sub>, 110 °C; (ii) N<sub>2</sub>H<sub>4</sub>·H<sub>2</sub>O, MeOH, reflux; (iii) AcOH, EtOH, reflux.</p></p></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Structure–Activity Relationship (SAR) of Diaryl Acylhydrazone Analogues</h3><div class="NLM_p">The activation curves of representative compounds were shown in Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>. Initial observation of the activities (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>) revealed that the compound bearing the <i>p</i>-Cl phenyl group (<b>9</b>) seemed to be inferior to analogues with the biphenyl (<b>7</b>, <b>8</b>) or naphthyl groups (<b>10</b>, <b>11</b>). Changing the <i>p</i>-Cl phenyl group to other phenyl groups such as <i>p</i>-tolyl (<b>16l</b>) and <i>p</i>-CF<sub>3</sub> phenyl (<b>16m</b>) groups slightly reduced the activity (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>). Introduction of 3,5-dimethoxyl phenyl (<b>16n</b>) group notably weakened SIRT1 activation. These results suggested that the modification of phenyl group might not give rise to remarkable gain of the activity, which turned our attention to optimize biphenyl and naphthyl analogues.</div><figure id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/medium/jm-2012-01032j_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. The dose–response curves of our compounds and <b>1</b> measured by the fluorescent assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm301032j&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Structures and Activities of the Biphenyl and Phenyl Hydrazones<a class="ref internalNav" href="#tbl2-fn1" aria-label="d">d</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/medium/jm-2012-01032j_0014.gif" alt="" id="fx2" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/medium/jm-2012-01032j_0015.gif" alt="" id="fx3" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn3"><div class="footnote" id="t2fn3"><sup>Table a</sup><p class="last">EC<sub>1.5</sub> value represents the concentration of an activator required to increase the enzyme activity by 50%.</p></div><div class="footnote" id="t2fn4"><sup>Table b</sup><p class="last">Percentage of maximum activation achieved.</p></div><div class="footnote" id="t2fn5"><sup>Table c</sup><p class="last">None detected.</p></div><div class="footnote" id="tbl2-fn1"><sup>Table d</sup><p class="last">* = R<sub>1</sub> at C-3 position; # = R<sub>1</sub> at C-3 and C-5 position.</p></div></div><div></div></div><div class="NLM_p">Compared with the parent compound <b>7</b> (EC<sub>1.5</sub> = 1.7 μM, maximum activation = 246%), introduction of various substituted phenyl moieties <b>16a</b>–<b>c</b> at the C-4 position (P2 Part) retained the comparable activity whereas the C-3 substituted analogue <b>16d</b> exhibited 10-fold less activity (EC<sub>1.5</sub> = 18.3 μM), indicating that a linear fragment was preferred over a branched fragment. 3,5-Dibromide substituents (P1 part) were shown to play an important role in binding affinity because the removal of two bromide <b>16e</b> resulted in a drop-off of EC<sub>1.5</sub> (10.4 μM). Methylation of either hydroxy group (<b>16f</b> and <b>16g</b>) also eroded the activity, which indicated that both hydroxyl groups at C-2 and C-4 positions (P1 part) might function as hydrogen donors. Replacement of the phenyl moieties with heteroaryl groups (P2 part) such as furan <b>16h</b> and pyridine <b>16i</b>–<b>k</b> maintained the similar EC<sub>1.5</sub> but with the improvement of the maximum activation.</div><div class="NLM_p">Next, a series of naphthyl analogues were explored (Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>). Interestingly, replacement of the naphthalene <b>11</b> (EC<sub>1.5</sub> = 1.6 μM, maximum activation = 331%) with the quinoline <b>20a</b> (EC<sub>1.5</sub> = 55.2 μM) resulted in a sharp decrease of the activity. Introduction of the bromide (<b>20b</b>) at the C-6 position of the naphthalene (P2 part) displayed a similar EC<sub>1.5</sub> (2.7 μM) but with the improvement of maximum activation up to 647%. A block of either hydroxyl at the C-2 and C-4 positions (P1 part) using methyl group (<b>20c</b> and <b>20d</b>) was not beneficial for the potency, which was also observed in biphenyl analogues. Changing the bromide to electron-withdrawing groups such as a methylsulfonyl (<b>20e</b>) or a cyano group (<b>20f</b>) gave rise to a decrease of maximum activation. Appending the methoxy group (<b>20g</b>) increased the EC<sub>1.5</sub> to 0.9 μM and the maximum activation to 624%, which showed the highest activation potency among the compounds. Although the incorporation of furan (<b>20h</b>) and benzene (<b>20i</b>) at the C-6 position of the naphthalene (P2 part) maintained a comparable EC<sub>1.5</sub>, the maximum activation tended to decrease.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Structures and Activities of the Naphthohydrazones</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/medium/jm-2012-01032j_0016.gif" alt="" id="fx4" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/medium/jm-2012-01032j_0017.gif" alt="" id="fx5" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>Table a</sup><p class="last">EC<sub>1.5</sub> value represents the concentration of an activator required to increase the enzyme activity by 50%.</p></div><div class="footnote" id="t3fn2"><sup>Table b</sup><p class="last">Percentage of maximum activation achieved.</p></div><div class="footnote" id="t3fn3"><sup>Table c</sup><p class="last">None detected.</p></div></div><div></div></div><div class="NLM_p last">To examine the selectivity of the SIRT1 activators, we tested their activities on SIRT1 and its closest sirtuin homologues: SIRT2 and SIRT3. The compounds were tested at the concentration of 50 μM. The data for SIRT1, SIRT2, and SIRT3 were listed in <a class="ref internalNav" href="#notes-4" aria-label="Supporting Information Table S1">Supporting Information Table S1</a>. The EC<sub>1.5</sub> values for SIRT2 and SIRT3 of the compounds unmentioned in the following parentheses were greater than 50 μM (SIRT2: <b>8</b>, EC<sub>1.5</sub> = 16.5 μM, maximum activation = 206%. <b>20i</b>, EC<sub>1.5</sub> = 11.6 μM, maximum activation = 257%. SIRT3: <b>20i</b>, EC<sub>1.5</sub> = 28.6 μM, maximum activation = 199%.). The results showed that these compounds (<b>1</b>, <b>7</b>–<b>11</b>, <b>16a</b>–<b>n</b>, <b>20a</b>–<b>i</b>) are selective for the activation of SIRT1.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Compounds Promote the Deacetylation of Fluorophore-Labeled Peptide but Not the Unlabeled Peptide</h3><div class="NLM_p">Further, we tested the activities of <b>20b</b>, <b>1</b>, and <b>4</b> on fluorophore-unlabeled acetyl peptide. A known SIRT1 inhibitor, 6-chloro-2,3,4,9-tetrahydro-1<i>H</i>-carbazole-1-carboxamide (Ex527), was used as control.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Acetyl-Arg-His-Lys-Lys (ε-acetyl)-NH<sub>2</sub> peptide (ac-RHKK<sub>ac</sub>-NH<sub>2</sub>) was synthesized without 4-amino-7-methylcoumarin (AMC) labeled at the C-terminus. The deacetylation reactions of the unlabeled peptide were carried out in the same way as the AMC-labeled peptide. The amount of deacetylated peptide was determined by HPLC. Deacetylation enhancement by compound <b>20b</b> was observed with the AMC-labeled peptide but not the unlabeled peptide (<a class="ref internalNav" href="#notes-4" aria-label="Supporting Information Figure S3">Supporting Information Figure S3</a> and Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>). Inconsistent with previous reports,<a onclick="showRef(event, 'ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref25 ref26">(25, 26)</a><b>1</b> and <b>4</b> did not activate SIRT1 in deacetylating the fluorophore-unlabeled peptide. This phenomenon aroused our guess that the compounds do not bind directly with SIRT1 but bind to the “SIRT1/fluorophore-labeled peptide” complex. ITC experiments carried out by Milne et al. and Pacholec et al. also support the conjecture.<a onclick="showRef(event, 'ref22 ref26'); return false;" href="javascript:void(0);" class="ref ref22 ref26">(22, 26)</a> When <b>4</b> was titrated to SIRT1-substrate (Tamra-labeled peptide) mixture, the binding was observed.<a onclick="showRef(event, 'ref22 ref26'); return false;" href="javascript:void(0);" class="ref ref22 ref26">(22, 26)</a> However, no binding isotherm was observed when <b>4</b> was titrated to SIRT1 and fluorophore-unlabeled peptide solution.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Thus, we believe that the fluorophore linked to the peptide is the key element for the binding activity.</div><figure id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/medium/jm-2012-01032j_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. The Lineweaver–Burk plots of (A) ac-RHKK<sub>ac</sub>-AMC peptide and (B) ac-RHKK<sub>ac</sub>-NH<sub>2</sub> peptide in the presence of 50 μM <b>4</b> and 10 μM <b>20b</b>, which were determined by HPLC method. <b>4</b> and <b>20b</b> exhibited substrate selectivity against native peptide and fluorophore-labeled peptide.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm301032j&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Crystal Structure of ac-RHKK<sub>ac</sub>-AMC Peptide in Complex with SIRT3</h3><div class="NLM_p">To confirm the role of the fluorophore, we tried to determine the conformation of the fluorophore conjugated with the peptide inside the enzyme. Up to now, the structures of several sirtuin proteins were solved, including Hst2 and Sir2p of yeast, Sir2-Af1 and Sir2-Af2 of archaea, Sir2Tm of bacteria, and SIRT2, SIRT3, SIRT6, and SIRT7 of humans. However, the structure of SIRT1 is still uncovered due to the aggregation feature of the protein. Both sequence alignment and structure superposition of the sirtuin homologues revealed that they share high sequence homology as well as structure homology.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> This suggests that SIRT1 model may be built based on its homologue protein. According to the sequence alignment of SIRT1 and SIRT3 (<a class="ref internalNav" href="#notes-4" aria-label="Supporting Information Figure S4">Supporting Information Figure S4</a>), the two proteins showed very high sequence homology in the catalytic region. The differences between SIRT1 and SIRT3 sequences were at the flexible loop connecting the zinc-binding motif and Rossmann fold region (between α2 and α3), a turn in the zinc-binding motif (between α9 and β6), and a short α-helix on the Rossmann fold domain (α12). These amino acids are distributed mainly on the surface of SIRT3 protein, which are far away from the acetyl peptide binding site. Because our attention was focused on the peptide binding site, the catalytic core domain of SIRT1 could be built by using the structure of SIRT3 as template. Thus, SIRT3 was chosen for cocrystallization with the AMC-labeled acetyl peptide.</div><div class="NLM_p">The purified SIRT3 protein was incubated with ac-RHKK<sub>ac</sub>-AMC peptide at 4 °C overnight. The mixture was screened with Hampton Research kits by using the hanging-drop method at 15 °C. After two weeks, crystals were obtained with crystallization solution containing 0.085 M Hepes-Na (pH 7.5), 8.5% (v/v) isopropyl alcohol, 17% (w/v) polyethylene glycol (PEG) 4000, and 15% (v/v) glycerol anhydrous. The X-ray diffractions of the crystals were collected at Shanghai Synchrotron Radiation Facility. The data was processed by HKL2000<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> and ccp4 suite.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> The data collection and refinement statistics of the complex structure were listed in Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>. The space group of the complex is <i>R</i>32, which has one molecule in an asymmetric unit. Interestingly, the AMC-labeled peptides are involved in a “head-to-head” packing, which makes the complexes exhibiting a dimeric arrangement (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>A,B). The acetyl-lysine (ALY) of the substrate inserts into the cleft between the zinc-binding motif (small domain) and the Rossmann fold region (large domain). The coumarins of the two peptide substrates are in sandwich stacking, with the side chain of arginine from another substrate packing aside the fluorophore. Because of such substrate arrangement, the two protein molecules are packed in a little twisted “shoulder-in-shoulder” conformation. Moreover, the π-stacking was also observed in the side chains of Trp 353 of two protein molecules (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>A). Though the dimerization of the fluorophores was observed under the crystallization condition, the complex structure suggests that the coumarin may intact with other aromatic rings such as small molecules under physiological conditions in like manner. This is because the configuration of a single aromatic fluorophore is energetically unfavorable at physiological conditions due to its high hydrophobicity and lacking of significant contacts with the protein. The interaction with another aromatic ring may stabilize the configuration of the fluorophore.</div><figure id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/medium/jm-2012-01032j_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Structure of human SIRT3 in complex with substrate ac-RHKK<sub>ac</sub>-AMC. (A) Ribbon diagram of the overall structure of the complexes in dimer. One molecule of SIRT3 protein is shown in green and the other in cyan. The substrate is shown in a stick representation; carbon, oxygen, and nitrogen are colored in magentas, red, and blue, respectively. Zinc ions are highlighted in red. (B) An enlarged view of the binding site in dimeric arrangement. The coumarin fluorophores of the substrates are packed in a “head-in-head” conformation, and the two peptides are represented with the 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density maps (1.0 σ-contoured). The labeled amino acids are involved in the packing. (C) The detailed interaction of Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>B without map. The black dashed lines represent the H-bonds formed between the substrate and the residues. (D) An enlarged view of the active site in an asymmetric cell. Four hydrogen bonds (represented with black dashed lines) are formed between the substrate and the protein. Water molecules are represented with yellow spheres, and the H-bond interactions of waters are shown as yellow dashed lines. Loop A and loop B were indicated with arrows.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm301032j&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Data Collection and Refinement Statistics of “SIRT3/ac-RHKK<sub>ac</sub>-AMC” Complex</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col></colgroup><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">diffraction data</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">space group</td><td class="colsep0 rowsep0" align="left"><i>R</i>32</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">resolution (Å)<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">50–2.1 (2.18–2.10)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">unit cell parameters</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>a</i>, <i>b</i>, <i>c</i> (Å)</td><td class="colsep0 rowsep0" align="left">114.809, 114.809, 123.202</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">α, β, γ (deg)</td><td class="colsep0 rowsep0" align="left">90.00, 90.00, 120.00</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">completeness (%)<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">99.9 (100.0)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">redundancy<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">11.0 (11.1)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">average <i>I</i>/σ<i>I</i><a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">24.8 (4.4)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>merge</sub> (%)<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">10.0 (66.5)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">refinement statistics</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>working</sub> (%)</td><td class="colsep0 rowsep0" align="left">20.76</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>free</sub> (%)</td><td class="colsep0 rowsep0" align="left">28.19</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rmsd in bond length (Å)</td><td class="colsep0 rowsep0" align="left">0.0242</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rmsd in bond angles (deg)</td><td class="colsep0 rowsep0" align="left">2.0687</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">mean <i>B</i>-factors</td><td class="colsep0 rowsep0" align="left">45.762</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="2" align="left">Ramachandran plot</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">in preferred regions (%)</td><td class="colsep0 rowsep0" align="left">93.85</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">in allowed regions (%)</td><td class="colsep0 rowsep0" align="left">6.15</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">outliers (%)</td><td class="colsep0 rowsep0" align="left">0.00</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup>a</sup><p class="last">Values for the highest resolution shells are represented in parentheses.</p></div></div></div><div class="NLM_p">The interaction details around the coumarins of two complex molecules were shown in Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>C. As described above, the arginine of one substrate packs head-to-head with the coumarin from another substrate. The γ-guanido group of arginine is usually protonated at physiological conditions. It may form cation−π interaction with the coumarin and Phe 294. Also, the arginine forms an H-bond with the carboxyl of Glu 181 of another protein molecule. The alkyl side chain of the arginine and the N-terminal acetyl group of the peptide are in van der Waals interactions with Tyr 175 and Glu 177 from another protein molecule. Besides, the phenyl group of Phe 294 hangs over the AMC fluorophore in a T-shaped stacking and the packed fluorophores are sandwiched by the arginine and Phe 294 at each side.</div><div class="NLM_p last">As shown in the enlarged view of the binding site in monomer (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>D), the substrate is trapped by a loop (SIRT3–291–299) with its ALY protruding into the active site. The loop between β6 and α10 on the small domain was defined as loop A (<a class="ref internalNav" href="#notes-4" aria-label="Supporting Information Figure S4">Supporting Information Figure S4</a>). The residues on loop A formed four H-bonds with the peptide (Val 292, Gly 295, Glu 296, and Leu 298). These H-bonds are strong drive forces that entrap the substrate to the protein cavity. His 248 is conserved in mammal sirtuins and critical for the deacetylase activity.<a onclick="showRef(event, 'ref37 ref38 ref39'); return false;" href="javascript:void(0);" class="ref ref37 ref38 ref39">(37-39)</a> The acetyl group of the substrate lysine is in between Phe 180 and His 248. Meanwhile, residues of Ile 291 and Ile 230 (both are conserved in mammal sirtuins) surround the methyl group of the ALY through van der Waals interactions. In the structure, the binding cavity is filled with water molecules as shown in Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>D. The water not only formed H-bond interactions between each other but also interacted with residues Gln 228, His 248, and Glu 325 of SIRT3, the acetyl group of ALY, and the linker of coumarin.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Structure Comparison of SIRT3 in Complex with Different Acetyl Peptides</h3><div class="NLM_p">Jin et. al reported the structure of SIRT3 in complex with its substrate AcsCS2-K<sub>ac</sub> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GLR">3GLR</a>).<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> The sequence of the peptide is TRSGK<sub>ac</sub>VMRRLLR. Only the region of RSGK<sub>ac</sub>VMR was observed in the crystal structure. As for other sirtuin–peptide complex, the ALY of the substrate inserts into the cleft between the small and large domains. loop A as well as a loop on the large domain (SIRT3–323–326 in between β7 and α11, defined as loop B) and the peptides form an enzyme–substrate antiparallel β-sheet. The AcsCS2-K<sub>ac</sub> peptide pulls two loops closer to each other in comparison with those in the apo-SIRT3 structure (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GLS">3GLS</a>). This suggests that binding of the acetyl peptide induced a conformational change of the enzyme.</div><div class="NLM_p">To examine whether the binding of the fluorophore-labeled peptide may induce further conformational change of the cleft, we superimposed the “SIRT3/AMC-acetyl peptide” complex structure with the “SIRT3/AcsCS2-K<sub>ac</sub> peptide” complex structure. The overall structures of SIRT3 in complex with two peptides are similar. The root-mean-square deviation (rmsd) of the C<sub>α</sub> atoms of SIRT3 in the two complexes was 0.97 Å. As shown in Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>A, the main difference was at the cleft in between loop A and loop B. In the “SIRT3/ac-RHKK<sub>ac</sub>-AMC” structure, the distance between the two loops increased and the cleft adopted a more open conformation. We speculated that the wider formation of the cleft was due to the steric block of the coumarin. As shown in Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>B, the residues on loop B from the large domain did not form any direct interaction with the peptide. Only the carbonyl oxygen of Glu 325 had indirect connection with the peptide via the H-bond network of the water molecules (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>D). The vertical orientation of the coumarin seems to repel the hydrogen bonding between the substrate and the large domain, while for AcsCS2-K<sub>ac</sub> peptide in the SIRT3 complex, the residues after ALY formed three H-bonds with the residues on loop B (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>C). The comparison of the protein surface at the binding sites clearly showed that the clefts in the complexes adopted different conformations due to the features of the acetyl substrates (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>D,E).</div><figure id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/medium/jm-2012-01032j_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Structure comparison of SIRT3 protein in complex with AceCS2-K<sub>ac</sub> peptide (RSGK<sub>ac</sub>VMR) and ac-RHKK<sub>ac</sub>-AMC peptide. (A) The structure alignment of SIRT3 in complex with two peptides. The “SIRT3/ac-RHKK<sub>ac</sub>-AMC” structure is colored in green with the peptide in stick representation and carbon atoms colored in purple. The “SIRT3/AceCS2-K<sub>ac</sub>” structure is colored in slate with the carbon atoms of the peptide colored in orange. In the loop region (SIRT3–159–167), no density map was observed in the “SIRT3/ac-RHKK<sub>ac</sub>-AMC” structure. Loop A and loop B on the ALY binding site were indicated with arrows (red arrows for “SIRT3/AceCS2-K<sub>ac</sub>” and orange arrows for “SIRT3/ac-RHKK<sub>ac</sub>-AMC”). (B) An enlarged view of the binding site in the structure of “SIRT3/ac-RHKK<sub>ac</sub>-AMC”. The peptide forms four H-bonds with the residues on the small domain. (C) An enlarged view of the binding site in the structure of “SIRT3/AceCS2-K<sub>ac</sub>”. The peptide forms four H-bonds with the residues on loop A and three H-bonds with that on loop B. The distances between the acetyl-lysine and loop B of the “SIRT3/AceCS2-K<sub>ac</sub>” structure were shorter than that of the “SIRT3/ac-RHKK<sub>ac</sub>-AMC” structure. (D) A lateral view of the binding site in (B). The protein molecule is in electrostatic surface representation. (E) A lateral view of (D).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm301032j&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Complex Model of SIRT1 with ac-RHKK<sub>ac</sub>-AMC Peptide</h3><div class="NLM_p">To elucidate the activation mechanism of our compounds, we built a SIRT1 model using SIRT3 as the template by homology modeling. The sequence of SIRT1 (244–498) was aligned with SIRT3 (<a class="ref internalNav" href="#notes-4" aria-label="Supporting Information Figure S4">Supporting Information Figure S4</a>). In the “SIRT3/ac-RHKK<sub>ac</sub>-AMC” complex, no density map was observed in the region of SIRT3–158–167, which is a flexible loop on the protein surface. Thus, the structure at the regions of 121–157 and 168–391 of “SIRT3/ac-RHKK<sub>ac</sub>-AMC” complex and the loop region (158–167) of SIRT3-AcsCS2-K<sub>ac</sub> complex (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GLR">3GLR</a>) were used as the templates. After structural refinement, the coordinates of AMC-labeled peptide were copied from “SIRT3/ac-RHKK<sub>ac</sub>-AMC” complex to the SIRT1 model. The “SIRT1/ac-RHKK<sub>ac</sub>-AMC” complex model was submitted to further structural refinement.</div><div class="NLM_p">The complex model of SIRT1 was shown in Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>A. The cleft adopted an open conformation as the “SIRT3/ac-RHKK<sub>ac</sub>-AMC” structure did. As shown in Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>B, the peptide is captured by the loop of SIRT1–411–419 which corresponds to loop A. Hydrogen bond interactions were formed between the peptide and the backbone atoms of SIRT1 residues on loop A (Val 412, Gly 415, Glu 416, and Leu 418). The phenyl ring of Phe 414 is adjacent to the AMC fluorophore. The acetyl group of the lysine is sandwiched by the phenyl ring of Phe 297 and imidazole ring of His 363. The side chains of Ile 347 and Ile 411 form van der Waals interactions with the methyl group of the ALY. All the residues mentioned above are conserved in SIRT1 and SIRT3. The binding mode is similar with the SIRT3–substrate complex. The substrate did not have direct interaction with the residues from loop B.</div><figure id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/medium/jm-2012-01032j_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. The homology model of SIRT1 in complex with ac-RHKK<sub>ac</sub>-AMC peptide. (A) The overall structure of the complex model. SIRT1 is represented in cartoon and colored in silver. Loop A and loop B were indicated with arrows. (B) The binding site of the complex. The peptide forms four H-bonds with the backbone atoms on loop A.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm301032j&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">According to the sirtuin catalytic mechanism, the carbonyl oxygen at the side chain of ALY carries out nucleophilic attacks to the C1′ position of the <i>N</i>-ribose of NAD<sup>+</sup>, which initiates the deacetylation. After that, an <i>O</i>-alkylamidate intermediate was formed.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> We suppose that the broadening of the cleft and the relative position shift of residues involved in catalysis may hinder the hydrolysis step in deacetylation. The fluorophore, whether it is coumarin or TAMRA, certainly will cause steric block in the cleft due to its big and rigid body. The increase of the distance between the two loops and the loss of the H-bonds between the substrate and the large domain will weaken the substrate binding. Both the kinetic experiment and ITC titration demonstrated that the existence of coumarin increased the <i>K</i><sub>m</sub> and <i>K</i><sub>d</sub> values of the peptide to SIRT1. The <i>K</i><sub>m</sub> values of two acetyl substrates (with or without coumarin) were 551.0 and 130.5 μM, respectively (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>), and the <i>K</i><sub>d</sub> values calculated by titrating the peptide into SIRT1 were 184 and 44 μM, respectively (<a class="ref internalNav" href="#notes-4" aria-label="Supporting Information Figure S1">Supporting Information Figure S1</a>). On the basis of above analysis, we built the binding modes of the activators and proposed their activation mechanism.</div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> SIRT1 Activators Bind in the Broadened Cleft Caused by the Fluorophore</h3><div class="NLM_p">To explore the binding sites of the activators, compounds <b>20b</b>, <b>1</b>, and <b>4</b> were docked into “SIRT1/ac-RHKK<sub>ac</sub>-AMC” complex model. The poses of <b>20b</b>, <b>1</b>, and <b>4</b> were superimposed in the complex model as shown in Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>A. Three compounds bound in the hydrophobic pocket formed by the side chains of His, Arg, Lys, and coumarin of the peptide and the residues at the large domain of SIRT1. All the compounds apparently have π–π interactions with the coumarin.</div><figure id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/medium/jm-2012-01032j_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. The binding modes of SIRT1 activators with “SIRT1/ac-RHKK<sub>ac</sub>-AMC”. (A) A superimposed view of three small compounds that bind to the complex. The fluorophore-labeled peptide is colored in magentas; compounds <b>20b</b>, <b>1</b>, and <b>4</b> are colored in cyan, yellow, and green, respectively. (B) An enlarged view of <b>20b</b> binding mode. The nitrogen atom on the linker forms hydrogen bond with the peptide. The <i>p</i>-hydroxyl on the 3,5-dibromo-2,4-dihydroxyphenyl fragment H-bonds with the amino group on the side chain of Gln 345, while the <i>o</i>-hydroxyl H-bonds with the linker between the substrate and the coumarin. (C) An enlarged view of <b>4</b> binding mode. (D) An enlarged view of <b>1</b> binding mode. The two hydroxyls H-bond with the sides chains of Arg 446 and His of the substrate, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm301032j&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div id="sec2_8_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> Binding Mode of Diaryl Acylhydrazone Derivatives</h4><div class="NLM_p">In the “SIRT3/ac-RHKK<sub>ac</sub>-AMC” crystal structure, two coumarins form π-stacking; therefore, we speculated that the activators may stack with the coumarin in the similar manner. As shown in Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>B, the 6-bromonaphthalene fragment of <b>20b</b> interacts with the coumarin in a “head-to-head” conformation, while the dibromo-dihydroxybenzaldehyde ring inserts into the cleft under the fluorophore. We monitored the hydrogen bonds between <b>20b</b> and “SIRT1/ac-RHKK<sub>ac</sub>-AMC” complex; the <i>p</i>-hydroxyl on the dibromo-dihydroxybenzaldehyde ring interacts with the amino group from the side chain of Gln 345. According to the SAR, the hydroxyl is crucial in the activation, and methylation of the group resulted in a complete loss of the activity (compare <b>20b</b> and <b>20c</b>). Thus, the H-bond interaction is vital for the activation activity of our compounds. In the mechanism of sirtuin deacetylation, Gln 345 plays an important role. The carbonyl group on the side chain of Gln 345 may make an H-bond with 3′-OH of the ribose from NAD<sup>+</sup>.<a onclick="showRef(event, 'ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref42 ref43">(42, 43)</a> We speculated that the interaction between the compound and Gln 345 as well as other van der Waals interactions may stabilize the catalytic core and compensate the shifting induced by the coumarin; as a result, the <i>V</i><sub>max</sub> was accelerated. For the explanation of the <i>K</i><sub>m</sub> decrease (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>A), we preferred that the activators caused tighter binding between the fluorophore-labeled peptide and SIRT1. In the previous publications, <b>1</b> and <b>4</b> were also proved to decrease the <i>K</i><sub>m</sub> values of the fluorophore-labeled peptides.<a onclick="showRef(event, 'ref14 ref22 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref14 ref22 ref25 ref26">(14, 22, 25, 26)</a> As shown in Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>B, besides the aromatic stacking, the imino group of <b>20b</b> makes an H-bond with the substrate, the <i>o</i>-hydroxyl on the dibromo-dihydroxybenzaldehyde fragment also H-bonds with the substrate. Through above interactions, compound <b>20b</b> increased the binding affinity of the substrate to the enzyme; the existence of <b>20b</b> in the broadened cleft may stabilize the complex conformation and facilitate the following catalysis.</div><div class="NLM_p">We synthesized compounds with varying aromatic fragments at the P2 part (Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>, and <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>). The <i>m</i>-biphenyl substitution (<b>16d</b>) was less active than <i>p</i>-biphenyl (<b>7</b>). The orientation of the <i>m</i>-biphenyl is not favorable in the π-stacking with the coumarin. Introduction of electron-withdrawing group such as trifluoromethyl group (<b>16a</b>) and heterocyclic atoms oxygen (<b>16c</b>, <b>16h</b>) or nitrogen (<b>16i</b>, <b>16j</b>, and <b>16k</b>) to the biphenyl fragment maintained the activity as compared with <b>7</b>. The diaryl acylhydrazone derivatives with phenyl substitutions at P2 part exhibited weaker activities (<b>9</b>, <b>16l</b>, <b>16m</b>, and <b>16n</b>). In contrast, naphthalene substitutions significantly improved the activity. The substitution at the R<sub>1</sub> position of the naphthalene showed great tolerance (<b>20b</b>, <b>20e</b>, <b>20f</b>, <b>20g</b>, <b>20h</b>, and <b>20i</b>). The above SAR at the R<sub>1</sub> position is in line with our binding mode. Naphthalene substitution is preferred, and the substitutions on the naphthalene ring may improve the π-stacking.</div><div class="NLM_p">To verify the binding mode, residues Gln 345 and Val 445, which were shown to have close contacts with <b>20b</b>, were mutated for enzymatic activity and kinetic studies. In the binding mode, Gln 345 exhibited direct interaction with <b>20b</b> through the H-bond; in contrast, Val 445 showed much weaker interaction (van der Waals interaction) with <b>20b</b> (Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>B). Therefore, Gln 345 was mutated to alanine to eliminate the H-bond and Val 445 was mutated to alanine to alter the van der Waals interaction. The activation effects of compound <b>20b</b> to mutant proteins were shown in <a class="ref internalNav" href="#notes-4" aria-label="Supporting Information Figure S5">Supporting Information Figure S5</a> and listed in Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a>A. For SIRT1<sup>Q345A</sup>, <b>20b</b> is about 4-fold less active to SIRT1<sup>Q345A</sup> than to wild-type SIRT1 (for SIRT1, EC<sub>1.5</sub> = 2.7 μM; for SIRT1<sup>Q345A</sup>, EC<sub>1.5</sub> = 12.2 μM). <b>20b</b> could only activate the enzymatic activity of SIRT1<sup>Q345A</sup> by ∼2-fold at the highest dose. For SIRT1<sup>V445A</sup>, approximately 2-fold change of EC<sub>1.5</sub> was observed (for SIRT1<sup>V445A</sup>, EC<sub>1.5</sub> = 4.8 μM). <b>20b</b> could only activate the enzymatic activity of SIRT1<sup>V445A</sup> by ∼3-fold. Furthermore, the kinetic parameters (<i>K</i><sub>m</sub> and <i>V</i><sub>max</sub>/<i>K</i><sub>m</sub>) of the fluorophore-peptide were determined in the absence or presence of <b>20b</b>, respectively. For SIRT1 wild-type, addition of <b>20b</b> resulted in a decrease of 135 μM of the <i>K</i><sub>m</sub> value (without <b>20b</b>, <i>K</i><sub>m</sub> = 240.4 μM; with <b>20b</b>, <i>K</i><sub>m</sub> = 104.8 μM), while for SIRT1<sup>Q345A</sup> and SIRT1<sup>V445A</sup>, the amount of <i>K</i><sub>m</sub> decrease reduced to 79 and 78 μM, respectively. Besides, after adding 10 μM of <b>20b</b>, the ratio of <i>V</i><sub>max</sub>/<i>K</i><sub>m</sub> significantly decreased after mutation of Gln 345 and Val 445 (for SIRT1, <i>V</i><sub>max</sub>/<i>K</i><sub>m</sub> = 13.1; for SIRT1<sup>Q345A</sup>, <i>V</i><sub>max</sub>/<i>K</i><sub>m</sub> = 1.0; for SIRT1<sup>V445A</sup>, <i>V</i><sub>max</sub>/<i>K</i><sub>m</sub> = 3.3). From the above analysis, we can see that for SIRT1<sup>Q345A</sup> and SIRT1<sup>V445A</sup> mutants, <b>20b</b> could not lower the <i>K</i><sub>m</sub> or increase <i>V</i><sub>max</sub>/<i>K</i><sub>m</sub> as much as that for the wild-type SIRT1. The mutants reduced the sensitivities to the compound, indicating that the side chains of Gln 345 and Val 445 take part in the interactions with <b>20b</b>.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. The Kinetic Parameters of the ac-RHKK<sub>ac</sub>-AMC Peptide to SIRT1 Wild-Type and Mutant Proteins<a class="ref internalNav" href="#tbl5-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="." /></col><col align="char" char="±" /></col><col align="char" char="." /></col><col align="char" char="±" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="rowsep1 colsep0" colspan="7" align="center">(A)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char="±"> </th><th class="rowsep1 colsep0" colspan="3" align="center" char=".">compound <b>20b</b></th><th class="rowsep1 colsep0" colspan="2" align="center" char="±">blank</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char="±">EC<sub>1.5</sub>(μM)<a class="ref internalNav" href="#t5fn1" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">Max A (%)<a class="ref internalNav" href="#t5fn2" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center" char="±"><i>K</i><sub>m</sub>(μM)<a class="ref internalNav" href="#t5fn3" aria-label="d">d</a></th><th class="colsep0 rowsep0" align="center" char="."><i>V</i><sub>max</sub>/<i>K</i><sub>m</sub><a class="ref internalNav" href="#t5fn4" aria-label="e">e</a></th><th class="colsep0 rowsep0" align="center" char="±"><i>K</i><sub>m</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char="."><i>V</i><sub>max</sub>/<i>K</i><sub>m</sub></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SIRT1</td><td class="colsep0 rowsep0" align="char" char="±">2.7 ± 0.3</td><td class="colsep0 rowsep0" align="char" char=".">647</td><td class="colsep0 rowsep0" align="char" char="±">104.8 ± 8.8</td><td class="colsep0 rowsep0" align="char" char=".">13.1</td><td class="colsep0 rowsep0" align="char" char="±">240.4 ± 14.5</td><td class="colsep0 rowsep0" align="char" char=".">3.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SIRT1<sup>Q345A</sup></td><td class="colsep0 rowsep0" align="char" char="±">12.2 ± 0.6</td><td class="colsep0 rowsep0" align="char" char=".">237</td><td class="colsep0 rowsep0" align="char" char="±">184.0 ± 3.7</td><td class="colsep0 rowsep0" align="char" char=".">1.0</td><td class="colsep0 rowsep0" align="char" char="±">262.6 ± 5.1</td><td class="colsep0 rowsep0" align="char" char=".">0.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SIRT1<sup>V445A</sup></td><td class="colsep0 rowsep0" align="char" char="±">4.8 ± 0.2</td><td class="colsep0 rowsep0" align="char" char=".">353</td><td class="colsep0 rowsep0" align="char" char="±">191.0 ± 12.6</td><td class="colsep0 rowsep0" align="char" char=".">3.3</td><td class="colsep0 rowsep0" align="char" char="±">268.7 ± 22.9</td><td class="colsep0 rowsep0" align="char" char=".">1.8</td></tr></tbody></table><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="." /></col><col align="char" char="±" /></col><col align="char" char="." /></col><col align="char" char="±" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="rowsep1 colsep0" colspan="7" align="center">(B)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char="±"> </th><th class="rowsep1 colsep0" colspan="3" align="center" char=".">compound <b>1</b></th><th class="rowsep1 colsep0" colspan="2" align="center" char="±">blank</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char="±">EC<sub>1.5</sub> (μM)<a class="ref internalNav" href="#t5fn1" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">Max A (%)<a class="ref internalNav" href="#t5fn2" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center" char="±"><i>K</i><sub>m</sub> (μM)<a class="ref internalNav" href="#t5fn3" aria-label="d">d</a></th><th class="colsep0 rowsep0" align="center" char="."><i>V</i><sub>max</sub>/<i>K</i><sub>m</sub><a class="ref internalNav" href="#t5fn4" aria-label="e">e</a></th><th class="colsep0 rowsep0" align="center" char="±"><i>K</i><sub>m</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char="."><i>V</i><sub>max</sub>/<i>K</i><sub>m</sub></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SIRT1</td><td class="colsep0 rowsep0" align="char" char="±">23.6 ± 3.4</td><td class="colsep0 rowsep0" align="char" char=".">802</td><td class="colsep0 rowsep0" align="char" char="±">111.3 ± 26.7</td><td class="colsep0 rowsep0" align="char" char=".">11.4</td><td class="colsep0 rowsep0" align="char" char="±">240.4 ± 14.5</td><td class="colsep0 rowsep0" align="char" char=".">3.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SIRT1<sup>R446A</sup></td><td class="colsep0 rowsep0" align="char" char="±">87.2 ± 9.1</td><td class="colsep0 rowsep0" align="char" char=".">239</td><td class="colsep0 rowsep0" align="char" char="±">134.3 ± 14.6</td><td class="colsep0 rowsep0" align="char" char=".">4.4</td><td class="colsep0 rowsep0" align="char" char="±">256.2 ± 10.6</td><td class="colsep0 rowsep0" align="char" char=".">2.6</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl5-fn1"><div class="footnote" id="tbl5-fn1"><sup>a</sup><p class="last">(A) The experiments were carried out in the absence or presence of 10 μM compound <b>20b</b>, respectively. (B) The experiments were carried out in the absence or presence of 50 μM compound <b>1</b>, respectively.</p></div><div class="footnote" id="t5fn1"><sup>b</sup><p class="last">EC<sub>1.5</sub> value represents the concentration of an activator required to increase the enzyme activity by 50%.</p></div><div class="footnote" id="t5fn2"><sup>c</sup><p class="last">Percentage of maximum activation achieved.</p></div><div class="footnote" id="t5fn3"><sup>d</sup><p class="last"><i>K</i><sub>m</sub>, the concentration of substrate that results in half-<i>V</i><sub>max</sub> for an enzymatic reaction.</p></div><div class="footnote" id="t5fn4"><sup>e</sup><p class="last"><i>V</i><sub>max</sub>, the maximum initial velocity of an enzymatic reaction.</p></div></div></div></div><div id="sec2_8_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> Binding Mode of Imidazo[1,2-<i>b</i>]thiazole Derivatives</h4><div class="NLM_p"><b>4</b> and <b>5</b> belong to imidazo[1,2-<i>b</i>]thiazole derivatives. In 2009, the Sirtris group reported the activities of these derivatives with various amide groups.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> In Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a>, we listed 13 compounds from the publication to discuss their SAR in combination with our model. As shown in Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>C, the piperazonomethyl fragment inserts into the cleft and forms van der Waals interactions with the loop on the large domain. The imidazo[2,1-<i>b</i>][1,3]thiazole group lies under the coumarin. The surfaces of the two aromatic fragments form a T-shaped stacking. <b>4</b> forms a bend at the phenyl group, and the trimethoxybenzene group forms a sandwich conformation with coumarin and arginine of the substrate.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Structures and Activities of the Imidazo[1,2-<i>b</i>]thiazole Derivatives Identified by the Sirtris Group<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/medium/jm-2012-01032j_0018.gif" alt="" id="fx6" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/medium/jm-2012-01032j_0019.gif" alt="" id="fx7" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup>Table a</sup><p class="last">The structures and SIRT1 activity data of the compounds came from table 2 in the ref <a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">23</a>.</p></div><div class="footnote" id="t6fn2"><sup>Table b</sup><p class="last">EC<sub>1.5</sub> value represents the concentration of an activator required to increase the enzyme activity by 50%;</p></div><div class="footnote" id="t6fn3"><sup>Table c</sup><p class="last">Percentage of maximum activation achieved.</p></div></div><div></div></div><div class="NLM_p last">In the publication by the Sirtris group, it is said that an aromatic group might be favorable at the R position because the cyclopentyl (<b>23</b>) and benzyl (<b>24</b>) exhibited low activities.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> According to the binding mode of <b>4</b> (Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>C), the trimethoxyphenyl group forms π–π interaction with the coumarin in which the aromatic substitutions at R position are indeed preferred. Substitution of methoxy groups at the benzene ring improved the activities as compared <b>4</b> and <b>25</b> to <b>27</b>. However, the low activity of <b>26</b> indicated that 2,4-dimethoxybenzene substitution is not well tolerated. A significant level of SIRT1 activation was observed if substituting the benzene ring (<b>27</b>) with monocyclic pyrimidine substitution (<b>28</b>). Furthermore, 2-substituted quinoline substitution (<b>29</b>) improved the potency in comparison with <b>4</b>, and 3-substituted quinoline substitution (<b>30</b>) was slightly more active than <b>29</b>. In like manner, introduction of heteroatoms such as nitrogen in the aromatic ring strengthened the stabilization of π-stacking. Notably, the 2-qunixaline group substitution (<b>5</b>) resulted in a significant increase of the activity. Besides, the activities of the compounds are highly dependent on the position of the heteroatom. For instance, different positions of the oxygen atom resulted in ∼10-fold change of EC<sub>1.5</sub> (<b>32</b> and <b>33</b>). This may be explained by the reason that the position of the heteroatom could induce different electron distributions of the π-cloud, which may affect the π–π interaction between the compound and the fluorophore. Interestingly, changing the 2-quinoline group to the 8-quinoline group (<b>31</b>) led to a sharp loss of potency. In our model, the orientation of the 8-quinoline group is not favorable to form π-stacking with the coumarin.</div></div><div id="sec2_8_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> Binding Mode of <b>1</b></h4><div class="NLM_p">Like compound <b>20b</b> and <b>4</b>, <b>1</b> was predicted to bind in the same site. As shown in Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>D, the 4-hydroxyphenyl ring of <b>1</b> and the coumarin are packed in “head-to-head” conformation; the <i>p</i>-hydroxyl may form H-bond with the coumarin. The histidine of the substrate is connected to Arg 446 on the large domain by 3,5-dihydroxyphenyl of <b>1</b> through H-bond. To validate the model, the mutagenesis studies were performed. <b>1</b> activated SIRT1 with an EC<sub>1.5</sub> of 23.6 μM and ∼8-fold enzymatic activity (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>), which is consistent with previous studies.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> We mutated Arg 446 to alanine, and the activation curve against the mutant was also determined (<a class="ref internalNav" href="#notes-4" aria-label="Supporting Information Figure S6">Supporting Information Figure S6</a>). The maximum activation of <b>1</b> against SIRT1<sup>R446A</sup> exhibited a significant decrease than that of the wild-type, with only ∼2-fold enhancement. Meanwhile, the EC<sub>1.5</sub> value decreased to 87.2 μM. Nevertheless, compound <b>20b</b> showed no difference in activating SIRT1 wild-type and SIRT1<sup>R446A</sup> (<a class="ref internalNav" href="#notes-4" aria-label="Supporting Information Figure S6A">Supporting Information Figure S6A</a>). We assume that the side chain of Arg 446 may interact with <b>1</b> but not <b>20b</b>. For further validation, the kinetic parameters of ac-RHKK<sub>ac</sub>-AMC against SIRT1 wild-type and mutant with or without <b>1</b> were determined. As shown in Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a>B, the <i>K</i><sub>m</sub> and <i>V</i><sub>max</sub>/<i>K</i><sub>m</sub> of the substrate for SIRT1 wild-type and SIRT1<sup>R446A</sup> were similar; the mutation of the side chain did not change the binding affinity of the substrate. While in the presence of 50 μM <b>1</b>, the comparison of the kinetic parameters in wild-type (<i>K</i><sub>m</sub> = 111.3 μM, <i>V</i><sub>max</sub>/<i>K</i><sub>m</sub> = 11.4) and mutant (<i>K</i><sub>m</sub> = 134.3 μM, <i>V</i><sub>max</sub>/<i>K</i><sub>m</sub> = 4.4) indicated a reduced affinity of SIRT1 to the coumarin-labeled substrate. Notably, the ratio of <i>V</i><sub>max</sub>/<i>K</i><sub>m</sub> displayed about 3-fold decrease in the presence of <b>1</b> when the Arg 446 was mutated to alanine. The mutagenesis studies revealed that the side chain of Arg 446 interacted with the hydroxyl of <b>1</b>, and the binding was beneficial in the <b>1</b> activation.</div><div class="NLM_p last">On the basis of the above analysis, we may summarize the common features of the binding modes of the three SIRT1 activators. The aromatic portion of the SIRT1 activator forms π-stacking with the fluorophore. Heterocyclic or substituted aromatic compounds are preferred in the stacking, and SIRT1 activators with these fragments showed improved activities. Such observations were supported by the previous studies in π-stacking. It was reported that introduction of the heteroatom to benzene such as nitrogen could create a dipole and reduce the π-electron cloud in the spatial extent compared with benzene.<a onclick="showRef(event, 'ref45 ref46 ref47'); return false;" href="javascript:void(0);" class="ref ref45 ref46 ref47">(45-47)</a> The substituted benzenes form stronger π-stacking than the unsubstituted ones.<a onclick="showRef(event, 'ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref48 ref49">(48, 49)</a> Furthermore, the other portion of the activators may insert into the cleft under the fluorophore in a T-shaped stacking or in an extended conformation, which interact with residues from the large domain of SIRT1. The two portions of the activators may stabilize the substrate binding and promote the following catalysis. It is worth mentioning that the linker between the two portions is usually composed of 2–4 carbon/nitrogen atoms. The length of the linker is appropriate to allow the two portions in the proper conformations to interact with the substrate or SIRT1 residues.</div></div></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> The Stability, Bioavailability, and Activity of Diaryl Acylhydrazone Derivatives in Cells</h3><div class="NLM_p">Apart from the above in vitro studies, we also tested the stability and bioavailability of our compound in cells. MCF-7 cells were treated with 100 μM <b>20b</b> and the cultured medium and cells were collected separately at different time points. Simultaneously, 100 μM <b>20b</b> was added to fresh medium to serve as control. In Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a>, squares represent the concentrations of <b>20b</b> in fresh medium at different time points (Control). Circles represent the total amount of <b>20b</b> in cultured medium and cells (Sample). Triangles represent the amount of <b>20b</b> in cells (In cells). In comparison of the two curves (Control and Sample), <b>20b</b> remained stable in fresh medium for 120 min (squares), while in cultured medium the concentration of <b>20b</b> started to decrease after 30 min and it decreased to ∼20 μM after 360 min in cell cultures (circles). These results indicated that the compound is stable in cells for a time period. As for the bioavailability, <b>20b</b> in cells achieved the maximum concentration at 30 min (triangles), at which over half of the compound could get into the cells.</div><figure id="fig8" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/medium/jm-2012-01032j_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Stability test of compound <b>20b</b>. “Control” (squares) represents the amount of <b>20b</b> in fresh cell medium. “Sample” (circles) represents the total amount of <b>20b</b> in cultured medium and cells. “In cells” (triangles) represents the amount of <b>20b</b> in cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm301032j&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Thus, the functional activity of our compound was further explored in a cell-based (MCF-7 cells) deacetylation assay.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> In the experiment, p53 becomes acetylated at Lys 382 after DNA damage, which was induced by doxorubicin (DOX).<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> To determine the effect of compound <b>20b</b> on SIRT1 activity, trichostatin (TSA) was used to inhibit other classes of deacetylases. A reported SIRT1 activator, <b>1</b>, was used as control. As shown in Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a>, <b>1</b> showed no effect on the acetyl-p53 level at the concentration lower than 100 μM. The acetyl-p53 level decreased along with the increase of the concentration of <b>1</b>. In the sample panel, the acetyl-p53 level decreased along with the increase amount of <b>20b</b>. However, no effect was detected when the concentration of <b>20b</b> decreased to 60 μM. The high concentrations of our compound used in the assay may be explained by the stability and bioavailability of the compound. In the Western blot, the cells were treated with the compounds for 6 h. The stability assay showed that the concentration of <b>20b</b> decreased from 100 to 20 μM after 6 h treatment in cells (Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a>). In vitro, compound <b>20g</b> showed a comparable activation activity as <b>20b</b> (Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>). Consistently, the Western blot results indicated that <b>20g</b> also decreased the acetyl-p53 on cell level (Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a>). <b>20g</b> was slightly more effective than <b>20b</b>. Both <b>20b</b> and <b>20g</b> showed better activities than <b>1</b>.</div><figure id="fig9" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/medium/jm-2012-01032j_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Western blot analysis of compounds <b>1</b>, <b>20b</b>, and <b>20g</b> on MCF-7 cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm301032j&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Implications in the Discovery of Novel SIRT1 Activators</h3><div class="NLM_p">The aim of this study is to understand the structural basis by which the SIRT1 activators promote the deacetylation. The study revealed that the acceleration of the catalysis was dependent on the structural features of the substrates. For a specific acetyl substrate (X<sub>1</sub>···X<sub>n</sub>)-K<sub>ac</sub>-Y, if residue Y comprises an aromatic ring, such as Phe, His, Tyr, or Trp, the steric bulk of Y would broaden the cleft of the catalysis domain as the fluorophore did and decrease the binding affinity of the substrate. In this case, SIRT1 activators may enhance the binding of substrate and accelerate the deacetylation. Studies by Dai et al. provided evidence to support the notion.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a><b>4</b> was found to activate the deacetylation of ac-RHKK<sub>ac</sub>-F-NH<sub>2</sub> and ac-RHKK<sub>ac</sub>-W-NH<sub>2</sub> in a dose–response way but not on the substrate ac-RHKK<sub>ac</sub>-A-NH<sub>2</sub>.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> However, in contrast with the inhibitory effect of <b>4</b> on ac-RHKK<sub>ac</sub>-AMC reported by Dai et al., <b>4</b> was observed to activate the deacetylation of AMC-labeled peptide in our study. Thus, in search of SIRT1 activators for a specific substrate, the activation mechanism presented here may be broadly applicative. For instance, among the five acetylated lysine in human FOXO3,<a onclick="showRef(event, 'ref7 ref51'); return false;" href="javascript:void(0);" class="ref ref7 ref51">(7, 51)</a> three of the ε-acetylated lysines (Lys 259, Lys 290, and Lys 569) are followed by residues bearing aromatic rings (Tyr 260, Trp 291, and His 570). The reported SIRT1 activators may accelerate the deacetylation of native FOXO3. DNA methyltransferase 1(DNMT1) can be deacetylated by SIRT1 and a Phe is linked to the C-terminus of acetyl-Lys 1349,<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> and the reported compounds might also have activation effect on DNMT1. The substrate feature-dependent activation of SIRT1 activators may explain the controversial results due to the diversity of SIRT1 substrates.</div><div class="NLM_p last">In addition, the structure of “SIRT3/AcsCS2-K<sub>ac</sub>” complex implicated that the binding of peptide induced the change of protein conformation, and the conformational change is beneficial for the following catalysis. In light of the binding modes of the above three SIRT1 activators, one can design compounds which bind directly with the substrate and the residues around the peptide binding site, such as Phe 414 from the small domain and Arg 446 from the large domain (Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>). The small molecules targeting the binding site could link the two domains and decrease the <i>K</i><sub>m</sub> value of the substrate.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i25">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63563" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63563" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In this paper, we discovered a new class of small compounds that activate SIRT1 deacetylation activity by a fluorescence method. In the process of this project, it was reported that some putative activators did not directly bind to SIRT1. We then tried to understand the reason for the phenomenon. First, we determined the substrate selectivity of the diaryl acylhydrazone derivatives and <b>4</b> by HPLC. Through enzyme kinetics studies, <b>20b</b> and <b>4</b> were confirmed to decrease the <i>K</i><sub>m</sub> values of the fluorophore-labeled peptide, while none of them affected the unlabeled peptide. After that, we tried to capture the structure of the fluorophore-labeled peptide in the enzyme and determine the changes of enzyme conformation induced by the fluorophore. To our delight, the cocrystal structure of SIRT3 in complex with ac-RHKK<sub>ac</sub>-AMC was obtained. In comparison with the structure of SIRT3 in complex with fluorophore-unlabeled peptide, the fluorophore was found to expand the substrate binding cleft and break the hydrogen bonds between the substrate and the loop on the Rossmann fold domain. A π-stacking of two coumarins were observed in a dimeric arrangement of the SIRT3 complexes. Our enzyme kinetics experiments and ITC data implied that the activators could be involved in π-stacking with the fluorophore. Computational analysis was then carried out to model the SIRT1 structure based on the “SIRT3/ac-RHKK<sub>ac</sub>-AMC” complex structure and dock the small molecule activators into the SIRT1 model. The binding modes of <b>20b</b>, <b>1</b>, and <b>4</b> were consistent with the SAR and mutagenesis results.</div><div class="NLM_p last">Next, we put forward the activation mechanism of SIRT1 activators on the basis of the complex structures. For a SIRT1 activator, one portion of its structure forms π-stacking with the fluorophore of the substrate and the other portion may interact with the large domain of SIRT1. The small molecules link the substrate and the large domain together to compensate for the broadened cleft induced by the fluorophore and thus tighten the binding of the substrate. Moreover, we examined the activity, stability, and bioavailability of our compound in cells, which suggests that the compounds may be used as potentially interesting probes to study the functions of SIRT1. In summary, we elaborated the activation mechanism of the activators from the structural point. The complex structures and the mechanism described here provide the basis for development of novel SIRT1 activators and the studies of sirtuin activator activities in vivo and in vitro.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i26">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64057" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64057" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Material</h3><div class="NLM_p last">Unless otherwise noted, the reagents (chemicals) were purchased from commercial sources and used without further purification. Analytical thin layer chromatography (TLC) was performed on HSGF 254 (0.15–0.2 mm thickness). Compound spots were visualized by UV light (254 nm). Column chromatography was performed on silica gel FCP 200–300. Mircrowave reactions were carried on CEM reactor. Nuclear magnetic resonance (NMR) spectra were recorded on a Bruker AMX-300 NMR 300 or 400 MHz (TMS as IS). Chemical shifts were reported in parts per million (ppm, δ) downfield from tetramethylsilane. Proton coupling patterns are described as singlet (s), doublet (d), triplet (t), quartet (q), multiplet (m), and broad (br). Low- and high-resolution mass spectra (LRMS and HRMS) were obtained with electric, electrospray, and matrix-assisted laser desorption ionization (EI and ESI) produced by FinniganMAT-95 and LCQ-DECA spectrometers. HPLC analysis was conducted for all compounds listed in Tables <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>–<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a> on an Agilent 1100 series HPLC with an Agilent Zorbax Eclipse XDB-C18 (4.6 mm × 50 mm, 5 μm) reversed phase column with two solvent systems (MeCN/H<sub>2</sub>O/TFA and MeOH/H<sub>2</sub>O/TFA). All the assayed compounds displayed a purity of 95–99% in both solvent systems.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> General Procedure for Preparation of Esters <b>13a</b>–<b>k</b> and <b>21h</b>–<b>i</b></h3><div class="NLM_p">A vial equipped with a magnetic stir bar was charged with Pd(PPh<sub>3</sub>)<sub>4</sub> (0.05 mmol), KF·2H<sub>2</sub>O (3.0 mmol), aryl bromide (1.0 mol), aryl boronic acid (1.2–1.5 mmol), and 4 mL of DME/H<sub>2</sub>O/EtOH (7/3/2, v/v/v). Then the vial was capped with septa and heated at 110 °C for 50 min under microwave irradiation. The reaction mixture was extracted with EA. The organic layer was washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and concentrated under vacuum. The crude product was purified by flash column chromatography to afford methyl esters <b>13a</b>–<b>k</b> and <b>21h</b>–<b>i</b>.</div><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> Methyl 4-[2-(Trifluoromethyl)phenyl]benzoate <b>13a</b></h4><div class="NLM_p last">Yield: 55%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.10–8.06 (m, 2H), 7.76 (d, <i>J</i> = 8.0 Hz, 1H), 7. (t, <i>J</i> = 7.6 Hz, 1H), 7.51 (t, <i>J</i> = 7.6 Hz, 1H), 7.41 (d, <i>J</i> = 8.0 Hz, 2H), 7.32 (d, <i>J</i> = 7.6 Hz, 1H), 3.95 (s, 3H).</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> Methyl 4-(5-Cyano-2-hydroxyphenyl)benzoate <b>13b</b></h4><div class="NLM_p last">Yield: 36%. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.17 (d, <i>J</i> = 8.4 Hz, 2H), 7.61–7.52 (m, 4H), 7.06 (d, <i>J</i> = 9.0, 1H), 6.06 (br, 1H), 3.96 (s, 3H).</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> Methyl 4-(2<i>H</i>-1,3-Benzodioxol-5-yl)benzoate <b>13c</b></h4><div class="NLM_p last">Yield: 83%. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.07 (d, <i>J</i> = 8.4 Hz, 2H), 7.58 (d, <i>J</i> = 8.4 Hz, 2H), 7.13–7.08 (m, 2H), 6.90 (d, <i>J</i> = 8.7 Hz, 1H), 6.02 (s, 2H), 3.93 (s, 3H).</div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> Methyl 3-Phenylbenzoate <b>13d</b></h4><div class="NLM_p last">Yield: 77%. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.29 (s, 1H), 8.02 (d, <i>J</i> = 7.8 Hz, 1H), 7.79 (d, <i>J</i> = 7.8 Hz, 1H), 7.63 (d, <i>J</i> = 7.2 Hz, 2H), 7.55–7.43 (m, 3H), 7.38 (t, <i>J</i> = 7.8 Hz, 1H), 3.95 (s, 3H).</div></div><div id="sec4_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> Methyl 4-(Furan-2-yl)benzoate <b>13h</b></h4><div class="NLM_p last">Yield: 64%. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.05 (d, <i>J</i> = 8.4 Hz, 2H), 7.73 (d, <i>J</i> = 8.4 Hz, 2H), 7.52 (d, <i>J</i> = 1.8 Hz, 1H), 6.79 (d, <i>J</i> = 3.6 Hz, 1H), 6.51 (dd, <i>J</i> = 1.8, 3.6 Hz, 1H), 3.92 (s, 3H).</div></div><div id="sec4_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> Methyl 4-(Pyridin-4-yl)benzoate <b>13i</b></h4><div class="NLM_p last">Yield: 79%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.71 (d, <i>J</i> = 6.0 Hz, 2H), 8.16 (d, <i>J</i> = 8.4 Hz, 2H), 7.71 (d, <i>J</i> = 8.4 Hz, 2H), 7.56 (d, <i>J</i> = 6.0 Hz, 2H), 3.96 (s, 3H).</div></div><div id="sec4_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> Methyl 4-(Pyridin-3-yl)benzoate <b>13j</b></h4><div class="NLM_p last">Yield: 68%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.89 (d, <i>J</i> = 2.0 Hz, 1H), 8.65 (dd, <i>J</i> = 1.2, 4.8 Hz, 1H), 8.15 (d, <i>J</i> = 8.4 Hz, 2H), 7.95–7.92 (m, 1H), 7.69–7.63 (d, <i>J</i> = 8.4 Hz, 2H), 7.42 (dd, <i>J</i> = 4.8, 8.0 Hz, 1H), 3.95 (s, 3H).</div></div><div id="sec4_2_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> Methyl 4-(Pyridin-2-yl)benzoate <b>13k</b></h4><div class="NLM_p last">Yield: 42%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.89 (d, <i>J</i> = 2.0 Hz, 1H), 8.65 (dd, <i>J</i> = 1.2, 4.8 Hz, 1H), 8.15 (d, <i>J</i> = 8.4 Hz, 2H), 7.95–7.92 (m, 1H), 7.69–7.63 (d, <i>J</i> = 8.4 Hz, 2H), 7.42 (dd, <i>J</i> = 4.8, 8.0 Hz, 1H), 3.95 (s, 3H).</div></div><div id="sec4_2_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> Methyl 6-(Furan-2-yl)naphthalene-2-carboxylate <b>21h</b></h4><div class="NLM_p last">Yield: 63%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.62 (s, 1H), 8.30 (s, 1H), 8.18 (d, <i>J</i> = 8.7 Hz, 1H), 8.08 (d, <i>J</i> = 9.0 Hz, 1H), 8.01–7.96 (m, 2H), 7.87 (d, <i>J</i> = 1.5 Hz, 1H), 7.20 (d, <i>J</i> = 3.6 Hz, 1H), 6.69 (dd, <i>J</i> = 1.5, 3.3 Hz, 1H), 3.92 (s, 3H).</div></div><div id="sec4_2_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> Methyl 6-Phenylnaphthalene-2-carboxylate <b>21i</b></h4><div class="NLM_p last">Yield: 53%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.68 (s, 1H), 8.33 (s, 1H), 8.24 (d, <i>J</i> = 8.7 Hz, 1H), 8.12 (d, <i>J</i> = 8.7 Hz, 1H), 8.04–7.94 (m, 2H), 7.86 (d, <i>J</i> = 7.5 Hz, 2H), 7.54 (t, <i>J</i> = 7.5 Hz, 2H), 7.44 (t, <i>J</i> = 7.5 Hz, 1H), 3.93 (s, 3H).</div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> 6-Bromonaphthalene-2-carbohydrazide <b>19b</b>. Representive Procedure for <b>14a</b>–<b>k</b>, <b>19a</b>–<b>e</b>, and <b>22h</b>–<b>i</b></h3><div class="NLM_p">To a solution of methyl 6-bromonaphthalene-2-carboxylate (513 mg, 1.94 mmol) in 9 mL of EtOH was added 3 mL of hydrazine hydrate (85%), and the resulting mixture was stirred under reflux for 3 h. After cooling to room temperature, the precipitate was filtered, washed with water, and dried to afford 6-bromonaphthalene-2-carbohydrazide as a white solid in 81% yield. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.95 (s, 1H), 8.44 (s, 1H), 8.28 (s, 1H), 8.03–7.94 (m, 3H), 7.71 (dd, <i>J</i> = 1.8, 8.7 Hz, 1H), 4.58 (s, 2H).</div><div id="sec4_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> 4-[2-(Trifluoromethyl)phenyl]benzohydrazide <b>14a</b></h4><div class="NLM_p last">Yield: 75%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.87 (s, 1H), 7.89–7.84 (m, 3H), 7.74 (t, <i>J</i> = 7.6 Hz, 1H), 7.64 (t, <i>J</i> = 7.6 Hz, 1H), 7.43–7.38 (m, 3H), 4.53 (s, 2H).</div></div><div id="sec4_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> 4-(5-Cyano-2-hydroxyphenyl)benzohydrazide <b>14b</b></h4><div class="NLM_p last">Yield: 72%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.94 (s, 1H), 9.82 (s, 1H), 7.86 (d, <i>J</i> = 8.4 Hz, 2H), 7.76 (d, <i>J</i> = 2.0 Hz, 1H), 7.69–7.63 (m, 3H), 7.10 (d, <i>J</i> = 8.4 Hz, 1H), 4.55 (s, 2H).</div></div><div id="sec4_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> 4-(2<i>H</i>-1,3-Benzodioxol-5-yl)benzohydrazide <b>14c</b></h4><div class="NLM_p last">Yield: 78%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.79 (s, 1H), 7.87 (d, <i>J</i> = 8.4 Hz, 2H), 7.68 (d, <i>J</i> = 8.4 Hz, 2H), 7.32 (d, <i>J</i> = 1.8 Hz, 1H), 7.22 (dd, <i>J</i> = 1.8, 8.1 Hz, 1H), 7.01 (d, <i>J</i> = 8.1 Hz, 1H), 6.07 (s, 2H), 4.54 (br, 2H).</div></div><div id="sec4_3_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> 3-Phenylbenzohydrazide <b>14d</b></h4><div class="NLM_p last">Yield: 85%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.89 (s, 1H), 8.10 (s, 1H), 7.81 (dd, <i>J</i> = 1.6, 7.6 Hz, 2H), 7.74 (d, <i>J</i> = 7.2 Hz, 2H), 7.56–7.48 (m, 3H), 7.40 (t, <i>J</i> = 7.6 Hz, 1H), 4.55 (s, 2H).</div></div><div id="sec4_3_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> 4-Phenylbenzohydrazide <b>14e</b></h4><div class="NLM_p last">Yield: 88%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.84 (s, 1H), 7.92 (d, <i>J</i> = 8.8 Hz, 2H), 7.76–7.71 (m, 4H), 7.49 (t, <i>J</i> = 7.6 Hz, 2H), 7.40 (t, <i>J</i> = 7.6 Hz, 1H), 4.52 (s, 2H).</div></div><div id="sec4_3_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> 4-(Furan-2-yl)benzohydrazide <b>14h</b></h4><div class="NLM_p last">Yield: 82%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.80 (s, 1H), 7.87 (d, <i>J</i> = 8.4 Hz, 2H), 7.81 (d, <i>J</i> = 0.8 Hz, 1H), 7.76 (d, <i>J</i> = 8.4 Hz, 2H), 7.09 (d, <i>J</i> = 3.6 Hz, 1H), 6.63 (dd, <i>J</i> = 1.6, 3.6 Hz, 1H), 4.53 (s, 2H).</div></div><div id="sec4_3_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> 4-(Pyridin-4-yl)benzohydrazide <b>14i</b></h4><div class="NLM_p last">Yield: 74%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.89 (s, 1H), 8.66 (dd, <i>J</i> = 1.5, 6.0 Hz, 2H), 7.97 (d, <i>J</i> = 8.4 Hz, 2H), 7.89 (d, <i>J</i> = 8.4 Hz, 2H), 7.76 (dd, <i>J</i> = 1.5, 6.0 Hz, 2H), 4.54 (s, 2H).</div></div><div id="sec4_3_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> 4-(Pyridin-3-yl)benzohydrazide <b>14j</b></h4><div class="NLM_p last">Yield: 79%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.87 (s, 1H), 8.95 (d, <i>J</i> = 2.0 Hz, 1H), 8.60 (dd, <i>J</i> = 2.0, 4.8 Hz, 1H), 8.17–8.11 (m, 1H), 7.95 (d, <i>J</i> = 8.4 Hz, 2H), 7.83 (d, <i>J</i> = 8.4 Hz, 2H), 7.51 (dd, <i>J</i> = 4.8, 7.2 Hz, 1H), 4.53 (s, 2H).</div></div><div id="sec4_3_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> 4-(Pyridin-2-yl)benzohydrazide <b>14k</b></h4><div class="NLM_p last">Yield: 80%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.87 (s, 1H), 8.69 (d, <i>J</i> = 4.8 Hz, 1H), 8.16 (d, <i>J</i> = 8.4 Hz, 2H), 8.04 (d, <i>J</i> = 8.0 Hz, 1H), 7.98–7.88 (m, 3H), 7.41–7.37 (m, 1H), 4.56 (s, 2H).</div></div><div id="sec4_3_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> Quinoline-6-carbohydrazide <b>19a</b></h4><div class="NLM_p last">Yield: 83%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.01 (s, 1H), 8.97 (dd, <i>J</i> = 1.6, 4.0 Hz, 1H), 8.51–8.43 (m, 2H), 8.15 (dd, <i>J</i> = 1.6, 8.8 Hz, 1H), 8.06 (d, <i>J</i> = 8.8 Hz, 1H), 7.60 (dd, <i>J</i> = 4.0, 8.4 Hz, 1H), 4.68 (br, 2H).</div></div><div id="sec4_3_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> 6-Methanesulfonylnaphthalene-2-carbohydrazide <b>19c</b></h4><div class="NLM_p last">Yield: 84%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.06 (s, 1H), 8.64 (s, 1H), 8.56 (s, 1H), 8.30–8.26 (m, 2H), 8.07–7.99 (m, 2H), 4.68 (s, 2H).</div></div><div id="sec4_3_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> 6-Cyanonaphthalene-2-carbohydrazide <b>19d</b></h4><div class="NLM_p last">Yield: 93%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.06 (s, 1H), 8.64 (s, 1H), 8.52 (s, 1H), 8.21 (d, <i>J</i> = 8.4 Hz, 1H), 8.13 (d, <i>J</i> = 8.7 Hz, 1H), 8.04 (dd, <i>J</i> = 1.5, 8.4 Hz, 1H), 7.86 (dd, <i>J</i> = 1.5, 8.4 Hz, 1H), 4.63 (s, 2H).</div></div><div id="sec4_3_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> 6-Methoxynaphthalene-2-carbohydrazide <b>19e</b></h4><div class="NLM_p last">Yield: 91%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.82 (s, 1H), 8.35 (s, 1H), 7.91 (d, <i>J</i> = 9.0 Hz, 1H), 7.86 (s, 2H), 7.37 (d, <i>J</i> = 2.4 Hz, 1H), 7.22 (dd, <i>J</i> = 2.4, 9.0 Hz, 1H), 4.54 (br, 2H), 3.90 (s, 3H).</div></div><div id="sec4_3_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> 6-(Furan-2-yl)naphthalene-2-carbohydrazide <b>22h</b></h4><div class="NLM_p last">Yield: 88%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.93 (s, 1H), 8.40 (s, 1H), 8.26 (s, 1H), 8.06–8.00 (m, 2H), 7.95–7.89 (m, 2H), 7.85 (d, <i>J</i> = 1.5 Hz, 1H), 7.16 (d, <i>J</i> = 3.3 Hz, 1H), 6.67 (dd, <i>J</i> = 1.8, 3.3 Hz, 1H), 4.57 (s, 2H).</div></div><div id="sec4_3_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> 6-Phenylnaphthalene-2-carbohydrazide <b>22i</b></h4><div class="NLM_p last">Yield: 84%. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.95 (s, 1H), 8.46 (s, 1H), 8.28 (s, 1H), 8.10 (d, <i>J</i> = 8.7 Hz, 1H), 8.05 (d, <i>J</i> = 10.0, Hz, 1H), 7.96–7.89 (m, 2H), 7.84 (d, <i>J</i> = 7.5 Hz, 2H), 7.53 (t, <i>J</i> = 7.5 Hz, 2H), 7.43 (t, <i>J</i> = 7.5 Hz, 1H), 4.57 (s, 2H).</div></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> 6-Bromo-<i>N</i>′-(3,5-dibromo-2,4-dihydroxybenzylidene)-2-naphthohydrazide <b>20b</b>. Representive Procedure for Compounds <b>16a</b>–<b>n</b> and <b>20a</b>–<b>i</b></h3><div class="NLM_p">6-Bromonaphthalene-2-carbohydrazide (100 mg, 0.38 mmol) and 3,5-dibromo-2,4-dihydroxybenzaldehyde (117 mg, 0.40 mmol) were suspended in 3 mL of EtOH/HAc (20/1, v/v). The resulting mixture was stirred for 3 h under reflux. The precipitate was collected by hot filtration and rinsed with EtOH to afford <b>20b</b> as a white solid in 72% yield. HPLC: 96.84%, <i>t</i><sub>R</sub> = 10.71. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.92 (s, 1H), 12.56 (s, 1H), 10.41 (br, 1H), 8.58 (s, 1H), 8.51 (s, 1H), 8.33 (s, 1H), 8.08–8.03 (m, 3H), 7.81–7.74 (m, 2H). EI-MS <i>m</i>/<i>z</i> (M<sup>+</sup>) 540. EI-HRMS calcd for (M<sup>+</sup>) 539.8320, found 539.8250.</div><div id="sec4_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> <i>N</i>′-(3,5-Dibromo-2,4-dihydroxybenzylidene)-2′-(trifluoromethyl)biphenyl-4-carbohydrazide <b>16a</b></h4><div class="NLM_p last">Yield: 56%. HPLC: 99.28%, <i>t</i><sub>R</sub> = 10.18. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.91 (s, 1H), 12.45 (s, 1H), 10.41 (s, 1H), 8.49 (s, 1H), 8.00 (d, <i>J</i> = 8.4 Hz, 2H), 7.88 (d, <i>J</i> = 7.6 Hz, 1H), 7.80–7.75 (m, 2H), 7.67 (t, <i>J</i> = 7.8 Hz, 1H), 7.51 (d, <i>J</i> = 8.0 Hz, 2H), 7.46 (d, <i>J</i> = 7.6 Hz, 1H). EI-MS <i>m</i>/<i>z</i> (M<sup>+</sup>) 556. EI-HRMS calcd for (M<sup>+</sup>) 555.9245, found 555.9200.</div></div><div id="sec4_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> 4-(5-Cyano-2-hydroxyphenyl)-<i>N</i>′-(3,5-dibromo-2,4-dihydroxybenzylidene)benzohydrazide <b>16b</b></h4><div class="NLM_p last">Yield: 62%. HPLC: 99.04%, <i>t</i><sub>R</sub> = 8.85. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.88 (s, 1H), 12.32 (s, 1H), 10.91 (s, 1H), 9.86 (br, 1H), 8.50 (s, 1H), 8.00 (d, <i>J</i> = 8.1 Hz, 2H), 7.81–7.73 (m, 4H), 7.67 (dd, <i>J</i> = 2.1, 8.4 Hz, 1H), 7.12 (d, <i>J</i> = 8.4 Hz, 1H). MS (ESI, <i>m</i>/<i>z</i>): 528.0 [M – H]<sup>−</sup>. HRMS (ESI) calcd for C<sub>21</sub>H<sub>13</sub>Br<sub>2</sub>N<sub>3</sub>O<sub>4</sub>Na ([M + Na]<sup>+</sup>) 551.9170; found 551.9160.</div></div><div id="sec4_4_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> 4-(Benzo[<i>d</i>][1,3]dioxol-5-yl)-<i>N</i>′-(3,5-dibromo-2,4-dihydroxybenzylidene)benzohydrazide <b>16c</b></h4><div class="NLM_p last">Yield: 73%. HPLC: 96.09%, <i>t</i><sub>R</sub> = 6.89. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.94 (s, 1H), 12.38 (s, 1H), 10.42 (br, 1H), 8.48 (s, 1H), 8.00 (d, <i>J</i> = 8.1 Hz, 2H), 7.85–7.73 (m, 3H), 7.37 (s, 1H), 7.27 (d, <i>J</i> = 8.1 Hz, 1H), 7.04 (dd, <i>J</i> = 1.5, 8.1 Hz, 1H), 6.09 (s, 2H). MS (ESI, <i>m</i>/<i>z</i>): 533.0 [M + H]<sup>+</sup>. HRMS (ESI) calcd for C<sub>21</sub>H<sub>13</sub>Br<sub>2</sub>N<sub>2</sub>O<sub>5</sub> ([M – H]<sup>−</sup>) 530.9191; found 530.9186.</div></div><div id="sec4_4_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> <i>N</i>′-(3,5-Dibromo-2,4-dihydroxybenzylidene)biphenyl-3-carbohydrazide <b>16d</b></h4><div class="NLM_p last">Yield: 64%. HPLC: 95.70%, <i>t</i><sub>R</sub> = 23.56. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.91 (s, 1H), 12.42 (s, 1H), 10.41 (s, 1H), 8.49 (s, 1H), 8.20 (s, 1H), 7.96–7.90 (m, 2H), 7.79–7.74 (m, 3H), 7.65 (t, <i>J</i> = 7.6 Hz, 1H), 7.53 (t, <i>J</i> = 7.6 Hz, 2H), 7.43 (t, <i>J</i> = 7.2 Hz, 1H). MS (ESI, <i>m</i>/<i>z</i>): 489.2 [M + H]<sup>+</sup>. HRMS (ESI) calcd for C<sub>20</sub>H<sub>14</sub>Br<sub>2</sub>N<sub>2</sub>O<sub>3</sub>Na ([M + H]<sup>+</sup>) 510.9269; found 510.9280.</div></div><div id="sec4_4_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> <i>N</i>′-(2,4-Dihydroxybenzylidene)biphenyl-4-carbohydrazide <b>16e</b></h4><div class="NLM_p last">Yield: 33%. HPLC: 98.13%, <i>t</i><sub>R</sub> = 6.43. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.98 (s, 1H), 11.50 (s, 1H), 9.97 (s, 1H), 8.53 (s, 1H), 8.02 (d, <i>J</i> = 8.4 Hz, 2H), 7.85 (d, <i>J</i> = 8.4 Hz, 2H), 7.76 (d, <i>J</i> = 7.2 Hz, 2H), 7.51 (t, <i>J</i> = 7.6 Hz, 2H), 7.43 (t, <i>J</i> = 7.2 Hz, 1H), 7.32 (d, <i>J</i> = 8.4 Hz, 1H), 6.41–6.31 (m, 2H). EI-MS <i>m</i>/<i>z</i> (M<sup>+</sup>) 332. EI-HRMS calcd for (M<sup>+</sup>) 332.1161; found 332.1163.</div></div><div id="sec4_4_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> <i>N</i>′-(3,5-Dibromo-2-hydroxy-4-methoxybenzylidene)biphenyl-4-carbohydrazide <b>16f</b></h4><div class="NLM_p last">Yield: 78%. HPLC: 99.29%, <i>t</i><sub>R</sub> = 16.59. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.98 (s, 1H), 12.55 (s, 1H), 8.55 (s, 1H), 8.06 (d, <i>J</i> = 8.4 Hz, 2H), 7.91 (s, 1H), 7.87 (d, <i>J</i> = 8.4 Hz, 2H), 7.76 (d, <i>J</i> = 7.6 Hz, 2H), 7.51 (t, <i>J</i> = 7.6 Hz, 2H), 7.43 (t, <i>J</i> = 7.6 Hz, 1H), 3.83 (s, 3H). EI-MS <i>m</i>/<i>z</i> (M<sup>+</sup>) 502. EI-HRMS calcd for (M<sup>+</sup>) 501.9528; found 501.9501.</div></div><div id="sec4_4_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> <i>N</i>′-(3,5-Dibromo-4-hydroxy-2-methoxybenzylidene)biphenyl-4-carbohydrazide <b>16g</b></h4><div class="NLM_p last">Yield: 81%. HPLC: 96.32%, <i>t</i><sub>R</sub> = 6.32. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.99 (s, 1H), 8.61 (s, 1H), 8.08–8.00 (m, 3H), 7.84 (d, <i>J</i> = 8.4 Hz, 2H), 7.75 (d, <i>J</i> = 7.2 Hz, 2H), 7.51 (t, <i>J</i> = 7.2 Hz, 2H), 7.42 (t, <i>J</i> = 7.2 Hz, 1H), 3.83 (s, 3H). EI-MS <i>m</i>/<i>z</i> (M<sup>+</sup>) 502. EI-HRMS calcd for (M<sup>+</sup>) 501.9528; found 501.9524.</div></div><div id="sec4_4_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> <i>N</i>′-(3,5-Dibromo-2,4-dihydroxybenzylidene)-4-(furan-2-yl)benzohydrazide <b>16h</b></h4><div class="NLM_p last">Yield: 55%. HPLC: 97.26%, <i>t</i><sub>R</sub> = 14.51. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.93 (s, 1H), 12.37 (s, 1H), 10.39 (s, 1H), 8.48 (s, 1H), 8.01 (d, <i>J</i> = 8.4 Hz, 2H), 7.90–7.83 (m, 3H), 7.77 (s, 1H), 7.16 (d, <i>J</i> = 3.6 Hz, 1H), 6.67 (dd, <i>J</i> = 1.8, 3.3 Hz, 1H). EI-MS <i>m</i>/<i>z</i> (M<sup>+</sup>) 478. EI-HRMS calcd for (M<sup>+</sup>) 477.9164; found 477.9148.</div></div><div id="sec4_4_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> <i>N</i>′-(3,5-Dibromo-2,4-dihydroxybenzylidene)-4-(pyridin-4-yl)benzohydrazide <b>16i</b></h4><div class="NLM_p last">Yield: 71%. HPLC: 95.71%, <i>t</i><sub>R</sub> = 2.27. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.90 (s, 1H), 12.46 (s, 1H), 10.32 (s, 1H), 8.70 (d, <i>J</i> = 4.4 Hz, 2H), 8.50 (s, 1H), 8.10 (d, <i>J</i> = 8.4 Hz, 2H), 8.01 (d, <i>J</i> = 8.4 Hz, 2H), 7.83–7.77 (m, 3H). MS (ESI, <i>m</i>/<i>z</i>): 488.1 [M – H]<sup>−</sup>. HRMS (ESI) calcd for C<sub>19</sub>H<sub>13</sub>Br<sub>2</sub>N<sub>3</sub>O<sub>3</sub>Na ([M + Na]<sup>+</sup>) 511.9221; found 511.9240.</div></div><div id="sec4_4_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> <i>N</i>′-(3,5-Dibromo-2,4-dihydroxybenzylidene)-4-(pyridin-3-yl)benzohydrazide <b>16j</b></h4><div class="NLM_p last">Yield: 63%. HPLC: 98.59%, <i>t</i><sub>R</sub> = 3.32. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.91 (s, 1H), 12.43 (s, 1H), 10.41 (s, 1H), 9.00 (d, <i>J</i> = 2.0 Hz 1H), 8.63 (d, <i>J</i> = 4.0 Hz, 1H), 8.50 (s, 1H), 8.23–8.16 (m, 1H), 8.08 (d, <i>J</i> = 8.4 Hz, 2H), 7.95 (d, <i>J</i> = 8.4 Hz, 2H), 7.78 (s, 1H), 7.54 (dd, <i>J</i> = 4.8, 8.0 Hz, 1H). MS (ESI, <i>m</i>/<i>z</i>): 490.1 [M + H]<sup>+</sup>. HRMS (ESI) calcd for C<sub>19</sub>H<sub>14</sub>Br<sub>2</sub>N<sub>3</sub>O<sub>3</sub> ([M + H]<sup>+</sup>) 489.9402; found 489.9391.</div></div><div id="sec4_4_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> <i>N</i>′-(3,5-Dibromo-2,4-dihydroxybenzylidene)-4-(pyridin-2-yl)benzohydrazide <b>16k</b></h4><div class="NLM_p last">Yield: 65%. HPLC: 99.03%, <i>t</i><sub>R</sub> = 3.90. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.93 (s, 1H), 12.44 (s, 1H), 10.38 (s, 1H), 8.72 (d, <i>J</i> = 4.4 Hz, 1H), 8.50 (s, 1H), 8.27 (d, <i>J</i> = 8.0 Hz, 2H), 8.11–8.04 (m, 3H), 7.94 (t, <i>J</i> = 7.6 Hz, 1H), 7.78 (s, 1H), 7.45–7.40 (m, 1H). MS (ESI, <i>m</i>/<i>z</i>): 490.2 [M + H]<sup>+</sup>. HRMS (ESI) calcd for C<sub>19</sub>H<sub>13</sub>Br<sub>2</sub>N<sub>3</sub>O<sub>3</sub>Na ([M + Na]<sup>+</sup>) 511.9221; found 511.9221.</div></div><div id="sec4_4_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> <i>N</i>′-(3,5-Dibromo-2,4-dihydroxybenzylidene)-4-methylbenzohydrazide <b>16l</b></h4><div class="NLM_p last">Yield: 42%. HPLC: 98.90%, <i>t</i><sub>R</sub> = 9.18. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.94 (s, 1H), 12.26 (s, 1H), 10.39 (s, 1H), 8.45 (s, 1H), 7.85 (d, <i>J</i> = 8.0 Hz, 2H), 7.74 (s, 1H), 7.36 (d, <i>J</i> = 8.0 Hz, 2H), 2.39 (s, 3H). EI-MS <i>m</i>/<i>z</i> (M<sup>+</sup>) 426. EI-HRMS calcd for (M<sup>+</sup>) 425.9215; found 425.9213.</div></div><div id="sec4_4_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> <i>N</i>′-(3,5-Dibromo-2,4-dihydroxybenzylidene)-4-(trifluoromethyl)benzohydrazide <b>16m</b></h4><div class="NLM_p last">Yield: 32%. HPLC: 98.94%, <i>t</i><sub>R</sub> = 13.82. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.79 (s, 1H), 12.54 (s, 1H), 10.43 (s, 1H), 8.49 (s, 1H), 8.14 (d, <i>J</i> = 8.1 Hz, 2H), 7.94 (d, <i>J</i> = 8.1 Hz, 2H), 7.80 (s, 1H). EI-MS <i>m</i>/<i>z</i> (M<sup>+</sup>) 480. EI-HRMS calcd for (M<sup>+</sup>) 479.8932; found 479.8916.</div></div><div id="sec4_4_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> <i>N</i>′-(3,5-Dibromo-2,4-dihydroxybenzylidene)-3,5-dimethoxybenzohydrazide <b>120b</b></h4><div class="NLM_p last">Yield: 54%. HPLC: 98.41%, <i>t</i><sub>R</sub> = 9.41. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.86 (s, 1H), 12.25 (s, 1H), 10.39 (s, 1H), 8.46 (s, 1H), 7.75 (s, 1H), 7.08 (d, <i>J</i> = 2.4 Hz, 2H), 6.74 (t, <i>J</i> = 2.4 Hz, 1H), 3.82 (s, 6H). MS (ESI, <i>m</i>/<i>z</i>): 471.1 [M – H]<sup>−</sup>. HRMS (ESI) calcd for C<sub>16</sub>H<sub>14</sub>Br<sub>2</sub>N<sub>2</sub>O<sub>5</sub>Na ([M + Na]<sup>+</sup>) 494.9167; found 494.9178.</div></div><div id="sec4_4_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> <i>N</i>′-(3,5-Dibromo-2,4-dihydroxybenzylidene)quinoline-6-carbohydrazide <b>20a</b></h4><div class="NLM_p last">Yield: 42%. HPLC: 97.19%, <i>t</i><sub>R</sub> = 3.59. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.89 (s, 1H), 12.59 (s, 1H), 10.41 (s, 1H), 9.04 (dd, <i>J</i> = 1.5, 4.2 Hz, 1H), 8.64 (d, <i>J</i> = 1.2 Hz, 1H), 8.60–8.51 (m, 2H), 8.26 (dd, <i>J</i> = 1.5, 8.4 Hz, 1H), 8.16 (d, <i>J</i> = 8.7 Hz, 1H), 7.80 (s, 1H), 7.67 (dd, <i>J</i> = 4.2, 8.1 Hz, 1H). MS (ESI, <i>m</i>/<i>z</i>): 462.0 [M – H]<sup>−</sup>. HRMS (ESI) calcd for C<sub>17</sub>H<sub>11</sub>Br<sub>2</sub>N<sub>3</sub>O<sub>3</sub>Na ([M + Na]<sup>+</sup>) 485.9065; found 485.9083.</div></div><div id="sec4_4_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> 6-Bromo-<i>N</i>′-(3,5-dibromo-2-hydroxy-4-methoxybenzylidene)-2-naphthohydrazide <b>20c</b></h4><div class="NLM_p last">Yield: 79%. HPLC: 97.64%, <i>t</i><sub>R</sub> = 8.04. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.95 (s, 1H), 12.68 (s, 1H), 8.60 (s, 1H), 8.57 (s, 1H), 8.34 (d, <i>J</i> = 1.8 Hz 1H), 8.11–8.05 (m, 3H), 7.95 (s, 1H), 7.77 (dd, <i>J</i> = 1.8, 8.7 Hz, 1H), 3.84 (s, 3H). EI-MS <i>m</i>/<i>z</i> (M<sup>+</sup>) 554. EI-HRMS calcd for (M<sup>+</sup>) 553.8476; found 553.8457.</div></div><div id="sec4_4_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> 6-Bromo-<i>N</i>′-(3,5-dibromo-4-hydroxy-2-methoxybenzylidene)-2-naphthohydrazide <b>20d</b></h4><div class="NLM_p last">Yield: 39%. HPLC: 95.16%, <i>t</i><sub>R</sub> = 8.59. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.12 (s, 1H), 10.56 (s, 1H), 8.62 (s, 1H), 8.57 (s, 1H), 8.33 (d, <i>J</i> = 1.2H, 1H), 8.09–8.00 (m, 4H), 7.76 (dd, <i>J</i> = 1.5, 8.7 Hz, 1H), 3.83 (s, 3H). EI-MS <i>m</i>/<i>z</i> (M<sup>+</sup>) 554. EI-HRMS calcd for (M<sup>+</sup>) 553.8476; found 553.8484.</div></div><div id="sec4_4_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> <i>N</i>′-(3,5-Dibromo-2,4-dihydroxybenzylidene)-6-(methylsulfonyl)-2-naphthohydrazide <b>20e</b></h4><div class="NLM_p last">Yield: 52%. HPLC: 96.63%, <i>t</i><sub>R</sub> = 5.64. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.89 (s, 1H), 12.65 (s, 1H), 10.44 (s, 1H), 8.70 (s, 2H), 8.53 (s, 1H), 8.39–8.33 (m, 2H), 8.15 (dd, <i>J</i> = 0.9, 9.3 Hz, 1H), 8.06 (dd, <i>J</i> = 1.5, 8.7 Hz, 1H), 7.82 (s, 1H), 3.3 (s, 3H). MS (ESI, <i>m</i>/<i>z</i>): 541.1 [M + H]<sup>+</sup>. HRMS (ESI) calcd for C<sub>19</sub>H<sub>13</sub>Br<sub>2</sub>N<sub>2</sub>O<sub>5</sub>S ([M – H]<sup>−</sup>) 538.8912; found 538.8919.</div></div><div id="sec4_4_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> 6-Cyano-<i>N</i>′-(3,5-dibromo-2,4-dihydroxybenzylidene)-2-naphthohydrazide <b>20f</b></h4><div class="NLM_p last">Yield: 37%. HPLC: 95.79%, <i>t</i><sub>R</sub> = 10.50. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.88 (s, 1H), 12.63 (s, 1H), 10.46 (s, 1H), 8.68 (d, <i>J</i> = 9.3 Hz, 2H), 8.52 (s, 1H), 8.30 (d, <i>J</i> = 8.4 Hz, 1H), 8.23 (d, <i>J</i> = 8.4 Hz, 1H), 8.13 (d, <i>J</i> = 9.3 Hz, 1H), 7.92 (d, <i>J</i> = 8.7 Hz, 1H), 7.81 (s, 1H). MS (ESI, <i>m</i>/<i>z</i>): 486.1 [M – H]<sup>−</sup>. HRMS (ESI) calcd for C<sub>19</sub>H<sub>11</sub>Br<sub>2</sub>N<sub>3</sub>O<sub>3</sub>Na ([M + Na]<sup>+</sup>) 509.9065; found 509.9043.</div></div><div id="sec4_4_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> <i>N</i>′-(3,5-Dibromo-2,4-dihydroxybenzylidene)-6-methoxy-2-naphthohydrazide <b>20g</b></h4><div class="NLM_p last">Yield: 58%. HPLC: 95.11%, <i>t</i><sub>R</sub> = 16.07. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.00 (s, 1H), 12.45 (s, 1H), 10.39 (s, 1H), 8.50 (s, 2H), 8.00 (d, <i>J</i> = 9.0 Hz, 1H), 7.96 (s, 2H), 7.78 (s, 1H), 7.43 (d, <i>J</i> = 2.4 Hz, 1H), 7.28 (dd, <i>J</i> = 2.7, 9.0 Hz, 1H), 3.92 (s, 3H). MS (ESI, <i>m</i>/<i>z</i>): 491.2 [M – H]<sup>−</sup>. HRMS (ESI) calcd for C<sub>19</sub>H<sub>14</sub>Br<sub>2</sub>N<sub>2</sub>O<sub>4</sub>Na ([M + Na]<sup>+</sup>) 514.9218; found 514.9199.</div></div><div id="sec4_4_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> <i>N</i>′-(3,5-Dibromo-2,4-dihydroxybenzylidene)-6-(furan-2-yl)-2-naphthohydrazide <b>20h</b></h4><div class="NLM_p last">Yield: 67%. HPLC: 98.20%, <i>t</i><sub>R</sub> = 10.79. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.97 (s, 1H), 12.53 (s, 1H), 10.41 (s, 1H), 8.55 (d, <i>J</i> = 0.9 Hz, 1H), 8.52 (s, 1H), 8.31 (s, 1H), 8.13 (dd, <i>J</i> = 4.5, 8.7 Hz, 2H), 8.03–7.96 (m, 2H), 7.87 (dd, <i>J</i> = 0.6, 1.8 Hz, 1H), 7.80 (s, 1H), 7.20 (dd, <i>J</i> = 0.6, 3.3 Hz, 1H), 6.69 (dd, <i>J</i> = 1.8, 3.3 Hz, 1H). MS (ESI, <i>m</i>/<i>z</i>): 529.1 [M + H]<sup>+</sup>. HRMS (ESI) calcd for C<sub>22</sub>H<sub>14</sub>Br<sub>2</sub>N<sub>2</sub>O<sub>4</sub>Na ([M + Na]<sup>+</sup>) 550.9218; found 550.9246.</div></div><div id="sec4_4_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> <i>N</i>′-(3,5-Dibromo-2,4-dihydroxybenzylidene)-6-phenyl-2-naphthohydrazide <b>20i</b></h4><div class="NLM_p last">Yield: 91%. HPLC: 97.09%, <i>t</i><sub>R</sub> = 15.38. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.96 (s, 1H), 12.54 (s, 1H), 8.60 (s, 1H), 8.53 (s, 1H), 8.33 (s, 1H), 8.22–8.13 (m, 2H), 8.06–7.95 (m, 2H), 7.89–7.84 (m, 2H), 7.79 (s, 1H), 7.58–7.51 (m, 2H), 7.44 (t, <i>J</i> = 7.2 Hz, 1H). MS (ESI, <i>m</i>/<i>z</i>): 539.1 [M + H]<sup>+</sup>. HRMS (ESI) calcd for C<sub>24</sub>H<sub>16</sub>Br<sub>2</sub>N<sub>2</sub>O<sub>3</sub>Na ([M + Na]<sup>+</sup>) 560.9425; found 560.9414.</div></div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i78" class="anchor-spacer"></div><h3 class="article-section__title" id="_i78"> Protein Expression and Purification</h3><div class="NLM_p">DNA fragment which encodes SIRT1 (156–664) was inserted into pET28a vector (<i>Nde</i><i>1</i>/<i>Xho</i><i>1</i>) with a hexa histidine-tag at the N-terminal. The plasmid was transformed into <i>Escherichia coli</i> BL21 (DE3) competent cells. A single colony was inoculated in the LB medium with 50 μg/mL kanamycin at 37 °C. Then 500 μM IPTG was added to the cell culture when OD<sub>600</sub> reached 0.8–1.0 and the protein was induced at 16 °C overnight. The cells were sonicated with SIRT1 lysis buffer (20 mM Tris, 200 mM NaCl, 5 mM β-ME, pH 8.0) and loaded to Ni-NTA. After washing with 40 mM imidazole, the protein was eluted with 500 mM imidazole. The elution was validated by SDS-PAGE and dialyzed into sirtuin deacetylation assay buffer in combination with glycerol (25 mM Tris, 137 mM NaCl, 2.7 mM KCl, 1 mM MgCl<sub>2</sub>, 10% glycerol, pH 8.0). SIRT1<sup>Q345A</sup>, SIRT1<sup>V445A</sup>, and SIRT1<sup>R446A</sup> plasmid were generated by site-directed mutagenesis kit (Stratagene, catalogue no 200518). The mutant was expressed and purified the same as the wild-type SIRT1.</div><div class="NLM_p">DNA fragment which encodes SIRT3 (118–399) was inserted into pET21a vector (<i>BamH</i><i>1</i>/<i>Xho</i><i>1</i>) with a hexa histidine-tag at the N-terminal and a TEV cleavage site between the protein sequence and his-tag. The plasmid was transformed into <i>E. coli</i> BL21 (DE3) competent cells. A single colony was inoculated in the LB medium with 100 μg/mL ampicillin at 37 °C. Then 100 μM IPTG was added to the cell culture when OD<sub>600</sub> reached 0.6–0.8 and the protein was induced at 16 °C overnight. The cells were sonicated with SIRT3 lysis buffer (25 mM Hepes, 200 mM NaCl, 5 mM β-ME, 5% glycerol, pH 7.5) and loaded to Ni-NTA. After washing with 40 mM imidazole, the protein was eluted with 250 mM imidazole. The elution was dialyzed into SIRT3 dialysis buffer (20 mM Tris, 200 mM NaCl, pH8.0) with TEV protease digestion at 4 °C overnight. The dialyzed protein was reloaded to Ni-NTA. The flow-through and 5 mM imizadole elution were collected and loaded to Superdex 200 (GE Healthcare) equilibrated with SIRT3 dialyze buffer. The peak representing SIRT3 monomer was collected and concentrated for crystallization.</div><div class="NLM_p last">DNA fragment which encodes SIRT2 (34–356) was inserted into pGET-4t-1 vector (<i>BamH</i><i>1</i>/<i>EcoR</i><i>1</i>) with a GST fusion protein at the N-terminus. The plasmid was transformed into <i>E. coli</i> BL21 (DE3) competent cells. A single colony was inoculated in the LB medium with 100 μg/mL ampicillin at 37 °C. Then 500 μM IPTG was added to the cell culture when OD<sub>600</sub> reached 0.3 and the protein was induced at 37 °C for 5 h. The protein was purified by GST resin with SIRT1 lysis buffer and eluted with 10 mM GSH in SIRT1 lysis buffer. The elution was validated by SDS-PAGE and dialyzed the same as SIRT1 protein.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i79" class="anchor-spacer"></div><h3 class="article-section__title" id="_i79"> Screening of Sirtuin Activators and the EC<sub>1.5</sub> Determination of the Compounds</h3><div class="NLM_p">A fluorescent assay was used to screen the compounds against SIRT1, SIRT2, and SIRT3. The C-terminus of the acetyl peptide acetyl-Arg-His-Lys-Lys(ε-acetyl) was labeled with 4-amino-7-methylcoumarin. The ALY was deacetylated by sirtuins. Then, AMC could be cleaved from the peptide by trypsin. All compounds were dissolved in DMSO, and the screening concentration was fixed at 50 μM; compound <b>1</b> was used as control. For the compound screening against SIRT1, the reaction system contained 750 μM NAD<sup>+</sup>, 31.25 μM ac-RHKK<sub>ac</sub>-AMC, 1 μM SIRT1, and compounds/DMSO in sirtuin deacetylation assay buffer (25 mM Tris, 137 mM NaCl, 2.7 mM KCl, 1 mM MgCl<sub>2</sub>, pH8.0). The reactions were carried out at 25 °C for 45 min, stopped with 10 mM nicotinamide, and developed with 0.5 μg/μL trysin for 30 min. The fluorescence intensity of AMC was measured with a microplate reader (POLARstar OPTIMA, BMG Labtech); the excitation and emission wavelengths were set 340 and 490 nm. To examine the selectivity of the compounds, we tested the activity of the compounds on SIRT2 and SIRT3. The reaction conditions and procedures were the same as the screening assay on SIRT1 (750 μM of NAD<sup>+</sup>, 31.25 μM of ac-RHKK<sub>ac</sub>-AMC, 50 μM of compounds). To achieve the same enzymatic activity as SIRT1, the concentrations of SIRT2 and SIRT3 were 4.2 and 3.6 μM, respectively.</div><div class="NLM_p last">The EC<sub>1.5</sub> determination was carried out the same as the screening assay. Compounds were 3-fold diluted in DMSO and added to reaction system. EC<sub>1.5</sub> and maximum activation were calculated by fitting the data points with the dose–response function in Origin 8.0 (Origin lab).</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i80" class="anchor-spacer"></div><h3 class="article-section__title" id="_i80"> Isothermal Titration Calorimetry</h3><div class="NLM_p last">SIRT1 protein was dialyzed into sirtuin deacetylation assay buffer (25 mM Tris, 137 mM NaCl, 2.7 mM KCl, 1 mM MgCl<sub>2</sub>, 5% glycerol, pH 8.0). The acetyl peptide was injected in 20 drops of 2 μL (the first drop was 0.5 μL and removed). The cell temperature was 26 °C, reference power was 5 μCal/s, initial delay was 60 s, and the stirring speed was 1000 rpm. All titrations were performed on a Microcal iTC200 (GE Healthcare). For the <i>K</i><sub>d</sub> detection of ac-RHKK<sub>ac</sub>-NH<sub>2</sub> to SIRT1, 2.5 mM ac-RHKK<sub>ac</sub>-NH<sub>2</sub> was titrated to 102 μM SIRT1. For the <i>K</i><sub>d</sub> detection of ac-RHKK<sub>ac</sub>-AMC to SIRT1, 2.5 mM ac-RHKK<sub>ac</sub>-AMC was titrated to 124 μM SIRT1. Raw data were integrated and analyzed with Origin 7.0 software (Origin lab). The data was fit to a one-site binding model.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i81" class="anchor-spacer"></div><h3 class="article-section__title" id="_i81"> Determination of <i>K</i><sub>m</sub> for Acetyl Peptide with or without AMC</h3><div id="sec4_8_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i82" class="anchor-spacer"></div><h4 class="article-section__title" id="_i82"> Fluorescent Assay</h4><div class="NLM_p last">To determine the <i>K</i><sub>m</sub> values of ac-RHKK<sub>ac</sub>-AMC for SIRT1 wild-type and mutants in the absence and presence of compounds, the concentrations of NAD<sup>+</sup> and enzyme were fixed, with gradient concentration of peptide. DMSO was used as blank, and compounds were added as indicated. The procedures were the same as the above screening assay. The velocities were calculated by the discrepancies of fluorescence divided by time. The reciprocal data of velocities were plotted against the reciprocal data of peptide concentration, and the <i>x</i> intercept equals to negative 1/<i>K</i><sub>m</sub>.</div></div><div id="sec4_8_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i83" class="anchor-spacer"></div><h4 class="article-section__title" id="_i83"> HPLC Assay</h4><div class="NLM_p">To determine the <i>K</i><sub>m</sub> values of ac-RHKK<sub>ac</sub>-NH<sub>2</sub> in the absence and presence of compounds, SIRT1 was fixed at 1 μM and NAD<sup>+</sup> was fixed at 2 mM, the concentrations of ac-RHKK<sub>ac</sub>-NH<sub>2</sub> were 1000, 500, 250, 125, 62.5, 31.25, 15.625, and 7.8125 μM, respectively. Indicated compounds were added to the reaction system (50 μM <b>4</b>, 10 μM compound <b>20b</b> or DMSO for blank). The reactions were carried out at 25 °C for 30 min and stopped by 10 mM nicotinamide. The <i>K</i><sub>m</sub> values of ac-RHKK<sub>ac</sub>-AMC were determined in the same way as ac-RHKK<sub>ac</sub>-NH<sub>2</sub> except that the concentrations of the peptide were set to 300, 150, 100, and 70 μM, respectively. After the reactions were stopped, the solution was centrifuged at 13000 rpm for 10 min, and the supernatant was transferred to glass vials for HPLC detection.</div><div class="NLM_p">The reactions of ac-RHKK<sub>ac</sub>-NH<sub>2</sub> were monitored by ultra performance liquid chromatography (UPLC). The separation was performed at 30 °C by using the Acquity UPLC BEH Shield RP 18 column (50 mm ×2.1 mm I.D., 1.7 μm) with the Waters Van Guard precolumn (5 mm × 2.1 mm I.D., 1.7 μm). The mobile phase consisted of buffer A (distilled water containing 0.1% TFA) and buffer B (acetonitrile containing 0.1% TFA) with a flow rate of 0.4 mL/min. The UV detection wavelength was set at 220 nm. The retention time of ac-RHKK<sub>ac</sub>-NH<sub>2</sub> was 1.7 min (<a class="ref internalNav" href="#notes-4" aria-label="Supporting Information Figure S2A">Supporting Information Figure S2A</a>).</div><div class="NLM_p">The reactions with ac-RHKK<sub>ac</sub>-AMC were monitored by Dionex UltiMate 3000 HPLC system. The separation was achieved at 30 °C by using the Zorbax SB-Aq C18 column (250 mm × 4.6 mm I.D., 5 μm, Agilent Corp., USA) with the Alltima C18 guard column (5 mm × 4.6 mm I.D., 5 μm, Grace Corp., USA). The mobile phase was the same as above with a flow rate of 1 mL/min. The UV detection wavelength was set at 325 nm. The retention times of ac-RHKK-AMC and ac-RHKK<sub>ac</sub>-AMC were 6 and 10.5 min, respectively (<a class="ref internalNav" href="#notes-4" aria-label="Supporting Information Figure S2B">Supporting Information Figure S2B</a>).</div><div class="NLM_p last">To determine the <i>K</i><sub>m</sub> values, the peak area of the remaining acetyl substrate was converted to molar concentration by using the peptide standard curve, and the deacetylated amount of substrate was determined. The velocities were calculated by the deacetylated amount of substrate divided by time. The calculation of <i>K</i><sub>m</sub> was the same as described above.</div></div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i84" class="anchor-spacer"></div><h3 class="article-section__title" id="_i84"> Crystallization, Data Collection, And Structure Determination</h3><div class="NLM_p">For “SIRT3/ac-RHKK<sub>ac</sub>-AMC peptide” cocrystallization, the molar ratio of ac-RHKK<sub>ac</sub>-AMC and SIRT3 protein was 1:1. SIRT3 was concentrated to 7 mg/mL, and the according amount of ac-RHKK<sub>ac</sub>-AMC peptide powder was added to the protein. The mixture was incubated at 4 °C overnight and centrifuged for crystallization. The “SIRT3/ac-RHKK<sub>ac</sub>-AMC” cocrystals were obtained by the hanging-drop vapor-diffusion method at 15 °C with crystallization solution containing 0.085 M Hepes-Na (pH 7.5), 8.5% (v/v) isopropyl alcohol, 17% (w/v) PEG 4000, and 15% (v/v) glycerol anhydrous.</div><div class="NLM_p last">The “SIRT3/ac-RHKKac-AMC” crystals were flash-frozen in liquid nitrogen. The diffraction data were collected on beamline BL17U at Shanghai Synchrotron Radiation Facility (SSRF) and processed by using HKL2000 program.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> The space group of the complex was <i>R</i>32 with one “SIRT3/ac-RHKK<sub>ac</sub>-AMC” complex in the asymmetric unit. The data set was reprocessed by Scalepack2mtz in the ccp4 suite,<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> and the structure was solved by MolRep<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> with apo-SIRT3 structure as the template (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GLS">3GLS</a>). The structure refinement was carried out by Shelx97,<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> Refmac5,<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> and COOT.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> The detailed information of diffraction data and refinement statistics were listed in Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i85" class="anchor-spacer"></div><h3 class="article-section__title" id="_i85"> Homology Modeling and Docking Simulation</h3><div class="NLM_p">The sequence of SIRT1 (244–498) was retrieved from the UniProt database (UniProtKB code: Q96EB6, <a href="http://www.uniprot.org/" class="extLink">http://www.uniprot.org/</a>). SIRT1 model was built with the crystal structures of “SIRT3/ac-RHKK<sub>ac</sub>-AMC” (SIRT3–121–157 and SIRT3–168–391) and “SIRT3/AcsCS2-K<sub>ac</sub>” (SIRT3–158–167, PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GLR">3GLR</a>) as templates. The sequences of SIRT1 and SIRT3 were aligned by ClustalW (<a class="ref internalNav" href="#notes-4" aria-label="Supporting Information Figure S4">Supporting Information Figure S4</a>).<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> The aligned sequences were input into Discovery Studio (Accelrys) and SIRT1 model structure was built by the Build Homology module. The model of SIRT1 was further validated by Profiles-3D,<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> and the overall compatibility score of the model was 100.37, which was much higher than the expected low score 52.15. The Ramachandran plot revealed two residues (Leu 404 and Ala 405) in the disallowed region (<a class="ref internalNav" href="#notes-4" aria-label="Supporting Information Figure S7B">Supporting Information Figure S7B</a>), but the residues are far away from the peptide binding site. Therefore, the quality of SIRT1 model was acceptable for the following study. The model of SIRT1 was superimposed into the SIRT3 complex structure, and the coordinates of ac-RHKK<sub>ac</sub>-AMC were copied from the complex structure to the SIRT1 model to build SIRT1/ac-RHKK<sub>ac</sub>-AMC model.</div><div class="NLM_p last">The structures of compounds <b>20b</b>, <b>1</b>, and <b>4</b> were built with the Simulate Structure package in Discovery Studio. The structures were assigned with CHARm forcefield parameters and optimized by energy minimization. Autodock 4.2 was used to predict the binding modes of the compounds in SIRT1.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> SIRT1/ac-RHKK<sub>ac</sub>-AMC model structure was applied with CHARm forcefield. The mol2 files of compounds and the pdb file of protein were pretreated with AutoDock Tools 1.5.4.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> In the docking procedure, the genetic algorithm (GA) was used. The number of GA runs was set to 50, population size was set to 200 and maximum number of top individuals that automatically survive was set to 2. The grid box was centered at the substrate binding cleft under the fluorophore. Other parameters were set as default. After running autogrid and autodock, the possible poses of compounds in SIRT1 were obtained.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i86" class="anchor-spacer"></div><h3 class="article-section__title" id="_i86"> Stability Test</h3><div class="NLM_p last">The stability of compound <b>20b</b> was tested on human breast cancer cells MCF-7. The cells were treated with <b>20b</b> for 0, 5, 15, 30, 60, 120, and 360 min. The medium was removed at the end point, and equal volume of acetonitrile was added. Cells were washed and treated with acetonitrile. The medium and cell lysate were collected separately for detection. <b>20b</b> was added to absolute cell medium and treated the same to serve as control. The samples were centrifuged and injected onto the Agilent 1260 Infinity HPLC system (Agilent Technologies, Waldbronn, Germany). Separation was achieved using a Hypersil GOLD Dim column (100 mm × 4.6 mm I.D., 3 μm; Thermo Fisher Scientific, Runcorn, Cheshire, UK) at a flow rate of 1 mL/min. The column was maintained at room temperature. A mobile phase of acetonitrile–5 mM ammonium acetate–formic acid (50:50:0.1, v/v/v) was employed. The effluent was monitored via ultraviolet detection at 340 nm.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i87" class="anchor-spacer"></div><h3 class="article-section__title" id="_i87"> Western Blotting</h3><div class="NLM_p last">Human cancer cell line MCF-7 cells were seeded at a density of 1 × 10<sup>5</sup> in 6-well plates with Dulbecco’s Modified Eagle’s Medium (DMEM) plus 10% fetal calf serum. Cells were incubated in a humidified atmosphere containing 5% CO<sub>2</sub> at 37 °C for 24 h. Gradient concentrations of <b>1</b>, <b>20b</b>, or <b>20g</b> plus 0.1 μM TSA and 5 μM DOX were added to MCF-7 cells. After incubation for 6 h, cells were collected and lysed by using a kit from Active Motif (Carlsbad, CA). Total protein concentration was determined by BCA kit (catalogue no. P0012, Beyotime Institute of Biotechnology, Shanghai, China). Samples containing 50 μg of protein were loaded onto a 9% gel. The proteins were transferred to a nitrocellulose membrane (Whatman, UK) after electrophoresis. Then 5% nonfat milk in TBST (10 mM Tris pH7.5, 150 mM NaCl, 0.05% Tween-20) was used to block the nonspecific reactivity. To determine the acetyl-p53, the membrane was incubated with polyclonal antibody against acetyl-p53 (Cell Signaling Technology, catalogue no. 2525) at 4 °C overnight. Immune complexes were detected with horseradish peroxidase-conjugated goat antirabbit IgG antibody and visualized with a SuperSignal West Dura Kit (Pierce). To determine the total p53, the same blot was reprobed and p53 antibody (Cell Signaling Technology, catalogue no. 2527) was used.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-4"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i88"><a href="/doi/suppl/10.1021/jm301032j">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00421" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00421" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Information about the synthesis of the chemical compounds, molecular modeling, HPLC data, isothermal titration calorimetry, enzyme kinetics data, etc. This material is available free of charge via the Internet at <a href="http://pubs.acs.org" class="extLink">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm301032j/suppl_file/jm301032j_si_001.pdf">jm301032j_si_001.pdf (811.14 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The coordinates of “SIRT3/ac-RHKK<sub>ac</sub>-AMC” complex structure were deposited in protein data bank with the PDB ID of <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FZ3">4FZ3</a>.</p><div class="testing" data-doi="10.1021/jm301032j" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47817" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47817" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hong Liu</span> - <span class="hlFld-Affiliation affiliation">†Department
of Pharmacology III, ‡Drug Discovery and Design Center, §Department of Analytical Chemistry, and ∥Center for Drug
Metabolism and Pharmacokinetics, Shanghai
Institute of Materia Medica, Chinese Academy of Sciences,
555 Zu-Chong-Zhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">School of Life Sciences, Fudan
University, Shanghai 200433, China</span>; 
    <span class="hlFld-Affiliation affiliation">Institute of Molecular Design and Drug
Discovery, School of Pharmaceutical Sciences, Central South University,
Changsha 410078, Hunan, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#f7938f9bb79a969e9bd9849f949994d99694d99499d79f9b9e82b79a969e9bd9849f949994d99694d99499"><span class="__cf_email__" data-cfemail="b0d4c8dcf0ddd1d9dc9ec3d8d3ded39ed1d39ed3de">[email protected]</span> <span class="__cf_email__" data-cfemail="a7cfcbced2e7cac6cecb89d4cfc4c9c489c6c489c4c9">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dongxiang Liu</span> - <span class="hlFld-Affiliation affiliation">†Department
of Pharmacology III, ‡Drug Discovery and Design Center, §Department of Analytical Chemistry, and ∥Center for Drug
Metabolism and Pharmacokinetics, Shanghai
Institute of Materia Medica, Chinese Academy of Sciences,
555 Zu-Chong-Zhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">School of Life Sciences, Fudan
University, Shanghai 200433, China</span>; 
    <span class="hlFld-Affiliation affiliation">Institute of Molecular Design and Drug
Discovery, School of Pharmaceutical Sciences, Central South University,
Changsha 410078, Hunan, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#76120e1a361b171f1a58051e151815581715581518561e1a1f03361b171f1a58051e151815581715581518"><span class="__cf_email__" data-cfemail="16726e7a567b777f7a38657e757875387775387578">[email protected]</span> <span class="__cf_email__" data-cfemail="01696d6874416c60686d2f7269626f622f60622f626f">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jiahui Wu</span> - <span class="hlFld-Affiliation affiliation">†Department
of Pharmacology III, ‡Drug Discovery and Design Center, §Department of Analytical Chemistry, and ∥Center for Drug
Metabolism and Pharmacokinetics, Shanghai
Institute of Materia Medica, Chinese Academy of Sciences,
555 Zu-Chong-Zhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">School of Life Sciences, Fudan
University, Shanghai 200433, China</span>; 
    <span class="hlFld-Affiliation affiliation">Institute of Molecular Design and Drug
Discovery, School of Pharmaceutical Sciences, Central South University,
Changsha 410078, Hunan, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dengyou Zhang</span> - <span class="hlFld-Affiliation affiliation">†Department
of Pharmacology III, ‡Drug Discovery and Design Center, §Department of Analytical Chemistry, and ∥Center for Drug
Metabolism and Pharmacokinetics, Shanghai
Institute of Materia Medica, Chinese Academy of Sciences,
555 Zu-Chong-Zhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">School of Life Sciences, Fudan
University, Shanghai 200433, China</span>; 
    <span class="hlFld-Affiliation affiliation">Institute of Molecular Design and Drug
Discovery, School of Pharmaceutical Sciences, Central South University,
Changsha 410078, Hunan, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lei Chen</span> - <span class="hlFld-Affiliation affiliation">†Department
of Pharmacology III, ‡Drug Discovery and Design Center, §Department of Analytical Chemistry, and ∥Center for Drug
Metabolism and Pharmacokinetics, Shanghai
Institute of Materia Medica, Chinese Academy of Sciences,
555 Zu-Chong-Zhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">School of Life Sciences, Fudan
University, Shanghai 200433, China</span>; 
    <span class="hlFld-Affiliation affiliation">Institute of Molecular Design and Drug
Discovery, School of Pharmaceutical Sciences, Central South University,
Changsha 410078, Hunan, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jianneng Li</span> - <span class="hlFld-Affiliation affiliation">School of Life Sciences, Fudan
University, Shanghai 200433, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jianling Wang</span> - <span class="hlFld-Affiliation affiliation">†Department
of Pharmacology III, ‡Drug Discovery and Design Center, §Department of Analytical Chemistry, and ∥Center for Drug
Metabolism and Pharmacokinetics, Shanghai
Institute of Materia Medica, Chinese Academy of Sciences,
555 Zu-Chong-Zhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">School of Life Sciences, Fudan
University, Shanghai 200433, China</span>; 
    <span class="hlFld-Affiliation affiliation">Institute of Molecular Design and Drug
Discovery, School of Pharmaceutical Sciences, Central South University,
Changsha 410078, Hunan, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chengqing Ning</span> - <span class="hlFld-Affiliation affiliation">Institute of Molecular Design and Drug
Discovery, School of Pharmaceutical Sciences, Central South University,
Changsha 410078, Hunan, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Niefang Yu</span> - <span class="hlFld-Affiliation affiliation">Institute of Molecular Design and Drug
Discovery, School of Pharmaceutical Sciences, Central South University,
Changsha 410078, Hunan, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fei Zhao</span> - <span class="hlFld-Affiliation affiliation">†Department
of Pharmacology III, ‡Drug Discovery and Design Center, §Department of Analytical Chemistry, and ∥Center for Drug
Metabolism and Pharmacokinetics, Shanghai
Institute of Materia Medica, Chinese Academy of Sciences,
555 Zu-Chong-Zhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">School of Life Sciences, Fudan
University, Shanghai 200433, China</span>; 
    <span class="hlFld-Affiliation affiliation">Institute of Molecular Design and Drug
Discovery, School of Pharmaceutical Sciences, Central South University,
Changsha 410078, Hunan, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dongying Chen</span> - <span class="hlFld-Affiliation affiliation">†Department
of Pharmacology III, ‡Drug Discovery and Design Center, §Department of Analytical Chemistry, and ∥Center for Drug
Metabolism and Pharmacokinetics, Shanghai
Institute of Materia Medica, Chinese Academy of Sciences,
555 Zu-Chong-Zhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">School of Life Sciences, Fudan
University, Shanghai 200433, China</span>; 
    <span class="hlFld-Affiliation affiliation">Institute of Molecular Design and Drug
Discovery, School of Pharmaceutical Sciences, Central South University,
Changsha 410078, Hunan, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaoyan Chen</span> - <span class="hlFld-Affiliation affiliation">†Department
of Pharmacology III, ‡Drug Discovery and Design Center, §Department of Analytical Chemistry, and ∥Center for Drug
Metabolism and Pharmacokinetics, Shanghai
Institute of Materia Medica, Chinese Academy of Sciences,
555 Zu-Chong-Zhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">School of Life Sciences, Fudan
University, Shanghai 200433, China</span>; 
    <span class="hlFld-Affiliation affiliation">Institute of Molecular Design and Drug
Discovery, School of Pharmaceutical Sciences, Central South University,
Changsha 410078, Hunan, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kaixian Chen</span> - <span class="hlFld-Affiliation affiliation">†Department
of Pharmacology III, ‡Drug Discovery and Design Center, §Department of Analytical Chemistry, and ∥Center for Drug
Metabolism and Pharmacokinetics, Shanghai
Institute of Materia Medica, Chinese Academy of Sciences,
555 Zu-Chong-Zhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">School of Life Sciences, Fudan
University, Shanghai 200433, China</span>; 
    <span class="hlFld-Affiliation affiliation">Institute of Molecular Design and Drug
Discovery, School of Pharmaceutical Sciences, Central South University,
Changsha 410078, Hunan, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hualiang Jiang</span> - <span class="hlFld-Affiliation affiliation">†Department
of Pharmacology III, ‡Drug Discovery and Design Center, §Department of Analytical Chemistry, and ∥Center for Drug
Metabolism and Pharmacokinetics, Shanghai
Institute of Materia Medica, Chinese Academy of Sciences,
555 Zu-Chong-Zhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">School of Life Sciences, Fudan
University, Shanghai 200433, China</span>; 
    <span class="hlFld-Affiliation affiliation">Institute of Molecular Design and Drug
Discovery, School of Pharmaceutical Sciences, Central South University,
Changsha 410078, Hunan, China</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>These authors contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d164e5212-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i91">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_40832" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_40832" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Prof. Tony Kouzarides for kindly providing the plasmid of SIRT1, Prof. Anita Büttner for kindly providing the plasmid of SIRT2, and Prof. Clemens Steegborn for kindly providing the plasmid of SIRT3. The work was supported by the National Science Fund (grant 21021063, grant 91229204) and the Science and Technology Commission of Shanghai Municipality (grant 08JC1422100).</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i92" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i92"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i93" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i93"> Abbreviations Used</h2><tr><td class="NLM_term">KDAC</td><td class="NLM_def"><p class="first last">lysine deacetylase</p></td></tr><tr><td class="NLM_term">NAD<sup>+</sup></td><td class="NLM_def"><p class="first last">nicotinamide adenine dinucleotide</p></td></tr><tr><td class="NLM_term">PPARγ, peroxisome proliferator activated receptor γ</td><td class="NLM_def"></td></tr><tr><td class="NLM_term">FOXO</td><td class="NLM_def"><p class="first last">forkhead homeobox type O</p></td></tr><tr><td class="NLM_term">BESTO</td><td class="NLM_def"><p class="first last">β cell-specific SIRT1-overexpressing</p></td></tr><tr><td class="NLM_term">Ucp2</td><td class="NLM_def"><p class="first last">uncoupling protein 2</p></td></tr><tr><td class="NLM_term">PGC-1α</td><td class="NLM_def"><p class="first last">peroxisome proliferator activated receptor gamma coactivator 1</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">AMC</td><td class="NLM_def"><p class="first last">4-amino-7-methylcoumarin</p></td></tr><tr><td class="NLM_term">ITC</td><td class="NLM_def"><p class="first last">isothermal titration calorimetry</p></td></tr><tr><td class="NLM_term">ALY</td><td class="NLM_def"><p class="first last">acetyl-lysine</p></td></tr><tr><td class="NLM_term">DNMT1</td><td class="NLM_def"><p class="first last">DNA methyltransferase 1</p></td></tr><tr><td class="NLM_term">DOX</td><td class="NLM_def"><p class="first last">doxorubicin</p></td></tr><tr><td class="NLM_term">TSA</td><td class="NLM_def"><p class="first last">trichostatin</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i94">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10065" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10065" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 60 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Landry, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sutton, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tafrov, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heller, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stebbins, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pillus, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sternglanz, R.</span><span> </span><span class="NLM_article-title">The silencing protein SIR2 and its homologs are NAD-dependent protein deacetylases</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">97</span><span class="NLM_x">, </span> <span class="NLM_fpage">5807</span><span class="NLM_x">–</span> <span class="NLM_lpage">5811</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2000&pages=5807-5811&author=J.+Landryauthor=A.+Suttonauthor=S.+T.+Tafrovauthor=R.+C.+Hellerauthor=J.+Stebbinsauthor=L.+Pillusauthor=R.+Sternglanz&title=The+silencing+protein+SIR2+and+its+homologs+are+NAD-dependent+protein+deacetylases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLandry%26aufirst%3DJ.%26aulast%3DSutton%26aufirst%3DA.%26aulast%3DTafrov%26aufirst%3DS.%2BT.%26aulast%3DHeller%26aufirst%3DR.%2BC.%26aulast%3DStebbins%26aufirst%3DJ.%26aulast%3DPillus%26aufirst%3DL.%26aulast%3DSternglanz%26aufirst%3DR.%26atitle%3DThe%2520silencing%2520protein%2520SIR2%2520and%2520its%2520homologs%2520are%2520NAD-dependent%2520protein%2520deacetylases%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2000%26volume%3D97%26spage%3D5807%26epage%3D5811" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">North, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdin, E.</span><span> </span><span class="NLM_article-title">Sirtuins: Sir2-related NAD-dependent protein deacetylases</span> <span class="citation_source-journal">Genome Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">224</span><span class="NLM_x">–</span> <span class="NLM_lpage">236</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2004&pages=224-236&author=B.+J.+Northauthor=E.+Verdin&title=Sirtuins%3A+Sir2-related+NAD-dependent+protein+deacetylases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DNorth%26aufirst%3DB.%2BJ.%26aulast%3DVerdin%26aufirst%3DE.%26atitle%3DSirtuins%253A%2520Sir2-related%2520NAD-dependent%2520protein%2520deacetylases%26jtitle%3DGenome%2520Biol.%26date%3D2004%26volume%3D5%26spage%3D224%26epage%3D236" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Vaquero, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scher, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Erdjument-Bromage, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tempst, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinberg, D.</span><span> </span><span class="NLM_article-title">Human SirT1 interacts with histone H1 and promotes formation of facultative heterochromatin</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">93</span><span class="NLM_x">–</span> <span class="NLM_lpage">105</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1016%2Fj.molcel.2004.08.031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm301032j&amp;key=15469825" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BD2cXptVKqtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2004&pages=93-105&author=A.+Vaqueroauthor=M.+Scherauthor=D.+H.+Leeauthor=H.+Erdjument-Bromageauthor=P.+Tempstauthor=D.+Reinberg&title=Human+SirT1+interacts+with+histone+H1+and+promotes+formation+of+facultative+heterochromatin"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Human SirT1 interacts with histone H1 and promotes formation of facultative heterochromatin</span></div><div class="casAuthors">Vaquero, Alejandro; Scher, Michael; Lee, Donghoon; Erdjument-Bromage, Hediye; Tempst, Paul; Reinberg, Danny</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">93-105</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">We characterized human SirT1, one of the human homologs of the budding yeast Sir2p, an NAD+-dependent histone deacetylase involved in establishing repressive chromatin and increased life span.  SirT1 deacetylates histone polypeptides with a preference for histone H4 lysine 16 (H4-K16Ac) and H3 lysine 9 (H3-K9Ac) in vitro.  RNAi-mediated decreased expression of SirT1 in human cells causes hyperacetylation of H4-K16 and H3-K9 in vivo.  SirT1 interacts with and deacetylates histone H1 at lysine 26.  Using an inducible system directing expression of SirT1 fused to the Gal4-DNA binding domain and a Gal4-reporter integrated in euchromatin, Gal4-SirT1 expression resulted in the deacetylation of H4-K16 and H3-K9, recruitment of H1 within the promoter vicinity, drastically reduced reporter expression, and loss of H3-K79 methylation, a mark restricting silenced chromatin.  We propose a model for SirT1-mediated heterochromatin formation that includes deacetylation of histone tails, recruitment and deacetylation of histone H1, and spreading of hypomethylated H3-K79 with resultant silencing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcmKK5RHZ4nrVg90H21EOLACvtfcHk0lgYfgrT3nd9Ow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXptVKqtrw%253D&md5=d342a46aac48b767ef497e34104750e5</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2004.08.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2004.08.031%26sid%3Dliteratum%253Aachs%26aulast%3DVaquero%26aufirst%3DA.%26aulast%3DScher%26aufirst%3DM.%26aulast%3DLee%26aufirst%3DD.%2BH.%26aulast%3DErdjument-Bromage%26aufirst%3DH.%26aulast%3DTempst%26aufirst%3DP.%26aulast%3DReinberg%26aufirst%3DD.%26atitle%3DHuman%2520SirT1%2520interacts%2520with%2520histone%2520H1%2520and%2520promotes%2520formation%2520of%2520facultative%2520heterochromatin%26jtitle%3DMol.%2520Cell%26date%3D2004%26volume%3D16%26spage%3D93%26epage%3D105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Vaquero, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sternglanz, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinberg, D.</span><span> </span><span class="NLM_article-title">NAD<sup>+</sup>-dependent deacetylation of H4 lysine 16 by class III HDACs</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">5505</span><span class="NLM_x">–</span> <span class="NLM_lpage">5520</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2007&pages=5505-5520&author=A.+Vaqueroauthor=R.+Sternglanzauthor=D.+Reinberg&title=NAD%2B-dependent+deacetylation+of+H4+lysine+16+by+class+III+HDACs"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVaquero%26aufirst%3DA.%26aulast%3DSternglanz%26aufirst%3DR.%26aulast%3DReinberg%26aufirst%3DD.%26atitle%3DNAD%252B-dependent%2520deacetylation%2520of%2520H4%2520lysine%252016%2520by%2520class%2520III%2520HDACs%26jtitle%3DOncogene%26date%3D2007%26volume%3D26%26spage%3D5505%26epage%3D5520" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Picard, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurtev, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Topark-Ngarm, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senawong, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Machado De Oliveira, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leid, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McBurney, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guarente, L.</span><span> </span><span class="NLM_article-title">Sirt1 promotes fat mobilization in white adipocytes by repressing PPAR-gamma</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">429</span><span class="NLM_x">, </span> <span class="NLM_fpage">771</span><span class="NLM_x">–</span> <span class="NLM_lpage">776</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=429&publication_year=2004&pages=771-776&author=F.+Picardauthor=M.+Kurtevauthor=N.+Chungauthor=A.+Topark-Ngarmauthor=T.+Senawongauthor=R.+Machado+De+Oliveiraauthor=M.+Leidauthor=M.+W.+McBurneyauthor=L.+Guarente&title=Sirt1+promotes+fat+mobilization+in+white+adipocytes+by+repressing+PPAR-gamma"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPicard%26aufirst%3DF.%26aulast%3DKurtev%26aufirst%3DM.%26aulast%3DChung%26aufirst%3DN.%26aulast%3DTopark-Ngarm%26aufirst%3DA.%26aulast%3DSenawong%26aufirst%3DT.%26aulast%3DMachado%2BDe%2BOliveira%26aufirst%3DR.%26aulast%3DLeid%26aufirst%3DM.%26aulast%3DMcBurney%26aufirst%3DM.%2BW.%26aulast%3DGuarente%26aufirst%3DL.%26atitle%3DSirt1%2520promotes%2520fat%2520mobilization%2520in%2520white%2520adipocytes%2520by%2520repressing%2520PPAR-gamma%26jtitle%3DNature%26date%3D2004%26volume%3D429%26spage%3D771%26epage%3D776" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Qiao, L. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shao, J. H.</span><span> </span><span class="NLM_article-title">SIRT1 regulates adiponectin gene expression through Foxo1-C/enhancer-binding protein alpha transcriptional complex</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">281</span><span class="NLM_x">, </span> <span class="NLM_fpage">39915</span><span class="NLM_x">–</span> <span class="NLM_lpage">39924</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=39915-39924&author=L.+P.+Qiaoauthor=J.+H.+Shao&title=SIRT1+regulates+adiponectin+gene+expression+through+Foxo1-C%2Fenhancer-binding+protein+alpha+transcriptional+complex"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DQiao%26aufirst%3DL.%2BP.%26aulast%3DShao%26aufirst%3DJ.%2BH.%26atitle%3DSIRT1%2520regulates%2520adiponectin%2520gene%2520expression%2520through%2520Foxo1-C%252Fenhancer-binding%2520protein%2520alpha%2520transcriptional%2520complex%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D39915%26epage%3D39924" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Brunet, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sweeney, L. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sturgill, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chua, K. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greer, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tran, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ross, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mostoslavsky, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohen, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, L. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jedrychowski, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gygi, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sinclair, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alt, F. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenberg, M. E.</span><span> </span><span class="NLM_article-title">Stress-dependent regulation of FOXO transcription factors by the SIRT1 deacetylase</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">303</span><span class="NLM_x">, </span> <span class="NLM_fpage">2011</span><span class="NLM_x">–</span> <span class="NLM_lpage">2015</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=303&publication_year=2004&pages=2011-2015&author=A.+Brunetauthor=L.+B.+Sweeneyauthor=J.+F.+Sturgillauthor=K.+F.+Chuaauthor=P.+L.+Greerauthor=Y.+Linauthor=H.+Tranauthor=S.+E.+Rossauthor=R.+Mostoslavskyauthor=H.+Y.+Cohenauthor=L.+S.+Huauthor=H.+L.+Chengauthor=M.+P.+Jedrychowskiauthor=S.+P.+Gygiauthor=D.+A.+Sinclairauthor=F.+W.+Altauthor=M.+E.+Greenberg&title=Stress-dependent+regulation+of+FOXO+transcription+factors+by+the+SIRT1+deacetylase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBrunet%26aufirst%3DA.%26aulast%3DSweeney%26aufirst%3DL.%2BB.%26aulast%3DSturgill%26aufirst%3DJ.%2BF.%26aulast%3DChua%26aufirst%3DK.%2BF.%26aulast%3DGreer%26aufirst%3DP.%2BL.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DTran%26aufirst%3DH.%26aulast%3DRoss%26aufirst%3DS.%2BE.%26aulast%3DMostoslavsky%26aufirst%3DR.%26aulast%3DCohen%26aufirst%3DH.%2BY.%26aulast%3DHu%26aufirst%3DL.%2BS.%26aulast%3DCheng%26aufirst%3DH.%2BL.%26aulast%3DJedrychowski%26aufirst%3DM.%2BP.%26aulast%3DGygi%26aufirst%3DS.%2BP.%26aulast%3DSinclair%26aufirst%3DD.%2BA.%26aulast%3DAlt%26aufirst%3DF.%2BW.%26aulast%3DGreenberg%26aufirst%3DM.%2BE.%26atitle%3DStress-dependent%2520regulation%2520of%2520FOXO%2520transcription%2520factors%2520by%2520the%2520SIRT1%2520deacetylase%26jtitle%3DScience%26date%3D2004%26volume%3D303%26spage%3D2011%26epage%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Bordone, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Motta, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picard, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robinson, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jhala, U. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Apfeld, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonagh, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lemieux, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McBurney, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szilvasi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Easlon, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guarente, L.</span><span> </span><span class="NLM_article-title">Sirt1 regulates insulin secretion by repressing UCP2 in pancreatic beta cells</span> <span class="citation_source-journal">PLoS Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">210</span><span class="NLM_x">–</span> <span class="NLM_lpage">220</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2006&pages=210-220&author=L.+Bordoneauthor=M.+C.+Mottaauthor=F.+Picardauthor=A.+Robinsonauthor=U.+S.+Jhalaauthor=J.+Apfeldauthor=T.+McDonaghauthor=M.+Lemieuxauthor=M.+McBurneyauthor=A.+Szilvasiauthor=E.+J.+Easlonauthor=S.+J.+Linauthor=L.+Guarente&title=Sirt1+regulates+insulin+secretion+by+repressing+UCP2+in+pancreatic+beta+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBordone%26aufirst%3DL.%26aulast%3DMotta%26aufirst%3DM.%2BC.%26aulast%3DPicard%26aufirst%3DF.%26aulast%3DRobinson%26aufirst%3DA.%26aulast%3DJhala%26aufirst%3DU.%2BS.%26aulast%3DApfeld%26aufirst%3DJ.%26aulast%3DMcDonagh%26aufirst%3DT.%26aulast%3DLemieux%26aufirst%3DM.%26aulast%3DMcBurney%26aufirst%3DM.%26aulast%3DSzilvasi%26aufirst%3DA.%26aulast%3DEaslon%26aufirst%3DE.%2BJ.%26aulast%3DLin%26aufirst%3DS.%2BJ.%26aulast%3DGuarente%26aufirst%3DL.%26atitle%3DSirt1%2520regulates%2520insulin%2520secretion%2520by%2520repressing%2520UCP2%2520in%2520pancreatic%2520beta%2520cells%26jtitle%3DPLoS%2520Biol.%26date%3D2006%26volume%3D4%26spage%3D210%26epage%3D220" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Ramsey, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mills, K. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Satoh, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Imai, S. I.</span><span> </span><span class="NLM_article-title">Age-associated loss of Sirt1-mediated enhancement of glucose-stimulated insulin secretion in beta cell-specific Sirt1-overexpressing (BESTO) mice</span> <span class="citation_source-journal">Aging Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">78</span><span class="NLM_x">–</span> <span class="NLM_lpage">88</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=78-88&author=K.+M.+Ramseyauthor=K.+F.+Millsauthor=A.+Satohauthor=S.+I.+Imai&title=Age-associated+loss+of+Sirt1-mediated+enhancement+of+glucose-stimulated+insulin+secretion+in+beta+cell-specific+Sirt1-overexpressing+%28BESTO%29+mice"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRamsey%26aufirst%3DK.%2BM.%26aulast%3DMills%26aufirst%3DK.%2BF.%26aulast%3DSatoh%26aufirst%3DA.%26aulast%3DImai%26aufirst%3DS.%2BI.%26atitle%3DAge-associated%2520loss%2520of%2520Sirt1-mediated%2520enhancement%2520of%2520glucose-stimulated%2520insulin%2520secretion%2520in%2520beta%2520cell-specific%2520Sirt1-overexpressing%2520%2528BESTO%2529%2520mice%26jtitle%3DAging%2520Cell%26date%3D2008%26volume%3D7%26spage%3D78%26epage%3D88" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Rodgers, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lerin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haas, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gygi, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spiegelman, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Puigserver, P.</span><span> </span><span class="NLM_article-title">Nutrient control of glucose homeostasis through a complex of PGC-1alpha and SIRT1</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">434</span><span class="NLM_x">, </span> <span class="NLM_fpage">113</span><span class="NLM_x">–</span> <span class="NLM_lpage">118</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=434&publication_year=2005&pages=113-118&author=J.+T.+Rodgersauthor=C.+Lerinauthor=W.+Haasauthor=S.+P.+Gygiauthor=B.+M.+Spiegelmanauthor=P.+Puigserver&title=Nutrient+control+of+glucose+homeostasis+through+a+complex+of+PGC-1alpha+and+SIRT1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRodgers%26aufirst%3DJ.%2BT.%26aulast%3DLerin%26aufirst%3DC.%26aulast%3DHaas%26aufirst%3DW.%26aulast%3DGygi%26aufirst%3DS.%2BP.%26aulast%3DSpiegelman%26aufirst%3DB.%2BM.%26aulast%3DPuigserver%26aufirst%3DP.%26atitle%3DNutrient%2520control%2520of%2520glucose%2520homeostasis%2520through%2520a%2520complex%2520of%2520PGC-1alpha%2520and%2520SIRT1%26jtitle%3DNature%26date%3D2005%26volume%3D434%26spage%3D113%26epage%3D118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Lagouge, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Argmann, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerhart-Hines, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meziane, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lerin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daussin, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Messadeq, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milne, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elliott, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geny, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laakso, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Puigserver, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Auwerx, J.</span><span> </span><span class="NLM_article-title">Resveratrol improves mitochondrial function and protects against metabolic disease by activating SIRT1 and PGC-1 alpha</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">127</span><span class="NLM_x">, </span> <span class="NLM_fpage">1109</span><span class="NLM_x">–</span> <span class="NLM_lpage">1122</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2006&pages=1109-1122&author=M.+Lagougeauthor=C.+Argmannauthor=Z.+Gerhart-Hinesauthor=H.+Mezianeauthor=C.+Lerinauthor=F.+Daussinauthor=N.+Messadeqauthor=J.+Milneauthor=P.+Lambertauthor=P.+Elliottauthor=B.+Genyauthor=M.+Laaksoauthor=P.+Puigserverauthor=J.+Auwerx&title=Resveratrol+improves+mitochondrial+function+and+protects+against+metabolic+disease+by+activating+SIRT1+and+PGC-1+alpha"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLagouge%26aufirst%3DM.%26aulast%3DArgmann%26aufirst%3DC.%26aulast%3DGerhart-Hines%26aufirst%3DZ.%26aulast%3DMeziane%26aufirst%3DH.%26aulast%3DLerin%26aufirst%3DC.%26aulast%3DDaussin%26aufirst%3DF.%26aulast%3DMessadeq%26aufirst%3DN.%26aulast%3DMilne%26aufirst%3DJ.%26aulast%3DLambert%26aufirst%3DP.%26aulast%3DElliott%26aufirst%3DP.%26aulast%3DGeny%26aufirst%3DB.%26aulast%3DLaakso%26aufirst%3DM.%26aulast%3DPuigserver%26aufirst%3DP.%26aulast%3DAuwerx%26aufirst%3DJ.%26atitle%3DResveratrol%2520improves%2520mitochondrial%2520function%2520and%2520protects%2520against%2520metabolic%2520disease%2520by%2520activating%2520SIRT1%2520and%2520PGC-1%2520alpha%26jtitle%3DCell%26date%3D2006%26volume%3D127%26spage%3D1109%26epage%3D1122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Blander, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guarente, L.</span><span> </span><span class="NLM_article-title">The sir2 family of protein deacetylases</span> <span class="citation_source-journal">Annu. Rev. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">73</span><span class="NLM_x">, </span> <span class="NLM_fpage">417</span><span class="NLM_x">–</span> <span class="NLM_lpage">435</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1146%2Fannurev.biochem.73.011303.073651" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Fjm301032j&amp;key=15189148" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmslags7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2004&pages=417-435&author=G.+Blanderauthor=L.+Guarente&title=The+sir2+family+of+protein+deacetylases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">The Sir2 family of protein deacetylases</span></div><div class="casAuthors">Blander, Gil; Guarente, Leonard</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">417-435</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  The yeast SIR protein complex has been implicated in transcription silencing and suppression of recombination.  The Sir complex represses transcription at telomeres, mating-type loci, and ribosomal DNA.  Unlike SIR3 and SIR4, the SIR2 gene is highly conserved in organisms ranging from archaea to humans.  Interestingly, Sir2 is active as an NAD+-dependent deacetylase, which is broadly conserved from bacteria to higher eukaryotes.  In this review, we discuss the role of NAD+, the unusual products of the deacetylation reaction, the Sir2 structure, and the Sir2 chem. inhibitors and activators that were recently identified.  We summarize the current knowledge of the Sir2 homologs from different organisms, and finally we discuss the role of Sir2 in caloric restriction and aging.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGOpp_Sk7r0rVg90H21EOLACvtfcHk0liqi0uB04FXQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmslags7c%253D&md5=0ee832d37d4865477d458ea5ce579c6a</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biochem.73.011303.073651&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biochem.73.011303.073651%26sid%3Dliteratum%253Aachs%26aulast%3DBlander%26aufirst%3DG.%26aulast%3DGuarente%26aufirst%3DL.%26atitle%3DThe%2520sir2%2520family%2520of%2520protein%2520deacetylases%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2004%26volume%3D73%26spage%3D417%26epage%3D435" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Longo, V. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennedy, B. K.</span><span> </span><span class="NLM_article-title">Sirtuins in aging and age-related disease</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">126</span><span class="NLM_x">, </span> <span class="NLM_fpage">257</span><span class="NLM_x">–</span> <span class="NLM_lpage">268</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1016%2Fj.cell.2006.07.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Fjm301032j&amp;key=16873059" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BD28Xot1Ont7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2006&pages=257-268&author=V.+D.+Longoauthor=B.+K.+Kennedy&title=Sirtuins+in+aging+and+age-related+disease"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Sirtuins in aging and age-related disease</span></div><div class="casAuthors">Longo, Valter D.; Kennedy, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">257-268</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Sirtuins have been the focus of intense scrutiny since the discovery of Sir2 as a yeast longevity factor.  Functioning as either deacetylases or ADP ribosylases, Sirtuins are regulated by the cofactor NAD and thus may serve as sensors of the metabolic state of the cell and organism.  Here we examine the roles of Sirtuins in diverse eukaryotic species, with special emphasis on their links to aging and age-related diseases including cancer, diabetes, and neurodegenerative disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVJ6UnhyJ7q7Vg90H21EOLACvtfcHk0liqi0uB04FXQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xot1Ont7g%253D&md5=a903a23be3b021edb7ed2b29f0f2c3b3</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2006.07.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2006.07.002%26sid%3Dliteratum%253Aachs%26aulast%3DLongo%26aufirst%3DV.%2BD.%26aulast%3DKennedy%26aufirst%3DB.%2BK.%26atitle%3DSirtuins%2520in%2520aging%2520and%2520age-related%2520disease%26jtitle%3DCell%26date%3D2006%26volume%3D126%26spage%3D257%26epage%3D268" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Howitz, K. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bitterman, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohen, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamming, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lavu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wood, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zipkin, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kisielewski, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, L.-L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scherer, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sinclair, D. A.</span><span> </span><span class="NLM_article-title">Small molecule activators of sirtuins extend <i>Saccharomyces cerevisiae</i> lifespan</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">425</span><span class="NLM_x">, </span> <span class="NLM_fpage">191</span><span class="NLM_x">–</span> <span class="NLM_lpage">196</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1038%2Fnature01960" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Fjm301032j&amp;key=12939617" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BD3sXntFCitr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=425&publication_year=2003&pages=191-196&author=K.+T.+Howitzauthor=K.+J.+Bittermanauthor=H.+Y.+Cohenauthor=D.+W.+Lammingauthor=S.+Lavuauthor=J.+G.+Woodauthor=R.+E.+Zipkinauthor=P.+Chungauthor=A.+Kisielewskiauthor=L.-L.+Zhangauthor=B.+Schererauthor=D.+A.+Sinclair&title=Small+molecule+activators+of+sirtuins+extend+Saccharomyces+cerevisiae+lifespan"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule activators of sirtuins extend Saccharomyces cerevisiae lifespan</span></div><div class="casAuthors">Howitz, Konrad T.; Bitterman, Kevin J.; Cohen, Haim Y.; Lamming, Dudley W.; Lavu, Siva; Wood, Jason G.; Zipkin, Robert E.; Chung, Phuong; Kisielewski, Anne; Zhang, Li-Li; Scherer, Brandy; Sinclair, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">425</span>
        (<span class="NLM_cas:issue">6954</span>),
    <span class="NLM_cas:pages">191-196</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In diverse organisms, calorie restriction slows the pace of ageing and increases max. lifespan.  In the budding yeast S. cerevisiae, calorie restriction extends lifespan by increasing the activity of Sir2, a member of the conserved sirtuin family of NAD+-dependent protein deacetylases.  Included in this family are SIR-2.1, a Caenorhabditis elegans enzyme that regulates lifespan, and SIRT1, a human deacetylase that promotes cell survival by neg. regulating the p53 tumor suppressor.  Here we report the discovery of 3 classes of small mols. that activate sirtuins.  We show that the potent activator resveratrol, a polyphenol found in red wine, lowers the Michaelis const. of SIRT1 for both the acetylated substrate and NAD+, and increases cell survival by stimulating SIRT1-dependent deacetylation of p53.  In yeast, resveratrol mimics calorie restriction by stimulating Sir2, increasing DNA stability and extending lifespan by 70%.  We discuss possible evolutionary origins of this phenomenon and suggest new lines of research into the therapeutic use of sirtuin activators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBrU9uI44d2bVg90H21EOLACvtfcHk0lg2fVMDHjG-dQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXntFCitr4%253D&md5=28d0ccc7b9b1c25358eebb8662177c96</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnature01960&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature01960%26sid%3Dliteratum%253Aachs%26aulast%3DHowitz%26aufirst%3DK.%2BT.%26aulast%3DBitterman%26aufirst%3DK.%2BJ.%26aulast%3DCohen%26aufirst%3DH.%2BY.%26aulast%3DLamming%26aufirst%3DD.%2BW.%26aulast%3DLavu%26aufirst%3DS.%26aulast%3DWood%26aufirst%3DJ.%2BG.%26aulast%3DZipkin%26aufirst%3DR.%2BE.%26aulast%3DChung%26aufirst%3DP.%26aulast%3DKisielewski%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DL.-L.%26aulast%3DScherer%26aufirst%3DB.%26aulast%3DSinclair%26aufirst%3DD.%2BA.%26atitle%3DSmall%2520molecule%2520activators%2520of%2520sirtuins%2520extend%2520Saccharomyces%2520cerevisiae%2520lifespan%26jtitle%3DNature%26date%3D2003%26volume%3D425%26spage%3D191%26epage%3D196" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Wood, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogina, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lavu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howitz, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Helfand, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tatar, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sinclair, D.</span><span> </span><span class="NLM_article-title">Sirtuin activators mimic caloric restriction and delay ageing in metazoans</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">430</span><span class="NLM_x">, </span> <span class="NLM_fpage">686</span><span class="NLM_x">–</span> <span class="NLM_lpage">689</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=430&publication_year=2004&pages=686-689&author=J.+G.+Woodauthor=B.+Roginaauthor=S.+Lavuauthor=K.+Howitzauthor=S.+L.+Helfandauthor=M.+Tatarauthor=D.+Sinclair&title=Sirtuin+activators+mimic+caloric+restriction+and+delay+ageing+in+metazoans"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWood%26aufirst%3DJ.%2BG.%26aulast%3DRogina%26aufirst%3DB.%26aulast%3DLavu%26aufirst%3DS.%26aulast%3DHowitz%26aufirst%3DK.%26aulast%3DHelfand%26aufirst%3DS.%2BL.%26aulast%3DTatar%26aufirst%3DM.%26aulast%3DSinclair%26aufirst%3DD.%26atitle%3DSirtuin%2520activators%2520mimic%2520caloric%2520restriction%2520and%2520delay%2520ageing%2520in%2520metazoans%26jtitle%3DNature%26date%3D2004%26volume%3D430%26spage%3D686%26epage%3D689" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Bauer, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goupil, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garber, G. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Helfand, S. L.</span><span> </span><span class="NLM_article-title">An accelerated assay for the identification of lifespan-extending interventions in <i>Drosophila melanogaster</i></span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">101</span><span class="NLM_x">, </span> <span class="NLM_fpage">12980</span><span class="NLM_x">–</span> <span class="NLM_lpage">12985</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=12980-12985&author=J.+H.+Bauerauthor=S.+Goupilauthor=G.+B.+Garberauthor=S.+L.+Helfand&title=An+accelerated+assay+for+the+identification+of+lifespan-extending+interventions+in+Drosophila+melanogaster"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBauer%26aufirst%3DJ.%2BH.%26aulast%3DGoupil%26aufirst%3DS.%26aulast%3DGarber%26aufirst%3DG.%2BB.%26aulast%3DHelfand%26aufirst%3DS.%2BL.%26atitle%3DAn%2520accelerated%2520assay%2520for%2520the%2520identification%2520of%2520lifespan-extending%2520interventions%2520in%2520Drosophila%2520melanogaster%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2004%26volume%3D101%26spage%3D12980%26epage%3D12985" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Valenzano, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terzibasi, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Genade, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cattaneo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Domenici, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cellerino, A.</span><span> </span><span class="NLM_article-title">Resveratrol prolongs lifespan and retards the onset of age-related markers in a short-lived vertebrate</span> <span class="citation_source-journal">Curr. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">296</span><span class="NLM_x">–</span> <span class="NLM_lpage">300</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1016%2Fj.cub.2005.12.038" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Fjm301032j&amp;key=16461283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlKltL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2006&pages=296-300&author=D.+R.+Valenzanoauthor=E.+Terzibasiauthor=T.+Genadeauthor=A.+Cattaneoauthor=L.+Domeniciauthor=A.+Cellerino&title=Resveratrol+prolongs+lifespan+and+retards+the+onset+of+age-related+markers+in+a+short-lived+vertebrate"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Resveratrol Prolongs Lifespan and Retards the Onset of Age-Related Markers in a Short-Lived Vertebrate</span></div><div class="casAuthors">Valenzano, Dario R.; Terzibasi, Eva; Genade, Tyrone; Cattaneo, Antonino; Domenici, Luciano; Cellerino, Alessandro</div><div class="citationInfo"><span class="NLM_cas:title">Current Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">296-300</span>CODEN:
                <span class="NLM_cas:coden">CUBLE2</span>;
        ISSN:<span class="NLM_cas:issn">0960-9822</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Resveratrol, a natural phytoalexin found in grapes and red wine, increases longevity in the short-lived invertebrates Caenorhabditis elegans and Drosophila and exerts a variety of biol. effects in vertebrates, including protection from ischemia and neurotoxicity.  Its effects on vertebrate lifespan were not yet known.  The relatively long lifespan of mice, which live at least 2.5 years, is a hurdle for life-long pharmacol. trials.  Here, the authors used the short-lived seasonal fish Nothobranchius furzeri with a max. recorded lifespan of 13 wk in captivity.  Short lifespan in this species is not the result of spontaneous or targeted genetic mutations, but a natural trait correlated with the necessity to breed in an ephemeral habitat and tied with accelerated development and expression of aging biomarkers at a cellular level.  Resveratrol was added to the food starting in early adulthood and caused a dose-dependent increase of median and max. lifespan.  In addn., resveratrol delays the age-dependent decay of locomotor activity and cognitive performances and reduces the expression of neurofibrillary degeneration in the brain.  These results demonstrate that food supplementation with resveratrol promotes longevity and retards the expression of age-dependent traits in a short-lived vertebrate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQmwGN6cM6-rVg90H21EOLACvtfcHk0lg2fVMDHjG-dQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlKltL4%253D&md5=522acaa576dae43ba50672e86716c45d</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.cub.2005.12.038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cub.2005.12.038%26sid%3Dliteratum%253Aachs%26aulast%3DValenzano%26aufirst%3DD.%2BR.%26aulast%3DTerzibasi%26aufirst%3DE.%26aulast%3DGenade%26aufirst%3DT.%26aulast%3DCattaneo%26aufirst%3DA.%26aulast%3DDomenici%26aufirst%3DL.%26aulast%3DCellerino%26aufirst%3DA.%26atitle%3DResveratrol%2520prolongs%2520lifespan%2520and%2520retards%2520the%2520onset%2520of%2520age-related%2520markers%2520in%2520a%2520short-lived%2520vertebrate%26jtitle%3DCurr.%2520Biol.%26date%3D2006%26volume%3D16%26spage%3D296%26epage%3D300" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Sun, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ge, X. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, T. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, X. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhai, Q. W.</span><span> </span><span class="NLM_article-title">SIRT1 improves insulin sensitivity under insulin-resistant conditions by repressing PTP1B</span> <span class="citation_source-journal">Cell Metab.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">307</span><span class="NLM_x">–</span> <span class="NLM_lpage">319</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1016%2Fj.cmet.2007.08.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Fjm301032j&amp;key=17908559" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1Sjsr3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=307-319&author=C.+Sunauthor=F.+Zhangauthor=X.+J.+Geauthor=T.+T.+Yanauthor=X.+M.+Chenauthor=X.+L.+Shiauthor=Q.+W.+Zhai&title=SIRT1+improves+insulin+sensitivity+under+insulin-resistant+conditions+by+repressing+PTP1B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">SIRT1 improves insulin sensitivity under insulin-resistant conditions by repressing PTP1B</span></div><div class="casAuthors">Sun, Cheng; Zhang, Fang; Ge, Xinjian; Yan, Tingting; Chen, Xingmiao; Shi, Xianglin; Zhai, Qiwei</div><div class="citationInfo"><span class="NLM_cas:title">Cell Metabolism</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">307-319</span>CODEN:
                <span class="NLM_cas:coden">CMEEB5</span>;
        ISSN:<span class="NLM_cas:issn">1550-4131</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Insulin resistance is often characterized as the most crit. factor contributing to the development of type 2 diabetes.  SIRT1 has been reported to be involved in the processes of glucose metab. and insulin secretion.  However, whether SIRT1 is directly involved in insulin sensitivity is still largely unknown.  Here we show that SIRT1 is downregulated in insulin-resistant cells and tissues and that knockdown or inhibition of SIRT1 induces insulin resistance.  Furthermore, increased expression of SIRT1 improved insulin sensitivity, esp. under insulin-resistant conditions.  Similarly, resveratrol, a SIRT1 activator, enhanced insulin sensitivity in vitro in a SIRT1-dependent manner and attenuated high-fat-diet-induced insulin resistance in vivo at a dose of 2.5 mg/kg/day.  Further studies demonstrated that the effect of SIRT1 on insulin resistance is mediated by repressing PTP1B transcription at the chromatin level.  Taken together, the finding that SIRT1 improves insulin sensitivity has implications toward resolving insulin resistance and type 2 diabetes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3juiZtxzRt7Vg90H21EOLACvtfcHk0ljig6MpI53WmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1Sjsr3L&md5=7b186a99719b49e9ecaf689007824c92</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2007.08.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2007.08.014%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DF.%26aulast%3DGe%26aufirst%3DX.%2BJ.%26aulast%3DYan%26aufirst%3DT.%2BT.%26aulast%3DChen%26aufirst%3DX.%2BM.%26aulast%3DShi%26aufirst%3DX.%2BL.%26aulast%3DZhai%26aufirst%3DQ.%2BW.%26atitle%3DSIRT1%2520improves%2520insulin%2520sensitivity%2520under%2520insulin-resistant%2520conditions%2520by%2520repressing%2520PTP1B%26jtitle%3DCell%2520Metab.%26date%3D2007%26volume%3D6%26spage%3D307%26epage%3D319" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Csiszar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labinskyy, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pinto, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ballabh, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Losonczy, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pearson, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Cabo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pacher, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ungvari, Z.</span><span> </span><span class="NLM_article-title">Resveratrol induces mitochondrial biogenesis in endothelial cells</span> <span class="citation_source-journal">Am. J. Physiol.: Heart Circ. Physiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">297</span><span class="NLM_x">, </span> <span class="NLM_fpage">13</span><span class="NLM_x">–</span> <span class="NLM_lpage">20</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1152%2Fajpheart.00368.2009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Fjm301032j&amp;key=19429820" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BD1MXovVKisL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=297&publication_year=2009&pages=13-20&author=A.+Csiszarauthor=N.+Labinskyyauthor=J.+T.+Pintoauthor=P.+Ballabhauthor=H.+Zhangauthor=G.+Losonczyauthor=K.+Pearsonauthor=R.+de+Caboauthor=P.+Pacherauthor=C.+Zhangauthor=Z.+Ungvari&title=Resveratrol+induces+mitochondrial+biogenesis+in+endothelial+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Resveratrol induces mitochondrial biogenesis in endothelial cells</span></div><div class="casAuthors">Csiszar, Anna; Labinskyy, Nazar; Pinto, John T.; Ballabh, Praveen; Zhang, Hanrui; Losonczy, Gyorgy; Pearson, Kevin; de Cabo, Rafael; Pacher, Pal; Zhang, Cuihua; Ungvari, Zoltan</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Physiology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">297</span>
        (<span class="NLM_cas:issue">1, Pt. 2</span>),
    <span class="NLM_cas:pages">H13-H20</span>CODEN:
                <span class="NLM_cas:coden">AJPHAP</span>;
        ISSN:<span class="NLM_cas:issn">0002-9513</span>.
    
            (<span class="NLM_cas:orgname">American Physiological Society</span>)
        </div><div class="casAbstract">Pathways that regulate mitochondrial biogenesis are potential therapeutic targets for the amelioration of endothelial dysfunction and vascular disease.  Resveratrol was shown to impact mitochondrial function in skeletal muscle and the liver, but its role in mitochondrial biogenesis in endothelial cells remains poorly defined.  The present study detd. whether resveratrol induces mitochondrial biogenesis in cultured human coronary arterial endothelial cells (CAECs).  In CAECs resveratrol increased mitochondrial mass and mitochondrial DNA content, upregulated protein expression of electron transport chain constituents, and induced mitochondrial biogenesis factors (proliferator-activated receptor-coactivator-1α, nuclear respiratory factor-1, mitochondrial transcription factor A).  Sirtuin 1 (SIRT1) was induced, and endothelial nitric oxide (NO) synthase (eNOS) was upregulated in a SIRT1-dependent manner.  Knockdown of SIRT1 (small interfering RNA) or inhibition of NO synthesis prevented resveratrol-induced mitochondrial biogenesis.  In aortas of type 2 diabetic (db/db) mice impaired mitochondrial biogenesis was normalized by chronic resveratrol treatment, showing the in vivo relevance of our findings.  Resveratrol increases mitochondrial content in endothelial cells via activating SIRT1.  We propose that SIRT1, via a pathway that involves the upregulation of eNOS, induces mitochondrial biogenesis.  Resveratrol induced mitochondrial biogenesis in the aortas of type 2 diabetic mice, suggesting the potential for new treatment approaches targeting endothelial mitochondria in metabolic diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSYfYD3QRgsLVg90H21EOLACvtfcHk0ljig6MpI53WmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXovVKisL0%253D&md5=342eac88c1cebf78b818521688f46224</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1152%2Fajpheart.00368.2009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fajpheart.00368.2009%26sid%3Dliteratum%253Aachs%26aulast%3DCsiszar%26aufirst%3DA.%26aulast%3DLabinskyy%26aufirst%3DN.%26aulast%3DPinto%26aufirst%3DJ.%2BT.%26aulast%3DBallabh%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DLosonczy%26aufirst%3DG.%26aulast%3DPearson%26aufirst%3DK.%26aulast%3Dde%2BCabo%26aufirst%3DR.%26aulast%3DPacher%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DUngvari%26aufirst%3DZ.%26atitle%3DResveratrol%2520induces%2520mitochondrial%2520biogenesis%2520in%2520endothelial%2520cells%26jtitle%3DAm.%2520J.%2520Physiol.%253A%2520Heart%2520Circ.%2520Physiol.%26date%3D2009%26volume%3D297%26spage%3D13%26epage%3D20" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Nayagam, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tan, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poulsen, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goh, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ni, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Entzeroth, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stunkel, W.</span><span> </span><span class="NLM_article-title">SIRT1 modulating compounds from high-throughput screening as anti-inflammatory and insulin-sensitizing agents</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">959</span><span class="NLM_x">–</span> <span class="NLM_lpage">967</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1177%2F1087057106294710" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Fjm301032j&amp;key=17099246" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BD2sXovFOlug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2006&pages=959-967&author=V.+M.+Nayagamauthor=X.+Wangauthor=Y.+C.+Tanauthor=A.+Poulsenauthor=K.+C.+Gohauthor=T.+Ngauthor=H.+Wangauthor=H.+Y.+Songauthor=B.+Niauthor=M.+Entzerothauthor=W.+Stunkel&title=SIRT1+modulating+compounds+from+high-throughput+screening+as+anti-inflammatory+and+insulin-sensitizing+agents"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">SIRT1 modulating compounds from high-throughput screening as anti-inflammatory and insulin-sensitizing agents</span></div><div class="casAuthors">Nayagam, Vasantha M.; Wang, Xukun; Tan, Yong Cheng; Poulsen, Anders; Goh, Kee Chuan; Ng, Tony; Wang, Haishan; Song, Hong Yan; Ni, Binhui; Entzeroth, Michael; Stunkel, Walter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">959-967</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The NAD (NAD+)-dependent protein deacetylase SIRT1 has been linked to fatty acid metab. via suppression of peroxysome proliferator-activated receptor gamma (PPAR-γ) and to inflammatory processes by deacetylating the transcription factor NF-κB.  First, modulation of SIRT1 activity affects lipid accumulation in adipocytes, which has an impact on the etiol. of a variety of human metabolic diseases such as obesity and insulin-resistant diabetes.  Second, activation of SIRT1 suppresses inflammation via regulation of cytokine expression.  Using high-throughput screening, the authors identified compds. with SIRT1 activating and inhibiting potential.  The biol. activity of these SIRT1-modulating compds. was confirmed in cell-based assays using mouse adipocytes, as well as human THP-1 monocytes.  SIRT1 activators were found to be potent lipolytic agents, reducing the overall lipid content of fully differentiated NIH L1 adipocytes.  In addn., the same compds. have anti-inflammatory properties, as became evident by the redn. of the proinflammatory cytokine tumor necrosis factor-alpha (TNF-α).  In contrast, a SIRT1 inhibitory compd. showed a stimulatory activity on the differentiation of adipocytes, a feature often linked to insulin sensitization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5D3Kk0JnfubVg90H21EOLACvtfcHk0ljaICKysMyNfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXovFOlug%253D%253D&md5=e20b0e3c3b84e8002a000c4d483bbc91</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1177%2F1087057106294710&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057106294710%26sid%3Dliteratum%253Aachs%26aulast%3DNayagam%26aufirst%3DV.%2BM.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DTan%26aufirst%3DY.%2BC.%26aulast%3DPoulsen%26aufirst%3DA.%26aulast%3DGoh%26aufirst%3DK.%2BC.%26aulast%3DNg%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DSong%26aufirst%3DH.%2BY.%26aulast%3DNi%26aufirst%3DB.%26aulast%3DEntzeroth%26aufirst%3DM.%26aulast%3DStunkel%26aufirst%3DW.%26atitle%3DSIRT1%2520modulating%2520compounds%2520from%2520high-throughput%2520screening%2520as%2520anti-inflammatory%2520and%2520insulin-sensitizing%2520agents%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2006%26volume%3D11%26spage%3D959%26epage%3D967" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Mai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valente, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meade, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carafa, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tardugno, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nebbioso, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galmozzi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitro, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Fabiani, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altucci, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kazantsev, A.</span><span> </span><span class="NLM_article-title">Study of 1,4-dihydropyridine structural scaffold: discovery of novel sirtuin activators and inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">5496</span><span class="NLM_x">–</span> <span class="NLM_lpage">5504</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm9008289" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=5496-5504&author=A.+Maiauthor=S.+Valenteauthor=S.+Meadeauthor=V.+Carafaauthor=M.+Tardugnoauthor=A.+Nebbiosoauthor=A.+Galmozziauthor=N.+Mitroauthor=E.+De+Fabianiauthor=L.+Altucciauthor=A.+Kazantsev&title=Study+of+1%2C4-dihydropyridine+structural+scaffold%3A+discovery+of+novel+sirtuin+activators+and+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjm9008289&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9008289%26sid%3Dliteratum%253Aachs%26aulast%3DMai%26aufirst%3DA.%26aulast%3DValente%26aufirst%3DS.%26aulast%3DMeade%26aufirst%3DS.%26aulast%3DCarafa%26aufirst%3DV.%26aulast%3DTardugno%26aufirst%3DM.%26aulast%3DNebbioso%26aufirst%3DA.%26aulast%3DGalmozzi%26aufirst%3DA.%26aulast%3DMitro%26aufirst%3DN.%26aulast%3DDe%2BFabiani%26aufirst%3DE.%26aulast%3DAltucci%26aufirst%3DL.%26aulast%3DKazantsev%26aufirst%3DA.%26atitle%3DStudy%2520of%25201%252C4-dihydropyridine%2520structural%2520scaffold%253A%2520discovery%2520of%2520novel%2520sirtuin%2520activators%2520and%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D5496%26epage%3D5504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Milne, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schenk, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carney, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gagne, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boss, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perni, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vu, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bemis, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Disch, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, P. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nunes, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lynch, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galonek, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Israelian, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choy, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iffland, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lavu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Medvedik, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sinclair, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olefsky, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jirousek, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elliott, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westphal, C. H.</span><span> </span><span class="NLM_article-title">Small molecule activators of SIRT1 as therapeutics for the treatment of type 2 diabetes</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">450</span><span class="NLM_x">, </span> <span class="NLM_fpage">712</span><span class="NLM_x">–</span> <span class="NLM_lpage">716</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1038%2Fnature06261" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Fjm301032j&amp;key=18046409" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlKmtbrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=450&publication_year=2007&pages=712-716&author=J.+C.+Milneauthor=P.+D.+Lambertauthor=S.+Schenkauthor=D.+P.+Carneyauthor=J.+J.+Smithauthor=D.+J.+Gagneauthor=L.+Jinauthor=O.+Bossauthor=R.+B.+Perniauthor=C.+B.+Vuauthor=J.+E.+Bemisauthor=R.+Xieauthor=J.+S.+Dischauthor=P.+Y.+Ngauthor=J.+J.+Nunesauthor=A.+V.+Lynchauthor=H.+Yangauthor=H.+Galonekauthor=K.+Israelianauthor=W.+Choyauthor=A.+Ifflandauthor=S.+Lavuauthor=O.+Medvedikauthor=D.+A.+Sinclairauthor=J.+M.+Olefskyauthor=M.+R.+Jirousekauthor=P.+J.+Elliottauthor=C.+H.+Westphal&title=Small+molecule+activators+of+SIRT1+as+therapeutics+for+the+treatment+of+type+2+diabetes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule activators of SIRT1 as therapeutics for the treatment of type 2 diabetes</span></div><div class="casAuthors">Milne, Jill C.; Lambert, Philip D.; Schenk, Simon; Carney, David P.; Smith, Jesse J.; Gagne, David J.; Jin, Lei; Boss, Olivier; Perni, Robert B.; Vu, Chi B.; Bemis, Jean E.; Xie, Roger; Disch, Jeremy S.; Ng, Pui Yee; Nunes, Joseph J.; Lynch, Amy V.; Yang, Hongying; Galonek, Heidi; Israelian, Kristine; Choy, Wendy; Iffland, Andre; Lavu, Siva; Medvedik, Oliver; Sinclair, David A.; Olefsky, Jerrold M.; Jirousek, Michael R.; Elliott, Peter J.; Westphal, Christoph H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">450</span>
        (<span class="NLM_cas:issue">7170</span>),
    <span class="NLM_cas:pages">712-716</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Calorie restriction extends lifespan and produces a metabolic profile desirable for treating diseases of aging such as type 2 diabetes.  SIRT1, an NAD+-dependent deacetylase, is a principal modulator of pathways downstream of calorie restriction that produce beneficial effects on glucose homeostasis and insulin sensitivity.  Resveratrol, a polyphenolic SIRT1 activator, mimics the antiaging effects of calorie restriction in lower organisms and in mice fed a high-fat diet ameliorates insulin resistance, increases mitochondrial content, and prolongs survival.  Here the authors describe the identification and characterization of small mol. activators of SIRT1 that are structurally unrelated to, and 1000-fold more potent than, resveratrol.  These compds. bind to the SIRT1 enzyme-peptide substrate complex at an allosteric site amino-terminal to the catalytic domain and lower the Michaelis const. for acetylated substrates.  In diet-induced obese and genetically obese mice, these compds. improve insulin sensitivity, lower plasma glucose, and increase mitochondrial capacity.  In Zucker fa/fa rats, hyperinsulinemic-euglycemic clamp studies demonstrate that SIRT1 activators improve whole-body glucose homeostasis and insulin sensitivity in adipose tissue, skeletal muscle and liver.  Thus, SIRT1 activation is a promising new therapeutic approach for treating diseases of aging such as type 2 diabetes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtZUMUqn50MbVg90H21EOLACvtfcHk0lhw8bCyTayUeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlKmtbrM&md5=8343b182d3596cbd8eb175c8b9cdcace</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnature06261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature06261%26sid%3Dliteratum%253Aachs%26aulast%3DMilne%26aufirst%3DJ.%2BC.%26aulast%3DLambert%26aufirst%3DP.%2BD.%26aulast%3DSchenk%26aufirst%3DS.%26aulast%3DCarney%26aufirst%3DD.%2BP.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DGagne%26aufirst%3DD.%2BJ.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DBoss%26aufirst%3DO.%26aulast%3DPerni%26aufirst%3DR.%2BB.%26aulast%3DVu%26aufirst%3DC.%2BB.%26aulast%3DBemis%26aufirst%3DJ.%2BE.%26aulast%3DXie%26aufirst%3DR.%26aulast%3DDisch%26aufirst%3DJ.%2BS.%26aulast%3DNg%26aufirst%3DP.%2BY.%26aulast%3DNunes%26aufirst%3DJ.%2BJ.%26aulast%3DLynch%26aufirst%3DA.%2BV.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DGalonek%26aufirst%3DH.%26aulast%3DIsraelian%26aufirst%3DK.%26aulast%3DChoy%26aufirst%3DW.%26aulast%3DIffland%26aufirst%3DA.%26aulast%3DLavu%26aufirst%3DS.%26aulast%3DMedvedik%26aufirst%3DO.%26aulast%3DSinclair%26aufirst%3DD.%2BA.%26aulast%3DOlefsky%26aufirst%3DJ.%2BM.%26aulast%3DJirousek%26aufirst%3DM.%2BR.%26aulast%3DElliott%26aufirst%3DP.%2BJ.%26aulast%3DWestphal%26aufirst%3DC.%2BH.%26atitle%3DSmall%2520molecule%2520activators%2520of%2520SIRT1%2520as%2520therapeutics%2520for%2520the%2520treatment%2520of%2520type%25202%2520diabetes%26jtitle%3DNature%26date%3D2007%26volume%3D450%26spage%3D712%26epage%3D716" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Vu, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bemis, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Disch, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, P. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nunes, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milne, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carney, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lynch, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lavu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gagne, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jirousek, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schenk, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olefsky, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perni, R. B.</span><span> </span><span class="NLM_article-title">Discovery of imidazo[1,2-<i>b</i>]thiazole derivatives as novel SIRT1 activators</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">1275</span><span class="NLM_x">–</span> <span class="NLM_lpage">1283</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm8012954" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=1275-1283&author=C.+B.+Vuauthor=J.+E.+Bemisauthor=J.+S.+Dischauthor=P.+Y.+Ngauthor=J.+J.+Nunesauthor=J.+C.+Milneauthor=D.+P.+Carneyauthor=A.+V.+Lynchauthor=J.+J.+Smithauthor=S.+Lavuauthor=P.+D.+Lambertauthor=D.+J.+Gagneauthor=M.+R.+Jirousekauthor=S.+Schenkauthor=J.+M.+Olefskyauthor=R.+B.+Perni&title=Discovery+of+imidazo%5B1%2C2-b%5Dthiazole+derivatives+as+novel+SIRT1+activators"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fjm8012954&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8012954%26sid%3Dliteratum%253Aachs%26aulast%3DVu%26aufirst%3DC.%2BB.%26aulast%3DBemis%26aufirst%3DJ.%2BE.%26aulast%3DDisch%26aufirst%3DJ.%2BS.%26aulast%3DNg%26aufirst%3DP.%2BY.%26aulast%3DNunes%26aufirst%3DJ.%2BJ.%26aulast%3DMilne%26aufirst%3DJ.%2BC.%26aulast%3DCarney%26aufirst%3DD.%2BP.%26aulast%3DLynch%26aufirst%3DA.%2BV.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DLavu%26aufirst%3DS.%26aulast%3DLambert%26aufirst%3DP.%2BD.%26aulast%3DGagne%26aufirst%3DD.%2BJ.%26aulast%3DJirousek%26aufirst%3DM.%2BR.%26aulast%3DSchenk%26aufirst%3DS.%26aulast%3DOlefsky%26aufirst%3DJ.%2BM.%26aulast%3DPerni%26aufirst%3DR.%2BB.%26atitle%3DDiscovery%2520of%2520imidazo%255B1%252C2-b%255Dthiazole%2520derivatives%2520as%2520novel%2520SIRT1%2520activators%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D1275%26epage%3D1283" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Bemis, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vu, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nunes, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, P. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Disch, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milne, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carney, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lynch, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lavu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iffland, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jirousek, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perni, R. B.</span><span> </span><span class="NLM_article-title">Discovery of oxazolo[4,5-<i>b</i>]pyridines and related heterocyclic analogs as novel SIRT1 activators</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">2350</span><span class="NLM_x">–</span> <span class="NLM_lpage">2353</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=2350-2353&author=J.+E.+Bemisauthor=C.+B.+Vuauthor=R.+Xieauthor=J.+J.+Nunesauthor=P.+Y.+Ngauthor=J.+S.+Dischauthor=J.+C.+Milneauthor=D.+P.+Carneyauthor=A.+V.+Lynchauthor=L.+Jinauthor=J.+J.+Smithauthor=S.+Lavuauthor=A.+Ifflandauthor=M.+R.+Jirousekauthor=R.+B.+Perni&title=Discovery+of+oxazolo%5B4%2C5-b%5Dpyridines+and+related+heterocyclic+analogs+as+novel+SIRT1+activators"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBemis%26aufirst%3DJ.%2BE.%26aulast%3DVu%26aufirst%3DC.%2BB.%26aulast%3DXie%26aufirst%3DR.%26aulast%3DNunes%26aufirst%3DJ.%2BJ.%26aulast%3DNg%26aufirst%3DP.%2BY.%26aulast%3DDisch%26aufirst%3DJ.%2BS.%26aulast%3DMilne%26aufirst%3DJ.%2BC.%26aulast%3DCarney%26aufirst%3DD.%2BP.%26aulast%3DLynch%26aufirst%3DA.%2BV.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DLavu%26aufirst%3DS.%26aulast%3DIffland%26aufirst%3DA.%26aulast%3DJirousek%26aufirst%3DM.%2BR.%26aulast%3DPerni%26aufirst%3DR.%2BB.%26atitle%3DDiscovery%2520of%2520oxazolo%255B4%252C5-b%255Dpyridines%2520and%2520related%2520heterocyclic%2520analogs%2520as%2520novel%2520SIRT1%2520activators%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D2350%26epage%3D2353" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Kaeberlein, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonagh, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heltweg, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hixon, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westman, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caldwell, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Napper, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curtis, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DiStefano, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fields, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedalov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennedy, B. K.</span><span> </span><span class="NLM_article-title">Substrate-specific activation of sirtuins by resveratrol</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">280</span><span class="NLM_x">, </span> <span class="NLM_fpage">17038</span><span class="NLM_x">–</span> <span class="NLM_lpage">17045</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2005&pages=17038-17045&author=M.+Kaeberleinauthor=T.+McDonaghauthor=B.+Heltwegauthor=J.+Hixonauthor=E.+A.+Westmanauthor=S.+D.+Caldwellauthor=A.+Napperauthor=R.+Curtisauthor=P.+S.+DiStefanoauthor=S.+Fieldsauthor=A.+Bedalovauthor=B.+K.+Kennedy&title=Substrate-specific+activation+of+sirtuins+by+resveratrol"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKaeberlein%26aufirst%3DM.%26aulast%3DMcDonagh%26aufirst%3DT.%26aulast%3DHeltweg%26aufirst%3DB.%26aulast%3DHixon%26aufirst%3DJ.%26aulast%3DWestman%26aufirst%3DE.%2BA.%26aulast%3DCaldwell%26aufirst%3DS.%2BD.%26aulast%3DNapper%26aufirst%3DA.%26aulast%3DCurtis%26aufirst%3DR.%26aulast%3DDiStefano%26aufirst%3DP.%2BS.%26aulast%3DFields%26aufirst%3DS.%26aulast%3DBedalov%26aufirst%3DA.%26aulast%3DKennedy%26aufirst%3DB.%2BK.%26atitle%3DSubstrate-specific%2520activation%2520of%2520sirtuins%2520by%2520resveratrol%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2005%26volume%3D280%26spage%3D17038%26epage%3D17045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Pacholec, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bleasdale, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chrunyk, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cunningham, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flynn, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garofalo, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffith, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffor, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loulakis, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pabst, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qiu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stockman, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thanabal, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varghese, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ward, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Withka, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahn, K.</span><span> </span><span class="NLM_article-title">SRT1720, SRT2183, SRT1460, and resveratrol are not direct activators of SIRT1</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">285</span><span class="NLM_x">, </span> <span class="NLM_fpage">8340</span><span class="NLM_x">–</span> <span class="NLM_lpage">8351</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=8340-8351&author=M.+Pacholecauthor=J.+E.+Bleasdaleauthor=B.+Chrunykauthor=D.+Cunninghamauthor=D.+Flynnauthor=R.+S.+Garofaloauthor=D.+Griffithauthor=M.+Grifforauthor=P.+Loulakisauthor=B.+Pabstauthor=X.+Qiuauthor=B.+Stockmanauthor=V.+Thanabalauthor=A.+Vargheseauthor=J.+Wardauthor=J.+Withkaauthor=K.+Ahn&title=SRT1720%2C+SRT2183%2C+SRT1460%2C+and+resveratrol+are+not+direct+activators+of+SIRT1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPacholec%26aufirst%3DM.%26aulast%3DBleasdale%26aufirst%3DJ.%2BE.%26aulast%3DChrunyk%26aufirst%3DB.%26aulast%3DCunningham%26aufirst%3DD.%26aulast%3DFlynn%26aufirst%3DD.%26aulast%3DGarofalo%26aufirst%3DR.%2BS.%26aulast%3DGriffith%26aufirst%3DD.%26aulast%3DGriffor%26aufirst%3DM.%26aulast%3DLoulakis%26aufirst%3DP.%26aulast%3DPabst%26aufirst%3DB.%26aulast%3DQiu%26aufirst%3DX.%26aulast%3DStockman%26aufirst%3DB.%26aulast%3DThanabal%26aufirst%3DV.%26aulast%3DVarghese%26aufirst%3DA.%26aulast%3DWard%26aufirst%3DJ.%26aulast%3DWithka%26aufirst%3DJ.%26aulast%3DAhn%26aufirst%3DK.%26atitle%3DSRT1720%252C%2520SRT2183%252C%2520SRT1460%252C%2520and%2520resveratrol%2520are%2520not%2520direct%2520activators%2520of%2520SIRT1%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D8340%26epage%3D8351" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Dai, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kustigian, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carney, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Case, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Considine, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubbard, B. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perni, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riera, T. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szczepankiewicz, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vlasuk, G. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stein, R. L.</span><span> </span><span class="NLM_article-title">SIRT1 activation by small molecules: kinetic and biophysical evidence for direct interaction of enzyme and activator</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">285</span><span class="NLM_x">, </span> <span class="NLM_fpage">32695</span><span class="NLM_x">–</span> <span class="NLM_lpage">32703</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=32695-32703&author=H.+Daiauthor=L.+Kustigianauthor=D.+Carneyauthor=A.+Caseauthor=T.+Considineauthor=B.+P.+Hubbardauthor=R.+B.+Perniauthor=T.+V.+Rieraauthor=B.+Szczepankiewiczauthor=G.+P.+Vlasukauthor=R.+L.+Stein&title=SIRT1+activation+by+small+molecules%3A+kinetic+and+biophysical+evidence+for+direct+interaction+of+enzyme+and+activator"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DH.%26aulast%3DKustigian%26aufirst%3DL.%26aulast%3DCarney%26aufirst%3DD.%26aulast%3DCase%26aufirst%3DA.%26aulast%3DConsidine%26aufirst%3DT.%26aulast%3DHubbard%26aufirst%3DB.%2BP.%26aulast%3DPerni%26aufirst%3DR.%2BB.%26aulast%3DRiera%26aufirst%3DT.%2BV.%26aulast%3DSzczepankiewicz%26aufirst%3DB.%26aulast%3DVlasuk%26aufirst%3DG.%2BP.%26aulast%3DStein%26aufirst%3DR.%2BL.%26atitle%3DSIRT1%2520activation%2520by%2520small%2520molecules%253A%2520kinetic%2520and%2520biophysical%2520evidence%2520for%2520direct%2520interaction%2520of%2520enzyme%2520and%2520activator%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D32695%26epage%3D32703" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Bass, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weinkove, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Houthoofd, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gems, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Partridge, L.</span><span> </span><span class="NLM_article-title">Effects of resveratrol on lifespan in <i>Drosophila melanogaster</i> and <i>Caenorhabditis elegans</i></span> <span class="citation_source-journal">Mech. Ageing Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">546</span><span class="NLM_x">–</span> <span class="NLM_lpage">552</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=546-552&author=T.+M.+Bassauthor=D.+Weinkoveauthor=K.+Houthoofdauthor=D.+Gemsauthor=L.+Partridge&title=Effects+of+resveratrol+on+lifespan+in+Drosophila+melanogaster+and+Caenorhabditis+elegans"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBass%26aufirst%3DT.%2BM.%26aulast%3DWeinkove%26aufirst%3DD.%26aulast%3DHouthoofd%26aufirst%3DK.%26aulast%3DGems%26aufirst%3DD.%26aulast%3DPartridge%26aufirst%3DL.%26atitle%3DEffects%2520of%2520resveratrol%2520on%2520lifespan%2520in%2520Drosophila%2520melanogaster%2520and%2520Caenorhabditis%2520elegans%26jtitle%3DMech.%2520Ageing%2520Dev.%26date%3D2007%26volume%3D128%26spage%3D546%26epage%3D552" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Feige, J. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lagouge, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Canto, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strehle, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Houten, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milne, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mataki, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elliott, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Auwerx, J.</span><span> </span><span class="NLM_article-title">Specific SIRT1 activation mimics low energy levels and protects against diet-induced metabolic disorders by enhancing fat oxidation</span> <span class="citation_source-journal">Cell Metab.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">347</span><span class="NLM_x">–</span> <span class="NLM_lpage">358</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1016%2Fj.cmet.2008.08.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Fjm301032j&amp;key=19046567" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlOgur7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2008&pages=347-358&author=J.+N.+Feigeauthor=M.+Lagougeauthor=C.+Cantoauthor=A.+Strehleauthor=S.+M.+Houtenauthor=J.+C.+Milneauthor=P.+D.+Lambertauthor=C.+Matakiauthor=P.+J.+Elliottauthor=J.+Auwerx&title=Specific+SIRT1+activation+mimics+low+energy+levels+and+protects+against+diet-induced+metabolic+disorders+by+enhancing+fat+oxidation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Specific SIRT1 activation mimics low energy levels and protects against diet-induced metabolic disorders by enhancing fat oxidation</span></div><div class="casAuthors">Feige, Jerome N.; Lagouge, Marie; Canto, Carles; Strehle, Axelle; Houten, Sander M.; Milne, Jill C.; Lambert, Philip D.; Mataki, Chikage; Elliott, Peter J.; Auwerx, Johan</div><div class="citationInfo"><span class="NLM_cas:title">Cell Metabolism</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">347-358</span>CODEN:
                <span class="NLM_cas:coden">CMEEB5</span>;
        ISSN:<span class="NLM_cas:issn">1550-4131</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The NAD+-dependent deacetylase SIRT1 controls metabolic processes in response to low nutrient availability.  We report the metabolic phenotype of mice treated with SRT1720, a specific and potent synthetic activator of SIRT1 that is devoid of direct action on AMPK.  SRT1720 administration robustly enhances endurance running performance and strongly protects from diet-induced obesity and insulin resistance by enhancing oxidative metab. in skeletal muscle, liver, and brown adipose tissue.  These metabolic effects of SRT1720 are mediated by the induction of a genetic network controlling fatty acid oxidn. through a multifaceted mechanism that involves the direct deacetylation of PGC-1α, FOXO1, and p53 and the indirect stimulation of AMPK signaling through a global metabolic adaptation mimicking low energy levels.  Combined with our previous work on resveratrol, the current study further validates SIRT1 as a target for the treatment of metabolic disorders and characterizes the mechanisms underlying the therapeutic potential of SIRT1 activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo90My01jFYurVg90H21EOLACvtfcHk0ljhI7ivfucukQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlOgur7O&md5=516fd7d72637dccb9a31a211c406ae0d</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2008.08.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2008.08.017%26sid%3Dliteratum%253Aachs%26aulast%3DFeige%26aufirst%3DJ.%2BN.%26aulast%3DLagouge%26aufirst%3DM.%26aulast%3DCanto%26aufirst%3DC.%26aulast%3DStrehle%26aufirst%3DA.%26aulast%3DHouten%26aufirst%3DS.%2BM.%26aulast%3DMilne%26aufirst%3DJ.%2BC.%26aulast%3DLambert%26aufirst%3DP.%2BD.%26aulast%3DMataki%26aufirst%3DC.%26aulast%3DElliott%26aufirst%3DP.%2BJ.%26aulast%3DAuwerx%26aufirst%3DJ.%26atitle%3DSpecific%2520SIRT1%2520activation%2520mimics%2520low%2520energy%2520levels%2520and%2520protects%2520against%2520diet-induced%2520metabolic%2520disorders%2520by%2520enhancing%2520fat%2520oxidation%26jtitle%3DCell%2520Metab.%26date%3D2008%26volume%3D8%26spage%3D347%26epage%3D358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kenney, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gagne, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frushour, B. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ladd, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galonek, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Israelian, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Razvadauskaite, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lynch, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carney, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lavu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iffland, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elliott, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elliston, K. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jirousek, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milne, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boss, O.</span><span> </span><span class="NLM_article-title">Small molecule activators of SIRT1 replicate signaling pathways triggered by calorie restriction in vivo</span> <span class="citation_source-journal">BMC Syst. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">31</span><span class="NLM_x">–</span> <span class="NLM_lpage">45</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1186%2F1752-0509-3-31" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Fjm301032j&amp;key=19284563" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A280%3ADC%252BD1M3jtVantQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2009&pages=31-45&author=J.+J.+Smithauthor=R.+D.+Kenneyauthor=D.+J.+Gagneauthor=B.+P.+Frushourauthor=W.+Laddauthor=H.+L.+Galonekauthor=K.+Israelianauthor=J.+Songauthor=G.+Razvadauskaiteauthor=A.+V.+Lynchauthor=D.+P.+Carneyauthor=R.+J.+Johnsonauthor=S.+Lavuauthor=A.+Ifflandauthor=P.+J.+Elliottauthor=P.+D.+Lambertauthor=K.+O.+Ellistonauthor=M.+R.+Jirousekauthor=J.+C.+Milneauthor=O.+Boss&title=Small+molecule+activators+of+SIRT1+replicate+signaling+pathways+triggered+by+calorie+restriction+in+vivo"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule activators of SIRT1 replicate signaling pathways triggered by calorie restriction in vivo</span></div><div class="casAuthors">Smith Jesse J; Kenney Renee Deehan; Gagne David J; Frushour Brian P; Ladd William; Galonek Heidi L; Israelian Kristine; Song Jeffrey; Razvadauskaite Giedre; Lynch Amy V; Carney David P; Johnson Robin J; Lavu Siva; Iffland Andre; Elliott Peter J; Lambert Philip D; Elliston Keith O; Jirousek Michael R; Milne Jill C; Boss Olivier</div><div class="citationInfo"><span class="NLM_cas:title">BMC systems biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">31</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Calorie restriction (CR) produces a number of health benefits and ameliorates diseases of aging such as type 2 diabetes.  The components of the pathways downstream of CR may provide intervention points for developing therapeutics for treating diseases of aging.  The NAD+-dependent protein deacetylase SIRT1 has been implicated as one of the key downstream regulators of CR in yeast, rodents, and humans.  Small molecule activators of SIRT1 have been identified that exhibit efficacy in animal models of diseases typically associated with aging including type 2 diabetes.  To identify molecular processes induced in the liver of mice treated with two structurally distinct SIRT1 activators, SIRT501 (formulated resveratrol) and SRT1720, for three days, we utilized a systems biology approach and applied Causal Network Modeling (CNM) on gene expression data to elucidate downstream effects of SIRT1 activation.  RESULTS:  Here we demonstrate that SIRT1 activators recapitulate many of the molecular events downstream of CR in vivo, such as enhancing mitochondrial biogenesis, improving metabolic signaling pathways, and blunting pro-inflammatory pathways in mice fed a high fat, high calorie diet.  CONCLUSION:  CNM of gene expression data from mice treated with SRT501 or SRT1720 in combination with supporting in vitro and in vivo data demonstrates that SRT501 and SRT1720 produce a signaling profile that mirrors CR, improves glucose and insulin homeostasis, and acts via SIRT1 activation in vivo.  Taken together these results are encouraging regarding the use of small molecule activators of SIRT1 for therapeutic intervention into type 2 diabetes, a strategy which is currently being investigated in multiple clinical trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQDlBeNme77BrqxQ2mb9Pu8fW6udTcc2eYHPTqH20nzSrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1M3jtVantQ%253D%253D&md5=042e7aec752f41e84c51b4adbda7925a</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1186%2F1752-0509-3-31&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1752-0509-3-31%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DKenney%26aufirst%3DR.%2BD.%26aulast%3DGagne%26aufirst%3DD.%2BJ.%26aulast%3DFrushour%26aufirst%3DB.%2BP.%26aulast%3DLadd%26aufirst%3DW.%26aulast%3DGalonek%26aufirst%3DH.%2BL.%26aulast%3DIsraelian%26aufirst%3DK.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DRazvadauskaite%26aufirst%3DG.%26aulast%3DLynch%26aufirst%3DA.%2BV.%26aulast%3DCarney%26aufirst%3DD.%2BP.%26aulast%3DJohnson%26aufirst%3DR.%2BJ.%26aulast%3DLavu%26aufirst%3DS.%26aulast%3DIffland%26aufirst%3DA.%26aulast%3DElliott%26aufirst%3DP.%2BJ.%26aulast%3DLambert%26aufirst%3DP.%2BD.%26aulast%3DElliston%26aufirst%3DK.%2BO.%26aulast%3DJirousek%26aufirst%3DM.%2BR.%26aulast%3DMilne%26aufirst%3DJ.%2BC.%26aulast%3DBoss%26aufirst%3DO.%26atitle%3DSmall%2520molecule%2520activators%2520of%2520SIRT1%2520replicate%2520signaling%2520pathways%2520triggered%2520by%2520calorie%2520restriction%2520in%2520vivo%26jtitle%3DBMC%2520Syst.%2520Biol.%26date%3D2009%26volume%3D3%26spage%3D31%26epage%3D45" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">He, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, H.</span><span> </span><span class="NLM_article-title">Discovering potent inhibitors against the beta-hydroxyacyl-acyl carrier protein dehydratase (FabZ) of <i>Helicobacter pylori</i>: structure-based design, synthesis, bioassay, and crystal structure determination</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">2465</span><span class="NLM_x">–</span> <span class="NLM_lpage">2481</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm8015602" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=2465-2481&author=L.+Heauthor=L.+Zhangauthor=X.+Liuauthor=X.+Liauthor=M.+Zhengauthor=H.+Liauthor=K.+Yuauthor=K.+Chenauthor=X.+Shenauthor=H.+Jiangauthor=H.+Liu&title=Discovering+potent+inhibitors+against+the+beta-hydroxyacyl-acyl+carrier+protein+dehydratase+%28FabZ%29+of+Helicobacter+pylori%3A+structure-based+design%2C+synthesis%2C+bioassay%2C+and+crystal+structure+determination"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm8015602&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8015602%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DYu%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DShen%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DH.%26atitle%3DDiscovering%2520potent%2520inhibitors%2520against%2520the%2520beta-hydroxyacyl-acyl%2520carrier%2520protein%2520dehydratase%2520%2528FabZ%2529%2520of%2520Helicobacter%2520pylori%253A%2520structure-based%2520design%252C%2520synthesis%252C%2520bioassay%252C%2520and%2520crystal%2520structure%2520determination%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D2465%26epage%3D2481" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Boehlow, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harburn, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spilling, C. D.</span><span> </span><span class="NLM_article-title">Approaches to the synthesis of some tyrosine-derived marine sponge metabolites: synthesis of verongamine and purealidin N</span> <span class="citation_source-journal">J. Org. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">3111</span><span class="NLM_x">–</span> <span class="NLM_lpage">3118</span></span><div class="citationLinks">[<a href="/doi/10.1021/jo010015v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BD3MXit1GjtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2001&pages=3111-3118&author=T.+R.+Boehlowauthor=J.+J.+Harburnauthor=C.+D.+Spilling&title=Approaches+to+the+synthesis+of+some+tyrosine-derived+marine+sponge+metabolites%3A+synthesis+of+verongamine+and+purealidin+N"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Approaches to the Synthesis of Some Tyrosine-Derived Marine Sponge Metabolites: Synthesis of Verongamine and Purealidin N</span></div><div class="casAuthors">Boehlow, Todd R.; Harburn, J. Jonathan; Spilling, Christopher D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3111-3118</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The oxidn. of tyrosine Et ester with Na2WO4/H2O2 in ethanol, dimethyldioxirane in acetone, or methyltrioxorhenium/H2O2 in EtOH gave the corresponding tyrosine oxime in high yield.  Controlled bromination of the arom. ring gave the monobromo oxime, the dibromo oxime, or the spiroisoxazoline depending upon reaction conditions.  Synthesis of the known metabolite verongamine (I, R1,R2 = H) was achieved by oxidn. of O-Me bromotyrosine Me ester and amidation of the resulting oxime ester with histamine.  The mono- and di-bromotyrosine oxime derivs. were further transformed into naturally occurring nitriles by base hydrolysis of the ester and acid-catalyzed decarboxylation.  Amidation of the oxime ester II with histamine, followed by deprotection of the MOM ether gave the first synthesis of purealidin N (I, R1 = Br, R2 = OH).  Oxidative spirocyclization of the phenolic oxime ester II with a polymer-bound iodosyl diacetate gave the spiroisoxazoline III and represents a formal synthesis of aerothionin IV (n = 4), homoaerothionin IV (n = 5), and aerophobin-1 V.  Synthetic purealidin N and three related oximes were tested against three sensitive cancer sell lines, lung NCI-H460, breast MCF7, and CNS SF-268.  At concns. of 10-4 M, all of the compds. were essentially inactive, showing inhibition of cell growth of between 30% and 93% of the control.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyxckqC9akc7Vg90H21EOLACvtfcHk0ljCWtXxYf1Bdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXit1GjtLc%253D&md5=457f8306cdc9b36097e1736c9196f8d9</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjo010015v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo010015v%26sid%3Dliteratum%253Aachs%26aulast%3DBoehlow%26aufirst%3DT.%2BR.%26aulast%3DHarburn%26aufirst%3DJ.%2BJ.%26aulast%3DSpilling%26aufirst%3DC.%2BD.%26atitle%3DApproaches%2520to%2520the%2520synthesis%2520of%2520some%2520tyrosine-derived%2520marine%2520sponge%2520metabolites%253A%2520synthesis%2520of%2520verongamine%2520and%2520purealidin%2520N%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2001%26volume%3D66%26spage%3D3111%26epage%3D3118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Napper, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hixon, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonagh, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keavey, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pons, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barker, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yau, W. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amouzegh, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flegg, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamelin, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kates, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Navia, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saunders, J. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DiStefano, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curtis, R.</span><span> </span><span class="NLM_article-title">Discovery of indoles as potent and selective inhibitors of the deacetylase SIRT1</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">8045</span><span class="NLM_x">–</span> <span class="NLM_lpage">8054</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm050522v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFyqsb3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=8045-8054&author=A.+D.+Napperauthor=J.+Hixonauthor=T.+McDonaghauthor=K.+Keaveyauthor=J.+F.+Ponsauthor=J.+Barkerauthor=W.+T.+Yauauthor=P.+Amouzeghauthor=A.+Fleggauthor=E.+Hamelinauthor=R.+J.+Thomasauthor=M.+Katesauthor=S.+Jonesauthor=M.+A.+Naviaauthor=J.+O.+Saundersauthor=P.+S.+DiStefanoauthor=R.+Curtis&title=Discovery+of+indoles+as+potent+and+selective+inhibitors+of+the+deacetylase+SIRT1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of indoles as potent and selective inhibitors of the deacetylase SIRT1</span></div><div class="casAuthors">Napper, Andrew D.; Hixon, Jeffrey; McDonagh, Thomas; Keavey, Kenneth; Pons, Jean-Francois; Barker, Jonathan; Yau, Wei Tsung; Amouzegh, Patricia; Flegg, Adam; Hamelin, Estelle; Thomas, Russell J.; Kates, Michael; Jones, Stephen; Navia, Manuel A.; Saunders, Jeffrey O.; DiStefano, Peter S.; Curtis, Rory</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">8045-8054</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">High-throughput screening against the human sirtuin SIRT1 led to the discovery of a series of indoles as potent inhibitors that are selective for SIRT1 over other deacetylases and NAD-processing enzymes.  The most potent compds. described herein inhibit SIRT1 with IC50 values of 60-100 nM, representing a 500-fold improvement over previously reported SIRT inhibitors.  Prepn. of enantiomerically pure indole derivs. allowed for their characterization in vitro and in vivo.  Kinetic analyses suggest that these inhibitors bind after the release of nicotinamide from the enzyme and prevent the release of deacetylated peptide and O-acetyl-ADP-ribose, the products of enzyme-catalyzed deacetylation.  These SIRT1 inhibitors are low mol. wt., cell-permeable, orally bioavailable, and metabolically stable.  These compds. provide chem. tools to study the biol. of SIRT1 and to explore therapeutic uses for SIRT1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpbdBlnlCEArVg90H21EOLACvtfcHk0lh6nzUJ4H6Xuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFyqsb3N&md5=5ba92c43b735dbdf383a988a64a06f80</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjm050522v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050522v%26sid%3Dliteratum%253Aachs%26aulast%3DNapper%26aufirst%3DA.%2BD.%26aulast%3DHixon%26aufirst%3DJ.%26aulast%3DMcDonagh%26aufirst%3DT.%26aulast%3DKeavey%26aufirst%3DK.%26aulast%3DPons%26aufirst%3DJ.%2BF.%26aulast%3DBarker%26aufirst%3DJ.%26aulast%3DYau%26aufirst%3DW.%2BT.%26aulast%3DAmouzegh%26aufirst%3DP.%26aulast%3DFlegg%26aufirst%3DA.%26aulast%3DHamelin%26aufirst%3DE.%26aulast%3DThomas%26aufirst%3DR.%2BJ.%26aulast%3DKates%26aufirst%3DM.%26aulast%3DJones%26aufirst%3DS.%26aulast%3DNavia%26aufirst%3DM.%2BA.%26aulast%3DSaunders%26aufirst%3DJ.%2BO.%26aulast%3DDiStefano%26aufirst%3DP.%2BS.%26aulast%3DCurtis%26aufirst%3DR.%26atitle%3DDiscovery%2520of%2520indoles%2520as%2520potent%2520and%2520selective%2520inhibitors%2520of%2520the%2520deacetylase%2520SIRT1%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D8045%26epage%3D8054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Sanders, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jackson, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marmorstein, R.</span><span> </span><span class="NLM_article-title">Structural basis for sirtuin function: what we know and what we don’t</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Proteins Proteomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">1804</span><span class="NLM_x">, </span> <span class="NLM_fpage">1604</span><span class="NLM_x">–</span> <span class="NLM_lpage">1616</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1016%2Fj.bbapap.2009.09.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Fjm301032j&amp;key=19766737" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntlaltLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1804&publication_year=2010&pages=1604-1616&author=B.+D.+Sandersauthor=B.+Jacksonauthor=R.+Marmorstein&title=Structural+basis+for+sirtuin+function%3A+what+we+know+and+what+we+don%E2%80%99t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for sirtuin function: What we know and what we don't</span></div><div class="casAuthors">Sanders, Brandi D.; Jackson, Brittany; Marmorstein, Ronen</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Proteins and Proteomics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1804</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1604-1616</span>CODEN:
                <span class="NLM_cas:coden">BBAPBW</span>;
        ISSN:<span class="NLM_cas:issn">1570-9639</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B. V.</span>)
        </div><div class="casAbstract">A review.  Sirtuin (silent information regulator 2) proteins are NAD-dependent deacetylases that are implicated in diverse biol. processes including DNA regulation, metab., and longevity.  Homologs of the prototypic yeast Sir2 protein have been identified in all 3 kingdoms of life, and while bacteria and archaea typically contain 1-2 sirtuins, eukaryotic organisms contain multiple members.  Sirtuins are regulated in part by the cellular concns. of the noncompetitive inhibitor, nicotinamide, and several synthetic modulators of these enzymes have been identified.  The x-ray crystal structures of several sirtuin proteins in various liganded forms have been detd.  This wealth of structural information, together with related biochem. studies, have provided important insights into the catalytic mechanism, substrate specificity, and inhibitory mechanism of sirtuin proteins.  Implications for future structural studies to address outstanding questions in the field are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrF0mq_taWVALVg90H21EOLACvtfcHk0lh6nzUJ4H6Xuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntlaltLc%253D&md5=7ddae49f9e55b39b4ad0abdb00d68596</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.bbapap.2009.09.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbapap.2009.09.009%26sid%3Dliteratum%253Aachs%26aulast%3DSanders%26aufirst%3DB.%2BD.%26aulast%3DJackson%26aufirst%3DB.%26aulast%3DMarmorstein%26aufirst%3DR.%26atitle%3DStructural%2520basis%2520for%2520sirtuin%2520function%253A%2520what%2520we%2520know%2520and%2520what%2520we%2520don%25E2%2580%2599t%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Proteins%2520Proteomics%26date%3D2010%26volume%3D1804%26spage%3D1604%26epage%3D1616" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Otwinowski, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minor, W.</span><span> </span><span class="NLM_article-title">Processing of X-ray diffraction data collected in oscillation mode</span> <span class="citation_source-journal">Macromol. Crystallogr., Part A</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">276</span><span class="NLM_x">, </span> <span class="NLM_fpage">307</span><span class="NLM_x">–</span> <span class="NLM_lpage">326</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=307-326&author=Z.+Otwinowskiauthor=W.+Minor&title=Processing+of+X-ray+diffraction+data+collected+in+oscillation+mode"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Processing of x-ray diffraction data collected in oscillation mode</span></div><div class="casAuthors">Otwinowski, Zbyszek; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part A</span>),
    <span class="NLM_cas:pages">307-326</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Macromol. crystallog. is an iterative process.  Rarely do the first crystals provide all the necessary data to solve the biol. problem being studied.  Each step benefits from experience learned in previous steps.  To monitor the progress, the HKL package provides 2 tools: (1) statistics, both weighted (χ2) and unweighted (R-merge), are provided, and the Bayesian reasoning and multicomponent error model facilitates obtaining the proper error ests. and (2) visualization of the process plays a double role by helping the operator to confirm that the process of data redn., including the resulting statistics, is correct, and allowing one to evaluate problems for which there are no good statistical criteria.  Visualization also provides confidence that the point of diminishing returns in data collection and redn. has been reached.  At that point, the effort should be directed to solving the structure.  The methods presented here have been applied to solve a large variety of problems, from inorg. mols. with 5 Å unit cell to rotavirus of 700 Å diam. crystd. in 700 × 1000 × 1400 Å cell.  Overall quality of the method was tested by many researchers by successful application of the programs to MAD structure detns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA3fvBOHqhhrVg90H21EOLACvtfcHk0lj3syoCgM_oFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D&md5=c9536971d4e32cc35352c40fb9368131</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3DProcessing%2520of%2520X-ray%2520diffraction%2520data%2520collected%2520in%2520oscillation%2520mode%26jtitle%3DMacromol.%2520Crystallogr.%252C%2520Part%2520A%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span> </span><span class="NLM_article-title">Collaborative computational project, number 4. The CCP4 suite: programs for protein crystallography</span>.  <span class="citation_source-journal">Acta. Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">760</span><span class="NLM_x">–</span> <span class="NLM_lpage">763</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Collaborative+computational+project%2C+number+4.+The+CCP4+suite%3A+programs+for+protein+crystallography.+Acta.+Crystallogr.%2C+Sect.+D%3A+Biol.+Crystallogr.+1994%2C+50%2C+760%E2%80%93763."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26atitle%3DCollaborative%2520computational%2520project%252C%2520number%25204.%2520The%2520CCP4%2520suite%253A%2520programs%2520for%2520protein%2520crystallography%26jtitle%3DActa.%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26spage%3D760%26epage%3D763" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Sauve, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Celic, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Avalos, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boeke, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schramm, V. L.</span><span> </span><span class="NLM_article-title">Chemistry of gene silencing: the mechanism of NAD<sup>+</sup>-dependent deacetylation reactions</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">15456</span><span class="NLM_x">–</span> <span class="NLM_lpage">15463</span></span><div class="citationLinks">[<a href="/doi/10.1021/bi011858j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2001&pages=15456-15463&author=A.+A.+Sauveauthor=I.+Celicauthor=J.+Avalosauthor=H.+Dengauthor=J.+D.+Boekeauthor=V.+L.+Schramm&title=Chemistry+of+gene+silencing%3A+the+mechanism+of+NAD%2B-dependent+deacetylation+reactions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fbi011858j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi011858j%26sid%3Dliteratum%253Aachs%26aulast%3DSauve%26aufirst%3DA.%2BA.%26aulast%3DCelic%26aufirst%3DI.%26aulast%3DAvalos%26aufirst%3DJ.%26aulast%3DDeng%26aufirst%3DH.%26aulast%3DBoeke%26aufirst%3DJ.%2BD.%26aulast%3DSchramm%26aufirst%3DV.%2BL.%26atitle%3DChemistry%2520of%2520gene%2520silencing%253A%2520the%2520mechanism%2520of%2520NAD%252B-dependent%2520deacetylation%2520reactions%26jtitle%3DBiochemistry%26date%3D2001%26volume%3D40%26spage%3D15456%26epage%3D15463" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Smith, B. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Denu, J. M.</span><span> </span><span class="NLM_article-title">Sir2 protein deacetylases: evidence for chemical intermediates and functions of a conserved histidine</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">272</span><span class="NLM_x">–</span> <span class="NLM_lpage">282</span></span><div class="citationLinks">[<a href="/doi/10.1021/bi052014t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2006&pages=272-282&author=B.+C.+Smithauthor=J.+M.+Denu&title=Sir2+protein+deacetylases%3A+evidence+for+chemical+intermediates+and+functions+of+a+conserved+histidine"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fbi052014t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi052014t%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DB.%2BC.%26aulast%3DDenu%26aufirst%3DJ.%2BM.%26atitle%3DSir2%2520protein%2520deacetylases%253A%2520evidence%2520for%2520chemical%2520intermediates%2520and%2520functions%2520of%2520a%2520conserved%2520histidine%26jtitle%3DBiochemistry%26date%3D2006%26volume%3D45%26spage%3D272%26epage%3D282" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Jackson, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppenheimer, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Denu, J. M.</span><span> </span><span class="NLM_article-title">Mechanism of nicotinamide inhibition and transglycosidation by Sir2 histone/protein deacetylases</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">278</span><span class="NLM_x">, </span> <span class="NLM_fpage">50985</span><span class="NLM_x">–</span> <span class="NLM_lpage">50998</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1074%2Fjbc.M306552200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Fjm301032j&amp;key=14522996" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BD3sXpslOhs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=50985-50998&author=M.+D.+Jacksonauthor=M.+T.+Schmidtauthor=N.+J.+Oppenheimerauthor=J.+M.+Denu&title=Mechanism+of+nicotinamide+inhibition+and+transglycosidation+by+Sir2+histone%2Fprotein+deacetylases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism of Nicotinamide Inhibition and Transglycosidation by Sir2 Histone/Protein Deacetylases</span></div><div class="casAuthors">Jackson, Michael D.; Schmidt, Manning T.; Oppenheimer, Norman J.; Denu, John M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">278</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">50985-50998</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Silent information regulator 2 (Sir2) enzymes catalyze NAD+-dependent protein/histone deacetylation, where the acetyl group from the lysine ε-amino group is transferred to the ADP-ribose moiety of NAD+, producing nicotinamide and the novel metabolite O-acetyl-ADP-ribose.  Sir2 proteins have been shown to regulate gene silencing, metabolic enzymes, and life span.  Recently, nicotinamide has been implicated as a direct neg. regulator of cellular Sir2 function; however, the mechanism of nicotinamide inhibition was not established.  Sir2 enzymes are multifunctional in that the deacetylase reaction involves the cleavage of the nicotinamide-ribosyl, cleavage of an amide bond, and transfer of the acetyl group ultimately to the 2'-ribose hydroxyl of ADP-ribose.  Here the authors demonstrate that nicotinamide inhibition is the result of nicotinamide intercepting an ADP-ribosyl-enzyme-acetyl peptide intermediate with regeneration of NAD+ (transglycosidation).  The cellular implications are discussed.  A variety of 3-substituted pyridines was found to be substrates for enzyme-catalyzed transglycosidation.  A Broensted plot of the data yielded a slope of +0.98, consistent with the development of a nearly full pos. charge in the transition state, and with basicity of the attacking nucleophile as a strong predictor of reactivity.  NAD+ analogs including β-2'-deoxy-2'-fluororibo-NAD+ and a His-to-Ala mutant were used to probe the mechanism of nicotinamide-ribosyl cleavage and acetyl group transfer.  The authors demonstrate that nicotinamide-ribosyl cleavage is distinct from acetyl group transfer to the 2'-OH ribose.  The obsd. enzyme-catalyzed formation of a labile 1'-acetylated-ADP-fluororibose intermediate using β-2'-deoxy-2'-fluororibo-NAD+ supports a mechanism where, after nicotinamide-ribosyl cleavage, the carbonyl oxygen of acetylated substrate attacks the C-1' ribose to form an initial iminium adduct.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqf6GUpwcEPWbVg90H21EOLACvtfcHk0lhKZsigGEHwEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXpslOhs78%253D&md5=89ebcb43d075c4738e261ee22e4b2893</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M306552200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M306552200%26sid%3Dliteratum%253Aachs%26aulast%3DJackson%26aufirst%3DM.%2BD.%26aulast%3DSchmidt%26aufirst%3DM.%2BT.%26aulast%3DOppenheimer%26aufirst%3DN.%2BJ.%26aulast%3DDenu%26aufirst%3DJ.%2BM.%26atitle%3DMechanism%2520of%2520nicotinamide%2520inhibition%2520and%2520transglycosidation%2520by%2520Sir2%2520histone%252Fprotein%2520deacetylases%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26spage%3D50985%26epage%3D50998" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cai, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dai, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choy, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bemis, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jirousek, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milne, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westphal, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perni, R. B.</span><span> </span><span class="NLM_article-title">Crystal structures of human SIRT3 displaying substrate-induced conformational changes</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">284</span><span class="NLM_x">, </span> <span class="NLM_fpage">24394</span><span class="NLM_x">–</span> <span class="NLM_lpage">24405</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=24394-24405&author=L.+Jinauthor=W.+Weiauthor=Y.+Jiangauthor=H.+Pengauthor=J.+Caiauthor=C.+Maoauthor=H.+Daiauthor=W.+Choyauthor=J.+E.+Bemisauthor=M.+R.+Jirousekauthor=J.+C.+Milneauthor=C.+H.+Westphalauthor=R.+B.+Perni&title=Crystal+structures+of+human+SIRT3+displaying+substrate-induced+conformational+changes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DL.%26aulast%3DWei%26aufirst%3DW.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DPeng%26aufirst%3DH.%26aulast%3DCai%26aufirst%3DJ.%26aulast%3DMao%26aufirst%3DC.%26aulast%3DDai%26aufirst%3DH.%26aulast%3DChoy%26aufirst%3DW.%26aulast%3DBemis%26aufirst%3DJ.%2BE.%26aulast%3DJirousek%26aufirst%3DM.%2BR.%26aulast%3DMilne%26aufirst%3DJ.%2BC.%26aulast%3DWestphal%26aufirst%3DC.%2BH.%26aulast%3DPerni%26aufirst%3DR.%2BB.%26atitle%3DCrystal%2520structures%2520of%2520human%2520SIRT3%2520displaying%2520substrate-induced%2520conformational%2520changes%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D24394%26epage%3D24405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Avalos, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bever, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolberger, C.</span><span> </span><span class="NLM_article-title">Mechanism of sirtuin inhibition by nicotinamide: altering the NAD<sup>+</sup> cosubstrate specificity of a Sir2 enzyme</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">855</span><span class="NLM_x">–</span> <span class="NLM_lpage">868</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2005&pages=855-868&author=J.+L.+Avalosauthor=K.+M.+Beverauthor=C.+Wolberger&title=Mechanism+of+sirtuin+inhibition+by+nicotinamide%3A+altering+the+NAD%2B+cosubstrate+specificity+of+a+Sir2+enzyme"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAvalos%26aufirst%3DJ.%2BL.%26aulast%3DBever%26aufirst%3DK.%2BM.%26aulast%3DWolberger%26aufirst%3DC.%26atitle%3DMechanism%2520of%2520sirtuin%2520inhibition%2520by%2520nicotinamide%253A%2520altering%2520the%2520NAD%252B%2520cosubstrate%2520specificity%2520of%2520a%2520Sir2%2520enzyme%26jtitle%3DMol.%2520Cell%26date%3D2005%26volume%3D17%26spage%3D855%26epage%3D868" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Min, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Landry, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sternglanz, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, R. M.</span><span> </span><span class="NLM_article-title">Crystal structure of a SIR2 homolog–NAD complex</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x">, </span> <span class="NLM_fpage">269</span><span class="NLM_x">–</span> <span class="NLM_lpage">279</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2001&pages=269-279&author=J.+Minauthor=J.+Landryauthor=R.+Sternglanzauthor=R.+M.+Xu&title=Crystal+structure+of+a+SIR2+homolog%E2%80%93NAD+complex"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DLandry%26aufirst%3DJ.%26aulast%3DSternglanz%26aufirst%3DR.%26aulast%3DXu%26aufirst%3DR.%2BM.%26atitle%3DCrystal%2520structure%2520of%2520a%2520SIR2%2520homolog%25E2%2580%2593NAD%2520complex%26jtitle%3DCell%26date%3D2001%26volume%3D105%26spage%3D269%26epage%3D279" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Avalos, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boeke, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolberger, C.</span><span> </span><span class="NLM_article-title">Structural basis for the mechanism and regulation of Sir2 enzymes</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">639</span><span class="NLM_x">–</span> <span class="NLM_lpage">648</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2004&pages=639-648&author=J.+L.+Avalosauthor=J.+D.+Boekeauthor=C.+Wolberger&title=Structural+basis+for+the+mechanism+and+regulation+of+Sir2+enzymes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAvalos%26aufirst%3DJ.%2BL.%26aulast%3DBoeke%26aufirst%3DJ.%2BD.%26aulast%3DWolberger%26aufirst%3DC.%26atitle%3DStructural%2520basis%2520for%2520the%2520mechanism%2520and%2520regulation%2520of%2520Sir2%2520enzymes%26jtitle%3DMol.%2520Cell%26date%3D2004%26volume%3D13%26spage%3D639%26epage%3D648" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Borra, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, B. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Denu, J. M.</span><span> </span><span class="NLM_article-title">Mechanism of human SIRT1 activation by resveratrol</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">280</span><span class="NLM_x">, </span> <span class="NLM_fpage">17187</span><span class="NLM_x">–</span> <span class="NLM_lpage">17195</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1074%2Fjbc.M501250200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Fjm301032j&amp;key=15749705" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjsFOgtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2005&pages=17187-17195&author=M.+T.+Borraauthor=B.+C.+Smithauthor=J.+M.+Denu&title=Mechanism+of+human+SIRT1+activation+by+resveratrol"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism of Human SIRT1 Activation by Resveratrol</span></div><div class="casAuthors">Borra, Margie T.; Smith, Brian C.; Denu, John M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">280</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">17187-17195</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The NAD+-dependent protein deacetylase family, Sir2 (or sirtuins), is important for many cellular processes including gene silencing, regulation of p53, fatty acid metab., cell cycle regulation, and life span extension.  Resveratrol, a polyphenol found in wines and thought to harbor major health benefits, was reported to be an activator of Sir2 enzymes in vivo and in vitro.  In addn., resveratrol was shown to increase life span in three model organisms through a Sir2-dependent pathway.  Here, we investigated the mol. basis for Sir2 activation by resveratrol.  Among the three enzymes tested (yeast Sir2, human SIRT1, and human SIRT2), only SIRT1 exhibited significant enzyme activation (∼8-fold) using the com. available Fluor de Lys kit (BioMol).  To examine the requirements for resveratrol activation of SIRT1, we synthesized three p53 acetylpeptide substrates either lacking a fluorophore or contg. a 7-amino-4-methylcoumarin (p53-AMC) or rhodamine 110 (p53-R110).  Although SIRT1 activation was independent of the acetylpeptide sequence, resveratrol activation was completely dependent on the presence of a covalently attached fluorophore.  Substrate competition studies indicated that the fluorophore decreased the binding affinity of the peptide, and, in the presence of resveratrol, fluorophore-contg. substrates bound more tightly to SIRT1.  Using available crystal structures, a model of SIRT1 bound to p53-AMC peptide was constructed.  Without resveratrol, the coumarin of p53-AMC peptide is solvent-exposed and makes no significant contacts with SIRT1.  We propose that binding of resveratrol to SIRT1 promotes a conformational change that better accommodates the attached coumarin group.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqP2qEn7gxZILVg90H21EOLACvtfcHk0li94v4X1sbMVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjsFOgtb8%253D&md5=f70797470cf27070646e3a1ccb5a0a2f</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M501250200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M501250200%26sid%3Dliteratum%253Aachs%26aulast%3DBorra%26aufirst%3DM.%2BT.%26aulast%3DSmith%26aufirst%3DB.%2BC.%26aulast%3DDenu%26aufirst%3DJ.%2BM.%26atitle%3DMechanism%2520of%2520human%2520SIRT1%2520activation%2520by%2520resveratrol%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2005%26volume%3D280%26spage%3D17187%26epage%3D17195" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Geronimo, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singh, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, K. S.</span><span> </span><span class="NLM_article-title">How different are electron-rich and electron-deficient pi interactions?</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">1931</span><span class="NLM_x">–</span> <span class="NLM_lpage">1934</span></span><div class="citationLinks">[<a href="/doi/10.1021/ct100182u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnsFGnsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=1931-1934&author=I.+Geronimoauthor=E.+C.+Leeauthor=N.+J.+Singhauthor=K.+S.+Kim&title=How+different+are+electron-rich+and+electron-deficient+pi+interactions%3F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">How Different are Electron-Rich and Electron-Deficient π Interactions?</span></div><div class="casAuthors">Geronimo, Inacrist; Lee, Eun Cheol; Singh, N. Jiten; Kim, Kwang S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1931-1934</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The intermol. interaction driven structural change is vital to mol. architecturing.  In the Cambridge Structural Database (CSD), we find that the preference for geometrical conformations of electron-deficient π systems is different from those of electron-rich π systems.  Indeed, ab initio calcns. find that electron-deficient π ring systems should involve different structures and energetics, consistent with the CSD search, due to the elec. multipole moments and the decrease in the spatial extent of π-electron d.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-_MjOdwiK6LVg90H21EOLACvtfcHk0li94v4X1sbMVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnsFGnsro%253D&md5=2d88740775072e6eba2e195631e1871c</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fct100182u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct100182u%26sid%3Dliteratum%253Aachs%26aulast%3DGeronimo%26aufirst%3DI.%26aulast%3DLee%26aufirst%3DE.%2BC.%26aulast%3DSingh%26aufirst%3DN.%2BJ.%26aulast%3DKim%26aufirst%3DK.%2BS.%26atitle%3DHow%2520different%2520are%2520electron-rich%2520and%2520electron-deficient%2520pi%2520interactions%253F%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2010%26volume%3D6%26spage%3D1931%26epage%3D1934" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Mishra, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sathyamurthy, N.</span><span> </span><span class="NLM_article-title">Pi–pi interaction in pyridine</span> <span class="citation_source-journal">J. Phys. Chem. A</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">6</span><span class="NLM_x">–</span> <span class="NLM_lpage">8</span></span><div class="citationLinks">[<a href="/doi/10.1021/jp045218c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2005&pages=6-8&author=B.+K.+Mishraauthor=N.+Sathyamurthy&title=Pi%E2%80%93pi+interaction+in+pyridine"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fjp045218c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp045218c%26sid%3Dliteratum%253Aachs%26aulast%3DMishra%26aufirst%3DB.%2BK.%26aulast%3DSathyamurthy%26aufirst%3DN.%26atitle%3DPi%25E2%2580%2593pi%2520interaction%2520in%2520pyridine%26jtitle%3DJ.%2520Phys.%2520Chem.%2520A%26date%3D2005%26volume%3D109%26spage%3D6%26epage%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Mishra, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sathyamurthy, N.</span><span> </span><span class="NLM_article-title">Stacking interaction in pyrazine dimer</span> <span class="citation_source-journal">J. Theor. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">609</span><span class="NLM_x">–</span> <span class="NLM_lpage">619</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1142%2FS0219633606002532" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVSitbfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=609-619&author=B.+K.+Mishraauthor=N.+Sathyamurthy&title=Stacking+interaction+in+pyrazine+dimer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Stacking interaction in pyrazine dimer</span></div><div class="casAuthors">Mishra, Brijesh Kumar; Sathyamurthy, N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Theoretical & Computational Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">609-619</span>CODEN:
                <span class="NLM_cas:coden">JTCCAC</span>;
        ISSN:<span class="NLM_cas:issn">0219-6336</span>.
    
            (<span class="NLM_cas:orgname">World Scientific Publishing Co. Pte. Ltd.</span>)
        </div><div class="casAbstract">The structure and stability of pyrazine dimer in different orientations have been investigated using second- and fourth-order Moller-Plesset perturbation theory (MP2, MP4) and coupled-cluster singles and doubles with non-iterative perturbative triples method [CCSD(T)] with various basis sets (6-31G*, 6-311G**, 6-311++G**, cc-pVDZ, aug-cc-pVDZ, cc-pVTZ, and aug-cc-pVTZ).  With the largest basis set (aug-cc-pVTZ) at the MP2 level of theory, the calcd. binding energies (basis set superposition error cor.) for the sandwich, N-N axial-displaced, lateral-displaced, cross-displaced, T-shaped (TN-ring and TH-ring), and wedge-shaped (WN-C and WC-C) geometries are found to be 3.01, 4.18, 4.63, 5.92, 4.54, 2.78, 2.89, and 3.23 kcal/mol, resp.  Although MP2 calcns. predict the cross-displaced geometry to be the most stable one, MP4 and CCSD(T) calcns. show the most stable geometry to be T-shaped (TN-ring), with a center-of-mass sepn. of 4.2 Å.  Dispersion interaction seems to be the major source of attraction in all the geometries considered, while the nature of electrostatic interaction depends on the mutual orientation of the two rings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAuB99DvoNSrVg90H21EOLACvtfcHk0lhQYYIMuA3MTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVSitbfN&md5=d882b9bb68905939640a6ef4f658eab2</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1142%2FS0219633606002532&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1142%252FS0219633606002532%26sid%3Dliteratum%253Aachs%26aulast%3DMishra%26aufirst%3DB.%2BK.%26aulast%3DSathyamurthy%26aufirst%3DN.%26atitle%3DStacking%2520interaction%2520in%2520pyrazine%2520dimer%26jtitle%3DJ.%2520Theor.%2520Comput.%2520Chem.%26date%3D2006%26volume%3D5%26spage%3D609%26epage%3D619" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Lee, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jurecka, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarakeshwar, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hobza, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, K. S.</span><span> </span><span class="NLM_article-title">Understanding of assembly phenomena by aromatic–aromatic interactions: benzene dimer and the substituted systems</span> <span class="citation_source-journal">J. Phys. Chem. A</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">3446</span><span class="NLM_x">–</span> <span class="NLM_lpage">3457</span></span><div class="citationLinks">[<a href="/doi/10.1021/jp068635t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2007&pages=3446-3457&author=E.+C.+Leeauthor=D.+Kimauthor=P.+Jureckaauthor=P.+Tarakeshwarauthor=P.+Hobzaauthor=K.+S.+Kim&title=Understanding+of+assembly+phenomena+by+aromatic%E2%80%93aromatic+interactions%3A+benzene+dimer+and+the+substituted+systems"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fjp068635t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp068635t%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DE.%2BC.%26aulast%3DKim%26aufirst%3DD.%26aulast%3DJurecka%26aufirst%3DP.%26aulast%3DTarakeshwar%26aufirst%3DP.%26aulast%3DHobza%26aufirst%3DP.%26aulast%3DKim%26aufirst%3DK.%2BS.%26atitle%3DUnderstanding%2520of%2520assembly%2520phenomena%2520by%2520aromatic%25E2%2580%2593aromatic%2520interactions%253A%2520benzene%2520dimer%2520and%2520the%2520substituted%2520systems%26jtitle%3DJ.%2520Phys.%2520Chem.%2520A%26date%3D2007%26volume%3D111%26spage%3D3446%26epage%3D3457" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Sinnokrot, M. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherrill, C. D.</span><span> </span><span class="NLM_article-title">Substituent effects in pi–pi interactions: sandwich and T-shaped configurations</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">126</span><span class="NLM_x">, </span> <span class="NLM_fpage">7690</span><span class="NLM_x">–</span> <span class="NLM_lpage">7697</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja049434a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2004&pages=7690-7697&author=M.+O.+Sinnokrotauthor=C.+D.+Sherrill&title=Substituent+effects+in+pi%E2%80%93pi+interactions%3A+sandwich+and+T-shaped+configurations"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fja049434a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja049434a%26sid%3Dliteratum%253Aachs%26aulast%3DSinnokrot%26aufirst%3DM.%2BO.%26aulast%3DSherrill%26aufirst%3DC.%2BD.%26atitle%3DSubstituent%2520effects%2520in%2520pi%25E2%2580%2593pi%2520interactions%253A%2520sandwich%2520and%2520T-shaped%2520configurations%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2004%26volume%3D126%26spage%3D7690%26epage%3D7697" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Sakaguchi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrera, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saito, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bustin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vassilev, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderson, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Appella, E.</span><span> </span><span class="NLM_article-title">DNA damage activates p53 through a phosphorylation–acetylation cascade</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">2831</span><span class="NLM_x">–</span> <span class="NLM_lpage">2841</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=1998&pages=2831-2841&author=K.+Sakaguchiauthor=J.+E.+Herreraauthor=S.+Saitoauthor=T.+Mikiauthor=M.+Bustinauthor=A.+Vassilevauthor=C.+W.+Andersonauthor=E.+Appella&title=DNA+damage+activates+p53+through+a+phosphorylation%E2%80%93acetylation+cascade"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSakaguchi%26aufirst%3DK.%26aulast%3DHerrera%26aufirst%3DJ.%2BE.%26aulast%3DSaito%26aufirst%3DS.%26aulast%3DMiki%26aufirst%3DT.%26aulast%3DBustin%26aufirst%3DM.%26aulast%3DVassilev%26aufirst%3DA.%26aulast%3DAnderson%26aufirst%3DC.%2BW.%26aulast%3DAppella%26aufirst%3DE.%26atitle%3DDNA%2520damage%2520activates%2520p53%2520through%2520a%2520phosphorylation%25E2%2580%2593acetylation%2520cascade%26jtitle%3DGenes%2520Dev.%26date%3D1998%26volume%3D12%26spage%3D2831%26epage%3D2841" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Wang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guan, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tong, Q.</span><span> </span><span class="NLM_article-title">Deacetylation of FOXO3 by SIRT1 or SIRT2 leads to Skp2-mediated FOXO3 ubiquitination and degradation</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">1546</span><span class="NLM_x">–</span> <span class="NLM_lpage">1557</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2012&pages=1546-1557&author=F.+Wangauthor=C.+H.+Chanauthor=K.+Chenauthor=X.+Guanauthor=H.+K.+Linauthor=Q.+Tong&title=Deacetylation+of+FOXO3+by+SIRT1+or+SIRT2+leads+to+Skp2-mediated+FOXO3+ubiquitination+and+degradation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DF.%26aulast%3DChan%26aufirst%3DC.%2BH.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DGuan%26aufirst%3DX.%26aulast%3DLin%26aufirst%3DH.%2BK.%26aulast%3DTong%26aufirst%3DQ.%26atitle%3DDeacetylation%2520of%2520FOXO3%2520by%2520SIRT1%2520or%2520SIRT2%2520leads%2520to%2520Skp2-mediated%2520FOXO3%2520ubiquitination%2520and%2520degradation%26jtitle%3DOncogene%26date%3D2012%26volume%3D31%26spage%3D1546%26epage%3D1557" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Peng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ling, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukasawa, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robertson, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olashaw, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koomen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lane, W. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seto, E.</span><span> </span><span class="NLM_article-title">SIRT1 deacetylates the DNA methyltransferase 1 (DNMT1) protein and alters its activities</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">4720</span><span class="NLM_x">–</span> <span class="NLM_lpage">4734</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1128%2FMCB.06147-11" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Fjm301032j&amp;key=21947282" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFakt7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2011&pages=4720-4734&author=L.+Pengauthor=Z.+Yuanauthor=H.+Lingauthor=K.+Fukasawaauthor=K.+Robertsonauthor=N.+Olashawauthor=J.+Koomenauthor=J.+Chenauthor=W.+S.+Laneauthor=E.+Seto&title=SIRT1+deacetylates+the+DNA+methyltransferase+1+%28DNMT1%29+protein+and+alters+its+activities"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">SIRT1 deacetylates the DNA methyltransferase 1 (DNMT1) protein and alters its activities</span></div><div class="casAuthors">Peng, Lirong; Yuan, Zhigang; Ling, Hongbo; Fukasawa, Kenji; Robertson, Keith; Olashaw, Nancy; Koomen, John; Chen, Jiandong; Lane, William S.; Seto, Edward</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">4720-4734</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">DNA methylation and histone acetylation/deacetylation are distinct biochem. processes that control gene expression.  While DNA methylation is a common epigenetic signal that inhibits gene transcription, histone deacetylation similarly represses transcription but can be both an epigenetic and non-epigenetic phenomenon.  Here it is reported that the histone deacetylase SIRT1 regulates the activities of DNMT1, a key enzyme responsible for DNA methylation.  In mass spectrometry anal., 12 new acetylated lysine sites were identified in DNMT1.  SIRT1 phys. assocs. with DNMT1 and can deacetylate acetylated DNMT1 in vitro and in vivo.  Interestingly, deacetylation of different lysines on DNMT1 has different effects on the functions of DNMT1.  For example, deacetylation of Lys1349 and Lys1415 in the catalytic domain of DNMT1 enhances DNMT1's methyltransferase activity, while deacetylation of lysine residues in the GK linker decreases DNMT1's methyltransferase-independent transcriptional repression function.  Furthermore, deacetylation of all identified acetylated lysine sites in DNMT1 abrogates its binding to SIRT1 and impairs its capability to regulate cell cycle G2/M transition.  Finally, inhibition of SIRT1 strengthens the silencing effects of DNMT1 on the expression of tumor suppressor genes ER-α and CDH1 in MDA-MB-231 breast cancer cells.  Together, these results suggest that SIRT1-mediated deacetylation of DNMT1 is crucial for DNMT1's multiple effects in gene silencing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpb75sh63SEw7Vg90H21EOLACvtfcHk0liG_YnhYcZqAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFakt7rO&md5=e8d6e4eeea1f81de86fb43193f3a60f4</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1128%2FMCB.06147-11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.06147-11%26sid%3Dliteratum%253Aachs%26aulast%3DPeng%26aufirst%3DL.%26aulast%3DYuan%26aufirst%3DZ.%26aulast%3DLing%26aufirst%3DH.%26aulast%3DFukasawa%26aufirst%3DK.%26aulast%3DRobertson%26aufirst%3DK.%26aulast%3DOlashaw%26aufirst%3DN.%26aulast%3DKoomen%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DLane%26aufirst%3DW.%2BS.%26aulast%3DSeto%26aufirst%3DE.%26atitle%3DSIRT1%2520deacetylates%2520the%2520DNA%2520methyltransferase%25201%2520%2528DNMT1%2529%2520protein%2520and%2520alters%2520its%2520activities%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2011%26volume%3D31%26spage%3D4720%26epage%3D4734" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Vagin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teplyakov, A.</span><span> </span><span class="NLM_article-title">MOLREP: an automated program for molecular replacement</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">1022</span><span class="NLM_x">–</span> <span class="NLM_lpage">1025</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1107%2FS0021889897006766" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1997&pages=1022-1025&author=A.+Vaginauthor=A.+Teplyakov&title=MOLREP%3A+an+automated+program+for+molecular+replacement"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">MOLREP: an automated program for molecular replacement</span></div><div class="casAuthors">Vagin, Alexei; Teplyakov, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1022-1025</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">MOLREP is an automated program for mol. replacement which uses effective new approaches in data processing and rotational and translational searching.  These include an automatic choice of all parameters, scaling by Patterson origin peaks and soft resoln. cutoff.  One of the cornerstones of the program is an original full-symmetry translation function combined with a packing function.  Information from the model already placed in the cell is incorporated in both translation and packing functions.  A no. of tests using exptl. data proved the ability of the program to find the correct soln. in difficult cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkGXdZ-nTKxLVg90H21EOLACvtfcHk0liG_YnhYcZqAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D&md5=96af64aedd8a8f018c3c18b57a6b4b21</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1107%2FS0021889897006766&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889897006766%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DTeplyakov%26aufirst%3DA.%26atitle%3DMOLREP%253A%2520an%2520automated%2520program%2520for%2520molecular%2520replacement%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1997%26volume%3D30%26spage%3D1022%26epage%3D1025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Sheldrick, G. M.</span><span> </span><span class="NLM_article-title">A short history of SHELX</span> <span class="citation_source-journal">Acta Crystallogr., Sect. A: Found. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">64</span><span class="NLM_x">, </span> <span class="NLM_fpage">112</span><span class="NLM_x">–</span> <span class="NLM_lpage">122</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1107%2FS0108767307043930" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Fjm301032j&amp;key=18156677" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVGhurzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2008&pages=112-122&author=G.+M.+Sheldrick&title=A+short+history+of+SHELX"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">A short history of SHELX</span></div><div class="casAuthors">Sheldrick, George M.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section A: Foundations of Crystallography</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">112-122</span>CODEN:
                <span class="NLM_cas:coden">ACACEQ</span>;
        ISSN:<span class="NLM_cas:issn">0108-7673</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">An account is given of the development of the SHELX system of computer programs from SHELX-76 to the present day.  In addn. to identifying useful innovations that have come into general use through their implementation in SHELX, a crit. anal. is presented of the less-successful features, missed opportunities and desirable improvements for future releases of the software.  An attempt is made to understand how a program originally designed for photog. intensity data, punched cards and computers over 10000 times slower than an av. modern personal computer has managed to survive for so long.  SHELXL is the most widely used program for small-mol. refinement and SHELXS and SHELXD are often employed for structure soln. despite the availability of objectively superior programs.  SHELXL also finds a niche for the refinement of macromols. against high-resoln. or twinned data; SHELXPRO acts as an interface for macromol. applications.  SHELXC, SHELXD and SHELXE are proving useful for the exptl. phasing of macromols., esp. because they are fast and robust and so are often employed in pipelines for high-throughput phasing.  This paper could serve as a general literature citation when one or more of the open-source SHELX programs (and the Bruker AXS version SHELXTL) are employed in the course of a crystal-structure detn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfLPYyJyJ2RbVg90H21EOLACvtfcHk0liG_YnhYcZqAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVGhurzO&md5=8f91f10be4a9df74b2a9dad522c71e6e</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1107%2FS0108767307043930&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0108767307043930%26sid%3Dliteratum%253Aachs%26aulast%3DSheldrick%26aufirst%3DG.%2BM.%26atitle%3DA%2520short%2520history%2520of%2520SHELX%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520A%253A%2520Found.%2520Crystallogr.%26date%3D2008%26volume%3D64%26spage%3D112%26epage%3D122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Vagin, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steiner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lebedev, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Potterton, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McNicholas, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Long, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murshudov, G. N.</span><span> </span><span class="NLM_article-title">REFMAC5 dictionary: organization of prior chemical knowledge and guidelines for its use</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">2184</span><span class="NLM_x">–</span> <span class="NLM_lpage">2195</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1107%2FS0907444904023510" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Fjm301032j&amp;key=15572771" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2184-2195&author=A.+A.+Vaginauthor=R.+A.+Steinerauthor=A.+A.+Lebedevauthor=L.+Pottertonauthor=S.+McNicholasauthor=F.+Longauthor=G.+N.+Murshudov&title=REFMAC5+dictionary%3A+organization+of+prior+chemical+knowledge+and+guidelines+for+its+use"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">REFMAC5 dictionary: organization of prior chemical knowledge and guidelines for its use</span></div><div class="casAuthors">Vagin, Alexei A.; Steiner, Roberto A.; Lebedev, Andrey A.; Potterton, Liz; McNicholas, Stuart; Long, Fei; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2184-2195</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">One of the most important aspects of macromol. structure refinement is the use of prior chem. knowledge.  Bond lengths, bond angles and other chem. properties are used in restrained refinement as subsidiary conditions.  This contribution describes the organization and some aspects of the use of the flexible and human/machine-readable dictionary of prior chem. knowledge used by the max.-likelihood macromol.-refinement program REFMAC5.  The dictionary stores information about monomers which represent the constitutive building blocks of biol. macromols. (amino acids, nucleic acids and saccharides) and about numerous org./inorg. compds. commonly found in macromol. crystallog.  It also describes the modifications the building blocks undergo as a result of chem. reactions and the links required for polymer formation.  More than 2000 monomer entries, 100 modification entries and 200 link entries are currently available.  Algorithms and tools for updating and adding new entries to the dictionary have also been developed and are presented here.  In many cases, the REFMAC5 dictionary allows entirely automatic generation of restraints within REFMAC5 refinement runs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRjDd-k3YHGrVg90H21EOLACvtfcHk0lhDKhP4RyYvwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73F&md5=005910422c5fb72fe2434c39b5321bf4</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1107%2FS0907444904023510&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904023510%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DSteiner%26aufirst%3DR.%2BA.%26aulast%3DLebedev%26aufirst%3DA.%2BA.%26aulast%3DPotterton%26aufirst%3DL.%26aulast%3DMcNicholas%26aufirst%3DS.%26aulast%3DLong%26aufirst%3DF.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DREFMAC5%2520dictionary%253A%2520organization%2520of%2520prior%2520chemical%2520knowledge%2520and%2520guidelines%2520for%2520its%2520use%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2184%26epage%3D2195" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">2126</span><span class="NLM_x">–</span> <span class="NLM_lpage">2132</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Fjm301032j&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lhDKhP4RyYvwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Thompson, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higgins, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibson, T. J.</span><span> </span><span class="NLM_article-title">Clustal-W improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">4673</span><span class="NLM_x">–</span> <span class="NLM_lpage">4680</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1994&pages=4673-4680&author=J.+D.+Thompsonauthor=D.+G.+Higginsauthor=T.+J.+Gibson&title=Clustal-W+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DThompson%26aufirst%3DJ.%2BD.%26aulast%3DHiggins%26aufirst%3DD.%2BG.%26aulast%3DGibson%26aufirst%3DT.%2BJ.%26atitle%3DClustal-W%2520improving%2520the%2520sensitivity%2520of%2520progressive%2520multiple%2520sequence%2520alignment%2520through%2520sequence%2520weighting%252C%2520position-specific%2520gap%2520penalties%2520and%2520weight%2520matrix%2520choice%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D1994%26volume%3D22%26spage%3D4673%26epage%3D4680" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group">Bowie, J. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luthy, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eisenberg, D.</span><span> </span><span class="NLM_article-title">A method to identify protein sequences that fold into a known three-dimensional structure</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">253</span><span class="NLM_x">, </span> <span class="NLM_fpage">164</span><span class="NLM_x">–</span> <span class="NLM_lpage">170</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1126%2Fscience.1853201" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Fjm301032j&amp;key=1853201" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADyaK3MXltV2qsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=253&publication_year=1991&pages=164-170&author=J.+U.+Bowieauthor=R.+Luthyauthor=D.+Eisenberg&title=A+method+to+identify+protein+sequences+that+fold+into+a+known+three-dimensional+structure"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">A method to identify protein sequences that fold into a known three-dimensional structure</span></div><div class="casAuthors">Bowie, James U.; Luthy, Roland; Eisenberg, David</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">253</span>
        (<span class="NLM_cas:issue">5016</span>),
    <span class="NLM_cas:pages">164-70</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">The inverse protein folding problem, the problem of finding which amino acid sequences fold into a known three-dimensional (3D) structure, can be effectively attacked by finding sequences that are most compatible with the environments of the residues in the 3D structure.  The environments are described by: (i) the area of the residue buried in the protein and inaccessible to solvent; (ii) the fraction of side-chain area that is covered by polar atoms (O and N); and (iii) the local secondary structure.  Examples of this 3D profile method are presented for four families of proteins: the globins, cAMP (adenosine 3',5'-monophosphate) receptor-like proteins, the periplasmic binding proteins, and the actins.  This method is able to detect the structural similarity of the actins and 70-kilodalton heat shock proteins, even though these protein families share no detectable sequence similarity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBNyBVziUN_bVg90H21EOLACvtfcHk0lhDKhP4RyYvwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXltV2qsbk%253D&md5=18345191c3ef68ca2c2475522c9ee699</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1126%2Fscience.1853201&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1853201%26sid%3Dliteratum%253Aachs%26aulast%3DBowie%26aufirst%3DJ.%2BU.%26aulast%3DLuthy%26aufirst%3DR.%26aulast%3DEisenberg%26aufirst%3DD.%26atitle%3DA%2520method%2520to%2520identify%2520protein%2520sequences%2520that%2520fold%2520into%2520a%2520known%2520three-dimensional%2520structure%26jtitle%3DScience%26date%3D1991%26volume%3D253%26spage%3D164%26epage%3D170" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">Morris, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodsell, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halliday, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huey, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hart, W. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belew, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, A. J.</span><span> </span><span class="NLM_article-title">Automated docking using a Lamarckian genetic algorithm and an empirical binding free energy function</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">1639</span><span class="NLM_x">–</span> <span class="NLM_lpage">1662</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1002%2F%28SICI%291096-987X%2819981115%2919%3A14%3C1639%3A%3AAID-JCC10%3E3.0.CO%3B2-B" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADyaK1cXntFemur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=1998&pages=1639-1662&author=G.+M.+Morrisauthor=D.+S.+Goodsellauthor=R.+S.+Hallidayauthor=R.+Hueyauthor=W.+E.+Hartauthor=R.+K.+Belewauthor=A.+J.+Olson&title=Automated+docking+using+a+Lamarckian+genetic+algorithm+and+an+empirical+binding+free+energy+function"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Automated docking using a Lamarckian genetic algorithm and an empirical binding free energy function</span></div><div class="casAuthors">Morris, Garrett M.; Goodsell, David S.; Halliday, Robert S.; Huey, Ruth; Hart, William E.; Belew, Richard K.; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1639-1662</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review, with 81 refs.  A novel and robust automated docking method that predicts the bound conformations of flexible ligands to macromol. targets has been developed and tested, in combination with a new scoring function that ests. the free energy change upon binding.  Interestingly, this method applies a Lamarckian model of genetics, in which environmental adaptations of an individual's phenotype are reverse transcribed into its genotype and become heritable traits (sic).  We consider three search methods, Monte Carlo simulated annealing, a traditional genetic algorithm, and the Lamarckian genetic algorithm, and compare their performance in dockings of seven protein-ligand test systems having known three-dimensional structure.  We show that both the traditional and Lamarckian genetic algorithms can handle ligands with more degrees of freedom than the simulated annealing method used in earlier versions of AuToDOCK, and that the Lamarckian genetic algorithm is the most efficient, reliable, and successful of the three.  The empirical free energy function was calibrated using a set of 30 structurally known protein-ligand complexes with exptl. detd. binding consts.  Linear regression anal. of the obsd. binding consts. in terms of a wide variety of structure-derived mol. properties was performed.  The final model had a residual std. error of 9.11 kJ mol-1 (2.177 kcal mol-1) and was chosen as the new energy function.  The new search methods and empirical free energy function are available in AUTODOCK, version 3.0.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWXZfPvnZWcbVg90H21EOLACvtfcHk0lhRjUfU3NRwbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXntFemur4%253D&md5=804b9f797c0b2cf942444c17b5653659</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291096-987X%2819981115%2919%3A14%3C1639%3A%3AAID-JCC10%3E3.0.CO%3B2-B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291096-987X%252819981115%252919%253A14%253C1639%253A%253AAID-JCC10%253E3.0.CO%253B2-B%26sid%3Dliteratum%253Aachs%26aulast%3DMorris%26aufirst%3DG.%2BM.%26aulast%3DGoodsell%26aufirst%3DD.%2BS.%26aulast%3DHalliday%26aufirst%3DR.%2BS.%26aulast%3DHuey%26aufirst%3DR.%26aulast%3DHart%26aufirst%3DW.%2BE.%26aulast%3DBelew%26aufirst%3DR.%2BK.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DAutomated%2520docking%2520using%2520a%2520Lamarckian%2520genetic%2520algorithm%2520and%2520an%2520empirical%2520binding%2520free%2520energy%2520function%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1998%26volume%3D19%26spage%3D1639%26epage%3D1662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group">Morris, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huey, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindstrom, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanner, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belew, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodsell, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, A. J.</span><span> </span><span class="NLM_article-title">AutoDock4 and AutoDockTools4: automated docking with selective receptor flexibility</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">2785</span><span class="NLM_x">–</span> <span class="NLM_lpage">2791</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1002%2Fjcc.21256" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Fjm301032j&amp;key=19399780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1GitrnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2009&pages=2785-2791&author=G.+M.+Morrisauthor=R.+Hueyauthor=W.+Lindstromauthor=M.+F.+Sannerauthor=R.+K.+Belewauthor=D.+S.+Goodsellauthor=A.+J.+Olson&title=AutoDock4+and+AutoDockTools4%3A+automated+docking+with+selective+receptor+flexibility"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">AutoDock and AutoDockTools: Automated docking with selective receptor flexibility</span></div><div class="casAuthors">Morris, Garrett M.; Huey, Ruth; Lindstrom, William; Sanner, Michel F.; Belew, Richard K.; Goodsell, David S.; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2785-2791</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We describe the testing and release of AutoDock4 and the accompanying graphical user interface AutoDockTools.  AutoDock4 incorporates limited flexibility in the receptor.  Several tests are reported here, including a redocking expt. with 188 diverse ligand-protein complexes and a cross-docking expt. using flexible sidechains in 87 HIV protease complexes.  We also report its utility in anal. of covalently bound ligands, using both a grid-based docking method and a modification of the flexible sidechain technique. © 2009 Wiley Periodicals, Inc.  J Comput Chem, 2009.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqktE6WAo4GEbVg90H21EOLACvtfcHk0lhRjUfU3NRwbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1GitrnK&md5=679ce22fc50e9291c9aa16e7a1855845</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21256%26sid%3Dliteratum%253Aachs%26aulast%3DMorris%26aufirst%3DG.%2BM.%26aulast%3DHuey%26aufirst%3DR.%26aulast%3DLindstrom%26aufirst%3DW.%26aulast%3DSanner%26aufirst%3DM.%2BF.%26aulast%3DBelew%26aufirst%3DR.%2BK.%26aulast%3DGoodsell%26aufirst%3DD.%2BS.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DAutoDock4%2520and%2520AutoDockTools4%253A%2520automated%2520docking%2520with%2520selective%2520receptor%2520flexibility%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2009%26volume%3D30%26spage%3D2785%26epage%3D2791" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 24 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Robin Bonomi, Vadim Popov, Maxwell T. Laws, David Gelovani, Anjoy Majhi, Aleksandr Shavrin, Xin Lu, Otto Muzik, Nashaat Turkman, Renshyan Liu, Thomas Mangner, <span class="NLM_string-name hlFld-ContribAuthor">Juri G. Gelovani</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular Imaging of Sirtuin1 Expression–Activity in Rat Brain Using Positron-Emission Tomography–Magnetic-Resonance Imaging with [18F]-2-Fluorobenzoylaminohexanoicanilide. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (16)
                                     , 7116-7130. <a href="https://doi.org/10.1021/acs.jmedchem.8b00253" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00253</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00253&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b00253%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DMolecular%252BImaging%252Bof%252BSirtuin1%252BExpression%2525E2%252580%252593Activity%252Bin%252BRat%252BBrain%252BUsing%252BPositron-Emission%252BTomography%2525E2%252580%252593Magnetic-Resonance%252BImaging%252Bwith%252B%25255B18F%25255D-2-Fluorobenzoylaminohexanoicanilide%26aulast%3DBonomi%26aufirst%3DRobin%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D14022018%26date%3D13082018%26date%3D27072018%26volume%3D61%26issue%3D16%26spage%3D7116%26epage%3D7130" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sergio  Valente</span>, <span class="hlFld-ContribAuthor ">Paolo  Mellini</span>, <span class="hlFld-ContribAuthor ">Francesco  Spallotta</span>, <span class="hlFld-ContribAuthor ">Vincenzo  Carafa</span>, <span class="hlFld-ContribAuthor ">Angela  Nebbioso</span>, <span class="hlFld-ContribAuthor ">Lucia  Polletta</span>, <span class="hlFld-ContribAuthor ">Ilaria  Carnevale</span>, <span class="hlFld-ContribAuthor ">Serena  Saladini</span>, <span class="hlFld-ContribAuthor ">Daniela  Trisciuoglio</span>, <span class="hlFld-ContribAuthor ">Chiara  Gabellini</span>, <span class="hlFld-ContribAuthor ">Maria  Tardugno</span>, <span class="hlFld-ContribAuthor ">Clemens  Zwergel</span>, <span class="hlFld-ContribAuthor ">Chiara  Cencioni</span>, <span class="hlFld-ContribAuthor ">Sandra  Atlante</span>, <span class="hlFld-ContribAuthor ">Sébastien  Moniot</span>, <span class="hlFld-ContribAuthor ">Clemens  Steegborn</span>, <span class="hlFld-ContribAuthor ">Roberta  Budriesi</span>, <span class="hlFld-ContribAuthor ">Marco  Tafani</span>, <span class="hlFld-ContribAuthor ">Donatella  Del Bufalo</span>, <span class="hlFld-ContribAuthor ">Lucia  Altucci</span>, <span class="hlFld-ContribAuthor ">Carlo  Gaetano</span>, and <span class="hlFld-ContribAuthor ">Antonello  Mai</span>  . </span><span class="cited-content_cbyCitation_article-title">1,4-Dihydropyridines Active on the SIRT1/AMPK Pathway Ameliorate Skin Repair and Mitochondrial Function and Exhibit Inhibition of Proliferation in Cancer Cells. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (4)
                                     , 1471-1491. <a href="https://doi.org/10.1021/acs.jmedchem.5b01117" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01117</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.5b01117%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3D1%25252C4-Dihydropyridines%252BActive%252Bon%252Bthe%252BSIRT1%25252FAMPK%252BPathway%252BAmeliorate%252BSkin%252BRepair%252Band%252BMitochondrial%252BFunction%252Band%252BExhibit%252BInhibition%252Bof%252BProliferation%252Bin%252BCancer%252BCells%26aulast%3DValente%26aufirst%3DSergio%26date%3D2016%26date%3D2016%26date%3D2015%26date%3D16072015%26date%3D05012016%26date%3D25022016%26date%3D21122015%26volume%3D59%26issue%3D4%26spage%3D1471%26epage%3D1491" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Man  Li</span>, <span class="hlFld-ContribAuthor ">Shi-chun  Li</span>, <span class="hlFld-ContribAuthor ">Bao-kai  Dou</span>, <span class="hlFld-ContribAuthor ">Ying-xiang  Zou</span>, <span class="hlFld-ContribAuthor ">Hao-zhen  Han</span>, <span class="hlFld-ContribAuthor ">Dong-xiang  Liu</span>, <span class="hlFld-ContribAuthor ">Zun-ji  Ke</span>, <span class="hlFld-ContribAuthor ">Zhi-fei  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Cycloastragenol upregulates SIRT1 expression, attenuates apoptosis and suppresses neuroinflammation after brain ischemia. </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmacologica Sinica</span><span> <strong>2020,</strong> <em>41 </em>
                                    (8)
                                     , 1025-1032. <a href="https://doi.org/10.1038/s41401-020-0386-6" title="DOI URL">https://doi.org/10.1038/s41401-020-0386-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41401-020-0386-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41401-020-0386-6%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmacologica%2520Sinica%26atitle%3DCycloastragenol%252Bupregulates%252BSIRT1%252Bexpression%25252C%252Battenuates%252Bapoptosis%252Band%252Bsuppresses%252Bneuroinflammation%252Bafter%252Bbrain%252Bischemia%26aulast%3DLi%26aufirst%3DMan%26date%3D2020%26date%3D2020%26volume%3D41%26issue%3D8%26spage%3D1025%26epage%3D1032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Haozhen  Han</span>, <span class="hlFld-ContribAuthor ">Chunpu  Li</span>, <span class="hlFld-ContribAuthor ">Man  Li</span>, <span class="hlFld-ContribAuthor ">Lisheng  Yang</span>, <span class="hlFld-ContribAuthor ">Sen  Zhao</span>, <span class="hlFld-ContribAuthor ">Zhifei  Wang</span>, <span class="hlFld-ContribAuthor ">Hong  Liu</span>, <span class="hlFld-ContribAuthor ">Dongxiang  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Design, Synthesis, and Biological Evaluation of 8-Mercapto-3,7-Dihydro-1H-Purine-2,6-Diones as Potent Inhibitors of SIRT1, SIRT2, SIRT3, and SIRT5. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2020,</strong> <em>25 </em>
                                    (12)
                                     , 2755. <a href="https://doi.org/10.3390/molecules25122755" title="DOI URL">https://doi.org/10.3390/molecules25122755</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules25122755&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules25122755%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DDesign%25252C%252BSynthesis%25252C%252Band%252BBiological%252BEvaluation%252Bof%252B8-Mercapto-3%25252C7-Dihydro-1H-Purine-2%25252C6-Diones%252Bas%252BPotent%252BInhibitors%252Bof%252BSIRT1%25252C%252BSIRT2%25252C%252BSIRT3%25252C%252Band%252BSIRT5%26aulast%3DHan%26aufirst%3DHaozhen%26date%3D2020%26date%3D2020%26volume%3D25%26issue%3D12%26spage%3D2755" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ting  Wang</span>, <span class="hlFld-ContribAuthor ">Yujue  Wang</span>, <span class="hlFld-ContribAuthor ">Li  Liu</span>, <span class="hlFld-ContribAuthor ">Zhongliang  Jiang</span>, <span class="hlFld-ContribAuthor ">Xingxing  Li</span>, <span class="hlFld-ContribAuthor ">Rongsheng  Tong</span>, <span class="hlFld-ContribAuthor ">Jun  He</span>, <span class="hlFld-ContribAuthor ">Jianyou  Shi</span>. </span><span class="cited-content_cbyCitation_article-title">Research progress on sirtuins family members and cell senescence. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>193 </em>, 112207. <a href="https://doi.org/10.1016/j.ejmech.2020.112207" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112207</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112207&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112207%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DResearch%252Bprogress%252Bon%252Bsirtuins%252Bfamily%252Bmembers%252Band%252Bcell%252Bsenescence%26aulast%3DWang%26aufirst%3DTing%26date%3D2020%26volume%3D193%26spage%3D112207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ling-Ling  Yang</span>, <span class="hlFld-ContribAuthor ">Hua-Li  Wang</span>, <span class="hlFld-ContribAuthor ">Yu-Hang  Yan</span>, <span class="hlFld-ContribAuthor ">Sha  Liu</span>, <span class="hlFld-ContribAuthor ">Zhu-Jun  Yu</span>, <span class="hlFld-ContribAuthor ">Meng-Yi  Huang</span>, <span class="hlFld-ContribAuthor ">Yubin  Luo</span>, <span class="hlFld-ContribAuthor ">Xi  Zheng</span>, <span class="hlFld-ContribAuthor ">Yamei  Yu</span>, <span class="hlFld-ContribAuthor ">Guo-Bo  Li</span>. </span><span class="cited-content_cbyCitation_article-title">Sensitive fluorogenic substrates for sirtuin deacylase inhibitor discovery. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>192 </em>, 112201. <a href="https://doi.org/10.1016/j.ejmech.2020.112201" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112201</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112201&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112201%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSensitive%252Bfluorogenic%252Bsubstrates%252Bfor%252Bsirtuin%252Bdeacylase%252Binhibitor%252Bdiscovery%26aulast%3DYang%26aufirst%3DLing-Ling%26date%3D2020%26volume%3D192%26spage%3D112201" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Azminah  Azminah</span>, <span class="hlFld-ContribAuthor ">Linda  Erlina</span>, <span class="hlFld-ContribAuthor ">Maksum  Radji</span>, <span class="hlFld-ContribAuthor ">Abdul  Mun’im</span>, <span class="hlFld-ContribAuthor ">Rezi Riadhi  Syahdi</span>, <span class="hlFld-ContribAuthor ">Arry  Yanuar</span>. </span><span class="cited-content_cbyCitation_article-title">In silico and in vitro identification of candidate SIRT1 activators from Indonesian medicinal plants compounds database. </span><span class="cited-content_cbyCitation_journal-name">Computational Biology and Chemistry</span><span> <strong>2019,</strong> <em>83 </em>, 107096. <a href="https://doi.org/10.1016/j.compbiolchem.2019.107096" title="DOI URL">https://doi.org/10.1016/j.compbiolchem.2019.107096</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.compbiolchem.2019.107096&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.compbiolchem.2019.107096%26sid%3Dliteratum%253Aachs%26jtitle%3DComputational%2520Biology%2520and%2520Chemistry%26atitle%3DIn%252Bsilico%252Band%252Bin%252Bvitro%252Bidentification%252Bof%252Bcandidate%252BSIRT1%252Bactivators%252Bfrom%252BIndonesian%252Bmedicinal%252Bplants%252Bcompounds%252Bdatabase%26aulast%3DAzminah%26aufirst%3DAzminah%26date%3D2019%26volume%3D83%26spage%3D107096" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chiara  Luise</span>, <span class="hlFld-ContribAuthor ">Tino  Heimburg</span>, <span class="hlFld-ContribAuthor ">Berin  Karaman</span>, <span class="hlFld-ContribAuthor ">Dina  Robaa</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Sippl</span>. </span><span class="cited-content_cbyCitation_article-title">Computer‐based Lead Identification for Epigenetic Targets. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 45-77. <a href="https://doi.org/10.1002/9783527809257.ch3" title="DOI URL">https://doi.org/10.1002/9783527809257.ch3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527809257.ch3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527809257.ch3%26sid%3Dliteratum%253Aachs%26atitle%3DComputer%2525E2%252580%252590based%252BLead%252BIdentification%252Bfor%252BEpigenetic%252BTargets%26aulast%3DLuise%26aufirst%3DChiara%26date%3D2019%26date%3D2019%26spage%3D45%26epage%3D77%26pub%3DWiley%26atitle%3DEpigenetic%252BDrug%252BDiscovery%26aulast%3DSippl%26aufirst%3DWolfgang%26date%3D2019%26date%3D2019%26volume%3D286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dina  Robaa</span>, <span class="hlFld-ContribAuthor ">Jelena  Melesina</span>, <span class="hlFld-ContribAuthor ">Chiara  Luise</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Sippl</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based Design of Epigenetic Inhibitors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 455-486. <a href="https://doi.org/10.1007/7355_2019_80" title="DOI URL">https://doi.org/10.1007/7355_2019_80</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7355_2019_80&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7355_2019_80%26sid%3Dliteratum%253Aachs%26atitle%3DStructure-Based%252BDesign%252Bof%252BEpigenetic%252BInhibitors%26aulast%3DRobaa%26aufirst%3DDina%26date%3D2019%26date%3D2019%26spage%3D455%26epage%3D486%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DChemical%252BEpigenetics%26aulast%3DMai%26aufirst%3DAntonello%26date%3D2020%26volume%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jiansong  Fang</span>, <span class="hlFld-ContribAuthor ">Chuipu  Cai</span>, <span class="hlFld-ContribAuthor ">Yanting  Chai</span>, <span class="hlFld-ContribAuthor ">Jingwei  Zhou</span>, <span class="hlFld-ContribAuthor ">Yujie  Huang</span>, <span class="hlFld-ContribAuthor ">Li  Gao</span>, <span class="hlFld-ContribAuthor ">Qi  Wang</span>, <span class="hlFld-ContribAuthor ">Feixiong  Cheng</span>. </span><span class="cited-content_cbyCitation_article-title">Quantitative and systems pharmacology 4. Network-based analysis of drug pleiotropy on coronary artery disease. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>161 </em>, 192-204. <a href="https://doi.org/10.1016/j.ejmech.2018.10.020" title="DOI URL">https://doi.org/10.1016/j.ejmech.2018.10.020</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2018.10.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2018.10.020%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DQuantitative%252Band%252Bsystems%252Bpharmacology%252B4.%252BNetwork-based%252Banalysis%252Bof%252Bdrug%252Bpleiotropy%252Bon%252Bcoronary%252Bartery%252Bdisease%26aulast%3DFang%26aufirst%3DJiansong%26date%3D2019%26volume%3D161%26spage%3D192%26epage%3D204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sreekanth  Thota</span>, <span class="hlFld-ContribAuthor ">Daniel A.  Rodrigues</span>, <span class="hlFld-ContribAuthor ">Pedro de Sena Murteira  Pinheiro</span>, <span class="hlFld-ContribAuthor ">Lídia M.  Lima</span>, <span class="hlFld-ContribAuthor ">Carlos A.M.  Fraga</span>, <span class="hlFld-ContribAuthor ">Eliezer J.  Barreiro</span>. </span><span class="cited-content_cbyCitation_article-title">N-Acylhydrazones as drugs. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2018,</strong> <em>28 </em>
                                    (17)
                                     , 2797-2806. <a href="https://doi.org/10.1016/j.bmcl.2018.07.015" title="DOI URL">https://doi.org/10.1016/j.bmcl.2018.07.015</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2018.07.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2018.07.015%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DN-Acylhydrazones%252Bas%252Bdrugs%26aulast%3DThota%26aufirst%3DSreekanth%26date%3D2018%26volume%3D28%26issue%3D17%26spage%3D2797%26epage%3D2806" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matthias  Schiedel</span>, <span class="hlFld-ContribAuthor ">Dina  Robaa</span>, <span class="hlFld-ContribAuthor ">Tobias  Rumpf</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Sippl</span>, <span class="hlFld-ContribAuthor ">Manfred  Jung</span>. </span><span class="cited-content_cbyCitation_article-title">The Current State of NAD
              +
              -Dependent Histone Deacetylases (Sirtuins) as Novel Therapeutic Targets. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Research Reviews</span><span> <strong>2018,</strong> <em>38 </em>
                                    (1)
                                     , 147-200. <a href="https://doi.org/10.1002/med.21436" title="DOI URL">https://doi.org/10.1002/med.21436</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/med.21436&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fmed.21436%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Research%2520Reviews%26atitle%3DThe%252BCurrent%252BState%252Bof%252BNAD%252B%25252B%252B-Dependent%252BHistone%252BDeacetylases%252B%252528Sirtuins%252529%252Bas%252BNovel%252BTherapeutic%252BTargets%26aulast%3DSchiedel%26aufirst%3DMatthias%26date%3D2018%26date%3D2017%26volume%3D38%26issue%3D1%26spage%3D147%26epage%3D200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rajitha  Sadashiva</span>, <span class="hlFld-ContribAuthor ">Damodara  Naral</span>, <span class="hlFld-ContribAuthor ">Jyothi  Kudva</span>, <span class="hlFld-ContribAuthor ">Naveen  Shivalingegowda</span>, <span class="hlFld-ContribAuthor ">Neratur Krishnappagowda  Lokanath</span>, <span class="hlFld-ContribAuthor ">Kudigana Jayaprakash  Pampa</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis, spectral, biological activity, and crystal structure evaluation of novel pyrazoline derivatives having sulfonamide moiety. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Chemistry Research</span><span> <strong>2017,</strong> <em>26 </em>
                                    (6)
                                     , 1213-1227. <a href="https://doi.org/10.1007/s00044-017-1838-5" title="DOI URL">https://doi.org/10.1007/s00044-017-1838-5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00044-017-1838-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00044-017-1838-5%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Chemistry%2520Research%26atitle%3DSynthesis%25252C%252Bspectral%25252C%252Bbiological%252Bactivity%25252C%252Band%252Bcrystal%252Bstructure%252Bevaluation%252Bof%252Bnovel%252Bpyrazoline%252Bderivatives%252Bhaving%252Bsulfonamide%252Bmoiety%26aulast%3DSadashiva%26aufirst%3DRajitha%26date%3D2017%26date%3D2017%26volume%3D26%26issue%3D6%26spage%3D1213%26epage%3D1227" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rahul  Kumar</span>, <span class="hlFld-ContribAuthor ">Lokesh  Nigam</span>, <span class="hlFld-ContribAuthor ">Amrendra Pratap  Singh</span>, <span class="hlFld-ContribAuthor ">Kusum  Singh</span>, <span class="hlFld-ContribAuthor ">Naidu  Subbarao</span>, <span class="hlFld-ContribAuthor ">Sharmistha  Dey</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis of allosteric peptide activator for human SIRT1 and its biological evaluation in cellular model of Alzheimer's disease. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>127 </em>, 909-916. <a href="https://doi.org/10.1016/j.ejmech.2016.11.001" title="DOI URL">https://doi.org/10.1016/j.ejmech.2016.11.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2016.11.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2016.11.001%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Bof%252Ballosteric%252Bpeptide%252Bactivator%252Bfor%252Bhuman%252BSIRT1%252Band%252Bits%252Bbiological%252Bevaluation%252Bin%252Bcellular%252Bmodel%252Bof%252BAlzheimer%252527s%252Bdisease%26aulast%3DKumar%26aufirst%3DRahul%26date%3D2017%26volume%3D127%26spage%3D909%26epage%3D916" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lionel  Sacconnay</span>, <span class="hlFld-ContribAuthor ">Pierre-Alain  Carrupt</span>, <span class="hlFld-ContribAuthor ">Alessandra  Nurisso</span>. </span><span class="cited-content_cbyCitation_article-title">Human sirtuins: Structures and flexibility. </span><span class="cited-content_cbyCitation_journal-name">Journal of Structural Biology</span><span> <strong>2016,</strong> <em>196 </em>
                                    (3)
                                     , 534-542. <a href="https://doi.org/10.1016/j.jsb.2016.10.008" title="DOI URL">https://doi.org/10.1016/j.jsb.2016.10.008</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jsb.2016.10.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jsb.2016.10.008%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Structural%2520Biology%26atitle%3DHuman%252Bsirtuins%25253A%252BStructures%252Band%252Bflexibility%26aulast%3DSacconnay%26aufirst%3DLionel%26date%3D2016%26volume%3D196%26issue%3D3%26spage%3D534%26epage%3D542" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wei  Gai</span>, <span class="hlFld-ContribAuthor ">He  Li</span>, <span class="hlFld-ContribAuthor ">Hualiang  Jiang</span>, <span class="hlFld-ContribAuthor ">Yaqiu  Long</span>, <span class="hlFld-ContribAuthor ">Dongxiang  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Crystal structures of SIRT3 reveal that the α2-α3 loop and α3-helix affect the interaction with long-chain acyl lysine. </span><span class="cited-content_cbyCitation_journal-name">FEBS Letters</span><span> <strong>2016,</strong> <em>590 </em>
                                    (17)
                                     , 3019-3028. <a href="https://doi.org/10.1002/1873-3468.12345" title="DOI URL">https://doi.org/10.1002/1873-3468.12345</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/1873-3468.12345&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F1873-3468.12345%26sid%3Dliteratum%253Aachs%26jtitle%3DFEBS%2520Letters%26atitle%3DCrystal%252Bstructures%252Bof%252BSIRT3%252Breveal%252Bthat%252Bthe%252B%2525CE%2525B12-%2525CE%2525B13%252Bloop%252Band%252B%2525CE%2525B13-helix%252Baffect%252Bthe%252Binteraction%252Bwith%252Blong-chain%252Bacyl%252Blysine%26aulast%3DGai%26aufirst%3DWei%26date%3D2016%26date%3D2016%26volume%3D590%26issue%3D17%26spage%3D3019%26epage%3D3028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ashwani  Kumar</span>, <span class="hlFld-ContribAuthor ">Shilpi  Chauhan</span>. </span><span class="cited-content_cbyCitation_article-title">How much successful are the medicinal chemists in modulation of SIRT1: A critical review. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>119 </em>, 45-69. <a href="https://doi.org/10.1016/j.ejmech.2016.04.063" title="DOI URL">https://doi.org/10.1016/j.ejmech.2016.04.063</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2016.04.063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2016.04.063%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DHow%252Bmuch%252Bsuccessful%252Bare%252Bthe%252Bmedicinal%252Bchemists%252Bin%252Bmodulation%252Bof%252BSIRT1%25253A%252BA%252Bcritical%252Breview%26aulast%3DKumar%26aufirst%3DAshwani%26date%3D2016%26volume%3D119%26spage%3D45%26epage%3D69" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Berin  Karaman</span>, <span class="hlFld-ContribAuthor ">Manfred  Jung</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Sippl</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based Design and Computational Studies of Sirtuin Inhibitors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2016,</strong>,, 297-325. <a href="https://doi.org/10.1016/B978-0-12-802808-7.00011-3" title="DOI URL">https://doi.org/10.1016/B978-0-12-802808-7.00011-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-802808-7.00011-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-802808-7.00011-3%26sid%3Dliteratum%253Aachs%26atitle%3DStructure-Based%252BDesign%252Band%252BComputational%252BStudies%252Bof%252BSirtuin%252BInhibitors%26aulast%3DKaraman%26aufirst%3DBerin%26date%3D2016%26spage%3D297%26epage%3D325%26pub%3DElsevier%26atitle%3DEpi-Informatics%26date%3D2016%26volume%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sung Pyo  Hong</span>, <span class="hlFld-ContribAuthor ">Ky Youb  Nam</span>, <span class="hlFld-ContribAuthor ">Young June  Shin</span>, <span class="hlFld-ContribAuthor ">Kil Won  Kim</span>, <span class="hlFld-ContribAuthor ">Soon Kil  Ahn</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of 11β-hydroxysteroid dehydrogenase type 1 inhibitor. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2015,</strong> <em>25 </em>
                                    (17)
                                     , 3501-3506. <a href="https://doi.org/10.1016/j.bmcl.2015.06.099" title="DOI URL">https://doi.org/10.1016/j.bmcl.2015.06.099</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2015.06.099&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2015.06.099%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDiscovery%252Bof%252B11%2525CE%2525B2-hydroxysteroid%252Bdehydrogenase%252Btype%252B1%252Binhibitor%26aulast%3DHong%26aufirst%3DSung%2BPyo%26date%3D2015%26volume%3D25%26issue%3D17%26spage%3D3501%26epage%3D3506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Cathrin D.  Ertl</span>, <span class="hlFld-ContribAuthor ">Daniel P.  Ris</span>, <span class="hlFld-ContribAuthor ">Stefan C.  Meier</span>, <span class="hlFld-ContribAuthor ">Edwin C.  Constable</span>, <span class="hlFld-ContribAuthor ">Catherine E.  Housecroft</span>, <span class="hlFld-ContribAuthor ">Markus  Neuburger</span>, <span class="hlFld-ContribAuthor ">Jennifer A.  Zampese</span>. </span><span class="cited-content_cbyCitation_article-title">Sticking and patching: tuning and anchoring cyclometallated ruthenium(
              ii
              ) complexes. </span><span class="cited-content_cbyCitation_journal-name">Dalton Transactions</span><span> <strong>2015,</strong> <em>44 </em>
                                    (4)
                                     , 1557-1570. <a href="https://doi.org/10.1039/C4DT02797C" title="DOI URL">https://doi.org/10.1039/C4DT02797C</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C4DT02797C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC4DT02797C%26sid%3Dliteratum%253Aachs%26jtitle%3DDalton%2520Transactions%26atitle%3DSticking%252Band%252Bpatching%25253A%252Btuning%252Band%252Banchoring%252Bcyclometallated%252Bruthenium%252528%252Bii%252B%252529%252Bcomplexes%26aulast%3DErtl%26aufirst%3DCathrin%2BD.%26date%3D2015%26date%3D2015%26volume%3D44%26issue%3D4%26spage%3D1557%26epage%3D1570" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wen  Ai</span>, <span class="hlFld-ContribAuthor ">Xiaoyi  Wei</span>, <span class="hlFld-ContribAuthor ">Xiuping  Lin</span>, <span class="hlFld-ContribAuthor ">Li  Sheng</span>, <span class="hlFld-ContribAuthor ">Zhen  Wang</span>, <span class="hlFld-ContribAuthor ">Zhengchao  Tu</span>, <span class="hlFld-ContribAuthor ">Xianwen  Yang</span>, <span class="hlFld-ContribAuthor ">Xuefeng  Zhou</span>, <span class="hlFld-ContribAuthor ">Jia  Li</span>, <span class="hlFld-ContribAuthor ">Yonghong  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Guignardins A–F, spirodioxynaphthalenes from the endophytic fungus Guignardia sp. KcF8 as a new class of PTP1B and SIRT1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Tetrahedron</span><span> <strong>2014,</strong> <em>70 </em>
                                    (35)
                                     , 5806-5814. <a href="https://doi.org/10.1016/j.tet.2014.06.041" title="DOI URL">https://doi.org/10.1016/j.tet.2014.06.041</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tet.2014.06.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tet.2014.06.041%26sid%3Dliteratum%253Aachs%26jtitle%3DTetrahedron%26atitle%3DGuignardins%252BA%2525E2%252580%252593F%25252C%252Bspirodioxynaphthalenes%252Bfrom%252Bthe%252Bendophytic%252Bfungus%252BGuignardia%252Bsp.%252BKcF8%252Bas%252Ba%252Bnew%252Bclass%252Bof%252BPTP1B%252Band%252BSIRT1%252Binhibitors%26aulast%3DAi%26aufirst%3DWen%26date%3D2014%26volume%3D70%26issue%3D35%26spage%3D5806%26epage%3D5814" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhiyong  Cheng</span>, <span class="hlFld-ContribAuthor ">Eva M.  Schmelz</span>, <span class="hlFld-ContribAuthor ">Dongmin  Liu</span>, <span class="hlFld-ContribAuthor ">Matthew W.  Hulver</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting mitochondrial alterations to prevent type 2 diabetes-Evidence from studies of dietary redox-active compounds. </span><span class="cited-content_cbyCitation_journal-name">Molecular Nutrition & Food Research</span><span> <strong>2014,</strong> <em>58 </em>
                                    (8)
                                     , 1739-1749. <a href="https://doi.org/10.1002/mnfr.201300747" title="DOI URL">https://doi.org/10.1002/mnfr.201300747</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/mnfr.201300747&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fmnfr.201300747%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Nutrition%2520%2526%2520Food%2520Research%26atitle%3DTargeting%252Bmitochondrial%252Balterations%252Bto%252Bprevent%252Btype%252B2%252Bdiabetes-Evidence%252Bfrom%252Bstudies%252Bof%252Bdietary%252Bredox-active%252Bcompounds%26aulast%3DCheng%26aufirst%3DZhiyong%26date%3D2014%26date%3D2014%26volume%3D58%26issue%3D8%26spage%3D1739%26epage%3D1749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">David A.  Sinclair</span>, <span class="hlFld-ContribAuthor ">Leonard  Guarente</span>. </span><span class="cited-content_cbyCitation_article-title">Small-Molecule Allosteric Activators of Sirtuins. </span><span class="cited-content_cbyCitation_journal-name">Annual Review of Pharmacology and Toxicology</span><span> <strong>2014,</strong> <em>54 </em>
                                    (1)
                                     , 363-380. <a href="https://doi.org/10.1146/annurev-pharmtox-010611-134657" title="DOI URL">https://doi.org/10.1146/annurev-pharmtox-010611-134657</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1146/annurev-pharmtox-010611-134657&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1146%2Fannurev-pharmtox-010611-134657%26sid%3Dliteratum%253Aachs%26jtitle%3DAnnual%2520Review%2520of%2520Pharmacology%2520and%2520Toxicology%26atitle%3DSmall-Molecule%252BAllosteric%252BActivators%252Bof%252BSirtuins%26aulast%3DSinclair%26aufirst%3DDavid%2BA.%26date%3D2014%26volume%3D54%26issue%3D1%26spage%3D363%26epage%3D380" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuji  Matsuya</span>, <span class="hlFld-ContribAuthor ">Yuta  Kobayashi</span>, <span class="hlFld-ContribAuthor ">Sayumi  Uchida</span>, <span class="hlFld-ContribAuthor ">Yukihiro  Itoh</span>, <span class="hlFld-ContribAuthor ">Hideyuki  Sawada</span>, <span class="hlFld-ContribAuthor ">Takayoshi  Suzuki</span>, <span class="hlFld-ContribAuthor ">Naoki  Miyata</span>, <span class="hlFld-ContribAuthor ">Kenji  Sugimoto</span>, <span class="hlFld-ContribAuthor ">Naoki  Toyooka</span>. </span><span class="cited-content_cbyCitation_article-title">Search for a novel SIRT1 activator: Structural modification of SRT1720 and biological evaluation. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2013,</strong> <em>23 </em>
                                    (17)
                                     , 4907-4910. <a href="https://doi.org/10.1016/j.bmcl.2013.06.070" title="DOI URL">https://doi.org/10.1016/j.bmcl.2013.06.070</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2013.06.070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2013.06.070%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DSearch%252Bfor%252Ba%252Bnovel%252BSIRT1%252Bactivator%25253A%252BStructural%252Bmodification%252Bof%252BSRT1720%252Band%252Bbiological%252Bevaluation%26aulast%3DMatsuya%26aufirst%3DYuji%26date%3D2013%26volume%3D23%26issue%3D17%26spage%3D4907%26epage%3D4910" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/medium/jm-2012-01032j_0020.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm301032j&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/medium/jm-2012-01032j_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical structures of representative SIRT1 activators.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm301032j&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/medium/jm-2012-01032j_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0011.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>16a</b>–<b>n</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm301032j&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) Pd(PPh<sub>3</sub>)<sub>4</sub>, KF·2H<sub>2</sub>O, DME/H<sub>2</sub>O/EtOH (7:3:2), <i>M</i><sub>w</sub>, 110 °C; (ii) N<sub>2</sub>H<sub>4</sub>·H<sub>2</sub>O, MeOH, reflux; (iii) AcOH, EtOH, reflux.</p></p></figure><figure data-id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/medium/jm-2012-01032j_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0012.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>20a</b>–<b>i</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm301032j&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) Pd(PPh<sub>3</sub>)<sub>4</sub>, KF·2H<sub>2</sub>O, DME/H<sub>2</sub>O/EtOH (7:3:2), <i>M</i><sub>w</sub>, 110 °C; (ii) N<sub>2</sub>H<sub>4</sub>·H<sub>2</sub>O, MeOH, reflux; (iii) AcOH, EtOH, reflux.</p></p></figure><figure data-id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/medium/jm-2012-01032j_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. The dose–response curves of our compounds and <b>1</b> measured by the fluorescent assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm301032j&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/medium/jm-2012-01032j_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. The Lineweaver–Burk plots of (A) ac-RHKK<sub>ac</sub>-AMC peptide and (B) ac-RHKK<sub>ac</sub>-NH<sub>2</sub> peptide in the presence of 50 μM <b>4</b> and 10 μM <b>20b</b>, which were determined by HPLC method. <b>4</b> and <b>20b</b> exhibited substrate selectivity against native peptide and fluorophore-labeled peptide.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm301032j&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/medium/jm-2012-01032j_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Structure of human SIRT3 in complex with substrate ac-RHKK<sub>ac</sub>-AMC. (A) Ribbon diagram of the overall structure of the complexes in dimer. One molecule of SIRT3 protein is shown in green and the other in cyan. The substrate is shown in a stick representation; carbon, oxygen, and nitrogen are colored in magentas, red, and blue, respectively. Zinc ions are highlighted in red. (B) An enlarged view of the binding site in dimeric arrangement. The coumarin fluorophores of the substrates are packed in a “head-in-head” conformation, and the two peptides are represented with the 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density maps (1.0 σ-contoured). The labeled amino acids are involved in the packing. (C) The detailed interaction of Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>B without map. The black dashed lines represent the H-bonds formed between the substrate and the residues. (D) An enlarged view of the active site in an asymmetric cell. Four hydrogen bonds (represented with black dashed lines) are formed between the substrate and the protein. Water molecules are represented with yellow spheres, and the H-bond interactions of waters are shown as yellow dashed lines. Loop A and loop B were indicated with arrows.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm301032j&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/medium/jm-2012-01032j_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Structure comparison of SIRT3 protein in complex with AceCS2-K<sub>ac</sub> peptide (RSGK<sub>ac</sub>VMR) and ac-RHKK<sub>ac</sub>-AMC peptide. (A) The structure alignment of SIRT3 in complex with two peptides. The “SIRT3/ac-RHKK<sub>ac</sub>-AMC” structure is colored in green with the peptide in stick representation and carbon atoms colored in purple. The “SIRT3/AceCS2-K<sub>ac</sub>” structure is colored in slate with the carbon atoms of the peptide colored in orange. In the loop region (SIRT3–159–167), no density map was observed in the “SIRT3/ac-RHKK<sub>ac</sub>-AMC” structure. Loop A and loop B on the ALY binding site were indicated with arrows (red arrows for “SIRT3/AceCS2-K<sub>ac</sub>” and orange arrows for “SIRT3/ac-RHKK<sub>ac</sub>-AMC”). (B) An enlarged view of the binding site in the structure of “SIRT3/ac-RHKK<sub>ac</sub>-AMC”. The peptide forms four H-bonds with the residues on the small domain. (C) An enlarged view of the binding site in the structure of “SIRT3/AceCS2-K<sub>ac</sub>”. The peptide forms four H-bonds with the residues on loop A and three H-bonds with that on loop B. The distances between the acetyl-lysine and loop B of the “SIRT3/AceCS2-K<sub>ac</sub>” structure were shorter than that of the “SIRT3/ac-RHKK<sub>ac</sub>-AMC” structure. (D) A lateral view of the binding site in (B). The protein molecule is in electrostatic surface representation. (E) A lateral view of (D).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm301032j&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/medium/jm-2012-01032j_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. The homology model of SIRT1 in complex with ac-RHKK<sub>ac</sub>-AMC peptide. (A) The overall structure of the complex model. SIRT1 is represented in cartoon and colored in silver. Loop A and loop B were indicated with arrows. (B) The binding site of the complex. The peptide forms four H-bonds with the backbone atoms on loop A.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm301032j&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/medium/jm-2012-01032j_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. The binding modes of SIRT1 activators with “SIRT1/ac-RHKK<sub>ac</sub>-AMC”. (A) A superimposed view of three small compounds that bind to the complex. The fluorophore-labeled peptide is colored in magentas; compounds <b>20b</b>, <b>1</b>, and <b>4</b> are colored in cyan, yellow, and green, respectively. (B) An enlarged view of <b>20b</b> binding mode. The nitrogen atom on the linker forms hydrogen bond with the peptide. The <i>p</i>-hydroxyl on the 3,5-dibromo-2,4-dihydroxyphenyl fragment H-bonds with the amino group on the side chain of Gln 345, while the <i>o</i>-hydroxyl H-bonds with the linker between the substrate and the coumarin. (C) An enlarged view of <b>4</b> binding mode. (D) An enlarged view of <b>1</b> binding mode. The two hydroxyls H-bond with the sides chains of Arg 446 and His of the substrate, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm301032j&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/medium/jm-2012-01032j_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Stability test of compound <b>20b</b>. “Control” (squares) represents the amount of <b>20b</b> in fresh cell medium. “Sample” (circles) represents the total amount of <b>20b</b> in cultured medium and cells. “In cells” (triangles) represents the amount of <b>20b</b> in cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm301032j&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/medium/jm-2012-01032j_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Western blot analysis of compounds <b>1</b>, <b>20b</b>, and <b>20g</b> on MCF-7 cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-3/jm301032j/production/images/large/jm-2012-01032j_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm301032j&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i94">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05407" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05407" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 60 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Landry, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sutton, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tafrov, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heller, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stebbins, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pillus, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sternglanz, R.</span><span> </span><span class="NLM_article-title">The silencing protein SIR2 and its homologs are NAD-dependent protein deacetylases</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">97</span><span class="NLM_x">, </span> <span class="NLM_fpage">5807</span><span class="NLM_x">–</span> <span class="NLM_lpage">5811</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2000&pages=5807-5811&author=J.+Landryauthor=A.+Suttonauthor=S.+T.+Tafrovauthor=R.+C.+Hellerauthor=J.+Stebbinsauthor=L.+Pillusauthor=R.+Sternglanz&title=The+silencing+protein+SIR2+and+its+homologs+are+NAD-dependent+protein+deacetylases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLandry%26aufirst%3DJ.%26aulast%3DSutton%26aufirst%3DA.%26aulast%3DTafrov%26aufirst%3DS.%2BT.%26aulast%3DHeller%26aufirst%3DR.%2BC.%26aulast%3DStebbins%26aufirst%3DJ.%26aulast%3DPillus%26aufirst%3DL.%26aulast%3DSternglanz%26aufirst%3DR.%26atitle%3DThe%2520silencing%2520protein%2520SIR2%2520and%2520its%2520homologs%2520are%2520NAD-dependent%2520protein%2520deacetylases%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2000%26volume%3D97%26spage%3D5807%26epage%3D5811" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">North, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdin, E.</span><span> </span><span class="NLM_article-title">Sirtuins: Sir2-related NAD-dependent protein deacetylases</span> <span class="citation_source-journal">Genome Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">224</span><span class="NLM_x">–</span> <span class="NLM_lpage">236</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2004&pages=224-236&author=B.+J.+Northauthor=E.+Verdin&title=Sirtuins%3A+Sir2-related+NAD-dependent+protein+deacetylases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DNorth%26aufirst%3DB.%2BJ.%26aulast%3DVerdin%26aufirst%3DE.%26atitle%3DSirtuins%253A%2520Sir2-related%2520NAD-dependent%2520protein%2520deacetylases%26jtitle%3DGenome%2520Biol.%26date%3D2004%26volume%3D5%26spage%3D224%26epage%3D236" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Vaquero, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scher, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Erdjument-Bromage, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tempst, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinberg, D.</span><span> </span><span class="NLM_article-title">Human SirT1 interacts with histone H1 and promotes formation of facultative heterochromatin</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">93</span><span class="NLM_x">–</span> <span class="NLM_lpage">105</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1016%2Fj.molcel.2004.08.031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm301032j&amp;key=15469825" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BD2cXptVKqtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2004&pages=93-105&author=A.+Vaqueroauthor=M.+Scherauthor=D.+H.+Leeauthor=H.+Erdjument-Bromageauthor=P.+Tempstauthor=D.+Reinberg&title=Human+SirT1+interacts+with+histone+H1+and+promotes+formation+of+facultative+heterochromatin"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Human SirT1 interacts with histone H1 and promotes formation of facultative heterochromatin</span></div><div class="casAuthors">Vaquero, Alejandro; Scher, Michael; Lee, Donghoon; Erdjument-Bromage, Hediye; Tempst, Paul; Reinberg, Danny</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">93-105</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">We characterized human SirT1, one of the human homologs of the budding yeast Sir2p, an NAD+-dependent histone deacetylase involved in establishing repressive chromatin and increased life span.  SirT1 deacetylates histone polypeptides with a preference for histone H4 lysine 16 (H4-K16Ac) and H3 lysine 9 (H3-K9Ac) in vitro.  RNAi-mediated decreased expression of SirT1 in human cells causes hyperacetylation of H4-K16 and H3-K9 in vivo.  SirT1 interacts with and deacetylates histone H1 at lysine 26.  Using an inducible system directing expression of SirT1 fused to the Gal4-DNA binding domain and a Gal4-reporter integrated in euchromatin, Gal4-SirT1 expression resulted in the deacetylation of H4-K16 and H3-K9, recruitment of H1 within the promoter vicinity, drastically reduced reporter expression, and loss of H3-K79 methylation, a mark restricting silenced chromatin.  We propose a model for SirT1-mediated heterochromatin formation that includes deacetylation of histone tails, recruitment and deacetylation of histone H1, and spreading of hypomethylated H3-K79 with resultant silencing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcmKK5RHZ4nrVg90H21EOLACvtfcHk0liMmjSrLXF_dQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXptVKqtrw%253D&md5=d342a46aac48b767ef497e34104750e5</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2004.08.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2004.08.031%26sid%3Dliteratum%253Aachs%26aulast%3DVaquero%26aufirst%3DA.%26aulast%3DScher%26aufirst%3DM.%26aulast%3DLee%26aufirst%3DD.%2BH.%26aulast%3DErdjument-Bromage%26aufirst%3DH.%26aulast%3DTempst%26aufirst%3DP.%26aulast%3DReinberg%26aufirst%3DD.%26atitle%3DHuman%2520SirT1%2520interacts%2520with%2520histone%2520H1%2520and%2520promotes%2520formation%2520of%2520facultative%2520heterochromatin%26jtitle%3DMol.%2520Cell%26date%3D2004%26volume%3D16%26spage%3D93%26epage%3D105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Vaquero, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sternglanz, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinberg, D.</span><span> </span><span class="NLM_article-title">NAD<sup>+</sup>-dependent deacetylation of H4 lysine 16 by class III HDACs</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">5505</span><span class="NLM_x">–</span> <span class="NLM_lpage">5520</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2007&pages=5505-5520&author=A.+Vaqueroauthor=R.+Sternglanzauthor=D.+Reinberg&title=NAD%2B-dependent+deacetylation+of+H4+lysine+16+by+class+III+HDACs"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVaquero%26aufirst%3DA.%26aulast%3DSternglanz%26aufirst%3DR.%26aulast%3DReinberg%26aufirst%3DD.%26atitle%3DNAD%252B-dependent%2520deacetylation%2520of%2520H4%2520lysine%252016%2520by%2520class%2520III%2520HDACs%26jtitle%3DOncogene%26date%3D2007%26volume%3D26%26spage%3D5505%26epage%3D5520" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Picard, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurtev, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Topark-Ngarm, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senawong, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Machado De Oliveira, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leid, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McBurney, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guarente, L.</span><span> </span><span class="NLM_article-title">Sirt1 promotes fat mobilization in white adipocytes by repressing PPAR-gamma</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">429</span><span class="NLM_x">, </span> <span class="NLM_fpage">771</span><span class="NLM_x">–</span> <span class="NLM_lpage">776</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=429&publication_year=2004&pages=771-776&author=F.+Picardauthor=M.+Kurtevauthor=N.+Chungauthor=A.+Topark-Ngarmauthor=T.+Senawongauthor=R.+Machado+De+Oliveiraauthor=M.+Leidauthor=M.+W.+McBurneyauthor=L.+Guarente&title=Sirt1+promotes+fat+mobilization+in+white+adipocytes+by+repressing+PPAR-gamma"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPicard%26aufirst%3DF.%26aulast%3DKurtev%26aufirst%3DM.%26aulast%3DChung%26aufirst%3DN.%26aulast%3DTopark-Ngarm%26aufirst%3DA.%26aulast%3DSenawong%26aufirst%3DT.%26aulast%3DMachado%2BDe%2BOliveira%26aufirst%3DR.%26aulast%3DLeid%26aufirst%3DM.%26aulast%3DMcBurney%26aufirst%3DM.%2BW.%26aulast%3DGuarente%26aufirst%3DL.%26atitle%3DSirt1%2520promotes%2520fat%2520mobilization%2520in%2520white%2520adipocytes%2520by%2520repressing%2520PPAR-gamma%26jtitle%3DNature%26date%3D2004%26volume%3D429%26spage%3D771%26epage%3D776" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Qiao, L. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shao, J. H.</span><span> </span><span class="NLM_article-title">SIRT1 regulates adiponectin gene expression through Foxo1-C/enhancer-binding protein alpha transcriptional complex</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">281</span><span class="NLM_x">, </span> <span class="NLM_fpage">39915</span><span class="NLM_x">–</span> <span class="NLM_lpage">39924</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=39915-39924&author=L.+P.+Qiaoauthor=J.+H.+Shao&title=SIRT1+regulates+adiponectin+gene+expression+through+Foxo1-C%2Fenhancer-binding+protein+alpha+transcriptional+complex"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DQiao%26aufirst%3DL.%2BP.%26aulast%3DShao%26aufirst%3DJ.%2BH.%26atitle%3DSIRT1%2520regulates%2520adiponectin%2520gene%2520expression%2520through%2520Foxo1-C%252Fenhancer-binding%2520protein%2520alpha%2520transcriptional%2520complex%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D39915%26epage%3D39924" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Brunet, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sweeney, L. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sturgill, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chua, K. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greer, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tran, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ross, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mostoslavsky, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohen, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, L. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jedrychowski, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gygi, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sinclair, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alt, F. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenberg, M. E.</span><span> </span><span class="NLM_article-title">Stress-dependent regulation of FOXO transcription factors by the SIRT1 deacetylase</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">303</span><span class="NLM_x">, </span> <span class="NLM_fpage">2011</span><span class="NLM_x">–</span> <span class="NLM_lpage">2015</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=303&publication_year=2004&pages=2011-2015&author=A.+Brunetauthor=L.+B.+Sweeneyauthor=J.+F.+Sturgillauthor=K.+F.+Chuaauthor=P.+L.+Greerauthor=Y.+Linauthor=H.+Tranauthor=S.+E.+Rossauthor=R.+Mostoslavskyauthor=H.+Y.+Cohenauthor=L.+S.+Huauthor=H.+L.+Chengauthor=M.+P.+Jedrychowskiauthor=S.+P.+Gygiauthor=D.+A.+Sinclairauthor=F.+W.+Altauthor=M.+E.+Greenberg&title=Stress-dependent+regulation+of+FOXO+transcription+factors+by+the+SIRT1+deacetylase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBrunet%26aufirst%3DA.%26aulast%3DSweeney%26aufirst%3DL.%2BB.%26aulast%3DSturgill%26aufirst%3DJ.%2BF.%26aulast%3DChua%26aufirst%3DK.%2BF.%26aulast%3DGreer%26aufirst%3DP.%2BL.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DTran%26aufirst%3DH.%26aulast%3DRoss%26aufirst%3DS.%2BE.%26aulast%3DMostoslavsky%26aufirst%3DR.%26aulast%3DCohen%26aufirst%3DH.%2BY.%26aulast%3DHu%26aufirst%3DL.%2BS.%26aulast%3DCheng%26aufirst%3DH.%2BL.%26aulast%3DJedrychowski%26aufirst%3DM.%2BP.%26aulast%3DGygi%26aufirst%3DS.%2BP.%26aulast%3DSinclair%26aufirst%3DD.%2BA.%26aulast%3DAlt%26aufirst%3DF.%2BW.%26aulast%3DGreenberg%26aufirst%3DM.%2BE.%26atitle%3DStress-dependent%2520regulation%2520of%2520FOXO%2520transcription%2520factors%2520by%2520the%2520SIRT1%2520deacetylase%26jtitle%3DScience%26date%3D2004%26volume%3D303%26spage%3D2011%26epage%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Bordone, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Motta, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picard, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robinson, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jhala, U. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Apfeld, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonagh, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lemieux, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McBurney, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szilvasi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Easlon, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guarente, L.</span><span> </span><span class="NLM_article-title">Sirt1 regulates insulin secretion by repressing UCP2 in pancreatic beta cells</span> <span class="citation_source-journal">PLoS Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">210</span><span class="NLM_x">–</span> <span class="NLM_lpage">220</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2006&pages=210-220&author=L.+Bordoneauthor=M.+C.+Mottaauthor=F.+Picardauthor=A.+Robinsonauthor=U.+S.+Jhalaauthor=J.+Apfeldauthor=T.+McDonaghauthor=M.+Lemieuxauthor=M.+McBurneyauthor=A.+Szilvasiauthor=E.+J.+Easlonauthor=S.+J.+Linauthor=L.+Guarente&title=Sirt1+regulates+insulin+secretion+by+repressing+UCP2+in+pancreatic+beta+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBordone%26aufirst%3DL.%26aulast%3DMotta%26aufirst%3DM.%2BC.%26aulast%3DPicard%26aufirst%3DF.%26aulast%3DRobinson%26aufirst%3DA.%26aulast%3DJhala%26aufirst%3DU.%2BS.%26aulast%3DApfeld%26aufirst%3DJ.%26aulast%3DMcDonagh%26aufirst%3DT.%26aulast%3DLemieux%26aufirst%3DM.%26aulast%3DMcBurney%26aufirst%3DM.%26aulast%3DSzilvasi%26aufirst%3DA.%26aulast%3DEaslon%26aufirst%3DE.%2BJ.%26aulast%3DLin%26aufirst%3DS.%2BJ.%26aulast%3DGuarente%26aufirst%3DL.%26atitle%3DSirt1%2520regulates%2520insulin%2520secretion%2520by%2520repressing%2520UCP2%2520in%2520pancreatic%2520beta%2520cells%26jtitle%3DPLoS%2520Biol.%26date%3D2006%26volume%3D4%26spage%3D210%26epage%3D220" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Ramsey, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mills, K. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Satoh, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Imai, S. I.</span><span> </span><span class="NLM_article-title">Age-associated loss of Sirt1-mediated enhancement of glucose-stimulated insulin secretion in beta cell-specific Sirt1-overexpressing (BESTO) mice</span> <span class="citation_source-journal">Aging Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">78</span><span class="NLM_x">–</span> <span class="NLM_lpage">88</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=78-88&author=K.+M.+Ramseyauthor=K.+F.+Millsauthor=A.+Satohauthor=S.+I.+Imai&title=Age-associated+loss+of+Sirt1-mediated+enhancement+of+glucose-stimulated+insulin+secretion+in+beta+cell-specific+Sirt1-overexpressing+%28BESTO%29+mice"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRamsey%26aufirst%3DK.%2BM.%26aulast%3DMills%26aufirst%3DK.%2BF.%26aulast%3DSatoh%26aufirst%3DA.%26aulast%3DImai%26aufirst%3DS.%2BI.%26atitle%3DAge-associated%2520loss%2520of%2520Sirt1-mediated%2520enhancement%2520of%2520glucose-stimulated%2520insulin%2520secretion%2520in%2520beta%2520cell-specific%2520Sirt1-overexpressing%2520%2528BESTO%2529%2520mice%26jtitle%3DAging%2520Cell%26date%3D2008%26volume%3D7%26spage%3D78%26epage%3D88" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Rodgers, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lerin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haas, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gygi, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spiegelman, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Puigserver, P.</span><span> </span><span class="NLM_article-title">Nutrient control of glucose homeostasis through a complex of PGC-1alpha and SIRT1</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">434</span><span class="NLM_x">, </span> <span class="NLM_fpage">113</span><span class="NLM_x">–</span> <span class="NLM_lpage">118</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=434&publication_year=2005&pages=113-118&author=J.+T.+Rodgersauthor=C.+Lerinauthor=W.+Haasauthor=S.+P.+Gygiauthor=B.+M.+Spiegelmanauthor=P.+Puigserver&title=Nutrient+control+of+glucose+homeostasis+through+a+complex+of+PGC-1alpha+and+SIRT1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRodgers%26aufirst%3DJ.%2BT.%26aulast%3DLerin%26aufirst%3DC.%26aulast%3DHaas%26aufirst%3DW.%26aulast%3DGygi%26aufirst%3DS.%2BP.%26aulast%3DSpiegelman%26aufirst%3DB.%2BM.%26aulast%3DPuigserver%26aufirst%3DP.%26atitle%3DNutrient%2520control%2520of%2520glucose%2520homeostasis%2520through%2520a%2520complex%2520of%2520PGC-1alpha%2520and%2520SIRT1%26jtitle%3DNature%26date%3D2005%26volume%3D434%26spage%3D113%26epage%3D118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Lagouge, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Argmann, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerhart-Hines, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meziane, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lerin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daussin, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Messadeq, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milne, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elliott, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geny, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laakso, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Puigserver, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Auwerx, J.</span><span> </span><span class="NLM_article-title">Resveratrol improves mitochondrial function and protects against metabolic disease by activating SIRT1 and PGC-1 alpha</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">127</span><span class="NLM_x">, </span> <span class="NLM_fpage">1109</span><span class="NLM_x">–</span> <span class="NLM_lpage">1122</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2006&pages=1109-1122&author=M.+Lagougeauthor=C.+Argmannauthor=Z.+Gerhart-Hinesauthor=H.+Mezianeauthor=C.+Lerinauthor=F.+Daussinauthor=N.+Messadeqauthor=J.+Milneauthor=P.+Lambertauthor=P.+Elliottauthor=B.+Genyauthor=M.+Laaksoauthor=P.+Puigserverauthor=J.+Auwerx&title=Resveratrol+improves+mitochondrial+function+and+protects+against+metabolic+disease+by+activating+SIRT1+and+PGC-1+alpha"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLagouge%26aufirst%3DM.%26aulast%3DArgmann%26aufirst%3DC.%26aulast%3DGerhart-Hines%26aufirst%3DZ.%26aulast%3DMeziane%26aufirst%3DH.%26aulast%3DLerin%26aufirst%3DC.%26aulast%3DDaussin%26aufirst%3DF.%26aulast%3DMessadeq%26aufirst%3DN.%26aulast%3DMilne%26aufirst%3DJ.%26aulast%3DLambert%26aufirst%3DP.%26aulast%3DElliott%26aufirst%3DP.%26aulast%3DGeny%26aufirst%3DB.%26aulast%3DLaakso%26aufirst%3DM.%26aulast%3DPuigserver%26aufirst%3DP.%26aulast%3DAuwerx%26aufirst%3DJ.%26atitle%3DResveratrol%2520improves%2520mitochondrial%2520function%2520and%2520protects%2520against%2520metabolic%2520disease%2520by%2520activating%2520SIRT1%2520and%2520PGC-1%2520alpha%26jtitle%3DCell%26date%3D2006%26volume%3D127%26spage%3D1109%26epage%3D1122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Blander, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guarente, L.</span><span> </span><span class="NLM_article-title">The sir2 family of protein deacetylases</span> <span class="citation_source-journal">Annu. Rev. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">73</span><span class="NLM_x">, </span> <span class="NLM_fpage">417</span><span class="NLM_x">–</span> <span class="NLM_lpage">435</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1146%2Fannurev.biochem.73.011303.073651" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Fjm301032j&amp;key=15189148" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmslags7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2004&pages=417-435&author=G.+Blanderauthor=L.+Guarente&title=The+sir2+family+of+protein+deacetylases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">The Sir2 family of protein deacetylases</span></div><div class="casAuthors">Blander, Gil; Guarente, Leonard</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">417-435</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  The yeast SIR protein complex has been implicated in transcription silencing and suppression of recombination.  The Sir complex represses transcription at telomeres, mating-type loci, and ribosomal DNA.  Unlike SIR3 and SIR4, the SIR2 gene is highly conserved in organisms ranging from archaea to humans.  Interestingly, Sir2 is active as an NAD+-dependent deacetylase, which is broadly conserved from bacteria to higher eukaryotes.  In this review, we discuss the role of NAD+, the unusual products of the deacetylation reaction, the Sir2 structure, and the Sir2 chem. inhibitors and activators that were recently identified.  We summarize the current knowledge of the Sir2 homologs from different organisms, and finally we discuss the role of Sir2 in caloric restriction and aging.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGOpp_Sk7r0rVg90H21EOLACvtfcHk0ljqakmkb3jrwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmslags7c%253D&md5=0ee832d37d4865477d458ea5ce579c6a</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biochem.73.011303.073651&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biochem.73.011303.073651%26sid%3Dliteratum%253Aachs%26aulast%3DBlander%26aufirst%3DG.%26aulast%3DGuarente%26aufirst%3DL.%26atitle%3DThe%2520sir2%2520family%2520of%2520protein%2520deacetylases%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2004%26volume%3D73%26spage%3D417%26epage%3D435" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Longo, V. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennedy, B. K.</span><span> </span><span class="NLM_article-title">Sirtuins in aging and age-related disease</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">126</span><span class="NLM_x">, </span> <span class="NLM_fpage">257</span><span class="NLM_x">–</span> <span class="NLM_lpage">268</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1016%2Fj.cell.2006.07.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Fjm301032j&amp;key=16873059" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BD28Xot1Ont7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2006&pages=257-268&author=V.+D.+Longoauthor=B.+K.+Kennedy&title=Sirtuins+in+aging+and+age-related+disease"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Sirtuins in aging and age-related disease</span></div><div class="casAuthors">Longo, Valter D.; Kennedy, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">257-268</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Sirtuins have been the focus of intense scrutiny since the discovery of Sir2 as a yeast longevity factor.  Functioning as either deacetylases or ADP ribosylases, Sirtuins are regulated by the cofactor NAD and thus may serve as sensors of the metabolic state of the cell and organism.  Here we examine the roles of Sirtuins in diverse eukaryotic species, with special emphasis on their links to aging and age-related diseases including cancer, diabetes, and neurodegenerative disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVJ6UnhyJ7q7Vg90H21EOLACvtfcHk0ljqakmkb3jrwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xot1Ont7g%253D&md5=a903a23be3b021edb7ed2b29f0f2c3b3</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2006.07.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2006.07.002%26sid%3Dliteratum%253Aachs%26aulast%3DLongo%26aufirst%3DV.%2BD.%26aulast%3DKennedy%26aufirst%3DB.%2BK.%26atitle%3DSirtuins%2520in%2520aging%2520and%2520age-related%2520disease%26jtitle%3DCell%26date%3D2006%26volume%3D126%26spage%3D257%26epage%3D268" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Howitz, K. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bitterman, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohen, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamming, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lavu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wood, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zipkin, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kisielewski, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, L.-L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scherer, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sinclair, D. A.</span><span> </span><span class="NLM_article-title">Small molecule activators of sirtuins extend <i>Saccharomyces cerevisiae</i> lifespan</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">425</span><span class="NLM_x">, </span> <span class="NLM_fpage">191</span><span class="NLM_x">–</span> <span class="NLM_lpage">196</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1038%2Fnature01960" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Fjm301032j&amp;key=12939617" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BD3sXntFCitr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=425&publication_year=2003&pages=191-196&author=K.+T.+Howitzauthor=K.+J.+Bittermanauthor=H.+Y.+Cohenauthor=D.+W.+Lammingauthor=S.+Lavuauthor=J.+G.+Woodauthor=R.+E.+Zipkinauthor=P.+Chungauthor=A.+Kisielewskiauthor=L.-L.+Zhangauthor=B.+Schererauthor=D.+A.+Sinclair&title=Small+molecule+activators+of+sirtuins+extend+Saccharomyces+cerevisiae+lifespan"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule activators of sirtuins extend Saccharomyces cerevisiae lifespan</span></div><div class="casAuthors">Howitz, Konrad T.; Bitterman, Kevin J.; Cohen, Haim Y.; Lamming, Dudley W.; Lavu, Siva; Wood, Jason G.; Zipkin, Robert E.; Chung, Phuong; Kisielewski, Anne; Zhang, Li-Li; Scherer, Brandy; Sinclair, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">425</span>
        (<span class="NLM_cas:issue">6954</span>),
    <span class="NLM_cas:pages">191-196</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In diverse organisms, calorie restriction slows the pace of ageing and increases max. lifespan.  In the budding yeast S. cerevisiae, calorie restriction extends lifespan by increasing the activity of Sir2, a member of the conserved sirtuin family of NAD+-dependent protein deacetylases.  Included in this family are SIR-2.1, a Caenorhabditis elegans enzyme that regulates lifespan, and SIRT1, a human deacetylase that promotes cell survival by neg. regulating the p53 tumor suppressor.  Here we report the discovery of 3 classes of small mols. that activate sirtuins.  We show that the potent activator resveratrol, a polyphenol found in red wine, lowers the Michaelis const. of SIRT1 for both the acetylated substrate and NAD+, and increases cell survival by stimulating SIRT1-dependent deacetylation of p53.  In yeast, resveratrol mimics calorie restriction by stimulating Sir2, increasing DNA stability and extending lifespan by 70%.  We discuss possible evolutionary origins of this phenomenon and suggest new lines of research into the therapeutic use of sirtuin activators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBrU9uI44d2bVg90H21EOLACvtfcHk0lhCsuOW1QxG1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXntFCitr4%253D&md5=28d0ccc7b9b1c25358eebb8662177c96</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnature01960&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature01960%26sid%3Dliteratum%253Aachs%26aulast%3DHowitz%26aufirst%3DK.%2BT.%26aulast%3DBitterman%26aufirst%3DK.%2BJ.%26aulast%3DCohen%26aufirst%3DH.%2BY.%26aulast%3DLamming%26aufirst%3DD.%2BW.%26aulast%3DLavu%26aufirst%3DS.%26aulast%3DWood%26aufirst%3DJ.%2BG.%26aulast%3DZipkin%26aufirst%3DR.%2BE.%26aulast%3DChung%26aufirst%3DP.%26aulast%3DKisielewski%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DL.-L.%26aulast%3DScherer%26aufirst%3DB.%26aulast%3DSinclair%26aufirst%3DD.%2BA.%26atitle%3DSmall%2520molecule%2520activators%2520of%2520sirtuins%2520extend%2520Saccharomyces%2520cerevisiae%2520lifespan%26jtitle%3DNature%26date%3D2003%26volume%3D425%26spage%3D191%26epage%3D196" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Wood, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogina, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lavu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howitz, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Helfand, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tatar, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sinclair, D.</span><span> </span><span class="NLM_article-title">Sirtuin activators mimic caloric restriction and delay ageing in metazoans</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">430</span><span class="NLM_x">, </span> <span class="NLM_fpage">686</span><span class="NLM_x">–</span> <span class="NLM_lpage">689</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=430&publication_year=2004&pages=686-689&author=J.+G.+Woodauthor=B.+Roginaauthor=S.+Lavuauthor=K.+Howitzauthor=S.+L.+Helfandauthor=M.+Tatarauthor=D.+Sinclair&title=Sirtuin+activators+mimic+caloric+restriction+and+delay+ageing+in+metazoans"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWood%26aufirst%3DJ.%2BG.%26aulast%3DRogina%26aufirst%3DB.%26aulast%3DLavu%26aufirst%3DS.%26aulast%3DHowitz%26aufirst%3DK.%26aulast%3DHelfand%26aufirst%3DS.%2BL.%26aulast%3DTatar%26aufirst%3DM.%26aulast%3DSinclair%26aufirst%3DD.%26atitle%3DSirtuin%2520activators%2520mimic%2520caloric%2520restriction%2520and%2520delay%2520ageing%2520in%2520metazoans%26jtitle%3DNature%26date%3D2004%26volume%3D430%26spage%3D686%26epage%3D689" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Bauer, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goupil, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garber, G. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Helfand, S. L.</span><span> </span><span class="NLM_article-title">An accelerated assay for the identification of lifespan-extending interventions in <i>Drosophila melanogaster</i></span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">101</span><span class="NLM_x">, </span> <span class="NLM_fpage">12980</span><span class="NLM_x">–</span> <span class="NLM_lpage">12985</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=12980-12985&author=J.+H.+Bauerauthor=S.+Goupilauthor=G.+B.+Garberauthor=S.+L.+Helfand&title=An+accelerated+assay+for+the+identification+of+lifespan-extending+interventions+in+Drosophila+melanogaster"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBauer%26aufirst%3DJ.%2BH.%26aulast%3DGoupil%26aufirst%3DS.%26aulast%3DGarber%26aufirst%3DG.%2BB.%26aulast%3DHelfand%26aufirst%3DS.%2BL.%26atitle%3DAn%2520accelerated%2520assay%2520for%2520the%2520identification%2520of%2520lifespan-extending%2520interventions%2520in%2520Drosophila%2520melanogaster%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2004%26volume%3D101%26spage%3D12980%26epage%3D12985" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Valenzano, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terzibasi, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Genade, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cattaneo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Domenici, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cellerino, A.</span><span> </span><span class="NLM_article-title">Resveratrol prolongs lifespan and retards the onset of age-related markers in a short-lived vertebrate</span> <span class="citation_source-journal">Curr. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">296</span><span class="NLM_x">–</span> <span class="NLM_lpage">300</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1016%2Fj.cub.2005.12.038" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Fjm301032j&amp;key=16461283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlKltL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2006&pages=296-300&author=D.+R.+Valenzanoauthor=E.+Terzibasiauthor=T.+Genadeauthor=A.+Cattaneoauthor=L.+Domeniciauthor=A.+Cellerino&title=Resveratrol+prolongs+lifespan+and+retards+the+onset+of+age-related+markers+in+a+short-lived+vertebrate"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Resveratrol Prolongs Lifespan and Retards the Onset of Age-Related Markers in a Short-Lived Vertebrate</span></div><div class="casAuthors">Valenzano, Dario R.; Terzibasi, Eva; Genade, Tyrone; Cattaneo, Antonino; Domenici, Luciano; Cellerino, Alessandro</div><div class="citationInfo"><span class="NLM_cas:title">Current Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">296-300</span>CODEN:
                <span class="NLM_cas:coden">CUBLE2</span>;
        ISSN:<span class="NLM_cas:issn">0960-9822</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Resveratrol, a natural phytoalexin found in grapes and red wine, increases longevity in the short-lived invertebrates Caenorhabditis elegans and Drosophila and exerts a variety of biol. effects in vertebrates, including protection from ischemia and neurotoxicity.  Its effects on vertebrate lifespan were not yet known.  The relatively long lifespan of mice, which live at least 2.5 years, is a hurdle for life-long pharmacol. trials.  Here, the authors used the short-lived seasonal fish Nothobranchius furzeri with a max. recorded lifespan of 13 wk in captivity.  Short lifespan in this species is not the result of spontaneous or targeted genetic mutations, but a natural trait correlated with the necessity to breed in an ephemeral habitat and tied with accelerated development and expression of aging biomarkers at a cellular level.  Resveratrol was added to the food starting in early adulthood and caused a dose-dependent increase of median and max. lifespan.  In addn., resveratrol delays the age-dependent decay of locomotor activity and cognitive performances and reduces the expression of neurofibrillary degeneration in the brain.  These results demonstrate that food supplementation with resveratrol promotes longevity and retards the expression of age-dependent traits in a short-lived vertebrate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQmwGN6cM6-rVg90H21EOLACvtfcHk0lhCsuOW1QxG1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlKltL4%253D&md5=522acaa576dae43ba50672e86716c45d</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.cub.2005.12.038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cub.2005.12.038%26sid%3Dliteratum%253Aachs%26aulast%3DValenzano%26aufirst%3DD.%2BR.%26aulast%3DTerzibasi%26aufirst%3DE.%26aulast%3DGenade%26aufirst%3DT.%26aulast%3DCattaneo%26aufirst%3DA.%26aulast%3DDomenici%26aufirst%3DL.%26aulast%3DCellerino%26aufirst%3DA.%26atitle%3DResveratrol%2520prolongs%2520lifespan%2520and%2520retards%2520the%2520onset%2520of%2520age-related%2520markers%2520in%2520a%2520short-lived%2520vertebrate%26jtitle%3DCurr.%2520Biol.%26date%3D2006%26volume%3D16%26spage%3D296%26epage%3D300" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Sun, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ge, X. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, T. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, X. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhai, Q. W.</span><span> </span><span class="NLM_article-title">SIRT1 improves insulin sensitivity under insulin-resistant conditions by repressing PTP1B</span> <span class="citation_source-journal">Cell Metab.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">307</span><span class="NLM_x">–</span> <span class="NLM_lpage">319</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1016%2Fj.cmet.2007.08.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Fjm301032j&amp;key=17908559" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1Sjsr3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=307-319&author=C.+Sunauthor=F.+Zhangauthor=X.+J.+Geauthor=T.+T.+Yanauthor=X.+M.+Chenauthor=X.+L.+Shiauthor=Q.+W.+Zhai&title=SIRT1+improves+insulin+sensitivity+under+insulin-resistant+conditions+by+repressing+PTP1B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">SIRT1 improves insulin sensitivity under insulin-resistant conditions by repressing PTP1B</span></div><div class="casAuthors">Sun, Cheng; Zhang, Fang; Ge, Xinjian; Yan, Tingting; Chen, Xingmiao; Shi, Xianglin; Zhai, Qiwei</div><div class="citationInfo"><span class="NLM_cas:title">Cell Metabolism</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">307-319</span>CODEN:
                <span class="NLM_cas:coden">CMEEB5</span>;
        ISSN:<span class="NLM_cas:issn">1550-4131</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Insulin resistance is often characterized as the most crit. factor contributing to the development of type 2 diabetes.  SIRT1 has been reported to be involved in the processes of glucose metab. and insulin secretion.  However, whether SIRT1 is directly involved in insulin sensitivity is still largely unknown.  Here we show that SIRT1 is downregulated in insulin-resistant cells and tissues and that knockdown or inhibition of SIRT1 induces insulin resistance.  Furthermore, increased expression of SIRT1 improved insulin sensitivity, esp. under insulin-resistant conditions.  Similarly, resveratrol, a SIRT1 activator, enhanced insulin sensitivity in vitro in a SIRT1-dependent manner and attenuated high-fat-diet-induced insulin resistance in vivo at a dose of 2.5 mg/kg/day.  Further studies demonstrated that the effect of SIRT1 on insulin resistance is mediated by repressing PTP1B transcription at the chromatin level.  Taken together, the finding that SIRT1 improves insulin sensitivity has implications toward resolving insulin resistance and type 2 diabetes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3juiZtxzRt7Vg90H21EOLACvtfcHk0liQKOQSe634HQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1Sjsr3L&md5=7b186a99719b49e9ecaf689007824c92</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2007.08.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2007.08.014%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DF.%26aulast%3DGe%26aufirst%3DX.%2BJ.%26aulast%3DYan%26aufirst%3DT.%2BT.%26aulast%3DChen%26aufirst%3DX.%2BM.%26aulast%3DShi%26aufirst%3DX.%2BL.%26aulast%3DZhai%26aufirst%3DQ.%2BW.%26atitle%3DSIRT1%2520improves%2520insulin%2520sensitivity%2520under%2520insulin-resistant%2520conditions%2520by%2520repressing%2520PTP1B%26jtitle%3DCell%2520Metab.%26date%3D2007%26volume%3D6%26spage%3D307%26epage%3D319" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Csiszar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labinskyy, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pinto, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ballabh, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Losonczy, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pearson, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Cabo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pacher, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ungvari, Z.</span><span> </span><span class="NLM_article-title">Resveratrol induces mitochondrial biogenesis in endothelial cells</span> <span class="citation_source-journal">Am. J. Physiol.: Heart Circ. Physiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">297</span><span class="NLM_x">, </span> <span class="NLM_fpage">13</span><span class="NLM_x">–</span> <span class="NLM_lpage">20</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1152%2Fajpheart.00368.2009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Fjm301032j&amp;key=19429820" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BD1MXovVKisL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=297&publication_year=2009&pages=13-20&author=A.+Csiszarauthor=N.+Labinskyyauthor=J.+T.+Pintoauthor=P.+Ballabhauthor=H.+Zhangauthor=G.+Losonczyauthor=K.+Pearsonauthor=R.+de+Caboauthor=P.+Pacherauthor=C.+Zhangauthor=Z.+Ungvari&title=Resveratrol+induces+mitochondrial+biogenesis+in+endothelial+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Resveratrol induces mitochondrial biogenesis in endothelial cells</span></div><div class="casAuthors">Csiszar, Anna; Labinskyy, Nazar; Pinto, John T.; Ballabh, Praveen; Zhang, Hanrui; Losonczy, Gyorgy; Pearson, Kevin; de Cabo, Rafael; Pacher, Pal; Zhang, Cuihua; Ungvari, Zoltan</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Physiology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">297</span>
        (<span class="NLM_cas:issue">1, Pt. 2</span>),
    <span class="NLM_cas:pages">H13-H20</span>CODEN:
                <span class="NLM_cas:coden">AJPHAP</span>;
        ISSN:<span class="NLM_cas:issn">0002-9513</span>.
    
            (<span class="NLM_cas:orgname">American Physiological Society</span>)
        </div><div class="casAbstract">Pathways that regulate mitochondrial biogenesis are potential therapeutic targets for the amelioration of endothelial dysfunction and vascular disease.  Resveratrol was shown to impact mitochondrial function in skeletal muscle and the liver, but its role in mitochondrial biogenesis in endothelial cells remains poorly defined.  The present study detd. whether resveratrol induces mitochondrial biogenesis in cultured human coronary arterial endothelial cells (CAECs).  In CAECs resveratrol increased mitochondrial mass and mitochondrial DNA content, upregulated protein expression of electron transport chain constituents, and induced mitochondrial biogenesis factors (proliferator-activated receptor-coactivator-1α, nuclear respiratory factor-1, mitochondrial transcription factor A).  Sirtuin 1 (SIRT1) was induced, and endothelial nitric oxide (NO) synthase (eNOS) was upregulated in a SIRT1-dependent manner.  Knockdown of SIRT1 (small interfering RNA) or inhibition of NO synthesis prevented resveratrol-induced mitochondrial biogenesis.  In aortas of type 2 diabetic (db/db) mice impaired mitochondrial biogenesis was normalized by chronic resveratrol treatment, showing the in vivo relevance of our findings.  Resveratrol increases mitochondrial content in endothelial cells via activating SIRT1.  We propose that SIRT1, via a pathway that involves the upregulation of eNOS, induces mitochondrial biogenesis.  Resveratrol induced mitochondrial biogenesis in the aortas of type 2 diabetic mice, suggesting the potential for new treatment approaches targeting endothelial mitochondria in metabolic diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSYfYD3QRgsLVg90H21EOLACvtfcHk0liQKOQSe634HQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXovVKisL0%253D&md5=342eac88c1cebf78b818521688f46224</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1152%2Fajpheart.00368.2009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fajpheart.00368.2009%26sid%3Dliteratum%253Aachs%26aulast%3DCsiszar%26aufirst%3DA.%26aulast%3DLabinskyy%26aufirst%3DN.%26aulast%3DPinto%26aufirst%3DJ.%2BT.%26aulast%3DBallabh%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DLosonczy%26aufirst%3DG.%26aulast%3DPearson%26aufirst%3DK.%26aulast%3Dde%2BCabo%26aufirst%3DR.%26aulast%3DPacher%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DUngvari%26aufirst%3DZ.%26atitle%3DResveratrol%2520induces%2520mitochondrial%2520biogenesis%2520in%2520endothelial%2520cells%26jtitle%3DAm.%2520J.%2520Physiol.%253A%2520Heart%2520Circ.%2520Physiol.%26date%3D2009%26volume%3D297%26spage%3D13%26epage%3D20" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Nayagam, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tan, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poulsen, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goh, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ni, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Entzeroth, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stunkel, W.</span><span> </span><span class="NLM_article-title">SIRT1 modulating compounds from high-throughput screening as anti-inflammatory and insulin-sensitizing agents</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">959</span><span class="NLM_x">–</span> <span class="NLM_lpage">967</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1177%2F1087057106294710" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Fjm301032j&amp;key=17099246" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BD2sXovFOlug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2006&pages=959-967&author=V.+M.+Nayagamauthor=X.+Wangauthor=Y.+C.+Tanauthor=A.+Poulsenauthor=K.+C.+Gohauthor=T.+Ngauthor=H.+Wangauthor=H.+Y.+Songauthor=B.+Niauthor=M.+Entzerothauthor=W.+Stunkel&title=SIRT1+modulating+compounds+from+high-throughput+screening+as+anti-inflammatory+and+insulin-sensitizing+agents"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">SIRT1 modulating compounds from high-throughput screening as anti-inflammatory and insulin-sensitizing agents</span></div><div class="casAuthors">Nayagam, Vasantha M.; Wang, Xukun; Tan, Yong Cheng; Poulsen, Anders; Goh, Kee Chuan; Ng, Tony; Wang, Haishan; Song, Hong Yan; Ni, Binhui; Entzeroth, Michael; Stunkel, Walter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">959-967</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The NAD (NAD+)-dependent protein deacetylase SIRT1 has been linked to fatty acid metab. via suppression of peroxysome proliferator-activated receptor gamma (PPAR-γ) and to inflammatory processes by deacetylating the transcription factor NF-κB.  First, modulation of SIRT1 activity affects lipid accumulation in adipocytes, which has an impact on the etiol. of a variety of human metabolic diseases such as obesity and insulin-resistant diabetes.  Second, activation of SIRT1 suppresses inflammation via regulation of cytokine expression.  Using high-throughput screening, the authors identified compds. with SIRT1 activating and inhibiting potential.  The biol. activity of these SIRT1-modulating compds. was confirmed in cell-based assays using mouse adipocytes, as well as human THP-1 monocytes.  SIRT1 activators were found to be potent lipolytic agents, reducing the overall lipid content of fully differentiated NIH L1 adipocytes.  In addn., the same compds. have anti-inflammatory properties, as became evident by the redn. of the proinflammatory cytokine tumor necrosis factor-alpha (TNF-α).  In contrast, a SIRT1 inhibitory compd. showed a stimulatory activity on the differentiation of adipocytes, a feature often linked to insulin sensitization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5D3Kk0JnfubVg90H21EOLACvtfcHk0li6PswHQTh7SQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXovFOlug%253D%253D&md5=e20b0e3c3b84e8002a000c4d483bbc91</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1177%2F1087057106294710&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057106294710%26sid%3Dliteratum%253Aachs%26aulast%3DNayagam%26aufirst%3DV.%2BM.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DTan%26aufirst%3DY.%2BC.%26aulast%3DPoulsen%26aufirst%3DA.%26aulast%3DGoh%26aufirst%3DK.%2BC.%26aulast%3DNg%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DSong%26aufirst%3DH.%2BY.%26aulast%3DNi%26aufirst%3DB.%26aulast%3DEntzeroth%26aufirst%3DM.%26aulast%3DStunkel%26aufirst%3DW.%26atitle%3DSIRT1%2520modulating%2520compounds%2520from%2520high-throughput%2520screening%2520as%2520anti-inflammatory%2520and%2520insulin-sensitizing%2520agents%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2006%26volume%3D11%26spage%3D959%26epage%3D967" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Mai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valente, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meade, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carafa, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tardugno, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nebbioso, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galmozzi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitro, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Fabiani, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altucci, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kazantsev, A.</span><span> </span><span class="NLM_article-title">Study of 1,4-dihydropyridine structural scaffold: discovery of novel sirtuin activators and inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">5496</span><span class="NLM_x">–</span> <span class="NLM_lpage">5504</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm9008289" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=5496-5504&author=A.+Maiauthor=S.+Valenteauthor=S.+Meadeauthor=V.+Carafaauthor=M.+Tardugnoauthor=A.+Nebbiosoauthor=A.+Galmozziauthor=N.+Mitroauthor=E.+De+Fabianiauthor=L.+Altucciauthor=A.+Kazantsev&title=Study+of+1%2C4-dihydropyridine+structural+scaffold%3A+discovery+of+novel+sirtuin+activators+and+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjm9008289&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9008289%26sid%3Dliteratum%253Aachs%26aulast%3DMai%26aufirst%3DA.%26aulast%3DValente%26aufirst%3DS.%26aulast%3DMeade%26aufirst%3DS.%26aulast%3DCarafa%26aufirst%3DV.%26aulast%3DTardugno%26aufirst%3DM.%26aulast%3DNebbioso%26aufirst%3DA.%26aulast%3DGalmozzi%26aufirst%3DA.%26aulast%3DMitro%26aufirst%3DN.%26aulast%3DDe%2BFabiani%26aufirst%3DE.%26aulast%3DAltucci%26aufirst%3DL.%26aulast%3DKazantsev%26aufirst%3DA.%26atitle%3DStudy%2520of%25201%252C4-dihydropyridine%2520structural%2520scaffold%253A%2520discovery%2520of%2520novel%2520sirtuin%2520activators%2520and%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D5496%26epage%3D5504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Milne, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schenk, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carney, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gagne, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boss, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perni, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vu, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bemis, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Disch, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, P. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nunes, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lynch, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galonek, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Israelian, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choy, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iffland, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lavu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Medvedik, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sinclair, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olefsky, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jirousek, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elliott, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westphal, C. H.</span><span> </span><span class="NLM_article-title">Small molecule activators of SIRT1 as therapeutics for the treatment of type 2 diabetes</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">450</span><span class="NLM_x">, </span> <span class="NLM_fpage">712</span><span class="NLM_x">–</span> <span class="NLM_lpage">716</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1038%2Fnature06261" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Fjm301032j&amp;key=18046409" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlKmtbrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=450&publication_year=2007&pages=712-716&author=J.+C.+Milneauthor=P.+D.+Lambertauthor=S.+Schenkauthor=D.+P.+Carneyauthor=J.+J.+Smithauthor=D.+J.+Gagneauthor=L.+Jinauthor=O.+Bossauthor=R.+B.+Perniauthor=C.+B.+Vuauthor=J.+E.+Bemisauthor=R.+Xieauthor=J.+S.+Dischauthor=P.+Y.+Ngauthor=J.+J.+Nunesauthor=A.+V.+Lynchauthor=H.+Yangauthor=H.+Galonekauthor=K.+Israelianauthor=W.+Choyauthor=A.+Ifflandauthor=S.+Lavuauthor=O.+Medvedikauthor=D.+A.+Sinclairauthor=J.+M.+Olefskyauthor=M.+R.+Jirousekauthor=P.+J.+Elliottauthor=C.+H.+Westphal&title=Small+molecule+activators+of+SIRT1+as+therapeutics+for+the+treatment+of+type+2+diabetes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule activators of SIRT1 as therapeutics for the treatment of type 2 diabetes</span></div><div class="casAuthors">Milne, Jill C.; Lambert, Philip D.; Schenk, Simon; Carney, David P.; Smith, Jesse J.; Gagne, David J.; Jin, Lei; Boss, Olivier; Perni, Robert B.; Vu, Chi B.; Bemis, Jean E.; Xie, Roger; Disch, Jeremy S.; Ng, Pui Yee; Nunes, Joseph J.; Lynch, Amy V.; Yang, Hongying; Galonek, Heidi; Israelian, Kristine; Choy, Wendy; Iffland, Andre; Lavu, Siva; Medvedik, Oliver; Sinclair, David A.; Olefsky, Jerrold M.; Jirousek, Michael R.; Elliott, Peter J.; Westphal, Christoph H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">450</span>
        (<span class="NLM_cas:issue">7170</span>),
    <span class="NLM_cas:pages">712-716</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Calorie restriction extends lifespan and produces a metabolic profile desirable for treating diseases of aging such as type 2 diabetes.  SIRT1, an NAD+-dependent deacetylase, is a principal modulator of pathways downstream of calorie restriction that produce beneficial effects on glucose homeostasis and insulin sensitivity.  Resveratrol, a polyphenolic SIRT1 activator, mimics the antiaging effects of calorie restriction in lower organisms and in mice fed a high-fat diet ameliorates insulin resistance, increases mitochondrial content, and prolongs survival.  Here the authors describe the identification and characterization of small mol. activators of SIRT1 that are structurally unrelated to, and 1000-fold more potent than, resveratrol.  These compds. bind to the SIRT1 enzyme-peptide substrate complex at an allosteric site amino-terminal to the catalytic domain and lower the Michaelis const. for acetylated substrates.  In diet-induced obese and genetically obese mice, these compds. improve insulin sensitivity, lower plasma glucose, and increase mitochondrial capacity.  In Zucker fa/fa rats, hyperinsulinemic-euglycemic clamp studies demonstrate that SIRT1 activators improve whole-body glucose homeostasis and insulin sensitivity in adipose tissue, skeletal muscle and liver.  Thus, SIRT1 activation is a promising new therapeutic approach for treating diseases of aging such as type 2 diabetes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtZUMUqn50MbVg90H21EOLACvtfcHk0lhtQgzfHiXrfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlKmtbrM&md5=8343b182d3596cbd8eb175c8b9cdcace</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnature06261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature06261%26sid%3Dliteratum%253Aachs%26aulast%3DMilne%26aufirst%3DJ.%2BC.%26aulast%3DLambert%26aufirst%3DP.%2BD.%26aulast%3DSchenk%26aufirst%3DS.%26aulast%3DCarney%26aufirst%3DD.%2BP.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DGagne%26aufirst%3DD.%2BJ.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DBoss%26aufirst%3DO.%26aulast%3DPerni%26aufirst%3DR.%2BB.%26aulast%3DVu%26aufirst%3DC.%2BB.%26aulast%3DBemis%26aufirst%3DJ.%2BE.%26aulast%3DXie%26aufirst%3DR.%26aulast%3DDisch%26aufirst%3DJ.%2BS.%26aulast%3DNg%26aufirst%3DP.%2BY.%26aulast%3DNunes%26aufirst%3DJ.%2BJ.%26aulast%3DLynch%26aufirst%3DA.%2BV.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DGalonek%26aufirst%3DH.%26aulast%3DIsraelian%26aufirst%3DK.%26aulast%3DChoy%26aufirst%3DW.%26aulast%3DIffland%26aufirst%3DA.%26aulast%3DLavu%26aufirst%3DS.%26aulast%3DMedvedik%26aufirst%3DO.%26aulast%3DSinclair%26aufirst%3DD.%2BA.%26aulast%3DOlefsky%26aufirst%3DJ.%2BM.%26aulast%3DJirousek%26aufirst%3DM.%2BR.%26aulast%3DElliott%26aufirst%3DP.%2BJ.%26aulast%3DWestphal%26aufirst%3DC.%2BH.%26atitle%3DSmall%2520molecule%2520activators%2520of%2520SIRT1%2520as%2520therapeutics%2520for%2520the%2520treatment%2520of%2520type%25202%2520diabetes%26jtitle%3DNature%26date%3D2007%26volume%3D450%26spage%3D712%26epage%3D716" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Vu, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bemis, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Disch, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, P. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nunes, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milne, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carney, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lynch, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lavu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gagne, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jirousek, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schenk, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olefsky, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perni, R. B.</span><span> </span><span class="NLM_article-title">Discovery of imidazo[1,2-<i>b</i>]thiazole derivatives as novel SIRT1 activators</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">1275</span><span class="NLM_x">–</span> <span class="NLM_lpage">1283</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm8012954" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=1275-1283&author=C.+B.+Vuauthor=J.+E.+Bemisauthor=J.+S.+Dischauthor=P.+Y.+Ngauthor=J.+J.+Nunesauthor=J.+C.+Milneauthor=D.+P.+Carneyauthor=A.+V.+Lynchauthor=J.+J.+Smithauthor=S.+Lavuauthor=P.+D.+Lambertauthor=D.+J.+Gagneauthor=M.+R.+Jirousekauthor=S.+Schenkauthor=J.+M.+Olefskyauthor=R.+B.+Perni&title=Discovery+of+imidazo%5B1%2C2-b%5Dthiazole+derivatives+as+novel+SIRT1+activators"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fjm8012954&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8012954%26sid%3Dliteratum%253Aachs%26aulast%3DVu%26aufirst%3DC.%2BB.%26aulast%3DBemis%26aufirst%3DJ.%2BE.%26aulast%3DDisch%26aufirst%3DJ.%2BS.%26aulast%3DNg%26aufirst%3DP.%2BY.%26aulast%3DNunes%26aufirst%3DJ.%2BJ.%26aulast%3DMilne%26aufirst%3DJ.%2BC.%26aulast%3DCarney%26aufirst%3DD.%2BP.%26aulast%3DLynch%26aufirst%3DA.%2BV.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DLavu%26aufirst%3DS.%26aulast%3DLambert%26aufirst%3DP.%2BD.%26aulast%3DGagne%26aufirst%3DD.%2BJ.%26aulast%3DJirousek%26aufirst%3DM.%2BR.%26aulast%3DSchenk%26aufirst%3DS.%26aulast%3DOlefsky%26aufirst%3DJ.%2BM.%26aulast%3DPerni%26aufirst%3DR.%2BB.%26atitle%3DDiscovery%2520of%2520imidazo%255B1%252C2-b%255Dthiazole%2520derivatives%2520as%2520novel%2520SIRT1%2520activators%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D1275%26epage%3D1283" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Bemis, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vu, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nunes, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, P. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Disch, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milne, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carney, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lynch, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lavu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iffland, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jirousek, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perni, R. B.</span><span> </span><span class="NLM_article-title">Discovery of oxazolo[4,5-<i>b</i>]pyridines and related heterocyclic analogs as novel SIRT1 activators</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">2350</span><span class="NLM_x">–</span> <span class="NLM_lpage">2353</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=2350-2353&author=J.+E.+Bemisauthor=C.+B.+Vuauthor=R.+Xieauthor=J.+J.+Nunesauthor=P.+Y.+Ngauthor=J.+S.+Dischauthor=J.+C.+Milneauthor=D.+P.+Carneyauthor=A.+V.+Lynchauthor=L.+Jinauthor=J.+J.+Smithauthor=S.+Lavuauthor=A.+Ifflandauthor=M.+R.+Jirousekauthor=R.+B.+Perni&title=Discovery+of+oxazolo%5B4%2C5-b%5Dpyridines+and+related+heterocyclic+analogs+as+novel+SIRT1+activators"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBemis%26aufirst%3DJ.%2BE.%26aulast%3DVu%26aufirst%3DC.%2BB.%26aulast%3DXie%26aufirst%3DR.%26aulast%3DNunes%26aufirst%3DJ.%2BJ.%26aulast%3DNg%26aufirst%3DP.%2BY.%26aulast%3DDisch%26aufirst%3DJ.%2BS.%26aulast%3DMilne%26aufirst%3DJ.%2BC.%26aulast%3DCarney%26aufirst%3DD.%2BP.%26aulast%3DLynch%26aufirst%3DA.%2BV.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DLavu%26aufirst%3DS.%26aulast%3DIffland%26aufirst%3DA.%26aulast%3DJirousek%26aufirst%3DM.%2BR.%26aulast%3DPerni%26aufirst%3DR.%2BB.%26atitle%3DDiscovery%2520of%2520oxazolo%255B4%252C5-b%255Dpyridines%2520and%2520related%2520heterocyclic%2520analogs%2520as%2520novel%2520SIRT1%2520activators%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D2350%26epage%3D2353" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Kaeberlein, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonagh, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heltweg, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hixon, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westman, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caldwell, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Napper, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curtis, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DiStefano, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fields, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedalov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennedy, B. K.</span><span> </span><span class="NLM_article-title">Substrate-specific activation of sirtuins by resveratrol</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">280</span><span class="NLM_x">, </span> <span class="NLM_fpage">17038</span><span class="NLM_x">–</span> <span class="NLM_lpage">17045</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2005&pages=17038-17045&author=M.+Kaeberleinauthor=T.+McDonaghauthor=B.+Heltwegauthor=J.+Hixonauthor=E.+A.+Westmanauthor=S.+D.+Caldwellauthor=A.+Napperauthor=R.+Curtisauthor=P.+S.+DiStefanoauthor=S.+Fieldsauthor=A.+Bedalovauthor=B.+K.+Kennedy&title=Substrate-specific+activation+of+sirtuins+by+resveratrol"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKaeberlein%26aufirst%3DM.%26aulast%3DMcDonagh%26aufirst%3DT.%26aulast%3DHeltweg%26aufirst%3DB.%26aulast%3DHixon%26aufirst%3DJ.%26aulast%3DWestman%26aufirst%3DE.%2BA.%26aulast%3DCaldwell%26aufirst%3DS.%2BD.%26aulast%3DNapper%26aufirst%3DA.%26aulast%3DCurtis%26aufirst%3DR.%26aulast%3DDiStefano%26aufirst%3DP.%2BS.%26aulast%3DFields%26aufirst%3DS.%26aulast%3DBedalov%26aufirst%3DA.%26aulast%3DKennedy%26aufirst%3DB.%2BK.%26atitle%3DSubstrate-specific%2520activation%2520of%2520sirtuins%2520by%2520resveratrol%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2005%26volume%3D280%26spage%3D17038%26epage%3D17045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Pacholec, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bleasdale, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chrunyk, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cunningham, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flynn, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garofalo, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffith, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffor, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loulakis, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pabst, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qiu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stockman, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thanabal, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varghese, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ward, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Withka, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahn, K.</span><span> </span><span class="NLM_article-title">SRT1720, SRT2183, SRT1460, and resveratrol are not direct activators of SIRT1</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">285</span><span class="NLM_x">, </span> <span class="NLM_fpage">8340</span><span class="NLM_x">–</span> <span class="NLM_lpage">8351</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=8340-8351&author=M.+Pacholecauthor=J.+E.+Bleasdaleauthor=B.+Chrunykauthor=D.+Cunninghamauthor=D.+Flynnauthor=R.+S.+Garofaloauthor=D.+Griffithauthor=M.+Grifforauthor=P.+Loulakisauthor=B.+Pabstauthor=X.+Qiuauthor=B.+Stockmanauthor=V.+Thanabalauthor=A.+Vargheseauthor=J.+Wardauthor=J.+Withkaauthor=K.+Ahn&title=SRT1720%2C+SRT2183%2C+SRT1460%2C+and+resveratrol+are+not+direct+activators+of+SIRT1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPacholec%26aufirst%3DM.%26aulast%3DBleasdale%26aufirst%3DJ.%2BE.%26aulast%3DChrunyk%26aufirst%3DB.%26aulast%3DCunningham%26aufirst%3DD.%26aulast%3DFlynn%26aufirst%3DD.%26aulast%3DGarofalo%26aufirst%3DR.%2BS.%26aulast%3DGriffith%26aufirst%3DD.%26aulast%3DGriffor%26aufirst%3DM.%26aulast%3DLoulakis%26aufirst%3DP.%26aulast%3DPabst%26aufirst%3DB.%26aulast%3DQiu%26aufirst%3DX.%26aulast%3DStockman%26aufirst%3DB.%26aulast%3DThanabal%26aufirst%3DV.%26aulast%3DVarghese%26aufirst%3DA.%26aulast%3DWard%26aufirst%3DJ.%26aulast%3DWithka%26aufirst%3DJ.%26aulast%3DAhn%26aufirst%3DK.%26atitle%3DSRT1720%252C%2520SRT2183%252C%2520SRT1460%252C%2520and%2520resveratrol%2520are%2520not%2520direct%2520activators%2520of%2520SIRT1%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D8340%26epage%3D8351" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Dai, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kustigian, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carney, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Case, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Considine, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubbard, B. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perni, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riera, T. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szczepankiewicz, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vlasuk, G. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stein, R. L.</span><span> </span><span class="NLM_article-title">SIRT1 activation by small molecules: kinetic and biophysical evidence for direct interaction of enzyme and activator</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">285</span><span class="NLM_x">, </span> <span class="NLM_fpage">32695</span><span class="NLM_x">–</span> <span class="NLM_lpage">32703</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=32695-32703&author=H.+Daiauthor=L.+Kustigianauthor=D.+Carneyauthor=A.+Caseauthor=T.+Considineauthor=B.+P.+Hubbardauthor=R.+B.+Perniauthor=T.+V.+Rieraauthor=B.+Szczepankiewiczauthor=G.+P.+Vlasukauthor=R.+L.+Stein&title=SIRT1+activation+by+small+molecules%3A+kinetic+and+biophysical+evidence+for+direct+interaction+of+enzyme+and+activator"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DH.%26aulast%3DKustigian%26aufirst%3DL.%26aulast%3DCarney%26aufirst%3DD.%26aulast%3DCase%26aufirst%3DA.%26aulast%3DConsidine%26aufirst%3DT.%26aulast%3DHubbard%26aufirst%3DB.%2BP.%26aulast%3DPerni%26aufirst%3DR.%2BB.%26aulast%3DRiera%26aufirst%3DT.%2BV.%26aulast%3DSzczepankiewicz%26aufirst%3DB.%26aulast%3DVlasuk%26aufirst%3DG.%2BP.%26aulast%3DStein%26aufirst%3DR.%2BL.%26atitle%3DSIRT1%2520activation%2520by%2520small%2520molecules%253A%2520kinetic%2520and%2520biophysical%2520evidence%2520for%2520direct%2520interaction%2520of%2520enzyme%2520and%2520activator%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D32695%26epage%3D32703" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Bass, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weinkove, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Houthoofd, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gems, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Partridge, L.</span><span> </span><span class="NLM_article-title">Effects of resveratrol on lifespan in <i>Drosophila melanogaster</i> and <i>Caenorhabditis elegans</i></span> <span class="citation_source-journal">Mech. Ageing Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">546</span><span class="NLM_x">–</span> <span class="NLM_lpage">552</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=546-552&author=T.+M.+Bassauthor=D.+Weinkoveauthor=K.+Houthoofdauthor=D.+Gemsauthor=L.+Partridge&title=Effects+of+resveratrol+on+lifespan+in+Drosophila+melanogaster+and+Caenorhabditis+elegans"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBass%26aufirst%3DT.%2BM.%26aulast%3DWeinkove%26aufirst%3DD.%26aulast%3DHouthoofd%26aufirst%3DK.%26aulast%3DGems%26aufirst%3DD.%26aulast%3DPartridge%26aufirst%3DL.%26atitle%3DEffects%2520of%2520resveratrol%2520on%2520lifespan%2520in%2520Drosophila%2520melanogaster%2520and%2520Caenorhabditis%2520elegans%26jtitle%3DMech.%2520Ageing%2520Dev.%26date%3D2007%26volume%3D128%26spage%3D546%26epage%3D552" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Feige, J. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lagouge, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Canto, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strehle, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Houten, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milne, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mataki, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elliott, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Auwerx, J.</span><span> </span><span class="NLM_article-title">Specific SIRT1 activation mimics low energy levels and protects against diet-induced metabolic disorders by enhancing fat oxidation</span> <span class="citation_source-journal">Cell Metab.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">347</span><span class="NLM_x">–</span> <span class="NLM_lpage">358</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1016%2Fj.cmet.2008.08.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Fjm301032j&amp;key=19046567" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlOgur7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2008&pages=347-358&author=J.+N.+Feigeauthor=M.+Lagougeauthor=C.+Cantoauthor=A.+Strehleauthor=S.+M.+Houtenauthor=J.+C.+Milneauthor=P.+D.+Lambertauthor=C.+Matakiauthor=P.+J.+Elliottauthor=J.+Auwerx&title=Specific+SIRT1+activation+mimics+low+energy+levels+and+protects+against+diet-induced+metabolic+disorders+by+enhancing+fat+oxidation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Specific SIRT1 activation mimics low energy levels and protects against diet-induced metabolic disorders by enhancing fat oxidation</span></div><div class="casAuthors">Feige, Jerome N.; Lagouge, Marie; Canto, Carles; Strehle, Axelle; Houten, Sander M.; Milne, Jill C.; Lambert, Philip D.; Mataki, Chikage; Elliott, Peter J.; Auwerx, Johan</div><div class="citationInfo"><span class="NLM_cas:title">Cell Metabolism</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">347-358</span>CODEN:
                <span class="NLM_cas:coden">CMEEB5</span>;
        ISSN:<span class="NLM_cas:issn">1550-4131</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The NAD+-dependent deacetylase SIRT1 controls metabolic processes in response to low nutrient availability.  We report the metabolic phenotype of mice treated with SRT1720, a specific and potent synthetic activator of SIRT1 that is devoid of direct action on AMPK.  SRT1720 administration robustly enhances endurance running performance and strongly protects from diet-induced obesity and insulin resistance by enhancing oxidative metab. in skeletal muscle, liver, and brown adipose tissue.  These metabolic effects of SRT1720 are mediated by the induction of a genetic network controlling fatty acid oxidn. through a multifaceted mechanism that involves the direct deacetylation of PGC-1α, FOXO1, and p53 and the indirect stimulation of AMPK signaling through a global metabolic adaptation mimicking low energy levels.  Combined with our previous work on resveratrol, the current study further validates SIRT1 as a target for the treatment of metabolic disorders and characterizes the mechanisms underlying the therapeutic potential of SIRT1 activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo90My01jFYurVg90H21EOLACvtfcHk0ljyPV_QTmiePg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlOgur7O&md5=516fd7d72637dccb9a31a211c406ae0d</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2008.08.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2008.08.017%26sid%3Dliteratum%253Aachs%26aulast%3DFeige%26aufirst%3DJ.%2BN.%26aulast%3DLagouge%26aufirst%3DM.%26aulast%3DCanto%26aufirst%3DC.%26aulast%3DStrehle%26aufirst%3DA.%26aulast%3DHouten%26aufirst%3DS.%2BM.%26aulast%3DMilne%26aufirst%3DJ.%2BC.%26aulast%3DLambert%26aufirst%3DP.%2BD.%26aulast%3DMataki%26aufirst%3DC.%26aulast%3DElliott%26aufirst%3DP.%2BJ.%26aulast%3DAuwerx%26aufirst%3DJ.%26atitle%3DSpecific%2520SIRT1%2520activation%2520mimics%2520low%2520energy%2520levels%2520and%2520protects%2520against%2520diet-induced%2520metabolic%2520disorders%2520by%2520enhancing%2520fat%2520oxidation%26jtitle%3DCell%2520Metab.%26date%3D2008%26volume%3D8%26spage%3D347%26epage%3D358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kenney, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gagne, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frushour, B. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ladd, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galonek, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Israelian, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Razvadauskaite, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lynch, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carney, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lavu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iffland, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elliott, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elliston, K. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jirousek, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milne, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boss, O.</span><span> </span><span class="NLM_article-title">Small molecule activators of SIRT1 replicate signaling pathways triggered by calorie restriction in vivo</span> <span class="citation_source-journal">BMC Syst. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">31</span><span class="NLM_x">–</span> <span class="NLM_lpage">45</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1186%2F1752-0509-3-31" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Fjm301032j&amp;key=19284563" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A280%3ADC%252BD1M3jtVantQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2009&pages=31-45&author=J.+J.+Smithauthor=R.+D.+Kenneyauthor=D.+J.+Gagneauthor=B.+P.+Frushourauthor=W.+Laddauthor=H.+L.+Galonekauthor=K.+Israelianauthor=J.+Songauthor=G.+Razvadauskaiteauthor=A.+V.+Lynchauthor=D.+P.+Carneyauthor=R.+J.+Johnsonauthor=S.+Lavuauthor=A.+Ifflandauthor=P.+J.+Elliottauthor=P.+D.+Lambertauthor=K.+O.+Ellistonauthor=M.+R.+Jirousekauthor=J.+C.+Milneauthor=O.+Boss&title=Small+molecule+activators+of+SIRT1+replicate+signaling+pathways+triggered+by+calorie+restriction+in+vivo"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule activators of SIRT1 replicate signaling pathways triggered by calorie restriction in vivo</span></div><div class="casAuthors">Smith Jesse J; Kenney Renee Deehan; Gagne David J; Frushour Brian P; Ladd William; Galonek Heidi L; Israelian Kristine; Song Jeffrey; Razvadauskaite Giedre; Lynch Amy V; Carney David P; Johnson Robin J; Lavu Siva; Iffland Andre; Elliott Peter J; Lambert Philip D; Elliston Keith O; Jirousek Michael R; Milne Jill C; Boss Olivier</div><div class="citationInfo"><span class="NLM_cas:title">BMC systems biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">31</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Calorie restriction (CR) produces a number of health benefits and ameliorates diseases of aging such as type 2 diabetes.  The components of the pathways downstream of CR may provide intervention points for developing therapeutics for treating diseases of aging.  The NAD+-dependent protein deacetylase SIRT1 has been implicated as one of the key downstream regulators of CR in yeast, rodents, and humans.  Small molecule activators of SIRT1 have been identified that exhibit efficacy in animal models of diseases typically associated with aging including type 2 diabetes.  To identify molecular processes induced in the liver of mice treated with two structurally distinct SIRT1 activators, SIRT501 (formulated resveratrol) and SRT1720, for three days, we utilized a systems biology approach and applied Causal Network Modeling (CNM) on gene expression data to elucidate downstream effects of SIRT1 activation.  RESULTS:  Here we demonstrate that SIRT1 activators recapitulate many of the molecular events downstream of CR in vivo, such as enhancing mitochondrial biogenesis, improving metabolic signaling pathways, and blunting pro-inflammatory pathways in mice fed a high fat, high calorie diet.  CONCLUSION:  CNM of gene expression data from mice treated with SRT501 or SRT1720 in combination with supporting in vitro and in vivo data demonstrates that SRT501 and SRT1720 produce a signaling profile that mirrors CR, improves glucose and insulin homeostasis, and acts via SIRT1 activation in vivo.  Taken together these results are encouraging regarding the use of small molecule activators of SIRT1 for therapeutic intervention into type 2 diabetes, a strategy which is currently being investigated in multiple clinical trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQDlBeNme77BrqxQ2mb9Pu8fW6udTcc2eaLUeUNIB0qXLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1M3jtVantQ%253D%253D&md5=042e7aec752f41e84c51b4adbda7925a</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1186%2F1752-0509-3-31&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1752-0509-3-31%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DKenney%26aufirst%3DR.%2BD.%26aulast%3DGagne%26aufirst%3DD.%2BJ.%26aulast%3DFrushour%26aufirst%3DB.%2BP.%26aulast%3DLadd%26aufirst%3DW.%26aulast%3DGalonek%26aufirst%3DH.%2BL.%26aulast%3DIsraelian%26aufirst%3DK.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DRazvadauskaite%26aufirst%3DG.%26aulast%3DLynch%26aufirst%3DA.%2BV.%26aulast%3DCarney%26aufirst%3DD.%2BP.%26aulast%3DJohnson%26aufirst%3DR.%2BJ.%26aulast%3DLavu%26aufirst%3DS.%26aulast%3DIffland%26aufirst%3DA.%26aulast%3DElliott%26aufirst%3DP.%2BJ.%26aulast%3DLambert%26aufirst%3DP.%2BD.%26aulast%3DElliston%26aufirst%3DK.%2BO.%26aulast%3DJirousek%26aufirst%3DM.%2BR.%26aulast%3DMilne%26aufirst%3DJ.%2BC.%26aulast%3DBoss%26aufirst%3DO.%26atitle%3DSmall%2520molecule%2520activators%2520of%2520SIRT1%2520replicate%2520signaling%2520pathways%2520triggered%2520by%2520calorie%2520restriction%2520in%2520vivo%26jtitle%3DBMC%2520Syst.%2520Biol.%26date%3D2009%26volume%3D3%26spage%3D31%26epage%3D45" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">He, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, H.</span><span> </span><span class="NLM_article-title">Discovering potent inhibitors against the beta-hydroxyacyl-acyl carrier protein dehydratase (FabZ) of <i>Helicobacter pylori</i>: structure-based design, synthesis, bioassay, and crystal structure determination</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">2465</span><span class="NLM_x">–</span> <span class="NLM_lpage">2481</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm8015602" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=2465-2481&author=L.+Heauthor=L.+Zhangauthor=X.+Liuauthor=X.+Liauthor=M.+Zhengauthor=H.+Liauthor=K.+Yuauthor=K.+Chenauthor=X.+Shenauthor=H.+Jiangauthor=H.+Liu&title=Discovering+potent+inhibitors+against+the+beta-hydroxyacyl-acyl+carrier+protein+dehydratase+%28FabZ%29+of+Helicobacter+pylori%3A+structure-based+design%2C+synthesis%2C+bioassay%2C+and+crystal+structure+determination"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm8015602&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8015602%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DYu%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DShen%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DH.%26atitle%3DDiscovering%2520potent%2520inhibitors%2520against%2520the%2520beta-hydroxyacyl-acyl%2520carrier%2520protein%2520dehydratase%2520%2528FabZ%2529%2520of%2520Helicobacter%2520pylori%253A%2520structure-based%2520design%252C%2520synthesis%252C%2520bioassay%252C%2520and%2520crystal%2520structure%2520determination%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D2465%26epage%3D2481" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Boehlow, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harburn, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spilling, C. D.</span><span> </span><span class="NLM_article-title">Approaches to the synthesis of some tyrosine-derived marine sponge metabolites: synthesis of verongamine and purealidin N</span> <span class="citation_source-journal">J. Org. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">3111</span><span class="NLM_x">–</span> <span class="NLM_lpage">3118</span></span><div class="citationLinks">[<a href="/doi/10.1021/jo010015v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BD3MXit1GjtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2001&pages=3111-3118&author=T.+R.+Boehlowauthor=J.+J.+Harburnauthor=C.+D.+Spilling&title=Approaches+to+the+synthesis+of+some+tyrosine-derived+marine+sponge+metabolites%3A+synthesis+of+verongamine+and+purealidin+N"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Approaches to the Synthesis of Some Tyrosine-Derived Marine Sponge Metabolites: Synthesis of Verongamine and Purealidin N</span></div><div class="casAuthors">Boehlow, Todd R.; Harburn, J. Jonathan; Spilling, Christopher D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3111-3118</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The oxidn. of tyrosine Et ester with Na2WO4/H2O2 in ethanol, dimethyldioxirane in acetone, or methyltrioxorhenium/H2O2 in EtOH gave the corresponding tyrosine oxime in high yield.  Controlled bromination of the arom. ring gave the monobromo oxime, the dibromo oxime, or the spiroisoxazoline depending upon reaction conditions.  Synthesis of the known metabolite verongamine (I, R1,R2 = H) was achieved by oxidn. of O-Me bromotyrosine Me ester and amidation of the resulting oxime ester with histamine.  The mono- and di-bromotyrosine oxime derivs. were further transformed into naturally occurring nitriles by base hydrolysis of the ester and acid-catalyzed decarboxylation.  Amidation of the oxime ester II with histamine, followed by deprotection of the MOM ether gave the first synthesis of purealidin N (I, R1 = Br, R2 = OH).  Oxidative spirocyclization of the phenolic oxime ester II with a polymer-bound iodosyl diacetate gave the spiroisoxazoline III and represents a formal synthesis of aerothionin IV (n = 4), homoaerothionin IV (n = 5), and aerophobin-1 V.  Synthetic purealidin N and three related oximes were tested against three sensitive cancer sell lines, lung NCI-H460, breast MCF7, and CNS SF-268.  At concns. of 10-4 M, all of the compds. were essentially inactive, showing inhibition of cell growth of between 30% and 93% of the control.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyxckqC9akc7Vg90H21EOLACvtfcHk0ljNrpoJkUvBIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXit1GjtLc%253D&md5=457f8306cdc9b36097e1736c9196f8d9</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjo010015v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo010015v%26sid%3Dliteratum%253Aachs%26aulast%3DBoehlow%26aufirst%3DT.%2BR.%26aulast%3DHarburn%26aufirst%3DJ.%2BJ.%26aulast%3DSpilling%26aufirst%3DC.%2BD.%26atitle%3DApproaches%2520to%2520the%2520synthesis%2520of%2520some%2520tyrosine-derived%2520marine%2520sponge%2520metabolites%253A%2520synthesis%2520of%2520verongamine%2520and%2520purealidin%2520N%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2001%26volume%3D66%26spage%3D3111%26epage%3D3118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Napper, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hixon, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonagh, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keavey, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pons, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barker, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yau, W. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amouzegh, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flegg, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamelin, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kates, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Navia, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saunders, J. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DiStefano, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curtis, R.</span><span> </span><span class="NLM_article-title">Discovery of indoles as potent and selective inhibitors of the deacetylase SIRT1</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">8045</span><span class="NLM_x">–</span> <span class="NLM_lpage">8054</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm050522v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFyqsb3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=8045-8054&author=A.+D.+Napperauthor=J.+Hixonauthor=T.+McDonaghauthor=K.+Keaveyauthor=J.+F.+Ponsauthor=J.+Barkerauthor=W.+T.+Yauauthor=P.+Amouzeghauthor=A.+Fleggauthor=E.+Hamelinauthor=R.+J.+Thomasauthor=M.+Katesauthor=S.+Jonesauthor=M.+A.+Naviaauthor=J.+O.+Saundersauthor=P.+S.+DiStefanoauthor=R.+Curtis&title=Discovery+of+indoles+as+potent+and+selective+inhibitors+of+the+deacetylase+SIRT1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of indoles as potent and selective inhibitors of the deacetylase SIRT1</span></div><div class="casAuthors">Napper, Andrew D.; Hixon, Jeffrey; McDonagh, Thomas; Keavey, Kenneth; Pons, Jean-Francois; Barker, Jonathan; Yau, Wei Tsung; Amouzegh, Patricia; Flegg, Adam; Hamelin, Estelle; Thomas, Russell J.; Kates, Michael; Jones, Stephen; Navia, Manuel A.; Saunders, Jeffrey O.; DiStefano, Peter S.; Curtis, Rory</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">8045-8054</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">High-throughput screening against the human sirtuin SIRT1 led to the discovery of a series of indoles as potent inhibitors that are selective for SIRT1 over other deacetylases and NAD-processing enzymes.  The most potent compds. described herein inhibit SIRT1 with IC50 values of 60-100 nM, representing a 500-fold improvement over previously reported SIRT inhibitors.  Prepn. of enantiomerically pure indole derivs. allowed for their characterization in vitro and in vivo.  Kinetic analyses suggest that these inhibitors bind after the release of nicotinamide from the enzyme and prevent the release of deacetylated peptide and O-acetyl-ADP-ribose, the products of enzyme-catalyzed deacetylation.  These SIRT1 inhibitors are low mol. wt., cell-permeable, orally bioavailable, and metabolically stable.  These compds. provide chem. tools to study the biol. of SIRT1 and to explore therapeutic uses for SIRT1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpbdBlnlCEArVg90H21EOLACvtfcHk0lg7CxJo8Kiyxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFyqsb3N&md5=5ba92c43b735dbdf383a988a64a06f80</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjm050522v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050522v%26sid%3Dliteratum%253Aachs%26aulast%3DNapper%26aufirst%3DA.%2BD.%26aulast%3DHixon%26aufirst%3DJ.%26aulast%3DMcDonagh%26aufirst%3DT.%26aulast%3DKeavey%26aufirst%3DK.%26aulast%3DPons%26aufirst%3DJ.%2BF.%26aulast%3DBarker%26aufirst%3DJ.%26aulast%3DYau%26aufirst%3DW.%2BT.%26aulast%3DAmouzegh%26aufirst%3DP.%26aulast%3DFlegg%26aufirst%3DA.%26aulast%3DHamelin%26aufirst%3DE.%26aulast%3DThomas%26aufirst%3DR.%2BJ.%26aulast%3DKates%26aufirst%3DM.%26aulast%3DJones%26aufirst%3DS.%26aulast%3DNavia%26aufirst%3DM.%2BA.%26aulast%3DSaunders%26aufirst%3DJ.%2BO.%26aulast%3DDiStefano%26aufirst%3DP.%2BS.%26aulast%3DCurtis%26aufirst%3DR.%26atitle%3DDiscovery%2520of%2520indoles%2520as%2520potent%2520and%2520selective%2520inhibitors%2520of%2520the%2520deacetylase%2520SIRT1%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D8045%26epage%3D8054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Sanders, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jackson, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marmorstein, R.</span><span> </span><span class="NLM_article-title">Structural basis for sirtuin function: what we know and what we don’t</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Proteins Proteomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">1804</span><span class="NLM_x">, </span> <span class="NLM_fpage">1604</span><span class="NLM_x">–</span> <span class="NLM_lpage">1616</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1016%2Fj.bbapap.2009.09.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Fjm301032j&amp;key=19766737" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntlaltLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1804&publication_year=2010&pages=1604-1616&author=B.+D.+Sandersauthor=B.+Jacksonauthor=R.+Marmorstein&title=Structural+basis+for+sirtuin+function%3A+what+we+know+and+what+we+don%E2%80%99t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for sirtuin function: What we know and what we don't</span></div><div class="casAuthors">Sanders, Brandi D.; Jackson, Brittany; Marmorstein, Ronen</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Proteins and Proteomics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1804</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1604-1616</span>CODEN:
                <span class="NLM_cas:coden">BBAPBW</span>;
        ISSN:<span class="NLM_cas:issn">1570-9639</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B. V.</span>)
        </div><div class="casAbstract">A review.  Sirtuin (silent information regulator 2) proteins are NAD-dependent deacetylases that are implicated in diverse biol. processes including DNA regulation, metab., and longevity.  Homologs of the prototypic yeast Sir2 protein have been identified in all 3 kingdoms of life, and while bacteria and archaea typically contain 1-2 sirtuins, eukaryotic organisms contain multiple members.  Sirtuins are regulated in part by the cellular concns. of the noncompetitive inhibitor, nicotinamide, and several synthetic modulators of these enzymes have been identified.  The x-ray crystal structures of several sirtuin proteins in various liganded forms have been detd.  This wealth of structural information, together with related biochem. studies, have provided important insights into the catalytic mechanism, substrate specificity, and inhibitory mechanism of sirtuin proteins.  Implications for future structural studies to address outstanding questions in the field are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrF0mq_taWVALVg90H21EOLACvtfcHk0lg7CxJo8Kiyxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntlaltLc%253D&md5=7ddae49f9e55b39b4ad0abdb00d68596</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.bbapap.2009.09.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbapap.2009.09.009%26sid%3Dliteratum%253Aachs%26aulast%3DSanders%26aufirst%3DB.%2BD.%26aulast%3DJackson%26aufirst%3DB.%26aulast%3DMarmorstein%26aufirst%3DR.%26atitle%3DStructural%2520basis%2520for%2520sirtuin%2520function%253A%2520what%2520we%2520know%2520and%2520what%2520we%2520don%25E2%2580%2599t%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Proteins%2520Proteomics%26date%3D2010%26volume%3D1804%26spage%3D1604%26epage%3D1616" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Otwinowski, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minor, W.</span><span> </span><span class="NLM_article-title">Processing of X-ray diffraction data collected in oscillation mode</span> <span class="citation_source-journal">Macromol. Crystallogr., Part A</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">276</span><span class="NLM_x">, </span> <span class="NLM_fpage">307</span><span class="NLM_x">–</span> <span class="NLM_lpage">326</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=307-326&author=Z.+Otwinowskiauthor=W.+Minor&title=Processing+of+X-ray+diffraction+data+collected+in+oscillation+mode"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Processing of x-ray diffraction data collected in oscillation mode</span></div><div class="casAuthors">Otwinowski, Zbyszek; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part A</span>),
    <span class="NLM_cas:pages">307-326</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Macromol. crystallog. is an iterative process.  Rarely do the first crystals provide all the necessary data to solve the biol. problem being studied.  Each step benefits from experience learned in previous steps.  To monitor the progress, the HKL package provides 2 tools: (1) statistics, both weighted (χ2) and unweighted (R-merge), are provided, and the Bayesian reasoning and multicomponent error model facilitates obtaining the proper error ests. and (2) visualization of the process plays a double role by helping the operator to confirm that the process of data redn., including the resulting statistics, is correct, and allowing one to evaluate problems for which there are no good statistical criteria.  Visualization also provides confidence that the point of diminishing returns in data collection and redn. has been reached.  At that point, the effort should be directed to solving the structure.  The methods presented here have been applied to solve a large variety of problems, from inorg. mols. with 5 Å unit cell to rotavirus of 700 Å diam. crystd. in 700 × 1000 × 1400 Å cell.  Overall quality of the method was tested by many researchers by successful application of the programs to MAD structure detns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA3fvBOHqhhrVg90H21EOLACvtfcHk0lgPLevEJuZv0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D&md5=c9536971d4e32cc35352c40fb9368131</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3DProcessing%2520of%2520X-ray%2520diffraction%2520data%2520collected%2520in%2520oscillation%2520mode%26jtitle%3DMacromol.%2520Crystallogr.%252C%2520Part%2520A%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span> </span><span class="NLM_article-title">Collaborative computational project, number 4. The CCP4 suite: programs for protein crystallography</span>.  <span class="citation_source-journal">Acta. Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">760</span><span class="NLM_x">–</span> <span class="NLM_lpage">763</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Collaborative+computational+project%2C+number+4.+The+CCP4+suite%3A+programs+for+protein+crystallography.+Acta.+Crystallogr.%2C+Sect.+D%3A+Biol.+Crystallogr.+1994%2C+50%2C+760%E2%80%93763."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26atitle%3DCollaborative%2520computational%2520project%252C%2520number%25204.%2520The%2520CCP4%2520suite%253A%2520programs%2520for%2520protein%2520crystallography%26jtitle%3DActa.%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26spage%3D760%26epage%3D763" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Sauve, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Celic, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Avalos, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boeke, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schramm, V. L.</span><span> </span><span class="NLM_article-title">Chemistry of gene silencing: the mechanism of NAD<sup>+</sup>-dependent deacetylation reactions</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">15456</span><span class="NLM_x">–</span> <span class="NLM_lpage">15463</span></span><div class="citationLinks">[<a href="/doi/10.1021/bi011858j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2001&pages=15456-15463&author=A.+A.+Sauveauthor=I.+Celicauthor=J.+Avalosauthor=H.+Dengauthor=J.+D.+Boekeauthor=V.+L.+Schramm&title=Chemistry+of+gene+silencing%3A+the+mechanism+of+NAD%2B-dependent+deacetylation+reactions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fbi011858j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi011858j%26sid%3Dliteratum%253Aachs%26aulast%3DSauve%26aufirst%3DA.%2BA.%26aulast%3DCelic%26aufirst%3DI.%26aulast%3DAvalos%26aufirst%3DJ.%26aulast%3DDeng%26aufirst%3DH.%26aulast%3DBoeke%26aufirst%3DJ.%2BD.%26aulast%3DSchramm%26aufirst%3DV.%2BL.%26atitle%3DChemistry%2520of%2520gene%2520silencing%253A%2520the%2520mechanism%2520of%2520NAD%252B-dependent%2520deacetylation%2520reactions%26jtitle%3DBiochemistry%26date%3D2001%26volume%3D40%26spage%3D15456%26epage%3D15463" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Smith, B. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Denu, J. M.</span><span> </span><span class="NLM_article-title">Sir2 protein deacetylases: evidence for chemical intermediates and functions of a conserved histidine</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">272</span><span class="NLM_x">–</span> <span class="NLM_lpage">282</span></span><div class="citationLinks">[<a href="/doi/10.1021/bi052014t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2006&pages=272-282&author=B.+C.+Smithauthor=J.+M.+Denu&title=Sir2+protein+deacetylases%3A+evidence+for+chemical+intermediates+and+functions+of+a+conserved+histidine"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fbi052014t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi052014t%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DB.%2BC.%26aulast%3DDenu%26aufirst%3DJ.%2BM.%26atitle%3DSir2%2520protein%2520deacetylases%253A%2520evidence%2520for%2520chemical%2520intermediates%2520and%2520functions%2520of%2520a%2520conserved%2520histidine%26jtitle%3DBiochemistry%26date%3D2006%26volume%3D45%26spage%3D272%26epage%3D282" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Jackson, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppenheimer, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Denu, J. M.</span><span> </span><span class="NLM_article-title">Mechanism of nicotinamide inhibition and transglycosidation by Sir2 histone/protein deacetylases</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">278</span><span class="NLM_x">, </span> <span class="NLM_fpage">50985</span><span class="NLM_x">–</span> <span class="NLM_lpage">50998</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1074%2Fjbc.M306552200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Fjm301032j&amp;key=14522996" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BD3sXpslOhs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=50985-50998&author=M.+D.+Jacksonauthor=M.+T.+Schmidtauthor=N.+J.+Oppenheimerauthor=J.+M.+Denu&title=Mechanism+of+nicotinamide+inhibition+and+transglycosidation+by+Sir2+histone%2Fprotein+deacetylases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism of Nicotinamide Inhibition and Transglycosidation by Sir2 Histone/Protein Deacetylases</span></div><div class="casAuthors">Jackson, Michael D.; Schmidt, Manning T.; Oppenheimer, Norman J.; Denu, John M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">278</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">50985-50998</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Silent information regulator 2 (Sir2) enzymes catalyze NAD+-dependent protein/histone deacetylation, where the acetyl group from the lysine ε-amino group is transferred to the ADP-ribose moiety of NAD+, producing nicotinamide and the novel metabolite O-acetyl-ADP-ribose.  Sir2 proteins have been shown to regulate gene silencing, metabolic enzymes, and life span.  Recently, nicotinamide has been implicated as a direct neg. regulator of cellular Sir2 function; however, the mechanism of nicotinamide inhibition was not established.  Sir2 enzymes are multifunctional in that the deacetylase reaction involves the cleavage of the nicotinamide-ribosyl, cleavage of an amide bond, and transfer of the acetyl group ultimately to the 2'-ribose hydroxyl of ADP-ribose.  Here the authors demonstrate that nicotinamide inhibition is the result of nicotinamide intercepting an ADP-ribosyl-enzyme-acetyl peptide intermediate with regeneration of NAD+ (transglycosidation).  The cellular implications are discussed.  A variety of 3-substituted pyridines was found to be substrates for enzyme-catalyzed transglycosidation.  A Broensted plot of the data yielded a slope of +0.98, consistent with the development of a nearly full pos. charge in the transition state, and with basicity of the attacking nucleophile as a strong predictor of reactivity.  NAD+ analogs including β-2'-deoxy-2'-fluororibo-NAD+ and a His-to-Ala mutant were used to probe the mechanism of nicotinamide-ribosyl cleavage and acetyl group transfer.  The authors demonstrate that nicotinamide-ribosyl cleavage is distinct from acetyl group transfer to the 2'-OH ribose.  The obsd. enzyme-catalyzed formation of a labile 1'-acetylated-ADP-fluororibose intermediate using β-2'-deoxy-2'-fluororibo-NAD+ supports a mechanism where, after nicotinamide-ribosyl cleavage, the carbonyl oxygen of acetylated substrate attacks the C-1' ribose to form an initial iminium adduct.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqf6GUpwcEPWbVg90H21EOLACvtfcHk0lgAriG_qNKn3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXpslOhs78%253D&md5=89ebcb43d075c4738e261ee22e4b2893</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M306552200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M306552200%26sid%3Dliteratum%253Aachs%26aulast%3DJackson%26aufirst%3DM.%2BD.%26aulast%3DSchmidt%26aufirst%3DM.%2BT.%26aulast%3DOppenheimer%26aufirst%3DN.%2BJ.%26aulast%3DDenu%26aufirst%3DJ.%2BM.%26atitle%3DMechanism%2520of%2520nicotinamide%2520inhibition%2520and%2520transglycosidation%2520by%2520Sir2%2520histone%252Fprotein%2520deacetylases%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26spage%3D50985%26epage%3D50998" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cai, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dai, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choy, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bemis, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jirousek, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milne, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westphal, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perni, R. B.</span><span> </span><span class="NLM_article-title">Crystal structures of human SIRT3 displaying substrate-induced conformational changes</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">284</span><span class="NLM_x">, </span> <span class="NLM_fpage">24394</span><span class="NLM_x">–</span> <span class="NLM_lpage">24405</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=24394-24405&author=L.+Jinauthor=W.+Weiauthor=Y.+Jiangauthor=H.+Pengauthor=J.+Caiauthor=C.+Maoauthor=H.+Daiauthor=W.+Choyauthor=J.+E.+Bemisauthor=M.+R.+Jirousekauthor=J.+C.+Milneauthor=C.+H.+Westphalauthor=R.+B.+Perni&title=Crystal+structures+of+human+SIRT3+displaying+substrate-induced+conformational+changes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DL.%26aulast%3DWei%26aufirst%3DW.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DPeng%26aufirst%3DH.%26aulast%3DCai%26aufirst%3DJ.%26aulast%3DMao%26aufirst%3DC.%26aulast%3DDai%26aufirst%3DH.%26aulast%3DChoy%26aufirst%3DW.%26aulast%3DBemis%26aufirst%3DJ.%2BE.%26aulast%3DJirousek%26aufirst%3DM.%2BR.%26aulast%3DMilne%26aufirst%3DJ.%2BC.%26aulast%3DWestphal%26aufirst%3DC.%2BH.%26aulast%3DPerni%26aufirst%3DR.%2BB.%26atitle%3DCrystal%2520structures%2520of%2520human%2520SIRT3%2520displaying%2520substrate-induced%2520conformational%2520changes%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D24394%26epage%3D24405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Avalos, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bever, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolberger, C.</span><span> </span><span class="NLM_article-title">Mechanism of sirtuin inhibition by nicotinamide: altering the NAD<sup>+</sup> cosubstrate specificity of a Sir2 enzyme</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">855</span><span class="NLM_x">–</span> <span class="NLM_lpage">868</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2005&pages=855-868&author=J.+L.+Avalosauthor=K.+M.+Beverauthor=C.+Wolberger&title=Mechanism+of+sirtuin+inhibition+by+nicotinamide%3A+altering+the+NAD%2B+cosubstrate+specificity+of+a+Sir2+enzyme"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAvalos%26aufirst%3DJ.%2BL.%26aulast%3DBever%26aufirst%3DK.%2BM.%26aulast%3DWolberger%26aufirst%3DC.%26atitle%3DMechanism%2520of%2520sirtuin%2520inhibition%2520by%2520nicotinamide%253A%2520altering%2520the%2520NAD%252B%2520cosubstrate%2520specificity%2520of%2520a%2520Sir2%2520enzyme%26jtitle%3DMol.%2520Cell%26date%3D2005%26volume%3D17%26spage%3D855%26epage%3D868" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Min, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Landry, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sternglanz, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, R. M.</span><span> </span><span class="NLM_article-title">Crystal structure of a SIR2 homolog–NAD complex</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x">, </span> <span class="NLM_fpage">269</span><span class="NLM_x">–</span> <span class="NLM_lpage">279</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2001&pages=269-279&author=J.+Minauthor=J.+Landryauthor=R.+Sternglanzauthor=R.+M.+Xu&title=Crystal+structure+of+a+SIR2+homolog%E2%80%93NAD+complex"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DLandry%26aufirst%3DJ.%26aulast%3DSternglanz%26aufirst%3DR.%26aulast%3DXu%26aufirst%3DR.%2BM.%26atitle%3DCrystal%2520structure%2520of%2520a%2520SIR2%2520homolog%25E2%2580%2593NAD%2520complex%26jtitle%3DCell%26date%3D2001%26volume%3D105%26spage%3D269%26epage%3D279" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Avalos, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boeke, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolberger, C.</span><span> </span><span class="NLM_article-title">Structural basis for the mechanism and regulation of Sir2 enzymes</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">639</span><span class="NLM_x">–</span> <span class="NLM_lpage">648</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2004&pages=639-648&author=J.+L.+Avalosauthor=J.+D.+Boekeauthor=C.+Wolberger&title=Structural+basis+for+the+mechanism+and+regulation+of+Sir2+enzymes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAvalos%26aufirst%3DJ.%2BL.%26aulast%3DBoeke%26aufirst%3DJ.%2BD.%26aulast%3DWolberger%26aufirst%3DC.%26atitle%3DStructural%2520basis%2520for%2520the%2520mechanism%2520and%2520regulation%2520of%2520Sir2%2520enzymes%26jtitle%3DMol.%2520Cell%26date%3D2004%26volume%3D13%26spage%3D639%26epage%3D648" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Borra, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, B. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Denu, J. M.</span><span> </span><span class="NLM_article-title">Mechanism of human SIRT1 activation by resveratrol</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">280</span><span class="NLM_x">, </span> <span class="NLM_fpage">17187</span><span class="NLM_x">–</span> <span class="NLM_lpage">17195</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1074%2Fjbc.M501250200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Fjm301032j&amp;key=15749705" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjsFOgtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2005&pages=17187-17195&author=M.+T.+Borraauthor=B.+C.+Smithauthor=J.+M.+Denu&title=Mechanism+of+human+SIRT1+activation+by+resveratrol"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism of Human SIRT1 Activation by Resveratrol</span></div><div class="casAuthors">Borra, Margie T.; Smith, Brian C.; Denu, John M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">280</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">17187-17195</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The NAD+-dependent protein deacetylase family, Sir2 (or sirtuins), is important for many cellular processes including gene silencing, regulation of p53, fatty acid metab., cell cycle regulation, and life span extension.  Resveratrol, a polyphenol found in wines and thought to harbor major health benefits, was reported to be an activator of Sir2 enzymes in vivo and in vitro.  In addn., resveratrol was shown to increase life span in three model organisms through a Sir2-dependent pathway.  Here, we investigated the mol. basis for Sir2 activation by resveratrol.  Among the three enzymes tested (yeast Sir2, human SIRT1, and human SIRT2), only SIRT1 exhibited significant enzyme activation (∼8-fold) using the com. available Fluor de Lys kit (BioMol).  To examine the requirements for resveratrol activation of SIRT1, we synthesized three p53 acetylpeptide substrates either lacking a fluorophore or contg. a 7-amino-4-methylcoumarin (p53-AMC) or rhodamine 110 (p53-R110).  Although SIRT1 activation was independent of the acetylpeptide sequence, resveratrol activation was completely dependent on the presence of a covalently attached fluorophore.  Substrate competition studies indicated that the fluorophore decreased the binding affinity of the peptide, and, in the presence of resveratrol, fluorophore-contg. substrates bound more tightly to SIRT1.  Using available crystal structures, a model of SIRT1 bound to p53-AMC peptide was constructed.  Without resveratrol, the coumarin of p53-AMC peptide is solvent-exposed and makes no significant contacts with SIRT1.  We propose that binding of resveratrol to SIRT1 promotes a conformational change that better accommodates the attached coumarin group.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqP2qEn7gxZILVg90H21EOLACvtfcHk0lgAIQbjxT0-cQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjsFOgtb8%253D&md5=f70797470cf27070646e3a1ccb5a0a2f</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M501250200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M501250200%26sid%3Dliteratum%253Aachs%26aulast%3DBorra%26aufirst%3DM.%2BT.%26aulast%3DSmith%26aufirst%3DB.%2BC.%26aulast%3DDenu%26aufirst%3DJ.%2BM.%26atitle%3DMechanism%2520of%2520human%2520SIRT1%2520activation%2520by%2520resveratrol%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2005%26volume%3D280%26spage%3D17187%26epage%3D17195" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Geronimo, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singh, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, K. S.</span><span> </span><span class="NLM_article-title">How different are electron-rich and electron-deficient pi interactions?</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">1931</span><span class="NLM_x">–</span> <span class="NLM_lpage">1934</span></span><div class="citationLinks">[<a href="/doi/10.1021/ct100182u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnsFGnsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=1931-1934&author=I.+Geronimoauthor=E.+C.+Leeauthor=N.+J.+Singhauthor=K.+S.+Kim&title=How+different+are+electron-rich+and+electron-deficient+pi+interactions%3F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">How Different are Electron-Rich and Electron-Deficient π Interactions?</span></div><div class="casAuthors">Geronimo, Inacrist; Lee, Eun Cheol; Singh, N. Jiten; Kim, Kwang S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1931-1934</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The intermol. interaction driven structural change is vital to mol. architecturing.  In the Cambridge Structural Database (CSD), we find that the preference for geometrical conformations of electron-deficient π systems is different from those of electron-rich π systems.  Indeed, ab initio calcns. find that electron-deficient π ring systems should involve different structures and energetics, consistent with the CSD search, due to the elec. multipole moments and the decrease in the spatial extent of π-electron d.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-_MjOdwiK6LVg90H21EOLACvtfcHk0lgAIQbjxT0-cQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnsFGnsro%253D&md5=2d88740775072e6eba2e195631e1871c</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fct100182u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct100182u%26sid%3Dliteratum%253Aachs%26aulast%3DGeronimo%26aufirst%3DI.%26aulast%3DLee%26aufirst%3DE.%2BC.%26aulast%3DSingh%26aufirst%3DN.%2BJ.%26aulast%3DKim%26aufirst%3DK.%2BS.%26atitle%3DHow%2520different%2520are%2520electron-rich%2520and%2520electron-deficient%2520pi%2520interactions%253F%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2010%26volume%3D6%26spage%3D1931%26epage%3D1934" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Mishra, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sathyamurthy, N.</span><span> </span><span class="NLM_article-title">Pi–pi interaction in pyridine</span> <span class="citation_source-journal">J. Phys. Chem. A</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">6</span><span class="NLM_x">–</span> <span class="NLM_lpage">8</span></span><div class="citationLinks">[<a href="/doi/10.1021/jp045218c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2005&pages=6-8&author=B.+K.+Mishraauthor=N.+Sathyamurthy&title=Pi%E2%80%93pi+interaction+in+pyridine"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fjp045218c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp045218c%26sid%3Dliteratum%253Aachs%26aulast%3DMishra%26aufirst%3DB.%2BK.%26aulast%3DSathyamurthy%26aufirst%3DN.%26atitle%3DPi%25E2%2580%2593pi%2520interaction%2520in%2520pyridine%26jtitle%3DJ.%2520Phys.%2520Chem.%2520A%26date%3D2005%26volume%3D109%26spage%3D6%26epage%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Mishra, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sathyamurthy, N.</span><span> </span><span class="NLM_article-title">Stacking interaction in pyrazine dimer</span> <span class="citation_source-journal">J. Theor. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">609</span><span class="NLM_x">–</span> <span class="NLM_lpage">619</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1142%2FS0219633606002532" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVSitbfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=609-619&author=B.+K.+Mishraauthor=N.+Sathyamurthy&title=Stacking+interaction+in+pyrazine+dimer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Stacking interaction in pyrazine dimer</span></div><div class="casAuthors">Mishra, Brijesh Kumar; Sathyamurthy, N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Theoretical & Computational Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">609-619</span>CODEN:
                <span class="NLM_cas:coden">JTCCAC</span>;
        ISSN:<span class="NLM_cas:issn">0219-6336</span>.
    
            (<span class="NLM_cas:orgname">World Scientific Publishing Co. Pte. Ltd.</span>)
        </div><div class="casAbstract">The structure and stability of pyrazine dimer in different orientations have been investigated using second- and fourth-order Moller-Plesset perturbation theory (MP2, MP4) and coupled-cluster singles and doubles with non-iterative perturbative triples method [CCSD(T)] with various basis sets (6-31G*, 6-311G**, 6-311++G**, cc-pVDZ, aug-cc-pVDZ, cc-pVTZ, and aug-cc-pVTZ).  With the largest basis set (aug-cc-pVTZ) at the MP2 level of theory, the calcd. binding energies (basis set superposition error cor.) for the sandwich, N-N axial-displaced, lateral-displaced, cross-displaced, T-shaped (TN-ring and TH-ring), and wedge-shaped (WN-C and WC-C) geometries are found to be 3.01, 4.18, 4.63, 5.92, 4.54, 2.78, 2.89, and 3.23 kcal/mol, resp.  Although MP2 calcns. predict the cross-displaced geometry to be the most stable one, MP4 and CCSD(T) calcns. show the most stable geometry to be T-shaped (TN-ring), with a center-of-mass sepn. of 4.2 Å.  Dispersion interaction seems to be the major source of attraction in all the geometries considered, while the nature of electrostatic interaction depends on the mutual orientation of the two rings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAuB99DvoNSrVg90H21EOLACvtfcHk0lhSsL2gmi3TWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVSitbfN&md5=d882b9bb68905939640a6ef4f658eab2</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1142%2FS0219633606002532&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1142%252FS0219633606002532%26sid%3Dliteratum%253Aachs%26aulast%3DMishra%26aufirst%3DB.%2BK.%26aulast%3DSathyamurthy%26aufirst%3DN.%26atitle%3DStacking%2520interaction%2520in%2520pyrazine%2520dimer%26jtitle%3DJ.%2520Theor.%2520Comput.%2520Chem.%26date%3D2006%26volume%3D5%26spage%3D609%26epage%3D619" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Lee, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jurecka, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarakeshwar, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hobza, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, K. S.</span><span> </span><span class="NLM_article-title">Understanding of assembly phenomena by aromatic–aromatic interactions: benzene dimer and the substituted systems</span> <span class="citation_source-journal">J. Phys. Chem. A</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">3446</span><span class="NLM_x">–</span> <span class="NLM_lpage">3457</span></span><div class="citationLinks">[<a href="/doi/10.1021/jp068635t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2007&pages=3446-3457&author=E.+C.+Leeauthor=D.+Kimauthor=P.+Jureckaauthor=P.+Tarakeshwarauthor=P.+Hobzaauthor=K.+S.+Kim&title=Understanding+of+assembly+phenomena+by+aromatic%E2%80%93aromatic+interactions%3A+benzene+dimer+and+the+substituted+systems"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fjp068635t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp068635t%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DE.%2BC.%26aulast%3DKim%26aufirst%3DD.%26aulast%3DJurecka%26aufirst%3DP.%26aulast%3DTarakeshwar%26aufirst%3DP.%26aulast%3DHobza%26aufirst%3DP.%26aulast%3DKim%26aufirst%3DK.%2BS.%26atitle%3DUnderstanding%2520of%2520assembly%2520phenomena%2520by%2520aromatic%25E2%2580%2593aromatic%2520interactions%253A%2520benzene%2520dimer%2520and%2520the%2520substituted%2520systems%26jtitle%3DJ.%2520Phys.%2520Chem.%2520A%26date%3D2007%26volume%3D111%26spage%3D3446%26epage%3D3457" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Sinnokrot, M. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherrill, C. D.</span><span> </span><span class="NLM_article-title">Substituent effects in pi–pi interactions: sandwich and T-shaped configurations</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">126</span><span class="NLM_x">, </span> <span class="NLM_fpage">7690</span><span class="NLM_x">–</span> <span class="NLM_lpage">7697</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja049434a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2004&pages=7690-7697&author=M.+O.+Sinnokrotauthor=C.+D.+Sherrill&title=Substituent+effects+in+pi%E2%80%93pi+interactions%3A+sandwich+and+T-shaped+configurations"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fja049434a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja049434a%26sid%3Dliteratum%253Aachs%26aulast%3DSinnokrot%26aufirst%3DM.%2BO.%26aulast%3DSherrill%26aufirst%3DC.%2BD.%26atitle%3DSubstituent%2520effects%2520in%2520pi%25E2%2580%2593pi%2520interactions%253A%2520sandwich%2520and%2520T-shaped%2520configurations%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2004%26volume%3D126%26spage%3D7690%26epage%3D7697" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Sakaguchi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrera, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saito, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bustin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vassilev, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderson, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Appella, E.</span><span> </span><span class="NLM_article-title">DNA damage activates p53 through a phosphorylation–acetylation cascade</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">2831</span><span class="NLM_x">–</span> <span class="NLM_lpage">2841</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=1998&pages=2831-2841&author=K.+Sakaguchiauthor=J.+E.+Herreraauthor=S.+Saitoauthor=T.+Mikiauthor=M.+Bustinauthor=A.+Vassilevauthor=C.+W.+Andersonauthor=E.+Appella&title=DNA+damage+activates+p53+through+a+phosphorylation%E2%80%93acetylation+cascade"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSakaguchi%26aufirst%3DK.%26aulast%3DHerrera%26aufirst%3DJ.%2BE.%26aulast%3DSaito%26aufirst%3DS.%26aulast%3DMiki%26aufirst%3DT.%26aulast%3DBustin%26aufirst%3DM.%26aulast%3DVassilev%26aufirst%3DA.%26aulast%3DAnderson%26aufirst%3DC.%2BW.%26aulast%3DAppella%26aufirst%3DE.%26atitle%3DDNA%2520damage%2520activates%2520p53%2520through%2520a%2520phosphorylation%25E2%2580%2593acetylation%2520cascade%26jtitle%3DGenes%2520Dev.%26date%3D1998%26volume%3D12%26spage%3D2831%26epage%3D2841" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Wang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guan, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tong, Q.</span><span> </span><span class="NLM_article-title">Deacetylation of FOXO3 by SIRT1 or SIRT2 leads to Skp2-mediated FOXO3 ubiquitination and degradation</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">1546</span><span class="NLM_x">–</span> <span class="NLM_lpage">1557</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2012&pages=1546-1557&author=F.+Wangauthor=C.+H.+Chanauthor=K.+Chenauthor=X.+Guanauthor=H.+K.+Linauthor=Q.+Tong&title=Deacetylation+of+FOXO3+by+SIRT1+or+SIRT2+leads+to+Skp2-mediated+FOXO3+ubiquitination+and+degradation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DF.%26aulast%3DChan%26aufirst%3DC.%2BH.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DGuan%26aufirst%3DX.%26aulast%3DLin%26aufirst%3DH.%2BK.%26aulast%3DTong%26aufirst%3DQ.%26atitle%3DDeacetylation%2520of%2520FOXO3%2520by%2520SIRT1%2520or%2520SIRT2%2520leads%2520to%2520Skp2-mediated%2520FOXO3%2520ubiquitination%2520and%2520degradation%26jtitle%3DOncogene%26date%3D2012%26volume%3D31%26spage%3D1546%26epage%3D1557" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Peng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ling, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukasawa, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robertson, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olashaw, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koomen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lane, W. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seto, E.</span><span> </span><span class="NLM_article-title">SIRT1 deacetylates the DNA methyltransferase 1 (DNMT1) protein and alters its activities</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">4720</span><span class="NLM_x">–</span> <span class="NLM_lpage">4734</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1128%2FMCB.06147-11" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Fjm301032j&amp;key=21947282" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFakt7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2011&pages=4720-4734&author=L.+Pengauthor=Z.+Yuanauthor=H.+Lingauthor=K.+Fukasawaauthor=K.+Robertsonauthor=N.+Olashawauthor=J.+Koomenauthor=J.+Chenauthor=W.+S.+Laneauthor=E.+Seto&title=SIRT1+deacetylates+the+DNA+methyltransferase+1+%28DNMT1%29+protein+and+alters+its+activities"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">SIRT1 deacetylates the DNA methyltransferase 1 (DNMT1) protein and alters its activities</span></div><div class="casAuthors">Peng, Lirong; Yuan, Zhigang; Ling, Hongbo; Fukasawa, Kenji; Robertson, Keith; Olashaw, Nancy; Koomen, John; Chen, Jiandong; Lane, William S.; Seto, Edward</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">4720-4734</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">DNA methylation and histone acetylation/deacetylation are distinct biochem. processes that control gene expression.  While DNA methylation is a common epigenetic signal that inhibits gene transcription, histone deacetylation similarly represses transcription but can be both an epigenetic and non-epigenetic phenomenon.  Here it is reported that the histone deacetylase SIRT1 regulates the activities of DNMT1, a key enzyme responsible for DNA methylation.  In mass spectrometry anal., 12 new acetylated lysine sites were identified in DNMT1.  SIRT1 phys. assocs. with DNMT1 and can deacetylate acetylated DNMT1 in vitro and in vivo.  Interestingly, deacetylation of different lysines on DNMT1 has different effects on the functions of DNMT1.  For example, deacetylation of Lys1349 and Lys1415 in the catalytic domain of DNMT1 enhances DNMT1's methyltransferase activity, while deacetylation of lysine residues in the GK linker decreases DNMT1's methyltransferase-independent transcriptional repression function.  Furthermore, deacetylation of all identified acetylated lysine sites in DNMT1 abrogates its binding to SIRT1 and impairs its capability to regulate cell cycle G2/M transition.  Finally, inhibition of SIRT1 strengthens the silencing effects of DNMT1 on the expression of tumor suppressor genes ER-α and CDH1 in MDA-MB-231 breast cancer cells.  Together, these results suggest that SIRT1-mediated deacetylation of DNMT1 is crucial for DNMT1's multiple effects in gene silencing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpb75sh63SEw7Vg90H21EOLACvtfcHk0lgPRs8dmU-SvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFakt7rO&md5=e8d6e4eeea1f81de86fb43193f3a60f4</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1128%2FMCB.06147-11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.06147-11%26sid%3Dliteratum%253Aachs%26aulast%3DPeng%26aufirst%3DL.%26aulast%3DYuan%26aufirst%3DZ.%26aulast%3DLing%26aufirst%3DH.%26aulast%3DFukasawa%26aufirst%3DK.%26aulast%3DRobertson%26aufirst%3DK.%26aulast%3DOlashaw%26aufirst%3DN.%26aulast%3DKoomen%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DLane%26aufirst%3DW.%2BS.%26aulast%3DSeto%26aufirst%3DE.%26atitle%3DSIRT1%2520deacetylates%2520the%2520DNA%2520methyltransferase%25201%2520%2528DNMT1%2529%2520protein%2520and%2520alters%2520its%2520activities%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2011%26volume%3D31%26spage%3D4720%26epage%3D4734" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Vagin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teplyakov, A.</span><span> </span><span class="NLM_article-title">MOLREP: an automated program for molecular replacement</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">1022</span><span class="NLM_x">–</span> <span class="NLM_lpage">1025</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1107%2FS0021889897006766" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1997&pages=1022-1025&author=A.+Vaginauthor=A.+Teplyakov&title=MOLREP%3A+an+automated+program+for+molecular+replacement"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">MOLREP: an automated program for molecular replacement</span></div><div class="casAuthors">Vagin, Alexei; Teplyakov, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1022-1025</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">MOLREP is an automated program for mol. replacement which uses effective new approaches in data processing and rotational and translational searching.  These include an automatic choice of all parameters, scaling by Patterson origin peaks and soft resoln. cutoff.  One of the cornerstones of the program is an original full-symmetry translation function combined with a packing function.  Information from the model already placed in the cell is incorporated in both translation and packing functions.  A no. of tests using exptl. data proved the ability of the program to find the correct soln. in difficult cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkGXdZ-nTKxLVg90H21EOLACvtfcHk0lgPRs8dmU-SvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D&md5=96af64aedd8a8f018c3c18b57a6b4b21</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1107%2FS0021889897006766&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889897006766%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DTeplyakov%26aufirst%3DA.%26atitle%3DMOLREP%253A%2520an%2520automated%2520program%2520for%2520molecular%2520replacement%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1997%26volume%3D30%26spage%3D1022%26epage%3D1025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Sheldrick, G. M.</span><span> </span><span class="NLM_article-title">A short history of SHELX</span> <span class="citation_source-journal">Acta Crystallogr., Sect. A: Found. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">64</span><span class="NLM_x">, </span> <span class="NLM_fpage">112</span><span class="NLM_x">–</span> <span class="NLM_lpage">122</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1107%2FS0108767307043930" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Fjm301032j&amp;key=18156677" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVGhurzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2008&pages=112-122&author=G.+M.+Sheldrick&title=A+short+history+of+SHELX"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">A short history of SHELX</span></div><div class="casAuthors">Sheldrick, George M.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section A: Foundations of Crystallography</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">112-122</span>CODEN:
                <span class="NLM_cas:coden">ACACEQ</span>;
        ISSN:<span class="NLM_cas:issn">0108-7673</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">An account is given of the development of the SHELX system of computer programs from SHELX-76 to the present day.  In addn. to identifying useful innovations that have come into general use through their implementation in SHELX, a crit. anal. is presented of the less-successful features, missed opportunities and desirable improvements for future releases of the software.  An attempt is made to understand how a program originally designed for photog. intensity data, punched cards and computers over 10000 times slower than an av. modern personal computer has managed to survive for so long.  SHELXL is the most widely used program for small-mol. refinement and SHELXS and SHELXD are often employed for structure soln. despite the availability of objectively superior programs.  SHELXL also finds a niche for the refinement of macromols. against high-resoln. or twinned data; SHELXPRO acts as an interface for macromol. applications.  SHELXC, SHELXD and SHELXE are proving useful for the exptl. phasing of macromols., esp. because they are fast and robust and so are often employed in pipelines for high-throughput phasing.  This paper could serve as a general literature citation when one or more of the open-source SHELX programs (and the Bruker AXS version SHELXTL) are employed in the course of a crystal-structure detn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfLPYyJyJ2RbVg90H21EOLACvtfcHk0ljHlaolliiD9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVGhurzO&md5=8f91f10be4a9df74b2a9dad522c71e6e</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1107%2FS0108767307043930&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0108767307043930%26sid%3Dliteratum%253Aachs%26aulast%3DSheldrick%26aufirst%3DG.%2BM.%26atitle%3DA%2520short%2520history%2520of%2520SHELX%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520A%253A%2520Found.%2520Crystallogr.%26date%3D2008%26volume%3D64%26spage%3D112%26epage%3D122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Vagin, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steiner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lebedev, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Potterton, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McNicholas, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Long, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murshudov, G. N.</span><span> </span><span class="NLM_article-title">REFMAC5 dictionary: organization of prior chemical knowledge and guidelines for its use</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">2184</span><span class="NLM_x">–</span> <span class="NLM_lpage">2195</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1107%2FS0907444904023510" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Fjm301032j&amp;key=15572771" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2184-2195&author=A.+A.+Vaginauthor=R.+A.+Steinerauthor=A.+A.+Lebedevauthor=L.+Pottertonauthor=S.+McNicholasauthor=F.+Longauthor=G.+N.+Murshudov&title=REFMAC5+dictionary%3A+organization+of+prior+chemical+knowledge+and+guidelines+for+its+use"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">REFMAC5 dictionary: organization of prior chemical knowledge and guidelines for its use</span></div><div class="casAuthors">Vagin, Alexei A.; Steiner, Roberto A.; Lebedev, Andrey A.; Potterton, Liz; McNicholas, Stuart; Long, Fei; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2184-2195</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">One of the most important aspects of macromol. structure refinement is the use of prior chem. knowledge.  Bond lengths, bond angles and other chem. properties are used in restrained refinement as subsidiary conditions.  This contribution describes the organization and some aspects of the use of the flexible and human/machine-readable dictionary of prior chem. knowledge used by the max.-likelihood macromol.-refinement program REFMAC5.  The dictionary stores information about monomers which represent the constitutive building blocks of biol. macromols. (amino acids, nucleic acids and saccharides) and about numerous org./inorg. compds. commonly found in macromol. crystallog.  It also describes the modifications the building blocks undergo as a result of chem. reactions and the links required for polymer formation.  More than 2000 monomer entries, 100 modification entries and 200 link entries are currently available.  Algorithms and tools for updating and adding new entries to the dictionary have also been developed and are presented here.  In many cases, the REFMAC5 dictionary allows entirely automatic generation of restraints within REFMAC5 refinement runs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRjDd-k3YHGrVg90H21EOLACvtfcHk0ljHlaolliiD9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73F&md5=005910422c5fb72fe2434c39b5321bf4</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1107%2FS0907444904023510&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904023510%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DSteiner%26aufirst%3DR.%2BA.%26aulast%3DLebedev%26aufirst%3DA.%2BA.%26aulast%3DPotterton%26aufirst%3DL.%26aulast%3DMcNicholas%26aufirst%3DS.%26aulast%3DLong%26aufirst%3DF.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DREFMAC5%2520dictionary%253A%2520organization%2520of%2520prior%2520chemical%2520knowledge%2520and%2520guidelines%2520for%2520its%2520use%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2184%26epage%3D2195" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">2126</span><span class="NLM_x">–</span> <span class="NLM_lpage">2132</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Fjm301032j&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0liI4G7FjqlgHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Thompson, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higgins, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibson, T. J.</span><span> </span><span class="NLM_article-title">Clustal-W improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">4673</span><span class="NLM_x">–</span> <span class="NLM_lpage">4680</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1994&pages=4673-4680&author=J.+D.+Thompsonauthor=D.+G.+Higginsauthor=T.+J.+Gibson&title=Clustal-W+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DThompson%26aufirst%3DJ.%2BD.%26aulast%3DHiggins%26aufirst%3DD.%2BG.%26aulast%3DGibson%26aufirst%3DT.%2BJ.%26atitle%3DClustal-W%2520improving%2520the%2520sensitivity%2520of%2520progressive%2520multiple%2520sequence%2520alignment%2520through%2520sequence%2520weighting%252C%2520position-specific%2520gap%2520penalties%2520and%2520weight%2520matrix%2520choice%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D1994%26volume%3D22%26spage%3D4673%26epage%3D4680" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group">Bowie, J. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luthy, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eisenberg, D.</span><span> </span><span class="NLM_article-title">A method to identify protein sequences that fold into a known three-dimensional structure</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">253</span><span class="NLM_x">, </span> <span class="NLM_fpage">164</span><span class="NLM_x">–</span> <span class="NLM_lpage">170</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1126%2Fscience.1853201" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Fjm301032j&amp;key=1853201" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADyaK3MXltV2qsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=253&publication_year=1991&pages=164-170&author=J.+U.+Bowieauthor=R.+Luthyauthor=D.+Eisenberg&title=A+method+to+identify+protein+sequences+that+fold+into+a+known+three-dimensional+structure"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">A method to identify protein sequences that fold into a known three-dimensional structure</span></div><div class="casAuthors">Bowie, James U.; Luthy, Roland; Eisenberg, David</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">253</span>
        (<span class="NLM_cas:issue">5016</span>),
    <span class="NLM_cas:pages">164-70</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">The inverse protein folding problem, the problem of finding which amino acid sequences fold into a known three-dimensional (3D) structure, can be effectively attacked by finding sequences that are most compatible with the environments of the residues in the 3D structure.  The environments are described by: (i) the area of the residue buried in the protein and inaccessible to solvent; (ii) the fraction of side-chain area that is covered by polar atoms (O and N); and (iii) the local secondary structure.  Examples of this 3D profile method are presented for four families of proteins: the globins, cAMP (adenosine 3',5'-monophosphate) receptor-like proteins, the periplasmic binding proteins, and the actins.  This method is able to detect the structural similarity of the actins and 70-kilodalton heat shock proteins, even though these protein families share no detectable sequence similarity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBNyBVziUN_bVg90H21EOLACvtfcHk0liI4G7FjqlgHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXltV2qsbk%253D&md5=18345191c3ef68ca2c2475522c9ee699</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1126%2Fscience.1853201&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1853201%26sid%3Dliteratum%253Aachs%26aulast%3DBowie%26aufirst%3DJ.%2BU.%26aulast%3DLuthy%26aufirst%3DR.%26aulast%3DEisenberg%26aufirst%3DD.%26atitle%3DA%2520method%2520to%2520identify%2520protein%2520sequences%2520that%2520fold%2520into%2520a%2520known%2520three-dimensional%2520structure%26jtitle%3DScience%26date%3D1991%26volume%3D253%26spage%3D164%26epage%3D170" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">Morris, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodsell, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halliday, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huey, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hart, W. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belew, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, A. J.</span><span> </span><span class="NLM_article-title">Automated docking using a Lamarckian genetic algorithm and an empirical binding free energy function</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">1639</span><span class="NLM_x">–</span> <span class="NLM_lpage">1662</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1002%2F%28SICI%291096-987X%2819981115%2919%3A14%3C1639%3A%3AAID-JCC10%3E3.0.CO%3B2-B" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADyaK1cXntFemur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=1998&pages=1639-1662&author=G.+M.+Morrisauthor=D.+S.+Goodsellauthor=R.+S.+Hallidayauthor=R.+Hueyauthor=W.+E.+Hartauthor=R.+K.+Belewauthor=A.+J.+Olson&title=Automated+docking+using+a+Lamarckian+genetic+algorithm+and+an+empirical+binding+free+energy+function"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Automated docking using a Lamarckian genetic algorithm and an empirical binding free energy function</span></div><div class="casAuthors">Morris, Garrett M.; Goodsell, David S.; Halliday, Robert S.; Huey, Ruth; Hart, William E.; Belew, Richard K.; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1639-1662</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review, with 81 refs.  A novel and robust automated docking method that predicts the bound conformations of flexible ligands to macromol. targets has been developed and tested, in combination with a new scoring function that ests. the free energy change upon binding.  Interestingly, this method applies a Lamarckian model of genetics, in which environmental adaptations of an individual's phenotype are reverse transcribed into its genotype and become heritable traits (sic).  We consider three search methods, Monte Carlo simulated annealing, a traditional genetic algorithm, and the Lamarckian genetic algorithm, and compare their performance in dockings of seven protein-ligand test systems having known three-dimensional structure.  We show that both the traditional and Lamarckian genetic algorithms can handle ligands with more degrees of freedom than the simulated annealing method used in earlier versions of AuToDOCK, and that the Lamarckian genetic algorithm is the most efficient, reliable, and successful of the three.  The empirical free energy function was calibrated using a set of 30 structurally known protein-ligand complexes with exptl. detd. binding consts.  Linear regression anal. of the obsd. binding consts. in terms of a wide variety of structure-derived mol. properties was performed.  The final model had a residual std. error of 9.11 kJ mol-1 (2.177 kcal mol-1) and was chosen as the new energy function.  The new search methods and empirical free energy function are available in AUTODOCK, version 3.0.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWXZfPvnZWcbVg90H21EOLACvtfcHk0liF6Uk7ee8-2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXntFemur4%253D&md5=804b9f797c0b2cf942444c17b5653659</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291096-987X%2819981115%2919%3A14%3C1639%3A%3AAID-JCC10%3E3.0.CO%3B2-B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291096-987X%252819981115%252919%253A14%253C1639%253A%253AAID-JCC10%253E3.0.CO%253B2-B%26sid%3Dliteratum%253Aachs%26aulast%3DMorris%26aufirst%3DG.%2BM.%26aulast%3DGoodsell%26aufirst%3DD.%2BS.%26aulast%3DHalliday%26aufirst%3DR.%2BS.%26aulast%3DHuey%26aufirst%3DR.%26aulast%3DHart%26aufirst%3DW.%2BE.%26aulast%3DBelew%26aufirst%3DR.%2BK.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DAutomated%2520docking%2520using%2520a%2520Lamarckian%2520genetic%2520algorithm%2520and%2520an%2520empirical%2520binding%2520free%2520energy%2520function%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1998%26volume%3D19%26spage%3D1639%26epage%3D1662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group">Morris, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huey, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindstrom, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanner, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belew, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodsell, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, A. J.</span><span> </span><span class="NLM_article-title">AutoDock4 and AutoDockTools4: automated docking with selective receptor flexibility</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">2785</span><span class="NLM_x">–</span> <span class="NLM_lpage">2791</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Fjm301032j&amp;key=10.1002%2Fjcc.21256" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Fjm301032j&amp;key=19399780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Fjm301032j&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1GitrnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2009&pages=2785-2791&author=G.+M.+Morrisauthor=R.+Hueyauthor=W.+Lindstromauthor=M.+F.+Sannerauthor=R.+K.+Belewauthor=D.+S.+Goodsellauthor=A.+J.+Olson&title=AutoDock4+and+AutoDockTools4%3A+automated+docking+with+selective+receptor+flexibility"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">AutoDock and AutoDockTools: Automated docking with selective receptor flexibility</span></div><div class="casAuthors">Morris, Garrett M.; Huey, Ruth; Lindstrom, William; Sanner, Michel F.; Belew, Richard K.; Goodsell, David S.; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2785-2791</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We describe the testing and release of AutoDock4 and the accompanying graphical user interface AutoDockTools.  AutoDock4 incorporates limited flexibility in the receptor.  Several tests are reported here, including a redocking expt. with 188 diverse ligand-protein complexes and a cross-docking expt. using flexible sidechains in 87 HIV protease complexes.  We also report its utility in anal. of covalently bound ligands, using both a grid-based docking method and a modification of the flexible sidechain technique. © 2009 Wiley Periodicals, Inc.  J Comput Chem, 2009.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqktE6WAo4GEbVg90H21EOLACvtfcHk0lhBP0Y8Regq_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1GitrnK&md5=679ce22fc50e9291c9aa16e7a1855845</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21256%26sid%3Dliteratum%253Aachs%26aulast%3DMorris%26aufirst%3DG.%2BM.%26aulast%3DHuey%26aufirst%3DR.%26aulast%3DLindstrom%26aufirst%3DW.%26aulast%3DSanner%26aufirst%3DM.%2BF.%26aulast%3DBelew%26aufirst%3DR.%2BK.%26aulast%3DGoodsell%26aufirst%3DD.%2BS.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DAutoDock4%2520and%2520AutoDockTools4%253A%2520automated%2520docking%2520with%2520selective%2520receptor%2520flexibility%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2009%26volume%3D30%26spage%3D2785%26epage%3D2791" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GLR" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GLR','PDB','3GLR'); return false;">PDB: 3GLR</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GLS" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GLS','PDB','3GLS'); return false;">PDB: 3GLS</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FZ3" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FZ3','PDB','4FZ3'); return false;">PDB: 4FZ3</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-4"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i88"><a href="/doi/suppl/10.1021/jm301032j">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_64811"></div></div></div></div></div><hr /></hr><p class="last">Information about the synthesis of the chemical compounds, molecular modeling, HPLC data, isothermal titration calorimetry, enzyme kinetics data, etc. This material is available free of charge via the Internet at <a class="extLink" href="http://pubs.acs.org">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm301032j/suppl_file/jm301032j_si_001.pdf">jm301032j_si_001.pdf (811.14 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The coordinates of “SIRT3/ac-RHKK<sub>ac</sub>-AMC” complex structure were deposited in protein data bank with the PDB ID of <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FZ3">4FZ3</a>.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm301032j&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Fjm301032j%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2013.56.issue-3%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm301032j" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679a988afb423c94","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
