
<html lang="en"     class="pb-page"  data-request-id="e73f7302-1f1e-4966-afec-398b3554c225"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2021.64.issue-5;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.0c01846;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Antibody-Mediated Delivery of Chimeric BRD4 Degraders. Part 2: Improvement of In Vitro Antiproliferation Activity and In Vivo Antitumor Efficacy" /></meta><meta name="dc.Creator" content="Peter S.  Dragovich" /></meta><meta name="dc.Creator" content="Thomas H.  Pillow" /></meta><meta name="dc.Creator" content="Robert A.  Blake" /></meta><meta name="dc.Creator" content="Jack D.  Sadowsky" /></meta><meta name="dc.Creator" content="Emel  Adaligil" /></meta><meta name="dc.Creator" content="Pragya  Adhikari" /></meta><meta name="dc.Creator" content="Jinhua  Chen" /></meta><meta name="dc.Creator" content="Nicholas  Corr" /></meta><meta name="dc.Creator" content="Josefa  dela Cruz-Chuh" /></meta><meta name="dc.Creator" content="Geoffrey  Del Rosario" /></meta><meta name="dc.Creator" content="Aaron  Fullerton" /></meta><meta name="dc.Creator" content="Steven J.  Hartman" /></meta><meta name="dc.Creator" content="Fan  Jiang" /></meta><meta name="dc.Creator" content="Susan  Kaufman" /></meta><meta name="dc.Creator" content="Tracy  Kleinheinz" /></meta><meta name="dc.Creator" content="Katherine R.  Kozak" /></meta><meta name="dc.Creator" content="Liling  Liu" /></meta><meta name="dc.Creator" content="Ying  Lu" /></meta><meta name="dc.Creator" content="Melinda M.  Mulvihill" /></meta><meta name="dc.Creator" content="Jeremy M.  Murray" /></meta><meta name="dc.Creator" content="Aimee  O’Donohue" /></meta><meta name="dc.Creator" content="Rebecca K.  Rowntree" /></meta><meta name="dc.Creator" content="William S.  Sawyer" /></meta><meta name="dc.Creator" content="Leanna R.  Staben" /></meta><meta name="dc.Creator" content="John  Wai" /></meta><meta name="dc.Creator" content="Jian  Wang" /></meta><meta name="dc.Creator" content="BinQing  Wei" /></meta><meta name="dc.Creator" content="Wentao  Wei" /></meta><meta name="dc.Creator" content="Zijin  Xu" /></meta><meta name="dc.Creator" content="Hui  Yao" /></meta><meta name="dc.Creator" content="Shang-Fan  Yu" /></meta><meta name="dc.Creator" content="Donglu  Zhang" /></meta><meta name="dc.Creator" content="Hongyan  Zhang" /></meta><meta name="dc.Creator" content="Shenhua  Zhang" /></meta><meta name="dc.Creator" content="Yongxin  Zhao" /></meta><meta name="dc.Creator" content="Hao  Zhou" /></meta><meta name="dc.Creator" content="Xiaoyu  Zhu" /></meta><meta name="dc.Description" content="Heterobifunctional compounds that direct the ubiquitination of intracellular proteins in a targeted manner via co-opted ubiquitin ligases have enormous potential to transform the field of medicinal..." /></meta><meta name="Description" content="Heterobifunctional compounds that direct the ubiquitination of intracellular proteins in a targeted manner via co-opted ubiquitin ligases have enormous potential to transform the field of medicinal..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="February 17, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c01846" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01846" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c01846" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01846" /></link>
        
    
    

<title>Antibody-Mediated Delivery of Chimeric BRD4 Degraders. Part 2: Improvement of In Vitro Antiproliferation Activity and In Vivo Antitumor Efficacy | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c01846" /></meta><meta property="og:title" content="Antibody-Mediated Delivery of Chimeric BRD4 Degraders. Part 2: Improvement of In Vitro Antiproliferation Activity and In Vivo Antitumor Efficacy" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0032.jpeg" /></meta><meta property="og:description" content="Heterobifunctional compounds that direct the ubiquitination of intracellular proteins in a targeted manner via co-opted ubiquitin ligases have enormous potential to transform the field of medicinal chemistry. These chimeric molecules, often termed proteolysis-targeting chimeras (PROTACs) in the chemical literature, enable the controlled degradation of specific proteins via their direction to the cellular proteasome. In this report, we describe the second phase of our research focused on exploring antibody–drug conjugates (ADCs), which incorporate BRD4-targeting chimeric degrader entities. We employ a new BRD4-binding fragment in the construction of the chimeric ADC payloads that is significantly more potent than the corresponding entity utilized in our initial studies. The resulting BRD4-degrader antibody conjugates exhibit potent and antigen-dependent BRD4 degradation and antiproliferation activities in cell-based experiments. Multiple ADCs bearing chimeric BRD4-degrader payloads also exhibit strong, antigen-dependent antitumor efficacy in mouse xenograft assessments that employ several different tumor models." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c01846"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01846">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c01846&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c01846&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c01846&amp;href=/doi/10.1021/acs.jmedchem.0c01846" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 5</span><span class="cit-fg-pageRange">, 2576-2607</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/5" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c01845" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c01137" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Antibody-Mediated Delivery of Chimeric BRD4 Degraders. Part 2: Improvement of In Vitro Antiproliferation Activity and In Vivo Antitumor Efficacy</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Peter S. Dragovich</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Peter S. Dragovich</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genentech Inc., 1 DNA Way, South San Francisco, California 94080, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#04607665636b726d676c2a74617061764463616a612a676b69"><span class="__cf_email__" data-cfemail="ceaabcafa9a1b8a7ada6e0beabbaabbc8ea9aba0abe0ada1a3">[email protected]</span></a>, Phone: 650-467-6854.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Peter+S.++Dragovich">Peter S. Dragovich</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7372-2862" title="Orcid link">http://orcid.org/0000-0001-7372-2862</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Thomas H. Pillow</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Thomas H. Pillow</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genentech Inc., 1 DNA Way, South San Francisco, California 94080, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Thomas+H.++Pillow">Thomas H. Pillow</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Robert A. Blake</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Robert A. Blake</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genentech Inc., 1 DNA Way, South San Francisco, California 94080, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Robert+A.++Blake">Robert A. Blake</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jack D. Sadowsky</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jack D. Sadowsky</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genentech Inc., 1 DNA Way, South San Francisco, California 94080, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jack+D.++Sadowsky">Jack D. Sadowsky</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0712-5450" title="Orcid link">http://orcid.org/0000-0002-0712-5450</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Emel Adaligil</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Emel Adaligil</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genentech Inc., 1 DNA Way, South San Francisco, California 94080, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Emel++Adaligil">Emel Adaligil</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Pragya Adhikari</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Pragya Adhikari</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genentech Inc., 1 DNA Way, South San Francisco, California 94080, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Pragya++Adhikari">Pragya Adhikari</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jinhua Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jinhua Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">WuXi AppTec, 288 Fute Zhong Road, Waigaoqiao Free Trade Zone, Shanghai 200131, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jinhua++Chen">Jinhua Chen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nicholas Corr</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nicholas Corr</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genentech Inc., 1 DNA Way, South San Francisco, California 94080, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nicholas++Corr">Nicholas Corr</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Josefa dela Cruz-Chuh</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Josefa dela Cruz-Chuh</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genentech Inc., 1 DNA Way, South San Francisco, California 94080, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Josefa++dela+Cruz-Chuh">Josefa dela Cruz-Chuh</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Geoffrey Del Rosario</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Geoffrey Del Rosario</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genentech Inc., 1 DNA Way, South San Francisco, California 94080, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Geoffrey++Del+Rosario">Geoffrey Del Rosario</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Aaron Fullerton</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Aaron Fullerton</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genentech Inc., 1 DNA Way, South San Francisco, California 94080, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Aaron++Fullerton">Aaron Fullerton</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Steven J. Hartman</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Steven J. Hartman</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genentech Inc., 1 DNA Way, South San Francisco, California 94080, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Steven+J.++Hartman">Steven J. Hartman</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Fan Jiang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Fan Jiang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Viva Biotech, Structural Biology, 334 Aidisheng Road, Zhangjiang High-Tech Park, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Fan++Jiang">Fan Jiang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Susan Kaufman</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Susan Kaufman</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genentech Inc., 1 DNA Way, South San Francisco, California 94080, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Susan++Kaufman">Susan Kaufman</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tracy Kleinheinz</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tracy Kleinheinz</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genentech Inc., 1 DNA Way, South San Francisco, California 94080, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tracy++Kleinheinz">Tracy Kleinheinz</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Katherine R. Kozak</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Katherine R. Kozak</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genentech Inc., 1 DNA Way, South San Francisco, California 94080, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Katherine+R.++Kozak">Katherine R. Kozak</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Liling Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Liling Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genentech Inc., 1 DNA Way, South San Francisco, California 94080, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Liling++Liu">Liling Liu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ying Lu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ying Lu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">WuXi AppTec, 288 Fute Zhong Road, Waigaoqiao Free Trade Zone, Shanghai 200131, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ying++Lu">Ying Lu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Melinda M. Mulvihill</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Melinda M. Mulvihill</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genentech Inc., 1 DNA Way, South San Francisco, California 94080, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Melinda+M.++Mulvihill">Melinda M. Mulvihill</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jeremy M. Murray</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jeremy M. Murray</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genentech Inc., 1 DNA Way, South San Francisco, California 94080, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jeremy+M.++Murray">Jeremy M. Murray</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2161-9326" title="Orcid link">http://orcid.org/0000-0002-2161-9326</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Aimee O’Donohue</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Aimee O’Donohue</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genentech Inc., 1 DNA Way, South San Francisco, California 94080, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Aimee++O%E2%80%99Donohue">Aimee O’Donohue</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rebecca K. Rowntree</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rebecca K. Rowntree</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genentech Inc., 1 DNA Way, South San Francisco, California 94080, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rebecca+K.++Rowntree">Rebecca K. Rowntree</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">William S. Sawyer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">William S. Sawyer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genentech Inc., 1 DNA Way, South San Francisco, California 94080, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=William+S.++Sawyer">William S. Sawyer</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Leanna R. Staben</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Leanna R. Staben</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genentech Inc., 1 DNA Way, South San Francisco, California 94080, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Leanna+R.++Staben">Leanna R. Staben</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">John Wai</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">John Wai</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">WuXi AppTec, 288 Fute Zhong Road, Waigaoqiao Free Trade Zone, Shanghai 200131, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=John++Wai">John Wai</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jian Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jian Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">WuXi AppTec, 288 Fute Zhong Road, Waigaoqiao Free Trade Zone, Shanghai 200131, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jian++Wang">Jian Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">BinQing Wei</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">BinQing Wei</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genentech Inc., 1 DNA Way, South San Francisco, California 94080, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=BinQing++Wei">BinQing Wei</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Wentao Wei</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wentao Wei</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Viva Biotech, Structural Biology, 334 Aidisheng Road, Zhangjiang High-Tech Park, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wentao++Wei">Wentao Wei</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Zijin Xu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zijin Xu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">WuXi AppTec, 288 Fute Zhong Road, Waigaoqiao Free Trade Zone, Shanghai 200131, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zijin++Xu">Zijin Xu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hui Yao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hui Yao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">WuXi AppTec, 288 Fute Zhong Road, Waigaoqiao Free Trade Zone, Shanghai 200131, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hui++Yao">Hui Yao</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Shang-Fan Yu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shang-Fan Yu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genentech Inc., 1 DNA Way, South San Francisco, California 94080, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shang-Fan++Yu">Shang-Fan Yu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Donglu Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Donglu Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genentech Inc., 1 DNA Way, South San Francisco, California 94080, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Donglu++Zhang">Donglu Zhang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hongyan Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hongyan Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">WuXi AppTec, 288 Fute Zhong Road, Waigaoqiao Free Trade Zone, Shanghai 200131, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hongyan++Zhang">Hongyan Zhang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Shenhua Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shenhua Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Viva Biotech, Structural Biology, 334 Aidisheng Road, Zhangjiang High-Tech Park, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shenhua++Zhang">Shenhua Zhang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yongxin Zhao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yongxin Zhao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">WuXi AppTec, 288 Fute Zhong Road, Waigaoqiao Free Trade Zone, Shanghai 200131, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yongxin++Zhao">Yongxin Zhao</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hao Zhou</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hao Zhou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">WuXi AppTec, 288 Fute Zhong Road, Waigaoqiao Free Trade Zone, Shanghai 200131, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hao++Zhou">Hao Zhou</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Xiaoyu Zhu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiaoyu Zhu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">WuXi AppTec, 288 Fute Zhong Road, Waigaoqiao Free Trade Zone, Shanghai 200131, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiaoyu++Zhu">Xiaoyu Zhu</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01846&amp;href=/doi/10.1021%2Facs.jmedchem.0c01846" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 5</span><span class="cit-pageRange">, 2576–2607</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">February 17, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>21 October 2020</li><li><span class="item_label"><b>Published</b> online</span>17 February 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 11 March 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c01846" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01846</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D2576%26pageCount%3D32%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DPeter%2BS.%2BDragovich%252C%2BThomas%2BH.%2BPillow%252C%2BRobert%2BA.%2BBlake%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D5%26contentID%3Dacs.jmedchem.0c01846%26title%3DAntibody-Mediated%2BDelivery%2Bof%2BChimeric%2BBRD4%2BDegraders.%2BPart%2B2%253A%2BImprovement%2Bof%2BIn%2BVitro%2BAntiproliferation%2BActivity%2Band%2BIn%2BVivo%2BAntitumor%2BEfficacy%26numPages%3D32%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D2607%26publicationDate%3DMarch%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c01846"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2477</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">5</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c01846" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Antibody-Mediated Delivery of Chimeric BRD4 Degraders. Part 2: Improvement of In Vitro Antiproliferation Activity and In Vivo Antitumor Efficacy&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;S. Dragovich&quot;},{&quot;first_name&quot;:&quot;Thomas&quot;,&quot;last_name&quot;:&quot;H. Pillow&quot;},{&quot;first_name&quot;:&quot;Robert&quot;,&quot;last_name&quot;:&quot;A. Blake&quot;},{&quot;first_name&quot;:&quot;Jack&quot;,&quot;last_name&quot;:&quot;D. Sadowsky&quot;},{&quot;first_name&quot;:&quot;Emel&quot;,&quot;last_name&quot;:&quot;Adaligil&quot;},{&quot;first_name&quot;:&quot;Pragya&quot;,&quot;last_name&quot;:&quot;Adhikari&quot;},{&quot;first_name&quot;:&quot;Jinhua&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Nicholas&quot;,&quot;last_name&quot;:&quot;Corr&quot;},{&quot;first_name&quot;:&quot;Josefa&quot;,&quot;last_name&quot;:&quot;dela Cruz-Chuh&quot;},{&quot;first_name&quot;:&quot;Geoffrey&quot;,&quot;last_name&quot;:&quot;Del Rosario&quot;},{&quot;first_name&quot;:&quot;Aaron&quot;,&quot;last_name&quot;:&quot;Fullerton&quot;},{&quot;first_name&quot;:&quot;Steven&quot;,&quot;last_name&quot;:&quot;J. Hartman&quot;},{&quot;first_name&quot;:&quot;Fan&quot;,&quot;last_name&quot;:&quot;Jiang&quot;},{&quot;first_name&quot;:&quot;Susan&quot;,&quot;last_name&quot;:&quot;Kaufman&quot;},{&quot;first_name&quot;:&quot;Tracy&quot;,&quot;last_name&quot;:&quot;Kleinheinz&quot;},{&quot;first_name&quot;:&quot;Katherine&quot;,&quot;last_name&quot;:&quot;R. Kozak&quot;},{&quot;first_name&quot;:&quot;Liling&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Ying&quot;,&quot;last_name&quot;:&quot;Lu&quot;},{&quot;first_name&quot;:&quot;Melinda&quot;,&quot;last_name&quot;:&quot;M. Mulvihill&quot;},{&quot;first_name&quot;:&quot;Jeremy&quot;,&quot;last_name&quot;:&quot;M. Murray&quot;},{&quot;first_name&quot;:&quot;Aimee&quot;,&quot;last_name&quot;:&quot;O’Donohue&quot;},{&quot;first_name&quot;:&quot;Rebecca&quot;,&quot;last_name&quot;:&quot;K. Rowntree&quot;},{&quot;first_name&quot;:&quot;William&quot;,&quot;last_name&quot;:&quot;S. Sawyer&quot;},{&quot;first_name&quot;:&quot;Leanna&quot;,&quot;last_name&quot;:&quot;R. Staben&quot;},{&quot;first_name&quot;:&quot;John&quot;,&quot;last_name&quot;:&quot;Wai&quot;},{&quot;first_name&quot;:&quot;Jian&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;BinQing&quot;,&quot;last_name&quot;:&quot;Wei&quot;},{&quot;first_name&quot;:&quot;Wentao&quot;,&quot;last_name&quot;:&quot;Wei&quot;},{&quot;first_name&quot;:&quot;Zijin&quot;,&quot;last_name&quot;:&quot;Xu&quot;},{&quot;first_name&quot;:&quot;Hui&quot;,&quot;last_name&quot;:&quot;Yao&quot;},{&quot;first_name&quot;:&quot;Shang-Fan&quot;,&quot;last_name&quot;:&quot;Yu&quot;},{&quot;first_name&quot;:&quot;Donglu&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Hongyan&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Shenhua&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Yongxin&quot;,&quot;last_name&quot;:&quot;Zhao&quot;},{&quot;first_name&quot;:&quot;Hao&quot;,&quot;last_name&quot;:&quot;Zhou&quot;},{&quot;first_name&quot;:&quot;Xiaoyu&quot;,&quot;last_name&quot;:&quot;Zhu&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;02&quot;,&quot;day&quot;:&quot;17&quot;,&quot;issue&quot;:&quot;5&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;2576-2607&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c01846&quot;},&quot;abstract&quot;:&quot;Heterobifunctional compounds that direct the ubiquitination of intracellular proteins in a targeted manner via co-opted ubiquitin ligases have enormous potential to transform the field of medicinal chemistry. These chimeric molecules, often termed proteolysis-targeting chimeras (PROTACs) in the chemical literature, enable the controlled degradation of specific proteins via their direction to the cellular proteasome. In this report, we describe the second phase of our research focused on exploring antibody–drug conjugates (ADCs), which incorporate BRD4-targeting chimeric degrader entities. We employ a new BRD4-binding fragment in the construction of the chimeric ADC payloads that is significantly more potent than the corresponding entity utilized in our initial studies. The resulting BRD4-degrader antibody conjugates exhibit potent and antigen-dependent BRD4 degradation and antiproliferation activities in cell-based experiments. Multiple ADCs bearing chimeric BRD4-degrader payloads also exhibit strong, antigen&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01846&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01846" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01846&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01846" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01846&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01846" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01846&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01846&amp;href=/doi/10.1021/acs.jmedchem.0c01846" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c01846" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c01846" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01846&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c01846%26sid%3Dliteratum%253Aachs%26pmid%3D33596073%26genre%3Darticle%26aulast%3DDragovich%26date%3D2021%26atitle%3DAntibody-Mediated%2BDelivery%2Bof%2BChimeric%2BBRD4%2BDegraders.%2BPart%2B2%253A%2BImprovement%2Bof%2BIn%2BVitro%2BAntiproliferation%2BActivity%2Band%2BIn%2BVivo%2BAntitumor%2BEfficacy%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D5%26spage%3D2576%26epage%3D2607%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (4)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291781" title="Degradation">Degradation</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291880" title="Crystal structure">Crystal structure</a>,</li><li><a href="/action/doSearch?ConceptID=291754" title="Molecules">Molecules</a>,</li><li><a href="/action/doSearch?ConceptID=291290" title="Conjugate acid-base pairs">Conjugate acid-base pairs</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/5" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/jmcmar.2021.64.issue-5/20210311/jmcmar.2021.64.issue-5.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0032.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0032.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Heterobifunctional compounds that direct the ubiquitination of intracellular proteins in a targeted manner via co-opted ubiquitin ligases have enormous potential to transform the field of medicinal chemistry. These chimeric molecules, often termed proteolysis-targeting chimeras (PROTACs) in the chemical literature, enable the controlled degradation of specific proteins via their direction to the cellular proteasome. In this report, we describe the second phase of our research focused on exploring antibody–drug conjugates (ADCs), which incorporate BRD4-targeting chimeric degrader entities. We employ a new BRD4-binding fragment in the construction of the chimeric ADC payloads that is significantly more potent than the corresponding entity utilized in our initial studies. The resulting BRD4-degrader antibody conjugates exhibit potent and antigen-dependent BRD4 degradation and antiproliferation activities in cell-based experiments. Multiple ADCs bearing chimeric BRD4-degrader payloads also exhibit strong, antigen-dependent antitumor efficacy in mouse xenograft assessments that employ several different tumor models.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78468" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78468" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">BRD2, BRD3, BRD4, and BRDT comprise the bromodomain and extraterminal (BET) epigenetic “reader” family of proteins that recognize acetylated lysine residues present in histones.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Among these entities, BRD4 is currently of high interest as a medicinal chemistry target since disruption of histone interactions mediated by its BD1 and BD2 bromodomain motifs may afford novel treatments for cancer and several other diseases.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Accordingly, a number of BRD4/BET inhibitors have been clinically evaluated as novel oncology therapies.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Although encouraging efficacy outcomes were observed in some of these studies, unwanted side effects, including likely on-target thrombocytopenia, have limited the clinical progression of these agents.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a> Several next-generation BRD4 inhibitors with improved binding selectivities for the BD1 or BD2 bromodomains have recently been disclosed, and these entities may improve clinical outcomes relative to the agents studied to date.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> In spite of such progress, new approaches to modulate the BRD4/BET biological activity remain highly desired as a means to achieve additional clinical benefit beyond that demonstrated to date by existing inhibitors.</div><div class="NLM_p">The use of synthetic heterobifunctional molecules to induce degradation of targeted proteins via co-opted ubiquitin ligases in a predictable and controllable manner is an increasingly important component of biology and medicinal chemistry research activities.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Such molecules have been termed proteolysis-targeting chimeras (PROTACs),<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> SNIPERS (specific and nongenetic IAP-dependent protein erasers),<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> and degronimers in the chemical literature, but for consistency and simplicity purposes, these entities will be referred to as “chimeric degraders” or “degraders” in this work. The BET family was among the first group of proteins to be targeted with such chimeric molecules, and many examples of BET/BRD4 degraders are now known in the literature.<a onclick="showRef(event, 'cit8b ref9'); return false;" href="javascript:void(0);" class="ref cit8b ref9">(8b,9)</a> Importantly, degradation of BRD4 was shown to impart broader transcriptional impacts relative to simple inhibition, and chimeric BET/BRD4 degraders may therefore have the potential for greater therapeutic impacts as compared to traditional small-molecule inhibitors.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a></div><div class="NLM_p">Because of their heterobifunctional nature, chimeric protein degraders typically possess molecular weights and other physicochemical properties that differ considerably from those associated with orally bioavailable compounds.<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11,12)</a> These characteristics can also complicate in vivo delivery of the chimeric degrader molecules by nonoral administration routes. Accordingly, such entities are often administered to animals using formulations and/or dosing routes (e.g., intraperitoneal) that are not readily transferred to human clinical settings. Although some progress has been made in the generation of chimeric degraders with favorable pharmacokinetic properties (including several orally bioavailable examples),<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13,14)</a> additional research is required to enable the routine and reliable creation of such molecules that are suitable for in vivo use.</div><div class="NLM_p last">In an effort to provide an alternate means to administer chimeric degrader compounds in vivo, we sought to conjugate these entities to monoclonal antibodies using technologies originally developed for cytotoxic payloads.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> In this report, we describe our continued efforts to improve the antiproliferation properties and antitumor efficacies associated with STEAP1-targeting antibody–drug conjugates (ADCs) derived from chimeric BET/BRD4 degraders. This research builds on our initial explorations that are described in the preceding paper in this journal,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> and a preliminary disclosure of some of our results has been recently published.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> We also leveraged learnings from our related studies of conjugates bearing chimeric degraders of the estrogen receptor α (ERα) protein.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Other researchers have disclosed their own approaches for conjugating chimeric BRD4 degraders to antibodies and have thereby expanded this new and growing field.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> As part of our studies, we were curious to learn whether the described chimeric degrader conjugates offered improvements to the thrombocytopenia risks often associated with unconjugated BET/BRD4 inhibitors.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a> In addition, we typically did not assess the BET family degradation selectivity of the chimeric molecules described in this work. Accordingly, we refer to these entities as “BRD4 degraders” simply to reflect this lack of characterization and do not imply their ability to selective degrade BRD4 over other BET family members.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_58851" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_58851" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We concluded our initial explorations with a desire to identify more potent chimeric BRD4 degraders relative to those derived from <i>tert</i>-butyl (<i>S</i>)-2-(4-(4-chlorophenyl)-2,3,9-trimethyl-6<i>H</i>-thieno[3,2-<i>f</i>][1,2,4]triazolo[4,3-<i>a</i>][1,4]diazepin-6-yl)acetate<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> (compound <b>1</b>, JQ1; <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) as a means of improving the antiproliferation outcomes of the corresponding conjugates.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> As shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, the VHL ubiquitin ligase-dependent chimeric degraders <b>2</b> (MZ1)<a onclick="showRef(event, 'cit9d'); return false;" href="javascript:void(0);" class="ref cit9d">(9d)</a> and <b>3</b> that we previously studied were capable of potently (DC<sub>50</sub> value) and extensively (<i>S</i><sub>inf</sub> value) degrading the BRD4 protein in PC3 prostate cancer cells that had been engineered to express the STEAP1 antigen at a relatively high level (see ref <a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> for descriptions of STEAP1, the engineered PC3-S1 cell line, and the relationship between BRD4 and prostate cancer). However, although both <b>2</b> and <b>3</b> also inhibited PC3-S1 cell proliferation in vitro, the measured IC<sub>50</sub> values for the compounds (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) were considerably weaker than those associated with typical cytotoxic ADC payloads, which resided in the low nanomolar to picomolar range.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> As one means to improve these antiproliferation activities, we sought to create new chimeric molecules that incorporated BRD4-binding elements with greater affinities for the protein than that afforded by compound <b>1</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). This strategy would ideally enable the continued use of favorable VHL-binding fragments and spacer groups that were identified during the initial phase of our research.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> It should also be noted that, while several extremely potent chimeric BRD4 degraders are now known,<a onclick="showRef(event, 'cit9a cit9b'); return false;" href="javascript:void(0);" class="ref cit9a cit9b">(9a,b)</a> these entities were not yet disclosed in the scientific literature at the time we conducted our research.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0002.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of compound <b>1</b> (JQ1) and related chimeric BRD4 degraders <b>2</b> (MZ1) and <b>3</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Biological Activities of Unconjugated BRD4 Inhibitors and Chimeric Degraders Described in This Work</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">cmpd</th><th class="colsep0 rowsep0" align="center">Log <i>k</i> IAM<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">BRD4 BD2 IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">VHL FP IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center">ternary complex SPR <i>T</i><sub>1/2</sub> (min)<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a></th><th class="colsep0 rowsep0" align="center">MYC IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t1fn5" aria-label="e">e</a></th><th class="colsep0 rowsep0" align="center">MYC max inhib (%)<a class="ref internalNav" href="#t1fn6" aria-label="f">f</a></th><th class="colsep0 rowsep0" align="center">PC3-S1 BRD4 DC<sub>50</sub> (nM)<a class="ref internalNav" href="#t1fn7" aria-label="g">g</a></th><th class="colsep0 rowsep0" align="center">PC3-S1 BRD4 <i>S</i><sub>inf</sub> (%)<a class="ref internalNav" href="#t1fn8" aria-label="h">h</a></th><th class="colsep0 rowsep0" align="center">PC3-S1 IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t1fn9" aria-label="i">i</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">27</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">79</td><td class="colsep0 rowsep0" align="left">76</td><td class="colsep0 rowsep0" align="left">>20 000</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">380</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="left">1.71</td><td class="colsep0 rowsep0" align="left">19</td><td class="colsep0 rowsep0" align="left">310</td><td class="colsep0 rowsep0" align="left">199<a class="ref internalNav" href="#t1fn10" aria-label="j">j</a></td><td class="colsep0 rowsep0" align="left">2.2</td><td class="colsep0 rowsep0" align="left">98</td><td class="colsep0 rowsep0" align="left">23</td><td class="colsep0 rowsep0" align="left">97</td><td class="colsep0 rowsep0" align="left">140</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">69</td><td class="colsep0 rowsep0" align="left">135</td><td class="colsep0 rowsep0" align="left">250<a class="ref internalNav" href="#t1fn10" aria-label="j">j</a></td><td class="colsep0 rowsep0" align="left">86</td><td class="colsep0 rowsep0" align="left">97</td><td class="colsep0 rowsep0" align="left">85</td><td class="colsep0 rowsep0" align="left">98</td><td class="colsep0 rowsep0" align="left">1300</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">1.7</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">3.3</td><td class="colsep0 rowsep0" align="left">81</td><td class="colsep0 rowsep0" align="left">>20 000</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">53</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">0.98</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">0.42</td><td class="colsep0 rowsep0" align="left">84</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">3.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">1.7</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">0.40</td><td class="colsep0 rowsep0" align="left">81</td><td class="colsep0 rowsep0" align="left">>20 000</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">3.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="left">1.32</td><td class="colsep0 rowsep0" align="left">1.4</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">77</td><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">98</td><td class="colsep0 rowsep0" align="left">7.5</td><td class="colsep0 rowsep0" align="left">98</td><td class="colsep0 rowsep0" align="left">28</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="left">2.08</td><td class="colsep0 rowsep0" align="left">4.3</td><td class="colsep0 rowsep0" align="left">613</td><td class="colsep0 rowsep0" align="left">3959</td><td class="colsep0 rowsep0" align="left">0.026</td><td class="colsep0 rowsep0" align="left">98</td><td class="colsep0 rowsep0" align="left">0.095</td><td class="colsep0 rowsep0" align="left">96</td><td class="colsep0 rowsep0" align="left">0.63</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="left">2.04</td><td class="colsep0 rowsep0" align="left">3.9</td><td class="colsep0 rowsep0" align="left">>200 000</td><td class="colsep0 rowsep0" align="left">0<a class="ref internalNav" href="#t1fn11" aria-label="k">k</a></td><td class="colsep0 rowsep0" align="left">0.65</td><td class="colsep0 rowsep0" align="left">83</td><td class="colsep0 rowsep0" align="left">>20 000</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">41</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="left">2.18</td><td class="colsep0 rowsep0" align="left">1.6</td><td class="colsep0 rowsep0" align="left">383</td><td class="colsep0 rowsep0" align="left">71</td><td class="colsep0 rowsep0" align="left">770</td><td class="colsep0 rowsep0" align="left">81</td><td class="colsep0 rowsep0" align="left">180</td><td class="colsep0 rowsep0" align="left">99</td><td class="colsep0 rowsep0" align="left">220</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b></td><td class="colsep0 rowsep0" align="left">3.15</td><td class="colsep0 rowsep0" align="left">2.7</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">6562</td><td class="colsep0 rowsep0" align="left">3.0</td><td class="colsep0 rowsep0" align="left">99</td><td class="colsep0 rowsep0" align="left">3.4</td><td class="colsep0 rowsep0" align="left">98</td><td class="colsep0 rowsep0" align="left">40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="left">1.52</td><td class="colsep0 rowsep0" align="left">8.1</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">650</td><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" align="left">98</td><td class="colsep0 rowsep0" align="left">2.1<a class="ref internalNav" href="#t1fn12" aria-label="l">l</a></td><td class="colsep0 rowsep0" align="left">98<a class="ref internalNav" href="#t1fn12" aria-label="l">l</a></td><td class="colsep0 rowsep0" align="left">91</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="left">1.59</td><td class="colsep0 rowsep0" align="left">1.7</td><td class="colsep0 rowsep0" align="left">229</td><td class="colsep0 rowsep0" align="left">742</td><td class="colsep0 rowsep0" align="left">1.9</td><td class="colsep0 rowsep0" align="left">98</td><td class="colsep0 rowsep0" align="left">1.4</td><td class="colsep0 rowsep0" align="left">100</td><td class="colsep0 rowsep0" align="left">10</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Cell membrane permeability assessment (larger values predict increased intracellular compound concentrations).</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">BRD4 BD2 bromodomain binding affinity; for BD1 binding affinities, see <a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Table S1</a>.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">VHL-binding affinity.</p></div><div class="footnote" id="t1fn4"><sup><sup>d</sup></sup><p class="last">SPR half-life of the BRD4/VHL/compound ternary complex (sequential injection; sum of half-lives determined using BRD4 BD1 and BD2 bromodomains; see <a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Table S4</a> for individual values).</p></div><div class="footnote" id="t1fn5"><sup><sup>e</sup></sup><p class="last">Concentration that suppresses MYC gene transcript 50% in MV4-11 AML cells; time point was 4 h.</p></div><div class="footnote" id="t1fn6"><sup><sup>f</sup></sup><p class="last">Maximum suppression of MYC gene transcript.</p></div><div class="footnote" id="t1fn7"><sup><sup>g</sup></sup><p class="last">Concentration that achieves 50% degradation of BRD4; time point was 4 h.</p></div><div class="footnote" id="t1fn8"><sup><sup>h</sup></sup><p class="last">Maximum reduction of BRD4 protein levels relative to DMSO-treated controls.</p></div><div class="footnote" id="t1fn9"><sup><sup>i</sup></sup><p class="last">Antiproliferation activity; time point was 6 days.</p></div><div class="footnote" id="t1fn10"><sup><sup>j</sup></sup><p class="last">Measured using alternate SPR assay conditions (coinjection; see ref <a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a>).</p></div><div class="footnote" id="t1fn11"><sup><sup>k</sup></sup><p class="last">No ternary complex detected.</p></div><div class="footnote" id="t1fn12"><sup><sup>l</sup></sup><p class="last">Measured in EOL-1 cells. NA, not applicable; ND, not determined. See the Supporting Information for a discussion on the error associated with each assay. Full chemical structures are depicted in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Table S7</a>.</p></div></div></div><div class="NLM_p">With the above objective in mind, we were intrigued by compound <b>4</b> and related molecules, which have been described in the literature to exhibit BRD4-binding properties that were significantly superior to <b>1</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>).<a onclick="showRef(event, 'cit2c ref21'); return false;" href="javascript:void(0);" class="ref cit2c ref21">(2c,21)</a> Our modeling studies suggested that <b>4</b> bound to BRD4 in a manner that exposed the aryl hydrogen adjacent to the sulfone moiety and para to the fused pyridone group toward the solvent and thus provided a vector that could potentially be utilized for the creation of chimeric degrader molecules (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a> and <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0003.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Structures of BRD4-binding compounds <b>4</b>–<b>7</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0004.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Crystal structure of compound <b>1</b> (cyan) bound to the BD1 bromodomain of BRD4 with the docked compound <b>4</b> (pink) superimposed. The yellow arrow indicates a vector from the docked compound <b>4</b> that protrudes away from the protein and out into the solvent. The orange circles depict the locations in compound <b>1</b> that were derivatized in our previous work to generate functional chimeric BRD4 degraders (ref <a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a>). The figure was created using the published structure: PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MXF">3MXF</a> (ref <a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">However, our modeling also indicated that this vector was oriented somewhat differently from those utilized to generate BRD4 degraders from compound <b>1</b> in the initial phase of our research (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, orange circles). Thus, at the outset of this part of our new studies, we did not know with certainty whether biologically active chimeric BRD4 degraders could be generated from the more-potent BRD4-binding fragment.</div><div class="NLM_p">As shown in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>, we prepared compounds <b>5</b>–<b>7</b>, which incorporated various functional groups at the solvent-exposed position of compound <b>4</b> that could subsequently be used to generate chimeric compounds. Encouragingly, all of these new molecules displayed potent BRD4-binding affinities that were dramatically improved relative to compound <b>1</b> and chimeric degraders <b>2</b> and <b>3</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>; the respective affinities of <b>5</b>–<b>7</b> for the BRD4 BD1 and BD2 bromodomains were nearly identical; see <a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Table S1</a>). As a likely result, compounds <b>5</b>–<b>7</b> were much more potent than <b>1</b> in cell-based assessments of MYC gene transcript suppression and antiproliferation properties (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>; see ref <a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> for information regarding the relationship between BRD4 inhibition and suppression of MYC gene transcription). In addition, a cocrystal structure of compound <b>6</b> bound to the BRD4 BD1 bromodomain was obtained, which confirmed the desired binding mode and resulting solvent exposure of the appended ester moiety (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Figures S1 and S2</a>; crystal structures of related molecules in complex with BRD4 have also recently been disclosed by others<a onclick="showRef(event, 'cit21a'); return false;" href="javascript:void(0);" class="ref cit21a">(21a)</a>).</div><div class="NLM_p">With the above information in hand, we transformed inhibitor <b>5</b> into the chimeric molecule <b>8</b> by utilizing the former compound’s amide moiety to attach the PEG spacer and VHL-binding fragment present in degrader <b>2</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>; cf., <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). As shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, <b>8</b> is an active and extensive BRD4 degrader with a DC<sub>50</sub> potency that improved on those displayed by compounds <b>2</b> and <b>3</b>. Encouragingly, the PC3-S1 antiproliferation properties exhibited by <b>8</b> were also improved relative to the latter two compounds, and this result supported further exploration of the new chimeric BRD4-degrader series derived from the more-potent BRD4-binding group (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). These cell-based outcomes were especially notable since compound <b>8</b> was predicted by immobilized artificial membrane (IAM) assessment to be somewhat less cell-permeable relative to degrader <b>2</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, larger log <i>k</i> IAM values = higher predicted cell permeability; see ref <a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> and corresponding IAM discussion in ref <a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> for additional details).</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0005.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Structures of chimeric BRD4-degrader <b>8</b> and corresponding linker-drugs.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We next constructed the antibody conjugates STEAP1–<b>8a</b> and STEAP1–<b>8b</b>, which incorporated chimeric degrader <b>8</b> using several glutathione-cleavable, disulfide-containing linkers that were previously employed during the initial phase of our research.<a onclick="showRef(event, 'ref16 ref23'); return false;" href="javascript:void(0);" class="ref ref16 ref23">(16,23)</a> As was performed in our earlier studies, these ADCs were prepared by reaction of STEAP1 antibodies containing engineered Cys residues at the LC-K149, HC-L174, and HC-Y373 sites with linker-drugs <b>8a</b> and <b>8b</b> bearing methanethiosulfonate (MTS) moieties (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>) to afford conjugates with drug/antibody ratios (DAR values) of close to 6 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Scheme S1</a>).<a onclick="showRef(event, 'ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref24 ref25">(24,25)</a> These conjugates were designed to release the attached degrader payloads following internalization into targeted cells, antibody catabolism, and subsequent cleavage of the disulfide-based linkers (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Scheme S2</a>).<a onclick="showRef(event, 'ref16 ref23'); return false;" href="javascript:void(0);" class="ref ref16 ref23">(16,23)</a> The engineered Cys mAb locations were shown in our previous studies of related cytotoxic ADCs to afford relatively stable attachments when conjugated to linker-drugs derived from maleimide or disulfide-based electrophiles.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> In addition, the DAR6 payload loading was demonstrated in the initial phase of our work to afford improved BRD4 degradation outcomes relative to more-traditional DAR2 ADCs.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> We therefore wished to maintain the higher drug/antibody ratio whenever possible during this portion of our assessments. Accordingly, unless otherwise indicated, the conjugates described in this work are DAR6 entities with additional descriptors included as appropriate to differentiate DAR6 (“D6”) ADCs from closely related DAR2 (“D2”) analogues. Finally, since conjugates derived from related carbonate-containing, disulfide-based linker-drugs were shown in the first phase of our research to exhibit favorable in vitro and/or in vivo stability properties, we were comfortable continuing to utilize such modalities in this portion of our studies.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a></div><div class="NLM_p">As shown in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, the STEAP1–<b>8a</b> and STEAP1–<b>8b</b> conjugates displayed potent and near-complete BRD4 degradation properties when tested using the PC3-S1 cell line. They also afforded measurable PC3-S1 antiproliferation effects that, although weaker than the corresponding BRD4 DC<sub>50</sub> vales, were nevertheless reasonably potent (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). However, all of the in vitro biological activities exhibited by STEAP1–<b>8a</b> and STEAP1–<b>8b</b> were not well-separated from those afforded by the corresponding control ADCs CLL1–<b>8a</b> and CLL1–<b>8b</b>, which targeted an antigen that was not highly expressed on the surface of the PC3-S1 cells (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>; see ref <a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> for additional discussion regarding the CLL1 antigen). As mentioned in the description of the initial phase of our research, these outcomes may be due to partial cleavage of the disulfide-containing ADC linkers under the culture conditions employed for the cell-based assay.<a onclick="showRef(event, 'ref16 ref27'); return false;" href="javascript:void(0);" class="ref ref16 ref27">(16,27)</a></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Biological Activities of Chimeric Degrader Conjugates Described in This Work</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0031.gif" alt="" id="fx1" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">conjugate</th><th class="colsep0 rowsep0" align="center">DAR<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">PC3-S1 BRD4 DC<sub>50</sub> (μg/mL)<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">PC3-S1 BRD4 DC<sub>50</sub> (nM)<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center">PC3-S1 BRD4 <i>S</i><sub>inf</sub> (%)<a class="ref internalNav" href="#t2fn4" aria-label="d">d</a></th><th class="colsep0 rowsep0" align="center">DC<sub>50</sub> ratio<a class="ref internalNav" href="#t2fn5" aria-label="e">e</a></th><th class="colsep0 rowsep0" align="center">PC3-S1 IC<sub>50</sub> (μg/mL)<a class="ref internalNav" href="#t2fn6" aria-label="f">f</a></th><th class="colsep0 rowsep0" align="center">PC3-S1 IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t2fn7" aria-label="g">g</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">STEAP1–<b>8a</b></td><td class="colsep0 rowsep0" align="left">5.9</td><td class="colsep0 rowsep0" align="left">0.022</td><td class="colsep0 rowsep0" align="left">0.86</td><td class="colsep0 rowsep0" align="left">94</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">1.5</td><td class="colsep0 rowsep0" align="left">61</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CLL1–<b>8a</b></td><td class="colsep0 rowsep0" align="left">5.9</td><td class="colsep0 rowsep0" align="left">0.19</td><td class="colsep0 rowsep0" align="left">7.6</td><td class="colsep0 rowsep0" align="left">94</td><td class="colsep0 rowsep0" align="left">8.6</td><td class="colsep0 rowsep0" align="left">5.4</td><td class="colsep0 rowsep0" align="left">211</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">STEAP1–<b>8b</b></td><td class="colsep0 rowsep0" align="left">5.9</td><td class="colsep0 rowsep0" align="left">0.034</td><td class="colsep0 rowsep0" align="left">1.3</td><td class="colsep0 rowsep0" align="left">96</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">1.8</td><td class="colsep0 rowsep0" align="left">70</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CLL1–<b>8b</b></td><td class="colsep0 rowsep0" align="left">5.9</td><td class="colsep0 rowsep0" align="left">0.46</td><td class="colsep0 rowsep0" align="left">18</td><td class="colsep0 rowsep0" align="left">93</td><td class="colsep0 rowsep0" align="left">13</td><td class="colsep0 rowsep0" align="left">>20</td><td class="colsep0 rowsep0" align="left">>780</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">STEAP1–<b>9a</b>–D2</td><td class="colsep0 rowsep0" align="left">2.0</td><td class="colsep0 rowsep0" align="left">0.017</td><td class="colsep0 rowsep0" align="left">0.23</td><td class="colsep0 rowsep0" align="left">96</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">0.86</td><td class="colsep0 rowsep0" align="left">11</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CLL1–<b>9a</b>–D2</td><td class="colsep0 rowsep0" align="left">2.0</td><td class="colsep0 rowsep0" align="left">0.029</td><td class="colsep0 rowsep0" align="left">0.38</td><td class="colsep0 rowsep0" align="left">96</td><td class="colsep0 rowsep0" align="left">1.7</td><td class="colsep0 rowsep0" align="left">1.0</td><td class="colsep0 rowsep0" align="left">13</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">STEAP1–<b>9b</b>–D2</td><td class="colsep0 rowsep0" align="left">1.9</td><td class="colsep0 rowsep0" align="left">0.034</td><td class="colsep0 rowsep0" align="left">0.44</td><td class="colsep0 rowsep0" align="left">96</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">0.69</td><td class="colsep0 rowsep0" align="left">8.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CLL1–<b>9b</b>–D2</td><td class="colsep0 rowsep0" align="left">2.0</td><td class="colsep0 rowsep0" align="left">0.061</td><td class="colsep0 rowsep0" align="left">0.81</td><td class="colsep0 rowsep0" align="left">95</td><td class="colsep0 rowsep0" align="left">1.8</td><td class="colsep0 rowsep0" align="left">0.96</td><td class="colsep0 rowsep0" align="left">13</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CLL1–<b>9b</b>–D6</td><td class="colsep0 rowsep0" align="left">5.5</td><td class="colsep0 rowsep0" align="left">0.010</td><td class="colsep0 rowsep0" align="left">0.37</td><td class="colsep0 rowsep0" align="left">92</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">0.52</td><td class="colsep0 rowsep0" align="left">19</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HER2–<b>9b</b>–D6</td><td class="colsep0 rowsep0" align="left">5.5</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">0.39</td><td class="colsep0 rowsep0" align="left">14</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">STEAP1–<b>9c–</b>D2</td><td class="colsep0 rowsep0" align="left">2.0</td><td class="colsep0 rowsep0" align="left">0.030</td><td class="colsep0 rowsep0" align="left">0.39</td><td class="colsep0 rowsep0" align="left">97</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">1.5</td><td class="colsep0 rowsep0" align="left">20</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CLL1–<b>9c</b>–D2</td><td class="colsep0 rowsep0" align="left">1.9</td><td class="colsep0 rowsep0" align="left">0.019</td><td class="colsep0 rowsep0" align="left">0.24</td><td class="colsep0 rowsep0" align="left">96</td><td class="colsep0 rowsep0" align="left">0.63</td><td class="colsep0 rowsep0" align="left">0.97</td><td class="colsep0 rowsep0" align="left">12</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">STEAP1–<b>9d</b></td><td class="colsep0 rowsep0" align="left">5.9</td><td class="colsep0 rowsep0" align="left">0.00063</td><td class="colsep0 rowsep0" align="left">0.025</td><td class="colsep0 rowsep0" align="left">92</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">0.16</td><td class="colsep0 rowsep0" align="left">6.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CLL1–<b>9d</b></td><td class="colsep0 rowsep0" align="left">6.0</td><td class="colsep0 rowsep0" align="left">0.15</td><td class="colsep0 rowsep0" align="left">6.0</td><td class="colsep0 rowsep0" align="left">95</td><td class="colsep0 rowsep0" align="left">238</td><td class="colsep0 rowsep0" align="left">17</td><td class="colsep0 rowsep0" align="left">670</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">STEAP1–<b>11a</b></td><td class="colsep0 rowsep0" align="left">6.1</td><td class="colsep0 rowsep0" align="left">0.011</td><td class="colsep0 rowsep0" align="left">0.46</td><td class="colsep0 rowsep0" align="left">90</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">0.19</td><td class="colsep0 rowsep0" align="left">7.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CLL1–<b>11a</b></td><td class="colsep0 rowsep0" align="left">6.0</td><td class="colsep0 rowsep0" align="left">5.1</td><td class="colsep0 rowsep0" align="left">205</td><td class="colsep0 rowsep0" align="left">84</td><td class="colsep0 rowsep0" align="left">460</td><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">448</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">STEAP1–<b>12a–</b>D2</td><td class="colsep0 rowsep0" align="left">1.9</td><td class="colsep0 rowsep0" align="left">0.11</td><td class="colsep0 rowsep0" align="left">1.4</td><td class="colsep0 rowsep0" align="left">87</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">>20</td><td class="colsep0 rowsep0" align="left">>250</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CLL1–<b>12a</b>–D2</td><td class="colsep0 rowsep0" align="left">2.0</td><td class="colsep0 rowsep0" align="left">6.7</td><td class="colsep0 rowsep0" align="left">88</td><td class="colsep0 rowsep0" align="left">88</td><td class="colsep0 rowsep0" align="left">61</td><td class="colsep0 rowsep0" align="left">>20</td><td class="colsep0 rowsep0" align="left">>250</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">STEAP1–<b>13a</b></td><td class="colsep0 rowsep0" align="left">5.9</td><td class="colsep0 rowsep0" align="left">0.04</td><td class="colsep0 rowsep0" align="left">1.6</td><td class="colsep0 rowsep0" align="left">90</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">0.74</td><td class="colsep0 rowsep0" align="left">29</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CLL1–<b>13a</b></td><td class="colsep0 rowsep0" align="left">5.8</td><td class="colsep0 rowsep0" align="left">8.0</td><td class="colsep0 rowsep0" align="left">313</td><td class="colsep0 rowsep0" align="left">89</td><td class="colsep0 rowsep0" align="left">200</td><td class="colsep0 rowsep0" align="left">>20</td><td class="colsep0 rowsep0" align="left">>772</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CLL1–<b>14a</b><a class="ref internalNav" href="#t2fn8" aria-label="h">h</a></td><td class="colsep0 rowsep0" align="left">5.9</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">ND</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">STEAP1–<b>15a</b></td><td class="colsep0 rowsep0" align="left">5.8</td><td class="colsep0 rowsep0" align="left">0.0066</td><td class="colsep0 rowsep0" align="left">0.26</td><td class="colsep0 rowsep0" align="left">94</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">2.8</td><td class="colsep0 rowsep0" align="left">108</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CLL1–<b>15a</b></td><td class="colsep0 rowsep0" align="left">5.8</td><td class="colsep0 rowsep0" align="left">14</td><td class="colsep0 rowsep0" align="left">549</td><td class="colsep0 rowsep0" align="left">75</td><td class="colsep0 rowsep0" align="left">2100</td><td class="colsep0 rowsep0" align="left">>20</td><td class="colsep0 rowsep0" align="left">>775</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Drug-to-antibody ratio.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">Absolute concentration of conjugate that achieves 50% degradation of BRD4; time point was 72 h.</p></div><div class="footnote" id="t2fn3"><sup><sup>c</sup></sup><p class="last">Concentration of conjugated chimeric degrader that achieves 50% BRD4 degradation (drug-equivalent units; see the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Supporting Information</a> for unit conversion details).</p></div><div class="footnote" id="t2fn4"><sup><sup>d</sup></sup><p class="last">Maximum degradation of BRD4 protein levels relative to DMSO-treated controls.</p></div><div class="footnote" id="t2fn5"><sup><sup>e</sup></sup><p class="last">Ratio of CLL1 and STEAP1 DC<sub>50</sub> values for the given linker-drug and DAR.</p></div><div class="footnote" id="t2fn6"><sup><sup>f</sup></sup><p class="last">Antiproliferation activity of the conjugate, absolute concentration; time point was 6 days.</p></div><div class="footnote" id="t2fn7"><sup><sup>g</sup></sup><p class="last">Antiproliferation activity, conjugated chimeric degrader drug-equivalents.</p></div><div class="footnote" id="t2fn8"><sup><sup>h</sup></sup><p class="last">Prepared as a control for in vivo studies and therefore not tested in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> assays. NA, not applicable; ND, not determined. See the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Supporting Information</a> for a discussion on the error associated with each assay. Full chemical structures are depicted in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Table S7</a>.</p></div></div></div><div class="NLM_p">In an effort to improve the potency of chimeric degrader <b>8</b>, we replaced the central PEG moiety present in the compound with an all-carbon spacer group. The resulting molecule (<b>9</b>, GNE-987,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a><a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>) displayed dramatically improved anti-MYC and PC3-S1 BRD4 degradation properties relative to compound <b>8</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Notably, these improvements were realized even though degrader <b>9</b> bound to BRD4 with an affinity that was very similar to that exhibited by <b>8</b> and also associated with the VHL protein in a manner comparable to compound <b>2</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). As was performed in the first part of our research,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> we utilized surface plasmon resonance (SPR) techniques to characterize the ternary complexes formed among BRD4, VHL, and degraders <b>8</b> and <b>9</b> to better understand the compounds’ biological potency differences.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Degrader <b>9</b> exhibited an extremely long ternary complex half-life in the SPR assessments that was driven primarily by interactions with the BRD4 BD1 bromodomain (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">S4</a>; see also ref <a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a>). In contrast, the half-life of the complex formed among BRD4, VHL, and degrader <b>8</b> was much shorter, and this difference helped explain the significantly weaker BRD4 degradation properties exhibited by the compound relative to <b>9</b>. We also confirmed that inversion of the secondary hydroxyl group present in <b>9</b> abrogated binding to VHL and thereby prevented formation of a ternary complex with BRD4 and VHL (compound <b>10</b>, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>; see also ref <a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a>). The anti-MYC and PC3-S1 antiproliferation properties associated with <b>10</b> were similar to those displayed by the potent BRD4-binding compounds <b>5</b>–<b>7</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) and were thus consistent with all of the compounds containing similar BRD4-binding moieties. These results helped quantitate the additional biological activities that resulted from the ability of compound <b>9</b> to degrade BRD4 as opposed to simply inhibiting the protein via a potent binding element. As was observed in the first part of our research,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> a reasonably good correlation was noted between the BRD4/VHL ternary complex half-lives measured for the chimeric degraders of the present work and their corresponding abilities to degrade BRD4 in PC3-S1 cells (especially if compound <b>12</b> was removed from the analysis; <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">S8</a>). As was also the case during our prior explorations, this trend was not fully apparent until a majority of the new chimeric molecules had been profiled in the related biological assessments. Additional discussion regarding the SPR performance of degraders other than <b>8</b> and <b>9</b> is provided in subsequent sections of this work. Importantly, the more potent BRD4 affinities of the compounds associated with this phase of our research enabled/required the use of a sequential injection SPR method (degrader followed by VHL protein, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Figure S7</a>) that differed from the coinjection approach employed in our previous studies.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> In spite of these assay differences, it was still possible to meaningfully compare half-life results from the earlier SPR testing with the ternary complex data generated in this work.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0006.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Structures of chimeric BRD4 degraders <b>9</b> and <b>10</b> and corresponding linker-drugs.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0007.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Correlation between SPR half-life of the ternary complex formed among the listed chimeric degrader molecules, BRD4, and VHL and PC3-S1 BRD4 degradation pDC<sub>50</sub> values. To generate the values depicted on the <i>x</i>-axis, the SPR half-lives determined independently for each compound with the BRD4 BD1 and BD2 bromodomains (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Table S4</a>) were added together. The number next to each point indicates the compound number from <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. *BRD4 degradation properties of <b>13</b> were determined in the EOL-1 cell line. <i>R</i><sup>2</sup> = 0.19 (improves to 0.66 if compound <b>12</b> is removed from the regression analysis).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We were intrigued by the dramatic improvements in multiple biological potencies exhibited by compound <b>9</b> relative to degrader <b>8</b> that appeared to result from the simple replacement of three oxygen atoms in the spacer region of the latter molecule with methylene moieties. Accordingly, we obtained a crystal structure of the ternary complex formed among BRD4 BD1, compound <b>9</b>, and the VHL protein to better understand the origins of these potency differences. As shown in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>, the new crystal structure confirmed that the BRD4-binding moiety present in <b>9</b> recapitulated the binding interactions observed in the BRD4-compound <b>6</b> crystal structure (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Figures S1 and S2</a>; see also <a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Figure S3</a>). The BRD4 Asn140 side chain formed a donor/acceptor interaction with the pyrrolopyridone of <b>9</b>, while the compound’s difluoropyridine moiety bound close to Trp81 of the BRD4 WPF motif. In addition, the sulfone group contained in <b>9</b> interacted with the backbone amide nitrogen of the BRD4 Asp88 residue. The attachment site for the all-carbon linker was positioned at the interface of the BRD4 and VHL-binding sites with the carbon linker atoms snaking in between the interface formed between the two proteins. Interestingly, the VHL-binding moiety of <b>9</b> adopted a unique conformation in which the hydroxyproline was anchored by canonical interactions with VHL residues Ser111 and His115 and the benzyl-amide −NH of <b>9</b> interacted with the VHL His110 carbonyl group. However, the phenylmethylthiazole present in <b>9</b> projected away from a hydrophobic VHL-binding pocket created by the movement of the Arg107 side chain in which it was previously observed to reside (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Figure S4</a>).<a onclick="showRef(event, 'cit9d ref28 ref29'); return false;" href="javascript:void(0);" class="ref cit9d ref28 ref29">(9d,28,29)</a> In this novel conformation, the phenyl portion of the phenylmethylthiazole moiety formed pi–pi stacking interactions with the VHL His110 side chain, while the methylthiazole was largely solvent-exposed (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>). These differences resulted in a less buried surface area noted in the BRD4 BD1/<b>9</b>/VHL crystal structure relative to that observed previously in the structure of the BRD4 BD2/<b>2</b>/VHL ternary complex (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Table S2 and Figure S4</a>).<a onclick="showRef(event, 'cit29b'); return false;" href="javascript:void(0);" class="ref cit29b">(29b)</a> In addition, the relative orientation of the BRD4 BD1 bromodomain and the VHL protein observed in the BRD4/<b>9</b>/VHL complex differed considerably from that noted for BRD4 BD2 and VHL in related ternary structures.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Extensive discussion regarding these differences is beyond the scope of the present work, and the observation will therefore be more completely described in a separate manuscript that is currently in preparation.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0008.jpeg" id="gr16" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Crystal structure of the ternary complex formed among VHL, BRD4 BD1, and compound <b>9</b>. The VHL protein is shown in cyan with interacting residues depicted as cyan sticks. BRD4 BD1 is shown in green with interacting residues depicted as green sticks. Compound <b>9</b> is shown as salmon sticks. Water molecules in the vicinity of the BRD4–VHL interface are represented as red spheres. Hydrogen-bond interactions are depicted as follows: black dashed lines, intermolecular between <b>9</b> and either BRD4 or VHL; red dashed lines, intermolecular between VHL and BRD4; yellow dashed lines, intramolecular that stabilizes the conformation of VHL R107. Resolution is 2.28 Å, and PDB code is <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7KHH">7KHH</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Interaction analysis of the BRD4/<b>9</b>/VHL crystal structure demonstrated that the all-carbon spacer present in the chimeric degrader was solvent-exposed and did not make significant contacts with either protein (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Figure S5</a>). This outcome indicated that the improved ternary complex half-life exhibited by <b>9</b> relative to the PEG-containing degrader <b>8</b> did not result from more favorable interactions of the former molecule with the BRD4 and/or VHL proteins. Instead, the analysis suggested that the improved half-life of <b>9</b> was likely due to hydrophobic collapse involving the compound’s <i>tert</i>-butyl moiety, the all-carbon spacer methylene units, and the aryl rings present in the molecule’s BRD4-binding fragment (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Figure S6</a>). Such a collapse helped stabilize a complementary protein–protein interaction interface between BRD4 and VHL, which was further strengthened by (1) a salt bridge interaction between Asp145 of BRD4 and R107 of VHL and (2) a H-bond interaction between the indole NH of BRD4 Trp81 and the carbonyl oxygen of VHL Pro97 (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>). We also note, however, that other explanations of half-life improvement are possible and additional structural and biophysical studies are required to fully define the phenomenon (e.g., obtaining crystal structures of the ternary complexes formed among VHL, compound <b>9</b>, and the BRD4 BD2 bromodomain and/or the full-length BRD4 protein).</div><div class="NLM_p">We next attempted to prepare antibody–drug conjugates that incorporated the highly potent chimeric BRD4 degrader <b>9</b> using disulfide-containing linkers that were similar (<b>9a</b>) or identical (<b>9b</b> and <b>9c</b>) to those employed to create ADCs from compound <b>8</b> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>; cf., <b>8a</b> and <b>8b</b>; <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). Disappointingly, DAR6 STEAP1-targeting conjugates could not be prepared from any of the <b>9a</b>, <b>9b</b>, or <b>9c</b> linker-drugs due to extensive aggregation of the corresponding ADCs that prevented their isolation in forms suitable for biological testing. These outcomes likely resulted from the increased lipophilicity of the all-carbon spacer present in <b>9</b> and related molecules relative to the corresponding PEG-containing entities (cf., <b>8</b>, <b>8a</b>, and <b>8b</b>; <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). Interestingly, a DAR6 CLL1-targeting conjugate with acceptable aggregation properties could be generated using the linker-drug <b>9b</b>, and the in vivo activities of the resulting ADC (CLL1–<b>9b</b>–D6) against mouse xenograft models of acute myeloid leukemia were previously disclosed (the corresponding DAR6 HER2–<b>9b</b>–D6 control conjugate was also successfully prepared).<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> However, similar attempts to prepare DAR6 CLL1-targeting ADCs using linker-drugs <b>9a</b> and <b>9c</b> once again resulted in significant aggregation issues. Collectively, these results illustrated how attempted conjugation of <b>9a</b>, <b>9b</b>, and/or <b>9c</b> was highly sensitive to both the antigen targeted by the engineered monoclonal antibody and the location of the methyl groups present in the disulfide-containing antibody linkers. Additional experiments are currently underway in our laboratories to better understand and control these aggregation phenomena.</div><div class="NLM_p">In one approach to address the above aggregation issues, we prepared DAR2 STEAP1–mAb conjugates from <b>9a</b>, <b>9b</b>, and <b>9c</b> via the LC-K149 mAb attachment site in lieu of the higher DAR entities that were originally sought. These conjugations proceeded uneventfully and provided the corresponding ADCs in good yields and with acceptable aggregation characteristics (STEAP1–<b>9a</b>–D2, STEAP1–<b>9b–</b>D2, and STEAP1–<b>9c–</b>D2; see <a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Table S9</a> for additional characterization information). Although this strategy diverged from our desire to maintain DAR6 values in this phase of our research, we hoped that the improved cell-based potencies of the new degraders relative to those employed in our prior work would compensate for the lower payload loadings. As shown in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, all of these conjugates exhibited highly potent BRD4 degradation and antiproliferation properties in PC3-S1 assessments. However, as was observed above for the related disulfide-linked ADCs STEAP1–<b>8a</b> and STEAP1–<b>8b</b>, these biological activities were poorly separated from those associated with the corresponding nontargeting control conjugates (CLL1–<b>9a</b>–D2, CLL1–<b>9b</b>–D2, and CLL1–<b>9c</b>–D2).</div><div class="NLM_p">As an alternate means to create DAR6 ADCs that incorporated chimeric degrader <b>9</b> that were free from aggregation issues, we synthesized the linker-drug <b>9d</b>, which contained a phosphatase-cleavable pyrophosphate moiety in its chemical structure (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). We envisioned that ionization of the pyrophosphate at physiological pH would significantly reduce the overall lipophilicity of <b>9d</b> and thereby help mitigate the aggregation issues encountered with the disulfide-containing linker-drugs. Related pyrophosphate linkers were used by others to successfully attach highly lipophilic glucocorticoid payloads to antibodies, but these studies only described the creation of DAR2 conjugates and not the DAR6 entities that we sought.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> The <b>9d</b> design also leveraged learnings from our prior explorations of conjugates bearing chimeric degraders of the ERα protein.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> These studies demonstrated that biologically active DAR6 conjugates containing highly lipophilic chimeric ERα degraders could be successfully prepared via the pyrophosphate linker. They also alleviated concerns from the literature<a onclick="showRef(event, 'cit30b'); return false;" href="javascript:void(0);" class="ref cit30b">(30b)</a> regarding efficient phosphatase-mediated hydrolysis of the hydroxyproline phosphate ester that was expected to result from pyrophosphate cleavage following antibody-mediated intracellular delivery (see <a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Scheme S3</a> for details associated with the payload release mechanism anticipated for ADCs containing the pyrophosphate linker).</div><div class="NLM_p">We were encouraged to discover that a DAR6 STEAP1-targeting ADC could be prepared from the linker-drug <b>9d</b> with acceptable aggregation characteristics (STEAP1–<b>9d</b>, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Tables <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">S9</a>). This conjugate exhibited potent and extensive BRD4 degradation properties when tested in the PC3-S1 cell line that were well-separated from those displayed by the corresponding CLL1–<b>9d</b> control ADC (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). In addition, STEAP1–<b>9d</b> showed potent antiproliferation activity against PC3-S1 cells and this property was clearly distinguishable from that displayed by the CLL1–<b>9d</b> control conjugate (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Thus, for the first time in our BRD4-degrader conjugate studies, we observed strong STEAP1-dependent BRD4 degradation activity <i>and</i> potent antigen-specific antiproliferation effects in PC3-S1 cells. These promising results were consistent with the hypotheses presented in the first phase of our research that (1) BRD4 degradation in excess of 85% was required to produce meaningful antiproliferation effects and (2) conjugation of highly potent chimeric BRD4 degraders such as those derived from BRD4 binders <b>5</b>–<b>7</b> instead of compound <b>1</b> was a means of improving antiproliferation outcomes (see below for additional discussion).<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Conjugate CLL1–<b>9d</b> also exhibited excellent stability properties when exposed to whole blood from various species for 24 h (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Table S5</a>). These results suggested that the conjugate and other ADCs derived from the <b>9d</b> pyrophosphate-containing linker-drug would display favorable stability outcomes if/when assessed in in vivo settings.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Accordingly, we continued our exploration of the BRD4 degraders described in this work in an effort to identify additional biologically active ADCs that could be considered for in vivo efficacy assessments.</div><div class="NLM_p">With the objective of employing protease-cleavable linkers in this phase of our research, we next introduced amine and aniline functional groups at various locations within the chimeric structures of the new BRD4 degraders under study. In one such example, inclusion of a piperidine moiety in the VHL-binding fragment of degrader <b>8</b> afforded a new chimeric molecule (<b>11</b>) that bound potently to VHL and formed a BRD4/VHL ternary complex with an SPR half-life similar to that produced by the former molecule (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). As described in the earlier section of our studies, related derivatization of this position in the VHL-binding fragment was tolerated in the chimeric BRD4 inhibitor designs and was also shown by others to afford functional chimeric degraders directed at multiple protein targets.<a onclick="showRef(event, 'ref16 ref32'); return false;" href="javascript:void(0);" class="ref ref16 ref32">(16,32)</a> However, in spite of the highly similar SPR outcomes, compound <b>11</b> displayed significantly weaker BRD4 degradation and antiproliferation activities as compared to <b>8</b> when tested in PC3-S1 cells (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Similar to the biological results we observed with compounds <b>8</b> and <b>9</b>, incorporation of an all-carbon spacer group into the new degrader design generated a chimeric molecule (<b>12</b>), which exhibited significantly improved anti-MYC, BRD4 degradation, and antiproliferation properties relative to the PEG-containing analogue (compare <b>11</b> with <b>12</b>, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). In addition, the ternary complex formed among BRD4, VHL, and compound <b>12</b> was extremely long-lived in the SPR-based assessments with a half-life that exceeded that measured for the related chimeric degrader <b>9</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">S4</a>). However, as was noted when comparing compounds <b>8</b> and <b>11</b> above, the in vitro biological potencies exhibited by degrader <b>12</b> were significantly attenuated relative to those displayed by degrader <b>9</b>, which lacked the new VHL-binding fragment modification (compare <b>9</b> with <b>12</b>, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0009.jpeg" id="gr17" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Structures of chimeric BRD4 degraders <b>11</b> and <b>12</b> and corresponding linker-drugs.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We suspected that the described potency differences might be caused by reduced cell permeability of the new molecules <b>11</b> and <b>12</b> relative to the earlier-studied analogues <b>8</b> and <b>9</b> as a result of the polar amine moieties that were incorporated in the structures of the former two compounds. Somewhat surprisingly, IAM assessments suggested the opposite trend and predicted degraders <b>11</b> and <b>12</b> to be more cell-permeable than the corresponding compounds <b>8</b> and <b>9</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>; larger log <i>k</i> IAM values = higher predicted cell permeability<a onclick="showRef(event, 'ref16 ref22'); return false;" href="javascript:void(0);" class="ref ref16 ref22">(16,22)</a>). We currently do not know the origin of these disparate results with certainty, but they may reflect the overestimation of cell permeability by IAM methods (which directly measure membrane affinities) for chimeric degraders that contain basic amine groups. Alternatively, the basic amine containing chimeric molecules may become partially sequestered in acidic cellular compartments after transitioning through the cell membranes (e.g., the lysosomes). In any event, compounds <b>11</b> and (especially) <b>12</b> appeared to be outliers from the correlations depicted in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">S8</a> established by the other four molecules characterized in this work (<b>8</b>, <b>9</b>, <b>13</b>, and <b>15</b>). A related chimeric degrader from the first phase of our studies that contained the same VHL-binding fragment present in <b>11</b> and <b>12</b> was a similar outlier in the correlation plot depicted in that work (see compound <b>7</b> in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a> from ref <a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a>). However, as the total number of molecules characterized in these assessments is relatively small (particularly in the current work), additional studies are needed to fully understand and explain the observed phenomena.</div><div class="NLM_p">We subsequently coupled degrader <b>11</b> to a peptidomimetic, protease-cleavable linker that we previously utilized to create cytotoxic ADCs with favorable in vivo efficacy profiles.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> The resulting linker-drug (<b>11a</b>) afforded conjugates that efficiently released the attached chimeric degrader <b>11</b> following intracellular internalization, lysosomal mAb catabolism, protease-mediated linker cleavage, and subsequent self-immolation of the <i>para</i>-amino-benzyloxy (PAB) moiety (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>; see <a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Scheme S4</a> for additional details). As shown in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, the DAR6 STEAP1–<b>11a</b> conjugate displayed potent BRD4 degradation and antiproliferation effects in PC3-S1 cells that were well-separated from those exhibited by the corresponding CLL1–<b>11a</b> control ADC. The CLL1–<b>11a</b> conjugate also exhibited favorable stability properties in in vitro assessments employing whole blood from various species (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Table S5</a>). These results suggested good potential for ADCs constructed using the protease-cleavable linker to display promising in vivo stability outcomes. However, as was observed with the highly lipophilic linker-drugs <b>9a</b>, <b>9b</b>, and <b>9c</b> above, attempts to generate the analogous DAR6 STEAP1-targeting conjugate from <b>12a</b> were unsuccessful due to significant aggregation issues (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>, cf. <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). Accordingly, we prepared the corresponding DAR2 ADC (STEAP1–<b>12a</b>–D2), which was a reasonably potent and antigen-dependent BRD4 degrader in the PC3-S1 cell-based assessment (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). In spite of these favorable properties and for reasons that we do not currently understand, STEAP1–<b>12a</b>–D2 failed to afford measurable PC3-S1 antiproliferation activity even when tested to relatively high levels (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>).</div><div class="NLM_p">We concluded this portion of our research activities by preparing compounds <b>13</b>, <b>14</b>, and <b>15</b> in which the functional groups required for ADC linker attachment were located in the central spacer regions of the chimeric molecules (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>). Both <b>13</b> and <b>15</b> were reasonably active BRD4 degraders and antiproliferative agents in cell-based assessments (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). In addition, each compound formed a BRD4/VHL ternary complex that exhibited an SPR half-life of intermediate duration relative to the extremes established by degraders <b>8</b> and <b>11</b> (short, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) and <b>9</b> and <b>12</b> (long, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Compound <b>14</b> incorporated a 4(<i>S</i>)-hydroxyproline isomer and was prepared to enable construction of control ADCs, which corresponded to conjugates derived from degrader <b>13</b>. Although <b>14</b> was not tested in our in vitro biological assessments, it was predicted to be a poor VHL binder and inactive BRD4 degrader by analogy to the results that we obtained earlier with the related compound <b>10</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>).</div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0010.jpeg" id="gr18" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Structures of chimeric BRD4 degraders <b>13</b>, <b>14</b>, and <b>15</b> and corresponding linker-drugs.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">As shown in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, a STEAP1-targeting DAR6 conjugate prepared from <b>13</b> using a peptidomimetic, protease-cleavable linker (STEAP1–<b>13a</b>) exhibited strong BRD4 degradation and antiproliferation properties when tested in the PC3-S1 cell line. These activities were well-separated from those observed for the corresponding CLL1-targeting control conjugate (CLL1–<b>13a</b>), thus demonstrating good antigen-dependent delivery of the chimeric degrader <b>13</b> to the PC3-S1 cells. Like CLL1–<b>11a</b>, which contained a similar protease-cleavable linker, CLL1–13a displayed good stability properties in in vitro assessments employing whole blood from a variety of different species (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Table S5</a>). These outcomes indicated the suitability of conjugates derived from linker-drugs <b>13a</b> and <b>14a</b> (e.g., STEAP1–<b>13a</b>, CLL1–<b>13a</b>, and CLL1–<b>14a</b>) for subsequent in vivo evaluations.</div><div class="NLM_p">We also prepared a DAR6 STEAP1 ADC using an ether-containing linker-drug derived from compound <b>15</b> (STEAP1–<b>15a</b>). This entity was anticipated to efficiently release the phenol-containing chimeric degrader <b>15</b> following cellular internalization, antibody catabolism, protease-mediated linker cleavage, and PAB self-immolation (see <a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Scheme S5</a> for details). Our prior explorations of cytotoxic ADCs containing related phenol-ether-derived linkers established the feasibility of the described payload release process.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> STEAP1–<b>15a</b> was a highly potent BRD4 degrader and an active antiproliferation agent when tested in PC3-S1 cells (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). As was observed for STEAP1–13a, these activities were clearly distinguishable from those exhibited by the corresponding CLL1–<b>15a</b> control conjugate (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). The large difference between BRD4 DC<sub>50</sub> potency and PC3-S1 antiproliferation activity displayed by STEAP1–<b>15a</b> relative to the other chimeric degrader conjugates described in this work is currently not understood.</div><div class="NLM_p">As we did in the first phase of our studies, we performed graphical analyses of the data we generated with the chimeric BRD4-degrader conjugates to better identify trends associated with optimizing these complex entities. <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a> depicts the BRD4 DC<sub>50</sub> and <i>S</i><sub>inf</sub> values for most of the ADCs listed in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> as a function of linker-drug classification. This new plot is analogous to a graphic that was included in the description of the first phase of our research (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a> in ref <a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a>), and a few comparisons between the two images are made below. Consistent with that prior analysis and the desired antigen-dependent delivery mechanism, nearly all of the STEAP1-targeting conjugates of the present study exhibited more potent BRD4 DC<sub>50</sub> values relative to the corresponding CLL1 control conjugates (i.e., purple dots in <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a> in this work were almost always lower on the <i>y</i>-axis relative to blue dots for a given linker-drug; <b>9c</b> is the only exception). As was also observed in our earlier data examination, the separation between STEAP1 and CLL1 ADC activity was the smallest for conjugates containing the disulfide-based, GSH-cleavable linkers (possibly as a result of partial linker cleavage during cell-based assessments).<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> In contrast to the ADCs associated with the initial phase of our studies, all of the STEAP1-targeting conjugates of the present work displayed potent (DC<sub>50</sub> < 3 nM drug-equivalents) and extensive (<i>S</i><sub>inf</sub> > 85%) BRD4 degradation properties in the PC3-S1 cells. Collectively, these observations indicated that the shift from the compound <b>1</b>-derived chimeric BRD4 degraders described in our earlier research (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) to the more potent entities detailed in this disclosure consistently afforded more favorable BRD4 degradation outcomes for the corresponding antibody conjugates (cf., ref <a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a>, <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>).</div><figure id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0011.jpeg" id="gr19" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Graphical depiction of the BRD4 DC<sub>50</sub> and <i>S</i><sub>inf</sub> values for the conjugates described in this work (DAR6 ADCs unless otherwise indicated). The > 85% BRD4 <i>S</i><sub>inf</sub> threshold was purposefully chosen to highlight the suspected requirement for achieving antiproliferation activity (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Figure S9</a>). See the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Supporting Information</a> for a discussion of the assay error.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Since most of the STEAP1-targeting conjugates described in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> also exhibited potent PC3-S1 antiproliferation activities, we conducted several analyses to better understand the relationships between such outcomes and BRD4 degradation properties. First, transitioning from chimeric degraders derived from compound <b>1</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) to those constructed using the more-potent BRD4-binder <b>5</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>) did not alter the requirement to achieve at least 85% BRD4 depletion in the 4 hour PC3-S1 degradation assessment to trigger meaningful 6 day PC3-S1 antiproliferation effects (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Figure S9</a>). Second, a plot of the PC3-S1 BRD4 DC<sub>50</sub> potency associated with each conjugate versus the corresponding PC3-S1 antiproliferation IC<sub>50</sub> result revealed a loose correlation, which suggested a drug-equivalent DC<sub>50</sub> value of approximately 3 nM coupled with >85% BRD4 degradation to be necessary (but not always sufficient) to afford measurable antiproliferation activity (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>). This analysis included data associated with the ADCs from the first phase of our research in which the described activity combination was approached but never achieved with conjugates employing non-GSH-cleavable linker release mechanisms. The latter caveat is important since, as mentioned multiple times throughout both phases of our studies, the in vitro activities of ADCs bearing GSH-cleavable linker-drugs may be impacted by partial linker cleavage under the employed cell assay conditions. Other parameters in addition to those depicted in <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a> may also contribute to antiproliferation outcomes, and additional studies involving larger numbers of conjugates are required to fully delineate the drivers of potent cell-killing effects. It is worth noting that the majority of STEAP1-targeting conjugates described in this phase of our research also displayed potent antiproliferation activities against the LNCaP prostate cancer cell line (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Table S6</a>). In addition, the anti-LNCaP properties of STEAP1 ADCs derived from linker-drugs <b>9d</b>, <b>11a</b>, <b>13a</b>, and <b>15a</b> were well-separated from outcomes afforded by the corresponding CLL1 control conjugates (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Table S6</a>). Collectively, these results demonstrated in vitro antigen-dependent delivery of the chimeric BRD4-degrader molecules to a second STEAP1-expressing cell line by the depicted conjugates (see ref <a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a>, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Figure S1</a> for STEAP1 expression levels in LNCaP cells).</div><figure id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0012.jpeg" id="gr20" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Graphical depiction of the PC3-S1 BRD4 DC<sub>50</sub> values for STEAP1-targeting chimeric degrader conjugates and corresponding PC3-S1 antiproliferation activities. DAR6 ADCs unless otherwise indicated. Conjugates depicted with circle and square symbols are described in ref <a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> and, for clarity purposes, are not numbered in this figure. BRD4 degradation time point was 72 h. PC3-S1 antiproliferation time point was 6 days. Conjugates residing on the y-axis above the dotted blue line were inactive in PC3-S1 antiproliferation assessments when tested to the depicted <i>y</i>-axis concentrations. The number next to each point indicates the linker-drug employed to prepare each DAR6 STEAP1 conjugate (see <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>–<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a>). The BRD4 degradation <i>S</i><sub>inf</sub> values of the conjugates are depicted by the point sizes in the figure, and the >85% BRD4 <i>S</i><sub>inf</sub> threshold was purposefully chosen to highlight the suspected requirement for achieving the antiproliferation activity (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Figure S9</a>). See the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Supporting Information</a> for a discussion of the assay error.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To further understand why the conjugates derived from more potent chimeric BRD4 degraders afforded improved PC3-S1 cell-killing properties relative to the entities we initially studied, we again examined the intracellular concentrations of unconjugated compounds that were achieved via either passive diffusion or antibody-mediated delivery. In particular, we compared the outcomes for the previously studied compound <b>3</b> and the related STEAP1–<b>3a</b> ADC (<a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a> and ref <a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a>) with those obtained for chimeric degrader <b>13</b> and the corresponding STEAP1–<b>13a</b> conjugate in the present work (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>). Although STEAP1–<b>3a</b> and STEAP1–<b>13a</b> differed only in the nature of the BRD4-binding groups contained in their respective conjugated chimeric degrader molecules, the latter ADC exhibited potent antiproliferation activity in PC3-S1 assessments while the former entity was inactive.</div><figure id="fig12" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0013.jpeg" id="gr21" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Structures of linker-drug <b>3a</b> and corresponding DAR6 STEAP1 ADC. The biological properties of the STEAP1<b>–3a</b> conjugate were measured in the PC3-S1 cell line and are depicted in drug-equivalent units (see ref <a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> for additional information).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">As shown in <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>, both compounds <b>3</b> and <b>13</b> rapidly diffused into PC3-S1 cells when administered in the unconjugated form via the extracellular culture media. As was the case in the first phase of our research,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> we did not determine whether such diffusion was passive or transporter-mediated. We also did not adjust the depicted intracellular concentrations for compound binding to intracellular proteins or other entities in that environment, and we did not quantitate the compounds in any subcellular compartments. The intracellular levels achieved by the former molecule were somewhat higher than those noted for <b>13</b> at both the 4 and 22 h time points across all extracellular concentrations examined. In addition, ADC-mediated delivery of the two chimeric degraders afforded similar intracellular levels that were consistent with the DAR6 loading of both the STEAP1–<b>3a</b> and STEAP1–<b>13a</b> conjugates. This observation discounted the possibility that chimeric degraders such as <b>13</b> that were derived from a more-potent BRD4-binding moiety were better retained inside cells following antibody-mediated delivery relative to molecules like <b>3</b>. In addition, and consistent with the desired antigen-dependent delivery of the chimeric molecules, significantly lower intracellular levels of <b>3</b> and <b>13</b> were associated with the CLL1<b>–3a</b> and CLL1–<b>13a</b> conjugates at the 22 h time point relative to their STEAP1-targeting ADC counterparts. Subsequent comparison of the observed intracellular quantitations with the biological potencies associated with the <b>3</b>, <b>13</b>, STEAP1–<b>3a</b>, and STEAP1–<b>13a</b> then helped explain the differing antiproliferation outcomes of the two conjugates.</div><figure id="fig13" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0014.jpeg" id="gr22" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 13. Intracellular quantitation of chimeric degraders <b>3</b> or <b>13</b> in PC3-S1 cells following non-antibody-mediated delivery or respective delivery via STEAP1<b>–3a</b> or STEAP1–<b>13a</b> ADCs. Time points are listed at the top of each plot. Non-antibody-mediated delivery: compound <b>3</b> (solid purple), compound <b>13</b> (solid orange). ADC delivery: STEAP1<b>–3a</b> (solid red), CLL1<b>–3a</b> (dashed red); deliver compound <b>3</b>. ADC delivery: STEAP1–<b>13a</b> (solid green), CLL1–<b>13a</b> (dashed green); deliver compound <b>13</b>. Extracellular conjugate concentrations were absolute (i.e., <i>not</i> the drug-equivalent concentrations employed elsewhere in this work). For comparison purposes, an extracellular conjugate concentration of 100 nM corresponds to approximately 15 μg/mL. Intracellular values for compounds <b>3</b> and <b>13</b> associated with the 0.1 nM concentrations of the test articles were often below the 0.002 nM LLOQ and are not depicted in the plots.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=fig13"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Per <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>, the extracellular concentration of 1300 nM that was associated with the compound <b>3</b> PC3-S1 antiproliferation IC<sub>50</sub> value corresponded to an intracellular concentration in excess of 50 000 nM. The intracellular compound <b>3</b> levels achieved after 22 h via delivery by the STEAP1–<b>3a</b> ADC were nowhere near this threshold and likely never approached it during the majority of the 6 day antiproliferation experiment (note that the maximum concentration of DAR6 STEAP1–<b>3a</b> tested for the antiproliferation activity was approximately 120 nM in the absolute conjugate concentration units employed on the <i>x</i>-axis of <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>). In contrast, the PC3-S1 IC<sub>50</sub> antiproliferation value of the more potent degrader <b>13</b> was 91 nM. This significantly lower extracellular concentration corresponded to an intracellular level of approximately 4,000 nM at the 22 h time point, which was 10-fold less than the corresponding value determined for degrader <b>3</b>. Since the STEAP1–<b>13a</b> conjugate delivered compound <b>13</b> to the PC3-S1 cells with similar efficiency as that observed for delivery of <b>3</b> by STEAP1–<b>3a</b>, the former conjugate was more likely than the latter to achieve intracellular thresholds of the released chimeric degrader required to drive meaningful antiproliferation activity. There are many uncertainties associated with precisely measuring the intracellular concentrations of the chimeric degraders, and the described studies involved a relatively small number of time points and test concentrations. Accordingly, we suggest that the depicted techniques be utilized to assess relative improvements in antibody-based delivery approaches and not as absolute predictors of the desired biological outcomes.</div><div class="NLM_p">Having established that the STEAP1–<b>13a</b>  conjugate successfully delivered degrader <b>13</b> to PC3-S1 cells (<a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>), exhibited potent PC3-S1 antiproliferation activity (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>), and was likely to display favorable in vivo stability properties (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Table S5</a>), we progressed the ADC into in vivo efficacy evaluations. As shown in <a data-tab="pane-pcw-Figures" href="#fig14" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig14" id="" class=" internalNav">14</a></a>, single intravenous administrations of STEAP1–<b>13a</b> to mice bearing PC3-S1 tumor xenografts afforded dose-dependent tumor growth inhibition (TGI) outcomes during the course of an 11 day experiment. The in vivo activity of STEAP1–<b>13a</b> was clearly distinguishable from the results obtained with the CLL1–<b>13a</b> control conjugate and was thus consistent with the antibody-mediated delivery of chimeric degrader <b>13</b> to the PC3-S1 tumors via the STEAP1-targeting ADC. In addition, an unconjugated STEAP1 mAb exhibited minimal antitumor efficacy when administered at the highest STEAP1–<b>13a</b> dose of 10 mg/kg and thereby confirmed that the TGI outcomes with the conjugate were associated with the released chimeric degrader payload and not the antibody. Importantly, all conjugates evaluated in this experiment were well-tolerated and afforded minimal body weight changes that were similar to those observed for the vehicle control (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Figure S11</a>).</div><figure id="fig14" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 14</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0015.jpeg" id="gr23" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 14. <i>In vivo</i> activity of conjugates STEAP1–<b>13a</b> and CLL1–<b>13a</b> in the PC3-S1 xenograft model. Efficacy (tumor growth inhibition). Blue traces (solid lines), STEAP1–<b>13a</b>; blue trace (dotted line), unconjugated STEAP1 mAb; purple traces, CLL1–<b>13a</b>; black trace, vehicle control. Numbers next to traces indicate doses of each conjugate (in mg/kg) that were administered once IV at the day 0 time point. Group mean tumor volumes (±SEM) are plotted over the duration of the study (<i>n</i> = 8/group). See the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Supporting Information</a> for additional details.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=fig14"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Encouraged by the in vivo activity of the STEAP1–<b>13a</b> ADC, we profiled several CLL1-targeting conjugates and their STEAP1 counterparts in the HL-60 acute myeloid leukemia (AML) cell line. BRD4/BET inhibitors previously afforded clinical efficacy in AML patients,<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> and the CLL1 antigen is known to be highly expressed on the surface of most AML cells.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> In an earlier disclosure, we described the in vitro activity of compound <b>9</b> against HL-60 cells along with the in vivo efficacy of the CLL1–<b>9b</b>-D6 ADC in the corresponding mouse xenograft tumor model.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> We were now curious to learn if additional CLL1-targeting conjugates prepared from the chimeric BRD4 degraders described in this phase of our research exhibited antigen-dependent in vitro and/or in vivo activities against AML cancers. As shown in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>, both the CLL1–<b>8a</b> and CLL1–<b>13a</b> ADCs displayed potent antiproliferation activity when tested against the HL-60 line and were significantly more active than the corresponding STEAP1-targeting conjugates. As the STEAP1 antigen is not highly expressed on the surface of HL-60 cells, the observed outcomes represented yet another example of antigen-dependent delivery of the chimeric BRD4-degrader molecules to cells that complemented the STEAP1-related results in the PC3-S1 line depicted in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> above.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Antiproliferation Activity of Unconjugated Chimeric Degraders <b>8</b> and <b>13</b> and Conjugates Derived from Linker-Drugs <b>8a</b> and <b>13a</b> in the HL-60 Cell Line</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">cmpd or conjugate</th><th class="colsep0 rowsep0" align="center">HL-60 IC<sub>50</sub> (μg/mL)<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">HL-60 IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">HL-60 IC<sub>50</sub> ratio<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">1.9</td><td class="colsep0 rowsep0" align="left">NA</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">4.2</td><td class="colsep0 rowsep0" align="left">NA</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">STEAP1–<b>8a</b></td><td class="colsep0 rowsep0" align="left">0.067</td><td class="colsep0 rowsep0" align="left">2.6</td><td class="colsep0 rowsep0" align="left">NA</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CLL1–<b>8a</b></td><td class="colsep0 rowsep0" align="left">0.0061</td><td class="colsep0 rowsep0" align="left">0.24</td><td class="colsep0 rowsep0" align="left">11</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">STEAP1–<b>13a</b></td><td class="colsep0 rowsep0" align="left">0.15</td><td class="colsep0 rowsep0" align="left">6.0</td><td class="colsep0 rowsep0" align="left">NA</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CLL1–<b>13a</b></td><td class="colsep0 rowsep0" align="left">0.0020</td><td class="colsep0 rowsep0" align="left">0.077</td><td class="colsep0 rowsep0" align="left">78</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Antiproliferation activity of the conjugate, absolute concentration; time point was 6 days.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">Antiproliferation activity, unconjugated degrader, or conjugated chimeric degrader drug-equivalents; time point was 6 days.</p></div><div class="footnote" id="t3fn3"><sup><sup>c</sup></sup><p class="last">Ratio of STEAP1 and CLL1 IC<sub>50</sub> values for the given linker-drug. NA, not applicable.</p></div></div></div><div class="NLM_p">The CLL1–<b>8a</b> conjugate was subsequently advanced into an in vivo efficacy study employing the HL-60 xenograft model where it exhibited promising dose-dependent antitumor activity following single IV administrations (<a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">15</a></a>). Importantly, at a matched dose of 10 mg/kg, the efficacy noted for the corresponding STEAP1–<b>8a</b> control conjugate was considerably attenuated relative to that displayed by CLL1–<b>8a</b>. These experiments provided additional evidence that antibody conjugates bearing chimeric BRD4 degraders could successfully deliver such payloads to tumors in an antigen-dependent manner. As was the case for the PC3-S1 in vivo studies described above, mouse body weight changes were acceptable throughout the course of the HL-60 experiments with CLL1–<b>8a</b> and STEAP1–<b>8a</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Figure S12</a>).</div><figure id="fig15" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Figure 15</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0016.jpeg" id="gr24" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 15. <i>In vivo</i> activity of conjugates STEAP1–<b>8a</b> and CLL1–<b>8a</b> in the HL-60 xenograft model. Efficacy (tumor growth inhibition). Blue trace, STEAP1–<b>8a</b>; purple traces, CLL1–<b>8a</b>; black trace, vehicle control. Numbers next to traces indicate doses of each conjugate (in mg/kg) that were administered once IV at the day 0 time point. All groups received 30 mg/kg anti-ragweed IP 4 h prior to ADC administrations. Group mean tumor volumes (±SEM) are plotted over the duration of the study (<i>n</i> = 5/group). See the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Supporting Information</a> for additional details.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=fig15"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We concluded our in vivo efficacy profiling work by examining conjugates derived from linker-drug <b>13a</b> in the HL-60 xenograft model. Similar to what was observed for the CLL1–<b>8a</b> ADC, CLL1–<b>13a</b> displayed strong, dose-dependent TGI effects in this assessment with the efficacy outcomes well-separated from those observed for the corresponding STEAP1–<b>13a</b> control conjugate (<a data-tab="pane-pcw-Figures" href="#fig16" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig16" id="" class=" internalNav">16</a></a>a). To probe the origin of the in vivo activity associated with CLL1–<b>13a</b>, we performed a second experiment in the HL-60 model, which compared the former conjugate to the DAR6 CLL1–<b>14a</b> ADC. The latter entity was constructed from chimeric degrader <b>14</b>, which contained a 4(<i>S</i>)-hydroxyproline moiety and was thus not expected to bind to VHL, form a ternary complex with BRD4 and VHL, and/or degrade the BRD4 protein in cells (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>; also see biological results for the related compound <b>10</b>, <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). When administered to xenografted mice at a dose of 10 mg/kg, CLL1–<b>14a</b> displayed no efficacy in the HL-60 assessment (<a data-tab="pane-pcw-Figures" href="#fig16" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig16" id="" class=" internalNav">16</a></a>b). This lack of activity was in stark contrast to the strong TGI outcomes observed for CLL1–<b>13a</b> at the same 10 mg/kg dose in this experiment (<a data-tab="pane-pcw-Figures" href="#fig16" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig16" id="" class=" internalNav">16</a></a>b). Collectively, the results underscored a critical requirement for the conjugated payloads to possess potent BRD4 <i>degradation</i> activities, as opposed to only potent BRD4 <i>binding</i> affinities, in order for the related ADCs to afford strong in vivo antitumor properties. In addition, a DAR6 CLL1-targeting conjugate that was derived from a linker-drug that lacked a BRD4-binding moiety was inactive in the HL-60 in vivo model (CLL1–16, <a data-tab="pane-pcw-Figures" href="#fig16" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig16" id="" class=" internalNav">16</a></a>b and <a data-tab="pane-pcw-Figures" href="#fig17" id="" class=" internalNav">17</a>). This outcome demonstrated that the encouraging HL-60 efficacy noted for CLL1–<b>13a</b> did not result from direct inhibition of the VHL protein. An unconjugated CLL1 mAb was also not active in the HL-60 xenograft assessment, thereby confirming that antibody-related bioactivity did not meaningfully contribute to the CLL1–<b>13a</b> efficacy results (<a data-tab="pane-pcw-Figures" href="#fig16" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig16" id="" class=" internalNav">16</a></a>b). Importantly, all of the tested conjugates were well-tolerated with acceptable mouse body weight changes noted throughout the course of the experiments (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Figure S13a,b</a>).</div><figure id="fig16" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Figure 16</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0017.jpeg" id="gr25" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 16. (a, b) <i>In vivo</i> activity of conjugates STEAP1–<b>13a</b>, CLL1–<b>13a</b>, CLL1–<b>14a</b>, and CLL1–<b>16</b> in the HL-60 xenograft model. Efficacy (tumor growth inhibition). (a) Blue traces, STEAP1–<b>13a</b>; purple traces, CLL1–<b>13a</b>; black trace, vehicle control. (b) Purple traces (solid lines), CLL1–<b>13a</b>; purple trace (dotted line), unconjugated CLL1 mAb; green trace, CLL1–<b>14a</b>; red trace, CLL1–<b>16</b>; black trace, vehicle control. Numbers next to traces indicate doses of each conjugate (in mg/kg) that were administered once IV at the day 0 time point. All groups received 30 mg/kg anti-ragweed IP 4 h prior to ADC administrations. Group mean tumor volumes (±SEM) are plotted over the duration of the study (<i>n</i> = 5/group). See the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Supporting Information</a> for additional details.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=fig16"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig17" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Figure 17</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0018.jpeg" id="gr26" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 17. Compound <b>16</b> (VHL fragment linker-drug) and the corresponding CLL1–<b>16</b> conjugate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=fig17"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In addition to profiling the CLL1–<b>13a</b> conjugate in the various in vivo efficacy studies described above, we also examined the in vivo stability and pharmacokinetic properties associated with the ADC. As shown in <a data-tab="pane-pcw-Figures" href="#fig18" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig18" id="" class=" internalNav">18</a></a>, analysis of plasma samples obtained from mice during the course of the initial HL-60 in vivo experiment (<a data-tab="pane-pcw-Figures" href="#fig16" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig16" id="" class=" internalNav">16</a></a>a) demonstrated that CLL1–<b>13a</b> exhibited excellent in vivo stability properties with minimal retro-Michael-reaction loss of the attached linker-drug or biotransformation of the chimeric degrader payload (also see <a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Figure S14</a>). Similar favorable stability outcomes were observed for the conjugate in a separate mouse pharmacokinetic study over a 7 day period (data not shown). Liquid chromatography–mass spectrometry (LCMS)/MS-based assessments of the same plasma samples also determined that CLL1–<b>13a</b> exhibited a favorable total antibody pharmacokinetic profile that was similar to that determined previously for the unconjugated CLL1 mAb (<a data-tab="pane-pcw-Figures" href="#fig19" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig19" id="" class=" internalNav">19</a></a>; note the differing doses of the two entities).<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> When combined with the in vivo stability and pharmacokinetic data from the initial phase of our studies,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> the results depicted in <a data-tab="pane-pcw-Figures" href="#fig18" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig18" id="" class=" internalNav">18</a></a> and <a data-tab="pane-pcw-Figures" href="#fig19" id="" class=" internalNav">19</a> demonstrate the general ability of the described conjugation approaches to generate chimeric protein degrader conjugates with attractive in vivo properties. However, as was also mentioned in the descriptions of our earlier research, additional experimentation is needed to (1) better define the pharmacokinetic profiles of the new ADCs (e.g., characterize them over longer time periods in vivo), and (2) determine whether the in vivo trends observed for BRD4-degrader conjugates extend to ADCs bearing chimeric degraders, which target other proteins.</div><figure id="fig18" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Figure 18</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0019.jpeg" id="gr27" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 18. <i>In vivo</i> stability of CLL1–<b>13a</b> following a single IV administration of 10 mg/kg to mice (normalized ave DAR plot).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=fig18"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig19" data-index="19" class="article__inlineFigure"><h2 class="fig-label">Figure 19</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0020.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0020.jpeg" id="gr28" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 19. Pharmacokinetic properties of the conjugate CLL1–<b>13a</b> (10 mg/kg) and the corresponding unconjugated mAb (5 mg/kg) following single IV administrations of each entity at the indicated dose to mice (analyte = total antibody).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=fig19"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We concluded this phase of our studies by examining whether conjugation of chimeric BRD4 degraders to antibodies could potentially improve the thrombocytopenia risks associated with BRD4 inhibitors.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a> Other researchers have exploited low expression levels of various ubiquitin ligases in platelets to achieve reduced platelet toxicity for chimeric BCL-2/BCL-X<sub>L</sub> degraders relative to simple inhibitors.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> As shown in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>, representative chimeric BRD4 degraders from this work were less potent when compared with the corresponding BRD4 inhibitors in an in vitro assessment of human megakaryocytopoiesis, which was known to correlate well with the in vivo thrombocytopenia potential (compare MKC IC<sub>50</sub> of <b>1</b> with <b>2</b> and of <b>7</b> with <b>8</b>, <b>9</b>, and <b>13</b>; see also <a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Figure S15</a>).<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> These potency differences may result from reduced cell permeability properties of the chimeric degraders relative to the simple inhibitors. In addition, as depicted in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a> and discussed elsewhere in this work, the chimeric degraders exhibited more potent antiproliferation activities against prostate cancer and AML cell lines with respect to the corresponding BRD4 inhibitors. As a result of these collective outcomes, the chimeric degraders displayed larger separations between in vitro megakaryocytopoiesis inhibition and cell-based anticancer activity relative to the tested BRD4 inhibitors (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). These improvements were likely driven by the ability of the chimeric molecules to utilize the VHL ubiquitin ligase to degrade BRD4 in cancer cells but not in the various cell types involved in the megakaryocytopoiesis assessment. Consistent with this possibility, chimeric molecule <b>10</b>, which contained a 4(<i>S</i>)-hydroxyproline moiety and thus did not efficiently engage the VHL protein (cf., <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), exhibited significantly smaller separations between anti-megakaryocytopoiesis and anticancer potencies relative to the corresponding 4(<i>R</i>)-hydroxyproline-containing analogue <b>9</b> (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). However, we did not measure or otherwise quantitate VHL protein levels in the cells, which comprised the megakaryocytopoiesis assay to further substantiate the hypothesis.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Testing of Representative BRD4 Inhibitors, Chimeric Degraders, and Antibody–Drug Conjugates in <i>In Vitro</i> Assessments of Human Megakaryocytopoiesis and Anticancer Activity<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">class</th><th class="colsep0 rowsep0" align="center">cmpd or CNJ</th><th class="colsep0 rowsep0" align="center">MKC IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">prostate IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">MKC/prostate ratio<a class="ref internalNav" href="#t4fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center">AML IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t4fn4" aria-label="d">d</a></th><th class="colsep0 rowsep0" align="center">MKC/AML ratio<a class="ref internalNav" href="#t4fn5" aria-label="e">e</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">inhibitor</td><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="left">27</td><td class="colsep0 rowsep0" align="left">273</td><td class="colsep0 rowsep0" align="left">0.10</td><td class="colsep0 rowsep0" align="left">60</td><td class="colsep0 rowsep0" align="left">0.45</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">inhibitor</td><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="left">0.06</td><td class="colsep0 rowsep0" align="left">2.63</td><td class="colsep0 rowsep0" align="left">0.02</td><td class="colsep0 rowsep0" align="left">0.58</td><td class="colsep0 rowsep0" align="left">0.10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">chimeric degrader</td><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="left">79</td><td class="colsep0 rowsep0" align="left">87.6</td><td class="colsep0 rowsep0" align="left">0.90</td><td class="colsep0 rowsep0" align="left">4.35</td><td class="colsep0 rowsep0" align="left">18.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">chimeric degrader</td><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="left">29</td><td class="colsep0 rowsep0" align="left">17.9</td><td class="colsep0 rowsep0" align="left">1.6</td><td class="colsep0 rowsep0" align="left">1.91</td><td class="colsep0 rowsep0" align="left">15.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">chimeric degrader</td><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="left">0.74</td><td class="colsep0 rowsep0" align="left">0.42</td><td class="colsep0 rowsep0" align="left">1.8</td><td class="colsep0 rowsep0" align="left">0.025</td><td class="colsep0 rowsep0" align="left">29.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">chimeric inhibitor</td><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="left">4.0</td><td class="colsep0 rowsep0" align="left">25.5</td><td class="colsep0 rowsep0" align="left">0.16</td><td class="colsep0 rowsep0" align="left">1.64</td><td class="colsep0 rowsep0" align="left">2.44</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">chimeric degrader</td><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">64.5</td><td class="colsep0 rowsep0" align="left">0.25</td><td class="colsep0 rowsep0" align="left">3.69</td><td class="colsep0 rowsep0" align="left">4.34</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ADC<a class="ref internalNav" href="#t4fn6" aria-label="f">f</a></td><td class="colsep0 rowsep0" align="left">STEAP1–<b>13a</b></td><td class="colsep0 rowsep0" align="left">310</td><td class="colsep0 rowsep0" align="left">26.9</td><td class="colsep0 rowsep0" align="left">11.5</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">NA</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ADC<a class="ref internalNav" href="#t4fn6" aria-label="f">f</a></td><td class="colsep0 rowsep0" align="left">CLL1–<b>13a</b></td><td class="colsep0 rowsep0" align="left">466</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">5.09</td><td class="colsep0 rowsep0" align="left">91.6</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last"><i>In vitro</i> inhibition of human megakaryocytopoiesis; see the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Supporting Information</a> and ref <a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> for additional details.</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">Antiproliferation activity in prostate cancer cell lines (average of PC3-S1 and LNCaP IC<sub>50</sub> values).</p></div><div class="footnote" id="t4fn3"><sup><sup>c</sup></sup><p class="last">MKC IC<sub>50</sub> divided by prostate IC<sub>50</sub>.</p></div><div class="footnote" id="t4fn4"><sup><sup>d</sup></sup><p class="last">Antiproliferation activity in AML cell lines (average of HL-60 and EOL-1 IC<sub>50</sub> values).</p></div><div class="footnote" id="t4fn5"><sup><sup>e</sup></sup><p class="last">MKC IC<sub>50</sub> divided by AML IC<sub>50</sub>.</p></div><div class="footnote" id="t4fn6"><sup><sup>f</sup></sup><p class="last">IC<sub>50</sub> values for ADCs are expressed in a “drug-equivalent” manner (i.e., reflects the concentration of the corresponding conjugated chimeric degrader payload) and are thus comparable to data for the unconjugated BRD4 degraders and inhibitors. Time point for all assays was 6 days. NA, not applicable.</p></div></div></div><div class="NLM_p">We also observed that conjugation of a representative chimeric BRD4 degrader to antibodies could further increase the separation between the undesired inhibition of megakaryocytopoiesis and anticancer effects in vitro. When normalized for payload loading (i.e., tested in “drug-equivalent” concentrations), both the STEAP1–<b>13a</b> and CLL1–<b>13a</b> conjugates exhibited significantly milder impacts in the megakaryocytopoiesis assay relative to the unconjugated chimeric degrader <b>13</b> (<a data-tab="pane-pcw-Figures" href="#fig20" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig20" id="" class=" internalNav">20</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). These outcomes presumably resulted from low or absent STEAP1 and CLL1 antigen expressions on the surfaces of the cells that comprised the megakaryocytopoiesis assessments, but we did not perform related antigen quantitation experiments to substantiate the possibility. The described example illustrates how potential differences in cytosolic ubiquitin ligase and surface antigen expressions can be combined to minimize undesired in vitro toxicities while maintaining potent cell-based anticancer impacts. However, additional studies that include dedicated in vivo toxicology assessments are needed to fully determine the generality and translatability of these initial findings.</div><figure id="fig20" data-index="20" class="article__inlineFigure"><h2 class="fig-label">Figure 20</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0021.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0021.jpeg" id="gr29" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 20. Impact of chimeric degrader <b>13</b> and conjugates STEAP1–<b>13a</b> and CLL1–<b>13a</b> on in vitro human megakaryocytopoiesis. Concentrations of STEAP1–<b>13a</b> and CLL1–<b>13a</b> conjugates reflect amounts of conjugated chimeric degrader <b>13</b> present in the assay. Time point was 6 days.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0021.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=fig20"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The preparations of chimeric degraders <b>9</b> and <b>10</b> as well as linker-drug <b>9b</b> are described in the initial disclosure of our research activities.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> The other unconjugated chimeric degraders described in this work and the corresponding linker-drugs required for ADC syntheses were generated by a variety of methods that are depicted in the following schemes. Hydrolysis of the ester moiety present in the known compound <b>6</b><a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> under mild basic conditions provided the corresponding carboxylic acid <b>17</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). Subsequent coupling of this intermediate with methylamine using hexafluorophosphate azabenzotriazole tetramethyl uronium (HATU) then afforded compound <b>5</b>. Alternatively, reduction of the ester contained in <b>6</b> with DIBAL-H gave alcohol <b>18</b>, and this entity was then oxidized with Dess–Martin periodinane (DMP) to generate intermediate <b>19</b>. The aldehyde group present in <b>19</b> was homologated to the corresponding alkyne (compound <b>7</b>) by treatment with dimethyl (1-diazo-2-oxopropyl)phosphonate in methanol.</div><figure id="sch1" data-index="21" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0022.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0022.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Syntheses of BRD4 Inhibitors <b>5</b>–<b>7</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0022.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) LiOH·H<sub>2</sub>O, 4:1:1 tetrahydrofuran (THF):MeOH:H<sub>2</sub>O, 23 °C, 2 h, 86%; (b) CH<sub>3</sub>NH<sub>2</sub>·HCl, HATU, <i>N,N</i>-diisopropylethylamine (DIEA), dimethylformamide (DMF), 23 °C, 1 h, 81%; (c) DIBAL-H, 1,2-dichloroethane, −78 °C, 2 h, 95%; (d) DMP, 1,2-dichloroethane, 23 °C, 12 h, 74%; (e) CH<sub>3</sub>COC(N<sub>2</sub>)PO(OCH<sub>3</sub>)<sub>2</sub>, K<sub>2</sub>CO<sub>3</sub>, MeOH, 23 °C, 12 h, 24%.</p></p></figure><div class="NLM_p">The preparation of chimeric BRD4 degrader <b>8</b> and linker-drugs <b>8a</b> and <b>8b</b> is shown in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. The known compound <b>20</b><a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> was coupled to carboxylic acid <b>17</b> via a HATU-mediated process to give chimeric degrader <b>8</b> in a single step. Alternatively, the carbonate-containing entities <b>21</b> and <b>24</b> (see ref <a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a>) were independently deprotected under acidic conditions and the resulting amine salts (not shown) were coupled with the PEG spacer <b>22</b><a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> to give the corresponding intermediates <b>23</b> and <b>25</b>. Subsequent Boc deprotection of <b>23</b> and <b>25</b> followed by independent coupling of the resulting amine salts (not shown) with carboxylic acid <b>17</b> then, respectively, provided linker-drugs <b>8a</b> and <b>8b</b> after prep-HPLC purification.</div><figure id="sch2" data-index="22" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0023.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0023.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Syntheses of Chimeric BRD4 Degrader <b>8</b> and Linker-Drugs <b>8a</b> and <b>8b</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0023.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) HATU, Et<sub>3</sub>N, DMF, 23 °C, 2 h, 31%; (b) trifluoroacetic acid (TFA), 1,1,1,3,3,3-hexafluoropropan-2-ol (HFIP), 23 °C, 1 h, 98–100%; (c) <b>22</b>, HATU, DIEA, DMF, 23 °C, 0.5–1 h, 82–83%; (d) <b>17</b>, HATU, DIEA, DMF, 23 °C, 1 h, 29–42%.</p></p></figure><div class="NLM_p"><a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a> depicts the syntheses of linker-drugs <b>9a</b> and <b>9c</b>. In independent experiments, the Boc groups present in the carbonate-containing entities <b>26</b> and <b>24</b> (see ref <a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a>) were removed under acidic conditions and the resulting amine salts (not shown) were coupled with the all-carbon spacer <b>27</b><a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> to, respectively, generate the corresponding intermediates <b>28</b> and <b>29</b>. A second round of acidic deprotections afforded the Boc-deprotected amines related to <b>28</b> and <b>29</b> (TFA salts, not shown), and these entitles were subsequently coupled with carboxylic acid <b>17</b> via HATU-mediated processes to, respectively, provide linker-drugs <b>9a</b> and <b>9c</b>.</div><figure id="sch3" data-index="23" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0024.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0024.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Syntheses of Linker-Drugs <b>9a</b> and <b>9c</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0024.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) TFA, HFIP, 23 °C, 1 h, 87–100%; (b) <b>27</b>, HATU, DIEA, dichloromethane (DCM) or DMF, 23 °C, 1–2 h, 61–72%; (c) <b>17</b>, HATU, DIEA, DMF, 23 °C, 1–2 h, 43–62%.</p></p></figure><div class="NLM_p">The preparation of pyrophosphate-containing linker-drug <b>9d</b> is shown in <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>. Carboxylic acid <b>17</b> was coupled to all-carbon spacer <b>30</b> (HCl salt, prepared from <b>27</b><a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> in one step) to give intermediate <b>31</b>. Separately, the known VHL-binding fragment <b>32</b><a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> was protected as the corresponding Alloc derivative (<b>33</b>), and this entity was subsequently converted into the phosphite-containing intermediate <b>34</b>. Activation/oxidation<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> of <b>34</b> as the corresponding imidazole-containing derivative (<b>35</b>) was followed by coupling with the known Fmoc-protected amino-phosphate <b>36</b><a onclick="showRef(event, 'cit30b ref44'); return false;" href="javascript:void(0);" class="ref cit30b ref44">(30b,44)</a> to give pyrophosphate analogue <b>37</b>. Removal of the Alloc group then provided amine <b>38</b>, and this intermediate was coupled with <b>31</b> to afford the chimeric molecule <b>39</b>. Fmoc deprotection of <b>39</b> followed by coupling of the resulting amine (not shown) with the commercially available reagent <b>40</b> then gave the linker-drug <b>9d</b>.</div><figure id="sch4" data-index="24" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0025.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0025.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of the Linker-Drug <b>9d</b>.<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0025.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>30</b>, HATU, DIEA, DMF, 23 °C, 2 h, 100%; (b) ClC(O)OCH<sub>2</sub>CH═CH<sub>2</sub>, Na<sub>2</sub>CO<sub>3</sub>, 1:1 THF:H<sub>2</sub>O, 0–25 °C, 12 h, 81%; (c) PCl<sub>3</sub>, Et<sub>3</sub>N, THF, −78 to 23 °C, 12 h and then 5:2 NaHCO<sub>3</sub>:H<sub>2</sub>O, 23 °C, 10 min, 89%; (d) TMS-imidazole, Et<sub>3</sub>N, 1:1 MeCN:CCl<sub>4</sub>, 23 °C, 40 min, 90%; (e) <b>36</b>, ZnCl<sub>2</sub>, DMF, 23 °C, 12 h, 38%; (f) 1,3-dimethylbarbituric acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, DCM, 23 °C, 2 h, 61%; (g) <b>31</b>, HATU, DIEA, DMF, 23 °C, 2 h, 45%; (h) piperidine, DMF, 23 °C, 1 h, 59%; (i) <b>40</b>, DIEA, DMF, 23 °C, 6 h, 24%.</p></p></figure><div class="NLM_p">The methodologies employed to generate chimeric BRD4 degrader <b>11</b> and linker-drug <b>11a</b> are illustrated in <a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>, while those utilized to prepare degrader <b>12</b> and linker-drug <b>12a</b> are shown in <a class="ref internalNav" href="#sch6" aria-label="Scheme 6">Scheme 6</a>. The Boc-containing intermediate <b>41</b><a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> was deprotected under acidic conditions, and the resulting amine (TFA salt, <b>42</b>) was coupled with carboxylic acid <b>17</b> to give the chimeric molecule <b>43</b> (<a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>). The Cbz moiety present in <b>4</b>3 was subsequently removed to provide chimeric degrader <b>11</b>, and this entity was coupled with mixed carbonate <b>44</b><a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> to afford linker-drug <b>11a</b> (<a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>).</div><figure id="sch5" data-index="25" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0026.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0026.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Syntheses of Chimeric BRD4 Degrader <b>11</b> and Linker-Drug <b>11a</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0026.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) TFA, DCM, 23 °C, 30 min, 97%; (b) <b>17</b>, HATU, DIEA, DMF, 23 °C, 1 h, 98%; (c) TMSI, DCM, 23 °C, 1 h, 27%; (d) <b>44</b>, DIEA, DMF, 23 °C, 2 h, 45%.</p></p></figure><figure id="sch6" data-index="26" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0027.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0027.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Syntheses of Chimeric BRD4 Degrader <b>12</b> and Linker-Drug <b>12a</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0027.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 4.0 M HCl in EtOAc, 23 °C, 1 h, 100%; (b) <b>27</b>, HATU, DIEA, DMF, 23 °C, 1 h, 36%; (c) TFA, HFIP, 23 °C, 1 h, 100%; (d) <b>17</b>, HATU, DIEA, DMF, 23 °C, 1 h, 23%; (e) TMSI, DCM, 23 °C, 30 min, 70%; (f) <b>44</b>, DIEA, DMF, 23 °C, 2 h, 41%.</p></p></figure><div class="NLM_p">In a related approach, Boc deprotection of compound <b>45</b><a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> followed by coupling of the resulting amine (HCl salt, not shown) with the all-carbon spacer <b>27</b><a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> gave the intermediate <b>46</b> (<a class="ref internalNav" href="#sch6" aria-label="Scheme 6">Scheme 6</a>). A second acidic deprotection then afforded amine <b>47</b> (TFA salt), which was subsequently coupled to carboxylic acid <b>17</b> under HATU-mediated conditions to give the chimeric compound <b>48</b>. Cleavage of the Cbz group present in <b>48</b> then afforded degrader <b>12</b>, and this compound was condensed with mixed carbonate <b>44</b><a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> to provide linker-drug <b>12a</b> (<a class="ref internalNav" href="#sch6" aria-label="Scheme 6">Scheme 6</a>).</div><div class="NLM_p"><a class="ref internalNav" href="#sch7" aria-label="Scheme 7">Scheme 7</a> shows the methods employed to prepare chimeric BRD4 degraders <b>13</b> and <b>14</b> as well as linker-drugs <b>13a</b>, <b>14a</b>, and <b>16</b>. The Boc and <i>tert</i>-butyl moieties present in compound <b>49</b><a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> were cleaved under acidic conditions to provide intermediate <b>50</b>. This entity was coupled with the known VHL-binding fragment <b>32</b><a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> to give <b>51</b>. The Cbz group present in <b>51</b> was then removed via treatment with TMSI, and the resulting amine (not shown) was coupled with carboxylic acid <b>17</b> under HATU-mediated conditions to provide chimeric degrader <b>13</b>. Derivatization of <b>13</b> via exposure to mixed carbonate <b>44</b><a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> at slightly elevated temperatures subsequently afforded linker-drug <b>13a</b>. Intermediate <b>50</b> was also combined with the epimeric VHL-binding fragment <b>52</b><a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> to give product <b>53</b>. Cbz deprotection and coupling of the resulting amine (not shown) with carboxylic acid <b>17</b> then converted <b>53</b> into chimeric degrader <b>14</b>. Subsequent condensation of <b>14</b> with mixed carbonate <b>44</b><a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> transformed the former entity into linker-drug <b>14a</b> (<a class="ref internalNav" href="#sch7" aria-label="Scheme 7">Scheme 7</a>). Finally, in a separate and independent transformation, linker-drug <b>16</b> was prepared by condensation of intermediate <b>51</b> with mixed carbonate <b>44</b>.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a></div><figure id="sch7" data-index="27" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0028.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0028.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Syntheses of Chimeric BRD4 Degraders <b>13</b> and <b>14</b> and Linker-Drugs <b>13a</b>, <b>14a</b>, and <b>16</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0028.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) TFA, DCM, 23 °C, 1 h, 40%; (b) <b>32</b> or <b>52</b>, HATU, DIEA, DMF, 23 °C, 1 h, 53–69%; (c) TMSI, DCM, 23 °C, 1 h, 98–100%; (d) <b>17</b>, HATU, DIEA, DMF, 23 °C, 1 h, 32–70%; (e) <b>44</b>, HOBt, pyridine, DMF, 38 °C, 1 or 24 h, 12–19%.</p></p></figure><div class="NLM_p">The syntheses of chimeric degrader <b>15</b> and the corresponding linker-drug <b>15a</b> are, respectively, depicted in <a class="ref internalNav" href="#sch8" aria-label="Schemes 8">Schemes 8</a> and <a class="ref internalNav" href="#sch9" aria-label="9">9</a>. Removal of the Cbz protecting group from intermediate <b>54</b><a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> provided the deprotected amine <b>55</b>, and this entity was coupled with carboxylic acid <b>17</b> under HATU-mediated conditions to afford compound <b>56</b> (<a class="ref internalNav" href="#sch8" aria-label="Scheme 8">Scheme 8</a>). Cleavage of the <i>tert</i>-butyl group present in <b>56</b> followed by coupling of the resulting carboxylic acid (not shown) with the known VHL-binding fragment <b>32</b><a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> then gave chimeric degrader <b>15</b> (<a class="ref internalNav" href="#sch8" aria-label="Scheme 8">Scheme 8</a>). Separately, the known Fmoc-protected <span class="smallcaps smallerCapital">l</span>-citrulline-PAB-OH derivative <b>57</b><a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> was converted to the corresponding bromide <b>58</b> (<a class="ref internalNav" href="#sch9" aria-label="Scheme 9">Scheme 9</a>). This material was used to alkylate the phenol moiety present in intermediate <b>59</b> (prepared in one step via Boc-protection of amino-phenol <b>55</b>) to give compound <b>60</b>. Simultaneous cleavage of the Boc and <i>tert</i>-butyl groups contained in <b>60</b> was accomplished by exposure to acidic conditions, and the resulting amino acid (<b>61</b>) was coupled with carboxylic acid <b>17</b> to provide intermediate <b>62</b>. This entity was subsequently coupled with the VHL-binding fragment <b>32</b><a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> to afford the chimeric molecule <b>63</b>. The Fmoc protecting group present in <b>63</b> was then removed under mild basic conditions, and the resulting amine (<b>64</b>) was condensed with <b>65</b><a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> to give linker-drug <b>15a</b> (<a class="ref internalNav" href="#sch9" aria-label="Scheme 9">Scheme 9</a>).</div><figure id="sch8" data-index="28" class="article__inlineFigure"><h2 class="fig-label">Scheme 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0029.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0029.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 8. Synthesis of Chimeric BRD4 Degrader <b>15</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0029.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=sch8"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Pd on C, H<sub>2</sub>, THF, 23 °C, 1 h, 100%; (b) <b>17</b>, HATU, DIEA, DMF, 23 °C, 2 h, 60%; (c) TFA, DCM, 23 °C, 1 h, 99%; (d) <b>32</b>, HATU, DIEA, DMF, 23 °C, 2 h, 21%.</p></p></figure><figure id="sch9" data-index="29" class="article__inlineFigure"><h2 class="fig-label">Scheme 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0030.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0030.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 9. Synthesis of Linker-Drug <b>15a</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0030.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=sch9"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 35% HBr in HOAc, 23 °C, 2 h, 89%; (b) Boc<sub>2</sub>O, Et<sub>3</sub>N, DCM, 23 °C, 1 h, 87%; (c) <b>58</b>, Cs<sub>2</sub>CO<sub>3</sub>, DMF, 23 °C, 1 h, 19%; (d) TFA, HFIP, 23 °C, 1 h, 99%; (e) <b>17</b>, HATU, DIEA, DMF, 23 °C, 1 h, 41%; (f) <b>32</b>, HATU, DIEA, DMF, 23 °C, 1 h, 22%; (g) piperidine, DMF, 23 °C, 2 h, 93%; (h) <b>65</b>, HATU, DIEA, DMF 23 °C, 1 h, 33%.</p></p></figure></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31226" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31226" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">This work details the second phase of our research activities aimed at optimizing the in vitro and in vivo bioactivities of chimeric BRD4-degrader antibody conjugates. It describes a shift in the focus of our conjugation efforts from chimeric molecules derived from compound <b>1</b> (JQ1, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>)<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> to exploration of BRD4-degrading entities containing the more-potent BRD4-binding fragment <b>5</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). As was performed in the first part of our research, we continued to thoroughly characterize the biological properties of the newly created chimeric degraders in the unconjugated form using assessments of BRD4 and VHL-binding, BRD4/VHL ternary complex formation, and IAM-based cell permeability in addition to cellular BRD4 degradation and antiproliferation profiling. These characterizations collectively demonstrated that BRD4 degraders derived from <b>5</b> were significantly more potent in cell-based assays relative to the chimeric molecules we studied in the initial phase of our research (e.g., compounds <b>2</b> and <b>3</b>, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). Importantly, the cell-based activities of the new chimeric entities described in this work were closer to the range of potencies typically associated with successful cytotoxic ADC payloads (i.e., low nanomolar to picomolar antiproliferation IC<sub>50</sub> values).</div><div class="NLM_p">We also continued to target the synthesis of ADCs with drug/antibody ratios of 6 (DAR6, “high-DAR” conjugates) in an effort to maximize the amounts of attached chimeric degraders delivered to the interior of cells. In contrast to the first part of our research, we periodically encountered challenges in achieving the desired DAR6 antibody loadings due to extensive aggregation issues that were observed following attachment of certain highly lipophilic linker-drugs (e.g., <b>9</b> and <b>12</b>) to the antibodies. These unwanted outcomes were highly sensitive to subtle changes in the antibody linker composition as well as the antigen targeted by the employed mAbs. The results highlighted the need for continued research in the antibody–drug conjugate field that seeks to identify linker and/or conjugation technologies that routinely enable the generation of high-DAR ADCs from lipophilic payloads.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a></div><div class="NLM_p">In spite of these challenges, multiple examples of cell-active DAR6 STEAP1-targeting BRD4-degrader conjugates were identified during the course of the research described in this work. Many of these entities exhibited large separations between their potent PC3-S1 BRD4 degradation and antiproliferation properties and the in vitro activities displayed by the corresponding CLL1-targeting control conjugates. These outcomes were consistent with antigen-dependent, antibody-mediated delivery of the chimeric degrader molecules to the targeted STEAP1-expressing prostate cancer cells. Alternatively, several conjugates directed at the CLL1 antigen were employed to selectively deliver several representative chimeric degrader payloads (e.g., compounds <b>8</b> and <b>13</b>) to HL-60 AML cells in an antigen-dependent manner. The collective in vitro results illustrated the ability to direct the delivery of the chimeric molecules to cancer cells of interest solely by proper mAb antigen selection without having to re-engineer the design of the associated antibody linkers.</div><div class="NLM_p last">Very encouragingly, the favorable cell culture outcomes translated into the in vivo realm where examples of strong, antigen-dependent antitumor activity were achieved in both prostate cancer and AML xenograft models. Importantly, the in vivo antitumor efficacy was demonstrated with structurally diverse conjugates that employed a peptidomimetic protease-cleavable or a disulfide-containing glutathione-cleavable linker and distinct chimeric BRD4-degrader payloads (compounds <b>8</b> and <b>13</b>). Equally important, control experiments clearly demonstrated that the in vivo activity of the CLL1–<b>13a</b> conjugate was associated with the delivery of a potent BRD4 protein degrader (as opposed to a potent BRD4 and/or VHL inhibitor). When combined with the initial disclosure of in vivo activity associated with conjugates derived from compound <b>9</b>,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> the collective results suggest that antibody conjugation of BRD4-targeted chimeric degraders can serve as a general means of enabling such modalities to achieve meaningful in vivo efficacy outcomes. Additional experiments are required to determine whether in vivo-active conjugates can be constructed using chimeric molecules that degrade proteins other than BRD4. However, we believe that the results described in this work and the related preceding paper clearly establish antibody-mediated delivery as an option that should strongly be considered during the preclinical exploration of chimeric degrader molecules.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10062" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10062" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> General Chemistry Methods</h3><div class="NLM_p">Unless otherwise indicated, all reagents and solvents were purchased from commercial sources and were used without further purification. Moisture- or oxygen-sensitive reactions were conducted under an atmosphere of argon or nitrogen gas. Unless otherwise stated, <sup>1</sup>H NMR spectra were recorded at 300 or 400 MHz using Varian or Bruker instruments operating at the indicated frequencies. Chemical shifts are expressed in ppm (δ) relative to an internal standard; tetramethylsilane (ppm = 0.00). The following abbreviations are used: br, broad signal; s, singlet; d, doublet; dd, doublet of doublets; t, triplet; q, quartet; p, pentet; m, multiplet. Purification by silica gel chromatography was carried out using Biotage systems with prepacked cartridges. Analytical LCMS methods are provided in the Supporting Information (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Table S8</a>). Chemical purities were ≥95% for all final compounds as assessed by LCMS analysis at UV 220 and 254 nm.</div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> 4-(3,5-Difluoropyridin-2-yl)-10-methyl-7-((methylsulfonyl)methyl)-11-oxo-3,4,10,11-tetrahydro-1<i>H</i>-1,4,10-triazadibenzo[cd,<i>f</i>]azulene-6-carboxylic Acid (<b>17</b>)</h4><div class="NLM_p last">To a 23 °C solution of compound <b>6</b><a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> (30 mg, 0.06 mmol) in 4:1:1 THF:MeOH:H<sub>2</sub>O (6.0 mL) was added LiOH·H<sub>2</sub>O (12 mg, 0.29 mmol). The mixture was stirred at 23 °C for 2 h and then was diluted with water (10 mL). The resulting mixture was washed with EtOAc (3 × 10 mL), and the aqueous layer was acidified with HCl (1.0 M) to pH = 4.0. This layer was then extracted with EtOAc (3 × 10 mL), and the combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to afford compound <b>17</b> (25 mg, 86%) as a yellow solid. LCMS (5–95, AB, 1.5 min): RT = 0.73 min, <i>m</i>/<i>z</i> = 501.0 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> 4-(3,5-Difluoropyridin-2-yl)-<i>N</i>,10-dimethyl-7-((methylsulfonyl)methyl)-11-oxo-3,4,10,11-tetrahydro-1<i>H</i>-1,4,10-triazadibenzo[cd,<i>f</i>]azulene-6-carboxamide (<b>5</b>)</h4><div class="NLM_p last">Compound <b>6</b> (40 mg, 0.08 mmol) was added to a solution of MeNH<sub>2</sub>·HCl (11 mg, 0.16 mmol), HATU (46 mg, 0.12 mmol), and DIEA (230 mg, 1.79 mmol) in DMF (10 mL) at 23 °C. The mixture was stirred at 23 °C for 1 h and then concentrated. The residue was purified by prep-HPLC (MeCN 20–50%/0.225% FA in H<sub>2</sub>O) to give compound <b>5</b> (33 mg, 81%) as a white solid. LCMS (5–95, AB, 1.5 min): RT = 0.61 min, <i>m</i>/<i>z</i> = 514.0 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.94 (brs, 1H), 8.42–8.34 (m, 1H), 8.08 (d, <i>J</i> = 2.8 Hz, 1H), 7.92 (s, 1H), 7.76 (s, 1H), 7.66–7.58 (m, 1H), 7.31–7.25 (m, 2H), 5.95 (brs, 1H), 5.22 (brs, 1H), 4.63 (brs, 1H), 4.25 (brs, 1H), 3.63 (s, 3H), 2.91 (s, 3H), 2.69 (d, <i>J</i> = 4.4 Hz, 3H).</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> 4-(3,5-Difluoropyridin-2-yl)-6-(hydroxymethyl)-10-methyl-7-((methylsulfonyl)methyl)-1,3,4,10-tetrahydro-11H-1,4,10-triazadibenzo[cd,<i>f</i>]azulen-11-one (<b>18</b>)</h4><div class="NLM_p last">To a −78 °C solution of compound <b>6</b><a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> (30 mg, 0.06 mmol) in 1,2-dichloroethane (20 mL) was added a solution of DIBAL-H in toluene (1.0 M, 0.23 mL, 0.23 mmol). The reaction solution was stirred at −78 °C for 2 h and then quenched with H<sub>2</sub>O (10 mL). The resulting mixture was extracted with DCM (2 × 30 mL), and the combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to afford the crude compound <b>18</b> (27 mg, 95%) as a light oil. This material was used in the next step without additional purification. LCMS (5–95, AB, 1.5 min): RT = 0.75 min, <i>m</i>/<i>z</i> = 487.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> 4-(3,5-Difluoropyridin-2-yl)-10-methyl-7-((methylsulfonyl)methyl)-11-oxo-3,4,10,11-tetrahydro-1<i>H</i>-1,4,10-triazadibenzo[cd,<i>f</i>]azulene-6-carbaldehyde (<b>19</b>)</h4><div class="NLM_p last">To a 23 °C solution of compound <b>18</b> (27 mg, 0.06 mmol) in 1,2-dichloroethane (10 mL) was added DMP (70 mg, 0.17 mmol). The reaction was stirred at 23 °C for 12 h; then, H<sub>2</sub>O (10 mL) was added. The resulting mixture was extracted with DCM (2 × 30 mL), and the combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. Purification of the residue by prep-TLC (15% MeOH in DCM, <i>R<sub>f</sub></i> = 0.3) then provided compound <b>19</b> (20 mg, 74%) as a yellow solid. LCMS (5–95, AB, 1.5 min): RT = 0.80 min, <i>m</i>/<i>z</i> = 485.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> 4-(3,5-Difluoropyridin-2-yl)-6-ethynyl-10-methyl-7-((methylsulfonyl)methyl)-1,3,4,10-tetrahydro-11H-1,4,10-triazadibenzo[cd,<i>f</i>]azulen-11-one (<b>7</b>)</h4><div class="NLM_p last">To a 23 °C solution of compound <b>19</b> (12 mg, 0.02 mmol) in MeOH (6 mL) were added K<sub>2</sub>CO<sub>3</sub> (12 mg, 0.09 mmol) and dimethyl (1-diazo-2-oxopropyl)phosphonate (12 mg, 0.06 mmol). The mixture was stirred at 23 °C for 12 h and then concentrated under reduced pressure. The resulting residue was partitioned between H<sub>2</sub>O (10 mL) and EtOAc (2 × 20 mL). The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The residue was purified by prep-HPLC (MeCN 36–66%/0.225% FA in H<sub>2</sub>O) to afford compound <b>7</b> (2.8 mg, 24%) as an off-white solid. LCMS (5–95, AB, 1.5 min): RT = 0.77 min, <i>m</i>/<i>z</i> = 503.1 [M + Na]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.94 (brs, 1H), 8.07 (d, <i>J</i> = 2.4 Hz, 1H), 7.94 (s, 1H), 7.72 (s, 1H), 7.67–7.63 (m, 1H), 7.28–7.25 (m, 2H), 5.90 (brs, 1H), 4.63 (brs, 2H), 4.51 (s, 1H), 4.25 (brs, 1H), 3.63 (s, 3H), 3.02 (s, 3H).</div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> 4-(3,5-Difluoropyridin-2-yl)-<i>N</i>-((<i>S</i>)-13-((2<i>S</i>,4<i>R</i>)-4-hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidine-1-carbonyl)-14,14-dimethyl-11-oxo-3,6,9-trioxa-12-azapentadecyl)-10-methyl-7-((methylsulfonyl)methyl)-11-oxo-3,4,10,11-tetrahydro-1<i>H</i>-1,4,10-triazadibenzo[cd,<i>f</i>]azulene-6-carboxamide (<b>8</b>)</h4><div class="NLM_p last">A mixture of compound <b>20</b><a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> (53 mg, 0.07 mmol), compound <b>17</b> (30 mg, 0.06 mmol), Et<sub>3</sub>N (18 mg, 0.18 mmol), and HATU (34 mg, 0.09 mmol) in DMF (3 mL) was stirred at 23 °C for 2 h. The mixture was then directly purified by prep-HPLC (MeCN 15–45%/0.225% FA in H<sub>2</sub>O) to give compound <b>8</b> (20 mg, 31%) as an off-white solid. LCMS (5–95, AB, 1.5 min): RT = 0.77 min, <i>m</i>/<i>z</i> =1102.6 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.88 (s, 1H), 8.94 (s, 1H), 8.58–8.55 (m, 1H), 8.46–8.44 (m, 1H), 8.03 (s, 1H), 7.89 (s, 1H), 7.73 (s, 1H), 7.59–7.54 (m, 1H), 7.46–7.29 (m, 5H), 7.26–7.24 (m, 2H), 5.90 (brs, 1H), 5.14 (brs, 1H), 4.54–4.52 (m, 2H), 4.46–4.29 (m, 3H), 4.27–4.15 (m, 1H), 4.09–4.03 (m, 1 H), 3.96–3.87 (m, 2H), 3.61–3.47 (m, 14H), 2.87 (s, 3H), 2.40 (s, 3H), 2.02–2.00 (m, 1H), 1.90–1.84 (m, 1H), 1.29–1.08 (m, 1H), 0.91 (s, 9H).</div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> <i>S</i>-(1-(((((3<i>R</i>,5<i>S</i>)-1-((<i>S</i>)-18-(<i>tert</i>-Butyl)-2,2-dimethyl-4,16-dioxo-3,8,11,14-tetraoxa-5,17-diazanonadecan-19-oyl)-5-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-3-yl)oxy)carbonyl)oxy)-2-methylpropan-2-yl) Methanesulfonothioate (<b>23</b>)</h4><div class="NLM_p last">TFA (0.1 mL) was added to a 23 °C solution of compound <b>21</b><a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> (107 mg, 0.14 mmol) in HFIP (1.9 mL). The reaction mixture was stirred at 23 °C for 1 h and then concentrated <i>in vacuo</i> to give the corresponding crude TFA salt (not shown, 109 mg, 100%) as a colorless oil. LCMS (5–95, AB, 1.5 min): RT = 0.63 min, <i>m</i>/<i>z</i> = 641.0 [M + H]<sup>+</sup>. Separately, DIEA (50 mg, 0.37 mmol) was added to a solution of compound <b>22</b><a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> (30 mg, 0.10 mmol) and HATU (42 mg, 0.11 mmol) in DMF (5 mL) at 23 °C. The mixture was stirred at that temperature for 10 min; then, a portion of the crude TFA salt obtained from compound <b>21</b> (56 mg, 0.07 mmol) was added. The resulting mixture was stirred at 23 °C for 1 h and then concentrated <i>in vacuo</i>. The residue was purified by prep-TLC (5% MeOH in DCM, <i>R<sub>f</sub></i> = 0.5) to afford compound <b>23</b> (65 mg, 82%) as a yellow solid. LCMS (5–95, AB, 1.5 min): RT = 0.85 min, <i>m</i>/<i>z</i> = 952.3 [M + Na]<sup>+</sup>.</div></div><div id="sec4_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> <i>S</i>-(1-(((((3<i>R</i>,5<i>S</i>)-1-((<i>S</i>)-15-(<i>tert</i>-Butyl)-1-(4-(3,5-difluoropyridin-2-yl)-10-methyl-7-((methylsulfonyl)methyl)-11-oxo-3,4,10,11-tetrahydro-1<i>H</i>-1,4,10-triazadibenzo[cd,<i>f</i>]azulen-6-yl)-1,13-dioxo-5,8,11-trioxa-2,14-diazahexadecan-16-oyl)-5-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-3-yl)oxy)carbonyl)oxy)-2-methylpropan-2-yl) Methanesulfonothioate (<b>8a</b>)</h4><div class="NLM_p last">To a 23 °C solution of compound <b>23</b> (65 mg, 0.06 mmol) in HFIP (6.0 mL) was added TFA (0.3 mL). The reaction mixture was stirred at 23 °C for 1 h and then concentrated <i>in vacuo</i> to afford the crude deprotected amine (TFA salt, not shown, 57 mg, 100%) as a yellow oil. LCMS (5–95, AB, 1.5 min): RT = 0.72 min, <i>m</i>/<i>z</i> = 830.3 [M + H]<sup>+</sup>. Separately, DIEA (23 mg, 0.17 mmol) was added to a 23 °C solution of compound <b>17</b> (20 mg, 0.04 mmol) and HATU (20 mg, 0.05 mmol) in DMF (5 mL). The reaction mixture was stirred at 23 °C for 10 min, and a portion of the crude TFA salt obtained from compound <b>23</b> (33 mg, 0.03 mmol) was then added. The resulting mixture was stirred at 23 °C for 1 h and then concentrated <i>in vacuo</i>. The residue was purified by prep-HPLC (MeCN 41–71%/0.225% FA in H<sub>2</sub>O) to give compound <b>8a</b> (22 mg, 42%) as a white solid. LCMS (5–95, AB, 1.5 min): RT = 0.82 min, <i>m</i>/<i>z</i> = 1313.6 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.91 (s, 1H), 8.97 (s, 1H), 8.65 (brs, 1H), 8.48 (brs, 1H), 8.06 (s, 1H), 7.92 (s, 1H), 7.76 (s, 1H), 7.62–7.57 (m, 1H), 7.45–7.39 (m, 5H), 7.28 (brs, 2H), 5.95 (br, 1H), 5.26–5.17 (m, 2H), 4.45–4.26 (m, 7H), 4.07–3.86 (m, 4H), 3.62 (brs, 5H), 3.53 (brs, 10H), 2.90 (s, 3H), 2.43–2.33 (m, 7H), 2.15–1.99 (m, 1H), 1.49 (d, <i>J</i> = 6.8 Hz, 6H), 1.23 (brs, 1H), 0.91 (s, 9H).</div></div><div id="sec4_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> <i>S</i>-(3-(((((3<i>R</i>,5<i>S</i>)-1-((<i>S</i>)-18-(<i>tert</i>-Butyl)-2,2-dimethyl-4,16-dioxo-3,8,11,14-tetraoxa-5,17-diazanonadecan-19-oyl)-5-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-3-yl)oxy)carbonyl)oxy)butan-2-yl) Methanesulfonothioate (<b>25</b>)</h4><div class="NLM_p last">TFA (0.3 mL) was added to a solution of compound <b>24</b><a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> (90 mg, 0.12 mmol) in HFIP (6.0 mL) at 23 °C. The reaction solution was stirred at 23 °C for 1 h and then concentrated <i>in vacuo</i> to afford the crude deprotected amine (TFA salt, not shown, 90 mg, 98%) as a colorless oil. LCMS (5–95, AB, 1.5 min): RT = 0.74 min, <i>m</i>/<i>z</i> = 641.1 [M + H]<sup>+</sup>. Separately, DIEA (77 mg, 0.60 mmol) was added to a 23 °C solution of compound <b>22</b><a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> (55 mg, 0.18 mmol) and HATU (91 mg, 0.24 mmol) in DMF (6 mL). The reaction solution was stirred at 23 °C for 15 min; then, the crude amine TFA salt obtained from compound <b>24</b> (90 mg, 0.12 mmol) was added. The resulting mixture was stirred at 23 °C for 0.5 h and then concentrated <i>in vacuo</i>. The residue was purified by prep-TLC (10% MeOH in DCM, <i>R<sub>f</sub></i> = 0.7) to afford compound <b>25</b> (100 mg, 83%) as a white solid. LCMS (10–80, AB, 7.0 min): RT = 4.03 min, <i>m</i>/<i>z</i> = 952.3 [M + Na]<sup>+</sup>.</div></div><div id="sec4_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> <i>S</i>-(3-(((((3<i>R</i>,5<i>S</i>)-1-((<i>S</i>)-15-(<i>tert</i>-Butyl)-1-(4-(3,5-difluoropyridin-2-yl)-10-methyl-7-((methylsulfonyl)methyl)-11-oxo-3,4,10,11-tetrahydro-1<i>H</i>-1,4,10-triazadibenzo[cd,<i>f</i>]azulen-6-yl)-1,13-dioxo-5,8,11-trioxa-2,14-diazahexadecan-16-oyl)-5-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-3-yl)oxy)carbonyl)oxy)butan-2-yl) Methanesulfonothioate (<b>8b</b>)</h4><div class="NLM_p last">To a solution of compound <b>25</b> (90 mg, 0.10 mmol) in HFIP (7.0 mL) was added TFA (0.35 mL) at 23 °C. The reaction solution was stirred at that temperature for 1 h and then concentrated <i>in vacuo</i> to afford the crude deprotected amine (TFA salt, not shown, 90 mg, 99%) as a colorless oil. LCMS (5–95, AB, 1.5 min): RT = 0.76 min, <i>m</i>/<i>z</i> = 830.5 [M + H]<sup>+</sup>. Separately, DIEA (21 mg, 0.16 mmol) and HATU (18 mg, 0.05 mmol) were added sequentially to a 23 °C solution of compound <b>17</b> (20 mg, 0.04 mmol) in DMF (4 mL). The reaction solution was stirred at 23 °C for 10 min; then, a portion of the crude amine TFA salt obtained from compound <b>25</b> (45 mg, 0.05 mmol) was added. The resulting solution was stirred at 23 °C for an additional 1 h and then directly purified by prep-HPLC (MeCN 36–66%/0.225% FA in H<sub>2</sub>O) to afford compound <b>8b</b> (15 mg, 29%) as a white solid. LCMS (5–95, AB, 1.5 min): RT = 0.84 min, <i>m</i>/<i>z</i> = 1334.2 [M + Na]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 8.87 (s, 1H), 7.94 (s, 1H), 7.88 (s, 1H), 7.69 (s, 2H), 7.62–7.58 (m, 1H), 7.43–7.35 (m, 3H), 7.25–7.22 (m, 2H), 5.29 (s, 1H), 4.62–4.51 (m, 4H), 4.34–4.22 (m, 4H), 4.04–3.92 (m, 4H), 3.71–3.64 (m, 11H), 3.51–3.49 (m, 2H), 3.43–3.38 (m, 3H), 2.98 (s, 3H), 2.44 (s, 4H), 2.29–2.26 (m, 2H), 1.46–1.43 (m, 2H), 1.38–1.25 (m, 4H), 1.00 (s, 9H).</div></div><div id="sec4_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> <i>S</i>-((<i>R</i>)-1-(((((3<i>R</i>,5<i>S</i>)-1-((<i>S</i>)-2-(11-((<i>tert</i>-Butoxycarbonyl)amino)undecanamido)-3,3-dimethylbutanoyl)-5-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-3-yl)oxy)carbonyl)oxy)propan-2-yl) Methanesulfonothioate (<b>28</b>)</h4><div class="NLM_p last">TFA (0.1 mL) was added to a 23 °C solution of compound <b>26</b><a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> (86 mg, 0.12 mmol) in HFIP (2 mL). The reaction mixture was stirred at that temperature for 1 h and then concentrated <i>in vacuo</i> to give the crude deprotected amine (TFA salt, not shown, 76 mg, 87%) as a colorless oil. LCMS (5–95, AB, 1.5 min): RT = 0.61 min, <i>m</i>/<i>z</i> = 627.0 [M + H]<sup>+</sup>. Separately, a mixture of compound <b>27</b><a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> (27 mg, 0.09 mmol), DIEA (26 mg, 0.20 mmol), and HATU (33 mg, 0.09 mmol) was stirred in DMF (5 mL) at 23 °C for 10 min. A portion of the crude TFA salt obtained from compound <b>26</b> (50 mg, 0.07 mmol) was then added. The resulting mixture was stirred at 23 °C for 2 h; then, the mixture was concentrated under reduced pressure. The residue was purified by flash chromatography on silica gel eluting with 0–10% MeOH in DCM to give compound <b>28</b> (44 mg, 72%) as a colorless oil. LCMS (5–95, AB, 1.5 min): RT = 1.05 min, <i>m</i>/<i>z</i> = 932.4 [M + Na]<sup>+</sup>.</div></div><div id="sec4_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> <i>S</i>-((<i>R</i>)-1-(((((3<i>R</i>,5<i>S</i>)-1-((<i>S</i>)-2-(11-(4-(3,5-Difluoropyridin-2-yl)-10-methyl-7-((methylsulfonyl)methyl)-11-oxo-3,4,10,11-tetrahydro-1<i>H</i>-1,4,10-triazadibenzo[cd,<i>f</i>]azulene-6-carboxamido)undecanamido)-3,3-dimethylbutanoyl)-5-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-3-yl)oxy)carbonyl)oxy)propan-2-yl) Methanesulfonothioate (<b>9a</b>)</h4><div class="NLM_p last">To a solution of compound <b>28</b> (44 mg, 0.05 mmol) in HFIP (2 mL) at 23 °C was added TFA (0.1 mL). The mixture was stirred at 23 °C for 1 h and then concentrated <i>in vacuo</i> to give the crude deprotected amine (TFA salt, not shown, 44 mg, 99%) as a colorless oil. LCMS (5–95, AB, 1.5 min): RT = 0.69 min, <i>m</i>/<i>z</i> = 810.3 [M + H]<sup>+</sup>. Separately, a 23 °C solution of compound <b>17</b> (18 mg, 0.04 mmol), DIEA (15 mg, 0.12 mmol), and HATU (14 mg, 0.04 mmol) in DMF (5 mL) was stirred for 10 min. A portion of the crude TFA salt obtained from compound <b>28</b> (22 mg, 0.02 mmol) was then added, and the resulting mixture was stirred at 23 °C for 2 h. The mixture was subsequently purified directly by prep-HPLC (MeCN 43–73%/0.225% FA in H<sub>2</sub>O) to give compound <b>9a</b> (19 mg, 62%) as a white solid. LCMS (10–80, AB, 7.0 min): RT = 4.44 min, <i>m</i>/<i>z</i> = 1314.6 [M + Na]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.89 (brs, 1H), 8.95 (s, 1H), 8.58 (brs, 1H), 8.37 (brs, 1H), 8.03 (s, 1H), 7.88–7.84 (m, 2H), 7.72 (brs, 1H), 7.59–7.37 (m, 2H), 7.37–7.35 (s, 3H), 7.23 (brs, 2H), 5.91 (br, 2H), 5.19 (s, 2H), 4.39–4.34 (m, 8H), 4.28–4.22 (m, 1H), 3.82–3.80 (m, 2H), 3.59 (s, 3H), 3.53 (s, 3H), 3.10 (s, 1H), 2.87 (s, 4H), 2.63 (s, 1H), 2.40 (s, 3H), 2.30–2.20 (m, 2H), 2.19–2.05 (m, 3H), 1.38–1.36 (m, 6H), 0.91 (m, 10H), 0.87 (s, 9H).</div></div><div id="sec4_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> <i>S</i>-(3-(((((3<i>R</i>,5<i>S</i>)-1-((<i>S</i>)-2-(11-((<i>tert</i>-Butoxycarbonyl)amino)undecanamido)-3,3-dimethylbutanoyl)-5-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-3-yl)oxy)carbonyl)oxy)butan-2-yl) Methanesulfonothioate (<b>29</b>)</h4><div class="NLM_p last">TFA (0.2 mL) was added to a solution of compound <b>24</b><a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> (100 mg, 0.13 mmol) in HFIP (4.0 mL) at 23 °C, and the resulting mixture was stirred at that temperature for 1 h. The solution was then concentrated <i>in vacuo</i> to remove the volatiles to afford the crude deprotected amine (TFA salt, not shown, 100 mg, 98%) as a colorless oil. LCMS (5–95, AB, 1.5 min): RT = 0.72 min, <i>m</i>/<i>z</i> = 641.1 [M + H]<sup>+</sup>. Separately, HATU (61 mg, 0.16 mmol) and DIEA (69 mg, 0.53 mmol) were added sequentially to a solution of compound <b>27</b><a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> (40 mg, 0.13 mmol) in DCM (10 mL) at 23 °C. The mixture was stirred at 23 °C for 15 min; then, a solution of crude TFA salt obtained from compound <b>24</b> (100 mg, 0.13 mmol) in DCM (5 mL) was added. The resulting reaction mixture was stirred at 23 °C for an additional 1 h and then concentrated <i>in vacuo</i>. The residue was purified by prep-TLC (10% MeOH in DCM, <i>R<sub>f</sub></i> = 0.5) to afford compound <b>29</b> (85 mg, 61%) as a white solid. LCMS (5–95, AB, 1.5 min): RT = 0.96 min, <i>m</i>/<i>z</i> = 946.3 [M + Na]<sup>+</sup>.</div></div><div id="sec4_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> <i>S</i>-(3-(((((3<i>R</i>,5<i>S</i>)-1-((<i>S</i>)-2-(11-(4-(3,5-Difluoropyridin-2-yl)-10-methyl-7-((methylsulfonyl)methyl)-11-oxo-3,4,10,11-tetrahydro-1<i>H</i>-1,4,10-triazadibenzo[cd,<i>f</i>]azulene-6-carboxamido)undecanamido)-3,3-dimethylbutanoyl)-5-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-3-yl)oxy)carbonyl)oxy)butan-2-yl) Methanesulfonothioate (<b>9c</b>)</h4><div class="NLM_p last">TFA (0.2 mL) was added to a solution of compound <b>29</b> (85 mg, 0.09 mmol) in HFIP (4.0 mL) at 23 °C, and the resulting mixture was stirred at that temperature for 1 h. The solution was subsequently concentrated <i>in vacuo</i> to afford the crude deprotected amine (TFA salt, not shown, 86 mg, 100%) as a crude colorless oil. LCMS (5–95, AB, 1.5 min): RT = 0.78 min, <i>m</i>/<i>z</i> = 824.4 [M + H]<sup>+</sup>. To a 23 °C solution of compound <b>17</b> (20 mg, 0.04 mmol) in DMF (4 mL) were added HATU (20 mg, 0.05 mmol) and DIEA (26 mg, 0.20 mmol). The mixture was stirred at 23 °C for 10 min; then, a portion of the crude TFA salt obtained from compound <b>29</b> (49 mg, 0.05 mmol) was added. The resulting reaction solution was stirred at 23 °C for 1 h and then directly purified by prep-HPLC (MeCN 5–80%/0.225% FA in H<sub>2</sub>O) to afford compound <b>9c</b> (23 mg, 43%) as a white solid. LCMS (5–95, AB, 1.5 min): RT = 0.89 min, <i>m</i>/<i>z</i> = 1329.1 [M + Na]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.93 (s, 1H), 8.98 (s, 1H), 8.62–8.60 (m, 1H), 8.41–8.39 (m, 1H), 8.06 (d, <i>J</i> = 2.4 Hz, 1H), 7.91–7.87 (m, 2H), 7.75 (s, 1H), 7.63–7.57 (m, 2H), 7.43–7.37 (m, 3H), 7.27–7.26 (m, 2H), 5.22–5.19 (m, 2H), 4.98–4.94 (m, 2H), 4.47–4.35 (m, 4H), 4.27–4.09 (m, 4H), 3.84–3.80 (m, 2H), 3.75–3.70 (m, 2H), 3.62 (s, 3H), 3.54 (d, <i>J</i> = 2.8 Hz, 3H), 3.13 (brs, 3H), 2.90 (s, 3H), 2.67 (brs, 1H), 2.44 (s, 3H), 2.33–2.09 (m, 3H), 1.45–1.32 (m, 10H), 1.21 (s, 8H), 0.94 (s, 9H).</div></div><div id="sec4_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> 11-Aminoundecanoic Acid (<b>30</b>)</h4><div class="NLM_p last">Compound <b>27</b><a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> (30 mg, 0.10 mmol) was dissolved in 4 M HCl in 1,4-dioxane (2 mL) at 23 °C, and the resulting mixture was stirred at that temperature for 1 h. The mixture was then concentrated under reduced pressure to give the crude compound <b>30</b> (HCl salt, 24 mg, 100%) as a gray solid. This material was used without additional purification in the preparation of compound <b>31</b> below.</div></div><div id="sec4_1_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> 11-(4-(3,5-Difluoropyridin-2-yl)-10-methyl-7-((methylsulfonyl)methyl)-11-oxo-3,4,10,11-tetrahydro-1<i>H</i>-1,4,10-triazadibenzo[cd,<i>f</i>]azulene-6-carboxamido)undecanoic Acid (<b>31</b>)</h4><div class="NLM_p last">A solution of compound <b>17</b> (25 mg, 0.05 mmol), DIEA (19 mg, 0.15 mmol), and HATU (23 mg, 0.06 mmol) in DMF (3 mL) was stirred at 23 °C for 1 h. The crude compound <b>30</b> (HCl salt, 18 mg, 0.07 mmol) was then added. The resulting mixture was stirred at 23 °C for 2 h and then concentrated under reduced pressure. The residue was purified by prep-TLC (12% MeOH in DCM, <i>R<sub>f</sub></i> = 0.4) to give compound <b>31</b> (34 mg, 100%) as a gray solid. LCMS (5–95, AB, 1.5 min): RT = 0.82 min, <i>m</i>/<i>z</i> = 684.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> Allyl ((<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-Hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)carbamate (<b>33</b>)</h4><div class="NLM_p last">Allyl chloroformate (485 mg, 4.02 mmol) was added to a 0 °C mixture of compound <b>32</b><a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> (1.65 g, 3.83 mmol) and NaHCO<sub>3</sub> (1.61 g, 19.2 mmol) in 1:1 THF:H<sub>2</sub>O (34 mL). The resulting mixture was allowed to warm to 25 °C and stirred at that temperature for 12 h. After diluting with water (50 mL), the reaction mixture was extracted with EtOAc (3 × 50 mL), and the combined organic layers were washed with brine (50 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The residue was purified by flash column chromatography on silica gel eluting with 0–3% MeOH in DCM to give compound <b>33</b> (1.60 g, 81%) as a gray solid. LCMS (10–80, AB, 7.0 min): RT = 2.57 min, <i>m</i>/<i>z</i> = 537.1 [M + Na]<sup>+</sup>.</div></div><div id="sec4_1_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> Allyl ((2<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-((Hydroxyhydrophosphoryl)oxy)-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)carbamate (<b>34</b>)</h4><div class="NLM_p last">To a solution of compound <b>33</b> (450 mg, 0.87 mmol) in THF (5 mL) at −78 °C was added a solution of PCl<sub>3</sub> (500 mg, 3.64 mmol) in THF (2 mL) followed by a solution of Et<sub>3</sub>N (0.73 mL, 5.25 mmol) in THF (2 mL). The reaction mixture was stirred at −78 °C for 20 min and then allowed to warm to 23 °C. After stirring at 23 °C for 12 h, the reaction was quenched by sequential addition of H<sub>2</sub>O (2 mL) and saturated aqueous NaHCO<sub>3</sub> (5 mL). The resulting mixture was stirred at 23 °C for 10 min and then acidified with 1 N HCl to pH = 3 and concentrated. The residue was purified by prep-TLC (12% MeOH in DCM, <i>R<sub>f</sub></i> = 0.4) to afford the compound <b>34</b> (450 mg, 89% yield) as a colorless solid. LCMS (0–60, CD, 3.0 min): RT = 1.42 min, <i>m</i>/<i>z</i> = 579.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> Allyl ((2<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-((Hydroxy(1<i>H</i>-imidazol-1-yl)phosphoryl)oxy)-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)carbamate (<b>35</b>)</h4><div class="NLM_p last">1-(Trimethylsilyl)-1<i>H</i>-imidazole (436 mg, 3.11 mmol) was added to a solution of compound <b>34</b> (450 mg, 0.78 mmol), and Et<sub>3</sub>N (0.43 mL, 3.11 mmol) in 1:1 MeCN:CCl<sub>4</sub> (8 mL) was added at 23 °C. The reaction mixture was stirred at that temperature for 40 min; then, MeOH (0.1 mL) was added, and stirring was continued for an additional 10 min. The solvent was removed under reduced pressure, and the remaining solid was suspended in 5:1 MTBE:EtOAc (3 mL) and collected by filtration. The white solid thus obtained was washed with MTBE (3 mL) to afford compound <b>35</b> (450 mg, 90%). LCMS (5–95, AB, 1.5 min): RT = 0.73 min, <i>m</i>/<i>z</i> = 645.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> Allyl ((<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-((3-(2-((((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)amino)ethoxy)-1,1,3,3-tetraoxo-1l6,3l6-diphosphoxaneyl)oxy)-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)carbamate (<b>37</b>)</h4><div class="NLM_p last">To a 23 °C solution of compound <b>35</b> (450 mg, 0.70 mmol) and compound <b>36</b><a onclick="showRef(event, 'cit30b ref44'); return false;" href="javascript:void(0);" class="ref cit30b ref44">(30b,44)</a> (355 mg, 0.98 mmol) in DMF (5 mL) was added ZnCl<sub>2</sub> (1.0 M in THF, 6.98 mL, 6.98 mmol). The reaction mixture was stirred at 23 °C for 12 h and then quenched by the addition of 1 N HCl (5 mL). The volatiles were removed under reduced pressure, and the residue was purified by prep-HPLC (MeCN 22–42%/10 mM NH<sub>4</sub>HCO<sub>3</sub> in H<sub>2</sub>O) to afford compound <b>37</b> (250 mg, 38%) as a white solid. LCMS (5–95, AB, 1.5 min): RT = 1.75 min, <i>m</i>/<i>z</i> = 940.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> (9<i>H</i>-Fluoren-9-yl)methyl (2-((3-(((3<i>R</i>,5<i>S</i>)-1-((<i>S</i>)-2-Amino-3,3-dimethylbutanoyl)-5-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-3-yl)oxy)-1,1,3,3-tetraoxo-1l6,3l6-diphosphoxaneyl)oxy)ethyl)carbamate (<b>38</b>)</h4><div class="NLM_p last">To a solution of compound <b>37</b> (100 mg, 0.11 mmol) and 1,3-dimethylbarbituric acid (83 mg, 0.53 mmol) in DCM (1 mL) and MeOH (0.20 mL) was added Pd(PPh<sub>3</sub>)<sub>4</sub> (25 mg, 0.02 mmol) at 23 °C. The reaction mixture was stirred at 23 °C for 2 h and then concentrated <i>in vacuo</i>. The resulting residue was purified by prep-HPLC (MeCN 18–48%/10 mM NH<sub>4</sub>HCO<sub>3</sub> in H<sub>2</sub>O) to afford compound <b>38</b> (55 mg, 61%) as a white solid. LCMS (0–60, CD, 3.0 min): RT = 1.44 min, <i>m</i>/<i>z</i> = 856.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> (9<i>H</i>-Fluoren-9-yl)methyl (2-((3-(((3<i>R</i>,5<i>S</i>)-1-((<i>S</i>)-2-(11-(4-(3,5-Difluoropyridin-2-yl)-10-methyl-7-((methylsulfonyl)methyl)-11-oxo-3,4,10,11-tetrahydro-1<i>H</i>-1,4,10-triazadibenzo[cd,<i>f</i>]azulene-6-carboxamido)undecanamido)-3,3-dimethylbutanoyl)-5-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-3-yl)oxy)-1,1,3,3-tetraoxo-1l6,3l6-diphosphoxaneyl)oxy)ethyl)carbamate (<b>39</b>)</h4><div class="NLM_p last">A solution of compound <b>31</b> (120 mg, 0.18 mmol), DIEA (8.0 mg, 0.06 mmol), and HATU (22 mg, 0.06 mmol) in anhydrous DMF (2 mL) was stirred at 23 °C for 20 min. Compound <b>38</b> (50 mg, 0.06 mmol) was then added, and the resulting mixture was stirred at 23 °C for 2 h. The reaction mixture was subsequently filtered, and the filtrate was directly purified by prep-HPLC (MeCN 27–57%/0.05% NH<sub>4</sub>OH in H<sub>2</sub>O) to afford compound <b>39</b> (40 mg, 45%) as a white solid. LCMS (0–60, CD, 3.0 min): RT =1.97 min, <i>m</i>/<i>z</i> = 761.3 [M/2 + H]<sup>+</sup>.</div></div><div id="sec4_1_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> 4-(3,5-Difluoropyridin-2-yl)-<i>N</i>-(11-(((<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-((3-(2-(6-(2,5-dioxo-2,5-dihydro-1<i>H</i>-pyrrol-1-yl)hexanamido)ethoxy)-1,1,3,3-tetraoxo-1l6,3l6-diphosphoxaneyl)oxy)-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-11-oxoundecyl)-10-methyl-7-((methylsulfonyl)methyl)-11-oxo-3,4,10,11-tetrahydro-1<i>H</i>-1,4,10-triazadibenzo[cd,<i>f</i>]azulene-6-carboxamide (<b>9d</b>)</h4><div class="NLM_p last">Piperidine (11 mg, 0.13 mmol) was added to a solution of compound <b>39</b> (40 mg, 0.03 mmol) in DMF (1 mL) at 23 °C. The reaction mixture was stirred at 23 °C for 1 h and then filtered. The filtrate was purified by prep-HPLC (MeCN 23–53%/10 mM NH<sub>4</sub>HCO<sub>3</sub> in H<sub>2</sub>O) to afford the deprotected amine (not shown, 20 mg, 59% yield) as a white solid. LCMS (0–60, CD, 3.0 min): RT = 1.73 min, <i>m</i>/<i>z</i> = 650.3 [M/2 + H]<sup>+</sup>. A portion of this material (10 mg, 0.01 mmol) was added to a solution of compound <b>40</b> (6.0 mg, 0.02 mmol) and DIEA (2.0 mg, 0.02 mmol) in DMF (1 mL) at 23 °C. The mixture was stirred at 23 °C for 6 h and then filtered. The filtrate was directly purified by prep-HPLC (MeCN 25–55%/0.1% TFA in H<sub>2</sub>O) to afford compound <b>9d</b> (2.7 mg, 24%) as a white solid. LCMS (10–80, AB, 7.0 min): RT = 3.72 min, <i>m</i>/<i>z</i> = 746.8 [M/2 + H]<sup>+</sup>. HRMS (ESI, 0-95_1_4min_CD): <i>m</i>/<i>z</i> = 1492.51 [M + 1]<sup>+</sup>.</div></div><div id="sec4_1_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> Benzyl 4-(2-(((2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-14-Amino-2-(<i>tert</i>-butyl)-4-oxo-6,9,12-trioxa-3-azatetradecanoyl)-4-hydroxypyrrolidine-2-carboxamido)methyl)-5-(4-methylthiazol-5-yl)phenoxy)piperidine-1-carboxylate (<b>42</b>)</h4><div class="NLM_p last">A solution of compound <b>41</b> (150 mg, 0.16 mmol) in 1:3 TFA:DCM (4 mL) was stirred at 23 °C for 30 min. The reaction mixture was then concentrated to give the crude Boc-deprotected amine (TFA salt, 130 mg, 97%) as a colorless oil. This material was used in the next step without additional purification. LCMS (5–95, AB, 1.5 min): RT = 0.78 min, <i>m</i>/<i>z</i> = 875.1 [M + Na]<sup>+</sup>.</div></div><div id="sec4_1_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> Benzyl 4-(2-(((2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-15-(<i>tert</i>-Butyl)-1-(4-(3,5-difluoropyridin-2-yl)-10-methyl-7-((methylsulfonyl)methyl)-11-oxo-3,4,10,11-tetrahydro-1<i>H</i>-1,4,10-triazadibenzo[cd,<i>f</i>]azulen-6-yl)-1,13-dioxo-5,8,11-trioxa-2,14-diazahexadecan-16-oyl)-4-hydroxypyrrolidine-2-carboxamido)methyl)-5-(4-methylthiazol-5-yl)phenoxy)piperidine-1-carboxylate (<b>43</b>)</h4><div class="NLM_p last">DIEA (78 mg, 0.60 mmol) and HATU (68 mg, 0.18 mmol) were sequentially added to a 23 °C solution of compound <b>17</b> (75 mg, 0.15 mmol) in DMF (5 mL). The mixture was stirred at 23 °C for 10 min; then, crude compound <b>42</b> (145 mg, 0.15 mmol) was added. The resulting solution was stirred at 23 °C for 1 h and then concentrated <i>in vacuo</i>. The residue was purified by prep-TLC (10% MeOH in DCM, <i>R<sub>f</sub></i> = 0.5) to afford compound <b>43</b> (198 mg, 98%) as a pale-yellow oil. LCMS (5–95, AB, 1.5 min): RT = 0.85 min, <i>m</i>/<i>z</i> = 1358.0 [M + Na]<sup>+</sup>.</div></div><div id="sec4_1_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> 4-(3,5-Difluoropyridin-2-yl)-<i>N</i>-((<i>S</i>)-13-((2<i>S</i>,4<i>R</i>)-4-hydroxy-2-((4-(4-methylthiazol-5-yl)-2-(piperidin-4-yloxy)benzyl)carbamoyl)pyrrolidine-1-carbonyl)-14,14-dimethyl-11-oxo-3,6,9-trioxa-12-azapentadecyl)-10-methyl-7-((methylsulfonyl)methyl)-11-oxo-3,4,10,11-tetrahydro-1<i>H</i>-1,4,10-triazadibenzo[cd,<i>f</i>]azulene-6-carboxamide (<b>11</b>)</h4><div class="NLM_p last">To a 23 °C solution of compound <b>43</b> (160 mg, 0.12 mmol) in DCM (4 mL) was added TMSI (240 mg, 1.2 mmol). The reaction mixture was stirred at 23 °C for 1 h; then, MeOH (20 mL) was added, and stirring was continued at 23 °C for an additional 10 min. The mixture was concentrated <i>in vacuo</i>, and the residue was purified by prep-HPLC (MeCN 0–40%/0.1% HCl in H<sub>2</sub>O) to afford compound <b>11</b> (42 mg, 27%) as a white solid. LCMS (5–95, AB, 1.5 min): RT = 0.77 min, <i>m</i>/<i>z</i> = 1201.6 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 8.86 (s, 1H), 8.53 (s, 1H), 7.94 (s, 1H), 7.87 (d, <i>J</i> = 2.8 Hz, 1H), 7.69 (s, 1H), 7.47 (d, <i>J</i> = 7.6 Hz, 1H), 7.36 (s, 1H), 7.23–7.21 (m, 2H), 7.05–7.00 (m, 2H), 4.88 (brs, 2H), 4.69 (s, 1H), 4.60–4.49 (m, 5H), 4.38–4.34 (m, 1H), 4.01 (d, <i>J</i> = 4.0 Hz, 2H), 3.88–3.79 (m, 2H), 3.70–3.63 (m, 13 H), 3.51–3.44 (m, 4 H), 3.31–3.24 (m, 2H), 2.99 (s, 3H), 2.46 (s, 3H), 2.21–2.03 (s, 6H), 1.02 (s, 9H).</div></div><div id="sec4_1_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> 4-((<i>S</i>)-2-(1-((5-(2,5-Dioxo-2,5-dihydro-1<i>H</i>-pyrrol-1-yl)pentyl)carbamoyl)cyclobutane-1-carboxamido)-5-ureidopentanamido)benzyl 4-(2-(((2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-15-(<i>tert</i>-Butyl)-1-(4-(3,5-difluoropyridin-2-yl)-10-methyl-7-((methylsulfonyl)methyl)-11-oxo-3,4,10,11-tetrahydro-1<i>H</i>-1,4,10-triazadibenzo[cd,<i>f</i>]azulen-6-yl)-1,13-dioxo-5,8,11-trioxa-2,14-diazahexadecan-16-oyl)-4-hydroxypyrrolidine-2-carboxamido)methyl)-5-(4-methylthiazol-5-yl)phenoxy)piperidine-1-carboxylate (<b>11a</b>)</h4><div class="NLM_p last">DIEA (10 mg, 0.08 mmol) was added to a 23 °C solution of compound <b>11</b> (25 mg, 0.02 mmol) and compound <b>44</b><a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> in DMF (2 mL). The reaction solution was stirred at 23 °C for 2 h and then directly purified by prep-HPLC (MeCN 35–65%/0.225% FA in H<sub>2</sub>O) to afford compound <b>11a</b> (17 mg, 45%) as a white solid. LCMS (5–95, AB, 1.5 min): RT = 0.80 min, <i>m</i>/<i>z</i> = 900.4 [M/2 + H]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 8.85 (s, 1H), 7.93 (s, 1H), 7.87 (d, <i>J</i> = 2.8 Hz, 1H), 7.69 (s, 1H), 7.62–7.59 (m, 3H), 7.43 (d, <i>J</i> = 7.6 Hz, 1H), 7.35 (s, 1H), 7.30 (d, <i>J</i> = 8.8 Hz, 2H), 7.26 (s, 1H), 7.21 (s, 1H), 7.01–6.94 (m, 1H), 6.75 (s, 2H), 5.06 (brs, 1H), 4.89–4.67 (m, 12H), 4.65–4.61 (m, 1H), 4.01 (d, <i>J</i> = 4.8 Hz, 2H), 3.71–3.61 (m, 15H), 3.49–3.42 (m, 5H), 3.31–3.22 (m, 3H), 3.25–3.21 (m, 4H), 3.16–3.13 (m, 2H), 2.97 (s, 3H), 2.56–2.51 (m, 2H), 2.45 (s, 3H), 1.94–1.92 (m, 3H), 1.78–1.74 (m, 2H), 1.55–1.51 (m, 6H), 1.28–1.26 (m, 4H), 1.00 (s, 9H).</div></div><div id="sec4_1_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> Benzyl 4-(2-(((2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(11-((<i>tert</i>-Butoxycarbonyl)amino)undecanamido)-3,3-dimethylbutanoyl)-4-hydroxypyrrolidine-2-carboxamido)methyl)-5-(4-methylthiazol-5-yl)phenoxy)piperidine-1-carboxylate (<b>46</b>)</h4><div class="NLM_p last">A solution of compound <b>45</b><a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> (200 mg, 0.26 mmol) in 4.0 M HCl in EtOAc (2 mL) was stirred at 23 °C for 1 h. The mixture was concentrated under reduced pressure to give the crude deprotected amine (HCl salt, not shown, 183 mg, 100%) as a white solid. LCMS (5–95, AB, 1.5 min): RT = 0.77 min, <i>m</i>/<i>z</i> = 664.1 [M + H]<sup>+</sup>. Separately, HATU (199 mg, 0.52 mmol) and DIEA (101 mg, 0.78 mmol) were sequentially added to a solution of compound <b>27</b><a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> (95 mg, 0.31 mmol) in DMF (10 mL) at 23 °C. The reaction mixture was stirred at that temperature for 10 min; then, the crude amine salt obtained from compound <b>45</b> (183 mg, 0.26 mmol) was added. The reaction mixture was stirred at 23 °C for 1 h and then concentrated under reduced pressure. The residue was purified by prep-HPLC (MeCN 65–95%/0.225% FA in H<sub>2</sub>O) to afford compound <b>46</b> (90 mg, 36% yield) as a white solid. LCMS (5–95, AB, 1.5 min): RT = 0.97 min, <i>m</i>/<i>z</i> = 969.4 [M + Na]<sup>+</sup>.</div></div><div id="sec4_1_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> Benzyl 4-(2-(((2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(11-Aminoundecanamido)-3,3-dimethylbutanoyl)-4-hydroxypyrrolidine-2-carboxamido)methyl)-5-(4-methylthiazol-5-yl)phenoxy)piperidine-1-carboxylate (<b>47</b>)</h4><div class="NLM_p last">TFA (0.15 mL) was added to a solution of compound <b>46</b> (100 mg, 0.11 mmol) in HFIP (2.85 mL) at 23 °C. The mixture was stirred at 23 °C for 1 h and then concentrated to give the crude compound <b>47</b> (TFA salt, 101 mg, 100%). This material was used in the next step without additional purification. LCMS (5–95, AB, 1.5 min): RT = 0.85 min, <i>m</i>/<i>z</i> = 847.5 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> Benzyl 4-(2-(((2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(11-(4-(3,5-Difluoropyridin-2-yl)-10-methyl-7-((methylsulfonyl)methyl)-11-oxo-3,4,10,11-tetrahydro-1<i>H</i>-1,4,10-triazadibenzo[cd,<i>f</i>]azulene-6-carboxamido)undecanamido)-3,3-dimethylbutanoyl)-4-hydroxypyrrolidine-2-carboxamido)methyl)-5-(4-methylthiazol-5-yl)phenoxy)piperidine-1-carboxylate (<b>48</b>)</h4><div class="NLM_p last">DIEA (39 mg, 0.30 mmol) was added to a solution of compound <b>17</b> (30 mg, 0.06 mmol) and HATU (25 mg, 0.07 mmol) in DMF (3.0 mL) at 23 °C. The mixture was stirred at 23 °C for 10 min; then, the crude compound <b>47</b> (63 mg, 0.07 mmol) was added. The reaction mixture was stirred at 23 °C for an additional 1 h, after which it was concentrated under reduced pressure. The residue was purified by prep-HPLC (MeCN 60–90%/0.225% FA in H<sub>2</sub>O) to afford compound <b>48</b> (18 mg, 23%) as a white solid. LCMS (5–95, AB, 1.5 min): RT = 1.00 min, <i>m</i>/<i>z</i> = 1329.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> 4-(3,5-Difluoropyridin-2-yl)-<i>N</i>-(11-(((<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-hydroxy-2-((4-(4-methylthiazol-5-yl)-2-(piperidin-4-yloxy)benzyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-11-oxoundecyl)-10-methyl-7-((methylsulfonyl)methyl)-11-oxo-3,4,10,11-tetrahydro-1<i>H</i>-1,4,10-triazadibenzo[cd,<i>f</i>]azulene-6-carboxamide (<b>12</b>)</h4><div class="NLM_p last">TMSI (1.00 mL, 0.06 mmol) was added to a solution of compound <b>48</b> (16 mg, 0.01 mmol) in DCM (4.0 mL) at 23 °C, and the resulting mixture was stirred at that temperature for 0.5 h. MeOH (2.0 mL) and aqueous 1 M HCl (2.00 mL) were then added sequentially, and the resulting mixture was stirred at 23 °C for 15 min. The volatiles were removed <i>in vacuo</i>, and the residue was purified by prep-HPLC (MeCN 30–50%/0.225% FA in H<sub>2</sub>O) to afford compound <b>12</b> (10 mg, 70%) as a white solid. LCMS (5–95, AB, 1.5 min): RT = 0.71 min, <i>m</i>/<i>z</i> = 1195.6 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.97 (s, 1H), 8.43–8.40 (m, 3H), 8.06 (d, <i>J</i> = 2.0 Hz, 1H), 7.91 (s, 1H), 7.87–7.85 (m, 1H), 7.75 (s, 1H), 7.60–7.57 (m, 1H), 7.46 (d, <i>J</i> = 7.6 Hz, 1H), 7.26 (d, <i>J</i> = 3.2 Hz, 2H), 7.06 (s, 1H), 6.90 (d, <i>J</i> = 8.4 Hz, 1H), 5.92 (br, 1H), 5.18 (br, 2H), 4.68 (br, 1H), 4.53 (d, <i>J</i> = 9.2 Hz, 1H), 4.45 (t, <i>J</i> = 8.0 Hz, 1H), 4.38–4.34 (m, 2H), 4.17–4.14 (m, 2H), 3.65 (s, 3H), 3.62 (s, 5H), 3.13–3.08 (m, 3H), 2.90 (s, 3H), 2.84 (br, 3H), 2.44 (s, 3H), 2.27–2.22 (m, 1H), 2.12–1.94 (m, 4H), 1.75–1.68 (m, 2H), 1.44 (br, 4H), 1.21 (br, 14H), 0.91 (s, 9H).</div></div><div id="sec4_1_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> 4-((<i>S</i>)-2-(1-((5-(2,5-Dioxo-2,5-dihydro-1<i>H</i>-pyrrol-1-yl)pentyl)carbamoyl)cyclobutane-1-carboxamido)-5-ureidopentanamido)benzyl 4-(2-(((2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(11-(4-(3,5-difluoropyridin-2-yl)-10-methyl-7-((methylsulfonyl)methyl)-11-oxo-3,4,10,11-tetrahydro-1<i>H</i>-1,4,10-triazadibenzo[cd,<i>f</i>]azulene-6-carboxamido)undecanamido)-3,3-dimethylbutanoyl)-4-hydroxypyrrolidine-2-carboxamido)methyl)-5-(4-methylthiazol-5-yl)phenoxy)piperidine-1-carboxylate (<b>12a</b>)</h4><div class="NLM_p last">DIEA (1.7 mg, 0.01 mmol) was added to a 23 °C solution of compound <b>44</b><a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> (3.7 mg, 0.01 mmol) and compound <b>12</b> (4.0 mg, 0.0033 mmol) in DMF (3.0 mL). The mixture was stirred at 23 °C for 2 h and then directly purified by prep-HPLC (MeCN 38–68%/0.225% FA in H<sub>2</sub>O) to afford compound <b>12a</b> (2.5 mg, 41%) as a white solid. LCMS (5–95, AB, 1.5 min): RT = 0.76 min, <i>m</i>/<i>z</i> = 896.8 [M/2 + H]<sup>+</sup>.</div></div><div id="sec4_1_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> 2-(3-Amino-5-(2-(((benzyloxy)carbonyl)amino)ethoxy)phenethoxy)acetic Acid (<b>50</b>)</h4><div class="NLM_p last">TFA (2.0 mL) was added to a 23 °C solution of compound <b>49</b><a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> (273 mg, 0.50 mmol) in DCM (2 mL), and the mixture was stirred at that temperature for 1 h. The mixture was concentrated <i>in vacuo</i>, and the residue was purified by prep-HPLC (MeCN 21–51%/0.225% FA in H<sub>2</sub>O) to give compound <b>50</b> (FA salt, 100 mg, 40%) as a white solid. LCMS (5–95, AB, 1.5 min): RT = 0.72 min, <i>m</i>/<i>z</i> = 389.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> Benzyl (2-(3-Amino-5-(2-(2-(((<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-2-oxoethoxy)ethyl)phenoxy)ethyl)carbamate (<b>51</b>)</h4><div class="NLM_p last">To a 23 °C solution of compound <b>32</b><a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> (256 mg, 0.59 mmol), compound <b>50</b> (110 mg, 0.28 mmol), and HATU (119 mg, 0.31 mmol) in DMF (5 mL) was added DIEA (183 mg, 1.42 mmol). The mixture was stirred at 23 °C for 1 h and then concentrated under reduced pressure. The residue was purified by prep-TLC (10% MeOH in DCM, <i>R<sub>f</sub></i> = 0.5) to give compound <b>51</b> (120 mg, 53%) as a yellow solid. LCMS (5–95, AB, 1.5 min): RT = 0.86 min, <i>m</i>/<i>z</i> = 823.3 [M + Na]<sup>+</sup>.</div></div><div id="sec4_1_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> <i>N</i>-(2-(3-Amino-5-(2-(2-(((<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-2-oxoethoxy)ethyl)phenoxy)ethyl)-4-(3,5-difluoropyridin-2-yl)-10-methyl-7-((methylsulfonyl)methyl)-11-oxo-3,4,10,11-tetrahydro-1<i>H</i>-1,4,10-triazadibenzo[cd,<i>f</i>]azulene-6-carboxamide (<b>13</b>)</h4><div class="NLM_p last">TMSI (100 mg, 0.50 mmol) was added to a solution of compound <b>51</b> (80 mg, 0.10 mmol) in DCM (5 mL) at 23 °C, and the resulting mixture was stirred at 23 °C for 1 h. The volatiles were removed <i>in vacuo</i>, and the resulting residue was triturated with EtOAc (3 × 10 mL) to afford the crude deprotected amine (not shown, 90 mg, 98%) as a yellow solid. LCMS (5–95, AB, 1.5 min): RT = 0.66 min, <i>m</i>/<i>z</i> = 667.5 [M + H]<sup>+</sup>. Separately, DIEA (65 mg, 0.50 mmol) was added to a 23 °C solution of compound <b>17</b> (50 mg, 0.10 mmol) and HATU (42 mg, 0.11 mmol) in DMF (8 mL). The mixture was stirred at 23 °C for 10 min; then, crude deprotected amine obtained from compound <b>51</b> (90 mg, 0.10 mmol) was added. The resulting mixture was stirred at 23 °C for 1 h and then concentrated under reduced pressure. Purification of the residue by prep-TLC (10% MeOH in DCM, <i>R<sub>f</sub></i> = 0.5) gave compound <b>13</b> (80 mg, 70%) as a yellow solid. LCMS (5–95, AB, 1.5 min): RT = 0.82 min, <i>m</i>/<i>z</i> = 1150.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> 4-((<i>S</i>)-2-(1-((5-(2,5-Dioxo-2,5-dihydro-1<i>H</i>-pyrrol-1-yl)pentyl)carbamoyl)cyclobutane-1-carboxamido)-5-ureidopentanamido)benzyl (3-(2-(4-(3,5-difluoropyridin-2-yl)-10-methyl-7-((methylsulfonyl)methyl)-11-oxo-3,4,10,11-tetrahydro-1<i>H</i>-1,4,10-triazadibenzo[cd,<i>f</i>]azulene-6-carboxamido)ethoxy)-5-(2-(2-(((<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-2-oxoethoxy)ethyl)phenyl)carbamate (<b>13a</b>)</h4><div class="NLM_p last">To a solution of compound <b>13</b> (50 mg, 0.04 mmol) and compound <b>44</b><a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> (96 mg, 0.13 mmol) in DMF (10 mL) at 23 °C were added HOBt (12 mg, 0.09 mmol) and pyridine (34 mg, 0.44 mmol). The mixture was stirred at 38 °C for 1 h. The mixture was then concentrated, and the residue was purified by prep-HPLC (MeCN 32–62%/0.225% FA in H<sub>2</sub>O) to give compound <b>13a</b> (15 mg, 19%) as a white solid. LCMS (5–95, AB, 1.5 min): RT = 0.75 min, <i>m</i>/<i>z</i> = 873.8 [M/2 + H]<sup>+</sup>. HRMS (ESI, 0–95_1_4min): <i>m</i>/<i>z</i> = 1745.66 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> Benzyl (2-(3-Amino-5-(2-(2-(((<i>S</i>)-1-((2S,4S)-4-hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-2-oxoethoxy)ethyl)phenoxy)ethyl)carbamate (<b>53</b>)</h4><div class="NLM_p last">To a 23 °C solution of compound <b>50</b> (185 mg, 0.37 mmol) and compound <b>52</b><a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> (360 mg, 0.77 mmol) in DMF (5.0 mL) were added DIEA (237 mg, 1.84 mmol) and HATU (154 mg, 0.40 mmol). The formed mixture was stirred at 23 °C for 0.5 h and then concentrated under reduced pressure. The residue was purified by prep-TLC (8% MeOH in DCM) to afford compound <b>53</b> (220 mg, 69%) as a pale-yellow solid. LCMS (5–95, AB, 1.5 min): RT = 0.72 min, <i>m</i>/<i>z</i> = 801.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> <i>N</i>-(2-(3-Amino-5-(2-(2-(((<i>S</i>)-1-((2<i>S</i>,4<i>S</i>)-4-hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-2-oxoethoxy)ethyl)phenoxy)ethyl)-4-(3,5-difluoropyridin-2-yl)-10-methyl-7-((methylsulfonyl)methyl)-11-oxo-3,4,10,11-tetrahydro-1<i>H</i>-1,4,10-triazadibenzo[cd,<i>f</i>]azulene-6-carboxamide (<b>14</b>)</h4><div class="NLM_p last">TMSI (275 mg, 1.37 mmol) was added to a solution of compound <b>53</b> (220 mg, 0.27 mmol) in DCM (4 mL) at 23 °C. The resulting mixture was stirred at 23 °C for 1 h; then, the volatiles were removed <i>in vacuo</i>. The residue was triturated with EtOAc (3 × 10 mL) to afford the crude deprotected amine (not shown, 253 mg, 100%) as a yellow solid. LCMS (5–95, AB, 1.5 min): RT = 0.74 min, <i>m</i>/<i>z</i> = 667.3 [M + H]<sup>+</sup>. Separately, DIEA (177 mg, 1.37 mmol) was added to a 23 °C solution of compound <b>17</b> (137 mg, 0.27 mmol) and HATU (115 mg, 0.30 mmol) in DMF (5 mL). The mixture was stirred at 23 °C for 10 min; then, the crude deprotected amine obtained from compound <b>53</b> (253 mg, 0.27 mmol) was added. The resulting mixture was stirred at 23 °C for 1 h and then concentrated under reduced pressure. Purification of the residue by prep-TLC (12% MeOH in DCM, <i>R<sub>f</sub></i> = 0.5) gave compound <b>14</b> (110 mg, 32%) as a pale-yellow solid. LCMS (10–80, AB, 2.0 min): RT = 0.98 min, <i>m</i>/<i>z</i> = 575.3 [M/2 + H]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 8.84 (s, 1H), 7.94 (s, 1H), 7.86 (d, <i>J</i> = 2.4 Hz, 1H), 7.69 (s, 1H), 7.41–7.39 (m, 5H), 7.31 (s, 1H), 7.23–7.21 (m, 3H), 4.64–4.55 (m, 8H), 4.53–4.46 (m, 3H), 4.40–4.34 (m, 3H), 4.11–4.07 (m, 2H), 3.98–3.94 (m, 2H), 3.73–3.69 (m, 5H), 3.68–3.63 (m, 3H), 2.89 (s, 3H), 2.79–2.73 (m, 3H), 2.44 (s, 4H), 2.41–2.38 (m, 1H), 2.00–1.92 (m, 2H), 0.97 (s, 9H).</div></div><div id="sec4_1_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> 4-((<i>S</i>)-2-(1-((5-(2,5-Dioxo-2,5-dihydro-1<i>H</i>-pyrrol-1-yl)pentyl)carbamoyl)cyclobutane-1-carboxamido)-5-ureidopentanamido)benzyl (3-(2-(4-(3,5-difluoropyridin-2-yl)-10-methyl-7-((methylsulfonyl)methyl)-11-oxo-3,4,10,11-tetrahydro-1<i>H</i>-1,4,10-triazadibenzo[cd,<i>f</i>]azulene-6-carboxamido)ethoxy)-5-(2-(2-(((<i>S</i>)-1-((2S,4S)-4-hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-2-oxoethoxy)ethyl)phenyl)carbamate (<b>14a</b>)</h4><div class="NLM_p last">To a 23 °C solution of compound <b>14</b> (100 mg, 0.09 mmol) and compound <b>44</b><a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> (192 mg, 0.26 mmol) in DMF (5 mL) were added HOBt (24 mg, 0.17 mmol) and pyridine (69 mg, 0.87 mmol). The mixture was stirred at 38 °C for 24 h and then concentrated under reduced pressure. The residue was purified by prep-HPLC (MeCN 40–50%/0.225% FA in H<sub>2</sub>O) to give compound <b>14a</b> (24 mg, 15%) as a pale-yellow solid. LCMS (5–95, AB, 1.5 min): RT = 0.77 min, <i>m</i>/<i>z</i> = 874.1 [M/2 + H]<sup>+</sup>. HRMS (ESI, 5–95 AB, 4 min): <i>m</i>/<i>z</i> = 1746.68 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> 4-((<i>S</i>)-2-(1-((5-(2,5-Dioxo-2,5-dihydro-1<i>H</i>-pyrrol-1-yl)pentyl)carbamoyl)cyclobutane-1-carboxamido)-5-ureidopentanamido)benzyl (3-(2-(((benzyloxy)carbonyl)amino)ethoxy)-5-(2-(2-(((<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-2-oxoethoxy)ethyl)phenyl)carbamate (<b>16</b>)</h4><div class="NLM_p last">To a 23 °C solution of compound <b>51</b> (100 mg, 0.12 mmol) and compound <b>44</b><a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> (138 mg, 0.19 mmol) in DMF (5 mL) were added HOBt (17 mg, 0.12 mmol) and pyridine (99 mg, 1.25 mmol). The mixture was stirred at 38 °C for 24 h and then concentrated <i>in vacuo</i>. Purification of the residue by prep-HPLC (MeCN 43–63%/0.225% FA in H<sub>2</sub>O) provided compound <b>16</b> (21 mg, 12%) as a pale-yellow solid. LCMS (5–95, AB, 1.5 min): RT = 0.91 min, <i>m</i>/<i>z</i> = 699.8 [M/2 + H]<sup>+</sup>.</div></div><div id="sec4_1_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> <i>tert</i>-Butyl 2-(3-(2-Aminoethoxy)-5-hydroxyphenethoxy)acetate (<b>55</b>)</h4><div class="NLM_p last">To a 23 °C solution of compound <b>54</b><a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> (270 mg, 0.61 mmol) in THF (5 mL) was added 10% palladium on carbon (194 mg). The mixture was stirred at 23 °C under H<sub>2</sub> (15 psi) for 1 h whereupon TLC analysis (30% EtOAc in petroleum ether) showed that most of the starting material was consumed. The mixture was filtered, and the filtrate was concentrated to give the crude compound <b>55</b> (188 mg, 100%) as a colorless oil. LCMS (5–95, AB, 1.5 min): RT = 0.71 min, <i>m</i>/<i>z</i> = 311.9 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> <i>tert</i>-Butyl 2-(3-(2-(4-(3,5-Difluoropyridin-2-yl)-10-methyl-7-((methylsulfonyl)methyl)-11-oxo-3,4,10,11-tetrahydro-1<i>H</i>-1,4,10-triazadibenzo[cd,f]azulene-6-carboxamido)ethoxy)-5-hydroxyphenethoxy)acetate (<b>56</b>)</h4><div class="NLM_p last">Compound <b>17</b> (121 mg, 0.24 mmol), HATU (92 mg, 0.24 mmol), and DIEA (104 mg, 0.80 mmol) were added sequentially to a solution of crude compound <b>55</b> (50 mg, 0.16 mmol) in DMF (4 mL) at 23 °C. The mixture was stirred at 23 °C for 2 h and then concentrated under reduced pressure. The residue was purified by column chromatography on silica gel eluting with 0–10% MeOH in DCM to give compound <b>56</b> (85 mg, 60%) as a white solid. LCMS (5–95, AB, 1.5 min): RT = 0.73 min, <i>m</i>/<i>z</i> = 794.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> 4-(3,5-Difluoropyridin-2-yl)-<i>N</i>-(2-(3-hydroxy-5-(2-(2-(((<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-2-oxoethoxy)ethyl)phenoxy)ethyl)-10-methyl-7-((methylsulfonyl)methyl)-11-oxo-3,4,10,11-tetrahydro-1<i>H</i>-1,4,10-triazadibenzo[cd,<i>f</i>]azulene-6-carboxamide (<b>15</b>)</h4><div class="NLM_p last">TFA (2.0 mL) was added to a 23 °C solution of compound <b>56</b> (85 mg, 0.11 mmol) in DCM (2 mL). The mixture was stirred at that temperature for 1 h and then concentrated <i>in vacuo</i> to give the crude deprotected carboxylic acid (not shown, 78 mg, 99%) as a white solid. LCMS (5–95, AB, 1.5 min): RT = 0.76 min, <i>m</i>/<i>z</i> = 738.2 [M + H]<sup>+</sup>. This material was dissolved in DMF (4 mL) at 23 °C, and compound <b>32</b><a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> (59 mg, 0.14 mmol), HATU (60 mg, 0.16 mmol), and DIEA (41 mg, 0.32 mmol) were added sequentially. The formed mixture was stirred at 23 °C for 2 h and then concentrated under reduced pressure. The residue was purified by sequential prep-TLC (10% MeOH in DCM <i>R<sub>f</sub></i> = 0.4) and prep-HPLC (MeCN 27–58%/ 0.225% FA in H<sub>2</sub>O) methods to give compound <b>15</b> (26 mg, 21%) as a white solid. LCMS (10–80, AB, 7.0 min): RT = 3.37 min, <i>m</i>/<i>z</i> = 1151.4 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 8.82 (s, 2H), 7.93 (s, 1H), 7.84 (d, <i>J</i> = 2.4 Hz, 1H), 7.67 (s, 1H), 7.43 (d, <i>J</i> = 10.8 Hz, 1H), 7.40–7.38 (m, 3H), 7.37–7.35 (m, 2H), 7.29 (s, 1H), 7.18 (s, 2H), 6.36–6.34 (m, 2H), 6.22–6.20 (m, 1H), 4.65–4.59 (m, 8H), 4.48 (brs, 2H), 4.36–4.32 (m, 1H), 4.09–4.06 (m, 2H), 3.96 (d, <i>J</i> = 2.8 Hz, 2H), 3.88–3.85 (m, 1H), 3.80–3.76 (m, 1H), 3.72–3.64 (m, 6H), 2.90 (s, 3H), 2.78 (t, <i>J</i> = 6.8 Hz, 2H), 2.42 (s, 3H), 2.24–2.19 (m, 2H), 2.09–2.03 (m, 2H), 0.98 (s, 9H).</div></div><div id="sec4_1_44" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> (9<i>H</i>-Fluoren-9-yl)methyl(<i>S</i>)-(1-((4-(bromomethyl)phenyl)amino)-1-oxo-5-ureidopentan-2-yl)carbamate (<b>58</b>)</h4><div class="NLM_p last">Compound <b>57</b><a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> (1.00 g, 1.99 mmol) was dissolved in 35% HBr in HOAc (15 mL) at 23 °C. The mixture was stirred at 23 °C for 2 h and then poured into 0 °C H<sub>2</sub>O (20 mL). The resulting precipitate was collected by filtration and washed with MTBE (2 × 20 mL) to give the crude compound <b>58</b> (1.00 g, 89%) as a gray solid. This material was used in the next step without additional purification. LCMS (5–95, AB, 1.5 min): RT = 0.783 min, <i>m</i>/<i>z</i> = 566.8 [M + 2 + H]<sup>+</sup>.</div></div><div id="sec4_1_45" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> <i>tert</i>-Butyl 2-(3-(2-((<i>tert</i>-Butoxycarbonyl)amino)ethoxy)-5-hydroxyphenethoxy)acetate (<b>59</b>)</h4><div class="NLM_p last">To a 23 °C solution of compound <b>55</b> (200 mg, 0.64 mmol) and Et<sub>3</sub>N (98 mg, 0.96 mmol) in DCM (10 mL) was added Boc<sub>2</sub>O (140 mg, 0.64 mmol). The reaction mixture was stirred at 23 °C for 1 h and then concentrated <i>in vacuo</i>. The residue was purified by prep-TLC (5% MeOH in DCM, <i>R<sub>f</sub></i> = 0.5) to give compound <b>59</b> (260 mg, 87%) as a yellow oil. LCMS (5–95, AB, 1.5 min): RT = 0.93 min, <i>m</i>/<i>z</i> = 434.2 [M + Na – 100]<sup>+</sup>.</div></div><div id="sec4_1_46" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> <i>tert</i>-Butyl (<i>S</i>)-2-(3-((4-(2-((((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)amino)-5-ureidopentanamido)benzyl)oxy)-5-(2-((<i>tert</i>-butoxycarbonyl)amino)ethoxy)phenethoxy)acetate (<b>60</b>)</h4><div class="NLM_p last">A mixture of Cs<sub>2</sub>CO<sub>3</sub> (285 mg, 0.87 mmol) and compound <b>59</b> (120 mg, 0.29 mmol) in DMF (5 mL) was stirred at 23 °C for 10 min. Crude compound <b>58</b> (495 mg, 0.87 mmol) was then added, and the resulting mixture was stirred at 23 °C for 1 h. The mixture was diluted with H<sub>2</sub>O (15 mL) and extracted with EtOAc (3 ×10 mL). The combined organic layers were washed with brine (2 × 10 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The residue was purified by prep-TLC (10% MeOH in DCM, <i>R<sub>f</sub></i> = 0.4) to afford compound <b>60</b> (50 mg, 19%) as a white solid. LCMS (5–95, AB, 1.5 min): RT =1.02 min, <i>m</i>/<i>z</i> = 897.4 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_47" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> (<i>S</i>)-2-(3-((4-(2-((((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)amino)-5-ureidopentanamido)benzyl)oxy)-5-(2-aminoethoxy)phenethoxy)acetic Acid (<b>61</b>)</h4><div class="NLM_p last">To a 23 °C solution of compound <b>60</b> (50 mg, 0.055 mmol) in HFIP (4.0 mL) was added TFA (0.2 mL). The reaction solution was stirred at 23 °C for 1 h and then concentrated <i>in vacuo</i> to afford the crude compound <b>61</b> (TFA salt, 72 mg, 99%) as a yellow oil. This material was used in the next step without additional purification. LCMS (5–95, AB, 1.5 min): RT = 0.66 min, <i>m</i>/<i>z</i> = 740.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_48" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> (<i>S</i>)-2-(3-((4-(2-((((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)amino)-5-ureidopentanamido)benzyl)oxy)-5-(2-(4-(3,5-difluoropyridin-2-yl)-10-methyl-7-((methylsulfonyl)methyl)-11-oxo-3,4,10,11-tetrahydro-1<i>H</i>-1,4,10-triazadibenzo[cd,<i>f</i>]azulene-6-carboxamido)ethoxy)phenethoxy)acetic Acid (<b>62</b>)</h4><div class="NLM_p last">A solution of compound <b>17</b> (50 mg, 0.10 mmol), DIEA (39 mg, 0.30 mmol), and HATU (42 mg, 0.11 mmol) in DMF (5 mL) was stirred at 23 °C for 10 min. The crude compound <b>61</b> (68 mg, 0.08 mmol) was then added, and the resulting mixture was stirred at 23 °C for 1 h. The reaction mixture was subsequently concentrated, and the residue was triturated with DCM (10 mL). The precipitate that formed was collected by filtration and was washed with DCM (5 mL) to afford compound <b>62</b> (50 mg, 41%) as a yellow solid. LCMS (5–95, AB, 1.5 min): RT = 0.87 min, <i>m</i>/<i>z</i> = 1222.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_49" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> (9<i>H</i>-Fluoren-9-yl)methyl ((<i>S</i>)-1-((4-((3-(2-(4-(3,5-Difluoropyridin-2-yl)-10-methyl-7-((methylsulfonyl)methyl)-11-oxo-3,4,10,11-tetrahydro-1<i>H</i>-1,4,10-triazadibenzo[cd,<i>f</i>]azulene-6-carboxamido)ethoxy)-5-(2-(2-(((<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-2-oxoethoxy)ethyl)phenoxy)methyl)phenyl)amino)-1-oxo-5-ureidopentan-2-yl)carbamate (<b>63</b>)</h4><div class="NLM_p last">To a solution of compound <b>62</b> (50 mg, 0.04 mmol) in DMF (4 mL) were sequentially added <b>32</b><a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> (21 mg, 0.05 mmol), HATU (17 mg, 0.04 mmol), and DIEA (16 mg, 0.12 mmol). The mixture was stirred at 23 °C for 2 h and then concentrated under reduced pressure. The residue was purified by prep-TLC (10% MeOH in DCM, <i>R<sub>f</sub></i> = 0.3) to give compound 64 (15 mg, 22%) as a white solid. LCMS (5–95, AB, 1.5 min): RT = 0.75 min, <i>m</i>/<i>z</i> = 817.8 [M/2 + H]<sup>+</sup>.</div></div><div id="sec4_1_50" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> <i>N</i>-(2-(3-((4-((<i>S</i>)-2-Qmino-5-ureidopentanamido)benzyl)oxy)-5-(2-(2-(((<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-2-oxoethoxy)ethyl)phenoxy)ethyl)-4-(3,5-difluoropyridin-2-yl)-10-methyl-7-((methylsulfonyl)methyl)-11-oxo-3,4,10,11-tetrahydro-1<i>H</i>-1,4,10-triazadibenzo[cd,<i>f</i>]azulene-6-carboxamide (<b>64</b>)</h4><div class="NLM_p last">A mixture of compound <b>63</b> (15 mg, 0.01 mmol) and piperidine (0.01 mL, 0.02 mmol) in DMF (2 mL) was stirred at 23 °C for 2 h. The mixture was then concentrated to give the crude compound <b>64</b> (12 mg, 93%) as a white solid. This material was used in the next step without additional purification. LCMS (5–95, AB, 1.5 min): RT = 0.77 min, <i>m</i>/<i>z</i> = 1412.9 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_51" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> <i>N</i>-((<i>S</i>)-1-((4-((3-(2-(4-(3,5-Difluoropyridin-2-yl)-10-methyl-7-((methylsulfonyl)methyl)-11-oxo-3,4,10,11-tetrahydro-1<i>H</i>-1,4,10-triazadibenzo[cd,<i>f</i>]azulene-6-carboxamido)ethoxy)-5-(2-(2-(((<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-2-oxoethoxy)ethyl)phenoxy)methyl)phenyl)amino)-1-oxo-5-ureidopentan-2-yl)-<i>N</i>-(5-(2,5-dioxo-2,5-dihydro-1<i>H</i>-pyrrol-1-yl)pentyl)cyclobutane-1,1-dicarboxamide (<b>15a</b>)</h4><div class="NLM_p last">To a solution of compound <b>64</b> (12 mg, 0.01 mmol) in DMF (4 mL) were sequentially added compound <b>65</b><a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> (3.2 mg, 0.01 mmol), HATU (4.9 mg, 0.01 mmol), and DIEA (3.5 mg, 0.03 mmol). The mixture was stirred at 23 °C for 1 h and then concentrated <i>in vacuo</i>. The residue was purified by prep-HPLC (MeCN 37–67%/0.225% FA in H<sub>2</sub>O) to afford compound <b>15a</b> (4.9 mg, 33%) as a white solid. LCMS (5–95, AB, 1.5 min): RT = 0.84 min, <i>m</i>/<i>z</i> = 852.3 [M/2 + H]<sup>+</sup>. HRMS (ESI, 0–95_1_4 min): <i>m</i>/<i>z</i> = 1702.68 [M + H]<sup>+</sup>.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i71"><a href="/doi/suppl/10.1021/acs.jmedchem.0c01846" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30705" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30705" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c01846?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01846</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">PC3-S1 cell line generation and characterization, in vitro inhibition of MYC expression assay, BRD4 degradation assays (PC3-S1 line), PC3-S1 antiproliferation assays, VHL-binding assessments, IAM assessments, molecular docking studies, intracellular quantitation of compound <b>8</b>, BRD4 BD1 and BD2 binding assessments, cocrystal structure of compound <b>6</b> bound to BRD4 BD1, analysis of the BRD4/VHL/<b>9</b> ternary complex crystal structure, BRD4–VHL-degrader ternary complex assessments, whole blood stability studies, PC3-S1 BRD4-degradation/antiproliferation relationships, LNCaP antiproliferation assays, in vivo efficacy studies, ADC in vivo stability and PK data determination, in vitro human megakaryocytopoiesis model, full chemical structures of chimeric BRD4 degraders and related linker-drugs, analytical LCMS methods, synthesis of antibody–drug conjugates, ADC mechanisms of action, ADC unit conversion example (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings of <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> compounds (<b>1</b>–<b>3</b>, <b>5</b>–<b>15</b>) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Molecular formula strings of new compounds in <a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a>–<a class="ref internalNav" href="#sch9" aria-label="9">9</a> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_003.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Compound characterization data; HPLC and LCMS reports (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_004.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf">jm0c01846_si_001.pdf (2.35 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_002.csv">jm0c01846_si_002.csv (7.91 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_003.csv">jm0c01846_si_003.csv (2.49 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_004.pdf">jm0c01846_si_004.pdf (8.19 MB)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">A. Cocrystal structure of compound <b>6</b> bound to BRD4 BD1: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7KHL">7KHL</a>.B. Crystal structure of the ternary complex formed among VHL, BRD4 BD1, and compound <b>9</b>: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7KHH">7KHH</a>. Authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.0c01846" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_70554" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_70554" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter S. Dragovich</span> - <span class="hlFld-Affiliation affiliation">Genentech
Inc., 1 DNA Way, South San Francisco, California 94080, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7372-2862" title="Orcid link">http://orcid.org/0000-0001-7372-2862</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#15716774727a637c767d3b65706170675572707b703b767a78"><span class="__cf_email__" data-cfemail="15716774727a637c767d3b65706170675572707b703b767a78">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Thomas H. Pillow</span> - <span class="hlFld-Affiliation affiliation">Genentech
Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Robert A. Blake</span> - <span class="hlFld-Affiliation affiliation">Genentech
Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jack D. Sadowsky</span> - <span class="hlFld-Affiliation affiliation">Genentech
Inc., 1 DNA Way, South San Francisco, California 94080, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0712-5450" title="Orcid link">http://orcid.org/0000-0002-0712-5450</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Emel Adaligil</span> - <span class="hlFld-Affiliation affiliation">Genentech
Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Pragya Adhikari</span> - <span class="hlFld-Affiliation affiliation">Genentech
Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jinhua Chen</span> - <span class="hlFld-Affiliation affiliation">WuXi
AppTec, 288 Fute Zhong
Road, Waigaoqiao Free Trade Zone, Shanghai 200131, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nicholas Corr</span> - <span class="hlFld-Affiliation affiliation">Genentech
Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Josefa dela Cruz-Chuh</span> - <span class="hlFld-Affiliation affiliation">Genentech
Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Geoffrey Del Rosario</span> - <span class="hlFld-Affiliation affiliation">Genentech
Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Aaron Fullerton</span> - <span class="hlFld-Affiliation affiliation">Genentech
Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Steven J. Hartman</span> - <span class="hlFld-Affiliation affiliation">Genentech
Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fan Jiang</span> - <span class="hlFld-Affiliation affiliation">Viva
Biotech, Structural Biology, 334 Aidisheng Road, Zhangjiang High-Tech Park, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><sup>∥</sup><span class="hlFld-ContribAuthor strong">Susan Kaufman</span> - <span class="hlFld-Affiliation affiliation">Genentech
Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tracy Kleinheinz</span> - <span class="hlFld-Affiliation affiliation">Genentech
Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Katherine R. Kozak</span> - <span class="hlFld-Affiliation affiliation">Genentech
Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Liling Liu</span> - <span class="hlFld-Affiliation affiliation">Genentech
Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ying Lu</span> - <span class="hlFld-Affiliation affiliation">WuXi
AppTec, 288 Fute Zhong
Road, Waigaoqiao Free Trade Zone, Shanghai 200131, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Melinda M. Mulvihill</span> - <span class="hlFld-Affiliation affiliation">Genentech
Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jeremy M. Murray</span> - <span class="hlFld-Affiliation affiliation">Genentech
Inc., 1 DNA Way, South San Francisco, California 94080, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2161-9326" title="Orcid link">http://orcid.org/0000-0002-2161-9326</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Aimee O’Donohue</span> - <span class="hlFld-Affiliation affiliation">Genentech
Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rebecca K. Rowntree</span> - <span class="hlFld-Affiliation affiliation">Genentech
Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">William S. Sawyer</span> - <span class="hlFld-Affiliation affiliation">Genentech
Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Leanna R. Staben</span> - <span class="hlFld-Affiliation affiliation">Genentech
Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">John Wai</span> - <span class="hlFld-Affiliation affiliation">WuXi
AppTec, 288 Fute Zhong
Road, Waigaoqiao Free Trade Zone, Shanghai 200131, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jian Wang</span> - <span class="hlFld-Affiliation affiliation">WuXi
AppTec, 288 Fute Zhong
Road, Waigaoqiao Free Trade Zone, Shanghai 200131, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">BinQing Wei</span> - <span class="hlFld-Affiliation affiliation">Genentech
Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wentao Wei</span> - <span class="hlFld-Affiliation affiliation">Viva
Biotech, Structural Biology, 334 Aidisheng Road, Zhangjiang High-Tech Park, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zijin Xu</span> - <span class="hlFld-Affiliation affiliation">WuXi
AppTec, 288 Fute Zhong
Road, Waigaoqiao Free Trade Zone, Shanghai 200131, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hui Yao</span> - <span class="hlFld-Affiliation affiliation">WuXi
AppTec, 288 Fute Zhong
Road, Waigaoqiao Free Trade Zone, Shanghai 200131, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shang-Fan Yu</span> - <span class="hlFld-Affiliation affiliation">Genentech
Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Donglu Zhang</span> - <span class="hlFld-Affiliation affiliation">Genentech
Inc., 1 DNA Way, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hongyan Zhang</span> - <span class="hlFld-Affiliation affiliation">WuXi
AppTec, 288 Fute Zhong
Road, Waigaoqiao Free Trade Zone, Shanghai 200131, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shenhua Zhang</span> - <span class="hlFld-Affiliation affiliation">Viva
Biotech, Structural Biology, 334 Aidisheng Road, Zhangjiang High-Tech Park, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yongxin Zhao</span> - <span class="hlFld-Affiliation affiliation">WuXi
AppTec, 288 Fute Zhong
Road, Waigaoqiao Free Trade Zone, Shanghai 200131, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hao Zhou</span> - <span class="hlFld-Affiliation affiliation">WuXi
AppTec, 288 Fute Zhong
Road, Waigaoqiao Free Trade Zone, Shanghai 200131, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaoyu Zhu</span> - <span class="hlFld-Affiliation affiliation">WuXi
AppTec, 288 Fute Zhong
Road, Waigaoqiao Free Trade Zone, Shanghai 200131, China</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br><sup>∥</sup>Susan Kaufman passed away in August 2020.<br /></br><br /></br></div></li></ul></div><div class="ack" id="ACK-d7e7966-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i74">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12015" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12015" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors would like to thank members of the WuXi Biologics Conjugation Group for the preparation and characterization of the antibody–drug conjugates described in this work. We also thank Douglas Leipold for assistance with the preparation of Figure 19. In addition, the authors acknowledge the Structural Biology group at Viva Biotech (Shanghai, China) for growing the crystals used to generate Figure 7, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">S1, and S2</a> and for collecting the associated data. This research employed resources of the Advanced Photon Source, a U.S. Department of Energy (DOE) Office of Science User Facility operated for the DOE Office of Science by Argonne National Laboratory under Contract No. DE-AC02-06CH11357. Part of the research described in this paper was performed at the Canadian Light Source, a national research facility of the University of Saskatchewan, which is supported by the Canada Foundation for Innovation (CFI), the Natural Sciences and Engineering Research Council (NSERC), the National Research Council (NRC), the Canadian Institutes of Health Research (CIHR), the Government of Saskatchewan, and the University of Saskatchewan. The authors are grateful for the help of the beamline staff at APS 21ID-D and CLS 08ID–1.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">ADC</td><td class="NLM_def"><p class="first last">antibody–drug conjugate</p></td></tr><tr><td class="NLM_term">Alloc</td><td class="NLM_def"><p class="first last">allyloxycarbonyl</p></td></tr><tr><td class="NLM_term">BET</td><td class="NLM_def"><p class="first last">bromodomain and extraterminal</p></td></tr><tr><td class="NLM_term">Boc</td><td class="NLM_def"><p class="first last"><i>tert</i>-butyloxycarbonyl</p></td></tr><tr><td class="NLM_term">BRD4</td><td class="NLM_def"><p class="first last">bromodomain-containing protein 4</p></td></tr><tr><td class="NLM_term">Cbz</td><td class="NLM_def"><p class="first last">benzyloxycarbonyl</p></td></tr><tr><td class="NLM_term">CLL1</td><td class="NLM_def"><p class="first last">C-type lectin-like molecule-1</p></td></tr><tr><td class="NLM_term">CRBN</td><td class="NLM_def"><p class="first last">cereblon</p></td></tr><tr><td class="NLM_term">DAR</td><td class="NLM_def"><p class="first last">drug/antibody ratio</p></td></tr><tr><td class="NLM_term">DCM</td><td class="NLM_def"><p class="first last">dichloromethane</p></td></tr><tr><td class="NLM_term">DIEA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last">dimethylformamide</p></td></tr><tr><td class="NLM_term">DIBAL-H</td><td class="NLM_def"><p class="first last">diisobutylaluminium hydride</p></td></tr><tr><td class="NLM_term">DMP</td><td class="NLM_def"><p class="first last">Dess–Martin periodinane</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">ELISA</td><td class="NLM_def"><p class="first last">enzyme-linked immunosorbent assay</p></td></tr><tr><td class="NLM_term">EtOAc</td><td class="NLM_def"><p class="first last">ethyl acetate</p></td></tr><tr><td class="NLM_term">EtOH</td><td class="NLM_def"><p class="first last">ethyl alcohol</p></td></tr><tr><td class="NLM_term">FA</td><td class="NLM_def"><p class="first last">formic acid</p></td></tr><tr><td class="NLM_term">Fmoc</td><td class="NLM_def"><p class="first last">fluorenylmethoxycarbonyl</p></td></tr><tr><td class="NLM_term">HATU</td><td class="NLM_def"><p class="first last">hexafluorophosphate azabenzotriazole tetramethyl uronium</p></td></tr><tr><td class="NLM_term">HER2</td><td class="NLM_def"><p class="first last">human epidermal growth factor receptor 2</p></td></tr><tr><td class="NLM_term">HFIP</td><td class="NLM_def"><p class="first last">1,1,1,3,3,3-hexafluoropropan-2-ol</p></td></tr><tr><td class="NLM_term">HOAc</td><td class="NLM_def"><p class="first last">acetic acid</p></td></tr><tr><td class="NLM_term">HOBt</td><td class="NLM_def"><p class="first last">hydroxybenzotriazole</p></td></tr><tr><td class="NLM_term">HPLC</td><td class="NLM_def"><p class="first last">high-performance liquid chromatography</p></td></tr><tr><td class="NLM_term">IAM</td><td class="NLM_def"><p class="first last">immobilized artificial membrane</p></td></tr><tr><td class="NLM_term">LCMS</td><td class="NLM_def"><p class="first last">liquid chromatography–mass spectrometry</p></td></tr><tr><td class="NLM_term">LLOQ</td><td class="NLM_def"><p class="first last">lower limit of quantification</p></td></tr><tr><td class="NLM_term">mAb</td><td class="NLM_def"><p class="first last">monoclonal antibody</p></td></tr><tr><td class="NLM_term">MDM2</td><td class="NLM_def"><p class="first last">mouse double minute 2 homolog</p></td></tr><tr><td class="NLM_term">MeCN</td><td class="NLM_def"><p class="first last">acetonitrile</p></td></tr><tr><td class="NLM_term">MeOH</td><td class="NLM_def"><p class="first last">methyl alcohol</p></td></tr><tr><td class="NLM_term">MS</td><td class="NLM_def"><p class="first last">molecular sieves</p></td></tr><tr><td class="NLM_term">MTBE</td><td class="NLM_def"><p class="first last">methyl <i>tert</i>-butyl ether</p></td></tr><tr><td class="NLM_term">MTS</td><td class="NLM_def"><p class="first last">methanethiosulfonate</p></td></tr><tr><td class="NLM_term">PAB</td><td class="NLM_def"><p class="first last"><i>para</i>-amino-benzyloxy</p></td></tr><tr><td class="NLM_term">PEG</td><td class="NLM_def"><p class="first last">poly(ethylene glycol)</p></td></tr><tr><td class="NLM_term">PROTAC</td><td class="NLM_def"><p class="first last">proteolysis-targeting chimera (PROTAC is a registered trademark of Arvinas Operations, Inc.)</p></td></tr><tr><td class="NLM_term">SEC</td><td class="NLM_def"><p class="first last">size-exclusion chromatography</p></td></tr><tr><td class="NLM_term">siRNA</td><td class="NLM_def"><p class="first last">small interfering RNA</p></td></tr><tr><td class="NLM_term">SPR</td><td class="NLM_def"><p class="first last">surface plasmon resonance</p></td></tr><tr><td class="NLM_term">STEAP1</td><td class="NLM_def"><p class="first last">six transmembrane epithelial antigen of the prostate 1</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr><tr><td class="NLM_term">TGI</td><td class="NLM_def"><p class="first last">tumor growth inhibition</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr><tr><td class="NLM_term">TLC</td><td class="NLM_def"><p class="first last">thin-layer chromatography</p></td></tr><tr><td class="NLM_term">TMSI</td><td class="NLM_def"><p class="first last">iodotrimethylsilane</p></td></tr><tr><td class="NLM_term">VHL</td><td class="NLM_def"><p class="first last">Von Hippel–Lindau tumor suppressor</p></td></tr><tr><td class="NLM_term">XIAP</td><td class="NLM_def"><p class="first last">X-linked inhibitor of apoptosis protein</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i77">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_73256" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_73256" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 48 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, L.-l.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.-j.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span> <span> </span><span class="NLM_article-title">Inhibition of BET bromodomains as a therapeutic strategy for cancer drug discovery</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">5501</span>– <span class="NLM_lpage">5516</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.3551</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.18632%2Foncotarget.3551" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=25849938" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A280%3ADC%252BC2MjitVGiuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=5501-5516&author=L.-l.+Fuauthor=M.+Tianauthor=X.+Liauthor=J.-j.+Liauthor=J.+Huangauthor=L.+Ouyangauthor=Y.+Zhangauthor=B.+Liu&title=Inhibition+of+BET+bromodomains+as+a+therapeutic+strategy+for+cancer+drug+discovery&doi=10.18632%2Foncotarget.3551"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1aR"><div class="casContent"><span class="casTitleNuber">1a</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of BET bromodomains as a therapeutic strategy for cancer drug discovery</span></div><div class="casAuthors">Fu Lei-lei; Tian Mao; Li Xiang; Li Jing-jing; Ouyang Liang; Zhang Yonghui; Liu Bo; Huang Jian; Zhang Yonghui</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">5501-16</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">As a conserved protein interaction module that recognizes and binds to acetylated lysine, bromodomain (BRD) contains a deep, largely hydrophobic acetyl lysine binding site.  Proteins that share the feature of containing two BRDs and an extra-terminal domain belong to BET family, including BRD2, BRD3, BRD4 and BRDT.  BET family proteins perform transcription regulatory function under normal conditions, while in cancer, they regulate transcription of several oncogenes, such as c-Myc and Bcl-2.  Thus, targeting BET proteins may be a promising strategy, and intense interest of BET proteins has fueled the development of structure-based bromodomain inhibitors in cancer.  In this review, we focus on summarizing several small-molecule BET inhibitors and their relevant anti-tumor mechanisms, which would provide a clue for exploiting new targeted BET inhibitors in the future cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQBHUaDfXsQehKDYG3bNImrfW6udTcc2eaFSUY5XScnsbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MjitVGiuw%253D%253D&md5=b3b070170e6da5585f87dffe1d568f47</span></div><a href="/servlet/linkout?suffix=cit1a&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.3551&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.3551%26sid%3Dliteratum%253Aachs%26aulast%3DFu%26aufirst%3DL.-l.%26aulast%3DTian%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DJ.-j.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DOuyang%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DB.%26atitle%3DInhibition%2520of%2520BET%2520bromodomains%2520as%2520a%2520therapeutic%2520strategy%2520for%2520cancer%2520drug%2520discovery%26jtitle%3DOncotarget%26date%3D2015%26volume%3D6%26spage%3D5501%26epage%3D5516%26doi%3D10.18632%2Foncotarget.3551" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit1b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Targeting bromodomains: epigenetic readers of lysine acetylation</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">337</span>– <span class="NLM_lpage">356</span>, <span class="refDoi"> DOI: 10.1038/nrd4286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1038%2Fnrd4286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=24751816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=337-356&author=P.+Filippakopoulosauthor=S.+Knapp&title=Targeting+bromodomains%3A+epigenetic+readers+of+lysine+acetylation&doi=10.1038%2Fnrd4286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1bR"><div class="casContent"><span class="casTitleNuber">1b</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bromodomains: epigenetic readers of lysine acetylation</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">337-356</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a key mechanism that regulates chromatin structure; aberrant acetylation levels have been linked to the development of several diseases.  Acetyl-lysine modifications create docking sites for bromodomains, which are small interaction modules found on diverse proteins, some of which have a key role in the acetylation-dependent assembly of transcriptional regulator complexes.  These complexes can then initiate transcriptional programs that result in phenotypic changes.  The recent discovery of potent and highly specific inhibitors for the BET (bromodomain and extra-terminal) family of bromodomains has stimulated intensive research activity in diverse therapeutic areas, particularly in oncol., where BET proteins regulate the expression of key oncogenes and anti-apoptotic proteins.  In addn., targeting BET bromodomains could hold potential for the treatment of inflammation and viral infection.  Here, we highlight recent progress in the development of bromodomain inhibitors, and their potential applications in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOSIcwBJKj4LVg90H21EOLACvtfcHk0liEKVnylbGZjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D&md5=77049c82ce6a19b4685e3b7950399a07</span></div><a href="/servlet/linkout?suffix=cit1b&amp;dbid=16384&amp;doi=10.1038%2Fnrd4286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4286%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DTargeting%2520bromodomains%253A%2520epigenetic%2520readers%2520of%2520lysine%2520acetylation%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D337%26epage%3D356%26doi%3D10.1038%2Fnrd4286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span> <span> </span><span class="NLM_article-title">Targeting Brd4 for cancer therapy: inhibitors and degraders</span>. <i>Med. Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1779</span>– <span class="NLM_lpage">1802</span>, <span class="refDoi"> DOI: 10.1039/C8MD00198G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1039%2FC8MD00198G" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=1779-1802&author=Y.+Duanauthor=Y.+Guanauthor=W.+Qinauthor=X.+Zhaiauthor=B.+Yuauthor=H.+Liu&title=Targeting+Brd4+for+cancer+therapy%3A+inhibitors+and+degraders&doi=10.1039%2FC8MD00198G"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2a&amp;dbid=16384&amp;doi=10.1039%2FC8MD00198G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC8MD00198G%26sid%3Dliteratum%253Aachs%26aulast%3DDuan%26aufirst%3DY.%26aulast%3DGuan%26aufirst%3DY.%26aulast%3DQin%26aufirst%3DW.%26aulast%3DZhai%26aufirst%3DX.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DH.%26atitle%3DTargeting%2520Brd4%2520for%2520cancer%2520therapy%253A%2520inhibitors%2520and%2520degraders%26jtitle%3DMed.%2520Chem.%2520Commun.%26date%3D2018%26volume%3D9%26spage%3D1779%26epage%3D1802%26doi%3D10.1039%2FC8MD00198G" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit2b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wold, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brasier, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span> <span> </span><span class="NLM_article-title">Drug discovery targeting bromodomain-containing protein 4</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4533</span>– <span class="NLM_lpage">4558</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01761</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01761" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXis1yiurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4533-4558&author=Z.+Liuauthor=P.+Wangauthor=H.+Chenauthor=E.+A.+Woldauthor=B.+Tianauthor=A.+R.+Brasierauthor=J.+Zhou&title=Drug+discovery+targeting+bromodomain-containing+protein+4&doi=10.1021%2Facs.jmedchem.6b01761"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2bR"><div class="casContent"><span class="casTitleNuber">2b</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Discovery Targeting Bromodomain-Containing Protein 4</span></div><div class="casAuthors">Liu, Zhiqing; Wang, Pingyuan; Chen, Haiying; Wold, Eric A.; Tian, Bing; Brasier, Allan R.; Zhou, Jia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4533-4558</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  BRD4, the most extensively studied member of BET family, is an epigenetic regulator that localizes to DNA via binding acetylated histones and controls the expression of therapeutically important gene regulatory networks through recruiting transcription factors to form mediator complexes, phosphorylating RNA polymerase II and by its intrinsic histone acetyltransferase activity.  Disrupting the protein-protein interactions between BRD4 and acetyl-lysine has been shown to effectively block cell proliferation in cancer, cytokine prodn. in acute inflammation, etc.  To date, significant efforts have been devoted to the development of BRD4 inhibitors, and consequently, a dozen have progressed into human clin. trials.  Herein, the authors summarize the advances in drug discovery and development of BRD4 inhibitors by focusing on their chemotypes, in vitro and in vivo activity, selectivity, relevant mechanisms of action and therapeutic potential.  Opportunities and challenges to achieve selective and efficacious BRD4 inhibitors as a viable therapeutic strategy for human diseases are also highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6IAERH4iggrVg90H21EOLACvtfcHk0lgh5yD8jICP2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXis1yiurg%253D&md5=6d7ba5844073fac8c916b1249af402e5</span></div><a href="/servlet/linkout?suffix=cit2b&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01761&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01761%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DWold%26aufirst%3DE.%2BA.%26aulast%3DTian%26aufirst%3DB.%26aulast%3DBrasier%26aufirst%3DA.%2BR.%26aulast%3DZhou%26aufirst%3DJ.%26atitle%3DDrug%2520discovery%2520targeting%2520bromodomain-containing%2520protein%25204%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D4533%26epage%3D4558%26doi%3D10.1021%2Facs.jmedchem.6b01761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit2c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côte′, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S.</span></span> <span> </span><span class="NLM_article-title">Disrupting acetyl-lysine recognition: progress in the development of bromodomain inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1271</span>– <span class="NLM_lpage">1298</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01514</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01514" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsr%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1271-1298&author=F.+A.+Romeroauthor=A.+M.+Taylorauthor=T.+D.+Crawfordauthor=V.+Tsuiauthor=A.+C%C3%B4te%E2%80%B2author=S.+Magnuson&title=Disrupting+acetyl-lysine+recognition%3A+progress+in+the+development+of+bromodomain+inhibitors&doi=10.1021%2Facs.jmedchem.5b01514"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2cR"><div class="casContent"><span class="casTitleNuber">2c</span><div class="casTitle"><span class="NLM_cas:atitle">Disrupting Acetyl-Lysine Recognition: Progress in the Development of Bromodomain Inhibitors</span></div><div class="casAuthors">Romero, F. Anthony; Taylor, Alexander M.; Crawford, Terry D.; Tsui, Vickie; Cote, Alexandre; Magnuson, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1271-1298</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains, small protein modules that recognize acetylated lysine on histones, play a significant role in the epigenome, where they function as "readers" that ultimately det. the functional outcome of the post-translational modification.  Because the initial discovery of selective BET inhibitors have helped define the role of that protein family in oncol. and inflammation, BET bromodomains have continued to garner the most attention of any other bromodomain.  More recently, non-BET bromodomain inhibitors that are potent and selective have been disclosed for ATAD2, CBP, BRD7/9, BRPF, BRPF/TRIM24, CECR2, SMARCA4, and BAZ2A/B.  Such novel inhibitors can be used to probe the physiol. function of these non-BET bromodomains and further understanding of their role in certain disease states.  Here, we provide an update to the progress in identifying selective bromodomain inhibitors and their use as biol. tools, as well as our perspective on the field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgbRrKGlHaVLVg90H21EOLACvtfcHk0lgh5yD8jICP2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsr%252FO&md5=260352f61301aa88aad48c0b63e6152f</span></div><a href="/servlet/linkout?suffix=cit2c&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01514%26sid%3Dliteratum%253Aachs%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DC%25C3%25B4te%25E2%2580%25B2%26aufirst%3DA.%26aulast%3DMagnuson%26aufirst%3DS.%26atitle%3DDisrupting%2520acetyl-lysine%2520recognition%253A%2520progress%2520in%2520the%2520development%2520of%2520bromodomain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1271%26epage%3D1298%26doi%3D10.1021%2Facs.jmedchem.5b01514" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amorim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stathis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gleeson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iyengar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magarotto, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leleu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morschhauser, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karlin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Broussais, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herait, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahatt, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokiec, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salles, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Facon, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palumbo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zucca, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thieblemont, C.</span></span> <span> </span><span class="NLM_article-title">Bromodomain inhibitor OTX015 in patients with lymphoma or multiple myeloma: a dose-escalation, open-label, pharmacokinetic, phase 1 study</span>. <i>Lancet Haematol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">e196</span>– <span class="NLM_lpage">e204</span>, <span class="refDoi"> DOI: 10.1016/S2352-3026(16)00021-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1016%2FS2352-3026%2816%2900021-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=27063978" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2016&pages=e196-e204&author=S.+Amorimauthor=A.+Stathisauthor=M.+Gleesonauthor=S.+Iyengarauthor=V.+Magarottoauthor=X.+Leleuauthor=F.+Morschhauserauthor=L.+Karlinauthor=F.+Broussaisauthor=K.+Rezaiauthor=P.+Heraitauthor=C.+Kahattauthor=F.+Lokiecauthor=G.+Sallesauthor=T.+Faconauthor=A.+Palumboauthor=D.+Cunninghamauthor=E.+Zuccaauthor=C.+Thieblemont&title=Bromodomain+inhibitor+OTX015+in+patients+with+lymphoma+or+multiple+myeloma%3A+a+dose-escalation%2C+open-label%2C+pharmacokinetic%2C+phase+1+study&doi=10.1016%2FS2352-3026%2816%2900021-1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2FS2352-3026%2816%2900021-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS2352-3026%252816%252900021-1%26sid%3Dliteratum%253Aachs%26aulast%3DAmorim%26aufirst%3DS.%26aulast%3DStathis%26aufirst%3DA.%26aulast%3DGleeson%26aufirst%3DM.%26aulast%3DIyengar%26aufirst%3DS.%26aulast%3DMagarotto%26aufirst%3DV.%26aulast%3DLeleu%26aufirst%3DX.%26aulast%3DMorschhauser%26aufirst%3DF.%26aulast%3DKarlin%26aufirst%3DL.%26aulast%3DBroussais%26aufirst%3DF.%26aulast%3DRezai%26aufirst%3DK.%26aulast%3DHerait%26aufirst%3DP.%26aulast%3DKahatt%26aufirst%3DC.%26aulast%3DLokiec%26aufirst%3DF.%26aulast%3DSalles%26aufirst%3DG.%26aulast%3DFacon%26aufirst%3DT.%26aulast%3DPalumbo%26aufirst%3DA.%26aulast%3DCunningham%26aufirst%3DD.%26aulast%3DZucca%26aufirst%3DE.%26aulast%3DThieblemont%26aufirst%3DC.%26atitle%3DBromodomain%2520inhibitor%2520OTX015%2520in%2520patients%2520with%2520lymphoma%2520or%2520multiple%2520myeloma%253A%2520a%2520dose-escalation%252C%2520open-label%252C%2520pharmacokinetic%252C%2520phase%25201%2520study%26jtitle%3DLancet%2520Haematol.%26date%3D2016%26volume%3D3%26spage%3De196%26epage%3De204%26doi%3D10.1016%2FS2352-3026%2816%2900021-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Collins, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattersley, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yates, J. W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mondal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mettetal, J. T.</span></span> <span> </span><span class="NLM_article-title">Translational modeling of drug-induced myelosuppression and effect of pretreatment myelosuppression for AZD5153, a selective BRD4 inhibitor</span>. <i>CPT: Pharmacometrics Syst. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">357</span>– <span class="NLM_lpage">364</span>, <span class="refDoi"> DOI: 10.1002/psp4.12194</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1002%2Fpsp4.12194" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=28378926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXos1Srsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2017&pages=357-364&author=T.+A.+Collinsauthor=M.+M.+Hattersleyauthor=J.+W.+T.+Yatesauthor=E.+Clarkauthor=M.+Mondalauthor=J.+T.+Mettetal&title=Translational+modeling+of+drug-induced+myelosuppression+and+effect+of+pretreatment+myelosuppression+for+AZD5153%2C+a+selective+BRD4+inhibitor&doi=10.1002%2Fpsp4.12194"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Translational Modeling of Drug-Induced Myelosuppression and Effect of Pretreatment Myelosuppression for AZD5153, a Selective BRD4 Inhibitor</span></div><div class="casAuthors">Collins, T. A.; Hattersley, M. M.; Yates, J. W. T.; Clark, E.; Mondal, M.; Mettetal, J. T.</div><div class="citationInfo"><span class="NLM_cas:title">CPT: Pharmacometrics & Systems Pharmacology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">357-364</span>CODEN:
                <span class="NLM_cas:coden">CPSPBR</span>;
        ISSN:<span class="NLM_cas:issn">2163-8306</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">In this work, we evaluate the potential risk of thrombocytopenia in man for a BRD4 inhibitor, AZD5153, based on the platelet count decreases from a Han Wistar rat study.  The effects in rat were modeled and used to make clin. predictions for human populations with healthy baseline blood counts.  At doses >10 mg, a dose-dependent effect on circulating platelets is expected, with similar predicted changes for both q.d. and b.i.d. dose schedules.  These results suggest that at predicted efficacious doses, AZD5153 is likely to have some redns. in the clin. platelet counts, but within the normal range at projected efficacious doses.  The model was then extended to incorporate preexisting myelosuppression where bone marrow function is inhibited by acute myeloid leukemia.  Under these conditions, duration of platelet count recovery has the potential to be prolonged due to drug-induced myelosuppression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnyVGxxPLv9LVg90H21EOLACvtfcHk0ljdyeJrrqAH-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXos1Srsrg%253D&md5=2bdc4d7bfd3ee8501d28db2c738631ff</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1002%2Fpsp4.12194&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpsp4.12194%26sid%3Dliteratum%253Aachs%26aulast%3DCollins%26aufirst%3DT.%2BA.%26aulast%3DHattersley%26aufirst%3DM.%2BM.%26aulast%3DYates%26aufirst%3DJ.%2BW.%2BT.%26aulast%3DClark%26aufirst%3DE.%26aulast%3DMondal%26aufirst%3DM.%26aulast%3DMettetal%26aufirst%3DJ.%2BT.%26atitle%3DTranslational%2520modeling%2520of%2520drug-induced%2520myelosuppression%2520and%2520effect%2520of%2520pretreatment%2520myelosuppression%2520for%2520AZD5153%252C%2520a%2520selective%2520BRD4%2520inhibitor%26jtitle%3DCPT%253A%2520Pharmacometrics%2520Syst.%2520Pharmacol.%26date%3D2017%26volume%3D6%26spage%3D357%26epage%3D364%26doi%3D10.1002%2Fpsp4.12194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Seal, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aylott, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copley, R. C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, J. R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayhow, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messenger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preston, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wall, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolven, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">The optimization of a novel, weak bromo and extra terminal domain (BET) bromodomain fragment ligand to a potent and selective second bromodomain (BD2) inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">9093</span>– <span class="NLM_lpage">9126</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00796</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00796" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVCmurzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=9093-9126&author=J.+T.+Sealauthor=S.+J.+Atkinsonauthor=H.+Aylottauthor=P.+Bamboroughauthor=C.-w.+Chungauthor=R.+C.+B.+Copleyauthor=L.+Gordonauthor=P.+Grandiauthor=J.+R.+J.+Grayauthor=L.+A.+Harrisonauthor=T.+G.+Hayhowauthor=M.+Lindonauthor=C.+Messengerauthor=A.-M.+Michonauthor=D.+Mitchellauthor=A.+Prestonauthor=R.+K.+Prinjhaauthor=I.+Riojaauthor=S.+Taylorauthor=I.+D.+Wallauthor=R.+J.+Watsonauthor=J.+M.+Woolvenauthor=E.+H.+Demont&title=The+optimization+of+a+novel%2C+weak+bromo+and+extra+terminal+domain+%28BET%29+bromodomain+fragment+ligand+to+a+potent+and+selective+second+bromodomain+%28BD2%29+inhibitor&doi=10.1021%2Facs.jmedchem.0c00796"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5aR"><div class="casContent"><span class="casTitleNuber">5a</span><div class="casTitle"><span class="NLM_cas:atitle">The Optimization of a Novel, Weak Bromo and Extra Terminal Domain (BET) Bromodomain Fragment Ligand to a Potent and Selective Second Bromodomain (BD2) Inhibitor</span></div><div class="casAuthors">Seal, Jonathan T.; Atkinson, Stephen J.; Aylott, Helen; Bamborough, Paul; Chung, Chun-wa; Copley, Royston C. B.; Gordon, Laurie; Grandi, Paola; Gray, James R. J.; Harrison, Lee A.; Hayhow, Thomas G.; Lindon, Matthew; Messenger, Cassie; Michon, Anne-Marie; Mitchell, Darren; Preston, Alex; Prinjha, Rab K.; Rioja, Inmaculada; Taylor, Simon; Wall, Ian D.; Watson, Robert J.; Woolven, James M.; Demont, Emmanuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9093-9126</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The profound efficacy, yet assocd. toxicity of pan-BET inhibitors is well documented.  The possibility of an ameliorated safety profile driven by significantly selective (>100-fold) inhibition of a subset of the eight bromodomains is enticing, but challenging given the close homol.  Herein, we describe the X-ray crystal structure-directed optimization of a novel weak fragment ligand with a pan-second bromodomain (BD2) bias, to potent and highly BD2 selective inhibitors.  A template hopping approach, enabled by our parallel research into an orthogonal template (15, GSK046, I), was the basis for the high selectivity obsd.  This culminated in two tool mols., 20 (GSK620) and 56 (GSK549), which showed an anti-inflammatory phenotype in human whole blood, confirming their cellular target engagement.  Excellent broad selectivity, developability, and in vivo oral pharmacokinetics characterize these tools, which we hope will be of broad utility to the field of epigenetics research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoS04XaSgoDyLVg90H21EOLACvtfcHk0ljdyeJrrqAH-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVCmurzI&md5=c562a8c9e4c38abac04149ee96e2f594</span></div><a href="/servlet/linkout?suffix=cit5a&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00796%26sid%3Dliteratum%253Aachs%26aulast%3DSeal%26aufirst%3DJ.%2BT.%26aulast%3DAtkinson%26aufirst%3DS.%2BJ.%26aulast%3DAylott%26aufirst%3DH.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DCopley%26aufirst%3DR.%2BC.%2BB.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DJ.%2BR.%2BJ.%26aulast%3DHarrison%26aufirst%3DL.%2BA.%26aulast%3DHayhow%26aufirst%3DT.%2BG.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DMessenger%26aufirst%3DC.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DMitchell%26aufirst%3DD.%26aulast%3DPreston%26aufirst%3DA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DWall%26aufirst%3DI.%2BD.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DWoolven%26aufirst%3DJ.%2BM.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DThe%2520optimization%2520of%2520a%2520novel%252C%2520weak%2520bromo%2520and%2520extra%2520terminal%2520domain%2520%2528BET%2529%2520bromodomain%2520fragment%2520ligand%2520to%2520a%2520potent%2520and%2520selective%2520second%2520bromodomain%2520%2528BD2%2529%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D9093%26epage%3D9126%26doi%3D10.1021%2Facs.jmedchem.0c00796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit5b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Preston, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craggs, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, J. R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rianjongdee, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wall, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolven, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of a highly selective and in vivo-capable inhibitor of the second bromodomain of the bromodomain and extra terminal domain family of proteins</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">9070</span>– <span class="NLM_lpage">9092</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00605</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00605" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVWrsLrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=9070-9092&author=A.+Prestonauthor=S.+Atkinsonauthor=P.+Bamboroughauthor=C.-w.+Chungauthor=P.+D.+Craggsauthor=L.+Gordonauthor=P.+Grandiauthor=J.+R.+J.+Grayauthor=E.+J.+Jonesauthor=M.+Lindonauthor=A.-M.+Michonauthor=D.+J.+Mitchellauthor=R.+K.+Prinjhaauthor=F.+Rianjongdeeauthor=I.+Riojaauthor=J.+Sealauthor=S.+Taylorauthor=I.+Wallauthor=R.+J.+Watsonauthor=J.+Woolvenauthor=E.+H.+Demont&title=Design+and+synthesis+of+a+highly+selective+and+in+vivo-capable+inhibitor+of+the+second+bromodomain+of+the+bromodomain+and+extra+terminal+domain+family+of+proteins&doi=10.1021%2Facs.jmedchem.0c00605"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5bR"><div class="casContent"><span class="casTitleNuber">5b</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of a Highly Selective and In Vivo-Capable Inhibitor of the Second Bromodomain of the Bromodomain and Extra Terminal Domain Family of Proteins</span></div><div class="casAuthors">Preston, Alex; Atkinson, Stephen; Bamborough, Paul; Chung, Chun-wa; Craggs, Peter D.; Gordon, Laurie; Grandi, Paola; Gray, James R. J.; Jones, Emma J.; Lindon, Matthew; Michon, Anne-Marie; Mitchell, Darren J.; Prinjha, Rab K.; Rianjongdee, Francesco; Rioja, Inmaculada; Seal, Jonathan; Taylor, Simon; Wall, Ian; Watson, Robert J.; Woolven, James; Demont, Emmanuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9070-9092</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pan-bromodomain and extra terminal domain (BET) inhibitors interact equipotently with the eight bromodomains of the BET family of proteins and have shown profound efficacy in a no. of in vitro phenotypic assays and in vivo pre-clin. models in inflammation or oncol.  A no. of these inhibitors have progressed to the clinic where pharmacol.-driven adverse events have been reported.  To better understand the contribution of each domain to their efficacy and improve their safety profile, selective inhibitors are required.  This article discloses the profile of GSK046, also known as iBET-BD2(I), a highly selective inhibitor of the second bromodomains of the BET proteins that has undergone extensive pre-clin. in vitro and in vivo characterization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXn2qUNkpo27Vg90H21EOLACvtfcHk0lhiwI4K9mNdWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVWrsLrK&md5=08f4c47868e0a5acd5d443adba06d016</span></div><a href="/servlet/linkout?suffix=cit5b&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00605%26sid%3Dliteratum%253Aachs%26aulast%3DPreston%26aufirst%3DA.%26aulast%3DAtkinson%26aufirst%3DS.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DCraggs%26aufirst%3DP.%2BD.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DJ.%2BR.%2BJ.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRianjongdee%26aufirst%3DF.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DSeal%26aufirst%3DJ.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DWall%26aufirst%3DI.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DWoolven%26aufirst%3DJ.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520a%2520highly%2520selective%2520and%2520in%2520vivo-capable%2520inhibitor%2520of%2520the%2520second%2520bromodomain%2520of%2520the%2520bromodomain%2520and%2520extra%2520terminal%2520domain%2520family%2520of%2520proteins%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D9070%26epage%3D9092%26doi%3D10.1021%2Facs.jmedchem.0c00605" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit5c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnett, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petretich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillipou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">GSK789: A selective inhibitor of the first bromodomains (BD1) of the bromo and extra terminal domain (BET) proteins</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">9045</span>– <span class="NLM_lpage">9069</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00614</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00614" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVWrsLzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=9045-9069&author=R.+J.+Watsonauthor=P.+Bamboroughauthor=H.+Barnettauthor=C.-w.+Chungauthor=R.+Davisauthor=L.+Gordonauthor=P.+Grandiauthor=M.+Petretichauthor=A.+Phillipouauthor=R.+K.+Prinjhaauthor=I.+Riojaauthor=P.+Sodenauthor=T.+Wernerauthor=E.+H.+Demont&title=GSK789%3A+A+selective+inhibitor+of+the+first+bromodomains+%28BD1%29+of+the+bromo+and+extra+terminal+domain+%28BET%29+proteins&doi=10.1021%2Facs.jmedchem.0c00614"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5cR"><div class="casContent"><span class="casTitleNuber">5c</span><div class="casTitle"><span class="NLM_cas:atitle">GSK789: A Selective Inhibitor of the First Bromodomains (BD1) of the Bromo and Extra Terminal Domain (BET) Proteins</span></div><div class="casAuthors">Watson, Robert J.; Bamborough, Paul; Barnett, Heather; Chung, Chun-wa; Davis, Rob; Gordon, Laurie; Grandi, Paola; Petretich, Massimo; Phillipou, Alex; Prinjha, Rab K.; Rioja, Inmaculada; Soden, Peter; Werner, Thilo; Demont, Emmanuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9045-9069</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pan-bromodomain and extra terminal (BET) inhibitors interact equipotently with all eight bromodomains of the BET family of proteins.  They have shown profound efficacy in vitro and in vivo in oncol. and immunomodulatory models, and a no. of them are currently in clin. trials where significant safety signals have been reported.  It is therefore important to understand the functional contribution of each bromodomain to assess the opportunity to tease apart efficacy and toxicity.  This article discloses the in vitro and cellular activity profiles of GSK789(I), a potent, cell-permeable, and highly selective inhibitor of the first bromodomains of the BET family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozid6PMjhvvbVg90H21EOLACvtfcHk0lhiwI4K9mNdWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVWrsLzJ&md5=c86b4d2c85dcfa62854debf225c8076c</span></div><a href="/servlet/linkout?suffix=cit5c&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00614&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00614%26sid%3Dliteratum%253Aachs%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBarnett%26aufirst%3DH.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DDavis%26aufirst%3DR.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DPetretich%26aufirst%3DM.%26aulast%3DPhillipou%26aufirst%3DA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DSoden%26aufirst%3DP.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DGSK789%253A%2520A%2520selective%2520inhibitor%2520of%2520the%2520first%2520bromodomains%2520%2528BD1%2529%2520of%2520the%2520bromo%2520and%2520extra%2520terminal%2520domain%2520%2528BET%2529%2520proteins%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D9045%26epage%3D9069%26doi%3D10.1021%2Facs.jmedchem.0c00614" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit5d"><span><span class="NLM_label">(d) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Wellaway, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craggs, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karamshi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willis, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolven, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wyspian′ska, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerr, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of a bromodomain and extraterminal domain (BET) inhibitor selective for the N-terminal bromodomains that retains an anti-inflammatory and antiproliferative phenotype</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">9020</span>– <span class="NLM_lpage">9044</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00566</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00566" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsV2lt77M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=9020-9044&author=C.+R.+Wellawayauthor=P.+Bamboroughauthor=S.+G.+Bernardauthor=C.-w.+Chungauthor=P.+D.+Craggsauthor=L.+Cutlerauthor=E.+H.+Demontauthor=J.+P.+Evansauthor=L.+Gordonauthor=B.+Karamshiauthor=A.+J.+Lewisauthor=M.+J.+Lindonauthor=D.+J.+Mitchellauthor=I.+Riojaauthor=P.+E.+Sodenauthor=S.+Taylorauthor=R.+J.+Watsonauthor=R.+Willisauthor=J.+M.+Woolvenauthor=B.+S.+Wyspian%E2%80%B2skaauthor=W.+J.+Kerrauthor=R.+K.+Prinjha&title=Structure-based+design+of+a+bromodomain+and+extraterminal+domain+%28BET%29+inhibitor+selective+for+the+N-terminal+bromodomains+that+retains+an+anti-inflammatory+and+antiproliferative+phenotype&doi=10.1021%2Facs.jmedchem.0c00566"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5dR"><div class="casContent"><span class="casTitleNuber">5d</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of a Bromodomain and Extraterminal Domain (BET) Inhibitor Selective for the N-Terminal Bromodomains That Retains an Anti-inflammatory and Antiproliferative Phenotype</span></div><div class="casAuthors">Wellaway, Christopher R.; Bamborough, Paul; Bernard, Sharon G.; Chung, Chun-wa; Craggs, Peter D.; Cutler, Leanne; Demont, Emmanuel H.; Evans, John P.; Gordon, Laurie; Karamshi, Bhumika; Lewis, Antonia J.; Lindon, Matthew J.; Mitchell, Darren J.; Rioja, Inmaculada; Soden, Peter E.; Taylor, Simon; Watson, Robert J.; Willis, Rob; Woolven, James M.; Wyspianska, Beata S.; Kerr, William J.; Prinjha, Rab K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9020-9044</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extraterminal domain (BET) family of epigenetic regulators comprises four proteins (BRD2, BRD3, BRD4, BRDT), each contg. tandem bromodomains.  To date, small mol. inhibitors of these proteins typically bind all eight bromodomains of the family with similar affinity, resulting in a diverse range of biol. effects.  To enable further understanding of the broad phenotype characteristic of pan-BET inhibition, the development of inhibitors selective for individual, or sets of, bromodomains within the family is required.  In this regard, we report the discovery of a potent probe mol. possessing up to 150-fold selectivity for the N-terminal bromodomains (BD1s) over the C-terminal bromodomains (BD2s) of the BETs.  Guided by structural information, a specific amino acid difference between BD1 and BD2 domains was targeted for selective interaction with chem. functionality appended to the previously developed I-BET151 scaffold.  Data presented herein demonstrate that selective inhibition of BD1 domains is sufficient to drive anti-inflammatory and antiproliferative effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkUREcHUhFfbVg90H21EOLACvtfcHk0lhiwI4K9mNdWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsV2lt77M&md5=52b2505bb0a36f526a08bb9bad1f2c26</span></div><a href="/servlet/linkout?suffix=cit5d&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00566&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00566%26sid%3Dliteratum%253Aachs%26aulast%3DWellaway%26aufirst%3DC.%2BR.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBernard%26aufirst%3DS.%2BG.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DCraggs%26aufirst%3DP.%2BD.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DEvans%26aufirst%3DJ.%2BP.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DKaramshi%26aufirst%3DB.%26aulast%3DLewis%26aufirst%3DA.%2BJ.%26aulast%3DLindon%26aufirst%3DM.%2BJ.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DSoden%26aufirst%3DP.%2BE.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DWillis%26aufirst%3DR.%26aulast%3DWoolven%26aufirst%3DJ.%2BM.%26aulast%3DWyspian%25E2%2580%25B2ska%26aufirst%3DB.%2BS.%26aulast%3DKerr%26aufirst%3DW.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26atitle%3DStructure-based%2520design%2520of%2520a%2520bromodomain%2520and%2520extraterminal%2520domain%2520%2528BET%2529%2520inhibitor%2520selective%2520for%2520the%2520N-terminal%2520bromodomains%2520that%2520retains%2520an%2520anti-inflammatory%2520and%2520antiproliferative%2520phenotype%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D9020%26epage%3D9044%26doi%3D10.1021%2Facs.jmedchem.0c00566" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit5e"><span><span class="NLM_label">(e) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Longenecker, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrent, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bui, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faivre, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magoc, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajaraman, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, K. F.</span></span> <span> </span><span class="NLM_article-title">Discovery of N-ethyl-4-[2-(4-fluoro-2,6-dimethyl-phenoxy)-5-(1-hydroxy-1-methyl-ethyl)phenyl]-6-methyl-7-oxo-1H-pyrrolo[2,3-c]pyridine-2-carboxamide (ABBV-744), a BET bromodomain inhibitor with selectivity for the second bromodomain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">5585</span>– <span class="NLM_lpage">5623</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00628</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00628" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5e&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnvVeisrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=5585-5623&author=G.+S.+Sheppardauthor=L.+Wangauthor=S.+D.+Fidanzeauthor=L.+A.+Hasvoldauthor=D.+Liuauthor=J.+K.+Prattauthor=C.+H.+Parkauthor=K.+L.+Longeneckerauthor=W.+Qiuauthor=M.+Torrentauthor=P.+Kovarauthor=M.+Buiauthor=E.+J.+Faivreauthor=X.+Huangauthor=X.+Linauthor=D.+Wilcoxauthor=L.+Zhangauthor=Y.+Shenauthor=D.+H.+Albertauthor=T.+J.+Magocauthor=G.+Rajaramanauthor=W.+M.+Katiauthor=K.+F.+McDaniel&title=Discovery+of+N-ethyl-4-%5B2-%284-fluoro-2%2C6-dimethyl-phenoxy%29-5-%281-hydroxy-1-methyl-ethyl%29phenyl%5D-6-methyl-7-oxo-1H-pyrrolo%5B2%2C3-c%5Dpyridine-2-carboxamide+%28ABBV-744%29%2C+a+BET+bromodomain+inhibitor+with+selectivity+for+the+second+bromodomain&doi=10.1021%2Facs.jmedchem.0c00628"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5eR"><div class="casContent"><span class="casTitleNuber">5e</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of N-Ethyl-4-[2-(4-fluoro-2,6-dimethyl-phenoxy)-5-(1-hydroxy-1-methyl-ethyl)phenyl]-6-methyl-7-oxo-1H-pyrrolo[2,3-c]pyridine-2-carboxamide (ABBV-744), a BET Bromodomain Inhibitor with Selectivity for the Second Bromodomain</span></div><div class="casAuthors">Sheppard, George S.; Wang, Le; Fidanze, Steven D.; Hasvold, Lisa A.; Liu, Dachun; Pratt, John K.; Park, Chang H.; Longenecker, Kenton; Qiu, Wei; Torrent, Maricel; Kovar, Peter J.; Bui, Mai; Faivre, Emily; Huang, Xiaoli; Lin, Xiaoyu; Wilcox, Denise; Zhang, Lu; Shen, Yu; Albert, Daniel H.; Magoc, Terrance J.; Rajaraman, Ganesh; Kati, Warren M.; McDaniel, Keith F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5585-5623</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The BET family of proteins consists of BRD2, BRD3, BRD4, and BRDt.  Each protein contains two distinct bromodomains (BD1 and BD2).  BET family bromodomain inhibitors under clin. development for oncol. bind to each of the eight bromodomains with similar affinities.  We hypothesized that it may be possible to achieve an improved therapeutic index by selectively targeting subsets of the BET bromodomains.  Both BD1 and BD2 are highly conserved across family members (>70% identity), whereas BD1 and BD2 from the same protein exhibit a larger degree of divergence (~ 40% identity), suggesting selectivity between BD1 and BD2 of all family members would be more straightforward to achieve.  Exploiting the Asp144/His437 and Ile146/Val439 sequence differences (BRD4 BD1/BD2 numbering) allowed the identification of compd. 27 demonstrating greater than 100-fold selectivity for BRD4 BD2 over BRD4 BD1.  Further optimization to improve BD2 selectivity and oral bioavailability resulted in the clin. development compd. 46 (ABBV-744).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJtZIcWro0xLVg90H21EOLACvtfcHk0lgPef_HCdgfDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnvVeisrk%253D&md5=675827659c3c7b6099ccd667f4c60f95</span></div><a href="/servlet/linkout?suffix=cit5e&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00628&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00628%26sid%3Dliteratum%253Aachs%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DFidanze%26aufirst%3DS.%2BD.%26aulast%3DHasvold%26aufirst%3DL.%2BA.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DLongenecker%26aufirst%3DK.%2BL.%26aulast%3DQiu%26aufirst%3DW.%26aulast%3DTorrent%26aufirst%3DM.%26aulast%3DKovar%26aufirst%3DP.%26aulast%3DBui%26aufirst%3DM.%26aulast%3DFaivre%26aufirst%3DE.%2BJ.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DWilcox%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DMagoc%26aufirst%3DT.%2BJ.%26aulast%3DRajaraman%26aufirst%3DG.%26aulast%3DKati%26aufirst%3DW.%2BM.%26aulast%3DMcDaniel%26aufirst%3DK.%2BF.%26atitle%3DDiscovery%2520of%2520N-ethyl-4-%255B2-%25284-fluoro-2%252C6-dimethyl-phenoxy%2529-5-%25281-hydroxy-1-methyl-ethyl%2529phenyl%255D-6-methyl-7-oxo-1H-pyrrolo%255B2%252C3-c%255Dpyridine-2-carboxamide%2520%2528ABBV-744%2529%252C%2520a%2520BET%2520bromodomain%2520inhibitor%2520with%2520selectivity%2520for%2520the%2520second%2520bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D5585%26epage%3D5623%26doi%3D10.1021%2Facs.jmedchem.0c00628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit5f"><span><span class="NLM_label">(f) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Faivre, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantena, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotnik, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bui, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sehgal, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uziel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hessler, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellin, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panchal, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicolette, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fossey, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Longenecker, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bigelow, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrent, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of the BD2 bromodomain of BET proteins in prostate cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>578</i></span>,  <span class="NLM_fpage">306</span>– <span class="NLM_lpage">310</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-1930-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5f&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1038%2Fs41586-020-1930-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5f&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=31969702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5f&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BB3cXis1Smsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=578&publication_year=2020&pages=306-310&author=E.+J.+Faivreauthor=K.+F.+McDanielauthor=D.+H.+Albertauthor=S.+R.+Mantenaauthor=J.+P.+Plotnikauthor=D.+Wilcoxauthor=L.+Zhangauthor=M.+H.+Buiauthor=G.+S.+Sheppardauthor=L.+Wangauthor=V.+Sehgalauthor=X.+Linauthor=X.+Huangauthor=X.+Luauthor=T.+Uzielauthor=P.+Hesslerauthor=L.+T.+Lamauthor=R.+J.+Bellinauthor=G.+Mehtaauthor=S.+Fidanzeauthor=J.+K.+Prattauthor=D.+Liuauthor=L.+A.+Hasvoldauthor=C.+Sunauthor=S.+C.+Panchalauthor=J.+J.+Nicoletteauthor=S.+L.+Fosseyauthor=C.+H.+Parkauthor=K.+Longeneckerauthor=L.+Bigelowauthor=M.+Torrentauthor=S.+H.+Rosenbergauthor=W.+M.+Katiauthor=Y.+Shen&title=Selective+inhibition+of+the+BD2+bromodomain+of+BET+proteins+in+prostate+cancer&doi=10.1038%2Fs41586-020-1930-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5fR"><div class="casContent"><span class="casTitleNuber">5f</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of the BD2 bromodomain of BET proteins in prostate cancer</span></div><div class="casAuthors">Faivre, Emily J.; McDaniel, Keith F.; Albert, Daniel H.; Mantena, Srinivasa R.; Plotnik, Joshua P.; Wilcox, Denise; Zhang, Lu; Bui, Mai H.; Sheppard, George S.; Wang, Le; Sehgal, Vasudha; Lin, Xiaoyu; Huang, Xiaoli; Lu, Xin; Uziel, Tamar; Hessler, Paul; Lam, Lloyd T.; Bellin, Richard J.; Mehta, Gaurav; Fidanze, Steve; Pratt, John K.; Liu, Dachun; Hasvold, Lisa A.; Sun, Chaohong; Panchal, Sanjay C.; Nicolette, John J.; Fossey, Stacey L.; Park, Chang H.; Longenecker, Kenton; Bigelow, Lance; Torrent, Maricel; Rosenberg, Saul H.; Kati, Warren M.; Shen, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">578</span>
        (<span class="NLM_cas:issue">7794</span>),
    <span class="NLM_cas:pages">306-310</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Proteins of the bromodomain and extra-terminal (BET) domain family are epigenetic readers that bind acetylated histones through their bromodomains to regulate gene transcription.  Dual-bromodomain BET inhibitors (DbBi) that bind with similar affinities to the first (BD1) and second (BD2) bromodomains of BRD2, BRD3, BRD4 and BRDt have displayed modest clin. activity in monotherapy cancer trials.  A reduced no. of thrombocytes in the blood (thrombocytopenia) as well as symptoms of gastrointestinal toxicity are dose-limiting adverse events for some types of DbBi1-5.  Given that similar haematol. and gastrointestinal defects were obsd. after genetic silencing of Brd4 in mice6, the platelet and gastrointestinal toxicities may represent on-target activities assocd. with BET inhibition.  The two individual bromodomains in BET family proteins may have distinct functions7-9 and different cellular phenotypes after pharmacol. inhibition of one or both bromodomains have been reported10,11, suggesting that selectively targeting one of the bromodomains may result in a different efficacy and tolerability profile compared with DbBi.  Available compds. that are selective to individual domains lack sufficient potency and the pharmacokinetics properties that are required for in vivo efficacy and tolerability assessment10-13.  Here we carried out a medicinal chem. campaign that led to the discovery of ABBV-744, a highly potent and selective inhibitor of the BD2 domain of BET family proteins with drug-like properties.  In contrast to the broad range of cell growth inhibition induced by DbBi, the antiproliferative activity of ABBV-744 was largely, but not exclusively, restricted to cell lines of acute myeloid leukemia and prostate cancer that expressed the full-length androgen receptor (AR).  ABBV-744 retained robust activity in prostate cancer xenografts, and showed fewer platelet and gastrointestinal toxicities than the DbBi ABBV-07514.  Analyses of RNA expression and chromatin immunopptn. followed by sequencing revealed that ABBV-744 displaced BRD4 from AR-contg. super-enhancers and inhibited AR-dependent transcription, with less impact on global transcription compared with ABBV-075.  These results underscore the potential value of selectively targeting the BD2 domain of BET family proteins for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrF98kER1y9pLVg90H21EOLACvtfcHk0lgPef_HCdgfDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXis1Smsr0%253D&md5=800d9106c33084390fd8cbf1ac7f1766</span></div><a href="/servlet/linkout?suffix=cit5f&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-1930-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-1930-8%26sid%3Dliteratum%253Aachs%26aulast%3DFaivre%26aufirst%3DE.%2BJ.%26aulast%3DMcDaniel%26aufirst%3DK.%2BF.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DMantena%26aufirst%3DS.%2BR.%26aulast%3DPlotnik%26aufirst%3DJ.%2BP.%26aulast%3DWilcox%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DBui%26aufirst%3DM.%2BH.%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DSehgal%26aufirst%3DV.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DUziel%26aufirst%3DT.%26aulast%3DHessler%26aufirst%3DP.%26aulast%3DLam%26aufirst%3DL.%2BT.%26aulast%3DBellin%26aufirst%3DR.%2BJ.%26aulast%3DMehta%26aufirst%3DG.%26aulast%3DFidanze%26aufirst%3DS.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DHasvold%26aufirst%3DL.%2BA.%26aulast%3DSun%26aufirst%3DC.%26aulast%3DPanchal%26aufirst%3DS.%2BC.%26aulast%3DNicolette%26aufirst%3DJ.%2BJ.%26aulast%3DFossey%26aufirst%3DS.%2BL.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DLongenecker%26aufirst%3DK.%26aulast%3DBigelow%26aufirst%3DL.%26aulast%3DTorrent%26aufirst%3DM.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DKati%26aufirst%3DW.%2BM.%26aulast%3DShen%26aufirst%3DY.%26atitle%3DSelective%2520inhibition%2520of%2520the%2520BD2%2520bromodomain%2520of%2520BET%2520proteins%2520in%2520prostate%2520cancer%26jtitle%3DNature%26date%3D2020%26volume%3D578%26spage%3D306%26epage%3D310%26doi%3D10.1038%2Fs41586-020-1930-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit5g"><span><span class="NLM_label">(g) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Gilan, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knezevic, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeung, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harker, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, E. Y. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petretich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassil, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vassiliadis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burr, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preston, A. G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wellaway, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gobbetti, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haynes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vappiani, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, S.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, M. A.</span></span> <span> </span><span class="NLM_article-title">Selective targeting of BD1 and BD2 of the BET proteins in cancer and immuno-inflammation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>368</i></span>,  <span class="NLM_fpage">387</span>– <span class="NLM_lpage">394</span>, <span class="refDoi"> DOI: 10.1126/science.aaz8455</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5g&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1126%2Fscience.aaz8455" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5g&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=32193360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5g&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnslKrtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2020&pages=387-394&author=O.+Gilanauthor=I.+Riojaauthor=K.+Knezevicauthor=M.+J.+Bellauthor=M.+M.+Yeungauthor=N.+R.+Harkerauthor=E.+Y.+N.+Lamauthor=C.-w.+Chungauthor=P.+Bamboroughauthor=M.+Petretichauthor=M.+Urhauthor=S.+J.+Atkinsonauthor=A.+K.+Bassilauthor=E.+J.+Robertsauthor=D.+Vassiliadisauthor=M.+L.+Burrauthor=A.+G.+S.+Prestonauthor=C.+Wellawayauthor=T.+Wernerauthor=J.+R.+Grayauthor=A.-M.+Michonauthor=T.+Gobbettiauthor=V.+Kumarauthor=P.+E.+Sodenauthor=A.+Haynesauthor=J.+Vappianiauthor=D.+F.+Toughauthor=S.+Taylorauthor=S.-J.+Dawsonauthor=M.+Bantscheffauthor=M.+Lindonauthor=G.+Drewesauthor=E.+H.+Demontauthor=D.+L.+Danielsauthor=P.+Grandiauthor=R.+K.+Prinjhaauthor=M.+A.+Dawson&title=Selective+targeting+of+BD1+and+BD2+of+the+BET+proteins+in+cancer+and+immuno-inflammation&doi=10.1126%2Fscience.aaz8455"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5gR"><div class="casContent"><span class="casTitleNuber">5g</span><div class="casTitle"><span class="NLM_cas:atitle">Selective targeting of BD1 and BD2 of the BET proteins in cancer and immunoinflammation</span></div><div class="casAuthors">Gilan, Omer; Rioja, Inmaculada; Knezevic, Kathy; Bell, Matthew J.; Yeung, Miriam M.; Harker, Nicola R.; Lam, Enid Y. N.; Chung, Chun-wa; Bamborough, Paul; Petretich, Massimo; Urh, Marjeta; Atkinson, Stephen J.; Bassil, Anna K.; Roberts, Emma J.; Vassiliadis, Dane; Burr, Marian L.; Preston, Alex G. S.; Wellaway, Christopher; Werner, Thilo; Gray, James R.; Michon, Anne-Marie; Gobbetti, Thomas; Kumar, Vinod; Soden, Peter E.; Haynes, Andrea; Vappiani, Johanna; Tough, David F.; Taylor, Simon; Dawson, Sarah-Jane; Bantscheff, Marcus; Lindon, Matthew; Drewes, Gerard; Demont, Emmanuel H.; Daniels, Danette L.; Grandi, Paola; Prinjha, Rab K.; Dawson, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">6489</span>),
    <span class="NLM_cas:pages">387-394</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">1095-9203</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The two tandem bromodomains of the BET (bromodomain and extraterminal domain) proteins enable chromatin binding to facilitate transcription.  Drugs that inhibit both bromodomains equally have shown efficacy in certain malignant and inflammatory conditions.  To explore the individual functional contributions of the first (BD1) and second (BD2) bromodomains in biol. and therapy, we developed selective BD1 and BD2 inhibitors.  We found that steady-state gene expression primarily requires BD1, whereas the rapid increase of gene expression induced by inflammatory stimuli requires both BD1 and BD2 of all BET proteins.  BD1 inhibitors phenocopied the effects of pan-BET inhibitors in cancer models, whereas BD2 inhibitors were predominantly effective in models of inflammatory and autoimmune disease.  These insights into the differential requirement of BD1 and BD2 for the maintenance and induction of gene expression may guide future BET-targeted therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr078AuJubsqbVg90H21EOLACvtfcHk0lgD3OnSoDsjpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnslKrtrg%253D&md5=d78cfea091fcf5ee18735b02b2120dd7</span></div><a href="/servlet/linkout?suffix=cit5g&amp;dbid=16384&amp;doi=10.1126%2Fscience.aaz8455&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aaz8455%26sid%3Dliteratum%253Aachs%26aulast%3DGilan%26aufirst%3DO.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DKnezevic%26aufirst%3DK.%26aulast%3DBell%26aufirst%3DM.%2BJ.%26aulast%3DYeung%26aufirst%3DM.%2BM.%26aulast%3DHarker%26aufirst%3DN.%2BR.%26aulast%3DLam%26aufirst%3DE.%2BY.%2BN.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DPetretich%26aufirst%3DM.%26aulast%3DUrh%26aufirst%3DM.%26aulast%3DAtkinson%26aufirst%3DS.%2BJ.%26aulast%3DBassil%26aufirst%3DA.%2BK.%26aulast%3DRoberts%26aufirst%3DE.%2BJ.%26aulast%3DVassiliadis%26aufirst%3DD.%26aulast%3DBurr%26aufirst%3DM.%2BL.%26aulast%3DPreston%26aufirst%3DA.%2BG.%2BS.%26aulast%3DWellaway%26aufirst%3DC.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DGray%26aufirst%3DJ.%2BR.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DGobbetti%26aufirst%3DT.%26aulast%3DKumar%26aufirst%3DV.%26aulast%3DSoden%26aufirst%3DP.%2BE.%26aulast%3DHaynes%26aufirst%3DA.%26aulast%3DVappiani%26aufirst%3DJ.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DDawson%26aufirst%3DS.-J.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DDawson%26aufirst%3DM.%2BA.%26atitle%3DSelective%2520targeting%2520of%2520BD1%2520and%2520BD2%2520of%2520the%2520BET%2520proteins%2520in%2520cancer%2520and%2520immuno-inflammation%26jtitle%3DScience%26date%3D2020%26volume%3D368%26spage%3D387%26epage%3D394%26doi%3D10.1126%2Fscience.aaz8455" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Nalawansha, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">PROTACs: An emerging therapeutic modality in precision medicine</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">998</span>– <span class="NLM_lpage">1014</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2020.07.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1016%2Fj.chembiol.2020.07.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=32795419" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhs1Wrs7vM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2020&pages=998-1014&author=D.+A.+Nalawanshaauthor=C.+M.+Crews&title=PROTACs%3A+An+emerging+therapeutic+modality+in+precision+medicine&doi=10.1016%2Fj.chembiol.2020.07.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6aR"><div class="casContent"><span class="casTitleNuber">6a</span><div class="casTitle"><span class="NLM_cas:atitle">PROTACs: An Emerging Therapeutic Modality in Precision Medicine</span></div><div class="casAuthors">Nalawansha, Dhanusha A.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">998-1014</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.,.  Targeted protein degrdn. (TPD) has emerged as an exciting new era in chem. biol. and drug discovery.  PROteolysis TArgeting Chimera (PROTAC) technol. targets cellular proteins for degrdn. by co-opting the ubiquitin-proteasome system.  Over the last 5 years, numerous studies have expanded our understanding of the unique mode of action and advantages of PROTACs, which has in turn spurred interest in both academia and industry to explore PROTACs as a novel therapeutic strategy.  In this review, we first highlight the key advantages of PROTACs and then discuss the spatiotemporal regulation of protein degrdn.  Next, we explore current chem. tractable E3 ligases focusing on expanding the existing repertoire with novel E3 ligases to uncover the full potential of TPD.  Collectively, these studies are guiding the development of the PROTAC technol. as it emerges as a new modality in precision medicine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4GkJzvvYnp7Vg90H21EOLACvtfcHk0lgD3OnSoDsjpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhs1Wrs7vM&md5=63c4eb78ee4fc9272b9ae2409889861e</span></div><a href="/servlet/linkout?suffix=cit6a&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2020.07.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2020.07.020%26sid%3Dliteratum%253Aachs%26aulast%3DNalawansha%26aufirst%3DD.%2BA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DPROTACs%253A%2520An%2520emerging%2520therapeutic%2520modality%2520in%2520precision%2520medicine%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2020%26volume%3D27%26spage%3D998%26epage%3D1014%26doi%3D10.1016%2Fj.chembiol.2020.07.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit6b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calabrese, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bullock, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeted protein degradation: expanding the toolbox</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">949</span>– <span class="NLM_lpage">963</span>, <span class="refDoi"> DOI: 10.1038/s41573-019-0047-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1038%2Fs41573-019-0047-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=31666732" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitVKrtLrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=949-963&author=M.+Schapiraauthor=M.+F.+Calabreseauthor=A.+N.+Bullockauthor=C.+M.+Crews&title=Targeted+protein+degradation%3A+expanding+the+toolbox&doi=10.1038%2Fs41573-019-0047-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6bR"><div class="casContent"><span class="casTitleNuber">6b</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted protein degradation: expanding the toolbox</span></div><div class="casAuthors">Schapira, Matthieu; Calabrese, Matthew F.; Bullock, Alex N.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">949-963</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review. Proteolysis-targeting chimeras (PROTACs) and related mols. that induce targeted protein degrdn. by the ubiquitin-proteasome system represent a new therapeutic modality and are the focus of great interest, owing to potential advantages over traditional occupancy-based inhibitors with respect to dosing, side effects, drug resistance and modulating 'undruggable' targets.  However, the technol. is still maturing, and the design elements for successful PROTAC-based drugs are currently being elucidated.  Importantly, fewer than 10 of the more than 600 E3 ubiquitin ligases have so far been exploited for targeted protein degrdn., and expansion of knowledge in this area is a key opportunity.  Here, we briefly discuss lessons learned about targeted protein degrdn. in chem. biol. and drug discovery and systematically review the expression profile, domain architecture and chem. tractability of human E3 ligases that could expand the toolbox for PROTAC discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwUvM27MU3WrVg90H21EOLACvtfcHk0lg76jnJU2m-IA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitVKrtLrM&md5=9bfd48c6c47fc1cce15129495fea7079</span></div><a href="/servlet/linkout?suffix=cit6b&amp;dbid=16384&amp;doi=10.1038%2Fs41573-019-0047-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-019-0047-y%26sid%3Dliteratum%253Aachs%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DCalabrese%26aufirst%3DM.%2BF.%26aulast%3DBullock%26aufirst%3DA.%2BN.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeted%2520protein%2520degradation%253A%2520expanding%2520the%2520toolbox%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D949%26epage%3D963%26doi%3D10.1038%2Fs41573-019-0047-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit6c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Small-molecule modulation of protein homeostasis</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">11269</span>– <span class="NLM_lpage">11301</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.7b00077</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.7b00077" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1Ohur3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2017&pages=11269-11301&author=G.+M.+Burslemauthor=C.+M.+Crews&title=Small-molecule+modulation+of+protein+homeostasis&doi=10.1021%2Facs.chemrev.7b00077"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6cR"><div class="casContent"><span class="casTitleNuber">6c</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Modulation of Protein Homeostasis</span></div><div class="casAuthors">Burslem, George M.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">11269-11301</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Control of protein levels by nucleic-acid-based technologies has proven to be a useful research tool but lacks the advantages of small mols. with respect to cell permeability, temporal control, and the potential generation of therapeutics.  In this Review, we discuss the technologies available for the control of intracellular protein levels with small mols. and compare the various systems available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcARydbgPK9bVg90H21EOLACvtfcHk0lg76jnJU2m-IA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1Ohur3K&md5=f9f8dd98e2c71c7166d2d500b514d578</span></div><a href="/servlet/linkout?suffix=cit6c&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.7b00077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.7b00077%26sid%3Dliteratum%253Aachs%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DSmall-molecule%2520modulation%2520of%2520protein%2520homeostasis%26jtitle%3DChem.%2520Rev.%26date%3D2017%26volume%3D117%26spage%3D11269%26epage%3D11301%26doi%3D10.1021%2Facs.chemrev.7b00077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhimschi, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamman, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishchenko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Lessons in PROTAC design from selective degradation with a promiscuous warhead</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">78</span>– <span class="NLM_lpage">87</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.09.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1016%2Fj.chembiol.2017.09.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=29129718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=78-87&author=D.+P.+Bondesonauthor=B.+E.+Smithauthor=G.+M.+Burslemauthor=A.+D.+Buhimschiauthor=J.+Hinesauthor=S.+Jaime-Figueroaauthor=J.+Wangauthor=B.+D.+Hammanauthor=A.+Ishchenkoauthor=C.+M.+Crews&title=Lessons+in+PROTAC+design+from+selective+degradation+with+a+promiscuous+warhead&doi=10.1016%2Fj.chembiol.2017.09.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7aR"><div class="casContent"><span class="casTitleNuber">7a</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons in PROTAC Design from Selective Degradation with a Promiscuous Warhead</span></div><div class="casAuthors">Bondeson, Daniel P.; Smith, Blake E.; Burslem, George M.; Buhimschi, Alexandru D.; Hines, John; Jaime-Figueroa, Saul; Wang, Jing; Hamman, Brian D.; Ishchenko, Alexey; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">78-87.e5</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Inhibiting protein function selectively is a major goal of modern drug discovery.  Here, we report a previously understudied benefit of small mol. proteolysis-targeting chimeras (PROTACs) that recruit E3 ubiquitin ligases to target proteins for their ubiquitination and subsequent proteasome-mediated degrdn.  Using promiscuous CRBN- and VHL-recruiting PROTACs that bind >50 kinases, we show that only a subset of bound targets is degraded.  The basis of this selectivity relies on protein-protein interactions between the E3 ubiquitin ligase and the target protein, as illustrated by engaged proteins that are not degraded as a result of unstable ternary complexes with PROTAC-recruited E3 ligases.  In contrast, weak PROTAC:target protein affinity can be stabilized by high-affinity target:PROTAC:ligase trimer interactions, leading to efficient degrdn.  This study highlights design guidelines for generating potent PROTACs as well as possibilities for degrading undruggable proteins immune to traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoISOBudOLyFrVg90H21EOLACvtfcHk0lg76jnJU2m-IA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN&md5=30fac502705a2344c4261cf531e6bf9d</span></div><a href="/servlet/linkout?suffix=cit7a&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.09.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.09.010%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DSmith%26aufirst%3DB.%2BE.%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DBuhimschi%26aufirst%3DA.%2BD.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DIshchenko%26aufirst%3DA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DLessons%2520in%2520PROTAC%2520design%2520from%2520selective%2520degradation%2520with%2520a%2520promiscuous%2520warhead%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D78%26epage%3D87%26doi%3D10.1016%2Fj.chembiol.2017.09.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit7b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Remillard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brien, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sonnett, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dastjerdi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wghr, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Degradation of the BAF complex factor BRD9 by heterobifunctional ligands</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">5738</span>– <span class="NLM_lpage">5743</span>, <span class="refDoi"> DOI: 10.1002/anie.201611281</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1002%2Fanie.201611281" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtFGrurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=5738-5743&author=D.+Remillardauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=G.+L.+Brienauthor=M.+Sonnettauthor=H.-S.+Seoauthor=S.+Dastjerdiauthor=M.+Wghrauthor=S.+Dhe-Paganonauthor=S.+A.+Armstrongauthor=J.+E.+Bradner&title=Degradation+of+the+BAF+complex+factor+BRD9+by+heterobifunctional+ligands&doi=10.1002%2Fanie.201611281"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7bR"><div class="casContent"><span class="casTitleNuber">7b</span><div class="casTitle"><span class="NLM_cas:atitle">Degradation of the BAF Complex Factor BRD9 by Heterobifunctional Ligands</span></div><div class="casAuthors">Remillard, David; Buckley, Dennis L.; Paulk, Joshiawa; Brien, Gerard L.; Sonnett, Matthew; Seo, Hyuk-Soo; Dastierdi, Shiva; Wuehr, Martin; Dhe-Paganon, Sirano; Armstrong, Scott A.; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">5738-5743</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The bromodomain-contg. protein BRD9, a subunit of the human BAF (SWI/SNF) nucleosome remodeling complex, has emerged as an attractive therapeutic target in cancer.  Despite the development of chem. probes targeting the BRD9 bromodomain, there is a limited understanding of BRD9 function beyond acetyl-lysine recognition.  We have therefore created the first BRD9-directed chem. degraders, through iterative design and testing of heterobifunctional ligands that bridge the BRD9 bromodomain and the cereblon E3 ubiquitin ligase complex.  Degraders of BRD9 exhibit markedly enhanced potency compared to parental ligands (10- to 100-fold).  Parallel study of degraders with divergent BRD9-binding chemotypes in models of acute myeloid leukemia resolves bromodomain polypharmacol. in this emerging drug class.  Together, these findings reveal the tractability of non-BET bromodomain contg. proteins to chem. degrdn., and highlight lead compd. dBRD9 as a tool for the study of BRD9.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQ9FU2Ipaq27Vg90H21EOLACvtfcHk0lg76jnJU2m-IA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtFGrurk%253D&md5=2eb6562cc9d976c7ae6070c9a419e9ea</span></div><a href="/servlet/linkout?suffix=cit7b&amp;dbid=16384&amp;doi=10.1002%2Fanie.201611281&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201611281%26sid%3Dliteratum%253Aachs%26aulast%3DRemillard%26aufirst%3DD.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DBrien%26aufirst%3DG.%2BL.%26aulast%3DSonnett%26aufirst%3DM.%26aulast%3DSeo%26aufirst%3DH.-S.%26aulast%3DDastjerdi%26aufirst%3DS.%26aulast%3DWghr%26aufirst%3DM.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DDegradation%2520of%2520the%2520BAF%2520complex%2520factor%2520BRD9%2520by%2520heterobifunctional%2520ligands%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26spage%3D5738%26epage%3D5743%26doi%3D10.1002%2Fanie.201611281" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morita, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ujikawa, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohoka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koyama, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sano, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imaeda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naito, M.</span></span> <span> </span><span class="NLM_article-title">Development of protein degradation inducers of androgen receptor by conjugation of androgen receptor ligands and inhibitor of apoptosis protein ligands</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">543</span>– <span class="NLM_lpage">575</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00168</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00168" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpslSlu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=543-575&author=N.+Shibataauthor=K.+Nagaiauthor=Y.+Moritaauthor=O.+Ujikawaauthor=N.+Ohokaauthor=T.+Hattoriauthor=R.+Koyamaauthor=O.+Sanoauthor=Y.+Imaedaauthor=H.+Naraauthor=N.+Choauthor=M.+Naito&title=Development+of+protein+degradation+inducers+of+androgen+receptor+by+conjugation+of+androgen+receptor+ligands+and+inhibitor+of+apoptosis+protein+ligands&doi=10.1021%2Facs.jmedchem.7b00168"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8aR"><div class="casContent"><span class="casTitleNuber">8a</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Protein Degradation Inducers of Androgen Receptor by Conjugation of Androgen Receptor Ligands and Inhibitor of Apoptosis Protein Ligands</span></div><div class="casAuthors">Shibata, Norihito; Nagai, Katsunori; Morita, Yoko; Ujikawa, Osamu; Ohoka, Nobumichi; Hattori, Takayuki; Koyama, Ryokichi; Sano, Osamu; Imaeda, Yasuhiro; Nara, Hiroshi; Cho, Nobuo; Naito, Mikihiko</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">543-575</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Targeted protein degrdn. using small mols. is a novel strategy for drug development.  The authors have developed hybrid mols. named specific and nongenetic inhibitor of apoptosis protein [IAP]-dependent protein erasers (SNIPERs) that recruit IAP ubiquitin ligases to degrade target proteins.  Here, the authors show novel SNIPERs capable of inducing proteasomal degrdn. of the androgen receptor (AR).  Through derivatization of the SNIPER(AR) mol. at the AR ligand and IAP ligand and linker, the authors developed SNIPER(AR)-51 (I), which shows effective protein knockdown activity against AR.  Consistent with the degrdn. of the AR protein, I inhibits AR-mediated gene expression and proliferation of androgen-dependent prostate cancer cells.  In addn., I efficiently induces caspase activation and apoptosis in prostate cancer cells, which was not obsd. in the cells treated with AR antagonists.  These results suggest that SNIPER(AR)s could be leads for an anticancer drug against prostate cancers that exhibit AR-dependent proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqW6OG4BZYg1rVg90H21EOLACvtfcHk0liuJWYO1pa7IA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpslSlu7k%253D&md5=eeefbe187f2f93ca32518f6a3258b903</span></div><a href="/servlet/linkout?suffix=cit8a&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00168&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00168%26sid%3Dliteratum%253Aachs%26aulast%3DShibata%26aufirst%3DN.%26aulast%3DNagai%26aufirst%3DK.%26aulast%3DMorita%26aufirst%3DY.%26aulast%3DUjikawa%26aufirst%3DO.%26aulast%3DOhoka%26aufirst%3DN.%26aulast%3DHattori%26aufirst%3DT.%26aulast%3DKoyama%26aufirst%3DR.%26aulast%3DSano%26aufirst%3DO.%26aulast%3DImaeda%26aufirst%3DY.%26aulast%3DNara%26aufirst%3DH.%26aulast%3DCho%26aufirst%3DN.%26aulast%3DNaito%26aufirst%3DM.%26atitle%3DDevelopment%2520of%2520protein%2520degradation%2520inducers%2520of%2520androgen%2520receptor%2520by%2520conjugation%2520of%2520androgen%2520receptor%2520ligands%2520and%2520inhibitor%2520of%2520apoptosis%2520protein%2520ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D543%26epage%3D575%26doi%3D10.1021%2Facs.jmedchem.7b00168" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit8b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Ohoka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okuhira, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ujikawa, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimokawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sano, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koyama, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujita, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teratani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imaeda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naito, M.</span></span> <span> </span><span class="NLM_article-title"><i>In vivo</i> knockdown of pathogenic proteins via specific and nongenetic inhibitor of apoptosis protein (IAP)-dependent protein erasers (SNIPERs)</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>292</i></span>,  <span class="NLM_fpage">4556</span>– <span class="NLM_lpage">4570</span>, <span class="refDoi"> DOI: 10.1074/jbc.M116.768853</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1074%2Fjbc.M116.768853" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=28154167" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXksVClur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=292&publication_year=2017&pages=4556-4570&author=N.+Ohokaauthor=K.+Okuhiraauthor=M.+Itoauthor=K.+Nagaiauthor=N.+Shibataauthor=T.+Hattoriauthor=O.+Ujikawaauthor=K.+Shimokawaauthor=O.+Sanoauthor=R.+Koyamaauthor=H.+Fujitaauthor=M.+Terataniauthor=H.+Matsumotoauthor=Y.+Imaedaauthor=H.+Naraauthor=N.+Choauthor=M.+Naito&title=In+vivo+knockdown+of+pathogenic+proteins+via+specific+and+nongenetic+inhibitor+of+apoptosis+protein+%28IAP%29-dependent+protein+erasers+%28SNIPERs%29&doi=10.1074%2Fjbc.M116.768853"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8bR"><div class="casContent"><span class="casTitleNuber">8b</span><div class="casTitle"><span class="NLM_cas:atitle">In Vivo Knockdown of Pathogenic Proteins via Specific and Nongenetic Inhibitor of Apoptosis Protein (IAP)-dependent Protein Erasers (SNIPERs)</span></div><div class="casAuthors">Ohoka, Nobumichi; Okuhira, Keiichiro; Ito, Masahiro; Nagai, Katsunori; Shibata, Norihito; Hattori, Takayuki; Ujikawa, Osamu; Shimokawa, Kenichiro; Sano, Osamu; Koyama, Ryokichi; Fujita, Hisashi; Teratani, Mika; Matsumoto, Hirokazu; Imaeda, Yasuhiro; Nara, Hiroshi; Cho, Nobuo; Naito, Mikihiko</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">292</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4556-4570</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Many diseases, esp. cancers, result from aberrant or overexpression of pathogenic proteins.  Specific inhibitors against these proteins have shown remarkable therapeutic effects, but these are limited mainly to enzymes.  An alternative approach that may have utility in drug development relies on selective degrdn. of pathogenic proteins via small chimeric mols. linking an E3 ubiquitin ligase to the targeted protein for proteasomal degrdn.  To this end, the authors recently developed a protein knockdown system based on hybrid small mol. SNIPERs (Specific and Nongenetic IAP-dependent Protein Erasers) that recruit inhibitor of the apoptosis protein (IAP) ubiquitin ligases to specifically degrade targeted proteins.  Here, the authors extend the authors' previous study to show a proof of concept of the SNIPER technol. in vivo.  By incorporating a high affinity IAP ligand, the authors developed a novel SNIPER against estrogen receptor α (ERα), SNIPER(ER)-87, that has a potent protein knockdown activity.  The SNIPER(ER) reduced ERα levels in tumor xenografts and suppressed the growth of ERα-pos. breast tumors in mice.  Mechanistically, it preferentially recruits X-linked IAP (XIAP) rather than cellular IAP1, to degrade ERα via the ubiquitin-proteasome pathway.  With this IAP ligand, potent SNIPERs against other pathogenic proteins, BCR-ABL, bromodomain-contg. protein 4 (BRD4), and phosphodiesterase-4 (PDE4) could also be developed.  These results indicate that forced ubiquitylation by SNIPERs is a useful method to achieve efficient protein knockdown with potential therapeutic activities and could also be applied to study the role of ubiquitylation in many cellular processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEzH5uNeAFabVg90H21EOLACvtfcHk0liuJWYO1pa7IA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXksVClur8%253D&md5=b09264d53c47c54343ed92d5a6d0eb79</span></div><a href="/servlet/linkout?suffix=cit8b&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M116.768853&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M116.768853%26sid%3Dliteratum%253Aachs%26aulast%3DOhoka%26aufirst%3DN.%26aulast%3DOkuhira%26aufirst%3DK.%26aulast%3DIto%26aufirst%3DM.%26aulast%3DNagai%26aufirst%3DK.%26aulast%3DShibata%26aufirst%3DN.%26aulast%3DHattori%26aufirst%3DT.%26aulast%3DUjikawa%26aufirst%3DO.%26aulast%3DShimokawa%26aufirst%3DK.%26aulast%3DSano%26aufirst%3DO.%26aulast%3DKoyama%26aufirst%3DR.%26aulast%3DFujita%26aufirst%3DH.%26aulast%3DTeratani%26aufirst%3DM.%26aulast%3DMatsumoto%26aufirst%3DH.%26aulast%3DImaeda%26aufirst%3DY.%26aulast%3DNara%26aufirst%3DH.%26aulast%3DCho%26aufirst%3DN.%26aulast%3DNaito%26aufirst%3DM.%26atitle%3DIn%2520vivo%2520knockdown%2520of%2520pathogenic%2520proteins%2520via%2520specific%2520and%2520nongenetic%2520inhibitor%2520of%2520apoptosis%2520protein%2520%2528IAP%2529-dependent%2520protein%2520erasers%2520%2528SNIPERs%2529%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2017%26volume%3D292%26spage%3D4556%26epage%3D4570%26doi%3D10.1074%2Fjbc.M116.768853" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of a small-molecule degrader of bromodomain and extra-terminal (BET) proteins with picomolar cellular potencies and capable of achieving tumor regression</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">462</span>– <span class="NLM_lpage">481</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01816</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01816" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvVajsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=462-481&author=B.+Zhouauthor=J.+Huauthor=F.+Xuauthor=Z.+Chenauthor=L.+Baiauthor=E.+Fernandez-Salasauthor=M.+Linauthor=L.+Liuauthor=C.-Y.+Yangauthor=Y.+Zhaoauthor=D.+McEachernauthor=S.+Przybranowskiauthor=B.+Wenauthor=D.+Sunauthor=S.+Wang&title=Discovery+of+a+small-molecule+degrader+of+bromodomain+and+extra-terminal+%28BET%29+proteins+with+picomolar+cellular+potencies+and+capable+of+achieving+tumor+regression&doi=10.1021%2Facs.jmedchem.6b01816"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9aR"><div class="casContent"><span class="casTitleNuber">9a</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Small-Molecule Degrader of Bromodomain and Extra-Terminal (BET) Proteins with Picomolar Cellular Potencies and Capable of Achieving Tumor Regression</span></div><div class="casAuthors">Zhou, Bing; Hu, Jiantao; Xu, Fuming; Chen, Zhuo; Bai, Longchuan; Fernandez-Salas, Ester; Lin, Mei; Liu, Liu; Yang, Chao-Yie; Zhao, Yujun; McEachern, Donna; Przybranowski, Sally; Wen, Bo; Sun, Duxin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">462-481</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extra-terminal (BET) family proteins, consisting of BRD2, BRD3, BRD4, and testis-specific BRDT members, are epigenetic "readers" and play a key role in the regulation of gene transcription.  BET proteins are considered to be attractive therapeutic targets for cancer and other human diseases.  Recently, heterobifunctional small-mol. BET degraders have been designed based upon the proteolysis targeting chimera (PROTAC) concept to induce BET protein degrdn.  Herein, we present our design, synthesis, and evaluation of a new class of PROTAC BET degraders.  One of the most promising compds., 23, effectively degrades BRD4 protein at concns. as low as 30 pM in the RS4;11 leukemia cell line, achieves an IC50 value of 51 pM in inhibition of RS4;11 cell growth and induces rapid tumor regression in vivo against RS4;11 xenograft tumors.  These data establish that compd. 23 (BETd-260/ZBC260) is a highly potent and efficacious BET degrader.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqz2Lwp52x1rVg90H21EOLACvtfcHk0liuJWYO1pa7IA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvVajsb8%253D&md5=6b3a10ecd192a5812ef581dca1fdc82a</span></div><a href="/servlet/linkout?suffix=cit9a&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01816&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01816%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DLin%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520a%2520small-molecule%2520degrader%2520of%2520bromodomain%2520and%2520extra-terminal%2520%2528BET%2529%2520proteins%2520with%2520picomolar%2520cellular%2520potencies%2520and%2520capable%2520of%2520achieving%2520tumor%2520regression%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D462%26epage%3D481%26doi%3D10.1021%2Facs.jmedchem.6b01816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit9b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziazadeh, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of QCA570 as an exceptionally potent and efficacious proteolysis targeting chimera (PROTAC) degrader of the bromodomain and extra-terminal (BET) proteins capable of inducing complete and durable tumor regression</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">6685</span>– <span class="NLM_lpage">6704</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00506</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00506" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlaqsLrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6685-6704&author=C.+Qinauthor=Y.+Huauthor=B.+Zhouauthor=E.+Fernandez-Salasauthor=C.-Y.+Yangauthor=L.+Liuauthor=D.+McEachernauthor=S.+Przybranowskiauthor=M.+Wangauthor=J.+Stuckeyauthor=J.+Meagherauthor=L.+Baiauthor=Z.+Chenauthor=M.+Linauthor=J.+Yangauthor=D.+N.+Ziazadehauthor=F.+Xuauthor=J.+Huauthor=W.+Xiangauthor=L.+Huangauthor=S.+Liauthor=B.+Wenauthor=D.+Sunauthor=S.+Wang&title=Discovery+of+QCA570+as+an+exceptionally+potent+and+efficacious+proteolysis+targeting+chimera+%28PROTAC%29+degrader+of+the+bromodomain+and+extra-terminal+%28BET%29+proteins+capable+of+inducing+complete+and+durable+tumor+regression&doi=10.1021%2Facs.jmedchem.8b00506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9bR"><div class="casContent"><span class="casTitleNuber">9b</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of QCA570 as an Exceptionally Potent and Efficacious Proteolysis Targeting Chimera (PROTAC) Degrader of the Bromodomain and Extra-Terminal (BET) Proteins Capable of Inducing Complete and Durable Tumor Regression</span></div><div class="casAuthors">Qin, Chong; Hu, Yang; Zhou, Bing; Fernandez-Salas, Ester; Yang, Chao-Yie; Liu, Liu; McEachern, Donna; Przybranowski, Sally; Wang, Mi; Stuckey, Jeanne; Meagher, Jennifer; Bai, Longchuan; Chen, Zhuo; Lin, Mei; Yang, Jiuling; Ziazadeh, Danya N.; Xu, Fuming; Hu, Jiantao; Xing, Weiguo; Huang, Liyue; Li, Siwei; Wen, Bo; Sun, Duxin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6685-6704</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proteins of the bromodomain and extra-terminal (BET) family are epigenetics "readers" and promising therapeutic targets for cancer and other human diseases.  We describe herein a structure-guided design of [1,4]oxazepines as a new class of BET inhibitors and our subsequent design, synthesis, and evaluation of proteolysis-targeting chimeric (PROTAC) small-mol. BET degraders.  Our efforts have led to the discovery of extremely potent BET degraders, exemplified by QCA570, which effectively induces degrdn. of BET proteins and inhibits cell growth in human acute leukemia cell lines even at low picomolar concns.  QCA570 achieves complete and durable tumor regression in leukemia xenograft models in mice at well-tolerated dose-schedules.  QCA570 is the most potent and efficacious BET degrader reported to date.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6ZUIsfX3p6LVg90H21EOLACvtfcHk0lhKxKCvdMFazQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlaqsLrE&md5=2a731730641372f4526543e03d9ac76f</span></div><a href="/servlet/linkout?suffix=cit9b&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00506%26sid%3Dliteratum%253Aachs%26aulast%3DQin%26aufirst%3DC.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DMeagher%26aufirst%3DJ.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DLin%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DZiazadeh%26aufirst%3DD.%2BN.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DXiang%26aufirst%3DW.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520QCA570%2520as%2520an%2520exceptionally%2520potent%2520and%2520efficacious%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%2520degrader%2520of%2520the%2520bromodomain%2520and%2520extra-terminal%2520%2528BET%2529%2520proteins%2520capable%2520of%2520inducing%2520complete%2520and%2520durable%2520tumor%2520regression%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D6685%26epage%3D6704%26doi%3D10.1021%2Facs.jmedchem.8b00506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit9c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, A. M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K. G.</span></span> <span> </span><span class="NLM_article-title">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">7124</span>– <span class="NLM_lpage">7129</span>, <span class="refDoi"> DOI: 10.1073/pnas.1521738113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1073%2Fpnas.1521738113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=27274052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=7124-7129&author=K.+Rainaauthor=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=D.+Gordonauthor=A.+M.+K.+Rossiauthor=J.+Wangauthor=X.+Chenauthor=H.+Dongauthor=K.+Siuauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=C.+M.+Crewsauthor=K.+G.+Coleman&title=PROTAC-induced+BET+protein+degradation+as+a+therapy+for+castration-resistant+prostate+cancer&doi=10.1073%2Fpnas.1521738113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9cR"><div class="casContent"><span class="casTitleNuber">9c</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span></div><div class="casAuthors">Raina, Kanak; Lu, Jing; Qian, Yimin; Altieri, Martha; Gordon, Deborah; Rossi, Ann Marie K.; Wang, Jing; Chen, Xin; Dong, Hanqing; Siu, Kam; Winkler, James D.; Crew, Andrew P.; Crews, Craig M.; Coleman, Kevin G.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">7124-7129</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Prostate cancer has the second highest incidence among cancers in men worldwide and is the second leading cause of cancer deaths of men in the United States.  Although androgen deprivation can initially lead to remission, the disease often progresses to castration-resistant prostate cancer (CRPC), which is still reliant on androgen receptor (AR) signaling and is assocd. with a poor prognosis.  Some success against CRPC has been achieved by drugs that target AR signaling, but secondary resistance invariably emerges, and new therapies are urgently needed.  Recently, inhibitors of bromodomain and extra-terminal (BET) family proteins have shown growth-inhibitory activity in preclin. models of CRPC.  Here, we demonstrate that ARV-771, a small-mol. pan-BET degrader based on proteolysis-targeting chimera (PROTAC) technol., demonstrates dramatically improved efficacy in cellular models of CRPC as compared with BET inhibition.  Unlike BET inhibitors, ARV-771 results in suppression of both AR signaling and AR levels and leads to tumor regression in a CRPC mouse xenograft model.  This study is, to our knowledge, the first to demonstrate efficacy with a small-mol. BET degrader in a solid-tumor malignancy and potentially represents an important therapeutic advance in the treatment of CRPC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD7QtYgZVig7Vg90H21EOLACvtfcHk0lhKxKCvdMFazQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D&md5=34218e685ad72950f3eb85586110c43d</span></div><a href="/servlet/linkout?suffix=cit9c&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1521738113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1521738113%26sid%3Dliteratum%253Aachs%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DGordon%26aufirst%3DD.%26aulast%3DRossi%26aufirst%3DA.%2BM.%2BK.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DSiu%26aufirst%3DK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26atitle%3DPROTAC-induced%2520BET%2520protein%2520degradation%2520as%2520a%2520therapy%2520for%2520castration-resistant%2520prostate%2520cancer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2016%26volume%3D113%26spage%3D7124%26epage%3D7129%26doi%3D10.1073%2Fpnas.1521738113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit9d"><span><span class="NLM_label">(d) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Selective small molecule induced degradation of the BET bromodomain protein BRD4</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1770</span>– <span class="NLM_lpage">1777</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00216</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00216" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1770-1777&author=M.+Zengerleauthor=K.-H.+Chanauthor=A.+Ciulli&title=Selective+small+molecule+induced+degradation+of+the+BET+bromodomain+protein+BRD4&doi=10.1021%2Facschembio.5b00216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9dR"><div class="casContent"><span class="casTitleNuber">9d</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Small Molecule Induced Degradation of the BET Bromodomain Protein BRD4</span></div><div class="casAuthors">Zengerle, Michael; Chan, Kwok-Ho; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1770-1777</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Bromo- and Extra-Terminal (BET) proteins BRD2, BRD3, and BRD4 play important roles in transcriptional regulation, epigenetics, and cancer and are the targets of pan-BET selective bromodomain inhibitor JQ1.  However, the lack of intra-BET selectivity limits the scope of current inhibitors as probes for target validation and could lead to unwanted side effects or toxicity in a therapeutic setting.  We designed Proteolysis Targeted Chimeras (PROTACs) that tether JQ1 to a ligand for the E3 ubiquitin ligase VHL, aimed at triggering the intracellular destruction of BET proteins.  Compd. MZ1 potently and rapidly induces reversible, long-lasting, and unexpectedly selective removal of BRD4 over BRD2 and BRD3.  The activity of MZ1 is dependent on binding to VHL but is achieved at a sufficiently low concn. not to induce stabilization of HIF-1α.  Gene expression profiles of selected cancer-related genes responsive to JQ1 reveal distinct and more limited transcriptional responses induced by MZ1, consistent with selective suppression of BRD4.  Our discovery opens up new opportunities to elucidate the cellular phenotypes and therapeutic implications assocd. with selective targeting of BRD4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEmSv7p9MZOrVg90H21EOLACvtfcHk0lhKxKCvdMFazQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D&md5=b7514fd274eff8e4bb6a771f5aaf9ab5</span></div><a href="/servlet/linkout?suffix=cit9d&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00216%26sid%3Dliteratum%253Aachs%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DK.-H.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DSelective%2520small%2520molecule%2520induced%2520degradation%2520of%2520the%2520BET%2520bromodomain%2520protein%2520BRD4%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D1770%26epage%3D1777%26doi%3D10.1021%2Facschembio.5b00216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit9e"><span><span class="NLM_label">(e) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Phthalimide conjugation as a strategy for in vivo target protein degradation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>348</i></span>,  <span class="NLM_fpage">1376</span>– <span class="NLM_lpage">1381</span>, <span class="refDoi"> DOI: 10.1126/science.aab1433</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9e&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1126%2Fscience.aab1433" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9e&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=25999370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9e&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=348&publication_year=2015&pages=1376-1381&author=G.+E.+Winterauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=J.+M.+Robertsauthor=A.+Souzaauthor=S.+Dhe-Paganonauthor=J.+E.+Bradner&title=Phthalimide+conjugation+as+a+strategy+for+in+vivo+target+protein+degradation&doi=10.1126%2Fscience.aab1433"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9eR"><div class="casContent"><span class="casTitleNuber">9e</span><div class="casTitle"><span class="NLM_cas:atitle">Phthalimide conjugation as a strategy for in vivo target protein degradation</span></div><div class="casAuthors">Winter, Georg E.; Buckley, Dennis L.; Paulk, Joshiawa; Roberts, Justin M.; Souza, Amanda; Dhe-Paganon, Sirano; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">348</span>
        (<span class="NLM_cas:issue">6241</span>),
    <span class="NLM_cas:pages">1376-1381</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The development of effective pharmacol. inhibitors of multidomain scaffold proteins, notably transcription factors, is a particularly challenging problem.  In part, this is because many small-mol. antagonists disrupt the activity of only one domain in the target protein.  The authors devised a chem. strategy that promotes ligand-dependent target protein degrdn. using as an example the transcriptional coactivator BRD4, a protein crit. for cancer cell growth and survival.  The authors appended a competitive antagonist of BET bromodomains to a phthalimide moiety to hijack the cereblon E3 ubiquitin ligase complex.  The resultant compd., dBET1, induced highly selective cereblon-dependent BET protein degrdn. in vitro and in vivo and delayed leukemia progression in mice.  A second series of probes resulted in selective degrdn. of the cytosolic protein FKBP12.  This chem. strategy for controlling target protein stability may have implications for therapeutically targeting previously intractable proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1ZW4zjRjXi7Vg90H21EOLACvtfcHk0ljacDULUpLQbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D&md5=b0277cc95d318d926b96e7a119066e3b</span></div><a href="/servlet/linkout?suffix=cit9e&amp;dbid=16384&amp;doi=10.1126%2Fscience.aab1433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aab1433%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DPhthalimide%2520conjugation%2520as%2520a%2520strategy%2520for%2520in%2520vivo%2520target%2520protein%2520degradation%26jtitle%3DScience%26date%3D2015%26volume%3D348%26spage%3D1376%26epage%3D1381%26doi%3D10.1126%2Fscience.aab1433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erb, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roderick, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vittori, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reyes, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">di Iulio, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeid, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lachance, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dastjerdi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelliher, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Churchman, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">BET bromodomain proteins function as master transcription elongation factors independent of CDK9 recruitment</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">5</span>– <span class="NLM_lpage">18</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2017.06.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1016%2Fj.molcel.2017.06.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=28673542" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFSmsbjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2017&pages=5-18&author=G.+E.+Winterauthor=A.+Mayerauthor=D.+L.+Buckleyauthor=M.+A.+Erbauthor=J.+E.+Roderickauthor=S.+Vittoriauthor=J.+M.+Reyesauthor=J.+di+Iulioauthor=A.+Souzaauthor=C.+J.+Ottauthor=J.+M.+Robertsauthor=R.+Zeidauthor=T.+G.+Scottauthor=J.+Paulkauthor=K.+Lachanceauthor=C.+M.+Olsonauthor=S.+Dastjerdiauthor=S.+Bauerauthor=C.+Y.+Linauthor=N.+S.+Grayauthor=M.+A.+Kelliherauthor=L.+S.+Churchmanauthor=J.+E.+Bradner&title=BET+bromodomain+proteins+function+as+master+transcription+elongation+factors+independent+of+CDK9+recruitment&doi=10.1016%2Fj.molcel.2017.06.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">BET Bromodomain Proteins Function as Master Transcription Elongation Factors Independent of CDK9 Recruitment</span></div><div class="casAuthors">Winter, Georg E.; Mayer, Andreas; Buckley, Dennis L.; Erb, Michael A.; Roderick, Justine E.; Vittori, Sarah; Reyes, Jaime M.; di Iulio, Julia; Souza, Amanda; Ott, Christopher J.; Roberts, Justin M.; Zeid, Rhamy; Scott, Thomas G.; Paulk, Joshiawa; Lachance, Kate; Olson, Calla M.; Dastjerdi, Shiva; Bauer, Sophie; Lin, Charles Y.; Gray, Nathanael S.; Kelliher, Michelle A.; Churchman, L. Stirling; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-18.e19</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Processive elongation of RNA Polymerase II from a proximal promoter paused state is a rate-limiting event in human gene control.  A small no. of regulatory factors influence transcription elongation on a global scale.  Prior research using small-mol. BET bromodomain inhibitors, such as JQ1, linked BRD4 to context-specific elongation at a limited no. of genes assocd. with massive enhancer regions.  Here, the mechanistic characterization of an optimized chem. degrader of BET bromodomain proteins, dBET6, led to the unexpected identification of BET proteins as master regulators of global transcription elongation.  In contrast to the selective effect of bromodomain inhibition on transcription, BET degrdn. prompts a collapse of global elongation that phenocopies CDK9 inhibition.  Notably, BRD4 loss does not directly affect CDK9 localization.  These studies, performed in translational models of T cell leukemia, establish a mechanism-based rationale for the development of BET bromodomain degrdn. as cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqseJAHmXAb-rVg90H21EOLACvtfcHk0ljacDULUpLQbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFSmsbjM&md5=ac6f4e33a8f43008b70d4ede80a526be</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2017.06.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2017.06.004%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DMayer%26aufirst%3DA.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DErb%26aufirst%3DM.%2BA.%26aulast%3DRoderick%26aufirst%3DJ.%2BE.%26aulast%3DVittori%26aufirst%3DS.%26aulast%3DReyes%26aufirst%3DJ.%2BM.%26aulast%3Ddi%2BIulio%26aufirst%3DJ.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DOtt%26aufirst%3DC.%2BJ.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DZeid%26aufirst%3DR.%26aulast%3DScott%26aufirst%3DT.%2BG.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DLachance%26aufirst%3DK.%26aulast%3DOlson%26aufirst%3DC.%2BM.%26aulast%3DDastjerdi%26aufirst%3DS.%26aulast%3DBauer%26aufirst%3DS.%26aulast%3DLin%26aufirst%3DC.%2BY.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DKelliher%26aufirst%3DM.%2BA.%26aulast%3DChurchman%26aufirst%3DL.%2BS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DBET%2520bromodomain%2520proteins%2520function%2520as%2520master%2520transcription%2520elongation%2520factors%2520independent%2520of%2520CDK9%2520recruitment%26jtitle%3DMol.%2520Cell%26date%3D2017%26volume%3D67%26spage%3D5%26epage%3D18%26doi%3D10.1016%2Fj.molcel.2017.06.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shultz, M. D.</span></span> <span> </span><span class="NLM_article-title">Two decades under the influence of the rule of five and the changing properties of approved oral drugs</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">1701</span>– <span class="NLM_lpage">1714</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00686</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00686" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslait73K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=1701-1714&author=M.+D.+Shultz&title=Two+decades+under+the+influence+of+the+rule+of+five+and+the+changing+properties+of+approved+oral+drugs&doi=10.1021%2Facs.jmedchem.8b00686"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Two Decades under the Influence of the Rule of Five and the Changing Properties of Approved Oral Drugs</span></div><div class="casAuthors">Shultz, Michael D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1701-1714</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Two decades have passed since the rule of five ushered in the concept of "drug-like" properties.  Attempts to quantify, correlate, and categorize mols. based on Ro5 parameters evolved into the introduction of efficiency metrics with far reaching consequences in decision making by industry leaders and scientists seeking to discover new medicines.  Examn. of oral drug parameters approved before and after the original Ro5 anal. demonstrates that some parameters such as clogP and HBD remained const. while the cutoffs for parameters such as mol. wt. and HBA have increased substantially over the past 20 years.  The time dependent increase in the mol. wt. of oral drugs during the past 20 years provides compelling evidence to disprove the hypothesis that mol. wt. is a "drug-like" property.  This anal. does not validate parameters that have not changed as being "drug-like" but instead calls into question the entire hypothesis that "drug-like" properties exist.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAnoxpQsv2kbVg90H21EOLACvtfcHk0ljapGr6dWyo0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslait73K&md5=d3c8b1107133dc0b7b277d22fa029597</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00686&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00686%26sid%3Dliteratum%253Aachs%26aulast%3DShultz%26aufirst%3DM.%2BD.%26atitle%3DTwo%2520decades%2520under%2520the%2520influence%2520of%2520the%2520rule%2520of%2520five%2520and%2520the%2520changing%2520properties%2520of%2520approved%2520oral%2520drugs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D1701%26epage%3D1714%26doi%3D10.1021%2Facs.jmedchem.8b00686" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Edmondson, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fallan, C.</span></span> <span> </span><span class="NLM_article-title">Proteolysis targeting chimeras (PROTACs) in ‘beyond rule-of-five’ chemical space: Recent progress and future challenges</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1555</span>– <span class="NLM_lpage">1564</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2019.04.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1016%2Fj.bmcl.2019.04.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=31047748" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1MXos12gs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=1555-1564&author=S.+D.+Edmondsonauthor=B.+Yangauthor=C.+Fallan&title=Proteolysis+targeting+chimeras+%28PROTACs%29+in+%E2%80%98beyond+rule-of-five%E2%80%99+chemical+space%3A+Recent+progress+and+future+challenges&doi=10.1016%2Fj.bmcl.2019.04.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12aR"><div class="casContent"><span class="casTitleNuber">12a</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis targeting chimeras (PROTACs) in 'beyond rule-of-five' chemical space: Recent progress and future challenges</span></div><div class="casAuthors">Edmondson, Scott D.; Yang, Bin; Fallan, Charlene</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1555-1564</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Proteolysis targeting chimeras (PROTACs) are heterobifunctional compds. with mol. wts. and other properties that lie outside the classic 'rule-of-five' space.  Consequently, PROTACs have unique challenges assocd. with their development as potential therapeutic agents.  This review summarizes and analyzes a representative set of recent PROTACs and highlights some of the potential future challenges facing this promising modality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDcS0gIul-K7Vg90H21EOLACvtfcHk0ljapGr6dWyo0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXos12gs74%253D&md5=62a74500178bd711b1ddaa33d731b960</span></div><a href="/servlet/linkout?suffix=cit12a&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2019.04.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2019.04.030%26sid%3Dliteratum%253Aachs%26aulast%3DEdmondson%26aufirst%3DS.%2BD.%26aulast%3DYang%26aufirst%3DB.%26aulast%3DFallan%26aufirst%3DC.%26atitle%3DProteolysis%2520targeting%2520chimeras%2520%2528PROTACs%2529%2520in%2520%25E2%2580%2598beyond%2520rule-of-five%25E2%2580%2599%2520chemical%2520space%253A%2520Recent%2520progress%2520and%2520future%2520challenges%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D29%26spage%3D1555%26epage%3D1564%26doi%3D10.1016%2Fj.bmcl.2019.04.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit12b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Maple, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clayden, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baron, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stacey, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felix, R.</span></span> <span> </span><span class="NLM_article-title">Developing degraders: principles and perspectives on design and chemical space</span>. <i>Med. Chem. Commum.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1755</span>– <span class="NLM_lpage">1764</span>, <span class="refDoi"> DOI: 10.1039/C9MD00272C</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1039%2FC9MD00272C" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1755-1764&author=H.+J.+Mapleauthor=N.+Claydenauthor=A.+Baronauthor=C.+Staceyauthor=R.+Felix&title=Developing+degraders%3A+principles+and+perspectives+on+design+and+chemical+space&doi=10.1039%2FC9MD00272C"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12b&amp;dbid=16384&amp;doi=10.1039%2FC9MD00272C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC9MD00272C%26sid%3Dliteratum%253Aachs%26aulast%3DMaple%26aufirst%3DH.%2BJ.%26aulast%3DClayden%26aufirst%3DN.%26aulast%3DBaron%26aufirst%3DA.%26aulast%3DStacey%26aufirst%3DC.%26aulast%3DFelix%26aufirst%3DR.%26atitle%3DDeveloping%2520degraders%253A%2520principles%2520and%2520perspectives%2520on%2520design%2520and%2520chemical%2520space%26jtitle%3DMed.%2520Chem.%2520Commum.%26date%3D2019%26volume%3D10%26spage%3D1755%26epage%3D1764%26doi%3D10.1039%2FC9MD00272C" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Petrylak, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogelzang, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garfield, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peck, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burris, H. A.</span></span> <span> </span><span class="NLM_article-title">First-in-human phase I study of ARV-110, an androgen receptor (AR) PROTAC degrader in patients (pts) with metastatic castrate-resistant prostate cancer (mCRPC) following enzalutamide (ENZ) and/or abiraterone (ABI)</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">3500</span>, <span class="refDoi"> DOI: 10.1200/JCO.2020.38.15_suppl.3500</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1200%2FJCO.2020.38.15_suppl.3500" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2020&pages=3500&author=D.+P.+Petrylakauthor=X.+Gaoauthor=N.+J.+Vogelzangauthor=M.+H.+Garfieldauthor=I.+Taylorauthor=M.+D.+Mooreauthor=R.+A.+Peckauthor=H.+A.+Burris&title=First-in-human+phase+I+study+of+ARV-110%2C+an+androgen+receptor+%28AR%29+PROTAC+degrader+in+patients+%28pts%29+with+metastatic+castrate-resistant+prostate+cancer+%28mCRPC%29+following+enzalutamide+%28ENZ%29+and%2For+abiraterone+%28ABI%29&doi=10.1200%2FJCO.2020.38.15_suppl.3500"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13a&amp;dbid=16384&amp;doi=10.1200%2FJCO.2020.38.15_suppl.3500&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2020.38.15_suppl.3500%26sid%3Dliteratum%253Aachs%26aulast%3DPetrylak%26aufirst%3DD.%2BP.%26aulast%3DGao%26aufirst%3DX.%26aulast%3DVogelzang%26aufirst%3DN.%2BJ.%26aulast%3DGarfield%26aufirst%3DM.%2BH.%26aulast%3DTaylor%26aufirst%3DI.%26aulast%3DMoore%26aufirst%3DM.%2BD.%26aulast%3DPeck%26aufirst%3DR.%2BA.%26aulast%3DBurris%26aufirst%3DH.%2BA.%26atitle%3DFirst-in-human%2520phase%2520I%2520study%2520of%2520ARV-110%252C%2520an%2520androgen%2520receptor%2520%2528AR%2529%2520PROTAC%2520degrader%2520in%2520patients%2520%2528pts%2529%2520with%2520metastatic%2520castrate-resistant%2520prostate%2520cancer%2520%2528mCRPC%2529%2520following%2520enzalutamide%2520%2528ENZ%2529%2520and%252For%2520abiraterone%2520%2528ABI%2529%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2020%26volume%3D38%26spage%3D3500%26doi%3D10.1200%2FJCO.2020.38.15_suppl.3500" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit13b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, R.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span> <span> </span><span class="NLM_article-title">A chemical approach for global protein knockdown from mice to non-human primates</span>. <i>Cell Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>5</i></span>, <span class="NLM_elocation-id">10</span> <span class="refDoi"> DOI: 10.1038/s41421-018-0079-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1038%2Fs41421-018-0079-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=30729032" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A280%3ADC%252BB3cjpvFKiug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2019&author=X.+Sunauthor=J.+Wangauthor=X.+Yaoauthor=W.+Zhengauthor=Y.+Maoauthor=T.+Lanauthor=L.+Wangauthor=Y.+Sunauthor=X.+Zhangauthor=Q.+Zhaoauthor=J.+Zhaoauthor=R.-P.+Xiaoauthor=X.+Zhangauthor=G.+Jiauthor=Y.+Rao&title=A+chemical+approach+for+global+protein+knockdown+from+mice+to+non-human+primates&doi=10.1038%2Fs41421-018-0079-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13bR"><div class="casContent"><span class="casTitleNuber">13b</span><div class="casTitle"><span class="NLM_cas:atitle">A chemical approach for global protein knockdown from mice to non-human primates</span></div><div class="casAuthors">Sun Xiuyun; Yao Xia; Lan Tianlong; Wang Liguo; Sun Yonghui; Zhao Qiuye; Rao Yu; Sun Xiuyun; Yao Xia; Lan Tianlong; Wang Liguo; Sun Yonghui; Zhao Qiuye; Rao Yu; Sun Xiuyun; Wang Jun; Mao Yang; Zhang Xinyi; Ji Guangju; Wang Jun; Mao Yang; Zhang Xinyi; Zheng Wen; Zhang Xiuqin; Zheng Wen; Zhang Xiuqin; Zhao Jianguo; Xiao Rui-Ping; Xiao Rui-Ping</div><div class="citationInfo"><span class="NLM_cas:title">Cell discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">10</span>
        ISSN:<span class="NLM_cas:issn">2056-5968</span>.
    </div><div class="casAbstract">Although conventional genetic modification approaches for protein knockdown work very successfully due to the increasing use of CRISPR/Cas9, effective techniques for achieving protein depletion in adult animals, especially in large animals such as non-human primates, are lacking.  Here, we report a chemical approach based on PROTACs technology that efficiently and quickly knocks down FKBP12 (12-kDa FK506-binding) protein globally in vivo.  Both intraperitoneal and oral administration led to rapid, robust, and reversible FKBP12 degradation in mice.  The efficiency and practicality of this method were successfully demonstrated in both large and small animals (mice, rats, Bama pigs, and rhesus monkeys).  Furthermore, we showed this approach can also be applied to effectively knockdown other target proteins such as Bruton's tyrosine kinase (BTK).  This chemical protein knockdown strategy provides a powerful research tool for gene function studies in animals, particularly in large animals, for which gene-targeted knockout strategies may remain unfeasible.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSOtjU0KY5kR9ts4TFFB11gfW6udTcc2eYdrehegPIdXrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cjpvFKiug%253D%253D&md5=199267298a928382ed1a91a7c066eefa</span></div><a href="/servlet/linkout?suffix=cit13b&amp;dbid=16384&amp;doi=10.1038%2Fs41421-018-0079-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41421-018-0079-1%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DYao%26aufirst%3DX.%26aulast%3DZheng%26aufirst%3DW.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DLan%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DZhao%26aufirst%3DJ.%26aulast%3DXiao%26aufirst%3DR.-P.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DJi%26aufirst%3DG.%26aulast%3DRao%26aufirst%3DY.%26atitle%3DA%2520chemical%2520approach%2520for%2520global%2520protein%2520knockdown%2520from%2520mice%2520to%2520non-human%2520primates%26jtitle%3DCell%2520Discovery%26date%3D2019%26volume%3D5%26doi%3D10.1038%2Fs41421-018-0079-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Pike, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harlfinger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGinnity, D. F.</span></span> <span> </span><span class="NLM_article-title">Optimising proteolysis-targeting chimeras (PROTACs) for oral drug delivery: a drug metabolism and pharmacokinetics perspective</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1793</span>– <span class="NLM_lpage">1800</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2020.07.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1016%2Fj.drudis.2020.07.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVeru7zF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2020&pages=1793-1800&author=A.+Pikeauthor=B.+Williamsonauthor=S.+Harlfingerauthor=S.+Martinauthor=D.+F.+McGinnity&title=Optimising+proteolysis-targeting+chimeras+%28PROTACs%29+for+oral+drug+delivery%3A+a+drug+metabolism+and+pharmacokinetics+perspective&doi=10.1016%2Fj.drudis.2020.07.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14aR"><div class="casContent"><span class="casTitleNuber">14a</span><div class="casTitle"><span class="NLM_cas:atitle">Optimising proteolysis-targeting chimeras (PROTACs) for oral drug delivery: a drug metabolism and pharmacokinetics perspective</span></div><div class="casAuthors">Pike, Andy; Williamson, Beth; Harlfinger, Stephanie; Martin, Scott; McGinnity, Dermot F.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1793-1800</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Proteolysis-targeting chimeras (PROTACs) are an emerging therapeutic modality with the potential to open target space not accessible to conventional small mols. via a degrdn.-based mechanism; however, their bifunctional nature can result in physicochem. properties that breach commonly accepted limits for small-mol. oral drugs.  We offer a drug metab. and pharmacokinetics (DMPK) perspective on the optimization of oral PROTACs across a diverse set of projects within Oncol. R&D at AstraZeneca, highlighting some of the challenges that they have presented to our established screening cascade.  Furthermore, we challenge some of the perceptions and dogma surrounding the feasibility of oral PROTACS and demonstrate that acceptable oral PK properties for this modality can be regularly achievable despite the physicochem. property challenges they present.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNes0zq6ODubVg90H21EOLACvtfcHk0lh3Zn69Cs3AXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVeru7zF&md5=a26048e1fe2eb5f9e134a9132593df23</span></div><a href="/servlet/linkout?suffix=cit14a&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2020.07.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2020.07.013%26sid%3Dliteratum%253Aachs%26aulast%3DPike%26aufirst%3DA.%26aulast%3DWilliamson%26aufirst%3DB.%26aulast%3DHarlfinger%26aufirst%3DS.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DMcGinnity%26aufirst%3DD.%2BF.%26atitle%3DOptimising%2520proteolysis-targeting%2520chimeras%2520%2528PROTACs%2529%2520for%2520oral%2520drug%2520delivery%253A%2520a%2520drug%2520metabolism%2520and%2520pharmacokinetics%2520perspective%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2020%26volume%3D25%26spage%3D1793%26epage%3D1800%26doi%3D10.1016%2Fj.drudis.2020.07.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit14b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Cantrill, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaturvedi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rynn, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaffland, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walter, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittwer, M. B.</span></span> <span> </span><span class="NLM_article-title">Fundamental aspects of DMPK optimization of targeted protein degraders</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">969</span>– <span class="NLM_lpage">982</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2020.03.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1016%2Fj.drudis.2020.03.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=32298797" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BB3cXntFGht7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2020&pages=969-982&author=C.+Cantrillauthor=P.+Chaturvediauthor=C.+Rynnauthor=J.+P.+Schafflandauthor=I.+Walterauthor=M.+B.+Wittwer&title=Fundamental+aspects+of+DMPK+optimization+of+targeted+protein+degraders&doi=10.1016%2Fj.drudis.2020.03.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14bR"><div class="casContent"><span class="casTitleNuber">14b</span><div class="casTitle"><span class="NLM_cas:atitle">Fundamental aspects of DMPK optimization of targeted protein degraders</span></div><div class="casAuthors">Cantrill, Carina; Chaturvedi, Prasoon; Rynn, Caroline; Petrig Schaffland, Jeannine; Walter, Isabelle; Wittwer, Matthias B.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">969-982</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Targeted protein degraders are an emerging modality.  Their properties fall outside the traditional small-mol. property space and are in the 'beyond rule of 5' space.  Consequently, drug discovery programs focused on developing orally bioavailable degraders are expected to face complex drug metab. and pharmacokinetics (DMPK) challenges compared with traditional small mols.  Nevertheless, little information is available on the DMPK optimization of oral degraders.  Therefore, in this review, we discuss our experience of these DMPK optimization challenges and present methodologies and strategies to overcome the hurdles dealing with this new small-mol. modality, specifically in developing oral degraders to treat cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoz2WUoP7TUvbVg90H21EOLACvtfcHk0lh3Zn69Cs3AXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXntFGht7s%253D&md5=465ad2dab5183e0f229aeec7e57e396a</span></div><a href="/servlet/linkout?suffix=cit14b&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2020.03.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2020.03.012%26sid%3Dliteratum%253Aachs%26aulast%3DCantrill%26aufirst%3DC.%26aulast%3DChaturvedi%26aufirst%3DP.%26aulast%3DRynn%26aufirst%3DC.%26aulast%3DSchaffland%26aufirst%3DJ.%2BP.%26aulast%3DWalter%26aufirst%3DI.%26aulast%3DWittwer%26aufirst%3DM.%2BB.%26atitle%3DFundamental%2520aspects%2520of%2520DMPK%2520optimization%2520of%2520targeted%2520protein%2520degraders%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2020%26volume%3D25%26spage%3D969%26epage%3D982%26doi%3D10.1016%2Fj.drudis.2020.03.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Beck, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goetsch, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumontet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corvaïa, N.</span></span> <span> </span><span class="NLM_article-title">Strategies and challenges for the next generation of antibody-drug conjugates</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">315</span>– <span class="NLM_lpage">337</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.268</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1038%2Fnrd.2016.268" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=28303026" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkt1Onsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=315-337&author=A.+Beckauthor=L.+Goetschauthor=C.+Dumontetauthor=N.+Corva%C3%AFa&title=Strategies+and+challenges+for+the+next+generation+of+antibody-drug+conjugates&doi=10.1038%2Fnrd.2016.268"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15aR"><div class="casContent"><span class="casTitleNuber">15a</span><div class="casTitle"><span class="NLM_cas:atitle">Strategies and challenges for the next generation of antibody-drug conjugates</span></div><div class="casAuthors">Beck, Alain; Goetsch, Liliane; Dumontet, Charles; Corvaia, Nathalie</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">315-337</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Antibody-drug conjugates (ADCs) are one of the fastest growing classes of oncol. therapeutics.  After half a century of research, the approvals of brentuximab vedotin (in 2011) and trastuzumab emtansine (in 2013) have paved the way for ongoing clin. trials that are evaluating more than 60 further ADC candidates.  The limited success of first-generation ADCs (developed in the early 2000s) informed strategies to bring second-generation ADCs to the market, which have higher levels of cytotoxic drug conjugation, lower levels of naked antibodies and more-stable linkers between the drug and the antibody.  Furthermore, lessons learned during the past decade are now being used in the development of third-generation ADCs.  In this Review, we discuss strategies to select the best target antigens as well as suitable cytotoxic drugs; the design of optimized linkers; the discovery of bioorthogonal conjugation chemistries; and toxicity issues.  The selection and engineering of antibodies for site-specific drug conjugation, which will result in higher homogeneity and increased stability, as well as the quest for new conjugation chemistries and mechanisms of action, are priorities in ADC research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqP7lV63Ws8rbVg90H21EOLACvtfcHk0lh3Zn69Cs3AXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkt1Onsbc%253D&md5=7358b6080dd1aed1d21e98c078cd9ef1</span></div><a href="/servlet/linkout?suffix=cit15a&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.268&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.268%26sid%3Dliteratum%253Aachs%26aulast%3DBeck%26aufirst%3DA.%26aulast%3DGoetsch%26aufirst%3DL.%26aulast%3DDumontet%26aufirst%3DC.%26aulast%3DCorva%25C3%25AFa%26aufirst%3DN.%26atitle%3DStrategies%2520and%2520challenges%2520for%2520the%2520next%2520generation%2520of%2520antibody-drug%2520conjugates%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D315%26epage%3D337%26doi%3D10.1038%2Fnrd.2016.268" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit15b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Sau, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alsaab, H. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kashaw, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tatiparti, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iyer, A. K.</span></span> <span> </span><span class="NLM_article-title">Advances in antibody-drug conjugates: A new era of targeted cancer therapy</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">1547</span>– <span class="NLM_lpage">1556</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2017.05.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1016%2Fj.drudis.2017.05.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=28627385" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVCrsLzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=1547-1556&author=S.+Sauauthor=H.+O.+Alsaabauthor=S.+K.+Kashawauthor=K.+Tatipartiauthor=A.+K.+Iyer&title=Advances+in+antibody-drug+conjugates%3A+A+new+era+of+targeted+cancer+therapy&doi=10.1016%2Fj.drudis.2017.05.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15bR"><div class="casContent"><span class="casTitleNuber">15b</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in antibody-drug conjugates: A new era of targeted cancer therapy</span></div><div class="casAuthors">Sau, Samaresh; Alsaab, Hashem O.; Kashaw, Sushil Kumar; Tatiparti, Katyayani; Iyer, Arun K.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1547-1556</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Antibody-drug conjugates (ADCs), a potent class of anticancer therapeutics, comprise a high-affinity antibody (Ab) and cytotoxic payload coupled via a suitable linker for selective tumor cell killing.  In the initial phase of their development, two ADCs, Mylotarg, and Adcetris were approved by the US Food and Drug Administration (FDA) for treating hematological cancer, but the real breakthrough came with the discovery of the breast cancer-targeting ADC, Kadcyla.  With advances in bioengineering, linker chem., and potent cytotoxic payload, ADC technol. has become a more powerful tool for targeted cancer therapy.  In addn., ADCs with improved safety using humanized Abs with a unified 'drug:antibody ratio' (DAR) have been achieved.  Concomitantly, there has been a significant increase in the no. of clin. trials with anticancer ADCs with high translation potential.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohgxck59tjx7Vg90H21EOLACvtfcHk0lisbe3IOy_MJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVCrsLzK&md5=dc5204e59d81ffa946b5d4ea07caec70</span></div><a href="/servlet/linkout?suffix=cit15b&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2017.05.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2017.05.011%26sid%3Dliteratum%253Aachs%26aulast%3DSau%26aufirst%3DS.%26aulast%3DAlsaab%26aufirst%3DH.%2BO.%26aulast%3DKashaw%26aufirst%3DS.%2BK.%26aulast%3DTatiparti%26aufirst%3DK.%26aulast%3DIyer%26aufirst%3DA.%2BK.%26atitle%3DAdvances%2520in%2520antibody-drug%2520conjugates%253A%2520A%2520new%2520era%2520of%2520targeted%2520cancer%2520therapy%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2017%26volume%3D22%26spage%3D1547%26epage%3D1556%26doi%3D10.1016%2Fj.drudis.2017.05.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit15c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Chari, R. V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Widdison, W. C.</span></span> <span> </span><span class="NLM_article-title">Antibody-drug conjugates: An emerging concept in cancer therapy</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">3796</span>– <span class="NLM_lpage">3827</span>, <span class="refDoi"> DOI: 10.1002/anie.201307628</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1002%2Fanie.201307628" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtVGmsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=3796-3827&author=R.+V.+J.+Chariauthor=M.+L.+Millerauthor=W.+C.+Widdison&title=Antibody-drug+conjugates%3A+An+emerging+concept+in+cancer+therapy&doi=10.1002%2Fanie.201307628"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15cR"><div class="casContent"><span class="casTitleNuber">15c</span><div class="casTitle"><span class="NLM_cas:atitle">Antibody-Drug Conjugates: An Emerging Concept in Cancer Therapy</span></div><div class="casAuthors">Chari, Ravi V. J.; Miller, Michael L.; Widdison, Wayne C.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3796-3827</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Traditional cancer chemotherapy is often accompanied by systemic toxicity to the patient.  Monoclonal antibodies against antigens on cancer cells offer an alternative tumor-selective treatment approach.  However, most monoclonal antibodies are not sufficiently potent to be therapeutically active on their own.  Antibody-drug conjugates (ADCs) use antibodies to deliver a potent cytotoxic compd. selectively to tumor cells, thus improving the therapeutic index of chemotherapeutic agents.  The recent approval of two ADCs, brentuximab vedotin and ado-trastuzumab emtansine, for cancer treatment has spurred tremendous research interest in this field.  This Review touches upon the early efforts in the field, and describes how the lessons learned from the first-generation ADCs have led to improvements in every aspect of this technol., i.e., the antibody, the cytotoxic compd., and the linker connecting them, leading to the current successes.  The design of ADCs currently in clin. development, and results from mechanistic studies and preclin. and clin. evaluation are discussed.  Emerging technologies that seek to further advance this exciting area of research are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4T2rhmji6DLVg90H21EOLACvtfcHk0lisbe3IOy_MJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtVGmsr0%253D&md5=fbfb2f843f516654ea7c0eb98949604c</span></div><a href="/servlet/linkout?suffix=cit15c&amp;dbid=16384&amp;doi=10.1002%2Fanie.201307628&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201307628%26sid%3Dliteratum%253Aachs%26aulast%3DChari%26aufirst%3DR.%2BV.%2BJ.%26aulast%3DMiller%26aufirst%3DM.%2BL.%26aulast%3DWiddison%26aufirst%3DW.%2BC.%26atitle%3DAntibody-drug%2520conjugates%253A%2520An%2520emerging%2520concept%2520in%2520cancer%2520therapy%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2014%26volume%3D53%26spage%3D3796%26epage%3D3827%26doi%3D10.1002%2Fanie.201307628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span>; <span class="NLM_string-name">Pillow, T. H.</span>; <span class="NLM_string-name">Blake, R. A.</span>; <span class="NLM_string-name">Sadowsky, J. D.</span>; <span class="NLM_string-name">Adaligil, E.</span>; <span class="NLM_string-name">Adhikari, P.</span>; <span class="NLM_string-name">Bhakta, S.</span>; <span class="NLM_string-name">Blaquiere, N.</span>; <span class="NLM_string-name">Chen, J.</span>; <span class="NLM_string-name">dela
Cruz-Chuh, J.</span>; <span class="NLM_string-name">Gascoigne, K. E.</span>; <span class="NLM_string-name">Hartman, S. J.</span>; <span class="NLM_string-name">He, M.</span>; <span class="NLM_string-name">Kaufman, S.</span>; <span class="NLM_string-name">Kleinheinz, T.</span>; <span class="NLM_string-name">Kozak, K. R.</span>; <span class="NLM_string-name">Liu, L.</span>; <span class="NLM_string-name">Liu, L.</span>; <span class="NLM_string-name">Liu, Q.</span>; <span class="NLM_string-name">Lu, Y.</span>; <span class="NLM_string-name">Meng, F.</span>; <span class="NLM_string-name">Mulvihill, M. M.</span>; <span class="NLM_string-name">O’Donohue, A.</span>; <span class="NLM_string-name">Rowntree, R. K.</span>; <span class="NLM_string-name">Staben, L. R.</span>; <span class="NLM_string-name">Staben, S. T.</span>; <span class="NLM_string-name">Wai, J.</span>; <span class="NLM_string-name">Wang, J.</span>; <span class="NLM_string-name">Wei, B.</span>; <span class="NLM_string-name">Wilson, C.</span>; <span class="NLM_string-name">Xin, J.</span>; <span class="NLM_string-name">Xu, Z.</span>; <span class="NLM_string-name">Yao, H.</span>; <span class="NLM_string-name">Zhang, D.</span>; <span class="NLM_string-name">Zhang, H.</span>; <span class="NLM_string-name">Zhou, H.</span>; <span class="NLM_string-name">Zhu, X.</span></span> <span> </span><span class="NLM_article-title">Antibody-mediated delivery of chimeric BRD4 degraders. Part 1: Exploration of antibody linker, payload loading, and payload molecular properties</span>.  <i>J. Med. Chem.</i> preceding paper in this issue.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Dragovich%2C+P.+S.%3B+Pillow%2C+T.+H.%3B+Blake%2C+R.+A.%3B+Sadowsky%2C+J.+D.%3B+Adaligil%2C+E.%3B+Adhikari%2C+P.%3B+Bhakta%2C+S.%3B+Blaquiere%2C+N.%3B+Chen%2C+J.%3B+dela%0ACruz-Chuh%2C+J.%3B+Gascoigne%2C+K.+E.%3B+Hartman%2C+S.+J.%3B+He%2C+M.%3B+Kaufman%2C+S.%3B+Kleinheinz%2C+T.%3B+Kozak%2C+K.+R.%3B+Liu%2C+L.%3B+Liu%2C+L.%3B+Liu%2C+Q.%3B+Lu%2C+Y.%3B+Meng%2C+F.%3B+Mulvihill%2C+M.+M.%3B+O%E2%80%99Donohue%2C+A.%3B+Rowntree%2C+R.+K.%3B+Staben%2C+L.+R.%3B+Staben%2C+S.+T.%3B+Wai%2C+J.%3B+Wang%2C+J.%3B+Wei%2C+B.%3B+Wilson%2C+C.%3B+Xin%2C+J.%3B+Xu%2C+Z.%3B+Yao%2C+H.%3B+Zhang%2C+D.%3B+Zhang%2C+H.%3B+Zhou%2C+H.%3B+Zhu%2C+X.+Antibody-mediated+delivery+of+chimeric+BRD4+degraders.+Part+1%3A+Exploration+of+antibody+linker%2C+payload+loading%2C+and+payload+molecular+properties.+J.+Med.+Chem.+preceding+paper+in+this+issue."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26atitle%3DAntibody-mediated%2520delivery%2520of%2520chimeric%2520BRD4%2520degraders.%2520Part%25201%253A%2520Exploration%2520of%2520antibody%2520linker%252C%2520payload%2520loading%252C%2520and%2520payload%2520molecular%2520properties%26jtitle%3DJ.%2520Med.%2520Chem." title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pillow, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adhikari, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blake, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Rosario, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshmukh, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Figueroa, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gascoigne, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamath, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleinheinz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozak, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Latifi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leipold, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulvihill, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Donohue, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowntree, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadowsky, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span></span> <span> </span><span class="NLM_article-title">Antibody conjugation of a chimeric BET degrader enables in vivo activity</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">17</span>– <span class="NLM_lpage">25</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201900497</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1002%2Fcmdc.201900497" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=31674143" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFCmt7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2020&pages=17-25&author=T.+H.+Pillowauthor=P.+Adhikariauthor=R.+A.+Blakeauthor=J.+Chenauthor=G.+Del+Rosarioauthor=G.+Deshmukhauthor=I.+Figueroaauthor=K.+E.+Gascoigneauthor=A.+V.+Kamathauthor=S.+Kaufmanauthor=T.+Kleinheinzauthor=K.+R.+Kozakauthor=B.+Latifiauthor=D.+D.+Leipoldauthor=C.+S.+Liauthor=R.+Liauthor=M.+M.+Mulvihillauthor=A.+O%E2%80%99Donohueauthor=R.+K.+Rowntreeauthor=J.+D.+Sadowskyauthor=J.+Waiauthor=X.+Wangauthor=C.+Wuauthor=Z.+Xuauthor=H.+Yaoauthor=S.-F.+Yuauthor=D.+Zhangauthor=R.+Zangauthor=H.+Zhangauthor=H.+Zhouauthor=X.+Zhuauthor=P.+S.+Dragovich&title=Antibody+conjugation+of+a+chimeric+BET+degrader+enables+in+vivo+activity&doi=10.1002%2Fcmdc.201900497"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Antibody Conjugation of a Chimeric BET Degrader Enables in vivo Activity</span></div><div class="casAuthors">Pillow, Thomas H.; Adhikari, Pragya; Blake, Robert A.; Chen, Jinhua; Del Rosario, Geoffrey; Deshmukh, Gauri; Figueroa, Isabel; Gascoigne, Karen E.; Kamath, Amrita V.; Kaufman, Susan; Kleinheinz, Tracy; Kozak, Katherine R.; Latifi, Brandon; Leipold, Douglas D.; Sing Li, Chun; Li, Ruina; Mulvihill, Melinda M.; O'Donohue, Aimee; Rowntree, Rebecca K.; Sadowsky, Jack D.; Wai, John; Wang, Xinxin; Wu, Cong; Xu, Zijin; Yao, Hui; Yu, Shang-Fan; Zhang, Donglu; Zang, Richard; Zhang, Hongyan; Zhou, Hao; Zhu, Xiaoyu; Dragovich, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">17-25</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The ability to selectively degrade proteins with bifunctional small mols. has the potential to fundamentally alter therapy in a variety of diseases.  However, the relatively large size of these chimeric mols. often results in challenging physico-chem. properties (e. g., low aq. soly.) and poor pharmacokinetics which may complicate their in vivo applications.  We recently discovered an exquisitely potent chimeric BET degrader (GNE-987) which exhibited picomolar cell potencies but also demonstrated low in vivo exposures.  In an effort to improve the pharmacokinetic properties of this mol., we discovered the first degrader-antibody conjugate by attaching GNE-987 to an anti-CLL1 antibody via a novel linker.  A single IV dose of the conjugate afforded sustained in vivo exposures that resulted in antigen-specific tumor regressions.  Enhancement of a chimeric protein degrader with poor in vivo properties through antibody conjugation thereby expands the utility of directed protein degrdn. as both a biol. tool and a therapeutic possibility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPzNLh7QlBR7Vg90H21EOLACvtfcHk0lisbe3IOy_MJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFCmt7zP&md5=90f9e1b5ae453ae38d61a36587feb558</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201900497&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201900497%26sid%3Dliteratum%253Aachs%26aulast%3DPillow%26aufirst%3DT.%2BH.%26aulast%3DAdhikari%26aufirst%3DP.%26aulast%3DBlake%26aufirst%3DR.%2BA.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DDel%2BRosario%26aufirst%3DG.%26aulast%3DDeshmukh%26aufirst%3DG.%26aulast%3DFigueroa%26aufirst%3DI.%26aulast%3DGascoigne%26aufirst%3DK.%2BE.%26aulast%3DKamath%26aufirst%3DA.%2BV.%26aulast%3DKaufman%26aufirst%3DS.%26aulast%3DKleinheinz%26aufirst%3DT.%26aulast%3DKozak%26aufirst%3DK.%2BR.%26aulast%3DLatifi%26aufirst%3DB.%26aulast%3DLeipold%26aufirst%3DD.%2BD.%26aulast%3DLi%26aufirst%3DC.%2BS.%26aulast%3DLi%26aufirst%3DR.%26aulast%3DMulvihill%26aufirst%3DM.%2BM.%26aulast%3DO%25E2%2580%2599Donohue%26aufirst%3DA.%26aulast%3DRowntree%26aufirst%3DR.%2BK.%26aulast%3DSadowsky%26aufirst%3DJ.%2BD.%26aulast%3DWai%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DC.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DYao%26aufirst%3DH.%26aulast%3DYu%26aufirst%3DS.-F.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DZang%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26atitle%3DAntibody%2520conjugation%2520of%2520a%2520chimeric%2520BET%2520degrader%2520enables%2520in%2520vivo%2520activity%26jtitle%3DChemMedChem%26date%3D2020%26volume%3D15%26spage%3D17%26epage%3D25%26doi%3D10.1002%2Fcmdc.201900497" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adhikari, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blake, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blaquiere, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">den Besten, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartman, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingalla, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamath, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleinheinz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leipold, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pillow, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowntree, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadowsky, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sampath, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staben, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staben, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wertz, I. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span> <span> </span><span class="NLM_article-title">Antibody-mediated delivery of chimeric protein degraders which target estrogen receptor alpha (ERα)</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>30</i></span>, <span class="NLM_elocation-id">126907</span> <span class="refDoi"> DOI: 10.1016/j.bmcl.2019.126907</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1016%2Fj.bmcl.2019.126907" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=31902710" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BB3cXksFWgtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2020&author=P.+S.+Dragovichauthor=P.+Adhikariauthor=R.+A.+Blakeauthor=N.+Blaquiereauthor=J.+Chenauthor=Y.-X.+Chengauthor=W.+den+Bestenauthor=J.+Hanauthor=S.+J.+Hartmanauthor=J.+Heauthor=M.+Heauthor=E.+R.+Ingallaauthor=A.+V.+Kamathauthor=T.+Kleinheinzauthor=T.+Laiauthor=D.+D.+Leipoldauthor=C.+S.+Liauthor=Q.+Liuauthor=J.+Luauthor=Y.+Luauthor=F.+Mengauthor=L.+Mengauthor=C.+Ngauthor=K.+Pengauthor=G.+L.+Phillipsauthor=T.+H.+Pillowauthor=R.+K.+Rowntreeauthor=J.+D.+Sadowskyauthor=D.+Sampathauthor=L.+Stabenauthor=S.+T.+Stabenauthor=J.+Waiauthor=K.+Wanauthor=X.+Wangauthor=B.+Weiauthor=I.+E.+Wertzauthor=J.+Xinauthor=K.+Xuauthor=H.+Yaoauthor=R.+Zangauthor=D.+Zhangauthor=H.+Zhouauthor=Y.+Zhao&title=Antibody-mediated+delivery+of+chimeric+protein+degraders+which+target+estrogen+receptor+alpha+%28ER%CE%B1%29&doi=10.1016%2Fj.bmcl.2019.126907"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Antibody-mediated delivery of chimeric protein degraders which target estrogen receptor alpha (ERα)</span></div><div class="casAuthors">Dragovich, Peter S.; Adhikari, Pragya; Blake, Robert A.; Blaquiere, Nicole; Chen, Jinhua; Cheng, Yun-Xing; den Besten, Willem; Han, Jinping; Hartman, Steven J.; He, Jintang; He, Mingtao; Rei Ingalla, Ellen; Kamath, Amrita V.; Kleinheinz, Tracy; Lai, Tommy; Leipold, Douglas D.; Li, Chun Sing; Liu, Qi; Lu, Jiawei; Lu, Ying; Meng, Fanwei; Meng, Lingyao; Ng, Carl; Peng, Kaishan; Lewis Phillips, Gail; Pillow, Thomas H.; Rowntree, Rebecca K.; Sadowsky, Jack D.; Sampath, Deepak; Staben, Leanna; Staben, Steven T.; Wai, John; Wan, Kunpeng; Wang, Xinxin; Wei, Bin Qing; Wertz, Ingrid E.; Xin, Jianfeng; Xu, Keyang; Yao, Hui; Zang, Richard; Zhang, Donglu; Zhou, Hao; Zhao, Yongxin</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">126907</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Chimeric mols. which effect intracellular degrdn. of target proteins via E3 ligase-mediated ubiquitination (e.g., PROTACs) are currently of high interest in medicinal chem.  However, these entities are relatively large compds. that often possess mol. characteristics which may compromise oral bioavailability, soly., and/or in vivo pharmacokinetic properties.  Accordingly, we explored whether conjugation of chimeric degraders to monoclonal antibodies using technologies originally developed for cytotoxic payloads might provide alternate delivery options for these novel agents.  In this report we describe the construction of several degrader-antibody conjugates comprised of two distinct ERα-targeting degrader entities and three independent ADC linker modalities.  We subsequently demonstrate the antigen-dependent delivery to MCF7-neo/HER2 cells of the degrader payloads that are incorporated into these conjugates.  We also provide evidence for efficient intracellular degrader release from one of the employed linkers.  In addn., preliminary data are described which suggest that reasonably favorable in vivo stability properties are assocd. with the linkers utilized to construct the degrader conjugates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdPWcGkXC85LVg90H21EOLACvtfcHk0lijTiN5tH_l4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXksFWgtw%253D%253D&md5=5f196b194df7f4622825c4b901130f7c</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2019.126907&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2019.126907%26sid%3Dliteratum%253Aachs%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26aulast%3DAdhikari%26aufirst%3DP.%26aulast%3DBlake%26aufirst%3DR.%2BA.%26aulast%3DBlaquiere%26aufirst%3DN.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DCheng%26aufirst%3DY.-X.%26aulast%3Dden%2BBesten%26aufirst%3DW.%26aulast%3DHan%26aufirst%3DJ.%26aulast%3DHartman%26aufirst%3DS.%2BJ.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DIngalla%26aufirst%3DE.%2BR.%26aulast%3DKamath%26aufirst%3DA.%2BV.%26aulast%3DKleinheinz%26aufirst%3DT.%26aulast%3DLai%26aufirst%3DT.%26aulast%3DLeipold%26aufirst%3DD.%2BD.%26aulast%3DLi%26aufirst%3DC.%2BS.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DMeng%26aufirst%3DF.%26aulast%3DMeng%26aufirst%3DL.%26aulast%3DNg%26aufirst%3DC.%26aulast%3DPeng%26aufirst%3DK.%26aulast%3DPhillips%26aufirst%3DG.%2BL.%26aulast%3DPillow%26aufirst%3DT.%2BH.%26aulast%3DRowntree%26aufirst%3DR.%2BK.%26aulast%3DSadowsky%26aufirst%3DJ.%2BD.%26aulast%3DSampath%26aufirst%3DD.%26aulast%3DStaben%26aufirst%3DL.%26aulast%3DStaben%26aufirst%3DS.%2BT.%26aulast%3DWai%26aufirst%3DJ.%26aulast%3DWan%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DWei%26aufirst%3DB.%26aulast%3DWertz%26aufirst%3DI.%2BE.%26aulast%3DXin%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DK.%26aulast%3DYao%26aufirst%3DH.%26aulast%3DZang%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DY.%26atitle%3DAntibody-mediated%2520delivery%2520of%2520chimeric%2520protein%2520degraders%2520which%2520target%2520estrogen%2520receptor%2520alpha%2520%2528ER%25CE%25B1%2529%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D30%26doi%3D10.1016%2Fj.bmcl.2019.126907" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maneiro, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forte, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shchepinova, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kounde, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chudasama, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tate, E. W.</span></span> <span> </span><span class="NLM_article-title">Antibody-PROTAC conjugates enable HER2-dependent targeted protein degradation of BRD4</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1306</span>– <span class="NLM_lpage">1312</span>, <span class="refDoi"> DOI: 10.1021/acschembio.0c00285</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.0c00285" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BB3cXotFehsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2020&pages=1306-1312&author=M.+Maneiroauthor=N.+Forteauthor=M.+M.+Shchepinovaauthor=C.+S.+Koundeauthor=V.+Chudasamaauthor=J.+R.+Bakerauthor=E.+W.+Tate&title=Antibody-PROTAC+conjugates+enable+HER2-dependent+targeted+protein+degradation+of+BRD4&doi=10.1021%2Facschembio.0c00285"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Antibody-PROTAC Conjugates Enable HER2-Dependent Targeted Protein Degradation of BRD4</span></div><div class="casAuthors">Maneiro, Maria; Forte, Nafsika; Shchepinova, Maria M.; Kounde, Cyrille S.; Chudasama, Vijay; Baker, James Richard; Tate, Edward W.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1306-1312</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Targeting protein degrdn. with Proteolysis-Targeting Chimeras (PROTACs) is an area of great current interest in drug discovery.  Nevertheless, although the high effectiveness of PROTACs against a wide variety of targets has been established, most degraders reported to date display limited intrinsic tissue selectivity and do not discriminate between cells of different types.  Here, we describe a strategy for selective protein degrdn. in a specific cell type.  We report the design and synthesis of a trastuzumab-PROTAC conjugate (Ab-PROTAC 3) in which E3 ligase-directed degrader activity is caged with an antibody linker which can be hydrolyzed following antibody-PROTAC internalization, releasing the active PROTAC and inducing catalytic protein degrdn.  We show that 3 selectively targets bromodomain-contg. protein 4 (BRD4) for degrdn. only in HER2 pos. breast cancer cell lines, while sparing HER2 neg. cells.  Using live cell confocal microscopy, we show internalization and lysosomal trafficking of the conjugate specifically in HER2 pos. cells, leading to the release of active PROTAC in quantities sufficient to induce potent BRD4 degrdn.  These studies demonstrate proof-of-concept for tissue-specific BRD4 degrdn., overcoming limitations of PROTAC selectivity, with significant potential for application to novel targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPwGNLaGmRbbVg90H21EOLACvtfcHk0lijTiN5tH_l4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXotFehsLo%253D&md5=544b53e54d3f0696f0b425c3f6e48e46</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facschembio.0c00285&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.0c00285%26sid%3Dliteratum%253Aachs%26aulast%3DManeiro%26aufirst%3DM.%26aulast%3DForte%26aufirst%3DN.%26aulast%3DShchepinova%26aufirst%3DM.%2BM.%26aulast%3DKounde%26aufirst%3DC.%2BS.%26aulast%3DChudasama%26aufirst%3DV.%26aulast%3DBaker%26aufirst%3DJ.%2BR.%26aulast%3DTate%26aufirst%3DE.%2BW.%26atitle%3DAntibody-PROTAC%2520conjugates%2520enable%2520HER2-dependent%2520targeted%2520protein%2520degradation%2520of%2520BRD4%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2020%26volume%3D15%26spage%3D1306%26epage%3D1312%26doi%3D10.1021%2Facschembio.0c00285" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morse, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickman, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christie, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Thangue, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiest, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of BET bromodomains</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1067</span>– <span class="NLM_lpage">1073</span>, <span class="refDoi"> DOI: 10.1038/nature09504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1038%2Fnature09504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=20871596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1067-1073&author=P.+Filippakopoulosauthor=J.+Qiauthor=S.+Picaudauthor=Y.+Shenauthor=W.+B.+Smithauthor=O.+Fedorovauthor=E.+M.+Morseauthor=T.+Keatesauthor=T.+T.+Hickmanauthor=I.+Felletarauthor=M.+Philpottauthor=S.+Munroauthor=M.+R.+McKeownauthor=Y.+Wangauthor=A.+L.+Christieauthor=N.+Westauthor=M.+J.+Cameronauthor=B.+Schwartzauthor=T.+D.+Heightmanauthor=N.+La+Thangueauthor=C.+A.+Frenchauthor=O.+Wiestauthor=A.+L.+Kungauthor=S.+Knappauthor=J.+E.+Bradner&title=Selective+inhibition+of+BET+bromodomains&doi=10.1038%2Fnature09504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of BET bromodomains</span></div><div class="casAuthors">Filippakopoulos, Panagis; Qi, Jun; Picaud, Sarah; Shen, Yao; Smith, William B.; Fedorov, Oleg; Morse, Elizabeth M.; Keates, Tracey; Hickman, Tyler T.; Felletar, Ildiko; Philpott, Martin; Munro, Shongah; McKeown, Michael R.; Wang, Yuchuan; Christie, Amanda L.; West, Nathan; Cameron, Michael J.; Schwartz, Brian; Heightman, Tom D.; La Thangue, Nicholas; French, Christopher; Wiest, Olaf; Kung, Andrew L.; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic proteins are intently pursued targets in ligand discovery.  So far, successful efforts have been limited to chromatin modifying enzymes, or so-called epigenetic 'writers' and 'erasers'.  Potent inhibitors of histone binding modules have not yet been described.  Here the authors report a cell-permeable small mol. (I,JQ1) that binds competitively to acetyl-lysine recognition motifs, or bromodomains.  High potency and specificity towards a subset of human bromodomains is explained by co-crystal structures with bromodomain and extra-terminal (BET) family member BRD4, revealing excellent shape complementarity with the acetyl-lysine binding cavity.  Recurrent translocation of BRD4 is obsd. in a genetically-defined, incurable subtype of human squamous carcinoma.  Competitive binding by JQ1 displaces the BRD4 fusion oncoprotein from chromatin, prompting squamous differentiation and specific antiproliferative effects in BRD4-dependent cell lines and patient-derived xenograft models.  These data establish proof-of-concept for targeting protein-protein interactions of epigenetic 'readers', and provide a versatile chem. scaffold for the development of chem. probes more broadly throughout the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ZbtmZM8m_LVg90H21EOLACvtfcHk0lifmDKfsD4A2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D&md5=bd9cadac86124d2c6c3cd5b2a92d68c2</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnature09504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09504%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DW.%2BB.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DHickman%26aufirst%3DT.%2BT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChristie%26aufirst%3DA.%2BL.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DSchwartz%26aufirst%3DB.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DLa%2BThangue%26aufirst%3DN.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSelective%2520inhibition%2520of%2520BET%2520bromodomains%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1067%26epage%3D1073%26doi%3D10.1038%2Fnature09504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantei, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holms, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguirre, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogdan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dietrich, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marjanovic, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutchins, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bui, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magoc, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajaraman, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, K. F.</span></span> <span> </span><span class="NLM_article-title">Discovery and optimization of novel constrained pyrrolopyridone BET family inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">1804</span>– <span class="NLM_lpage">1810</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2018.04.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1016%2Fj.bmcl.2018.04.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=29678460" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1cXotVamtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=1804-1810&author=S.+D.+Fidanzeauthor=D.+Liuauthor=R.+A.+Manteiauthor=L.+A.+Hasvoldauthor=J.+K.+Prattauthor=G.+S.+Sheppardauthor=L.+Wangauthor=J.+H.+Holmsauthor=Y.+Daiauthor=A.+Aguirreauthor=A.+Bogdanauthor=J.+D.+Dietrichauthor=J.+Marjanovicauthor=C.+H.+Parkauthor=C.+W.+Hutchinsauthor=X.+Linauthor=M.+H.+Buiauthor=X.+Huangauthor=D.+Wilcoxauthor=L.+Liauthor=R.+Wangauthor=P.+Kovarauthor=T.+J.+Magocauthor=G.+Rajaramanauthor=D.+H.+Albertauthor=Y.+Shenauthor=W.+M.+Katiauthor=K.+F.+McDaniel&title=Discovery+and+optimization+of+novel+constrained+pyrrolopyridone+BET+family+inhibitors&doi=10.1016%2Fj.bmcl.2018.04.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21aR"><div class="casContent"><span class="casTitleNuber">21a</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and optimization of novel constrained pyrrolopyridone BET family inhibitors</span></div><div class="casAuthors">Fidanze, Steven D.; Liu, Dachun; Mantei, Robert A.; Hasvold, Lisa A.; Pratt, John K.; Sheppard, George S.; Wang, Le; Holms, James H.; Dai, Yujia; Aguirre, Ana; Bogdan, Andrew; Dietrich, Justin D.; Marjanovic, Jasmina; Park, Chang H.; Hutchins, Charles W.; Lin, Xiaoyu; Bui, Mai H.; Huang, Xiaoli; Wilcox, Denise; Li, Leiming; Wang, Rongqi; Kovar, Peter; Magoc, Terrance J.; Rajaraman, Ganesh; Albert, Daniel H.; Shen, Yu; Kati, Warren M.; McDaniel, Keith F.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1804-1810</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Novel conformationally constrained BET bromodomain inhibitors have been developed.  These inhibitors were optimized in two similar, yet distinct chem. series, the 6-methyl-1H-pyrrolo[2,3-c]pyridin-7(6H)-ones (A) and the 1-methyl-1H-pyrrolo[2,3-c]pyridin-7(6H)-ones (B).  Each series demonstrated excellent activity in binding and cellular assays, and lead compds. from each series demonstrated significant efficacy in in vivo tumor xenograft models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbjffbFRUNM7Vg90H21EOLACvtfcHk0li3UbLoVTJ8xw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXotVamtbg%253D&md5=2ac2fba2c203dabfc34216f6b0717d1d</span></div><a href="/servlet/linkout?suffix=cit21a&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2018.04.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2018.04.020%26sid%3Dliteratum%253Aachs%26aulast%3DFidanze%26aufirst%3DS.%2BD.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DMantei%26aufirst%3DR.%2BA.%26aulast%3DHasvold%26aufirst%3DL.%2BA.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DHolms%26aufirst%3DJ.%2BH.%26aulast%3DDai%26aufirst%3DY.%26aulast%3DAguirre%26aufirst%3DA.%26aulast%3DBogdan%26aufirst%3DA.%26aulast%3DDietrich%26aufirst%3DJ.%2BD.%26aulast%3DMarjanovic%26aufirst%3DJ.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DHutchins%26aufirst%3DC.%2BW.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DBui%26aufirst%3DM.%2BH.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DWilcox%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DKovar%26aufirst%3DP.%26aulast%3DMagoc%26aufirst%3DT.%2BJ.%26aulast%3DRajaraman%26aufirst%3DG.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DKati%26aufirst%3DW.%2BM.%26aulast%3DMcDaniel%26aufirst%3DK.%2BF.%26atitle%3DDiscovery%2520and%2520optimization%2520of%2520novel%2520constrained%2520pyrrolopyridone%2520BET%2520family%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D28%26spage%3D1804%26epage%3D1810%26doi%3D10.1016%2Fj.bmcl.2018.04.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit21b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S. D.</span>; <span class="NLM_string-name">Liu, D.</span>; <span class="NLM_string-name">Mantei, R. A.</span>; <span class="NLM_string-name">Mcdaniel, K. F.</span>; <span class="NLM_string-name">Pratt, J.</span>; <span class="NLM_string-name">Sheppard, G. S.</span>; <span class="NLM_string-name">Wang, L.</span>; <span class="NLM_string-name">Bogdan, A.</span>; <span class="NLM_string-name">Holms, J. H.</span>; <span class="NLM_string-name">Dietrich, J. D.</span>; <span class="NLM_string-name">Marjanovic, J.</span>; <span class="NLM_string-name">Hasvold, L. A.</span>; <span class="NLM_string-name">Dai, Y.</span></span> <span> </span><span class="NLM_article-title">Tetracyclic Bromodomain Inhibitors</span>. International patent application, <span class="NLM_patent">WO2014139324</span>, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=S.+D.+Fidanze&author=D.+Liu&author=R.+A.+Mantei&author=K.+F.+Mcdaniel&author=J.+Pratt&author=G.+S.+Sheppard&author=L.+Wang&author=A.+Bogdan&author=J.+H.+Holms&author=J.+D.+Dietrich&author=J.+Marjanovic&author=L.+A.+Hasvold&author=Y.+Dai&title=Tetracyclic+Bromodomain+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DFidanze%26aufirst%3DS.%2BD.%26atitle%3DTetracyclic%2520Bromodomain%2520Inhibitors%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artursson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hann, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leavens, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mateus, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Readshaw, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valko, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wayne, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, A.</span></span> <span> </span><span class="NLM_article-title">Direct measurement of intracellular compound concentration by RapidFire mass spectrometry offers insights into cell permeability</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">156</span>– <span class="NLM_lpage">164</span>, <span class="refDoi"> DOI: 10.1177/1087057115604141</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1177%2F1087057115604141" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=26336900" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC28XmvVSrtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=156-164&author=L.+J.+Gordonauthor=M.+Allenauthor=P.+Arturssonauthor=M.+M.+Hannauthor=B.+J.+Leavensauthor=A.+Mateusauthor=S.+Readshawauthor=K.+Valkoauthor=G.+J.+Wayneauthor=A.+West&title=Direct+measurement+of+intracellular+compound+concentration+by+RapidFire+mass+spectrometry+offers+insights+into+cell+permeability&doi=10.1177%2F1087057115604141"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22aR"><div class="casContent"><span class="casTitleNuber">22a</span><div class="casTitle"><span class="NLM_cas:atitle">Direct measurement of intracellular compound concentration by RapidFire mass spectrometry offers insights into cell permeability</span></div><div class="casAuthors">Gordon, Laurie J.; Allen, Morven; Artursson, Per; Hann, Michael M.; Leavens, Bill J.; Mateus, Andre; Readshaw, Simon; Valko, Klara; Wayne, Gareth J.; West, Andy</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">156-164</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">One of the key challenges facing early stage drug discovery is understanding the commonly obsd. difference between the activity of compds. in biochem. assays and cellular assays.  Traditionally, indirect or estd. cell permeability measurements such as estns. from logP or artificial membrane permeability are used to explain the differences.  The missing link is a direct measurement of intracellular compd. concn. in whole cells.  This can, in some circumstances, be estd. from the cellular activity, but this may also be problematic if cellular activity is weak or absent.  Advances in sensitivity and throughput of anal. techniques have enabled us to develop a high-throughput assay for the measurement of intracellular compd. concn. for routine use to support lead optimization.  The assay uses a RapidFire-MS based readout of compd. concn. in HeLa cells following incubation of cells with test compd.  The initial assay validation was performed by ultra-high performance liq. chromatog. tandem mass spectrometry, and the assay was subsequently transferred to RapidFire tandem mass spectrometry.  Further miniaturization and optimization were performed to streamline the process, increase sample throughput, and reduce cycle time.  This optimization has delivered a semi-automated platform with the potential of prodn. scale compd. profiling up to 100 compds. per day.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLd62-6firibVg90H21EOLACvtfcHk0li3UbLoVTJ8xw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmvVSrtLc%253D&md5=34a5d2a18c1a36973425d00b87112f20</span></div><a href="/servlet/linkout?suffix=cit22a&amp;dbid=16384&amp;doi=10.1177%2F1087057115604141&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057115604141%26sid%3Dliteratum%253Aachs%26aulast%3DGordon%26aufirst%3DL.%2BJ.%26aulast%3DAllen%26aufirst%3DM.%26aulast%3DArtursson%26aufirst%3DP.%26aulast%3DHann%26aufirst%3DM.%2BM.%26aulast%3DLeavens%26aufirst%3DB.%2BJ.%26aulast%3DMateus%26aufirst%3DA.%26aulast%3DReadshaw%26aufirst%3DS.%26aulast%3DValko%26aufirst%3DK.%26aulast%3DWayne%26aufirst%3DG.%2BJ.%26aulast%3DWest%26aufirst%3DA.%26atitle%3DDirect%2520measurement%2520of%2520intracellular%2520compound%2520concentration%2520by%2520RapidFire%2520mass%2520spectrometry%2520offers%2520insights%2520into%2520cell%2520permeability%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2016%26volume%3D21%26spage%3D156%26epage%3D164%26doi%3D10.1177%2F1087057115604141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit22b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Beigi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gottschalk, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hägglund, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haneskog, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brekkan, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Österberg, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundahl, P.</span></span> <span> </span><span class="NLM_article-title">Immobilized liposome and biomembrane partitioning chromatography of drugs for prediction of drug transport</span>. <i>Int. J. Pharm.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>164</i></span>,  <span class="NLM_fpage">129</span>– <span class="NLM_lpage">137</span>, <span class="refDoi"> DOI: 10.1016/S0378-5173(97)00398-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1016%2FS0378-5173%2897%2900398-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADyaK1cXitlSrt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=164&publication_year=1998&pages=129-137&author=F.+Beigiauthor=I.+Gottschalkauthor=C.+L.+H%C3%A4gglundauthor=L.+Haneskogauthor=E.+Brekkanauthor=Y.+Zhangauthor=T.+%C3%96sterbergauthor=P.+Lundahl&title=Immobilized+liposome+and+biomembrane+partitioning+chromatography+of+drugs+for+prediction+of+drug+transport&doi=10.1016%2FS0378-5173%2897%2900398-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22bR"><div class="casContent"><span class="casTitleNuber">22b</span><div class="casTitle"><span class="NLM_cas:atitle">Immobilized liposome and biomembrane partitioning chromatography of drugs for prediction of drug transport</span></div><div class="casAuthors">Beigi, Farideh; Gottschalk, Ingo; Lagerquist Hagglund, Christine; Haneskog, Lars; Brekkan, Eggert; Zhang, Yanxiao; Osterberg, Thomas; Lundahl, Per</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Pharmaceutics</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">164</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">129-137</span>CODEN:
                <span class="NLM_cas:coden">IJPHDE</span>;
        ISSN:<span class="NLM_cas:issn">0378-5173</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">Drug partitioning into lipid bilayers was studied by chromatog. on liposomes and biomembranes immobilized in gel beads by freeze-thawing.  The drug retention vol. was expressed as a capacity factor, Ks, normalized with respect to the amt. of immobilized phospholipid.  Log Ks values for pos. charged drugs on brain phosphatidylserine (PS)/egg phosphatidylcholine (PC) liposomes decreased as the ionic strength was increased, increased as the PS:PC ratio or the pH was increased and varied linearly with the temp.  Log Ks values for beta-blockers, phenothiazines and benzodiazepines on egg phospholipid (EPL) liposomes correlated well with corresponding values on red cell membrane lipid liposomes (r2=0.96), and on human red cell membrane vesicles contg. transmembrane proteins (r2=0.96).  A fair correlation was obsd. between the values on EPL liposomes and those on native membranes of adsorbed red cells (r2=0.86).  Compared to the data obtained with liposomes, the retentions of hydrophilic drugs became larger and the range of log Ks values more narrow on the vesicles and the membranes, which expose hydrophilic protein surfaces and oligosaccharides.  Lower correlations were obsd. between drug retention on EPL liposomes and egg PC liposomes; and between retention on liposomes (or vesicles) and immobilized artificial membrane (IAM) monolayers of PC analogs.  Absorption of orally administered drugs in humans (literature data) was nearly complete for drugs of log Ks values in the interval 1.2-2.5 on vesicles.  Both vesicles and liposomes can thus be used for chromatog. anal. of drug-membrane interaction and prediction of drug absorption.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_r5N-IU8WaLVg90H21EOLACvtfcHk0lizb37FjMXzVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXitlSrt7g%253D&md5=f930fe3b4f61a23889e2dc42f510b8bc</span></div><a href="/servlet/linkout?suffix=cit22b&amp;dbid=16384&amp;doi=10.1016%2FS0378-5173%2897%2900398-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0378-5173%252897%252900398-0%26sid%3Dliteratum%253Aachs%26aulast%3DBeigi%26aufirst%3DF.%26aulast%3DGottschalk%26aufirst%3DI.%26aulast%3DH%25C3%25A4gglund%26aufirst%3DC.%2BL.%26aulast%3DHaneskog%26aufirst%3DL.%26aulast%3DBrekkan%26aufirst%3DE.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3D%25C3%2596sterberg%26aufirst%3DT.%26aulast%3DLundahl%26aufirst%3DP.%26atitle%3DImmobilized%2520liposome%2520and%2520biomembrane%2520partitioning%2520chromatography%2520of%2520drugs%2520for%2520prediction%2520of%2520drug%2520transport%26jtitle%3DInt.%2520J.%2520Pharm.%26date%3D1998%26volume%3D164%26spage%3D129%26epage%3D137%26doi%3D10.1016%2FS0378-5173%2897%2900398-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Pillow, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadowsky, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Rosario, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhakta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohri, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozak, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ha, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Junutula, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flygare, J. A.</span></span> <span> </span><span class="NLM_article-title">Decoupling stability and release in disulfide bonds with antibody-small molecule conjugates</span>. <i>Chem Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">366</span>– <span class="NLM_lpage">370</span>, <span class="refDoi"> DOI: 10.1039/C6SC01831A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1039%2FC6SC01831A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=28451181" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlOqsL%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=366-370&author=T.+H.+Pillowauthor=J.+D.+Sadowskyauthor=D.+Zhangauthor=S.-F.+Yuauthor=G.+Del+Rosarioauthor=K.+Xuauthor=J.+Heauthor=S.+Bhaktaauthor=R.+Ohriauthor=K.+R.+Kozakauthor=E.+Haauthor=J.+R.+Junutulaauthor=J.+A.+Flygare&title=Decoupling+stability+and+release+in+disulfide+bonds+with+antibody-small+molecule+conjugates&doi=10.1039%2FC6SC01831A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23aR"><div class="casContent"><span class="casTitleNuber">23a</span><div class="casTitle"><span class="NLM_cas:atitle">Decoupling stability and release in disulfide bonds with antibody-small molecule conjugates</span></div><div class="casAuthors">Pillow, Thomas H.; Sadowsky, Jack D.; Zhang, Donglu; Yu, Shang-Fan; Del Rosario, Geoffrey; Xu, Keyang; He, Jintang; Bhakta, Sunil; Ohri, Rachana; Kozak, Katherine R.; Ha, Edward; Junutula, Jagath R.; Flygare, John A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">366-370</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Disulfide bonds provide a bioactivatable connection with applications in imaging and therapy.  The circulation stability and intracellular release of disulfides are problematically coupled in that increasing stability causes a corresponding decrease in cleavage and payload release.  However, an antibody offers the potential for a reversible stabilization.  We examd. this by attaching a small mol. directly to engineered cysteines in an antibody.  At certain sites this unhindered disulfide was stable in circulation yet cellular internalization and antibody catabolism generated a disulfide catabolite that was rapidly reduced.  We demonstrated that this stable connection and facile release is applicable to a variety of payloads.  The ability to reversibly stabilize a labile functional group with an antibody may offer a way to improve targeted probes and therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocdfG3fjz5LLVg90H21EOLACvtfcHk0lizb37FjMXzVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlOqsL%252FP&md5=1ef5dd94930854c9fba36a9ddea49f3a</span></div><a href="/servlet/linkout?suffix=cit23a&amp;dbid=16384&amp;doi=10.1039%2FC6SC01831A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6SC01831A%26sid%3Dliteratum%253Aachs%26aulast%3DPillow%26aufirst%3DT.%2BH.%26aulast%3DSadowsky%26aufirst%3DJ.%2BD.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DYu%26aufirst%3DS.-F.%26aulast%3DDel%2BRosario%26aufirst%3DG.%26aulast%3DXu%26aufirst%3DK.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DBhakta%26aufirst%3DS.%26aulast%3DOhri%26aufirst%3DR.%26aulast%3DKozak%26aufirst%3DK.%2BR.%26aulast%3DHa%26aufirst%3DE.%26aulast%3DJunutula%26aufirst%3DJ.%2BR.%26aulast%3DFlygare%26aufirst%3DJ.%2BA.%26atitle%3DDecoupling%2520stability%2520and%2520release%2520in%2520disulfide%2520bonds%2520with%2520antibody-small%2520molecule%2520conjugates%26jtitle%3DChem%2520Sci.%26date%3D2017%26volume%3D8%26spage%3D366%26epage%3D370%26doi%3D10.1039%2FC6SC01831A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit23b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Pillow, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schutten, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohri, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadowsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Achilles-Poon, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solis, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Rosario, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Go, M. A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leipold, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamath, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masterson, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gregson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunzner-Toste, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozak, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spencer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polakis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polson, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flygare, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Junutula, J. R.</span></span> <span> </span><span class="NLM_article-title">Modulating therapeutic activity and toxicity of pyrrolobenzodiazepine antibody-Drug conjugates with self-immolative disulfide linkers</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">871</span>– <span class="NLM_lpage">878</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-16-0641</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1158%2F1535-7163.MCT-16-0641" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=28223423" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmvFemtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=871-878&author=T.+H.+Pillowauthor=M.+Schuttenauthor=S.-F.+Yuauthor=R.+Ohriauthor=J.+Sadowskyauthor=K.+Achilles-Poonauthor=W.+Solisauthor=F.+Zhongauthor=G.+Del+Rosarioauthor=M.+A.+T.+Goauthor=J.+Lauauthor=S.+Yeeauthor=J.+Heauthor=L.+Liuauthor=C.+Ngauthor=K.+Xuauthor=D.+D.+Leipoldauthor=A.+V.+Kamathauthor=D.+Zhangauthor=L.+Mastersonauthor=S.+J.+Gregsonauthor=P.+W.+Howardauthor=F.+Fangauthor=J.+Chenauthor=J.+Gunzner-Tosteauthor=K.+R.+Kozakauthor=S.+Spencerauthor=P.+Polakisauthor=A.+G.+Polsonauthor=J.+A.+Flygareauthor=J.+R.+Junutula&title=Modulating+therapeutic+activity+and+toxicity+of+pyrrolobenzodiazepine+antibody-Drug+conjugates+with+self-immolative+disulfide+linkers&doi=10.1158%2F1535-7163.MCT-16-0641"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23bR"><div class="casContent"><span class="casTitleNuber">23b</span><div class="casTitle"><span class="NLM_cas:atitle">Modulating Therapeutic Activity and Toxicity of Pyrrolobenzodiazepine Antibody-Drug Conjugates with Self-Immolative Disulfide Linkers</span></div><div class="casAuthors">Pillow, Thomas H.; Schutten, Melissa; Yu, Shang-Fan; Ohri, Rachana; Sadowsky, Jack; Poon, Kirsten Achilles; Solis, Willy; Zhong, Fiona; Del Rosario, Geoffrey; Go, Mary Ann T.; Lau, Jeffrey; Yee, Sharon; He, Jintang; Liu, Luna; Ng, Carl; Xu, Keyang; Leipold, Douglas D.; Kamath, Amrita V.; Zhang, Donglu; Masterson, Luke; Gregson, Stephen J.; Howard, Philip W.; Fang, Fan; Chen, Jinhua; Gunzner-Toste, Janet; Kozak, Katherine K.; Spencer, Susan; Polakis, Paul; Polson, Andrew G.; Flygare, John A.; Junutula, Jagath R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">871-878</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A novel disulfide linker was designed to enable a direct connection between cytotoxic pyrrolobenzodiazepine (PBD) drugs and the cysteine on a targeting antibody for use in antibody-drug conjugates (ADCs).  The ADCs composed of a cysteine-engineered antibody were armed with a PBD using a self-immolative disulfide linker.  Both the chem. linker and the antibody site were optimized for this new bioconjugation strategy to provide a highly stable and efficacious ADC.  This novel disulfide ADC was compared with a conjugate contg. the same PBD drug, but attached to the antibody via a peptide linker.  Both ADCs had similar efficacy in mice bearing human tumor xenografts.  Safety studies in rats revealed that the disulfide-linked ADC had a higher MTD than the peptide-linked ADC.  Overall, these data suggest that the novel self-immolative disulfide linker represents a valuable way to construct ADCs with equiv. efficacy and improved safety.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZP_imFtNeBrVg90H21EOLACvtfcHk0lizb37FjMXzVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmvFemtrw%253D&md5=6bf9501951de0cfad9d890a4ef9492b1</span></div><a href="/servlet/linkout?suffix=cit23b&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-16-0641&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-16-0641%26sid%3Dliteratum%253Aachs%26aulast%3DPillow%26aufirst%3DT.%2BH.%26aulast%3DSchutten%26aufirst%3DM.%26aulast%3DYu%26aufirst%3DS.-F.%26aulast%3DOhri%26aufirst%3DR.%26aulast%3DSadowsky%26aufirst%3DJ.%26aulast%3DAchilles-Poon%26aufirst%3DK.%26aulast%3DSolis%26aufirst%3DW.%26aulast%3DZhong%26aufirst%3DF.%26aulast%3DDel%2BRosario%26aufirst%3DG.%26aulast%3DGo%26aufirst%3DM.%2BA.%2BT.%26aulast%3DLau%26aufirst%3DJ.%26aulast%3DYee%26aufirst%3DS.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DNg%26aufirst%3DC.%26aulast%3DXu%26aufirst%3DK.%26aulast%3DLeipold%26aufirst%3DD.%2BD.%26aulast%3DKamath%26aufirst%3DA.%2BV.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DMasterson%26aufirst%3DL.%26aulast%3DGregson%26aufirst%3DS.%2BJ.%26aulast%3DHoward%26aufirst%3DP.%2BW.%26aulast%3DFang%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DGunzner-Toste%26aufirst%3DJ.%26aulast%3DKozak%26aufirst%3DK.%2BR.%26aulast%3DSpencer%26aufirst%3DS.%26aulast%3DPolakis%26aufirst%3DP.%26aulast%3DPolson%26aufirst%3DA.%2BG.%26aulast%3DFlygare%26aufirst%3DJ.%2BA.%26aulast%3DJunutula%26aufirst%3DJ.%2BR.%26atitle%3DModulating%2520therapeutic%2520activity%2520and%2520toxicity%2520of%2520pyrrolobenzodiazepine%2520antibody-Drug%2520conjugates%2520with%2520self-immolative%2520disulfide%2520linkers%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2017%26volume%3D16%26spage%3D871%26epage%3D878%26doi%3D10.1158%2F1535-7163.MCT-16-0641" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span> The mAb nomenclature
follows the EU convention described in:<span class="NLM_contrib-group"><span class="NLM_string-name">Edelman, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gall, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gotlieb, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rutishauser, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waxdal, M. J.</span></span> <span> </span><span class="NLM_article-title">The covalent structure of an entire γG immunoglobin molecule</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1969</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">78</span>– <span class="NLM_lpage">85</span>, <span class="refDoi"> DOI: 10.1073/pnas.63.1.78</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1073%2Fpnas.63.1.78" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=5257969" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADyaE3cXivFyjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=1969&pages=78-85&author=G.+M.+Edelmanauthor=B.+A.+Cunninghamauthor=W.+E.+Gallauthor=P.+D.+Gotliebauthor=U.+Rutishauserauthor=M.+J.+Waxdal&title=The+covalent+structure+of+an+entire+%CE%B3G+immunoglobin+molecule&doi=10.1073%2Fpnas.63.1.78"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent structure of an entire γG immunoglobulin molecule</span></div><div class="casAuthors">Edelman, Gerald M.; Cunningham, Bruce A.; Gall, W. Einar; Gottlieb, Paul D.; Rutishauser, Urs; Waxdal, Myron J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1969</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">78-85</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">The complete amino acid sequence of a human γGl immunoglobulin (Eu) has been detd. and the arrangement of all of the disulfide bonds has been established.  Comparison of the sequence with that of another myeloma protein (He) suggests that the variable regions of heavy and light chains are homologous and similar in length.  The const. portion of the heavy chain contains 3 homology regions each of which is similar in size and homologous to the const. region of the light chain.  Each variable region and each const. homology region contains 1 intrachain disulfide bond.  The half-cystines participating in the interchain bonds are all clustered within a stretch of 10 residues at the middle of the heavy chains.  These data support the hypothesis that immunoglobulins evolved by gene duplication after early divergence of V genes, which specified antigen-binding functions, and C genes, which specified other functions of antibody mols.  Each polypeptide chain may therefore be specified by 2 genes, V and C, which are fused to form a single gene (translocation hypothesis).  The internal homologies and symmetry of the mol. suggest that homology regions may have similar 3-dimensional structures each consisting of a compact domain which contributes to at least 1 active site (domain hypothesis).  Both hypotheses are in accord with the linear regional differentiation of function in antibody mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjqW9i9JX9g7Vg90H21EOLACvtfcHk0ljn8MTrfhQ_eQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE3cXivFyjtQ%253D%253D&md5=18ba9b6417ce609be8870e1d142e7fc9</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1073%2Fpnas.63.1.78&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.63.1.78%26sid%3Dliteratum%253Aachs%26aulast%3DEdelman%26aufirst%3DG.%2BM.%26aulast%3DCunningham%26aufirst%3DB.%2BA.%26aulast%3DGall%26aufirst%3DW.%2BE.%26aulast%3DGotlieb%26aufirst%3DP.%2BD.%26aulast%3DRutishauser%26aufirst%3DU.%26aulast%3DWaxdal%26aufirst%3DM.%2BJ.%26atitle%3DThe%2520covalent%2520structure%2520of%2520an%2520entire%2520%25CE%25B3G%2520immunoglobin%2520molecule%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1969%26volume%3D63%26spage%3D78%26epage%3D85%26doi%3D10.1073%2Fpnas.63.1.78" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="note"><p class="first last">For a comparison of the EU and the alternate Kabat mAb nomenclature systems, see Table S1 in ref <a onclick="showRef(event, 'cit26a'); return false;" href="javascript:void(0);" class="ref cit26a">(26a)</a>.</p></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sadowsky, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pillow, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">dela Cruz-Chuh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Donohue, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Rosario, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, G. D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozak, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leipold, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wai, J.</span></span> <span> </span><span class="NLM_article-title">Development of efficient chemistry to generate site-specific disulfide-linked protein- and peptide-payload conjugates: Application to THIOMAB antibody-drug conjugates</span>. <i>Bioconjugate Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">2086</span>– <span class="NLM_lpage">2098</span>, <span class="refDoi"> DOI: 10.1021/acs.bioconjchem.7b00258</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.bioconjchem.7b00258" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVaktLvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2017&pages=2086-2098&author=J.+D.+Sadowskyauthor=T.+H.+Pillowauthor=J.+Chenauthor=F.+Fanauthor=C.+Heauthor=Y.+Wangauthor=G.+Yanauthor=H.+Yaoauthor=Z.+Xuauthor=S.+Martinauthor=D.+Zhangauthor=P.+Chuauthor=J.+dela+Cruz-Chuhauthor=A.+O%E2%80%99Donohueauthor=G.+Liauthor=G.+Del+Rosarioauthor=J.+Heauthor=L.+Liuauthor=C.+Ngauthor=D.+Suauthor=G.+D.+L.+Phillipsauthor=K.+R.+Kozakauthor=S.-F.+Yuauthor=K.+Xuauthor=D.+Leipoldauthor=J.+Wai&title=Development+of+efficient+chemistry+to+generate+site-specific+disulfide-linked+protein-+and+peptide-payload+conjugates%3A+Application+to+THIOMAB+antibody-drug+conjugates&doi=10.1021%2Facs.bioconjchem.7b00258"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Efficient Chemistry to Generate Site-Specific Disulfide-Linked Protein- and Peptide-Payload Conjugates: Application to THIOMAB Antibody-Drug Conjugates</span></div><div class="casAuthors">Sadowsky, Jack D.; Pillow, Thomas H.; Chen, Jinhua; Fan, Fang; He, Changrong; Wang, Yanli; Yan, Gang; Yao, Hui; Xu, Zijin; Martin, Shanique; Zhang, Donglu; Chu, Phillip; dela Cruz-Chuh, Josefa; ODonohue, Aimee; Li, Guangmin; Del Rosario, Geoffrey; He, Jintang; Liu, Luna; Ng, Carl; Su, Dian; Lewis Phillips, Gail D.; Kozak, Katherine R.; Yu, Shang-Fan; Xu, Keyang; Leipold, Douglas; Wai, John</div><div class="citationInfo"><span class="NLM_cas:title">Bioconjugate Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2086-2098</span>CODEN:
                <span class="NLM_cas:coden">BCCHES</span>;
        ISSN:<span class="NLM_cas:issn">1043-1802</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Conjugation of small mol. payloads to specific cysteine residues on proteins via a disulfide bond represents an attractive strategy to generate redox-sensitive bioconjugates, which have value as potential diagnostic reagents or therapeutics.  Advancement of such "direct-disulfide" bioconjugates to the clinic necessitates chem. methods to form disulfide connections efficiently, without byproducts.  The disulfide connection must also be resistant to premature cleavage by thiols prior to arrival at the targeted tissue.  We show here that commonly-employed methods to generate direct disulfide-linked bioconjugates are inadequate for addressing these challenges.  We describe our efforts to optimize direct-disulfide conjugation chem., focusing on the generation of conjugates between cytotoxic payloads and cysteine-engineered antibodies (i.e., THIOMAB antibody-drug conjugates, or TDCs).  This work culminates in the development of novel, high-yielding conjugation chem. for creating direct payload disulfide connections to any of several Cys mutation sites in THIOMAB antibodies or to Cys sites in other biomols. (e.g., human serum albumin and cell-penetrating peptides).  We conclude by demonstrating that hindered direct disulfide TDCs with two Me groups adjacent to the disulfide, which have heretofore not been described for any bioconjugate, are more stable and more efficacious in mouse tumor xenograft studies than less hindered analogs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIap6CQnm0X7Vg90H21EOLACvtfcHk0ljn8MTrfhQ_eQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVaktLvL&md5=9c437d3e1aa16e51e3f571fea95785c4</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.bioconjchem.7b00258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.bioconjchem.7b00258%26sid%3Dliteratum%253Aachs%26aulast%3DSadowsky%26aufirst%3DJ.%2BD.%26aulast%3DPillow%26aufirst%3DT.%2BH.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DFan%26aufirst%3DF.%26aulast%3DHe%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DYan%26aufirst%3DG.%26aulast%3DYao%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DChu%26aufirst%3DP.%26aulast%3Ddela%2BCruz-Chuh%26aufirst%3DJ.%26aulast%3DO%25E2%2580%2599Donohue%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DDel%2BRosario%26aufirst%3DG.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DNg%26aufirst%3DC.%26aulast%3DSu%26aufirst%3DD.%26aulast%3DPhillips%26aufirst%3DG.%2BD.%2BL.%26aulast%3DKozak%26aufirst%3DK.%2BR.%26aulast%3DYu%26aufirst%3DS.-F.%26aulast%3DXu%26aufirst%3DK.%26aulast%3DLeipold%26aufirst%3DD.%26aulast%3DWai%26aufirst%3DJ.%26atitle%3DDevelopment%2520of%2520efficient%2520chemistry%2520to%2520generate%2520site-specific%2520disulfide-linked%2520protein-%2520and%2520peptide-payload%2520conjugates%253A%2520Application%2520to%2520THIOMAB%2520antibody-drug%2520conjugates%26jtitle%3DBioconjugate%2520Chem.%26date%3D2017%26volume%3D28%26spage%3D2086%26epage%3D2098%26doi%3D10.1021%2Facs.bioconjchem.7b00258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Ohri, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhakta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Donohue, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bos, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farahi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhakta, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pillow, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raab, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polakis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erickson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Junutula, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozak, K. R.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">High-throughput cysteine scanning to identify stable antibody conjugation sites for maleimide- and disulfide-based linkers</span>. <i>Bioconjugate Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">473</span>– <span class="NLM_lpage">485</span>, <span class="refDoi"> DOI: 10.1021/acs.bioconjchem.7b00791</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.bioconjchem.7b00791" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVKku7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2018&pages=473-485&author=R.+Ohriauthor=S.+Bhaktaauthor=A.+O%E2%80%99Donohueauthor=J.+Chuhauthor=S.+P.+Tsaiauthor=R.+Cookauthor=B.+Weiauthor=C.+Ngauthor=A.+W.+Wongauthor=A.+B.+Bosauthor=F.+Farahiauthor=J.+Bhaktaauthor=T.+H.+Pillowauthor=H.+Raabauthor=P.+Polakisauthor=Y.+Liuauthor=H.+Ericksonauthor=J.+R.+Junutulaauthor=K.+R.+Kozak&title=High-throughput+cysteine+scanning+to+identify+stable+antibody+conjugation+sites+for+maleimide-+and+disulfide-based+linkers&doi=10.1021%2Facs.bioconjchem.7b00791"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26aR"><div class="casContent"><span class="casTitleNuber">26a</span><div class="casTitle"><span class="NLM_cas:atitle">High-Throughput Cysteine Scanning To Identify Stable Antibody Conjugation Sites for Maleimide- and Disulfide-Based Linkers</span></div><div class="casAuthors">Ohri, Rachana; Bhakta, Sunil; Fourie-O'Donohue, Aimee; dela Cruz-Chuh, Josefa; Tsai, Siao Ping; Cook, Ryan; Wei, Binqing; Ng, Carl; Wong, Athena W.; Bos, Aaron B.; Farahi, Farzam; Bhakta, Jiten; Pillow, Thomas H.; Raab, Helga; Vandlen, Richard; Polakis, Paul; Liu, Yichin; Erickson, Hans; Junutula, Jagath R.; Kozak, Katherine R.</div><div class="citationInfo"><span class="NLM_cas:title">Bioconjugate Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">473-485</span>CODEN:
                <span class="NLM_cas:coden">BCCHES</span>;
        ISSN:<span class="NLM_cas:issn">1043-1802</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">THIOMAB antibody technol. utilizes cysteine residues engineered onto an antibody to allow for site-specific conjugation.  The technol. has enabled the exploration of different attachment sites on the antibody in combination with small mols., peptides, or proteins to yield antibody conjugates with unique properties.  As reported previously, the specific location of the site of conjugation on an antibody can impact the stability of the linkage to the engineered cysteine for both thio-succinimide and disulfide bonds.  High stability of the linkage is usually desired to maximize the delivery of the cargo to the intended target.  In the current study, cysteines were individually substituted into every position of the anti-HER2 antibody (trastuzumab), and the stabilities of drug conjugations at those sites were evaluated.  We screened a total of 648 THIOMAB antibody-drug conjugates, each generated from a trastuzamab prepd. by sequentially mutating non-cysteine amino acids in the light and heavy chains to cysteine.  Each THIOMAB antibody variant was conjugated to either maleimidocaproyl-valine-citrulline-p-aminobenzyloxycarbonyl-monomethyl auristatin E (MC-vc-PAB-MMAE) or pyridyl disulfide monomethyl auristatin E (PDS-MMAE) using a high-throughput, on-bead conjugation and purifn. method.  Greater than 50% of the THIOMAB antibody variants were successfully conjugated to both MMAE derivs. with a drug to antibody ratio (DAR) of >0.5 and <50% aggregation.  The relative in vitro plasma stabilities for approx. 750 conjugates were assessed using enzyme-linked immunosorbent assays, and stable sites were confirmed with affinity-capture LC/MS-based detection methods.  Highly stable conjugation sites for the two types of MMAE derivs. were identified on both the heavy and light chains.  Although the stabilities of maleimide conjugates were shown to be greater than those of the disulfide conjugates, many sites were identified that were stable for both.  Furthermore, in vitro stabilities of selected stable sites translated across different cytotoxic payloads and different target antibodies as well as to in vivo stability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLlgFVNRmHMbVg90H21EOLACvtfcHk0liK-Gy_gp_C1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVKku7g%253D&md5=318146c1baea8094b671e99b5c429627</span></div><a href="/servlet/linkout?suffix=cit26a&amp;dbid=16384&amp;doi=10.1021%2Facs.bioconjchem.7b00791&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.bioconjchem.7b00791%26sid%3Dliteratum%253Aachs%26aulast%3DOhri%26aufirst%3DR.%26aulast%3DBhakta%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Donohue%26aufirst%3DA.%26aulast%3DChuh%26aufirst%3DJ.%26aulast%3DTsai%26aufirst%3DS.%2BP.%26aulast%3DCook%26aufirst%3DR.%26aulast%3DWei%26aufirst%3DB.%26aulast%3DNg%26aufirst%3DC.%26aulast%3DWong%26aufirst%3DA.%2BW.%26aulast%3DBos%26aufirst%3DA.%2BB.%26aulast%3DFarahi%26aufirst%3DF.%26aulast%3DBhakta%26aufirst%3DJ.%26aulast%3DPillow%26aufirst%3DT.%2BH.%26aulast%3DRaab%26aufirst%3DH.%26aulast%3DPolakis%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DErickson%26aufirst%3DH.%26aulast%3DJunutula%26aufirst%3DJ.%2BR.%26aulast%3DKozak%26aufirst%3DK.%2BR.%26atitle%3DHigh-throughput%2520cysteine%2520scanning%2520to%2520identify%2520stable%2520antibody%2520conjugation%2520sites%2520for%2520maleimide-%2520and%2520disulfide-based%2520linkers%26jtitle%3DBioconjugate%2520Chem.%26date%3D2018%26volume%3D29%26spage%3D473%26epage%3D485%26doi%3D10.1021%2Facs.bioconjchem.7b00791" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit26b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Vollmar, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohri, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leipold, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cosino, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fourie-O’Donohue, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozak, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamath, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazar, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erickson, H. K.</span></span> <span> </span><span class="NLM_article-title">Attachment site cysteine thiol pKa is a key driver for site-dependent stability of THIOMAB antibody-drug conjugates</span>. <i>Bioconjugate Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">2538</span>– <span class="NLM_lpage">2548</span>, <span class="refDoi"> DOI: 10.1021/acs.bioconjchem.7b00365</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.bioconjchem.7b00365" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVGks7vL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2017&pages=2538-2548&author=B.+S.+Vollmarauthor=B.+Weiauthor=R.+Ohriauthor=J.+Zhouauthor=J.+Heauthor=S.-F.+Yuauthor=D.+Leipoldauthor=E.+Cosinoauthor=S.+Yeeauthor=A.+Fourie-O%E2%80%99Donohueauthor=G.+Liauthor=G.+L.+Phillipsauthor=K.+R.+Kozakauthor=A.+Kamathauthor=K.+Xuauthor=G.+Leeauthor=G.+A.+Lazarauthor=H.+K.+Erickson&title=Attachment+site+cysteine+thiol+pKa+is+a+key+driver+for+site-dependent+stability+of+THIOMAB+antibody-drug+conjugates&doi=10.1021%2Facs.bioconjchem.7b00365"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26bR"><div class="casContent"><span class="casTitleNuber">26b</span><div class="casTitle"><span class="NLM_cas:atitle">Attachment Site Cysteine Thiol pKa Is a Key Driver for Site-Dependent Stability of THIOMAB Antibody-Drug Conjugates</span></div><div class="casAuthors">Vollmar, Breanna S.; Wei, Binqing; Ohri, Rachana; Zhou, Jianhui; He, Jintang; Yu, Shang-Fan; Leipold, Douglas; Cosino, Ely; Yee, Sharon; Fourie-O'Donohue, Aimee; Li, Guangmin; Phillips, Gail L.; Kozak, Katherine R.; Kamath, Amrita; Xu, Keyang; Lee, Genee; Lazar, Greg A.; Erickson, Hans K.</div><div class="citationInfo"><span class="NLM_cas:title">Bioconjugate Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2538-2548</span>CODEN:
                <span class="NLM_cas:coden">BCCHES</span>;
        ISSN:<span class="NLM_cas:issn">1043-1802</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The incorporation of cysteines into antibodies by mutagenesis allows for the direct conjugation of small mols. to specific sites on the antibody via disulfide bonds.  The stability of the disulfide bond linkage between the small mol. and the antibody is highly dependent on the location of the engineered cysteine in either the heavy chain (HC) or the light chain (LC) of the antibody.  Here, we explore the basis for this site-dependent stability.  We evaluated the in vivo efficacy and pharmacokinetics of five different cysteine mutants of trastuzumab conjugated to a pyrrolobenzodiazepine (PBD) via disulfide bonds.  A significant correlation was obsd. between disulfide stability and efficacy for the conjugates.  We hypothesized that the obsd. site-dependent stability of the disulfide-linked conjugates could be due to differences in the attachment site cysteine thiol pKa.  We measured the cysteine thiol pKa using isothermal titrn. calorimetry (ITC) and found that the variants with the highest thiol pKa (LC K149C and HC A140C) were found to yield the conjugates with the greatest in vivo stability.  Guided by homol. modeling, we identified several mutations adjacent to LC K149C that reduced the cysteine thiol pKa and, thus, decreased the in vivo stability of the disulfide-linked PBD conjugated to LC K149C.  We also present results suggesting that the high thiol pKa of LC K149C is responsible for the sustained circulation stability of LC K149C TDCs utilizing a maleimide-based linker.  Taken together, our results provide evidence that the site-dependent stability of cys-engineered antibody-drug conjugates may be explained by interactions between the engineered cysteine and the local protein environment that serves to modulate the side-chain thiol pKa.  The influence of cysteine thiol pKa on stability and efficacy offers a new parameter for the optimization of ADCs that utilize cysteine engineering.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAfrN62jixCbVg90H21EOLACvtfcHk0liK-Gy_gp_C1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVGks7vL&md5=ba1805c493f468d4ffbb198dbf8b3d02</span></div><a href="/servlet/linkout?suffix=cit26b&amp;dbid=16384&amp;doi=10.1021%2Facs.bioconjchem.7b00365&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.bioconjchem.7b00365%26sid%3Dliteratum%253Aachs%26aulast%3DVollmar%26aufirst%3DB.%2BS.%26aulast%3DWei%26aufirst%3DB.%26aulast%3DOhri%26aufirst%3DR.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DS.-F.%26aulast%3DLeipold%26aufirst%3DD.%26aulast%3DCosino%26aufirst%3DE.%26aulast%3DYee%26aufirst%3DS.%26aulast%3DFourie-O%25E2%2580%2599Donohue%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DPhillips%26aufirst%3DG.%2BL.%26aulast%3DKozak%26aufirst%3DK.%2BR.%26aulast%3DKamath%26aufirst%3DA.%26aulast%3DXu%26aufirst%3DK.%26aulast%3DLee%26aufirst%3DG.%26aulast%3DLazar%26aufirst%3DG.%2BA.%26aulast%3DErickson%26aufirst%3DH.%2BK.%26atitle%3DAttachment%2520site%2520cysteine%2520thiol%2520pKa%2520is%2520a%2520key%2520driver%2520for%2520site-dependent%2520stability%2520of%2520THIOMAB%2520antibody-drug%2520conjugates%26jtitle%3DBioconjugate%2520Chem.%26date%3D2017%26volume%3D28%26spage%3D2538%26epage%3D2548%26doi%3D10.1021%2Facs.bioconjchem.7b00365" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blake, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">den Besten, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fourie, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartman, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingalla, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozak, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leong, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nannini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohri, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pillow, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowntree, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sampath, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandlen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vollmar, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wertz, I. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span> <span> </span><span class="NLM_article-title">Conjugation of indoles to antibodies through a novel self-immolating linker</span>. <i>Chem. – Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">4830</span>– <span class="NLM_lpage">4834</span>, <span class="refDoi"> DOI: 10.1002/chem.201800859</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1002%2Fchem.201800859" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=29493023" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1cXlslSnu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=4830-4834&author=P.+S.+Dragovichauthor=R.+A.+Blakeauthor=C.+Chenauthor=J.+Chenauthor=J.+Chuhauthor=W.+den+Bestenauthor=F.+Fanauthor=A.+Fourieauthor=S.+J.+Hartmanauthor=C.+Heauthor=J.+Heauthor=E.+R.+Ingallaauthor=K.+R.+Kozakauthor=S.+R.+Leongauthor=J.+Luauthor=Y.+Maauthor=L.+Mengauthor=M.+Nanniniauthor=J.+Oehauthor=R.+Ohriauthor=G.+L.+Phillipsauthor=T.+H.+Pillowauthor=R.+K.+Rowntreeauthor=D.+Sampathauthor=R.+Vandlenauthor=B.+Vollmarauthor=J.+Waiauthor=I.+E.+Wertzauthor=K.+Xuauthor=Z.+Xuauthor=D.+Zhang&title=Conjugation+of+indoles+to+antibodies+through+a+novel+self-immolating+linker&doi=10.1002%2Fchem.201800859"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Conjugation of Indoles to Antibodies through a Novel Self-Immolating Linker</span></div><div class="casAuthors">Dragovich, Peter S.; Blake, Robert A.; Chen, Chunjiao; Chen, Jinhua; Chuh, Josefa; den Besten, Willem; Fan, Fang; Fourie, Aimee; Hartman, Steven J.; He, Changrong; He, Jintang; Ingalla, Ellen Rei; Kozak, Katherine R.; Leong, Steven R.; Lu, Jiawei; Ma, Yong; Meng, Lingyao; Nannini, Michelle; Oeh, Jason; Ohri, Rachana; Lewis Phillips, Gail; Pillow, Thomas H.; Rowntree, Rebecca K.; Sampath, Deepak; Vandlen, Richard; Vollmar, Breanna; Wai, John; Wertz, Ingrid E.; Xu, Keyang; Xu, Zijin; Zhang, Donglu</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">4830-4834</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A novel strategy to attach indole-contg. payloads to antibodies through a carbamate moiety and a self-immolating, disulfide-based linker is described.  This new strategy was employed to connect a selective estrogen receptor down-regulator (SERD) to various antibodies in a site-selective manner.  The resulting conjugates displayed potent, antigen-dependent down-regulation of estrogen receptor levels in MCF7-neo/HER2 and MCF7-hB7H4 cells.  They also exhibited similar antigen-dependent modulation of the estrogen receptor in tumors when administered i.v. to mice bearing MCF7-neo/HER2 tumor xenografts.  The indole-carbamate moiety present in the new linker was stable in whole blood from various species and also exhibited good in vivo stability properties in mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKhcOeE6xSRLVg90H21EOLACvtfcHk0liK-Gy_gp_C1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXlslSnu78%253D&md5=f1c82d8a721646a690e735c3ed38b53a</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fchem.201800859&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.201800859%26sid%3Dliteratum%253Aachs%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26aulast%3DBlake%26aufirst%3DR.%2BA.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DChuh%26aufirst%3DJ.%26aulast%3Dden%2BBesten%26aufirst%3DW.%26aulast%3DFan%26aufirst%3DF.%26aulast%3DFourie%26aufirst%3DA.%26aulast%3DHartman%26aufirst%3DS.%2BJ.%26aulast%3DHe%26aufirst%3DC.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DIngalla%26aufirst%3DE.%2BR.%26aulast%3DKozak%26aufirst%3DK.%2BR.%26aulast%3DLeong%26aufirst%3DS.%2BR.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DMeng%26aufirst%3DL.%26aulast%3DNannini%26aufirst%3DM.%26aulast%3DOeh%26aufirst%3DJ.%26aulast%3DOhri%26aufirst%3DR.%26aulast%3DPhillips%26aufirst%3DG.%2BL.%26aulast%3DPillow%26aufirst%3DT.%2BH.%26aulast%3DRowntree%26aufirst%3DR.%2BK.%26aulast%3DSampath%26aufirst%3DD.%26aulast%3DVandlen%26aufirst%3DR.%26aulast%3DVollmar%26aufirst%3DB.%26aulast%3DWai%26aufirst%3DJ.%26aulast%3DWertz%26aufirst%3DI.%2BE.%26aulast%3DXu%26aufirst%3DK.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DD.%26atitle%3DConjugation%2520of%2520indoles%2520to%2520antibodies%2520through%2520a%2520novel%2520self-immolating%2520linker%26jtitle%3DChem.%2520%25E2%2580%2593%2520Eur.%2520J.%26date%3D2018%26volume%3D24%26spage%3D4830%26epage%3D4834%26doi%3D10.1002%2Fchem.201800859" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span> Others have used similar
SPR techniques to examine ternary complexes formed between BRD4, VHL,
and related chimeric degrader molecules:<span class="NLM_contrib-group"><span class="NLM_string-name">Roy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitworth, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galant, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farnaby, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumpel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">SPR-measured dissociation kinetics of PROTAC ternary complexes influence target degradation rate</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">361</span>– <span class="NLM_lpage">368</span>, <span class="refDoi"> DOI: 10.1021/acschembio.9b00092</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.9b00092" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVCisrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=361-368&author=M.+Royauthor=S.+Winklerauthor=S.+J.+Hughesauthor=C.+Whitworthauthor=M.+Galantauthor=W.+Farnabyauthor=K.+Rumpelauthor=A.+Ciulli&title=SPR-measured+dissociation+kinetics+of+PROTAC+ternary+complexes+influence+target+degradation+rate&doi=10.1021%2Facschembio.9b00092"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">SPR-Measured Dissociation Kinetics of PROTAC Ternary Complexes Influence Target Degradation Rate</span></div><div class="casAuthors">Roy, Michael J.; Winkler, Sandra; Hughes, Scott J.; Whitworth, Claire; Galant, Michael; Farnaby, William; Rumpel, Klaus; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">361-368</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bifunctional degrader mols., known as proteolysis-targeting chimeras (PROTACs), function by recruiting a target to an E3 ligase, forming a target/PROTAC/ligase ternary complex.  Despite the importance of this key intermediate species, no detailed validation of a method to directly det. binding parameters for ternary complex kinetics has been reported, and it remains to be addressed whether tuning the kinetics of PROTAC ternary complexes may be an effective strategy to improve the efficiency of targeted protein degrdn.  Here, we develop an SPR-based assay to quantify the stability of PROTAC-induced ternary complexes by measuring for the first time the kinetics of their formation and dissocn. in vitro using purified proteins.  We benchmark our assay using four PROTACs that target the bromodomains (BDs) of bromodomain and extraterminal domain proteins Brd2, Brd3, and Brd4 to the von Hippel-Lindau E3 ligase (VHL).  We reveal marked differences in ternary complex off-rates for different PROTACs that exhibit either pos. or neg. cooperativity for ternary complex formation relative to binary binding.  The pos. cooperative degrader MZ1 forms comparatively stable and long-lived ternary complexes with either Brd4BD2 or Brd2BD2 and VHL.  Equivalent complexes with Brd3BD2 are destabilized due to a single amino acid difference (Glu/Gly swap) present in the bromodomain.  We observe that this difference in ternary complex dissociative half-life correlates to a greater initial rate of intracellular degrdn. of Brd2 and Brd4 relative to Brd3.  These findings establish a novel assay to measure the kinetics of PROTAC ternary complexes and elucidate the important kinetic parameters that drive effective target degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzFhvuKYztcbVg90H21EOLACvtfcHk0li854tqrnBP7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVCisrg%253D&md5=629d1acbd218b668a2696b727946962b</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facschembio.9b00092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.9b00092%26sid%3Dliteratum%253Aachs%26aulast%3DRoy%26aufirst%3DM.%26aulast%3DWinkler%26aufirst%3DS.%26aulast%3DHughes%26aufirst%3DS.%2BJ.%26aulast%3DWhitworth%26aufirst%3DC.%26aulast%3DGalant%26aufirst%3DM.%26aulast%3DFarnaby%26aufirst%3DW.%26aulast%3DRumpel%26aufirst%3DK.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DSPR-measured%2520dissociation%2520kinetics%2520of%2520PROTAC%2520ternary%2520complexes%2520influence%2520target%2520degradation%2520rate%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2019%26volume%3D14%26spage%3D361%26epage%3D368%26doi%3D10.1021%2Facschembio.9b00092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of a macrocyclic PROTAC</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1727</span>– <span class="NLM_lpage">1734</span>, <span class="refDoi"> DOI: 10.1002/anie.201914396</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1002%2Fanie.201914396" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlyitbjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2020&pages=1727-1734&author=A.+Testaauthor=S.+J.+Hughesauthor=X.+Lucasauthor=J.+E.+Wrightauthor=A.+Ciulli&title=Structure-based+design+of+a+macrocyclic+PROTAC&doi=10.1002%2Fanie.201914396"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29aR"><div class="casContent"><span class="casTitleNuber">29a</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of a Macrocyclic PROTAC</span></div><div class="casAuthors">Testa, Andrea; Hughes, Scott J.; Lucas, Xavier; Wright, Jane E.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1727-1734</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Constraining a mol. in its bioactive conformation via macrocyclization represents an attractive strategy to rationally design functional chem. probes.  While this approach has been applied to enzyme inhibitors or receptor antagonists, to date it remains unprecedented for bifunctional mols. that bring proteins together, such as PROTAC degraders.  Herein, the authors report the design and synthesis of a macrocyclic PROTAC by adding a cyclizing linker to the BET degrader MZ1.  A cocrystal structure of macroPROTAC-1 bound in a ternary complex with VHL and the second bromodomain of Brd4 validated the rational design.  Biophys. studies revealed enhanced discrimination between the second and the first bromodomains of BET proteins.  Despite a 12-fold loss of binary binding affinity for Brd4, macroPROTAC-1 exhibited cellular activity comparable to MZ1.  The findings support macrocyclization as an advantageous strategy to enhance PROTAC degrdn. potency and selectivity between homologous targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZjLgy_V3_abVg90H21EOLACvtfcHk0li854tqrnBP7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlyitbjK&md5=6f3f279f6220ec30b30ca3d2c931da95</span></div><a href="/servlet/linkout?suffix=cit29a&amp;dbid=16384&amp;doi=10.1002%2Fanie.201914396&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201914396%26sid%3Dliteratum%253Aachs%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DHughes%26aufirst%3DS.%2BJ.%26aulast%3DLucas%26aufirst%3DX.%26aulast%3DWright%26aufirst%3DJ.%2BE.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructure-based%2520design%2520of%2520a%2520macrocyclic%2520PROTAC%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2020%26volume%3D59%26spage%3D1727%26epage%3D1734%26doi%3D10.1002%2Fanie.201914396" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit29b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamont, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Structural basis of PROTAC cooperative recognition for selective protein degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">514</span>– <span class="NLM_lpage">521</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2329</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1038%2Fnchembio.2329" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=28288108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=514-521&author=M.+S.+Gaddauthor=A.+Testaauthor=X.+Lucasauthor=K.-H.+Chanauthor=W.+Chenauthor=D.+J.+Lamontauthor=M.+Zengerleauthor=A.+Ciulli&title=Structural+basis+of+PROTAC+cooperative+recognition+for+selective+protein+degradation&doi=10.1038%2Fnchembio.2329"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29bR"><div class="casContent"><span class="casTitleNuber">29b</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of PROTAC cooperative recognition for selective protein degradation</span></div><div class="casAuthors">Gadd, Morgan S.; Testa, Andrea; Lucas, Xavier; Chan, Kwok-Ho; Chen, Wenzhang; Lamont, Douglas J.; Zengerle, Michael; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">514-521</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Inducing macromol. interactions with small mols. to activate cellular signaling is a challenging goal.  PROTACs (proteolysis-targeting chimeras) are bifunctional mols. that recruit a target protein in proximity to an E3 ubiquitin ligase to trigger protein degrdn.  Structural elucidation of the key ternary ligase-PROTAC-target species and its impact on target degrdn. selectivity remain elusive.  Here, we solved the crystal structure of Brd4 degrader MZ1 in complex with human VHL and the Brd4 bromodomain (Brd4BD2).  The ligand folds into itself to allow formation of specific intermol. interactions in the ternary complex.  Isothermal titrn. calorimetry studies, supported by surface mutagenesis and proximity assays, were consistent with pronounced cooperative formation of ternary complexes with Brd4BD2.  Structure-based-designed compd. AT1 exhibited highly selective depletion of Brd4 in cells.  These results elucidate how PROTAC-induced de novo contacts dictate preferential recruitment of a target protein into a stable and cooperative complex with an E3 ligase for selective degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJri00jB5YJLVg90H21EOLACvtfcHk0li854tqrnBP7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D&md5=dfee13913106d362d0972d7b1a62d700</span></div><a href="/servlet/linkout?suffix=cit29b&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2329&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2329%26sid%3Dliteratum%253Aachs%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DLucas%26aufirst%3DX.%26aulast%3DChan%26aufirst%3DK.-H.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DLamont%26aufirst%3DD.%2BJ.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructural%2520basis%2520of%2520PROTAC%2520cooperative%2520recognition%2520for%2520selective%2520protein%2520degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D514%26epage%3D521%26doi%3D10.1038%2Fnchembio.2329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Brandish, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmieri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antonenko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beaumont, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benso, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cancilla, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fayadat-Dilman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Figueroa, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Firdos, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garbaccio, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garvin-Queen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gately, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geda, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haines, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hseih, S.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodges, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kern, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knudsen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwasnjuk, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manibusan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moy, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stivers, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomazela, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woo, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaller, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zielstorff, M.</span></span> <span> </span><span class="NLM_article-title">Development of anti-CD74 antibody-drug conjugates to target glucocorticoids to immune cells</span>. <i>Bioconjugate Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">2357</span>– <span class="NLM_lpage">2369</span>, <span class="refDoi"> DOI: 10.1021/acs.bioconjchem.8b00312</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.bioconjchem.8b00312" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFGlsb7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2018&pages=2357-2369&author=P.+E.+Brandishauthor=A.+Palmieriauthor=S.+Antonenkoauthor=M.+Beaumontauthor=L.+Bensoauthor=M.+Cancillaauthor=M.+Chengauthor=L.+Fayadat-Dilmanauthor=G.+Fengauthor=I.+Figueroaauthor=J.+Firdosauthor=R.+Garbaccioauthor=L.+Garvin-Queenauthor=D.+Gatelyauthor=P.+Gedaauthor=C.+Hainesauthor=S.-C.+Hseihauthor=D.+Hodgesauthor=J.+Kernauthor=N.+Knudsenauthor=K.+Kwasnjukauthor=L.+Liangauthor=H.+Maauthor=A.+Manibusanauthor=P.+L.+Millerauthor=L.+Y.+Moyauthor=Y.+Quauthor=S.+Shahauthor=J.+S.+Shinauthor=P.+Stiversauthor=Y.+Sunauthor=D.+Tomazelaauthor=H.+C.+Wooauthor=D.+Zallerauthor=S.+Zhangauthor=Y.+Zhangauthor=M.+Zielstorff&title=Development+of+anti-CD74+antibody-drug+conjugates+to+target+glucocorticoids+to+immune+cells&doi=10.1021%2Facs.bioconjchem.8b00312"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30aR"><div class="casContent"><span class="casTitleNuber">30a</span><div class="casTitle"><span class="NLM_cas:atitle">Development of anti-CD74 antibody-drug conjugates to target glucocorticoids to immune cells</span></div><div class="casAuthors">Brandish, Philip E.; Palmieri, Anthony; Antonenko, Svetlana; Beaumont, Maribel; Benso, Lia; Cancilla, Mark; Cheng, Mangeng; Fayadat-Dilman, Laurence; Feng, Guo; Figueroa, Isabel; Firdos, Juhi; Garbaccio, Robert; Garvin-Queen, Laura; Gately, Dennis; Geda, Prasanthi; Haines, Christopher; Hseih, SuChun; Hodges, Douglas; Kern, Jeffrey; Knudsen, Nickolas; Kwasnjuk, Kristen; Liang, Linda; Ma, Huiping; Manibusan, Anthony; Miller, Paul L.; Moy, Lily Y.; Qu, Yujie; Shah, Sanjiv; Shin, John S.; Stivers, Peter; Sun, Ying; Tomazela, Daniela; Woo, Hyun Chong; Zaller, Dennis; Zhang, Shuli; Zhang, Yiwei; Zielstorff, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Bioconjugate Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2357-2369</span>CODEN:
                <span class="NLM_cas:coden">BCCHES</span>;
        ISSN:<span class="NLM_cas:issn">1043-1802</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Glucocorticoids (GCs) are excellent anti-inflammatory drugs but are dose-limited by on-target toxicity.  We sought to solve this problem by delivering GCs to immune cells with antibody-drug conjugates (ADCs) using antibodies contg. site-specific incorporation of a non-natural amino acid, novel linker chem. for in vitro and in vivo stability, and existing and novel glucocorticoid receptor (GR) agonists as payloads.  We directed fluticasone propionate to human antigen-presenting immune cells to afford GR activation that was dependent on the targeted antigen.  However, mechanism of action studies pointed to accumulation of free payload in the tissue culture supernatant as the dominant driver of activity and indeed administration of the ADC to human CD74 transgenic mice failed to activate GR target genes in splenic B cells.  Suspecting dissipation of released payload, we designed an ADC bearing a novel GR agonist payload with reduced permeability which afforded cell-intrinsic activity in human B cells.  Our work shows that antibody-targeting offers significant potential for rescuing existing and new dose-limited drugs outside the field of oncol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqL223IFiibVrVg90H21EOLACvtfcHk0li_oyEZ6xCvuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFGlsb7P&md5=16f03a31902538a233676f30cc1c26c7</span></div><a href="/servlet/linkout?suffix=cit30a&amp;dbid=16384&amp;doi=10.1021%2Facs.bioconjchem.8b00312&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.bioconjchem.8b00312%26sid%3Dliteratum%253Aachs%26aulast%3DBrandish%26aufirst%3DP.%2BE.%26aulast%3DPalmieri%26aufirst%3DA.%26aulast%3DAntonenko%26aufirst%3DS.%26aulast%3DBeaumont%26aufirst%3DM.%26aulast%3DBenso%26aufirst%3DL.%26aulast%3DCancilla%26aufirst%3DM.%26aulast%3DCheng%26aufirst%3DM.%26aulast%3DFayadat-Dilman%26aufirst%3DL.%26aulast%3DFeng%26aufirst%3DG.%26aulast%3DFigueroa%26aufirst%3DI.%26aulast%3DFirdos%26aufirst%3DJ.%26aulast%3DGarbaccio%26aufirst%3DR.%26aulast%3DGarvin-Queen%26aufirst%3DL.%26aulast%3DGately%26aufirst%3DD.%26aulast%3DGeda%26aufirst%3DP.%26aulast%3DHaines%26aufirst%3DC.%26aulast%3DHseih%26aufirst%3DS.-C.%26aulast%3DHodges%26aufirst%3DD.%26aulast%3DKern%26aufirst%3DJ.%26aulast%3DKnudsen%26aufirst%3DN.%26aulast%3DKwasnjuk%26aufirst%3DK.%26aulast%3DLiang%26aufirst%3DL.%26aulast%3DMa%26aufirst%3DH.%26aulast%3DManibusan%26aufirst%3DA.%26aulast%3DMiller%26aufirst%3DP.%2BL.%26aulast%3DMoy%26aufirst%3DL.%2BY.%26aulast%3DQu%26aufirst%3DY.%26aulast%3DShah%26aufirst%3DS.%26aulast%3DShin%26aufirst%3DJ.%2BS.%26aulast%3DStivers%26aufirst%3DP.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DTomazela%26aufirst%3DD.%26aulast%3DWoo%26aufirst%3DH.%2BC.%26aulast%3DZaller%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DZielstorff%26aufirst%3DM.%26atitle%3DDevelopment%2520of%2520anti-CD74%2520antibody-drug%2520conjugates%2520to%2520target%2520glucocorticoids%2520to%2520immune%2520cells%26jtitle%3DBioconjugate%2520Chem.%26date%3D2018%26volume%3D29%26spage%3D2357%26epage%3D2369%26doi%3D10.1021%2Facs.bioconjchem.8b00312" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit30b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Kern, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cancilla, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dooney, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwasnjuk, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beaumont, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Figueroa, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsieh, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomazela, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandish, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmieri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stivers, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geda, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bresson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Firdos, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gately, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knudsen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manibusan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garbaccio, R. M.</span></span> <span> </span><span class="NLM_article-title">Discovery of pyrophosphate diesters as tunable, soluble, and bioorthogonal linkers for site-specific antibody-drug conjugates</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">1430</span>– <span class="NLM_lpage">1445</span>, <span class="refDoi"> DOI: 10.1021/jacs.5b12547</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.5b12547" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC28XltFCgsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2016&pages=1430-1445&author=J.+C.+Kernauthor=M.+Cancillaauthor=D.+Dooneyauthor=K.+Kwasnjukauthor=R.+Zhangauthor=M.+Beaumontauthor=I.+Figueroaauthor=S.+C.+Hsiehauthor=L.+Liangauthor=D.+Tomazelaauthor=J.+Zhangauthor=P.+E.+Brandishauthor=A.+Palmieriauthor=P.+Stiversauthor=M.+Chengauthor=G.+Fengauthor=P.+Gedaauthor=S.+Shahauthor=A.+Beckauthor=D.+Bressonauthor=J.+Firdosauthor=D.+Gatelyauthor=N.+Knudsenauthor=A.+Manibusanauthor=P.+G.+Schultzauthor=Y.+Sunauthor=R.+M.+Garbaccio&title=Discovery+of+pyrophosphate+diesters+as+tunable%2C+soluble%2C+and+bioorthogonal+linkers+for+site-specific+antibody-drug+conjugates&doi=10.1021%2Fjacs.5b12547"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30bR"><div class="casContent"><span class="casTitleNuber">30b</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Pyrophosphate Diesters as Tunable, Soluble, and Bioorthogonal Linkers for Site-Specific Antibody-Drug Conjugates</span></div><div class="casAuthors">Kern, Jeffrey C.; Cancilla, Mark; Dooney, Deborah; Kwasnjuk, Kristen; Zhang, Rena; Beaumont, Maribel; Figueroa, Isabel; Hsieh, SuChun; Liang, Linda; Tomazela, Daniela; Zhang, Jeffrey; Brandish, Philip E.; Palmieri, Anthony; Stivers, Peter; Cheng, Mangeng; Feng, Guo; Geda, Prasanthi; Shah, Sanjiv; Beck, Andrew; Bresson, Damien; Firdos, Juhi; Gately, Dennis; Knudsen, Nick; Manibusan, Anthony; Schultz, Peter G.; Sun, Ying; Garbaccio, Robert M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1430-1445</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">As part of an effort to examine the utility of antibody-drug conjugates (ADCs) beyond oncol. indications, a novel pyrophosphate ester linker was discovered to enable the targeted delivery of glucocorticoids.  As small mols., these highly sol. phosphate ester drug linkers were found to have ideal orthogonal properties: robust plasma stability coupled with rapid release of payload in a lysosomal environment.  Building upon these findings, site-specific ADCs were made between this drug linker combination and an antibody against human CD70, a receptor specifically expressed in immune cells but also found aberrantly expressed in multiple human carcinomas.  Full characterization of these ADCs enabled procession to in vitro proof of concept, wherein ADCs 1-22 and 1-37 were demonstrated to afford potent, targeted delivery of glucocorticoids to a representative cell line, as measured by changes in glucocorticoid receptor-mediated gene mRNA levels.  These activities were found to be antibody-, linker-, and payload-dependent.  Preliminary mechanistic studies support the notion that lysosomal trafficking and enzymic linker cleavage are required for activity and that the utility for the pyrophosphate linker may be general for internalizing ADCs as well as other targeted delivery platforms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0or81gNuulrVg90H21EOLACvtfcHk0li_oyEZ6xCvuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XltFCgsw%253D%253D&md5=07aa87ec657d4506e63b35f73af61e78</span></div><a href="/servlet/linkout?suffix=cit30b&amp;dbid=16384&amp;doi=10.1021%2Fjacs.5b12547&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.5b12547%26sid%3Dliteratum%253Aachs%26aulast%3DKern%26aufirst%3DJ.%2BC.%26aulast%3DCancilla%26aufirst%3DM.%26aulast%3DDooney%26aufirst%3DD.%26aulast%3DKwasnjuk%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DBeaumont%26aufirst%3DM.%26aulast%3DFigueroa%26aufirst%3DI.%26aulast%3DHsieh%26aufirst%3DS.%2BC.%26aulast%3DLiang%26aufirst%3DL.%26aulast%3DTomazela%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DBrandish%26aufirst%3DP.%2BE.%26aulast%3DPalmieri%26aufirst%3DA.%26aulast%3DStivers%26aufirst%3DP.%26aulast%3DCheng%26aufirst%3DM.%26aulast%3DFeng%26aufirst%3DG.%26aulast%3DGeda%26aufirst%3DP.%26aulast%3DShah%26aufirst%3DS.%26aulast%3DBeck%26aufirst%3DA.%26aulast%3DBresson%26aufirst%3DD.%26aulast%3DFirdos%26aufirst%3DJ.%26aulast%3DGately%26aufirst%3DD.%26aulast%3DKnudsen%26aufirst%3DN.%26aulast%3DManibusan%26aufirst%3DA.%26aulast%3DSchultz%26aufirst%3DP.%2BG.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DGarbaccio%26aufirst%3DR.%2BM.%26atitle%3DDiscovery%2520of%2520pyrophosphate%2520diesters%2520as%2520tunable%252C%2520soluble%252C%2520and%2520bioorthogonal%2520linkers%2520for%2520site-specific%2520antibody-drug%2520conjugates%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2016%26volume%3D138%26spage%3D1430%26epage%3D1445%26doi%3D10.1021%2Fjacs.5b12547" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fourie-O’Donohue, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, P. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">dela Cruz Chuh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tchelepi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawyer, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pillow, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadowsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozak, K. R.</span></span> <span> </span><span class="NLM_article-title">Improved translation of stability for conjugated antibodies using an in vitro whole blood assay</span>. <i>mAbs</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">1715705</span> <span class="refDoi"> DOI: 10.1080/19420862.2020.1715705</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1080%2F19420862.2020.1715705" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=31997712" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvFSrt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2020&author=A.+Fourie-O%E2%80%99Donohueauthor=P.+Y.+Chuauthor=J.+dela+Cruz+Chuhauthor=R.+Tchelepiauthor=S.+P.+Tsaiauthor=J.+C.+Tranauthor=W.+S.+Sawyerauthor=D.+Suauthor=C.+Ngauthor=K.+Xuauthor=S.-F.+Yuauthor=T.+H.+Pillowauthor=J.+Sadowskyauthor=P.+S.+Dragovichauthor=Y.+Liuauthor=K.+R.+Kozak&title=Improved+translation+of+stability+for+conjugated+antibodies+using+an+in+vitro+whole+blood+assay&doi=10.1080%2F19420862.2020.1715705"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Improved translation of stability for conjugated antibodies using an in vitro whole blood assay</span></div><div class="casAuthors">Fourie-O'Donohue, Aimee; Chu, Phillip Y.; dela Cruz Chuh, Josefa; Tchelepi, Robert; Tsai, Siao Ping; Tran, John C.; Sawyer, William S.; Su, Dian; Ng, Carl; Xu, Keyang; Yu, Shang-Fan; Pillow, Thomas H.; Sadowsky, Jack; Dragovich, Peter S.; Liu, Yichin; Kozak, Katherine R.</div><div class="citationInfo"><span class="NLM_cas:title">mAbs</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1715705/1</span>CODEN:
                <span class="NLM_cas:coden">MABSCP</span>;
        ISSN:<span class="NLM_cas:issn">1942-0870</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis, Inc.</span>)
        </div><div class="casAbstract">For antibody-drug conjugates to be efficacious and safe, they must be stable in circulation to carry the payload to the site of the targeted cell.  Several components of a drug-conjugated antibody are known to influence stability: 1) the site of drug attachment on the antibody, 2) the linker used to attach the payload to the antibody, and 3) the payload itself.  In order to support the design and optimization of a high vol. of drug conjugates and avoid unstable conjugates prior to testing in animal models, we wanted to proactively identify these potential liabilities.  Therefore, we sought to establish an in vitro screening method that best correlated with in vivo stability.  While traditionally plasma has been used to assess in vitro stability, our evaluation using a variety of THIOMABTM antibody-drug conjugates revealed several disconnects between the stability assessed in vitro and the in vivo outcomes when using plasma.  When drug conjugates were incubated in vitro for 24 h in mouse whole blood rather than plasma and then analyzed by affinity capture LC-MS, we found an improved correlation to in vivo stability with whole blood (R2 = 0.87, coeff. of detn.) compared to unfrozen or frozen mouse plasma (R2 = 0.34, 0.01, resp.).  We further showed that this whole blood assay was also able to predict in vivo stability of other preclin. species such as rat and cynomolgus monkey, as well as in human.  The screening method utilized short (24 h) incubation times, as well as a custom anal. software, allowing increased throughput and in-depth biotransformation characterization.  While some instabilities that were more challenging to identify remain, the method greatly enhanced the process of screening, optimizing, and lead candidate selection, resulting in the substantial redn. of animal studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhca0BIA00H7Vg90H21EOLACvtfcHk0lhsMfJZER01cQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvFSrt7g%253D&md5=c4b05b4fa7b3bf4d84955ec3e5e8dee5</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1080%2F19420862.2020.1715705&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F19420862.2020.1715705%26sid%3Dliteratum%253Aachs%26aulast%3DFourie-O%25E2%2580%2599Donohue%26aufirst%3DA.%26aulast%3DChu%26aufirst%3DP.%2BY.%26aulast%3Ddela%2BCruz%2BChuh%26aufirst%3DJ.%26aulast%3DTchelepi%26aufirst%3DR.%26aulast%3DTsai%26aufirst%3DS.%2BP.%26aulast%3DTran%26aufirst%3DJ.%2BC.%26aulast%3DSawyer%26aufirst%3DW.%2BS.%26aulast%3DSu%26aufirst%3DD.%26aulast%3DNg%26aufirst%3DC.%26aulast%3DXu%26aufirst%3DK.%26aulast%3DYu%26aufirst%3DS.-F.%26aulast%3DPillow%26aufirst%3DT.%2BH.%26aulast%3DSadowsky%26aufirst%3DJ.%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DKozak%26aufirst%3DK.%2BR.%26atitle%3DImproved%2520translation%2520of%2520stability%2520for%2520conjugated%2520antibodies%2520using%2520an%2520in%2520vitro%2520whole%2520blood%2520assay%26jtitle%3DmAbs%26date%3D2020%26volume%3D12%26doi%3D10.1080%2F19420862.2020.1715705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harbin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamman, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Differential PROTAC substrate specificity dictated by orientation of recruited E3 ligase</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>, <span class="NLM_elocation-id">131</span> <span class="refDoi"> DOI: 10.1038/s41467-018-08027-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1038%2Fs41467-018-08027-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=30631068" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A280%3ADC%252BB3cjgt1Sksg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&author=B.+E.+Smithauthor=S.+L.+Wangauthor=S.+Jaime-Figueroaauthor=A.+Harbinauthor=J.+Wangauthor=B.+D.+Hammanauthor=C.+M.+Crews&title=Differential+PROTAC+substrate+specificity+dictated+by+orientation+of+recruited+E3+ligase&doi=10.1038%2Fs41467-018-08027-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32aR"><div class="casContent"><span class="casTitleNuber">32a</span><div class="casTitle"><span class="NLM_cas:atitle">Differential PROTAC substrate specificity dictated by orientation of recruited E3 ligase</span></div><div class="casAuthors">Smith Blake E; Wang Stephen L; Jaime-Figueroa Saul; Crews Craig M; Harbin Alicia; Wang Jing; Hamman Brian D; Crews Craig M; Crews Craig M</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">131</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">PROteolysis-TArgeting Chimeras (PROTACs) are hetero-bifunctional molecules that recruit an E3 ubiquitin ligase to a given substrate protein resulting in its targeted degradation.  Many potent PROTACs with specificity for dissimilar targets have been developed; however, the factors governing degradation selectivity within closely-related protein families remain elusive.  Here, we generate isoform-selective PROTACs for the p38 MAPK family using a single warhead (foretinib) and recruited E3 ligase (von Hippel-Lindau).  Based on their distinct linker attachments and lengths, these two PROTACs differentially recruit VHL, resulting in degradation of p38α or p38δ.  We characterize the role of ternary complex formation in driving selectivity, showing that it is necessary, but insufficient, for PROTAC-induced substrate ubiquitination.  Lastly, we explore the p38δ:PROTAC:VHL complex to explain the different selectivity profiles of these PROTACs.  Our work attributes the selective degradation of two closely-related proteins using the same warhead and E3 ligase to heretofore underappreciated aspects of the ternary complex model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRYPr3dBOGbeuR_TJajKxJDfW6udTcc2ebYU6ohGbNp67ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cjgt1Sksg%253D%253D&md5=bd56c51f5966709ca5d1ff3dadb7cc79</span></div><a href="/servlet/linkout?suffix=cit32a&amp;dbid=16384&amp;doi=10.1038%2Fs41467-018-08027-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-018-08027-7%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DB.%2BE.%26aulast%3DWang%26aufirst%3DS.%2BL.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DHarbin%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DDifferential%2520PROTAC%2520substrate%2520specificity%2520dictated%2520by%2520orientation%2520of%2520recruited%2520E3%2520ligase%26jtitle%3DNat.%2520Commun.%26date%3D2019%26volume%3D10%26doi%3D10.1038%2Fs41467-018-08027-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit32b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Farnaby, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koegl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roy, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitworth, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diers, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trainor, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zollman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steurer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karolyi-Oezguer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riedmueller, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmaschitz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wachter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dank, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galant, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharps, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumpel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traxler, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstberger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnitzer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petermann, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greb, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinstabl, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoephel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss-Puxbaum, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ehrenhöfer-Wölfer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wöhrle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehmelt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinnenthal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnhof, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiechens, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, M.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen-Hughes, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ettmayer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConnell, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">BAF complex vulnerabilities in cancer demonstrated via structure-based PROTAC design</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">672</span>– <span class="NLM_lpage">680</span>, <span class="refDoi"> DOI: 10.1038/s41589-019-0294-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1038%2Fs41589-019-0294-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=31178587" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFeqtLfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=672-680&author=W.+Farnabyauthor=M.+Koeglauthor=M.+J.+Royauthor=C.+Whitworthauthor=E.+Diersauthor=N.+Trainorauthor=D.+Zollmanauthor=S.+Steurerauthor=J.+Karolyi-Oezguerauthor=C.+Riedmuellerauthor=T.+Gmaschitzauthor=J.+Wachterauthor=C.+Dankauthor=M.+Galantauthor=B.+Sharpsauthor=K.+Rumpelauthor=E.+Traxlerauthor=T.+Gerstbergerauthor=R.+Schnitzerauthor=O.+Petermannauthor=P.+Grebauthor=H.+Weinstablauthor=G.+Baderauthor=A.+Zoephelauthor=A.+Weiss-Puxbaumauthor=K.+Ehrenh%C3%B6fer-W%C3%B6lferauthor=S.+W%C3%B6hrleauthor=G.+Boehmeltauthor=J.+Rinnenthalauthor=H.+Arnhofauthor=N.+Wiechensauthor=M.-Y.+Wuauthor=T.+Owen-Hughesauthor=P.+Ettmayerauthor=M.+Pearsonauthor=D.+B.+McConnellauthor=A.+Ciulli&title=BAF+complex+vulnerabilities+in+cancer+demonstrated+via+structure-based+PROTAC+design&doi=10.1038%2Fs41589-019-0294-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32bR"><div class="casContent"><span class="casTitleNuber">32b</span><div class="casTitle"><span class="NLM_cas:atitle">BAF complex vulnerabilities in cancer demonstrated via structure-based PROTAC design</span></div><div class="casAuthors">Farnaby, William; Koegl, Manfred; Roy, Michael J.; Whitworth, Claire; Diers, Emelyne; Trainor, Nicole; Zollman, David; Steurer, Steffen; Karolyi-Oezguer, Jale; Riedmueller, Carina; Gmaschitz, Teresa; Wachter, Johannes; Dank, Christian; Galant, Michael; Sharps, Bernadette; Rumpel, Klaus; Traxler, Elisabeth; Gerstberger, Thomas; Schnitzer, Renate; Petermann, Oliver; Greb, Peter; Weinstabl, Harald; Bader, Gerd; Zoephel, Andreas; Weiss-Puxbaum, Alexander; Ehrenhofer-Wolfer, Katharina; Wohrle, Simon; Boehmelt, Guido; Rinnenthal, Joerg; Arnhof, Heribert; Wiechens, Nicola; Wu, Meng-Ying; Owen-Hughes, Tom; Ettmayer, Peter; Pearson, Mark; McConnell, Darryl B.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">672-680</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Targeting subunits of BAF/PBAF chromatin remodeling complexes has been proposed as an approach to exploit cancer vulnerabilities.  Here, we develop proteolysis targeting chimera (PROTAC) degraders of the BAF ATPase subunits SMARCA2 and SMARCA4 using a bromodomain ligand and recruitment of the E3 ubiquitin ligase VHL.  High-resoln. ternary complex crystal structures and biophys. investigation guided rational and efficient optimization toward ACBI1(I), a potent and cooperative degrader of SMARCA2, SMARCA4 and PBRM1.  I induced anti-proliferative effects and cell death caused by SMARCA2 depletion in SMARCA4 mutant cancer cells, and in acute myeloid leukemia cells dependent on SMARCA4 ATPase activity.  These findings exemplify a successful biophysics- and structure-based PROTAC design approach to degrade high profile drug targets, and pave the way toward new therapeutics for the treatment of tumors sensitive to the loss of BAF complex ATPases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonEh4pux4Bs7Vg90H21EOLACvtfcHk0lhl_-ypdOqzDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFeqtLfJ&md5=e95fad12c727aedcf08aed9320dae905</span></div><a href="/servlet/linkout?suffix=cit32b&amp;dbid=16384&amp;doi=10.1038%2Fs41589-019-0294-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-019-0294-6%26sid%3Dliteratum%253Aachs%26aulast%3DFarnaby%26aufirst%3DW.%26aulast%3DKoegl%26aufirst%3DM.%26aulast%3DRoy%26aufirst%3DM.%2BJ.%26aulast%3DWhitworth%26aufirst%3DC.%26aulast%3DDiers%26aufirst%3DE.%26aulast%3DTrainor%26aufirst%3DN.%26aulast%3DZollman%26aufirst%3DD.%26aulast%3DSteurer%26aufirst%3DS.%26aulast%3DKarolyi-Oezguer%26aufirst%3DJ.%26aulast%3DRiedmueller%26aufirst%3DC.%26aulast%3DGmaschitz%26aufirst%3DT.%26aulast%3DWachter%26aufirst%3DJ.%26aulast%3DDank%26aufirst%3DC.%26aulast%3DGalant%26aufirst%3DM.%26aulast%3DSharps%26aufirst%3DB.%26aulast%3DRumpel%26aufirst%3DK.%26aulast%3DTraxler%26aufirst%3DE.%26aulast%3DGerstberger%26aufirst%3DT.%26aulast%3DSchnitzer%26aufirst%3DR.%26aulast%3DPetermann%26aufirst%3DO.%26aulast%3DGreb%26aufirst%3DP.%26aulast%3DWeinstabl%26aufirst%3DH.%26aulast%3DBader%26aufirst%3DG.%26aulast%3DZoephel%26aufirst%3DA.%26aulast%3DWeiss-Puxbaum%26aufirst%3DA.%26aulast%3DEhrenh%25C3%25B6fer-W%25C3%25B6lfer%26aufirst%3DK.%26aulast%3DW%25C3%25B6hrle%26aufirst%3DS.%26aulast%3DBoehmelt%26aufirst%3DG.%26aulast%3DRinnenthal%26aufirst%3DJ.%26aulast%3DArnhof%26aufirst%3DH.%26aulast%3DWiechens%26aufirst%3DN.%26aulast%3DWu%26aufirst%3DM.-Y.%26aulast%3DOwen-Hughes%26aufirst%3DT.%26aulast%3DEttmayer%26aufirst%3DP.%26aulast%3DPearson%26aufirst%3DM.%26aulast%3DMcConnell%26aufirst%3DD.%2BB.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DBAF%2520complex%2520vulnerabilities%2520in%2520cancer%2520demonstrated%2520via%2520structure-based%2520PROTAC%2520design%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D15%26spage%3D672%26epage%3D680%26doi%3D10.1038%2Fs41589-019-0294-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit32c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span>; <span class="NLM_string-name">Dong, H.</span>; <span class="NLM_string-name">Wang, J.</span>; <span class="NLM_string-name">Berlin, M.</span>; <span class="NLM_string-name">Siu, K.</span>; <span class="NLM_string-name">Crew, A. P.</span>; <span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Compounds and Methods for the Targeted Degradation of Bromodomain-containing Proteins</span>. International patent application, <span class="NLM_patent">WO2017030814</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=Y.+Qian&author=H.+Dong&author=J.+Wang&author=M.+Berlin&author=K.+Siu&author=A.+P.+Crew&author=C.+M.+Crews&title=Compounds+and+Methods+for+the+Targeted+Degradation+of+Bromodomain-containing+Proteins"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32c&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DQian%26aufirst%3DY.%26atitle%3DCompounds%2520and%2520Methods%2520for%2520the%2520Targeted%2520Degradation%2520of%2520Bromodomain-containing%2520Proteins%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit32d"><span><span class="NLM_label">(d) </span><span class="NLM_contrib-group"><span class="NLM_string-name">Jin, M.</span>; <span class="NLM_string-name">Crew, A.</span>; <span class="NLM_string-name">Dong, H.</span>; <span class="NLM_string-name">Wang, J.</span>; <span class="NLM_string-name">Siu, K.</span>; <span class="NLM_string-name">Ferraro, C.</span>; <span class="NLM_string-name">Chen, X.</span>; <span class="NLM_string-name">Qian, Y.</span></span> <span> </span><span class="NLM_article-title">Compounds and Methods for the Targeted Degradation of the Androgen Receptor</span>.. International patent application, <span class="NLM_patent">WO2016118666</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=M.+Jin&author=A.+Crew&author=H.+Dong&author=J.+Wang&author=K.+Siu&author=C.+Ferraro&author=X.+Chen&author=Y.+Qian&title=Compounds+and+Methods+for+the+Targeted+Degradation+of+the+Androgen+Receptor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32d&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DM.%26atitle%3DCompounds%2520and%2520Methods%2520for%2520the%2520Targeted%2520Degradation%2520of%2520the%2520Androgen%2520Receptor%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Su, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cosino, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">dela Cruz-Chuh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Rosario, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Figueroa, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamath, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaur, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozak, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leipold, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pillow, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polakis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polson, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowntree, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saad, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Safina, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stagg, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tercel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandlen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vollmar, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span></span> <span> </span><span class="NLM_article-title">Antibody-drug conjugates derived from cytotoxic seco-CBI-dimer payloads are highly efficacious in xenograft models and form protein adducts in vivo</span>. <i>Bioconjugate Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">1356</span>– <span class="NLM_lpage">1370</span>, <span class="refDoi"> DOI: 10.1021/acs.bioconjchem.9b00133</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.bioconjchem.9b00133" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmvFart74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2019&pages=1356-1370&author=D.+Suauthor=J.+Chenauthor=E.+Cosinoauthor=J.+dela+Cruz-Chuhauthor=H.+Davisauthor=G.+Del+Rosarioauthor=I.+Figueroaauthor=L.+Goonauthor=J.+Heauthor=A.+V.+Kamathauthor=S.+Kaurauthor=K.+R.+Kozakauthor=J.+Lauauthor=D.+Leeauthor=M.+V.+Leeauthor=D.+Leipoldauthor=L.+Liuauthor=P.+Liuauthor=G.-L.+Luauthor=C.+Nelsonauthor=C.+Ngauthor=T.+H.+Pillowauthor=P.+Polakisauthor=A.+G.+Polsonauthor=R.+K.+Rowntreeauthor=O.+Saadauthor=B.+Safinaauthor=N.+J.+Staggauthor=M.+Tercelauthor=R.+Vandlenauthor=B.+S.+Vollmarauthor=J.+Waiauthor=T.+Wangauthor=B.+Weiauthor=K.+Xuauthor=J.+Xueauthor=Z.+Xuauthor=G.+Yanauthor=H.+Yaoauthor=S.-F.+Yuauthor=D.+Zhangauthor=F.+Zhongauthor=P.+S.+Dragovich&title=Antibody-drug+conjugates+derived+from+cytotoxic+seco-CBI-dimer+payloads+are+highly+efficacious+in+xenograft+models+and+form+protein+adducts+in+vivo&doi=10.1021%2Facs.bioconjchem.9b00133"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33aR"><div class="casContent"><span class="casTitleNuber">33a</span><div class="casTitle"><span class="NLM_cas:atitle">Antibody-drug conjugates derived from cytotoxic seco-CBI-dimer payloads are highly efficacious in xenograft models and form protein adducts in vivo</span></div><div class="casAuthors">Su, Dian; Chen, Jinhua; Cosino, Ely; dela Cruz-Chuh, Josefa; Davis, Helen; Del Rosario, Geoffrey; Figueroa, Isabel; Goon, Leanne; He, Jintang; Kamath, Amrita V.; Kaur, Surinder; Kozak, Katherine R.; Lau, Jeffrey; Lee, Donna; Lee, M. Violet; Leipold, Douglas; Liu, Luna; Liu, Peter; Lu, Guo-Liang; Nelson, Chris; Ng, Carl; Pillow, Thomas H.; Polakis, Paul; Polson, Andrew G.; Rowntree, Rebecca K.; Saad, Ola; Safina, Brian; Stagg, Nicola J.; Tercel, Moana; Vandlen, Richard; Vollmar, Breanna S.; Wai, John; Wang, Tao; Wei, BinQing; Xu, Keyang; Xue, Juanjuan; Xu, Zijin; Yan, Gang; Yao, Hui; Yu, Shang-Fan; Zhang, Donglu; Zhong, Fiona; Dragovich, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Bioconjugate Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1356-1370</span>CODEN:
                <span class="NLM_cas:coden">BCCHES</span>;
        ISSN:<span class="NLM_cas:issn">1043-1802</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This work discloses the first examples of antibody-drug conjugates (ADCs) that are constructed from linker-drugs bearing dimeric seco-CBI payloads (duocarmycin analogs).  Several homogeneous, CD22-targeting THIOMAB antibody-drug conjugates (TDCs) contg. the dimeric seco-CBI entities are shown to be highly efficacious in the WSU-DLCL2 and BJAB mouse xenograft models.  Surprisingly, the seco-CBI-contg. conjugates are also obsd. to undergo significant biotransformation in vivo in mice, rats, and monkeys and thereby form 1:1 adducts with the Alpha-1-Microglobulin (A1M) plasma protein from these species.  Variation of both the payload mAb attachment site and length of the linker-drug is shown to alter the rates of adduct formation.  Subsequent expts. demonstrated that adduct formation attenuates the in vitro antiproliferation activity of the affected seco-CBI-dimer TDCs, but does not significantly impact the in vivo efficacy of the conjugates.  In vitro assays employing phosphatase-treated whole blood suggest that A1M adduct formation is likely to occur if the seco-CBI-dimer TDCs are administered to humans.  Importantly, protein adduct formation leads to the underestimation of total antibody (Tab) concns. using an ELISA assay but does not affect Tab values detd. via an orthogonal LC-MS/MS method.  Several recommendations regarding bioanal. of future in vivo studies involving related seco-CBI-contg. ADCs are provided based on these collective findings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKI29fSnevNLVg90H21EOLACvtfcHk0lhl_-ypdOqzDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmvFart74%253D&md5=3072806260862be9d2578a2bbd4aebc6</span></div><a href="/servlet/linkout?suffix=cit33a&amp;dbid=16384&amp;doi=10.1021%2Facs.bioconjchem.9b00133&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.bioconjchem.9b00133%26sid%3Dliteratum%253Aachs%26aulast%3DSu%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DCosino%26aufirst%3DE.%26aulast%3Ddela%2BCruz-Chuh%26aufirst%3DJ.%26aulast%3DDavis%26aufirst%3DH.%26aulast%3DDel%2BRosario%26aufirst%3DG.%26aulast%3DFigueroa%26aufirst%3DI.%26aulast%3DGoon%26aufirst%3DL.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DKamath%26aufirst%3DA.%2BV.%26aulast%3DKaur%26aufirst%3DS.%26aulast%3DKozak%26aufirst%3DK.%2BR.%26aulast%3DLau%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DD.%26aulast%3DLee%26aufirst%3DM.%2BV.%26aulast%3DLeipold%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DLu%26aufirst%3DG.-L.%26aulast%3DNelson%26aufirst%3DC.%26aulast%3DNg%26aufirst%3DC.%26aulast%3DPillow%26aufirst%3DT.%2BH.%26aulast%3DPolakis%26aufirst%3DP.%26aulast%3DPolson%26aufirst%3DA.%2BG.%26aulast%3DRowntree%26aufirst%3DR.%2BK.%26aulast%3DSaad%26aufirst%3DO.%26aulast%3DSafina%26aufirst%3DB.%26aulast%3DStagg%26aufirst%3DN.%2BJ.%26aulast%3DTercel%26aufirst%3DM.%26aulast%3DVandlen%26aufirst%3DR.%26aulast%3DVollmar%26aufirst%3DB.%2BS.%26aulast%3DWai%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DWei%26aufirst%3DB.%26aulast%3DXu%26aufirst%3DK.%26aulast%3DXue%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DYan%26aufirst%3DG.%26aulast%3DYao%26aufirst%3DH.%26aulast%3DYu%26aufirst%3DS.-F.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DZhong%26aufirst%3DF.%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26atitle%3DAntibody-drug%2520conjugates%2520derived%2520from%2520cytotoxic%2520seco-CBI-dimer%2520payloads%2520are%2520highly%2520efficacious%2520in%2520xenograft%2520models%2520and%2520form%2520protein%2520adducts%2520in%2520vivo%26jtitle%3DBioconjugate%2520Chem.%26date%3D2019%26volume%3D30%26spage%3D1356%26epage%3D1370%26doi%3D10.1021%2Facs.bioconjchem.9b00133" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit33b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunzner-Toste, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nonomiya, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozak, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leipold, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamath, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eigenbrot, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinbacher, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohri, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raab, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staben, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flygare, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pillow, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masterson, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Safina, B.</span></span> <span> </span><span class="NLM_article-title">Discovery of peptidomimetic antibody-drug conjugate linkers with enhanced protease specificity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">989</span>– <span class="NLM_lpage">1000</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFKitrbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=989-1000&author=B.+Weiauthor=J.+Gunzner-Tosteauthor=H.+Yaoauthor=T.+Wangauthor=J.+Wangauthor=Z.+Xuauthor=J.+Chenauthor=J.+Waiauthor=J.+Nonomiyaauthor=S.+P.+Tsaiauthor=J.+Chuhauthor=K.+R.+Kozakauthor=Y.+Liuauthor=S.-F.+Yuauthor=J.+Lauauthor=G.+Liauthor=G.+D.+Phillipsauthor=D.+Leipoldauthor=A.+Kamathauthor=D.+Suauthor=K.+Xuauthor=C.+Eigenbrotauthor=S.+Steinbacherauthor=R.+Ohriauthor=H.+Raabauthor=L.+R.+Stabenauthor=G.+Zhaoauthor=J.+A.+Flygareauthor=T.+H.+Pillowauthor=V.+Vermaauthor=L.+A.+Mastersonauthor=P.+W.+Howardauthor=B.+Safina&title=Discovery+of+peptidomimetic+antibody-drug+conjugate+linkers+with+enhanced+protease+specificity&doi=10.1021%2Facs.jmedchem.7b01430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33bR"><div class="casContent"><span class="casTitleNuber">33b</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Peptidomimetic Antibody-Drug Conjugate Linkers with Enhanced Protease Specificity</span></div><div class="casAuthors">Wei, BinQing; Gunzner-Toste, Janet; Yao, Hui; Wang, Tao; Wang, Jing; Xu, Zijin; Chen, Jinhua; Wai, John; Nonomiya, Jim; Tsai, Siao Ping; Chuh, Josefa; Kozak, Katherine R.; Liu, Yichin; Yu, Shang-Fan; Lau, Jeff; Li, Guangmin; Phillips, Gail D.; Leipold, Doug; Kamath, Amrita; Su, Dian; Xu, Keyang; Eigenbrot, Charles; Steinbacher, Stefan; Ohri, Rachana; Raab, Helga; Staben, Leanna R.; Zhao, Guiling; Flygare, John A.; Pillow, Thomas H.; Verma, Vishal; Safina, Brian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">989-1000</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Antibody-drug conjugates (ADCs) have become an important therapeutic modality for oncol., with three approved by the FDA and over 60 others in clin. trials.  Despite the progress, improvements in ADC therapeutic index are desired.  Peptide-based ADC linkers that are cleaved by lysosomal proteases have shown sufficient stability in serum and effective payload-release in targeted cells.  If the linker can be preferentially hydrolyzed by tumor-specific proteases, safety margin may improve.  However, the use of peptide-based linkers limits our ability to modulate protease specificity.  Here we report the structure-guided discovery of novel, nonpeptidic ADC linkers.  We show that a cyclobutane-1,1-dicarboxamide-contg. linker is hydrolyzed predominantly by cathepsin B while the valine-citrulline dipeptide linker is not.  ADCs bearing the nonpeptidic linker are as efficacious and stable in vivo as those with the dipeptide linker.  Our results strongly support the application of the peptidomimetic linker and present new opportunities for improving the selectivity of ADCs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohInS8FeMmxbVg90H21EOLACvtfcHk0litDsOwWad0PA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFKitrbI&md5=8646f801d6292eaa2c3e2cc1b9e2e6cc</span></div><a href="/servlet/linkout?suffix=cit33b&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01430%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DB.%26aulast%3DGunzner-Toste%26aufirst%3DJ.%26aulast%3DYao%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DWai%26aufirst%3DJ.%26aulast%3DNonomiya%26aufirst%3DJ.%26aulast%3DTsai%26aufirst%3DS.%2BP.%26aulast%3DChuh%26aufirst%3DJ.%26aulast%3DKozak%26aufirst%3DK.%2BR.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DS.-F.%26aulast%3DLau%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DPhillips%26aufirst%3DG.%2BD.%26aulast%3DLeipold%26aufirst%3DD.%26aulast%3DKamath%26aufirst%3DA.%26aulast%3DSu%26aufirst%3DD.%26aulast%3DXu%26aufirst%3DK.%26aulast%3DEigenbrot%26aufirst%3DC.%26aulast%3DSteinbacher%26aufirst%3DS.%26aulast%3DOhri%26aufirst%3DR.%26aulast%3DRaab%26aufirst%3DH.%26aulast%3DStaben%26aufirst%3DL.%2BR.%26aulast%3DZhao%26aufirst%3DG.%26aulast%3DFlygare%26aufirst%3DJ.%2BA.%26aulast%3DPillow%26aufirst%3DT.%2BH.%26aulast%3DVerma%26aufirst%3DV.%26aulast%3DMasterson%26aufirst%3DL.%2BA.%26aulast%3DHoward%26aufirst%3DP.%2BW.%26aulast%3DSafina%26aufirst%3DB.%26atitle%3DDiscovery%2520of%2520peptidomimetic%2520antibody-drug%2520conjugate%2520linkers%2520with%2520enhanced%2520protease%2520specificity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D989%26epage%3D1000%26doi%3D10.1021%2Facs.jmedchem.7b01430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">dela Cruz-Chuh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bobba, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staben, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozak, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, G. D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vollmar, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadowsky, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandlen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khojasteh, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hop, C. E. C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pillow, T. H.</span></span> <span> </span><span class="NLM_article-title">Immolation of <i>p</i>-aminobenzyl ether linker and payload potency and stability determine the cell-killing activity of antibody-drug conjugates with phenol-containing payloads</span>. <i>Bioconjugate Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">267</span>– <span class="NLM_lpage">274</span>, <span class="refDoi"> DOI: 10.1021/acs.bioconjchem.7b00576</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.bioconjchem.7b00576" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1yrs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2018&pages=267-274&author=D.+Zhangauthor=H.+Leauthor=J.+dela+Cruz-Chuhauthor=S.+Bobbaauthor=J.+Guoauthor=L.+Stabenauthor=C.+Zhangauthor=Y.+Maauthor=K.+R.+Kozakauthor=G.+D.+L.+Phillipsauthor=B.+S.+Vollmarauthor=J.+D.+Sadowskyauthor=R.+Vandlenauthor=B.+Weiauthor=D.+Suauthor=P.+Fanauthor=P.+S.+Dragovichauthor=S.+C.+Khojastehauthor=C.+E.+C.+A.+Hopauthor=T.+H.+Pillow&title=Immolation+of+p-aminobenzyl+ether+linker+and+payload+potency+and+stability+determine+the+cell-killing+activity+of+antibody-drug+conjugates+with+phenol-containing+payloads&doi=10.1021%2Facs.bioconjchem.7b00576"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34aR"><div class="casContent"><span class="casTitleNuber">34a</span><div class="casTitle"><span class="NLM_cas:atitle">Immolation of p-Aminobenzyl Ether Linker and Payload Potency and Stability Determine the Cell-Killing Activity of Antibody-Drug Conjugates with Phenol-Containing Payloads</span></div><div class="casAuthors">Zhang, Donglu; Le, Hoa; Cruz-Chuh, Josefa dela; Bobba, Sudheer; Guo, Jun; Staben, Leanna; Zhang, Chenghong; Ma, Yong; Kozak, Katherine R.; Lewis Phillips, Gail D.; Vollmar, Breanna S.; Sadowsky, Jack D.; Vandlen, Richard; Wei, BinQing; Su, Dian; Fan, Peter; Dragovich, Peter S.; Khojasteh, S. Cyrus; Hop, Cornelis E. C. A.; Pillow, Thomas H.</div><div class="citationInfo"><span class="NLM_cas:title">Bioconjugate Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">267-274</span>CODEN:
                <span class="NLM_cas:coden">BCCHES</span>;
        ISSN:<span class="NLM_cas:issn">1043-1802</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The valine-citrulline (Val-Cit) dipeptide and p-aminobenzyl (PAB) spacer have been commonly used as a cleavable self-immolating linker in ADC design including in the clin. approved ADC, brentuximab vedotin (Adcetris).  When the same linker was used to connect to the phenol of the cyclopropabenzindolone (CBI) (P1), the resulting ADC1 showed loss of potency in CD22 target-expressing cancer cell lines (e.g., BJAB, WSU-DLCL2).  In comparison, the conjugate (ADC2) of a cyclopropapyrroloindolone (CPI) (P2) was potent despite the two corresponding free drugs having similar picomolar cell-killing activity.  Although the corresponding spirocyclization products of P1 and P2, responsible for DNA alkylation, are a prominent component in buffer, the linker immolation was slow when the PAB was connected as an ether (PABE) to the phenol in P1 compared to that in P2.  Addnl. immolation studies with two other PABE-linked substituted phenol compds. showed that electron-withdrawing groups accelerated the immolation to release an acidic phenol-contg. payload (to delocalize the neg. charge on the anticipated anionic phenol oxygen during immolation).  In contrast, efficient immolation of LD4 did not result in an active ADC4 because the payload (P4) had a low potency to kill cells.  In addn., nonimmolation of LD5 did not affect the cell-killing potency of its ADC5 since immolation is not required for DNA alkylation by the center-linked pyrrolobenzodiazepine.  Therefore, careful evaluation needs to be conducted when the Val-Cit-PAB linker is used to connect antibodies to a phenol-contg. drug as the linker immolation, as well as payload potency and stability, affects the cell-killing activity of an ADC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrw2sCnqYYrJbVg90H21EOLACvtfcHk0litDsOwWad0PA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1yrs7g%253D&md5=8ff229fed0a454b3d38d7f2a045b1209</span></div><a href="/servlet/linkout?suffix=cit34a&amp;dbid=16384&amp;doi=10.1021%2Facs.bioconjchem.7b00576&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.bioconjchem.7b00576%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DLe%26aufirst%3DH.%26aulast%3Ddela%2BCruz-Chuh%26aufirst%3DJ.%26aulast%3DBobba%26aufirst%3DS.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DStaben%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DKozak%26aufirst%3DK.%2BR.%26aulast%3DPhillips%26aufirst%3DG.%2BD.%2BL.%26aulast%3DVollmar%26aufirst%3DB.%2BS.%26aulast%3DSadowsky%26aufirst%3DJ.%2BD.%26aulast%3DVandlen%26aufirst%3DR.%26aulast%3DWei%26aufirst%3DB.%26aulast%3DSu%26aufirst%3DD.%26aulast%3DFan%26aufirst%3DP.%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26aulast%3DKhojasteh%26aufirst%3DS.%2BC.%26aulast%3DHop%26aufirst%3DC.%2BE.%2BC.%2BA.%26aulast%3DPillow%26aufirst%3DT.%2BH.%26atitle%3DImmolation%2520of%2520p-aminobenzyl%2520ether%2520linker%2520and%2520payload%2520potency%2520and%2520stability%2520determine%2520the%2520cell-killing%2520activity%2520of%2520antibody-drug%2520conjugates%2520with%2520phenol-containing%2520payloads%26jtitle%3DBioconjugate%2520Chem.%26date%3D2018%26volume%3D29%26spage%3D267%26epage%3D274%26doi%3D10.1021%2Facs.bioconjchem.7b00576" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit34b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, S. T.</span></span> <span> </span><span class="NLM_article-title">A self-immolative spacer that enables tunable controlled release of phenols under neutral conditions</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">4363</span>– <span class="NLM_lpage">4374</span>, <span class="refDoi"> DOI: 10.1021/jo300400q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo300400q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC38Xltl2hu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2012&pages=4363-4374&author=K.+M.+Schmidauthor=L.+Jensenauthor=S.+T.+Phillips&title=A+self-immolative+spacer+that+enables+tunable+controlled+release+of+phenols+under+neutral+conditions&doi=10.1021%2Fjo300400q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34bR"><div class="casContent"><span class="casTitleNuber">34b</span><div class="casTitle"><span class="NLM_cas:atitle">A Self-Immolative Spacer That Enables Tunable Controlled Release of Phenols under Neutral Conditions</span></div><div class="casAuthors">Schmid, Kyle M.; Jensen, Lasse; Phillips, Scott T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4363-4374</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A current challenge in the area of responsive materials is the design of reagents and polymers that provide controlled release of phenols in environments that are less polar than water.  In these contexts, a mol. strategy that enables release of nearly any phenol with predictable and tunable rates and without complication from background hydrolysis would substantially increase the precision with which materials can be designed to respond to a particular signal.  This Article addresses this problem at the fundamental level by describing the design, synthesis, and phys.-org. characterization of two small mol. self-immolative spacers that are capable of releasing phenols in org. and mixed org.-aq. solns.  The rate of release from these small mol. model systems is predictable and tunable, such that nearly any type of phenol, regardless of pKa value, can be released in neutral solns. without complications from nonspecific background release due to hydrolysis.  Furthermore, the release properties of the spacers can be predicted from bond length and conformation data (obtained from crystal structures).  On the basis of these results, it should now be possible to incorporate these design elements into materials to enable precise response properties in environments that are not 100% aq.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrc9lFN689qnrVg90H21EOLACvtfcHk0ljKPzaYcLN-Mw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xltl2hu70%253D&md5=b0bb0447e0dccf2d26bb12b2c08ed596</span></div><a href="/servlet/linkout?suffix=cit34b&amp;dbid=16384&amp;doi=10.1021%2Fjo300400q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo300400q%26sid%3Dliteratum%253Aachs%26aulast%3DSchmid%26aufirst%3DK.%2BM.%26aulast%3DJensen%26aufirst%3DL.%26aulast%3DPhillips%26aufirst%3DS.%2BT.%26atitle%3DA%2520self-immolative%2520spacer%2520that%2520enables%2520tunable%2520controlled%2520release%2520of%2520phenols%2520under%2520neutral%2520conditions%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2012%26volume%3D77%26spage%3D4363%26epage%3D4374%26doi%3D10.1021%2Fjo300400q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berthon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raffoux, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vey, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomez-Roca, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taussig, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roumier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herait, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahatt, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quesnel, B.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">OTX015 in patients with acute leukaemia: A dose-escalation, phase 1 study</span>. <i>Lancet Haematol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">e186</span>– <span class="NLM_lpage">e195</span>, <span class="refDoi"> DOI: 10.1016/S2352-3026(15)00247-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1016%2FS2352-3026%2815%2900247-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=27063977" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2016&pages=e186-e195&author=C.+Berthonauthor=E.+Raffouxauthor=X.+Thomasauthor=N.+Veyauthor=C.+Gomez-Rocaauthor=K.+Yeeauthor=D.+C.+Taussigauthor=K.+Rezaiauthor=C.+Roumierauthor=P.+Heraitauthor=C.+Kahattauthor=B.+Quesnel&title=OTX015+in+patients+with+acute+leukaemia%3A+A+dose-escalation%2C+phase+1+study&doi=10.1016%2FS2352-3026%2815%2900247-1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2FS2352-3026%2815%2900247-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS2352-3026%252815%252900247-1%26sid%3Dliteratum%253Aachs%26aulast%3DBerthon%26aufirst%3DC.%26aulast%3DRaffoux%26aufirst%3DE.%26aulast%3DThomas%26aufirst%3DX.%26aulast%3DVey%26aufirst%3DN.%26aulast%3DGomez-Roca%26aufirst%3DC.%26aulast%3DYee%26aufirst%3DK.%26aulast%3DTaussig%26aufirst%3DD.%2BC.%26aulast%3DRezai%26aufirst%3DK.%26aulast%3DRoumier%26aufirst%3DC.%26aulast%3DHerait%26aufirst%3DP.%26aulast%3DKahatt%26aufirst%3DC.%26aulast%3DQuesnel%26aufirst%3DB.%26atitle%3DOTX015%2520in%2520patients%2520with%2520acute%2520leukaemia%253A%2520A%2520dose-escalation%252C%2520phase%25201%2520study%26jtitle%3DLancet%2520Haematol.%26date%3D2016%26volume%3D3%26spage%3De186%26epage%3De195%26doi%3D10.1016%2FS2352-3026%2815%2900247-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">del Rosario, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leong, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, G. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vij, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, W.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chalouni, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadowsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hendricks, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Achilles Poon, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pillow, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schutten, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flygare, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polson, A. G.</span></span> <span> </span><span class="NLM_article-title">An anti-CLL1 antibody-drug conjugate for the treatment of acute myeloid leukemia</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1358</span>– <span class="NLM_lpage">1368</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-18-0333</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1158%2F1078-0432.CCR-18-0333" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=29959143" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtlOmt7bE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2019&pages=1358-1368&author=B.+Zhengauthor=S.-F.+Yuauthor=G.+del+Rosarioauthor=S.+R.+Leongauthor=G.+Y.+Leeauthor=R.+Vijauthor=C.+Chiuauthor=W.-C.+Liangauthor=Y.+Wuauthor=C.+Chalouniauthor=J.+Sadowskyauthor=V.+Clarkauthor=A.+Hendricksauthor=K.+Achilles+Poonauthor=W.+Chuauthor=T.+Pillowauthor=M.+M.+Schuttenauthor=J.+Flygareauthor=A.+G.+Polson&title=An+anti-CLL1+antibody-drug+conjugate+for+the+treatment+of+acute+myeloid+leukemia&doi=10.1158%2F1078-0432.CCR-18-0333"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">An anti-CLL-I antibody-drug conjugate for the treatment of acute myeloid leukemia</span></div><div class="casAuthors">Zheng, Bing; Yu, Shang-Fan; del Rosario, Geoffrey; Leong, Steven R.; Lee, Genee Y.; Vij, Rajesh; Chiu, Cecilia; Liang, Wei-Ching; Wu, Yan; Chalouni, Cecile; Sadowsky, Jack; Clark, Vanessa; Hendricks, Angela; Poon, Kirsten Achilles; Chu, Wayne; Pillow, Thomas; Schutten, Melissa M.; Flygare, John; Polson, Andrew G.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1358</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: The treatment of acute myeloid leukemia (AML) has not significantly changed in 40 years.  Cytarabine- and anthracycline-based chemotherapy induction regimens (7 + 3) remain the std. of care, and most patients have poor long-term survival.  The reapproval of Mylotarg, an anti-CD33-calicheamicin antibody-drug conjugate (ADC), has demonstrated ADCs as a clin. validated option to enhance the effectiveness of induction therapy.  We are interested in developing a next-generation ADC for AML to improve upon the initial success of Mylotarg.  Exptl. Design: The expression pattern of CLL-1 and its hematopoietic potential were investigated.  A novel anti-CLL-1-ADC, with a highly potent pyrrolobenzodiazepine (PBD) dimer conjugated through a self-immolative disulfide linker, was developed.  The efficacy and safety profiles of this ADC were evaluated in mouse xenograft models and in cynomolgus monkeys.  Results: We demonstrate that CLL-1 shares similar prevalence and trafficking properties that make CD33 an excellent ADC target for AML, but lacks expression on hematopoietic stem cells that hampers current CD33-targeted ADCs.  Our anti-CLL-1-ADC is highly effective at depleting tumor cells in AML xenograft models and lacks target independent toxicities at doses that depleted target monocytes and neutrophils in cynomolgus monkeys.  Conclusions: Collectively, our data suggest that an anti-CLL-1-ADC has the potential to become an effective and safer treatment for AML in humans, by reducing and allowing for faster recovery from initial cytopenias than the current generation of ADCs for AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJdzTurW_r4LVg90H21EOLACvtfcHk0ljKPzaYcLN-Mw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtlOmt7bE&md5=c701a514d19baec87815539f0eaa9275</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-18-0333&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-18-0333%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DB.%26aulast%3DYu%26aufirst%3DS.-F.%26aulast%3Ddel%2BRosario%26aufirst%3DG.%26aulast%3DLeong%26aufirst%3DS.%2BR.%26aulast%3DLee%26aufirst%3DG.%2BY.%26aulast%3DVij%26aufirst%3DR.%26aulast%3DChiu%26aufirst%3DC.%26aulast%3DLiang%26aufirst%3DW.-C.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DChalouni%26aufirst%3DC.%26aulast%3DSadowsky%26aufirst%3DJ.%26aulast%3DClark%26aufirst%3DV.%26aulast%3DHendricks%26aufirst%3DA.%26aulast%3DAchilles%2BPoon%26aufirst%3DK.%26aulast%3DChu%26aufirst%3DW.%26aulast%3DPillow%26aufirst%3DT.%26aulast%3DSchutten%26aufirst%3DM.%2BM.%26aulast%3DFlygare%26aufirst%3DJ.%26aulast%3DPolson%26aufirst%3DA.%2BG.%26atitle%3DAn%2520anti-CLL1%2520antibody-drug%2520conjugate%2520for%2520the%2520treatment%2520of%2520acute%2520myeloid%2520leukemia%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2019%26volume%3D25%26spage%3D1358%26epage%3D1368%26doi%3D10.1158%2F1078-0432.CCR-18-0333" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leipold, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Figueroa, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masih, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Latifi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yip, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, B.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dere, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carrasco-Triguero, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saad, O. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vuillemenot, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laing, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schutten, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozak, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polson, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamath, A. V.</span></span> <span> </span><span class="NLM_article-title">Preclinical pharmacokinetics and pharmacodynamics of DCLL9718A: An antibody-drug conjugate for the treatment of acute myeloid leukemia</span>. <i>mAbs</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1312</span>– <span class="NLM_lpage">1321</span>, <span class="refDoi"> DOI: 10.1080/19420862.2018.1517565</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1080%2F19420862.2018.1517565" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=30183491" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVGjtLjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=1312-1321&author=D.+D.+Leipoldauthor=I.+Figueroaauthor=S.+Masihauthor=B.+Latifiauthor=V.+Yipauthor=B.-Q.+Shenauthor=R.+C.+Dereauthor=M.+Carrasco-Trigueroauthor=M.+V.+Leeauthor=O.+M.+Saadauthor=L.+Liuauthor=J.+Heauthor=D.+Suauthor=K.+Xuauthor=B.+R.+Vuillemenotauthor=S.+T.+Laingauthor=M.+Schuttenauthor=K.+R.+Kozakauthor=B.+Zhengauthor=A.+G.+Polsonauthor=A.+V.+Kamath&title=Preclinical+pharmacokinetics+and+pharmacodynamics+of+DCLL9718A%3A+An+antibody-drug+conjugate+for+the+treatment+of+acute+myeloid+leukemia&doi=10.1080%2F19420862.2018.1517565"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Preclinical pharmacokinetics and pharmacodynamics of DCLL9718A: An antibody-drug conjugate for the treatment of acute myeloid leukemia</span></div><div class="casAuthors">Leipold, Douglas D.; Figueroa, Isabel; Masih, Shabkhaiz; Latifi, Brandon; Yip, Victor; Shen, Ben-Quan; Dere, Randall C.; Carrasco-Triguero, Montserrat; Lee, M. Violet; Saad, Ola M.; Liu, Luna; He, Jintang; Su, Dian; Xu, Keyang; Vuillemenot, Brian R.; Laing, Steven T.; Schutten, Melissa; Kozak, Katherine R.; Zheng, Bing; Polson, Andrew G.; Kamath, Amrita V.</div><div class="citationInfo"><span class="NLM_cas:title">mAbs</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1312-1321</span>CODEN:
                <span class="NLM_cas:coden">MABSCP</span>;
        ISSN:<span class="NLM_cas:issn">1942-0870</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis, Inc.</span>)
        </div><div class="casAbstract">Few treatment options are available for acute myeloid leukemia (AML) patients.  DCLL9718A is an antibody-drug conjugate that targets C-type lectin-like mol.-1 (CLL-1).  Here, we characterize the in vitro and in vivo stability, the pharmacokinetics (PK) and pharmacodynamics (PD) of DCLL9718A and MCLL0517A in rodents and cynomolgus monkeys.  Three key PK analytes were measured in these studies: total antibody, antibody-conjugated PBD dimer and unconjugated PBD dimer.  In vitro, DCLL9718A, was stable with most (> 80%) of the PBD dimer payload remaining conjugated to the antibody over 96 h.  This was recapitulated in vivo with antibody-conjugated PBD dimer clearance ests. similar to DCLL9718A total antibody clearance.  Both DCLL9718A and MCLL0517A showed linear PK in the non-binding rodent species, and non-linear PK in cynomolgus monkeys, a binding species.  The PK data indicated minimal impact of conjugation on the disposition of DCLL9718A total antibody.  Finally, in cynomolgus monkey, MCLL0517A showed target engagement at all doses tested (0.5 and 20 mg/kg) as measured by receptor occupancy, and DCLL9718A (at doses of 0.05, 0.1 and 0.2 mg/kg) showed strong PD activity as evidenced by notable redn. in monocytes and neutrophils.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsFXdb4ZZBT7Vg90H21EOLACvtfcHk0linBiR0BEFzbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVGjtLjN&md5=d1370e943ab7a89e07a60f56b624785b</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1080%2F19420862.2018.1517565&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F19420862.2018.1517565%26sid%3Dliteratum%253Aachs%26aulast%3DLeipold%26aufirst%3DD.%2BD.%26aulast%3DFigueroa%26aufirst%3DI.%26aulast%3DMasih%26aufirst%3DS.%26aulast%3DLatifi%26aufirst%3DB.%26aulast%3DYip%26aufirst%3DV.%26aulast%3DShen%26aufirst%3DB.-Q.%26aulast%3DDere%26aufirst%3DR.%2BC.%26aulast%3DCarrasco-Triguero%26aufirst%3DM.%26aulast%3DLee%26aufirst%3DM.%2BV.%26aulast%3DSaad%26aufirst%3DO.%2BM.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DSu%26aufirst%3DD.%26aulast%3DXu%26aufirst%3DK.%26aulast%3DVuillemenot%26aufirst%3DB.%2BR.%26aulast%3DLaing%26aufirst%3DS.%2BT.%26aulast%3DSchutten%26aufirst%3DM.%26aulast%3DKozak%26aufirst%3DK.%2BR.%26aulast%3DZheng%26aufirst%3DB.%26aulast%3DPolson%26aufirst%3DA.%2BG.%26aulast%3DKamath%26aufirst%3DA.%2BV.%26atitle%3DPreclinical%2520pharmacokinetics%2520and%2520pharmacodynamics%2520of%2520DCLL9718A%253A%2520An%2520antibody-drug%2520conjugate%2520for%2520the%2520treatment%2520of%2520acute%2520myeloid%2520leukemia%26jtitle%3DmAbs%26date%3D2018%26volume%3D10%26spage%3D1312%26epage%3D1321%26doi%3D10.1080%2F19420862.2018.1517565" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thummuri, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, G.</span></span> <span> </span><span class="NLM_article-title">Discovery of PROTAC BCL-X<sub>L</sub> degraders as potent anticancer agents with low on-target platelet toxicity</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>192</i></span>, <span class="NLM_elocation-id">112186</span> <span class="refDoi"> DOI: 10.1016/j.ejmech.2020.112186</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1016%2Fj.ejmech.2020.112186" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=32145645" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BB3cXksFarur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=192&publication_year=2020&author=X.+Zhangauthor=D.+Thummuriauthor=X.+Liuauthor=W.+Huauthor=P.+Zhangauthor=S.+Khanauthor=Y.+Yuanauthor=D.+Zhouauthor=G.+Zheng&title=Discovery+of+PROTAC+BCL-XL+degraders+as+potent+anticancer+agents+with+low+on-target+platelet+toxicity&doi=10.1016%2Fj.ejmech.2020.112186"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38aR"><div class="casContent"><span class="casTitleNuber">38a</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of PROTAC BCL-XL degraders as potent anticancer agents with low on-target platelet toxicity</span></div><div class="casAuthors">Zhang, Xuan; Thummuri, Dinesh; Liu, Xingui; Hu, Wanyi; Zhang, Peiyi; Khan, Sajid; Yuan, Yaxia; Zhou, Daohong; Zheng, Guangrong</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">192</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112186</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Anti-apoptotic protein BCL-XL plays a key role in tumorigenesis and cancer chemotherapy resistance, rendering it an attractive target for cancer treatment.  However, BCL-XL inhibitors such as ABT-263 cannot be safely used in the clinic because platelets solely depend on BCL-XL to maintain their viability.  To reduce the on-target platelet toxicity assocd. with the inhibition of BCL-XL, we designed and synthesized PROTAC BCL-XL degraders that recruit CRBN or VHL E3 ligase because both of these enzymes are poorly expressed in human platelets compared to various cancer cell lines.  We confirmed that platelet-toxic BCL-XL/2 dual inhibitor ABT-263 can be converted into platelet-sparing CRBN/VHL-based BCL-XL specific degraders.  A no. of BCL-XL degraders are more potent in killing cancer cells than their parent compd. ABT-263.  Specifically, XZ739, a CRBN-dependent BCL-XL degrader, is 20-fold more potent than ABT-263 against MOLT-4 T-ALL cells and has >100-fold selectivity for MOLT-4 cells over human platelets.  Our findings further demonstrated the utility of PROTAC technol. to achieve tissue selectivity through recruiting differentially expressed E3 ligases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYSnEzXDSn97Vg90H21EOLACvtfcHk0linBiR0BEFzbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXksFarur4%253D&md5=879c25c55e21f3136072477d8368499b</span></div><a href="/servlet/linkout?suffix=cit38a&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112186&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112186%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DThummuri%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DHu%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DKhan%26aufirst%3DS.%26aulast%3DYuan%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DD.%26aulast%3DZheng%26aufirst%3DG.%26atitle%3DDiscovery%2520of%2520PROTAC%2520BCL-XL%2520degraders%2520as%2520potent%2520anticancer%2520agents%2520with%2520low%2520on-target%2520platelet%2520toxicity%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D192%26doi%3D10.1016%2Fj.ejmech.2020.112186" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit38b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H.-N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thummuri, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiegand, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortiz, Y. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Budamagunta, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elisseeff, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campisi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almeida, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, D.</span></span> <span> </span><span class="NLM_article-title">Using proteolysis-targeting chimera technology to reduce navitoclax platelet toxicity and improve its senolytic activity</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>, <span class="NLM_elocation-id">1996</span> <span class="refDoi"> DOI: 10.1038/s41467-020-15838-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1038%2Fs41467-020-15838-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=32332723" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BB3cXot1amtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&author=Y.+Heauthor=X.+Zhangauthor=J.+Changauthor=H.-N.+Kimauthor=P.+Zhangauthor=Y.+Wangauthor=S.+Khanauthor=X.+Liuauthor=X.+Zhangauthor=D.+Lvauthor=L.+Songauthor=W.+Liauthor=D.+Thummuriauthor=Y.+Yuanauthor=J.+S.+Wiegandauthor=Y.+T.+Ortizauthor=V.+Budamaguntaauthor=J.+H.+Elisseeffauthor=J.+Campisiauthor=M.+Almeidaauthor=G.+Zhengauthor=D.+Zhou&title=Using+proteolysis-targeting+chimera+technology+to+reduce+navitoclax+platelet+toxicity+and+improve+its+senolytic+activity&doi=10.1038%2Fs41467-020-15838-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38bR"><div class="casContent"><span class="casTitleNuber">38b</span><div class="casTitle"><span class="NLM_cas:atitle">Using proteolysis-targeting chimera technology to reduce navitoclax platelet toxicity and improve its senolytic activity</span></div><div class="casAuthors">He, Yonghan; Zhang, Xuan; Chang, Jianhui; Kim, Ha-Neui; Zhang, Peiyi; Wang, Yingying; Khan, Sajid; Liu, Xingui; Zhang, Xin; Lv, Dongwen; Song, Lin; Li, Wen; Thummuri, Dinesh; Yuan, Yaxia; Wiegand, Janet S.; Ortiz, Yuma T.; Budamagunta, Vivekananda; Elisseeff, Jennifer H.; Campisi, Judith; Almeida, Maria; Zheng, Guangrong; Zhou, Daohong</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1996</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Small mols. that selectively kill senescent cells (SCs), termed senolytics, have the potential to prevent and treat various age-related diseases and extend healthspan.  The use of Bcl-xl inhibitors as senolytics is largely limited by their on-target and dose-limiting platelet toxicity.  Here, we report the use of proteolysis-targeting chimera (PROTAC) technol. to reduce the platelet toxicity of navitoclax (also known as ABT263), a Bcl-2 and Bcl-xl dual inhibitor, by converting it into PZ15227 (PZ), a Bcl-xl PROTAC, which targets Bcl-xl to the cereblon (CRBN) E3 ligase for degrdn.  Compared to ABT263, PZ is less toxic to platelets, but equally or slightly more potent against SCs because CRBN is poorly expressed in platelets.  PZ effectively clears SCs and rejuvenates tissue stem and progenitor cells in naturally aged mice without causing severe thrombocytopenia.  With further improvement, Bcl-xl PROTACs have the potential to become safer and more potent senolytic agents than Bcl-xl inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoA89Bee3qtEbVg90H21EOLACvtfcHk0linBiR0BEFzbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXot1amtrc%253D&md5=bb66a0a79d0180c8abbee076324ec6b3</span></div><a href="/servlet/linkout?suffix=cit38b&amp;dbid=16384&amp;doi=10.1038%2Fs41467-020-15838-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-020-15838-0%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DChang%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DH.-N.%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DKhan%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DLv%26aufirst%3DD.%26aulast%3DSong%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DThummuri%26aufirst%3DD.%26aulast%3DYuan%26aufirst%3DY.%26aulast%3DWiegand%26aufirst%3DJ.%2BS.%26aulast%3DOrtiz%26aufirst%3DY.%2BT.%26aulast%3DBudamagunta%26aufirst%3DV.%26aulast%3DElisseeff%26aufirst%3DJ.%2BH.%26aulast%3DCampisi%26aufirst%3DJ.%26aulast%3DAlmeida%26aufirst%3DM.%26aulast%3DZheng%26aufirst%3DG.%26aulast%3DZhou%26aufirst%3DD.%26atitle%3DUsing%2520proteolysis-targeting%2520chimera%2520technology%2520to%2520reduce%2520navitoclax%2520platelet%2520toxicity%2520and%2520improve%2520its%2520senolytic%2520activity%26jtitle%3DNat.%2520Commun.%26date%3D2020%26volume%3D11%26doi%3D10.1038%2Fs41467-020-15838-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit38c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thummuri, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiegand, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCurdy, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuruvilla, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baran, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrando, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.-m.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogojina, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houghton, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hromas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konopleva, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daohong
Zhou, D.</span></span> <span> </span><span class="NLM_article-title">A selective BCL-X<sub>L</sub> PROTAC degrader achieves safe and potent antitumor activity</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1938</span>– <span class="NLM_lpage">1947</span>, <span class="refDoi"> DOI: 10.1038/s41591-019-0668-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1038%2Fs41591-019-0668-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=31792461" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1yjsrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2019&pages=1938-1947&author=S.+Khanauthor=X.+Zhangauthor=D.+Lvauthor=Q.+Zhangauthor=Y.+Heauthor=P.+Zhangauthor=X.+Liuauthor=D.+Thummuriauthor=Y.+Yuanauthor=J.+S.+Wiegandauthor=J.+Peiauthor=W.+Zhangauthor=A.+Sharmaauthor=C.+R.+McCurdyauthor=V.+M.+Kuruvillaauthor=N.+Baranauthor=A.+A.+Ferrandoauthor=Y.-m.+Kimauthor=A.+Rogojinaauthor=P.+J.+Houghtonauthor=G.+Huangauthor=R.+Hromasauthor=M.+Konoplevaauthor=G.+Zhengauthor=D.+Daohong%0AZhou&title=A+selective+BCL-XL+PROTAC+degrader+achieves+safe+and+potent+antitumor+activity&doi=10.1038%2Fs41591-019-0668-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38cR"><div class="casContent"><span class="casTitleNuber">38c</span><div class="casTitle"><span class="NLM_cas:atitle">A selective BCL-XL PROTAC degrader achieves safe and potent antitumor activity</span></div><div class="casAuthors">Khan, Sajid; Zhang, Xuan; Lv, Dongwen; Zhang, Qi; He, Yonghan; Zhang, Peiyi; Liu, Xingui; Thummuri, Dinesh; Yuan, Yaxia; Wiegand, Janet S.; Pei, Jing; Zhang, Weizhou; Sharma, Abhisheak; McCurdy, Christopher R.; Kuruvilla, Vinitha M.; Baran, Natalia; Ferrando, Adolfo A.; Kim, Yong-mi; Rogojina, Anna; Houghton, Peter J.; Huang, Guangcun; Hromas, Robert; Konopleva, Marina; Zheng, Guangrong; Zhou, Daohong</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1938-1947</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">B-cell lymphoma extra large (BCL-XL) is a well-validated cancer target.  However, the on-target and dose-limiting thrombocytopenia limits the use of BCL-XL inhibitors, such as ABT263, as safe and effective anticancer agents.  To reduce the toxicity of ABT263, we converted it into DT2216, a BCL-XL proteolysis-targeting chimera (PROTAC), that targets BCL-XL to the Von Hippel-Lindau (VHL) E3 ligase for degrdn.  We found that DT2216 was more potent against various BCL-XL-dependent leukemia and cancer cells but considerably less toxic to platelets than ABT263 in vitro because VHL is poorly expressed in platelets.  In vivo, DT2216 effectively inhibits the growth of several xenograft tumors as a single agent or in combination with other chemotherapeutic agents, without causing appreciable thrombocytopenia.  These findings demonstrate the potential to use PROTAC technol. to reduce on-target drug toxicities and rescue the therapeutic potential of previously undruggable targets.  Furthermore, DT2216 may be developed as a safe first-in-class anticancer agent targeting BCL-XL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGposmmgxVNCRLVg90H21EOLACvtfcHk0lhJOZ2LF_A8Mw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1yjsrrO&md5=b07678c880569acac428ec479f002974</span></div><a href="/servlet/linkout?suffix=cit38c&amp;dbid=16384&amp;doi=10.1038%2Fs41591-019-0668-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41591-019-0668-z%26sid%3Dliteratum%253Aachs%26aulast%3DKhan%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DLv%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DThummuri%26aufirst%3DD.%26aulast%3DYuan%26aufirst%3DY.%26aulast%3DWiegand%26aufirst%3DJ.%2BS.%26aulast%3DPei%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DSharma%26aufirst%3DA.%26aulast%3DMcCurdy%26aufirst%3DC.%2BR.%26aulast%3DKuruvilla%26aufirst%3DV.%2BM.%26aulast%3DBaran%26aufirst%3DN.%26aulast%3DFerrando%26aufirst%3DA.%2BA.%26aulast%3DKim%26aufirst%3DY.-m.%26aulast%3DRogojina%26aufirst%3DA.%26aulast%3DHoughton%26aufirst%3DP.%2BJ.%26aulast%3DHuang%26aufirst%3DG.%26aulast%3DHromas%26aufirst%3DR.%26aulast%3DKonopleva%26aufirst%3DM.%26aulast%3DZheng%26aufirst%3DG.%26aulast%3DDaohong%2BZhou%26aufirst%3DD.%26atitle%3DA%2520selective%2520BCL-XL%2520PROTAC%2520degrader%2520achieves%2520safe%2520and%2520potent%2520antitumor%2520activity%26jtitle%3DNat.%2520Med.%26date%3D2019%26volume%3D25%26spage%3D1938%26epage%3D1947%26doi%3D10.1038%2Fs41591-019-0668-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Tarrant, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhawan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zabka, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DosSantos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sampath, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCray, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kauss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dambach, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Misner, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uppal, H.</span></span> <span> </span><span class="NLM_article-title">Preclinical models of nicotinamide phosphoribosyltransferase inhibitor-mediated hematotoxicity and mitigation by co-treatment with nicotinic acid</span>. <i>Toxicol. Mech. Methods</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">201</span>– <span class="NLM_lpage">211</span>, <span class="refDoi"> DOI: 10.3109/15376516.2015.1014080</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.3109%2F15376516.2015.1014080" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=25894564" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptVSjt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=201-211&author=J.+M.+Tarrantauthor=P.+Dhawanauthor=J.+Singhauthor=T.+S.+Zabkaauthor=E.+Clarkeauthor=G.+DosSantosauthor=P.+S.+Dragovichauthor=D.+Sampathauthor=T.+Linauthor=B.+McCrayauthor=N.+Laauthor=T.+Nguyenauthor=A.+Kaussauthor=D.+Dambachauthor=D.+L.+Misnerauthor=D.+Diazauthor=H.+Uppal&title=Preclinical+models+of+nicotinamide+phosphoribosyltransferase+inhibitor-mediated+hematotoxicity+and+mitigation+by+co-treatment+with+nicotinic+acid&doi=10.3109%2F15376516.2015.1014080"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39aR"><div class="casContent"><span class="casTitleNuber">39a</span><div class="casTitle"><span class="NLM_cas:atitle">Preclinical models of nicotinamide phosphoribosyltransferase inhibitor-mediated hematotoxicity and mitigation by co-treatment with nicotinic acid</span></div><div class="casAuthors">Tarrant, Jacqueline M.; Dhawan, Preeti; Singh, Jatinder; Zabka, Tanja S.; Clarke, Emer; DosSantos, Garry; Dragovich, Peter S.; Sampath, Deepak; Lin, Tori; McCray, Bobbi; La, Nghi; Nguyen, Trung; Kauss, Ariel; Dambach, Donna; Misner, Dinah L.; Diaz, Dolores; Uppal, Hirdesh</div><div class="citationInfo"><span class="NLM_cas:title">Toxicology Mechanisms and Methods</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">201-211</span>CODEN:
                <span class="NLM_cas:coden">TMMOCP</span>;
        ISSN:<span class="NLM_cas:issn">1537-6516</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">NAD (NAD) is an essential co-factor in glycolysis and is a key mol. involved in maintaining cellular energy metab.  Nicotinamide phosphoribosyltransferase (NAMPT) catalyzes the rate-limiting step of an important salvage pathway in which nicotinamide is recycled into NAD.  NAMPT is up-regulated in many types of cancer and NAMPT inhibitors (NAMPTi) have potential therapeutic benefit in cancer by impairing tumor metab.  Clin. trials with NAMPTi APO-866 and GMX-1778, however, failed to reach projected efficacious exposures due to dose-limiting thrombocytopenia.  We evaluated preclin. models for thrombocytopenia that could be used in candidate drug selection and risk mitigation strategies for NAMPTi-related toxicity.  Rats treated with a suite of structurally diverse and potent NAMPTi at max. tolerated doses had decreased reticulocyte and lymphocyte counts, but no thrombocytopenia.  We therefore evaluated and qualified a human colony forming unit-megakaryocyte (CFU-MK) as in vitro predictive model of NAMPTi-induced MK toxicity and thrombocytopenia.  We further demonstrate that the MK toxicity is on-target based on the evidence that nicotinic acid (NA), which is converted to NAD via a NAMPT-independent pathway, can mitigate NAMPTi toxicity to human CFU-MK in vitro and was also protective for the hematotoxicity in rats in vivo.  Finally, assessment of CFU-MK and human platelet bioenergetics and function show that NAMPTi was toxic to MK and not platelets, which is consistent with the clin. obsd. time-course of thrombocytopenia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVoRVrRU14ubVg90H21EOLACvtfcHk0lhJOZ2LF_A8Mw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptVSjt7k%253D&md5=536137fdb1961f4dbf5807d9655e6549</span></div><a href="/servlet/linkout?suffix=cit39a&amp;dbid=16384&amp;doi=10.3109%2F15376516.2015.1014080&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F15376516.2015.1014080%26sid%3Dliteratum%253Aachs%26aulast%3DTarrant%26aufirst%3DJ.%2BM.%26aulast%3DDhawan%26aufirst%3DP.%26aulast%3DSingh%26aufirst%3DJ.%26aulast%3DZabka%26aufirst%3DT.%2BS.%26aulast%3DClarke%26aufirst%3DE.%26aulast%3DDosSantos%26aufirst%3DG.%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26aulast%3DSampath%26aufirst%3DD.%26aulast%3DLin%26aufirst%3DT.%26aulast%3DMcCray%26aufirst%3DB.%26aulast%3DLa%26aufirst%3DN.%26aulast%3DNguyen%26aufirst%3DT.%26aulast%3DKauss%26aufirst%3DA.%26aulast%3DDambach%26aufirst%3DD.%26aulast%3DMisner%26aufirst%3DD.%2BL.%26aulast%3DDiaz%26aufirst%3DD.%26aulast%3DUppal%26aufirst%3DH.%26atitle%3DPreclinical%2520models%2520of%2520nicotinamide%2520phosphoribosyltransferase%2520inhibitor-mediated%2520hematotoxicity%2520and%2520mitigation%2520by%2520co-treatment%2520with%2520nicotinic%2520acid%26jtitle%3DToxicol.%2520Mech.%2520Methods%26date%3D2015%26volume%3D25%26spage%3D201%26epage%3D211%26doi%3D10.3109%2F15376516.2015.1014080" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit39b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Pessina, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parent-Massin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albella, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Den
Heuvel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casati, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Croera, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malerba, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sibiril, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomez, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Smedt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gribaldo, L.</span></span> <span> </span><span class="NLM_article-title">Application of human CFU-Mk assay to predict potential thrombocytotoxicity of drugs</span>. <i>Toxicol. In Vitro</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">194</span>– <span class="NLM_lpage">200</span>, <span class="refDoi"> DOI: 10.1016/j.tiv.2008.11.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1016%2Fj.tiv.2008.11.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=19084588" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BD1MXktlKrsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2009&pages=194-200&author=A.+Pessinaauthor=D.+Parent-Massinauthor=B.+Albellaauthor=R.+Van+Den%0AHeuvelauthor=S.+Casatiauthor=C.+Croeraauthor=I.+Malerbaauthor=Y.+Sibirilauthor=S.+Gomezauthor=A.+de+Smedtauthor=L.+Gribaldo&title=Application+of+human+CFU-Mk+assay+to+predict+potential+thrombocytotoxicity+of+drugs&doi=10.1016%2Fj.tiv.2008.11.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39bR"><div class="casContent"><span class="casTitleNuber">39b</span><div class="casTitle"><span class="NLM_cas:atitle">Application of human CFU-Mk assay to predict potential thrombocytotoxicity of drugs</span></div><div class="casAuthors">Pessina, A.; Parent-Massin, D.; Albella, B.; Van Den Heuvel, R.; Casati, S.; Croera, C.; Malerba, I.; Sibiril, Y.; Gomez, S.; de Smedt, A.; Gribaldo, L.</div><div class="citationInfo"><span class="NLM_cas:title">Toxicology in Vitro</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">194-200</span>CODEN:
                <span class="NLM_cas:coden">TIVIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0887-2333</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Megakaryocytopoiesis gives rise to platelets by proliferation and differentiation of lineage-specific progenitors, identified in vitro as Colony Forming Unit-Megakaryocytes (CFU-Mk).  The aim of this study was to refine and optimize the in vitro Std. Operating Procedure (SOP) of the CFU-Mk assay for detecting drug-induced thrombocytopenia and to prevalidate a model for predicting the acute exposure levels that cause max. tolerated decreases in the platelets count, based on the correlation with the maximal plasma concns. (C max) in vivo.  The assay was linear under the SOP conditions, and the in vitro endpoints (percentage of colonies growing) were reproducible within and across labs.  The protocol performance phase was carried out testing 10 drugs (selected on the base of their recognized or potential in vivo haematotoxicity, according to the literature).  Results showed that a relationship can be established between the maximal concn. in plasma (C max) and the in vitro concns. that inhibited the 10-50-90% of colonies growth (ICs).  When C max is lower than IC10, it is possible to predict that the chems. have no direct toxicity effect on CFU-Mk and could not induce thrombocytopenia due to bone marrow damage.  When the C max is higher than IC90 and/or IC50, thrombocytopenia can occur due to direct toxicity of chems. on CFU-Mk progenitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4ZYnZ3DfsdLVg90H21EOLACvtfcHk0liaR9G-ZL0Oqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXktlKrsg%253D%253D&md5=6994a99dee97665057c0862450d79350</span></div><a href="/servlet/linkout?suffix=cit39b&amp;dbid=16384&amp;doi=10.1016%2Fj.tiv.2008.11.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tiv.2008.11.006%26sid%3Dliteratum%253Aachs%26aulast%3DPessina%26aufirst%3DA.%26aulast%3DParent-Massin%26aufirst%3DD.%26aulast%3DAlbella%26aufirst%3DB.%26aulast%3DVan%2BDen%2BHeuvel%26aufirst%3DR.%26aulast%3DCasati%26aufirst%3DS.%26aulast%3DCroera%26aufirst%3DC.%26aulast%3DMalerba%26aufirst%3DI.%26aulast%3DSibiril%26aufirst%3DY.%26aulast%3DGomez%26aufirst%3DS.%26aulast%3Dde%2BSmedt%26aufirst%3DA.%26aulast%3DGribaldo%26aufirst%3DL.%26atitle%3DApplication%2520of%2520human%2520CFU-Mk%2520assay%2520to%2520predict%2520potential%2520thrombocytotoxicity%2520of%2520drugs%26jtitle%3DToxicol.%2520In%2520Vitro%26date%3D2009%26volume%3D23%26spage%3D194%26epage%3D200%26doi%3D10.1016%2Fj.tiv.2008.11.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Popow, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnhof, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Covini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dank, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmaschitz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greb, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karolyi-Özguer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koegl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConnell, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rieger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinnenthal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roessler, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrenk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spina, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steurer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trainor, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traxler, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wieshofer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoephel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ettmayer, P.</span></span> <span> </span><span class="NLM_article-title">Highly selective PTK2 proteolysis targeting chimeras to probe focal adhesion kinase scaffolding functions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">2508</span>– <span class="NLM_lpage">2520</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01826</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01826" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisl2ls78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=2508-2520&author=J.+Popowauthor=H.+Arnhofauthor=G.+Baderauthor=H.+Bergerauthor=A.+Ciulliauthor=D.+Coviniauthor=C.+Dankauthor=T.+Gmaschitzauthor=P.+Grebauthor=J.+Karolyi-%C3%96zguerauthor=M.+Koeglauthor=D.+B.+McConnellauthor=M.+Pearsonauthor=M.+Riegerauthor=J.+Rinnenthalauthor=V.+Roesslerauthor=A.+Schrenkauthor=M.+Spinaauthor=S.+Steurerauthor=N.+Trainorauthor=E.+Traxlerauthor=C.+Wieshoferauthor=A.+Zoephelauthor=P.+Ettmayer&title=Highly+selective+PTK2+proteolysis+targeting+chimeras+to+probe+focal+adhesion+kinase+scaffolding+functions&doi=10.1021%2Facs.jmedchem.8b01826"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40aR"><div class="casContent"><span class="casTitleNuber">40a</span><div class="casTitle"><span class="NLM_cas:atitle">Highly Selective PTK2 Proteolysis Targeting Chimeras to Probe Focal Adhesion Kinase Scaffolding Functions</span></div><div class="casAuthors">Popow, Johannes; Arnhof, Heribert; Bader, Gerd; Berger, Helmut; Ciulli, Alessio; Covini, David; Dank, Christian; Gmaschitz, Teresa; Greb, Peter; Karolyi-Oezguer, Jale; Koegl, Manfred; McConnell, Darryl B.; Pearson, Mark; Rieger, Maria; Rinnenthal, Joerg; Roessler, Vanessa; Schrenk, Andreas; Spina, Markus; Steurer, Steffen; Trainor, Nicole; Traxler, Elisabeth; Wieshofer, Corinna; Zoephel, Andreas; Ettmayer, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2508-2520</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Focal adhesion tyrosine kinase (PTK2) is often overexpressed in human hepatocellular carcinoma (HCC), and several reports have linked PTK2 depletion and/or pharmacol. inhibition to reduced tumorigenicity.  However, the clin. relevance of targeting PTK2 still remains to be proven.  Here, we present two highly selective and functional PTK2 proteolysis-targeting chimeras utilizing von Hippel-Lindau and cereblon ligands to hijack E3 ligases for PTK2 degrdn.  BI-3663 (cereblon-based) degrades PTK2 with a median DC50 of 30 nM to >80% across a panel of 11 HCC cell lines.  Despite effective PTK2 degrdn., these compds. did not phenocopy the reported antiproliferative effects of PTK2 depletion in any of the cell lines tested.  By disclosing these compds., we hope to provide valuable tools for the study of PTK2 degrdn. across different biol. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsL2R_QBrfk7Vg90H21EOLACvtfcHk0liaR9G-ZL0Oqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisl2ls78%253D&md5=f0ad27d3ff46ab12e8a90e8d40a6f3c6</span></div><a href="/servlet/linkout?suffix=cit40a&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01826%26sid%3Dliteratum%253Aachs%26aulast%3DPopow%26aufirst%3DJ.%26aulast%3DArnhof%26aufirst%3DH.%26aulast%3DBader%26aufirst%3DG.%26aulast%3DBerger%26aufirst%3DH.%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DCovini%26aufirst%3DD.%26aulast%3DDank%26aufirst%3DC.%26aulast%3DGmaschitz%26aufirst%3DT.%26aulast%3DGreb%26aufirst%3DP.%26aulast%3DKarolyi-%25C3%2596zguer%26aufirst%3DJ.%26aulast%3DKoegl%26aufirst%3DM.%26aulast%3DMcConnell%26aufirst%3DD.%2BB.%26aulast%3DPearson%26aufirst%3DM.%26aulast%3DRieger%26aufirst%3DM.%26aulast%3DRinnenthal%26aufirst%3DJ.%26aulast%3DRoessler%26aufirst%3DV.%26aulast%3DSchrenk%26aufirst%3DA.%26aulast%3DSpina%26aufirst%3DM.%26aulast%3DSteurer%26aufirst%3DS.%26aulast%3DTrainor%26aufirst%3DN.%26aulast%3DTraxler%26aufirst%3DE.%26aulast%3DWieshofer%26aufirst%3DC.%26aulast%3DZoephel%26aufirst%3DA.%26aulast%3DEttmayer%26aufirst%3DP.%26atitle%3DHighly%2520selective%2520PTK2%2520proteolysis%2520targeting%2520chimeras%2520to%2520probe%2520focal%2520adhesion%2520kinase%2520scaffolding%2520functions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D2508%26epage%3D2520%26doi%3D10.1021%2Facs.jmedchem.8b01826" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit40b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">impact of target warhead and linkage vector on inducing protein degradation: Comparison of bromodomain and extra-terminal (BET) degraders derived from triazolodiazepine (JQ1) and tetrahydroquinoline (I-BET726) BET inhibitor scaffolds</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">504</span>– <span class="NLM_lpage">513</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01912</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01912" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpslSktL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=504-513&author=K.-H.+Chanauthor=M.+Zengerleauthor=A.+Testaauthor=A.+Ciulli&title=impact+of+target+warhead+and+linkage+vector+on+inducing+protein+degradation%3A+Comparison+of+bromodomain+and+extra-terminal+%28BET%29+degraders+derived+from+triazolodiazepine+%28JQ1%29+and+tetrahydroquinoline+%28I-BET726%29+BET+inhibitor+scaffolds&doi=10.1021%2Facs.jmedchem.6b01912"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40bR"><div class="casContent"><span class="casTitleNuber">40b</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of Target Warhead and Linkage Vector on Inducing Protein Degradation: Comparison of Bromodomain and Extra-Terminal (BET) Degraders Derived from Triazolodiazepine (JQ1) and Tetrahydroquinoline (I-BET726) BET Inhibitor Scaffolds</span></div><div class="casAuthors">Chan, Kwok-Ho; Zengerle, Michael; Testa, Andrea; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">504-513</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The design of proteolysis-targeting chimeras (PROTACs) is a powerful small-mol. approach for inducing protein degrdn.  PROTACs conjugate a target warhead to an E3 ubiquitin ligase ligand via a linker.  Here the authors examd. the impact of derivatizing two different BET bromodomain inhibitors, triazolodiazepine JQ1 and the more potent tetrahydroquinoline I-BET726, via distinct exit vectors, using different polyethylene glycol linkers to VHL ligand VH032.  Triazolodiazepine PROTACs exhibited pos. cooperativities of ternary complex formation and were more potent degraders than tetrahydroquinoline compds., which showed neg. cooperativities instead.  Marked dependency on linker length was obsd. for BET-degrading and cMyc-driven antiproliferative activities in acute myeloid leukemia cell lines.  This work exemplifies as a cautionary tale how a more potent inhibitor does not necessarily generate more potent PROTACs and underscores the key roles played by the conjugation.  The provided insights and framework for structure-activity relationships of bivalent degraders are anticipated to have wide future applicability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvEL41TefwTbVg90H21EOLACvtfcHk0liaR9G-ZL0Oqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpslSktL4%253D&md5=c3bfd9377d8e7805ba962cd0180431af</span></div><a href="/servlet/linkout?suffix=cit40b&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01912&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01912%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DK.-H.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3Dimpact%2520of%2520target%2520warhead%2520and%2520linkage%2520vector%2520on%2520inducing%2520protein%2520degradation%253A%2520Comparison%2520of%2520bromodomain%2520and%2520extra-terminal%2520%2528BET%2529%2520degraders%2520derived%2520from%2520triazolodiazepine%2520%2528JQ1%2529%2520and%2520tetrahydroquinoline%2520%2528I-BET726%2529%2520BET%2520inhibitor%2520scaffolds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D504%26epage%3D513%26doi%3D10.1021%2Facs.jmedchem.6b01912" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amara, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mashiach, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amar, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krief, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spieser, S. A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bottomley, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aharoni, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meijler, M. M.</span></span> <span> </span><span class="NLM_article-title">Covalent inhibition of bacterial quorum sensing</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>131</i></span>,  <span class="NLM_fpage">10610</span>– <span class="NLM_lpage">10619</span>, <span class="refDoi"> DOI: 10.1021/ja903292v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja903292v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BD1MXot1ejtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2009&pages=10610-10619&author=N.+Amaraauthor=R.+Mashiachauthor=D.+Amarauthor=P.+Kriefauthor=S.+A.+H.+Spieserauthor=M.+J.+Bottomleyauthor=A.+Aharoniauthor=M.+M.+Meijler&title=Covalent+inhibition+of+bacterial+quorum+sensing&doi=10.1021%2Fja903292v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent Inhibition of Bacterial Quorum Sensing</span></div><div class="casAuthors">Amara, Neri; Mashiach, Roi; Amar, Dotan; Krief, Pnina; Spieser, Stephane A. H.; Bottomley, Matthew J.; Aharoni, Amir; Meijler, Michael M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">10610-10619</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Chem. coordination of gene expression among bacteria as a function of population d. is regulated by a mechanism known as 'quorum sensing' (QS).  QS in Pseudomonas aeruginosa, an opportunistic pathogen that causes disease in immunocompromised patients, is mediated by binding of the transcriptional activator, LasR, to its ligand, 3-oxo-C12-HSL, leading to population-wide secretion of virulence factors and biofilm formation.  We have targeted QS in P. aeruginosa with a set of electrophilic probes designed to covalently bind Cys79 in the LasR binding pocket, leading to specific inhibition of QS-regulated gene expression and concomitant redn. of virulence factor secretion and biofilm formation.  This first example of covalent modification of a QS receptor provides a new tool to study mol. mechanisms of bacterial group behavior and could lead to new strategies for targeting bacterial virulence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeWMbH025eh7Vg90H21EOLACvtfcHk0lha5oNoxI_CHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXot1ejtLw%253D&md5=4f58a662d8ec92f6d44d920d53f2906c</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fja903292v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja903292v%26sid%3Dliteratum%253Aachs%26aulast%3DAmara%26aufirst%3DN.%26aulast%3DMashiach%26aufirst%3DR.%26aulast%3DAmar%26aufirst%3DD.%26aulast%3DKrief%26aufirst%3DP.%26aulast%3DSpieser%26aufirst%3DS.%2BA.%2BH.%26aulast%3DBottomley%26aufirst%3DM.%2BJ.%26aulast%3DAharoni%26aufirst%3DA.%26aulast%3DMeijler%26aufirst%3DM.%2BM.%26atitle%3DCovalent%2520inhibition%2520of%2520bacterial%2520quorum%2520sensing%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2009%26volume%3D131%26spage%3D10610%26epage%3D10619%26doi%3D10.1021%2Fja903292v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delproposto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of ARD-69 as a highly potent proteolysis targeting chimera (PROTAC) degrader of androgen receptor (AR) for the treatment of prostate cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">941</span>– <span class="NLM_lpage">964</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01631</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01631" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlvFCjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=941-964&author=X.+Hanauthor=C.+Wangauthor=C.+Qinauthor=W.+Xiangauthor=E.+Fernandez-Salasauthor=C.-Y.+Yangauthor=M.+Wangauthor=L.+Zhaoauthor=T.+Xuauthor=K.+Chinnaswamyauthor=J.+Delpropostoauthor=J.+Stuckeyauthor=S.+Wang&title=Discovery+of+ARD-69+as+a+highly+potent+proteolysis+targeting+chimera+%28PROTAC%29+degrader+of+androgen+receptor+%28AR%29+for+the+treatment+of+prostate+cancer&doi=10.1021%2Facs.jmedchem.8b01631"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of ARD-69 as a highly potent proteolysis targeting chimera (PROTAC) degrader of androgen receptor (AR) for the treatment of prostate cancer</span></div><div class="casAuthors">Han, Xin; Wang, Chao; Qin, Chong; Xiang, Weiguo; Fernandez-Salas, Ester; Yang, Chao-Yie; Wang, Mi; Zhao, Lijie; Xu, Tianfeng; Chinnaswamy, Krishnapriya; Delproposto, James; Stuckey, Jeanne; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">941-964</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report herein the discovery of highly potent PROTAC degraders of androgen receptor (AR), as exemplified by IARD-69.  I induces degrdn. of AR protein in AR-pos. prostate cancer cell lines in a dose- and time-dependent manner.  I achieves DC50 values of 0.86, 0.76, and 10.4 nM in LNCaP, VCaP, and 22Rv1 AR+ prostate cancer cell lines, resp.  I is capable of reducing the AR protein level by >95% in these prostate cancer cell lines and effectively suppressing AR-regulated gene expression.  I potently inhibits cell growth in these AR-pos. prostate cancer cell lines and is >100 times more potent than AR antagonists.  A single dose of I effectively reduces the level of AR protein in xenograft tumor tissue in mice.  Further optimization of I may ultimately lead to a new therapy for AR+, castration-resistant prostate cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTlOD69tfhtbVg90H21EOLACvtfcHk0lha5oNoxI_CHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlvFCjtQ%253D%253D&md5=2376573fb1a1e19c517d64c70199700a</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01631&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01631%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DQin%26aufirst%3DC.%26aulast%3DXiang%26aufirst%3DW.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DT.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DDelproposto%26aufirst%3DJ.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520ARD-69%2520as%2520a%2520highly%2520potent%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%2520degrader%2520of%2520androgen%2520receptor%2520%2528AR%2529%2520for%2520the%2520treatment%2520of%2520prostate%2520cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D941%26epage%3D964%26doi%3D10.1021%2Facs.jmedchem.8b01631" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kadokura, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urashima, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sekine, M.</span></span> <span> </span><span class="NLM_article-title">Efficient synthesis of γ-methyl-capped guanosine 5′-triphosphate as a 5′-terminal unique structure of U6 RNA via a new triphosphate bond formation involving activation of methyl phosphorimidazolidate using ZnCl<sub>2</sub> as a catalyst in DMF under anhydrous conditions</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">8359</span>– <span class="NLM_lpage">8362</span>, <span class="refDoi"> DOI: 10.1016/S0040-4039(97)10263-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1016%2FS0040-4039%2897%2910263-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADyaK2sXnslekur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1997&pages=8359-8362&author=M.+Kadokuraauthor=T.+Wadaauthor=C.+Urashimaauthor=M.+Sekine&title=Efficient+synthesis+of+%CE%B3-methyl-capped+guanosine+5%E2%80%B2-triphosphate+as+a+5%E2%80%B2-terminal+unique+structure+of+U6+RNA+via+a+new+triphosphate+bond+formation+involving+activation+of+methyl+phosphorimidazolidate+using+ZnCl2+as+a+catalyst+in+DMF+under+anhydrous+conditions&doi=10.1016%2FS0040-4039%2897%2910263-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient synthesis of γ-methyl-capped guanosine 5'-triphosphate as a 5'-terminal unique structure of U6 RNA via a new triphosphate bond formation involving activation of methyl phosphorimidazolidate using ZnCl2 as a catalyst in DMF under anhydrous conditions</span></div><div class="casAuthors">Kadokura, Michinori; Wada, Takeshi; Urashima, Chihiro; Sekine, Mitsuo</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">8359-8362</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">P1-Me P3-(5'-guanosyl) triphosphate (CH3pppG), which exists as a unique structure at the 5'-terminal region of U6 RNA, was synthesized in an excellent yield by Me phosphorimidazolidate with the bis(tetrabutylammonium) salt of GDP in the presence of ZnCl2 as an effective catalyst in dry DMF.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrv2xGEhgGgYLVg90H21EOLACvtfcHk0lha5oNoxI_CHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXnslekur4%253D&md5=2baad1e3cfefaed7bb985ba5ef93fa6a</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2FS0040-4039%2897%2910263-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0040-4039%252897%252910263-5%26sid%3Dliteratum%253Aachs%26aulast%3DKadokura%26aufirst%3DM.%26aulast%3DWada%26aufirst%3DT.%26aulast%3DUrashima%26aufirst%3DC.%26aulast%3DSekine%26aufirst%3DM.%26atitle%3DEfficient%2520synthesis%2520of%2520%25CE%25B3-methyl-capped%2520guanosine%25205%25E2%2580%25B2-triphosphate%2520as%2520a%25205%25E2%2580%25B2-terminal%2520unique%2520structure%2520of%2520U6%2520RNA%2520via%2520a%2520new%2520triphosphate%2520bond%2520formation%2520involving%2520activation%2520of%2520methyl%2520phosphorimidazolidate%2520using%2520ZnCl2%2520as%2520a%2520catalyst%2520in%2520DMF%2520under%2520anhydrous%2520conditions%26jtitle%3DTetrahedron%2520Lett.%26date%3D1997%26volume%3D38%26spage%3D8359%26epage%3D8362%26doi%3D10.1016%2FS0040-4039%2897%2910263-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitehead, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammer, R. P.</span></span> <span> </span><span class="NLM_article-title">Application of in situ silylation for improved, convenient preparation of fluorenylmethoxycarbonyl (Fmoc)-protected phosphinate amino acids</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">3116</span>– <span class="NLM_lpage">3118</span>, <span class="refDoi"> DOI: 10.1021/jo070266p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo070266p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BD2sXktVygsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2007&pages=3116-3118&author=S.+Liauthor=J.+K.+Whiteheadauthor=R.+P.+Hammer&title=Application+of+in+situ+silylation+for+improved%2C+convenient+preparation+of+fluorenylmethoxycarbonyl+%28Fmoc%29-protected+phosphinate+amino+acids&doi=10.1021%2Fjo070266p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Application of in Situ Silylation for Improved, Convenient Preparation of Fluorenylmethoxycarbonyl (Fmoc)-Protected Phosphinate Amino Acids</span></div><div class="casAuthors">Li, Shunzi; Whitehead, John K.; Hammer, Robert P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3116-3118</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A convenient and efficient method has been developed for the prepn. of 9-fluorenylmethoxycarbonyl (Fmoc)-protected 1-aminoalkylphosphinic acids.  Reproducible procedures for the synthesis and purifn. of free α-amino H-phosphinates are provided.  Protection of free amino phosphinates as the N-Fmoc deriv. was achieved by in situ trimethylsilylation of aminoalkylphosphinic acids, which then reacted with Fmoc-Cl to provide corresponding products in excellent yields and in high purity after simple extractive isolation.  Mechanistic aspects of the silylation are discussed, and the application of the procedure to another class of amino phosphorus acids is presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFCbe8r82ohbVg90H21EOLACvtfcHk0liDaqXh8o5SvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXktVygsbo%253D&md5=705946bb116c94961f725d1e32bb91eb</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fjo070266p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo070266p%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DS.%26aulast%3DWhitehead%26aufirst%3DJ.%2BK.%26aulast%3DHammer%26aufirst%3DR.%2BP.%26atitle%3DApplication%2520of%2520in%2520situ%2520silylation%2520for%2520improved%252C%2520convenient%2520preparation%2520of%2520fluorenylmethoxycarbonyl%2520%2528Fmoc%2529-protected%2520phosphinate%2520amino%2520acids%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2007%26volume%3D72%26spage%3D3116%26epage%3D3118%26doi%3D10.1021%2Fjo070266p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Flygare, J.</span>; <span class="NLM_string-name">Gunzner-Toste, J.</span>; <span class="NLM_string-name">Pillow, T.</span>; <span class="NLM_string-name">Safina, B.</span>; <span class="NLM_string-name">Verma, V.</span>; <span class="NLM_string-name">Wei, B.</span>; <span class="NLM_string-name">Zhao, G.</span>; <span class="NLM_string-name">Staben, L.</span></span> <span> </span><span class="NLM_article-title">Peptidomimetic Compounds and Antibody-conjugates Thereof</span>. International patent application, <span class="NLM_patent">WO2015095223</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=J.+Flygare&author=J.+Gunzner-Toste&author=T.+Pillow&author=B.+Safina&author=V.+Verma&author=B.+Wei&author=G.+Zhao&author=L.+Staben&title=Peptidomimetic+Compounds+and+Antibody-conjugates+Thereof"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DFlygare%26aufirst%3DJ.%26atitle%3DPeptidomimetic%2520Compounds%2520and%2520Antibody-conjugates%2520Thereof%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mondal, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinney, K. G.</span></span> <span> </span><span class="NLM_article-title">Improved methodology for the synthesis of a cathepsin B cleavable dipeptide linker, widely used in antibody-drug conjugate research</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">3594</span>– <span class="NLM_lpage">3599</span>, <span class="refDoi"> DOI: 10.1016/j.tetlet.2018.08.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1016%2Fj.tetlet.2018.08.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=31156276" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1enu7%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2018&pages=3594-3599&author=D.+Mondalauthor=J.+Fordauthor=K.+G.+Pinney&title=Improved+methodology+for+the+synthesis+of+a+cathepsin+B+cleavable+dipeptide+linker%2C+widely+used+in+antibody-drug+conjugate+research&doi=10.1016%2Fj.tetlet.2018.08.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Improved methodology for the synthesis of a cathepsin b cleavable dipeptide linker, widely used in antibody-drug conjugate research</span></div><div class="casAuthors">Mondal, Deboprosad; Ford, Jacob; Pinney, Kevin G.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">3594-3599</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Antibody-drug conjugates (ADCs) represent an emerging class of biopharmaceutical agents that deliver highly potent anticancer agents (payloads) selectively to tumors or components assocd. with the tumor microenvironment.  The linker, responsible for the connection between the antibody and payload, is a crucial component of ADCs.  In certain examples the linker is composed of a cleavable short peptide which imparts an addnl. aspect of selectivity.  Esp. prevalent is the cathepsin B cleavable Mc-Val-Cit-PABOH linker utilized in many pre-clin. ADC candidates, as well as the FDA approved ADC ADCETRIS (brentuximab vedotin).  An alternative route for the synthesis of the cathepsin B cleavable Mc-Val-Cit-PABOH linker is reported herein that involved six steps from L-Citrulline and proceeded with a 50% overall yield.  In this modified route, the spacer (a para-aminobenzyl alc. moiety) was incorporated via HATU coupling followed by dipeptide formation.  Importantly, this route avoided undesirable epimerization and proceeded with improved overall yield.  Utilizing this methodol., a drug-linker construct incorporating a potent small-mol. inhibitor of tubulin polymn. (referred to as KGP05), was synthesized as a representative example.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmlphUMG1MVbVg90H21EOLACvtfcHk0liDaqXh8o5SvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1enu7%252FM&md5=60399fc5e1c2755e60f6d26c52de2b0b</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.tetlet.2018.08.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tetlet.2018.08.021%26sid%3Dliteratum%253Aachs%26aulast%3DMondal%26aufirst%3DD.%26aulast%3DFord%26aufirst%3DJ.%26aulast%3DPinney%26aufirst%3DK.%2BG.%26atitle%3DImproved%2520methodology%2520for%2520the%2520synthesis%2520of%2520a%2520cathepsin%2520B%2520cleavable%2520dipeptide%2520linker%252C%2520widely%2520used%2520in%2520antibody-drug%2520conjugate%2520research%26jtitle%3DTetrahedron%2520Lett.%26date%3D2018%26volume%3D59%26spage%3D3594%26epage%3D3599%26doi%3D10.1016%2Fj.tetlet.2018.08.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Polakis, P.</span>; <span class="NLM_string-name">Dragovich, P.</span>; <span class="NLM_string-name">Pillow, T. H.</span>; <span class="NLM_string-name">Yu, S.-F.</span></span> <span> </span><span class="NLM_article-title">Antibodies and Immunoconjugates</span>. International patent application, <span class="NLM_patent">WO2016205176</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=P.+Polakis&author=P.+Dragovich&author=T.+H.+Pillow&author=S.-F.+Yu&title=Antibodies+and+Immunoconjugates"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DPolakis%26aufirst%3DP.%26atitle%3DAntibodies%2520and%2520Immunoconjugates%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, S.</span>; <span class="NLM_string-name">Wang, K.-P.</span></span> <span> </span><span class="NLM_article-title">ADCs With Thiol Multiplex Linkers</span>. International patent application, <span class="NLM_patent">WO2020132655</span>, <span class="NLM_year">2020</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=S.+Jeffrey&author=K.-P.+Wang&title=ADCs+With+Thiol+Multiplex+Linkers"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DJeffrey%26aufirst%3DS.%26atitle%3DADCs%2520With%2520Thiol%2520Multiplex%2520Linkers%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit48b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Le Joncour, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martins, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puhka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isola, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salmikangas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laakkonen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joensuu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barok, M.</span></span> <span> </span><span class="NLM_article-title">A movel anti-HER2 antibody-drug conjugate XMT-1522 for HER2-positive breast and gastric cancers resistant to trastuzumab emtansine</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1721</span>– <span class="NLM_lpage">1730</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-19-0207</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1158%2F1535-7163.MCT-19-0207" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=31292166" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BB3cXmsFKlug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=1721-1730&author=V.+Le+Joncourauthor=A.+Martinsauthor=M.+Puhkaauthor=J.+Isolaauthor=M.+Salmikangasauthor=P.+Laakkonenauthor=H.+Joensuuauthor=M.+Barok&title=A+movel+anti-HER2+antibody-drug+conjugate+XMT-1522+for+HER2-positive+breast+and+gastric+cancers+resistant+to+trastuzumab+emtansine&doi=10.1158%2F1535-7163.MCT-19-0207"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48bR"><div class="casContent"><span class="casTitleNuber">48b</span><div class="casTitle"><span class="NLM_cas:atitle">A novel anti-HER2 antibody-drug conjugate XMT-1522 for HER2-positive breast and gastric cancers resistant to trastuzumab emtansine</span></div><div class="casAuthors">Le Joncour, Vadim; Martins, Ana; Puhka, Maija; Isola, Jorma; Salmikangas, Marko; Laakkonen, Pirjo; Joensuu, Heikki; Barok, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1721-1730</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Most patients with HER2-pos. breast or gastric cancer exhibit primary or acquired resistance to trastuzumab emtansine (T-DM1), and such patients may have limited therapeutic options.  XMT-1522 is a novel anti-HER2 antibody-drug conjugate.  We compared XMT-1522 to T-DM1 in preclin. models.  The effects of XMT-1522 and T-DM1 on cell survival and apoptosis were compared in six HER2-pos. breast cancer or gastric cancer cell lines, of which three lines were T-DM1-sensitive (N-87, OE-19, JIMT-1) and three T-DM1-resistant (RN-87, ROE-19, SNU-216).  We compared these agents also in the HER2-neg. breast cancer cell line MCF-7, and in mouse RN-87 and JIMT-1 xenograft models.  Cell survival was assessed using the AlamarBlue method and apoptosis with the Caspase-Glo 3/7 method.  XMT-1522 inhibited the growth of all six HER2-pos. cell lines.  The proportions of cells that survived XMT-1522 treatment were smaller as compared with T-DM1, particularly in the T-DM1-resistant cell lines.  XMT-1522 induced more cell apoptosis compared with T-DM1.  While RN-87 and JIMT-1 xenograft tumors progressed on T-DM1 treatment, all tumors responded to XMT-1522, and all but one tumor disappeared during the XMT-1522 treatment.  XMT-1522 had a strong antitumor effect on RN-87 and JIMT-1 xenografts that progressed on T-DM1.  We conclude that XMT-1522 was effective in HER2-pos. breast cancer and gastric cancer cell lines resistant to T-DM1, and in xenograft models resistant to T-DM1.  The results support the testing of XMT-1522 in clin. trials in patients with HER2-pos. cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_Pci-NaVO6bVg90H21EOLACvtfcHk0liDaqXh8o5SvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXmsFKlug%253D%253D&md5=71fd86427ea0bac472f679076eb4c680</span></div><a href="/servlet/linkout?suffix=cit48b&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-19-0207&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-19-0207%26sid%3Dliteratum%253Aachs%26aulast%3DLe%2BJoncour%26aufirst%3DV.%26aulast%3DMartins%26aufirst%3DA.%26aulast%3DPuhka%26aufirst%3DM.%26aulast%3DIsola%26aufirst%3DJ.%26aulast%3DSalmikangas%26aufirst%3DM.%26aulast%3DLaakkonen%26aufirst%3DP.%26aulast%3DJoensuu%26aufirst%3DH.%26aulast%3DBarok%26aufirst%3DM.%26atitle%3DA%2520movel%2520anti-HER2%2520antibody-drug%2520conjugate%2520XMT-1522%2520for%2520HER2-positive%2520breast%2520and%2520gastric%2520cancers%2520resistant%2520to%2520trastuzumab%2520emtansine%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2019%26volume%3D18%26spage%3D1721%26epage%3D1730%26doi%3D10.1158%2F1535-7163.MCT-19-0207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit48c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Viricel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fournet, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beaumel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perrial, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papotd, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumontet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joseph, B.</span></span> <span> </span><span class="NLM_article-title">Monodisperse polysarcosine-based highly-loaded antibody-drug conjugates</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">4048</span>– <span class="NLM_lpage">4053</span>, <span class="refDoi"> DOI: 10.1039/C9SC00285E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1039%2FC9SC00285E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=31015945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1MXkt1Wqurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=4048-4053&author=W.+Viricelauthor=G.+Fournetauthor=S.+Beaumelauthor=E.+Perrialauthor=S.+Papotdauthor=C.+Dumontetauthor=B.+Joseph&title=Monodisperse+polysarcosine-based+highly-loaded+antibody-drug+conjugates&doi=10.1039%2FC9SC00285E"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48cR"><div class="casContent"><span class="casTitleNuber">48c</span><div class="casTitle"><span class="NLM_cas:atitle">Monodisperse polysarcosine-based highly-loaded antibody-drug conjugates</span></div><div class="casAuthors">Viricel, Warren; Fournet, Guy; Beaumel, Sabine; Perrial, Emeline; Papot, Sebastien; Dumontet, Charles; Joseph, Benoit</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4048-4053</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Antibody-drug conjugates (ADCs) convey highly potent anticancer drugs to antigen-expressing tumor cells, thereby sparing healthy tissues throughout the body.  Pharmacokinetics and tolerability of ADCs are predominantly influenced by the drug-antibody ratio (DAR) of the conjugates, which is to-date limited to a value of 3-4 drugs per antibody in Antibody-drug conjugates under clin. investigations.  Here, we report the synthesis of monodisperse (i.e. discrete) polysarcosine compds. and their use as a hydrophobicity masking entity for the construction of highly-loaded homogeneous glucuronidase-responsive antibody-drug conjugates (ADCs).  The highly hydrophilic drug-linker platform described herein improves drug-loading, physicochem. properties, pharmacokinetics and in vivo antitumor efficacy of the resulting conjugates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyJWZGJpnUOrVg90H21EOLACvtfcHk0lhzjil6h0wvQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXkt1Wqurk%253D&md5=3f6556a5f22ba298ae4d504a68bca3c0</span></div><a href="/servlet/linkout?suffix=cit48c&amp;dbid=16384&amp;doi=10.1039%2FC9SC00285E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC9SC00285E%26sid%3Dliteratum%253Aachs%26aulast%3DViricel%26aufirst%3DW.%26aulast%3DFournet%26aufirst%3DG.%26aulast%3DBeaumel%26aufirst%3DS.%26aulast%3DPerrial%26aufirst%3DE.%26aulast%3DPapotd%26aufirst%3DS.%26aulast%3DDumontet%26aufirst%3DC.%26aulast%3DJoseph%26aufirst%3DB.%26atitle%3DMonodisperse%2520polysarcosine-based%2520highly-loaded%2520antibody-drug%2520conjugates%26jtitle%3DChem.%2520Sci.%26date%3D2019%26volume%3D10%26spage%3D4048%26epage%3D4053%26doi%3D10.1039%2FC9SC00285E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit48d"><span><span class="NLM_label">(d) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Neumann, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olivas, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loftus, L. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neale, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nix, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pittman, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simmons, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ulrich, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waight, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaval, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyon, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senter, P. D.</span></span> <span> </span><span class="NLM_article-title">Targeted delivery of cytotoxic NAMPT inhibitors using antibody-drug conjugates</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">2633</span>– <span class="NLM_lpage">2642</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-18-0643</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1158%2F1535-7163.MCT-18-0643" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48d&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=30242091" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjs1elsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=2633-2642&author=C.+S.+Neumannauthor=K.+C.+Olivasauthor=M.+E.+Andersonauthor=J.+H.+Cochranauthor=S.+Jinauthor=F.+Liauthor=L.+V.+Loftusauthor=D.+W.+Meyerauthor=J.+Nealeauthor=J.+C.+Nixauthor=P.+G.+Pittmanauthor=J.+K.+Simmonsauthor=M.+L.+Ulrichauthor=A.+B.+Waightauthor=A.+Wongauthor=M.+C.+Zavalauthor=W.+Zengauthor=R.+P.+Lyonauthor=P.+D.+Senter&title=Targeted+delivery+of+cytotoxic+NAMPT+inhibitors+using+antibody-drug+conjugates&doi=10.1158%2F1535-7163.MCT-18-0643"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48dR"><div class="casContent"><span class="casTitleNuber">48d</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted delivery of cytotoxic NAMPT inhibitors using antibody-drug conjugates</span></div><div class="casAuthors">Neumann, Christopher S.; Olivas, Kathleen C.; Anderson, Martha E.; Cochran, Julia H.; Jin, Steven; Li, Fu; Loftus, Luke V.; Meyer, David W.; Neale, Jason; Nix, Jay C.; Pittman, Paul G.; Simmons, Jessica K.; Ulrich, Michelle L.; Waight, Andrew B.; Wong, Abbie; Zaval, Margo C.; Zeng, Weiping; Lyon, Robert P.; Senter, Peter D.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2633-2642</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Antibody-drug conjugates (ADCs) are a therapeutic modality that enables the targeted delivery of cytotoxic drugs to cancer cells.  Identification of active payloads with unique mechanisms of action is a key aim of research efforts in the field.  Herein, we report the development of inhibitors of nicotinamide phosphoribosyltransferase (NAMPT) as a novel payload for ADC technol.  NAMPT is a component of a salvage biosynthetic pathway for NAD, and inhibition of this enzyme results in disruption of primary cellular metab. leading to cell death.  Through derivatization of the prototypical NAMPT inhibitor FK-866, we identified potent analogs with chem. functionality that enables the synthesis of hydrophilic enzyme-cleavable drug linkers.  The resulting ADCs displayed NAD depletion in both cell-based assays and tumor xenografts.  Antitumor efficacy is demonstrated in five mouse xenograft models using ADCs directed to indication-specific antigens.  In rat toxicol. models, a nonbinding control ADC was tolerated at >10-fold the typical efficacious dose used in xenografts.  Moderate, reversible hematol. effects were obsd. with ADCs in rats, but there was no evidence for the retinal and cardiac toxicities reported for small-mol. inhibitors.  These findings introduce NAMPT inhibitors as active and well-tolerated payloads for ADCs with promise to improve the therapeutic window of NAMPT inhibition and enable application in clin. settings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpx2rsX00yD6LVg90H21EOLACvtfcHk0lhzjil6h0wvQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjs1elsL0%253D&md5=43118eed82fe96b192cac5dca45a7b44</span></div><a href="/servlet/linkout?suffix=cit48d&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-18-0643&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-18-0643%26sid%3Dliteratum%253Aachs%26aulast%3DNeumann%26aufirst%3DC.%2BS.%26aulast%3DOlivas%26aufirst%3DK.%2BC.%26aulast%3DAnderson%26aufirst%3DM.%2BE.%26aulast%3DCochran%26aufirst%3DJ.%2BH.%26aulast%3DJin%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DLoftus%26aufirst%3DL.%2BV.%26aulast%3DMeyer%26aufirst%3DD.%2BW.%26aulast%3DNeale%26aufirst%3DJ.%26aulast%3DNix%26aufirst%3DJ.%2BC.%26aulast%3DPittman%26aufirst%3DP.%2BG.%26aulast%3DSimmons%26aufirst%3DJ.%2BK.%26aulast%3DUlrich%26aufirst%3DM.%2BL.%26aulast%3DWaight%26aufirst%3DA.%2BB.%26aulast%3DWong%26aufirst%3DA.%26aulast%3DZaval%26aufirst%3DM.%2BC.%26aulast%3DZeng%26aufirst%3DW.%26aulast%3DLyon%26aufirst%3DR.%2BP.%26aulast%3DSenter%26aufirst%3DP.%2BD.%26atitle%3DTargeted%2520delivery%2520of%2520cytotoxic%2520NAMPT%2520inhibitors%2520using%2520antibody-drug%2520conjugates%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2018%26volume%3D17%26spage%3D2633%26epage%3D2642%26doi%3D10.1158%2F1535-7163.MCT-18-0643" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit48e"><span><span class="NLM_label">(e) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamilton, J. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pires, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, H. W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leiske, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emmerton, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waight, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senter, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyon, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, S. C.</span></span> <span> </span><span class="NLM_article-title">Glucuronide-linked antibody-tubulysin conjugates display activity in MDR+ and heterogeneous tumor models</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1752</span>– <span class="NLM_lpage">1760</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-18-0073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48e&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1158%2F1535-7163.MCT-18-0073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48e&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=29866744" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48e&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVWrtrfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=1752-1760&author=P.+J.+Burkeauthor=J.+Z.+Hamiltonauthor=T.+A.+Piresauthor=H.+W.+H.+Laiauthor=C.+I.+Leiskeauthor=K.+K.+Emmertonauthor=A.+B.+Waightauthor=P.+D.+Senterauthor=R.+P.+Lyonauthor=S.+C.+Jeffrey&title=Glucuronide-linked+antibody-tubulysin+conjugates+display+activity+in+MDR%2B+and+heterogeneous+tumor+models&doi=10.1158%2F1535-7163.MCT-18-0073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48eR"><div class="casContent"><span class="casTitleNuber">48e</span><div class="casTitle"><span class="NLM_cas:atitle">Glucuronide-Linked Antibody-Tubulysin Conjugates Display Activity in MDR+ and Heterogeneous Tumor Models</span></div><div class="casAuthors">Burke, Patrick J.; Hamilton, Joseph Z.; Pires, Thomas A.; Lai, Holden W. H.; Leiske, Christopher I.; Emmerton, Kim K.; Waight, Andrew B.; Senter, Peter D.; Lyon, Robert P.; Jeffrey, Scott C.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1752-1760</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Although antibody-drug conjugates (ADCs) find increasing applications in cancer treatment, de novo or treatment-emergent resistance mechanisms may impair clin. benefit.  Two resistance mechanisms that emerge under prolonged exposure include upregulation of transporter proteins that confer multidrug resistance (MDR+) and loss of cognate antigen expression.  Recently, we developed the quaternary ammonium linker system to expand the scope of conjugatable payloads to include tertiary amines and applied the linker to tubulysins, a highly potent class of tubulin binders that maintain activity in MDR+ cell lines.  In this work, tubulysin M, which contains an unstable acetate susceptible to enzymic hydrolysis, and two stabilized tubulysin analogs were prepd. as quaternary ammonium-linked glucuronide-linkers and assessed as ADC payloads in preclin. models.  The conjugates were potent across a panel of cancer cell lines and active in tumor xenografts, including those displaying the MDR+ phenotype.  The ADCs also demonstrated potent bystander activity in a coculture model comprised of a mixt. of antigen-pos. and -neg. cell lines, and in an antigen-heterogeneous tumor model.  Thus, the glucuronide-tubulysin drug-linkers represent a promising ADC payload class, combining conjugate potency in the presence of the MDR+ phenotype and robust activity in models of tumor heterogeneity in a structure-dependent manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpi-liIa9oi_rVg90H21EOLACvtfcHk0lhzjil6h0wvQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVWrtrfM&md5=57268b599402601821db672b6d645936</span></div><a href="/servlet/linkout?suffix=cit48e&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-18-0073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-18-0073%26sid%3Dliteratum%253Aachs%26aulast%3DBurke%26aufirst%3DP.%2BJ.%26aulast%3DHamilton%26aufirst%3DJ.%2BZ.%26aulast%3DPires%26aufirst%3DT.%2BA.%26aulast%3DLai%26aufirst%3DH.%2BW.%2BH.%26aulast%3DLeiske%26aufirst%3DC.%2BI.%26aulast%3DEmmerton%26aufirst%3DK.%2BK.%26aulast%3DWaight%26aufirst%3DA.%2BB.%26aulast%3DSenter%26aufirst%3DP.%2BD.%26aulast%3DLyon%26aufirst%3DR.%2BP.%26aulast%3DJeffrey%26aufirst%3DS.%2BC.%26atitle%3DGlucuronide-linked%2520antibody-tubulysin%2520conjugates%2520display%2520activity%2520in%2520MDR%252B%2520and%2520heterogeneous%2520tumor%2520models%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2018%26volume%3D17%26spage%3D1752%26epage%3D1760%26doi%3D10.1158%2F1535-7163.MCT-18-0073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit48f"><span><span class="NLM_label">(f) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Ogitani, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aida, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagihara, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishii, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harada, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soma, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oitate, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arakawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atsumi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayakawa, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abe, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agatsuma, T.</span></span> <span> </span><span class="NLM_article-title">DS-8201a, a novel HER2-targeting ADC with a novel DNA topoisomerase I inhibitor, demonstrates a promising antitumor efficacy with differentiation from T-DM1</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">5097</span>– <span class="NLM_lpage">5108</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-15-2822</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48f&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1158%2F1078-0432.CCR-15-2822" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48f&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=27026201" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48f&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslOhs77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2016&pages=5097-5108&author=Y.+Ogitaniauthor=T.+Aidaauthor=K.+Hagiharaauthor=J.+Yamaguchiauthor=C.+Ishiiauthor=N.+Haradaauthor=M.+Somaauthor=H.+Okamotoauthor=M.+Oitateauthor=S.+Arakawaauthor=T.+Hiraiauthor=R.+Atsumiauthor=T.+Nakadaauthor=I.+Hayakawaauthor=Y.+Abeauthor=T.+Agatsuma&title=DS-8201a%2C+a+novel+HER2-targeting+ADC+with+a+novel+DNA+topoisomerase+I+inhibitor%2C+demonstrates+a+promising+antitumor+efficacy+with+differentiation+from+T-DM1&doi=10.1158%2F1078-0432.CCR-15-2822"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48fR"><div class="casContent"><span class="casTitleNuber">48f</span><div class="casTitle"><span class="NLM_cas:atitle">DS-8201a, A Novel HER2-Targeting ADC with a Novel DNA Topoisomerase I Inhibitor, Demonstrates a Promising Antitumor Efficacy with Differentiation from T-DM1</span></div><div class="casAuthors">Ogitani, Yusuke; Aida, Tetsuo; Hagihara, Katsunobu; Yamaguchi, Junko; Ishii, Chiaki; Harada, Naoya; Soma, Masako; Okamoto, Hiromi; Oitate, Masataka; Arakawa, Shingo; Hirai, Takehiro; Atsumi, Ryo; Nakada, Takashi; Hayakawa, Ichiro; Abe, Yuki; Agatsuma, Toshinori</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5097-5108</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">An anti-HER2 antibody-drug conjugate with a novel topoisomerase I inhibitor, DS-8201a, was generated as a new antitumor drug candidate, and its preclin. pharmacol. profile was assessed.  In vitro and in vivo pharmacol. activities of DS-8201a were evaluated and compared with T-DM1 in several HER2-pos. cell lines and patient-derived xenograft (PDX) models.  The mechanism of action for the efficacy was also evaluated.  Pharmacokinetics in cynomolgus monkeys and the safety profiles in rats and cynomolgus monkeys were assessed.  DS-8201a exhibited a HER2 expression-dependent cell growth-inhibitory activity and induced tumor regression with a single dosing at more than 1 mg/kg in a HER2-pos. gastric cancer NCI-N87 model.  Binding activity to HER2 and ADCC activity of DS-8201a were comparable with unconjugated anti-HER2 antibody.  DS-8201a also showed an inhibitory activity to Akt phosphorylation.  DS-8201a induced phosphorylation of Chk1 and Histone H2A.X, the markers of DNA damage.  Pharmacokinetics and safety profiles of DS-8201a were favorable and the highest non-severely toxic dose was 30 mg/kg in cynomolgus monkeys, supporting DS-8201a as being well tolerated in humans.  DS-8201a was effective in a T-DM1-insensitive PDX model with high HER2 expression.  DS-8201a, but not T-DM1, demonstrated antitumor efficacy against several breast cancer PDX models with low HER2 expression.  DS-8201a exhibited a potent antitumor activity in a broad selection of HER2-pos. models and favorable pharmacokinetics and safety profiles.  The results demonstrate that DS-8201a will be a valuable therapy with a great potential to respond to T-DM1-insensitive HER2-pos. cancers and low HER2-expressing cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCZitEmBAqAbVg90H21EOLACvtfcHk0li4YQca2zHyTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslOhs77F&md5=3fc9a072b2ba05c80929713c843df3f5</span></div><a href="/servlet/linkout?suffix=cit48f&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-15-2822&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-15-2822%26sid%3Dliteratum%253Aachs%26aulast%3DOgitani%26aufirst%3DY.%26aulast%3DAida%26aufirst%3DT.%26aulast%3DHagihara%26aufirst%3DK.%26aulast%3DYamaguchi%26aufirst%3DJ.%26aulast%3DIshii%26aufirst%3DC.%26aulast%3DHarada%26aufirst%3DN.%26aulast%3DSoma%26aufirst%3DM.%26aulast%3DOkamoto%26aufirst%3DH.%26aulast%3DOitate%26aufirst%3DM.%26aulast%3DArakawa%26aufirst%3DS.%26aulast%3DHirai%26aufirst%3DT.%26aulast%3DAtsumi%26aufirst%3DR.%26aulast%3DNakada%26aufirst%3DT.%26aulast%3DHayakawa%26aufirst%3DI.%26aulast%3DAbe%26aufirst%3DY.%26aulast%3DAgatsuma%26aufirst%3DT.%26atitle%3DDS-8201a%252C%2520a%2520novel%2520HER2-targeting%2520ADC%2520with%2520a%2520novel%2520DNA%2520topoisomerase%2520I%2520inhibitor%252C%2520demonstrates%2520a%2520promising%2520antitumor%2520efficacy%2520with%2520differentiation%2520from%2520T-DM1%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2016%26volume%3D22%26spage%3D5097%26epage%3D5108%26doi%3D10.1158%2F1078-0432.CCR-15-2822" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1a','cit1b'],'ref2':['cit2a','cit2b','cit2c'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5a','cit5b','cit5c','cit5d','cit5e','cit5f','cit5g'],'ref6':['cit6a','cit6b','cit6c'],'ref7':['cit7a','cit7b'],'ref8':['cit8a','cit8b'],'ref9':['cit9a','cit9b','cit9c','cit9d','cit9e'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12a','cit12b'],'ref13':['cit13a','cit13b'],'ref14':['cit14a','cit14b'],'ref15':['cit15a','cit15b','cit15c'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21a','cit21b'],'ref22':['cit22a','cit22b'],'ref23':['cit23a','cit23b'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26a','cit26b'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29a','cit29b'],'ref30':['cit30a','cit30b'],'ref31':['cit31'],'ref32':['cit32a','cit32b','cit32c','cit32d'],'ref33':['cit33a','cit33b'],'ref34':['cit34a','cit34b'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38a','cit38b','cit38c'],'ref39':['cit39a','cit39b'],'ref40':['cit40a','cit40b'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48a','cit48b','cit48c','cit48d','cit48e','cit48f']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 5 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Jing Liu, He Chen, Yi Liu, Yudao Shen, Fanye Meng, H. Ümit Kaniskan, Jian Jin, <span class="NLM_string-name hlFld-ContribAuthor">Wenyi Wei</span>. </span><span class="cited-content_cbyCitation_article-title">Cancer Selective Target Degradation by Folate-Caged PROTACs. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2021,</strong> <em>143 </em>
                                    (19)
                                     , 7380-7387. <a href="https://doi.org/10.1021/jacs.1c00451" title="DOI URL">https://doi.org/10.1021/jacs.1c00451</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.1c00451&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.1c00451%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DCancer%252BSelective%252BTarget%252BDegradation%252Bby%252BFolate-Caged%252BPROTACs%26aulast%3DLiu%26aufirst%3DJing%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D13012021%26date%3D10052021%26volume%3D143%26issue%3D19%26spage%3D7380%26epage%3D7387" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Peter S. Dragovich, Thomas H. Pillow, Robert A. Blake, Jack D. Sadowsky, Emel Adaligil, Pragya Adhikari, Sunil Bhakta, Nicole Blaquiere, Jinhua Chen, Josefa dela Cruz-Chuh, Karen E. Gascoigne, Steven J. Hartman, Mingtao He, Susan Kaufman, Tracy Kleinheinz, Katherine R. Kozak, Liang Liu, Liling Liu, Qi Liu, Ying Lu, Fanwei Meng, Melinda M. Mulvihill, Aimee O’Donohue, Rebecca K. Rowntree, Leanna R. Staben, Steven T. Staben, John Wai, Jian Wang, BinQing Wei, Catherine Wilson, Jianfeng Xin, Zijin Xu, Hui Yao, Donglu Zhang, Hongyan Zhang, Hao Zhou, <span class="NLM_string-name hlFld-ContribAuthor">Xiaoyu Zhu</span>. </span><span class="cited-content_cbyCitation_article-title">Antibody-Mediated Delivery of Chimeric BRD4 Degraders. Part 1: Exploration of Antibody Linker, Payload Loading, and Payload Molecular Properties. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (5)
                                     , 2534-2575. <a href="https://doi.org/10.1021/acs.jmedchem.0c01845" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01845</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01845&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01845%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DAntibody-Mediated%252BDelivery%252Bof%252BChimeric%252BBRD4%252BDegraders.%252BPart%252B1%25253A%252BExploration%252Bof%252BAntibody%252BLinker%25252C%252BPayload%252BLoading%25252C%252Band%252BPayload%252BMolecular%252BProperties%26aulast%3DDragovich%26aufirst%3DPeter%2BS.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D21102020%26date%3D17022021%26volume%3D64%26issue%3D5%26spage%3D2534%26epage%3D2575" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jing  Liu</span>, <span class="hlFld-ContribAuthor ">Yunhua  Peng</span>, <span class="hlFld-ContribAuthor ">Wenyi  Wei</span>. </span><span class="cited-content_cbyCitation_article-title">Light-Controllable PROTACs for Temporospatial Control of Protein Degradation. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Cell and Developmental Biology</span><span> <strong>2021,</strong> <em>9 </em><a href="https://doi.org/10.3389/fcell.2021.678077" title="DOI URL">https://doi.org/10.3389/fcell.2021.678077</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fcell.2021.678077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffcell.2021.678077%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Cell%2520and%2520Developmental%2520Biology%26atitle%3DLight-Controllable%252BPROTACs%252Bfor%252BTemporospatial%252BControl%252Bof%252BProtein%252BDegradation%26aulast%3DLiu%26aufirst%3DJing%26date%3D2021%26date%3D2021%26volume%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qun  Zhou</span>, <span class="hlFld-ContribAuthor ">Josephine  Kyazike</span>, <span class="hlFld-ContribAuthor ">Ekaterina  Boudanova</span>, <span class="hlFld-ContribAuthor ">Michael  Drzyzga</span>, <span class="hlFld-ContribAuthor ">Denise  Honey</span>, <span class="hlFld-ContribAuthor ">Robert  Cost</span>, <span class="hlFld-ContribAuthor ">Lihui  Hou</span>, <span class="hlFld-ContribAuthor ">Francis  Duffieux</span>, <span class="hlFld-ContribAuthor ">Marie-Priscille  Brun</span>, <span class="hlFld-ContribAuthor ">Anna  Park</span>, <span class="hlFld-ContribAuthor ">Huawei  Qiu</span>. </span><span class="cited-content_cbyCitation_article-title">Site-Specific Antibody Conjugation to Engineered Double Cysteine Residues. </span><span class="cited-content_cbyCitation_journal-name">Pharmaceuticals</span><span> <strong>2021,</strong> <em>14 </em>
                                    (7)
                                     , 672. <a href="https://doi.org/10.3390/ph14070672" title="DOI URL">https://doi.org/10.3390/ph14070672</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ph14070672&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fph14070672%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmaceuticals%26atitle%3DSite-Specific%252BAntibody%252BConjugation%252Bto%252BEngineered%252BDouble%252BCysteine%252BResidues%26aulast%3DZhou%26aufirst%3DQun%26date%3D2021%26date%3D2021%26volume%3D14%26issue%3D7%26spage%3D672" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Louise M.  Sternicki</span>, <span class="hlFld-ContribAuthor ">Jim  Nonomiya</span>, <span class="hlFld-ContribAuthor ">Miaomiao  Liu</span>, <span class="hlFld-ContribAuthor ">Melinda M.  Mulvihill</span>, <span class="hlFld-ContribAuthor ">Ronald J.  Quinn</span>. </span><span class="cited-content_cbyCitation_article-title">Native Mass Spectrometry for the Study of PROTAC GNE‐987‐Containing Ternary Complexes. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2021,</strong> <em>18 </em><a href="https://doi.org/10.1002/cmdc.202100113" title="DOI URL">https://doi.org/10.1002/cmdc.202100113</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.202100113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.202100113%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DNative%252BMass%252BSpectrometry%252Bfor%252Bthe%252BStudy%252Bof%252BPROTAC%252BGNE%2525E2%252580%252590987%2525E2%252580%252590Containing%252BTernary%252BComplexes%26aulast%3DSternicki%26aufirst%3DLouise%2BM.%26date%3D2021%26date%3D2021%26volume%3D18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0032.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0032.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0002.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of compound <b>1</b> (JQ1) and related chimeric BRD4 degraders <b>2</b> (MZ1) and <b>3</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0003.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Structures of BRD4-binding compounds <b>4</b>–<b>7</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0004.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Crystal structure of compound <b>1</b> (cyan) bound to the BD1 bromodomain of BRD4 with the docked compound <b>4</b> (pink) superimposed. The yellow arrow indicates a vector from the docked compound <b>4</b> that protrudes away from the protein and out into the solvent. The orange circles depict the locations in compound <b>1</b> that were derivatized in our previous work to generate functional chimeric BRD4 degraders (ref <a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a>). The figure was created using the published structure: PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MXF">3MXF</a> (ref <a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0005.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Structures of chimeric BRD4-degrader <b>8</b> and corresponding linker-drugs.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0006.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Structures of chimeric BRD4 degraders <b>9</b> and <b>10</b> and corresponding linker-drugs.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0007.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Correlation between SPR half-life of the ternary complex formed among the listed chimeric degrader molecules, BRD4, and VHL and PC3-S1 BRD4 degradation pDC<sub>50</sub> values. To generate the values depicted on the <i>x</i>-axis, the SPR half-lives determined independently for each compound with the BRD4 BD1 and BD2 bromodomains (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Table S4</a>) were added together. The number next to each point indicates the compound number from <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. *BRD4 degradation properties of <b>13</b> were determined in the EOL-1 cell line. <i>R</i><sup>2</sup> = 0.19 (improves to 0.66 if compound <b>12</b> is removed from the regression analysis).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0008.jpeg" id="rightTab-gr16" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Crystal structure of the ternary complex formed among VHL, BRD4 BD1, and compound <b>9</b>. The VHL protein is shown in cyan with interacting residues depicted as cyan sticks. BRD4 BD1 is shown in green with interacting residues depicted as green sticks. Compound <b>9</b> is shown as salmon sticks. Water molecules in the vicinity of the BRD4–VHL interface are represented as red spheres. Hydrogen-bond interactions are depicted as follows: black dashed lines, intermolecular between <b>9</b> and either BRD4 or VHL; red dashed lines, intermolecular between VHL and BRD4; yellow dashed lines, intramolecular that stabilizes the conformation of VHL R107. Resolution is 2.28 Å, and PDB code is <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7KHH">7KHH</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0009.jpeg" id="rightTab-gr17" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Structures of chimeric BRD4 degraders <b>11</b> and <b>12</b> and corresponding linker-drugs.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0010.jpeg" id="rightTab-gr18" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Structures of chimeric BRD4 degraders <b>13</b>, <b>14</b>, and <b>15</b> and corresponding linker-drugs.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0011.jpeg" id="rightTab-gr19" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Graphical depiction of the BRD4 DC<sub>50</sub> and <i>S</i><sub>inf</sub> values for the conjugates described in this work (DAR6 ADCs unless otherwise indicated). The > 85% BRD4 <i>S</i><sub>inf</sub> threshold was purposefully chosen to highlight the suspected requirement for achieving antiproliferation activity (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Figure S9</a>). See the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Supporting Information</a> for a discussion of the assay error.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0012.jpeg" id="rightTab-gr20" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Graphical depiction of the PC3-S1 BRD4 DC<sub>50</sub> values for STEAP1-targeting chimeric degrader conjugates and corresponding PC3-S1 antiproliferation activities. DAR6 ADCs unless otherwise indicated. Conjugates depicted with circle and square symbols are described in ref <a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> and, for clarity purposes, are not numbered in this figure. BRD4 degradation time point was 72 h. PC3-S1 antiproliferation time point was 6 days. Conjugates residing on the y-axis above the dotted blue line were inactive in PC3-S1 antiproliferation assessments when tested to the depicted <i>y</i>-axis concentrations. The number next to each point indicates the linker-drug employed to prepare each DAR6 STEAP1 conjugate (see <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>–<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a>). The BRD4 degradation <i>S</i><sub>inf</sub> values of the conjugates are depicted by the point sizes in the figure, and the >85% BRD4 <i>S</i><sub>inf</sub> threshold was purposefully chosen to highlight the suspected requirement for achieving the antiproliferation activity (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Figure S9</a>). See the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Supporting Information</a> for a discussion of the assay error.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig12" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0013.jpeg" id="rightTab-gr21" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Structures of linker-drug <b>3a</b> and corresponding DAR6 STEAP1 ADC. The biological properties of the STEAP1<b>–3a</b> conjugate were measured in the PC3-S1 cell line and are depicted in drug-equivalent units (see ref <a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> for additional information).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig13" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0014.jpeg" id="rightTab-gr22" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 13. Intracellular quantitation of chimeric degraders <b>3</b> or <b>13</b> in PC3-S1 cells following non-antibody-mediated delivery or respective delivery via STEAP1<b>–3a</b> or STEAP1–<b>13a</b> ADCs. Time points are listed at the top of each plot. Non-antibody-mediated delivery: compound <b>3</b> (solid purple), compound <b>13</b> (solid orange). ADC delivery: STEAP1<b>–3a</b> (solid red), CLL1<b>–3a</b> (dashed red); deliver compound <b>3</b>. ADC delivery: STEAP1–<b>13a</b> (solid green), CLL1–<b>13a</b> (dashed green); deliver compound <b>13</b>. Extracellular conjugate concentrations were absolute (i.e., <i>not</i> the drug-equivalent concentrations employed elsewhere in this work). For comparison purposes, an extracellular conjugate concentration of 100 nM corresponds to approximately 15 μg/mL. Intracellular values for compounds <b>3</b> and <b>13</b> associated with the 0.1 nM concentrations of the test articles were often below the 0.002 nM LLOQ and are not depicted in the plots.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=fig13"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig14" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 14</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0015.jpeg" id="rightTab-gr23" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 14. <i>In vivo</i> activity of conjugates STEAP1–<b>13a</b> and CLL1–<b>13a</b> in the PC3-S1 xenograft model. Efficacy (tumor growth inhibition). Blue traces (solid lines), STEAP1–<b>13a</b>; blue trace (dotted line), unconjugated STEAP1 mAb; purple traces, CLL1–<b>13a</b>; black trace, vehicle control. Numbers next to traces indicate doses of each conjugate (in mg/kg) that were administered once IV at the day 0 time point. Group mean tumor volumes (±SEM) are plotted over the duration of the study (<i>n</i> = 8/group). See the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Supporting Information</a> for additional details.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=fig14"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig15" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Figure 15</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0016.jpeg" id="rightTab-gr24" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 15. <i>In vivo</i> activity of conjugates STEAP1–<b>8a</b> and CLL1–<b>8a</b> in the HL-60 xenograft model. Efficacy (tumor growth inhibition). Blue trace, STEAP1–<b>8a</b>; purple traces, CLL1–<b>8a</b>; black trace, vehicle control. Numbers next to traces indicate doses of each conjugate (in mg/kg) that were administered once IV at the day 0 time point. All groups received 30 mg/kg anti-ragweed IP 4 h prior to ADC administrations. Group mean tumor volumes (±SEM) are plotted over the duration of the study (<i>n</i> = 5/group). See the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Supporting Information</a> for additional details.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=fig15"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig16" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Figure 16</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0017.jpeg" id="rightTab-gr25" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 16. (a, b) <i>In vivo</i> activity of conjugates STEAP1–<b>13a</b>, CLL1–<b>13a</b>, CLL1–<b>14a</b>, and CLL1–<b>16</b> in the HL-60 xenograft model. Efficacy (tumor growth inhibition). (a) Blue traces, STEAP1–<b>13a</b>; purple traces, CLL1–<b>13a</b>; black trace, vehicle control. (b) Purple traces (solid lines), CLL1–<b>13a</b>; purple trace (dotted line), unconjugated CLL1 mAb; green trace, CLL1–<b>14a</b>; red trace, CLL1–<b>16</b>; black trace, vehicle control. Numbers next to traces indicate doses of each conjugate (in mg/kg) that were administered once IV at the day 0 time point. All groups received 30 mg/kg anti-ragweed IP 4 h prior to ADC administrations. Group mean tumor volumes (±SEM) are plotted over the duration of the study (<i>n</i> = 5/group). See the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf" class="ext-link">Supporting Information</a> for additional details.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=fig16"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig17" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Figure 17</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0018.jpeg" id="rightTab-gr26" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 17. Compound <b>16</b> (VHL fragment linker-drug) and the corresponding CLL1–<b>16</b> conjugate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=fig17"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig18" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Figure 18</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0019.jpeg" id="rightTab-gr27" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 18. <i>In vivo</i> stability of CLL1–<b>13a</b> following a single IV administration of 10 mg/kg to mice (normalized ave DAR plot).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=fig18"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig19" data-index="19" class="article__inlineFigure"><h2 class="fig-label">Figure 19</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0020.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0020.jpeg" id="rightTab-gr28" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 19. Pharmacokinetic properties of the conjugate CLL1–<b>13a</b> (10 mg/kg) and the corresponding unconjugated mAb (5 mg/kg) following single IV administrations of each entity at the indicated dose to mice (analyte = total antibody).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=fig19"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig20" data-index="20" class="article__inlineFigure"><h2 class="fig-label">Figure 20</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0021.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0021.jpeg" id="rightTab-gr29" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 20. Impact of chimeric degrader <b>13</b> and conjugates STEAP1–<b>13a</b> and CLL1–<b>13a</b> on in vitro human megakaryocytopoiesis. Concentrations of STEAP1–<b>13a</b> and CLL1–<b>13a</b> conjugates reflect amounts of conjugated chimeric degrader <b>13</b> present in the assay. Time point was 6 days.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0021.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=fig20"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="21" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0022.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0022.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Syntheses of BRD4 Inhibitors <b>5</b>–<b>7</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0022.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) LiOH·H<sub>2</sub>O, 4:1:1 tetrahydrofuran (THF):MeOH:H<sub>2</sub>O, 23 °C, 2 h, 86%; (b) CH<sub>3</sub>NH<sub>2</sub>·HCl, HATU, <i>N,N</i>-diisopropylethylamine (DIEA), dimethylformamide (DMF), 23 °C, 1 h, 81%; (c) DIBAL-H, 1,2-dichloroethane, −78 °C, 2 h, 95%; (d) DMP, 1,2-dichloroethane, 23 °C, 12 h, 74%; (e) CH<sub>3</sub>COC(N<sub>2</sub>)PO(OCH<sub>3</sub>)<sub>2</sub>, K<sub>2</sub>CO<sub>3</sub>, MeOH, 23 °C, 12 h, 24%.</p></p></figure><figure data-id="sch2" data-index="22" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0023.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0023.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Syntheses of Chimeric BRD4 Degrader <b>8</b> and Linker-Drugs <b>8a</b> and <b>8b</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0023.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) HATU, Et<sub>3</sub>N, DMF, 23 °C, 2 h, 31%; (b) trifluoroacetic acid (TFA), 1,1,1,3,3,3-hexafluoropropan-2-ol (HFIP), 23 °C, 1 h, 98–100%; (c) <b>22</b>, HATU, DIEA, DMF, 23 °C, 0.5–1 h, 82–83%; (d) <b>17</b>, HATU, DIEA, DMF, 23 °C, 1 h, 29–42%.</p></p></figure><figure data-id="sch3" data-index="23" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0024.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0024.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Syntheses of Linker-Drugs <b>9a</b> and <b>9c</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0024.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) TFA, HFIP, 23 °C, 1 h, 87–100%; (b) <b>27</b>, HATU, DIEA, dichloromethane (DCM) or DMF, 23 °C, 1–2 h, 61–72%; (c) <b>17</b>, HATU, DIEA, DMF, 23 °C, 1–2 h, 43–62%.</p></p></figure><figure data-id="sch4" data-index="24" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0025.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0025.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of the Linker-Drug <b>9d</b>.<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0025.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>30</b>, HATU, DIEA, DMF, 23 °C, 2 h, 100%; (b) ClC(O)OCH<sub>2</sub>CH═CH<sub>2</sub>, Na<sub>2</sub>CO<sub>3</sub>, 1:1 THF:H<sub>2</sub>O, 0–25 °C, 12 h, 81%; (c) PCl<sub>3</sub>, Et<sub>3</sub>N, THF, −78 to 23 °C, 12 h and then 5:2 NaHCO<sub>3</sub>:H<sub>2</sub>O, 23 °C, 10 min, 89%; (d) TMS-imidazole, Et<sub>3</sub>N, 1:1 MeCN:CCl<sub>4</sub>, 23 °C, 40 min, 90%; (e) <b>36</b>, ZnCl<sub>2</sub>, DMF, 23 °C, 12 h, 38%; (f) 1,3-dimethylbarbituric acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, DCM, 23 °C, 2 h, 61%; (g) <b>31</b>, HATU, DIEA, DMF, 23 °C, 2 h, 45%; (h) piperidine, DMF, 23 °C, 1 h, 59%; (i) <b>40</b>, DIEA, DMF, 23 °C, 6 h, 24%.</p></p></figure><figure data-id="sch5" data-index="25" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0026.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0026.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Syntheses of Chimeric BRD4 Degrader <b>11</b> and Linker-Drug <b>11a</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0026.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) TFA, DCM, 23 °C, 30 min, 97%; (b) <b>17</b>, HATU, DIEA, DMF, 23 °C, 1 h, 98%; (c) TMSI, DCM, 23 °C, 1 h, 27%; (d) <b>44</b>, DIEA, DMF, 23 °C, 2 h, 45%.</p></p></figure><figure data-id="sch6" data-index="26" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0027.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0027.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Syntheses of Chimeric BRD4 Degrader <b>12</b> and Linker-Drug <b>12a</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0027.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 4.0 M HCl in EtOAc, 23 °C, 1 h, 100%; (b) <b>27</b>, HATU, DIEA, DMF, 23 °C, 1 h, 36%; (c) TFA, HFIP, 23 °C, 1 h, 100%; (d) <b>17</b>, HATU, DIEA, DMF, 23 °C, 1 h, 23%; (e) TMSI, DCM, 23 °C, 30 min, 70%; (f) <b>44</b>, DIEA, DMF, 23 °C, 2 h, 41%.</p></p></figure><figure data-id="sch7" data-index="27" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0028.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0028.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Syntheses of Chimeric BRD4 Degraders <b>13</b> and <b>14</b> and Linker-Drugs <b>13a</b>, <b>14a</b>, and <b>16</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0028.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) TFA, DCM, 23 °C, 1 h, 40%; (b) <b>32</b> or <b>52</b>, HATU, DIEA, DMF, 23 °C, 1 h, 53–69%; (c) TMSI, DCM, 23 °C, 1 h, 98–100%; (d) <b>17</b>, HATU, DIEA, DMF, 23 °C, 1 h, 32–70%; (e) <b>44</b>, HOBt, pyridine, DMF, 38 °C, 1 or 24 h, 12–19%.</p></p></figure><figure data-id="sch8" data-index="28" class="article__inlineFigure"><h2 class="fig-label">Scheme 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0029.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0029.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 8. Synthesis of Chimeric BRD4 Degrader <b>15</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0029.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=sch8"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Pd on C, H<sub>2</sub>, THF, 23 °C, 1 h, 100%; (b) <b>17</b>, HATU, DIEA, DMF, 23 °C, 2 h, 60%; (c) TFA, DCM, 23 °C, 1 h, 99%; (d) <b>32</b>, HATU, DIEA, DMF, 23 °C, 2 h, 21%.</p></p></figure><figure data-id="sch9" data-index="29" class="article__inlineFigure"><h2 class="fig-label">Scheme 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/medium/jm0c01846_0030.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0030.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 9. Synthesis of Linker-Drug <b>15a</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01846/20210305/images/large/jm0c01846_0030.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01846&amp;id=sch9"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 35% HBr in HOAc, 23 °C, 2 h, 89%; (b) Boc<sub>2</sub>O, Et<sub>3</sub>N, DCM, 23 °C, 1 h, 87%; (c) <b>58</b>, Cs<sub>2</sub>CO<sub>3</sub>, DMF, 23 °C, 1 h, 19%; (d) TFA, HFIP, 23 °C, 1 h, 99%; (e) <b>17</b>, HATU, DIEA, DMF, 23 °C, 1 h, 41%; (f) <b>32</b>, HATU, DIEA, DMF, 23 °C, 1 h, 22%; (g) piperidine, DMF, 23 °C, 2 h, 93%; (h) <b>65</b>, HATU, DIEA, DMF 23 °C, 1 h, 33%.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i77">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71024" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71024" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 48 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, L.-l.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.-j.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span> <span> </span><span class="NLM_article-title">Inhibition of BET bromodomains as a therapeutic strategy for cancer drug discovery</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">5501</span>– <span class="NLM_lpage">5516</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.3551</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.18632%2Foncotarget.3551" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=25849938" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A280%3ADC%252BC2MjitVGiuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=5501-5516&author=L.-l.+Fuauthor=M.+Tianauthor=X.+Liauthor=J.-j.+Liauthor=J.+Huangauthor=L.+Ouyangauthor=Y.+Zhangauthor=B.+Liu&title=Inhibition+of+BET+bromodomains+as+a+therapeutic+strategy+for+cancer+drug+discovery&doi=10.18632%2Foncotarget.3551"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1aR"><div class="casContent"><span class="casTitleNuber">1a</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of BET bromodomains as a therapeutic strategy for cancer drug discovery</span></div><div class="casAuthors">Fu Lei-lei; Tian Mao; Li Xiang; Li Jing-jing; Ouyang Liang; Zhang Yonghui; Liu Bo; Huang Jian; Zhang Yonghui</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">5501-16</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">As a conserved protein interaction module that recognizes and binds to acetylated lysine, bromodomain (BRD) contains a deep, largely hydrophobic acetyl lysine binding site.  Proteins that share the feature of containing two BRDs and an extra-terminal domain belong to BET family, including BRD2, BRD3, BRD4 and BRDT.  BET family proteins perform transcription regulatory function under normal conditions, while in cancer, they regulate transcription of several oncogenes, such as c-Myc and Bcl-2.  Thus, targeting BET proteins may be a promising strategy, and intense interest of BET proteins has fueled the development of structure-based bromodomain inhibitors in cancer.  In this review, we focus on summarizing several small-molecule BET inhibitors and their relevant anti-tumor mechanisms, which would provide a clue for exploiting new targeted BET inhibitors in the future cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQBHUaDfXsQehKDYG3bNImrfW6udTcc2ebAMGmld5e_Cbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MjitVGiuw%253D%253D&md5=b3b070170e6da5585f87dffe1d568f47</span></div><a href="/servlet/linkout?suffix=cit1a&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.3551&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.3551%26sid%3Dliteratum%253Aachs%26aulast%3DFu%26aufirst%3DL.-l.%26aulast%3DTian%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DJ.-j.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DOuyang%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DB.%26atitle%3DInhibition%2520of%2520BET%2520bromodomains%2520as%2520a%2520therapeutic%2520strategy%2520for%2520cancer%2520drug%2520discovery%26jtitle%3DOncotarget%26date%3D2015%26volume%3D6%26spage%3D5501%26epage%3D5516%26doi%3D10.18632%2Foncotarget.3551" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit1b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Targeting bromodomains: epigenetic readers of lysine acetylation</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">337</span>– <span class="NLM_lpage">356</span>, <span class="refDoi"> DOI: 10.1038/nrd4286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1038%2Fnrd4286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=24751816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=337-356&author=P.+Filippakopoulosauthor=S.+Knapp&title=Targeting+bromodomains%3A+epigenetic+readers+of+lysine+acetylation&doi=10.1038%2Fnrd4286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1bR"><div class="casContent"><span class="casTitleNuber">1b</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bromodomains: epigenetic readers of lysine acetylation</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">337-356</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a key mechanism that regulates chromatin structure; aberrant acetylation levels have been linked to the development of several diseases.  Acetyl-lysine modifications create docking sites for bromodomains, which are small interaction modules found on diverse proteins, some of which have a key role in the acetylation-dependent assembly of transcriptional regulator complexes.  These complexes can then initiate transcriptional programs that result in phenotypic changes.  The recent discovery of potent and highly specific inhibitors for the BET (bromodomain and extra-terminal) family of bromodomains has stimulated intensive research activity in diverse therapeutic areas, particularly in oncol., where BET proteins regulate the expression of key oncogenes and anti-apoptotic proteins.  In addn., targeting BET bromodomains could hold potential for the treatment of inflammation and viral infection.  Here, we highlight recent progress in the development of bromodomain inhibitors, and their potential applications in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOSIcwBJKj4LVg90H21EOLACvtfcHk0lihQEbXd6LLDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D&md5=77049c82ce6a19b4685e3b7950399a07</span></div><a href="/servlet/linkout?suffix=cit1b&amp;dbid=16384&amp;doi=10.1038%2Fnrd4286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4286%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DTargeting%2520bromodomains%253A%2520epigenetic%2520readers%2520of%2520lysine%2520acetylation%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D337%26epage%3D356%26doi%3D10.1038%2Fnrd4286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span> <span> </span><span class="NLM_article-title">Targeting Brd4 for cancer therapy: inhibitors and degraders</span>. <i>Med. Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1779</span>– <span class="NLM_lpage">1802</span>, <span class="refDoi"> DOI: 10.1039/C8MD00198G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1039%2FC8MD00198G" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=1779-1802&author=Y.+Duanauthor=Y.+Guanauthor=W.+Qinauthor=X.+Zhaiauthor=B.+Yuauthor=H.+Liu&title=Targeting+Brd4+for+cancer+therapy%3A+inhibitors+and+degraders&doi=10.1039%2FC8MD00198G"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2a&amp;dbid=16384&amp;doi=10.1039%2FC8MD00198G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC8MD00198G%26sid%3Dliteratum%253Aachs%26aulast%3DDuan%26aufirst%3DY.%26aulast%3DGuan%26aufirst%3DY.%26aulast%3DQin%26aufirst%3DW.%26aulast%3DZhai%26aufirst%3DX.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DH.%26atitle%3DTargeting%2520Brd4%2520for%2520cancer%2520therapy%253A%2520inhibitors%2520and%2520degraders%26jtitle%3DMed.%2520Chem.%2520Commun.%26date%3D2018%26volume%3D9%26spage%3D1779%26epage%3D1802%26doi%3D10.1039%2FC8MD00198G" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit2b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wold, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brasier, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span> <span> </span><span class="NLM_article-title">Drug discovery targeting bromodomain-containing protein 4</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4533</span>– <span class="NLM_lpage">4558</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01761</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01761" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXis1yiurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4533-4558&author=Z.+Liuauthor=P.+Wangauthor=H.+Chenauthor=E.+A.+Woldauthor=B.+Tianauthor=A.+R.+Brasierauthor=J.+Zhou&title=Drug+discovery+targeting+bromodomain-containing+protein+4&doi=10.1021%2Facs.jmedchem.6b01761"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2bR"><div class="casContent"><span class="casTitleNuber">2b</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Discovery Targeting Bromodomain-Containing Protein 4</span></div><div class="casAuthors">Liu, Zhiqing; Wang, Pingyuan; Chen, Haiying; Wold, Eric A.; Tian, Bing; Brasier, Allan R.; Zhou, Jia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4533-4558</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  BRD4, the most extensively studied member of BET family, is an epigenetic regulator that localizes to DNA via binding acetylated histones and controls the expression of therapeutically important gene regulatory networks through recruiting transcription factors to form mediator complexes, phosphorylating RNA polymerase II and by its intrinsic histone acetyltransferase activity.  Disrupting the protein-protein interactions between BRD4 and acetyl-lysine has been shown to effectively block cell proliferation in cancer, cytokine prodn. in acute inflammation, etc.  To date, significant efforts have been devoted to the development of BRD4 inhibitors, and consequently, a dozen have progressed into human clin. trials.  Herein, the authors summarize the advances in drug discovery and development of BRD4 inhibitors by focusing on their chemotypes, in vitro and in vivo activity, selectivity, relevant mechanisms of action and therapeutic potential.  Opportunities and challenges to achieve selective and efficacious BRD4 inhibitors as a viable therapeutic strategy for human diseases are also highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6IAERH4iggrVg90H21EOLACvtfcHk0lh-PQL-UOFvVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXis1yiurg%253D&md5=6d7ba5844073fac8c916b1249af402e5</span></div><a href="/servlet/linkout?suffix=cit2b&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01761&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01761%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DWold%26aufirst%3DE.%2BA.%26aulast%3DTian%26aufirst%3DB.%26aulast%3DBrasier%26aufirst%3DA.%2BR.%26aulast%3DZhou%26aufirst%3DJ.%26atitle%3DDrug%2520discovery%2520targeting%2520bromodomain-containing%2520protein%25204%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D4533%26epage%3D4558%26doi%3D10.1021%2Facs.jmedchem.6b01761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit2c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côte′, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S.</span></span> <span> </span><span class="NLM_article-title">Disrupting acetyl-lysine recognition: progress in the development of bromodomain inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1271</span>– <span class="NLM_lpage">1298</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01514</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01514" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsr%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1271-1298&author=F.+A.+Romeroauthor=A.+M.+Taylorauthor=T.+D.+Crawfordauthor=V.+Tsuiauthor=A.+C%C3%B4te%E2%80%B2author=S.+Magnuson&title=Disrupting+acetyl-lysine+recognition%3A+progress+in+the+development+of+bromodomain+inhibitors&doi=10.1021%2Facs.jmedchem.5b01514"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2cR"><div class="casContent"><span class="casTitleNuber">2c</span><div class="casTitle"><span class="NLM_cas:atitle">Disrupting Acetyl-Lysine Recognition: Progress in the Development of Bromodomain Inhibitors</span></div><div class="casAuthors">Romero, F. Anthony; Taylor, Alexander M.; Crawford, Terry D.; Tsui, Vickie; Cote, Alexandre; Magnuson, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1271-1298</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains, small protein modules that recognize acetylated lysine on histones, play a significant role in the epigenome, where they function as "readers" that ultimately det. the functional outcome of the post-translational modification.  Because the initial discovery of selective BET inhibitors have helped define the role of that protein family in oncol. and inflammation, BET bromodomains have continued to garner the most attention of any other bromodomain.  More recently, non-BET bromodomain inhibitors that are potent and selective have been disclosed for ATAD2, CBP, BRD7/9, BRPF, BRPF/TRIM24, CECR2, SMARCA4, and BAZ2A/B.  Such novel inhibitors can be used to probe the physiol. function of these non-BET bromodomains and further understanding of their role in certain disease states.  Here, we provide an update to the progress in identifying selective bromodomain inhibitors and their use as biol. tools, as well as our perspective on the field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgbRrKGlHaVLVg90H21EOLACvtfcHk0lh-PQL-UOFvVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsr%252FO&md5=260352f61301aa88aad48c0b63e6152f</span></div><a href="/servlet/linkout?suffix=cit2c&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01514%26sid%3Dliteratum%253Aachs%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DC%25C3%25B4te%25E2%2580%25B2%26aufirst%3DA.%26aulast%3DMagnuson%26aufirst%3DS.%26atitle%3DDisrupting%2520acetyl-lysine%2520recognition%253A%2520progress%2520in%2520the%2520development%2520of%2520bromodomain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1271%26epage%3D1298%26doi%3D10.1021%2Facs.jmedchem.5b01514" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amorim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stathis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gleeson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iyengar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magarotto, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leleu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morschhauser, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karlin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Broussais, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herait, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahatt, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokiec, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salles, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Facon, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palumbo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zucca, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thieblemont, C.</span></span> <span> </span><span class="NLM_article-title">Bromodomain inhibitor OTX015 in patients with lymphoma or multiple myeloma: a dose-escalation, open-label, pharmacokinetic, phase 1 study</span>. <i>Lancet Haematol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">e196</span>– <span class="NLM_lpage">e204</span>, <span class="refDoi"> DOI: 10.1016/S2352-3026(16)00021-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1016%2FS2352-3026%2816%2900021-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=27063978" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2016&pages=e196-e204&author=S.+Amorimauthor=A.+Stathisauthor=M.+Gleesonauthor=S.+Iyengarauthor=V.+Magarottoauthor=X.+Leleuauthor=F.+Morschhauserauthor=L.+Karlinauthor=F.+Broussaisauthor=K.+Rezaiauthor=P.+Heraitauthor=C.+Kahattauthor=F.+Lokiecauthor=G.+Sallesauthor=T.+Faconauthor=A.+Palumboauthor=D.+Cunninghamauthor=E.+Zuccaauthor=C.+Thieblemont&title=Bromodomain+inhibitor+OTX015+in+patients+with+lymphoma+or+multiple+myeloma%3A+a+dose-escalation%2C+open-label%2C+pharmacokinetic%2C+phase+1+study&doi=10.1016%2FS2352-3026%2816%2900021-1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2FS2352-3026%2816%2900021-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS2352-3026%252816%252900021-1%26sid%3Dliteratum%253Aachs%26aulast%3DAmorim%26aufirst%3DS.%26aulast%3DStathis%26aufirst%3DA.%26aulast%3DGleeson%26aufirst%3DM.%26aulast%3DIyengar%26aufirst%3DS.%26aulast%3DMagarotto%26aufirst%3DV.%26aulast%3DLeleu%26aufirst%3DX.%26aulast%3DMorschhauser%26aufirst%3DF.%26aulast%3DKarlin%26aufirst%3DL.%26aulast%3DBroussais%26aufirst%3DF.%26aulast%3DRezai%26aufirst%3DK.%26aulast%3DHerait%26aufirst%3DP.%26aulast%3DKahatt%26aufirst%3DC.%26aulast%3DLokiec%26aufirst%3DF.%26aulast%3DSalles%26aufirst%3DG.%26aulast%3DFacon%26aufirst%3DT.%26aulast%3DPalumbo%26aufirst%3DA.%26aulast%3DCunningham%26aufirst%3DD.%26aulast%3DZucca%26aufirst%3DE.%26aulast%3DThieblemont%26aufirst%3DC.%26atitle%3DBromodomain%2520inhibitor%2520OTX015%2520in%2520patients%2520with%2520lymphoma%2520or%2520multiple%2520myeloma%253A%2520a%2520dose-escalation%252C%2520open-label%252C%2520pharmacokinetic%252C%2520phase%25201%2520study%26jtitle%3DLancet%2520Haematol.%26date%3D2016%26volume%3D3%26spage%3De196%26epage%3De204%26doi%3D10.1016%2FS2352-3026%2816%2900021-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Collins, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattersley, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yates, J. W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mondal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mettetal, J. T.</span></span> <span> </span><span class="NLM_article-title">Translational modeling of drug-induced myelosuppression and effect of pretreatment myelosuppression for AZD5153, a selective BRD4 inhibitor</span>. <i>CPT: Pharmacometrics Syst. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">357</span>– <span class="NLM_lpage">364</span>, <span class="refDoi"> DOI: 10.1002/psp4.12194</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1002%2Fpsp4.12194" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=28378926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXos1Srsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2017&pages=357-364&author=T.+A.+Collinsauthor=M.+M.+Hattersleyauthor=J.+W.+T.+Yatesauthor=E.+Clarkauthor=M.+Mondalauthor=J.+T.+Mettetal&title=Translational+modeling+of+drug-induced+myelosuppression+and+effect+of+pretreatment+myelosuppression+for+AZD5153%2C+a+selective+BRD4+inhibitor&doi=10.1002%2Fpsp4.12194"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Translational Modeling of Drug-Induced Myelosuppression and Effect of Pretreatment Myelosuppression for AZD5153, a Selective BRD4 Inhibitor</span></div><div class="casAuthors">Collins, T. A.; Hattersley, M. M.; Yates, J. W. T.; Clark, E.; Mondal, M.; Mettetal, J. T.</div><div class="citationInfo"><span class="NLM_cas:title">CPT: Pharmacometrics & Systems Pharmacology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">357-364</span>CODEN:
                <span class="NLM_cas:coden">CPSPBR</span>;
        ISSN:<span class="NLM_cas:issn">2163-8306</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">In this work, we evaluate the potential risk of thrombocytopenia in man for a BRD4 inhibitor, AZD5153, based on the platelet count decreases from a Han Wistar rat study.  The effects in rat were modeled and used to make clin. predictions for human populations with healthy baseline blood counts.  At doses >10 mg, a dose-dependent effect on circulating platelets is expected, with similar predicted changes for both q.d. and b.i.d. dose schedules.  These results suggest that at predicted efficacious doses, AZD5153 is likely to have some redns. in the clin. platelet counts, but within the normal range at projected efficacious doses.  The model was then extended to incorporate preexisting myelosuppression where bone marrow function is inhibited by acute myeloid leukemia.  Under these conditions, duration of platelet count recovery has the potential to be prolonged due to drug-induced myelosuppression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnyVGxxPLv9LVg90H21EOLACvtfcHk0lh-PQL-UOFvVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXos1Srsrg%253D&md5=2bdc4d7bfd3ee8501d28db2c738631ff</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1002%2Fpsp4.12194&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpsp4.12194%26sid%3Dliteratum%253Aachs%26aulast%3DCollins%26aufirst%3DT.%2BA.%26aulast%3DHattersley%26aufirst%3DM.%2BM.%26aulast%3DYates%26aufirst%3DJ.%2BW.%2BT.%26aulast%3DClark%26aufirst%3DE.%26aulast%3DMondal%26aufirst%3DM.%26aulast%3DMettetal%26aufirst%3DJ.%2BT.%26atitle%3DTranslational%2520modeling%2520of%2520drug-induced%2520myelosuppression%2520and%2520effect%2520of%2520pretreatment%2520myelosuppression%2520for%2520AZD5153%252C%2520a%2520selective%2520BRD4%2520inhibitor%26jtitle%3DCPT%253A%2520Pharmacometrics%2520Syst.%2520Pharmacol.%26date%3D2017%26volume%3D6%26spage%3D357%26epage%3D364%26doi%3D10.1002%2Fpsp4.12194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Seal, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aylott, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copley, R. C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, J. R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayhow, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messenger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preston, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wall, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolven, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">The optimization of a novel, weak bromo and extra terminal domain (BET) bromodomain fragment ligand to a potent and selective second bromodomain (BD2) inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">9093</span>– <span class="NLM_lpage">9126</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00796</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00796" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVCmurzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=9093-9126&author=J.+T.+Sealauthor=S.+J.+Atkinsonauthor=H.+Aylottauthor=P.+Bamboroughauthor=C.-w.+Chungauthor=R.+C.+B.+Copleyauthor=L.+Gordonauthor=P.+Grandiauthor=J.+R.+J.+Grayauthor=L.+A.+Harrisonauthor=T.+G.+Hayhowauthor=M.+Lindonauthor=C.+Messengerauthor=A.-M.+Michonauthor=D.+Mitchellauthor=A.+Prestonauthor=R.+K.+Prinjhaauthor=I.+Riojaauthor=S.+Taylorauthor=I.+D.+Wallauthor=R.+J.+Watsonauthor=J.+M.+Woolvenauthor=E.+H.+Demont&title=The+optimization+of+a+novel%2C+weak+bromo+and+extra+terminal+domain+%28BET%29+bromodomain+fragment+ligand+to+a+potent+and+selective+second+bromodomain+%28BD2%29+inhibitor&doi=10.1021%2Facs.jmedchem.0c00796"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5aR"><div class="casContent"><span class="casTitleNuber">5a</span><div class="casTitle"><span class="NLM_cas:atitle">The Optimization of a Novel, Weak Bromo and Extra Terminal Domain (BET) Bromodomain Fragment Ligand to a Potent and Selective Second Bromodomain (BD2) Inhibitor</span></div><div class="casAuthors">Seal, Jonathan T.; Atkinson, Stephen J.; Aylott, Helen; Bamborough, Paul; Chung, Chun-wa; Copley, Royston C. B.; Gordon, Laurie; Grandi, Paola; Gray, James R. J.; Harrison, Lee A.; Hayhow, Thomas G.; Lindon, Matthew; Messenger, Cassie; Michon, Anne-Marie; Mitchell, Darren; Preston, Alex; Prinjha, Rab K.; Rioja, Inmaculada; Taylor, Simon; Wall, Ian D.; Watson, Robert J.; Woolven, James M.; Demont, Emmanuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9093-9126</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The profound efficacy, yet assocd. toxicity of pan-BET inhibitors is well documented.  The possibility of an ameliorated safety profile driven by significantly selective (>100-fold) inhibition of a subset of the eight bromodomains is enticing, but challenging given the close homol.  Herein, we describe the X-ray crystal structure-directed optimization of a novel weak fragment ligand with a pan-second bromodomain (BD2) bias, to potent and highly BD2 selective inhibitors.  A template hopping approach, enabled by our parallel research into an orthogonal template (15, GSK046, I), was the basis for the high selectivity obsd.  This culminated in two tool mols., 20 (GSK620) and 56 (GSK549), which showed an anti-inflammatory phenotype in human whole blood, confirming their cellular target engagement.  Excellent broad selectivity, developability, and in vivo oral pharmacokinetics characterize these tools, which we hope will be of broad utility to the field of epigenetics research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoS04XaSgoDyLVg90H21EOLACvtfcHk0lioIav4skB5zw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVCmurzI&md5=c562a8c9e4c38abac04149ee96e2f594</span></div><a href="/servlet/linkout?suffix=cit5a&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00796%26sid%3Dliteratum%253Aachs%26aulast%3DSeal%26aufirst%3DJ.%2BT.%26aulast%3DAtkinson%26aufirst%3DS.%2BJ.%26aulast%3DAylott%26aufirst%3DH.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DCopley%26aufirst%3DR.%2BC.%2BB.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DJ.%2BR.%2BJ.%26aulast%3DHarrison%26aufirst%3DL.%2BA.%26aulast%3DHayhow%26aufirst%3DT.%2BG.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DMessenger%26aufirst%3DC.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DMitchell%26aufirst%3DD.%26aulast%3DPreston%26aufirst%3DA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DWall%26aufirst%3DI.%2BD.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DWoolven%26aufirst%3DJ.%2BM.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DThe%2520optimization%2520of%2520a%2520novel%252C%2520weak%2520bromo%2520and%2520extra%2520terminal%2520domain%2520%2528BET%2529%2520bromodomain%2520fragment%2520ligand%2520to%2520a%2520potent%2520and%2520selective%2520second%2520bromodomain%2520%2528BD2%2529%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D9093%26epage%3D9126%26doi%3D10.1021%2Facs.jmedchem.0c00796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit5b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Preston, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craggs, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, J. R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rianjongdee, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wall, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolven, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of a highly selective and in vivo-capable inhibitor of the second bromodomain of the bromodomain and extra terminal domain family of proteins</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">9070</span>– <span class="NLM_lpage">9092</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00605</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00605" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVWrsLrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=9070-9092&author=A.+Prestonauthor=S.+Atkinsonauthor=P.+Bamboroughauthor=C.-w.+Chungauthor=P.+D.+Craggsauthor=L.+Gordonauthor=P.+Grandiauthor=J.+R.+J.+Grayauthor=E.+J.+Jonesauthor=M.+Lindonauthor=A.-M.+Michonauthor=D.+J.+Mitchellauthor=R.+K.+Prinjhaauthor=F.+Rianjongdeeauthor=I.+Riojaauthor=J.+Sealauthor=S.+Taylorauthor=I.+Wallauthor=R.+J.+Watsonauthor=J.+Woolvenauthor=E.+H.+Demont&title=Design+and+synthesis+of+a+highly+selective+and+in+vivo-capable+inhibitor+of+the+second+bromodomain+of+the+bromodomain+and+extra+terminal+domain+family+of+proteins&doi=10.1021%2Facs.jmedchem.0c00605"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5bR"><div class="casContent"><span class="casTitleNuber">5b</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of a Highly Selective and In Vivo-Capable Inhibitor of the Second Bromodomain of the Bromodomain and Extra Terminal Domain Family of Proteins</span></div><div class="casAuthors">Preston, Alex; Atkinson, Stephen; Bamborough, Paul; Chung, Chun-wa; Craggs, Peter D.; Gordon, Laurie; Grandi, Paola; Gray, James R. J.; Jones, Emma J.; Lindon, Matthew; Michon, Anne-Marie; Mitchell, Darren J.; Prinjha, Rab K.; Rianjongdee, Francesco; Rioja, Inmaculada; Seal, Jonathan; Taylor, Simon; Wall, Ian; Watson, Robert J.; Woolven, James; Demont, Emmanuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9070-9092</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pan-bromodomain and extra terminal domain (BET) inhibitors interact equipotently with the eight bromodomains of the BET family of proteins and have shown profound efficacy in a no. of in vitro phenotypic assays and in vivo pre-clin. models in inflammation or oncol.  A no. of these inhibitors have progressed to the clinic where pharmacol.-driven adverse events have been reported.  To better understand the contribution of each domain to their efficacy and improve their safety profile, selective inhibitors are required.  This article discloses the profile of GSK046, also known as iBET-BD2(I), a highly selective inhibitor of the second bromodomains of the BET proteins that has undergone extensive pre-clin. in vitro and in vivo characterization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXn2qUNkpo27Vg90H21EOLACvtfcHk0lioIav4skB5zw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVWrsLrK&md5=08f4c47868e0a5acd5d443adba06d016</span></div><a href="/servlet/linkout?suffix=cit5b&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00605%26sid%3Dliteratum%253Aachs%26aulast%3DPreston%26aufirst%3DA.%26aulast%3DAtkinson%26aufirst%3DS.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DCraggs%26aufirst%3DP.%2BD.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DJ.%2BR.%2BJ.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRianjongdee%26aufirst%3DF.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DSeal%26aufirst%3DJ.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DWall%26aufirst%3DI.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DWoolven%26aufirst%3DJ.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520a%2520highly%2520selective%2520and%2520in%2520vivo-capable%2520inhibitor%2520of%2520the%2520second%2520bromodomain%2520of%2520the%2520bromodomain%2520and%2520extra%2520terminal%2520domain%2520family%2520of%2520proteins%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D9070%26epage%3D9092%26doi%3D10.1021%2Facs.jmedchem.0c00605" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit5c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnett, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petretich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillipou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">GSK789: A selective inhibitor of the first bromodomains (BD1) of the bromo and extra terminal domain (BET) proteins</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">9045</span>– <span class="NLM_lpage">9069</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00614</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00614" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVWrsLzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=9045-9069&author=R.+J.+Watsonauthor=P.+Bamboroughauthor=H.+Barnettauthor=C.-w.+Chungauthor=R.+Davisauthor=L.+Gordonauthor=P.+Grandiauthor=M.+Petretichauthor=A.+Phillipouauthor=R.+K.+Prinjhaauthor=I.+Riojaauthor=P.+Sodenauthor=T.+Wernerauthor=E.+H.+Demont&title=GSK789%3A+A+selective+inhibitor+of+the+first+bromodomains+%28BD1%29+of+the+bromo+and+extra+terminal+domain+%28BET%29+proteins&doi=10.1021%2Facs.jmedchem.0c00614"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5cR"><div class="casContent"><span class="casTitleNuber">5c</span><div class="casTitle"><span class="NLM_cas:atitle">GSK789: A Selective Inhibitor of the First Bromodomains (BD1) of the Bromo and Extra Terminal Domain (BET) Proteins</span></div><div class="casAuthors">Watson, Robert J.; Bamborough, Paul; Barnett, Heather; Chung, Chun-wa; Davis, Rob; Gordon, Laurie; Grandi, Paola; Petretich, Massimo; Phillipou, Alex; Prinjha, Rab K.; Rioja, Inmaculada; Soden, Peter; Werner, Thilo; Demont, Emmanuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9045-9069</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pan-bromodomain and extra terminal (BET) inhibitors interact equipotently with all eight bromodomains of the BET family of proteins.  They have shown profound efficacy in vitro and in vivo in oncol. and immunomodulatory models, and a no. of them are currently in clin. trials where significant safety signals have been reported.  It is therefore important to understand the functional contribution of each bromodomain to assess the opportunity to tease apart efficacy and toxicity.  This article discloses the in vitro and cellular activity profiles of GSK789(I), a potent, cell-permeable, and highly selective inhibitor of the first bromodomains of the BET family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozid6PMjhvvbVg90H21EOLACvtfcHk0ljZzDLuHxPvEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVWrsLzJ&md5=c86b4d2c85dcfa62854debf225c8076c</span></div><a href="/servlet/linkout?suffix=cit5c&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00614&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00614%26sid%3Dliteratum%253Aachs%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBarnett%26aufirst%3DH.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DDavis%26aufirst%3DR.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DPetretich%26aufirst%3DM.%26aulast%3DPhillipou%26aufirst%3DA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DSoden%26aufirst%3DP.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DGSK789%253A%2520A%2520selective%2520inhibitor%2520of%2520the%2520first%2520bromodomains%2520%2528BD1%2529%2520of%2520the%2520bromo%2520and%2520extra%2520terminal%2520domain%2520%2528BET%2529%2520proteins%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D9045%26epage%3D9069%26doi%3D10.1021%2Facs.jmedchem.0c00614" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit5d"><span><span class="NLM_label">(d) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Wellaway, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craggs, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karamshi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willis, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolven, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wyspian′ska, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerr, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of a bromodomain and extraterminal domain (BET) inhibitor selective for the N-terminal bromodomains that retains an anti-inflammatory and antiproliferative phenotype</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">9020</span>– <span class="NLM_lpage">9044</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00566</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00566" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsV2lt77M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=9020-9044&author=C.+R.+Wellawayauthor=P.+Bamboroughauthor=S.+G.+Bernardauthor=C.-w.+Chungauthor=P.+D.+Craggsauthor=L.+Cutlerauthor=E.+H.+Demontauthor=J.+P.+Evansauthor=L.+Gordonauthor=B.+Karamshiauthor=A.+J.+Lewisauthor=M.+J.+Lindonauthor=D.+J.+Mitchellauthor=I.+Riojaauthor=P.+E.+Sodenauthor=S.+Taylorauthor=R.+J.+Watsonauthor=R.+Willisauthor=J.+M.+Woolvenauthor=B.+S.+Wyspian%E2%80%B2skaauthor=W.+J.+Kerrauthor=R.+K.+Prinjha&title=Structure-based+design+of+a+bromodomain+and+extraterminal+domain+%28BET%29+inhibitor+selective+for+the+N-terminal+bromodomains+that+retains+an+anti-inflammatory+and+antiproliferative+phenotype&doi=10.1021%2Facs.jmedchem.0c00566"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5dR"><div class="casContent"><span class="casTitleNuber">5d</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of a Bromodomain and Extraterminal Domain (BET) Inhibitor Selective for the N-Terminal Bromodomains That Retains an Anti-inflammatory and Antiproliferative Phenotype</span></div><div class="casAuthors">Wellaway, Christopher R.; Bamborough, Paul; Bernard, Sharon G.; Chung, Chun-wa; Craggs, Peter D.; Cutler, Leanne; Demont, Emmanuel H.; Evans, John P.; Gordon, Laurie; Karamshi, Bhumika; Lewis, Antonia J.; Lindon, Matthew J.; Mitchell, Darren J.; Rioja, Inmaculada; Soden, Peter E.; Taylor, Simon; Watson, Robert J.; Willis, Rob; Woolven, James M.; Wyspianska, Beata S.; Kerr, William J.; Prinjha, Rab K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9020-9044</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extraterminal domain (BET) family of epigenetic regulators comprises four proteins (BRD2, BRD3, BRD4, BRDT), each contg. tandem bromodomains.  To date, small mol. inhibitors of these proteins typically bind all eight bromodomains of the family with similar affinity, resulting in a diverse range of biol. effects.  To enable further understanding of the broad phenotype characteristic of pan-BET inhibition, the development of inhibitors selective for individual, or sets of, bromodomains within the family is required.  In this regard, we report the discovery of a potent probe mol. possessing up to 150-fold selectivity for the N-terminal bromodomains (BD1s) over the C-terminal bromodomains (BD2s) of the BETs.  Guided by structural information, a specific amino acid difference between BD1 and BD2 domains was targeted for selective interaction with chem. functionality appended to the previously developed I-BET151 scaffold.  Data presented herein demonstrate that selective inhibition of BD1 domains is sufficient to drive anti-inflammatory and antiproliferative effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkUREcHUhFfbVg90H21EOLACvtfcHk0ljZzDLuHxPvEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsV2lt77M&md5=52b2505bb0a36f526a08bb9bad1f2c26</span></div><a href="/servlet/linkout?suffix=cit5d&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00566&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00566%26sid%3Dliteratum%253Aachs%26aulast%3DWellaway%26aufirst%3DC.%2BR.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBernard%26aufirst%3DS.%2BG.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DCraggs%26aufirst%3DP.%2BD.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DEvans%26aufirst%3DJ.%2BP.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DKaramshi%26aufirst%3DB.%26aulast%3DLewis%26aufirst%3DA.%2BJ.%26aulast%3DLindon%26aufirst%3DM.%2BJ.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DSoden%26aufirst%3DP.%2BE.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DWillis%26aufirst%3DR.%26aulast%3DWoolven%26aufirst%3DJ.%2BM.%26aulast%3DWyspian%25E2%2580%25B2ska%26aufirst%3DB.%2BS.%26aulast%3DKerr%26aufirst%3DW.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26atitle%3DStructure-based%2520design%2520of%2520a%2520bromodomain%2520and%2520extraterminal%2520domain%2520%2528BET%2529%2520inhibitor%2520selective%2520for%2520the%2520N-terminal%2520bromodomains%2520that%2520retains%2520an%2520anti-inflammatory%2520and%2520antiproliferative%2520phenotype%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D9020%26epage%3D9044%26doi%3D10.1021%2Facs.jmedchem.0c00566" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit5e"><span><span class="NLM_label">(e) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Longenecker, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrent, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bui, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faivre, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magoc, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajaraman, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, K. F.</span></span> <span> </span><span class="NLM_article-title">Discovery of N-ethyl-4-[2-(4-fluoro-2,6-dimethyl-phenoxy)-5-(1-hydroxy-1-methyl-ethyl)phenyl]-6-methyl-7-oxo-1H-pyrrolo[2,3-c]pyridine-2-carboxamide (ABBV-744), a BET bromodomain inhibitor with selectivity for the second bromodomain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">5585</span>– <span class="NLM_lpage">5623</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00628</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00628" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5e&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnvVeisrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=5585-5623&author=G.+S.+Sheppardauthor=L.+Wangauthor=S.+D.+Fidanzeauthor=L.+A.+Hasvoldauthor=D.+Liuauthor=J.+K.+Prattauthor=C.+H.+Parkauthor=K.+L.+Longeneckerauthor=W.+Qiuauthor=M.+Torrentauthor=P.+Kovarauthor=M.+Buiauthor=E.+J.+Faivreauthor=X.+Huangauthor=X.+Linauthor=D.+Wilcoxauthor=L.+Zhangauthor=Y.+Shenauthor=D.+H.+Albertauthor=T.+J.+Magocauthor=G.+Rajaramanauthor=W.+M.+Katiauthor=K.+F.+McDaniel&title=Discovery+of+N-ethyl-4-%5B2-%284-fluoro-2%2C6-dimethyl-phenoxy%29-5-%281-hydroxy-1-methyl-ethyl%29phenyl%5D-6-methyl-7-oxo-1H-pyrrolo%5B2%2C3-c%5Dpyridine-2-carboxamide+%28ABBV-744%29%2C+a+BET+bromodomain+inhibitor+with+selectivity+for+the+second+bromodomain&doi=10.1021%2Facs.jmedchem.0c00628"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5eR"><div class="casContent"><span class="casTitleNuber">5e</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of N-Ethyl-4-[2-(4-fluoro-2,6-dimethyl-phenoxy)-5-(1-hydroxy-1-methyl-ethyl)phenyl]-6-methyl-7-oxo-1H-pyrrolo[2,3-c]pyridine-2-carboxamide (ABBV-744), a BET Bromodomain Inhibitor with Selectivity for the Second Bromodomain</span></div><div class="casAuthors">Sheppard, George S.; Wang, Le; Fidanze, Steven D.; Hasvold, Lisa A.; Liu, Dachun; Pratt, John K.; Park, Chang H.; Longenecker, Kenton; Qiu, Wei; Torrent, Maricel; Kovar, Peter J.; Bui, Mai; Faivre, Emily; Huang, Xiaoli; Lin, Xiaoyu; Wilcox, Denise; Zhang, Lu; Shen, Yu; Albert, Daniel H.; Magoc, Terrance J.; Rajaraman, Ganesh; Kati, Warren M.; McDaniel, Keith F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5585-5623</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The BET family of proteins consists of BRD2, BRD3, BRD4, and BRDt.  Each protein contains two distinct bromodomains (BD1 and BD2).  BET family bromodomain inhibitors under clin. development for oncol. bind to each of the eight bromodomains with similar affinities.  We hypothesized that it may be possible to achieve an improved therapeutic index by selectively targeting subsets of the BET bromodomains.  Both BD1 and BD2 are highly conserved across family members (>70% identity), whereas BD1 and BD2 from the same protein exhibit a larger degree of divergence (~ 40% identity), suggesting selectivity between BD1 and BD2 of all family members would be more straightforward to achieve.  Exploiting the Asp144/His437 and Ile146/Val439 sequence differences (BRD4 BD1/BD2 numbering) allowed the identification of compd. 27 demonstrating greater than 100-fold selectivity for BRD4 BD2 over BRD4 BD1.  Further optimization to improve BD2 selectivity and oral bioavailability resulted in the clin. development compd. 46 (ABBV-744).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJtZIcWro0xLVg90H21EOLACvtfcHk0li7ZZ4nbfOu6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnvVeisrk%253D&md5=675827659c3c7b6099ccd667f4c60f95</span></div><a href="/servlet/linkout?suffix=cit5e&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00628&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00628%26sid%3Dliteratum%253Aachs%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DFidanze%26aufirst%3DS.%2BD.%26aulast%3DHasvold%26aufirst%3DL.%2BA.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DLongenecker%26aufirst%3DK.%2BL.%26aulast%3DQiu%26aufirst%3DW.%26aulast%3DTorrent%26aufirst%3DM.%26aulast%3DKovar%26aufirst%3DP.%26aulast%3DBui%26aufirst%3DM.%26aulast%3DFaivre%26aufirst%3DE.%2BJ.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DWilcox%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DMagoc%26aufirst%3DT.%2BJ.%26aulast%3DRajaraman%26aufirst%3DG.%26aulast%3DKati%26aufirst%3DW.%2BM.%26aulast%3DMcDaniel%26aufirst%3DK.%2BF.%26atitle%3DDiscovery%2520of%2520N-ethyl-4-%255B2-%25284-fluoro-2%252C6-dimethyl-phenoxy%2529-5-%25281-hydroxy-1-methyl-ethyl%2529phenyl%255D-6-methyl-7-oxo-1H-pyrrolo%255B2%252C3-c%255Dpyridine-2-carboxamide%2520%2528ABBV-744%2529%252C%2520a%2520BET%2520bromodomain%2520inhibitor%2520with%2520selectivity%2520for%2520the%2520second%2520bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D5585%26epage%3D5623%26doi%3D10.1021%2Facs.jmedchem.0c00628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit5f"><span><span class="NLM_label">(f) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Faivre, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantena, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotnik, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bui, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sehgal, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uziel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hessler, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellin, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panchal, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicolette, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fossey, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Longenecker, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bigelow, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrent, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of the BD2 bromodomain of BET proteins in prostate cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>578</i></span>,  <span class="NLM_fpage">306</span>– <span class="NLM_lpage">310</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-1930-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5f&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1038%2Fs41586-020-1930-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5f&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=31969702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5f&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BB3cXis1Smsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=578&publication_year=2020&pages=306-310&author=E.+J.+Faivreauthor=K.+F.+McDanielauthor=D.+H.+Albertauthor=S.+R.+Mantenaauthor=J.+P.+Plotnikauthor=D.+Wilcoxauthor=L.+Zhangauthor=M.+H.+Buiauthor=G.+S.+Sheppardauthor=L.+Wangauthor=V.+Sehgalauthor=X.+Linauthor=X.+Huangauthor=X.+Luauthor=T.+Uzielauthor=P.+Hesslerauthor=L.+T.+Lamauthor=R.+J.+Bellinauthor=G.+Mehtaauthor=S.+Fidanzeauthor=J.+K.+Prattauthor=D.+Liuauthor=L.+A.+Hasvoldauthor=C.+Sunauthor=S.+C.+Panchalauthor=J.+J.+Nicoletteauthor=S.+L.+Fosseyauthor=C.+H.+Parkauthor=K.+Longeneckerauthor=L.+Bigelowauthor=M.+Torrentauthor=S.+H.+Rosenbergauthor=W.+M.+Katiauthor=Y.+Shen&title=Selective+inhibition+of+the+BD2+bromodomain+of+BET+proteins+in+prostate+cancer&doi=10.1038%2Fs41586-020-1930-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5fR"><div class="casContent"><span class="casTitleNuber">5f</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of the BD2 bromodomain of BET proteins in prostate cancer</span></div><div class="casAuthors">Faivre, Emily J.; McDaniel, Keith F.; Albert, Daniel H.; Mantena, Srinivasa R.; Plotnik, Joshua P.; Wilcox, Denise; Zhang, Lu; Bui, Mai H.; Sheppard, George S.; Wang, Le; Sehgal, Vasudha; Lin, Xiaoyu; Huang, Xiaoli; Lu, Xin; Uziel, Tamar; Hessler, Paul; Lam, Lloyd T.; Bellin, Richard J.; Mehta, Gaurav; Fidanze, Steve; Pratt, John K.; Liu, Dachun; Hasvold, Lisa A.; Sun, Chaohong; Panchal, Sanjay C.; Nicolette, John J.; Fossey, Stacey L.; Park, Chang H.; Longenecker, Kenton; Bigelow, Lance; Torrent, Maricel; Rosenberg, Saul H.; Kati, Warren M.; Shen, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">578</span>
        (<span class="NLM_cas:issue">7794</span>),
    <span class="NLM_cas:pages">306-310</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Proteins of the bromodomain and extra-terminal (BET) domain family are epigenetic readers that bind acetylated histones through their bromodomains to regulate gene transcription.  Dual-bromodomain BET inhibitors (DbBi) that bind with similar affinities to the first (BD1) and second (BD2) bromodomains of BRD2, BRD3, BRD4 and BRDt have displayed modest clin. activity in monotherapy cancer trials.  A reduced no. of thrombocytes in the blood (thrombocytopenia) as well as symptoms of gastrointestinal toxicity are dose-limiting adverse events for some types of DbBi1-5.  Given that similar haematol. and gastrointestinal defects were obsd. after genetic silencing of Brd4 in mice6, the platelet and gastrointestinal toxicities may represent on-target activities assocd. with BET inhibition.  The two individual bromodomains in BET family proteins may have distinct functions7-9 and different cellular phenotypes after pharmacol. inhibition of one or both bromodomains have been reported10,11, suggesting that selectively targeting one of the bromodomains may result in a different efficacy and tolerability profile compared with DbBi.  Available compds. that are selective to individual domains lack sufficient potency and the pharmacokinetics properties that are required for in vivo efficacy and tolerability assessment10-13.  Here we carried out a medicinal chem. campaign that led to the discovery of ABBV-744, a highly potent and selective inhibitor of the BD2 domain of BET family proteins with drug-like properties.  In contrast to the broad range of cell growth inhibition induced by DbBi, the antiproliferative activity of ABBV-744 was largely, but not exclusively, restricted to cell lines of acute myeloid leukemia and prostate cancer that expressed the full-length androgen receptor (AR).  ABBV-744 retained robust activity in prostate cancer xenografts, and showed fewer platelet and gastrointestinal toxicities than the DbBi ABBV-07514.  Analyses of RNA expression and chromatin immunopptn. followed by sequencing revealed that ABBV-744 displaced BRD4 from AR-contg. super-enhancers and inhibited AR-dependent transcription, with less impact on global transcription compared with ABBV-075.  These results underscore the potential value of selectively targeting the BD2 domain of BET family proteins for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrF98kER1y9pLVg90H21EOLACvtfcHk0li7ZZ4nbfOu6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXis1Smsr0%253D&md5=800d9106c33084390fd8cbf1ac7f1766</span></div><a href="/servlet/linkout?suffix=cit5f&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-1930-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-1930-8%26sid%3Dliteratum%253Aachs%26aulast%3DFaivre%26aufirst%3DE.%2BJ.%26aulast%3DMcDaniel%26aufirst%3DK.%2BF.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DMantena%26aufirst%3DS.%2BR.%26aulast%3DPlotnik%26aufirst%3DJ.%2BP.%26aulast%3DWilcox%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DBui%26aufirst%3DM.%2BH.%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DSehgal%26aufirst%3DV.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DUziel%26aufirst%3DT.%26aulast%3DHessler%26aufirst%3DP.%26aulast%3DLam%26aufirst%3DL.%2BT.%26aulast%3DBellin%26aufirst%3DR.%2BJ.%26aulast%3DMehta%26aufirst%3DG.%26aulast%3DFidanze%26aufirst%3DS.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DHasvold%26aufirst%3DL.%2BA.%26aulast%3DSun%26aufirst%3DC.%26aulast%3DPanchal%26aufirst%3DS.%2BC.%26aulast%3DNicolette%26aufirst%3DJ.%2BJ.%26aulast%3DFossey%26aufirst%3DS.%2BL.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DLongenecker%26aufirst%3DK.%26aulast%3DBigelow%26aufirst%3DL.%26aulast%3DTorrent%26aufirst%3DM.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DKati%26aufirst%3DW.%2BM.%26aulast%3DShen%26aufirst%3DY.%26atitle%3DSelective%2520inhibition%2520of%2520the%2520BD2%2520bromodomain%2520of%2520BET%2520proteins%2520in%2520prostate%2520cancer%26jtitle%3DNature%26date%3D2020%26volume%3D578%26spage%3D306%26epage%3D310%26doi%3D10.1038%2Fs41586-020-1930-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit5g"><span><span class="NLM_label">(g) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Gilan, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knezevic, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeung, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harker, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, E. Y. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petretich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassil, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vassiliadis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burr, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preston, A. G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wellaway, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gobbetti, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haynes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vappiani, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, S.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, M. A.</span></span> <span> </span><span class="NLM_article-title">Selective targeting of BD1 and BD2 of the BET proteins in cancer and immuno-inflammation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>368</i></span>,  <span class="NLM_fpage">387</span>– <span class="NLM_lpage">394</span>, <span class="refDoi"> DOI: 10.1126/science.aaz8455</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5g&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1126%2Fscience.aaz8455" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5g&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=32193360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5g&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnslKrtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2020&pages=387-394&author=O.+Gilanauthor=I.+Riojaauthor=K.+Knezevicauthor=M.+J.+Bellauthor=M.+M.+Yeungauthor=N.+R.+Harkerauthor=E.+Y.+N.+Lamauthor=C.-w.+Chungauthor=P.+Bamboroughauthor=M.+Petretichauthor=M.+Urhauthor=S.+J.+Atkinsonauthor=A.+K.+Bassilauthor=E.+J.+Robertsauthor=D.+Vassiliadisauthor=M.+L.+Burrauthor=A.+G.+S.+Prestonauthor=C.+Wellawayauthor=T.+Wernerauthor=J.+R.+Grayauthor=A.-M.+Michonauthor=T.+Gobbettiauthor=V.+Kumarauthor=P.+E.+Sodenauthor=A.+Haynesauthor=J.+Vappianiauthor=D.+F.+Toughauthor=S.+Taylorauthor=S.-J.+Dawsonauthor=M.+Bantscheffauthor=M.+Lindonauthor=G.+Drewesauthor=E.+H.+Demontauthor=D.+L.+Danielsauthor=P.+Grandiauthor=R.+K.+Prinjhaauthor=M.+A.+Dawson&title=Selective+targeting+of+BD1+and+BD2+of+the+BET+proteins+in+cancer+and+immuno-inflammation&doi=10.1126%2Fscience.aaz8455"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5gR"><div class="casContent"><span class="casTitleNuber">5g</span><div class="casTitle"><span class="NLM_cas:atitle">Selective targeting of BD1 and BD2 of the BET proteins in cancer and immunoinflammation</span></div><div class="casAuthors">Gilan, Omer; Rioja, Inmaculada; Knezevic, Kathy; Bell, Matthew J.; Yeung, Miriam M.; Harker, Nicola R.; Lam, Enid Y. N.; Chung, Chun-wa; Bamborough, Paul; Petretich, Massimo; Urh, Marjeta; Atkinson, Stephen J.; Bassil, Anna K.; Roberts, Emma J.; Vassiliadis, Dane; Burr, Marian L.; Preston, Alex G. S.; Wellaway, Christopher; Werner, Thilo; Gray, James R.; Michon, Anne-Marie; Gobbetti, Thomas; Kumar, Vinod; Soden, Peter E.; Haynes, Andrea; Vappiani, Johanna; Tough, David F.; Taylor, Simon; Dawson, Sarah-Jane; Bantscheff, Marcus; Lindon, Matthew; Drewes, Gerard; Demont, Emmanuel H.; Daniels, Danette L.; Grandi, Paola; Prinjha, Rab K.; Dawson, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">6489</span>),
    <span class="NLM_cas:pages">387-394</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">1095-9203</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The two tandem bromodomains of the BET (bromodomain and extraterminal domain) proteins enable chromatin binding to facilitate transcription.  Drugs that inhibit both bromodomains equally have shown efficacy in certain malignant and inflammatory conditions.  To explore the individual functional contributions of the first (BD1) and second (BD2) bromodomains in biol. and therapy, we developed selective BD1 and BD2 inhibitors.  We found that steady-state gene expression primarily requires BD1, whereas the rapid increase of gene expression induced by inflammatory stimuli requires both BD1 and BD2 of all BET proteins.  BD1 inhibitors phenocopied the effects of pan-BET inhibitors in cancer models, whereas BD2 inhibitors were predominantly effective in models of inflammatory and autoimmune disease.  These insights into the differential requirement of BD1 and BD2 for the maintenance and induction of gene expression may guide future BET-targeted therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr078AuJubsqbVg90H21EOLACvtfcHk0ljLEQLch9lfVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnslKrtrg%253D&md5=d78cfea091fcf5ee18735b02b2120dd7</span></div><a href="/servlet/linkout?suffix=cit5g&amp;dbid=16384&amp;doi=10.1126%2Fscience.aaz8455&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aaz8455%26sid%3Dliteratum%253Aachs%26aulast%3DGilan%26aufirst%3DO.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DKnezevic%26aufirst%3DK.%26aulast%3DBell%26aufirst%3DM.%2BJ.%26aulast%3DYeung%26aufirst%3DM.%2BM.%26aulast%3DHarker%26aufirst%3DN.%2BR.%26aulast%3DLam%26aufirst%3DE.%2BY.%2BN.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DPetretich%26aufirst%3DM.%26aulast%3DUrh%26aufirst%3DM.%26aulast%3DAtkinson%26aufirst%3DS.%2BJ.%26aulast%3DBassil%26aufirst%3DA.%2BK.%26aulast%3DRoberts%26aufirst%3DE.%2BJ.%26aulast%3DVassiliadis%26aufirst%3DD.%26aulast%3DBurr%26aufirst%3DM.%2BL.%26aulast%3DPreston%26aufirst%3DA.%2BG.%2BS.%26aulast%3DWellaway%26aufirst%3DC.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DGray%26aufirst%3DJ.%2BR.%26aulast%3DMichon%26aufirst%3DA.-M.%26aulast%3DGobbetti%26aufirst%3DT.%26aulast%3DKumar%26aufirst%3DV.%26aulast%3DSoden%26aufirst%3DP.%2BE.%26aulast%3DHaynes%26aufirst%3DA.%26aulast%3DVappiani%26aufirst%3DJ.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DDawson%26aufirst%3DS.-J.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DDawson%26aufirst%3DM.%2BA.%26atitle%3DSelective%2520targeting%2520of%2520BD1%2520and%2520BD2%2520of%2520the%2520BET%2520proteins%2520in%2520cancer%2520and%2520immuno-inflammation%26jtitle%3DScience%26date%3D2020%26volume%3D368%26spage%3D387%26epage%3D394%26doi%3D10.1126%2Fscience.aaz8455" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Nalawansha, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">PROTACs: An emerging therapeutic modality in precision medicine</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">998</span>– <span class="NLM_lpage">1014</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2020.07.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1016%2Fj.chembiol.2020.07.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=32795419" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhs1Wrs7vM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2020&pages=998-1014&author=D.+A.+Nalawanshaauthor=C.+M.+Crews&title=PROTACs%3A+An+emerging+therapeutic+modality+in+precision+medicine&doi=10.1016%2Fj.chembiol.2020.07.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6aR"><div class="casContent"><span class="casTitleNuber">6a</span><div class="casTitle"><span class="NLM_cas:atitle">PROTACs: An Emerging Therapeutic Modality in Precision Medicine</span></div><div class="casAuthors">Nalawansha, Dhanusha A.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">998-1014</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.,.  Targeted protein degrdn. (TPD) has emerged as an exciting new era in chem. biol. and drug discovery.  PROteolysis TArgeting Chimera (PROTAC) technol. targets cellular proteins for degrdn. by co-opting the ubiquitin-proteasome system.  Over the last 5 years, numerous studies have expanded our understanding of the unique mode of action and advantages of PROTACs, which has in turn spurred interest in both academia and industry to explore PROTACs as a novel therapeutic strategy.  In this review, we first highlight the key advantages of PROTACs and then discuss the spatiotemporal regulation of protein degrdn.  Next, we explore current chem. tractable E3 ligases focusing on expanding the existing repertoire with novel E3 ligases to uncover the full potential of TPD.  Collectively, these studies are guiding the development of the PROTAC technol. as it emerges as a new modality in precision medicine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4GkJzvvYnp7Vg90H21EOLACvtfcHk0ljLEQLch9lfVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhs1Wrs7vM&md5=63c4eb78ee4fc9272b9ae2409889861e</span></div><a href="/servlet/linkout?suffix=cit6a&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2020.07.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2020.07.020%26sid%3Dliteratum%253Aachs%26aulast%3DNalawansha%26aufirst%3DD.%2BA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DPROTACs%253A%2520An%2520emerging%2520therapeutic%2520modality%2520in%2520precision%2520medicine%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2020%26volume%3D27%26spage%3D998%26epage%3D1014%26doi%3D10.1016%2Fj.chembiol.2020.07.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit6b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calabrese, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bullock, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeted protein degradation: expanding the toolbox</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">949</span>– <span class="NLM_lpage">963</span>, <span class="refDoi"> DOI: 10.1038/s41573-019-0047-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1038%2Fs41573-019-0047-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=31666732" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitVKrtLrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=949-963&author=M.+Schapiraauthor=M.+F.+Calabreseauthor=A.+N.+Bullockauthor=C.+M.+Crews&title=Targeted+protein+degradation%3A+expanding+the+toolbox&doi=10.1038%2Fs41573-019-0047-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6bR"><div class="casContent"><span class="casTitleNuber">6b</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted protein degradation: expanding the toolbox</span></div><div class="casAuthors">Schapira, Matthieu; Calabrese, Matthew F.; Bullock, Alex N.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">949-963</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review. Proteolysis-targeting chimeras (PROTACs) and related mols. that induce targeted protein degrdn. by the ubiquitin-proteasome system represent a new therapeutic modality and are the focus of great interest, owing to potential advantages over traditional occupancy-based inhibitors with respect to dosing, side effects, drug resistance and modulating 'undruggable' targets.  However, the technol. is still maturing, and the design elements for successful PROTAC-based drugs are currently being elucidated.  Importantly, fewer than 10 of the more than 600 E3 ubiquitin ligases have so far been exploited for targeted protein degrdn., and expansion of knowledge in this area is a key opportunity.  Here, we briefly discuss lessons learned about targeted protein degrdn. in chem. biol. and drug discovery and systematically review the expression profile, domain architecture and chem. tractability of human E3 ligases that could expand the toolbox for PROTAC discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwUvM27MU3WrVg90H21EOLACvtfcHk0ljLEQLch9lfVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitVKrtLrM&md5=9bfd48c6c47fc1cce15129495fea7079</span></div><a href="/servlet/linkout?suffix=cit6b&amp;dbid=16384&amp;doi=10.1038%2Fs41573-019-0047-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-019-0047-y%26sid%3Dliteratum%253Aachs%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DCalabrese%26aufirst%3DM.%2BF.%26aulast%3DBullock%26aufirst%3DA.%2BN.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeted%2520protein%2520degradation%253A%2520expanding%2520the%2520toolbox%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D949%26epage%3D963%26doi%3D10.1038%2Fs41573-019-0047-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit6c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Small-molecule modulation of protein homeostasis</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">11269</span>– <span class="NLM_lpage">11301</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.7b00077</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.7b00077" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1Ohur3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2017&pages=11269-11301&author=G.+M.+Burslemauthor=C.+M.+Crews&title=Small-molecule+modulation+of+protein+homeostasis&doi=10.1021%2Facs.chemrev.7b00077"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6cR"><div class="casContent"><span class="casTitleNuber">6c</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Modulation of Protein Homeostasis</span></div><div class="casAuthors">Burslem, George M.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">11269-11301</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Control of protein levels by nucleic-acid-based technologies has proven to be a useful research tool but lacks the advantages of small mols. with respect to cell permeability, temporal control, and the potential generation of therapeutics.  In this Review, we discuss the technologies available for the control of intracellular protein levels with small mols. and compare the various systems available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcARydbgPK9bVg90H21EOLACvtfcHk0ljLEQLch9lfVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1Ohur3K&md5=f9f8dd98e2c71c7166d2d500b514d578</span></div><a href="/servlet/linkout?suffix=cit6c&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.7b00077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.7b00077%26sid%3Dliteratum%253Aachs%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DSmall-molecule%2520modulation%2520of%2520protein%2520homeostasis%26jtitle%3DChem.%2520Rev.%26date%3D2017%26volume%3D117%26spage%3D11269%26epage%3D11301%26doi%3D10.1021%2Facs.chemrev.7b00077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhimschi, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamman, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishchenko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Lessons in PROTAC design from selective degradation with a promiscuous warhead</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">78</span>– <span class="NLM_lpage">87</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.09.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1016%2Fj.chembiol.2017.09.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=29129718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=78-87&author=D.+P.+Bondesonauthor=B.+E.+Smithauthor=G.+M.+Burslemauthor=A.+D.+Buhimschiauthor=J.+Hinesauthor=S.+Jaime-Figueroaauthor=J.+Wangauthor=B.+D.+Hammanauthor=A.+Ishchenkoauthor=C.+M.+Crews&title=Lessons+in+PROTAC+design+from+selective+degradation+with+a+promiscuous+warhead&doi=10.1016%2Fj.chembiol.2017.09.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7aR"><div class="casContent"><span class="casTitleNuber">7a</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons in PROTAC Design from Selective Degradation with a Promiscuous Warhead</span></div><div class="casAuthors">Bondeson, Daniel P.; Smith, Blake E.; Burslem, George M.; Buhimschi, Alexandru D.; Hines, John; Jaime-Figueroa, Saul; Wang, Jing; Hamman, Brian D.; Ishchenko, Alexey; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">78-87.e5</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Inhibiting protein function selectively is a major goal of modern drug discovery.  Here, we report a previously understudied benefit of small mol. proteolysis-targeting chimeras (PROTACs) that recruit E3 ubiquitin ligases to target proteins for their ubiquitination and subsequent proteasome-mediated degrdn.  Using promiscuous CRBN- and VHL-recruiting PROTACs that bind >50 kinases, we show that only a subset of bound targets is degraded.  The basis of this selectivity relies on protein-protein interactions between the E3 ubiquitin ligase and the target protein, as illustrated by engaged proteins that are not degraded as a result of unstable ternary complexes with PROTAC-recruited E3 ligases.  In contrast, weak PROTAC:target protein affinity can be stabilized by high-affinity target:PROTAC:ligase trimer interactions, leading to efficient degrdn.  This study highlights design guidelines for generating potent PROTACs as well as possibilities for degrading undruggable proteins immune to traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoISOBudOLyFrVg90H21EOLACvtfcHk0lg0NuKHzPnC2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN&md5=30fac502705a2344c4261cf531e6bf9d</span></div><a href="/servlet/linkout?suffix=cit7a&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.09.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.09.010%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DSmith%26aufirst%3DB.%2BE.%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DBuhimschi%26aufirst%3DA.%2BD.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DIshchenko%26aufirst%3DA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DLessons%2520in%2520PROTAC%2520design%2520from%2520selective%2520degradation%2520with%2520a%2520promiscuous%2520warhead%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D78%26epage%3D87%26doi%3D10.1016%2Fj.chembiol.2017.09.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit7b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Remillard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brien, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sonnett, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dastjerdi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wghr, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Degradation of the BAF complex factor BRD9 by heterobifunctional ligands</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">5738</span>– <span class="NLM_lpage">5743</span>, <span class="refDoi"> DOI: 10.1002/anie.201611281</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1002%2Fanie.201611281" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtFGrurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=5738-5743&author=D.+Remillardauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=G.+L.+Brienauthor=M.+Sonnettauthor=H.-S.+Seoauthor=S.+Dastjerdiauthor=M.+Wghrauthor=S.+Dhe-Paganonauthor=S.+A.+Armstrongauthor=J.+E.+Bradner&title=Degradation+of+the+BAF+complex+factor+BRD9+by+heterobifunctional+ligands&doi=10.1002%2Fanie.201611281"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7bR"><div class="casContent"><span class="casTitleNuber">7b</span><div class="casTitle"><span class="NLM_cas:atitle">Degradation of the BAF Complex Factor BRD9 by Heterobifunctional Ligands</span></div><div class="casAuthors">Remillard, David; Buckley, Dennis L.; Paulk, Joshiawa; Brien, Gerard L.; Sonnett, Matthew; Seo, Hyuk-Soo; Dastierdi, Shiva; Wuehr, Martin; Dhe-Paganon, Sirano; Armstrong, Scott A.; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">5738-5743</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The bromodomain-contg. protein BRD9, a subunit of the human BAF (SWI/SNF) nucleosome remodeling complex, has emerged as an attractive therapeutic target in cancer.  Despite the development of chem. probes targeting the BRD9 bromodomain, there is a limited understanding of BRD9 function beyond acetyl-lysine recognition.  We have therefore created the first BRD9-directed chem. degraders, through iterative design and testing of heterobifunctional ligands that bridge the BRD9 bromodomain and the cereblon E3 ubiquitin ligase complex.  Degraders of BRD9 exhibit markedly enhanced potency compared to parental ligands (10- to 100-fold).  Parallel study of degraders with divergent BRD9-binding chemotypes in models of acute myeloid leukemia resolves bromodomain polypharmacol. in this emerging drug class.  Together, these findings reveal the tractability of non-BET bromodomain contg. proteins to chem. degrdn., and highlight lead compd. dBRD9 as a tool for the study of BRD9.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQ9FU2Ipaq27Vg90H21EOLACvtfcHk0lg0NuKHzPnC2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtFGrurk%253D&md5=2eb6562cc9d976c7ae6070c9a419e9ea</span></div><a href="/servlet/linkout?suffix=cit7b&amp;dbid=16384&amp;doi=10.1002%2Fanie.201611281&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201611281%26sid%3Dliteratum%253Aachs%26aulast%3DRemillard%26aufirst%3DD.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DBrien%26aufirst%3DG.%2BL.%26aulast%3DSonnett%26aufirst%3DM.%26aulast%3DSeo%26aufirst%3DH.-S.%26aulast%3DDastjerdi%26aufirst%3DS.%26aulast%3DWghr%26aufirst%3DM.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DDegradation%2520of%2520the%2520BAF%2520complex%2520factor%2520BRD9%2520by%2520heterobifunctional%2520ligands%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26spage%3D5738%26epage%3D5743%26doi%3D10.1002%2Fanie.201611281" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morita, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ujikawa, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohoka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koyama, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sano, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imaeda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naito, M.</span></span> <span> </span><span class="NLM_article-title">Development of protein degradation inducers of androgen receptor by conjugation of androgen receptor ligands and inhibitor of apoptosis protein ligands</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">543</span>– <span class="NLM_lpage">575</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00168</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00168" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpslSlu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=543-575&author=N.+Shibataauthor=K.+Nagaiauthor=Y.+Moritaauthor=O.+Ujikawaauthor=N.+Ohokaauthor=T.+Hattoriauthor=R.+Koyamaauthor=O.+Sanoauthor=Y.+Imaedaauthor=H.+Naraauthor=N.+Choauthor=M.+Naito&title=Development+of+protein+degradation+inducers+of+androgen+receptor+by+conjugation+of+androgen+receptor+ligands+and+inhibitor+of+apoptosis+protein+ligands&doi=10.1021%2Facs.jmedchem.7b00168"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8aR"><div class="casContent"><span class="casTitleNuber">8a</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Protein Degradation Inducers of Androgen Receptor by Conjugation of Androgen Receptor Ligands and Inhibitor of Apoptosis Protein Ligands</span></div><div class="casAuthors">Shibata, Norihito; Nagai, Katsunori; Morita, Yoko; Ujikawa, Osamu; Ohoka, Nobumichi; Hattori, Takayuki; Koyama, Ryokichi; Sano, Osamu; Imaeda, Yasuhiro; Nara, Hiroshi; Cho, Nobuo; Naito, Mikihiko</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">543-575</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Targeted protein degrdn. using small mols. is a novel strategy for drug development.  The authors have developed hybrid mols. named specific and nongenetic inhibitor of apoptosis protein [IAP]-dependent protein erasers (SNIPERs) that recruit IAP ubiquitin ligases to degrade target proteins.  Here, the authors show novel SNIPERs capable of inducing proteasomal degrdn. of the androgen receptor (AR).  Through derivatization of the SNIPER(AR) mol. at the AR ligand and IAP ligand and linker, the authors developed SNIPER(AR)-51 (I), which shows effective protein knockdown activity against AR.  Consistent with the degrdn. of the AR protein, I inhibits AR-mediated gene expression and proliferation of androgen-dependent prostate cancer cells.  In addn., I efficiently induces caspase activation and apoptosis in prostate cancer cells, which was not obsd. in the cells treated with AR antagonists.  These results suggest that SNIPER(AR)s could be leads for an anticancer drug against prostate cancers that exhibit AR-dependent proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqW6OG4BZYg1rVg90H21EOLACvtfcHk0lg0NuKHzPnC2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpslSlu7k%253D&md5=eeefbe187f2f93ca32518f6a3258b903</span></div><a href="/servlet/linkout?suffix=cit8a&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00168&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00168%26sid%3Dliteratum%253Aachs%26aulast%3DShibata%26aufirst%3DN.%26aulast%3DNagai%26aufirst%3DK.%26aulast%3DMorita%26aufirst%3DY.%26aulast%3DUjikawa%26aufirst%3DO.%26aulast%3DOhoka%26aufirst%3DN.%26aulast%3DHattori%26aufirst%3DT.%26aulast%3DKoyama%26aufirst%3DR.%26aulast%3DSano%26aufirst%3DO.%26aulast%3DImaeda%26aufirst%3DY.%26aulast%3DNara%26aufirst%3DH.%26aulast%3DCho%26aufirst%3DN.%26aulast%3DNaito%26aufirst%3DM.%26atitle%3DDevelopment%2520of%2520protein%2520degradation%2520inducers%2520of%2520androgen%2520receptor%2520by%2520conjugation%2520of%2520androgen%2520receptor%2520ligands%2520and%2520inhibitor%2520of%2520apoptosis%2520protein%2520ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D543%26epage%3D575%26doi%3D10.1021%2Facs.jmedchem.7b00168" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit8b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Ohoka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okuhira, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ujikawa, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimokawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sano, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koyama, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujita, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teratani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imaeda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naito, M.</span></span> <span> </span><span class="NLM_article-title"><i>In vivo</i> knockdown of pathogenic proteins via specific and nongenetic inhibitor of apoptosis protein (IAP)-dependent protein erasers (SNIPERs)</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>292</i></span>,  <span class="NLM_fpage">4556</span>– <span class="NLM_lpage">4570</span>, <span class="refDoi"> DOI: 10.1074/jbc.M116.768853</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1074%2Fjbc.M116.768853" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=28154167" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXksVClur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=292&publication_year=2017&pages=4556-4570&author=N.+Ohokaauthor=K.+Okuhiraauthor=M.+Itoauthor=K.+Nagaiauthor=N.+Shibataauthor=T.+Hattoriauthor=O.+Ujikawaauthor=K.+Shimokawaauthor=O.+Sanoauthor=R.+Koyamaauthor=H.+Fujitaauthor=M.+Terataniauthor=H.+Matsumotoauthor=Y.+Imaedaauthor=H.+Naraauthor=N.+Choauthor=M.+Naito&title=In+vivo+knockdown+of+pathogenic+proteins+via+specific+and+nongenetic+inhibitor+of+apoptosis+protein+%28IAP%29-dependent+protein+erasers+%28SNIPERs%29&doi=10.1074%2Fjbc.M116.768853"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8bR"><div class="casContent"><span class="casTitleNuber">8b</span><div class="casTitle"><span class="NLM_cas:atitle">In Vivo Knockdown of Pathogenic Proteins via Specific and Nongenetic Inhibitor of Apoptosis Protein (IAP)-dependent Protein Erasers (SNIPERs)</span></div><div class="casAuthors">Ohoka, Nobumichi; Okuhira, Keiichiro; Ito, Masahiro; Nagai, Katsunori; Shibata, Norihito; Hattori, Takayuki; Ujikawa, Osamu; Shimokawa, Kenichiro; Sano, Osamu; Koyama, Ryokichi; Fujita, Hisashi; Teratani, Mika; Matsumoto, Hirokazu; Imaeda, Yasuhiro; Nara, Hiroshi; Cho, Nobuo; Naito, Mikihiko</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">292</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4556-4570</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Many diseases, esp. cancers, result from aberrant or overexpression of pathogenic proteins.  Specific inhibitors against these proteins have shown remarkable therapeutic effects, but these are limited mainly to enzymes.  An alternative approach that may have utility in drug development relies on selective degrdn. of pathogenic proteins via small chimeric mols. linking an E3 ubiquitin ligase to the targeted protein for proteasomal degrdn.  To this end, the authors recently developed a protein knockdown system based on hybrid small mol. SNIPERs (Specific and Nongenetic IAP-dependent Protein Erasers) that recruit inhibitor of the apoptosis protein (IAP) ubiquitin ligases to specifically degrade targeted proteins.  Here, the authors extend the authors' previous study to show a proof of concept of the SNIPER technol. in vivo.  By incorporating a high affinity IAP ligand, the authors developed a novel SNIPER against estrogen receptor α (ERα), SNIPER(ER)-87, that has a potent protein knockdown activity.  The SNIPER(ER) reduced ERα levels in tumor xenografts and suppressed the growth of ERα-pos. breast tumors in mice.  Mechanistically, it preferentially recruits X-linked IAP (XIAP) rather than cellular IAP1, to degrade ERα via the ubiquitin-proteasome pathway.  With this IAP ligand, potent SNIPERs against other pathogenic proteins, BCR-ABL, bromodomain-contg. protein 4 (BRD4), and phosphodiesterase-4 (PDE4) could also be developed.  These results indicate that forced ubiquitylation by SNIPERs is a useful method to achieve efficient protein knockdown with potential therapeutic activities and could also be applied to study the role of ubiquitylation in many cellular processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEzH5uNeAFabVg90H21EOLACvtfcHk0lji3BYlW650yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXksVClur8%253D&md5=b09264d53c47c54343ed92d5a6d0eb79</span></div><a href="/servlet/linkout?suffix=cit8b&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M116.768853&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M116.768853%26sid%3Dliteratum%253Aachs%26aulast%3DOhoka%26aufirst%3DN.%26aulast%3DOkuhira%26aufirst%3DK.%26aulast%3DIto%26aufirst%3DM.%26aulast%3DNagai%26aufirst%3DK.%26aulast%3DShibata%26aufirst%3DN.%26aulast%3DHattori%26aufirst%3DT.%26aulast%3DUjikawa%26aufirst%3DO.%26aulast%3DShimokawa%26aufirst%3DK.%26aulast%3DSano%26aufirst%3DO.%26aulast%3DKoyama%26aufirst%3DR.%26aulast%3DFujita%26aufirst%3DH.%26aulast%3DTeratani%26aufirst%3DM.%26aulast%3DMatsumoto%26aufirst%3DH.%26aulast%3DImaeda%26aufirst%3DY.%26aulast%3DNara%26aufirst%3DH.%26aulast%3DCho%26aufirst%3DN.%26aulast%3DNaito%26aufirst%3DM.%26atitle%3DIn%2520vivo%2520knockdown%2520of%2520pathogenic%2520proteins%2520via%2520specific%2520and%2520nongenetic%2520inhibitor%2520of%2520apoptosis%2520protein%2520%2528IAP%2529-dependent%2520protein%2520erasers%2520%2528SNIPERs%2529%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2017%26volume%3D292%26spage%3D4556%26epage%3D4570%26doi%3D10.1074%2Fjbc.M116.768853" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of a small-molecule degrader of bromodomain and extra-terminal (BET) proteins with picomolar cellular potencies and capable of achieving tumor regression</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">462</span>– <span class="NLM_lpage">481</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01816</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01816" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvVajsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=462-481&author=B.+Zhouauthor=J.+Huauthor=F.+Xuauthor=Z.+Chenauthor=L.+Baiauthor=E.+Fernandez-Salasauthor=M.+Linauthor=L.+Liuauthor=C.-Y.+Yangauthor=Y.+Zhaoauthor=D.+McEachernauthor=S.+Przybranowskiauthor=B.+Wenauthor=D.+Sunauthor=S.+Wang&title=Discovery+of+a+small-molecule+degrader+of+bromodomain+and+extra-terminal+%28BET%29+proteins+with+picomolar+cellular+potencies+and+capable+of+achieving+tumor+regression&doi=10.1021%2Facs.jmedchem.6b01816"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9aR"><div class="casContent"><span class="casTitleNuber">9a</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Small-Molecule Degrader of Bromodomain and Extra-Terminal (BET) Proteins with Picomolar Cellular Potencies and Capable of Achieving Tumor Regression</span></div><div class="casAuthors">Zhou, Bing; Hu, Jiantao; Xu, Fuming; Chen, Zhuo; Bai, Longchuan; Fernandez-Salas, Ester; Lin, Mei; Liu, Liu; Yang, Chao-Yie; Zhao, Yujun; McEachern, Donna; Przybranowski, Sally; Wen, Bo; Sun, Duxin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">462-481</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extra-terminal (BET) family proteins, consisting of BRD2, BRD3, BRD4, and testis-specific BRDT members, are epigenetic "readers" and play a key role in the regulation of gene transcription.  BET proteins are considered to be attractive therapeutic targets for cancer and other human diseases.  Recently, heterobifunctional small-mol. BET degraders have been designed based upon the proteolysis targeting chimera (PROTAC) concept to induce BET protein degrdn.  Herein, we present our design, synthesis, and evaluation of a new class of PROTAC BET degraders.  One of the most promising compds., 23, effectively degrades BRD4 protein at concns. as low as 30 pM in the RS4;11 leukemia cell line, achieves an IC50 value of 51 pM in inhibition of RS4;11 cell growth and induces rapid tumor regression in vivo against RS4;11 xenograft tumors.  These data establish that compd. 23 (BETd-260/ZBC260) is a highly potent and efficacious BET degrader.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqz2Lwp52x1rVg90H21EOLACvtfcHk0lji3BYlW650yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvVajsb8%253D&md5=6b3a10ecd192a5812ef581dca1fdc82a</span></div><a href="/servlet/linkout?suffix=cit9a&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01816&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01816%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DLin%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520a%2520small-molecule%2520degrader%2520of%2520bromodomain%2520and%2520extra-terminal%2520%2528BET%2529%2520proteins%2520with%2520picomolar%2520cellular%2520potencies%2520and%2520capable%2520of%2520achieving%2520tumor%2520regression%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D462%26epage%3D481%26doi%3D10.1021%2Facs.jmedchem.6b01816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit9b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziazadeh, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of QCA570 as an exceptionally potent and efficacious proteolysis targeting chimera (PROTAC) degrader of the bromodomain and extra-terminal (BET) proteins capable of inducing complete and durable tumor regression</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">6685</span>– <span class="NLM_lpage">6704</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00506</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00506" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlaqsLrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6685-6704&author=C.+Qinauthor=Y.+Huauthor=B.+Zhouauthor=E.+Fernandez-Salasauthor=C.-Y.+Yangauthor=L.+Liuauthor=D.+McEachernauthor=S.+Przybranowskiauthor=M.+Wangauthor=J.+Stuckeyauthor=J.+Meagherauthor=L.+Baiauthor=Z.+Chenauthor=M.+Linauthor=J.+Yangauthor=D.+N.+Ziazadehauthor=F.+Xuauthor=J.+Huauthor=W.+Xiangauthor=L.+Huangauthor=S.+Liauthor=B.+Wenauthor=D.+Sunauthor=S.+Wang&title=Discovery+of+QCA570+as+an+exceptionally+potent+and+efficacious+proteolysis+targeting+chimera+%28PROTAC%29+degrader+of+the+bromodomain+and+extra-terminal+%28BET%29+proteins+capable+of+inducing+complete+and+durable+tumor+regression&doi=10.1021%2Facs.jmedchem.8b00506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9bR"><div class="casContent"><span class="casTitleNuber">9b</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of QCA570 as an Exceptionally Potent and Efficacious Proteolysis Targeting Chimera (PROTAC) Degrader of the Bromodomain and Extra-Terminal (BET) Proteins Capable of Inducing Complete and Durable Tumor Regression</span></div><div class="casAuthors">Qin, Chong; Hu, Yang; Zhou, Bing; Fernandez-Salas, Ester; Yang, Chao-Yie; Liu, Liu; McEachern, Donna; Przybranowski, Sally; Wang, Mi; Stuckey, Jeanne; Meagher, Jennifer; Bai, Longchuan; Chen, Zhuo; Lin, Mei; Yang, Jiuling; Ziazadeh, Danya N.; Xu, Fuming; Hu, Jiantao; Xing, Weiguo; Huang, Liyue; Li, Siwei; Wen, Bo; Sun, Duxin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6685-6704</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proteins of the bromodomain and extra-terminal (BET) family are epigenetics "readers" and promising therapeutic targets for cancer and other human diseases.  We describe herein a structure-guided design of [1,4]oxazepines as a new class of BET inhibitors and our subsequent design, synthesis, and evaluation of proteolysis-targeting chimeric (PROTAC) small-mol. BET degraders.  Our efforts have led to the discovery of extremely potent BET degraders, exemplified by QCA570, which effectively induces degrdn. of BET proteins and inhibits cell growth in human acute leukemia cell lines even at low picomolar concns.  QCA570 achieves complete and durable tumor regression in leukemia xenograft models in mice at well-tolerated dose-schedules.  QCA570 is the most potent and efficacious BET degrader reported to date.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6ZUIsfX3p6LVg90H21EOLACvtfcHk0lh4sagtbWckOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlaqsLrE&md5=2a731730641372f4526543e03d9ac76f</span></div><a href="/servlet/linkout?suffix=cit9b&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00506%26sid%3Dliteratum%253Aachs%26aulast%3DQin%26aufirst%3DC.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DMeagher%26aufirst%3DJ.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DLin%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DZiazadeh%26aufirst%3DD.%2BN.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DXiang%26aufirst%3DW.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520QCA570%2520as%2520an%2520exceptionally%2520potent%2520and%2520efficacious%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%2520degrader%2520of%2520the%2520bromodomain%2520and%2520extra-terminal%2520%2528BET%2529%2520proteins%2520capable%2520of%2520inducing%2520complete%2520and%2520durable%2520tumor%2520regression%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D6685%26epage%3D6704%26doi%3D10.1021%2Facs.jmedchem.8b00506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit9c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, A. M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K. G.</span></span> <span> </span><span class="NLM_article-title">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">7124</span>– <span class="NLM_lpage">7129</span>, <span class="refDoi"> DOI: 10.1073/pnas.1521738113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1073%2Fpnas.1521738113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=27274052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=7124-7129&author=K.+Rainaauthor=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=D.+Gordonauthor=A.+M.+K.+Rossiauthor=J.+Wangauthor=X.+Chenauthor=H.+Dongauthor=K.+Siuauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=C.+M.+Crewsauthor=K.+G.+Coleman&title=PROTAC-induced+BET+protein+degradation+as+a+therapy+for+castration-resistant+prostate+cancer&doi=10.1073%2Fpnas.1521738113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9cR"><div class="casContent"><span class="casTitleNuber">9c</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span></div><div class="casAuthors">Raina, Kanak; Lu, Jing; Qian, Yimin; Altieri, Martha; Gordon, Deborah; Rossi, Ann Marie K.; Wang, Jing; Chen, Xin; Dong, Hanqing; Siu, Kam; Winkler, James D.; Crew, Andrew P.; Crews, Craig M.; Coleman, Kevin G.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">7124-7129</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Prostate cancer has the second highest incidence among cancers in men worldwide and is the second leading cause of cancer deaths of men in the United States.  Although androgen deprivation can initially lead to remission, the disease often progresses to castration-resistant prostate cancer (CRPC), which is still reliant on androgen receptor (AR) signaling and is assocd. with a poor prognosis.  Some success against CRPC has been achieved by drugs that target AR signaling, but secondary resistance invariably emerges, and new therapies are urgently needed.  Recently, inhibitors of bromodomain and extra-terminal (BET) family proteins have shown growth-inhibitory activity in preclin. models of CRPC.  Here, we demonstrate that ARV-771, a small-mol. pan-BET degrader based on proteolysis-targeting chimera (PROTAC) technol., demonstrates dramatically improved efficacy in cellular models of CRPC as compared with BET inhibition.  Unlike BET inhibitors, ARV-771 results in suppression of both AR signaling and AR levels and leads to tumor regression in a CRPC mouse xenograft model.  This study is, to our knowledge, the first to demonstrate efficacy with a small-mol. BET degrader in a solid-tumor malignancy and potentially represents an important therapeutic advance in the treatment of CRPC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD7QtYgZVig7Vg90H21EOLACvtfcHk0lh4sagtbWckOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D&md5=34218e685ad72950f3eb85586110c43d</span></div><a href="/servlet/linkout?suffix=cit9c&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1521738113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1521738113%26sid%3Dliteratum%253Aachs%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DGordon%26aufirst%3DD.%26aulast%3DRossi%26aufirst%3DA.%2BM.%2BK.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DSiu%26aufirst%3DK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26atitle%3DPROTAC-induced%2520BET%2520protein%2520degradation%2520as%2520a%2520therapy%2520for%2520castration-resistant%2520prostate%2520cancer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2016%26volume%3D113%26spage%3D7124%26epage%3D7129%26doi%3D10.1073%2Fpnas.1521738113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit9d"><span><span class="NLM_label">(d) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Selective small molecule induced degradation of the BET bromodomain protein BRD4</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1770</span>– <span class="NLM_lpage">1777</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00216</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00216" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1770-1777&author=M.+Zengerleauthor=K.-H.+Chanauthor=A.+Ciulli&title=Selective+small+molecule+induced+degradation+of+the+BET+bromodomain+protein+BRD4&doi=10.1021%2Facschembio.5b00216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9dR"><div class="casContent"><span class="casTitleNuber">9d</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Small Molecule Induced Degradation of the BET Bromodomain Protein BRD4</span></div><div class="casAuthors">Zengerle, Michael; Chan, Kwok-Ho; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1770-1777</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Bromo- and Extra-Terminal (BET) proteins BRD2, BRD3, and BRD4 play important roles in transcriptional regulation, epigenetics, and cancer and are the targets of pan-BET selective bromodomain inhibitor JQ1.  However, the lack of intra-BET selectivity limits the scope of current inhibitors as probes for target validation and could lead to unwanted side effects or toxicity in a therapeutic setting.  We designed Proteolysis Targeted Chimeras (PROTACs) that tether JQ1 to a ligand for the E3 ubiquitin ligase VHL, aimed at triggering the intracellular destruction of BET proteins.  Compd. MZ1 potently and rapidly induces reversible, long-lasting, and unexpectedly selective removal of BRD4 over BRD2 and BRD3.  The activity of MZ1 is dependent on binding to VHL but is achieved at a sufficiently low concn. not to induce stabilization of HIF-1α.  Gene expression profiles of selected cancer-related genes responsive to JQ1 reveal distinct and more limited transcriptional responses induced by MZ1, consistent with selective suppression of BRD4.  Our discovery opens up new opportunities to elucidate the cellular phenotypes and therapeutic implications assocd. with selective targeting of BRD4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEmSv7p9MZOrVg90H21EOLACvtfcHk0lh4sagtbWckOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D&md5=b7514fd274eff8e4bb6a771f5aaf9ab5</span></div><a href="/servlet/linkout?suffix=cit9d&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00216%26sid%3Dliteratum%253Aachs%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DK.-H.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DSelective%2520small%2520molecule%2520induced%2520degradation%2520of%2520the%2520BET%2520bromodomain%2520protein%2520BRD4%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D1770%26epage%3D1777%26doi%3D10.1021%2Facschembio.5b00216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit9e"><span><span class="NLM_label">(e) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Phthalimide conjugation as a strategy for in vivo target protein degradation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>348</i></span>,  <span class="NLM_fpage">1376</span>– <span class="NLM_lpage">1381</span>, <span class="refDoi"> DOI: 10.1126/science.aab1433</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9e&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1126%2Fscience.aab1433" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9e&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=25999370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9e&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=348&publication_year=2015&pages=1376-1381&author=G.+E.+Winterauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=J.+M.+Robertsauthor=A.+Souzaauthor=S.+Dhe-Paganonauthor=J.+E.+Bradner&title=Phthalimide+conjugation+as+a+strategy+for+in+vivo+target+protein+degradation&doi=10.1126%2Fscience.aab1433"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9eR"><div class="casContent"><span class="casTitleNuber">9e</span><div class="casTitle"><span class="NLM_cas:atitle">Phthalimide conjugation as a strategy for in vivo target protein degradation</span></div><div class="casAuthors">Winter, Georg E.; Buckley, Dennis L.; Paulk, Joshiawa; Roberts, Justin M.; Souza, Amanda; Dhe-Paganon, Sirano; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">348</span>
        (<span class="NLM_cas:issue">6241</span>),
    <span class="NLM_cas:pages">1376-1381</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The development of effective pharmacol. inhibitors of multidomain scaffold proteins, notably transcription factors, is a particularly challenging problem.  In part, this is because many small-mol. antagonists disrupt the activity of only one domain in the target protein.  The authors devised a chem. strategy that promotes ligand-dependent target protein degrdn. using as an example the transcriptional coactivator BRD4, a protein crit. for cancer cell growth and survival.  The authors appended a competitive antagonist of BET bromodomains to a phthalimide moiety to hijack the cereblon E3 ubiquitin ligase complex.  The resultant compd., dBET1, induced highly selective cereblon-dependent BET protein degrdn. in vitro and in vivo and delayed leukemia progression in mice.  A second series of probes resulted in selective degrdn. of the cytosolic protein FKBP12.  This chem. strategy for controlling target protein stability may have implications for therapeutically targeting previously intractable proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1ZW4zjRjXi7Vg90H21EOLACvtfcHk0lg3Au5-uvyToA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D&md5=b0277cc95d318d926b96e7a119066e3b</span></div><a href="/servlet/linkout?suffix=cit9e&amp;dbid=16384&amp;doi=10.1126%2Fscience.aab1433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aab1433%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DPhthalimide%2520conjugation%2520as%2520a%2520strategy%2520for%2520in%2520vivo%2520target%2520protein%2520degradation%26jtitle%3DScience%26date%3D2015%26volume%3D348%26spage%3D1376%26epage%3D1381%26doi%3D10.1126%2Fscience.aab1433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erb, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roderick, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vittori, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reyes, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">di Iulio, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeid, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lachance, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dastjerdi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelliher, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Churchman, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">BET bromodomain proteins function as master transcription elongation factors independent of CDK9 recruitment</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">5</span>– <span class="NLM_lpage">18</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2017.06.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1016%2Fj.molcel.2017.06.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=28673542" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFSmsbjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2017&pages=5-18&author=G.+E.+Winterauthor=A.+Mayerauthor=D.+L.+Buckleyauthor=M.+A.+Erbauthor=J.+E.+Roderickauthor=S.+Vittoriauthor=J.+M.+Reyesauthor=J.+di+Iulioauthor=A.+Souzaauthor=C.+J.+Ottauthor=J.+M.+Robertsauthor=R.+Zeidauthor=T.+G.+Scottauthor=J.+Paulkauthor=K.+Lachanceauthor=C.+M.+Olsonauthor=S.+Dastjerdiauthor=S.+Bauerauthor=C.+Y.+Linauthor=N.+S.+Grayauthor=M.+A.+Kelliherauthor=L.+S.+Churchmanauthor=J.+E.+Bradner&title=BET+bromodomain+proteins+function+as+master+transcription+elongation+factors+independent+of+CDK9+recruitment&doi=10.1016%2Fj.molcel.2017.06.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">BET Bromodomain Proteins Function as Master Transcription Elongation Factors Independent of CDK9 Recruitment</span></div><div class="casAuthors">Winter, Georg E.; Mayer, Andreas; Buckley, Dennis L.; Erb, Michael A.; Roderick, Justine E.; Vittori, Sarah; Reyes, Jaime M.; di Iulio, Julia; Souza, Amanda; Ott, Christopher J.; Roberts, Justin M.; Zeid, Rhamy; Scott, Thomas G.; Paulk, Joshiawa; Lachance, Kate; Olson, Calla M.; Dastjerdi, Shiva; Bauer, Sophie; Lin, Charles Y.; Gray, Nathanael S.; Kelliher, Michelle A.; Churchman, L. Stirling; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-18.e19</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Processive elongation of RNA Polymerase II from a proximal promoter paused state is a rate-limiting event in human gene control.  A small no. of regulatory factors influence transcription elongation on a global scale.  Prior research using small-mol. BET bromodomain inhibitors, such as JQ1, linked BRD4 to context-specific elongation at a limited no. of genes assocd. with massive enhancer regions.  Here, the mechanistic characterization of an optimized chem. degrader of BET bromodomain proteins, dBET6, led to the unexpected identification of BET proteins as master regulators of global transcription elongation.  In contrast to the selective effect of bromodomain inhibition on transcription, BET degrdn. prompts a collapse of global elongation that phenocopies CDK9 inhibition.  Notably, BRD4 loss does not directly affect CDK9 localization.  These studies, performed in translational models of T cell leukemia, establish a mechanism-based rationale for the development of BET bromodomain degrdn. as cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqseJAHmXAb-rVg90H21EOLACvtfcHk0lg3Au5-uvyToA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFSmsbjM&md5=ac6f4e33a8f43008b70d4ede80a526be</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2017.06.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2017.06.004%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DMayer%26aufirst%3DA.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DErb%26aufirst%3DM.%2BA.%26aulast%3DRoderick%26aufirst%3DJ.%2BE.%26aulast%3DVittori%26aufirst%3DS.%26aulast%3DReyes%26aufirst%3DJ.%2BM.%26aulast%3Ddi%2BIulio%26aufirst%3DJ.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DOtt%26aufirst%3DC.%2BJ.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DZeid%26aufirst%3DR.%26aulast%3DScott%26aufirst%3DT.%2BG.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DLachance%26aufirst%3DK.%26aulast%3DOlson%26aufirst%3DC.%2BM.%26aulast%3DDastjerdi%26aufirst%3DS.%26aulast%3DBauer%26aufirst%3DS.%26aulast%3DLin%26aufirst%3DC.%2BY.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DKelliher%26aufirst%3DM.%2BA.%26aulast%3DChurchman%26aufirst%3DL.%2BS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DBET%2520bromodomain%2520proteins%2520function%2520as%2520master%2520transcription%2520elongation%2520factors%2520independent%2520of%2520CDK9%2520recruitment%26jtitle%3DMol.%2520Cell%26date%3D2017%26volume%3D67%26spage%3D5%26epage%3D18%26doi%3D10.1016%2Fj.molcel.2017.06.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shultz, M. D.</span></span> <span> </span><span class="NLM_article-title">Two decades under the influence of the rule of five and the changing properties of approved oral drugs</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">1701</span>– <span class="NLM_lpage">1714</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00686</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00686" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslait73K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=1701-1714&author=M.+D.+Shultz&title=Two+decades+under+the+influence+of+the+rule+of+five+and+the+changing+properties+of+approved+oral+drugs&doi=10.1021%2Facs.jmedchem.8b00686"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Two Decades under the Influence of the Rule of Five and the Changing Properties of Approved Oral Drugs</span></div><div class="casAuthors">Shultz, Michael D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1701-1714</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Two decades have passed since the rule of five ushered in the concept of "drug-like" properties.  Attempts to quantify, correlate, and categorize mols. based on Ro5 parameters evolved into the introduction of efficiency metrics with far reaching consequences in decision making by industry leaders and scientists seeking to discover new medicines.  Examn. of oral drug parameters approved before and after the original Ro5 anal. demonstrates that some parameters such as clogP and HBD remained const. while the cutoffs for parameters such as mol. wt. and HBA have increased substantially over the past 20 years.  The time dependent increase in the mol. wt. of oral drugs during the past 20 years provides compelling evidence to disprove the hypothesis that mol. wt. is a "drug-like" property.  This anal. does not validate parameters that have not changed as being "drug-like" but instead calls into question the entire hypothesis that "drug-like" properties exist.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAnoxpQsv2kbVg90H21EOLACvtfcHk0lildWlBZZg1-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslait73K&md5=d3c8b1107133dc0b7b277d22fa029597</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00686&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00686%26sid%3Dliteratum%253Aachs%26aulast%3DShultz%26aufirst%3DM.%2BD.%26atitle%3DTwo%2520decades%2520under%2520the%2520influence%2520of%2520the%2520rule%2520of%2520five%2520and%2520the%2520changing%2520properties%2520of%2520approved%2520oral%2520drugs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D1701%26epage%3D1714%26doi%3D10.1021%2Facs.jmedchem.8b00686" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Edmondson, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fallan, C.</span></span> <span> </span><span class="NLM_article-title">Proteolysis targeting chimeras (PROTACs) in ‘beyond rule-of-five’ chemical space: Recent progress and future challenges</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1555</span>– <span class="NLM_lpage">1564</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2019.04.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1016%2Fj.bmcl.2019.04.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=31047748" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1MXos12gs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=1555-1564&author=S.+D.+Edmondsonauthor=B.+Yangauthor=C.+Fallan&title=Proteolysis+targeting+chimeras+%28PROTACs%29+in+%E2%80%98beyond+rule-of-five%E2%80%99+chemical+space%3A+Recent+progress+and+future+challenges&doi=10.1016%2Fj.bmcl.2019.04.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12aR"><div class="casContent"><span class="casTitleNuber">12a</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis targeting chimeras (PROTACs) in 'beyond rule-of-five' chemical space: Recent progress and future challenges</span></div><div class="casAuthors">Edmondson, Scott D.; Yang, Bin; Fallan, Charlene</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1555-1564</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Proteolysis targeting chimeras (PROTACs) are heterobifunctional compds. with mol. wts. and other properties that lie outside the classic 'rule-of-five' space.  Consequently, PROTACs have unique challenges assocd. with their development as potential therapeutic agents.  This review summarizes and analyzes a representative set of recent PROTACs and highlights some of the potential future challenges facing this promising modality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDcS0gIul-K7Vg90H21EOLACvtfcHk0lh6QLn2zN4V4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXos12gs74%253D&md5=62a74500178bd711b1ddaa33d731b960</span></div><a href="/servlet/linkout?suffix=cit12a&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2019.04.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2019.04.030%26sid%3Dliteratum%253Aachs%26aulast%3DEdmondson%26aufirst%3DS.%2BD.%26aulast%3DYang%26aufirst%3DB.%26aulast%3DFallan%26aufirst%3DC.%26atitle%3DProteolysis%2520targeting%2520chimeras%2520%2528PROTACs%2529%2520in%2520%25E2%2580%2598beyond%2520rule-of-five%25E2%2580%2599%2520chemical%2520space%253A%2520Recent%2520progress%2520and%2520future%2520challenges%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D29%26spage%3D1555%26epage%3D1564%26doi%3D10.1016%2Fj.bmcl.2019.04.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit12b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Maple, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clayden, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baron, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stacey, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felix, R.</span></span> <span> </span><span class="NLM_article-title">Developing degraders: principles and perspectives on design and chemical space</span>. <i>Med. Chem. Commum.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1755</span>– <span class="NLM_lpage">1764</span>, <span class="refDoi"> DOI: 10.1039/C9MD00272C</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1039%2FC9MD00272C" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1755-1764&author=H.+J.+Mapleauthor=N.+Claydenauthor=A.+Baronauthor=C.+Staceyauthor=R.+Felix&title=Developing+degraders%3A+principles+and+perspectives+on+design+and+chemical+space&doi=10.1039%2FC9MD00272C"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12b&amp;dbid=16384&amp;doi=10.1039%2FC9MD00272C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC9MD00272C%26sid%3Dliteratum%253Aachs%26aulast%3DMaple%26aufirst%3DH.%2BJ.%26aulast%3DClayden%26aufirst%3DN.%26aulast%3DBaron%26aufirst%3DA.%26aulast%3DStacey%26aufirst%3DC.%26aulast%3DFelix%26aufirst%3DR.%26atitle%3DDeveloping%2520degraders%253A%2520principles%2520and%2520perspectives%2520on%2520design%2520and%2520chemical%2520space%26jtitle%3DMed.%2520Chem.%2520Commum.%26date%3D2019%26volume%3D10%26spage%3D1755%26epage%3D1764%26doi%3D10.1039%2FC9MD00272C" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Petrylak, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogelzang, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garfield, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peck, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burris, H. A.</span></span> <span> </span><span class="NLM_article-title">First-in-human phase I study of ARV-110, an androgen receptor (AR) PROTAC degrader in patients (pts) with metastatic castrate-resistant prostate cancer (mCRPC) following enzalutamide (ENZ) and/or abiraterone (ABI)</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">3500</span>, <span class="refDoi"> DOI: 10.1200/JCO.2020.38.15_suppl.3500</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1200%2FJCO.2020.38.15_suppl.3500" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2020&pages=3500&author=D.+P.+Petrylakauthor=X.+Gaoauthor=N.+J.+Vogelzangauthor=M.+H.+Garfieldauthor=I.+Taylorauthor=M.+D.+Mooreauthor=R.+A.+Peckauthor=H.+A.+Burris&title=First-in-human+phase+I+study+of+ARV-110%2C+an+androgen+receptor+%28AR%29+PROTAC+degrader+in+patients+%28pts%29+with+metastatic+castrate-resistant+prostate+cancer+%28mCRPC%29+following+enzalutamide+%28ENZ%29+and%2For+abiraterone+%28ABI%29&doi=10.1200%2FJCO.2020.38.15_suppl.3500"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13a&amp;dbid=16384&amp;doi=10.1200%2FJCO.2020.38.15_suppl.3500&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2020.38.15_suppl.3500%26sid%3Dliteratum%253Aachs%26aulast%3DPetrylak%26aufirst%3DD.%2BP.%26aulast%3DGao%26aufirst%3DX.%26aulast%3DVogelzang%26aufirst%3DN.%2BJ.%26aulast%3DGarfield%26aufirst%3DM.%2BH.%26aulast%3DTaylor%26aufirst%3DI.%26aulast%3DMoore%26aufirst%3DM.%2BD.%26aulast%3DPeck%26aufirst%3DR.%2BA.%26aulast%3DBurris%26aufirst%3DH.%2BA.%26atitle%3DFirst-in-human%2520phase%2520I%2520study%2520of%2520ARV-110%252C%2520an%2520androgen%2520receptor%2520%2528AR%2529%2520PROTAC%2520degrader%2520in%2520patients%2520%2528pts%2529%2520with%2520metastatic%2520castrate-resistant%2520prostate%2520cancer%2520%2528mCRPC%2529%2520following%2520enzalutamide%2520%2528ENZ%2529%2520and%252For%2520abiraterone%2520%2528ABI%2529%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2020%26volume%3D38%26spage%3D3500%26doi%3D10.1200%2FJCO.2020.38.15_suppl.3500" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit13b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, R.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span> <span> </span><span class="NLM_article-title">A chemical approach for global protein knockdown from mice to non-human primates</span>. <i>Cell Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>5</i></span>, <span class="NLM_elocation-id">10</span> <span class="refDoi"> DOI: 10.1038/s41421-018-0079-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1038%2Fs41421-018-0079-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=30729032" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A280%3ADC%252BB3cjpvFKiug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2019&author=X.+Sunauthor=J.+Wangauthor=X.+Yaoauthor=W.+Zhengauthor=Y.+Maoauthor=T.+Lanauthor=L.+Wangauthor=Y.+Sunauthor=X.+Zhangauthor=Q.+Zhaoauthor=J.+Zhaoauthor=R.-P.+Xiaoauthor=X.+Zhangauthor=G.+Jiauthor=Y.+Rao&title=A+chemical+approach+for+global+protein+knockdown+from+mice+to+non-human+primates&doi=10.1038%2Fs41421-018-0079-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13bR"><div class="casContent"><span class="casTitleNuber">13b</span><div class="casTitle"><span class="NLM_cas:atitle">A chemical approach for global protein knockdown from mice to non-human primates</span></div><div class="casAuthors">Sun Xiuyun; Yao Xia; Lan Tianlong; Wang Liguo; Sun Yonghui; Zhao Qiuye; Rao Yu; Sun Xiuyun; Yao Xia; Lan Tianlong; Wang Liguo; Sun Yonghui; Zhao Qiuye; Rao Yu; Sun Xiuyun; Wang Jun; Mao Yang; Zhang Xinyi; Ji Guangju; Wang Jun; Mao Yang; Zhang Xinyi; Zheng Wen; Zhang Xiuqin; Zheng Wen; Zhang Xiuqin; Zhao Jianguo; Xiao Rui-Ping; Xiao Rui-Ping</div><div class="citationInfo"><span class="NLM_cas:title">Cell discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">10</span>
        ISSN:<span class="NLM_cas:issn">2056-5968</span>.
    </div><div class="casAbstract">Although conventional genetic modification approaches for protein knockdown work very successfully due to the increasing use of CRISPR/Cas9, effective techniques for achieving protein depletion in adult animals, especially in large animals such as non-human primates, are lacking.  Here, we report a chemical approach based on PROTACs technology that efficiently and quickly knocks down FKBP12 (12-kDa FK506-binding) protein globally in vivo.  Both intraperitoneal and oral administration led to rapid, robust, and reversible FKBP12 degradation in mice.  The efficiency and practicality of this method were successfully demonstrated in both large and small animals (mice, rats, Bama pigs, and rhesus monkeys).  Furthermore, we showed this approach can also be applied to effectively knockdown other target proteins such as Bruton's tyrosine kinase (BTK).  This chemical protein knockdown strategy provides a powerful research tool for gene function studies in animals, particularly in large animals, for which gene-targeted knockout strategies may remain unfeasible.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSOtjU0KY5kR9ts4TFFB11gfW6udTcc2ebNW0fhNDj0BLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cjpvFKiug%253D%253D&md5=199267298a928382ed1a91a7c066eefa</span></div><a href="/servlet/linkout?suffix=cit13b&amp;dbid=16384&amp;doi=10.1038%2Fs41421-018-0079-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41421-018-0079-1%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DYao%26aufirst%3DX.%26aulast%3DZheng%26aufirst%3DW.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DLan%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DZhao%26aufirst%3DJ.%26aulast%3DXiao%26aufirst%3DR.-P.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DJi%26aufirst%3DG.%26aulast%3DRao%26aufirst%3DY.%26atitle%3DA%2520chemical%2520approach%2520for%2520global%2520protein%2520knockdown%2520from%2520mice%2520to%2520non-human%2520primates%26jtitle%3DCell%2520Discovery%26date%3D2019%26volume%3D5%26doi%3D10.1038%2Fs41421-018-0079-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Pike, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harlfinger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGinnity, D. F.</span></span> <span> </span><span class="NLM_article-title">Optimising proteolysis-targeting chimeras (PROTACs) for oral drug delivery: a drug metabolism and pharmacokinetics perspective</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1793</span>– <span class="NLM_lpage">1800</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2020.07.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1016%2Fj.drudis.2020.07.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVeru7zF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2020&pages=1793-1800&author=A.+Pikeauthor=B.+Williamsonauthor=S.+Harlfingerauthor=S.+Martinauthor=D.+F.+McGinnity&title=Optimising+proteolysis-targeting+chimeras+%28PROTACs%29+for+oral+drug+delivery%3A+a+drug+metabolism+and+pharmacokinetics+perspective&doi=10.1016%2Fj.drudis.2020.07.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14aR"><div class="casContent"><span class="casTitleNuber">14a</span><div class="casTitle"><span class="NLM_cas:atitle">Optimising proteolysis-targeting chimeras (PROTACs) for oral drug delivery: a drug metabolism and pharmacokinetics perspective</span></div><div class="casAuthors">Pike, Andy; Williamson, Beth; Harlfinger, Stephanie; Martin, Scott; McGinnity, Dermot F.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1793-1800</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Proteolysis-targeting chimeras (PROTACs) are an emerging therapeutic modality with the potential to open target space not accessible to conventional small mols. via a degrdn.-based mechanism; however, their bifunctional nature can result in physicochem. properties that breach commonly accepted limits for small-mol. oral drugs.  We offer a drug metab. and pharmacokinetics (DMPK) perspective on the optimization of oral PROTACs across a diverse set of projects within Oncol. R&D at AstraZeneca, highlighting some of the challenges that they have presented to our established screening cascade.  Furthermore, we challenge some of the perceptions and dogma surrounding the feasibility of oral PROTACS and demonstrate that acceptable oral PK properties for this modality can be regularly achievable despite the physicochem. property challenges they present.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNes0zq6ODubVg90H21EOLACvtfcHk0lgc-3VqS0Pmhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVeru7zF&md5=a26048e1fe2eb5f9e134a9132593df23</span></div><a href="/servlet/linkout?suffix=cit14a&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2020.07.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2020.07.013%26sid%3Dliteratum%253Aachs%26aulast%3DPike%26aufirst%3DA.%26aulast%3DWilliamson%26aufirst%3DB.%26aulast%3DHarlfinger%26aufirst%3DS.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DMcGinnity%26aufirst%3DD.%2BF.%26atitle%3DOptimising%2520proteolysis-targeting%2520chimeras%2520%2528PROTACs%2529%2520for%2520oral%2520drug%2520delivery%253A%2520a%2520drug%2520metabolism%2520and%2520pharmacokinetics%2520perspective%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2020%26volume%3D25%26spage%3D1793%26epage%3D1800%26doi%3D10.1016%2Fj.drudis.2020.07.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit14b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Cantrill, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaturvedi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rynn, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaffland, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walter, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittwer, M. B.</span></span> <span> </span><span class="NLM_article-title">Fundamental aspects of DMPK optimization of targeted protein degraders</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">969</span>– <span class="NLM_lpage">982</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2020.03.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1016%2Fj.drudis.2020.03.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=32298797" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BB3cXntFGht7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2020&pages=969-982&author=C.+Cantrillauthor=P.+Chaturvediauthor=C.+Rynnauthor=J.+P.+Schafflandauthor=I.+Walterauthor=M.+B.+Wittwer&title=Fundamental+aspects+of+DMPK+optimization+of+targeted+protein+degraders&doi=10.1016%2Fj.drudis.2020.03.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14bR"><div class="casContent"><span class="casTitleNuber">14b</span><div class="casTitle"><span class="NLM_cas:atitle">Fundamental aspects of DMPK optimization of targeted protein degraders</span></div><div class="casAuthors">Cantrill, Carina; Chaturvedi, Prasoon; Rynn, Caroline; Petrig Schaffland, Jeannine; Walter, Isabelle; Wittwer, Matthias B.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">969-982</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Targeted protein degraders are an emerging modality.  Their properties fall outside the traditional small-mol. property space and are in the 'beyond rule of 5' space.  Consequently, drug discovery programs focused on developing orally bioavailable degraders are expected to face complex drug metab. and pharmacokinetics (DMPK) challenges compared with traditional small mols.  Nevertheless, little information is available on the DMPK optimization of oral degraders.  Therefore, in this review, we discuss our experience of these DMPK optimization challenges and present methodologies and strategies to overcome the hurdles dealing with this new small-mol. modality, specifically in developing oral degraders to treat cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoz2WUoP7TUvbVg90H21EOLACvtfcHk0lgc-3VqS0Pmhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXntFGht7s%253D&md5=465ad2dab5183e0f229aeec7e57e396a</span></div><a href="/servlet/linkout?suffix=cit14b&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2020.03.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2020.03.012%26sid%3Dliteratum%253Aachs%26aulast%3DCantrill%26aufirst%3DC.%26aulast%3DChaturvedi%26aufirst%3DP.%26aulast%3DRynn%26aufirst%3DC.%26aulast%3DSchaffland%26aufirst%3DJ.%2BP.%26aulast%3DWalter%26aufirst%3DI.%26aulast%3DWittwer%26aufirst%3DM.%2BB.%26atitle%3DFundamental%2520aspects%2520of%2520DMPK%2520optimization%2520of%2520targeted%2520protein%2520degraders%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2020%26volume%3D25%26spage%3D969%26epage%3D982%26doi%3D10.1016%2Fj.drudis.2020.03.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Beck, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goetsch, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumontet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corvaïa, N.</span></span> <span> </span><span class="NLM_article-title">Strategies and challenges for the next generation of antibody-drug conjugates</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">315</span>– <span class="NLM_lpage">337</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.268</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1038%2Fnrd.2016.268" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=28303026" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkt1Onsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=315-337&author=A.+Beckauthor=L.+Goetschauthor=C.+Dumontetauthor=N.+Corva%C3%AFa&title=Strategies+and+challenges+for+the+next+generation+of+antibody-drug+conjugates&doi=10.1038%2Fnrd.2016.268"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15aR"><div class="casContent"><span class="casTitleNuber">15a</span><div class="casTitle"><span class="NLM_cas:atitle">Strategies and challenges for the next generation of antibody-drug conjugates</span></div><div class="casAuthors">Beck, Alain; Goetsch, Liliane; Dumontet, Charles; Corvaia, Nathalie</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">315-337</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Antibody-drug conjugates (ADCs) are one of the fastest growing classes of oncol. therapeutics.  After half a century of research, the approvals of brentuximab vedotin (in 2011) and trastuzumab emtansine (in 2013) have paved the way for ongoing clin. trials that are evaluating more than 60 further ADC candidates.  The limited success of first-generation ADCs (developed in the early 2000s) informed strategies to bring second-generation ADCs to the market, which have higher levels of cytotoxic drug conjugation, lower levels of naked antibodies and more-stable linkers between the drug and the antibody.  Furthermore, lessons learned during the past decade are now being used in the development of third-generation ADCs.  In this Review, we discuss strategies to select the best target antigens as well as suitable cytotoxic drugs; the design of optimized linkers; the discovery of bioorthogonal conjugation chemistries; and toxicity issues.  The selection and engineering of antibodies for site-specific drug conjugation, which will result in higher homogeneity and increased stability, as well as the quest for new conjugation chemistries and mechanisms of action, are priorities in ADC research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqP7lV63Ws8rbVg90H21EOLACvtfcHk0lgc-3VqS0Pmhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkt1Onsbc%253D&md5=7358b6080dd1aed1d21e98c078cd9ef1</span></div><a href="/servlet/linkout?suffix=cit15a&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.268&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.268%26sid%3Dliteratum%253Aachs%26aulast%3DBeck%26aufirst%3DA.%26aulast%3DGoetsch%26aufirst%3DL.%26aulast%3DDumontet%26aufirst%3DC.%26aulast%3DCorva%25C3%25AFa%26aufirst%3DN.%26atitle%3DStrategies%2520and%2520challenges%2520for%2520the%2520next%2520generation%2520of%2520antibody-drug%2520conjugates%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D315%26epage%3D337%26doi%3D10.1038%2Fnrd.2016.268" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit15b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Sau, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alsaab, H. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kashaw, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tatiparti, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iyer, A. K.</span></span> <span> </span><span class="NLM_article-title">Advances in antibody-drug conjugates: A new era of targeted cancer therapy</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">1547</span>– <span class="NLM_lpage">1556</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2017.05.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1016%2Fj.drudis.2017.05.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=28627385" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVCrsLzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=1547-1556&author=S.+Sauauthor=H.+O.+Alsaabauthor=S.+K.+Kashawauthor=K.+Tatipartiauthor=A.+K.+Iyer&title=Advances+in+antibody-drug+conjugates%3A+A+new+era+of+targeted+cancer+therapy&doi=10.1016%2Fj.drudis.2017.05.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15bR"><div class="casContent"><span class="casTitleNuber">15b</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in antibody-drug conjugates: A new era of targeted cancer therapy</span></div><div class="casAuthors">Sau, Samaresh; Alsaab, Hashem O.; Kashaw, Sushil Kumar; Tatiparti, Katyayani; Iyer, Arun K.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1547-1556</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Antibody-drug conjugates (ADCs), a potent class of anticancer therapeutics, comprise a high-affinity antibody (Ab) and cytotoxic payload coupled via a suitable linker for selective tumor cell killing.  In the initial phase of their development, two ADCs, Mylotarg, and Adcetris were approved by the US Food and Drug Administration (FDA) for treating hematological cancer, but the real breakthrough came with the discovery of the breast cancer-targeting ADC, Kadcyla.  With advances in bioengineering, linker chem., and potent cytotoxic payload, ADC technol. has become a more powerful tool for targeted cancer therapy.  In addn., ADCs with improved safety using humanized Abs with a unified 'drug:antibody ratio' (DAR) have been achieved.  Concomitantly, there has been a significant increase in the no. of clin. trials with anticancer ADCs with high translation potential.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohgxck59tjx7Vg90H21EOLACvtfcHk0liMJE_UuaHbyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVCrsLzK&md5=dc5204e59d81ffa946b5d4ea07caec70</span></div><a href="/servlet/linkout?suffix=cit15b&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2017.05.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2017.05.011%26sid%3Dliteratum%253Aachs%26aulast%3DSau%26aufirst%3DS.%26aulast%3DAlsaab%26aufirst%3DH.%2BO.%26aulast%3DKashaw%26aufirst%3DS.%2BK.%26aulast%3DTatiparti%26aufirst%3DK.%26aulast%3DIyer%26aufirst%3DA.%2BK.%26atitle%3DAdvances%2520in%2520antibody-drug%2520conjugates%253A%2520A%2520new%2520era%2520of%2520targeted%2520cancer%2520therapy%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2017%26volume%3D22%26spage%3D1547%26epage%3D1556%26doi%3D10.1016%2Fj.drudis.2017.05.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit15c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Chari, R. V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Widdison, W. C.</span></span> <span> </span><span class="NLM_article-title">Antibody-drug conjugates: An emerging concept in cancer therapy</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">3796</span>– <span class="NLM_lpage">3827</span>, <span class="refDoi"> DOI: 10.1002/anie.201307628</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1002%2Fanie.201307628" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtVGmsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=3796-3827&author=R.+V.+J.+Chariauthor=M.+L.+Millerauthor=W.+C.+Widdison&title=Antibody-drug+conjugates%3A+An+emerging+concept+in+cancer+therapy&doi=10.1002%2Fanie.201307628"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15cR"><div class="casContent"><span class="casTitleNuber">15c</span><div class="casTitle"><span class="NLM_cas:atitle">Antibody-Drug Conjugates: An Emerging Concept in Cancer Therapy</span></div><div class="casAuthors">Chari, Ravi V. J.; Miller, Michael L.; Widdison, Wayne C.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3796-3827</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Traditional cancer chemotherapy is often accompanied by systemic toxicity to the patient.  Monoclonal antibodies against antigens on cancer cells offer an alternative tumor-selective treatment approach.  However, most monoclonal antibodies are not sufficiently potent to be therapeutically active on their own.  Antibody-drug conjugates (ADCs) use antibodies to deliver a potent cytotoxic compd. selectively to tumor cells, thus improving the therapeutic index of chemotherapeutic agents.  The recent approval of two ADCs, brentuximab vedotin and ado-trastuzumab emtansine, for cancer treatment has spurred tremendous research interest in this field.  This Review touches upon the early efforts in the field, and describes how the lessons learned from the first-generation ADCs have led to improvements in every aspect of this technol., i.e., the antibody, the cytotoxic compd., and the linker connecting them, leading to the current successes.  The design of ADCs currently in clin. development, and results from mechanistic studies and preclin. and clin. evaluation are discussed.  Emerging technologies that seek to further advance this exciting area of research are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4T2rhmji6DLVg90H21EOLACvtfcHk0liMJE_UuaHbyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtVGmsr0%253D&md5=fbfb2f843f516654ea7c0eb98949604c</span></div><a href="/servlet/linkout?suffix=cit15c&amp;dbid=16384&amp;doi=10.1002%2Fanie.201307628&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201307628%26sid%3Dliteratum%253Aachs%26aulast%3DChari%26aufirst%3DR.%2BV.%2BJ.%26aulast%3DMiller%26aufirst%3DM.%2BL.%26aulast%3DWiddison%26aufirst%3DW.%2BC.%26atitle%3DAntibody-drug%2520conjugates%253A%2520An%2520emerging%2520concept%2520in%2520cancer%2520therapy%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2014%26volume%3D53%26spage%3D3796%26epage%3D3827%26doi%3D10.1002%2Fanie.201307628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span>; <span class="NLM_string-name">Pillow, T. H.</span>; <span class="NLM_string-name">Blake, R. A.</span>; <span class="NLM_string-name">Sadowsky, J. D.</span>; <span class="NLM_string-name">Adaligil, E.</span>; <span class="NLM_string-name">Adhikari, P.</span>; <span class="NLM_string-name">Bhakta, S.</span>; <span class="NLM_string-name">Blaquiere, N.</span>; <span class="NLM_string-name">Chen, J.</span>; <span class="NLM_string-name">dela
Cruz-Chuh, J.</span>; <span class="NLM_string-name">Gascoigne, K. E.</span>; <span class="NLM_string-name">Hartman, S. J.</span>; <span class="NLM_string-name">He, M.</span>; <span class="NLM_string-name">Kaufman, S.</span>; <span class="NLM_string-name">Kleinheinz, T.</span>; <span class="NLM_string-name">Kozak, K. R.</span>; <span class="NLM_string-name">Liu, L.</span>; <span class="NLM_string-name">Liu, L.</span>; <span class="NLM_string-name">Liu, Q.</span>; <span class="NLM_string-name">Lu, Y.</span>; <span class="NLM_string-name">Meng, F.</span>; <span class="NLM_string-name">Mulvihill, M. M.</span>; <span class="NLM_string-name">O’Donohue, A.</span>; <span class="NLM_string-name">Rowntree, R. K.</span>; <span class="NLM_string-name">Staben, L. R.</span>; <span class="NLM_string-name">Staben, S. T.</span>; <span class="NLM_string-name">Wai, J.</span>; <span class="NLM_string-name">Wang, J.</span>; <span class="NLM_string-name">Wei, B.</span>; <span class="NLM_string-name">Wilson, C.</span>; <span class="NLM_string-name">Xin, J.</span>; <span class="NLM_string-name">Xu, Z.</span>; <span class="NLM_string-name">Yao, H.</span>; <span class="NLM_string-name">Zhang, D.</span>; <span class="NLM_string-name">Zhang, H.</span>; <span class="NLM_string-name">Zhou, H.</span>; <span class="NLM_string-name">Zhu, X.</span></span> <span> </span><span class="NLM_article-title">Antibody-mediated delivery of chimeric BRD4 degraders. Part 1: Exploration of antibody linker, payload loading, and payload molecular properties</span>.  <i>J. Med. Chem.</i> preceding paper in this issue.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Dragovich%2C+P.+S.%3B+Pillow%2C+T.+H.%3B+Blake%2C+R.+A.%3B+Sadowsky%2C+J.+D.%3B+Adaligil%2C+E.%3B+Adhikari%2C+P.%3B+Bhakta%2C+S.%3B+Blaquiere%2C+N.%3B+Chen%2C+J.%3B+dela%0ACruz-Chuh%2C+J.%3B+Gascoigne%2C+K.+E.%3B+Hartman%2C+S.+J.%3B+He%2C+M.%3B+Kaufman%2C+S.%3B+Kleinheinz%2C+T.%3B+Kozak%2C+K.+R.%3B+Liu%2C+L.%3B+Liu%2C+L.%3B+Liu%2C+Q.%3B+Lu%2C+Y.%3B+Meng%2C+F.%3B+Mulvihill%2C+M.+M.%3B+O%E2%80%99Donohue%2C+A.%3B+Rowntree%2C+R.+K.%3B+Staben%2C+L.+R.%3B+Staben%2C+S.+T.%3B+Wai%2C+J.%3B+Wang%2C+J.%3B+Wei%2C+B.%3B+Wilson%2C+C.%3B+Xin%2C+J.%3B+Xu%2C+Z.%3B+Yao%2C+H.%3B+Zhang%2C+D.%3B+Zhang%2C+H.%3B+Zhou%2C+H.%3B+Zhu%2C+X.+Antibody-mediated+delivery+of+chimeric+BRD4+degraders.+Part+1%3A+Exploration+of+antibody+linker%2C+payload+loading%2C+and+payload+molecular+properties.+J.+Med.+Chem.+preceding+paper+in+this+issue."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26atitle%3DAntibody-mediated%2520delivery%2520of%2520chimeric%2520BRD4%2520degraders.%2520Part%25201%253A%2520Exploration%2520of%2520antibody%2520linker%252C%2520payload%2520loading%252C%2520and%2520payload%2520molecular%2520properties%26jtitle%3DJ.%2520Med.%2520Chem." title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pillow, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adhikari, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blake, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Rosario, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshmukh, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Figueroa, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gascoigne, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamath, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleinheinz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozak, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Latifi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leipold, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulvihill, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Donohue, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowntree, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadowsky, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span></span> <span> </span><span class="NLM_article-title">Antibody conjugation of a chimeric BET degrader enables in vivo activity</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">17</span>– <span class="NLM_lpage">25</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201900497</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1002%2Fcmdc.201900497" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=31674143" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFCmt7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2020&pages=17-25&author=T.+H.+Pillowauthor=P.+Adhikariauthor=R.+A.+Blakeauthor=J.+Chenauthor=G.+Del+Rosarioauthor=G.+Deshmukhauthor=I.+Figueroaauthor=K.+E.+Gascoigneauthor=A.+V.+Kamathauthor=S.+Kaufmanauthor=T.+Kleinheinzauthor=K.+R.+Kozakauthor=B.+Latifiauthor=D.+D.+Leipoldauthor=C.+S.+Liauthor=R.+Liauthor=M.+M.+Mulvihillauthor=A.+O%E2%80%99Donohueauthor=R.+K.+Rowntreeauthor=J.+D.+Sadowskyauthor=J.+Waiauthor=X.+Wangauthor=C.+Wuauthor=Z.+Xuauthor=H.+Yaoauthor=S.-F.+Yuauthor=D.+Zhangauthor=R.+Zangauthor=H.+Zhangauthor=H.+Zhouauthor=X.+Zhuauthor=P.+S.+Dragovich&title=Antibody+conjugation+of+a+chimeric+BET+degrader+enables+in+vivo+activity&doi=10.1002%2Fcmdc.201900497"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Antibody Conjugation of a Chimeric BET Degrader Enables in vivo Activity</span></div><div class="casAuthors">Pillow, Thomas H.; Adhikari, Pragya; Blake, Robert A.; Chen, Jinhua; Del Rosario, Geoffrey; Deshmukh, Gauri; Figueroa, Isabel; Gascoigne, Karen E.; Kamath, Amrita V.; Kaufman, Susan; Kleinheinz, Tracy; Kozak, Katherine R.; Latifi, Brandon; Leipold, Douglas D.; Sing Li, Chun; Li, Ruina; Mulvihill, Melinda M.; O'Donohue, Aimee; Rowntree, Rebecca K.; Sadowsky, Jack D.; Wai, John; Wang, Xinxin; Wu, Cong; Xu, Zijin; Yao, Hui; Yu, Shang-Fan; Zhang, Donglu; Zang, Richard; Zhang, Hongyan; Zhou, Hao; Zhu, Xiaoyu; Dragovich, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">17-25</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The ability to selectively degrade proteins with bifunctional small mols. has the potential to fundamentally alter therapy in a variety of diseases.  However, the relatively large size of these chimeric mols. often results in challenging physico-chem. properties (e. g., low aq. soly.) and poor pharmacokinetics which may complicate their in vivo applications.  We recently discovered an exquisitely potent chimeric BET degrader (GNE-987) which exhibited picomolar cell potencies but also demonstrated low in vivo exposures.  In an effort to improve the pharmacokinetic properties of this mol., we discovered the first degrader-antibody conjugate by attaching GNE-987 to an anti-CLL1 antibody via a novel linker.  A single IV dose of the conjugate afforded sustained in vivo exposures that resulted in antigen-specific tumor regressions.  Enhancement of a chimeric protein degrader with poor in vivo properties through antibody conjugation thereby expands the utility of directed protein degrdn. as both a biol. tool and a therapeutic possibility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPzNLh7QlBR7Vg90H21EOLACvtfcHk0liMJE_UuaHbyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFCmt7zP&md5=90f9e1b5ae453ae38d61a36587feb558</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201900497&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201900497%26sid%3Dliteratum%253Aachs%26aulast%3DPillow%26aufirst%3DT.%2BH.%26aulast%3DAdhikari%26aufirst%3DP.%26aulast%3DBlake%26aufirst%3DR.%2BA.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DDel%2BRosario%26aufirst%3DG.%26aulast%3DDeshmukh%26aufirst%3DG.%26aulast%3DFigueroa%26aufirst%3DI.%26aulast%3DGascoigne%26aufirst%3DK.%2BE.%26aulast%3DKamath%26aufirst%3DA.%2BV.%26aulast%3DKaufman%26aufirst%3DS.%26aulast%3DKleinheinz%26aufirst%3DT.%26aulast%3DKozak%26aufirst%3DK.%2BR.%26aulast%3DLatifi%26aufirst%3DB.%26aulast%3DLeipold%26aufirst%3DD.%2BD.%26aulast%3DLi%26aufirst%3DC.%2BS.%26aulast%3DLi%26aufirst%3DR.%26aulast%3DMulvihill%26aufirst%3DM.%2BM.%26aulast%3DO%25E2%2580%2599Donohue%26aufirst%3DA.%26aulast%3DRowntree%26aufirst%3DR.%2BK.%26aulast%3DSadowsky%26aufirst%3DJ.%2BD.%26aulast%3DWai%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DC.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DYao%26aufirst%3DH.%26aulast%3DYu%26aufirst%3DS.-F.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DZang%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26atitle%3DAntibody%2520conjugation%2520of%2520a%2520chimeric%2520BET%2520degrader%2520enables%2520in%2520vivo%2520activity%26jtitle%3DChemMedChem%26date%3D2020%26volume%3D15%26spage%3D17%26epage%3D25%26doi%3D10.1002%2Fcmdc.201900497" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adhikari, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blake, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blaquiere, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">den Besten, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartman, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingalla, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamath, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleinheinz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leipold, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pillow, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowntree, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadowsky, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sampath, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staben, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staben, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wertz, I. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span> <span> </span><span class="NLM_article-title">Antibody-mediated delivery of chimeric protein degraders which target estrogen receptor alpha (ERα)</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>30</i></span>, <span class="NLM_elocation-id">126907</span> <span class="refDoi"> DOI: 10.1016/j.bmcl.2019.126907</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1016%2Fj.bmcl.2019.126907" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=31902710" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BB3cXksFWgtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2020&author=P.+S.+Dragovichauthor=P.+Adhikariauthor=R.+A.+Blakeauthor=N.+Blaquiereauthor=J.+Chenauthor=Y.-X.+Chengauthor=W.+den+Bestenauthor=J.+Hanauthor=S.+J.+Hartmanauthor=J.+Heauthor=M.+Heauthor=E.+R.+Ingallaauthor=A.+V.+Kamathauthor=T.+Kleinheinzauthor=T.+Laiauthor=D.+D.+Leipoldauthor=C.+S.+Liauthor=Q.+Liuauthor=J.+Luauthor=Y.+Luauthor=F.+Mengauthor=L.+Mengauthor=C.+Ngauthor=K.+Pengauthor=G.+L.+Phillipsauthor=T.+H.+Pillowauthor=R.+K.+Rowntreeauthor=J.+D.+Sadowskyauthor=D.+Sampathauthor=L.+Stabenauthor=S.+T.+Stabenauthor=J.+Waiauthor=K.+Wanauthor=X.+Wangauthor=B.+Weiauthor=I.+E.+Wertzauthor=J.+Xinauthor=K.+Xuauthor=H.+Yaoauthor=R.+Zangauthor=D.+Zhangauthor=H.+Zhouauthor=Y.+Zhao&title=Antibody-mediated+delivery+of+chimeric+protein+degraders+which+target+estrogen+receptor+alpha+%28ER%CE%B1%29&doi=10.1016%2Fj.bmcl.2019.126907"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Antibody-mediated delivery of chimeric protein degraders which target estrogen receptor alpha (ERα)</span></div><div class="casAuthors">Dragovich, Peter S.; Adhikari, Pragya; Blake, Robert A.; Blaquiere, Nicole; Chen, Jinhua; Cheng, Yun-Xing; den Besten, Willem; Han, Jinping; Hartman, Steven J.; He, Jintang; He, Mingtao; Rei Ingalla, Ellen; Kamath, Amrita V.; Kleinheinz, Tracy; Lai, Tommy; Leipold, Douglas D.; Li, Chun Sing; Liu, Qi; Lu, Jiawei; Lu, Ying; Meng, Fanwei; Meng, Lingyao; Ng, Carl; Peng, Kaishan; Lewis Phillips, Gail; Pillow, Thomas H.; Rowntree, Rebecca K.; Sadowsky, Jack D.; Sampath, Deepak; Staben, Leanna; Staben, Steven T.; Wai, John; Wan, Kunpeng; Wang, Xinxin; Wei, Bin Qing; Wertz, Ingrid E.; Xin, Jianfeng; Xu, Keyang; Yao, Hui; Zang, Richard; Zhang, Donglu; Zhou, Hao; Zhao, Yongxin</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">126907</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Chimeric mols. which effect intracellular degrdn. of target proteins via E3 ligase-mediated ubiquitination (e.g., PROTACs) are currently of high interest in medicinal chem.  However, these entities are relatively large compds. that often possess mol. characteristics which may compromise oral bioavailability, soly., and/or in vivo pharmacokinetic properties.  Accordingly, we explored whether conjugation of chimeric degraders to monoclonal antibodies using technologies originally developed for cytotoxic payloads might provide alternate delivery options for these novel agents.  In this report we describe the construction of several degrader-antibody conjugates comprised of two distinct ERα-targeting degrader entities and three independent ADC linker modalities.  We subsequently demonstrate the antigen-dependent delivery to MCF7-neo/HER2 cells of the degrader payloads that are incorporated into these conjugates.  We also provide evidence for efficient intracellular degrader release from one of the employed linkers.  In addn., preliminary data are described which suggest that reasonably favorable in vivo stability properties are assocd. with the linkers utilized to construct the degrader conjugates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdPWcGkXC85LVg90H21EOLACvtfcHk0lgfnpdB0PpjuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXksFWgtw%253D%253D&md5=5f196b194df7f4622825c4b901130f7c</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2019.126907&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2019.126907%26sid%3Dliteratum%253Aachs%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26aulast%3DAdhikari%26aufirst%3DP.%26aulast%3DBlake%26aufirst%3DR.%2BA.%26aulast%3DBlaquiere%26aufirst%3DN.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DCheng%26aufirst%3DY.-X.%26aulast%3Dden%2BBesten%26aufirst%3DW.%26aulast%3DHan%26aufirst%3DJ.%26aulast%3DHartman%26aufirst%3DS.%2BJ.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DIngalla%26aufirst%3DE.%2BR.%26aulast%3DKamath%26aufirst%3DA.%2BV.%26aulast%3DKleinheinz%26aufirst%3DT.%26aulast%3DLai%26aufirst%3DT.%26aulast%3DLeipold%26aufirst%3DD.%2BD.%26aulast%3DLi%26aufirst%3DC.%2BS.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DMeng%26aufirst%3DF.%26aulast%3DMeng%26aufirst%3DL.%26aulast%3DNg%26aufirst%3DC.%26aulast%3DPeng%26aufirst%3DK.%26aulast%3DPhillips%26aufirst%3DG.%2BL.%26aulast%3DPillow%26aufirst%3DT.%2BH.%26aulast%3DRowntree%26aufirst%3DR.%2BK.%26aulast%3DSadowsky%26aufirst%3DJ.%2BD.%26aulast%3DSampath%26aufirst%3DD.%26aulast%3DStaben%26aufirst%3DL.%26aulast%3DStaben%26aufirst%3DS.%2BT.%26aulast%3DWai%26aufirst%3DJ.%26aulast%3DWan%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DWei%26aufirst%3DB.%26aulast%3DWertz%26aufirst%3DI.%2BE.%26aulast%3DXin%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DK.%26aulast%3DYao%26aufirst%3DH.%26aulast%3DZang%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DY.%26atitle%3DAntibody-mediated%2520delivery%2520of%2520chimeric%2520protein%2520degraders%2520which%2520target%2520estrogen%2520receptor%2520alpha%2520%2528ER%25CE%25B1%2529%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D30%26doi%3D10.1016%2Fj.bmcl.2019.126907" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maneiro, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forte, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shchepinova, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kounde, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chudasama, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tate, E. W.</span></span> <span> </span><span class="NLM_article-title">Antibody-PROTAC conjugates enable HER2-dependent targeted protein degradation of BRD4</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1306</span>– <span class="NLM_lpage">1312</span>, <span class="refDoi"> DOI: 10.1021/acschembio.0c00285</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.0c00285" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BB3cXotFehsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2020&pages=1306-1312&author=M.+Maneiroauthor=N.+Forteauthor=M.+M.+Shchepinovaauthor=C.+S.+Koundeauthor=V.+Chudasamaauthor=J.+R.+Bakerauthor=E.+W.+Tate&title=Antibody-PROTAC+conjugates+enable+HER2-dependent+targeted+protein+degradation+of+BRD4&doi=10.1021%2Facschembio.0c00285"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Antibody-PROTAC Conjugates Enable HER2-Dependent Targeted Protein Degradation of BRD4</span></div><div class="casAuthors">Maneiro, Maria; Forte, Nafsika; Shchepinova, Maria M.; Kounde, Cyrille S.; Chudasama, Vijay; Baker, James Richard; Tate, Edward W.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1306-1312</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Targeting protein degrdn. with Proteolysis-Targeting Chimeras (PROTACs) is an area of great current interest in drug discovery.  Nevertheless, although the high effectiveness of PROTACs against a wide variety of targets has been established, most degraders reported to date display limited intrinsic tissue selectivity and do not discriminate between cells of different types.  Here, we describe a strategy for selective protein degrdn. in a specific cell type.  We report the design and synthesis of a trastuzumab-PROTAC conjugate (Ab-PROTAC 3) in which E3 ligase-directed degrader activity is caged with an antibody linker which can be hydrolyzed following antibody-PROTAC internalization, releasing the active PROTAC and inducing catalytic protein degrdn.  We show that 3 selectively targets bromodomain-contg. protein 4 (BRD4) for degrdn. only in HER2 pos. breast cancer cell lines, while sparing HER2 neg. cells.  Using live cell confocal microscopy, we show internalization and lysosomal trafficking of the conjugate specifically in HER2 pos. cells, leading to the release of active PROTAC in quantities sufficient to induce potent BRD4 degrdn.  These studies demonstrate proof-of-concept for tissue-specific BRD4 degrdn., overcoming limitations of PROTAC selectivity, with significant potential for application to novel targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPwGNLaGmRbbVg90H21EOLACvtfcHk0lgfnpdB0PpjuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXotFehsLo%253D&md5=544b53e54d3f0696f0b425c3f6e48e46</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facschembio.0c00285&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.0c00285%26sid%3Dliteratum%253Aachs%26aulast%3DManeiro%26aufirst%3DM.%26aulast%3DForte%26aufirst%3DN.%26aulast%3DShchepinova%26aufirst%3DM.%2BM.%26aulast%3DKounde%26aufirst%3DC.%2BS.%26aulast%3DChudasama%26aufirst%3DV.%26aulast%3DBaker%26aufirst%3DJ.%2BR.%26aulast%3DTate%26aufirst%3DE.%2BW.%26atitle%3DAntibody-PROTAC%2520conjugates%2520enable%2520HER2-dependent%2520targeted%2520protein%2520degradation%2520of%2520BRD4%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2020%26volume%3D15%26spage%3D1306%26epage%3D1312%26doi%3D10.1021%2Facschembio.0c00285" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morse, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickman, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christie, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Thangue, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiest, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of BET bromodomains</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1067</span>– <span class="NLM_lpage">1073</span>, <span class="refDoi"> DOI: 10.1038/nature09504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1038%2Fnature09504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=20871596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1067-1073&author=P.+Filippakopoulosauthor=J.+Qiauthor=S.+Picaudauthor=Y.+Shenauthor=W.+B.+Smithauthor=O.+Fedorovauthor=E.+M.+Morseauthor=T.+Keatesauthor=T.+T.+Hickmanauthor=I.+Felletarauthor=M.+Philpottauthor=S.+Munroauthor=M.+R.+McKeownauthor=Y.+Wangauthor=A.+L.+Christieauthor=N.+Westauthor=M.+J.+Cameronauthor=B.+Schwartzauthor=T.+D.+Heightmanauthor=N.+La+Thangueauthor=C.+A.+Frenchauthor=O.+Wiestauthor=A.+L.+Kungauthor=S.+Knappauthor=J.+E.+Bradner&title=Selective+inhibition+of+BET+bromodomains&doi=10.1038%2Fnature09504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of BET bromodomains</span></div><div class="casAuthors">Filippakopoulos, Panagis; Qi, Jun; Picaud, Sarah; Shen, Yao; Smith, William B.; Fedorov, Oleg; Morse, Elizabeth M.; Keates, Tracey; Hickman, Tyler T.; Felletar, Ildiko; Philpott, Martin; Munro, Shongah; McKeown, Michael R.; Wang, Yuchuan; Christie, Amanda L.; West, Nathan; Cameron, Michael J.; Schwartz, Brian; Heightman, Tom D.; La Thangue, Nicholas; French, Christopher; Wiest, Olaf; Kung, Andrew L.; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic proteins are intently pursued targets in ligand discovery.  So far, successful efforts have been limited to chromatin modifying enzymes, or so-called epigenetic 'writers' and 'erasers'.  Potent inhibitors of histone binding modules have not yet been described.  Here the authors report a cell-permeable small mol. (I,JQ1) that binds competitively to acetyl-lysine recognition motifs, or bromodomains.  High potency and specificity towards a subset of human bromodomains is explained by co-crystal structures with bromodomain and extra-terminal (BET) family member BRD4, revealing excellent shape complementarity with the acetyl-lysine binding cavity.  Recurrent translocation of BRD4 is obsd. in a genetically-defined, incurable subtype of human squamous carcinoma.  Competitive binding by JQ1 displaces the BRD4 fusion oncoprotein from chromatin, prompting squamous differentiation and specific antiproliferative effects in BRD4-dependent cell lines and patient-derived xenograft models.  These data establish proof-of-concept for targeting protein-protein interactions of epigenetic 'readers', and provide a versatile chem. scaffold for the development of chem. probes more broadly throughout the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ZbtmZM8m_LVg90H21EOLACvtfcHk0lgGq1WH7XLa1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D&md5=bd9cadac86124d2c6c3cd5b2a92d68c2</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnature09504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09504%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DW.%2BB.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DHickman%26aufirst%3DT.%2BT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChristie%26aufirst%3DA.%2BL.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DSchwartz%26aufirst%3DB.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DLa%2BThangue%26aufirst%3DN.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSelective%2520inhibition%2520of%2520BET%2520bromodomains%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1067%26epage%3D1073%26doi%3D10.1038%2Fnature09504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantei, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holms, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguirre, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogdan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dietrich, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marjanovic, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutchins, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bui, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magoc, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajaraman, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, K. F.</span></span> <span> </span><span class="NLM_article-title">Discovery and optimization of novel constrained pyrrolopyridone BET family inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">1804</span>– <span class="NLM_lpage">1810</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2018.04.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1016%2Fj.bmcl.2018.04.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=29678460" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1cXotVamtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=1804-1810&author=S.+D.+Fidanzeauthor=D.+Liuauthor=R.+A.+Manteiauthor=L.+A.+Hasvoldauthor=J.+K.+Prattauthor=G.+S.+Sheppardauthor=L.+Wangauthor=J.+H.+Holmsauthor=Y.+Daiauthor=A.+Aguirreauthor=A.+Bogdanauthor=J.+D.+Dietrichauthor=J.+Marjanovicauthor=C.+H.+Parkauthor=C.+W.+Hutchinsauthor=X.+Linauthor=M.+H.+Buiauthor=X.+Huangauthor=D.+Wilcoxauthor=L.+Liauthor=R.+Wangauthor=P.+Kovarauthor=T.+J.+Magocauthor=G.+Rajaramanauthor=D.+H.+Albertauthor=Y.+Shenauthor=W.+M.+Katiauthor=K.+F.+McDaniel&title=Discovery+and+optimization+of+novel+constrained+pyrrolopyridone+BET+family+inhibitors&doi=10.1016%2Fj.bmcl.2018.04.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21aR"><div class="casContent"><span class="casTitleNuber">21a</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and optimization of novel constrained pyrrolopyridone BET family inhibitors</span></div><div class="casAuthors">Fidanze, Steven D.; Liu, Dachun; Mantei, Robert A.; Hasvold, Lisa A.; Pratt, John K.; Sheppard, George S.; Wang, Le; Holms, James H.; Dai, Yujia; Aguirre, Ana; Bogdan, Andrew; Dietrich, Justin D.; Marjanovic, Jasmina; Park, Chang H.; Hutchins, Charles W.; Lin, Xiaoyu; Bui, Mai H.; Huang, Xiaoli; Wilcox, Denise; Li, Leiming; Wang, Rongqi; Kovar, Peter; Magoc, Terrance J.; Rajaraman, Ganesh; Albert, Daniel H.; Shen, Yu; Kati, Warren M.; McDaniel, Keith F.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1804-1810</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Novel conformationally constrained BET bromodomain inhibitors have been developed.  These inhibitors were optimized in two similar, yet distinct chem. series, the 6-methyl-1H-pyrrolo[2,3-c]pyridin-7(6H)-ones (A) and the 1-methyl-1H-pyrrolo[2,3-c]pyridin-7(6H)-ones (B).  Each series demonstrated excellent activity in binding and cellular assays, and lead compds. from each series demonstrated significant efficacy in in vivo tumor xenograft models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbjffbFRUNM7Vg90H21EOLACvtfcHk0lgGq1WH7XLa1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXotVamtbg%253D&md5=2ac2fba2c203dabfc34216f6b0717d1d</span></div><a href="/servlet/linkout?suffix=cit21a&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2018.04.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2018.04.020%26sid%3Dliteratum%253Aachs%26aulast%3DFidanze%26aufirst%3DS.%2BD.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DMantei%26aufirst%3DR.%2BA.%26aulast%3DHasvold%26aufirst%3DL.%2BA.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DHolms%26aufirst%3DJ.%2BH.%26aulast%3DDai%26aufirst%3DY.%26aulast%3DAguirre%26aufirst%3DA.%26aulast%3DBogdan%26aufirst%3DA.%26aulast%3DDietrich%26aufirst%3DJ.%2BD.%26aulast%3DMarjanovic%26aufirst%3DJ.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DHutchins%26aufirst%3DC.%2BW.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DBui%26aufirst%3DM.%2BH.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DWilcox%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DKovar%26aufirst%3DP.%26aulast%3DMagoc%26aufirst%3DT.%2BJ.%26aulast%3DRajaraman%26aufirst%3DG.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DKati%26aufirst%3DW.%2BM.%26aulast%3DMcDaniel%26aufirst%3DK.%2BF.%26atitle%3DDiscovery%2520and%2520optimization%2520of%2520novel%2520constrained%2520pyrrolopyridone%2520BET%2520family%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D28%26spage%3D1804%26epage%3D1810%26doi%3D10.1016%2Fj.bmcl.2018.04.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit21b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S. D.</span>; <span class="NLM_string-name">Liu, D.</span>; <span class="NLM_string-name">Mantei, R. A.</span>; <span class="NLM_string-name">Mcdaniel, K. F.</span>; <span class="NLM_string-name">Pratt, J.</span>; <span class="NLM_string-name">Sheppard, G. S.</span>; <span class="NLM_string-name">Wang, L.</span>; <span class="NLM_string-name">Bogdan, A.</span>; <span class="NLM_string-name">Holms, J. H.</span>; <span class="NLM_string-name">Dietrich, J. D.</span>; <span class="NLM_string-name">Marjanovic, J.</span>; <span class="NLM_string-name">Hasvold, L. A.</span>; <span class="NLM_string-name">Dai, Y.</span></span> <span> </span><span class="NLM_article-title">Tetracyclic Bromodomain Inhibitors</span>. International patent application, <span class="NLM_patent">WO2014139324</span>, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=S.+D.+Fidanze&author=D.+Liu&author=R.+A.+Mantei&author=K.+F.+Mcdaniel&author=J.+Pratt&author=G.+S.+Sheppard&author=L.+Wang&author=A.+Bogdan&author=J.+H.+Holms&author=J.+D.+Dietrich&author=J.+Marjanovic&author=L.+A.+Hasvold&author=Y.+Dai&title=Tetracyclic+Bromodomain+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DFidanze%26aufirst%3DS.%2BD.%26atitle%3DTetracyclic%2520Bromodomain%2520Inhibitors%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artursson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hann, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leavens, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mateus, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Readshaw, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valko, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wayne, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, A.</span></span> <span> </span><span class="NLM_article-title">Direct measurement of intracellular compound concentration by RapidFire mass spectrometry offers insights into cell permeability</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">156</span>– <span class="NLM_lpage">164</span>, <span class="refDoi"> DOI: 10.1177/1087057115604141</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1177%2F1087057115604141" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=26336900" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC28XmvVSrtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=156-164&author=L.+J.+Gordonauthor=M.+Allenauthor=P.+Arturssonauthor=M.+M.+Hannauthor=B.+J.+Leavensauthor=A.+Mateusauthor=S.+Readshawauthor=K.+Valkoauthor=G.+J.+Wayneauthor=A.+West&title=Direct+measurement+of+intracellular+compound+concentration+by+RapidFire+mass+spectrometry+offers+insights+into+cell+permeability&doi=10.1177%2F1087057115604141"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22aR"><div class="casContent"><span class="casTitleNuber">22a</span><div class="casTitle"><span class="NLM_cas:atitle">Direct measurement of intracellular compound concentration by RapidFire mass spectrometry offers insights into cell permeability</span></div><div class="casAuthors">Gordon, Laurie J.; Allen, Morven; Artursson, Per; Hann, Michael M.; Leavens, Bill J.; Mateus, Andre; Readshaw, Simon; Valko, Klara; Wayne, Gareth J.; West, Andy</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">156-164</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">One of the key challenges facing early stage drug discovery is understanding the commonly obsd. difference between the activity of compds. in biochem. assays and cellular assays.  Traditionally, indirect or estd. cell permeability measurements such as estns. from logP or artificial membrane permeability are used to explain the differences.  The missing link is a direct measurement of intracellular compd. concn. in whole cells.  This can, in some circumstances, be estd. from the cellular activity, but this may also be problematic if cellular activity is weak or absent.  Advances in sensitivity and throughput of anal. techniques have enabled us to develop a high-throughput assay for the measurement of intracellular compd. concn. for routine use to support lead optimization.  The assay uses a RapidFire-MS based readout of compd. concn. in HeLa cells following incubation of cells with test compd.  The initial assay validation was performed by ultra-high performance liq. chromatog. tandem mass spectrometry, and the assay was subsequently transferred to RapidFire tandem mass spectrometry.  Further miniaturization and optimization were performed to streamline the process, increase sample throughput, and reduce cycle time.  This optimization has delivered a semi-automated platform with the potential of prodn. scale compd. profiling up to 100 compds. per day.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLd62-6firibVg90H21EOLACvtfcHk0lid4GH_1rjveQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmvVSrtLc%253D&md5=34a5d2a18c1a36973425d00b87112f20</span></div><a href="/servlet/linkout?suffix=cit22a&amp;dbid=16384&amp;doi=10.1177%2F1087057115604141&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057115604141%26sid%3Dliteratum%253Aachs%26aulast%3DGordon%26aufirst%3DL.%2BJ.%26aulast%3DAllen%26aufirst%3DM.%26aulast%3DArtursson%26aufirst%3DP.%26aulast%3DHann%26aufirst%3DM.%2BM.%26aulast%3DLeavens%26aufirst%3DB.%2BJ.%26aulast%3DMateus%26aufirst%3DA.%26aulast%3DReadshaw%26aufirst%3DS.%26aulast%3DValko%26aufirst%3DK.%26aulast%3DWayne%26aufirst%3DG.%2BJ.%26aulast%3DWest%26aufirst%3DA.%26atitle%3DDirect%2520measurement%2520of%2520intracellular%2520compound%2520concentration%2520by%2520RapidFire%2520mass%2520spectrometry%2520offers%2520insights%2520into%2520cell%2520permeability%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2016%26volume%3D21%26spage%3D156%26epage%3D164%26doi%3D10.1177%2F1087057115604141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit22b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Beigi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gottschalk, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hägglund, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haneskog, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brekkan, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Österberg, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundahl, P.</span></span> <span> </span><span class="NLM_article-title">Immobilized liposome and biomembrane partitioning chromatography of drugs for prediction of drug transport</span>. <i>Int. J. Pharm.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>164</i></span>,  <span class="NLM_fpage">129</span>– <span class="NLM_lpage">137</span>, <span class="refDoi"> DOI: 10.1016/S0378-5173(97)00398-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1016%2FS0378-5173%2897%2900398-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADyaK1cXitlSrt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=164&publication_year=1998&pages=129-137&author=F.+Beigiauthor=I.+Gottschalkauthor=C.+L.+H%C3%A4gglundauthor=L.+Haneskogauthor=E.+Brekkanauthor=Y.+Zhangauthor=T.+%C3%96sterbergauthor=P.+Lundahl&title=Immobilized+liposome+and+biomembrane+partitioning+chromatography+of+drugs+for+prediction+of+drug+transport&doi=10.1016%2FS0378-5173%2897%2900398-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22bR"><div class="casContent"><span class="casTitleNuber">22b</span><div class="casTitle"><span class="NLM_cas:atitle">Immobilized liposome and biomembrane partitioning chromatography of drugs for prediction of drug transport</span></div><div class="casAuthors">Beigi, Farideh; Gottschalk, Ingo; Lagerquist Hagglund, Christine; Haneskog, Lars; Brekkan, Eggert; Zhang, Yanxiao; Osterberg, Thomas; Lundahl, Per</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Pharmaceutics</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">164</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">129-137</span>CODEN:
                <span class="NLM_cas:coden">IJPHDE</span>;
        ISSN:<span class="NLM_cas:issn">0378-5173</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">Drug partitioning into lipid bilayers was studied by chromatog. on liposomes and biomembranes immobilized in gel beads by freeze-thawing.  The drug retention vol. was expressed as a capacity factor, Ks, normalized with respect to the amt. of immobilized phospholipid.  Log Ks values for pos. charged drugs on brain phosphatidylserine (PS)/egg phosphatidylcholine (PC) liposomes decreased as the ionic strength was increased, increased as the PS:PC ratio or the pH was increased and varied linearly with the temp.  Log Ks values for beta-blockers, phenothiazines and benzodiazepines on egg phospholipid (EPL) liposomes correlated well with corresponding values on red cell membrane lipid liposomes (r2=0.96), and on human red cell membrane vesicles contg. transmembrane proteins (r2=0.96).  A fair correlation was obsd. between the values on EPL liposomes and those on native membranes of adsorbed red cells (r2=0.86).  Compared to the data obtained with liposomes, the retentions of hydrophilic drugs became larger and the range of log Ks values more narrow on the vesicles and the membranes, which expose hydrophilic protein surfaces and oligosaccharides.  Lower correlations were obsd. between drug retention on EPL liposomes and egg PC liposomes; and between retention on liposomes (or vesicles) and immobilized artificial membrane (IAM) monolayers of PC analogs.  Absorption of orally administered drugs in humans (literature data) was nearly complete for drugs of log Ks values in the interval 1.2-2.5 on vesicles.  Both vesicles and liposomes can thus be used for chromatog. anal. of drug-membrane interaction and prediction of drug absorption.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_r5N-IU8WaLVg90H21EOLACvtfcHk0lid4GH_1rjveQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXitlSrt7g%253D&md5=f930fe3b4f61a23889e2dc42f510b8bc</span></div><a href="/servlet/linkout?suffix=cit22b&amp;dbid=16384&amp;doi=10.1016%2FS0378-5173%2897%2900398-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0378-5173%252897%252900398-0%26sid%3Dliteratum%253Aachs%26aulast%3DBeigi%26aufirst%3DF.%26aulast%3DGottschalk%26aufirst%3DI.%26aulast%3DH%25C3%25A4gglund%26aufirst%3DC.%2BL.%26aulast%3DHaneskog%26aufirst%3DL.%26aulast%3DBrekkan%26aufirst%3DE.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3D%25C3%2596sterberg%26aufirst%3DT.%26aulast%3DLundahl%26aufirst%3DP.%26atitle%3DImmobilized%2520liposome%2520and%2520biomembrane%2520partitioning%2520chromatography%2520of%2520drugs%2520for%2520prediction%2520of%2520drug%2520transport%26jtitle%3DInt.%2520J.%2520Pharm.%26date%3D1998%26volume%3D164%26spage%3D129%26epage%3D137%26doi%3D10.1016%2FS0378-5173%2897%2900398-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Pillow, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadowsky, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Rosario, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhakta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohri, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozak, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ha, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Junutula, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flygare, J. A.</span></span> <span> </span><span class="NLM_article-title">Decoupling stability and release in disulfide bonds with antibody-small molecule conjugates</span>. <i>Chem Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">366</span>– <span class="NLM_lpage">370</span>, <span class="refDoi"> DOI: 10.1039/C6SC01831A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1039%2FC6SC01831A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=28451181" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlOqsL%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=366-370&author=T.+H.+Pillowauthor=J.+D.+Sadowskyauthor=D.+Zhangauthor=S.-F.+Yuauthor=G.+Del+Rosarioauthor=K.+Xuauthor=J.+Heauthor=S.+Bhaktaauthor=R.+Ohriauthor=K.+R.+Kozakauthor=E.+Haauthor=J.+R.+Junutulaauthor=J.+A.+Flygare&title=Decoupling+stability+and+release+in+disulfide+bonds+with+antibody-small+molecule+conjugates&doi=10.1039%2FC6SC01831A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23aR"><div class="casContent"><span class="casTitleNuber">23a</span><div class="casTitle"><span class="NLM_cas:atitle">Decoupling stability and release in disulfide bonds with antibody-small molecule conjugates</span></div><div class="casAuthors">Pillow, Thomas H.; Sadowsky, Jack D.; Zhang, Donglu; Yu, Shang-Fan; Del Rosario, Geoffrey; Xu, Keyang; He, Jintang; Bhakta, Sunil; Ohri, Rachana; Kozak, Katherine R.; Ha, Edward; Junutula, Jagath R.; Flygare, John A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">366-370</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Disulfide bonds provide a bioactivatable connection with applications in imaging and therapy.  The circulation stability and intracellular release of disulfides are problematically coupled in that increasing stability causes a corresponding decrease in cleavage and payload release.  However, an antibody offers the potential for a reversible stabilization.  We examd. this by attaching a small mol. directly to engineered cysteines in an antibody.  At certain sites this unhindered disulfide was stable in circulation yet cellular internalization and antibody catabolism generated a disulfide catabolite that was rapidly reduced.  We demonstrated that this stable connection and facile release is applicable to a variety of payloads.  The ability to reversibly stabilize a labile functional group with an antibody may offer a way to improve targeted probes and therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocdfG3fjz5LLVg90H21EOLACvtfcHk0lid4GH_1rjveQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlOqsL%252FP&md5=1ef5dd94930854c9fba36a9ddea49f3a</span></div><a href="/servlet/linkout?suffix=cit23a&amp;dbid=16384&amp;doi=10.1039%2FC6SC01831A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6SC01831A%26sid%3Dliteratum%253Aachs%26aulast%3DPillow%26aufirst%3DT.%2BH.%26aulast%3DSadowsky%26aufirst%3DJ.%2BD.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DYu%26aufirst%3DS.-F.%26aulast%3DDel%2BRosario%26aufirst%3DG.%26aulast%3DXu%26aufirst%3DK.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DBhakta%26aufirst%3DS.%26aulast%3DOhri%26aufirst%3DR.%26aulast%3DKozak%26aufirst%3DK.%2BR.%26aulast%3DHa%26aufirst%3DE.%26aulast%3DJunutula%26aufirst%3DJ.%2BR.%26aulast%3DFlygare%26aufirst%3DJ.%2BA.%26atitle%3DDecoupling%2520stability%2520and%2520release%2520in%2520disulfide%2520bonds%2520with%2520antibody-small%2520molecule%2520conjugates%26jtitle%3DChem%2520Sci.%26date%3D2017%26volume%3D8%26spage%3D366%26epage%3D370%26doi%3D10.1039%2FC6SC01831A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit23b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Pillow, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schutten, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohri, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadowsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Achilles-Poon, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solis, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Rosario, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Go, M. A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leipold, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamath, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masterson, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gregson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunzner-Toste, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozak, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spencer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polakis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polson, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flygare, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Junutula, J. R.</span></span> <span> </span><span class="NLM_article-title">Modulating therapeutic activity and toxicity of pyrrolobenzodiazepine antibody-Drug conjugates with self-immolative disulfide linkers</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">871</span>– <span class="NLM_lpage">878</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-16-0641</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1158%2F1535-7163.MCT-16-0641" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=28223423" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmvFemtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=871-878&author=T.+H.+Pillowauthor=M.+Schuttenauthor=S.-F.+Yuauthor=R.+Ohriauthor=J.+Sadowskyauthor=K.+Achilles-Poonauthor=W.+Solisauthor=F.+Zhongauthor=G.+Del+Rosarioauthor=M.+A.+T.+Goauthor=J.+Lauauthor=S.+Yeeauthor=J.+Heauthor=L.+Liuauthor=C.+Ngauthor=K.+Xuauthor=D.+D.+Leipoldauthor=A.+V.+Kamathauthor=D.+Zhangauthor=L.+Mastersonauthor=S.+J.+Gregsonauthor=P.+W.+Howardauthor=F.+Fangauthor=J.+Chenauthor=J.+Gunzner-Tosteauthor=K.+R.+Kozakauthor=S.+Spencerauthor=P.+Polakisauthor=A.+G.+Polsonauthor=J.+A.+Flygareauthor=J.+R.+Junutula&title=Modulating+therapeutic+activity+and+toxicity+of+pyrrolobenzodiazepine+antibody-Drug+conjugates+with+self-immolative+disulfide+linkers&doi=10.1158%2F1535-7163.MCT-16-0641"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23bR"><div class="casContent"><span class="casTitleNuber">23b</span><div class="casTitle"><span class="NLM_cas:atitle">Modulating Therapeutic Activity and Toxicity of Pyrrolobenzodiazepine Antibody-Drug Conjugates with Self-Immolative Disulfide Linkers</span></div><div class="casAuthors">Pillow, Thomas H.; Schutten, Melissa; Yu, Shang-Fan; Ohri, Rachana; Sadowsky, Jack; Poon, Kirsten Achilles; Solis, Willy; Zhong, Fiona; Del Rosario, Geoffrey; Go, Mary Ann T.; Lau, Jeffrey; Yee, Sharon; He, Jintang; Liu, Luna; Ng, Carl; Xu, Keyang; Leipold, Douglas D.; Kamath, Amrita V.; Zhang, Donglu; Masterson, Luke; Gregson, Stephen J.; Howard, Philip W.; Fang, Fan; Chen, Jinhua; Gunzner-Toste, Janet; Kozak, Katherine K.; Spencer, Susan; Polakis, Paul; Polson, Andrew G.; Flygare, John A.; Junutula, Jagath R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">871-878</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A novel disulfide linker was designed to enable a direct connection between cytotoxic pyrrolobenzodiazepine (PBD) drugs and the cysteine on a targeting antibody for use in antibody-drug conjugates (ADCs).  The ADCs composed of a cysteine-engineered antibody were armed with a PBD using a self-immolative disulfide linker.  Both the chem. linker and the antibody site were optimized for this new bioconjugation strategy to provide a highly stable and efficacious ADC.  This novel disulfide ADC was compared with a conjugate contg. the same PBD drug, but attached to the antibody via a peptide linker.  Both ADCs had similar efficacy in mice bearing human tumor xenografts.  Safety studies in rats revealed that the disulfide-linked ADC had a higher MTD than the peptide-linked ADC.  Overall, these data suggest that the novel self-immolative disulfide linker represents a valuable way to construct ADCs with equiv. efficacy and improved safety.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZP_imFtNeBrVg90H21EOLACvtfcHk0lj4g2HLVYLFuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmvFemtrw%253D&md5=6bf9501951de0cfad9d890a4ef9492b1</span></div><a href="/servlet/linkout?suffix=cit23b&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-16-0641&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-16-0641%26sid%3Dliteratum%253Aachs%26aulast%3DPillow%26aufirst%3DT.%2BH.%26aulast%3DSchutten%26aufirst%3DM.%26aulast%3DYu%26aufirst%3DS.-F.%26aulast%3DOhri%26aufirst%3DR.%26aulast%3DSadowsky%26aufirst%3DJ.%26aulast%3DAchilles-Poon%26aufirst%3DK.%26aulast%3DSolis%26aufirst%3DW.%26aulast%3DZhong%26aufirst%3DF.%26aulast%3DDel%2BRosario%26aufirst%3DG.%26aulast%3DGo%26aufirst%3DM.%2BA.%2BT.%26aulast%3DLau%26aufirst%3DJ.%26aulast%3DYee%26aufirst%3DS.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DNg%26aufirst%3DC.%26aulast%3DXu%26aufirst%3DK.%26aulast%3DLeipold%26aufirst%3DD.%2BD.%26aulast%3DKamath%26aufirst%3DA.%2BV.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DMasterson%26aufirst%3DL.%26aulast%3DGregson%26aufirst%3DS.%2BJ.%26aulast%3DHoward%26aufirst%3DP.%2BW.%26aulast%3DFang%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DGunzner-Toste%26aufirst%3DJ.%26aulast%3DKozak%26aufirst%3DK.%2BR.%26aulast%3DSpencer%26aufirst%3DS.%26aulast%3DPolakis%26aufirst%3DP.%26aulast%3DPolson%26aufirst%3DA.%2BG.%26aulast%3DFlygare%26aufirst%3DJ.%2BA.%26aulast%3DJunutula%26aufirst%3DJ.%2BR.%26atitle%3DModulating%2520therapeutic%2520activity%2520and%2520toxicity%2520of%2520pyrrolobenzodiazepine%2520antibody-Drug%2520conjugates%2520with%2520self-immolative%2520disulfide%2520linkers%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2017%26volume%3D16%26spage%3D871%26epage%3D878%26doi%3D10.1158%2F1535-7163.MCT-16-0641" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span> The mAb nomenclature
follows the EU convention described in:<span class="NLM_contrib-group"><span class="NLM_string-name">Edelman, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gall, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gotlieb, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rutishauser, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waxdal, M. J.</span></span> <span> </span><span class="NLM_article-title">The covalent structure of an entire γG immunoglobin molecule</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1969</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">78</span>– <span class="NLM_lpage">85</span>, <span class="refDoi"> DOI: 10.1073/pnas.63.1.78</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1073%2Fpnas.63.1.78" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=5257969" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADyaE3cXivFyjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=1969&pages=78-85&author=G.+M.+Edelmanauthor=B.+A.+Cunninghamauthor=W.+E.+Gallauthor=P.+D.+Gotliebauthor=U.+Rutishauserauthor=M.+J.+Waxdal&title=The+covalent+structure+of+an+entire+%CE%B3G+immunoglobin+molecule&doi=10.1073%2Fpnas.63.1.78"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent structure of an entire γG immunoglobulin molecule</span></div><div class="casAuthors">Edelman, Gerald M.; Cunningham, Bruce A.; Gall, W. Einar; Gottlieb, Paul D.; Rutishauser, Urs; Waxdal, Myron J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1969</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">78-85</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">The complete amino acid sequence of a human γGl immunoglobulin (Eu) has been detd. and the arrangement of all of the disulfide bonds has been established.  Comparison of the sequence with that of another myeloma protein (He) suggests that the variable regions of heavy and light chains are homologous and similar in length.  The const. portion of the heavy chain contains 3 homology regions each of which is similar in size and homologous to the const. region of the light chain.  Each variable region and each const. homology region contains 1 intrachain disulfide bond.  The half-cystines participating in the interchain bonds are all clustered within a stretch of 10 residues at the middle of the heavy chains.  These data support the hypothesis that immunoglobulins evolved by gene duplication after early divergence of V genes, which specified antigen-binding functions, and C genes, which specified other functions of antibody mols.  Each polypeptide chain may therefore be specified by 2 genes, V and C, which are fused to form a single gene (translocation hypothesis).  The internal homologies and symmetry of the mol. suggest that homology regions may have similar 3-dimensional structures each consisting of a compact domain which contributes to at least 1 active site (domain hypothesis).  Both hypotheses are in accord with the linear regional differentiation of function in antibody mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjqW9i9JX9g7Vg90H21EOLACvtfcHk0lj4g2HLVYLFuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE3cXivFyjtQ%253D%253D&md5=18ba9b6417ce609be8870e1d142e7fc9</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1073%2Fpnas.63.1.78&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.63.1.78%26sid%3Dliteratum%253Aachs%26aulast%3DEdelman%26aufirst%3DG.%2BM.%26aulast%3DCunningham%26aufirst%3DB.%2BA.%26aulast%3DGall%26aufirst%3DW.%2BE.%26aulast%3DGotlieb%26aufirst%3DP.%2BD.%26aulast%3DRutishauser%26aufirst%3DU.%26aulast%3DWaxdal%26aufirst%3DM.%2BJ.%26atitle%3DThe%2520covalent%2520structure%2520of%2520an%2520entire%2520%25CE%25B3G%2520immunoglobin%2520molecule%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1969%26volume%3D63%26spage%3D78%26epage%3D85%26doi%3D10.1073%2Fpnas.63.1.78" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="note"><p class="first last">For a comparison of the EU and the alternate Kabat mAb nomenclature systems, see Table S1 in ref <a onclick="showRef(event, 'cit26a'); return false;" href="javascript:void(0);" class="ref cit26a">(26a)</a>.</p></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sadowsky, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pillow, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">dela Cruz-Chuh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Donohue, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Rosario, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, G. D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozak, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leipold, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wai, J.</span></span> <span> </span><span class="NLM_article-title">Development of efficient chemistry to generate site-specific disulfide-linked protein- and peptide-payload conjugates: Application to THIOMAB antibody-drug conjugates</span>. <i>Bioconjugate Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">2086</span>– <span class="NLM_lpage">2098</span>, <span class="refDoi"> DOI: 10.1021/acs.bioconjchem.7b00258</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.bioconjchem.7b00258" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVaktLvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2017&pages=2086-2098&author=J.+D.+Sadowskyauthor=T.+H.+Pillowauthor=J.+Chenauthor=F.+Fanauthor=C.+Heauthor=Y.+Wangauthor=G.+Yanauthor=H.+Yaoauthor=Z.+Xuauthor=S.+Martinauthor=D.+Zhangauthor=P.+Chuauthor=J.+dela+Cruz-Chuhauthor=A.+O%E2%80%99Donohueauthor=G.+Liauthor=G.+Del+Rosarioauthor=J.+Heauthor=L.+Liuauthor=C.+Ngauthor=D.+Suauthor=G.+D.+L.+Phillipsauthor=K.+R.+Kozakauthor=S.-F.+Yuauthor=K.+Xuauthor=D.+Leipoldauthor=J.+Wai&title=Development+of+efficient+chemistry+to+generate+site-specific+disulfide-linked+protein-+and+peptide-payload+conjugates%3A+Application+to+THIOMAB+antibody-drug+conjugates&doi=10.1021%2Facs.bioconjchem.7b00258"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Efficient Chemistry to Generate Site-Specific Disulfide-Linked Protein- and Peptide-Payload Conjugates: Application to THIOMAB Antibody-Drug Conjugates</span></div><div class="casAuthors">Sadowsky, Jack D.; Pillow, Thomas H.; Chen, Jinhua; Fan, Fang; He, Changrong; Wang, Yanli; Yan, Gang; Yao, Hui; Xu, Zijin; Martin, Shanique; Zhang, Donglu; Chu, Phillip; dela Cruz-Chuh, Josefa; ODonohue, Aimee; Li, Guangmin; Del Rosario, Geoffrey; He, Jintang; Liu, Luna; Ng, Carl; Su, Dian; Lewis Phillips, Gail D.; Kozak, Katherine R.; Yu, Shang-Fan; Xu, Keyang; Leipold, Douglas; Wai, John</div><div class="citationInfo"><span class="NLM_cas:title">Bioconjugate Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2086-2098</span>CODEN:
                <span class="NLM_cas:coden">BCCHES</span>;
        ISSN:<span class="NLM_cas:issn">1043-1802</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Conjugation of small mol. payloads to specific cysteine residues on proteins via a disulfide bond represents an attractive strategy to generate redox-sensitive bioconjugates, which have value as potential diagnostic reagents or therapeutics.  Advancement of such "direct-disulfide" bioconjugates to the clinic necessitates chem. methods to form disulfide connections efficiently, without byproducts.  The disulfide connection must also be resistant to premature cleavage by thiols prior to arrival at the targeted tissue.  We show here that commonly-employed methods to generate direct disulfide-linked bioconjugates are inadequate for addressing these challenges.  We describe our efforts to optimize direct-disulfide conjugation chem., focusing on the generation of conjugates between cytotoxic payloads and cysteine-engineered antibodies (i.e., THIOMAB antibody-drug conjugates, or TDCs).  This work culminates in the development of novel, high-yielding conjugation chem. for creating direct payload disulfide connections to any of several Cys mutation sites in THIOMAB antibodies or to Cys sites in other biomols. (e.g., human serum albumin and cell-penetrating peptides).  We conclude by demonstrating that hindered direct disulfide TDCs with two Me groups adjacent to the disulfide, which have heretofore not been described for any bioconjugate, are more stable and more efficacious in mouse tumor xenograft studies than less hindered analogs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIap6CQnm0X7Vg90H21EOLACvtfcHk0lj4g2HLVYLFuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVaktLvL&md5=9c437d3e1aa16e51e3f571fea95785c4</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.bioconjchem.7b00258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.bioconjchem.7b00258%26sid%3Dliteratum%253Aachs%26aulast%3DSadowsky%26aufirst%3DJ.%2BD.%26aulast%3DPillow%26aufirst%3DT.%2BH.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DFan%26aufirst%3DF.%26aulast%3DHe%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DYan%26aufirst%3DG.%26aulast%3DYao%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DChu%26aufirst%3DP.%26aulast%3Ddela%2BCruz-Chuh%26aufirst%3DJ.%26aulast%3DO%25E2%2580%2599Donohue%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DDel%2BRosario%26aufirst%3DG.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DNg%26aufirst%3DC.%26aulast%3DSu%26aufirst%3DD.%26aulast%3DPhillips%26aufirst%3DG.%2BD.%2BL.%26aulast%3DKozak%26aufirst%3DK.%2BR.%26aulast%3DYu%26aufirst%3DS.-F.%26aulast%3DXu%26aufirst%3DK.%26aulast%3DLeipold%26aufirst%3DD.%26aulast%3DWai%26aufirst%3DJ.%26atitle%3DDevelopment%2520of%2520efficient%2520chemistry%2520to%2520generate%2520site-specific%2520disulfide-linked%2520protein-%2520and%2520peptide-payload%2520conjugates%253A%2520Application%2520to%2520THIOMAB%2520antibody-drug%2520conjugates%26jtitle%3DBioconjugate%2520Chem.%26date%3D2017%26volume%3D28%26spage%3D2086%26epage%3D2098%26doi%3D10.1021%2Facs.bioconjchem.7b00258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Ohri, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhakta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Donohue, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bos, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farahi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhakta, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pillow, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raab, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polakis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erickson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Junutula, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozak, K. R.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">High-throughput cysteine scanning to identify stable antibody conjugation sites for maleimide- and disulfide-based linkers</span>. <i>Bioconjugate Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">473</span>– <span class="NLM_lpage">485</span>, <span class="refDoi"> DOI: 10.1021/acs.bioconjchem.7b00791</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.bioconjchem.7b00791" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVKku7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2018&pages=473-485&author=R.+Ohriauthor=S.+Bhaktaauthor=A.+O%E2%80%99Donohueauthor=J.+Chuhauthor=S.+P.+Tsaiauthor=R.+Cookauthor=B.+Weiauthor=C.+Ngauthor=A.+W.+Wongauthor=A.+B.+Bosauthor=F.+Farahiauthor=J.+Bhaktaauthor=T.+H.+Pillowauthor=H.+Raabauthor=P.+Polakisauthor=Y.+Liuauthor=H.+Ericksonauthor=J.+R.+Junutulaauthor=K.+R.+Kozak&title=High-throughput+cysteine+scanning+to+identify+stable+antibody+conjugation+sites+for+maleimide-+and+disulfide-based+linkers&doi=10.1021%2Facs.bioconjchem.7b00791"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26aR"><div class="casContent"><span class="casTitleNuber">26a</span><div class="casTitle"><span class="NLM_cas:atitle">High-Throughput Cysteine Scanning To Identify Stable Antibody Conjugation Sites for Maleimide- and Disulfide-Based Linkers</span></div><div class="casAuthors">Ohri, Rachana; Bhakta, Sunil; Fourie-O'Donohue, Aimee; dela Cruz-Chuh, Josefa; Tsai, Siao Ping; Cook, Ryan; Wei, Binqing; Ng, Carl; Wong, Athena W.; Bos, Aaron B.; Farahi, Farzam; Bhakta, Jiten; Pillow, Thomas H.; Raab, Helga; Vandlen, Richard; Polakis, Paul; Liu, Yichin; Erickson, Hans; Junutula, Jagath R.; Kozak, Katherine R.</div><div class="citationInfo"><span class="NLM_cas:title">Bioconjugate Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">473-485</span>CODEN:
                <span class="NLM_cas:coden">BCCHES</span>;
        ISSN:<span class="NLM_cas:issn">1043-1802</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">THIOMAB antibody technol. utilizes cysteine residues engineered onto an antibody to allow for site-specific conjugation.  The technol. has enabled the exploration of different attachment sites on the antibody in combination with small mols., peptides, or proteins to yield antibody conjugates with unique properties.  As reported previously, the specific location of the site of conjugation on an antibody can impact the stability of the linkage to the engineered cysteine for both thio-succinimide and disulfide bonds.  High stability of the linkage is usually desired to maximize the delivery of the cargo to the intended target.  In the current study, cysteines were individually substituted into every position of the anti-HER2 antibody (trastuzumab), and the stabilities of drug conjugations at those sites were evaluated.  We screened a total of 648 THIOMAB antibody-drug conjugates, each generated from a trastuzamab prepd. by sequentially mutating non-cysteine amino acids in the light and heavy chains to cysteine.  Each THIOMAB antibody variant was conjugated to either maleimidocaproyl-valine-citrulline-p-aminobenzyloxycarbonyl-monomethyl auristatin E (MC-vc-PAB-MMAE) or pyridyl disulfide monomethyl auristatin E (PDS-MMAE) using a high-throughput, on-bead conjugation and purifn. method.  Greater than 50% of the THIOMAB antibody variants were successfully conjugated to both MMAE derivs. with a drug to antibody ratio (DAR) of >0.5 and <50% aggregation.  The relative in vitro plasma stabilities for approx. 750 conjugates were assessed using enzyme-linked immunosorbent assays, and stable sites were confirmed with affinity-capture LC/MS-based detection methods.  Highly stable conjugation sites for the two types of MMAE derivs. were identified on both the heavy and light chains.  Although the stabilities of maleimide conjugates were shown to be greater than those of the disulfide conjugates, many sites were identified that were stable for both.  Furthermore, in vitro stabilities of selected stable sites translated across different cytotoxic payloads and different target antibodies as well as to in vivo stability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLlgFVNRmHMbVg90H21EOLACvtfcHk0ljdsBdSxI8BjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVKku7g%253D&md5=318146c1baea8094b671e99b5c429627</span></div><a href="/servlet/linkout?suffix=cit26a&amp;dbid=16384&amp;doi=10.1021%2Facs.bioconjchem.7b00791&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.bioconjchem.7b00791%26sid%3Dliteratum%253Aachs%26aulast%3DOhri%26aufirst%3DR.%26aulast%3DBhakta%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Donohue%26aufirst%3DA.%26aulast%3DChuh%26aufirst%3DJ.%26aulast%3DTsai%26aufirst%3DS.%2BP.%26aulast%3DCook%26aufirst%3DR.%26aulast%3DWei%26aufirst%3DB.%26aulast%3DNg%26aufirst%3DC.%26aulast%3DWong%26aufirst%3DA.%2BW.%26aulast%3DBos%26aufirst%3DA.%2BB.%26aulast%3DFarahi%26aufirst%3DF.%26aulast%3DBhakta%26aufirst%3DJ.%26aulast%3DPillow%26aufirst%3DT.%2BH.%26aulast%3DRaab%26aufirst%3DH.%26aulast%3DPolakis%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DErickson%26aufirst%3DH.%26aulast%3DJunutula%26aufirst%3DJ.%2BR.%26aulast%3DKozak%26aufirst%3DK.%2BR.%26atitle%3DHigh-throughput%2520cysteine%2520scanning%2520to%2520identify%2520stable%2520antibody%2520conjugation%2520sites%2520for%2520maleimide-%2520and%2520disulfide-based%2520linkers%26jtitle%3DBioconjugate%2520Chem.%26date%3D2018%26volume%3D29%26spage%3D473%26epage%3D485%26doi%3D10.1021%2Facs.bioconjchem.7b00791" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit26b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Vollmar, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohri, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leipold, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cosino, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fourie-O’Donohue, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozak, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamath, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazar, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erickson, H. K.</span></span> <span> </span><span class="NLM_article-title">Attachment site cysteine thiol pKa is a key driver for site-dependent stability of THIOMAB antibody-drug conjugates</span>. <i>Bioconjugate Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">2538</span>– <span class="NLM_lpage">2548</span>, <span class="refDoi"> DOI: 10.1021/acs.bioconjchem.7b00365</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.bioconjchem.7b00365" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVGks7vL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2017&pages=2538-2548&author=B.+S.+Vollmarauthor=B.+Weiauthor=R.+Ohriauthor=J.+Zhouauthor=J.+Heauthor=S.-F.+Yuauthor=D.+Leipoldauthor=E.+Cosinoauthor=S.+Yeeauthor=A.+Fourie-O%E2%80%99Donohueauthor=G.+Liauthor=G.+L.+Phillipsauthor=K.+R.+Kozakauthor=A.+Kamathauthor=K.+Xuauthor=G.+Leeauthor=G.+A.+Lazarauthor=H.+K.+Erickson&title=Attachment+site+cysteine+thiol+pKa+is+a+key+driver+for+site-dependent+stability+of+THIOMAB+antibody-drug+conjugates&doi=10.1021%2Facs.bioconjchem.7b00365"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26bR"><div class="casContent"><span class="casTitleNuber">26b</span><div class="casTitle"><span class="NLM_cas:atitle">Attachment Site Cysteine Thiol pKa Is a Key Driver for Site-Dependent Stability of THIOMAB Antibody-Drug Conjugates</span></div><div class="casAuthors">Vollmar, Breanna S.; Wei, Binqing; Ohri, Rachana; Zhou, Jianhui; He, Jintang; Yu, Shang-Fan; Leipold, Douglas; Cosino, Ely; Yee, Sharon; Fourie-O'Donohue, Aimee; Li, Guangmin; Phillips, Gail L.; Kozak, Katherine R.; Kamath, Amrita; Xu, Keyang; Lee, Genee; Lazar, Greg A.; Erickson, Hans K.</div><div class="citationInfo"><span class="NLM_cas:title">Bioconjugate Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2538-2548</span>CODEN:
                <span class="NLM_cas:coden">BCCHES</span>;
        ISSN:<span class="NLM_cas:issn">1043-1802</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The incorporation of cysteines into antibodies by mutagenesis allows for the direct conjugation of small mols. to specific sites on the antibody via disulfide bonds.  The stability of the disulfide bond linkage between the small mol. and the antibody is highly dependent on the location of the engineered cysteine in either the heavy chain (HC) or the light chain (LC) of the antibody.  Here, we explore the basis for this site-dependent stability.  We evaluated the in vivo efficacy and pharmacokinetics of five different cysteine mutants of trastuzumab conjugated to a pyrrolobenzodiazepine (PBD) via disulfide bonds.  A significant correlation was obsd. between disulfide stability and efficacy for the conjugates.  We hypothesized that the obsd. site-dependent stability of the disulfide-linked conjugates could be due to differences in the attachment site cysteine thiol pKa.  We measured the cysteine thiol pKa using isothermal titrn. calorimetry (ITC) and found that the variants with the highest thiol pKa (LC K149C and HC A140C) were found to yield the conjugates with the greatest in vivo stability.  Guided by homol. modeling, we identified several mutations adjacent to LC K149C that reduced the cysteine thiol pKa and, thus, decreased the in vivo stability of the disulfide-linked PBD conjugated to LC K149C.  We also present results suggesting that the high thiol pKa of LC K149C is responsible for the sustained circulation stability of LC K149C TDCs utilizing a maleimide-based linker.  Taken together, our results provide evidence that the site-dependent stability of cys-engineered antibody-drug conjugates may be explained by interactions between the engineered cysteine and the local protein environment that serves to modulate the side-chain thiol pKa.  The influence of cysteine thiol pKa on stability and efficacy offers a new parameter for the optimization of ADCs that utilize cysteine engineering.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAfrN62jixCbVg90H21EOLACvtfcHk0ljdsBdSxI8BjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVGks7vL&md5=ba1805c493f468d4ffbb198dbf8b3d02</span></div><a href="/servlet/linkout?suffix=cit26b&amp;dbid=16384&amp;doi=10.1021%2Facs.bioconjchem.7b00365&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.bioconjchem.7b00365%26sid%3Dliteratum%253Aachs%26aulast%3DVollmar%26aufirst%3DB.%2BS.%26aulast%3DWei%26aufirst%3DB.%26aulast%3DOhri%26aufirst%3DR.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DS.-F.%26aulast%3DLeipold%26aufirst%3DD.%26aulast%3DCosino%26aufirst%3DE.%26aulast%3DYee%26aufirst%3DS.%26aulast%3DFourie-O%25E2%2580%2599Donohue%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DPhillips%26aufirst%3DG.%2BL.%26aulast%3DKozak%26aufirst%3DK.%2BR.%26aulast%3DKamath%26aufirst%3DA.%26aulast%3DXu%26aufirst%3DK.%26aulast%3DLee%26aufirst%3DG.%26aulast%3DLazar%26aufirst%3DG.%2BA.%26aulast%3DErickson%26aufirst%3DH.%2BK.%26atitle%3DAttachment%2520site%2520cysteine%2520thiol%2520pKa%2520is%2520a%2520key%2520driver%2520for%2520site-dependent%2520stability%2520of%2520THIOMAB%2520antibody-drug%2520conjugates%26jtitle%3DBioconjugate%2520Chem.%26date%3D2017%26volume%3D28%26spage%3D2538%26epage%3D2548%26doi%3D10.1021%2Facs.bioconjchem.7b00365" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blake, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">den Besten, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fourie, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartman, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingalla, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozak, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leong, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nannini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohri, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pillow, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowntree, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sampath, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandlen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vollmar, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wertz, I. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span> <span> </span><span class="NLM_article-title">Conjugation of indoles to antibodies through a novel self-immolating linker</span>. <i>Chem. – Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">4830</span>– <span class="NLM_lpage">4834</span>, <span class="refDoi"> DOI: 10.1002/chem.201800859</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1002%2Fchem.201800859" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=29493023" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1cXlslSnu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=4830-4834&author=P.+S.+Dragovichauthor=R.+A.+Blakeauthor=C.+Chenauthor=J.+Chenauthor=J.+Chuhauthor=W.+den+Bestenauthor=F.+Fanauthor=A.+Fourieauthor=S.+J.+Hartmanauthor=C.+Heauthor=J.+Heauthor=E.+R.+Ingallaauthor=K.+R.+Kozakauthor=S.+R.+Leongauthor=J.+Luauthor=Y.+Maauthor=L.+Mengauthor=M.+Nanniniauthor=J.+Oehauthor=R.+Ohriauthor=G.+L.+Phillipsauthor=T.+H.+Pillowauthor=R.+K.+Rowntreeauthor=D.+Sampathauthor=R.+Vandlenauthor=B.+Vollmarauthor=J.+Waiauthor=I.+E.+Wertzauthor=K.+Xuauthor=Z.+Xuauthor=D.+Zhang&title=Conjugation+of+indoles+to+antibodies+through+a+novel+self-immolating+linker&doi=10.1002%2Fchem.201800859"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Conjugation of Indoles to Antibodies through a Novel Self-Immolating Linker</span></div><div class="casAuthors">Dragovich, Peter S.; Blake, Robert A.; Chen, Chunjiao; Chen, Jinhua; Chuh, Josefa; den Besten, Willem; Fan, Fang; Fourie, Aimee; Hartman, Steven J.; He, Changrong; He, Jintang; Ingalla, Ellen Rei; Kozak, Katherine R.; Leong, Steven R.; Lu, Jiawei; Ma, Yong; Meng, Lingyao; Nannini, Michelle; Oeh, Jason; Ohri, Rachana; Lewis Phillips, Gail; Pillow, Thomas H.; Rowntree, Rebecca K.; Sampath, Deepak; Vandlen, Richard; Vollmar, Breanna; Wai, John; Wertz, Ingrid E.; Xu, Keyang; Xu, Zijin; Zhang, Donglu</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">4830-4834</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A novel strategy to attach indole-contg. payloads to antibodies through a carbamate moiety and a self-immolating, disulfide-based linker is described.  This new strategy was employed to connect a selective estrogen receptor down-regulator (SERD) to various antibodies in a site-selective manner.  The resulting conjugates displayed potent, antigen-dependent down-regulation of estrogen receptor levels in MCF7-neo/HER2 and MCF7-hB7H4 cells.  They also exhibited similar antigen-dependent modulation of the estrogen receptor in tumors when administered i.v. to mice bearing MCF7-neo/HER2 tumor xenografts.  The indole-carbamate moiety present in the new linker was stable in whole blood from various species and also exhibited good in vivo stability properties in mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKhcOeE6xSRLVg90H21EOLACvtfcHk0ljFXLzU4z2MoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXlslSnu78%253D&md5=f1c82d8a721646a690e735c3ed38b53a</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fchem.201800859&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.201800859%26sid%3Dliteratum%253Aachs%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26aulast%3DBlake%26aufirst%3DR.%2BA.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DChuh%26aufirst%3DJ.%26aulast%3Dden%2BBesten%26aufirst%3DW.%26aulast%3DFan%26aufirst%3DF.%26aulast%3DFourie%26aufirst%3DA.%26aulast%3DHartman%26aufirst%3DS.%2BJ.%26aulast%3DHe%26aufirst%3DC.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DIngalla%26aufirst%3DE.%2BR.%26aulast%3DKozak%26aufirst%3DK.%2BR.%26aulast%3DLeong%26aufirst%3DS.%2BR.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DMeng%26aufirst%3DL.%26aulast%3DNannini%26aufirst%3DM.%26aulast%3DOeh%26aufirst%3DJ.%26aulast%3DOhri%26aufirst%3DR.%26aulast%3DPhillips%26aufirst%3DG.%2BL.%26aulast%3DPillow%26aufirst%3DT.%2BH.%26aulast%3DRowntree%26aufirst%3DR.%2BK.%26aulast%3DSampath%26aufirst%3DD.%26aulast%3DVandlen%26aufirst%3DR.%26aulast%3DVollmar%26aufirst%3DB.%26aulast%3DWai%26aufirst%3DJ.%26aulast%3DWertz%26aufirst%3DI.%2BE.%26aulast%3DXu%26aufirst%3DK.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DD.%26atitle%3DConjugation%2520of%2520indoles%2520to%2520antibodies%2520through%2520a%2520novel%2520self-immolating%2520linker%26jtitle%3DChem.%2520%25E2%2580%2593%2520Eur.%2520J.%26date%3D2018%26volume%3D24%26spage%3D4830%26epage%3D4834%26doi%3D10.1002%2Fchem.201800859" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span> Others have used similar
SPR techniques to examine ternary complexes formed between BRD4, VHL,
and related chimeric degrader molecules:<span class="NLM_contrib-group"><span class="NLM_string-name">Roy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitworth, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galant, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farnaby, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumpel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">SPR-measured dissociation kinetics of PROTAC ternary complexes influence target degradation rate</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">361</span>– <span class="NLM_lpage">368</span>, <span class="refDoi"> DOI: 10.1021/acschembio.9b00092</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.9b00092" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVCisrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=361-368&author=M.+Royauthor=S.+Winklerauthor=S.+J.+Hughesauthor=C.+Whitworthauthor=M.+Galantauthor=W.+Farnabyauthor=K.+Rumpelauthor=A.+Ciulli&title=SPR-measured+dissociation+kinetics+of+PROTAC+ternary+complexes+influence+target+degradation+rate&doi=10.1021%2Facschembio.9b00092"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">SPR-Measured Dissociation Kinetics of PROTAC Ternary Complexes Influence Target Degradation Rate</span></div><div class="casAuthors">Roy, Michael J.; Winkler, Sandra; Hughes, Scott J.; Whitworth, Claire; Galant, Michael; Farnaby, William; Rumpel, Klaus; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">361-368</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bifunctional degrader mols., known as proteolysis-targeting chimeras (PROTACs), function by recruiting a target to an E3 ligase, forming a target/PROTAC/ligase ternary complex.  Despite the importance of this key intermediate species, no detailed validation of a method to directly det. binding parameters for ternary complex kinetics has been reported, and it remains to be addressed whether tuning the kinetics of PROTAC ternary complexes may be an effective strategy to improve the efficiency of targeted protein degrdn.  Here, we develop an SPR-based assay to quantify the stability of PROTAC-induced ternary complexes by measuring for the first time the kinetics of their formation and dissocn. in vitro using purified proteins.  We benchmark our assay using four PROTACs that target the bromodomains (BDs) of bromodomain and extraterminal domain proteins Brd2, Brd3, and Brd4 to the von Hippel-Lindau E3 ligase (VHL).  We reveal marked differences in ternary complex off-rates for different PROTACs that exhibit either pos. or neg. cooperativity for ternary complex formation relative to binary binding.  The pos. cooperative degrader MZ1 forms comparatively stable and long-lived ternary complexes with either Brd4BD2 or Brd2BD2 and VHL.  Equivalent complexes with Brd3BD2 are destabilized due to a single amino acid difference (Glu/Gly swap) present in the bromodomain.  We observe that this difference in ternary complex dissociative half-life correlates to a greater initial rate of intracellular degrdn. of Brd2 and Brd4 relative to Brd3.  These findings establish a novel assay to measure the kinetics of PROTAC ternary complexes and elucidate the important kinetic parameters that drive effective target degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzFhvuKYztcbVg90H21EOLACvtfcHk0ljFXLzU4z2MoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVCisrg%253D&md5=629d1acbd218b668a2696b727946962b</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facschembio.9b00092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.9b00092%26sid%3Dliteratum%253Aachs%26aulast%3DRoy%26aufirst%3DM.%26aulast%3DWinkler%26aufirst%3DS.%26aulast%3DHughes%26aufirst%3DS.%2BJ.%26aulast%3DWhitworth%26aufirst%3DC.%26aulast%3DGalant%26aufirst%3DM.%26aulast%3DFarnaby%26aufirst%3DW.%26aulast%3DRumpel%26aufirst%3DK.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DSPR-measured%2520dissociation%2520kinetics%2520of%2520PROTAC%2520ternary%2520complexes%2520influence%2520target%2520degradation%2520rate%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2019%26volume%3D14%26spage%3D361%26epage%3D368%26doi%3D10.1021%2Facschembio.9b00092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of a macrocyclic PROTAC</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1727</span>– <span class="NLM_lpage">1734</span>, <span class="refDoi"> DOI: 10.1002/anie.201914396</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1002%2Fanie.201914396" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlyitbjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2020&pages=1727-1734&author=A.+Testaauthor=S.+J.+Hughesauthor=X.+Lucasauthor=J.+E.+Wrightauthor=A.+Ciulli&title=Structure-based+design+of+a+macrocyclic+PROTAC&doi=10.1002%2Fanie.201914396"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29aR"><div class="casContent"><span class="casTitleNuber">29a</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of a Macrocyclic PROTAC</span></div><div class="casAuthors">Testa, Andrea; Hughes, Scott J.; Lucas, Xavier; Wright, Jane E.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1727-1734</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Constraining a mol. in its bioactive conformation via macrocyclization represents an attractive strategy to rationally design functional chem. probes.  While this approach has been applied to enzyme inhibitors or receptor antagonists, to date it remains unprecedented for bifunctional mols. that bring proteins together, such as PROTAC degraders.  Herein, the authors report the design and synthesis of a macrocyclic PROTAC by adding a cyclizing linker to the BET degrader MZ1.  A cocrystal structure of macroPROTAC-1 bound in a ternary complex with VHL and the second bromodomain of Brd4 validated the rational design.  Biophys. studies revealed enhanced discrimination between the second and the first bromodomains of BET proteins.  Despite a 12-fold loss of binary binding affinity for Brd4, macroPROTAC-1 exhibited cellular activity comparable to MZ1.  The findings support macrocyclization as an advantageous strategy to enhance PROTAC degrdn. potency and selectivity between homologous targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZjLgy_V3_abVg90H21EOLACvtfcHk0ljFXLzU4z2MoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlyitbjK&md5=6f3f279f6220ec30b30ca3d2c931da95</span></div><a href="/servlet/linkout?suffix=cit29a&amp;dbid=16384&amp;doi=10.1002%2Fanie.201914396&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201914396%26sid%3Dliteratum%253Aachs%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DHughes%26aufirst%3DS.%2BJ.%26aulast%3DLucas%26aufirst%3DX.%26aulast%3DWright%26aufirst%3DJ.%2BE.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructure-based%2520design%2520of%2520a%2520macrocyclic%2520PROTAC%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2020%26volume%3D59%26spage%3D1727%26epage%3D1734%26doi%3D10.1002%2Fanie.201914396" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit29b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamont, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Structural basis of PROTAC cooperative recognition for selective protein degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">514</span>– <span class="NLM_lpage">521</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2329</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1038%2Fnchembio.2329" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=28288108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=514-521&author=M.+S.+Gaddauthor=A.+Testaauthor=X.+Lucasauthor=K.-H.+Chanauthor=W.+Chenauthor=D.+J.+Lamontauthor=M.+Zengerleauthor=A.+Ciulli&title=Structural+basis+of+PROTAC+cooperative+recognition+for+selective+protein+degradation&doi=10.1038%2Fnchembio.2329"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29bR"><div class="casContent"><span class="casTitleNuber">29b</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of PROTAC cooperative recognition for selective protein degradation</span></div><div class="casAuthors">Gadd, Morgan S.; Testa, Andrea; Lucas, Xavier; Chan, Kwok-Ho; Chen, Wenzhang; Lamont, Douglas J.; Zengerle, Michael; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">514-521</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Inducing macromol. interactions with small mols. to activate cellular signaling is a challenging goal.  PROTACs (proteolysis-targeting chimeras) are bifunctional mols. that recruit a target protein in proximity to an E3 ubiquitin ligase to trigger protein degrdn.  Structural elucidation of the key ternary ligase-PROTAC-target species and its impact on target degrdn. selectivity remain elusive.  Here, we solved the crystal structure of Brd4 degrader MZ1 in complex with human VHL and the Brd4 bromodomain (Brd4BD2).  The ligand folds into itself to allow formation of specific intermol. interactions in the ternary complex.  Isothermal titrn. calorimetry studies, supported by surface mutagenesis and proximity assays, were consistent with pronounced cooperative formation of ternary complexes with Brd4BD2.  Structure-based-designed compd. AT1 exhibited highly selective depletion of Brd4 in cells.  These results elucidate how PROTAC-induced de novo contacts dictate preferential recruitment of a target protein into a stable and cooperative complex with an E3 ligase for selective degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJri00jB5YJLVg90H21EOLACvtfcHk0lgskLJDH8MZsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D&md5=dfee13913106d362d0972d7b1a62d700</span></div><a href="/servlet/linkout?suffix=cit29b&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2329&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2329%26sid%3Dliteratum%253Aachs%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DLucas%26aufirst%3DX.%26aulast%3DChan%26aufirst%3DK.-H.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DLamont%26aufirst%3DD.%2BJ.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructural%2520basis%2520of%2520PROTAC%2520cooperative%2520recognition%2520for%2520selective%2520protein%2520degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D514%26epage%3D521%26doi%3D10.1038%2Fnchembio.2329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Brandish, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmieri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antonenko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beaumont, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benso, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cancilla, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fayadat-Dilman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Figueroa, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Firdos, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garbaccio, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garvin-Queen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gately, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geda, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haines, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hseih, S.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodges, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kern, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knudsen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwasnjuk, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manibusan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moy, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stivers, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomazela, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woo, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaller, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zielstorff, M.</span></span> <span> </span><span class="NLM_article-title">Development of anti-CD74 antibody-drug conjugates to target glucocorticoids to immune cells</span>. <i>Bioconjugate Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">2357</span>– <span class="NLM_lpage">2369</span>, <span class="refDoi"> DOI: 10.1021/acs.bioconjchem.8b00312</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.bioconjchem.8b00312" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFGlsb7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2018&pages=2357-2369&author=P.+E.+Brandishauthor=A.+Palmieriauthor=S.+Antonenkoauthor=M.+Beaumontauthor=L.+Bensoauthor=M.+Cancillaauthor=M.+Chengauthor=L.+Fayadat-Dilmanauthor=G.+Fengauthor=I.+Figueroaauthor=J.+Firdosauthor=R.+Garbaccioauthor=L.+Garvin-Queenauthor=D.+Gatelyauthor=P.+Gedaauthor=C.+Hainesauthor=S.-C.+Hseihauthor=D.+Hodgesauthor=J.+Kernauthor=N.+Knudsenauthor=K.+Kwasnjukauthor=L.+Liangauthor=H.+Maauthor=A.+Manibusanauthor=P.+L.+Millerauthor=L.+Y.+Moyauthor=Y.+Quauthor=S.+Shahauthor=J.+S.+Shinauthor=P.+Stiversauthor=Y.+Sunauthor=D.+Tomazelaauthor=H.+C.+Wooauthor=D.+Zallerauthor=S.+Zhangauthor=Y.+Zhangauthor=M.+Zielstorff&title=Development+of+anti-CD74+antibody-drug+conjugates+to+target+glucocorticoids+to+immune+cells&doi=10.1021%2Facs.bioconjchem.8b00312"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30aR"><div class="casContent"><span class="casTitleNuber">30a</span><div class="casTitle"><span class="NLM_cas:atitle">Development of anti-CD74 antibody-drug conjugates to target glucocorticoids to immune cells</span></div><div class="casAuthors">Brandish, Philip E.; Palmieri, Anthony; Antonenko, Svetlana; Beaumont, Maribel; Benso, Lia; Cancilla, Mark; Cheng, Mangeng; Fayadat-Dilman, Laurence; Feng, Guo; Figueroa, Isabel; Firdos, Juhi; Garbaccio, Robert; Garvin-Queen, Laura; Gately, Dennis; Geda, Prasanthi; Haines, Christopher; Hseih, SuChun; Hodges, Douglas; Kern, Jeffrey; Knudsen, Nickolas; Kwasnjuk, Kristen; Liang, Linda; Ma, Huiping; Manibusan, Anthony; Miller, Paul L.; Moy, Lily Y.; Qu, Yujie; Shah, Sanjiv; Shin, John S.; Stivers, Peter; Sun, Ying; Tomazela, Daniela; Woo, Hyun Chong; Zaller, Dennis; Zhang, Shuli; Zhang, Yiwei; Zielstorff, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Bioconjugate Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2357-2369</span>CODEN:
                <span class="NLM_cas:coden">BCCHES</span>;
        ISSN:<span class="NLM_cas:issn">1043-1802</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Glucocorticoids (GCs) are excellent anti-inflammatory drugs but are dose-limited by on-target toxicity.  We sought to solve this problem by delivering GCs to immune cells with antibody-drug conjugates (ADCs) using antibodies contg. site-specific incorporation of a non-natural amino acid, novel linker chem. for in vitro and in vivo stability, and existing and novel glucocorticoid receptor (GR) agonists as payloads.  We directed fluticasone propionate to human antigen-presenting immune cells to afford GR activation that was dependent on the targeted antigen.  However, mechanism of action studies pointed to accumulation of free payload in the tissue culture supernatant as the dominant driver of activity and indeed administration of the ADC to human CD74 transgenic mice failed to activate GR target genes in splenic B cells.  Suspecting dissipation of released payload, we designed an ADC bearing a novel GR agonist payload with reduced permeability which afforded cell-intrinsic activity in human B cells.  Our work shows that antibody-targeting offers significant potential for rescuing existing and new dose-limited drugs outside the field of oncol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqL223IFiibVrVg90H21EOLACvtfcHk0lgskLJDH8MZsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFGlsb7P&md5=16f03a31902538a233676f30cc1c26c7</span></div><a href="/servlet/linkout?suffix=cit30a&amp;dbid=16384&amp;doi=10.1021%2Facs.bioconjchem.8b00312&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.bioconjchem.8b00312%26sid%3Dliteratum%253Aachs%26aulast%3DBrandish%26aufirst%3DP.%2BE.%26aulast%3DPalmieri%26aufirst%3DA.%26aulast%3DAntonenko%26aufirst%3DS.%26aulast%3DBeaumont%26aufirst%3DM.%26aulast%3DBenso%26aufirst%3DL.%26aulast%3DCancilla%26aufirst%3DM.%26aulast%3DCheng%26aufirst%3DM.%26aulast%3DFayadat-Dilman%26aufirst%3DL.%26aulast%3DFeng%26aufirst%3DG.%26aulast%3DFigueroa%26aufirst%3DI.%26aulast%3DFirdos%26aufirst%3DJ.%26aulast%3DGarbaccio%26aufirst%3DR.%26aulast%3DGarvin-Queen%26aufirst%3DL.%26aulast%3DGately%26aufirst%3DD.%26aulast%3DGeda%26aufirst%3DP.%26aulast%3DHaines%26aufirst%3DC.%26aulast%3DHseih%26aufirst%3DS.-C.%26aulast%3DHodges%26aufirst%3DD.%26aulast%3DKern%26aufirst%3DJ.%26aulast%3DKnudsen%26aufirst%3DN.%26aulast%3DKwasnjuk%26aufirst%3DK.%26aulast%3DLiang%26aufirst%3DL.%26aulast%3DMa%26aufirst%3DH.%26aulast%3DManibusan%26aufirst%3DA.%26aulast%3DMiller%26aufirst%3DP.%2BL.%26aulast%3DMoy%26aufirst%3DL.%2BY.%26aulast%3DQu%26aufirst%3DY.%26aulast%3DShah%26aufirst%3DS.%26aulast%3DShin%26aufirst%3DJ.%2BS.%26aulast%3DStivers%26aufirst%3DP.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DTomazela%26aufirst%3DD.%26aulast%3DWoo%26aufirst%3DH.%2BC.%26aulast%3DZaller%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DZielstorff%26aufirst%3DM.%26atitle%3DDevelopment%2520of%2520anti-CD74%2520antibody-drug%2520conjugates%2520to%2520target%2520glucocorticoids%2520to%2520immune%2520cells%26jtitle%3DBioconjugate%2520Chem.%26date%3D2018%26volume%3D29%26spage%3D2357%26epage%3D2369%26doi%3D10.1021%2Facs.bioconjchem.8b00312" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit30b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Kern, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cancilla, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dooney, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwasnjuk, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beaumont, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Figueroa, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsieh, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomazela, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandish, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmieri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stivers, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geda, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bresson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Firdos, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gately, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knudsen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manibusan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garbaccio, R. M.</span></span> <span> </span><span class="NLM_article-title">Discovery of pyrophosphate diesters as tunable, soluble, and bioorthogonal linkers for site-specific antibody-drug conjugates</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">1430</span>– <span class="NLM_lpage">1445</span>, <span class="refDoi"> DOI: 10.1021/jacs.5b12547</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.5b12547" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC28XltFCgsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2016&pages=1430-1445&author=J.+C.+Kernauthor=M.+Cancillaauthor=D.+Dooneyauthor=K.+Kwasnjukauthor=R.+Zhangauthor=M.+Beaumontauthor=I.+Figueroaauthor=S.+C.+Hsiehauthor=L.+Liangauthor=D.+Tomazelaauthor=J.+Zhangauthor=P.+E.+Brandishauthor=A.+Palmieriauthor=P.+Stiversauthor=M.+Chengauthor=G.+Fengauthor=P.+Gedaauthor=S.+Shahauthor=A.+Beckauthor=D.+Bressonauthor=J.+Firdosauthor=D.+Gatelyauthor=N.+Knudsenauthor=A.+Manibusanauthor=P.+G.+Schultzauthor=Y.+Sunauthor=R.+M.+Garbaccio&title=Discovery+of+pyrophosphate+diesters+as+tunable%2C+soluble%2C+and+bioorthogonal+linkers+for+site-specific+antibody-drug+conjugates&doi=10.1021%2Fjacs.5b12547"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30bR"><div class="casContent"><span class="casTitleNuber">30b</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Pyrophosphate Diesters as Tunable, Soluble, and Bioorthogonal Linkers for Site-Specific Antibody-Drug Conjugates</span></div><div class="casAuthors">Kern, Jeffrey C.; Cancilla, Mark; Dooney, Deborah; Kwasnjuk, Kristen; Zhang, Rena; Beaumont, Maribel; Figueroa, Isabel; Hsieh, SuChun; Liang, Linda; Tomazela, Daniela; Zhang, Jeffrey; Brandish, Philip E.; Palmieri, Anthony; Stivers, Peter; Cheng, Mangeng; Feng, Guo; Geda, Prasanthi; Shah, Sanjiv; Beck, Andrew; Bresson, Damien; Firdos, Juhi; Gately, Dennis; Knudsen, Nick; Manibusan, Anthony; Schultz, Peter G.; Sun, Ying; Garbaccio, Robert M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1430-1445</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">As part of an effort to examine the utility of antibody-drug conjugates (ADCs) beyond oncol. indications, a novel pyrophosphate ester linker was discovered to enable the targeted delivery of glucocorticoids.  As small mols., these highly sol. phosphate ester drug linkers were found to have ideal orthogonal properties: robust plasma stability coupled with rapid release of payload in a lysosomal environment.  Building upon these findings, site-specific ADCs were made between this drug linker combination and an antibody against human CD70, a receptor specifically expressed in immune cells but also found aberrantly expressed in multiple human carcinomas.  Full characterization of these ADCs enabled procession to in vitro proof of concept, wherein ADCs 1-22 and 1-37 were demonstrated to afford potent, targeted delivery of glucocorticoids to a representative cell line, as measured by changes in glucocorticoid receptor-mediated gene mRNA levels.  These activities were found to be antibody-, linker-, and payload-dependent.  Preliminary mechanistic studies support the notion that lysosomal trafficking and enzymic linker cleavage are required for activity and that the utility for the pyrophosphate linker may be general for internalizing ADCs as well as other targeted delivery platforms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0or81gNuulrVg90H21EOLACvtfcHk0liwAuljkWBv3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XltFCgsw%253D%253D&md5=07aa87ec657d4506e63b35f73af61e78</span></div><a href="/servlet/linkout?suffix=cit30b&amp;dbid=16384&amp;doi=10.1021%2Fjacs.5b12547&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.5b12547%26sid%3Dliteratum%253Aachs%26aulast%3DKern%26aufirst%3DJ.%2BC.%26aulast%3DCancilla%26aufirst%3DM.%26aulast%3DDooney%26aufirst%3DD.%26aulast%3DKwasnjuk%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DBeaumont%26aufirst%3DM.%26aulast%3DFigueroa%26aufirst%3DI.%26aulast%3DHsieh%26aufirst%3DS.%2BC.%26aulast%3DLiang%26aufirst%3DL.%26aulast%3DTomazela%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DBrandish%26aufirst%3DP.%2BE.%26aulast%3DPalmieri%26aufirst%3DA.%26aulast%3DStivers%26aufirst%3DP.%26aulast%3DCheng%26aufirst%3DM.%26aulast%3DFeng%26aufirst%3DG.%26aulast%3DGeda%26aufirst%3DP.%26aulast%3DShah%26aufirst%3DS.%26aulast%3DBeck%26aufirst%3DA.%26aulast%3DBresson%26aufirst%3DD.%26aulast%3DFirdos%26aufirst%3DJ.%26aulast%3DGately%26aufirst%3DD.%26aulast%3DKnudsen%26aufirst%3DN.%26aulast%3DManibusan%26aufirst%3DA.%26aulast%3DSchultz%26aufirst%3DP.%2BG.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DGarbaccio%26aufirst%3DR.%2BM.%26atitle%3DDiscovery%2520of%2520pyrophosphate%2520diesters%2520as%2520tunable%252C%2520soluble%252C%2520and%2520bioorthogonal%2520linkers%2520for%2520site-specific%2520antibody-drug%2520conjugates%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2016%26volume%3D138%26spage%3D1430%26epage%3D1445%26doi%3D10.1021%2Fjacs.5b12547" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fourie-O’Donohue, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, P. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">dela Cruz Chuh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tchelepi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawyer, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pillow, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadowsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozak, K. R.</span></span> <span> </span><span class="NLM_article-title">Improved translation of stability for conjugated antibodies using an in vitro whole blood assay</span>. <i>mAbs</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">1715705</span> <span class="refDoi"> DOI: 10.1080/19420862.2020.1715705</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1080%2F19420862.2020.1715705" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=31997712" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvFSrt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2020&author=A.+Fourie-O%E2%80%99Donohueauthor=P.+Y.+Chuauthor=J.+dela+Cruz+Chuhauthor=R.+Tchelepiauthor=S.+P.+Tsaiauthor=J.+C.+Tranauthor=W.+S.+Sawyerauthor=D.+Suauthor=C.+Ngauthor=K.+Xuauthor=S.-F.+Yuauthor=T.+H.+Pillowauthor=J.+Sadowskyauthor=P.+S.+Dragovichauthor=Y.+Liuauthor=K.+R.+Kozak&title=Improved+translation+of+stability+for+conjugated+antibodies+using+an+in+vitro+whole+blood+assay&doi=10.1080%2F19420862.2020.1715705"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Improved translation of stability for conjugated antibodies using an in vitro whole blood assay</span></div><div class="casAuthors">Fourie-O'Donohue, Aimee; Chu, Phillip Y.; dela Cruz Chuh, Josefa; Tchelepi, Robert; Tsai, Siao Ping; Tran, John C.; Sawyer, William S.; Su, Dian; Ng, Carl; Xu, Keyang; Yu, Shang-Fan; Pillow, Thomas H.; Sadowsky, Jack; Dragovich, Peter S.; Liu, Yichin; Kozak, Katherine R.</div><div class="citationInfo"><span class="NLM_cas:title">mAbs</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1715705/1</span>CODEN:
                <span class="NLM_cas:coden">MABSCP</span>;
        ISSN:<span class="NLM_cas:issn">1942-0870</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis, Inc.</span>)
        </div><div class="casAbstract">For antibody-drug conjugates to be efficacious and safe, they must be stable in circulation to carry the payload to the site of the targeted cell.  Several components of a drug-conjugated antibody are known to influence stability: 1) the site of drug attachment on the antibody, 2) the linker used to attach the payload to the antibody, and 3) the payload itself.  In order to support the design and optimization of a high vol. of drug conjugates and avoid unstable conjugates prior to testing in animal models, we wanted to proactively identify these potential liabilities.  Therefore, we sought to establish an in vitro screening method that best correlated with in vivo stability.  While traditionally plasma has been used to assess in vitro stability, our evaluation using a variety of THIOMABTM antibody-drug conjugates revealed several disconnects between the stability assessed in vitro and the in vivo outcomes when using plasma.  When drug conjugates were incubated in vitro for 24 h in mouse whole blood rather than plasma and then analyzed by affinity capture LC-MS, we found an improved correlation to in vivo stability with whole blood (R2 = 0.87, coeff. of detn.) compared to unfrozen or frozen mouse plasma (R2 = 0.34, 0.01, resp.).  We further showed that this whole blood assay was also able to predict in vivo stability of other preclin. species such as rat and cynomolgus monkey, as well as in human.  The screening method utilized short (24 h) incubation times, as well as a custom anal. software, allowing increased throughput and in-depth biotransformation characterization.  While some instabilities that were more challenging to identify remain, the method greatly enhanced the process of screening, optimizing, and lead candidate selection, resulting in the substantial redn. of animal studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhca0BIA00H7Vg90H21EOLACvtfcHk0liwAuljkWBv3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvFSrt7g%253D&md5=c4b05b4fa7b3bf4d84955ec3e5e8dee5</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1080%2F19420862.2020.1715705&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F19420862.2020.1715705%26sid%3Dliteratum%253Aachs%26aulast%3DFourie-O%25E2%2580%2599Donohue%26aufirst%3DA.%26aulast%3DChu%26aufirst%3DP.%2BY.%26aulast%3Ddela%2BCruz%2BChuh%26aufirst%3DJ.%26aulast%3DTchelepi%26aufirst%3DR.%26aulast%3DTsai%26aufirst%3DS.%2BP.%26aulast%3DTran%26aufirst%3DJ.%2BC.%26aulast%3DSawyer%26aufirst%3DW.%2BS.%26aulast%3DSu%26aufirst%3DD.%26aulast%3DNg%26aufirst%3DC.%26aulast%3DXu%26aufirst%3DK.%26aulast%3DYu%26aufirst%3DS.-F.%26aulast%3DPillow%26aufirst%3DT.%2BH.%26aulast%3DSadowsky%26aufirst%3DJ.%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DKozak%26aufirst%3DK.%2BR.%26atitle%3DImproved%2520translation%2520of%2520stability%2520for%2520conjugated%2520antibodies%2520using%2520an%2520in%2520vitro%2520whole%2520blood%2520assay%26jtitle%3DmAbs%26date%3D2020%26volume%3D12%26doi%3D10.1080%2F19420862.2020.1715705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harbin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamman, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Differential PROTAC substrate specificity dictated by orientation of recruited E3 ligase</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>, <span class="NLM_elocation-id">131</span> <span class="refDoi"> DOI: 10.1038/s41467-018-08027-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1038%2Fs41467-018-08027-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=30631068" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A280%3ADC%252BB3cjgt1Sksg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&author=B.+E.+Smithauthor=S.+L.+Wangauthor=S.+Jaime-Figueroaauthor=A.+Harbinauthor=J.+Wangauthor=B.+D.+Hammanauthor=C.+M.+Crews&title=Differential+PROTAC+substrate+specificity+dictated+by+orientation+of+recruited+E3+ligase&doi=10.1038%2Fs41467-018-08027-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32aR"><div class="casContent"><span class="casTitleNuber">32a</span><div class="casTitle"><span class="NLM_cas:atitle">Differential PROTAC substrate specificity dictated by orientation of recruited E3 ligase</span></div><div class="casAuthors">Smith Blake E; Wang Stephen L; Jaime-Figueroa Saul; Crews Craig M; Harbin Alicia; Wang Jing; Hamman Brian D; Crews Craig M; Crews Craig M</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">131</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">PROteolysis-TArgeting Chimeras (PROTACs) are hetero-bifunctional molecules that recruit an E3 ubiquitin ligase to a given substrate protein resulting in its targeted degradation.  Many potent PROTACs with specificity for dissimilar targets have been developed; however, the factors governing degradation selectivity within closely-related protein families remain elusive.  Here, we generate isoform-selective PROTACs for the p38 MAPK family using a single warhead (foretinib) and recruited E3 ligase (von Hippel-Lindau).  Based on their distinct linker attachments and lengths, these two PROTACs differentially recruit VHL, resulting in degradation of p38α or p38δ.  We characterize the role of ternary complex formation in driving selectivity, showing that it is necessary, but insufficient, for PROTAC-induced substrate ubiquitination.  Lastly, we explore the p38δ:PROTAC:VHL complex to explain the different selectivity profiles of these PROTACs.  Our work attributes the selective degradation of two closely-related proteins using the same warhead and E3 ligase to heretofore underappreciated aspects of the ternary complex model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRYPr3dBOGbeuR_TJajKxJDfW6udTcc2ealNBXkXa7XC7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cjgt1Sksg%253D%253D&md5=bd56c51f5966709ca5d1ff3dadb7cc79</span></div><a href="/servlet/linkout?suffix=cit32a&amp;dbid=16384&amp;doi=10.1038%2Fs41467-018-08027-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-018-08027-7%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DB.%2BE.%26aulast%3DWang%26aufirst%3DS.%2BL.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DHarbin%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DDifferential%2520PROTAC%2520substrate%2520specificity%2520dictated%2520by%2520orientation%2520of%2520recruited%2520E3%2520ligase%26jtitle%3DNat.%2520Commun.%26date%3D2019%26volume%3D10%26doi%3D10.1038%2Fs41467-018-08027-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit32b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Farnaby, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koegl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roy, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitworth, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diers, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trainor, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zollman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steurer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karolyi-Oezguer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riedmueller, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmaschitz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wachter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dank, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galant, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharps, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumpel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traxler, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstberger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnitzer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petermann, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greb, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinstabl, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoephel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss-Puxbaum, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ehrenhöfer-Wölfer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wöhrle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehmelt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinnenthal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnhof, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiechens, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, M.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen-Hughes, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ettmayer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConnell, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">BAF complex vulnerabilities in cancer demonstrated via structure-based PROTAC design</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">672</span>– <span class="NLM_lpage">680</span>, <span class="refDoi"> DOI: 10.1038/s41589-019-0294-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1038%2Fs41589-019-0294-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=31178587" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFeqtLfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=672-680&author=W.+Farnabyauthor=M.+Koeglauthor=M.+J.+Royauthor=C.+Whitworthauthor=E.+Diersauthor=N.+Trainorauthor=D.+Zollmanauthor=S.+Steurerauthor=J.+Karolyi-Oezguerauthor=C.+Riedmuellerauthor=T.+Gmaschitzauthor=J.+Wachterauthor=C.+Dankauthor=M.+Galantauthor=B.+Sharpsauthor=K.+Rumpelauthor=E.+Traxlerauthor=T.+Gerstbergerauthor=R.+Schnitzerauthor=O.+Petermannauthor=P.+Grebauthor=H.+Weinstablauthor=G.+Baderauthor=A.+Zoephelauthor=A.+Weiss-Puxbaumauthor=K.+Ehrenh%C3%B6fer-W%C3%B6lferauthor=S.+W%C3%B6hrleauthor=G.+Boehmeltauthor=J.+Rinnenthalauthor=H.+Arnhofauthor=N.+Wiechensauthor=M.-Y.+Wuauthor=T.+Owen-Hughesauthor=P.+Ettmayerauthor=M.+Pearsonauthor=D.+B.+McConnellauthor=A.+Ciulli&title=BAF+complex+vulnerabilities+in+cancer+demonstrated+via+structure-based+PROTAC+design&doi=10.1038%2Fs41589-019-0294-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32bR"><div class="casContent"><span class="casTitleNuber">32b</span><div class="casTitle"><span class="NLM_cas:atitle">BAF complex vulnerabilities in cancer demonstrated via structure-based PROTAC design</span></div><div class="casAuthors">Farnaby, William; Koegl, Manfred; Roy, Michael J.; Whitworth, Claire; Diers, Emelyne; Trainor, Nicole; Zollman, David; Steurer, Steffen; Karolyi-Oezguer, Jale; Riedmueller, Carina; Gmaschitz, Teresa; Wachter, Johannes; Dank, Christian; Galant, Michael; Sharps, Bernadette; Rumpel, Klaus; Traxler, Elisabeth; Gerstberger, Thomas; Schnitzer, Renate; Petermann, Oliver; Greb, Peter; Weinstabl, Harald; Bader, Gerd; Zoephel, Andreas; Weiss-Puxbaum, Alexander; Ehrenhofer-Wolfer, Katharina; Wohrle, Simon; Boehmelt, Guido; Rinnenthal, Joerg; Arnhof, Heribert; Wiechens, Nicola; Wu, Meng-Ying; Owen-Hughes, Tom; Ettmayer, Peter; Pearson, Mark; McConnell, Darryl B.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">672-680</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Targeting subunits of BAF/PBAF chromatin remodeling complexes has been proposed as an approach to exploit cancer vulnerabilities.  Here, we develop proteolysis targeting chimera (PROTAC) degraders of the BAF ATPase subunits SMARCA2 and SMARCA4 using a bromodomain ligand and recruitment of the E3 ubiquitin ligase VHL.  High-resoln. ternary complex crystal structures and biophys. investigation guided rational and efficient optimization toward ACBI1(I), a potent and cooperative degrader of SMARCA2, SMARCA4 and PBRM1.  I induced anti-proliferative effects and cell death caused by SMARCA2 depletion in SMARCA4 mutant cancer cells, and in acute myeloid leukemia cells dependent on SMARCA4 ATPase activity.  These findings exemplify a successful biophysics- and structure-based PROTAC design approach to degrade high profile drug targets, and pave the way toward new therapeutics for the treatment of tumors sensitive to the loss of BAF complex ATPases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonEh4pux4Bs7Vg90H21EOLACvtfcHk0lhtTLIT9eOtAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFeqtLfJ&md5=e95fad12c727aedcf08aed9320dae905</span></div><a href="/servlet/linkout?suffix=cit32b&amp;dbid=16384&amp;doi=10.1038%2Fs41589-019-0294-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-019-0294-6%26sid%3Dliteratum%253Aachs%26aulast%3DFarnaby%26aufirst%3DW.%26aulast%3DKoegl%26aufirst%3DM.%26aulast%3DRoy%26aufirst%3DM.%2BJ.%26aulast%3DWhitworth%26aufirst%3DC.%26aulast%3DDiers%26aufirst%3DE.%26aulast%3DTrainor%26aufirst%3DN.%26aulast%3DZollman%26aufirst%3DD.%26aulast%3DSteurer%26aufirst%3DS.%26aulast%3DKarolyi-Oezguer%26aufirst%3DJ.%26aulast%3DRiedmueller%26aufirst%3DC.%26aulast%3DGmaschitz%26aufirst%3DT.%26aulast%3DWachter%26aufirst%3DJ.%26aulast%3DDank%26aufirst%3DC.%26aulast%3DGalant%26aufirst%3DM.%26aulast%3DSharps%26aufirst%3DB.%26aulast%3DRumpel%26aufirst%3DK.%26aulast%3DTraxler%26aufirst%3DE.%26aulast%3DGerstberger%26aufirst%3DT.%26aulast%3DSchnitzer%26aufirst%3DR.%26aulast%3DPetermann%26aufirst%3DO.%26aulast%3DGreb%26aufirst%3DP.%26aulast%3DWeinstabl%26aufirst%3DH.%26aulast%3DBader%26aufirst%3DG.%26aulast%3DZoephel%26aufirst%3DA.%26aulast%3DWeiss-Puxbaum%26aufirst%3DA.%26aulast%3DEhrenh%25C3%25B6fer-W%25C3%25B6lfer%26aufirst%3DK.%26aulast%3DW%25C3%25B6hrle%26aufirst%3DS.%26aulast%3DBoehmelt%26aufirst%3DG.%26aulast%3DRinnenthal%26aufirst%3DJ.%26aulast%3DArnhof%26aufirst%3DH.%26aulast%3DWiechens%26aufirst%3DN.%26aulast%3DWu%26aufirst%3DM.-Y.%26aulast%3DOwen-Hughes%26aufirst%3DT.%26aulast%3DEttmayer%26aufirst%3DP.%26aulast%3DPearson%26aufirst%3DM.%26aulast%3DMcConnell%26aufirst%3DD.%2BB.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DBAF%2520complex%2520vulnerabilities%2520in%2520cancer%2520demonstrated%2520via%2520structure-based%2520PROTAC%2520design%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D15%26spage%3D672%26epage%3D680%26doi%3D10.1038%2Fs41589-019-0294-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit32c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span>; <span class="NLM_string-name">Dong, H.</span>; <span class="NLM_string-name">Wang, J.</span>; <span class="NLM_string-name">Berlin, M.</span>; <span class="NLM_string-name">Siu, K.</span>; <span class="NLM_string-name">Crew, A. P.</span>; <span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Compounds and Methods for the Targeted Degradation of Bromodomain-containing Proteins</span>. International patent application, <span class="NLM_patent">WO2017030814</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=Y.+Qian&author=H.+Dong&author=J.+Wang&author=M.+Berlin&author=K.+Siu&author=A.+P.+Crew&author=C.+M.+Crews&title=Compounds+and+Methods+for+the+Targeted+Degradation+of+Bromodomain-containing+Proteins"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32c&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DQian%26aufirst%3DY.%26atitle%3DCompounds%2520and%2520Methods%2520for%2520the%2520Targeted%2520Degradation%2520of%2520Bromodomain-containing%2520Proteins%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit32d"><span><span class="NLM_label">(d) </span><span class="NLM_contrib-group"><span class="NLM_string-name">Jin, M.</span>; <span class="NLM_string-name">Crew, A.</span>; <span class="NLM_string-name">Dong, H.</span>; <span class="NLM_string-name">Wang, J.</span>; <span class="NLM_string-name">Siu, K.</span>; <span class="NLM_string-name">Ferraro, C.</span>; <span class="NLM_string-name">Chen, X.</span>; <span class="NLM_string-name">Qian, Y.</span></span> <span> </span><span class="NLM_article-title">Compounds and Methods for the Targeted Degradation of the Androgen Receptor</span>.. International patent application, <span class="NLM_patent">WO2016118666</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=M.+Jin&author=A.+Crew&author=H.+Dong&author=J.+Wang&author=K.+Siu&author=C.+Ferraro&author=X.+Chen&author=Y.+Qian&title=Compounds+and+Methods+for+the+Targeted+Degradation+of+the+Androgen+Receptor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32d&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DM.%26atitle%3DCompounds%2520and%2520Methods%2520for%2520the%2520Targeted%2520Degradation%2520of%2520the%2520Androgen%2520Receptor%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Su, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cosino, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">dela Cruz-Chuh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Rosario, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Figueroa, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamath, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaur, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozak, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leipold, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pillow, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polakis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polson, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowntree, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saad, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Safina, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stagg, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tercel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandlen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vollmar, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span></span> <span> </span><span class="NLM_article-title">Antibody-drug conjugates derived from cytotoxic seco-CBI-dimer payloads are highly efficacious in xenograft models and form protein adducts in vivo</span>. <i>Bioconjugate Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">1356</span>– <span class="NLM_lpage">1370</span>, <span class="refDoi"> DOI: 10.1021/acs.bioconjchem.9b00133</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.bioconjchem.9b00133" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmvFart74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2019&pages=1356-1370&author=D.+Suauthor=J.+Chenauthor=E.+Cosinoauthor=J.+dela+Cruz-Chuhauthor=H.+Davisauthor=G.+Del+Rosarioauthor=I.+Figueroaauthor=L.+Goonauthor=J.+Heauthor=A.+V.+Kamathauthor=S.+Kaurauthor=K.+R.+Kozakauthor=J.+Lauauthor=D.+Leeauthor=M.+V.+Leeauthor=D.+Leipoldauthor=L.+Liuauthor=P.+Liuauthor=G.-L.+Luauthor=C.+Nelsonauthor=C.+Ngauthor=T.+H.+Pillowauthor=P.+Polakisauthor=A.+G.+Polsonauthor=R.+K.+Rowntreeauthor=O.+Saadauthor=B.+Safinaauthor=N.+J.+Staggauthor=M.+Tercelauthor=R.+Vandlenauthor=B.+S.+Vollmarauthor=J.+Waiauthor=T.+Wangauthor=B.+Weiauthor=K.+Xuauthor=J.+Xueauthor=Z.+Xuauthor=G.+Yanauthor=H.+Yaoauthor=S.-F.+Yuauthor=D.+Zhangauthor=F.+Zhongauthor=P.+S.+Dragovich&title=Antibody-drug+conjugates+derived+from+cytotoxic+seco-CBI-dimer+payloads+are+highly+efficacious+in+xenograft+models+and+form+protein+adducts+in+vivo&doi=10.1021%2Facs.bioconjchem.9b00133"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33aR"><div class="casContent"><span class="casTitleNuber">33a</span><div class="casTitle"><span class="NLM_cas:atitle">Antibody-drug conjugates derived from cytotoxic seco-CBI-dimer payloads are highly efficacious in xenograft models and form protein adducts in vivo</span></div><div class="casAuthors">Su, Dian; Chen, Jinhua; Cosino, Ely; dela Cruz-Chuh, Josefa; Davis, Helen; Del Rosario, Geoffrey; Figueroa, Isabel; Goon, Leanne; He, Jintang; Kamath, Amrita V.; Kaur, Surinder; Kozak, Katherine R.; Lau, Jeffrey; Lee, Donna; Lee, M. Violet; Leipold, Douglas; Liu, Luna; Liu, Peter; Lu, Guo-Liang; Nelson, Chris; Ng, Carl; Pillow, Thomas H.; Polakis, Paul; Polson, Andrew G.; Rowntree, Rebecca K.; Saad, Ola; Safina, Brian; Stagg, Nicola J.; Tercel, Moana; Vandlen, Richard; Vollmar, Breanna S.; Wai, John; Wang, Tao; Wei, BinQing; Xu, Keyang; Xue, Juanjuan; Xu, Zijin; Yan, Gang; Yao, Hui; Yu, Shang-Fan; Zhang, Donglu; Zhong, Fiona; Dragovich, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Bioconjugate Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1356-1370</span>CODEN:
                <span class="NLM_cas:coden">BCCHES</span>;
        ISSN:<span class="NLM_cas:issn">1043-1802</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This work discloses the first examples of antibody-drug conjugates (ADCs) that are constructed from linker-drugs bearing dimeric seco-CBI payloads (duocarmycin analogs).  Several homogeneous, CD22-targeting THIOMAB antibody-drug conjugates (TDCs) contg. the dimeric seco-CBI entities are shown to be highly efficacious in the WSU-DLCL2 and BJAB mouse xenograft models.  Surprisingly, the seco-CBI-contg. conjugates are also obsd. to undergo significant biotransformation in vivo in mice, rats, and monkeys and thereby form 1:1 adducts with the Alpha-1-Microglobulin (A1M) plasma protein from these species.  Variation of both the payload mAb attachment site and length of the linker-drug is shown to alter the rates of adduct formation.  Subsequent expts. demonstrated that adduct formation attenuates the in vitro antiproliferation activity of the affected seco-CBI-dimer TDCs, but does not significantly impact the in vivo efficacy of the conjugates.  In vitro assays employing phosphatase-treated whole blood suggest that A1M adduct formation is likely to occur if the seco-CBI-dimer TDCs are administered to humans.  Importantly, protein adduct formation leads to the underestimation of total antibody (Tab) concns. using an ELISA assay but does not affect Tab values detd. via an orthogonal LC-MS/MS method.  Several recommendations regarding bioanal. of future in vivo studies involving related seco-CBI-contg. ADCs are provided based on these collective findings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKI29fSnevNLVg90H21EOLACvtfcHk0ljOSv3CDBuvFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmvFart74%253D&md5=3072806260862be9d2578a2bbd4aebc6</span></div><a href="/servlet/linkout?suffix=cit33a&amp;dbid=16384&amp;doi=10.1021%2Facs.bioconjchem.9b00133&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.bioconjchem.9b00133%26sid%3Dliteratum%253Aachs%26aulast%3DSu%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DCosino%26aufirst%3DE.%26aulast%3Ddela%2BCruz-Chuh%26aufirst%3DJ.%26aulast%3DDavis%26aufirst%3DH.%26aulast%3DDel%2BRosario%26aufirst%3DG.%26aulast%3DFigueroa%26aufirst%3DI.%26aulast%3DGoon%26aufirst%3DL.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DKamath%26aufirst%3DA.%2BV.%26aulast%3DKaur%26aufirst%3DS.%26aulast%3DKozak%26aufirst%3DK.%2BR.%26aulast%3DLau%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DD.%26aulast%3DLee%26aufirst%3DM.%2BV.%26aulast%3DLeipold%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DLu%26aufirst%3DG.-L.%26aulast%3DNelson%26aufirst%3DC.%26aulast%3DNg%26aufirst%3DC.%26aulast%3DPillow%26aufirst%3DT.%2BH.%26aulast%3DPolakis%26aufirst%3DP.%26aulast%3DPolson%26aufirst%3DA.%2BG.%26aulast%3DRowntree%26aufirst%3DR.%2BK.%26aulast%3DSaad%26aufirst%3DO.%26aulast%3DSafina%26aufirst%3DB.%26aulast%3DStagg%26aufirst%3DN.%2BJ.%26aulast%3DTercel%26aufirst%3DM.%26aulast%3DVandlen%26aufirst%3DR.%26aulast%3DVollmar%26aufirst%3DB.%2BS.%26aulast%3DWai%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DWei%26aufirst%3DB.%26aulast%3DXu%26aufirst%3DK.%26aulast%3DXue%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DYan%26aufirst%3DG.%26aulast%3DYao%26aufirst%3DH.%26aulast%3DYu%26aufirst%3DS.-F.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DZhong%26aufirst%3DF.%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26atitle%3DAntibody-drug%2520conjugates%2520derived%2520from%2520cytotoxic%2520seco-CBI-dimer%2520payloads%2520are%2520highly%2520efficacious%2520in%2520xenograft%2520models%2520and%2520form%2520protein%2520adducts%2520in%2520vivo%26jtitle%3DBioconjugate%2520Chem.%26date%3D2019%26volume%3D30%26spage%3D1356%26epage%3D1370%26doi%3D10.1021%2Facs.bioconjchem.9b00133" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit33b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunzner-Toste, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nonomiya, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozak, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leipold, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamath, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eigenbrot, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinbacher, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohri, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raab, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staben, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flygare, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pillow, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masterson, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Safina, B.</span></span> <span> </span><span class="NLM_article-title">Discovery of peptidomimetic antibody-drug conjugate linkers with enhanced protease specificity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">989</span>– <span class="NLM_lpage">1000</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFKitrbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=989-1000&author=B.+Weiauthor=J.+Gunzner-Tosteauthor=H.+Yaoauthor=T.+Wangauthor=J.+Wangauthor=Z.+Xuauthor=J.+Chenauthor=J.+Waiauthor=J.+Nonomiyaauthor=S.+P.+Tsaiauthor=J.+Chuhauthor=K.+R.+Kozakauthor=Y.+Liuauthor=S.-F.+Yuauthor=J.+Lauauthor=G.+Liauthor=G.+D.+Phillipsauthor=D.+Leipoldauthor=A.+Kamathauthor=D.+Suauthor=K.+Xuauthor=C.+Eigenbrotauthor=S.+Steinbacherauthor=R.+Ohriauthor=H.+Raabauthor=L.+R.+Stabenauthor=G.+Zhaoauthor=J.+A.+Flygareauthor=T.+H.+Pillowauthor=V.+Vermaauthor=L.+A.+Mastersonauthor=P.+W.+Howardauthor=B.+Safina&title=Discovery+of+peptidomimetic+antibody-drug+conjugate+linkers+with+enhanced+protease+specificity&doi=10.1021%2Facs.jmedchem.7b01430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33bR"><div class="casContent"><span class="casTitleNuber">33b</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Peptidomimetic Antibody-Drug Conjugate Linkers with Enhanced Protease Specificity</span></div><div class="casAuthors">Wei, BinQing; Gunzner-Toste, Janet; Yao, Hui; Wang, Tao; Wang, Jing; Xu, Zijin; Chen, Jinhua; Wai, John; Nonomiya, Jim; Tsai, Siao Ping; Chuh, Josefa; Kozak, Katherine R.; Liu, Yichin; Yu, Shang-Fan; Lau, Jeff; Li, Guangmin; Phillips, Gail D.; Leipold, Doug; Kamath, Amrita; Su, Dian; Xu, Keyang; Eigenbrot, Charles; Steinbacher, Stefan; Ohri, Rachana; Raab, Helga; Staben, Leanna R.; Zhao, Guiling; Flygare, John A.; Pillow, Thomas H.; Verma, Vishal; Safina, Brian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">989-1000</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Antibody-drug conjugates (ADCs) have become an important therapeutic modality for oncol., with three approved by the FDA and over 60 others in clin. trials.  Despite the progress, improvements in ADC therapeutic index are desired.  Peptide-based ADC linkers that are cleaved by lysosomal proteases have shown sufficient stability in serum and effective payload-release in targeted cells.  If the linker can be preferentially hydrolyzed by tumor-specific proteases, safety margin may improve.  However, the use of peptide-based linkers limits our ability to modulate protease specificity.  Here we report the structure-guided discovery of novel, nonpeptidic ADC linkers.  We show that a cyclobutane-1,1-dicarboxamide-contg. linker is hydrolyzed predominantly by cathepsin B while the valine-citrulline dipeptide linker is not.  ADCs bearing the nonpeptidic linker are as efficacious and stable in vivo as those with the dipeptide linker.  Our results strongly support the application of the peptidomimetic linker and present new opportunities for improving the selectivity of ADCs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohInS8FeMmxbVg90H21EOLACvtfcHk0ljOSv3CDBuvFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFKitrbI&md5=8646f801d6292eaa2c3e2cc1b9e2e6cc</span></div><a href="/servlet/linkout?suffix=cit33b&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01430%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DB.%26aulast%3DGunzner-Toste%26aufirst%3DJ.%26aulast%3DYao%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DWai%26aufirst%3DJ.%26aulast%3DNonomiya%26aufirst%3DJ.%26aulast%3DTsai%26aufirst%3DS.%2BP.%26aulast%3DChuh%26aufirst%3DJ.%26aulast%3DKozak%26aufirst%3DK.%2BR.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DS.-F.%26aulast%3DLau%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DPhillips%26aufirst%3DG.%2BD.%26aulast%3DLeipold%26aufirst%3DD.%26aulast%3DKamath%26aufirst%3DA.%26aulast%3DSu%26aufirst%3DD.%26aulast%3DXu%26aufirst%3DK.%26aulast%3DEigenbrot%26aufirst%3DC.%26aulast%3DSteinbacher%26aufirst%3DS.%26aulast%3DOhri%26aufirst%3DR.%26aulast%3DRaab%26aufirst%3DH.%26aulast%3DStaben%26aufirst%3DL.%2BR.%26aulast%3DZhao%26aufirst%3DG.%26aulast%3DFlygare%26aufirst%3DJ.%2BA.%26aulast%3DPillow%26aufirst%3DT.%2BH.%26aulast%3DVerma%26aufirst%3DV.%26aulast%3DMasterson%26aufirst%3DL.%2BA.%26aulast%3DHoward%26aufirst%3DP.%2BW.%26aulast%3DSafina%26aufirst%3DB.%26atitle%3DDiscovery%2520of%2520peptidomimetic%2520antibody-drug%2520conjugate%2520linkers%2520with%2520enhanced%2520protease%2520specificity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D989%26epage%3D1000%26doi%3D10.1021%2Facs.jmedchem.7b01430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">dela Cruz-Chuh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bobba, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staben, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozak, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, G. D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vollmar, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadowsky, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandlen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khojasteh, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hop, C. E. C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pillow, T. H.</span></span> <span> </span><span class="NLM_article-title">Immolation of <i>p</i>-aminobenzyl ether linker and payload potency and stability determine the cell-killing activity of antibody-drug conjugates with phenol-containing payloads</span>. <i>Bioconjugate Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">267</span>– <span class="NLM_lpage">274</span>, <span class="refDoi"> DOI: 10.1021/acs.bioconjchem.7b00576</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.bioconjchem.7b00576" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1yrs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2018&pages=267-274&author=D.+Zhangauthor=H.+Leauthor=J.+dela+Cruz-Chuhauthor=S.+Bobbaauthor=J.+Guoauthor=L.+Stabenauthor=C.+Zhangauthor=Y.+Maauthor=K.+R.+Kozakauthor=G.+D.+L.+Phillipsauthor=B.+S.+Vollmarauthor=J.+D.+Sadowskyauthor=R.+Vandlenauthor=B.+Weiauthor=D.+Suauthor=P.+Fanauthor=P.+S.+Dragovichauthor=S.+C.+Khojastehauthor=C.+E.+C.+A.+Hopauthor=T.+H.+Pillow&title=Immolation+of+p-aminobenzyl+ether+linker+and+payload+potency+and+stability+determine+the+cell-killing+activity+of+antibody-drug+conjugates+with+phenol-containing+payloads&doi=10.1021%2Facs.bioconjchem.7b00576"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34aR"><div class="casContent"><span class="casTitleNuber">34a</span><div class="casTitle"><span class="NLM_cas:atitle">Immolation of p-Aminobenzyl Ether Linker and Payload Potency and Stability Determine the Cell-Killing Activity of Antibody-Drug Conjugates with Phenol-Containing Payloads</span></div><div class="casAuthors">Zhang, Donglu; Le, Hoa; Cruz-Chuh, Josefa dela; Bobba, Sudheer; Guo, Jun; Staben, Leanna; Zhang, Chenghong; Ma, Yong; Kozak, Katherine R.; Lewis Phillips, Gail D.; Vollmar, Breanna S.; Sadowsky, Jack D.; Vandlen, Richard; Wei, BinQing; Su, Dian; Fan, Peter; Dragovich, Peter S.; Khojasteh, S. Cyrus; Hop, Cornelis E. C. A.; Pillow, Thomas H.</div><div class="citationInfo"><span class="NLM_cas:title">Bioconjugate Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">267-274</span>CODEN:
                <span class="NLM_cas:coden">BCCHES</span>;
        ISSN:<span class="NLM_cas:issn">1043-1802</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The valine-citrulline (Val-Cit) dipeptide and p-aminobenzyl (PAB) spacer have been commonly used as a cleavable self-immolating linker in ADC design including in the clin. approved ADC, brentuximab vedotin (Adcetris).  When the same linker was used to connect to the phenol of the cyclopropabenzindolone (CBI) (P1), the resulting ADC1 showed loss of potency in CD22 target-expressing cancer cell lines (e.g., BJAB, WSU-DLCL2).  In comparison, the conjugate (ADC2) of a cyclopropapyrroloindolone (CPI) (P2) was potent despite the two corresponding free drugs having similar picomolar cell-killing activity.  Although the corresponding spirocyclization products of P1 and P2, responsible for DNA alkylation, are a prominent component in buffer, the linker immolation was slow when the PAB was connected as an ether (PABE) to the phenol in P1 compared to that in P2.  Addnl. immolation studies with two other PABE-linked substituted phenol compds. showed that electron-withdrawing groups accelerated the immolation to release an acidic phenol-contg. payload (to delocalize the neg. charge on the anticipated anionic phenol oxygen during immolation).  In contrast, efficient immolation of LD4 did not result in an active ADC4 because the payload (P4) had a low potency to kill cells.  In addn., nonimmolation of LD5 did not affect the cell-killing potency of its ADC5 since immolation is not required for DNA alkylation by the center-linked pyrrolobenzodiazepine.  Therefore, careful evaluation needs to be conducted when the Val-Cit-PAB linker is used to connect antibodies to a phenol-contg. drug as the linker immolation, as well as payload potency and stability, affects the cell-killing activity of an ADC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrw2sCnqYYrJbVg90H21EOLACvtfcHk0lj5Kz0IteHDmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1yrs7g%253D&md5=8ff229fed0a454b3d38d7f2a045b1209</span></div><a href="/servlet/linkout?suffix=cit34a&amp;dbid=16384&amp;doi=10.1021%2Facs.bioconjchem.7b00576&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.bioconjchem.7b00576%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DLe%26aufirst%3DH.%26aulast%3Ddela%2BCruz-Chuh%26aufirst%3DJ.%26aulast%3DBobba%26aufirst%3DS.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DStaben%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DKozak%26aufirst%3DK.%2BR.%26aulast%3DPhillips%26aufirst%3DG.%2BD.%2BL.%26aulast%3DVollmar%26aufirst%3DB.%2BS.%26aulast%3DSadowsky%26aufirst%3DJ.%2BD.%26aulast%3DVandlen%26aufirst%3DR.%26aulast%3DWei%26aufirst%3DB.%26aulast%3DSu%26aufirst%3DD.%26aulast%3DFan%26aufirst%3DP.%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26aulast%3DKhojasteh%26aufirst%3DS.%2BC.%26aulast%3DHop%26aufirst%3DC.%2BE.%2BC.%2BA.%26aulast%3DPillow%26aufirst%3DT.%2BH.%26atitle%3DImmolation%2520of%2520p-aminobenzyl%2520ether%2520linker%2520and%2520payload%2520potency%2520and%2520stability%2520determine%2520the%2520cell-killing%2520activity%2520of%2520antibody-drug%2520conjugates%2520with%2520phenol-containing%2520payloads%26jtitle%3DBioconjugate%2520Chem.%26date%3D2018%26volume%3D29%26spage%3D267%26epage%3D274%26doi%3D10.1021%2Facs.bioconjchem.7b00576" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit34b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, S. T.</span></span> <span> </span><span class="NLM_article-title">A self-immolative spacer that enables tunable controlled release of phenols under neutral conditions</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">4363</span>– <span class="NLM_lpage">4374</span>, <span class="refDoi"> DOI: 10.1021/jo300400q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo300400q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC38Xltl2hu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2012&pages=4363-4374&author=K.+M.+Schmidauthor=L.+Jensenauthor=S.+T.+Phillips&title=A+self-immolative+spacer+that+enables+tunable+controlled+release+of+phenols+under+neutral+conditions&doi=10.1021%2Fjo300400q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34bR"><div class="casContent"><span class="casTitleNuber">34b</span><div class="casTitle"><span class="NLM_cas:atitle">A Self-Immolative Spacer That Enables Tunable Controlled Release of Phenols under Neutral Conditions</span></div><div class="casAuthors">Schmid, Kyle M.; Jensen, Lasse; Phillips, Scott T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4363-4374</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A current challenge in the area of responsive materials is the design of reagents and polymers that provide controlled release of phenols in environments that are less polar than water.  In these contexts, a mol. strategy that enables release of nearly any phenol with predictable and tunable rates and without complication from background hydrolysis would substantially increase the precision with which materials can be designed to respond to a particular signal.  This Article addresses this problem at the fundamental level by describing the design, synthesis, and phys.-org. characterization of two small mol. self-immolative spacers that are capable of releasing phenols in org. and mixed org.-aq. solns.  The rate of release from these small mol. model systems is predictable and tunable, such that nearly any type of phenol, regardless of pKa value, can be released in neutral solns. without complications from nonspecific background release due to hydrolysis.  Furthermore, the release properties of the spacers can be predicted from bond length and conformation data (obtained from crystal structures).  On the basis of these results, it should now be possible to incorporate these design elements into materials to enable precise response properties in environments that are not 100% aq.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrc9lFN689qnrVg90H21EOLACvtfcHk0lj5Kz0IteHDmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xltl2hu70%253D&md5=b0bb0447e0dccf2d26bb12b2c08ed596</span></div><a href="/servlet/linkout?suffix=cit34b&amp;dbid=16384&amp;doi=10.1021%2Fjo300400q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo300400q%26sid%3Dliteratum%253Aachs%26aulast%3DSchmid%26aufirst%3DK.%2BM.%26aulast%3DJensen%26aufirst%3DL.%26aulast%3DPhillips%26aufirst%3DS.%2BT.%26atitle%3DA%2520self-immolative%2520spacer%2520that%2520enables%2520tunable%2520controlled%2520release%2520of%2520phenols%2520under%2520neutral%2520conditions%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2012%26volume%3D77%26spage%3D4363%26epage%3D4374%26doi%3D10.1021%2Fjo300400q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berthon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raffoux, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vey, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomez-Roca, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taussig, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roumier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herait, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahatt, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quesnel, B.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">OTX015 in patients with acute leukaemia: A dose-escalation, phase 1 study</span>. <i>Lancet Haematol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">e186</span>– <span class="NLM_lpage">e195</span>, <span class="refDoi"> DOI: 10.1016/S2352-3026(15)00247-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1016%2FS2352-3026%2815%2900247-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=27063977" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2016&pages=e186-e195&author=C.+Berthonauthor=E.+Raffouxauthor=X.+Thomasauthor=N.+Veyauthor=C.+Gomez-Rocaauthor=K.+Yeeauthor=D.+C.+Taussigauthor=K.+Rezaiauthor=C.+Roumierauthor=P.+Heraitauthor=C.+Kahattauthor=B.+Quesnel&title=OTX015+in+patients+with+acute+leukaemia%3A+A+dose-escalation%2C+phase+1+study&doi=10.1016%2FS2352-3026%2815%2900247-1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2FS2352-3026%2815%2900247-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS2352-3026%252815%252900247-1%26sid%3Dliteratum%253Aachs%26aulast%3DBerthon%26aufirst%3DC.%26aulast%3DRaffoux%26aufirst%3DE.%26aulast%3DThomas%26aufirst%3DX.%26aulast%3DVey%26aufirst%3DN.%26aulast%3DGomez-Roca%26aufirst%3DC.%26aulast%3DYee%26aufirst%3DK.%26aulast%3DTaussig%26aufirst%3DD.%2BC.%26aulast%3DRezai%26aufirst%3DK.%26aulast%3DRoumier%26aufirst%3DC.%26aulast%3DHerait%26aufirst%3DP.%26aulast%3DKahatt%26aufirst%3DC.%26aulast%3DQuesnel%26aufirst%3DB.%26atitle%3DOTX015%2520in%2520patients%2520with%2520acute%2520leukaemia%253A%2520A%2520dose-escalation%252C%2520phase%25201%2520study%26jtitle%3DLancet%2520Haematol.%26date%3D2016%26volume%3D3%26spage%3De186%26epage%3De195%26doi%3D10.1016%2FS2352-3026%2815%2900247-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">del Rosario, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leong, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, G. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vij, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, W.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chalouni, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadowsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hendricks, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Achilles Poon, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pillow, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schutten, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flygare, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polson, A. G.</span></span> <span> </span><span class="NLM_article-title">An anti-CLL1 antibody-drug conjugate for the treatment of acute myeloid leukemia</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1358</span>– <span class="NLM_lpage">1368</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-18-0333</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1158%2F1078-0432.CCR-18-0333" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=29959143" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtlOmt7bE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2019&pages=1358-1368&author=B.+Zhengauthor=S.-F.+Yuauthor=G.+del+Rosarioauthor=S.+R.+Leongauthor=G.+Y.+Leeauthor=R.+Vijauthor=C.+Chiuauthor=W.-C.+Liangauthor=Y.+Wuauthor=C.+Chalouniauthor=J.+Sadowskyauthor=V.+Clarkauthor=A.+Hendricksauthor=K.+Achilles+Poonauthor=W.+Chuauthor=T.+Pillowauthor=M.+M.+Schuttenauthor=J.+Flygareauthor=A.+G.+Polson&title=An+anti-CLL1+antibody-drug+conjugate+for+the+treatment+of+acute+myeloid+leukemia&doi=10.1158%2F1078-0432.CCR-18-0333"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">An anti-CLL-I antibody-drug conjugate for the treatment of acute myeloid leukemia</span></div><div class="casAuthors">Zheng, Bing; Yu, Shang-Fan; del Rosario, Geoffrey; Leong, Steven R.; Lee, Genee Y.; Vij, Rajesh; Chiu, Cecilia; Liang, Wei-Ching; Wu, Yan; Chalouni, Cecile; Sadowsky, Jack; Clark, Vanessa; Hendricks, Angela; Poon, Kirsten Achilles; Chu, Wayne; Pillow, Thomas; Schutten, Melissa M.; Flygare, John; Polson, Andrew G.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1358</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: The treatment of acute myeloid leukemia (AML) has not significantly changed in 40 years.  Cytarabine- and anthracycline-based chemotherapy induction regimens (7 + 3) remain the std. of care, and most patients have poor long-term survival.  The reapproval of Mylotarg, an anti-CD33-calicheamicin antibody-drug conjugate (ADC), has demonstrated ADCs as a clin. validated option to enhance the effectiveness of induction therapy.  We are interested in developing a next-generation ADC for AML to improve upon the initial success of Mylotarg.  Exptl. Design: The expression pattern of CLL-1 and its hematopoietic potential were investigated.  A novel anti-CLL-1-ADC, with a highly potent pyrrolobenzodiazepine (PBD) dimer conjugated through a self-immolative disulfide linker, was developed.  The efficacy and safety profiles of this ADC were evaluated in mouse xenograft models and in cynomolgus monkeys.  Results: We demonstrate that CLL-1 shares similar prevalence and trafficking properties that make CD33 an excellent ADC target for AML, but lacks expression on hematopoietic stem cells that hampers current CD33-targeted ADCs.  Our anti-CLL-1-ADC is highly effective at depleting tumor cells in AML xenograft models and lacks target independent toxicities at doses that depleted target monocytes and neutrophils in cynomolgus monkeys.  Conclusions: Collectively, our data suggest that an anti-CLL-1-ADC has the potential to become an effective and safer treatment for AML in humans, by reducing and allowing for faster recovery from initial cytopenias than the current generation of ADCs for AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJdzTurW_r4LVg90H21EOLACvtfcHk0lgBrcif0b84UQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtlOmt7bE&md5=c701a514d19baec87815539f0eaa9275</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-18-0333&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-18-0333%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DB.%26aulast%3DYu%26aufirst%3DS.-F.%26aulast%3Ddel%2BRosario%26aufirst%3DG.%26aulast%3DLeong%26aufirst%3DS.%2BR.%26aulast%3DLee%26aufirst%3DG.%2BY.%26aulast%3DVij%26aufirst%3DR.%26aulast%3DChiu%26aufirst%3DC.%26aulast%3DLiang%26aufirst%3DW.-C.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DChalouni%26aufirst%3DC.%26aulast%3DSadowsky%26aufirst%3DJ.%26aulast%3DClark%26aufirst%3DV.%26aulast%3DHendricks%26aufirst%3DA.%26aulast%3DAchilles%2BPoon%26aufirst%3DK.%26aulast%3DChu%26aufirst%3DW.%26aulast%3DPillow%26aufirst%3DT.%26aulast%3DSchutten%26aufirst%3DM.%2BM.%26aulast%3DFlygare%26aufirst%3DJ.%26aulast%3DPolson%26aufirst%3DA.%2BG.%26atitle%3DAn%2520anti-CLL1%2520antibody-drug%2520conjugate%2520for%2520the%2520treatment%2520of%2520acute%2520myeloid%2520leukemia%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2019%26volume%3D25%26spage%3D1358%26epage%3D1368%26doi%3D10.1158%2F1078-0432.CCR-18-0333" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leipold, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Figueroa, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masih, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Latifi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yip, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, B.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dere, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carrasco-Triguero, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saad, O. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vuillemenot, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laing, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schutten, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozak, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polson, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamath, A. V.</span></span> <span> </span><span class="NLM_article-title">Preclinical pharmacokinetics and pharmacodynamics of DCLL9718A: An antibody-drug conjugate for the treatment of acute myeloid leukemia</span>. <i>mAbs</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1312</span>– <span class="NLM_lpage">1321</span>, <span class="refDoi"> DOI: 10.1080/19420862.2018.1517565</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1080%2F19420862.2018.1517565" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=30183491" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVGjtLjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=1312-1321&author=D.+D.+Leipoldauthor=I.+Figueroaauthor=S.+Masihauthor=B.+Latifiauthor=V.+Yipauthor=B.-Q.+Shenauthor=R.+C.+Dereauthor=M.+Carrasco-Trigueroauthor=M.+V.+Leeauthor=O.+M.+Saadauthor=L.+Liuauthor=J.+Heauthor=D.+Suauthor=K.+Xuauthor=B.+R.+Vuillemenotauthor=S.+T.+Laingauthor=M.+Schuttenauthor=K.+R.+Kozakauthor=B.+Zhengauthor=A.+G.+Polsonauthor=A.+V.+Kamath&title=Preclinical+pharmacokinetics+and+pharmacodynamics+of+DCLL9718A%3A+An+antibody-drug+conjugate+for+the+treatment+of+acute+myeloid+leukemia&doi=10.1080%2F19420862.2018.1517565"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Preclinical pharmacokinetics and pharmacodynamics of DCLL9718A: An antibody-drug conjugate for the treatment of acute myeloid leukemia</span></div><div class="casAuthors">Leipold, Douglas D.; Figueroa, Isabel; Masih, Shabkhaiz; Latifi, Brandon; Yip, Victor; Shen, Ben-Quan; Dere, Randall C.; Carrasco-Triguero, Montserrat; Lee, M. Violet; Saad, Ola M.; Liu, Luna; He, Jintang; Su, Dian; Xu, Keyang; Vuillemenot, Brian R.; Laing, Steven T.; Schutten, Melissa; Kozak, Katherine R.; Zheng, Bing; Polson, Andrew G.; Kamath, Amrita V.</div><div class="citationInfo"><span class="NLM_cas:title">mAbs</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1312-1321</span>CODEN:
                <span class="NLM_cas:coden">MABSCP</span>;
        ISSN:<span class="NLM_cas:issn">1942-0870</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis, Inc.</span>)
        </div><div class="casAbstract">Few treatment options are available for acute myeloid leukemia (AML) patients.  DCLL9718A is an antibody-drug conjugate that targets C-type lectin-like mol.-1 (CLL-1).  Here, we characterize the in vitro and in vivo stability, the pharmacokinetics (PK) and pharmacodynamics (PD) of DCLL9718A and MCLL0517A in rodents and cynomolgus monkeys.  Three key PK analytes were measured in these studies: total antibody, antibody-conjugated PBD dimer and unconjugated PBD dimer.  In vitro, DCLL9718A, was stable with most (> 80%) of the PBD dimer payload remaining conjugated to the antibody over 96 h.  This was recapitulated in vivo with antibody-conjugated PBD dimer clearance ests. similar to DCLL9718A total antibody clearance.  Both DCLL9718A and MCLL0517A showed linear PK in the non-binding rodent species, and non-linear PK in cynomolgus monkeys, a binding species.  The PK data indicated minimal impact of conjugation on the disposition of DCLL9718A total antibody.  Finally, in cynomolgus monkey, MCLL0517A showed target engagement at all doses tested (0.5 and 20 mg/kg) as measured by receptor occupancy, and DCLL9718A (at doses of 0.05, 0.1 and 0.2 mg/kg) showed strong PD activity as evidenced by notable redn. in monocytes and neutrophils.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsFXdb4ZZBT7Vg90H21EOLACvtfcHk0lgBrcif0b84UQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVGjtLjN&md5=d1370e943ab7a89e07a60f56b624785b</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1080%2F19420862.2018.1517565&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F19420862.2018.1517565%26sid%3Dliteratum%253Aachs%26aulast%3DLeipold%26aufirst%3DD.%2BD.%26aulast%3DFigueroa%26aufirst%3DI.%26aulast%3DMasih%26aufirst%3DS.%26aulast%3DLatifi%26aufirst%3DB.%26aulast%3DYip%26aufirst%3DV.%26aulast%3DShen%26aufirst%3DB.-Q.%26aulast%3DDere%26aufirst%3DR.%2BC.%26aulast%3DCarrasco-Triguero%26aufirst%3DM.%26aulast%3DLee%26aufirst%3DM.%2BV.%26aulast%3DSaad%26aufirst%3DO.%2BM.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DSu%26aufirst%3DD.%26aulast%3DXu%26aufirst%3DK.%26aulast%3DVuillemenot%26aufirst%3DB.%2BR.%26aulast%3DLaing%26aufirst%3DS.%2BT.%26aulast%3DSchutten%26aufirst%3DM.%26aulast%3DKozak%26aufirst%3DK.%2BR.%26aulast%3DZheng%26aufirst%3DB.%26aulast%3DPolson%26aufirst%3DA.%2BG.%26aulast%3DKamath%26aufirst%3DA.%2BV.%26atitle%3DPreclinical%2520pharmacokinetics%2520and%2520pharmacodynamics%2520of%2520DCLL9718A%253A%2520An%2520antibody-drug%2520conjugate%2520for%2520the%2520treatment%2520of%2520acute%2520myeloid%2520leukemia%26jtitle%3DmAbs%26date%3D2018%26volume%3D10%26spage%3D1312%26epage%3D1321%26doi%3D10.1080%2F19420862.2018.1517565" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thummuri, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, G.</span></span> <span> </span><span class="NLM_article-title">Discovery of PROTAC BCL-X<sub>L</sub> degraders as potent anticancer agents with low on-target platelet toxicity</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>192</i></span>, <span class="NLM_elocation-id">112186</span> <span class="refDoi"> DOI: 10.1016/j.ejmech.2020.112186</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1016%2Fj.ejmech.2020.112186" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=32145645" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BB3cXksFarur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=192&publication_year=2020&author=X.+Zhangauthor=D.+Thummuriauthor=X.+Liuauthor=W.+Huauthor=P.+Zhangauthor=S.+Khanauthor=Y.+Yuanauthor=D.+Zhouauthor=G.+Zheng&title=Discovery+of+PROTAC+BCL-XL+degraders+as+potent+anticancer+agents+with+low+on-target+platelet+toxicity&doi=10.1016%2Fj.ejmech.2020.112186"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38aR"><div class="casContent"><span class="casTitleNuber">38a</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of PROTAC BCL-XL degraders as potent anticancer agents with low on-target platelet toxicity</span></div><div class="casAuthors">Zhang, Xuan; Thummuri, Dinesh; Liu, Xingui; Hu, Wanyi; Zhang, Peiyi; Khan, Sajid; Yuan, Yaxia; Zhou, Daohong; Zheng, Guangrong</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">192</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112186</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Anti-apoptotic protein BCL-XL plays a key role in tumorigenesis and cancer chemotherapy resistance, rendering it an attractive target for cancer treatment.  However, BCL-XL inhibitors such as ABT-263 cannot be safely used in the clinic because platelets solely depend on BCL-XL to maintain their viability.  To reduce the on-target platelet toxicity assocd. with the inhibition of BCL-XL, we designed and synthesized PROTAC BCL-XL degraders that recruit CRBN or VHL E3 ligase because both of these enzymes are poorly expressed in human platelets compared to various cancer cell lines.  We confirmed that platelet-toxic BCL-XL/2 dual inhibitor ABT-263 can be converted into platelet-sparing CRBN/VHL-based BCL-XL specific degraders.  A no. of BCL-XL degraders are more potent in killing cancer cells than their parent compd. ABT-263.  Specifically, XZ739, a CRBN-dependent BCL-XL degrader, is 20-fold more potent than ABT-263 against MOLT-4 T-ALL cells and has >100-fold selectivity for MOLT-4 cells over human platelets.  Our findings further demonstrated the utility of PROTAC technol. to achieve tissue selectivity through recruiting differentially expressed E3 ligases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYSnEzXDSn97Vg90H21EOLACvtfcHk0lgBrcif0b84UQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXksFarur4%253D&md5=879c25c55e21f3136072477d8368499b</span></div><a href="/servlet/linkout?suffix=cit38a&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112186&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112186%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DThummuri%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DHu%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DKhan%26aufirst%3DS.%26aulast%3DYuan%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DD.%26aulast%3DZheng%26aufirst%3DG.%26atitle%3DDiscovery%2520of%2520PROTAC%2520BCL-XL%2520degraders%2520as%2520potent%2520anticancer%2520agents%2520with%2520low%2520on-target%2520platelet%2520toxicity%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D192%26doi%3D10.1016%2Fj.ejmech.2020.112186" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit38b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H.-N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thummuri, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiegand, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortiz, Y. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Budamagunta, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elisseeff, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campisi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almeida, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, D.</span></span> <span> </span><span class="NLM_article-title">Using proteolysis-targeting chimera technology to reduce navitoclax platelet toxicity and improve its senolytic activity</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>, <span class="NLM_elocation-id">1996</span> <span class="refDoi"> DOI: 10.1038/s41467-020-15838-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1038%2Fs41467-020-15838-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=32332723" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BB3cXot1amtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&author=Y.+Heauthor=X.+Zhangauthor=J.+Changauthor=H.-N.+Kimauthor=P.+Zhangauthor=Y.+Wangauthor=S.+Khanauthor=X.+Liuauthor=X.+Zhangauthor=D.+Lvauthor=L.+Songauthor=W.+Liauthor=D.+Thummuriauthor=Y.+Yuanauthor=J.+S.+Wiegandauthor=Y.+T.+Ortizauthor=V.+Budamaguntaauthor=J.+H.+Elisseeffauthor=J.+Campisiauthor=M.+Almeidaauthor=G.+Zhengauthor=D.+Zhou&title=Using+proteolysis-targeting+chimera+technology+to+reduce+navitoclax+platelet+toxicity+and+improve+its+senolytic+activity&doi=10.1038%2Fs41467-020-15838-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38bR"><div class="casContent"><span class="casTitleNuber">38b</span><div class="casTitle"><span class="NLM_cas:atitle">Using proteolysis-targeting chimera technology to reduce navitoclax platelet toxicity and improve its senolytic activity</span></div><div class="casAuthors">He, Yonghan; Zhang, Xuan; Chang, Jianhui; Kim, Ha-Neui; Zhang, Peiyi; Wang, Yingying; Khan, Sajid; Liu, Xingui; Zhang, Xin; Lv, Dongwen; Song, Lin; Li, Wen; Thummuri, Dinesh; Yuan, Yaxia; Wiegand, Janet S.; Ortiz, Yuma T.; Budamagunta, Vivekananda; Elisseeff, Jennifer H.; Campisi, Judith; Almeida, Maria; Zheng, Guangrong; Zhou, Daohong</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1996</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Small mols. that selectively kill senescent cells (SCs), termed senolytics, have the potential to prevent and treat various age-related diseases and extend healthspan.  The use of Bcl-xl inhibitors as senolytics is largely limited by their on-target and dose-limiting platelet toxicity.  Here, we report the use of proteolysis-targeting chimera (PROTAC) technol. to reduce the platelet toxicity of navitoclax (also known as ABT263), a Bcl-2 and Bcl-xl dual inhibitor, by converting it into PZ15227 (PZ), a Bcl-xl PROTAC, which targets Bcl-xl to the cereblon (CRBN) E3 ligase for degrdn.  Compared to ABT263, PZ is less toxic to platelets, but equally or slightly more potent against SCs because CRBN is poorly expressed in platelets.  PZ effectively clears SCs and rejuvenates tissue stem and progenitor cells in naturally aged mice without causing severe thrombocytopenia.  With further improvement, Bcl-xl PROTACs have the potential to become safer and more potent senolytic agents than Bcl-xl inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoA89Bee3qtEbVg90H21EOLACvtfcHk0liaEbNXw4hxLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXot1amtrc%253D&md5=bb66a0a79d0180c8abbee076324ec6b3</span></div><a href="/servlet/linkout?suffix=cit38b&amp;dbid=16384&amp;doi=10.1038%2Fs41467-020-15838-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-020-15838-0%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DChang%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DH.-N.%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DKhan%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DLv%26aufirst%3DD.%26aulast%3DSong%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DThummuri%26aufirst%3DD.%26aulast%3DYuan%26aufirst%3DY.%26aulast%3DWiegand%26aufirst%3DJ.%2BS.%26aulast%3DOrtiz%26aufirst%3DY.%2BT.%26aulast%3DBudamagunta%26aufirst%3DV.%26aulast%3DElisseeff%26aufirst%3DJ.%2BH.%26aulast%3DCampisi%26aufirst%3DJ.%26aulast%3DAlmeida%26aufirst%3DM.%26aulast%3DZheng%26aufirst%3DG.%26aulast%3DZhou%26aufirst%3DD.%26atitle%3DUsing%2520proteolysis-targeting%2520chimera%2520technology%2520to%2520reduce%2520navitoclax%2520platelet%2520toxicity%2520and%2520improve%2520its%2520senolytic%2520activity%26jtitle%3DNat.%2520Commun.%26date%3D2020%26volume%3D11%26doi%3D10.1038%2Fs41467-020-15838-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit38c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thummuri, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiegand, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCurdy, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuruvilla, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baran, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrando, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.-m.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogojina, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houghton, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hromas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konopleva, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daohong
Zhou, D.</span></span> <span> </span><span class="NLM_article-title">A selective BCL-X<sub>L</sub> PROTAC degrader achieves safe and potent antitumor activity</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1938</span>– <span class="NLM_lpage">1947</span>, <span class="refDoi"> DOI: 10.1038/s41591-019-0668-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1038%2Fs41591-019-0668-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=31792461" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1yjsrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2019&pages=1938-1947&author=S.+Khanauthor=X.+Zhangauthor=D.+Lvauthor=Q.+Zhangauthor=Y.+Heauthor=P.+Zhangauthor=X.+Liuauthor=D.+Thummuriauthor=Y.+Yuanauthor=J.+S.+Wiegandauthor=J.+Peiauthor=W.+Zhangauthor=A.+Sharmaauthor=C.+R.+McCurdyauthor=V.+M.+Kuruvillaauthor=N.+Baranauthor=A.+A.+Ferrandoauthor=Y.-m.+Kimauthor=A.+Rogojinaauthor=P.+J.+Houghtonauthor=G.+Huangauthor=R.+Hromasauthor=M.+Konoplevaauthor=G.+Zhengauthor=D.+Daohong%0AZhou&title=A+selective+BCL-XL+PROTAC+degrader+achieves+safe+and+potent+antitumor+activity&doi=10.1038%2Fs41591-019-0668-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38cR"><div class="casContent"><span class="casTitleNuber">38c</span><div class="casTitle"><span class="NLM_cas:atitle">A selective BCL-XL PROTAC degrader achieves safe and potent antitumor activity</span></div><div class="casAuthors">Khan, Sajid; Zhang, Xuan; Lv, Dongwen; Zhang, Qi; He, Yonghan; Zhang, Peiyi; Liu, Xingui; Thummuri, Dinesh; Yuan, Yaxia; Wiegand, Janet S.; Pei, Jing; Zhang, Weizhou; Sharma, Abhisheak; McCurdy, Christopher R.; Kuruvilla, Vinitha M.; Baran, Natalia; Ferrando, Adolfo A.; Kim, Yong-mi; Rogojina, Anna; Houghton, Peter J.; Huang, Guangcun; Hromas, Robert; Konopleva, Marina; Zheng, Guangrong; Zhou, Daohong</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1938-1947</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">B-cell lymphoma extra large (BCL-XL) is a well-validated cancer target.  However, the on-target and dose-limiting thrombocytopenia limits the use of BCL-XL inhibitors, such as ABT263, as safe and effective anticancer agents.  To reduce the toxicity of ABT263, we converted it into DT2216, a BCL-XL proteolysis-targeting chimera (PROTAC), that targets BCL-XL to the Von Hippel-Lindau (VHL) E3 ligase for degrdn.  We found that DT2216 was more potent against various BCL-XL-dependent leukemia and cancer cells but considerably less toxic to platelets than ABT263 in vitro because VHL is poorly expressed in platelets.  In vivo, DT2216 effectively inhibits the growth of several xenograft tumors as a single agent or in combination with other chemotherapeutic agents, without causing appreciable thrombocytopenia.  These findings demonstrate the potential to use PROTAC technol. to reduce on-target drug toxicities and rescue the therapeutic potential of previously undruggable targets.  Furthermore, DT2216 may be developed as a safe first-in-class anticancer agent targeting BCL-XL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGposmmgxVNCRLVg90H21EOLACvtfcHk0liaEbNXw4hxLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1yjsrrO&md5=b07678c880569acac428ec479f002974</span></div><a href="/servlet/linkout?suffix=cit38c&amp;dbid=16384&amp;doi=10.1038%2Fs41591-019-0668-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41591-019-0668-z%26sid%3Dliteratum%253Aachs%26aulast%3DKhan%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DLv%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DThummuri%26aufirst%3DD.%26aulast%3DYuan%26aufirst%3DY.%26aulast%3DWiegand%26aufirst%3DJ.%2BS.%26aulast%3DPei%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DSharma%26aufirst%3DA.%26aulast%3DMcCurdy%26aufirst%3DC.%2BR.%26aulast%3DKuruvilla%26aufirst%3DV.%2BM.%26aulast%3DBaran%26aufirst%3DN.%26aulast%3DFerrando%26aufirst%3DA.%2BA.%26aulast%3DKim%26aufirst%3DY.-m.%26aulast%3DRogojina%26aufirst%3DA.%26aulast%3DHoughton%26aufirst%3DP.%2BJ.%26aulast%3DHuang%26aufirst%3DG.%26aulast%3DHromas%26aufirst%3DR.%26aulast%3DKonopleva%26aufirst%3DM.%26aulast%3DZheng%26aufirst%3DG.%26aulast%3DDaohong%2BZhou%26aufirst%3DD.%26atitle%3DA%2520selective%2520BCL-XL%2520PROTAC%2520degrader%2520achieves%2520safe%2520and%2520potent%2520antitumor%2520activity%26jtitle%3DNat.%2520Med.%26date%3D2019%26volume%3D25%26spage%3D1938%26epage%3D1947%26doi%3D10.1038%2Fs41591-019-0668-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Tarrant, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhawan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zabka, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DosSantos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sampath, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCray, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kauss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dambach, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Misner, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uppal, H.</span></span> <span> </span><span class="NLM_article-title">Preclinical models of nicotinamide phosphoribosyltransferase inhibitor-mediated hematotoxicity and mitigation by co-treatment with nicotinic acid</span>. <i>Toxicol. Mech. Methods</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">201</span>– <span class="NLM_lpage">211</span>, <span class="refDoi"> DOI: 10.3109/15376516.2015.1014080</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.3109%2F15376516.2015.1014080" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=25894564" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptVSjt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=201-211&author=J.+M.+Tarrantauthor=P.+Dhawanauthor=J.+Singhauthor=T.+S.+Zabkaauthor=E.+Clarkeauthor=G.+DosSantosauthor=P.+S.+Dragovichauthor=D.+Sampathauthor=T.+Linauthor=B.+McCrayauthor=N.+Laauthor=T.+Nguyenauthor=A.+Kaussauthor=D.+Dambachauthor=D.+L.+Misnerauthor=D.+Diazauthor=H.+Uppal&title=Preclinical+models+of+nicotinamide+phosphoribosyltransferase+inhibitor-mediated+hematotoxicity+and+mitigation+by+co-treatment+with+nicotinic+acid&doi=10.3109%2F15376516.2015.1014080"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39aR"><div class="casContent"><span class="casTitleNuber">39a</span><div class="casTitle"><span class="NLM_cas:atitle">Preclinical models of nicotinamide phosphoribosyltransferase inhibitor-mediated hematotoxicity and mitigation by co-treatment with nicotinic acid</span></div><div class="casAuthors">Tarrant, Jacqueline M.; Dhawan, Preeti; Singh, Jatinder; Zabka, Tanja S.; Clarke, Emer; DosSantos, Garry; Dragovich, Peter S.; Sampath, Deepak; Lin, Tori; McCray, Bobbi; La, Nghi; Nguyen, Trung; Kauss, Ariel; Dambach, Donna; Misner, Dinah L.; Diaz, Dolores; Uppal, Hirdesh</div><div class="citationInfo"><span class="NLM_cas:title">Toxicology Mechanisms and Methods</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">201-211</span>CODEN:
                <span class="NLM_cas:coden">TMMOCP</span>;
        ISSN:<span class="NLM_cas:issn">1537-6516</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">NAD (NAD) is an essential co-factor in glycolysis and is a key mol. involved in maintaining cellular energy metab.  Nicotinamide phosphoribosyltransferase (NAMPT) catalyzes the rate-limiting step of an important salvage pathway in which nicotinamide is recycled into NAD.  NAMPT is up-regulated in many types of cancer and NAMPT inhibitors (NAMPTi) have potential therapeutic benefit in cancer by impairing tumor metab.  Clin. trials with NAMPTi APO-866 and GMX-1778, however, failed to reach projected efficacious exposures due to dose-limiting thrombocytopenia.  We evaluated preclin. models for thrombocytopenia that could be used in candidate drug selection and risk mitigation strategies for NAMPTi-related toxicity.  Rats treated with a suite of structurally diverse and potent NAMPTi at max. tolerated doses had decreased reticulocyte and lymphocyte counts, but no thrombocytopenia.  We therefore evaluated and qualified a human colony forming unit-megakaryocyte (CFU-MK) as in vitro predictive model of NAMPTi-induced MK toxicity and thrombocytopenia.  We further demonstrate that the MK toxicity is on-target based on the evidence that nicotinic acid (NA), which is converted to NAD via a NAMPT-independent pathway, can mitigate NAMPTi toxicity to human CFU-MK in vitro and was also protective for the hematotoxicity in rats in vivo.  Finally, assessment of CFU-MK and human platelet bioenergetics and function show that NAMPTi was toxic to MK and not platelets, which is consistent with the clin. obsd. time-course of thrombocytopenia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVoRVrRU14ubVg90H21EOLACvtfcHk0ljeVPd4FqAXsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptVSjt7k%253D&md5=536137fdb1961f4dbf5807d9655e6549</span></div><a href="/servlet/linkout?suffix=cit39a&amp;dbid=16384&amp;doi=10.3109%2F15376516.2015.1014080&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F15376516.2015.1014080%26sid%3Dliteratum%253Aachs%26aulast%3DTarrant%26aufirst%3DJ.%2BM.%26aulast%3DDhawan%26aufirst%3DP.%26aulast%3DSingh%26aufirst%3DJ.%26aulast%3DZabka%26aufirst%3DT.%2BS.%26aulast%3DClarke%26aufirst%3DE.%26aulast%3DDosSantos%26aufirst%3DG.%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26aulast%3DSampath%26aufirst%3DD.%26aulast%3DLin%26aufirst%3DT.%26aulast%3DMcCray%26aufirst%3DB.%26aulast%3DLa%26aufirst%3DN.%26aulast%3DNguyen%26aufirst%3DT.%26aulast%3DKauss%26aufirst%3DA.%26aulast%3DDambach%26aufirst%3DD.%26aulast%3DMisner%26aufirst%3DD.%2BL.%26aulast%3DDiaz%26aufirst%3DD.%26aulast%3DUppal%26aufirst%3DH.%26atitle%3DPreclinical%2520models%2520of%2520nicotinamide%2520phosphoribosyltransferase%2520inhibitor-mediated%2520hematotoxicity%2520and%2520mitigation%2520by%2520co-treatment%2520with%2520nicotinic%2520acid%26jtitle%3DToxicol.%2520Mech.%2520Methods%26date%3D2015%26volume%3D25%26spage%3D201%26epage%3D211%26doi%3D10.3109%2F15376516.2015.1014080" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit39b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Pessina, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parent-Massin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albella, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Den
Heuvel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casati, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Croera, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malerba, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sibiril, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomez, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Smedt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gribaldo, L.</span></span> <span> </span><span class="NLM_article-title">Application of human CFU-Mk assay to predict potential thrombocytotoxicity of drugs</span>. <i>Toxicol. In Vitro</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">194</span>– <span class="NLM_lpage">200</span>, <span class="refDoi"> DOI: 10.1016/j.tiv.2008.11.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1016%2Fj.tiv.2008.11.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=19084588" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BD1MXktlKrsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2009&pages=194-200&author=A.+Pessinaauthor=D.+Parent-Massinauthor=B.+Albellaauthor=R.+Van+Den%0AHeuvelauthor=S.+Casatiauthor=C.+Croeraauthor=I.+Malerbaauthor=Y.+Sibirilauthor=S.+Gomezauthor=A.+de+Smedtauthor=L.+Gribaldo&title=Application+of+human+CFU-Mk+assay+to+predict+potential+thrombocytotoxicity+of+drugs&doi=10.1016%2Fj.tiv.2008.11.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39bR"><div class="casContent"><span class="casTitleNuber">39b</span><div class="casTitle"><span class="NLM_cas:atitle">Application of human CFU-Mk assay to predict potential thrombocytotoxicity of drugs</span></div><div class="casAuthors">Pessina, A.; Parent-Massin, D.; Albella, B.; Van Den Heuvel, R.; Casati, S.; Croera, C.; Malerba, I.; Sibiril, Y.; Gomez, S.; de Smedt, A.; Gribaldo, L.</div><div class="citationInfo"><span class="NLM_cas:title">Toxicology in Vitro</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">194-200</span>CODEN:
                <span class="NLM_cas:coden">TIVIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0887-2333</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Megakaryocytopoiesis gives rise to platelets by proliferation and differentiation of lineage-specific progenitors, identified in vitro as Colony Forming Unit-Megakaryocytes (CFU-Mk).  The aim of this study was to refine and optimize the in vitro Std. Operating Procedure (SOP) of the CFU-Mk assay for detecting drug-induced thrombocytopenia and to prevalidate a model for predicting the acute exposure levels that cause max. tolerated decreases in the platelets count, based on the correlation with the maximal plasma concns. (C max) in vivo.  The assay was linear under the SOP conditions, and the in vitro endpoints (percentage of colonies growing) were reproducible within and across labs.  The protocol performance phase was carried out testing 10 drugs (selected on the base of their recognized or potential in vivo haematotoxicity, according to the literature).  Results showed that a relationship can be established between the maximal concn. in plasma (C max) and the in vitro concns. that inhibited the 10-50-90% of colonies growth (ICs).  When C max is lower than IC10, it is possible to predict that the chems. have no direct toxicity effect on CFU-Mk and could not induce thrombocytopenia due to bone marrow damage.  When the C max is higher than IC90 and/or IC50, thrombocytopenia can occur due to direct toxicity of chems. on CFU-Mk progenitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4ZYnZ3DfsdLVg90H21EOLACvtfcHk0ljeVPd4FqAXsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXktlKrsg%253D%253D&md5=6994a99dee97665057c0862450d79350</span></div><a href="/servlet/linkout?suffix=cit39b&amp;dbid=16384&amp;doi=10.1016%2Fj.tiv.2008.11.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tiv.2008.11.006%26sid%3Dliteratum%253Aachs%26aulast%3DPessina%26aufirst%3DA.%26aulast%3DParent-Massin%26aufirst%3DD.%26aulast%3DAlbella%26aufirst%3DB.%26aulast%3DVan%2BDen%2BHeuvel%26aufirst%3DR.%26aulast%3DCasati%26aufirst%3DS.%26aulast%3DCroera%26aufirst%3DC.%26aulast%3DMalerba%26aufirst%3DI.%26aulast%3DSibiril%26aufirst%3DY.%26aulast%3DGomez%26aufirst%3DS.%26aulast%3Dde%2BSmedt%26aufirst%3DA.%26aulast%3DGribaldo%26aufirst%3DL.%26atitle%3DApplication%2520of%2520human%2520CFU-Mk%2520assay%2520to%2520predict%2520potential%2520thrombocytotoxicity%2520of%2520drugs%26jtitle%3DToxicol.%2520In%2520Vitro%26date%3D2009%26volume%3D23%26spage%3D194%26epage%3D200%26doi%3D10.1016%2Fj.tiv.2008.11.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Popow, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnhof, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Covini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dank, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmaschitz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greb, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karolyi-Özguer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koegl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConnell, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rieger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinnenthal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roessler, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrenk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spina, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steurer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trainor, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traxler, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wieshofer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoephel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ettmayer, P.</span></span> <span> </span><span class="NLM_article-title">Highly selective PTK2 proteolysis targeting chimeras to probe focal adhesion kinase scaffolding functions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">2508</span>– <span class="NLM_lpage">2520</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01826</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01826" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisl2ls78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=2508-2520&author=J.+Popowauthor=H.+Arnhofauthor=G.+Baderauthor=H.+Bergerauthor=A.+Ciulliauthor=D.+Coviniauthor=C.+Dankauthor=T.+Gmaschitzauthor=P.+Grebauthor=J.+Karolyi-%C3%96zguerauthor=M.+Koeglauthor=D.+B.+McConnellauthor=M.+Pearsonauthor=M.+Riegerauthor=J.+Rinnenthalauthor=V.+Roesslerauthor=A.+Schrenkauthor=M.+Spinaauthor=S.+Steurerauthor=N.+Trainorauthor=E.+Traxlerauthor=C.+Wieshoferauthor=A.+Zoephelauthor=P.+Ettmayer&title=Highly+selective+PTK2+proteolysis+targeting+chimeras+to+probe+focal+adhesion+kinase+scaffolding+functions&doi=10.1021%2Facs.jmedchem.8b01826"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40aR"><div class="casContent"><span class="casTitleNuber">40a</span><div class="casTitle"><span class="NLM_cas:atitle">Highly Selective PTK2 Proteolysis Targeting Chimeras to Probe Focal Adhesion Kinase Scaffolding Functions</span></div><div class="casAuthors">Popow, Johannes; Arnhof, Heribert; Bader, Gerd; Berger, Helmut; Ciulli, Alessio; Covini, David; Dank, Christian; Gmaschitz, Teresa; Greb, Peter; Karolyi-Oezguer, Jale; Koegl, Manfred; McConnell, Darryl B.; Pearson, Mark; Rieger, Maria; Rinnenthal, Joerg; Roessler, Vanessa; Schrenk, Andreas; Spina, Markus; Steurer, Steffen; Trainor, Nicole; Traxler, Elisabeth; Wieshofer, Corinna; Zoephel, Andreas; Ettmayer, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2508-2520</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Focal adhesion tyrosine kinase (PTK2) is often overexpressed in human hepatocellular carcinoma (HCC), and several reports have linked PTK2 depletion and/or pharmacol. inhibition to reduced tumorigenicity.  However, the clin. relevance of targeting PTK2 still remains to be proven.  Here, we present two highly selective and functional PTK2 proteolysis-targeting chimeras utilizing von Hippel-Lindau and cereblon ligands to hijack E3 ligases for PTK2 degrdn.  BI-3663 (cereblon-based) degrades PTK2 with a median DC50 of 30 nM to >80% across a panel of 11 HCC cell lines.  Despite effective PTK2 degrdn., these compds. did not phenocopy the reported antiproliferative effects of PTK2 depletion in any of the cell lines tested.  By disclosing these compds., we hope to provide valuable tools for the study of PTK2 degrdn. across different biol. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsL2R_QBrfk7Vg90H21EOLACvtfcHk0lgw1twY_gDZfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisl2ls78%253D&md5=f0ad27d3ff46ab12e8a90e8d40a6f3c6</span></div><a href="/servlet/linkout?suffix=cit40a&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01826&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01826%26sid%3Dliteratum%253Aachs%26aulast%3DPopow%26aufirst%3DJ.%26aulast%3DArnhof%26aufirst%3DH.%26aulast%3DBader%26aufirst%3DG.%26aulast%3DBerger%26aufirst%3DH.%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DCovini%26aufirst%3DD.%26aulast%3DDank%26aufirst%3DC.%26aulast%3DGmaschitz%26aufirst%3DT.%26aulast%3DGreb%26aufirst%3DP.%26aulast%3DKarolyi-%25C3%2596zguer%26aufirst%3DJ.%26aulast%3DKoegl%26aufirst%3DM.%26aulast%3DMcConnell%26aufirst%3DD.%2BB.%26aulast%3DPearson%26aufirst%3DM.%26aulast%3DRieger%26aufirst%3DM.%26aulast%3DRinnenthal%26aufirst%3DJ.%26aulast%3DRoessler%26aufirst%3DV.%26aulast%3DSchrenk%26aufirst%3DA.%26aulast%3DSpina%26aufirst%3DM.%26aulast%3DSteurer%26aufirst%3DS.%26aulast%3DTrainor%26aufirst%3DN.%26aulast%3DTraxler%26aufirst%3DE.%26aulast%3DWieshofer%26aufirst%3DC.%26aulast%3DZoephel%26aufirst%3DA.%26aulast%3DEttmayer%26aufirst%3DP.%26atitle%3DHighly%2520selective%2520PTK2%2520proteolysis%2520targeting%2520chimeras%2520to%2520probe%2520focal%2520adhesion%2520kinase%2520scaffolding%2520functions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D2508%26epage%3D2520%26doi%3D10.1021%2Facs.jmedchem.8b01826" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit40b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">impact of target warhead and linkage vector on inducing protein degradation: Comparison of bromodomain and extra-terminal (BET) degraders derived from triazolodiazepine (JQ1) and tetrahydroquinoline (I-BET726) BET inhibitor scaffolds</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">504</span>– <span class="NLM_lpage">513</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01912</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01912" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpslSktL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=504-513&author=K.-H.+Chanauthor=M.+Zengerleauthor=A.+Testaauthor=A.+Ciulli&title=impact+of+target+warhead+and+linkage+vector+on+inducing+protein+degradation%3A+Comparison+of+bromodomain+and+extra-terminal+%28BET%29+degraders+derived+from+triazolodiazepine+%28JQ1%29+and+tetrahydroquinoline+%28I-BET726%29+BET+inhibitor+scaffolds&doi=10.1021%2Facs.jmedchem.6b01912"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40bR"><div class="casContent"><span class="casTitleNuber">40b</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of Target Warhead and Linkage Vector on Inducing Protein Degradation: Comparison of Bromodomain and Extra-Terminal (BET) Degraders Derived from Triazolodiazepine (JQ1) and Tetrahydroquinoline (I-BET726) BET Inhibitor Scaffolds</span></div><div class="casAuthors">Chan, Kwok-Ho; Zengerle, Michael; Testa, Andrea; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">504-513</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The design of proteolysis-targeting chimeras (PROTACs) is a powerful small-mol. approach for inducing protein degrdn.  PROTACs conjugate a target warhead to an E3 ubiquitin ligase ligand via a linker.  Here the authors examd. the impact of derivatizing two different BET bromodomain inhibitors, triazolodiazepine JQ1 and the more potent tetrahydroquinoline I-BET726, via distinct exit vectors, using different polyethylene glycol linkers to VHL ligand VH032.  Triazolodiazepine PROTACs exhibited pos. cooperativities of ternary complex formation and were more potent degraders than tetrahydroquinoline compds., which showed neg. cooperativities instead.  Marked dependency on linker length was obsd. for BET-degrading and cMyc-driven antiproliferative activities in acute myeloid leukemia cell lines.  This work exemplifies as a cautionary tale how a more potent inhibitor does not necessarily generate more potent PROTACs and underscores the key roles played by the conjugation.  The provided insights and framework for structure-activity relationships of bivalent degraders are anticipated to have wide future applicability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvEL41TefwTbVg90H21EOLACvtfcHk0lgw1twY_gDZfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpslSktL4%253D&md5=c3bfd9377d8e7805ba962cd0180431af</span></div><a href="/servlet/linkout?suffix=cit40b&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01912&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01912%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DK.-H.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3Dimpact%2520of%2520target%2520warhead%2520and%2520linkage%2520vector%2520on%2520inducing%2520protein%2520degradation%253A%2520Comparison%2520of%2520bromodomain%2520and%2520extra-terminal%2520%2528BET%2529%2520degraders%2520derived%2520from%2520triazolodiazepine%2520%2528JQ1%2529%2520and%2520tetrahydroquinoline%2520%2528I-BET726%2529%2520BET%2520inhibitor%2520scaffolds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D504%26epage%3D513%26doi%3D10.1021%2Facs.jmedchem.6b01912" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amara, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mashiach, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amar, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krief, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spieser, S. A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bottomley, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aharoni, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meijler, M. M.</span></span> <span> </span><span class="NLM_article-title">Covalent inhibition of bacterial quorum sensing</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>131</i></span>,  <span class="NLM_fpage">10610</span>– <span class="NLM_lpage">10619</span>, <span class="refDoi"> DOI: 10.1021/ja903292v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja903292v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BD1MXot1ejtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2009&pages=10610-10619&author=N.+Amaraauthor=R.+Mashiachauthor=D.+Amarauthor=P.+Kriefauthor=S.+A.+H.+Spieserauthor=M.+J.+Bottomleyauthor=A.+Aharoniauthor=M.+M.+Meijler&title=Covalent+inhibition+of+bacterial+quorum+sensing&doi=10.1021%2Fja903292v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent Inhibition of Bacterial Quorum Sensing</span></div><div class="casAuthors">Amara, Neri; Mashiach, Roi; Amar, Dotan; Krief, Pnina; Spieser, Stephane A. H.; Bottomley, Matthew J.; Aharoni, Amir; Meijler, Michael M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">10610-10619</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Chem. coordination of gene expression among bacteria as a function of population d. is regulated by a mechanism known as 'quorum sensing' (QS).  QS in Pseudomonas aeruginosa, an opportunistic pathogen that causes disease in immunocompromised patients, is mediated by binding of the transcriptional activator, LasR, to its ligand, 3-oxo-C12-HSL, leading to population-wide secretion of virulence factors and biofilm formation.  We have targeted QS in P. aeruginosa with a set of electrophilic probes designed to covalently bind Cys79 in the LasR binding pocket, leading to specific inhibition of QS-regulated gene expression and concomitant redn. of virulence factor secretion and biofilm formation.  This first example of covalent modification of a QS receptor provides a new tool to study mol. mechanisms of bacterial group behavior and could lead to new strategies for targeting bacterial virulence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeWMbH025eh7Vg90H21EOLACvtfcHk0lizPlkM7rqQgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXot1ejtLw%253D&md5=4f58a662d8ec92f6d44d920d53f2906c</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fja903292v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja903292v%26sid%3Dliteratum%253Aachs%26aulast%3DAmara%26aufirst%3DN.%26aulast%3DMashiach%26aufirst%3DR.%26aulast%3DAmar%26aufirst%3DD.%26aulast%3DKrief%26aufirst%3DP.%26aulast%3DSpieser%26aufirst%3DS.%2BA.%2BH.%26aulast%3DBottomley%26aufirst%3DM.%2BJ.%26aulast%3DAharoni%26aufirst%3DA.%26aulast%3DMeijler%26aufirst%3DM.%2BM.%26atitle%3DCovalent%2520inhibition%2520of%2520bacterial%2520quorum%2520sensing%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2009%26volume%3D131%26spage%3D10610%26epage%3D10619%26doi%3D10.1021%2Fja903292v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delproposto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of ARD-69 as a highly potent proteolysis targeting chimera (PROTAC) degrader of androgen receptor (AR) for the treatment of prostate cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">941</span>– <span class="NLM_lpage">964</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01631</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01631" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlvFCjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=941-964&author=X.+Hanauthor=C.+Wangauthor=C.+Qinauthor=W.+Xiangauthor=E.+Fernandez-Salasauthor=C.-Y.+Yangauthor=M.+Wangauthor=L.+Zhaoauthor=T.+Xuauthor=K.+Chinnaswamyauthor=J.+Delpropostoauthor=J.+Stuckeyauthor=S.+Wang&title=Discovery+of+ARD-69+as+a+highly+potent+proteolysis+targeting+chimera+%28PROTAC%29+degrader+of+androgen+receptor+%28AR%29+for+the+treatment+of+prostate+cancer&doi=10.1021%2Facs.jmedchem.8b01631"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of ARD-69 as a highly potent proteolysis targeting chimera (PROTAC) degrader of androgen receptor (AR) for the treatment of prostate cancer</span></div><div class="casAuthors">Han, Xin; Wang, Chao; Qin, Chong; Xiang, Weiguo; Fernandez-Salas, Ester; Yang, Chao-Yie; Wang, Mi; Zhao, Lijie; Xu, Tianfeng; Chinnaswamy, Krishnapriya; Delproposto, James; Stuckey, Jeanne; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">941-964</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report herein the discovery of highly potent PROTAC degraders of androgen receptor (AR), as exemplified by IARD-69.  I induces degrdn. of AR protein in AR-pos. prostate cancer cell lines in a dose- and time-dependent manner.  I achieves DC50 values of 0.86, 0.76, and 10.4 nM in LNCaP, VCaP, and 22Rv1 AR+ prostate cancer cell lines, resp.  I is capable of reducing the AR protein level by >95% in these prostate cancer cell lines and effectively suppressing AR-regulated gene expression.  I potently inhibits cell growth in these AR-pos. prostate cancer cell lines and is >100 times more potent than AR antagonists.  A single dose of I effectively reduces the level of AR protein in xenograft tumor tissue in mice.  Further optimization of I may ultimately lead to a new therapy for AR+, castration-resistant prostate cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTlOD69tfhtbVg90H21EOLACvtfcHk0lizPlkM7rqQgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlvFCjtQ%253D%253D&md5=2376573fb1a1e19c517d64c70199700a</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01631&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01631%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DQin%26aufirst%3DC.%26aulast%3DXiang%26aufirst%3DW.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DT.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DDelproposto%26aufirst%3DJ.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520ARD-69%2520as%2520a%2520highly%2520potent%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%2520degrader%2520of%2520androgen%2520receptor%2520%2528AR%2529%2520for%2520the%2520treatment%2520of%2520prostate%2520cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D941%26epage%3D964%26doi%3D10.1021%2Facs.jmedchem.8b01631" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kadokura, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urashima, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sekine, M.</span></span> <span> </span><span class="NLM_article-title">Efficient synthesis of γ-methyl-capped guanosine 5′-triphosphate as a 5′-terminal unique structure of U6 RNA via a new triphosphate bond formation involving activation of methyl phosphorimidazolidate using ZnCl<sub>2</sub> as a catalyst in DMF under anhydrous conditions</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">8359</span>– <span class="NLM_lpage">8362</span>, <span class="refDoi"> DOI: 10.1016/S0040-4039(97)10263-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1016%2FS0040-4039%2897%2910263-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADyaK2sXnslekur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1997&pages=8359-8362&author=M.+Kadokuraauthor=T.+Wadaauthor=C.+Urashimaauthor=M.+Sekine&title=Efficient+synthesis+of+%CE%B3-methyl-capped+guanosine+5%E2%80%B2-triphosphate+as+a+5%E2%80%B2-terminal+unique+structure+of+U6+RNA+via+a+new+triphosphate+bond+formation+involving+activation+of+methyl+phosphorimidazolidate+using+ZnCl2+as+a+catalyst+in+DMF+under+anhydrous+conditions&doi=10.1016%2FS0040-4039%2897%2910263-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient synthesis of γ-methyl-capped guanosine 5'-triphosphate as a 5'-terminal unique structure of U6 RNA via a new triphosphate bond formation involving activation of methyl phosphorimidazolidate using ZnCl2 as a catalyst in DMF under anhydrous conditions</span></div><div class="casAuthors">Kadokura, Michinori; Wada, Takeshi; Urashima, Chihiro; Sekine, Mitsuo</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">8359-8362</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">P1-Me P3-(5'-guanosyl) triphosphate (CH3pppG), which exists as a unique structure at the 5'-terminal region of U6 RNA, was synthesized in an excellent yield by Me phosphorimidazolidate with the bis(tetrabutylammonium) salt of GDP in the presence of ZnCl2 as an effective catalyst in dry DMF.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrv2xGEhgGgYLVg90H21EOLACvtfcHk0lgAH0bQqRVZPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXnslekur4%253D&md5=2baad1e3cfefaed7bb985ba5ef93fa6a</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2FS0040-4039%2897%2910263-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0040-4039%252897%252910263-5%26sid%3Dliteratum%253Aachs%26aulast%3DKadokura%26aufirst%3DM.%26aulast%3DWada%26aufirst%3DT.%26aulast%3DUrashima%26aufirst%3DC.%26aulast%3DSekine%26aufirst%3DM.%26atitle%3DEfficient%2520synthesis%2520of%2520%25CE%25B3-methyl-capped%2520guanosine%25205%25E2%2580%25B2-triphosphate%2520as%2520a%25205%25E2%2580%25B2-terminal%2520unique%2520structure%2520of%2520U6%2520RNA%2520via%2520a%2520new%2520triphosphate%2520bond%2520formation%2520involving%2520activation%2520of%2520methyl%2520phosphorimidazolidate%2520using%2520ZnCl2%2520as%2520a%2520catalyst%2520in%2520DMF%2520under%2520anhydrous%2520conditions%26jtitle%3DTetrahedron%2520Lett.%26date%3D1997%26volume%3D38%26spage%3D8359%26epage%3D8362%26doi%3D10.1016%2FS0040-4039%2897%2910263-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitehead, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammer, R. P.</span></span> <span> </span><span class="NLM_article-title">Application of in situ silylation for improved, convenient preparation of fluorenylmethoxycarbonyl (Fmoc)-protected phosphinate amino acids</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">3116</span>– <span class="NLM_lpage">3118</span>, <span class="refDoi"> DOI: 10.1021/jo070266p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo070266p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BD2sXktVygsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2007&pages=3116-3118&author=S.+Liauthor=J.+K.+Whiteheadauthor=R.+P.+Hammer&title=Application+of+in+situ+silylation+for+improved%2C+convenient+preparation+of+fluorenylmethoxycarbonyl+%28Fmoc%29-protected+phosphinate+amino+acids&doi=10.1021%2Fjo070266p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Application of in Situ Silylation for Improved, Convenient Preparation of Fluorenylmethoxycarbonyl (Fmoc)-Protected Phosphinate Amino Acids</span></div><div class="casAuthors">Li, Shunzi; Whitehead, John K.; Hammer, Robert P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3116-3118</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A convenient and efficient method has been developed for the prepn. of 9-fluorenylmethoxycarbonyl (Fmoc)-protected 1-aminoalkylphosphinic acids.  Reproducible procedures for the synthesis and purifn. of free α-amino H-phosphinates are provided.  Protection of free amino phosphinates as the N-Fmoc deriv. was achieved by in situ trimethylsilylation of aminoalkylphosphinic acids, which then reacted with Fmoc-Cl to provide corresponding products in excellent yields and in high purity after simple extractive isolation.  Mechanistic aspects of the silylation are discussed, and the application of the procedure to another class of amino phosphorus acids is presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFCbe8r82ohbVg90H21EOLACvtfcHk0lgAH0bQqRVZPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXktVygsbo%253D&md5=705946bb116c94961f725d1e32bb91eb</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fjo070266p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo070266p%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DS.%26aulast%3DWhitehead%26aufirst%3DJ.%2BK.%26aulast%3DHammer%26aufirst%3DR.%2BP.%26atitle%3DApplication%2520of%2520in%2520situ%2520silylation%2520for%2520improved%252C%2520convenient%2520preparation%2520of%2520fluorenylmethoxycarbonyl%2520%2528Fmoc%2529-protected%2520phosphinate%2520amino%2520acids%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2007%26volume%3D72%26spage%3D3116%26epage%3D3118%26doi%3D10.1021%2Fjo070266p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Flygare, J.</span>; <span class="NLM_string-name">Gunzner-Toste, J.</span>; <span class="NLM_string-name">Pillow, T.</span>; <span class="NLM_string-name">Safina, B.</span>; <span class="NLM_string-name">Verma, V.</span>; <span class="NLM_string-name">Wei, B.</span>; <span class="NLM_string-name">Zhao, G.</span>; <span class="NLM_string-name">Staben, L.</span></span> <span> </span><span class="NLM_article-title">Peptidomimetic Compounds and Antibody-conjugates Thereof</span>. International patent application, <span class="NLM_patent">WO2015095223</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=J.+Flygare&author=J.+Gunzner-Toste&author=T.+Pillow&author=B.+Safina&author=V.+Verma&author=B.+Wei&author=G.+Zhao&author=L.+Staben&title=Peptidomimetic+Compounds+and+Antibody-conjugates+Thereof"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DFlygare%26aufirst%3DJ.%26atitle%3DPeptidomimetic%2520Compounds%2520and%2520Antibody-conjugates%2520Thereof%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mondal, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinney, K. G.</span></span> <span> </span><span class="NLM_article-title">Improved methodology for the synthesis of a cathepsin B cleavable dipeptide linker, widely used in antibody-drug conjugate research</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">3594</span>– <span class="NLM_lpage">3599</span>, <span class="refDoi"> DOI: 10.1016/j.tetlet.2018.08.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1016%2Fj.tetlet.2018.08.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=31156276" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1enu7%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2018&pages=3594-3599&author=D.+Mondalauthor=J.+Fordauthor=K.+G.+Pinney&title=Improved+methodology+for+the+synthesis+of+a+cathepsin+B+cleavable+dipeptide+linker%2C+widely+used+in+antibody-drug+conjugate+research&doi=10.1016%2Fj.tetlet.2018.08.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Improved methodology for the synthesis of a cathepsin b cleavable dipeptide linker, widely used in antibody-drug conjugate research</span></div><div class="casAuthors">Mondal, Deboprosad; Ford, Jacob; Pinney, Kevin G.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">3594-3599</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Antibody-drug conjugates (ADCs) represent an emerging class of biopharmaceutical agents that deliver highly potent anticancer agents (payloads) selectively to tumors or components assocd. with the tumor microenvironment.  The linker, responsible for the connection between the antibody and payload, is a crucial component of ADCs.  In certain examples the linker is composed of a cleavable short peptide which imparts an addnl. aspect of selectivity.  Esp. prevalent is the cathepsin B cleavable Mc-Val-Cit-PABOH linker utilized in many pre-clin. ADC candidates, as well as the FDA approved ADC ADCETRIS (brentuximab vedotin).  An alternative route for the synthesis of the cathepsin B cleavable Mc-Val-Cit-PABOH linker is reported herein that involved six steps from L-Citrulline and proceeded with a 50% overall yield.  In this modified route, the spacer (a para-aminobenzyl alc. moiety) was incorporated via HATU coupling followed by dipeptide formation.  Importantly, this route avoided undesirable epimerization and proceeded with improved overall yield.  Utilizing this methodol., a drug-linker construct incorporating a potent small-mol. inhibitor of tubulin polymn. (referred to as KGP05), was synthesized as a representative example.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmlphUMG1MVbVg90H21EOLACvtfcHk0lgJlUYL13CCrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1enu7%252FM&md5=60399fc5e1c2755e60f6d26c52de2b0b</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.tetlet.2018.08.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tetlet.2018.08.021%26sid%3Dliteratum%253Aachs%26aulast%3DMondal%26aufirst%3DD.%26aulast%3DFord%26aufirst%3DJ.%26aulast%3DPinney%26aufirst%3DK.%2BG.%26atitle%3DImproved%2520methodology%2520for%2520the%2520synthesis%2520of%2520a%2520cathepsin%2520B%2520cleavable%2520dipeptide%2520linker%252C%2520widely%2520used%2520in%2520antibody-drug%2520conjugate%2520research%26jtitle%3DTetrahedron%2520Lett.%26date%3D2018%26volume%3D59%26spage%3D3594%26epage%3D3599%26doi%3D10.1016%2Fj.tetlet.2018.08.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Polakis, P.</span>; <span class="NLM_string-name">Dragovich, P.</span>; <span class="NLM_string-name">Pillow, T. H.</span>; <span class="NLM_string-name">Yu, S.-F.</span></span> <span> </span><span class="NLM_article-title">Antibodies and Immunoconjugates</span>. International patent application, <span class="NLM_patent">WO2016205176</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=P.+Polakis&author=P.+Dragovich&author=T.+H.+Pillow&author=S.-F.+Yu&title=Antibodies+and+Immunoconjugates"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DPolakis%26aufirst%3DP.%26atitle%3DAntibodies%2520and%2520Immunoconjugates%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, S.</span>; <span class="NLM_string-name">Wang, K.-P.</span></span> <span> </span><span class="NLM_article-title">ADCs With Thiol Multiplex Linkers</span>. International patent application, <span class="NLM_patent">WO2020132655</span>, <span class="NLM_year">2020</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=S.+Jeffrey&author=K.-P.+Wang&title=ADCs+With+Thiol+Multiplex+Linkers"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DJeffrey%26aufirst%3DS.%26atitle%3DADCs%2520With%2520Thiol%2520Multiplex%2520Linkers%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit48b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Le Joncour, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martins, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puhka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isola, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salmikangas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laakkonen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joensuu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barok, M.</span></span> <span> </span><span class="NLM_article-title">A movel anti-HER2 antibody-drug conjugate XMT-1522 for HER2-positive breast and gastric cancers resistant to trastuzumab emtansine</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1721</span>– <span class="NLM_lpage">1730</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-19-0207</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1158%2F1535-7163.MCT-19-0207" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=31292166" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BB3cXmsFKlug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=1721-1730&author=V.+Le+Joncourauthor=A.+Martinsauthor=M.+Puhkaauthor=J.+Isolaauthor=M.+Salmikangasauthor=P.+Laakkonenauthor=H.+Joensuuauthor=M.+Barok&title=A+movel+anti-HER2+antibody-drug+conjugate+XMT-1522+for+HER2-positive+breast+and+gastric+cancers+resistant+to+trastuzumab+emtansine&doi=10.1158%2F1535-7163.MCT-19-0207"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48bR"><div class="casContent"><span class="casTitleNuber">48b</span><div class="casTitle"><span class="NLM_cas:atitle">A novel anti-HER2 antibody-drug conjugate XMT-1522 for HER2-positive breast and gastric cancers resistant to trastuzumab emtansine</span></div><div class="casAuthors">Le Joncour, Vadim; Martins, Ana; Puhka, Maija; Isola, Jorma; Salmikangas, Marko; Laakkonen, Pirjo; Joensuu, Heikki; Barok, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1721-1730</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Most patients with HER2-pos. breast or gastric cancer exhibit primary or acquired resistance to trastuzumab emtansine (T-DM1), and such patients may have limited therapeutic options.  XMT-1522 is a novel anti-HER2 antibody-drug conjugate.  We compared XMT-1522 to T-DM1 in preclin. models.  The effects of XMT-1522 and T-DM1 on cell survival and apoptosis were compared in six HER2-pos. breast cancer or gastric cancer cell lines, of which three lines were T-DM1-sensitive (N-87, OE-19, JIMT-1) and three T-DM1-resistant (RN-87, ROE-19, SNU-216).  We compared these agents also in the HER2-neg. breast cancer cell line MCF-7, and in mouse RN-87 and JIMT-1 xenograft models.  Cell survival was assessed using the AlamarBlue method and apoptosis with the Caspase-Glo 3/7 method.  XMT-1522 inhibited the growth of all six HER2-pos. cell lines.  The proportions of cells that survived XMT-1522 treatment were smaller as compared with T-DM1, particularly in the T-DM1-resistant cell lines.  XMT-1522 induced more cell apoptosis compared with T-DM1.  While RN-87 and JIMT-1 xenograft tumors progressed on T-DM1 treatment, all tumors responded to XMT-1522, and all but one tumor disappeared during the XMT-1522 treatment.  XMT-1522 had a strong antitumor effect on RN-87 and JIMT-1 xenografts that progressed on T-DM1.  We conclude that XMT-1522 was effective in HER2-pos. breast cancer and gastric cancer cell lines resistant to T-DM1, and in xenograft models resistant to T-DM1.  The results support the testing of XMT-1522 in clin. trials in patients with HER2-pos. cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_Pci-NaVO6bVg90H21EOLACvtfcHk0lgClQ1kD534wA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXmsFKlug%253D%253D&md5=71fd86427ea0bac472f679076eb4c680</span></div><a href="/servlet/linkout?suffix=cit48b&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-19-0207&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-19-0207%26sid%3Dliteratum%253Aachs%26aulast%3DLe%2BJoncour%26aufirst%3DV.%26aulast%3DMartins%26aufirst%3DA.%26aulast%3DPuhka%26aufirst%3DM.%26aulast%3DIsola%26aufirst%3DJ.%26aulast%3DSalmikangas%26aufirst%3DM.%26aulast%3DLaakkonen%26aufirst%3DP.%26aulast%3DJoensuu%26aufirst%3DH.%26aulast%3DBarok%26aufirst%3DM.%26atitle%3DA%2520movel%2520anti-HER2%2520antibody-drug%2520conjugate%2520XMT-1522%2520for%2520HER2-positive%2520breast%2520and%2520gastric%2520cancers%2520resistant%2520to%2520trastuzumab%2520emtansine%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2019%26volume%3D18%26spage%3D1721%26epage%3D1730%26doi%3D10.1158%2F1535-7163.MCT-19-0207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit48c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Viricel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fournet, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beaumel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perrial, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papotd, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumontet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joseph, B.</span></span> <span> </span><span class="NLM_article-title">Monodisperse polysarcosine-based highly-loaded antibody-drug conjugates</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">4048</span>– <span class="NLM_lpage">4053</span>, <span class="refDoi"> DOI: 10.1039/C9SC00285E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1039%2FC9SC00285E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=31015945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1MXkt1Wqurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=4048-4053&author=W.+Viricelauthor=G.+Fournetauthor=S.+Beaumelauthor=E.+Perrialauthor=S.+Papotdauthor=C.+Dumontetauthor=B.+Joseph&title=Monodisperse+polysarcosine-based+highly-loaded+antibody-drug+conjugates&doi=10.1039%2FC9SC00285E"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48cR"><div class="casContent"><span class="casTitleNuber">48c</span><div class="casTitle"><span class="NLM_cas:atitle">Monodisperse polysarcosine-based highly-loaded antibody-drug conjugates</span></div><div class="casAuthors">Viricel, Warren; Fournet, Guy; Beaumel, Sabine; Perrial, Emeline; Papot, Sebastien; Dumontet, Charles; Joseph, Benoit</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4048-4053</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Antibody-drug conjugates (ADCs) convey highly potent anticancer drugs to antigen-expressing tumor cells, thereby sparing healthy tissues throughout the body.  Pharmacokinetics and tolerability of ADCs are predominantly influenced by the drug-antibody ratio (DAR) of the conjugates, which is to-date limited to a value of 3-4 drugs per antibody in Antibody-drug conjugates under clin. investigations.  Here, we report the synthesis of monodisperse (i.e. discrete) polysarcosine compds. and their use as a hydrophobicity masking entity for the construction of highly-loaded homogeneous glucuronidase-responsive antibody-drug conjugates (ADCs).  The highly hydrophilic drug-linker platform described herein improves drug-loading, physicochem. properties, pharmacokinetics and in vivo antitumor efficacy of the resulting conjugates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyJWZGJpnUOrVg90H21EOLACvtfcHk0lgClQ1kD534wA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXkt1Wqurk%253D&md5=3f6556a5f22ba298ae4d504a68bca3c0</span></div><a href="/servlet/linkout?suffix=cit48c&amp;dbid=16384&amp;doi=10.1039%2FC9SC00285E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC9SC00285E%26sid%3Dliteratum%253Aachs%26aulast%3DViricel%26aufirst%3DW.%26aulast%3DFournet%26aufirst%3DG.%26aulast%3DBeaumel%26aufirst%3DS.%26aulast%3DPerrial%26aufirst%3DE.%26aulast%3DPapotd%26aufirst%3DS.%26aulast%3DDumontet%26aufirst%3DC.%26aulast%3DJoseph%26aufirst%3DB.%26atitle%3DMonodisperse%2520polysarcosine-based%2520highly-loaded%2520antibody-drug%2520conjugates%26jtitle%3DChem.%2520Sci.%26date%3D2019%26volume%3D10%26spage%3D4048%26epage%3D4053%26doi%3D10.1039%2FC9SC00285E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit48d"><span><span class="NLM_label">(d) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Neumann, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olivas, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loftus, L. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neale, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nix, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pittman, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simmons, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ulrich, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waight, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaval, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyon, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senter, P. D.</span></span> <span> </span><span class="NLM_article-title">Targeted delivery of cytotoxic NAMPT inhibitors using antibody-drug conjugates</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">2633</span>– <span class="NLM_lpage">2642</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-18-0643</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1158%2F1535-7163.MCT-18-0643" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48d&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=30242091" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjs1elsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=2633-2642&author=C.+S.+Neumannauthor=K.+C.+Olivasauthor=M.+E.+Andersonauthor=J.+H.+Cochranauthor=S.+Jinauthor=F.+Liauthor=L.+V.+Loftusauthor=D.+W.+Meyerauthor=J.+Nealeauthor=J.+C.+Nixauthor=P.+G.+Pittmanauthor=J.+K.+Simmonsauthor=M.+L.+Ulrichauthor=A.+B.+Waightauthor=A.+Wongauthor=M.+C.+Zavalauthor=W.+Zengauthor=R.+P.+Lyonauthor=P.+D.+Senter&title=Targeted+delivery+of+cytotoxic+NAMPT+inhibitors+using+antibody-drug+conjugates&doi=10.1158%2F1535-7163.MCT-18-0643"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48dR"><div class="casContent"><span class="casTitleNuber">48d</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted delivery of cytotoxic NAMPT inhibitors using antibody-drug conjugates</span></div><div class="casAuthors">Neumann, Christopher S.; Olivas, Kathleen C.; Anderson, Martha E.; Cochran, Julia H.; Jin, Steven; Li, Fu; Loftus, Luke V.; Meyer, David W.; Neale, Jason; Nix, Jay C.; Pittman, Paul G.; Simmons, Jessica K.; Ulrich, Michelle L.; Waight, Andrew B.; Wong, Abbie; Zaval, Margo C.; Zeng, Weiping; Lyon, Robert P.; Senter, Peter D.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2633-2642</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Antibody-drug conjugates (ADCs) are a therapeutic modality that enables the targeted delivery of cytotoxic drugs to cancer cells.  Identification of active payloads with unique mechanisms of action is a key aim of research efforts in the field.  Herein, we report the development of inhibitors of nicotinamide phosphoribosyltransferase (NAMPT) as a novel payload for ADC technol.  NAMPT is a component of a salvage biosynthetic pathway for NAD, and inhibition of this enzyme results in disruption of primary cellular metab. leading to cell death.  Through derivatization of the prototypical NAMPT inhibitor FK-866, we identified potent analogs with chem. functionality that enables the synthesis of hydrophilic enzyme-cleavable drug linkers.  The resulting ADCs displayed NAD depletion in both cell-based assays and tumor xenografts.  Antitumor efficacy is demonstrated in five mouse xenograft models using ADCs directed to indication-specific antigens.  In rat toxicol. models, a nonbinding control ADC was tolerated at >10-fold the typical efficacious dose used in xenografts.  Moderate, reversible hematol. effects were obsd. with ADCs in rats, but there was no evidence for the retinal and cardiac toxicities reported for small-mol. inhibitors.  These findings introduce NAMPT inhibitors as active and well-tolerated payloads for ADCs with promise to improve the therapeutic window of NAMPT inhibition and enable application in clin. settings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpx2rsX00yD6LVg90H21EOLACvtfcHk0lgClQ1kD534wA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjs1elsL0%253D&md5=43118eed82fe96b192cac5dca45a7b44</span></div><a href="/servlet/linkout?suffix=cit48d&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-18-0643&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-18-0643%26sid%3Dliteratum%253Aachs%26aulast%3DNeumann%26aufirst%3DC.%2BS.%26aulast%3DOlivas%26aufirst%3DK.%2BC.%26aulast%3DAnderson%26aufirst%3DM.%2BE.%26aulast%3DCochran%26aufirst%3DJ.%2BH.%26aulast%3DJin%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DLoftus%26aufirst%3DL.%2BV.%26aulast%3DMeyer%26aufirst%3DD.%2BW.%26aulast%3DNeale%26aufirst%3DJ.%26aulast%3DNix%26aufirst%3DJ.%2BC.%26aulast%3DPittman%26aufirst%3DP.%2BG.%26aulast%3DSimmons%26aufirst%3DJ.%2BK.%26aulast%3DUlrich%26aufirst%3DM.%2BL.%26aulast%3DWaight%26aufirst%3DA.%2BB.%26aulast%3DWong%26aufirst%3DA.%26aulast%3DZaval%26aufirst%3DM.%2BC.%26aulast%3DZeng%26aufirst%3DW.%26aulast%3DLyon%26aufirst%3DR.%2BP.%26aulast%3DSenter%26aufirst%3DP.%2BD.%26atitle%3DTargeted%2520delivery%2520of%2520cytotoxic%2520NAMPT%2520inhibitors%2520using%2520antibody-drug%2520conjugates%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2018%26volume%3D17%26spage%3D2633%26epage%3D2642%26doi%3D10.1158%2F1535-7163.MCT-18-0643" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit48e"><span><span class="NLM_label">(e) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamilton, J. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pires, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, H. W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leiske, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emmerton, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waight, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senter, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyon, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, S. C.</span></span> <span> </span><span class="NLM_article-title">Glucuronide-linked antibody-tubulysin conjugates display activity in MDR+ and heterogeneous tumor models</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1752</span>– <span class="NLM_lpage">1760</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-18-0073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48e&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1158%2F1535-7163.MCT-18-0073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48e&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=29866744" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48e&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVWrtrfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=1752-1760&author=P.+J.+Burkeauthor=J.+Z.+Hamiltonauthor=T.+A.+Piresauthor=H.+W.+H.+Laiauthor=C.+I.+Leiskeauthor=K.+K.+Emmertonauthor=A.+B.+Waightauthor=P.+D.+Senterauthor=R.+P.+Lyonauthor=S.+C.+Jeffrey&title=Glucuronide-linked+antibody-tubulysin+conjugates+display+activity+in+MDR%2B+and+heterogeneous+tumor+models&doi=10.1158%2F1535-7163.MCT-18-0073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48eR"><div class="casContent"><span class="casTitleNuber">48e</span><div class="casTitle"><span class="NLM_cas:atitle">Glucuronide-Linked Antibody-Tubulysin Conjugates Display Activity in MDR+ and Heterogeneous Tumor Models</span></div><div class="casAuthors">Burke, Patrick J.; Hamilton, Joseph Z.; Pires, Thomas A.; Lai, Holden W. H.; Leiske, Christopher I.; Emmerton, Kim K.; Waight, Andrew B.; Senter, Peter D.; Lyon, Robert P.; Jeffrey, Scott C.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1752-1760</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Although antibody-drug conjugates (ADCs) find increasing applications in cancer treatment, de novo or treatment-emergent resistance mechanisms may impair clin. benefit.  Two resistance mechanisms that emerge under prolonged exposure include upregulation of transporter proteins that confer multidrug resistance (MDR+) and loss of cognate antigen expression.  Recently, we developed the quaternary ammonium linker system to expand the scope of conjugatable payloads to include tertiary amines and applied the linker to tubulysins, a highly potent class of tubulin binders that maintain activity in MDR+ cell lines.  In this work, tubulysin M, which contains an unstable acetate susceptible to enzymic hydrolysis, and two stabilized tubulysin analogs were prepd. as quaternary ammonium-linked glucuronide-linkers and assessed as ADC payloads in preclin. models.  The conjugates were potent across a panel of cancer cell lines and active in tumor xenografts, including those displaying the MDR+ phenotype.  The ADCs also demonstrated potent bystander activity in a coculture model comprised of a mixt. of antigen-pos. and -neg. cell lines, and in an antigen-heterogeneous tumor model.  Thus, the glucuronide-tubulysin drug-linkers represent a promising ADC payload class, combining conjugate potency in the presence of the MDR+ phenotype and robust activity in models of tumor heterogeneity in a structure-dependent manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpi-liIa9oi_rVg90H21EOLACvtfcHk0lgNCXHLISq6_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVWrtrfM&md5=57268b599402601821db672b6d645936</span></div><a href="/servlet/linkout?suffix=cit48e&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-18-0073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-18-0073%26sid%3Dliteratum%253Aachs%26aulast%3DBurke%26aufirst%3DP.%2BJ.%26aulast%3DHamilton%26aufirst%3DJ.%2BZ.%26aulast%3DPires%26aufirst%3DT.%2BA.%26aulast%3DLai%26aufirst%3DH.%2BW.%2BH.%26aulast%3DLeiske%26aufirst%3DC.%2BI.%26aulast%3DEmmerton%26aufirst%3DK.%2BK.%26aulast%3DWaight%26aufirst%3DA.%2BB.%26aulast%3DSenter%26aufirst%3DP.%2BD.%26aulast%3DLyon%26aufirst%3DR.%2BP.%26aulast%3DJeffrey%26aufirst%3DS.%2BC.%26atitle%3DGlucuronide-linked%2520antibody-tubulysin%2520conjugates%2520display%2520activity%2520in%2520MDR%252B%2520and%2520heterogeneous%2520tumor%2520models%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2018%26volume%3D17%26spage%3D1752%26epage%3D1760%26doi%3D10.1158%2F1535-7163.MCT-18-0073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit48f"><span><span class="NLM_label">(f) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Ogitani, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aida, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagihara, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishii, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harada, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soma, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oitate, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arakawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atsumi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayakawa, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abe, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agatsuma, T.</span></span> <span> </span><span class="NLM_article-title">DS-8201a, a novel HER2-targeting ADC with a novel DNA topoisomerase I inhibitor, demonstrates a promising antitumor efficacy with differentiation from T-DM1</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">5097</span>– <span class="NLM_lpage">5108</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-15-2822</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48f&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=10.1158%2F1078-0432.CCR-15-2822" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48f&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=27026201" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48f&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslOhs77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2016&pages=5097-5108&author=Y.+Ogitaniauthor=T.+Aidaauthor=K.+Hagiharaauthor=J.+Yamaguchiauthor=C.+Ishiiauthor=N.+Haradaauthor=M.+Somaauthor=H.+Okamotoauthor=M.+Oitateauthor=S.+Arakawaauthor=T.+Hiraiauthor=R.+Atsumiauthor=T.+Nakadaauthor=I.+Hayakawaauthor=Y.+Abeauthor=T.+Agatsuma&title=DS-8201a%2C+a+novel+HER2-targeting+ADC+with+a+novel+DNA+topoisomerase+I+inhibitor%2C+demonstrates+a+promising+antitumor+efficacy+with+differentiation+from+T-DM1&doi=10.1158%2F1078-0432.CCR-15-2822"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48fR"><div class="casContent"><span class="casTitleNuber">48f</span><div class="casTitle"><span class="NLM_cas:atitle">DS-8201a, A Novel HER2-Targeting ADC with a Novel DNA Topoisomerase I Inhibitor, Demonstrates a Promising Antitumor Efficacy with Differentiation from T-DM1</span></div><div class="casAuthors">Ogitani, Yusuke; Aida, Tetsuo; Hagihara, Katsunobu; Yamaguchi, Junko; Ishii, Chiaki; Harada, Naoya; Soma, Masako; Okamoto, Hiromi; Oitate, Masataka; Arakawa, Shingo; Hirai, Takehiro; Atsumi, Ryo; Nakada, Takashi; Hayakawa, Ichiro; Abe, Yuki; Agatsuma, Toshinori</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5097-5108</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">An anti-HER2 antibody-drug conjugate with a novel topoisomerase I inhibitor, DS-8201a, was generated as a new antitumor drug candidate, and its preclin. pharmacol. profile was assessed.  In vitro and in vivo pharmacol. activities of DS-8201a were evaluated and compared with T-DM1 in several HER2-pos. cell lines and patient-derived xenograft (PDX) models.  The mechanism of action for the efficacy was also evaluated.  Pharmacokinetics in cynomolgus monkeys and the safety profiles in rats and cynomolgus monkeys were assessed.  DS-8201a exhibited a HER2 expression-dependent cell growth-inhibitory activity and induced tumor regression with a single dosing at more than 1 mg/kg in a HER2-pos. gastric cancer NCI-N87 model.  Binding activity to HER2 and ADCC activity of DS-8201a were comparable with unconjugated anti-HER2 antibody.  DS-8201a also showed an inhibitory activity to Akt phosphorylation.  DS-8201a induced phosphorylation of Chk1 and Histone H2A.X, the markers of DNA damage.  Pharmacokinetics and safety profiles of DS-8201a were favorable and the highest non-severely toxic dose was 30 mg/kg in cynomolgus monkeys, supporting DS-8201a as being well tolerated in humans.  DS-8201a was effective in a T-DM1-insensitive PDX model with high HER2 expression.  DS-8201a, but not T-DM1, demonstrated antitumor efficacy against several breast cancer PDX models with low HER2 expression.  DS-8201a exhibited a potent antitumor activity in a broad selection of HER2-pos. models and favorable pharmacokinetics and safety profiles.  The results demonstrate that DS-8201a will be a valuable therapy with a great potential to respond to T-DM1-insensitive HER2-pos. cancers and low HER2-expressing cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCZitEmBAqAbVg90H21EOLACvtfcHk0liA3U3BA_nPEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslOhs77F&md5=3fc9a072b2ba05c80929713c843df3f5</span></div><a href="/servlet/linkout?suffix=cit48f&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-15-2822&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-15-2822%26sid%3Dliteratum%253Aachs%26aulast%3DOgitani%26aufirst%3DY.%26aulast%3DAida%26aufirst%3DT.%26aulast%3DHagihara%26aufirst%3DK.%26aulast%3DYamaguchi%26aufirst%3DJ.%26aulast%3DIshii%26aufirst%3DC.%26aulast%3DHarada%26aufirst%3DN.%26aulast%3DSoma%26aufirst%3DM.%26aulast%3DOkamoto%26aufirst%3DH.%26aulast%3DOitate%26aufirst%3DM.%26aulast%3DArakawa%26aufirst%3DS.%26aulast%3DHirai%26aufirst%3DT.%26aulast%3DAtsumi%26aufirst%3DR.%26aulast%3DNakada%26aufirst%3DT.%26aulast%3DHayakawa%26aufirst%3DI.%26aulast%3DAbe%26aufirst%3DY.%26aulast%3DAgatsuma%26aufirst%3DT.%26atitle%3DDS-8201a%252C%2520a%2520novel%2520HER2-targeting%2520ADC%2520with%2520a%2520novel%2520DNA%2520topoisomerase%2520I%2520inhibitor%252C%2520demonstrates%2520a%2520promising%2520antitumor%2520efficacy%2520with%2520differentiation%2520from%2520T-DM1%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2016%26volume%3D22%26spage%3D5097%26epage%3D5108%26doi%3D10.1158%2F1078-0432.CCR-15-2822" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MXF" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MXF','PDB','3MXF'); return false;">PDB: 3MXF</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7KHH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7KHH','PDB','7KHH'); return false;">PDB: 7KHH</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7KHL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7KHL','PDB','7KHL'); return false;">PDB: 7KHL</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i71"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01846">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_25784"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c01846?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01846</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">PC3-S1 cell line generation and characterization, in vitro inhibition of MYC expression assay, BRD4 degradation assays (PC3-S1 line), PC3-S1 antiproliferation assays, VHL-binding assessments, IAM assessments, molecular docking studies, intracellular quantitation of compound <b>8</b>, BRD4 BD1 and BD2 binding assessments, cocrystal structure of compound <b>6</b> bound to BRD4 BD1, analysis of the BRD4/VHL/<b>9</b> ternary complex crystal structure, BRD4–VHL-degrader ternary complex assessments, whole blood stability studies, PC3-S1 BRD4-degradation/antiproliferation relationships, LNCaP antiproliferation assays, in vivo efficacy studies, ADC in vivo stability and PK data determination, in vitro human megakaryocytopoiesis model, full chemical structures of chimeric BRD4 degraders and related linker-drugs, analytical LCMS methods, synthesis of antibody–drug conjugates, ADC mechanisms of action, ADC unit conversion example (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings of <a class="ref showTableEvent internalNav" href="#tbl1">Table <a class="ref showTableEvent internalNav" href="#tbl1">1</a></a> compounds (<b>1</b>–<b>3</b>, <b>5</b>–<b>15</b>) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_002.csv">CSV</a>)</p></li><li><p class="inline">Molecular formula strings of new compounds in <a class="ref internalNav" href="#sch1">Schemes 1</a>–<a class="ref internalNav" href="#sch9">9</a> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_003.csv">CSV</a>)</p></li><li><p class="inline">Compound characterization data; HPLC and LCMS reports (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_004.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_001.pdf">jm0c01846_si_001.pdf (2.35 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_002.csv">jm0c01846_si_002.csv (7.91 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_003.csv">jm0c01846_si_003.csv (2.49 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01846/suppl_file/jm0c01846_si_004.pdf">jm0c01846_si_004.pdf (8.19 MB)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">A. Cocrystal structure of compound <b>6</b> bound to BRD4 BD1: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7KHL">7KHL</a>.B. Crystal structure of the ternary complex formed among VHL, BRD4 BD1, and compound <b>9</b>: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7KHH">7KHH</a>. Authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c01846&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-5%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c01846%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c01846" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679ab0aebf9dd1a7","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
