
<html lang="en"     class="pb-page"  data-request-id="d77ba7fc-4768-45ba-9507-276bacea9424"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";issue:issue:10.1021/jmcmar.2020.63.issue-8;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.9b02076;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Achieving In Vivo Target Depletion through the Discovery and Optimization of Benzimidazolone BCL6 Degraders" /></meta><meta name="dc.Creator" content="Benjamin R.  Bellenie" /></meta><meta name="dc.Creator" content="Kwai-Ming J.  Cheung" /></meta><meta name="dc.Creator" content="Ana  Varela" /></meta><meta name="dc.Creator" content="Olivier A.  Pierrat" /></meta><meta name="dc.Creator" content="Gavin W.  Collie" /></meta><meta name="dc.Creator" content="Gary M.  Box" /></meta><meta name="dc.Creator" content="Michael D.  Bright" /></meta><meta name="dc.Creator" content="Sharon  Gowan" /></meta><meta name="dc.Creator" content="Angela  Hayes" /></meta><meta name="dc.Creator" content="Matthew J.  Rodrigues" /></meta><meta name="dc.Creator" content="Kartika N.  Shetty" /></meta><meta name="dc.Creator" content="Michael  Carter" /></meta><meta name="dc.Creator" content="Owen A.  Davis" /></meta><meta name="dc.Creator" content="Alan T.  Henley" /></meta><meta name="dc.Creator" content="Paolo  Innocenti" /></meta><meta name="dc.Creator" content="Louise D.  Johnson" /></meta><meta name="dc.Creator" content="Manjuan  Liu" /></meta><meta name="dc.Creator" content="Selby  de Klerk" /></meta><meta name="dc.Creator" content="Yann-Vaï  Le Bihan" /></meta><meta name="dc.Creator" content="Matthew G.  Lloyd" /></meta><meta name="dc.Creator" content="P. Craig  McAndrew" /></meta><meta name="dc.Creator" content="Erald  Shehu" /></meta><meta name="dc.Creator" content="Rachel  Talbot" /></meta><meta name="dc.Creator" content="Hannah L.  Woodward" /></meta><meta name="dc.Creator" content="Rosemary  Burke" /></meta><meta name="dc.Creator" content="Vladimir  Kirkin" /></meta><meta name="dc.Creator" content="Rob L. M.  van Montfort" /></meta><meta name="dc.Creator" content="Florence I.  Raynaud" /></meta><meta name="dc.Creator" content="Olivia W.  Rossanese" /></meta><meta name="dc.Creator" content="Swen  Hoelder" /></meta><meta name="dc.Description" content="Deregulation of the transcriptional repressor BCL6 enables tumorigenesis of germinal center B-cells, and hence BCL6 has been proposed as a therapeutic target for the treatment of diffuse large B-ce..." /></meta><meta name="Description" content="Deregulation of the transcriptional repressor BCL6 enables tumorigenesis of germinal center B-cells, and hence BCL6 has been proposed as a therapeutic target for the treatment of diffuse large B-ce..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="April 10, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b02076" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b02076" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b02076" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b02076" /></link>
        
    
    

<title>Achieving In Vivo Target Depletion through the Discovery and Optimization of Benzimidazolone BCL6 Degraders | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b02076" /></meta><meta property="og:title" content="Achieving In Vivo Target Depletion through the Discovery and Optimization of Benzimidazolone BCL6 Degraders" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0022.jpeg" /></meta><meta property="og:description" content="Deregulation of the transcriptional repressor BCL6 enables tumorigenesis of germinal center B-cells, and hence BCL6 has been proposed as a therapeutic target for the treatment of diffuse large B-cell lymphoma (DLBCL). Herein we report the discovery of a series of benzimidazolone inhibitors of the protein–protein interaction between BCL6 and its co-repressors. A subset of these inhibitors were found to cause rapid degradation of BCL6, and optimization of pharmacokinetic properties led to the discovery of 5-((5-chloro-2-((3R,5S)-4,4-difluoro-3,5-dimethylpiperidin-1-yl)pyrimidin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2H-benzo[d]imidazol-2-one (CCT369260), which reduces BCL6 levels in a lymphoma xenograft mouse model following oral dosing." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b02076"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b02076">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b02076&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b02076&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b02076&amp;href=/doi/10.1021/acs.jmedchem.9b02076" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 8</span><span class="cit-fg-pageRange">, 4047-4068</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/8" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b02039" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b02083" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Achieving <i>In Vivo</i> Target Depletion through the Discovery and Optimization of Benzimidazolone BCL6 Degraders</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Benjamin R. Bellenie</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Benjamin R. Bellenie</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SM2 5NG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Benjamin+R.++Bellenie">Benjamin R. Bellenie</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9987-3079" title="Orcid link">http://orcid.org/0000-0001-9987-3079</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kwai-Ming J. Cheung</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kwai-Ming J. Cheung</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SM2 5NG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kwai-Ming+J.++Cheung">Kwai-Ming J. Cheung</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ana Varela</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ana Varela</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SM2 5NG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ana++Varela">Ana Varela</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Olivier A. Pierrat</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Olivier A. Pierrat</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SM2 5NG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Olivier+A.++Pierrat">Olivier A. Pierrat</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Gavin W. Collie</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gavin W. Collie</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit,  Division of Structural Biology, The Institute of Cancer Research, London SM2 5NG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gavin+W.++Collie">Gavin W. Collie</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0406-922X" title="Orcid link">http://orcid.org/0000-0002-0406-922X</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Gary M. Box</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gary M. Box</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SM2 5NG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gary+M.++Box">Gary M. Box</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Michael D. Bright</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Michael D. Bright</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SM2 5NG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Michael+D.++Bright">Michael D. Bright</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sharon Gowan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sharon Gowan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SM2 5NG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sharon++Gowan">Sharon Gowan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Angela Hayes</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Angela Hayes</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SM2 5NG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Angela++Hayes">Angela Hayes</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Matthew J. Rodrigues</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Matthew J. Rodrigues</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit,  Division of Structural Biology, The Institute of Cancer Research, London SM2 5NG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Matthew+J.++Rodrigues">Matthew J. Rodrigues</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kartika N. Shetty</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kartika N. Shetty</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit,  Division of Structural Biology, The Institute of Cancer Research, London SM2 5NG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kartika+N.++Shetty">Kartika N. Shetty</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Michael Carter</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Michael Carter</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SM2 5NG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Michael++Carter">Michael Carter</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Owen A. Davis</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Owen A. Davis</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SM2 5NG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Owen+A.++Davis">Owen A. Davis</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Alan T. Henley</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alan T. Henley</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SM2 5NG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alan+T.++Henley">Alan T. Henley</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Paolo Innocenti</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Paolo Innocenti</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SM2 5NG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Paolo++Innocenti">Paolo Innocenti</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Louise D. Johnson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Louise D. Johnson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SM2 5NG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Louise+D.++Johnson">Louise D. Johnson</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Manjuan Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Manjuan Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SM2 5NG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Manjuan++Liu">Manjuan Liu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Selby de Klerk</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Selby de Klerk</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SM2 5NG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Selby++de+Klerk">Selby de Klerk</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yann-Vaï Le Bihan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yann-Vaï Le Bihan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SM2 5NG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yann-Va%C3%AF++Le+Bihan">Yann-Vaï Le Bihan</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6850-9706" title="Orcid link">http://orcid.org/0000-0002-6850-9706</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Matthew G. Lloyd</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Matthew G. Lloyd</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SM2 5NG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Matthew+G.++Lloyd">Matthew G. Lloyd</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">P. Craig McAndrew</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">P. Craig McAndrew</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SM2 5NG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=P.+Craig++McAndrew">P. Craig McAndrew</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Erald Shehu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Erald Shehu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SM2 5NG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Erald++Shehu">Erald Shehu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rachel Talbot</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rachel Talbot</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SM2 5NG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rachel++Talbot">Rachel Talbot</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hannah L. Woodward</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hannah L. Woodward</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SM2 5NG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hannah+L.++Woodward">Hannah L. Woodward</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8429-2546" title="Orcid link">http://orcid.org/0000-0001-8429-2546</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rosemary Burke</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rosemary Burke</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SM2 5NG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rosemary++Burke">Rosemary Burke</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Vladimir Kirkin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Vladimir Kirkin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SM2 5NG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Vladimir++Kirkin">Vladimir Kirkin</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rob L. M. van Montfort</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rob L. M. van Montfort</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit,  Division of Structural Biology, The Institute of Cancer Research, London SM2 5NG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rob+L.+M.++van+Montfort">Rob L. M. van Montfort</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Florence I. Raynaud</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Florence I. Raynaud</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SM2 5NG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Florence+I.++Raynaud">Florence I. Raynaud</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Olivia W. Rossanese</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Olivia W. Rossanese</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SM2 5NG, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Olivia+W.++Rossanese">Olivia W. Rossanese</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Swen Hoelder</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Swen Hoelder</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London SM2 5NG, U.K.</div></div><span class="conrtib-corresp"><strong>*</strong>S.W.: email, <a href="/cdn-cgi/l/email-protection#cebda6a1aba2aaabbc8ea7adbce0afade0bba5"><span class="__cf_email__" data-cfemail="63100b0c060f070611230a00114d02004d1608">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Swen++Hoelder">Swen Hoelder</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8636-1488" title="Orcid link">http://orcid.org/0000-0001-8636-1488</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b02076&amp;href=/doi/10.1021%2Facs.jmedchem.9b02076" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 8</span><span class="cit-pageRange">, 4047–4068</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">April 10, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>13 December 2019</li><li><span class="item_label"><b>Published</b> online</span>10 April 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 23 April 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b02076" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b02076</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice_termsofuse.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b02076"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">7679</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">9</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b02076" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Achieving In Vivo Target Depletion through the Discovery and Optimization of Benzimidazolone BCL6 Degraders&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Benjamin&quot;,&quot;last_name&quot;:&quot;R. Bellenie&quot;},{&quot;first_name&quot;:&quot;Kwai-Ming&quot;,&quot;last_name&quot;:&quot;J. Cheung&quot;},{&quot;first_name&quot;:&quot;Ana&quot;,&quot;last_name&quot;:&quot;Varela&quot;},{&quot;first_name&quot;:&quot;Olivier&quot;,&quot;last_name&quot;:&quot;A. Pierrat&quot;},{&quot;first_name&quot;:&quot;Gavin&quot;,&quot;last_name&quot;:&quot;W. Collie&quot;},{&quot;first_name&quot;:&quot;Gary&quot;,&quot;last_name&quot;:&quot;M. Box&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;D. Bright&quot;},{&quot;first_name&quot;:&quot;Sharon&quot;,&quot;last_name&quot;:&quot;Gowan&quot;},{&quot;first_name&quot;:&quot;Angela&quot;,&quot;last_name&quot;:&quot;Hayes&quot;},{&quot;first_name&quot;:&quot;Matthew&quot;,&quot;last_name&quot;:&quot;J. Rodrigues&quot;},{&quot;first_name&quot;:&quot;Kartika&quot;,&quot;last_name&quot;:&quot;N. Shetty&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;Carter&quot;},{&quot;first_name&quot;:&quot;Owen&quot;,&quot;last_name&quot;:&quot;A. Davis&quot;},{&quot;first_name&quot;:&quot;Alan&quot;,&quot;last_name&quot;:&quot;T. Henley&quot;},{&quot;first_name&quot;:&quot;Paolo&quot;,&quot;last_name&quot;:&quot;Innocenti&quot;},{&quot;first_name&quot;:&quot;Louise&quot;,&quot;last_name&quot;:&quot;D. Johnson&quot;},{&quot;first_name&quot;:&quot;Manjuan&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Selby&quot;,&quot;last_name&quot;:&quot;de Klerk&quot;},{&quot;first_name&quot;:&quot;Yann-Vaï&quot;,&quot;last_name&quot;:&quot;Le Bihan&quot;},{&quot;first_name&quot;:&quot;Matthew&quot;,&quot;last_name&quot;:&quot;G. Lloyd&quot;},{&quot;first_name&quot;:&quot;P.&quot;,&quot;last_name&quot;:&quot;Craig McAndrew&quot;},{&quot;first_name&quot;:&quot;Erald&quot;,&quot;last_name&quot;:&quot;Shehu&quot;},{&quot;first_name&quot;:&quot;Rachel&quot;,&quot;last_name&quot;:&quot;Talbot&quot;},{&quot;first_name&quot;:&quot;Hannah&quot;,&quot;last_name&quot;:&quot;L. Woodward&quot;},{&quot;first_name&quot;:&quot;Rosemary&quot;,&quot;last_name&quot;:&quot;Burke&quot;},{&quot;first_name&quot;:&quot;Vladimir&quot;,&quot;last_name&quot;:&quot;Kirkin&quot;},{&quot;first_name&quot;:&quot;Rob&quot;,&quot;last_name&quot;:&quot;L. M. van Montfort&quot;},{&quot;first_name&quot;:&quot;Florence&quot;,&quot;last_name&quot;:&quot;I. Raynaud&quot;},{&quot;first_name&quot;:&quot;Olivia&quot;,&quot;last_name&quot;:&quot;W. Rossanese&quot;},{&quot;first_name&quot;:&quot;Swen&quot;,&quot;last_name&quot;:&quot;Hoelder&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;04&quot;,&quot;day&quot;:&quot;10&quot;,&quot;issue&quot;:&quot;8&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;4047-4068&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b02076&quot;},&quot;abstract&quot;:&quot;Deregulation of the transcriptional repressor BCL6 enables tumorigenesis of germinal center B-cells, and hence BCL6 has been proposed as a therapeutic target for the treatment of diffuse large B-cell lymphoma (DLBCL). Herein we report the discovery of a series of benzimidazolone inhibitors of the protein–protein interaction between BCL6 and its co-repressors. A subset of these inhibitors were found to cause rapid degradation of BCL6, and optimization of pharmacokinetic properties led to the discovery of 5-((5-chloro-2-((3R,5S)-4,4-difluoro-3,5-dimethylpiperidin-1-yl)pyrimidin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2H-benzo[d]imidazol-2-one (CCT369260), which reduces BCL6 levels in a lymphoma xenograft mouse model following oral dosing.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b02076&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b02076" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b02076&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b02076" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b02076&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b02076" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b02076&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b02076&amp;href=/doi/10.1021/acs.jmedchem.9b02076" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b02076" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b02076" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (10 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b02076&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b02076%26sid%3Dliteratum%253Aachs%26pmid%3D32275432%26genre%3Darticle%26aulast%3DBellenie%26date%3D2020%26atitle%3DAchieving%2BIn%2BVivo%2BTarget%2BDepletion%2Bthrough%2Bthe%2BDiscovery%2Band%2BOptimization%2Bof%2BBenzimidazolone%2BBCL6%2BDegraders%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D8%26spage%3D4047%26epage%3D4068%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291316" title="Mixtures">Mixtures</a>,</li><li><a href="/action/doSearch?ConceptID=291781" title="Degradation">Degradation</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=291454" title="Solubility">Solubility</a>,</li><li><a href="/action/doSearch?ConceptID=291754" title="Molecules">Molecules</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/8" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/jmcmar.2020.63.issue-8/20200423/jmcmar.2020.63.issue-8.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/medium/jm9b02076_0022.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0022.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02076&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Deregulation of the transcriptional repressor BCL6 enables tumorigenesis of germinal center B-cells, and hence BCL6 has been proposed as a therapeutic target for the treatment of diffuse large B-cell lymphoma (DLBCL). Herein we report the discovery of a series of benzimidazolone inhibitors of the protein–protein interaction between BCL6 and its co-repressors. A subset of these inhibitors were found to cause rapid degradation of BCL6, and optimization of pharmacokinetic properties led to the discovery of 5-((5-chloro-2-((3<i>R</i>,5<i>S</i>)-4,4-difluoro-3,5-dimethylpiperidin-1-yl)pyrimidin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (CCT369260), which reduces BCL6 levels in a lymphoma xenograft mouse model following oral dosing.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_28386" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_28386" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">BCL6 (B-cell lymphoma 6 protein) is a transcriptional repressor that plays a key role in the formation and maintenance of germinal centers during the process of antibody affinity maturation.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1−3)</a> By binding to DNA via its zinc fingers and recruiting one of its co-repressors (NCoR, SMRT, or BCOR) to its dimeric BTB domain, BCL6 represses genes involved in cell cycle control, cell death, differentiation, and the DNA damage response. This action enables B-cells in the germinal center to proliferate rapidly, evade growth checkpoint controls, and tolerate high levels of DNA damage. These features are required for the process of somatic hypermutation of antibodies but are also among the hallmarks of cancer;<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4,5)</a> hence deregulation of BCL6 can lead to lymphomagenesis. Most B-cell lymphomas arise from germinal center B-cells, which are dependent on continued expression of BCL6 for survival.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Recruitment of co-repressors is essential for the repressive and oncogenic functions of BCL6, and disruption of this interaction is sufficient to inhibit lymphoma cell growth.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a> We sought to discover compounds that disrupt the protein–protein interaction (PPI) between the BTB domain of BCL6 and its co-repressors to alleviate BCL6-mediated gene repression, inhibit lymphoma cell growth, and identify new treatments for BCL6-driven lymphomas. Following on from earlier publications that identified small molecule or peptidomimetic compounds with moderate potency against BCL6,<a onclick="showRef(event, 'ref8 ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref10">(8−10)</a> the recent publication of high-potency small molecules has established the ligandability of the BCL6 BTB domain dimer.<a onclick="showRef(event, 'ref11 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref13">(11−13)</a> Proteolysis targeting chimeras (PROTAC)<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> and non-PROTAC degrader compounds that trigger the proteosomal degradation of BCL6<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> have also been reported. However, to date, no compound has been reported to be suitable for use as an <i>in vivo</i> probe to investigate the effect of degradation of BCL6. In this study we report the discovery not only of compounds that can potently inhibit BCL6 function by disrupting the PPI with its co-repressors but also of a subset of molecules that trigger the rapid degradation of BCL6. Further optimization of this series led to compound <b>1</b> (CCT369260), which shows degradation of tumoral BCL6 <i>in vivo</i> following oral dosing in a lymphoma xenograft mouse model.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63041" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63041" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Chemistry: Synthesis of Compounds</h3><div class="NLM_p">Initial hit compounds <b>2</b> and <b>3</b> and follow-up compound <b>4</b> were obtained from commercial vendors, while the cyclopropyl compound <b>5</b> and benzimidazolone <b>6</b> (CCT365386) were prepared by single step nucleophilic aromatic substitution (S<sub>N</sub>Ar) reaction from available building blocks. Compounds with alkyl substituents in the benzimidazolone-N<sup>3</sup> position could be synthesized by stepwise alkylation of nitro-compound <b>7</b>, reduction, and S<sub>N</sub>Ar reaction, as exemplified in the preparation of <b>11a</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). In order to facilitate the optimization of this position (as shown in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>), we developed conditions to allow the addition of this group as the final step (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>, steps d and e), using intermediate <b>9</b>. The use of alkyl halides or epoxides as electrophiles under these conditions gave primarily the desired regioisomer, although in most cases minor products corresponding to alkylation at the 4-pyridylamine were also observed in the reaction mixture and required HPLC purification to separate.</div><figure id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/medium/jm9b02076_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0009.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Final Step Diversification of Benzimidazolone N<sup>3</sup> Position<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02076&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 2-ethyloxirane, cesium carbonate, DMF, 120 °C, 1 h; (b) sodium dithionite, ethanol/DMSO, rt to 90 °C; (c) 2,4-dichloropyridine-3-carbonitrile, DIPEA, DMA, 120 °C, 30–45 min; (d) (2<i>R</i>)- or (2<i>S</i>)-ethyloxirane, cesium carbonate, DMF, 120 °C, 1 h, then 140 °C, 1 h; (e) alkyl bromide or tosylate, cesium carbonate, DMF, 60 °C, 3 days or 120–140 °C, 1 h.</p></p></figure><div class="NLM_p">In order to vary the substituent of the 5-amino group on the benzimidazolone (labeled as R<sup>4</sup> in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>), it was necessary first to scale up the synthesis of the 1,3-disubstituted 5-aminobenzimidazol-2-one. Cyclization of commercially available nitrodianiline <b>12</b> using disuccinimidylcarbonate proceeded in high yield and was followed by alkylation with tosylate <b>14a</b>, then hydrogenation to reduce the nitro group and form aniline <b>16</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). These steps were readily amenable to scale up, and multigram quantities were produced without the need for column chromatography.</div><figure id="sch2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/medium/jm9b02076_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0010.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Key Intermediate <b>16</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02076&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) disuccinimidylcarbonate, MeCN, rt, 18 h, up to 11g scale, 86–99% yield; (b) cesium carbonate, MeCN, reflux, 4 h, 12 g scale, 92% yield; (c) TsCl, NEt<sub>3</sub>, DCM, 0 °C to rt, up to 40g scale, 60–72% yield; (d) 10% Pd/C, 1 atm H<sub>2</sub>, ethanol, 60 °C, 3 h, 6 g scale, quant.</p></p></figure><div class="NLM_p">Aromatic nucleophilic substitution or (for less reactive pyridine substituents) palladium coupling reactions were used to prepare the set of compounds shown in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>).</div><figure id="sch3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/medium/jm9b02076_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0011.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds from <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>, Investigating Different Substitution Patterns on Pyrimidine or Pyridine Rings<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02076&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) heteroaryl chloride, DIPEA, NMP or DMF, 80–120 °C, 30 min; (b) 4-bromo- or iodopyridine, Xantphos, Pd<sub>2</sub>(dba)<sub>3</sub>, cesium carbonate, NMP/toluene, 140 °C, 1 h.</p></p></figure><div class="NLM_p">In order to explore substitution at the pyrimidine-2-position (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Tables <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a>), dichloropyrimidine <b>18</b> was prepared on multigram scale. Final compounds were prepared from this, again via S<sub>N</sub>Ar or metal-catalyzed reactions (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>).</div><figure id="sch4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/medium/jm9b02076_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0012.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of 2-Substituted Pyrimidine Compounds from <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Tables <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a>, Starting from Common Intermediate <b>18</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02076&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) dimethylamine HCl, cesium carbonate, NMP, 180 °C, 1 h; (b) cyclic amine, DIPEA, NMP, 140–180 °C, 1–2 h; (c) pyrazole, cesium carbonate, NMP, 170 °C, 1 h; (d) 2,4-dimethyl-5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)thiazole, sodium carbonate, PdCl<sub>2</sub>(PPh<sub>3</sub>)<sub>2</sub>, 1,4-dioxane/water, 130 °C, 30 min; (e) 1-methyl-2-(tributylstannyl)-1<i>H</i>-imidazole, PdCl<sub>2</sub>(PPh<sub>3</sub>)<sub>2</sub>, 1,4-dioxane, 90 °C, 18 h; (f) bicyclic amine, DIPEA, NMP, 140 °C, 2–8 h.</p></p></figure><div class="NLM_p">The reduced reactivity of the pyridine core in comparison to the pyrimidine meant that metal-catalyzed reactions on bromo- or iodopyridines or S<sub>N</sub>Ar reactions using 2-fluoropyridines were used to prepare the pyridyl examples in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a> (<a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>).</div><figure id="sch5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/medium/jm9b02076_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0013.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of 2,4,5-Substituted Pyridine Derivatives<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02076&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>16,</b> DIPEA, NMP, 180 °C, 1 h; (b) 3,5-dimethylpyrazole, Xantphos, cesium carbonate. Pd<sub>2</sub>(dba)<sub>3</sub>, DMF/toluene, 140 °C, 1 h; (c) DIPEA, THF, 100 °C, 16 h; (d) <b>16,</b> Xantphos, Pd<sub>2</sub>(dba)<sub>3</sub>, cesium carbonate, DMF/toluene, 80 °C, 1 h.</p></p></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Hit Discovery</h3><div class="NLM_p">A detailed description of our hit discovery campaign for BCL6 will be presented elsewhere.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Briefly, we identified hit compounds <b>2</b> and <b>3</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) from an in-house high-throughput screen using a fluorescence polarization (FP) assay, based on the displacement of a fluorescently labeled peptide derived from the BCOR co-repressor.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> In common with other hits from this screen, these compounds showed weak potency (IC<sub>50</sub> ∼100 μM). Initial attempts to obtain ligand-bound X-ray structures were hindered by solubility which was lower than their biochemical assay activity (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). To address this, we made analogues with reduced lipophilicity and fewer aromatic rings. Compounds <b>4</b> and <b>5</b> demonstrated comparable potency to initial hits and improved solubility (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), enabling X-ray structure determination.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Like the BCL6 co-repressors SMRT and BCOR,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> compounds were found to bind at the dimer interface of the BTB domain of BCL6 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A and <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B). Two interaction features are common to both <b>4</b> and <b>5</b>: a hydrogen bond from an NH to the backbone carbonyl of Met51 and the intercalation of their cyanopyridine moieties into a cleft between Tyr58 and Asn21 on the protein surface, creating a critical hydrophobic interaction between the cyanopyridine and the side chain of Tyr58. Compound <b>4</b> forms additional interactions via the oxygen atom of its benzimidazolone core to the backbone N–H of Glu115 and to the backbone N–H of His116 via a mediating water molecule. The cyclopropyl group of <b>5</b> makes only weak hydrophobic interactions with the backbone of residues 53–55 (in blue in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>, also observed for the benzimidazole of <b>4</b>), suggesting that the chlorocyanopyridine of <b>5</b> provides much of the binding affinity of this compound. We therefore hypothesized that combining the benzimidazolone core with the chlorocyanopyridine of <b>5</b> would lead to an increase in potency. Gratifyingly, the resulting compound <b>6</b> (CCT365386) showed a significant improvement in binding affinity, down to ∼10 μM (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Potency and Solubility Data for Hit and Early Lead Compounds</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/medium/jm9b02076_0014.gif" alt="" id="gr6" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Data represent the geometric mean. See <a href="/doi/suppl/10.1021/acs.jmedchem.9b02076/suppl_file/jm9b02076_si_001.pdf" class="ext-link">Supporting Information Table S1</a> for full statistics. Potency was later confirmed in the TR-FRET assay described in the manuscript; see <a href="/doi/suppl/10.1021/acs.jmedchem.9b02076/suppl_file/jm9b02076_si_001.pdf" class="ext-link">Supporting Information Table S2</a>.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">Kinetic solubility measured by NMR in HEPES buffer (containing 5% DMSO) at pH 8. LE calculated as 1.4 × pIC<sub>50</sub>/HAC.</p></div></div><div></div></div><figure id="fig1" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/medium/jm9b02076_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0001.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structure-guided merging of <b>5</b> (A, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOG">6TOG</a>) and <b>4</b> (B, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOF">6TOF</a>) yielded the more potent <b>6</b> (CCT365386) (C, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOH">6TOH</a>). The surface of the BCL6 dimer is shown as a grey transparent surface. The two individual monomers are highlighted in gray and cyan ribbons except for residues 53–55 which are indicated in blue. Key protein residues are shown in line representation. The compounds are shown as orange ball and sticks, a selected water molecule is shown as a red sphere, and H-bonds are shown as yellow dashed lines. In panel C the orange arrow indicates an exit vector towards a large accessible pocket close to our compounds. Additional images containing overlays of <b>4</b> and <b>5</b> with <b>6</b> are shown in the <a href="/doi/suppl/10.1021/acs.jmedchem.9b02076/suppl_file/jm9b02076_si_001.pdf" class="ext-link">Supporting Information Figure S1</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02076&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The X-ray structure of <b>6</b> bound to BCL6 confirmed the hypothesized binding mode, combining the features of <b>4</b> and <b>5</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C). Low microsomal clearance and high PAMPA permeability, combined with good ligand efficiency for a protein–protein interaction inhibitor hit compound (LE = 0.32 based on the FP assay) further established <b>6</b> as a promising hit compound for optimization. Despite these good properties, the suboptimal biochemical activity and solubility meant that we did not observe significant displacement of co-repressor SMRT from BCL6 in the NanoBRET cellular target engagement assay (IC<sub>50</sub> > 30 μM).<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Initial Hit Optimization</h3><div class="NLM_p">Having discovered an attractive starting point, our next objective was to improve the biochemical potency. To avoid being constrained by the tight binding limit of the FP assay, we developed new TR-FRET assay conditions with reduced protein concentration (from 3 μM used in the FP assay to 1 nM), allowing us to measure compound activities in the nM range.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> From the X-ray structure of <b>6</b>, we identified two main areas for optimization: first, growing from the benzimidazolone-N<sup>3</sup> position into a relatively large cavity, indicated by the arrow in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C, and second, by further optimization of the interactions between the pyridine and the pocket formed by Tyr58.</div><div class="NLM_p">Our aim was to gain potency by extending from the N<sup>3</sup> position and filling the pocket while ensuring that hydrogen bond donor or acceptor groups in the binding site were appropriately satisfied. In our design, we sought to exploit a hydrophobic patch close to Val18 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C) with a <i>n-</i>butyl chain. This chain would pass close to Asn21, so we incorporated an alcohol (donor) or nitrile (acceptor) group to satisfy this interaction, which we hypothesized would also address the moderate solubility. However, the resulting compounds <b>11a</b> and <b>11b</b> showed no improvement in potency (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). The X-ray structure of <b>11a</b> shows both enantiomers bound to BCL6 with similar occupancies (0.46 and 0.54 for enantiomers (<i>S</i>) and (<i>R</i>), respectively, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). We hypothesized that the weak (3.25 Å) hydrogen bond between the hydroxyl of the (<i>S</i>) enantiomer of <b>11a</b> and the side chain of Asn21 is not sufficient to outweigh the desolvation penalty<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18,19)</a> and is hence not contributing positively to binding. This was supported by testing of single enantiomers <b>11c</b> and <b>11d</b> and des-hydroxy <b>11e</b>, which all have comparable potency.</div><figure id="fig2" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/medium/jm9b02076_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0002.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Binding mode of enantiomers of <b>11a</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOO">6TOO</a>). Panel A and panel B show the binding mode of the respective (<i>S</i>)- and (<i>R</i>)-enantiomers of <b>11a</b>. Panel C shows the (<i>S</i>)-enantiomer and highlights the four water molecules targeted to gain potency. In all panels the surface of the BCL6 dimer is shown as a gray transparent surface, with the two individual monomers highlighted as ribbons and colored in gray and cyan, respectively. Selected residues are shown in line representation. Enantiomers of <b>11a</b> are shown as orange ball and sticks, selected water molecules as a red spheres, and H-bonds as yellow dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02076&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Structure-Activity Relationships of the Benzimidazolone-N<sup>3</sup> Position (Labeled as R<sup>3</sup> Group)</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/medium/jm9b02076_0015.gif" alt="" id="gr10" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/medium/jm9b02076_0025.gif" alt="" id="GRAPHIC-d7e1000-autogenerated" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Data represent the geometric mean of at least three replicates. See <a href="/doi/suppl/10.1021/acs.jmedchem.9b02076/suppl_file/jm9b02076_si_001.pdf" class="ext-link">Supporting Information Table S2</a> for full statistics.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">Kinetic solubility measured by NMR in HEPES buffer (containing 5% DMSO) at pH 8. Where this was <30 μM, solubility by HPLC in PBS buffer and 1% DMSO at pH 7.4 was also measured and is shown in parentheses.</p></div></div><div></div></div><div class="NLM_p">We sought instead to interact indirectly with BCL6 via the network of water molecules (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C) which were observed in the <b>11a</b>-bound crystal structure, by changing the position of the hydroxyl group from the C-2 to C-3 carbon of the butyl chain. Compound <b>11f</b> showed a 3-fold improvement in potency (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>), and a crystal structure of <b>11f</b> bound to BCL6 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A) confirmed the expected binding mode with interactions to three water molecules. The addition of a methyl group to form the achiral tertiary alcohol <b>17a</b> provided a further improvement in potency, consistent with its increase in lipophilicity, while maintaining the binding mode to BCL6 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B).</div><figure id="fig3" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/medium/jm9b02076_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0003.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Binding mode of <b>11f</b> and <b>17a</b>. Panel A and Panel B show the respective BCL6 binding modes of <b>11f</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOI">6TOI</a>) and <b>17a</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOJ">6TOJ</a>). Compounds are shown as orange ball and sticks, selected water molecules as red spheres, and H-bonds as yellow dashed lines. The surface of BCL6 dimer is shown as a gray transparent surface. The two individual monomers are displayed as ribbons and colored in gray and cyan, respectively. Key residues are shown as lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02076&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p"><b>17a</b> represented our first submicromolar compound. Further characterization in the NanoBRET assay demonstrated that <b>17a</b> is able to disrupt the protein–protein interaction between full-length co-repressor and BCL6 in cells, albeit with a ∼10-fold drop-off in activity (NanoBRET IC<sub>50</sub> = 8.8 μM, compared with TR-FRET IC<sub>50</sub> = 0.86 μM). To gain further binding affinity and hence improve cellular potency, we explored modifications to the pyridine ring (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). The 3-nitrile group points into a small lipophilic pocket, does not appear to be making polar interactions, and could be replaced by a chloro atom (<b>17b</b>). We also assessed whether the potentially reactive 2-chloropyridine in <b>17b</b> could be removed and found that resulting compound <b>17c</b> maintains activity and improves solubility.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Structure–Activity Relationships of the R<sup>4</sup>-Group-Substituted Pyridine and Pyrimidines</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/medium/jm9b02076_0016.gif" alt="" id="gr13" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/medium/jm9b02076_0026.gif" alt="" id="GRAPHIC-d7e1109-autogenerated" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Data represent the geometric mean of at least three replicates. See <a href="/doi/suppl/10.1021/acs.jmedchem.9b02076/suppl_file/jm9b02076_si_001.pdf" class="ext-link">Supporting Information Table S2</a> for full statistics.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">Kinetic solubility measured by NMR in HEPES buffer (containing 5% DMSO) at pH 8. Where this was <30 μM, solubility by HPLC in PBS buffer and 1% DMSO at pH 7.4 was measured and is shown in parentheses.</p></div></div><div></div></div><div class="NLM_p">The crystal structure of BCL6 bound to <b>17a</b> shows the pyridine ring making hydrophobic contacts with the electron rich π-system of Tyr58, and the pyridine nitrogen engaging in a water-mediated H-bond with Arg28 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). We hypothesized that a pyrimidine ring could maintain both these features, and indeed <b>17d</b> and <b>17e</b> showed comparable activity to their pyridine equivalents.</div><div class="NLM_p">Further functionalization of the pyrimidine or pyridine ring was explored in order to gain potency. The pyridine 2-position offers only limited scope for further substitution; the 2-chloro group in <b>17a</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>) is largely filling the available space in this region. Substitution in the pyridine 5-position could give rise to a steric clash with the phenyl ring of the benzimidazolone, preventing the molecule from adopting the binding conformation observed crystallographically: for example, addition of a chlorine at this position in <b>17f</b> leads to a 37-fold drop in activity compared to parent <b>17c</b>. The remaining position (the 2-position on the pyrimidine) was hence selected for further exploration. Initial examples <b>17g</b>, <b>18</b>, and <b>19</b> showed that substitution was tolerated, and a set of 2-heterocyclyl- and heteroarylpyrimidines was therefore prepared to probe this region (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). A variety of groups including aliphatic amines and both carbon- or nitrogen-linked heteroaryl groups were well tolerated in this position, consistent with the space available in this solvent-exposed region of the binding site, with hydrophilic groups such as morpholine <b>20a</b> showing 4-fold better potency than the more hydrophobic piperidine <b>20b</b>.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Structure–Activity Relationships of the Pyrimidine and Pyridine 2-Position (Labeled as R<sup>3</sup> Group)</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/medium/jm9b02076_0017.gif" alt="" id="gr16" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/medium/jm9b02076_0018.gif" alt="" id="gr17" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">Data represent the geometric mean of at least three replicates. See <a href="/doi/suppl/10.1021/acs.jmedchem.9b02076/suppl_file/jm9b02076_si_001.pdf" class="ext-link">Supporting Information Table S2</a> for full statistics.</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">Data represent the geometric mean of at least two replicates except where superscript 1 is indicated. n.d. = not done.</p></div><div class="footnote" id="t4fn3"><sup><sup>c</sup></sup><p class="last">Kinetic solubility measured by NMR in HEPES buffer and 5% DMSO at pH 8, and/or where data are shown in brackets, by HPLC in PBS buffer and 1% DMSO at pH 7.4. <sup>#</sup>These compounds showed <10% degradation of BCL6 in single concentration experiments at 10 μM, <i>n</i> = 2.</p></div></div><div></div></div><div class="NLM_p">We attempted to increase potency further by increasing steric bulk to more completely fill this area of the pocket and derive additional hydrophobic surface contacts. Addition of one or two methyl groups to pyrazole (compare <b>23a</b>, <b>23b</b>, <b>23c</b>, and <b>23d</b>), piperidine (compare <b>20b</b>, <b>24a</b>, and <b>24b</b>) and morpholine (compare <b>20a</b> and <b>25a</b>) subseries did indeed lead to an increase in potency, as measured in the TR-FRET assay. For the most potent examples, pyridine matched-pairs were prepared but were less potent than their pyrimidine counterparts (compare <b>23d</b> and <b>26a</b>, or <b>24b</b> and <b>26b</b>, or <b>25a</b> and <b>26c</b>, <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>), in contrast to the matched-pairs for simpler analogues (compare <b>17a</b> and <b>17d</b>, and <b>17b</b> and <b>17e</b>; shown in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). This may be due to the different conformational preferences of the added substituent: the additional steric clash resulting from the pyridyl CH compared to the pyrimidine N is likely to lead to a more twisted minimum energy conformation. Compounds with submicromolar activity in TR-FRET were profiled in the cellular NanoBRET assay and showed inhibition in the low micromolar range. Further substitution on the morpholine moiety led to <b>25b</b>, which was our first compound to show both sub-100 nM activity in the TR-FRET assay and submicromolar activity in the cellular NanoBRET assay.</div><div class="NLM_p">X-ray structures obtained for <b>23d</b> and <b>25b</b> provided possible explanations for the observed increases in potency. The pyrazole group of <b>23d</b> forms new interactions: a possible cation−π stacking interaction with Arg24, and a hydrogen bond from one of the pyrazole nitrogen atoms to Arg28, which adopts an alternative conformation (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C and <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D). In contrast, no new polar interactions were observed for <b>25b</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A and <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B), which adopts the same binding mode as parent <b>17a</b>. We hypothesize that the observed potency enhancement results from the improved hydrophobic contacts with Tyr58 and the displacement of water molecules in this region (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A and <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B).</div><figure id="fig4" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/medium/jm9b02076_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0004.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Binding mode of <b>25b</b> and <b>23d</b>. Panels A and B show two different views of the binding mode of 25b (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TON">6TON</a>). Panels C and D show two views of the binding mode of <b>23d</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOK">6TOK</a>). The panels show the changed conformation of Arg28 and the new interaction with one of the compound’s pyrazole nitrogen atoms. In all panels, the surface of the BCL6 BTB dimer is shown as a gray transparent surface, with the two individual monomers displayed as ribbons and colored in gray and cyan. Key residues are shown as lines. Compounds are shown as orange ball and sticks, selected water molecules as red spheres, and H-bonds as yellow dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02076&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Discovery of BCL6 Degraders</h3><div class="NLM_p">In contrast to the morpholine analogues <b>25a</b> and <b>25b</b>, we were surprised to see no cellular activity for dimethylpiperidine <b>24b</b> (CCT368682) in the NanoBRET assay, despite submicromolar potency in the TR-FRET assay, and good passive permeability (PAMPA <i>P</i><sub>app</sub> is high [36 × 10<sup>–6</sup> cm/s]). A large drop-off from TR-FRET to NanoBRET potencies was also observed for monomethylpiperidine <b>24a</b>. Upon detailed examination of the NanoBRET data for <b>24b</b>, we observed a dose-dependent reduction in <i>total</i> luminescence, suggesting either cellular toxicity or depletion of one of the assay components, for example, BCL6 degradation, as has been previously reported for another series of inhibitors.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> To investigate this further, we treated SU-DHL-4 and OCI-Ly1 cells with <b>24b</b>; Western analysis of cell lysates demonstrated a concentration-dependent reduction in BCL6 protein levels, consistent with compound-mediated BCL6 degradation (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). We note also an apparent elevation in concentration of BCL6 at compound concentrations below the binding IC<sub>50</sub> which is discussed further below.</div><figure id="fig5" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/medium/jm9b02076_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0005.jpeg" id="gr18" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Treatment of DLBCL cell lines with <b>24b</b> (CCT368682) leads to degradation of BCL6. SU-DHL-4 or OCI-Ly1 cells were treated with <b>24b</b> at the indicated concentration for 4 h at 37 °C. Cells were collected, lysed, and examined by Western blot for BCL6 and GAPDH protein levels. D: DMSO control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02076&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Degradation of BCL6 represents an attractive alternative to inhibition. As the protein is removed rather than just inhibited, activity at substoichiometric concentrations of compound is possible as the small molecule degrader may act catalytically. This removes the need for complete continuous occupancy of BCL6, as the loss of function would be maintained until the target is regenerated.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> We decided to focus on optimization of this novel BCL6 degrader, aiming to identify compounds with suitable physicochemical and pharmacokinetic properties to investigate <i>in vivo</i> degradation of BCL6 in a tumor xenograft model.</div><div class="NLM_p last">To support optimization, we developed an immunofluorescence assay to quantify BCL6 degradation in SU-DHL-4 cells. DC<sub>50</sub> values from this assay are shown in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>. Initial SAR appeared tight, with relatively small changes in chemical structure resulting in the abolition of degradation activity (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>): only the monomethyl- and dimethylpiperidines <b>24a</b>, <b>24b</b>, and <b>26b</b> were identified as degraders, with DC<sub>50</sub> values similar to or below their IC<sub>50</sub> values in the TR-FRET assay. No degradation was observed for pyrazole and morpholine analogues.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Optimization of DMPK Properties</h3><div class="NLM_p">Piperidine degraders <b>24a</b> and <b>24b</b> were subject to rapid metabolism in mouse microsomes, which we expected would limit our ability to achieve sufficient exposure to demonstrate degradation <i>in vivo</i>. We noted that morpholine <b>25a</b> (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>) had lower microsomal clearance, leading us to hypothesize that metabolism of <b>24a</b> and <b>24b</b> may be occurring on the more lipophilic piperidine group. We aimed to reduce microsomal clearance and hence improve the likelihood of sufficient exposure<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> first by replacing hydrogen atoms with fluorine to block the potential metabolic sites and second by lowering lipophilicity.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Modifications to the Pyrimidine 2-Substituent</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/medium/jm9b02076_0019.gif" alt="" id="gr19" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/medium/jm9b02076_0020.gif" alt="" id="gr20" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">Data represent the geometric mean of at least three replicates. See <a href="/doi/suppl/10.1021/acs.jmedchem.9b02076/suppl_file/jm9b02076_si_001.pdf" class="ext-link">Supporting Information Table S2</a> for full statistics.</p></div><div class="footnote" id="t5fn2"><sup><sup>b</sup></sup><p class="last">Kinetic solubility measured by NMR in HEPES buffer and 5% DMSO at pH 8, and/or where data are shown in brackets, by HPLC in PBS buffer and 1% DMSO at pH 7.4. <sup>#</sup>These compounds showed <25% degradation of BCL6 in single concentration experiments at 10 μM. n.d. = not done.</p></div></div><div></div></div><div class="NLM_p">Trifluorination of the piperidine 3-methyl group showed modest improvement in microsomal stability (<b>27a</b>), and a more substantial reduction in clearance was achieved by the introduction of a difluoro group in the 4-position of the piperidine. The addition of these fluorine atoms also provided an improvement in both degradation (DC<sub>50</sub>) and biochemical inhibition (IC<sub>50</sub>), for monomethyl (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>, compare <b>24a</b> and <b>27c</b>) and dimethyl (compare <b>24b</b> and <b>1</b>) piperidine analogues.</div><div class="NLM_p">In the previously reported series of BCL6 degraders, only hydrophobic groups were shown to cause degradation.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> We therefore designed analogues with lower lipophilicity to investigate whether degradation could be retained with more hydrophilic groups in this region and as an alternative approach to lowering metabolic clearance. We chose to modify the 3-position of the piperidine, as we had already shown that compounds with hydrophilic groups in the 4-position (morpholine <b>25a</b> and piperazine <b>25c</b>) were not capable of inducing BCL6 degradation. Hydroxymethyl <b>27d</b> induced incomplete degradation, with the response plateauing at ∼60%. Addition of the 4,4-difluoro moiety was found again to improve both metabolic stability and the ability of the molecule to trigger BCL6 degradation, with <b>27e</b> (CCT369900) giving potent and full (>85%) degradation while maintaining excellent aqueous solubility. With <b>27e</b> we had thus identified an alternative and, importantly, less lipophilic group that induced degradation.</div><div class="NLM_p">Further modifications demonstrated that the structural requirements for induction of degradation are quite specific: methylation of the OH reduced degradation (<b>27f</b>); alternative hydrophilic groups such as nitrile <b>27g</b> in this position did not enable degradation; other discrete changes in structure such as moving the trifluoromethyl group (<b>27h)</b> and constraining the ring by forming bicyclic groups (<b>28a</b>, <b>28b</b>) also led to a loss of degradation activity.</div><div class="NLM_p">To confirm that degradation was not cell line specific, we developed an MSD (Meso Scale Discovery) assay to measure BCL6 levels. Compound <b>1</b> (CCT369260), along with analogues <b>27b</b> and <b>27e</b>, shows full (>85%) degradation of BCL6 in OCI-Ly1 and Karpas 422 cells, at DC<sub>50</sub> values comparable to those observed in the SU-DHL-4 immunofluorescence assay (<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>). With degradation activity confirmed, we investigated whether this translated into antiproliferative activity; examination of degrader compounds in a 14-day proliferation assay revealed growth inhibition in both SU-DHL-4 and OCI-Ly1 cell lines. Inhibitor <b>25b</b> was less effective, suggesting that depletion of BCL6 gives a stronger effect on proliferation than inhibition alone, consistent with previous findings.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> Degraders showed a 5- to 40-fold reduction in activity in the BCL6 low-expressing<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> OCI-Ly3 cell line compared to OCI-Ly1 (<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>). Degrader <b>1</b> was further profiled in a panel of BCL6-negative cell lines and showed no antiproliferative activty (<a href="/doi/suppl/10.1021/acs.jmedchem.9b02076/suppl_file/jm9b02076_si_001.pdf" class="ext-link">Supporting Information Figure S2</a>). These findings suggest that the observed antiproliferative activity is driven by on-target effects.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Antiproliferative Activity of Key Compounds</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/medium/jm9b02076_0021.gif" alt="" id="gr21" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/medium/jm9b02076_0027.gif" alt="" id="GRAPHIC-d7e1605-autogenerated" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last">Data represent the geometric mean of at least three replicates. See <a href="/doi/suppl/10.1021/acs.jmedchem.9b02076/suppl_file/jm9b02076_si_001.pdf" class="ext-link">Supporting Information Table S2</a> for full statistics.</p></div><div class="footnote" id="t6fn2"><sup><sup>b</sup></sup><p class="last">Data represent the geometric mean of at least two replicates except where superscript i is indicated. <sup>#</sup>This compound showed <10% degradation of BCL6 in a single concentration experiment at 10 μM, <i>n</i> = 2.</p></div></div><div></div></div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> <i>In Vivo</i> Profiling of <b>1</b> (CCT369260)</h3><div class="NLM_p">On the basis of its acceptable microsomal clearance and robust effects in degradation and proliferation assays, compound <b>1</b> was selected for further profiling. A pharmacokinetic study was carried out in female Balb/C mice, dosing at 1 mg/kg iv (<i>n</i> = 3) and 5 mg/kg po (<i>n</i> = 3). All mice appeared normal after dosing and 24 h after dose. Compound <b>1</b> demonstrated moderate clearance (CL 20 mL min<sup>–1</sup> kg<sup>–1</sup>) with mean oral bioavailability of 54%. Protein binding measurements using equilibrium dialysis showed the compound is highly bound (0.07% free in SCID mouse plasma, <i>n</i> = 6). To enable comparison of free concentrations <i>in vivo</i> with free levels in the OCI-Ly1 DC<sub>50</sub> MSD assay, the free fraction in the DC<sub>50</sub> assay medium was also measured (2.2% fraction unbound in IMDM medium, <i>n</i> = 3). This was used to estimate that a free concentration of greater than 1.1 nM (based on the OCI-Ly1 DC<sub>50</sub>) is required to elicit degradation <i>in vivo</i>, corresponding to a total plasma concentration of 1.5 μM or a total blood concentration of 2.8 μM (SCID mouse blood to plasma ratio of 1.9). PK studies were carried out at three doses (5, 15, and 50 mg/kg po, <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>) to determine if sufficient compound exposure could be achieved in the SCID mouse strain that would be used for xenograft studies. Exposure increased with dose between 5 and 15 mg/kg; however, no improvement was obtained by a further increase to 50 mg/kg. This is likely due to compound solubility; the compound was dosed as a clear solution in saline containing 10% DMSO and 5% Tween 20 at the lower doses but was not completely soluble and hence was dosed as a suspension in the same vehicle at the 50 mg/kg dose. The mean total blood concentration at the 15 mg/kg dose was above the desired concentration for approximately 10 h, confirming that this compound was suitable for progression to a PK/PD study at this dose to determine if, and at what concentration, BCL6 depletion could be achieved.</div><figure id="fig6" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/medium/jm9b02076_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0006.jpeg" id="gr23" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (Left) Total SCID mouse blood concentrations of <b>1</b> (CCT369260) after po dosing at 5 (blue triangles), 15 (red circles), and 50 (green squares) mg/kg vs predicted active concentration (dashed line). (Right) Data for individual animals showing <i>c</i><sub>max</sub>, <i>t</i><sub>max</sub>, and total AUC<sub>10h</sub> in SCID mouse at 5, 15, and 50 mg/kg doses. All experiments were carried out according to the U.K. guidelines for animal experimentation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02076&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">A PK/PD study was carried out in mice using a single dose of <b>1</b> at 15 mg/kg po in an OCI-Ly1 DLBCL xenograft model. A clear decrease in levels of BCL6 in the tumor was observed up to 10 h after dosing with compound <b>1</b> (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b02076/suppl_file/jm9b02076_si_001.pdf" class="ext-link">Supporting Information Figure S3</a>), with a maximal effect at ∼4 h. No visible adverse events were observed in this study. Examination of the relationship between compound <b>1</b> and BCL6 levels (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b02076/suppl_file/jm9b02076_si_001.pdf" class="ext-link">Supporting Information Figure S4</a>) shows that, consistent with predictions, free concentrations of >1 nM degrader <b>1</b> lead to degradation of BCL6. Less substantial degradation is observed at 0.5 h; this delayed onset may be due to the time required for the compound to distribute into tumor. After compound levels drop below the calculated free DC<sub>50</sub> at the 10 h time point, BCL6 levels remain low but rebound by 12 and 16 h, presumably due to the time required for BCL6 resynthesis. This study demonstrates that degradation of BCL6 in tumors following oral dosing with a small molecule is a feasible approach.</div><figure id="fig7" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/medium/jm9b02076_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0007.jpeg" id="gr24" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. PK/PD study with <b>1</b> (CCT369260) at 15 mg/kg po. Tumour xenografts were prepared by subcutaneous injection of 1 × 10<sup>7</sup> OCI-Ly1 cells in female SCID mice, with dosing of compound commencing 20 days after injection, to mice with xenografts between 0.5 and 0.8 cm<sup>3</sup>, as described in more detail in the <a href="/doi/suppl/10.1021/acs.jmedchem.9b02076/suppl_file/jm9b02076_si_001.pdf" class="ext-link">Supporting Information</a>. Sampling took place at 0.5, 2, 4, 6, 10, 12, 16 and 24 h postdose. All experiments were carried out according to the U.K. guidelines for animal experimentation. BCL6 levels in tumor were quantified using capillary electrophoresis and normalized to a GAPDH loading control and are shown as gray (vehicle-treated) or black (compound treated) bars. Free compound levels at 0.5–24 h are shown (blue dots); dotted line indicates free DC<sub>50</sub>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02076&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i21">Discussion and Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_28880" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_28880" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We describe the discovery of a series of benzimidazolone inhibitors of BCL6, including <b>1</b> (CCT369260), an orally bioavailable degrader of BCL6. Weakly active (100 μM) hits derived from high throughput screening were merged to give benzimidazolone lead compound <b>6</b> (CCT365386). By filling a hydrophobic pocket and forming hydrogen bonds to water molecules, we were able to both improve binding affinity and incorporate hydrophilic groups to improve solubility. Further potency improvements were achieved through additional hydrophobic surface contacts in a solvent-exposed region of the pocket, resulting in sub-100 nM inhibitors. A subset of these compounds, including <b>24b</b> (CCT368682), were shown to cause rapid degradation of BCL6, initially by Western blot and by an immunofluorescence assay in SU-DHL-4, and later confirmed in different cell lines using a MSD assay.</div><div class="NLM_p">We noted that our initial Western blot data for <b>24b</b> showed an apparent increase in BCL6 levels at low concentrations. A similar effect is also visible in previously presented work on BCL6 degraders,<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> but analysis of curves from MSD experiments shows that this effect is not observed in all experiments and, when observed, corresponds to a < 25% increase. It is noteworthy that we and others<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> also observe an increase in BCL6 levels when cells are treated with nondegrading <i>inhibitors</i> of BCL6. This suggests that binding may trigger a feedback mechanism leading to increased expression of BCL6.</div><div class="NLM_p">Further optimization of initially identified degrader <b>24b</b> enabled us to identify compounds with reduced metabolic clearance including <b>27e</b> (CCT369900) and <b>1</b> (CCT369260). CCT369260 showed sub-100 nM activity in a degradation assay in SU-DHL-4 cells, and robust antiproliferative activity, in common with previously identified degraders<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> but in contrast to BCL6-targetting PROTACs.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a></div><div class="NLM_p">Similar to previously reported degraders,<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> we found that degradation mediated by our compounds is proteasome-dependent. It is noteworthy that we observe no significant difference in binding mode (as observed by X-ray crystallography) between <b>1</b> (CCT369260) and closely related nondegrader <b>25a</b> (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>), suggesting that the inhibitor or degrader behavior of a compound may depend on the nature of the substituents in this region of the molecule rather than on a distinct binding mode.</div><figure id="fig8" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/medium/jm9b02076_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0008.jpeg" id="gr25" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Binding modes of the BCL6 degrader <b>1</b> (CCT369260) (A) and inhibitor <b>25a</b> (B). Crystal structures of the BCL6 BTB domain bound to the BCL6 degrader <b>1</b> (CCT369260) (A, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOM">6TOM</a>) and inhibitor <b>25a</b> (B, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOL">6TOL</a>) show a nearly identical binding mode. In both panels, the surface of the BCL6 dimer is shown as a gray transparent surface, with the two individual monomers displayed in ribbons and colored in gray and cyan. Selected residues are shown as lines. Compounds are shown as orange ball and sticks, selected water molecules as red spheres, and H-bonds as yellow dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02076&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">One possible hypothesis for how degradation is induced is via formation of a ternary complex of the degrader, BCL6, and an E3 ligase. In this context, we note that the degrader compounds including CCT369260 are largely buried in the BTB domain, with a relatively small area remaining solvent-exposed and available to bind to another component (for example, an E3 ligase) and that in contrast to bivalent PROTAC-type degraders, no hook effect is observed with increased concentration.<a onclick="showRef(event, 'ref23 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref23 ref24 ref25">(23−25)</a> These data suggest that the degrader molecule alone does not bind potently to an E3 ligase in the absence of BCL6. One related potential mechanism by which a small molecule could promote proteosomal degradation is through “hydrophobic tagging”; the degrader binds to the target protein and presents as a hydrophobic patch on the protein surface, mimicking a partially unfolded protein state, which is then targeted for proteosomal degradation.<a onclick="showRef(event, 'ref23 ref24 ref26'); return false;" href="javascript:void(0);" class="ref ref23 ref24 ref26">(23,24,26)</a> Should this be the case for our BCL6 degraders, then the incorporation of hydrophilic groups in the solvent-exposed region would not be expected to be tolerated, and a broad range of lipophilic groups could be expected to trigger degradation. However, we show first that certain hydrophilic groups are tolerated with degradation retained and second that the structural requirements for degradation are quite specific. These data together support an alternative hypothesis: that the combination of the protein surface and the bound degrader surface generates a “neo-substrate” or “neo-degron” that can be recognized by an E3 ubiquitin ligase or other effector of proteasomal degradation. This is similar conceptually to the mechanism of action of IMiDs which bind strongly to an E3 ligase and generate a “neo-enzyme” surface which modifies the substrate specificity of the ligase, although in our case it is the substrate surface, rather than that of the E3 ligase, that we hypothesize to be modified.</div><div class="NLM_p last">CCT369260 was progressed into PK studies, and we showed that the levels of compound needed to mediate degradation could be achieved <i>in vivo</i> following oral dosing, enabling us to demonstrate depletion of BCL6 levels in tumors following oral dosing in a mouse xenograft model. Ongoing work includes further optimization of CCT369260 and the use of our BCL6 degraders as <i>in vitro</i> and <i>in vivo</i> chemical tools to pharmacologically validate BCL6 as a target for the treatment of hematological cancer.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i22">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_83066" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_83066" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">All <i>in vivo</i> experiments were carried out according to the U.K. guidelines for animal experimentation. Cell lines were supplied by the German Collection of Microorganisms and Cell Cultures (DSMZ). Cell lines were authenticated by STR profiling and were routinely screened for <i>Mycoplasma</i>, using an in-house PCR-based assay.</div><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> General Synthetic Information</h3><div class="NLM_p last">All anhydrous solvents and reagents were obtained from commercial suppliers and used without further purification. Evaporation of solvent was carried out using a rotary evaporator under reduced pressure at a bath temperature of up to 60 °C. Flash column chromatography was carried out using a Biotage purification system using SNAP KP-Sil cartridges or on reverse-phase mode using SNAP Ultra C18 cartridges. Semipreparative separations were carried out using an Agilent 1200 series preparative HPLC instrument over a 15 min gradient elution. Microwave-assisted reactions were carried out using a Biotage Initiator microwave system. Final compounds were purified to ≥95% purity. NMR data were collected on a Bruker Avance 500 spectrometer equipped with a 5 mm BBO/QNP probe or on a Bruker Avance Neo 600 spectrometer equipped with a 5 mm TCI CryoProbe. NMR data are presented in the form of chemical shift δ (multiplicity, coupling constants, integration) for major diagnostic protons, given in parts per million (ppm) relative to tetramethylsilane (TMS), referenced to the internal deuterated solvent. HRMS was assessed using an Agilent 1200 series HPLC instrument and diode array detector coupled to a 6120 time-of-flight mass spectrometer with dual multimode APCI/ESI source or on a Waters Acquity UHPLC and diode array detector coupled to a Waters G2 QToF mass spectrometer fitted with a multimode ESI/APCI source.</div></div><div id="sec4_2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Preparation of Compounds. 5-((5-Chloro-2-((3<i>R</i>,5<i>S</i>)-4,4-difluoro-3,5-dimethylpiperidin-1-yl)pyrimidin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>1</b>, CCT369260)</h3><div class="NLM_p last">To a mixture of 5-((2,5-dichloropyrimidin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>18</b>) (2.0 g, 5.0 mmol) were added (3<i>R</i>,5<i>S</i>)-4,4-difluoro-3,5-dimethylpiperidine hydrochloride (1.03 g, 5.6 mmol), NMP (20.2 mL), and DIPEA (2.64 mL, 15.1 mmol). The resulting mixture was heated to 140 °C for 2 h, cooled to room temperature, and then poured into ice–water. The resulting precipitate was collected and washed thoroughly with water to remove residual NMP, then further washed with ethanol to give a brown solid. Ethanol filtrate was concentrated and washed with water, then ethanol to give a second crop of solid. The two solid crops were combined and suspended in water. Ethanol was added until the brown color was removed from the solid, which was collected by filtration, washed again with water, and dried under vacuum to give compound <b>1</b> (1.97 g, 3.87 mmol, 77%) as an off-white solid. HRMS (ESI +ve): found 509.2269, expected 509.2243 for C<sub>24</sub>H<sub>32</sub>ClF<sub>2</sub>N<sub>6</sub>O<sub>2</sub> [M + H]<sup>+</sup>. δ<sub>H</sub> (600 MHz, CDCl<sub>3</sub>) 7.98 (s, 1H), 7.32 (d, <i>J</i> = 2.0 Hz, 1H), 7.22 (dd, <i>J</i> = 8.3, 2.0 Hz, 1H), 6.99 (s, 1H), 6.94 (d, <i>J</i> = 8.3 Hz, 1H), 4.60 (br d, <i>J</i> = 12.7 Hz, 2H), 4.04 (t, <i>J</i> = 7.4 Hz, 2H), 3.43 (s, 3H), 2.73 (t, <i>J</i> = 12.7 Hz, 2H), 2.32 (s, 1H), 2.02–1.90 (m, 2H), 1.89 (t, <i>J</i> = 7.4 Hz, 2H), 1.29 (s, 6H), 1.05 (d, <i>J</i> = 6.7 Hz, 6H).</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> 2-Chloro-4-((cyclopropylmethyl)amino)nicotinonitrile (<b>5</b>)</h3><div class="NLM_p last">A suspension of cyclopropylmethanamine (17 μL, 0.2 mmol), triethylamine (25 μL, 0.18 mmol), and 2,4-dichloronicotinonitrile (30 mg, 0.17 mmol) in DMA (0.35 mL) was stirred at rt for 18 h. The reaction was quenched with brine and extracted with EtOAc. The combined organics were washed with water and brine, dried, and concentrated, then purified by flash column chromatography (0–15% EtOAc in cyclohexane) to give <b>5</b> (26 mg, 0.13 mmol, 72%). δ<sub>H</sub> (500 MHz, CDCl<sub>3</sub>) 8.09 (d, <i>J</i> = 6.1 Hz, 1H), 6.48 (d, <i>J</i> = 6.1 Hz, 1H), 5.31 (br s, 1H), 3.16–3.10 (m, 2H), 1.19–1.07 (m, 1H), 0.74–0.64 (m, 2H), 0.36–0.30 (m, 2H).</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> 2-Chloro-4-((1,3-dimethyl-2-oxo-2,3-dihydro-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)amino)nicotinonitrile (<b>6</b>, CCT365386)</h3><div class="NLM_p last">A mixture of 5-amino-1,3-dimethyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (30 mg, 0.17 mmol), 2,4-dichloronicotinonitrile (30 mg, 0.17 mmol), and DIPEA (0.032 mL, 0.18 mmol) in DMA (0.4 mL) was heated in the microwave to 120 °C for 30 min, then added to a 1:1 mixture of methanol:water. The resulting precipitate was collected, washed sequentially with water, methanol, diethyl ether and dried to give <b>6</b> (32.3 mg, 0.10 mmol, 61%) as a brown solid. HRMS (ESI +ve): found 314.0810, expected 314.0803 for C<sub>15</sub>H<sub>13</sub>ClN<sub>5</sub>O<sup>+</sup> [M + H]<sup>+</sup>. δ<sub>H</sub> (500 MHz, DMSO-<i>d</i><sub>6</sub>) 9.50 (s, 1H), 8.01 (d, <i>J</i> = 6.2 Hz, 1H), 7.21 (d, <i>J</i> = 8.3 Hz, 1H), 7.16 (d, <i>J</i> = 2.0 Hz, 1H), 7.00 (dd, <i>J</i> = 8.2, 2.0 Hz, 1H), 6.67 (d, <i>J</i> = 6.2 Hz, 1H), 3.36 (s, 3H), 3.32 (s, 3H).</div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> 1-Methyl-5-nitro-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>7</b>)</h3><div class="NLM_p last">To a mixture of <i>N</i><sup>1</sup>-methyl-4-nitrobenzene-1,2-diamine (10.5 g, 62.8 mmol) in acetonitrile (150 mL) cooled to 0 °C was added portionwise disuccinimidyl carbonate (21.7 g, 84.7 mmol) over 20 min. The mixture was stirred at rt for 18 h and then poured into ice–water, forming a precipitate. The solid was collected by suction filtration and washed sequentially with water, DCM, and diethyl ether to give <b>7</b> (11.12 g) as an orange-red solid. HRMS (ESI +ve): found 194.0572, expected 194.0560 for C<sub>8</sub>H<sub>8</sub>N<sub>3</sub>O<sub>3</sub><sup>+</sup> [M + H]<sup>+</sup>. δ<sub>H</sub> (500 MHz, DMSO-<i>d</i><sub>6</sub>) 11.43 (s, 1H), 8.03 (dd, <i>J</i> = 8.7, 2.3 Hz, 1H), 7.76 (d, <i>J</i> = 2.3 Hz, 1H), 7.31 (d, <i>J</i> = 8.7 Hz, 1H), 3.36 (s, 3H).</div></div><div id="sec4_2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> 2-Chloro-4-((1-methyl-2-oxo-2,3-dihydro-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)amino)nicotinonitrile (<b>9</b>)</h3><div class="NLM_p last">To a mixture of 5-amino-1-methyl-1<i>H</i>-benzo[<i>d</i>]imidazol-2(3<i>H</i>)-one (750 mg, 4.6 mmol) and 2,4-dichloropyridine-3-carbonitrile (760 mg, 4.4 mmol) under argon was added DMA (10 mL) followed by DIPEA (0.90 mL, 5.19 mmol). The reaction mixture was heated at 120 °C under microwave irradiation for 45 min then allowed to cool to rt and added dropwise to a stirring mixture of methanol:water (1:1; 120 mL). The resulting precipitate was filtered, washed with water (2 × 25 mL) and diethyl ether (2 × 30 mL), affording 2-chloro-4-[(1-methyl-2-oxo-3<i>H</i>-benzimidazol-5-yl)amino]pyridine-3-carbonitrile (1.30 g, 99%, 4.3 mmol) as a beige solid. δ<sub>H</sub> (500 MHz, DMSO-<i>d</i><sub>6</sub>) 10.95 (br s, 1H), 9.39 (br s, 1H), 7.99 (d, <i>J</i> = 6.2 Hz, 1H), 7.13 (d, <i>J</i> = 8.3 Hz, 1H), 6.95 (dd, <i>J</i> = 8.3, 1.9 Hz, 1H), 6.90 (d, <i>J</i> = 1.9 Hz, 1H), 6.65 (d, <i>J</i> = 6.2 Hz, 1H), 3.29 (s, 3H).</div></div><div id="sec4_2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> 3-(2-Hydroxybutyl)-1-methyl-5-nitro-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>10</b>)</h3><div class="NLM_p last">To a mixture of 1-methyl-5-nitro-1<i>H</i>-benzo[<i>d</i>]imidazol-2(3<i>H</i>)-one (240 mg, 1.242 mmol) and cesium carbonate (485 mg, 1.489 mmol) in DMF (3 mL) was added 2-ethyloxirane (0.119 mL, 1.367 mmol), and the resulting mixture was heated in the microwave to 120 °C for 1 h, then added to water (10 mL). The resulting mixture was extracted with DCM, and combined organics were dried over sodium sulfate, filtered, and evaporated onto silica gel for purification by flash column chromatography (10 g silica, 40–60% ethyl acetate in cyclohexane). Fractions were combined and evaporated to give the title compound (200 mg, 60%) as a yellow-orange oil which solidified on standing. δ<sub>H</sub> (500 MHz, DMSO-<i>d</i><sub>6</sub>) 8.13 (d, <i>J</i> = 2.2 Hz, 1H), 8.05 (dd, <i>J</i> = 8.7, 2.3 Hz, 1H), 7.36 (d, <i>J</i> = 8.8 Hz, 1H), 4.90 (d, <i>J</i> = 5.4 Hz, 1H), 3.88 (dd, <i>J</i> = 14.2, 4.2 Hz, 1H), 3.82 (dd, <i>J</i> = 14.2, 7.4 Hz, 1H), 3.70 (m, 1H), 3.41 (s, 3H), 1.49 (m, 1H), 1.37 (m, 1H), 0.92 (t, <i>J</i> = 7.4 Hz, 3H).</div></div><div id="sec4_2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> 2-Chloro-4-((3-(2-hydroxybutyl)-1-methyl-2-oxo-2,3-dihydro-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)amino)nicotinonitrile (<b>11a</b>)</h3><div class="NLM_p last">To a suspension of 3-(2-hydroxybutyl)-1-methyl-5-nitro-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>10</b>) (34 mg, 0.13 mmol) in ethanol (1.5 mL) and DMSO (0.5 mL) was added sodium dithionite (116 mg, 0.67 mmol). The resulting mixture was heated to 90 °C for 5 h, then stirred at room temperature for 2 days, then heated again to 90 °C for 2 h. Additional sodium dithionite (116 mg, 0.67 mmol) was added, and the mixture was heated to 90 °C for 2 h. Water (1 mL) was added, and the resulting cloudy suspension was stirred at rt for 3 days, then diluted with water until all was in solution. The pH was adjusted to 10 using 2 M sodium carbonate, and the resulting mixture was extracted with DCM (×3), dried by passing through a phase separator, and evaporated under reduced pressure. The resulting material was diluted with DMA (0.5 mL), and then 2,4-dichloronicotinonitrile (15 mg, 0.087 mmol) and DIPEA (25 μL, 0.143 mmol) were added. The resulting mixture was heated in the microwave to 120 °C for 30 min, then partitioned between DCM and water. The aqueous layer was acidified to pH 7 using 10% citric acid, then extracted with DCM (×3). Combined organics were evaporated and purified by HPLC (ACE 5 C18-PFP 250 mm × 21.2 mm column, 15 min gradient from 60:40 to 0:100 water:methanol, 0.1% formic acid modifier) at a flow rate of 20 mL/min. The resulting yellow oil was triturated with diethyl ether to give compound <b>11a</b> (4.2 mg, 0.011 mmol, 8.5%) as pale yellow solid. HRMS (ESI +ve): found 372.1230, expected 372.1222 for C<sub>18</sub>H<sub>19</sub>ClN<sub>5</sub>O<sub>2</sub><sup>+</sup> [M + H]<sup>+</sup>. δ<sub>H</sub> (500 MHz, CD<sub>3</sub>OD) 7.94 (d, <i>J</i> = 6.2 Hz, 1H), 7.22 (d, <i>J</i> = 8.2 Hz, 1H) overlapping with 7.22 (d, <i>J</i> = 1.9 Hz, 1H), 7.07 (dd, <i>J</i> = 8.4, 1.9 Hz, 1H), 6.70 (d, <i>J</i> = 6.3 Hz, 1H), 3.98–3.89 (m, 1H), 3.89–3.80 (m, 2H), 3.47 (s, 3H), 1.67–1.55 (m, 1H), 1.48 (dp, <i>J</i> = 14.6, 7.3 Hz, 1H), 1.03 (t, <i>J</i> = 7.4 Hz, 3H).</div></div><div id="sec4_2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> 2-Chloro-4-((3-(2-cyanobutyl)-1-methyl-2-oxo-2,3-dihydro-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)amino)nicotinonitrile (<b>11b</b>)</h3><div class="NLM_p last">To a mixture of cesium carbonate (70 mg, 0.21 mmol) and 2-chloro-4-((1-methyl-2-oxo-2,3-dihydro-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)amino)nicotinonitrile (<b>9</b>) (25 mg, 0.08 mmol) in DMF (0.50 mL) was added 2-(bromomethyl)butanenitrile (25 mg, 0.15 mmol). The resulting mixture was heated to 60 °C for 3 days, then partitioned between water and DCM, and the aqueous layer acidified to pH 5 using a 10% citric acid solution. Layers were separated, the aqueous phase was extracted with further DCM, and combined organics were evaporated under reduced pressure and then purified by HPLC (ACE 5 C18-PFP 250 mm × 21.2 mm column, 15 min gradient from 45:55 to 30:70 water:methanol, 0.1% formic acid modifier) at a flow rate of 20 mL/min to give compound <b>11b</b> as an off-white solid. HRMS (ESI +ve): found 381.1209, expected 381.1225 for C<sub>19</sub>H<sub>18</sub>ClN<sub>6</sub>O<sup>+</sup> [M + H]<sup>+</sup>. δ<sub>H</sub> (500 MHz, CDCl<sub>3</sub>) 8.04 (d, <i>J</i> = 6.1 Hz, 1H), 7.08 (d, <i>J</i> = 1.9 Hz, 1H), 7.07 (d, <i>J</i> = 8.2 Hz, 1H), 7.02 (dd, <i>J</i> = 8.2, 1.9 Hz, 1H), 6.96 (s, 1H), 6.71 (d, <i>J</i> = 6.1 Hz, 1H), 4.11 (dd, <i>J</i> = 14.4, 5.7 Hz, 1H), 4.04 (dd, <i>J</i> = 14.5, 8.8 Hz, 1H), 3.48 (s, 3H), 3.11 (tt, <i>J</i> = 9.1, 5.4 Hz, 1H), 1.89–1.69 (m, 2H), 1.19 (t, <i>J</i> = 7.4 Hz, 3H).</div></div><div id="sec4_2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> (<i>S</i>)-2-Chloro-4-((3-(2-hydroxybutyl)-1-methyl-2-oxo-2,3-dihydro-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)amino)nicotinonitrile (<b>11c</b>)</h3><div class="NLM_p last">To a mixture of 2-chloro-4-((1-methyl-2-oxo-2,3-dihydro-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)amino)nicotinonitrile (<b>9</b>) (20 mg, 0.067 mmol) and cesium carbonate (23 mg, 0.07 mmol) in DMF (0.50 mL) was added (2<i>S</i>)-2-ethyloxirane (7 μL, 0.08 mmol), and the resulting mixture was heated in the microwave to 120 °C for 1 h. Two further batches of (2<i>S</i>)-2-ethyloxirane (7 μL, 0.08 mmol) were added, with microwave heating to 140 °C for 30 min after each. The mixture was partitioned between water and DCM, and the aqueous layer was acidified to pH 5 using a 10% citric acid solution. Layers were separated, the aqueous phase was extracted with further DCM, and combined organics were evaporated under reduced pressure and then purified by HPLC (ACE 5 C18-PFP 250 mm × 21.2 mm column, 15 min gradient from 50:50 to 35:65 water:methanol, 0.1% formic acid modifier) at a flow rate of 20 mL/min to give compound <b>11c</b> (7.5 mg, 0.020 mmol, 30%) as an off white solid. HRMS (ESI +ve): found 372.1218, expected 372.1222 for C<sub>18</sub>H<sub>19</sub>ClN<sub>5</sub>O<sub>2</sub><sup>+</sup> [M + H]<sup>+</sup>. δ<sub>H</sub> (500 MHz, CD<sub>3</sub>OD) identical to racemate <b>11a</b>.</div></div><div id="sec4_2_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> (<i>R</i>)-2-Chloro-4-((3-(2-hydroxybutyl)-1-methyl-2-oxo-2,3-dihydro-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)amino)nicotinonitrile (<b>11d</b>)</h3><div class="NLM_p last">Prepared by the same method as its enantiomer <b>11c</b>, starting from (2<i>R</i>)-2-ethyloxirane. HRMS (ESI +ve): found 372.1218, expected 372.1222 for C<sub>18</sub>H<sub>19</sub>ClN<sub>5</sub>O<sub>2</sub><sup>+</sup> [M + H]<sup>+</sup>. δ<sub>H</sub> (500 MHz, CD<sub>3</sub>OD) identical to racemate <b>11a</b>.</div></div><div id="sec4_2_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> 4-((3-Butyl-1-methyl-2-oxo-2,3-dihydro-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)amino)-2-chloronicotinonitrile (<b>11e</b>)</h3><div class="NLM_p last">To 2-chloro-4-((1-methyl-2-oxo-2,3-dihydro-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)amino)nicotinonitrile (<b>9</b>) (25 mg, 0.084 mmol) and cesium carbonate (33 mg, 0.10 mmol) in DMF (0.85 mL) was added 1-bromobutane (9.5 μL, 0.089 mmol), and the resulting mixture was stirred at 140 °C under microwave irradiation for 1 h. Water (10 mL) was added, and the aqueous mixture was acidified with 1 M HCl. The aqueous mixture was extracted with EtOAc (5 × 10 mL). The organic extracts were combined, dried (Na<sub>2</sub>SO<sub>4</sub>), and concentrated in vacuo, then purified by HPLC (ACE 5 C18-PFP 250 mm × 21.2 mm column; 15 min gradient of 40:60 to 20:80 water:methanol, 0.1% formic acid modifier) at a flow rate of 20 mL/min to give compound <b>11e</b> (12.3 mg, 41%, 0.035 mmol) as an off-white solid. HRMS (ESI +ve): found 356.1263, expected 356.1273 for C<sub>18</sub>H<sub>19</sub>ClN<sub>5</sub>O<sup>+</sup> [M + H]<sup>+</sup>. δ<sub>H</sub> (500 MHz, CDCl<sub>3</sub>) 8.03 (d, <i>J</i> = 6.0 Hz, 1H), 7.02 (d, <i>J</i> = 8.1 Hz, 1H), 6.98 (dd, <i>J</i> = 8.1, 1.7 Hz, 1H), 6.93 (br s, 1H), 6.87 (d, <i>J</i> = 1.7 Hz, 1H), 6.60 (d, <i>J</i> = 6.0 Hz, 1H), 3.88 (t, <i>J</i> = 7.3 Hz, 2H), 3.46 (s, 3H), 1.77–1.68 (m, 2H), 1.45–1.35 (m, 2H), 0.97 (t, <i>J</i> = 7.3 Hz, 3H).</div></div><div id="sec4_2_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> (<i>R</i>)-2-Chloro-4-((3-(3-hydroxybutyl)-1-methyl-2-oxo-2,3-dihydro-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)amino)nicotinonitrile (<b>11f</b>)</h3><div class="NLM_p last">To 2-chloro-4-((1-methyl-2-oxo-2,3-dihydro-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)amino)nicotinonitrile (<b>9</b>) (40 mg, 0.13 mmol) and cesium carbonate (60 mg, 0.18 mmol) in DMF (0.6 mL) was added [(3<i>R</i>)-3-hydroxybutyl] 4-methylbenzenesulfonate (<b>14b</b>) (55 mg, 0.23 mmol), and the resulting mixture was heated in the microwave to 120 °C for 1 h. The mixture was partitioned between water and DCM, and the aqueous layer was acidified to pH 5 using a 10% citric acid solution. Layers were separated, the aqueous phase was extracted with further DCM, and combined organics were evaporated under reduced pressure and then purified by HPLC (ACE 5 C18-PFP 250 mm × 21.2 mm column, 15 min gradient from 50:50 to 35:65 water:methanol, 0.1% formic acid modifier) at a flow rate of 20 mL/min to give compound <b>11f</b> (21.5 mg, 0.057 mmol, 42%). HRMS (ESI +ve): found 372.1228, expected 372.1222 for C<sub>18</sub>H<sub>19</sub>ClN<sub>5</sub>O<sub>2</sub><sup>+</sup> [M + H]<sup>+</sup>. δ<sub>H</sub> (500 MHz, CDCl<sub>3</sub>) 8.05 (d, <i>J</i> = 6.2 Hz, 1H), 7.07 (d, <i>J</i> = 8.2 Hz, 1H), 7.03 (dd, <i>J</i> = 2.0, 8.2 Hz, 1H), 6.97 (d, <i>J</i> = 2.0 Hz, 1H), 6.94 (s, 1H), 6.63 (d, <i>J</i> = 6.2 Hz, 1H), 4.25 (ddd, <i>J</i> = 15.0, 11.0, 4.6 Hz, 1H), 3.88 (dt, <i>J</i> = 15.0, 5.5, 3.8 Hz, 1H), 3.72 (dqd, <i>J</i> = 10.2, 6.2, 2.8 Hz, 1H), 3.49 (s, 3H), 1.88 (dddd, <i>J</i> = 14.1, 11, 5.5, 2.8 Hz, 1H), 1.71 (dddd, <i>J</i> = 14.1, 10.2, 4.6, 3.8 Hz, 1H), 1.22 (d, <i>J</i> = 6.2 Hz, 3H).</div></div><div id="sec4_2_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> (<i>S</i>)-2-Chloro-4-((3-(3-hydroxybutyl)-1-methyl-2-oxo-2,3-dihydro-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)amino)nicotinonitrile (<b>11g</b>)</h3><div class="NLM_p last">Prepared by the same method as its enantiomer <b>11f</b>. HRMS (ESI +ve): found 372.1219, expected 372.1222 for C<sub>18</sub>H<sub>19</sub>ClN<sub>5</sub>O<sub>2</sub><sup>+</sup> [M + H]<sup>+</sup>. δ<sub>H</sub> (500 MHz, DMSO-<i>d</i><sub>6</sub>) 9.48 (s, 1H), 8.00 (d, <i>J</i> = 6.0 Hz, 1H), 7.20 (d, <i>J</i> = 8.2 Hz, 1H), 7.17 (d, <i>J</i> = 1.9 Hz, 1H), 7.00 (dd, <i>J</i> = 1.9, 8.2 Hz, 1H), 6.68 (d, <i>J</i> = 6.0 Hz, 1H), 4.58 (d, <i>J</i> = 4.7 Hz, 1H), 3.94–3.79 (m, 2H), 3.65–3.58 (m, 1H), 3.34 (s, 3H), 1.72–1.58 (m, 2H), 1.08 (d, <i>J</i> = 6.3 Hz, 3H).</div></div><div id="sec4_2_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> 3-Hydroxy-3-methylbutyl 4-methylbenzenesulfonate (<b>14a</b>)</h3><div class="NLM_p last">To a solution of 3-methylbutane-1,3-diol (27 g, 0.26 mol) and triethylamine (40 mL, 0.29 mol) in DCM (250 mL) at 0 °C was added dropwise 4-methylbenzenesulfonyl chloride (50 g, 84 mmol) in 200 mL of DCM over 1 h. The resulting mixture was stirred at 0 °C for 4 h, then water was added (200 mL) and stirred for 45 min. Adjustment to pH 10 was made by addition of 2 M NaOH and separated. Organic phase was washed sequentially with 200 mL of half-saturated sodium bicarbonate solution (×2), water, 1 M HCl, brine, then dried over magnesium sulfate, filtered, and concentrated to give a yellow oil. This was dissolved in DCM/cyclohexane and purified by flash column chromatography (0–40% ethyl acetate in cyclohexane) to give the title compound (39.9 g) as a colorless oil. δ<sub>H</sub> (500 MHz, CDCl<sub>3</sub>) 7.85–7.78 (m, 2H), 7.40–7.34 (m, 2H), 4.23 (t, <i>J</i> = 6.8 Hz, 2H), 2.47 (s, 3H), 1.88 (t, <i>J</i> = 6.8 Hz, 2H), 1.23 (s, 6H).</div></div><div id="sec4_2_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> [(3<i>R</i>)-3-Hydroxybutyl] 4-methylbenzenesulfonate (<b>14b</b>)</h3><div class="NLM_p last">A solution of (3<i>R</i>)-butane-1,3-diol (0.86 mL, 9.5 mmol) in dry dichloromethane (10 mL) under a nitrogen atmosphere was cooled in a salt–ice bath (bath temp −12 °C). Triethylamine (2.25 mL, 16.1 mmol) was added followed by a solution of 4-methylbenzenesulfonyl chloride (2 g, 10.5 mmol) in dry dichloromethane (6 mL) over 10 min. The resulting mixture was allowed to warm slowly to room temperature and stirred for 20 h, then diluted with DCM, washed with 10% citric acid, sat. sodium bicarbonate solution, brine, dried over sodium sulfate, filtered, and evaporated under reduced pressure to give a clear oil. This was purified by flash column chromatography (50 g silica, 10–30% ethyl acetate in cyclohexane) to give <b>14b</b> (1.65 g, 6.75 mmol, 71%) as a clear oil. δ<sub>H</sub> (500 MHz, CDCl<sub>3</sub>) 7.82 (br d, <i>J</i> = 8.3 Hz, 2H), 7.37 (br d, <i>J</i> = 7.9 Hz, 2H), 4.26 (ddd, <i>J</i> = 10.0, 8.7, 5.0 Hz, 1H), 4.13 (dt, <i>J</i> = 10.0, 5.5 Hz, 1H), 3.96 (dqd, <i>J</i> = 9.7, 6.2, 3.5 Hz, 1H), 2.47 (s, 3H), 1.85 (dddd, <i>J</i> = 14.5, 8.7, 5.8, 3.6 Hz, 1H), 1.71 (ddt, <i>J</i> = 14.2, 8.9, 5.0 Hz, 1H), 1.21 (d, <i>J</i> = 6.3 Hz, 3H).</div></div><div id="sec4_2_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> 3-(3-Hydroxy-3-methylbutyl)-1-methyl-5-nitro-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>15</b>)</h3><div class="NLM_p last">A mixture of 1-methyl-5-nitro-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>7</b>, 9.05 g, 46.9 mmol), cesium carbonate (30.5 g, 93.7 mmol), and 3-hydroxy-3-methylbutyl 4-methylbenzenesulfonate (<b>14a</b>, 16.4 g, 63.5 mmol) in acetonitrile (234 mL) was heated to reflux for 4 h. The mixture was concentrated under reduced pressure. The residue was then diluted with water (500 mL) and stirred for 30 min, forming a precipitate, which was filtered and washed thoroughly with water. The collected solid was dried under vacuum to give <b>15</b> (12 g, 43 mmol, 92%) as a red-brown solid. δ<sub>H</sub> (500 MHz, CDCl<sub>3</sub>) 8.13 (dd, <i>J</i> = 8.7, 2.0 Hz, 1H), 7.95 (d, <i>J</i> = 2.0 Hz, 1H), 7.04 (d, <i>J</i> = 8.7 Hz, 1H), 4.13 (m, 2H), 3.50 (s, 3H), 1.93 (m, 2H), 1.34 (s, 6H).</div></div><div id="sec4_2_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> 5-Amino-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>16</b>)</h3><div class="NLM_p last">Palladium on activated charcoal (10% Pd, 240 mg) was added to a solution of 3-(3-hydroxy-3-methylbutyl)-1-methyl-5-nitro-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one <b>(15)</b> (6.3 g, 22.6 mmol) in ethanol (113 mL), which was then stirred at 60 °C under an atmosphere of hydrogen for 3 h. The mixture was filtered through a pad of Celite and washed through with ethanol. The resulting solution was then concentrated under reduced pressure to give compound <b>16</b> (5.85 g) as a pale pink solid. δ<sub>H</sub> (500 MHz, CD<sub>3</sub>OD) 6.90 (d, <i>J</i> = 8.1 Hz, 1H), 6.63 (d, <i>J</i> = 1.9 Hz, 1H), 6.58 (dd, <i>J</i> = 8.1, 1.9 Hz, 1H), 3.95 (m, 2H), 3.36 (s, 3H), 1.84 (m, 2H), 1.29 (s, 6H).</div></div><div id="sec4_2_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> 2-Chloro-4-((3-(3-hydroxy-3-methylbutyl)-1-methyl-2-oxo-2,3-dihydro-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)amino)nicotinonitrile (<b>17a</b>)</h3><div class="NLM_p last">A mixture of 2,4-dichloropyridine-3-carbonitrile (11 mg, 0.064 mmol), 5-amino-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>16</b>) (15 mg, 0.06 mmol), and DIPEA (15 μL, 0.086 mmol) in NMP (0.5 mL) was heated in the microwave to 120 °C for 30 min and then purified by HPLC (ACE 5 C18-PFP 250 mm × 21.2 mm column, 15 min gradient from 45:55 to 20:80 water:methanol, 0.1% formic acid modifier) at a flow rate of 20 mL/min to give compound <b>17a</b> (11.2 mg, 0.027 mmol, 46%) as a white solid. HRMS (ESI +ve): found 386.1388, expected 386.1378 for C<sub>19</sub>H<sub>21</sub>ClN<sub>5</sub>O<sub>2</sub><sup>+</sup> [M + H]<sup>+</sup>. δ<sub>H</sub> (500 MHz, DMSO-<i>d</i><sub>6</sub>) 9.46 (s, 1H), 8.00 (d, <i>J</i> = 6.1 Hz, 1H), 7.20 (d, <i>J</i> = 8.3 Hz, 1H), 7.11 (d, <i>J</i> = 1.9 Hz, 1H), 7.00 (dd, <i>J</i> = 8.3, 2.0 Hz, 1H), 6.68 (d, <i>J</i> = 6.2 Hz, 1H), 4.44 (s, 1H), 3.92–3.85 (m, 2H), 3.34 (s, 3H), 1.73–1.66 (m, 2H), 1.15 (s, 6H).</div></div><div id="sec4_2_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> 5-((2,3-Dichloropyridin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>17b</b>)</h3><div class="NLM_p last">A mixture of 5-amino-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>16</b>) (20 mg, 0.08 mmol), Xantphos (12 mg, 0.02 mmol), 2,3-dichloro-4-iodopyridine (30 mg, 0.11 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (2.3 mg, 0.004 mmol), and cesium carbonate (67 mg, 0.21 mmol) in NMP:toluene (1:1 v/v, 1.8 mL) was degassed and heated in the microwave under a nitrogen atmosphere at 140 °C for 1 h. Purification was by reverse-phase flash chromatography (Biotage Ultra C18 12g, 30–90% methanol in water (0.1% formic acid modifier)) to give <b>17b</b> (7 mg, 0.018 mmol, 22%). HRMS (ESI +ve): found 395.1038, expected 395.1036 for C<sub>18</sub>H<sub>21</sub>Cl<sub>2</sub>N<sub>4</sub>O<sub>2</sub><sup>+</sup> [M + H]<sup>+</sup>. δ<sub>H</sub> (600 MHz, CDCl<sub>3</sub>) 7.90 (d, <i>J</i> = 5.6 Hz, 1H), 7.03 (d, <i>J</i> = 8.2 Hz, 1H), 7.01 (dd, <i>J</i> = 8.2, 1.8 Hz, 1H), 6.95 (d, <i>J</i> = 1.8 Hz, 1H), 6.68 (s, 1H), 6.64 (d, <i>J</i> = 5.7 Hz, 1H), 4.09–4.03 (m, 2H), 3.46 (s, 3H), 1.93–1.87 (m, 2H), 1.32 (s, 6H).</div></div><div id="sec4_2_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> 5-((3-Chloropyridin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>17c</b>)</h3><div class="NLM_p last">Method as for <b>17b</b>, using 4-bromo-3-chloropyridine. Purified by HPLC (ACE 5 C18-PFP 250 mm × 21.2 mm column, 15 min gradient from 40:60 to 25:75 water:methanol, 0.1% formic acid modifier) at a flow rate of 20 mL/min to give compound <b>17c</b> (21 mg, 69%, 0.055 mmol) as the formic acid salt. HRMS (ESI +ve): found 361.1422, expected 361.1426 for C<sub>18</sub>H<sub>22</sub>ClN<sub>4</sub>O<sub>2</sub><sup>+</sup> [M + H]<sup>+</sup>. δ<sub>H</sub> (500 MHz, acetone-<i>d</i><sub>6</sub>) 8.30 (s, 1H), 8.16 (s, 1H), 8.06 (d, <i>J</i> = 5.7 Hz, 1H), 7.18–7.13 (m, 2H), 7.06 (dd, <i>J</i> = 8.1, 2.0 Hz, 1H), 6.84 (d, <i>J</i> = 5.7 Hz, 1H), 4.03 (t, <i>J</i> = 8.2 Hz, 2H), 3.41 (s, 3H), 1.85 (t, <i>J</i> = 8.2 Hz, 2H), 1.25 (s, 6H).</div></div><div id="sec4_2_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> 4-Chloro-6-((3-(3-hydroxy-3-methylbutyl)-1-methyl-2-oxo-2,3-dihydro-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)amino)pyrimidine-5-carbonitrile (<b>17d</b>)</h3><div class="NLM_p last">Method as for <b>17a</b>, using 4,6-dichloropyrimidine-5-carbonitrile and heating at 80 °C for 30 min. <b>17d</b> (8.2 mg, 0.02 mmol, 33%) obtained as an off white solid. HRMS (ESI +ve): found 387.1340, expected 387.1331 for C<sub>18</sub>H<sub>20</sub>ClN<sub>6</sub>O<sub>2</sub><sup>+</sup> [M + H]<sup>+</sup> . δ<sub>H</sub> (500 MHz, DMSO-<i>d</i><sub>6</sub>) 10.25 (br s, 1H), 8.49 (s, 1H), 7.23 (s, 1H), 7.14 (s, 2H), 4.45 (br s, 1H), 3.93–3.81 (m, 2H), 3.33 (s, 3H), 1.75–1.64 (m, 2H), 1.16 (s, 6H).</div></div><div id="sec4_2_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> 5-((5,6-Dichloropyrimidin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>17e</b>)</h3><div class="NLM_p last">A mixture of 5-amino-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>16</b>) (30 mg, 0.12 mmol), DIPEA (30 uL, 0.17 mmol), and 4,5,6-trichloropyrimidine (26 mg, 0.14 mmol) in DMF (0.9 mL) was heated in the microwave to 120 °C for 30 min. The mixture was partitioned between water and DCM, and the aqueous layer was acidified to pH 4 using a 10% citric acid solution. Layers were separated, the aqueous phase was extracted with further DCM, and combined organics were evaporated under reduced pressure and then purified by reverse phase flash chromatography (Biotage 12g SNAP Ultra C18, 20–100% methanol in water, 0.1% formic acid modifier) and then further triturated with diethyl ether to give compound <b>17e</b> as a beige solid (41 mg, 0.10 mmol, 86%). HRMS (ESI +ve): found 396.0997, expected 396.0989 for C<sub>17</sub>H<sub>20</sub>Cl<sub>2</sub>N<sub>5</sub>O<sub>2</sub><sup>+</sup> [M + H]<sup>+</sup>. δ<sub>H</sub> (500 MHz, DMSO-<i>d</i><sub>6</sub>) 9.46 (s, 1H), 8.26 (s, 1H), 7.30 (d, <i>J</i> = 2.0 Hz, 1H), 7.21 (dd, <i>J</i> = 8.4, 2.0 Hz, 1H), 7.13 (d, <i>J</i> = 8.4 Hz, 1H), 4.45 (s, 1H), 3.91–3.84 (m, 2H), 3.33 (s, 3H, obscured by solvent), 1.74–1.66 (m, 2H), 1.16 (s, 6H).</div></div><div id="sec4_2_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> 5-((3,5-Dichloropyridin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>17f</b>)</h3><div class="NLM_p last">Method as for <b>17b</b>, using 4-bromo-3,5-dichloropyridine. Obtained compound <b>17f</b> (8 mg, 0.02 mmol, 25%). HRMS (ESI +ve): found 395.1027, expected 395.1036 for C<sub>18</sub>H<sub>21</sub>Cl<sub>2</sub>N<sub>4</sub>O<sub>2</sub><sup>+</sup> [M + H]<sup>+</sup>. δ<sub>H</sub> (600 MHz, CDCl<sub>3</sub>) 8.34 (s, 2H), 6.92 (d, <i>J</i> = 8.2 Hz, 1H), 6.78 (dd, <i>J</i> = 8.2, 2 Hz, 1H), 6.76 (d, <i>J</i> = 2.0 Hz, 1H), 6.58 (s, 1H), 4.04 (t, <i>J</i> = 7.4 Hz, 2H), 3.45 (s, 3H), 1.89 (t, <i>J</i> = 7.4 Hz, 2H), 1.30 (s, 6H).</div></div><div id="sec4_2_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> 5-((5-Chloro-2-(methylthio)pyrimidin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>17g</b>)</h3><div class="NLM_p last">Method as for <b>17e</b>, using 4,5-dichloro-2-methylsulfanylpyrimidine. Obtained compound <b>17g</b> as white solid (27 mg, 0.063 mmol, 78%). HRMS (ESI +ve): found 408.1208, expected 408.1255 for C<sub>18</sub>H<sub>23</sub>ClN<sub>5</sub>O<sub>2</sub>S<sup>+</sup> [M + H]<sup>+</sup> ]<sup>+</sup>. δ<sub>H</sub> (500 MHz, DMSO-<i>d</i><sub>6</sub>) 8.43 (br s, 1H), 8.24 (s, 1H), 7.41 (d, <i>J</i> = 1.6 Hz, 1H), 7.27 (dd, <i>J</i> = 8.4, 1.6 Hz, 1H), 7.13 (d, <i>J</i> = 8.4 Hz, 1H), 3.87 (m, 2H), 3.32 (s, 3H), 2.38 (s, 3H), 1.72 (m, 2H), 1.17 (s, 6H).</div></div><div id="sec4_2_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> 5-((2,5-Dichloropyrimidin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>18</b>)</h3><div class="NLM_p last">A mixture of 5-amino-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>16</b>) (1.61 g, 6.47 mmol), 2,4,5-trichloropyrimidine (0.88 mL, 7.69 mmol), and cesium carbonate (4.21 g, 12.93 mmol) in DMF (15 mL) was heated in the microwave to 120 °C for 30 min. The resulting mixture was diluted with water, acidified to pH 5 by addition of 10% citric acid, and extracted with DCM. The combined organics were evaporated under reduced pressure, and the resulting sticky solid was dissolved in a minimum volume of ethyl acetate and precipitated by addition of diethyl ether. The resulting solid was collected by filtration and washed with diethyl ether and dried under reduced pressure, giving the title product (1.96 g, 72%, 4.69 mmol) as a solid. δ<sub>H</sub> (500 MHz, DMSO-<i>d</i><sub>6</sub>) 9.57 (s, 1H), 8.34 (s, 1H), 7.35 (d, <i>J</i> = 1.9 Hz, 1H), 7.19 (dd, <i>J</i> = 8.4, 1.9 Hz, 1H), 7.15 (d, <i>J</i> = 8.4 Hz, 1H), 4.44 (s, 1H), 3.92–3.86 (m, 2H), 3.33 (s, 3H), 1.76–1.69 (m, 2H), 1.17 (s, 6H).</div></div><div id="sec4_2_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> 5-((5-Chloro-2-(dimethylamino)pyrimidin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>19</b>)</h3><div class="NLM_p last">A mixture of 5-((2,5-dichloropyrimidin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>18</b>) (20 mg, 0.05 mmol), dimethylamine hydrochloride (37 mg, 0.45 mmol), cesium carbonate (164 mg, 0.50 mmol) in NMP (0.5 mL) was heated in the microwave to 180 °C for 1 h. Mixture was partitioned between DCM and water, aqueous layer was extracted with DCM, and combined organics were washed with Brine. Organic layers were combined and evaporated, and the resulting NMP solution was purified by HPLC (ACE 5 C18-PFP 250 mm × 21.2 mm column; 15 min gradient of 40:60 to 25:75 water:methanol, 0.1% formic acid modifier) at a flow rate of 20 mL/min to give compound <b>19</b> (8 mg, 0.0188 mmol, 37%). HRMS (ESI +ve): found 405.1790, expected 405.1800 for C<sub>19</sub>H<sub>26</sub>ClN<sub>6</sub>O<sub>2</sub><sup>+</sup>. δ<sub>H</sub> (500 MHz, acetone-<i>d</i><sub>6</sub>) 7.96 (s, 1H), overlapping with 7.98–7.94 (0.6H, partly exchanged NH), 7.73 (d, <i>J</i> = 1.7 Hz, 1H), 7.35 (dd, <i>J</i> = 8.1, 1.7 Hz, 1H), 7.05 (d, <i>J</i> = 8.1 Hz, 1H), 4.02 (t, <i>J</i> = 7.9 Hz, 2H), 3.38 (s, 3H), 3.13 (s, 6H), 1.85 (t, <i>J</i> = 7.9 Hz, 2H), 1.25 (s, 6H).</div></div><div id="sec4_2_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> 5-((5-Chloro-2-morpholinopyrimidin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>20a</b>)</h3><div class="NLM_p last">A mixture of morpholine (0.031 mL, 0.25 mmol), 5-((2,5-dichloropyrimidin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>18,</b> 20 mg, 0.05 mmol), and DIPEA (0.04 mL, 0.25 mmol) in NMP (1 mL) was heated in the microwave to 180 °C for 1 h, then partitioned between DCM and water. The aqueous layer was extracted with DCM, and combined organics were washed with brine. The organic layers were combined and evaporated, and the resulting material (containing residual NMP) was purified by preparative HPLC (ACE 5 C18-PFP column (5 μm, 250 mm × 21.2 mm), 15 min gradient elution from 40:60 to 25:75 water:methanol (both modified with 0.1% formic acid) at a flow rate of 20 mL/min) to give <b>20a</b> (18 mg, 0.038 mmol, 76%). HRMS (ESI +ve): found 447.1897, expected 447.1906 for C<sub>21</sub>H<sub>28</sub>ClN<sub>6</sub>O<sub>3</sub><sup>+</sup> [M + H]<sup>+</sup>. δ<sub>H</sub> (500 MHz, CD<sub>3</sub>OD) 7.92 (s, 1H), 7.47 (d, <i>J</i> = 1.7 Hz, 1H), 7.31 (dd, <i>J</i> = 8.4, 1.7 Hz, 1H), 7.10 (d, <i>J</i> = 8.4 Hz, 1H), 4.00 (m, 2H), 3.70–3.67 (m, 4H), 3.66–3.62 (m, 4H), 3.41 (s, 3H), 1.84 (m, 2H), 1.28 (s, 6H).</div></div><div id="sec4_2_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> 5-((5-Chloro-2-(piperidin-1-yl)pyrimidin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>20b</b>)</h3><div class="NLM_p last">Method as for <b>20a</b>, using piperidine and heating to 170 °C for 1 h. Obtained compound <b>20b</b> (19 mg, 0.04 mmol, 80%) following HPLC. HRMS (ESI +ve): found 445.2111, expected 445.2113 for C<sub>22</sub>H<sub>30</sub>ClN<sub>6</sub>O<sub>2</sub><sup>+</sup> [M + H]<sup>+</sup>. δ<sub>H</sub> (500 MHz, acetone-<i>d</i><sub>6</sub>) 7.97 (br, partly exchanged NH, 0.3H), 7.96 (s, 1H), 7.61 (d, <i>J</i> = 1.8 Hz, 1H), 7.31 (dd, <i>J</i> = 8.7, 1.3 Hz, 1H), 7.05 (d, <i>J</i> = 8.7 Hz, 1H), 4.02 (m, 2H), 3.73 (br t, J 4.8 Hz, 4H), 3.38 (s, 3H), 1.86 (m, 2H), 1.67–1.62 (m, 2H), 1.59–1.52 (m, 4H), 1.25 (s, 6H).</div></div><div id="sec4_2_29" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> 5-((5-Chloro-2-(2,4-dimethylthiazol-5-yl)pyrimidin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>21</b>)</h3><div class="NLM_p last">A mixture of 5-((2,5-dichloropyrimidin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>18</b>) (20 mg, 0.05 mmol), 2,4-dimethyl-5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)thiazole (12.07 mg, 0.05 mmol), sodium carbonate (10.7 mg, 0.10 mmol), and bis(triphenylphosphine)palladium(II) chloride (1.8 mg, 0.0025 mmol) in 1,4-dioxane (0.40 mL) and water (0.40 mL) was heated in the microwave at 130 °C for 30 min. The mixture was partitioned between DCM and water and pH adjusted to 5 using 10% citric acid before separation and extraction with further DCM. The organic layers were combined and evaporated, and the resulting solution was purified by HPLC (ACE 5 C18-PFP 250 mm × 21.2 mm column, 15 min gradient from 40:60 to 25:75 water:methanol, 0.1% formic acid modifier) at a flow rate of 20 mL/min to give compound <b>21</b> (2 mg, 0.0042 mmol, 8%) as a white solid. HRMS (ESI +ve): found 473.1501, expected 473.1521 for C<sub>22</sub>H<sub>26</sub>ClN<sub>6</sub>O<sub>2</sub>S<sup>+</sup> [M + H]<sup>+</sup>. δ<sub>H</sub> (500 MHz, acetone-<i>d</i><sub>6</sub>) 8.50 (br, 1H), 8.38 (s, 1H), 7.54 (d, <i>J</i> = 1.8 Hz, 1H), 7.34 (dd, <i>J</i> = 8.3, 1.8 Hz, 1H), 7.13 (d, <i>J</i> = 8.3 Hz, 1H), 4.07 (t, <i>J</i> = 7.9 Hz, 2H), 3.41 (s, 3H), 2.63 (s, 3H), 2.60 (s, 3H), 1.89 (t, <i>J</i> = 7.9 Hz, 2H), 1.24 (s, 6H).</div></div><div id="sec4_2_30" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> 5-((5-Chloro-2-(1-methyl-1<i>H</i>-imidazol-2-yl)pyrimidin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>22</b>)</h3><div class="NLM_p last">A mixture of 1-methyl-2-(tributylstannyl)-1<i>H</i>-imidazole (21 mg, 0.06 mmol), 5-((2,5-dichloropyrimidin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>18</b>) (20 mg, 0.05 mmol), and bis(triphenylphosphine)palladium(II) chloride (3.5 mg, 0.005 mmol) in 1,4-dioxane (1 mL) was heated for 18 h at 90 °C. The mixture was partitioned between DCM and water and pH adjusted to 5 using 10% citric acid before separation and extraction with further DCM. The organic layers were combined and evaporated, and the resulting solution was purified by HPLC (ACE 5 C18-PFP 250 mm × 21.2 mm column, 15 min gradient from 40:60 to 25:75 water:methanol, 0.1% formic acid modifier) at a flow rate of 20 mL/min to give compound <b>22</b> (3 mg, 0.0068 mmol, 13%) as the formic acid salt. HRMS (ESI +ve): found 442.1740, expected 442.1753 for C<sub>21</sub>H<sub>25</sub>ClN<sub>7</sub>O<sub>2</sub><sup>+</sup> [M + H]<sup>+</sup>. δ<sub>H</sub> (500 MHz,, acetone-<i>d</i><sub>6</sub>) 8.50 (br, 1H), 8.37 (s, 1H), 8.12 (s, 1H), 7.74 (br s, 1H), 7.64 (br s, 1H), 7.55 (d, <i>J</i> = 2.0 Hz, 1H), 7.35 (dd, <i>J</i> = 8.6, 2.0 Hz, 1H), 7.14 (d, <i>J</i> = 8.6 Hz, 1H), 4.04 (t, <i>J</i> = 8.3 Hz, 2H), 3.91 (s, 3H), 3.40 (s, 3H), 1.88 (t, <i>J</i> = 8.3 Hz, 2H), 1.24 (s, 6H).</div></div><div id="sec4_2_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> 5-((5-Chloro-2-(1<i>H</i>-pyrazol-1-yl)pyrimidin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>23a</b>)</h3><div class="NLM_p last">Method as for <b>19</b>, using 1<i>H</i>-pyrazole, heating in the microwave to 170 °C for 1 h. Purified by HPLC (ACE 5 C18-PFP 250 mm × 21.2 mm column; 15 min gradient of 60:40 to 0:100 water:methanol, 0.1% formic acid modifier) at a flow rate of 20 mL/min to give compound <b>23a</b>. HRMS (ESI +ve): found 428.1552, expected 428.1596 for C<sub>20</sub>H<sub>23</sub>ClN<sub>7</sub>O<sub>2</sub><sup>+</sup> [M + H]<sup>+</sup>. δ<sub>H</sub> (500 MHz, DMSO-<i>d</i><sub>6</sub>) 9.42 (br, 1H), 8.45 (s, 1H), 8.38 (d, <i>J</i> = 2.2 Hz, 1H), 7.79 (s, 1H), 7.63 (d, <i>J</i> = 2.2 Hz, 1H), 7.42 (dd, <i>J</i> = 8.4, 2.2 Hz, 1H), 7.17 (d, <i>J</i> = 8.4 Hz, 1H), 6.52 (s, 1H), 3.91 (t, <i>J</i> = 7.6 Hz, 2H), 3.34 (s, 3H), 1.75 (t, <i>J</i> = 7.6 Hz, 2H), 1.14 (s, 6H).</div></div><div id="sec4_2_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> 5-((5-Chloro-2-(3-methyl-1<i>H</i>-pyrazol-1-yl)pyrimidin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>23b</b>) and 5-((5-Chloro-2-(5-methyl-1<i>H</i>-pyrazol-1-yl)pyrimidin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>23c</b>)</h3><div class="NLM_p last">A mixture of 5-((2,5-dichloropyrimidin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>18</b>) (25 mg, 0.063 mmol), 3-methylpyrazole (0.05 mL, 0.62 mmol), and cesium carbonate (100 mg, 0.31 mmol) in NMP (0.5 mL) was heated in the microwave to 170 °C for 1 h. The mixture was partitioned between DCM and water and pH adjusted to 5 using 10% citric acid before separation and extraction with further DCM. The organic layers were combined and evaporated, and the resulting solution was purified by HPLC (ACE 5 C18-PFP 250 mm × 21.2 mm column; 15 min gradient of 45:55 to 20:80 water:methanol, 0.1% formic acid modifier) at a flow rate of 20 mL/min. Two products were obtained: the major, later eluting regioisomer was assigned as compound <b>23b</b> (18 mg, 0.039 mmol, 61%). HRMS (ESI +ve): found 442.1755, expected 442.1753 for C<sub>21</sub>H<sub>25</sub>ClN<sub>7</sub>O<sub>2</sub><sup>+</sup> [M + H]<sup>+</sup>. δ<sub>H</sub> (500 MHz, CDCl<sub>3</sub>) 8.34 (s, 1H), 8.27 (d, <i>J</i> = 2.6 Hz, 1H), 7.63 (d, <i>J</i> = 2.0 Hz, 1H), 7.32 (s, 1H, NH), 7.18 (dd, <i>J</i> = 8.3, 2.0 Hz, 1H), 7.02 (d, <i>J</i> = 8.3 Hz, 1H), 6.23 (d, <i>J</i> = 2.6 Hz, 1H), 4.15–4.08 (m, 2H), 3.47 (s, 3H), 2.40 (s, 3H), 1.94 (t, <i>J</i> = 7.3 Hz, 2H), 1.30 (s, 6H). The minor, earlier eluting regioisomer was assigned as compound <b>23c</b> (2.5 mg, 0.005 mmol, 8.5%). HRMS (ESI +ve): found 442.1756, expected 442.1753 for C<sub>21</sub>H<sub>25</sub>ClN<sub>7</sub>O<sub>2</sub><sup>+</sup> [M + H]<sup>+</sup>. δ<sub>H</sub> (500 MHz,, CDCl<sub>3</sub>) 8.37 (s, 1H), 7.74 (d, <i>J</i> = 2.0 Hz, 1H), 7.63 (m, 1H), 7.29 (s, 1H, NH), 7.08 (dd, <i>J</i> = 8.3, 2.0 Hz, 1H), 6.98 (d, <i>J</i> = 8.3 Hz, 1H), 6.16 (s, 1H), 4.13–4.06 (m, 2H), 3.45 (s, 3H), 2.46 (s, 3H), 1.97–1.91 (m, 2H), 1.29 (s, 6H).</div></div><div id="sec4_2_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> 5-((5-Chloro-2-(3,5-dimethyl-1<i>H</i>-pyrazol-1-yl)pyrimidin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>23d</b>)</h3><div class="NLM_p last">Method as for <b>23b</b>, using 3,5-dimethyl-1<i>H</i>-pyrazole. Purified by HPLC (ACE 5 C18-PFP 250 mm × 21.2 mm column; 15 min gradient of 40:60 to 25:75 water:methanol, 0.1% formic acid modifier) at a flow rate of 20 mL/min, then further purified by flash column chromatography (12 g KP-SIL, 0–6% methanol in DCM) to give compound <b>23d</b>. HRMS (ESI +ve): found 456.1928, expected 456.1909 for C<sub>22</sub>H<sub>27</sub>ClN<sub>7</sub>O<sub>2</sub> [M + H]<sup>+</sup>. δ<sub>H</sub> (500 MHz, CDCl<sub>3</sub>) 8.37 (s, 1H), 7.44 (d, <i>J</i> = 1.9 Hz, 1H), 7.22 (s, 1H), 7.13 (dd, <i>J</i> = 1.9, 8.2 Hz, 1H), 6.98 (d, <i>J</i> = 8.2 Hz, 1H), 5.95 (s, 1H), 4.08 (t, <i>J</i> = 7.3 Hz, 2H), 3.45 (s, 3H), 2.33 (s, 3H), 2.31 (s, 3H), 1.89 (t, <i>J</i> = 7.4 Hz, 2H), 1.35–1.22 (m, 6H).</div></div><div id="sec4_2_34" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> 5-((5-Chloro-2-(3-methylpiperidin-1-yl)pyrimidin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>24a</b>)</h3><div class="NLM_p last">Method as for <b>20a</b>, using 3-methylpiperidine and heating to 170 °C for 1 h. Obtained <b>24a</b> (18 mg, 0.037 mmol, 74%) as the formic acid salt after HPLC purification. HRMS (ESI +ve): found 459.2239, expected 459.2270 for C<sub>23</sub>H<sub>32</sub>ClN<sub>6</sub>O<sub>2</sub><sup>+</sup> [M + H]<sup>+</sup>. δ<sub>H</sub> (500 MHz, acetone-<i>d</i><sub>6</sub>) 8.14 (s, 1H, formic acid), 7.98 (br, 0.8H, partly exchanged NH), 7.95 (s, 1H), 7.57 (d, <i>J</i> = 2.0 Hz, 1H), 7.35 (dd, <i>J</i> = 8.8, 2.0 Hz, 1H), 7.06 (d, <i>J</i> = 8.8 Hz, 1H), 4.59–4.50 (m, 2H), 4.01 (m, 2H), 3.37 (s, 3H), 2.83 (td, <i>J</i> = 12.5, 2.9 Hz, 1H), 2.51 (dd, <i>J</i> = 12.9, 10.6 Hz, 1H), 1.86 (m, 2H), 1.80 (dm, <i>J</i> = 12.9 Hz, 1H),1.69 (dp, <i>J</i> = 13.2, 3.4 Hz, 1H), 1.57 (m, 1H), 1.46 (qt, <i>J</i> = 13.2, 12.5, 12, 4 Hz, 1H), 1.25 (s, 6H), 1.16 (app qd, <i>J</i> = 12.1, 3.8 Hz, 1H), 0.90 (d, <i>J</i> = 6.1 Hz, 2H).</div></div><div id="sec4_2_35" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> 5-((5-Chloro-2-((3<i>S</i>,5<i>R</i>)-3,5-dimethylpiperidin-1-yl)pyrimidin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>24b</b>)</h3><div class="NLM_p last">Method as for <b>20a</b> using (3<i>R</i>,5<i>S</i>)-3,5-dimethylpiperidine. Obtained <b>24b</b> (11 mg, 0.022 mmol, 88%) as the formic acid salt. HRMS (ESI +ve): found 473.2392, expected 473.2426 for C<sub>24</sub>H<sub>34</sub>ClN<sub>6</sub>O<sub>2</sub><sup>+</sup> [M + H]<sup>+</sup>. δ<sub>H</sub> (500 MHz, acetone-<i>d</i><sub>6</sub>) 8.31 (br s, 0.6H, partly exchanged NH), 8.13 (s, 1H), 7.96 (s, 1H), 7.51 (d, <i>J</i> = 2.0 Hz, 1H), 7.39 (dd, <i>J</i> = 8.4, 2.0 Hz, 1H), 7.09 (d, <i>J</i> = 8.4 Hz, 1H), 4.66 (br d, <i>J</i> = 12.6 Hz, 2H), 4.04 (m, 2H), 3.39 (s, 3H), 2.33 (dd, <i>J</i> = 12.6, 11.7 Hz, 2H), 1.86 (m, 2H), 1.81 (m, 1H), 1.59 (m, 2H), 1.26 (s, 6H), 0.90 (d, <i>J</i> = 6.5 Hz, 6H), 0.82 (m, 1H).</div></div><div id="sec4_2_36" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> 5-((5-Chloro-2-((2<i>R</i>,6<i>S</i>)-2,6-dimethylmorpholino)pyrimidin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>25a</b>)</h3><div class="NLM_p last">Method as for <b>20a</b>, using (2<i>R</i>,6<i>S</i>)-2,6-dimethylmorpholine and heating to 170 °C for 1 h. Obtained compound <b>25a</b> (16 mg, 0.032 mmol, 63%) after HPLC purification. HRMS (ESI +ve): found 475.2169, expected 475.2219 for C<sub>23</sub>H<sub>32</sub>ClN<sub>6</sub>O<sub>3</sub> [M + H]<sup>+</sup>. δ<sub>H</sub> (500 MHz, acetone-<i>d</i><sub>6</sub>) 8.07 (br, 1H), 7.98 (s, 1H), 7.57 (d, <i>J</i> = 1.8 Hz, 1H), 7.31 (dd, <i>J</i> = 8.7, 1.8 Hz, 1H), 7.07 (d, <i>J</i> = 8.7 Hz, 1H), 4.46 (dd, <i>J</i> = 13.3, 2.3 Hz, 2H), 4.03 (t, <i>J</i> = 8.0 Hz, 2H), 3.55 (dqd, <i>J</i> = 10.5, 6.2, 2.3 Hz, 2H), 3.37 (s, 3H), 2.48 (dd, <i>J</i> = 13.3, 10.5 Hz, 2H), 1.86 (t, <i>J</i> = 8.0 Hz, 2H), 1.26 (s, 6H), 1.16 (d, <i>J</i> = 6.2 Hz, 6H).</div></div><div id="sec4_2_37" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> 5-((5-Chloro-2-(2,2,6,6-tetramethylmorpholino)pyrimidin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>25b</b>)</h3><div class="NLM_p last">A mixture of 5-((2,5-dichloropyrimidin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>18</b>) (30 mg, 0.076 mmol), 2,2,6,6-tetramethylmorpholine (22 mg, 0.15 mmol), and DIPEA (40 μL, 0.23 mmol) in NMP (0.67 mL) was heated in the microwave to 140 °C for 2 h. Once cooled, the mixture was diluted with DMSO (0.5 mL) and then purified by reverse-phase chromatography eluting from 30% to 100% methanol in water (each containing 0.1% formic acid) to give compound <b>25b</b> (32 mg, 0.064 mmol, 84%) as a pale brown solid. HRMS (ESI +ve): found 503.2521, expected 503.2532 for C<sub>25</sub>H<sub>36</sub>ClN<sub>6</sub>O<sub>3</sub><sup>+</sup> [M + H]<sup>+</sup>. δ<sub>H</sub> (600 MHz, CDCl<sub>3</sub>) 7.97 (s, 1H), 7.36 (d, <i>J</i> = 2.0 Hz, 1H), 7.24 (dd, <i>J</i> = 8.4, 2.0 Hz, 1H), 7.00 (s, 1H), 6.93 (d, <i>J</i> = 8.4 Hz, 1H), 4.04 (t, <i>J</i> = 7.2 Hz, 2H), 3.58 (s, 4H), 3.42 (s, 3H), 1.89 (t, <i>J</i> = 7.2 Hz, 2H), 1.28 (s, 6H), 1.23 (s, 12H).</div></div><div id="sec4_2_38" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> 5-((5-Chloro-2-((3<i>S</i>,5<i>R</i>)-3,4,5-trimethylpiperazin-1-yl)pyrimidin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>25c</b>)</h3><div class="NLM_p last">Method as for <b>25b</b>, using (2<i>S</i>,6<i>R</i>)-1,2,6-trimethylpiperazine and heating for 1 h at 140 °C. Purification by HPLC (ACE 5 C18-PFP 250 mm × 21.2 mm column; 15 min gradient of 90:10 to 0:100 water:methanol, 0.1% formic acid modifier) at a flow rate of 20 mL/min followed by further purification using an SCX-2 column gave compound <b>25c</b> (2 mg, 0.004 mmol, 11%). HRMS (ESI +ve): found 488.2518, expected 488.2535 for C<sub>24</sub>H<sub>35</sub>ClN<sub>7</sub>O<sub>2</sub><sup>+</sup> [M + H]<sup>+</sup>. δ<sub>H</sub> (600 MHz, CD<sub>3</sub>OD) 7.94 (s, 1H), 7.43 (d, <i>J</i> = 2.0 Hz, 1H), 7.37 (dd, <i>J</i> = 8.4, 2.0 Hz, 1H), 7.14 (d, <i>J</i> = 8.4 Hz, 1H), 4.40 (br d, <i>J</i> = 13.4 Hz, 2H), 4.04 (m, 2H), 3.45 (s, 3H), 2.70 (dd, <i>J</i> = 13.4, 11.0 Hz, 2H), 2.34 (s, 3H), 2.32–2.22 (m, 2H), 1.87 (m, 2H), 1.30 (s, 6H), 1.15 (d, <i>J</i> = 6.2 Hz, 6H).</div></div><div id="sec4_2_39" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> 5-((5-Chloro-2-(3,5-dimethyl-1<i>H</i>-pyrazol-1-yl)pyridin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>26a</b>)</h3><div class="NLM_p last">A mixture of 3,5-dimethyl-1<i>H</i>-pyrazole (3 mg, 0.03 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (1.5 mg, 0.0017 mmol), Xantphos (4.1 mg, 0.007 mmol), cesium carbonate (14 mg, 0.04 mmol), and 5-((2-bromo-5-chloropyridin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>29</b>, 9 mg, 0.02 mmol) in NMP:toluene (1:1 v/v, 0.8 mL) was heated in the microwave to 140 °C for 1 h. The resulting mixture was diluted with water, acidified to pH 5 by addition of 10% citric acid, then extracted with DCM (5 mL × 3). Organic layer was collected and passed through a Si-DMT palladium scavenging column, then evaporated under reduced pressure. Product was purified by HPLC (ACE 5 C18-PFP 250 mm × 21.2 mm column; 15 min gradient of 40:60 to 25:75 water:methanol, 0.1% formic acid modifier) at a flow rate of 20 mL/min to give <b>26a</b> (4 mg, 0.0084 mmol, 41%). HRMS (ESI +ve): found 455.1943, expected 455.1957 for C<sub>23</sub>H<sub>28</sub>ClN<sub>6</sub>O<sub>2</sub><sup>+</sup> [M + H]<sup>+</sup>. δ<sub>H</sub> (500 MHz, acetone-<i>d</i><sub>6</sub>) 8.19–8.14 (m, 1.6H, including partly exchanged NH), 7.39 (s, 1H), 7.19–7.16 (m, 2H), 7.09 (dd, <i>J</i> = 8.2, 2.0 Hz, 1H), 5.97 (s, 1H), 4.03 (t, <i>J</i> = 7.9 Hz, 2H), 3.42 (s, 3H), 2.57 (s, 3H), 2.10 (s, 3H), 1.87 (t, <i>J</i> = 7.9 Hz, 2H), 1.23 (6H, s).</div></div><div id="sec4_2_40" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> 5-((5-Chloro-2-((3<i>S</i>,5<i>R</i>)-3,5-dimethylpiperidin-1-yl)pyridin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>26b</b>)</h3><div class="NLM_p last">A mixture of 5-amino-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>16</b>, 15 mg, 0.06 mmol), 5-chloro-2-((3<i>S</i>,5<i>R</i>)-3,5-dimethylpiperidin-1-yl)-4-iodopyridine <b>30a</b> (25.3 mg, 0.072 mmol), cesium carbonate (157 mg, 0.48 mmol), Xantphos (20.9 mg, 0.036 mmol), and Pd<sub>2</sub>(dba)<sub>3</sub> (5.5 mg, 0.006 mmol) in toluene:DMF (3:1 v/v, 0.8 mL) was heated in the microwave at 80 °C for 1 h. EtOAc and aq NH<sub>4</sub>Cl were added, layers were separated, and aqueous layer was extracted with further EtOAc. The organic layers were combined, dried over MgSO<sub>4</sub>, and concentrated under vacuum. Purification was by flash column chromatography (0–10% MeOH in DCM), followed by further purification by HPLC (ACE 5 C18-PFP 250 mm × 21.2 mm column; 15 min gradient of 60:40 to 0:100 water:methanol, 0.1% formic acid modifier) at a flow rate of 20 mL/min to give compound <b>26b</b> (13 mg, 0.028 mmol, 46%) as formic acid salt. HRMS (ESI +ve): found 472.2482, expected 472.2474 for C<sub>25</sub>H<sub>35</sub>ClN<sub>5</sub>O<sub>2</sub><sup>+</sup> [M + H]<sup>+</sup>. δ<sub>H</sub> (500 MHz, CD<sub>3</sub>OD) 8.27 (br, 1H), 7.84 (s, 1H), 7.22 (d, <i>J</i> = 8.3 Hz, 1H), 7.18 (d, <i>J</i> = 1.8 Hz, 1H), 7.10 (dd, <i>J</i> = 8.3, 1.8 Hz, 1H), 6.12 (s, 1H), 4.03 (m, 2H), 3.81 (dm, <i>J</i> = 13 Hz, 2H), 3.46 (s, 3H), 2.30 (dd, <i>J</i> = 13, 11.4 Hz, 2H), 1.86 (m, 2H), 1.80 (m, 1H), 1.62 (m, 2H), 1.28 (s, 6H), 0.87 (d, <i>J</i> = 6.6 Hz, 6H), 0.76 (dt, <i>J</i> = 12.9, 11.8 Hz, 1H).</div></div><div id="sec4_2_41" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> 5-((5-Chloro-2-((2<i>R</i>,6<i>S</i>)-2,6-dimethylmorpholino)pyridin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>26c</b>)</h3><div class="NLM_p last">Method as for <b>26b</b>, using (2<i>S</i>,6<i>R</i>)-4-(5-chloro-4-iodo-2-pyridyl)-2,6-dimethylmorpholine <b>30b</b>. Further purification by HPLC (ACE 5 C18-PFP 250 mm × 21.2 mm column; 15 min gradient of 45:55 to 20:80 water:methanol, 0.1% formic acid modifier) at a flow rate of 20 mL/min gave compound <b>26c</b> (5 mg, 0.011 mmol, 26%). HRMS (ESI +ve): found 474.2246, expected 474.2266 for C<sub>24</sub>H<sub>33</sub>ClN<sub>5</sub>O<sub>3</sub><sup>+</sup> [M + H]<sup>+</sup>. δ<sub>H</sub> (500 MHz,, CD<sub>3</sub>OD) 7.86 (s, 1H), 7.21 (d, <i>J</i> = 8.3 Hz, 1H), 7.16 (d, <i>J</i> = 1.6 Hz, 1H), 7.09 (dd, <i>J</i> = 8.3, 1.6 Hz, 1H), 6.13 (s, 1H), 4.03 (m, 2H), 3.73 (br dd, <i>J</i> = 12.3, 2.3 Hz, 2H), 3.63 (m, 2H), 3.46 (s, 3H), 2.34 (dd, <i>J</i> = 12.5, 10.6 Hz, 2H), 1.86 (m, 2H), 1.29 (s, 6H), 1.16 (d, <i>J</i> = 6.3 Hz, 6H).</div></div><div id="sec4_2_42" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> 5-((5-Chloro-2-(3-(trifluoromethyl)piperidin-1-yl)pyrimidin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>27a</b>)</h3><div class="NLM_p last">Method as for <b>20a</b>, using 3-(trifluoromethyl)piperidine, with heating for 1 h at 170 °C. Obtained compound <b>27a</b> (13 mg, 0.025 mmol, 50%) after purification by HPLC. HRMS (ESI +ve): found 513.1980, expected 513.1987 for C<sub>23</sub>H<sub>29</sub>ClF<sub>3</sub>N<sub>6</sub>O<sub>2</sub><sup>+</sup> [M + H]<sup>+</sup>. δ<sub>H</sub> (500 MHz, acetone-<i>d</i><sub>6</sub>) 8.11 (br s, 1H), 8.00 (s, 1H), 7.49 (d, <i>J</i> = 1.6 Hz, 1H), 7.37 (dd, <i>J</i> = 8.4, 1.6 Hz, 1H), 7.05 (d, <i>J</i> = 8.4 Hz, 1H), 4.88 (br d, <i>J</i> = 12.5 Hz, 1H), 4.63 (br d, <i>J</i> = 13.3 Hz, 1H), 4.06–3.98 (m, 2H), 3.38 (s, 3H), 2.96–2.85 (m, 3H), 2.40 (m, 1H), 1.88–1.79 (m, 3H), 1.66–1.50 (m, 2H), 1.29 (s, 6H).</div></div><div id="sec4_2_43" class="NLM_sec NLM_sec_level_2"><div id="ac_i66" class="anchor-spacer"></div><h3 class="article-section__title" id="_i66"> 5-((5-Chloro-2-(4,4-difluoropiperidin-1-yl)pyrimidin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>27b</b>)</h3><div class="NLM_p last">Method as for <b>25b</b>, using 4,4-difluoropiperidine, with heating for 1 h at 140 °C. Purification by HPLC (ACE 5 C18-PFP 250 mm × 21.2 mm column; 15 min gradient of 90:10 to 0:100 water:methanol, 0.1% formic acid modifier) at a flow rate of 20 mL/min gave compound <b>27b</b> (11 mg, 0.023 mmol. 55%). HRMS (ESI +ve): found 503.1729, expected 503.1744 for C<sub>22</sub>H<sub>27</sub>ClF<sub>2</sub>N<sub>6</sub>NaO<sub>2</sub><sup>+</sup> [M + Na]<sup>+</sup>. δ<sub>H</sub> (600 MHz, CD<sub>3</sub>OD) 7.94 (s, 1H), 7.44 (d, <i>J</i> = 1.9 Hz, 1H), 7.32 (dd, <i>J</i> = 8.4, 2.0 Hz, 1H), 7.12 (d, <i>J</i> = 8.4 Hz, 1H), 4.11–3.95 (m, 2H), 3.84 (br t, <i>J</i> = 5.9 Hz, 4H), 3.43 (s, 3H), 1.95 (tt, <i>J</i> = 13.8, 5.9 Hz, 4H), 1.89–1.79 (m, 2H), 1.29 (s, 6H).</div></div><div id="sec4_2_44" class="NLM_sec NLM_sec_level_2"><div id="ac_i67" class="anchor-spacer"></div><h3 class="article-section__title" id="_i67"> 5-((5-Chloro-2-(4,4-difluoro-3-methylpiperidin-1-yl)pyrimidin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>27c</b>)</h3><div class="NLM_p last">Method as for <b>27b</b>, using 4,4-difluoro-3-methylpiperidine. Obtained compound <b>27c</b> (11 mg, 0.022 mmol, 52%) as hemiformic acid salt. HRMS (ESI +ve): found 495.2087, expected 495.2081 for C<sub>23</sub>H<sub>30</sub>ClF<sub>2</sub>N<sub>6</sub>O<sub>2</sub><sup>+</sup> [M + H]<sup>+</sup>. δ<sub>H</sub> (600 MHz, CD<sub>3</sub>OD) 8.28 (s, 0.5H, formic), 7.94 (s, 1H), 7.42 (d, <i>J</i> = 1.9 Hz, 1H), 7.35 (dd, <i>J</i> = 8.4, 2.0 Hz, 1H), 7.13 (d, <i>J</i> = 8.4 Hz, 1H), 4.40 (br d, <i>J</i> = 13.6 Hz, 1H), 4.32 (br d, <i>J</i> = 13.4 Hz, 1H), 4.09–3.96 (m, 2H), 3.44 (s, 3H), 3.28 (ddd, <i>J</i> = 13.6, 11.4, 3.4 Hz, 1H)), 2.98 (dd, <i>J</i> = 13.4, 10.2 Hz, 1H), 2.15–1.95 (m, 2H), 1.87 (m, 2H) overlapping with 1.88–1.78 (m, 1H), 1.29 (s, 6H), 1.00 (d, <i>J</i> = 6.8 Hz, 3H).</div></div><div id="sec4_2_45" class="NLM_sec NLM_sec_level_2"><div id="ac_i68" class="anchor-spacer"></div><h3 class="article-section__title" id="_i68"> 5-((5-Chloro-2-(3-(hydroxymethyl)piperidin-1-yl)pyrimidin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>27d</b>)</h3><div class="NLM_p last">Method as for <b>27b</b>, using piperidin-3-ylmethanol. Obtained <b>27d</b> (18.5 mg, 0.039 mmol, 77%) as formic acid salt. HRMS (ESI +ve): found 475.2224, expected 475.2219 for C<sub>23</sub>H<sub>32</sub>ClN<sub>6</sub>O<sub>3</sub><sup>+</sup> [M + H]<sup>+</sup>. δ<sub>H</sub> (600 MHz, CD<sub>3</sub>OD) 8.14 (br s, 1H), 7.89 (s, 1H), 7.46 (d, <i>J</i> = 1.9 Hz, 1H), 7.39 (dd, <i>J</i> = 8.4, 2.0 Hz, 1H), 7.10 (d, <i>J</i> = 8.4 Hz, 1H), 4.47 (dd, <i>J</i> = 13.1, 3.7 Hz, 1H), 4.32 (dt, <i>J</i> = 13.1, 4.2 Hz, 1H), 4.02 (m, 2H), 3.46 (dd, <i>J</i> = 10.9, 5.7 Hz, 1H), 3.42 (s, 3H), overlapping with 3.41 (dd, <i>J</i> = 10.9, 7.2 Hz, 1H), 3.01 (td, <i>J</i> = 13.1, 11.2, 3.1 Hz, 1H), 2.80 (dd, <i>J</i> = 13.1, 10.1 Hz, 1H), 1.89–1.80 (m, 3H), 1.74–1.66 (m, 2H), 1.49 (m, 1H), 1.34–1.26 (m, 1H) overlapping with 1.29 (s, 6H).</div></div><div id="sec4_2_46" class="NLM_sec NLM_sec_level_2"><div id="ac_i69" class="anchor-spacer"></div><h3 class="article-section__title" id="_i69"> 5-((5-Chloro-2-(4,4-difluoro-3-(hydroxymethyl)piperidin-1-yl)pyrimidin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>27e</b>)</h3><div class="NLM_p last">Method as for <b>27b</b>, using (4,4-difluoro-3-piperidyl)methanol. Obtained <b>27e</b> (13 mg, 0.025 mmol, 65%) as formic acid salt. HRMS (ESI +ve): found 511.2036, expected 511.2030 for C<sub>23</sub>H<sub>30</sub>ClF<sub>2</sub>N<sub>6</sub>O<sub>3</sub><sup>+</sup> [M + H]<sup>+</sup>. δ<sub>H</sub> (600 MHz, CD<sub>3</sub>OD) 8.22 (s, 1H), 7.95 (s, 1H), 7.43 (d, <i>J</i> = 1.9 Hz, 1H), 7.41 (dd, <i>J</i> = 8.4, 1.9 Hz, 1H), 7.12 (d, <i>J</i> = 8.4 Hz, 1H), 4.50 (br d, <i>J</i> = 13.6 Hz, 1H), 4.26 (m, 1H), 4.03 (m, 2H), 3.88 (dd, <i>J</i> = 11.2, 4.1 Hz, 1H), 3.48 (dd, <i>J</i> = 11.2, 9.2 Hz, 1H), 3.44 (m, 1H) overlapping with 3.43 (s, 3H), 3.31 (dd, <i>J</i> = 13.6, 9.4 Hz, 1H), 2.13 (m, 1H), 2.01 (m, 1H), 1.96–1.86 (m, 1H) overlapping with 1.87 (m, 2H), 1.30 (s, 6H).</div></div><div id="sec4_2_47" class="NLM_sec NLM_sec_level_2"><div id="ac_i70" class="anchor-spacer"></div><h3 class="article-section__title" id="_i70"> 5-((5-Chloro-2-(4,4-difluoro-3-(methoxymethyl)piperidin-1-yl)pyrimidin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>27f</b>)</h3><div class="NLM_p last">Method as for <b>27b</b>, using 4,4-difluoro-3-(methoxymethyl)piperidine. Obtained <b>27f</b> (5 mg, 0.001 mmol, 22%) as formic acid salt. HRMS (ESI +ve): found 525.2171, expected 525.2187 for C<sub>24</sub>H<sub>32</sub>ClF<sub>2</sub>N<sub>6</sub>O<sub>3</sub><sup>+</sup> [M + H]<sup>+</sup>. δ<sub>H</sub> (600 MHz, CD<sub>3</sub>OD) 8.22 (formic acid), 7.96 (s, 1H), 7.47 (d, <i>J</i> = 1.9 Hz, 1H), 7.36 (dd, <i>J</i> = 8.4, 1.9 Hz, 1H), 7.12 (d, <i>J</i> = 8.4 Hz, 1H), 4.56–4.46 (br d, <i>J</i> = 13.6 Hz, 1H), 4.39–4.29 (dm, <i>J</i> = 13.6 Hz, 1H), 4.05 (m, 2H), 3.66 (dd, <i>J</i> = 9.6, 3.6 Hz, 1H), 3.45 (s, 3H), 3.41–3.35 (m, 1H), 3.32–3.33 (m, 1H), 3.27–3.22 (m, 1H), 3.22 (s, 3H), 2.28–2.17 (m, 1H), 2.07–1.97 (m, 1H), 1.97–1.90 (m, 1H), 1.87 (dd, <i>J</i> = 9.1, 7.3 Hz, 2H), 1.30 (s, 6H).</div></div><div id="sec4_2_48" class="NLM_sec NLM_sec_level_2"><div id="ac_i71" class="anchor-spacer"></div><h3 class="article-section__title" id="_i71"> 1-(5-Chloro-4-((3-(3-hydroxy-3-methylbutyl)-1-methyl-2-oxo-2,3-dihydro-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)amino)pyrimidin-2-yl)piperidine-3-carbonitrile (<b>27g</b>)</h3><div class="NLM_p last">Method as for <b>27b</b>, using piperidine-3-carbonitrile hydrochloride. Obtained <b>27g</b> (20 mg, 0.04 mmol, 80%). HRMS (ESI +ve): found 470.2053, expected 470.2066 for C<sub>23</sub>H<sub>29</sub>ClN<sub>7</sub>O<sub>2</sub><sup>+</sup> [M + H]<sup>+</sup>. δ<sub>H</sub> (600 MHz, CD<sub>3</sub>OD) 7.93 (d, <i>J</i> = 1.1 Hz, 1H), 7.45 (d, <i>J</i> = 2.0 Hz, 1H), 7.33 (dd, <i>J</i> = 8.4, 2.0 Hz, 1H), 7.11 (dd, <i>J</i> = 8.4, 1.1 Hz, 1H), 4.06–3.99 (m, 2H), 3.93 (d, <i>J</i> = 5.2 Hz, 2H), 3.78–3.66 (m, 2H), 3.42 (s, 3H), 2.95–2.88 (m, 1H), 2.06–1.98 (m, 1H), 1.96–1.84 (m, 2H + 1H), 1.80–1.70 (m, 1H), 1.64–1.54 (m, 1H), 1.29 (s, 6H).</div></div><div id="sec4_2_49" class="NLM_sec NLM_sec_level_2"><div id="ac_i72" class="anchor-spacer"></div><h3 class="article-section__title" id="_i72"> 5-((5-Chloro-2-(4-(trifluoromethyl)piperidin-1-yl)pyrimidin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>27h</b>)</h3><div class="NLM_p last">Method as for <b>27b</b>, using 4-(trifluoromethyl)piperidine. Obtained <b>27h</b> (18 mg, 0.035 mmol, 82%) as formic acid salt. HRMS (ESI +ve): found 535.1787, expected 535.1807 for C<sub>23</sub>H<sub>28</sub>ClF<sub>3</sub>N<sub>6</sub>NaO<sub>2</sub><sup>+</sup> [M + Na]<sup>+</sup>. δ<sub>H</sub> (600 MHz, CD<sub>3</sub>OD) 8.21 (s, 1H), 7.92 (s, 1H), 7.46 (d, <i>J</i> = 1.9 Hz, 1H), 7.33 (dd, <i>J</i> = 8.4, 1.9 Hz, 1H), 7.11 (d, <i>J</i> = 8.4 Hz, 1H), 4.68 (dt, <i>J</i> = 13.5, 2.8 Hz, 2H), 4.01 (m, 2H), 3.43 (s, 3H), 2.85 (td, <i>J</i> = 13.5, 12.7, 2.6 Hz, 2H), 2.42 (m (tqt), 1H), 1.95–1.77 (m, 4H), 1.46 (qd, <i>J</i> = 12.6, 4.4 Hz, 2H), 1.29 (s, 6H).</div></div><div id="sec4_2_50" class="NLM_sec NLM_sec_level_2"><div id="ac_i73" class="anchor-spacer"></div><h3 class="article-section__title" id="_i73"> 5-((5-Chloro-2-(8,8-difluoro-3-azabicyclo[3.2.1]octan-3-yl)pyrimidin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>28a</b>)</h3><div class="NLM_p last">Method as for <b>25b</b>, using 8,8-difluoro-3-azabicyclo[3.2.1]octane hydrochloride. Purification by HPLC (ACE 5 C18-PFP 250 mm × 21.2 mm column; 15 min gradient of 90:10 to 0:100 water:methanol, 0.1% formic acid modifier) at a flow rate of 20 mL/min gave compound <b>28a</b> (9.5 mg, 0.019 mmol, 45%) as hemiformic acid salt. HRMS (ESI +ve): found 507.2073, expected 507.2081 for C<sub>24</sub>H<sub>30</sub>F<sub>2</sub>N<sub>6</sub>O<sub>2</sub>Cl<sup>+</sup> [M + H]<sup>+</sup>. δ<sub>H</sub> (600 MHz, CD<sub>3</sub>OD) δ 8.19 (s, 0.5H, formic), 7.94 (s, 1H), 7.47 (d, <i>J</i> = 2.0 Hz, 1H), 7.33 (dd, <i>J</i> = 8.4, 2.0 Hz, 1H), 7.12 (d, <i>J</i> = 8.4 Hz, 1H), 4.41 (br d, <i>J</i> = 13.4 Hz, 2H), 4.02 (m, 2H), 3.43 (s, 3H), 3.23 (br d, <i>J</i> = 12.9 Hz, 2H), 2.28 (m, 2H), 1.90–1.83 (m, 4H), 1.65–1.53 (m, 2H), 1.29 (s, 6H).</div></div><div id="sec4_2_51" class="NLM_sec NLM_sec_level_2"><div id="ac_i74" class="anchor-spacer"></div><h3 class="article-section__title" id="_i74"> 5-((5-Chloro-2-(3,3-difluoro-8-azabicyclo[3.2.1]octan-8-yl)pyrimidin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>28b</b>)</h3><div class="NLM_p last">Method as for <b>27b</b>, using 3,3-difluoro-8-azabicyclo[3.2.1]octane hydrochloride and heating in the microwave at 140 °C for 8 h. Obtained compound <b>28b</b> (12.5 mg, 0.025 mmol, 59%) as a cream solid. HRMS (ESI +ve): found 507.2087, expected 507.2081 for C<sub>24</sub>H<sub>30</sub>ClF<sub>2</sub>N<sub>6</sub>O<sub>2</sub><sup>+</sup> [M + H]<sup>+</sup>. δ<sub>H</sub> (600 MHz, DMF-<i>d</i><sub>7</sub>) 8.84 (s, 1H), 8.14 (s, 1H), 7.66 (d, <i>J</i> = 1.9 Hz, 1H), 7.47 (dd, <i>J</i> = 8.4, 2.0 Hz, 1H), 7.18 (d, <i>J</i> = 8.4 Hz, 1H), 4.71 (br s, 2H), 4.52 (s, 1H), 4.04 (m, 2H), 3.42 (s, 3H), 2.33–2.12 (m, 4H), 2.09–1.93 (m, 4H), 1.86 (m, 2H), 1.26 (s, 6H).</div></div><div id="sec4_2_52" class="NLM_sec NLM_sec_level_2"><div id="ac_i75" class="anchor-spacer"></div><h3 class="article-section__title" id="_i75"> 5-((2-Bromo-5-chloropyridin-4-yl)amino)-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>29</b>)</h3><div class="NLM_p last">A mixture of 5-amino-3-(3-hydroxy-3-methylbutyl)-1-methyl-1,3-dihydro-2<i>H</i>-benzo[<i>d</i>]imidazol-2-one (<b>16</b>) (20 mg, 0.08 mmol), 2,4-dibromo-5-chloropyridine (25 mg, 0.092 mmol), and DIPEA (16 mg, 0.12 mmol) in NMP (0.5 mL) was heated in the microwave to 180 °C for 1 h, then diluted with water, acidified to pH 5 by addition of 10% citric acid, then extracted with DCM (5 mL × 3). Combined organics were evaporated under reduced pressure and purified by reverse phase column chromatography (12 g C18 column, 30–80% MeOH in water, both modified with 0.1% formic acid) to give <b>29</b> (9 mg, 0.019 mmol, 24%). HRMS (ESI +ve): found 439.0512, expected 439.0531 for C<sub>18</sub>H<sub>21</sub>BrClN<sub>4</sub>O<sub>2</sub><sup>+</sup> [M + H]<sup>+</sup>. δ<sub>H</sub> (500 MHz, acetone-<i>d</i><sub>6</sub>) 8.08 (s, 1H), 7.92 (br, 0.6H, partly exchanged NH), 7.17 (d, <i>J</i> = 8.4 Hz, 1H), 7.15 (m, 1H), 7.08 (dt, <i>J</i> = 8.4, 1.7 Hz, 1H), 6.83 (d, <i>J</i> = 4.3 Hz, 1H), 4.04 (m, 2H), 3.42 (s, 3H), 1.86 (m, 2H), 1.26 (s, 6H).</div></div><div id="sec4_2_53" class="NLM_sec NLM_sec_level_2"><div id="ac_i76" class="anchor-spacer"></div><h3 class="article-section__title" id="_i76"> 5-Chloro-2-((3<i>S</i>,5<i>R</i>)-3,5-dimethylpiperidin-1-yl)-4-iodopyridine (<b>30a</b>)</h3><div class="NLM_p last">A solution of <i>cis</i>-3,5-dimethylpiperidine (0.23 g, 2.0 mmol), 5-chloro-2-fluoro-4-iodopyridine (0.53 g, 2.0 mmol), and DIPEA (0.53 mL, 3.1 mmol) in THF (8 mL) was heated in a sealed vial to 100 °C for 16 h. When cooled, water was added to the THF solution and extracted with EtOAc. The combined organic layers were washed with water twice and dried with sodium sulfate. Flash column chromatography (4% ethyl acetate in cyclohexane) gave <b>30a</b> (488 mg, 1.39 mmol, 68%) as a white solid. δ<sub>H</sub> (500 MHz, DMSO-<i>d</i><sub>6</sub>) 8.08 (s, 1H), 7.40 (s, 1H), 4.36–4.10 (m, 2H), 2.25 (dd, <i>J</i> = 12.9, 11.4 Hz, 2H), 1.75 (dtd, <i>J</i> = 10.6, 3.6, 1.9 Hz, 1H), 1.62–1.46 (m, 2H), 0.88 (d, <i>J</i> = 6.6 Hz, 6H), 0.84–0.70 (m, 1H).</div></div><div id="sec4_2_54" class="NLM_sec NLM_sec_level_2"><div id="ac_i77" class="anchor-spacer"></div><h3 class="article-section__title" id="_i77"> (2<i>S</i>,6<i>R</i>)-4-(5-Chloro-4-iodopyridin-2-yl)-2,6-dimethylmorpholine (<b>30b</b>)</h3><div class="NLM_p last">Prepared as for <b>30a</b>, using (2<i>S</i>,6<i>R</i>)-2,6-dimethylmorpholine, with heating for 3 days. Flash column chromatography (10% ethyl acetate in cyclohexane) gave <b>30b</b> (54 mg, 0.15 mmol, 39%) as a white waxy solid. δ<sub>H</sub> (500 MHz, DMSO-<i>d</i><sub>6</sub>) 8.13 (s, 1H), 7.43 (s, 1H), 4.22–4.06 (m, 2H), 3.66–3.50 (m, 2H), 2.39 (dd, <i>J</i> = 12.8, 10.5 Hz, 2H), 1.14 (d, <i>J</i> = 6.1 Hz, 6H).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i78"><a href="/doi/suppl/10.1021/acs.jmedchem.9b02076" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41048" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41048" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.9b02076?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b02076</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Supplementary experimental details on protein synthesis and purification, assay conditions, PK and PD methods; analytical data for compound <b>1</b>; tables listing statistics and replicate values and crystallographic data; figures showing binding modes, cell proliferation assay data, PD effects, and PK/PD relationship (<a href="/doi/suppl/10.1021/acs.jmedchem.9b02076/suppl_file/jm9b02076_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a href="/doi/suppl/10.1021/acs.jmedchem.9b02076/suppl_file/jm9b02076_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b02076/suppl_file/jm9b02076_si_001.pdf">jm9b02076_si_001.pdf (958.64 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b02076/suppl_file/jm9b02076_si_002.csv">jm9b02076_si_002.csv (5.3 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Atomic coordinates and structure factors for the crystal structures of BCL6 with compounds <b>1</b>, <b>4</b>, <b>5</b>, <b>6</b>, <b>11a</b>, <b>11f</b>, <b>17a</b>, <b>23d</b>, <b>25a</b>, and <b>25b</b> can be accessed using PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOM">6TOM</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOF">6TOF</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOG">6TOG</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOH">6TOH</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOO">6TOO</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOI">6TOI</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOJ">6TOJ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOK">6TOK</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOL">6TOL</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TON">6TON</a> respectively. Authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.9b02076" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03564" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03564" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Swen Hoelder</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit and , The Institute of Cancer Research, London SM2 5NG, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8636-1488" title="Orcid link">http://orcid.org/0000-0001-8636-1488</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#3d4e5552585159584f7d545e4f135c5e134856"><span class="__cf_email__" data-cfemail="e192898e848d858493a1888293cf8082cf948a">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Benjamin R. Bellenie</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit and , The Institute of Cancer Research, London SM2 5NG, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9987-3079" title="Orcid link">http://orcid.org/0000-0001-9987-3079</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kwai-Ming J. Cheung</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit and , The Institute of Cancer Research, London SM2 5NG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ana Varela</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit and , The Institute of Cancer Research, London SM2 5NG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Olivier A. Pierrat</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit and , The Institute of Cancer Research, London SM2 5NG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gavin W. Collie</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit, 
    Division of Structural Biology, 
     and , The Institute of Cancer Research, London SM2 5NG, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0406-922X" title="Orcid link">http://orcid.org/0000-0002-0406-922X</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gary M. Box</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit and , The Institute of Cancer Research, London SM2 5NG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael D. Bright</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit and , The Institute of Cancer Research, London SM2 5NG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sharon Gowan</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit and , The Institute of Cancer Research, London SM2 5NG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Angela Hayes</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit and , The Institute of Cancer Research, London SM2 5NG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthew J. Rodrigues</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit, 
    Division of Structural Biology, 
     and , The Institute of Cancer Research, London SM2 5NG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kartika N. Shetty</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit, 
    Division of Structural Biology, 
     and , The Institute of Cancer Research, London SM2 5NG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael Carter</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit and , The Institute of Cancer Research, London SM2 5NG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Owen A. Davis</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit and , The Institute of Cancer Research, London SM2 5NG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alan T. Henley</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit and , The Institute of Cancer Research, London SM2 5NG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paolo Innocenti</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit and , The Institute of Cancer Research, London SM2 5NG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Louise D. Johnson</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit and , The Institute of Cancer Research, London SM2 5NG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Manjuan Liu</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit and , The Institute of Cancer Research, London SM2 5NG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Selby de Klerk</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit and , The Institute of Cancer Research, London SM2 5NG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yann-Vaï Le Bihan</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit and , The Institute of Cancer Research, London SM2 5NG, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6850-9706" title="Orcid link">http://orcid.org/0000-0002-6850-9706</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthew G. Lloyd</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit and , The Institute of Cancer Research, London SM2 5NG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">P. Craig McAndrew</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit and , The Institute of Cancer Research, London SM2 5NG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Erald Shehu</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit and , The Institute of Cancer Research, London SM2 5NG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rachel Talbot</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit and , The Institute of Cancer Research, London SM2 5NG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hannah L. Woodward</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit and , The Institute of Cancer Research, London SM2 5NG, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8429-2546" title="Orcid link">http://orcid.org/0000-0001-8429-2546</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rosemary Burke</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit and , The Institute of Cancer Research, London SM2 5NG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Vladimir Kirkin</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit and , The Institute of Cancer Research, London SM2 5NG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rob L. M. van Montfort</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit, 
    Division of Structural Biology, 
     and , The Institute of Cancer Research, London SM2 5NG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Florence I. Raynaud</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit and , The Institute of Cancer Research, London SM2 5NG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Olivia W. Rossanese</span> - <span class="hlFld-Affiliation affiliation">Cancer
Research UK Cancer Therapeutics Unit and , The Institute of Cancer Research, London SM2 5NG, U.K.</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li><span class="author-information-subsection-header">Funding</span><p>This work was supported by Cancer Research UK (Grant C309/A11566), CRT Pioneer Fund and Sixth Element Capital, who we thank for their generous funding. We also acknowledge NHS funding to the NIHR Biomedical Research Centre.</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i81">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_36816" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_36816" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Dr. Nora Cronin from the Division of Structural Biology at the Institute of Cancer Research and the staff of Diamond Light Source and the European Synchrotron Radiation Facility for their support during X-ray crystallography data collection. We also thank Meirion Richards and Dr. Amin Mirza of the Structural Chemistry team for their expertise and assistance with LC, NMR, and mass spectrometry, and Dr. Matthew Cheeseman for input concerning statistical analysis of data.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">APCI</td><td class="NLM_def"><p class="first last">atmospheric pressure chemical ionization</p></td></tr><tr><td class="NLM_term">BCL6</td><td class="NLM_def"><p class="first last">B-cell lymphoma 6 protein</p></td></tr><tr><td class="NLM_term">BCOR</td><td class="NLM_def"><p class="first last">BCL-6 co-repressor protein</p></td></tr><tr><td class="NLM_term">BRET</td><td class="NLM_def"><p class="first last">bioluminescence resonance energy transfer</p></td></tr><tr><td class="NLM_term">BTB</td><td class="NLM_def"><p class="first last">broad-complex, tramtrack, and bric a brac (domain), also known as POZ (poxvirus and zinc finger) domain</p></td></tr><tr><td class="NLM_term">CL</td><td class="NLM_def"><p class="first last">clearance</p></td></tr><tr><td class="NLM_term">CL<sub>int</sub></td><td class="NLM_def"><p class="first last">intrinsic clearance</p></td></tr><tr><td class="NLM_term"><i>c</i><sub>max</sub></td><td class="NLM_def"><p class="first last">maximum (peak) concentration achieved after a single dose</p></td></tr><tr><td class="NLM_term">cPr</td><td class="NLM_def"><p class="first last">cyclopropyl</p></td></tr><tr><td class="NLM_term">DC<sub>50</sub></td><td class="NLM_def"><p class="first last">concentration of compound at which 50% of the maximally observed protein degradation (<i>D</i><sub>max</sub>) is achieved</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">DLBCL</td><td class="NLM_def"><p class="first last">diffuse large B-cell lymphoma</p></td></tr><tr><td class="NLM_term"><i>D</i><sub>max</sub></td><td class="NLM_def"><p class="first last">maximal percentage degradation of protein achieved</p></td></tr><tr><td class="NLM_term">EtOAc</td><td class="NLM_def"><p class="first last">ethyl acetate</p></td></tr><tr><td class="NLM_term">GAPDH</td><td class="NLM_def"><p class="first last">glyceraldehyde 3-phosphate dehydrogenase</p></td></tr><tr><td class="NLM_term">HAC</td><td class="NLM_def"><p class="first last">heavy atom count</p></td></tr><tr><td class="NLM_term">IMDM</td><td class="NLM_def"><p class="first last">Iscove’s modified Dulbecco’s medium</p></td></tr><tr><td class="NLM_term">IMiD</td><td class="NLM_def"><p class="first last">immunomodulatory imide drug, such as thalidomide and its analogues</p></td></tr><tr><td class="NLM_term">MeCN</td><td class="NLM_def"><p class="first last">acetonitrile</p></td></tr><tr><td class="NLM_term">MLM</td><td class="NLM_def"><p class="first last">mouse liver microsome</p></td></tr><tr><td class="NLM_term">MSD</td><td class="NLM_def"><p class="first last">meso scale discovery, an assay method similar to ELISA, using electrochemiluminescence as a detection technique</p></td></tr><tr><td class="NLM_term">NanoBRET</td><td class="NLM_def"><p class="first last">a BRET-based assay format commercialized by Promega that uses NanoLuc luciferase to generate the donor signal and a HaloTag ligand as the fluorescent energy acceptor</p></td></tr><tr><td class="NLM_term">NCoR</td><td class="NLM_def"><p class="first last">nuclear receptor co-repressor 1</p></td></tr><tr><td class="NLM_term">PFP</td><td class="NLM_def"><p class="first last">pentafluorophenyl</p></td></tr><tr><td class="NLM_term">PPI</td><td class="NLM_def"><p class="first last">protein–protein interaction</p></td></tr><tr><td class="NLM_term">PROTAC</td><td class="NLM_def"><p class="first last">proteolysis targeting chimera</p></td></tr><tr><td class="NLM_term">QToF</td><td class="NLM_def"><p class="first last">quadrupole time-of-flight</p></td></tr><tr><td class="NLM_term">SCID</td><td class="NLM_def"><p class="first last">severe combined immunodeficient (mouse model)</p></td></tr><tr><td class="NLM_term">SCX-2</td><td class="NLM_def"><p class="first last">SCX (strong cation exchange)-2, which is a propylsulfonic acid bonded sorbent</p></td></tr><tr><td class="NLM_term">Si-DMT</td><td class="NLM_def"><p class="first last">silica bound equivalent of 2,4,6-trimercaptotriazine</p></td></tr><tr><td class="NLM_term">SMRT</td><td class="NLM_def"><p class="first last">silencing mediator for retinoid or thyroid hormone receptor (also known as nuclear receptor co-repressor 2 or NCoR2)</p></td></tr><tr><td class="NLM_term">S<sub>N</sub>Ar</td><td class="NLM_def"><p class="first last">nucleophilic aromatic substitution</p></td></tr><tr><td class="NLM_term">sol.</td><td class="NLM_def"><p class="first last">solubility</p></td></tr><tr><td class="NLM_term">STR</td><td class="NLM_def"><p class="first last">short tandem repeat (analysis)</p></td></tr><tr><td class="NLM_term"><i>t</i><sub>max</sub></td><td class="NLM_def"><p class="first last">time after dosing at which the maximum (peak) concentration <i>c</i><sub>max</sub> is observed</p></td></tr><tr><td class="NLM_term">TR-FRET</td><td class="NLM_def"><p class="first last">time-resolved fluorescence energy transfer</p></td></tr><tr><td class="NLM_term">Xantphos</td><td class="NLM_def"><p class="first last">4,5-bis(diphenylphosphino)-9,9-dimethylxanthene</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i83">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_11810" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_11810" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 26 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hatzi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnick, A.</span></span> <span> </span><span class="NLM_article-title">Breaking bad in the germinal center: how deregulation of BCL6 contributes to lymphomagenesis</span>. <i>Trends Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>20</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">343</span>– <span class="NLM_lpage">352</span>, <span class="refDoi"> DOI: 10.1016/j.molmed.2014.03.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10.1016%2Fj.molmed.2014.03.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=24698494" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltl2htrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=343-352&issue=6&author=K.+Hatziauthor=A.+Melnick&title=Breaking+bad+in+the+germinal+center%3A+how+deregulation+of+BCL6+contributes+to+lymphomagenesis&doi=10.1016%2Fj.molmed.2014.03.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Breaking bad in the germinal center: how deregulation of BCL6 contributes to lymphomagenesis</span></div><div class="casAuthors">Hatzi, Katerina; Melnick, Ari</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Molecular Medicine</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">343-352</span>CODEN:
                <span class="NLM_cas:coden">TMMRCY</span>;
        ISSN:<span class="NLM_cas:issn">1471-4914</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The B cell lymphoma 6 (BCL6) transcriptional repressor is a master regulator of the germinal center (GC) B cell program, required for their unique proliferative and stress tolerant phenotype.  Most B cell lymphomas arise from GC B cells and are dependent on the continued or deregulated expression of BCL6 to maintain their survival.  The actions of BCL6 in B cells involve formation of distinct chromatin modifying complexes that silence specific promoter and enhancer networks, resp.  The same biochem. mechanisms are maintained in malignant lymphoma cells.  Targeted inhibition of these BCL6 functions has emerged as the basis for rational design of lymphoma therapies and combinatorial regimens.  In this review, we summarize recent advances on BCL6 mechanisms of action and the deregulation of its target gene networks in lymphoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrroimnIsIZJ7Vg90H21EOLACvtfcHk0lgl7_hMRZ7dgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltl2htrc%253D&md5=bbc52827778c0995ea400f814c66cfe4</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.molmed.2014.03.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molmed.2014.03.001%26sid%3Dliteratum%253Aachs%26aulast%3DHatzi%26aufirst%3DK.%26aulast%3DMelnick%26aufirst%3DA.%26atitle%3DBreaking%2520bad%2520in%2520the%2520germinal%2520center%253A%2520how%2520deregulation%2520of%2520BCL6%2520contributes%2520to%2520lymphomagenesis%26jtitle%3DTrends%2520Mol.%2520Med.%26date%3D2014%26volume%3D20%26issue%3D6%26spage%3D343%26epage%3D352%26doi%3D10.1016%2Fj.molmed.2014.03.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Basso, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalla-Favera, R.</span></span> <span> </span><span class="NLM_article-title">Roles of BCL6 in normal and transformed germinal center B cells</span>. <i>Immunological Reviews</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>247</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">172</span>– <span class="NLM_lpage">183</span>, <span class="refDoi"> DOI: 10.1111/j.1600-065X.2012.01112.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10.1111%2Fj.1600-065X.2012.01112.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=22500840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVaks7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=247&publication_year=2012&pages=172-183&issue=1&author=K.+Bassoauthor=R.+Dalla-Favera&title=Roles+of+BCL6+in+normal+and+transformed+germinal+center+B+cells&doi=10.1111%2Fj.1600-065X.2012.01112.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Roles of BCL6 in normal and transformed germinal center B cells</span></div><div class="casAuthors">Basso, Katia; Dalla-Favera, Riccardo</div><div class="citationInfo"><span class="NLM_cas:title">Immunological Reviews</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">247</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">172-183, 12 pp.</span>CODEN:
                <span class="NLM_cas:coden">IMRED2</span>;
        ISSN:<span class="NLM_cas:issn">1600-065X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  BCL6 is a transcriptional repressor required in mature B cells during the germinal center (GC) reaction.  Multiple mechanisms act coordinately to timely modulate BCL6 expression at transcriptional and post-transcriptional levels.  BCL6 prevents premature activation and differentiation of GC B cells and provides an environment tolerant of the DNA breaks assocd. with Ig gene remodeling mechanisms involved in the prodn. of high-affinity antibodies of different isotypes.  The crit. functions exerted by BCL6 during normal B-cell development can be hijacked by the malignant transformation process.  Indeed, BCL6 is targeted by genetic aberrations and acts as an oncogene in GC-derived lymphomas.  The aberrations affecting BCL6 interfere with the multiple levels of regulation that grant a fine tuning of BCL6 expression and activity in physiol. conditions.  This review summarizes the current knowledge on BCL6 function and its role in lymphomagenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXfjT3UG13QrVg90H21EOLACvtfcHk0lgl7_hMRZ7dgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVaks7jO&md5=6f923c689f545b55faf1116e87f6afec</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1111%2Fj.1600-065X.2012.01112.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1600-065X.2012.01112.x%26sid%3Dliteratum%253Aachs%26aulast%3DBasso%26aufirst%3DK.%26aulast%3DDalla-Favera%26aufirst%3DR.%26atitle%3DRoles%2520of%2520BCL6%2520in%2520normal%2520and%2520transformed%2520germinal%2520center%2520B%2520cells%26jtitle%3DImmunological%2520Reviews%26date%3D2012%26volume%3D247%26issue%3D1%26spage%3D172%26epage%3D183%26doi%3D10.1111%2Fj.1600-065X.2012.01112.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jardin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruminy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bastard, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tilly, H.</span></span> <span> </span><span class="NLM_article-title">The BCL6 proto-oncogene: a leading role during germinal center development and lymphomagenesis</span>. <i>Pathol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>55</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">73</span>– <span class="NLM_lpage">83</span>, <span class="refDoi"> DOI: 10.1016/j.patbio.2006.04.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10.1016%2Fj.patbio.2006.04.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=16815642" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhs1Sntbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2007&pages=73-83&issue=1&author=F.+Jardinauthor=P.+Ruminyauthor=C.+Bastardauthor=H.+Tilly&title=The+BCL6+proto-oncogene%3A+a+leading+role+during+germinal+center+development+and+lymphomagenesis&doi=10.1016%2Fj.patbio.2006.04.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The BCL6 proto-oncogene: a leading role during germinal center development and lymphomagenesis</span></div><div class="casAuthors">Jardin, Fabrice; Ruminy, Philippe; Bastard, Christian; Tilly, Herve</div><div class="citationInfo"><span class="NLM_cas:title">Pathologie Biologie</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">73-83</span>CODEN:
                <span class="NLM_cas:coden">PTBIAN</span>;
        ISSN:<span class="NLM_cas:issn">0369-8114</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  The BCL6 proto-oncogene encodes a nuclear transcriptional repressor, with pivotal roles in germinal center (GC) formation and regulation of lymphocyte function, differentiation, and survival.  BCL6 suppresses p53 in GCB-cells and its constitutive expression can protect B-cell lines from apoptosis induced by DNA damage.  BCL6-mediated expression may allow GCB-cells to sustain the low levels of physiol. DNA breaks related to somatic mutation (SM) and Ig class switch recombination which physiol. occur in GCB-cells.  Three types of genetic events occur in the BCL6 locus and involve invariably the 5' non-coding region and include translocations, deletions and SM actively targeted to the 5' untranslated region.  These acquired mutations occur independently of translocations but may be involved in the deregulation of the gene and/or translocation mechanisms.  The favorable prognostic value of high levels of BCL6 gene expression in NHL seems well-established.  By contrast, the relevance of SM or translocation of the gene remains unclear.  However, it is likely that non-Hodgkin's lymphomas (NHL) harboring the most frequent translocation involving BCL6, i.e. t(3;14), are characterized by a common cell of origin and similar oncogenic mechanisms.  Several expts. and mouse models mimicking BCL6 translocation occurring in human lymphoma have demonstrated the oncogenic role of BCL6 and constitute a rational to consider BCL6 as a new therapeutic target in NHL.  BCL6 blockade can be achieved by different strategies which include siRNA, interference by specific peptides or regulation of BCL6 acetylation by pharmacol. agents such as SAHA or niacinamide and would be applicable to most type of B-cell NHL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqDg4nsDWOqLVg90H21EOLACvtfcHk0ljO4V_Yd5xZxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhs1Sntbc%253D&md5=eaa157769f2ba4d2f91a729d02e12f66</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.patbio.2006.04.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.patbio.2006.04.001%26sid%3Dliteratum%253Aachs%26aulast%3DJardin%26aufirst%3DF.%26aulast%3DRuminy%26aufirst%3DP.%26aulast%3DBastard%26aufirst%3DC.%26aulast%3DTilly%26aufirst%3DH.%26atitle%3DThe%2520BCL6%2520proto-oncogene%253A%2520a%2520leading%2520role%2520during%2520germinal%2520center%2520development%2520and%2520lymphomagenesis%26jtitle%3DPathol.%2520Biol.%26date%3D2007%26volume%3D55%26issue%3D1%26spage%3D73%26epage%3D83%26doi%3D10.1016%2Fj.patbio.2006.04.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hanahan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinberg, R. A.</span></span> <span> </span><span class="NLM_article-title">The Hallmarks of Cancer</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>100</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">57</span>– <span class="NLM_lpage">70</span>, <span class="refDoi"> DOI: 10.1016/S0092-8674(00)81683-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10.1016%2FS0092-8674%2800%2981683-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10647931" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BD3cXks1CktA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2000&pages=57-70&issue=1&author=D.+Hanahanauthor=R.+A.+Weinberg&title=The+Hallmarks+of+Cancer&doi=10.1016%2FS0092-8674%2800%2981683-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The hallmarks of cancer</span></div><div class="casAuthors">Hanahan, Douglas; Weinberg, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, Massachusetts)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">57-70</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review is given with many refs. on principles governing the transformation of normal human cells into malignant cancers.  The authors suggest that research over the past decades has revealed a small no. of mol., biochem., and cellular traits (acquired capabilities) shared by most and perhaps all types of cancer.  Topics included are the acquired capabilities self-sufficiency in growth signals, insensitivity to antigrowth signals, evading apoptosis, limitless replicative potential, sustained angiogenesis, and tissue invasion and metastasis followed by genome instability as an enabling characteristic and alternative pathways to cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooXmGqKjM1C7Vg90H21EOLACvtfcHk0ljO4V_Yd5xZxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXks1CktA%253D%253D&md5=1d23c4400dad34c7f32d4a0c41890349</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2800%2981683-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252800%252981683-9%26sid%3Dliteratum%253Aachs%26aulast%3DHanahan%26aufirst%3DD.%26aulast%3DWeinberg%26aufirst%3DR.%2BA.%26atitle%3DThe%2520Hallmarks%2520of%2520Cancer%26jtitle%3DCell%26date%3D2000%26volume%3D100%26issue%3D1%26spage%3D57%26epage%3D70%26doi%3D10.1016%2FS0092-8674%2800%2981683-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hanahan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinberg, R. A.</span></span> <span> </span><span class="NLM_article-title">Hallmarks of Cancer: The Next Generation</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>144</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">646</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2011.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10.1016%2Fj.cell.2011.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=21376230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjsFeqtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=144&publication_year=2011&pages=646-674&issue=5&author=D.+Hanahanauthor=R.+A.+Weinberg&title=Hallmarks+of+Cancer%3A+The+Next+Generation&doi=10.1016%2Fj.cell.2011.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Hallmarks of cancer: the next generation</span></div><div class="casAuthors">Hanahan, Douglas; Weinberg, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">144</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">646-674</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The hallmarks of cancer comprise six biol. capabilities acquired during the multistep development of human tumors.  The hallmarks constitute an organizing principle for rationalizing the complexities of neoplastic disease.  They include sustaining proliferative signaling, evading growth suppressors, resisting cell death, enabling replicative immortality, inducing angiogenesis, and activating invasion and metastasis.  Underlying these hallmarks are genome instability, which generates the genetic diversity that expedites their acquisition, and inflammation, which fosters multiple hallmark functions.  Conceptual progress in the last decade has added two emerging hallmarks of potential generality to this list-reprogramming of energy metab. and evading immune destruction.  In addn. to cancer cells, tumors exhibit another dimension of complexity: they contain a repertoire of recruited, ostensibly normal cells that contribute to the acquisition of hallmark traits by creating the "tumor microenvironment.".  Recognition of the widespread applicability of these concepts will increasingly affect the development of new means to treat human cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTcpqXALcb6LVg90H21EOLACvtfcHk0ljO4V_Yd5xZxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjsFeqtrk%253D&md5=b36f160c417a712e83cb83c577f0018e</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DHanahan%26aufirst%3DD.%26aulast%3DWeinberg%26aufirst%3DR.%2BA.%26atitle%3DHallmarks%2520of%2520Cancer%253A%2520The%2520Next%2520Generation%26jtitle%3DCell%26date%3D2011%26volume%3D144%26issue%3D5%26spage%3D646%26epage%3D674%26doi%3D10.1016%2Fj.cell.2011.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Polo, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dell’Oso, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranuncolo, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerchietti, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Da Silva, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prive, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Licht, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnick, A.</span></span> <span> </span><span class="NLM_article-title">Specific peptide interference reveals BCL6 transcriptional and oncogenic mechanisms in B-cell lymphoma cells</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">1329</span>– <span class="NLM_lpage">1335</span>, <span class="refDoi"> DOI: 10.1038/nm1134</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10.1038%2Fnm1134" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=15531890" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVCisbfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2004&pages=1329-1335&issue=12&author=J.+M.+Poloauthor=T.+Dell%E2%80%99Osoauthor=S.+M.+Ranuncoloauthor=L.+Cerchiettiauthor=D.+Beckauthor=G.+F.+Da+Silvaauthor=G.+G.+Priveauthor=J.+D.+Lichtauthor=A.+Melnick&title=Specific+peptide+interference+reveals+BCL6+transcriptional+and+oncogenic+mechanisms+in+B-cell+lymphoma+cells&doi=10.1038%2Fnm1134"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Specific peptide interference reveals BCL6 transcriptional and oncogenic mechanisms in B-cell lymphoma cells</span></div><div class="casAuthors">Polo, Jose M.; Dell'Oso, Tania; Ranuncolo, Stella Maris; Cerchietti, Leandro; Beck, David; Da Silva, Gustavo F.; Prive, Gilbert G.; Licht, Jonathan D.; Melnick, Ari</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1329-1335</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The BTB/POZ transcriptional repressor and candidate oncogene BCL6 is frequently misregulated in B-cell lymphomas.  The interface through which the BCL6 BTB domain mediates recruitment of the SMRT, NCoR and BCoR corepressors was recently identified.  To det. the contribution of this interface to BCL6 transcriptional and biol. properties, the authors generated a peptide that specifically binds BCL6 and blocks corepressor recruitment in vivo.  This inhibitor disrupts BCL6-mediated repression and establishment of silenced chromatin, reactivates natural BCL6 target genes, and abrogates BCL6 biol. function in B cells.  In BCL6-pos. lymphoma cells, peptide blockade caused apoptosis and cell cycle arrest.  BTB domain peptide inhibitors may constitute a novel therapeutic agent for B-cell lymphomas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAPaMuV1BGLrVg90H21EOLACvtfcHk0liLZaC5wuWo3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVCisbfJ&md5=21f9d4b802d65897d666af1ac77a083c</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnm1134&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm1134%26sid%3Dliteratum%253Aachs%26aulast%3DPolo%26aufirst%3DJ.%2BM.%26aulast%3DDell%25E2%2580%2599Oso%26aufirst%3DT.%26aulast%3DRanuncolo%26aufirst%3DS.%2BM.%26aulast%3DCerchietti%26aufirst%3DL.%26aulast%3DBeck%26aufirst%3DD.%26aulast%3DDa%2BSilva%26aufirst%3DG.%2BF.%26aulast%3DPrive%26aufirst%3DG.%2BG.%26aulast%3DLicht%26aufirst%3DJ.%2BD.%26aulast%3DMelnick%26aufirst%3DA.%26atitle%3DSpecific%2520peptide%2520interference%2520reveals%2520BCL6%2520transcriptional%2520and%2520oncogenic%2520mechanisms%2520in%2520B-cell%2520lymphoma%2520cells%26jtitle%3DNat.%2520Med.%26date%3D2004%26volume%3D10%26issue%3D12%26spage%3D1329%26epage%3D1335%26doi%3D10.1038%2Fnm1134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghetu, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corcoran, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerchietti, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bardwell, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Privé, G. G.</span></span> <span> </span><span class="NLM_article-title">Structure of a BCOR corepressor peptide in complex with the BCL6 BTB domain dimer</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>29</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">384</span>– <span class="NLM_lpage">391</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2007.12.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10.1016%2Fj.molcel.2007.12.026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=18280243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BD1cXisFSns7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2008&pages=384-391&issue=3&author=A.+F.+Ghetuauthor=C.+M.+Corcoranauthor=L.+Cerchiettiauthor=V.+J.+Bardwellauthor=A.+Melnickauthor=G.+G.+Priv%C3%A9&title=Structure+of+a+BCOR+corepressor+peptide+in+complex+with+the+BCL6+BTB+domain+dimer&doi=10.1016%2Fj.molcel.2007.12.026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of a BCOR corepressor peptide in complex with the BCL6 BTB domain dimer</span></div><div class="casAuthors">Ghetu, Alexandru F.; Corcoran, Connie M.; Cerchietti, Leandro; Bardwell, Vivian J.; Melnick, Ari; Prive, Gilbert G.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">384-391</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The transcriptional corepressors BCOR, SMRT, and NCoR are known to bind competitively to the BCL6 BTB domain despite the fact that BCOR has no detectable sequence similarity to the other two corepressors.  We have identified a 17 residue motif from BCOR that binds directly to the BCL6 BTB domain and detd. the crystal structure of the complex to a resoln. of 2.6 Å.  Remarkably, the BCOR BCL6 binding domain (BCORBBD) peptide binds in the same BCL6 binding site as the SMRTBBD peptide despite the lack of any significant sequence similarity between the two peptides.  Mutations of crit. BCORBBD residues cause the disruption of the BCL6 corepression activities of BCOR, and a BCORBBD peptide blocks BCL6-mediated transcriptional repression and kills lymphoma cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-FAcEvWcMKbVg90H21EOLACvtfcHk0liLZaC5wuWo3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXisFSns7c%253D&md5=f064d9c9e3c157a960eb0bda26fcdece</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2007.12.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2007.12.026%26sid%3Dliteratum%253Aachs%26aulast%3DGhetu%26aufirst%3DA.%2BF.%26aulast%3DCorcoran%26aufirst%3DC.%2BM.%26aulast%3DCerchietti%26aufirst%3DL.%26aulast%3DBardwell%26aufirst%3DV.%2BJ.%26aulast%3DMelnick%26aufirst%3DA.%26aulast%3DPriv%25C3%25A9%26aufirst%3DG.%2BG.%26atitle%3DStructure%2520of%2520a%2520BCOR%2520corepressor%2520peptide%2520in%2520complex%2520with%2520the%2520BCL6%2520BTB%2520domain%2520dimer%26jtitle%3DMol.%2520Cell%26date%3D2008%26volume%3D29%26issue%3D3%26spage%3D384%26epage%3D391%26doi%3D10.1016%2Fj.molcel.2007.12.026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cerchietti, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghetu, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Da Silva, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunting, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polo, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farès, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coop, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKerell, A. D.,  Jr</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Privé, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnick, A.</span></span> <span> </span><span class="NLM_article-title">A small-molecule inhibitor of BCL6 kills DLBCL cells in vitro and in vivo</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>17</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">400</span>– <span class="NLM_lpage">411</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2009.12.050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10.1016%2Fj.ccr.2009.12.050" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=20385364" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmsF2msb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=400-411&issue=4&author=L.+C.+Cerchiettiauthor=A.+F.+Ghetuauthor=X.+Zhuauthor=G.+F.+Da+Silvaauthor=S.+Zhongauthor=M.+Matthewsauthor=K.+L.+Buntingauthor=J.+M.+Poloauthor=C.+Far%C3%A8sauthor=C.+H.+Arrowsmithauthor=S.+N.+Yangauthor=M.+Garciaauthor=A.+Coopauthor=A.+D.+MacKerellauthor=G.+G.+Priv%C3%A9author=A.+Melnick&title=A+small-molecule+inhibitor+of+BCL6+kills+DLBCL+cells+in+vitro+and+in+vivo&doi=10.1016%2Fj.ccr.2009.12.050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">A small-molecule inhibitor of BCL6 kills DLBCL cells in vitro and in vivo</span></div><div class="casAuthors">Cerchietti, Leandro C.; Ghetu, Alexandru F.; Zhu, Xiao; Da Silva, Gustavo F.; Zhong, Shijun; Matthews, Marilyn; Bunting, Karen L.; Polo, Jose M.; Fares, Christophe; Arrowsmith, Cheryl H.; Yang, Shao Ning; Garcia, Monica; Coop, Andrew; MacKerell, Alexander D., Jr.; Prive, Gilbert G.; Melnick, Ari</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">400-411</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The BCL6 transcriptional repressor is the most frequently involved oncogene in diffuse large B cell lymphoma (DLBCL).  We combined computer-aided drug design with functional assays to identify low-mol.-wt. compds. that bind to the corepressor binding groove of the BCL6 BTB domain.  One such compd. disrupted BCL6/corepressor complexes in vitro and in vivo, and was obsd. by X-ray crystallog. and NMR to bind the crit. site within the BTB groove.  This compd. could induce expression of BCL6 target genes and kill BCL6-pos. DLBCL cell lines.  In xenotransplantation expts., the compd. was nontoxic and potently suppressed DLBCL tumors in vivo.  The compd. also killed primary DLBCLs from human patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdcScjNPjhiLVg90H21EOLACvtfcHk0liLZaC5wuWo3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmsF2msb8%253D&md5=64f30a813c26394b69da6de95991f8dd</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2009.12.050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2009.12.050%26sid%3Dliteratum%253Aachs%26aulast%3DCerchietti%26aufirst%3DL.%2BC.%26aulast%3DGhetu%26aufirst%3DA.%2BF.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DDa%2BSilva%26aufirst%3DG.%2BF.%26aulast%3DZhong%26aufirst%3DS.%26aulast%3DMatthews%26aufirst%3DM.%26aulast%3DBunting%26aufirst%3DK.%2BL.%26aulast%3DPolo%26aufirst%3DJ.%2BM.%26aulast%3DFar%25C3%25A8s%26aufirst%3DC.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DYang%26aufirst%3DS.%2BN.%26aulast%3DGarcia%26aufirst%3DM.%26aulast%3DCoop%26aufirst%3DA.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26aulast%3DPriv%25C3%25A9%26aufirst%3DG.%2BG.%26aulast%3DMelnick%26aufirst%3DA.%26atitle%3DA%2520small-molecule%2520inhibitor%2520of%2520BCL6%2520kills%2520DLBCL%2520cells%2520in%2520vitro%2520and%2520in%2520vivo%26jtitle%3DCancer%2520Cell%26date%3D2010%26volume%3D17%26issue%3D4%26spage%3D400%26epage%3D411%26doi%3D10.1016%2Fj.ccr.2009.12.050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cerchietti, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaknovich, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatzi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polo, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chadburn, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowdy, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnick, A.</span></span> <span> </span><span class="NLM_article-title">A peptomimetic inhibitor of BCL6 with potent antilymphoma effects in vitro and in vivo</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>113</i></span> (<span class="NLM_issue">15</span>),  <span class="NLM_fpage">3397</span>– <span class="NLM_lpage">3405</span>, <span class="refDoi"> DOI: 10.1182/blood-2008-07-168773</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10.1182%2Fblood-2008-07-168773" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=18927431" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BD1MXkslCkurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2009&pages=3397-3405&issue=15&author=L.+C.+Cerchiettiauthor=S.+N.+Yangauthor=R.+Shaknovichauthor=K.+Hatziauthor=J.+M.+Poloauthor=A.+Chadburnauthor=S.+F.+Dowdyauthor=A.+Melnick&title=A+peptomimetic+inhibitor+of+BCL6+with+potent+antilymphoma+effects+in+vitro+and+in+vivo&doi=10.1182%2Fblood-2008-07-168773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">A peptomimetic inhibitor of BCL6 with potent antilymphoma effects in vitro and in vivo</span></div><div class="casAuthors">Cerchietti, Leandro C.; Yang, Shao Ning; Shaknovich, Rita; Hatzi, Katerina; Polo, Jose M.; Chadburn, Amy; Dowdy, Steven F.; Melnick, Ari</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3397-3405</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">The BCL6 transcriptional repressor is the most commonly involved oncogene in diffuse large B-cell lymphomas (DLBCLs).  BCL6 lymphomagenic activity is dependent on its ability to recruit corepressor proteins to a unique binding site on its N-terminal BTB domain.  A recombinant peptide fragment of the SMRT (silencing mediator for retinoid and thyroid hormone receptor) corepressor that blocks this site can inhibit BCL6 biol. functions.  Shortening and conversion of this peptide to D-amino acid and retro configuration as well as the addn. of a fusogenic motif yielded a far more potent and stable BCL6 inhibitor that still retained the specificity of the original SMRT fragment.  Like the L-peptide, retroinverso BCL6 peptide inhibitor (RI-BPI) selectively killed BCR rather than OxPhos-type DLBCL cells.  The RI-BPI could recapitulate the failure to form germinal centers seen in BCL6 null mice yet was nontoxic and non-immunogenic even when administered for up to 52 wk.  RI-BPI showed superior duration of tissue penetration and could accordingly powerfully suppress the growth of human DLBCLs xenografts in a dose-dependent manner.  Finally, RI-BPI could kill primary human DLBCL cells but had no effect on normal lymphoid tissue or other tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxsckch1D41rVg90H21EOLACvtfcHk0ljmnCz6beFXRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXkslCkurw%253D&md5=8a12df08b153ba0712b5d06f9bc550c2</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1182%2Fblood-2008-07-168773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2008-07-168773%26sid%3Dliteratum%253Aachs%26aulast%3DCerchietti%26aufirst%3DL.%2BC.%26aulast%3DYang%26aufirst%3DS.%2BN.%26aulast%3DShaknovich%26aufirst%3DR.%26aulast%3DHatzi%26aufirst%3DK.%26aulast%3DPolo%26aufirst%3DJ.%2BM.%26aulast%3DChadburn%26aufirst%3DA.%26aulast%3DDowdy%26aufirst%3DS.%2BF.%26aulast%3DMelnick%26aufirst%3DA.%26atitle%3DA%2520peptomimetic%2520inhibitor%2520of%2520BCL6%2520with%2520potent%2520antilymphoma%2520effects%2520in%2520vitro%2520and%2520in%2520vivo%26jtitle%3DBlood%26date%3D2009%26volume%3D113%26issue%3D15%26spage%3D3397%26epage%3D3405%26doi%3D10.1182%2Fblood-2008-07-168773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cardenas, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beguelin, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teater, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldstein, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oswald, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatzi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.-N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaknovich, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanommeslaeghe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abbott, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tam, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonard, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elemento, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerchietti, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKerell, A. D.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnick, A. M.</span></span> <span> </span><span class="NLM_article-title">Rationally designed BCL6 inhibitors target activated B cell diffuse large B cell lymphoma</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>126</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">3351</span>– <span class="NLM_lpage">3362</span>, <span class="refDoi"> DOI: 10.1172/JCI85795</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10.1172%2FJCI85795" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=27482887" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A280%3ADC%252BC2s3mvFagsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2016&pages=3351-3362&issue=9&author=M.+G.+Cardenasauthor=W.+Yuauthor=W.+Beguelinauthor=M.+R.+Teaterauthor=H.+Gengauthor=R.+L.+Goldsteinauthor=E.+Oswaldauthor=K.+Hatziauthor=S.-N.+Yangauthor=J.+Cohenauthor=R.+Shaknovichauthor=K.+Vanommeslaegheauthor=H.+Chengauthor=D.+Liangauthor=H.+J.+Choauthor=J.+Abbottauthor=W.+Tamauthor=W.+Duauthor=J.+P.+Leonardauthor=O.+Elementoauthor=L.+Cerchiettiauthor=T.+Cierpickiauthor=F.+Xueauthor=A.+D.+MacKerellauthor=A.+M.+Melnick&title=Rationally+designed+BCL6+inhibitors+target+activated+B+cell+diffuse+large+B+cell+lymphoma&doi=10.1172%2FJCI85795"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Rationally designed BCL6 inhibitors target activated B cell diffuse large B cell lymphoma</span></div><div class="casAuthors">Cardenas Mariano G; Yu Wenbo; Beguelin Wendy; Teater Matthew R; Geng Huimin; Goldstein Rebecca L; Oswald Erin; Hatzi Katerina; Yang Shao-Ning; Cohen Joanna; Shaknovich Rita; Vanommeslaeghe Kenno; Cheng Huimin; Liang Dongdong; Cho Hyo Je; Abbott Joshua; Tam Wayne; Du Wei; Leonard John P; Elemento Olivier; Cerchietti Leandro; Cierpicki Tomasz; Xue Fengtian; MacKerell Alexander D Jr; Melnick Ari M</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of clinical investigation</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3351-62</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Diffuse large B cell lymphomas (DLBCLs) arise from proliferating B cells transiting different stages of the germinal center reaction.  In activated B cell DLBCLs (ABC-DLBCLs), a class of DLBCLs that respond poorly to current therapies, chromosomal translocations and amplification lead to constitutive expression of the B cell lymphoma 6 (BCL6) oncogene.  The role of BCL6 in maintaining these lymphomas has not been investigated.  Here, we designed small-molecule inhibitors that display higher affinity for BCL6 than its endogenous corepressor ligands to evaluate their therapeutic efficacy for targeting ABC-DLBCL.  We used an in silico drug design functional-group mapping approach called SILCS to create a specific BCL6 inhibitor called FX1 that has 10-fold greater potency than endogenous corepressors and binds an essential region of the BCL6 lateral groove.  FX1 disrupted formation of the BCL6 repression complex, reactivated BCL6 target genes, and mimicked the phenotype of mice engineered to express BCL6 with corepressor binding site mutations.  Low doses of FX1 induced regression of established tumors in mice bearing DLBCL xenografts.  Furthermore, FX1 suppressed ABC-DLBCL cells in vitro and in vivo, as well as primary human ABC-DLBCL specimens ex vivo.  These findings indicate that ABC-DLBCL is a BCL6-dependent disease that can be targeted by rationally designed inhibitors that exceed the binding affinity of natural BCL6 ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRAVUzYiXt5ZL5XHngOXiV_fW6udTcc2ebMXWXf2rwmlbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s3mvFagsw%253D%253D&md5=90a313fd7adce1922773212f0960bad7</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1172%2FJCI85795&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI85795%26sid%3Dliteratum%253Aachs%26aulast%3DCardenas%26aufirst%3DM.%2BG.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DBeguelin%26aufirst%3DW.%26aulast%3DTeater%26aufirst%3DM.%2BR.%26aulast%3DGeng%26aufirst%3DH.%26aulast%3DGoldstein%26aufirst%3DR.%2BL.%26aulast%3DOswald%26aufirst%3DE.%26aulast%3DHatzi%26aufirst%3DK.%26aulast%3DYang%26aufirst%3DS.-N.%26aulast%3DCohen%26aufirst%3DJ.%26aulast%3DShaknovich%26aufirst%3DR.%26aulast%3DVanommeslaeghe%26aufirst%3DK.%26aulast%3DCheng%26aufirst%3DH.%26aulast%3DLiang%26aufirst%3DD.%26aulast%3DCho%26aufirst%3DH.%2BJ.%26aulast%3DAbbott%26aufirst%3DJ.%26aulast%3DTam%26aufirst%3DW.%26aulast%3DDu%26aufirst%3DW.%26aulast%3DLeonard%26aufirst%3DJ.%2BP.%26aulast%3DElemento%26aufirst%3DO.%26aulast%3DCerchietti%26aufirst%3DL.%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DXue%26aufirst%3DF.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26aulast%3DMelnick%26aufirst%3DA.%2BM.%26atitle%3DRationally%2520designed%2520BCL6%2520inhibitors%2520target%2520activated%2520B%2520cell%2520diffuse%2520large%2520B%2520cell%2520lymphoma%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2016%26volume%3D126%26issue%3D9%26spage%3D3351%26epage%3D3362%26doi%3D10.1172%2FJCI85795" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kamada, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakai, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sogabe, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ida, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snell, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iida, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imaeda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsui, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a B-cell lymphoma 6 protein–protein interaction inhibitor by a biophysics-driven fragment-based approach</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">4358</span>– <span class="NLM_lpage">4368</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00313</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00313" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BC2sXntFSkt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4358-4368&issue=10&author=Y.+Kamadaauthor=N.+Sakaiauthor=S.+Sogabeauthor=K.+Idaauthor=H.+Okiauthor=K.+Sakamotoauthor=W.+Laneauthor=G.+Snellauthor=M.+Iidaauthor=Y.+Imaedaauthor=J.+Sakamotoauthor=J.+Matsui&title=Discovery+of+a+B-cell+lymphoma+6+protein%E2%80%93protein+interaction+inhibitor+by+a+biophysics-driven+fragment-based+approach&doi=10.1021%2Facs.jmedchem.7b00313"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a B-Cell Lymphoma 6 Protein-Protein Interaction Inhibitor by a Biophysics-Driven Fragment-Based Approach</span></div><div class="casAuthors">Kamada, Yusuke; Sakai, Nozomu; Sogabe, Satoshi; Ida, Koh; Oki, Hideyuki; Sakamoto, Kotaro; Lane, Weston; Snell, Gyorgy; Iida, Motoo; Imaeda, Yasuhiro; Sakamoto, Junichi; Matsui, Junji</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4358-4368</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">B-cell lymphoma 6 (BCL6) is a transcriptional factor that expresses in lymphocytes and regulates the differentiation and proliferation of lymphocytes.  Therefore, BCL6 is a therapeutic target for autoimmune diseases and cancer treatment.  This report presents the discovery of BCL6-corepressor interaction inhibitors by using a biophysics-driven fragment-based approach.  Using the surface plasmon resonance (SPR)-based fragment screening, we successfully identified fragment 1 (SPR KD = 1200 μM, ligand efficiency (LE) = 0.28), a competitive binder to the natural ligand BCoR peptide.  Moreover, we elaborated 1 into the more potent compd. 7 (SPR KD = 0.078 μM, LE = 0.37, cell-free protein-protein interaction (PPI) IC50 = 0.48 μM (ELISA), cellular PPI IC50 = 8.6 μM (M2H)) by a structure-based design and structural integration with a second high-throughput screening hit.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0yLaTbq3FqbVg90H21EOLACvtfcHk0lh70wJq8S3Tyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXntFSkt7o%253D&md5=c1c6a4e57ec5655ed2c937f5274ea6ec</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00313%26sid%3Dliteratum%253Aachs%26aulast%3DKamada%26aufirst%3DY.%26aulast%3DSakai%26aufirst%3DN.%26aulast%3DSogabe%26aufirst%3DS.%26aulast%3DIda%26aufirst%3DK.%26aulast%3DOki%26aufirst%3DH.%26aulast%3DSakamoto%26aufirst%3DK.%26aulast%3DLane%26aufirst%3DW.%26aulast%3DSnell%26aufirst%3DG.%26aulast%3DIida%26aufirst%3DM.%26aulast%3DImaeda%26aufirst%3DY.%26aulast%3DSakamoto%26aufirst%3DJ.%26aulast%3DMatsui%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%2520B-cell%2520lymphoma%25206%2520protein%25E2%2580%2593protein%2520interaction%2520inhibitor%2520by%2520a%2520biophysics-driven%2520fragment-based%2520approach%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26issue%3D10%26spage%3D4358%26epage%3D4368%26doi%3D10.1021%2Facs.jmedchem.7b00313" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yasui, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakai, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asano, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshitomi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takagi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sogabe, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamada, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snell, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwata, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inooka, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, J.-i.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakada, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imaeda, Y.</span></span> <span> </span><span class="NLM_article-title">Discovery of a novel B-cell lymphoma 6 (BCL6)–corepressor interaction inhibitor by utilizing structure-based drug design</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>25</i></span> (<span class="NLM_issue">17</span>),  <span class="NLM_fpage">4876</span>– <span class="NLM_lpage">4886</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2017.07.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10.1016%2Fj.bmc.2017.07.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=28760529" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1GisbvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=4876-4886&issue=17&author=T.+Yasuiauthor=T.+Yamamotoauthor=N.+Sakaiauthor=K.+Asanoauthor=T.+Takaiauthor=Y.+Yoshitomiauthor=M.+Davisauthor=T.+Takagiauthor=K.+Sakamotoauthor=S.+Sogabeauthor=Y.+Kamadaauthor=W.+Laneauthor=G.+Snellauthor=M.+Iwataauthor=M.+Gotoauthor=H.+Inookaauthor=J.-i.+Sakamotoauthor=Y.+Nakadaauthor=Y.+Imaeda&title=Discovery+of+a+novel+B-cell+lymphoma+6+%28BCL6%29%E2%80%93corepressor+interaction+inhibitor+by+utilizing+structure-based+drug+design&doi=10.1016%2Fj.bmc.2017.07.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a novel B-cell lymphoma 6 (BCL6)-corepressor interaction inhibitor by utilizing structure-based drug design</span></div><div class="casAuthors">Yasui, Takeshi; Yamamoto, Takeshi; Sakai, Nozomu; Asano, Kouhei; Takai, Takafumi; Yoshitomi, Yayoi; Davis, Melinda; Takagi, Terufumi; Sakamoto, Kotaro; Sogabe, Satoshi; Kamada, Yusuke; Lane, Weston; Snell, Gyorgy; Iwata, Masashi; Goto, Masayuki; Inooka, Hiroshi; Sakamoto, Jun-ichi; Nakada, Yoshihisa; Imaeda, Yasuhiro</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4876-4886</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">B-cell lymphoma 6 (BCL6) is a transcriptional repressor that can form complexes with corepressors via protein-protein interactions (PPIs).  The complexes of BCL6 and corepressors play an important role in the formation of germinal centers (GCs), and differentiation and proliferation of lymphocytes.  Therefore, BCL6-corepressor interaction inhibitors would be drug candidates for managing autoimmune diseases and cancer.  Starting from high-throughput screening hits I and II, the authors identified a novel BCL6-corepressor interaction inhibitor III (cell-free ELISA [ELISA] IC50 = 0.10 μM, cell-based mammalian two-hybrid [M2H] assay IC50 = 0.72 μM) by utilizing structure-based drug design (SBDD) based on an x-ray crystal structure of I bound to BCL6.  Compd. III also showed a good pharmacokinetic profile, which was acceptable for both in vitro and in vivo studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorDO1ByXavNrVg90H21EOLACvtfcHk0lh70wJq8S3Tyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1GisbvF&md5=2a9cae7f5e2ec480bd5ae106cc4cba31</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2017.07.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2017.07.037%26sid%3Dliteratum%253Aachs%26aulast%3DYasui%26aufirst%3DT.%26aulast%3DYamamoto%26aufirst%3DT.%26aulast%3DSakai%26aufirst%3DN.%26aulast%3DAsano%26aufirst%3DK.%26aulast%3DTakai%26aufirst%3DT.%26aulast%3DYoshitomi%26aufirst%3DY.%26aulast%3DDavis%26aufirst%3DM.%26aulast%3DTakagi%26aufirst%3DT.%26aulast%3DSakamoto%26aufirst%3DK.%26aulast%3DSogabe%26aufirst%3DS.%26aulast%3DKamada%26aufirst%3DY.%26aulast%3DLane%26aufirst%3DW.%26aulast%3DSnell%26aufirst%3DG.%26aulast%3DIwata%26aufirst%3DM.%26aulast%3DGoto%26aufirst%3DM.%26aulast%3DInooka%26aufirst%3DH.%26aulast%3DSakamoto%26aufirst%3DJ.-i.%26aulast%3DNakada%26aufirst%3DY.%26aulast%3DImaeda%26aufirst%3DY.%26atitle%3DDiscovery%2520of%2520a%2520novel%2520B-cell%2520lymphoma%25206%2520%2528BCL6%2529%25E2%2580%2593corepressor%2520interaction%2520inhibitor%2520by%2520utilizing%2520structure-based%2520drug%2520design%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2017%26volume%3D25%26issue%3D17%26spage%3D4876%26epage%3D4886%26doi%3D10.1016%2Fj.bmc.2017.07.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoull, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abrams, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blades, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Box, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byth, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuaqui, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carbajo, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Code, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fillery, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuller, N. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gangl, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grist, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hargreaves, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kemmitt, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connell, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prince, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raubo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rawlins, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robb, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waring, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whittaker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wylot, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span> <span> </span><span class="NLM_article-title">Discovery of pyrazolo[1,5-a]pyrimidine B-cell lymphoma 6 (BCL6) binders and optimization to high affinity macrocyclic inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">4386</span>– <span class="NLM_lpage">4402</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00359</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00359" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BC2sXntFOmsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4386-4402&issue=10&author=W.+McCoullauthor=R.+D.+Abramsauthor=E.+Andersonauthor=K.+Bladesauthor=P.+Bartonauthor=M.+Boxauthor=J.+Burgessauthor=K.+Bythauthor=Q.+Caoauthor=C.+Chuaquiauthor=R.+J.+Carbajoauthor=T.+Cheungauthor=E.+Codeauthor=A.+D.+Fergusonauthor=S.+Filleryauthor=N.+O.+Fullerauthor=E.+Ganglauthor=N.+Gaoauthor=M.+Gristauthor=D.+Hargreavesauthor=M.+R.+Howardauthor=J.+Huauthor=P.+D.+Kemmittauthor=J.+E.+Nelsonauthor=N.+O%E2%80%99Connellauthor=D.+B.+Princeauthor=P.+Rauboauthor=P.+B.+Rawlinsauthor=G.+R.+Robbauthor=J.+Shiauthor=M.+J.+Waringauthor=D.+Whittakerauthor=M.+Wylotauthor=X.+Zhu&title=Discovery+of+pyrazolo%5B1%2C5-a%5Dpyrimidine+B-cell+lymphoma+6+%28BCL6%29+binders+and+optimization+to+high+affinity+macrocyclic+inhibitors&doi=10.1021%2Facs.jmedchem.7b00359"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Pyrazolo[1,5-a]pyrimidine B-Cell Lymphoma 6 (BCL6) Binders and Optimization to High Affinity Macrocyclic Inhibitors</span></div><div class="casAuthors">McCoull, William; Abrams, Roman D.; Anderson, Erica; Blades, Kevin; Barton, Peter; Box, Matthew; Burgess, Jonathan; Byth, Kate; Cao, Qing; Chuaqui, Claudio; Carbajo, Rodrigo J.; Cheung, Tony; Code, Erin; Ferguson, Andrew D.; Fillery, Shaun; Fuller, Nathan O.; Gangl, Eric; Gao, Ning; Grist, Matthew; Hargreaves, David; Howard, Martin R.; Hu, Jun; Kemmitt, Paul D.; Nelson, Jennifer E.; O'Connell, Nichole; Prince, D. Bryan; Raubo, Piotr; Rawlins, Philip B.; Robb, Graeme R.; Shi, Junjie; Waring, Michael J.; Whittaker, David; Wylot, Marta; Zhu, Xiahui</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4386-4402</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibition of the protein-protein interaction between B-cell lymphoma 6 (BCL6) and corepressors has been implicated as a therapeutic target in diffuse large B-cell lymphoma (DLBCL) cancers and profiling of potent and selective BCL6 inhibitors are crit. to test this hypothesis.  We identified a pyrazolo[1,5-a]pyrimidine series of BCL6 binders from a fragment screen in parallel with a virtual screen.  Using structure-based drug design, binding affinity was increased 100000-fold.  This involved displacing crystallog. water, forming new ligand-protein interactions and a macrocyclization to favor the bioactive conformation of the ligands.  Optimization for slow off-rate const. kinetics was conducted as well as improving selectivity against an off-target kinase, CK2.  Potency in a cellular BCL6 assay was further optimized to afford highly selective probe mols.  Only weak antiproliferative effects were obsd. across a no. of DLBCL lines and a multiple myeloma cell line without a clear relationship to BCL6 potency.  As a result, we conclude that the BCL6 hypothesis in DLBCL cancer remains unproven.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWp2OlXAZDFLVg90H21EOLACvtfcHk0lh70wJq8S3Tyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXntFOmsbo%253D&md5=fc4142453a0580edbf04802e15d4d9aa</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00359&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00359%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoull%26aufirst%3DW.%26aulast%3DAbrams%26aufirst%3DR.%2BD.%26aulast%3DAnderson%26aufirst%3DE.%26aulast%3DBlades%26aufirst%3DK.%26aulast%3DBarton%26aufirst%3DP.%26aulast%3DBox%26aufirst%3DM.%26aulast%3DBurgess%26aufirst%3DJ.%26aulast%3DByth%26aufirst%3DK.%26aulast%3DCao%26aufirst%3DQ.%26aulast%3DChuaqui%26aufirst%3DC.%26aulast%3DCarbajo%26aufirst%3DR.%2BJ.%26aulast%3DCheung%26aufirst%3DT.%26aulast%3DCode%26aufirst%3DE.%26aulast%3DFerguson%26aufirst%3DA.%2BD.%26aulast%3DFillery%26aufirst%3DS.%26aulast%3DFuller%26aufirst%3DN.%2BO.%26aulast%3DGangl%26aufirst%3DE.%26aulast%3DGao%26aufirst%3DN.%26aulast%3DGrist%26aufirst%3DM.%26aulast%3DHargreaves%26aufirst%3DD.%26aulast%3DHoward%26aufirst%3DM.%2BR.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DKemmitt%26aufirst%3DP.%2BD.%26aulast%3DNelson%26aufirst%3DJ.%2BE.%26aulast%3DO%25E2%2580%2599Connell%26aufirst%3DN.%26aulast%3DPrince%26aufirst%3DD.%2BB.%26aulast%3DRaubo%26aufirst%3DP.%26aulast%3DRawlins%26aufirst%3DP.%2BB.%26aulast%3DRobb%26aufirst%3DG.%2BR.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DWaring%26aufirst%3DM.%2BJ.%26aulast%3DWhittaker%26aufirst%3DD.%26aulast%3DWylot%26aufirst%3DM.%26aulast%3DZhu%26aufirst%3DX.%26atitle%3DDiscovery%2520of%2520pyrazolo%255B1%252C5-a%255Dpyrimidine%2520B-cell%2520lymphoma%25206%2520%2528BCL6%2529%2520binders%2520and%2520optimization%2520to%2520high%2520affinity%2520macrocyclic%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26issue%3D10%26spage%3D4386%26epage%3D4402%26doi%3D10.1021%2Facs.jmedchem.7b00359" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoull, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byth, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castaldi, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiarparin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carbajo, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Code, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davey, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fillery, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuller, N. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hargreaves, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawatkar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kemmitt, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molina, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connell, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petteruti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rasmusson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raubo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rawlins, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ricchiuto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robb, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schenone, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waring, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fawell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. M.</span></span> <span> </span><span class="NLM_article-title">Development of a novel B-cell lymphoma 6 (BCL6) PROTAC to provide insight into small molecule targeting of BCL6</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">3131</span>– <span class="NLM_lpage">3141</span>, <span class="refDoi"> DOI: 10.1021/acschembio.8b00698</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.8b00698" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFaisrnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=3131-3141&issue=11&author=W.+McCoullauthor=T.+Cheungauthor=E.+Andersonauthor=P.+Bartonauthor=J.+Burgessauthor=K.+Bythauthor=Q.+Caoauthor=M.+P.+Castaldiauthor=H.+Chenauthor=E.+Chiarparinauthor=R.+J.+Carbajoauthor=E.+Codeauthor=S.+Cowanauthor=P.+R.+Daveyauthor=A.+D.+Fergusonauthor=S.+Filleryauthor=N.+O.+Fullerauthor=N.+Gaoauthor=D.+Hargreavesauthor=M.+R.+Howardauthor=J.+Huauthor=A.+Kawatkarauthor=P.+D.+Kemmittauthor=E.+Leoauthor=D.+M.+Molinaauthor=N.+O%E2%80%99Connellauthor=P.+Petterutiauthor=T.+Rasmussonauthor=P.+Rauboauthor=P.+B.+Rawlinsauthor=P.+Ricchiutoauthor=G.+R.+Robbauthor=M.+Schenoneauthor=M.+J.+Waringauthor=M.+Zindaauthor=S.+Fawellauthor=D.+M.+Wilson&title=Development+of+a+novel+B-cell+lymphoma+6+%28BCL6%29+PROTAC+to+provide+insight+into+small+molecule+targeting+of+BCL6&doi=10.1021%2Facschembio.8b00698"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a Novel B-Cell Lymphoma 6 (BCL6) PROTAC To Provide Insight into Small Molecule Targeting of BCL6</span></div><div class="casAuthors">McCoull, William; Cheung, Tony; Anderson, Erica; Barton, Peter; Burgess, Jonathan; Byth, Kate; Cao, Qing; Castaldi, M. Paola; Chen, Huawei; Chiarparin, Elisabetta; Carbajo, Rodrigo J.; Code, Erin; Cowan, Suzanna; Davey, Paul R.; Ferguson, Andrew D.; Fillery, Shaun; Fuller, Nathan O.; Gao, Ning; Hargreaves, David; Howard, Martin R.; Hu, Jun; Kawatkar, Aarti; Kemmitt, Paul D.; Leo, Elisabetta; Molina, Daniel M.; O'Connell, Nichole; Petteruti, Philip; Rasmusson, Timothy; Raubo, Piotr; Rawlins, Philip B.; Ricchiuto, Piero; Robb, Graeme R.; Schenone, Monica; Waring, Michael J.; Zinda, Michael; Fawell, Stephen; Wilson, David M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3131-3141</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">B-cell lymphoma 6 (BCL6) inhibition is a promising mechanism for treating hematol. cancers but high quality chem. probes are necessary to evaluate its therapeutic potential.  Here we report potent BCL6 inhibitors that demonstrate cellular target engagement and exhibit exquisite selectivity for BCL6 based on mass spectrometry analyses following chem. proteomic pull down.  Importantly, a proteolysis-targeting chimera, 4-((R)-25-Chloro-41-methyl-42-oxo-41,42,43,44-tetrahydro-5,8-dioxa-3-aza-4(6,8)-quinolina2(2,4)-pyrimidina-1(1,2)-piperazinacyclononaphane-14-yl)-N-(3-(2-(2-((2-((S)-2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)ethoxy)ethoxy)propyl)-4-oxobutanamide trifluoroacetate, (PROTAC, 15) was also developed and shown to significantly degrade BCL6 in a no. of diffuse large B-cell lymphoma (DLBCL) cell lines, but neither BCL6 inhibition nor degrdn. selectively induced marked phenotypic response.  To investigate, we monitored PROTAC directed BCL6 degrdn. in DLBCL OCI-Ly1 cells by immunofluorescence and discovered a residual BCL6 population.  Anal. of subcellular fractions also showed incomplete BCL6 degrdn. in all fractions despite having measurable PROTAC concns., together providing a rationale for the weak antiproliferative response seen with both BCL6 inhibitor and degrader.  In summary, we have developed potent and selective BCL6 inhibitors and a BCL6 PROTAC that effectively degraded BCL6, but both modalities failed to induce a significant phenotypic response in DLBCL despite achieving cellular concns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEe9aUjqcaHLVg90H21EOLACvtfcHk0ljAAurRPMpDow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFaisrnF&md5=3d2c2d8acf143f05babee6ebdabddc8c</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facschembio.8b00698&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.8b00698%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoull%26aufirst%3DW.%26aulast%3DCheung%26aufirst%3DT.%26aulast%3DAnderson%26aufirst%3DE.%26aulast%3DBarton%26aufirst%3DP.%26aulast%3DBurgess%26aufirst%3DJ.%26aulast%3DByth%26aufirst%3DK.%26aulast%3DCao%26aufirst%3DQ.%26aulast%3DCastaldi%26aufirst%3DM.%2BP.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DChiarparin%26aufirst%3DE.%26aulast%3DCarbajo%26aufirst%3DR.%2BJ.%26aulast%3DCode%26aufirst%3DE.%26aulast%3DCowan%26aufirst%3DS.%26aulast%3DDavey%26aufirst%3DP.%2BR.%26aulast%3DFerguson%26aufirst%3DA.%2BD.%26aulast%3DFillery%26aufirst%3DS.%26aulast%3DFuller%26aufirst%3DN.%2BO.%26aulast%3DGao%26aufirst%3DN.%26aulast%3DHargreaves%26aufirst%3DD.%26aulast%3DHoward%26aufirst%3DM.%2BR.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DKawatkar%26aufirst%3DA.%26aulast%3DKemmitt%26aufirst%3DP.%2BD.%26aulast%3DLeo%26aufirst%3DE.%26aulast%3DMolina%26aufirst%3DD.%2BM.%26aulast%3DO%25E2%2580%2599Connell%26aufirst%3DN.%26aulast%3DPetteruti%26aufirst%3DP.%26aulast%3DRasmusson%26aufirst%3DT.%26aulast%3DRaubo%26aufirst%3DP.%26aulast%3DRawlins%26aufirst%3DP.%2BB.%26aulast%3DRicchiuto%26aufirst%3DP.%26aulast%3DRobb%26aufirst%3DG.%2BR.%26aulast%3DSchenone%26aufirst%3DM.%26aulast%3DWaring%26aufirst%3DM.%2BJ.%26aulast%3DZinda%26aufirst%3DM.%26aulast%3DFawell%26aufirst%3DS.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26atitle%3DDevelopment%2520of%2520a%2520novel%2520B-cell%2520lymphoma%25206%2520%2528BCL6%2529%2520PROTAC%2520to%2520provide%2520insight%2520into%2520small%2520molecule%2520targeting%2520of%2520BCL6%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26issue%3D11%26spage%3D3131%26epage%3D3141%26doi%3D10.1021%2Facschembio.8b00698" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kerres, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steurer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlager, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caligiuri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dank, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engen, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ettmayer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischerauer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flotzinger, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerlach, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstberger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmaschitz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greb, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heyes, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iacob, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kölle, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamarre, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lancia, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayr, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mischerikow, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mück, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peinsipp, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petermann, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiser, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudolph, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumpel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salomon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scharn, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnitzer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrenk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schweifer, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traxler, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varecka, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voss, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss-Puxbaum, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoephel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraut, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConnell, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koegl, M.</span></span> <span> </span><span class="NLM_article-title">Chemically induced degradation of the oncogenic transcription factor BCL6</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>20</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">2860</span>– <span class="NLM_lpage">2875</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2017.08.081</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10.1016%2Fj.celrep.2017.08.081" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=28930682" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFeqsrfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2017&pages=2860-2875&issue=12&author=N.+Kerresauthor=S.+Steurerauthor=S.+Schlagerauthor=G.+Baderauthor=H.+Bergerauthor=M.+Caligiuriauthor=C.+Dankauthor=J.+R.+Engenauthor=P.+Ettmayerauthor=B.+Fischerauerauthor=G.+Flotzingerauthor=D.+Gerlachauthor=T.+Gerstbergerauthor=T.+Gmaschitzauthor=P.+Grebauthor=B.+Hanauthor=E.+Heyesauthor=R.+E.+Iacobauthor=D.+Kesslerauthor=H.+K%C3%B6lleauthor=L.+Lamarreauthor=D.+R.+Lanciaauthor=S.+Lucasauthor=M.+Mayerauthor=K.+Mayrauthor=N.+Mischerikowauthor=K.+M%C3%BCckauthor=C.+Peinsippauthor=O.+Petermannauthor=U.+Reiserauthor=D.+Rudolphauthor=K.+Rumpelauthor=C.+Salomonauthor=D.+Scharnauthor=R.+Schnitzerauthor=A.+Schrenkauthor=N.+Schweiferauthor=D.+Thompsonauthor=E.+Traxlerauthor=R.+Vareckaauthor=T.+Vossauthor=A.+Weiss-Puxbaumauthor=S.+Winklerauthor=X.+Zhengauthor=A.+Zoephelauthor=N.+Krautauthor=D.+McConnellauthor=M.+Pearsonauthor=M.+Koegl&title=Chemically+induced+degradation+of+the+oncogenic+transcription+factor+BCL6&doi=10.1016%2Fj.celrep.2017.08.081"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Chemically Induced Degradation of the Oncogenic Transcription Factor BCL6</span></div><div class="casAuthors">Kerres, Nina; Steurer, Steffen; Schlager, Stefanie; Bader, Gerd; Berger, Helmut; Caligiuri, Maureen; Dank, Christian; Engen, John R.; Ettmayer, Peter; Fischerauer, Bernhard; Flotzinger, Gerlinde; Gerlach, Daniel; Gerstberger, Thomas; Gmaschitz, Teresa; Greb, Peter; Han, Bingsong; Heyes, Elizabeth; Iacob, Roxana E.; Kessler, Dirk; Koelle, Heike; Lamarre, Lyne; Lancia, David R.; Lucas, Simon; Mayer, Moriz; Mayr, Katharina; Mischerikow, Nikolai; Mueck, Katja; Peinsipp, Christoph; Petermann, Oliver; Reiser, Ulrich; Rudolph, Dorothea; Rumpel, Klaus; Salomon, Carina; Scharn, Dirk; Schnitzer, Renate; Schrenk, Andreas; Schweifer, Norbert; Thompson, Diane; Traxler, Elisabeth; Varecka, Roland; Voss, Tilman; Weiss-Puxbaum, Alexander; Winkler, Sandra; Zheng, Xiaozhang; Zoephel, Andreas; Kraut, Norbert; McConnell, Darryl; Pearson, Mark; Koegl, Manfred</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2860-2875</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The transcription factor BCL6 is a known driver of oncogenesis in lymphoid malignancies, including diffuse large B cell lymphoma (DLBCL).  Disruption of its interaction with transcriptional repressors interferes with the oncogenic effects of BCL6.  We used a structure-based drug design to develop highly potent compds. that block this interaction.  A subset of these inhibitors also causes rapid ubiquitylation and degrdn. of BCL6 in cells.  These compds. display significantly stronger induction of expression of BCL6-repressed genes and anti-proliferative effects than compds. that merely inhibit co-repressor interactions.  This work establishes the BTB domain as a highly druggable structure, paving the way for the use of other members of this protein family as drug targets.  The magnitude of effects elicited by this class of BCL6-degrading compds. exceeds that of our equipotent non-degrading inhibitors, suggesting opportunities for the development of BCL6-based lymphoma therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXfmjxa_7LnrVg90H21EOLACvtfcHk0licF2oLE0jYVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFeqsrfK&md5=45b538db6643abbd5b8f3fa04770a772</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2017.08.081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2017.08.081%26sid%3Dliteratum%253Aachs%26aulast%3DKerres%26aufirst%3DN.%26aulast%3DSteurer%26aufirst%3DS.%26aulast%3DSchlager%26aufirst%3DS.%26aulast%3DBader%26aufirst%3DG.%26aulast%3DBerger%26aufirst%3DH.%26aulast%3DCaligiuri%26aufirst%3DM.%26aulast%3DDank%26aufirst%3DC.%26aulast%3DEngen%26aufirst%3DJ.%2BR.%26aulast%3DEttmayer%26aufirst%3DP.%26aulast%3DFischerauer%26aufirst%3DB.%26aulast%3DFlotzinger%26aufirst%3DG.%26aulast%3DGerlach%26aufirst%3DD.%26aulast%3DGerstberger%26aufirst%3DT.%26aulast%3DGmaschitz%26aufirst%3DT.%26aulast%3DGreb%26aufirst%3DP.%26aulast%3DHan%26aufirst%3DB.%26aulast%3DHeyes%26aufirst%3DE.%26aulast%3DIacob%26aufirst%3DR.%2BE.%26aulast%3DKessler%26aufirst%3DD.%26aulast%3DK%25C3%25B6lle%26aufirst%3DH.%26aulast%3DLamarre%26aufirst%3DL.%26aulast%3DLancia%26aufirst%3DD.%2BR.%26aulast%3DLucas%26aufirst%3DS.%26aulast%3DMayer%26aufirst%3DM.%26aulast%3DMayr%26aufirst%3DK.%26aulast%3DMischerikow%26aufirst%3DN.%26aulast%3DM%25C3%25BCck%26aufirst%3DK.%26aulast%3DPeinsipp%26aufirst%3DC.%26aulast%3DPetermann%26aufirst%3DO.%26aulast%3DReiser%26aufirst%3DU.%26aulast%3DRudolph%26aufirst%3DD.%26aulast%3DRumpel%26aufirst%3DK.%26aulast%3DSalomon%26aufirst%3DC.%26aulast%3DScharn%26aufirst%3DD.%26aulast%3DSchnitzer%26aufirst%3DR.%26aulast%3DSchrenk%26aufirst%3DA.%26aulast%3DSchweifer%26aufirst%3DN.%26aulast%3DThompson%26aufirst%3DD.%26aulast%3DTraxler%26aufirst%3DE.%26aulast%3DVarecka%26aufirst%3DR.%26aulast%3DVoss%26aufirst%3DT.%26aulast%3DWeiss-Puxbaum%26aufirst%3DA.%26aulast%3DWinkler%26aufirst%3DS.%26aulast%3DZheng%26aufirst%3DX.%26aulast%3DZoephel%26aufirst%3DA.%26aulast%3DKraut%26aufirst%3DN.%26aulast%3DMcConnell%26aufirst%3DD.%26aulast%3DPearson%26aufirst%3DM.%26aulast%3DKoegl%26aufirst%3DM.%26atitle%3DChemically%2520induced%2520degradation%2520of%2520the%2520oncogenic%2520transcription%2520factor%2520BCL6%26jtitle%3DCell%2520Rep.%26date%3D2017%26volume%3D20%26issue%3D12%26spage%3D2860%26epage%3D2875%26doi%3D10.1016%2Fj.celrep.2017.08.081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pierrat, O. A.</span>; <span class="NLM_string-name">Liu, M.</span>; <span class="NLM_string-name">Rodrigues, M.J.</span>; <span class="NLM_string-name">Collie, G. W.</span>; <span class="NLM_string-name">Le Bihan, Y-V</span>; <span class="NLM_string-name">Shetty, K. N.</span>; <span class="NLM_string-name">McAndrew, P. C.</span>; <span class="NLM_string-name">Richards, M.</span>; <span class="NLM_string-name">Stubbs, M.</span>; <span class="NLM_string-name">Rowlands, M. G.</span>; <span class="NLM_string-name">Yahya, N.</span>; <span class="NLM_string-name">Pickard, L.</span>; <span class="NLM_string-name">Shehu, E.</span>; <span class="NLM_string-name">Talbot, R.</span>; <span class="NLM_string-name">Bellenie, B. R.</span>; <span class="NLM_string-name">Davis, O. A.</span>; <span class="NLM_string-name">Varela, A.</span>; <span class="NLM_string-name">Cheung, K-M. J.</span>; <span class="NLM_string-name">Drouin, L.</span>; <span class="NLM_string-name">Innocenti, P.</span>; <span class="NLM_string-name">Woodward, H.</span>; <span class="NLM_string-name">Lloyd, M. G.</span>; <span class="NLM_string-name">Huckvale, R.</span>; <span class="NLM_string-name">Broccatelli, F.</span>; <span class="NLM_string-name">Carter, M.</span>; <span class="NLM_string-name">Galiwango, D.</span>; <span class="NLM_string-name">Hayes, A.</span>; <span class="NLM_string-name">Raynaud, F. I.</span>; <span class="NLM_string-name">Bryant, C.</span>; <span class="NLM_string-name">Whittaker, S.</span>; <span class="NLM_string-name">Hoelder, S.</span>; <span class="NLM_string-name">Burke, R.</span>; <span class="NLM_string-name">van Montfort, R. L. M.</span></span> Unpublished results.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Pierrat%2C+O.+A.%3B+Liu%2C+M.%3B+Rodrigues%2C+M.J.%3B+Collie%2C+G.+W.%3B+Le+Bihan%2C+Y-V%3B+Shetty%2C+K.+N.%3B+McAndrew%2C+P.+C.%3B+Richards%2C+M.%3B+Stubbs%2C+M.%3B+Rowlands%2C+M.+G.%3B+Yahya%2C+N.%3B+Pickard%2C+L.%3B+Shehu%2C+E.%3B+Talbot%2C+R.%3B+Bellenie%2C+B.+R.%3B+Davis%2C+O.+A.%3B+Varela%2C+A.%3B+Cheung%2C+K-M.+J.%3B+Drouin%2C+L.%3B+Innocenti%2C+P.%3B+Woodward%2C+H.%3B+Lloyd%2C+M.+G.%3B+Huckvale%2C+R.%3B+Broccatelli%2C+F.%3B+Carter%2C+M.%3B+Galiwango%2C+D.%3B+Hayes%2C+A.%3B+Raynaud%2C+F.+I.%3B+Bryant%2C+C.%3B+Whittaker%2C+S.%3B+Hoelder%2C+S.%3B+Burke%2C+R.%3B+van+Montfort%2C+R.+L.+M.+Unpublished+results."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DPierrat%26aufirst%3DO.%2BA." title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hill, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. J.</span></span> <span> </span><span class="NLM_article-title">Getting physical in drug discovery: a contemporary perspective on solubility and hydrophobicity</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">15–16</span>),  <span class="NLM_fpage">648</span>– <span class="NLM_lpage">655</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2010.05.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10.1016%2Fj.drudis.2010.05.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=20570751" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVeksb3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2010&pages=648-655&issue=15%E2%80%9316&author=A.+P.+Hillauthor=R.+J.+Young&title=Getting+physical+in+drug+discovery%3A+a+contemporary+perspective+on+solubility+and+hydrophobicity&doi=10.1016%2Fj.drudis.2010.05.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Getting physical in drug discovery: a contemporary perspective on solubility and hydrophobicity</span></div><div class="casAuthors">Hill, Alan P.; Young, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">15/16</span>),
    <span class="NLM_cas:pages">648-655</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Suboptimal phys. properties have been identified as a particular shortcoming of compds. in contemporary drug discovery, contributing to high attrition levels.  An anal. of the relationship between hydrophobicity (calcd. and measured) and ∼100 k measured kinetic soly. values has been undertaken.  In line with the General Soly. Equation, ests. of hydrophobicity, particularly ACD c log D pH7.4, give a useful indication of the likely soly. classification of particular mols.  Taking ACD c log D pH7.4 values together with the no. of arom. rings in a given mol. provides enhanced prediction.  The Soly. Forecast Index' (SFI = c log D pH7.4 + #Ar) is proposed as a simple, yet effective, guide to predicting soly.  Moreover, anal. of measured distribution/partition coeff. values highlighted statistically significant shortcomings in the applicability of octanol/water as a model system for hydrophobicity detn. with poorly sol. compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdtQyfNjbFUrVg90H21EOLACvtfcHk0licF2oLE0jYVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVeksb3P&md5=7b201081c5586a6ea4b7af00d507d462</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2010.05.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2010.05.016%26sid%3Dliteratum%253Aachs%26aulast%3DHill%26aufirst%3DA.%2BP.%26aulast%3DYoung%26aufirst%3DR.%2BJ.%26atitle%3DGetting%2520physical%2520in%2520drug%2520discovery%253A%2520a%2520contemporary%2520perspective%2520on%2520solubility%2520and%2520hydrophobicity%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2010%26volume%3D15%26issue%3D15%25E2%2580%259316%26spage%3D648%26epage%3D655%26doi%3D10.1016%2Fj.drudis.2010.05.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bissantz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahl, M.</span></span> <span> </span><span class="NLM_article-title">A medicinal chemist’s guide to molecular interactions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span> (<span class="NLM_issue">14</span>),  <span class="NLM_fpage">5061</span>– <span class="NLM_lpage">5084</span>, <span class="refDoi"> DOI: 10.1021/jm100112j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100112j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjvFKjtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5061-5084&issue=14&author=C.+Bissantzauthor=B.+Kuhnauthor=M.+Stahl&title=A+medicinal+chemist%E2%80%99s+guide+to+molecular+interactions&doi=10.1021%2Fjm100112j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">A Medicinal Chemist's Guide to Molecular Interactions</span></div><div class="casAuthors">Bissantz, Caterina; Kuhn, Bernd; Stahl, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5061-5084</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraV8HPbwsihbVg90H21EOLACvtfcHk0lih7aBZN53OCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjvFKjtbs%253D&md5=f39cf8700267cdc311c73de4d433bab0</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm100112j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100112j%26sid%3Dliteratum%253Aachs%26aulast%3DBissantz%26aufirst%3DC.%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DA%2520medicinal%2520chemist%25E2%2580%2599s%2520guide%2520to%2520molecular%2520interactions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26issue%3D14%26spage%3D5061%26epage%3D5084%26doi%3D10.1021%2Fjm100112j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bodnarchuk, M. S.</span></span> <span> </span><span class="NLM_article-title">Water, water, everywhere...It’s time to stop and think</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>21</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">1139</span>– <span class="NLM_lpage">1146</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2016.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10.1016%2Fj.drudis.2016.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=27210724" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BC28XoslWltb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=1139-1146&issue=7&author=M.+S.+Bodnarchuk&title=Water%2C+water%2C+everywhere...It%E2%80%99s+time+to+stop+and+think&doi=10.1016%2Fj.drudis.2016.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Water, water, everywhere... It's time to stop and think</span></div><div class="casAuthors">Bodnarchuk, Michael S.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1139-1146</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Despite the numerous methods available for predicting the location and affinity of water mols., there is still a degree of scepticism and reluctance towards using such information within a drug discovery program.  Here, I review some of the most common and popular methods to assess whether these apparent concerns are justified.  I suggest that the field is approaching maturity and that some methods are capable of giving quant. predictions, which are confirmed exptl.  This suggests that water-placement methods should be used more widely to help direct chem. efforts, although more successful examples are required to help validate the techniques.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxfKWV5PXTkrVg90H21EOLACvtfcHk0lih7aBZN53OCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XoslWltb4%253D&md5=dc2a72fa764bb19cd0c0ca2921fc974a</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2016.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2016.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DBodnarchuk%26aufirst%3DM.%2BS.%26atitle%3DWater%252C%2520water%252C%2520everywhere...It%25E2%2580%2599s%2520time%2520to%2520stop%2520and%2520think%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2016%26volume%3D21%26issue%3D7%26spage%3D1139%26epage%3D1146%26doi%3D10.1016%2Fj.drudis.2016.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Induced protein degradation: an emerging drug discovery paradigm</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">101</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10.1038%2Fnrd.2016.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=27885283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=101&author=A.+C.+Laiauthor=C.+M.+Crews&title=Induced+protein+degradation%3A+an+emerging+drug+discovery+paradigm&doi=10.1038%2Fnrd.2016.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Induced protein degradation: an emerging drug discovery paradigm</span></div><div class="casAuthors">Lai, Ashton C.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">101-114</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. drug discovery has traditionally focused on occupancy of a binding site that directly affects protein function, and this approach typically precludes targeting proteins that lack such amenable sites.  Furthermore, high systemic drug exposures may be needed to maintain sufficient target inhibition in vivo, increasing the risk of undesirable off-target effects.  Induced protein degrdn. is an alternative approach that is event-driven: upon drug binding, the target protein is tagged for elimination.  Emerging technologies based on proteolysis-targeting chimaeras (PROTACs) that exploit cellular quality control machinery to selectively degrade target proteins are attracting considerable attention in the pharmaceutical industry owing to the advantages they could offer over traditional small-mol. strategies.  These advantages include the potential to reduce systemic drug exposure, the ability to counteract increased target protein expression that often accompanies inhibition of protein function and the potential ability to target proteins that are not currently therapeutically tractable, such as transcription factors, scaffolding and regulatory proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-O7zXGOpXLVg90H21EOLACvtfcHk0lih7aBZN53OCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP&md5=a01ff43e9b696fe4dc03f0815158a98f</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.211%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DInduced%2520protein%2520degradation%253A%2520an%2520emerging%2520drug%2520discovery%2520paradigm%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D101%26doi%3D10.1038%2Fnrd.2016.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Broccatelli, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hop, C. E. C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, M.</span></span> <span> </span><span class="NLM_article-title">Strategies to optimize drug half-life in lead candidate identification</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">221</span>– <span class="NLM_lpage">230</span>, <span class="refDoi"> DOI: 10.1080/17460441.2019.1569625</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10.1080%2F17460441.2019.1569625" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=30675823" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvVCisbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=221-230&issue=3&author=F.+Broccatelliauthor=C.+E.+C.+A.+Hopauthor=M.+Wright&title=Strategies+to+optimize+drug+half-life+in+lead+candidate+identification&doi=10.1080%2F17460441.2019.1569625"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Strategies to optimize drug half-life in lead candidate identification</span></div><div class="casAuthors">Broccatelli, Fabio; Hop, Marcel; Wright, Matthew</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">221-230</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  : The PK optimization of drug candidates is one of the most resource-intensive tasks in pharmaceutical research and development.  With the increasing availability of in silico, in vitro and mechanistic in vivo ADME models, drug discovery scientists have progressively learned to recognize common SAR patterns and engineer data-driven strategies to accelerate the resoln. of ADME issues in lead optimization.  Many of these strategies gravitate toward the concept of drug-likeness, which defines a no. of optimal holistic physicochem. parameters (such as lipophilicity) that idealized oral drugs possess.  Herein, the authors discuss the interplay of lipophilicity with in vitro and in vivo ADME data in order to refine existing thought around drug half-life optimization.  Strategies to prolong the half-life of oral drugs via formulation are beyond the scope of this review.  Optimizing active properties such as potency, selectivity, and intrinsic metabolic clearance is an unambiguously beneficial strategy for small mols. within or beyond the Lipinski rule of five chem. space.  The data that we present in this work suggests that emphasis should be primarily placed on optimizing active properties such as potency, selectivity, and metabolic stability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrWxmcwdPTmLVg90H21EOLACvtfcHk0lih7aBZN53OCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvVCisbo%253D&md5=0a5b590ffdb385a2f36205874345a72b</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1080%2F17460441.2019.1569625&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F17460441.2019.1569625%26sid%3Dliteratum%253Aachs%26aulast%3DBroccatelli%26aufirst%3DF.%26aulast%3DHop%26aufirst%3DC.%2BE.%2BC.%2BA.%26aulast%3DWright%26aufirst%3DM.%26atitle%3DStrategies%2520to%2520optimize%2520drug%2520half-life%2520in%2520lead%2520candidate%2520identification%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2019%26volume%3D14%26issue%3D3%26spage%3D221%26epage%3D230%26doi%3D10.1080%2F17460441.2019.1569625" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berberich-Siebelt, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, R.</span></span> <span> </span><span class="NLM_article-title">The feedback loop of LITAF and BCL6 is involved in regulating apoptosis in B cell non-Hodgkin’s-lymphoma</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">47</span>),  <span class="NLM_fpage">77444</span>– <span class="NLM_lpage">77456</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.12680</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10.18632%2Foncotarget.12680" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=27764808" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A280%3ADC%252BC2srksFWmug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=77444-77456&issue=47&author=Y.+Shiauthor=Y.+Kuaiauthor=L.+Leiauthor=Y.+Wengauthor=F.+Berberich-Siebeltauthor=X.+Zhangauthor=J.+Wangauthor=Y.+Zhouauthor=X.+Jiangauthor=G.+Renauthor=H.+Panauthor=Z.+Maoauthor=R.+Zhou&title=The+feedback+loop+of+LITAF+and+BCL6+is+involved+in+regulating+apoptosis+in+B+cell+non-Hodgkin%E2%80%99s-lymphoma&doi=10.18632%2Foncotarget.12680"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">The feedback loop of LITAF and BCL6 is involved in regulating apoptosis in B cell non-Hodgkin's-lymphoma</span></div><div class="casAuthors">Shi Yaoyao; Kuai Yue; Lei Lizhen; Weng Yuanyuan; Jiang Xin; Mao Zhengrong; Zhou Ren; Shi Yaoyao; Berberich-Siebelt Friederike; Zhang Xinxia; Pan Hongyang; Wang Jinjie; Zhou Yuan; Ren Guoping</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">77444-77456</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Dysregulation of the apoptotic pathway is widely recognized as a key step in lymphomagenesis.  Notably, LITAF was initially identified as a p53-inducible gene, subsequently implicated as a tumor suppressor.  Our previous study also showed LITAF to be methylated in 89.5% B-NHL samples.  Conversely, deregulated expression of BCL6 is a pathogenic event in many lymphomas.  Interestingly, our study found an oppositional expression of LITAF and BCL6 in B-NHL.  In addition, LITAF was recently identified as a novel target gene of BCL6.  Therefore, we sought to explore the feedback loop between LITAF and BCL6 in B-NHL.  Here, our data for the first time show that LITAF can repress expression of BCL6 by binding to Region A (-87 to +65) containing a putative LITAF-binding motif (CTCCC) within the BCL6 promoter.  Furthermore, the regulation of BCL6 targets ( PRDM1 or c-Myc) by LITAF may be associated with B-cell differentiation.  Results also demonstrate that ectopic expression of LITAF induces cell apoptosis, activated by releasing cytochrome c, cleaving PARP and caspase 3 in B-NHL cells whereas knockdown of LITAF robustly protected cells from apoptosis.  Interestingly, BCL6, in turn, could reverse cell apoptosis mediated by LITAF.  Collectively, our findings provide a novel apoptotic regulatory pathway in which LITAF, as a transcription factor, inhibits the expression of BCL6, which leads to activation of the intrinsic mitochondrial pathway and tumor apoptosis.  Our study is expected to provide a possible biomarker as well as a target for clinical therapies to promote tumor cell apoptosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTvAmjS16coryjiVMCqkKxVfW6udTcc2eanq1uzJAriJ7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2srksFWmug%253D%253D&md5=4a71f277ffcfc61c5b2a2c0a781e486b</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.12680&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.12680%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DKuai%26aufirst%3DY.%26aulast%3DLei%26aufirst%3DL.%26aulast%3DWeng%26aufirst%3DY.%26aulast%3DBerberich-Siebelt%26aufirst%3DF.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DX.%26aulast%3DRen%26aufirst%3DG.%26aulast%3DPan%26aufirst%3DH.%26aulast%3DMao%26aufirst%3DZ.%26aulast%3DZhou%26aufirst%3DR.%26atitle%3DThe%2520feedback%2520loop%2520of%2520LITAF%2520and%2520BCL6%2520is%2520involved%2520in%2520regulating%2520apoptosis%2520in%2520B%2520cell%2520non-Hodgkin%25E2%2580%2599s-lymphoma%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26issue%3D47%26spage%3D77444%26epage%3D77456%26doi%3D10.18632%2Foncotarget.12680" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixit, V. M.</span></span> <span> </span><span class="NLM_article-title">Drugging the undruggables: exploring the ubiquitin system for drug development</span>. <i>Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">484</span>– <span class="NLM_lpage">498</span>, <span class="refDoi"> DOI: 10.1038/cr.2016.31</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10.1038%2Fcr.2016.31" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=27002218" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BC28XkvVyhs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=484-498&issue=4&author=X.+Huangauthor=V.+M.+Dixit&title=Drugging+the+undruggables%3A+exploring+the+ubiquitin+system+for+drug+development&doi=10.1038%2Fcr.2016.31"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Drugging the undruggables: exploring the ubiquitin system for drug development</span></div><div class="casAuthors">Huang, Xiaodong; Dixit, Vishva M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">484-498</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">NPG Nature Asia-Pacific</span>)
        </div><div class="casAbstract">Dynamic modulation of protein levels is tightly controlled in response to physiol. cues.  In mammalian cells, much of the protein degrdn. is carried out by the ubiquitin-proteasome system (UPS).  Similar to kinases, components of the ubiquitin system are often dysregulated, leading to a variety of diseases, including cancer and neurodegeneration, making them attractive drug targets.  However, so far there are only a handful of drugs targeting the ubiquitin system that have been approved by the FDA.  Here, we review possible therapeutic intervention nodes in the ubiquitin system, analyze the challenges, and highlight the most promising strategies to target the UPS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnGVCksLBzN7Vg90H21EOLACvtfcHk0lgkhafkHohvZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XkvVyhs70%253D&md5=5bda209d50a9179aa4e9c17e7fd491ad</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fcr.2016.31&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcr.2016.31%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DDixit%26aufirst%3DV.%2BM.%26atitle%3DDrugging%2520the%2520undruggables%253A%2520exploring%2520the%2520ubiquitin%2520system%2520for%2520drug%2520development%26jtitle%3DCell%2520Res.%26date%3D2016%26volume%3D26%26issue%3D4%26spage%3D484%26epage%3D498%26doi%3D10.1038%2Fcr.2016.31" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Collins, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldwell, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chopra, R.</span></span> <span> </span><span class="NLM_article-title">Chemical approaches to targeted protein degradation through modulation of the ubiquitin–proteasome pathway</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>474</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">1127</span>– <span class="NLM_lpage">1147</span>, <span class="refDoi"> DOI: 10.1042/BCJ20160762</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10.1042%2FBCJ20160762" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=28298557" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvVygt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=474&publication_year=2017&pages=1127-1147&issue=7&author=I.+Collinsauthor=H.+Wangauthor=J.+J.+Caldwellauthor=R.+Chopra&title=Chemical+approaches+to+targeted+protein+degradation+through+modulation+of+the+ubiquitin%E2%80%93proteasome+pathway&doi=10.1042%2FBCJ20160762"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical approaches to targeted protein degradation through modulation of the ubiquitin-proteasome pathway</span></div><div class="casAuthors">Collins, Ian; Wang, Hannah; Caldwell, John J.; Chopra, Raj</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">474</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1127-1147</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">Manipulation of the ubiquitin-proteasome system to achieve targeted degrdn. of proteins within cells using chem. tools and drugs has the potential to transform pharmacol. and therapeutic approaches in cancer and other diseases.  An increased understanding of the mol. mechanism of thalidomide and its analogs following their clin. use has unlocked small-mol. modulation of the substrate specificity of the E3 ligase cereblon (CRBN), which in turn has resulted in the advancement of new immunomodulatory drugs (IMiDs) into the clinic.  The degrdn. of multiple context-specific proteins by these pleiotropic small mols. provides a means to uncover new cell biol. and to generate future drug mols. against currently undruggable targets.  In parallel, the development of larger bifunctional mols. that bring together highly specific protein targets in complexes with CRBN, von Hippel-Lindau, or other E3 ligases to promote ubiquitin-dependent degrdn. has progressed to generate selective chem. compds. with potent effects in cells and in vivo models, providing valuable tools for biol. target validation and with future potential for therapeutic use.  In this review, we survey recent breakthroughs achieved in these two complementary methods and the discovery of new modes of direct and indirect engagement of target proteins with the proteasome.  We discuss the exptl. characterization that validates the use of mols. that promote protein degrdn. as chem. tools, the preclin. and clin. examples disclosed to date, and the future prospects for this exciting area of chem. biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMkoPeMHzjTbVg90H21EOLACvtfcHk0lgkhafkHohvZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvVygt7Y%253D&md5=8c9ccac22f5088d776b81203dc8ebf2b</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1042%2FBCJ20160762&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBCJ20160762%26sid%3Dliteratum%253Aachs%26aulast%3DCollins%26aufirst%3DI.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DCaldwell%26aufirst%3DJ.%2BJ.%26aulast%3DChopra%26aufirst%3DR.%26atitle%3DChemical%2520approaches%2520to%2520targeted%2520protein%2520degradation%2520through%2520modulation%2520of%2520the%2520ubiquitin%25E2%2580%2593proteasome%2520pathway%26jtitle%3DBiochem.%2520J.%26date%3D2017%26volume%3D474%26issue%3D7%26spage%3D1127%26epage%3D1147%26doi%3D10.1042%2FBCJ20160762" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mares, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, I. E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campos, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miah, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulholland, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Routly, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinn, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergamini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faelth-Savitski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willard, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flanagan, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casillas, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Votta, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">den Besten, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Famm, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruidenier, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harling, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Catalytic in vivo protein knockdown by small-molecule PROTACs</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">611</span>– <span class="NLM_lpage">617</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1858</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10.1038%2Fnchembio.1858" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=26075522" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=611-617&issue=8&author=D.+P.+Bondesonauthor=A.+Maresauthor=I.+E.+D.+Smithauthor=E.+Koauthor=S.+Camposauthor=A.+H.+Miahauthor=K.+E.+Mulhollandauthor=N.+Routlyauthor=D.+L.+Buckleyauthor=J.+L.+Gustafsonauthor=N.+Zinnauthor=P.+Grandiauthor=S.+Shimamuraauthor=G.+Bergaminiauthor=M.+Faelth-Savitskiauthor=M.+Bantscheffauthor=C.+Coxauthor=D.+A.+Gordonauthor=R.+R.+Willardauthor=J.+J.+Flanaganauthor=L.+N.+Casillasauthor=B.+J.+Vottaauthor=W.+den+Bestenauthor=K.+Fammauthor=L.+Kruidenierauthor=P.+S.+Carterauthor=J.+D.+Harlingauthor=I.+Churcherauthor=C.+M.+Crews&title=Catalytic+in+vivo+protein+knockdown+by+small-molecule+PROTACs&doi=10.1038%2Fnchembio.1858"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic in vivo protein knockdown by small-molecule PROTACs</span></div><div class="casAuthors">Bondeson, Daniel P.; Mares, Alina; Smith, Ian E. D.; Ko, Eunhwa; Campos, Sebastien; Miah, Afjal H.; Mulholland, Katie E.; Routly, Natasha; Buckley, Dennis L.; Gustafson, Jeffrey L.; Zinn, Nico; Grandi, Paola; Shimamura, Satoko; Bergamini, Giovanna; Faelth-Savitski, Maria; Bantscheff, Marcus; Cox, Carly; Gordon, Deborah A.; Willard, Ryan R.; Flanagan, John J.; Casillas, Linda N.; Votta, Bartholomew J.; den Besten, Willem; Famm, Kristoffer; Kruidenier, Laurens; Carter, Paul S.; Harling, John D.; Churcher, Ian; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">611-617</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The current predominant therapeutic paradigm is based on maximizing drug-receptor occupancy to achieve clin. benefit.  This strategy, however, generally requires excessive drug concns. to ensure sufficient occupancy, often leading to adverse side effects.  Here, we describe major improvements to the proteolysis targeting chimeras (PROTACs) method, a chem. knockdown strategy in which a heterobifunctional mol. recruits a specific protein target to an E3 ubiquitin ligase, resulting in the target's ubiquitination and degrdn.  These compds. behave catalytically in their ability to induce the ubiquitination of super-stoichiometric quantities of proteins, providing efficacy that is not limited by equil. occupancy.  We present two PROTACs that are capable of specifically reducing protein levels by >90% at nanomolar concns.  In addn., mouse studies indicate that they provide broad tissue distribution and knockdown of the targeted protein in tumor xenografts.  Together, these data demonstrate a protein knockdown system combining many of the favorable properties of small-mol. agents with the potent protein knockdown of RNAi and CRISPR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonhnajD3CxS7Vg90H21EOLACvtfcHk0ljUWzW_a6kSbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP&md5=31eb8756c5661f78ab4527a50dc1e251</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1858&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1858%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DMares%26aufirst%3DA.%26aulast%3DSmith%26aufirst%3DI.%2BE.%2BD.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DCampos%26aufirst%3DS.%26aulast%3DMiah%26aufirst%3DA.%2BH.%26aulast%3DMulholland%26aufirst%3DK.%2BE.%26aulast%3DRoutly%26aufirst%3DN.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DGustafson%26aufirst%3DJ.%2BL.%26aulast%3DZinn%26aufirst%3DN.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DShimamura%26aufirst%3DS.%26aulast%3DBergamini%26aufirst%3DG.%26aulast%3DFaelth-Savitski%26aufirst%3DM.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DCox%26aufirst%3DC.%26aulast%3DGordon%26aufirst%3DD.%2BA.%26aulast%3DWillard%26aufirst%3DR.%2BR.%26aulast%3DFlanagan%26aufirst%3DJ.%2BJ.%26aulast%3DCasillas%26aufirst%3DL.%2BN.%26aulast%3DVotta%26aufirst%3DB.%2BJ.%26aulast%3Dden%2BBesten%26aufirst%3DW.%26aulast%3DFamm%26aufirst%3DK.%26aulast%3DKruidenier%26aufirst%3DL.%26aulast%3DCarter%26aufirst%3DP.%2BS.%26aulast%3DHarling%26aufirst%3DJ.%2BD.%26aulast%3DChurcher%26aufirst%3DI.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DCatalytic%2520in%2520vivo%2520protein%2520knockdown%2520by%2520small-molecule%2520PROTACs%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26issue%3D8%26spage%3D611%26epage%3D617%26doi%3D10.1038%2Fnchembio.1858" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neklesa, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tae, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundberg, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stagg, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonnell, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Small-molecule-mediated degradation of the androgen receptor through hydrophobic tagging</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span> (<span class="NLM_issue">33</span>),  <span class="NLM_fpage">9659</span>– <span class="NLM_lpage">9662</span>, <span class="refDoi"> DOI: 10.1002/anie.201503720</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10.1002%2Fanie.201503720" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVegsbvO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=9659-9662&issue=33&author=J.+L.+Gustafsonauthor=T.+K.+Neklesaauthor=C.+S.+Coxauthor=A.+G.+Rothauthor=D.+L.+Buckleyauthor=H.+S.+Taeauthor=T.+B.+Sundbergauthor=D.+B.+Staggauthor=J.+Hinesauthor=D.+P.+McDonnellauthor=J.+D.+Norrisauthor=C.+M.+Crews&title=Small-molecule-mediated+degradation+of+the+androgen+receptor+through+hydrophobic+tagging&doi=10.1002%2Fanie.201503720"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule-Mediated Degradation of the Androgen Receptor Through Hydrophobic Tagging</span></div><div class="casAuthors">Gustafson, Jeffrey L.; Neklesa, Taavi K.; Cox, Carly S.; Roth, Anke G.; Buckley, Dennis L.; Tae, Hyun Seop; Sundberg, Thomas B.; Stagg, D. Blake; Hines, John; McDonnell, Donald P.; Norris, John D.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">33</span>),
    <span class="NLM_cas:pages">9659-9662</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Androgen receptor (AR)-dependent transcription is a major driver of prostate tumor cell proliferation.  Consequently, it is the target of several antitumor chemotherapeutic agents, including the AR antagonist MDV3100/enzalutamide.  Recent studies have shown that a single AR mutation (F876L) converts MDV3100 action from an antagonist to an agonist.  Here we describe the generation of a novel class of selective androgen receptor degraders (SARDs) to address this resistance mechanism.  Mols. contg. hydrophobic degrons linked to small-mol. AR ligands induce AR degrdn., reduce expression of AR target genes and inhibit proliferation in androgen-dependent prostate cancer cell lines.  These results suggest that selective AR degrdn. may be an effective therapeutic prostate tumor strategy in the context of AR mutations that confer resistance to second-generation AR antagonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBoEt08CZb-rVg90H21EOLACvtfcHk0ljUWzW_a6kSbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVegsbvO&md5=024225436378fcff0445d3b66696d69e</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1002%2Fanie.201503720&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201503720%26sid%3Dliteratum%253Aachs%26aulast%3DGustafson%26aufirst%3DJ.%2BL.%26aulast%3DNeklesa%26aufirst%3DT.%2BK.%26aulast%3DCox%26aufirst%3DC.%2BS.%26aulast%3DRoth%26aufirst%3DA.%2BG.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DTae%26aufirst%3DH.%2BS.%26aulast%3DSundberg%26aufirst%3DT.%2BB.%26aulast%3DStagg%26aufirst%3DD.%2BB.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DMcDonnell%26aufirst%3DD.%2BP.%26aulast%3DNorris%26aufirst%3DJ.%2BD.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DSmall-molecule-mediated%2520degradation%2520of%2520the%2520androgen%2520receptor%2520through%2520hydrophobic%2520tagging%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D54%26issue%3D33%26spage%3D9659%26epage%3D9662%26doi%3D10.1002%2Fanie.201503720" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 9 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Frances P. Rodriguez-Rivera, <span class="NLM_string-name hlFld-ContribAuthor">Samuel M. Levi</span>. </span><span class="cited-content_cbyCitation_article-title">Unifying Catalysis Framework to Dissect Proteasomal Degradation Paradigms. </span><span class="cited-content_cbyCitation_journal-name">ACS Central Science</span><span> <strong>2021,</strong> <em>7 </em>
                                    (7)
                                     , 1117-1125. <a href="https://doi.org/10.1021/acscentsci.1c00389" title="DOI URL">https://doi.org/10.1021/acscentsci.1c00389</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acscentsci.1c00389&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facscentsci.1c00389%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Central%2520Science%26atitle%3DUnifying%252BCatalysis%252BFramework%252Bto%252BDissect%252BProteasomal%252BDegradation%252BParadigms%26aulast%3DRodriguez-Rivera%26aufirst%3DFrances%2BP.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D27032021%26date%3D16062021%26volume%3D7%26issue%3D7%26spage%3D1117%26epage%3D1125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yong Ai, Lucia Hwang, Alexander D. MacKerell Jr., Ari Melnick, <span class="NLM_string-name hlFld-ContribAuthor">Fengtian Xue</span>. </span><span class="cited-content_cbyCitation_article-title">Progress toward B-Cell Lymphoma 6 BTB Domain Inhibitors for the Treatment of Diffuse Large B-Cell Lymphoma and Beyond. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (8)
                                     , 4333-4358. <a href="https://doi.org/10.1021/acs.jmedchem.0c01686" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01686</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01686&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01686%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DProgress%252Btoward%252BB-Cell%252BLymphoma%252B6%252BBTB%252BDomain%252BInhibitors%252Bfor%252Bthe%252BTreatment%252Bof%252BDiffuse%252BLarge%252BB-Cell%252BLymphoma%252Band%252BBeyond%26aulast%3DAi%26aufirst%3DYong%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D25092020%26date%3D12042021%26volume%3D64%26issue%3D8%26spage%3D4333%26epage%3D4358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andrew C.  Pearce</span>, <span class="hlFld-ContribAuthor ">Mark J.  Bamford</span>, <span class="hlFld-ContribAuthor ">Ruth  Barber</span>, <span class="hlFld-ContribAuthor ">Angela  Bridges</span>, <span class="hlFld-ContribAuthor ">Maire A.  Convery</span>, <span class="hlFld-ContribAuthor ">Constantinos  Demetriou</span>, <span class="hlFld-ContribAuthor ">Sian  Evans</span>, <span class="hlFld-ContribAuthor ">Thomas  Gobbetti</span>, <span class="hlFld-ContribAuthor ">David J.  Hirst</span>, <span class="hlFld-ContribAuthor ">Duncan S.  Holmes</span>, <span class="hlFld-ContribAuthor ">Jonathan P.  Hutchinson</span>, <span class="hlFld-ContribAuthor ">Sandrine  Jayne</span>, <span class="hlFld-ContribAuthor ">Larissa  Lezina</span>, <span class="hlFld-ContribAuthor ">Michael T.  McCabe</span>, <span class="hlFld-ContribAuthor ">Cassie  Messenger</span>, <span class="hlFld-ContribAuthor ">Joanne  Morley</span>, <span class="hlFld-ContribAuthor ">Melissa C.  Musso</span>, <span class="hlFld-ContribAuthor ">Paul  Scott-Stevens</span>, <span class="hlFld-ContribAuthor ">Ana Sousa  Manso</span>, <span class="hlFld-ContribAuthor ">Jennifer  Schofield</span>, <span class="hlFld-ContribAuthor ">Tom  Slocombe</span>, <span class="hlFld-ContribAuthor ">Don  Somers</span>, <span class="hlFld-ContribAuthor ">Ann L.  Walker</span>, <span class="hlFld-ContribAuthor ">Anastasia  Wyce</span>, <span class="hlFld-ContribAuthor ">Xi-Ping  Zhang</span>, <span class="hlFld-ContribAuthor ">Simon D.  Wagner</span>. </span><span class="cited-content_cbyCitation_article-title">GSK137, a potent small-molecule BCL6 inhibitor with in vivo activity, suppresses antibody responses in mice. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biological Chemistry</span><span> <strong>2021,</strong> <em>297 </em>
                                    (2)
                                     , 100928. <a href="https://doi.org/10.1016/j.jbc.2021.100928" title="DOI URL">https://doi.org/10.1016/j.jbc.2021.100928</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jbc.2021.100928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jbc.2021.100928%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biological%2520Chemistry%26atitle%3DGSK137%25252C%252Ba%252Bpotent%252Bsmall-molecule%252BBCL6%252Binhibitor%252Bwith%252Bin%2525C2%2525A0vivo%252Bactivity%25252C%252Bsuppresses%252Bantibody%252Bresponses%252Bin%252Bmice%26aulast%3DPearce%26aufirst%3DAndrew%2BC.%26date%3D2021%26volume%3D297%26issue%3D2%26spage%3D100928" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kalyn M  Rambacher</span>, <span class="hlFld-ContribAuthor ">Matthew F  Calabrese</span>, <span class="hlFld-ContribAuthor ">Masaya  Yamaguchi</span>. </span><span class="cited-content_cbyCitation_article-title">Perspectives on the development of first-in-class protein degraders. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2021,</strong> <em>13 </em>
                                    (14)
                                     , 1203-1226. <a href="https://doi.org/10.4155/fmc-2021-0033" title="DOI URL">https://doi.org/10.4155/fmc-2021-0033</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2021-0033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2021-0033%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DPerspectives%252Bon%252Bthe%252Bdevelopment%252Bof%252Bfirst-in-class%252Bprotein%252Bdegraders%26aulast%3DRambacher%26aufirst%3DKalyn%2BM%26date%3D2021%26volume%3D13%26issue%3D14%26spage%3D1203%26epage%3D1226" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Natalie S.  Scholes</span>, <span class="hlFld-ContribAuthor ">Cristina  Mayor-Ruiz</span>, <span class="hlFld-ContribAuthor ">Georg E.  Winter</span>. </span><span class="cited-content_cbyCitation_article-title">Identification and selectivity profiling of small-molecule degraders via multi-omics approaches. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2021,</strong> <em>28 </em>
                                    (7)
                                     , 1048-1060. <a href="https://doi.org/10.1016/j.chembiol.2021.03.007" title="DOI URL">https://doi.org/10.1016/j.chembiol.2021.03.007</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2021.03.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2021.03.007%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DIdentification%252Band%252Bselectivity%252Bprofiling%252Bof%252Bsmall-molecule%252Bdegraders%252Bvia%252Bmulti-omics%252Bapproaches%26aulast%3DScholes%26aufirst%3DNatalie%2BS.%26date%3D2021%26volume%3D28%26issue%3D7%26spage%3D1048%26epage%3D1060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zuzanna  Kozicka</span>, <span class="hlFld-ContribAuthor ">Nicolas Holger  Thomä</span>. </span><span class="cited-content_cbyCitation_article-title">Haven't got a glue: Protein surface variation for the design of molecular glue degraders. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2021,</strong> <em>28 </em>
                                    (7)
                                     , 1032-1047. <a href="https://doi.org/10.1016/j.chembiol.2021.04.009" title="DOI URL">https://doi.org/10.1016/j.chembiol.2021.04.009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2021.04.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2021.04.009%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DHaven%252527t%252Bgot%252Ba%252Bglue%25253A%252BProtein%252Bsurface%252Bvariation%252Bfor%252Bthe%252Bdesign%252Bof%252Bmolecular%252Bglue%252Bdegraders%26aulast%3DKozicka%26aufirst%3DZuzanna%26date%3D2021%26volume%3D28%26issue%3D7%26spage%3D1032%26epage%3D1047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Barbara  Orth</span>, <span class="hlFld-ContribAuthor ">Bodo  Sander</span>, <span class="hlFld-ContribAuthor ">Andreas  Möglich</span>, <span class="hlFld-ContribAuthor ">Kay  Diederichs</span>, <span class="hlFld-ContribAuthor ">Martin  Eilers</span>, <span class="hlFld-ContribAuthor ">Sonja  Lorenz</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of an atypical interaction site in the BTB domain of the MYC-interacting zinc-finger protein 1. </span><span class="cited-content_cbyCitation_journal-name">Structure</span><span> <strong>2021,</strong> <em>66 </em><a href="https://doi.org/10.1016/j.str.2021.06.005" title="DOI URL">https://doi.org/10.1016/j.str.2021.06.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.str.2021.06.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.str.2021.06.005%26sid%3Dliteratum%253Aachs%26jtitle%3DStructure%26atitle%3DIdentification%252Bof%252Ban%252Batypical%252Binteraction%252Bsite%252Bin%252Bthe%252BBTB%252Bdomain%252Bof%252Bthe%252BMYC-interacting%252Bzinc-finger%252Bprotein%252B1%26aulast%3DOrth%26aufirst%3DBarbara%26date%3D2021%26volume%3D66" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matthew J.  Henley</span>, <span class="hlFld-ContribAuthor ">Angela N.  Koehler</span>. </span><span class="cited-content_cbyCitation_article-title">Advances in targeting ‘undruggable’ transcription factors with small molecules. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Drug Discovery</span><span> <strong>2021,</strong> <em>172 </em><a href="https://doi.org/10.1038/s41573-021-00199-0" title="DOI URL">https://doi.org/10.1038/s41573-021-00199-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41573-021-00199-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41573-021-00199-0%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Drug%2520Discovery%26atitle%3DAdvances%252Bin%252Btargeting%252B%2525E2%252580%252598undruggable%2525E2%252580%252599%252Btranscription%252Bfactors%252Bwith%252Bsmall%252Bmolecules%26aulast%3DHenley%26aufirst%3DMatthew%2BJ.%26date%3D2021%26date%3D2021%26volume%3D172" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mikołaj  Słabicki</span>, <span class="hlFld-ContribAuthor ">Hojong  Yoon</span>, <span class="hlFld-ContribAuthor ">Jonas  Koeppel</span>, <span class="hlFld-ContribAuthor ">Lena  Nitsch</span>, <span class="hlFld-ContribAuthor ">Shourya S.  Roy Burman</span>, <span class="hlFld-ContribAuthor ">Cristina  Di Genua</span>, <span class="hlFld-ContribAuthor ">Katherine A.  Donovan</span>, <span class="hlFld-ContribAuthor ">Adam S.  Sperling</span>, <span class="hlFld-ContribAuthor ">Moritz  Hunkeler</span>, <span class="hlFld-ContribAuthor ">Jonathan M.  Tsai</span>, <span class="hlFld-ContribAuthor ">Rohan  Sharma</span>, <span class="hlFld-ContribAuthor ">Andrew  Guirguis</span>, <span class="hlFld-ContribAuthor ">Charles  Zou</span>, <span class="hlFld-ContribAuthor ">Priya  Chudasama</span>, <span class="hlFld-ContribAuthor ">Jessica A.  Gasser</span>, <span class="hlFld-ContribAuthor ">Peter G.  Miller</span>, <span class="hlFld-ContribAuthor ">Claudia  Scholl</span>, <span class="hlFld-ContribAuthor ">Stefan  Fröhling</span>, <span class="hlFld-ContribAuthor ">Radosław P.  Nowak</span>, <span class="hlFld-ContribAuthor ">Eric S.  Fischer</span>, <span class="hlFld-ContribAuthor ">Benjamin L.  Ebert</span>. </span><span class="cited-content_cbyCitation_article-title">Small-molecule-induced polymerization triggers degradation of BCL6. </span><span class="cited-content_cbyCitation_journal-name">Nature</span><span> <strong>2020,</strong> <em>588 </em>
                                    (7836)
                                     , 164-168. <a href="https://doi.org/10.1038/s41586-020-2925-1" title="DOI URL">https://doi.org/10.1038/s41586-020-2925-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41586-020-2925-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41586-020-2925-1%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%26atitle%3DSmall-molecule-induced%252Bpolymerization%252Btriggers%252Bdegradation%252Bof%252BBCL6%26aulast%3DS%25C5%2582abicki%26aufirst%3DMiko%25C5%2582aj%26date%3D2020%26date%3D2020%26volume%3D588%26issue%3D7836%26spage%3D164%26epage%3D168" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/medium/jm9b02076_0022.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0022.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02076&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/medium/jm9b02076_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0009.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Final Step Diversification of Benzimidazolone N<sup>3</sup> Position<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02076&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 2-ethyloxirane, cesium carbonate, DMF, 120 °C, 1 h; (b) sodium dithionite, ethanol/DMSO, rt to 90 °C; (c) 2,4-dichloropyridine-3-carbonitrile, DIPEA, DMA, 120 °C, 30–45 min; (d) (2<i>R</i>)- or (2<i>S</i>)-ethyloxirane, cesium carbonate, DMF, 120 °C, 1 h, then 140 °C, 1 h; (e) alkyl bromide or tosylate, cesium carbonate, DMF, 60 °C, 3 days or 120–140 °C, 1 h.</p></p></figure><figure data-id="sch2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/medium/jm9b02076_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0010.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Key Intermediate <b>16</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02076&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) disuccinimidylcarbonate, MeCN, rt, 18 h, up to 11g scale, 86–99% yield; (b) cesium carbonate, MeCN, reflux, 4 h, 12 g scale, 92% yield; (c) TsCl, NEt<sub>3</sub>, DCM, 0 °C to rt, up to 40g scale, 60–72% yield; (d) 10% Pd/C, 1 atm H<sub>2</sub>, ethanol, 60 °C, 3 h, 6 g scale, quant.</p></p></figure><figure data-id="sch3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/medium/jm9b02076_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0011.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds from <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>, Investigating Different Substitution Patterns on Pyrimidine or Pyridine Rings<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02076&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) heteroaryl chloride, DIPEA, NMP or DMF, 80–120 °C, 30 min; (b) 4-bromo- or iodopyridine, Xantphos, Pd<sub>2</sub>(dba)<sub>3</sub>, cesium carbonate, NMP/toluene, 140 °C, 1 h.</p></p></figure><figure data-id="sch4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/medium/jm9b02076_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0012.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of 2-Substituted Pyrimidine Compounds from <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Tables <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a>, Starting from Common Intermediate <b>18</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02076&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) dimethylamine HCl, cesium carbonate, NMP, 180 °C, 1 h; (b) cyclic amine, DIPEA, NMP, 140–180 °C, 1–2 h; (c) pyrazole, cesium carbonate, NMP, 170 °C, 1 h; (d) 2,4-dimethyl-5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)thiazole, sodium carbonate, PdCl<sub>2</sub>(PPh<sub>3</sub>)<sub>2</sub>, 1,4-dioxane/water, 130 °C, 30 min; (e) 1-methyl-2-(tributylstannyl)-1<i>H</i>-imidazole, PdCl<sub>2</sub>(PPh<sub>3</sub>)<sub>2</sub>, 1,4-dioxane, 90 °C, 18 h; (f) bicyclic amine, DIPEA, NMP, 140 °C, 2–8 h.</p></p></figure><figure data-id="sch5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/medium/jm9b02076_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0013.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of 2,4,5-Substituted Pyridine Derivatives<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02076&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>16,</b> DIPEA, NMP, 180 °C, 1 h; (b) 3,5-dimethylpyrazole, Xantphos, cesium carbonate. Pd<sub>2</sub>(dba)<sub>3</sub>, DMF/toluene, 140 °C, 1 h; (c) DIPEA, THF, 100 °C, 16 h; (d) <b>16,</b> Xantphos, Pd<sub>2</sub>(dba)<sub>3</sub>, cesium carbonate, DMF/toluene, 80 °C, 1 h.</p></p></figure><figure data-id="fig1" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/medium/jm9b02076_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0001.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structure-guided merging of <b>5</b> (A, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOG">6TOG</a>) and <b>4</b> (B, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOF">6TOF</a>) yielded the more potent <b>6</b> (CCT365386) (C, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOH">6TOH</a>). The surface of the BCL6 dimer is shown as a grey transparent surface. The two individual monomers are highlighted in gray and cyan ribbons except for residues 53–55 which are indicated in blue. Key protein residues are shown in line representation. The compounds are shown as orange ball and sticks, a selected water molecule is shown as a red sphere, and H-bonds are shown as yellow dashed lines. In panel C the orange arrow indicates an exit vector towards a large accessible pocket close to our compounds. Additional images containing overlays of <b>4</b> and <b>5</b> with <b>6</b> are shown in the <a href="/doi/suppl/10.1021/acs.jmedchem.9b02076/suppl_file/jm9b02076_si_001.pdf" class="ext-link">Supporting Information Figure S1</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02076&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/medium/jm9b02076_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0002.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Binding mode of enantiomers of <b>11a</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOO">6TOO</a>). Panel A and panel B show the binding mode of the respective (<i>S</i>)- and (<i>R</i>)-enantiomers of <b>11a</b>. Panel C shows the (<i>S</i>)-enantiomer and highlights the four water molecules targeted to gain potency. In all panels the surface of the BCL6 dimer is shown as a gray transparent surface, with the two individual monomers highlighted as ribbons and colored in gray and cyan, respectively. Selected residues are shown in line representation. Enantiomers of <b>11a</b> are shown as orange ball and sticks, selected water molecules as a red spheres, and H-bonds as yellow dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02076&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/medium/jm9b02076_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0003.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Binding mode of <b>11f</b> and <b>17a</b>. Panel A and Panel B show the respective BCL6 binding modes of <b>11f</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOI">6TOI</a>) and <b>17a</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOJ">6TOJ</a>). Compounds are shown as orange ball and sticks, selected water molecules as red spheres, and H-bonds as yellow dashed lines. The surface of BCL6 dimer is shown as a gray transparent surface. The two individual monomers are displayed as ribbons and colored in gray and cyan, respectively. Key residues are shown as lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02076&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/medium/jm9b02076_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0004.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Binding mode of <b>25b</b> and <b>23d</b>. Panels A and B show two different views of the binding mode of 25b (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TON">6TON</a>). Panels C and D show two views of the binding mode of <b>23d</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOK">6TOK</a>). The panels show the changed conformation of Arg28 and the new interaction with one of the compound’s pyrazole nitrogen atoms. In all panels, the surface of the BCL6 BTB dimer is shown as a gray transparent surface, with the two individual monomers displayed as ribbons and colored in gray and cyan. Key residues are shown as lines. Compounds are shown as orange ball and sticks, selected water molecules as red spheres, and H-bonds as yellow dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02076&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/medium/jm9b02076_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0005.jpeg" id="rightTab-gr18" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Treatment of DLBCL cell lines with <b>24b</b> (CCT368682) leads to degradation of BCL6. SU-DHL-4 or OCI-Ly1 cells were treated with <b>24b</b> at the indicated concentration for 4 h at 37 °C. Cells were collected, lysed, and examined by Western blot for BCL6 and GAPDH protein levels. D: DMSO control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02076&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/medium/jm9b02076_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0006.jpeg" id="rightTab-gr23" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (Left) Total SCID mouse blood concentrations of <b>1</b> (CCT369260) after po dosing at 5 (blue triangles), 15 (red circles), and 50 (green squares) mg/kg vs predicted active concentration (dashed line). (Right) Data for individual animals showing <i>c</i><sub>max</sub>, <i>t</i><sub>max</sub>, and total AUC<sub>10h</sub> in SCID mouse at 5, 15, and 50 mg/kg doses. All experiments were carried out according to the U.K. guidelines for animal experimentation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02076&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/medium/jm9b02076_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0007.jpeg" id="rightTab-gr24" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. PK/PD study with <b>1</b> (CCT369260) at 15 mg/kg po. Tumour xenografts were prepared by subcutaneous injection of 1 × 10<sup>7</sup> OCI-Ly1 cells in female SCID mice, with dosing of compound commencing 20 days after injection, to mice with xenografts between 0.5 and 0.8 cm<sup>3</sup>, as described in more detail in the <a href="/doi/suppl/10.1021/acs.jmedchem.9b02076/suppl_file/jm9b02076_si_001.pdf" class="ext-link">Supporting Information</a>. Sampling took place at 0.5, 2, 4, 6, 10, 12, 16 and 24 h postdose. All experiments were carried out according to the U.K. guidelines for animal experimentation. BCL6 levels in tumor were quantified using capillary electrophoresis and normalized to a GAPDH loading control and are shown as gray (vehicle-treated) or black (compound treated) bars. Free compound levels at 0.5–24 h are shown (blue dots); dotted line indicates free DC<sub>50</sub>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02076&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/medium/jm9b02076_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0008.jpeg" id="rightTab-gr25" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Binding modes of the BCL6 degrader <b>1</b> (CCT369260) (A) and inhibitor <b>25a</b> (B). Crystal structures of the BCL6 BTB domain bound to the BCL6 degrader <b>1</b> (CCT369260) (A, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOM">6TOM</a>) and inhibitor <b>25a</b> (B, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOL">6TOL</a>) show a nearly identical binding mode. In both panels, the surface of the BCL6 dimer is shown as a gray transparent surface, with the two individual monomers displayed in ribbons and colored in gray and cyan. Selected residues are shown as lines. Compounds are shown as orange ball and sticks, selected water molecules as red spheres, and H-bonds as yellow dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.9b02076/20200416/images/large/jm9b02076_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02076&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i83">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_22087" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_22087" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 26 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hatzi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnick, A.</span></span> <span> </span><span class="NLM_article-title">Breaking bad in the germinal center: how deregulation of BCL6 contributes to lymphomagenesis</span>. <i>Trends Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>20</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">343</span>– <span class="NLM_lpage">352</span>, <span class="refDoi"> DOI: 10.1016/j.molmed.2014.03.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10.1016%2Fj.molmed.2014.03.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=24698494" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltl2htrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=343-352&issue=6&author=K.+Hatziauthor=A.+Melnick&title=Breaking+bad+in+the+germinal+center%3A+how+deregulation+of+BCL6+contributes+to+lymphomagenesis&doi=10.1016%2Fj.molmed.2014.03.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Breaking bad in the germinal center: how deregulation of BCL6 contributes to lymphomagenesis</span></div><div class="casAuthors">Hatzi, Katerina; Melnick, Ari</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Molecular Medicine</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">343-352</span>CODEN:
                <span class="NLM_cas:coden">TMMRCY</span>;
        ISSN:<span class="NLM_cas:issn">1471-4914</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The B cell lymphoma 6 (BCL6) transcriptional repressor is a master regulator of the germinal center (GC) B cell program, required for their unique proliferative and stress tolerant phenotype.  Most B cell lymphomas arise from GC B cells and are dependent on the continued or deregulated expression of BCL6 to maintain their survival.  The actions of BCL6 in B cells involve formation of distinct chromatin modifying complexes that silence specific promoter and enhancer networks, resp.  The same biochem. mechanisms are maintained in malignant lymphoma cells.  Targeted inhibition of these BCL6 functions has emerged as the basis for rational design of lymphoma therapies and combinatorial regimens.  In this review, we summarize recent advances on BCL6 mechanisms of action and the deregulation of its target gene networks in lymphoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrroimnIsIZJ7Vg90H21EOLACvtfcHk0lhJlzp846lkeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltl2htrc%253D&md5=bbc52827778c0995ea400f814c66cfe4</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.molmed.2014.03.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molmed.2014.03.001%26sid%3Dliteratum%253Aachs%26aulast%3DHatzi%26aufirst%3DK.%26aulast%3DMelnick%26aufirst%3DA.%26atitle%3DBreaking%2520bad%2520in%2520the%2520germinal%2520center%253A%2520how%2520deregulation%2520of%2520BCL6%2520contributes%2520to%2520lymphomagenesis%26jtitle%3DTrends%2520Mol.%2520Med.%26date%3D2014%26volume%3D20%26issue%3D6%26spage%3D343%26epage%3D352%26doi%3D10.1016%2Fj.molmed.2014.03.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Basso, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalla-Favera, R.</span></span> <span> </span><span class="NLM_article-title">Roles of BCL6 in normal and transformed germinal center B cells</span>. <i>Immunological Reviews</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>247</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">172</span>– <span class="NLM_lpage">183</span>, <span class="refDoi"> DOI: 10.1111/j.1600-065X.2012.01112.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10.1111%2Fj.1600-065X.2012.01112.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=22500840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVaks7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=247&publication_year=2012&pages=172-183&issue=1&author=K.+Bassoauthor=R.+Dalla-Favera&title=Roles+of+BCL6+in+normal+and+transformed+germinal+center+B+cells&doi=10.1111%2Fj.1600-065X.2012.01112.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Roles of BCL6 in normal and transformed germinal center B cells</span></div><div class="casAuthors">Basso, Katia; Dalla-Favera, Riccardo</div><div class="citationInfo"><span class="NLM_cas:title">Immunological Reviews</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">247</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">172-183, 12 pp.</span>CODEN:
                <span class="NLM_cas:coden">IMRED2</span>;
        ISSN:<span class="NLM_cas:issn">1600-065X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  BCL6 is a transcriptional repressor required in mature B cells during the germinal center (GC) reaction.  Multiple mechanisms act coordinately to timely modulate BCL6 expression at transcriptional and post-transcriptional levels.  BCL6 prevents premature activation and differentiation of GC B cells and provides an environment tolerant of the DNA breaks assocd. with Ig gene remodeling mechanisms involved in the prodn. of high-affinity antibodies of different isotypes.  The crit. functions exerted by BCL6 during normal B-cell development can be hijacked by the malignant transformation process.  Indeed, BCL6 is targeted by genetic aberrations and acts as an oncogene in GC-derived lymphomas.  The aberrations affecting BCL6 interfere with the multiple levels of regulation that grant a fine tuning of BCL6 expression and activity in physiol. conditions.  This review summarizes the current knowledge on BCL6 function and its role in lymphomagenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXfjT3UG13QrVg90H21EOLACvtfcHk0lhJlzp846lkeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVaks7jO&md5=6f923c689f545b55faf1116e87f6afec</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1111%2Fj.1600-065X.2012.01112.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1600-065X.2012.01112.x%26sid%3Dliteratum%253Aachs%26aulast%3DBasso%26aufirst%3DK.%26aulast%3DDalla-Favera%26aufirst%3DR.%26atitle%3DRoles%2520of%2520BCL6%2520in%2520normal%2520and%2520transformed%2520germinal%2520center%2520B%2520cells%26jtitle%3DImmunological%2520Reviews%26date%3D2012%26volume%3D247%26issue%3D1%26spage%3D172%26epage%3D183%26doi%3D10.1111%2Fj.1600-065X.2012.01112.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jardin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruminy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bastard, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tilly, H.</span></span> <span> </span><span class="NLM_article-title">The BCL6 proto-oncogene: a leading role during germinal center development and lymphomagenesis</span>. <i>Pathol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>55</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">73</span>– <span class="NLM_lpage">83</span>, <span class="refDoi"> DOI: 10.1016/j.patbio.2006.04.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10.1016%2Fj.patbio.2006.04.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=16815642" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhs1Sntbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2007&pages=73-83&issue=1&author=F.+Jardinauthor=P.+Ruminyauthor=C.+Bastardauthor=H.+Tilly&title=The+BCL6+proto-oncogene%3A+a+leading+role+during+germinal+center+development+and+lymphomagenesis&doi=10.1016%2Fj.patbio.2006.04.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The BCL6 proto-oncogene: a leading role during germinal center development and lymphomagenesis</span></div><div class="casAuthors">Jardin, Fabrice; Ruminy, Philippe; Bastard, Christian; Tilly, Herve</div><div class="citationInfo"><span class="NLM_cas:title">Pathologie Biologie</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">73-83</span>CODEN:
                <span class="NLM_cas:coden">PTBIAN</span>;
        ISSN:<span class="NLM_cas:issn">0369-8114</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  The BCL6 proto-oncogene encodes a nuclear transcriptional repressor, with pivotal roles in germinal center (GC) formation and regulation of lymphocyte function, differentiation, and survival.  BCL6 suppresses p53 in GCB-cells and its constitutive expression can protect B-cell lines from apoptosis induced by DNA damage.  BCL6-mediated expression may allow GCB-cells to sustain the low levels of physiol. DNA breaks related to somatic mutation (SM) and Ig class switch recombination which physiol. occur in GCB-cells.  Three types of genetic events occur in the BCL6 locus and involve invariably the 5' non-coding region and include translocations, deletions and SM actively targeted to the 5' untranslated region.  These acquired mutations occur independently of translocations but may be involved in the deregulation of the gene and/or translocation mechanisms.  The favorable prognostic value of high levels of BCL6 gene expression in NHL seems well-established.  By contrast, the relevance of SM or translocation of the gene remains unclear.  However, it is likely that non-Hodgkin's lymphomas (NHL) harboring the most frequent translocation involving BCL6, i.e. t(3;14), are characterized by a common cell of origin and similar oncogenic mechanisms.  Several expts. and mouse models mimicking BCL6 translocation occurring in human lymphoma have demonstrated the oncogenic role of BCL6 and constitute a rational to consider BCL6 as a new therapeutic target in NHL.  BCL6 blockade can be achieved by different strategies which include siRNA, interference by specific peptides or regulation of BCL6 acetylation by pharmacol. agents such as SAHA or niacinamide and would be applicable to most type of B-cell NHL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqDg4nsDWOqLVg90H21EOLACvtfcHk0lhJlzp846lkeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhs1Sntbc%253D&md5=eaa157769f2ba4d2f91a729d02e12f66</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.patbio.2006.04.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.patbio.2006.04.001%26sid%3Dliteratum%253Aachs%26aulast%3DJardin%26aufirst%3DF.%26aulast%3DRuminy%26aufirst%3DP.%26aulast%3DBastard%26aufirst%3DC.%26aulast%3DTilly%26aufirst%3DH.%26atitle%3DThe%2520BCL6%2520proto-oncogene%253A%2520a%2520leading%2520role%2520during%2520germinal%2520center%2520development%2520and%2520lymphomagenesis%26jtitle%3DPathol.%2520Biol.%26date%3D2007%26volume%3D55%26issue%3D1%26spage%3D73%26epage%3D83%26doi%3D10.1016%2Fj.patbio.2006.04.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hanahan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinberg, R. A.</span></span> <span> </span><span class="NLM_article-title">The Hallmarks of Cancer</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>100</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">57</span>– <span class="NLM_lpage">70</span>, <span class="refDoi"> DOI: 10.1016/S0092-8674(00)81683-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10.1016%2FS0092-8674%2800%2981683-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10647931" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BD3cXks1CktA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2000&pages=57-70&issue=1&author=D.+Hanahanauthor=R.+A.+Weinberg&title=The+Hallmarks+of+Cancer&doi=10.1016%2FS0092-8674%2800%2981683-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The hallmarks of cancer</span></div><div class="casAuthors">Hanahan, Douglas; Weinberg, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, Massachusetts)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">57-70</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review is given with many refs. on principles governing the transformation of normal human cells into malignant cancers.  The authors suggest that research over the past decades has revealed a small no. of mol., biochem., and cellular traits (acquired capabilities) shared by most and perhaps all types of cancer.  Topics included are the acquired capabilities self-sufficiency in growth signals, insensitivity to antigrowth signals, evading apoptosis, limitless replicative potential, sustained angiogenesis, and tissue invasion and metastasis followed by genome instability as an enabling characteristic and alternative pathways to cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooXmGqKjM1C7Vg90H21EOLACvtfcHk0lgrOTzOt-fRwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXks1CktA%253D%253D&md5=1d23c4400dad34c7f32d4a0c41890349</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2800%2981683-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252800%252981683-9%26sid%3Dliteratum%253Aachs%26aulast%3DHanahan%26aufirst%3DD.%26aulast%3DWeinberg%26aufirst%3DR.%2BA.%26atitle%3DThe%2520Hallmarks%2520of%2520Cancer%26jtitle%3DCell%26date%3D2000%26volume%3D100%26issue%3D1%26spage%3D57%26epage%3D70%26doi%3D10.1016%2FS0092-8674%2800%2981683-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hanahan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinberg, R. A.</span></span> <span> </span><span class="NLM_article-title">Hallmarks of Cancer: The Next Generation</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>144</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">646</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2011.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10.1016%2Fj.cell.2011.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=21376230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjsFeqtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=144&publication_year=2011&pages=646-674&issue=5&author=D.+Hanahanauthor=R.+A.+Weinberg&title=Hallmarks+of+Cancer%3A+The+Next+Generation&doi=10.1016%2Fj.cell.2011.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Hallmarks of cancer: the next generation</span></div><div class="casAuthors">Hanahan, Douglas; Weinberg, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">144</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">646-674</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The hallmarks of cancer comprise six biol. capabilities acquired during the multistep development of human tumors.  The hallmarks constitute an organizing principle for rationalizing the complexities of neoplastic disease.  They include sustaining proliferative signaling, evading growth suppressors, resisting cell death, enabling replicative immortality, inducing angiogenesis, and activating invasion and metastasis.  Underlying these hallmarks are genome instability, which generates the genetic diversity that expedites their acquisition, and inflammation, which fosters multiple hallmark functions.  Conceptual progress in the last decade has added two emerging hallmarks of potential generality to this list-reprogramming of energy metab. and evading immune destruction.  In addn. to cancer cells, tumors exhibit another dimension of complexity: they contain a repertoire of recruited, ostensibly normal cells that contribute to the acquisition of hallmark traits by creating the "tumor microenvironment.".  Recognition of the widespread applicability of these concepts will increasingly affect the development of new means to treat human cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTcpqXALcb6LVg90H21EOLACvtfcHk0lgrOTzOt-fRwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjsFeqtrk%253D&md5=b36f160c417a712e83cb83c577f0018e</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DHanahan%26aufirst%3DD.%26aulast%3DWeinberg%26aufirst%3DR.%2BA.%26atitle%3DHallmarks%2520of%2520Cancer%253A%2520The%2520Next%2520Generation%26jtitle%3DCell%26date%3D2011%26volume%3D144%26issue%3D5%26spage%3D646%26epage%3D674%26doi%3D10.1016%2Fj.cell.2011.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Polo, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dell’Oso, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranuncolo, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerchietti, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Da Silva, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prive, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Licht, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnick, A.</span></span> <span> </span><span class="NLM_article-title">Specific peptide interference reveals BCL6 transcriptional and oncogenic mechanisms in B-cell lymphoma cells</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">1329</span>– <span class="NLM_lpage">1335</span>, <span class="refDoi"> DOI: 10.1038/nm1134</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10.1038%2Fnm1134" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=15531890" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVCisbfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2004&pages=1329-1335&issue=12&author=J.+M.+Poloauthor=T.+Dell%E2%80%99Osoauthor=S.+M.+Ranuncoloauthor=L.+Cerchiettiauthor=D.+Beckauthor=G.+F.+Da+Silvaauthor=G.+G.+Priveauthor=J.+D.+Lichtauthor=A.+Melnick&title=Specific+peptide+interference+reveals+BCL6+transcriptional+and+oncogenic+mechanisms+in+B-cell+lymphoma+cells&doi=10.1038%2Fnm1134"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Specific peptide interference reveals BCL6 transcriptional and oncogenic mechanisms in B-cell lymphoma cells</span></div><div class="casAuthors">Polo, Jose M.; Dell'Oso, Tania; Ranuncolo, Stella Maris; Cerchietti, Leandro; Beck, David; Da Silva, Gustavo F.; Prive, Gilbert G.; Licht, Jonathan D.; Melnick, Ari</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1329-1335</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The BTB/POZ transcriptional repressor and candidate oncogene BCL6 is frequently misregulated in B-cell lymphomas.  The interface through which the BCL6 BTB domain mediates recruitment of the SMRT, NCoR and BCoR corepressors was recently identified.  To det. the contribution of this interface to BCL6 transcriptional and biol. properties, the authors generated a peptide that specifically binds BCL6 and blocks corepressor recruitment in vivo.  This inhibitor disrupts BCL6-mediated repression and establishment of silenced chromatin, reactivates natural BCL6 target genes, and abrogates BCL6 biol. function in B cells.  In BCL6-pos. lymphoma cells, peptide blockade caused apoptosis and cell cycle arrest.  BTB domain peptide inhibitors may constitute a novel therapeutic agent for B-cell lymphomas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAPaMuV1BGLrVg90H21EOLACvtfcHk0lgrOTzOt-fRwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVCisbfJ&md5=21f9d4b802d65897d666af1ac77a083c</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnm1134&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm1134%26sid%3Dliteratum%253Aachs%26aulast%3DPolo%26aufirst%3DJ.%2BM.%26aulast%3DDell%25E2%2580%2599Oso%26aufirst%3DT.%26aulast%3DRanuncolo%26aufirst%3DS.%2BM.%26aulast%3DCerchietti%26aufirst%3DL.%26aulast%3DBeck%26aufirst%3DD.%26aulast%3DDa%2BSilva%26aufirst%3DG.%2BF.%26aulast%3DPrive%26aufirst%3DG.%2BG.%26aulast%3DLicht%26aufirst%3DJ.%2BD.%26aulast%3DMelnick%26aufirst%3DA.%26atitle%3DSpecific%2520peptide%2520interference%2520reveals%2520BCL6%2520transcriptional%2520and%2520oncogenic%2520mechanisms%2520in%2520B-cell%2520lymphoma%2520cells%26jtitle%3DNat.%2520Med.%26date%3D2004%26volume%3D10%26issue%3D12%26spage%3D1329%26epage%3D1335%26doi%3D10.1038%2Fnm1134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghetu, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corcoran, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerchietti, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bardwell, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Privé, G. G.</span></span> <span> </span><span class="NLM_article-title">Structure of a BCOR corepressor peptide in complex with the BCL6 BTB domain dimer</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>29</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">384</span>– <span class="NLM_lpage">391</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2007.12.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10.1016%2Fj.molcel.2007.12.026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=18280243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BD1cXisFSns7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2008&pages=384-391&issue=3&author=A.+F.+Ghetuauthor=C.+M.+Corcoranauthor=L.+Cerchiettiauthor=V.+J.+Bardwellauthor=A.+Melnickauthor=G.+G.+Priv%C3%A9&title=Structure+of+a+BCOR+corepressor+peptide+in+complex+with+the+BCL6+BTB+domain+dimer&doi=10.1016%2Fj.molcel.2007.12.026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of a BCOR corepressor peptide in complex with the BCL6 BTB domain dimer</span></div><div class="casAuthors">Ghetu, Alexandru F.; Corcoran, Connie M.; Cerchietti, Leandro; Bardwell, Vivian J.; Melnick, Ari; Prive, Gilbert G.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">384-391</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The transcriptional corepressors BCOR, SMRT, and NCoR are known to bind competitively to the BCL6 BTB domain despite the fact that BCOR has no detectable sequence similarity to the other two corepressors.  We have identified a 17 residue motif from BCOR that binds directly to the BCL6 BTB domain and detd. the crystal structure of the complex to a resoln. of 2.6 Å.  Remarkably, the BCOR BCL6 binding domain (BCORBBD) peptide binds in the same BCL6 binding site as the SMRTBBD peptide despite the lack of any significant sequence similarity between the two peptides.  Mutations of crit. BCORBBD residues cause the disruption of the BCL6 corepression activities of BCOR, and a BCORBBD peptide blocks BCL6-mediated transcriptional repression and kills lymphoma cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-FAcEvWcMKbVg90H21EOLACvtfcHk0ljxvQqAbJdZbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXisFSns7c%253D&md5=f064d9c9e3c157a960eb0bda26fcdece</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2007.12.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2007.12.026%26sid%3Dliteratum%253Aachs%26aulast%3DGhetu%26aufirst%3DA.%2BF.%26aulast%3DCorcoran%26aufirst%3DC.%2BM.%26aulast%3DCerchietti%26aufirst%3DL.%26aulast%3DBardwell%26aufirst%3DV.%2BJ.%26aulast%3DMelnick%26aufirst%3DA.%26aulast%3DPriv%25C3%25A9%26aufirst%3DG.%2BG.%26atitle%3DStructure%2520of%2520a%2520BCOR%2520corepressor%2520peptide%2520in%2520complex%2520with%2520the%2520BCL6%2520BTB%2520domain%2520dimer%26jtitle%3DMol.%2520Cell%26date%3D2008%26volume%3D29%26issue%3D3%26spage%3D384%26epage%3D391%26doi%3D10.1016%2Fj.molcel.2007.12.026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cerchietti, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghetu, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Da Silva, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunting, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polo, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farès, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coop, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKerell, A. D.,  Jr</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Privé, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnick, A.</span></span> <span> </span><span class="NLM_article-title">A small-molecule inhibitor of BCL6 kills DLBCL cells in vitro and in vivo</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>17</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">400</span>– <span class="NLM_lpage">411</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2009.12.050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10.1016%2Fj.ccr.2009.12.050" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=20385364" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmsF2msb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=400-411&issue=4&author=L.+C.+Cerchiettiauthor=A.+F.+Ghetuauthor=X.+Zhuauthor=G.+F.+Da+Silvaauthor=S.+Zhongauthor=M.+Matthewsauthor=K.+L.+Buntingauthor=J.+M.+Poloauthor=C.+Far%C3%A8sauthor=C.+H.+Arrowsmithauthor=S.+N.+Yangauthor=M.+Garciaauthor=A.+Coopauthor=A.+D.+MacKerellauthor=G.+G.+Priv%C3%A9author=A.+Melnick&title=A+small-molecule+inhibitor+of+BCL6+kills+DLBCL+cells+in+vitro+and+in+vivo&doi=10.1016%2Fj.ccr.2009.12.050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">A small-molecule inhibitor of BCL6 kills DLBCL cells in vitro and in vivo</span></div><div class="casAuthors">Cerchietti, Leandro C.; Ghetu, Alexandru F.; Zhu, Xiao; Da Silva, Gustavo F.; Zhong, Shijun; Matthews, Marilyn; Bunting, Karen L.; Polo, Jose M.; Fares, Christophe; Arrowsmith, Cheryl H.; Yang, Shao Ning; Garcia, Monica; Coop, Andrew; MacKerell, Alexander D., Jr.; Prive, Gilbert G.; Melnick, Ari</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">400-411</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The BCL6 transcriptional repressor is the most frequently involved oncogene in diffuse large B cell lymphoma (DLBCL).  We combined computer-aided drug design with functional assays to identify low-mol.-wt. compds. that bind to the corepressor binding groove of the BCL6 BTB domain.  One such compd. disrupted BCL6/corepressor complexes in vitro and in vivo, and was obsd. by X-ray crystallog. and NMR to bind the crit. site within the BTB groove.  This compd. could induce expression of BCL6 target genes and kill BCL6-pos. DLBCL cell lines.  In xenotransplantation expts., the compd. was nontoxic and potently suppressed DLBCL tumors in vivo.  The compd. also killed primary DLBCLs from human patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdcScjNPjhiLVg90H21EOLACvtfcHk0ljxvQqAbJdZbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmsF2msb8%253D&md5=64f30a813c26394b69da6de95991f8dd</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2009.12.050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2009.12.050%26sid%3Dliteratum%253Aachs%26aulast%3DCerchietti%26aufirst%3DL.%2BC.%26aulast%3DGhetu%26aufirst%3DA.%2BF.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DDa%2BSilva%26aufirst%3DG.%2BF.%26aulast%3DZhong%26aufirst%3DS.%26aulast%3DMatthews%26aufirst%3DM.%26aulast%3DBunting%26aufirst%3DK.%2BL.%26aulast%3DPolo%26aufirst%3DJ.%2BM.%26aulast%3DFar%25C3%25A8s%26aufirst%3DC.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DYang%26aufirst%3DS.%2BN.%26aulast%3DGarcia%26aufirst%3DM.%26aulast%3DCoop%26aufirst%3DA.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26aulast%3DPriv%25C3%25A9%26aufirst%3DG.%2BG.%26aulast%3DMelnick%26aufirst%3DA.%26atitle%3DA%2520small-molecule%2520inhibitor%2520of%2520BCL6%2520kills%2520DLBCL%2520cells%2520in%2520vitro%2520and%2520in%2520vivo%26jtitle%3DCancer%2520Cell%26date%3D2010%26volume%3D17%26issue%3D4%26spage%3D400%26epage%3D411%26doi%3D10.1016%2Fj.ccr.2009.12.050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cerchietti, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaknovich, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatzi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polo, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chadburn, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowdy, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnick, A.</span></span> <span> </span><span class="NLM_article-title">A peptomimetic inhibitor of BCL6 with potent antilymphoma effects in vitro and in vivo</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>113</i></span> (<span class="NLM_issue">15</span>),  <span class="NLM_fpage">3397</span>– <span class="NLM_lpage">3405</span>, <span class="refDoi"> DOI: 10.1182/blood-2008-07-168773</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10.1182%2Fblood-2008-07-168773" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=18927431" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BD1MXkslCkurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2009&pages=3397-3405&issue=15&author=L.+C.+Cerchiettiauthor=S.+N.+Yangauthor=R.+Shaknovichauthor=K.+Hatziauthor=J.+M.+Poloauthor=A.+Chadburnauthor=S.+F.+Dowdyauthor=A.+Melnick&title=A+peptomimetic+inhibitor+of+BCL6+with+potent+antilymphoma+effects+in+vitro+and+in+vivo&doi=10.1182%2Fblood-2008-07-168773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">A peptomimetic inhibitor of BCL6 with potent antilymphoma effects in vitro and in vivo</span></div><div class="casAuthors">Cerchietti, Leandro C.; Yang, Shao Ning; Shaknovich, Rita; Hatzi, Katerina; Polo, Jose M.; Chadburn, Amy; Dowdy, Steven F.; Melnick, Ari</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3397-3405</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">The BCL6 transcriptional repressor is the most commonly involved oncogene in diffuse large B-cell lymphomas (DLBCLs).  BCL6 lymphomagenic activity is dependent on its ability to recruit corepressor proteins to a unique binding site on its N-terminal BTB domain.  A recombinant peptide fragment of the SMRT (silencing mediator for retinoid and thyroid hormone receptor) corepressor that blocks this site can inhibit BCL6 biol. functions.  Shortening and conversion of this peptide to D-amino acid and retro configuration as well as the addn. of a fusogenic motif yielded a far more potent and stable BCL6 inhibitor that still retained the specificity of the original SMRT fragment.  Like the L-peptide, retroinverso BCL6 peptide inhibitor (RI-BPI) selectively killed BCR rather than OxPhos-type DLBCL cells.  The RI-BPI could recapitulate the failure to form germinal centers seen in BCL6 null mice yet was nontoxic and non-immunogenic even when administered for up to 52 wk.  RI-BPI showed superior duration of tissue penetration and could accordingly powerfully suppress the growth of human DLBCLs xenografts in a dose-dependent manner.  Finally, RI-BPI could kill primary human DLBCL cells but had no effect on normal lymphoid tissue or other tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxsckch1D41rVg90H21EOLACvtfcHk0ljxvQqAbJdZbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXkslCkurw%253D&md5=8a12df08b153ba0712b5d06f9bc550c2</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1182%2Fblood-2008-07-168773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2008-07-168773%26sid%3Dliteratum%253Aachs%26aulast%3DCerchietti%26aufirst%3DL.%2BC.%26aulast%3DYang%26aufirst%3DS.%2BN.%26aulast%3DShaknovich%26aufirst%3DR.%26aulast%3DHatzi%26aufirst%3DK.%26aulast%3DPolo%26aufirst%3DJ.%2BM.%26aulast%3DChadburn%26aufirst%3DA.%26aulast%3DDowdy%26aufirst%3DS.%2BF.%26aulast%3DMelnick%26aufirst%3DA.%26atitle%3DA%2520peptomimetic%2520inhibitor%2520of%2520BCL6%2520with%2520potent%2520antilymphoma%2520effects%2520in%2520vitro%2520and%2520in%2520vivo%26jtitle%3DBlood%26date%3D2009%26volume%3D113%26issue%3D15%26spage%3D3397%26epage%3D3405%26doi%3D10.1182%2Fblood-2008-07-168773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cardenas, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beguelin, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teater, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldstein, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oswald, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatzi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.-N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaknovich, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanommeslaeghe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abbott, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tam, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonard, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elemento, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerchietti, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKerell, A. D.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnick, A. M.</span></span> <span> </span><span class="NLM_article-title">Rationally designed BCL6 inhibitors target activated B cell diffuse large B cell lymphoma</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>126</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">3351</span>– <span class="NLM_lpage">3362</span>, <span class="refDoi"> DOI: 10.1172/JCI85795</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10.1172%2FJCI85795" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=27482887" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A280%3ADC%252BC2s3mvFagsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2016&pages=3351-3362&issue=9&author=M.+G.+Cardenasauthor=W.+Yuauthor=W.+Beguelinauthor=M.+R.+Teaterauthor=H.+Gengauthor=R.+L.+Goldsteinauthor=E.+Oswaldauthor=K.+Hatziauthor=S.-N.+Yangauthor=J.+Cohenauthor=R.+Shaknovichauthor=K.+Vanommeslaegheauthor=H.+Chengauthor=D.+Liangauthor=H.+J.+Choauthor=J.+Abbottauthor=W.+Tamauthor=W.+Duauthor=J.+P.+Leonardauthor=O.+Elementoauthor=L.+Cerchiettiauthor=T.+Cierpickiauthor=F.+Xueauthor=A.+D.+MacKerellauthor=A.+M.+Melnick&title=Rationally+designed+BCL6+inhibitors+target+activated+B+cell+diffuse+large+B+cell+lymphoma&doi=10.1172%2FJCI85795"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Rationally designed BCL6 inhibitors target activated B cell diffuse large B cell lymphoma</span></div><div class="casAuthors">Cardenas Mariano G; Yu Wenbo; Beguelin Wendy; Teater Matthew R; Geng Huimin; Goldstein Rebecca L; Oswald Erin; Hatzi Katerina; Yang Shao-Ning; Cohen Joanna; Shaknovich Rita; Vanommeslaeghe Kenno; Cheng Huimin; Liang Dongdong; Cho Hyo Je; Abbott Joshua; Tam Wayne; Du Wei; Leonard John P; Elemento Olivier; Cerchietti Leandro; Cierpicki Tomasz; Xue Fengtian; MacKerell Alexander D Jr; Melnick Ari M</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of clinical investigation</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3351-62</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Diffuse large B cell lymphomas (DLBCLs) arise from proliferating B cells transiting different stages of the germinal center reaction.  In activated B cell DLBCLs (ABC-DLBCLs), a class of DLBCLs that respond poorly to current therapies, chromosomal translocations and amplification lead to constitutive expression of the B cell lymphoma 6 (BCL6) oncogene.  The role of BCL6 in maintaining these lymphomas has not been investigated.  Here, we designed small-molecule inhibitors that display higher affinity for BCL6 than its endogenous corepressor ligands to evaluate their therapeutic efficacy for targeting ABC-DLBCL.  We used an in silico drug design functional-group mapping approach called SILCS to create a specific BCL6 inhibitor called FX1 that has 10-fold greater potency than endogenous corepressors and binds an essential region of the BCL6 lateral groove.  FX1 disrupted formation of the BCL6 repression complex, reactivated BCL6 target genes, and mimicked the phenotype of mice engineered to express BCL6 with corepressor binding site mutations.  Low doses of FX1 induced regression of established tumors in mice bearing DLBCL xenografts.  Furthermore, FX1 suppressed ABC-DLBCL cells in vitro and in vivo, as well as primary human ABC-DLBCL specimens ex vivo.  These findings indicate that ABC-DLBCL is a BCL6-dependent disease that can be targeted by rationally designed inhibitors that exceed the binding affinity of natural BCL6 ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRAVUzYiXt5ZL5XHngOXiV_fW6udTcc2eb864SDHksdeLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s3mvFagsw%253D%253D&md5=90a313fd7adce1922773212f0960bad7</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1172%2FJCI85795&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI85795%26sid%3Dliteratum%253Aachs%26aulast%3DCardenas%26aufirst%3DM.%2BG.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DBeguelin%26aufirst%3DW.%26aulast%3DTeater%26aufirst%3DM.%2BR.%26aulast%3DGeng%26aufirst%3DH.%26aulast%3DGoldstein%26aufirst%3DR.%2BL.%26aulast%3DOswald%26aufirst%3DE.%26aulast%3DHatzi%26aufirst%3DK.%26aulast%3DYang%26aufirst%3DS.-N.%26aulast%3DCohen%26aufirst%3DJ.%26aulast%3DShaknovich%26aufirst%3DR.%26aulast%3DVanommeslaeghe%26aufirst%3DK.%26aulast%3DCheng%26aufirst%3DH.%26aulast%3DLiang%26aufirst%3DD.%26aulast%3DCho%26aufirst%3DH.%2BJ.%26aulast%3DAbbott%26aufirst%3DJ.%26aulast%3DTam%26aufirst%3DW.%26aulast%3DDu%26aufirst%3DW.%26aulast%3DLeonard%26aufirst%3DJ.%2BP.%26aulast%3DElemento%26aufirst%3DO.%26aulast%3DCerchietti%26aufirst%3DL.%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DXue%26aufirst%3DF.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26aulast%3DMelnick%26aufirst%3DA.%2BM.%26atitle%3DRationally%2520designed%2520BCL6%2520inhibitors%2520target%2520activated%2520B%2520cell%2520diffuse%2520large%2520B%2520cell%2520lymphoma%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2016%26volume%3D126%26issue%3D9%26spage%3D3351%26epage%3D3362%26doi%3D10.1172%2FJCI85795" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kamada, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakai, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sogabe, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ida, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snell, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iida, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imaeda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsui, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a B-cell lymphoma 6 protein–protein interaction inhibitor by a biophysics-driven fragment-based approach</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">4358</span>– <span class="NLM_lpage">4368</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00313</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00313" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BC2sXntFSkt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4358-4368&issue=10&author=Y.+Kamadaauthor=N.+Sakaiauthor=S.+Sogabeauthor=K.+Idaauthor=H.+Okiauthor=K.+Sakamotoauthor=W.+Laneauthor=G.+Snellauthor=M.+Iidaauthor=Y.+Imaedaauthor=J.+Sakamotoauthor=J.+Matsui&title=Discovery+of+a+B-cell+lymphoma+6+protein%E2%80%93protein+interaction+inhibitor+by+a+biophysics-driven+fragment-based+approach&doi=10.1021%2Facs.jmedchem.7b00313"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a B-Cell Lymphoma 6 Protein-Protein Interaction Inhibitor by a Biophysics-Driven Fragment-Based Approach</span></div><div class="casAuthors">Kamada, Yusuke; Sakai, Nozomu; Sogabe, Satoshi; Ida, Koh; Oki, Hideyuki; Sakamoto, Kotaro; Lane, Weston; Snell, Gyorgy; Iida, Motoo; Imaeda, Yasuhiro; Sakamoto, Junichi; Matsui, Junji</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4358-4368</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">B-cell lymphoma 6 (BCL6) is a transcriptional factor that expresses in lymphocytes and regulates the differentiation and proliferation of lymphocytes.  Therefore, BCL6 is a therapeutic target for autoimmune diseases and cancer treatment.  This report presents the discovery of BCL6-corepressor interaction inhibitors by using a biophysics-driven fragment-based approach.  Using the surface plasmon resonance (SPR)-based fragment screening, we successfully identified fragment 1 (SPR KD = 1200 μM, ligand efficiency (LE) = 0.28), a competitive binder to the natural ligand BCoR peptide.  Moreover, we elaborated 1 into the more potent compd. 7 (SPR KD = 0.078 μM, LE = 0.37, cell-free protein-protein interaction (PPI) IC50 = 0.48 μM (ELISA), cellular PPI IC50 = 8.6 μM (M2H)) by a structure-based design and structural integration with a second high-throughput screening hit.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0yLaTbq3FqbVg90H21EOLACvtfcHk0ljrWlV2miRbKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXntFSkt7o%253D&md5=c1c6a4e57ec5655ed2c937f5274ea6ec</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00313&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00313%26sid%3Dliteratum%253Aachs%26aulast%3DKamada%26aufirst%3DY.%26aulast%3DSakai%26aufirst%3DN.%26aulast%3DSogabe%26aufirst%3DS.%26aulast%3DIda%26aufirst%3DK.%26aulast%3DOki%26aufirst%3DH.%26aulast%3DSakamoto%26aufirst%3DK.%26aulast%3DLane%26aufirst%3DW.%26aulast%3DSnell%26aufirst%3DG.%26aulast%3DIida%26aufirst%3DM.%26aulast%3DImaeda%26aufirst%3DY.%26aulast%3DSakamoto%26aufirst%3DJ.%26aulast%3DMatsui%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%2520B-cell%2520lymphoma%25206%2520protein%25E2%2580%2593protein%2520interaction%2520inhibitor%2520by%2520a%2520biophysics-driven%2520fragment-based%2520approach%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26issue%3D10%26spage%3D4358%26epage%3D4368%26doi%3D10.1021%2Facs.jmedchem.7b00313" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yasui, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakai, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asano, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshitomi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takagi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sogabe, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamada, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snell, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwata, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inooka, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, J.-i.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakada, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imaeda, Y.</span></span> <span> </span><span class="NLM_article-title">Discovery of a novel B-cell lymphoma 6 (BCL6)–corepressor interaction inhibitor by utilizing structure-based drug design</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>25</i></span> (<span class="NLM_issue">17</span>),  <span class="NLM_fpage">4876</span>– <span class="NLM_lpage">4886</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2017.07.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10.1016%2Fj.bmc.2017.07.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=28760529" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1GisbvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=4876-4886&issue=17&author=T.+Yasuiauthor=T.+Yamamotoauthor=N.+Sakaiauthor=K.+Asanoauthor=T.+Takaiauthor=Y.+Yoshitomiauthor=M.+Davisauthor=T.+Takagiauthor=K.+Sakamotoauthor=S.+Sogabeauthor=Y.+Kamadaauthor=W.+Laneauthor=G.+Snellauthor=M.+Iwataauthor=M.+Gotoauthor=H.+Inookaauthor=J.-i.+Sakamotoauthor=Y.+Nakadaauthor=Y.+Imaeda&title=Discovery+of+a+novel+B-cell+lymphoma+6+%28BCL6%29%E2%80%93corepressor+interaction+inhibitor+by+utilizing+structure-based+drug+design&doi=10.1016%2Fj.bmc.2017.07.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a novel B-cell lymphoma 6 (BCL6)-corepressor interaction inhibitor by utilizing structure-based drug design</span></div><div class="casAuthors">Yasui, Takeshi; Yamamoto, Takeshi; Sakai, Nozomu; Asano, Kouhei; Takai, Takafumi; Yoshitomi, Yayoi; Davis, Melinda; Takagi, Terufumi; Sakamoto, Kotaro; Sogabe, Satoshi; Kamada, Yusuke; Lane, Weston; Snell, Gyorgy; Iwata, Masashi; Goto, Masayuki; Inooka, Hiroshi; Sakamoto, Jun-ichi; Nakada, Yoshihisa; Imaeda, Yasuhiro</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4876-4886</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">B-cell lymphoma 6 (BCL6) is a transcriptional repressor that can form complexes with corepressors via protein-protein interactions (PPIs).  The complexes of BCL6 and corepressors play an important role in the formation of germinal centers (GCs), and differentiation and proliferation of lymphocytes.  Therefore, BCL6-corepressor interaction inhibitors would be drug candidates for managing autoimmune diseases and cancer.  Starting from high-throughput screening hits I and II, the authors identified a novel BCL6-corepressor interaction inhibitor III (cell-free ELISA [ELISA] IC50 = 0.10 μM, cell-based mammalian two-hybrid [M2H] assay IC50 = 0.72 μM) by utilizing structure-based drug design (SBDD) based on an x-ray crystal structure of I bound to BCL6.  Compd. III also showed a good pharmacokinetic profile, which was acceptable for both in vitro and in vivo studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorDO1ByXavNrVg90H21EOLACvtfcHk0lh18KxtRRbEsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1GisbvF&md5=2a9cae7f5e2ec480bd5ae106cc4cba31</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2017.07.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2017.07.037%26sid%3Dliteratum%253Aachs%26aulast%3DYasui%26aufirst%3DT.%26aulast%3DYamamoto%26aufirst%3DT.%26aulast%3DSakai%26aufirst%3DN.%26aulast%3DAsano%26aufirst%3DK.%26aulast%3DTakai%26aufirst%3DT.%26aulast%3DYoshitomi%26aufirst%3DY.%26aulast%3DDavis%26aufirst%3DM.%26aulast%3DTakagi%26aufirst%3DT.%26aulast%3DSakamoto%26aufirst%3DK.%26aulast%3DSogabe%26aufirst%3DS.%26aulast%3DKamada%26aufirst%3DY.%26aulast%3DLane%26aufirst%3DW.%26aulast%3DSnell%26aufirst%3DG.%26aulast%3DIwata%26aufirst%3DM.%26aulast%3DGoto%26aufirst%3DM.%26aulast%3DInooka%26aufirst%3DH.%26aulast%3DSakamoto%26aufirst%3DJ.-i.%26aulast%3DNakada%26aufirst%3DY.%26aulast%3DImaeda%26aufirst%3DY.%26atitle%3DDiscovery%2520of%2520a%2520novel%2520B-cell%2520lymphoma%25206%2520%2528BCL6%2529%25E2%2580%2593corepressor%2520interaction%2520inhibitor%2520by%2520utilizing%2520structure-based%2520drug%2520design%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2017%26volume%3D25%26issue%3D17%26spage%3D4876%26epage%3D4886%26doi%3D10.1016%2Fj.bmc.2017.07.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoull, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abrams, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blades, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Box, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byth, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuaqui, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carbajo, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Code, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fillery, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuller, N. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gangl, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grist, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hargreaves, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kemmitt, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connell, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prince, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raubo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rawlins, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robb, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waring, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whittaker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wylot, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span> <span> </span><span class="NLM_article-title">Discovery of pyrazolo[1,5-a]pyrimidine B-cell lymphoma 6 (BCL6) binders and optimization to high affinity macrocyclic inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">4386</span>– <span class="NLM_lpage">4402</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00359</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00359" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BC2sXntFOmsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4386-4402&issue=10&author=W.+McCoullauthor=R.+D.+Abramsauthor=E.+Andersonauthor=K.+Bladesauthor=P.+Bartonauthor=M.+Boxauthor=J.+Burgessauthor=K.+Bythauthor=Q.+Caoauthor=C.+Chuaquiauthor=R.+J.+Carbajoauthor=T.+Cheungauthor=E.+Codeauthor=A.+D.+Fergusonauthor=S.+Filleryauthor=N.+O.+Fullerauthor=E.+Ganglauthor=N.+Gaoauthor=M.+Gristauthor=D.+Hargreavesauthor=M.+R.+Howardauthor=J.+Huauthor=P.+D.+Kemmittauthor=J.+E.+Nelsonauthor=N.+O%E2%80%99Connellauthor=D.+B.+Princeauthor=P.+Rauboauthor=P.+B.+Rawlinsauthor=G.+R.+Robbauthor=J.+Shiauthor=M.+J.+Waringauthor=D.+Whittakerauthor=M.+Wylotauthor=X.+Zhu&title=Discovery+of+pyrazolo%5B1%2C5-a%5Dpyrimidine+B-cell+lymphoma+6+%28BCL6%29+binders+and+optimization+to+high+affinity+macrocyclic+inhibitors&doi=10.1021%2Facs.jmedchem.7b00359"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Pyrazolo[1,5-a]pyrimidine B-Cell Lymphoma 6 (BCL6) Binders and Optimization to High Affinity Macrocyclic Inhibitors</span></div><div class="casAuthors">McCoull, William; Abrams, Roman D.; Anderson, Erica; Blades, Kevin; Barton, Peter; Box, Matthew; Burgess, Jonathan; Byth, Kate; Cao, Qing; Chuaqui, Claudio; Carbajo, Rodrigo J.; Cheung, Tony; Code, Erin; Ferguson, Andrew D.; Fillery, Shaun; Fuller, Nathan O.; Gangl, Eric; Gao, Ning; Grist, Matthew; Hargreaves, David; Howard, Martin R.; Hu, Jun; Kemmitt, Paul D.; Nelson, Jennifer E.; O'Connell, Nichole; Prince, D. Bryan; Raubo, Piotr; Rawlins, Philip B.; Robb, Graeme R.; Shi, Junjie; Waring, Michael J.; Whittaker, David; Wylot, Marta; Zhu, Xiahui</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4386-4402</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibition of the protein-protein interaction between B-cell lymphoma 6 (BCL6) and corepressors has been implicated as a therapeutic target in diffuse large B-cell lymphoma (DLBCL) cancers and profiling of potent and selective BCL6 inhibitors are crit. to test this hypothesis.  We identified a pyrazolo[1,5-a]pyrimidine series of BCL6 binders from a fragment screen in parallel with a virtual screen.  Using structure-based drug design, binding affinity was increased 100000-fold.  This involved displacing crystallog. water, forming new ligand-protein interactions and a macrocyclization to favor the bioactive conformation of the ligands.  Optimization for slow off-rate const. kinetics was conducted as well as improving selectivity against an off-target kinase, CK2.  Potency in a cellular BCL6 assay was further optimized to afford highly selective probe mols.  Only weak antiproliferative effects were obsd. across a no. of DLBCL lines and a multiple myeloma cell line without a clear relationship to BCL6 potency.  As a result, we conclude that the BCL6 hypothesis in DLBCL cancer remains unproven.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWp2OlXAZDFLVg90H21EOLACvtfcHk0lh18KxtRRbEsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXntFOmsbo%253D&md5=fc4142453a0580edbf04802e15d4d9aa</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00359&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00359%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoull%26aufirst%3DW.%26aulast%3DAbrams%26aufirst%3DR.%2BD.%26aulast%3DAnderson%26aufirst%3DE.%26aulast%3DBlades%26aufirst%3DK.%26aulast%3DBarton%26aufirst%3DP.%26aulast%3DBox%26aufirst%3DM.%26aulast%3DBurgess%26aufirst%3DJ.%26aulast%3DByth%26aufirst%3DK.%26aulast%3DCao%26aufirst%3DQ.%26aulast%3DChuaqui%26aufirst%3DC.%26aulast%3DCarbajo%26aufirst%3DR.%2BJ.%26aulast%3DCheung%26aufirst%3DT.%26aulast%3DCode%26aufirst%3DE.%26aulast%3DFerguson%26aufirst%3DA.%2BD.%26aulast%3DFillery%26aufirst%3DS.%26aulast%3DFuller%26aufirst%3DN.%2BO.%26aulast%3DGangl%26aufirst%3DE.%26aulast%3DGao%26aufirst%3DN.%26aulast%3DGrist%26aufirst%3DM.%26aulast%3DHargreaves%26aufirst%3DD.%26aulast%3DHoward%26aufirst%3DM.%2BR.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DKemmitt%26aufirst%3DP.%2BD.%26aulast%3DNelson%26aufirst%3DJ.%2BE.%26aulast%3DO%25E2%2580%2599Connell%26aufirst%3DN.%26aulast%3DPrince%26aufirst%3DD.%2BB.%26aulast%3DRaubo%26aufirst%3DP.%26aulast%3DRawlins%26aufirst%3DP.%2BB.%26aulast%3DRobb%26aufirst%3DG.%2BR.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DWaring%26aufirst%3DM.%2BJ.%26aulast%3DWhittaker%26aufirst%3DD.%26aulast%3DWylot%26aufirst%3DM.%26aulast%3DZhu%26aufirst%3DX.%26atitle%3DDiscovery%2520of%2520pyrazolo%255B1%252C5-a%255Dpyrimidine%2520B-cell%2520lymphoma%25206%2520%2528BCL6%2529%2520binders%2520and%2520optimization%2520to%2520high%2520affinity%2520macrocyclic%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26issue%3D10%26spage%3D4386%26epage%3D4402%26doi%3D10.1021%2Facs.jmedchem.7b00359" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoull, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byth, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castaldi, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiarparin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carbajo, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Code, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davey, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fillery, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuller, N. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hargreaves, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawatkar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kemmitt, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molina, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connell, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petteruti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rasmusson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raubo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rawlins, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ricchiuto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robb, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schenone, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waring, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fawell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. M.</span></span> <span> </span><span class="NLM_article-title">Development of a novel B-cell lymphoma 6 (BCL6) PROTAC to provide insight into small molecule targeting of BCL6</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">3131</span>– <span class="NLM_lpage">3141</span>, <span class="refDoi"> DOI: 10.1021/acschembio.8b00698</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.8b00698" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFaisrnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=3131-3141&issue=11&author=W.+McCoullauthor=T.+Cheungauthor=E.+Andersonauthor=P.+Bartonauthor=J.+Burgessauthor=K.+Bythauthor=Q.+Caoauthor=M.+P.+Castaldiauthor=H.+Chenauthor=E.+Chiarparinauthor=R.+J.+Carbajoauthor=E.+Codeauthor=S.+Cowanauthor=P.+R.+Daveyauthor=A.+D.+Fergusonauthor=S.+Filleryauthor=N.+O.+Fullerauthor=N.+Gaoauthor=D.+Hargreavesauthor=M.+R.+Howardauthor=J.+Huauthor=A.+Kawatkarauthor=P.+D.+Kemmittauthor=E.+Leoauthor=D.+M.+Molinaauthor=N.+O%E2%80%99Connellauthor=P.+Petterutiauthor=T.+Rasmussonauthor=P.+Rauboauthor=P.+B.+Rawlinsauthor=P.+Ricchiutoauthor=G.+R.+Robbauthor=M.+Schenoneauthor=M.+J.+Waringauthor=M.+Zindaauthor=S.+Fawellauthor=D.+M.+Wilson&title=Development+of+a+novel+B-cell+lymphoma+6+%28BCL6%29+PROTAC+to+provide+insight+into+small+molecule+targeting+of+BCL6&doi=10.1021%2Facschembio.8b00698"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a Novel B-Cell Lymphoma 6 (BCL6) PROTAC To Provide Insight into Small Molecule Targeting of BCL6</span></div><div class="casAuthors">McCoull, William; Cheung, Tony; Anderson, Erica; Barton, Peter; Burgess, Jonathan; Byth, Kate; Cao, Qing; Castaldi, M. Paola; Chen, Huawei; Chiarparin, Elisabetta; Carbajo, Rodrigo J.; Code, Erin; Cowan, Suzanna; Davey, Paul R.; Ferguson, Andrew D.; Fillery, Shaun; Fuller, Nathan O.; Gao, Ning; Hargreaves, David; Howard, Martin R.; Hu, Jun; Kawatkar, Aarti; Kemmitt, Paul D.; Leo, Elisabetta; Molina, Daniel M.; O'Connell, Nichole; Petteruti, Philip; Rasmusson, Timothy; Raubo, Piotr; Rawlins, Philip B.; Ricchiuto, Piero; Robb, Graeme R.; Schenone, Monica; Waring, Michael J.; Zinda, Michael; Fawell, Stephen; Wilson, David M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3131-3141</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">B-cell lymphoma 6 (BCL6) inhibition is a promising mechanism for treating hematol. cancers but high quality chem. probes are necessary to evaluate its therapeutic potential.  Here we report potent BCL6 inhibitors that demonstrate cellular target engagement and exhibit exquisite selectivity for BCL6 based on mass spectrometry analyses following chem. proteomic pull down.  Importantly, a proteolysis-targeting chimera, 4-((R)-25-Chloro-41-methyl-42-oxo-41,42,43,44-tetrahydro-5,8-dioxa-3-aza-4(6,8)-quinolina2(2,4)-pyrimidina-1(1,2)-piperazinacyclononaphane-14-yl)-N-(3-(2-(2-((2-((S)-2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)ethoxy)ethoxy)propyl)-4-oxobutanamide trifluoroacetate, (PROTAC, 15) was also developed and shown to significantly degrade BCL6 in a no. of diffuse large B-cell lymphoma (DLBCL) cell lines, but neither BCL6 inhibition nor degrdn. selectively induced marked phenotypic response.  To investigate, we monitored PROTAC directed BCL6 degrdn. in DLBCL OCI-Ly1 cells by immunofluorescence and discovered a residual BCL6 population.  Anal. of subcellular fractions also showed incomplete BCL6 degrdn. in all fractions despite having measurable PROTAC concns., together providing a rationale for the weak antiproliferative response seen with both BCL6 inhibitor and degrader.  In summary, we have developed potent and selective BCL6 inhibitors and a BCL6 PROTAC that effectively degraded BCL6, but both modalities failed to induce a significant phenotypic response in DLBCL despite achieving cellular concns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEe9aUjqcaHLVg90H21EOLACvtfcHk0lgMw2UvkhE6Qg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFaisrnF&md5=3d2c2d8acf143f05babee6ebdabddc8c</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facschembio.8b00698&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.8b00698%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoull%26aufirst%3DW.%26aulast%3DCheung%26aufirst%3DT.%26aulast%3DAnderson%26aufirst%3DE.%26aulast%3DBarton%26aufirst%3DP.%26aulast%3DBurgess%26aufirst%3DJ.%26aulast%3DByth%26aufirst%3DK.%26aulast%3DCao%26aufirst%3DQ.%26aulast%3DCastaldi%26aufirst%3DM.%2BP.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DChiarparin%26aufirst%3DE.%26aulast%3DCarbajo%26aufirst%3DR.%2BJ.%26aulast%3DCode%26aufirst%3DE.%26aulast%3DCowan%26aufirst%3DS.%26aulast%3DDavey%26aufirst%3DP.%2BR.%26aulast%3DFerguson%26aufirst%3DA.%2BD.%26aulast%3DFillery%26aufirst%3DS.%26aulast%3DFuller%26aufirst%3DN.%2BO.%26aulast%3DGao%26aufirst%3DN.%26aulast%3DHargreaves%26aufirst%3DD.%26aulast%3DHoward%26aufirst%3DM.%2BR.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DKawatkar%26aufirst%3DA.%26aulast%3DKemmitt%26aufirst%3DP.%2BD.%26aulast%3DLeo%26aufirst%3DE.%26aulast%3DMolina%26aufirst%3DD.%2BM.%26aulast%3DO%25E2%2580%2599Connell%26aufirst%3DN.%26aulast%3DPetteruti%26aufirst%3DP.%26aulast%3DRasmusson%26aufirst%3DT.%26aulast%3DRaubo%26aufirst%3DP.%26aulast%3DRawlins%26aufirst%3DP.%2BB.%26aulast%3DRicchiuto%26aufirst%3DP.%26aulast%3DRobb%26aufirst%3DG.%2BR.%26aulast%3DSchenone%26aufirst%3DM.%26aulast%3DWaring%26aufirst%3DM.%2BJ.%26aulast%3DZinda%26aufirst%3DM.%26aulast%3DFawell%26aufirst%3DS.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26atitle%3DDevelopment%2520of%2520a%2520novel%2520B-cell%2520lymphoma%25206%2520%2528BCL6%2529%2520PROTAC%2520to%2520provide%2520insight%2520into%2520small%2520molecule%2520targeting%2520of%2520BCL6%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26issue%3D11%26spage%3D3131%26epage%3D3141%26doi%3D10.1021%2Facschembio.8b00698" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kerres, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steurer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlager, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caligiuri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dank, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engen, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ettmayer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischerauer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flotzinger, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerlach, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstberger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmaschitz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greb, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heyes, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iacob, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kölle, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamarre, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lancia, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayr, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mischerikow, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mück, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peinsipp, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petermann, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiser, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudolph, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumpel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salomon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scharn, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnitzer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrenk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schweifer, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traxler, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varecka, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voss, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss-Puxbaum, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoephel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraut, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConnell, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koegl, M.</span></span> <span> </span><span class="NLM_article-title">Chemically induced degradation of the oncogenic transcription factor BCL6</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>20</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">2860</span>– <span class="NLM_lpage">2875</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2017.08.081</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10.1016%2Fj.celrep.2017.08.081" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=28930682" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFeqsrfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2017&pages=2860-2875&issue=12&author=N.+Kerresauthor=S.+Steurerauthor=S.+Schlagerauthor=G.+Baderauthor=H.+Bergerauthor=M.+Caligiuriauthor=C.+Dankauthor=J.+R.+Engenauthor=P.+Ettmayerauthor=B.+Fischerauerauthor=G.+Flotzingerauthor=D.+Gerlachauthor=T.+Gerstbergerauthor=T.+Gmaschitzauthor=P.+Grebauthor=B.+Hanauthor=E.+Heyesauthor=R.+E.+Iacobauthor=D.+Kesslerauthor=H.+K%C3%B6lleauthor=L.+Lamarreauthor=D.+R.+Lanciaauthor=S.+Lucasauthor=M.+Mayerauthor=K.+Mayrauthor=N.+Mischerikowauthor=K.+M%C3%BCckauthor=C.+Peinsippauthor=O.+Petermannauthor=U.+Reiserauthor=D.+Rudolphauthor=K.+Rumpelauthor=C.+Salomonauthor=D.+Scharnauthor=R.+Schnitzerauthor=A.+Schrenkauthor=N.+Schweiferauthor=D.+Thompsonauthor=E.+Traxlerauthor=R.+Vareckaauthor=T.+Vossauthor=A.+Weiss-Puxbaumauthor=S.+Winklerauthor=X.+Zhengauthor=A.+Zoephelauthor=N.+Krautauthor=D.+McConnellauthor=M.+Pearsonauthor=M.+Koegl&title=Chemically+induced+degradation+of+the+oncogenic+transcription+factor+BCL6&doi=10.1016%2Fj.celrep.2017.08.081"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Chemically Induced Degradation of the Oncogenic Transcription Factor BCL6</span></div><div class="casAuthors">Kerres, Nina; Steurer, Steffen; Schlager, Stefanie; Bader, Gerd; Berger, Helmut; Caligiuri, Maureen; Dank, Christian; Engen, John R.; Ettmayer, Peter; Fischerauer, Bernhard; Flotzinger, Gerlinde; Gerlach, Daniel; Gerstberger, Thomas; Gmaschitz, Teresa; Greb, Peter; Han, Bingsong; Heyes, Elizabeth; Iacob, Roxana E.; Kessler, Dirk; Koelle, Heike; Lamarre, Lyne; Lancia, David R.; Lucas, Simon; Mayer, Moriz; Mayr, Katharina; Mischerikow, Nikolai; Mueck, Katja; Peinsipp, Christoph; Petermann, Oliver; Reiser, Ulrich; Rudolph, Dorothea; Rumpel, Klaus; Salomon, Carina; Scharn, Dirk; Schnitzer, Renate; Schrenk, Andreas; Schweifer, Norbert; Thompson, Diane; Traxler, Elisabeth; Varecka, Roland; Voss, Tilman; Weiss-Puxbaum, Alexander; Winkler, Sandra; Zheng, Xiaozhang; Zoephel, Andreas; Kraut, Norbert; McConnell, Darryl; Pearson, Mark; Koegl, Manfred</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2860-2875</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The transcription factor BCL6 is a known driver of oncogenesis in lymphoid malignancies, including diffuse large B cell lymphoma (DLBCL).  Disruption of its interaction with transcriptional repressors interferes with the oncogenic effects of BCL6.  We used a structure-based drug design to develop highly potent compds. that block this interaction.  A subset of these inhibitors also causes rapid ubiquitylation and degrdn. of BCL6 in cells.  These compds. display significantly stronger induction of expression of BCL6-repressed genes and anti-proliferative effects than compds. that merely inhibit co-repressor interactions.  This work establishes the BTB domain as a highly druggable structure, paving the way for the use of other members of this protein family as drug targets.  The magnitude of effects elicited by this class of BCL6-degrading compds. exceeds that of our equipotent non-degrading inhibitors, suggesting opportunities for the development of BCL6-based lymphoma therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXfmjxa_7LnrVg90H21EOLACvtfcHk0lgMw2UvkhE6Qg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFeqsrfK&md5=45b538db6643abbd5b8f3fa04770a772</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2017.08.081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2017.08.081%26sid%3Dliteratum%253Aachs%26aulast%3DKerres%26aufirst%3DN.%26aulast%3DSteurer%26aufirst%3DS.%26aulast%3DSchlager%26aufirst%3DS.%26aulast%3DBader%26aufirst%3DG.%26aulast%3DBerger%26aufirst%3DH.%26aulast%3DCaligiuri%26aufirst%3DM.%26aulast%3DDank%26aufirst%3DC.%26aulast%3DEngen%26aufirst%3DJ.%2BR.%26aulast%3DEttmayer%26aufirst%3DP.%26aulast%3DFischerauer%26aufirst%3DB.%26aulast%3DFlotzinger%26aufirst%3DG.%26aulast%3DGerlach%26aufirst%3DD.%26aulast%3DGerstberger%26aufirst%3DT.%26aulast%3DGmaschitz%26aufirst%3DT.%26aulast%3DGreb%26aufirst%3DP.%26aulast%3DHan%26aufirst%3DB.%26aulast%3DHeyes%26aufirst%3DE.%26aulast%3DIacob%26aufirst%3DR.%2BE.%26aulast%3DKessler%26aufirst%3DD.%26aulast%3DK%25C3%25B6lle%26aufirst%3DH.%26aulast%3DLamarre%26aufirst%3DL.%26aulast%3DLancia%26aufirst%3DD.%2BR.%26aulast%3DLucas%26aufirst%3DS.%26aulast%3DMayer%26aufirst%3DM.%26aulast%3DMayr%26aufirst%3DK.%26aulast%3DMischerikow%26aufirst%3DN.%26aulast%3DM%25C3%25BCck%26aufirst%3DK.%26aulast%3DPeinsipp%26aufirst%3DC.%26aulast%3DPetermann%26aufirst%3DO.%26aulast%3DReiser%26aufirst%3DU.%26aulast%3DRudolph%26aufirst%3DD.%26aulast%3DRumpel%26aufirst%3DK.%26aulast%3DSalomon%26aufirst%3DC.%26aulast%3DScharn%26aufirst%3DD.%26aulast%3DSchnitzer%26aufirst%3DR.%26aulast%3DSchrenk%26aufirst%3DA.%26aulast%3DSchweifer%26aufirst%3DN.%26aulast%3DThompson%26aufirst%3DD.%26aulast%3DTraxler%26aufirst%3DE.%26aulast%3DVarecka%26aufirst%3DR.%26aulast%3DVoss%26aufirst%3DT.%26aulast%3DWeiss-Puxbaum%26aufirst%3DA.%26aulast%3DWinkler%26aufirst%3DS.%26aulast%3DZheng%26aufirst%3DX.%26aulast%3DZoephel%26aufirst%3DA.%26aulast%3DKraut%26aufirst%3DN.%26aulast%3DMcConnell%26aufirst%3DD.%26aulast%3DPearson%26aufirst%3DM.%26aulast%3DKoegl%26aufirst%3DM.%26atitle%3DChemically%2520induced%2520degradation%2520of%2520the%2520oncogenic%2520transcription%2520factor%2520BCL6%26jtitle%3DCell%2520Rep.%26date%3D2017%26volume%3D20%26issue%3D12%26spage%3D2860%26epage%3D2875%26doi%3D10.1016%2Fj.celrep.2017.08.081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pierrat, O. A.</span>; <span class="NLM_string-name">Liu, M.</span>; <span class="NLM_string-name">Rodrigues, M.J.</span>; <span class="NLM_string-name">Collie, G. W.</span>; <span class="NLM_string-name">Le Bihan, Y-V</span>; <span class="NLM_string-name">Shetty, K. N.</span>; <span class="NLM_string-name">McAndrew, P. C.</span>; <span class="NLM_string-name">Richards, M.</span>; <span class="NLM_string-name">Stubbs, M.</span>; <span class="NLM_string-name">Rowlands, M. G.</span>; <span class="NLM_string-name">Yahya, N.</span>; <span class="NLM_string-name">Pickard, L.</span>; <span class="NLM_string-name">Shehu, E.</span>; <span class="NLM_string-name">Talbot, R.</span>; <span class="NLM_string-name">Bellenie, B. R.</span>; <span class="NLM_string-name">Davis, O. A.</span>; <span class="NLM_string-name">Varela, A.</span>; <span class="NLM_string-name">Cheung, K-M. J.</span>; <span class="NLM_string-name">Drouin, L.</span>; <span class="NLM_string-name">Innocenti, P.</span>; <span class="NLM_string-name">Woodward, H.</span>; <span class="NLM_string-name">Lloyd, M. G.</span>; <span class="NLM_string-name">Huckvale, R.</span>; <span class="NLM_string-name">Broccatelli, F.</span>; <span class="NLM_string-name">Carter, M.</span>; <span class="NLM_string-name">Galiwango, D.</span>; <span class="NLM_string-name">Hayes, A.</span>; <span class="NLM_string-name">Raynaud, F. I.</span>; <span class="NLM_string-name">Bryant, C.</span>; <span class="NLM_string-name">Whittaker, S.</span>; <span class="NLM_string-name">Hoelder, S.</span>; <span class="NLM_string-name">Burke, R.</span>; <span class="NLM_string-name">van Montfort, R. L. M.</span></span> Unpublished results.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Pierrat%2C+O.+A.%3B+Liu%2C+M.%3B+Rodrigues%2C+M.J.%3B+Collie%2C+G.+W.%3B+Le+Bihan%2C+Y-V%3B+Shetty%2C+K.+N.%3B+McAndrew%2C+P.+C.%3B+Richards%2C+M.%3B+Stubbs%2C+M.%3B+Rowlands%2C+M.+G.%3B+Yahya%2C+N.%3B+Pickard%2C+L.%3B+Shehu%2C+E.%3B+Talbot%2C+R.%3B+Bellenie%2C+B.+R.%3B+Davis%2C+O.+A.%3B+Varela%2C+A.%3B+Cheung%2C+K-M.+J.%3B+Drouin%2C+L.%3B+Innocenti%2C+P.%3B+Woodward%2C+H.%3B+Lloyd%2C+M.+G.%3B+Huckvale%2C+R.%3B+Broccatelli%2C+F.%3B+Carter%2C+M.%3B+Galiwango%2C+D.%3B+Hayes%2C+A.%3B+Raynaud%2C+F.+I.%3B+Bryant%2C+C.%3B+Whittaker%2C+S.%3B+Hoelder%2C+S.%3B+Burke%2C+R.%3B+van+Montfort%2C+R.+L.+M.+Unpublished+results."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DPierrat%26aufirst%3DO.%2BA." title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hill, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. J.</span></span> <span> </span><span class="NLM_article-title">Getting physical in drug discovery: a contemporary perspective on solubility and hydrophobicity</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">15–16</span>),  <span class="NLM_fpage">648</span>– <span class="NLM_lpage">655</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2010.05.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10.1016%2Fj.drudis.2010.05.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=20570751" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVeksb3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2010&pages=648-655&issue=15%E2%80%9316&author=A.+P.+Hillauthor=R.+J.+Young&title=Getting+physical+in+drug+discovery%3A+a+contemporary+perspective+on+solubility+and+hydrophobicity&doi=10.1016%2Fj.drudis.2010.05.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Getting physical in drug discovery: a contemporary perspective on solubility and hydrophobicity</span></div><div class="casAuthors">Hill, Alan P.; Young, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">15/16</span>),
    <span class="NLM_cas:pages">648-655</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Suboptimal phys. properties have been identified as a particular shortcoming of compds. in contemporary drug discovery, contributing to high attrition levels.  An anal. of the relationship between hydrophobicity (calcd. and measured) and ∼100 k measured kinetic soly. values has been undertaken.  In line with the General Soly. Equation, ests. of hydrophobicity, particularly ACD c log D pH7.4, give a useful indication of the likely soly. classification of particular mols.  Taking ACD c log D pH7.4 values together with the no. of arom. rings in a given mol. provides enhanced prediction.  The Soly. Forecast Index' (SFI = c log D pH7.4 + #Ar) is proposed as a simple, yet effective, guide to predicting soly.  Moreover, anal. of measured distribution/partition coeff. values highlighted statistically significant shortcomings in the applicability of octanol/water as a model system for hydrophobicity detn. with poorly sol. compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdtQyfNjbFUrVg90H21EOLACvtfcHk0lhWX-xp4lBuwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVeksb3P&md5=7b201081c5586a6ea4b7af00d507d462</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2010.05.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2010.05.016%26sid%3Dliteratum%253Aachs%26aulast%3DHill%26aufirst%3DA.%2BP.%26aulast%3DYoung%26aufirst%3DR.%2BJ.%26atitle%3DGetting%2520physical%2520in%2520drug%2520discovery%253A%2520a%2520contemporary%2520perspective%2520on%2520solubility%2520and%2520hydrophobicity%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2010%26volume%3D15%26issue%3D15%25E2%2580%259316%26spage%3D648%26epage%3D655%26doi%3D10.1016%2Fj.drudis.2010.05.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bissantz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahl, M.</span></span> <span> </span><span class="NLM_article-title">A medicinal chemist’s guide to molecular interactions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span> (<span class="NLM_issue">14</span>),  <span class="NLM_fpage">5061</span>– <span class="NLM_lpage">5084</span>, <span class="refDoi"> DOI: 10.1021/jm100112j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100112j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjvFKjtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5061-5084&issue=14&author=C.+Bissantzauthor=B.+Kuhnauthor=M.+Stahl&title=A+medicinal+chemist%E2%80%99s+guide+to+molecular+interactions&doi=10.1021%2Fjm100112j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">A Medicinal Chemist's Guide to Molecular Interactions</span></div><div class="casAuthors">Bissantz, Caterina; Kuhn, Bernd; Stahl, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5061-5084</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraV8HPbwsihbVg90H21EOLACvtfcHk0lhWX-xp4lBuwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjvFKjtbs%253D&md5=f39cf8700267cdc311c73de4d433bab0</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm100112j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100112j%26sid%3Dliteratum%253Aachs%26aulast%3DBissantz%26aufirst%3DC.%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DA%2520medicinal%2520chemist%25E2%2580%2599s%2520guide%2520to%2520molecular%2520interactions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26issue%3D14%26spage%3D5061%26epage%3D5084%26doi%3D10.1021%2Fjm100112j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bodnarchuk, M. S.</span></span> <span> </span><span class="NLM_article-title">Water, water, everywhere...It’s time to stop and think</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>21</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">1139</span>– <span class="NLM_lpage">1146</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2016.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10.1016%2Fj.drudis.2016.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=27210724" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BC28XoslWltb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=1139-1146&issue=7&author=M.+S.+Bodnarchuk&title=Water%2C+water%2C+everywhere...It%E2%80%99s+time+to+stop+and+think&doi=10.1016%2Fj.drudis.2016.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Water, water, everywhere... It's time to stop and think</span></div><div class="casAuthors">Bodnarchuk, Michael S.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1139-1146</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Despite the numerous methods available for predicting the location and affinity of water mols., there is still a degree of scepticism and reluctance towards using such information within a drug discovery program.  Here, I review some of the most common and popular methods to assess whether these apparent concerns are justified.  I suggest that the field is approaching maturity and that some methods are capable of giving quant. predictions, which are confirmed exptl.  This suggests that water-placement methods should be used more widely to help direct chem. efforts, although more successful examples are required to help validate the techniques.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxfKWV5PXTkrVg90H21EOLACvtfcHk0lhWX-xp4lBuwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XoslWltb4%253D&md5=dc2a72fa764bb19cd0c0ca2921fc974a</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2016.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2016.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DBodnarchuk%26aufirst%3DM.%2BS.%26atitle%3DWater%252C%2520water%252C%2520everywhere...It%25E2%2580%2599s%2520time%2520to%2520stop%2520and%2520think%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2016%26volume%3D21%26issue%3D7%26spage%3D1139%26epage%3D1146%26doi%3D10.1016%2Fj.drudis.2016.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Induced protein degradation: an emerging drug discovery paradigm</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">101</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10.1038%2Fnrd.2016.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=27885283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=101&author=A.+C.+Laiauthor=C.+M.+Crews&title=Induced+protein+degradation%3A+an+emerging+drug+discovery+paradigm&doi=10.1038%2Fnrd.2016.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Induced protein degradation: an emerging drug discovery paradigm</span></div><div class="casAuthors">Lai, Ashton C.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">101-114</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. drug discovery has traditionally focused on occupancy of a binding site that directly affects protein function, and this approach typically precludes targeting proteins that lack such amenable sites.  Furthermore, high systemic drug exposures may be needed to maintain sufficient target inhibition in vivo, increasing the risk of undesirable off-target effects.  Induced protein degrdn. is an alternative approach that is event-driven: upon drug binding, the target protein is tagged for elimination.  Emerging technologies based on proteolysis-targeting chimaeras (PROTACs) that exploit cellular quality control machinery to selectively degrade target proteins are attracting considerable attention in the pharmaceutical industry owing to the advantages they could offer over traditional small-mol. strategies.  These advantages include the potential to reduce systemic drug exposure, the ability to counteract increased target protein expression that often accompanies inhibition of protein function and the potential ability to target proteins that are not currently therapeutically tractable, such as transcription factors, scaffolding and regulatory proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-O7zXGOpXLVg90H21EOLACvtfcHk0lh4m6lHC5842w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP&md5=a01ff43e9b696fe4dc03f0815158a98f</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.211%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DInduced%2520protein%2520degradation%253A%2520an%2520emerging%2520drug%2520discovery%2520paradigm%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D101%26doi%3D10.1038%2Fnrd.2016.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Broccatelli, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hop, C. E. C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, M.</span></span> <span> </span><span class="NLM_article-title">Strategies to optimize drug half-life in lead candidate identification</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">221</span>– <span class="NLM_lpage">230</span>, <span class="refDoi"> DOI: 10.1080/17460441.2019.1569625</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10.1080%2F17460441.2019.1569625" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=30675823" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvVCisbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=221-230&issue=3&author=F.+Broccatelliauthor=C.+E.+C.+A.+Hopauthor=M.+Wright&title=Strategies+to+optimize+drug+half-life+in+lead+candidate+identification&doi=10.1080%2F17460441.2019.1569625"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Strategies to optimize drug half-life in lead candidate identification</span></div><div class="casAuthors">Broccatelli, Fabio; Hop, Marcel; Wright, Matthew</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">221-230</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  : The PK optimization of drug candidates is one of the most resource-intensive tasks in pharmaceutical research and development.  With the increasing availability of in silico, in vitro and mechanistic in vivo ADME models, drug discovery scientists have progressively learned to recognize common SAR patterns and engineer data-driven strategies to accelerate the resoln. of ADME issues in lead optimization.  Many of these strategies gravitate toward the concept of drug-likeness, which defines a no. of optimal holistic physicochem. parameters (such as lipophilicity) that idealized oral drugs possess.  Herein, the authors discuss the interplay of lipophilicity with in vitro and in vivo ADME data in order to refine existing thought around drug half-life optimization.  Strategies to prolong the half-life of oral drugs via formulation are beyond the scope of this review.  Optimizing active properties such as potency, selectivity, and intrinsic metabolic clearance is an unambiguously beneficial strategy for small mols. within or beyond the Lipinski rule of five chem. space.  The data that we present in this work suggests that emphasis should be primarily placed on optimizing active properties such as potency, selectivity, and metabolic stability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrWxmcwdPTmLVg90H21EOLACvtfcHk0lh4m6lHC5842w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvVCisbo%253D&md5=0a5b590ffdb385a2f36205874345a72b</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1080%2F17460441.2019.1569625&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F17460441.2019.1569625%26sid%3Dliteratum%253Aachs%26aulast%3DBroccatelli%26aufirst%3DF.%26aulast%3DHop%26aufirst%3DC.%2BE.%2BC.%2BA.%26aulast%3DWright%26aufirst%3DM.%26atitle%3DStrategies%2520to%2520optimize%2520drug%2520half-life%2520in%2520lead%2520candidate%2520identification%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2019%26volume%3D14%26issue%3D3%26spage%3D221%26epage%3D230%26doi%3D10.1080%2F17460441.2019.1569625" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berberich-Siebelt, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, R.</span></span> <span> </span><span class="NLM_article-title">The feedback loop of LITAF and BCL6 is involved in regulating apoptosis in B cell non-Hodgkin’s-lymphoma</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">47</span>),  <span class="NLM_fpage">77444</span>– <span class="NLM_lpage">77456</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.12680</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10.18632%2Foncotarget.12680" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=27764808" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A280%3ADC%252BC2srksFWmug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=77444-77456&issue=47&author=Y.+Shiauthor=Y.+Kuaiauthor=L.+Leiauthor=Y.+Wengauthor=F.+Berberich-Siebeltauthor=X.+Zhangauthor=J.+Wangauthor=Y.+Zhouauthor=X.+Jiangauthor=G.+Renauthor=H.+Panauthor=Z.+Maoauthor=R.+Zhou&title=The+feedback+loop+of+LITAF+and+BCL6+is+involved+in+regulating+apoptosis+in+B+cell+non-Hodgkin%E2%80%99s-lymphoma&doi=10.18632%2Foncotarget.12680"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">The feedback loop of LITAF and BCL6 is involved in regulating apoptosis in B cell non-Hodgkin's-lymphoma</span></div><div class="casAuthors">Shi Yaoyao; Kuai Yue; Lei Lizhen; Weng Yuanyuan; Jiang Xin; Mao Zhengrong; Zhou Ren; Shi Yaoyao; Berberich-Siebelt Friederike; Zhang Xinxia; Pan Hongyang; Wang Jinjie; Zhou Yuan; Ren Guoping</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">77444-77456</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Dysregulation of the apoptotic pathway is widely recognized as a key step in lymphomagenesis.  Notably, LITAF was initially identified as a p53-inducible gene, subsequently implicated as a tumor suppressor.  Our previous study also showed LITAF to be methylated in 89.5% B-NHL samples.  Conversely, deregulated expression of BCL6 is a pathogenic event in many lymphomas.  Interestingly, our study found an oppositional expression of LITAF and BCL6 in B-NHL.  In addition, LITAF was recently identified as a novel target gene of BCL6.  Therefore, we sought to explore the feedback loop between LITAF and BCL6 in B-NHL.  Here, our data for the first time show that LITAF can repress expression of BCL6 by binding to Region A (-87 to +65) containing a putative LITAF-binding motif (CTCCC) within the BCL6 promoter.  Furthermore, the regulation of BCL6 targets ( PRDM1 or c-Myc) by LITAF may be associated with B-cell differentiation.  Results also demonstrate that ectopic expression of LITAF induces cell apoptosis, activated by releasing cytochrome c, cleaving PARP and caspase 3 in B-NHL cells whereas knockdown of LITAF robustly protected cells from apoptosis.  Interestingly, BCL6, in turn, could reverse cell apoptosis mediated by LITAF.  Collectively, our findings provide a novel apoptotic regulatory pathway in which LITAF, as a transcription factor, inhibits the expression of BCL6, which leads to activation of the intrinsic mitochondrial pathway and tumor apoptosis.  Our study is expected to provide a possible biomarker as well as a target for clinical therapies to promote tumor cell apoptosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTvAmjS16coryjiVMCqkKxVfW6udTcc2eb8ZFAoTHMv0rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2srksFWmug%253D%253D&md5=4a71f277ffcfc61c5b2a2c0a781e486b</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.12680&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.12680%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DKuai%26aufirst%3DY.%26aulast%3DLei%26aufirst%3DL.%26aulast%3DWeng%26aufirst%3DY.%26aulast%3DBerberich-Siebelt%26aufirst%3DF.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DX.%26aulast%3DRen%26aufirst%3DG.%26aulast%3DPan%26aufirst%3DH.%26aulast%3DMao%26aufirst%3DZ.%26aulast%3DZhou%26aufirst%3DR.%26atitle%3DThe%2520feedback%2520loop%2520of%2520LITAF%2520and%2520BCL6%2520is%2520involved%2520in%2520regulating%2520apoptosis%2520in%2520B%2520cell%2520non-Hodgkin%25E2%2580%2599s-lymphoma%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26issue%3D47%26spage%3D77444%26epage%3D77456%26doi%3D10.18632%2Foncotarget.12680" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixit, V. M.</span></span> <span> </span><span class="NLM_article-title">Drugging the undruggables: exploring the ubiquitin system for drug development</span>. <i>Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">484</span>– <span class="NLM_lpage">498</span>, <span class="refDoi"> DOI: 10.1038/cr.2016.31</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10.1038%2Fcr.2016.31" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=27002218" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BC28XkvVyhs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=484-498&issue=4&author=X.+Huangauthor=V.+M.+Dixit&title=Drugging+the+undruggables%3A+exploring+the+ubiquitin+system+for+drug+development&doi=10.1038%2Fcr.2016.31"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Drugging the undruggables: exploring the ubiquitin system for drug development</span></div><div class="casAuthors">Huang, Xiaodong; Dixit, Vishva M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">484-498</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">NPG Nature Asia-Pacific</span>)
        </div><div class="casAbstract">Dynamic modulation of protein levels is tightly controlled in response to physiol. cues.  In mammalian cells, much of the protein degrdn. is carried out by the ubiquitin-proteasome system (UPS).  Similar to kinases, components of the ubiquitin system are often dysregulated, leading to a variety of diseases, including cancer and neurodegeneration, making them attractive drug targets.  However, so far there are only a handful of drugs targeting the ubiquitin system that have been approved by the FDA.  Here, we review possible therapeutic intervention nodes in the ubiquitin system, analyze the challenges, and highlight the most promising strategies to target the UPS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnGVCksLBzN7Vg90H21EOLACvtfcHk0lj13EhCy7-EpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XkvVyhs70%253D&md5=5bda209d50a9179aa4e9c17e7fd491ad</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fcr.2016.31&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcr.2016.31%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DDixit%26aufirst%3DV.%2BM.%26atitle%3DDrugging%2520the%2520undruggables%253A%2520exploring%2520the%2520ubiquitin%2520system%2520for%2520drug%2520development%26jtitle%3DCell%2520Res.%26date%3D2016%26volume%3D26%26issue%3D4%26spage%3D484%26epage%3D498%26doi%3D10.1038%2Fcr.2016.31" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Collins, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldwell, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chopra, R.</span></span> <span> </span><span class="NLM_article-title">Chemical approaches to targeted protein degradation through modulation of the ubiquitin–proteasome pathway</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>474</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">1127</span>– <span class="NLM_lpage">1147</span>, <span class="refDoi"> DOI: 10.1042/BCJ20160762</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10.1042%2FBCJ20160762" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=28298557" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvVygt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=474&publication_year=2017&pages=1127-1147&issue=7&author=I.+Collinsauthor=H.+Wangauthor=J.+J.+Caldwellauthor=R.+Chopra&title=Chemical+approaches+to+targeted+protein+degradation+through+modulation+of+the+ubiquitin%E2%80%93proteasome+pathway&doi=10.1042%2FBCJ20160762"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical approaches to targeted protein degradation through modulation of the ubiquitin-proteasome pathway</span></div><div class="casAuthors">Collins, Ian; Wang, Hannah; Caldwell, John J.; Chopra, Raj</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">474</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1127-1147</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">Manipulation of the ubiquitin-proteasome system to achieve targeted degrdn. of proteins within cells using chem. tools and drugs has the potential to transform pharmacol. and therapeutic approaches in cancer and other diseases.  An increased understanding of the mol. mechanism of thalidomide and its analogs following their clin. use has unlocked small-mol. modulation of the substrate specificity of the E3 ligase cereblon (CRBN), which in turn has resulted in the advancement of new immunomodulatory drugs (IMiDs) into the clinic.  The degrdn. of multiple context-specific proteins by these pleiotropic small mols. provides a means to uncover new cell biol. and to generate future drug mols. against currently undruggable targets.  In parallel, the development of larger bifunctional mols. that bring together highly specific protein targets in complexes with CRBN, von Hippel-Lindau, or other E3 ligases to promote ubiquitin-dependent degrdn. has progressed to generate selective chem. compds. with potent effects in cells and in vivo models, providing valuable tools for biol. target validation and with future potential for therapeutic use.  In this review, we survey recent breakthroughs achieved in these two complementary methods and the discovery of new modes of direct and indirect engagement of target proteins with the proteasome.  We discuss the exptl. characterization that validates the use of mols. that promote protein degrdn. as chem. tools, the preclin. and clin. examples disclosed to date, and the future prospects for this exciting area of chem. biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMkoPeMHzjTbVg90H21EOLACvtfcHk0lgGi6aneBJGXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvVygt7Y%253D&md5=8c9ccac22f5088d776b81203dc8ebf2b</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1042%2FBCJ20160762&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBCJ20160762%26sid%3Dliteratum%253Aachs%26aulast%3DCollins%26aufirst%3DI.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DCaldwell%26aufirst%3DJ.%2BJ.%26aulast%3DChopra%26aufirst%3DR.%26atitle%3DChemical%2520approaches%2520to%2520targeted%2520protein%2520degradation%2520through%2520modulation%2520of%2520the%2520ubiquitin%25E2%2580%2593proteasome%2520pathway%26jtitle%3DBiochem.%2520J.%26date%3D2017%26volume%3D474%26issue%3D7%26spage%3D1127%26epage%3D1147%26doi%3D10.1042%2FBCJ20160762" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mares, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, I. E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campos, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miah, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulholland, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Routly, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinn, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergamini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faelth-Savitski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willard, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flanagan, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casillas, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Votta, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">den Besten, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Famm, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruidenier, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harling, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Catalytic in vivo protein knockdown by small-molecule PROTACs</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">611</span>– <span class="NLM_lpage">617</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1858</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10.1038%2Fnchembio.1858" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=26075522" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=611-617&issue=8&author=D.+P.+Bondesonauthor=A.+Maresauthor=I.+E.+D.+Smithauthor=E.+Koauthor=S.+Camposauthor=A.+H.+Miahauthor=K.+E.+Mulhollandauthor=N.+Routlyauthor=D.+L.+Buckleyauthor=J.+L.+Gustafsonauthor=N.+Zinnauthor=P.+Grandiauthor=S.+Shimamuraauthor=G.+Bergaminiauthor=M.+Faelth-Savitskiauthor=M.+Bantscheffauthor=C.+Coxauthor=D.+A.+Gordonauthor=R.+R.+Willardauthor=J.+J.+Flanaganauthor=L.+N.+Casillasauthor=B.+J.+Vottaauthor=W.+den+Bestenauthor=K.+Fammauthor=L.+Kruidenierauthor=P.+S.+Carterauthor=J.+D.+Harlingauthor=I.+Churcherauthor=C.+M.+Crews&title=Catalytic+in+vivo+protein+knockdown+by+small-molecule+PROTACs&doi=10.1038%2Fnchembio.1858"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic in vivo protein knockdown by small-molecule PROTACs</span></div><div class="casAuthors">Bondeson, Daniel P.; Mares, Alina; Smith, Ian E. D.; Ko, Eunhwa; Campos, Sebastien; Miah, Afjal H.; Mulholland, Katie E.; Routly, Natasha; Buckley, Dennis L.; Gustafson, Jeffrey L.; Zinn, Nico; Grandi, Paola; Shimamura, Satoko; Bergamini, Giovanna; Faelth-Savitski, Maria; Bantscheff, Marcus; Cox, Carly; Gordon, Deborah A.; Willard, Ryan R.; Flanagan, John J.; Casillas, Linda N.; Votta, Bartholomew J.; den Besten, Willem; Famm, Kristoffer; Kruidenier, Laurens; Carter, Paul S.; Harling, John D.; Churcher, Ian; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">611-617</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The current predominant therapeutic paradigm is based on maximizing drug-receptor occupancy to achieve clin. benefit.  This strategy, however, generally requires excessive drug concns. to ensure sufficient occupancy, often leading to adverse side effects.  Here, we describe major improvements to the proteolysis targeting chimeras (PROTACs) method, a chem. knockdown strategy in which a heterobifunctional mol. recruits a specific protein target to an E3 ubiquitin ligase, resulting in the target's ubiquitination and degrdn.  These compds. behave catalytically in their ability to induce the ubiquitination of super-stoichiometric quantities of proteins, providing efficacy that is not limited by equil. occupancy.  We present two PROTACs that are capable of specifically reducing protein levels by >90% at nanomolar concns.  In addn., mouse studies indicate that they provide broad tissue distribution and knockdown of the targeted protein in tumor xenografts.  Together, these data demonstrate a protein knockdown system combining many of the favorable properties of small-mol. agents with the potent protein knockdown of RNAi and CRISPR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonhnajD3CxS7Vg90H21EOLACvtfcHk0lgGi6aneBJGXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP&md5=31eb8756c5661f78ab4527a50dc1e251</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1858&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1858%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DMares%26aufirst%3DA.%26aulast%3DSmith%26aufirst%3DI.%2BE.%2BD.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DCampos%26aufirst%3DS.%26aulast%3DMiah%26aufirst%3DA.%2BH.%26aulast%3DMulholland%26aufirst%3DK.%2BE.%26aulast%3DRoutly%26aufirst%3DN.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DGustafson%26aufirst%3DJ.%2BL.%26aulast%3DZinn%26aufirst%3DN.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DShimamura%26aufirst%3DS.%26aulast%3DBergamini%26aufirst%3DG.%26aulast%3DFaelth-Savitski%26aufirst%3DM.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DCox%26aufirst%3DC.%26aulast%3DGordon%26aufirst%3DD.%2BA.%26aulast%3DWillard%26aufirst%3DR.%2BR.%26aulast%3DFlanagan%26aufirst%3DJ.%2BJ.%26aulast%3DCasillas%26aufirst%3DL.%2BN.%26aulast%3DVotta%26aufirst%3DB.%2BJ.%26aulast%3Dden%2BBesten%26aufirst%3DW.%26aulast%3DFamm%26aufirst%3DK.%26aulast%3DKruidenier%26aufirst%3DL.%26aulast%3DCarter%26aufirst%3DP.%2BS.%26aulast%3DHarling%26aufirst%3DJ.%2BD.%26aulast%3DChurcher%26aufirst%3DI.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DCatalytic%2520in%2520vivo%2520protein%2520knockdown%2520by%2520small-molecule%2520PROTACs%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26issue%3D8%26spage%3D611%26epage%3D617%26doi%3D10.1038%2Fnchembio.1858" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neklesa, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tae, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundberg, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stagg, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonnell, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Small-molecule-mediated degradation of the androgen receptor through hydrophobic tagging</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span> (<span class="NLM_issue">33</span>),  <span class="NLM_fpage">9659</span>– <span class="NLM_lpage">9662</span>, <span class="refDoi"> DOI: 10.1002/anie.201503720</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=10.1002%2Fanie.201503720" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVegsbvO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=9659-9662&issue=33&author=J.+L.+Gustafsonauthor=T.+K.+Neklesaauthor=C.+S.+Coxauthor=A.+G.+Rothauthor=D.+L.+Buckleyauthor=H.+S.+Taeauthor=T.+B.+Sundbergauthor=D.+B.+Staggauthor=J.+Hinesauthor=D.+P.+McDonnellauthor=J.+D.+Norrisauthor=C.+M.+Crews&title=Small-molecule-mediated+degradation+of+the+androgen+receptor+through+hydrophobic+tagging&doi=10.1002%2Fanie.201503720"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule-Mediated Degradation of the Androgen Receptor Through Hydrophobic Tagging</span></div><div class="casAuthors">Gustafson, Jeffrey L.; Neklesa, Taavi K.; Cox, Carly S.; Roth, Anke G.; Buckley, Dennis L.; Tae, Hyun Seop; Sundberg, Thomas B.; Stagg, D. Blake; Hines, John; McDonnell, Donald P.; Norris, John D.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">33</span>),
    <span class="NLM_cas:pages">9659-9662</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Androgen receptor (AR)-dependent transcription is a major driver of prostate tumor cell proliferation.  Consequently, it is the target of several antitumor chemotherapeutic agents, including the AR antagonist MDV3100/enzalutamide.  Recent studies have shown that a single AR mutation (F876L) converts MDV3100 action from an antagonist to an agonist.  Here we describe the generation of a novel class of selective androgen receptor degraders (SARDs) to address this resistance mechanism.  Mols. contg. hydrophobic degrons linked to small-mol. AR ligands induce AR degrdn., reduce expression of AR target genes and inhibit proliferation in androgen-dependent prostate cancer cell lines.  These results suggest that selective AR degrdn. may be an effective therapeutic prostate tumor strategy in the context of AR mutations that confer resistance to second-generation AR antagonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBoEt08CZb-rVg90H21EOLACvtfcHk0lgFpQvzxTjrjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVegsbvO&md5=024225436378fcff0445d3b66696d69e</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1002%2Fanie.201503720&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201503720%26sid%3Dliteratum%253Aachs%26aulast%3DGustafson%26aufirst%3DJ.%2BL.%26aulast%3DNeklesa%26aufirst%3DT.%2BK.%26aulast%3DCox%26aufirst%3DC.%2BS.%26aulast%3DRoth%26aufirst%3DA.%2BG.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DTae%26aufirst%3DH.%2BS.%26aulast%3DSundberg%26aufirst%3DT.%2BB.%26aulast%3DStagg%26aufirst%3DD.%2BB.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DMcDonnell%26aufirst%3DD.%2BP.%26aulast%3DNorris%26aufirst%3DJ.%2BD.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DSmall-molecule-mediated%2520degradation%2520of%2520the%2520androgen%2520receptor%2520through%2520hydrophobic%2520tagging%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D54%26issue%3D33%26spage%3D9659%26epage%3D9662%26doi%3D10.1002%2Fanie.201503720" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOG" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOG','PDB','6TOG'); return false;">PDB: 6TOG</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOF" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOF','PDB','6TOF'); return false;">PDB: 6TOF</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOH','PDB','6TOH'); return false;">PDB: 6TOH</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOO" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOO','PDB','6TOO'); return false;">PDB: 6TOO</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOI" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOI','PDB','6TOI'); return false;">PDB: 6TOI</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOJ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOJ','PDB','6TOJ'); return false;">PDB: 6TOJ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TON" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TON','PDB','6TON'); return false;">PDB: 6TON</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOK','PDB','6TOK'); return false;">PDB: 6TOK</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOM" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOM','PDB','6TOM'); return false;">PDB: 6TOM</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOL','PDB','6TOL'); return false;">PDB: 6TOL</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i78"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b02076">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_78078"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.9b02076?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b02076</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Supplementary experimental details on protein synthesis and purification, assay conditions, PK and PD methods; analytical data for compound <b>1</b>; tables listing statistics and replicate values and crystallographic data; figures showing binding modes, cell proliferation assay data, PD effects, and PK/PD relationship (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b02076/suppl_file/jm9b02076_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b02076/suppl_file/jm9b02076_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b02076/suppl_file/jm9b02076_si_001.pdf">jm9b02076_si_001.pdf (958.64 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b02076/suppl_file/jm9b02076_si_002.csv">jm9b02076_si_002.csv (5.3 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Atomic coordinates and structure factors for the crystal structures of BCL6 with compounds <b>1</b>, <b>4</b>, <b>5</b>, <b>6</b>, <b>11a</b>, <b>11f</b>, <b>17a</b>, <b>23d</b>, <b>25a</b>, and <b>25b</b> can be accessed using PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOM">6TOM</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOF">6TOF</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOG">6TOG</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOH">6TOH</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOO">6TOO</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOI">6TOI</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOJ">6TOJ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOK">6TOK</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TOL">6TOL</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TON">6TON</a> respectively. Authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b02076&amp;pbContext=%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-8%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b02076%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b02076" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                10MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679989a5e8883d42","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
